<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - AcrR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">AcrR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">AcrR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>It should be noted that all plasmids used for the measurement of transcriptional activity , including pNN602 , pNN608 , pNN609 and pNN610 , were true transcriptional fusions , in which the translationof the truncated AcrA or AcrR polypeptide is stopped by the presence of three stop codons in all possible reading frames ( supplied by the plasmid pDC401 , see Ma et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Mol Microbiol 79:1136 -- 1150 Gu R , Li M , Su CC , Long F , Routh MD , Yang F , McDermott G , Yu EW ( 2008 ) Conformational change of the AcrR regulator reveals a possible mechanism of induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>We identiﬁed a high proportion of TFs belonging to the AsnC , ArsR , MarR , TetR/AcrR , BirA , AraC/XylS , Fur , LysR , and PhoU families in bacteria and archaea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>In fact , K46 of QacR and K47 of TetR ( correspond to the first residue , K55 , in helix α4 of AcrR ) have been seen in contact with the phosphate backbones in QacR-IR123 and TetR-tetO ,27 respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Moreover , these ﬁndings raise the question of whether the effect of other MDR regulators , such as AcrR , a repressor of acrA , or the MarAhomologs SoxS and Rob , as well as marA orthologs or marAlike regulators in other clinically relevant pathogens , are also subject to this kind of ﬁne-tuned regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>For the overexpressionof AcrR , strain D1210harbouring the plasmid pHCacrR was induced by 1mM IPTG for 3 h before the A600 reached 0.5 ; the overproduction was confirmed by the SDS-PAGE denaturing gel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Furthermore , AcrR binds to micF with higher affinity than to marRAB and soxRS promoters due to greater conservation of the AcrR motif in micF at +29 to +53 ( 5 ′ - gtCATtCATT-tctg-AATG blue , and Fis in yellow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>As CmeR is lacking the recognition helix α3 , we superimposed the Cα atoms of helices α1 and α2 , between residues 8 and 42 , of AcrR with their corresponding residues in CmeR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Overexpression of AcrR in a background strain of wild-type E. coli and in the OST strain LMB015 ( ΔfadR ΔmarR ; acrR + and ΔfadR ΔmarR acrR + strain , respectively ) significantly increased cell growth in the presence of n-hexane/cyclohex-ane , which attenuated the membrane reduction capacity of the wild-type strain below 50 % of the control level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_10">
Ma , D. , Alberti , M. , Lynch , C. , Nikaido , H. , and Hearst , J.E. ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></label>
<input id="act_similarity_10", type="checkbox">
<div class="ACT_similar">
<li>[ 8 ] Ma , D. , Alberti , M. , Lynch , C. , Nikaido , H. and Hearst , J.E. ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11886757" target="_blank">11886757</a>]<br></li></div>
<li>The upregulation in the ΔacrR strain indicates that the involvement of SoxRS in defense against organic solvents ( Alekshun and Levy 1997 , 1999 ; Demple 1996 ) could be controlled by AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>This suggests that AcrR functions as a transcriptional-repressor of acrA , which is consistent with the findings of Ma et al. ( 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Conversely , in the ΔacrR strain , the expression of all tested genes ( marR , marA , marB , soxR , and soxS ) , other than the constitutive gene rob ( Kwon et al. 2000 ) , increased significantly regardless of the presence or absence of solvent , suggesting that AcrR functions as a transcriptional-repressor of those genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>It appeared contradictory to our inference that AcrAB overproduction in the strain lacking the repressor AcrR ( ΔacrR ) would enhance the OST of E. coli cells ( White et al. 1997 ) , but instead the ΔacrR strain failed to grow at all in the presence of solvent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Second , the prevention of overexpression ( dampening ) of acrAB appears to be achieved by directly increasing the amount of its repressor , AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Lane 1 , no repressor ; lanes 2 , 3 , 4 , and 8 , AcrR protein ( lane 2 , 1.5 pmol ; lane 3 , 4.5 pmol ; lanes 4 and 8 , 13.5 pmol ) ; lanes 5 , 6 , 7 , and 9 , AcrS protein ( lane 5 , 1.5 pmol ; lane 6 , 4.5 pmol ; lanes 7 and 9 , 13.5 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>For instance , the expression of acrAB in E. coli is controlled by repressor protein AcrR , global activator protein MarA , SoxS , Rob and quorum sensing regulator LuxR homologue protein SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>AcrB is regulated by a global transcriptional-activator MarA and a local transcriptional-repressor AcrR [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The expression of AcrAB is controlled by the transcriptional-regulator AcrR , whose open reading frame is located 141-base-pair upstream of the acrAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>To evaluate whether AcrR and AcrS directly regulate acrAB expression , a DNase I footprinting analysis was performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>However , these acrAB induction rates are low ( about twofold increase ) , probably because of the semi-constitutive expression of acrAB caused by the leakiness of AcrR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>However , it does not appear that modulation of acrAB expression in response to these environmental stresses operates through AcrR , MarA , or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>However , the expression of these regulators was not affected by AcrS and AcrR ( data not shown ) , indicating that the acrAB repression by AcrR and AcrS is unlikely to be mediated by MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>Apparently , factor or factors other than AcrR are primarily responsible for the stress induction of acrAB ( and probably acrR as well ) , and AcrR could , therefore , functionas a secondary modulator to prevent the excessiveexpression of acrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Nevertheless , the general stress conditions increased transcription of acrAB in the absence of functional AcrR , and such conditions surprisingly increasedthe transcription of acrR even more strongly than that of acrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>~ Induction Ratio of LacZa 4 % stationary 0.5 M Increased transcription of acrAB , caused by the general stress conditions , is not mediated by MarA or soxs acrAB transcription becomes elevated by the general stress conditions ( Ma et al. , 1995 ) , but this response occurs in the absence of functional AcrR , as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Electrophoretic mobility shift assay demonstrated that AcrR binds directly to the promoter region of acrRAB , marAB , and soxRS , indicating that AcrR acts on global regulators to affect mar-sox-rob regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Our gel mobility shift assays indeed showed that some factor other than AcrR , which could correspond to the regulator responding to global stress conditions , can bind to the promoter region of acrR-acrAB to form band 3 ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>In addition , purified AcrR could bind to the promoter region of soxRS in vitro in the absence of solvent ( Fig. 5 ) , indicating that the OST enhancement resulting from acrR overexpression was mediated by upregulation of soxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>As one can see from Fig. 3B , both the absolute amount and the extent of the induction level of acrR transcription were higher in WZM124 than in the AcrR ' parent strain W4680 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Quantitative real-time PCR analysis of RNA from the wild-type , ΔacrR , and acrR + strains showed that AcrR represses the transcription of marRAB and soxRS , and its own gene cluster , acrRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>AcrR repressed the transcription of marRAB and soxRS by binding to their promoter The effect of AcrR on the transcription of 14 target genes was investigated by quantitative real-time PCR in wild-type , ΔacrR , and acrR + strains under n-hexane/cyclohexane stress ( Table 2 ; Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The induction of acrAB , marRAB , and soxRS following deletion of acrR , and the in vitro binding of purified AcrR to the promoter DNA of each operon , indicate that AcrR controls expression of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Presumably , AcrR represses the transcription of acrAB by directly binding to the inverting 24-bp IR sequence in the promoter region of the efflux operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>AcrR represses both itself and acrAB transcription , although the repression is very weak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>Altogether , these data suggest that AcrR regulates the transcription of the global regulators marRAB and soxRS , in addition to acrRAB , by direct binding to their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The consequence is that transcription of the global regulators , MarA and SoxS , are modulated by repression of AcrR at the predicted AcrR binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>As far as we are aware , the way in which AcrR regulates the transcription of acrAB reveals two novel features ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Such regulators increase the transcription of both acrAB and acrR , and AcrR , whose production is self-repressed , will limit the extent of overexpression of acrA6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Characterization of the multidrug efflux regulator AcrR from Escherichia coli Chih-Chia Su1 , Den ae J. Rutherford2 , and Edward W. Yu1 ,2 , * 1 Department of Bioch USA emistry , Biophysics and Molecular Biology , Iowa State University , Ames , IA 50011 , 2 Department of Physics and Astronomy , Iowa State University , Ames , IA 50011 , USA bstract A The Escherichia coli AcrR represses transcription of the acrB gene , which encodes the multidrug efflux pump AcrB that extrudes a wide variety of toxic compounds , by binding its target operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The KD values for these ligands range from 4.2 to 10.1 μM , suggesting that AcrR is capable of recognizing a wide range of structurally dissimilar toxic compounds as it is in the case of the AcrB multidrug efflux pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_43">
This change in conformation results in the release of AcrR from its operator DNA , and thus allows transcription from cognate promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></label>
<input id="act_similarity_43", type="checkbox">
<div class="ACT_similar">
<li>This change in conformation results in the release of AcrR from its operator DNA , and thus allows transcription from its cognate promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li></div>
<li>This value is in good agreement with the theoretical value of 120.7 kDa for four AcrR molecules bound to the 28-bp IR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The experiment demonstrated that four AcrR molecules are bound to one IR operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The induced conformational change triggered by the bound drug is accompanied with an increase in center-to-center distance between the DNA recognition helices α3 and α3 ′ from 37 Å in the DNA-bound form to 48 Å in the ligandbound form .23 Thus , similar conformational change within two HTH motifs of the AcrR dimer is expected upon DNA and ligand bindings .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The affinities of AcrR to its drugs are in the micromolar region , and the dissociation constants are quite similar to those of QacR [ 25 ] , BmrR [ 26 ] , and TtgV [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Fluorescence polarization assays were also applied to characterize the affinity and specificity of AcrR interaction with three different fluorescent ligands , rhodamine 6G , ethidium and proflavin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>This negative potential suggests that AcrR may be more favorable to-bind neutral and positively charged ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Thus , its function in AcrR is expected to recognize and bind multiple AcrR drugs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Therefore , AcrR can repress its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Fig. 1a illustrates the binding isotherm of AcrR in the presence of 1 nM fluoresceinated DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>( a ) Binding isotherm of AcrR in the presence of 10 mM Na-phosphate ( pH 7.2 ) , 100 mM NaCl , 1 nM fluoresceinated DNA , and 1 μg of poly ( dI-dC ) as non-specific DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Speculative model of AcrR in its DNA-bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The inflection point at an AcrR dimer concentration of 1,000 nM in the presence of 500 nM DNA , indicating the stoichiometry of 2:1 protein-to-DNA ratio .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Band 3 should representa specific protein-DNA complex independent of AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The AcrR binding motif is ( a ) AcrR ( μg ) 0 2.5 13 0 0 2.5 13 0 2.5 13 EBNA 5 protein-DNA complex DNA and increasing amount of AcrR ( cell lysate of AcrR overproducing strain BL21DE3 pMB32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>For drug binding , we fitted these data using a monomeric AcrR as a single structural unit since many of the TetR family members bind ligands in both subunits of the homodimers , although QacR only binds one drug molecule in the dimer [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Hence , the differences in the degree of acrA repression between AcrS and AcrR can be explained in part by the difference in their binding afﬁnities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>In each case , the bound drug was completely buried in the AcrR molecule , and strong interaction between the bound drug and the regulator was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>In the case of Et and Pf , Pf is bound by both apo-AcrR and AcrR-Et in equally affinities , indicating that Et and Pf may exhibit noncompetitive binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>We found that the binding of rhodamine 6G to AcrR is inhibited by the presence of ethidium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>( d ) Hill plot of the data obtained for R6G binding to AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>This KD is about thirteen times of that of R6G in the absence of Et , suggesting the presence of Et severely interferes with the binding of R6G in AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Our experimental results suggest that AcrR is most likely to-bind Et and R6G competitively as the binding affinity of R6G is greatly decreased in the presence of Et .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The binding of AcrR with several ligands , including rhodamine 6G ( R6G ) , ethidium ( Et ) and proflavin ( Pf ) , have been studied and their dissociation constants have been measured , ranging from 4.2 to 10.7 μM .24 How inducing ligands bind to AcrR and modulate the expression of AcrB is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>We then determined the binding affinity of this alanine mutant , E67A , with three different AcrR ligands , Pf , R6G and Et , using fluorescence polarization assays .37,38 Fig. 5 illustrates the binding isotherm of E67A in the presence of 1 μM Pf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The polarization experiments indicated that AcrR binds Pf , with a KD of 14.9 ± 0.9 μM , in the presence of 100 μM Et .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Binding of ligands by AcrR in the presence of 100 μM of Et as determined by fluorescence polarization assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>In fact , many of the AcrR ligands , including R6G and Et , are positive in charge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Thus , AcrR appears to have stronger interaction with Et and Pf , but weaker with R6G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Fluorescence polarization of AcrR with Pf and R6G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Fluorescence polarization assays suggested that AcrR binds this inverting IR in a manner of 2:1 dimeric AcrR-to-IR molar ratio , with the dissociation constant of 20 nM .24 Thus , there is a good chance that the binding of AcrR to IR resembles the QacR-IR1 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Recently , fluorescence polarization experiments have suggested that AcrR binds R6G , Et and Pf in a manner of 1:1 monomer-to-drug molar ratio .24 Thus , the crystal structure , demonstrating two independent ligand-binding pockets in the homodimer , is in good agreement with the fluorescence polarization study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Based on the fluorescence polarization assays , titrations of AcrR to the ligand binding solutions give a stoichiometry of 1:1 monomeric AcrR-to-drug molar ratio .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>We reasoned that when AcrR binds its inducing ligand , the volume of the ligand-binding pocket might need to expand to accommodate the large size of the drug molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>This pocket can accommodate a variety of ligands , which suggests that AcrR is a multidrug binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>crR ligands f A o g in d in B The goal of this experiment was to determine the binding affinities of a variety of AcrR ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Fluorescence polarization assays37 ,38 were used to determine the ligand binding affinities of the mutant AcrR regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The crystal structure provides novel insight into the mechanisms of ligand binding and AcrR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The DNA-binding ability of AcrR is greatly diminished , if not abolished , in WZM124 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The experimental procedures for determining ligand binding affinities of AcrR using the technique of fluorescence polarization were similar to that of the DNA binding assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Thus , it is likely that AcrR induction may be governed by steric repulsion that takes place during drug binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>In comparing the DNA and drug binding modes of AcrR with those of TetR and QacR , AcrR is distinct in that its DNA binding mode is similar to that of QacR , however , it employs the mode of drug binding similar to that of TetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">AcrR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>As CmeR is lacking the recognition helix α3 , we superimposed the Cα atoms of helices α1 and α2 , between residues 8 and 42 , of AcrR with their corresponding residues in CmeR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Sequence alignment of the HTH motif among members of the TetR family indicates that residues L34 , R45 , G46 , A47 , Y49 , W50 , H51 and K55 in AcrR are important for repressoroperator interaction .22 Three of these residues , Y49 , H51 and K55 , are highly conserved , suggesting that these residues may play a major role in the recognition of target DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>According to our speculative model , a mutation on residue R45 should affect the function of the AcrR regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The interactions between two AcrR subunits are extensive , and this dimer-interface is further strengthened by two cross-interface hydrogen bonds between R131 and F180 ′ ; and R162 and Q190 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The dimer is an all-helical protein , and its overall architecture is in agreement with the structures of the homologues TetR ,26,27 QacR ,23,28 CprB ,29 EthR30 ,31 and CmeR .32 Each subunit of AcrR in the dimer is composed of nine helices ( α1-α9 and α1 ′ - α9 ′ , respectively ) and can be divided into two domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>In fact , K46 of QacR and K47 of TetR ( correspond to the first residue , K55 , in helix α4 of AcrR ) have been seen in contact with the phosphate backbones in QacR-IR123 and TetR-tetO ,27 respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Using recombinant AcrR protein that contains a 6xHis tag at the C-terminus , we confirmed that AcrR interacts with the 28 bp DNA , 5 ′ TTTACATACATTTGTGAATGTATGTACC3 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Although the ligand-binding pocket of AcrR is predominantly hydrophobic , the inner surface of the pocket displays a patch of negative potential , created by the completely buried acidic residue E67 , in the electrostatic surface diagram ( Fig. 3b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>This was clearly demonstrated in the cases of BmrR35 ,36 and QacR .28 For AcrR , the crystal structure illustrates that there is a completely buried acidic residue , E67 , located at the center of the drug-binding binding pocket ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>To demonstrate that the buried acidic glutamate , E67 , in AcrR is critical for drug-binding , we mutated this residue to an alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Based on the crystal structure , it is expected that the AcrR dimer binds two ligand molecules , and each subunit of which contribute to bind one ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Recently , fluorescence polarization experiments have suggested that AcrR binds R6G , Et and Pf in a manner of 1:1 monomer-to-drug molar ratio .24 Thus , the crystal structure , demonstrating two independent ligand-binding pockets in the homodimer , is in good agreement with the fluorescence polarization study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>For drug-binding , we fitted these data using a monomeric AcrR as a single structural unit since many of the TetR family members bind ligands in both subunits of the homodimers , although QacR only binds one drug molecule in the dimer [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>As the recent preliminary X-ray diffraction data suggest a dimeric assembly of the AcrR regulator [ 28 ] , thus we believe that each C-terminal ligand-binding site of AcrR ( each subunit of which contains one site ) in the homodimer binds a drug molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>This view shows the large cavity spanning from the side surface ( front ) to subunit interface ( back ) of the C-terminal domain of one subunit of AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>In view of the AcrR crystal structure , the C-terminal α-helical bundle forms a large internal cavity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Superimposition of the C-terminal domain of AcrR with that of QacR28 indicates that this large internal cavity overlaps with the multidrug-binding pocket of QacR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The C-terminal domain of AcrR consists of six α helices , with helices 4 , 5 , 7 , 8 and 9 forming an antiparallel five-helix bundle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The overall structure of the N-terminal domain of AcrR is very similar to those of TetR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Although the C-terminal region displays no primary sequence conservation among members of the TetR family , the overall structure of the C-terminal domain of AcrR displays topological similarity to those of QacR28 and CmeR .32 A pair-wise alignment of the C-terminal domains is difficult as the six C-terminal helices ( seven in CmeR ) differ both in length and orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The binding of drugs to the C-terminal domain of AcrR triggers a conformational change in the N-terminal DNA-binding region , resulting in the release of AcrR from DNA and allowing transcription from its cognate promoter ( Su et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>It is expected that binding of drugs to the C-terminal ligand-binding domain of AcrR triggers conformational change in the Nterminal DNA-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>As a member of the TetR family of transcriptional regulators , it is expected that binding of drug to the C-terminal ligand-binding domain of AcrR triggers conformational change in its N-terminal DNA-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The N-terminal domain of AcrR contains a DNA-binding helix-turn-helix ( HTH ) motif , and the C-terminal domain has a unique sequence that is predicted to bind ligands ( Ramos et al. 2005 ; Su et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Like other members in the TetR family , the N-terminal domain of AcrR contains a predicted DNA-binding helix-turn-helix ( HTH ) motif , while its C-terminal domain has unique sequences and is expected to form a multi-drug-binding site for its inducing drugs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>We used the program `` putative active sites with spheres '' ( PASS ) 33 to search for potential ligand-binding sites in the AcrR structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Each monomer of AcrR forms a multi-entrance pocket of 350 cubic angstroms in the ligand-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The ligand-binding pocket of AcrR is unique in that it possesses three openings ( Fig. 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The Figure was prepared using PyMOL .52 Views of the cavity formed in the ligand-binding domain of AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Fig. 2a illustrates the superposition of the N-terminal domains of AcrR , QacR and CmeR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>We generated a speculative model of DNA-bound AcrR by aligning its individual domains with those of the DNA-bound QacR ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>All The model of the DNA-bound form of AcrR was generated using O. 46 In brief , the N - and Cterminal domains of AcrR were separately aligned with the corresponding domains of the DNA-bound QacR ( 1JT023 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The N - and C-terminal domains of the ligand-bound dimeric AcrR were individually aligned with those of the DNA-bound QacR ( 1JT0 ) to generate the DNA-bound form of AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The model of this DNA-bound AcrR suggests that Y49 , H51 and K55 make phosphate contacts with the IR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The crystal structure of the DNA-bound AcrR would be necessary to confirm our model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The asymmetric unit contains a single monomer of AcrR ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The structure reveals that AcrR is an all-alpha helical protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The crystal structure of AcrR belongs to space group P31 similar to P2221 , indicating that solvents bind at Glu67 ( Gu et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">AcrR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The mtr locu consists of mtrR , encoding a transcriptional repressor protein related to AcrR and AcrS ( 203 ) , and an operon containing the mtrC , mtrD , and mtrE genes ( 95 ) , which encode members of the MFP , RND , and OMF families , respectively ( Table 4 ; Fig. 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>The crystal structure of AcrR belongs to space group P31 similar to P2221 , indicating that solvents bind at Glu67 ( Gu et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_3">
Gristwood , T. , Fineran , P. C. , Everson , L. , and Salmond , G. P. ( 2008 ) PigZ , a TetR/AcrR family repressor , modulates secondary metabolism via the expression of a putative four-component resistance-nodulation-cell-divi-sion efflux pump , ZrpADBC , in Serratia sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22474332" target="_blank">22474332</a>]<br></label>
<input id="evo_similarity_3", type="checkbox">
<div class="EVO_similar">
<li>[ 83 ] T. Gristwood , P.C. Fineran , L. Everson , G.P. Salmond , PigZ , a TetR/AcrR family repressor , modulates secondary metabolism via the expression of a putative four-component resistance-nodulation-cell-division efﬂux pump , ZrpADBC , in Serratia sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li></div>
<li>Pathogenic bacteria , such as P. aeruginosa exhibit a high number of TFs associated to antibiotic resistance of the TetR/AcrR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>The mHTH class includes the three large families , AraC/XylS ( 8.5 % ) , LuxR/UhpA ( 5.7 % ) , and TetR/AcrR ( 9.4 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>We identiﬁed a high proportion of TFs belonging to the AsnC , ArsR , MarR , TetR/AcrR , BirA , AraC/XylS , Fur , LysR , and PhoU families in bacteria and archaea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Sequence alignment of the HTH motif among members of the TetR family indicates that residues L34 , R45 , G46 , A47 , Y49 , W50 , H51 and K55 in AcrR are important for repressoroperator interaction .22 Three of these residues , Y49 , H51 and K55 , are highly conserved , suggesting that these residues may play a major role in the recognition of target DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Although the C-terminal region displays no primary sequence conservation among members of the TetR family , the overall structure of the C-terminal domain of AcrR displays topological similarity to those of QacR28 and CmeR .32 A pair-wise alignment of the C-terminal domains is difficult as the six C-terminal helices ( seven in CmeR ) differ both in length and orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>To understand the structural basis of AcrR regulation , we have determined the crystal structure of AcrR , to 2.55 Å resolution , revealing a dimeric two-domain molecule with an entirely helical architecture similar to members of the TetR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The overall structure of the N-terminal domain of AcrR is very similar to those of TetR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>As a member of the TetR family of transcriptional regulators , it is expected that binding of drug to the C-terminal ligand-binding domain of AcrR triggers conformational change in its N-terminal DNA-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The relatedness of the predicted YtfA and TetR proteins is highest in the N-terminal domain that contains a helix-turn-helix motif required for DNA binding in the TetR/AcrR family of transcription factors ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11705922" target="_blank">11705922</a>]<br></li>
<li>Like other members in the TetR family , the N-terminal domain of AcrR contains a predicted DNA-binding helix-turn-helix ( HTH ) motif , while its C-terminal domain has unique sequences and is expected to form a multi-drug binding site for its inducing drugs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>[ 6 ] Deng , W. , Li , C. and Xie , J. ( 2013 ) The underlying mechanism of bacterial TetR / AcrR family transcriptional repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25685666" target="_blank">25685666</a>]<br></li>
<li>Thus , our experimental data strongly The affinity of AcrR for IR is in the nanomolar range ( 20.2 nM ) , and this dissociation constant is closer to that of QacR among members of the TetR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by AcrR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Productb ybiC b0801 Putative dehydrogenase yqhD b3011 Putative alcohol dehydrogenase 3.1 3.3 Inorganic-ion-transport and metabolism cirA b2155 Outer membrane pore protein , receptor for colicin I , requires TonB cysA b2422 ATP-binding component of sulfate permease A protein ; chromate resistance cysC b2750 Adenosine 5 - phosphosulfate kinase cysI b2763 Sulﬁte reductase , alpha subunit cysJ b2764 Sulﬁte reductase ( NADPH ) , ﬂavoprotein beta subunit cysN b2751 ATP-sulfurylase ( ATP : sulfate adenylyltransferase ) , subunit 1 , probably a GTPase cysP b2425 Thiosulfate binding protein fecA b4291 KpLE2 phage-like element ; outer membrane porin , receptor for ferric citrate fecR b4292 KpLE2 phage-like element ; regulator in multicomponent regulatory system with FecI/FecA fepA b0584 Outer membrane porin , receptor for ferric enterobactin and colicins B and D fepB b0592 Ferric enterobactin ( enterochelin ) binding protein ; periplasmic component fepC b0588 ATP-binding component of ferric enterobactin-transport fepG b0589 Ferric enterobactin-transport protein fes b0585 Enterochelin esterase ycdO b1018 Hypothetical protein 2.8 2.3 2.7 2.2 2.7 2.1 3.1 2.5 3.3 2.0 1.9 2.0 2.0 2.7 1.8 Lipid transport and metabolism acs b4069 Acetyl-coenzyme A synthetase 2.4 1.8 1.8 fadE b0221 Medium-long-chain fatty acyl-coenzyme A dehydrogenase garR b3125 Tartronate semialdehyde reductase ( TSAR ) etabolism Allantoinase Nucleotide transport and m allB b0512 rihA b0651 rihC b0030 1.9 2.0 3.3 Pyrimidine-speciﬁc nucleoside hydrolase Nucleoside hydrolase Posttranslational modiﬁcation , protein turnover , chaperones ccmC b2199 Heme export protein ( ABC superfamily , membrane ) ccmG b2195 Heme lyase/disul ﬁde oxidoreductase ( thiol-disulﬁde interchange protein ) cysU b2424 Sulfate , thiosulfate-transport system permease T protein yeeD b2012 Hypothetical protein 2.3 1.8 2.7 3.0 Replication , recombination , and repair nohB b0560 DLP12 prophage ; bacteriophage DNA - packaging protein sbcB b2011 Exonuclease I , 3 3 - speciﬁc ; deoxyribophosphodiesterase b0259 CP4 -- 6 prophage ; IS5 protein 1 b3218 IS5 protein 10 b3505 IS5 protein 11 b0656 IS5 protein b1331 IS5 protein b1370 IS5Y transposase b1994 IS5 protein b2030 IS5 protein b2192 IS5 protein b2982 IS5 protein 2.0 2.0 2.0 2.3 2.2 2.2 2.1 2.4 2.4 2.6 2.5 3.2 Secondary-metabolite biosynthesis , transport and catabolism entB b0595 2,3-Dihydro-2 ,3 - dihydroxybenzoate synthetase , isochroismatase entE b0594 2,3-Dihydroxybenzoate-AMP ligase idnO b4266 5-Keto-D-gluconate 5-reductase srlD b2705 Glucitol ( sorbitol ) -6 - phosphate dehydrogenase yeiN b2165 Hypothetical protein 2.1 2.2 2.0 1.9 3.1 Signal transduction mechanisms ycdT b1025 Putative transmembrane protein 1.7 Transcription KpLE2 phage-like element ; sigma 19 factor of RNA polymerase fecI b4293 lrhA b2289 yijC b3963 2.2 4.4 2.4 NADH dehydrogenase transcriptional regulator , LysR family ; modulates SprE ( RssB ) activity Putative regulator ( TetR/AcrR family ) General function prediction or function unknown yadS b0157 Putative membrane protein 2.2 Continued on following pag Not in COGS database b1172 b1172 Hypothetical protein 2.0 2.3 1.9 2.6 2.4 2.7 2.0 2.5 3.5 3.5 8.0 2.4 4.9 2.1 3.4 2.4 2.1 2.3 b2191 b2191 Unknown CDS cdaR b0162 Regulator of D-galactarate , D-glucarate , and D-glycerate metabolism cysW b2423 Sulfate transport system permease W protein malM b4037 Periplasmic protein of mal regulon pppA b2972 Bifunctional prepilin peptidase srlA b2702 PTS family enzyme IIC , glucitol/sorbitol speciﬁc srlE b2703 PTS family enzyme IIBC , glucitol/sorbitol speciﬁc ybcT b0556 DLP12 prophage ; bacteriophage lambda endopeptidase homolog ybcU b0557 DLP12 prophage ; bacteriophage lambda Bor lipoprotein homolog , involved in serum resistance yddB b1495 Hypothetical protein yedO b1919 D-cysteine desulfhydrase , PLP-dependent enzyme ygaW b2670 Putative membrane protein yhcE b3217 Unknown CDS yijD b3964 Putative membrane protein yjiD b4326 Unknown CDS ylaC b0458 Putative membrane protein ymfL b1147 e14 prophage ; putative negative regulator a Genes involved in iron utilization and/or known to be Fe2 - Fur regulated are shown in boldface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Gene product OD 4a NaClb pH 5c Metabolism ( 26 ) adhP b1478 b1927 b3012 b2097 b2661 b1619 b2720 b0186 b1469 b1467 b0871 b4051 b2464 b1519 b2465 b3449 b0485 Alcohol dehydrogenase , propanol preferring Cytoplasmic alpha-amylase 5.38 12.21 4.37 8.00 4.81 2.71 10.67 3.76 2.75 5.24 17.67 2.39 6.79 2.76 14.08 2.30 14.05 9.19 11.02 6.33 9.61 4.08 6.97 24.06 3.72 3.62 5.70 45.89 2.73 13.08 8.68 35.87 3.69 9.46 3.43 10.57 2.37 9.33 2.89 2.43 9.11 3.88 2.07 4.57 29.60 2.64 13.83 3.53 10.76 2.01 2.70 amyA dkgV fbaB gabD hdhA hycF IdcC narU narY poxB qor talA tam tktB ugp ybaS ybaY ycaC yfcF ygjG yhfW yjcS yjgB yncG 2,5-Diketo-D-gluconate reductase A Fructose-bisphosphate aldolase class I Succinate-semialdehyde dehydrogenase NADP dependent alpha-hydroxysteroid dehydrogenase , NAD dependent Hydrogenase 3 , putative quinone oxidoreductase , FeS related Lysine decarboxylase 2 , constitutive Nitrate extrusion protein Nitrate reductase 2 , beta-subunit Pyruvate dehydrogenase/oxidase : FAD and thiamine PPi-binding Quinone oxidoreductase , NADPH dependent Transaldolase A Trans-aconitate methyltransferase Transketolase 2 , thiamin binding , isozyme Glycerophosphodiester phosphodiesterase , cytosolic Putative glutaminase/carboxypeptidase with beta-lactamase/D-Ala-carboxypeptidase domain Glycoprotein/polysaccharide-metabolism Putative enzyme with cysteine hydrolase domain Putative glutathione S-transferase enzyme with thioredoxin-like domain Putrescine :2 - oxoglutaric acid aminotransferase Putative phosphopentomutase , alkaline phosphatase-like domain Putative enzyme with 2 metallohydrolase/oxidoreductase domains Putative alcohol dehydrogenase with NAD ( P ) - binding and GroES domains Putative glutathione S-transferase with thioredoxin-like and glutathione Stransferases , C-terminal domain Putative dienelactone hydrolase b0453 b0897 b2301 b3073 b3380 b4083 b4269 b1454 6.15 13.34 2.55 8.93 4.00 10.73 6.98 3.42 8.67 5.92 2.44 16.90 6.67 10.38 10.80 13.77 6.09 5.88 2.40 7.45 4.46 4.71 7.40 3.64 ysgA b3830 2.69 2.27 3.68 Regulation ( 11 ) bolA b0435 b1217 b2664 b3515 Transcriptional activator of morphogenic pathway ( BolA family ) , important in general stress response Cation-transport regulator Putative transcriptional repressor with DNA-binding winged helix domain ( GntR family ) Transcriptional regulator for GadX ( regulatory protein ) , glutamic acid decarboxylase ( GadA , - B ) , and glutamate transport protein ( GadC ) ( AraC-XylS family ) Transcriptional regulator for glutamic acid decarboylase and transporter ( GadA , GadBC ) ( AraC/XylS family ) RNase II modulator with PYP-like sensor domain Transcriptional repressor for pyruvate dehydrogenase complex ( GntR family ) Response regulator involved in protein turnover , controls stability of RpoS Flavodoxin-like protein , trp repressor-binding protein Putative transcriptional regulator with DNA-binding domain Putative regulator with homeodomain-like DNA-binding domain ( TetR/AcrR family ) 4.77 3.38 4.30 3.09 4.74 3.36 4.84 6.12 8.47 5.54 10.09 5.98 chaB csiR gadW gadX gem pdhR rssB wrbA yiaG yjdC b3516 b1285 b0113 b1235 b1004 b3555 b4135 6.09 6.78 5.01 4.68 9.38 8.40 2.49 7.45 3.83 3.42 7.19 6.83 9.20 2.34 3.14 6.94 2.86 4.08 14.77 4.08 2.46 Transport and membrane ( 20 ) artM artP blc eutH gabP mscL potF rssA ugpB b0861 b0864 b4149 b2452 b2663 b3291 b0854 b1234 b3453 2.61 4.71 8.10 2.59 2.83 2.59 2.24 2.26 3.29 2.79 4.56 8.99 3.06 4.70 2.31 4.25 3.19 5.55 2.20 3.50 4.77 3.11 4.06 2.94 2.76 2.50 3.29 Arginine transport protein ( ABC superfamily , membrane ) Arginine transport protein ( ABC superfamily , ATP-binding subunit ) Outer membrane lipoprotein ( lipocalin ) Putative transport protein , ethanolamine utilization Gamma-aminobutyrate transport protein Mechanosensitive channel Putrescine transport protein ( ABC superfamily , periplasmic binding protein ) Putative transmembrane protein Sn-Glycerol 3-phosphate-transport protein ( ABC superfamily , periplasmic binding protein ) Sn-Glycerol 3-phosphate-transport protein ( ABC superfamily , ATP-binding subunit ) Putative transport protein , integral membrane location Putative transmembrane protein ( N terminal ) ; putative phosphatase ( C terminal ) Putative transport protein ( ABC superfamily , periplasmic binding protein ) Putative multidrug-resistance membrane protein Putative membrane protein Putative membrane protein Putative permease ( PerM family ) Putative membrane protein Putative transmembrane protein Putative transport protein ( YqaE family ) ugpC ybiO ybiP ydcS ydhJ yeaY yebS yhhT yhiD yphA yqaE 2.39 8.49 7.73 21.66 2.21 2.43 2.08 2.30 4.11 2.35 2.42 b3450 b0808 b0815 b1440 b1644 b1806 b1833 b3474 b3522 b2543 b2666 2.44 2.02 2.43 4.36 2.22 4.06 2.48 2.92 7.24 2.53 3.16 2.50 4.51 5.89 4.83 4.29 3.21 2.50 3.89 5.93 3.38 4.78 Continued on following pag TABLE 1 -- Continued Ratio Function ( reference ) and gene b no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Quantitative real-time PCR analysis of RNA from the wild-type , ΔacrR , and acrR + strains showed that AcrR represses the transcription of marRAB and soxRS , and its own gene cluster , acrRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>This hypothesis is consistent with the notion that a genome evolves from a set of precursor genes to a mature size by gene duplications and increasing modiﬁcations ( Yanai et al. , 2000 ) Table 1 TFs identiﬁed in complete genomes Family DBD Role PFAM Annotations Total 63 328 Function MarR w R 502 Multiple antibiotic resistance response Mercury resistance regulator MerR c R 244 MetJ -- Helmann et al. ( 1989 ) ; Zeng et al. ( 1998 ) Old et al. ( 1993 ) ; Wu et al. ( 1993 ) 48 235 10 R 0 Methionine biosynthesis repressor Molybdenum metabolism regulation R ModE 30 NadRa c 798 w 0 20 12 R 0 Foster and Penfound ( 1993 ) Regulation of NAD-synthesis and nucleotide-transport Regulation of the nag ( N-acetylglucosamine ) operon Biosynthesis of membrane components Probable growth inhibitor regulator NagR/XylRa w 244 A 2 31 1 A 135 OmpR w 722 638 n/d n/d 0 PemK -- PhoU -- Pérez - Rueda and Collado-Vides ( 2000 ) 36 62 0 0 -- -- Phosphate assimilation Ribose phosphate isomerase 137 R R 94 RpiR c Sorensen and Hove-Jensen ( 1996 ) Aramaki et al. ( 1995 ) 157 6 865 12 897 TetR/AcrR m Tetracycline resistance repressors Tryptophan operon repressor 12 29 TrpR WrbA R m Yang et al. ( 1993 ) Yang et al. ( 1993 ) -- -- R -- Enhancer in the formation and/or stability of noncovalent complexes between TrpR and DNA The notation is as follows : family name ( Hyp : hypothetical families ) ; DBD structure ( w , winged ; m , miscellaneous , and c , classic ) ; HLH , helix-loop-helix ; , beta-strand antiparallel ; CSD , cold shock DNA-binding domain-like ; and Zn , zinc-ﬁnger ; ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Characterization of the multidrug efflux regulator AcrR from Escherichia coli Chih-Chia Su1 , Den ae J. Rutherford2 , and Edward W. Yu1 ,2 , * 1 Department of Bioch USA emistry , Biophysics and Molecular Biology , Iowa State University , Ames , IA 50011 , 2 Department of Physics and Astronomy , Iowa State University , Ames , IA 50011 , USA bstract A The Escherichia coli AcrR represses transcription of the acrB gene , which encodes the multidrug efflux pump AcrB that extrudes a wide variety of toxic compounds , by binding its target operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Finally , the effect of AcrR modification on the expression of several genes , including mar-sox-rob genes , was studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Appl Microbiol Biotechnol ( 2014 ) 98:8763 -- 8773 DOI 10.1007 / s00253-014-6024-9 BIOENERGY AND BIOFUELS Overproduction o AcrR increases organic solvent tolerance mediated by modulation of SoxS regulon in Escherichia coli f Jae Ok Lee & Kyung-Suk Cho & Ok Bin Kim Abstract Acriflavine resistance regulator ( AcrR ) , a local transcription factor , regulates the expression of the acrRAB genes associated with the AcrAB-TolC multidrug efflux pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The acriflavine resistance regulator ( AcrR ) is a transcriptional repressor that regulates expression of the AcrAB-TolC multidrug efflux pump ( Ma et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>In the present study , we showed that the local transcriptional factor associated with the AcrAB-TolC pump AcrR functions as a repressor of transcription of global regulator genes marRAB and soxRS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The strongest similarity was to EnvR of Escherichia coli , but the sequence also resembled MtrR , AcrR , TetC , and TcmR , all of which are involved in regulating resistance to antibiotics or toxic hydrophobic substances in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592375" target="_blank">7592375</a>]<br></li>
<li>Cloning , expression , purification , crystallization and preliminary X-ray diffraction analysis of the regulator AcrR from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Ma et al. ( 163 ) have proposed that regulation of acrAB expression is mediated primarily by global regulatory pathways , and AcrR acts as a secondary modulator to prevent excessive expression of acrAB ( 163 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>AcrR appears to play a modulating role in preventing excessive expression of acrAB ( Ma eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>For instance , the expression of acrAB in E. coli is controlled by repressor protein AcrR , global activator protein MarA , SoxS , Rob and quorum-sensing regulator LuxR homologue protein SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_15">
Mol Microbiol 16:45 -- 55 Ma D , Alberti M , Lynch C , Nikaido H , Hearst JE ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></label>
<input id="rp_similarity_15", type="checkbox">
<div class="RP_similar">
<li>Ma , D. , Alberti , M. , Lynch , C. , Nikaido , H. , and Hearst , J.E. ( 1996 ) In the regulation of the acrAB genes of Escherichia coli by global stress signals , the local repressor AcrR plays a modulating role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Ma , D. , Alberti , M. , Lynch , C. , Nikaido , H. , and Hearst , J.E. ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>[ 8 ] Ma , D. , Alberti , M. , Lynch , C. , Nikaido , H. and Hearst , J.E. ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11886757" target="_blank">11886757</a>]<br></li></div>
<li>The local repressor AcrR plays a moderating role in the regulation of acrAB genes of Escherichia coli by gobal stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_20">
The local repressor AcrR plays a modulating role in the regulation of acraAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></label>
<input id="rp_similarity_20", type="checkbox">
<div class="RP_similar">
<li>The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562820" target="_blank">12562820</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11914367" target="_blank">11914367</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18297445" target="_blank">18297445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885437" target="_blank">16885437</a>]<br></li></div>
<li>A homologue of RutR , AcrR , also functions as a secondary modulator of gene-expression by 70-holoenzyme ( 16 ) , as does ArgR , an unrelated protein , at the 54-dependent ast promoter ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_23">
Gristwood , T. , Fineran , P. C. , Everson , L. , and Salmond , G. P. ( 2008 ) PigZ , a TetR/AcrR family repressor , modulates secondary-metabolism via the expression of a putative four-component resistance-nodulation-cell-divi-sion efflux pump , ZrpADBC , in Serratia sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22474332" target="_blank">22474332</a>]<br></label>
<input id="rp_similarity_23", type="checkbox">
<div class="RP_similar">
<li>[ 83 ] T. Gristwood , P.C. Fineran , L. Everson , G.P. Salmond , PigZ , a TetR/AcrR family repressor , modulates secondary-metabolism via the expression of a putative four-component resistance-nodulation-cell-division efﬂux pump , ZrpADBC , in Serratia sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">AcrR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>A gel mobility shift assay , using a DNA fragment bearing the sequence of the promoter region between acrR and acrAB , has demonstrated that AcrR binds to the promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Fluorescence polarization was performed using purified , recombinant AcrR that contains a 6xHis tag at the C-terminus and a fluorescein-labeled 28-base pair oligonucleotide bearing a predicted palindrome ( IR ) operator sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Using recombinant AcrR protein that contains a 6xHis tag at the C-terminus , we confirmed that AcrR interacts with the 28 bp DNA , 5 ′ TTTACATACATTTGTGAATGTATGTACC3 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The titration experiment indicated that AcrR binds the 28 bp operator , with a dissociation constant , KD , of 20.2 ± 1.4 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Fluorescence polarization-based assay was carried out to study the interaction between AcrR and a 28 bp DNA containing the IR sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>The IR sequence of AcrR is similar in length to that of the long 28 bp IR1 recognized by QacR , but is different from the 15 bp tetO bound by TetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>However , the crystal structure of the QacR-DNA complex reveals that QacR binds its 28-bp IR1 operator as a dimer of dimers .23 In the case of AcrR , there is strong evidence that the IR of AcrR consists of a 24-bp inverting sequence , 5 ′ TACATACATTTGTGAATGTATGTA3 ′ .21,24,25 This nucleotide sequence is located upstream of the acrAB operon and overlaps with the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Both AcrR and AcrS directly bound to the acrA promoter containing the previously predicted 24-bp palindrome sequence ( TACATACATT-TATG-AATGTAT GTA ) ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>Presumably , AcrR represses the transcription of acrAB by directly binding to the inverting 24-bp IR sequence in the promoter region of the efflux operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>As mentioned above , the predicted IR operator of AcrR is composed of an inverting 24 bp sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>These oligodeoxynucleotides contain the predicted 24 bp IR site for AcrR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Binding site prediction for AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Since the Bglll site is located at the 3 ' end of the helix-turn-helix coding sequence of acrR , it is possible that the truncated AcrR could be expressed in WZM124 and retain the potential DNA-bindingability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The induced conformational change triggered by the bound drug is accompanied with an increase in center-to-center distance between the DNA recognition helices α3 and α3 ′ from 37 Å in the DNA-bound form to 48 Å in the ligandbound form .23 Thus , similar conformational change within two HTH motifs of the AcrR dimer is expected upon DNA and ligand bindings .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Thus , its function in AcrR is expected to recognize and bind multiple AcrR drugs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Although AcrR and QacR recognize a similar range of cationic lipophilic drugs , their C-terminal sequences , which form the multidrug binding pockets , are quite diverse .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by AcrR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Characterization of the multidrug efflux regulator AcrR from Escherichia coli Chih-Chia Su1 , Den ae J. Rutherford2 , and Edward W. Yu1 ,2 , * 1 Department of Bioch USA emistry , Biophysics and Molecular Biology , Iowa State University , Ames , IA 50011 , 2 Department of Physics and Astronomy , Iowa State University , Ames , IA 50011 , USA bstract A The Escherichia coli AcrR represses transcription of the acrB gene , which encodes the multidrug efflux pump AcrB that extrudes a wide variety of toxic compounds , by binding its target operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Conversely , in the ΔacrR strain , the expression of all tested genes ( marR , marA , marB , soxR , and soxS ) , other than the constitutive gene rob ( Kwon et al. 2000 ) , increased significantly regardless of the presence or absence of solvent , suggesting that AcrR functions as a transcriptional repressor of those genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>acrF and yhiV are located in operons together with acrE and yhiU ( orfA ) ( Fig. 16 ) , which encode MFP constituents , and the acrEF operon is probably regulated by the upstream acrS gene , whose product is similar to AcrR ( Fig. 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>The mtr locu consists of mtrR , encoding a transcriptional repressor protein related to AcrR and AcrS ( 203 ) , and an operon containing the mtrC , mtrD , and mtrE genes ( 95 ) , which encode members of the MFP , RND , and OMF families , respectively ( Table 4 ; Fig. 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>Materials and methods Expression and Purification of Recombinant AcrR The cloning , expression , and purification of recombinant AcrR that contains a 6xHis tag at the C-terminus were done as described before [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
<li>Therefore , AcrR can repress its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The expression of tolC was similar to that observed in the wild-type , acrR + and ΔacrR strains , indicating that tolC transcription was not regulated by AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Quantitative real-time PCR analysis of RNA from the wild-type , ΔacrR , and acrR + strains showed that AcrR represses the transcription of marRAB and soxRS , and its own gene cluster , acrRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>AcrR repressed the transcription of marRAB and soxRS by binding to their promoter The effect of AcrR on the transcription of 14 target genes was investigated by quantitative real-time PCR in wild-type , ΔacrR , and acrR + strains under n-hexane/cyclohexane stress ( Table 2 ; Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>However , these acrAB induction rates are low ( about twofold increase ) , probably because of the semi-constitutive expression of acrAB caused by the leakiness of AcrR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>For instance , the expression of acrAB in E. coli is controlled by repressor protein AcrR , global activator protein MarA , SoxS , Rob and quorum sensing regulator LuxR homologue protein SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>However , it does not appear that modulation of acrAB expression in response to these environmental stresses operates through AcrR , MarA , or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>However , the expression of these regulators was not affected by AcrS and AcrR ( data not shown ) , indicating that the acrAB repression by AcrR and AcrS is unlikely to be mediated by MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>The drug susceptibilities of W3104 hns acrAB were hardly affected even when AcrS and AcrR were overexpressed ( Table 1 ) , suggesting that overexpression of neither AcrS nor AcrR suppresses the expression of acrEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>Presumably , AcrR represses the transcription of acrAB by directly binding to the inverting 24-bp IR sequence in the promoter region of the efflux operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>AcrR represses both itself and acrAB transcription , although the repression is very weak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>The induction of acrAB , marRAB , and soxRS following deletion of acrR , and the in vitro binding of purified AcrR to the promoter DNA of each operon , indicate that AcrR controls expression of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The transcription of the acrAB operon in E. coli is regulated by the repressor AcrR ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11914367" target="_blank">11914367</a>]<br></li>
<li>The expression of AcrAB is controlled by the transcriptional regulator AcrR , whose open reading frame is located 141-base-pair upstream of the acrAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>The acrR gene is located upstream of acrA , and the AcrR protein represses expression of the acrAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>Analysis of acrAB-lacZ fusions has suggested that expression of this operon is subject to regulation by the E. coli mar regulon ( 166 ) and also by AcrR ( 163 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>AcrS also represses the expression of acrA more efﬁciently than AcrR does .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>