<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Ada</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Ada</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Ada transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>MeAda and 0.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>However , in the presence of meAda , the transcriptional-activator responsible for alkylation-damage induction of aid6 the levels of Eos-dependenttranscription were significantly increased , indicating that meAdacan activate transcription by both Eo7 ' and EnS.Interaction of a transcriptional-activator with both forms of RNA polymerase has already been describedfor the CAMPreceptorprotein ( CRP ) at boththe gatP I and the Iac promoters ( Kolb eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>In the absence of meAda , RNA polymerase can bind to the promoters via its aCTD , but it fails to establish any strong interaction with the core promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Results of gel retardation experiments are shown in Figure 3 ; a retarded band , corresponding to the RNA polymerase-DNA complex appears at an enzyme concentration of 0.25 pmol , regardless of the presence of meAda , for both the ada and aidB promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>meAda might bind to a region of the alkA promoter important for recognition by EsS , but not by Es70 ; alternatively , binding of meAda might alter the alkA conformation to make it less favorable for interaction with EsS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Results of the DNasel-protection experiment shown in Fig. 16 are consistent with the in vitro transcription assay : in the absence of meAda , Ea7 ' ( 0.2 pM ) forms a binary complex that is inactive in transcription initiation ( Landini and Volkert , 1995b ) , and strongly protects the 01996 Blackwell Science Ltd , Mo/ecu/ar Microbiology , 20 , 947-95 Repression of os-dependent transcription by Lrp 949 aidB promoter region upstream of -40 ( lane 2 , Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Thus , it is possible that meAda could contact amino acids conserved in both s factors ; future experiments of site-directed mutagenesis in ss will allow better understanding of activation of ss-depend-ent transcription by meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Instead , upon binding of a to the UP element , meAda interacts with s70 , activating transcription ; therefore , a subunit-pro-moter and meAda-s70 interactions act at separate but interdependent steps of transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>When cells reach stationary phase , intracellular concentrations of sS increase ; meAda prevents EsS from binding to alkA , thus reducing the amount of RNA polymerase available for alkA transcription and , in turn , its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Thus , meAda activates transcription either by improving initial binding of RNA polymerase to the 235 or by facilitating a later step in transcription initiation , such as isomerization to open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Values are percentages of transcription by wild-type Es70 in the presence of meAda and are averages of three independent experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>9 ; when necessary , 0.2 mM of meAda was added to the transcription reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The function of meAda as a negative regulator of EsSdependent transcription suggests a speciﬁc mechanism for turning off high-level expression of the alkA gene : alkA expression is activated as long as transcription in the cell is mostly dependent on Es70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The function of meAda as a negative regulator of EsSdependent transcription suggests an alternative mechanism : alkA expression is activated as long as transcription is mostly s70 dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>In contrast , not only does meAda fail to stimulate alkA transcription by EsS , consistent with the lack of conservation of K593 , K597 , and R603 in sS , but it negatively affects EsS-dependent transcription both in vivo and in vitro ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Our present results show that meAda negatively regulates EsS-de-pendent transcription of alkA , and this mechanism is used by E. coli to down-regulate alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>The specificity of meAda - 70 interaction , the location of its s determinants in the terminal 39 amino acids of s70 , and the strong effect of the E575V mutation in vitro point to s70CTD as the principal target of activation by meAda at the ada and aidB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Model for transcription activation by meAda at ada and alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The evidence for this model was based on the observation that truncations in aCTD abolish transcription from the ada promoter in the presence of meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>meAda activates ada transcription by either Es70 or EsS with roughly the same efﬁciency , while the lack of a functional rpoS gene results in lower expression of the aidB gene even in the presence of meAda ( 22 , 69 ) , showing that the aidB promoter is dependent on both Ada and sS for optimal expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>The results presented in Fig. 3 show that 1 pmol of unmethylated Ada or unmethylate pBR322 None 52 60 1.1 pDS411 Wild type 216 1,302 6.0 ( lysine 178 ) pK178L Leucine 4.6 pK178V Valine 4.0 pK178G Glycine 178 73 327 4.5 pK178Y Tyrosine 3.4 pK178R Arginine 5.2 pK178C Cysteine 3.9 pK178P Proline 178 58 53 0.9 pK178S Serine 3.2 TABLE 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Bacterial strains and plasmids used in this study Source or reference Genotype or description Parental strain or plasmid vector Strain or plasmid E. coli strains AB1157 4 F9 thr-1 leu-6 proA2 his-4 thi-1 argE3 lacY1 galK2 ara-14 xyl-5 mtl-1 tsx-33 rpsL31 supE37 ada-lacZ argE3 his-4 leu-6 proA2 thr-1 ara-1 galK2 lacY1 mtl-1 xyl-5 thi-1 rpsL31 supE44 tsx-33 rfa-550 alkA104 : : lpSG1 Dada-25 ada-lacZ DlacU169 pro1 [ F9 proA1B1 lacIq lacZDM15 Tn10 ( Tcr ) ] alkA-lacZ DlacU169 pro1 [ F9 proA1B1 lacIq lacZ : : Tn5 ( Kanr ) ] GW5354 MV1902 AB1157 7 33 GW7102 GW7105 GW7106 MV1902 GW5354 GW7102 29 26 26 Plasmids pBR322 Apr Tcr ada1 alkB1 Apr Laboratory stock 7 26 26 26 This study This study pGW2607 pDS400 pDS411 pDS415 pBS179 pBS178/179 pBR322 pGW2607 pDS400 pDS411 pDS411 pBR322 ada1 Apr ; contains entire ada gene and a small segment of alkB ada1 ; entire ada gene under lac control N terminus of Ada up to lysine 178 under lac control C terminus of Ada starting at glutamine 179 under lac control N terminus and C terminus of Ada , each under lac control , transcribed in divergent orientations Lysine 178 changed to leucine Lysine 178 changed to valine Lysine 178 changed to glycine Lysine 178 changed to tyrosine Lysine 178 changed to arginine Lysine 178 changed to cysteine Lysine 178 changed to proline Lysine 178 changed to serine pK178L pK178V pK178G pK178Y pK178R pK178C pK178P pK178S This study This study This study This study This study This study This study This study pDS411 pDS411 pDS411 pDS411 pDS411 pDS411 pDS411 pDS411 b-Galactosidase activity ( OD600 ) of ada9-lacZ strain Ada Fold Plasmid derivative induction 2MNNG 1MNNG pBR322 None 52 60 1.1 pDS411 Wild type 216 1,302 6.0 ( lysine 178 ) pK178L Leucine 4.6 pK178V Valine 4.0 pK178G Glycine 178 73 327 4.5 pK178Y Tyrosine 3.4 pK178R Arginine 5.2 pK178C Cysteine 3.9 pK178P Proline 178 58 53 0.9 pK178S Serine 3.2 RESULTS Alteration of Lys-178 residue of Ada protein differentially affects regulation of ada and alkA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Effects of Ada hinge mutations on alkA transcription in vivoa b-Galactosidase activity ( OD600 ) of alkA9-lacZ strain Ada Fold Plasmid derivative induction 1MNNG 2MNNG 1.0 3.8 6.0 119 6.0 31 pBR322 pDS411 None Wild type ( lysine 178 ) Leucine 178 Valine 178 Glycine 178 Tyrosine 178 Arginine 178 Cysteine 178 Proline 178 Serine 178 32 33 32 25 35 29 35 32 pK178L pK178V pK178G pK178Y pK178R pK178C pK178P pK178S 3.2 3.2 3.9 4.6 3.2 3.7 3.3 3.6 FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Each track contained the following materials : I and I I , 50 pg protein of Fraction I and 11 ; III ,30 pg protein of Fraction 111 ; IV , 7 pg protein of Fraction IV ; V , 5 pg protein of Fraction V ; M , molecularmass markers asdescribed in the legend to Fig. 2 Amino acid composition of Ada protein Amino acid cleotide sequence Predicted fromnu-residueslmlecule 28.0 17 11 22.0 ' 21 17.5 ' 18 43.2 21 22 13.0 15 19.0 19 49.9 11.8 ' 12 17.5 d 18 5.3 6 11.7 13 33Ad 6.0 12.2 14.9 8.1 2.5 ' 34.3 Aspartic acid Asparagine Threonine Serine Glutamic acid Glutamine Proline Glycine Ala4n8ine Cysteine Valine Methionine Isoleucine Leucin3e4 T6yrosine Phenylalanine 12 Lysine 14 Histidine 8 Tryptophan 5 Arg34 inine Total 354 a Average values obtained from 24 - , 48 - , and 72-h hydrolyses with 5.7 N HCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Although methylated phosphates are innocuous , this lesion is readily produced by methylating agents ( 37 ) and provides a sensitive regulatory signal that leads to induction of the Ada regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Induction by alkylating agents is mediated by the transcriptional-activator Ada , in its methylated form ( meAda ) ; the other forms of induction are Ada independent and require as , the alternative a factor mainly expressed during the stationary phase of bacterial growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Potentially mutagenic and cytotoxic alkylating agents are The Ada response is induced by bacterial cell exposure to sublethal doses of methylating agent [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Regulatory mechanisms for induction of syn enzymes in response to alkylating agents : A acts as a transcriptional-regulator ABSTRACT Expression of the ada and alkA genes , both of which are involved in the adaptive response of Escherichia coil to alkylating agents , is positively controlled by Ada protein , the product of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_31">
Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada acts as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></label>
<input id="act_similarity_31", type="checkbox">
<div class="ACT_similar">
<li>Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_33">
J Biol Chem 270 : 6664-6670 Nakabeppu Y , Sekiguchi M ( 1986 ) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></label>
<input id="act_similarity_33", type="checkbox">
<div class="ACT_similar">
<li>Nakabeppu Y , Sekiguchi M ( 1986 ) : Regulatory mechanismsfor induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Nakabeppu , Y. and Sekiguchi , M. ( 1986 ) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Nakabeppu , Y. , and Sekiguchi , M. ( 1986 ) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li></div>
<li>However , one must take into account the fact that an additional feature of Ada is altered by methylation , since the transactivating capacity of Ada C69H is increased up to 6-fold in cells treated with methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Expression levels of C321A and C69H/C321A mutant proteins were almost the same as those of the wild type and C69H Ada , but the level of C69A/C321A Ada was much lower .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In cells expressing wild-type Ada , a considerable amount ( 773 Units ) of / % galactosidase activity was produced in the absence of M M S treatment , and the level increased about 14-fold after exposure of the cells to 1 mM MMS for 2 h .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The level of / ~ - galactosidase activity in cells with Ada C69H was almost equal to that of cells with wild-type Ada , and the activity in both types of cells increased 5 - to 7-fold after MMS treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Although the zinc-binding potential of aspartate ( Vallee and Auld 1990 ) may be responsible for its reasonably high degree of stability , Ada C69D remains inactive as a transactivator , probably due to its inability to-bind to the promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Lrp-mediated repression of aidB is abolished by high concentrations of leucine , which inhibits Lrp binding to the promoter region ; binding of the transcriptional-activator '' `` Ada to Pa , ds also results in dissociation of Lrp from the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Dissociation of Lrp from the aid5 promoter is observed even in the presence of a twofold excess of Lrp over meAda ( data not shown ) , suggestingthat the displacement of Lrp is mediated by binding of meAdato PaidSrather than protein : protein interaction and titration of Lrp by the transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>meAda binds to its DNA site via its NTD and stimulates transcription initiation ( black arrow ) via protein-protein interaction between its own CTD and the CTD of s70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Direct protein-protein interaction between meAda and RNA polymerase is also presumed to be necessary for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>As a result , on the basis of our in vitro and in vivo observations , it appears that sufficient levels of unmethylated Ada will inhibit methylate suggests that the mechanism of inhibition does not require a supercoiled template or any accessory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Recent results indicate that the Ada regulon is indeed induced during stationary phase and protects against active alkylators produced by nitrosation of amino acids in nongrowing cells ( 55 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In contrast , GW7105 cells containing either parental plasmid pBR322 or the plasmid pDS415 , which expresses only the N-terminal domain of Ada , exhibit a mutation frequency that increases by at least 2 orders of magnitude in the presence of 1 mg of MNNG per ml and by 3 orders of magnitude after the addition of 10 mg of MNNG per ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In addition , this observation suggests that the ability of Ada to recognize and bind to an Ada box , a property of the N-terminal domain , is unimpaired in these eight Ada derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>AB LE T Op Pr Reg Act Ada ada 0 0 0 1,2 126 AraC araBAD 1 1 0 1,2 59 , 70 AraC araC 1 1 0 1,2 48 AraC araE 0 1 0 2 145 CRP cat 0 0 0 2 72 CRP ccd 0 1 p 2,4 165 CRP colElpl 1 1 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Effect of Singly Methylated Forms of Ada Protein on Protein Synthesis Directed by the ada Gene in Plasmid pCS66 Experimental conditions were as described in Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In in Gtro reconstituted systems , both adn and alkA transcripts were formed , in an Ada prot ' eindependent manner ( Teo et al. , 1986 ; Kakabeppu dz Sekiguehi , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Carcinogenesis 13:837 -843 Demple B ( 1986 ) Mutant Escherichia coil Ada proteins simultaneously defectivein the repair of O6-methylguanineand in gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Demple B ( 1986 ) : Mutant Eschen ` chiu coli Ada proteins stimultaneouslydefective in the repair of 06-methylguanine and in gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Demple B. Mutant Escherichia coli Ada proteins simultaneously defective in the repair of O6-methylguanine and in gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>We will focus on Ada-dependent induction of the adaptive response genes , ﬁne tuning of individual gene expression according to the growth-phase , and the role played by Ada in shutting off the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Instead , once repair is completed , cleavage of Ada would result in more rapid turn-off of the adaptive response gene expression [ Sedgwick , 1987 ; Sekiguchi and Naka-beppu , 1987 ; Teo et al. , 19861 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Given that the decrease in MNNG mutation frequency was not accompanied by an increase in survival , oxyS is also not acting through the Ada regulator ( Table 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>As predicted , overproduction of Ada in GM56 Δada Δogt results in a striking increase in survival after MNNG exposure , equivalent to that of GM56 at the permissive temperature ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>Ada ( f ) Binding of mutant Ada proteins to the promoter Takano et al. ( 1988 ) constructed mutant forms of Ada proteins , Ala69 and Ala321 , in which each of the methyl-accepting cysteine residues was altered to alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>It seems that the cysteine residue at position 69 is important for accepting the methyl group and to maintain the conformation of Ada prot ' ein so as to exhibit the basic transcriptionpromoting activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Ada has two active methyl acceptor cysteine residues , Cys-69 and Cys-321 , that are required for demethylation of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Thus , substitution of the cysteine residue 321 by alanine renders Ada protein constitutively active , regardless of whether or not the Cys 69 residue is intact or is substituted by histidine 97-69 - 46 ¬ 30 ¬ 14.3 ¬ Fig. 4a , b Methyl acceptor activity - of C69H Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Residue K593 of 70 has been found to be important for several transcription activators , including AraC , UhpA , cI , FNR , Ada , RhaR ( this study ) , and RhaS ( 4 , 24 , 27 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Consistent with this , neither AraC nor Ada have a negatively charged residue that aligns with RhaS D241 , and in both of these cases s70 R599A was not defective for activation ( 33 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>In contrast , the ability of these same derivatives to activate alkA expression remains inducible by methylating agents just as with wild-type Ada ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Genetic studies have demonstrated that Ada derivatives lacking 10-20 % of the carboxyl terminus promote high levels of expression of the ada gene in the absence of a methylating agent but their ability to activate alkA expression remains inducible by methylating agents just as with wild-type Ada ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The lower panel depicts the interaction of two activated forms of Ada with the operators of the & - alkB operon , and the alkA and aidB genes , resulting in induced expression of all four genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>is presumed to result in a conformational change and converts Ada into a strong transcriptional-activator of its own gene ( ada ) and also of several other genes ( alkB , alkA , and aidB ) ( 17 , 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Ada is also positive transcriptional-regulator of the alkA and other BER genes [ 19,30 ] and this response can not be mounted in the Δada mutant we have used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>Results of DNA-Dependent Positive Autoregulation Ada Protein Synthesis of Virology , Royal Postgraduate Medi ¬ ' Present address : Department cal School , London , England .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Thus , the methyl-ated phosphates in DNA serve as the signal that converts Ada to its transcriptionally active form , which , in turn , induces the Ada regulon , resulting in increased alkylation repair activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>As expected , a number of previously identified DNA alkylation repair and recombination proteins were identified in our set of 99 MMS-toxicity modulating proteins , including Ada , AlkA , AlkB , and RecA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>Expression of these genes is regulated by Ada , and their induction provides protection against alkylation damage to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>MMS , apart from the Ada response , induces the SOS system that increases expression of over 40 genes involved in DNA recombination , repair , replication and mutagenesis [ 25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>O6-meG is a product of N-methyl-N ' - nitro-N-nitrosoguanidine ( MNNG ) action on DNA in E. coli and is acted upon by the Ada and Ogt methyltransferases to remove the methyl group [ 19,20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>Methylation of Ada was performed as in Landini and Volkert ( 1995a ) , except that the concentration of methylated calf-thymus DNA was lowered to 0.06 pg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Figure 3a shows transfer of the methyl group from methylated DNA to wild-type and C69H Ada proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Expression of Ada response starts when a methyl group from S ( but not R ) diastereoisomer of methyl phosphotriester from DNA is transferred onto Cys-38 of Ada , or when Cys-38 of Ada is methylated directly by SN2 methylating agents ( e.g. , methyl methanesulphonate , MMS , used in this work ) [ 9,10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Therefore , when the acceptor site for base demethylation is occupied , Ada is not activated to perform its regulator function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Presumably , a single Ada prot , ein molecule binds to the adn regulatory sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Elements present in the carboxyl terminus of Ada appear to be necessary for this negative regulatory function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>In contrast , the K593 , K597 , and R603 residues , which appear to be the target amino acids for activation by Ada at the alkA promoter , are not conserved in sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Transcriptional regulation of an ada-lacZ chromosomal operon fusion is therefore dependent upon the level of IPTG , which controls the concentration of Ada produced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>RESULTS Promoter Elements Determinant for Selective Recognition of the aidB Promoter by E S -- The aidB promoter ( PaidB ) is S strongly - dependent in the absence of Ada activation , i.e. in physiological growth conditions ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12853450" target="_blank">12853450</a>]<br></li>
<li>Furthermore , transactivation of the ada promoter by Ada C69H was enhanced up to 6-fold by treatment with methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Portions of one helix recognize the 235 element and are preceded and followed by resi-dues that affect activation by proteins that bind near 235 , such as activation of lPRM by cI protein ( 24 ) , of the ada and aidB promoters by Ada ( 23 ) , of the pho regulon promoters by PhoB ( 21 ) , and of the phage Mu middle promoter by Mor protein ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>In these conditions , both '' ` Ada and a bind specifically to the aidB and ada promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>This indicates that the zinc-binding capacity of histidine at residue 69 is likely to be sufficient for Ada to recognize and bind to the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Transcription of the mutant ada promoter by the wild-type enzyme was activated about eightfold by Ada , but virtually no activation was observed with the mutant enzymes ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Because this substitution also replaces the Ada binding site , ada-dependent activation is almost completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>However , more recent data indicate that RNA polymerase binds to the 260 to 240 region of the ada and aidB promoters via its aCTD , regardless of the presence of Ada ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>However , in a previous report ( 17 ) , it was shown that RNA polymerase binds the ada and aidB promoters via aCTD , regardless of the presence of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>To either one or a mixture of DNA templates , E. coli RNA polymerase holo-enzyme was added , and the preparation was incubated at 37 ° C for 1 hr in the presence or absence of a purified preparation of Ada protein , in order to produce a transcrip-tion-initiation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>In the presence of either form of Ada , a stable ternary complex , ( me ) Ada-RNA polymerase-alkA , which is resistant to challenge with heparin is formed and can be detected in gel retardation assays ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>The binding affinity of unmethylated Ada to the alkA promoter region in the ternary complex is increased at least 25-fold ( Fig. 5 ) by the presence of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>In contrast to what is seen at the ada and aidB promoters , the a subunit of RNA polymerase does not bind the alkA promoter in the absence of the Ada protein ; however , the presence of either Ada or meAda together with a results in formation of ternary complexes detectable in band shift experiments ( Fig. 6 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>On the other hand , both forms of Ada protein were equally active in promoting transcription of alkA , when excess RNA polymerase was present in the reaction mixture ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>At alkA , no RNA polymerase-promoter binary complex in the absence of Ada is detectable , and both forms of the Ada protein appear to increase RNA polymerase binding afﬁnity to the promoter region ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Furthermore , since all the Ada derivatives are capable of activating alkA transcription in response to MNNG treatment , this result implies that their methylphosphotriester DNA methyltransferase activities are active and that they are capable of recognizing and binding to an Ada box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>First , this observation provides support that methylphosphotriester DNA methyltransferase activity functions normally since transfer of a methyl group to Cys-69 is necessary to activate Ada for its role in alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Although the DNA binding function of the a subunit is required for binding of RNA polymerase to the alkA promoter , this requirement does not exclude the possibility that transcription is activated by Ada through a Class I transcriptional-activator mechanism , as proposed ( 1 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>This DNA-binding domain has also been shown to be involved in transcription activation in a number of AraC/XylS family proteins including Ada , RhaS , AraC , MelR , MarA , SoxS , XylS , and UreR ( 1 , 4 , 5 , 15 , 16 , 20 -- 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>It is possible that Ada CTD is responsible for this interaction ; deletions in Ada CTD abolish transcription activation at the ada promoter without affecting DNA binding by meAda ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The N-terminal half of Ada , which includes the AraC/XylS family domain , is capable of binding to DNA and activating transcription at the alkA promoter but is not sufﬁcient for transcription activation at promoters where Ada binds upstream of the 235 region ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Methylation of Cys-69 , in the N-terminal domain , triggers specific DNA binding by Ada and is required for transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Regulation of ada and alkA transcription in cells a containing separate Ada N-terminal and C-terminal domains b-Galactosidase activity of strain ada9-lacZ ( OD600 ) Fold alkA9-lacZ ( OD600 Fold 2MNNG 1MNNG induction 2MNNG 1MNNG induction Plasmid 1.0 3.8 6.4 1.0 9.2 6.0 119 147 6.0 103 6.0 31 23 6.0 11 1.1 6.0 1.2 1.2 1.2 60 52 pBR322 pDS411 1,302 82 52 68 216 66 45 59 pDS415 pBS179 pBS178/179 Ada , these methylated Ada molecules , because of Ada 's auto-regulatory circuit , continue to activate ada transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>We examined the ability of these Ada derivatives to activate transcription at a b-Galactosidase assays were performed as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>As shown in Table 2 , seven of the eight Ada derivatives altered at position 178 ( Leu , Val , Gly , Tyr , Arg , Cys , and Ser ) showed a partial loss of their ability to activate ada transcription in response to MNNG treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Furthermore , no inhibition was demonstrated at the alkA promoter by higher concentrations of unmethylated Ada in vitro or in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Both Ada and meAda proteins are able to activate transcription of alkA in vitro ( 9 ) , although higher concentrations of Ada are required ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>In vitro transcription experiments ( Fig. 4A ) indicated that unmethylated Ada could activate alkA transcription when present at sufficiently high concentrations , but higher levels of unmethylated Ada did not inhibit alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Nevertheless , in the presence of methylated Ada , transcription by ErrSis significantly stimulated , indicating that '' `` Ada is able to interact with both forms of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Despite these differences , both unmethylated and methylated Ada share the ability to stimulate a binding and trigger more efficient transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Both the unmethylated and the methylated forms of Ada ( Ada and meAda ) stimulated in vitro transcription , although the highest levels of transcription occurred with meAda ( Fig. 2 , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>In vitro transcription in the presence of Lrp and methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Dark grey bars , transcription levels in the absence of Ada ; light grey bars , transcription levels in the presence of unmethylated Ada ; black bars , transcription levels in the presence of methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>In light of our observation that higher concentrations of unmethylated Ada protein inhibited transcription at the ada promoter , we wondered whether Ada derivatives lacking 10-20 % ofthe carboxyl-terminal end are constitutive activators of ada transcription , at least in part , because they have lost their ability to cause this inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The experiments presented here show that concentrations of unmethylated Ada corresponding to -200 -500 molecules per cell activate ada transcription in vitro , but at higher concentrations , ada transcription is unexpectedly inhibited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>In addition , physiologically relevant concentrations of the unmethylated form of Ada speciﬁcally inhibit methylated Ada-promoted ada transcription both in vitro and in vivo and it has been suggested that this phenomenon plays a pivotal role in the down-regulation of the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The results ofthese studies indicated that unmethylated Ada could inhibit the ability of methylated Ada to activate transcription at the ada promoter in vivo as well as in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>We have previously demonstrated that physiologically relevant concentrations of unmethylated Ada protein are able to inhibit methylated Ada-promoted ada transcription both in vitro and in vivo ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Collectively , these in vitro and in vivo observations support our conclusion that concentrations of unmethylated Ada within the physiologically relevant concentration range can inhibit transcription at the ada promoter activated by meth-ylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>In contrast , the same concentrations of unmethylated Ada do not inhibit the activation of aLkA transcription by methylated Ada , either in vitro or in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>We have found that physiologically relevant higher concentrations of unmethylated Ada are able to inhibit the activation of ada transcription by methylated Ada , both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>We suggest that , at concentrations of IPTG of 30 LM and higher , unmethylated Ada molecules are inhibiting activation of transcription at the ada promoter by methylated Ada molecules just as they do in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Before examining the effect of higher concentrations of methylated Ada on ada transcription , we tried a variety of strategies for purifying the methylated form of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The higher levels of `` competitor '' M. luteus DNA resulted in a decrease in the levels of both ada and lacUV5 transcription at the highest concentrations of methylated Ada tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>To test the physiological relevance of these results , we utilized a system that enabled us to alter the ratio of unmethylated Ada to methylated Ada molecules in a living cell and examined the effects of this on transcription at the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>When all of the repairable methylphosphotriester lesions have been repaired via methyltransfer to Ada , methylated Ada mole-cules will continue to activate transcription at ada , generating higher levels of unmethylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>A threshold of unmethylated Ada molecules per cell that inhibits any further methylated Ada-promoted ada transcription is then reached .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Furthermore , in in vitro experiments we found that concentrations of unmethylated Ada corresponding to > 200 molecules per cell strongly inhibited ada transcription promoted by levels of methylated Ada corresponding to -10 molecules per cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>However , in contrast to unmethylated Ada , there was only a modest inhibition of ada transcription relative to lacUV5 transcription as methylated Ada was increased to 200 pmol per reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>As shown in Fig. 2 , the presence of > 20 pmol of unmethylated Ada per reaction inhibited ada transcription activated by 1 pmol of methylated Ada with transcription being mostly inhibited by 100 pmol of unmethylated Ada per reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The ada transcript , however , was not detected in the presence of nonmethylated Ada or a methylated N-terminal half-molecule of 20 kDa * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Therefore , once bound , the 20-kDa amino-terminal fragment would prevent methylated Ada from further activating ada transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_137">
When methylated at Cys-69 , the amino-terminal half of Ada will bind to the Ada box of the ada gene but activation of transcription does not occur ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></label>
<input id="act_similarity_137", type="checkbox">
<div class="ACT_similar">
<li>When methylated at Cys-69 , the amino-terminal half ofAda will bind to the Ada box ofthe ada gene but activation of transcription does not occur ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li></div>
<li>When methylated at Cys-69 , Ada is converted into a potent activator ofada and alkA transcription and binds to a sequence ( Ada box ) present in both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Thus , the methylated form of Ada binds the alkA site more efficiently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>During stationary phase , however , the methylated form of Ada shuts off expression of alkA ; this repression is speciﬁc for sS-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Expression of the ada and the related genes.such as alkA , is controlled by the Ada prot ' ein at , the t ' ranscriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Once methylated , Ada becomes able to activate the expression of the ada , alkA , alkB , and aidB genes , which confer cellular resistance to the mutagenic and cytotoxic effects of these agents ( 7 , 8 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>Upon self-methylation , Ada stimulates transcription of four genes , ada , aidB , alkA , and alkB ( whose transcription is directed by the ada promoter ) ( 10 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>At the ada and aidB promoters , the methylated form of Ada ( meAda ) is essential for transcription activation , whereas transcription from the alkA promoter is activated by both the unmethylated and methylated forms ( 4 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Overexpression of the unmethylated Ada in vivo is sufficient to induce alkA ( i.e. in the absence of alkylation damage ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>First , methylation of Ada is not necessary for alkA induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Higher Concentrations of Unmethylated Ada Do Not Inhibit Transcription of alkA in Vitro or in Vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The negative effect of meAda on EsS-dependent transcription of alkA is likely to have important physiological consequences for shutting off this component of the Ada regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The main features of transcription activation by Ada are summarized in Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The activator-independent , basal level of transcription of alkA was reduced substantially by the R265A mutation ( lane 6 ) , and no activation of trancription was detected by either Ada or meAda ( lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>The amount of factor-independent transcription attained by E d is roughly identical to the amount of Eo7 ' - dependent transcription in the presence of '' ' `` Ada ( compare Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>The a R265A-RNA polymerase was reconstituted in vitro and used in transcription of alkA in the presence or absence of Ada and meAda ( Fig. 2 , lanes 5 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>70 The presence of two distinct activation targets in s determines the ability by Ada to activate transcription by RNA polymerase containing sS ( EsS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>The fact that this cluster is not conserved in sS neatly explains why Ada is unable to activate transcription by EsS at the alkA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Moreover , the Ada protein is unable to activate transcription by EsS at the alkA promoter , consistent with the fact that the K593 , K597 , and R603 residues are not conserved in sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Enhancement of alkA transcription by Ada protein was also demonstrated ( for a review , see Sekiguchi & Nakabeppu , 1987 ; Lindahl et aZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>This promoter is not regulated by the Ada protein and does not affect levels of alkA transcription either in vivo or in vitro .2 The Ada protein was purified as in Saget and Walker ( 17 ) , and methylation of the Ada protein was obtained as in Nakabeppu and Sekiguchi ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Although the Ada protein is known to be unable to stimulate EsS-dependent transcription from the alkA promoter in vitro ( 7 ) , we expected some induction in vivo , since Es70 is also available .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Addition of either form of the Ada protein stimulated binding of Es70 to alkA ( Fig. 3 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>While either form of the Ada protein clearly activated transcription by Es70 ( lanes 1 to 3 ) , no stimulation of transcription by EsS was detected ( lanes 4 to 6 ) ; in fact , meAda displayed a weak negative effect on alkA transcription by EsS ( compare lane 4 to lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>The results summarized in Fig. 2 show that EsS is more efﬁcient than Es70 in carrying out transcription from the alkA promoter in the absence of Ada and that the methylated form of the Ada protein represses EsS-dependent transcription approximately threefold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>The results in Fig. 3 show that no interaction between Es70 and the alkA promoter was detected in the absence of Ada ( lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Finally , the elements of Ada required to activate alkA transcription ( also located in the N-terminal domain of Ada ) must be essentially normal in these Ada deriv-atives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The 20-kDa amino-terminal domain of Ada , in the presence of a methylating agent , is sufﬁcient to promote transcriptional activation at the alkA promoter in vitro and in vivo but will not activate transcription at ada ( 1 , 26 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_166">
Moreover , the amino-terminal half of the Ada protein is capable of activating transcription at alkA in the presence of a methylating agent but it is not sufﬁcient to activate transcription at ada ( 1 , 26 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></label>
<input id="act_similarity_166", type="checkbox">
<div class="ACT_similar">
<li>Moreover , the amino-terminal half of the Ada protein is capable of activating transcription at alkA in the presence of a methylating agent but is not sufficient to activate transcription at ada ( 20 , 22 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li></div>
<li>The truncated Ada proteins that constitutively activate ada transcription require additional activation in order to induce alkA transcription , again demonstrating that the regulatory regions required for alkA and ada induction are distinct and separable by mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>These observations indicate that elements in the carboxyl-ter-minal domain of Ada are necessary for the activation of ada transcription but not for the activation of alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Ada produced as separate domains can activate alkA but not ada transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In contrast , all of the eight Ada derivatives including AdaK178P exhibited normal MNNG-induced activation of alkA transcription ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In contrast , only 1 pmol of methylated Ada or meth-ylated AdaK178P was necessary to induce signiﬁcant alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Thus , we propose that Ada activates alkA transcription via direct interaction 70 with s region 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Further , they seemed as efﬁcient as wild-type Ada in weakly activating alkA transcription in the absence of MNNG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Leucine shows little or no effect on Ada binding ( Fig. 5 , lanes 8-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Ada binding was competed for by a 200-fold excess of the unlabeled fragment but not by the presence of 1 , ug of pUC18 DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>Our results provide no support for the cleavage of the Lys-178 -- Gln-179 bond being involved in Ada inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>An alternative hypothesis is that the Ada protein can be inactivated in vivo by proteolytic cleavage at its Lys-178-Gln-179 bond ( 4 , 5 , 25-28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The solution structure of the methylated form of the N-terminal 16-kDa domain of Escherichia coli Ada protein HIROTO TAKINOWAKI , YASUHIRO MATSUDA , TAKUYA 1 1 YOSHIDA , 1 YUJI KOBAYASHI , AND TADAYASU OHKUBO 1,2 1 1 Graduate School of Pharmaceutical Sciences , Osaka University , Osaka 565-0871 , Japan 2 Osaka University of Pharmaceutical Sciences , Osaka 569-1094 , Japan ( RECEIVED August 16 , 2005 ; FINAL REVISION November 19 , 2005 ; ACCEPTED November 25 , 2005 ) Abstract Reprint requests to : Tadayasu Ohkubo , Graduate School of Pharmaceutical Sciences , Osaka University , 1-6 Yamadaoka , Suita , Osaka 565-0871 , Japan ; e-mail : ohkubo@protein.osaka-u.ac.jp; fax : 81-6-6879-8221 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Abbreviations : N-Ada16k , the N-terminal 16-kDa domain of the Ada protein ; meC38 N-Ada16k , the Cys38 methylated form of N-Ada16k ; MTase , methyltransferase ; HTH , helix-turn-helix ; NMR , nuclear magnetic resonance ; MALDI-TOF MS , matrix assisted laser desorption/ioni-zation time of flight mass spectrometry ; MNU , methylnitrosourea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Expression of the Escherichia coli Ada Regulon in Stationary Evidence for rpoS-Dependent Negative Regulation of alkA Transcription Phase : PAOLO LANDINI1 ,2 * AND STEPHEN J. W. BUSBY2 Swiss Federal Institute for Environmental Science and Technology ( EAWAG ) , 8600 Dübendorf , Switzerland ,1 and School of Biosciences , The University of Birmingham , Edgbaston , Birmingham B15 2TT , United Kingdom2 Received 28 May 1999/Accepted 30 July 1999 The Escherichia coli Ada protein activates s70-dependent transcription at three different promoters ( ada , aidB , and alkA ) in response to alkylation damage of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Ada pro- appears that ada is aregulatory gene for the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>We have recently shown that the a subunit of RNA polym-erase binds to the promoters of the adaptive response genes ada and aidB in the absence of Ada protein ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>DISCUSSION Expression of the ada and alkA genes , which are involved in adaptive response in E. coli , is positively regulated by Ada protein , which is encoded by ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>One of these is by the induction of the adaptive response by the Ada protein that acts as a methyl transferase ; in its methylated form , Ada acts as a transcriptional-regulator of the Ada operon genes ada , aidB , alkA , and alkB ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>175 , No. 8 The Ada Protein Is a Class I Transcription Escherichia coli Factor of KUNIHIKO SAKUMI ,1 KAZUHIKO IGARASHI ,2 MUTSUO SEKIGUCHI , ' AND AKIRA ISHIHAMA2 * Department of Biochemistry , Medical Institute of Bioregulation , Kyushu University , Fukuoka 812,1 and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka 411,2 Japan Received 10 September 1992/Accepted 5 February 1993 The methylated Ada protein of Escherichia coli , a regulatory protein for the adaptive response , binds to a target DNA from positions -62 to -31 upstream of the ada gene and facilitates the binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_187">
Transcriptional activation of the Escherichia coli adaptive response gene aidB is mediated by binding of methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></label>
<input id="act_similarity_187", type="checkbox">
<div class="ACT_similar">
<li>Landini , P. and Volkert , M.R. ( 1995 ) Transcriptional activation of the Escherichia coli adaptive response gene aidB is mediated by binding of methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Landini , P. , and Volkert , M. ( 1995a ) Transcriptional activation of the Escherichia coli adaptive response gene aidB is mediated by binding of methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li></div>
<li>In Escherichia coli , exposure to sublethal doses of DNA-methylating agents like methyl methanesulfonate ( MMS ) , N-methyl-N ' - nitro-N-nitrosoguanidine ( MNNG ) , and N-methyl-N-nitrosourea ( MNU ) activates the expression of several genes via positive regulation mediated by Ada protein in a process termed adaptive response ( 6 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>The Ada protein acts as both a positive and a negative modulator of Escherichia coli 's response to methylating agents ABSTRACT The adaptive response of Escherichia coli protects the cells against the toxic and mutagenic effects of ceilain alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_192">
The Ada protein acts as both a positive and negative modulator of Escherichia coli 's response to methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></label>
<input id="act_similarity_192", type="checkbox">
<div class="ACT_similar">
<li>The Ada protein acts as both a positive and a negative modulator of Escherichia coli 's response to methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li></div>
<li>( i ) The mutant enzymes are defective in response to activation by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>The expression of the adaptive response is controlled by the multifunctional 39-kDa Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The Ada protein or some modified or processed form of it is required for the adaptive response of E. coli to methylating and ethylating agents , a response that includes the induction of 06-methylguanine-DNA methyltransferase and 3-methylad-enine-DNA glycosylase II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>The aidB gene belongs to the adaptive response to DNA alkylating agents and is activated by the Ada regulatory protein ( 29 ) ( for reviews , see Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12853450" target="_blank">12853450</a>]<br></li>
<li>The adaptive response of Escherichia coli to alkylating damage of DNA is regulated by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Our results suggest that the Ada protein plays a pivotal role in the negative modulation of Its own synthesis and therefore in the down-regulation of the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Thus , at the alkA promoter , the Ada protein can act as both a positive and negative modulator of the adaptive response to alkylation damage , depending on the cell 's physiological state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>J Bacteriol 170:5263 -5271 Shevell DE , Friedman BM , Walker GC ( 1990 ) Resistance to alkylation damage in Escherichia coli : role of the Ada protein in induction of the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_202">
[ 6 ] D.E. Shevell , M.B.M. Friedman , G.C. Walker , Resistance to alkylation damage in Escherichia coli : role of the Ada protein in induction of the adaptive response , Mutation Res .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></label>
<input id="act_similarity_202", type="checkbox">
<div class="ACT_similar">
<li>Resistance to alkylation damage in Escherichia coli : role of the Ada protein in induction of the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li></div>
<li>In E. coli , the Ada protein mediates the response to alkylation damage , which has been called the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>Multiple functions of the Ada protein in the adaptive response to alkylation damage in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Since the adaptive response takes place in ada ' , but not ada - , cells that have been treated with methylating agents ( 6 , 11 , 13 , 17-19 ) , it seemed likely that Ada protein , which itself is a methyltransferase and accepts the methyl group , would be a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>Thus the current model of regulation of the adaptive response is that Ada protein first interacts with methylated DNA , removing methyl groups from damaged bases and phosphotriesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>The Ada protein is the key enzyme of this process ; Ada acts both as a methyltransferase able to remove methyl groups from damaged DNA and as a transcriptional-activator for the adaptive response genes ( 13 , 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>The product of the ada gene , the Ada protein , plays a dual role in the adaptive response ; Ada transfers methyl groups from DNA to two of its cysteine residues , thereby functioning as a DNA repair protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>In the adaptive response , Ada protein plays important roles both in regulation of gene expression and in repair of alkylated DNA ( reviewed in Sekiguchi and Nakabeppu 1987 ; Lindahl et al. 1988 ; Shevell et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Moreover , methyltransfera ase activity for DNA phosphotriesters , induced as part of the adaptive response ( McCarthy et al. , 1983 ) , has also been shown to reside in the Ada protein ( T.McCarthy , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6092060" target="_blank">6092060</a>]<br></li>
<li>This is consistent with the proposal of Teo ef al. ( 1986 ) , that transfer of the methyl group from methylphosphotriester of DNA to Ada protein is a trigger for induction of the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>f ( d ) Biological consequences of repair o DNA Since both the ada and alkA gene products are known to be inducible as part , of an adaptive response ( Demple et al. , 1982 ; Evensen & , Seeberg , 1982 ) , an in-vitro transcription system was used to examine the ability of mutant Ada proteins to promote transcription of the ada and aEkA genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Methylation 4 of a cysteine residue in N-Ada20k dramatically enhances a sequence-specific DNA binding affinity of the Ada protein for the promoter regions of its own gene , ada , and other alkylation resistance genes , alkA , aidB , and 3 alkB , by 10 - fold ( Teo et al. 1986 ; Rebeck et al. 1988 ; Sakumi and Sekiguchi 1989 ; Akimaru et al. 1990 ; Myers et al. 1992 ; Sakashita et al. 1993 ; He and Verdine 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>- sequence ( -- 53 to -46 ) result , ed in a dec * rrased potential t , o initiate the ada transcription ( Nakamura et al. , 198X ) , phosphate bonds at -19 and - 48 were not pro & ted by t , he met ~ hylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The cloned ada and alkA genes and purified intact Ada protein were used to analyze the role of Ada in the regulation of the adaptive response in studies by Teo et al. [ 1986 ] and Nakabeppu and Sekiguchi [ 1986 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>In in-vitro reconstituted systems , bot , h ada and alkA transcripts were formed in an Ada protein-dependent manner ( Teo et al. , 1986 ; Nakabeppu & Sekiguchi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>When Ada is methylated at position 69 ( reAda ) , it can bind to specific DNA sequences in the promoter region of the adaptive response genes , and activate their transcription ( Nakabeppu and Sekiguchi , 1986 ; Teo et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>When methylated Ada binds to this sequence , transcription from both ada and alkA promoters is stimulated , as measured by run-off transcription assays [ Nakabeppu and Sekiguchi , 1986 ; Teo et al. , 19861 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>The methylated form of Ada protein acquires the ability to-bind to the promoter regions of genes belonging to the Ada regulon , and enhances initiation of their transcription ( Nakabeppu et al. 1985a ; Teo et al. 1986 ; Nakamura et al. 1988 ; Sakumi and Sekiguchi 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>These two studies are in agreement that Ada protein stimulates transcription from the ada promoter only when methyl-ated , although they disagree as to whether only methylated Ada [ Teo et al. , 19861 or both methylated and unmethylated Ada [ Nakabeppu and Sekiguchi , 19861are capable of stimulating transcription from the aZkA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>This notion is supported by the recent findings that Ada protein , especially in its methylated form , promotes transcription of the ada gene in vitro ( Teo et al. , 1986 ; Nakabeppu & Sekiguchi , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Positive and negative regulation of transcription by a cleavage product of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_224">
References Akimaru , H. , Sakumi , K. , Yoshikai , T. , Anai , M. and Sekiguchi , M. ( 1990 ) Positive and negative regulation of transcription by a cleavage product of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></label>
<input id="act_similarity_224", type="checkbox">
<div class="ACT_similar">
<li>References Akimaru H , Sakumi K , Yoshikai T , Anai M , Sekiguchi M ( 1990 ) Positive and negative regulation of transcription by a cleavage product of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li></div>
<li>They found that interaction of Ada protein with synthetically constructed 06-meG containing DNA , which results in methylation of the cysteine residue at position 329 of Ada , does not activate it to stimulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Results of the in vitro transcription experiments demonstrate that EDs-dependenttranscrip-tion can be stimulated by methylatedAda protein.Another activator might sharewith Ada the ability of stimulatingED ' as well as the mechanismfor bypassing Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>When transcription was performed in the absence of Ada protein , the result shown in Figure 9 ( b ) was obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>To elucidate the role of specific methylation of Ada protein in regulation of expression of the inducible genes , we performed in-vitro transcription experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>pAT2101 is a derivative of pATada lacking the coding region for Ada protein , which served as control vector , pTV2101 , pTVada , pTV69H , pTV321A , and pTV69H321A were constructed by replacing the replication origins of pAT2101 and its derivatives with that of the high copy-number plasmid , pTVllgN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Proc Natl Acad Sci USA 88 :9001 -9005 Shevell DE , LeMotte PK , Walker GC ( 1988 ) Alteration of the carboxyl-terminaldomain of Ada protein influences its inducibility , specificity , and strength as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Altera-tion of the carboxyl-terminal domain of Ada protein influences its inducibility , specificity , and strength as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2180908" target="_blank">2180908</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_233">
Alteration of the carboxyl-terminal domain of Ada protein inﬂuences its inducibility , speciﬁcity , and strength as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></label>
<input id="act_similarity_233", type="checkbox">
<div class="ACT_similar">
<li>Alteration of the carboxyl-terminal domain of Ada protein inﬂuences its inducibility , speciﬁc ¬ ity , and strength as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li></div>
<li>We propose that the transcrip-tion-regulating function of Ada protein is under dual control by methylation of cysteine residues at positions 69 and 321 ; the former enhances DNA binding , while the latter enhances the transactivating capacity of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It has been shown that C321A Ada protein has a high level of transactivating capacity in unadapted cells , but its DNA binding activity is further enhanced by treatment with alkylating agents ( Takano et al. 1988 ; Sakumi and Sekiguchi 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It has been proposed that an increase in the DNA-binding ability of Ada protein is the main cause of the acquisition of transcriptional-activator capacity ( Teo et al. 1986 ; Myers et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Formation of S-methylcysteine at position 321 itself may not lead to an increase in DNA-binding affinity , since an Ada protein accepted a methyl group only from 0 6-methylguanine did not show an increase in its DNA-binding activity in vitro ( Teo et al. 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>To determine whether the overproduced Ada protein had the ability to accept the methyl group from alkylated DNA , an extract from induced YN3 cells harboring pYN3059 was incubated with [ 3H ] MNU-treatedDNA and thensubjected to SDS-gel electrophoresis followed by fluorography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>These data , together with those shown in Figure 3A , strongly indicate that transfer of a methyl group to the Ada protein from a phosphotriester rather than from 06-methylguanine is the critical event in the generation of an activated form of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Ada protein has been demonstrated to accept a methyl group from methyl-phosphotriester thereby acquiring the ability to-bind tightly to the Ada box of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It is likely that the C321A Ada protein adopts an activated conformation , which is acquired by wild-type Ada protein only upon addition of a methyl group to Cys321 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The ability to-bind DNA both speciﬁcally and nonspeciﬁcally is a similarity to the Ada protein , which binds DNA nonspeciﬁcally in its unmethylated form and becomes able to recognize speciﬁc target sequences upon self-methylation ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>AlkA protein functions as a glycosylase that repairs N3-meA , N3-meG , O * - meT , and 02-meC residues in DNA , and Ada protein functions as an alkyltransferase that removes alkyl groups from 06-meG , 04-meT residues as well as methylphosphotriesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>The intact Ada protein acts as an alkyltransferasethat demethylates methylphosphotriesters as well as 06-meG and 04-meT in DNA [ Margison et al. , 1985 ; McCarthy et al. , 1983 ; McCarthy et al. , 1984 ; McCarthy and Lindahl , 19851 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Once Ada protein transfers a methyl group from the methyl-phosphate to the Cys-69 resi-due , it becomes a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Ada protein transfers methyl groups from damaged DNA to two of its own cysteine residues ; in doing so , it becomes a transcriptional-activator ( 17 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>The latter finding suggests that the cysteine residue at position 321 , which is located near the C terminus of the Ada protein , IS involved in regulating activity , as the transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Tt is evident t , hat the cysteine residue at , position 69 is important for accepting the methyl group and for maintaining the conformation of Ada protein as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Using a reconstituted syst 'em with purified components , we demonstrated that met , hylation of the cysteine residue at position 69 is responsible for activation of the Ada protein , as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>This activation of Ada protein is achieved by transfer of the methyl group from one of the stereoisomers of methylphosphotriester of DNA to cysteine residue 69 of Ada protein , by intrinsic methyltransferase activity ( Sedgwick et al. , 1987 ; Takano et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>I tt addition to the transcriptional-regulator activity , the Ada protein possessestwo methyltransferase activities , one to transfer a methyl group from one of two stereoisomers of methyl-phosphotriesters of methylated DNA to a cpsteine residue at position 69 of the Ada protein and t ) he ot , her to transfer a methyl group from 06-met .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Adaptive-response induction occurs when the Ada protein transfers a methyl group from methylphosphotriesters formed in DNA by methylating agents to its own Cys-69 residue ( for reviews , see references 12 , 15 , and 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>The mode of activity control by NemR repressor is thus similar to that of Ada protein , which repairs DNA methyl phosphotriester lesions by transferring the methyl group to its Cys residue , thereby acting as a positive regulator for its own synthesis as well as other DNA alkylation response genes ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567656" target="_blank">18567656</a>]<br></li>
<li>The presence of positively charged amino acids in the proximity of the cysteine residues that function as methyl acceptor sites was proposed to be important for interaction with DNA and DNA repair activity of the Ada protein ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The methyl phosphotriester in DNA stereospecifically recognized by the Ada protein has been demonstrated by nuclear magnetic resonance experiments to be in the Sp configuration ; this methyl group protrudes from the DNA double helix rather than being accommodated in a groove ( Weinfeld et al. , 1985 ; Hamblin and Potter , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Acquisition of a methyl group at the Cys69 residue must cause a subtle conformational change in the Ada protein so as to provide easier access to the DNA interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The C69H Ada protein has the capacity to accept a methyl group only from O6-methylguanine and O4-methylthymine resi-dues in methylated DNA , and the enhanced transactivating capacity is attributable to the acquisition of a methyl group at the Cys321 residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The E. coli Ada protein , which regulates the expression of genes in response to DNA methylation , is activated by the transfer of a methyl group from the DNA to the Ada protein ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The data described above suggest that transfer of a methyl group from DNA to Ada protein can activate Ada as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>The purified 38-kDa Ada protein catalyzed transfer of the methyl group from alkylated DNA to its own protein molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>The methyl group of 06-methylguanine in DNA is transferred to cysteine-321 of the Ada protein by its own methyltransferase activity ( 4 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>group from the OB-methylguanine residuealokfylated The presentstudies were undertaken to purify andto DNA to the Ada protein , per 86 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Activation of Ada Protein Requires Methylation of a Specific Cysteine Residue The most important enzymatic function of the Ada protein is to counteract alkylation mutagenesis by direct reversal of 06-methylguanine in DNA to unsubstituted guanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>( iii ) The direct effect of the Ada protein is to induce an altered promoter , perhaps by allowing DNA curvature , which is recognized by the oa subunit but is little recognized by the mutant ax .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>These two activities are closely tied to one another , as the Ada protein must be activated to perform its regulatory function and activation is a consequence of its DNA repair activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Different types of DNA damage are detected by Ada and RecA , and activation of each protein will initiate transcriptional responses that facilitate cell survival after alkylation damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>The t * wo reactions occur independently and lead to different ) biological consequences ; the former is responsible for removing premutagenic and lethal lesions from the DNA , and the latter for converting Ada protein to an active form as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Not only does the Ada protein possess two DNA repair activities but it is also a very potent transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The Ada protein is both a DNA repair protein and a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>The major effector-molecule regulating this response is the 39-kDa Ada protein , which functions as both a DNA repair protein and a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Amounts of the 38-kDa Ada protein decreased while bands for smaller polypeptides ( 20,18 , and 16 kDa ) increased onincubation of the extracts in buffers whose pH is 7.4 and which contain high concentrations of NaCl and 2-mercaptoethanol ( lanes K and M ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>We found , in the present study , that this processing is dependent on both pH and ionic strength of extraction buffer ; the 38-kDa Ada protein was fairly stable in low ionic strength buffer at pH 8.5 while it was rapidly degraded when the extractwas incubated in the presence of salts at pH7 .4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>In our experiment using a multicopy plasmid system , in which a relatively large amount of Ada protein is formed , we found that Ada C69H possesses transactivating capacity equivalent to that of wild-type Ada protein , in the absence of an alkylating agent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In contrast , Fig. 1 clearly shows that AdaK178P is as active as wild-type Ada protein in repair of both O6-methylguanine and methylphosphotriester substrates in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Moreover , Myers et al. ( 1993 ) showed that a truncated form of Ada ( residues 1-179 ) carrying a histidine substitution for Cys69 binds cadmium-113 ; thus it is reasonable to assume that C69H Ada protein binds zinc as well as does wild-type Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It has been shown that the S-methylcysteine residue in the methylated form of Ada , binds a zinc ion ; thus methylation may not cause a dramatic change in the structure of Ada protein ( Ohkubo et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Since wild-type Ada protein is also required for methyla-tion-induced expression of the aidB gene [ Volkert et al. , 19861 , Ada protein could function in the same fashion to regulate aidB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Since the a & gene product appears to induce its own synthesis in vivo , the pCS88-directed system was supplemented with different forms of the purified Ada protein in attempts to stimulate expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>We showed that synthesis of the ada gene product is induced by exposure to the methylating agent MNNG and that the Ada protein or some derivative of it acts positively in trans to increase expression of an ada-lacZ operon fusion , suggesting a stimulation oftranscription of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>It is evident , therefore , that both sigma 70 and methylated Ada protein are required for ada transcription , These two proteins would function in the initiation of transcription in a different manner ; the sigma factor binds to the RNA polymerase core enzyme to enhance the potential to link up with the proper promoter sequence , while the methylated TN266 ( C+G at -47 ) TN260 ( T+A at -34 ) G Ada MeAda Ada MeAda I 36b1361 G O A ` 0 1 3 6 '' 1 3 6 ' A ' 01 -61 -61 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Substitution of Cys321 by alanine does not increase the DNA-binding ability of Ada protein but does increase its ability to-bind to the ada promoter in the presence of RNA polymerase holo-enzyme in vitro ( Sakumi and Sekiguchi 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Previously , Sakumi and Sekiguchi ( 10 ) showed that the methylated form of Ada protein binds in vitro to the target DNA signal generally located upstream of the promoters of the ada regulon including its own gene and facilitates the binding of RNA polymerase to the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>As shown in Discussion The presence of the Ada protein , in its active ( methylated ) form , is necessary for transcriptional initiation at the ada and aidB promoters , and only in its presence can RNA polymerase produce an open complex ( Sakumi and Sekiguchi , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>However , binding affinity of RNA polymerase to the extended promoters of both ada and aidB is not affected by the presence of methylated Ada protein , as shown by both gel retardation and DNase I protection experiments ( Figures 3 and 4 , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Therefore , bound methylated Ada may facilitate RNA polymerase binding by direct protein-protein contact or by causing a conformational change in the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Because the formation of lac RNA drastically decreased with increasing amounts of the methylated Ada protein , it seems that most of-the active enzymes were associated with the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>The Ada protein , in me its methylated form ( Ada ) , can activate transcription at the aidB promoter ( PaidB ) by both the E 70 and E S forms of RNA polymerase ( 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12853450" target="_blank">12853450</a>]<br></li>
<li>The footprints showed that the methylated form of Ada protein binds t * o the adu promot ' er and promotes binding of the R'NA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Binding of methylated Ada protein and R , h ' A polymerase tu the ccda promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>the methylated Ada protein could bind to the promoter , but ' binding of RXA polymerase did not ) occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>RNA polymerase did bind to the promoter once the methylat , ed Ada protein had bound to the upstream sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Ada protein would bind to the upstream region of the promoter to facilitate the proper binding of the RSA polymerase to the promoter sequence , To clarify the mechanism of transcriptional activation by the methylated Ada protein , we investigated interactions of the protein with the ada promoter and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Presence of unmethylated Ada protein , which is inactive in transcriptional activation ( Teo et al. , 1986 ) , does not strongly affect the nature of RNA polymerase binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>RNA polymerase assays and in vitro protein synthesis in presence of Ada protein were performed as described by Teo et al. ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>We found that RNA polymerase could bind specifically to the promoters only in the presence of the Ada protein and that either RNA polymerase Ada or protein alone did not bind tightly to the promoters ( unpublished result ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_297">
Lanes 1-5 , no Ada protein and increasing concentrations of RNA polymerase ( 0 , 0.13 , 0.25 , 0.5 , 1 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></label>
<input id="act_similarity_297", type="checkbox">
<div class="ACT_similar">
<li>Lanes 1-4 , no Ada protein and increasing concentrations of RNA polymerase ( 0 , 0.25 , 1 , 4 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li></div>
<li>We directly examined whether higher concentrations of unmethylated Ada protein would inhibit transcriptional activation at the ada promoter mediated by methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Previous in vitro analyses of the roles of methylated Ada in transcriptional activation have utilized concentrations ofAda protein ( corresponding to 10-60 molecules per cell ) that are at the lower end of the concentration range that is physiologically relevant ( 12 , 19 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Methylation of cysteine at position 69 triggers a conformational change in the Ada protein ; cysteine-69-methylated Ada ( meAda ) is a speciﬁc DNA binding protein able to promote transcription of its own gene , ada , and of the alkB gene , whose transcription is also me driven by the ada promoter ( 25 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>`` The Ada protein with the cysteine residue at posit * ion 69 replaced by alanine was incapable of binding to the ada promoter even when the cysteine residue at position 321 of the protein was methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The methylated NTD of Ada as well as the unmethylated form of the protein have been shown to negatively regulate the ada promoter and were proposed to b involved in shutting off the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Since the methylated form of the amino-terminal cleavage product of Ada is able to-bind the ada protein but is incapable of promoting transcription , it seemed possible that it competes with the methylated intact Ada protein for binding to the ada promoter and thus plays a role in shutting off the adaptive response ( 17 , 25 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>It should be noted that there is an UP-element binding protein in E.coli , methylated Ada protein , which binds to an UP-like element sequence in ada and aidB promoters , and stimulates transcription by modifying the nature of the RNP ± promoter interaction ( Landini and Volkert , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Footprinting analyses revealed that t , he methylated Ada protein binds t , o a region from positions - 63 to - 31 , which includes the ada regulatory sequence AAAGCGCA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_307">
Transcription of the ada and alkA genes promoted by the methylated form of Ada involves the binding of the Ada protein to a sequence referred to as an Ada box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></label>
<input id="act_similarity_307", type="checkbox">
<div class="ACT_similar">
<li>Transcription of the ada and alkA genes promoted by the methylated form of Ada involves the binding ot the Ada protein to a sequence referred to as an `` Ada box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li></div>
<li>Upon addition of methylated Ada protein ( IOpmol ) , aid6 transcription is restored almost to the same extent as meAda-activatedtranscription in the absence of Lrp ( Fig. 6 , lanes 2 and 4 for En '' , and lanes 6 and 8 for EoS ) , suggesting that activation by meAdais dominant over repression by Lrp in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>When the promoter DNAs derived from wild type or base substitution mutants that showed normal expression in vivo were used as templates for transcription in vitro , the ada-specific RNA was formed in the presence of a methylated form of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Under these conditions only the 90-nucleotide ada transcript was formed , which was produced in the presence of the methylated Ada protein as a minor product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>By ada the addition of methylated Ada protein , transcription was markedly stimulated for the but ada wild-type enzyme , transcript was hardly detected for the mutant holoenzymes , even in the presence of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Ada protein plays a direct role in aidB regulation : methylated Ada is able to-bind to the aidB promoter region and to activate transcription from aidB in an in vitro transcription-translation system using crude E. coil extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>Taken together , a most plausible model would be that both methylated and unmeth-ylated forms of Ada protein function as positive regulators , though the former is more active than the latter.Thus , it would be of interest to examine effects of the two forms of Ada protein on in vitro transcription of the cloned alkA and ada genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Indeed , in contrast with the ada and aidB promoters , the unmethylated form of the Ada protein , as well as the methylated form of the AdaNTD , is able to activate transcription at alkA , although with a lower efﬁciency than the full-length methylated Ada protein ( 1 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Deletion of the carboxyl-trminal 67 amino acids of Ada abolished the ability of the unmethylated form of the protein to inhibit activation of ada transcription but not the ability of the methylated form to activate ada or alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>However , both methylated AdaNTD and unmethylated Ada protein can activate transcription at alkA , so that neither mechanism would result in down-regulation of this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>and the methylated forms of the Ada protein can activate alkA transcription in vivo and in vitro ( 9 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>While methylated Ada protein is able to induce transcription of th 200 POL , MLM 0 0 0 1 2.5 5 7.5 1 MNNG ( gg/plate ) FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>In reaction conditions in which methylated Ada protein clearly stimulated expression of the ada promoter , no Ada-induced transcription from the BglII-EcoRI fragment of aidB was detected ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>Once Ada had been methylated , strong activation of ada transcription was observed with as little as 1 pmol of methylated Ada protein per reaction as previously reported ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>( ii , ) This repair reaction gives rise to the methylated form of Ada protein , which acts on the ada promoter to enhance transcription of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>This level of methylated Ada protein strongly activates transcription at ada ( Figs. 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>It was shown that ada transcription by the holo enzyme depends on the methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>It was shown that the methylated form of Ada protein is more effective in promoting ada t , ranscription than is the unmethylated form , whilst the effects of both forms are much the same with regard t ' o aZkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>It was shown that the methylated form of Ada protein is more effective than the nonmrthylated form in promoting ada transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>A low but significant level of the alkA gene expression occurs in the absence of Ada protein , and unmethylated and methylated forms of Ada proteins are equally effective in promoting alkA transcription in vitro ( Nakabeppu & Sekiguchi , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>However , since the methylated protein strongly aggregated , we instead generated methylated Ada in situ ( 19 ) by treatment of small quantities of Ada with methylnitrosourea-methylated M. luteus DNA prior to using it in the in vitro transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>An in vitro transcription competition experiment analogous to that described above revealed that higher concentrations of unmethylated Ada do not inhibit methylated Ada protein-promoted alkA transcription ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>We then performed a transcription experiment in vitro in the presence of a methylated form of Ada protein ( see Fig. 9 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>We performed an in vitro transcription competition experiment using a constant amount ( 1 pmol per reaction ) of methylated Ada protein while increasing the level of unmethylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>We estimate that the methylated Ada protein is about 100 times more efficient as a transcriptional-activator than the unmethylated protein in these in vitro experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>We prepared a methylated form of Ada protein by an in vitro reaction and compared the activity with that of the normal , unmethylated form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>To obtain support for this assumption , we prepared a methylated form of Ada protein and compared its transcription-promoting activity with that of normal , unmethylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>A HindllllEcoRl DNA fragment of the ada gene ( Figure 4 ) radioactively labeled in the transcribed strand , was incubated in the presence of a 50-fold molar excess of unmethylated or methylated Ada protein and was briefly challenged with DNAase I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Analysis of Run-Off Transcripts from Hindlll-EcoRI Fragment Containing the ada Promoter Region ( A ) Lane 2 , transcription of the DNA fragment alone ; lane 3 , in the presence of unmethylated Ada protein ; lane 4 , in the presence of fully methylated Ada protein ; lane 1 , size markers , 5 ' 32P-labeled Hinfl digest of pSR322 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Analysis of Run-Off Transcripts from a 431 bp Rsal-Hincll Fragment Containing the alkA Promoter Region Lanes 1 and 2 , transcription in the presence of fully methylated Ada protein ; lane 3 , in the presence of unmethylated Ada protein ; lane 4 , in the presence of the DNA fragment alone ; lane 5 , size markers , 5 ' 32PVlabeled Hinfl digest of pBR322 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Reaction mixtures contained different forms of the Ada protein , as follows : lane 1 , unmethylated Ada protein ; lane 2 , doubly methylated Ada protein ( preincubated with alkylated DNA ) ; lane 3 , unmethylated Ada protein ( preincubated with unmethylated oligonucleotide ) ; lane 4 , singly methylated Ada protein ( preincubated with Ds-methytguanine-containing oligonucleotide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>When the DNA of pTN268 ( - C47 + G ) was used as template , no 98-nucleotide RNA was formed in the presence of methylated Ada protein but a large amount of 90-nucleotide RNA did form in the absence of Ada protein ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Formation of the 98-nucleotide transcript was greatly enha , nced by the methylated form of the wild-type Ada protein , and a lesser degree of stimulation was observed wit , h the nonmethylated form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>This singly methylated Ada protein , in contrast to the doubly methylated protein , was completely ineffective as an activator of ada-depen-dent protein synthesis ( Figure 3A ) 321 Lys ; Ala - LyS-Glr ~ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In contrast , the methylated form of the Ada protein , obtained by incubation of the intact purified protein with an excess of MNU-treated DNA , efficiently activated synthesis of 36S-labeled Ada protein ( Figure 2 , lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In the case of the wild type , more RNAs were formed with increasing concentrations of methylated DNA , the methyl donor for Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>In contrast , Ada protein singly methylated at Cys-321 is not capable of specific DNA binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>In addition to commercial reagents , reaction mixtures ( 30 JII ) contained 3-4 pg plasmid DNA , 1 Hg Ada protein ( methylated or unmethytated ) , and 30 t & i 35S-methionine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In this reaction , 98 % of the methyltransferase activity in the crude extract was inactivated , indicating that most of the Ada protein had been methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>The lanes represent reaction mixtures with lane 1 , no Ada protein ; lane 2 , unmethylated Ada protein ; lane 3 , methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The contents of the reaction mixtures in each lane ( see Experimental Procedures ) are as follows : lanes 1 and 7 , no Ada protein ; lane 2 , addition of methylated DNA but no Ada protein ; lane 3 , unmethylated Ada protein ; lane 4 , the unmethylated 19 kd domain of Ada protein ; lanes 5 and 6 , methylated Ada protein ; lanes 6 and 9 , methylated 19 kd domain of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Analysis of Lrp and Ada binding by gel retardation assay Lane 1 , PardBDNA , no protein added ; lane 2 , 5 prnol of unmethyl-ated Ada protein ; lane 3 , 5 pmol of methylated Ada protein ; lane 4 , 5 prnol of Lrp ; lane 5 , 5 pmol of Lrp and unmethylated Ada : lane 6 , 5pmol of Lrp and methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>aidB ; lane 1 , control DNA ; lane 2 , 2 pmol unmethylated Ada protein ; lane 3 , 2 pmol methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Lane I. no protein : lane 2 , the methylated Ada prot , ein ( 6 pmol ) ; lane 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Lanes are as follows : lane 1 , promoter regions , no protein added ; lane 2 , 12 pmol of unmethylated Ada protein ; lane 3 , 12 pmol of methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Lanes 1 , 4 , 7 , and 10 , no Ada protein ; lanes 2 , 5 , 8 , and 11 , 0.2 mM unmethylated Ada ; lanes 3 , 6 , 9 , and 12 , 0.2 mM methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Reactions performed without Ada protein was were ( lanes 1 , 2 , and 5 ) or with addition of 10 pmol of Ada ( unmethylated ; lanes 3 and 6 ) or of 10 pmol of methylated Ada ( lanes 4 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>Transcription of the a/kA gene might conceivably be activated by either unmethylated or methylated Ada protein , since induced cells contain high levels of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>This latter transcript occurred at the same low level in the presence of either unmethylated or methylated Ada protein , and its relevance is uncertain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>It is even possible that the Ada protein becomes methylated and then is cleaved to generate the active positive regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>Consequently , the most abundant form of the activated Ada protein would be expected to be the doubly methylated species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>This sequence is bound b methylated Ada protein , as deduced from nuclease protection experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>even in the presence of a relatively high concentration of the methylated Ada protein ( Fig. 7 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>methylated or non-methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>+ spec ~ fic RXA in t ' he presence of the methylated Ada protein ( A ) or the non-methylatrd Ada prot ) rin ( A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>a , Without Ada protein : h. in the presence of 6 pmol of the non-methylated Ada protein : c , in t , hr presence of 6 pmol of the methylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>When the Ada protein wrth alanine at posit , ion 321 was methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Examination of the f ' oot.printing patterns revealed that , the methylated Ada protein does not bind to nor interact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The repair of these lesions by Ada protein results in four products : repaired DNA , the two singly methylated forms of Ada , and the doubly methylated form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>method of The Ada protein purified according the was to Takano et al. ( 13 ) and methylated by incubation in the presence of methylated DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>The Ada protein was methylated as described by Nakabeppu and Sekiguchi ( 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>The non-met hylated and methylated Ada protein are eyually effective in promoting the alleil transcription ( Nakabeppu C!LSekiguchi , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The methylated Ada protein , however , effi ciently activated transcription of the DNA fragment , and a distinct RNA product of 101 f 5 nucleotides was obtained ( Figure 5A , lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The methylated Ada protein greatly enhanced transcription of the DNA fragment , whereas the unmethyl-ated protein was much less effective as an activator ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Japan ( Received 22 June 198X ) The Ada protein of Escherichia , coli catalyzes transfer of methyl groups from methylated DNA to its own molecule , and the methylated form of Ada protein promotes transcription of its own gene , ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Methyl-groups of alkylated DNA are transferred to Ada protein by its own methyltransferase activity and the methylated Ada protein then acts as a positive regulator to overproduce the ada and related gene products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>These data are in good agreement with the results on DNA-dependent protein synthesis and demonstrate that the fully methylated Ada protein is a transcriptional-activator of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Upon methylation , the Ada protein ( meAda ) activates its own operon , adalalkB , and at least two other genes , a / M and aidB , by binding to a specific sequence immediately upstream of its target promoters ( Furuichi et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>This is indeed the case for the Ada protein , which binds the alkA promoter between 247 and 235 , i.e. , one helical turn downstream compared to the location of the Ada binding site in ada ( between 257 and 245 ) and aidB ( between 255 and 243 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In control gel retardation experiments , no significant binding by meAda and s70 was detected with unrelated DNA fragments ; s70 was unable to promote binding of the unmethylated Ada protein to the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Ada - and meAda-stimulated binding of a could be mediated by protein-protein me interaction of a with Ada and Ada ; an alternative possibility is that binding of Ada or meAda might facilitate binding of a to a specific DNA site in the alkA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>We conclude that methylation of the Ada protein is required for its efficient specific binding to the a/kA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>A number of possible mechanisms could account for the increased alkylation resistance resulting from some aidB mutations : decreased cell permeability , decreased metabolic activation of mutagens , or perhaps increased stability of Ada protein , which could cause increased expression of the alkA gene and the ada-alkB operon ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>In the absence of activated Ada protein , primarily rpoS-mediated transcription is induced , since rpoS most severely impairs ada-indepen-dent forms of aidB induction ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>Although only three promoters ( ada , aidB , and alkA ) are the target of the Ada protein , major differences exist in the mechanisms for their activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In contrast , the ada and aidB promoters are only activated by meAda protein ( 6 , 8 ) , and high concentrations of Ada protein inhibit activation by meAda ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Since the CTD of Ada is dispensable for activation at alkA , the NTD is shown to be involved in both meAda-a and meAda-s contacts CONCLUSIONS In this report , we have reviewed the mechanisms of transcription activation by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In this report , we present evidence that region 4 of s70 is also the target of activation by the Ada protein at the alkA promoter ; however , different determinants are involved in Ada-s70 interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Since both of these plasmids appear to direct the synthesis of truncated derivatives of the Ada protein , it appears that it is the amino terminus of this protein which is responsible for the positive activation of ada and that a truncated product of Ada is a considerably more potent transcription activator than the intact Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>When Ada protein interacts with this type of substrate , it is activated and stimulates transcription from the ada and alkA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Trends Genet 3 :51 -5 Shevell DE , Walker GC ( 1991 ) A region of the Ada DNA-repair protein required for the activation of ada transcription is not necessary for activation of alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_389">
A region of the Ada DNA-repair protein required for the activation of ada transcription is not necessary for activation of alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></label>
<input id="act_similarity_389", type="checkbox">
<div class="ACT_similar">
<li>A region of the Ada DNA repair protein required for the activation of ada transcription is not necessary for activation of alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li></div>
<li>The C321A mutation of Ada protein results in constitutive activation of the ada promoter ( 60 , 61 ) , again suggesting a direct role of the meth-ylation site in the AdaCTD in transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>A possible reason for this discrepancy is the very strong negative effect of s70 over-EV575 expression on transcription of ada in vivo ( 18 ) , which results in lower intracellular concentrations of the Ada protein ; this , in turn , would result in suboptimal activation of the alkA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>For this purpose the ada gene , protein was degraded to smallerpolypeptideswhen cloned from E. coli K12 , was placed under control of the lac crude extracts or partially purified preparationswere promoter in a multicopy plasmid and Ada protein was overincubated in haigh ionic-strengthbuffer at neutralpH , produced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Expression of the aZkA gene is also controlled by the Ada protein , but the mode of regulation is somewhat different from that for the ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Discussion Transcription of the ada gene is dependent on the Ada protein , and methylation of the protein greatly increases t , he rat , e of formaCon of the transcription initiation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>In contrast to the 98-base ada RNA , formation of the 90-base RNA was not enhanced by either form of Ada protein but rather was suppressed when formation of the 98-base RNA was enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>In this work , we constructed such a mutant form of Ada protein by site-directed mutagenesis of the cloned ada , gene , and found that this is actually the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>To transactivate the ada promoter , Ada protein must primarily bind to the Ada box in the promoter ( Teo et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In this system the amount of Ada protein relative to the copy number of the ada promoter was increased up to factor of 6 by altering the replication origins for both the reporter plasmid and Ada expression plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In this system , Ada protein would be constitutivel ) produced , since the lac promot , er ( Plac ) is placed upstream from the ada gene lacking its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The possibility that unmethylated Ada protein can serve to induce aZkA is consistent with the in vivo result that aZkA gene expression is elevated when the wild-type unmethylated Ada protein is overproduced from a high copy number plasmid carrying the ada gene [ Sedgwick , 19831 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>\ \ ` e developed a system to measurethe activitv of Ada protein to induce the ada and alkA expression in viva ( see Fig. 8 ) , and obtained results t , hat conform in part with those of experiments Zn vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>It only demonstrates that , if unmethylated Ada protein can serve as an inducer of alkA , then ada protein levels must increase before induction of aZkA can occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>However , with 10 pmol of protein , unmethylated Ada and unmethylated AdaK178P effectively activate alkA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>More importantly , the cyclic AMP receptor protein and related regulatory proteins are activated by the noncovalent binding of a small effector-molecule , while the Ada protein is irreversibly activated by a specific posttranslational covalent modification event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Effect of MMR inactivation and Ada overproduction on MNNG-induced DSBs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>holo enzyme and Ada : lane 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Methyltransferase activities of Ada , Ada-178 , and AdaK178P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Ada 528 , 107-118 21 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2934380" target="_blank">2934380</a>]<br></li>
<li>Ada 260 , 244-258 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6397464" target="_blank">6397464</a>]<br></li>
<li>Ada 1185 , 129-151 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11275556" target="_blank">11275556</a>]<br></li>
<li>Ada 211 , 158 3G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4564569" target="_blank">4564569</a>]<br></li>
<li>Ada 216 , 393402 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4568262" target="_blank">4568262</a>]<br></li>
<li>Ada was purified as in Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>However , DNAase I was slightly inhibited by the Ada protein preparations , so enzyme concentrations and incubation times were adjusted to obtain approximately equal levels of digestion of nonprotected DNA , as detailed in the figure legends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Deletion of the C-terminal region of Ada renders it constitutively active as a transactivator ( Shevell and Walker 1991 ) , suggesting that the deleted region may negatively regulate the transactivation function of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>For this purpose , hybrid plasmids encoding C321A , C69H/C321A , or C69A/C321A Ada proteins were constructed and the expression levels of each mutant protein were analyzed by Western blotting ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>One mutant protein , C69H , in which cysteine 69 was replaced by histidine , retained the transactivation function of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Of the 19 Ada mutants examined , only Ada C69H , in which Cys69 is replaced by histidine was formed at levels similar to those of the wild-type Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>; I relatively small increase in glyc ~ osylaw ~ activity was observed , probably reflecting the finding that t IIP normal and methylated forms of Ada protein ar ( ' equally effective for promoting t ' ranscription of t ' hr alkA gent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Format , ion of the 23 nuclrotide transcript , was stimulated by alkA both normal and met , hylat ' ed forms of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The amount of ' t , he alkA transcript formed without Ada protein ( at 30 min ) was about ' 814 of that formed in thfh present ' of the methylat , ed Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Whether the overproduction ofAda protein , we purified the protein processed or unprocessed form of Ada protein is active as a to apparenpthysicahlomogeneityT.hpeurified 38,000-dalton Ada proteintransferredthemethyl regulatory protein has remained unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Purification of Ada Protein-Taking advantage of overproduction of Ada proteinin cells harboring pYN3059 after induction of lac promoter by IPTG , Ada protein was purified to apparent physical homogeneity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>In uninduced E. coii cells only extremely low amounts of Ada protein are present , while a second methyltransferase , the Ogt protein , is constitutively expressed ( Rebeck et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>In certain experiments , cells were treated with 2 , ug of MNNG per ml 20 min before labeling with [ 35S ] methionine to examine induction of the Ada protein by MNNG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>The mode of action of the Ada protein strongly resembles that of transcriptional-activators such as the cyclic AMP receptor protein ( de Crombrugghe et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The enhanced levels of Ada and AlkA protein result in an alkylation-resistant phenotype of the cells by the improved repair of OE-methylguanine and 3-methylpurines , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The basic nature of Ada protein could explain its unique chromatographic behavior ; adsorption of Ada protein to DEAE-Sephacel at pH8 .5 is very weak , and it adsorbs to phosphocellulose even at pH8 .5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>The concentration of Ada protein was determined spectrophotometrically , and the amounts were calculated using 6280 = 3.8 x 104 M ' 1 cm - ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>The strain was grown in the presence of different concentrations of IPTG to achieve different intracellular levels of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Formation of a ternary complex was not detected when unmethylated Ada protein was added , indicating that the interaction is specific and not due to protein aggregation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Binding to the promoter regions by the a-R265A mutant was not observed either in the absence or in the presence of Ada protein ( Figure 6 , lanes 7-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Addition of the unmethylated Ada protein resulted in no detectable increase in protein synthesis ( Figure 2 , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Little RNA synthesis was observed in the presence or absence of the unmethylated Ada protein ( Figure 5A , lanes 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>A small stimulatory effect was observed in the presence of unmethylated Ada protein ( Figure 5B , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Formation ofthis 90-base RNA as well as of the control 63-base lac transcript was not significantly affected by the presence of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>We also found that for degradation of Ada protein , the presence of NaCl is more important than that of 2-mercap-toethanol ( datanotshown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>t , he t , ranscription of which depends on the presence of the Ada protein ( data not , shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>In the presence of the mrt , hylat ~ ~ tl Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>In the presence of t , he Ada protein , cert , ain upstream regions were protected by high levels of R?r ' A polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>However , when a homogeneous preparation of Ada protein ( Fraction V ) was incubated in buffer B , in which Ada protein in less purified preparations was degraded , no degradation of Ada protein was observed ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Lane A , purified Ada protein ( FractionV , 5 units ) ; B to D , crude extract in buffer A ( B ,1 pg of protein ; C ,2 pg of protein ; D , 5 pg of protein ) ; E to G , crude extract inbuffer B ( E , 1pg of protein ; F ,2 pg of protein ; G , 5 pgof protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>2 ; B , Ada protein without incubation ; C , Ada protein incubated in buffer A + s ; I ) , Ada protein incubated in buffer A ' + s ; E , Ada protein incubated in buffer B ; F , extract prepared in buffer A ( without incubation ) ; G , extract incubated in buffer A ; H , extract incubated in buffer A + s ; I , extract prepared in buffer A ' ( without incubation ) ; J , extract incubated in huffer A ' ; K , extract incubated inbuffer A ' + s ; L , extract prepared in bufferB ( without incubation ) ; M , extract incubated in buffer R. 6 7 43 30 20 14.4 FIG. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>No firm binding was observed with the nonmethylated Ada protein , although some DNase I-hypersensitive sites were produced in the promo & by both types of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Thus , activation of the Ada protein seems to be induced by replacement of the cysteine residue by alanine as well as by formation of S-methylcysteine at the same site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>Mutant RNA polymerases consisting of C-terminal-deleted a subunits are virtually inactive in response to activation by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Defining the existence of such mutant forms of Ada will facilitate our understanding of the molecular mechanisms of activation of Ada protein by methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Thus Ada cleavage is not required for its activation as a regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>This observation raises questions about the actual role played by the Ada protein in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>That is , the cysteine methylation triggers a conformational change of the Ada protein , which should induce some rearrangements of the surface of the MTase subdomain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Ada protein would be present at elevated levels only after its synthesis is induced , which requires methylation damage to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>We assume that a specific methylation of the Ada protein leads to a conformational change of the protein that allows it to-bind to the specific DNA sequence identified in the present study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>On the basis of the finding that t , hr methylnted form of the Ada protein binds to the nda promot ) er .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>These results show that the Ada protein can activate its own synthesis in a cell-free system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Table 1 Transcription-promoting activity of Ada protein in vivo Chloramphenicol acetyltransferase ( nmol min - ' m : l - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Lanes 1 and 4 , no Ada protein ; lanes 2 and 5 , 10prnol of unrnethylatedAda protein ; lanes 3 and 6 , 10prnol of rnethylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lanes 1 and 4 , no Ada protein ; lanes 2 and 5 , 0.2 mM Ada ; lanes 3 and 6 , 0.2 mM meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>and this protection occurred even in the absence of the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>When necessary , the Ada protein was present at 0.2 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>the methylat , ed Ada protein is present in the preparation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>( a ) Binding of the methylat ' ed Ada protein to the nda prornot , er .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>This may be achieved by dilution of Ada protein by cell division or by inactivation of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>Truncation of the Ada protein in our deletion derivatives may mimic this conformational change .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>It is likely that , the metIhylatJt ~ ti Ada protein may bind to the DSA from one side .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>It is of interest to learn how the Ada protein is activated by methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>It has been demonstrated that the Ada protein autoregulates its own synthesis ( 17 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Overproduction of the Ada protein by incorporation of the gene into a multicopy plasmid leads to elevated expression of the activities associated with the res-ponse , such as the alkA + - encoded DNA glycosylase , even in the absence of alkylating agents ( Sedgwick , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6092060" target="_blank">6092060</a>]<br></li>
<li>A similar activation can be achieved by direct methylation of the Ada protein by certain methylating agents ( Takahashi et al. , unpublished result ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>[ 9 ] K. Takahashi , Y. Kawazoe , K. Sakumi , Y. Nakabeppu , M. Sekiguchi , Activation of Ada protein as a transcriptional-regulator by direct alkylation with methylating agents , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Further characterization of the structure and functions of the Ada protein may serve to elucidate the cellular mechanisms for induction of resistance to alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6092060" target="_blank">6092060</a>]<br></li>
<li>When cells are exposed to alkylating agents , a large amount ' of Ada protein is formed , which in turn causes to enhanced expression of the own and other genes belonging to this regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>( it ) When methylating agents attack the cells , the DNAs are rapidly methylated , and the 06 methylguanine and methylphosphotriester formed are demethylated by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>Ada protein thus formed c ~ mltl lw methylated when cells were exposed to alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The Ada protein was strongly induced upon exposure of cells to methylating agents such as N-methyl-N ' - nitro-N-nitrosoguanidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>In this strain , the levels of the Ada protein can be controlled by varying the IPITG concentration and are therefore not affected by exposure to methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>These differences appear to be qualitative rather than quantitative , since different DNase I protection patterns are produced even in the presence of a large excess of unmethylated Ada ( 2.5 mM , data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>C. Binding sites for Ecr '' and EoSat Pajd ~ i , n the presence and in the absence of methylated Ada , as determined by the DNasel-protection assay shown in Fig. 16 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>However , these studies were performed using relatively low concentrations of unmethylated Ada ( up to 6 pmol in a standard reaction volume of 35 pj ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The lacUV5 transcript , used as our control , was unaffected by increasing the amounts of unmethylated Ada in the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>When required , 10 pmol of unmethylated or methylated Ada were added , and the mixture was incubated for 15 min at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>When required , 10pmol of Lrp , unmethylated Ada , or methylatedAda were added and the mixture was incubated for 15min at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Thus , once sufficient IPTG has been added to induce enough Ada molecules to repair all the phosphotriesters induced by 1 pg of MNNG per ml , any further increases in IPTG concentration will result in the synthesis of more Ada molecules than can be methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>As a result , the pools of unmethylated Ada molecules per cell increase while the level of methylated Ada does not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The pools of unmethyl-ated Ada molecules per cell will then increase but the concentration of methylated Ada will not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Lrp binding ( lane 5 ) is barely affected by 0.25 pM unmethylated Ada , while at the same concentration of methylated Ada the Lrp / PaldB complex is hardly detectable and PardB is almost completely shifted as the meAda/P , , dB complex ( Fig. 5 , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lane 1 , RNA polymerase alone ; lane 2 , 1 pmol of unmethylated Ada ; lane 3 , 10 pmol of unmethylated Ada ; lane 4 , 1 pmol of methylated Ada ; lane 5 , 10 pmol of methylated Ada ; lane 6 , 1 pmol of unmethylated AdaK178P ; lane 7 , 10 pmol of unmethylated AdaK178P ; lane 8 , 1 pmol of methylated AdaK178P ; lane 9 , 10 pmol of methylated AdaK178P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Lanes 4 and 5 , are the same as lanes 2 and 3 but with the addition of 10pmol of methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lanes 1 and 5 , RNA polymerase only ; lanes 2 and 6 , 10pmol of methylated Ada ; lanes 3 and 7 , lopmol of Lrp ; lanes 4 and 8 , 10pmol of methylated Ada and 10pmol of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>This reaction is irrevers-ible , and methylated Ada ( meAda ) is the terminal end product of the demethylation reaction ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>If this reaction proves to be physiologically relevant , then it would be interesting to learn if methylated and unmethylated forms of Ada are cleaved at different rates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>As shown in Fig. 5 , in the presence of an equimolar concentration of meAda , Lrp binding to PaidB is almost completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Odd-numbered lanes , no meAda ; even-numbered lanes , 0.2 mM meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Ada protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>DNA-binding activity , rather than to a defect in proteinprotein interaction between a-R265A and meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>The a subunit of RNA polymerase is believed to be involved in direct contact with reAda ( Sakumi et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>s70CTD suggests that meAda interacts with a set of negatively charged residues both in and downstream of region 4.2 ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>With the exception of I590A , all the substitutions that significantly affect ada-dependent transcription involve negatively charged amino acids ( Glu-574 , Glu-575 , Glu-591 , Glu-605 and Asp-612 ) ; substitution to alanine of several positively charged residues ( Arg-596 , Lys-597 , Arg-608 ) results in increased levels of ada-dependent transcription ( Fig. 7 ) , supporting the hypothesis that meAda interacts with a negatively charged patch in s70 CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Thus , it is possible that meAda could contact amino acids conserved in both s factors ; future experiments of site-directed mutagenesis in ss will allow better understanding of activation of ss-depend-ent transcription by meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>These observations are consistent with the fact that the s70 amino acids important for activation by meAda at ada and aidB are also conserved in sS ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>These observations are consistent with the fact that the s70 amino acids important for activation by meAda are also conS served in s ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Gel retardation experiments ( 20 ) show that direct protein-protein interaction takes place between meAda and s70 and that the determinants for such interaction reside in the C-terminal 39 amino acids of s70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>We provide evidence that meAda activates transcription by direct interaction with the C-terminal domain of RNA polymerase s70 subunit ( amino acids 574 -- 613 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>We further investigated the role of aCTD in activation by meAda by performing in vitro transcription experiments using two forms of reconstituted RNA polymerase different with respect to their a subunits : one form carried wild type a , the other a mutant a deleted of the C-terminal 94 amino acids ( a-235 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Several negatively charged residues in the s70 C-terminal domain are important for transcription-activation by meAda ; in particular , a glutamic acid to valine substitution at position 575 has a dramatic effect on meAda-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Substitutions in s70 Affect meAda-dependent Transcription -- The above gel retardation experiments suggest that the Cterminal domain of s70 could be the target for activation by meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>meAda might bind to a region of the alkA promoter important for recognition by EsS , but not by Es70 ; alternatively , binding of meAda might alter the alkA conformation to make it less favorable for interaction with EsS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>At the alkA promoter , meAda protects residues from 254 to 229 ( 16 , 18 ) , overlapping the 235 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>With the sole exception of D612 , these residues are conserved in sS , consistent with the ability of meAda to promote transcription by EsS at both ada and aidB ( 16 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>At the ada and aidB promoters , meAda appears either to increase binding to the core promoter region or to favor the formation of the open complex , functions that are indeed typical for activators that interact with the s subunit of RNA polymerase ( 29 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>At alkA , meAda interacts with both the a and s subunits , recruiting RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Unlike the s70 amino acids involved in the interaction with meAda at the ada and aidB promoters , K593 , K597 , and R603 are not conserved in sS , an alternative s subunit of RNA polymerase mainly expressed during the stationary phase of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>These observations suggest that meAda activates transcription by affecting RNA polymerase interaction with both the 235 and 210 elements of aidB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Gene product OD 4a NaClb pH 5c Adaptation to stress ( 16 ) aidB b4187 b3336 b0812 b3517 b1493 b1732 b1283 b1482 b4376 Putative acyl-CoA dehydrogenase ( ﬂavoprotein ) , adaptive response ( transcription activated by Ada ) Bacterioferritin , an iron storage homoprotein Stress response DNA-binding protein with ferritin-like domain Glutamate-decarboxylase , isozyme A Glutamate-decarboxylase , isozyme B Catalase ; hydroperoxidase HPII ( III ) , RpoS dependent Lipoprotein , osmotically inducible Resistance protein , osmotically inducible Hyperosmotically inducible periplasmic protein , RpoS-dependent stationary-phase gene Trehalose-6-phosphate synthase Trehalose-6-phosphate phosphatase , biosynthetic Induced by phosphate starvation Trehalase , periplasmic Trehalase , cytoplasmic Universal-stress-protein-B Putative glutamate : gamma-aminobutyric acid antiporter ( APC family ) 4.11 8.84 8.92 32.38 16.44 11.14 3.34 4.93 50.66 2.08 3.28 24.19 5.80 3.96 9.00 2.36 6.19 68.76 3.47 4.72 4.32 2.33 2.22 7.37 2.34 3.51 19.08 bfr dps gadA gadB katE osmB osmC osmY otsA otsB psiF treA treF uspB gadC b1896 b1897 b0384 b1197 b3519 b3494 b1492 10.27 5.56 9.48 7.98 2.07 4.52 10.28 52.44 38.22 10.53 12.71 9.25 5.91 3.48 13.79 9.09 7.42 2.50 5.20 3.61 2.22 Protein processing ( 7 ) cbpA hchA b1000 b1967 Curved DNA-binding protein , cochaperone of DnaK ( Hsp40 family ) Molecular chaperone independent of ATP/ADP cycle , with class I glutamine amidotransferase-like domain Protease involved in processing C-terminal end of HycE 30S ribosomal subunit protein S22 Putative inhibitor of protein kinase C ; contains a transferase domain Putative translation factor Putative intracellular proteinase with class I glutamine amidotransferase-like domain 4.97 4.16 2.05 6.02 15.64 4.89 hycI rpsV ycfF ylgH yhbO b2717 b1480 b1103 b4248 b3153 4.00 5.20 2.68 6.04 4.94 4.16 6.07 2.99 4.70 23.00 2.79 4.60 2.10 3.06 12.10 ther or undeﬁned O function ( 60 ) csiD elaB erfK ﬁc msyB phnB yaiA yajO ybdK ybeL ybgA ybgS ybhE yhjP yccJ yceK ycgB ycgZ yciF yciG ydaM ydfN ydgA yeaG yeaH yebF yeeP yegP yegS yﬁL ygaF ygaM ygaU Conserved protein with clavaminate synthase-like domain Unknown CDSd b2659 b2266 b1990 b3361 b1051 b4107 b0389 b0419 b0581 b0643 b0707 b0753 b0767 b0865 b1003 b1050 b1188 b1164 b1258 b1259 b1341 b1547 b1614 b1783 b1784 b1847 b1999 b2080 b2086 b2602 b2660 b2672 b2665 6.90 8.11 2.24 12.94 17.37 4.81 3.92 2.21 8.78 2.42 6.53 8.43 2.66 3.66 10.42 14.93 10.36 5.11 2.98 18.16 6.74 2.25 2.99 19.44 11.89 4.04 2.80 17.85 8.87 3.22 5.28 9.39 7.54 7.82 17.82 4.55 17.76 8.60 5.37 4.72 6.76 14.70 9.80 16.97 38.08 5.57 6.53 17.09 14.50 35.43 5.53 20.43 84.92 9.54 4.71 5.89 26.61 14.66 2.79 4.87 17.67 6.52 3.63 6.57 14.13 11.75 3.68 11.80 2.18 10.82 8.89 3.38 2.41 3.42 12.71 4.27 13.24 16.12 2.38 6.24 16.33 10.14 11.19 4.00 2.04 19.19 3.15 2.35 2.63 21.48 14.73 2.17 3.08 7.86 13.93 3.75 5.18 13.02 8.51 Conserved-hypothetical-protein with NAD ( P ) - binding Rossmann-fold domain Possible cell ﬁlamentation protein , induced in stationary phase Acidic protein suppresses mutants lacking function of protein export Conserved protein with glyoxalase and dihydroxybiphenyl dioxygenase domain Unknown CDS Putative oxidoreductase , NAD ( P ) dependent Conserved-hypothetical-protein Conserved-hypothetical-protein Conserved protein Conserved protein Putative isomerase Conserved-hypothetical-protein Unknown CDS Unknown CDS Conserved-hypothetical-protein Unknown CDS Conserved protein Conserved-hypothetical-protein Conserved protein with PYP-like sensor domain Qin prophage ; putative tail ﬁber protein Conserved protein Conserved protein , nucleotide triphosphate hydrolase domain Conserved-hypothetical-protein Conserved-hypothetical-protein CP4-44 prophage ; putative GTP binding factor Conserved-hypothetical-protein Conserved protein with PDZ-like domain Conserved-hypothetical-protein Putative enzyme Conserved-hypothetical-protein Conserved-hypothetical-protein with LysM domain Continued on following pag TABLE 1 -- Continued Function ( reference ) Ratio and gene OD 4a NaClb pH 5c b no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>DOMAIN ORGANIZATION OF AraC/XylS POLYPEPTIDES Size and Location of the Conserved Domain in AraC/XylS Members Most members of the AraC/XylS-family of regulators are 250 to 300 residues long , although a few exceptions are found HprB from Burkholderia solanacearum , Ada from M. tuberculosis , Ybbb from B. subtilis , and Hrp from X. campestris and X. oryzae are about 500 amino acids long ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>In theAda protein , acidic and basic residues are unevenly distributed ( see Fig. 8 ) , suggesting that the protein consists of two or more functional domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Regulation of ada and alkA transcription in cells a containing separate Ada N-terminal and C-terminal domains b-Galactosidase activity of strain ada9-lacZ ( OD600 ) Fold alkA9-lacZ ( OD600 Fold 2MNNG 1MNNG induction 2MNNG 1MNNG induction Plasmid 1.0 3.8 6.4 1.0 9.2 6.0 119 147 6.0 103 6.0 31 23 6.0 11 1.1 6.0 1.2 1.2 1.2 60 52 pBR322 pDS411 1,302 82 52 68 216 66 45 59 pDS415 pBS179 pBS178/179 Ada , these methylated Ada molecules , because of Ada 's auto-regulatory circuit , continue to activate ada transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>We have also demonstrated , using Ada derivatives with 10 to 20 % of the carboxyl terminus de-leted , that this speciﬁc inhibition requires , at least in part , these elements in the C terminus of Ada that have been de-leted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Elements present in the carboxyl terminus of Ada appear to be necessary for this negative regulatory function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Consistent with this , neither AraC nor Ada have a negatively charged residue that aligns with RhaS D241 , and in both of these cases s70 R599A was not defective for activation ( 33 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Interestingly , although s70 region 4 is also the target for Ada activation at the ada and aidB promoters , two different sets of amino acids are involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Our previous study ( 7 ) showed that the activation of the alkA promoter is dependent on the interaction between Ada and a positively charged cluster in s70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>In contrast to the s70 residues necessary for meAda-dependent transcription at ada and aidB , the amino acids involved in transcription-activation by Ada at the alkA promoter are positively charged .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>A negatively charged set of amino acids in s70 is the target for Ada activation at ada and aidB , while a positively charged cluster interacts with Ada at the alkA promoter ( 6 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>At the alkA promoter , a set of positively charged amino acids in s70 was important for activation by Ada ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>However , the nature of these contacts and their possible functions remain to be determined Ada-s70 INTERACTION AT THE alkA PROMOTER Although the C-terminal region of s70 is also a target for Ada activation at the alkA promoter , a different set of amino acids ( K593 , K597 , and R603 ) is involved ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In contrast , the K593 , K597 , and R603 residues , which appear to be the target amino acids for activation by Ada at the alkA promoter , are not conserved in sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>At the ada and aidB promoters , Ada interacts with a negatively charged patch in s70 : with the single exception of residue I590 , substitutions affecting Ada-dependent transcription are in negatively charged amino acids ( E574 , E575 , E591 , E605 , and D612 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In a recent study ( 18 ) , we showed me that the target for activation by Ada is in region 4 of the s70 subunit of RNA polymerase ( amino acids 574 to 613 ) ; several negatively charged amino acids in this region ( E574 , E575 , E591 , E605 , and D612 , as well as hydrophobic residue I590 ) appear to be involved in meAda-s70 interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Methylated Ada protein and a subunit form a ternary complex with ada and aidB UP elements To determine if meAda and a can bind simultaneously to the ada and aidB promoter regions or if they exclude one another , gel retardation experiments in the presence of the two proteins were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>This implies that such amino acid changes would cause a local alteration of conformational structure of the Ada protein so that th altered form of the protein would bind more readily to the ada promoter region or RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>This distance may be important for direct contact between the Ada protein and the polymerase on the DNA and our results lend support to this notion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Thi suggests that Cys321 is involved in modulation of the protein structure to permit a more intimate interaction between Ada protein and R N A polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In particular , Lys-178 could be important for a conformation that allows another region of the Ada protein to interact with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In the presence of t , he Ada protein , cert , ain upstream regions were protected by high levels of R?r ' A polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>All of these observations together strongly suggested proteinprotein contact between RNA polymerase and Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>The mutation at position 178 might interfere with activation of ada transcription by changing a critical contact with RNA polymerase , by causing a conformational change of Ada , or by interfering with the communication of conformational information between the amino - and the carboxylterminal domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>A second possibility is that the alterations at position 178 lead to alterations in the conformation of Ada that interfere with the ability of Ada to activate ada transcription but that there is no direct contact between Lys-178 and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>70 The presence of two distinct activation targets in s determines the ability by Ada to activate transcription by RNA polymerase containing sS ( EsS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_46">
The Ada protein is the ﬁrst example of a transcription activator able to contact two distinct determinants in the s70 subunit of RNA polymerase in a promoter-speciﬁc fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></label>
<input id="dom_similarity_46", type="checkbox">
<div class="DOM_similar">
<li>To our knowledge , the Ada protein is the ﬁrst example of a transcription activator able to contact two distinct determinants in the s70 subunit of RNA polymerase in a promoterspeciﬁc fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li></div>
<li>Direct protein-protein interaction between meAda and RNA polymerase is also presumed to be necessary for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>In the model for meAda activation previously proposed ( 16 ) , meAda contacts RNA polymerase through protein-protein interaction with aCTD and recruits RNA polymerase to the promoter region ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>We find that 1 ) the RNA polymerase subunit plays an important role in both basal and Ada-a activated transcription , 2 ) both Ada and meAda stimulate specific binding of both RNA polymerase holoenzyme and the purified a subunit of RNA polymerase , and 3 ) the AdazRNA polymerasezalkA promoter ternary complex differs from the ternary complex formed by meAda in the extent of upstream DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Thus , it appears that there is cooperative interaction of RNA polymerase and Ada at this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>The adaptive response promoters also differ in their interaction with RNA polymerase : at ada and aidB , RNA polymerase binds the promoters in the absence of Ada , mostly through a subunit/UPlike element contacts , to form an unusual binary complex which is poorly active in transcription initiation ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Unlike at the ada and aidB promoters , where meAda does not stimulate binding of the aCTD , interaction between the a subunit of RNA polymerase and the Ada protein appears to make an important contribution to transcription-activation of the alkA promoter ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_54">
Ada protein-RNA polymerase s subunit interaction and a subunit-promoter DNA interactions are necessary at different steps in transcription-activation at the Escherichia coli ada and aidB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></label>
<input id="dom_similarity_54", type="checkbox">
<div class="DOM_similar">
<li>Ada protein-RNA polymerase s subunit interaction and a subunit-promoter DNA interaction are necessary at different steps in transcription initiation at the Escherichia coli ada and aidB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li></div>
<li>This observation raises questions about the actual role played by the Ada protein in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>It may be that these two proteins , cII and Ada , have some similarities in the mechanism of transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>Because this substitution also replaces the Ada binding site , ada-dependent activation is almost completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>) Thus , the arginine 265 residue in the carboxyl-terminal domain of the a subunit is required for formation of the Ada - or meAda-stimulated ternary complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>MECHANISM OF TRANSCRIPTION ACTIVATION BY meAda According to the model for meAda activation proposed by Sakumi et al. ( 52 ) , meAda contacts RNA polymerase through protein-protein interaction with the C-terminal domain ( CTD ) of its a subunit ( aCTD ) and recruits RNA polymerase to the Ada-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In this report , we show that although aCTD is responsible for the formation of the initial RNA polymerase-promoter binary complex , transcription-activation requires protein-protein interaction between meAda and the C-terminal domain of the s70 factor of RNA polymerase ( s70 CTD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Thus , we conclude that the Ada protein is a class I transcription factor which requires the C-terminal region of the RNA polymerase a subunit for transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Altogether , the results indicated that the Ada protein requires the C-terminal region of the ot subunit for transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Formation of Complexes of a Subunit and Either Ada or meAda at the alkA Promoter -- The carboxyl-terminal domain of the RNA polymerase a subunit plays a role in both basal and activated transcription at alkA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>We therefore asked whether the carboxyl-terminal domain of the RNA polymerase a subunit is essential for activation of transcription by Ada at the alkA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Recently , Ada has been classiﬁed as a class I transcription factor because of its requirement for the carboxyl-terminal domain of the a subunit of RNA polymerase for transcription-activation at ada ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Recently , Ada has been classified as a class I transcription factor due to its requirement for the carboxyl-terminal domain of the a subunit of RNA polymer-ase for transcription-activation ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>These observations indicate that the activation of ada differs from the activation of alkA in requiring elements in the carboxyl-terminal half of Ada and that the carboxyl-terminal 10-20 % o of the protein plays an important negative role in Ada 's regulatory functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Finally , the C-terminal domain of Ada , which is necessary for activation at the ada and aidB promoters , is dispensable at alkA ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>These observations indicate that elements in the carboxyl-ter-minal domain of Ada are necessary for the activation of ada transcription but not for the activation of alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Activation of ada , but not alkA , requires elements contained within the carboxyl-terminal domain of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Finally , the elements of Ada required to activate alkA transcription ( also located in the N-terminal domain of Ada ) must be essentially normal in these Ada deriv-atives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>A striking feature of the activation of ada transcription is that the Ada box-binding capability of Ada is located in the N-terminal domain but that elements required for ada transcription ( but not alkA transcription ) are located in the Cterminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The 20-kDa amino-terminal domain of Ada , in the presence of a methylating agent , is sufﬁcient to promote transcriptional activation at the alkA promoter in vitro and in vivo but will not activate transcription at ada ( 1 , 26 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The Ada protein has two activation domains , one of which is an AraC/XylS-family domain which is required to activate transcription of the alkA operon ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>When the N-terminal and the C-terminal domains of Ada were expressed separately within the same cell , the results were similar to what was observed when only the N-terminal domain of the protein was expressed : i.e. , the coexpressed 19-kDa C-terminal protein had no inﬂuence on the ability of the N-terminal domain to activate alkA transcription , nor did it permit the activation of ada transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Trends Genet 3 :51 -5 Shevell DE , Walker GC ( 1991 ) A region of the Ada DNA-repair protein required for the activation of ada transcription is not necessary for activation of alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_78">
A region of the Ada DNA-repair protein required for the activation of ada transcription is not necessary for activation of alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></label>
<input id="dom_similarity_78", type="checkbox">
<div class="DOM_similar">
<li>A region of the Ada DNA-repair-protein required for the activation of ada transcription is not necessary for activation of alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li></div>
<li>It is possible that Ada CTD is responsible for this interaction ; deletions in Ada CTD abolish transcription-activation at the ada promoter without affecting DNA-binding by meAda ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Methylation of Cys-69 , in the N-terminal domain , triggers specific DNA-binding by Ada and is required for transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>However , the Ada Cterminal domain does play a role in transcription-activation ; deletions in Ada CTD affect transcription of ada , even though they do not affect specific DNA-binding ( 12 -- 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Since both of these plasmids appear to direct the synthesis of truncated derivatives of the Ada protein , it appears that it is the amino terminus of this protein which is responsible for the positive activation of ada and that a truncated product of Ada is a considerably more potent transcription activator than the intact Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>Since the CTD of Ada is dispensable for activation at alkA , only the NTD is shown to make contact with both a and s70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Since the CTD of Ada is dispensable for activation at alkA , the NTD is shown to be involved in both meAda-a and meAda-s contacts CONCLUSIONS In this report , we have reviewed the mechanisms of transcription-activation by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The azAda or azmeAda complexes were not detected , however , when purified R265A mutant a subunit was used ( lanes 7 -- 9 ) , indicating that the DNA-binding function of the carboxyl-terminal domain is necessary for the formation of these complexes as well as for activation of this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>The region of greatest amino acid sequence homology identiﬁed in XylS/AraC members is clearly a set of 99 residues found in most of the proteins at the C-terminal end of the regulators , although in some cases it is at the N-terminal end ( CafR and Rob from E. coli ) or in the central domain ( Ada from E. coli and S. typhimurium and Ybbb from B. subtilis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>The crystal structure of the 19 k D a C-terminal fragment ( residues Thr176 to Arg354 ) of Ada revealed that this region consists of two distinct structural domains , with the Cys321 in the C-terminal domain buried inside the molecule ( Moore et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The C-terminal domain of s70 ( sCTD ) contains a putative helix-turn-helix motif ( region 4.2 , residues 570 to 613 ) which recognizes the 235 promoter element ( 15 ) and some of the transcription activators that bind near the 235 region , including l cI , PhoB , Ada , cyclic AMP ( cAMP ) receptor protein ( CAP ) -- activating region-3 ( AR3 ) , FNR , and phage Mu Mor proteins ( 1 , 21 , 23 -- 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>This region of s is located in the C-terminal 70 domain ( CTD ; amino acids 590 ± 613 ) , which also contains residues required for transcription-activation by Ada ( Landini and Busby , 1999 ) , lcI ( Kuldell and Hochschild , 1994 ; Li et al. , 1994 ) , AraC ( Hu and Gross , 1985 ) , RhaS ( Bhende and Egan , 2000 ) and the cyclic AMP receptor protein ( CRP ) ( Lonetto et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Whatever the mechanism , it appears that it will involve elements located within the carboxyl-terminal 67 amino acids of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Recently , it has been shown that the amino-terminal domain of Ada has a high afﬁnity for binding a single zinc atom ( 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In addition , this observation suggests that the ability of Ada to recognize and bind to an Ada box , a property of the N-terminal domain , is unimpaired in these eight Ada derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In order to do this , we performed methyltransferase assays in vitro , using puriﬁed wild-type Ada and AdaK178P proteins and the 20-kDa N-terminal domain of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>h , pBR322 ( parental plasmid ) ; å , pDS411 ( wild-type Ada ) ; F , pDS415 ( the N-terminal domain of Ada ) ; ■ , pK178P ( AdaK178P ) ; Ç , pBS179 ( the C-terminal domain of Ada ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The Ada binding site is centrally located in a long but imperfect region of dyad symmetry ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The target for activation by Ada is located immediately downstream of the helix-turn-helix DNA-binding motif , which is responsible for recognition of the 235 pro moter element ( 20 ) and is the target of several transcription activators ( 12 , 21 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>According to both models , residues K593 and K597 belong to a surface-exposed patch , thus providing an accessible target for activator proteins such as Ada , while R603 is removed from the protein surface and in closer contact with the helix-turn-helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The positions of the main lacUV5 and alkA transcripts are indicated Ada , while R603 is removed from the protein surface and in closer contact with the helix-turn-helix DNA-binding motif ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>This indicates that the zinc-binding capacity of histidine at residue 69 is likely to be sufficient for Ada to recognize and bind to the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In the case of Ada , zinc-binding may play important roles both in maintenance of a stable structure and the DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It has been shown that the S-methylcysteine residue in the methylated form of Ada , binds a zinc ion ; thus methylation may not cause a dramatic change in the structure of Ada protein ( Ohkubo et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It has been shown that the replacement of a zinc ion to a cadmium ion and the mutation of those cysteine residues to an aspartic acid or a histidine residue cause the loss of the Ada 's repair activity ( Myers et al. 1995 ; Sun et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>In particular , the amino acid sequence Pro-Cys-His-Arg , which includes the methyl-accepting cysteine residue ( Cys321 in the case of Ada ) is conser-ved in 11 known molecular species of the enzyme and can not be substituted with any other amino acids without loss of function ( Takano et al. 1988 ; Chueh et al. 1992 ; Ihara et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Ada ( f ) Binding of mutant Ada proteins to the promoter Takano et al. ( 1988 ) constructed mutant forms of Ada proteins , Ala69 and Ala321 , in which each of the methyl-accepting cysteine residues was altered to alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Moreover , the Ada protein is unable to activate transcription by EsS at the alkA promoter , consistent with the fact that the K593 , K597 , and R603 residues are not conserved in sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>A distinct region of the promoter , corresponding to residues -65 to -48 , was well protected against DNAase I digestion by the methylated Ada protein ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>First , Ada protein is basic , having a net charge of +10 ; it has 38 acidic residues ( Asp + Glu ) and 48 basic residues ( Arg + Lys ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Incubations of Ada protein with the Gs-methyfguanine-containing ofigonucleotide ( 10 pmol protein per 0.05 pg oligonucleotide containing 15 pmol OS-methylgua-nine ) were followed by acid hydrolysis and fluorescence spectrometry of aliquots of the reaction mixture to ascertain that stoichiometric transfer of methyl groups from 0 '' methylguanine residues to the protein had occurred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The unique amino terminus provides additional evidence for the physical homogeneity of the Ada protein preparation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>The protein was purified to physical homogeneity , the purified Ada proteinremainedstableunderthe and its amino-terminal sequence and total amino acid comsame conditions , indicating that theAda protein may position were determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>The Ada protein cont , ains no amino acid sequence analogous to the zinc-finger motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The E. coli Ada protein may have arisen as a consequence of fusion of the two genes , and might thus have acquired the characteristic whereby the two domains affect each other via physical contact to give rise to more subtle regulatory effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The proteolytic processing generates two domains with retained ability to repair 06-methylguanine and phosphotriesters , whereas the regulatory activity of the Ada protein is destroyed ( our unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>As shown in Fig. 1 , the Ada protein is structured in two independent domains , linked by a hinge region that is highly susceptible to proteolytic cleavage ( 7 , 63 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The 39-kDa Ada protein is comprised of two functional domains linked with the hinge region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Abbreviations : N-Ada16k , the N-terminal 16-kDa domain of the Ada protein ; meC38 N-Ada16k , the Cys38 methylated form of N-Ada16k ; MTase , methyltransferase ; HTH , helix-turn-helix ; NMR , nuclear magnetic resonance ; MALDI-TOF MS , matrix assisted laser desorption/ioni-zation time of flight mass spectrometry ; MNU , methylnitrosourea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>J Bacteriol 173:7834 -7840 Myers LC , Terranova MP , Nash HM , Markus MA , Verdine GL ( 1992 ) Zinc binding by the methylation signaling domain of the Escherichia coli Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Zinc binding by the methylation signaling domain of the Escherichia coli Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Chemical communication across the zinc tetrathiolate cluster in Escherichia coli Ada , a metalloactivated DNA-repair-protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>J Bacteriol 171 : 2249-2251 Sedgwick B , Robins P , Totty N , Lindahl T ( 1988 ) Functional domains and methyl acceptor sites of the Escherichia coli Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Functional domains and methyl acceptor sites of the Escherichia coli Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>We have shown that mutant Ada ( C321A ) , in which Cys321 is replaced by alanine , constitutively transactivates both ada and alkA genes in vivo ( Takano et al. 1988 ) , suggesting that the alanine substitution causes a subtle structural change in the C-terminal domain which renders the mutant protein constitutively active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Deletion of the C-terminal region of Ada renders it constitutively active as a transactivator ( Shevell and Walker 1991 ) , suggesting that the deleted region may negatively regulate the transactivation function of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>A sequence-specific DNA-binding domain of Ada protein ties in the N-terminal region ( Akimaru et al. 1990 ; Sakashita et al. 1993 ) , and it is likely that the DNA-binding capacity depends on the structure of the metal ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>There is accumulating evidence to suggest that the C-terminal region of Ada protein is involved in a transactivating function but does not affect its DNA-binding capacity ( Demple 1986 ; Shevell et al. 1988 ; Shevell and Walker 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The hinge region of Ada , which is susceptible to various proteases in vitro , joins the N-terminal domain containing the methylphosphotriester DNA methyltransferase-activity to the C-terminal domain containing the O6-methylgua-nine/O4-methylthymine DNA methyltransferase-activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Treatment of this DNA substrate with puriﬁed wild-type Ada protein removes both types of lesions , whereas treatment with only the N-terminal domain of Ada removes only the methylphosphotriesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In addition , the mutation frequency of arginine reversion in cells expressing only the C-terminal domain of Ada ( from plasmid pBS179 ) , which contains only O6-methylguanine DNA methyltransferase-activity , was similar to that of cells expressing wild-type Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Altera-tion of the carboxyl-terminal domain of Ada protein influences its inducibility , specificity , and strength as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2180908" target="_blank">2180908</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_131">
Alteration of the carboxyl-terminal domain of Ada protein inﬂuences its inducibility , speciﬁcity , and strength as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></label>
<input id="dom_similarity_131", type="checkbox">
<div class="DOM_similar">
<li>Alteration of the carboxyl-terminal domain of Ada protein inﬂuences its inducibility , speciﬁc ¬ ity , and strength as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li></div>
<li>H , C and N resonance assignments of the N-terminal 16 1 13 15 kDa domain of Escherichia coli Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Therefore , as demonstrated by the data in Fig. 1 , the amount of radioactivity transferred to the 20-kDa N-terminal domain of Ada was one-half of that in the fully intact protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>h , Ada protein ; å , Ada-178 protein ( the 20-kDa N-terminal domain of Ada ) ; F , AdaK178P protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Keywords : N-terminal domain of Ada protein ; solution structure ; methyl acceptor residue ; mass spectrometry ; DNA methyltransferase ; transcriptional factor ; Cys38 methylation Supplemental material : see www.proteinscience.org DNA in living organisms is constantly subject to modification by intracellular and extracellular DNA alkylation agents which are cytotoxic , mutagenic , and carcinogenic ( Wood et al. 2001 ; Lindahl 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>There are 12 families of enzyme domains , of which at least three are certainly catalytically active : the signal peptidase domain in LexA , the methyl-DNA protein methyltransferase as in Ada and the P-loop nucleotide triphosphate hydrolase in DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>The N-terminal domain of Escherichia coli Ada exhibits an interesting function of direct removal of a methyl group on a methyl phosphotriester backbone lesion ( 3,4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15004242" target="_blank">15004242</a>]<br></li>
<li>Solution structure of the DNA methyl phosphotriester repair domain of Escherichia coli Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>The Ada protein , unlike the CAMP receptor protein , has a monomeric structure and does not seem to contain a helix-turn-helix domain that might function to recognize DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The Ada protein is able to remove methyl groups from the 6 4 premutagenic lesions O - methylguanine and O - methylthy-mine and transfer them irreversibly to Cys-321 , a cysteine located in its carboxyl-terminal domain ( 2 , 3 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The Ada protein removes methyl groups from the premutagenic lesions 06-methylguanine and 04-methylthymine , transferring them irreversibly to Cys-321 , a cysteine located in its carboxylterminal domain ( 4-6 ) , and also removes methyl groups from the S diastereoisomers of DNA methylphosphotriesters , transferring them irreversibly to Cys-69 , which is located in its amino-terminal domain ( 7-10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The N-terminal 16-kDa domain of Escherichia coli Ada protein ( N-Ada16k ) repairs DNA methyl phosphotriester lesions by an irreversible methyl transfer to its cysteine residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>The N-terminal domain of the Ada protein ( AdaNTD ) carries the determinants for speciﬁc DNA-binding : methylation of cysteine-69 , the methyl acceptor site for methyl-phosphotriesters , allows AdaNTD to speciﬁcally bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The N-terminal region ( Metl to Lys178 ) of Ada protein , on the other hand , possesses methylphospho-triester-DNA methyltransferase-activity and cysteine residue 69 is the methyl acceptor site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>A purified C-terminal 19 kd domain of the Ada protein , unmethylated or carrying a single methyl group derived from 06-methylguanine , failed to stimulate Ada protein synthesis ( Figure 2 , lanes 4 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The Ada protein consists of t , wo domains , the II ' - domain with a methyl accept , or site , ( lys69 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The 19 kd C-terminal domain of the Ada protein was isolated as described ( Demple et al. , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>1 The abbreviations used are : aCTD , a subunit C-terminal domain ; MMS , methyl methanesulfonate ; meAda , the methylated form of the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>AlkA protein functions as a glycosylase that repairs N3-meA , N3-meG , O * - meT , and 02-meC residues in DNA , and Ada protein functions as an alkyltransferase that removes alkyl groups from 06-meG , 04-meT residues as well as methylphosphotriesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Activation of Ada Protein Requires Methylation of a Specific Cysteine Residue The most important enzymatic function of the Ada protein is to counteract alkylation mutagenesis by direct reversal of 06-methylguanine in DNA to unsubstituted guanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Since 06-methylguanine is a more frequent alkylation lesion than the methylphosphotriester Sp isomer , Ada protein molecules that would encounter the latter DNA lesion in vivo would be likely to have already reacted with an 06-methylguanine residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>These two activities are closely tied to one another , as the Ada protein must be activated to perform its regulatory function and activation is a consequence of its DNA repair activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In contrast , Ada protein singly methylated at Cys-321 is not capable of specific DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The plasmids thus formed would produce mutant Ada proteins whose cysteine residues at positions 69 and 321 are replaced by alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The result obtained in vivo suggests that conversion of cysteine at position 321 to alanine would ca , use some subtle conformational alteration of Ada protein so as to make for more ready accessto the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Direct amino acid analysis of 3H-labeled , 19,000 M , protein ( Demplr et al. , 1985 ) revealed that the methyl acceptor site is a cvsteine residue , which corresponds to the 321st amino acid residue of the int , act Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Substitution of Cys69 by each of 19 other amino acids revealed that this cysteine residue is essential for full activity of Ada as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The presence of positively charged amino acids in the proximity of the cysteine residues that function as methyl acceptor sites was proposed to be important for interaction-with-DNA and DNA repair activity of the Ada protein ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Structural data show that cysteine-321 and the ﬂanking amino acids are buried inside the protein and are not accessible to solvents in the unmethylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In the present study , we have constructed mutant forms of Ada protein in which the cysteine residue 69 was substituted with 19 other amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>We constructed mutant forms of Ada protein whose cysteine residues at the plausible methyl acceptor sites are replaced by other amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Requirement for two conserved cysteine residues in the Ada protein of Escherichia coli for transactivation of the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Mol Gen Genet ( 1996 ) 250:523 -532 © Springer-Vertag 1996 A. T a k e t o m i • Y. Nakabeppu • K. Ihara • D. J. Hart M. Furuichi • M. Sekiguchi Requirement for two conserved cysteine Escherichia coli for transactivation of th da protein of residues in the A e ada promoter Received : 17 August 1995 / Accepted : 6 November 1995 Abstract Cysteine residue 69 of the Escherichia coli Ada transcription factor , which accepts a methyl group from methylphosphotriester in methylated DNA , was substituted by each of 19 other amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Ada protein transfers methyl groups from damaged DNA to two of its own cysteine residues ; in doing so , it becomes a transcriptional activator ( 17 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>The product of the ada gene , the Ada protein , plays a dual role in the adaptive response ; Ada transfers methyl groups from DNA to two of its cysteine residues , thereby functioning as a DNA-repair-protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The Ada protein ( 39 kDa ) is able to recognize DNA damage and to transfer methyl groups from DNA to two of its own cysteine residues ( Demple eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>The product of the ada gene , Ada , is a methyltransferase able to transfer methyl groups from damaged DNA to two of its own cysteine residues ( 6 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Ada is able to transfer methyl groups from DNA to two of its own cysteine residues ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>Ada has two active methyl acceptor cysteine residues , Cys-69 and Cys-321 , that are required for demethylation of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Ada contains several cysteine residues , two of which act as methyl acceptors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Ada protein is composed of two functional regions , each containing one of the methyl-accepting cysteine residues ( Demple et al. 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Ada is a 39-kDa protein organized in two independently structured domains , each with one methyl-acceptor cysteine ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The two cysteine methyl group acceptor sites , Cys-321 and tentatively Cys-69 , were localized by amino acid sequencing of relevant peptides from the methylated form of the Ada protein ( Demple et al. , 1965 ; T McCarthy et al. , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Results ( a ) Replacement of cysteine residues 6.9 and 321 of Ada protein by alar & P Direct sequenceanalysis of the [ J-terminal half of the methylated Ada protein revealed that cysteine 321 , which is located between a proline and a histidine residue , is the acceptor site for a methyl group from 06-methylguanine of alkylated DNA ( Demple et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Thus , substitution of the cysteine residue 321 by alanine renders Ada protein constitutively active , regardless of whether or not the Cys 69 residue is intact or is substituted by histidine 97-69 - 46 ¬ 30 ¬ 14.3 ¬ Fig. 4a , b Methyl acceptor activity - of C69H Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>We propose that the transcrip-tion-regulating function of Ada protein is under dual control by methylation of cysteine residues at positions 69 and 321 ; the former enhances DNA-binding , while the latter enhances the transactivating capacity of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>One mutant protein , C69H , in which cysteine 69 was replaced by histidine , retained the transactivation function of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It seems that the cysteine residue at position 69 is important for accepting the methyl group and to maintain the conformation of Ada prot ' ein so as to exhibit the basic transcriptionpromoting activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Methylation of cysteine residue 321 , which accepts a methyl group from O6-methylguanine or O4-methylthymine is involved in modulation of the transactivating capacity of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The C69H Ada protein has the capacity to accept a methyl group only from O6-methylguanine and O4-methylthymine resi-dues in methylated DNA , and the enhanced transactivating capacity is attributable to the acquisition of a methyl group at the Cys321 residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Since Ada protein with alanine at position 321 had no such effect on ada - cells , it is likely that 06-methylguanine and 04-methyl-thymine , the repair of which is handled by the cysteine residue at position 321 , are prematageni and lethal t ' o cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>This result is compatible with the idea that the methyl group of 04-methylthymine can be transferred to the cysteine residue at position 321 , but not that at position 69 , of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Tt is evident t , hat the cysteine residue at , position 69 is important for accepting the methyl group and for maintaining the conformation of Ada protein as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>t.hen the Ada protein whose cysteine residue at position 69 is converted to another amino acid could not accept t , he methyl group from methylphosphotriester , but would retain the ability to accept the methyl group from 06-methylguanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Methylation of the second cysteine acceptor site in the Ada protein , with the methyl group derived from an 06-methylguanine residue , was inefficient as an activating event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In a parallel experiment , we constructed Ada protein whose cysteine residue at position 321 is converted to alanine , thereby supporting the notion that this cysteine residue is the acceptor sit ) e for t , he methyl group from 06-methylguanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Adaptive-response induction occurs when the Ada protein transfers a methyl group from methylphosphotriesters formed in DNA by methylating agents to its own Cys-69 residue ( for reviews , see references 12 , 15 , and 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>Once Ada protein transfers a methyl group from the methyl-phosphate to the Cys-69 resi-due , it becomes a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>I tt addition to the transcriptional regulator activity , the Ada protein possessestwo methyltransferase activities , one to transfer a methyl group from one of two stereoisomers of methyl-phosphotriesters of methylated DNA to a cpsteine residue at position 69 of the Ada protein and t ) he ot , her to transfer a methyl group from 06-met .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>This activation of Ada protein is achieved by transfer of the methyl group from one of the stereoisomers of methylphosphotriester of DNA to cysteine residue 69 of Ada protein , by intrinsic methyltransferase-activity ( Sedgwick et al. , 1987 ; Takano et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>It , was later found that the intact Ada protein possesses anot , her methyltransferase-activity , which ( Batalyzes transfer of a methyl group from one of the two stereoisomers of methylphosphotriesters in alkplated DNA to a cysteine residue in the N-terminal half of the protein ( Margison et al. , 1985 : Mc ( : arthy & LindahI , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Self-methylation of the Ada protein by transfer of a methyl group from a phosphotriester in alkylated DNA to a cysteine residue in the protein converts it to an activator of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Acquisition of a methyl group at the Cys69 residue must cause a subtle conformational change in the Ada protein so as to provide easier access to the DNA interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>We recently produced an altered form of Ada protein , in which this cysteine residue is replaced by alanine , by site-directed mutagenesis of the cloned ada gene ( unpublished work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>Thus , activation of the Ada protein seems to be induced by replacement of the cysteine residue by alanine as well as by formation of S-methylcysteine at the same site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>These results are compatible with the idea that methylation of the cysteine residue at position 69 renders Ada protein active as a transcriptional regulator , whilst the cysteine residue at position 321 is responsible for repair of pre-mutagenic and lethal lesions in DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Using a reconstituted syst 'em with purified components , we demonstrated that met , hylation of the cysteine residue at position 69 is responsible for activation of the Ada protein , as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>They found that interaction of Ada protein with synthetically constructed 06-meG containing DNA , which results in methylation of the cysteine residue at position 329 of Ada , does not activate it to stimulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>`` The Ada protein with the cysteine residue at posit * ion 69 replaced by alanine was incapable of binding to the ada promoter even when the cysteine residue at position 321 of the protein was methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Replacement ' of the cysteine residue at position 69 , but not at position 321 , abolished the ability of Ada protein to promote transcription of both ada and alkA genes in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The Ada protein with the alanine residue at , position 69 was inactive for promoting in-vitro transcription of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Using this mutant we have investigated how methylation of the Cys321 residue of Ada protein affects its transcription-regulating function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>The role of methvlation of each cysteinr residue was investigated using mutant forms of the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The reaction involving methylation of a cysteine residue that serves to convert the Ada protein to a transcriptional activator is so far unique , since S-methylcysteine has not been found in any other enzymatically modified protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The latter finding suggests that the cysteine residue at position 321 , which is located near the C terminus of the Ada protein , IS involved in regulating activity , as the transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The methylated form of the Ada protein employed as activator contained two methyl groups derived from an 06-methylguanine and a phosphotriester residue ( McCarthy and Lindahl , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The Ada protein would consequently be methylated only at residue Cys-321 ( see Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Moore et al. ( 13 ) have proposed that , when Cys-321 accepts a methyl group 6 from O - methylguanine , the C-terminal helix of Ada ( residues 340 to 350 ) would be repositioned because of a movement about residues 325 to 335 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>This is an extremely interesting portion of the molecule since the recent determination of the crystal structure of the carboxyl-terminal half of the Ada protein has led to the proposal that the mechanism of methyl removal from 06-methylguanine , which involves a transfer of the methyl group to cysteine 321 ( 38 ) , involves a movement about residues 325-335 of Ada that would reposition the carboxyl-terminal helix ( residues 340-350 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>An additional substitution was made with alanine in Ada C69H , replacing Cys321 , the site for acceptance of a methyl group from O6-methylguanine and O4-methylthymine residues in DNA , with alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Figure 3a shows transfer of the methyl group from methylated DNA to wild-type and C69H Ada proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>First , this observation provides support that methylphosphotriester DNA methyltransferase-activity functions normally since transfer of a methyl group to Cys-69 is necessary to activate Ada for its role in alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Furthermore , Ada contains an additional methyltransferase-activity that removes methyl groups from the S-diastereoisomers of DNA methylphosphotriesters and transfers them irreversibly to Cys-69 , which is located in its amino-terminal domain ( 11 , 24 , 25 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Ada removes methyl groups from phosphotriester and 0Pmethyluanine lesions in DNA , irreversibly transferring them to cystehie residues at positions 69 and 321 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>In this paper , we describe the isolation and characterization of mutants altered at the Lys-178 residue within the proposed hinge region of Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>In order to assess whether alteration of the in vitro prote-olysis-sensitive hinge region of Ada would affect communication between the N-terminal and C-terminal domains , resulting in altered transcriptional activation activity , site-directed mutagenesis was employed , with a random oligonucleotide used , to alter the Lys-178 residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>We isolated and characterized eight different Ada derivatives that had the following amino acids substituted for Lys-178 : leucine , valine , glycine , tyrosine , arginine , cysteine , proline , and serine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Padaand Placindicate the ada and lac promoter , respectively ; St , Stu R ~ u ~ Construction and expression of mutant Ada proteins carrying amino acid substitutions Nineteen different types of m u t a n t A d a proteins , in which the Cys69 residue was substituted with one of 19 o t h e r a m i n o acids , were c o n s t r u c t e d b y m e a n s of P C R - m e d i a t e d site-directed mutagenesis ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>( i ) A part of the Ada contact site remains in the truncated at subunit upstream of amino acid residue 235 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">Ada evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>DISCUSSION The AidB protein belongs to the adaptive response to alkyl-ating agents ( 25 ) , which is deﬁned by the Ada regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>DOMAIN ORGANIZATION OF AraC/XylS POLYPEPTIDES Size and Location of the Conserved Domain in AraC/XylS Members Most members of the AraC/XylS family of regulators are 250 to 300 residues long , although a few exceptions are found HprB from Burkholderia solanacearum , Ada from M. tuberculosis , Ybbb from B. subtilis , and Hrp from X. campestris and X. oryzae are about 500 amino acids long ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>There are 12 families of enzyme domains , of which at least three are certainly catalytically active : the signal peptidase domain in LexA , the methyl-DNA protein methyltransferase as in Ada and the P-loop nucleotide triphosphate hydrolase in DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>This DNA-binding domain has also been shown to be involved in transcription activation in a number of AraC/XylS family proteins including Ada , RhaS , AraC , MelR , MarA , SoxS , XylS , and UreR ( 1 , 4 , 5 , 15 , 16 , 20 -- 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>The Ada protein has two activation domains , one of which is an AraC/XylS family domain which is required to activate transcription of the alkA operon ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>The N-terminal half of Ada , which includes the AraC/XylS family domain , is capable of binding to DNA and activating transcription at the alkA promoter but is not sufﬁcient for transcription activation at promoters where Ada binds upstream of the 235 region ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Ada</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Repressor of its own synthesis Similar to E. coli AraC Similar to E. coli AraC Unknown Unknown Positive regulator of caf1MA and caf1 operons for the production and transport of the capsule antigen F1 Repressor of the celABCF operon involved in the degradation of cellobiose , arbutin , and salicin Transcriptional activator of the cfaABCE operon for the production of CFA/I ﬁmbriae in enterotoxigenic E. coli strains Transcriptional activator of the operon involved in the production of CS5 ﬁmbriae in enterotoxigenic E. coli strains Transcriptional temperature-dependent activator of several E. coli envelope proteins , most notably the porins OmpF and OmpC and the l receptor , LamB Transcriptional activator of the exsCBA operon and exsD , exoS , and ORF1 genes required for the synthesis and secretion of exoenzyme S Transcriptional activator of the 987P operon for ﬁmbrial proteins in enterotoxigenic E. coli strains Transcriptional activator of hpaBC operon for catabolism of p-hydroxyphenylacetic acid SP : P43463 135 Ada Escherichia coli 57 , 138 , 150 , 180 SP : P06134 354 Ada Ada Mycobacterium tuberculosis Salmonella typhimurium Bacillus subtilis SP : Q10630 SP : P26189 SP : P19219 AdaA AdiY 253 Escherichia coli SP : P33234 32 , 237 TE : Q07681 AfrR 272 258 Escherichia coli AggR 265 183 Escherichia coli SP : P43464 AppY 43 Escherichia coli SP : P05052 12 , 131 2 AraC 281 30 Citrobacter freundii SP : P11765 AraC 292 Escherichia coli SP : P03021 174 , 238 , 248 , 266 301 SP : P07642 SP : P03022 SP : Q03320 SP : P35319 SP : P26950 AraC AraC AraL AraL CafR Erwinia chrysanthemi Salmonella typhimurium Streptomyces antibioticus Streptomyces lividans Yersinia pestis 149 46 265 43 85 , 128 199 280 CelD Escherichia coli SP : P17410 265 222 CfaD Escherichia coli SP : P25393 55 301 CsvR Escherichia coli SP : P43460 159 53 EnvY Escherichia coli 2 SP : P10805 77 298 ExsA Pseudomonas aeruginosa SP : P26993 133 260 FapR Escherichia coli SP : P23774 TE : Q46985 204 295 HpaA Escherichia coli TABLE 1 -- Continued Function Protein HrpB Accession no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Gene product OD 4a NaClb pH 5c Adaptation to stress ( 16 ) aidB b4187 b3336 b0812 b3517 b1493 b1732 b1283 b1482 b4376 Putative acyl-CoA dehydrogenase ( ﬂavoprotein ) , adaptive response ( transcription activated by Ada ) Bacterioferritin , an iron storage homoprotein Stress response DNA-binding protein with ferritin-like domain Glutamate decarboxylase , isozyme A Glutamate decarboxylase , isozyme B Catalase ; hydroperoxidase HPII ( III ) , RpoS dependent Lipoprotein , osmotically inducible Resistance protein , osmotically inducible Hyperosmotically inducible periplasmic protein , RpoS-dependent stationary-phase gene Trehalose-6-phosphate synthase Trehalose-6-phosphate phosphatase , biosynthetic Induced by phosphate starvation Trehalase , periplasmic Trehalase , cytoplasmic Universal stress protein B Putative glutamate : gamma-aminobutyric acid antiporter ( APC family ) 4.11 8.84 8.92 32.38 16.44 11.14 3.34 4.93 50.66 2.08 3.28 24.19 5.80 3.96 9.00 2.36 6.19 68.76 3.47 4.72 4.32 2.33 2.22 7.37 2.34 3.51 19.08 bfr dps gadA gadB katE osmB osmC osmY otsA otsB psiF treA treF uspB gadC b1896 b1897 b0384 b1197 b3519 b3494 b1492 10.27 5.56 9.48 7.98 2.07 4.52 10.28 52.44 38.22 10.53 12.71 9.25 5.91 3.48 13.79 9.09 7.42 2.50 5.20 3.61 2.22 Protein processing ( 7 ) cbpA hchA b1000 b1967 Curved DNA-binding protein , cochaperone of DnaK ( Hsp40 family ) Molecular chaperone independent of ATP/ADP cycle , with class I glutamine amidotransferase-like domain Protease involved in processing C-terminal end of HycE 30S ribosomal subunit protein S22 Putative inhibitor of protein kinase C ; contains a transferase domain Putative translation factor Putative intracellular proteinase with class I glutamine amidotransferase-like domain 4.97 4.16 2.05 6.02 15.64 4.89 hycI rpsV ycfF ylgH yhbO b2717 b1480 b1103 b4248 b3153 4.00 5.20 2.68 6.04 4.94 4.16 6.07 2.99 4.70 23.00 2.79 4.60 2.10 3.06 12.10 ther or undeﬁned O function ( 60 ) csiD elaB erfK ﬁc msyB phnB yaiA yajO ybdK ybeL ybgA ybgS ybhE yhjP yccJ yceK ycgB ycgZ yciF yciG ydaM ydfN ydgA yeaG yeaH yebF yeeP yegP yegS yﬁL ygaF ygaM ygaU Conserved protein with clavaminate synthase-like domain Unknown CDSd b2659 b2266 b1990 b3361 b1051 b4107 b0389 b0419 b0581 b0643 b0707 b0753 b0767 b0865 b1003 b1050 b1188 b1164 b1258 b1259 b1341 b1547 b1614 b1783 b1784 b1847 b1999 b2080 b2086 b2602 b2660 b2672 b2665 6.90 8.11 2.24 12.94 17.37 4.81 3.92 2.21 8.78 2.42 6.53 8.43 2.66 3.66 10.42 14.93 10.36 5.11 2.98 18.16 6.74 2.25 2.99 19.44 11.89 4.04 2.80 17.85 8.87 3.22 5.28 9.39 7.54 7.82 17.82 4.55 17.76 8.60 5.37 4.72 6.76 14.70 9.80 16.97 38.08 5.57 6.53 17.09 14.50 35.43 5.53 20.43 84.92 9.54 4.71 5.89 26.61 14.66 2.79 4.87 17.67 6.52 3.63 6.57 14.13 11.75 3.68 11.80 2.18 10.82 8.89 3.38 2.41 3.42 12.71 4.27 13.24 16.12 2.38 6.24 16.33 10.14 11.19 4.00 2.04 19.19 3.15 2.35 2.63 21.48 14.73 2.17 3.08 7.86 13.93 3.75 5.18 13.02 8.51 Conserved hypothetical protein with NAD ( P ) - binding Rossmann-fold domain Possible cell ﬁlamentation protein , induced in stationary phase Acidic protein suppresses mutants lacking function of protein export Conserved protein with glyoxalase and dihydroxybiphenyl dioxygenase domain Unknown CDS Putative oxidoreductase , NAD ( P ) dependent Conserved hypothetical protein Conserved hypothetical protein Conserved protein Conserved protein Putative isomerase Conserved hypothetical protein Unknown CDS Unknown CDS Conserved hypothetical protein Unknown CDS Conserved protein Conserved hypothetical protein Conserved protein with PYP-like sensor domain Qin prophage ; putative tail ﬁber protein Conserved protein Conserved protein , nucleotide triphosphate hydrolase domain Conserved hypothetical protein Conserved hypothetical protein CP4-44 prophage ; putative GTP binding factor Conserved hypothetical protein Conserved protein with PDZ-like domain Conserved hypothetical protein Putative enzyme Conserved hypothetical protein Conserved hypothetical protein with LysM domain Continued on following pag TABLE 1 -- Continued Function ( reference ) Ratio and gene OD 4a NaClb pH 5c b no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Repair proteinsa E ( g ) Group Gene E. coli V. cholerae H. inﬂuenzae B. subtilis Direct repair 0.40 0.51 -- 0.53 0.38 -- 0.62 -- -- -- -- Deoxyribodipyrimidine photolyase O6 -- Methylguanine DNA alkyltransferase 0.79 0.59 0.76 0.68 -- Methylated DNA-protein-cysteine methyltransferase Regulatory protein , methyltransferase , alkyltransferase Ada Base excision repair 0.47 0.53 0.57 0.53 0.61 0.56 0.64 0.53 0.46 0.40 0.44 0.43 0.46 0.85 0.66 0.57 0.40 0.68 0.38 0.47 0.46 0.57 0.51 0.54 -- 0.53 0.47 0.35 0.37 0.53 0.44 0.48 0.78 0.35 -- b 0.38 0.50 0.41 0.69 0.55 0.62 0.58 0.68 -- 0.70 -- 0.48 0.47 0.62 0.63 0.56 0.88 0.44 0.56 0.53 0.51 0.45 0.53 0.49 0.50 0.54 -- 0.69 -- 0.57 0.35 0.42 -- -- 0.39 1.09 0.40 0.50 0.61 0.51 0.37 Uracil DNA glycosylase A/G-speci ﬁc adenine glycosylase Endonuclease III Formamidopyrimidine DNA glycosylase 3-Methyladenine DNA glycosylase I DNA 3-methyladenine glycosidase II Exonuclease III , apurinic-apyrimidinic endonuclease Endonuclease IV DNA mismatch-repair protein MutS DNA mismatch-repair protein MutL GATC endonuclease GATC methylase Exodeoxyribonuclease large subunit Exodeoxyribonuclease small subunit Excision nuclease UvrA Excision nuclease UvrB Excision nuclease UvrC DNA helicase II Transcription repair coupling factor AP endonucleases Mismatch excision repair Nucleotide excision repair Recombinational repair 1.48 0.28 0.32 0.39 0.51 0.39 0.59 0.71 0.56 0.45 0.40 0.35 0.45 0.50 0.80 -- -- -- 0.43 0.37 -- 0.72 0.39 0.48 -- 0.39 0.52 0.51 0.69 0.39 0.37 0.42 0.64 0.48 0.63 0.75 0.53 0.53 0.53 -- -- 0.63 0.88 0.41 0.46 0.41 0.47 0.44 0.43 0.55 0.49 0.53 0.43 0.37 0.47 0.61 RecA protein ExoV helicase RecB ExoV nuclease RecC ExoV helicase RecD Single-stranded DNA binding protein RecF 59 3 39 single-stranded DNA exonuclease RecJ RecO protein RecR protein ATP binding protein RecN DNA helicase RecQ Exodeoxyribonuclease I Double-stranded DNA exonuclease SbcC Double-stranded DNA exonuclease SbcD DNA-repair protein RadC RuvA protein RuvB protein 0.55 0.64 -- -- 0.65 0.65 0.52 0.58 0.55 0.47 0.40 0.50 0.59 0.56 0.48 0.61 Branch migration resolution RecG protein Holliday junction endonuclease RuvC NAD-dependent DNA ligase GTPase triphosphatase MutT 0.51 0.54 0.89 0.44 0.74 0.71 0.39 0.63 -- 0.56 0.72 0.97 DNA ligases Nucleotide pools Deoxyuridine 59-triphosphate nucleotide hydrolase 0.48 0.32 0.42 0.40 DNA polymerase I 0.43 0.36 -- -- DNA polymerase II 0.64 0.34 0.41 0.30 DNA polymerase III alpha chain Replication 0.78 0.56 0.78 0.53 SOS regulon transcription repressor LexA 0.50 0.53 0.56 0.56 ATP-dependent protease Sms/RadA Other repair proteins 0.40 0.37 0.48 0.36 Primosomal protein replication factor PriA 1.48 0.66 0.94 0.80 Single-stranded DNA binding protein Ssb 0.99 0.52 0.49 -- ATP-dependent helicase HepA a The classiﬁcation of repair proteins was adopted in part from data in reference 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489855" target="_blank">11489855</a>]<br></li>
<li>The function of meAda as a negative regulator of EsSdependent transcription suggests a speciﬁc mechanism for turning off high-level expression of the alkA gene : alkA expression is activated as long as transcription in the cell is mostly dependent on Es70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>These activities represented components of direct repair ( DR ) , base excision repair ( BER ) , mismatch-repair ( MMR ) and recombinational-repair ( RR ) ( Ada , AlkA , AlkB , RecA , RecC , RuvA , RecB , RuvB , Dam , RecO , and RecN ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>Our results provide no support for the cleavage of the Lys-178 -- Gln-179 bond being involved in Ada inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>As a second test of the contribution of MMR , overproduction of Ada protein ( which removes O6-meG as a substrate for MutS ) in GM56 Δada Δogt cells challenged with MNNG , produced no detectable linear DNA compared to control cells ( Fig. 4 and Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>Effect of MMR inactivation and Ada overproduction on MNNG-induced DSBs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>Some regulators are involved in the response-to-stressors , e.g. , response to alkylating agents ( Ada from E. coli , S. typhi-murium , and Mycobacterium tuberculosis and AdaA from Ba-cillus subtilis ) ( 57 , 100 , 176 , 177 ) ; response-to-oxidative-stress ( SoxS from E. coli and S. typhimurium ) ( 4 , 260 ) ; tolerance to antibiotics , organic solvents , and heavy metals ( AarP from Providencia stuartii , MarA and Rob from E. coli , PqrA from Proteus vulgaris , and RamA from Klebsiella pneumoniae ) ( 90 , 121 , 161 , 229 , 240 ) ; and transition from exponential growth t the stationary phase ( AppY from E. coli ) ( 12 , 131 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Some of the proteins that regulate stress response genes are SoxS ( oxidative stress , E. coli ) , MarA ( multiple antibiotic resistance , E. coli ) , Ada ( alkylating agents , E. coli ) , and Rob ( originally identiﬁed as binding to oriC but cross-activates SoxS - and MarA-regulated genes , E. coli ) ( 6 , 18 -- 21 , 30 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>aCTD participates in the function of many transcriptional activators , including CAP ( 34 ) , OmpR ( 38 ) , SoxS ( 17 ) , Ada ( 23 ) , and BvgA of Bordetella pertussis ( 4 ) , as well as in antitermination by l N protein ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>is presumed to result in a conformational change and converts Ada into a strong transcriptional activator of its own gene ( ada ) and also of several other genes ( alkB , alkA , and aidB ) ( 17 , 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Methylation at Cys-69 converts Ada into a strong transcriptional activator of its own gene ( ada ) and also of several other genes ( alkB , alkA , and aidW ) ( 11-15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>A number of possible mechanisms could account for the increased alkylation resistance resulting from some aidB mutations : decreased cell permeability , decreased metabolic activation of mutagens , or perhaps increased stability of Ada protein , which could cause increased expression of the alkA gene and the ada-alkB operon ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Ada protein plays a central role in control of expression of the ada regulon , which is composed of at least three operons , the ada-alkB , the alkA and the aidB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>As expected , a number of previously identified DNA-alkylation repair and recombination proteins were identified in our set of 99 MMS-toxicity modulating proteins , including Ada , AlkA , AlkB , and RecA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>Discussion In E. coli MMS induces two complex systems , the Ada response , protecting cells against alkylanes [ 4 -- 7 ] , and the SOS-response , expressing dozens of proteins including the umuDC-encoded Pol V DNA polymerase , which enables DNA-replication past the non-coding lesions [ 25 -- 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>E. coli cells depends on expression of two global systems , SOS and Ada response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>This may indicate that the HK82 strain is defective in yet another gene involved in repair of MMS-induced lesions or that the balance between Ada and SOS expression has been disturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>MMS , apart from the Ada response , induces the SOS system that increases expression of over 40 genes involved in DNA-recombination , repair , replication and mutagenesis [ 25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Expression of genes involved in the Ada response system by repair of alkylate The results of the second approach revealed that overexIt is amazing that among the four known in nature proteins , Summing up , the potential of MMS-induced mutations in lesions in DNA counteracts potential mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Bhattacharyya D , Hazra TK , Behnke WD , Chong PL , Kurosky A , Lee JC , Mitra S ( 1998 ) Reversible folding of Ada protein ( O6-methylguanine-DNA methyltransferase ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10398829" target="_blank">10398829</a>]<br></li>
<li>Proc Natl Acad Sci USA 83:6297 -6301 Nakabeppu Y , Kondo H , Kawabata S , Iwanaga S , Sekiguchi M ( 1985a ) Purification and structure of the intact Ada regulator protein of Escherichia coli K12 , O6-methylguanine-DNAmethyl-transferase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>( 1988 ) 202 , 483494 Expression of the ada Gene of Escherichia coli in Response to Alkylating Agents Identification of Transcriptional Regulatory Elements Takanori Nakamura , Yasuhito Tokumoto , Kunihiko Sakumi George Koike , Yusaku Nakabeppu and Mutsuo Sekiguchif Department of Biochemistry , Faculty of Medicine Kyushu University 60 , Fukuolca 812 , Japan ( Received 12 October 1987 ) Ada protein plays a central role in the regulatory synthesis of DNA-repair enzymes , following exposure of Escherichia coli to alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Regulatory mechanisms for induction of syn enzymes in response to alkylating agents : A acts as a transcriptional regulator ABSTRACT Expression of the ada and alkA genes , both of which are involved in the adaptive response of Escherichia coil to alkylating agents , is positively controlled by Ada protein , the product of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_26">
J Biol Chem 270 : 6664-6670 Nakabeppu Y , Sekiguchi M ( 1986 ) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></label>
<input id="rp_similarity_26", type="checkbox">
<div class="RP_similar">
<li>Nakabeppu Y , Sekiguchi M ( 1986 ) : Regulatory mechanismsfor induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Nakabeppu , Y. and Sekiguchi , M. ( 1986 ) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Nakabeppu , Y. , and Sekiguchi , M. ( 1986 ) Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada protein acts as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Regulatory mechanisms for induction of synthesis of repair enzymes in response to alkylating agents : Ada acts as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li></div>
<li>Expression of the Escherichia coli Ada Regulon in Stationary Evidence for rpoS-Dependent Negative Regulation of alkA Transcription Phase : PAOLO LANDINI1 ,2 * AND STEPHEN J. W. BUSBY2 Swiss Federal Institute for Environmental Science and Technology ( EAWAG ) , 8600 Dübendorf , Switzerland ,1 and School of Biosciences , The University of Birmingham , Edgbaston , Birmingham B15 2TT , United Kingdom2 Received 28 May 1999/Accepted 30 July 1999 The Escherichia coli Ada protein activates s70-dependent transcription at three different promoters ( ada , aidB , and alkA ) in response to alkylation damage of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>This observation raised the possibility that unmethylated Ada might play a key role in inhibiting transcription of the ada gene in vivo and that this inhibition may be a mechanism ( s ) for down-regulation of the adaptive response after completion of DNA damage repair .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_34">
( 16 ) Sedgwick , B. , and Lindahl , T. ( 2002 ) Recent progress on the Ada response for inducible repair of DNA-alkylation damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25537480" target="_blank">25537480</a>]<br></label>
<input id="rp_similarity_34", type="checkbox">
<div class="RP_similar">
<li>Sedgwick , B. and Lindahl , T. ( 2002 ) Recent progress on the Ada response for inducible repair of DNA-alkylation damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15004242" target="_blank">15004242</a>]<br></li></div>
<li>We will focus on Ada-dependent induction of the adaptive response genes , ﬁne tuning of individual gene-expression according to the growth phase , and the role played by Ada in shutting off the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>- Ida prot4n : ( A ) Ala69 Ada protein ( c ) Repair of 04-methylthym , ine It has been shown that 04-methylthymine , which is lessabundant in methylated DNA , is repaired by the ada gene product ( McCarthy et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Recent studies have revealed that the Ada protein , coded by the ada gene , functions as a DNA-repair enzyme as well as a transcriptional regulator ( for a review , see Sekiguchi & Nakabeppu , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Recent studies have shown that the Ada protein , the product of the ada gene , functions as a positive regulator both for its own synthesis and for the expression of other genes belonging to the adn regulon ( for a review , see Lindahl et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Ada protein is shown as a five-sided polygon ( see text for details ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Proteolytic cleavage of Ada protein that carries methyltransferase and transcriptional regulatory activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The Ada protein or some derivative of it also acts to increase the transcription of an alkA : : Mu d ( Ap lac ) fusion ( 27 ; LeMotte and Walker , unpublished data ) and of other loci in the E. coli chromosome ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>This was also the case in partiallypurified Ada protein ( data notshown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Chemical communication across the zinc tetrathiolate cluster in Escherichia coli Ada , a metalloactivated DNA-repair protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Japan ( Received 22 June 198X ) The Ada protein of Escherichia , coli catalyzes transfer of methyl groups from methylated DNA to its own molecule , and the methylated form of Ada protein promotes transcription of its own gene , ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The Ada protein acts as both a positive and a negative modulator of Escherichia coli 's response to methylating agents ABSTRACT The adaptive response of Escherichia coli protects the cells against the toxic and mutagenic effects of ceilain alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_47">
The Ada protein acts as both a positive and negative modulator of Escherichia coli 's response to methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></label>
<input id="rp_similarity_47", type="checkbox">
<div class="RP_similar">
<li>The Ada protein acts as both a positive and a negative modulator of Escherichia coli 's response to methylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li></div>
<li>In E. coli , the Ada protein mediates the response to alkylation damage , which has been called the adaptive response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>[ 6 ] D.E. Shevell , M.B.M. Friedman , G.C. Walker , Resistance to alkylation damage in Escherichia coli : role of the Ada protein in induction of the adaptive response , Mutation Res .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>The E. coli Ada protein , which regulates the expression of genes in response to DNA-methylation , is activated by the transfer of a methyl group from the DNA to the Ada protein ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>In Escherichia coli , exposure to sublethal doses of DNA-methylating agents like methyl methanesulfonate ( MMS ) , N-methyl-N ' - nitro-N-nitrosoguanidine ( MNNG ) , and N-methyl-N-nitrosourea ( MNU ) activates the expression of several genes via positive regulation mediated by Ada protein in a process termed adaptive response ( 6 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>The methyl-ated Ada protein is a positive regulator of its own synthesis ( ada ) and that of other genes , such as alkA , alkB and aidB Similar to E. coli Ada Similar to E. coli Ada One of the two proteins required for the ad-aptative response to alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>One of these is by the induction of the adaptive response by the Ada protein that acts as a methyl transferase ; in its methylated form , Ada acts as a transcriptional regulator of the Ada operon genes ada , aidB , alkA , and alkB ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>After it interacts with methylated DNA , Ada protein functions as a positive regulatory element that controls the expression of the adaptive response by stimulating the expression of the ah-alkB operon , and the alkA and aidB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>DISCUSSION Expression of the ada and alkA genes , which are involved in adaptive response in E. coli , is positively regulated by Ada protein , which is encoded by ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>The mode of activity control by NemR repressor is thus similar to that of Ada protein , which repairs DNA methyl phosphotriester lesions by transferring the methyl group to its Cys residue , thereby acting as a positive regulator for its own synthesis as well as other DNA-alkylation response genes ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567656" target="_blank">18567656</a>]<br></li>
<li>In the adaptive response , Ada protein plays important roles both in regulation-of-gene-expression and in repair of alkylated DNA ( reviewed in Sekiguchi and Nakabeppu 1987 ; Lindahl et al. 1988 ; Shevell et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>A crucial role in this process is played by ada-encoded Ada protein , which is a sensor of alkylation , a regulator of expression of Ada response , and dealkylator of O6alkylG in DNA [ 5 -- 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>The Ada protein plays a key role in the adaptive response to alkylating agents , apparently in a similar way to the RecA protein during the SOS-response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6092060" target="_blank">6092060</a>]<br></li>
<li>The Ada protein is both a DNA-repair protein and a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>The major effector molecule regulating this response is the 39-kDa Ada protein , which functions as both a DNA-repair protein and a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>The Ada protein plays a dual role in the adaptive response , being both a DNA-repair protein and a transcription activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>The product of the ada gene , the Ada protein , plays a dual role in the adaptive response ; Ada transfers methyl groups from DNA to two of its cysteine residues , thereby functioning as a DNA-repair protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>In the SOS-response , the RecA protein indirectly enhances transcription of certain genes by promoting proteolytic cleavage of a repressor , while the Ada protein shows a direct stimulatory effect on transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>To elucidate the role of specific methylation of Ada protein in regulation of expression of the inducible genes , we performed in-vitro transcription experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>AlkA protein functions as a glycosylase that repairs N3-meA , N3-meG , O * - meT , and 02-meC residues in DNA , and Ada protein functions as an alkyltransferase that removes alkyl groups from 06-meG , 04-meT residues as well as methylphosphotriesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>Methylation of cysteine residue 321 , which accepts a methyl group from O6-methylguanine or O4-methylthymine is involved in modulation of the transactivating capacity of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Stability of Ada Protein-Since buffers A and B differ in composition as well as in pH , we investigated what factor ( s ) are involved in the degradation of the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>The system fhr measuring transcription promoting act , ivit ' y of mutant Ada Fxotein in GCV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>la , b Construction and expression of mutant Ada proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Fig. 2a , b Western blotting analysis of mutant Ada proteins expressed in E. coli strain EAT50 harboring pYM3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>This DNA-binding domain has also been shown to be involved in transcription activation in a number of AraC/XylS family proteins including Ada , RhaS , AraC , MelR , MarA , SoxS , XylS , and UreR ( 1 , 4 , 5 , 15 , 16 , 20 -- 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Since a number of regulatory proteins , including RpoS , Fur , PhoPQ , Ada , PhoBR , EvgAS , GadE , and YdeO , have been reported to be involved in expression of the acid survival system ( 2 , 19 , 33 ) , transcription regulations of the ASP genes must be different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Ada binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The binding experiments shown in Figure 6 indicate that wild-type a and meAda can bind simultaneously to their binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>The hatched boxes indicate the binding sites for the polymerases alone , and the solid boxes indicate the binding sites for the polymerases in the presence of meAda.The location of the binding site for meAda , the -35 and -10 sequences , and the transcriptional start site ( indicated by the arrow ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>The Ada consensus sequences separated by six base pairs are underlined ( Landini and Volkert 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
<li>Neither of the two sequences is present in the aidB promoter , and a different common sequence , AAT-NNNNNNGCAA , has recently been proposed as the meAda recognition site ( Landini and Volkert , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>All Ada-dependent promoters display a common sequence , AAT ( N ) 6GCAA , thought to be the recognition sequence for meAda ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Sequence Homology between the Ada Protein Binding Sites in the ada and a / @ Promoter Regions The ada sequence ( top ) is from Demple et al. ( 1965 ) and the elkA sequence ( bottom ) is from Nakabeppu et al. ( 1964b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>As the phosphate bonds belonging to the ado regulatory sequence are not always completely protected , t ' hr methylat , ed Ada protein make extended regions including the - 10 and the - 35 box sequence , are protected ( Fig. 5 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Presumably , a single Ada prot , ein molecule binds to the adn regulatory sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The ability to bind DNA both speciﬁcally and nonspeciﬁcally is a similarity to the Ada protein , which binds DNA nonspeciﬁcally in its unmethylated form and becomes able to recognize speciﬁc target sequences upon self-methylation ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>We assume that a specific methylation of the Ada protein leads to a conformational change of the protein that allows it to bind to the specific DNA sequence identified in the present study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>The Ada binding site is centrally located in a long but imperfect region of dyad symmetry ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>No sequence could he vonsidrretl as an inverted-repeat in the DNA region around Ada protrinbinding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Based on the amino-terminal sequence data of the purified Ada protein ( Table 111 ) , we deduced that the ATG codon at position 102 is the initiation site for the translation of ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
<li>Footprinting analyses revealed that t , he methylated Ada protein binds t , o a region from positions - 63 to - 31 , which includes the ada regulatory sequence AAAGCGCA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The methylated Ada protein would occupy a region ext ' ending from the nda regulatory sequence ( AAA-GCGCA ) t > o the edge of the -35 box ( TTGGTT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>This sequence is bound b methylated Ada protein , as deduced from nuclease protection experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>A major groove considered t ' o be recognized by the methylated Ada protein is shown by the dotted area of the methylat.ed complete inaccessibility protein to t ' his sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>This indicates that zinc-binding by histidine in Ada C69H is sufficient for recognition of the ada sequence and binding to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>It should be noted that there is an UP-element binding protein in E.coli , methylated Ada protein , which binds to an UP-like element sequence in ada and aidB promoters , and stimulates transcription by modifying the nature of the RNP ± promoter interaction ( Landini and Volkert , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The sequence-specific DNA binding regions of meC38 N-Ada16k To determine the sequence-specific DNA binding sites of the methylated form of the Ada protein , 2D H - N 1 15 HSQC spectra of meC38 N-Ada16k with DNA fragments containing an ada or alkA promoter sequence were recorded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16452614" target="_blank">16452614</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_21">
Transcription of the ada and alkA genes promoted by the methylated form of Ada involves the binding of the Ada protein to a sequence referred to as an Ada box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></label>
<input id="sit_similarity_21", type="checkbox">
<div class="SIT_similar">
<li>Transcription of the ada and alkA genes promoted by the methylated form of Ada involves the binding ot the Ada protein to a sequence referred to as an `` Ada box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li></div>
<li>The Ada Box The DNA sequences in the ada and a/kA promoters to which methylated Ada protein bind show distinct sequence homology ( Figure 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Nevertheless , Ada protein with the S-methylcysteine at position 69 exhibits an increased ability to bind a specific sequence in the ada promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>Methylation of Ada converts this protein to a positive regulatory factor that binds sequences upstream of the ada , alkA , and aidB promoters , stimulating their transcription ( 12 , 15 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>Transcription of the ada and the aidB genes is triggered by binding of the methylated Ada protein ( meAda ) to a specific sequence located 40-60 base pairs upstream of the transcriptional start , which is internal to an A/T-rich region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>The methyl-ated Ada protein ( meAda ) recognizes the sequence AAT ( N ) 6GCAA , which at ada and aidB is located 5 and 7 base pairs upstream of the 235 sequence , respectively ( 10 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The Cys-69-methylated Ada protein ( meAda ) 1 is able to recognize specific sequences upstream of its own promoter and at least two other promoters , alkA and aidB , and activate their transcription me ( 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Upon methylation , the Ada protein ( meAda ) activates its own operon , adalalkB , and at least two other genes , a / M and aidB , by binding to a specific sequence immediately upstream of its target promoters ( Furuichi et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>175 , No. 8 The Ada Protein Is a Class I Transcription Escherichia coli Factor of KUNIHIKO SAKUMI ,1 KAZUHIKO IGARASHI ,2 MUTSUO SEKIGUCHI , ' AND AKIRA ISHIHAMA2 * Department of Biochemistry , Medical Institute of Bioregulation , Kyushu University , Fukuoka 812,1 and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka 411,2 Japan Received 10 September 1992/Accepted 5 February 1993 The methylated Ada protein of Escherichia coli , a regulatory protein for the adaptive response , binds to a target DNA from positions -62 to -31 upstream of the ada gene and facilitates the binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>There is a six-nucleotide stretch between the AAAGCGCA sequence : the possible binding site for the methylated Ada portein , and the -35 box , to which RNA polymerase would bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>The fact that the potential -35 region of the ada promoter has poor homology with the -35 consensus sequence is consistent with the idea that Ada or an Ada-derived product acts as a positively acting factor by stimulating the binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>RNA polymerase did bind to the promoter once the methylat , ed Ada protein had bound to the upstream sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Ada protein would bind to the upstream region of the promoter to facilitate the proper binding of the RSA polymerase to the promoter sequence , To clarify the mechanism of transcriptional activation by the methylated Ada protein , we investigated interactions of the protein with the ada promoter and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>The covalently modified Ada protein binds specifically to promoter reglons containing the sequence d ( AAANNAAAGCGCA ) immediately upstream of the RNA polymerase binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>The present ' study revealed that the methylated Ada protein binds t ) o the region containing the ada regulatory sequence , AAAGCGCA , which has been assigned by sit,e-directed mutagenesis ( Nakamura et al. , 1988 ) , an t ' hat this binding facilitates further binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>12 345 -4631 DNA Binding Site of the Activated Ada Protein DNA sequences upstream of the RNA polymerase binding site of the ada gene were required for the activation of transcription by the methylated Ada protein , indicating that this promoter region contains a binding site for the activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>These data directly dem-onstrate that the methylated Ada protein binds specifically to a DNA region immediately upstream of the putative RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In the heat shock response the regulatory gene encodes a sigma factor , whereas the Ada protein binds specifically at a distinct site in the promoter region upstream of the RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In these conditions , both '' ` Ada and a bind specifically to the aidB and ada promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Ada differs from many other AraC/XylS family proteins in that it can activate transcription from either a site that overlaps the 235 region ( at alkA ) or from a site that is 5 to 7 bp upstream of the 235 region ( at ada and aidB ) ( 1 , 15 , 35 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>However , direct interaction of Ada and meAda with other subunits such as s can not be ruled out ; the location of the Ada binding site in the alkA promoter would favor such interaction , which might not be possible in the other Ada-de-pendent promoters where the Ada binding site is 6 -- 8 base pairs upstream of the 235 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>In ada and aidB , the Ada binding site is , respectively , 5 and 7 bp upstream of a fairly well conserved 235 sequence ( 14 , 28 ) ; at the alkA promoter , the Ada binding site overlaps a 235 hexamer which bears no similarity to the consensus sequence ( 1 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>The 252 to 230 area includes the recognition sequence we have recently proposed for methylated Ada , AATnnnnnnGCAA ( where n represents the part of the sequence that varies ) ( 10 ) , which in the alkA promoter spans between 246 and 234 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>Thus , the methylated form of Ada binds the alkA site more efficiently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>When methylated Ada binds to this sequence , transcription from both ada and alkA promoters is stimulated , as measured by run-off transcription assays [ Nakabeppu and Sekiguchi , 1986 ; Teo et al. , 19861 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>When methylated at Cys-69 , Ada is converted into a potent activator ofada and alkA transcription and binds to a sequence ( Ada box ) present in both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7937881" target="_blank">7937881</a>]<br></li>
<li>Taking this in account , Teo et al. ( 1986 ) proposed that the - 10 box of the Ada promoter is the sequence CAGCCT , which is located six nucleotides upstream from the actual transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>When Ada is methylated at position 69 ( reAda ) , it can bind to specific DNA sequences in the promoter region of the adaptive response genes , and activate their transcription ( Nakabeppu and Sekiguchi , 1986 ; Teo et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Thus , substitutions in aCTD result in reduced transcription from ada-dependent promoters because they prevent a from binding the promoter 260 to 240 transcription enhancer sequence , rather than impairing its direct interaction with meAda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In the absence of meAda , RNA polymerase binds to the 260 to 240 region , which also includes the meAda binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>This region serves as the meAda binding site , but it also closely resembles , in A1T content , location , and function , the UP element transcription enhancer sequence identiﬁed in the rrnBP1 promoter ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>RNA polym-erase holoenzyme binds the two promoter regions between 260 and 240 via its a subunit ; this area overlaps the meAda binding site and closely resembles the UP element of the rrnB P1 promoter with respect to location , high A/T content , and recognition by a ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>In both ada and aidB , meAda binds farther upstream , protecting sequences from 262 to 238 in DNase I protection experiments ( 10 , 19 ) ( see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>In contrast , changing either the 235 or 210 sequences of aidB to consensus results in a significant stimulation of ada-independent transcription in vivo , suggesting that meAda improves RNA polymerase interaction with the core promoter region ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Two sequences ( AAAGCGCA and AAANNAA ) present respectively in the ada and the alkA promoter regions have been proposed as the binding sites for meAda ( Teo et al. , 1986 ; Akimaru et al. , 1990 ; Furuichi et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Figure 6 , incubation of the promoter regions with 80 pmol of a subunit results in a shift ( Figure 6 , lanes 4 and 5 ) ; subsequent addition of meAda ( 1 pmol for the ada promoter , 2 pmol for aidB ) results in a supershift ( C2 , Figure 6 , lanes 6 ) , suggesting that the two proteins can bind simultaneously to their binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by Ada</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>MeAda and 0.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Recent results indicate that the Ada regulon is indeed induced during stationary phase and protects against active alkylators produced by nitrosation of amino acids in nongrowing cells ( 55 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>When cells reach stationary phase , intracellular concentrations of sS increase ; meAda prevents EsS from binding to alkA , thus reducing the amount of RNA polymerase available for alkA transcription and , in turn , its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>In contrast , not only does meAda fail to stimulate alkA transcription by EsS , consistent with the lack of conservation of K593 , K597 , and R603 in sS , but it negatively affects EsS-dependent transcription both in vivo and in vitro ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Our present results show that meAda negatively regulates EsS-de-pendent transcription of alkA , and this mechanism is used by E. coli to down-regulate alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>These observations suggest that meAda activates transcription by affecting RNA polymerase interaction with both the 235 and 210 elements of aidB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Deletion of the aCTD dramatically affects both meAda-depend-ent and independent transcription from the wild type aidB promoter , even though activation by meAda is not totally abolished ( Fig. 3 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Induction by alkylating agents is mediated by the transcriptional activator Ada , in its methylated form ( meAda ) ; the other forms of induction are Ada independent and require as , the alternative a factor mainly expressed during the stationary phase of bacterial growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>MMS , apart from the Ada response , induces the SOS system that increases expression of over 40 genes involved in DNA recombination , repair , replication and mutagenesis [ 25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Expression of these genes is regulated by Ada , and their induction provides protection against alkylation damage to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Overexpression of the unmethylated Ada in vivo is sufficient to induce alkA ( i.e. in the absence of alkylation damage ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9188491" target="_blank">9188491</a>]<br></li>
<li>The function of meAda as a negative regulator of EsSdependent transcription suggests a speciﬁc mechanism for turning off high-level expression of the alkA gene : alkA expression is activated as long as transcription in the cell is mostly dependent on Es70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>During stationary phase , however , the methylated form of Ada shuts off expression of alkA ; this repression is speciﬁc for sS-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>In contrast , only 1 pmol of methylated Ada or meth-ylated AdaK178P was necessary to induce signiﬁcant alkA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Model for transcription activation by meAda at ada and alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>These results indicate that the hinge region of Ada is important for ada but not alkA transcription and further support the notion that the mechanism ( s ) by which Ada activates ada transcription differs from that by which it activates transcription at alkA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>The fact that this cluster is not conserved in sS neatly explains why Ada is unable to activate transcription by EsS at the alkA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>meAda activates ada transcription by either Es70 or EsS with roughly the same efﬁciency , while the lack of a functional rpoS gene results in lower expression of the aidB gene even in the presence of meAda ( 22 , 69 ) , showing that the aidB promoter is dependent on both Ada and sS for optimal expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>meAda is able to activate transcription by EsS as well as Es70 at both the ada and aidB promoters ( 24 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>The methyl-ated form of the Ada protein is able to activate transcription by EsS as well as Es70 at both the ada and aidB promoters ( 16 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>Upon methylation , the Ada protein ( meAda ) activates its own operon , adalalkB , and at least two other genes , a / M and aidB , by binding to a specific sequence immediately upstream of its target promoters ( Furuichi et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Ada is also positive transcriptional regulator of the alkA and other BER genes [ 19,30 ] and this response can not be mounted in the Δada mutant we have used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827074" target="_blank">17827074</a>]<br></li>
<li>The actions of mutant Ada proteins on the ada and alkA promoters in viva were investigated using an artificially composed gene expression system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>( f ) Effects of ada smutations on the expression of the ada and alkA genes in vivo To investigate the effects of mutant Ada proteins on the expression of the ada and alkA genes in viao .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>The lower panel depicts the interaction of two activated forms of Ada with the operators of the & - alkB operon , and the alkA and aidB genes , resulting in induced expression of all four genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>is presumed to result in a conformational change and converts Ada into a strong transcriptional activator of its own gene ( ada ) and also of several other genes ( alkB , alkA , and aidB ) ( 17 , 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>After it interacts with methylated DNA , Ada protein functions as a positive regulatory element that controls the expression of the adaptive response by stimulating the expression of the ah-alkB operon , and the alkA and aidB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3278898" target="_blank">3278898</a>]<br></li>
<li>The methyl-ated Ada protein is a positive regulator of its own synthesis ( ada ) and that of other genes , such as alkA , alkB and aidB Similar to E. coli Ada Similar to E. coli Ada One of the two proteins required for the ad-aptative response to alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>One of these is by the induction of the adaptive response by the Ada protein that acts as a methyl transferase ; in its methylated form , Ada acts as a transcriptional regulator of the Ada operon genes ada , aidB , alkA , and alkB ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>Ada protein plays a central role in control of expression of the ada regulon , which is composed of at least three operons , the ada-alkB , the alkA and the aidB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Upon self-methylation , Ada stimulates transcription of four genes , ada , aidB , alkA , and alkB ( whose transcription is directed by the ada promoter ) ( 10 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>]<br></li>
<li>The cloned ada and a/kA genes and purified Ada protein have been used in cell-free systems to identify the inducing signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Expression of the aZkA gene is also controlled by the Ada protein , but the mode of regulation is somewhat different from that for the ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>DISCUSSION Expression of the ada and alkA genes , which are involved in adaptive response in E. coli , is positively regulated by Ada protein , which is encoded by ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3529081" target="_blank">3529081</a>]<br></li>
<li>Expression of the ada and the related genes , such as alkA , belonging to the ada regulon is positively controlled by the Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Expression of the ada and the related genes.such as alkA , is controlled by the Ada prot ' ein at , the t ' ranscriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Recently , Sekiguchi et al. ( 1966 ) also found stimulation of in vitro transcription of the ada and alkA genes by unmethylated Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Results ( a ) Promotion of the ada and alkA transcription by Ada protein Ada prot , ein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>Transcription of the a/kA gene might conceivably be activated by either unmethylated or methylated Ada protein , since induced cells contain high levels of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>In this system the amount of Ada protein relative to the copy number of the ada promoter was increased up to factor of 6 by altering the replication origins for both the reporter plasmid and Ada expression plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8676855" target="_blank">8676855</a>]<br></li>
<li>In this system , Ada protein would be constitutivel ) produced , since the lac promot , er ( Plac ) is placed upstream from the ada gene lacking its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3047400" target="_blank">3047400</a>]<br></li>
<li>Transcription in vitro by these reconstituted holoenzymes was examined by using a truncated DNA fragment containing the ada promoter ( 10 ) and in the presence and absence of Ada protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>The high levels of Ada protein synthesized in induced cells reflect the high levels of ada transcription that we were able to observe with the ada-lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>In the absence of activated Ada protein , primarily rpoS-mediated transcription is induced , since rpoS most severely impairs ada-indepen-dent forms of aidB induction ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002588" target="_blank">8002588</a>]<br></li>
<li>In reaction conditions in which methylated Ada protein clearly stimulated expression of the ada promoter , no Ada-induced transcription from the BglII-EcoRI fragment of aidB was detected ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961409" target="_blank">7961409</a>]<br></li>
<li>Methylation of cysteine at position 69 triggers a conformational change in the Ada protein ; cysteine-69-methylated Ada ( meAda ) is a speciﬁc DNA binding protein able to promote transcription of its own gene , ada , and of the alkB gene , whose transcription is also me driven by the ada promoter ( 25 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li>
<li>A low but significant level of the alkA gene expression occurs in the absence of Ada protein , and unmethylated and methylated forms of Ada proteins are equally effective in promoting alkA transcription in vitro ( Nakabeppu & Sekiguchi , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>This notion is supported by the recent findings that Ada protein , especially in its methylated form , promotes transcription of the ada gene in vitro ( Teo et al. , 1986 ; Nakabeppu & Sekiguchi , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3139888" target="_blank">3139888</a>]<br></li>
<li>Using an in vitro reconstituted system , we found that both the sigma factor and the methylated Ada protein are required for transcription of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648001" target="_blank">2648001</a>]<br></li>
<li>It has been demonstrated that the Ada protein autoregulates its own synthesis ( 17 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868601" target="_blank">7868601</a>]<br></li>
<li>Since the a & gene product appears to induce its own synthesis in vivo , the pCS88-directed system was supplemented with different forms of the purified Ada protein in attempts to stimulate expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>We showed that synthesis of the ada gene product is induced by exposure to the methylating agent MNNG and that the Ada protein or some derivative of it acts positively in trans to increase expression of an ada-lacZ operon fusion , suggesting a stimulation oftranscription of the ada gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982792" target="_blank">2982792</a>]<br></li>
<li>Interestingly , the adaptive response genes , including alkA , are transcribed more efﬁciently by EsS than by Es70 both in vitro and in vivo in the absence of meAda protein ( 22 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Since the adaptive response takes place in & a + , but not in ada - cells , which have been treated with methylating agents , it seemed likely that theAda protein which accepted the methyl group would be an activated form which promotes expression of the two inducible genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2987251" target="_blank">2987251</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>