<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - AppY</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">AppY</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">AppY transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The signal is probably not the oxygen or redox state , since the AppY-dependent growth-phase induction also took place in well-aerated cultures , but could be a change in the concentration of a metabolite interacting with the AppY protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>We think that it is very likely that during the deceleration phase of growth the AppY protein is responding to the same signal as that elicited by anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>Thus synthesis of AppY is transcriptionally regulated by s s in response to the growth-phase w84x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>176 , No. 17 f R ole of the Transcriptional Activator AppY in Regulation o the cyx appA Qperon of Escherichia coli by Anaerobiosis , Phosphate Starvation , and Growth Phase TOVE ATLUNGl * AND LONE BR0NDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , The Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 6 December 1993/Accepted 13 June 1994 Transcriptional lacZ fusions have been used to analyze the regulation of the appA operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Apr. 1997 , p. 2141 -- / 97 / $ 04.0010 Copyright q 1997 , American Society for Microbiology Effects of sS and the Transcriptional Activator AppY on Induction o the Escherichia coli hya and cbdAB-appA Operons in Response to f Carbon and Phosphate Starvation TOVE ATLUNG ,1 * KASPER KNUDSEN ,1 LOTTE HEERFORDT ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , 2800 Lyngby ,2 Denmark Received 21 October 1996/Accepted 29 January 1997 The transcriptional regulation of two energy metabolism operons , hya and cbdAB-appA , has been investigated during carbon and phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1996 , p. 3418 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology e Histone-Like Protein H-NS Acts as a Transcriptional Repressor for Expression of the Anaerobic and Growth Phase Th Activator AppY of Escherichia coli TOVE ATLUNG ,1 * SUSANNE SUND ,2 KIRSTEN OLESEN ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 12 October 1995/Accepted 9 April 1996 The transcriptional-activator AppY is required for anaerobic and stationary-phase induction of the cyx-appA and hya operons of Escherichia coli , and expression of the appY gene itself is induced by these environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_7">
References Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></label>
<input id="act_similarity_7", type="checkbox">
<div class="ACT_similar">
<li>Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>We are presently dissecting the relatively complex transcriptional unit which contains the appA gene , by isolation of the three promoters that contribute to its transcription , to study their individual regulation with respect to anaerobiosis , phosphate-starvation , and activation by AppY protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>Atlung , T. , Knudsen , K. , Heerfordt , L. and Brondsted , L. ( 1997 ) Effects of sigmaS and the transcriptional-activator AppY on induction of the Escherichia coli hya and cbdAB-appA operons in response to carbon and phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_12">
Effects of sS and the transcriptional-activator AppY on induction of the Escherichia coli and cbdAB-appA operons in response to carbon and phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></label>
<input id="act_similarity_12", type="checkbox">
<div class="ACT_similar">
<li>Effects of σs and the transcriptional-activator AppY on induction of the Escherichia coli hya and cbd -- appA operons in response to carbon and phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Effects of sS and the transcriptional-activator AppY on induction of the Escherichia coli hya and cbdAB-appA operons in response to carbon and phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Effects of S and the transcriptional-activator AppY on induction of the Escherichia coli hya and cbdAB-appA operons in response to carbon and phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11807064" target="_blank">11807064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057949" target="_blank">12057949</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_16">
Role of the transcriptional-activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobi-osis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11807064" target="_blank">11807064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></label>
<input id="act_similarity_16", type="checkbox">
<div class="ACT_similar">
<li>Role of the transcriptional-activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobi-osis , phosphate-starvation and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222606" target="_blank">11222606</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Role of the transcriptional-activator AppY in regulation of the cyx appA operon of Esche-richia coli by anaerobiosis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>Role of the transcriptional-activator AppY in regulation of the cpx appA operon of Escherichia coli by anaerobiosis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Role of the transcriptional-activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Atlung , T. , and Brendsted , L. ( 1994 ) Role of the transcriptional-activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Atlung , T. , and Bronsted , L. ( 1994 ) Role of the transcriptional-activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung T , Brøndsted L ( 1994 ) Role of the transcriptional-activator AppY in regulation of the cyx-appA operon of Escherichia coli by anaerobiosis , phosphate-starvation , and growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li></div>
<li>In addition , we identiﬁed a transcriptional-regulator , AppY , that is likely to regulate synthesis of yet another potential anti-adaptor ; overproduction of AppY stabilizes sS independently of IraP , IraD and IraM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>In addition to RpoS , appB is regulated by AppY ( 6 ) , which itself is subject to regulation by RpoS ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>AppY , on the other hand , is a known transcriptional-regulator , suggesting that it may act by increasing the transcription of one or more anti-adaptors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>AppY and sS AppY , a transcriptional-regulator , causes sS stabilization when overexpressed , and the effect is seen even in cells deleted for iraP , iraD and iraM ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Furthermore , induction also inﬂu-enced the steady-state amounts of sS ( Fig. 2A and B , cf. the time 0 min for each set of samples ) , presumably owing to the effect of IraD , AppY and IraM on sS stability before the chase was begun .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Regulated expression of 21 small proteins Comparison of growth under the two indicated conditions Minimal glucose vs minimal glycerol Proteina M b ap c LB vs Aerobic vs Mock treatment vs pH 7.5 vs 30 ° C vs 37 ° C vs Mock Mock low oxygen SDS EDTA pH 5.5 45 ° C 10 ° C treatment treatment vs vs H2O2 diamide RegulonDB minimal glucose YpfM YccB Monocistronic None d appAB-yccB - 38 , 70 , AppY , appC ArcA Monocistronic None Monocistronic None cydAB-ybgTE 70 , ArcA , Fnr , FruR , HNS ykgMO 70 , Zur yqgB-speAB None Monocistronic None yjhBC-ythA None Monocistronic NL yqeKL None Monocistronic NL Monocistronic NL ilvXGMEDA 70 , Lrp , IHF yeaP-yoaKJ None Monocistronic NL yeaP-yoaKJ None yoeI-yeeF None Monocistronic None In IsrB sRNA NL yobF-cspC None YncL YneM - - e YbgT - ¬ YkgO YqgB YnhF YthA YobI YqeL YohP YkgR IlvX YoaJ YpdK YoaK YoeI YbhT AzuC YobF - - - - - ¬ - ¬ - ¬ - - - ¬ - - ¬ - - - ¬ - ¬ - -- ¬ -- - ¬ - ¬ - ¬ - - ¬ a All small proteins that had at least a 4-fold change in protein levels in at least one experiment are listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>This protein , UspA ( universai stress protein A ) , is thus a generai , non-specific responder to growth arrest ( this paper ; Nystrom and Neidhardt , 1992 ) , and its induction is independent of RpoS as weli as of aii other giobal reguiators so far tested , inciuding PhoB , ReiA , SpoT , AppY , OmpR , H-ns , Lrp , and RpoH ( Nystrom and Neidhardt , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152377" target="_blank">8152377</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>Fat arrows , strong stimulation of expression ( .10 - fold ) ; medium arrows , medium stimulatory effects ( 5 - to 10-fold ) ; dotted arrows , weak stimulatory effects ( 2 - to 5-fold ) ; curved arrows , reactions ; X , unknown low-molecular-weight effector for AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>Synthesis of the AppY protein is regulated by the ArcA regulator , which responds to changes in respiration ( 21 ) , and the AppY protein is itself activated in response to anaerobiosis ( 7 ) , probably by association with a low-molecular-weight effector like some of the other members of the AraC family of activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>However , the effect was dependent on an intact appY gene , and there was no speciﬁc binding of H-NS to these promoters , indicating that the effect is indirect , being mediated by increased concentrations of AppY protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>AppY overproduction stimulated expression of the cyx promoter 20-fold , whereas PappA , like the control promoter pd , , A , was virtually unaffected by the presence of the appY + plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>During aerobic-growth in glucose minimal medium , there is a two - to threefold reduction in the specific 3-galactosidase activity in the appY mutant strain ( Table 4 and see Fig. 8 ) , indicating that there is also a small stimulation of the cyx promoter by AppY protein under this growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>In contrast , for the hyaA and hybO promoters , IscR repressed the previously identiﬁed O2-regulated transcription start sites ; under anaerobic-growth conditions , AppY and the ArcA/B two-component regulatory system have been implicated in the activation of transcription of the hya operon ( Brondsted and Atlung , 1994 ; Richard et al. , 1999 ) , whereas FNR appears to contribute to activation of the hybO promoter ( Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>ArcA and AppY antagonize IscR repression of hydrogenase-1 expression under anaerobic conditions , revealing a novel mode of O2 regulation of gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>ArcA and AppY Antagonize IscR Repression of Hydrogenase-1 Expression under Anaerobic Conditions , Revealing a Novel Mode O2 Regulation of Gene Expression in Escherichia coli of A. D. Nesbit , * A. S. Fleischhacker , S. J. Teter , and P. J. Kiley Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , USA Transcription of the Escherichia coli hydrogenase-1 operon ( hyaABCDEF ) is increased by the transcription factors ArcA and AppY under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Surprisingly , we also found that even in the absence of AppY and ArcA , wild-type IscR could not repress PhyaA anaerobically as well as an IscR mutant that lacked the Fe-S cluster or as well as wild-type IscR aerobically at the same protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The repression by IscR in the absence of either ArcA or AppY appears to be a direct result of IscR binding , since the increase in expression observed from a mutation that is known to disrupt the IscR binding site ( C-to-G mutation at position 50 in PhyaA [ PhyaA ( 50G ) ] ) ( 28 ) mimicked the increase in expression of the iscR strains ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>thermore , by similarly increasing the levels of IscR in a strain lacking ArcA and AppY , we found that less IscR was required for repression anaerobically ( Fig. 3C ) than in strains containing AppY and ArcA ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , much greater repression was observed aerobically than anaerobically at the same concentrations of IPTG , consistent with direct IscR repression at PhyaA without competition from ArcA and AppY aerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In sum , we have found that a combinatorial strategy leads to ArcA and AppY overcoming IscR repression at PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Although isolation of AppY will be required before we can elucidate how this protein inhibits IscR repression at PhyaA , the simplest explanation is that both AppY and ArcA block IscR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The ability of ArcA and AppY to act as antirepressors of IscR repression of PhyaA depended on IscR levels , suggesting that IscR competes with ArcA and/or AppY for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Therefore , while the role of AppY in preventing repression by IscR is unclear , the simplest model from the combined results is that both AppY and ArcA act as antirepressors , competing with the repressor IscR for binding to PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>As IscR protected a region of the hyaA promoter that overlaps the − 35 promoter element and site ( s ) where an activator may bind , one mechanism to explain the lack of IscR repression of the hyaA and hybO promoters under anaerobic conditions would be the occlusion of IscR binding by anaerobic regulators ( ArcA , AppY or FNR ) and/or RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>ArcA and AppY had no effect on PhyaA expression under aerobic conditions , as expected ( Fig. 1C ) , since ArcA is not active aerobically and since AppY is poorly expressed under these growth conditions ( 1 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>For example , the decreased expression of PhyaA that arises when strains lack ArcA ( AppY and IscR ) or AppY ( ArcA and IscR ) is reversed by the further elimination of IscR from either strain ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , elimination of all three transcription factors ( IscR , AppY , and ArcA ) resulted in an unexpectedly high level of expression under anaerobic conditions , which was slightly greater than the level of expression in the parent strain ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Therefore , the decreased expression in strains lacking ArcA and/or AppY under anaerobic conditions does not appear to result from a defect in activation by ArcA or AppY but , rather , from unmasking IscR repression in the absence of these other transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Our data indicate that the decrease in expression upon elimination of ArcA or AppY is not due to activation by either transcription factor , but , rather , to repression by IscR in the absence of ArcA and/or AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>More recently , another transcription factor , IscR , has been shown to directly repress the hyaA promoter ( PhyaA ) but only under aerobic conditions when ArcA and AppY are not active ( 14 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Previous studies ( 5 , 31 ) showed that deletion of ArcA or AppY results in decreased expression of PhyaA under anaerobic conditions , which led to the hypothesis that ArcA and AppY activated PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>To test whether increasing the concentration of IscR could overcome antirepression by ArcA and AppY , we varied the levels of both the wild-type and a clusterless mutant of IscR and monitored the expression of the hya promoter under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Therefore , downregulation of IscR levels under anaerobic conditions appears to be important for proper regulation of PhyaA , such that IscR competition with ArcA and AppY for PhyaA binding is maintained at a low level , keeping PhyaA from being repressed anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Hence , the concentration of IscR , in addition to regulation by ArcA and AppY , is important for determining the anaerobic expression levels of PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Our studies revealed that IscR repressor function is diminished under anaerobic conditions because of antirepression by ArcA and AppY , decreased IscR levels , and decreased repressor function of [ 2Fe-2S ] - IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In sum , a combination of ArcA and AppY antirepression of IscR function , lower levels of IscR , and weak repression by [ 2Fe-2S ] - IscR leads to increased PhyaA expression under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Antirepression by the anaerobic transcription factors ArcA and AppY , in addition to a combination of decreased levels of wild-type IscR and the weak repressing function of [ 2Fe-2S ] - IscR , resulted in the lack of repression by IscR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The ﬁnding that PhyaA was repressed by IscR under aerobic conditions was surprising since activation of PhyaA under anaerobic conditions by ArcA and AppY would make an aerobic repressor unnecessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The lack of repression of PhyaA under anaerobic conditions appears to be linked to the lower anaerobic levels of IscR ( 10-fold less than aerobic protein levels [ 13 ] ) since artiﬁcially increasing the levels of wild-type IscR to those found under aerobic conditions resulted in increased repression of PhyaA by IscR , even in the presence of ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>RESULTS AppY and ArcA prevent IscR repression of the hyaA promoter under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>hyaA In this study , we examined the roles of IscR , ArcA , and AppY in regulating the expression of the hyaA promoter under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA , AppY , and IscR act combinatorially to control hyaA promoter expression levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Consistent with the notion that PhyaA expression is regulated by IscR repression aerobically and by ArcA and AppY antirepression of IscR anaerobically , PhyaA-lacZ expression levels in the absence of any transcription factors were similar under anaerobic and aerobic conditions ( Fig. 1A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Despite the fact that we observed more repression of the hyaA promoter with increasing levels of IscR , the extent of PhyaA repression under anaerobic conditions was never as great as that measured under aerobic conditions , even in the absence of AppY and ArcA ( Fig. 1 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Here , we report that ArcA and AppY increase PhyaA expression under anaerobic conditions by antagonizing IscR binding at PhyaA , since IscR repression is observed when either ArcA or AppY is eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>A further , but small , increase in repression of PhyaA was observed with IscRC92A when ArcA and AppY were eliminated ( Fig. 3C ) , indicating that the ability of apo-IscR to repress PhyaA under anaerobic conditions is largely indifferent to the presence of ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>These data also show that both AppY and ArcA must be present to prevent IscR repression of PhyaA under anaerobic conditions , since loss of either transcription factor is sufﬁcient to permit repression by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>We found that under anaerobic conditions two transcription factors , ArcA and AppY , work together to overcome the repression of PhyaA by IscR that occurs under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>This could reﬂect the elevated levels of appY transcription in sta-tionary-phase cultures ( 10 ) , which could presumably lead to increased AppY stimulation of hya expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Expression of the hya operon , like that of cbd , is stimulated by an increased AppY concentration ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>The cbd promoter seems to be stimulated quite efﬁciently by increased AppY protein under nonactivating conditions ( 1 ) , while the hya promoter is primarily stimulated by activated AppY protein ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>Since transcription of the appY gene is induced under anaerobic conditions ( 1 ) , this could be due to increased synthesis of AppY protein at the onset of anaerobiosis , and thus only as the concentration of AppY builds up does the cyx promoter become fully activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>Expression of the appY gene is itself induced by anaerobiosis , entry into stationary phase , and phosphate-starvation ( 8 ) , indicating that the increased AppY concentration under these conditions contributes to the induction of the cbd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>The AppY-depen-dent stimulation of the cbd promoter might be mediated by increased levels of AppY protein , since appY expression is also strongly induced by phosphate-starvation ( 8 ) ( see Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>This operon is induced under an-aerobic conditions in the presence of formate or fumarate , repressed in the presence of nitrate , and requires acidic pH , ArcA , and AppY for optimal expression ( 12 , 21 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Anaerobic induction of hya expression was dependent on the ArcA and AppY regulators , which conﬁrms earlier observations by other authors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>In addition , anaerobic induction of hya expression was partially dependent on AppY , while hybO expression was partially dependent on NarP even in the absence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The expression of the appY gene is induced immediately by anaerobiosis , and this anaerobic induction is independent of Fnr , and AppY , but dependent on ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>The activation by AppY includes an indirect effect of ArcA , since the anaerobic induction of the appY gene is partially dependent on ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>We found that the transcription of the hya operon was induced when the growth condition was changed from aerobic to anaerobic and that this induction was independent of Fnr but dependent on regulators AppY and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>We found that transcription was induced by a shift from aerobic to anaerobic conditions and that this induction was regulated by ArcA and AppY but not Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>Under anaerobic conditions , the activity of the response-regulator ArcA is increased through phosphorylation by the sensor kinase ArcB ( 1 ) , while the activity of AppY is presumably increased by changes in appY transcription ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The anaerobic transcription of the hydrogenase 1 operon is activated by ArcA and AppY , which seem to act independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>NarL and NarP appear to be responsible for repression of hya expression in response to nitrate ( 5 , 31 ) , whereas the anaerobic transcription factors ArcA and AppY are required for the increased expression of the hya operon in response to anaerobiosis ( 5 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The effector for AppY might be formate since we found that both the hya operon ( 9 ) and the cyx-appA operon were induced by formate and that the formate inductions were independent of FhlA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>AppY regulates the appCBA operon ( 5 ) , which was induced by EvgA overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>Furthermore , the cyx promoter is induced by anaerobic-growth , and we have recently shown that this induction is dependent on AppY but independent of Fnr ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>As shown in Table 2 , the same sequences of the cyx promoter are required for stimulation by AppY overproduction and anaerobic induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>In the accompanying paper ( 7 ) , we show that the AppY protein is also involved in anaerobic induction of transcription of the hya operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>It is possible that period of anaerobic-growth a is needed to accumulate sufficient AppY protein obtain to full activation of transcription of the cyx-appA and hya operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>Regulation of stationary-phase induction of the cyx-lacZ fusion by ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>ArcA and AppY activated transcription of the cyx-appA operon during entry into stationary phase and under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Since the stationary-phase induction was found to be dependent on ArcA and AppY , we examined the effect of ArcA on the anaerobic induction of the cyx-appA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Anaerobic regulation of the cyx-appA operon by ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>A different pattern was found in the anaerobic regulation of the acid phosphatase operon , suggesting that here ArcA and AppY act coordinately in activating the anaerobic transcription of the cyx-appA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>AppY activates transcription of the hydrogenase 1 operon and the acid phosphatase operon during anaerobic-growth , and we have suggested that AppY is a third global transcriptional-regulator involved in anaerobic gene regulation ( 1 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Virtually no anaerobic induction was found when one of the two activators was absent , indicating that AppY and Arc depend on each other when activating the transcription of the acid phosphatase operon during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>It is not known whether AppY responds directly or indirectly to the O supply .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>C. Effect of AppY over-expression on s S stability in the absence of IraP , IraM , and IraD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>In the absence of RpoS or AppY , the response of hya expression onset to external pH was evident and maximal hya levels remained greater in acidic than in alkaline medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>The response of hya expression to pHe was tested in the absence of AppY to determine whether AppY was involved in the response of hya to pHe ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Expression of hya in response to pHe in the absence of AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>hya responds to pHe in the absence of AppY but at reduced levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Since carbon starvation caused induction of the appY gene and therefore presumably an increase in AppY protein concentration , we investigated the carbon starvation response of the pcbd9-lacZ , and phya9-lacZ fusions in appY mutants ( but not the pappY9-lacZ fusion , since appY is not autoregulated [ 8 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>The weak induction of the pcbd9-lacZ fusion was dependent on AppY in both the absence and presence of RpoS , whereas TABLE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">AppY protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>A Kyte-Doolittle hydro-pathy plot of the related protein , AppY , does not reveal an obvious membrane-spanning region between amino acids 150 and 190 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>The only significant amino acid sequence similarity found upon inspection of the diagon outprints coincided with a probable DNA-binding domain in the AppY sequence , amino acids 149 to 168 ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>Using the weight matrix of Dodd and Egan for evaluating DNA-binding domains ( 13 ) , we found a score of 1,210 for AppY ; only 3 % of the ca. 2,500 proteins in a data base contained amino acid sequences giving scores above 1,200 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">AppY evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>continued Subcellular localization f Function lrp a 30.69 c 2.72 2.69 8.72 2.60 23.70 2 2.22 2 2.00 2 2.00 23.33 2 2.22 2 2.08 2 4.00 2 2.00 2 2.70 2 2.77 2 3.13 2 2.08 2 2.00 2 2.50 3.75 2 2.04 2 2.99 2 2.00 2 2.17 2 2.50 22.86 9.76 14.00 Cytoplasmic Cytoplasmic Cytoplasmic f Cytoplasmic f Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic f Leucine-responsive regulatory protein Regulator of the two-component system PhoP/PhoQ 5E-2 1E-2 7E-3 3E-3 1E-3 2E-2 3E-2 1E-2 1E-2 5E-2 4E-3 2E-3 5E-2 4E-4 2E-2 2E-2 1E-2 8E-3 6E-3 5E-2 5E-2 5E-2 2E-2 2E-2 5E-2 4E-2 2E-2 5E-2 4E-2 P19494 P23836 P77170 P77574 P24210 P02384 P02387 P02386 P02390 P02418 P02409 P02392 P02411 P02413 P02416 P02424 P02425 P02429 P02435 P02436 P13445 P02361 P02363 P02369 P02370 P02373 P02375 P30017 P75989 phoP b pinQ b pinR b rcsA ab rplA b rplB rplC rplF rplI b rplK rplL rplN rplO rplQ rplW rplX rpmC rpmF rpmG rpoS a rpsH rpsI rpsM rpsN rpsQ rpsS b stpA a ycgE DNA invertase DNA invertase Regulatory protein 50S ribosomal subunit protein L1 50S ribosomal subunit protein L2 50S ribosomal subunit protein L3 50S ribosomal subunit protein L6 50S ribosomal subunit protein L9 50S ribosomal subunit protein L11 50S ribosomal subunit protein L7/L12 50S ribosomal subunit protein L14 50S ribosomal subunit protein L15 50S ribosomal subunit protein L17 50S ribosomal subunit protein L23 50S ribosomal subunit protein L24 50S ribosomal subunit protein L29 50S ribosomal subunit protein L32 50S ribosomal subunit protein L33 RNA polymerase sigma factor 38 30S ribosomal subunit protein S8 30S ribosomal subunit protein S9 30S ribosomal subunit protein S13 30S ribosomal subunit protein S14 30S ribosomal subunit protein S17 30S ribosomal subunit protein S19 DNA-binding protein Similar to P33358 , YehV putative transcriptional regulator ( E. coli ) PS00552 : Bacterial regulatory protein MerR family signature Similar to P05052 , AppY regulatory protein ( E. coli ) PS00041 : Bacterial regulatory proteins AraC family signature Similar to O44006 , regulatory protein CzcR ( Ralstonia eutropha ) Similar to U32711 , sigma54 modulation protein , putative ( H. influenzae ) Similar to P05052 , AppY regulatory protein ( E. coli ) PS00041 : Bacterial regulatory proteins AraC family signature Similar to P10805 , regulatory protein EnvY ( E. coli ) PS00041 : Bacterial regulatory proteins AraC family signature Similar to P76268 , KdgR transcriptional regulator ( E. coli ) PS01051 : Bacterial regulatory proteins , IclR family signature ydeO b P76135 2E-2 3.87 Cytoplasmic 35.36 c 3.72 32.41 c yedW b yfiA yhiW b P76340 P11285 P37638 5E-2 4E-2 2E-2 Cytoplasmic probable Cytoplasmic f Cytoplasmic f yhiX b Cytoplasmic f P37639 4E-4 13.92 24.07 c yjhI b Cytoplasmic f P39360 4E-2 Other functions 2.50 d 4.10 D appA b cspB 3E-2 5E-2 Acid phosphatase , pH 2.5 Similar to cold shock protein P07102 P36995 Periplasmic Cytoplasmic probable ain signature PS00352 : Cold shock DNA-binding dom Similar to cold shock protein PS00352 : Cold shock DNA-binding dom P36996 4.73 1E-2 cspC Cytoplasmic probable ain signatur Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>u000 .00 opeono transcriptional regulator 27.8 % PS00 1 : HTH-ARAC-FAMILY [ Bacterial regulat0ory proteins , AraC family signature ] I00 mtcoh SimlarIto14eoeralo regulaors i0 AraC fomily contr ( oligouter-rnembrane protein cxpression : ENVY ECOLI ( SW ) 253 Escheric4hiacoliEnvY , poron heroreogulatboryprotein 40.1 % STYPEFABCD -.13 ( GB ) 259 Salmoneolla typhimuroum ORF ( produton90W kb virulence ploasmid 40.4 % APPYOECOLI ( SW ) 243 0146eri46 ( 0 41o1 ) AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8041620" target="_blank">8041620</a>]<br></li>
<li>Primer extension analysis defined the promoter.The amino acid sequence of AdiY showed homologyto the XylS/AraC family of transcriptionalregulators , which includes Envy and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>- v % 1500 m 2s 1000 c , U m 0 500 W @k Overlapping interactions between AdiY , AppY and Envy AdiY has sequence similarity to the XylS/AraC family of transcriptional activators which includes AppY and Envy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Synthesis of the AppY protein is regulated by the ArcA regulator , which responds to changes in respiration ( 21 ) , and the AppY protein is itself activated in response to anaerobiosis ( 7 ) , probably by association with a low-molecular-weight effector like some of the other members of the AraC family of activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Like CfaD and VirF -- its close relatives in the AraC family -- AppY is a link in a regulatory cascade which involves the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>The ﬁnal gene product , SrgC ( ORF11 ) , is homologous to several regulatory proteins of the AraC family including EnvY , AppY , and Rns ( from E. coli ) and VirF ( from Y. enterocolitica ) ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495757" target="_blank">9495757</a>]<br></li>
<li>The subject of this report , adiY , is a member of the XylS/AraC family of transcriptional regulators which now contains more than 30 members , including AppY and Envy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>The anaerobic induction is dependent on two transcriptional activators ( 2 , 8 ) : ( i ) the ArcA response regulator ( 18 ) , which is activated by the ArcB sensor in response to reduced respiration ( 17 , 19 ) , and ( ii ) the AppY protein , which is a member of the AraC family of transcriptional activators ( 3 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>The AppY protein is homologous to the transcriptional activators of the AraC family ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>The AppY protein is a member of the AraC superfamily of transcriptional activators ( 17 ) and is most homologous to the members of the family involved in pathogenesis , e.g. , CfaD of E. coli and VirF of S. ﬂexneri .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by AppY</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>ﬂuoride or Pefabloc SC ; BCA , bicinchoninic acid ; delta Cn , correlation value ( difference between the ﬁrst hit and the second hit ) ; DTE , dithioerythritol ; DTT , dithiothreitol ; iTRAQ , a multiplexed set of isobaric reagents that yield amine-derivatized peptides ( iTRAQ reagents ; Applied Biosystems , CA ) ( 253 ) ; PMSF , phenylmethylsulfonyl ﬂuoride ; RSp , rank preliminary score ; SELDI-TOF-MS , surface-enhanced laser desorption ionization -- time of ﬂight mass spectrometry ; TCA , trichloroacetic acid ; Xcorr , cross-correlation ( measures how close the spectrum ﬁts to the ideal spectrum ) Biotin carboxyl carrier protein of acetyl-CoA carboxylase AccB P0ABD8 179 , 286 , 287 , 323 4.66 / 16,687.21 4.57 / 22,029 4.74 / 18,338 ( 4 -- 5 ) 5.01 / 18,459 ( 4 -- 5 ) 4.78 / 15,809 ( 4 -- 5 ) Biotin carboxylase 6.65 / 49,320.74 7.58 / 33,321.89 P24182 P0A9Q5 AccC AccD 179 179 Acetyl-CoA carboxylase carboxyl transferase subunit beta Isocitrate lyase Protein function and expressione Acetyl-CoA carboxylase is a heterodecamer of four copies of biotin carboxyl carrier protein ( AccB ) , two copies of biotin carboxylase ( AccC ) , and two copies each of the two subunits of carboxyl transferase ( AccA and AccD ) ; AccC catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA See AccA ; induced by high pH during anaerobic growth 4.57 / 22,029 4.74 / 18,338 ( 4 -- 5 ) 5.01 / 18,459 ( 4 -- 5 ) 4.78 / 15,809 ( 4 -- 5 ) See AccA See AccA 6.65 / 49,320.74 7.58 / 33,321.89 P24182 P0A9Q5 AccC AccD 179 179 Involved in glyoxylate-bypass ; activated by phosphorylation ( on histidine ) and inhibited by phosphoenolpyruvate ( PEP ) , 3-phosphoglycerate , and succinate ; induced by pH change ; increases in the physiological short-term adaptation to glucose limitation The pyruvate dehydrogenase complex contains multiple copies of three enzymatic components : pyruvate dehydrogenase ( E1 ) , dihydrolipoamide acetyltransferase ( E2 ) , and lipoamide dehydrogenase ( E3 ) ; catalyzes the overall conversion of pyruvate to acetyl-CoA and CO2 ; increases during aerobic growth and after benzoic acid treatment See AceE ; increases during aerobic growth , during the low temp at 10 ° C , and after benzoic acid treatment Involved in the activation of acetate to acetyl-CoA and the secretion of acetate ; involved in the synthesis of most of the ATP formed catabolically during anaerobic growth ; induced by low pH during anaerobic growth Aconitate hydratases ( AcnA and AcnB ) serve as a protective buffer against the basal level of oxidative stress that accompanies aerobic growth by acting as a sink for reactive-oxygen-species and by modulating translation of the sodA transcript ; AcnA enhances the stability of the sodA transcript , whereas AcnB lowers its stability ; it is induced by iron and oxidative stress and may have an ironresponsive regulatory function See AcnA AceA 27 , 179 , 287 , 311 5.16 / 47,521.57 P0A9G6 5.19 / 44,130 ( 4.5 -- 5.5 ) 5.12 / 44,130 ( 4.5 -- 5.5 ) 5.01 / 33,761 ( 4.5 -- 5.5 ) 4.99 / 33,624 ( 4.5 -- 5.5 ) 5.01 / 51,702 ( 5 -- 6 ) AceE 5.40 / 99,289 5.46 / 99,537.29 P0AFG8 Pyruvate dehydrogenase E1 component 179 , 228 , 270 , 294 , 321 5.39 / 94,195 ( DIGE 4.5 -- 6.5 ) AceF Dihydrolipoyllysine residue acetyltransferase component of pyruvate dehydrogenase complex Acetate kinase P06959 133 , 179 , 228 , 270 , 294 , 321 5.01 / 77,450 5.09 / 65,964.87 5.09 / 78,105 ( DIGE 4.5 -- 6.5 ) AckA 5.84 / 44,001 5.85 / 43,290.45 P0A6A3 179 , 228 , 294 , 287 , 321 , 323 5.79 / 41,417 ( DIGE 4.5 -- 6.5 ) 5.72 / 31,846 ( 5 -- 6 ) 5.59 / 97,516.00 Aconitate hydratase 1 P25516 AcnA 151 , 278 AcnB 151 , 179 , 278 , 287 , 321 Aconitate hydratase 2 P36683 5.30 / 95,000 ( DIGE 4.5 -- 6.5 ) 5.24 / 75,907 ( 4.5 -- 5.5 ) 5.24 / 93,498.11 Continued on facing pag TABLE 2 -- Continued pI/MW Descriptionb Protein function and expressione Experimentald Theoreticalc AcpP P0A6A8 Acyl carrier protein 3.98 / 8,508.33 A carrier of the growing fatty acid chain in fatty-acid-biosynthesis ; even though AcpP is an abundant protein ( ca. 60,000 copies per cell ) , its small size ( 8.5 kDa ) makes it difﬁcult to detect by typical 2-DE ; it can be detected after subcellular fractionation AcrAB is a drug efﬂux protein with broad substrate speciﬁcity 1 Acriﬂavine resistance protein A Adenosine deaminase Adenine deaminase AcrA P0AE06 179 6.08 / 39,723.72 Add P22333 Ade P31441 ( YicP ) 5.36 / 36,397.46 5.63 / 34,723.76 179 321 5.22 / 61,539 ( DIGE 4.5 -- 6.5 ) Repressed by H-NS but activated by insertion of a variety of insertion elements into a region extending from 145 to 13 relative to the transcription start site ; decreases after benzoic acid treatment AdhE has three activities : alcohol dehydrogenase , acetaldehyde dehydrogenase , and PFL deactivase ; PFL deactivase activity catalyzes the quenching of the pyruvate-formate-lyase catalyst in an iron - , NAD - , and CoA-dependent reaction ; in aerobic conditions , it acts as a hydrogen peroxide scavenger ; induced under anaerobic conditions in the absence of nitrate Catalyzes the reversible transfer of the terminal phosphate group between ATP and AMP ; essential for maintenance and cell-growth ; increases during phosphate limitation Absolutely required for the growth of E. coli in a high-phosphate medium containing glucose-1-phosphate as the sole carbon source ; positively controlled by cAMP-CRP Directly reduces organic hydroperoxides in its reduced dithiol form ; may be directly degraded by ClpXP or modulated by a protease-dependent mechanism ; induced by oxidative stress but repressed by sulfate or cysteine ; increases during high-cell-density cultivation and acid or propionate condition during aerobic growth 6.33 / 95,996.05 Aldehyde-alcohol dehydrogenase P0A9Q7 AdhE 179 Adk 5.49 / 28,491 5.60 / 22,470 5.55 / 23,586.02 Adenylate kinase P69441 179 , 228 , 287 , 293 5.00 / 9,929 ( 4 -- 5 ) 4.99 / 9,991 ( 4.5 -- 5.5 ) 5.48 / 31,661 ( 5 -- 6 ) 5.38 / 43,560.36 Glucose-1-phosphatase P19926 Agp 79 1 AhpC Alkyl hydroperoxide reductase subunit C P0AE08 5.01 / 21,554 5.03 / 20,630.25 27 , 179 , 228 , 266 , 287 , 294 , 308 , 321 , 325 5.09 / 23,123 ( DIGE 4.5 -- 6.5 ) 5.08 / 21,705 ( 4.5 -- 5.5 ) 5.03 / 22,439 ( 4.5 -- 5.5 ) AhpF Serves to protect the cell against DNA damage by alkyl hydroperoxides ; can use either NADH or NADPH as electron donor for direct reduction of redox dyes or of alkyl hydroperoxides when combined with the AhpC protein ; induced by oxidative stress and after benzoic acid treatment 5.44 / 53,485 ( DIGE 4.5 -- 6.5 ) 5.20 / 41,503 ( 5 -- 6 ) 5.47 / 56,177.11 Alkyl hydroperoxide reductase subunit F P35340 179 , 203 , 266 , 287 , 321 Alanyl -- tRNA synthetase Aldehyde dehydrogenase A AlaS 29 , 179 , 228 , 294 5.53 / 96,032.40 5.50 / 96,123 P00957 AldA Acts on lactaldehyde as well as other aldehydes ; induced by growth on fucose , rhamnose , arabinose , and amino acids such as glutamate ; increases in the physiological shortterm adaptation to glucose limitation 5.11 / 52,363 ( DIGE 4.5 -- 6.5 ) 5.29 / 42,547 ( 4.5 -- 5.5 ) 5.53 / 38,913 ( 4.5 -- 5.5 ) 5.43 / 50,034 ( 5 -- 6 ) 5.07 / 52,141.60 P25553 179 , 287 , 308 , 311 , 321 , 325 Continued on following pag Involved in regulation of AMP concentrations ; allosterically activated by Mg-ATP but inactivated by inorganic phosphate ; induced by cAMP at limiting phosphate concentrations Serine beta-lactamase with substrate speciﬁcity for cephalosporins AnsA converts L-asparagine into L-aspartate ; there are two L-asparaginase isoenzymes : L-asparaginase I ( AnsA ) , a low-afﬁnity enzyme located in the cytoplasm , and L-asparaginase II ( AnsB ) , a high-afﬁnity secreted enzyme See AnsA ; induced by cAMP and anaerobiosis It is induced during entry into the stationary phase ; its synthesis is triggered by carbon starvation , phosphate starvation , osmotic shift , or a shift from aerobic to anaerobic conditions ; controlled by S and AppY Catalyzes a purine-salvage reaction , resulting in the formation of AMP that is energically less costly than de novo synthesis Involved in L-arabinose-catabolism 293 Amn P0AE12 AMP nucleosidase 5.90 / 53,994.91 6.38 / 47,693 87 Beta-lactamase AmpC P00811 8.78 / 39,551.14 9.06 / 43,647 ( 6 -- 11 ) 2 179 AnsA P0A962 L-Asparaginase I 5.52 / 37,127.24 AnsB P00805 5.66 / 34,593.94 L-Asparaginase II 179 6.11 / 44,689.86 5.42 / 46,694 Periplasmic AppA protein AppA P07102 13 , 228 , 293 Apt 5.30 / 23,792 ( D 5.26 / 19,858.88 Adenine P69503 IG E 4.5 -- 6.5 ) 179 , 321 phosphoribosyltransferase 5.73 / 25,518.91 6.06 / 29,165 L-Ribulose-5-phosphate 4-epimerase L-Arabinose-binding periplasmic protein AraD P08203 294 Involved in high-afﬁnity L-arabinose membrane transport system ; binds with high afﬁnity to arabinose but can also bind D-galactose ( approx 2-fold reduction ) and D-fucose ( approx 40-fold reduction ) ArcA/ArcB two-component regulatory system represses a wide variety of aerobic enzymes under anaerobic conditions , controls the resistance of E. coli to dyes , and may be involved in the osmoregulation of envelope proteins ; when activated by arcB , it negatively regulates the expression of genes of aerobic function ; it activates the transcription of the plfB operon by binding to its promoter Involved in both the arginine and lysine biosynthetic pathways 5.61 / 33,210.05 AraF P02924 179 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 5.21 / 27,292.02 P0A9Q1 Aerobic respiration control protein ArcA ArcA 125 , 321 Acetylornithine / ArgD P18335 287 , 321 5.75 / 41,241 ( DIGE 4.5 -- 6.5 ) 5.19 / 33,559 ( 5 -- 6 ) 5.80 / 43,635.78 succinyldiaminopimelate aminotransferase Ornithine carbamoyltransferase chain F Ornithine carbamoyltransferase consists of ArgF and ArgI ; involved in ﬁrst step of arginine biosynthetic pathway Involved in seventh step of arginine biosynthetic pathway 5.73 / 30,486 ( 5 -- 6 ) 5.86 / 29,835 ( 5.5 -- 6.7 ) 5.63 / 36,695.95 P06960 ArgF 287 ArgG 179 , 287 , 321 5.27 / 46,667 ( DIGE 4.5 -- 6.5 ) 5.28 / 46,209 ( 4.5 -- 5.5 ) 5.08 / 51,667 ( 5 -- 6 ) 5.43 / 39,776 5.23 / 49,767.20 P0A6E4 Argininosuccinate synthase See ArgF ; repressed during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases its level with increasing growth rate Involved in an arginine-transport system ; increases in the physiological short-term adaptation to glucose limitation ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism P0A6E4 Argininosuccinate synthase 5.46 / 36,775.92 Ornithine carbamoyltransferase chain I Arginyl-tRNA synthetase 228 , 293 P04391 ArgI P11875 ArgS 29 , 109 , 228 , 230 , 294 5.32 / 60,405 5.32 / 64,682.96 ArgT 5.15 / 26,055 ( DIGE 4.5 -- 6.5 ) 5.17 / 26,881 ( 4.5 -- 5.5 ) 5.22 / 25,784.99 Lysine-arginine-ornithine-binding periplasmic protein P09551 179 , 287 , 308 , 311 , 321 Continued on facing pag Involved in sixth step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP See AroA : seventh ( ﬁnal ) step See AroA : third step Stereospeciﬁc condensation of PEP and D-erythrose-4-phosphate , giving rise to 3-deoxy-D-arabino - heptulosonate-7-phosphate ( DAHP ) ; involved in ﬁrst step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP See AroF : ﬁrst step AroA P0A6D3 3-Phosphoshikimate 1-carboxyvinyltransferase 5.37 / 46,095.78 5.34 / 53,476 ( 5 -- 6 ) 5.23 / 50,990 ( 5 -- 6 ) Chorismate synthase P12008 P05194 P00888 AroC AroD AroF 179 5.82 / 39,006.25 5.19 / 27,466.65 5.42 / 38,803.96 3-Dehydroquinate dehydratase Phospho-2-dehydro-3-deoxyheptonate aldolase , Tyr sensitive 179 , 287 321 5.31 / 27,332 ( 4.5 -- 5.5 ) 5.38 / 37,715 ( DIGE 4.5 -- 6.5 ) AroG 179 , 286 Phospho-2-dehydro-3 - P0AB91 6.14 / 38,009.52 6.12 / 39,462 6.13 / 38,005 deoxyheptonate aldolase , Phe sensitive Shikimate kinase I 179 , 274 , 286 , 287 , 321 Involved in ﬁfth step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP ; induced by high pH Involved in the arginine periplasmic transport system ( artPIQMJ gene products ) ; increases after benzoic acid treatment See ArtI ; decreases after benzoic acid treatment See ArtI ; probably responsible for energy coupling to the transport system Involved in amino-acid-biosynthetic pathway : Lys , Met , and Thr Involved in asparagine-biosynthesis ; decreases after benzoic acid treatment Involved in asparagine-biosynthesis AroK 5.26 / 19,406.85 P0A6D7 5.30 / 17,959 5.30 / 19,078 ( DIGE 4.5 -- 6.5 ) 5.18 / 21,250 ( 5 -- 6 ) ArtI 5.32 / 25,042.20 Arginine-binding periplasmic protein 1 P30859 179 , 286 , 287 , 321 5.24 / 26,625 5.27 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.07 / 31,478 ( 5 -- 6 ) 5.94 / 25,009 ( DIGE 4.5 -- 6.5 ) 5.59 / 24,831 ( DIGE 4.5 -- 6.5 ) ArtJ 5.97 / 24,908.11 Arginine-binding periplasmic protein 2 Arginine transport ATP-binding protein ArtP P30860 321 6.17 / 27,022.05 P0AAF6 ArtP 179 Asd Aspartate-semialdehyde dehydrogenase 5.37 / 40,017.88 5.20 / 50,339 ( 5 -- 6 ) 179 , 287 P0A9Q9 AsnA 5.45 / 36,650.55 Aspartate-ammonia ligase P00963 179 , 321 5.45 / 37,876 ( DIGE 4.5 -- 6.5 ) 5.38 / 37,715 ( DIGE 4.5 -- 6.5 ) 5.55 / 62,527.82 AsnB P22106 Asparagine synthetase B ( glutamine hydrolyzing ) P0A8M0 Asparaginyl-tRNA synthetase 179 5.64 / 92,858 5.17 / 52,439.25 AsnS 228 , 236 , 287 , 294 , 321 4.69 / 50,800 ( DIGE 4.5 -- 6.5 ) 5.01 / 38,992 ( 4.5 -- 5.5 ) 5.19 / 52,356.13 5.54 / 43,573.36 Aspartate ammonia-lyase Aspartate aminotransferase P0AC38 P00509 AspA AspC 179 5.40 / 41,960 5.52 / 41,877 179 , 228 , 236 , 287 , 294 5.53 / 50,441 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.42 / 61,614 Changes very little at normal temp range ( 23 -- 37 ° C ) Involved in short-chain-fatty-acid-metabolism ; induced during phosphate limitation and at phosphonate growth F-type ATPases consist of the two complex components CF ( 0 ) , the membrane proton channel , and CF ( 1 ) , the catalytic core ; CF ( 1 ) has ﬁve subunits : alpha ( 3 ) , beta ( 3 ) , gamma ( 1 ) ; delta ( 1 ) , and epsilon ( 1 ) ; CF ( 0 ) has three main subunits : a , b , and c ; produces ATP from ADP in the presence of a proton gradient across the membrane ; increases in the physiological short-term adaptation to glucose limitation See AtpA 5.53 / 50,441 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.42 / 61,614 5.47 / 65,913.45 29 , 109 , 228 , 294 Aspartyl-tRNA synthetase P21889 AspS Acetate CoA-transferase beta subunit P76459 AtoA 5.65 / 22,959.65 5.41 / 22,852 293 AtpA 5.98 / 43,053 5.84 / 53,108 5.80 / 55,222.08 P0ABB0 ATP synthase alpha chain 179 , 228 , 286 , 287 , 294 , 311 , 321 5.82 / 52,204 ( DIGE 4.5 -- 6.5 ) 5.15 / 39,861 ( DIGE 4.5 -- 6.5 ) 5.81 / 36,727 ( 5 -- 6 ) 6.11 / 57,637 ( 6 -- 11 ) 6.02 / 30,657 ( 6 -- 11 ) AtpC 179 , 228 , 287 , 294 5.46 / 14,937.07 5.48 / 14,811 5.34 / 10,379 ( 4.5 -- 5.5 ) 5.45 / 13,773 ( 5 -- 6 ) ATP synthase epsilon chain P0A6E6 Continued on following pag See AtpA ; decreases during phosphate limitation AtpD P0ABB4 ATP synthase beta chain 4.90 / 50,194.23 4.90 / 47,721 4.96 / 48,103 ( DIGE 4.5 -- 6.5 ) 4.99 / 48,103 ( DIGE 4.5 -- 6.5 ) 4.95 / 43,643 ( 4 -- 5 ) 4.95 / 47,782 ( 4 -- 5 ) 4.55 / 43,745 ( 4 -- 5 ) 5.00 / 45,494 ( 4.5 -- 5.5 ) 4.96 / 42,894 ( 4.5 -- 5.5 ) 5.53 / 19,289 ( DIGE 4.5 -- 6.5 ) 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) AtpF P0ABA0 ATP synthase B chain 5.99 / 17,263.96 See AtpA ; decreases after benzoic acid treatment See AtpA See AtpA 179 , 321 P0ABA6 ATP synthase gamma chain P0ABA4 ATP synthase delta chain P0AE52 Putative peroxiredoxin Bcp AtpG AtpH Bcp 8.84 / 31,577.42 4.94 / 19,332.22 5.03 / 17,633.94 179 179 5.02 / 15,872 4.92 / 16,060 179 , 228 , 286 , 294 , 321 5.10 / 17,526 ( DIGE 4.5 -- 6.5 ) 5.34 / 16,350 May perform functions in iron detoxiﬁcation and storage analogous to those of animal ferritins P0ABD3 Bacterioferritin Bfr 4.69 / 18,495.03 294 5.77 / 81,406.5 5.32 / 38,648.09 Periplasmic beta-glucosidase Biotin synthase P33363 P12996 BglX BioB 179 179 Involved in ﬁnal step of biotin-biosynthesis : biotin from 6-carboxyhexanoyl-CoA See BioB : third step Involved in the active translocation of vitamin B12 ( cyanocobalamin ) across the outer membrane to the periplasmic space ; derives its energy for transport by interacting with the transperiplasmic membrane protein TonB May be an auxiliary component of the transport system Involved in amino acid biosynthesis , L-arginine-biosynthesis , and synthesis of carbamoyl phosphate from HCO3 ; decreases during phosphate limitation early See CarA ; decreases signiﬁcantly during phosphate limitation Scavenges exogenous and endogenous cytidine and 2-deoxycytidine for UMP-synthesis ; increases during high-cell-density cultivation and overexpression of recombinant protein in large scale Involved in the transmission of sensory signals from the chemoreceptors to the ﬂagellar motors ; autophosphorylated ; can transfer its phosphate group to either CheB or CheY ; a histidine kinase See CheA ; the active CheY ( phosphorylated or acetylated form ) exhibits enhanced binding to a switch component , FliM , at the ﬂagellar motor , which induces a change from counterclockwise to clockwise ﬂagellar rotation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Genes transcribed more efﬁciently in the presence of EsS Genes and Gene product b promoter regions a Transcription levels ( EsS/Es70 ratio ) d Stress response yieF sE ( 59 ) ; sS ( 19 ) sS , RcsBA ( 60 ) sS , RcsBA ( 60 ) Chromate reductase b3713 N.A. 1.50 1.51 osmB Intergenic region including osmB promoter Osmolarity-inducible lipoprotein b1283 b2688 1.51 2.21 gshA g-glutamate -- cysteine ligase ( glutathione-biosynthesis ) ahpF Alkyl-peroxidase reductase pphA Protein phosphatase macA Macrolide resistance efﬂux pump DNA and RNA-metabolism and modiﬁcation ppk Polyphosphate kinase , part of RNA degradosome , Intergenic region including xapA promoter xapA Xanthosine phosphorylase , nucleotide-synthesis/degradation gyrB B subunit of DNA gyrase recT Rac prophage , recombinase , in recET-lar-ydaCQ operon lar Rac prophage , restriction alleviation and modiﬁcation enhancement , in recET-lar-ydaCQ operon ydaC Rac prophage , in recET-lar-ydaCQ operon ydaQ Rac prophage , possible recombinase , in recET-lar-ydaCQ operon Polyamine-metabolism puuB g-glutamyl -- putrescine oxidase gabD Succinate semialdehyde dehydrogenase speB Agmatinase ( in arginine/putrescine-degradation pathway ) Transcription regulation Intergenic region including appY promoter appY Regulator of hyaABCDEF operon yciT Putative deoR-type transcription regulator ilvY Regulator of ilvC operon rhaS Regulator of rhamnose-transport Sugar metabolism treF Trehalase OxyR ( 61 ) sH ( 62,63 ) b0606 b1838 b0878 1.52 1.54 1.54 Regulator of intracellular sS concentrations ( 3 ) XapR ( 64 ) XapR ( 64 ) b2501 1.50 N.A. b2407 1.62 1.56 b3699 b1349 1.65 1.93 Fis ( 65 ) sS-Dependent in bioﬁlm cells ( 18 ) - growing b1348 1.93 b1347 b1346 1.74 1.66 as recT as recT sS ( 19 ) b1301 b2661 b2937 1.61 1.65 1.66 sS , Nac ( 9,66 ) 1.69 1.51 1.56 1.75 1.81 N.A. b0564 b1284 b3773 b3905 H-NS ( 67 ) H-NS ( 67 ) CRP ( 29 ) ; RhaS ( 68 ) sS ( 9,19 ) sS ( 19 ) sS ( 7 ) 1.53 1.55 1.56 1.57 b3519 b4230 b1897 b1901 ytfT Putative galactose ABC transporter otsB Trehalose-6-phosphate phosphatise araF Arabinose transporter , component of an ABC transport system glgP Glycogen phosphorylase , part of glgCAP operon gpmM Putative 2,3-bisphosphoglycerate mutase glvC Sugar transport phosphotransferase Other metabolic functions Intergenic region including hyaABCDEF promoter hyaA Hydrogenase small subunit ; in hyaABCDEF operon ( 69 ) ; sS ( 19 ) CRP , AraC CRP ( 70 ) ; sS controls glgCAP operon ( 19 ) 1.60 b3428 1.61 1.79 b3612 b3683 2.02 N.A. sS , AppY , anaerobic regulation by ArcA and NarP/NarL ( 20,71,72 ) as hyaA 1.68 b0972 Hydrogenase large subunit ; in hyaABCDEF operon Hydrogenase subunit ( nickel-binding protein ) ; in hyaABCDEF operon SecY-interacting protein Acetyl-muramic acid permease Transcriptional repressor of murQP Trans-aconitate methylatransferase Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon hyaB 1.66 b0973 as hyaA hyaF 1.50 b0977 syd 1.59 1.59 1.62 1.60 b2793 b2429 b2427 b1519 murP murR tam CRP , MurR ( 73 ) sS ( 74 ) ydhY FNR , NarL ( 75 ) 1.80 b1674 FNR , NarL ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>u000 .00 opeono transcriptional regulator 27.8 % PS00 1 : HTH-ARAC-FAMILY [ Bacterial regulat0ory proteins , AraC family signature ] I00 mtcoh SimlarIto14eoeralo regulaors i0 AraC fomily contr ( oligouter-rnembrane protein cxpression : ENVY ECOLI ( SW ) 253 Escheric4hiacoliEnvY , poron heroreogulatboryprotein 40.1 % STYPEFABCD -.13 ( GB ) 259 Salmoneolla typhimuroum ORF ( produton90W kb virulence ploasmid 40.4 % APPYOECOLI ( SW ) 243 0146eri46 ( 0 41o1 ) AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8041620" target="_blank">8041620</a>]<br></li>
<li>In sum , a combination of ArcA and AppY antirepression of IscR function , lower levels of IscR , and weak repression by [ 2Fe-2S ] - IscR leads to increased PhyaA expression under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>hyaA In this study , we examined the roles of IscR , ArcA , and AppY in regulating the expression of the hyaA promoter under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA , AppY , and IscR act combinatorially to control hyaA promoter expression levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Repressor of its own synthesis Similar to E. coli AraC Similar to E. coli AraC Unknown Unknown Positive regulator of caf1MA and caf1 operons for the production and transport of the capsule antigen F1 Repressor of the celABCF operon involved in the degradation of cellobiose , arbutin , and salicin Transcriptional activator of the cfaABCE operon for the production of CFA/I ﬁmbriae in enterotoxigenic E. coli strains Transcriptional activator of the operon involved in the production of CS5 ﬁmbriae in enterotoxigenic E. coli strains Transcriptional temperature-dependent activator of several E. coli envelope proteins , most notably the porins OmpF and OmpC and the l receptor , LamB Transcriptional activator of the exsCBA operon and exsD , exoS , and ORF1 genes required for the synthesis and secretion of exoenzyme S Transcriptional activator of the 987P operon for ﬁmbrial proteins in enterotoxigenic E. coli strains Transcriptional activator of hpaBC operon for catabolism of p-hydroxyphenylacetic acid SP : P43463 135 Ada Escherichia coli 57 , 138 , 150 , 180 SP : P06134 354 Ada Ada Mycobacterium tuberculosis Salmonella typhimurium Bacillus subtilis SP : Q10630 SP : P26189 SP : P19219 AdaA AdiY 253 Escherichia coli SP : P33234 32 , 237 TE : Q07681 AfrR 272 258 Escherichia coli AggR 265 183 Escherichia coli SP : P43464 AppY 43 Escherichia coli SP : P05052 12 , 131 2 AraC 281 30 Citrobacter freundii SP : P11765 AraC 292 Escherichia coli SP : P03021 174 , 238 , 248 , 266 301 SP : P07642 SP : P03022 SP : Q03320 SP : P35319 SP : P26950 AraC AraC AraL AraL CafR Erwinia chrysanthemi Salmonella typhimurium Streptomyces antibioticus Streptomyces lividans Yersinia pestis 149 46 265 43 85 , 128 199 280 CelD Escherichia coli SP : P17410 265 222 CfaD Escherichia coli SP : P25393 55 301 CsvR Escherichia coli SP : P43460 159 53 EnvY Escherichia coli 2 SP : P10805 77 298 ExsA Pseudomonas aeruginosa SP : P26993 133 260 FapR Escherichia coli SP : P23774 TE : Q46985 204 295 HpaA Escherichia coli TABLE 1 -- Continued Function Protein HrpB Accession no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>AppY and sS AppY , a transcriptional regulator , causes sS stabilization when overexpressed , and the effect is seen even in cells deleted for iraP , iraD and iraM ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>In addition , we identiﬁed a transcriptional regulator , AppY , that is likely to regulate synthesis of yet another potential anti-adaptor ; overproduction of AppY stabilizes sS independently of IraP , IraD and IraM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>We report the isolation and characterization of two new regulators of sS turnover encoded by the previously uncharacterized yjiD and ycgW genes , and the identiﬁcation of a role for the transcriptional regulator AppY , possibly by regulating an unidentiﬁed anti-adaptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Therefore , it seems likely that AppY regulates an unidentiﬁed regulator ( possibly an anti-adaptor ) for sS turnover .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>The possibility that AppY is a third global regulator of energy metabolism genes is discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>Other regulators ( AppY , Fis , H-NS , StpA , and IHF ) The AppY protein Ž acid poly phosphatase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>Other identiﬁed genes include those encoding DNA helicases ( UvrD , RecG , RecQ ) , truncated forms of the DNA mismatch-repair protein ( MutS ) and a primosomal component ( DnaT ) , a negative modulator of initiation of replication/GATC-binding protein ( SeqA ) , a stationary phase regulator AppY , a transcriptional regulator PaaX and three putative open reading frames , ycgW , yfjY and yjiD , encoding hypothetical proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225322" target="_blank">15225322</a>]<br></li>
<li>It would be interesting to determine if it is involved in the regulation of other genes in a fashion similar to AppY , which can affect the synthesis of more than 30 cellular proteins ( Atlung e t al. , 1989 ; Atlung & Brsndsted , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Some regulators are involved in the response-to-stressors , e.g. , response to alkylating agents ( Ada from E. coli , S. typhi-murium , and Mycobacterium tuberculosis and AdaA from Ba-cillus subtilis ) ( 57 , 100 , 176 , 177 ) ; response-to-oxidative-stress ( SoxS from E. coli and S. typhimurium ) ( 4 , 260 ) ; tolerance to antibiotics , organic solvents , and heavy metals ( AarP from Providencia stuartii , MarA and Rob from E. coli , PqrA from Proteus vulgaris , and RamA from Klebsiella pneumoniae ) ( 90 , 121 , 161 , 229 , 240 ) ; and transition from exponential growth t the stationary phase ( AppY from E. coli ) ( 12 , 131 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Although the physiological function of Hyd1 has not yet been deﬁned , the response of hya expression to carbon and phosphate starvation ( 2 ) and dependence on RpoS and AppY ( 2 , 9 ) suggest that it is involved in response-to-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>The response of hya expression to pHe was tested in the absence of AppY to determine whether AppY was involved in the response of hya to pHe ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>The AppY amino acid sequence was therefore searched for sequence similarity to a number of other proteins involved in transcriptional-control of gene-expression by using the diagon program ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>Recently , Bronsted & Atlung ( 1994 ) initiated an analysis of hya expression using a Φ ( hya -- lacZ ) gene fusion and determined that anaerobic induction is dependent on the global transcriptional regulator ArcA ( Lynch & Lin , 1996a ) and a new anaerobic regulator termed AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>These proteins include : CfaD , a regulator of fimbria production ( Savelkoul e t al. , 1990 ) ; Rns , a regulator of colonization factor antigens and adhesin production of enterotoxigenic E. coli ( Caron e t al. , 1989 ) ; AggR , a regulator of fimbria production by enteroaggregative E. culi ( Nataro e t al. , 1994 ) ; FapR , a regulator of the 987P operon in enterotoxigenic E. coli ( Klaasen & de Graff , 1990 ) ; AppY , a regulator of growth-phase-dependent expression of acid phosphatase ( Atlung e t al. , 1989 ) ; and UreR , a regulator of urease expression present in many urinary tract pathogens ( D'Orazio & Collins , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>ArcA and AppY Antagonize IscR Repression of Hydrogenase-1 Expression under Anaerobic Conditions , Revealing a Novel Mode O2 Regulation of Gene Expression in Escherichia coli of A. D. Nesbit , * A. S. Fleischhacker , S. J. Teter , and P. J. Kiley Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , USA Transcription of the Escherichia coli hydrogenase-1 operon ( hyaABCDEF ) is increased by the transcription factors ArcA and AppY under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In the accompanying paper ( 7 ) , we show that the AppY protein is also involved in anaerobic induction of transcription of the hya operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>These results indicate that both AppY and ArcA are involved in activating the transcription of the hya operon during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>A model incorporating the anaerobic regulation of the appY gene and the two operons which are controlled by AppY , the hydrogenase 1 ( hya ) operon and the acid phosphatase ( cyx-appA ) operon , is presented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Two operons , the hya operon and the cyx-appA operon , encoding enzymes involved in the anaerobic energy metabolism have been identiﬁed as targets for regulation by AppY ( 1 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Subsequently , we showed that the appY gene product , AppY , is required for the anaerobic induction of two operons involved in energy metabolism , the hya operon encoding hydrogenase 1 and the cyx-appA operon encoding a putative third cytochrome x oxidase and AppA ( 1 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>AppY activates transcription of the hydrogenase 1 operon and the acid phosphatase operon during anaerobic growth , and we have suggested that AppY is a third global transcriptional regulator involved in anaerobic gene regulation ( 1 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Recently , we identified a third anaerobic transcriptional activator , AppY , which is required for induction of the acid phosphatase ( cyx-appA ) operon during anaerobic growth ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_31">
The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></label>
<input id="rp_similarity_31", type="checkbox">
<div class="RP_similar">
<li>Atlung , T. , Sund , S. , Olesen , K. , and Brønsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>References Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>JOURNAL OF BACTERIOLOGY , Apr. 1997 , p. 2141 -- / 97 / $ 04.0010 Copyright q 1997 , American Society for Microbiology Effects of sS and the Transcriptional Activator AppY on Induction o the Escherichia coli hya and cbdAB-appA Operons in Response to f Carbon and Phosphate Starvation TOVE ATLUNG ,1 * KASPER KNUDSEN ,1 LOTTE HEERFORDT ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , 2800 Lyngby ,2 Denmark Received 21 October 1996/Accepted 29 January 1997 The transcriptional regulation of two energy metabolism operons , hya and cbdAB-appA , has been investigated during carbon and phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1996 , p. 3418 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology e Histone-Like Protein H-NS Acts as a Transcriptional Repressor for Expression of the Anaerobic and Growth Phase Th Activator AppY of Escherichia coli TOVE ATLUNG ,1 * SUSANNE SUND ,2 KIRSTEN OLESEN ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 12 October 1995/Accepted 9 April 1996 The transcriptional activator AppY is required for anaerobic and stationary-phase induction of the cyx-appA and hya operons of Escherichia coli , and expression of the appY gene itself is induced by these environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Atlung , T. , Knudsen , K. , Heerfordt , L. and Brondsted , L. ( 1997 ) Effects of sigmaS and the transcriptional activator AppY on induction of the Escherichia coli hya and cbdAB-appA operons in response to carbon and phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_39">
Effects of sS and the transcriptional activator AppY on induction of the Escherichia coli and cbdAB-appA operons in response to carbon and phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></label>
<input id="rp_similarity_39", type="checkbox">
<div class="RP_similar">
<li>Effects of σs and the transcriptional activator AppY on induction of the Escherichia coli hya and cbd -- appA operons in response to carbon and phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Effects of sS and the transcriptional activator AppY on induction of the Escherichia coli hya and cbdAB-appA operons in response to carbon and phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Effects of S and the transcriptional activator AppY on induction of the Escherichia coli hya and cbdAB-appA operons in response to carbon and phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11807064" target="_blank">11807064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057949" target="_blank">12057949</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_43">
Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobi-osis , phosphate starvation and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222606" target="_blank">11222606</a>]<br></label>
<input id="rp_similarity_43", type="checkbox">
<div class="RP_similar">
<li>Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobi-osis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11807064" target="_blank">11807064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>References Atlung T , Bronsted L ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Atlung , T. , and Bronsted , L. ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung , T. , and Brendsted , L. ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Atlung T , Brøndsted L ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx-appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the y x a p p A operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the cpx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the cyx appA operon of Esche-richia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li></div>
<li>The ﬁnal gene product , SrgC ( ORF11 ) , is homologous to several regulatory proteins of the AraC family including EnvY , AppY , and Rns ( from E. coli ) and VirF ( from Y. enterocolitica ) ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495757" target="_blank">9495757</a>]<br></li>
<li>Among these , the AppY protein shows a high degree of homology to the CfaD protein of E. coli involved in expression of adhesins CS1 and CS2 and the VirF protein of Shigella ﬂexneri involved in the expression of virulence genes ( 16 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">AppY binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Plasmids pQE-IraD , pQE-IraM and pQE-AppY were constructed by amplifying the coding sequences of iraD , iraM and appY by PCR using the primers EcoRI-rbs-yjiD F ( 5 ′ - atcgGAATTCATTAAAGAGGAGAAATTAACTatgatgcgac aatcacttcaggctg-3 ′ ) ; PstI-yjiD R ( 5 ′ - atcgCTGCAGttagctgac attctccagcgtcgcac-3 ′ ) for iraD , EcoRI-rbs-ycgW F ( 5 ′ - atcgGAATTCATTAAAGAGGAGAAATTAACTatgaagtggatag taattgacacggtaat-3 ′ ) and PstI-ycgW R ( 5 ′ - atcgCTGCAGttac ttgagcccatatgggcat-3 ′ ) for iraM ; EcoRI-rbs-appY F ( 5 ′ - atcgGAATTCATTAAAGAGGAGAAATTAACTatggattatgtttgc tccgtagttttcat-3 ′ ) and PstI-appY R ( 5 ′ - atcgCTGCAGTC AGTCAATTGTTTTGTTTATTCCATC-3 ′ ) for appY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Our genetic analysis of the cyx promoter region indicates that the presence of sequences upstream of the -10 region are required for activation of the cyx promoter by AppY protein and anaerobiosis , suggesting that the AppY binding site is located in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by AppY</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Since the appC promoter is transcribed in vivo by RNA polymerase containing the RpoS sigma factor and is induced by AppY [ 32,33 ] , one possibility is that AppY and/or RpoS sigma are required for the efficient binding of YdeO to the appC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>ArcA and AppY had no effect on PhyaA expression under aerobic conditions , as expected ( Fig. 1C ) , since ArcA is not active aerobically and since AppY is poorly expressed under these growth conditions ( 1 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>More recently , another transcription factor , IscR , has been shown to directly repress the hyaA promoter ( PhyaA ) but only under aerobic conditions when ArcA and AppY are not active ( 14 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Consistent with the notion that PhyaA expression is regulated by IscR repression aerobically and by ArcA and AppY antirepression of IscR anaerobically , PhyaA-lacZ expression levels in the absence of any transcription factors were similar under anaerobic and aerobic conditions ( Fig. 1A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA , AppY , and IscR act combinatorially to control hyaA promoter expression levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Transcription of hya is induced by anaerobiosis and repressed by nitrate ( 9 ) and requires the sigma factor RpoS ( 2 ) and the transcriptional regulators ArcA and AppY ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>This operon is induced under an-aerobic conditions in the presence of formate or fumarate , repressed in the presence of nitrate , and requires acidic pH , ArcA , and AppY for optimal expression ( 12 , 21 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>We found that the anaerobic expression of adhE is independent of AppY but is inﬂuenced by DNA topology ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></li>
<li>Since transcription of the appY gene is induced under anaerobic conditions ( 1 ) , this could be due to increased synthesis of AppY protein at the onset of anaerobiosis , and thus only as the concentration of AppY builds up does the cyx promoter become fully activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>Furthermore , the cyx promoter is induced by anaerobic growth , and we have recently shown that this induction is dependent on AppY but independent of Fnr ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>The expression of the appY gene is induced immediately by anaerobiosis , and this anaerobic induction is independent of Fnr , and AppY , but dependent on ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>We found that the transcription of the hya operon was induced when the growth condition was changed from aerobic to anaerobic and that this induction was independent of Fnr but dependent on regulators AppY and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>We found that transcription was induced by a shift from aerobic to anaerobic conditions and that this induction was regulated by ArcA and AppY but not Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>The AppY-depen-dent stimulation of the cbd promoter might be mediated by increased levels of AppY protein , since appY expression is also strongly induced by phosphate starvation ( 8 ) ( see Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>Both operons were induced by carbon starvation , but only induction of the hya operon was dependent on sS , whereas that of the cbd promoter was dependent on AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>The phosphate starvation induction of the promoter region ( cyx ) transcribing the entire operon is dependent on AppY ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Since the phosphate starvation induction of the hydrogenase 1 ( hya ) operon is also dependent on AppY ( 10 ) , we examined the effect of phosphate starvation on the expression of the appY gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Expression of the appY gene is itself induced by anaerobiosis , entry into stationary phase , and phosphate starvation ( 8 ) , indicating that the increased AppY concentration under these conditions contributes to the induction of the cbd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>Using the appY mutant constructed in this work , we found that AppY is essential for anaerobic induction of pcy Induction of the cyx promoter by entry into stationary phase and by phosphate starvation is severely reduced in the appY mutant , whereas induction of the appA promoter is unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>Since anaerobic induction of the hydrogenase 1 operon ( 9 ) and the acid phosphatase operon ( 1 ) ( Table 2 ) is dependent on both AppY and ArcA , we investigated whether anaerobic induction of the appY gene was regulated by ArcA or was auto-regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>The expression of the hydrogenase 1 operon ( 9 ) and the acid phosphatase operon ( Table 3 ) is affected by the spe-ciﬁc anaerobic growth conditions , and this could be mediated by AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Thus , the observed nitrate repression and formate induction of the hydrogenase 1 operon ( 9 ) and the acid phosphatase operon ( Table 3 ) can not be mediated by changes in the expression of AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>ArcA and AppY Antagonize IscR Repression of Hydrogenase-1 Expression under Anaerobic Conditions , Revealing a Novel Mode O2 Regulation of Gene Expression in Escherichia coli of A. D. Nesbit , * A. S. Fleischhacker , S. J. Teter , and P. J. Kiley Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , USA Transcription of the Escherichia coli hydrogenase-1 operon ( hyaABCDEF ) is increased by the transcription factors ArcA and AppY under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In contrast , for the hyaA and hybO promoters , IscR repressed the previously identiﬁed O2-regulated transcription start sites ; under anaerobic growth conditions , AppY and the ArcA/B two-component regulatory system have been implicated in the activation of transcription of the hya operon ( Brondsted and Atlung , 1994 ; Richard et al. , 1999 ) , whereas FNR appears to contribute to activation of the hybO promoter ( Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The anaerobic transcription of the hydrogenase 1 operon is activated by ArcA and AppY , which seem to act independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>These results indicate that both AppY and ArcA are involved in activating the transcription of the hya operon during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>Expression of the hya operon , like that of cbd , is stimulated by an increased AppY concentration ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>AppY regulates the appCBA operon ( 5 ) , which was induced by EvgA overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>The only other operon that has been identiﬁed as a target for the AppY activator is the hya operon , which is located immediately upstream of the cbdAB-appA operon ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></li>
<li>In the accompanying paper ( 7 ) , we show that AppY also regulates transcription of the hya operon , which encodes hydrogenase 1 and is located immediately upstream of the appA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>The effector for AppY might be formate since we found that both the hya operon ( 9 ) and the cyx-appA operon were induced by formate and that the formate inductions were independent of FhlA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>A model incorporating the anaerobic regulation of the appY gene and the two operons which are controlled by AppY , the hydrogenase 1 ( hya ) operon and the acid phosphatase ( cyx-appA ) operon , is presented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Furthermore , we tested the anaerobic expression of the hya-lacZ fusion in an appY mutant , since we recently have shown that AppY acts as a transcriptional activator of the cyx-appA operon during anaerobic growth ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>ArcA and AppY activated transcription of the cyx-appA operon during entry into stationary phase and under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Anaerobic regulation of the cyx-appA operon by ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>]<br></li>
<li>Other than the demonstration of its involvement in the regulation of expression of the hya and cbdAB -- appA operons , little information is available concerning the recognition sequence for AppY or its mode of action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>176 , No. 17 f R ole of the Transcriptional Activator AppY in Regulation o the cyx appA Qperon of Escherichia coli by Anaerobiosis , Phosphate Starvation , and Growth Phase TOVE ATLUNGl * AND LONE BR0NDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , The Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 6 December 1993/Accepted 13 June 1994 Transcriptional lacZ fusions have been used to analyze the regulation of the appA operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1996 , p. 3418 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology e Histone-Like Protein H-NS Acts as a Transcriptional Repressor for Expression of the Anaerobic and Growth Phase Th Activator AppY of Escherichia coli TOVE ATLUNG ,1 * SUSANNE SUND ,2 KIRSTEN OLESEN ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 12 October 1995/Accepted 9 April 1996 The transcriptional activator AppY is required for anaerobic and stationary-phase induction of the cyx-appA and hya operons of Escherichia coli , and expression of the appY gene itself is induced by these environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>In the present study , we have constructed transcriptional fusions of the appA operon promoters to the lacZ and phoA indicator genes and used these fusions to study regulation of the appA operon by environmental factors and by AppY protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>We are presently dissecting the relatively complex transcriptional unit which contains the appA gene , by isolation of the three promoters that contribute to its transcription , to study their individual regulation with respect to anaerobiosis , phosphate starvation , and activation by AppY protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_41">
Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobi-osis , phosphate starvation and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222606" target="_blank">11222606</a>]<br></label>
<input id="tu_similarity_41", type="checkbox">
<div class="TU_similar">
<li>Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobi-osis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11807064" target="_blank">11807064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>References Atlung T , Bronsted L ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Atlung , T. , and Bronsted , L. ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung , T. , and Brendsted , L. ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Atlung T , Brøndsted L ( 1994 ) Role of the transcriptional activator AppY in regulation of the cyx-appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the cpx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the cyx appA operon of Escherichia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464194" target="_blank">10464194</a>]<br></li>
<li>Role of the transcriptional activator AppY in regulation of the cyx appA operon of Esche-richia coli by anaerobiosis , phosphate starvation , and growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li></div>
				</div></ul></div><br>
	</div>
</body>
</html>