<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - AsnC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">AsnC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">AsnC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Structural insight into gene transcriptional regulation and effector binding by the Lrp/AsnC family Paul Thaw , Svetlana E. Sedelnikova , Tatyana Muranova , Sebastian Wiese , Sylvia Ayora , 1,2 Juan C. Alonso , Arie B. Brinkman , Jasper Akerboom , John van der Oost and 2 3 4 4 John B. Rafferty * Department of Molecular Biology and Biotechnology , Krebs Institute , University of Sheffield , Western Bank , Sheffield S10 2TN , UK , Departamento de Biologia Molecular , Universidad Autonoma de Madrid , Cantoblanco , 1 28049 Madrid , Spain , Departamento de Biotecnologia Microbiana , Centro Nacional de Biotecnologia , CSIC , Campus 2 Universidad Autonoma de Madrid , Cantoblanco , 28049 Madrid , Spain , Department of Molecular Biology , 3 NCMLS M850/3 .79 , Geert Grooteplein 30 , 6525 GA , Nijmegen , The Netherlands and Laboratory of Microbiology , 4 Wageningen University , Hesselink van Suchtelenweg 4 , 6307 CT Wageningen , The Netherlands Received November 21 , 2005 ; Revised and Accepted February 10 , 2006 ABSTRACT The Lrp/AsnC family of transcriptional-regulatory proteins is found in both archaea and bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>We identiﬁed a high proportion of TFs belonging to the AsnC , ArsR , MarR , TetR/AcrR , BirA , AraC/XylS , Fur , LysR , and PhoU families in bacteria and archaea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>For instance , proteins classiﬁed in the Fur and PhoU families regulate iron and phosphate metabolism , the AsnC and MarR families belong to the earliest repressor super-group ( Pérez - Rueda and Collado-Vides , 2001 ) , while the ArsR family contains the shortest repressor proteins described so far ( Pérez - Rueda and Collado-Vides , 2000 ; Sato and Kobayashi , 1998 ) , very close in length to the proposed hypothetical ancestor of TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Five changes were made to the AsnC structure ( Y82 to F90 ; Y100 to L108 ; T101 to V109 ; S106 to D114 and Q128 to L136 ) based on the sequence alignment ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The Lrp leucine response-mutations correspond to the following residues of AsnC : Y100 , S106 , I116 , Q128 , L139 , I140 and V141 ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The substitution of residues Y82 , Y100 and Q128 in AsnC by a phenylalanine and two leucines , respect-ively , in Lrp , changes the size and nature of the pocket such that it would be ideally suited to-binding the aliphatic side chain of leucine as opposed to the carboxyamide group of asparagine ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Cro , I CI , GylR , AsnC , Fnr , and TnpR ( Dodd and Egan , 1987 ; Brennan and Matthews , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1995431" target="_blank">1995431</a>]<br></li>
<li>Members of the family influence cellular metabolism in both a global ( Lrp ) and specific ( AsnC ) manner , often in response to exogenous amino acid effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>From left to right , increasing amounts of AsnC + extract ( CM1671 plus pLSK35-3 , in micrograms of protein ) were added as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>Galactokinase activity , expressed as counts per minute of [ 14C ] galactose phosphorylated , was measured after the transcription-translation system was incubated with different amounts of complementing extracts , i.e. , AsnC + extract ( CM1671 plus pLSK35-3 , 0 ) and AslC - extract ( CM1671 , 0 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>In this context , it is worth noticing that methionine binds to the Lrp-like regulator NMB0573 ( annotated as AsnC ) of Neisseria meningitidis and stabilizes the octameric form of the protein ( Ren et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The resultant binding pocket is smaller in size than that of AsnC , mainly due to the presence of R96 , although it still possesses a channel into the central cavity of the octamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Lane 1 , AsnC - extract ; lane 2 , AsnC ~ complemented with 1 pl AsnC + extract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Lane 1 , EcoRI + XhoI cut DNA , AsnC - ; lane 2 , EeoRI + XhoI cut DNA , AsnC - with 1 gl AsnC + extract ; lane 3 , supercoiled DNA , AsnC ; lane 4 , supercoiled DNA , AsnC - with 1 pl AsnC + extract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>However , a model of straight B-form DNA bound to a dimer of either AsnC or LrpC suggests that the interactions are not optimal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Thus it seems likely that AsnC exerts its effects by binding to specific DNA sequences within the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>Implications for DNA binding It is clear that the N-terminal HtH motif and in particular helix aC is responsible for the binding of DNA in both AsnC and LrpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Models of AsnC and LrpC binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Therefore , in agreement with earlier observations , this control must be entirely mediated by AsnC [ 12,13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11814655" target="_blank">11814655</a>]<br></li>
<li>Therefore we suggest that AsnC not only specifically binds to D N A but also to RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Our data support the notion that the genes from asnC to gidB , with oriC located in between , constitute a regulatory unit , since asnC and gidA are negatively controlled by AsnC protein , although at different levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>The latter gene codes for asparagine synthase , and is regulated by AsnC in an asparagine-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>AsnC has also been demonstrated to autoregulate its own expression in an asparagine-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_25">
Effect of asparagine on AsnC function in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></label>
<input id="act_similarity_25", type="checkbox">
<div class="ACT_similar">
<li>Effect of asparagine on AsnC function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li></div>
<li>For instance , if the cells are growing under N-excess conditions , asparagine will be synthesized by AsnA and AsnB ; if asparagine is introduced under this condition , AsnC will adjust the expression of asnA according to the asparagine requirement of the cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11814655" target="_blank">11814655</a>]<br></li>
<li>As noted before the expression of AsnA was stimulated and the expression of AsnC was repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>AsnA synthesis was stimulated , while the synthesis of AsnC was repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>In contrast to Lrp , AsnC has only been shown to exert speciﬁc control of its own gene and that of asnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The effector switch Like Lrp , AsnC binds DNA and regulates gene expression in the absence or presence of effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>When the upstream AsnC-regulated transcript is cut off , the expression of gidA is still AsnC regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Besides being regulated by AsnC , gidA expression may be subject to additional controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Its amino acid sequence is related to that of AsnC , a protein that regulates the expression of both asnC and asnA ( deWind et al. , 1985 ; Kolling and Lother , 1985 ; Willins et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Using the minichromosome pOC77 ( Fig. 1A ) as DNA template in the Zubay system we found that the AsnC protein acts as a positive regulator of asnA expression and that it regulates its own synthesis negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Lrp is thought to be evolutionarily related to AsnC , a regulatory protein that can positively stimulate the expression of one operon and negatively affect its own promoter ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>AsnC stimulates expression from the asnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>The changes in protein pattern induced by the addition of AsnC protein are depicted in Fig. 2A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>The synthesis of the 17-kilodalton protein , which we now call AsnC , is autogenously regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>Here we present evidence that this 17-KDa protein , which we now call AsnC , is an autogenously regulated activator of asnA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>The expression of the gidA gene which is located immediately counterclockwise of the replication origin of Escherichia eoli , oriC , was found to be negatively regulated by the AsnC protein in an in vitro transcription-translation system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>The asnC promoter is negatively regulated by AsnC protein while the rnioC promoter is not ( K611ing and Lother 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>This effect is not due to simple repression of transcription originating at the gidA promoter , because the AsnC protein did not change the level of gidA promoter dependent transcription as analysed by promoter-galK fusions and by $ 1 mapping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Mapping of the gidA promoter by deletion analys & and by S1 mapping To decide whether AsnC affects transcription or translation of gidA we localized the gidA promoter and analysed its transcript levels in the absence and in the presence of AsnC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Effect of the AsnC protein on transcription in the gidA region In order to test whether gidA promoter activity is affected by the AsnC protein we compared galK expression from gidA promoter-galK fusions in the Zubay system in the absence and in the presence of the AsnC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Thus there seems to be a step , sensitive to a conformational change in AsnC protein , which is required for asnA activation but not for asnC repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>The stimulating effect of AsnC on asnA transcription is abolished by asparagine , while the autoregulation of asnC is not affected by asparagine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>With increasing concentration of asparagine , the stimulating effect of AsnC on asnA transcription disappeared , while the repression at the asnC promoter was not affected ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">AsnC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>We tested the possibility that AsnC bears a specific protease function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Although E137 in AsnC becomes R145 in Lrp , there is a compensatory change in an interacting residue of an H for a D at a position equivalent to residue 104 of AsnC ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Crystals were found to belong to space group P4 and contain two copies of the AsnC monomer in the asymmetric unit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Asparagine binding in AsnC In the reﬁned structure of the AsnC octamer , clear additional density was observed in the cleft between the turn from strand b3 to b4 of one monomer and strand b5 of another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>AsnC discriminates between asparagine and aspartate and an analysis of the binding pocket suggests that unfavourable interactions would arise between the charged sidechain of the effector molecule and residue E128 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>DNA-binding region Mapping of residues identiﬁed in E.coli Lrp as being involved in DNA-binding [ residues D13 , L34 , L40 , S41 , P44 , L46 , R48 , Y61 , L65 and L70 , ( 50 ) ] onto the structures of AsnC and LrpC reveals two main clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The remainder of the pocket is largely unchanged with the AsnC TET motif ( residues 136 -- 138 ) mimicked by TRT ( residues 144 -- 146 ) in Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The substitution of residues Y82 , Y100 and Q128 in AsnC by a phenylalanine and two leucines , respect-ively , in Lrp , changes the size and nature of the pocket such that it would be ideally suited to binding the aliphatic side chain of leucine as opposed to the carboxyamide group of asparagine ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Using a structure-based sequence alignment , mapping of equivalent Lrp residues onto the AsnC structure results in one end of the binding pocket becoming predominantly hydrophobic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The AsnC crystal structure reveals that three of these residues are directly involved in effector-binding pocket formation ( Y100 , S106 and Q128 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>A study of the equivalent region of LrpA ﬁnds the binding pocket obstructed by residues M101 and P133 ( equivalent to S106 and T138 of AsnC ) and is consistent with its reported effector-independent activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Modelling of the AsnC octamer bound to a curved piece of DNA ( based on PDB code 1CGP ) indicates that the pairs of DNA-binding N-terminal domains of individual dimers must be reoriented relative to the effector-binding C-terminal domains to enable binding of an octamer simultaneously to multiple 13 bp sites with 18 bp spacings ( Figure 6a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Both and AsnC have helix-turn-helix ( HTH ) motif at predicted to ' a were their amino-terminal ends ( 7 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>AsnC has a region centered at amino acid 35 that is similar to regions of other proteins known to have helix-turn-helix ( HTH ) motifs ( 38 , 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The G37 to E37 mutation in our AsnC construct ( Materials and Methods ) lies on the surface at the N-a terminal end of helix C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The majority of the increase in diameter of AsnC and LrpC arises from the packing of the C-terminal residues into the space between elements of strands b3 and b4 and the helical turn around residues P61 to L64 ( numbering as for AsnC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>RESULTS Overall structure The monomer structure of both AsnC and LrpC comprises two domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The relative overall orientation of the N - and C-terminal domains is essentially the same in both AsnC and LrpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Implications for DNA-binding It is clear that the N-terminal HtH motif and in particular helix aC is responsible for the binding of DNA in both AsnC and LrpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The DNA-binding helices in dimers of both AsnC and LrpC ( helix aC ) adopt a different relative orientation to those of CAP and consequently the distance between the regions of helix aC that contact the DNA are slightly closer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The possible exceptions are R48 ( equivalent to AsnC R42 and LrpC R39 ) , which forms an ion pair network with two totally conserved aspartates in helix aA and L34 ( equivalent to AsnC L28 and LrpC L25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>With the exception of a triple network found in the N-terminal region of AsnC and LrpC that may contribute to stabilizing the relative orientation of helices aA and aC in the DNA reading heads , only FL11 contained any other ion pair networks with more than two partners ( FL11 contained two ion pair networks of four residues in the dimer ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">AsnC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Other common regulator families are ArsR ( 175 ) , AsnC ( 140 ) , Crp ( 233 ) , DeoR ( 17 ) , GntR ( 29 , 104 , 213 ) , IclR ( 214 ) , LacI ( 253 ) , LuxR/UphA ( 112 ) , LysR ( 111 ) , MarR ( 56 ) , MerR ( 107 ) , NtrC ( 189 ) , TetR ( 198 ) , YedF/YeeD/YhhP ( 14 ) , and YhdG/YjbN/YohI ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Members Group function Ec1 So2 Bs3 Group function LuxR/UhpA or OmpR family transcriptional response regulator of two-component regulatory system LysR family transcriptional regulator GntR or DeoR family transcriptional regulator sensory histidine kinase in two-component regulatory system sigma54 activator protein , enhancer binding protein AraC/XylS family transcriptional regulator ROK family transcriptional regulator/sugar kinase IclR family transcriptional regulator methyl-accepting chemotaxis protein MerR family transcriptional regulator DNA-binding regulatory protein AraC/XylS family transcriptional regulator MarR family transcriptional reguator AsnC family transcriptional regulator 46 20 18 18 13 13 11 10 9 18 14 13 13 11 16 13 8 1 oxidoreductase , Fe-S-binding 31 10 11 13 6 oxidoreductase , NAD ( P ) - binding oxidoreductase4 , NAD ( P ) - binding aldehyde dehydrogenase , NAD ( P ) - binding oxidoreductase , FAD/NAD ( P ) - binding sugar kinase respiratory reductase , alpha subunit aldo-keto reductase , NAD ( P ) - binding phosphatase nucleoside diphosphate ( Nudix ) hydrolase 43 30 22 5 14 14 7 8 8 7 5 8 8 2 7 5 1No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>We identiﬁed a high proportion of TFs belonging to the AsnC , ArsR , MarR , TetR/AcrR , BirA , AraC/XylS , Fur , LysR , and PhoU families in bacteria and archaea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>For instance , proteins classiﬁed in the Fur and PhoU families regulate iron and phosphate metabolism , the AsnC and MarR families belong to the earliest repressor super-group ( Pérez - Rueda and Collado-Vides , 2001 ) , while the ArsR family contains the shortest repressor proteins described so far ( Pérez - Rueda and Collado-Vides , 2000 ; Sato and Kobayashi , 1998 ) , very close in length to the proposed hypothetical ancestor of TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>The leucine regulator protein ( LRP ) , a member of the AsnC family identiﬁed in Pyrococcus furiosus , contains a wHTH towards the N-terminus , and a reminiscent RNA - and DNA-binding structure at its C-terminal domain ( Leonard et al. , 2001 ; Napoli et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>AsnC has a region centered at amino acid 35 that is similar to regions of other proteins known to have helix-turn-helix ( HTH ) motifs ( 38 , 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Members of the Lrp / AsnC family typically have a molecular mass of 15 kDa but populate a range of multimeric species in solution , which include dimers , tetramers , octamers and hexadecamers ( 4 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>As Lrp-regu - Introduction Leucine-responsive regulatory protein ( Lrp ) belongs to the widely distributed Lrp -- AsnC family of small , basic transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Bacillus subtilis LrpC was one of seven genes encoding proteins belonging to the Lrp/AsnC family identiﬁed in its genomic sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>INTRODUCTION Proteins belonging to the Lrp/AsnC family of global or speciﬁc transcriptional regulators are widely distributed in numerous prokaryotes , including bacteria and archaea ( 1,2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Members of the family influence cellular metabolism in both a global ( Lrp ) and specific ( AsnC ) manner , often in response to exogenous amino acid effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Sequence alignment of the Lrp/AsnC family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Members of the Lrp/AsnC family are present in both Gram-positive and Gram-negative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>It shares 34 and 25 % identity with E.coli Lrp and AsnC , respectively , and has reported functionality in both sporulation and amino acid metabolism ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Lrp has some amino acid sequence similarity to AsnC , a protein that regulates asnA expression ( Kolling , R. , and Lother , H. ( 1985 ) J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>The amino acid sequences of Lrp and AsnC are 25 % identical , and at least another 25 % show close similarity ( 168 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>E.coli AsnC shows notable sequence identity ( 25 % ) with Lrp ( Figure 1 ) , which has resulted in them being classiﬁed as a distinct evolutionary protein family ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The AsnC protein apparently belongs to this group of proteins , because it shows the same sequence homologies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by AsnC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Table 1 TFs identiﬁed in complete genomes Family DBD Role PFAM Annotations Total 1 16 Function AlpA C n/d -- Lipopolysaccharide adhesion to human gastric tissue Tatusova et al. ( 1999 ) ; Odenbreit et al. ( 2002 ) AraC/XylS A M 733 124 813 Carbon metabolism , cell-wall-synthesis , stress responses , and pathogenesis regulation Arginine biosynthesis regulation Tobes and Ramos ( 2002 ) R 56 ArgR ArsR W 14 60 Campoy et al. ( 2002 ) 335 R W 50 313 Pérez - Rueda and Collado-Vides ( 2000 ) ; Sato and Kobayashi ( 1998 ) ; Diorio et al. ( 1995 ) Arsenic resistance regulation -- AslB n/d AsnC D 87 -- 39 72 -- w 277 Leucine repressor protein Biosynthesis of biotin Napoli et al. ( 1999 ) ; Leonard et al. ( 2001 ) Wilson et al. ( 1992 ) -- -- 24 0 BirA R CaiF n/d w 75 6 Co-activator of the carnitine operon Low temperature adaptation Buchet et al. ( 1998 ) Obokata et al. ( 1991 ) 201 -- 198 8 Cold shock CSD A Crl -- R 0 0 Schnetz ( 2002 ) Global regulator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Mol Gen Genet ( 1988 ) 212 : 99-104 © Springer-Verlag 1988 AsnC , a multifunctional regulator of genes located around the replication origin of Escherichia coil , oriC Ralf K • lling , Annette Gielow , Wolfgang Seufert , Claudia Kiieherer , and Walter Messer Max-Planck-Institut fiir molekulare Genetik , Ihnestrasse 73 , D-1000 Berlin 33 Summary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>AsnC , a multifunctional regulator of genes located around the replication origin of Escherichia coli , oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22337945" target="_blank">22337945</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Metabolic integration could account for why nitrogen limitation represses both enzymes of glutamate-synthesis , the ﬁrst enzyme of serine synthesis , and AsnC ( the activator of asnA ) ( 12 , 35 , 64 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Members of the family influence cellular-metabolism in both a global ( Lrp ) and specific ( AsnC ) manner , often in response to exogenous amino acid effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Using the minichromosome pOC77 ( Fig. 1A ) as DNA template in the Zubay system we found that the AsnC protein acts as a positive regulator of asnA expression and that it regulates its own synthesis negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Lrp is thought to be evolutionarily related to AsnC , a regulatory protein that can positively stimulate the expression of one operon and negatively affect its own promoter ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>It is related to AsnC , an E. coli protein that regulates expression of asnA ( which encodes asparagine synthetase A ) ( 37 , 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Its amino acid sequence is evolutionarily related to that of AsnC , a regulatory protein that controls * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Lrp has some amino acid sequence similarity to AsnC , a protein that regulates asnA expression ( Kolling , R. , and Lother , H. ( 1985 ) J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Its amino acid sequence is related to that of AsnC , a protein that regulates the expression of both asnC and asnA ( deWind et al. , 1985 ; Kolling and Lother , 1985 ; Willins et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>The effector switch Like Lrp , AsnC binds DNA and regulates gene-expression in the absence or presence of effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>In contrast to Lrp , AsnC has only been shown to exert speciﬁc control of its own gene and that of asnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>From these data we conclude that the AsnC protein controls gidA gene-expression at a post-transcriptional level , gidA is the third gene in the oriC region , besides asnA and asnC , whose expression is under AsnC control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>AsnC has also been demonstrated to autoregulate its own expression in an asparagine-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>There are indications that AsnC affects another function besides asnA that is involved in asparagine-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">AsnC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>A study of promoter regions regulated by the Lrp/AsnC family revealed several inverted-repeat sequences of 5 bp with an intervening 3 bp , i.e. a 13 bp site ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>There is also a repeating pattern to the distribution of these 13 bp sites in many of the upstream regions of genes regulated by Lrp/AsnC family members such that either a shorter 7 or 8 bp spacing or a longer 18 bp spacing between sites is generally observed ( 55 ) Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Kolling and Lother ( 23 ) proposed a consensus sequence for AsnC binding as TT ( A/T ) TT ( T/G ) ( A/C ) ATG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Analysis of the central 3 bp in the DNA-binding site of AsnC shows a marked preference for TA base pairs , consistent with DNA bending ( 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Thus it seems likely that AsnC exerts its effects by binding to specific DNA sequences within the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by AsnC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Instead we find a cluster of functions around oriC that are regulated by AsnC , supporting the view that oriC is located within an asn operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>In contrast to Lrp , AsnC has only been shown to exert speciﬁc control of its own gene and that of asnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>For instance , if the cells are growing under N-excess conditions , asparagine will be synthesized by AsnA and AsnB ; if asparagine is introduced under this condition , AsnC will adjust the expression of asnA according to the asparagine requirement of the cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11814655" target="_blank">11814655</a>]<br></li>
<li>AsnC has also been demonstrated to autoregulate its own expression in an asparagine-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>As noted before the expression of AsnA was stimulated and the expression of AsnC was repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>The expression of the gidA gene which is located immediately counterclockwise of the replication origin of Escherichia eoli , oriC , was found to be negatively regulated by the AsnC protein in an in vitro transcription-translation system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>From these data we conclude that the AsnC protein controls gidA gene expression at a post-transcriptional level , gidA is the third gene in the oriC region , besides asnA and asnC , whose expression is under AsnC control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Lrp is thought to be evolutionarily related to AsnC , a regulatory protein that can positively stimulate the expression of one operon and negatively affect its own promoter ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Using the minichromosome pOC77 ( Fig. 1A ) as DNA template in the Zubay system we found that the AsnC protein acts as a positive regulator of asnA expression and that it regulates its own synthesis negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>Its amino acid sequence is related to that of AsnC , a protein that regulates the expression of both asnC and asnA ( deWind et al. , 1985 ; Kolling and Lother , 1985 ; Willins et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>It is related to AsnC , an E. coli protein that regulates expression of asnA ( which encodes asparagine synthetase A ) ( 37 , 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>, / The proposal that Nac only represses asnC derives from the fact that the expression of asnA was induced in N-limiting growth conditions when AsnC was expressed from the lac promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11814655" target="_blank">11814655</a>]<br></li>
<li>The stimulating effect of AsnC on asnA transcription is abolished by asparagine , while the autoregulation of asnC is not affected by asparagine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2864330" target="_blank">2864330</a>]<br></li>
<li>All these transcripts were repressed by AsnC protein ( Fig. 2A , tracks 4 and 5 ) and , therefore , must originate from the asnC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>The asnC promoter is negatively regulated by AsnC protein while the rnioC promoter is not ( K611ing and Lother 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836709" target="_blank">2836709</a>]<br></li>
<li>c is not very effective : ( i ) The low rate of transcription observed at the end of the asnC gene when assayed in high copy number vectors is due to repression of asnC transcription by AsnC protein ( K611ing and Lother 1985 ; de Wind et al. 1985 ; Schaefer and Messer 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>