<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - BasR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">BasR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">BasR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>After prolonged incubation , the level of phosphorylated RRs increased up to completion for pBasR , pHydG , pOmpR , pPhoB , and pUvrY ( group B1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The trans-phos-phorylation was experimentally demonstrated in vitro , for the first time , for 9 pairs , CusS-CusR , RstB-RstA , YedV-YedW , BaeS-BaeR , YehU-YehT , YfhK-YfhA , HydH-HydG , BasSBasR , and CreC-CreB ( see Fig. 3 ) , supporting the predicted HK-RR pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>A total of 30 sensor HKs , including as yet uncharacterized puta - tive HKs ( BaeS , BasS , CreC , CusS , HydH , RstB , YedV , and YfhK ) , and a total of 34 RRs , including putative RRs ( BaeR , BasR , CreB , CusR , HydG , RstA , YedW , YfhA , YgeK , and YhjB ) , have been suggested to exist in Esche - richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Trans-phosphorylation of presumed cog - nate RRs by HKs was detected , for the first time , for eight pairs , BaeS-BaeR , BasS-BasR , CreC-CreB , CusS-CusR , HydH-HydG , RstB-RstA , YedV-YedW , and YfhK-YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>During growth in the presence of high concentrations of Zn ( II ) , BasS senses the cation and activates BasR to induce transcription of a number of genes , some of which are known to be involved in the modiÔ¨Åcation of lipopolysaccharides and , hence , resistance to polymyxin B. TABLE 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659689" target="_blank">15659689</a>]<br></li>
<li>There is an approximate correlation for BasR regulation : BasR binding near the initiation codon represses trans-cription , while BasR binding far from the initiation codon activates the target gene ( see Figs 5b and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>DOI 10.1099 / mic.0.057745-0 Novel regulation targets of the metal-response BasS -- BasR two-component system of Escherichia coli Hiroshi Ogasawara ,3 Shota Shinohara , Kaneyoshi Yamamoto and Akira Ishihama Correspondence Akira Ishihama Department of Frontier Bioscience , Hosei University , Koganei , Tokyo 184-8584 , Japan aishiham@hosei.ac.jp The BasS -- BasR two-component system is known as an iron - and zinc-sensing transcription regulator in Escherichia coli , but so far only a few genes have been identified to be under the direct control of phosphorylated BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Most of the predicted targets under the direct control of phosphorylated BasR ( see Tables 1 and 2 ) can be classified into three groups : group 1 , genes for formation and modification of membrane structure ; group 2 , genes for membrane-associated functions ; and group 3 , genes for stress-response gene regulation and stress-response metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>One to four bands of DNA -- BasR complexes were detected for all the probes , even though some DNA fragments did not give clear gel bands , but instead showed smeared bands , implying co-operative binding of phosphorylated BasR to the DNA probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Probes with high affinity to BasR , such as eptA , formed multiple bands , probably due to co-operative BasR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>+ + Pedicted inner-membrane protein yjjP , BasR in the presence of acetyl phosphate , and after incubation were immediately subjected to PAGE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Effect of BasR on DgkA protein amounts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>This stimulating effect of BasR on DgkA-Flag was further increased in the case of the BasR53 mutant protein ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>The site of BasR binding on the csgD promoter overlaps with those of IHF , CpxR and H-NS ( Fig. 7 ) , implying that the silencing effect of H-NS might be antagonized by the combination of three regulators , BasR , IHF and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>For the same three constructions , the signal was enhanced with BasR or BasR53 overproduction , but not with DcuR overproduction , showing the speciÔ¨Åcity of this activation ( Fig. 5B and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>This suggests that while BasR activation can overcome the activity of the basal promoter of plsB , the activation of the sE promoter of plsB overcomes BasR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>As the activity of the eptC-lacZ transcriptional fusion was not totally abolished in phoB mutants , it suggests that a constitutive basal transcriptional activity is present , which is independent of both PhoB and BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23372159" target="_blank">23372159</a>]<br></li>
<li>Binding activity in vitro of BasR to target DNA probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>This in vitro experiment demonstrates the direct and speciÔ¨Åc binding of BasR to the BasR consensus sequence in the promoter of dgkA that we have evidenced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>dgkA is activated by BasR That dgkA is part of the BasRS regulon appeared as a surprise , because it was never identiÔ¨Åed before in transcriptomic or bioinformatic approaches aimed at deÔ¨Åning this regulon , either in S. typhimurium or in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Finally , dgkA is activated by the two-component regulator BasR , linking dgkA function of phospholipid recycling to LPS modiÔ¨Åcations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>In this paper , we aimed at elucidating the genetic regulation of plsB and dgkA by three stress response-regulators : sE , ppGpp and the two-component response-regulator BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Indeed , our results show that dgkA expression is activated by BasR , whereas it is inhibited by sE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Conversely , plsB expression is activated by sE , whereas it is partially inhibited by BasR as can be observed using transcriptional fusions in GK1 strain ( Fig. 7C ) , or by quantiÔ¨Åcation of plsB mRNA during BasR53 overproduction ( Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>dgkR , epilogue The results reported above demonstrated that the response-regulator BasR activates dgkA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Strong expression of dgkA was only observed in condition of BasRS activation : by overproducing BasR or BasR53 regulators , or in the dgkR-1 ( basS ( D312N ) ) genetic context .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>The E. coli BL21 pBasR transformant was grown in LB broth in the presence of 50 mg ampicillin ml , and 21 expression of BasR was induced in mid-exponential phase by adding 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>RESULTS SELEX screening system ( Shimada et al. , 2005 ) , in which purified His-tagged BasR was mixed with a collection of E. coli genome fragments of 200 -- 300 bp in length in the presence of acetyl phosphate , which phosphorylates BasR in the absence of BasS kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>We further checked the expression of ais , ugd and arnB , three genes that are known to be activated by BasR ( PmrA ) in E. coli or Salmonella ( Wosten and Groisman , 1999 ; Froelich et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>In contrary to eptA that is activated by BasR ( Wosten and Groisman , 1999 ; Froelich et al. , 2006 ) , eptB is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">BasR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The DNA-binding activity of phosphorylated BasR was examined by gel shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>This suggests that while BasR activation can overcome the activity of the basal promoter of plsB , the activation of the sE promoter of plsB overcomes BasR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by BasR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Doubling regulated regulated of altered time IdentiÔ¨Åed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiÔ¨Åcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 SigniÔ¨Åcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DÔ¨ÅmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>During growth in the presence of high concentrations of Zn ( II ) , BasS senses the cation and activates BasR to induce transcription of a number of genes , some of which are known to be involved in the modiÔ¨Åcation of lipopolysaccharides and , hence , resistance to polymyxin B. TABLE 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659689" target="_blank">15659689</a>]<br></li>
<li>This sensing of high zinc concentrations by BasS could in turn ( as it does in Sal-monella ) cause phosphorylation of BasR and ultimately upregulation of numerous genes ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659689" target="_blank">15659689</a>]<br></li>
<li>DOI 10.1099 / mic.0.057745-0 Novel regulation targets of the metal-response BasS -- BasR two-component system of Escherichia coli Hiroshi Ogasawara ,3 Shota Shinohara , Kaneyoshi Yamamoto and Akira Ishihama Correspondence Akira Ishihama Department of Frontier Bioscience , Hosei University , Koganei , Tokyo 184-8584 , Japan aishiham@hosei.ac.jp The BasS -- BasR two-component system is known as an iron - and zinc-sensing transcription regulator in Escherichia coli , but so far only a few genes have been identified to be under the direct control of phosphorylated BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Taken together , these observations support the notion that BasSR is involved in Fe ( II ) - dependent regulation , at least for the target genes examined , and that BasR is a dual regulator , acting as an activator or a repressor depending on the target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Here we identified the involvement of BasR as an activator in the regulation of the csgD promoter , which indicates that iron and/or zinc influences biofilm-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Based on the orientation of transcription , we predicted the eptA gene , encoding an innermembrane protein with binding activity to ZipA ( cell-division machinery component ) , to be a regulation target of BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The dgkA gene , encoding glycerol-3-phosphate acyltransferase , an enzyme involved in synthesis of membrane phospholipid , has been found to be under the direct control of BasR ( Wahl et al. , 2011 ) , linking the function of dgkA in phospholipid recycling to LPS modification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Also noteworthy is that the anti-silencer LeuO ( also a global regulator of membrane function ) ( Shimada et al. , 2011a ) and CsgD ( the master regulator of biofilm-formation ) ( Ogasawara et al. , 2010a , b , 2011 ) were regulated by BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The SELEX-clos analysis indicated that the regulation targets of BasR are involved in various cell functions , including the formation and modification of cell membranes , and also include stress-response proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The BasR regulon can be classified into three groups of genes : group 1 includes the genes for the formation and modification of membrane structure ; group 2 includes genes for modulation of membrane functions ; and group 3 includes genes for stress-response cell functions , including Received 16 January 2012 Revised 10 March 2012 csgD , the master regulator of biofilm-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The predicted regulation targets of BasR include three groups of genes : group 1 includes a set of genes for the synthesis and modulation of membrane structure , such as arnB ( LPS modification ) and rfaB ( LPS galactosyltransferase ) ; group Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Regulation targets of BasSR : group 2 genes for modification of membrane functions In agreement with the involvement of BasR in the remodelling of membrane structure , genes for mem-brane-associated functions were also found to be under the direct control of BasR , including csgB ( curli fimbriae curlin subunit ) , eptA ( ZipA-binding inner-membrane protein ) , fimB ( fim promoter switch recombinase ) , putP ( Pro : Na symporter ) , ompC ( outer-membrane porin C ) , phoE ( outer-membrane phosphoporin E ) , potF ( putrescine-transporter ) , mlaF ( ABC transporter ) , yrbG ( putative metal transporter ) and yfaL ( adhesin-like protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>1.5 T W basR 0.8 mM 1.0 1.6 mM 0.5 3.2 mM 0 1 0 2 3 0 1 2 3 Culture me ( h ) WT basR WT basR WT -- + -- + -- + -- + -- + ( b ) WT Fe ( II ) -- + basR -- + basR -- + Regulation targets of BasSR : group 3 genes for modulation of gene regulation and metabolism under stressful conditions A number of stress-responsive regulatory proteins were also found to be directly regulated by BasR , including the YafQ -- DinJ toxin -- antitoxin pair , TomB ( Hha toxicity modulator ) , ElbA ( RpoS level control factor ) , Eco ( serine protease inhibitor ) , EptA ( ZipA-binding septation protein ) and FimB ( fim promoter switch recombinase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">BasR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>From Northern blot analysis ( see Fig. 6 ) , the targets carrying the BasR-binding site up to at least 250 bp from the initiation codon were found to be under the direct control of BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>BasR-binding sites identified by SELEX-clos A total of 84 independent clones were isolated after SELEX-clos analysis of DNA segments binding phosphorylated BasR , and the DNA fragment in each SELEX clone was sequenced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Searching for the regulation targets of BasR by Genomic SELEX screening : SELEX-clos Genomic SELEX screening of BasR-binding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Location of BasR box sequences on target promoters Using a direct repeat of the consensus 9 bp CTTAAGGTT sequence , the location of the BasR box sequence was searched for within the putative regulation target promo-ters or genes identified by Genomic SELEX screening .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>By DNase I footprint analysis for high-affinity BasR-binding sites , a direct repeat of a TTAAnnTT sequence was identified as the BasR box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The BasR box-like sequence was detected for the major peak regions identified by SELEX-chip ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The WebLogo motif that was derived using these conserved sequences as probes indicated 8 bp TTAAnnTT to be the consensus sequence of the BasR box ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The consensus sequence of the BasR box was then identified after motif analysis using WebLogo ( http://weblogo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>B. W3110/DgkA-ProtA strains with or without the mut # mutation introduced in the BasR consensus binding site ( EB395 and EB612 ) were transformed with pBAD and pBAD-basR53 plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>The consensus regions for BasR binding , promoters and transcription start sites are indicated by boxes and arrows , in blue for dgkA elements and in green for plsB elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>We demonstrated that dgkA is part of the BasRS regulon and identiÔ¨Åed the BasR binding sequence , at the beginning of the reverse plsB coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>This in vitro experiment demonstrates the direct and speciÔ¨Åc binding of BasR to the BasR consensus sequence in the promoter of dgkA that we have evidenced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>The predicted promoter has a + 1 transcription start site 180 base pairs before the start codon of dgkA , good -10 and -35 consensus sequences , and the BasR ( PmrA ) binding site is located on top of the -35 region , as described for other promoters regulated by PmrA ( Marchal et al. , 2004 ) ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>However , the PmrA ( BasR ) binding site region that we identiÔ¨Åed upstream dgkA in E. coli is exactly conserved in S. typhimurium , and matches the reported consensus sequence ( Fig. 1B ) ( Marchal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>For dgkA promoter , an alignment of the BasR binding region with the consensus sequence for PmrA binding of Salmonella ( Marchal et al. , 2004 ) is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Except for the last nine BasR-binding sites , which are located within the ORFs of the genes shown in the BasR site column , all other BasR-binding sites were located within intergenic spacer regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Therefore , we constructed new transcriptional fusions of different lengths , comprising the potential BasR binding site : dgkA2 includes the BasR binding site and the full plsBdgkA intergenic region , dgkA3 excludes the plsBPsE promoter sequence , and dgkA4 excludes both the plsBPsE and plsBP2 promoter regions ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>In brief , probes with BasR-binding sequences identified by Genomic SELEX were generated by PCR amplification using a pair of primers , 59 FITC-labelled T7-F primer and T7-R primer ( for sequences , see Table S1 available with the online version of this paper ) , the genomic SELEX plasmids containing the respective BasR recognition sequences as templates , and Ex Taq DNA polymerase ( Takara ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Phosphorylated BasR-bound DNA fragments were affinity-purified for the identification of BasR recognition sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by BasR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>As the activity of the eptC-lacZ transcriptional fusion was not totally abolished in phoB mutants , it suggests that a constitutive basal transcriptional activity is present , which is independent of both PhoB and BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23372159" target="_blank">23372159</a>]<br></li>
<li>The orientation of the BasR box-like sequence relative to the gene organization ( or the transcription direction ) was the same for eptA , arnB , fimB and cydA , but in the opposite direction for putA , yrbL , csgD , cspI and tomB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>A single BasR box is shown by a single red arrow , which also indicates the direction of the BasR box relative to the transcription of the respective target gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>The E. coli BL21 pBasR transformant was grown in LB broth in the presence of 50 mg ampicillin ml , and 21 expression of BasR was induced in mid-exponential phase by adding 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>We further checked the expression of ais , ugd and arnB , three genes that are known to be activated by BasR ( PmrA ) in E. coli or Salmonella ( Wosten and Groisman , 1999 ; Froelich et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>In contrary to eptA that is activated by BasR ( Wosten and Groisman , 1999 ; Froelich et al. , 2006 ) , eptB is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Furthermore , overproduction of the mutated BasS ( D312N ) protein in a MG1655DbasS strain induced the expression of dgkA and ais ( Fig. 7C ) , showing that the mutation alone was sufficient to constitutively induce the BasR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Finally , the activation of dgkA expression by BasR was veriÔ¨Åed using quantitative real-time PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Indeed , our results show that dgkA expression is activated by BasR , whereas it is inhibited by sE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
<li>Conversely , plsB expression is activated by sE , whereas it is partially inhibited by BasR as can be observed using transcriptional fusions in GK1 strain ( Fig. 7C ) , or by quantiÔ¨Åcation of plsB mRNA during BasR53 overproduction ( Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463370" target="_blank">21463370</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>