<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - BluR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">BluR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">BluR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>It acts as a blue-light activated direct antagonist to the repressor protein BluR ( YcgE ) and thereby can induce several small proteins involved in the control of activity of the Rcs phosphorelay system , which , via the sRNA RprA , can downregulate the expression of CsgD ( 24 , 35 , 69 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26303830" target="_blank">26303830</a>]<br></li>
<li>Interestingly , BluR could also downregulate csgD and curli expression when expressed from a plasmid , especially when inducer was added ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Since the expression of the curli regulator CsgD is under the control of the BluR paralogue MlrA ( Brown et al. , 2001 ; Ogasawara et al. , 2010 ) , we hypothesized that BluF , when present at elevated levels , may be able to-bind MlrA and thereby also act as an anti-activator to MlrA and csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In summary , BluR may have evolved from a duplicate of MlrA , as is suggested not only by its structural and functional similarity to MlrA , but is also evident from its residual ability to directly bind to the MlrA-dependent csgD promoter , which -- when overproduced -- allows BluR to negatively inﬂuence the expression of curli ﬁbres in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>S2 ) , it would be interesting to clarify , how and why BluR lost its ability to activate gene expression and now acts as a repressor only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>These observations suggest that during evolution BluR may have lost its ability to-bind a speciﬁc ligand and therefore can not assume an activating conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In contrast to its interaction with MlrACTD only , BluF binds to both domains of BluR , with interaction to the BluRNTD being even stronger ( Fig. 4 ; and Tschowri et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response to drugs , QseB for quorum sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">BluR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>This situation is still found for instance in B. avium and A. macleodii , which do not possess any BluR homologues but a BluF variant that contains all key residues required for a c-di-GMP-hydrolysing PDE ( Figs 1C and S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In conclusion , re-introducing amino acids typically conserved in enzymatically active EAL-domain does not restore PDE activity of BluF , but rather compromises its ability to counteract BluR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In comparison , BluR contains neither the RxxD motif nor any cysteines in its CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Thus , the interaction with the DNA-binding N-terminal domain of BluR may be a novel evolutionary acquisition speciﬁc for its function as a direct antagonist that releases BluRNTD from the DNA , and may involve a direct competition of BluF and operator DNA for BluRNTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Moreover , BluR and MlrA have similar N-terminal DNA-binding domains with only four residues differing in the helix -- turn -- helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Since BluR has a similar DNA-binding helix -- turn -- helix motif ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>MlrA and BluR have a characteristic MerR-like DNA-binding N-terminal domain ( NTD ) and also share a spe-ciﬁc C-terminal domain ( CTD ; Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Origin and molecular function of the repressor BluR MerR-like transcriptional regulators contain an N-terminal helix -- turn -- helix DNA-binding region and a ligand-binding CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>All these observations taken together indicate that ( i ) both domains of BluF contribute in a complex way to the interactions with BluR or MlrA ( that an interaction between the puriﬁed BluFNTD and the two BluR domains was not observed in vitro ( Tschowri et al. , 2009 ) , may have been due to non-native protein conformation of BluFNTD alone ) , ( ii ) BluF is not just a defective PDE but has evolved to speciﬁcally bind to BluR , which involves contacts to both domains of BluR and ( iii ) BluF probably has evolved from a protein that originally had PDE activity and bound to the ligand-binding C-terminal domain of MlrA , the activator of csgD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Moreover , this assay showed that BluF interacted with the C-terminal domain of MlrA , which at ﬁrst glance seemed surprising given the previous result that BluF interacts with the N-terminal domain of the MlrA paralogue BluR ( Tschowri et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In contrast to its interaction with MlrACTD only , BluF binds to both domains of BluR , with interaction to the BluRNTD being even stronger ( Fig. 4 ; and Tschowri et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>For comparison , we also included the N-terminal as well as the C-terminal domains of BluR in this in vivo interaction analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Two-hybrid assays with the isolated N-terminal and EAL-domains of BluF showed that both BluF domains contribute to the interactions with MlrA and BluR , and conﬁrmed that only MlrACTD is contacted whereas in BluR , both domains are involved ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Two-hybrid analysis reveals that restoration of consensus amino acids in the EAL-domain of BluF improves in vivo binding to MlrACTD and that BluF interacts with N - and C-terminal domains of BluR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">BluR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>BluR does not only bind to the ycgZ promoter region , but also shows residual binding to the csgD promoter region BluR and MlrA are not only MerR-like regulators ( which are deﬁned by similar NTDs ) , but rather are direct paral-ogues that also share the CTD , which results in the same length of 243 amino acids as well as 49 % overall amino acid identity ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>The latter protein is annotated as ` MlrA ' , but in fact shows 65 % identity to BluR and 48 % identity only to MlrA from E. coli , whereas another MerR-like protein ( ) in the K. pneumoniae chromosome shows 37 % identity to BluR and 44 % identity to MlrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In comparison , BluR shows only low overall identity to other MerR-like proteins of E. coli ( 12 % to SoxR , 17 % to CueR and 14 % to ZntR ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by BluR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response-to-drugs , QseB for quorum-sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Thus , BluR and MlrA have most likely evolved from a common ancestral gene , possibly involving a gene duplication event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Among the two BLUF-EAL proteins , shows a canonical EAL domain protein that most likely acts as a PDE , whereas is degenerate to some extent and forms a coding unit with the BluR homologue and ycgZ-ymgAB-like small genes ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>It acts as a blue-light activated direct antagonist to the repressor protein BluR ( YcgE ) and thereby can induce several small proteins involved in the control of activity of the Rcs phosphorelay system , which , via the sRNA RprA , can downregulate the expression of CsgD ( 24 , 35 , 69 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26303830" target="_blank">26303830</a>]<br></li>
<li>Graphical summary of the roles of the BluF/BluR and YdaM/YciR/MlrA systems in the control of expression of CsgD and curli ﬁbres .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">BluR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>BluR binding sites were mapped to the ycgZ promoter sequence and pointed out with arrows ( arrows in the same direction indicate a direct sequence repeat , arrows pointing towards each other an inverted-repeat ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>This site is located within the ycgZ promoter , i.e. BluR binds to two half-sites overlapping with the -35 region and the spacer region of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>S2 ) as MlrA and binds to a GTACA sequence motif , which is also present in the MlrA binding site in the csgD promoter region , we wondered whether BluR is able to bind to the csgD promoter region , too .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by BluR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>S2 ) , it would be interesting to clarify , how and why BluR lost its ability to activate gene expression and now acts as a repressor only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>A. Overexpressed wild-type BluF and its mutated variants downregulate csgB : : lacZ expression independently of BluR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Also at its nonphysiological target , the csgD promoter , overexpressed BluR inactivates expression , although it replaces an activator , i.e. MlrA , in this situation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>BluF has residual affinity for the BluR paralogue MlrA which is improved upon restoring of consensus amino acids in the degenerate EAL domain Knocking out bluF in E. coli does not inﬂuence the c-di-GMP-responsive expression of curli ﬁbres as monitored by a csgB : : lacZ reporter gene fusion ( Sommerfeldt et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In summary , BluR may have evolved from a duplicate of MlrA , as is suggested not only by its structural and functional similarity to MlrA , but is also evident from its residual ability to directly bind to the MlrA-dependent csgD promoter , which -- when overproduced -- allows BluR to negatively inﬂuence the expression of curli ﬁbres in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Graphical summary of the roles of the BluF/BluR and YdaM/YciR/MlrA systems in the control of expression of CsgD and curli ﬁbres .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>