<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - CadC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">CadC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">CadC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>For example , Pb ( II ) - , Cd ( II ) - , Zn ( II ) - , and Bi ( III ) - bound CadC proteins have signi cents - cantly decreased a ⁄ nities ( 340 - , 230 - , 185 - , and 170-fold decrease respectively ) for the DNA compared to the apo-protein [ 23,39,40 ] ; addition of these metal ions to the preformed apo-CadC ^ DNA complex also results in CadC dissociation from the DNA ( M.L. VanZile and D.P. Gie-droc , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Recent comparative spectroscopic and functional studies of highly puri cents ed wild-type and ligand substitution mutants of Synechococcus SmtB , S. aureus pI258 CadC , Synechocystis ZiaR , S. aureus CzrA ( M. Pennella , C. Eicken , J. Sacchettini and D.P. Giedroc , manuscript in preparation ) and M. tuberculosis NmtR are consistent with a unifying ` theme and variations ' model for metal binding by this family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>In fact , several repressor proteins in this family do utilize ligands from this ` box ' including ArsR and CadC proteins ; however , it is now known that this parallel can not be made to all other members of the family [ 39,45,46 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>The O/P binding properties of other members of the SmtB/ArsR family , including the K5C sensor NmtR and the K3N sensor CadC , are also strongly negatively regulated by metal binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Hsieh , P. F. , Lin , H. H. , Lin , T. L. , and Wang , J. T. ( 2010 ) CadC regulates cad and tdc operons in response to gastrointestinal stresses and enhances intestinal colonization of Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25403562" target="_blank">25403562</a>]<br></li>
<li>Acid induction is mediated by the CadC regulator ( Watson et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9534244" target="_blank">9534244</a>]<br></li>
<li>( v ) CadC is limiting in the cell for Pcad induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Although the mechanism of this LysP-mediated repression is not understood , this result led to the suggestions that LysP inhibits the ability of CadC to activate transcription and that lysine , possibly through an interaction with CadC or LysP , eliminates this inhibition ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Neely MN , Dell CL , Olson ER : Roles of LysP and CadC in mediating the lysine requirement for acid induction of the Escherichia coli cad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_10">
Roles of LysP and CadC in mediating the lysine requirement for acid induction of the Escherichia coli 29 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7608084" target="_blank">7608084</a>]<br></label>
<input id="act_similarity_10", type="checkbox">
<div class="ACT_similar">
<li>Roles of LysP and CadC in mediating the lysine requirement for acid induction of the Escherichia coli cad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594828" target="_blank">14594828</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601715" target="_blank">15601715</a>]<br></li></div>
<li>According to the proposed model , LysP and CadC interact via their transmembrane domains in the absence of lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Thus , in addition to transport activity , LysP senses lysine availability and transduces the signal to CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The cognate anti-porters , GadC for glutamate , AdiC for arginine , and CadC for lysine , expel the decarboxylation product in exchange for new amino acid substrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Although the arginine and lysine-dependent acid resistance pathways are regulated by H-NS [ 1 ] , it is not known whether AdiY and CadC , the specific regulators of these pathways respectively , are controlled by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>cadBA expression is induced only in medium of low pH that is also supplemented with Lys and is dependent on an activator , CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>In lysP-defective mutants , cadBA is induced at low pH even in the absence of Lys , and the model is that LysP negatively regulates cadBA by sequestering CadC in the absence of Lys ( 34 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>regulation of the cadBA operon , by sequestering the CadC activator in the absence of Lys but not in its presence ( 34 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>By whatever speciﬁc mechanism , these models assume that the conformation or physical state of CadC is ultimately changed as a function of pH and lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>In the presence of lysine , LysP releases CadC and CadC becomes susceptible to activation by low pH. These ﬁndings suggest that LysP has an additional regulatory function , which is typical for so-called trigger transporters ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Two general models can be considered for how CadC becomes activated as a result of the cell 's exposure to the two stimuli ( low pH and lysine ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>In this scenario , CadC would be altered by pH regardless of the presence of lysine and lysine would alter CadC regardless of the pH. Examination of the kinetics of induction when the stimuli were added sequentially in the two different orders ( low pH and then lysine ; lysine and then low pH ) and when they were added simultaneously allowed us to experimentally address these two models .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Together , these data suggest that CadC senses the lysine - and pH-induced signals separately and that one of the roles of lysine in inducing cadBA may be to repress expression of lysP , thus eliminating the repressing effects of LysP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Some of these mutations result in pH-independent cadBA expression but do not alter the requirement for lysine , suggesting that separate mechanisms are involved in CadC 's ability to sense and respond to the pH - and lysine-induced signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Thus , we favor the second model ( nonsequential ) in which the ability of CadC to respond to low pH and lysine is due to mechanistically independent processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Since CadC is present at similar levels in cells grown in neutral and acidic external pH media , it appears that CadC 's ability to activate Pcad is altered as a function of pH - and lysine-induced signals ( 3 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>The membrane-localized CadC protein is required for activation of this operon under inducing conditions , which include acidic external pH , lysine , and oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>How CadC senses the pH and lysine signals , how these signals are transmitted across the membrane , which steps in Pcad activation are regulated by CadC , and what molecular properties deﬁne active and inactive CadC have not been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Basically , CadC is proposed to act as an environmental sensor detecting changes in extracellular-pH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383109" target="_blank">8383109</a>]<br></li>
<li>CadC is a transcriptional-activator for other cadaverin associated gene products ( cadA and cadB ) and is know to sense external stimuli associated with low pH and low oxygen [ 42 ] , and we can speculate that this transcriptional-activator can sense environmental conditions associated with MMS in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>The second hypothesis is that CadC is most active when synthesized in the immediate vicinity of its target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>The cadC gene product , CadC , acts in trans to activate Pcad .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Together , this suggests that transcription of cadC is not induced by low external pH. The hypothesis that CadC is limiting in the cell , even at pH 5.8 , was invoked to explain differences in the induced levels of Pcad-lacZ plasmids shown in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>1 117 241 PhoP o294 Strain ~ ~ ~ Vvm ToxR 263 - 512 I = r CadC Sequences upstream of Pcad interact with a factor necessary for its activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Comparison between PhoP , ToxR , and CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Third , since the presence of pPH2200 increased expression in the wild-type background , CadC must normally be limiting for Pcad activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>If CadC expression is required for Pcad activation , one would expect detection of a cadCspecific mRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>pHo induction of adi does not depend upon CadC , but is altered in the presence of novobiocin ( a DNA gyrase inhibitor ) , indicating that DNA supercoiting is involved in adi expression ( Stim and Bennett , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Specifically , is CadC a DNA-binding protein ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Binding of UspC was specific for KdpD , since almost no binding bet-ween UspC and CadC or LuxU was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">CadC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>According to the proposed model , LysP and CadC interact via their transmembrane domains in the absence of lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In the presence of lysine , LysP releases CadC and CadC becomes susceptible to activation by low pH. These ﬁndings suggest that LysP has an additional regulatory function , which is typical for so-called trigger transporters ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Conversely , for the K3N sensor CadC , a cluster of K3N-only residues is conserved in the N-terminal ` arm ' , while others are found in the K3 and KR helices ( cf. Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>At first glance , FeoA ( 75 amino acid resi-dues ) and FeoB ( 773 amino acid residues ) seemed to be comparable to the cadmium export system ( cadAC ) , in which one relatively large membrane protein , CadA ( 727 amino acid residues ) , and CadC ( 122 amino acid residues ) catalyze the ATP-driven export of Cd2 '' ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8407793" target="_blank">8407793</a>]<br></li>
<li>The spectrum for C11G-CadC is also characterized by Co ( II ) binding to both the K3N and K5 sites , except that the strong blue shift in the low energy transitions suggests that just three cysteine thiolate ^ Co ( II ) coordination bonds are formed in the mutant K3N Co ( II ) complex , i.e. , Cys11 is a ligand [ 39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Plasmid pPH2249 differs from pPH2200 by the presence of a frameshift mutation at the ClaI site at nucleotide 1031 ( 53 amino acids from the amino terminus of CadC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The comparisons are summarized below the sequences as follows : ( i ) asterisks correspond to residues that are conserved between CadC and at least one of the other two proteins , ( ii ) lowercase letters correspond to residues that are identical between CadC and at least one of the other proteins , and ( iii ) capital letters correspond to residues that are identical in all three proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The CadC protein is schematically shown and labeled with its three separate domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Equal molarities of the purified KdpD derivatives and , as controls , two soluble signaling proteins , the purified periplasmic domain of CadC ( sensor domain of a membrane-bound transcriptional activator in E. coli ) 38 and LuxU ( histidine-containing phosphotransfer protein ( HPt ) in Vibrio harveyi ) 39 were spotted in equal volumes onto a nitrocellulose membrane , which was incubated with purified UspC in buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>One of these proteins is the membrane-integrated transcriptional activator CadC , which consists of a periplasmic sensor domain and a cytoplasmic DNA-binding domain .38 In accordance with our in vitro observations , comparative studies with E. coli wild type cells and a ΔuspC mutant indicated that UspC boosts KdpFABC production in response to salt stress , but UspC is without effect under K + limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>This outer domain senses , either directly or indirectly , fluctuations in periplasmic pH resulting in an unspecified alteration of CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The model proposed by Watson et ai ( 1992 ) postulates that the extemal domain of CadC senses fiuctuations in periplasmic pH resulting in conformational modifications of CadC which ailow the amino-terminal cytoplasmic domain to function as a transcriptional activator specific for the cad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>A region of about 100 amino acids at the amino terminus of CadC was found to be similar to the carboxy termini of several bacterial proteins known to be transcriptional activators ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The predicted amino terminal region of the cadC gene product ( CadC ) resembles the carboxy-terminal domain of prokaryotic transcriptional activators involved in environmental sensing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The predicted amino-terminal domain of CadC displays high homoiogy to the carboxy-terminal region of prokaryotic transcriptional activators involved in environmental sensing , including PhoP from Bacillus subtilis , OmpR from E. coti , VirG from Agrobacter-ium tumefaciens and ToxR from Vibrio choierae ( Watson e ( ai , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Although a direct interaction between CadC and Pcad has not been demonstrated , the genetic evidence presented here along with the similarity between CadC and other DNA-binding proteins strongly suggests that such an interaction is responsible for pH-induced transcriptional response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Does CadC , like ToxR , have both cytoplasmic and periplasmic domains ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The basic premise of the model is that like ToxR , the amino domain of CadC is a DNA-binding domain and resides in the cytoplasm with one or more domains localized in the outer face of the cytoplasmic membrane or periplasmic space .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Specifically , is CadC a DNA-binding protein ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>Unlike the other members of this class , however , the DNA-binding domain in ToxR , like in CadC , is located at its amino terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>ToxR , however , differs from the OmpR class of response proteins in at least two fundamental ways : it is membrane bound , and like CadC , the conserved DNA-binding domain is at the amino terminus ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The mechanism by which CadC functions to activate Pcad is not clear ; however the observation that its amino terminus is similar to the carboxyl DNA-binding domain of a group of transcriptional activators ( OmpR-like proteins ) suggests that a CadC-DNA interaction is involved ( 24 , 31 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The amino terminus of CadC protein has signiﬁcant homology to the DNA-binding domain of a number of activators and is overall most similar to ToxR of Vibrio cholerae ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836317" target="_blank">7836317</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">CadC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Given the high degree of sequence similarity ( 25 ^ 50 % identity ) and the lack of a need to introduce signi cents cant gaps in a multiple sequence alignment ( Fig. 1 ) , threading of the amino acid sequence of E. coli R773 ArsR [ 44 ] , S. aureus pI258 CadC [ 40 ] and M. tuberculosis NmtR [ 27 ] through the structure of apo-SmtB [ 41 ] has allowed the generation of robust , albeit low resolution , models of these homodimeric repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>The sequence alignment of YgaV , using ClustalW ( 6 ) , with other small regulatory proteins also bearing an ArsR helix-turn-helix ( HTH ) motif ( Fig. 2 ) , shows that YgaV has greater identity ( 30 % ) with HlyU ( 18 ) of Vibrio cholerae , NolR ( 13 ) of Rhizobium meliloti , PagR ( 10 ) of Bacillus anthracis , and SoxR ( 2 , 14 ) of Pseudaminobacter salicylatoxidans than with the SmtB/ArsR regulatory proteins ( 4 ) ( SmtB , ZiaR , ArsR1 , ArsR2 , ArsR , and CadC ; 17 % identities ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245262" target="_blank">18245262</a>]<br></li>
<li>In fact , several repressor proteins in this family do utilize ligands from this ` box ' including ArsR and CadC proteins ; however , it is now known that this parallel can not be made to all other members of the family [ 39,45,46 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>The closest homologue of zntA , cadA from Staphylococcus aureus ( Nucifora et al. , 1989 ) , is regulated by a repressor CadC ( Endo and Silver , 1995 ) , a member of the ArsR family of transcriptional regulators ( Shi et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>The only protein to which the ArsR protein shows similarity is the CadC protein of the plasmid-encoded cadmium resist-ance determinant ( Nucifora et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1838573" target="_blank">1838573</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by CadC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Results : We combined mutational , phenotypic and gene-expression analyses , to unravel the regulatory hierarchy in acid resistance involving H-NS , RcsB-P/GadE complex , HdfR , CadC , AdiY regulators , and DNA-binding assays to separate direct effects from indirect ones .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Identification of the targets directly controlled by H-NS H-NS modulates the expression of several regulators controlling acid stress resistance including HdfR , RcsD , EvgA , YdeO , YdeP , GadE , GadW , GadX , AdiY and CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Conclusion In E. coli , the control of acid stress resistance is achieved by the concerted efforts of multiple regulators and overlapping systems , most of the genes directly involved in acid resistance being both controlled by RcsB-P/GadE complex and by at least one other regulator such as H-NS , HdfR , CadC or AdiY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>The cognate anti-porters , GadC for glutamate , AdiC for arginine , and CadC for lysine , expel the decarboxylation product in exchange for new amino acid substrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>CadC is a positive regulator of the H-NS-repressed cadBA locus , encoding an acid-inducible lysine decarboxylase required in the acid stress response ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>This suggests a role of CadC regulator in the H-NS regulation of the lysine-dependent acid stress resistance and a role of AdiY regulator in the arginine - and glutamate-dependent pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>The new characterized H-NS targets , CadC and AdiY , have no effect on motility ( data not shown ) and are involved in the H-NS-dependent regulation of lysine and arginine-dependent response to acid stress , respectively ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>These regulators activate expression of the corresponding target genes in response to lysine availability ( ArgP for lysP and CadC via interaction with LysP for cadBA ) ( 55 , 58 ) and low pH ( CadC ) ( 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Effects of CadC , LysP , and ArgP on cadBA-lac expression - Gal sp act ( Miller units ) a TABLE 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>The cadC gene product , CadC , acts in trans to activate Pcad .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The Cad system , which plays a role in pH homeostasis in Enterobacteria , comprises the membrane-integrated transcriptional activator CadC and the cadBA operon , encoding the lysine decarboxylase CadA and the lysine-cadaverine antiporter CadB ( 26 , 27 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The results indicate that the cadBA operon plays important roles in cellular regulation at acidic pH. Introduction The Escherichia coli cadBA operon and the adjacent cadC gene encode lysine decarboxylase ( cadA ) , a putative cadaverine-lysine antiporter ( cadB ) , and CadC , a positive regulator of cadBA expression ( Meng and Bennett , 1992 ; Watson et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14982633" target="_blank">14982633</a>]<br></li>
<li>Investigation of the molecular events that control the expression of the genes encoding these decarboxylases provides the opportunity to understand the natures of the signal-transduction-pathways that are involved in sensing and responding to changes in external pH. The Escherichia coli cadBA operon and the adjacent cadC gene encode lysine decarboxylase ( cadA ) , a lysine-cadaverine antiporter ( cadB ) , and CadC , a positive regulator of cadBA expression ( 1 , 9 , 14 , 17 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Among other targets , LeuO acts as a transcriptional activator of the bgl ( b-glucoside ) operon ( Ueguchi et al. , 1998 ) , of outer membrane porins ompS1 and ompS2 ( Flores-Valdez et al. , 2003 ; Fernandez-Mora et al. , 2004 ; Rodriguez-Morales et al. , 2006 ) , the yjjQ-bglJ operon ( Stratmann et al. , 2008 ) , and the CRISPR-associated cas operon ( Hernandez-Lucas et al. , 2008 ; Westra et al. , 2010 ; Medina-Aparicio et al. , 2011 ) , while LeuO represses transcription of the acid stress regulator CadC ( Shi and Bennett , 1995 ) and the small RNA dsrA ( Repoila and Gottesman , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>These proteins are associated with the regulation of a large ( ArcA and Fis ) or small number ( CadC , UidR , and OxyR ) of downstream targets , indicating that both global and specific transcriptional regulators were identified in our screen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>Note that besides the csgD promoter , the genes for two additional transcription factors , CadC and YrbA , are also under the control of MlrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>SmtB , E. coli ArsR , and Sta-phylococcus aureus pI258 CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Although localization of CsgD in the cytoplasmic membrane may be unusual for a transcription factor , other membraneassociated proteins , such as the ToxR protein of Vibrio cholerae ( 14 ) and CadC in E. coli ( 13 ) , can act as transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16513732" target="_blank">16513732</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">CadC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>1 117 241 PhoP o294 Strain ~ ~ ~ Vvm ToxR 263 - 512 I = r CadC Sequences upstream of Pcad interact with a factor necessary for its activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>The absorption spectrum of wildtype S. aureus pI258 CadC is a superposition of Co ( II ) complexes formed at the K3N and K5 metal binding sites ( see Fig. 5 ) [ 39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>As measured by DNase I footprinting , the binding of CadC protects a 20-bp region between the 310 hexamer and the translational start site of the cad O/P [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by CadC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>On the basis of the identiﬁcation of cadC , it was proposed that its gene product , CadC , is this required factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>Note that besides the csgD promoter , the genes for two additional transcription factors , CadC and YrbA , are also under the control of MlrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The transcription of the cad operon is positively controlled by the CadC protein , which is composed of periplasmic , transmembrane and cytoplasmic domains ( Watson et ai , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>cadC is located immediately upstream of Pcad , and if CadC is inserted into the membrane as it is being transcribed , as one would expect during coupled transcription-translation , it would localize the cad operon near the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>The genes cadB and ldcI / cadA are on the same operon ( 60 , 61 ) , and the operon is activated by an inner membrane anchored transcription activator , CadC , that binds a Pcad element in the promoter region of cadBA ( 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301313" target="_blank">16301313</a>]<br></li>
<li>Expression of the cadBA operon under inducing conditions has an absolute requirement for an activator protein , CadC , whose structural gene ( cadC ) is located about 300 bp upstream from the cadBA promoter ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836317" target="_blank">7836317</a>]<br></li>
<li>It has been demonstrated that Lrp does not alter intracellular levels of CadC , the transcriptional activator of the cadBA operon ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Regarding cadBA transcriptional regulation , it is known that H-NS represses cadBA expression under noninducing conditions ( 46 ) , and according to the current model , CadC binding dissolves the repressor complex formed by H-NS ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Together , these data suggest that CadC senses the lysine - and pH-induced signals separately and that one of the roles of lysine in inducing cadBA may be to repress expression of lysP , thus eliminating the repressing effects of LysP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>cadBA expression is induced only in medium of low pH that is also supplemented with Lys and is dependent on an activator , CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>In lysP-defective mutants , cadBA is induced at low pH even in the absence of Lys , and the model is that LysP negatively regulates cadBA by sequestering CadC in the absence of Lys ( 34 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>regulation of the cadBA operon , by sequestering the CadC activator in the absence of Lys but not in its presence ( 34 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>