<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Cra</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Cra</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Cra transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul>				<label class="ACT_collapse_similarity" for="act_similarity_1">
Mol Microbiol 11:913 -- 918 Terui Y , Higashi K , Taniguchi S , Shigemasa A , Nishimura K , Yamamoto K , Kashiwagi K , Ishihama A , Igarashi K ( 2007 ) Enhancement of the synthesis of RpoN , Cra , and H-NS by polyamines at the level of translation in Escherichia coli cultured with glucose and glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></label>
<input id="act_similarity_1", type="checkbox">
<div class="ACT_similar">
<li>Terui , Y. , Higashi , K. , Taniguchi , S. , Shigemasa , A. , Nishimura , K. , Yamamoto , K. , Kashiwagi , K. , Ishihama , A. and Igarashi , K. ( 2007 ) Enhancement of the synthesis of RpoN , Cra , and H-NS by polyamines at the level of translation in Escherichia coli cultured with glucose and glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>holixsnhgbi , gtdLen.tdom SELE matic search for the Cra-binding promoters using genomic SELEX system Tomohiro Shimada1 ,2 , Nobuyuki Fujita3 , Michihisa Maeda2 and Akira Ishihama1 ,4 , * 1Nippon Institute for Biological Science , Division of Molecular Biology , Ome , Tokyo 198-0024 , Japan 2Meiji University , Faculty of Agriculture , Kawasaki , Kanagawa 214-8571 , Japan 3National Institute of Technology and Evaluation , Genome Analysis Center , Nishihara , Shibuya , Tokyo 151-0066 , Japan 4Hosei University , Faculty of Engineering and Research Center for Micro-Nano Technology , Koganei 184-8584 , Tokyo 184-8584 , Japan Cra ( or FruR ) , a global transcription factor with both repression and activation activities , controls a large number of the genes for glycolysis and gluconeogenesis.To get insights into the entire network of transcription regulation of the E. coli genome by Cra , we isolated a set of Cra-binding sequences using an improved method of genomic SELEX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>These results are similar to previous reports that Cra represses transcription of genes that encode glycolytic ( sugar-catabolizing ) enzymes such as key enzymes in the Embden-Meyerhof and Enter-Doudoroff pathways , but activates transcription of genes that encode biosynthetic and oxidative enzymes such as key enzymes in the Krebs cycle , the glyoxylate shunt , the gluconeogenic pathway , and electron transfer [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>All genes that are negatively regulated by Cra encode glycolytic or Entner-Doudoroff pathway enzymes , whereas all genes that are positively controlled by Cra encode enzymesof the Krebs cycle , the glyoxylate shunt , or gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Cra binds to the control regions of target genes and exerts a negative effect on the expression of genes encoding glyco-lytic and Entner-Doudoroff enzymes , while exerting a positive effect on genes encoding Krebs cycle , glyoxylate shunt and gluconeogenic enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Genes encoding key enzymes concerned with oxidative and gluconeogenic carbon flow are activated by era while genes encoding key enzymes involved in fermentative carbon flow are repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Target genes encoding enzymes concerned with fermentative carbon flow ( glycolysis and Enmer-Doudoroff pathway ) are repressed by Cra , while target genes involved in oxidative and gluconeogenic carbon flow ( Krebs cycle , glyoxylate shunt and gluconeogenesis ) are activated by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>The double mutant , arcA cra , exhibited very low activities compared with either of the single mutants , showing that the combined effects of the Cra and ArcA proteins in wild-type cells are less than additive The fnr null mutation lowered cyd-lacZ fusion activity under stationary phase , microaerophilic conditions but increased this activity under logarithmic phase , anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>The cra mutation significantly reduced cyd operon expression under both microaero-philic and anaerobic conditions , suggesting that Cra serves as an activator of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Results Cra inXuences E. coli bioWlm formation independently of cAMP Attachment of E. coli to abiotic surfaces , like polystyrene and glass , leads to the formation of a bioWlm ( Pratt and Kolter 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Anti-bioWlm agents targeting Cra may be employed to Wght persistent infections and to prevent fouling of in-dwelling medical devices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>After one cycle of SELEX , mixtures of the Cra-associated DNA formed smear bands on gel , but discrete bands could be detected after the second cycle , indicating that DNA fragments with high afﬁnity to Cra were enriched but some bands with weak afﬁnity to Cra were lost during the repeated SELEX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In order to identify Cra binding sites in the E. coli genome , experimentally-determined 16-base Cra binding site sequences from the following genes were used as input for the GRASP DNA program ; fruB , icdA , aceB , epd , ppsA , mtlA , pykF , ptsH , nirB , fbp , pckA , adhE and edd ( Schilling et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>The six known Cra-regulated promoters were isolated in our SELEX screening , but seven other known promoters with Cra-binding sequences were not included in our SELEX collection.Two ( edd and pfkA ) are repressed by Cra while ﬁve ( icdA , aceB , ppsA , acnA and pckA ) are activated by Cra.To conﬁrm the Cra-binding activity for these seven promoters under the conditions employed , we prepared these promoter fragments and tested for the Cra-binding activity by the gel shift assay.All these DNA fragments formed Cra complexes ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Under the genomic SELEX conditions employed ( protein/DNA input molar ratio of 2 , two cycles of SELEX and in the absence of inducers ) , we failed to detect the promoters ( icdA , aceB , ppsA , acnA and pckA ) , which are known to be activated by Cra.We then prepared the Cra box-containing DNA fragments from these activationtype promoters , and measured the Cra-binding afﬁnity in the absence of inducers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>SELEX-clos allows the screening of DNA sequences with high afﬁnity to the test transcription factor , but it is not practical for identiﬁcation of the whole set of its binding targets because the number of isolated clones correlates with the binding afﬁnity as demonstrated for a variety of transcription factors such as Cra , CRP , and LeuO ( Shimada et al. 2009 , 2011a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>With respect to the number of DNA-binding sites , Dan is different from the well-characterized global regulators so far examined by the genomic SELEX , including CRP , Cra , RstA , PdhR , RutR , TyrR , NemR , AllR and CitB ( 7 -- 10,12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Chin et aL ( 1989 ) , however , showed that the enzyme activities of theptk and thefbp gene products are regulated by Cra ( 2.5-fold and 2-fold , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>The survival and growth of bacteria under such changing environments largely depend on the adjustments in the expression proﬁles of several global regulatory proteins , such as RpoD , RpoS , SoxRS , Cra , FadR and IclR ( Aronis 2002a , b ; Cortay et al. 1991 ; Jordan et al. 1999 ; Liochev and Fridovich 1992 ; Lu et al. 2003 ; Moat 2002 ; Tang et al. 2002 ; Varghese et al. 2003 ; Perrenoud and Sauer 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>Cra mediates catabolite repression or catabolite activation depending on the target gene era controls gene expression in vivo Expression of the negatively Cra-regulated pykF gene and the positively Cra-regulated ppsA gene was recently studied as a function of carbon source Beta-galactosidase activity measurement of singlecopy , chromosomal fusions of these genes to lacZ in E. coli strains wild-type for or lacking the Cra protein showed that Cra controls the response to carbon source independently of the cAMP-cAMP receptor protein ( CRP ) complex ( Bledig et al. , 1996 ; T.M. Ramseier and M.H. Saier , Jr. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>For instance , the genes for sugar catabolism such as fruB ( fructose-speciﬁc PTS ) , pfkA ( 6-phosphofructokinase I ) and pykF ( pyruvate kinase I ) are repressed by Cra while the genes for glyconeogenesis such as ppsA ( phosphoenolpyruvate synthase ) and fbp ( fructose-1 ,6 - bisphosphatase ) are activated by Cra ( Chin et al. 1989 ; Ramseier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The genes such as ptsHI , pfkA , pykF , acnB , edd -- eda , fruBKA , mtlADR , gapB are reported to be negatively controlled , while ppsA , fbp , pckA , acnA , icd , aceA , aceB , and cydA , B are positively controlled by the Cra protein ( Saier and Ramseier 1996 ; Cunningham et al. 1997 ; Moat et al. 2002 ; Perrenoud and Sauer 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>On the other hand , aconitase genes ( acnA and acnB ) are transcriptionally regulated by a variety of global regulatory genes , where acnB expression is activated by Crp and repressed by ArcA , Fis , and Cra , while acnA expression is initiated by r38 , activated by Crp , Fur , SoxR/S , Cra and repressed by ArcA and Fnr ( Cunningham et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>The cydA , B operon is known to be controlled by ArcA and Fnr in response to oxygen limitation , and also controlled by Cra by carbon source availability ( Ramseier et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Although it has been reported that cyd operon is regulated by the interdependency of Cra , Fnr , and ArcA ( Ramseier et al. 1996 ) , the down-regulation of this operon was due to Cra in the present case since fnr and arcA changed little for the continuous culture at the present aerobic condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Repression of transcription in vitro of the repressor-type promoters was also observed at lower concentrations of Cra than that used for transcription activation of the activation-type promoters ( Prost et al. 1999 ; T. Shimada , unpublished observation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The promoter of the pdhR-aceEF-lpd operon is repressed by pyruvate-sensing PdhR ( 33 ) and Cra ( this study ) and is activated by CRP ( 55 ; T. Shimada et al. , manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The upstream ptsH promoter ( p0 ) has been shown to be subject to both induction by growth-on-glucose and activation by cAMP/CAP ( De Reuse and Danchin , 1988 ; De Reuse et al. , 1992 ; Ryu and Garges , 1994 ) , while the second , downstream promoter ( p1 ) is regulated by FruR ( Cra ) , the repressor for the fructose PTS ( Ryu et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></li>
<li>In particular , it is noteworthy that the crp gene is under the control of Cra because we identiﬁed more than 200 novel targets for cAMPbound CRP ( Shimada et al. , in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Although extensive evidence shows that Crp autoregulates expression of its own structural gene ( Hanamura and Aiba , 1992 ; Ishizuka et al. , 1993 ; Tagami et al. , 1995 ) , this is the first evidence that Cra cross controls crp gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Discussion References Activation of the crp gene by Crp , but not by Cra , had been reported previously ( Hanamura and Aiba , 1992 ; Ishizuka et al. , 1993 ; Tagami et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Transcription regulation of the eno gene by Cra Previously , the eno gene was cloned and its expression was measured by Klein et al. ( 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra is reported to repress adhE gene in the aerobic condition ( Mikulskis et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>These last observations can be explained because growth of wild type cells in the presence of fructose yields fructose-1-phosphate , the high affinity ligand for Cra , while growth-on-glucose , the most rapidly utilized sugar in E. coli , and the primary substrate of both the glucose and mannose permeases , should produce the highest level of cytoplasmic fructose-1 ,6 - bisphosphate , the primary low affinity ligand for Cra ( Ramseier et al. , 1993 ; Ramseier et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>As expected , the Pcrp activity was unaffected by the loss of Cra when cells were grown with fructose since fructose , when converted to fructose-1-P , releases Cra from its operator ( Fig. 2 ; Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Tyson et al. ( 1997 ) showed that this results from the Cra protein , which is active as a DNA-binding repressor when cells are cultured in minimal medium ( reviewed by Saier and Ramseier , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Binding of Cra protein to a site centred at position -16.5 represses the nir promoter when cells are in minimal medium ( Tyson et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Ramseier et al. ( 1995 ) showed that carbohydrate , oligosaccharides , monosaccharides , carbohydrate acids , and carbophosphate ester are utilized via glycolysis and by the PP pathway under the negative control of Cra protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>However , only the presence of promoter epd P0 , its activation in the presence of glucose and its downregulation by the Cra protein in the presence of pyruvate were experimentally demonstrated ( Ramseier et al. , 1995 ; Saier and Ramseier , 1996 ; Charpentier et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16102000" target="_blank">16102000</a>]<br></li>
<li>Through its regulation by the Cra protein , promoter epd P0 is also expected to be modulated by the metabolic ﬂux during both glycolysis and gluconeogenesis ( Ramseier et al. , 1995 ; Saier and Ramseier , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16102000" target="_blank">16102000</a>]<br></li>
<li>Cra is a DNA binding protein Extensive protein-DNA binding studies which included DNA band mobility shift assays and vari ¬ ous footprinting techniques revealed that the purified Cra protein recognizes and binds asymmetrically to specific operator sites in the control regions of several target genes in vitro ( Ramseier et al. , 1993 , 1995 ; Cortay et al. , 1994 ; Ryu et al. , 1995 , Bledig et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Interestingly , Cra bound to all of the genes identified in the computer searches that were involved in carbon metabolism , but it did not bind to genes that functioned in other capacities Effect of catabolites on era binding to target sites In vitro Cra-DNA binding experiments revealed that micromolar concentrations of fructose l-phos-phate ( F1P ) or millimolar concentrations of fructose 1,6-bisphosphate ( FBP ) , but not other catabolites or gluconeogenic substances , displaced the Cra protein from all examined operator sites in the regulatory regions of era-controlled genes ( Ramseier er al , 1993 , 1995 ; Bledig et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>As outlined above , millimolar concentrations of FBP or micromolar concentrations of F1P were shown to inhibit Cra binding in vitro , irrespective of the target gene investigated ( Ramseier et al. , 1993 , 1995 ; Bledig et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Crp generally regulates the initiation of carbon metabolism ( Kolb et al. , 1993 ) while Cra more frequently regulates carbon flux through the dominant metabolic pathways ( Ramseier et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>However , there is extensive overlap as Crp , for example , regulates transcription of most Krebs cycle enzymes while Cra directly regulates the initiation of the metabolism of some sugars ( e.g. , fructose , allose , etc. ) ( Ramseier et al. , 1993 ; Ramseier et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Cra was initially characterized as the fructose repressor ( FruR ) , but was later recognized as a pleiotropic regulator that controls transcription of the genes in major pathways for carbon and energy metabolism ( Ramseier et al. 1993 , 1995 ; Ramseier 1996 ; Saier 1996 ; Saier & Ramseier 1996 ; Saier et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>FruR ( Cra ) is a transcriptional-regulator which , together with CRP , controls the expression of genes involved in C metabolism , particularly with respect to the direction of metabolic flow , glycolysis being repressed whereas gluconeogenesis is activated ( Ramseier et al. , 1993 ; Saier & Ramseier , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>However , the binding of fructose 1-phosphate or fructose 1,6-diphosphate , generated from the metabolism of fructose and glucose , respectively , dissociates Cra from the DNA , resulting in catabolite repression ( Cra-activated genes ) or activation ( Cra-repressed genes ) ( Ramseier et al. 1993 , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>For transcription activation of these promoters by Cra , the level of Cra must be high ( Ramseier et al. 1993 ) , but in addition , speciﬁc inducers or effectors may be needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra operators are found upstream of promoters that are activated by Cra as well as downstream or overlapping promoters that are repressed by Cra ( Ramseier et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Neither Cra ( purified as described in ref [ 27 ] ) nor Crp ( generously provided by J.C. Lee , University of Texas , Galveston , Texas ) in the presence of cyclic AMP alone shifted the DNA ( lanes 5-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Increasing amounts of Cra in the presence of Fnr shifted the 258-bp DNA fragment towards a position corresponding to the second , more slowly migrating band observed in the presence of Fnr alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>As shown in Fig. 5 , activity was decreased by the loss of Cra during growth in minimal medium containing 0.5 % Dxylose ( A ) or 0.5 % glycerol ( B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Although zwf is probably under control of Cra as shown in Table 4 , the activity of G6PDH may be enhanced by the activation of ED pathway in the cra mutant due to enzyme level regulation via reduced 6-phosphogluconate , which ﬂowed toward ED pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>In the wild-type E. coli , the GFP activity relative to the RFP , which is under the control of internal reference promoter , was very low for F1 but markedly increased in the mutant lacking Cra ( Fig. 4D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Among 20 species of the regulation targets shown in Fig. 3 , the promoter activity of 19 species except for the pps promoter increased in the cra mutant , indicating that Cra is involved as a repressor for these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Cra is involved in regulation of the whole set of enzymes in central carbon metabolism ( 43 ) , while CRP regulates the genes for carbon source utilization outside the central metab-olism , including transport of various carbon sources , glycolysisgluconeogenesis switching , the tricarboxylic acid ( TCA ) cycle , the pyruvate dehydrogenase pathway , and aerobic respiration ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In reﬂecting the physiological role of GapA as a real GAPDH , the promoter organization is very unique , being controlled by two global regulators , CRP and Cra , for a large set of genes for carbohydrate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Results Two global regulators , CRP and Cra , of E. coli recognize and regulate as many as 378 and 164 promoters , respectively , of the genes for carbon source utilization ( Shimada et al. , 2005 , 2011a , 2011b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In our previous study , we tried to identify the regulation targets of Cra after the cloning and sequencing of SELEX fragments ( 46 ) , but using the SELEX-clos method , it is practically difﬁcult to identify the whole set of regulation targets , including those with a low afﬁnity for Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>After Genomic SELEX screening of Cra , originally identified as the repressor FruR but later renamed to Cra ( catabolite repressor and activator ) [ 31 ] , its regulation targets suddenly increased more than 8-fold to 178 [ 30 ] , the number as many as that of CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Thus , the total number of regulation targets of Cra increased 7.76-fold from 23 to 178.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In fact , the number of regulation targets of Cra increased 8-fold from 20 to 178 after the Genomic SELEX screening [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>These observations are different from previous reports that Cra deletion in E. coli MG1655 contributed to accumulation of pyruvate by suppressing gluconeogenesis due to the down regulation of ppsA and pckA [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>A minus sign ( - ) indicates negative regulation exerted by Cra , while a plus sign ( + ) indicates positive regulation by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>In order to gain insight into genome regulation by Cra for the control of carbon metabolism in E. coli , we tried to identify the whole set of target promoters , genes , and operons under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Conﬁrmation of regulation in vivo of the predicted target by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Based on these criteria , we propose that the total number of regulation target genes under the control of Cra is 178.5 , as listed in Table 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Transcription regulation of the yahA and hypF genes by Cra Transcription in vitro of the gapA and eno genes was repressed by Cra in dose-dependent ways ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Lower panel : Most of the genes for the enzymes involved in glycolysis are controlled by Cra [ 30 ] while only three genes , fbaA , gapA and pgk , had been identified as the regulation targets of CRP ( shown in while under blue background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>These data suggest that carbon source-dependent regulation by Cra is mediated by the catabolic effectots FBP and/or FIP and that the concentrations of these catabolites are dependent on the growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>This broad role of Cra in the regulation of carbon ﬂow has recently become more fully articulated ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>This finding raises the need for reevaluation of the differential roles of two global regulators , CRP and Cra , in the overall regulation network of carbon source metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>This finding agrees with elaborate footprinting studies which established that Cra binds asymmetrically to its operator sites , it was found that era binds predominantly to the first half-site of the era binding consensus sequence ( Cortay et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Examination of the aligned operators revealed that the consensus sequence for Cra binding is the oame for operons that are activated and repressed by era .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Cra activates transcription of csgDEFG operon on binding to its operator sites The negative strand of the curli intergenic region was cloned upstream of a promoterless gfp reporter gene , and the contribution of each of the three Cra-binding sites identiWed upstream of the csgD coding sequence to the regulation of the csgDEFG operon was studied by mutagenizing conserved bases in the Cra-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>In the light of our results , this correlation may not hold true for csgDEGF too , which is transcriptionally activated by Cra though one of its binding sites ( O1 ) is downstream of the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>In the case of the positively Cra-regulated ppsA gene , cat-abolite-induced Cra displacement from its binding site results in deactivation of ppsA gene transcription , a form of cAMP-independent catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>On the other hand , in the case of the negatively Cra-controlled pykF gene , catabolite-it , duced Cra displacement from its binding site results in Jerepression of pykF gene transcription , a cAMP-independent phenomenon for which the appropriate term catabolite activation is to put to use .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Introduction The phospohoenolpyruvate : carbohydrate transferase system ( PTS ) , in conjunction with Crp and Cra , constitute the main regulatory system in Escherichia coli involved in catabolite carbon repression that allows the utilization of glucose as the preferred carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Key words Catabolite regulation · Cra protein · Nitrite reductase regulation · Escherichia coli · Catabolite repression Abbreviations Cra Catabolite repressor-activator protein · nir Genes encoding NADH-dependent nitrite reduction complex · nrf genes encoding the formate-dependent nitrite reduction complex Introduction The generation of ATP is an essential process in all living organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>It involves repression by the GB regulator ICLR and activation by the nucleoid protein IHF and the global regulator Cra ( also known as FruR ) ( reviewed in reference 95 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>FruR , also known as , Cra ( Catabolic repressor/activator ) , is a global transcription regulatory protein in enteric bacteria that regulates gene expression by binding to a specific DNA sequence [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The regulation of PTS system was most probably exerted by the global transcriptional control factors such as cAMP -- CRP complex and the catabolite repressor/activator protein ( Cra ) ( for reviews , see Saier and Ramseier , 1996 ; Titgemeyer and Hillen , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
<li>In contrast , the Cra protein ( catabolite repressor-activator , formerly designated the fructose repressor , FruR ) mediates catabolite regulation independently of cAMP ( Geerse et al. 1986 , 1989 ; Chin et al. 1987 , 1989 ; Ramseier et al. 1993 ; Saier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>The catabolite repressor/activator Cra in E. coli binds DNA operators in the absence of fructose 1-phos-phate or fructose 1,6-biphosphate and is inactivated upon the interaction with the effectors ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>The second protein , Cra ( for catabolite repressor activator ) , represses adhE expression ( 19 ) , but this transcriptional-regulator is functional only when complexed with fructose-1-phosphate or fructose-1 ,6 - bisphosphate as an effector ( for a review , see reference 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></li>
<li>The operon is also positively controlled by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex and negatively by the catabolite repressor / activator ( Cra ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The fru operon is inducible in wild-type E. coli K12 due to the presence of the fructose repressor , FruR , also known as the catabolite repressor/activator ( Cra ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The LacI protein family catabolite repressor/activator , Cra , was initially characterized as the fructose repressor , FruR , because it negatively regulates transcription of the fructose utilization operon ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>It is worthwhile to compare Mlc with the FruR repressor ( recently renamed Cra for catabolite repressor/activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>Therefore , Cra , the catabolite repressor-activator p ~ tein , mediates cAMP-independent eatabolite repression and activation in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>A number of the genes for both glycolysis and gluconeogenesis are under the control of catabolite repressor activator Cra , but independent of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The 50S L20 operonb rplT < rpmI infC bamD > raiA Translation inhibitor set of genes that are involved in carbon utilization for energy production is controlled by two global regulators , CRP ( cAMP receptor protein or catabolite activation protein ) and Cra ( catabolite repressor and activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The 63His-tagged Cra and Cra * proteins were puriﬁed from isopropyl-b-D-thiogalactopyranoside ( IPTG ) - induced E. coli XL-1 Blue cells ( Stratagene ) transformed with pAM47-5 and pAM46-3 , respectively , by using Ni-NTA resin ( QIAGEN ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The 63His-tagged Cra and Cra * proteins were overproduced and puriﬁed by one-step afﬁnity chromotography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>For convenience , we will refer to the puriﬁed proteins simply as Cra and Cra * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The utilization of gluconate as well as glycerol + CAA was slightly decreased ; glucose utilization was only slightly decreased , and fructose utilization was substantially increased by the loss of Cra ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>The utilization of xylose , glycerol , lactose , succinate and sorbitol are subject to stringent Crp control as well as weak Cra control ( Table 2 ) , but there is no Cra binding site upstream of their catabolic enzyme-encoding operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Protein concentrations are expressed ( top ) in terms of the monomeric protein , although Crp and Fnr are dimeric and Cra is tetrameric .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>The three restriction fragments correspond to those depicted in Figure 1 B. Lane 1 : control with no protein added ; lanes 2-4 : increasing concentrations of Fnr ; lanes 5-7 : increasing concentrations of Cra ; lanes 8 and 9 : increasing concentrations of Crp ; lanes 10-12 : increasing concentrations of Cra in the presence of 400 nM Fnr ; lanes 13 and 14 : increasing concentrations of Cra in the presence of 600 nM Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>The effect of such an alteration on the Pcrp activity was determined using the altered promoterlacZ fusion in the MG1655ΔlacY and MG1655ΔlacYΔfruR strains in both the presence and absence of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>The two arrows to the right of the figure indicate the two shifted bands that were observed when Fnr alone was present , and which markedly increased in intensity when the Cra protein was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>The mechanism by which Cra acts to stimulate bioWlm formation is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>The absence of Cra protein was conﬁrmed by immunoblotting of whole cell lysates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The Cra targets listed in RegulonDB are indicated in the `` RegulonDB '' column .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The whole set of regulation targets of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The up-regulation of fru operon in the mutant is consistent with the previous study , since this operon is under the negative control of Cra protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>The Fnr protein is required for Cra-mediated transcriptional control , but the ArcA protein antagonizes the response to Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>As also revealed by the data in Table 2 , the loss of Fur function completely abolished the effect of Cra under both conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>On the other hand , loss of Cra function clearly did not abolish the activation observed when Fnr function was lost under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>In earlier studies we found that cydAB operon expression is not influenced by Crp but is controlled by Cra ( A.M. Chin , R.B. Gennis , and M.H. Saier , Jr. , unpublished results , cited in ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>In vitro studies of Cra and Cra * function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>In this paper we report both in vivo and in vitro studies that suggest that the mechanism of Cra action depends both on Fnr and ArcA in opposing ways : Fnr is required for the Cra response , while ArcA counteracts it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Yanofsky , C. , T. Platt , I. P. Cra , wford , B. P. Nichols , G. E. Christie , H. Horowitz , M. VanCleemput , and A. M. Wu .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>Additionally , physical interactions between Crp and/or Fur with other regulators such as IHF , Cra , and FadR could be important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659676" target="_blank">15659676</a>]<br></li>
<li>No signiﬁcant effect of Cra was found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></li>
<li>Alternatively or in addition , Cra binding could antagonize the repressive effect of Crp bound to CrpII .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>These results are consistent with the conclusion that Cra mediates its effect , at least in part , by binding to the proposed Cra binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>These results are in agreement with the in vivo results ( Table 2 ) showing that an effect of Cra is dependent on the presence of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>We then concluded that the Cra boxes within F2 region contribute to the repression of ene gene transcription from P1 , P2 or P3 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The repression of the above-described four cases , however , takes place by Cra bound more than 100 bp upstream of the respective target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The repression level of in vitro transcription , however , differed among gapA , eno , yahA and hypF , reﬂecting not only the afﬁnity of each Cra box to Cra protein but also the location of Cra box relative to the respective promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The Cra protein can bind overlapping or downstream of certain promoters to inhibit transcription and can bind upstream of other promoters to activate their expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>This may explain the relatively modest 1.5 - to 4-fold effect of adhE repression by Cra observed in our study , in contrast to the 20-fold repression of the fru operon ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The stronger repression of adhE by Cra under aerobic than anaerobic-growth conditions gives the appearance of a redox control , but in fact , this difference in gene expression may reﬂect F1P and F16P levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The mutant protein ( Cra * ) sustained an Arg1483His substitution causing 1.5 - and 3-fold stronger repression of adhE transcription under anaerobic and aerobic conditions , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>We suggest that Cra helps to tighten the control of adhE transcription under aerobic conditions by its repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>It is likely that the expression of adhE is de-repressed by the adhC81 mutation due to the afinity of Cra to the Cra-box being decreased by the G !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>This result again conﬁrmed that repression of the adhE gene by Cra was canceled by the adhC81 mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>This mutation caused de-repression of the adhE gene under aerobic conditions probably because of decreased afinity of the Cra protein to the mutant Cra-box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>The double mutant ECL4004 , lacking both Cra and Fis , expressed F ( adhE-lacZ ) at very low levels either aerobically or anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>Cra ( 24 ) , it is possible that in the absence of this global regulator , Fis is no longer required for adhE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>Fis dependence of adhE expression in the presence or absence of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>Thus , Fis is required for adhE expression independently of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>Inactivation of cra , on the other hand , resulted in elevated levels of adhE and adhE9-lacZ expression under those growth conditions , suggesting that Cra acts as a repressor of adhE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Our results showed that Cra repressed adhE expression 1.5-fold during anaerobic-growth-on-LB-glucose and 4-fold during aerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Thus , Cra bound on sgrS may lead to the repression of sgrR , thereby resulting in a decrease in activator SgrR levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Thus , the upstream-bound Cra may interact with one or more of these factors for effective repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>However , disruption of cra had no effect on the levels of β-galactosidase from pKT53 , indicating that Cra is not responsible for catabolite repression of the nrf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>These observations suggest that in several cases , catabolite repression depends on both Crp and Cra ( 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>Instead , catabolite repression is mediated by the Fis and Cra proteins ( 14 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>The physiological effect of Cra on gene expression was assessedby the construction of single-copy , chro-mosomal lacZ fusions to ppsA , pckA , gapB , edd , mtlA and pykF in the genetic backgrounds of a era mutant and its isogenic wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Cra activates expression of the crp gene when xylose , glycerol ± CAA or gluconate was used as the sole carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Since both Crp and Cra are recognized to regulate gene expression in response to carbon source availability , either one might regulate cyd gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>In most cases , however , the effects of Cra on gene expression were found to be no more than 2 - to 8-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>We show that Cra regulates growth by activating expression of the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Neither Crp nor Cra alone can fully activate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Cra and Crp both activate expression of the crp gene , apparently by a mechanism that requires the simultaneous presence of both proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>FruR ( also known as Cra ) regulates the expression of many genes involved in carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>All of the results presented in Figs. 2 , 3 , 4 and 5 as well as Table 2 are consistent with the conclusion that Cra regulates crp gene expression and thereby indirectly controls expression of numerous operons involved in the initiation of carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>As the curli biosynthesis operons , csgBAC and csg-DEFG , are highly repressed in the cra mutant ( Table 2 ) , it was hypothesized that Cra may directly regulate expression of these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Moreover , since Cra positively regulate ace operon , the glyoxylate pathway is repressed in cra gene knockout mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Since Cra activated ED pathway , and repressed TCA cycle and the glyoxylate pathway , we constructed cra.edd.iclR genes knockout mutant to reduce the acetate production by activating the glyoxylate pathway , thus improving the cellular performance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>The molecular mechanism by which CcpA activates transcription is not known ; although its E. coli ortholog Cra ( 367 , 389 ) activates transcription of ppsA , which encodes a gluconeogenic enzyme , from a site located at position 45.5 ( 319 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Whereas Cra positively regulates the genes that encode the GB and gluconeogenesis ( 319 , 366 , 389 , 439 ) , there exists no evidence that this global regulator exerts a signiﬁcant effect on acs transcription ( Beatty and Wolfe , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Cra also activates pps and pck , thus stimulating gluconeogenesis ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Note that Cra also activates genes that encode gluconeogenesis enzymes ( 319 , 366 , 367 , 389 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The possible role of Cra in modulating the expression of pfkA and pykF genes and the epd-pgk-fbaA operon downregulated in PB11 , could be part of a more general mechanism , in which due to the absence of PTS , not only the Cra regulated genes but most glycolytic genes were downregulated in PB11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Nevertheless , downregulation of these genes may be due in part to Cra repression ; however , pykF and epd-pgk-fbaA transcription in both PTS strains was only slightly downregulated as compared to 2 JM101 , suggesting , a minor role of Cra in the expression of these genes [ 66,67 ] in the tested conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>This phenomena is reinforced by the differential expression Biomass ( g/L ) Biomass Indole of some genes regulated by Cra which respond to low F1 ,6 P concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>All six known SELEX fragments ( fruB , epd , adhE , ptsH , nirB , pykF ) are the group of genes , which are repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Genes that are activated by Cra ( e.g. ppsA , pckA , and aceBA ) , are generally subject to catabolite repression , while genes that are repressed by Cra ( e.g. fru , pts , and edd ) , are subjected to catabolite activation [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Both the pykA and pykF genes , encoding pyruvate kinase , are also repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The afﬁnity to Cra is generally higher for the promoters repressed by Cra , while in the absence of inducers ( or effectors ) , the promoters from the genes activated by Cra showed low afﬁnity to Cra protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Generally the genes , which are repressed by Cra , have higher afﬁnity to Cra than those that are activated by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The icdA and aceA genes are known to be repressed by ArcA/B and activated by Cra ( Additional file 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19055729" target="_blank">19055729</a>]<br></li>
<li>Here we demonstrated that between the genes under the control of Cra , there is a hierarchy in the afﬁnity to Cra ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The genes that most clearly showed dual control of catabolite repression by Crp and Cra were gat , mgl , tre , and dsd ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>The genes shown in blue boxes are under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The presence of different promoters in these genes could Remarkably , few general transcription regulators have been reported to be involved in modulating the expression of most glycolytic genes - Cra and Crp , among others - especially at the upper part of the glycolytic pathway involved in the transformation of glucose-6P into pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>The pykA and pykF genes have putative promoters that could be transcribed by RpoS , and pykF is negatively regulated by Cra [ 20,53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Cra represses pfkA , pykF , and glk , thereby accelerating glycolysis ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Since Cra represses the glycolysis pathway genes such as ptsHI , pfkA , pykF , and activates the gluconeogenic pathway genes such as fbp , ppsA , pckA , etc. , cra gene knockout may activate glycolysis , and the glucose consumption rate may be increased as shown in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>It has been shown that cra disruption causes pyruvate accumulation , which is caused by the increased transcription of glycolysis genes , since transcription from such genes is negatively regulated by Cra ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24212579" target="_blank">24212579</a>]<br></li>
<li>In gluconeogenic conditions , where E. coli grows relatively slow , transcription of glycolytic Cra regulated genes should be decreased , while transcription of Cra regulated gluconeogenic genes should be increased ; in agreement , cra inactivation inhibits growth on gluconeogenic substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>In contrast , transcription from the nrf promoter was not regulated by Cra under any of the growth conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>In certain cases , such as those of aroP4Cra3pdhR and fruB4Cra3setB ( Table 2 ) , the spacer-bound Cra regulates transcription toward both directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>For most of the targets identiﬁed herein , Cra represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>For isolation of DNA sequences that are recognized by E. coli Cra protein , genomic DNA fragments were regenerated by PCR using a pair of primers , which bind to the vector sequences at the 5 ′ and 3 ′ vector-insert junctions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Promoter fragments from the yahA ( A ) and hypF ( B ) were transcribed in vitro under the standard assay conditions , as described in Experimental procedures , by RNA polymerase 70 Eσ holoenzyme and in the absence or presence of increasing concentrations of Cra protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>F1P and F16P2 can displace Cra from DNA in vitro at micromolar and millimolar concentrations , respectively , and are thought to be the effectors in vivo ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The wild-type Cra protein binds to DNA as a homotetramer and is readily displaced from DNA in vitro by micromolar concentrations of F1P and milli-molar concentrations of F16P2 ( 23 ) , leading to derepression of negatively controlled promoters and deactivation of positively controlled promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>However , low but clear peaks were identiﬁed in the presence of effectors by expanding the y axis , indicating that the afﬁnity for target sequences decreased for effector-bound Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In the presence of effectors , either D-fructose-1-phosphate or D-fructose-1 ,6 - bisphosphate , SELEX bands remained smeared on PAG gels even after 2 cycles of genomic SELEX , indicating that the sequence selectivity of Cra is higher without binding the effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The SELEX chip analysis in the absence of an effector indicated a total of 194 peaks of Cra binding , but no clear peaks of Cra binding were identiﬁed in the presence of D-fructose-1 ,6 - bisphosphate ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Cra forms two different conformations depending on the presence or absence of the effector D-fructose-1-phosphate or D-fructose-1 ,6 - phos-phate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Here we identiﬁed three in vivo promoters by S1 mapping , and all three promoters appeared to be derepressed in the absence of Cra protein ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Both the gapA ( glyceraldehyde 3-phosphate dehydrogenase ) and eno ( enolase ) promoters showed high afﬁnity to Cra in the absence of inducers ( or effectors ) as in the case of repression-type gene promoters ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In contrast , all the promoters that are activated by Cra were not selected in our SELEX , because the afﬁnity of Cra to this group promoters is weak under the conditions employed ( in the absence of effectors ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In the absence of inducers , Cra should ﬁrst bind to the promoters of repression-group genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In the presence of Cra , the level of P1 RNA decreased in Cra dose-dependent way ( up to 40 pmol Cra ) , while P2 RNA level stays almost constant in the presence and absence of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In the absence of the effectors , F1P and F16P2 , the Cra protein can act either as a repressor or as an activator , depending on the target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>In the absence of glucose , Cra binds to the promoters of target operons and activates or represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>In the presence of glucose , the intracellular concentrations of the inducers increase , which interact with Cra to prevent its binding to the target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In both modes , the level of binding of Cra to the target Cra box decreases in the presence of effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>This regulatory function of Cra is controlled by the intracellular concentrations of key metabolites , including D-fructose-1-phosphate and - fructore-1 ,6 - bisphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Derepression of the genes for glycolysis takes place when Cra is inactivated after interaction with inducers ( or effectors ) such as d-fructose-1-phosphate and d-fructose-1 ,6 - bisphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The induction of the fructose operon takes place when the repressor Cra is inactivated after interactions with inducers such as D-fructose-1-phosphate and D-fructose-1 ,6 - bisphosphate ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Derepression of the glycolytic network and deactivation of the gluconeogenic network by the effectors of Cra also help t ensure the preferential utilization of carbon sources such as glucose and fructose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>When wild type E. coli grows fast aerobically on glucose , such as strain JM101 , Cra whose DNA binding activity is modulated by sugar catabolites ( fructose-1-P and fructose-1-6P ) , does not significantly repress glycolytic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>This was not surprising since , in the presence of glucose , Cra binds the glycolytic intermediate fructose 1,6-diphosphate , causing the protein to dissociate from the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>SpeciWc binding of puriWed His-tagged Cra protein to a 200-base pair DNA fragment encompassing the region upstream of the coding sequence of csgB conWrmed the presence of a Cra-binding motif in the region ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Furthermore , the inclusion of fructose 1-phosphate , an effector of Cra DNA-binding activity , in the assay decreased the ability of Cra to retard the nir promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Furthermore , the amount of retarded nir promoter fragment decreased when 0.5 mM fructose 1-phosphate was included in the reaction ( Fig. 3 compare lanes 3 and 4 ) , confirming that the observed retardation is due to the binding of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>However , only 63His-tagged Cra , and not 63His-Cra * , was displaced from the DNA target by the effector , fructose-1-phosphate ( F1P ) , suggesting that the mutant protein is locked in a promoter-binding conformation and is no longer responsive to F1P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Another possibility is that Cra * is locked in a conformation insensitive to fructose-1-phosphate ( F1P ) or fructose-1 ,6 - bisphosphate ( F16P2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Our results indicated that the enhanced adhE repression in the mutant strain was due to the inability of the effectors to displace Cra * from DNA because the protein was locked in the binding conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Results indicate the presence of two Cra-binding sequences on the gapA promoter , Cra box-1 on F1 and Cra box-2 on F2 fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>These results suggested that Cra does bind directly to the adhE promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>As expected , both Cra and Cra * bound to the adhE promoter fragment , resulting in a single band shift ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>To test the hypothesis of locked conformation , we examined Cra and Cra * binding to the adhE and fruB promoter fragments with or without F1P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Moreover , both Cra and Cra * bound to DNA with similar afﬁnities and both proteins bound more strongly to the fruB promoter fragment than to the adhE promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>However , comparable afﬁnities of Cra and Cra * binding to DNA ruled out the possibility of a more stable binding of Cra * to the adhE promoter ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>No difference , however , was observed in the afﬁnity of Cra and Cra * binding to the adhE or fruB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Results of DNA band shift assays demonstrating Cra and Cra * binding to the adhE and fruB promoter regions but no binding to the cra gene region lacking the Cra-binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Results of a DNA band shift assay representing Cra and Cra * binding to the adhE and fruB promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Among seven promoters that were hitherto identiﬁed to be under the control of Cra but not selected by this SELEX , two ( pfkA and edd ) belong to the promoter group that is repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Moreover we identiﬁed in this study that gapA P1 is repressed by Cra , which binds to the promoter-proximal Cra-box 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Transcription from the nir promoter was repressed by Cra when cells were grown in minimal me-dium with glycerol and fumarate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>To conﬁrm that the newly identiﬁed target candidates are indeed under the control of Cra , we constructed a set of promoter-lacZ fusions and performed the promoter assay in the presence and absence of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The regulation of newly identiﬁed genes by Cra was conﬁrmed by the in vivo promoter strength assay using a newly developed TFP ( two-ﬂuorescent protein ) vector for promoter assay or by in vitro transcription assay in the presence of Cra protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Interestingly , nitrate was less effective in stimulating transcription from the nir promoter in poor media than in rich media , irrespective of the presence of functional Cra protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Using lacZ reporter fusions , we investigated the role of the Cra protein in mediating glucose repression and in the activation of the nrf and nir operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>The binding of NarL or NarP to the 7-2-7 element , centred at position -69.5 , counteracts the effects of IHF and Fis , and activates the promoter provided that Cra is not present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Effect of a Cra operator mutation on Cra activation of Pcrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>One possible explanation is that an as yet unidentiﬁed promoter in F1 requires Cra for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The reason for this phenomenon is unknown at present , but the results clearly demonstrate that Cra represses the nir promoter , resulting in catabolite activation of nir operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>The activation of the crp promoter by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Mobility shift assay experiments were used to confirm that Cra binds to the nir promoter and that the observed Cra-dependent repression of nir expression is not due to indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Cra represses the promoter by blocking access to the -10 element and this repression can not be overridden by the other upstream activators ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>A recent compilation of data on Cra-mediated gene regulation indicates that binding of Cra upstream from a promoter activates gene transcription , whereas downstream binding causes gene repression ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Glucose-6-phosphate , NAD , NADH , NADP , NADPH , and ATP also failed to interfere with the binding of Cra or Cra * to DNA even at a concentration of 5 mM ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Some of the DNA micro-array results are consistent with the previously published data , while several others seem to be under control of Cra as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>During glueoneogenic growth , the concentrations of FBP and FIP are apparently insufficient to cause dissociation of Cra from its operator sites , resulting in Cra-inhibited transcription of pykF and Cra-activated transcription of ppsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>However , in gluconeogenic conditions these sugar catabolites are present in relatively lower concentrations thus allowing Cra to repress glycolytic and activate gluconeogenic genes [ 49,66,67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>These results can be explained by the fact that sugar catabolites present at higher concentrations when growing rapidly on glucose , bind to Cra and displace it from the operator sites in target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Sugar catabolites tend to-bind to Cra and displace it from the operator sites in target operons [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Negative control is exerted by Cra on a number of operons involved in glycolysis permitting derepression of the glycolytic pathways by F1P and/or F16P2 in the presence of exogenous six-carbon compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Positive control is exerted by Cra over a number of operons involved in gluconeogenesis , permitting induction of the gluconeogenic pathways by F1P and/or F16P2 in the presence of nonfermentable compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Seventeen of these carbon sources were oxidized more slowly or not at all by the era mutant ( carbon sources positively controlled by Cra ) whereas nineteen were oxidized more rapidly ( carbon sources negatively regulated by Cra ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>This finding indicated that in the presence of FIP or FBP , operons under positive era control are deactivated while operons under negative Cra control are derepressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Consequently , operons which are positively regulated by Cra are deactivated or in other words , repressed by certain catabolites ; i.e. , catabolite repression occurs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Only those operons that are regulated by Cra are indicated in italics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Cra and the control of carbon flux via metabolic pathways T.M. Ramseier Department of Biology , University of California at San Diego , La Jolla , California 92093-0116 ( USA ) Summary Weisser , P. , Krtimer , R. , Sahm , H. & Sprenger , G.A. ( 1995 ) , Functional expression of the glucose transporter of Zymomonas mobilis leads to restoration of glucose and fructose up-take in Escherichia coli mutants and provides evidence for its facilit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Regulation of crp gene expression by the catabolite repressor / activator , Cra , in Escherichia coli Zhongge Zhanga , Mohammad Aboulwafaa , b , and Milton H. Saier Jr. a Zhongge Zhang : zzhang@biomail.ucsd.edu; Mohammad Aboulwafa : maboulwafa@yahoo.com; Milton H. Saier : msaier@ucsd.edu aDepartment of Molecular Biology , Division of Biological Sciences , University of California at San Diego , 9500 Gilman Drive , La Jolla , CA 92093-0116 bDepartment of Microbiology and Immunology , Faculty of Pharmacy , Ain Shams University , Al Khalifa Al Maamoun St. , Abbassia , Cairo , Egypt Abstract Growth on several carbon sources is dependent on the catabolite repressor/activator ( Cra ) protein although a Cra consensus DNA binding site is not present in the control regions of the relevant catabolic operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Thus , both HexR and Cra are dual transcriptional-regulators that mediate cAMP-independent catabolite control in response to key catab-olite intermediates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>In addition to Cra and ArcA/B , other metabolic regulators are involved in acetate metabolism in E.coli : acetyl-CoA synthetase is under the control of the sigma factors RpoS and RpoD and requires cAMP receptor protein , and the oxygen regulator ( Fnr ) for full expression [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>In Escherichia coli two global transcriptional-regulatory proteins , Crp and Cra , are known to control transcriptional responses to carbon availability [ 28 , 30 ] , while two other global regulators , Fnr and ArcA , control gene expression in response to oxygen availability [ 14 , 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Crp and Cra are controlled by cytoplasmic cyclic AMP and metabolite concentrations respectively [ 3 , 4 , 24-281 , while Fnr and ArcA are activated for transcriptional regulation by anoxia [ 14 , 18 , 33 , 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Enzyme Microb Technol 38:512 -- 520 Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>Antonie Van Leeuwenhoek 71 ( 3 ) :217 -- 230 Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li>
<li>J Bacteriol 179:4138 -- 4142 Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_256">
Impact of global transcriptional regula ¬ tion by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></label>
<input id="act_similarity_256", type="checkbox">
<div class="ACT_similar">
<li>Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li></div>
<li>They lead us to propose that protein -- protein interactions between Fnr and Cra are crucial to the transcriptional regulation of the cyd operon , and that activation by Cra involves an ordered sequence of events in which Fnr binding to the transcriptional initiation complex precedes Cra binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>It therefore appears that Crp and the fruR gene product , Cra , cooperate in the regulation of many operons in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>Cooperative interaction between Cra and Fnr in the regulation of the cydAB operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Mol Microbiol 16:1157 -- 1169 Ramseier TM , Chien SY , Saier MH Jr ( 1996 ) Cooperative interaction between Cra and Fnr in the regulation of the cydAB operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Cra protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>On the other hand , loss of Cra function clearly did not abolish the activation observed when Fnr function was lost under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Cra , which recognition site is located 197 nucleotides upstream the ATG initiation codon [ 49 , Regulon DB http://www.ccg.unam.mx] or some other factor , could play a role in the utilization of these putative promoters in which no clear s 210 recognition elements were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Using the improved method of genomic SELEX , we identiﬁed in this study at least four new members of the Cra regulon , including two genes with the known functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Here we identiﬁed three in vivo promoters by S1 mapping , and all three promoters appeared to be derepressed in the absence of Cra protein ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra and Cra * were puriﬁed as fusion proteins tagged with an N-terminal 63His element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Cra is a member of the LacI-GalR family of transcriptional regulatory proteins with N-terminal helix-turn-helix DNA-binding motifs and C-terminal ligand-binding domains ( Nguyen and Saier 1995 ; Weickert and Adhya 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Therefore , DNA-binding studies in vitro were essential to conﬁrm Cra interaction with the adhE promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>DNA-binding activity of Cra is triggered by two central intermediates of the EMP pathway , fructose 1-phosphate and fructose 1,6-biphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>This putative Cra-binding motif , however , differed from the Cra consensus by two conserved nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Cra consists of two functional domains , an N-terminal DNA-binding domain with an H-T-H motif and a C-terminal inducer-binding and subunit-subunit contact domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The regulatory regions present upstream of this operon or upstream of pdeL ( in the strains that do not contain the AidA-I-like adhesin gene ) share important regulatory motifs , e.g. , a Cra binding site and the putative promoter ( 53 ) , with some divergence in between these motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26303830" target="_blank">26303830</a>]<br></li>
<li>A transition at ) 257 , is located in the Cra protein-binding site ( Fig. 2A ) [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>Unlabelled oligonucleotides containing the putative csgB Cra-binding site used in excess were used to demonstrate speciWc binding of Cra to the identiWed operator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Cra with a molecular size of 334 amino acid residues forms a tetramer with a molecular mass of 148 kDa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra forms two different conformations depending on the presence or absence of the effector D-fructose-1-phosphate or D-fructose-1 ,6 - phos-phate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">Cra evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Crp , the cyclic AMP receptor protein , and Cra , the catabolite repressor/activator , belong to two distinct families of helix-turn-helix DNA-binding proteins , and they recognize two completely different DNA-binding consensus sequences [ 24-26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Cra proved to be homologous to a family of helix-turn-helix DNA binding proteins called the GalR-Lacl family ( Jahreis et al. , 1991 ; Vartak et al. , 1991 ; Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Cra is a member of the LacI-GalR family of transcriptional regulatory proteins with N-terminal helix-turn-helix DNA-binding motifs and C-terminal ligand-binding domains ( Nguyen and Saier 1995 ; Weickert and Adhya 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Cra , a member of the GalR-LacI family , was originally iden ¬ * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>On the basis of sequence similarity , Cra is classiﬁed in the GalR-LacI family ( Leclerc et al. 1990 ; N. Fujita , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra is a member of the LacI-GalR family ( Feldheim et al. , 1990 ; Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Cra</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Oxidative stress ( ) Enterobactin pathway Respiration ) ( PTS + Glycolysis ( ) TCA ) ( PP Pathway ) ( Specific CO2 production rate Specific glucose-uptake Figure 5 Overall illustration for the effects of cyoA , cydB , fnr , and fur gene knockout on the metabolism based on the fermentation characteristics Intracellular energy senso Glucose PEP soxR/S + PTS ( ptsHI.crr , ptsG ) + - - PYR G6PDH ( zwf ) 6PGL ( pgl ) G6P GL6P Pgi ( pgi ) PTS ¬ + + PG 6 fur ¬ Mlc 6PGDH ( gnd ) X5P E4P F6P rp e rpoS + - Pfk ( pfkA , B ) Fbp ( fbp ) tktA , B + Rub5P talA , B + S7P ¬ F-1 ,6 - BP ¬ Cra rpiA , B Shikim Fba ( fbaA , B ) DHAP Tpi ( tpiA G3P + ) - Gap ( gapA ) + 1,3-PG Pgk ( pgk ) R5 P aroKLAC ate ¬ cyaA Chorismate entCBA ¬ DHB ¬ cAMP Crp entEFDB ¬ 3-PG - Gpm ( gpmA ) + ¬ Enterobactin 2-PG Crp-cAMP Eno ( eno ) PEP Intracellular N - level indicator + Pck ( pck ) Pyk ( pykA , F ) Pps ( ppsA ) AcP ¬ PYR lpdA - KG/Glu Acetate + PDH aceE , F ¬ AcCoA glnALG glnAp2 glnAp1 Ppc ( ppc ) ¬ OAA MDH ( mdh ) CS ( gltA ) Nac nac + ¬ + MAL CT Fum Acn ( acnAB ) ( fumABC ) Fum ICT SDH Frd ( frd ) ICDH NADPH + ( sdhCDAB ) ( icdA ) GDH + Suc KGDH - KG ( sucAB ) SCS ( sucCD ) SucCoA ¬ arcA ¬ + gdhA + ATP GS ADP rpoN NADP + ¬ ¬ fnr + Glutamate Glutamine ¬ + GOGAT gltBD NADP + H O 2 Glutamate - KG NADH NADPH O e ¬ 2 NAD + e ¬ e ¬ ¬ N-limited N-rich + NDH-1 NH + Q-Pool Cytbo Cytbd 4 NDH-II 2H + / e - + ¬ 0H / e ¬ 2H + / e ¬ + 1H + / e ¬ Global regulator Signaling molecules Substrates and singals Complexes Promoters Figure 6 Overall illustration of the gene regulation in terms of global regulators in central carbon metabolism of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_2">
Impact of global transcriptional regula ¬ tion by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose-catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></label>
<input id="rp_similarity_2", type="checkbox">
<div class="RP_similar">
<li>Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose-catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li></div>
<li>Enzyme Microb Technol 38:512 -- 520 Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose-catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>Antonie Van Leeuwenhoek 71 ( 3 ) :217 -- 230 Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose-catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li>
<li>J Bacteriol 179:4138 -- 4142 Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose-catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Mol Microbiol 49:639 -- 654 Gore AL , Payne SM ( 2010 ) CsrA and Cra inXuence Shigella Xexneri pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated Glycolysis , citric-acid-cycle , and respiratory chain genes times following the introduction of air 5 min 10 min 15 min 60 min 6.0 6.7 6.9 4.7 3.6 5.7 6.1 3.2 Relevant regulatory proteins aceE pyruvate dehydrogenase ( decarboxylase component ) aceF pyruvate dehydrogenase ( dihydrolipoyltransacetylase component ) acnA aconitase A acnB aconitase B cyoA cytochrome bo terminal oxidase subunit II cyoB cytochrome bo terminal oxidase subunit I cyoC cytochrome bo terminal oxidase subunit III cyoD cytochrome bo terminal oxidase subunit IV cyoE heme o synthase gltA citrate synthase gpmA phosphoglycerate mutase icdA isocitrate dehydrogenase ( NADP ) lpdA lipoamide dehydrogenase ( E3 monomer ) mdh malate dehydrogenase mqo malate dehydrogenase ( quinone ) ndh NADH dehydrogenase II pdhR pyruvate-responsive transcription regulator poxB pyruvate oxidase sdhA succinate dehydrogenase flavoprotein sdhB succinate dehydrogenase iron-sulfur protein PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) ArcA ( ) FNR ( ) Cra ( ) CRP ( ) SoxRS ( ) ArcA ( ) Cra ( ) CRP ( ) Fis ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA * ( ) Fur ( ) P1 , ArcA * ( ) P2 , Cra * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) Plpd-ArcA * ( ) FNR ( ) CRP ( ) PdhR ( ) Fis ( ) ArcA ( ) CRP ( ) FlhD ( ) ArcA ( ) FNR * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) SoxRS ( ) MarR ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2-CRP ( ) P1 - ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) 2.0 3.1 2.6 2.9 6.8 8.6 9.9 7.3 5.2 2.7 1.9 5.0 3.3 3.2 7.4 3.2 5.8 7.3 9.1 11.7 7.8 6.5 4.7 2.1 6.1 4.9 10.4 12.9 9.2 5.7 4.4 1.5 4.8 8.6 10.0 7.9 6.3 4.0 2.1 5.3 3.0 2.4 4.8 2.9 3.5 3.9 3.4 6.9 4.2 4.6 3.5 8.6 5.0 5.0 2.2 3.5 2.4 6.6 3.1 3.0 4.7 2.1 3.7 6.8 9.9 5.5 9.1 5.8 8.7 4.3 2.9 5.8 2.8 sdhC succinate dehydrogenase membrane protein sdhD succinate dehydrogenase membrane protein sucA 2-oxoglutarate decarboxylase 16.6 4.6 3.2 3.6 7.8 1.9 2.0 2.1 2.1 3.2 3.6 3.1 sucB dihydrolipoamide succinate transferase sucC succinyl-CoA synthetase , subunit sucD succinyl-CoA synthetase , subunit 3.5 4.7 2.7 TABLE 2 Transcripts encoding proteins involved in the oxidative stress response that are present in increased abundance after the introduction of air to anaerobic cultures of E. coli MG1655 Fold regulation ( at least 2-fold at one or more time point ) is the ratio of transcript levels at the indicated time after the introduction of air to the transcript levels at t 0 ( the initial anaerobic state ; p 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>holixsnhgbi , gtdLen.tdom SELE matic search for the Cra-binding promoters using genomic SELEX system Tomohiro Shimada1 ,2 , Nobuyuki Fujita3 , Michihisa Maeda2 and Akira Ishihama1 ,4 , * 1Nippon Institute for Biological Science , Division of Molecular Biology , Ome , Tokyo 198-0024 , Japan 2Meiji University , Faculty of Agriculture , Kawasaki , Kanagawa 214-8571 , Japan 3National Institute of Technology and Evaluation , Genome Analysis Center , Nishihara , Shibuya , Tokyo 151-0066 , Japan 4Hosei University , Faculty of Engineering and Research Center for Micro-Nano Technology , Koganei 184-8584 , Tokyo 184-8584 , Japan Cra ( or FruR ) , a global transcription factor with both repression and activation activities , controls a large number of the genes for glycolysis and gluconeogenesis.To get insights into the entire network of transcription regulation of the E. coli genome by Cra , we isolated a set of Cra-binding sequences using an improved method of genomic SELEX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>After the reporter assay of promoter-lacZ fusions in both wild-type E. coli and cra mutants was performed , all the genes tested were found to be under the control of Cra ( Fig. 3 ; for the gene list , see Table 2 ) , including the newly identiﬁed genes sgrS ( ptsG stabilizer small RNA ) , prpB ( 2-methylisocitrate lyase ) , cyoA ( cytochrome o ubiquinol oxidase ) , marR ( multidrug-resistance regulator ) , zwf ( glucose-6-phosphate dehydrogenase ) , fbaB ( fructose-bisphos-phate aldolase ) , setB ( lactose/glucose efﬂux system ) , maeB ( NADP-linked malic enzyme ) , glgC ( glucose-1-phosphate ad-enylyltransferase ) , gpmM ( phosphoglyceromutase III ) , tpiA ( triosephosphate isomerase ) , ppc ( phosphoenolpyruvate carboxylase ) , and mpl ( UDP-N-acetylmuramate : L-alanyl-gamma-D-glutamyl-meso-diaminopimelate ligase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>( A ) E. coli B ( BL21 ) , ( B ) E. coli B ( BL21 ) cra ¯ Intensity ration ( Log2 ( Cra - / Cra + ) ) Intensity ration ( Log2 ( Cra - / Cra + ) Figure 5 Growth , glucose consumption , conductivity , and acetate production of E. coli K-12 ( JM109 ) in LB media containing 15 g/L NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>This property , together with this strain insensitivity to high glucose concentrations Intensity ration ( Log2 ( Cra - / Cra + ) ) ) ( A 5 ( B ) 5 4 4 fbp pck ppsA fbp pck ppsA 3 2 3 2 1 1 0 0 -1 -2 -1 -2 -3 -4 -3 -4 -5 -5 Late log Mid log Mid log Fermentation stages Fermentation stages Early log Early log Late log Figure 7 Microarray analysis of glucoenogensis related genes in E. coli B ( BL21 ) and E. coli K-12 ( JM109 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>( A ) E. coli B ( BL21 ) , ( B ) E. coli K-12 ( JM109 ) Abbreviations : fbp , fructose-1 ,6 - bisphosphatase ; pckA , phosphoenolpyruvate carboxykinase ; ppsA , phosphoenolpyruvate synthase Intensity ration ( Log2 ( Cra - / Cra + ) Figure 8 Microarray analysis of bet operon genes in E. coli B ( BL21 ) and E. coli K-12 ( JM109 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The survival and growth of bacteria under such changing environments largely depend on the adjustments in the expression proﬁles of several global regulatory proteins , such as RpoD , RpoS , SoxRS , Cra , FadR and IclR ( Aronis 2002a , b ; Cortay et al. 1991 ; Jordan et al. 1999 ; Liochev and Fridovich 1992 ; Lu et al. 2003 ; Moat 2002 ; Tang et al. 2002 ; Varghese et al. 2003 ; Perrenoud and Sauer 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>Bioinformatics 23:2947 -- 2948 Lewis JA , Boyd JM , Downs DM , Escalante-Semerena JC ( 2009 ) Involvement of the Cra global regulatory protein in the expression of the iscRSUA operon , revealed during studies of tricarballylate catabolism in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Polyamines , in particular putrescine , play an important role in various cellular-processes and can affect intracellular concentrations of the regulatory proteins sN , Cra and H-NS , thus impacting global gene-expression ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Later , Cra was characterized as a pleiotropic transcriptional regulator that 3 The abbreviations used are : TF , transcription factor ; EMP pathway , Embden Meyerhof-Parnas pathway ; ED pathway , Entner-Doudoroff pathway ; KDPG , 2-keto-3-deoxy-6-phosphogluconate ; FPA , fluorescence polarization assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>Cra represses pfkA , pykF , and glk , thereby accelerating glycolysis ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Since Cra represses the glycolysis pathway genes such as ptsHI , pfkA , pykF , and activates the gluconeogenic pathway genes such as fbp , ppsA , pckA , etc. , cra gene knockout may activate glycolysis , and the glucose consumption rate may be increased as shown in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Cra also activates pps and pck , thus stimulating gluconeogenesis ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Note that Cra also activates genes that encode gluconeogenesis enzymes ( 319 , 366 , 367 , 389 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Whereas Cra positively regulates the genes that encode the GB and gluconeogenesis ( 319 , 366 , 389 , 439 ) , there exists no evidence that this global regulator exerts a signiﬁcant effect on acs transcription ( Beatty and Wolfe , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Since Cra activated ED pathway , and repressed TCA-cycle and the glyoxylate pathway , we constructed cra.edd.iclR genes knockout mutant to reduce the acetate production by activating the glyoxylate pathway , thus improving the cellular performance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Cra binds to the control regions of target genes and exerts a negative effect on the expression of genes encoding glyco-lytic and Entner-Doudoroff enzymes , while exerting a positive effect on genes encoding Krebs-cycle , glyoxylate shunt and gluconeogenic enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>These results are similar to previous reports that Cra represses transcription of genes that encode glycolytic ( sugar-catabolizing ) enzymes such as key enzymes in the Embden-Meyerhof and Enter-Doudoroff pathways , but activates transcription of genes that encode biosynthetic and oxidative enzymes such as key enzymes in the Krebs-cycle , the glyoxylate shunt , the gluconeogenic pathway , and electron-transfer [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Figure 1 provides an overview of the regulatory effects of Cra on key enzymes in glycolysis , the Entner-Doudoroff pathway , the Krebs-cycle , the glyoxylate shunt and gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>All genes that are negatively regulated by Cra encode glycolytic or Entner-Doudoroff pathway enzymes , whereas all genes that are positively controlled by Cra encode enzymesof the Krebs-cycle , the glyoxylate shunt , or gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>In order to test the possibility that Cra controls transcription of the crp gene , we constructed a chromosomal Pcrp-lacZ fusion in wild type and fruR genetic backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>However , there is extensive overlap as Crp , for example , regulates transcription of most Krebs-cycle enzymes while Cra directly regulates the initiation of the metabolism of some sugars ( e.g. , fructose , allose , etc. ) ( Ramseier et al. , 1993 ; Ramseier et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Although extensive evidence shows that Crp autoregulates expression of its own structural gene ( Hanamura and Aiba , 1992 ; Ishizuka et al. , 1993 ; Tagami et al. , 1995 ) , this is the first evidence that Cra cross controls crp gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Arch Microbiol ( 2011 ) 193:711 -- 722 DOI 10.1007 / s00203-011-0708-7 ORIGINAL PAPER BioWlm formation in Escherichia coli cra mutants is impaired due to down-regulation of curli biosynthesis Shamlan M. S. Reshamwala · Santosh B. Noronha Received : 26 October 2010 / Revised : 18 January 2011 / Accepted : 13 April 2011 / Published online : 11 May 2011 © Springer-Verlag 2011 Keywords Cra · Curli · BioWlm · Consensus sequence Abstract Cra is a pleiotropic regulatory protein that controls carbon and energy Xux in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Effect of Cra on the anaerobic expression of F ( adhE-lacZ ) 291 operon and protein fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></li>
<li>Our results showed that Cra repressed adhE expression 1.5-fold during anaerobic growth on LB-glucose and 4-fold during aerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The stronger repression of adhE by Cra under aerobic than anaerobic growth conditions gives the appearance of a redox control , but in fact , this difference in gene-expression may reﬂect F1P and F16P levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Thus , both HexR and Cra are dual transcriptional regulators that mediate cAMP-independent catabolite control in response to key catab-olite intermediates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>Key words Catabolite regulation · Cra protein · Nitrite reductase regulation · Escherichia coli · Catabolite repression Abbreviations Cra Catabolite repressor-activator protein · nir Genes encoding NADH-dependent nitrite reduction complex · nrf genes encoding the formate-dependent nitrite reduction complex Introduction The generation of ATP is an essential process in all living organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Among them , the catabolite repressor/activator protein ( Cra ) initially characterized as the fructose repressor , FruR plays an important role in the control of carbon ﬂow in E. coli ( Moat et al. 2002 ; Saier and Ramseier 1996 ; Saier et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>In E. coli , two dominant mechanisms involve transcriptional regulation by the cyclic AMP receptor protein ( Crp ) encoded by the crp gene , and by the catabolite repressor/activator ( Cra ) protein , encoded by the fruR gene ( Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Transcription of the gene is under the negative control of the Cra ( catabolite repressor-activator ) protein , whereas translation of the adhE mRNA requires processing by RNase III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>In vivo and in vitro experiments are reported demonstrating that the catabolite repressor-activator ( Cra ) protein ( formerly designated FruR ) regulates expression of the cydAB operon of Escherichia coli encoding cytochrome d oxidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Regulation of expression of the ethanol dehydrogenase gene ( adhE ) in Escherichia coli by the catabolite repressor activator protein Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_42">
J Biol Chem 279:12588 -- 12597 Mikulskis A , Aristarkhov A , Lin ECC ( 1997 ) Regulation of expression of ethanol dehydrogenase gene ( adhE ) in Escherichia coli by catabolite repressor activator protein Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></label>
<input id="rp_similarity_42", type="checkbox">
<div class="RP_similar">
<li>Mikulskis , A. , Aristarkhov , A. & Lin , E.C. ( 1997 ) Regulation of expression of the ethanol dehydrogenase gene ( adhE ) in Escherichia coli by catabolite repressor activator protein Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>[ 14 ] A. Mikulskis , A. Aristarkhov , E.C.C. Lin , Regulation of expression of the ethanol dehydrogenase gene ( adhE ) in Escherichia coli by catabolite repressor activator protein Cra , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li></div>
<li>FruR , also known as , Cra ( Catabolic repressor/activator ) , is a global transcription regulatory protein in enteric bacteria that regulates gene-expression by binding to a specific DNA sequence [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The cra gene encodes a catabolite repressor-activator protein ( Cra ) involved in the modulation of carbon ﬂow in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The catabolite repressor-activator ( Cra ) protein controls the direction of carbon flux through meta-bolic pathways in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_48">
The catabolite repressor / activator ( Cra ) protein of enteric Bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></label>
<input id="rp_similarity_48", type="checkbox">
<div class="RP_similar">
<li>The catabolite repressor / activator ( Cra ) protein of enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753455" target="_blank">10753455</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li></div>
<li>The fru operon is inducible in wild-type E. coli K12 due to the presence of the fructose repressor , FruR , also known as the catabolite repressor/activator ( Cra ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The LacI protein family catabolite repressor/activator , Cra , was initially characterized as the fructose repressor , FruR , because it negatively regulates transcription of the fructose utilization operon ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>Cra works as a repressor of certain glycolytic genes such as ptsHI , pfkA , pykF , and the epd-pgk-fbaA operon and as an activator of some gluconeogenic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>For example , the free ( unliganded ) form of Cra serves as an activator of catabolite repressible , oxidative Krebs-cycle and gluconeogenic enzyme genes , but as a repressor of catabolite activatable , anaerobic glycolytic enzyme genes ( Chin et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Cra is a bifunctional transcription factor , acting as a repressor for most of the targets and as an activator for a small set of targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>A number of the genes for both glycolysis and gluconeogenesis are under the control of catabolite repressor activator Cra , but independent of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The 50S L20 operonb rplT < rpmI infC bamD > raiA Translation inhibitor set of genes that are involved in carbon-utilization for energy production is controlled by two global regulators , CRP ( cAMP receptor protein or catabolite activation protein ) and Cra ( catabolite repressor and activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Besides CRP , another transcription factor , Cra ( catabolite repressor activator ) , has been indicated to play a role in the global regulation of the genes for carbon metabolism ( 16 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The promoter activity for the pps gene encoding phosphoenolpyruvate synthase , however , markedly decreased in the cra mutant ( Fig. 3 ) , indicating that Cra is involved as an activator for the pps operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Cra is a common activator of the gene set ppsA , fbp , pckA , aceA , which are vital for acetate uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Comparative genomic analysis of the Cra ( FruR ) regulon revealed substantial variability in regulon gene content among Enterobacteriales and that it acted as a local regulator of the fructose utilization operon in other - proteobacteria such as Vibrio and Pseudomonas species ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>The Cra ( FruR ) regulons reconstructed in other - proteobacteria ( Aeromonadales , Pseudomonadales , Psychromonadaceae , are Vibrionales ) are implicated in the local control of the fructose utilization operon ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>It involves repression by the GB regulator ICLR and activation by the nucleoid protein IHF and the global regulator Cra ( also known as FruR ) ( reviewed in reference 95 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Recent studies have shown that Cra also regulates other cell processes and inXuences bioWlm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Table 1 Up-regulated genes in E. coli MG1655 cra mutant ( fold change > 1.5 ) Gene Gene product Fold change ( log ) 2 6.680 5.842 4.561 3.411 3.313 fruK fruA Fructose-1-phosphate kinase Fused fructose-speciWc PTS enzymes : IIB component/IIC components D-Erythrose 4-phosphate dehydrogenase Conserved protein Sulphur carrier protein epd ymdF thiS prep 2.644 2.518 2.416 2.362 Predicted propionyl-CoA synthetase with ATPase domain 2-Methylcitrate synthase 2-Methylcitrate dehydratase Fused acetaldehyde-CoA dehydrogenase/iron-dependent alcohol dehydrogenase/pyruvate-formate lyase deactivase Nitrate reductase 2 ( NRZ ) , beta subunit Periplasmic protein , combats stress prpC prpD adhE narY cpxP 1.938 1.849 Phosphoglyceromutase III , cofactor-independent Nitrate/nitrite-transporter Fructose-bisphosphate aldolase , class II Predicted phosphate starvation inducible protein 2-Methylisocitrate lyase gpmM narU 1.736 1.706 1.695 1.639 1.592 fbaA psiE prpB ydfN Qin prophage ; predicted side tail Wbre assembly protein Modulator of gene-expression , with H-NS 16S rRNA pseudouridylate 516 synthase Nitrate reductase 2 ( NRZ ) , gamma subunit 1.568 1.558 1.540 1.524 hha rsuA narV Discussion Curli biosynthesis is regulated by Cra Cra is a global plieotropic transcription factor in enteric bacteria that regulates a number of genes involved in carbon and energy metabolism ( Saier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Cra also regulates the equilibrium reaction of glycolytic pathway by repressing the dicistronic gapB ( epd ) - pgk operon ( Charpentier et al. 1998 ) that encode d-erythrose 4-phosphate dehydrogenase and phosphoglycerate kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Among these ten promoters , six are associated with genes already identiﬁed to be under the control of Cra , including adhE ( aldehyde-alcohol dehydrogenase ) ( Mikulskis et al. 1997 ) , pykF ( pyruvate kinase I ) , fruB ( 6-phosphofructokinase isozyme I ) ( Bledig et al. 1996 ) , ptsH ( phosphocarrier protein HPr [ fructose ] ) ( Ryu et al. 1995 ) , epd ( d-erythrose 4-phosphate dehydrogenase ) , and nirB ( NADH-dependent nitrite reductase large subunit ) ( Tyson et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra proved to bind to the following five E. coli genes : ( i ) pckA , encoding PEP carboxykinase , ( ii ) edd-eda , encoding the two enzymes of the Entner-Doudoroff pathway ( 6-phos-phogluconate dehydratase and 2-keto-3-deoxy-6-phosphogluconate aldolase ) , ( iii ) gapB , encoding erythrose 4-phosphate dehydrogenase , ( iv ) mtlADR , encoding the enzymes of mannitol-catabolism ( enzyme II of the mannitol PTS , mannitol l-phos-phate dehydrogenase and the mannitol operon repressor ) , and ( v ) pykF , encoding one of the two pyruvate kinase isoenzymes ( Ramseier et al. , 1995 ; Bledig et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Cra box-like sequences also exist within the coding regions of some other genes , besides the mppA gene , including the the alsB , garD , hchA , mesJ , pepD , ppx , ptsA , rspB , sun , and rig genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>To investigate this possibility and better understand the function of Cra in the two strains , cra - negative E. coli B ( BL21 ) and E. coli K-12 ( JM109 ) were prepared and their growth behaviour and gene-expression at high glucose were evaluated using microarray and real-time PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>( A ) E. coli B ( BL21 ) , ( B ) E. coli K-12 ( JM109 ) Abbreviations : aceA , isocitrate lyase ; aceB , malate synthase A ; ackA , acetate kinase ; pta , phosphotransacetylase ; poxB , pyruvate oxidase B ; acs , acetyl-CoA synthetase Conclusions To evaluate the role of the global regulator Cra in the central carbon metabolism of E. coli K and E. coli B , the cra gene was deleted and the mutant strains were compared to the parental strains in their gene-expression , growth and metabolic behaviour .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>In the present work , we have investigated the effect of cra deletion on the central carbon metabolism in E. coli B ( BL21 ) and E. coli K-12 ( JM109 ) to verify the above assumption and to understand further the function of Cra in high cell density at high glucose , by following growth kinetics and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>By creating cra deleted mutants of these two strains and by analyzing their growth , metabolism and gene-expression , we anticipated to have more information on the role of the Cra regulating the central carbon metabolism in E. coli , and perhaps be able to create a strain tolerant to high glucose concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Since both Crp and Cra are recognized to regulate gene-expression in response to carbon source availability , either one might regulate cyd gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>We show that Cra regulates growth by activating expression of the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>All of the results presented in Figs. 2 , 3 , 4 and 5 as well as Table 2 are consistent with the conclusion that Cra regulates crp gene-expression and thereby indirectly controls expression of numerous operons involved in the initiation of carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Novel Members of the Cra Regulon Involved in Carbon Metabolism in Escherichia coli † Tomohiro Shimada , Kaneyoshi Yamamoto , and Akira Ishihama * Department of Frontier Bioscience and Micro-Nano Technology Research Center , Hosei University , Koganei , Tokyo 184-8584 , Japan Received 10 October 2010/Accepted 12 November 2010 Cra ( catabolite repressor activator ) is a global regulator of the genes for carbon metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Introduction The phospohoenolpyruvate : carbohydrate transferase system ( PTS ) , in conjunction with Crp and Cra , constitute the main regulatory system in Escherichia coli involved in catabolite carbon repression that allows the utilization of glucose as the preferred carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_78">
Novel members of the Cra regulon involved in carbon metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26303830" target="_blank">26303830</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></label>
<input id="rp_similarity_78", type="checkbox">
<div class="RP_similar">
<li>Shimada , T. , Yamamoto , K. & Ishihama , A. ( 2011c ) Novel members of the Cra regulon involved in carbon metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23301696" target="_blank">23301696</a>]<br></li>
<li>Shimada T , Yamamoto K & Ishihama A ( 2011a ) Novel members of the Cra regulon in carbon metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li></div>
<li>It therefore appears that Crp and the fruR gene product , Cra , cooperate in the regulation of many operons in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>ybaS -1.32 Probable glutaminase ybaS ( EC 3.5.1.2 ) ( e ) Global and metabolic regulatory genes ﬁs DNA-binding protein ﬁs ( factor-for-inversion stimulation protein ) ( HIN recombinational enhancer binding protein ) 1.52 fucR -0.47 lysR 1.51 - Fucose operon activator L Transcriptional activator protein lysR purR -0.45 rpoS 1.62 Purine nucleotide-synthesis repressor RNA polymerase sigma-stationary phase Discussion In Escherichia coli , Crp and Cra are known to control transcriptional responses to carbon availability ( Saier and Ramseier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>In addition to Cra and ArcA/B , other metabolic regulators are involved in acetate-metabolism in E.coli : acetyl-CoA synthetase is under the control of the sigma factors RpoS and RpoD and requires cAMP receptor protein , and the oxygen regulator ( Fnr ) for full expression [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Escherichia coli and related Enterobacteria utilize two global transcription factors ( TFs ) ,3 Crp and Cra ( FruR ) , to control central and peripheral carbohydrate-metabolism ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>In Escherichia coli two global transcriptional regulatory proteins , Crp and Cra , are known to control transcriptional responses to carbon availability [ 28 , 30 ] , while two other global regulators , Fnr and ArcA , control gene-expression in response to oxygen availability [ 14 , 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Cra mediates catabolite repression or catabolite activation depending on the target gene era controls gene-expression in vivo Expression of the negatively Cra-regulated pykF gene and the positively Cra-regulated ppsA gene was recently studied as a function of carbon source Beta-galactosidase activity measurement of singlecopy , chromosomal fusions of these genes to lacZ in E. coli strains wild-type for or lacking the Cra protein showed that Cra controls the response to carbon source independently of the cAMP-cAMP receptor protein ( CRP ) complex ( Bledig et al. , 1996 ; T.M. Ramseier and M.H. Saier , Jr. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Conclusions Over the past six years , extensive experimental data have accumulated showing that Cra is a global transcriptional regulatory protein which controls the expression of numerous genes encoding key enzymes of virtually every major pathway of carbon metabolism in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Introduction Catabolite repressor/activator ( Cra ) protein ( also known as FruR ) is a pleiotropic regulatory agent that inXuences , in a cyclic AMP-independent manner , the relative activities of every major pathway concerned with carbon and energy metabolism in enteric bacteria ( Saier 1996 ) , as also genes involved in other processes ( Lewis et al. 2009 ; Ow et al. 2007 ; Shimada et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Discussion Cra ( formally designated the fructose repressor FruR ) is a key regulator controlling the balance between glycolysis and gluconeogenesis ( Saier 1996 ; Saier & Ramseier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>FruR ( Cra ) is a transcriptional regulator which , together with CRP , controls the expression of genes involved in C metabolism , particularly with respect to the direction of metabolic flow , glycolysis being repressed whereas gluconeogenesis is activated ( Ramseier et al. , 1993 ; Saier & Ramseier , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Cra was initially characterized as the fructose repressor ( FruR ) , but was later recognized as a pleiotropic regulator that controls transcription of the genes in major pathways for carbon and energy metabolism ( Ramseier et al. 1993 , 1995 ; Ramseier 1996 ; Saier 1996 ; Saier & Ramseier 1996 ; Saier et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The fructose repressor FruR ( Cra ) , which plays a pleiotropic role in the transcriptional regulation of many genes encoding enzymes of carbon and energy metabolism ( Saier 1996 ) , was thought to be a possible candidate for this putative repressor because the pts promoter region contains a FruR-binding site ( Ryu et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>In reﬂecting the physiological role of GapA as a real GAPDH , the promoter organization is very unique , being controlled by two global regulators , CRP and Cra , for a large set of genes for carbohydrate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Results Two global regulators , CRP and Cra , of E. coli recognize and regulate as many as 378 and 164 promoters , respectively , of the genes for carbon source utilization ( Shimada et al. , 2005 , 2011a , 2011b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Moreover , the differences in the regulation targets and regulatory roles of two global regulators , CRP and Cra , in the control of a number of genes for carbon source utilization are totally left undetermined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>This finding raises the need for reevaluation of the differential roles of two global regulators , CRP and Cra , in the overall regulation network of carbon source metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The HexR regulon in Shewanella can be considered as a partial functional replacement of the Enterobacterial Cra ( FruR ) regulon that plays a pleiotropic role , modulating the direction of carbon flow through the different metabolic pathways of energy metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>Taken together , we propose that Cra plays a key role in balancing the levels of the enzymes for carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>While the cyclic AMP-CRP complex primarily controls the initiation of carbon source utilization , Cra primarily influences the direction of carbon metabolic flux within the central pathways of carbon metabolism by regulating the levels of key enzymes in these pathways [ 26 , 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>A balanced aerobic growth on glycerol depends on three global regulators : cAMP-CRP as the principal inducer of the glycerol catabolic regulon ( including glpF , glpK and glpD ) ; Cra ( FruR ) as regulator of some gluconeogenic genes , and ArcA as regulator of several central metabolic genes including the TCA-cycle and others involved in respiration [ 1,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Genes encoding key enzymes concerned with oxidative and gluconeogenic carbon flow are activated by era while genes encoding key enzymes involved in fermentative carbon flow are repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Target genes encoding enzymes concerned with fermentative carbon flow ( glycolysis and Enmer-Doudoroff pathway ) are repressed by Cra , while target genes involved in oxidative and gluconeogenic carbon flow ( Krebs-cycle , glyoxylate shunt and gluconeogenesis ) are activated by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>We have recently found that Cra influences the expression of many cyclic AMP-Crp activatable genes that initiate carbon metabolism even though no Cra binding site could be identified in the control regions of the operons encoding the relevant transport systems and metabolic enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>This mechanism of Cra action differs from those examined previously for regulation of the expression of genes encoding enzymes of carbon metabolism [ 25 , 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Cra is involved in regulation of the whole set of enzymes in central carbon metabolism ( 43 ) , while CRP regulates the genes for carbon source utilization outside the central metab-olism , including transport of various carbon sources , glycolysisgluconeogenesis switching , the tricarboxylic acid ( TCA ) cycle , the pyruvate-dehydrogenase-pathway , and aerobic-respiration ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The majority of Cra targets are the genes coding for the enzymes involved in central carbon metabolism , covering all the genes for the enzymes involved in glycolysis and metabolism downstream of glycolysis , including the tricarboxylic acid ( TCA ) cycle and aerobic-respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>This library was used to search for DNA sequences recognized by Cra , a global regulator of genes involved in carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>We then proposed a novel model that Cra is another global regulator of the genes for carbon metabolism , playing a role in balancing the level of enzymes involved in glycolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Apparently , another global carbon regulator , Cra ( also known as FruR ) , also is not involved ( 339 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>FruR ( also known as Cra ) regulates the expression of many genes involved in carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>For the identiﬁcation of as-yet-un-identiﬁed genes under the control of Cra , we developed the genomic SELEX ( systematic evolution of ligands by exponential enrichment ) system and isolated a number of Cra-binding sequences from mixtures of E. coli genome fragments ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The integration of the genes encoding as many as 33 transcription factors into the Cra regulon indicates that a number of E. coli genes are under the indirect control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The genes encoding transcription factors that are organized in the Cra regulon are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The genes that most clearly showed dual control of catabolite repression by Crp and Cra were gat , mgl , tre , and dsd ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>The genes shown in blue boxes are under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>After genomic SELEX screening , we discovered that a total of 33 genes encoding transcription factors are under the control of Cra , including 11 regulators of the genes for carbon metabolism , 4 regulators of the genes for nitrogen metabolism , 9 regulators for stress response genes , and 7 putative regulators with unidentiﬁed regulatory roles ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The marked increase in the numbers of target genes under the control of Cra also indicates that the regulation targets for each of a total 300 species of transcription factors in E. coli ( 16 , 17 ) must be more than those identiﬁed so far , and genomic SELEX screening should be a useful experimental system for searches of the whole set of regulation targets for each transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Many genes involved in both pathways are known to be under the control of Cra.Yet the whole set of E. coli genes , which are regulated by Cra , has not been identiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In order to gain insight into genome regulation by Cra for the control of carbon metabolism in E. coli , we tried to identify the whole set of target promoters , genes , and operons under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The presence of different promoters in these genes could Remarkably , few general transcription regulators have been reported to be involved in modulating the expression of most glycolytic genes - Cra and Crp , among others - especially at the upper part of the glycolytic pathway involved in the transformation of glucose-6P into pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Therefore , neither Crp in the case of gapA nor Cra for pykF and the epd-pgk-fbaA operon were apparently significantly involved in modulating the expression of these genes in the tested conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>It has been shown that cra disruption causes pyruvate accumulation , which is caused by the increased transcription of glycolysis genes , since transcription from such genes is negatively regulated by Cra ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24212579" target="_blank">24212579</a>]<br></li>
<li>In agreement with the prediction of metabolic switching by Cra , both enzymes are involved in the equilibrium reactions in the pathways of glycolysis and gluconeogenesis ( Harris & Waters 1976 ; Cunningham et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Lower panel : Most of the genes for the enzymes involved in glycolysis are controlled by Cra [ 30 ] while only three genes , fbaA , gapA and pgk , had been identified as the regulation targets of CRP ( shown in while under blue background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Cra has been indicated to regulate all these rate-controlling genes and is thus considered to play a role in switching between glycolysis and gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Later , FruR was indicated to be involved in the regulation of other genes encoding enzymes involved in the catabolic pathways of both glycolysis and gluconeogenesis and was later renamed Cra ( 38 -- 41 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The genes carrying the repression-type promoters are mostly involved in the metabolic pathway of glycolysis while the genes with the activation-type promoters are involved in gluconeogenesis.The molecular switching of Cra distribution is a critical factor for metabolic switching between glycolysis and gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The promoter assay indicated that Cra represses the newly identiﬁed genes that are involved in glycolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In addition , Cra was found to repress the genes for the energy production pathway downstream of glycolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In fact , Cra regulates the expression of a number of genes from glycolysis , TCA-cycle , GS and gluconeogenesis and plays an important role in the control of carbon fluxes [ 42-45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19852855" target="_blank">19852855</a>]<br></li>
<li>Heretofore , it has not been suspected that regulation by Cra extended also to operons involved in terminal fermentation pathways , e.g. , reduction of acetyl coenzyme A into acetaldehyde and then into ethanol by AdhE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Negative control is exerted by Cra on a number of operons involved in glycolysis permitting derepression of the glycolytic pathways by F1P and/or F16P2 in the presence of exogenous six-carbon compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Positive control is exerted by Cra over a number of operons involved in gluconeogenesis , permitting induction of the gluconeogenic pathways by F1P and/or F16P2 in the presence of nonfermentable compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Cra and the control of carbon flux via metabolic pathways T.M. Ramseier Department of Biology , University of California at San Diego , La Jolla , California 92093-0116 ( USA ) Summary Weisser , P. , Krtimer , R. , Sahm , H. & Sprenger , G.A. ( 1995 ) , Functional expression of the glucose-transporter of Zymomonas mobilis leads to restoration of glucose and fructose up-take in Escherichia coli mutants and provides evidence for its facilit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Mol Microbiol 28:1187 -- 1197 Ramseier TM ( 1996 ) Cra and the control of carbon Xux via metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_136">
Cra and the control of carbon ﬂux via metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></label>
<input id="rp_similarity_136", type="checkbox">
<div class="RP_similar">
<li>Ramseier , T.M. ( 1996 ) Cra and the control of carbon ﬂux via metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Ramseier TM ( 1996 ) Cra and the control of carbon flux via metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li></div>
<li>Among a number of PTS transport systems for sugars ( 27 ) , the ptsI gene , encoding PTS-E1 , is under the control of Cra ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In addition , we identiﬁed PTS-E2 ( PtsG ) for glucose-transport as an additional target of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>So far , we have identiﬁed four new Cra target sequences , including two genes ( eno and gapA ) with known functions , and two unidentiﬁed genes ( hypF and yahA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>As a shortcut approach to identifying the whole set of targets under the control of Cra , we have subjected a mixture of genomic SELEX fragments to chip analysis using an E. coli tilling array and identiﬁed a total of 164 target candidates under the control of Cra , including 20 known targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Cra binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>So far , we have identiﬁed four new Cra target sequences , including two genes ( eno and gapA ) with known functions , and two unidentiﬁed genes ( hypF and yahA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Table 1 DNA fragments isolated by SELEX Fragment size ( bp ) Number of clones Location of SELEX fragment Cra box Regulatory function 109 126 64 + fruB < S > setB epd < S < yggC cysK > S > ptsH gapA < S < yeaA adhE < S > ychE eno < S < pyrG yahA < S < betT nirB < S < cytX pykF < S > ydhZ hypF < S > ascF fruB repression epd repression ptsH repression unidentiﬁed adhE repression unidentiﬁed unidentiﬁed nirB repression pykF repression unidentiﬁed 43 12 11 10 8 + + + + 5 + 3 + 2 + 1 + 1 + A total of 97 DNA fragments have been isolated by the genomic SELEX and sequenced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra , which recognition site is located 197 nucleotides upstream the ATG initiation codon [ 49 , Regulon DB http://www.ccg.unam.mx] or some other factor , could play a role in the utilization of these putative promoters in which no clear s 210 recognition elements were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>For isolation of DNA sequences that are recognized by E. coli Cra protein , genomic DNA fragments were regenerated by PCR using a pair of primers , which bind to the vector sequences at the 5 ′ and 3 ′ vector-insert junctions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>For the genomic SELEX screening of Cra-binding sequences , 5 pmol of DNA fragments and 10 pmol His-tagged Cra were mixed in a binding buffer ( 10 mm Tri-HCl , pH 7.8 at 4 ° C , 3 mm Mgacetate , 150 mm NaCl , BSA 1.25 µg / mL ) and incubated for 30 min at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>For this purpose , we employed `` genomic SELEX '' screening , in which puriﬁed His-tagged Cra was mixed with a collection of E. coli genome fragments of 200 to 300 bp in length and Cra-bound DNA fragments were afﬁnity puriﬁed for the identiﬁcation of Cra recognition sequences ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>This library was used to search for DNA sequences recognized by Cra , a global regulator of genes involved in carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>DNA band migration retardation experiments coupled with restriction enzyme digestion of the labelled DNA fragments were conducted to ascertain the ability of Cra to bind in vitro to the operator-like sequences of all the elucidated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>The molecular mechanism by which CcpA activates transcription is not known ; although its E. coli ortholog Cra ( 367 , 389 ) activates transcription of ppsA , which encodes a gluconeogenic enzyme , from a site located at position 45.5 ( 319 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The increase in CitB-binding afinity due to acetylphosphate correlated with the number of independent isolates ( Table 1 ) , as observed in the case of Cra .9 ) The putative CitB-binding sequences among three SELEX fragments ( S1 to S3 ) were found to be high in A/T content , as in the case of K. pneumoniae CitB .12 ) Taking this together we predicted that at least three regions exist in the E. coli genome that speciﬁcally bind pCitB , between mdh and argR ( S1 ) , between uxaC and exuT ( S2 ) , and between citC and citA ( S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18997424" target="_blank">18997424</a>]<br></li>
<li>Black bar above the DNA sequences represents the respective SELEX fragment , in which a single Cra box was identiﬁed.The numbers represent the positions from the respective translation initiation codon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The 186 bp-long SELEX fragment corresponds to the sequence between − 589 and − 774 from the translation initiation codon , and contains a single Cra box , which overlaps with P3 promoter.As expected from the gene organization , transcription repression was observed , albeit at lower extent than yahA repression , for both P3 and P2 promoters ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Communicated by : Hiroji Aiba * Correspondence : E-mail : aishiham@nibs.or.jp DOI : 10.1111 / j.1365-2443.2005.00888.x © Blackwell Publishing Limited Previously , Negre et al. ( 1996 ) performed a systematic search for the Cra-binding sequences from a pool of synthetic oligonucleotides with all possible sequences using the CAST ( cyclic ampliﬁcation and selection of targets ) procedure , which was developed by Norby et al. ( 1992 ) and Wright & Funk ( 1993 ) [ note that CAST is essentially the same approach as SELEX ( systematic evolution of ligands by exponential enrichment ) ( Tuerk & Gold 1990 ) ] , and identiﬁed 20 imperfect palindromic DNA sequences , indicating that the Cra tetramer interacts asymmetrically with two half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>For pfk , encoding phosphofructokinase andfbp , encoding fructose 1,6-bis-phosphatase , no Cra binding site hasbeen identified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Effector-free Cra has been indicated to bind the 14-bp-long Cra box with an imperfect palindromic sequence , GCTG AAACGTTTCA ( 16 , 31 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>By using a collection of 500-bp sequences centered on each peak and BioProspector ( ) , which was successfully employed previously for the identiﬁcation of the RutR-box sequence ( 47 ) , we identiﬁed a 14-bp Cra-box motif from all 167 Cra targets ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Consensus sequence for Cra binding From the ten Cra-binding sequences ( six newly identiﬁed in this study and seven known Cra-box sequences ) , we can propose a complete palindromic sequence , TGAAAC-GTTTCA , with an extended GC at the 5 ′ end , for the typical Cra box ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Consensus recognition sequences of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Consensus sequences of Cra binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In order to identify Cra binding sites in the E. coli genome , experimentally-determined 16-base Cra binding site sequences from the following genes were used as input for the GRASP DNA program ; fruB , icdA , aceB , epd , ppsA , mtlA , pykF , ptsH , nirB , fbp , pckA , adhE and edd ( Schilling et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>These catabolites prevent Cra from binding to its operator sites in the regulatory regions of target operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Sugar catabolites tend to bind to Cra and displace it from the operator sites in target operons [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The Cra protein binds to a speciﬁc DNA sequence ( 23 ) , the consensus of which was successively reﬁned with studies of additional Cra-controlled promoters ( 7 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The in vitro DNA retardation assay showed that Cra binds to a region on the 59 side of the ORF , as predicted by the consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>FruR , also known as , Cra ( Catabolic repressor/activator ) , is a global transcription regulatory protein in enteric bacteria that regulates gene expression by binding to a specific DNA sequence [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>Results of DNA band shift assays demonstrating Cra and Cra * binding to the adhE and fruB promoter regions but no binding to the cra gene region lacking the Cra-binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>SpeciWc binding of puriWed His-tagged Cra protein to a 200-base pair DNA fragment encompassing the region upstream of the coding sequence of csgB conWrmed the presence of a Cra-binding motif in the region ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Electrophoretic mobility shift assays were also performed to demonstrate binding of puriWed Cra protein to Cra-bind-ing motifs upstream of the csgD coding sequence ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>c Single Cra targets located downstream of Cra-binding sites ( group 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Site-directed mutagenesis of three Cra-binding sites in the promoter region of the csgDEFG operon suggests that Cra activates transcription of this operon upon binding to operator regions both downstream and upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Alternatively , Cra may act as a positive transcription regulator by binding to its operator region located downstream of the csgB transcription start site ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Appendix B Method for ﬁnding the Cra-binding sites The method of Berg and von Hippel ( 1987 ) was implemented in Matlab to score the E. coli genome sequence with the Cra weight matrix developed from foot-printed promoter regions of fourteen Cra-controlled operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>The Z value for the mean score li of the Cra rec-ognition sites of the 14 input foot printed binding sites is also given in Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Cra activates transcription of csgDEFG operon on binding to its operator sites The negative strand of the curli intergenic region was cloned upstream of a promoterless gfp reporter gene , and the contribution of each of the three Cra-binding sites identiWed upstream of the csgD coding sequence to the regulation of the csgDEFG operon was studied by mutagenizing conserved bases in the Cra-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>The degeneracy of the era binding conseasus sequence correlates well with the pleiotropic function of Cra , i.e. the necessity to recognize numerous operator sites in the genomes of enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_37">
The degeneracy of the Crabinding consensus sequence is considered to correlate well with the pleiotropic function of Cra , i.e. , the need to recognize a number of operator sites in the genome ( Ramseier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></label>
<input id="sit_similarity_37", type="checkbox">
<div class="SIT_similar">
<li>The degeneracy of the Cra-binding sequence correlates well with the pleiotropic function of Cra , i.e. , the need to recognize a number of operator sites in the genome ( Ramseier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li></div>
<li>Also shown is the similarity between the consensus sequence for Cra binding sites and the putative Cra binding site located between bases 10 and 23 upstream of the transcription start in the nir promoter , and ( not to scale ) the EcoRI and HindIII restriction sites used to generate the 186 bp DNA fragment for the mobility shift assays Fig. 2 A , B Effects of changes in the composition of the growth medium and cra on expression from the nir and nrf promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Within the nir promoter sequence , from -- 23 bp to -- 10 bp with respect to the transcription start , there is an im perfect palindrome , ( GC ) TGAATCGTTAAG , that is identical to the proposed consensus for Cra binding sites , ( GC ) TGAAACGTTTCA , at 10 out of 14 positions ( Fig. 1 ) ( Ramseier et al. 1993 , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>As in the case of other Cra binding sites , the upstream half of the palindrome corresponds better to the consensus than the downstream half ( Cortay et al. 1994 ; Ramseier et al. 1995 ) , which , at the nir promoter , overlaps the -- 10 hexamer and , thus , must be subject to additional sequence constraints .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>This putative Cra-binding motif , however , differed from the Cra consensus by two conserved nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>This finding agrees with elaborate footprinting studies which established that Cra binds asymmetrically to its operator sites , it was found that era binds predominantly to the first half-site of the era binding consensus sequence ( Cortay et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Elucidation of six Cra binding sites in the investigated genes ( i.e. two operator sites in the control region of the fru operon and one operator site in each of the other above mentioned genes ) allowed formulation of a preliminary Cra operator consensus sequence ( Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Formulation of a Cra binding consensus sequence The elucidation of eleven Cra operator sites allowed the formulation of a era binding consensus sequence : RS [ TGAAWC I SNTHHWI where R = AorG , S = C orG , W = A orT , H = A or C or T , and N = any nucleotide ( Ramseier et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>The Cra binding site overlaps one of the two Crp binding sites , the upstream CrpI site , by two nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>The two Crp binding sites ( CrpI and CrpII ) are indicated by lines above the sequence while the Cra binding site is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Since we obtained a reﬁned consensus sequence of Cra binding ( Fig. 2 ) , the location of Cra-binding sites around the newly identiﬁed target promoters was also analyzed after an in silico search for the Cra-box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>We found a DNA sequence similar to the consensus for Cra binding downstream from the adhE promoter ( Fig. 2 ) , consistent with adhE repression by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The position of this sequence relative to a promoter determines Cra function : binding sites upstream from the promoter for activation and binding sites downstream from the promoter for repression ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The binding site of Cra on some of the known targets listed in RegulonDB were experimentally identiﬁed by gel shift and DNase I footprinting assays , but in most cases , the Cra-binding sites were predicted on the basis of the location of the Cra consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Mutations introduced in the Cra binding site are labeled with letters above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>This consensus sequence was used to screen the DNA data-bases for potential , unidentified Cra binding sites in the E. coil and S. typhimurium genomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Examination of the aligned operators revealed that the consensus sequence for Cra binding is the oame for operons that are activated and repressed by era .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Based on the Cra-binding sites identiWed in this and other studies , the Cra consensus sequence is reWned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Figure 3 The consensus sequence for Cra binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Lane 1 DIG-labelled probe ; lane 2 DIG-labelled probe plus Cra ; lane 3 DIG-labelled probe plus Cra plus unlabelled oligo containing the fruB O2 Cra-binding site ( 5 - GAGGCTGAATCGTTTCAATTC ) ; lane 4 DIG-labelled probe plus Cra plus unlabelled oligo containing the csgB Cra-binding site ( 5 - CAAGTTGAAATGATTTAATT ) ; lane 5 DIG-labelled probe plus Cra plus unlabelled oligo containing mutated Cra-binding site ( 5 - CAATTCACAATGACTGAATT ) O 1 0 10 20 30 40 50 60 70 80 O 2 0 10 20 30 40 50 60 70 80 90 O 3 ReWnement of the Cra-binding consensus sequence Cra binds asymmetrically to a degenerate , imperfect palindromic consensus sequence of 14 nucleotides ( RSTG AAWCSNTHHW ) ( Ramseier et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Cra is a DNA binding protein Extensive protein-DNA binding studies which included DNA band mobility shift assays and vari ¬ ous footprinting techniques revealed that the purified Cra protein recognizes and binds asymmetrically to specific operator sites in the control regions of several target genes in vitro ( Ramseier et al. , 1993 , 1995 ; Cortay et al. , 1994 ; Ryu et al. , 1995 , Bledig et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>In addition , the putative Cra site at the nir promoter also contains the GC motif , located immediately upstream of the inverted-repeat , that has been shown to be of prime importance for high-affinity Cra binding in vitro ( Cortay et al. 1994 ; Ramseier et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by Cra</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>JOURNAL OF BACTERIOLOGY , Nov. 1997 , p. 7129 -- / 97 / $ 04.0010 Copyright © 1997 , American Society for Microbiology Regulation of Expression of the Ethanol Dehydrogenase Gene ( adhE ) in Escherichia coli by Catabolite Repressor Activator Protein Cra ALVYDAS MIKULSKIS , † ALEXANDER ARISTARKHOV , ‡ AND EDMUND C. C. LIN * Department of Microbiology and Molecular Genetics , Harvard Medical School , Boston , Massachusetts 02115 Received 30 June 1997/Accepted 4 September 1997 The adhE gene encodes ethanol dehydrogenase and is located at min 27.9 of the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Under the genomic SELEX conditions employed ( protein/DNA input molar ratio of 2 , two cycles of SELEX and in the absence of inducers ) , we failed to detect the promoters ( icdA , aceB , ppsA , acnA and pckA ) , which are known to be activated by Cra.We then prepared the Cra box-containing DNA fragments from these activationtype promoters , and measured the Cra-binding afﬁnity in the absence of inducers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Cra also regulates the equilibrium reaction of glycolytic pathway by repressing the dicistronic gapB ( epd ) - pgk operon ( Charpentier et al. 1998 ) that encode d-erythrose 4-phosphate dehydrogenase and phosphoglycerate kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The cydA , B operon is known to be controlled by ArcA and Fnr in response to oxygen limitation , and also controlled by Cra by carbon source availability ( Ramseier et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Although it has been reported that cyd operon is regulated by the interdependency of Cra , Fnr , and ArcA ( Ramseier et al. 1996 ) , the down-regulation of this operon was due to Cra in the present case since fnr and arcA changed little for the continuous culture at the present aerobic condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Microarray analysis of wild-type and mutant E. coli cultivated in conditions promoting bioWlm formation revealed that the curli biosynthesis genes , csgBAC and csgDEFG , are poorly expressed in the mutant , suggesting that transcription of genes required for curli production is regulated by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>When wild type E. coli grows fast aerobically on glucose , such as strain JM101 , Cra whose DNA binding activity is modulated by sugar catabolites ( fructose-1-P and fructose-1-6P ) , does not significantly repress glycolytic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>These results can be explained by the fact that sugar catabolites present at higher concentrations when growing rapidly on glucose , bind to Cra and displace it from the operator sites in target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Beta-galactosidase assays with the constructed strains revealed that Cra positively regulates gene expression of ppsA ( 11-fold ) and pckA ( 5-fold ) , whereas era negatively regulates gene expression of gapB ( 4-fold ) , edd ( 3-fold ) , mtlA ( 2-fold ) and pykF ( 3-fold ) ( Ramseiereral .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Seventeen of these carbon sources were oxidized more slowly or not at all by the era mutant ( carbon sources positively controlled by Cra ) whereas nineteen were oxidized more rapidly ( carbon sources negatively regulated by Cra ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>The genes such as ptsHI , pfkA , pykF , acnB , edd -- eda , fruBKA , mtlADR , gapB are reported to be negatively controlled , while ppsA , fbp , pckA , acnA , icd , aceA , aceB , and cydA , B are positively controlled by the Cra protein ( Saier and Ramseier 1996 ; Cunningham et al. 1997 ; Moat et al. 2002 ; Perrenoud and Sauer 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Cra has been known to affect positively and negatively the transcription of several other genes such as acnA/B ( aconitase ) , icdA ( isocitrate dehydrogenase ) , eda ( 2-keto-3-deoxy-6-phosphogluconate adolase ) , edd ( 6-phosphogluco-nate dehydrase ) , pfkA ( phosphofructokinase ) , and pykF ( pyruvate kinase isoenzymes ) , but the transcription patterns of these genes in this work were not significantly different between the two strains ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The molecular mechanism by which CcpA activates transcription is not known ; although its E. coli ortholog Cra ( 367 , 389 ) activates transcription of ppsA , which encodes a gluconeogenic enzyme , from a site located at position 45.5 ( 319 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Cra activates transcription of csgDEFG operon on binding to its operator sites The negative strand of the curli intergenic region was cloned upstream of a promoterless gfp reporter gene , and the contribution of each of the three Cra-binding sites identiWed upstream of the csgD coding sequence to the regulation of the csgDEFG operon was studied by mutagenizing conserved bases in the Cra-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Site-directed mutagenesis of three Cra-binding sites in the promoter region of the csgDEFG operon suggests that Cra activates transcription of this operon upon binding to operator regions both downstream and upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Alternatively , Cra may act as a positive transcription regulator by binding to its operator region located downstream of the csgB transcription start site ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Considering what is currently known about Cra regulation , and the difference between E. coli K-12 ( JM109 ) and E. coli B ( BL21 ) in regard to glucose metabolism , it was suggested that in E. coli B ( BL21 ) cra is constitutively expressed , which explains why acetate is not accumulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>To investigate this possibility and better understand the function of Cra in the two strains , cra - negative E. coli B ( BL21 ) and E. coli K-12 ( JM109 ) were prepared and their growth behaviour and gene expression at high glucose were evaluated using microarray and real-time PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>By creating cra deleted mutants of these two strains and by analyzing their growth , metabolism and gene expression , we anticipated to have more information on the role of the Cra regulating the central carbon metabolism in E. coli , and perhaps be able to create a strain tolerant to high glucose concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>These Cra boxes are located within the Cterminal portion of the upstream gene , pyrG encoding CTP synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Among a number of PTS transport systems for sugars ( 27 ) , the ptsI gene , encoding PTS-E1 , is under the control of Cra ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Since Cra represses the glycolysis pathway genes such as ptsHI , pfkA , pykF , and activates the gluconeogenic pathway genes such as fbp , ppsA , pckA , etc. , cra gene knockout may activate glycolysis , and the glucose consumption rate may be increased as shown in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>From DNA microarray data for cra gene knockout mutant , the Cra protein might be responsible for the repression of not only the fru and mlt operons but also other operons such as fuc and xyl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Moreover , since Cra positively regulate ace operon , the glyoxylate pathway is repressed in cra gene knockout mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>As the curli biosynthesis operons , csgBAC and csg-DEFG , are highly repressed in the cra mutant ( Table 2 ) , it was hypothesized that Cra may directly regulate expression of these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>FruR , also known as , Cra ( Catabolic repressor/activator ) , is a global transcription regulatory protein in enteric bacteria that regulates gene expression by binding to a specific DNA sequence [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The Cra protein encoded by the cra gene is known to act as a repressor of adhE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>Mikulskis , A. , Aristarkhov , A. & Lin , E.C. ( 1997 ) Regulation of expression of the ethanol dehydrogenase gene ( adhE ) in Escherichia coli by catabolite repressor activator protein Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Transcription of the gene is under the negative control of the Cra ( catabolite repressor-activator ) protein , whereas translation of the adhE mRNA requires processing by RNase III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>In vivo and in vitro experiments are reported demonstrating that the catabolite repressor-activator ( Cra ) protein ( formerly designated FruR ) regulates expression of the cydAB operon of Escherichia coli encoding cytochrome d oxidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>The operon is also positively controlled by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex and negatively by the catabolite repressor / activator ( Cra ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The LacI protein family catabolite repressor/activator , Cra , was initially characterized as the fructose repressor , FruR , because it negatively regulates transcription of the fructose utilization operon ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>Cra is reported to repress adhE gene in the aerobic condition ( Mikulskis et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18648770" target="_blank">18648770</a>]<br></li>
<li>Transcription regulation of the gapA gene by Cra Figure 4 Identiﬁcation and characterization of the gapA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Transcription regulation of the eno gene by Cra Previously , the eno gene was cloned and its expression was measured by Klein et al. ( 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Although extensive evidence shows that Crp autoregulates expression of its own structural gene ( Hanamura and Aiba , 1992 ; Ishizuka et al. , 1993 ; Tagami et al. , 1995 ) , this is the first evidence that Cra cross controls crp gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Cra activates expression of the crp gene when xylose , glycerol ± CAA or gluconate was used as the sole carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Regulation of crp gene expression by Cra and Crp is therefore suggested to be part of a single regulatory mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Neither Crp nor Cra alone can fully activate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>Cra and Crp both activate expression of the crp gene , apparently by a mechanism that requires the simultaneous presence of both proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>We show that Cra regulates growth by activating expression of the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>We here document this phenomenon and provide compelling evidence that it is due to the fact that Cra positively controls expression of the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923415" target="_blank">24923415</a>]<br></li>
<li>To conﬁrm transcription repression of the eno gene by Cra , both F1 and F12 fragments were inserted into the promoter assay TFP vector and measured the GFP expression level relative to RFP level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>As can be seen in figures 2f-1 and S1f-1 transcription from the epd gene initiated in all strains from the same reported promoter whose expression is controlled by Crp and Cra [ 45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Cra ( 24 ) , it is possible that in the absence of this global regulator , Fis is no longer required for adhE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>These results indicated that expression of the adhE gene was regulated transcriptionally by Cra and posttranscriptionally by RNase G. 2002 Elsevier Science ( USA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>It is likely that the expression of adhE is de-repressed by the adhC81 mutation due to the afinity of Cra to the Cra-box being decreased by the G !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>Our results showed that Cra repressed adhE expression 1.5-fold during anaerobic growth on LB-glucose and 4-fold during aerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Effect of Cra on the anaerobic expression of F ( adhE-lacZ ) 291 operon and protein fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></li>
<li>Fis dependence of adhE expression in the presence or absence of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>Thus , Fis is required for adhE expression independently of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>]<br></li>
<li>This result again conﬁrmed that repression of the adhE gene by Cra was canceled by the adhC81 mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12083772" target="_blank">12083772</a>]<br></li>
<li>We suggest that Cra helps to tighten the control of adhE transcription under aerobic conditions by its repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>The stronger repression of adhE by Cra under aerobic than anaerobic growth conditions gives the appearance of a redox control , but in fact , this difference in gene expression may reﬂect F1P and F16P levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>In earlier studies we found that cydAB operon expression is not influenced by Crp but is controlled by Cra ( A.M. Chin , R.B. Gennis , and M.H. Saier , Jr. , unpublished results , cited in ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Nevertheless , downregulation of these genes may be due in part to Cra repression ; however , pykF and epd-pgk-fbaA transcription in both PTS strains was only slightly downregulated as compared to 2 JM101 , suggesting , a minor role of Cra in the expression of these genes [ 66,67 ] in the tested conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>The possible role of Cra in modulating the expression of pfkA and pykF genes and the epd-pgk-fbaA operon downregulated in PB11 , could be part of a more general mechanism , in which due to the absence of PTS , not only the Cra regulated genes but most glycolytic genes were downregulated in PB11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Therefore , neither Crp in the case of gapA nor Cra for pykF and the epd-pgk-fbaA operon were apparently significantly involved in modulating the expression of these genes in the tested conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Of these three targets , the involvement of Cra in expression regulation was experimentally identiﬁed only for the pck operon ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The cra deletion affected the bet operon expression only in E. coli K-12 ( JM109 ) and not in E. coli B ( BL21 ) ; it is possible that Cra affects the bet operon indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>The cra mutation significantly reduced cyd operon expression under both microaero-philic and anaerobic conditions , suggesting that Cra serves as an activator of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>Although Cra box-like sequences exist upstream from promoters of the adk , aldH , degP , infC , pasB , rseB , sdaB , and sodC genes ( data not shown ) , it is unlikely that the Cra box-like sequences play regulatory roles in transcription of these genes because these sequences are located far from the respective initiation codons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Likewise , all three genes , aceE ( PDH decarboxylase component E1 ) , lpd ( lipoamide dehydrogenase , PDH component E3 ) , and aceF ( PDH dihydrolipoyltransacetylase component E2 ) , coding for the components of the PDH complex were all found to be repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The genes that most clearly showed dual control of catabolite repression by Crp and Cra were gat , mgl , tre , and dsd ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>Regulation of the eno and gapA genes by Cra was conﬁrmed in both in vivo and in vitro transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Transcription regulation of the yahA and hypF genes by Cra Transcription in vitro of the gapA and eno genes was repressed by Cra in dose-dependent ways ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In the case of gapA and eno transcription in vitro , transcription was almost completely repressed by the addition of 5 pmol Cra ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>It has been shown that cra disruption causes pyruvate accumulation , which is caused by the increased transcription of glycolysis genes , since transcription from such genes is negatively regulated by Cra ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24212579" target="_blank">24212579</a>]<br></li>
<li>We then concluded that Cra represses transcription of the newly identiﬁed yahA and hypF genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The genes encoding transcription factors that are organized in the Cra regulon are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Genes that are activated by Cra ( e.g. ppsA , pckA , and aceBA ) , are generally subject to catabolite repression , while genes that are repressed by Cra ( e.g. fru , pts , and edd ) , are subjected to catabolite activation [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>All six known SELEX fragments ( fruB , epd , adhE , ptsH , nirB , pykF ) are the group of genes , which are repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Genes encoding key enzymes concerned with oxidative and gluconeogenic carbon flow are activated by era while genes encoding key enzymes involved in fermentative carbon flow are repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Both the pykA and pykF genes , encoding pyruvate kinase , are also repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Generally the genes , which are repressed by Cra , have higher afﬁnity to Cra than those that are activated by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The icdA and aceA genes are known to be repressed by ArcA/B and activated by Cra ( Additional file 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19055729" target="_blank">19055729</a>]<br></li>
<li>Many genes involved in both pathways are known to be under the control of Cra.Yet the whole set of E. coli genes , which are regulated by Cra , has not been identiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The pykA and pykF genes have putative promoters that could be transcribed by RpoS , and pykF is negatively regulated by Cra [ 20,53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Among seven promoters that were hitherto identiﬁed to be under the control of Cra but not selected by this SELEX , two ( pfkA and edd ) belong to the promoter group that is repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>glk carries a putative RpoS promoter [ 20 ] and is apparently repressed by Cra ( FruR ) [ 49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Here we also identiﬁed that the glgC promoter is repressed by Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Transcription from the nir promoter was repressed by Cra when cells were grown in minimal me-dium with glycerol and fumarate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>The promoter of the pdhR-aceEF-lpd operon is repressed by pyruvate-sensing PdhR ( 33 ) and Cra ( this study ) and is activated by CRP ( 55 ; T. Shimada et al. , manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>This correlation has been found to not hold true in the case of glk , whose expression is repressed by Cra by binding to a site upstream of the glk promoter region ( Meyer et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Cra represses the promoter by blocking access to the -10 element and this repression can not be overridden by the other upstream activators ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The result indicates that the newly identi-ﬁed upstream promoters P1 ′ / P2 ′ contribute , to certain extent , the expression of gapA at least under the repression conditions.The promoter assay conﬁrmed that gapA transcription from the P1 -- P5 promoters is repressed by Cra-box 1-bound Cra protein , but the role of Cra-box 2 remains unsolved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>The transcription of the pdhR gene is also repressed by upstream-bound Cra ( position 185.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In contrast , transcription from the nrf promoter was not regulated by Cra under any of the growth conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>In the absence of glucose , Cra binds to the promoters of target operons and activates or represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Of a total of 194 peaks , a group of 164 peaks is located upstream of open reading frames ( ORFs ) , implying that these ORFs are the putative target genes or operons under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>These six operons are known to be repressed by Cra , and the Cra-binding sequences are located near the cognate promoters ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Examination of the aligned operators revealed that the consensus sequence for Cra binding is the oame for operons that are activated and repressed by era .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>This finding indicated that in the presence of FIP or FBP , operons under positive era control are deactivated while operons under negative Cra control are derepressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Only those operons that are regulated by Cra are indicated in italics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Consequently , operons which are positively regulated by Cra are deactivated or in other words , repressed by certain catabolites ; i.e. , catabolite repression occurs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>