<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - CreB</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">CreB</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">CreB transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The RR-bound 32P was stably retained for the A1 group RRs such as pArcA , pCpxR , pCusR , pKdpE , and pNarL but was rapidly released for the A2 group RRs , including pBaeR , pCheB , pCheY , pCreB , pDcuR , pNarP , pNtrC , pTorR , pUhpA , pYehT , and pYfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Location and functional classification of 32-regulon members A. Location Periplasm ( 2 ) Regulon members DsbC YceI CreC CutE FtsH FxsA GntYa HflX HflK HflC HtpX LipBa LspA MenA PgpA PhoQ YbeXa YbeZa YceJa YcjFa ClpB ClpP ClpX CreB DnaJ DnaK FkpB GapA GlnS GroELS GrpE HepA Hfq HolC Hsp33 HslU HslV HtpG IbpA IbpB IleS IspH Lon MiaA Mlc MutL MutM NarP NusB PhoP PrlC PyrF RecA RecJ RdgB RpoD rrnB RrmJ SdaA TopA TyrR ValS XerD YciH YfjN YrfH ZntR CreA FolP RibE ThiL YadF YafD YafE YbeD YbeY YceP YciH YciM YciS YcjX YdhQ YeaD YfjN YgaD YgbF YggW YhdN YhiQ YnfK Inner membrane ( 18 ) Cytoplasm ( 48 ) Unknown ( 25 ) B. Functional category Regulon members Metabolism CutE FolP GapA IspH LipB LspA MenA PgpA PyrF RibE SdaA ThiL YadF YafE YceJ YggW Chaperone/folding catalysts ClpB DnaJ DnaK DsbC FkpB GroELS GrpE Hsp33 HtpG IbpA IbpB Protein degradation ClpP ClpX FtsH HflX HflK HflC HslU HslV HtpX Lon PrlC DNA modification HolC MutL MutM RecA RecJ RdgB TopA XerD RNA state GlnS Hfq IleS MiaA rrnB RrmJ ValS YciH YfjN YrfH Transcription regulators CreB CreC HepA Mlc NarP NusB PhoP PhoQ RpoD TyrR ZntR Transporter YbeX YbeZ Miscellaneous FxsA GntY YciH YfjN YhiQ Unknown function CreA YafD YbeD YbeY YceI YceP YciM YciS YcjF YcjX YdhQ YeaD YgaD YgbF YhdN YnfK Only regulon members encoded on the chromosome and with a confirmed 32-dependent promoter are considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16818608" target="_blank">16818608</a>]<br></li>
<li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>PG-V-SR -- LL 60 70 80 90 i00 SRV RDEEVYDRS IDVLILRLRR KLEADPSSPQLIKTARGAGYFF SRV REEEVYDRS IDVLIFRLRR KLEGDPTSPQLIKTARGAGYFF SRV REEEVYDRS IDVLI LRLRR KLEGDPTTPQLIKTARGAGYFF VTG RELKPHDRTVDVTIRRIRK HFESTPDTPEIIATIHGEGYRF '' ~ 2 , ~ GY TPERHL DTRVVDVHISRLRA KLEDDPSNPELILTSRGTGYLF \ q , ~ GY TAERQV DTRVVDVHISRLRA KLEDDPSNPDLILTARGTGYLF LQLY PDAELRESHTIDVLMGRLRK KIQAQYPH DVITTVRGQGYLF '' , ~ G PNAVEHSHYLRIYMGHLRQ KLEQDPARPRHFITETGIGYRF 2 , GR EYSAME RSIDVQISRLRR MVEEDPAHPRYIQTVWGLGYVF , ~ A2RSASAEEAAGDRAVDVQ ITRLRR KIEPDPREPRYLQTVRGLGYML '' , ' > ~ G TNVYVEDRTVDVHIRRLRK ALEPG GHDRMVQTVRGTGYRF \ ~ ' , ; G GNVYVEERTVDMD IRRLRK ALGE VYENLVQTVRGTGYRF V > ~ E DAQDTYDRTVDTHIKTLRA KLRAINPDLSPINTHRGMGYSL VS 'N YDFAGDTRIVDVHISHLRPTKIENNTKKPIYIKTIRGLGYKL VLG KRLTPFDRAIDMHISNLRR KLPDRKDGHPWFKTLRGRGYLM AWD ENTDPFTNVVRVTVMTLRR KLGE PPVIVTVPGSGYRI EQGFEVDDSSLTQAISTLRK MLKDSTKSPQYVKTVPKRGYQL VirG Atu6 VirG Arh VirG Atu5 ArcA Eco RRHM CYca RRHM POae PhoP Sty KdpE Eco OmpR Eco PetR Rca PhoB Eco PhoB Pae CreB Eco PhoB Bsu YjjE Eco CutR Sli ToxR Vch 119 Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response-regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Proteins in classes 4 , 5 and 6 are exceptions 7O ENAF I0 20 30 40 50 60 NtrC NtrC NifRl NtrC PgtA DctD FixJ NodW CheY SpoOF RRHM RRHM OmpR PhoB CreB ArcA PetR VirG CutR KdpE PhoB PhoP TctD AIgB YLB3 RcsC BvgS HydG CheY FrzE BarA TrpO HvgA FimZ DegU NarL AIgRI YecB UhpA RcsB Agm & CheB CheB FrzG Spo0A MrkE AgrA Coma F!bD Bpa Rme Rca Eco Sty Rle Rme Bja Bsu Bsu Cca Pae Eco Bsu ECO Eco Rca Atu Sli Eco Eco Sty Sty Pae Lin Eco Bpe Eco Eco M ~ a Eco Pae Bpe Eco Bsu Eco Pae Eco Eco Eco Pae Bsu Eco Mxa Bsu Kpn Sau Bsu Ccr 6 ILVADDDTAIKTVLNQA 6 ILVADDD ~ IRTVLNQA 5 VLVADDDRTIRTVLTQA 6 V % ~ VDDDSSIR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>) Major clusters 1 , 2 , and 3 are associated primarily or exclusively with the three major families of DNA binding NtrC Bpa NtrC Rme NifRl Rca Nt rC Eco PgtA Sty DetD Rle FixJ Kme NodW Bja CheY Bsu SpoOF BSu RRHM Cca RRHM Pae OmpR Eco PhoB Bsu CreB Eco ArcA Eco PetR Rca VirG Atu CutR Sli KdpE ECO PhoB Eco PhoP Sty TctD Sty A!gB Pae YLB3 Lin RcsC Eco BvgS Bpe HydG Eco CheY Eco FrzE Nc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>These include CreB and PhoB of enteric bacteria which are responsive to cytoplasmic catabolite concentrations and extracellular inorganic phosphate , respectively ; OmpR of enteric bacteria , involved in responses to osmotic conditions ; VirG proteins of agrobacterial species , activators of the Ti or Ri plasmid-encoded virulence op-erons ; and ArcA , an E. coli activator of genes encoding proteins which function in aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>The first approximately 200 residues of KdpE and three other effector-proteins , PhoB ( 28 ) , CreB ( formerly PhoM-orf2 , the hypothetical PhoB protein of PhoM/Pho2 ) ( 2 , 48 ) , and OmpR ( 8 ) of E. coli are aligned with the TREE program ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>lPV 1LLTARsava 1eaatrkpDL iILDlgLPdgdGiefirdI.Rqvsav ... PV ivLsARseEs ArcA VirG PhoP Va `` R PhOB CreB TctD KdpE CO `` SEX `` SUS DL LD LP G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>pvirighFeLnepaaqisw fDtp.LaLTryEFlLLktL1 kspgrVwSRq hevqqlgeLi Fhdegy.fll qgqp.LaLTp rEqaLLtvb yrrtrpvSRq dplvkfsdvtvdlaar.vih rgeeavhLTp iEFrLagrcstmpekyspsg ~ PR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE and Silverman , with the second viously published by Weber and Silverman ( Weber 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>PhoB CreB OmpR FIG. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>227 OmpR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE kINnlaRgre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>Asterisks ( * ) indicate the conserved aa residues crucial for the transcriptional regula ¬ 232 CpxR nvsas ... OmpR ArcA VirG PhoP V & `` R PhoB CreB TctD KdpE hrpdgska .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>The level of phosphorylated RRs was less than 1 % for CreB , HydG , YedW , NarL , CheB , OmpR , DcuR , CpxR , YehT , and NarP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The trans-phos-phorylation was experimentally demonstrated in vitro , for the first time , for 9 pairs , CusS-CusR , RstB-RstA , YedV-YedW , BaeS-BaeR , YehU-YehT , YfhK-YfhA , HydH-HydG , BasSBasR , and CreC-CreB ( see Fig. 3 ) , supporting the predicted HK-RR pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>A total of 30 sensor HKs , including as yet uncharacterized puta - tive HKs ( BaeS , BasS , CreC , CusS , HydH , RstB , YedV , and YfhK ) , and a total of 34 RRs , including putative RRs ( BaeR , BasR , CreB , CusR , HydG , RstA , YedW , YfhA , YgeK , and YhjB ) , have been suggested to exist in Esche - richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Trans-phosphorylation of presumed cog - nate RRs by HKs was detected , for the first time , for eight pairs , BaeS-BaeR , BasS-BasR , CreC-CreB , CusS-CusR , HydH-HydG , RstB-RstA , YedV-YedW , and YfhK-YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>However , since CreB-His clearly is biologically active and more easily puriﬁed than native CreB , it was used to test for the ability of CreB to-bind to the cre tag sequence in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Thus it is probable that CreB and BlrA bind to the same DNA binding motif but that they are activated to do this by their partner signal sensor in response to different signals and that their binding sites control the expression of functionally unrelated genes in E. coli and Aeromonas spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>CreB and CreC form a classical two-component regulatory system , and we recently showed that CreBC are activated upon fermentation of glucose in minimal medium or during aerobic-growth-on-minimal-medium containing fermentation products , such as pyruvate , lactate , or acetate , as the sole carbon and energy source ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20418396" target="_blank">20418396</a>]<br></li>
<li>Thus , the simplest explanation of the observed effects is that CreB binds to the cre tag directly , and that the activation of CreC after a switch from complex to minimal medium leads to increased phosphorylation of CreB via phospho-relay , resulting in CreB binding to the cre tag more tightly and stimulating ( repressing in the case of malE ) transcription of the cre tag gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>Therefore , we are conﬁdent in proposing the following model : CreC responds to changes in the growth medium and/or me-tabolite pool levels and autophosphorylates ; this phosphate is passed on to CreB , which binds to cre tag sequences ; at some promoters ( e.g. , creD or talA ) , CreB works as a transcriptional-activator by recruiting RNA polymerase in a manner typical of other winged-helix RR proteins , such as PhoB ( 7 ) ; at other promoters ( e.g. , malE ) , the binding of CreB to the cre tag represses transcription through promoter occlusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>The creC gene is part of a four-gene cluster , creABCD ( 11 ) ; the functions of CreA and CreD are unknown , but CreB is a putative response-regulator and is homologous to PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>CreB and CreC are essential for the activation of cre regulon gene expression , but CreA and CreD , encoded as part of the creABCD gene cluster , are not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Given that PhoB and CreB are highly similar ( 25 ) , we next tested whether CreB overproduction in E. coli DH5 would stimulate cre regulon gene expression even during aerobic-growth-on-glucose , when the CreBC TCS is not activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>CreC and CreB are known to form an active TCS in vitro ( 3 ) , and we have shown here that CreB binds to the cre tag in vitro and speciﬁcally stimulates cre regulon gene expression in vivo when overproduced in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Instead , the complete Cre regulon is seen only under conditions where CreC is activated and the activity of ArcA ( and possibly other regulators ) is also modi-ﬁed , perhaps releasing a block on activation of gene expression by CreB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20418396" target="_blank">20418396</a>]<br></li>
<li>We must conclude , however , that acetyl phosphate accumulation does not lead to the phosphorylation and activation of the CreB RR under these conditions , because even if cellular acetyl phosphate levels are high during our experiments , as might be expected ( 13 ) , the induction of cre regulon gene expression during pyruvate growth is entirely CreC dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Under other conditions , the HK CreC detects a signal ( as yet unknown ) , is autophosphorylated and transfers the phosphoryl group to both its partner RR CreB and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Cma sensitivity may be a convenient tool to study CreBC-mediated gene transcription and , in particular , to identify the signal that regulates CreC phosphorylation and , in turn , CreB phosphorylation and transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22773789" target="_blank">22773789</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">CreB protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>BlrAB is highly homologous to E. coli CreBC ( 60 -- 70 % at the amino acid level ) , and there is particularly good homology between the DNA-binding domains of BlrA and CreB ( 2 ) but poor homology between the signal-recognition domains of BlrB and CreC ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by CreB</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Cma sensitivity may be a convenient tool to study CreBC-mediated gene transcription and , in particular , to identify the signal that regulates CreC phosphorylation and , in turn , CreB phosphorylation and transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22773789" target="_blank">22773789</a>]<br></li>
<li>Overproduction of BaeR also increased expression of the CreB regulon ( 4 ) , including expression of creD , malE , yidS , and yieI ( Table 2 ) and the creB response regulator gene ( 2.6-fold increase as determined by microarray analysis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
<li>These data conﬁrm that CreB and CreC both are essential for controlling the expression of cre regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>These results ( Fig. 7A ) clearly demonstrate that the overproduction of CreB activates cre regulon gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>These include CreB and PhoB of enteric bacteria which are responsive to cytoplasmic catabolite concentrations and extracellular inorganic phosphate , respectively ; OmpR of enteric bacteria , involved in responses to osmotic conditions ; VirG proteins of agrobacterial species , activators of the Ti or Ri plasmid-encoded virulence op-erons ; and ArcA , an E. coli activator of genes encoding proteins which function in aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>To examine whether overexpressed CbrA inhibits Cma uptake , the Cma transport system via FhuA , TonB , ExbB , and ExbD was bypassed by osmotic TABLE 4 - Galactosidase activities of E. coli strains carrying pcbrA-lacZ fusions on plasmid pKH55a Strain Medium Growth Activity ( U ) BW25113 ( pKH55 ) M9 Aerobic 16.2 JW4361 creB ( pKH55 ) M9 Aerobic 3.6 JW5631 cbrA ( pKH55 ) M9 Aerobic 14.4 BW25113 ( pKH55 ) M9 Anaerobic 14.5 JW4361 creB ( pKH55 ) M9 Anaerobic 1.6 JW5631 cbrA ( pKH55 ) M9 Anaerobic 13.3 BW25113 ( pKH55 ) LB Aerobic 2.1 JW4361 creB ( pKH55 ) LB Aerobic 0.7 JW5631 cbrA ( pKH55 ) LB Aerobic 1.5 a Plasmid pKH55 encodes the pcbrA-lacZ operon fusion under the control of CreB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22773789" target="_blank">22773789</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">CreB binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>We are conﬁdent that yieJ can be directly assigned to the Cre regulon because it shares with yieI a promoter containing a Cre tag sequence ( 5 ) , which is th binding site of CreB ( 10 ) , and its expression is upregulated in a CreB-dependent manner upon activation of CreC ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20418396" target="_blank">20418396</a>]<br></li>
<li>cbrA is preceded by the repeat sequence TTCACnnnnnnTTCAC ( 2 ) , which serves as a binding site of the transcriptional regulator CreB ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22773789" target="_blank">22773789</a>]<br></li>
<li>A so-called cre/blr tag sequence , TTCACnnnnnnTTCAC , located at around 60 relative to the transcriptional start site , is essential for the control of cepH expression by the RR , BlrA ( 6 ) , so we postulated that CreB also controls cloned cepH expression through the cre/blr tag , and further , that the cre/blr tag is found upstream of CreB-regulated genes native to E. coli ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>CreB binds to a TTCACnnnnnnTTCAC direct repeat ( the cre tag ) in vitro , and this sequence , which is associated with cre regulon gene promoters , is required for the control of gene expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>These data conﬁrm that CreB binds speciﬁcally to the cre tag sequence in vitro and , taken together with the data presented in Fig. 7 and the determined transcriptional start site for creD ( Fig. 1B ) , provide strong evidence that CreB is a transcriptional activator of creD and other cases of cre regulon gene expression through interaction with the cre tag .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by CreB</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>DH5 ( pUB6070 ) ( A ) or DH5 ( creC ) ( pUB6070 ) ( B ) containing either empty pBAD vector ( CONT ) , pUB6073 ( CreB ) , or pUB6074 ( CreB-His ) were grown to early log phase ( OD600 0.3 ) in M9 minimal medium containing glucose , and then 0.2 % ( wt/vol ) arabinose was added ( ARA ) to induce CreB or CreB-His production , and the cells were grown for a further 2 h. Alternatively , no arabinose was added ( ARA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>To examine whether overexpressed CbrA inhibits Cma uptake , the Cma transport system via FhuA , TonB , ExbB , and ExbD was bypassed by osmotic TABLE 4 - Galactosidase activities of E. coli strains carrying pcbrA-lacZ fusions on plasmid pKH55a Strain Medium Growth Activity ( U ) BW25113 ( pKH55 ) M9 Aerobic 16.2 JW4361 creB ( pKH55 ) M9 Aerobic 3.6 JW5631 cbrA ( pKH55 ) M9 Aerobic 14.4 BW25113 ( pKH55 ) M9 Anaerobic 14.5 JW4361 creB ( pKH55 ) M9 Anaerobic 1.6 JW5631 cbrA ( pKH55 ) M9 Anaerobic 13.3 BW25113 ( pKH55 ) LB Aerobic 2.1 JW4361 creB ( pKH55 ) LB Aerobic 0.7 JW5631 cbrA ( pKH55 ) LB Aerobic 1.5 a Plasmid pKH55 encodes the pcbrA-lacZ operon fusion under the control of CreB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22773789" target="_blank">22773789</a>]<br></li>
<li>cbrA transcription is under the control of CreB ( 2 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22773789" target="_blank">22773789</a>]<br></li>
<li>Thus it is probable that CreB and BlrA bind to the same DNA binding motif but that they are activated to do this by their partner signal sensor in response to different signals and that their binding sites control the expression of functionally unrelated genes in E. coli and Aeromonas spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>The signal sensor BlrB is 65 % identical to CreC ( 21 ) , and the response regulator , BlrA , is 63 % identical to CreB ( 1 , 21 ) and uses the same Cre tag motif to bind DNA and thus activate gene expression ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20418396" target="_blank">20418396</a>]<br></li>
<li>Increased expression of the CreB regulon members ( 4 ) , creD , malE , yidS , and yieI was observed by both microarray and qRT-PCR analyses ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
<li>Overproduction of BaeR also increased expression of the CreB regulon ( 4 ) , including expression of creD , malE , yidS , and yieI ( Table 2 ) and the creB response regulator gene ( 2.6-fold increase as determined by microarray analysis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
<li>A so-called cre/blr tag sequence , TTCACnnnnnnTTCAC , located at around 60 relative to the transcriptional start site , is essential for the control of cepH expression by the RR , BlrA ( 6 ) , so we postulated that CreB also controls cloned cepH expression through the cre/blr tag , and further , that the cre/blr tag is found upstream of CreB-regulated genes native to E. coli ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Given that PhoB and CreB are highly similar ( 25 ) , we next tested whether CreB overproduction in E. coli DH5 would stimulate cre regulon gene expression even during aerobic growth on glucose , when the CreBC TCS is not activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Of course it is possible that the regulation of cre tag gene expression by CreBC is indirect and mediated by the activation of a transcription factor other than CreB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>CreB and CreC are essential for the activation of cre regulon gene expression , but CreA and CreD , encoded as part of the creABCD gene cluster , are not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>These data conﬁrm that CreB and CreC both are essential for controlling the expression of cre regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>However , the role of the cre tag in the CreBC-mediated control of cre regulon gene expression has not been examined experimentally , so it was the secondary aim of the work described in this paper to do so , by conﬁrming that CreB binds to the cre tag sequence in vitro and that this sequence is essential for the CreBC-depen-dent control of gene expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>CreC and CreB are known to form an active TCS in vitro ( 3 ) , and we have shown here that CreB binds to the cre tag in vitro and speciﬁcally stimulates cre regulon gene expression in vivo when overproduced in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>