<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - CRP</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">CRP</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">CRP transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>CSklaCZexpression in minimal media in response to starvation for carbon , nitrogen and phosphate sources and the roles of global regulators such as the alternative sigma factor ( encoded by 0 ' rpoS ) , cAMPICRP and the relA gene product were investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>Lanes 1 and 8 , no added protein ; lane 2 , 50 ni % - CAP plus O-2mM-CAMP ; lane 3 , NagC extract from JMlOl , 28 pg/ml ; lane 4 ; as lane 3 plus @ 2mM-CAMP ; lane 5 , NagC extract from TP2139 ( Acrp ) 130pg/ml ; 1ane 6 , as lane 5 plus 0.2 mivr-CAMP ; lane 7 , as lane 6 plus 50 n & r-CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Beta-galactoatdase activity '' Compound added '' Giucose Galactose 3530 9740 320 3480 2460 NG * ̂ NG NG NG 4160 320 Xylose Giyceroi Acetale Succinate Fumarate Buffered L-broth Buffered L-broth-i-glucose Effect of crp and fruR mutations on sdhC-lacZ expression As a relatively wide range in sd/7C - / acZ expression was obseived when cells were grown in different types of media ( Table 2 ) , we tested whether the ap-gene product contributes to ttiis control as previously proposed ( Wilde and Guest , 1986 ) , A Acrp mutation was introduced into the MC4100 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783618" target="_blank">7783618</a>]<br></li>
<li>When osmotic regulation of osmY in minimal medium was assayed in a Acya Acrp background ( RO82 ) , csi-5 : : lacZ expression was already increased in exponential phase and could be further stimulated by adding 0.3 M NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The increased exponential phase expression of both fusion constructs in the Acrp background is probably due to the strongly elevated levels of os in cya and crp mutants ( Lange and Hengge-Aronis , 1991b ; Lange and Hengge-Aronis , 1994a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Only the high strength promoter was sensitive to catabolite repression , while the other was active in Acrp or Acya strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>The complexity of addition two new minor groove contacts , ascribed to negative regulation was first shown by the fact that CytR phosphate-CytR interactions , are observed at -60 be - regulation was missing in cells devoid of the CAMP-CPR tween theCRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>) 1,2 139 CRP cytR 0 1 p 1,2 142 CRP deop2 1 1 p 1,2 142 , 162 CRP galpi 1 1 0 1,2 60 CRP glpD 0 1 0 2,4 174 CRP glnApl 1 1 0 4 119 CRP ilvB 0 0 0 4 39 CRP lac 0 1 0 1,2 87 CRP malT 0 0 0 1,2 88 CRP melR 0 0 0 1,2 169 CRP nagB 1 0 0 4 108 CRP nagE 1 0 0 4 108 CRP pBRp4 0 0 0 2,4 113 CRP putA 1 1 0 4 47 CRP rpoHpS 1 0 0 4 99 CRP tnaA 0 0 0 2,4 27 CRP tsx 0 1 p 4 142 , 165 FNR dmsA 0 0 0 4 35 FNR fdrA 0 0 0 1,4 35 , 61 FNR , Y gipA 0 0 0 4 35 FNR , Y narG 0 0 0 1,4 35 , 75 FNR nirB 0 1 0 1,4 35 , 61 , 62 FNR PFLp6 1 0 0 4 133 FNR PFLp7 1 0 0 4 75 , 133 IlvY ilvC 1 0 p 3 170 MalT malE 1 1 R 1,2 114 , 115 MalT malK 1 1 R 1,2 114 , 115 MalT malPQ 0 0 0 1,4 138 MetR metE 1 1 0 1,3 161 MetR metH 0 0 0 4 160 , 161 OmpR ompC 0 0 p 1,2 80 , 81 , 154 OmpR ompF 0 0 p 1,2 38 , 116 , 154 OxyR ahpC 0 0 0 3 146 OxyR katG 0 0 0 3 146 , 149 PapB papB 1 1 R 1,2 37 , 41 PhoB phoA 0 0 0 4 84 PhoB phoB 0 0 0 1,4 84 , 154 PhoB phoE 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The question arises as to the involvement of the different activating regions in CRP and FNR at promoters in which both CRP and FNR function in tandem in various combinations , such as in the ansB promoters of Salmonella and Escherichia coil ( Jennings and Beacham , 1993 ; Jennings et aL , 1993 ; see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_11">
Scott , S. , Busby , S. , and Beacham , I. ( 1995 ) Transcriptional co-activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></label>
<input id="act_similarity_11", type="checkbox">
<div class="ACT_similar">
<li>Transcriptional co-activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10715008" target="_blank">10715008</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_13">
Scott , S. , Busby , S. , and Beacham , I. ( 1995 ) Transcriptional coactivation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></label>
<input id="act_similarity_13", type="checkbox">
<div class="ACT_similar">
<li>[ 20 ] Scott , S. , Busby , S. and Beacham , I. ( 1995 ) Transcriptional co-acti-vation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>[ 149 ] Scott , S. , Busby , S. and Beacham , I. ( 1995 ) Transcriptional co-acti-vation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li></div>
<li>We conclude that , at the ansB promoters , AR1 is required when CRP binds upstream ( around -- 91 ) , whether functioning together with downstream FNR in a co-dependent manner , or with CRP downstream in a synergistic manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>At class II CRP-dependent promoters , CRP binds to a site centered near position 41.5 , and transcription is stimulated through interactions between CTD and AR1 of the upstream CRP monomer and between NTD and an additional surface , referred to as activating region 2 ( AR2 ; residues 19 , 21 , 96 , and 101 ) , on the downstream CRP monomer ( 11 , 24 , 76 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Bands of interest were excised and analyzed by gas-phase sequencing at the Biomo ¬ I Abbreviations : aa , amino acid ; BisTris , [ bis ( 2-hydroxyethyl ) - amino ] tris ( hydroxymethyl ) methane ; BME , 8-mercaptoethanol ; CNBr , cyanogen bromide ; CRP , cataboliterepressorprotein ; CTNE , N-carbo-benzoxy-L-tyrosine-p-nitrophenyl ester ; dATP , 2 ' - deoxyadenosine 5 ' - triphosphate ; DEAE , diethylaminoethyl ; DNA , deoxyribonucleic acid ; DNase I , deoxyribonuclease I ; DTT , m-dithiothreitol ; EDTA , ethyl-enediaminetetraaceticacid ; IDA , iminodiacetic acid ; LacR , lac repressor ; MOPS , 3 - ( N-morpholino ) propanesulfonicacid ; PMSF , phenylmethanesulfonyl fluoride ; SDS-PAGE , sodiumdodecyl sulfate-polyacrylamide gel electrophoresis ; TAME , p-tosyl-L-arginine methyl ester ; TPCK , L-1-tosylamino-2-phenylethylchloromethyl ketone ; Tris , tris ( hydroxy-methy1 ) aminomethane ; TrpR , trp repressor ; UV , ultraviolet lecular Resource Center , University of California , San Francisco .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>( 1991 ) 218 , 529-542 Supercoiling is Essential for the Formation and Stability Initiation Complex at the Divergent malEp and malKp Promoters of the Evelyne Richet and Olivier Raibaud Unite ' de Ge 'n e ' tique Mole ' culaire A 1149 du CNRS , Institut Pasteur , 25 rue du Dr. Roux 75724 Paris Cedex 15 , France UR ( Received 5 July 1990 ; accepted 2 January 1991 ) m , alEp and malKp are divergent and partially overlapping promoters of the Escherichia coli maltose regulon , whose activity depends on the presence of two transcriptional-activators , MalT and CRP ( CAMP receptor protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_20">
CRP induces the repositioning of MalT at the Escherichia col malKp promoter primarily through DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></label>
<input id="act_similarity_20", type="checkbox">
<div class="ACT_similar">
<li>CRP induces the repositioning of MalT at the Escherichi coli malKp promoter primarily through DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14529615" target="_blank">14529615</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_22">
Richet , E. , and Søgaard-Andersen , L. ( 1994 ) CRP induces the repositioning of MalT at Escherichia coli malKp promoter primarily through DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></label>
<input id="act_similarity_22", type="checkbox">
<div class="ACT_similar">
<li>Richet , E. , and Sogaard-Andersen , L. ( 1994 ) CRP induces the repositioning of MalT at the Escherichia coli malKp promoter primarily through DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Richet , E. , and Sogaard , A.L. ( 1994 ) CRP induces the repositioning of MalT at the Escherichia coli malKp promoter primarily through DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9822819" target="_blank">9822819</a>]<br></li>
<li>CRP induces the repositioning of MalT at the Escherichia coli malKp promoter primarily through DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9440505" target="_blank">9440505</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9401025" target="_blank">9401025</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li></div>
<li>To determine the effect of different carbon sources on E. coli CRP activity as well as the functional relevance between the active CRP level and the expression of FlhDC operon in a range of conditions with the sequenced E. coli strain MG1655 , we performed the following assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>To determine whether the observed regulation by glucose was mediated by CRP , the Pa-lacZ and Pz-lacZ fusions were integrated into the chromosome of an isogenic CRP2 strain , E. coli AFSB , as described under `` Experimental Procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Cra is involved in regulation of the whole set of enzymes in central carbon metabolism ( 43 ) , while CRP regulates the genes for carbon source utilization outside the central metab-olism , including transport of various carbon sources , glycolysisgluconeogenesis switching , the tricarboxylic acid ( TCA ) cycle , the pyruvate dehydrogenase pathway , and aerobic respiration ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Results Search for the regulation targets of CRP by using Genomic SELEX screening : SELEX-clos analysis In order to get insight into the genome regulation by CRP for control of carbon source utilization in E. coli , we first tried to identify the whole set of target promoters , genes and operons under the control of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In addition to the operon-specific regulation mediated by the PaaX repressor , the paa catabolic cluster of E. coli is subjected to control by the two global regulators , CRP ( 23 ) and IHF ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>We have shown in this work that the specific PaaX-mediated control of the Pa and Pz promoters for PA degradation in E. coli is , in turn , subordinated to a superimposed regulation mediated by global regulators such as CRP and IHF , which connect the expression of the paa catabolic genes to the metabolic and energetic status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>CRP Modulates fis Transcription by Alternate Formation of Activating and Repressing Nucleoprotein Complexes * Received for publication , January 23 , 2001 , and in revised form , February 22 , 2001 Published , JBC Papers in Press , February 23 , 2001 , DOI 10.1074 / jbc.M100632200 William Nasser ‡ , Robert Schneider ‡ § , Andrew Travers ¶ , and Georgi Muskhelishvili ‡ i From the ‡ Institut für Genetik und Mikrobiologie , Ludwig-Maximilians-Univesitaet , Maria-Ward-Strasse 1a , 80638 München , Germany and the ¶ Medical Research Council Laboratory of Molecular Biology , Hills Road , Cambridge CB2 2QH , United Kingdom The DNA architectural proteins FIS and CRP are global regulators of transcription in Escherichia coli involved in the adjustment of cellular metabolism to varying growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>43 2 Biochemistry 2010 , 49 , 432 -- 442 DOI : 10.1021 / bi901583n An Unusual Pattern of CytR and CRP Binding Energetics at Escherichia coli cddP Suggests a Unique Blend of Class I and Class II Mediated Activation † Allison K. Holt and Donald F. Senear * Department of Molecular Biology and Biochemistry , University of California , Irvine , California 92697 Received September 9 , 2009 ; Revised Manuscript Received November 4 , 2009 ABSTRACT : Two transcription factors , CRP and CytR , mediate positive and negative control of nine cistrons involved in nucleoside catabolism and recycling in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_34">
Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigma S and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></label>
<input id="act_similarity_34", type="checkbox">
<div class="ACT_similar">
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on σS and requires activation by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10406809" target="_blank">10406809</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sS and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731280" target="_blank">14731280</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigmaS and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and s requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and S requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_41">
Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigma s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26020590" target="_blank">26020590</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24981318" target="_blank">24981318</a>]<br></label>
<input id="act_similarity_41", type="checkbox">
<div class="ACT_similar">
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli , that is exclusively dependent on ss and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on S and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621818" target="_blank">16621818</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and requires activation by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730174" target="_blank">12730174</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li></div>
<li>6 ( 1992 ) 3257 -- 3265 [ 31 ] Xu J. , Johnson R.C. , aldB , an RpoS-dependent gene in Escheri-chia coli encoding an aldehyde dehydrogenase that is repressed by FIS and activated by CRP , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>Xu , J.M. , and Johnson , R.C. ( 1995a ) aldB , an RpoSdependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>aldB , an RpoS-depen-dent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_49">
Xu , J. and Johnson , R.C. ( 1995b ) aldB an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by FIS and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></label>
<input id="act_similarity_49", type="checkbox">
<div class="ACT_similar">
<li>aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Xu , J.M. , and Johnson , R.C. ( 1995 ) aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li></div>
<li>References Aiba , H , ( 1983 ) Autoreguiation of the Escherichia coli crp gene : CRP is a transcriptional-repressor for its own gene , Ceii 32 : 141-149 , Aiba , H , , and Krakow , J.S , ( 1981 ) Isolation and characterization of the amino and carboxyl proximal fragments of the adenosine cyclic 3 ' , 5 * - phosphate receptor protein of Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Natl. Aiba , H. : Autoregulation of the Escher & u coli crp gene : CRP is a transcriptional-repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_54">
H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></label>
<input id="act_similarity_54", type="checkbox">
<div class="ACT_similar">
<li>Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2999070" target="_blank">2999070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6193097" target="_blank">6193097</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2180916" target="_blank">2180916</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>]<br></li>
<li>Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126461" target="_blank">15126461</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523115" target="_blank">14523115</a>]<br></li>
<li>[ 15 ] Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>References Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>References Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional-repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11555297" target="_blank">11555297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li></div>
<li>0 5-minute induction of σF 5 A 0 2 = σF β ' Motility control by carbon source : CRP binds to and scalably activates FlhDC operon in alternative carbon sources Recently , Liu and coworkers ( 20 ) proposed a new model for carbon source foraging strategy by E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>As shown in more detail for the sfm operon , this catabolite repression is exerted through the CRP protein , therefore recognizing these ﬁmbrial operons as new CRP-activated operons not previously identiﬁed Multiple functional ﬁmbriae in Escherichia coli K-12 1967 Morphological and genetic diversities between the studied ﬁmbriae suggested that they can interact with a variety of structures or ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Epst , ream repredsion and CRP stimulation of the Escherichia coli r,-arabinose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1560456" target="_blank">1560456</a>]<br></li>
<li>Upstream repression and CRP stimulation of the Escherichia co/i L-arabinose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2871943" target="_blank">2871943</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_66">
and Schleif , R. ( 1984 ) Upstream repression and CRP stimulation of Escherichia coli L-arabinose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></label>
<input id="act_similarity_66", type="checkbox">
<div class="ACT_similar">
<li>Hahn S , Dunn T , Schleif R. Upstream repression and CRP stimulation of the Escherichia coli L-arabinose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17173282" target="_blank">17173282</a>]<br></li>
<li>Upstream repression and CRP stimulation of the Escherichia coli L-arabinose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9440505" target="_blank">9440505</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600836" target="_blank">9600836</a>]<br></li></div>
<li>J Bacteriol 114:1068 1073 Fraser ADE , Yamazaki H ( 1978 ) Determination of the rate of synthesis and degradation of adenosine 3 ' ,5 ' - monophosphate in Eseherichia coli CRP - and CRP + strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>, Anabaena species ; Btr Bp , Bordetella pertussis ; CLP Xc , Xanthomonas campestris ; CooA Rr , Rhodospirillum rubrum ; CRP Ec , Escherichia coli ; CRP Hi , Haemophilus influenzae : CRP Ka , Klebsiella aerogenes ; CRP Mt , Mycobacterium tuberculosis ; CRP Ph , Pasteurelhl haemolytica ; CRP Pm , Pasteurella multicida ; CRP SJ ~ Shigella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Claret , L. , and Rouvière - Yaniv , J. ( 1996 ) Regulation of HUa and HUb by CRP and FIS in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9570400" target="_blank">9570400</a>]<br></li>
<li>We have previously shown that CAMP and CRP repress the activity of the putrescine biosynthetic enzymes ( i.e. , ODC , arginine decarboxylase and agmatine ureohydrolase ) and glucose-6-phosphate dehydrogenase in E. coli ( Wright and Boyle , 1982 ; Satishchandran and Boyle , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>CRP down-regulates adenylate cyclase activity by reducing the level of phosphorylated IIA ( Glc ) , the glucose-specific phosphotransferase protein , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>Takahashi , H. , Inada , T. , Postma , P. & Aiba , H. ( 1998 ) CRP down-regulates adenylate cyclase activity by reducing the level of phosphorylated IIAGlc , the glucose-speciﬁc phosphotransferase protein , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>Predicted targets ( divergently transcribed gene sets with CRP sites on ORF of spacer genes ) ( total , 14 ) B. Targets not identified by Genomic SELEX ( total , 103 ) azoR , hrpA.ydcF , csrB , syd.queF , garP , garD.sohA , kduI , yqeF.yqeG , lpxH , ppiB.cysB , lysP , yeiE.yeiH , matA , matB.matC , minD , minC.ycgJ , rplM , yhcM.yhcB , uraA , upp.purM , yebA , znuA.znuC , yeaZ , yoaA.yoaB , yedJ , yedR.xseX , yqfF , argP.scpA aceB , acnA , acnB , acs , aer , agp , aldA , aldB , ansB , araJ , argG , bglG , chbB , cirA , crp , csgD , csiE , cyaA , cyaR , cyoA , dcuB , dcuR , dsdA , ebgA , entC , fadD , feaR , fecA , fepA , fiu , focA , fumA , fur , gadA , gadB , gadE , galE , glcC , glgC , glnA , glpD , glpE , glpF , gltB , gntX , grpE , gyrA , hlyE , hofM , hupA , hupB , isrB , lpd , marR , mdh , melA , melR , metK , metY , mhpA , modA , mpl , murQ , nanA , nirB , nrdA , ompA , ompR , osmY , oxyR , pdhR , pncB , ppdD , prpB , prpR , rbsD , relA , rrhaT , rpoS , sbmC , sdhD , sfsA , sodA , sodB , sohB , speC , spf , srlA , tnaC , trg , trxA , ubiG , ugpB , uxaB , uxuR , xylA , xylF , ychH , yfiD , yhfA , yiaJ , yiaK , ykgR , zraS C. Predicted targets ( genes transcribed toward opposite directly from known target gene ) ( total , 63 ) acnB , acs , aer , araF , ascF , aspA , cpdB , csgD , cyaA , dadA , deoC , dsdX , entC , fadL , feaR , fepA , fumA , gapA , gdhA , glcC , glgS , glnA , glnL , glpF , gltB , gntX , grpE , hlyE , hofM , ivbL , kbaZ , tsrA , malS , malT , manX , marR , mdh , mhpA , mtlA , murQ , nanC , nmpC , nrdA , nupC , ompR , oxyR , pdhR , pncB , ppiA , putP , serA , sohB , speC , srlA , tdcA , treB , trxA , udp , ulaA , ychH , yfiD , ykgR , zraS The regulation targets of cAMP-CRP listed in RegulonDB [ 27 ] are classified into two groups .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Genes induced at the onset of stationary phase with s - dependent expression as identiﬁed by microarray analyses Functional Mean WT d Ratio classa valueb ( WT/rpoS ) Regulation SDc Gene Gene product Starvation and stress response genes dpsS DNA protection protein , bacterial ferritin IHFS , OxyRS , rssBTC , MMC , PQ , NaSal , Lrp , BG , DHCP , OS FisS , NaSal , Lrp , OS CRPS , LrpS , IHFS , H-NSS , arcBTC , MMC , Lrp , Rcs , BG , DHCP ompR/envZTC , CD uvrYTC , arcBTC SSR 1,597 2.5 0.4 osmES osmYS Stress-inducible outer membrane lipoprotein Hyperosmotically inducible periplasmic protein Induced by phosphate-starvation Superoxide dismulase precursor ( Cu-Zn ) SSR SSR 1,728 424 9.6 4.4 2.1 1.3 656 459 psiF sodCS 4.8 2.7 0.4 0.5 SSR Protein synthesis rpsVS infA Amino acid transport and metabolism artP artI 5.2 2.8 FISS , IHFS , ppGppS 2.7 0.8 30S ribosomal subunit protein S22 Protein chain initiation factor IF-1 3,466 1,011 PS PS Arginine transport , ATP-binding component Arginine transport , periplasmic binding protein L-Asparagine permease Dihydroxyacid dehydratase Tryptophanase AATM AATM 714 346 2.6 0.7 PQ , NaSal , Lrp 3.8 1.9 PQ , Lrp ansP ilvD tnaA AATM AATM AATM 572 364 DHCP ompR/envZTC 2.5 4.8 0.4 2.1 4.3 phoQTC , ompR/envZTC , CRP , uvrYTC , arcATC , arcBTC , rcsBTC , citABTC , MMC , PQ ( ilvG : Lrp ) NaSal , Lrp , BG , DHCP , OS 13.7 7,808 wrbAS Flavodoxin-like protein ; binds the trp repressor SSR 364 5.0 2.0 Iron uptake and storage bfr entD Carbohydrate metabolism crr Bacterioferrin , iron storage homoprotein Enterochelin synthetase , component D IUS IUS 905 508 5.1 2.1 DHCP , ryhB RNA 4.2 0.5 acrBTC Phosphotransferase system , glucose-speciﬁc IIA component Fructose 1,6-bisphosphate aldolase ( b2097 ) Glyoxylate-induced protein Putative oxidase , glycolate utilization ( b2977 ) Glucosamine-6-phosphate deaminase Phosphofructokinase-2 Transaldolase A CHM 443 2.7 0.6 CD 1,264 1,097 CHM 4.6 3.6 3.0 2.6 2.5 2.9 1.4 1.0 1.7 0.6 2 1.8 MMC , Lrp kdpABCDETC fbaB gip uvrYTC , arcATC , arcBTC ompR/envZTC , arcBTC MMC , CD , DHCP rssBTC , MMC , DHCP , CreBC CHM CHM CHM CHM glcG nagB pfkB talA Miscellaneous function or unknown function deoD moeB msyB yafN yahOS ybgA 3.0 1.2 4.0 0.1 2.2 1.7 4.8 3.4 6.1 2.6 4.5 4.5 349 Purine nucleoside phosphorylase Molybdopterin biosynthesis NM CB 1,100 2,038 CD , Lrp , DHCP MMC , GadX Lrp Membrane protein , protein export ( b1051 ) Hypothetical ( adherence ) protein ( b0232 ) Hypothetical protein ( b0329 ) Hypothetical ( inner membrane ) protein ( b0707 ) Putative enzyme ( b0865 ) Hypothetical protein , bioﬁlm dependent ( b1481 ) Hypothetical protein ( b1724 ) Hypothetical protein ( b2427 ) Putative dehydrogenase protein ( b2660 ) Putative immunogenic protein ( b2922 ) Putative transport periplasmic protein ( b3020 ) Hypothetical ( outer membrane ) protein ( b3024 ) Hypothetical ( regulatory ) protein ( b3555 ) Hypothetical protein ( b4045 ) Hypothetical ( membrane ) protein ( b1582 ) Hypothetical ( periplasmic ) protein ( b3097 ) Hypothetical protein ( b4217 ) MF ( MF ) rssBTC , baeSRTC , atoSCTC , Lrp ybjP yddX 2.5 1.4 4.2 4.2 1,317 376 ( EF ) rssBTC , OS ompR/envZTC ydiZ yfeT ygaF yggE ygiS 1.7 0.9 0.5 1.7 0.8 3.1 3.5 2.8 4.8 2.7 436 rssBTC , NarXLTC rssBTC , MMC ; Rcs , phoQP ( EF ) ( MF ) rssBTC , citABTC , arcATC , NaSal ( MF ) ygiW 1.9 4.0 892 rssBTC , ntrBCTC , NaSal yiaG yjbJ ynfA yqjC ytfK 0.5 1.2 1.1 1.3 3.0 2.7 4.5 3.5 5.9 3.9 1,227 601 3,334 1,121 556 ( MF ) PQ , Fur , BG a SSR , starvation and stress response ; PS , protein synthesis ; AATM , amino acid transport and metabolism ; IUS , iron uptake and storage ; CHM , carbohydrate metabolism ; NM , nucleoside metabolism ; CB , cofactor biosynthesis ; MF , membrane function ; EF , extracellular function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>Gene characteristics Gene Predicted Band no.a Gene function ( reference ) Regulation of corresponding gene ( reference ) productb molecular mass ( Da ) c PﬂB * 85,357 GadA * 52,685 1 Pyruvate formate lyase I ( 56 ) Glutamate decarboxylase ( 64 ) Induced by ArcA and FNR ( 61 ) 2 Positively regulated by rpoS , CRP , gadE , gadW , and gadX ; negatively regulated by H-NS ( 21 , 66 ) Positively activated by rpoS and by CRP ( 33 , 74 ) Positively activated by CRP ( 74 ) ; repressed by ArcA ( 60 ) Same transcription unit as gatZ Positively regulated by rpoS ( 33 , 46 , 71 ) Positively regulated by rpoS ( 2 , 33 , 46 , 71 ) 52,773 TnaA † 3 Tryptophan deaminase ( tryptophanase ) ( 22 ) 47,109 GatZ † 3 Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Quinone reductase ; response to oxidative-stress ( 45 ) Bacterial ferritin , DNA protection during stationary phase ( 39 , 73 ) Transcriptional regulator ( 28 ) 30,812 20,846 18,695 GatY † WrbA * Dps * 4 5 6 , 7 , and 11 24,935 CsgD * 8 Positively regulated by rpoS , hns , and ompR ( 25 , 58 ) Same transcription unit as csgD Positively regulated by FNR ( 18 ) Outer membrane protein ; curli transport component ( 38 ) Outer membrane protein ; receptor for colicin S4 ( 49 ) 30,557 22,928 CsgG * OmpW * 9 10 a As in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>Examples of transcriptional control in which the action of ArgR occurs in combination with ( an ) other regulatory protein ( s ) and co-factors have been documented previously for CRP , NtrC , Lrp , and IHF .22,23,29,50 Furthermore , artP mRNA levels were found to be induced in the stationary phase by EσS , the general stress-specific RNA polymerase ,51 upregulated ( 1.9-fold ) by the superoxide generating agent paraquat , an inducer of the soxRS system , and by sodium salicylate ( NaSal ) , an inducer of the marRA system ( 2.5-fold ) ,52 and down-regulated by Lrp ( about sixfold ) .53 Genome-wide expression profiling studies in E. coli indicate that argT and hisJ transcription are activated by NtrC ( GlnG ) ,45 and that in anaerobiosis hisJ expression is strongly increased in fnr ( 25-fold ) and arcA ( 60-fold ) mutants .54,55 None of the underlying molecular mechanisms has been unraveled yet , but the induction of E. coli artP in the stationary phase appears to depend on several secondary promoters ,51 upstream of PartP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic-growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic-growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The phosphoryl group from PEP is transferred to the phosphoryl carrier protein HPr by enzyme I ( PtsI ) ; phosphoHPr then transfers it to the permease ( enzymes II/III ) ; HPr is common to all PTS ; regulated by the cAMP-CRP complex and also by growth-on-glucose ; induced by low pH and phosphonate growth See PtsH ; decreases after benzoic acid treatment but increases at phosphonate growth Plays an important role in de novo pathway of purine nucleotide biosynthesis : AMP biosynthesis ; increases after benzoic acid treatment Involved in IMP biosynthesis : 5-amino-1 - ( 5-phospho-D-ribosyl ) imidazole-4 - carboxamide from N ( 2 ) - formyl-N ( 1 ) - ( 5-phospho - dribosyl ) glycinamide ( step 5 ) See PurB : step 4 ; increases after benzoic acid treatment See PurB : step 2 ; increases after benzoic acid treatment See PurB : step 3 PstS P0AG82 Phosphate-binding periplasmic 6.92 / 34,421.76 6.85 / 45,258 ( 6 -- 11 ) protein Pta P0A9M8 Phosphate acetyltransferase 5.28 / 77,040.90 5.29 / 85,417 179 , 228 , 274 , 293 , 294 PtrA 179 Protease III P05458 5.66 / 105,113.29 5.65 / 9,119.37 5.65 / 10,366 P0AA04 Phosphocarrier protein HPr PtsH 27 , 228 , 293 , 294 PtsI 4.78 / 59,810 Phosphoenolpyruvate-protein phosphotransferase P08839 56 , 179 , 228 , 293 , 294 , 321 4.78 / 63,561.84 4.91 / 63,671 ( DIGE 4.5 -- 6.5 ) PurA P0A7D4 Adenylosuccinate synthetase 179 , 321 5.30 / 44,524 ( DIGE 4.5 -- 6.5 ) 5.30 / 43,494 ( DIGE 4.5 -- 6.5 ) 5.32 / 47,213.76 PurB 179 5.68 / 51,542.81 P0AB89 Adenylosuccinate lyase PurC 5.09 / 26,148 ( DIGE 4.5 -- 6.5 ) 5.07 / 26,995.00 P0A7D7 Phosphoribosylaminoimidazolesuccinocarboxamide synthase Phosphoribosylamine-glycine ligase Phosphoribosylaminoimidazole carboxylase catalytic subunit Bifunctional purine biosynthesis protein Phosphoribosylaminoimidazole carboxylase ATPase subunit 179 , 321 PurD 179 , 321 5.06 / 47,667 ( DIGE 4.5 -- 6.5 ) 4.96 / 45,940.36 P15640 PurE 179 , 321 6.05 / 18,899 ( DIGE 4.5 -- 6.5 ) 6.03 / 17,649.13 P0AG18 See PurB : ﬁnal step ; increases after benzoic acid treatment See PurB : step 3 ; possesses an ATPase activity that is dependent on the presence of AIR ( aminoimidazole ribonucleotide ) ; the association of PurK and PurE produces an enzyme complex capable of converting AIR to CAIR efﬁciently under physiological condition See PurB : step 2 PurH 179 , 321 5.50 / 57,000 ( DIGE 4.5 -- 6.5 ) 5.53 / 57,329.21 P15639 5.75 / 28,100 ( 5 -- 6 ) 5.60 / 39,461.12 P09029 PurK 287 179 P08178 Phosphoribosylformylgly-cinamidine cyclo-ligase HTH-type transcriptional-repressor 4.83 / 36,722.85 PurM Repressor that binds to the purF operator and coregulates other genes for de novo purine nucleotide synthesis ; involved in regulation of purB , purC , purEK , purHD , purL , purMN , and guaBA expression ; binds hypoxanthine and guanine as inducers 4.83 / 36,722.85 PurM P0ACP7 6.3 / 38,043.62 PurR 179 Oxidizes proline to glutamate for use as a carbon and nitrogen source and also functions as a transcriptional-repressor of the put operon ; induced by proline Involved in ﬁnal step of glycolysis Involved in UMP biosynthesis : UMP from HCO3 ( step 2 ) See PyrB : step 3 See PyrB : step 4 See PyrB : step 6 ( ﬁnal ) Bifunctional protein 5.69 / 143,815.16 PykF Pyruvate kinase I P0AD61 5.77 / 50,729.42 5.65 / 56,143 ( DIGE 4.5 -- 6.5 ) 179 , 321 PyrB Aspartate carbamoyltransferase catalytic chain Dihydroorotase Dihydroorotate dehydrogenase Orotidine 5 -- phosphate decarboxylase CTP synthase P0A786 6.13 / 34,296.17 5.70 / 93,664 6.13 / 35,321 179 , 212 , 228 , 294 P05020 P0A7E1 P08244 PyrC 5.77 / 38,696.19 7.66 / 36,774.52 5.81 / 26,350.24 179 179 , 287 PyrD PyrF 7.28 / 47,786 ( 6 -- 11 ) 5.76 / 56,695 179 , 228 , 294 , 321 5.83 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.61 / 58,475 ( DIGE 4.5 -- 6.5 ) Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen ; allosterically activated by GTP when glutamine is the substrate and also activated by magnesium but inhibited by CTP and by divalent metal ions such as copper and zinc Catalyzes the phosphorylation of UMP to UDP , with ATP as the preferred donor Involved in allosteric regulation of aspartate carbamoyltransferase Essential for efﬁcient processing of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; cold-shock protein essential for E. coli cells to adapt to low temp PyrG P0A7E5 5.63 / 60,242.96 179 , 321 7.07 / 25,839.07 P0A7E9 Uridylate kinase PyrH PyrI P0A7F3 Aspartate carbamoyltransferase 6.84 / 16,989.44 6.85 / 17,023 regulatory chain 7.01 / 17,642 ( 6 -- 11 ) 179 , 212 , 228 , 287 , 294 RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 Involved in the high-afﬁnity D-ribose membrane transport system and also serves as the primary chemoreceptor for chemotaxis ; repressed in the presence of glucose but induced in the physiological short-term adaptation to glucose limitation Involved in ribose metabolism Catalyzes the hydrolysis of ATP in the presence of single-stranded DNA , the ATP-dependent uptake of single-stranded DNA by duplex DNA , and the ATP-dependent hybridization of homologous single-stranded DNAs ; interacts with LexA , causing its activation and leading to its autocatalytic cleavage ; induced by low temp at 10 ° C as well as by cadmium chloride , hydrogen peroxide , and ACDQ Counteracts the effect of relE by means of direct proteinprotein interaction , enabling the reversion of translation inhibition ; acts as an autorepressor of relBE transcription ; increased transcription rate of relBE and activation of relE is consistent with a lower level of relB in starved cells due to degradation of relB by protease lon ; induced by amino acid starvation RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 RbsB P02925 6.85 / 30,900 5.99 / 28,474.47 D-Ribose-binding periplasmic protein 99 , 179 , 311 , 321 E 4.5 -- 6.5 ) IG 5.92 / 29,066 ( D 4.99 / 32,290.52 P0A9J6 Ribokinase P0A7G6 Protein RecA RbsK RecA 179 5.09 / 37,842.18 5.08 / 41,137 133 , 179 , 228 , 294 , 297 4.81 / 9,071.48 4.81 / 9,100 Antitoxin P0C079 RelB 99 Continued on facing pag Catalyzes the interconversion between ADP-D-glycero-beta-D-manno-heptose and ADP-L-glycero-beta-D-manno-heptose via an epimerization at carbon 6 of the heptose ; completely inhibited by ADP and ADP-glucose and partially inhibited by ATP and NADH ; induced by heat-shock Involved in carbohydrate biosynthesis and dTDP-L-rhamnose biosynthesis See RfbB RfaD P67910 ADP-L-glycero-D-manno-heptose-6-epimerase 4.80 / 34,893.17 4.85 / 36,820 4.94 / 36,106 ( DIGE 4.5 -- 6.5 ) 4.98 / 35,245 ( 4 -- 5 ) P37759 RfbB 179 dTDP-glucose 4,6-dehydratase 5.47 / 40,558.33 RfbC P37745 dTDP-4-dehydrorhamnose 3,5 - 5.48 / 21,270.11 epimerase P0AG30 Transcription termination factor 6.75 / 47,004.21 179 Facilitates transcription Rho termination by a mechanism that involves rho binding to the nascent RNA , activation of rho 's RNA-dependent ATPase activity , and release of the mRNA from the DNA template Involved in riboﬂavin biosynthesis ; repressed by heat-shock but induced by low pH Involved in ﬁnal steps of riboﬂavin synthesis ; riboﬂavin synthase is a bifunctional enzyme complex catalyzing the formation of riboﬂavin from 5-amino-6 - ( 1 - D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione and L-3 ,4 - dihydrohy-2-butanone-4 - phosphate via 6,7-dimethyl-8-lumazine ; the alpha subunit catalyzes the dismutation of 6,7-dimethyl-8-lumazine to riboﬂavin and 5-amino-6 - ( 1-D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione See RibE ; decreases after benzoic acid treatment Acetylates the N-terminal serine of ribosomal protein L7/L12 ; increases during phosphate limitation and phosphonate growth Essential for efﬁcient processing of 16S rRNA ; probably part of the 30S subunit prior to or during the ﬁnal step in the processing of 16S free 30S ribosomal subunits ; could be some accessory protein needed for efﬁcient assembly of the 30S subunit ; needed in a step prior to RbfA during the maturation of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; decreases after benzoic acid treatment P0A7J0 3,4-Dihydroxy-2-butanone 4-phosphate synthase 4.90 / 23,353.47 RibB 274 RibE P0AFU8 Riboﬂavin synthase alpha chain 5.64 / 23,444.90 5.67 / 26,821 ( 5 -- 6 ) 5.73 / 25,731 ( 5.5 -- 6.7 ) 179 , 287 RibH 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.19 / 12,875 ( 4.5 -- 5.5 ) 5.42 / 40,651 5.15 / 16,156.51 P61714 6,7-Dimethyl-8-ribityllumazine synthase Ribosomal protein-serine acetyltransferase 179 , 287 , 321 5.86 / 20,680.55 P13857 RimL 228 , 293 , 294 4.68 / 24,436 ( DIGE 4.5 -- 6.5 ) 4.61 / 20,605.43 P0A7X6 16S rRNA-processing protein RimM 321 179 321 5.09 / 35,712.71 5.39 / 32,693.56 Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA Catalyzes the formation of dTDP-glucose from dTTP and glucose-1-phosphate as well as its pyrophosphorolysis ; decreases after benzoic acid treatment Digests double-stranded RNA ; involved in the processing of rRNA precursors and of some mRNAs Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA 5.41 / 32,544 ( DIGE 4.5 -- 6.5 ) RmlA1 6.4 / 25,550.04 P0A7Y0 RNase III Rnc 179 Matures 5S rRNA from its precursors from all the rRNA genes ; cleaves RNA I , a molecule that controls the replication of colE1 plasmid DNA ; the major endoribonuclease participating in mRNA turnover in E. coli May be part of a membrane complex involved in electron transport Responsible for the end turnover of tRNA : speciﬁcally removes the terminal AMP residue from uncharged tRNA ( tRNA-C-C-A ) ; involved in tRNA biosynthesis , especially in strains lacking other exoribonucleases P21513 Rne RNase E 5.48 / 118,196.73 Electron transport complex protein P77285 RnfG 179 6.59 / 21,911.91 RNase T P30014 Rnt 179 5.19 / 23,522.7 5.13 / 24,554.25 Ribulose-phosphate 3-epimerase Ribose-5-phosphate isomerase A P0AG07 Rpe 179 Involved in nonoxidative branch of the pentose phosphate pathway ; base-induced protein One of the primary rRNA-binding proteins ; binds very close to the 3 end of the 23S rRNA and forms part of the L1 stalk ; translational repressor protein ; controls the translation of the L11 operon by binding to its mRNA See RplA ; increases at phosphonate growth RpiA 5.20 / 22,860.40 5.06 / 25,943 P0A7Z0 179 , 274 , 286 RplA 50S ribosomal protein 9.64 / 24,598.44 8.24 / 25,577 ( 6 -- 11 ) L1 P0A7L0 179 , 287 RplI 6.20 / 19,831 6.17 / 15,769 6.17 / 15,769.06 50S ribosomal protein L9 P0A7R1 29 , 179 , 228 , 286 , 287 , 293 , 294 6.71 / 10,366 5.10 / 8,830 ( 4.5 -- 5.5 ) 6.36 / 15,124 ( 6 -- 11 ) 9.04 / 17,580.4 Translational repressor protein ; controls the translation of the rplJL-rpoBC operon by binding to its mRNA Seems to be the binding site for several of the factors involved in protein synthesis and appears to be essential for accurate translation ; decreases during phosphate limitation Binds the 5S rRNA ; required for the late stages of subunit assembly Binds to 23S rRNA in the presence of protein L20 One of the proteins that binds to the 5S RNA in the ribosome , where it forms part of the central protuberance ; binds to the 5S rRNA independently of L5 and L18 ; not required for binding of the 5S rRNA/L5/L18 subcomplex to 23S rRNA May be induced under zinc-limiting conditions ; may be repressed in that case by the zinc uptake regulation protein Zur DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Regulatory regions studied in this work Identiﬁcation ‰ Predicted best DNA site for CRP Position Score ROMA ChIP-Chip w z Gene Gene function ATCTGTGAGTGATTTCACAGTA 259 6.654 Yes No mglB Galactose-binding transport protein , receptor for galactose taxis gatY Tagatose-bisphosphate aldolase 1 TTTTGTGATCGTTATCTCGATA aer Aerotaxis sensor receptor AATTGCGATCTAAATCAAATTA 7.895 Yes 4.054 No No Yes 64 93 sdaC Serine transporter ATTTGAGATCAAGATCACTGAT ychH Hypothetical protein AATTGTGATCACGCCCGCACAT 7.618 Yes 6.771 Yes Yes No 168 94 malX PTS system , maltose and glucose-speciﬁc IIABC component TAATGTGATTTATGCCTCACTA yffQ Hypothetical protein TTTTGTGATGCAGATCGCTTTT 7.892 Yes 4.098 Yes Yes Yes 83 145 ycdZ Hypothetical protein AAGTGTGATCTACGTCACTCAT fdoG Formate dehydrogenase-O major subunit ACCTGTGATATTTGTCACATTT 5.461 Yes 6.281 No No Yes 44 237 ribB Riboﬂavin biosynthesis ; 3,4-dihydroxy-2-butanone 4-phosphate synthase Yes No yidW Regulator protein for dgo operon TTTTGTGATCTAAATTGTAGTA 144 7.852 Yes No slp Outer membrane protein induced after carbon starvation AAAATTAACTGAGTTCACCTAA 100 11.96 No Yes Sites as predicted by Robison et al. ( 1998 ) ( ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional-regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine metabolism ) was controlled by protein regulators such as FNR ( transcriptional-regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>It is therefore tempting to speculate that MalT might activate transcription initiation by a mechanism different from that used by CI or CRP to activate PR or lacZp ( de Crombrugghe et al. , 1984 ; Ptashne , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2524384" target="_blank">2524384</a>]<br></li>
<li>Furthermore , FNR-AR1 is proposed to interact with the a-C-terminal domain ( a-CTD ) of RNAP to activate transcription as has been observed with CRP ( Ebright , 1993 ; Busby and Ebright , 1997 ; Williams et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>This is surprising because it was expected that tandem-bound FNR dimers would act synergistically to activate transcription , as has been reported for the closely related transcription factor CRP ( Savery et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>However , it is possible that the change in transcription activity of PguaB in a fis background occurs as a result of altered regulation of PguaB by a transcription factor other than CRP , for example H-NS or HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ) or by changes in supercoiling ( Schneider et al. , 1997 ; Weinstein-Fischer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>Fis transcription is subject to a plethora of control mechanisms including , autoregulation , stringent control , the availability of its initiating NTP and the CRP transcription factor ( Ball et al. , 1992 ; Ninnemann et al. , 1992 ; Nasser et al. , 2001 ; Walker and Osuna , 2002 ; Mallik et al. , 2004 ; Walker et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Results Our previous studies revealed that cells regulate acs transcription through the combined action of multiple proteins , including , but not limited to , IHF and CRP ( Kumari et al. , 2000a ; Browning et al. , 2002 ; 2004 ; 2005 ; 2006 ; Beatty et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Expression of FlhDC is tightly regulated by multiple transcription factors , including CRP , H-NS and OmpR , and hence the genes of the ﬂagellar cascade are regulated in response to many different environmental inputs ( Shin and Park , 1995 ; Sout-ourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>More than ten transcription regulators including EvgA , YdeO , GadE , TorR , H-NS , PhoP , RpoS , CRP , MnmE , GadX , and GadW have been found to participate in the regulation of gadE during various circumstances of acid stresses through at least ﬁve different regulatory circuits ( Ma et al. , 2004 ; Sayed and Foster , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>These results , in conjunction with those from footprint analysis , suggest that CRP and FIS can bind simultaneously to CRP II , FIS II and CRP I. IHF binds the acs regulatory region simultaneously with either CRP or FIS Previously , we showed that IHF in vitro binds to a DNA site ( IHF I ) that overlaps acsP1 and in vivo represses transcription from both acsP1 and pnrfA ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>To achieve maximal transcription , however , a second dimer also must bind to CRP II , the distal , lower-affinity site ( Beatty et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>CRP binding upstream of aer was ﬁrst detected by Grainger et al. ( 2005 ) in a whole genome chromatin immunoprecipitation analysis , and it was subsequently shown that CRP activates transcription by binding to a single DNA site in the aer regulatory region ( Hollands et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>However , the position of the CRP site 1 , which is centred on position − 76.5 , is not expected to allow direct contacts with the polymerase : when acting alone , CRP indeed activates initiation of transcription by contacting the polymerase , but its ability to activate requires its binding site be located on the same face of the DNA helix as the polymerase binding site , i.e. at positions − 41.5 , − 61.5 , − 71.5 , − 82.5 or − 92.521 bp ( Gaston et al. , 1990 ; Ushida & Aiba , 1990 ; Valentin-Hansen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In a study by Gaston et al. ( 19 ) , CRP binding at the 83.5 center position did not lead to any CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>We have also examined the reciprocal co-dependence of binding of CaiF and CRP , which are both absolutely required for the activation of transcription of the two operons ( Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Regarding this sites we note that its location is suboptimal for CRP-mediated transcription activation ( Gaston et al. , 1990 ) and , furthermore , it contains suboptimal ﬂanking sequences for bending , and thus for binding of CRP ( Gar-tenberg and Crothers , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>SELEX-clos allows the screening of DNA sequences with high afﬁnity to the test transcription factor , but it is not practical for identiﬁcation of the whole set of its binding targets because the number of isolated clones correlates with the binding afﬁnity as demonstrated for a variety of transcription factors such as Cra , CRP , and LeuO ( Shimada et al. 2009 , 2011a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>In summary , during the early stages of exponential phase , FIS inhibits acs transcription by steric hindrance of CRP binding ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Mutants of CRP that bind to the target site on the DNA , but fail to activate transcription ( positive control or pc mutants ) , have been used to show that a particular region ( activating region I , ARI ) located at the surface of the known structure of the CRP dimer is responsible for activation at lac-type promoters ( Bell et al. , 1990 ; Eschenlauer & Reznikoff , 1991 ; Williams et al. , 1991 ; Zhou et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>A variety of evidence indicates that transcriptional activation by CRP at both Class I and II promoters involves direct contact with RNA polymerase , including the identification of amino acid substitutions in CRP which result in retention of DNA binding but which can not activate transcription ( reviewed by Ishihama , 1992 ; Ebright , 1993 ; Busby and Ebright , 1994 ; Busby et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Despite the loose distance requirement , activation of transcription can take place only when CRP and RNA polymerase bind to the same face of the DNA helix ( Gaston et al. , 1990 ; Ushida and Aiba , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Our in vitro studies have shown that MelR alone binds to site 1 cents , site 1 and site 2 , but that occupation of site 2 cents , which is essential for transcription activation , requires the presence of melibiose and CRP ( Williams et al. , 1994 ; Belyaeva et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>These results verify that activation of acsP2 transcription requires CRP I ( Beatty et al. , 2003 ) and shows that IHF-dependent inhibition requires CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In vivo studies have clearly established that both CRP and CaiF are required for the activation of transcription of the cai and ® x operons ( Eichler et al. , 1995 ; 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Similarly , activation of transcription in vivo required CRP , its AR1 surface and an intact CRP I. On the basis of these observations , we proposed that CRP ` focused ' RNAP to acsP2 ( Beatty et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>This is consistent with previous observations ( West et aL , 1993 ) that these substitutions suppress at Class II promoters where CRP is bound around - 4 1 , but not at Class I promoters where CRP is bound further upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This is in contrast to the situation at previously studied Class I and Class II CRP-dependent promoters where the optimal locations for activation by CRP are positions -61.5 and -41.5 respectively , and where CRP activates only very weakly when bound to a site centred near position -50 ( Gaston et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In conclusion , the structure of the cdd promoter supports and extends the idea that CytR and CRP are rivals and , consequently , that bound CytR repressor inhibits RNA synthesis by antagonizing CRP activation -LCB- Martinussen et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>In support of this model , Dethiollaz et al. ( 1996 ) have recently detected such DNA ± RNAP interactions in the transcriptional activation process mediated by CRP at the malT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512708" target="_blank">9512708</a>]<br></li>
<li>In the case of the lac promoter , a surface-exposed loop around amino acids 158-161 of CRP constitutes an activating region ( ARI ) essential for transcriptional activation ( Bell et al. , 1990 ; Eschenlauer and Reznikoff , 1991 ; Zhou et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Binding reactions contained 5 ml of each DNA template ( 20 nM ) in buffer B ( without DTT ) and 5 ml of 400 nM CRP or buffer A and were kept at room temperature for 15 min ( this CRP concentration was previously found to be optimal for activation of the csiD promoter ; Marschall et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Transcription activation at the aer promoter requires CRP binding at an atypical location Mutational analysis showed that CRP-dependent activation of the aer200 : : lacZ fusion requires CRP binding to the single DNA target indicated in Fig. 2B ( Hollands et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>At the divergently transcribed malE , malK promoter , the active form of the CRP protein is also required for transcriptional activation ( Raibaud et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9822819" target="_blank">9822819</a>]<br></li>
<li>IHF has the ability to bend DNA to a similar degree as does CRP ( Kosturko et al. , 1988 ) and thus allowed us to analyse the respective contributions of protein-protein contacts and DNA curvature to the mechanism of transcriptional activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The latter would be consistent with the ® nding that EsS can also support CRP activation ( albeit less ef ® ciently than Es70 ) of the galp1 and lac promo-ters , where CRP boxes are centered around ÿ41 .5 and ÿ61 .5 , respectively ( Kolb et al. , 1995 ; Tanaka et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The H1 59L mutation in ARI drastically affects transcriptional activation by CRP ( Bell et al. , 1990 ; Zhou et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Elegant studies by Raibaud , Richet and colleagues have shown that CRP serves to reposition MalT to an alternative set of T boxes that results in activation ( Richet et al. , 1991 ; Richet and Sogaard , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9822819" target="_blank">9822819</a>]<br></li>
<li>Peaks of maximal transcriptional activation occur in ViVo when CRP is bound at positions -60.5 , -71.5 and -81.5 ( Déthiollaz et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Merkel et al. ( 17 ) found that CRP AR1 substitutions were not defective for uhpT activation , suggesting that CRP activation of uhpT does not depend on the previously deﬁned a-CTD -- AR1 interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Their activation depends on the synergistic action of MalT , the activator of the maltose regulon ( Schwartz , 1987 ) , and CRP , a global regulator controlling the utilization of carbon sources ( Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Activation by CRP is strongest at the positions corresponding to the natural promoters lac and malT Comparison of the -81.5 activation peak with natural promoters is difficult because such promoters are generally activated by CRP only in conjunction with a secondary activator ( Kolb et al. , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>To ` proﬁle ' CRP-dependent activation of candidate promoters , b-galactosi-dase activities were also measured in M182 Dcrp cells carrying pDCRP derivatives encoding wild-type or mutant CRP , as described by Rhodius et al. ( 1997 ) , with pDU9 encoding no CRP as a control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Nevertheless , FNR-AR1 is also expected to be required in the activation of class II promoters , as AR1 of CRP is required for the activation of class II promoters via its interaction with the RNAP a-CTD ( Fig. 1 ) ( Lombardo et al. , 1991 ; Busby and Ebright , 1997 ; 1999 ; Wing et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Anti-inhibition is through an interaction between the AR1 of CRP and a-CTD , which overcomes an inhibitory effect of a-CTD ( Figure 1 ) ; thereby , the CRP activation of class II CRP-dependent promoter is unaltered by the mutant RNP devoid of a-CTD ( Attey et al. , 1994 ; Zhou et al. , 1994 ; Belyaeva et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>If this is so , then activation of dad P2 by Lrp-alanine may provide suf ® cient expression of dadX under conditions in which levels of cyclic AMP-CRP , which activates P1 ± P3 ( Zhi et al. , 1998 ) , are low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>CyaR is positively regulated by cyclic AMP and CRP , while Spot 42 is negatively regulated ( Møller et al. , 2002 ; Johansen et al. , 2008 ; Papenfort et al. , 2008 ; De Lay and Gottesman , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22040174" target="_blank">22040174</a>]<br></li>
<li>Furthermore , the ACS enzyme is positively regulated by CRP ( generating acetyl-CoA ) , high levels of AMP , and FNR ( at low oxygen partial pressure ) , whereas it is negatively regulated by the j2 factor ( Kumari et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>These sites are designated CRP-S sites because although CRP can still bind to these sites with low affinity , CRP alone is not sufficient to induce expression from promoters containing CRP-S sites , but also requires Sxy ( Redﬁeld et al. , 2005 ; Cameron and Redﬁeld , 2006 ; 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>In contrast to other maltose operon promoters , which are regulated only by MalT : malEp and malKp are also positively controlled by CRP , a pleiotropic regulator involved in the control of carbon source utilization ( De Crombrugghe et al. , 1984 ; Chapon , 1982 ; Richet & Raibaud , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>This simultaneous CRP control is not unique to the mhp pathway , since it has been also described for the genes for propionate catabolism in E. coli and Salmonella enterica ( Lee et al. , 2005 ) and for some regulons from E. coli such as those for L-rhamnose ( Holcroft & Egan , 2000 ; Wickstrum et al. , 2005 ) , melibiose ( Webster et al. , 1988 ; Belyaeva et al. , 2000 ) and maltose ( Chapon & Kolb , 1983 ; Richet & Sogaard-Andersen , 1994 ; Richet , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Results Two global regulators , CRP and Cra , of E. coli recognize and regulate as many as 378 and 164 promoters , respectively , of the genes for carbon source utilization ( Shimada et al. , 2005 , 2011a , 2011b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>CRP activates transcription from over 150 different promoters in E. coli ( Kolb et al. , 1993 ; Zheng et al. , 2004 ) and this CRP regulon includes several operons mainly involved in catabolism of carbon sources other than glucose ( Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Involvement of EvgA , YdeO , H-NS , CRP , GadX , as well as a small RNA DsrA in the regulation of the mdtEF gene in the same operon has also been reported ( Nishino and Yamaguchi , 2002 , 2004 ; Oshima et al. , 2006 ; Nishino et al. , 2008a , b , 2009 , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>INVOLVEMENT OF DIFFERENT REGULATORS IN THE ANAEROBIC EXPRESSION OF mdtEF FROM THAT OF AEROBIC CONDITION Previous studies focusing on the expression of gadE-mdtEF operon during acid resistance have revealed that several regulators , such as EvgA , YdeO , MnmE , GadE , GadX , CRP , and H-NS , are involved in the regulation of gadE-mdtEF operon ( Gong et al. , 2004 ; Ma et al. , 2004 ; Hirakawa et al. , 2006 ; Tramonti et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>FruR ( Cra ) is a transcriptional-regulator which , together with CRP , controls the expression of genes involved in C metabolism , particularly with respect to the direction of metabolic flow , glycolysis being repressed whereas gluconeogenesis is activated ( Ramseier et al. , 1993 ; Saier & Ramseier , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>M o r e recently , studies with lac2 fusions have shown that sdh gene expression is anaerobically repressed some 70-fold by the combined effects of ArcA and FNR , a n d repressed up to threefold by glucose in a CRP - ( and FruR - ) independent manner ( Iuchi et al. , 1994 ; Park et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>For example , the oxidative acid resistance system is regulated by rs and CRP ( Foster 2004 ) , and rs regulates both gene expression for osmotic stress ( Hengge-Aronis 1996 ) and regulates many acid resistance genes including the uncharacterized ygiW and yodD genes ( Weber et al. 2005 ) identiﬁed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>With increasing acetate formation , expression of many genes known to be differentially expressed in glucose-grown and acetate-grown cells ( Oh et al. 2002 ) or in cells lacking CRP ( Gosset et al. 2004 ) decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>For example , the PEP carboxykinase gene pckA , the tryptophanase gene tnaA , and the aspartase gene aspA showed lower expression when acetate is formed in overflow metabolism ( Fig. 1 ) and their expression is lower in glucose-grown cells than in acetate-grown cells ( Oh et al. 2002 ) and it is lower in cells lacking CRP ( Gosset et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>By contrast , repression complex formation requires a CRP-CRP centre-to-centre distance of strictly 52 or 53bp ( SQgaard-Andersen et al. , 1990 ; Hoist et aL , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Conclusions Members of the LacI -- GalR family of transcriptional-repressors are usually functional as dimers , although in some cases , repression depends on the dimerization of dimers or interactions with other proteins , such as CRP ( Weickert & Adhya , 1992 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>CRP regulation There are experimental data showing that the expression of gntKU and gntT from the GntR regulon and uxuAB from the UxuR/ExuR regulon is under catabolite repression by CRP ( Izu et al. , 1997 ; Peekhaus and Conway , 1998b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>In E. coli , CRP is the global regulator of catabolite repression and previously was shown to negatively regulate Spot 42 RNA and positively regulate CyaR RNA expression ( De Lay & Gottesman , 2009 , Johansen et al. , 2008 , Polayes et al. , 1988 , Papenfort et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>In the previous paper , we showed that the intracellular CRP level is markedly reduced by glucose and that this reduction in CRP level is an important determinant of catabolite repression ( Ishizuka et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Gosset et al completed a similar in vivo transcriptomic study of CRP and identiﬁed 39 operons under CRP-dependent glucose repression and 19 operons under CRP-dependent glucose activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>One-third of operons subject to CRP-dependent glucose repression ( 13/39 ) in Gosset et al. were activated by CRP in ROMA experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP activates both the psAD and P , promoters from a single site located between araO , / Pc and aral ( Lee et al. , 1981 ; I ) unn & Schleif , 1984 ; Stoltzfus & Wilcox , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>All three appea ) r to he activated by a single CRP site , since it , s partial deletion abolished all basal and starvation-inducible activity as occurs in a Acya or Acrp mutant , ( Schukz et al. , 1988 ; Schultz & Matin , 1988 ; Blum c , t al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>In particular , it is noteworthy that the crp gene is under the control of Cra because we identiﬁed more than 200 novel targets for cAMPbound CRP ( Shimada et al. , in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In spite of this , Won et al. [ 22,23 ] obtained low resolution CRP backbone NMR assignments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In contrast , most PTS-E2 systems are under the control of CRP ( Shimada et al. , in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>In addition , PhlyE was characterized as a class I promoter that could be activated by the oxygenresponsive regulator FNR and/or the glucose-responsive regulator CRP ( Green and Baldwin , 1997 ; Ralph et al. , 1998 ; Westermark et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In addition , both FNR and CRP contribute to regulation of hlyE expression ( Green and Baldwin , 1997 ; Ralph et al. , 1998 ; Westermark et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Increasing CRP concentration progressively decreased modiﬁcation at P2A , while generally increasing modiﬁcation at P2UP , as we observed previously ( Beatty et al. , 2003 ) , especially at -6 and -7 ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>LuxS was previously shown by Wang et al. ( 65 ) to be negatively regulated by crp , but it lacked a sequence upstream of the coding sequence for luxS that was able to-bind CRP in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18978044" target="_blank">18978044</a>]<br></li>
<li>As far as we are aware , this is the first report of a direct interaction between CRP and an AraC family member ( although there may be a parallel with the interaction between CRP and CaiF reported by Buchet et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>This approach has led to several models for how CRP might interact and for the structure of the interaction sites ( Gilbert , 1976 , O'Neill et al. , 1981 ; McKay and Steitz , 1981 ; Queen and Rosenberg , 1981 ; Ebright , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>This observation supports the hypothesis by Krueger et al. [ 37 ] that the conformational change of CRP would be driven from a relatively open to closed conformation ( refer to Section 3.4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>FNR is structurally related to CRP and , like CRP , binds to a 22bp sequence that accommodates two FNR subunits ( Green et al. , 1991 ; Sharrocks et aL , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Thus , the major regulation of adenylate cyclase by CRP appears to be exerted posttranscriptionally ( reviewed by Bot-sford and Harman 1992 ; Postma et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Our data also conﬁrm that adenylate cyclase is regulated posttranscriptionally by CRP as previously noted ( reviewed by Botsford and Harman 1992 ; Postma et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Of the eight V.cholerae promoters , two ( purE and uraA ) are members of the PurR regulon predicted by TractorDB and by Ravcheev et al. ( 41 ) , and are also regulated by both CRP and PurR in H.inﬂuenzae ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Mgl-BAC is involved in the galactose transport , GalR and GalS negatively regulate MglBAC , and GalR and GalS are regulated by the global regulator CRP ( Semsey et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>Moreover , almost all genes from the KdgR regulon are regulated by CRP in E. chrysanthemi ( Nasser et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Conversely , CRP can repress ﬁs , which helps to ensure that FIS levels continue to diminish ( Nasser et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Fis hinders access of CRP to CRP I ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The CytR anti-activator and CRP act in concert to repress transcription from genes encoding transport proteins and enzymes necessary for uptake and catabolism of ribonucleosides and deoxyribonucleosides ( reviewed by Hammer-Jespersen , 1983 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Keywords : CRP -- FNR protein superfamily , iron -- sulphur proteins , transcriptional regulation , nitrogen starvation , stationary phase INTRODUCTION The CRP -- FNR family of transcription regulators contains at least 70 members that respond to a variety of metabolic and environmental signals to control a diverse range of physiological functions in an equally diverse range of bacteria ( Spiro , 1994 ; Guest et al. , 1996 ; Vollack et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The inability of CRP to activate malEp on its own is most likely due to the fact that none of the CRP sites is located at a permissive position ( Figure 1 ; Gaston et al. , 1990 ; Ushida & Aiba , 1990 ; Valentin-Hansen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The role of IHF has remained less clear : while in vitro it is known to-bind simultaneously with CRP and to cause inhibition of acsP2 transcription , in vivo the upstream sequence containing the three IHF sites has a positive effect on transcription ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The idea that CRT ' acGvates pBAo by opening the repression loop was further supported by an in vitro study that showed that the requirement for CRP in the transcription of pRAD from supercoiled templates is greatly reduced by eit ' her deleting ara0 , or by misorienting the ara0 , and nrnf sites ( Hahn et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The position of these sites and the involvement of certain CRP and RNAP surfaces support a mechanism by which tandem CRP proteins synergistically activate acsP2 transcription ( Beatty et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The major effect of CRP at the gal promoter is acceleration of the rate of isomerization to the open complex ( increase in k2 ) ( Herbert et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The promoter of the pdhR-aceEF-lpd operon is repressed by pyruvate-sensing PdhR ( 33 ) and Cra ( this study ) and is activated by CRP ( 55 ; T. Shimada et al. , manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The Pl promoter is activated by CAMP-CRP ( Muss0 et al. , 1977b ) , whereas P2 is inhibited by CAMP-CRP and is stimulated by glycerol ( Nakanishi et al. , 1974 ) or by superhelicity of the DNA template ( C. Queen , K. McKenny and M. Rosenberg , manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>The mechanism by which CRP activates independently of repression looping in araCRAD ( Hahn et al. , 1986 ; Lichenstein et al. , 1987 : Stolzfus & Wilcox , 1989 ) , is likely to be different from that in other systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The activity of sS is ﬁne tuned by a number of global regulators , including CRP , Lrp , IHF , H-NS and Crl ( Hengge-Aronis , 1996b ; Loewen et al. , 1998 ; Pratt and Silhavy , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>The expression of acs encoding acetyl-coenzyme A synthetase is controlled by the regulator proteins CRP , Fis and IHF ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>The mutant CRP proteins ( CRP KE101 and CRP HL159 ) were prepared , as described previously ( Ghosaini et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The direct involvement of CRP in their regulation may therefore simply be a reﬂection of the role of CRP in facilitating the acquisition and metabolism of nutrients other than glucose ( Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The csgDEFG operon is part of the CRP regulon ( Zheng et al. 2004 ) and thus is activated in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23824318" target="_blank">23824318</a>]<br></li>
<li>It has been shown previously that , in the presence of an UP-element immediately downstream of the CRP box , CC promoter activity obtained with Es is approximately 70 ﬁvefold higher when the CRP box was located at 269.5 than at 272.5 or even 261.5 , i.e. the presence of a full UP-element shifted the optimum for CRP box location to 269.5 ( Law et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>It has been shown that relief of H-NS-mediated hlyE silencing by SlyA is much less efficient in the absence of CRP as the hlyE promoter normally needs CRP or FNR to be activated to higher levels ( Westermark et al. , 2000 ; Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>P1 promoter activity is controlled by medium osmolarity and negatively regulated by CRP ( Culham et al. 1993 ; Mellies et al. 1995 , this paper ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>CRP , CpxR , CsgD , H-NS , IHF , MlrA , OmpR , and RstA on the csgD promoter ( Ogasawara et al. 2010a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>Recall that , unlike the malX promoter , the malI promoter is active in the absence of CRP ( Lloyd et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>CRP and FIS are putative effectors of the hup promoters We have previously observed , using gene fusions to malPQ , that both hup transcription contro sequences are of comparable or greater strength than the induced malP promoter when these cells are grown in M9 medium supplemented with Casamino acids and maltose ( Bonnefoy et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>These results are consistent with those of Musso et al. ( 1977a ) and demonstrate that , on the pKgalS template , gal transcription initiates from the Pl promoter in the presence of CAMP-CRP and from the P2 promoter five base pairs upstream in the presence of glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>Discussion Beatty et al. ( 2003 ) showed that CRP binds tandem sites located upstream of the major promoter acsP2 to activate its transcription synergistically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>This is because CRP is required to activate transcription of the ﬂhDC operon that encodes an essential activator of transcription from the ﬂiA promoter ( Soutourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>However , it is conceivable that the reduced biofilm formation observed in several E. coli K-12 strains in the presence of glucose and in a CRP mutant strain ( Jackson et al. , 2002 ) might be the indirect result of decreased McaS activation of flhD and pgaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>When only one activator was added , low levels of PF ~ H transcription were obtained in the presence of CRP but not AraC protein , while previous studies have shown that transcription of Psst ) proceeds with AraC protein and not with CRP alone -LCB- Hahn et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Both cai and ® x operons require the presence of FNR and CRP for their expression , while they are partially repressed by H-NS ( Eichler et al. , 1995 ; Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>While the expression of the mhpR , melR and malT regulators depends only on the presence of CRP ( Webster et al. , 1988 ; Chapon & Kolb , 1983 ) , the expression of rhaSR and prpR depends on the presence of CRP and the transcription activators RhaR and PrpR , respectively ( Tobin & Schleif , 1990a , b ; Wickstrum et al. , 2005 ; Lee et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>While the requirement for MalT is absolute ( Bedouelle , 1983 ) , the dependence on CRP is less stringent : malEp and malKp still work at 20 % and lo % , respectively , in the presence of MalT alone ( Raibaud et al. , 1989 ; Vidal-Ingigliardi & Raibaud , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Also , these terminations were in the absence of CAMP-CRP , and it was proposed that in their presence , the laci mRNA would terminate further upstream ( Cone et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>We began by verifying previous reports ( Muss0 et al. , 1977a ) that the Pl transcripts are synthesized in vitro in the presence of CAMP-CRP and the P2 transcripts in the presence of glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>Transcription control of acs -- yjcH -- actP is complex and the intergenic region between acs and nrfA is bound by CRP , FNR , IHF , FIS and NarL or NarP and is folded into an ordered nucleoprotein structure that permits the two divergent promoters to be regulated independently ( Browning et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Of the two tandem promoters driving the expression of the crp gene coding for the global transcriptional-regulator , CRP , only one requires DNA negative supercoiling and is subject to stringent control ( GonzaÂlez-Gil et al. , 1998 ; Johansson et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>We emphasize , however , that the regulation of ® s expression is a complex phenomenon subject to growth-phase-dependent and stringent control , and involving in addition to IHF the effects of other factors such as FIS itself , CRP , the superhelical density of DNA and , perhaps , NTP pools ( this study ; Ball et al. , 1992 ; Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>This promoter is controlled by a number of global regulators such as CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , heat-shock proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) and H-NS ( Bertin et al. , 1994 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) , as well as by several environmental signals such as temperature control ( Qualding and Stocker , 1962 ; Adler and Templeton , 1967 ) , high concentrations of inorganic salts , carbohydrates or alcohols ( Li et al. , 1993 ; Shi et al. , 1993 ) , growth-phase ( Nishimura and Hirota , 1989 ) , DNA supercoiling ( Li et al. , 1993 ; Shi et al. , 1993 ; Kutsukake , 1997 ; Das et al. , 1998 ) , phosphatidylethanolamine and phosphatidylglycerol synthesis ( Shi et al. , 1993 ; Mizushima et al. , 1995 ) , cell cycle control ( Morgan et al. , 1995 ; Prüß and Matsumura , 1996 ; Mizushima et al. , 1997 ) and quorum sensing ( Sperandio et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>]<br></li>
<li>In those bacteria that possess more than one member of the DNR/FLP subgroup , problems due to cross-talk may not arise because the regulators appear to be expressed in response to different environmental stimuli ( Vollack et al. , 1999 ) and their active forms may therefore never coexist Despite possessing the same structural framework , differences in primary structure between CRP and FNR within , and probably outside , the DNA-recognition helix may sufficiently alter DNA-target recognition such that CRP and FNR are able to regulate distinct modulons whilst sharing the same cytoplasm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Concerning the regulation of CRP synthesis in vivo , early immunological studies suggested that the levels of CRP in cells do not vary under different genetic and nutritional conditions ( Guiso and Blazy , 1980 ; Guiso et ai , 1982 ) , However , the existence of the complex autoregulatory mechanisms of the crp gene suggests that CRP synthesis should be controlled and the levels of CRP may vary according to growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>This suggests that the lowered level of CRP in glucose-containing medium is primarily due to the reduction in crp mRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>In order to examine whether CRP is involved in this glucose repression , we constructed crp and cya mutants ( where cya adenylate cyclase gene ) , and we measured their PGA activities during growth in M9 minimal medium containing glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>This repression required a functional CRP as CAMP failed to repress ODC synthesis in vitro in an extract prepared from a crp - strain ; the addition of purified CRP to the clp - extract restored the ability of CAMP to repress ODC synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>These studies in vivo and in vitro demonstrate this by showing that CAMP and CRP repress ODC synthesis as measured by either the amount of enzyme or mRNA produced from the speC gene in the absence of other potential regulatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>i219 ) was generated by cloning the 1.3 kb Pst \ - Xho \ \ fragment of pGS81 into pGS1406 , For testing the effects of global regulators ( ArcA , FNR and CRP ) , monoiysogenic derivatives of the corresponding Alac parental , mutant , and plasm id-transformed mutant , containing each of the pdhR - , aceE - and / pd - / acZfusion phages , were constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>By these criteria , ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp were designated as global TFs in E. coli K-12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , it is concluded that ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp are truly global regulators , at least as far as the response to oxygen availability is concerned , in the sense that their outputs create the transcriptional framework upon which local regulators operate in response to specific unstable environments Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Such is the case for complex II in E. coli , with the complex II bio-synthetic operon ( sdhCDAB ) being negatively regulated by ArcAB and FNR in response to oxygen conditions ( i.e. anaerobic ) and positively regulated by CRP in the absence of glucose [ 108 -- 110 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>Transcriptional monocopy lacZ fusion studies showed that both operons are coexpressed during anaerobic-growth in the presence of carnitine , respond to common environmental stimuli ( like glucose and nitrate ) , and are modulated positively by the same general regulators , CRP and FNR , and negatively by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Like cai and ﬁx operons , the caiF gene appeared to be positively controlled by CRP and FNR and negatively controlled by H-NS and the NarL ( plus nitrate ) regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The acnA gene is positively regulated by Crp , SoxRS , FNR and repressed by ArcA [ 59 ] , whereas acnB is positively regulated by CRP and negatively regulated by ArcA , FruR and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Genes most highly repressed upon IbsC , ShoB , LdrD and TisB overexpression.a Average fold repression dusB/yhdG ﬁs yeeF apt efeU ( ycdN ) CRP , IHF , Fis tRNA dihydrouridine synthase B Transcriptional activator Putative transporter Adenine phosphoribosyl transferase Inactive ferrous iron permease 6.5 7.7 0.9 0.9 0.8 0.9 1.2 1.1 1.1 1.2 1.2 2.1 4.3 3.8 4.7 2.3 1.2 17.8 6.1 21.2 CpxR Inactive ferrous iron transporter Inactive ferrous iron transporter Porin Unknown efeO ( ycdO ) efeB ( ycdB ) ompF yagU 14.3 2.6 6.1 26.7 1.0 1.8 1.4 1.7 1.2 1.8 0.9 2.5 1.2 1.2 0.5 1.2 OmpR , EnvYd a. Genes and operons which are repressed 5-fold in three independent experiments are listed with the average fold repression for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>However , when using a CTD deletion mutant RNAP ( 1 -- 235 ) , CRP neither activated lacP1-6 nor repressed lacP1-6S1 ( lanes 5 -- 8 ) , suggesting that CRP - CTD interaction is also needed for CRP repression at lacP1-6S1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>An CTD-deleted mutant RNAP abolished not only the enhancement of transcription from the pa-rental lacP1-6 promoter but also CRP-mediated repression of transcription at lacP1-6S1 , suggesting that an CRP - CTD interaction is also the basis of repression at lacP1-6S1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Expressionfrom the suc promoter was repressed by IHF and partially activated by d8but it was not regulated by ArcA , FNR , CRP , FruR or Fis , and not repressed by glucose or anaerobiosis , indicating that the well-established catabolite and anaerobic repression of ODH synthesis is imposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>This showed that the acnB gene is subject to repression by ArcA , Fis and FruR and activation by CRP , whereas the a c n A gene is repressed by ArcA and FNR , activated by FruR , CRP , Fur and SoxRS , and initiated by a38in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Although increased transcription of uxuR in this case is explained by the relieved autorepression ( due to the presence of fructuronate ) and by the functionality of cyclic AMP-CRP ( due to the absence of glucose ) , this assumes `` unreasonable '' interference with the required induction of UxuR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Lanes : a , primer PE-1 alone ; b , primer extension products of RNA made in vivo ; remaining lanes , primer extension products of RNA made in vitro in the absence of cyclic AMP-CRP ( c ) , in the presence of 50 mM cyclic AMP and 40 nM CRP ( d ) , and in the presence of cyclic AMP-CRP and 50 nM Lrp ( e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>4 Transcript mapping and regulatory studies with a dctA-lacZ transcriptional fusion showed that the dctA gene is monocis-tronic , has a single transcriptional start site , and is activated by cyclic AMP receptor protein ( CRP ) in the absence of glucose , repressed by ArcA during anaerobiosis , and weakly activated by the recently identiﬁed DcuS-DcuR system ( 13 , 57 ) in the presence of C4-dicarboxylates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Together , these observations indicate that the structure of site I is constrained in quite a different form in the presence of RNAP and the activator protein , in a manner very reminiscent of signals observed in ternary complexes involving cyclic AMP receptor protein ( CRP ) and RNAP at the lac promoter .22 RNAP contacts FIS site III in the absence of FIS Remarkably , in the absence of FIS , RNAP interacts with the upstream activating sequence ( UAS ) , inducing structural modiﬁcations in site III ( at 118 and 2122 on the TS and NTS , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054813" target="_blank">12054813</a>]<br></li>
<li>This situation exactly parallels that of fumarate reductase and succinate dehydrogenase , in which the corresponding frdABCD and sdhCDAB operons are subject to the same differential regulation by FNR in one case and the cyclic AMP-CRP complex plus the anaerobic repressor in the other , and the enzyme complexes are specifically adapted to accept electrons from menaquinione in fumarate respiration ( fumarate reductase ) or to donate electrons to ubiquinone in aerobic respiration ( succinate dehydrogenase ) ( 11 , 17 , 24 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>This contrasts with fumA expression , which has recently been shown to be activated by the cyclic AMP-CRP complex and repressed anaerobically by the arcA ( dye ) - encoded regulator ( J. R. Guest and R. S. Buxton , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>This indicates that the cyclic AMP ( cAMP ) - CRP complex weakly activates dcuB expression in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Alternatively , some of the negative effects of Lrp may be the result of direct effects on cyclic AMP-CRP , an activator of dad that binds in proximity to Lrp ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>This positive and negative control by cyclic AMP-CRP is apparent in both the IHF + and IHF - strains ( Fig. 2A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>expression in vivo appears to be a composite of activation and inhibition by cyclic AMP-CRP of multiple ompB proCyclic AMP-CRP is negative for those moters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Transcript Ti was strongly inhibited by cyclic AMP-CRP , with the concurrent activation of a second transcript , T2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Perhaps the altered activity of the multiple ompB promoters due to cyclic AMP-CRP control leads to a change in the ratio of OmpR and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>unusual , since only in rare cases does cyclic AMP-CRP both activate and inhibit expression from an individual operon ( 1 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Tr1 is not made in the absence of cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Note that in the absence of cyclic AMP CRP does not interfere with P , expression ( Figure 5 ) demonstrating that the inhibition is a result of CRP-CAMP acting specifically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>( a ) CRP increases the dissociation rate of AraC from aral , on minicircles in a cyclic AMP-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Incubations contained 0.75 mM labelled MelR , 75 nM CRP and 200 mM cyclic AMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>which are achieved by an interplay of the activities of the two repressors as well as cyclic AMP and CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>These promoters , P1-P3 , overlap and are positively regulated in vivo by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>A number of in vivo experiments suggest that the ilvB operon is positively controlled by cyclic AMP and CRP ( 4 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>Cyclic AMP titration of WT and mutant forms of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>These transcripts appear to be reciprocally regulated by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>phosphotransferasesystem ; CRP , cyclic AMP receptorprotein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>Although the overall effect of cyclic AMP-CRP in vivo is positive , there is no evidence that increased expression of ompB leads to a differential increase in OmpF levels in relation to OmpC ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Although the mechanism of this effect is unknown , it has been hypothesized that it is indirect and due to the putative regulation of the ompB operon by cyclic AMP-CRP ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>These observations suggest a physiological basis for cyclic AMP-CRP regulation of ompB , since this control could , as previously postulated ( 53 ) , lead to the observed positive action by cyclic AMP on OmpF synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In vitro footprinting showed that cyclic AMP-CRP binds to a 34-bp site centered at -53 and at -75 in relation to the start sites of the major transcripts that are inhibited and activated , respectively , by this complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In the case of glpD , besides regulation by GlpR , the promoter is subject to regulation by cyclic AMP-CRP and by the ArcA / ArcB systems ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>In vivo primer extension analysis indicated that this control is complex and involves both positive and negative effects by cydic AMP-CRP on multiple ompB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In the current study , we present evidence that cyclic AMP-CRP is a direct effector of ompB in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_253">
Zhi , J. , Mathew , E. , and Freundlich , M. ( 1998 ) In vitro and in vivo characterization of three major dadAX promoters in Escherichia coli that are regulated by cyclic AMP-CRP and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></label>
<input id="act_similarity_253", type="checkbox">
<div class="ACT_similar">
<li>In vitro and in vivo characterization of three major dadAX promoters in Escherichia coli that are regulated by cyclic AMP-CRP and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li></div>
<li>70-Dependent Transcription ... 3 Activation of acsP2 by CRP ... 3 Negative Regulation by Nucleoid Proteins ... 3 Negative regulation by FIS ... 3 Negative regulation by IHF ... 3 Independent regulation of acs transcription by FIS and IHF ... 3 Other Regulatory Factors ... 3 Signals ... 3 OST-TRANSCRIPTIONAL CONTROL ... 3 Post-Transcriptional Control by the Csr System ... 3 Post-Translational Control by Acetylation-Deacetylation ... 3 AMP-ACS-Dependent Acetylation and Chemotaxis ... 3 CETATE SWITCH IN OTHER ORGANISMS ... 3 PTA-ACKA/AMP-ACS Acetate Switch of the Gram-Positive Bacterium B. subtilis ... 3 Acetate Assimilation by the PTA-ACK Pathway of an Industrial Corynebacterium sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>This activation by alanine or leucine was not simply relief of repression , as P2 transcription from a constitutive template was increased ® vefold compared with the basal level of transcription found in the absence of Lrp and the co-activator cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>This activation of Tr2 by Lrp plus alanine or leucine was best seen when transcription was performed in the absence of cyclic AMP-CRP ( Fig. 5B ) , a known activator of this transcript ( Zhi et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>In vitro transcription from the ilvB promoter was greatly increased by cyclic AMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>In vitro transcription in the presence of CRP and cyclic AMP resulted in a marked stimulation , of the putative ilvB leader transcript ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>Taken together , these data strongly suggest that the ilvB biosynthetic operon is negatively controlled by multivalent transcription termination and is positively regulated ` by cyclic AMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>Inhibition and activation of ompB transcription in vitro by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Expression of the melR gene depends on transcription activation by CRP , whose activity depends on cyclic AMP ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>Positive and negative effects of cyclic AMP-CRP on ompB transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Our in vitro and in vivo experiments indicate that cyclic AMP-CRP positively and negatively alters ompB transcription by differentially regulating multiple ompB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>279 , No. 52 , Issue of December 24 , pp. 54552 -- 54557 , 2004 Printed in U.S.A. lacP1 Promoter with an Extended 10 Motif PLEIOTROPIC EFFECTS OF CYCLIC AMP PROTEIN AT DIFFERENT STEPS OF TRANSCRIPTION INITIATION * Received for publication , July 29 , 2004 , and in revised form , September 21 , 2004 Published , JBC Papers in Press , September 22 , 2004 , DOI 10.1074 / jbc.M408609200 Mofang Liu , Susan Garges ‡ , and Sankar Adhya § From the Laboratory of Molecular Biology , Center for Cancer Research , N Bethesda , Maryland 20892-4264 The cyclic AMP receptor protein ( CRP ) , which activates transcription from the wild-type lacP1 promoter and most of its mutants , represses productive RNA synthesis from a lacP1 promoter variant that contains an extended 10 element , although CRP enhances RNA po-lymerase binding as well as open complex formation in both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Memorial University of Newfoundland , St. John 's , Newfoundland , CANADA AlB 3V6 Tel. 709-737-6885 ( Received May 15th , 1985 ) ( Revision received November 1lth , 1985 ) ( Accepted February 3rd , 1986 ) SUMMARY The speC gene encoding ornithine decarboxylase ( ODC ) in Escherichia coli is negatively regulated by CAMP and the CAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>Cyclic AMP Receptor Protein-Dependent Activation of the Escherichia coli acsP2 Promoter by a Synergistic Class III Mechanism Christine M. Beatty ,1 Douglas F. Browning ,1,2 Stephen J. W. Busby ,2 and Alan J. Wolfe1 * Department of Microbiology and Immunology , Stritch School of Medicine , Loyola University Chicago , Maywood , Illinois 60153,1 and School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom2 Received 12 February 2003/Accepted 16 June 2003 The cyclic AMP receptor protein ( CRP ) activates transcription of the Escherichia coli acs gene , which encodes an acetate-scavenging enzyme required for ﬁtness during periods of carbon starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>278 , No. 41 , Issue of October 10 , pp. 39755 -- 39761 , 2003 Printed in U.S.A. Kinetics of Transcription Initiation at lacP1 MULTIPLE ROLES OF CYCLIC AMP RECEPTOR PROTEIN * Received for publication , June 6 , 2003 , and in revised form , July 21 , 2003 Published , JBC Papers in Press , July 23 , 2003 , DOI 10.1074 / jbc.M305995200 Mofang Liu ‡ § , Geeta Gupte ‡ , Siddhartha Roy ‡ ¶ , Rajiv P. Bandwar , Smita S. Patel , and Susan Garges ‡ From the ‡ Laboratory of Molecular Biology , Center for Cancer Research , NCI , National Institutes of Health , Bethesda , Maryland 20892-4264 , the ¶ Department of Biophysics , Bose Institute , P-1 / 12 , C.I.T. Scheme VII M , Calcutta 700 054 , India , and the Department of Biochemistry , Robert Wood Johnson Medical School , Piscataway , New Jersey 08854 The cyclic AMP receptor protein ( CRP ) acts as a transcription activator at many promoters of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>P1 activation depends on protein-protein contact between the RNA polymerase a subunit and CRP ( 13,19 , also reviewed in 20 ) , while P2 transcription is repressed by CRP binding ( Fig. 6 , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>A very long stretch of amino acids in a completely extended conformation would be necessary to explain almost equal activation factors when CRP is placed at -60 or -80 solely on the basis of identical protein-protein interactions between CRP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>However , we also identiﬁed novel regulation , such as Zur repression of ykgMO , cyclic AMP response protein ( CRP ) repression of azuC , and CRP activation of ykgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Although it is outstanding that many transport operons are under the transcriptional regulation of activation by CRP , such coupling of CRP with many transport protein operons may be the result of selection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>0378-l 119/86 / $ 03.50 0 1986 Elsevier Science Publishers B.V. ( Biomedical Division INTRODUCTION The role of CAMP and its receptor protein ( CRP ) in activation of catabolite-sensitive operons is well established ( reviewed in Ullmann and Danchin , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>There are also built-in sites for activation by CRP and the AraC protein in its role of activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>receptor protein ( CRP ) or catabolite gene activator protein ( CAP ) of E. coil , and the homology suggested that FNR performs a role in anaerobic gene activation analogous to that of CRP in catabolite gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Finally , in case of deoP2-wt the ability of the CRP variants to interact with the tandem CRP sites is reduced considerably compared with CRP-wt protein and only modest activation and repression is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>It seems possible that the indirect activation is due to action of the FIS protein , expression of which is activated by CRP binding to three discontinuous sites [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Prominent examples are the bacterial nitrogen uptake regulating transcription factor , NtrC , which is phosphoryiated in its active form by NtrB ( Austin and Dixon , 1992 ; Sanders etai , 1992 ) , or the catabolite regulatory protein , CRP , which is activated by oAMP ( Botsford and Harman , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>It may be speculated that , along with the simple repression of genes encoding temporarily unneeded enzymes , these TFs perform another important biological function , that of keeping the expression of target genes on a controllable level when their transcription is strongly induced by the global catabolic activator protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>The RhaS autoregulation mechanism is unusual in that a transcriptional-activator protein functions to decrease expression of an operon by activating transcription , but not allowing a second activator protein -- in this case CRP -- to coactivate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Since the Pl mRNA , although present in twice the amount of the P2 mRNA , produced only half as much E protein , E must be translated four times more efficiently from P2 than from Pl This 4-fold difference was also demonstrated when only half as much of the CAMP-CRP reaction was used , so that equal amounts of the Pl and P2 mRNAs were translated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>This negative autoreguiation was shown to be operating in vivo by crp-iac fusion studies ( Garges and Adhya , 1983 ; Cossart and Gicquel-Sanzey , 1985 ) , Evidence for the negative autoreguiation of crp in vivo was aiso presented by quantitative analyses of crp mRNA -LCB- Okamoto and Freundlich , 1986 ) and CRP protein ( Okamoto et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Although we have demonstrated that the negative and positive autoregulations are operating in cells carrying the multicopy crp plasmid , how these autoregulatory mechanisms actually contribute to determining the levels of CRP protein in wild-type cells remains to be studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>We are currently testing this hypothesis by measunng the amounts of crp mRNA and CRP protein in cells under various nutritional and growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>As additional negative controls , under the same test conditions , adding 0.5 mM IPTG to induce protein CRP or Crl did not significantly affect the value of the specific GFP fluorescence of the strain MG1655/pPyncC-gfp/pCA24N-crp or MG1655 / pPyncC-gfp/pCA24N-crl ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18309357" target="_blank">18309357</a>]<br></li>
<li>Therefore , two of the four types of regulation found for the uidA gene also apply to the uidR gene since this regulatory gene is repressed by its own product and is positively controlled by the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>CRP-mediated regulation of gcvT-lacZ is dependent on the GcvA protein , and CRP stimulates expression of a gcvA-lacZ fusion about twofold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>It should be noted that regulation of L-asparaginase II by CRP is not via the FNR protein ( i.e. , regulation of FNR by CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>Its regulation is partially dependent on σS in a manner that is not understood , as well as on the catabolite repressor protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>Abbreviations : CRP , CAMPreceptor protein ; FNR , fumarate and nitrate reduction regulator ; IHF , integration host factor ; ODH , 2-oxoglutarate dehydrogenase ; PDH , pyruvate dehydrogenase ; SCS , succinyl-CoA synthetase ; SDH , succinate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>CRP protein was induced by the addition of 0.2 mM isopropyl - - D-thiogalactopyranoside ( IPTG ) , and the bound protein was eluted using 50 to 500 mM imidazole gradient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>This requirement for ATP is a property which , together with other features of the MalT protein ( its size and the asymmetry of its recognition sequence ) distinguishes it from the classical bacterial regulators such as the CI or the CRP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2524384" target="_blank">2524384</a>]<br></li>
<li>This change caused the fdn operon to be expressed aerobically , subject to CRP protein control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Transcription is activated by CRP and repressed by a multi-protein CRPzCytRzCRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank J. Krakow for the gift of CRP protein , R. Ebright and S. Busby for CRP plasmids , and Leah Corselli for puriﬁcation of the a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Reactions were brought to a final concentration of 0.5 , tg of AraC protein per ml and 1.0 p , g of CRP per ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>CRP , CAMP receptor protein ; Xgal , 5-bromo-4-chloro-3-indolyl - ~ - ~ - galactopyranosidc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>CRP is mediated by protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Band intensities corresponding to CRP protein or CRP-tRNA were quantiﬁed using ImageQuant software , and the efﬁciency of PTH was calculated as the ratio of CRP protein to CRP-tRNA plus CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24322300" target="_blank">24322300</a>]<br></li>
<li>Interestingly , the carbon catabolite repression protein ( CRP ) has been shown to regulate several types of CU ﬁmbriae expression including type 1 and Pap ﬁmbriae [ 10 , 91 , 92 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21344461" target="_blank">21344461</a>]<br></li>
<li>CRP binds to a site from about base ® 324 to ® 303 , and although its speciﬁc mechanism for regulating gcvT : : lacZ expression has not yet been identiﬁed , CRP likely interferes with repression by the GcvA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>]<br></li>
<li>Purification of CRP protein E. coli BL21 ( DE3 ) transformant with the CRP expression plasmid pCRP encoding His-tagged CRP was grown in LB broth , and protein expression was induced by adding 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>While CRP activates the expression of both genes , the FIS protein positively regulates the hupA gene but negatively regulates the tandem P2-P3 promoters of hupB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>It was demonstrated in several other systems such as the ara and mal regulons that CRP regulates speciﬁc promoters in conjunction with another regulator and , in addition , regulates the expression of the coregulatory protein itself ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Putrescine is a biosynthetic precursor of another poly ¬ C , L-arginine P , adenosine Abbreviations : ODC , L-ornithine decarboxylase ; AD decarbo × ylase ; CRP , cAMP receptor protein ; cAM -3 ' :5 ' - cyclicmonophosphate Offprint requests to : S.M. Boyle 0026-8925/82 / 0186/0482 / $ 01.2 We assayed ODC and ADC activities in extracts from cells grown under conditions known to modulate intracellular levels of cAMP : by varying the major carbon source ( Buettner et al. 1973 ; Pastan and Adhya 1976 ) , addition of cAMP to growth medium ( Perlman and Pastan 1968 ) , and in bacterial strains carrying mutations in the adenylate cyclase ( EC 4.6.1.1 ) structural gene ( cya ) or in the crp structural gene encoding the cAMP receptor protein ( Brickman et al. 1973 ; Fraser and Yamazaki 1978 , 1980 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>coli RNA polymerase holoenzyme , 100 nM CRP , 20 mM Tris '' HC1 pH 7.9 , 8 mM MgC12 , 250 pM cAMP , 44 mM L-arabinose , 100 / aM DTT , 100 / ~ M EDTA , 250 / ~ M ATP , GTP , 20 pM in the nonradioactive pyrimidine nucleotide triphosphate , 5 p.M in the [ a-a2p ] pyrimi-dine nucleotide triphosphate ; araC protein , when present , was at 100 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7019009" target="_blank">7019009</a>]<br></li>
<li>The CAP ( = CRP = cAMP receptor protein = catabolite activator protein ) binds , in the presence of cAMP , to a site adjacent to the distal NagC site ( Fig. 2 , Lanes 2 and 11 ) centred at -83.5 from the transcriptional start site although protection extends back as far as -100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Structural basis for cAMP-mediated allosteric control Nataliya Popovycha , b ,1 , Shiou-Ru Tzenga , b ,1 , Marco Tonellic , Richard H. Ebrighta , d , and Charalampos G. Kalodimosa , b ,2 aDepartment of Chemistry and Chemical Biology , Rutgers University , Piscataway , NJ 08854 ; bDepartment of Biomedical Engineering , Rutgers University , Piscataway , NJ 08854 ; cDepartment of Biochemistry , National Magnetic Resonance Facility at Madison , University of Wisconsin-Madison , Madison , WI 53706 ; and dHoward Hughes Medical Institute , Waksman Institute , Rutgers University , Piscataway , NJ 08854 Edited by Peter H. von Hippel , University of Oregon , Eugene , OR , and approved March 10 , 2009 ( received for review January 18 , 2009 ) The cAMP-mediated allosteric transition in the catabolite activator reported structures of CAP in the presence of cAMP ( 6 -- 8 ) , protein ( CAP ; also known as the cAMP receptor protein , CRP ) is a complete the structural basis for understanding the allosteric textbook example of modulation of DNA-binding activity by small - control of CAP , a prototypic transcriptional-regulatory factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19359484" target="_blank">19359484</a>]<br></li>
<li>The shaded box indicates the lacZ region derived from pMC1403 Translation of cya mRNA is not regulated by CRP To test whether CRP can regulate adenylate cyclase at the translational level , we introduced the cya-lacZ protein fusion plasmid ( pIT296 ) into * cya crp ` and * cya crp ~ cells and measured the levels of cya mRNA , cAMP and b-galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>The regulation of PTS system was most probably exerted by the global transcriptional control factors such as cAMP -- CRP complex and the catabolite repressor/activator protein ( Cra ) ( for reviews , see Saier and Ramseier , 1996 ; Titgemeyer and Hillen , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
<li>The results indicate that functional interaction of cAMP and CRP is required for maximal production of Cir and that in addition to regulation by iron and temperature , synthesis of the Cir protein is controlled either directly or indirectly by levels of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>To do that , we used the cAMP receptor protein ( CRP , bends DNA by about 901 Lawson et al. , 2004 ) and the integration host factor ( IHF , bends DNA by 140 -- 1601 Rice et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Class 2 operons are transcribed by s - RNA 70 polymerase in the presence of FlhD4C2 , which binds to the FlhD4C2 box DNA upstream of class 2 promoters ( Liu & Matsumura , 1994 ; Liu et al. , 1995 ; Ikebe et al. , 1999b ; Abbreviations : CRP , cAMP receptor protein ; 59-UTR , 59 untranslated region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>Interestingly , AraC-dependent transcription initiation at the araBAD promoter is increased by the cyclic AMP receptor protein ( CRP ) , a well-characterized global regulator , controlled by cyclic AMP ( cAMP ) , that activates the expression of a number of genes in response to a variety of stresses , including glucose-starvation ( reviewed by Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>These genes for the GntI system have been extensively characterized and shown to be regulated negatively by GntR and positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex [ Tong et al. , 1996 ; Izu et al. , 1997a , b ; Porco et al. , 1997 ; Peekhaus and Conway , 1998 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>Factors required for motility and full flhDC expression include cyclic AMP ( cAMP ) and cAMP receptor protein ( CRP ; Kutsukake , 1997 ; Soutourina et al. , 1999 ) , histone-like nucleoid-structuring ( H-NS ) protein ( Bertin et al. , 1994 ) , the heat-shock response network proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) , Pss and Psd , which are involved in the synthesis of phosphatidy-lethanolamine in cell membranes ( Shi et al. , 1993 ) , and inorganic polyphosphate ( Rashid et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>In addition to the transcription activators RhaS and RhaR , the cAMP receptor protein ( CRP ) is also required for the full activation of all three of the rha operons ( Egan and Schleif , 1993 ; Holcroft and Egan , 2000 ; Via et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li>
<li>In the absence of glucose and/or other PTS sugars , the phosphorylated form of Crr may be accumulated in the cell , and , in connection , it is suggested that the phosphorylated Crr activates adenylate cyclase to raise the synthesis rate of cAMP ( Postma et al. , 1993 ) , which is needed for CRP to activate the transcription of many other transport protein genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>Furthermore , cAMP -- CRP is reported to be a repressor of its own transcription ( Aiba , 1983 ; Cossart and Gicquel-Sanzey , 1985 ; Okamoto and Freundlich , 1986 ; Hanamura and Aiba , 1991 ) , of genes for adenylate cyclase ( Majerfeld et al. , 1981 ; Mori and Aiba , 1985 ) , of genes encoding enzymes participating in glutamine metabolism ( Prusiner et al. , 1972 ) and of genes for the outer membrane protein III ( Mallick and Herrlich , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>These include the transcription factor FNR that mediates anaerobic regulation ( Unden and Schirawski , 1997 ) , the cAMP receptor protein ( CRP ) , which activates transcription of catabolic operons ( Kolb et al. , 1993 ) , and the nucleoid-associated protein H-NS ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>In enteric bacteria , it acts as an allosteric-effector of the cAMP receptor protein ( CRP ) and is essential for the regulation of the transcription of a large number of genes ( reviewed by Botsford and Harman 1992 ; Kolb et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>The cAMP receptor protein , CRP , is required for activation of ptsG transcription ( De Reuse and Danchin , 1988 ; Fox et al. , 1992 ) , whereas the negative regulatory protein Mlc represses ptsG transcription in the absence of glucose ( Kimata et al. , 1998 ; Plumbridge , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>The ptsG transcription is regulated simultaneously by two global transcription factors , CRP ( cAMP receptor protein ) -- cAMP and Mlc , in response to glucose availability ( Kimata et al. 1997 , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) - AMPc complex activates transcription of many gluconeogenic genes such as acs , actP , aceBAK and maeB [ Gosset et al. , 2004 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18679018" target="_blank">18679018</a>]<br></li>
<li>For induction , all three operons require ( a ) the L-arabinose-specific positive regulatory protein , AraC ( b ) the catabolite gene activator protein ( CAP ; also referred to as the cAMP receptor protein , CRP ) , ( c ) and cAMP ( Greenblatt and Schleif 1971 ; Englesberg and Wil-cox 1974 ; Lee et al. 1974 ; Kolodrubetz and Schleif 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997582" target="_blank">2997582</a>]<br></li>
<li>Although the precise mechanism of transcriptional regulation has yet to be elucidated , it has been established that the cAMP receptor protein ( CRP ) , integration host factor and two proteins , termed TdcR and TdcA , are necessary for positive regulation ( Wu et al. , 1992 ; Hagewood et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Transcription of the gcv operon is under the control of three global regulatory proteins [ Lrp , PurR and cAMP-receptor protein ( CRP ) ] and two gcv-speciﬁc transcriptional-regulatory proteins ( GcvA and GcvR ) ( Ghrist & Stauffer , 1995 ; Lin et al. , 1992 ; Stauffer & Stauffer , 1994 ; Wilson et al. , 1993a ; Wilson & Stauffer , 1994 ; Wonderling & Stauffer , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>Known regulators involved in the control of acs expression in E. coli are cAMP receptor protein ( CRP ) , factor for inversion stimulation ( Fis ) , integration host factor ( IHF ) ( Wolfe 2005 ) and possibly the dual transcriptional-regulator for an-oxic redox control ( ArcA ) ( Covert et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li>
<li>The E. coli cAMP receptor protein ( CRP ) is a global transcriptional-regulator that can serve as an activator , repressor , coactivator , and corepressor ( Kallipolitis et al. 1997 ; Busby and Ebright 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>Introduction cAMP receptor protein ( CRP ) is the best characterized global transcriptional-regulator in Escherichia coli ( Kolb et al. , 1993 ; Busby & Ebright , 1999 ; Harman , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) also plays a role in this RpoS-GadX pathway by tempering expression of RpoS during growth under conditions that result in high cAMP levels ( Ma et al. , 2003b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>ftnA -- lacZ expression is dependent on the growth-phase , iron and Fur The cAMP-receptor protein ( CRP ) does not control ftnA or fur expression It was previously reported that the fur gene is induced approximately fourfold during post-exponential-growth in rich medium in a manner that is largely dependent on the cAMP -- CRP ( cAMP-receptor protein ) complex ( de Lorenzo et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In enteric bacteria , expression of the pts operon increases during growth-on-glucose and requires an intact cAMP ± cAMP receptor protein ( CRP ) system , although growth-on-glucose reduces the concentration of cAMP and CRP in the cell ( Postma et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11157755" target="_blank">11157755</a>]<br></li>
<li>In this paper , we report that a sequence-specific binding protein , the cAMP receptor protein ( CRP ) , whose primary function is to control transcription as a function of catabolite conditions ( de Crombrugghe et al. 1984 ; Kolb et al. 1993 ) , can also function as an osmoregulator of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>We had shown previously that the Pa and Pz promoters of the two PA catabolic operons in E. coli were under control of global regulators such as the cyclic adenosine monophosphate ( cAMP ) receptor protein ( CRP ) , that accounts for the catabolite repression control mediated by glucose , and the integration host factor ( IHF ) ( Ferrández et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Superimposed in the specific PaaX-mediated repression , two global regulators , the cAMP receptor protein ( CRP ) and the integration host factor protein ( IHF ) , act as activators of the gene expression driven by Pa and Pz promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Cra mediates catabolite repression or catabolite activation depending on the target gene era controls gene expression in vivo Expression of the negatively Cra-regulated pykF gene and the positively Cra-regulated ppsA gene was recently studied as a function of carbon source Beta-galactosidase activity measurement of singlecopy , chromosomal fusions of these genes to lacZ in E. coli strains wild-type for or lacking the Cra protein showed that Cra controls the response to carbon source independently of the cAMP-cAMP receptor protein ( CRP ) complex ( Bledig et al. , 1996 ; T.M. Ramseier and M.H. Saier , Jr. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>Because the glucose repression effect is usually mediated by the cAMP -- CRP complex in enteric bacteria ( Kolb et al. , 1993 ; Saier et al. , 1996 ) , we checked whether CRP was required for mhpR expression by using an E. coli strain lacking CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Expression of Pa promoter is also influenced by the cAMP receptor protein ( CRP ) , which allows expression of the mhp catabolic genes when the preferred carbon source ( glucose ) is not available but 3HPP is present in the medium ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>Superimposed in the specific MhpR-mediated regulation of the Pa promoter , we have observed a strict catabolite repression control carried out by the cAMP receptor protein ( CRP ) that allows expression of the mhp catabolic genes when the preferred carbon source ( glucose ) is not available and 3HPP is present in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>Altogether , our ﬁndings indicated that mcaS transcription might be subject to catabolite repression by the cAMP receptor protein ; however , we observed only minor effects of eliminating cAMP or CRP synthesis ( cya or crp ) on mcaS expression ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>To determine whether the expression of the serC-aroA operon was subject to catabolite repression , lacZ expression was examined in cya and crp mutants deficient in adenylate cyclase and cAMP receptor protein ( CRP ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>To determine whether synthesis of the colicin I receptor was subject to catabolite repression , receptor levels in a wild-type strain ( JK213 ; trpR55 trpA9605 his-85 metE ) , a cya derivative defective in cAMP production ( JK214 ) , and a crp derivative unable to produce functional cAMP receptor protein ( CRP ) ( JK215 ) were measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>Novel Roles of cAMP Receptor Protein ( CRP ) in Regulation of Transport and Metabolism of Carbon Sources Tomohiro Shimada , Nobuyuki Fujita , Kaneyoshi Yamamoto , Akira Ishihama * 1,2 3 1,2 1,2 1 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo , Japan , 2 Micro-Nanotechnology Research Center , Hosei University , Koganei , Tokyo , Japan , 3 Department of Biotechnology , National Institute of Technology and Evaluation , Shibuya-ku , Tokyo , Japan Abstract CRP ( cAMP receptor protein ) , the global regulator of genes for carbon source utilization in the absence of glucose , is the best-studied prokaryotic transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R Involvement of cAMP-CRP in transcription activation and repression of the pck gene encoding PEP carboxykinase , the key enzyme of gluconeogenesis Masahiro Nakano1 , Hiroshi Ogasawara1 ,2 , Tomohiro Shimada1 ,3 , Kaneyoshi Yamamoto1 & Akira Ishihama1 1 2 Department of Frontier Bioscience and Research Center for Micro-Nano Technology , Hosei University , Koganei , Tokyo , Japan ; Research Center for Human and Environmental Sciences , Shinshu University , Ueda , Nagano , Japan ; and Chemical Resources Laboratory , Tokyo Institute of 3 Technology , Nagatsuda , Yokohama , Japan Correspondence : Akira Ishihama , MicroNano Technology Research Center , Hosei Abstract cAMP receptor protein ( CRP ) is the best characterized global regulator of Esc-herichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Although the enzymatic activity of AcnB is lower than that of aconitase AcnA and the sequence homology covers only 17 % [ 49 ] , the protagonistic role of AcnA in at least four global regulatory pathways , cAMP receptor protein ( CRP ) - mediated catabolite activation , ArcAmediated anaerobic repression , iron and Fur ( ferric uptake regulator ) - dependent activation and oxidative-stress regulatory system ( SoxRS ) activation , points to the possible effects of binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21972224" target="_blank">21972224</a>]<br></li>
<li>Here , we show that in contrast to other characterised Gamma-proteobacteria , E. coli Sxy is positively autoregulated at the level of transcription by a mechanism that requires cAMP receptor protein ( CRP ) , cyclic AMP ( cAMP ) and a CRP-S site in the sxy promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>volume 12 Number 21 1984 Interaction of cycnc AMP receptor protein with the ilvB biosynthetic operon in E. coli Phillip Friden , Ping Tsui , Keinosuke Okamoto and Martin Freundlich Department of Biochemistry , State University of New York , Stony Brook , NY 11794 , USA Received 29 May 1984 ; Revised and Accepted 15 October 1984 ABSTRACT DNase and restriction site protection studies show that cAMP and its receptor protein ( CRP ) bind to the promoter of the ilvB operon at approximately position -44 to -82 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Similarly , the hlyE promoter includes a 9 of 10 fit to the core FNR-binding site , but this is also a binding site for CRP , the cAMP receptor protein , and H-HNS ( heat-stable nucleoid-structuring protein ) interferes with FNR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The proP regulatory region also contains a highaf ® nity binding site for the cyclic AMP ( cAMP ) receptor protein ( CRP ) located at ÿ121 .5 relative to the P2 promoter start site ( Figure 1 ) .28 In the absence of Fis , CRP-cAMP provides little activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>This repression is presumably mediated by 3 ' - 5 ' cyclic AMP ( CAMP ) and the cAMP receptor protein ( CRP ) , which form a CAMP-CRP complex that activates transcription by interacting with the CRP-binding sites in the promoter regions of the corresponding genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3309132" target="_blank">3309132</a>]<br></li>
<li>In addition , we have recently found that cyclic AMP ( cAMP ) receptor protein ( CRP ) - cAMP complex has a strong inhibitory effect on proP1 , perhaps by binding to the almost perfect consensus CRP binding site located within its 235 region ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545153" target="_blank">7545153</a>]<br></li>
<li>In addition to Fis activation , there is an upstream cyclic AMP ( cAMP ) receptor protein ( CRP ) binding site located at 2121.5 relative to the start of P2 transcription ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a CRP binding site positioned at 271 upstream of the gntT transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>1 Abbreviations : cAMP , cyclic adenosine 3 ′ :5 ′ - cyclic monophosphate ; CRP , cAMP receptor protein ; wt , wild type I promoters possess a CRP binding site centered -61.5 base pairs from the start site of transcription or at integral helical turns further upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>This burst of proP transcription is correlated with the osmolarity-dependent binding of the cAMP receptor protein CRP to a site within the proP P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>We showed the presence of a unique transcription promoter , gapB P0 , with a cyclic AMP ( cAMP ) receptor protein ( CRP ) - cAMP binding site centered 70.5 bp upstream of the start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>We demonstrate that the binding of the cyclic AMP ( cAMP ) receptor protein ( CRP ) to a site centered at 234.5 within the promoter is responsible for the low promoter activity under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>The mechanism of transcriptional regulation involves the cAMP receptor protein ( CRP ) , which first binds CAMP , alters its conformation in response to cAMP binding , and then binds to the promoter region of a target operon to promote transcriptional initiation ( Schultz e t al. , 1991 ; Kolb e t al. , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The binding of CRP to the wild-type ara regulatory region required cAMP [ Fig. 2A , compare lane 1 ( no protein ) , lane 2 ( CRP but no cAMP ) , and lane 3 ( CRP and cAMP ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>The observation that the levels of gadA and gadBC transcripts are greatly enhanced in an hns-defective strain when compared with that found in wild-type E. coli cells led several 1 The abbreviations used are : CRP , cAMP receptor protein ; MalE , maltose-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The cents rst member of the family was the cyclic aden-osine monophosphate ( cAMP ) - binding protein , CRP ( cAMP receptor protein ) , also called CAP for catabolite activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>The transcriptional control of a number of genes is mediated by the binding of cAMP to the cAMP receptor protein ( CRP ) ( for reviews , see references 6 and 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>[ 48 ] G. Tan , P. Kelly , J. Kim , R.M. Wartell , Comparison of cAMP receptor protein ( CRP ) and a cAMP-independent form of CRP by Raman spectroscopy and DNA binding , Biochemistry 30 ( 1991 ) 5076 -- 5080 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Abbreviations : CRP , cAMP receptor protein ; FNR , fumarate and nitrate reduction regulator ; MALDI-TOF , matrix-associated laser desorption ionization-time of ﬂight ; MBP , maltose-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>CRP , cAMP receptor protein binding site ; I to VI , Fis binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226664" target="_blank">8226664</a>]<br></li>
<li>Keywords : Transcription activation ; DNA footprinting ; Operon ; Filter binding ; CD ; FTIR The Escherichia coli cyclic AMP ( cAMP ) 1 receptor protein ( CRP ) is a 47,000 Da DNA-binding protein responsible for regulating the expression of over 100 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Moreover , it is subject to global control by the same regulatory proteins which are known to modulate expression of carnitine metabolism , i.e. , the cyclic AMP ( cAMP ) receptor protein ( CRP ) which mediates the activation of catabolic operons , the transcriptional-regulator FNR that is responsible for anaerobic induction , and the DNA-binding protein H-NS ( 19 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>One well-characterized regulator of the genes for carbon source utilization is CRP ( cyclic AMP [ cAMP ] receptor protein ) ( sometimes called catabolite activator protein [ CAP ] ) , which is converted into the functional form after binding cAMP that is synthesized in the absence of glucose ( 10 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR-binding element for gntK has been proposed ( 12 , 34 ) , and two GntR-binding elements for gntT were experimentally deﬁned ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618441" target="_blank">12618441</a>]<br></li>
<li>Cyclic AMP ( cAMP ) also negatively regulates speA gene expression either directly or indirectly via binding to cAMP receptor protein ( CRP ) ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2198270" target="_blank">2198270</a>]<br></li>
<li>Introduction In Escherichiacoli and related bacteria , cAMP receptor protein ( CRP ) forms a complex with cyclic AMP ( cAMP ) , and regulates gene expression by binding to CRP-binding sites on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>To clarify the role of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex in the control of the dsdXA promoter , we measured the D-serine-induced transcriptional activity of plasmid pNM23 when present in three different genetic backgrounds : wild-type , Ddsd , and Dcrp strains ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>Wanner and coworkers ( 24 , 28 ) had previously reported that expression of the psiE9-9lacZ gene in a cya or crp mutant was greater than that in the wild-type strain , indicating that psiE expression is negatively regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>In this report , we show that both AI-2 synthesis and uptake in Escherichia coli are subject to catabolite repression through the cyclic AMP ( cAMP ) - CRP complex , which directly stimulates transcription of the lsr ( for `` luxS regulated '' ) operon and indirectly represses luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In this study , we show that catabolite repression inﬂu-ences AI-2 accumulation through the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which directly stimulates transcription of the lsr operon and indirectly represses luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In this study , we found that transcription from the prpBCDE and prpR promoters was down-regulated by the addition of glucose or glycerol , indicating that these genes may be regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In these cases , the H-NS protein represses transcription speciﬁcally at low temperature and seems to act by antagonizing the transcriptional-activators CfaD , cyclic AMP ( cAMP ) / CRP , and possibly VirF ( 16 , 22 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Key words : CRP targets/DNA sequence/footprinting/gene fusions / gene regulation Introduction In bacteria such as Escherichia coli and Salmonella typhi-murium , the complex of adenosine 3 ' ,5 ' - monophosphate ( cAMP ) and the cAMP receptor protein ( CRP ) activates the expression of certain genes concerned with transport and utilization of energy sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Molecular mechanism of negative autoregulation of ABSTRACT Transcription of the Eschenchia coli crp gene encoding cAMP receptor protein ( CRP ) is negatively regulated by CRP-cAMP complex that binds to a specific site located downstream from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>In the case of the Escherichia coli lactose operon , glucose lowers transcription by increasing the level of free lac repressor and/or by decreasing the level of activator , CRP ( cAMP receptor protein ) - cAMP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>de Quimica Fisica ` Rocasolano ' , Serrano 119 , Madrid 6 , Spain Communicated by H.Buc In Escherichia coli , the transcription of the malT gene is activated by the complex formed between cAMP and its receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Three promoter/operator regions have been demonstrated : at P01 and P02 , transcription is repressed by the deoR repressor and induced by deoxyribose-5-P ; at P02 , is also the transcription repressed by cytR repressor , induced by cytidine or adenosine , and depends on cAMP and CRP ; at P03 , transciption is controlled by unknown protein factor ( s ) and induced by inosine or guanosine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453417" target="_blank">16453417</a>]<br></li>
<li>We are currently attempting to isolate the DsdC protein in order to deﬁne how this activator in concert with the cAMP-CRP complex synergistically activate transcription from the dsdXA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>At low cAMP levels , transcription initiates from the galP2 promoter but as cAMP levels increase , the cAMP receptor protein ( CRP ) - cAMP complex binds to its recognition sequence preventing transcription from galP2 and activating transcription from the galP1 promoter , which is located 5 bp downstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li>
<li>Transcription activation at CRP ( cAMP-receptor protein ) - dependent promoters provides a paradigm for elucidating the mechanism by which an activator makes multiple interactions with RNP to enhance transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>For transcription activation by cAMP receptor protein -LCB- CRP ) , for instance , several lines of evidence indicate that Received 28 July , 1992 ; revised and accepted 17 October , 1992 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Three trans-acting proteins , GalR ( the gal repressor ) , cAMP receptor protein ( CRP ) - cAMP complex , and the histone-like protein HU , control transcription from these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>To understand the role of the CRP-cAMP complex on P1 transcription dynamics , we measured changes in the amount of CRP protein and cAMP during different growth-phases in MG1655 ( Figs. 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>DemonSlration o / ¡ he method 011 sites cAMP protein , CRP , is involved in the transcription regulation of several E.coll genes , either as an activa tor or as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>Transcription of these genes is activated in response to intracellular cAMP levels by the cAMP receptor protein ( CRP ) , a transcription factor that activates expression of over 100 E. coli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>A second regulator , cAMP/CRP , activates P2 , an effect that is largely inhibited by Lrp , such that catabolite repressor protein ( Crp ) and Lrp are rival activators of serA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>We show that this regulation occurs through reduction in transcription and depends on both cAMP and the cAMP receptor protein ( CRP ) but is independent of de novo protein synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>Clearly , acs transcription responds to catabolite repression through cAMP and its receptor protein , CRP ( 249 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Activation of acs transcription depends upon the cAMP receptor protein ( CRP , also known as CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>A functional cAMP receptor protein ( CRP ) together with cAMP is required to act as a transcription activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>In these operons , cAMP and its receptor protein On ( CRP ) act aspositive effector for transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>The 50S L20 operonb rplT < rpmI infC bamD > raiA Translation inhibitor set of genes that are involved in carbon utilization for energy production is controlled by two global regulators , CRP ( cAMP receptor protein or catabolite activation protein ) and Cra ( catabolite repressor and activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The activation of two of the promo-ters , malEp and malKp , aiso requires the cAMP receptor protein ( CRP ) ( Vidal-lngigiiardi and Raibaud .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7830577" target="_blank">7830577</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) binds to a site from bp 299 to 326 relative to the transcription initiation site ( Fig. 1 ( a ) ) and is required for a 4-fold activation of the gcvTHP operon in glucose minimal ( GM ) medium [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>The increased expression of the hya-lacZ fusion when glycerol rather than glucose is used as the carbon source could therefore be due to activation of transcription by the cAMP-cAMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>The consequences of adding a transcriptional-activator , the cAMP receptor proteinycAMP complex ( CRP ) , are monitored in real time , permitting us to assign specific interactions to the activation of discrete steps in transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>The physiological function of the GutM protein is shown to be activation of gut operon transcription , while that it involves participation of both a glucitolGxpressional regulation of the operon is unusual in that it invokes participation of both a glucitolspecific repressor and a glucitol-specific activator in addition to the general activator , the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3062173" target="_blank">3062173</a>]<br></li>
<li>15302 Biochemistry 1996 , 35 , 15302-15312 Influence of the Location of the cAMP Receptor Protein Binding Site on the Geometry of a Transcriptional Activation Complex in Escherichia coli † Patrick Eichenberger , ‡ Sylvie Déthiollaz , ‡ Nobuyuki Fujita , § Akira Ishihama , § and Johannes Geiselmann * , ‡ Département de Biologie Moléculaire , UniVersité de GenèVe , 30 Quai Ernest Ansermet , CH-1211 GenèVe 4 , Switzerland , and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka-Ken 411 , Japan ReceiVed June 10 , 1996 ; ReVised Manuscript ReceiVed August 27 , 1996X ABSTRACT : The interactions between the cAMP receptor protein ( CRP ) and RNA polymerase during transcriptional activation at the Escherichia coli malT promoter have been analyzed using a combination of footprinting methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Effect of GalR bound to OE on transcription from gal P1 and P2 promoters using reconstituted RNA polymerases carrying different amino acid substitutions at aCTD Enhancement of P2 The ® nding of GalR enhancement of P2 parallels the activation of transcription of the E. coli lac promoter by cAMP receptor protein ( CRP or CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>J Mol Biol 232 : 406-418 , Lavlgna , M. , Kolb , A. , and Buc , H. ( 1992 ) Transcription activation by cAMP receptor protein ( CRP ) at the Escherichia coli gal PI promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_407">
Transcription activation by cAMP receptor protein ( CRP ) at the Escherichia coli gal P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></label>
<input id="act_similarity_407", type="checkbox">
<div class="ACT_similar">
<li>Transcription activation by cAMP receptor protein ( CRP ) at the Escherichia coli galP1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li></div>
<li>Upregulation in E. coli is mediated via activation of the flhDC operon by cAMP -- CRP ( cAMP receptor protein ) ( Yokota & Gots , 1970 ; Silverman & Simon , 1974 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>Two noted exceptions are X repressor -LCB- XC \ ) activation of PRM ( Meyer and Ptashne , 1980 ) and cAMP receptor protein ( CRP ) repression of cya ( Aiba , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Volume 10 Number 4 1982 Molecular cloning and nucleotide sequencing of the gene for E. coli cAMP receptor protein Hiroji Aiba , Shinji Fujimoto and Norihito Ozaki Radioisotope Laboratory , Faculty of Medicine , and * Department of Biophysics , Faculty of Science , Kyoto University , Kyoto 606 , Japan Received 23 December 1981 ; Accepted 27 January 1982 ABSTRACT The crp gene of E. coli , which codes for cAMP receptor protein ( CRP ) , has been cloned in the plasmid pBR322 on the basis of a genetic complementation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The cAMP receptor protein CRP can function as an osmoregulator of transcription in Escherichia coli Lenore Landis ,1,4 Jimin Xu ,3,4 and Reid C. Johnson1 ,2,5 1Department of Biological Chemistry , School of Medicine , University of California , Los Angeles , Los Angeles , California 90095-1737 USA ; 2Molecular Biology Institute , University of California , Los Angeles , Los Angeles , California 90095 USA Transcription of the P1 promoter of the Escherichia coli proP gene , which encodes a transporter of osmoprotectants , is strongly induced by a shift to hyperosmotic media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The Escherichia coli cyclic AMP ( cAMP ) receptor protein ( CRP ; also known as the catabolite activator protein ) activates transcription from more than 100 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>The cyclic AMP receptor protein ( CRP ; also referred to as catabolite gene activator protein , CAP ) from Escherichia coli , a classic transcription regulator in prokaryotes , is also allosterically modulated by the effector-molecule cAMP [ reviewed in 5 -- 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Introduction The lac and ara operons of Escherichia coli , like other catabolite-scnsitive ( CS ) operons , normally require for their transcription , adenosine-3 ' ,5 ' - cyclic mono-phosphate ( cAMP ) , and at least one protein factor , variously termed CAP , CRP , and CGA ( Perlman and Pastan , 1968 ; Ullman and Monod , 1968 ; Zubay , Schwartz and Beckwith , 1970 ; Emmer , de Crombrugghe , Pastan and Perlman , 1970 ; l % iggs , Reiness and Zubay , 1971 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4343249" target="_blank">4343249</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) complexed to cAMP binds at specific sites in promoters to regulate initiation of transcription of many operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Department of Microbiology and Immunology and Department of Zoology , University of British Columbia , Vancouver , BC , Canada INTRODUCTION The Escherichia coli cAMP receptor protein ( CRP ) , also called the catabolite activator protein ( CAP ) , was the ﬁrst transcription factor to be puriﬁed and the ﬁrst to have its structure solved ( 1,2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_418">
The cAMP receptor protein CRP can function as an osmoregulator of transcription in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>]<br></label>
<input id="act_similarity_418", type="checkbox">
<div class="ACT_similar">
<li>Landis , L. , Xu , J. , and Johnson , R.C. ( 1999 ) The cAMP receptor protein CRP can function as an osmoregulator of transcription in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16420371" target="_blank">16420371</a>]<br></li>
<li>[ 40 ] L. Landis , J. Xu , R.C. Johnson , The cAMP receptor protein CRP can function as an osmoregulator of transcription in Escherichia coli , Genes Dev .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li></div>
<li>The P5 promoter is totally dependent on the cAMP receptor protein ( CRP ) , whereas the P3 and P4 promoters have been reported to be negatively controlled by the DnaA protein ( Wang and Kaguni , 1989b ; Nagai et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The glnAp1 is a s - dependent 70 promoter that is activated by the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_423">
The Escherichia coli cAMP receptor protein ( CRP ) represses the Rhizobium meliloti dctA promoter in a cAMP-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></label>
<input id="act_similarity_423", type="checkbox">
<div class="ACT_similar">
<li>Wang , Y.P. , Giblin , L. , Boesten , B. , and O'Gara , F. ( 1993 ) The Escherichia coli cAMP receptor protein ( CRP ) represses the Rhizobium meliloti dctA promoter in a cAMP-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li></div>
<li>The acsyjcHactP and aceBAK operons are activated by the cAMP receptor protein ( CRP ) -- cAMP complex ; additionally , transcription of the former operon occurs from the acsP2 promoter , which is dependent on CRP ( Wolfe , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375081" target="_blank">24375081</a>]<br></li>
<li>The P1 promoter is constitutively active , but , on entry into stationary phase , relA transcription is driven by P2 and it is thought that transcription from P2 is inﬂuenced by the global transcriptional-regulator cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Abbreviations used : cAMP , 30,50-cyclic adenosine monophosphate ; CRP , cAMP receptor protein ; Lrp , leucine-responsive regulatory protein ; E , RNA polymerase core enzyme ; SDS-PAGE , sodium dodecyl sulfate-polyacrylamide gel electrophoresis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>1 Abbreviations : RNA , ribonucleic acid ; RNAP , RNA polymerase ; LacI , Lac repressor ; GalR , Gal repressor ; CAP , catabolite activator protein ( also known as CRP , cAMP receptor protein ) ; O , `` external '' E operator ; OI , `` internal '' operator ; bp , base pair ; BSA , bovine serum albumin ; DTT , dithiothreitol ; EDTA , ethylenediaminetetraacetic acid ; Bis-tris , bis ( 2-hydroxyethyl ) iminotris ( hydroxymethyl ) methane ; DNase I , deoxyribonuclease I ( EC 3.1.21.1 ) ; DNA , deoxyribonucleic acid ; cAMP , adenosine 3 ′ 5 ′ - cyclic monophosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Multiple Specific CytR Binding Sites at the Escherichia coli deoP2 Promoter Mediate Both Cooperative and Competitive Interactions between CytR and cAMP Receptor Protein * Binding of cAMP receptor protein ( CRP ) and CytR mediates both positive and negative control of transcription from Escherichia coli deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In particular , the transcription of genes needed for utilization of nucleosides in Escherichia coli is regulated by a repressor protein , CytR , in concert with the cyclic AMP ( cAMP ) - activated form of cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Much of the expression of nucleoside utilization genes is coordinated by a repressor protein , CytR , acting in concert with the cyclic AMP ( cAMP ) - activated form of the cAMP receptor protein ( CRP ) ( 2 , 9 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>( 2008 ) 383 , 1 -- 9 Available online at www.sciencedirect.com C OMMUNICA TION Do n-regulation uter mbrane Proteins Non oding R Unraveling the cAMP -- C σ E - D e -- pX Regulatory o As : w f O Me by N c RP - and ent C yaR om ep nd Case Jesper Johansen , Maiken Eriksen , Birgitte Kallipolitis and Poul Valentin-Hansen Abbreviations used : σE , extracytoplasmic stress response sigma factor in Escherichia coli ; sRNA , small noncoding RNA ; OMP , outer membrane protein ; ppGpp , guanosine 3 ′ ,5 ′ - bispyrophosphate ; CRP , cAMP receptor protein ; σ70 , `` housekeeping '' sigma factor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>Sequence and Characterization of the sfsl G Is Involved in CRP * - Dependent mal Gene Expression in Escherichia coli Cyclic AMP receptor protein ( CRP ) binds to cAMP and regulates the expression of many genes negatively or positively in Escherichia coli ( 1 , 9 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>Alone or in synergy with CRP ( cAMP receptor protein ) , MalT activates all the operons comprising the Escherichia coli maltose regulon , which encodes proteins involved in the uptake and degradation of maltodextrins ( Boos and Shuman , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692154" target="_blank">10692154</a>]<br></li>
<li>INTRODUCTION Transcription of the lac operon as well as other ` cata-bolite sensitive ' operons in E.coli is stimulated by a cAMP ( adenosine-3 ' :5 ' - cyclic monophosphate ) receptor protein ( CRP ) when complexed with cAMP ( 13 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>126 , No. 3 Printed in U.S.A. Mutations Affecting Catabolite Repression of the L-Arabinose Regulon in Escherichia coli B/r LAUREL HEFFERNAN , ' * RICHARD BASS , AND ELLIS ENGLESBERG Section in Biochemistry and Molecular Biology , Department ofBiological Sciences , University ofCalifornia , Santa Barbara , California 93106 Received for publication 2 January 1976 Expression of the L-arabinose regulon in Escherichia coli B/r requires , among other things , cyclic adenosine-3 ' ,5 ' - monophosphate ( cAMP ) and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>For the cAMP receptor protein ( CRP ) - regulated sRNA CyaR ( 18 , 19 ) , we detected as yet unidentiﬁed targets in primary metabolism ( sdhA ) and the phosphotransferase system ( ptsI ) , constituting previously unreported links of the CyaR regulon to carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Novel roles of cAMP receptor protein ( CRP ) in regulation of transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752906" target="_blank">22752906</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25790494" target="_blank">25790494</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Cold Spring Harbor Laboratory , NY , pp 1.119 -- 111.122 Shimada T , Fujita N , Yamamoto K , Ishihama A ( 2011 ) Novel roles of cAMP receptor protein ( CRP ) in regulation of transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_440">
Shimada T , Fujita N , Yamamoto K & Ishihama A ( 2011b ) Novel roles of cAMP receptor protein ( CRP ) in regulation of transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></label>
<input id="act_similarity_440", type="checkbox">
<div class="ACT_similar">
<li>Shimada , T. , Fujita , N. , Yamamoto , K. & Ishihama , A. ( 2011b ) Novel roles of cAMP receptor protein ( CRP ) in regulation of transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23301696" target="_blank">23301696</a>]<br></li></div>
<li>Abbreviations used : ADPG , ADP -- glucose ; CRP , cAMP receptor protein ; KmR , kanamycin resistance ; ppGpp , guanosine 5 ′ - diphosphate 3 ′ - diphosphate ; RACE , rapid ampliﬁcation of cDNA ends ; RT , reverse transcriptase ; TAP , tobacco acid pyrophosphatase ; UTR , untranslated region ; WT , wild-type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>Expression of mhp catabolic genes depends on the transcriptional-activator MhpR belonging to the IclR family of transcriptional-regulators ( Fig. 1A ) comprising more 2 The abbreviations used are : 3HPP , 3-hydroxyphenylpropionate ; CRP , cAMP receptor protein ; DHPP , 3 - ( 2,3-dihydroxyphenyl ) propionic acid ; PA , phe-nylacetic acid ; 3HPA , 3-hydroxyphenylacetic acid ; PP , phenylpropionic acid ; CI , cinnamic acid ; 3HCI , 3-hydroxycinnamic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) 1 is a regulatory protein , which binds cAMP and mediates transcriptional regulation at several promoters ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) is responsible for catabolic regulation in response to cAMP , and the AraC protein responds to L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>( A ) The organization of the operon and its regulation by the positive regulator cAMP receptor protein ( CRP ) and the negative regulator GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>The increased levels of CRP protein and cAMP during the INCREASE period ( Figs. 6 and 7 ) support both possibilities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Because other IclR-type regulators 1 The abbreviations used are : 3HPP , 3 - ( 3-hydroxyphenyl ) propionic acid ; CRP , cAMP receptor protein ; OR , operator region ; LB , Luria Bertani ; Km , kanamycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>1 The abbreviations used are : PTS , phosphoenolpyruvate : sugar phosphotransferase system ; ArcA-P , phospho-ArcA ; CRP , cAMP receptor protein ; DTT , dithiothreitol ; MALDI-TOF , matrix-assisted laser de-sorption ionization time-of-flight ; ORF , open reading frame .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>The abbreviations used are : CRP , cAMP receptor protein ; Ac , acetate ; bp , base pair ; DTT , dithiothreitol ; IPTG , isopropyl-0-D-thiogalactopyranoside ; PEG , polyethylene glycol ; SDS , sodium dodecyl sulfate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>1 The abbreviations used are : CRP , cAMP receptor protein ; kb , kilobase pair ( s ) ; X-gal , 5-bromo-4-chloro-3-indolyl-b-D-galactopyranoside ; IPTG , isopropyl-b-D-thio-galactopyranoside ; FPLC , fast protein liquid chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>1 The abbreviations used are : IS , insertion elements ; CRP , cAMP receptor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>1 The abbreviations used are : PTS , phosphoenolpyruvate-carbohy-drate phosphotransferase system ; CRP , cAMP receptor protein ; DTT , dithiothreitol ; HPr , histidine protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464268" target="_blank">10464268</a>]<br></li>
<li>1 The abbreviations used are : PGA , penicillin G acylase ; 6-APA , 6-aminopenicillanic acid ; PAA , phenylacetic acid ; PAA-CoA , phenylac-etic acid-coenzyme A ; CRP , cAMP-receptor protein ; IHF , integration host factor ; RT , reverse transcriptase ; EMSA , electrophoretic mobility shift assay ; ChIP , chromatin immunoprecipitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>1 The abbreviations used are : PA , phenylacetic acid ; bp , base pair ( s ) ; CRP , cAMP receptor protein ; CoA , coenzyme A ; IHF , integration host factor ; LH , left-half ; OR , operator region ; PA-CoA , phenylacetyl-coen-zyme A ; PCR , polymerase chain reaction ; RH , right-half .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>We showed that these surface ﬁmbriae are all under the negative control of the H-NS repressor therefore explaining their cryptic character and are subjected to cAMP receptor protein ( CRP ) - mediated catabolite repression ( except for yad ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Glucose causes catabolite repression by lowering the intracellular levels of both cAMP and cAMP receptor protein ( CRP ) in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The production of fumarase is subject to catabolite and anaerobic repression ( 27 , 28 ) and is likely to be controlled by the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6328431" target="_blank">6328431</a>]<br></li>
<li>The role of the cAMP receptor protein ( CRP ) in the repression of ODC and A D C by cAMP is shown in Table 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>As an enzyme that is subject to anaerobic and catabolite repression , MDH might be controlled by the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>Furthermore the repression exerted by glucose is reversed by addition of cAMP , suggesting that the catabolite repression is mediated by cAMP and its receptor protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>Catabolite repression in bacteria is a phenomenon by which glucose and its analogs decrease the intracellular level of both cAMP and its receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>When cAMP-CRP and AraC protein plus arabinose were both added , protection extended from - 3 7 tp - 1 0 0 and the protection in the +44 region could be seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>PItYSIOLOGY VOL 44 ISBN 0-12-027744 - I Copyright © 2001 Academic Press All rights of reproduction in any tiwm reserve ABBREVIATIONS cAMP CAP CRP FLP FNR Adenosine 3 ' ,5 ' - monophosphate Catabo -RCB- ite gene activator protein cAMP receptor protein FNR-like protein fumarate and nitrate reduction regulator 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>As expected , there was no change in SecB when cAMP was added in the cya crp double-mutant RH77 , which lacks the ability to synthesize endogenous cAMP and its functional mediator , cAMP receptor protein ( CRP ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023186" target="_blank">9023186</a>]<br></li>
<li>The induction proﬁle raised the possibility that the yjiY gene might also be regulated by cAMP and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>Like other cytR-regulated transcriptionalunits , the synthesis is also activated by the cAMP receptor protein ( CRP ) control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>If the m u t a t i o n changed the CRP protein enabling it to act without cAMP , alt-1 should map in the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4343249" target="_blank">4343249</a>]<br></li>
<li>At present , it is not known why only these r-protein operons are under the control of cAMP-CRP , but we predict the following .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Under the culture conditions thus established , we next examined the possible inﬂuence of cAMP and CRP on the production of RMF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Lane 1 , labeled ; lane 2 , cAMP-CRP protein and ycfRp ycfRp labeled ycfRp ; lanes 3 to 7 : cAMP-CRP protein , labeled ycfRp , and nonspeciﬁc competitor unlabeled ycfRp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>Lanes 1 -- 4 and lane - cAMP contained 0 , 2 , 4 , 6 , and 6 pmol CRP protein , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Alone or together with CRP ( cAMP receptor protein ) MalT activates the promoters of all of the operons comprising the maltose regulon ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10559195" target="_blank">10559195</a>]<br></li>
<li>p1 is dependent on cAMP receptor protein ( CRP ) and VAMP for full activity while p2 is ( ` KP independent ( Musso ef al. , f 977 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3316663" target="_blank">3316663</a>]<br></li>
<li>Expression in rich media is largely dependent on the alternate sigma factor σS and is repressed by the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>More recently , it has been proposed that CRP binds 4 mol of cAMP per mole of protein [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>A third permease involved in the ac - protein ( cAMP receptor protein [ CRP ] ) that cumulation of L-arabinose is not part of the ara binds cAMP has been isolated and characterregulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>This is due to the catabolite eVect mediated in part by cAMP and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>This step was enhanced by the cAMP receptor protein ( CRP ) , which is an activator of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Abbreviations : bp , nucleotides or nucleotide pairs ; CB , cyanogen bromide cleavage products ; CNBr , cyanogen bromide ; CRP , cAMP receptor protein ; DTT , dithiothreitol ; kb , kilobase pairs ; NTP , nucleotide triphosphate ; panc , pancreatic RNase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7019009" target="_blank">7019009</a>]<br></li>
<li>Keywords Putrescine Transport Feedback inhibition Energy source cAMP Abbreviations CadB Cadaverine-lysine antiporter CCCP Carbonyl-cyanide-m-chlorophenylhydrazone CRP cAMP receptor protein PotE Putrescine-ornithine antiporter Y. Terui and S. D. Saroj contributed equally to this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>More recently it was realized that gluconate lowers both cAMP receptor protein ( CRP ) and cAMP to nearly the same extent as glucose ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>It is generally believed that in most of these processes the function of cAMP is mediated by its receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6344011" target="_blank">6344011</a>]<br></li>
<li>Both enzymes are positively regulated by the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>One of the carbon starvation-inducibie regulons involves cAMP and the cAMP-receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022277" target="_blank">8022277</a>]<br></li>
<li>Abbreviations used : CRP , cAMP receptor protein ; FIS , factor for inversion stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Abbreviations : CRP , cAMP receptor protein ; MM , minim external pH. ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>Abbreviations used : CRP , cAMP receptor protein ; DMS , dimethyl sulphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In addition , we found that three r-protein operons , rpmI-rplT-pheM , rplM-rpsI , and rpsF-priB-rpsR-rplI , are under the control of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In contrast , expression of the malKpromoter of the maltose regulon requires two activator proteins , MalT and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In addition to regulation by its own protein product , araC gene expression is under positive control of the cAMP receptor protein ( CRP ) ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>In Haemophilus inﬂuenzae , competence gene expression also requires the cAMP receptor protein ( CRP ) , a global regulator involved in the sugar starvation response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>The fact that gntT expression is very low in the absence of adenylate cyclase ( cya ) or catabolite receptor protein ( crp ) , even in the presence of gluconate , shows that the positive regulation by cAMP-CRP is essential for full induction of gntT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>The complex of cAMP and CRP ( cAMP receptor protein ) inhibited the expression of PuuR in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>The crp * l gene encodes an altered cAMP receptor protein ( CRP * ) which is functional in the expression of some sugar fermentation genes , such as lac , ara , and man , in the absence of cAMP ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670911" target="_blank">2670911</a>]<br></li>
<li>This is particularly severe for nagE whose expression is very low in the absence of a functional cAMP/CAP complex ( CAP = catabolite activating protein , also known as CRP = cAMP receptor protein ) ( Ptumbridge , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>In the absence of the cAMP receptor protein ( CRP ) complex ( cAMP-CRP ) , FadR and ArcA-ArcB repression of fadL and fadD expression was relatively weak ( Fig. 8 ) and similar to that seen with fadBA and fadH [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Furthermore , we showed that a complex of cAMP receptor protein ( CRP ) and cAMP binds to a speciﬁc P3hfq promoter region of hfq and regulates hfq expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>We recently demonstrated that , in vivo , FIS regulates the expression of several genes needed for catabolism of sugars and nucleic acids , a majority of which are also transcriptionally regulated by cAMP -- cAMP-receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth-phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Thus far , the cAMP receptor protein ( CRP ) complex and Mlc are known to be the major regulators of ptsHIcrr and ptsG expression in response to the availability of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>Transcription of the gal regulon promoters is regulated in different ways by Gal repressor ( GalR ) and Gal isorepressor ( GalS ) and a global regulator , adenosine cyclic-3 ′ ,5 ′ - monophosphate ( cAMP ) receptor protein ( CRP ) , responding to two signals , D-galactose and cAMP respectively ( Weickert and Adhya , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>1 Abbreviations used : cAMP , cyclic AMP ; CRP , cAMP receptor protein ; RNAP , RNA polymerase ; lacP , lactose operon promoter ; AR1 , activation region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>All three operons in the L-rhamnose regulon also require a second activator protein , cyclic AMP ( cAMP ) receptor protein ( CRP ) , for full transcription activation ( 11 , 16 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>We show here that gntP is induced by fructuronate , regulated by UxuR , and requires activation by cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>RhaR positively regulates rhaSR in response to L-rhamnose , and RhaR activation can be enhanced by the cyclic AMP ( cAMP ) receptor protein ( CRP ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>An Escherichia coli global regulatory protein , cyclic AMP ( cAMP ) receptor protein ( CRP ) , is involved in the regulation of transcription , either positively or negatively , in many genes involved in carbon metabolism ( 4 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative-stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The cyclic AMP recep - s tor protein and cyclic AMP complex ( CRP cAMP ) in - z creased expression from P2 but showed negative effect on transcription from P1 by Es70 , although it had little effect on transcription from P2 by Es32 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11340070" target="_blank">11340070</a>]<br></li>
<li>The observed changes in protein levels could reﬂect regulation by cyclic AMP ( cAMP ) receptor protein ( CRP ) , a transcription factor that modulates the expression of hundreds of genes , depending on glucose availability ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>The cyclic AMP receptor protein ( CRP ) complexed with 3 0250 cyclic AMP ( cAMP ) binds to its cognate recognition site around the 241 position and activates transcription from P1 , while P2 is repressed [ 11,12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298753" target="_blank">11298753</a>]<br></li>
<li>The in vitro coupled transcription-translation of two of the biosynthetic genes , glgC ( ADPglucose pyrophosphorylase ) and gigA ( glycogen synthase ) , was enhanced up to 26 - and 10-fold , respectively , by cyclic AMP ( cAMP ) and cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The cyclic AMP receptor protein ( CRP ) is a cyclic AMP ( cAMP ) - dependent global transcription regulator ( for reviews , see references 11 and 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The more extensively studied system is dependent on cyclic AMP ( cAMP ) and its receptor protein , CRP , which activates the transcription of many catabolic operons ( Busby and Kolb 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>The second major regulator of ptsG transcription is the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855233" target="_blank">16855233</a>]<br></li>
<li>The regulation of these genes is complex and involves the transcription factors RpoS ( 3 , 14 ) , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) ( 3 ) , HN-S ( 11 ) , and EvgA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>For example , ptsG transcription is activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) ( 2 ) , and negative control is exerted by the repressor Mlc , which inhibits ptsG transcription in the absence of glucose ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873911" target="_blank">23873911</a>]<br></li>
<li>Enzyme synthesis of the cycle is activated by cyclic AMP ( cAMP ) and cAMP receptor protein ( CRP ) in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>We show here that Fis and cyclic AMP ( cAMP ) receptor protein ( CRP ) - cAMP collaborate to activate transcription synergistically in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Previously , the only in vivo information regarding SET family members was limited to sotA of D. dadantii : the transcription of sotA is activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) and weakly repressed by the 2-keto-3-deoxygluconate repressor KdgR ( 4 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>171 , No. 2 Effect of Mutations in the Cyclic AMP Receptor Protein-Bindi Site on araBAD and araC Expression LORI STOLTZFUSt AND GARY WILCOXt * Department of Microbiology , University of California at Los Angeles , Los Angeles , California 90024 ng Received 27 July 1988/Accepted 16 November 1988 Maximum expression of the adjacent but divergently transcribed araBAD operon and araC gene requires the presence of cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>In addition , maximum expression of the araBAD op-eron and araC gene requires the presence of cyclic AMP ( cAMP ) and the pleiotropic regulator cAMP receptor protein ( CRP ) ( 1 , 6 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>We also have found that the timing and/or magnitude of this induction depends , in part , on the carbon regulator cyclic AMP ( cAMP ) receptor protein ( CRP ) , the oxygen regulator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Cyclic AMP ( cAMP ) receptor protein ( CRP ) , also known as catabolite activator protein ( CAP ) is one of the global regulatory factors involved in station-ary-phase induction of large groups of genes in E. coli ( reviewed in reference 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In Escherichia coli , cyclic AMP ( cAMP ) receptor protein ( CRP , also referred to as CAP ) is a global transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>We therefore examined the effects of cpx mutations upon cyaR-lux expression under highglucose conditions where cyclic AMP ( cAMP ) levels would be expected to be low , and therefore cAMP receptor protein ( CRP ) / cAMP would have little effect on cyaR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>The ula regulon also seems to be controlled by the global regulators cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and fumarate nitrate reductase regu - lator ( FNR ) , since mutations in the corresponding genes abol - ished or delayed , respectively , growth on L-ascorbate ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The results in this report also demonstrate that gntT is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which was found be essential for full induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>The cyclic AMP ( cAMP ) receptor protein ( CRP ) mediates a fourfold positive effect on gcv expression as measured from a gcvT-lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex stimulates the expression of lsrR and the lsr operon , which includes lsrACDBFG , while LsrR represses their expression and is located adjacent to , while being transcribed divergently from , the lsr operon [ 14-16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>The distal P1 promoter appeared to be independent of carbon sources , whereas the proximal P2 promoter was shown to be subject to control by the cyclic AMP ( cAMP ) receptor protein ( CRP ) - cAMP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>The in vivo rate of glycogen biosynthesis has been shown to be affected by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) system ( 17 , 18 , 37 ) and by the relA gene , which is required for the synthesis of guanosine 5 ' - diphos-phate 3 ' - diphosphate ( ppGpp ) during the stringent response ( 10 , 38 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Sequence analysis of the DNA fragments ﬂanking the mini-Tn10kan insertion showed that both plasmids contained the same target gene , cra , involved in cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) - independent global modulation of carbon ﬂow ( 22 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>Transcriptional regulation is modulated by the level of cyclic AMP ( cAMP ) synthesized by the membrane-bound adenylate cyclase and cAMP receptor protein ( CRP ) , a global transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In the course of identiﬁcation of regulation targets by cyclic AMP ( cAMP ) receptor protein ( CRP ) , we realized that the cAMP-CRP complex is involved in regulation of some ribosomal protein genes , including the rmf gene ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In this study , the cyclic AMP ( cAMP ) receptor protein ( CRP ) is shown to be involved in positive regulation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>It was shown recently that the functions of the E. coli lsr operon and its regulators , LsrR and LsrK , are similar to those in S. enterica serovar Typhimurium , and cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) are involved in regulation of the lsr operon ( 50 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>For glucose metabolism , E. coli has developed pleiotropic regulators for the transcriptional regulation , such as cyclic AMP ( cAMP ) and its receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>in the presence of acetate and is regulated by cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and FNR as well as by IclR and its activator , FadR ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>Several operons encoding b-oxidation cycle enzymes are also activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) -- cAMP complex and therefore subjected to carbon catabolite repression in the presence of glucose11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21832992" target="_blank">21832992</a>]<br></li>
<li>Many of the gal regulon genes are activated in the presence of the adenosine cyclic-3 ,5 - monophosphate ( cAMP ) -- cAMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Transcription of tnaA is under the control of cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) ( 78 ) ; hence , catabolite repression is an important regulatory mechanism involved in indole synthesis and bioﬁlm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>Some of the major effects of catabolite repression are believed to be mediated through cyclic AMP ( cAMP ) plus the cAMP receptor protein ( cAMP-CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3902796" target="_blank">3902796</a>]<br></li>
<li>In enteric bacteria , expression of the pts operon increases about threefold during growth on PTS substrates and requires an intact cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) system ( 72 , 280 , 390 , 471 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Unlike phoA and appA , agp expression in vivo is not influenced by inorganic phosphate in the medium , and opposite to appA , it is positively controlled by cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2153660" target="_blank">2153660</a>]<br></li>
<li>Moreover , expression was found to be dependent on cyclic AMP ( cAMP ) / cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>This , in turn , results in cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) - dependent expression of lacZ , which can be detected as blue colonies on LB plates containing both X-Gal and IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22328664" target="_blank">22328664</a>]<br></li>
<li>Glucose represses the ula regulon via at least two mechanisms , one dependent on cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and the other ( possibly inducer exclusion ) independent of it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Initiations from P2 depend on the cyclic AMP/cyclic AMP receptor protein complex ( cAMP/CRP ) and are negatively controlled by the cytR repressor , the inducer being cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453417" target="_blank">16453417</a>]<br></li>
<li>Next , we assessed the role of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex in arb gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11994151" target="_blank">11994151</a>]<br></li>
<li>KEINOSUKE OKAMOTO AND MARTIN FREUNDLICH Department of Biochemistry , State University of New York , Stony Brook , NY 11794 ABSTRACT Expression of the crp gene is negatively autoregulated by the complex of cyclic AMP and its receptor protein ( cAMP-CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) ( 56 ) activate AI-2 uptake via lsrACDB ( 56 ) and indole synthesis via tnaAB ( 55 ) and inhibit 244 AI-2 synthesis via luxS ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16597943" target="_blank">16597943</a>]<br></li>
<li>Cyclic AMP ( cAMP ) and its receptor protein ( CRP ) have opposite effects on the gal promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>All CytRregulated promoters studied so far are activated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>With the exception of fadL each is also positively controlled by cyclic adenosine monophosphate ( cAMP ) and cAMP-receptor protein ( CRP ) [ 88 ± 90 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>We present evidence that galS is ( i ) negatively autoregulated and ( ii ) positively regulated by cyclic AMP ( cAMP ) receptor protein ( CRP ) and that galR is constitutively expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>We show that the former is regulated by cyclic AMP ( cAMP ) and its receptor protein , CRP , and that the latter is regulated by phosphorylated PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>cAMP , cyclic AMP ; CRP , cAMP receptor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Subsequently , it was found to be engaged in other site-specific DNA recombination processes ( Ball & Johnson , 1991 ; Thompson et al. , 1987 ; Weinreich & Reznikoff , 1992 ) , to alter DNA supercoiling ( Schneider et al. , 1997 , 2001 ; Skoko et al. , 2005 ; Travers et al. , 2001 ) , to influence the initiation of DNA replication Abbreviations : CRP , cyclic AMP receptor protein ; H-NS , histone-like nucleoid structuring protein ; IHF , integration host factor ; qRT-PCR , reverse transcriptase quantitative PCR ; RNAP , RNA polymerase ; WMW , Wilcoxon -- Mann -- Whitney ; WT , wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>At least two regulatory classes of these genes exist : those whose induction is positively regulated by adenosine 3 ' ,5 ' - cyclic monophosphate ( cAMPf ) and its receptor protein ( CRP ) ( the cst genes ) and those independent of this control ( the pex genes ) ( Schultz et al. , 1988 ; Schultz & Matin , 1988 ; Blum et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>ExuR , another Abbreviations : Ap , ampicillin ; bp , base pair ( s ) ; CAMP , cyclic AMP ; Cm , chloramphenicol ; CRP , CAMP receptor protein ; kb , repressor involved in the regulation of the hexuronate system genes , has no effect on the expression of uidA ( Ritzenthaler et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>For example , cyclic AMP receptor protein ( CRP ) and IHF are also known to play similar roles in gene regulation in other promoters ( Lobell and Schleif , 1990 ; Søgaard-Andersen et al. , 1991 ; Richet , 1996 ; Yang et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Abbreviations : amp , / I-lactamase gene ( ApR ) ; Ap , ampicillin ; bp , base pair ( s ) ; BLA , / 3-lactamase ; CAMP , cyclic adenosine-3 ' : 5 ' - monophosphate ; Cm , chloramphenicol ; crp , CRP-coding gene ; CRP , CAMP receptor protein ; cya , adenylcyclase gene ; d , deletion ; EtdBr , ethidium bromide ; kb , kilobases ; IPTG , isopropyl-B-thio-D-galactoside ; MOPS , 3 - ( N-morpholino ) - pro-panesulfonic acid ; ODC , ornithine decarboxylase ; PA , polyacrylamide ; R , resistance ; SDS , sodium dodecyl sulfate ; speC , gene coding for ODC ; Tc , tetracycline ; ret , Tc resistance gene ( TcR ) ; [ 1 , designates the plasmid-carrier state ; : : , novel joint .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; Ap , ampicillin ; bp , base pair ( s ) ; BSA , bovine serum albumin ; CAMP , adenosine 3 ' ,5 ' - cyclic monophosphate ; CRP , CAMP receptor protein ; gp , gene product ; GDH , glutamate dehydrogenase ; GOGAT , glutamate 2-oxoglutarate + NH , ' + NADPH + L-glutamate + NADP ' + H , O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2465204" target="_blank">2465204</a>]<br></li>
<li>At 661 0 1991 Academic Press Limite t Abbreviations used : CAP = CRP , catabolite activator protein , CAMP receptor protein ; CAMP , cyclic AMP ; GlcNAc , N-acetylglucosamine ; PEP , phosphoenolpyruvate ; PTS , phosphotransferase system ; GlcN , glucosamine ; bp , base pair ( s ) ; kb , kilobase pairs ; DMS , dimethyl sulphoxide ; DTE , dithioerythritol ; IPTG , isopropyl-fi-n-thiogalactoside ; IHF , integration host factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>$ Abbreviations used : P!t 'S , phosphoenol pyruvate-dependent sugar phosphotransferase system ; PEP , phosphoenol pyruvate ; CRP , CAMP receptor protein ; REP , repetitive extragenic palindromic ; ORF , open reading frame ; kb , 10 ' bases or base-pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3062173" target="_blank">3062173</a>]<br></li>
<li>Also , it has been demonstrated that there exist internal sites for transcription initiation of this operon in both Abbreviations : Ap , ampicillin ; bp , base pair ( s ) ; CAMP , 3 ' , 5 ' - cyclic AMP ; CRP , CAMP receptor protein ; EtdBr , ethidium bromide ; GalK , galactokinase ; ILV , isoleucine , leucine , valine ; kb , 1000 bp ; LB , Luria Bertani ; nt , nucleotide ( s ) ; PolIk , Klenow ( large ) fragment of E. coli DNA polymerase I ; RNAP , RNA polymerase ; SDS , sodium dodecylsulfate ; Tn , transposon ; [ 1 , designates plasmid-carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2473940" target="_blank">2473940</a>]<br></li>
<li>In E. cofi , a CAMP-receptor protein but lies outside the I1A site , and the plasmid was not ( CRP ) - induced DNA bend that contributes t o trans-reconstructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>Abbreviations : bp , base pairs ; CRP , CAMP receptor protein ; EtBr , ethidium bromide ; kb , kilobase pairs ; PMI , phosphomannose isomerase ; SDS , sodium dodecyl sulphate ; ssDNA , single-stranded DNA ; [ 1 , indicates plasmid-carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6397402" target="_blank">6397402</a>]<br></li>
<li>However , open complex formation in the presence of RNA polymerase , which is an indication for active FNR and transcriptional activation , is stimulated by Fe ( II ) Most of the FNR-dependent promoters are regulated by FNR in cooperation with other regulators such as NarL , NarP ( nitrate regulation ) , ArcA , Fur ( ferric uptake regulation ) , or CRP ( cyclic AMP receptor protein , glucose repression ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>32 , No. 19 doi :10.1093 / nar/gkh908 Identification of the CRP regulon using in vitro and in vivo transcriptional profiling Dongling Zheng , Chrystala Constantinidou * , Jon L. Hobman and Stephen D. Minchin School of Biosciences , University of Birmingham , Birmingham B15 2TT , UK Received July 2 , 2004 ; Revised September 3 , 2004 ; Accepted October 7 , 2004 ABSTRACT The Escherichia coli cyclic AMP receptor protein ( CRP ) is a global regulator that controls transcription initiation from more than 100 promoters by binding to a specific DNA sequence within cognate promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>INTRODUCTION The cyclic AMP receptor protein ( CRP or CAP ) of Escherichia coli is a sequence-specific DNA binding protein which regulates transcription of a number of genes in response to the carbon source ( 1-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>The effects of 11 mutations within the DNA region thought to-bind the cyclic AMP receptor protein correlate well with the CRP consensus binding sequence and confirm that this region is responsible for cyclic AMP regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>The acceleration of Fur proteolysis in the presence of metal is analogous to the behavior of the E. coli CRP protein : cyclic AMP binding to the N-terminal domain of CRP induces protease sensitivity in the C-terminal DNA-binding domain ( Heyduk & Lee , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>Its expression is also dependent on the global regulatory proteins mediating catabolic repression via cyclic AMP receptor protein ( CRP ) , or anaerobiosis via fumarate nitrate reduction ( FNR ) transcriptional-regulator , and the nucleoid-associated DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>It is structurally-related to CRP ( the cyclic AMP receptor protein ) except for the presence of a cysteine-rich / V-terminat extension , which may form part of an iron-binding , redox-sensing domain in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Transcription of the glp operons is subject to multiple controls , including catabolite repression mediated by cyclic AMP-CRP and respiratory control mediated by the FNR and ArcA/ArcB systems ( 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Expression of the gal operon is regulated by cellular levels of CAMP and the CAMP receptor protein ( CRP ) through the mechanism of catabolite repression ( for review , see de Crumbrugghe and Pastan , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>The synthesis of PM1 is probably subject to catabolite repression mediated by CAMP and the CAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6397402" target="_blank">6397402</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_581">
The dcuB gene is strongly induced anaerobically by FNR , repressed in the presence of nitrate by NarL , and is subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="act_similarity_581", type="checkbox">
<div class="ACT_similar">
<li>The dcuB gene is strongly activated anaerobically by FNR , repressed in the presence of nitrate by NarL , and subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li></div>
<li>Regulation studies with a dctA-lacZ ( f428-lacZ ) transcriptional fusion showed that dctA is subject to cyclic AMP receptor protein ( CRP ) - dependent catabolite repression and ArcA-mediated anaerobic repression and is weakly induced by the DcuS-DcuR system in response to C4-dicarboxylates and citrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Growth with glucose causes catabolite repression , one manifestation of which is lowered CRP protein activity ( due to lowered cyclic AMP [ for a review , see reference 21 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Transcription of these operons is sensitive to catabolite repression and requires cyclic AMP and the catabolite repressor protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094697" target="_blank">10094697</a>]<br></li>
<li>1 Abbreviations : bistris , 2 - [ bis ( 2-hydroxyethyl ) amino ] -2 - ( hydroxymethyl ) - propane-1 ,3 - diol ; CRP , Escherichia coli cyclic AMP receptor protein ; CytR , Escherichia coli cytidine repressor protein ; DNase I , deoxyribonuclease I ; RNAP , bacterial RNA polymerase , Tris , tris ( hydroxymethyl ) amino-methane ; EDTA , ethylenediaminetetraacetic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In E. coli , the nanATEK-yhcH operon is positively controlled by cyclic AMP receptor protein ( CRP ) and negatively regulated by the upstream nanR-encoded transcriptional-repressor , NanR ( 6 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Summary Activation of the two divergent Escherichia coli cai and ® x operons involved in anaerobic carnitine meta-bolism is co-dependent on the cyclic AMP receptor protein ( CRP ) and on CaiF , the speci ® c carnitine ¬ sensitive transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>oside catabolizing enzymes in Escherichia coil : effects of 3 ' 5 ' cyclic AMP and CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Multiple regulation of nucleoside catabolizing enzymes in Escherichia coli : effects of 3 ' 5 ` cyclic AMP and CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Hammer-Jespersen , K. , and Nygaard , P. ( 1976 ) Multiple regulation of nucleoside catabolizing enzymes in Escheri-chia coli : effects of 3 9 cyclic AMP and CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>( d ) Mapping the psiE locus Hfr crosses showed that the p & E-la & fusion maps within the 88 to 8 minute interval on the E. coli map ( Bachmann , 1983 ) ; and , the results here show that the p & E promoter is negatively regulated by the adenosine 3 ' ,5 ' - cyclic mono-phosphate ( CAMP ) - CAMP receptor protein ( CRP ) system , as was reported ( Wanner , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3540312" target="_blank">3540312</a>]<br></li>
<li>Effect of altering the putative upstream promoter on ilvB expression DISCUSSION Transcription of the ilvB operon is positively activated by cyclic AMP and its receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Expression from the galS promoter was stimulated by the addition of D-fucose , repressed by glucose , and dependent on cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>CRP , the cyclic AMP receptor protein that regulates the expression of a number of catabolite-sensitive operons , is known to activate the lac operon by direct interaction with the a subunit of the RNA polymerase complex ( 15 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836282" target="_blank">7836282</a>]<br></li>
<li>It has been reported that Mlc regulates manXYZ encoding enzyme II of the mannose PTS ( 4 ) , 1 The abbreviations used are : CRP , cyclic AMP receptor protein ; PTS , phosphoenolpyruvate : carbohydrate phosphotransferase system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11340070" target="_blank">11340070</a>]<br></li>
<li>The feaR gene is upregulated by carbon or nitrogen limitation , which we propose reﬂects regulation of feaR by the cyclic AMP receptor protein ( CRP ) and the nitrogen assimilation control protein ( NAC ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>The global regulator cyclic AMP receptor protein ( CRP ) administers additional control over GadX production , tying it to the carbon energy state of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>The operon is known to be subject to activation by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex , and mutations that specifically render the mtl operon independent of CRP frequently also render it constitutive ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300537" target="_blank">8300537</a>]<br></li>
<li>The mechanism of FruR action proved to be independent of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex , but the extent of regulation nevertheless correlated roughly with sensitivity to repression/activation by exo-genous sugars ( Chin efai , 1987 ; 1989 ; Feldheim etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>tre operon expression is under the control of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex as well as that of TreR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>As expected , gut operon expression is under the control of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>This operon is also subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>ptsG but not crr is subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The operon is also positively controlled by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex and negatively by the catabolite repressor / activator ( Cra ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_606">
The operon is subject to positive control by cyclic AMP and the cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></label>
<input id="act_similarity_606", type="checkbox">
<div class="ACT_similar">
<li>The manXYZ operon is also subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The gat operon is subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li></div>
<li>FNR shows sequence and functional similarity to CRP , the cyclic AMP receptor protein ( BELLet al. 1989 ; SPIROand GUEST1987 ) , although the mechanism by which FNR senses ana-erobiosis is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>One operon , araFGH , codes for the well-studied AraC protein and by cyclic AMP ( cAMPt ) receptor protein ( CRP ) ( Englesberg et al. , 1965 ; Greenblatt & Schleif , 1971 ; Wilcox et al. , 1974 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>We have used in vitro and in vivo experiments to show that cyclic AMP and the cyclic AMP receptor protein ( CRP ) directly regulate ompB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>It has previously been established that the synthesis of L-asparaginase II is subject to regulation by cyclic AMP receptor protein ( CRP ) and is also induced by anaerobiosis ( 6 , 8 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>Here , we present evidence that this switch occurs primarily through the induction of acs and that the timing and magnitude of this induction depend , in part , on the direct action of the carbon regulator cyclic AMP receptor protein ( CRP ) and the oxygen regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>This mechanism will activate the cyclic AMP receptor protein ( CRP ) ( 43 ) , a global transcriptional factor that positively regulates most carbon catabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>erase , 20 ng of Hpa I1 fragment ( fragment 3 ) , and as indicated in the he : ' abbreviations used are : CRP , cyclic AMP receptor protein ; n , nucleotide ; bp , base pair ; SDS , sodium dodecyl sulfate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>pBADis also positively regulated by AraC-arabinose and the cyclic AMP receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The araBALl genes are also regulated by the cyclic AMP receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Here , we show that cspD is positively regulated by cyclic AMP receptor protein ( CRP ) , a global metabolic regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Here , we show that the cyclic AMP receptor protein ( CRP ) , a global regulator involved in sugar metabolism , upregulates cspE in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>A comparison with similar sequences of other bacterial or phage operons could aid in elucidating general features of Abbreviations : CRP , cyclic AMP receptor protein ; P-RNase , bovine pancreatic RNase A ; CM-RNase , e-lysine 41 carboxy-methylated pancreatic RNase A ; NTP , nucleoside 5 ' - triphos-phate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=4612533" target="_blank">4612533</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_621">
[ 60 ] S. Garges , S. Adhya , Cyclic AMP-induced conformational change of cyclic AMP receptor protein ( CRP ) : intragenic suppressors of cyclic AMP-independent CRP mutations , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></label>
<input id="act_similarity_621", type="checkbox">
<div class="ACT_similar">
<li>Cyclic AMP-induced conformational change of cyclic AMP receptor protein ( CRP ) : intragenic suppressors of cyclic AMP-independent CRP mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2165479" target="_blank">2165479</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2180908" target="_blank">2180908</a>]<br></li></div>
<li>TCA , tricarboxylic acid cycle ; MDH , malate dehydrogenase ; kbp , kilopase-pairs ; CRP , cyclic adenosinemonophosphate receptor protein Offprint requests to : K.-D .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>galactose operon : bp , base-pair : CRP , cyclic AMP receptor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>KD , apparent KD ; CRP , cyclic AMP receptor protein ; DcuR-P , phosphoryl DcuR ; TBE , Tris/borate/EDTA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073297" target="_blank">15073297</a>]<br></li>
<li>Antibodies against AscG , cyclic AMP receptor protein ( CRP ) , and PrpR were raised in rabbits by injection of the respective puriﬁed proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>The dadAX operon is expressed by multiple promoters that are repressed by leucine-responsive regulatory protein ( Lrp ) and activated by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>in vitro and in vivo experiIn this report , we show by ments , including primer extension analysis and site-directed mutagenesis , that cyclic AMP and the cyclic AMP receptor protein ( CRP ) directly regulate the expression of the ompB The overall positive effect by cyclic AMP ompB on operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Transcription from acsP2 is totally dependent on activation by the cyclic AMP receptor protein ( CRP ) , which binds at two sites ( CRP I and CRP II ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Evidence for AraC interactions with 70 come from early 70 mutations ( eventually identiﬁed at R596 ) that increased araBAD expression in the absence of activation by cyclic AMP receptor protein ( CRP ) ( 18 , 39 , 44 ) , as well as the ﬁnding that araBAD expression in a cya strain was increased by 70 E591A and R596A and decreased by 70 K593A ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Thus , if O ~ CTDis removed , transcription activation from Class I promoters is abolished ( lgarash Plate 1 Structural lold of the cyclic AMP receptor protein ( CRP ) bound in dimeric form to target DNA ( Schultz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , can not activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>Cyclic AMP receptor protein ( CRP ) is required for full expression of all three of the rha operons , but it functions as a coactivator that substantially activates transcription only when RhaS or RhaR also binds to the promoter regions ( 11 , 15 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Transcription from pmelAB is totally dependent on MelR and the presence of melibiose in the medium ( 2 ) , while transcription from pmelR depends on the global transcription activator , the cyclic AMP receptor protein ( CRP ) ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>An additional level of repression of gad genes is provided by the cyclic AMP receptor protein ( CRP ) , which was demonstrated to repress RpoS-de-pendent gad transcription during the exponential phase in rich ( LB ) medium ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Regulation of transcription of the araBAD and araFGH operons depends upon promoter occupancy by cyclic AMP receptor protein ( CRP ) ' and AraC protein ( 6 , 8 , 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447222" target="_blank">1447222</a>]<br></li>
<li>Here , we show that cyclic AMP receptor protein ( CRP ) upregulates cspD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Abbreviations : CRP , cyclic AMP receptor protein ; Fis , factor for inversion stimulation ; FNR , transcription factor for fumarate-nitrate reduction anced production of GMP with that of AMP and with the pyrimidine nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>These studies also show that the cyclic AMP receptor protein ( CRP ) interacts with the melAB promoter and increases MelR-dependent transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Substitution to alanine of any of these res-idues severely affects transcription activation by the Fnr protein and by cyclic AMP receptor protein ( CRP ) , at the dmsA and pmelRcon promoters , respectively , while R596 appears to be the target site for cI ( 30 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>1984 ; Monta etai , 1985 ) whereas this important sequence is not preserved in CRP site II , This suggests that CRP site II may have a lower affinity to CRP-cAMP than CRP site I. To test this , we performed a DNase I footprinting assay of CRP and the two CRP sites in Taq \ - Hin6 \ \ \ fragments in the presence of various concentrations of cAMP , Figure 7 shows the effect of the cAMP concentration on the footprint of the wild-type fragment by CRP , Strong protection by CRP-cAMP was observed at CRP site I and the cAMP concentration needed for full protection was about 16 ^ M. On the other hand , the protection of CRP site II was rather moderate and higher concentrations ( more than 32 \ iU ) of cAMP were required for clear protection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Also this time appeared to be dependent on the UTP concentration ; it was further increased , at least in the absence of Table H. Effects of CRP-cAMP on kinetic constants for substrates at the malT promoter ( I0O ( DA ( DB ( I ) AB amax/min KA uM KB FAM Kia FM x 102 min x 106 min x 107 min M x 10 '' min M2 1/1o 4A/40 ( B / cents ' O ( ABO+B - CRP 1.27 0.23 2 ± 0.17 2 ± 0.17 5.7 ± 0.7 78 14 160 50 16 ± 5 280 ± 80 + CRP 1.04 ± 0.06 0.53 ± 0.1 0.23 A 0.07 0.97 ± 0.13 96 ± 6 52 ± 10 2.2 A 0.7 420 100 Assay conditions are described in the legend of Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Reaction mixtures contained 0,65 nM template , 20 nM RNA polymerase , 100 nM CRP , and indicated concentrations ot cAMP , B , The relative amounts of transcripts from crp and divergent promoters were determined by densitometric tracing of the autoradiogram and plotted as a function of cAMP concentration : • , crp RNA ; A , divergent RNA , The amount of crp RNA in the absence of cAMP was taken as 100 , B CAMP (  iiM -RCB- 0 0.5 1,5 4,5 13,5 40.5 -- divergent RNA 3 2 1 4 S 6 0 ' ' 0,1 CAMP Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>B , The relative amounts of transcripts from crp and divergent promoters were determined by densitometric tracing of the autoradiogram and plotted as a function of cAMP concentration : • , crp RNA ; A , divergent RNA , The amount ot crp RNA in the absence of cAMP was taken as 100 Differentiai binding affinities of CRP-cAhJiP to fwo CRP sites The differential sensitivity of two promoters to cAf ^ P mentioned above may be simply explained by a difference in the relative affinities of CRP-cAMP complex to the CRP sites 1 and II , since it is well documented that deviations from homology with the consensus sequence in CRP recognition sites significantly lower the binding affinity between CRP sites and CRP-cAMP ( Kolb etai , 1983a , b ; Berg and von Hippel , 1988 ) , As shown in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Based on these results , we propose the following model for transcriptional regulation of crp and divergent promoters -LCB- Fig. 9 ) , Three extreme states are characterized by the concentrations of CRP and cAMP , In the absence of the CRP-cAMP complex -LCB- A ) , only the crp promoter is functional and RNA polymerase transcribes crp at a basal level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Thus , the divergent promoter is activated and the crp promoter is inhibited , since RNA polymerase occupancies at two promoters are mutually exclusive -LCB- Hanamura and Aiba , 1991 ) , At higher concentrations of CRP-cAMP ( C ) , the complex also binds to CRP site II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>In the presence of CRP-cAMP , the complex first binds to CRP site I to direct RNA polymerase to the divergent promoter , At this stage the divergent gene is activated and the crp gene is repressed ( B ) , In the presence of higher concentrations ot CRP-cAMP , CRP site II becomes occupied by CRP-cAMP , This causes again repositioning of RNA polymerase resulting in the activation of crp transcription ( C ) , RNA polymetase CRP site I B Experimental procedures Proteins and chemicais RNA polymerase was purified from E co/f strain W3350 essentially according to the method of Fukuda etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>In contrast to Es , Es can use this UP-element half-site 70 S and cAMP-CRP bound at 268.5 independently for activation ( i.e. each element or component activates in the absence of the other ; Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In the absence of cAMP , no specific peaks were identified for CRP ( Fig. 1 , open circles ; but note that open circles are hidden behind closed circles ) , indicating that the peaks thus identified represent specific binding of cAMP-bound CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In the presence of both CRP and cAMP , we could observe CRP-bound complexes ( Fig. 3b ) , but the CRP-DNA complexes could not be observed in the absence of cAMP ( Fig. 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In the absence of CytR , cAMP-CRP interacts at one set of sites ( CRP-1 and CRP-2 ) and both of these binding sites are required for full promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the absence of the CytR repressor , cAMP-CRP binds to two targets ( CRP-1 and CRP-2 ; Figure 1 ) and it is evident from our analysis of deletion mutants that the high affinity site , CRP-1 , is the essential site for promoter activation ( Table IA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In contrast to the strong CytR stimulation of cAMP-CRP binding to CRP-1 and CRP-2 in the wild-type promoter , in conjunction with protection of the spacer region ( compare lanes 3 and 6 , right panel ) , there is only a weak effect of CytR in the promoter fragment carrying the single mutation at - 4 5 at the lowest cAMP concentration ( lanes 3 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In this context it should be noted that the dependence of CytR on cAMP-CRP is in striking contrast to the action of the classical bacterial repressors , which appear to regulate promoter activity in the absence of additional regulator proteins ( for reviews , see Adhya 1989 ; Gralla 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In the presence of CytR ( B ) , the cAMP ± CRP , especially that bound at the promoter-proximal site , undergoes an allosteric change such that it no longer interacts with a-NTD positively , but instead negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of both cAMP-CRP and CytR , however , this footprint remains unchanged , in contrast to the strong effects observed in the wild-type promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>of Operator site experimentsb CytR at CRP2 CytR CRP1 CRP2 213.5 6 0.3 -- All CRP None 4 -- 212.0 6 0.1 All CytR None 13 210.7 6 0.4 -- WT/CRP12 CytR None 8 -- CRP22 CytR None 3 -- Wild type CRP CytR 2 213.9 6 0.1 -- 213.5 6 0.1 CRP12 CRP CytR 2 213.7 6 0.4 -- -- CRP22 CRP CytR 2 -- -- 212.9 6 0.1 Wild type CytR CRP 5 -- 212.8 6 0.2 -- CRP12 CytR CRP 3 -- 212.4 6 0.3 -- CRP22 CytR CRP 4 -- 212.7 6 0.2 -- Wild type CytR CRP , cytidine 3 -- 212.2 6 0.4 -- CRP12 CytR CRP , cytidine 2 -- 211.9 6 0.3 -- CRP22 CytR CRP , cytidine 2 -- 211.2 6 0.1 -- a Effector concentrations : CRP , 0.1 mM ( total dimer ) ; cAMP , 150 mM ( present in all experiments ) ; CytR , 75 nM ( dimer ) ; cytidine , 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>of expts b c CytR at CRP2 CytR at CRP1 CytO CRP2 CRP1 wild type CRP1 CRP2 CRP none 7 - ¬ -12.4 ( 0.4 -13.8 ( 0.3 wild type CRP1 CRP2 CytR none 10 -- wild type CRP CytR 2 CRP1 CRP CytR 2-CRP2 CRP CytR 3-wild type CytR CRP 4 CRP1 CytR CRP 2-CRP2 CytR CRP 3-wild type CRP CytR , cytidine 3 CRP1 CRP CytR , cytidine 2-CRP2 CRP CytR , cytidine 2-wild type CytR CRP , cytidine 5 -12.9 ( 0.4 CRP1 CytR CRP , cytidine 2 - -12.3 ( 0.2 CRP2 CytR CRP , cytidine 2 - -12.8 ( 0.1 a Free energy changes for saturation of cdd operators with either CRP ( cAMP ) 1 or CytR in the presence or absence of effector ligands are as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The mechanism of repression by CytR proposed in this study is that the cAMP ± CRP bound at ± 41.5 undergoes an allosteric change upon direct interaction with CytR such that it no longer maintains a productive interaction with the N-terminal domain of a , but instead acts as a repressor to interfere with RNA polymerase acting on deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Neither CRP in the absence of cAMP , nor the CytR extract changes the cleavage pattern of DNase I ( lanes 2-3 ) relative to that in the control in which no protein is added ( lane 1 ) , Th ` footprint ' produced by cAMP-CRP alone is displayed in lanes 4-6 ; cAMP-CRP binds to two targets , with high and low affinity located around positions - 9 3 ( CRP-2 ) and - 4 0 ( CRP-1 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In order to analyze the possibility that CytR afﬁnity to different promoters was mediated by their ability to-bind cAMP -- CRP , gel retardation experiments with the same 32P-end-labeled DNA fragments were performed in the presence of puriﬁed E. coli CRP protein ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Binding of cAMP - CRP and CytR to wild-type and mutant promoters In order to investigate the interaction of cAMP-CRP and CytR with cddP and to define the role of the different protein binding sites , we performed DNase I footprinting experiments on 32P-end-labelled fragments in the presence of purified proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>These observations are not easily explained by current models for the action of pro-karyotic repressors ; instead we favour a model in which the interaction of CytR with deoP2 depends on the presence of tandem DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Hence , CytR relies on interactions with DNA-bound cAMP-CRP complexes in order to-bind strongly to deoP2 , and cAMP-CRP may be viewed as an adaptor that allows CytR to associate with deoP2 ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>1989 ; L. Sogaard-Andersen and P. Valentin-Hansen , in prepar-ation ) , strongly indicate that the structure recognized by CytR includes tandem DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>The region protected from DNase I by formation of the CRP -- CytR -- DNA complex corresponds to the combined footprint of cAMP -- CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The inducer for the CytR repressor , cytidine , does not prevent in vitro DNA binding of CytR , but releases the repressor from the nucleoprotein complex and leaves the cAMP-CRP activator bound to its two DNA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In this paper , we show that the cAMP-CRP complex is indeed a crucial component of fhe CytR repression apparatus , and that CyfR relies on interactions with DNA-bound activator proteins in order to-bind at deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Detailed investigation of the deo operon has shown that cAMP-CRP and CytR bind cooperatively to the deoP2 promoter forming a nucleoprotein complex in which CytR is sandwiched between two DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Summary Repression by CytR depends on the formation of nucleoprotein complexes in which the CytR repressor and the cAMP-CRP activator complex bind cooperatively to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Therefore , in the mobility shift assay , CytR and one cAMP-CRP complex still bind co-operatively to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>CytR simultaneously makes specific DNA contacts as well as protein-protein contacts to cAMP-CRP in the repression complexes ( Pedersen et ai , 1991 ; Sogaard-Andersen and ValentinHansen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>It has consistently been shown that CytR and cAMP-CRP bind cooperatively to deoP2 forming a nucleoprotein complex in which the repressor protein bridges the tandem DNA-bound cAMP-CRP ( Pedersen et al. , et complexes 1991 ; S0gaard-Andersen al. , The 1991a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>These results are similar to those reported for deoP2 ( Pedersen et al. , 1991 ) and the in vitro analyses strongly indicate that the same type of repression complex , consisting of CytR sandwiched between tandem DNA-bound cAMP-CRP complexes , is involved in negative regulation of these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Taken together , these data point to a strong coupling between D N A binding of cAMP-CRP and the extent of CytR regulation in deoP2 , and , thus , strongly corroborate the conclusion obtained from in vitro experiments that CytR relies on interaction with tandem DNA-bound cAMP-CRP complexes in order to-bind to deoP2 D N A ( Sogaard-Andersen et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Binding of CytR to nupG is stimulated more than 3000-fold by cAMP-CRP ( Fig. 2 , compare lanes 3 and 14 ) , and also , the presence of CytR increases protection in the downstream region considerably ( Fig. 2 , lanes 6 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Binding of cAMP-CRP is , however , enhanced by the CytR extract ( Fig. 2 , right panel ; compare lanes 4-6 with lanes 7-9 ; see also Fig. 4 , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Firstly , the cleavage pattern in CRP-l is identical to that obtained in the presence of saturating concentrations of cAMP-CRP alone ( Figure 3A , compare lanes 3-4 with 5-6 and 9-10 with 11-12 ) indicating that cAMP - CRP also occupies this target in the presence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>First , the DNase I cleavage pattern in the CRP targets is identical to that obtained in the presence of saturating concentrations of cAMP-CRP alone ( compare lanes 6-9 ; see also Fig. 4 , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>combined footprint of the tsx-503 allele is that the cAMP-CRP complex can interact strongly with the CRP-2 target in the presence of CytR ( Fig. 7B , lane 11 ) , although the affinity of cAMP-CRP for the mutated target is severely reduced ( Fig. 6 , lanes 31 to 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In contrast , the presence of cytidine eliminates the full repression complex ( lanes 5 and 11 ) ; it does not , however , abolish co-operative binding of CytR and one cAMP-CRP complex ( Fig. 4 , compare lanes 2 , 9 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In the intermediate CRP-CytR-nupG complex , cAMP-CRP bound at CRP2 also strongly stimulates binding of CytR ( Fig. 2 , compare lanes 3 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Initiation of transcription from deoP2 is negatively controlled by two repressors , DeoR and CytR , and is also dependent on the cAMP/CRP complex Location of transcription terminators in the 3 ' - end of the deo operon In order to locate termination signals , we inserted restriction fragments from the 3 ' - end of the deo operon between the deoP2 promoter and the lacZ gene of pJEL144 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3299264" target="_blank">3299264</a>]<br></li>
<li>Unlike the other members of this protein family , CytR binds with only modest afﬁnity to its operators and transcription repression thus relies on the formation of nucleoprotein complexes with the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Thus , CytR does not regulate transcription from deoP2 in cells depleted for the cAMP-CRP complex ( Acya or Acrp strains ) , even when the level of the repressor in the cells is high ( Sogaard-Andersen et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>As shown in Table 1 , initiation of transcription is regulated strongly both by the CytR repressor ( 40-fold ) and by the cAMP / CRP complex ( 100-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Release of CytR from the nucleoprotein complex in response to cytidine leaves the cAMP-CRP activator complex bound to tSx-p2 , which will allow immediate stimulation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Speciﬁcally , CytR has a relaxed spacing requirement between operator half-sites and can , in concert with cAMP -- CRP , repress transcription in vivo from octamer boxes separated with centre distances up to three helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>At P1 and P2 transcription is repressed by the DeoR repressor and induced by deoxyribo-se-5-P ; at P2 transcription is also repressed by the CytR repressor ( the inducer being cytidine or adenosine ) and depends on cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Addition of increasing amounts of CytR to transcription assays containing cAMP ± CRP gradually decreased transcription ( Figure 5 , middle panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Thus , it appears that CytR ± CRP communication at deoP2 also affected the cAMP ± CRP-independent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the absence of cAMP ± CRP , addition of CytR had no effect on deoP2 transcription ( Figure 5 , left panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of the cAMP-CRP complex , initiation of transcription from deoCp2 was repressed 10-fold by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Furthermore , CytR only regulates initiations of transcription from deoP2 in the presence of cAMP-CRP ( 7-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Consistently , CytR represses transcription in vivo only in the presence of the cAMP-CRP complex ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>CytR ± CRP-mediated repression at deoP2 To assess CytR regulation at deoP2 in vitro , we ® rst examined transcription with intact RNP in the presence of CytR , CytR and cAMP ± CRP , and CytR , cAMP ± CRP and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In an attempt to determine the role of a-CTD in CytR / CRP-mediated repression at deoP2 , we next investigated transcription by the reconstituted RNP derivatives in the presence of a constant concentration of cAMP ± CRP and varying amounts of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of cAMP-CRP , however , the affinity of CytR for this sequence is greatly increased ( 100-fold ) and , moreover , the two proteins bind cooperatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In the presence of CytR , however , the two regulators bind cooperatively to cddP forming a nucleoprotein complex in which cAMP - CRP binds to CRP-1 and CRP-3 and CytR occupies the sequence between these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the presence of cAMP-CRP , however , the affinity of CytR for this sequence is increased dramatically ( approximately 1,000-fold ) , and CytR and cAMP-CRP bind cooperatively to deo-p2 , forming a nucleoprotein complex in which CytR is sandwiched between two DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>These data indicate that the CytR repressor relies on the presence of the cAMP-CRP activator complex to regulate tsr-p2 promoter activity and that the formation of an active repression complex requires the combined interactions of cAMP-CRP and CytR at tsX-p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Cooperative binding of the CytR repressor and the extent in the presence of cytidine that the promoter no cAMP-CRP activator complex longer responds to the CytR repressor in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Combined DNase I footprint of the wild-type tsx-p2 promoter in the presence of both the CytR repressor and the cAMP-CRP activator complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_704">
To this end , we titrated in parallel the wild-type and mutant promoters with the cAMP-CRP complex and the CytR repressor in the presence of 50 pRM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></label>
<input id="act_similarity_704", type="checkbox">
<div class="ACT_similar">
<li>To this end we titrated , in parallel , the wild-type and mutant promoters with the cAMP-CRP complex in the absence or the presence of the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li></div>
<li>Isolation of DNA sequences that support CRP2 -- CytR -- DNA complex formation We next performed the selection in the presence of both CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Next , we performed footprinting experiments in the presence of both the CytR repressor and the cAMP-CRP activator complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Recently , we have shown in vitro that CytR interacts strongly with deoP2 only in presence of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Protection of the wild-type deoP2 promoter ( right panel ) and a deoP2 promoter deleted tor CRP-2 ( left panel ) from DNase I digestion by CRP alone ( lane 2 ) , by CRP and CytR ( lane 3 ) , by CRP in the presence ot cAMP ( lanes 4 , 5 and 6 ) or by a mixture of CRP , cAMP and CytR extract ( lanes 7 , 8 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Ihe DNase 1 digestion pattern in the wild-type ( wl ) and three mutant derivatives of deoP2 in the presence of CRP , cAMP and CytR extract , as indicated above each lane , are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The cleavage pattern in the CRP2 region is identical to that obtained in the presence of cAMP-CRP alone ( Fig. 2 , lane 5 ; Fig. 3 , lane 6 ) , indicating that cAMP-CRP also occupies this target in the presence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The DNase I digestion pattem in the presence of CRP and various concentrations of cAMP ( shown above each lane -RCB- ( lanes 2 , 3 and 4 ) and in the presence ot CRP , cAMP and CytR extract ( lanes 5 , 6 and 7 ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The presence of both CytR and cAMP-CRP results in protection of a region of approximately 80 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The action of CytR in vivo is absolutely dependent on the presence of cAMP ± CRP , and therefore CRP interacts alternatively , and mutually exclusively , with CytR or RNP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Gel retardation of 32P-end-labeled udpP-containing DNA fragments , using puriﬁed CytR and CRP in the presence of 50 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Two nanomolar DNA was incubated with CytR and CRP in concentrations shown above the gel in the conditions described in Materials and Methods in the presence of 50 mM cAMP , and the mixtures were treated with 0.5 U of DNase I ( Promega ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>To investigate whether cAMP -- CRP has any effect on the DNA recognition speciﬁcity of CytR , we repeated the selection/ampli ﬁcation experiments in the presence of both CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>It thus appears that selection in the presence of cAMP -- CRP has produced DNA targets to which CytR binds in a conformation that is designated to interaction with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the presence of CRP ( Pro-110 - + Ser ) , addition of the CytR extract results in formation of both cAMP-CRP/CytR/deoP2 complexes , but , again , the amounts of complexes formed are less than in the presence of CRP-wt ( compare lanes 3 and 8 , curved arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In this complex , the presence of CytR forces cAMP-CRP to interact at CRP-3 by means of protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>B-E : Underlined letters represent base pairs at which the G is protected from DMS methylation by cAMP-CRP or CytR ; letters in bold indicate increased reactivity of adenine or guanine towards DMS methylation in the presence of protein ( this work ; Reznikoff and Abelson , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Moreover , the efﬁciency of E. coli CytR protein to-bind udpPV c increased markedly in the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>In the presence of cAMP -- CRP , CytR binds the C1 , C2 and C3 sequences with very similar afﬁnity ( Figure 3 , hatched bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the presence of both CytR and cAMP-CRP , guanine positions - 99 , - 74 , - 57 and - 47 on the ( + ) strand ( lane 4 ) and - 8 8 , - 8 6 , - 7 3 and - 5 6 on the ( - ) strand ( lane 8 ) are protected from DMS methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In the presence of both cAMP-CRP and CytR , a combined footprint is obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Two sequences from each of groups C1 , C2 and C3 were footprinted by DNase I and DMS in the presence of CytR and cAMP -- CRP ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Notably , the footprint patterns in the CytR region are relatively independent of the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Second , the remaining parts of the footprint strongly deviate from those obtained in the presence of cAMP-CRP or CytR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The footprint obtained with cAMP-CRP clearly indicated that the same targets ( CRP-1 and CRP-3 ) were occupied both in the presence and in the absence of CytR ( Figure 8 , compare lanes 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR titration in the absence ( left panel ) and presence ( middle and right panels ) of cAMP ± CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Figure 7B shows the cAMP ± CRP footprint at 0.1 mM CRP and 0.1 mM cAMP in the absence ( lane 2 ) or presence of CytR ( 20 nM ) ( lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>( B ) cAMP ( 0.1 mM ) and CRP ( 0.1 mM ) were reacted with pdeoP2 in the presence or absence of CytR ( 15 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The dilutions of the CytR ' extract used are indicated , cAMP , purified CRP protein , and each of the nucleosides cytidine , adenosine , and uridine were added to the reaction mixture to final concentrations of 50 [ LM , 13 p.M , and 10 mM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The concentrations o * cAMP in the binding reactions are indicated above lanes 4-9 , and the amounts of purified CRP and CytR extract used are 0.5 ng and 0.1 ( ig per reaction , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The concentrations of both CytR and CRP ( cAMP ) 1 , whether used as titrant or as held constant , were the independent variables .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The region protected from DNase I by cAMP -- CRP and CytR is indicated to the left of each panel by a thin and thick line respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>rot expression from pNM91 is regulated 5.7-fold by CylR and slrongly activated by cAMP-CRP whereas rot expression from pNlV195 is unaffected by CytR and exhibits a 35 % decrease in activity in a Crp strain ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Of particular interest were mutations that resulted in CytR-dependent constitutive udp expression when intracellular cAMP-CRP concentrations were limiting ( glucose medium ) ( del224 , del364 , dup1 , and dup361 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>b Extent of regulation by CytR protein was estimated as the ratio of activities in cytR1 versus cytR strains ; regulation by cAMP-CRP was estimated as the ratio of activities in crp1 cytR cells grown in minimal medium with glycerol ( fully derepressed udp expression ) versus crp cells grown in the same medium FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>As all strains used in this work are deoR + and the DeoR operator is not perturbed in any of the plasmid constructs , P-galactosidase expression from pJEL134 reflects only cAMP-CRP activation and CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>CytR and cAMP -- CRP regulation of rpoH -- lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>In this second model , protein-protein interactions between CytR and DNA-bound CRP could be a prerequisite for the interaction of CytR with deoCp2 , with CytR forming a bridge between the two DNA-bound cAMP-CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In most cases , these complexes consist of a DNA-bound CytR protein , sandwiched between two DNA-bound cAMP -- CRP complexes [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Hydroxyl-radicalfootprint of cAMP-CRP and CytR bound to nupG.CRP targets are indicated ; final protein concentrations are in pg m1-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>DNase I footprint of deoP2 in thepresence of purified CytR protein , cAMP * CRP , and inducer ( cytidine ) A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>These results imply that a CytR protein bound to C-sequences does not change shape upon addition of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>These experiments clearly demonstrate that E. coli CytR protein bound to heterologous promoters in a highly cooperative fashion in concert with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The DMS footprints show that CytR simultaneously contacts guanines in both half-sites when bound co-operatively with cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The probabilities are given by where DGs is the sum of free energy contributions for configuration s ( Table III ) , and where i and j are the stoichiometries of bound CRP - ( cAMP ) 1 complexes and CytR dimers in configuration s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Crystalline cAMP ( > 99 % pure as 99 % pure as free acid ) were purchase ΔGs is the sum of all free energy contributions for a given configuration , s ( Table 3 ) ; i and j are the stoichiometries of bound CRP ( cAMP ) 1 and CytR in configuration s , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>These results can be explained in a rigorous manner in light of our recent findings concerning regulation of the deoP2 promoter by cAMP-CRP and CytR and by taking the ability of the various CRP proteins to interact at the different CRP targets into consideration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>For example , most of the in vitro studies have been conducted as combinations of plus/minus the various components ( CRP , CytR , cAMP , cytidine , and promoter elements ) with insufficient understanding of the mutual effects of interactions between these molecules to know what concentrations are necessary to achieve particular effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The nupG promoter is activated 50-fold by cAMP-CRP , and repressed fourfold by CytR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The deoP2 promoter is activated 30-fold by cAMP-CRP and repressed 10-fold by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Moreover , this promoter was only weakly activated by cAMP-CRP ( 3-4 fold ) and not regulated by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>cAMP-CRP and CytR bind cooperatively to this promoter and the combined footprint is the sum of the independent footprints .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Hence , cAMP-CRP and CytR bind cooperatively to the tSx-p2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>CytR and cAMP CRP to thdeeoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>RESULTS Binding of cAMP-CRP and CytR to the tsx-p2 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In DMS footprinting , CytR invariably protected the central guanine of each half-operator , whereas cAMP -- CRP , alone and in combination with CytR , protected two guanines in the CRP-target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In the first model , cAMP-CRP still binds to CRP-1 and CytR occupies the sequences on both sides of CRP-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the absence of cAMP -- CRP , the CytR repressor binds two such octamer motifs in inverted or direct repeat arrangement , preferably separated by 2 bp ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>( A ) In the absence of cAMP -- CRP , the CytR repressor preferentially binds octamer repeats in direct or inverted orientation , separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In this report we have addressed the question of how cAMP-CRP and CytR interact at deoP2 by isolating CRP mutants that severely decrease CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In this context , cAMP ± CRP , an activator , is converted through an interaction with CytR to a repressor that interferes with RNP acting at deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In order to gain further insight into the mechanism by which the CytR repressor functions , we define here the interaction of cAMP-CRP and CytR with decP2 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>When both CytR and cAMP-CRP are added , at the same concentrations as in the experiments of lanes 2 and 3 , two new retarded bands appear ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Samples of lane 2 contained CRP ( 30 nM ) and cAMP ( 100p ~ ) sa , mples of lanes 3contained CRP ( 30 nM ) , RNAP ( 250 nM ) and cAMP ( 100 pM ) , and samples of lanes 4contained CRP ( 30 nM ) , cAMP ( 100 pM ) , and CytR ( 26 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>At high CytR concentration , favorable interactions with CytR would increase the apparent affinity of CRP ( cAMP ) 1 for CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>When the combined footprints ( lanes 7 and 8 ) are compared with the individual footprints of cAMP-CRP and CytR ( lanes 3-6 ) it appears that the positive interactions between CytR and cAMP-CRP bound at CRP2 is not fully abolished by the inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>First , pairwise cooperativity between CytR and CRP ( cAMP ) 1 bound to CRP2 yields about a 10-fold increase in effective CytR affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As the pattern of the 50 bp combined footprint corresponds to the sum of the independent footprints , this complex most likely consists of cAMP-CRP bound to CRP2 , and CytR to the CytR target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>At this higher concentration of CRP , addition of CytR almost exclusively results in formation of the slowest moving cAMP-CRP/CytR/deoP2 complex ( lane 3 , curved arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Thus , in the full repression complex cAMP-CRP occupies CRP2 , and CytR a target immediately downstream of CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Most strikingly , CytR can adopt widely different conformations that are equally favored energetically for complex formation with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Negative regulation by CytR depends on both CRP-1 and CRP-2 and is confined to cells containing the cAMP-CRP complex ( Sogaard-Andersen etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Transcription is activated in response to intracellular cAMP levels by CRP , repressed by CytR , and induced by cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Transcription at the various promoters is activated by cAMP-CRP , and repressed by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>A subset of these promoters is activated by the cAMP-CRP complex and repressed by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>1989 ; L. Segaard-Andersen and P. Valentin-Hansen , in preparation ) strongly suggests the involvement of specific interactions between cAMP-CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>These include strict spacing requirements for the interacting sets of proteins , CytR and cAMP -- CRP or cAMP -- CRP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Furthermore , our results reveal that widely different conformations of the CytR repressor can cooperate with cAMP -- CRP to form nucleoprotein complexes of equal stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>We can now consider why CytR requires cAMP -- CRP to-bind DNA efﬁciently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>cAMP-CRP is tentatively showntoiniroduce t * io significant bends in CRP-1 and CRP-2 and CytR is shown not to deform the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Moreover , mutant cdd promoters in which CRP-2 and CRP-3 have been deleted are partially regulated by CytR , and cAMP-CRP and CytR still bind cooperatively to these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>These findings provide clues to an understanding of how cAMP - CRP and CytR interact at a structurally diverse set of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>On the other hand , cAMP-CRP occupies CRP-2 in the absence of CytR in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>bEffector concentrations : CRP , 0.1 μM ( total dimer ) ; cAMP , 150 μM ; CytR , 0.1 μM ( dimer ) ; cytidine , 2 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>We previously showed that deoP1 is regulated neither by CytR nor by cAMP-CRP ( Vaientin-Hansen etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>It is not obvious from the DNase I footprint whether CytR and/or cAMP-CRP binds in the downstream part .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Thus , cAMP -- CRP induces conformational changes in CytR so that the repressor ﬁts the natural targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Subsequently , we examined the DNase I footprint of cAMP ± CRP and CytR bound to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_793">
DMS footprint of cAMP-CRP and CytR bound to nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_793", type="checkbox">
<div class="ACT_similar">
<li>DNase I footprint of cAMP-CRP and CytR bound to nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li></div>
<li>Hence , cAMP-CRP and CytR bind cooperatively to cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR and cAMP-CRP bind co-operatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Purified CRP proteins , CytR extract , and cAMP were added as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Evidence for CytR induced repositioning of cAMP - CRP Several predictions follow from this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Arrows denotes guanines protected from DMS methylation ; ( ← ) DMS protection by cAMP -- CRP ; ( ⇐ ) additional DMS protection by cAMP -- CRP þ CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>cAMP-CRP and CytR targets , as determined in the DNase I footprints , are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>DNase I and DMS footprints of CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>cAMP -- CRP does , however , induce drastic changes in CytR 's quaternary structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>cAMP -- CRP complexes are hatched ; CytR is in black .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>was to A B cAMP CytR CytR CRP 1 80 ¬ 1 1 1 1 00 CytR4 ¬ 1 1 1 ¬ cytidine urid ine cytidine - - + ¬ - + - - + - + adenoslne * + - - + - + ¬ AMML - - - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>( A ) Binding of either CRP ( cAMP ) 1 alone or of CytR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to a cddP derivative with CRP2/3 deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to wt cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Binding of CytR and cAMP -- CRP to spacing variants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>However , when couched in quantitative terms as in Table II , it is evident that this phenomenology reflects the failure of Fig. 5 to account for all configurations of bound CytR and/or cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Addition of puriﬁed CytR to reactions containing cAMP -- CRP gave rise to a single new band with lower mobility ( Fig. 6A -- D , lines 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>These changes in the digestion pattern are detected in nearly all gels ( see Fig. 2 , 3 , 5 , 6 , and 7 ) , irrespective of whether cAMP-CRP or CytR proteins were present in the assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to deoP2 derivatives with point mutations in the CRP sites As mutations in each of the two CRP targets of deoP2 decrease CytR regulation in vivo , we were interested to see whether this effect could be demonstrated at the molecular level in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>At the high CRP ( cAMP ) 1 concentration required to significantly load CRP1 , the net effect of cooperativity and competition produces only a slight difference in CytR occupancy whether deoP2 is wild type or CRP22 ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Results Binding of cAMP-CRP and CytR to wild-type deoP2 In order to test the importance of cAMP-CRP in negative regulation by CytR , we used nuclease protection experiments to probe specific interactions between these proteins and deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Although maximum levels of expression decreased greatly owing to the lack of the speciﬁc NtrC-phosphate-binding sites that form the glnA enhancer , signiﬁcant inhibition by CRP in the presence of exogenous cAMP was still observed : 12-fold for pKU102 compared with 20-fold for pKU101 , and fourfold for pKU100 respectively ( Fig. 4 for TP2006 ; data not shown for TP2339 -- 1 containing pLG339CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The poor reversibility of glucose repression by exogenous cAMP in the wild type and the cya mutant may be explained by assuming that the binding of CRP to DNA interferes with transcription from P4 ( Fig. 4 ) and that the strains used are not very permeable to cAMP ; cAMP at 10 mM could reverse the glucose repression of Plac-directed lacZ expression by only 10 % ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>The osmotic sensitivity of CRP binding to DNA is not mediated by changes in cAMP binding The presence of cAMP is required for repression by wildtype CRP as proP P1 transcription is constitutively expressed in a Dcya crp + mutant ( Fig. 7B ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>This is also suggested by the ® nding that in vitro , even low concentrations of Lrp interfere with cAMP-CRP binding ( Figure 7B ) , which can not be relevant in vivo where cAMP-CRP is clearly the main activator of csiD expression and Lrp seems to play a modulatory , perhaps `` back-up '' role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Although agp expression is subject to catabolite repression in vivo , the upstream region does not contain any consensuslike CRP-binding sequence , and band-shift assays in the presence of cAMP and of purified CRP protein ( up to 200 nM ) showed no detectable binding of the CRP-cAMP complex to this region ( E. Pradel , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2153660" target="_blank">2153660</a>]<br></li>
<li>These results clearly demonstrate that the cAMP-CRP binding site at 271 is indeed involved in the catabolite repression and the positive regulation of gntT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Transcription of the cstA - lacZ fusion containing the native cstA promoter and CRP binding site was dependent on cAMP and CRP , consistent with previously published results demonstrating that cstA expression is subject to catabolite repression ( 42 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>Increasing FIS concentrations relative to cAMP -- CRP prevented binding of cAMP -- CRP at site I , but had no effect on cAMP -- CRP binding at site II , as evidenced from both the protection pattern by cAMP -- CRP and the DNase I-hypersensitive sites ( Figure 5B , lane 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>When the fragment carrying the mutated CRP site II was used , the binding of CRP-cAMP to CRP site II was no longer observed while the binding of CRP-cAMP to the CRP site I was unaffected , indicating that the mutation of site II completely abolished the affinity of CRP-cAMP for CRP site II ( Fig. 7B , lanes 4-6 ) , Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>When the concentration of CRP was increased 10-fold , protection was greater at the cAMP-CRP binding site within the wild-type ara regulatory region ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>In gel shift assays , this mutant fragment had a similar affinity for cAMP ± CRP compared with the wild-type fragment , and binding was cAMP dependent ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>A In the absence of gluco-nate , GntR molecules bind to all of 3 binding sites ( HRI , HRII and HRIII ) and obstruct binding of cAMP-CRP complexes to its binding sites ( cCI and cCII ) , resulting in repression of the gntV and idnD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>In an early study of CRP-cAMP binding to DNA , a fragment with the CRP consensus site was shown to-bind CRP with an increased afﬁnity compared to that of naturally occurring CRP binding sites , even in the absence of high levels of cAMP ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In view of the fact that the operator for PaaX covers only the CRP-binding site I , cAMP-CRP could conceivably bind to site II when PAA is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In the absence of PAA , pga is poorly expressed because the cAMP-CRP complex can not bind to the CRP-binding site I as the latter is occupied by PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In the absence of this cAMP-CRP binding site , the amount of functional araC protein in the cell remains close to wild-type levels because autoregulation of the araC gene by its own protein product dominates cAMP-CRP-mediated regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>CRP is an allosteric protein in which binding of cAMP causes a conformational transition resulting in an enhanced affinity for DNA In 0 Y ¬ Frag - I - Frag .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>In contrast , the binding of cGMP , of which guanine ring possess the 6-keto group replacing the 6-amino group of cAMP , does not induce proper allosteric change and fails to activate the protein , even though it binds to CRP with comparable afﬁnity as cAMP [ 5 -- 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>By introducing CRP proteins that exhibit an altered DNA binding specificity into a strain containing a mutant deoP2 promoter in which cAMP-CRP activation was decreased and CytR regulation completely abolished , we show that CytR regulation of this promoter can be reestablished by restored the DNA binding of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>These findings support a general model for CytR binding and CytR regulation , in which ( i ) CytR and the CRP/cAMP complex bind to similar or Identical targets , ( ii ) two or more targets are necessary for proper binding of CytR to a promoter region , and ( iii ) CytR represses transcription by antagonizing cAMP/CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>By introducing point mutations in the CRP sites , which strongly reduce binding of the cAMP-CRP activator complex , activation of transcription was severely reduced and CytR regulation was completely abolished , even when CytR was overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Our results support the model in which repression normally involves the binding of CytR to cAMP-CRP to form a complex which binds to speciﬁc sites in the udp promoter , without direct interaction between CytR protein and a speciﬁc operator DNA sequence , and in which induction by speciﬁc inducer cytidine involves dissociation of CytR from cAMP-CRP and then RNA polymerase interaction with cAMP-CRP bound to a site upstream of the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>A model for the mode of action of CytR must account for the observations in this study that CytR only inhibits cAMP-CRP-dependent initiation of transcription and that tandem , intact CRP binding sites are required for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Structural ﬂexibility and induced ﬁt While it is not surprising that transcription factors can exhibit promiscuous DNA-binding characteristics , or undergo induced ﬁt when interacting with other molecules , it is remarkable that CytR can assume multiple conformations of about equal stability when co-binding with cAMP -- CRP ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Taking together the results presented in this study , we propose a simple mechanism to account for the repression by CytR : the cAMP ± CRP complex at the CRP1 site , upon interaction with CytR , undergoes allosteric changes such that its binding to the DNA site is improved but its interaction with a-NTD is no longer productive , but rather destructive , leading to transcription repression ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Although no unique estimate for the affinity of CRP ( cAMP ) 2 for binding to either CRP1 or CRP2 was obtainable , the analysis did yield as an upper limit to the operator binding affinity of CRP ( cAMP ) 2 a value 100-fold lower than that of CRP ( cAMP ) 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Cooperativity contrib-utes unequally to CRP1 and CRP2 , dependent on their relative intrinsic affinities for CRP ( cAMP ) 1 binding ( DG1 and DG2 ; Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Estimates of the free energy changes for binding of CRP ( cAMP ) 1 to CRP1 and to CRP2 are in Table I. Affinity for CRP2 is 10-fold higher than for CRP1 , consistent with results reported at other experimental conditions ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These data were analyzed quantitatively using Equation 4 of Heyduk and Lee ( 41 ) to estimate the free energy changes corresponding to mac-roscopic , step-wise association constants for binding of one ( K1 ) and two ( K2 ) cAMPs to free CRP dimers and for binding of CRP ( cAMP ) 1 and CRP ( cAMP ) 2 to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Therefore , the CytR-mediated differences in loading free energy changes for CRP ( cAMP ) 1 binding to CRP1 and CRP2 are summed to yield DG123 5 ( DGCytR CRP CRP1 2 DGCRP1 ) 1 ( DGCytR2 2 DGCRP2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>FIGURE 5 : Comparison of the distributions of CRP ligation states of cddP and udpP calculated using the free energy changes for binding of CRP ( cAMP ) 1 to CRP1 and CRP2 as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>As previously shown , binding of cAMP -- CRP to the wild type udpPEc resulted in the formation of two retarded complexes : the complex with the highest mobility contained cAMP -- CRP bound at CRP2 and the complex with the lowest mobility contained cAMP -- CRP bound at both CRP2 and CRP1 [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>To obtain binding equations for the two reduced-valence operators , CRP1 and CRP2 , configurations in which CRP ( cAMP ) 1 is -- bound to a mutated site were excluded from the summations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>revealed a higher degree of cooperative DNA binding of the two regulators and , as predicted , no rearranged binding of cAMP-CRP in the upstream region 363 in cddP , and that a change between activated and repressed states in cddP involves a switch in the combinations of CRP sites used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CRP * mutants which can actively affect gene expression without binding cAMP were isolated by many groups ( 3 , 10 , 11 , 13-15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>CRP -- cAMP binding after addition of different concentrations of NaCl to cells growing in M9 + glycerol was also determined ( Fig. 4B , C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>However , the CRP binding site on gcv diverges from consensus at many positions ( Fig. 1 ) and it is not clear how CRP can achieve its maximum range of regulation of the gcv operon at the cAMP levels found in GM medium-grown cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A common feature of the ® ve operons thus far identi ® ed as members of the Mlc regulon is that they possess at least one cAMP ± CRP binding site as well as an Mlc binding site , and thus are under dual regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11157755" target="_blank">11157755</a>]<br></li>
<li>) Since cAMP-CRP binding required sequences between positions -103 and -77 , we conclude that the CRP site centered around position -84 is required for spf gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>After addition of 0.3 M NaCl , there was a small decrease in CRP -- cAMP binding , and occupancy of the CRP site became increasingly reduced with increasing NaCl such that at 0.5 M NaCl there was almost a 70 % loss of binding ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>We provide physiological , genetic and bio-chemical data demonstrating that CRP inhibits glnAp2 in a CRP-binding site-independent ( Figs 3 and 4 ) but cAMP-dependent manner ( Figs 3 , 4 , 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Due to the overlap with the activating cAMP-CRP-binding site , phosphorylated ArcA would not only act as a `` classical `` repressor , but also as an antiactivator for cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16291649" target="_blank">16291649</a>]<br></li>
<li>Cyclic AMP binding to the Wrst CRP binding-site caused a substantial conformational change , however , this change was only completed with second cAMP binding site saturation [ 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>These data conﬁrm that cAMP-CRP speciﬁcally recognizes and binds to the CRP binding site located in site 1 , and not site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>As the concentration of CRP-cAMP increases , its binding to the CRP site increases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>Thus , binding of CRP -- cAMP to the galP1D4 site was osmotically sensitive in a transient manner that resembles the proP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In thisreport we further investigate the combinatorial binding of cAMP - CRP andCytR in order to gain an understanding of the interactions involved in the formation of this nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>WT CRP : cAMP complex bound to LCAP with a Ka value of 2.9 # 107 M ¡ 1 , a 74-fold increase in LCAP binding aYnity compared to apo-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Therefore , it could in principle decrease the binding of cAMP-CRP complex in a manner analogous to that of the tsx-502 allele .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Moreover , the absence of cAMP in the assay abolished the binding activity of CRP ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Despite the uncertainty regarding the number of bound cAMP to CRP , we do know that cAMP binding exerts minor eVects on CRP secondary structure and appears to function by causing rigid body movements on CRP structure [ 8 -- 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Non-specific interactions of CRP from E. coli with native and denatured DNAs : control of binding by cAMP and cGMP and by cation concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987134" target="_blank">1987134</a>]<br></li>
<li>Therefore it was necessary to analyze the cAMP concentration dependence of CRP binding to deoP2 to determine concentrations of active CRP ( cAMP ) 1 dimer in our experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Thus , the observed differences in secondary structure between apo-CRP and cAMP -- CRP probably arise from the direct effect of cAMP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>However , as a consequence of overlapping transitions for binding of the first and second cAMPs , only a fraction of CRP is in this functionally active form even at optimal cAMP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However , it would be reasonable to assume that the CRP mutants without cAMP already resemble the active-conformation of cAMP -- CRP , since they are constitutively active regardless of cAMP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Two fundamental questions regarding this process are how the conformational changes are induced and how the signal is transmitted from cAMP binding pocket to aVect a global change in the CRP dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>This group attributed most of the CRP conformational change resulting from cAMP binding in a 1:1 molar ratio .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>This prediction may explain one of the modes by which cAMP binding mediates rigid body motion in CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>CRP requires coupling of cAMP binding with the necessary conformational changes through cAMP interaction with residues 72 and 82 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Despite comprehensive understanding of CRP-family structures , representative one of the complicated issues on the CRP allostery is the cAMP-binding cooperativity and its functional implications .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>CRP -- cAMP binding was found to be more sensitive to osmotic upshift in the synthetic media as compared to LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Free energy changes reported for CRP binding use a standard state of 1 M dimeric CRP ( cAMP ) 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>After addition of 0.6 M NaCl , essentially no CRP -- cAMP binding was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>WT CRP exhibits negative cooperativity in cAMP binding with a ratio of Ka1 -- Ka2 of 1600 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>CRP binding to e2DGs/RT z @CRP ~ cAMP !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Five speciWc ionic or polar interactions occur in cAMP binding pocket of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Concentrations of the binding reaction components were as follows : restriction fragments , 20 nM ; CRP , 1 FtM ; and cAMP , 200 , uM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>At intermediate K + glutamate concentrations , the binding equilibrium of CRP -- cAMP is shifted such that over the course of a 2-min DMS reaction , the guanines become partially methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>For the effector ( + ) reaction of CRP , 10 insert.symbol mmM adenosine 29,39-cyclic monophosphate ( cAMP ) ( SIGMA ) was added during the binding reaction and also subsequent isolation procedure of CRP-DNA complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>This cAMP concentration maximizes the fraction of CRP present as the active DNA-binding species in which cAMP is bound to one but not both of the subunits ( denoted by CRP ( cAMP ) 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Experiments in which CRP was present also contained 150 µM cAMP , a concentration that maximizes the fraction of CRP present as the active DNA-binding species in which half of the cAMP sites are filled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These changes in conformation promote CRP ( cAMP ) 1 complex binding to speciWc sites in DNA and CRP interaction with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Then , based on electrostatics calculations , the authors have proposed that the open conformation moves to closed conformation upon binding to DNA , resulting in the DNA bending observed in the cAMP -- CRP -- DNA complex structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Because of its low DNA binding affinity , repression by CyfR is confined to cells containing cAMP-CRP ( Sogaard-Andersen etat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>cAMP binding is negatively cooperative , thus favoring the singly liganded species , CRP ( cAMP ) 1 , which is the functionally active , site-specific DNA binding form ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However , only the anti-cAMP binding is assumed to be responsible for the global allosteric transition of CRP , since the overall conformations of the DNA-bound CRP structures were similar , regardless of syn-cAMP binding [ 18 -- 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>This is probably owing to the fact that CRP -- cAMP can activate glnAp1 through recruiting and helping Es to compete with 70 NtrC-phosphate for DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>It has been reported that micromolar rather than millimolar concentrations of cAMP often favor a higher afﬁnity for DNA binding by CRP , and that millimolar concentrations of cAMP can even inhibit binding of CRP-cAMP to DNA ( 2 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>We show here that the osmotic regulation of proP P1 correlates with CRP -- cAMP binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>We wondered if the osmotically sensitive DNA binding by CRP -- cAMP could reflect a salt sensitivity of cAMP binding to CRP , which would thereby affect DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>These experiments showed that mutation of the cAMP binding pocket eVects CRP speciWc DNA binding drastically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Surprisingly , these DNA conﬁgurations are not optimal for co-binding with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>cAMP ± CRP binding to deoP2 DNA as probed with DNase I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>This result indicates that pH , regulation of malT may be in part related to either pH , modulation of the cAMP pool or conformational changes in CRP induced by cAMP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank Kevin O'Hare for helping to begin sequencing experiments during the stay of M.L. in his lab as an exchange student under the program of the International Association for the Exchange of Students for Technical Experience , Pradip Parrack and Gauranga Mukhopadhyay for help in planning the cAMP-CRP binding assay , and Michael Yarmolinsky for critical comments on the manuscript This work was supported by the Polish Academy of Sciences within Project 3.13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>Bandshift experiments confirmed that E.coli CRP-S sequences are low affinity binding sites for CRP , and mRNA analysis showed that they require CRP , cAMP ( CRP 's allosteric-effector ) and Sxy for gene induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>It is therefore surprising that CRP -- cAMP binding to specific sites within the regulatory regions of proP , galP1 , pmelRcon ( data not shown ) , and by inference lacP , appears so sensitive to changes in medium osmolarity in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Indeed , if the CRP binding sites are only partially occupied in vivo , their destruction is expected to have greater consequences under growth conditions in which the cAMP concentration is high and which consequently favor the occupancy of the sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>It has been demonstrated that cAMP-CRP binds to many sites with an 8-bp spacer in vitro ( 3 , 4 , 9 , 49 ) , although we have found no reports showing such CRP binding sites are functional in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In one such promoter , deoP2 , CytR binds to the region between the tandem CRP binding sites with a relatively low affinity ; in the presence of cAMP-CRP , however , the repressor and activator bind cooperatively to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In the third model , CytR interacts with the same DNA sequences in the combined footprint as in the independent footprint , and cAMP-CRP , rather than binding to CRP-1 , binds to the region located immediately downstream of the CytR-binding sequence ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>If the unusually high affinity for CRP imparts both high sensitivity to cAMP concentration and also contributes to high levels of CRP-mediated activation , the high affinity for CytR binding to CytO counteracts this with high sensitivity repression , thus making cddP both the first to turn on and the first to shut off of the CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Furthermore , we propose that different combinations of these CRP sites are used during activation and repression : activation of cddP involves CRP - 1 and CRP-2 whereas cAMP-CRP binding in the presence of CytR involves CRP-1 and CRP-3 , i.e. the cooperative binding of cAMP-CRP and CytR induces a 2 bp repositioning of from to cAMP-CRP CRP-2 CRP-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR was in .100 - fold excess to the binding site ( cAMP -- CRP absent ) ; in the presence of cAMP -- CRP , both proteins were in at least 5-fold excess to the binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Consistently , it was noted that cAMP ± CRP binding was greatly enhanced in the presence of CytR , especially at the CRP1 site , suggesting a protein ± protein interaction between two proteins ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>We have taken this approach one step further , to deﬁne the optimal binding site of the CytR repressor , alone as well as in the presence of its helper protein , cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>B -- D , CytR and CRP ( cAMP ) 1 binding together , i.e. the concentration of one protein is varied in presence of a fixed concentration of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>DG123 can also be evaluated by comparing CRP ( cAMP ) 1 binding in the presence versus the absence of saturating CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As there appear to be more than three CytR binding sites , the saturating CytR stoichiometry , whether in the presence or absence of CRP ( cAMP ) 1 , is n't obvious .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>TABLE I Loading free energy changes for binding of CRP and CytR to the udp regulatory region Free energy changes for saturation of udp operators with either CRP ( cAMP ) 1 or CytR in the presence or absence of effector ligands are as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To evaluate the cooperative free energy change in this manner , loading free energy changes were determined for binding of CytR and CRP ( cAMP ) 1 to deoP2 , each in the presence and absence of a fixed concentration of the other ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR binding to wild type , [ squlf ] ) , CRP12 ( [ trif ] ) , and CRP22 ( [ itrif ] ) udp promotere in the presence of saturating CRP ( cAMP ) ( 64 nM ) and 1 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR binding to CytO in wild-type ( 9 ) , CRP1 ( 2 ) , andCRP2 ( 1 ) cddP in the presence of saturating CRP ( cAMP ) 1 ( 64 nM ) - and cytidine ( 2 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Binding of cAMP-CRP to the mutated CRP-2 site is strongly reduced ; however , in the presence of CytR , binding of cAMP-CRP is reestablished ( Fig. 7B , lanes 10 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Fig. 7 demonstrates how the Specific Additional Sites model rationalizes both the apparent high cooperativity of CytR binding in the presence of CRP ( cAMP ) 1 at fixed , saturating concentration and apparent weak cooperativity of CRP ( cAMP ) 1 binding in the presence of CytR at fixed , saturating concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Finally , in the combined footprint , cAMP-CRP binding to CRP-2 is enhanced , and in the presence of cAMP-CRP , binding of CytR is enhanced ( Fig. 3A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The stimulation of udp expression by CytR in certain mutants may reﬂect its stabilization of cAMP-CRP binding to target DNA and illustrates that only modest evolutionary changes could allow particular multiprotein complexes to serve as either repressors or transcriptional-activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>DG is the sum of free energy contributions for configuration s ( Table III ) ; i and j are the stoichiometries of bound CRP ( cAMP ) 1 complexes and CytR dimers in configuration s. Summation of the relative probabilities for all configurations in which protein is bound to any given site derives the binding equation for that site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The combined footprint is the sum of the individual cAMP-CRP and CytR footprints , suggesting that tbis footprint is obtained by binding of two oAMP-CRP molecules and the CytR protein bound in belween .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The equation for binding of a protein , CRP ( cAMP ) 1 or CytR , to an individual site is obtained by summing fs over all configurations in which that protein is bound to that individual site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The CytR repressor inhibits expression of the deoP2 promoter by binding cooperatively to deoP2 together with cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Isolation of the optimal CytR-binding site The course of the selection for CytR operators ( in the absence of cAMP -- CRP ) was evaluated by mobility shift assays ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The cAMP ± CRP binding site is shown as a shaded box and CytR as an open box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The intensities of the bands containing CytR ( lanes 2 and 8 ) show that cAMP-CRP stimulates binding of CytR more than 30-fold in the mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The loading free energy change for CytR binding to deoP2 that is saturated by CRP ( cAMP ) 1 ( DGCRP CytR ) includes contributions from both intrinsic binding and cooperativity , i.e. DG3 1 DG123 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The cooperative binding of CytR and cAMP -- CRP to this sequence is very inefﬁcient , even though the independent CytR afﬁnity is similar to that of other C-sequences ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The CytR half-operator consensus for cooperative binding with cAMP -- CRP is slightly different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Here we demonstrate the presence of two binding sites for the cAMP-CRP complex , CRP-1 and CRP-2 , and show that cAMP-CRP and CytR bind cooperatively to tSx-p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In this report , we have tested this model in vivo by assessing the relationship between binding of the cAMP-CRP complex to the two CRP binding sites and the extent of CytR regulation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>It should be emphasized that this analysiswas not designedto provide quantitative estimations of promoter activity Discussion Here we have analysed in vivo how CytR regulation is related to-binding of the cAMP-CRP complex to the two CRP sites in deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Moreover , the results show that ( i ) , the co-operative binding of cAMP-CRP and CytR to nupG stimulates binding of CytR drastically , by a factor of more than 3000 ; ( ii ) , CytR triggers a repositioning of the cAMP-CRP complex in the - 3 5 region ; and ( iii ) , the inducer of CytR , cytidine , does not completely prevent the co-operative formation of an intermediate repression complex consisting of CytR and one cAMP-CRP molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>These results indicate that strong CytR-udpP interaction depends on binding by the cAMP-CRP complex and , conversely , that CytR itself enhances CRP binding to the udpP region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Therefore , the reduced repression by CytR observed for the tsx-503 allele is likely to be a consequence of reduced binding of the cAMP-CRP complex to the CRP-2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Hence , binding of CytR to deoP2 is strongly dependent on proper binding of the cAMP-CRP complex to both CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>While the cAMP-CRP complex is able to-bind independently to the tandem targets in deoP2 , no detect-able binding of CytR could be observed in the absence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Binding of CytR to its target excludes binding of cAMP-CRP to CRP1 , and promotes binding of cAMP-CRP to CRP0 , therefore allowing co-operative binding of the two protein species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Here we show that CytR binding to the deoP2 promoter relies on interaction with the master gene regulatory protein , CRP , and , furthermore , that cAMP-CRP and CytR bind co-opera-tively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Here evidence is presented to show that rot is expressed from four promoters , three of whioh are activated by cAMP-CRP and repressed by the co-operative binding of CytR and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>We could not ﬁnd A-sequences for which it would make sense to test for cooperative binding , since most of the A-fragments contain mutations in the CRP targets , or because the binding sites are too close to allow simultan-eous binding of cAMP -- CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Taken together , the observation that neither of the CRP sites by itself titrates CytR , and the lack of co-operativity in CytR interaction with the two CRP sites , seem to ruie out the possibility that CytR can compete with cAMP-CRP for binding to the two CRP sites , as hypothesized above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>For the cooperative binding with cAMP -- CRP , the corresponding CytR concentrations were 7310 -- 10 M and 2310 -- 11 M. CRP and cAMP were employed at concentrations of 2310 -- 7 M and 50 µM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Strong synergy between CytR and CRP in binding to deoP2 is particularly obvious in experiments where the cAMP-CRP comlex is present at low concentrations ( Fig. 2 and Fig. 4 , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>( ii ) CytR could interact with deoCp2 without precluding cAMP-CRP binding to the DNA and antagonize the activating effect of cAMP-CRP either by inducing a conformational change in the DNA or by directly contacting CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>17808 DNA Binding of the C tion pattern of CytR and cAMP.CRP with deoP2 , strongly indicate that the cooperative binding of CytR and CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Here , in vitro selection experiments have been used to examine the DNA-binding characteristics of CytR , by itself and when co-binding with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Rather , cAMP -- CRP stabilizes alternative DNA-binding modes of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Furthermore , the inducer ( cytidine ) does not affect independ-ent DNA binding of CytR , rather the CytR/cAMP * CRP cooperativity is perturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to deoP2 deletion derivatives Next we probed cAMP-ORP and CytR binding to two deoP2 derivatives in which CytR regulation in vivo is abolished ( Sogaard-Andersen ef ai , 1990b ; our unpu-blished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Hence , CytR regulation of deoP2 can be modulated by simply varying DNA binding of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Second , the efficiency of CytR regulation of deoP2 can be varied by changing the D N A binding strength of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Individual site binding of CytR and CRP ( cAMP ) 1 to the udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Many different CytR-binding site conﬁgurations seem to allow cooperative interaction of CytR with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>FIGURE 3 : Individual site isotherms for binding of CytR and CRP ( cAMP ) 1 to cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Co-operative binding of cAMP-CRP and CytR Next we tested binding of CytR and cAMP-CRP to a 160 bp fragment containing nupG sequences from - 1 1 8 to - 1 0 in the DNase I footprinting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Furthermore , the reduction in CytR titration parallels the reduction in binding of cAMP-CRP to the mutated CRP sites in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Specifically , we demonstrate that point mutations reducing cAMP-CRP binding result in drastic reductions in CytR interaction with deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>This approach revealed that CytR uses different settings of half-operators when binding independently and in concert with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>A , CytR and CRP ( cAMP ) 1 binding alone , each in the absence of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Obviously , binding of CytR is strongly stimulated by cAMP-CRP ( compare lanes 5 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Independent binding of CytR and cAMP-CRP to the mutant tSX-p2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>( B - - - D ) Simultaneous binding of CRP ( cAMP ) 1 and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Combined binding of cAMP-CRP and CytR to the mutant tsx-p .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>So tt is not enough to have two DNA-bound cAMP-CRP complexes to obtain simultaneous binding of CRP and CytR ; the structure of the spacer region or the spatial orientation ot the tandem CRP targets is also critical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Taken together , the CytR ( 20 nM ) binding signi ® cantly enhanced the cAMP ± CRP binding , particularly at the downstream site , suggesting that a protein ± protein interaction between two proteins resulted in increased cAMP ± CRP binding to the DNA sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>First , the effect of the mutations in the CRP sites on CytR regulation is an indirect effect mediated through reduced binding of the cAMP-CRP complex , rather than a direct effect on CytR-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>This implies that it is the conﬁguration of the CytR half-sites and the position of these half-sites relative to the ﬂanking CRP targets , and not the strength of the CytR -- DNA interaction , that are important for cooperative binding of CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>A few simple possibilities are likely : binding of CytR to A-sites may deform the DNA in a way that impedes wrapping the DNA helix around the proteins in the CRP2 -- CytR -- DNA complex , or it may induce DNA conformational changes in the ﬂanking CRP targets , unfavorable for binding of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>( i ) CytR and cAMP-CRP compete for the same DNA sequences , with CytR binding cooperatively to the tandem CRP sites , i.e. , a model which would resemble the cI/Cro system of bacteriophage lambda , in which the cI and Cro proteins bind the DNA and each excludes to same sequences the binding of the other ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Circular dichroism experiments Results A semi-log plot of cyclic AMP titration of CRP : ANS complexes produced biphasic cAMP binding isotherms ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Since the genes for - D-glucoside utilization were predicted to be expressed in the absence of glucose and since the AscG box sequence is similar to the CRP box sequence ( Fig. 4 ) , we analyzed binding in vitro of cyclic AMP ( cAMP ) - CRP to the ascF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>With Ea38 , transcription from the P1 promoter was only slightly activated , even in the presence of CRP and cAMP , although P2 transcription was repressed by CRP binding , just as in the case of Ea70 ( cAMP-CRP functions as a repressor for lac transcription by Ea38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>The affinity of the cAMP-CRP complex for the two CRP targets of tsx-500 , tsx-501 , and tsx-502 alleles seems to be very similar to that of the wild-type promoter , as determined from binding experiments in which the CRP concentration was varied from 50 nM to 13 pRM ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>g006 Transcription from the P1 promoter is likely regulated by DNA binding of CRP-cAMP The P2 transcription dominance during exponential-growth in strains lacking CRP or cAMP ( Fig. 3 ) suggested that the CRP-cAMP complex regulates P1 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>CRP complex ( 18 ) , and the specific binding of cAMP - CRP to the gal promoter region has been demonstrated by DNase protection experiments ( 27 ; Majumdar and Adhya , in preparation ; see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>The initial increase of both CRP and cAMP suggests that increased binding of the CRP-cAMP complex to the gal operon promoter region caused the initial increase in P1 transcripts during the INCREASE period ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Speciﬁc binding site ( s ) for the CRP -- cAMP complex located upstream of these promo ¬ glnAp2 expression is affected by different carbon sources and that the CRP -- cAMP complex inhibits the glnAp2 promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Activation occurs through binding of cAMP -- CRP to the CRP II site located upstream of the crp promoter ( Hanamura and Aiba , 1992 ) , while repression depends on the CRP I site located downstream of the crp promoter ( Aiba , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>In IG this model the transcriptional-repressor , PaaX , and the transcriptional-activator , the cAMP-CRP complex , compete for binding to the upstream region of the pga promoter that overlaps with the CRP-binding site I. Details are given in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Speciﬁcally , cAMP-CRP is shown to-bind to a CRP binding site located in the upstream region of the lsr promoter and works with the LsrR repressor to regulate AI-2 uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Indeed , at low FIS concentrations , binding of cAMP -- CRP to site I occurs simultaneously with binding of FIS to its high afﬁnity site centered at 168 ( Figure 5 ) which overlaps the -- 10 region of the putative crp2 promoter and thus may occlude binding of RNAP at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>At low K + glutamate concentrations , CRP -- cAMP is tightly bound to the promoter effectively preventing RNA polymerase binding or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>We were unable to directly measure binding of CRP -- cAMP to the lac promoter site by in vivo DMS footprinting ( e.g. , Fig. 5 ) , probably because of its relatively weak af-finity to the nonconsensus lacP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Binding of Mlc and CRP to the pts regulatory region We performed electrophoretic mobility-shift assay ( EMSA ) by using the 311 bp HindIII-EcoRI fragment III to examine the binding of Mlc and CRP-cAMP to the pts promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>Therefore we suggest that the cooperative interaction between RNAP and the cAMP -- CRP complex could explain the improvement in the binding of both proteins to the Pr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Under low-glu-cose and LP conditions , where both proteins are abundant , PhoB and cAMP-CRP would compete for binding to the psiE promoter region , which contains overlapping sequences recognized by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>However , the CRP-promoter complexes were not detected in the absence of cAMP , indicating that the CRP binding to these ribosome-rerated genes is speciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>We could not detect speciﬁc binding of puriﬁed CRP to the ast promoter by DNase I footprinting with or without cAMP ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>To test CRP binding to the mcaS promoter , a fragment of the promoter , radiolabeled on either the top or bottom strand , was incubated with increasing amounts of purified CRP with cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>Mutually exclusive binding of PaaX and cAMP-CRP to the pga promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Results show the combined effect of cAMP -- CRP and GalR ( A ) or GalS ( B ) binding on promoter activities ( right lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In vitro CRP binding to the Pr promoter To confirm the direct binding of the cAMP -- CRP complex in vitro and to localize its operator at the Pr promoter , DNase I footprint experiments were performed using the Pa -- Pr fragment as a DNA probe that covers the entire DNA intergenic region between the mhpR and mhpABCDFE genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>In fact , the response to cAMP concentration was experimentally demonstrated , and putative CRP binding sites were in silico predicted in patZp , revealing that this promoter is regulated by CRP ( Figs 5 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>( B ) In contrast , after the addition of glucose the cAMP levels are reduced , preventing the cAMP-CRP complex from binding to the hfq promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>In E. coli , the cAMP-CRP complex stimulates the expression of both lsrR and the lsr operon by binding to the promoter regions , while LsrR represses their expression [ 14 , 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>In the following experiment , we wished to determine whether CRP -- cAMP could function as an osmotically-sensitive repressor in vitro , which would provide evidence that CRP -- cAMP binding directly mediates the osmotic control of the P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In vivo as well as in vitro , the binding of CRP-cAMP to the P1 promoter inhibits transcription at a step leading to open-complex formation , as judged by the decreased hypersensitivity of bases within the P1 promoter to potassium permanganate attack when CRP is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>In vivo binding of CRP -- cAMP to the proP P1 promoter is osmotically sensitive For CRP -- cAMP to function as an osmoregulator of proP transcription in vivo , its binding to the P1 promoter would be expected to be osmotically sensitive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Nucleotides identical to the CRP binding site consensus are indicated by vertical lines In vitro analysis of binding sites for cAMP-CRP and Lrp in the csiD promoter region In gel retardation assays , puri ® ed CRP protein ( in the presence of cAMP ) was found to reduce mobility of a digoxigenin-labeled PCR fragment containing 475 nucleotides upstream and 111 nucleotides downstream of the csiD transcriptional start site ( data not shown ) , suggesting that CRP regulation of csiD is probably due to direct binding of CRP in the regulatory region of csiD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We believe that in the presence of glucose the level of CRP-cAMP complex is not sufficient to occupy the CRP binding site II , especially as we have previously shown that the CRP binding site II has a lower affinity for CRP-cAMP than the downstream CRP binding site I ( Hanamura and Aiba , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>This protection was observed only in the presence of cAMP , again indicating that CRP binding on these promoters is speciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>CRP functions in the presence of cAMP by binding to speciWc sites located in the control region of CRP-depen-dent promoters [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>CRP was found to-bind , under in vitro conditions in the presence of cAMP , to the binding site at 271 ( Fig. 6B and 8 ) but not to the putative site at 213 , which was postulated to be a hybrid of GntR and CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>A region of protection from position 253 to 272 , which encompasses the putative CRP binding site , was caused by CRP in the presence of cAMP and RNA polymerase ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>As an additional negative control , biotinlabeled gadA promoter ( 8 ng , 294 bp , consisting of 285 bp upstream and 9 bp 5 of gadA ) lacking a CRP binding site was also ampliﬁed by PCR ( primers 5 - GAT GTG GAT GAT ATC GTA-3 and 5 - CTG GTC CAT TTC GAA CTC-3 ) and incubated with in vitro-synthesized CRP ( 1 l ) in the presence of 1 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>Es70 did not show any evidence of unwinding the csiD transcriptional initiation region , indicating that its binding to the csiD promoter remains unproductive , even in the presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>While in the absence of cAMP-CRP , EsS is superior in binding to the csiD promoter ( and its binding is not further improved by the presence of cAMP-CRP ) , cAMP-CRP stimulates the relatively weak binding of Es70 ( Figure 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>It is possible that termination after sdhB may be overcome in the presence of cAMP , allowing the suc operon to be expressed from an sdh promoter , and this may explain the absence of a CRP binding site in the vicinity of the suc promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3897791" target="_blank">3897791</a>]<br></li>
<li>In the presence of appropriate concentrations of CRP-cAMP -LCB- B ) , the complex preferentially binds to CRP site I and this binding allows RNA polymerase to-bind at the divergent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>This promoter would overlap with the cAMP-CRP binding site , and open complex formation in vitro was indeed completely abolished in the presence of cAMP-CRP ( Figure 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Hence , due to the presence of a CRP consensus binding site at a functional distance from the gapB P0 start site , we had to test for the involvement of the CRP-cAMP complex on gapB P0 activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>Despite the presence of a perfect consensus CRP binding site in the galP cis-regulatory region , the PgalP promoter activity reached only about 50 % of the activity of the PmglB , PgalS and P1galE promoters in the presence of cAMP -- CRP in our assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>However , the activity of the PgalR promoter remained unchanged in the presence of the cAMP -- CRP complex ( Fig. 2 ) , showing that the putative cAMP -- CRP binding site in the PgalR promoter region ( Weickert and Adhya , 1993b ) is not functional in the puriﬁed system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The far UV CD spectrum for Table 1 Microscopic binding constants of WT CRP , R82Q , E72D and E72Q mutants CRP proteins was measured in the presence and absence of cAMP ligand at room temperature with an OLIS spectrometer in the wavelength range of 180 -- 260 nm [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>The binding is completely cAMP dependent as shown by the fact that no change in the DNasel digestion pattern is seen : either in the presence of cAMP alone ( lane 2 ) or CRP alone ( lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The shaded box indicates the lacZ region derived from pMC1403 Determination of cAMP levels by a gel mobility shift assay We developed a simple method for cAMP determination based on the ability of CRP to form a stable complex , in the presence of cAMP , with a DNA fragment containing the consensus CRP-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>At 100 nM CRP dimer , all of the wild-type DNA fragment was shifted in the presence and absence of cAMP , probably the result of nonspeciﬁc binding by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The mutant forms of CRP , in the presence of cAMP , exhibited DNA binding aYnity similar to that of the apo-form of WT CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In the presence of cAMP WT CRP binding to DNA increase approximately 100-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Such mutations ( namely CRP − phenotype ) , isolated in the β4 -- β5 ﬂap region , can be represented by the K52N mutation and the deletion of E54 -- G56 segment , which commonly decrease the DNA-binding afﬁnity of CRP in the presence of cAMP [ 55 -- 57 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Furthermore , this strong EsS preference in combination with a requirement for high concentrations of the essential activator cAMP-CRP University of Konstanz P.O. Box 5560 , 78434 Konsta Germany 2 UniteÂ de Physicochimie des MacromoleÂcules Biologiques UMR 321 , Institut Pasteur 25 rue du Dr. Roux 75724 Paris , Cedex , France z n * Corresponding author Present addresses : C. Marschall , Department of Dermatology , Ludwigs-Maximilians-University , 80337 Mu Germany ; D. Weichart , Institute of Biological Sciences , University of Wales , Aberystwyth , Dyfed SY23 3DA , United Kingdom .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The relative extents of inhibition and activation of crp transcription by CRP-cAMP are determined primarily by the concentrations of CRP and cAMP , The data which led us to these conclusions can be summarized as follows , ( i ) The activation of crp transcription was observed with increasing concentrations of CRP and/or cAMP when a DNA fragment containing the crp promoterregulatory region was transcribed in vitro in certain conditions , -LCB- ii ) The activation of crp transcription by CRP-cAMP was demonstrated more clearly when the fragment lacking downstream CRP site I was used for in vitro transcription , -LCB- iii -RCB- The concentrations needed for activation of the crp promoter were significantly higher than those required for activation of the divergent promoter , ( iv ) CRP-cAMP was shown to-bind to the CRP site II with a lower affinity compared with the CRP site I by a DNase I footprinting assay , -LCB- v ) The mutation in the CRP site II was detrimental for the binding of CRP-cAMP and for the activation of crp by CRP-cAMP , -LCB- vi ) The positive and negative autoreguiation of crp was shown , by SI analysis , to be operating in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Furthermore , we found that the inhibition of crp transcription by CRP-cAMP was relieved with increasing concentrations of CRP or cAMP when DNA fragments containing the upstream region of the M/ul site were used ( Fig , 1 ) , Figure 2 shows an example of this observation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>When CRP site II was mutated , the transcription of crp was strongly inhibited by increasing concentrations of cAMP and the activation was no longer observed ( Fig , 8A , lanes 6-9 ) , The analysis of divergent RNA showed that the mutation in CRP site II did not affect the activation of divergent RNA ( Fig. 8B , lanes 6-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>0002-8080 G 2005 SGM Printed in Great Britain Silencing involves two sites ( Dole et al. , 2004b ) : H-NS represses transcription initiation at the cAMP receptor protein ( CRP ) - dependent promoter by binding to an AT-rich and presumably bent upstream silencer sequence ( Schnetz , 1995 ; Singh et al. , 1995 ; Schnetz & Wang , 1996 ; Mukerji & Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Previously , we reported that cAMP receptor protein ( CRP ) activates acs transcription by binding tandem DNA sites located upstream of the major acsP2 promoter and that the nucleoid protein IHF ( integration host factor ) binds three speciﬁc sites located just upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>This effect of His-TcbC on tdc transcription was unexpected , and it was considered that the presence of the protein may have prevented cAMP ± CRP binding to the DNA , inactivated RNAP or compacted the DNA such that it impeded CRP-dependent activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The binding of the CRP * protein to DNA allows the transcription of CRP-dependent genes when the cAMP concentration is very low ( Botsford and Harman , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>The factor for inversion stimulation ( FIS ) regulates transcription by binding to highly degenerate 15 bp DNA Abbreviations : CRP , cAMP receptor protein ; EMSA , electromobility shift assay ; FIS , factor for inversion stimulation ; GRDC , growth rate-dependent control ; RNAP , RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>However , despite the presence of putative DNA binding sites for the cAMP receptor protein ( CRP ) ( 472 ) , it appears that cAMP-CRP does no control sdh-suc transcription directly ( 339 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>cAMP-regulatory protein ( CRP ) Many cellular signalling pathways operate through the production of cAMP , which can regulate DNA transcription by binding to cAMP-regulatory protein ( CRP ; also known as catabolite gene activator protein , CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li>
<li>Often the binding of the cAMP-CRP complex to specific sequences in the DNA results in the stimulation of transcription ; however , the expression of several genes has been shown to be negatively regulated , apparently by the inhibition of transcription initiation ( 1 , 2 , 12 , 17 , 23 , 27 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>When bound to its allosteric-effector , cAMP , the CRP homodimer binds spe - ciﬁc DNA sites near target promoters , enhancing the binding of RNA polymerase holoenzyme ( RNAP ) and facilitating the initiation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>We propose that this reduction happens by the following chain of events : poor carbon source -- high cAMP cAMP-CRP binding at -84 - * displacement of factor binding at -80 / -68 - > decrease in spf transcription - * lower spot 42 RNA -- lower DNA polymerase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>GalR and cAMP -- CRP can bind simultaneously to their cognate binding sites ( Adhya , 1989 ; Dalma-Weiszhausz and Brenowitz , 1992 ; Goodrich and McClure , 1992 ) , cancelling each other 's effect , and allowing transcription of both promoters to levels similar to their intrinsic activities ( Lewis , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>To determine whether cAMP binding was important for the osmotic regulation of proP P1 transcription , a CRP * mutant that could bind DNA in the absence of cAMP was assayed for osmotic control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Because CRP-cAMP bound at 241 enhances transcription of P2 to a small extent ( 48 ) , levels of LacZ activity in the absence of CRP binding are actually decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Our analysis indicates that the increase in P1 transcription is the result of cAMP-CRP binding to increasing numbers of galactose operons in the cell population .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Similar osmotic regulation of proP P1 transcription by the CRP * mutant implies that binding of cAMP is not responsible for the unusual osmotic sensitivity of CRP activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>However , Fis and CRP-cAMP together activate transcription synergistically .29 Even though the CRP-binding site is located unusually far upstream from the transcriptional start site , co-acti-vation is dependent on both the correct helical phasing of the CRP site and activation region 1 ( AR1 ) on CRP , a patch shown to contact the aCTD at other promoters .30 The presence of CRP promotes the binding of an aCTD peptide on its pro-moter-proximal side ( nucleotides ÿ97 to ÿ105 ) , providing further evidence that CRP activates proP P2 transcription through interactions with the aCTD .29 Here , we further investigate the nature of the Fis-aCTD interaction by identifying the residues on aCTD that are important for Fis-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The P2 promoter is completely dependent on Es38 , while the P1 promoter is transcribed strongly by Es70 and weakly by Es38 .6 CRP-cAMP , as well as Fis bound at site II , inhibits P1 transcription .6,28,66 Fis binding to site I and CRP binding at ÿ121 .5 co-activate transcription at P2 .29 Results Fis-dependent activation of proP P2 by aCTD mutants in vivo We have shown that the aCTD is necessary for activated transcription of the proP P2 promoter by the Fis protein .24 To identify the region on the aCTD required for activation by Fis , a collection of a subunit mutants containing single alanine substitutions at each position in the C-terminal domain ( residues 255 to 329 ) 31 ± 33 was assayed in vivo for the ability to promote proP P2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>This suggests that , when located at 243.5 , it is the misalignment of the CRP binding site that results in lower , suboptimal transcription in the presence of glucose , perhaps by increasing the apparent threshold of cAMP required to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Complete repression of the upstream initiation site by binding of FIS to secondary sites would be prevented by binding of cAMP -- CRP at site II , which relieves repression by FIS and activates crp1 transcription ( Hanamura and Aiba , 1992 ; this study ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>If the cAMP-CRP complex had bound to the CRP site to activate araBAD in mutants containing either the 3 - or the 6-base-pair substitution , then the binding was below the level necessary to activate araC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>In such conditions , the transcription of crp may be moderately repressed by the binding of CRP-cAMP to the downstream CRP binding site I. Upon depletion of glucose , the cAMP level would increase resulting in the increased level of CRP-cAMP complex to allow the occupation of CRP-cAMP at the CRP binding site II , and the transcription of crp is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>In other words the binding itself of CRP-cAMP to the CRP site is not sufficient to inhibit the crp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>In this connection we have observed that the binding of CRP-cAMP to the second CRP site activates transcription in certain conditions ( manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>Although a potential CRP binding site , 5 - TGGGAagaaagagtTCAGA-3 , was present ( spacer shown in lowercase letters ) , a gel mobility shift assay showed no binding of the cAMP-CRP complex to the luxS promoter region ( data not shown ) , suggesting that th effect of CRP and cAMP on luxS transcription was probably indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>These results demonstrate that cAMP-CRP stimulates transcription of the lsrACDBFG operon by directly binding to the upstream region of the lsr promoter and that mutation in the CRP binding site abolishes this stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>CRP-cAMP binding at the CRP site activates tranhe primers used to prepare DNA probes scription from the P1 promoter , and inhibits transcription from for Northern blot analysis are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18374359" target="_blank">18374359</a>]<br></li>
<li>When PAA is supplied it enters the nucleoplasm and is converted to PAA-CoA by PaaK , and this then inhibits PaaX from binding to the pga operator by blocking the physical interaction between PaaX monomers .2 The cAMP-CRP complex is thus now able to-bind to the unoccupied CRP-binding site I after binding to the higher affinity CRP-binding site II , and then fully activates pga transcription in the manner typical of class III CRP-dependent promoter ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>( b ) The consensus nucleotide sequence for the cAMP -- CRP binding site ( Busby & Ebright , 1999 ) is compared to the Pr promoter sequence from nt '' 51 to '' 30 and the known cAMP -- CRP binding sites at pgalP1 and pmelR , which are located from '' 52 to '' 31 with respect to its transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>With pHAlOm , which carries point mutations ( Figure 7B ) in the CRP binding site II that prevent the transcriptional activation by CRP-cAMP ( Hanamura and Aiba , 1992 ) , the CRP level was clearly reduced relative to pHAlOw in the absence of glucose ( Figure 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Mutations in the upstream CRP binding site in the crp promoter region abolish the effect of glucose One simple explanation for the requirement for the crp promoter and CRP-cAMP for the effect of glucose on crp expression would be to propose that the activation of crp transcription by CRP-cAMP is reduced in the presence of glucose while the positive autoregulation of crp transcription is efficiently operating in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>If this were the case , the disruption of the CRP binding site II , which is located upstream of the crp promoter and responsible for the activation of crp transcription by CRP-cAMP ( Hanamura and Aiba , 1992 ) , should lower the expression of crp in the absence of glucose and eliminate the down-regulation of crp by glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Together these results demonstrated that the cAMP-CRP complex activates the transcription of gntT by binding to the 271 binding site and that full activity of the gntT promoter is dependent on this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>A promoter which carries an altered CRP-binding site II is no ionger activated by CRP-cAMP , indicating that CRP site II mediates the activation of crp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Furthermore , in the promoter , the putative cAMP -- CRP binding site is located at a proper distance for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>We have found that CRP-cAMP can activate transcription of fhe crp gene both in vitro and in vivo in certain conditions and this activation is caused by the binding of CRP-cAMP to CRP site ii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>This demonstrates that , on a linear template , the location of the cAMP ± CRP binding site does not permit transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>This diversity in the mechanism of transcriptional repression by PaaX is noteworthy because it acts as an upstream repressor and functions by displacing the activator , cAMP-CRP , in the pga promoter , unlike most bacterial repressors that inhibit either RNA polym-erase , open-complex formation , or transcriptional activation by binding close to the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Exogenous cAMP relieved catabolite repression of glucuronate-dependent induction of gntP transcription caused by glucose , presumably by allowing formation of cAMP-CRP complex and activation by binding to the CRP site ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Unexpectedly , while GalS inhibited cAMP -- CRP-mediated P1galE activation , allowing low-level transcription of P1galE , it did not prevent P2galE repression caused by binding of the cAMP -- CRP complex in our assays ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>This is a weak CRP activation site since the activation of both promoters caused by binding of CRP -- cAMP complex is only two-fold w2x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>This location is not compatible with a transcription activation mechanism by a cAMP-CRP complex , since the absence of specific CRP binding to this sequence renders its physiological activity unlikely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>Autoregulation of sxy transcription depends on CRP , cAMP and a CRP-S site As Sxy itself is not sufficient to induce expression from Sxy-dependent promoters but also requires CRP and cAMP for transcriptional activation ( Cameron and Redﬁeld , 2006 ) , induction of the sxy : : lacZ fusions by Sxy overexpression was re-examined in crp and cyaA strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>The major finding is that transcription of crp is not only negatively but also positively regulated by CRP-cAMP , and upstream CRP site II is responsible for this positive autoreguiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The data also suggest that transcription from this start site is positively regulated by PhoB and negatively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>to the proposed model , the cAMP-CRP complex binds to this single site in the ara regulatory region to activate transcription of both the araC gene and araBAD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Based on in vitro studies of the cloned crp gene , we found previously that the transcription of crp is negatively regulated by CRP-cAMP that binds to a specific site -LCB- CRP site I ) located about 40 bp downstream from the transcription start site -LCB- Aiba , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Discussion It is well established that transcription of crp is negatively autoreguiated by the CRP-cAMP that binds to downstream CRP site I ( Aiba , 1983 ; Okamoto and Freundlich , 1986 ; Hanamura and Aiba , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>In vitro transcription studies effectively demonstrated this , whereby no CRP-dependent transcription activation was observed on a linear template ; however , use of a supercoiled template permitted transcription activation to occur , presumably through providing sufficient helical twist in the DNA to align cAMP ± CRP and RNAP correctly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>When bound to this site , the cAMP ± CRP dimer will be slightly misaligned with respect to RNAP and , consequently , it would be expected that transcription activation does not occur unless both molecules are realigned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>With CRP at 0.1 mM , the cAMP ± CRP footprint was barely detectable only at the upstream site ( centered at ± 93 ) with a hyper-reactive T ( ± 99 , vertical arrows ) , even though the activation by cAMP ± CRP , as determined by transcription assay ( see Figure 2 ) , was maximal at this concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Taken together , these results confirm the existence of three functional CRP sites 364 We have distinguished two functional regions within cddP that are involved in cAMP-CRP activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>On linear templates , in vitro transcription of csiD and its activation by cAMP-CRP is clearly stronger with EsS than with Es70 ( Figure 9 ; on supercoiled templates , however , this strong preference for EsS is conditional ; see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>While strong EsS preference and more ef ® cient activation by cAMP-CRP in the presence of EsS were observed with linear templates ( Figures 9 and 10A ) , Es70-mediated transcription was improved with supercoiled templates , and could also be stimulated by cAMP-CRP ( Figure 10B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>This presumably facilitates activation of transcription in the presence of cAMP ± CRP and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>This specific feature of the B strain is attributed to the activation of the transcription factor CRP due to the higher cAMP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25524697" target="_blank">25524697</a>]<br></li>
<li>This activation of transcription was dependent on the addition of both cAMP and CRP and on the concentration of CRP used ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1085">
Nakano , M. , Ogasawara , H. , Shimada , T. , Yamamoto , K. & Ishihama , A. ( 2014 ) Involvement of cAMP-CRP in transcription activation and repression of the pck gene encoding PEP carboxykinase , the key enzyme of gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></label>
<input id="act_similarity_1085", type="checkbox">
<div class="ACT_similar">
<li>Involvement of cAMP-CRP in transcription activation and repression of the pck gene encoding PEP carboxykinase , the key enzyme of gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_1087">
Dissection of a surface-exposed portion of the cAMP-CRP complex that mediates transcription activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></label>
<input id="act_similarity_1087", type="checkbox">
<div class="ACT_similar">
<li>Meibom , K.L. , Sùgaard-Andersen , L. , Mironov , A.S. and ValentinHansen , P. ( 1999 ) Dissection of a surface-exposed portion of the cAMP ± CRP complex that mediates transcription activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Meibom , K. L. , Sogaard-Andersen , L. , Mironov , A. S. , and ValentinHansen , P. ( 1999 ) Dissection of a surface-exposed portion of the cAMP-CRP complex that mediates transcription activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>When CRP and cAMP were added in combination with RhaR and L-rhamnose , an approximately fourfold synergistic activation of transcription from pSR was seen ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Biochemical and biophysical results revealed that regulation of transcription activation depends on alignment of CRP tertiary structure through inter-domain communication and it was concluded that positions 72 and 82 are essential in the activation of CRP by cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Our results suggest that this mechanism is the activation by cAMP-CRP of a divergent RNA that strongly inhibits crp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>These results conﬁrm that cAMP-CRP stimulates ﬂiA transcription , most probably through ﬂhDC activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>These results demonstrate that transcription of gntP requires activation by the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>It is interesting to notice that the transcription level of the acsyjc-HactP operon and maeB gene , both activated by CRP , was downregulated in strains PB11 and PB12 as compared to JM101 strain ; again , this effect could be associated with the reduced cAMP concentrations in the PTS -- strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18679018" target="_blank">18679018</a>]<br></li>
<li>For example , in the galactose utilization network of E. coli , transcription of both the spf gene and the gal operon is simultaneously regulated by cAMP -- CRP ( 11 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Furthermore , the gal regulon genes are differentially regulated by cAMP-CRP and D-galactose , and in the absence of D-galactose the expression of the mglBAC operon can be activated by cAMP-CRP , whereas the transcription of galP gene is strongly repressed [ 38,39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>CRP , a homodimer , undergoes a conformational change when complexed with its allosteric-effector cAMP and binds to a speciﬁc sequence located near or within target promoters to regulate transcription ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>When complexed with its allosteric-effector cAMP , CRP undergoes a conformational transition ( 7 , 8 ) and binds to 22-bp target sites ( 9-12 ) within or near promoters of catabolite-sensitive operons to activate their transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>CRP is able to-bind DNA to positively or negatively regulate transcription after it complexes with cAMP , the levels of which increase in response to low glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>To test the effect of CRP on proP P2 regulation , single-round in vitro transcription reactions were performed with CRP-cAMP and RNA polymerase complexed with s38 ( Es38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In this paper , we report that the regulation of spot 42 RNA synthesis occurs at the level of transcription of the spf gene and depends on both cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>In this study , we identiﬁed , for the ﬁrst time , that cAMP-CRP bound downstream initiation represses transcription of the pck gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In particular , the importance of cAMP -- CRP for CyaR transcription can be seen for the cyaA strain , where addition of the alarmone to the growth medium restored sRNA expression ( lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>In contrast to the normal fragment , the addition of cAMP-CRP did not activate the divergent transcript when the altered template was transcribed and no inhibition of crp transcription was found ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>It was shown conclusively , by using cya-lac fusion strains , that one of the cya promoters , P2 , is negatively regulated by the cAMP-CRP complex and that the decrease in cya transcription under repressing conditions is about fourfold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>It was concluded from this KMnO4 footprinting results that the cAMP-CRP regulates the gal promoters at the level of [ RP ] O formation ; transcription bubble formation is stimulated at P1 but repressed at P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760580" target="_blank">10760580</a>]<br></li>
<li>We conclude that the transcription of crp is both negatively and positively autoreguiated depending on the concentrations of CRP and/or cAMP , Thus , the transcriptional control of crp by CRP-cAMP is more complicated than originally thought crp RNA ( 160 nt ) ^ ^ „ , , , , , ̂ , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Taking these results together , we conclude that crp transcription is regulated both negatively and positively by CRP-cAMP depending on the concentrations of CRP and cAMP in vivo as well as in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Also , the in vitro transcription data ( Fig. 3 ) 70 demonstrate that Es can use cAMP-CRP bound to 70 268.5 only to some very minor extent , whereas this activator position is optimal for Es .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>We observed that not only cAMP-CRP , but also Lrp could stimulate csiD transcription by EsS ( on linear as well as on supercoiled templates ) , and that the effects of cAMP-CRP and Lrp were additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>To investigate whether or not transcription of fadL was stimulated by CRP and cAMP , these compounds were also used in the transcription reactions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Discussion Regulation of crp transcription Here we have shown that in addition to cAMP -- CRP , another trans-acting factor , FIS , is indispensable for repression of crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>PTS sugars are present in the growth medium , low intracellular levels of cAMP and CRP result , and there is almost no transcription of the lsr operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>When glucose is absent from the medium , cAMP is present at high concentrations and binds with CRP to inﬂuence gene transcription ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16597943" target="_blank">16597943</a>]<br></li>
<li>At these D-galactose concentrations the level of active GalS would depend on both the cAMP -- CRP concentration and the active GalS level itself , which represses galS transcription ( Weickert and Adhya , 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Since the O c mutant lacks the cytPO site , CRP + cAMP promoted transcription in this strain can not occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6780756" target="_blank">6780756</a>]<br></li>
<li>Effect of cAMP concentrations on transcription ot the fragment carrying mutated CRP site II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>No transcription took place at the P3 site in the absence of cAMP ( Fig. 7A ) or CRP ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>Hence our results stronglsyuggest that in vivoas in vitrogal transcription initiatesa t SI in the presenceof CAMP-CRP and at S pin the absence of cAMP or when CRP is deficient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>CRP designates a cAMP-independent mutant of CRP ; even in the absence of cAMP it activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Accessory factors are a decrease of acetyl phosphate levels under energy limitation conditions ( McCleary and Stock 1994 ) and increased levels of cAMP-CRP , which now acts as a class I activator for transcription from rpoSp1 ( F. Mika and R. Hengge , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16291649" target="_blank">16291649</a>]<br></li>
<li>cAMP and CRP , acting as a complex , stimulate ( positive control ) transcription at Pi , whereas they inhibit transcription from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>It is possible that the CRP -- cAMP complex may activate transcription of another gene ( such as dnaK ) which , in turn , acts as an anti-s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>It is now clear that cAMP and CRP , in vivo as in vitro , act both to stimulate transcription from P1 and to inhibit that from P2 ( Table 2 , line 3 versus line 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>With linear templates , cAMP-CRP signi ® cantly stimulates EsS-mediated in vitro transcription , whereas transcription by Es70 is negligible and hardly stimulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In in vitro studies , the effects of cAMP and CRP on rpoH transcription have not been tested directly ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>When cAMP -- CRP level is increased , spf transcription is inhibited , while gal transcription is activated , therefore removal of Spot42 sRNA can be facilitated , resulting in a faster but still graded recovery of GalK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>As expected , only when transcribed by RNP containing the intact a-subunit ( a-WT ; Figure 3 , lane 2 ) could the cAMP ± CRP activate transcription at lacZP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Here , we demonstrate that the effect of glucose on dad transcription in vivo is mediated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>Under high-glucose and LP conditions , where the amount of PhoB is large and that of cAMP-CRP is small , PhoB activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>cAMP and CRP requirement of negative regulation of spf transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>Thus it seems that regulation by both FadR and ArcA-ArcB rely on the cAMP-CRP complex to activate transcription such that it can be further modulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Thus , FIS and cAMP -- CRP regulate transcription from both the crp and divergent RNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Effect of cAMP -- CRP on transcription of the gal regulon promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Finally , the cAMP -- CRP machinery is now presumed to regulate transcription at nearly 200 different promoters [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>A common feature of these FIS-regulated operons was that their transcription is activated by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Thus , it seems likely that CRP acts directly to activate acs transcription and that rising cAMP levels help to trigger that transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>By contrast , cAMP ± CRP stimulated the transcription at deoP2 when transcribed by RNP reconstituted with truncated a-subunits as well as intact a-subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>As expected , cAMP-CRP is required to activate gntP transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>As expected , transcription from P1galE was enhanced and that from P2galE repressed by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>cAMP and CRP had no effect on Nac-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>]<br></li>
<li>We showed that cAMP and CRP negatively regulate luxS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Another role of cAMP , which is independent of CRP transcription , has been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>While CRP-cAMP is able to completely inhibit transcription at 0.2 M potassium glutamate , CRP only slightly inhibits transcription at 0.4 M potassium glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Effects of cAMP and CRP on the transcription of luxS and pfs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>When 50 pmol of the CpdA protein was preincubated with cAMP and CRP before the addition of template DNA and RNA polymer-ase , transcription from the lacP1 promoter was partially inhibited , and instead a small amount of the lacP2 transcript was detected ( Fig. 5 , lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>When 100 pmol of the CpdA protein was preincubated with cAMP and CRP , transcription from the lacP1 promoter was completely repressed ( Fig. 5 , lane 5 ) , and the level of the lacP2 transcript was as high as that in the absence of cAMP and CRP ( compare Fig. 5 , lanes 1 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>An alternative explanation is that tandem DNA-bound cAMP-CRP complexes function synergistically to activate transcription from P2 , as has recently been found for the odd promoter ( Hoist ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Regulation of the P2galE promoter , located 5 bp upstream of P1galE , is opposite ; transcription from this promoter is further enhanced by OE-bound GalR and repressed by cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>specific promoter ( P5 ) which is sensitive to glucose repression and which is regulated by cAMP and CRP is involved in the control of rpoH transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>These results establish that transcription from the P5 promoter is mediated by RNA polymerase-u70 and depends on both cAMP and CRP under the conditions employed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>In vivo and in vitro experiments have shown that transcription from the Pr promoter is downregulated by the addition of glucose and this catabolic repression is also mediated by the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>In this study , we demonstrate that crp transcription is regulated by FIS and that both FIS and cAMP -- CRP are required to keep the crp promoter in a repressed state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>In line with previous reports ( Hanamura and Aiba , 1992 ) , addition of cAMP -- CRP activated transcription from the divergent promoter and markedly reduced initiation at crp1 ( Figure 6 , lanes 2 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>In vitro transcription from the ilvB promoter was markedly increased by the addition of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>It is known that cAMP forms a complex with CRP ( cAMP-CRP complex ) that binds to a speciﬁc promoter region of genes to either activate or inhibit transcription depending upon the position bound by the complex relative to the promoter ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>In the absence of glucose , cAMP binds to CRP and forms a complex that regulates transcription by binding to the promoter regions of cAMP-regulated genes ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16597943" target="_blank">16597943</a>]<br></li>
<li>However , when glucose or other PTS sugars are absent , the cAMP-CRP complex binds to the upstream region of the lsr promoter and stimulates its transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>( A ) In LB medium without the addition of glucose , cAMP forms a complex with CRP and speciﬁcally binds to the hfq promoter to lower the transcription of hfq gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>It was therefore of interest to determine directly if CRP binds to the ilvB promoter and to further investigate the effect of cAMP-CRP on iji vitro transcription in this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Aiba ( 4 ) found that cAMP-CRP strongly inhibited transcription of the crp gene in vitro and that this complex could bind with high affinity to the crp promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Under conditions in which cAMP levels are high ( e.g. , when cells are growing in glycerol minimal medium or in Luria-Bertani [ LB ] medium ) CRP activates transcription of many promoters , including those required for the utilization of such alternative carbohydrate carbon sources as lactose , galactose , arabinose , and maltose ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Under aerobic conditions with excess glucose , control of sdhCDAB -- b0725 -- sucABCD transcription by Arc , FNR , FruR and cAMP-CRP will play a minor role and , thus , reduced expression of this operon during glucose overflow metab-olism is likely due to the control by the cAMP-CRP independent catabolite control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Since CRP is activated by cAMP that is produced in the absence of glucose , detailed analysis was focused on CRP-dependent regulation of the genes for the ribosome modulation factors RMF and RaiA , which are induced during growth transition from exponential to stationary phase ( 19 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>PTS not only transports specific sugars but also in the absence of its substrates , stimulates , through adenylate cyclase ( Cya ) , the production of cAMP which in turn activates the transcription of many cAMP-CRP dependent catabolic genes , including those involved in glycerol utilization [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Expression of these genes is activated in the absence of glucose through functional conversion of CRP into an active form after interaction with an effector cAMP [ 17 ] , which is synthesized by the membrane-bound adenylate cyclase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>cAMP-bound active form of CRP is then regulates transcription of a number of genes that are involved in the utilization of non-glucose carbon sources in the absence of glucose [ 9,11 -- 13 ] , and thus it is an antagonist of catabolite repression [ 9,12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The results of band shift assays indicated that both hupA and hupB promoter DNA fragments do indeed bind CRP in the presence of cAMP , with a stability comparable to the lac operator ( Kolb et al. , 1983 ) , ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>AraC , one of the members in Group 2 , is suggested to-bind to at least two sites around the promoter region and to form a double stranded DNA loop of more than 200 bp in the absence of arabinose , cAMP or CRP , while the loop could open in the presence of both arabinose and cAMP-CRP ( Dunn et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>We incubated fragments corresponding to the entire acs-nrfA intergenic region , its 59 portion , or its 39 portion with increasing concentrations of puriﬁed CRP protein in the presence of cAMP ( 0.2 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Fragments were incubated with increasing concentrations of puriﬁed CRP protein as shown , in the presence of cAMP , as described in Lloyd et al. ( 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>However , even with supercoiled csiD templates , EsS preference , especially in the presence of cAMP-CRP , was restored in the presence of the histone-like protein H-NS ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Initiation at P0 is enhanced by the presence of exogenous glucose ( 12 ) , whereas in the absence of glucose , initiation depends strictly on the presence of the complex formed between the cAMP receptor protein ( CRP ) and cAMP ( CRP-cAMP complex ) ( 14 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>We find that H1 makes sequence-specific contacts and that the final I Abbreviations : DNaseI , deoxyribonuclease I ; CRP , cAMP receptor protein ( presence of cAMP bound to protein is implied ) ; bp , base pair ; ( OP ) 2Cu +,2:1 1,lO-phenanthroline-cuprous ion complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>We located the Hae-437 fragment in a HaelII digest of pNL4 plasmid DNA by its size , its possession of a BamHI site , and by its ability to direct the synthesis of the 5 ' - end of araBAD mRNA in vitro in the presence of araC protein , CRP and their ligands t-arabinose and cAMP ( Fig. 4 , band a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7019009" target="_blank">7019009</a>]<br></li>
<li>At 5 mM cAMP , CRP dimer binds and shifts gcv DNA at protein concentrations similar to those seen in Fig. 3 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>This result suggests that CRP can bind to gcv DNA at similar concentrations of protein in the presence of high or low levels of cAMP , supporting the in vivo data demonstrating that CRP can regulate gcv optimally in GM medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Data obtained in the presence be fitted by the equation : as in the absence of CRP-cAMP could Table I. Activities of the malT and Px promoters MalT In the presence of Px Px In the presence of malT Alone Alone Template 214 bp 214 bp cut by HIphIl 214 bp 214 bp cut by HaeIll Product ApUpU ApUpU CpUpUpU ApUpC CpUpUpU ApUpC - CRP 38 49 16 7 12 6 19 i 9 12 6 + CRP 71 74 2 I 1 1 I1 1 1 +1 The activities or tum-over numbers of the two promoters are expressed as the amount of mols of UMP incorporated/mol of template/min with SD 20 % unless otherwise indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Transcription of the gal operon is initiated from two overlapping promoters , P1 , in the presence of the activating complex cAMP/CRP , and P2 , in its absence ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>Transcription of csiD by s variants with alterations in S region 2.5 as a function of the nucleotides at the 213 and 214 positions : results in the absence of the activator cAMP-CRP In the absence of CRP , overall expression of csiD is reduced , but the stimulating effect of replacing A ( 213 ) by a C in csiDp is even more pronounced than in the presence of CRP ( compare data in Figs 4A and 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Regulation by cAMP-CRP has both positive and negative elements , with the latter being dependent on the presence of os , whose expression is negatively influenced by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Therefore , the presence of CRP -- cAMP-activated glnAp1 may partially compensate for CRP -- cAMP-inhibited glnAp2 , allowing a low and signiﬁcant level of expression of the important genes transcribed from it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Expression of the psiE9-9cat fusions ( pKE135 , pKE136 , and pKE137 ) was reduced in DphoB strain transformants when transformants were grown in the presence of cAMP in both Tris-glucose LP and HP media ( Table 1 ) , suggesting that expression of these constructs is also negatively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Whereas Mlc is responsible for induction of ptsG ( and several other genes ) in the presence of glucose , the catabolite activator complex cAMP-CRP is also absolutely necessary for ptsG expression ( 15 , 26 , 32 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855233" target="_blank">16855233</a>]<br></li>
<li>Glucose is known to affect gene expression through the cAMP-CRP complex , and the presence of glucose in the growth medium results in decreased levels of cAMP and CRP ( 26 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Interestingly , we found that the expression of pfs was reduced by the presence of glucose , but not through the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Exogenous cAMP partially compensates for the repressed level of SecB in the presence of glucose , and the compensatory recovery by the addition of cAMP is lost in cells lacking functional production of CRP from a cya and crp mutant strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>As shown in Fig. 4b and c , the level of cAMP was higher when cells were cultured in the presence of 0.04 % glucose compared to 0.4 % glucose , while the level of CRP was nearly equal in cells cultured in the presence of either 0.4 or 0.04 % glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>CRP and cAMP levels are at maximum cellular levels in bacteria growing in media lacking glucose as a carbon source and decrease about 2.5 - to 5-fold in bacteria in the presence of glucose ( 26 , 27 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>In this experiment , the 301 bp T'aq ` l-H/ndlll fragment carrying crp and divergent promoters ( Fig. 1 ) was transcribed in the presence of 100 pM of cAMP and various concentrations of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>In contrast to CRP -- cAMP , the Lac repressor functioned effectively at K + glutamate concentrations up to 0.6 M ( Fig. 3B ) ; transcript levels obtained with 0.1 or 0.6 M K + glutamate in the presence of repressor were # 10 % of the no-repressor control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In contrast , substituting the P3hfq promoter with a lacZ promoter ( that is positively regulated by cAMP-CRP ) increased the stability of the ompA mRNA in the presence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>In order to determine whether the increased activity of the crp promoter in ﬁs cells is the result of a reduced cAMP level such that no active cAMP -- CRP complexes are formed , we grew the cultures in the presence of 1 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>In vitro kinetic measurements suggest that the stimulation of P1 is due to the product KB X kf being five times greater in the presence of cAMP CRP than in the absence of the complex ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>In the case of SecB , the cAMP-CRP complex appears to function as an activator since its presence leads to an increase in SecB levels in glucose medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023186" target="_blank">9023186</a>]<br></li>
<li>In these conditions , we observed the disappearance of the hyper-reactive C ( 234 ) that is formed in the presence of the cAMP -- CRP complex alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>In the presence of catabolite-repressing sugars such as D-glucose and D-gluconate , cAMP and CRP levels are low and transcription of the idn genes is not induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>In the presence of CRP , expression of glpD and glpE increased 5 - and 3-fold upon the addition of cAMP , respectively ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>In the presence of glucose , the cAMP level is expected to be decreased , and hence the cAMP CRP level , thus the CRP-dependent regulation is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In the presence of glucose , CyaA does not catalyze the conversion of ATP to cAMP and CRP is not activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In the presence of excess glucose , cAMP levels are low and cAMP-CRP thus is unavailable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>In the presence of CRP -- cAMP , overexpression of nifA from pMJ220 resulted in equally high level expression of glnAp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In a previous study ( 50 ) , we showed that the presence of glucose strongly inhibits expression of the lsr operon through control of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In the presence of glucose , low levels of cAMP and CRP result in almost no expression of the lsr operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>We conclude that , at the csiD promoter , H-NS generates high stringency with respect to the position of the activator cAMP-CRP and thereby contributes to maximal expression in the presence of cAMP-CRP ( when bound to its wild-type position at 268.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>However , in the presence of cAMP and CRP , purified RNAP induces strong reactivity near the melR transcript start due to open complex formation at the melR promoter ( Fig. 3 , lanes e and f ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>However , the presence of cAMP -- CRP and RNAP induces a strong reactivity of the thymine residues from position 21 to 212 with respect to the transcription start site due to open complex formation at the Pr promoter ( Fig. 4a , lanes 4 and 5 ; Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Multiple-round transcription reactions were performed with PguaB ( 133 to 36 ) and the class II CRP-dependent promoter CC ( 41.5 ) , using 10 nM RNAP holoenzyme in the presence or absence of 20 nM CRP and 200 M cAMP , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>We began by cloning the aer200 fragment upstream of the loop terminator in plasmid pSR , and tested the ability of puriﬁed Es28 and Es70 to drive transcription from the aer promoter in an in vitro multiround transcription assay , in the presence and absence of puriﬁed CRP and cAMP ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In the presence of both cAMP and CRP , a transcript from the lacP1 promoter was observed ( Fig. 5 , lane 2 ) , whereas a transcript from the lacP2 promoter was observed in the absence of cAMP and CRP ( Fig. 5 , lane 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>In the absence of CRP , Ea70 transcribes mainly from the P2 promoter , while in the presence of CRP and cAMP , EaY70 transcribes efficiently from the P1 promoter , but P2 transcription is repressed ( Fig. 6 , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>We incubated the acs promoter fragment pacs444 , an AatII-HindIII fragment that carries acs sequences from position 379 to position 65 ( Table 1 ) , with increasing concentrations of puriﬁed CRP in the presence of cAMP ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Second , the effect of cAMP/CRP on promoter activity was measured in a cytR ~ , Acya , Mac strain with glucose as carbon source in the absence or presence of 5mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Therefore , the P1galE input function resembles a ` D-galactose ' gate , with higher promoter activity in the presence of cAMP -- CRP , while the P2galE input function can be described by a ` D-galactose AND ( NOT cAMP ) ' logic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Even in the presence of CRP and cAMP , the yield was still significantly lower than when the CRP independent malT P1 promoter mutant was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Binding of RNAP to the Pa promoter depends on the presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>To begin a detailed examination of the role of the divergent promoter in crp autoregulation we purified divergent RNA made when fragment 3 is transcribed in the presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>RNA polymerase titrations of the malT promoter in the presence of 50 nM CRP and 200 j4M cAMP ( 0 ) or in its absence ( 0 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>They demonstrated that in the presence of cAMP -- CRP the RNA polymerase preferentially binds to the divergent RNA promoter , while in its absence it binds to the crp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>We found that only EsS could form an open complex at the csiD promoter in vitro , which was further stimulated by the presence of cAMP-CRP ( Figure 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>DNase I footprinting analysis of the tsx-p2 promoter and its mutant derivatives in the presence of the cAMP-CRP activator complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Fig. 7 , panel A , shows that PaaX binds very tightly to the pga promoter in the absence of PAA but fails to-bind in its presence and that , conversely , the cAMP-CRP complex binds strongly to the pga promoter in the presence of PAA but hardly at all in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>EsS was unable to form an open complex in this putative promoter region , both in the absence and presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We performed the retardation assays in the presence of CRP and cAMP , although we have observed that RNAP can bind to the Pz promoter even in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>RNAP is able to-bind to the Pa promoter even in the presence of increasing concentrations of PaaX , and a low mobility complex due to the interaction of the PA probe with RNAP , PaaX and cAMP-CRP is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The Taqi-HindWl fragments ^ ^ P-labeiled at their Taql 5 ' ends denved pCR5 carrying the wild-type CRP site II ( A and B , lanes 1-3 ) and pCR41 carrying the mutated site II ( B , tanes 4-6 ) were treated with DNase I in the presence ( A , lanes 3-9 and B , lanes 3 and 6 ) or absence ( A , lane 2 and B , lanes 2 and 5 ) of CRP and cAMP , The concentrations of DNA and CRP were 1 nM and 100 nM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Each DNA probe ( 0.5 pmol each ) was subjected to a gel shift assay with increasing concentrations of CRP ( lanes 1 to 6 , 0 , 0.1 , 0.25 , 0.5 , 1.0 , and 1.0 M , respectively ) in the presence ( lanes 1 to 5 ) or absence ( lane 6 ) of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The truncated DNA fragment did not show a speciﬁc band shift at 10 nM CRP in the presence or absence of cAMP ( Fig. 2 , lanes 7 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Labeled DNA ( ﬁnal concentration , 0.4 nM ) was incubated at room temperature for 30 min in 10 l of a buffer containing 20 mM HEPES ( pH 8.0 ) , 5 mM MgCl2 , 50 mM potassium glutamate , 1 mM dithiothreitol , 10 % ( vol/vol ) glycerol , 20 g ml 1 sonicated calf thymus DNA ( GE Healthcare ) , and , where appropriate , 200 M cAMP , in the absence or presence of 200 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>RNAP in buffer ( 50 mM Tris acetate , pH 7.5 , 50 mM sodium acetate , 10 mM magnesium acetate , 0.1 mM cAMP , and 1 mM dithiothreitol ) was loaded in one syringe , and the single 2-AP-labeled lac promoter DNA , either in the presence or absence of CRP , in the same buffer was loaded in the other syringe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Binding of cAMP-CRP to the the ulaG-ulaA intergenic region was undertaken in gel shift experiments performed with different probes and increasing amounts of cell extracts of strain JA211 overexpressing CRP from plasmid pBS-CRP in the presence of various concentrations of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>To confirm these genetic experiments , gel retardation assays were performed with purified CRP in the presence of cAMP and three different DNA fragments ( Pz-Pa , Pz , and Pa ) used as probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>To confirm the in vivo experiments reported above , gel retardation assays were performed with purified CRP in the presence of cAMP and the Pa-Pr fragment as probe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>It is also noteworthy that gel mobility shift experiments on these systems have not provided evidence for multiple liganded complexes , whether in the presence or absence of CRP ( cAMP ) 1 binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>It was also evident from briefer exposure of the gel shown in Figure 3 that , in the presence of cAMP and CRP , formation of the longer transcripts from the HaeIII and AvaII fragments was partially inhibited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453417" target="_blank">16453417</a>]<br></li>
<li>A second gel mobility shift assay was performed to determine the lowest concentration at which CRP could bind and shift gcv DNA in the presence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Binding of CRP to DNA at this concentration was dependent on the presence of cAMP ( Fig. 2 , compare lanes 2 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Using the full-sized DNA probe and in the presence of cAMP , a single-shifted band was observed at the input CRP : DNA molar ratio of 2 ( Fig. 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>A labeled 170-bp DNA fragment containing secB regulatory regions was incubated with increasing concentrations of CRP in the presence of cAMP and then treated with DNase I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Plasmid DNA was digested with HindIII and EcoRI and incubated in the presence or absence of purified CRP and cAMP , as described by Garner and Revzin ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>32P-labeled DNA fragments ( 0.1 nM ) were incubated with CRP proteins ( 0 to 6.4 3 1029 M ) in the presence of cAMP for 30 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>After repeating the Genomic SELEX cycle in the continuous presence of cAMP , DNA fragments with affinity to CRP were enriched , leading to form sharp PAGE bands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>cAMP was titrated in the presence of 0.1 mM CRP using deoP2 DNA template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>CRP is a well-studied global transcription regulator that binds to DNA as a dimer and activates transcription in the presence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>CRP binds as a dimer to speciﬁc DNA sites and induces a DNA bend of approximately 90 degrees in the presence of its ligand , cAMP ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Non-conservative mutations at these positions produced poor activators of lacP ; cells containing either the E72Q CRP or the R82Q CRP and incubated in the presence of cAMP failed to produce - galactosidase activity above the background levels observed for cells cultured in the absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Results indicated that dnaK expression was not affected by the presence or absence of CRP -- cAMP , as indicated below : dnaK expression ( b-galactosidase activity in Miller units ) in TP2006 without cAMP is 1929 ^ 20 , with cAMP is 1656 ^ 13 ; in TP2339 -- 1 ( pLG339CRP ) without cAMP is 1611 ^ 6 , with cAMP is 1554 ^ 21 ; in TP2339 -- 1 ( pLG339DRS ) without cAMP is 1509 ^ 4 , with cAMP is 1416 ^ 18 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Purified KK81 fragment was end-labelled at the HindIII site and treated with permanganate after preincubation with different combinations of purified RNAP , cAMP-CRP , and MelR , in the presence or absence of melibiose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>cAMP ± CRP footprint at the downstream site ( centered at ± 41 ) was detected only in the presence of 1 mM CRP ( Figure 7A , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Lanes 2-4 show the results of digesting the fragment with DNase I in the presence of CRP ( Lane 2 ) ; and in the presence of CRP and , respectively , 10 and 20 ltM cAMP ( Lanes 3-4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>( Top ) EMSA results as shown by 32P-labeled hfq or NCH fragment in the presence of 200 uM cAMP and different concentrations of CRP as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>Finally , the MelR-induced hyper-reactivity near position 210 is significantly increased by the presence of cAMP and CRP , both in the presence and the absence of melibiose ( Fig. 3 , lanes i and j ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>However , at 100 nM CRP dimer all of the truncated template shifted in the presence and absence of cAMP ( Fig. 2 , lanes 8 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>( B ) Binding of cAMP-CRP to probe P65 , using 2 ( lanes 2 and 3 ) or 10 ( lanes 4 and 5 ) g of crude extracts of strain JA211 transformed with pBS-CRP , in the absence ( lanes 2 and 4 ) or presence ( lanes 3 and 5 ) of 125 M cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>rbs , measured in the presence of 50 nM CRP and 200 uM cAMP ( 0 ) and in its absence ( 0 ) is plotted versus the reciprocal of the RNA polymerase concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>For the gel shift assay , 0.2 pmol each of the FITC-labeled probes was incubated at 37 ° C for 30 min with various amounts of CRP in the presence or absence of 0.02 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>We therefore used EMSAs with purified CRP in the presence or absence of cAMP to test the proposed sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>After the addition of cAMP in the presence or absence of CRP in vivo , the flgalactosidase activity resulting from the fused genes was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Again , whereas induction of ppdD , yrfD and sbmC absolutely required both CRP and Sxy , mglB was induced by exogenous cAMP to the same levels in the presence or absence of Sxy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>( B ) Schematic representation of DNase I and DMS footprints of sequences isolated in the absence of cAMP -- CRP ( A12-09 and A12-44 ) or in its presence ( C13-20 , C13-40 , C09-20 , C13-26 , C13-05 and C09-17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Three CRP conformations have been detected experimentally : the apo-CRP , which predominates in the absence of cAMP ; CRP ( cAMP ) 1 , which predominates in the presence of micromolar cAMP concentrations ; and , CRP ( cAMP ) 2 , which predominates at millimolar cAMP concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Reactions were carried out in the presence of various concentrations of puriﬁed CsrA in the absence ( ) or presence ( ) of cAMP ( 0.1 mM ) and CRP ( 1.6 g ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>LCAP ( 27 pM ) was incubated with various concentrations of CRP in the absence and presence of 200 M cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Since CRP exhibits functional inter-conversion depending on the presence or absence of specific effector cAMP [ 11 ] , we carried out Genomic SELEX in the presence and absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>It should be emphasized that the CRP ` footprint ' is cAMP depend-ent , while the repressor ` footprint ' is not influenced by the presence or absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>CD and FTIR experiments showed that the secondary structure content of position 72 and 82 CRP mutants remained essentially the same as that of WT-CRP in the absence and in the presence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>WT CRP showed no footprinting in the absence of cAMP ( lane 3 ) , however WT CRP footprinted in the presence of cAMP ( lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>NMR experiments designed to compare CRP structures in the presence and absence of cAMP encountered solubility and size limitation problems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Reactions which contained NtrA , NtrC , or both factors were tested in the presence or absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Figure 7A shows cAMP ± CRP footprints where CRP was present at 0.1 mM ( lane 2 ) or 1 mM ( lane 3 ) in the presence of a saturating concentration of cAMP ( 0.1 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>When present , CRP was added to a final concentration of 75 nM in the presence of 200 µM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1273">
228 and 230 indicate guanines which are protected from DMS reactivity in the presence of CRP , and R indicates a guanine used as a reference , whose reactivity was not inﬂuenced by the presence of CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></label>
<input id="act_similarity_1273", type="checkbox">
<div class="ACT_similar">
<li>228 and 230 indicate the guanines which are protected from DMS reactivity in vitro in the presence of CRP ( Fig. 5B ) , and R indicates a guanine used as a reference , whose reactivity was not inﬂuenced by the presence of CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li></div>
<li>Binding of CRP to fragment EP was shown to be dependent on the presence of cAMP in the reaction mixture ( Fig. 5c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>A single complex was formed by CRP in the presence of cAMP on the fragment ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Indeed , the region containing nucleotides 388 through 413 was protected in the presence of 70 or 140 nM CRP plus cAMP ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>Fragment 1 in the presence of cAMP-CRP produced a transcript of 75 nucleotides ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Transcription of the spf gene encoding the Spot42 sRNA is reduced 3 - to 5-fold in the presence of cAMP-CRP ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Because the GalK level is not affected by the 3 - to 5-fold reduced Spot42 level in the presence of cAMP -- CRP ( 11 ) , we assume that mRNA -- sRNA pairing is negligible when the Spot42 sRNA is present at 50 copies per cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>When the labeled gntT fragment was incubated with CRP , only one band with reduced mobility was formed , and as expected , this occurred only in the presence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>ppGpp activated the synthesis of glgC independently of cAMP and CRP ; however , its effects were more pronounced in the presence of these factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>CRP -- cAMP inhibitory effect on glnAp2 and its derivatives in presence of different activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>( A ) Different concentrations of CRP were reacted with deoP2 in the presence of 0.1 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>These FITC-labeled probes ( 0.2 pmol ) were incubated with CRP ( lanes 1 -- 3 : 0 , 0.2 , and 0.8 pmol ) in the presence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Fragment 3 ( 1.2 pmol ) was transcribed in the presence of CRP ( 10 Ag/ml ) and cAMP ( 50 , uM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>First , the protection pattern in CRP-2 corresponds to that obtained in the presence of cAMP-CRP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>( b ) Effects of increasing CRP levels in the presence of saturating levels of cAMP ( 100 F.M ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1289">
( a ) Effects of cAMP in the presence of 2 p.g of exogenous CRP per reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></label>
<input id="act_similarity_1289", type="checkbox">
<div class="ACT_similar">
<li>( c ) Effects of ppGpp in the presence of cAMP ( 100 , M ) and CRP ( 1 p.g per reaction ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li></div>
<li>` C ' indicates that this selection was done in the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Effect of the presence of cAMP and CRP on gapA mRNA composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>For the wild-type fragment , clear footprints were again obtained with two CRP sites in the presence of 100 nM of CRP and 100 ) jM cAMP ( Fig. 7B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Experiments were conducted in the presence of 150 µM cAMP , a concentration that maximizes the proportion of the active CRP species in which cAMP is bound to only the low affinity cAMP sites ( 9 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The ﬁgure shows the transcripts produced in multi-round in vitro transcription assays using the aer200 fragment cloned in pSR , incubated with various concentrations of Es28 or Es70 , and in the presence or absence of 100 nM CRP and 0.2 mM cAMP , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The ﬁgure shows the results of KMnO4 footprinting using an aer200 PstI-HindIII fragment , end-labelled on the template strand and incubated with a ﬁnal concentration of 50 nM Es28 or Es70 , in the presence or absence of 100 nM CRP and 0.2 mM cAMP , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The central position of CRP , between AraC protein and RNA polymerase , is consistent with the sensitivity of araFGH to the presence of CRP-cAMP both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>The P1 and P4 transcripts that are known to be synthesized by RNA polymerase-U70 were produced regardless of the presence or absence of cAMP or CRP ; the interference between promoters P4 and P5 observed in vivo was not apparent under the in vitro conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>The following experiments were performed to investigate the effects of CRP on the repressor-operator interactions : CRP in the presence of cAMP was allowed to-bind to the sites on fragment A as shown m figure 4 , lane 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The csiD transcript was found almost exclusively with EsS and its formation was stimulated by the presence of cAMP-CRP ( Figure 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The right panel is in the presence of 20 mM cytidine in addition to cAMP ± CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The fragment was incubated with DNasel lor 15s ( lane 1 ) , in the presence of CRP alone ( lane 2 ) , and in the presence of CRP and 1-5jiM of cAMP ( lanes 3-4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>The Sau3A2-HpaI fragment was incubated with various concentrations of CRP in the presence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>The bar graph depicts the levels of the proP P1 transcript synthesized in the presence versus the absence of CRP -- cAMP at each K + glutamate concentration from the data shown above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The addition of cAMP increased the intensities of all three bands in the presence and absence of GntR , suggesting that cAMP-CRP acts as a positive regulator of gntT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>The A G sequencing ladder is indicated by an asterisk tightly even in the presence of a ® vefold excess of cAMP-CRP , i.e. the footprint pattern obtained with Lrp was identical in the absence and presence of cAMP-CRP ( Figure 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The results altogether indicate that a basal level of RMF production takes place in the absence of cAMP-CRP but that RMF production is enhanced in the presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The observation that cAMP is able to cause some stimulation in the absence of added CRP is probably the result of the presence of endogenous CRP from the S-30 extract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The reaction was run at 37 ° C in the presence or absence of cAMP and purified CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>The presence of glucose prevents E.coli from catabolizing alternative sugars by several mechanisms , one of which is that glucose lowers the level of cAMP , the inducer for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The afﬁnity of CRP for these promoters in the presence of cAMP is apparently the same among these promoters , indicating that these operons are regulated simultaneously in the absence of glucose , probably as a response to a decrease in the energy level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The activity of CA8306 ( cya ) harboring these clones was higher in the presence of cAMP , suggesting that gntK expression is controlled by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The expression of yﬁQ occurs from its own promoter ; it is upregulated in the stationary phase and in the presence of non-PTS carbon sources and is positively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>The right hand panels show the results of electromobility shift assays using the end-labelled promoter fragments , incubated with 0 , 50 , 100 or 200 nM CRP , in the presence of 0.2 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The P1 promoter is activated by the positive regulator CRP in the presence of cAMP ; the P2 promoter is repressed by CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>The gntT promoter fragment was incubated with different amounts of CRP and GntR in the presence of 5 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>The 311 bp HindIII-EcoRI fragment containing the pts promoter was incubated with or without indicated amount of CRP and/or Mlc in the presence of cAMP and then with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>The filled and open circles mark the G residues whose methylation by DMS is diminished in the presence of low ( 10 nM ) and high ( 100 nM ) concentrations of cAMP-CRP complex , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The results show that the presence of the CRP -- cAMP complex prevents NtrC-dependent open complex formation on glnAp2 ( Fig. 6A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The results showed that only probe P65 , encompassing the putative CRP site located at 61 of ulaA , yielded a retarded complex in the presence of cAMP ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Here we show that ( i ) CytR selectively regulated cAMP-CRP-dependent initiations , although transcription started from the same site in deoCp2 in the absence or presence of cAMP-CRP ; ( ii ) deletion of the uppermost cAMP-CRP target ( CRP-2 ) resulted in loss of CytR regulation , but had only a minor effect on positive control by the cAMP-CRP complex ; ( iui ) introduction of point mutations in either CRP target resulted in loss of CytR regulation ; and ( iv ) regulation by CytR of deletion mutants lacking CRP-2 could be specifically reestablished by increasing the intracellular concentration of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In the case of Pa ( Manso et al. , 2009 ) , MhpR binds at the operator sequence and in the presence of 3HPP the MhpR -- HPP interaction facilitates the recruitment of the cAMP -- CRP complex , which synergistically activates transcription of the mhp catabolic genes as occurs in typical class III promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>In the presence of cAMP , the cAMP-CRP complex ( hereinafter called CRP ) will bind to a 22 bp 2-fold-symmetric recognition site located within or near target promoters ( Berg and von Hippel , 1988 ) , and thereby activate transcription from these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Synthesis of the araC transcript from the wild-type promoter was stimulated 1.6-fold by the addition of cAMP and CRP ; the addition of araC protein ( repressor form ) reduced by 60 % the amount of araC transcription , even in the presence of the cAMP-CRP complex ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>Regardless of the presence of propionate , PprpR activity was reduced 2.5-fold in strains lacking CRP or adenylate cyclase ( JSW2 or JSW3 , respectively ) compared to that in the wild type ( JSW1 ) , indicating that the cAMP-CRP complex acts as a positive regulator of prpR transcription ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Because three of the promoters do not have detectable intrinsic activity , the effect of GalR and GalS in transcription reactions was studied in the presence of the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In the absence of the activator , the CyaR band disappeared , indicating that transcription initiation at cyaR-P in vitro is highly dependent on the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>( a ) In vitro 1.6 4.0 25.0 ND Activation factor transcription assays on supercoiled or linear templates with native Es54 in the absence or presence of cAMP receptor protein ( CRP ) H159L or integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Transcription reactions In vitro transcription reactions were carried out as described ( Howard et al. , 1977 ) , either in the absence or presence of 500 cAMP and 10 AM , g/ml CRP ( a gift from J.Krakow ) with an RNA polymerase to template ratio of 3:1 or 1:1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453417" target="_blank">16453417</a>]<br></li>
<li>Therefore , aerobic acetate formation in the presence of excess glucose is negatively correlated with transcription of acs -- yjcH -- actP due to cAMP-CRP dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>E ¡ ect of cAMPWCRP on gal promoters Transcription from the gal promoters was monitored in the presence of various concentrations of cAMP and the saturating concentration ( 100 nM ) of CRP [ 18,19 ] by both transcription and KMnO4 assays ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760580" target="_blank">10760580</a>]<br></li>
<li>We found that transcription of galR is independent of cAMP -- CRP concentrations ; therefore , its input function resembles a TRUE gate with different levels of expression in the presence and absence of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>gal transcription in vitro initiates at + 1 ( Fig. 1 ) in the presence of cAMP and CRP and at -5 in their absence ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>( A ) In vitro single-round transcription reactions were performed using the proP substrate pRJ4069 in the presence ( + ) or absence ( − ) of CRP -- cAMP and in the presence of 0.1 -- 0.6 M K + glutamate as denoted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>We performed in vitro transcription on pRPGSM DNA in the presence of cAMP -- CRP ( Fig. 2 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In vitro transcription assays were performed on pRPGSM DNA template in the presence of cAMP -- CRP ( left lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Results of the in vitro transcription assay performed on pRPGSM DNA template in the presence and absence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In vitro transcription on supercoiled plasmid DNA was performed in the presence and absence of CRP -- cAMP at different K + glutamate concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Whereas the presence of CRP -- cAMP inhibited in vitro transcription by purified Es70 at K + glutamate concentrations up to 0.3 M , repression was lost at $ 0.4 M K + glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Lanes : a-e , transcription for 2 , 3 , 4 , 5 and 10 min ; f-j , transcription in the presence of cAMP ( 5 mM ) and CRP ( 6 yg/ml ) for 2 , 3 , A , 5 and 10 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Taken together , we propose that cAMP-CRP associated at downstream CRP box-2 plays as a repressor for pck transcription only in the presence of high levels of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>At 0.3 M K + glutamate a small amount of P1 transcription was detected in the presence of CRP -- cAMP , and above 0.4 M K + glutamate CRP -- cAMP was no longer an effective repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>At the beginning of the addition of gluconate , a relatively weakly bound GntR at HRI or HRIII may be released as a complex with gluconate , and it is replaced by the cAMP-CRP complex as cAMP is accumulated , which in turn interacts with GntR bound at HRII ( the GntR may still be free from gluconate ) and activates transcriptions to some extent from the gntV and idnD promoters to produce some amount of GntII proteins including IdnD , IdnO and GntH proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>In the gal operon , one of two overlapping promoters is activated ; the other is inhibited by the complex ( Adhya and Miller , 1979 ) , and expression of the ompA gene in vitro is likewise inhibited by cAMP/CRP ( Mowa et al. , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>In gal the two promoters are controlled negatively by the gal repressor and one of the promoters is inhibited by cAMP/CRP , the other activated ( Adhya and Miller , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453417" target="_blank">16453417</a>]<br></li>
<li>g003 The whole set of regulation targets of CRP A number of approaches in vitro , in vivo and in silico have been employed for the identification of regulation targets by cAMP-CRP , and mixed together , a total of 195 target promoters are listed In Regulon DB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Taken together , these data suggest that cAMP-CRP may inhibit P3 in vitro by an indirect mechanism through its ability to activate promoters which overlap P3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>These results suggest that CRP , in the presence of cAMP , binds speciﬁcally to the wild-type DNA template but b crpAR1 , crpAR2 , and crpAR3 , crp with mutant activation domains AR1 , AR2 , and AR3 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>These results suggest that in PB11 , basal levels of active Cya supply enough cAMP required for CRP activation to allow growth on acetate almost as fast as the wild-type strain ( = 0.21 h -- 1 ) , while in PB12 apparently there is not enough cAMP necessary for fast growth on this carbon source even when the cAMP concentration is slightly higher in this strain as compared to PB11 ( table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18679018" target="_blank">18679018</a>]<br></li>
<li>Subsequently , this negative autoreguiation was shown to be caused due to the activation of an overlapping divergent promoter by CRP-cAMP , ( Okamoto and Freundlich , 1986 ; Okamoto etai , 1988 ; Hanamura and Aiba , 1991 -RCB- , During the course of these studies , we noticed that the extent of negative autoreguiation varied depending on the DNA fragments used , and the concentrations of DNA , CRP , cAMP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Although much remains to be learned about the molecular processes which occur during activation , it appears that binding of cAMP-CRP results in activation in most systems by increasing the affinity of RNA polymerase for the promoter or by blocking RNA polymer-ase from binding to competing , overlapping sites ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>However as CRP ( cAMP ) 1 subsequently binds CRP1 , the additional pairwise cooperative interaction should now cause CytR to compete effectively for CRP2-RNAP interactions and so tend to shut down CRP2-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In addition to its role in activation , cAMP-CRP also plays a crucial role in negative regulation of cdd ] P. cAMP-CRP and CytR bind cooperatively to cddP forming a nucleoprotein complex that covers 84 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In vivo analyses have shown that the uppermost CRP target ( CRP-2 ) is dispensable for cAMP-CRP activation , whereas a deletion of this site completely abolishes negative control by CytR , and point mutations in either CRP-1 or CRP-2 result in a partial loss of CytR regulation ( Valentin-Hansen , 1985 , Sogaard-Andersenefa / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In order to test cAMP-CRP activation and CytR regulation in this mutant deoP2 promoter i '' Enzymelevels are measured during exponential-growth at 30 ° C in AB minimal medium using glycerol ( crp ÷ ) or glucose ( Acrp ) as carbon source as described by Miller ( Miller 1972 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>We have undertaken a detailed study of the in vivo activity of the deoCp2 promoter to clarify the mechanism of CytR regulation and cAMP-CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>However , a direct analysis of the effect of mutations in CRP-1 on CytR regulation is complicated as such mutations also affect promoter structure and cAMP-CRP activation ( L. Sogaard-Andersen et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>This plasmid is a derivative of the well characterized transcription vector pSA508 ( Choy and Adhya , 1993 ) , and carries all the deoP2 sequence inform-ation ( from ± 116 to +25 ) required for cAMP ± CRP activation and CytR repression , followed by a strong factor-independent transcription terminator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Expression of heterlogous udp promoters in E. coli cells To verify the level of CytR regulation and cAMP -- CRP activation of heterologous promoters we constructed a series of plasmids carrying the udpP -- lacZ transcriptional fusions ( plasmids pMZ32 , pMZ37 , pMZ39 and pMZ43 , see Materials and methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>cAMP-CRP activation and CytR repression of the mutant promoters were determined by measuring the activity of/Lgalacto-sidase in isogenic strains carrying mutations in cytR or crp ( Table 1 and Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>As shown in Table 3 , CytR regulation was reduced from approximately 8-fold in udpPEc and udpPSt to approximately 3-fold in udpPYp , and to 1.6-fold in udpPV c whereas cAMP -- CRP activation was approximately the same in all promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Thus , CytR probably inhibits RNA synthesis by antagonizing cAMP / CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Here we present evidence that activation by cAMP-CRP and repression by CytR is coupled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>When the lac promoter fused with the lacZ reporter gene on pGD926 ( pGD926Plac ) was introduced in the reporting-strain TP2339 -- 1 , it gave 1800 Miller units of b-galactosidase of sufﬁcient basal level expression in the absence of CRP -- cAMP , and 32 000 Miller units upon CRP -- cAMP activation , suggesting that the lac promoter could be used to increase expression of the nifA and the rpoN genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Effect of mutations in the CRP site on P-galactosidase activity in strains carrying an araC-lacZ fusion '' f-Galactosidase sp act % Glucose Glycerol Glucose 5 cAMP repression mM Strain DISCUSSION The results described in this report support the conclusion that a CRP site at -82 to -110 in the ara regulatory region is involved in the activation of both araC and araBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>The CRP2 site , centred at -92.5 , probably is important for activation of nupG as it was previously shown that deletion of sequences upstream of - 8 0 decreases cAMP-CRP activation more than 10-fold ( Munch-Petersen and Jensen , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The interaction between a-NTD and cAMP ± CRP at ± 41 ( CRP1 site ) should , therefore , be important for activation of deoP2 , class II CRP-dependent promoter ( A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The activation role of the cAMP -- CRP regulatory system was further investigated by DNase I footprinting assays , which showed that the cAMP -- CRP complex binds to the Pr promoter sequence , protecting a region centred at position '' 40.5 , which allowed the classification of Pr as a class II CRP-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>The synergistic transcription activation by the cAMP -- CRP complex and the MhpR activator allowed the Pa promoter of the mhp cluster to be classified as a class III CRP-dependent promoter ( Torres et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>This is a likely possibility since cAMP ± CRP activation of a CRP II-dependent promoter should require precise interaction between those amino acid determinants on CRP and a-NTD , and therefore a subtle change in protein conformation could interfere with RNP and core promoter interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In contrast , by using strains impaired in CRP or cAMP production , we showed that the CRP-cAMP complex has a strong indirect effect on glucose-mediated activation of the gapB P0 promoter ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>The absence of typical -35 and -10 promoter elements in promoters requiring activation by cAMP -- CRP for full activity is common .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Furthermore , FIS prevented activation of the divergent promoter even in the presence of cAMP -- CRP ( Figure 6 , lanes 10 -- 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Remarkably , transcription in vivo from the mutant promoter was unaffected aerobically , which indicates that TdcA , bound to the presumptive co-inducer , together with IHF and TdcR are all essential for transcriptional activation , even when the cAMP ± CRP complex is not limiting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1375">
Malan , T.P. , Kolb , A. , Buc , H. and McClure , W.R. ( 1984 ) Mechanism of CRP -- cAMP activation of lac operon transcription initiation activation of the P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218806" target="_blank">9218806</a>]<br></label>
<input id="act_similarity_1375", type="checkbox">
<div class="ACT_similar">
<li>Malan , T.P. , Kolb , A. , Buc , H. & McClure , W.R. ( 1984 ) Mechanism of CRP ± cAMP activation of lac operon transcription initiation activation of the P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li></div>
<li>Finally , by in vitro transcription assays we have demonstrated the Received 13 July 2010 Revised 14 October 2010 absolute requirement of the cAMP -- CRP complex for the activation of the Pr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Expression from this promoter can be further enhanced ( about 7-fold ) in the presence of the cAMP-CRP complex , leading to a more than 200-fold activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>Moreover , both the ptsH P0 promoter and the ptsG P1 promoter have been shown to be subject to activation by the cAMP CRP complex and repression by Mlc ( 13 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>Under native conditions , in the absence of TcbC , these proteins must assume the role of realigning the cAMP ± CRP ± RNAP complex to cause promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Acetyl-CoA synthetase is a high-affinity enzyme that permits the scavenging for , and metabolism of , small amounts of acetate previously excreted into the medium ; thus , as the primary carbon source becomes depleted , the activation of the acs promoter by small amounts of CRP -- cAMP would delay entrance into stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Taken together , these results demonstrate that the dramatic decrease in prpBCDE promoter activity is most likely the result of decreased PrpR production in the crp and cya mutant strains in combination with the loss of cAMP-CRP activation of PprpBCDE itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Furthermore , it was shown that the negative autoreguiation of crp was coupled with the activation of a second divergent promoter by CRP-cAMP bound to CRP site i ( Okamoto and Freundlich , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>It is evident from the analysis of deletion derivatives of deoCp2 that efficient cAMP-CRP activation of deoCp2 require CRP-2 , and therefore does not CRP-1 is the essential site for promoter activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>These data suggest that the negative autoregulation of the crp gene is a direct consequence of the activation of the divergent promoter by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>This result indicates that promoter agaZp requires induction of AgaR and activation by cAMP-CRP for full activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>However , the observation with deoP2 suggests that the synergistic activation by two or more cAMP ± CRP should depend on the context of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Thus , activation of the lac promoter by CRP -- cAMP appears transiently sensitive to an osmotic upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Indeed , the homology between FNR and CRP is such that their DNA-binding specificities can be interconverted by appropriate substitutions in the recognition helices , such that catabolite repression can be lifted in response to anoxia and anaerobic gene activation can be induced by glucose-starvation ( cAMP ) ] 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>These NMR experiments indicate a possible role of E72 and R82 in cAMP binding and subsequent cAMP-mediated activation of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Because cAMP -- CRP-mediated P1galE activation is strongly inhibited by GalS , it is unlikely that the differential effect of GalR and GalS on cAMP -- CRP action would be due to the different binding affinities of the two proteins towards the galETKM OE operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>During the course of this work , it became apparent that GntR might affect gntT expression by a mechanism that is independent of negative control by operator binding as well as activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>CRP -- cAMP activation of lacP and binding to galP1D4 is osmotically sensitive CRP is a positive and negative regulator of many different genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>For example , cells growing in an amino acid-rich glucose medium would have low expression of P1 and P3 because of the lack of cAMP-CRP , but P2 expression would be high because of activation by Lrp-alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The observation that , similar to the situation in the naturally competent Gamma-proteobacterium , H. inﬂuenzae , the activation of sxy transcription requires both CRP and cAMP , suggests that growth in the presence of poor carbon sources or the exhaustion of preferred sugars may be required for sxy expression and might also hint at possible conditions that could favour competence development in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>The glucose repression of the ula regulon is mediated by at least two mechanisms , one independent of cAMP-CRP and the other mediated by this activator complex , which causes a weak , though signiﬁcant , activation of the ulaA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The strong activation by cAMP-CRP of P1 and the modest activation of P2 found in vitro and in vivo were absent in the CRP1 mutant ( Fig. 5A , lane e , bands 1 and 2 ; lane h , bands 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>The presence of glucose interferes with the activation of membrane-bound ad-enylate cyclase ( Cya ) , decreases cAMP levels in the cell , and represses the expression of CRP ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16597943" target="_blank">16597943</a>]<br></li>
<li>The presence of the cAMP-CRP complex has a stimulatory effect on activation ; however , CRP does not activate expression alone or directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>The substrates for EIICBGlc transport , glucose and aMG , are both strong catabolite-repressing mole-cules that cause a reduction in cellular cAMP levels , and presumably a reduced potential for ptsG activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>The GalR -- OE complex also prevents P1galE activation by the cAMP -- CRP complex ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The CRP -- cAMP activation of glnAp1 could be a ` backup ' system for when s or NtrC-phosphate is not 54 available to activate glnAp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The evidence indicates a role for E72 and R82 in the molecular mechanism for cAMP-mediated CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>The physiological role of CRP -- cAMP activation of glnAp1 is not clear ( Magasanik , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The 235 site is relatively poor , and this would explain why dctA expression is weak in the absence of cAMP-CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>The histone-like proteins H-NS and Lrp positively modulate csiD expression by differentially affecting activation by cAMP-CRP Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The evidence implies that activation of permease synthesis absolutely requires cAMP and CRP , but that expression ofaraBAD is activated at a low level if enough internal L-arabinose is provided .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>These results suggest that activation of ptsG P1 by heat-shock was not mediated by Mlc or CRP cAMP even though Mlc repression z might be dominant over activation of ptsG P1 by heat-shock .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11340070" target="_blank">11340070</a>]<br></li>
<li>We have shown that osmotically sensitive regulation by CRP -- cAMP is not limited to repression at the proP locus as activation of lacP is also transiently sensitive to osmotic upshifts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Expression of GalP requires the presence of both cAMP and D-galactose , for transcriptional activation ( by cAMP -- CRP ) and inhibition of repression ( by GalR and GalS ) respectively ( AND function ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>That H-NS acts by inﬂuencing activation by cAMP-CRP was also conﬁrmed by the observation that , in cya mutant backgrounds , the presence or absence of H-NS no longer made a difference ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>A possibility was that the presence of multiple copies of the arb genes was overcoming the absence of cAMP-CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11994151" target="_blank">11994151</a>]<br></li>
<li>Some operons such as malEFG are under the transcriptional activation by the response-regulators in Group 1 as well as by CRP , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>Contact inhibition by GalR prevents cAMP -- CRP activation of P1galE , and reduces P1galE activity by about 75 % in the absence of cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Although synthesis of isomerase and permease is activated in the mutant in the absence of cAMP and CRP , activation can be further stimulated by the addition of these products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Formation of a CRP -- cAMP complex is a prerequisite for activation of catabolic genes and CRP mediates the cAMP effect on these genes at the transcriptional level ( reviewed in Ullmann and Danchin , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>This demonstrates that the cAMP-CRP complex is necessary only for activation of pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>To further conﬁrm the involvement of the cAMP-CRP complex in the activation of the prpBCDE promoter , PprpBCDE-rfp expression was measured in crp - and cya-deﬁcient strains expressing PrpR in trans .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In contrast , CRP -- cAMP-dependent activation of glnA was not observed under nitrogen-excess conditions , presumably because activation of glnAp1 was reduced ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In gal and lac , these mutations have severe effects on cAMP-CRP activation and in vitro binding of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In nupG , activation is independently stimulated by cAMP-CRP , and full activation requires both of the CRP2 and CRP1 targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In the ® rst set of experiments , activation of deoP2 by cAMP ± CRP was examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>DNA helix phase dependency of cAMP-CRP activation of csiD expression mediated by Es or Es .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In summary , both histone-like proteins , H-NS and Lrp , modulate csiD expression by inﬂuencing activation by cAMP-CRP , but they obviously do so by different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>As the lrp mutation had 70 no effect when the CRP box was located at the nonproductive positions between 272.5 and 276.5 , Lrp seems to act as a co-activator for cAMP-CRP without affecting the ` topological ' rules for activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>S Both holoenzymes can be activated by cAMP-CRP bound at the classical class I position 260.5 When cAMP-CRP was positioned at 260.5 , csiD activation seemed to ` switch ' to a different mode .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Therefore , it seems improbable that CRP -- cAMP speciﬁcally interferes with NtrC-mediated transcriptional activation , for example by somehow interfering with NtrC0s activity , e.g. by disrupting the histidyl -- aspartyl phosphorelay between NtrB and NtrC proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Therefore , the fold activation of deoP2 by cAMP ± CRP when transcribed by a the RNP reconstituted with the truncated - subunit appeared to be much greater than that obtained with the a RNP reconstituted with intact , although the actual activity is less .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Derepression of fadF appears to be the result of the combined e $ ects of inactivation of FadR , cAMP ± CRP dependent gene activation and , to a lesser extent , accumulation of the alarmone ppGpp [ 262 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Finally , the constructs with S the relocated CRP box were used to test whether modulation by the histone-like proteins H-NS and Lrp is connected to activation by cAMP-CRP or proceeds independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>cAMP - CRP activation of cddP ( Figure 7B , compare lanes 4-6 with 10-12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Thus , the ` topological ' function of H-NS in activation by cAMP-CRP suggested by the data in Fig. 7A seems to be speciﬁc for Es .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>This is a result of an increase in cAMP levels which facilitates activation of fadD , fadE , fadF , fadG , fadBA and fadH by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>CRP activation of P1 and inhibition of P2 depends on the effective concentration of the cAMP - CRP in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>d Fold of cAMP -- CRP activation is the ratio between activity in a cytR − crp + strain over the activity in a cytR − crp − .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Transcription of galS requires activation by cAMP -- CRP in our in vitro system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>cAMP ± CRP activation of deoP2 Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Therefore , in the case of deoP2 , the cAMP ± CRP at ± 41.5 should be primarily responsible for the activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Although both H-NS and Lrp stimulate csiD activation by cAMP-CRP , they obviously do so by different mechan-isms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Thus , Lrp somehow improves the efﬁciency but does not alter the ` geometry ' of activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Promoter strength and cAMP - CRP activation , however , remained unaffected by the mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Selection for trans-acting mutations that make the entire regulon independ-ent of cAMP and CRP may be possible , since the three permeases that transport L-arabinose are unlinked to araBAD ; two are under the control of the araC gene product , and all three are severely catabolite repressed ( 8 , 12 , 16 , 31 , 32 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Finally , the crp * GQJ41 mutation , which renders the crp gene product CRP partially independent of cAMP for stimulation of cAMP-CRP-dependent promoters due to a glycine ( G ) to glutamine ( Q ) change at position 141 ( 12 ; S. Garges , personal communication ) , was examined for its ability to suppress the cAMP requirement for induction of the cst : : lacZ fusions ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2163388" target="_blank">2163388</a>]<br></li>
<li>The CRP encoded by the crp * GQJ41 allele is relatively independent of cAMP in activating cAMP-CRP-dependent promoters ( 12 ; J. Kim , S. Garges , and S. Adhya , unpublished data ) , and the Acya mutation eliminates changes in cAMP concentration in response to starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2163388" target="_blank">2163388</a>]<br></li>
<li>As stated above , the CRP protein encoded by the crp * GQ141 allele is believed to be independent of cAMP in activating the cAMP-CRP-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2163388" target="_blank">2163388</a>]<br></li>
<li>More recent studies showed , however , that the synthesis of the flagella , of the Che and was proteins , of the EIls cAMP-CRP dependent , whereas PTS-dependent chemotaxis per se was normal in the absence of IIAGc , adenylate cyclase , cAMP , and the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>consequence of the pH , regulation of malT The crp '' mutation affects the coding sequence of crp and makes the CRP ' ) protein able to activate CAMP - As the pH , modulation of the plasmid-borne ( malTpl CRP-dependent promoters even in the absence of cAMP malTpl0 ' - lad + ) 1 operon fusion was abolished ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>As predicted from these results , an equivalent reduction of sfm activity was observed in a Dcya or strain ( cya encodes the adenylate cyclase ) , therefore indicating that the CRP effect was also cAMP dependent ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>To test whether this effect is mediated by cAMP-CRP , we measured the activity of 0-galactosidase in a strain containing the Xd4 ( dadA ' - lacZ ) 1 transcriptional fusion and its Acya and Acrp derivatives ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>To check whether cAMP and CRP are involved in the production of extracellular AI-2 activity , we compared the extracellular AI-2 activity produced by W3110 with that of isogenic crp and cyaA mutants grown in LB ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>To examine the dependence of Pugp2 on cAMP and CRP , we introduced pKD5061 into the nearly isogenic strains with deletions in cya or crp and assayed the CAT activity in these strains grown in excess or limited carbon medium ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>CRP gave a clear cAMP-dependent footprint at around -40 and -90 in both the wild type and Oc mutants when saturating concentrations of both CRP and cAMP were used ( Fig. 4 , left panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>( Middle and right ) Autoradiographs show the effect of CRP ( 2 , ug/ml ) and increasing concentrations of cAMP ( shown at the top ) on DNase I footprint analyses of the wild-type and O ' promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Under these conditions , cAMP-CRP and IHF do not contribute to the specific regulatory pattern of starvation induction ( which is no longer found in a strain that is deficient for as and Lrp but has wild-type cya , crp , himA , and hip alleles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Since both the crp deletion and the cyaA deletion caused about a fourfold decrease in gcvT-lacZ expression compared to the level of expression in the wild-type strain when it was grown in GM medium , we wanted to conﬁrm that the cAMP effect was mediated through CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>For cAMP-CRP , this relief seems to be total , since wild-type and cya or crp strains reached similar final levels of osmY expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Since cstA expression is dependent on cAMP-CRP ( 42 , 43 ) , lower expression was observed throughout the growth curve in both the wild-type and csrA mutant strains under this growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>These results indicate that , unlike prpBCDE expression , expression of prpR is not regulated by propionate or its catabolism but is regulated by glucose or glycerol availability , suggesting a classic catabolite repression mechanism mediated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>These results give further support to the previous proposal that glucose repression is also mediated by another mechanism independent of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Although the intracellular CRP level could be another determinant of the level of active CRP-cAMP complex , it had not been seriously tested whether the level of CRP is involved in the modulation of catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>These results taken together suggested that one or more promoters located on the HindIII-EcoRV segment were sensitive to glucose ( or catabolite ) repression and require the cAMP-CRP complex for maximum activity , although glucose repression could not be reversed by exog-enous cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>Therefore , both the Pa and Pz promoters seem to be activated by the cAMP-CRP complex , which explains their catabolite repression by glucose reported above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>This further supports the view that dctA is subject to CRP-mediated catabolite repression , which is consistent with studies by Kay and Kornberg ( 20 ) showing that CRP , cyclic AMP ( cAMP ) , and glucose regulate Dct activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>This catabolic promoter is also subject to catabolic repression in the presence of glucose mediated by the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>This result is probably the superposition of two effects : the lack of inducer ( presumably GlcNAc ) and catabolite repression via the cAMP/CRP complex , as was found for enzyme IIG1 '' and enzyme IIMannose ( Rephaeli and Saier , 1980 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3284790" target="_blank">3284790</a>]<br></li>
<li>This indicates that the tna promoter is not regulated by tryptophan but that it is sensitive to catabolite repression , presumably mediated through cAMP-CRP ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3902796" target="_blank">3902796</a>]<br></li>
<li>This suggested that the hyb operon is subject to catabolite repression mediated by the cAMP-CRP system ( Busby & Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>These regulatory proteins seem not only to be provided for the repression that is removed by cAMP-CRP , but also to have abilities to control their own repression in response to the concentration of substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>We proposed that glucose causes catabolite repression by reducing both intracellular CRP and cAMP levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Clearly , more studies are needed to characterize cAMP-CRP independent glucose repression by EIICBGlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>In the previous study , we showed that glucose causes catabolite repression of the lactose operon by reducing both the CRP and cAMP levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>In addition , recent studies on catabolite repression caused by non-PTS sugars such as glucose-6-phosphate or gluconate concluded that it is the amount of CRP , as well as cAMP , that is altered in response to non-PTS sugars ( 14 , 16 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>J Mol Biol 165 : 197 -- 202 Tagami H , Inada T , Kunimura T , Aiba H ( 1995 ) Catabolite repression in Escherichia coli cells lacking cAMP is mediated by ﬂuctuations in CRP levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1474">
Tagami , H. , Inada , T. , Kunimura , T. , and Aiba , H. ( 1995 ) Glucose lowers CRP * levels resulting in repression of the lac operon in cells lacking cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></label>
<input id="act_similarity_1474", type="checkbox">
<div class="ACT_similar">
<li>Glucose lowers CRP * levels resulting in repression of the lac operon in cells lacking cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Glucose lowers CRP levels resulting in repression of the lac operon in cells lacking cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li></div>
<li>First , TcbC could cause increased cAMP levels within glucose-grown cells , in which case its effect would be indirect or , secondly , TcbC could increase the sensitivity of the tdc operon operator sequence to cAMP ± CRP and potentially its interaction with other components of the transcriptional initiation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>A possible explanation for this is that positive autoreguiation ma ensure maximum production of CRP during glucose-starvation , since the co-ordinate increase in both CRP and cAMP levels is apparently the most efficient way of activating catabolite-sensitive operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>As glucose is consumed the intracellular levels of cAMP rise , and an active cAMP-CRP complex forms , which then modulates the activity of numerous genes at the transcriptional level ( 24 , 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Previous studies showed that Dcu activity is only slightly reduced by glucose and is unaffected by cAMP or a cya mutation , suggesting that CRP does not regulate Dcu synthesis ( 8 , 9 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In summary , we conclude that the csiD promoter has an intrinsic preference for EsS , but that also protein factors such as cAMP-CRP , Lrp and probably H-NS as well as DNA conformation contribute to its strong EsS selectivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In the case of cAMP-CRP-dependent reactions with lac and gal promoters , the promoter DNA was incubated with 2.5 pmol CRP protein in the presence jM of 10 cAMP prior to the addition of each holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>The x-ray structure of αCTD bound to DNA ( and to the cAMP-receptor protein , CRP ) in the context of the lac promoter ( 12 ) is in complete agreement with the predictions of the biochemical and genetic analyses and brings our understanding of the αCTD-DNA interaction to the atomic level ( Figure 2 , see color insert ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15568992" target="_blank">15568992</a>]<br></li>
<li>[ Key Words : CRP -- cAMP ; osmotic gene regulation ; protein -- DNA interactions ; ProP transporter ; lac promoter ] Received September 3 , 1999 ; revised version accepted October 13 , 1999 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>These results , together with measurements of the in vivo psiE promoter activity under various conditions , show that expression of the psiE gene is under direct positive and negative control by PhoB and cAMP-CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>cAMP-CRP is shown to-bind to the upstream region of the promoter of the lsr operon and works with the LsrR repressor to regulate AI-2 uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>For example , LacI in Group 4 is well known to-bind to the promoter region at the low concentration of some inducers , lactose or certain other galactosides , while it is released from DNA at the high concentration of the inducer even in the absence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>IRL Press Limited , Oxford , England Results Organization of the DNA region upstream the RNA start at malT As already shown in Chapon and Kolb ( 1983 ) , the malT promoter is extremely weak in the absence of added CRP - cAMP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>cAMP did not restore normal GRDC to the P ( 133 to guaB 36 , G-122C ) promoter , which lacked a functional CRP site , nor did it result in repression of this promoter at low growth rates ( compare Fig. 6C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Experiments with primer extension and operon fusions suggested that the major promoter , P1 , of the gene is repressed by GntR and the repression is relieved by the addition of gluconate , and that cAMP-CRP functions as a positive regulator for the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>These in vivo results clearly indicate that both proteins , FIS and cAMP -- CRP , are required for crp repression , since the lack of either transcriptional-regulator leads to a derepressed crp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Employing crp promoter -- lacZ fusions , we demonstrate that both FIS and cAMP -- CRP are required to keep the crp promoter in a repressed state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>These results are consistent with the Northern analyses and indicate that both FIS and cAMP -- CRP are required throughout the growth-phase to keep the crp promoter in a repressed state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Synergistic regulation of bgl by BglJ -- RcsB and CRP -- cAMP BglJ -- RcsB and CRP synergistically activate the bgl P2 promoter and relief repression by HNS and StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>At high cAMP -- CRP concentrations , the repression of the crp1 promoter was relieved ( Figure 6 , lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Addition of cAMP -- CRP in combination with FIS slightly alleviated the repression of the crp1 promoter , but not of the divergent promoter , indicating that the divergent promoter is repressed by FIS even more efﬁciently than is the crp1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Transcription from the promoter would be blocked by cAMP-CRP under low-glucose and HP conditions , because of a high concentration of cAMP-CRP and low concentration of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>cAMP - CRP enhances-the-activity of P1 , possibly by enhancing the isomerization of RNA polymerase at this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>lacZP acted as a control promoter , which is activated by cAMP ± CRP only when transcribed by RNP carrying an intact a-subunit ( Igarashi and Ishihama , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Binding of CRP -- cAMP to the galP1D4 promoter is also osmotically sensitive , but this promoter is not significantly derepressed by an osmotic upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Transcription levels were normalized to the divergent p82 promoter , which does not respond to RhaR , CRP , L-rhamnose , or cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Activation by cAMP -- CRP is not expected to inﬂuence tsp selection ; it increases promoter strength without affecting the tsp in the P1galE promoter ( Lewis and Adhya , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>To test for the roles of CRP and cAMP on gapB P0 promoter activity , we looked for the amount of gapB transcripts in E. coli POP4129Dcrp , which is deﬁcient in CRP production , and in the E. coli TP2006cya mutant strain , defective in cAMP production ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>Moreover , the crp promoter on a low copy number plasmid was derepressed in CSH50 ∆ ﬁs after overnight culture ( measured by β-galactosidase activity ) , although these cells overproduced CRP and cAMP was added to the culture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>CRP -- cAMP functions to negatively regulate this promoter ; however , we were only able to demonstrate a small increase in b-galactosidase activity by galP1D4 -- lacZ after addition of 0.5 M NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>) Thus , the CRP protein encoded by the crp * GQ141 allele activated the lac promoter up to ca. 30 % of its full activity in the absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2163388" target="_blank">2163388</a>]<br></li>
<li>When we used E & 38 in place of Ea70 , we unexpectedly found that the P1 promoter was efficiently recognized by Ea38 even in the absence of CRP and cAMP ( essentially the same results have been found by others ; 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>Although the down-regulation of crp expression by glucose can be explained by the autoregulatory loop , we also noticed that glucose slightly but consistently reduces the level of CRP independently of cAMP or the crp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Since the malT promoter is controlled by the cAMP-CRP complex , we considered the possibility that the decreased expression of MalT observed in hns and hns-stpA strains was in part also due to a reduced level of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Significantly , the transcripts generated by the Pr promoter were not detected in the strain lacking CRP , indicating that the cAMP -- CRP complex is required to activate mhpR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>When the gntT promoter fragment was incubated with both GntR and CRP-cAMP , a ternary complex consisting of one GntR molecule and one cAMP-CRP complex bound or two GntR molecules and one cAMP-CRP complex bound was formed ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Third , only the cAMP-CRP complex , but not cAMP or CRP alone , could directly bind to the promoter of the hfq gene ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>However , when cAMP -- CRP complex was added to the reaction mixture , the Pr activity was stimulated and a 132 nt transcript generated from the Pr promoter was detected ( Fig. 3 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>On induction , GutR is inactivated and thus increases the synthesis of the activator GutM from the main promoter , a process which is dependent on the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>TdcA-dependent anaerobic induction is atypical , in that it requires the cAMP ± CRP complex to-bind near the 235 region of the tdcA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>It has been well known that the cAMP CRP complex binds to the ptsG promoter centered at 40.5 and 95 , respectively ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>Therefore , these results indicate that the cAMP -- CRP complex is mediating the glucose effect at the Pr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Surprisingly , these experiments also revealed that P3 , a promoter that we found to be inhibited by cAMP-CRP in vitro ( Fig. 1 ) , is positively regulated by this complex in vivo ( Fig. 5B , lanes a and b , band 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>CAT fusion analysis revealed that the operon has a promoter for gntR and another for gntKU , and that the gntR gene is constitutively expressed , while that of gntKU is regulated positively by the cAMP-CRP complex and negatively by GntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The PK promoter is regulated positively by cAMP-CRP complex and negatively by GntR , and the PR promoter is constitutively expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>If so , we would expect that GalS will be induced at lower galactose concentrations than GalR , since the mgl operon is the more efficient transport system at low galactose concentrations , ( ii ) The galS gene is autoregulated and its expression is CRP cAMP dependent , while gaIR expression appears to be constitutive -LCB- Weickert and Adhya , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>The repressing effect of cAMP-CRP may be responsible for the low increase of osmY expression in stationary-phase cells starved for glucose in minimal medium ( which is in pronounced contrast to a more than 30-fold induction in LB medium ) , because the at onset of starvation the synthesis of cAMP is drastically stimulated under these conditions ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The cAMP/CRP complex is a positive transcriptional-regulator of a number of catabolic operons , and as such , plays a role in catabolite repression , whereby secondary carbon sources are not catabolized in the presence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>The sdaB gene is expressed only in rich medium in the absence of glucose , under the control of the cAMP/CRP catabolite repression system [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>CRP is responsible for the repression in complex media of gadA transcription The difference in the induction pattern observed between minimal and complex media indicated that either there is an inhibiting molecule ( s ) in LB or the glucose present in EG is helping to induce gadA and gadB in response to acidic pH. The complex versus minimal media effect on gad expression suggested the involvement of CRP and -RCB- or cAMP as a negative regulator of gadA transcription in complex LB media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>The ; 30-fold postexponential repression of dctA expression by glucose in L broth observed under aerobic conditions was largely reversed by the addition of cAMP ( from 0.17 to 2.9 mmol/min/mg ) , which provides a strong indication that dctA is activated by the cAMP-CRP complex ( Fig. 8C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>The discoordinate expression is more severe in the absence of a functional cAMP -- CRP complex and can be suppressed in glucose-grown cells or in cya − strains by the addition of cAMP to the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>The P2 , P3 and P4 transcripts become detect-able in the poorer media and with glyoGiol as carbon source -- the medium that results in the highest concentration of cAMP-CRP -- all the three upstream transcripts are clearly present ( Fig. 2A , lane 6 ; Fig , 2B , lane 9 ) , On fhe other hand , the P1 transcript is present at all growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The same initiation site was seen in CytR1 and CytR2 strains ( data not shown ) and also under growth conditions that should lead to high and low levels of cAMP and thus of CRP ( Fig. 2 , ﬁrst two lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The results of the protein nitrocellulose Wlter binding assay show that in the absence of cAMP WT and mutant forms of CRP bound duplex LCAP with similar aYnities having association constant ( Ka ) values that ranged from 2.0 # 105 M ¡ 1 to 9.6 # 105 M ¡ 1 ( Fig. 3 , Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>The speciﬁc activator ( NtrC-phosphate ) is not essential for CRP -- cAMP-mediated inhibition of glnAp2 At high concentration , NtrC-phosphate can activate glnAp2 in a binding site-independent fashion ( Reitzer and Magasanik , 1986 ; see also above ) , suggesting that the CRP -- cAMP complex might somehow modulate the quantity or activity of NtrC-phosphate through the complicated Ntr system which , in turn , results in the repressive downregulation of glnAp2 lacking NtrC-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The high intracellular levels of cAMP and therefore of cAMP-CRP , that are required for the relatively inef ® cient binding of cAMP-CRP upstream of the csiD promoter ( Figure 6 ) , are probably present only in response to starvation for carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The activation process of CRP upon binding to cAMP involves unique features of allostery distinguished from those of other enzymes and membrane receptors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The larger , N-terminal domain ( NTD ) , which is predominantly β-stranded , is basically responsible for CRP dimerization ( see Fig. 1 ) , and the cAMP binding to this domain results in the functional activation of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The strong CRP binding site is correctly positioned to function in aspA transcriptional activation by the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The third important observation was the existence of a third binding site , CRP-3 , for cAMP-CRP that is absolutely required for full repression but dispensable for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The normal osmotic control by the CRP * mutant also rules out the possibility that changes in cAMP levels are responsible for the osmotically sensitive binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The activity of CRP is triggered by binding of the second messenger cAMP , whose conversion from ATP is catalyzed by the adenylate cyclase enzyme CyaA in response to glucose-starvation and other stresses ( references 20 and 35 and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The binding of the cAMP-CRP complex is essential for full induction of gntT , and this binding is neither cooperative nor inhibited by GntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>The greater sensitivity of CRP -- cAMP binding to increases in medium osmolarity in minimal media correlates with the greater induction at lower osmolarity in minimal as compared to complex media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>A in Figure Overproduction of CRP by the Recombinant Plasmids It is of interest to measure the level of CRP in cells harbouring the 42.1 90.2 0.2 0.2 * The assay for cAMP binding was carried out as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The wavy underline shows the sequence for cAMP - CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>The activity of puriﬁed CRP was determined by a cAMP binding assay as follows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The cAMP -- CRP site is in blue lettering .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>The CRP site was sensitive to DMS modification 15 min after salt addition as observed previously but became increasingly resistant to DMS modification after 30 min , indicating rebinding of CRP -- cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The b-galactosidase synthesis from the trxA-lac fusion lacking a plausible cAMP-CRP binding site was not lowered in the presence of lactose or in the addition of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>The twofold inhibition of P2 by cAMP in vivo may reflect the relative binding affinities of cAMP - CRP and RNA polymer-ase competing for the same site and their concentrations in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>The binding of CRP-cAMP to the CRP site allows RN polymerase to-bind predominantly at Pd and the divergent gene is actively transcribed ( Figure 9B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>The binding of cAMP-CRP to the synthetic CRP site is comparable to that of CRP-2 , whilst binding to CRP-1 remains unaltered ( tested for constructs pl3-1361 +22 , p13-1361 + 18 , pl3-1362-1-22 and p13-1364-t-22 , data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>The recruitment is specific for this site and binding of CRP is dependent on cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The binding of CRP to this site is strictly cAMP-dependent ( fig. 2 B , lanes 2 , 3 ; 3 B , lanes 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The CRP group mainly contains transcription factors that regulate catabolite-sensitive genes in response to glucose-starvation as signalled by cAMP ( Busby & Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The addition of CRP and cAMP , however , increased the P1 promoter activity dramatically ( Fig. 5 , lane 2 ) , suggesting that the transcription initiation at ptsG P1 is totally dependent on cAMP CRP as expected from a previous report ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>The concentration dynamics of cAMP , rather than CRP , appear to be more closely related to P1 transcription dynamics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>The bound cAMP CRP would then stimulate gal transcription from the P1 promoter only in this population .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>The cAMP -- CRP complex stimulates transcription from the downstream promoter P1 and inhibits transcription from the upstream promoter P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>The divergent RNA requires cAMP-CRP for expression ; it strongly blocks crp transcription ; and , if this RNA is not made , cAMP-CRP inhibition of crp expression is abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The formation of this structure by the activation of the divergent promoter by cAMP-CRP could therefore lead to very early termination of crp mRNA synthesis and could account for the observed inhibition of crp transcription by divergent RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The autoradiograph shows the effect of CRP ( IOM-9 ml ' ) and increasing concentrations of cAMP upon digestion by DNasel ( 5 \ i.g ml ' ) of the BssHI-Scal restriction fragment spanning the cdd promoter region -LCB- [ ^ ^ P ] - labelled at the OssHII end ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>The deoP2 promoter sequence ( ± 105 to ± 25 ) is 5 cents - AATTATTTGAACCAGATCGCATTA-CAGTGATGCAACTTGTAAGTAGATTTCCTTAATTGTGATGTG - TATCGAAGTGTGTTG-3 cents , in which the two cAMP ± CRP sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The major goal of this study was to quantify the contribution of transcriptional regulation at the cya promoter to the downregulation of cAMP levels by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>The developing model for the control circuit is that , when E. coli is actively growing under conditions where cAMP levels are high ( e.g. , exponential-growth in media like LB medium ) , CRP will repress gadX by inhibiting RpoS production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The maximal stimulation of expression by cAMP and CRP observed in the experiment shown in Fig. 2 was 26-fold for glgC and 10-fold for gigA ; ppGpp increased the expression 3.6 - and 1.8-fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The addition ot cAMP and CRP to the in vitro reaction resulted in a 3 to 10 - fold stimulation in the amount of IlvB leader RNA synthesized ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>The observed CRP regulation is also consistent with the inability of mutants lacking phosphotransferase system components to grow on succinate ( 54 ) and the reestablishment of Dct activity in cya ( adenyl cyclase ) mutants by the addition of cAMP to the growth medium ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>The synthesis of most of these proteins is also affected strongly by positive regulation through the cAMP -- CRP complex and the enzymes are highly expressed under appropriate growth conditions ( i.e. conditions in which rpoH-P5 is activated ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The result is also consistent with the recent finding that the starvation-induced synthesis of GroEL and DnaK proteins is independent of cAMP and CRP ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>DISCUSSION The cAMP CRP complex binds to the promoters of many operons , e.g. , lac , srl , etc. , and dramatically stimulates their rate of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>The cAMP-CRP complex binds the fadL , fadD and fadH promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The complex of cAMP and CRP binds to a region of the DNA near the beginning of a regulated gene and enhances transcription of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>The cAMP-CRP complex thus seems to-bind to the putative sequence and to enhance gntT transcription ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>The transcription of cyaA is negatively regulated by the cAMP-CRP complex ; thus , a large increase in cAMP synthesis occurs in a crp mutant [ 43 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>The CRP -- cAMP complex inhibits s - dependent glnAp2 strongly , by 54 21-fold ( see Results obtained from glnAp1 ` dead ' construct various contributions in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The results are consistent with our previous studies on a heterologous system , in E. coli are all downregulated by the CRP -- cAMP complex ( glnAp2 , this work ; glnHp2 , unpublished observations ; glnK , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The genes encoding enzymes involved in glycogen metabo-lism ( glycogen synthase and glycogen phosphorylase ) are linked on a single operon that is regulated by the cAMP CRP z complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9334229" target="_blank">9334229</a>]<br></li>
<li>The regulation by the cAMP-CRP complex was investigated by comparing the strains cultured in LB and in LB supplemented-with-glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>The positive effect of the cAMP-CRP complex , however , is seen only in concert with DsdC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>The cAMP-CRP complex stimulates the expression of lsrR and the lsr operon , while LsrR represses their expression [ 14 , 15 ] , suggesting that cAMP , the CRP , or both may be affected by sdiA-ydiV double mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>The GntH molecule bound to HRII strongly recruits cAMP-CRP complexes to activate the expression of the gntV and idnD genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>The expression of tnaA is under cAMP and CRP regulation ( 78 ) and is also inducible by high pH ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>The mel locus is induced by several a-D-galactosides ( 2,3 ) and its expression is controlled by the cAMP-CRP regulatory circuit ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3031590" target="_blank">3031590</a>]<br></li>
<li>The expression of fMet-Gln , the amino-terminal dipeptide of gigA was enhanced 2.4 - to 2.8-fold by cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The synthesis of fMet-Val directed by the HpaI fragment was enhanced 2.5-fold by cAMP and CRP , versus the 7 - to 10-fold effect with plasmid pOP12 ; however , cAMP and CRP no longer enhanced the synthesis of fMetGln from this fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The synthesis of numerous proteins in E. coli appears to be negatively controlled by cAMP-CRP ( 19 ) , but in only a few systems has this effect been studied at the molecular level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>As many factors are involved ( CRP , cAMP , s , Es , NtrC , 54 54 NtrC-phosphate , NifA , etc. ) , it is difﬁcult to quantify their The precise mechanisms involved in inhibition by CRP -- cAMP at glnAp2 are not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1586">
The experiments with CRP contained 150 μM cAMP , a concentration that maximizes the fraction of CRP in its active form ( 0.64 ( 0.2 ) under these buffer conditions ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></label>
<input id="act_similarity_1586", type="checkbox">
<div class="ACT_similar">
<li>The concentration of cAMP used in these experiments ( 150 μM ) maximizes the fraction of CRP dimer in the active form as CRP ( cAMP ) 1 ( 0.64 ( 0.02 ) ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>The final concentration of cAMP used in the reactions was 0.2 mM , whereas CRP was used at a final concentration of 500 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The components cAMP , CRP , NtrA , and NtrC610 were added at 5 p.M , 2 p.g per reaction , 80 p.g per reaction , and 800 p.g per reaction , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The concentration of cAMP was 100 , uM ; CRP , NtrA , and NtrC610 were added at 1 , ug , 80 ng , and 800 ng per reaction , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>1981 ) , The activity of CRP is reguiated by the intracellular concentration of cAMP which is modulated by the carbon source on which cells grow ( Pastan and Adhya , 1976 ; Uilman and Danchin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The in vitro assays were performed in the absence of CRP and cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The ability of DksA to substitute for a complete absence of ppGpp argues that DksA and ppGpp are not cofactors ( like cAMP and CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18532980" target="_blank">18532980</a>]<br></li>
<li>The araCi mutants , therefore , act trans to confer the ability to utilize L-arabinose in the absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>The addition of cAMP and CRP increased the pOP12-directed synthesis of ADPglucose pyrophosphor-ylase ( glgC ) and glycogen synthase ( glgA ) but not that of branching enzyme ( glgB ) or aspartate semialdehyde dehydrogenase ( asd ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The mutants have gained the ability to utilize L-arabinose in a cya - crp - background but not other carbohydrates that require cAMP and CRP for utilization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>The glucose - and cAMP-medi-ated activations occur independently of each other ( 2 , 4 , 5 ) , even though glucose lowers the level of CRP and cAMP in the cell ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464268" target="_blank">10464268</a>]<br></li>
<li>Fraction of CRP as Functionally Active CRP ( cAMP ) -- The 1 role of cAMP as an allosteric-effector of CRP functional states is well known ( 26 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The lactose operon absolutely requires CRP and cAMP to be expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>b and c The levels of cAMP and CRP in E. coli KK3131/pACYC-PuuP and E. coli KK3131/pACYC-PuuP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>The cst genes including cstA are activated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>The concentrations of cAMP and CRP are indicated above the panels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>The final balance between Pc and Pd occupancies would be determined by the concentrations of CRP and cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>The cAMP concentration was 50 µM whenever CRP was employed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The effect of cAMP and CRP on SecB synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023186" target="_blank">9023186</a>]<br></li>
<li>The last group depicts a fully active , namely `` on '' state including the cAMP-bound CRP ( cAMP -- CRP ) and constitutively active mutants : T127L/S128A-CRP ( CRP LA ) [ 24 ] , G145S-PrfA ( PrfA 145 ) [ 31 ] , and N127L/S128L-ChCooA ( ChCooA LL ) [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The wild-type PrfA and PrfA 145 , both of which are active , assume the same orientation of αD as that of cAMP -- CRP , whereas the DdCprK and CHPA-DhCprKox , both of which are inactive , show broader αC-to-αD angles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Two hypotheses could account for these results : ( i ) osmoregulation of osmY requires yet another unidentified regulatory mechanism that is totally independent of as , Lrp , cAMP-CRP , and IHF ; or ( ii ) osmotic induction could be mediated by several independently and additively acting components , some of which could be those identified in the present study as being involved in the regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Stationary-phase induction of osmY in rich medium is mediated by the combined action of as , Lrp , cAMP-CRP , and IHF , with the latter three proteins acting as transition state regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>None of the several regulatory genes mentioned above seems to be essential for hyperosmotic induction of osmY , which could still be observed in mutants deficient in as , Lrp , cAMP-CRP , or IHF and even in strains carrying rpoS lrp or rpoS hip double mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>When the fragment was incubated with 50 nM of CRP and 200 AM of cAMP prior to enzyme and substrate addition , abortive synthesis of ApUpC or CpUpUpU was completely abolished , while the specific activity in the steady-state release of ApUpU was markedly increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Figure 3A shows that , given constant cAMP levels ( 50 mM ) , just one band was retarded when the CRP concentration was very low ( 0.25 ng/ml ) and that an additional , more slowly migrating band appeared when higher CRP levels were used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Since in vivo regulation by CRP is observed in GM medium , where the cAMP concentration is low , a gel mobility shift assay was performed with 5 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>WT CRP , E72D , E72Q and R82Q CRP were run without cAMP ( lanes 3 , 6 , 9 and 12 ) , with cAMP ( lanes 4 , 7 , 10 and 11 ) and with cAMP and RNAP ( lanes 12 , 13 and 14 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>An analysis of dipeptides formed with pOP12 as the genetic template confirmed previous results which showed that the synthesis of the amino-terminal dipeptide of glgC was enhanced by cAMP and CRP ( 7 - to 10-fold in our experiments ) and that neither cAMP nor CRP alone was sufficient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>We found that the degree of overproduction of cAMP in crp ~ cells is reduced to 50-fold , suggesting that the negative regulation of cya transcription is indeed involved in the downregulation of cAMP production by CRP , while the major regulation of cAMP production by CRP is exerted at the posttranscriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Since the cAMP added exogenously is sufﬁcient to overcome the deletion of the cyaA gene , the results indicate that the cAMP level was probably sufﬁcient in the wild-type lysogen to allow the maximum range of regulation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A deletion of the cyaA gene also resulted in a fourfold decrease in b-galactosi-dase levels ( Table 2 ) , indicating that CRP requires cAMP for regulation of the gcvT-lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>All the putative target genes examined were indeed found to be under the control of CRP , classified in group-A in Table 2 , supporting the prediction that the Genomic SELEX screening allows the identification of regulation targets of cAMP-CRP with high accuracy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Key words Adenylate cyclase · cAMP synthesis · CRP · Transcriptional regulation · Posttranscriptional regulation Introduction Cyclic AMP is an important regulatory molecule involved in various cellular activities in many organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>This result suggested that the putative CRP box does not play a direct role in serC ( pdxF ) - aroA regulation and that the negative regulatory effects of CRP-cAMP are indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>However , when the levels of cAMP increase and CRP is inactive ( i.e. growth on acetate ) the regulation due to IclR becomes predominant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17426033" target="_blank">17426033</a>]<br></li>
<li>This negative regulation of cAMP level by CRP could be crucial to cell physiology although the biological signiﬁcance of this regulation is not fully understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Among these regulators , it is likely that the observed up-regulation in acetate and glycerol was mediated by cAMP-CRP , as cAMP level should be increased in nonglucose media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753455" target="_blank">10753455</a>]<br></li>
<li>Thus , the level of cAMP present in GM medium appears to be sufﬁcient for regulation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Quantitative analysis of cya mRNA , cya protein , and cAMP establishes that transcriptional control is partly responsible for the down-regulation of cAMP levels by CRP , although the major regulation is exerted at posttranscriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Our data indicate that elimination of transcriptional regulation of cya by CRP reduces the extent of overproduction of cAMP in a crp ~ background from 200-fold to 50-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>To assess the role o transcriptional regulation of cya in mediating the strong downregulation of cAMP production by CRP , the cAMP levels of cells harboring pSE3 were determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>This means that transcriptional inhibition of cya by CRP is indeed involved in the regulation of cAMP levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>So far , no experiment has been done to test directly whether the transcriptional regulation of cya contrib-utes to the repression of cAMP level by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Alternatively , transcriptional inhibition of cya by CRP may play little part in the regulation of cAMP level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>We conclude that transcriptional regulation of cya does indeed play a role in the down-regulation of cAMP production by CRP , although the major regulation is exerted at the posttranscriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>We investigated the quantitative contributions to the cAMP level of transcriptional and posttranscriptional regulation of adenylate cyclase by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>These in vitro results conﬁrm the previously reported in vivo observations that expression of these three genes is highly dependent upon cAMP and CRP ( Weickert and Adhya , 1993b ; Geanacopoulos and Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Our ® nding that cAMP-CRP not only activates csiD expression , but does so to different extents when EsS and Es70 are used in in vitro assays ( see discussion above ) , indicates that CRP contributes to sigma factor selectivity at the csiD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>1981 ) , Four different transcripts originate in its upstream regutatory region , and it has been shown that in vitro expression of gIgC is stimulated by cAMP/CRP and weakly stimulated by ppGpp ( Romeo and Preiss , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1324388" target="_blank">1324388</a>]<br></li>
<li>Because Sxy is thought not to-bind DNA , we also examined gene expression in cyaA cells to test whether Sxy might act by overriding-CRP 's dependence on its allosteric-effector cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>This is in contrast with what is found for S. enterica serovar Typhimurium , where CRP seems to be essential for stationary-phase expression ( 23 ) , but is in agreement with results for E. coli arguing against a role of cAMP in the regulation of an ast-lacZ fusion ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>This work represents an initial contribution to the elucidation of the role of cobB and yﬁQ gene products in the control of the metabolism of E. coli , describing for the ﬁrst time how their expression is regulated and co-ordinated by central metabolism regulator CRP and the concentration of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>Moreover , this activity decreased immediately after the addition of cAMP to the culture ( Fig. 5 ) , indicating that cAMP-CRP negatively controls osmY expression by an rpoS-independent mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Effect of exogenous cAMP on the expression of serA : : lacZ fusions If serA is indeed subject to regulation by cAMP/CRP , one would expect that the addition of cAMP to glucose ¬ Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Both genes are regulated by CRP , which activates their expression in response to the elevation of cAMP levels upon glucose-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25524697" target="_blank">25524697</a>]<br></li>
<li>Expression is also inﬂuenced by growth on carbohydrates through CRP , which dampens expression of RpoS , an effect more evident in LB-grown cells , where cAMP levels are high , than in cells grown on glucose , where cAMP levels are low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Finally , we examined the requirement of cAMP -- CRP for in vivo expression of CyaR by monitoring the cellular levels of the sRNA in strain SØ928 cultivated with different carbon sources , as well as in a mutant strain defective in cAMP synthesis ( cyaA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>Expression of the enzymes encoded by the galETKM operon can be described by the sum of the P1galE and P2galE input functions , which results in a ` D-galactose ' gate , with lower level and dis-coordinated expression of the proteins in the absence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>cAMP-CRP , a positive regulator , stimulates expression of both operons in the absence of glucose , while LsrR prevents their expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In the absence of glucose , cAMP-CRP is needed to stimulate lsr expression , while LsrR represses its expression in the absence of the inducer phospho -- AI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>No increase in the gene expression , however , was observed in the absence of CRP or cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Regulation of regulators Autoregulation of GalR allows its low-level expression in the absence of cAMP -- CRP and D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>This indicates that a functional CRP is required for cAMP to exert its negative effect on ODC and A D C expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>This negative influence of cAMP can be attributed to the presenceof os , which plays a positive role in csiE expression and which is itself under negative control by cAMP-CRP ( Lange and Hengge-Aronis , 1994a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>This suggests that cAMP may act , via CRP , as a positive regulator of acrAB expression during carbon limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11886757" target="_blank">11886757</a>]<br></li>
<li>This variation is further extended by the fmding that cAMP/CRP can act not only as a positive effector but also as a negative regulator of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Although , it is generally believed that CRP is synthesized constitutively and its function is controled by the level of cAMP ( 11 ) , little is known about the expression of the crp gene , which codes for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>These results suggest that the expression of the luxS gene is negatively regulated by cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>These results suggest that cAMP together with CRP can have a negative inﬂuence on the trxA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>Although CRP and cAMP are necessary for uidA expression ( Novel 1973 ) , the precise mechanism whereby they act in vivo remains unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>Interestingly , expression of both genes is activated by cAMP-CRP and repressed by CsrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>Both cAMP and CRP are required for expression of glgS and PTS ( phosphotransferase system ) - related genes required for normal glycogen production [ 22,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>Then cAMP-CRP complexes bind to cCI and cCII and activate the gntV and idnD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>CsrA and cAMP-CRP both activate motility in E. coli through effects on ﬂhDC expression ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>Therefore , changes in dna K expression are probably not involved in the CRP -- cAMP-mediated downregulation of glnAp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Effects of ArcA , FNR , cAMP , and CRP on dctA-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>On this basis we conclude that cAMP -- CRP also stimulates galE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>( i ) cAMP and CRP enhanced glgC expression more strongly than gigA expression ( approximately threefold ) and had weak or negligible effects on glgB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Instead , cAMP-CRP and IHF seem to determine the absolute level of osmY expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>cAMP and CRP , therefore , are required for ope-ron expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>cAMP and CRP negatively regulate the expression of luxS but not pfs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>CRP requires cAMP to regulate gcv expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Moreover , since Lrp can positively modulate csiD expression under certain conditions in vivo ( Figure 2 ) , effects of Lrp were also tested besides those of cAMP-CRP ( Figure 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>By 54 activating glnAp1 and downregulating glnAp2 , the overall effect of CRP -- cAMP on glnA expression is an approximately fourfold reduction , which correlates with the reduction of g-glutamyl transferase activities regulon , NtrB ( also called NRII ) and NtrC ( also called NRI ) ( Reitzer and Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Regulation of pga Expression by PaaX and the cAMP-CRP Complex -- To demonstrate antagonistic regulation of pga expression by PaaX as repressor , and CRP as activator , we constructed 3 - cMyc-tagged derivatives of recombinant paaX and crp and performed ChIP assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Furthermore , the introduction of a cya mutation also abolished TcbC-dependent induction of tdc expression ( data not shown ) , which confirms the dependence on CRP and cAMP for regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Thus , the observed expression patterns for pckA , tnaA , aspA and others under conditions of aerobic acetate formation can be explained by cAMP-CRP dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>When cells contained the crp * allele , the synthesis of spot 42 RNA was low even in the absence of cAMP ( Fig. 1 , lane 7 ) ; the further reduction in spot 42 RNA upon addition of cAMP was consistent with the fact that CRP * is in a partially active-conformation in the absence of cAMP ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>Effects of cAMP , CRP , and ppGpp on the synthesis of glycogen biosynthesis enzymes in S-30 extracts Presence ( + ) or absence ( - ) of : Expression '' of : cAMP CRP ppGpp IA gg gg ( 100 , uM ) ( 28.6 , ug/ml ) ( 250 FM ) lC ggA gIgB - - - 1.0 1.0 1.0 + - - 2.5 1.3 1.0 + + - 6.3 2.2 1.0 - - + 1.6 1.1 1.1 + - + 6.0 2.4 1.0 + + + 18.8 4.8 1.3 `` Results are expressed as the relative incorporation ( in counts per minute ) of [ 35Slmethionine for each gene product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Since cya + crp + strains containing deletions of araC are unable to use L-arabi-nose , Ara + revertants from strains SB5611 and SB5612 must be constitutive as well as allow the expression of araBAD in the absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Ara + revertants of a cya - crp - strain were isolated on 0.2 % minimal L-arabinose plates , conditions which require the entire ara regulon to be activated in the absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>DISCUSSION The isolation of Ara + revertants of a cya-crp strain under conditions in which the entire regulon must be inducible in the absence of cAMP and CRP has been described .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Mutants deficient in adenyl cyclase ( cya - ) , the enzyme which synthesizes cAMP , or CRP ( crp - ) are unable to utilize a variety of carbohydrates , including L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Since the recipient is cya - crp - , enzyme production must also occur in the absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>In the absence of CRP , this stimulation of RMF production was not observed , leading to the formation of a basal level of 100S ribosomes in the absence of cAMP-CRP ( Fig. 8C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In this phase , CRP is inactive due to the absence of cAMP , and acetyl-CoA synthetase gene is not expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25524697" target="_blank">25524697</a>]<br></li>
<li>When the fragment was incubated with CRP , but in the absence of cAMP , an identical pattern was observed ( lane 2 ) , whereas extensive protection around 187 and moderate protection around 137 was observed when the fragment was incubated with both cAMP ( 1-5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>No specific tdcA transcript was observed in the absence of CRP ( Fig. 3 , lane 5 ) or when cAMP was omitted from the assay ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>This is consistent with our observation of the absence of S1-initiated transcript and normal modulation of S2 by cAMP CRP as was previously shown in vitro ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>11 when Spot42 sRNA is expressed at full strength ( not repressed by cAMP -- CRP ) and the gal operon is transcribed at the maximum rate possible in the absence of cAMP -- CRP and GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Transcription of P1 remained high with up to 0.6 M K + glutamate in the reaction ; RNA synthesis at 0.6 M K + glutamate was 40 % greater than at 0.1 M K + glutamate in the absence of CRP -- cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>CRP-2 CRP-1 -10 deoC in deoP2 in the absence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>` A ' indicates that the selection was performed in the absence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Therefore , it can be activated by cAMP -- CRP even in the absence of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In the CRP defective strain , HY27 , neither the activities of ODC and ADC , nor the growth rate were effected when cAMP was increased to a concentration of 10 mM ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>In other words , the cAMP-CRP complex functions as a positive regulator for many genes controlled by catabolites , whereas , in the case of mdtEF , it is actually a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>In contrast to station-ary-phase induction of osmY , which now can be explained by the concerted action of us and the negatively acting transitio state regulators Lrp , cAMP-CRP , and IHF , the problem of osmotic regulation of osmY still awaits its solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In contrast , in addition to transcriptional regulation by cAMP-CRP ( Gibert and Barbe 1990 ) , ompA is growth-rate-regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204185" target="_blank">16204185</a>]<br></li>
<li>In summary , expression of yjiY is induced at the transition of the E. coli culture into the stationary-growth-phase , and induction requires cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>In an rpoS mutant , csiE expression is significantly but not totally reduced , and can still be stimulated by cAMP-CRP ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>In order to conﬁrm the role of CRP on the expression of cobB , patZ and acs genes , cAMP response experiments were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>In this genetic background , exogenous cAMP was required for induction of all four genes ( Figure 7C ) , indicating that Sxy does not bypass CRP 's cAMP-dependence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In galP18 the single-base-pair deletion at -18 must cause a rotational perturbation that precludes stimulation and facilitates inhibition by cAMP - CRP as seen for galP + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>In the CRP : cAMP2 crystal , ionic interaction between the R82 guanidinium group and the negatively charged oxygen of the cAMP phosphate was perturbed by introducing a polar group ( Q ) to position 82 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In this case , CRP -- cAMP might 54 compete with activators , such as NtrC , NifA , DctD , XylR , etc. for Es to downregulate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In one tube M cAMP and 5 pg of CRP were added and in the other tube 20 % glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>In addition , it has long been suggested that the mono-ligated form ( CRP -- cAMP1 ) is the only or the most active species of CRP , while it is evident that CRP can bind four molecules of cAMP in total [ 7,8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>In addition , since this is the first operon shown to be controlled by attenuation and cAMP-CRP , it was feasible to investigate © IRL Press Limited , Oxford , England .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>In fact , cAMP and CRP were proposed to be involved indirectly by affecting the synthesis of an unknown enzyme whose substrate or reaction product could alter the activity of ADPglucose pyro-phosphorylase ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>In turn , the intracellular concentrations of the small molecules cAMP and D-galactose control-the-activity of CRP , GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>It appears , therefore , that the cAMP-CRP complex has a higher affinity for wild-type CRP-2 than wild-type CRP-1 and that the affinity of cAMP and CRP for the mutated CRP-2 sites is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>deoPl deoR by the Initiation from is under negative control repressor , whereas initiation from deoP2 is dependent upon the cAMP/CRP complex and under the joint negative control of the deoR and cytR repressors , respectively ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2653814" target="_blank">2653814</a>]<br></li>
<li>Second , different promoters show different regulation on the same plasmid , i.e. glnAp1 is upregulated by the CRP -- cAMP complex ; glnAp2 is downregulated by the CRP -- cAMP complex ; tetR is constant and not affected by the CRP -- cAMP complex ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>We also show evidence that both promoters are directly activated by binding of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Furthermore , the nifB , nifE , nifF , nifJ , nifLA and nifU promoters from K. pneumoniae are also downregulated by the CRP -- cAMP complex ( unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>We propose that the inhibitory effect mediated by the CRP -- cAMP complex is probably through the classic PTS system , an extension of the ` glucose effect ' from carbon metabolism to nitrogen assimilation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>To investigate deoCp2 activity and regulation in the absence of the cAMP-CRP complex , we introduced pJEL134 into strains unable to synthesize cAMP ( Acya ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>another carbon source , for instance , or Under such conditions , the existence of a transcript induced by the cAMP-CRP complex may give a selective advantage to the cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>However , it is not obvious why cells would use the CRP -- cAMP complex to activate glnAp1 under the nitrogen-excess , carbon-limiting conditions ( for a review , see Magasanik , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>cAMP -- CRP complex is estimated to control more than 100 genes in E. coli , including ptsH and crr genes ( Titgemeyer and Hillen , 2002 ; Reuse and Danchin , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
<li>CRP adopts an active-conformation only when it binds cAMP , and the concentration of the active cAMP-CRP complex is influenced by the intracellular concentration of cAMP [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>This suggests that the effects are mediated by the CRP -- cAMP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Third , the opposing regulation of spf and galK by the cAMP -- CRP complex was striking .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>'' auto-regulation and positive control by the cAMP/CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Although GntR , GntH and cAMP-CRP complex bind as a dimer to each element , they are depicted as a single molecule for simplification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>Expression of lsrR and the lsr operon is inhibited by LsrR and stimulated by the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>Thus , cAMP - CRP inhibits P2 activity by precluding closed complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>CRP : ANS complex cAMP titration data were analyzed to determine the Adair constants for the formation of CRP ( cAMP ) 1 ( K1 ) and CRP ( cAMP ) 2 ( K2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>First , we prepared a full-length recombinant CRP and incubated it with cAMP to form the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>Since it is known that the amount of the cAMP-CRP complex increases under glucose-starvation conditions such as Tris-glycerol medium , these results suggested that expression of psiE is negatively regulated by cAMP-CRP independent of the phosphate concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>In the case of Pr , when glucose is not available , the cAMP -- CRP complex induces mhpR expression as occurs in typical class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Because the glucose repression effect is usually mediated by the CRP-cAMP complex in enteric bacteria ( 46 , 47 ) , we checked whether CRP was necessary for mhp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>Consequently , it was thought necessary to investigate whether the CRP -- cAMP complex inﬂuenced glnAp2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Because , in this case , the cAMP-CRP complex acts as a negative regulator of this operon , the release causes derepression of mdtEF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>CRP and cAMP maximally regulate CRP-dependent genes when the level of cAMP is elevated due to growth on a poor carbon source ( for reviews , see references 2 and 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Since CRP and cAMP regulate gcvT-lacZ over a fourfold range in GM medium , a preferred carbon source where the cAMP level is low , we tested whether CRP would regulate gcvT-lacZ over a larger range if the level of cAMP was elevated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1734">
This suggests that glucose repression is mediated by cAMP-CRP bound to this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></label>
<input id="act_similarity_1734", type="checkbox">
<div class="ACT_similar">
<li>This suggests that glucose repression of araBAD is not mediated by cAMP-CRP bound to this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li></div>
<li>Binding of cAMP -- CRP at site I increased DNase I cleavage at positions 135 and 145 ( Figure 5B , lanes 8 and 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1737">
For reduced valence , operators CRP12 and CRP22 , configurations in which CRP ( cAMP ) 1 is bound to the mutated site were excluded from the summation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_1737", type="checkbox">
<div class="ACT_similar">
<li>For reduced valency , CRP12 and CRP22 operators , configurations in which CRP ( cAMP ) 1 is bound to the mutated site were excluded from the summation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li></div>
<li>TAACGCCTTT ) , located at 257.5 and 255.5 from the transcriptional start site of glnAp2 , respectively , might bind CRP -- cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Effect of cAMP and CRP on initiation at the P3 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>Transcription of sxy depends on CRP , cAMP and the CRP-S site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>This might be as a result of the fact that the concentrations or the activities of the repressor CRP -- cAMP versus the activator NtrC or NifA are different under the conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In the absence of other proteins , cAMP-CRP binds to three CRP targets centred at -92.5 ( CRP2 ) , -42.5 ( CRP1 ) and +56.5 ( CRPE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Apparently , cAMP-CRP binds to the high-affinity target CRP2 and only one of the weak CRP targets ( CRP1 or CRPE ) under the conditions of the mobility-shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Affinities for CRP1 and CRP2 parallel each other over the entire range of cAMP concentrations , consistent with the conclusion that cAMP is an effector of pre-existing conformational states of free CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>deoO is the sites recognized by DeoR ; CRP and CRP2 the cAMP/CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Secondly , inudpPEcthe CRP2 target has higher afﬁnity for cAMP -- CRP as compared to CRP1 [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Although the asymmetry observed in the cAMP -- CRP crystal has been interpreted as an important nature for its functional mechanism , NMR studies in solution could never ﬁnd any evidence of structural asymmetry in both the apo-CRP and cAMP -- CRP [ 12,14,25,32 -- 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Many of the CRP mutation sites , particularly the cluster in the αD , are also involved in the interdomain networking of cAMP -- CRP via the hinge region [ 6,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Our recent NMR investigation that aimed to identify the region of interdomain interaction change in CRP has implied all of the known positions for CRP mutations are involved in the interdomain interaction sites , either in apo - or in cAMP -- CRP [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Since CRP LA is constitutively active even without cAMP , we can identify the change in the T127 -- L137 region as the critical factor responsible for global transition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Second point is the fact that only T127 and S128 among the six residues directly contacting to cAMP provide the sites for CRP mutation ; the T127L/S128A or T127L/S128I mutant of CRP is constitutively active without cAMP [ 24,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>During logarithmic growth-phase , the concentration of FIS decreases , whereas increasing cAMP -- CRP levels activate divergent RNA synthesis , limiting the frequency of initiation at the crp1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Optimal cAMP -- CRP levels might then activate divergent RNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>cAMP -- CRP complexes are formed which , at lowered FIS levels , activate divergent RNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Thus , it can not be excluded yet that for some of group-B1 and -- B2 cases , both of the divergently transcribed operons are regulated by cAMP-CRP bound within intergenic regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Unlike the Gal repressosome , the OE-bound GalR dimer does not inhibit cAMP -- CRP action at the galETKM promoters ; instead , GalR and cAMP -- CRP cancels each other 's effect ( Lewis , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>We conclude that Es , but not Es , can optimally S 70 use the activator cAMP-CRP when bound to 268.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>It thus seems reasonable to conclude that cAMP-CRP is bound to CRP-2 in the combined footprint .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In lanes 6 -- 8 , increasing concentrations of cAMP -- CRP were added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>For each panel : lane 1 , without cAMP and CRP ; lane 2 , only CRP added : lanes 3-5 , CRP and increasing concentrations of cAMP , Templates : left panel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>although we have no data concerning the intracellular concentrations of cAMP , CRP , and RNA polymerase in our experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>It should be pointed out that it is still not known how gluconate , a non-PTS carbon source , regulates the intracellular concentrations of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>( Left ) Effect of saturating amounts of CRP ( 10 , u.g / ml ) and cAMP ( 15 , uM ) on digestion by DNase I. Lane 1 , No addition ; lane 2 , CRP alone ; lane 3 , CRP and cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Lane 1 , no additions ; iane2 , 10 nM FadR ; lane 3 , 45 nM CRP and 10 ( jm cAMP ; lane 4 , 10 nM FadR , 45nM CRP and lO ^ McAMP Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>cAMP to 10 fiM and CRP fo 45 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>cAMP was added to the reaction mixture to a final concentration of 50 F.M , and the CRP concentration present in the reaction is given in micromolar amounts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>When present , CRP is at a concentration of 100 nM and 200 pM cAMP is included in the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>To study the e ¡ ect of cAMP , di ¡ erent amounts of cAMP and 100 nM CRP were included in the preincubation mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760580" target="_blank">10760580</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1770">
Additional regulators such as 40 nM CRP , 100 mM cAMP , or 0 -- 1.5 mM Mlc were added to the reaction as needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11340070" target="_blank">11340070</a>]<br></label>
<input id="act_similarity_1770", type="checkbox">
<div class="ACT_similar">
<li>Additional regulators such as 40 nM CRP , 100 mM cAMP , or 0 -- 200 ng of Mlc were added to the reaction as needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464268" target="_blank">10464268</a>]<br></li></div>
<li>Binding reactions 2 , 4 and 6 contained 100 nM CRP ; 1 mM cAMP was present in all the reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>KMnO4 footprinting reactions contained 0.2 mM cAMP , 100 nM CRP and 50 nM Es28 or Es70 , as required .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>FeBABE footprinting reactions contained 0.2 mM cAMP , 100 nM CRP and 200 nM FeBABE-labelled Es28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Final concentrations in the assay are given : DNA , 6 nM ; polymerase , 175 nM or 100 nM ; ApU , 500 uM ; UTP , 50 ytM ; CRP and cAMP as indicated above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>ppGpp was added in various concentrations ( 0 , 25 , 50 , 100 , 250 , and 500 , uM ) to reactions containing cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Final concentrations of CRP and cAMP were 50 nM and 200 jiM respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>By comparison with DMS protection patterns obtained for other CRP targets ( see Fig. 8 ) , it is proposed that , when present alone , cAMP-CRP binds CRP targets centred at - 9 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Similarly , when constitutively expressed NifA is used as the activator , CRP -- cAMP downregulates glnAp2 less than when NtrC is used as the activator ( 11-fold versus 21-fold , Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Quantification of incorporated [ 35S ] methionine ( Fig. 2 ) allowed the estimation of saturation levels for cAMP , CRP , p.l , , uM , and ppGpp ( 100 p.M , 2 p.g / 35 and 150 to 200 respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>For example , the αC-αD angle is about 40 ° in the active CooA LL Ch , which angle is consistent with that in the closed subunit of cAMP -- CRP ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>It is additionally observed in Fig. 3B that the αD in both the CprK structures inactive was stretched to the opposite direction of what is observed in cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Beyond this point , osmYis finally strongly induced , which also implies that the repressing activity of cAMP-CRP , Lrp , and IHF is at least partially relieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Our future workwill be directedtowards identifying and characterizing the putative activator that binds upstream of position -38 and that mediates positive control by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>A number of the genes for both glycolysis and gluconeogenesis are under the control of catabolite repressor activator Cra , but independent of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>A closer investigation of the effects of crp and cya mutations and of exogenously added cAMP revealed that the control of csiE by cAMP-CRP has both positive and negative elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>cAMP-CRP and LsrR are involved in transcriptional control of lsr and lsrRK operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>Under these conditions , cAMP-CRP and os act in an additive , and therefore apparently independent , manner in the control of csiE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Under these conditions any gene under positive cAMP/CRP control should be at least weakly induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>These results indicate that CRP -- cAMP-mediated inhibition is not limited to a speciﬁc activator , speciﬁc growth conditions or medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>This occurs late in the growth-phase and results in the accumulation of cAMP , the allosteric-effector for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>This is also highlighted by our results which implicate both cAMP-CRP and EIIA in this Glc regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li>
<li>Taken together , these results demonstrate that csiE is under positive control of both as and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>These results further substantiate the possible involvement of cAMP and CRP in the synthesis of SecB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023186" target="_blank">9023186</a>]<br></li>
<li>gene system , the exwession of g/pD and g/pE was found to be positively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Transcription of both P1galE and P2galE can be regulated by GalR and cAMP -- CRP , while GalS affects only P1galE activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Many of the genes showing decreased RNA levels after adaptation to acetate or propionate are known to be regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>We also confirmed that intracellular levels of the translation products and the transcripts from g/pD and g/pE were positively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Moreover , it is known that cAMP , possibly in combination with CRP , could regulate cell morphology ( 16-18 ) , cell division ( 19 ) and the interest to study how these factors are controled in a cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>A preliminary report which showed that the in vitro synthesis of the first dipeptide and tripeptide of glgC was stimulated by cAMP and CRP was published previously ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>It is not clear how cAMP-CRP activates P3 in vivo but inhibits P3 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>We predict that CRP -- cAMP -- mediated inhibition of s - dependent 54 promoters is quite common .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Experiments are under way to study the mechanisms involved in CRP -- cAMP-mediated inhibition of s - dependent 54 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>This protection was dependent on the addition of both cAMP and CRP ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Various genes , including rpoS and several sigma S-dependent genes , such as bolA , are negatively controlled by cAMP-CRP ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Specific bin-ding of CRP is completely cAMP-dependent , since CRP without cAMP has no general effect upon the digestion pattern .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Downregulation of outer membrane proteins by noncoding RNAs : Unraveling the cAMP -- CRP - and s - dependent CyaR -- ompX regulatory case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665928" target="_blank">21665928</a>]<br></li>
<li>Binding of CRP is dependent on cAMP and possibly on another factor present in crude cell extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>CRP -- cAMP inhibits s - dependent glnAp2 strongly , by 21-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>CRP abolished cAMP-dependent stimulation of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>Therefore , the induction of these csi : : phoA fusions is not dependent on cAMP or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022277" target="_blank">8022277</a>]<br></li>
<li>Based on these values of K1 and K2 , the fraction of CRP - ( cAMP ) 1 is calculated to reach a maximum of 0.635 ( 60.021 by propagation of errors ) of total CRP at 150 mM cAMP , the concentration used in all subsequent experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As a practical approximation to the limit of saturating CRP ( cAMP ) 1 , 0.1 mM CRP ( total dimer ) and 150 mM cAMP were used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>When glnAp2 is activated by NifA , a similar inhibitory effect by CRP -- cAMP is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>This arrangement is in contrast to the location of the sequence on the sense strand in most of those genes that are activated by cAMP-CRP ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Unlike the above promoters , P2galE is repressed by cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Binding of cAMP -- CRP has the opposite effect ; it activates P1galE and represses P2galE ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>When present ( as indicated above the gel ) , the His6-RhaS-CTD concentration was 10 M and the CRP concentration was 1 M. cAMP was present in all reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>When D-galactose is present , cAMP-CRP can activate P1galE and repress P2galE ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Experiments in which CRP was present also contained 150 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1821">
As CRP was present in these reactions , 0.2 mM cAMP was also included .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_1821", type="checkbox">
<div class="ACT_similar">
<li>When CRP was present in the reactions , 0.2 mM cAMP was also included .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>When CRP was present in the reactions , 0.2 mM cAMP also was included .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li></div>
<li>Asymmetry between subunits was also appreciable , as in cAMP -- CRP , with relatively more open and more closed conformations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Pyruvate oxydase B is also under the control of RpoS and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21718532" target="_blank">21718532</a>]<br></li>
<li>No CRP-cAMP sample was also prepared without CRP and cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>Purified glp repressor ( with or without inducer , glycerol-P ) or CRP ( with or without cAMP ) was added at the concentration indicated in the legends to the figures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>As shown in Figure 3A , CRP -- cAMP was an effective repressor at 0.1 M and 0.2 M K + glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Even the closed subunit of RrCooA and open subunit of cAMP -- CRP is in good correlation with activity , by showing larger and smaller αC-to-αD angles , respectively , than that of the intermediate state of CooA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>[ 35 ] A.E. García , J.G. Harman , Simulations of CRP : ( cAMP ) 2 in noncrystalline environments show a subunit transition from the open to the closed conformation , Protein Sci .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>A region which binds cAMP-CRP and therefore probably mediates the cAMP effects was detected upstream from glgC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>One unit is the amount of CRP that binds one pmol of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>There , aCTD is positioned upstream of cAMP-CRP , and the latter interacts with aCTD , aNTD and s ( Busby and Ebright , 1999 ; Rhodius 70 and Busby , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Therefore , it was decided to investigate whether sequences upstream of glnAp2 are essential for CRP -- cAMP-mediated inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>CRP -- cAMP-mediated inhibition at glnAp2 might occur by one or more mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Conversely , glucose-starvation leads to an increase in cAMP levels allowing a molecule of cAMP to-bind CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>Effects of glucose , CRP , and cAMP on gapB mRNA level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>CRP is an indirect sensor of glucose-starvation by virtue of its ability to monitor the intracellular cAMP pool , which becomes elevated when glucose is exhausted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Transcription of the plas-mid-encoded rnaI gene was unaffected by CRP -- cAMP , as expected , and decreased somewhat at the higher salt concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>: : : a - ` However , this picture changed when the cAMP and CRP concentrations were reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>At low cAMP concentrations , CRP fails to maximally stimulate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Thus , it is inferred again that the stimulation at T127 and S128 by cAMP would be the critical factor directing the allosteric transition of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Cell extracts were prepared , and CRP was puriﬁed by cAMP afﬁnity chromatography followed by ion-exchange chromatography under conditions described previously ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>CRP was puriﬁed essentially as described previously ( 64 ) , with the exception that CRP was precipitated with ( NH4 ) 2SO4 to obtain CRP free from cAMP instead of removing the cAMP by chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Inﬂuence of CRP and cAMP on the level of gapB transcripts in total RNA extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>While the total amount of RNA made increased when cAMP-CRP were added there was little difference in the percent of transcripts that were terminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>CRP was activated by the addition of 200 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Transcription of PgalP was activated by cAMP -- CRP to obtain detectable level of transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Supershift by GntR/cAMP-CRP , however , was hardly observed under the conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>Received in revised form 29 April 2009 Accepted 30 April 2009 Available online 9 May 2009 Keywords : Allostery cAMP CooA CprK CRP PrfA 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Titration of the effects of cAMP , CRP , and ppGpp on the synthesis of glycogen biosynthesis enzymes in S-30 extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>'' s j t sx - 50 0 tsx-501 t s x - 502 tax - 503 tsx - wt + + +2 13 +1 + + + cAMP - + + + + + 3 + + + + - ¬ - 13 13 2.6 0.5 Q3 CLO5 - 13 13 2.6 05 0.3 0.05 i CRP - 13 13 2.6 0S 03 005 - 3 13 2.6 05 03 0.05 - 3 13 2.6 05 3 I b F * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Where indicated , lo `` M cAMP and/or 0.1 pgof CRP were added to the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>cAMP was added to 50 µM in reactions containing CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Also , 250 , uM ppGpp was added to reactions which lacked cAMP , CRP , or both factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Thus , we investigated the role of cAMP and CRP in AI-2 uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Of course all of the CRP mutants might adopt similar conformation as cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>On this basis , the simplifying assumption was made that only CRP ( cAMP ) 1 is functionally active in all subsequent analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP -- cAMP could interact directly with s or its holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>E 70 did not direct synthesis of this transcript by itself or with cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>]<br></li>
<li>Where do PhoB and cAMP-CRP bind ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>cAMP - - +1 + + + + C R P - 13 13 2.6 0.5 0.3 0.05 290 ¬ CR P - 2 320 ¬ CRP - 1 350 ¬ 380 A/G 1 2 3 4 5 6 7 FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>468 S. Semsey , S. Krishna , K. Sneppen and S. Adhya Regulation by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Reactions with CRP contained 2mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Interestingly , cGMP can be substituted for cAMP as an effector of CRP * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670911" target="_blank">2670911</a>]<br></li>
<li>Thus , ArcA and FNR regulate respiration in response to oxygen availability ( the variable in these experiments ) , CRP and Lrp sense nutritional state via cAMP and L-leucine concentrations , and Fis , H-NS and IHF modify DNA topology , which is itself dependent on energy status [ 7,25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1867">
The active cAMP-CRP complex binds to speciﬁc DNA sites located at or upstream of CRP-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></label>
<input id="act_similarity_1867", type="checkbox">
<div class="ACT_similar">
<li>As a consequence , the active cAMP-CRP complex binds to speciﬁc DNA sites located at or upstream of CRP-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li></div>
<li>To determine whether the CRP-cAMP complex binds to this region , a DNase I footprinting experiment with purified CRP and end-labeled DNA was performed ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>Figure 3B depicts that the cAMP-CRP complex bound to this synthetic fragment in a CRP dose-dependent manner ( 0 to 200 nM ) , whereas cAMP or CRP alone ( Fig. 3B , bottom ) failed to-bind to the proposed region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1871">
Blazy , B. , Takahashi , M. and Baudras , A. ( 1980 ) Binding of CRP to DNA-dependent RNA polymerase from E. coli : modulation by cAMP of the interactions with free and DNA-bound holo and core enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></label>
<input id="act_similarity_1871", type="checkbox">
<div class="ACT_similar">
<li>and Baudras , A. ( 1980 ) Binding of CRP to DNA-dependent RNA polymerase from E. coli : modulation by cAMP of the interactions with free and DNA-bound holo and core enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2041473" target="_blank">2041473</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160397" target="_blank">2160397</a>]<br></li></div>
<li>Up to now , more than ten 3-dimensional structure coordinates of CRP ( Table 1 ) have been deposited in the Protein Data Bank ( PDB ) , including th three different functional states of complexes ( CRP -- cAMP [ 16,17 ] , CRP -- cAMP -- DNA [ 18 -- 20 ] , and CRP -- cAMP -- DNA -- RNAP [ 21 ] ) and some mutants ( namely CRP ) that are constitutively active even in the absence of cAMP [ 22 -- 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Two nM DNA , 40 nM CRP , 200 ng of Mlc , and 100 mM cAMP were mixed in transcription buffer in the combinations as indicated in Fig. 5 and incubated at 37 ° C for 10 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464268" target="_blank">10464268</a>]<br></li>
<li>Total radioactivity in DNA was taken as 100 % Discussion The magnitude of the transcriptional repression of cya exerted by CRP is much less than the degree of repression of cAMP level seen in crp ` cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>The assay was carried out as above with the DNA fragment containing malEp and malKp , except that 0.2 mM cAMP and 30 nM CRP were included in the reaction mixture ( Richet and Raibaud , 1987 ) and that ApA and [ a-32P ] CTP were replaced by GpU and [ c & 32P ] UTP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2524384" target="_blank">2524384</a>]<br></li>
<li>The earlier approach of MD simulation by García and Harman indicated the closed conformation of both subunits [ 35 ] , consistent with that observed in the crystal structures of cAMP -- CRP -- DNA and cAMP -- CRP -- DNA -- RNAP complexes [ 18 -- 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The DNA fragments were shifted by the addition of CRP and cAMP and supershifted by further addition of GntH and 5-ketogluconate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>The interaction of cAMP-CRP with restriction fragments was detected by virtue of the decreased electrophoretic mobility of bound versus free DNA ( 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Since CRP does not bind speciﬁcally to DNA in the absence of cAMP ( 21 ) , we tested whether CRP 's regulation of gcvT-lacZ requires cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Transcription was performed at increasing cAMP concentrations with constant CRP and wild-type RNP using supercoiled pdeoP2 as the DNA template ( Figure 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Our results revealed that cAMP-CRP binds to the DNA fragment containing both site 1 and site 2 , DISCUSSION AI-2 was ﬁrst discovered in V. harveyi as a quorum signal to regulate bioluminescence ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In the absence of cAMP , CRP binds to DNA unspeciWcally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In contrast , run-off assays showed that the yield of the full-length transcripts was markedly enhanced by prior incubation of the DNA fragment with CRP - cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Further , addition of cAMP and RNAP caused additional footprinting indicating the ternary complex formed between CRP : cAMP , RNAP and DNA ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>With ﬁxed amounts of CRP and DNA , the amount of CRP-cAMP/DNA complex was found to be proportional to the concentration of cAMP in the assay mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Based on the cAMP -- CRP and cAMP -- CRP -- DNA structures [ 16 -- 19 ] , it has long been assumed that a CRP dimer can bind two molecules of anti-conformation cAMP , one at each NTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>This deﬁciency is overcome by interaction with DNA-bound cAMP -- CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Likewise , the amount of DNA-bound AscG decreased with an increase in cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Polymerase was at 150 nM , DNA template at 4 nM , cAMP at 200 AM and CRP , when present , at 100 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Prior to DNAase I treatment , the labelled DNA was preincubated with melibiose , MelR , cAMP ± CRP and RNAP ( 200 nM ) as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Prior to DNase I treatment , the labelled DNA was preincubated with melibiose , MelR303 , MelRl73 , CRP and cAMP as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Transcriptions were performed in a 50 ml reaction volume containing 1.6 pmol of Fis , 1.1 pmol of CRP , 0.1 mM cAMP , and 0.1 pmol of supercoiled plasmid DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Reactions were initiated with the addition of the enzyme , following a 10-min pre-incubation of DNA and nucleotide substrates and eventually CRP and cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1895">
As only activated CRP binds DNA , 0.20 mM cAMP was included in all buffers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_1895", type="checkbox">
<div class="ACT_similar">
<li>As only activated CRP binds DNA , 0.2 mM cAMP was included in all buffers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>Competitor DNA was not included , cAMP ( 101 ~ M ) was included in reactions containing CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Detailed secondary and tertiary structural information for CRP ( cAMP ) 2 and CRP ( cAMP ) 2-DNA co-crystals are available .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>When cAMP-CRP binds DNA , it induces DNA bending of about 87 [ 16,22,23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Finally , it has been conﬁrmed in solution without DNA that CRP binds four cAMP molecules as its maximum [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Interaction Reference Galactose transport by GalP 1 2 Riordan and Kornberg ( 1977 ) Robbins et al. ( 1976 ) Geanacopoulos and Adhya ( 1997 ) Geanacopoulos and Adhya ( 1997 ) This work Geanacopoulos and Adhya ( 1997 ) Weickert and Adhya ( 1992 ) Geanacopoulos and Adhya ( 1997 ) Weickert and Adhya ( 1992 ) Majumdar and Adhya ( 1984 ) ; Majumdar et al. ( 1987 ) Weickert and Adhya ( 1993b ) Weickert and Adhya ( 1993b ) This work This work This work Polayes et al. ( 1988 ) Rice and Dahlberg ( 1982 ) Weickert and Adhya ( 1992 ) Aki et al. ( 1996 ) ; Choy and Adhya ( 1992 ) Choy and Adhya ( 1992 ) Aki et al. ( 1996 ) Choy and Adhya ( 1992 ; 1993 ) Choy and Adhya ( 1992 ; 1993 ) Adhya ( 2003 ) ; Ullmann et al. ( 1979 ) Adhya ( 2003 ) ; Ullmann et al. ( 1979 ) Hucho and Wallenfels ( 1971 ) Møller et al. ( 2002 ) Sherman and Adler ( 1963 ) Kurahashi and Sugimura ( 1960 ) ; Wong et al. ( 1977 ) Wilson and Hogness ( 1964 ) Weissborn et al. ( 1994 ) Joshi and Handler ( 1964 ) Hogema et al. ( 1997 ) Hogema et al. ( 1998 ) Galactose transport by the Mgl system cAMP -- CRP activates PgalP cAMP -- CRP activates PmglB GalS represses PgalP GalR represses PgalP GalS represses PmglB GalR represses PmglB D-galactose binding inactivates GalS D-galactose binding inactivates GalR 3 4 5 6 7 8 9 10 cAMP -- CRP activates PgalS GalS autorepression 11 12 13 14 15 16 17 18 19 GalS represses PgalR GalR represses PgalS GalR autorepression cAMP -- CRP represses spf transcription spf encodes Spot42 RNA GalS represses P1galE GalR represses P1galE by contact inhibition and DNA looping GalR binding to OE activates P2galE under non-looping conditions GalR represses P2galE by DNA looping 20 21 22 23 24 25 26 27 28 29 cAMP -- CRP activates P1galE cAMP -- CRP represses P2galE P1galE makes equimolar expression of the galETKM operon genes P2galE expresses primarily the promoter proximal galE gene ( natural polarity ) GalM catalyses the b-D-galactose ↔ a-D-galactose reaction Spot42 RNA inhibits translation of galK GalK catalyses the a-D-galactose + ATP ↔ galactose-1-phosphate + ADP reaction GalT catalyses the galactose-1-phosphate + UDP-glucose ↔ glucose 1-phosphate + UDP-galactose reaction GalE catalyses the UDP-galactose ↔ UDP-glucose reaction GalU catalyses the UDP-glucose ↔ glucose 1-phosphate reaction Pgm catalyses the glucose 1-phosphate ↔ glucose 6-phosphate reaction Glucose 6-phosphate decreases the cAMP -- CRP level Galactose transport is inhibited by inducer exclusion in the presence of glucose 6-phosphate 30 31 32 33 34 Numbering refers to interactions in Fig. 6A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>It has been shown that at some arti ® cial CRP-dependent promoters , cAMP ± CRP centered near position ± 103 , ± 93 or ± 83 could synergistically activate transcription with a cAMP ± CRP centered near position ± 42 , a class III CRP-dependent promoter ( Belyaeva et al. , 1996 , 1998 ; Murakami et al. , 1997 ; Busby and Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>It has been proposed that repression is accomplished through synthesis of a divergent RNA molecule whose transcription starts 2 bp upstream from the crp mRNA , runs in the opposite direction and is strongly dependent on cAMP -- CRP ( Okamoto and Freund-lich , 1986 ; Okamoto et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>The present study reveals that , in addition to transcriptional repression by phosphorylated OmpR ( Shin and Park , 1995 ) and HdfR ( Ko and Park , 2000 ) and activation by cAMP-CRP ( Soutourina et al. , 1999 ) , flhDC is regulated post-transcriptionally by CsrA-mediated transcript stabilization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>This enhancement is due to cAMP -- CRP-mediated activation of the class 1 operon flhDC ( Yokota & Gots , 1970 ; Silverman & Simon , 1974 ) , since the cAMP -- CRP system is more active in low-nutrient conditions than in high-nutrient conditions ( Ishizuka et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>When we relocated the CRP box to different locations between 260.5 and 280.5 , we observed DNA helix phase dependency of cAMP-CRP activation to be equally stringent for Es - dependent expression from the csiD promoter S ( Fig. 2 ) , as was reported previously for Es - mediated 70 expression from the lac or CC promoters ( Gaston et al. , 1990 ; Ushida and Aiba , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>We propose therefore that a physiological role of CRP -- cAMP activation of glnAp1 is to partially compensate for CRP -- cAMP downregulation of Q 2001 Blackwell Science Lt For s - dependent promoters , upstream activator 54 sequences ( UAS ) are the speciﬁc target sites for transcriptional-activators belonging to the NtrC family ( NtrC , NifA , etc. , reviewed in Kustu et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>These results are consistent with the previous reports ( Sùgaard-Andersen et al. , 1990a ) that CRP-2 plays no role , or a very small one , in cAMP ± CRP activation of transcription at deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>A unifying model for activation is difficult to reconcile with the ability of cAMP-CRP to activate transcription from sites lying at different positions within the various promoters ( Gaston et al. , 1990 ; Ushida and Aiba , 1990 ; ValentinHansen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>It has been proposed that transcription activation of class II CRP-dependent promoters is established by positioning a-CTD to the DNA site immediately upstream of the cAMP ± CRP site in addition to direct activation ( Attey et al. , 1994 ; Belyaeva et al. , 1996 , 1998 ; Niu et al. , 1996 ; Busby and Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Direct activation is through an interaction of a-NTD with AR2 of cAMP ± CRP ( see Figure 1B ) ( Attey , 1994 ; Belyaeva et al. , 1996 ; Murakami et al. , 1997 ; Rhodius et al. , 1997 ; Savery , 1998 ; Busby and Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>This DNA sequence-independent interaction between the DNA adjacent to cAMP ± CRP and a-CTD of RNP has been implicated as playing a role in class II cAMP ± CRP-dependent activation ( Savery et al. , 1995 , 1998 ; Lloyd et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>This model also offers a plausible explanation for the recent findings that CytR does not regulate transcription from deoP2 in Icya or Acrp strains , although transcription occurs from the same site in deoP2 in the absence and presence of cAMP-CRP ( L. Sogaard-Andersen et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Thus , in the presence of both cAMP-CRP and CytR , a nucleoprotein complex is formed that most probably contains cAMP-CRP at each CRP site and CytR bound in the intervening region ( Sogaard-Ander-sen et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>The open bar indicates the operator region protected by CytR in the presence of cAMP-CRP ( Sogaard-Andersen et al. 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Transcription of the rpoH gene from these promoters is regulated by CRP -- cAMP , CytR ( Ramirez-Santos et al. , 2001 ; Kallipolitis & Valentin-Hansen , 1998 ) and DnaA ( Wang & Kaguni , 1989b ; Messer & Weigel , 1997 ) proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>For example , to-bind to and achieve repression of the deoP2 , cdd and udp promoters , the CytR repressor has to interact with cAMP -- CRP ( Holst et al. , 1992 ; Sogaard-Andersen and Valentin-Hansen , 1993 ; Brikun et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Mol Gen Genet ( 1991 ) 231:76 -80 OIGG © Springer-Verlag 1991 Restored DNA-binding of the cAMP-CRP activator complex reestablishes negative regulation by the CytR repressor in the deoP2 promoter in Escherichia coli Lotte Sogaard-Andersen and Poul Valcntin-Hanscn Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark ReceivedMay 13 , 1991 July 25 , 1991 1990a , b ) whereas cAMP-CRP activation only requires an intact CRP-1 site ( Valentin-Hansen et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Firstly , cAMP-CRP activation of deoP2 is mediated by the CRP-1 site , whereas proper regulation of this promoter by CytR requires tandem intact CRP binding sites ( Martinussen et al. 1989 ; Sogaard-Andersen et al. 1990a , b ; Valentin-Hansen 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In CytR regulated promoters , cAMP-CRP constitutes an essential part of the repression apparatus and the available data suggest that two appropriately positioned CRP binding sites are a prerequisite for negative regulation by CytR ( S0gaard-Andersen et al. , 1990a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>cddPD79 bears a single target for CytR and cAMP -- CRP , respectively , and is repressed ﬁvefold by the co-operative binding of the two regulators ( see Fig. 1 and Table 1 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In vitro , CytR exhibits a low affinity for the sequence located between the two CRP sites ; however , its binding to deoP2 can be stimulated as much as 1000-fold by cAMP-CRP ( Sogaard-Andersen et al. , 1991b ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Repression of deoP2 Repression of deoP2 by CytR was only observed in the presence of cAMP ± CRP , in agreement with the in vivo observation that CytR binding requires cAMP ± CRP ( Sùgaard-Andersen et al. , 1990a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In vitro , CRP-3 is only occupied in the presence of CytR and is located at a position relative to CRP-1 that has been shown to be optimal for the cooperative binding of cAMP-CRP and CytR in deoP2 ( S0gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Based on these observations , it has been suggested that CytR and RNP are rivals that compete for binding with CRP to deoP2 , and that CytR acts as an anti-activator , nullifying the activation by cAMP ± CRP ( Sùgaard-Andersen et al. , 1990a , b ; Mùllegaard et al. , 1993 ; Kristensen et al. , 1997 ; Meibom et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>This ability of cAMP - CRP to stimulate the binding of CytR cooperatively requires a set of amino acids that are located in a specific region on the surface of cAMP-CRP ( Pedersen et al. , 1991 ; Sogaard-Andersen et al. , 1991a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR binds to its cognate promoters in concert with cAMP-CRP in a highly co-operative fashion torming repression complex that generally consists of two mol-ecules of cAMP-CRP separated by 52 or 53 bp and CytR ioound in between ( Sogaard-Andersen et ai , 1991b ; Gerlach etai , 1991 ; Holsi etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The synergy in the system is prominent ; depending on which promoter is examined , the cAMP -- CRP activator complex strengthens binding of CytR from 100 - to several thousandfold in vitro ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 , 1992 , 1995 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Therefore , the cAMP-CRP complex strengthens binding of CytR to its operators up to 1000-fold ( Pedersen et aL , 1991 ) , increasing the affinity of the repressor for the promoters to a level adequate for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>H Discussion Recent studies have indicated that repression of the deoP2 promoter by the CytR repressor requires the cAMP-CRP complex and that there is a coupling between cAMP-CRP binding to tandem CRP targets and the interaction of the CytR repressor ( Sogaard-Andersen et aL , 1990a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>On the one hand , the recent isolation of CRP mutants that can not activate transcription but still bend the DNA like wt CRP point to the importance of direct interactions between cAMP-CRP and RNA polymerase ( Bell et al. , 1990 ; Eschenlauer and Reznikoff , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Binding of the cAMP -- CRP complex to its target sites is known to bend DNA ( Schultz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>On the other hand , the observation that the CRP sites in gal and lac can be replaced by properly phased intrinsically bent DNA sequences points to the importance of the cAMP - CRP induced DNA bend in the activation process ( Bracco et al. , 1989 ; Gartenberg and Crothers , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>At its site of interaction , cAMP -- CRP induces sharp bends in DNA ranging from 90 to 180 ° ( Wu and Crothers , 1984 ; Liu-Johnson et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>In these complexes , the DNA bends strongly around cAMP -- CRP , bringing the DNA-bound regulators into close proximity ( Søgaard-Andersen et al. , 1991b ; Crothers and Steitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>When bound with cAMP , CRP is converted into a functional form and recognizes a spe-ciﬁc DNA sequence , referred to CRP box , with the consensus sequence of a 22-bp inverted repeat sequence , 50-AAATGTGATCTAGATCACATTT-30 [ underlines are highly conserved ] ( De Crombrugghe et al. , 1984 ; Berg & von Hippel , 1988 ; Gunaskerra et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>These results are consistent with previous studies showing that nitrogen-deﬁciency-induced glnA expression is controlled by s - dependent glnAp2 , while poor carbon-induced glnA 54 expression is controlled by CRP -- cAMP-activated , s - dependent glnAp1 ( MacNeil et al. , 1982b ; Reitzer 70 To explore the role of CRP -- cAMP and/or NtrC DNA-binding sites in the control of glnA , two derivatives of pKU100 were constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The presence of a putative cAMP-CRP binding site , sdh -- galK fusion analyses and DNA micro-array experiments revealed that glucose repression of Psdh is in part cAMP-CRP dependent ( Wilde and Guest 1986 ; Gosset et al. 2004 ; Zheng et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>The binding of CRP -- cAMP after an osmotic upshift was measured by dimethylsulfate ( DMS ) footprinting in growing cells where molecular-crowding forces caused by the loss of free water are operating to stabilize protein -- DNA interactions ( Record et al. 1998a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>CRP -- cAMP could interact with a-CTD in solution and inhibit binding of Es RNA polymerase to some target 54 promoters , or alter the conformation of promoter-bound Es RNA polymerase and so limit activation by NtrC 54 ( Wang et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>It has been shown that cAMP-CRP binding to this region is required for the activation of P1 and the inhibition of P2 ( Musso et al. , 1978 ; Aiba et al. , 1981 ; Busby et al. , 1982b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>cAMP levels are high when cells are ters are essential for transcriptional activation , and the location of binding sites and their afﬁnity for CRP help to determine the extent of activation ( reviewed in Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>However , the near-normal osmotic control by the cAMP-independent crp * mutant in the absence of ade-nylate cyclase , together with the relative insensitivity of cAMP binding to CRP measured in vitro with increasing salt concentration ( Takahashi et al. 1980 ) , argues against such a model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1944">
As CRP -- cAMP can exert its effect on a ` core ' promoter , which lacks a speciﬁc CRP-binding site , it was proposed that the effect might be general , with potential relevance to other s - dependent promoters 54 ( Wang et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></label>
<input id="act_similarity_1944", type="checkbox">
<div class="ACT_similar">
<li>As CRP -- cAMP can exert its effect on a ` core ' promoter , which lacks a speciﬁc CRP-binding site , it was proposed that the effect might be general , with potential relevance to other s - dependent promoters ( Wang et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li></div>
<li>Since the results of Stojcevic et al. ( 21 ) are well accounted for by our model and , moreover , we provide evidence that the cAMP-CRP complex binds directly to the pga promoter , it seems clear that it is the CRP-binding site II that has the higher affinity for cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Similarly , Webster et al. ( 1988 ) were unable to detect the binding of cAMP -- CRP complex alone to the melR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>It is well known that the cAMP -- CRP complex prevents the Eσ70 holoenzyme from binding at lacP2 and activates transcription at lacP1 ( Malan and McClure , 1984 ; Spassky et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10406809" target="_blank">10406809</a>]<br></li>
<li>As expected from numerous prior investigations ( e.g. , Lawson et al. [ 31 ] ) , CRP lacked DNA binding activity in the absence of its cAMP ligand ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Moreover , the primary effect of a replacement of C at position -97 with A would be expected to increase binding of cAMP-CRP to CRP-3 marginally ( de Crombrugghe et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>This di $ erence is probably re ¯ ected in the stronger binding of cAMP-CRP to CRP1 ( Fig. 4 ; Lo-bocka et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Also based on the SANS results , Krueger et al. have ﬁnally suggested that the cAMP binding induces conformational change in CRP from the open form to more compacted closed form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Based on the SANS measurements , Krueger et al. have revealed that the T127L / S128A-CRP , already in the active-conformation , undergoes little , or possibly no , structural change upon binding of cAMP [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>A precise insight into the interdomain network in cAMP -- CRP by Passner et al. [ 17 ] has enabled them to predict a movement of the β-hairpin ﬂap consisting of β-strands 4 and 5 , upon cAMP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Bending of DNA is also caused by the binding of the cAMP.CRP complex as in the case of the lac and otherpromoters ( Kim et al. , 1989 ; Wuand Crothers , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>One is that osmotically induced changes in DNA structure could influence DNA binding by CRP -- cAMP , as has been proposed for osmotic control of the proU system ( Jordi et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The addition of cAMP to a ﬁnal concentration of 5 mM did not affect glucose repression ( see Fig. 7 ) , indicating that the glucose effect is due not only to the reduction of the cAMP-CRP level , as reported by Zhang et al. ( 35 ) , but also to other mechanisms , such as inducer exclusion , which is mediated by the phosphoenolpyruvate-dependent sugar phosphotransferase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The putative cAMP-CRP binding sequence resembles that of canonical FNR ( Scott et al. , 1995 ) , but it appears that FNR is not involved in the regulation of gntK expression , because the gluconate Transcriptional attenuation within the beginning of gntU The CAT activity from pGNTKU-CAT7 also depended on cAMP , like pGNTK-CAT1 but it was three to sixfold lower under all conditions tested ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The expression of gapA , encoding GAPDH , could be related to CRP -- cAMP ( Charpentier and Branlant , 1994 ) , a regulatory complex indicating that energy is in short supply , and orchestrating gene regulation cascades in response to the growth conditions and environmental cues ( Domka et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22176949" target="_blank">22176949</a>]<br></li>
<li>The RNA is transcribed from the spf ( spot forty-two ) gene , whose expression is negatively regulated by the cAMP -- CRP complex ( Joyce and Grindley 1982 ; Rice and Dahlberg 1982 ; Polayes et al. 1988b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>The gntR gene is constitutively expressed , whereas the gntKU genes are positively regulated by the cAMP -- CRP complex and negatively by GntR , and gntU expression is further repressed by attenuation , which occurs after gntK ( Izu et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>The dual regulation by GntR and GntH in addition to that by cAMP-CRP is proposed to be important for expressional control of the GntII genes and also for switching of the GntI to GntII systems along with cell growth as demonstrated [ Tsunedomi et al. , 2003 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>The cAMP -- CRP complex activates many promoters of cata-bolic genes ( Busby et al. , 1995 ; Busby and Ebright , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1964">
The Salmonella flhDC operon is also known to be activated by the cAMP -- CRP complex ( Kutsukake et al. , 1990 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></label>
<input id="act_similarity_1964", type="checkbox">
<div class="ACT_similar">
<li>The flhDC operon of Salmonella is also positively regulated by cAMP -- CRP ( Kutsukake et al. , 1990 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li></div>
<li>However , csiD expression is strongly activated by cAMP-CRP ( Marschall et al. , 1998 ) , and the presence of cAMP-CRP compensates for the absence of the C ( 213 ) / K173 contact , as observed in the present study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Monitoring dnaK expression in the presence or absence of CRP -- cAMP in vivo As proteins such as DnaK co-purify with s ( Jishage et al. , 54 1996 ) , it is possible that the CRP -- cAMP complex may activate transcription of another gene ( such as dnaK ) which , in turn , acts as an anti-s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Although Wang et al ( 2005 ) observed a negative regulation of luxS expression by cAMP and CRP , an alternative explanation is MicA involvement as it is up-regulated in a cya-deficient background [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20976191" target="_blank">20976191</a>]<br></li>
<li>Thus , if the level of cAMP is high in cells cultured in the presence of 0.04 % glucose , negative regulation by the cAMP-CRP complex probably occurs in the puuR gene expression ( Shimada et al. 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Disassembly of the repressosome in the presence of D-galactose ( Semsey et al. , 2006 ) results in derepression of both promoters , allowing differential regulation of the promoters by the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In 70 addition , csiD expression is activated by cAMP-CRP , which binds to a site centred around 268.5 upstream from the transcriptional start site ( Weichart et al. , 1993 ; Marschall et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>In accordance with previous work , the expression of spf was enhanced ( Sahagan and Dahlberg 1979 ; Polayes et al. 1988b ) and the expression of galK was reduced in cells lacking the cAMP -- CRP complex ( Fig. 3 , lanes 5,6 ) or deficient in cAMP and CRP ( Fig. 3 , lane 2 , glucose grown ; Ullmann et al. 1979 ; Joseph et al. 1981 ; Queen and Rosenberg 1981 ; Adhya 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>In addition , the intracellular cAMP level is strongly reduced by CRP ( Botsford and Drexler , 1978 ) due to the negative regulation of adenylate cyclase by CRP ( Aiba , 1985 ; Mori and Aiba , 1985 ; Botsford and Harman , 1992 ; Postma et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>In this regard , this would be similar to the reduced affinity of the cAMP-CRP complex for the melibiose operon in the absence of its activator MelR-meli-biose ( Belyaeva et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>In contrast , recent MD simulation by Berrera et al. showed open conformation of both subunits in cAMP -- CRP [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Accessory components are acetyl phosphate , which contributes to RssB and perhaps also to ArcA phosphorylation ( Bouché et al. 1998 ; Pena-Sandoval et al. 2005 ) , as well as cAMP-CRP , which in log-phase cells has an indirect inhibitory effect on rpoS transcription ( Lange and Hengge-Aronis 1994 ; F. Mika and R. Hengge , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16291649" target="_blank">16291649</a>]<br></li>
<li>For example , glucose lowers the rate of transcription of the lac operon by reducing both intracellular cAMP and CRP levels when added to cells growing on other carbon sources ( Ishizuka et al. 1993 , 1994 ; Tagami et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>A striking ﬁnding is that a large increase ( about 100-fold ) in cAMP synthesis occurs when the crp gene is defective : in other words , the synthesis of cAMP is strictly down-regulated by CRP ( reviewed by Botsford and Harman 1992 ; Postma et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Moreover , work on the galactose operon has revealed that the cAMP -- CRP complex , directly or indirectly , influences synthesis of the products of this operon differentially ( Ullmann et al. 1979 ; Joseph et al. 1981 ; Adhya 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>As glucose is consumed , cAMP and CRP levels increase and the cAMP -- CRP complex activates a set of operons whose products are necessary for the recruitment and metabolism of alternative carbon sources ( Ullmann and Danchin , 1983 ; Ishizuka et al. , 1993 ; Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>However , Reznikoff et al. ( 1982 ) observed an extremely weak transcript initiated at about -45 in an in vitro reaction containing 25 % glycerol , but it was inhibited by cAMP + CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>To study the effect of CRP -- cAMP ( Dumay et al. , 1996 ) , 0.4 % w/v of glycerol was used as the sole carbon source instead of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Because of its cAMP-CRP dependency , csiDp is activated exclusively under carbon starvation conditions and stationary phase in LB ( Weichart et al. , 1993 ; Marschall et al. , 1998 ; Germer et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731280" target="_blank">14731280</a>]<br></li>
<li>CRP was puriﬁed as described by Savery et al. ( 1998 ) using the method of Ghosaini et al. ( 1988 ) and was activated by the addition of 200 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>cAMP-CRP ( Jackson et al. 2002a ) , RpoS ( Adams and McLean 1999 ) and OmpR ( Vidal et al. 1998 ) stimulate or repress bioWlm formation in E. coli by regulating , directly Communicated by Jorge Membrillo-Hernández .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>These include activating factors as , e.g. cAMP-CRP , H-NS and CsrA as well as repressing factors such as RcsB/A , OmpR and LrhA ( recently summarized in Prüß et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537210" target="_blank">17537210</a>]<br></li>
<li>C. Minimalrecognitionsequencesfor cAMP-CRP and CytRproteins ( de Crombruggheet aL , 1984 ; Rasmussenet al. , 1993 ; this work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>When compiexed with cAMP , CRP undergoes a conformational change ( Krakow and Pastan , 1973 ; Eilen et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>It has been shown that the regulation of the ¯ hDC operon of E. coli and S. typhimurium is affected by increasing concentrations of glucose ( via cAMP-CRP ) , NaCl , and ethanol , and also by temperature increase ( Macnab , 1996 ; Kutsukake , 1997 ; Shi et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>This may also explain why CRP -- cAMP did not show any signiﬁcant effect on the expression of XylR activated operons under activating conditions with high inducer concentrations ( Hugouvieux-Cotte-Pattat et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>It also leads to a decrease in the intracellular levels of cAMP and the cAMP Receptor Protein ( CRP ) , which are required for the optimal expression of a number of catabolic operons ( Epstein et al. 1975 ; Ishizuka et al. 1993 ; reviewed by Botsford & Harman 1992 ; Kolb et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li>
<li>CRP -- cAMP represses the s - dependent dctA 54 promoter when the R. meliloti dct system is reconstituted in Escherichia coli ( Wang et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Keywords : CRP , cAMP , two-component regulatory system Escherichia coli can use C4-dicarboxylates ( succinate , fuma-rate , malate , and aspartate ) as sole carbon and energy sources during both aerobic and anaerobic-growth ( Engel et al. , 1992 ; Golby et al. , 1998 ; Golby et al. , 1999 ; Abo-Amer et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>In the presence of CRP -- cAMP , the g-GT activities were reduced in E. coli cya mutant TP2006 and cya crp double mutant TP2339 -- 1 ( data not shown ) , a reduction which parallels the reduction in b-galactosidase activities observed with the glnAp1p2 : : lacZ fusion ( pKU100 , which the dctA promoter was shown to be repressed by the CRP -- cAMP complex ( Wang et al. , 1993 ; 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>edtsx mRNA are indicated by arrows Genetic experiments have previously indicated that tsx expression is mediated by two differently regulated promoters : a minor promoter , PI , which is negatively controlled by DeoR and a main promoter , P2 , which is repressi-ble by CytR and activated by the cAMP/CRP complex ( Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Taken together , the results of the S 1 mapping experiment mad the regulatory features of tsx gene expression ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) strongly suggest that transcription initiating from ~ he weak tsx P1 promoter is repressible by DeoR , whereas the activity of the main tsx promoter , 1 ' 2 , is stimulated by the cAMP/CRP complex and negatively controlled by the CytR repre ~ sor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Gene expression from the Pa promoter is also under a strict global control system mediated by the cAMP -- CRP complex that allows expression of the mhp catabolic genes when glucose is not available and 3HPP is present in the medium ( Torres et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>RESULTS Influence of glucose on mhpR expression As mentioned above , it has been demonstrated that the transcription of the mhp cluster from the Pa promoter is activated by the cAMP -- CRP complex ( Torres et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Expression of the deoCA genes from the P-2 promoter is known to be controlled positively by cAMP/CRP and negatively by the cytR repressor , the inducer being cytidine ( J0rgensen et al. , 1977 ; Valentin-Hansen et al. , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Transcription from the main promoter , PsdhC , is thought to be activated by cAMP-CRP and was shown to be activated by reduced FNR and phosphorylated ArcA ( Park et al. 1997 ; Shen and Gunsalus 1997 ; Cunningham and Guest 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>As the deduced amino acid sequence of TcbC indicates that it has similarity to histone-like proteins ( Sawers et al. , 1998 ) , it is conceivable that it alters the topological status or distorts the DNA of the tdc promoter such that it is more readily activated , in particular by the cAMP ± CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>First , P5 is a catabolite-sensitive promoter that requires the cAMP -- CRP complex for its function ( Nagai et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Indeed , we have previously observed that when DctD is fully induced by DctB and succinate , the dctA promoter is much less sensitive to repression by the CRP -- cAMP complex ( Wang et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Microbiology ( 2011 ) , 157 , 593 -- 600 DOI 10.1099 / mic.0.043620-0 Escherichia coli mhpR gene expression is regulated by catabolite repression mediated cAMP -- CRP complex by the I. Manso , J. L. Garcıa and B. Galán Correspondence B. Galán Departamento de Biologıa Medioambiental , Centro de Investigaciones Biológicas , Consejo Superior de Investigaciones Cientıficas ( CSIC ) , Ramiro de Maeztu 9 , 28040 Madrid , Spain bgalan@cib.csic.es The expression of the mhp genes involved in the degradation of the aromatic compound 3 - ( 3-hydroxyphenyl ) propionic acid ( 3HPP ) in Escherichia coli is dependent on the MhpR transcriptional-activator at the Pa promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>The centre-to-centre distance of this CRP target and CRP2 is 52 bp , identical to the spacing between the CRP targets used in repression complex formation at the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The putative repression complex formed by CytR and CRP overlaps promoter elements for P3 , P4 and P5 , suggesting that CytR may affect these promoters in particular ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The cAMP -- CRP complex and the CytR anti-activator bind co-operatively to the promoter region forming a repression complex that overlaps the sE-dependent P3 promoter and the s70-dependent P4 and P5 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The glpE promoter was apparently controlled by the cAMP -- CRP complex , but none of the promoters was responsive to specific repression by GlpR , to anaerobiosis or to FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>However , in more complex promoters , cAMP-CRP interacts co-operatively or antagonistically with other regulators , and it may function as a repressor ( Musso et aL , 1977 ) , co-activator ( Jennings and Beacham , 1993 ) , co-repressor ( S ~ gaard-Andersen etaL , 1991b ; Hoist etaL , 1992 ; Pedersen etaL , 1991 ; Pedersen et aL , 1992 ; Gerlach et aL , 1991 ) , anti-repressor ( Forsman et aL , 1992 ) or as an architectural element ( Lobell and Schleif , 1991 ) , depending on the context .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Søgaard-Andersen L , Møllegaard NE , Douthwaite SR , Valentin-Hansen P CRP CRP distances .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Nucleic Acids Res 14:2215 -2228 Valentin-Hansen P , Holst B , Sogaard-Andersen L , Martinussen J , Nesvera J , Douthwaite SR ( 1991 ) Design of cAMP-CRP activated promoters in Escheriehia eoli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>75 , 557-566 , November 5 , 1993 , Copyright 0 1993 by Cell Press Protein-Protein Interactions in Gene Regulation : The CAMP-CRP Complex Sets the Specificity of a Second DNA-Binding Protein , the CytR Repressor Lotte Segaard-Andersen and Poul Valentin-Hansen Department of Molecular Biology Odense University DK-5230 Odense M Denmark Summary Maximal repression by the CytR protein depends on the formation of nucleoprotein complexes in which CytR interacts with DNA and with CAMP-CAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>172 , No. 10 The CytR Repressor Antagonizes Cyclic AMP-Cyclic AMP Protein Activation of the deoCp2 Promoter of Escherichia coli K-12 Receptor LOTTE S0GAARD-ANDERSEN , JAN MARTINUSSEN , NIELS ERIK M0LLEGAARD , t STEPHEN R. DOUTHWAITE , AND POUL VALENTIN-HANSEN * Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark Received 19 April 1990/Accepted 7 July 1990 We have investigated the regulation of the Escherichia coli deoCp2 promoter by the CytR repressor and the cyclic AMP ( cAMP ) receptor protein ( CRP ) complexed to cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>173 , No. 17 The Cyclic AMP ( cAMP ) - cAMP Receptor Protein Complex Functions Both as an Activator and as a Corepressor at the tSx-p2 Promoter of Escherichia coli K-12 PETRA GERLACH , ' LOTTE S0GAARD-ANDERSEN ,2 HENRIK PEDERSEN ,2 JAN MARTINUSSEN ,2 POUL VALENTIN-HANSEN ,2 AND ERHARD BREMER ' * Department of Biology , University of Konstanz , P.O. Box 5560 , D-7750 Konstanz , Germany , ' and Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark2 Received 6 March 1991/Accepted 2 July 1991 The tsx-p2 promoter is one of at least seven Escherichia coli promoters that are activated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The EMBO Journal Vol .16 No. 8 pp. 2108 -- 2118 , 1997 Protein-induced ﬁt : the CRP activator protein changes sequence-speciﬁc DNA recognition by CytR repressor , a highly ﬂexible LacI member the Henrik Pedersen1 and Poul Valentin-Hansen2 Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark 1Present address : Department of Chemistry , University of California , Berkeley , Berkeley , CA 94720 , USA 2Corresponding author The CytR repressor and the cAMP receptor protein ( CRP ) bind cooperatively to several promoters in Escherichia coli to repress transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>3812 Biochemistry 2003 , 42 , 3812-3825 Role of Protein-Protein Bridging Interactions on Cooperative Assembly of DNA-Bound CRP-CytR-CRP Complex and Regulation of the Escherichia Coli CytR Regulon † Mayy Chahla , ‡ John Wooll , § Thomas M. Laue , § Nghia Nguyen , ‡ and Donald F. Senear * , ‡ Department of Molecular Biology and Biochemistry , UniVersity of California , IrVine , California 92697 , and Department of Biochemistry , UniVersity of New Hampshire , Durham , New Hampshire 03824 ReceiVed NoVember 5 , 2002 ; ReVised Manuscript ReceiVed February 10 , 2003 ABSTRACT : The unlinked operons that comprise the Escherichia coli CytR regulon are controlled coordinately through interactions between two gene regulatory proteins , the cAMP receptor protein ( CRP ) and the cytidine repressor ( CytR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; cAMP , cyclic AMP ; CRP , catabolite activator protein ; cytR , gene encoding CytR ; CytR and DeoR repressor proteins ; A , deletion ; deoR , gene encoding DeoR ; kb , is part of this regulon ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>The CytR regulon in Escherichia coli constitutes a combinatorial regulatory system in which the cyclic AMP receptor protein ( CRP ) functions both as an independent transcriptional-activator and as an ` adaptor ' for a specific repressor , CytR ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ; ( © ) Oxford University Press S0gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>DNA sequences involved in expression and regulation of DeoR , CytR and cAMI ' : CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Cold Spring Harbor Laboratory Press , Cold Spring Harbor , New York Martinussen J , Mollegaard NE , Holst B , Douthwaite SR , ValentinHansen P ( 1989 ) A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2022">
DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></label>
<input id="act_similarity_2022", type="checkbox">
<div class="ACT_similar">
<li>DNA sequences involved in expression and regulation of DeoR , CytR , and cAMP-CRP controlled genes in Escherichia coli , p. 273-288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>DNA sequences involved in expression and regulation of CytR and cAMP/CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li></div>
<li>Martinussen , J , , Mollegaard , N.E. , Hoist , B. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1988 ) A new version of negative controt , DNA sequences involved in expression and regulation of CytR and cAMP/CRP controtled genes in Escherichia coli In DNA-protein interactions in Transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Valentin-Hansen , P. ( 1985 ) DNA sequences involved In expression and regulation of DeoR , CytR and cAMP-CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Since E. coli possesses almost all genes homologous to those in the lsr operon of serovar Typhimurium , we were curious whether the function and regulation of the E. coli Lsr transporter were similar to those of serovar Typhimurium and whether cAMP and CRP were involved in regulation of AI-transport .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP -- cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Goransson , M. , Forsman , P. , Nilsson , P. , and Uhlin , B.E. ( 1989 ) Upstream activating sequences that are shared by two divergently transcribed operons mediate cAMP-CRP regulation of pilus-adhesin in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8861216" target="_blank">8861216</a>]<br></li>
<li>Botsford , J.L. and Drexler , M. ( 1978 ) CRP and the regulation of cAMP synthesis in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Gerlach , P. , Valentin-Hansen , P , , and Bremer , E , -LCB- 1990 ) Trans-criptional regulation of the cytR repressor gene of Escherichia coli : autoregulation and positive control by the cAMP/CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>In E. coli , catabolite repression is mediated in part through the cAMP-CRP complex ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16597943" target="_blank">16597943</a>]<br></li>
<li>Seok , Glucose repression of the Escherichia coli sdhCDAB operon , revisited : regulation by the CRP · cAMP complex , Nucleic Acids Res .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>Archives of Biochemistry and Biophysics 453 ( 2006 ) 217 -- 223 www.elsevier.com/locate/yabbi EVect of salt bridge on transcription activation of CRP-dependent lactose operon in Escherichia coli Yusuf Tutar a , b , $ , James G. Harman a , c a Department of Chemistry and Biochemistry , Texas Tech University , Lubbock , TX 79409 , USA b Department of Chemistry , Cumhuriyet University , Sivas 58140 , Turkey c Chemistry Department , South Plains College , Levelland , TX 79336 , USA Received 26 March 2006 , and in revised form 8 June 2006 Available online 10 July 2006 Abstract Expression of catabolite-sensitive operons in Escherichia coli is cAMP-dependent and mediated through the CRP : cAMP complex binding to speciWc sequences in DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>, two Synechocystis species ; NtcA Sy3 , Svnechococcus species ; NtcA Sy7 , Synechococcus species ; PrfA Li , Listeria ivanovii ; PflA Lm , Listeria mono ~ wtogenes ; PtrA Sy , ~ vnechococcus species ; Vfr Pa , Pseudomonas aeruk , inosa ; Ycf Pp , Porphyra purpurea : YeiL Ec , Escherichia coli ( 3 ) have a predicted DNA-binding specificity corresponding to that of FNR rather than CRP ; ( 4 ) fail to respond to cAMP ; ( 5 ) respond to oxygen limitation ; and ( 6 ) retain all or most of the characteristic ` cysteine signature ' of FNR , which comprises three essential N-terminal cysteine residues and a fourth essential cysteine within the [ 3-roll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A protein which is related to both FNR and CRP may be assigned to the FNR family on the basis of : - an ability to complement anfnr mutation in E. coli , or its equivalent in another organism ; - a greater degree of sequence identity to FNR than to CRP ; - h a v i n g the predicted DNA-binding specificity of FNR rather than that of CRP ; - failing to respond to cAMP , or lacking amino acids which are involved in liganding cAMP ; - responding to the same physiological signal as FNR , namely oxygen limitation or - retaining some or all of the cysteine residues of FNR which are thought to be involved in its response to oxygen limitation , and which are absent from CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>20 No. 19 pp. 5392 ± 5399 , 2001 Repression of deoP2 in Escherichia coli by C conversion of a transcription activator into a repressor ytR : Minsang Shin , Soim Kang , Seok-Jin Hyun , Nobuyuki Fujita , Akira Ishihama , 1 1 Poul Valentin-Hansen and Hyon E.Choy 2 3 Department of Microbiology , Chonnam University Medical College , 5 Hakdong , Dongku , Gwangju , South Korea 501-714 , Department of 1 Molecular Genetics , National Institutes of Genetics , Mishima , Shizuoka 411 , Japan and Department of Molecular Biology , 2 Odense University , Campusvej 55 , DK-5230 Odense M , Denmark 3 Corresponding author e-mail : hyonchoy@chonnam.ac.kr In the deoP2 promoter of Escherichia coli , a transcription activator , cAMP ± CRP , binds at two sites , centered at ± 41.5 and ± 93.5 from the start site of transcription , while a repressor , CytR , binds to a space between the two cAMP ± CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>cAMP -- CRP activator complex and the Cyt repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP -- CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2039">
Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , Gerlach , P. , Bremer , E. , and Valentin-Hansen , P. ( 1992 ) cAMP-CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP-CRP-induced DNA bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></label>
<input id="act_similarity_2039", type="checkbox">
<div class="ACT_similar">
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , Gerlach , P. , Bremer , E. , and Valentin-Hansen , P. ( 1992 ) cAMP -- CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP -- CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , Gerlach , P. , Bremer , E. and Valentin-Hansen , P. ( 1992 ) cAMP -- CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP -- CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Pedersen , H. , Segaard-Andersen , L. , Hoist , B. , Gerlach , P. , Bremer , E. , and Valentin-Hansen , P. ( 1992 ) cAMP-CRP activator complex and the CytR repressor protein bind cooperatively to the cytRP promoter in Escherichia coil and CytR antagonizes the cAMP-CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li></div>
<li>E , , and Valentin-Hansen , P , ( 1992 ) cAMP-CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia cot and GytR antagonizes the cAMP-CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>CAMP-CRP activator complex and the CytR repressor protein bind co-operatively to the cytRPpromoter in Escherichia co/i and CytR antagonizes the CAMP-CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Pedersen H , Valentin-Hansen P. Protein-induced fit : the CRP activator protein changes sequencespecific DNA recognition by the CytR repressor , ahighly flexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144778" target="_blank">12144778</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2046">
Tandem DNA-bound CAMP-CRP complexes are required for transcriptional repression of the deof2 promoter by the CytR repressor in Escherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></label>
<input id="act_similarity_2046", type="checkbox">
<div class="ACT_similar">
<li>Tandem DNA-bound CAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Tandem DNA bound CAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li></div>
<li>Segaard-Andersen , L. , Mollegaard , N.E. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1990 ) Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deeP2 promoter by the CytR repressor in Escherichia coil Mo / Microbiol 4 : 1595-1601 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2050">
Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></label>
<input id="act_similarity_2050", type="checkbox">
<div class="ACT_similar">
<li>Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li></div>
<li>J Bacteriol 172:5706 -5713 Sogaard-Andersen L , Mollegaard NE , Douthwaite SR , ValentinHanson P ( 1990b ) Tandem DNA-bound cAMP-CRP complexes are required for tanscriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2053">
Sogaard-Andersen , L. , Mollegaard , N. E. , Douthwaite , S. R. , and Valentin-Hansen , P. ( 1990 ) Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></label>
<input id="act_similarity_2053", type="checkbox">
<div class="ACT_similar">
<li>Søgaard-Andersen , L. , Møllegaard , N.E. , Douthwaite , S.R. and ValentinHansen , P. ( 1990 ) Tandem DNA-bound cAMP -- CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Sùgaard-Andersen , L. , Mùllegaard , N.E. , Douthwaite , S.R. and ValentinHansen , P. ( 1990b ) Tandem DNA-bound cAMP ± CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>Moreover , the observation that CAMP-CRP recruits CytR to a deoP2 promoter with no CytR operator rules out the possibility that CAMP-CRP only acts by increasing the functional DNA-binding specificity of CytR as was recently shown to be the way the MCMl protein assists a2 binding in the MCMl-a2 system ( Smith and Johnson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Restored DNA-binding of the CAMP-CRP activator complex reestablishes negative regulation by the CytR repressor in the deoP2 promoter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Heterologous co-operativity in Escherichia coli : the CytR repressor both contacts DNA and the CRP protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The role of the tandem CRP sites in negative and positive control has been studied most intensively in the deoP2 promoter , in vt/hich transcriptional initiation is modulated 10-foid by CytR and 30-fold by the cAMP-CRP complex -LCB- Valentin-Hansen , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>The weak tsx-pl promoter is negatively regulated by the DeoR repressor , whereas the main promoter , tSx-p2 , is subjected to both negative control by CytR and activation by the cAMP-CRP complex ( 3 , 4 , 17 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The minor tsx-p1 promoter is negatively controlled by the DeoR repressor , whereas the main promoter , tsX-p2 , is subjected to positive regulation by the cAMP-CRP activator complex and negative control by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>It was shown recently that to achieve maximal repression , the CytR repressor has to interact with DNA and the cAMP-CRP complex to speciﬁcally repress the deoP2 promoter ( Sogaard-Andersen & Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Using a combination of genetic , biochemical , and gene fusion techniques on single-copy plasmids , we have analyzed one such promoter , deoCp2 , to define the DNA sequences involved in positive and negative control by the cAMP-CRP complex and the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>INTRODUCTION The structural genes encoding the ~ ystems for nucleoside uptake and metabolism in E. coli are organized in a regulon that is controlled positively by the cAMP/CRP activator complex and negatively by the DeoR and CytR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>The three most upstream rot promoters are activated by the global gene regulatory cAMP-CRP complex and negatively regulated by the CytR repressor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The gene encoding the periplasmic cyclophilin homoloque , PPIase A , in Esche-richia coli , is expressed from four promoters , three of which are activated by the cAMP-CRP complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9159398" target="_blank">9159398</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2067">
Nørregaard-Madsen , M. , Mygind , B. , Pedersen , R. , ValentinHansen , P. , and Søgaard-Andersen , L. ( 1994 ) The gene encoding the periplasmic cyclophilin homologue , PPIase A , in Escherichia coli , is expressed from four promoters , three of which are activated by the cAMP -- CRP complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></label>
<input id="act_similarity_2067", type="checkbox">
<div class="ACT_similar">
<li>Nørregaard-Madsen , M , Mygind , B. , Pedersen , R. , Valentin-Hansen , P. and Søgaard-Andersen , L. ( 1994 ) The gene encoding the periplasmic cyclophilin homologue , PPIaseA , in Escherichia coli , is expressed from four promoters , three of which are activated by the cAMP -- CRP complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Nørregaard-Madsen M , Mygind B , Pedersen R , Valentin-Hansen P , Søgaard-Andersen L ( 1994 ) The gene encoding the periplasmic cyclophilin homologue , PPIase A , in Escherichia coli , is expressed from four promoters , three of which are activated by the cAMP-CRP complex and negatively regulated by the CytR References 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li></div>
<li>Initiations from P1 have been shown to be controlled negatively by the DeoR repressor , whereas initiations from P2 are dependent on the cAMP/CRP complex and are negatively controlled by both the CytR and DeoR repressor ( Valentin-Hansen , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2071">
Søgaard-Andersen , L. , Pedersen , H. , Holst , B. and Valentin-Hansen , P. ( 1991b ) A novel function of the cAMP -- CRP complex in Escherichia coli : cAMP -- CRP functions as an adaptor for the CytR repressor in the deo operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_2071", type="checkbox">
<div class="ACT_similar">
<li>Sùgaard-Andersen , L. , Pedersen , H. , Holst , B. and Valentin-Hansen , P. ( 1991b ) A novel function of the cAMP ± CRP complex in Escherichia coli : cAMP ± CRP functions as an adaptor for the CytR repressor in the deo operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>Moreover , it was reported that CytR repressor binds to deop2 of the deo gene as a large complex sandwiched by cAMP-CRP complexes [ Søgaard-Andersen and Valentin-Hansen , 1993 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2074">
, Valentin-Hansen , P. and Søgaard-Andersen , L. ( 1996 ) CytR/cAMP -- CRP nucleoprotein formation in E.coli : the CytR repressor binds its operator as a stable dimer in a ternary complex with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_2074", type="checkbox">
<div class="ACT_similar">
<li>Kristensen , P. Valentin-Hansen , L. Søgaard-Andersen , CytR/cAMP -- CRP nucleoprotein formation in E. coli : The CytR repressor binds its operator as a stable dimer in a ternary complex with cAMP -- CRP , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li></div>
<li>1 1 no. 10 pp. 3635 - 3643 , 1992 The cAMP - CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP - CRP activator complex are involved in combinatorial regulation of the cdd promoter B.Holst , L.Sogaard-Andersen , H.Pedersen and P.Valentin-Hansen1 Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark Communicated by H.Buc ` Corresponding author Transcription initiation at CytR regulated promoters in Escherichia coli is controlled by a combinatorial regulatory system in which the cAMP receptor protein ( CRP ) functions as both an activator and a co-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2077">
Holst , B. , Søgaard-Andersen , L. , Pedersen , H. , and ValentinHansen , P. ( 1992 ) The cAMP -- CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP -- CRP activator complex are involved in combinational regulation of the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></label>
<input id="act_similarity_2077", type="checkbox">
<div class="ACT_similar">
<li>Holst , B. , Søgaard-Andersen , L. , Pedersen , H. and Valentin-Hansen , P. ( 1992 ) The cAMP -- CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP -- CRP activator complex are involved in combinatorial regulation of the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Holst , B. , Sogaard-Andersen , L. , Pedersen , H. and Valentin-Hansen , P. ( 1992 ) The cAMP -- CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP -- CRP activator complex are involved in combinatorial regulation of the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_2080">
, Valentin-Hansen , P. , and Søgaard-Andersen , L. ( 1997 ) Design of CytR regulated , cAMP -- CRP dependent class II promoters in Escherichia coli : RNA polymerase-promoter interactions modulate the efﬁciency of CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></label>
<input id="act_similarity_2080", type="checkbox">
<div class="ACT_similar">
<li>Kristensen , H.H. , Valentin-Hansen , P. and Sùgaard-Andersen , L. ( 1997 ) Design of CytR regulated , cAMP ± CRP dependent class II promoters in Escherichia coli : RNA polymerase ± promoter interactions modulate the ef ® ciency of CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Kristensen , H. H. , Valentin-Hansen , P. , and Sogaard-Andersen , L. ( 1997 ) Design of CytR regulated , cAMP-CRP dependent class II promoters in Escherichia coli : RNA polymerase-promoter interactions modulate the efficiency of CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Gene 51:255 -267 Sogaard-Andersen L , Martinussen J , Mollegaard NE , Douthwaite SR , Valentin-Hansen P ( 1990a ) The CytR repressor antagonizes cAMP-CRP activation in the deoP2 promoter of Escherichia coli-K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Sagaard-Andersen , L. , Martinussen , J. , Mallegaard , N.E. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1990a ) The CytR repressor antagonizes cAMP-CRP activation in the deoP2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Rasmussen , P.B. , Søgaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Identiﬁcation of the nucleotide sequence recognized by the cAMP -- CRP dependent CytR repressor protein in the deoP2 promoter in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2086">
Pedersen , H. , Dall , J. , Dandanell , G. and Valentin-Hansen , P. ( 1995 ) Gene regulatory modules in Escherichia coli : nucleoprotein complexes formed by cAMP -- CRP and CytR at the nupG promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_2086", type="checkbox">
<div class="ACT_similar">
<li>Pedersen , H. , Dall , J. , Dandanell , G. , and Valentin-Hansen , P. ( 1995 ) Gene-regulatory modules in Escherichia coli : nucleoprotein complexes formed by cAMP -- CRP and CytR at the nupG promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_2088">
Valentin-Hansen , P. , Søgaard-Andersen , L. and Pedersen , H. ( 1996 ) A ﬂexible partnership : the CytR anti-activator and the cAMP -- CRP activator protein , comrades in transcriptional control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_2088", type="checkbox">
<div class="ACT_similar">
<li>Valentin-Hansen , P. , Søgaard-Andersen , L. , and Pedersen , H. ( 1996 ) A ﬂexible partnership : the CytR anti-activator and the cAMP -- CRP activator protein , comrades in transcriptional control Mol Microbiol 20 : 461 -- 466 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
<li>[ 32 ] P. Valentin-Hansen , L. Søgaard-Andersen , H. Pedersen , A ﬂexible partnership : The CytR anti-activator and the cAMP -- CRP activator protein , comrades in transcription control , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Valentin-Hansen , P. , Sùgaard-Andersen , L. and Pedersen , H. ( 1996 ) A ¯ exible partnership : the CytR anti-activator and the cAMP ± CRP activator protein , comrades in transcription control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Transcription is repressed by a three-protein complex ( cAMP receptor protein ( CRP ) - CytR-CRP ) that is stabilized by cooperative interactions between CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Meibom , K.L. , Kalliipolitis , B.H. , Ebright , R. and Valentin-Hansen , P. ( 2000 ) Identi ® cation of the subunit of cAMP receptor protein ( CRP ) that functionally interacts with CytR in CRP ± CytR-mediated transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In deoCp1 , promoter control is mediated by the DeoR repressor , whereas deoCp2 is subject to negative regulation by the DeoR and CytR repressors and activation by the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex ( 29 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In CytR regulated promoters , including the udpP promoter , the cAMP -- CRP complex acts as a transcriptional-activator as well as a co-repressor for the CytR protein [ 5,23,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>In this system , the global activator protein cAMP receptor protein ( CRP ; also referred to as the catabolite gene activator protein , CAP ) , guides another DNA-binding protein , the CytR repressor , to a number of binding sites that share minimal sequence homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Single amino acid substitutions in the cAMP receptor prote ABSTRACT Promoters in Escherichia coli that are negatively regulated by the CytR repressor are also activated by the cAMP receptor protein ( CRP ) complexed to cAMP ; as a characteristic , these promoters encode tandem binding sites for cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Heterologous Cooperativityin Escherichia coli Promoters in Escherichiacoli that arenegatively regulated by the CytR repressor are alsoactivated by the cAMP receptor protein ( CRP ) complexedto CAMP ; as a characteristic , these promotersencodetwo binding sites for the cAMP * CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Protein-protein interactions between CytR and cAMP-CRP mediate this co-operative binding ( S ~ gaard-Andersen et aL , 1991a ; Pedersen et aL , 1991 ; S ~ gaard-Andersen and Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>As the cooperative binding of cAMP-CRP and CytR relies on both protein-DNA and protein-protein interactions and is dramatically reduced by mutations that affect these parameters ( Sogaard-Andersen al. , 1990a , b , et 1991a ; S0gaard-Andersen and Valentin-Hansen , 1991 ) , this regulatory difference might be explained by the relative affinity of cAMP-CRP for its binding sites in the two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>We have investigated in vivo the coupling between CytR regulation of the deoP2 promoter in Escherichia coli and the DNA-binding specificity of the cAMP-CRP ( cAMP receptor protein ) complex in order to obtain a more detailed picture of the role played by cAMP-CRP in CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2102">
[ 28 ] L. Søgaard-Andersen , P. Valentin-Hansen , Protein -- protein interactions in gene regulation : The cAMP -- CRP complex sets the speciﬁcity of a second DNA-binding protein , the CytR repressor , Cell 75 ( 1993 ) 557 -- 568 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></label>
<input id="act_similarity_2102", type="checkbox">
<div class="ACT_similar">
<li>Søgaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Protein -- protein interactions in gene regulation : the cAMP -- CRP complex sets the speciﬁcity of a second DNA-binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Sùgaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Protein ± protein interactions in gene regulation : the cAMP ± CRP complex sets the speci ® city of a second DNA-binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Sogaard-Andersen , L. , and Valentin-Hansen , P. ( 1993 ) Protein ± protein interactions in gene regulation : the cAMP ± CRP complex sets the speci ® city of a second DNA-binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Sogaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Protein -- protein interactions in gene regulation : the cAMP -- CRP complex sets the speciﬁcity for a second DNA binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li></div>
<li>Furthermore , E. coli CytR proteins deleted from the DNA binding domain are able to repress transcription from CytR-regulated promoters , implying that the protein -- protein interactions between CytR and cAMP -- CRP play a key role in specifying the site of action of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The inducer of CytR perturbs protein-protein interactions to cAMP-CRP Cytidine , the inducer of the CytR regulon , does not affect individual CytR-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The physiological importance of the cooperative DNA binding of cAMP-CRP and CytR to tSx-p2 is emphasized by the observation that cytidine , the inducer of CytR , does not interfere with independent DNA binding of the repressor protein ; rather , the cooperative binding of cAMP-CRP and CytR is interrupted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>three possibilities can be envisaged for how CytR interacts with deoP2 : ( i ) CytR binds to the DNA and directly contacts cAMP-CRP , i.e. the dependence of CytR on cAMP-CRP is mediated through protein-protein interactions ; ( ii ) cAMP-CRP binding to DNA induces a conformational change in the DNA which is specifically recognized by CytR , i.e. the dependence of CytR on cAMP-CRP is mediated through changes in DNA structure/conformation ; ( iii ) CytR does not contact DNA but sits on top of cAMP-CRP , i.e. a system closely resembling the Gal4/Gal80 system in yeast -LCB- Johnston etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Although CytR is a sequence-specific DNA-binding protein , it requires cAMP-CRP for effective binding to its cognate promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>These results strongly indicate that cooperative DNA binding of CytR and cAMP-CRP is achieved through protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Two mechanisms can be envisaged to participate in this heterologous cooperativity : changes in DNA conformation induced by DNA binding of the two proteins and/or protein-protein interactions between CytR and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>1 The abbreviations used are : CRP , cAMP receptor protein ; CRP1 and CRP2 , CRP operator sites 1 and 2 , respectively ; RNAP , E. coli RNA member of the LacI family of bacterial repressors , and is induced when CytR binds cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Transcription is repressed by the cytidine repressor protein ( CytR ) , a regulon-specific member of the 1 Abbreviations : RCTD , R-subunit of E. coli RNA polymerase ; bp , base pair ; cAMP , cyclic-5 ′ ,3 ′ - adenosine monophosphate ; CRP , E. coli cAMP receptor protein ; CRP1 and CRP2 , proximal and distal CRP operators , respectively ; CytO , CytR operator site ; CytR , E. coli cytidine repressor protein ; Core domain , CytR mutants with N-terminal , DNA binding domain deleted ; deoP2 , E. coli proximal deo promoter ; dNTP , 5 ′ - deoxyribonucleotide triphosphate ; udpP , E. coli uridine dephosphorylase promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>One should recall that the nucleoprotein assembly formed at , malEp-malKp is thought to involve wrapping of the DNA around a core formed by the bound molecules of CRP and MalT ( Raibaud et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>The interaction between CRP and DNA induces DNA bending by up to 90 ° ( Schultz et al. , 1991 ; Plate 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>CRP and IHF have been studied in detail and shown to bend DNA by 90 and - 1400 , respectively ( Wu and Crothers , 1984 ; Liu-Johnson et al. , 1986 ; Robertson and Nash , 1988 ; Thompson and Landy , 1988 ; Schultz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>CRP binds to gal DNA at the -30 to -52 region , which contains the 5 ' - AANTGTGANNT consensus sequence developed by Ebright et al. ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>To avoid any potential interaction between CRP and the aCTD of Es54 , the CRP mutant was used , which is unable to interact H159L with the aCTD of Es70 , but binds and bends DNA as wild type CRP ( Zhou et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Likewise , it has been established that another set of amino acids of CRP are critical for stimulation of transcriptional initiation , but are not critical for either DNA binding or DNA bending , at both gal - and lac-like promoters ( Bell et al. , 1990 ; Williams et al. , 1991 ; Eschenlauer and Reznikoff , 1991 ; Reznikoff , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>In the CRP and Trp repressors , changes analogous to these are observed upon ligand binding in the domain of the protein that interacts with DNA ( Heyduk & Lee , 1989 ; Lawson et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>By analogy with CRP ( Buckle et al. , 1992 ; Zinkel and Crothers , 1992 ) , it has been proposed that binding of FIS and the consequent bending of the UAS DNA ( Verbeek et al. , 1991 ) stabilizes a toroidal loop ( Lazarus and Travers , 1993 ) , which could facilitate binding of RNAP either by itself or in conjunction with protein-protein interactions between FIS and RNAP ( Lazarus , 1992 ; Newlands et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7537215" target="_blank">7537215</a>]<br></li>
<li>To attempt to show that CAMP-CRP is bound to this sequence we performed gel binding assays with purified CRP and labelled DNA fragments from pKK60D75 , exactly as described by Kolb et al. ( 1983 , a and b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>( b ) CRP-induced loop breakiny in vitro is dependent upon the orientation and distances of the CRP binding site relative to ara1 CRP bends the DNA when it binds to lac operon DNA ( Wu & Crothers , 1984 ) , and to other operons that it regulates , including araCBAll ( Lichenstein et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>It should be noted that , because of the ability of CRP to bend DNA ( Kolb et al. , 1993 ) , we can not rule out the possibility of topological changes induced by the binding of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>While X-repressor binds through close contacts to the major groove of the DNA , the binding of CRP is relying more on surface-surface interactions analogous to those presentin nucleosomes between DNA and the histones ( Richmond et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>This model is compatible with the known properties of CRP , since the direction of the bend induced by the binding of CRP to its recognition site ( Liu-Johnson et al. , 1986 ) is the same as the one resulting from DNA wrapping around MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>Due to the large bend induced in the DNA upon binding of CRP , interactions of the DNA upstream of the CRP binding site with the ` backside ' of the RNA polymerase are conceivable ( Bracco et al. , 1989 ; Gartenberg and Crothers , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Fis can also affect DNA supercoiling indirectly , because of its roles in repressing transcription of gyrA and gyrB to reduce the DNA gyrase activity ( Schneider et al. , 1999 ) , increasing topA transcription to enhance topoisomerase I activity ( WeinsteinFischer et al. , 2000 ) , and regulating the expression of other DNA-binding proteins such as H-NS , HU and CRP ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) that also affect DNA supercoiling directly or indirectly ( Broyles & Pettijohn , 1986 ; Gomez-Gomez et al. , 1996 ; Tupper et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Since ® s expression is known to be modulated by three DNA architectural proteins , IHF , CRP and FIS itself ( Ball et al. , 1992 ; Ninnemann et al. , 1992 ; Pratt et al. , 1997 ; Nasser et al. , 2001 ) , the differences in relative b-galacto-sidase activities of deletion mutants could be explained by binding of these regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Consistent with other studies ( Busby et aL , 1994 ; 1995 ) , AR1 in CRP is required at both upstream and downstream binding sites in all ` versions ' of the ansB promoters with CRP sites ; the CC-FF and FF-CC promoters specifically demonstrate a requirement upstream and downstream , respectively , when binding in tandem with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>] Furthermore , a semi-synthetic equivalent to ansB Pe ( CC-FF ) , derived from ML1 ( Busby et aL , 1994 ) , is active in the absence of CRP and shows enhancement by CRP binding upstream of ~ 1.7-fold ( H. Wing and S. Busby , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The consensus sequence for FNR binding is very similar to that for CRP , differing at just two symmetrically related positions in the inverted repeat ; appropriate changes at these positions can result in interconversion of CRP and FNR sites ( Bell et aL , 1989 ; Zhang and Ebright , 1990 ; Jennings and Beacham , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>, 1990 ) , and the mannitol operon repressor , which negatively regulates mannitol operon expression ( mtlR ; Figge et al. , 1994 ) , The regulatory region ot this operon is rather unusual in that it exhibits five putative binding sites for CRP ( Lee and Qaier .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>One could also imagine that the CRP molecule bound to site 1 contacts the polymerase even though its binding site is not at a permissive location ( Choy et al. , 1995 ) , and that the MalT protomers bound downstream transform this contact into a productive one .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The upstream weaker-binding CRP site 2 plays little or no role in activation , despite being centred at a position ( 101.5 ) where , potentially , it could function synergistically with the downstream bound CRP ( Belyaeva et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>The CRP preparation used was partially inactive , as judged from its ability to remain bound to a lac binding site in a gel retardation assay ( Kolb et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>The structure of CRP bound to DNA is known to atomic resolution ( Schultz et al. , 1991 ) and confirms solution studies that have shown that CRP bends the DNA by about 90 ° upon binding to its specific binding site ( LiuJohnson et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Ap , ampicillin ; Cm , chloramphenicol ; Ec , E. coli ; Km , kanamycin ; Kp , K. pneumoniae ; dnaKp , dnaK promoter ; lacp , lac promoter ; mCRP , mutated CRP-binding site ; Tc , tetracycline ; US , upstream sequences ; , resistance ; D , deletion ; : : , novel joint ; lacDX74 , complete deletion of the lac operon ; R glp * is mutation near the argH gene that allows growth of this strain on glycerol ( Roy et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The contactsare compatiblewith complex and thatproper binding of CAMP-CRPto promoter a model in which the promotD erNA is wrapped around a complexof two CRPsandoneCytR.IntheRNA targets was a prerequisite for negative regulation ( Sbgaard-polymerase-CRP complex , the CRP-1 phosphate con - Andersen et al. , 1990b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>The increased spacing between the 210 and CRP binding site is reminiscent of that observed for the MerR-dependent mer promoter ( Summers , 1992 ; Ansari et al. , 1995 ) and SoxR-dependent regulation of soxS ( Hidalgo and Demple , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The activity of the CRP -RCB- cAMP-dependent bgl promoter is also enhanced and expression of the operon is activated by spontaneous mutations that disrupt a silencer sequence located upstream of the promoter , and by point mutations that improve the CRP-binding site ( Reynolds et al. , 1981 , 1986 ; Schnetz & Rak , 1992 ; Singh et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
<li>In an in vitro system in the absence of CAMP-CRP , RNA polymerase binds preferentially to a site located 22 bp before the principal ( Pl ) binding site ( Malan & McClure , 1984 Malan et al. , 1984 ) and under certain reaction conditions can result in initiation of a transcript from this position in vitro ( $ - U-U-C-C-G - ; Reznikoff et al. , 1982 ; Malan & McClure , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>Gel retardation and DNase I footprinting assays As CRP binds to the cai ± ® x regulatory region ( Buchet et al. , 1998 ) , bandshift assays were performed in the conditions favourable to CRP binding , as described by Sogaard-Ander-sen and Valentin-Hansen ( 1993 ) , with minor modi ® cations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Although it was not possible to detect the binding of purified CRP to the Pr promoter by electrophoretic mobility shift assay ( Torres et al. , 2003 ) , we have shown a weak but detectable binding of CRP to the Pr promoter by DNase I footprinting assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Because low CRP concentrations favoured occupancy of CRP I , while high concentrations favoured binding of both CRP sites ( Beatty et al. , 2003 ; data not shown ) , bubble extension to P2DOWN thus inversely correlates with occupancy of CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Footprints for binding of either CRP or CaiF alone were found to be similar to those already reported ( Fig. 5A , lanes 3 and 5 ; Buchet et al. , 1998 ; Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>As observed previously with the linearized pacs444 template ( Beatty et al. , 2003 ; Browning et al. , 2004 ) , WT CRP elicited patterns of protection and hypersensitive ( HS ) sites characteristic of binding to both CRP I and CRP II ( Fig. 3A , lanes 6 and 19 ; Fig. 3B ; Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Interestingly , the proposed SoxS binding site overlaps both with one of the CRP and one of the ArcA binding sites ( Jeong et al. 2004 ) , suggesting the possibility of complex regulation in response to changes in the redox conditions of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18368388" target="_blank">18368388</a>]<br></li>
<li>In CRP this domain is responsible for specific binding of the effector cyclic AMP ( Weber & Steitz 1987 ; Shaw et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Dam-methylation can in ¯ uence Lrp binding ( Huisman & de Graaf , 1995 ; Nou et al. , 1993 ) , whereas CRP binding is hardly affected by methylation ( Gunasekara et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Ogden et al. [ 9 ] proposed that araBAD and araC are regulated by binding of CRP to distinct regions , CRPsAD and CRPc , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>-1 +3 -12 -10 -9 -30 -29 -28 P2A P2 RNAP IHF + RNAP 200 nM CRP + IHF + RNAP 12 nM CRP + IHF + RNAP 200 nM KE101 + IHF + RNAP CRP stabilizes the TOC in the presence of binding to lower-affinity sites by IHF Because 20 nM IHF did not exert a large effect on the RNAP footprint , we speculated that the negative effect of IHF observed previously on pacs444 in vitro ( Browning et al. , 2004 ) might be due to its binding to lower-affinity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In the presence of CAMP-CRP , however , binding of CytR is enhanced more than lOOO-fold ( Holst et al. , 1992 ; Gerlath et al. , 1991 ; Pedersen et al. , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In the presence of the low molecular effectors cytidine or adenosine , however , the repression loopis broken and CAMP-CRP at the CRP-1-binding site is free to interact with the polymerase ( Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>There is no CRP binding site in the uidA promoter region We searched for the presence of CRP binding sites in the Sau3A2-HpaI fragment by the method already described ( Fried and Crothers 1983 ; Kolb et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>Moreover , the presence of good quality binding sites for the two regulators are a prerequisite for negative regulation of a ` single CRP site ' cdd promoter ( Holst et al. , 1992 , Kristensen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Interestingly , Kumari et al [ 60 ] , inferred two TSSs based on the presence of binding sites for the regulatory Fnr and CRP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>In a similar fashion , at the acs promoter , for example , the binding of CRP to similarly placed activating sites stabilizes RNAP binding in the presence of nucleoid proteins such as FIS and IHF ( Sclavi et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Hypersensitive site 1 ( -93 ) is associated with CRP binding to CRP I ( Beatty et al. , 2003 ) , site 2 ( -94 ) is associated with FIS binding to FIS II , and site 3 ( -95 ) occurs in the presence of CRP and a high concentration of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Our in vitro studies , showed that MelR can bind to this site in both the presence and absence of melibiose , that this site accommodates just one MelR subunit , and that binding is tightened by CRP ( Belyaeva et al. , 2000 ; T. A. Belyaeva , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760179" target="_blank">10760179</a>]<br></li>
<li>Here , we used electrophoretic mobility shift assays and DNase I footprint analyses to demonstrate that the nucleoid proteins FIS and IHF each bind multiple sites within the acs regulatory region , that FIS competes successfully with CRP for binding to their overlapping and neighbouring sites and that IHF binds independently of either FIS or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Mechanisms that are independent of CRP are also involved in osmotic regulation of the P1 promoter since its expression still displays osmotic control in the absence of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Ueguchi and Mizuno ( 1993 ) have recently demonstrated that H-NS functions as a transcriptional-repressor by direct binding to the promoter region of the proVWX operon in E coli It has also been shown that , in the absence of H-NS , promo-ters may become independent of positively acting regulators such as CRP , suggesting that these regulators may act as antirepressors for H-NS ( Forsman et ai , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>The specific binding of cyclic AMP-CRP to the ompB promoter region , taken together with the results obtained in the in vivo experiments , suggests that cyclic AMP-CRP is a direct effector of ompB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Thus , the CRP binding site overlaps the putative promoters for the transcripts that are inhibited by cyclic AMP-CRP , while the promoters that are activated by this complex are downstream from the CRP-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>However , cells containing the plasmid with the altered CRP binding site showed little or no cyclic AMP-dependent increase in chlor-amphenicol acetyltransferase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Effect of deleting the CRP binding site on cyclic AMP regulation of ompB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In addition , inactivation of the CRP binding site by site-specific mutagenesis strongly reduces effects by cyclic AMP on ompB expression in an ompB-cat fusion plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Genes that are subject to catabolite repression are activated by the complex formed between cyclic AMP and its receptor protein ( CRP ) , which interacts with a conserved sequence of nucleotides close to the corresponding RNA polymerase binding site ( Valentin-Hansen , 1982 ; Chapon & Kolb , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6383359" target="_blank">6383359</a>]<br></li>
<li>This repression was indicated to be due to interference of binding of cyclic AMP-CRP to the CRP box , which overlaps with the AscG-binding site 1 , as well as binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>If polymerase is made to enter the DNA by binding in vitro at this P , site , cyclic AMP receptor protein ( CRP ) actively removes polymer-ase and redirects it to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>However , with CRP activated during growth on a non-catab-olite-repressing substrate like glycerol , nanA expression increases , presumably by RNA polymerase blocking repressor binding ( 14 ) , followed by at least another 10-fold increase resulting from inactivating NanR with Neu5Ac ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>In the case of the double mutant , the affinity for the mutated CRP-2 target is also reduced to a level comparable to that of wild-type CRP-1 ( lanes 3-5 ) , whereas no binding of CRP to the triple mutant is observed ( lanes 3-5 of -45 , -89 , - 9 8 mutant ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In general , as the FIS concentration increased , the signatures characteristic of CRP binding disappeared , replaced by signatures characteristic of complete FIS binding ( Fig. 3A , compare lane 9 with lanes 10 -- 12 ; see also Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>When FIS and CRP were added in combination , the DNase I hypersensitivity around positions 238/239 induced by binding of FIS at site II and the protection of FIS site III were noticeably enhanced ( Fig. 3A , compare lanes 13 -- 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>However , when the left half of BCRP binding sequences was truncated ( probe Pr-60 ) , the ability of this fragment to-bind CRP was almost completely abolished ( Fig. 5C , lanes 7 to 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>We propose that CRP decreases energetic barriers to RNA polymerase binding , isomerization , and abortive RNA synthesis but stabilizes the abortive RNA initiating complex , which results in increasing the activation energy of the transition state before the elongation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>We hypothesize that by stabilizing the corresponding transition states , the CRP - CTD interaction ( s ) decreases the energetic barrier to closed , open , and initiating complex formation and helps increased RNAP binding , open complex formation , and abortive RNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Finally , the possibility that the enhanced occupation of MalT sites 1 and 2 observed in the malEp + context results from a local effect , e.g. , from a direct interaction with CRP bound to the adjacent CRP 1 site , rather than from the formation of the higher-order structure is excluded by the observation that in vivo ( this work ) and in vitro ( unpublished results ) CRP binding to site 1 does not signiﬁcantly increase the afﬁnity of MalT for sites 1 and 2 in malEpD92 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The absence of any cooperativity in the binding of MalT and CRP to sites 1/2 and 1 , respectively , as revealed by the in vivo footprinting experiments makes it unlikely that contact with MalT bound to site 2 mediates this stimulatory effect of CRP , and lends support to the hypothesis of an interaction with the polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The mechanism of activation by CRP is clearly different from that occurring at the divergent malEp-malKp promoters , which are simultaneously controlled by an array of three binding sites for CRP , all located at distal positions in between ( 36 ) , and forming a high-order structure responsible for activation , assisting direct activation by the MalT primary regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The gene organization and the binding sites of CRP within the spacer regions are shown : ( A ) divergent promoters are regulated by single and the same CRP bound within spacers ; ( B ) divergent promoters are regulated by different CRP molecules bound within spacers ; and ( C ) multiple CRP molecules bind to spacer between divergent promoters , but the role of each CRP remains unidentified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Consistent with our observations at PhlyE , it has been suggested that SlyA binding to such sites remodels the local nucleoprotein structure , counteracting the effect of negative regulators such as H-NS , and clearing the way for classical transcriptional-activators such as PhoP , or as in the case of hlyE , FNR and CRP ( Ellison and Miller , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>55 different TFs binding sites were detected with high confidence , of which the most frequently found were those for CRP , Fis , PhoB , RhaS , PurR and FNR ( Table S4 and Figure S6 ) , of which , CRP , Fis and FNR are general TFs that regulate many genes in E. coli ( Table S5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838305" target="_blank">19838305</a>]<br></li>
<li>Whether the two CRP binding sites are independent or cooperate in stimulation of transcription of lsrR and the lsr operon needs further investigation ; however , our results clearly indicate that th promoters of lsrR and the lsr operon are both subject to LsrR repression and CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>Although it is not surprising that CRP shows specific binding to the two promoters , since its binding sites to the promoters are well characterized ( 15 , 18 ) , it is intriguing that ArcA binds specifically to the two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>This immediately suggests that one of the possible mechanisms of ascFB repression by AscG is competition between two regulators , AscG and CRP , for binding to their respective target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>The cryptic bgl operon is activated by point mutations that improve the CRP binding site of the bgl promoter ( 6 ) , and several other H-NS repressed genes are activated by specific DNA-binding proteins that antagonize H-NS-mediated repression in response to specific stimuli ( 41 -- 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>These results support previous data indicating that the upstream silencer is necessary for repression of the promoter by H-NS and that efﬁcient binding of CRP counteracts the H-NS-mediated repression of the bgl promoter ( Schnetz and Wang , 1996 ; Mukerji and Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Rhamnose metabolism is under catabolite repression by glucose , which is overcome by the binding of CAMP-catabolite repression protein ( CRP ) to a consensus recognition sequence found in the promoter of rbaBAD ( Egan & Schleif , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8757746" target="_blank">8757746</a>]<br></li>
<li>The binding of H-NS to an upstream silencer results in an ~ threefold repression of the catabolite gene regulator protein ( CRP ) dependent bgl promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Of the 25 CRP sites , 8 fall around -40 on the gal position , 6 are around -60 as in lac , and 6 are centered at -70 as in the malT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>, Peakman , T. , Busby , S. , Quincey , R. and Cole , Kahn , M. , Kolter , R. , Thomas , C. , Figurski , D. , Meyer , R. , Remaut , E. and Helinski , D. : Plasmid cloning vehicles Kolb , A. , Busby , S. , Herbert , M. , Kotlarz , D. and But , H. : Comparison of the binding sites for the Escherichiacoli CAMP Kolb , A. , Spassky , A. , Chapon , C. , Blazy , B. and But , H. : On the different binding affmities of CRP at the lac , gal and malT promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Addition of CRP enhanced the reactivity of the bases within the 210 region of the fis promoter to permanganate , but as observed by DNase I footprinting experiments described above , increasing CRP concentrations abolished this enhancing effect ( Fig. 4A ) , probably because of binding of CRP at a low affinity site overlapping the 210 region ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Although CRP is also necessary for activity of the Pz promoter , no binding of CRP to the Pz fragment was observed in gel retardation assays ( Fig. 6 ) , suggesting that CRP bound to Pa is able to activate the divergent Pz promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>As shown in Fig. 1 , we found that ArcA as well as CRP bound to the two promoter regions , but not to the ORF of ptsG , to a significant level , implying specific binding of these two proteins to the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>As shown in Fig. 4A , three types of complexes were observed , depending on the concentration of CRP , which may indicate that multiple CRP-binding sites are present in the cai-ﬁx promoter-regulatory region or that oligomerization of CRP bound to one or more sites might occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Characterization of open complex formation Because disruption of the lower-affinity CRP binding site , CRP II , increased in vivo promoter activity from pacs236 , we predicted that within this context , low concentrations of CRP would favour transcription , while high concentrations would inhibit it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , a degree of reciprocity of binding-site recognition can be inferred from the observation that FNR can activate transcription from a promoter containing a consensus CRP site , but only in a crp mutant , and CRP can likewise activate an FNR-dependent promoter in the absence of FNR ( Sawers et aL , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Taken together , these results strongly suggest that TcbC activates transcription of tdc by altering the local topological status of the tdc promoter and that , in the wild-type tdc promoter , the CRP binding site is misaligned to allow transcription to occur only under optimal conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>However , the position of the CRP binding site for manA relative to the promoter for & m4 suggests that it may control transcription of both genes during catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6397402" target="_blank">6397402</a>]<br></li>
<li>Since the CRP binding site located in the cya promoter region is responsible for the negative regulation of the cya transcription by CRP ( Aiba 1985 ) , the removal of this CRP site should eliminate the effect of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Third , certain promoter structures result in a situation where CRP can activate transcription by binding to an upstream site but can also repress transcription by binding to a downstream site , e.g. the crp promoter ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>We propose that CRP , binding to a near-consensus site centred at 3117.5 , activates transcription from guaP and obstructs transcription from the xseA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Furthermore , no effect of CRP was found in vivo by either mutation of the CRP binding site or use of strains that lacked CRP ( data not shown ) , indicating that under the conditions tested , CRP does not regulate the FNR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>We hypothesized that the remarkably enhanced bioﬁlm formation of the ycfR mutant when cultured in LB glu medium , but not in LB medium , may be due to regulation by CRP , and a putative CRP binding site ( 9 ) was identiﬁed between positions 70 and 83 in the ycfR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>CRP Facilitates FIS Binding at Site II both in Vitro and in Vivo -- To clarify the mechanism of synergy between FIS and CRP in repressing the fis promoter activity , we investigated the effect of CRP on the binding of FIS at site II by DMS footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Although footprinting failed to identify the exact binding site of CRP at the guaBA promoter , the most likely site at which CRP could bind to repress xseP and activate guaP is the near-consensus binding site centred at 3117.5 with respect to guaP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>At the araBAD promoter , CRP binds to a site upstream of AraC Sites I1 and I2 , and this binding is independent of the binding of AraC ( Zhang and Schleif , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Positive autoregulation of crp is caused by the binding of CRP-CAMP to the CRP binding site II located upstream of the crp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>However , in the presence of CRP , speciﬁc protection of DNase I cleavage was observed immediately adjacent to the downstream side of CRP at concentrations of a-CTD used to detect binding at the rrnB P1 promoter containing a strong UP element ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>DNAase I footprinting of RNAP at the melAB promoter Figure 6 shows a footprinting experiment in which DNAase I was used to investigate the binding of RNAP to the KK81 fragment in the presence of either MelR or MelR and CRP ( with or without melibiose ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>In contrast , at the melAB promoter , CRP binds between two MelR subunits and binding appears to be contingent upon the presence of MelR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Although the sites in the glpFK promoter do not match the consensus very well , cooperative binding of CRP to the tandemly repeated sites may be at least part of the reason this promoter is relatively sensitive to catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>A prediction that both of the transcripts are subject to catabolite repression via the CRP protein was made on the basis of the presence of four putative CRP binding sites within the gltA promoter region ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8051021" target="_blank">8051021</a>]<br></li>
<li>This is consistent with the presence of potential CRP binding sites in the acnA promoter region ( Prodromou etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>Both in vitro and in vivo ﬁndings support the model that CRP plays as a dual regulator , functioning as an activator by binding to the promoter-distal high-afﬁn-ity CRP box-1 but as a repressor by binding to the pro-moter-proximal weak-afﬁnity box-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>DNAase I footprinting of CRP at the melAB promoter In the next part of the study , stimulated by the CRP-dependent effects seen in Fig. 2 and Fig. 3 , we used DNAase I footprinting to monitor the binding of CRP to the mel regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>We suggest that the effects of CRP on the activity of the melAB promoter may be due to an increase in the melibiose-dependent occupation of binding Site 20 by MelR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Results illustrated in Figure 5B show that MelR-dependent repression of the melR promoter is unaffected when CRP binding at this target is prevented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>This indicates that CRP binding plays little or no role in MelR-dependent repression of the melR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>This yields ∆ Gi , M is the free energy change for binding of i CRP dimers to the unliganded udp promoter ( i.e. , related to the macroscopic product association equilibrium constant by Ki , M ) e - ∆ Gi , M/RT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This change , in the 5 ' - TGTGA-3 ' CRP consensus sequence ( 9 , 15 ) , abolished CRP binding to the ompB promoter region in vitro ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In the current study , we show that the binding of CRP does not substantially increase the extent of TOC formation by RNAP on the acsP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2227">
In vitro binding of puriﬁed PaaX , RNAP and CRP to the Pz promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></label>
<input id="act_similarity_2227", type="checkbox">
<div class="ACT_similar">
<li>In vitro binding of puriﬁed PaaX , RNAP and CRP to the Pa promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li></div>
<li>Consist-ent with this view , we show here that although CRP andRNA polymerase interact cooperatively at deoP2 an even higher synergismbetween CRP andCytR in binding to this promoter preventstranscriptioninitiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Activation of the lac promoter by CRP is also transiently inhibited after an osmotic upshift , as is the binding of CRP to the galD4 P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Step 2 , binding of CRP displaces RNAP from the divergent promoter and activates the fis promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Figure 7 EMSA of binding of Mlc and CRP to the pts promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>However , for both pEUcspEgfp48 and pEUcspEgfp60 , the SFI level was negligible and the differential regulation across the two strains disappeared , demonstrating that CRP-mediated PcspE activation is completely abolished in the absence of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Figure 6B shows that the CRP mutants T158A and H159L ( 52 ) , which contain mutations in AR1 , had similar levels of activity regardless of the presence of the CRP binding site , indicating that they were unable to stimulate P2 transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Although CRP requires the presence of RhaR for efﬁcient in vivo activation of rhaSR , DNase I footprinting assays indicated that cooperative binding between RhaR and CRP does not make a signiﬁcant contribution to the mechanism of CRP activation at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We found s that activation by RhaS was defective in the presence of several 70 s derivatives , most notably K593A and R599A , but only at truncated promoters that lacked the binding sites for the second activator , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>In the rhaS rhaR strain , where RhaR but not RhaS can activate , we found that the wild-type CRP binding site [ ( rhaS-lacZ ) 128 ] enabled a 30-fold increase in activation compared with no CRP binding site [ ( rhaS-lacZ ) 85 ] ( 216 versus 6.9 Miller units , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In contrast , the addition of the CRP sites 2 or 3 bp upstream of the RhaR binding site did not result in increased expression compared to ( rhaS-lacZ ) 85 , indicating that there was no contribution of CRP to activation in combination with RhaR from these CRP site positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In addition , AR mutants complementation data from PcspD with CRP site-I ( 83.5 ) alone ( Fig. 5 ) also supports that CRP activation mediated by binding to this target site takes place in a typical class I manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In contrast , our previous in vivo experiments showed that overexpression of - 235 resulted in only a two - to threefold defect at promoters that included a CRP binding site , although the total activation by CRP is normally 100-fold ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>To further examine the mode of transcription activation by MelR , in this study we have explored the binding of MelR173 to these additional sites and the effect of CRP and compared this to the binding of full-length MelR ( MelR303 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Cameron , A.D. , and Redﬁeld , R.J. ( 2008 ) CRP binding and transcription activation at CRP-S sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>In addition to these interactions , cooperative binding of MelR with CRP also enhances transcription activation ( 20,21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>In this study , CRP binding was measured at each of the selected loci and CRP-dependent transcription activation was assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>At the rhaSR promoter , CRP alone can weakly activate transcription , but binding of the other L-rhamnose-responsive activator , RhaR , is required for maximal CRP-dependent activation in vivo ( 15 ; unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>However , because classical parameters for binding and isomerization are not modified by CRP ( 5 ) , the mechanism of transcription activation at this promoter is expected to be quite novel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>To assess the involvement of CRP in the activation of transcription of cai and ﬁx operons ( Tables 2 and 3 ) , the binding speciﬁcity of puriﬁed CRP to the cai-ﬁx intergenic promoter region was tested in vitro by performing gel mobility shift assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Evidence that bending alone can activate transcription comes from replacing CRP sites by DNA-bending A-tracts ( Bracco ef ai , 1989 ; Gartenburg and Crothers , 1991 ) , and replacing X repressor sites by an IHF-binding site to give IHF-dependent XPL activation ( Giladi etai , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>For example , at the E. coli melAB promoter , CRP binds to DNA cooperatively with the second activator , MelR , to increase MelR binding to the site that is required for transcription activation ( 4 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>It is unlikely that CRP binding to site 4 contributes directly to an increase in rhaSR expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Instead , the presence of C rather than T at position 6 of the CRP half-site appears to make Sxy essential for CRP-DNA binding and transcription activation ( 13,14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Although this seemed a much more plausible position for activation of transcription , experimental verification of the CRP binding site was required .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>( ii ) Although base substitutions at the downstream site I of the acs promoter prevented CRP binding and CRP-dependent activation , those at the upstream site I of the pga promoter permitted CRP binding but rendered CRP unable to activate transcription ( Fig. 6 and Table II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>To test whether these transcriptional defects are speci ® c to Fis-dependent activation , the aCTD mutants not previously implicated in DNA binding were evaluated at another Es38-dependent promo-ter , aldB , which is activated by CRP when bound to a site 59.5 bp upstream from the start of transcription ( Figure 4 ( a ) ) .37 CRP has been shown to activate transcription at other class II promoters through the `` 287 determinant '' on aCTD ,30,33 so residues involved in Fis-dependent transcription are not expected to have an effect at the CRP-dependent aldB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Coactivation is dependent on the phasing between CRP and Fis at site I. To determine if the relative helical orientation between Fis and CRP was important for activation , a 5-bp insertion was created between nucleotides 281 and 282 in the Fis site II binding site which would shift the orientation of CRP by a half helical turn of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>NarL binding to the site results in reduced napF expression relative to NarP binding , including a reduced contribution to activation by the Fnr protein ( related to CRP ) from its DNA site upstream of the NarP/NarL site ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In addition , our results indicate that neither CRP-induced DNA bending nor cooperative binding between CRP and RhaR make signiﬁcant contributions to CRP activation at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>acs activation required the proximal DNA site for the CRP site , involved the distal DNA binding site for CRP , required AR1 of CRP , and involved the 287 , 261 , and 265 determinants of the - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>CRP activation of rhaSR expression is primarily due to neither cooperative binding with RhaR nor DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2260">
These results conﬁrmed that the DNA-binding ability of CRP is essential for PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="act_similarity_2260", type="checkbox">
<div class="ACT_similar">
<li>These results conﬁrmed that the DNA binding ability of CRP is essential for PcspE activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_2262">
Effect of defective DNA-binding domain of CRP on PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="act_similarity_2262", type="checkbox">
<div class="ACT_similar">
<li>Effect of defective DNA binding domain of CRP on PcspE activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>Full activation of both the rhaBAD and rhaT promoters requires activation by CRP binding immediately upstream of RhaS ( 13 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>However , when the promoter included the RhaR and CRP binding sites , the level of activation by RhaS and CRP was much lower than that by RhaR and CRP , suggesting that CRP could not fully coactivate with RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Three adjacent CRP binding sites are involved in the activation of the divergent maZEp-rnaZKp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Model for MelR and CRP binding during activation at the melAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Binding of FIS to this region could interfere with or exclude the binding of CRP and thus decreases or eliminate the activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>We have recently demonstrated that several of the four CRP binding sites are involved in the activation of both malEp a.nd malKp in , rh .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Comparing activation levels of operons with strong CRP sites to those with weak sites indicated that there is no signiﬁcant relationship between the degree of activation and the ` quality ' of CRP-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Therefore , conservation of CRP-binding sequence is not sufﬁcient to predict CRP activation levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The next activation peak is observed when the CRP binding site is moved one helical turn further upstream to position -81.5 , with an activation factor almost as strong as for the more promoter-proximal positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>A smaller , secondary activation peak is observed when the centre of the CRP binding site is at -63 , 2.5 bp upstream of the activation peak corresponding to a lactype promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>An additional 50-fold activation of rhaBAD expression occurs when CRP occupies its binding site centered at 292.5 , which places CRP adjacent to RhaS ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>However , our data ( Table 1 ) show that CRP binding to site 2 , centred at position 117.5 , causes little or no repression , while CRP binding to site 1 , centred at position 43.5 causes only weak activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>This suggests that CRP binding at site 3 may be destabilized at F ( rhaS-lacZ ) D216 , and this destabilization may result in CRP activation from site 3 becoming more dependent upon RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>activation of pBAn by CRT ' would depend on the distance and orientation of the CRP binding site relative to ad , as is seen with bhe CRP-induced loop breaking in , vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Coactivation both in vivo and in vitro is dependent on CRP binding to a site centered at 2121.5 , but CRP without Fis provides little activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>BAD by CRP in vivo is dependent ( c ) Activation of p upon the orientation and distance of the CR P binding site relative to araT Loop breaking by CRP could function in the activation of psAn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>This type of interaction is largely independent of the location of the CRP binding site and easily explains almost equal transcriptional activation of the -70.5 and the -81.5 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2281">
Sawers , G. , Kaiser , M. , Sirko , A. and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding-siterecognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></label>
<input id="act_similarity_2281", type="checkbox">
<div class="ACT_similar">
<li>Sawers , G. , Kaiser , M. , Sirko , A. , and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding-site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Sawers , G. , Kaiser , M. , Sirko , A. , and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li></div>
<li>Depending on the position of the CRP binding site , either AR1 or both regions are required for efﬁcient activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Some CRP activation of pSR was seen with the template carrying the CRP down mutations , suggesting that there may be some re-sidual CRP binding to the weakened CRP site 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>From the location of the CRP-binding site , CRP was indicated to be involved in activation of the ascFB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Seemingly , CRP uses different mechanisms of transcriptional activation depending on the position of its binding site , and FNR may be less versatile in this respect TM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>If transcriptional activation remains due to a direct contact between the R-subunits and CRP , at least one of the R-subunits has to move when the CRP binding site is relocated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>As expected , the decrease in transcriptional activation upon misaligning the CRP binding site is less dramatic for the activation peaks around -70 and -80 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>CRP binding in the absence of Core domain produced the expected pattern of mobility-shifted bands corresponding to species with either one or two CRP dimers bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Together with the observation that IHF can bind simultaneously with CRP , these data suggest a working model in which the binding of IHF to IHF III alters the interaction of one CRP dimer with the other and/or with RNAP itself ( C. M. Beatty , D. Thach , and A. J. Wolfe , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>This observation supports the hypothesis that the AR2 surface is involved in an interaction between RNAP and CRP bound at CRP I and that the binding of WT CRP to CRP II can stabilize this interaction and affect the structure of the resulting TOC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>CRP performs its dual role by binding to this site , although it has been suggested more recently that a second molecule of CRP can bind in a sequence-nonspe-cific way at the -60 region when RNA polymerase occupies P1 ( 8 , 20 , 21 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>Deletions of either of the two core motifs as deﬁned in Fig. 1B , or both of them together , abolished CRP binding ( Fig. 3A , lane 3 to 8 ) , thereby validating that CRP binds to the putative target site sequence , which is centered at 61.5 ( relative to the TSS ) in the cspE upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Indicated also are the UP element and discriminator sequence ( involved in growth-rate-dependent regulation and stringent control ) , sites identi cents ed for the binding of PurR and DnaA and the putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Where non-integral numbers of turns were inserted , a compensatory insert between the CR.P binding site and ara0 , was introduced so that the total insertion between ara0 , and aral was an integral number of DXA helical turns : this maintains t , h and distance between CRP to activate , but also by affecting formation of the repression loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>A possible CRP binding site in the promotor region of the mdh gene provides evidence for a co-regulation with fumA gene , the structural gene of fumarase , which is also subject to catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>The simplest model is that repression is dependent on the formation of a complex involving more than one molecule of MelR and we have found that optimal repression requires the binding of MelR at sites R , 2 , 1 and 10 , but not the binding of CRP at its targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>The repressor could in that way prevent the productive interaction of the activator complex by binding co-operatively to the tandem CRP targets ( Martinussen ef aL , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The high-affinity Fis binding site at 241 , the interactions between Fis and aCTD of RNAP , and the collaboration with CRP at this promoter may all contribute to the effective utilization of Fis , even when its levels are low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The CRP site mutant had 126 U of b-galactosidase , while the natural promoter produced 391 U. Up to a sevenfold increase in b-galactosidase activity by CRP binding has been measured in M9 glycerol medium , although the overall level of expression was considerably lower ( 80 U of b-galactosidase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The involvement of CRP in the regulation of clyA gene expression and the presence of a potential CRP binding site in the clyA upstream region suggested a direct interaction of CRP with the clyA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The in vitro fore , binding of CRP could block RNApolymerase from binding transcription results show that POb is active only when the to P l a and is in a position to act as an activator of Plb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>The partial deletion of this target site , leading to the loss of CRP binding ( Fig. 3 ) , as well as the use of CRP mutants having defective DNA binding domains ( Fig. 4A ) failed to activate PcspE transcription , indicating that the binding of CRP to the target site in PcspE is a prerequisite for this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The CRP binding site overlaps the -35 hexamer and thereby the binding of CRP represses P2 transcription not only by Ea70 , but also by Eo38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>The twofold stimulation of transcription in the presence of a functional CRP binding site was maintained in the AR2-CRP mutant , although overall transcription in both reporters was elevated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The presence of the [ 4Fe-4S ] cluster increases dimerization of FNR which is correlated with an increase in site-specific DNA binding of FNR , a property expected of transcription factors of the FNR/CRP family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The attenuated lytic activity of BEU705 could be restored both with and without the presence of oxygen by the introduction of CRP on a plasmid ( pDW300 ) , and to some extent , under anaerobic conditions only , by the introduction of FNR ( plasmid pGS24 ) or altered FNR proteins having CRP binding speciﬁcities ( plasmids pGS215 and pGS297 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The presence of a CRP-binding consensus sequence on the 50-upstream of the yeiT -- YeiA gene suggests its regulation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>The presence of CRP did not show any effect on ArcA-P binding to the site I. FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>The level of CRP binding ( Y-axis ) represents the fluorescent intensity ratio between SELEX samples in the presence and absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The presence of CRP binding is depicted by a plus sign , and no binding is shown by a minus sign .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The results , illustrated in Fig. 6 , show that CRP binds to a single site in the aer200 fragment , but binding of CRP to speciﬁc targets in the tsr and trg regulatory regions was also detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2314">
The ability of CRP to break the loop in vitro and to activate pBAD in uivo is dependent upon the orientation and distance of the CRP binding site relative to aral 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></label>
<input id="act_similarity_2314", type="checkbox">
<div class="ACT_similar">
<li>The ability of CRP to break the loop in vitro is dependent upon the distance and orientation of the CRP binding site relative to nmf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li></div>
<li>The stronger binding of CRP to the consensus CRP site resulted in a 30 to 50 % decrease in PguaB activity , depending on the growth rate ( Fig. 3D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The mutation introduced into site 4 , matPp40 , is a transition which , when present in MalT-binding site 1 , dramatically decreases matPp activity ( Gutierrez and Raibaud , 1984 ) ; furthermore , it was not expected to affect the affinity of CRP for its site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7830577" target="_blank">7830577</a>]<br></li>
<li>The last column ( * ) represents experimental data about regulation : S , known binding sites ; R , known regulation , binding site not known.Regulons : a. GntR ; b. IdnR ; c. UxuR/ExuR ; d. ExuR ; e. UidR ; f. KdgR ; g. CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The CRP site 1 down mutation resulted in a loss of detectable binding to CRP site 1 , but no change in binding to the regio of CRP sites 2 , 3 , and 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The changes induced by binding of CRP to its recognition site map to the boundaries of the CRP binding site ( Figure 2a , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The weak binding of puri cents ed CRP to fragment EP hampered attempts to locate the exact binding site of CRP by DNaseI footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The occupancy of araO2 was still further reduced upon deleting araI and the CRP binding site and was eliminated when araI , the CRP site , and araOl were all deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3041410" target="_blank">3041410</a>]<br></li>
<li>The core binding motifs in the CRP site that contribute most strongly to the interaction with CRP are overlined ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The trg promoter fragment binds CRP with similar affinity to the aer fragment , while CRP binding to the tsr fragment is much tighter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The operator DNA binding affinity of unliganded CRP is assumed equal to 0 in this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The conformation of the promoter DNA might be changed by binding of CRP , which resulted in some of the bases becoming more easily accessible to KMnO4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The change in RNAP binding at P0 by CRP was undetectable under our experimental conditions ( Fig. 8 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>The currently accepted model for the mechanism of action of CAMP is that CRP undergoes a conformational change upon binding of CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>The efficacy of our CRP preparation for binding to the mtl operon CBSs 2,3,4 , and 5 proved to be greater than that for binding to the two well-CharacterizedpckA CBSs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The CRP , FadR , and ArcA binding sites are indicated with blue , gray , and black rectangles , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The binding of CRP to both sites is enhanced by glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The effect of CRP on RNAP binding is to decrease the Kd value about 4-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The inverted repeat sequence known to be important for CRP binding is in capital letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The regions binding FIS , CRP , and RNAP are indicated by thick , thin , and dashed lines , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The stars indicate hypersensitivities induced by binding of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The arrowheads indicate hypersensitivities induced by binding of both FIS and CRP or both CRP and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>First , to test whether CRP binds speciﬁcally to E.coli CRP-S sites , we puriﬁed His-tagged E.coli CRP under native conditions and used electrophoretic mobility-shift assays ( EMSA ) to detect site-speciﬁc DNA-binding ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>DNase I footprinting conﬁrms that CRP binds to site 3 ; CRP binding to the other potential CRP sites at rhaSR was not detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>First , the pattern of synergistic activation due to CRP binding at both sites changes when CRP binds to the CRP1 site first .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In conditions where CRP is active , but melibiose is absent , MelR is made and occupies binding sites 2 , 1 , 10 and site R , but it can not occupy site 20 and activate the melAB promoter , and hence the system is ` poised ' ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>Although there are very few known examples of E. coli proteins binding to noncanonical DNA sites , CRP binds to a noncanonical site in the melAB promoter in cooperation with MelR bound to adjacent DNA sites ( Wade et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>To localize the binding sites precisely , we analysed the binding of CRP to these regions on the two DNA strands , by DNase I footprinting , and conﬁrmed the presence of one CRP site on each promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>To examine whether MelR that is frozen in the activating conformation has the same requirement for CRP , we constructed the JK141 promoter , which lacks MelR-binding sites 1 and 1 and the pmelAB DNA site for CRP ( Fig. 1 ) , and cloned the resulting fragment in pRW50 to give plasmid JK141-pRW50 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>In this communication we report the results of in vitro CRP-DNA interaction studies which establish that four of the five putative binding sequences bind CRP with high affinity , and that the fifth ( the most distal one relative to the transcriptional start site ) binds CRP with lower avidity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>In this study , we found that the addition of EDTA and ~ - phenanthroline inhibited the repression activity of FNR ( see Fig. 2C or D ) , Despite the sequence homology in the DNA binding domain between FNR and CRP , the two regulatory proteins respond to different signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>In lanes 13 -- 16 , CRP binding was assayed with increasing concentrations of FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>In vivo binding of CRP after addition of different concentrations of NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In vitro studies have previously noted that CRP binding to DNA is quite sensitive to increasing KCl concentrations above 0.1 M , an effect that appears to be primarily mediated by the cation ( 16 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>In vitro binding of CRP to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>In vitro binding of CRP to sites at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>So even though independent DNA binding of CytRA9-48 can not be detected , CytRA9-48 may be brought into such close contact with the DNA in the presence of CAMP-CRP that interactions with the backbone contribute to stabilizing the repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>As shown in Fig. 6 , whereas a CRP-DNA complex was observed with the Pz-Pa and the Pa probes , no binding of CRP to the Pz probe was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>On the basis of these findings , we propose that the app Y gene product is a DNA-binding protein which binds to some promoters including that of appA and stimulates transcription in analogy with CRP or OmpR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>At both promoters , direct contact between AraC and CRP may stabilize AraC in an inducing conformation , although such interactions are not required for protein binding to the DNA ( Hendrickson & Schleif , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Some coregulators act independently of CRP ; others affect CRP binding either by modifying DNA conformation or by increasing the local CRP concentration through protein -- protein contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>position function of the Sfsl protein is as a kind of DNA binding protein or interacting protein with an activating protein such as MalT or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>Quantitative analysis of cyclic-AMP induced binding of CRP from Escherchia coli to non-specific and specific DNA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Cyclic AMP binding to CRP induces conformational changes that reorient the DNA-binding domains and exposes CRP surface residues important to CRP interaction with RNA E-mail addresses : ytutar@cumhuriyet.edu.tr, ytutar@yahoo.com ( Y. Tutar ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Under the reaction conditions used , the fraction of CRP in its active , sitespecific DNA-binding form in which cytidine is bound to one but not both subunits , denoted as CRP ( cAMP1 ) , is 0.64 ( 0.2 ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>There are 21 base pairs between CRP site 3 and the RhaR binding site , which may be too far to allow cooperative binding ; however , RhaR binding has been estimated to bend DNA by 160 degrees , and CRP can bend DNA by 87 degrees ( 21 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Does the R-subunit ( s ) maintain its contact with CRP , or does it remain bound to the DNA site upstream of -35 when the CRP binding site is moved ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Nevertheless , at least one of the R-subunits remains bound to the DNA upstream of the -35 hexamer for all positions of the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Many repressor proteins , such as LacR , TrpR , and CRP , have been found to have a two-domain structure , in which one domain is involved in DNA binding while the other is involved in ligand binding ( Steitz , 1990 ; Pabo & Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>DNase I protectionexperiments revealed thatCAMP-CRP binds to the cya DNA region between +11 and -20 , in which a consensus CRP binding sequence is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>We repeated the analysis , using equally stringent search criteria , for other global DNA-binding proteins ( LexA , Fur , FruR , FNR , CRP , NarL and Fis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>Reaction mixtures were incubated for 5 min at 37 ° C , and 2 ml of puriﬁed CRP diluted in DNA binding buffer was added to the mixtures as indicated in Fig. 2 and 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>FNR differs from CRP in three amino acid residues ( Val-208 , Ser-212 and Gly-216 in FNR ) which are known to be critical for specific binding of CRP to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Since bot8h ,4 raC and CRP bend the DNA and bind closth to each other along the DNA , the proteins caould interfere with each ot ' her ` s binding to the DIVA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>This fact suggests that binding to DNA and associations with CRP are coupled structurally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>DNase I footprint analysis of MelR and CRP binding to pmelAB DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Our finding that the binding of CRP to DNA requires a helper factor is unusual .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Band shift assays for the binding of CRP to DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>How is CRP binding to DNA regulated by osmolarity ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>With respect to the number of DNA-binding sites , Dan is different from the well-characterized global regulators so far examined by the genomic SELEX , including CRP , Cra , RstA , PdhR , RutR , TyrR , NemR , AllR and CitB ( 7 -- 10,12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>There is also a perfect match in this region between the extent of protection towards the 5 ' end of the upper DNA strand and the register of CRP induced DNasel enhanced cutting sites , suggesting the importance of this DNA sequence for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Here , FIS binds to sites that overlap tandem DNA sites for CRP , probably hindering the binding of the activator during exponential-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Is the sensitivity to the relative angular orientation of the proximal and distal sets of MalT sites a property intrinsic to MalT or is it due to the increased torsional stiffness of the intervening DNA resulting from CRP binding ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>One of its function is architectural : via binding to the three CRP binding sites and bending the DNA , it facilitates cooperative binding of MalT onto sites 1/2 and the distal sites ( Richet & Sogaard-Andersen , 1994 ) , thereby ensuring high occupancy of the critical MalT sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Taken together , these results support the hypothesis that the CRP binding site at rhaSR , but not the RhaS binding site at rhaBAD , is required for the RhaS-depen-dent decrease in rhaSR expression , or negative autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Our results strongly suggest that binding of CRP to the site located at position 117.5 is required for repression of PguaB at low growth rates and that this plays a role in GRDC of P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Disruption of CRP guaB binding to PguaB results in derepression of PguaB activity in media supporting low growth rates and concomitantly abolishes GRDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Thus , the occupation of IHF III both increases the probability that a TOC will form at acsP2 and , as described above , increases the probability that the TOC eventually will become poised , i.e. when the CRP concentration rises to levels that favour binding at CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Higher concentrations of CRP inhibited DNase activity in these footprinting reactions , so we can not rule out the possibility that CRP is capable of binding to sites 2 and/or 4 at higher protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>β-Galactosidase expression can not be activated by CRP protein since there is no binding site for this protein in the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>It merely summarizes t #hedata to state that the loop is destabilized when the CRP binding site is near the AraC protein , but not otherwise .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>No binding of CRP on mutated site 1 was observed , whatever the protein concentration used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>One is CRP protein bound at 2121.5 , but we have not observed any cooperative binding between Fis and CRP at proP that could support such a model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Therefore , the unexpectedly low affinity of CRP binding can be attributed to partial inactivation of the purified protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Full induction also requires binding of the catabolite repressor protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17173282" target="_blank">17173282</a>]<br></li>
<li>In operons where CRP is the sole activator , its binding site is centred at positions -41 -5 , -61.5 or -71.5 relative to the transcriptional start site , and CRP directly stimulates transcriptional initiation by RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>By contrast , when CRP acts together with another transcriptional-activator , and CRP is not the primary activator , its binding site is not localized at these positions ( Kolb e t al. , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Site-speciﬁc alterations of the CRP binding site to match the consensus resulted in substantially higher levels of ClyA expression , while alterations that were predicted to reduce CRP binding reduced ClyA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In contrast , the altered CRP binding site that matched the consensus more closely resulted in substantially increased ClyA expression in crp1 but not crp strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In particular , H-NS directly or indirectly increases positional stringency for the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>It is not possible to draw conclusions about how or whether differences in the RhaS - 70 and RhaR - 70 interfaces might relate to this difference in CRP binding site position since all but one of the 70 derivatives tested were defective only in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>This result argues for the absence of a specific CRP binding site in the Sau3A2-HpaI fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2397">
A , binding in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></label>
<input id="act_similarity_2397", type="checkbox">
<div class="ACT_similar">
<li>C , binding in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li></div>
<li>If this were the case , it would explain the absence of CRP binding to site 2 when footprinting CRP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>No binding to CRP site 2 is observed in the absence of glycerol ( lanes 4 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Increasing protection from DNase I mediated cleavage is observed as the CRP concentration is increased , indicating site-specifi binding of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>For the CRP12 and CRP22 mutants , no specific binding of CRP to either mutated CRP site was observed , and intrinsic CRP binding to the nonmutated site was indistinguishable from what was observed for wild type udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>No specific binding of CRP to either mutated site was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Further , the demonstration of CRP binding to the target site , centered at 61.5 , in PcspE ( Fig. 2 ) strongly suggested a direct interaction of CRP with PcspE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>It is interesting to note that a CRP binding site has been bound in a similar position in the CRP operon ( Aiba , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>It was thus of interest to examine the effects of the CRP binding-site mutations under these growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>It is interesting to note that this upstream interaction is conserved , whereas the interaction downstream of the CRP binding site is lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Effect of CRP binding site mutations on the in vitro activity of maZEp and malKp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>homologous sequence in CRP site 2 is : 5 ' g-A-A-t-T-G-T-G-A-g-c 3 ' CRP seems to-bind with similar affinities to site 1 and 2 in binding buffer containing 25 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Even when the CRP-binding-site sequence is not optimal for CRP binding alone , other factors , such as AraC binding , may stabilize CRP binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Deletion of a putative CRP binding site abolished carbon starvation-mediated induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>This observation suggests that residues between positions 133 and 118 , which include the CRP binding site , contribute in some way to GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>This stimulation was not found when the ilvB template lacked the proposed CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>This suggested that the CRP binding site is required for maximal repression by RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Here , on the contrary , we propose that the occupancy of the overlapping site by CRP prevents FIS binding and therefore alleviates the repression by FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>How then can CRP binding at site 1 antagonize GcvA-medi-ated repression ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>These results further support the importance of CRP binding to CRP I and the contribution of CRP binding to CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>These results demonstrate that CaiF facilitates the binding of CRP and vice versa , suggesting that both regulators bind co-operatively to the F1/CRP2 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Protein-binding sites are as follows : RNA polymerase for araBAD , +23 to -39 ; araI2 , -38 to -54 ; araIl , -54 to -73 ; CRP , -82 to -110 ; araOl , -108 to -146 ; RNA polymerase for araC , -112 to -170 ; araO2 , -269 to -294 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Binding of IHF to low-affinity sites inhibits RNAP binding in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Simultaneous binding of IHF , CRP and RNAP results in protection between the tandem CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Taken together , these data show that RNAP , CRP and IHF can bind simultaneously at saturating concentrations for their respective speciﬁc binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>It is thought that Sxy might stabilise the binding of CRP to the low-affinity CRP-S sites , although the precise mechanism is unknown ( Cameron and Redﬁeld , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>, 1989 ; Jennings & Beacham , 1993 ) that hybrid binding sites of this type respond to both CRP and FNR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Mutations that were predicted to decrease the strength of CRP binding were introduced to each of the four CRP sites i the rhaSR-rhaBAD intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Arrows indicate hypersensitive sites induced by CRP binding , while stars show those induced by FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Hypersensitive sites induced by CRP binding are shown by arrows , while those induced by FIS are indicated by stars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Suggested binding sites for CRP , NAC , PhoB , and NsrR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>Using restrictionalanalysis to separate , delineate and destroy the various putative CRP bindingsites , all five sites were shown to be functional for CRP binding in witno .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>This is in contrast to other binding of SAM plus such as TrpR or that undergo large regulatory proteins , CRP , on the binding of effector which affect conformational changes specific binding to target DNAs ( 5,6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862523" target="_blank">7862523</a>]<br></li>
<li>However , this negative e $ ect is seen at CRP concentrations below those required for binding of CRP2 in vitro ( Figs. 1 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>At cddP , initial binding of CRP to is primarily to CRP1 where it will activate via a class II mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Effect of CRP on expression and binding to the Pa and Pr promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>Because FIS bound at FIS III displaces CRP from CRP I , we suspect that simultaneous binding occurs at CRP I and Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>1989 ) , by forcing AraC onto araf , , CRP would lower the affinity of AraC binding to aral , as we observed ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Stronger binding of CRP to P ( 133 guaB to 36 , A-111C ) was conﬁrmed by EMSA ( Fig. 3E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>It would be reasonable to assume that the ligand binding to CRP ﬁrst provokes the local perturbation at the directly contacting points , which then propagates to whole molecule and leads to global transition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>This result is in agreement with the observation , noted in Fig. 2 , that CRP 4 and CRP 5 exhibit sequences which closely approximate the CRP binding consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>This shows that the loop-breaking effect of CRP requires binding to the DSA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>This suggests some sort of functional interaction between CRP and XylR , e.g. co-opera-tive binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>This form of FNR is therefore the equivalent , in terms of binding preferences , to CRP EV181 ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This is consistent with the conclusion that CRP binding is noncooperative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Autoradiogram of EMSA , analyzing the binding of puriﬁed CRP to end-labeled fragments of the acs-nrfA intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Free energy changes obtained for CRP binding alone in this experiment were DGl ,1 5 211.8 6 0.2 kcal/mol and DGl ,2 5 213.3 6 0.1 kcal/mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Concerning the lack of binding of CRP to the ribB , yidW and slp targets in vitro , it is possible that other factors are necessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Cyclic AMP binding to CRP results in an up-Weld shift in the proton NMR signals of R82 and W85 indicating a change in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Cyclic AMP binding pocket of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Discussion Cyclic AMP binding aVects CRP conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Kolb , A , , Spassky , A. Chapon , C , Blazy , B , , and Buc , H , -LCB- 1983a ) On the different binding affinities of CRP at lhe iac .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Rep Rep CRP CRP rAMP rAMP 1 2 I t M : s = z - J Is repressor binding influenced by CRP interactions ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>However , the assays provide no evidence for speciﬁc binding of CRP to the ribB , yidW and slp fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>No clear binding of CRP was found with the ﬂiC / ﬂiD , ﬂgMN , ﬂgKL , motAB/cheAW or tar/tap/cheRBYZ fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>CRP activates peAD tenfold when it 's binding sit , e is directly adjacent to aral .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>CRP1 has a much lower afﬁnity for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Additionally , CRP binding to the gap between Site 1 and Site 2 increases the efficiency of this process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Implications of PrpR and CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>CRP binding to the malEp-maEKpregion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>Preliminary in vivo DMS footprinting indicates that CRP does bind at P1 in an osmo-larity-sensitive manner , while the binding of other proteins such as Fis and Lac repressor are not sensitive to increasing medium osmolarity ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Repressor replaces CRP in site 2 ( operator ) in less than 4 minutes , as shown by the displacement of CRP specific interspersed bands in the ` footprint ' by repressor specific ones , allowing an upper estimate for ` the half-life of the CRP-site 2 complex of less than 1 minute , if one assumes independ-ent UNA binding of these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Hence , CytR and RNA polymerase donot form co-complexes at deoP2 , and thepresence of even low concentrations of the repressor prevents the formation of the transcription initiation complex ( lane 15 ; the cleavage patterns seen in lane 14 and 15 are identical to those obtained with CRP and CytR alone , lanes 16 and 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>The apparent explanation for our observation is that CRP makes a direct interaction with MelR bound to Site 1 and Site 2 and this recruits the CRP dimer to the 20 bp gap between the two bound MelR subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The model accounts for Core domain association only when CRP is bound to both CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The pairwise interaction between CytR and CRP bound to CRP1 is greatly reduced , but because CRP2 is the weaker site and as such is rarely occupied by itself , the reduction in pairwise CytR-CRP2 has a negligible effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Not only is the relative affinity of CRP higher for CRP1 than for CRP2 , but also the absolute affinity of CRP for CRP1 is unusually high .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CRP binds separately to CRP1 and to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This concentration of CRP yields 0.98 and .0.99 fractional saturation of CRP1 and CRP2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Affinity for CRP2 is higher than for CRP1 as evidenced by the CRP concentration dependence of the protection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>A large effect is observed in the case when CRP is bound to CRP2 but not to CRP1 ( reduced-valence CRP1 cddP ) , suggesting a substantial-Δ decrease in pairwise cooperativity , G23 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In contrast , there is essentially no effect when CRP is bound to CRP1 but not to CRP2 ( reduced-valence CRP2 cddP ) and only a small effect when CRP-is bound to both sites ( wild-type cddP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In activation , one consequence of the competition between CytR and CRP is to facilitate configurations in which CRP is bound either to CRP1 or to CRP2 , but not to both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast , it has a negligible effect on pairwise cooperativity between CytR and CRP bound to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In dup1 , CRP was bound to CRP2 and to each CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Therefore , it is tempting to hypothesize that CRP and/or CytR will bind to one or more of the putative CRP targets present in the DNA sequence in front of cytR and that this region is involved in regulation of expression of the repressor gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>In contrast , the tsx-500 mutation , which is located outside of the two CRP targets , affects only CytR regulation of tSx-p2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Table 2 summarizes the expression data and show that the ability of the different CRP proteins to activate deoP2-wt varied in parallel with the efficiency of regulation by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2477">
This work and previous reports regarding CRP and CytR regulation show that E. coil uses activating and repressing modules in different stoichiometric and structural arrangements to differentially adjust the expression of genes belonging to the same regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_2477", type="checkbox">
<div class="ACT_similar">
<li>In summary , this work conﬁrms previous reports on CRP and CytR regulation which show that bacteria use activating and repressing modules in different stoichiometric and structural arrangements to differentially adjust the expression of genes belonging to the same regulon [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li></div>
<li>These results , therefore , confirm our location of the cdd-promoter and suggest that all sequences necessary for proper regulation of this promoter by CytR and CRP are present in the Hga \ fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Most importantly , our results clearly showed that CytR regulation of the mutant deoP2 promoter was reestablished in the presence of either of the altered CRP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Thus , the strong reduction in promoter activity observed upon replacement of CRP-wt protein with either of the altered CRP proteins was parallelled by a reduction in the degree of CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>This decrease in efficiency of CytR regulation is not simply a consequence of the reduced promoter activ ity , as the promoter activity in the cytR ÷ strain was higher ( 1.0 ) in presence of the altered CRP proteins than in presence of CRP-wt protein ( 0.6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2483">
Brikun , I. , Suziedelis , K. , Stemmann , O. , Zhong , R. , Alikhanian , L. , Linkova , E. , Mironov , A. and Berg , D.E. ( 1996 ) Analysis of CRP -- CytR interactions at the Escherichia coli udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></label>
<input id="act_similarity_2483", type="checkbox">
<div class="ACT_similar">
<li>[ 5 ] I. Brikun , K. Suziedelis , O. Stemmann , R. Zhong , L. Alikhanian , E. Linkova , A. Mironov , D.E. Berg , Analysis of CRP -- CytR interactions at the Escherichia coli udp promoter , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>[ 38 ] I. Brikun , K. Suziedelis , O. Stemmann , R. Zhong , L. Alikhanian , E. Linkova , A. Mironov , D.E. Berg , Analysis of CRP -- CytR interactions at the Escherichia coli udp promoter , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li></div>
<li>Cooperative interactions between CytR , when bound to an operator ( CytO ) located upstream of a CytR-regulated promoter , and CRP , when bound to flanking tandem promoters , are critical to the regulatory role of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Analysis of cooperative interactions between CytR and CRP at the deoP2 and udpP promoters has implicated the importance of promoter architecture in controlling repression and induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Binding of CytR and CRP to the rpoH promoter region tested by gel shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>a , interaction of CRP and CytR with the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Here this cooperativity isolating investigated by we have mutants of the CRP protein that abolish CytR regulation without exhib-iting a concomitant loss in their ability to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>To test this idea , we have investigated CRP and CytR regulation of transcription initiation at deoP2 with reconstituted wild-type RNP and RNP derivatives lacking a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In all cases , these studies have implicated interactions of CRP and CytR with sequences located in the 80 -- 100 bp immediately upstream of the various transcription start sites and interactions between the proteins as the basis for positive and negative gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Arrows in panel C indicate sites of IS insertions and the new transcriptional start in ins1 ( position 264 , the same as in del18 ) ; newly created 235 and 210 regions are underlined DISCUSSION We used biochemical and mutational tests to better understand how CRP and CytR repressor interact with each other and target DNA to result in the regulation of transcription of the udp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>An important feature of the regulation mediated by CytR and CRP is that extents of activation , repression , and induction vary among the promoters , even among those with a similar arrangement of regulatory elements and similar binding affinities for the two proteins ( 4-6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This is achieved by nesting levels of local repression , mediated by DeoR and CytR , on a more global regulation mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Coordinate gene regulation is achieved by nesting local repression by the LacI family member , CytR , on the global activation mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Second , while mutant deoP2 with only CRP1 functional ( i.e. CRP22 ) displays nearly the same extent of CRP activation ( presumably class II-mediated ) as wild type , it is not effectively repressed by CytR ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This effect occurs first at CRP2 due to the higher affinity of CytR for its competing site ( s ) , thus helping to facilitate the switch from class I to class II CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Increasing the distance between CRP - 1 and CRP-2 by 2 bp in the wt promoter and in the promoter containing the point mutation at position -60 would be expected to result in decreased activation and increased CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Although we can not yet account for the effect of this amino acid substitution on activation , the salient point here is that amino acid substitutions in CRP can abolish CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In this view the state of CRP activation is as important as occupancy by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As such , CytR would act initially as a switch from one kinetic mode of CRP activation to another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this model , induction of expression by cytidine was shown to involve dissociation of CytR from CRP complexes , which results in transcription activation , possibly through effects on DNA topology or direct interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>In this model , CytR ± CRP and CytR ± DNA interactions block transcription activation by CRP , by sterically blocking the functional AR1 of CRP and by sterically preventing a-CTD from interacting with the DNA segment immediately upstream of the CRP dimer at the CRP1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2505">
Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymer ¬ ase for activation of transcription on the E. co/i deoP2 promoter , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></label>
<input id="act_similarity_2505", type="checkbox">
<div class="ACT_similar">
<li>Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymer-ase for activation of transcription on the Escherichia coli deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li></div>
<li>and Nielsen , P.E , ( 1993 ) Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymerase for activation of transcription on the Escherichia coti deoP2 promoler , J Biol Chem 26B : 17471-17477 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2508">
Mollegaard , N.E. , Rasmussen , P.B. , Valentin-Hansen , P. , and Nielsen , P.E. ( 1993 ) Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymerase for activation of transcription on the Escherichia coil deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_2508", type="checkbox">
<div class="ACT_similar">
<li>Mùllegaard , N.E. , Rasmussen , P.B. , Valentin-Hansen , P. and Nielsen , P.E. ( 1993 ) Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymerase for activation of transcription on the Escherichia coli deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>However , the ® nding that the formation of ( CRP ) ± CytR ± promoter complexes resulted 2 in a complete repression of deoP2 transcription ( i.e. to a promoter activity that was considerably lower than that in the absence of cAMP ± CRP ) was unexpected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In this paper , we address how the formation of CytR ± CRP promoter complexes at deoP2 might result in transcription repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>A prediction of the ® rst model is that the formation of CytR ± CRP promoter complexes should not result in repression if transcription is conducted by RNP derivatives lacking a-CTD ( Igarashi and Ishihama , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>We found that constitutive CRP-independent udp expression in many of the isolated mutants was caused by loss of part or all of CRP1 , indicating that it is important in CRP activation and CytR repression of udp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>To test these hypotheses and to assess how linkage might be involved in differential regulation of transcription , we have investigated cooperative and competitive CytR and CRP binding to the udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR has no effect on basal level transcription but instead requires CRP binding to mediate repression ( 16 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP directly interacts with CytR , thereby shifting its binding mode for activation or repression of transcription of its targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>This is unlike the protein -- protein contact-mediated repression by CytR , which involves antiactivation by contact 39 between CytR repressor and CRP activator at promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>These results indicate that CytR is highly adaptable to different arrangements of CytR and CRP sites in the absence of cytidine in order to form the three-protein repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2519">
Kallipolitis , B.H. , Norregaard-Madsen , M. and Valentin-Hansen , P. ( 1997 ) Protein ± protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></label>
<input id="act_similarity_2519", type="checkbox">
<div class="ACT_similar">
<li>Kallipolitis , B.H. , Nørregaard-Madsen , N. , Valentin-Hansen , P. ( 1997 ) Protein-protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Kallipolitis , B.H. , Nørregaard-Madsen , M. , and ValentinHansen , P. ( 1997 ) Protein-protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_2522">
Protein -- protein communication : Structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></label>
<input id="act_similarity_2522", type="checkbox">
<div class="ACT_similar">
<li>Protein -- protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li></div>
<li>First , CytR and CRP bind cooperatively to form a three-protein complex on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Together , CytR and CRP bind co-opera-tively to their respective targets , and a nucleoprotein complex with protein stoichiometries of 1:2 is formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2526">
Role of protein-protein bridging interactions on cooperative assembly of DNA-bound CRP-CytR-CRP complex and regulation of the Escherichia coli CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></label>
<input id="act_similarity_2526", type="checkbox">
<div class="ACT_similar">
<li>Chahla , M. , Wooll , J. , Laue , T. M. , Nguyen , N. , and Senear , D. F. ( 2003 ) Role of protein-protein bridging interactions on cooperative assembly of DNA-bound CRP-CytR-CRP complex and regulation of the Escherichia coli CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Thus , protein-protein communication between DNA-bound CytR and CRP underlies repression , co-activation , and induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This was interpreted to indicate that CytR mutants associate via bridging protein-protein interactions alone to form a functional , DNA-bound CRP-CytR-CRP repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank Steven A. Short for the gift of puriﬁed CytR protein , Sankar Adhya for sending us the CRP overproducer strain SA500 , and Kalpana Srivastava for technical assistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>However , there is a small but significant decrease in cooperative free energy for the three-protein complex CRP-CytR-CRP when bound to udp as compared with that seen when bound to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Despite only weak association between CytR and CRP in solution , our results demonstrate the formation of a relatively stable complex in which the Core domain forms a protein bridge between tandem CRP dimers when bound to either udpP or deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>A direct way to evaluate the linkage would be to compare association of operator-oligonucleotide-bound CRP and CytR to association of free CRP and CytR ( 26 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR forms a protein bridge between the bound CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This would suggest that CytR is primarily a protein-protein bridge , dependent on its interactions with CRP bound to the flanking CRP sites for association with its operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Recently , it was shown by DNase I footprinting in vitro that therepression complex most likely consists of one molecule CytR and two molecules of CRP , i.e. that the activatorprotein CRP is also required for repression ( S ~ gaard-Andersen eatl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Consistent with the expectation that cooperativity should enhance-the-activity of CytR as an RCTD antagonist , the three protein , CRP-CytR-CRP complex is required for repression ; CytR alone does not repress basal level transcription ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The three-protein CRP-CytR-CRP complex was shown not only to reverse CRP-mediated activation but also to actively block basal level activity , reducing transcription to a level significantly below that in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In this complex , CytR binds to an operator site ( usually referred to as CytO ) that is flanked by tandem CRP operators , CRP1 and CRP2 , as are found in most CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Implicit in this model is that CytR binds co-operatively to tandem , properly spaced CRP targets , since only a small set of CRP-activated promoters are repressed by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Lanes 1-12 show promoter DNA being titrated with increasing concentrations of CRP dimer in the presence of 4.0 µM CytR ∆ 2-63 Core domain dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>DNase I footprinting experiments performed in the presence of increasing concentrations of CytR ( Figure 3B , lanes 12 - 13 ) revealed that CytR interacts with the region located between the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The direction arrows below the helix indicate the phosphodiester bonds in the CytR-binding site that are cleaved in the DNAase I protection experiments in the presence of CAMP-CRP and CytR-wt or CytRA9-48 24 - + Glu , or CytRA9-48 to reactions containing CAMP-CRP gives rise to a single new band with a lower mobility ( Figure 4C , lanes 3-l 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In the first type of binding experiment , each protein , either CRP or CytR , is ti-trated alone ; in the second , each protein , either CRP or CytR , is titrated in the presence of a constant , near the saturating concentration of the other protein ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Second , binding of each protein ( CytR and CRP ) to each promoter is analyzed in the presence of a fixed and saturating concentration of the other protein ( CRP and CytR , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>For a system in which only intrinsic binding and cooperativity contribute , the difference between ΔGLoad , i 's obtained for binding of either protein ( CytR or CRP ) in the presence versus absence of the other yields the cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In principle , the same value should be obtained by summing the Δ ( ΔGLoad , i ) values for CRP binding to CRP1 and to CRP2 in the presence versus absence of saturating CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Competition contributes differently to ΔG1 and ΔG2 for CRP binding to CRP1 and to CRP2 in the presence of fixed CytR concentration than it does to ΔG3 for CytR binding to CytO in the presence of fixed CRP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In Fig. 2B the binding of the two CRP proteins is compared in the presence of various amounts of partially purified CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>To test this prediction quantitatively , we conducted parallel titrations of CRP binding in the presence of saturating cytidine , with and without fixed but now subsaturating CytR ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The free energy change for the association of CytR Core domain binding with the CRP2-promoter complex ( ∆ Gbridge ) was determined from paired experiments in which the promoter was titrated with CRP in the presence or absence of a fixed concentration of CytR Core domain ( 0.5-15 µM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Since this association favors promoter complexes with two CRP dimers bound , thus mimicking cooperative CRP binding , ∆ Gbridge can be evaluated by comparing CRP binding in the presence and absence of fixed concentrations of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Figure 6 depicts CRP binding to deoP2 in the presence and absence of 4.0 µM CytR ∆ 2-62 dimer ( left-hand panels ) and 13.1 µM CytR ∆ 2-63 dimer ( right-hand panels ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To address this question experimentally , matched CRP binding titrations of both deoP2 and udpP were conducted with and without CytR Core domain , as before ( Figure 6 ) , but now also comparing the presence and absence of 1 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Individual-site binding isotherms were generated by footprint titration of CRP binding in the presence ( panels B and D ) and absence ( panels A and C ) of fixed concentrations of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Parallel experiments of individual-site binding of CRP to the deoP2 promoter in the presence ( panels B and D ) and absence ( panels A and C ) of 13.1 µM CytR ∆ 2-63 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This simplification has a small but still observable effect on the accuracy of the fits to the titrations when CytR and CRP bind DNA together , with one protein in the presence of a near saturating concentration of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , the result of this pattern of effects is that the free energy change for formation of the ternary protein complex of CRP-CytR-CRP bound to DNA is only slightly smaller in the presence of cytidine than in its absence ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In the presence of CAMP-CRP and CytRA9-48 , a complex with the same borders as the CytR-wt protein is formed ; however , CytRA9-48 only results in modest changes in the DNAase I digestion pattern ( see Figure 4A , lane 5 ; Figure 4B , lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>When the same experiment is performed using CRP ( Pro-110 - * Ser ) only the fastest moving CRP/CytR/deoP2 complex appears ( lane 7 ) , and this complex is formed with a significantly lower efficiency in than the presence of CRP-wt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In the presence of both CytR and CRP , a third retarded complex with a lower mobility is observed ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>A direct connection between CytR regulation and the presence of intact tandem CRP targets in deoCp2 ( Fig. 1 ) was evident from analyses of deletion and point mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>This was done by comparing the energetics of CytR-CRP association to the energetics of cooperativity when CytR and CRP bind to deoP2 and to udpP both in the presence and in the absence of cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Second , the region between the CRP targets is strongly protected in the presence of both proteins , which suggests that this region is occupied by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>( C ) Binding of the three CytR proteins to deoP2 in the presence of CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Figure 5 presents a footprint autoradiogram from a representative experiment in which deoP2 was titrated with CRP in the presence of 4.0 µM CytR ∆ 2-62 dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>It seems fairly evident that this difference , accounting for only about a 3-fold effect on affinity , is insufficient to account for induction simply on the basis of reduced CytO occupancy Our cddP data demonstrate yet a third pattern of effects on cooperativity , in which the pairwise interaction between CytR and CRP bound to CRP1 is completely unaffected while the pairwise interaction between CytR and CRP bound to CRP1 is greatly reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>These results also a Free energy changes for intrinsic binding of CRP to operator sites CRP1 and CRP2 of the E. coli udp and deoP2 promoters and for subsequent binding of CytR Core domain to form the DNA-bound CRP-Core domain-CRP bridged complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR Core domain associates when CRP is bound to both CRP1 and CRP2 as a result of favorably oriented protein-protein contacts , despite no interaction between Core domain and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Even with its DNA-binding domain deleted , CytR will associate to form a protein bridge between CRP dimers bound to CRP1 and to CRP2 , thus indicating that direct interaction between the DNA-bound proteins is the mechanism for the cooperativity ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Additionally , while cytidine binding to CytR nearly eliminates the pairwise interactions between CytR and CRP bound at CRP1 , it has little effect on pairwise cooperativity between CytR and CRP bound at CRP2 or as a consequence on the overall cooperativity of the three-protein complex in which CRP is bound to both sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Whereas at deoP2 cytidine binding to CytR essentially eliminates cooperativity in the three-protein complex , at udpP it serves to eliminate cooperativity only between CytR and CRP bound to CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Interaction of CytR with these sites provides a complex pattern of both competitive and cooperative interactions between CytR and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As a result of this cooperativity CytR is recruited to CytO when CRP binds to the flanking sites , CRP1 and CRP2 , to yield a three-protein , DNA-bound complex , CRP-CytR-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus , it appears that CytR not only occludes interactions involving RNAP RCTD as a result of bridging interactions but also modulates the activity of CRP1-bound CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Second , it may also be significant that the three-way cooperativity , i.e. with both CRP sites filled and CytR bound in between , equals the additive sum of free energy changes for these pairwise interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Whereas the effect of cytidine on pairwise cooperativity between CytR and CRP bound to CRP2 is modest , similar to that for wild type udp , pairwise cooperativity between CytR and CRP bound to CRP1 is nearly eliminated , similar to that for deoP2 ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This model assumes no significant association of CytR Core domain when CRP is bound to either , but not both , CRP1 or CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This suggests that CytR bound to the intervening site simultaneously contacts CRP bound to both CRP1 and CRP2 and with no influence of either pairwise interaction on the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Since CytR and CRP bind in a cooperative manner , the separate interactions between CytR and CytO , and between CytR and both CRP1-bound and CRP2-bound CRP , must combine to yield a net free energy change that is more favorable than for any of these individual interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>A second surprise uncovered by our analysis of udpP is that the inducer cytidine does not eliminate all CytR-CRP cooperativity , as it does at deoP2 , but instead eliminates only the pairwise interaction between CRP bound to CRP1 and CytR bound to CytO ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Second , the free energy changes for pairwise cooperativity between CytR and CRP are the same whether CRP is bound to CRP1 or to CRP2 and are the same for udp as for deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>An important result of our study is that whereas cytidine is an effector of CytR interactions with CRP bound to udp-CRP1 , cytidine is not an effector of CytR interactions with CRP bound to udp-CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>At udpP , however , the pairwise interaction between CytR and CRP bound to CRP2 is essentially unaffected ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>'' How can CytR can make the same interaction with CRP bound to CRP1 and CRP2 given spacers of 15 and 8 bps ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At cddP , cooperativity between CytR and CRP bound to CRP1 is largely retained , while cooperativity between CytR and CRP bound to CRP2 is largely eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Presumably , either the spacing between the CytR and CRP sites or something inherent in the CytR site , such as the spacing between recognition motifs , is responsible for controlling whether interactions between CytR and CRP are still possible when cytidine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CRP bound to this duplicate site seemed not to interact with CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>By contrast , at high CRP concentration , favorable interactions with CRP bound to CRP1 and CRP2 would increase apparent affinity of CytR for its intervening site , and this would not be offset by unfavorable competitive interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>If direct protein-protein interactions provide the driving force for cooperativity , as has been presumed ( 23 58 ) , then CytR bound to the intervening site must make substantially identical interactions with CRP , whether bound to CRP1 or to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These results strongly indicate that CytR can be directed to deoP2 in the absence of specific CytR-DNA interactions and , by implication , that the protein-protein interactions between CytR and CAMP-CRP play a key role in specifying the site of action of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In contrast to the low affinity for solution assembly of CytR and CRP , these two CytR Core domain constructs show appreciable affinity for bridging between tandem , DNA-bound CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Thus , DNA-bound CRP molecules engaged in nucleoprotein formation by CytR are no longer accessible for RNA polymerase , andsterichindrance could account for negative regulation of deoP2 by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Either competition between CytR and RNAP for interaction with CRP is the main mechanism of repression , with RNAP-DNA interactions being essentially ancillary , or cytidine-liganded CytR remains bound to the DNA in a conformation which prevents CytR from acting as an allosteric antiactivator of either or both of the flanking CRP molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Gel Retardation Analysis of the Binding of CytR-wt , CytR-Ala-24-Glu , and CytRA9-48 to deoP2 and Random DNA in the Absence and Presence of CAMP-CRP ( A ) Binding of the three CytR proteins to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Figure 3 presents typical results of experiments in which deoP2 DNA was first titrated with CRP , and subsequently , CytR ∆ 2-63 was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These results suggest strong interactions between CytR and CRP rather than between CytR and any particular DNA sequence within udpP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>We study the interactions between DNA , CytR , and CRP proteins in the udpP region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>( A ) Titration of CytR and CRP on WT udpP ; ( B ) CytR-CRP-DNA interactions in WT , del252 , del264 , and del18 udpPs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Repression by the cooperative CRP-CytR-CRP complex results from competition between CytR and RNAP , both of which are recruited by CRP to-bind to the DNA sequence flanked by CRP1 and CRP2 ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>n i induced DNA bends in the circular permutation restriction nuclease cleavag CAMP-CRP causes the largest variations in complex mobilities and , therefore , induces a greater DNA bend than CytR ( Thompson & Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>For CytR to bridge the gap between promoter-bound CRP dimers and juxtapose the interacting surfaces of CytR and CRP as delineated by mutagenesis experiments ( 39 ) , the promoter DNA must be bent substantially .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>But a three way cooperative complex also forms in which CytR forms a protein bridge between tandem , DNA-bound CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>On the basis of this value for ∆ Gobridge , the connection free energy that pertains to the cooperatively bound complex in which CytR both bridges between tandem DNA-bound CRP dimers and also binds CytO is easily calculated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This indicates an interaction between CytR ∆ 2-63 and both DNA-bound CRP dimers , which is consistent with formation of a bridged complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>One can imagine scenarios in which CytR remains bound to DNA , if not to CRP , under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>One potential source of unfavorable contribution to ∆ Goconnect is from DNA bending in forming a cooperatively bound , CRP-CytR-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Whereas -7 kcal/mol is a large free energy change as compared to the heterologous cooperativity exhibited when full-length CytR binds to CRP - deoP2 ( ∆ G ≈ -2.5 to -3 kcal/mol ( 9 , 11 ) ) , 2 it indicates a very low affinity for analysis by gel-shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The effective affinity of CytR for CytO ( in the presence of CRP ) is only 1.2 times higher than cytidine-liganded CytR , yet induction is fairly robust , accounting for a roughly 20-fold increase in transcription according to reporter gene assays ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Cooperative and Competitive Interactions of CytR -- The initial indication and one functional consequence of these interactions is that while saturation of CRP1 and CRP2 greatly increases the apparent affinity of CytR for its putative operator located between these , saturation by CytR has a much lesser effect on the apparent affinity of CRP for CRP1 and CRP2 ( Table I ) ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The total effect of cooperativity with both CRP sites on CytR affinity is 100-1000-fold in different promoters and conditions ( cf. Table I and Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>DISCUSSION The most significant aspect of the CytR regulon is that a complex pattern of coordinate and differential regulation of a large number of unlinked transcriptional units is provided by interactions between CRP , CytR , and the DNA sequences of the various promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Binding in vitro of CytR and CRP to spacing variants The relative afﬁnity of the promoter variants for CytR and CRP , alone and in combination , was measured in a simple competition assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The apparent afﬁnity of CytR for selected DNA sequences and deoP2-wt , -- 9 in the absence ( black bars ) or presence ( gray bars ) of 6310 M CRP , was determined by the mobility shift assay ( see Materials and methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The interaction of CytR with udpP DNA ( full protection ) required the presence of CRP and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The formation of these complexes involves both CytR ± DNA interaction and CytR ± CRP interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Role of Competition and Cooperativity in deoP2 Regulation -- The important question remaining is how these multiple competitive and cooperative interactions between CytR and CRP regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The proteins were added according to the scheme below the autoradiograms to the following final concentrations : CRP , 1 n & l ; CytR-wt , 3 nglpl in lanes 4 , 9 , and 13 and 44 nglul in lane 15 ; CytRA9-48 , 30 nglul .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The final concentrations of CRP and CytR in ng/41 are indicated below each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The results indicate only weak association between free CytR and CRP to form a hetero-tetrameric complex in solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The free energy change we obtain for this interaction , ∆ Gobridge ) -7.8 kcal/mol , is essentially double what we estimate for solution assembly of CytR and CRP to form a heterotetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The deoC , nupG and tsx operons that are under CytR regulation were activated by CRP in our study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The central importance of CAMP-CRP in CytR repression is illustrated by several observations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The stimulation by CytR also required CRP in the cases of dup1 and dup361 but apparently not in the cases of del224 and del364 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The structure of nupG is an inversion of deoP2 , and similar interactions between CytR and CRP , no doubt , explain the apparent one-way stimulation ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The ∆ G obtained for this reaction accounts for only a small fraction of CytR and CRP dimers associated as a tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The latter is stabilized by cooperative interactions between CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Additionally , puriﬁed E. coli CytR and CRP proteins were used in in vitro assays ( gel retardation ) to determine their ability to-bind heterologous promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>However , in strains containing altered CRP proteins that bind to the mutated CRP sites with a higher affinity than CRP-wt protein , CytR regulation was reestabl-ished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>For example , CRP depends on direct protein -- protein interactions with MelR and CytR to-bind low-afﬁnity CRP sites in the melA and deoC promoters , respectively ( 58 -- 60 ) Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>CRP requires cooperativity with MelR and CytR to-bind to low affinity sites in the melAB and deoCABD promoters , among others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>CRP alone activates transcription , whereas the CRP-CytR cooperatively bound to OCRP and OCYTR , respectively , represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Our results support and extend a model in which transcription is repressed when CRP moieties are bound to two appropriately positioned sites in the udpP and are themselves bound and stabilized by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CRP and CytR designated filled sites according to the transcription factor bound ; O designates empty sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Upon repression , the CytR repressor binds between the two bound CRP molecules , inactivating them and causing repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>A summary of the protection experiments ( Figures 5C and 5D ) demonstrates that the overall structure of the two repression complexes is identical ; i.e. , CAMP-CRP is bound at the tandem CRP sites , and CytR is located in between .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Figure 3 also shows the effect when CytR ∆ 2-63 was added to reaction mixtures containing CRP dimer at a concentration of 13 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Strikingly , the effect on CytR regulation varies in the four promoters tested ( Table 2 ) : all CRP mutants almost completely abolish CytR regulation in tsx , whereas a low residual CytR regulation is still present in deoP2 and cdd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>When 0.2 mM cytidine-liganded CytR was subsequently added , CRP affinity for CRP1 and CRP2 dropped by over 2-fold , giving DGCRP1 5 211.3 6 0.1 kcal/mol and DGCRP2 5 212.9 6 0.2 kcal/mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However , we do observe an additional few-fold decrease in apparent CRP affinity when the CytR concentration is increased to 0.4 mM , consistent with a simple competitive model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Configuration 5 pertains to the complex formed by association of CytR Core domain in which ∆ G represents the free energy change for bridging CytR-CRP bridge interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>On the basis of ∆ Gbridge ≈ -8 kcal/mol , ∆ G of the order -3 to -4 kcal/mol might be anticipated for a 1:1 association between CytR and CRP to form a hetero-tetrameric complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In the well-characterized case of the CytR regulon , CRP expresses positive or negative inﬂuences in collaboration with CytR ( Pedersen & Valentin-Hansen , 1997 ; Holt & Senear , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In this context , the suggestion that the primary competition between CytR and RNAP is for interaction with CRP ( 31 ) is attractive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In this model , the association of the z-average molecular weight monomer corresponds to assocation of CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These results support our hypothesis that cytidine is an allosteric-effector of CytR conformations that constrain its simultaneous association with CytO and the flanking CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These findings indicate that both CRP targets are required for efficient CytR repression of deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>These results clearly show the importance of the tandem CRP targets for CytR repression of deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>RESULTS Our goal is to understand how CytR and CRP mediate differential regulation of CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Senear , D. F. , Perini , L. T. , and Gavigan , S. A. ( 1998 ) Analysis of interactions between CytR and CRP at CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus , in addition to the tandem CRP targets , the distance between the two targets seems to be an important factor for CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>A key to CytR-mediated regulation is that despite roles as functional antagonists , CytR and CRP bind with positive cooperativity to their respective operators ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These studies have also identified competition between CytR and CRP as an additional contributor to differential regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Central to this process and to the differential regulation of expression are the interactions between CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To analyze the mechanism ( s ) involved in this combinatorial regulation , we endeavored to isolate CRP mutants that could not support CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Hence , both CRP targets are involved in and required for efficient CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Indeed , the effect of competition at deoP2 is to largely eliminate recruitment of CRP by CytR , thereby negating the role of CytR as a co-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , cooperativity also means that CytR can recruit CRP and , by doing so , can act as a co-activa-tor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Because cytidine is an effector of CytR-CRP cooperativity , its effect should also be linked to how CytR binds at the various promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This suggests that the separate interactions between CytR and each CRP dimer are additive , yielding a connection free energy of zero for bridging .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>On the other hand , the free energy changes for cooperativity between CytR and CRP are independent of the particular model and so appear to provide accurate descriptions of these interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>It is ironic , given the results we report here , that until relatively recently CRP and CytR were believed to compete to-bind to the same sites ( cf. Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Only when CRP occupies the two sites can CytR bind and repress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>But simple recruitment of this sort makes little functional sense , as it would yield only two significantly populated states , i.e. CRP and CytR sites empty or CRP and CytR sites filled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Alternatively , CytR may adopt a conformation at these sites that is incompatible with positive CRP -- CytR interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>There seem to be rather strict rules for the position of the L and R CytR half-sites relative to the ﬂanking CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Binding sites for regulatory proteins CRP , CytR and DnaA are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>As practical approximations to the limit of saturating concentrations , 0.1 mM CRP ( total dimer ) and 75 nM CytR were used here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Given the relatively low solubility limit for CytR , a high CRP concentration was used to drive the assembly and maximize sensitivity to hetero-oligomers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>As a practical limit to saturating concentration , 0.1 μM dimer was used for both CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>We propose instead that the CytR-CRP interactions themselves are critical to CytR-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>We evaluated the cooperative interactions and competition between CytR and CRP as described previously ( 19 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>ACKNOWLEDGMENT We thank Dr. Vira Tretyachenko-Ladokhina for the CytR and CRP used in this work as well as for discussion and insights .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Udp activity in this strain was as high as in constitutive mutants with larger deletions and was completely CytR and CRP independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>This indicates that the model is a fair approximation to the heteromeric association of CytR and CRP to form tetramers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This phenomenology is consistent with recent reports of `` unidirectional stimulation '' in CRP and CytR interaction with nupG ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Third , the truncated CytR proteins still have the ability to interact with CAMP-CRP , to respond to inducer , and to oligomerize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Simple cooperativity is of course a two-way street , and as such , CytR would be a net recruiter of CRP as well as the reverse .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR and CRP bind cooperatively when CytR binds CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR and CRP proteins were added to the following ﬁnal concentrations : Lanes 1 , no proteins ; 2 , 1.3 nM CytR ; 3 , 0.4 nM CRP ; 4 , 4.2 nM CRP ; 5 , 0.65 nM CytR and 0.2 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>With this information , the distributions of ligation states of each of the promoters under physiologically relevant concentrations of CRP and CytR are easily computed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2682">
Final concentrations of CRP and CytR are indicated in ng/pl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></label>
<input id="act_similarity_2682", type="checkbox">
<div class="ACT_similar">
<li>Final concentrations of CRP and CytR are indicated in ng/4l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Final concentrations of CRP and CytR are indicated in ng/1ul .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li></div>
<li>Cells were loaded with 1:1 , 1:2 , 1:4 , and 1:8 dilutions from initial concentrations of ∼ 2-3 mg/mL for CRP ; ∼ 0.4 mg/mL for CytR ; and 1:1 , 1:3 , and 1:9 dilutions of 2.7 mg/mL for Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Mixtures of CRP and CytR were 1:1 and 1:3 dilutions of these initial concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Furthermore , the CytR operators in nupG , cdd and deoP2 are not symmetrically positioned with respect to the flanking CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>DNase I footprinting of complexes between WT , del252 , del264 , and del18 udpPs , CRP , and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Unfortunately , this approach is not feasible in this case because of the low affinity for CytR-CRP association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Sedimentation equilibrium analysis was employed to investigate the association between free CytR and free CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>∆ G values as obtained from this analysis are tightly bounded , both for the association of CRP with free CytR and with cytidine-liganded CytR ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2692">
Lane 9 , 135 nM CRP + 220 nM RNAP + 66 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></label>
<input id="act_similarity_2692", type="checkbox">
<div class="ACT_similar">
<li>Lane 10,135 nM CRP + 66 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li></div>
<li>Lune 11:135 nM CRP + 2.6 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Lane 16,42 nM CRP + 260 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Lane 17 , 42 nM CRP + 26 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Results of sedimentation equilibrium experiments conducted on a mixture of CytR and CRP are summarized in Table 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Analysis of Cooperative and Competitive Binding of CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Regions protected by CytR and CRP are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Gel Retardation in the Presence of CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Solution Assembly of CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CRP and CytR Purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In the case of CRP -RCB- CytR , the antiactivator binds DNA between activator-binding sites , forming a nucleoprotein complex which prevents activation ( ValentinHansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>Moreover , Thus , a model is suggestedinwhichthe DNA is the synergistic action of CRPand CytR relies on direct wrappedaround a complex of RNA polymerase and protein-protein interaction on the DNA helix as single amino one CRP.Finally , the resultsshow thatCytR and RNA acid substitutions intwo surface exposed loops of CRP abolish polymerase are rivals that compete for binding with regulation by CytR without altering CAMP-CRP activation CRP at deoP2 and that CytR functions as an antiactivator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>-9.6 ( 0.5 -9.7 ( 0.2 -11.5 ( 0.2 -12.8 ( 0.1 -13.3 ( 0.1 -13.7 ( 0.2 -14.1 ( 0.1 -13.2 ( 0.3 -13.4 ( 0.4 -13.3 ( 0.3 -12.2 ( 0.3 -12.1 ( 0.0 -14.3 ( 0.3 -14.5 ( 0.03 Table 2 : Apparent Cooperativity As Assessed by Differences in Indivia dual Site Loading Free Energy Changes Described in Table 1 a Table 3 : Configurations and Free Energy States for CRP and CytR Binding to the cdd Promoter binding sites a CRP1 free energy contributions free energy state b CRP2 CytR at CRP1 CytO CytR at CRP2 O O O 1 O O reference state ΔGs ,1 2 O O O CRP O ΔG1 ΔGs ,2 3 CRP O O O O ΔG2 ΔGs ,3 4 O O CytR O O ΔG3 ΔGs ,4 5 O O O O CytR ΔG4 ΔGs ,5 6 O CytR O O O ΔG5 ΔGs ,6 7 CRP O O CRP O ΔG1 þ ΔG2 ΔGs ,7 8 O O CytR CRP O ΔG1 þ ΔG3 þ ΔG13 ΔGs ,8 9 O CytR O CRP O ΔG1 þ ΔG5 ΔGs ,9 10 CRP O CytR O O ΔG2 þ ΔG3 þ ΔG23 ΔGs ,10 11 CRP O O O CytR ΔG2 þ ΔG4 ΔGs ,11 12 O O CytR O CytR ΔG3 þ ΔG4 ΔGs ,12 13 O CytR CytR O O ΔG3 þ ΔG5 ΔGs ,13 14 O CytR O O CytR ΔG4 þ ΔG5 ΔGs ,14 15 CRP O CytR CRP O ΔG1 þ ΔG2 þ ΔG3 þ ΔG123 ΔGs ,15 16 CRP O CytR O CytR ΔG2 þ ΔG3 þ ΔG4 þ ΔG23 ΔGs ,16 17 O CytR CytR O CytR ΔG3 þ ΔG4 þ ΔG5 ΔGs ,17 18 O CytR CytR CRP O ΔG1 þ ΔG3 þ ΔG5 þ ΔG13 ΔGs ,18 a Binding sites are listed in columns 2-6 corresponding to the order , from promoter distal to promoter proximal , on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CRP concentrations were correcte configuration ( fs ) as given by Table 1 : Configurations and Free Energy States for CRP and CytR Core Domain Association with deoP2 and udpP Promoter Configurationa free energy free energy CRP2 CytR CRP1 contributionsb state - ∆ Gs/RT i j e [ CRP ( cAMP ) ] [ Core ] fs ) 1 ∑ - ∆ Gs/RT 1 s , ie [ CRP ( cAMP ) ] i [ Core ] j ( 2 ) 1 O O O reference state ∆ Gs ,1 2 O O CRP ∆ G1 ∆ Gs ,2 3 CRP O O ∆ G2 ∆ Gs ,3 4 CRP O CRP ∆ G + ∆ G ∆ G 1 2 s ,4 5 CRP Core domain CRP ∆ G1 + ∆ G2 + ∆ Gbridge ∆ Gs ,5 6 O CytR O ∆ G3 ∆ Gs ,6 7 O CytR CRP ∆ G1 + ∆ G3 + ∆ G13 ∆ Gs ,7 8 CRP CytR O ∆ G2 + ∆ G3 + ∆ G23 ∆ Gs ,8 9 CRP CytR CRP ∆ G + ∆ G + ∆ G + ∆ G 1 2 3 s ,9 ∆ G123 a Promoter configurations with sites denoted as filled ( CRP , Core domain , or CytR ) or empty ( O ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Model for Cooperative and Competitive Binding of CytR and cAMP-CRP -- To determine whether such interactions of CytR with DNA sequences overlapping and thereby occluding or competing with CRP binding to CRP1 and CRP2 can account quantitatively for both the loading free energy changes in Table I and apparent free energy changes in Fig. 6 , two models that incorporate two additional CytR binding sites and which constitute opposing possibilities were evaluated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The inducer of CytR , cytidine : perturbs the cooperative t ) KA binding of CAMP-CRP and CytR by disrupting the protein-protein contacts between t , hr two proteins ( Pedersen et al. , 1991 ; Gertach rf nl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2709">
The co-operative binding of CytR and CRP relies on direct protein -- protein interactions ( Søgaard-Andersen and Valentin-Hansen , 1993 ; Kallipolitis et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></label>
<input id="act_similarity_2709", type="checkbox">
<div class="ACT_similar">
<li>The co-operative binding of CytR and CRP relies on direct protein -- protein interactions ( Søgaard-Andersen and Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>The results of these experiments ( summarized in Figure 3 ) strongly point to the conclusions that CytRAS-48-mediated repression does not rely on the exact sequence in the CytR-binding site and that the specificity of CytR can be achieved solely through the protein-protein interactions to CAMP-CRP , whereas the CytR-DNA interactions may primarily serve to increase the stability of the repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>These data thus provide a picture of the CytRlcAMP-CRP regulatory system in which the site of action of CytR can be specified by the protein-protein interactions to CAMP-CRP and in which the CytR-DNA interactions may primarily contribute to CytR binding and repression by stabilizing the CytRl CAMP-CRPIDNA repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Second , CytR and CRP are reported to mutually antagonize each other 's protein binding induced bends in the cytRP sequence ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP , CytR and DnaA protein binding region is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>The cooperative binding of CytR and CAMP-CRP is mediated by protein-protein interactions between the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The CytR protein stabilized binding of CRP to the CRP sites in udpP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CytR is a co-activator when it recruits CRP to its DNA binding site ( s ) via cooperative binding to DNA , and it is a repressor when it is part of a three-protein complex that is stabilized by CytR-CRP cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Therefore , it is probable that it is the CytR core protein , not the DNA-binding domains of CytR , that interacts with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Whereas inducer or co-repressor binding to other family members controls a switch between DNA-binding protein and non-DNA-binding protein conformations , cytidine binding to CytR affects only cooperative interactions between CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This special mode of protein-DNA interaction in which CytR can either enhance CRP binding or compete for CRP binding affects both repression and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>It has been suggested that the significant competition underlying CytR-mediated repression is between CytR and RNAP for making protein-protein contacts with CRP , rather than competition between CytR and RNAP for binding DNA ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The current view is that transcription from these promoters involves CRP binding to CRP1 and that repression involves the binding of CytR protein to CRPs that are , in turn , bound to both CRP1 and CRP2 in the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The unusual location of the CytR operator in the deoP-2 regulatory region suggests that this repressor inhibits transcription by preventing the binding of the CRP protein or by changing the DNA conformation in such a way that CRP fails to function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The cAMP.CRP / CytR regulatory case has a remarkable functional relationship to several eukaryotic systems , e.g. in yeast the activator protein GRM is required for efficient DNA binding of the repressor protein a2 , and GRM can by itself bind toDNA and activate transcription ( Benderand Sprague , 1987 ; Keleher etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Only a small set of CRP-regulated promoters are repressed by CytR , however , as at least two properly spaced CRP binding sites seem to be required for efficient binding of this repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Altogether the binding experiments demonstrate that CytR and CRP bind synergistically to rpoH forming a nucleoprotein complex , which in analogy to the repression complexes formed at most CytR-regulated promoters , appears to consist of one molecule of CytR bound in between two molecules of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>This band was interpreted to contain a bound complex of CytR and CRP bound to one of the two CRP sites at a low CRP concentration ( 0.25 ng/ml ) ; no band attributable to the binding of only CRP was seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Table 5 : Global Analysis of Individual Site Titration Data Including Those with Cytidine Present As Represented in Table 1 according to the Model Described in Table 3 parameter value a - þ-0 .2 ( 0.2 , 0.3 ) þ1 .1 ( 0.4 þ0 .2 ( 0.4 ΔΔG13 ΔΔG23 ΔΔG123 s b 0.054 a Consequently , the subsequent binding of CRP to CRP2 should contribute only a relatively modest increment to the overall activation , and even this would be muted to a substantial extent as a result of concomitant recruitment of CytR to CytO that results from the strong three-way cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Consistent with the suggestion that the additional CytR-DNA interactions provide differential modulation of CRP-mediated activation , our results indicate that CytR competes for CRP binding to udp-CRP2 but does not compete for CRP binding to CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>We proposed that different patterns of competition between CytR and CRP for binding to CRP1 and CRP2 at different promoters could provide differential modulation of CRP-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>We suggest that cooperative CytR binding has very different effects on class I CRP activation mediated by CRP2 versus class II activation mediated by CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We start with the premise that CytR is a modulator of CRP activation rather than a repressor per se , a view consistent with the fact that CytR is ineffective in the absence of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Cytidine binding to CytR releases the latter from the complex , hence the activation by CRP resumes and the gene is derepressed [ 33,34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>However , with so little bridged complex formed at these CytR ∆ 2-63 concentrations ( 13-15 µM ) , the stabilizing effect on CRP binding is small , and a lower affinity limit for CytR ∆ 2-63 association is not precisely determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Under the conditions used in the gel mobility shift assay , CRP stimulates binding of CytR < 20-fold ( compare the intensities of the bands containing CytR in lanes 2 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>When we analyzed CytR and CRP binding to the deoP2 promoter ( 5 ) , we also considered an alternative explanation for the competition between CytR and CRP for CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>But when CRP is added to this same CytR concentration , cooperative binding of CytR and CRP produces a footprint that extends beyond CRP1 to the 210 region of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Subsequently , global analysis of the individual site CRP and CytR binding data was conducted using equations that describe cooperative and competitive binding of CRP and CytR according to the model defined by the promoter configurations specified in Table II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Ratios of variances were also separately calculated considering the data obtained for each binding site and set of effectors , e.g. , for binding of CytR to CytO of the wild-type promoter in the presence of CRP and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>To investigate the chemical and structural basis for these effects , we investigated how cytidine binding affects association between CytR and CRP in solution and how it affects the binding of CytR deletion mutants lacking the DNA binding HTH domain , with tandem CRP dimers bound to either udpP or deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To assess these effects and their linkage to cytidine binding , we investigated the assembly of CytR and CRP in solution and compared this to the association of the Core domain dimer with tandem , promoter-bound CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>If so , elimination of CytR-CRP interactions in response to cytidine binding might allow the CRP-RNAP interaction necessary for full activation even in the presence of DNA-bound CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Thus , defining these sites thermodynamically , by their effect on th coupled energetics for binding of CytR and CRP to their primary operators , but without regard to any particular sequence of DNA , generates parameter values that are indistinguishable from those obtained from the protection actually observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>A critical component of the regulation is that CytR and CRP bind DNA cooperatively , meaning that CRP binding to CRP1 and CRP2 enhanced the affinity of CytR for CytO and vice versa ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Two nanomolar DNA was used for binding with CRP and CytR proteins in concentrations shown in the ﬁgure legends , using the same as for gel retardation binding buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The challenge then is to explain both the very broad specificity of CytR for DNA binding and for CytR-CRP cooperativity when cytidine is not bound , together with the control of cooperative interactions when cytidine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR is unique among LacI/GalR family members in that binding of its regulatory ligand , cytidine , is linked not to DNA-binding affinity but instead to the cooperative interactions between CytR bound at CytO and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This indicates that cytidine binding to CytR and bridging of CytR between pairs of DNA-bound CRP dimers are thermodynamically couple processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Analysis of Heterologous Cooperative Interaction between deoP2-bound CRP and CytR -- To evaluate cooperative interactions between different proteins binding to DNA , we considered the thermodynamic cycle for their simultaneous binding ( cf. Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We speculate that these different conformational modes restrict interactions between bound DNA-bound CytR and DNA-bound CRP in ways that are reflected in the different effects of cytidine binding to CytR in CytR-CRP cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Perhaps the simplest explanation to-tie together these various effects is to suggest that cytidine binding results in conformational changes that reorient the interactions between CytR , CRP , and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>What then is the significance of multiple CytR-DNA interactions and competition between CytR and CRP , and presumably also between CytR and RNAP , for binding DNA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Inhibition of CRP Binding by Cytidine-liganded CytR -- A second testable prediction of our model is based on the unusual mechanism of induction in which cytidine is an effector of heterologous cooperative interaction between CytR and CRP ( 6 ) 2 rather than of intrinsic binding of CytR to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>To address this question requires CytR Core domain in which the interaction between CytR and DNA is eliminated but other CytR functions ( i.e. , dimerization , cytidine binding , and CRP association ) are unperturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Alternatively , CytR , CRP , and/or the DNA would have to be distorted from the symmetric arrangement proposed in the model to accommodate off-center binding by CytR to CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , one can not rule out possible CytR interaction with the DNA after binding to CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2757">
The total Gibbs free energy of each configuration relative to the unliganded reference state is given as a sum of contributions from four free energy changes for intrinsic binding of CRP and CytR ( DGk ) and three free energy changes for cooperative interaction between liganded sites ( DGij ( k ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_2757", type="checkbox">
<div class="ACT_similar">
<li>The total Gibbs free energy of each configuration relative to the unliganded reference state is given as a sum of contributions from b five free energy changes for intrinsic binding of CRP and CytR and three free energy changes for cooperative interaction between liganded sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>The guanine residues protected by CAMP-CRP and CytR in the DMS footprints are indicated by open arrows , and the residues with enhanced reactivity in the CytR-binding site are indicated by closed arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The combined binding of CAMP-CRP and CytR-wt , additionally , resulted in diminished reactivity of three G residues and weakly enhanced reactivity of three A residues and one G residue ( Figure 5A , lane 9 ; Figure 58 , lane 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The molecular model defined by the promoter configurations listed in Table 3 accounts for CRP binding to CRP1 and to CRP2 and for CytR binding to CytO , with both pairwise and three-way CytR-CRP cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The results also confirm the expectation that CytR and CRP compete for binding to CRP2 but not for binding to CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The reason for this is that competition between CRP for binding to CRP1 and CRP2 and CytR for binding to satellite sites that occlude CRP1 and CRP2 complicates this simple analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The net effect on CytR binding is substantial while the net effect on CRP binding to CRP2 is slight .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The values of ΔG1 , ΔG2 , and ΔG3 are statistically indistinguishable from the ΔGLoad , i values listed in Table 1 for CytR binding alone to CytO and for CRP binding alone to both CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The values in Table I demonstrate a similar behavior , although with a different pattern of effects , for CytR and CRP binding to udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The central role of cooperativity in regulation is highlighted by the fact that binding of the inducer , cyt-idine , to CytR is coupled to CytR-CRP cooperativity ; this underlies the mechanism for induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The significance of CytR-CRP cooperativity to this competition is underscored by the fact that binding of cytidine ( the inducer ligand ) to CytR is coupled to CytR-CRP cooperativity ( 19 , 21 ) , a feature that is unique among LacI family repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Yet , regulation of cytRP , in which there is only one CRP site , also depends on cooperative CytR and CRP binding ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>For binding experiments in which both CytR and CRP are present , analysis according to Equation 1 provides an accurate estimate of the individual site loading free energy change , DGl , i ( 25 ) , and its confidence limits ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Fitting to Equation 1 was also used to estimate individual site loading free energy changes DGl , i ( 38 ) for binding experiments in which both CytR and CRP were present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Thus , differential and coordinate gene regulation must depend on different dispositions of CRP and CytR binding sites , different protein binding affinities , and/or different levels of site-site interaction or cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Also , point mutations in either CRP site reduce or abolish CytR titration ; however , no co-operativity is observed in the interaction of CytR with the two CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Titrant DGapp app app 123 DG13 DG23 CytR 22.7 6 0.4 21.9 6 0.7 22.0 6 0.4 CRP 21.8 6 0.6 20.6 6 0.4 20.5 6 0.4 In particular , pairwise binding of CytR and CRP to either site appears to be essentially noncooperative when evaluated from CRP titrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this view , CytR is presumed to be an ineffective inhibitor of transcription in the absence of CRP binding only because its free concentration is insufficient to fill the site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In addition , it was recently observed ( 11 , 21 ) that the apparent cooperativity , when assessed by the effect of saturating CRP concentration on CytR affinity , is substantially greater than when cooperativity is assessed by the effect of saturating CytR concentration on CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>b In order to analyze the effect of inducer binding to CytR on CytR-CRP cooperativity , a second set of experiments was conducted with cytidine-liganded CytR and global analysis performed as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In contrast , the interaction between CytR and CRP bound to CRP2 of udpP is virtually unaffected by cytidine ( 11 ) resulting in only a small decrease in overall CytR binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In addition , concomitant with CRP binding to CRP1 , CytR will tend binding to CytO , due to its recruitment by the strong threeway cooperative interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In this situation , CytR competes directly for CRP binding to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast , whereas CytR is a net recruiter for CRP binding to CRP1 , it is not a net recruiter for CRP binding to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2782">
In this model , CRP and CytR compete for binding to mutually exclusive sites at CRP2 , but CRP alone binds to CRP1 with no competition from CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_2782", type="checkbox">
<div class="ACT_similar">
<li>In this model , binding of CytR at the sites overlapping CRP1 and CRP2 and of CRP binding to CRP1 and CRP2 is mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>In addition , it is only by including competition between CytR and CRP for binding to CRP2 in the global analysis of the footprint data that we can accurately account for the experimentally determined loading free energy changes ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In energetic terms , if most of the loss of translational , vibrational , and rotational energy that accompanies macromolecular association is paid by the first interaction , the result can be a free energy change for binding of an oligomer , such as CytR-CRP heterotetramer , that is more favorable than the sum of free energy changes for separate binding of the individual proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To assess these effects together with competition between CytR and CRP for binding CRP2 , we extended the model defined in Table II to include terms to describe the effects of cytidine on cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Finally , how does pairwise cooperativity between CytR binding to CytO and CRP binding to CRP2 of udpP survive cytidine binding , particularly when pairwise cooperative interactions with CRP bound to CRP1 of udpP or either CRP site of deoP2 does not ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>DG13 and DG23 , which pertain to pairwise cooperative interactions between CytR binding and CRP bound to either CRP1 or CRP2 , were similarly evaluated using reduced valency mutants in which specific binding to either CRP1 ( CRP12 ) or CRP2 ( CRP22 ) was eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>One effect of competition between CytR and CRP for binding to CRP1 and CRP2 of deoP2 is that whereas CRP , whether bound to CRP1 alone , to CRP2 alone , or to both together , is a net recruiter of CytR to CytO ; the reverse is not so .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>First , the relative affinities for CRP binding to CRP1 and CRP2 and for CytR binding to CytO are very similar in udp and deoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>By eliminating cooperativity between CytR and CRP bound to flanking sites , cytidine should make CytR into a simple competitor for CRP binding to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Cytidine binding to CytR largely eliminates pairwise cooperativity between CytR and CRP bound to CRP1 of udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Here too , CRP bound to either CRP1 or CRP2 or both is a net recruiter for CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Although CytR binds to an additional site located just downstream from the transcription start site as noted above , there is no interaction between CytR binding to this site and CytR/CRP binding to CRP2 , CytO , and CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>First , the relative affinities of CRP for binding to CRP1 and to CRP2 are reversed as compared to other CytR-regulated promoters ; at cddP , the CRP1 site is the higher affinity site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Symbols denote the -- different binding interactions : 1 , CRP binding to CRP1 ; 2 , CRP binding to CRP2 ; 9 CytR binding to CytO ; [ , CytR binding to the CytR site located at CRP1 ; b , CytR binding to the CytR site located at CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Recently , we showed that CytR binds specifically to multiple sites in the E. coli deoP promoter , thereby providing competition for CRP binding to CRP operator site 1 ( CRP1 ) and CRP2 as well as cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>But this would also be partly offset by unfavorable competition between CytR and CRP for binding to overlapping sites at CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>One surprise uncovered by our analysis of deoP2 is that , in addition to its site-specific binding to CytO , CytR also binds with weak specificity to satellite sites and that this binding competes with CRP for binding to both CRP1 and CRP2 ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>However , this loss accounts for only a 0.6 kcal/mol effect on cooperativity according to the loading free energy changes listed in Table I. Thus , in contrast to other promoters , binding of the inducer has only a modest effect on CytR-CRP cooperativity at udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Binding of either CRP or CytR alone was analyzed to obtain the Gibbs free energy change corresponding to ki in Equation 1 ( DGi 5 2RT lnki ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>DGCytR , the loading free energy change for CytR binding alone ( no CRP ) is equal to the intrinsic free energy change , DG3 ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2803">
For simple binding of either CRP or CytR alone , Equation 1 gives the intrinsic free energy change for local binding , DGi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_2803", type="checkbox">
<div class="ACT_similar">
<li>For simple binding of either CRP or CytR alone , eq 1 gives the intrinsic free energy change for local binding , ∆ G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li></div>
<li>Finally , the occurrence of tandem CRP binding sites in all CytRregulated promoters studied so far ( 14 , 32 ; unpublished data ) suggests that this is a structural requirement for CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>We propose that competition and cooperativity in CytR and CRP binding are important to differential regulation of these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Our previous analysis of the binding of CytR and CRP to the deoP2 and udp promoters has highlighted unique aspects of these interactions that appear to be related to differential regulation of the CytRregulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>To ascertain whether this result of the conformational change is widespread or unique to deoP2 , we also assessed the effect of cytidine binding to CytR on CytR-CRP cooperativity in binding to udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Panels A and B represent paired titrations to show the effect of 4.0 µM CytR ∆ 2-63 Core domain dimer on CRP binding to udpP ; panels C and D represent paired titrations to show the effect of 13.1 µM CytR ∆ 2-63 Core domain dimer on CRP binding to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Despite this role of CytR as a functional antagonist of CRP , the two proteins interact cooperatively , resulting in substantially increased CytR binding af-finity when CRP is present ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>On the basis of the view that the thermodynamic driving force for cooperativity is CytR-CRP association , the presumption was that cytidine binding induces a concerted change in the conformation of the CytR dimer , the result of which is to decrease greatly its affinity for association with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>As expected , the stabilization resulting from the association of CytR Core domain is more acute for CRP1 as a consequence of its lower intrinsic affinity for CRP binding ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Do differences between CytR binding sites in udp and deoP2 result in different patterns of competition between CytR and CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Because titration studies were not conducted it is n't possible to assess either individual affinities for CytR or the mutual effects of CRP binding and of CytR binding to different sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Experimental resolution of CRP binding is limited at higher CytR concentrations ; substantial protection from DNase I cleavage by CytR binding alone obscures any changes when CytR is displaced by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Third , by binding to the CytR operator with high affinity , CytR acts as an antiactivator at lower CytR concentrations and recruits CRP at lower CRP concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>whereas CytR is able to stimulate binding of ( CAMP - ( `` RI ' under the conditions of the get retardation assa ; v. LV ' e are ( turrrntly pursuing a more detailed analysis of thr co-operativta binding of CAMP-CRP and ( $ tR to rytR1 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>FIGURE 6 : Effect of CytR Core domain on binding of CRP to the deoP2 and udp promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>We first discovered competition between CytR and CRP while investigating their cooperative binding to deoP2 ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Equations that describe the cooperative binding of CRP and CytR were derived by considering the relative probability of each of the deoP2 configurations ( Table III ) as specified by the Specific , and Nonspecific Additional Sites models that are introduced in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Panel A : CRP binding in the absence of CytR Core domain ( CytR ∆ 2-63 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Third , under inducing conditions in which binding of cytidine to CytR eliminates positive cooperativity between CRP and CytR ( 6 ) ,2 CytR was found to be a competitive inhibitor of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Thus , CytR is a modular regulator that undergoes induced ﬁt when co-binding with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>This reflects an increase in apparent affinity for CRP binding when CytR ∆ 2-63 is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This is accomplished by partially decoupling CRP binding from CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Although not explicitly stated , our expectation was that cytidine binding would produce a CytR conformation that would not interact with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CRP and CytR ) eliminate the binding of the less efficient partners ( CRP andRNA polymerase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>However , competition between CytR and CRP for binding to CRP1 and CRP2 provides a second mode of CytR-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Cooperative binding of CytR and CRP is a critical component of both repression and induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Analysis of Cooperative and Competitive Binding of CytR and CRP -- Cooperative interactions between CytR and CRP and competition between CytR and CRP for binding to CRP2 were evaluated by considering the thermodynamic cycles for simultaneous binding of both proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This result confirms that competition between CRP and CytR for binding CRP2 is a significant feature of the interaction of these proteins with udp , just as it is for deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Equations to describe CRP binding to CRP1 and to CRP2 were derived by considering the relative probability of each promoter ∆ Gs is the sum of free energy contributions for configuration s , ( Table 1 ) ; i and j are the stoichiometries of bound CRP complexes ( 0 , 1 , or 2 ) and Core domain dimers ( 0 or 1 ) in configuration s. Summation of the relative probabilities for the configurations with protein bound to any given site yields the binding equation for that site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Following the lead provided by the electrophoretic mobility-shift results , we evaluated Core domain binding to the CRP2-deoP2 and CRP2-udpP complexes by considering the thermodynamic linkage between this reaction and binding of CRP to CRP1 and CRP2 , as depicted in Figure 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Converting the fitted values obtained for ∆ G1 , M and ∆ G2 , M into free energy changes for intrinsic , noncooperative binding of CRP yielded -10.9 ( 0.1 and -11.4 ( 0.1 kcal/mol for binding to CRP1 and to CRP2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>∆ G1 and ∆ G2 denote the free energy changes for intrinsic ( and noncooperative ) binding of CRP to CRP1 and CRP2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In deoP2 the upstream CRP target ( CRP-2 ; Fig. 1 ) is required for repression , as shown in a series of experiments in vivo -LCB- Valentin-Hansen , 1985 ; Martinussen ef al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Regulation of the gcv operon by GcvA and GcvR is part of a growing number of prokaryotic regulatory systems involving an activator -RCB- antiactivator complex , including TraR -RCB- TraM regulation of conjugal transfer of the Ti plasmid in Agrobacterium tumefaciens ( Hwang et al. , 1999 ) , NifA -RCB- NifL regulation of nitrogen ﬁxation in Klebsiella pneumoniae ( Narberhaus et al. , 1995 ) , ComK -RCB- MecA -RCB- ClpC regulation of competence in Ba-cillus subtilis ( Turgay et al. , 1998 ) and CRP -RCB- CytR regulation of operons encoding the nucleoside transport and biosynthesis enzymes ( Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>Under conditions in which CytR is inactive ( i.e. in the presence of cytidine ) , CRP interacts with RNP to form transcriptionally active CRP ± RNP ± promoter complexes , whereas under conditions in which CytR is active ( in the absence of cytidine ) , two CRPs interact with CytR to form transcriptionally inactive ( CRP ) 2 ± CytR ± deoP2 complexes ( Sùgaard-Andersen et al. , 1990a , b , 1991a , b ; Sùgaard-Andersen and Valentin-Hansen , 1993 ; Meibom et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>CAMP-CRP activation takes place from CRP-1 ( Valentin-Hansen , 1985 ) , whereas CytR regulation is dependent onthe presence of both CRP targets awsell as the distance and the sequence between these targets ( Sld-gaard-Andersen et al. , 1990a , 1990b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>We have previously shown that this band corresponds to the repression complex formed at deof2 in which CytR is sandwiched between tandem CAMP-CRP complexes ( Pedersen et al. , 1991 ; Segaard-Andersen et al. , 1991a ; see also the footprinting experiments below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CRPa-CytR-Promotor Complex-It has previously been shown that both the CRP-1 and tC heRP-2 sites arneecessary for CytR repression of the deoP2 promoter , and DNase I footprintinghas indicated that bothsitesare occupiedby CRP in the repression complex with CytR bound in between the CRP sites ( Sbgaard-Andersen et al. , 199Ob ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>As previously shown CytR and CAMP-CRP bind simultaneously to deoP2 in a highly cooperative manner and the protection pattern of the repressor complex is consistent with a model in which CytR is trapped between r -93 40 Uranyl Photofootprinting CRP-2 CRP-CytCRomplex-Uranyl photofootprinting reveals contacts between a DNA-bound ligand and thephosphates of the DNA backbone ( Nielsen et al. , 1988 ; Jeppesen and Niel-sen , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>The CRP2 -- CytR -- DNA repression complex covers ~ 80 bp on one face of the DNA helix , and is held together by CRP -- DNA , CytR -- DNA and CRP -- CytR interactions ( Pedersen et al. , 1991 ; Søgaard-Andersen et al. , 1991a , b ; Rasmussen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The second repressor of the system , CytR , binds in concert with the CAMP-CRP complex in a highly cooperative fashion , forming a repression complex in which the DNA is wrapped around CytR and one or two molecules of CAMP-CRP ( Pedersen et al. , 1991 , 1992 ; Gerlach et al. , 1991 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CytR and CRP turn off transcription by binding in a highly co-operative fashion to speciﬁc targets in the promoter regions ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ; 1992 ; 1995 ; Søgaard-Andersen et al. , 1991 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>In this context , it should be noted that the inducers of CytR ( cytidine and adenosine ) exert their effect on repression by interrupting the protein-protein interactions between CytR and CAMP-CRP ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Discussion Previous results have shown that CytR can interact and bind co-operatively with CRP from a number of different settings of operator half-sites ( Valentin-Hansen et al. , 1996 ; Pedersen and Valentin-Hansen , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Binding studies have shown that CytR binds to deoP2 with a dissociation constant of approximately 1Om7M and that CAMP-CRP stimulates CytR binding approximately lOOO-fold ( Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Thus , CRP with single amino acid substitutions ( Hid7Arg , Cys18Arg , Val ' '' Ala , Pro ' 1oSer ) have been isolated that specifically abolish CytR regulation in uiuo ( Sb-gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Proteins CytR and CRP proteins were purified according to Pedersen et al. ( 1991 ) and Ghosaini et aL ( 1988 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>CytR and CRP proteins were puriﬁed as described ( Ghosaini et al. , 1988 ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>CRP and CytR were purified as described ( Ghosaini et al. , 1988 ; Petersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>This moderate decrease in transcription , together with the fact that 10 element and CRP I substitutions resulted in signiﬁcant decreases in transcription , suggests that CRP bound to CRP II functions as a coactivator of the CRP bound at CRP I. To conﬁrm that the G126C mutation in CRP II does not exert a CRP-independent effect upon transcription , we generated crp cells carrying fusions to the wild-type , G75C mutant , and G126C mutant promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>While the localization of a-CTD by DNase I footprinting performed here is with the a-CTD not tethered to RNA polymerase , in contrast to the previously mentioned experiments , CRP bound at 2121.5 nonetheless functions to activate transcription of proP P2 in a manner dependent on AR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>* -35 CCATGGCCATTACGTTGGCTGAACTGGTTTATTCGGAACTGATCGGACTTGTTCAGCGTA -10 +1 CACGTGTTAGCTATCCTGCGTGCTTCAATAAAATAAGGCTTACAGAGAACATG - 3 5 * - 1 0 AGATCATATTTGAAAAAAGATAGAAAGATAGTTGCAACATTCCAGGTGGTCCGACCTATA + 1 * Specific Activities ' ^ Relevant genotype Strain / Plasmid ^ G M rowth edium '' ase ( J-oxidatior1 l ^ - galactosid LSI345 \ <t> fadB-lacZ TB 0.71 ± 0.2 3.65 +0.1 ND 3.11 ± 0,2 3.53 ± 0,6 ND 0.47 '' ND ND 4.88 ° ND ND 0.6610.2 2.55 ± 0,2 ND 2.5010.4 2.50 ± 0.1 ND 320187 Regulation o ^ fadL and fabA transcription in vitro To compare the role of FadR in regulating the transcription of fadL and fabA , we performed in vitro transcription experiments using purified FadR , CRP and RNA poiy-merase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The consensus sequence for FNR is similar to that for CRP consensus , and previous studies have shown that FNR activates transcription initiation in a similar way to CRP and that both proteins can bind to the DNA site for the other protein ( 41,42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>A uniquely 59-end 32P-labeled fragment from 2190 to 1103 ( with respect to P2 transcriptional initiation ) of proP was used ; 1.1 pmol of CRP protein and 2.2 nmol of a-CTD were bound to the DNA fragment for 10 min under the same conditions used in the in vitro transcription reaction in a total volume of 20 ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>A heterologous protein ( IHF ) that bends the DNA to a similar degree as does CRP activates transcription when bound at sites equivalent to activating positions for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>These results suggest that in complex media CRP represses gadA transcription and inhibits induction by acidic pH. However , it is unlikely that CRP is directly involved in pH control since an rpoS crp mutant still exhibits acid induction ( EF677 , Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>The results from this study are consistent with a model where CRP 's role in the regulation of gcv is to interfere with repression by GcvA , rather than to activate transcription via interactions with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>These results suggest that CRP regulation contributes to the pH-dependent regulation of AzuC levels , most likely by repressing AzuC transcription in neutral media , but that there is also an as-yet-unidentiﬁed transcriptional-regulator of azuC induction in acidic medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>These results , taken together with previous data ( 32 , 47 ) , indicate that no fewer than four proteins ( IHF , 54 , PrpR , and CRP ) are involved in the regulation of prpB-CDE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>The use of similar sensing mechanisms by structurally unrelated transcription factors , such as OxyR and SoxR , suggests that there may be a limited number of biologically effective solutions to the problem of setting transcriptional switches within specific redox windows The conformational changes induced in CRP , FNR and FNR-like proteins upon perception of the appropriate signal ` switch ' the regulators from their inactive to active forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Three proteins were present at higher levels in minimal glucose medium than in minimal glycerol medium , suggesting that their synthesis may be repressed by the CRP transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>The dot blot assays showed that YkgR-SPA levels are higher in medium containing glycerol rather than glucose , suggesting that CRP activates ykgR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>The number of CRP targets should also increase if CRP bound within intergenic regions between divergent promoters regulates transcription toward both directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>1.0 * 1.0 ** 0.3 2.2 / 2.5 3.3 The transcriptional regulation by CRP and FIS is mirrored in the synthesis of HU subunits To examine how variations at the level of HU transcription caused by the FIS and CRP regulation affected the a and b subunits , we performed Western blots to measure the accumulation of the two HU subunits in the absence of FIS and/or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The bgalactosidase activities were measured in the presence / absence of plasmid-encoding CRP in nitrogen-limiting H159L medium M63 containing 2.5 % glutamine [ weight in volume ( w/v ) ] ; for in vitro assays , the promoters were inserted into the transcription vector pJCD01 ( Marschall et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF-lacZ fusion by these two proteins ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>The bar graph shows fold activation over the basal level ( no added activator ) from in vitro transcription reactions performed with CRP , Fis , and Fis plus CRP in the presence of 0.2 to 0.6 M potassium glutamate ( K1Glu ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>CRP can stimulate transcription when looping is impossible due to t ' he absence of the ara0 , site t I 're srnt address : Department of Rheumatology and Immunology , Harvard Medical School , The Seeley G. Mudd Building .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The data in Table 1A show that transcription is dependent on CRP but was sharply reduced by the presence of CRP containing the HL159 substitution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The transcription from P l a is completely shut off in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>The progression of the transcription bubble could clearly be observed , both in the absence and presence of CRP ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>The results suggest that , in the presence of CRP , more transcription complexes are formed , and the conformation of those complexes might be different from those formed in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The observed severe decrease in malEFG and malKlamB malM transcription as well as LamB levels in hns and hns-stpA mutant strains was most likely caused by the combined and additive effects of the lower levels of both MalT and CRP which characterized these strains ( this work and unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The in vitro transcription proﬁles of CRP derivatives were compared with wild-type CRP , and signiﬁcant changes were identiﬁed by outlier iteration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Effects of CRP and FIS on heterodimeric HU The results obtained on the hupA transcription show that the positive effects of CRP and FIS are additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The cyagene is an additional example of which transcription is negatively regulated by CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>The action of many global transcription factors such as ArcA , Fnr , CRP , and IHF commonly involved both NADH and ATP , whereas others responded to either ATP or NADH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>The additional occupation of IHF III either increases or decreases transcription , depending upon the status of CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The development of an in vitro transcription assay with CRP , RNAP and TcbC clearly demonstrated that TcbC increased CRP-dependent transcription from a relaxed DNA template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Discussion The multiple roles of CRP during activation of acs transcription Previously , we reported that in vitro RNAP alone could form an open complex ( TOC ) at both acsP2A and acsP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The increased stability of this ternary CRP -- RNAP -- acsP2 complex activates transcription at multiple levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>This allosteric transition increases the affinity of CRP for promoter DNA and thereby directly stimulates initiation of transcription by RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>This indicates that CRP is unable to activate transcription from the aer promoter when its DNA site is moved to the opposite face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>AraC-dependent transcription initiation at the araBAD promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>To activate transcription , CRP binds to a specific DNA site located upstream from a promoter and interacts with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>One possibility was that CRP binds to DNA cooperatively with RhaR to improv the ability of RhaR to activate transcription , similar to the cooperative binding of MelR and CRP at the melAB promoter ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Since this site is far upstream of RNAP 's binding site , it seems unlikely that CRP is contacting RNAP to activate transcription of gcv unless DNA looping occurs at the gcv promoter , allowing contact between one of CRP 's activating regions and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>As expected , positive control mutations that impair the ability of CRP to activate transcription but not DNA binding have no effect on P1 repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>These observations suggested that CRP repression is alleviated upon acid stress or that other DNA binding proteins regulate azuC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Mutants of CRP that bind normally to the target site on the DNA and do not perturb bending of the DNA , but fail to activate transcription ( positive control or pc-mutants ) , have been isolated by several laboratories .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Mutants of CRP mapping in this region , called positive control or pc-mutants , are ( nearly ) completely unable to activate transcription , but bind DNA with the same affinity and bend DNA to the same extent as does wt CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>When the DNA upstream of the araFGl site or CRP site was deleted , no increase in the basal level of transcription was seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>As a prerequisite to activating transcription , CRP must bind to a 22-bp twofold-symmetric recognition site on DNA ( 2 , 9 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>CRP binding to these sites bends the DNA and causes MalT to shift 3 bp downstream to-binding sites from which MalT can activate transcription ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>It is shown that CRP can activate transcription at ﬁve of these targets and the functional DNA sites for CRP are identiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Another mechanism by which CRP can activate transcription at class III promoters involves DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>However , it appears that this bending is not essential for CRP function because CRP variants that can not bend DNA are still effective transcription regulators ( Busby and Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>It has been proposed that CRP correctly positions the R-subunit on the DNA and thereby activates transcription ( Busby & Ebright , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>It was also reported that the nucleoproteins , FIS and IHF , bound directly to multiple regions of the acsP2 promoter , that their binding reduced CRP-dependent acs transcription , and that FIS competed with CRP to occupy the acsP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In vitro transcription assay Binding in vitro of CRP to the pck promoter : AFM observation For mapping of the CRP dose-dependent binding sites on the pck promoter , we next performed AFM analysis at increasing concentrations of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In the experiments shown in Figure 6 , binding of CRP to the gal and lac promoters enhanced transcription from the respective P1 promoter by Ea38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>Repression of pck transcription was observed using the truncated pck-del3 template lacking the CRP box-1 ( Fig. 4Ab ) , indicating that the repression of pck transcription takes place by CRP-binding to the promoter-proximal box-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Binding of CRP to the proP promoter in vivo is transiently destabilized after a hyperosmotic shift with kinetics that correspond to the derepression of transcription , whereas Fis and Lac repressor binding is not osmotically sensitive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Thus , these data suggest that under such conditions , FNR and CRP are in competition for binding at PhlyE , and that the simplest explanation of these observations is that FNR is more efﬁciently bound than CRP at PhlyE but is not as effective as CRP in activating hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Transcription of both genes might be regulated by CRP , as glucose effectively inactivated transcription and there was a sequence similar to the consensus for CRP binding in the intergenic region of nagE and nagB ; nagA is probably co-transcribed with nagB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3284790" target="_blank">3284790</a>]<br></li>
<li>Based on the position of the putative FadR binding site , it seemed likely that FadR and ArcA negatively regulate fadH expression , as previously reported ( 8 ) , whereas CRP activates fadH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>At many promoters where CRP binding to upstream sites serves to regulate transcription by contacting CTD , displacement of the CRP site by one helical turn from its optimal position does not completely abolish CRP activity ( 8 , 12 , 24 , 44 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP is known to activate transcription by binding to sites located at various positions upstream of the 235 site ( from 241.5 to 2103 ) ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>When CRP II is not occupied , the binding of IHF enhances transcription , perhaps by increasing the stability of the open complex ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>When both CRP sites are occupied , the binding of IHF to IHF III results in signiﬁcantly reduced transcription , possibly because compaction of the upstream elements overstabilizes the complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>when CRP carrying HL159 is targeted upstream in XX-CC , transcription is severely reduced , strongly suggesting a requirement for AR1 in CRP when binding upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This result is consistent with our proposal that binding of CRP to the PguaB upstream region serves to downregulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP activates acs transcription by binding tandem sites located upstream of the major promoter , acsP2 , whereas , both IHF and Fis each act at different times during growth as anti-activators of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>CRP activates acs transcription by binding tandem DNA sites located upstream of the major promoter , acsP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>CRP binding to the - 4 0 sites activates transcription from the Pi promoter of the gal operon as well as from the mgl , galS and galP promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>CRP activates transcription by binding to speciﬁc sites located upstream of promoters and contacting RNAP ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Thus , it was proposed that during exponential-growth , Fis represses acs expression indirectly by repressing crp transcription and directly by binding to sites that overlap the binding sites of CRP and the -10 hexamer of RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>CRP directly activates or represses the transcription of about 200 genes , although it probably interacts with many other low-afﬁnity binding sites in the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Thus , the competition between FIS and CRP for binding to their overlapping and tandem sites helps to keep acsP2 transcription low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>vel mechanism controls anaerobic a catabolite regulation o herichia coli tdc peron nd A no f the Esc o Gary Sawers * ing of the CRP binding site to 241.5 bp resulted in high-level , CRP-dependent transcription , even under catabolite-repressing conditions , and that transcription was no longer influenced by TcbC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Given the similarity in position of CRP sites in deoP2 , cddP , and udpP , it is interesting to consider how each site contributes to regulation of transcription of distal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>From this position , or a site located one helical turn upstream , CRP can activate s38 - , but not s70-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Binding of CRP to the putative downstream site would be expected to repress transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>All of these differences are present in the CRP site in the ara regulatory region , since the CRP site activates transcription in both directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>1991 ) , in addition , a number of studies have been done concerning the molecular mechanism by which CRP activates transcription ( Reznii < off , 1992 ) , in contrast , studies of how the crp gene encoding CRP is regulated are still limited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Further studies with acn-lac2 gene fusions and a wider range of transcription regulators indicated that acnA expression is initiated by G ~ ~ from PIacnAa , nd from PZacnA it is activated directly or indirectly by CRP , FruR , Fur and SoxRS , and repressed by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Regulation of acnB gene transcription Apart from being activated by CRP and repressed by ArcA , the acnB gene exhibited a very different pattern of regulation to that of acnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>In vivo , mutations that disrupt FIS II or alter the balance between FIS III and CRP I result in elevated acs transcription , indicating that these sites and the competition to-bind to them help to keep acsP2 transcription low during exponential-growth and to modulate transcription after CRP-dependent induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>High concentrations of FIS could negatively regulate CRP-dependent acsP2 transcription during early exponential phase using both indirect and direct mechanisms that FIS , when present at a sufﬁciently high concentration , can displace CRP from both its sites and , thus , antiactivate CRP-dependent acsP2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>A. Schematic showing acs transcription ( acs , TRXN ) , growth ( OD ) and the relative concentrations of active CRP , FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In transcription in vitro , no obvious lac RNA band was seen in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>In previous studies with galp19T , it was shown that P1 can function in the absence of CRP but that transcription initiation at P1 is stimulated by CRP ( 2 , 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>In the absence of CRP , FIS yielded no transcription product ( data not shown ) , indicating that FIS alone can not activate acsP2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In promoters where CRP acts as a sole transcription activator or synergistic activator , it binds to highaffinity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2938">
In the absence of CRP , MalT binds to three sites from which it can not activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></label>
<input id="act_similarity_2938", type="checkbox">
<div class="ACT_similar">
<li>In the absence of CRP , MalT binds three high affinity sites but is unable to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li></div>
<li>In addition to the stimulating catabolitesensitive operons , CAMP and CRP influence the synthesis of several cell envelope components , lysogeny of bacteriophage , antibiotic susceptibility , replication of certain plasmids and possibly act as a modulator of polarity in polycistronic transcriptional units by decreasing transcription termination ( reviewed in Botsford , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>In the previous transcription reactions , Fis and CRP were allowed to-bind to the DNA prior to Es38 addition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In addition , it is possible , although we consider it unlikely , that DNA bending promoted by the B-C loop Fis mutants is sufﬁcient to stimulate transcription when CRP is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In vitro transcription assays were performed to ﬁnd out whether CRP alone is suf-ﬁcient to activate PcspD transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In vitro repression of transcription by CRP and Lac repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In contrast , FIS and CRP compete in the regulation of hupB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>In these cases , it is not ruled out that CRP regulates transcription toward opposite directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In addition , CRP can regulate transcription of other transcription factors , such as MelR , RpoH , BlgG , Fis and PdhR , which could further regulate transcription of genes involved in the above processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In addition , CRP could regulate gyrA transcription through the regulation of topA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>We observed in this study that CytR could block transcription in vitro to a level that is signi ® cantly lower than that observed in the absence of CRP ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of cytidine , CytR is inactive , and thus CRP can activate transcription in a Class III dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In the absence of cytidine , CytR is active and binds between the two CRP dimers , repressing CRP-dependent transcription [ reviewed in ( 42 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Thus , we propose a complex regulatory cascade in which global regulators ( RpoS , CRP , HN-S , EvgA , etc. ) inﬂuence the expression levels and/or activities of GadX and GadW , which in turn activate the expression and/or activities of transcription activators ( e.g. , YdeO and YhiE ) that directly activate subsets of target genes involved in AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>CRP activated expression of sotA ( 4 ) , while KdgR , which represses transcription of many genes involved in transport and catabolism of 2-keto-3-deoxy-gluconate in E. coli ( 22 , 27 ) , was shown to weakly repress the expression of sotA in D. dadantii ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Here , we show that CRP , a global regulator involved in sugar me-tabolism , regulates cspE transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>CRP activates transcription of about 200 genes in E. coli ( 9 ) in response to the carbon source available .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>These results suggested that lsrR expression was subject to catabolite repression , and CRP was needed for stimulation of lsrR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>This raises the question of why CRP should directly regulate transcription of aer and trg , particularly when CRP indirectly regulates transcription of all genes in the ﬂagellar cascade by controlling expression of the master regulator , FlhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Interestingly , the transcription of glpF and glpK is regulated by CRP and GlpR , whereas the expression of glpD is regulated by CRP , GlpR and ArcA , as proposed elsewhere [ 1,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>To further investigate this we examined the e $ ect of increasing amounts of CRP on dad expression in a puri ® ed in vitro transcription system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Genes which are under positive transcriptio-nal control frequently lack the -35 consensus region , as they use regulatory proteins such as CRP to enhance transcription initiation ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3031590" target="_blank">3031590</a>]<br></li>
<li>( iii ) Most of the sugar transport operons , whose transcription is activated by CRP , are also under the control of repression by other transcriptional-regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>Thus the global regulators are proteins that control responses to changing food levels and carbon degradation ( mlc , lrp , CRP ) and transcription factors that respond to changes in redox status or ion levels of the cell ( arcA , narL , FNR , fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>These seemingly include direct interactions by CRP and FNR to activate transcription initiation and indirect effects by acetate metabolic enzymes and transcription factors that control carbon ﬂux .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>This substitution increased the ability of AraC to activate transcription in the absence of CRP , such that cya mutant cells regained the ability to use arabinose as the sole carbon source ( 50 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>araFGH is highly sensitive to catabolite repression and is not capable of significant transcription in the absence of CRP either in vivo or in vitro ( 29 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>694 J. K. Lithgow , F. Haider , I. S. Roberts and J. Green effect on hlyE transcription in vivo in the absence of H-NS , and active CRP and FNR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>However , in the absence of a productive Fis-aIICTD interaction , seen with the heterodimeric Es38 preparation lacking aIICTD , CRP is unable to stimulate transcription from its position at ÿ121 .5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>To activate acsP2 transcription ( in the absence of IHF ) , CRP performs two roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>We also found that CRP and His6-RhaS-CTD together could activate transcription to a level that was threefold higher than that by His6-RhaS-CTD alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>Transcription depends upon CRP , the anaerobically triggered transcription activator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Transcription from galP1 is stimulated by CRP , ynfE is FNR dependent and yeaR transcription requires NarL [ 25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23341772" target="_blank">23341772</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535077" target="_blank">12535077</a>]<br></li>
<li>Previous studies have shown that the Lys52Asn substitution in CRP greatly increases the ability of CRP to activate transcription at class II CRP-dependent promoters ( 11 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>DNase I analysis and in vitro transcription assays showed that IHF can bind simultaneously with CRP and can anti-activate CRP-dependent acsP2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Also , evidence has accumulated to indicate that CAMP-CRP complex represses the transcription of as many genes as it activates ( Mallick and Herrlich , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>sites , CRP is proposed to increase the rate of formation of the open transcription complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>There are many examples of transcription regulators that interact with the - CTD of RNAP to activate transcription , and interactions between - CTD and CRP have been particularly well studied ( reviewed in reference 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Given that a-CTD and CRP can interact to activate transcription at a variety of promoters ( 29 ; reviewed in reference 6 ) , we looked for evidence of such an interaction at rhaBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>CRP bound to these sites is thought to activate transcription via two distinct interactions with RNAP ( reviewed in ref 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>1993 ) , one such model explaining how CRP can activate transcription when bound at positions from - 4 1 to -- 103 in different promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>As expected , the activities of both transport promoters were modestly decreased upon the loss of crp ( Fig. 7A and B ) verifying that CRP activates fadL and fadD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>In the proposed network ( Fig. 6C ) , RpoS slightly represses the transcription of CRP itself known to activate the ﬂhDC operon ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>If anything , CRP increased transcription in the latter template ( Fig. 1B , lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>These results support the hypothesis that CRP activates acs transcription by means of class I interactions and not via class II interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Varying the concentration of the transcription-initiating nucleotide ( GTP ) , salt ( KCl ) , or CRP did not enhance the responsiveness of PguaB to CRP in multiple - or single-round reactions ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>We demonstrate that CRP is involved in the formation of distinct nucleoprotein complexes that can either activate or repress fis transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>CRP modulates ﬁs transcription by alternate formation of activating and repressing complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12888342" target="_blank">12888342</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2987">
Nasser , W. , Schneider , R. , Travers , A. , and Muskhelishvili , G. ( 2001 ) CRP modulates ﬁs transcription by alternating formation of activating and repressing nucleoprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></label>
<input id="act_similarity_2987", type="checkbox">
<div class="ACT_similar">
<li>Nasser , W. , Schneider , R. , Travers , A. , and Muskhelishvili , G. ( 2001 ) CRP modulates ﬁs transcription by alternate formation of activating and repressing nucleoprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>This ﬁnding indicates the repression of pck transcription when CRP binds to both CRP boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>A novel ﬁnding of this study is the involvement of CRP in transcription repression of the pck gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>CRP2 at 293.5 , a location from which CRP can not by itself activate transcription ( 29 ) , is important along with CRP1 for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CRP appears to activate and also transcription directly indirectly by breaking the repression loop ( 23 , 24 , 39 , 41 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>This indicates that in the malEpD92 context , CRP does not cause increased transcription by stabilizing MalT at sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>This compound is known , to increase - in vitro transcription of operons - regulated by CRP ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>This successive appearance of HU mRNAs is independent of FIS which regulates the hup genes FIS and CRP regulate the transcription of both hup genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>This result indicates that CRP positively regulates the transcription of gyrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>As predicted , if CRP II enhanced transcription , pacs155 exhibited about one-half the activity displayed by pacs205 , which includes both CRP I and CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2998">
Antagonistic effect of CRP and KdgR in the transcription control of the Erwinia chrysanthemi pectinolysis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></label>
<input id="act_similarity_2998", type="checkbox">
<div class="ACT_similar">
<li>Nasser , W. , Robert-Baudouy , J. , and Reverchon , S. ( 1997 ) Antagonistic effect of CRP and KdgR in the transcription control of the Erwinia chrysanthemi pectinolysis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li></div>
<li>Similarly , the effect of CRP on the induced level of setA and sgrS transcription was mediated through SgrR-dependent signaling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>CRP appears to prevent σS-dependent transcription of the gad genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>Thus it was concluded that the CAMP-CRP complex is required to activate transcription of the a m 4 gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>Thus , CRP inhibits transcription of cy from a multicopy plasmid to the same extent as it does that of the chromosomal cya gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Taken together , these findings indicate that , in the deletion construct CRP , TdcA and TdcR are functional in controlling transcription of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>We found that transcription of the ykgMO operon is repressed by Zur and that azuC and ykgR transcription is repressed and activated by CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Third , at the HNS-repressed bgl operon BglJ -- RcsB activates transcription synergistically with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Finally , because of the necessity for data manipulation to identify only the most robust changes in transcription , promoters that are only weakly activated by CRP in vitro will not be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP also activates transcription at all three of the rha promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>These data show that Fis and CRP activate transcription synergistically at proP P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>We proposed that CRP activates transcription from acsP2 by a synergistic class III mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>By these criteria , CRP , a paradigm of activator of transcription , becomes bifunctional ; it activates and represses different steps of the initiation pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>CRP activates transcription by using two different modes of RNAP interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Specific sequences to which the CRP and FNR proteins bind , to stimulate transcription , have been proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>AraC and CRP were both required for maximal transcription in vitro , although a low level of transcription was detected with CRP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Effects of CRP and CAMP on in vitro transcription of the ompA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>Here , we have shown that CRP and CAMP inhibit in vitro transcription of the ompA gene but not the Zppgene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>CRP activates transcription in vitro by focusing RNAP to P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Using an in vitro transcription assay , we determined that CRP activates transcription of acs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>CRP represses P1 transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>CRP increases PcspD transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>RESULTS Fis and CRP activate transcription synergistically in vitro at proP P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>RhaR and CRP synergistically activate transcription of rhaSR in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Both CRP I and CRP II contribute to acs transcription in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Effects of FNR , CRP , and H-NS on in vivo transcription of hlyE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Global transcription factors such as ArcA , Fnr , IHF , Fur , and CRP respond to both NADH and ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>CRP is required to activate guaBA transcription and repress xseA transcription following downshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>CRP can activate or repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Consequently , it was assumed that TcbC somehow replaced CRP to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Effect of CRP on P1 transcription as assayed by primer extension .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Active CRP was then added , and at regular intervals samples were removed and assayed by runoff transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>When CRP EV181 is introduced instead of wild-type CRP , transcription from the XX-CC promoter is increased , whereas expression from CC-CC is decreased by nearly twofold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Osmotic sensitivity of CRP -- cAMPmediated activation of the lac promoter was assessed by comparing activated transcription of the CRP-dependent wild-type lac promoter with the CRP-independent lacL8UV5 promoter after addition of 0.5 M NaCl to cells growing in LBN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Our study shows that both [ Leu159 Glu101 Asn52 ] CRP and wild-type CRP , in conditions where they can stimulate transcription and participate in ternary CRP-RNAP-galp1 complexes , have no effect on the location of s70 region 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>When triggered by melibiose , wild-type MelR requires the assistance of CRP to activate transcription at pmelAB ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>Maximal transcription was still dependent on AraC and CRP ; low levels of transcription , similar to those obtained with the wild type , were obtained for WH592 and WH603 with CRP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>Together , these results suggest that while CRP may repress basal levels of setA transcription , neither CRP nor KdgR is likely to exert any direct effect on setA transcription under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Recall that , in the conditions used in our experiments , direct effects of CRP on transcription of promoter : : lacZ fusions in pRW50 can be measured independent of the effect of CRP on s28 levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>It has been reported that even though the main role of the CAMP-CRP complex is to activate transcription , it can also , in some cases repress it , as it does in the case of the P2 promoter in the gal operon , and the ompA and clp promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2465204" target="_blank">2465204</a>]<br></li>
<li>Full transcription from the rhaBAD promoter requires CRP ( catabolite repression protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Plasmid-borne promoter fusions to the receptor gene for chloramphenicol acetyl transferase were used to assess the role of CRP in controlling transcription from guaP and xseP following a downshift of cultures from rich into minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>When AR2 is disrupted by mutation , or when either the promoter-distal CRP II or the promoter-proximal IHF III sites remain unoccupied , some of these upstream contacts are lost , and transcription can proceed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In this case , activating region 1 ( AR1 ) of the downstream monomer of CRP contacts the RNAP - CTD to recruit RNAP to the promoter and activate transcription ( reviewed in reference 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>At the rhaBAD promoter , CRP can activate transcription only when RhaS is bound adjacent to RNAP ( 12 ; unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>This indicates that CRP can not activate transcription from the aer promoter when bound at a Class II location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>CRP is considered to behave as a simple transcription activator when it binds closer to the promoter start , i.e. , at positions 241.5 , 261.5 , and 271.5 ( 8 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>First , some operons require other factors for CRP to activate transcription , e.g. AraC at the araBAD promoter ( 25 ) , which are missing in the ROMA system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>When glucose or othe ( Table 3 ) , indicating that CRP is needed to activate transcription from the lsr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In addition , the synchronized pattern of increasing CRP activity causing increasing FlhDC expression with decreasing carbon source quality , together with the apparent coupling of motility activity with the activation of motility and chemotaxis genes in poor quality carbon sources , highlights the importance of CRP activation in allowing E. coli to devote progressively more of its limited reserves to search out better conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The correlation between the activity of CRP and the motility of cells indicates that activation of CRP plays an important role in triggering E. coli ` foraging ' - like behavior ( 20 -- 22 ) to actively search for better conditions as the quality of the available substrate ( s ) decreases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The upregulation of motility genes correlated with increasing motility of cells in poor quality carbon sources , together with the synchronized pattern of increasing CRP activity causing increasing FlhDC transcription , further suggests CRP is a factor ( or at least one of multiple factors ) to play a positive role in triggering E. coli ` foraging ' - like behavior to actively search for better conditions as the quality of the available substrate ( s ) decreases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>This stabilizing effect most likely results from the assembly of the higher-order structure formed by MalT and CRP at malEp-malKp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>CRP bound to site 1 plays a speciﬁc role in malEp + activation The observation that CRP site 1 still strongly stimulated malEp in the absence of any distal MalT sites ( Raibaud et al. , 1989 ; Vidal-Ingigliardi et al. , 1991 ) suggested that the CRP molecule bound to site 1 might play an active role in malEp activation , besides its structural role in the assembly of the active complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Discussion A unique nucleoprotein assembly activates malEp and malKp The MalT sites 1 and 2 , as well as the CRP sites 1 , 2 and 3 had all been shown to play a role in malEp activation ( Vidal-Ingigliardi & Raibaud , 1991 ; Vidal-Ingigliardi et al. , 1991 ) , but so far it was not clear which of the MalT sites present in the distal region are involved in promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>One role of the higher-order structure formed by MalT and CRP when bound to the entire regulatory region is to ensure high occupation of MalT sites 1 and 2 , but it is not its only function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>This study indeed revealed the following : ( 1 ) occupation of MalT sites 1 and 2 is sufﬁcient to promote malEp activation ; ( 2 ) in vivo , under conditions of full induction , the activity of the truncated malEpD64 promoter is severely limited by the degree of occupancy of MalT sites 1 and 2 ; ( 3 ) one role of the upstream regulatory elements is to stabilize MalT at sites 1 and 2 ; ( 4 ) full occupation of these sites is not sufﬁcient , however , to ensure full promoter activity ; ( 5 ) CRP site 1 and the upstream MalT sites can independently increase the capacity of MalT bound to sites 1 and 2 to stimulate malEp ; and ( 6 ) this effect of CRP bound to site 1 does not involve stabilization of MalT at sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Hence , this suggests that malEpD64 might actually represent a minimal MalT-dependent promoter that is inactive because of a too low occupation of MalT sites 1 and 2 , and that the role of the upstream elements in malEp activation might simply be to ensure high occupation of MalT sites 1 and 2 through the assembly of a higher-order structure with MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Some elements of this structure , namely the CRP site 1 , located at − 76.5 , and the distal MalT sites , seem to play a direct role in malEp activation besides their participation in the assembly of the higher-order structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Formation of this structure is disfavored when MalT is present alone and the role of CRP in malKp activation is to facilitate the repositioning of MalT onto sites 3 ' / 4 ' / 5 ' by stabilizing this higher-order structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Hence , malEp activation primarily relies on MalT sites 1/2 , 3 ' / 4 ' / 5 ' and on the CRP sites 1/2/3 , i.e. on the same set of regulatory elements as malKp activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>An insertion of one DNA helical turn , 11 base-pairs , between CRP and aral only partially inhibits CRP loop breaking and activation of pBAD , while an insertion of less than one DNA helical turn , 4 base-pairs , not only abolishes CRP activation and loop breaking , but actually causes CRP to stabilize the loop and increases the araO,-mediated repression of pBAD.Both integral and non-integral insertions of greater than one helical turn completely abolish CR , P activation and loop breaking in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>In addition , the expression level of the lsrACDBFG operon is much higher in wild-type cells than in the luxS mutant , as CRP is required for its activation ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In the wild-type background expression of the gntP-lacZ fusion was approximately 100-fold higher than in either the crp or cya mutants , indicating that cAMP and CRP are required for activation ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>In order to test the involvement of different domains of CRP in PcspD activation , PcspD activity was studied in M182 crp cells complemented with different activation region mutant alleles of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>CRP-dependent activation requires activating region I. - Galactosidase activity of a CB12 lysogen deleted for crp ( strain AJW2163 ) and transformed with plasmids expressing either wild-type CRP , CRP H159L , CRP K101E , or CRP H159L/K52N was determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Selectivity of the Escherichia coli E 38 for overlapping promoters and ability to support CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Li , B. , Wing , H. , Lee , D. et al. ( 1998 ) Transcription activation by Escherichia coli FNR protein : similarities to and differences from , the CRP paradigm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Transcription activation by Escherichia coli FNR protein : similarities to , and differences from , the CRP paradigm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Li , B. , Wing , H. , Lee , D. , Wu , H. , and Busby , S. ( 1998 ) Transcription activation by Escherichia coli FNR protein : similarities to , and differences from , the CRP paradigm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Repositioning of the CRP site at 241.5 bp abolished activation by the TcbC protein and allowed CRP-dependent transcription to occur on linear , relaxed and supercoiled templates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Most notably , in E.coli CRP binds to and stimulates transcription at novel CRP sites with a distinct consensus ( CRP-S ) that makes transcription activation dependent on an additional protein factor , Sxy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>At some promoters , transcription activation by CRP can be repressed by a transcriptional-regulator protein , CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3072">
Orientation of functional activating regions in the Escherichia coli CRP protein during transcription activation at class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></label>
<input id="act_similarity_3072", type="checkbox">
<div class="ACT_similar">
<li>Orientation of functional activating regions in the E. coli CRP protein during transcription activation at class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_3074">
Williams , R.M. , Rhodius , V.A. , Bell , A.I. , Kolb , A. , and Busby , S.J.W. ( 1996 ) Orientation of functional activating regions in the Escherichia coli CRP protein during transcription activation at Class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></label>
<input id="act_similarity_3074", type="checkbox">
<div class="ACT_similar">
<li>Williams , R. , Rhodius , V. , Bell , A. , Kolb , A. and Busby , S. ( 1996 ) Orientation of functional activating regions in the E.coli CRP protein during transcription activation at class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_3076">
Williams , R , , Bell , A. , Sims , G. , and Busby , S. ( 1991 ) The role of two surface exposed loops in transcriptional activation by the Escherichia cofi CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></label>
<input id="act_similarity_3076", type="checkbox">
<div class="ACT_similar">
<li>Williams , R. , Bell , A. , Sima , G. and Busby , S. ( 1991 ) The role of two surface exposed loops in transcription activation by the Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Williams , R. , Bell , A. , Sims , G. and Busby , S. ( 1991 ) The role of two surface exposed loops in transcription activation by the Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Williams , R. , Bell , A. , Sims , G. , and Busby , S. ( 1991 ) The role of two surface exposed loops in transcription activation by Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li></div>
<li>7 1996 Academic Press Limited Keywords : MalT ; CRP ; Escherichia coli ; transcription activation ; DNA loops Unité de Génétique Moléculaire URA CNRS 1149 Institut Pasteur 25 rue du Dr. Roux 75724 Paris Cedex 15 France Introduction Prokaryotic transcriptional-activators act either alone or synergistically , depending on the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3081">
Synergistic transcription activation : a dual role for CRP in the activation of an Escherichia coli promoter depending on MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></label>
<input id="act_similarity_3081", type="checkbox">
<div class="ACT_similar">
<li>Richet , E. ( 2000 ) Synergistic transcription activation : a dual role for CRP in the activation of an Escherichia coli promoter depending on MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_3083">
Role of CRP in transcription activation at Escherichia coli lac promoter : CRP is dispensable after the formation of open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9737918" target="_blank">9737918</a>]<br></label>
<input id="act_similarity_3083", type="checkbox">
<div class="ACT_similar">
<li>Tagami , H. & Aiba , H. ( 1995 ) Role of CRP in transcription activation at Escherichia coli lac promoter : CRP is dispensable after the formation of open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li></div>
<li>Identiﬁcation of the effectors involved in GadX activation and further investigation of the interactions between regulatory components of the gad system ( H-NS , RpoS , CRP , and GadX ) may provide further insight into the molecular circuitry underlying the glutamic acid-based acid stress response of E. coli A. Tramonti and P. Visca contributed equally to this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Given the importance of AR in the ecology of E. coli , it is not surprising that several regulatory inputs , involving at least four global regulators , HN-S , CRP , RpoS , and EvgA , regulate the AR response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>However , in the purR gcvA gcvR strain carrying the gcvAF31A allele ( little activation and no repression ) , the crp deletion had no effect ( Table 7 ) , suggesting that CRP 's role in the regulation of gcvT-lacZ is dependent on GcvA 's ability to repress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>While it is known that carbon catabolite repression affects the ﬂagellar synthesis and the CRP activation might be involved in alleviating this repression ( 41,42,44,45 ) , much less is known regarding the role of CRP in motility regulation under a range of carbon source conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>Here we report that a new sRNA regulator , McaS , through activation by CRP responds to nutrient limitation and coordinates the regulation of motility , yet another physiological response not previously known to be targeted by sRNAs , and biofilm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>Also , Lu and Abdelal proposed that expression of the putative 54-dependent astC promoter in S. enterica serovar Typhimurium is subject to carbon catabolite repression and requires CRP , together with arginine and ArgR , for activation of the ast operon ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>We do not believe that an overlapping class II CRP-dependent promoter contributes in any signiﬁcant manner to rhaBAD activation since expression from this promoter is extremely low ( 0.01 U ) in the absence of RhaS ( unde conditions where CRP would be expected to activate if it could ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>In both cases , deletion of the distal MalTbinding sites impaired promoter function to about the same extent as in a crp ' strain ( compare Kp228 to Kp240 , and Epl99 to Ep236 ) , showing that in the absence of CRP MalT boxes located between positions -200 and - 240 upstream from the transcription start site are still involved in transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>In order to simplify our analysis , we exploited an engineered derivative of the melAB promoter in which MelR binding to site 2 and site 2 cents , in the absence of CRP , is sufﬁcient for transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>In contrast to the expected repression , we found that in the absence of both RhaR and the CRP binding site at the rhaSR promoter , RhaS activated expression to a level comparable with RhaR activation of the same promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In our model for synergistic activation of proP P2 , Fis minimally contacts one a-CTD domain , and the second a-CTD domain is bound to the CRP subunit oriented proximal to the promoter ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In the acsP2 promoter , the upstream CRP site II centered at position 122.5 has low affinity and CRP that is bound there acts as a co-activator for optimal activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In addition to stimulation of proP P2 transcription by Fis , CRP co-activates this promoter when bound to a remote site upstream from the promoter ( centered at ÿ121 .5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>In this case , the upstream CRP is needed for transcription activation in conjunction with AraC , but the role of CRP bound at downstream of the promoter remains unsolved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Our results show that the fucPIK promoter , a complex promoter which ordinarily depends on both CRP and the fucose-speciﬁc regulator FucR for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>We show furthermore that direct contacts between RNA polymerase and regions of IHF corresponding to ARI , ARII or ARIII of CRP are not responsible for activation : the class II promoter variants of malT are repressed by IHF binding , whereas the homologous CRP promoters are activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>acs promoter and that its activation in vitro and in vivo requires only CRP , which binds tandem DNA sites ( centered at positions 69.5 and 122.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>The function of H-NS at the csiD promoter would be to counteract this ﬂexibility and thereby improve the precision of activation from the optimal CRP box position , maybe by binding to several sites along this region as a ` local DNA scaffold ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In this study , we have shown that the 54-dependent prpBCDE promoter is positively regulated by CRP together with PrpR , which implies direct contact between CRP and E 54 or activation via DNA conformational changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Our results suggest that DNA topology and direct protein-protein contacts contribute to a comparable degree to the process of transcriptional activation by CRP at the malT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>In general , proximal activation sites are located to overlap the promoter -35 region , or , in the case of CRP , alternative upstream in-phase positions are sometimes used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Here , we used our series of strains with the CRP box located at different positions in the csiD promoter region in order to test whether this modulation by H-NS and Lrp is coupled to activation by CRP or proceeds by CRP-independent mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>the activation of t , he lac promoter and arnC ' B. 4 I ) 1 ) ~ CRP is similar in that CRT ' shows the S ~ IIIP orientation tlependcnce in both operons : as in nrtc ( ` H ~ ~ lO , CRT ' can partially activate the lnc promoter when its distance from the promoter is increased by 11 base-pairs , but not by five base-pairs , both in uiuo ( Mandecki $ Carut ' hers , 1984 ) and in r * itro ( St.raney et n / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Our study provides a first example of synergistic repression by FIS and CRP , which contrasts the synergistic activation reported for the RpoS-depend-ent proP P2 promoter and also , for the malEp promoter ( 30 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>These observation also imply that the organization of the different CRP sites in cddP makes it possible for the promoter to achieve both maximal activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Thus , although it is possible that part of the effects of the a mutations could be mediated through Lrp and CRP , the results are more consistent with a GcvA Æ aCTD interaction likely being required for GcvA-mediated activation at the PgcvT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>These results are consistent with the hypothesis that activation of the previously identiﬁed rhaSR promoter as well as the alternative promoters that result in the CRP A and CRP B transcripts are all due to interactions between CRP and - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Previous results by Holcroft and Egan did not provide conclusive evidence for or against an interaction between CRP and - CTD at the rhaSR promoter ( 15 ) , so we ﬁrst tested other possible mechanisms of CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Our ﬁnding that several point mutations in - CTD also decreased CRP-dependent in vitro activation of the rhaSR promoter further supports our conclusion that CRP activation of rhaSR requires interaction with - CTD ( unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>When CRP is bound around -40.5 , the activation factor is -10 ( Table I ) and the basal promoter activity is very low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>When we measure the activation factors of this CRP mutant with our malT promoter variants , we obtain the same activation profile as for the wt CRP , but all activation factors are decreased -10 - fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>This is not surprising because CRP positioned upstream of the promoter usually does not mediate transcriptional activation through AR2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>This confirms that CRP is a primary activator of the promoter and that AraC alone is not capable of activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>This appears to occur in CRP activation of the lacP1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>We discuss models that account for the different activities of CRP in transcriptional activation as a function of promoter geometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Future efforts will be directed to understanding the synergy between CRP and MhpR and the molecular mechanisms governing the transcriptional activation of the Pa promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>Wang , Y.P. , Kolb , A. , Buck , M. , Wen , J. , O'Gara , F. , and Buc , H. ( 1998 ) CRP interacts with promoter-bound s RNA 54 polymerase and blocks transcriptional activation of the dctA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>CRP interacts with promoter-bound s - RNAP and blocks tran-54 scriptional activation of the dctA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>We observe another strong peak of activation when CRP is bound at approximately the position of the wt malT promoter ( -70.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Previous studies have indicated that the ptsG promoter , which contains a CRP activation site ( see Table 2 ) ( 36 ) , is both activated by CRP and strongly repressed by the pleiotropic transcriptional-repressor Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Our in vivo transcription assays indicated that CRP site 3 is required for CRP activation of rhaSR expression and con-ﬁrmed that CRP site 1 is the only site required for CRP activation at the rhaBAD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Effect of moving the DNA site for CRP on activation of the aer promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Why does CRP site 1 not mediate activation by wild type CRP at the IS5-disrupted fucPIK promoter ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>CRP site 3 is critical for in vitro CRP activation of the rhaSR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The class I activation mechanism that CRP utilizes when bound to CRP2 recruits RNAP via an interaction between the promoter proximal subunit of the CRP dimer and the polymerase R-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The class II mechanism that CRP utilizes when bound to CRP1 also features this interaction , although in this case with the promoter distal subunit of bound CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The modesty of the activation is partly due to the activator-independent strength of the malI promoter , but principally it is because CRP site 1 is located 34 bp upstream of the 10 hexamer element , which is 2 bases upstream of the optimal position for Class II activation ( Busby & Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>The CRP protein represses the P1 promoter under low osmotic growth conditions ; however , the effect of CRP on P2 activation has not been reported ( 33 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The requirement for AR1 on CRP for co-acti-vation of proP P2 indicates that the function of CRP is mediated through the aCTD .29,30 In our model for co-activation of proP P2 by Fis and CRP , the aI subunit interacts with the upstream CRP protein through its promoter-proximal subunit ( Figure 7 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The promoter activity was reduced by the single base pair substitutions at the 10 and extended 10 elements , and CRP repression was switched back to activation by the mutations ( Table II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3135">
Sclavi , B. , Beatty , C. , Thach , D. , Fredericks , C. , Buckle , M. , and Wolfe , A. ( 2007 ) The multiple roles of CRP at the complex acs promoter depend on activation region 2 and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></label>
<input id="act_similarity_3135", type="checkbox">
<div class="ACT_similar">
<li>Sclavi , B. , Beatty , C.M. , Thach , D.S. , Fredericks , C.E. , Buckle , M. , and Wolfe , A.J. ( 2007 ) The multiple roles of CRP at the complex acs promoter depend on activation region 2 and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li></div>
<li>The lack of defect from 70 K593A , R599A , or R608A at the ( rhaS-lacZ ) 216 promoter is similar to previous ﬁndings with RhaS and AraC that substitutions at 70 some residues were defective only in the absence of CRP activation ( 4 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The mechanism of CRP activation varies depending on where CRP binds relative to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The intrinsically defective lacP1 promoter requires CRP for activation of transcription both in vitro and in vivo ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The predominance of mutations affecting CRP is consistent with the central role of CRP in activation of the promoter ; CRP alone can stimulate the promoter both in vivo and in vitro , whereas AraC alone has no effect ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>With the aI / aII mutant Es38 , Fis alone activated transcription 17-fold , similar to its effect on the wild-type enzyme , but in the presence of both Fis and CRP there was little or no increase in activation ( Figure 6 ( a ) , lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Further , if these s70 residues are important for activation by RhaS in the absence of CRP , it is possible that other AraC / XylS family proteins that activate transcription without the aid of a second activator , such as CRP , may also require these residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>CRP and RhaR could each partially activate transcription in the absence of the other two activators , and a-CTD was not capable of activation alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>This in vitro transcription system mimics the in vivo activation of rhaBAD by RhaS-CTD and also , to a great extent , the in vivo activation of rhaBAD by full-length RhaS , each in the presence and absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>At the rhaSR promoter , the 70 L595A derivative was unique in that it was defective in both the presence and absence of the second activator , CRP , but it 's role in RhaR-mediated transcription activation is not yet known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Our results show that the pattern of cleavage is identical in both the absence and the presence of CRP , implying that CRP does not reposition s70 region 4 during transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>CRP alone does not activate rhaBAD expression , but in the presence of RhaS CRP can contribute 30 - to 50-fold additional activation ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>CRP activation at some promoters , such as melAB and araBAD , is dependent on the presence of an additional regulator ( MelR and AraC , respectively ) ( 33,25 ) that is only induced by a speciﬁc substrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>This smaller defect in the presence of CRP activation was not expected given the hypothesis that the primary role of a-CTD would be interaction with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Fis and CRP co-activation of proP P2 in vitro in the presence of a-heterodimeric Es38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Comparison of transcripts synthesized by the - 235 and - WT enzymes in the absence and presence of CRP demonstrated that nearly all of the CRP-dependent activation of pSR depended on - CTD in the presence of RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>pSR activation by CRP was determined by dividing the intensity in the presence of CRP by the intensity in the absence of CRP for each pair of lanes ( intensity of pSR was corrected on the basis of the intensity for control p82 transcripts ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The lack of sequence dependence for the CRP site in the ara region may also be because the araBAD operon , unlike the gal and lac operons , depends not just on CRP activation but also on activation by a specific activator , AraC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>The contribution of DNA bending to transcriptional activation is more important when CRP is bound further upstream and protein-protein contacts are more difficult to establish .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The requirement for Fis for activation by CRP could , at least in part , re ¯ ect the ability of Fis to bend DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The requirement of both RhaR and - CTD for signiﬁcant CRP activation of pSR suggests a possible mechanism by which RhaR bends DNA to allow CRP to interact with - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The mechanism of CRP activation has been studied in detail and is believed to involve DNA bending , direct interaction with RNA polymerase ( Ebright , 1993 ; Ishihama , 1993 ; Kolb e t al. , 1993b ; West e t a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The activation factors are smaller than for CRP either because additional interactions exist between the genuine activator CRP and RNA polymerase , or because the geometrical distortions introduced by IHF do not exactly mimic the DNA conformation induced by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The mechanism by which RhaR increases CRP activation is not known , but it does not appear to be through cooperative binding on the basis of our DNase footprinting results ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The binding of MalT and CRP to this region is known to result in the formation of a higher-order structure that is responsible for malKp activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The steep decrease of transcriptional activation when the CRP binding site is moved away from the -60.5 position ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The decrease in transcriptional activation by CRP upon shifting the CRP binding site I bp away from the optimal position is indeed of this magnitude .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The 287 determinant is required for activation by CRP but is not required for DNA binding or UP element-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>The mechanism of activation of pBAD by CRP apparently does not involve stimulation or stabilization of Arab binding to aral .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The question however , is why CRP requires co-binding of CaiF for full activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The position of the CRP site at rhaSR ( 111.5 ) is less typical , and in the tandem CRP site studies , CRP sites near this position made little , if any , contribution to activation ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>The lower afﬁnity site , CRP II , centred at position -122.5 , is necessary for maximal activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The contribution by CRP requires the correct helical phasing of the CRP site and a functional activation region 1 on CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The location of the CRP site at 2106.5 is therefore appropriate to permit CRP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The presence of this site was shown to be essential for PcspE activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The full-length fusions include the binding site for CRP in addition to that for RhaS , indicating that in the presence of CRP the contribution of these residues to rhaBAD and rhaT activation was either decreased or eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>The threefold contribution to activation by CRP in this experiment is very similar to the twofold contribution found in the above in vivo experiment ( Table 3 ) , conﬁrming that RhaS-CTD is sufﬁcient to allow at least partial CRP activation of rhaBAD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>The increased expression from the longer of these fusions is due to the contribution of CRP to the activation ( 15 , 34 ) , which in this case was nearly 100-fold in combination with RhaR but was less than fourfold in combination with RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>The following shows that CR ( P activation in viva is also dependent upon the distance and orientation between the CRP and aral sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The data in Table 1 show that , like CRP-induced loop breaking , activation of peAD by CRP is dependent upon its orientation and distance from aral .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The full level of activation that is seen with CRP and wild-type Fis is not achieved ; however , it is evident that the transcriptionally impaired Fis mutants are still competent for coactivation with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The requirement for activating region 1 of CRP in PcspD activation , along with the involvement of the 287 , 265 , and 261 determinants of the - CTD , suggest that CRP activates by a class I-type mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>The small defect upon substitution of CRP G162 leaves open the possibility that CRP activation of rhaBAD could involve AR1 contacts with a-CTD ; however , the evidence for such an interaction is not strong .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>The magnitude of the synergistic effects was usually greater with just two activators than with all three , suggesting possible redundancies in the mechanisms of activation by CRP , a-CTD , and RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>The stabilizing effect of CRP is lost , suggesting that the R-subunits are involved in the CRP-mediated activation , most likely by a direct interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The activation factor Position CRP ( activation ) Position IHF ( inhibition ) Activation factors are given as the ratio of luminosities in wt strains compared with strains deleted for the activator ( CRP or IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The fact that Lrp and CRP are required for full activation of PgcvT [ 14,15 ] complicates interpretation of the results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>The magnitude of IHF activation is smaller than the activation obtained with CRP bound at the same positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The significance of this difference is easily understood in the context of the dual transcriptional activation mechanisms mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The allostery of CRP activation has been understood mainly in terms of conformational change .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The rate-limiting step , therefore , in induction of the regulon be ara may activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>4 The strong CRP activation of dctA ensures that external C4-dicarboxylates are taken up for catabolic purposes only in the absence of `` more preferable '' carbon and energy sources , such as glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>The - WT RNAP showed levels of activation by CRP and RhaR that were comparable to those obtained with RNAP puriﬁed as a holoenzyme ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The level of prpB mRNA in M9-glucose culture ( Fig. 6 , lanes 3 and 4 ) was lower than that in LB culture ( Fig. 6 , lanes 1 and 2 ) , in agreement with the involvement of CRP in activation of the prpBCDE operon ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>When CRP represses at the ompA and cya promoters , it is bound outside of its activation position and within the common zone of repression ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This would necessarily lead to a two-step activation mechanism at both deoP2 and udpP : first , a relatively modest activation due to the initial recruitment of RNAP by CRP bound at CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This component of CRP activation at rhaSR could account for the majority of the CRP activation in a wild-type context and may involve a mechanism , such as DNA bending , that can act from a distance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>If an interaction between CRP and a-CTD does occur at rhaSR , it appears to make a relatively small contribution to activation in the wild-type context .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Gaston et al. ( 9 ) demonstrated that a DNA site for CRP located at 69.5 resulted in weak activation compared to the preferred 61.5 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Nevertheless , in the case of deoP2 , if a-CTD ± UP-element interactions and a-CTD ± CRP interactions both recruit a-CTD to the DNA site , the activation system appears to be redundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>This smaller defect may indicate that CRP displaces a-CTD to a weaker DNA site , thereby reducing the ability of a-CTD to contribute to activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>In addition to the possible interaction between a-CTD and CRP , the alanine scan results indicate that DNA contacts by a-CTD are important for its activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>We wished to identify residues in a-CTD involved in this activation and to investigate whether a-CTD activation depends on interactions with DNA and/or CRP and / or RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>First , the only contribution of a-CTD to activation could be by interaction with DNA , and all of the activation signals that CRP and RhaS transmit to RNAP could be transmitted by a-CTD-in-dependent mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Overall , our results suggest that CRP interaction with a-CTD may not be required for rhaBAD activation ; however , a-CTD does contribute to full activation , probably through interactions with DNA and possibly RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Overall , our results are most consistent with a model for rhaBAD activation in which CRP activates by a mechanism other than interaction with a-CTD and in which a-CTD activates by interacting with DNA and possibly RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>When supercoiled templates were used , the activation effect of DNA bending was about 1.6-fold for CRP and fourfold for IHF ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>H159L When linear templates were used , the activation effect of DNA bending was about 25-fold for CRP ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Recall that for the sigma 70 activators OmpR and CRP , activation diminishes precipitously as the DNA elements are moved out of the nearby proximal position ( 40 , 81 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Removal of this DNA almost completely abolishes transcriptional activation by CRP in vitro ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>In addition to the effect of CRP , activation of cai and ﬁx promoters has been shown to be subject to the essential function of CaiF , the speciﬁc activator of carnitine metabolism ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>In contrast to activation , the repression levels of most operons were not signiﬁcantly affected by either HL159-CRP or KE101-CRP , which indicates that most CRP repression is not dependent on either the AR1 or the AR2 determinant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In addition , most CRP-dependent promoters are subjected to repression by other factors , which will mask the CRP activation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In this report , we further investigate the regulation of proP by examining the effects of CRP on transcriptional activation of the P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In contrast , when the CRP box is located at 260.5 , efﬁcient activation is obtained with both holoenzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>DISCUSSION In the past few years , genetic and biochemical experiments have identified a surface-exposed loop of CRP ( ARI ) that is involved in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>This deletion also removed CRP site 2 and hence our result conﬁrms that this site plays little or no role in CRP-dependent activation of the malX promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3214">
PcspD with CRP site-I ( 83.5 ) alone also requires activation region I for CRP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="act_similarity_3214", type="checkbox">
<div class="ACT_similar">
<li>Therefore , similar to the wild-type promoter , PcspD with CRP site-I ( 83.5 ) alone also requires activation region 1 for CRP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li></div>
<li>In addition , the CRP molecule bound to site 1 seems to play a direct but so far undeﬁned role in the activation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In addition to decreasing CRP activation of pSR , mutation of CRP site 3 also decreased the amount of the CRP A and CRP B transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In the lac and gal systems , single base-pair changes in the CRP site may result in greatly decreased activation by CRP ( 5 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Our results demonstrated that CRP site 3 was most critical for CRP activation of pSR ( Fig. 5 ) , in agreement with the in vivo results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3220">
In contrast to deletion of CRP site-II , deletion of CRP site-I lead to complete loss of CRP-mediated activation of PcspD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="act_similarity_3220", type="checkbox">
<div class="ACT_similar">
<li>CRP activation was not affected by deletion of CRP site-II , whereas deletion of CRP site-I lead to complete loss of CRP-mediated activation of PcspD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li></div>
<li>Therefore , it can be concluded that CRP site-I plays an important functional role in CRP-mediated PcspD activation , while CRP site-II plays a minor role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>As in the CRP studies , the activation is maximal when the OmpR sites are restricted to the same side of the helix as the natural site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>CRP activation of pBAD in ciz ) o shows the same dependence on t , he distance and orientation of the CRP site relative to aral .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>It was found that CRP mediates activation of araC through this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Partial deletion of CRP target site leads to loss of PcspE activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Both mobility shift assay and mutagenesis results suggest that site 3 has a direct role in CRP activation of rhaSR ; thus we conclude that CRP bound at site 3 is required for full rhaSR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>CRP bound to the lower-afﬁnity upstream CRP II site , however , may substitute for the UP element to enhance activation of P2 from the suboptimal 69.5 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>CRP utilizes a class I activation mechanism when bound to either CRP1 or distal CRP2 site ( 11-16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Synergistic activation of a transcriptional-regulator together with CRP has been shown at several loci including the ara locus by AraC and CRP , malE by MalT and CRP , rha by RhaS / RhaR and CRP and asc by two CRP dimers , among others ( 37,44 -- 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>We also dem-onstrate the role of CRP in the direct activation of the ulaAB-CDEF operon and show that ula regulon expression was not affected in an FNR mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>We hypothesized that the residual FNR protein-indepen-dent expression of the CC fusion construct was due to activation by the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>As expression of rhaS is also dependent on CRP activation ( C. C. Holcroft and S. M. Egan , submitted for publication ) , it was possible that the small effects of the CRP AR1 and AR2 substitutions were indirect , due to decreased RhaS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>However , the CRP contribution to rhaSR activation is substantially lower when RhaS is the primary activator than when RhaR is the primary activator , resulting in a relative decrease in rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>However , the contribution to activation by CRP at the promoter activated by RhaS was greatly reduced compared with the CRP-RhaR combination , thereby resulting in a relative repression of rhaSR expression upon activation by RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>This suggested that CRP is important for activation of ClyA expression by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>This suggests that CRP site 3 is responsible for at least part of the CRP activation of rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Although the single CRP site is involved in activation of both araBAD expression and araC expression , different mutations in the site did not affect expression to the same extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Taken together , our results suggest that CRP site 3 is the major site required for CRP activation of rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Evidence is presented that CRP is directly involved in the expression of the cai and ﬁx genes and plays a crucial role in the activation of carnitine metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>We can deduce that the former group of deletions cause a reduction in both P1 and P2 expression , whilst the latter group prevents the activation of P1 by CRP and the switch from P2 to P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>CRP activation of rhaBAD expression appears to occur by a mechanism other than interaction with a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>To begin to understand the mechanism of CRP activation of rhaBAD expression , we wished to determine whether AR1 and/or AR2 of CRP are necessary for activation at rhaBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Since we now had evidence that CRP was a direct activator of rhaSR expression , we wished to determine whether AR1 and/or AR2 of CRP were necessary for this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>In both cases , there is little or no activation by CRP in the absence of RhaS or RhaR ( Table 2 ) ( 16 ) ; presumably , CRP requires DNA bending by the primary activator to enable contacts with - CTD ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In the absence of both CRP and RhaR , a-CTD did not contribute any activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Since there is no detectable rhaBAD activation by CRP in the absence of RhaS , it was not possible to test the effect of a-D235 expression on CRP activation independent of RhaS activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>We have some evidence that a-CTD activation , at least in the absence of CRP , may involve contacts with RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>As mentioned above , it was not possible to test the contribution of a-CTD to CRP activation in the absence of RhaS since CRP does not activate under such conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>First , the fold activation by - CTD increased in a the absence of CRP [ compare the defects upon a-CTD deletion at F ( rhaB-lacZ ) D110 , 13-fold , and F ( rhaB-lacZ ) D84 , 54-fold ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>This is similar to the increased activation by 70 two of the s substitutions ( E591A and R596A ) when tested at araBAD in the absence of CRP ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>This increased activation by CRP and a-CTD in the absence of the other is the opposite of what would be expected if CRP activation required contacts with a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>This would represent the 4-fold activation by RhaR , and the 20-fold and 13-fold activation by CRP , in the absence of other activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Effect of CRP AR1 and AR2 on activation Previous studies have shown that CRP activation is dependent on the interactions of its activating regions with subunits of RNAP [ reviewed in ( 3 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>To test whether the - CTD was required for full activation by CRP or RhaR , we reconstituted RNAP from its individual subunits in order to obtain RNAP preparations in which both copies of the - CTD were deleted for all residues after amino acid 235 ( - 235 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We expected that CRP activation would involve contacts with - CTD ; hence , - CTD would be required a a for full activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>We next tested whether both or only one of the RhaS half-sites contribute to RhaS-CTD activation and whether RhaS-CTD is sufﬁcient to allow CRP to contribute to rhaBAD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>We tested whether it would be possible to obtain increased CRP coactivation at rhaSR in combination with RhaS by screening for RhaS variants with increased activation at ( rhaS-lacZ ) 128 , but without any increase in activation at ( rhaS-lacZ ) 85 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>We investigated the mechanism of CRP activation at rhaSR both alone and in combination with RhaR in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We found that RhaS-CTD allowed signiﬁcant activation by CRP , both in vivo and in vitro , although full-length RhaS allowed somewhat greater CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>We also found that CRP , RhaR , and a-CTD each have synergistic effects on activation by the others , suggesting direct or indirect interactions among all three .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>We have no evidence at this time to argue either for or against a direct interaction between CRP and RhaR ; however , we do have some evidence to suggest that a direct a-CTD -- CRP interaction could contrib-ute to rhaSR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>To distinguish between these two possible roles for CRP , the positive-control ( PC ) gcvA mutant gcvAF31A ( 16 ) was used to separate the activation and repression functions of GcvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Thus , there may be nothing special about the involvement of CRP ; it may simply be that physiology has evolved to demand activation in addition to repression principally for catabolite-sensitive operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Due to the dual action of GcvA , it is possible that CRP interferes with repression by GcvA or facilitates activation by GcvA , since both scenarios can explain the phenotypes of the Dcrp and the Dcrp DgcvA strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>For instance , the apparent repression by PdhR could be due to an interference of activation by another factor such as CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513468" target="_blank">17513468</a>]<br></li>
<li>This was expected , since the CRP requirement for MelR-dependent activation of pmelAB is well established ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>However , the molecular mechanisms underlying the IIA - dependent activation of G - # adenylate cyclase through the C-terminal region and the inhibition of this activation process by CRP remain to be elucidated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>These results are consistent with the locations of various regulatory sites proposed and imply an involvement of CRP and XylR in PA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>This is followed by strong allosteric activation as CRP subsequently fills CRP1 , the lower affinity of the two CRP sites at these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This compares to 19 % of repressible promoters containing CRP sites involved in activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This is unlike other promoters , where tandem CRP target sites lead to a synergistic activation ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Thus , when CRP sites are moved to different proximal positions , the activation is strongest from the gal position , still strong from the lac position , and detectable from the malT position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Might Sxy also enhance CRP activation at sites that do not ﬁt the CRP-S consensus ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Recall that CRP activation sites are mostly restricted to three phased position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Only the latter ` CRP-S ' sites require both CRP and the coregulator Sxy for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>We have identi ® ed its recognition sequence and demonstrated that it corresponds to the previously described inverted repeats surrounding the two CRP boxes involved in ® x activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>CRP is also involved in activation with regulon-speciﬁc proteins at several pairs of divergent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>CRP contributes approximately 50 - to 100-fold activation to each of the rha promoters in vivo ; however , the mechanism of CRP activation has not been identiﬁed for any of these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>It will be interesting to see to what extent the latter is related to CRP function in the activation of other promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Both H159L and H159L K101E CRP mutants signiﬁcantly decreased ( 3.5-fold ) PcspD activity , indicating that the H159 residue is critical for PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>We do not know if this mechanism is also involved in CRP activation of the ilvB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>To further support this conclusion , the AR1 and AR2 CRP substitution derivatives were also tested for activation at F ( rhaB-lacZ ) D84 ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Transcriptional activation by CRP in vivo was also examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Therefore , we also tested whether RhaS-CTD was capable of fulﬁlling the function of RhaS that allows CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>There is also a second component to CRP activation of rhaSR that appears to function through synergism with RhaR and a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Much less is known about how the activation of CRP is affected as well as how this activation will affect cell motility by different carbon source supplies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>We tested alanine substitutions in activating regions ( AR ) 1 and 2 of CRP for their effect on rhaBAD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>this bend is crucial for activation in promot ' ers in which ( IAMP-CRP is the only activator , several lines of evidence indicate that bending may contribute to the activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Transcriptional activation by CRP constitutes an ideal model because the of the system geometry activation is variable and complexes possibly different modes of activation can be studied in the same As a system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>These data suggest that AR2 of CRP is not involved in co-activation of P2bgl by BglJ -- RcsB and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Thus , AR1 , but not AR2 , is required for CRP activation of proP P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Nearly 90 % of operons identi-ﬁed by ROMA showed dependence on the AR1 determinant of CRP for full activation , while AR2 is required for activation of 59 % ( 104 ) of operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Activation of pBAD by CRP is abolished in all of the spacing variants except for the 11 base-pair insert ' ion , which still allows a sixfold activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Studies of the activation of malT by CRP and of ompF by OmpR give clues to how more distant activators can regulate RNA polymer-ase ( 81 , 88 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Similar reasoning may account in part for the lower CRP activation from the upstream malT position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This set of experiments determined the roles of key residues at CRP activation mechanism and aVect on CRP biological function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>This location is non-optimal for CRP activation with Es70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Interestingly though , for synergistic activation of bgl by BglJ -- RcsB and CRP , the CRP AR1 but not the AR2 is important ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Thus , the synergistic activation of bgl by BglJ -- RcsB and CRP apparently involves a novel mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Alternatively , the smaller defect could indicate that the roles of a-CTD and CRP in rhaBAD activation are partially redundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>a pheno-menon that could contribute to the transcriptional activation of the operon by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>What physical interactions lead to transcriptional activation by CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>We further report the results of investigations into the mechanisms of CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>All activation factors are between 7 - and 12-fold lower for H159L than for wt CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Thus , CRP ma somehow inhibit the activation process of adenylate cyclase by phosphorylated IAG - # .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Does the same activating region on the surface of CRP participate in the activation process ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3308">
FIG 4 Role of different functional domains of CRP in PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="act_similarity_3308", type="checkbox">
<div class="ACT_similar">
<li>Role of different functional domains of CRP in PcspE activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_3310">
Effect of CRP with modiﬁed AR1/AR2/AR3 regions on PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="act_similarity_3310", type="checkbox">
<div class="ACT_similar">
<li>Effect of CRP with modiﬁed AR1/AR2 and AR3 on PcspE activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>Carbon source effects on cell motility through activation of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>Does activation by a-CTD involve contacts with CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Interdependence of activation by CRP , a-CTD , and RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>These locations correspond to peaks of activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Positions 83 , 127 and 128 are important but not essential for CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>PrfA PrfA is distinguished from CRP in that it has no known effector for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>CRP is now ,90 ° out of phase compared with all other activation peaks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>CRP activation of ykgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Mechanism of activation by CRP at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Thus , we propose that four subunits of MelR and two subunits of CRP cooperate together to form a higher structure that is needed to-coerce MelR into Site 20 , which is crucial for activation ( illustrated schematically in Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>As shown in Fig. 5 , this change in the helical position of the CRP dimer abolished any contribution by CRP on proP P2 activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Both KdgR and CRP had modest effects on the activation of setA - lacZ and more signiﬁcant effects on sgrS - lacZ under stress conditions ( Fig. 4A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Results showed that K101E mutation in CRP did not affect PcspE activation , while both H159L and H159L K101E alleles adversely affected PcspE activation ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Partial disappearance of the bands in this area could also be Involvement of CRP activating regions AR1 and AR3 in cai and ® x transcription To determine further whether direct contact between CRP and RNA polymerase is involved in transcription activation of both cai and ® x promoters , the effect of amino acid substitutions in CRP was investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Much of our knowledge about transcription activation comes from studies of CRP and CRP homologs that can activate from either class I or class II positions .7 Despite the importance of transcription activation to an understanding of gene expression , little is known at a level of detail comparable to that for CRP family members about the mechanism of activation by other transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866514" target="_blank">11866514</a>]<br></li>
<li>These observations may be explained by a mechanism in which FNR recognizes PhlyE efﬁciently but acts only as a relatively poor activator of hlyE transcription ( 10 , 11 ) , whereas , conversely , the recognition of PhlyE by CRP is poor , but once CRP is bound , CRP-mediated activation of hlyE expression is efﬁcient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>As mentioned above , previous in vivo experiments from this lab did not provide conclusive evidence for or against an interaction between CRP and - CTD ; therefore , we used in vitro transcription assays to more directly measure the role of - CTD in transcription activation at the rhaSR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In particular , the same s residues that affect FNR-dependent transcrip-70 tion activation ( K593 , R596 and K597 ) are also required for transcription activation by the related transcription factor CRP , under conditions that specifically use activating region 3 ( AR3 ) , one of three potential CRP activating regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>In addition , we have demonstrated that CRP can further stimulate rhaSR expression in vitro and have obtained evidence that CRP activation at rhaSR requires contact with - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Tobin and Schleif ﬁrst demonstrated RhaR activation of rhaSR transcription in vitro ( 31 ) ; however , at that time it was not known that CRP also regulates rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In this study , both in vivo and in vitro transcription assay methods demonstrated that CRP site 3 was required for signiﬁcant transcription activation of rhaSR by CRP ( Fig. 2A and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In vivo , we showed that acs activation absolutely requires a functional CRP I , while optimal transcription also requires CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>In vitro transcription results also indicated that CRP activation at rhaSR requires the - CTD of RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>This twofold activation ( which was lost in a strain carrying an ihfB deletion [ 8 ] ) suggests that DNA bending may make a small contribution to CRP activation of rhaSR transcription , but it is at most a small part of the approximately 100-fold activation by CRP at this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>It has also been shown that FNR induces an 82-92 ° bend in target DNA ( Ziegelhoffer and Kiley , 1995 ) but , as with CRP , factor-medi-ated DNA bending may not be essential tbr transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>I ) i i -RCB- -RCB- li i Activation of transcription by IHF In order to assess the contribution of DNA bending to transcriptional activation by CRP , we replaced CRP by the heterologous DNA-bending protein IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>If IHF can activate transcription by bending the DNA it is quite likely that DNA-bending by CRP has the same effect , namely transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Regulation of the hup promoters A. Regulation by CRP P-hupA P2-hupB P3-hupB 0.5 / 0.6 0.2 / 0.2 0.6 0.3 / 0.2 2.0 1.0 ** 1.0 * 0.6 1.0 * 0.5 2.0 wt LB wt LB glu 0.8 / 0.9 1.0 ** 1.0 / 0.9 no effect wt LB crp LB CRP activation B. Regulation by FIS when CRP is absent P-hupA P2-hupB P3-hupB 0.1 / 0.2 0.4 / 0.7 wt LB glu ﬁs LB glu 1.0 * 1.0 ** 0.2 2.7 / 2.5 FIS activation 5.0 FIS repression 2.6 3.7 C. Regulation by FIS when CRP is present P-hupA P2-hupB P3-hupB 0.6 / 0.8 0.9 / 0.7 wt LB ﬁs LB 1.0 * 1.0 ** 0.3 2.2 / 2.5 FIS activation 3.3 FIS repression 2.3 no effect Effect of CRP , glucose and FIS on hupA and hupB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Opposing activation and repression , mediated by CRP and a LacI family member , respectively , is one of the most common ways in which transcription of catabolic genes is regulated in bacteria to meet the cell 's metabolic needs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Fis activates proP P2 transcription predominantly through aII Co-activation with Fis and CRP requires both aCTDs We have previously shown that co-activation of proP P2 transcription by Fis and CRP is dependen on the aCTD .24,29 To determine whether Fis and CRP are each acting independently through different aCTDs , the oriented a-heterodimers were used in transcription reactions containing one or both activators ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>To determine the effect of this region on coactivation with CRP , in vitro transcription reactions were performed with CRP together with Fis mutants that by themselves are strongly defective in P2 activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Single-round in vitro transcription reactions were performed with CRP and Fis mutants containing substitutions in the B-C loop region that is required for P2 activation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The hypothesis of a direct role of CRP sites 1 and 2 in the activation of transcription of cai and ﬁx , respectively , was tested by examining the effect of alterations in the two sites on expression of caiT-lacZ and ﬁxA-lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The three - to ﬁvefold activation of these mutant promoters by CRP can be ascribed to a repositioning of CRP2 from 93 to 71 bp from the transcriptional start , a position from which CRP activates transcription from the WT malT promoter ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The observed negative correlation between acetate formation and transcription of acs -- yjcH -- actP can be explained by CRP dependent activation as different glucose concentrations were present in the feed medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>CRP Repression Is CTD-dependent -- The interaction of CRP with the CTD of RNAP is the basis of transcription activation in wild-type lacP1 and the CRP-independent lacP1 variants mentioned above ( 10 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The more recent ﬁnding that other s70 substitutions at positions near R596 , especially K593A , signiﬁcantly reduce activation by AraC in the absence of CRP supports the hypothesis that wild-type AraC and s70 make an interaction that contributes to transcription activation ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>The CRP AR1 mutants were found to strongly reduce activation , while the AR2 mutants had no effect on activation of transcription at proP P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The araC gene is regulated by CRP activation of transcription and by autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>The double-headed arrow means synergistic transcription activation by MhpR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>a The goal of this work was to characterize rhaBAD transcription activation by CRP and a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>The in vivo and in vitro transcription assays and the DNase footprinting assays show little or no role for CRP site 2 in rhaSR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The above results identify CRP site 3 as the most important site for activation of rhaSR transcription , but they do not provide information about the mechanism of activation by CRP bound at site 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The orientation in which MelR binds to site 2 and site 2 cents has important consequences , as it deﬁnes the surfaces of MelR that can make contact with the RNA polymerase s subunit and CRP during transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>- C TD substitutions that affect acs transcription Substitutions that increased acs transcription - CTD Substitutions that decreased acs transcription Substitutions adjacent to known determinants determinant T285A , V287A , E288A D259A , E261A L260A , L262A ( both increased transcription ) P293A , K297A ( decreased transcription ) ; T301 ( increased transcription ) N268A , N294A , G296A , K298A , S299A R265A C269A , I275A , Y277A , I278A , D280A , L281A , Q283A , V306A Other residues I303A DISCUSSION Previously , we demonstrated that CRP plays a critical role in the activation of acs transcription ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>However , we propose that , although NarP and NarL bind as dimers , transcription activation requires activity in just one of the subunits of the dimer ( this has been shown for both CRP and FNR : Zhou etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Although CRP sites appear from different distances upstream to cognate promoters , maximal activation is obtained when CRP is located at bp 261.5 or 241.5 upstream of the transcription start site ( reference 25 and citations therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Site-directed mutagenesis of CRP sites 1 and 2 clearly demonstrated that site 1 is required to activate only transcription of the cai operon whereas site 2 only plays a role in the activation of the ﬁx operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>One of the most intriguing features of transcriptional activation by CRP is the observation that CRP can activate transcription when placed at many different positions upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>CRP site 3 therefore emerges as the critical CRP site for rhaSR transcription activation , consistent with our previous , less comprehensive , results obtained using multicopy fusions ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>DISCUSSION CRP site 3 is critical for CRP activation of rhaSR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Therefore , even though CRP and FNR share the same structural framework , there are major differences in the way they interact with CZCTD , such that different contacts are used to modulate the extent of transcription activation at specific promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A variation of CRP-dependent activation , designated class III , utilizes two or more CRP dimers or a combination of CRP and other activators to achieve maximal transcription activation ( reviewed in references 5 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>CRP-dependent activation requires activating region I. Given the location of CRP I ( 69.5 ) and CRP II ( 122.5 ) , we predicted that CRP activates acs transcription by class I interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>indicating strongly that CRP sites I and II are responsible for inhibition and activation of crp transcription , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>However , transcription of pap is independent of CRP activation in the E. coli K-12 mutant lacking H-NS ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Expression of the E. coli K-12 melAB genes , which encode products essential for melibiose metabolism , depends on transcription activation by MelR and CRP , and is triggered by melibiose ( 7,9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3369">
Stringent spacing requirements for t ' ranscriptional activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1324322" target="_blank">1324322</a>]<br></label>
<input id="act_similarity_3369", type="checkbox">
<div class="ACT_similar">
<li>Stringent spacing requirements for transcriptional activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Stringent spacing requirements for transcription activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1735706" target="_blank">1735706</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768815" target="_blank">7768815</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_3372">
Gaston , K. , Bell , A , , Kolb , A. , Buc , H. & Busby , S. ( 1990 ) Stringent spacing requirements for transcription activation by CRP , Cell 62 , 733-743 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></label>
<input id="act_similarity_3372", type="checkbox">
<div class="ACT_similar">
<li>Gaston , K. , Bell , A. , Kolb , A. , Buc , H. and Busby , S. ( 1990 ) Stringent spacing requirements for transcription activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Gaston , K. , Bell , A. , Kolb , A. , Buc , H. , and Busby , S. ( 1990 ) Stringent spacing requirements for transcription activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li></div>
<li>A peculiar synergistic transcription activation of the mhp genes by the specific MhpR activator and the CRP global regulator is described .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>This tightly regulated system uses two transcription factors for its activation , MhpR and CRP , and maximizes the efficiency of activation by using two synergistic effectors for the same transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>This is fully consistent with strong activation of dctA transcription by CRP and weak induction in response to C4-dicarboxylates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>This indicates that the other tdc regulators , TdcA and TdcR , govern anaerobic transcription activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Plasmid DNAs were incubated with MalT and CRP , both present at concentrations that were optimal for activation of transcription , and then treated with dimethylsulphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Finally , we demonstrated the ability of RhaS-CTD to activate transcription in vitro and further that CRP was capable of signiﬁcant in vitro activation in combination with RhaS-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>CRP activation of rhaSR transcription is dependent on - CTD in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We therefore investigated the importance of the - CTD for CRP and RhaR activation of rhaSR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>CRP activation of rhaSR transcription requires - CTD and RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Taken together , we propose that CRP regulates transcription activation of the rmf gene for formation of 100S ribosome dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>CRP at 43.5 position exhibits only weak activation , but CRP bound at +17.5 causes no inﬂu-ence on malI transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>These data further support the notion that CRP is required for activation of bgl transcription by BglJ -- RcsB , and they demonstrate that the AR1 of CRP is important for this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Both BglJ -- RcsB and CRP are required for transcription activation of the newly identiﬁed P2bgl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Therefore , stimulation of CRP transcription activation was examined by perturbing these residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>MhpR shows a synergistic transcription activation mechanism with CRP ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>In of CRP-mediated control since the presence of the CAMP-theCRP2-CytRcomplex all proteinDNA-phosphate CRP complex is absolutely required for both activation and contactsatCRP-1andCRP-2areretained , andin repression of initiation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Most of the in vivo properties malKp promoters can be explained by this model if we assumethat the integrity of these nucleoprotein structures , i.e. the anchorage points on both sides and the correct wrapping of the DNA , is essential for transcription activation by this complex and if we assume that the complex involving both MalT and CRP is the most active species.Deletions of the distal MalT boxes destroy the anchorage points .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3392">
A common role of CRP in transcription activation : CRP acts transiently to stimulate events leading to open complex formation at a diverse set of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054813" target="_blank">12054813</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17504942" target="_blank">17504942</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></label>
<input id="act_similarity_3392", type="checkbox">
<div class="ACT_similar">
<li>Tagami , H. , and Aiba , H. ( 1998 ) A common role of CRP in transcription activation : CRP acts transiently to stimulate events leading to open complex formation at a diverse set of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207706" target="_blank">12207706</a>]<br></li>
<li>Tagami , H. & Aiba , H. ( 1998 ) A common role of CRP in transcription activation : CRP acts transiently to stimulate events leading to open complex formation at a diverse set of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li></div>
<li>End-labelled pacs AatII -- HindIII fragment was incubated with increasing concentrations of IHF protein in the absence or presence of a constant amount of CRP or FIS proteins and subjected to DNase I footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>A ﬂuorescently labeled SELEX segment ( 1.0 pmol ) from the ascG-ascF spacer ( A ) or the prpR-prpB spacer ( B ) was incubated in the absence or presence of increasing concentrations of puriﬁed AscG ( 2.5 , 5.0 , 10 , and 20 pmol , from right to left ) or CRP ( 0.3 , 0.6 , 1.25 , and 2.5 pmol , from right to left ) and then subjected to DNase I footprinting assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Transcription reactions contained : RNAP alone ( lane 1 ) ; RNAP with increasing concentrations of CRP ( lanes 2 -- 5 ) ; RNAP with increasing concentrations of FIS ( lanes 6 -- 9 ) ; and RNAP , FIS and increasing concentrations of CRP ( lanes 10 -- 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3398">
Reactions contained puriﬁed RNA polymerase , puriﬁed CRP and increasing concentrations of puriﬁed IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_3398", type="checkbox">
<div class="ACT_similar">
<li>Reactions contained puriﬁed RNA polymerase , puriﬁed CRP and increasing concentrations of puriﬁed FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Reactions contained puriﬁed RNA polymerase , puriﬁed CRP , puriﬁed IHF and increasing concentrations of puriﬁed FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>End-labelled pacs444 AatII -- HindIII fragment was incubated with increasing concentrations of CRP and FIS proteins and subjected to DNase I footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3402">
An end-labelled FIS III fragment was incubated with increasing concentrations of puriﬁed FIS , CRP or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_3402", type="checkbox">
<div class="ACT_similar">
<li>An end-labelled FIS II + III fragment was incubated with increasing concentrations of puriﬁed FIS , CRP or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>However , because the superhelical density of DNA increases after the nutritional shift-up and CRP might be involved in this increase by activating gyrA ( 47 , 48 ) , it is conceivable that this indirect effect would contribute to the rapid attainment of maximal fis expression levels in wild type cells ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>While the CRP site 2 down mutation had a small effect on rhaSR expression in vivo ( 66 % of wild-type activity [ Fig. 2A ] ) that was dependent on CRP , we predict that this defect is no due to decreased CRP binding to site 2 but more likely is due to an indirect effect on CRP bound at site 3 , such as a change in DNA bending or a change in the ability of - CTD to contact CRP at site 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>When the deletion extends into the CRP binding site II , the crp expression was significantly reduced even in the absence of glucose and the repressive effect of glucose on crp expression was almost eliminated ( Figure 8A , lanes 5-8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>While the substitutions changing CRP site 1 to the consensus sequence at the 10 most conserved positions resulted in a small increase in rhaBAD expression , a similar increase in expression was observed in the absence of CRP , suggesting that the increase in expression may be due to the DNA sequence change and not a change in CRP occupancy of the site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Second , the importance of CRP increased in the absence of a-CTD ( compare the defects upon deleting the CRP-bind-ing site with wild-type a , 38-fold , and with expression of a-D235 , 162-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>To do that , we mutated the core promoter of CC-55 .5 to get CC-55 .5 - M ( Fig. 1 ) , which has a lower afﬁnity for Es54 than CC ¬ 55.5 ( data not shown ) , and investigated the effect of CRP - mediated DNA bending on CC-55 .5 - M. Figure H159L 4b shows that the in vivo expression of CC-55 .5 - M in the absence of CRP decreased to about 27 % of native H159L glnAp2 , whereas CRP increased the expression of CC-H159L 55.5-M up to 6-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Previous in vivo experiments also indicate that CRP is not capable of activating rhaBAD expression in the absence of RhaS ( 11 ) , perhaps suggesting that RhaS must bend the DNA to allow CRP to activate , although other possibilities exist as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>For this purpose , I constructed malEp variants containing point mutations destroying the three CRP sites ( malEpKp1 , -2 , and -3 ) and mutations in either sites 3 and 4 , sites 3 ' and 4 ' , or sites 3 , 4 , 3 ' and 4 ' , and assayed them in vivo in the presence of glycerol and maltose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In this system , malEp and malKp are active only in the presence of both MalT and CRP , and various mutations in the MalT or CRP binding sites affect the promoters in the same way as they do in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>( ( a ) malEp and malKp function in vitro as in vivo To ensure tha.t our in oitro system really mimics events occ * urring in , z ' iz > o , we examined whether malflp and malKp activity depended on the pre-sence of the distal MalT binding sites and on the presence of the same set of CRP binding sites as in Gvo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>To determine whether the higher efﬁciency of the 3 ' / 4 ' / 5 ' set of sites in promoting assembly of a higher-order structure depended on the presence of CRP , I measured the effect of the distal sets of MalT sites on malEp in the absence of functional CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The EcoRI-EcoRI DNA fragments of were transferred in both orientations in front of the mulPQ maZEp and of malKp promoter was then measured by assaying ( d ) DNase I footprint of MalT bound to malEpmalKp ( c ) Effect of deletions and insertions in the absence of CRP To determine whether the involvement of the distal MalT binding sites depended on the presence of CRP and to study their role in the DNA-turn dependence , we examined the effect of deletions and insertions on promoter activity in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>The hypothesis of a direct role for CRP site 1 was tested by examining whether occupation of MalT sites 1 and 2 in the presence of CRP site 1 increased malEp activity to a higher level than that observed with malEpD64 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The fact that in the malPQ − overproducing strain , malEpD92 was almost as active as malEp + in pop2226 for a similar occupancy of MalT sites 1 and 2 , indicates that ﬁlling MalT sites 1 and 2 in the presence of CRP site 1 efﬁciently restores promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The fact that the insertions have the same qualitative effect on malKp activity whether or not CRP is present suggests that in the presence of CRP the distal MalT binding sites have also to be stereoaligned with some downstream regulatory elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>The observations that the 3 ' / 4 ' / 5 ' set of sites has a stronger stimulatory effect on malEp than the 3/4/5 sites regardless of the presence of functional CRP sites and that CRP similarly enhances the stimulatory effect of both sets of MalT sites , indicates that the sensitivity to the phasing is actually essentially determined by MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>However , the fact that in pop2226 , the malEpKp1 ,2,3 , I3 variant is only 29 % as active as the wild-type promoter , while displaying similar occupation of MalT sites 1 and 2 , suggests that complete occupation of MalT sites 1 and 2 in the presence of the distal MalT sites is not sufﬁcient to restore malEp activity fully and that one of the CRP sites also plays a direct role in promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Even when the defect in MalT synthesis in the absence of CRP was counteracted the of by presence a mutation in the malTp promoter that renders its activity CRP-independent , some problems persisted that impeded any quantitative assessment of the role played by CRP in the activation of the promoters of the three mal operons , malPQ , malEFG , and malK-lamB-malM ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>It is possible that these residues are also important for RhaS activation in the full-length promoter , but for reasons described in the Discussion , such as redundancy , they did not show any detectable defect in the presence of CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Crosses symbolize the malEpKp1 , -2 and -3 mutations , which correspond to 3 bp substitutions and which inactivate the CRP sites 1 , 2 and 3 , respectively ( Vidal-Ingigliardi & Raibaud , 1991 ) Occupation of MalT sites 1 and 2 requires the presence of upstream regulatory elements As mentioned above , the truncated malEpD64 promoter , which comprises only MalT sites 1 and 2 , is inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Mutations inactivating either MalT site 4 or the CRP site -LCB- malPp40 and malPpSO ) had no effect at all on the uninduced or induced activity of the promoter , clearly demonstrating the absence of any functional role for these two sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7830577" target="_blank">7830577</a>]<br></li>
<li>Moreover , inactivation of both distal sets of MalT sites together with the three CRP sites abolishes promoter activity as efﬁciently as deletion malEpD64 ( Figure 2 ( b ) ) , thereby indicating that , in the absence of functional CRP sites , malEp activity is fully dependent on the presence of either set of distal MalT sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Cooperative Binding to deoP2 To determine the ability of the different repressor proteins to-bind cooperatively with CAMP-CRP to deoP2 , gel retardation was performed in the presence of CAMP-CRP ( Figure 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>We propose that the presence of CRP bound at CRP II contributes to the recruitment of the RNAP to the promoter through an AR1 -- a-CTD interaction ( see below ) , causing enhanced transcription as expected for a Class III promoter composed of tandem Class I sites ( Fig. 7 , compare panels A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Second , with IHF bound at its speciﬁc sites , additional protection was observed between the two CRP sites , but only in the presence of the intact AR2 surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , in the presence of both RNAP and CRP , IHF does mediate increased protection of the region between the two CRP sites ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>It is associated with the occupancy of both CRP sites by WT CRP and , because of the presence of additional upstream elements ( e.g. CRP-bound CRP II ) , with a more stable RNAP -- DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>S1 and data not shown ) to determine whether the presence of IHF bound at IHF III and CRP bound to CRP II can affect the binding of RNAP and the structure of the open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>While we did not observe a substantial difference in the overall affinity of RNAP for acsP2 , or of CRP for its sites , we can not exclude the possibility that the presence of IHF exclusively at IHF III and the resulting changes in DNA structure might affect the on-and-off rates of binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , in the presence of meAda , the transcriptional-activator responsible for alkylation-damage induction of aid6 the levels of Eos-dependenttranscription were significantly increased , indicating that meAdacan activate transcription by both Eo7 ' and EnS.Interaction of a transcriptional-activator with both forms of RNA polymerase has already been describedfor the CAMPreceptorprotein ( CRP ) at boththe gatP I and the Iac promoters ( Kolb eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Interestingly , the inactivation of AR1 by the H159L substitution does not impair the ability of CRP to activate transcription from the wild-type promoter under inducing conditions ( i.e. , in the presence of fucose ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>As expected , the activity of this promoter ( XX-CC ) in the presence of wild-type CRP was less , due to the absence of enhanced transcription via CRP binding upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Figure 6B shows that in the presence of wild-type CRP ( pZYCRP1 ) and a functional CRP binding site , transcription at proP P2 was stimulated 2.5-fold in vivo when the cells were grown in Luria broth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In the absence of a functional crp gene or in the presence of a CRP binding-site mutation , P1 transcription is elevated throughout the growth cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>In vivo analyses revealed that moving the CRP binding site to 241.5 bp increased anaerobic transcription in the presence of glucose to levels seen without glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>In vivo , the presence of IHF III strongly reduces CRP-dependent transcription , sug gesting that the binding of IHF to this site interferes with the ability of CRP to activate transcription ( 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Down mutations in CRP site 3 reduced transcription levels of pSR to approximately 50 % of the wild-type levels in the presence of CRP alone as well as in the presence of CRP and RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We then wanted to conﬁrm that in the presence of a functional CRP site 1 , CRPK52N exerted its constitutive effect on lacZ transcription independently of the fucose-speciﬁc regulator , FucR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>B $ 1 nuclease mapping of the start site of transcription in deoP2-wt and deoP2-36-89-98 in the presence of different CRP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>This was observed both in the presence and in the absence of CRP , which is consistent with our ﬁnding that transcription initiation by Es28 is independent of CRP in vitro at the high RNA polymerase concentrations used in these reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Furthermore , potassium permanganate probing shows that the conformation of the open complex in the presence of CRP appears qualitatively and quantitatively different from that in the absence of CRP , suggesting that contact with RNA polymerase is maintained throughout the transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>When the amount of CRP added was reduced to l / 10 , which was equivalent to -20 CRP molecules/DNA fragment , the in vitro transcription was not completely inhibited even in the presence of CAMP ( fig. 1 B ( 4 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>Primer extension In vitro transcription was performed using the pRPGSM DNA template in the presence of 50 nM CRP protein as described above , but using non-radioactive UTP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In the presence of CRP , transcription proceeds from a site slightly further upstream , which places the operator in a position where it reaches -45 and the polymerase ; this repression might then proceed in the conventional manner as discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>In both cases , transcription repression was observed in the presence of high concentrations of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In contrast , the presence of CRP favoured TOC formation at acsP2 and inhibited TOC formation at acsP2A and , as a result , activated transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In vitro transcription from the rhaSR promoter in the presence ( ) and absence ( ) of CRP , RhaR , and L-rhamnose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In the presence of chitobiose , ChbR , along with CRP , activates transcription from the chb promoter ( Plum-bridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>In the presence of glucose ( or in the absence of CRP ) , the other two regulators have little transcription to regulate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>In both cases , mutations of CRP sites had no effect on the corresponding distal promoters ; i.e. , the level of ﬁx transcription remained unaffected in the absence of CRP ﬁxation on site 1 , and cai was normally expressed in the presence of a mutated site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>In vitro run-off transcription assays have revealed no significant transcription initiation in either direction from this site , either in the presence or absence of MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>In the presence of wild-type CRP , the V287 mutation decreased PcspE transcription by the maximum level ( 64 % ) ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>However , it showed significant reduction of transcription in the presence of CRP ( Fig. 1A , lanes 13 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Maximal transcription from pacs236 crp II ( which includes CRP I and IHF III , but not CRP II ) depended upon the presence of both IHF and AR2 , as well as the inability of CRP to-bind at CRP II ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>As expected , Fis still activated transcription on this template in both the presence and absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>proP P2 transcription was activated approximately 70-fold in the presence of both Fis and CRP ( Figure 6 , lanes 4 and 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Our in vitro transcription assay results indicate that CRP can activate transcription only from the `` normal '' pSR transcription start point in the presence of RhaR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Figure 3 shows the effects on in vitro transcription of different concentrations of potassium glutamate in the presence of Fis and CRP , both as solitary activators and in combination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>B , in vitro transcription from supercoiled pWN1 ( 0.5 mg ) in the presence of RNAP ( 10 nM ) and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>At lacP1-6 , no distortion of lacP2 was detected in the absence or presence of CRP ; DNA was distorted with polymer-ase alone at position 11 , which represents the lacP1 locus ; the presence of CRP slightly stimulated DNA distortion at this site ( Fig. 3 , lanes 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Results presented in Fig. 5A show that , in the presence of CRP and Es28 ( lane 3 ) , DNA cleavage on the template strand of the aer promoter is enhanced around positions -72 and -64 upstream of the DNA site for CRP , and around positions -38 and -30 downstream of the CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Addition of CRP in the presence of MelR173 causes protection of DNA between sites 1 and 2 and increases the protection at sites 1 and 2 , but does not extend the footprint at site 1 ′ or 2 ′ ( Fig. 3 , lane b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Plasmids pJG24 , JG14 , and S 70 JG18 ( which carry the CRP box at 260.5 , 268.5 and 274.5 , respectively , upstream of the csiD transcriptional start site ) were incubated with either Es or Es in the presence or absence of CRP S 70 as detailed in Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Therefore , a comparison of the effects of 3 - and 6-base-pair substitution and deletion mutations demonstrates that spacing changes in the CRP site did not affect araBAD expression to any greater extent than did substitution mutations in the presence of L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>DNase I footprinting was carried out in the presence of various concentrations of CRP ( Fig. 2a ) and th gated by site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>Although the presence of both core motifs was found to be necessary for CRP to-bind to the target site in PcspE ( Fig. 3A ) , an intact second core motif was shown to be more important for CRP-PcspE interaction ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Transcription was reduced in the presence of CRP containing HL159 and this reduction was suppressed by KN52 confirming that AR1 is required at the downstream site ( Table 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Additionally , the boundaries of the RhaR and CRP footprints did not change when both proteins were present compared with their individual footprints , which argues against the possibility that CRP binds to site 2 in the presence of RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Spacing changes in the CRP site would thus not be predicted to directly alter loop formation in the presence of L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>EMSA , using a 274-bp fragment ( 274 to 1 ) from the cspD upstream region and puriﬁed CRP protein , revealed binding of CRP to this region , as evidenced by an additional slow moving band in the presence of CRP ( Fig. 2A , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>EMSA using a 312-base fragment ( 300 to 12 ) from the cspE upstream region and puriﬁed CRP revealed the binding of CRP to this region as evident by an additional , slow-moving band in the presence of CRP ( Fig. 2A , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Additionally , although CRP is unable to-bind to the melAB promoter in the absence of MelR , in the presence of MelR , it binds to a site located between MelR binding Received 19 November , 1999 ; revised 21 January , 2000 ; accepted 24 January , 2000 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>However , binding of CRP to thseecond site inthe gal operon requires the presence of RNA polymerase , while CRP binds to both sites in the glpFK operon in the absence of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Nevertheless , the conservation of the CRP binding site , the repression of McaS in the presence of glucose and the DNase I footprinting data provide evidence for direct CRP regulation of McaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>When cells were shocked with 0.6 M NaCl , levels of induction of ProP-LacZ activity were similar regardless of the presence of the CRP binding-site mutation ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Because P1 expression can be further induced by osmotic upshifts in a Dcrp strain or in the presence of the CRP binding-site mutation , additional controls exist to osmotically regulate P1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>This difference might be related to the presence of a putative CRP binding site in the 5 ' flanking region of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2465204" target="_blank">2465204</a>]<br></li>
<li>In the presence of CRP , binding of CRP to the P0 site is increased by P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>In the presence of CRP ( lanes 4 and 5 ) the longer fragment yields several-fold more transcripts than the fragment truncated at -93 , just upstream of the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Likewise , when the CRP site was mutated to prevent binding ( 58 ) , little coactivation was obtained in the presence of Fis ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>CRP binding site To determine the CRP binding site , a DNase I protection experiment was carried out in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>Each mutant was analyzed in the presence ( crp ) and absence ( crp ) of CRP to distinguish the effects on CRP binding from other effects of the DNA sequence changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Moreover , they indicate a co-operative binding of CaiF and CRP in the F1/CRP2 and F2/CRP3 regions , as the presence of either regulator at saturating concentration strongly stimulates the binding of the other ( Figs 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Whether the destabilization is due to the bend introduced by the binding of CRP or t , he mere presence of the protein was not ) determined by the experiments reported hen , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>These modifications of the ( OP ) , Cu + pattern seen in the presence of H1 were further analyzed in the Pribnow box and in the CRP binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>F , binding in the presence of 30 nM CRP and 50 g/ml heparin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3490">
B , binding in the presence of 30 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></label>
<input id="act_similarity_3490", type="checkbox">
<div class="ACT_similar">
<li>D , binding in the presence of 30 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li></div>
<li>A more interesting observation is that 10 mM KC1 also inhibits CRP binding to site 2 even in the presence of glycerol ( fig. 5 , lanes 1 , 2 , 3 ) , while the interactions of CRP with CRP site 1 are unaffected by the increased salt concentration ( fig. 5 , lanes 1 , 2 , 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>However , Fig. 4 lanes d and e show that , in the presence of MelR , CRP affords clear protection of the DNA between MelR binding Site 1 and Site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Interestingly , in the presence of melibiose , the recruitment of CRP results in increased protection of MelR binding site 20 ( Fig. 4 , compare lanes e and h ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>However , consistent with DNase I footprinting data in the presence of CRP , the reactivity signatures attributable to FIS binding at site II were substantially increased ( Fig. 6 , A and B , compare lanes 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>In Bacteria , an abrupt slope in the NWD curve reveals the presence of a handful of high-scoring binding sites for highly speciﬁc TFs ( e.g. TrpR , LexA ) , whereas a progressive increase of the NWD slope is indicative of global factors , such as CRP , FNR ( Figure 3 ) , Fur , FruR , IHF and FIS ( Supplementary Data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>In contrast , IHF at this higher concentration does not inhibit the binding of RNAP in the presence of CRP , even when the transcription factor is present at the lower concentration ( 12 nM ) tested ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , alternative explanations for the effect of CRP at diverse promoters suggest that CRP acts at a recruitment step essentially to stimulate the formation of transcriptionally competent complexe and to inhibit the appearance of unproductive initiation complexes .30 This concurs well with our observations , wherein RNAP forms upstream contacts either with DNA in the absence of FIS , or with the protein in the presence of FIS ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054813" target="_blank">12054813</a>]<br></li>
<li>In our case , the dependence of the fucPIK promoter on a speciﬁc complex of regulatory proteins for its activity can apparently be overcome in the presence of a mutant CRP protein with three , rather than two , activating regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>The ansB 5 ' - untrans-lated sequence required for regulation by FNR protein and CRP , as defined by deletion analysis , was examined for the presence of the regulatory elements described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>lacP1 mutants , lacPs , lacP uv5 , and lacP1-6 , have been reported that are active in the absence of CRP but are also further stimulated by the regulatory protein ( 26 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>In this paper , we have examined the ability of the puri - ® ed CaiF protein to-bind to the cai ± ® x regulatory region alone or in the presence of CRP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>It also requires the presence of the CRP activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>, I FIGURE 7 : Schemeillustrating DNaseI attack of the CRP site in the presence of protein on a double helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>CRP and araC protein in the presence of arabinose protected from - 116 to -37 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6319708" target="_blank">6319708</a>]<br></li>
<li>deoP2 in the presence of CRP and further to do so with only moderate reduction in overall affinity ( 23 ) compared with the full-length protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>RNA isolated from S ~ 2929 containing pJEL134-wt in absence of CRP protein or in presence of CRP-wt , CRP181EL or CRP181EV was used in the reactions in lane 1 , 2 , 3 and 4 , respectively ; RNA from $ 5252929 containing pJEL134-36-89-98 in the absence of CRP protein or in the presence of CRP-wt , CRP181EL or CRP181EV was used in the reactions in lane 5 , 6 , 7 and 8 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>'' and indicate the absence and presence , respectively , of 250 nM purified CRP and 1 g of total protein of MhpR extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>In the absence of CRP no 125 base product is made even in the presence of maximal AraC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>In the presence of Es28 , but in the absence of CRP ( lane 2 ) , the pattern of DNA cleavage is similar to the background detected in the absence of any protein ( lane 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>For example , the presence of fructose in our growth conditions meant that the fru operon was identiﬁed as CRP regulated in our study , but not regulated in the Gosset study due to the lack of fructose in LB media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>It is impossible to include all inducers in the growth media and , moreover , the presence of extra inducers may trigger expression of other genes that are independent on CRP , furthe complicating the interpretation of the data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>H159L As control , the expression of native glnAp2 was not affected by the presence of either CRP or IHF ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Tn the presence of cyclic AMP-CRP , a complex i formed that contains CRP and AraC bound to the DNA ; AraC dissociates from this complex with a half-time of 2.5 minutes , resulting in a complex that contains only CRP bound to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>However , because the comparative assays reported here were performed in the presence of a saturating concentration of MalT , a differential effect of CRP might have been overlooked , and the possibility remains that CRP may also increase the bias toward the 1/2 -3 ' / 4 ' / 5 ' looped structures via an increase in the torsional rigidity of the DNA in the central region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In the presence of both MalT and CRP , CRP molecules bind to their recognition sites , replacing several of the MalT monomers which nonspecifically contacted the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>In contrast to what was seen with the supercoiled template , the tdcA transcript was seen only in the presence of His-TcbC and CRP with the relaxed , circular DNA template ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>In particular , positions -51 and -52 , which are hyperreactive in free DNA , are no longer very reactive in the presence of CRP ( Figure 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>In contrast , the additional presence of CRP allowed formation of a band which migrated more slowly than the DNA-CRP band in the gel retardation assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>One key to this apparent paradox was provided by the hydroxyl radical footprints that indicate that CytRA9-48 may contact the DNA backbone in the presence of CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Thus , the presence of CRP shifts the dissociation rate of AraC from aral on minicircles to a value characteristic of unlooped DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Within the protected region are DNA fragments , whose rate of production is unaffected or in the presence of CRP even enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>In the presence of RhaS , CRP activates rhaBAD an additional 30 - to 50-fold ; however , CRP is unable to activate to any signiﬁcant extent in the absence of RhaS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>In the absence of added sugars , CRP indeed appeared to be an activator of sfm both in the presence and in the absence of H-NS , as shown by decreased sfm expression in crp mutants ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>In addition , the mutation also results in elevated gcvA-lacZ expression in the presence and absence of CRP ( Table 6 ) and presumably of the gcvA gene itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In the presence of CRP , however , RNAP transcribed from P2 quite efﬁciently ( Fig. 4A , lanes 2 to 5 ) , producing a 139-nucle-otide band with no change in the intensity of the RNA 1 control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>In the presence of a constant CRP concentration , increasing the concentration of FIS resulted in the disappearance of the CRP -- DNA complex and the appearance of a FIS -- DNA complex ( Fig. 4B , lanes 7 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In the presence of a constant CRP concentration , increasing the FIS concentration resulted in the appearance of a high-molecular-mass complex , the mobility of which corresponds to a complex containing both FIS and CRP , and the disappearance of the complex containing only CRP ( Fig. 4C , lane 7 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In the presence of CRP , RNAP binds with similar affinity to acsP2 , and no longer forms an open complex at acsP2A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In contrast , the combined presence of IHF and 12 nM CRP resulted in the formation of virtually no open complex at acsP2A ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In the top strand ( Fig. 9B ) , compared with the control ( Fig. 9B , lane 5 ) , the thymine residues at 36 , 35 , 34 , 28 , 8 , 7 , 3 , 3 , and 4 exhibited hyperreactivity to permanganate either in the presence ( Fig. 9B , lane 7 ) or the absence of CRP ( Fig. 9B , lane 6 ) , but the stops at these positions are stronger in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>In addition , CRP increased the apparent first order rate constant of formation of the initiating complex by a factor of 3 ( t1y2 5 1.5 6 0.2 min in the presence of CRP vs. t1y2 5 4.4 6 0.4 min in its absence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>In Fig. 4b we compare the kinetics of formation of this RPinit , deduced from the 112 signal , in the presence or absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>In the presence of CRP , A 0.052 ( Fig. 8B ) ; in the absence of CRP , A 0.041 ( Fig. 8A ) , indicating more open complex available in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>In the presence of CRP , MalT is repositioned to three adjacent sites of lower affinity and is thereby properly aligned to activate malK expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In vitro ( lane 1 ; without the presence of CRP ) and in vivo ( lane 2 ) DMS footprinting experiments were performed as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>In addition , the presence of a CRP effect in an hns mutant background also suggests than CRP acts independently of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>In addition , bands at 29 , 27 , 22 , 21 , 15 , 12 , 10 , and 5 occurred specifically or much more strongly in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>In the presence of CytRA9-48 together with CAMP-CRP , a very similar cutting pattern was observed ( Figures 5A and 5B , lanes 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In some of these proteins the presence of extensive sequence similarity has allowed specific families with common ancestries and similar modes of action to be recognised , e.g. the XCro , LysR and CRP families [ 2-5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2060763" target="_blank">2060763</a>]<br></li>
<li>In a similar fashion , the 6-fold change in the value of 4AB reflected an increased affinity of the enzyme for UTP at low ApU concentration ; 4 ) A was less affected by the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>In the presence of the KE101 mutant or at low CRP concentrations ( 12 nM ) , the band at -1 of acsP2 increases in intensity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In this case , low b-galactosidase activities were obtained regardless of the presence of the CRP mutation ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>In either case , the presence of low-affinity sites for both IHF and CRP raises a much larger issue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>'' Lanes 1 -- 4 , the sequences of the lacP1 promoter ; lanes 5 , 8 , 11 , 14 , and 17 , KMnO treatment in the absence of RNAP ; lanes 6 , 9 , 12 , 15 , and 18 , 4 KMnO treatment in the presence of RNAP ; lanes 7 , 10 , 13 , 16 , and 4 19 , KMnO4 treatment in the presence of RNAP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>This result demonstrates that CRP has the ability to induce the expression at the Pr promoter in vitro and this can occur independently of the presence of the Pa promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Electrophoretic mobility shift assays Fluorescent-labelled promoter fragments were prepared by standard PCR using a 5 cents Hex-labelled primer and incubated with varying amounts of puriﬁed FIS in the absence or presence of a constant concentration of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Incubation with Es70 did not result in promoter melting around the aer transcript start site , either in the presence or in the absence of CRP ( Fig. 3B , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>After RNAP was cleared from the promoter , a time-dependent decrease in fluorescence was observed either in the presence or the absence of CRP ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>As we observed similar behaviour with pacs444 in either the absence or presence of IHF ( Fig. 5 ) , this effect of CRP at the acs promoter appears to be general .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In the presence of CRP , this promoter behaves identically to malTp1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>In the presence of CRP , the dissociation constant reduces about 4-fold to 1.5 10 7 M. Clearly , CRP strongly augments the recruitment step , and in its presence the dissociation constant becomes comparable with that of a strong promoter like - PR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Therefore , the presence of a CRP motif at position 241.5 in the promoter of caiF strongly suggests that it is of physiological signiﬁcance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>We conclude that the malT promoter is limited at the escape step , even in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>We determined the effect of mutations in gaiR , galS , and crp ( which encodes CRP ) on galR and galS promoter activity in the presence and absence of inducers ( D-galactose and D-fucose ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Panel A , competitive binding of PaaX and CRP to the pga promoter according to the presence or absence of PAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Expression from the melAB promoter in the absence or presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>DNase I protection of the ompA promoter in the absence and presence of CRP and CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>Interestingly , CRP repression at the promoter was dependent upon the presence of the 35 element ; mutation of the 35 element abolished CRP repression at the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>( Unlike the native crp promoter , the bla promoter is not subject to feedback regulation by CRP or to downregulation in the presence of glucose [ 36 , 54 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>This is surprising since one of the two major promoters of hupB , the proximal P2 promoter , is insensitive to the presence of CRP ( Figures 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Alternatively , in the presence of CRP the total number of interactions at this promoter may be large enough that the loss of any one interaction does not result in a signiﬁcant defect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>At lacP1-6M ( 35 ) , formation of open complex was very weak in the absence of CRP , which is consistent with the electrophoresis mobility assay data of this template , but was stimulated in the presence of the regulatory protein ( Fig. 3 , lanes 11 -- 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Formation of open complex was very strong at the extended 10 promoters lacP1-6S1 and lacP1-6S1M ( 35 ) in the absence of CRP ; slight increases were observed in the presence of CRP ( Fig. 3 , lanes 14 -- 16 and lanes 17 -- 19 , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Likewise , open-complex formation at P1 in vivo is inhibited by the presence of CRP under low-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Because the CRP -- RNAP complex can not compete with preformed TOC at acsP2A , the presence of 12 nM CRP only partially inhibited TOC formation at acsP2A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , a great deal of stabilization of the open complex occurs in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>This concentration of CRP suffices to stabilize the open complex in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Blue line , in the absence of CRP ; red line , in the presence of CRP ; R , RNAP ; P , promoter DNA ; ( R.P ) ‡ , the transition state ; ( R.P ) C , closed complex ; ( R.P ) O , open complex ; ( R.P ) i , initiating complex ; and ( R.P ) e , elongating complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>About three times as much initiating complex was formed in the presence of CRP than in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>KMnO probing of the promoters open complex forma-IG 4 tion by RNAP in the presence or absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>When using the CTD mutant RNAP , abortive RNA synthesis was not detected in lacP1-6 and was made in somewhat reduced amounts in lacP1-6S1 either in the absence or presence of CRP ( Fig. 4 , lanes 9 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>A 170 base transcript of unknown origin ( marked x in Fig. 4 ) is produced in the presence of RNA polymerase and only in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>At low Es28 concentrations , this transcript is detected only in the presence of CRP ( lanes 3 -- 6 ) , although some transcript is generated in the absence of CRP as the RNA polymerase concentration is increased ( lanes 7 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Under these conditions , IHF inhibits TOC formation by RNAP , which can be overcome by the presence of low concentrations of CRP ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Fig. 7 showsthatthetranscriptmade in the presence of CAMP-CRP protects the probe overa length of -45 n ( lane 2 ) , whereas the transcript made in the presence of glycerol protects -50 n ( lane3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>Lane I , size markers ; lane 2 , RNA made in the presence of CAMP-CRP ; lane 3 , RNA made in the presence of ments as suggesting that the CAMP-CRcPomplex had no role 20 % glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>When RhaR was footprinted in the presence of CRP , there was no signiﬁcant change in the strength of the RhaR footprint , suggesting that RhaR and CRP do not bind cooperatively ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We noted that very little abortive transcripts were synthesized from lacP either in the ab-1-6M ( 35 ) sence or presence of CRP ( Fig. 4 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>We further propose that two distinct versions of the TOC can form at acsP2 in the presence of CRP ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>CRP influenced these footprinting patterns only quantitatively ( compare the left half of Fig. 3 to the right half ) ; all signals were stronger in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>only fails to break the loop but appears to stabilize it ; AraC dissociates three times more slowly in the presence of CRP in this spacing variant , ( Fig. 5 ( a ) and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>With the 11 base-pair insert , the halftime for dissociation of AraC from aral I decreases from 40 minutes to 4.5 minutes in the presence of ' CRP ( Fig. 5 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Site 2 was largely occupied in the presence of an intermediate concentration of CRP ( 123 nM ) ( Fig. 4 , compare lanes 3 in panels B and C with lane 3 in panel A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Specifically , bands at 11 , 1 , 13 , 15 , and 16 along the bottom strand ( Fig. 9A ) become reactive both in the presence of RNAP and RNAP CRP , but the intensities of these bands in the presence of CRP ( Fig. 9A , lane 7 ) are about 2 -- 8-fold for those without CRP ( Fig. 9A , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Increasing the UTP concentration brought the specific activity measured in the abortive initiation assay to a level comparable to that observed in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>We detected only one CRP protected region located from 112 to 87 ( including a hypersensitive region ) ( Fig. 3C , bottom ) when comparing sequence patterns in the absence or presence of CRP ( Fig. 3C , top and middle ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>These were obtained by global analysis of matched CRP titrations conducted in the presence and in the absence of fixed concentrations of Core domain dimer , as shown in Figure 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>A DNA fragment from positions 251 to 32 was subjected to DNase I footprinting in the absence or presence of CRP ( 400 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>Promoter DNA was incubated with RNAP either in the presence or absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Control experiments confirmed that this protection requires the presence of CRP ; no protection was observed in the absence of CRP , even at a concentration of Core domain of 48 µM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Multiple-round tran scription reactions were performed as previously described , in the presence or absence of CRP , except that KCl was used at a concentration of 100 mM ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Indeed , repression in the absence of CRP appears to be slightly stronger than in its presence ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>There are changes detected by the potassium permanganate probing in between the 10 and the 35 seen in the presence of CRP that are not seen in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Linearized plasmid pJCD01 was incubated with either Es70 or EsS in the presence or absence of CRP as detailed in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Reactions were performed in the presence ( ) or absence ( ) of 800 nM CRP and 75 nM RhaR as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Two pairs of reactions with no CRP are represented : the ﬁrst was performed in the absence of IHF , and the second was performed in its presence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Expressions of glnAp2 , CC-55 .5 and CC-55 .5 - M in the absence or presence of CRP are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>A plus sign ( ) indicates the presence of CRP , and a minus sign ( ) indicates the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Transcription reactions were performed in the presence or absence of CRP at 0.2 or 0.4 M potassium glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Transcription is carried out in the presence or absence of CRP as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>We emphasize here that fluorescence increase amplitudes of the steps are functions of quantum yields of the initial and final states , which may be affected by presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>We propose that pH , may indirectly affect CAMP-CRPbinding at malTp , either by pH,-induced alterations of malTp topology or by the presence of a pH,-dependent effector that may act in concert with CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>We transcribed the 235-bp fragment 2 , either in the presence of glycerol or in the presence of CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>To analyse the simultaneous action of both FIS and CRP , these studies were repeated in ﬁs + and ﬁs − strains grown in the presence of active CRP ( LB medium ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Consequently in the presence of active CRP , an excess of HUa is found in ﬁs + strains whereas in the same conditions , in ﬁs − cells , there is an excess of HUb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Both promoters show some activity in the presence of MalT alone while they are completely inactive in the presence of CRP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Taken together , these results indicated that the presence of CRP is important for cell growth at low temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Densitometric scanning of the stained gel revealed that the relative level of CRP decreased 2 - to 3-fold in the presence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>These same concentrations of RhaR were used in the presence of 400 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Figure 3 shows the response of malEp and malKp to inczreasing concent , rations of MalT in the presence of a saturating concentration of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>mal & and malKp depended on the presence of both MalT and CRP , as observed in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>the activities of mdEp and malKp depend on the presence of both CRP and MalT as they do in viva .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>However , the detailed pattern of protection is different : positions -59 , -60 , and -69 are accessible to DNaseI digestion in the presence of CRP but not H 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>Importantly , the differences are not only in magnitude but show that bases at 11 , 3 , 5 , 1 , 2 , 3 , and 4 have disproportionately more reactivity in the presence of CRP and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>A. RNAP titrations in the presence of WT CRP at 12 nM ( thin bars ) or 200 nM ( thick bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Strikingly , the TOC formed in the presence of the KE101 mutation was more open at its downstream end , independently of the CRP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Overall , the amount of abortive RNA synthesis in the presence of CRP is highest in lacP1-6S1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>On the nontemplate strand , in the presence of CRP positions -66 , -67 , -55 , and -56 are accessible to DNaseI attack ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>Further , the presence of CRP was found to be critical for growth at low temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>As expected , this weak cleavage is only seen clearly in the presence of melibiose and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>( B ) Effect of the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>lb , top ) barely entered the 5 % polyacrylamide gel in the presence of 30ng CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>To determine if AR1 was required upstream , the activity of XX-CC ( and CC-CC ) was also assayed in the presence of wild-type and variant CRP , in a crp + background ( Table 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>To investigate this point , the activity of ansB Ps ( of configuration CC-CC ; see Fig. 1A ) was compared in the presence of either wild-type or mutant versions of CRP introduced into a Acrp host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>However , the presence of either mutant CRP did not result in a signiﬁcant change in GRDC of PguaB in comparison to that with wild-type CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Results Glucose lowers the expression of crp on a multicopy plasmid We found recently that the presence of glucose in the culture medium lowers the CRP level 2 - to 3-fold in exponentially growing wild-type cells ( Ishizuka et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>It should be noted that in the presence of glucose the levels of CRP and crp mRNA in cya - cells are than those in cells significantly higher wild-type ( Figures 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The differences in AI-2 activity of cell-free ﬂuids from the wild type and crp and cya mutants grown with glucose ( data not shown ) were not as marked as the differences observed when cells were grown without glucose , probably due to the already very low level of cAMP and CRP in the cells in the presence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>E. coli AFMCRL and AFSBRL ( CRP '' ) were grown in M63 minimal medium containing 20 mM glycerol or 10 mM glucose in the absence or presence of 1 mM 3HPP until the cultures reached an OD600 of 0.9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>When E. coli AFSBRL ( CRP ) ( Table 1 ) cells were 2 grown in either glycerol - or glucose-containing minimal medium in the absence or presence of 3HPP , only basal bgalactosidase activities were observed ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>When E. coli AFSBRAL and E. coli AFSBAL ( pBT18 ) CRP strains ( Table I ) were grown either in glycerol - or glucose-containing minimal medium in the presence of 3HPP , no - galactosidase activity was observed ( Fig. 8 , A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>For example , it was shown previously that the intracellular concentration of CRP is lower in E. coli cells growing in the presence of glucose than in cells utilizing glycerol as a sole carbon source ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP Mediates a Superimposed Level of Regulation on the Pa Promoter -- It is known that some aromatic catabolic pathways in E. coli , such as the paa , hca , hpa , and mao-encoded pathways responsible for the catabolism of phenylacetate , phenylpropionate , 4-hydroxyphenylacetate , and 2-phenylethylamine , respectively , are repressed by the presence of glucose in the culture medium ( 26 , 27 , 42 -- 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>For cells grown in a glucose-containing medium , the amount of the gapB P0 transcript was dramatically reduced in the absence of CRP ( compare lanes 1 and 2 in Fig. 3A ) , whereas the absence of CRP had no marked effect on the low level of gapB P0 mRNA in cells grown in a succinate-glycerol-containing medium ( compare lanes 3 and 4 in Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>Although the ykgR-SPA transcript was barely detectable , the levels were higher for cells grown in minimal glycerol medium than those grown in mini mal glucose medium , and this difference was CRP dependent ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Regulation of acnA gene expression by CRP , ArcA and FNR The 0-galactosidase activity expressed from a single-copy A.acnA-lacZ fusion prophage in the crp ' parental strain was reduced 3-5-foldin the crp deletion strain but restored by the presence of a multicopy crp + plasmid ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>The crp mutation resulted in a basal level of setA - lacZ activity that was 2.4-fold higher than in the wild-type strain ( Fig. 4A , MG ) , suggesting that CRP might repress setA expression under nonstress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>The results illustrated in Fig. 2 conﬁrm that overexpression of wild-type CRP by plasmid pDCRP results in greatly increased expression from each of the six promo-ters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>However , rather than the approximately eightfold enhancement in hlyE expression observed in an hns mutant on solid medium ( 36 ) , in liquid culture H-NS has a positive effect on CRP-driven hlyE expression , a negative effect on FNR-driven hlyE expression , and little intrinsic regulatory activity in the absence of FNR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The ﬁndings that SlyA does not enhance hlyE expression in hns cultures grown in liquid medium under anaerobic conditions and that SlyA appears unable to activate hlyE expression in the absence of FNR and CRP and the determination of the location of the region of PhlyE occupied by SlyA ( Fig. 6 ) suggest that SlyA activates hlyE expression by antagonizing the negative action of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The HindIII-EcoRV fragment of pDW30 was isolated in the same manner and end-labeled at the HindIII site with [ a-35S ] dATP and [ CX - ~ ~ SI ~ GPuTriPfi.ed repressor ( in thepresence or absence of glyc-erol-P ) or CRP ( in thepresence or absence of CAMP ) was added to the labeled DNA in a reaction mixture containing 10 mM Tris-HC1 ( pH 7.4 ) , 1mM EDTA , 50 mM KC1 , 2 mM CaC12 ,4 mM MgC12 ,5 % glycerol , 0.025 % Triton X-100 ,100 pg/ml bovine serum albumin , and 5 pg/mI sonicated salmon sperm DNA in a total volume of0 .1 ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>The particular mutations introduced into the upstream site in XX-CC allow the targeting of CRP containing an EV181 change to this site , since CRP EV181 , but not wild-type CRP , is able to-bind at such sites ( Ebright et aL , 1987 ; Gunasekera et aL , 1992 ; and see Zhou et aL , FF-CC ansB Pe - ¬ Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The presence of both CRP sites as found in pMP30 and pMP32 gives rise to a normally regulated enzyme synthesis in cytR and deoR mutants , whereas with pMP34 , which lacks the putative CRP2 site but has retained the CRPl site , the yield of enzyme is low and enzyme synthesis is not derepressed , neither in wild-type strains , nor in cytR mutants ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>The resulting complex ( composed of IHF bound at IHF III , CRP dimers bound at CRP II and CRP I , and RNAP bound at acsP2 ) could fold into a wrapped conformation that inhibits escape due to numerous interactions between RNAP and several upstream elements ( Fig. 7D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The latter should contain CRP 3 , the 322 bp Sau3AI restriction fragment was CRP 4 , and as shown in Fig. 5 , this fragment was retarded purified on a polyacrylamide gel as described in Methods , with low concentrations of CRP ( 10-90 ng ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The level of pck RNA slightly increased at low CRP concentrations , but markedly decreased at high CRP concentrations ( Fig. 4Aa ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The addition of CRP decreased the ArcA-specific protection on binding sites II and III ( Fig. 6 , compare lanes 8 and 10 ) , and the CRP-specific protection was decreased as the ArcA-P level was increased ( lanes 9 -- 11 ) , implying that ArcA-P and CRP compete with each other for the binding to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>The reverse also was true ; the presence of RNAP inﬂuenced the binding of WT CRP , substantially increasing the HS associated with all three CRP binding sites and with the HS sites between CRP I and CRP II ( Fig. 3A , lanes 7 -- 9 ; Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The combination of RNAP , IHF and CRP ( Fig. 3A , lanes 3 -- 5 ; Fig. 3C ) increased protection throughout the region downstream of CRP II to about -40 , especially between CRP II and CRP I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The combination of CRP and CaiF at subsaturating concentrations resulted in a discrete protected area observed in the presence of the two sole regulators ( Fig. 5A , lane 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The presence of 20 nM IHF ( Fig. 3A , lanes 2 and 15 ; Fig. 3C ) caused the loss of the CRP III-associated HS sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The replacement of WT CRP with the KE101 mutant in the presence of IHF resulted in a signiﬁcant decrease in the protection between CRP I and CRP II ( Fig. 3A , lanes 16 -- 18 versus lanes 3 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The regulatory effect of CRP was ﬁrst inferred from the absence of induction of caiT lacZ and ﬁxA-lacZ fusions in a crp mutant ( Tables 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The results show that MalI-dependent repression is slightly greater in the absence of CRP , but each of the different mutations has a similar effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>The relative levels of hup mRNAs were determined for the two strains in LB supplemented-with-glucose since the addition of glucose is equivalent to the absence of active CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The relief of H-NS silencing by SlyA was also much less efﬁcient in the absence of CRP , since only a very weak cytolytic activity could be detected in CRP-deﬁcient strains overexpressing SlyA ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The E. coli BL21 pCRP transformant was grown in LB broth in the presence of 50 g/ml ampicillin , and expression of CRP was induced in mid-log phase by the addition of 1 mM isopropyl - - D-thiogalactopyranoside ( IPTG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The presence of CRP * led to the restoration of LamB expression at 37 ° C in hns and hns-stpA mutant strains ; the expression levels were similar to that in a wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The effects of the global regulators ( ArcA , FNR and CRP ) on the pdh , ace and Ipd promoters were investigated in wiid-type and mutant strains containing singie-copy X. / acZ-fusion prophages in the presence and absence of the corresponding multicopy regulator piasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>The data ( Fig. 3 ( a ) ) show that AraC dissociates 16-fold faster in the presence of cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>( a ) The dissociation r & es from muI , in a spacing mutant cont , aining 4 bp inserted between CRP and aral were compared in the presence and absence of cyclic AMP-CRP by t ' he looping-restric ~ t , ion cutting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The concentrations of the proteins were 100 nM RNAP and 25 nM CRP ; '' and + indicate the absence and presence of the protein in the assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>The fold activation was expressed as a ratio of the intensity values in the presence or absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>The fact that hupB-P2 expression is not affected by CRP certainly explains why FIS expression of hupB-P2 is effective in the presence or absence of CRP ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Run-off assays The assays reported above indicated that similar amounts of competent complex were formed at malT in the absence and presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>The main conclusion from this work comes from the study with galp19T , where RNAP can form open complexes at P1 both in the absence and in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3668">
The obtained rate constants are 1.75 10 3 and 3 1 3.04 10 s , in the absence and the presence of CRP , respectively ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></label>
<input id="act_similarity_3668", type="checkbox">
<div class="ACT_similar">
<li>The rate constants for this step in the 3 absence and in the presence of CRP are 1.75 10 and 3.04 3 1 10 s , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li></div>
<li>The dissociation rate of Ara ( l from minicircles containing these different spacings was measured in the presence or absence of CRP by the looping-restriction cutting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The 32P-labeled fragment was digested with DNase I in the absence and the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>The seen in the absence than in the presence of CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>The fluorescence increases both in the absence and in the presence of CRP and can be fitted to a single exponential .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The representative kinetic traces are shown in Fig. 4 , in which DNA ( 35 nM ) was mixed with RNAP ( 550 nM ) either in the absence ( Fig. 4A ) or presence of CRP ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The results of KMnO4 probing of the different promoters in the absence and presence of CRP are shown in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The primer extension pattern obtained when KMnO4 was used to modify the lac promoter either in the presence or the absence of CRP is shown in Fig. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The stimulation by CRP in vivo was completely dependent on the presence of Fis ; cells lacking Fis have ,5 U of b-galacto-sidase ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The transcriptional level of each gene in the presence of either HL159-CRP or KE101-CRP was compared with those from the control reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The activities of the oriented RNA polymerase heterodimers imply that the pre-ference for the aIICTD by Fis is maintained in the presence of both Fis and CRP , leaving aICTD to contact CRP ( Figure 7 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The stopped-flow experiments also indicate the presence of a rapid isomerization step in which the fluorescence of 2-AP increases with a forward rate constant of 9.1 and 15.1 s 1 , without and with CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The cells were grown in minimal medium in the presence of glycerol and maltose , i.e. under conditions in which both MalT and CRP are active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Results Properties of witd-type and mutant FNR proteins The major structural difference between FNR and CRP is the presence of an W-terminal cysteine cluster in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The potassium permanganate results show that there are increases in reactivity in the presence of CRP and RNAP compared with RNAP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The cleavage pattern in this region there have not been enough experimental data to permit de-resembles more closely that obtained in the presence of CRP tailed discussion of this hypothesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>The presence of CAMP and of an active CRP deter Two Functional gal Promoters in Intact E .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>The malTplO mutation alters the CRP lacZ + ) loperon fusion in the presence of chromosomal malTp2 malTp2O mutations was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>The rate of [ i-oxldatlon was unaffected by the presence of the / acZ fusion and demonstrated control of p-oxldatlon by FadR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>( i ) The presence of IHF on its three speciﬁc sites in the full-length pacs444 , or on the IHF III site of the truncated pacs236 , does not inhibit the formation of an open complex in vitro in either the presence or the absence of CRP ( Figs 4 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The gal mechanism is still controversial , but in one model , in the presence of CRP the repressor binds to an adjacent upstream position to form repression complex containing both proteins ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The presence of melibiose induces a conformation change in MelR such that it is able to occupy Site 20 : this requires a complex of at least four MelR subunits and one CRP dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The open complex formed in the presence of CRP is structurally identical to the one found at a CRP-independent promoter up-mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The presence of either WT CRP or KE101 allows RNAP to form an open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The dependence of these interactions on cAMP and the differences in the pattern of interspersed bands in the ` footprint ' of repressor ( fig. 3 B , lanes 2 , 3 ; fig. 4 , lanes 2 , 3 ; fig. 6 B , C ) and CRP in the operator and the difference in salt dependence ( see subsequent paragraph ) exclude the possibility that a contamination by repressor in the CRP preparation causes the observed ` footprint ' of CRP in the operator sequence ( cAMP does not change the repressor ` footprint ' , fig. 4 and The comparison of the two CRP ` footprints1 reveals some expected similarities as to size of the protected fragments of 25-28 bases on each DNA strand and the positions of the interspersed CRP-induced DNasel enhanced cutting sites ( fig. 6 A , B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The presence of CRP shift the sites of DNA cleavage downstream to positions 229 and 240 , as P2 is repressed and P1 is activated ( this pattern of cleavage is similar to that seen with galp19T DNA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>The CRP-induced loop-breaking effect , that is , t ' he amount of the increase in the AraC dissociation rate , is reduced from a 16-fold to a ninefold effect by inserting one DNA helical turn between the CRP and aral sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The number of CRP targets should increase by changing the experimental conditions such as that : i ) the number of SELEX cycles was reduced to pick up DNA sequences with weak affinity to CRP ; or ii ) the cut-off level of SELEX-chip pattern was set at lower levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The result with [ Leu159 Glu101 ] CRP was unexpected and implies that , in our conditions , mere occupation of the DNA site for CRP is insufﬁcient to shift RNAP from P2 to P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>The afﬁnity of CRP for the secB gene , expressed as the apparent Kd value , was estimated to be about 0.8 3 1029 M , where the ratio of free DNA and bound DNA is 1 ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>The arrows on the right point to CRP enhanced or unaffected DNA bands in the ` footprint ' ( lane 3 ) - Further descriptions are as in fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The mechanism by which Sxy facilitates CRP -- DNA interactions is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>The stimulatory effect of the H159L substitution suggests that , when functional , AR1 may mediate an inhibitory interaction between CRP and the a-CTD , disruption of which promotes the formation of additional productive interactions , presumably involving FucR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>The ﬁnding of synergistic effects between all pairs of activators suggests the possibility of direct RhaR-CRP , RhaR -- a-CTD , and CRP -- a-CTD interactions , although other mechanisms could also account for the synergism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>The CRP protein , which is known to act as a dimer , was also able to-bind to F45 ( Fig. 3A , lane 6 ) but not to the half-site present in F24 ( Fig. 3A , lane 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The isolated proP CRP site was efficiently bound in cells growing in LBN , but protection was lost 10 minutes after addition of 0.6 M NaCl ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The RNAs labeled Pl , synthesized in the presence of CAMP-CRP , and those labeled P2 , synthesized in the presence of glycerol , both derive from the 5 ' - proximal region of the gal operon and extend to the Hint II site within the E gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>The data presented here clearly establish that occupation of MalT site 1 and 2 is indeed not sufﬁcient to fully activate malEp , and that both the CRP site 1 and the distal MalT sites further stimulate malEp , 14 and ﬁvefold , respectively , when the proximal MalT sites are similarly occupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The E.coli sbmC gene was included in this analysis ; it has no H.inﬂuenzae homolog but is CRP regulated , and its predicted CRP site resembles the CRP-S motif ( 3 ) ( Figure 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>The location of Fis site I at nt 41 is analo-D 2 gous to the positions of other activators , such as the l repressor , MalT , AraC , and CRP ( in some cases ) , in which direct contact with s70 has been implicated ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545153" target="_blank">7545153</a>]<br></li>
<li>The aCTD binds to its target upstream of the CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The resulting displacement reactions are collected in Figure 5 , where it is clear that the rate at which polymerase leaves the P , site depends on how much active CRP was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>The P0 and P1 promoters ( in cis ) are controlled by CRP , since fusions of the chloramphenicol acetyltransferase gene to these promoters show increased chloramphenicol acetyltransferase activity ( 125 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The phoA and p & E promoters are negatively controlled by the phoCT product : p & E and psi0 are repressed by t.he CAMP-CRP system ; and psi0 may be negatively controlled by a pho gene product that is required for the full induct ' ion of BAPase synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3540312" target="_blank">3540312</a>]<br></li>
<li>The regulation observed in the trp operon might also have resulted from an indirect effect , as there is no evidence from previous studies to indicate that CRP regulates this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The objective of this work was to provide a preliminary study of the role of CRP at the divergent promoters of guaBA and xseA and the data presented here o ¡ ers the cents rst evidence that CRP is involved in their regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The abbreviations used are : CRP , CAMP receptorprotein ; SDS , sodium dodecyl sulfate ; IPTG , isopropyl-P-D-thiogalactosidek ; b , kil-obase ; bp , base pair ; DTT , dithiothreitol ; dNTP , deoxynucleoside triphosphate ; avian myeloblastosis AMV , virus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3017988" target="_blank">3017988</a>]<br></li>
<li>No Effect of CRP on Promoter Clearance -- As an assay for a The dissociation constant was calculated from RNA polymerase concentration dependence of 2-AP fluorescence increase in the stoppedflow experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The decrease in crp RNA and the increase in divergent RNA follow the similar CRP concentration dependence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>The concentration of CRP was as follows : lane 1 , no protein ; lanes 2 -- 6 , 0.025 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3719">
The concentration of CRP in each reaction was as follows : lanes 2 and 5 -- 8 , no protein ; lane 3 , 0.10 mM ; lanes 4 and 9 -- 12 , 0.30 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_3719", type="checkbox">
<div class="ACT_similar">
<li>The concentration of CRP in each reaction was as follows : lanes 2 -- 6 and 11 -- 15 , no protein ; lanes 7 -- 10 , 0.30 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The concentration of CRP was as follows : lanes 1 and 6 -- 8 , no protein ; lanes 2 -- 5 , 0.10 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The concentration of CRP in each reaction was as follows : lanes 1 -- 5 , no protein ; lanes 6 -- 10 , 1.67 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>The concentration of CRP in each reaction was as follows : lane 1 , no protein ; lane 2 , 30 nM ; lane 3 , 125 nM ; lane 4 , 250 nM ; lane 5 , 500 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>The concentration of CRP was as follows : lane 1 , no protein ; lane 2 , 25 nM ; lane 3 , 50 nM ; lane 4 , 100 nM ; lane 5 , 300 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>The reaction mixture contained 0.2 mM-CAMP instead of ATP and maltotriose and various concentrations of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>The used concentrations of CRP and FIS are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The CRP concentrations are as follows : lane 1 , 0 ; lane 2 , 6.17 nM ; lane 3 , 123 nM ; lane 4 , 1.23 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The concentrations of proteins were 100 nM RNAp , 100 nM MhpR-His , and 200 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>The concentrations of CRP dimer used were 0 , 200 , 400 , and 800 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>The final concentration of CRP in each reaction was 500 nM , and TcbC was added to a final concentration of 2 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The ﬁnal concentration of CRP , when present , was 800 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The CRP concentration is 75 nM in lanes 1 , 3 , 5 , 7 , 9 , 11 , 13 , and 15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The concentration of CRP was 0.10 mM in all lanes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The concentration of RNAP was 20 nM ; the concentration of FIS was 20 nM in lanes 2 and 6 and 100 nM in lanes 7 and 8 ; the concentration of CRP was 20 nM in lanes 2 and 3 and 10 nM in lane 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The reaction mixtures loaded in lanes 1 and 4 contained no CRP , that loaded in lane 2 contained 320 nM CRP dimer , and those loaded in lanes 3 and 5 contained 64 nM CRP dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>The concentrations of CRP ( dimer ) in lanes 2 , 3 , 4 , 5 , and 6 were 10 , 20 , 40 , 100 , and 100 nM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The concentrations of CRP ( dimer ) in lanes 2 , 3 , 4 , and 5 were 4 , 10 , 15 , and 80 nM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The results show that CRP binds to speciﬁc targets in the control mglB fragment ( as expected ) , and in the gatY , aer , sdaC , ychH , malX , yffQ , ycdZ and fdoG fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>The failure of CRP to activate psAD in the ot , her spacing mutations is not due to lack of ara02-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The results indicate that CRP repression of gadX occurs primarily , but not completely , through its control over rpoS ( Fig. 7B , compare EF528 and EF678 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The number in boxes represent in the order from left to right : The number of CRP target genes listed in Regulon DB ( white ) ; the number of genes identified to be regulated by CRP in this study ( green ) ; the number of genes predicted to be under the control of CRP in this study ( yellow ) ; the total number of genes under the control of CRP ( orange ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The expression of CRP was induced by the addition of arabinose at various concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The requirement of CRP and FNR for anaerobic clyA expression was subsequently investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The fact that the expression level of IIC72 -- CRP retaining the membrane-targeting ability is markedly lower compared to that of IIC72FS -- CRP , the nonmembrane counterpart of IIC72 -- CRP , supports this assumption .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>The level of FLAG-tagged CRP expression was measured by Western blotting using anti-FLAG antibody .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The highest level of lacZ expression was achieved with CRPH159L , and , intriguingly , the inactivation of CRP site 1 completely abrogated the superinduction mediated by CRPH159L , K52N ( compare Tables 2 and 3 [ 1,600 versus 31 Miller units ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>The presence of this site may explain the ability of CRP to repress σSdependent expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>The gal operon encoding the enzymes for galactose meta-bolism has two promoters regulated by CRP in opposite ways ; one [ P ^ ) is stimulated and the other [ P ^ inhibited by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>The glgCAP operon , encoding glycogen biosynthesis and degradation , is known to be regulated by CRP ( 2 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Role of RpoS and Crl in bgl operon regulation The bgl operon is regulated by various pleiotropic regulators including H-NS and CRP , as essential factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
<li>The negative control of speC by CAMP in a cell-free system was shown to be dependent upon CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>The clear levels of control observed , however , suggests that if CRP is acting indirectly then it is probably through inﬂuencing a direct transcriptional-regulator of hyb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>The stimulating effect of Lrp is not sigma factor speciﬁc , and it can only be observed when the CRP box is optimally located ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The footprinting study also shows that MelR173 , when bound at sites 1 and 2 , is sufficient to recruit CRP to pmelAB , between sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>The thick arrow drawn on the right points to the CRP induced weak DNasel enhanced cutting sites ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Evolution of CRP and Sxy in g-proteobacteria The above analysis revealed that specialized CRP sites regulate competence genes in the Enterobacteriaceae , Pasteurellaceae and Vibrionaceae ( the ` EPV ' clade ) , but not in the Pseudomonadaceae or Xanthomonadaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>The influence of CRP on the transcriptome progressively increased with increased aerobiosis , whereas the activities of ArcA , FNR , IHF and Lrp decreased as aerobiosis increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The CRP regulators represent a fairly homogeneous group of proteins that control the response to glucose-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The overlapping control of gadX by CRP , RpoS , and GadW makes E. coli highly attuned to situations that can lead to lethal acid stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The data , in sum , suggest that the majority of CRP control over gadA and gadBC occurs through its control over RpoS , which , in turn , is needed to maximally express gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The mixture was incubated for 10 more minutes after addition 54 of ArgR and IHF in that order for reactions with or of ArgR and CRP S simultaneously for reactions with .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>The case of ptsG demonstrates that an in vivo indirect effect can mask the real regulation by CRP , sometimes leading to the opposite conclusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The interaction between the three repressors and CRP in the regulation of deo is an aspect that needs to be studied in the future .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The invitro CRP regulation of PI seems rep -- c FIG3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>The CRP-McaS-csgD circuit forms an incoherent feedforward loop where CRP and McaS act in opposition ( CRP activates csgD and McaS while McaS represses csgD ) , while the regulatory circuit formed by CRP-McaS - flhD is a coherent feedforward loop ( CRP activates both mcaS and flhD while McaS activates flhD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>The residues involved in cAMPbinding by CRP and the cysteine residues in YeiL and FNR are shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The putative consensus sequences for the CRP , FNR , Fur , and FhlA/HyfR proteins are enclosed by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The column was then equilibrated with 10 ml of buffer A. Subsequent to application of the sample to the column , the flow was stopped to allow interaction of CRP with the ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The fusions were designed to produce IIC -- CRP proteins in which the N-ter-minal 14 , 40 , 72 , 107 , and 139 residues of IICBGlc were joined to the CRP ORF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>The mechanism by which CRP activates peAD is of interest for two reasons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>The sole exception , spf ( see CRP in Tables 1 and 2 ) , is in fact just marginally remote , approaching the -70 position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The effect of CRP is not as pronounced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The speciﬁc mechanism for the observed effect of CRP is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>The cluster , however , is lacking in CRP and FixK , which are not O2-sensitive regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>( Right ) The arrowheads indicate the phosphate bonds accessible to DNase1 when CRP is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>The thymine residue at 2 appeared to be specifically reactive when CRP was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The properties of CRP and Fis coactivation are explored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The relative order of aral , araP , and ara ( CRP ) is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>The number of CRP targets should be more because : 1 ) both of the divergently transcribed genes or operons might be regulated for some of group-B and group-C cases and even for some known targets ; and 2 ) regulation targets increase if the cut-off level of SELEX-chip is set at lower levels [ note that only about half of the known CRP targets were picked up at the cut-off level herein employed ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The rhamnose regulon is also induced by the pentose L-lyxose ( Badia e t al. , 1991 ) , which is structurally similar to L-rhamnose and is metabolized by the Abbreviations : CRP , catabolite repression protein ; tsp , transcriptional start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8757746" target="_blank">8757746</a>]<br></li>
<li>Glucose lowers the crp mRNA level The lowered level of CRP in cells grown on glucose-containing medium could be due to destabilization of the CRP protein and/or a decrease in the level of crp mRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The CRP protein is another global regulator that , on exhaustion of glucose or its analogues , switches the cellular metabo-lism to the utilization of alternative carbon sources , such as lactose , maltose , and arabinose ( 24 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The observations reported here have many notable features : in the deﬁned MOPS minimum medium with different carbon sources , CRP is activated in poor quality ( deﬁned by growth rate ) carbon sources relative to rich quality ones ; this activated CRP associates with the ﬂhDC operon encoding the master regulator with roles in ﬂagellar regulation and development ; the activation of CRP is correlated with the increase of expression of sF as well as the increase of cell motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The activation of FlhDC expression by CRP is reduced by repression of crp by RpoS , a pathway independent of the csgD and cpxR genes ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>Key words : autoregulation/catabolite repression/CRP - cAMP/crp expression/transcriptional activation The inhibition of gene expression by glucose in culture medium has been a paradigm for understanding how genes are regulated in response to nutrient status in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The transcriptional-activators MalT and CRP regulate the expression of the E. coli malt.ose regulon ( Schwartz , 1987 ) with regard to the availability of maltodextrins and of some other sugars such as glucose , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>The glucose , oxygen , and nitrate repression of dcuB expression suggests that dcuB could be subject to regulation by the global transcriptional-regulatory proteins FNR , CRP , FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Under these conditions the formation of active complexes at malEp and malKp could be easily followed as a function of time after addition of polymerase to the template that had been preincubated for 20 minutes with MalT and CRP ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>These mechanisms could involve interactions of Lrp with other proteins , such as cyclic AMP-CRP , that bind in the dad promoter region , as well as direct effects of Lrp on RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>These could either be contacts to RNA polymer-ase itself or CRP induced changes in DNA structure providing an improved contact surface for polymerase to-bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>Whether IHF-induced bending of the paaZ-paaA intergenic region can bring together RNA polymerase bound to Pz and CRP bound to Pa could be an interesting model that remains to be tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>At both promoters AraC protein binds adjacent to RNA polymerase and CRP binds further upstream next to AraC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Alternatively , Shanblatt & Revzin ( 1983 ) have recently proposed that RNA polymerase bound at P induces the fixation of a second CRP molecule , which binds to the sequence upstream from - 5 0 and is responsible for the protection as far as - 7 0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>Type III promoters , such as araBAD ( 48 ) , malE , and malK ( 52 ) , require the coordinate function of CRP , usually bound at upstream sequences , and one or more additional activators , usually bound between CRP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>A direct contact between CRP and RNA polymerase could be more difficult to establish due to the increased distance between the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>CRP activates guaP and represses xseP when cells are placed under nutrient-limited conditions , presumably to remove competition for RNA polymerase between the closely-spaced promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Under these conditions RNA polymerase can successfully compete with CRP to form transcriptionally active complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>On this view FIS , and possibly also CRP , act to ﬁne tune a fundamental mode of transcriptional regulation , not only by their direct effects on DNA topology but also from the indirect effects of their transcriptional control of other nucleoid-associated proteins , such as HU and H-NS [ 3 , 4 ] , which also act as determinants of chromosomal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>In summary , we found families involved in the regulation of processes that require unique regulators in bacteria and probably different from those in archaea , such as the global regulator protein , CRP , the positive regulator of the cytochrome c552 , DniR , or the DNA repair system regulated by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>-80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC DISCUSSION osmY ( csi-5 ) is one of many stationary-phase-induced and osmotically regulated genes in E. coli that require us , the gene product of rpoS , for expression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Lrp Lrp -200 GACCGTGTCCTTGCAATAGGCCTGCAAGTCAAGGTGGTCTGGCGCTCGTAATTAAGAAC -140 -80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC ACKNOWLEDGMENTS We thank Erhard Bremer and his coworkers for providing bacterial strains and Merna Villarejo for sharing unpublished data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>For example , CRP is considerably larger than Fis , and aCTD binds approximately one turn of the helix downstream of the promoter-proximal CRP subunit in the CRP-aCTD-DNA complex .28 Furthermore , the regions on CRP ( AR1 ) and Fis ( the B-C turn ) that interact with aCTD are positioned differently with respect to the bound DNA : the Fis B-C turn is immediately adjacent to and in line with the path of the DNA ,17 while CRP AR1 is further away from the DNA and off to one side .47 Therefore , in the complex with CRP , aCTD is not subject to the constraints that appear to alter its mode of DNA binding in the Fis complex , and it interacts with DNA in the same manner in which it interacts with the UP element ( Benoff , B. , Yang , H. , Lawson , C. , Ebright , Y. , Ebright , R. & Berman , H. , unpublished results ) A model for aCTD-Fis-DNA interactions in the rrnB P1 activation complex Materials and Methods Strains and plasmids Table 1 lists the strains and plasmids used in this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866514" target="_blank">11866514</a>]<br></li>
<li>Rather , we could explain these observations by suggesting the formation of a nucleoprotein complex in which the proteins form a scaffold for wrapping the regulatory DNA in a way somewhat similar to the structure adopted by the interspersed bound MalT and CRP proteins responsible for malK promoter activation Table !! .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3804">
To determine whether FIS bound at FIS III can displace CRP from CRP I , we performed a competitive EMSA , incubating puriﬁed FIS and CRP with a DNA fragment that contains only sites FIS III and CRP I ( Fig. 4A ; FIS III ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_3804", type="checkbox">
<div class="ACT_similar">
<li>To determine whether FIS and CRP can bind simultaneously , we performed a competitive EMSA incubating puriﬁed FIS and CRP with a DNA fragment that contained sites FIS III , FIS II and CRP I , but not CRP II ( Fig. 4A ; FIS II + FIS III ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>To test CRP binding to the four predicted CRP sites and to determine whether CRP could bind to any other potential sites , we used DNase I footprinting assays to monitor occupancy of CRP on 32P-labeled DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>To determine whether occupancy of these DNA sites for IHF affects the binding of CRP , we performed DNase I footprint analysis and compared the signatures of CRP alone or in combination with increasing concentrations of IHF ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>As shown ( Fig. 3 , lane 9 ) , CytR aione binds weakiy to the DNA and results in protection against digestion by DNase I in the region between the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>As expected , two bands resulted from the titration of this DNA fragment with CRP , and the range of CRP concentrations giving rise to BMR ( 10-90 ng ) suggested that both sites bind CRP with high affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>In this complex , MelR binds to two pairs of DNA sites , sites 2 and 20 and sites 1 and 10 , and CRP binds between the two pairs of sites ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>To investigate whether CRP could directly bind to the clyA promoter region , gel mobility shift assays and DNase I footprint analysis with puriﬁed CRP and the clyA DNA were carried out as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Simultaneous addition of both CaiF and CRP substantially increased permanganate reactivity of thymines within the ÿ10 hexamer region ( Fig. 5B , lanes 7 ± 9 ) , suggesting an increase in the extent of DNA unwinding necessary for promoter opening .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Transcription repression by CRP can occur through the occupation of DNA sites that overlap core promoter regions ( 69 ) or by stabilization of a transcriptional-repressor or corepressor bound to the promoter region ( 53 , 77 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP is not needed for MelR-dependent repression of the melR promoter To investigate the possibility that the activator CRP is required for repression of the melR promoter by MelR , we constructed derivatives of the TB20 fragment carrying mutations that inactivated either or both DNA sites for CRP , and measured MelR-dependent repression ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>CRP complex fails to-bind to the galP35 DNA in vivo , then the normal 2:1 ratio of S1 to S2 for the wild-type promoter should be skewed in the direction of S2 in a cya + background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>Bandshifts are apparent in lanes 2 and 3 , although very little DNA is shifted relative to the lacZ promoter in lane 5 , indicating that CRP has low but speciﬁc afﬁnity for CRP-S sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Finally , it is surprising that the three highest afﬁnity DNA sites for CRP are located in the yffQ , ycdZ and fdoG fragments , where substantial effects of CRP on promoter activity could not be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>It can be structural , as found in the araBAD promoter , in which case a CRP-induced DNA bend acts to disrupt a repressive DNA loop , or steric , as in the malK promoter , in which case CRP induces a repositioning of the MalT activator into an activating register .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>Although , CRP binds to this promoter during sugar starvation , no trans-cription occurs unless the LacI repressor binds lactose and releases the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Thus , DNA phasing seems to be crucial in case of the bgl promoter which is HNS repressed and synergistically activated by BglJ -- RcsB and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Removal of this DNA decreases the capacity of the malT promoter to be activated by CRP in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>By analogy , CRP might work as a DNA destabilizing protein ( 2 ) , whose interactions with DNA are facilitated by glycerol in vitro , although effects of glycerol on the protein itself can not be ruled out to explain the observed changes in the protection pattern .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Increasing salt concentrations generally have a destabilizing effect on protein -- DNA interactions in vitro , and this has been specifically reported for CRP [ Fried and Stickle 1993 ; but note that chloride salts , which strongly inhibit CRP -- DNA interactions ( J.Xu , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Without this DNA bend , AraC protein alone can not contact polymerase ; however , , the combined bending caused by CRP and AraC protein ( unpublished results ) may allow AraC directly to contact a second region of polymerase and further stimulate its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>This increase in DNA bending could change the position/geometry of the CRP protein and potentially decrease the ability of CRP to contact - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>CAMPhas been shown to induce protease sensitivity in the E. coli CRP , and this conformational change has been correlated with the ability of the protein to-bind DNA ( Heyduk & Lee , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>Typically , bacterial transcriptional-regulators use a helix-turn-helix ( HTH ) motif to-bind DNA and commonly form homodimers ( e.g. , cI , cro , trp repressor , CRP , NtrC , MarR , and FadR ) ( 2 , 22 , 36 , 40 , 42 , 43 , 49 , 60 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>We show that repression requires a complex involving four DNA-bound MelR molecules , and that the global CRP regulator plays little or no role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>Although our results indicate that CRP 's role is to inhibit repression by GcvA at the gcv promoter , it is still possible that direct contact occurs between CRP and RNAP via DNA bending at gcv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>However , evidence suggest that Lrp s role is structural , bending the PgcvT DNA to allow a GcvA Æ RNAP interaction [ 14 ] and CRP s role is to antagonize GcvA-mediated repression [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>CRP prevents formatIon of the repression DNA loop in the araBAD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>ara02-ara1 , loop breaking by ( XP could be due to bending of the DNA by CRP in a manner that inhibits the formation of the repression loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>This suggests that the mechanism of loop breaking is only partially due to a direct , negative interaction of CRP with AratC bound at aral : the majority of th AraC ¬ linear DNA ¬ lb ) Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>This long-range interaction results in a bend in the upstream DNA that permits an additional stabilizing contact via the AR2 surface between RNAP and CRP bound to CRP I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>When puri cents ed CRP was used , bands derived from unbound DNA ( which is in excess of DNA bound by puri cents ed CRP ) obscured the footprint and no increase in protection was seen with higher concentrations of CRP ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>In addition , dimethylsulphate protection experiments provide evidence that the nucleoprotein complexes formed by CRP and MalT bound to malEp and TmalKp on supercoiled and relaxed DNA are different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>In this structure , the DNA appears to be wrapped around a core of proteins made up of MalT molecules ( when CRP is omitted ) or of a mixture of MalT and CRP molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2664421" target="_blank">2664421</a>]<br></li>
<li>CRP binds to a DNA target that overlaps the 235 element of both promoters and switches RNAP from P2 to P1 by repressing P2 and activating P1 ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>We predict that CRP bound at site 4 would be on the same `` face '' of the DNA as CRP at site 1 and RhaS and RNAP at rhaBAD , while CRPs bound at sites 2 and 3 are predicted to be on the same `` face '' of the DNA as RhaR and RNAP at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>We then constructed the TB211 fragment in which the CRP2 site was inactivated , and TB222 , in which both DNA sites for CRP were inactivated ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>CRP is bound at ÿ121 .5 with the aICTD ( b-associ-ated ) bound to the DNA on the downstream side of CRP .29 Fis is bound to a site at ÿ41 , overlapping the ÿ35 DNA recognition element for s38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>For example , replacement of T : A6 in a consensus CRP site with C : G6 causes an 80-fold reduction in CRP afﬁnity by increasing the free energy required to bend the DNA ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>CRP-DNA afﬁnity increases with increasing similarity of a DNA site to the CRP consensus , but CRP 's afﬁnity for a site matching the consensus is too strong to be biologically useful ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>A CRP induced DNA destabilization between site 1 and 2 , however , is not excluded , since it might not influence the DNasel cleavage pattern .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 and site 2 ( CRP1 ) is inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>CRP binds to a single DNA site upstream of I1 and I2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Additionally , as in araCBdD , CRP binds adjacent to the promoterproximal operator site within the DNA loop both in lac and dro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Sxy is hypothesized to interact directly with CRP , and may also bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>They also support the possibility that FIS and CRP may bind simultaneously to form a FIS -- CRP -- DNA complex at lower FIS concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>However , when the distance between CRP and aral is increased by 1I base-pairs , i.e. one complete DNA helical turn , CRP breaks t ' he loop , although less effectively than when it is at the wildtype spacing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>RhaR was found to bend DNA by 160 degrees in vitro ( 30 ) , and this bending may be required to bring CRP bound at 111.5 close enough to interact with - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Besides the free DNA ( F ) , three DNA bands with reduced mobility , FC , FCR1 , and FCR2 , corresponding to DNA bound to one CRP , one CRP and one GntR , and one CRP and two GntR molecules , respectively , were formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>We conclude that efﬁcient coactivation by Fis and CRP requires that both activators be bound to the DNA prior to Es38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Protein species bound to the shifted DNA complexes were determined by immunoblot analysis , using speciﬁc antibodies against AscG and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>CRP is bound at 2121.5 with one a-CTD bound to the DNA on the downstream side of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Probe DNA-bound CRP was decreased by adding increasing amounts of AscG ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>A 310bp DNA fragment was next amplified by PCR in order to determine if CRP can bind to the two putative Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>Fleoent DNA band mobility shift experiments established that GRP can bind to all five CRP operator sites in vitro ( Ramseier and Saier̂ 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>However , in most cases , the positions of CRP and XylR sites conform to the period of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Because only DNase I hypersensitivity indicative of DNA distortion but no extensive protection of these putative sites by CRP was observed , it appears that CRP does not bind tightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>A B ATG ECDRV Hpa \ -35 -10 CRP CRP o +1 CRP CRP CRP mtIA 11050 3S6 bp PCR DNA fragment : 10704 11059 197 227 159 129 - * EcoRV Hpa \ 6 5 M4 bp PCR DNA ttagmenl fused lo lac ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>Lanes : 1 , no addition ; 2 , 0.25 , uM CRP ; 3 , 0.5 , uM CRP ; 4 , 1 jiM CRP ; 5 , 1.5 , uM CRP ; 6 , pBR322 DNA cut with MspI used as size markers , which are shown in nucleotides at the right .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>How these weakened interactions aVected CRP secondary structure as well as CRP interaction with DNA and RNAP were investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Transcription of Hint II-Cleaved pKgalS DNA in the Presence of CAMP-CRP and of Glycerol An autoradiogram of the transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>Digoxigenin-labeled DNA fragments of seq1 , seq2 , seq3 , and seq4 were incubated with 0 to 80 nM puriﬁed CRP , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Binding of CRP at proP on supercoiled DNA was examined by in vitro DMS footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Concentrations of CRP that lead to the occupation of CRP I alone favour the formation of the forward-leaning TOCA , in which the DNA is open to the -1 to +3 positions ( A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>It is thought that CRP binds in the major groove of DNA , and by analogy , we suggest that HI is a major-groove binder .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>Time ( min ) ¬ 27 810.25 81 minicircle CRPiAraCIDNA-CRP/DNA ¬ AraUDNA ¬ linear DNA ¬ ( 4 E = 0 , 000 $ $ + I 1bp CRP-am / $ + AraC + CRP + AraC - CRP - 0.25 2 4 12 36 0.25 2 4 12 36 8s !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>These interactions are facilitated by looping of the DNA between the CRP and Fis activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>When four base-pairs , nearly half a DNA helical turn , are inserted between CRP and aral , CRP not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>thus CRP-CYTR-DNA - f 7 $ CYTR-DNA - $ 4 Free DNA ¬ - - - 10.25.25.25.25 -13.06 - * 25 - .3 .06 - - ~ 3 ,06.01 .3 .3 .3 ,06 + + CRP CYTR Cyttdtne .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>c -- CYTR-DNA + FREE DNA + h CRP w - I FREE DNA - t ill '' ` 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CRP and the lipoprotein DNA were gifts from M. Inouye .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>These results for CRP activation at proP through AR1 are similar to those found where CRP is acting as an upstream activator alone or in conjunction with either another molecule of CRP , FNR , or the l cI protein bound in an analogous position to Fis site I at proP P2 ( 11 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Synergy has been clearly demonstrated in artiﬁcial bacterial promoters between CRP bound upstream of the promoter and either another CRP or lambda cI protein bound to a site overlapping the 235 element ( 11 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Full rhaBAD activation requires the AraC family activator RhaS ( bound to a site that overlaps the 235 region of the promoter ) and the cyclic AMP receptor protein ( CRP ; bound immediately upstream of RhaS at 292.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Based on the results of Busby and coworkers ( 3 , 22 ) , we anticipated that the CRPK52N protein likely exerted its effect when bound to a CRP recognition site centered ; 41.5 bp upstream of the start point of fucPIK transcription for both the wild-type and IS5-disrupted promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>With regard to the speciﬁc role played by CRP bound to site 1 , it is unlikely that the protein acts by preventing an aggregation phenomenon since its stimulatory effect does not increase with the concentration of MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Because this low-affinity CRP site also appears to be dependent on the supercoiling of the DNA , and the negative supercoiling of the DNA decreases as the cells reach stationary phase ( Travers and Muskhelishvili , 2005 ) , the affinity of CRP III would be expected to be at its lowest when IHF approaches its highest concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>By competing with H-NS for the region of PhlyE downstream of the FNR/CRP site , SlyA may prevent the formation of the negatively acting H-NS -- PhlyE complexes by blocking a primary interaction between H-NS and PhlyE , allowing FNR and CRP to continue to activate hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Mutationalchanges , with respect to the parentalsequence , are double underlined.I I I Wild-typeCRP bindingsite ; ~ m , JH , Derivative of CC-CC ansB Ps containingtwo symmetry-relatedpoint mutationsthat change each CRP half site ; ~ FNR bindingsite ; Derivativeof FF-FF anaBPecontainingtwo symmetry-relatedpoint mutationsthat change each FNR half site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Such an inhibi-tory interaction might involve the CRP dimer bound at site 1 and/or one or more additional CRP dimers bound further upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>One possibility is that the CRP molecule bound to site 1 or/and one of the MalT protomers distally bound facilitate the initiation process , either by directly contacting the polymerase or by aiding one of the MalT protomers bound downstream to interact with the enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In contrast , when CRP was ® rst bound , the addition of Lrp followed by a 20 minute incubation displaced CRP from its site , resulting in full occupancy by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We conclude that CRP bound to this site regulates expression in both directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>In the cya mutant strain , CRP would not be bound to its site , and AraC would be the only activator of araBAD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>We then sought to determine if CRP site 1 was required for the ability of CRPK52N to activate both the IS5-disrupted and the wild-type fucPIK promoters under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>When the CRP site is misoriented relative to its wild-type position within araBAD , CRP can act ' ually enhance the looping interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Therefore , although we expected the CRP site in the CRP 3 construct to be optimally positioned for CRP coactivation with RhaS , there was no increase in CRP coactivation compared with the wild-type CRP position at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Another possible way that CRP and RhaR could bind cooperatively would be if CRP could bind to site 2 but only cooperatively as part of a complex with RhaR and CRP at site 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>As such , we hypothesize that the four phased A tracts may contribute to the ability of CRP to activate well from such a distant site at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Thus , the question remains whether this low-affinity CRP site functions in vivo and , if so , whether that function is positive or negative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>d Sequences corresponding to the original CRP upstream half-site in PguaB ( CRP 106.5 ) ( positions 125 to 121 ) or the original CRP downstream half-site in PguaB ( CRP 128.5 ) ( positions 114 to 110 ) were inactivated by substitution with the sequence 5 - GACTG-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Activation absolutely requires that one dimer binds CRP I , the proximal , higher-affinity site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , and as expected from its low affinity for CRP , site 4 was still dispensable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>Moreover , occupancy of this site by CRP is required for GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Furthermore , we showed in vitro that disruption of the AR2 surface results in a different structure of the TOC , similar to the one observed at low WT CRP concentrations , when CRP II is less likely to be occupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In vitro results suggest that it could also occur on the WT promoter whenever CRP concentrations are low and , hence , CRP II remains unoccupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Results from experiments using a low-copy plasmid-borne clyA promoter-lacZ fusion were consistent with the idea that both CRP and FNR are involved in the transcriptional regulation of clyA under anaerobic-growth conditions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>To determine whether CRP and KdgR were involved in one of these modes of regulation , activities of setA - lacZ and sgrS - lacZ fusions were monitored in two different host strains : host 1 , an sgrR host in which stress was induced by addition of MG to cul tures , and host 2 , a sgrR host complemented with a stress signal-independent sgrR allele ( i.e. , an sgrR mutant that does not require stress to activate the sgrS promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>To investigate the role of CRP and adenylate cyclase in catabolite repression of prpR or prpBCDE by glucose and glycerol , we compared the activities of the prpBCDE and prpR promoters ( with the PprpBCDE-rfp or PprpR-lacZ reporter fusion ) in strains lacking CRP or adenylate cyclase activity ( crp and cya mutants ) to promoter activity in the parent strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>If repression of ODC synthesis by CAMP-CRP occurs directly at the speC promoter , the amount of ODC synthesized in minicells would be expected to decrease with increasing concentrations of exogenously supplied CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>Thus , high-er-than-normal levels of CRP do not result in repression of guaB promoter activity relative to that observed in wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Upon further increase in CRP level , however , the level of b-galactosi-dase activity deceased again ( Fig. 4B ) , indicating the repression of pck promoter at high concentrations of Single dot complex Double dot complex CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>When FIS and CRP were added in combination , the KMnO4 reactivity of the fis promoter 210 element and especially that of bases at 2101 and 2102 was strongly reduced , suggesting a cooperative effect ( Fig. 3B , compare lanes 2 , 3 , and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Increasing concentrations of CRP up to 45 nM inhibited the transcription of crp and activated the divergent promoter ( Fig , 2 , lanes 2-4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Indeed , we observe that in vitro CRP greatly augments the repressing effect of low FIS concentrations on fis promoter activity ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Thus , a combination of FIS and CRP at concentrations that themselves have little , if any , effect on transcription , strongly inhibit fis promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>As expected , addition of increasing FIS concentrations gradually decreased the transcription of the fis promoter ( Fig. 5 , upper panel , compare lanes 1 -- 4 ) , whereas under the reaction conditions CRP alone had no detectable effect ( Fig. 5 , compare lanes 1 , 7 , and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>However , when these same CRP concentrations were combined with a low FIS concentration ( corresponding to that in lane 2 ) , which alone had only marginal effect on transcription , the activity of the fis promoter was strongly inhibited ( Fig. 5 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>A CRP-independent lac Mutant Is Repressed by CRP -- When the CRP-independent lacP1-6 was changed to become 35 ele-ment-independent by converting it to an extended 10 ele-ment-containing promoter , lacP1-6S1 , it remained highly active without CRP and a 35 element ( Fig. 1A , lane 13 and Fig. 1B , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>It should be noted that CRP , when present at low concentrations ( Fig. 6 , lane 3 ) , bound more efﬁciently to the udpPEc , udpPSt , udpPYp promoters than to the udpPV c promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The fact that CRP * overproduction did not suppress this effect excluded the possibility that it was due solely to a decrease in the level of active CRP protein , the activator of the malT promoter , observed in hns and hns-stpA strains ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The activity ot carnitine dehydratase was dependent on the CRP regulatory protein and strongly enhanced in the absence of a functional H-NS protein , in relation to the consensus sequences detected in the promoter region of the cai operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7815937" target="_blank">7815937</a>]<br></li>
<li>Using a malTpl malTplO promoter that is CAMP-CRP ( CAMPreceptor protein ) - independent , it was demonstrated that CRP is essential for malT pH , regulation and that the pH,-dependent activity of malKp is a direct consequence of malT regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>The Mechanism of Synergy between CRP and FIS in Repressing fis -- After the shift-up the modulation of fis promoter activity by CRP occurs concomitantly with the accumulation of FIS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Therefore , it is surprising that this promoter requires araC protein and CRP for significant activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6319708" target="_blank">6319708</a>]<br></li>
<li>TABLE II The effect of single base-pair substitutions at the extended 10 motif and 10 element on promoter activity and CRP regulation Promoters lacP1 -- 6S1 15T 3 G 14G 3 T 12T 3 G 11A 3 T Transcription CRP 100 60 50 70 20 Percentage ( % ) a CRP 120 a The amount of the transcripts was normalized to the percentage of that of lacP1 -- 6S1 in the absence of CRP , which was taken as 100 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The second line of data shows that , in the absence of crp , MelR induces low levels of expression from the melAB promoter on the KK43 fragment ; expression is less than 10 % of that seen in the presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The first line of data shows expression from the melAB promoter in the presence of both MelR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The scarcity of transcripts as well as the very low expression of 13-galactosidase from the galS-lacZ fusions , even in the presence of CRP , suggests that it is a poor promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>The presence of a promoter-external CRP target has not been observed in other CRP/CytR regulated operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The accumulation of abortive RNA in the presence of CRP suggested that CRP provides a barrier to promoter clearance in both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The presence of CRP on this site , which prevents FIS access to it , suppresses FIS repression of the promoter P3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The results showed that PcspD is transcribed more efﬁciently in the presence of CRP [ ( 3.3 0.24 ) - fold ] ( Fig. 7 ) , although the increase in the promoter activity in vitro is not commensurate with the fold increase observed for in vivo assays ( Fig. 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>The activity of the CRP + promoter increases ~ 1.5-fold in the hns mutant ( from 865 to 1275 units ) to the level of the D , CRP + double mutant promoter , which is not affected by H-NS ( Fig. 2C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The response pattems further indicated that the pdh and Ipd promoters are activated by CRP whereas the ace promoter is severely repressed by CRP ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>The final , somewhat unusual case is the papB promoter , in which adjacent CRP and PapB sites activate from a far-remote position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The overlap of the FIS and CRP sites could certainly explain why FIS represses the P3 promoter only when CRP is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The CRP - CTD contact may further increase the strength of one or more of the RNAP-promoter contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The decreased effect of this substitution at F ( rhaS-lacZ ) D216 is consistent with the reduced synergism between CRP and a-CTD at this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>The tdc promoter is unusual , as it appears to be regulated by both a LysR-type regulator and CRP , as well as being responsive to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The malT promoter is much weaker than the malTp1 mutant , but it is activated to a much greater extent by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>The activity of the Ep92 promoter still depends on MalT ; its dependence on CRP has not been tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>The effects of CRP and supercoilingon the activity of switches fromP l a toP l b , and transcriptionincreases from POa pte promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>In contrast , the activity of the rplM promoter increased in the crp deletion mutant ( Fig. 5E ) , implying that CRP is involved as a repressor of this rplM operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In the context of the present work , the K52N substitution presumably renders CRP a more potent activator that can function independently to activate a promoter whose activity ordinarily depends on multiple activators working in concert .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>In the absence of CRP , promoter activity remains low throughout the growth cycle , showing that the requirement for CRP for expression from the aer regulatory region is independent of the effect of CRP on s28 levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In the absence of CRP , P exhibited guaB an increase in activity at low growth rates relative to that in wild-type cells , and there was no longer a positive correlation between the promoter activity and the growth rate ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In good agreement with the reduction of rmf promoter activity , we observed decreases in RMF production and 100S ribosome dimerization in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In cells lacking CRP , therefore , the gua promoter maintains constant low activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>In the absence of FIS , CRP activates fis tra-nscription , thereby displacing the polymerase from the divergent promoter , whereas together FIS and CRP synergistically repress fis gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>In low-osmolarity medium , this promoter is strongly repressed by the CRP protein ( Xu and Johnson 1997b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In this report , we provide evidence that CRP functions directly as a negative regulator of the P1 promoter to maintain a low basal expression under low or standard medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>In contrast , FIS repressed expression of the hupB-P3 promoter only in the LB glucose medium when the activity of CRP was low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>In reﬂecting the physiological role of GapA as a real GAPDH , the promoter organization is very unique , being controlled by two global regulators , CRP and Cra , for a large set of genes for carbohydrate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>In addition to the repression by YjjQ shown here , transcription initiation at the ﬂhDC promoter is known to be controlled by OmpR , CRP , RcsA-RcsB and RcsB , HfdR , DksA / ppGpp , LrhA , and MatA ( EcpR ) ( 37 , 39 , 41 -- 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>In addition to being controlled by H-NS , StpA , LeuO , and BglJ-RcsB , the bgl promoter is CRP dependent and is repressed by FIS ( 4 , 48 , 52 ) DISCUSSION BglJ , a LuxR-type transcriptional-regulator , and LeuO , a LysR-type transcriptional-regulator , relieve repression of the bgl operon by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>In addition to being controlled by H-NS , StpA , LeuO , and BglJ-RcsB , the bgl promoter is CRP dependent and is repressed by FIS ( 4 , 48 , 52 ) ACKNOWLEDGMENTS This work was supported by a grant from the Deutsche Forschungsgemeinschaft ( Schn371/10 -1 ) and by the Cologne Graduate School of Genetics and Functional Genomics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>In contrast , the FIS-dependent repression of the hupB-P3 promoter was observed only when CRP was absent or inactive ( Figure 5 , Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>In contrast to FIS CRP activates the gyrA promoter and increases the supercoiling level of reporter plasmids [ 47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>A crp mutation increases the basal level of setA - lacZ activity ( Fig. 4A and B ) , but not the activity of sgrS - lacZ , suggesting there may be another promoter controlling setA expression that is repressed by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>These results suggest that CRP negatively regulates proP expression and point to the Fis-independent P1 promoter as the target of repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>It is now clear that CAMP and CRP regulate ODC expression as a function of the speC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>CRP Activates the fis Promoter in the Absence of FIS -- To evaluate the functional importance of the observed interactions , we first investigated the effect of CRP on the open complex formation at the fis promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>We thus conclude that in the absence of FIS , CRP can activate the fis promoter by facilitating open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>We showed that , in vivo , the melAB promoter carried by the JK19 fragment is fully active , dependent on melibiose and MelR , but independent of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>We concluded that the clyA promoter is dependent on CRP also during anoxic growth conditions and that FNR to some extent can complement the requirement for CRP during anaerobic-growth only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Perhaps this or other features of the fucPIK core promoter help account for the fact that the wild-type fucPIK promoter can not be activated by CRP in the absence of FucR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>This is consistent with our observation that in the absence of CRP and FNR , addition of SlyA only leads to partial derepression of the hlyE promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Here , we report that the malI promoter is active in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>Expressionof the E. coil ansB promoter,CC-FF , in the presenceand absence of CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Finally , the third line of data shows an important control ; expression in every case is minimal in the absence of MelR and CRP alone is unable to activate the melAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Measurements of the expression of this fusion in strain M182 , and in its Dcrp derivative , show that , in contrast to the malX promoter , the malI promoter is active in the absence of CRP , with a modest 1.7-fold stimulation by CRP ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>Furthermore , even when CRP works as an activator , part of its role is to repress ineffective secondary promoters which would otherwise reduce the efficiency of the main promoter ( Malan and McClure , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2664421" target="_blank">2664421</a>]<br></li>
<li>Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner & Schleif , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Does the location of the CRP box around 268.5 70 contribute to the strong Es selectivity of this promoter in S vivo and even in vitro ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>We infer that FIS and CRP synergistically repress the fis promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Effect of CRP and FIS on transcription of the fis promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>To investigate whether KdgR and CRP directly or indirectly regulate sgrS promoter activity , we again utilized the plasmid carrying the sgrR1 mutant allele .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>For example , at the uhpT promoter , CRP binds at position 2103.5 and acts in conjunction with the uhp-speciﬁc activator , UhpA , bound at position 264 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>CRP bound at -40 , it would prevent RNA polymer-ase access to the -35 of P2 and thus inhibit SI starts from the proposed hybrid promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>With the JK19 fragment , the changes in Site 20 result in a promoter where the requirement for MelR bound at Site 1 and Site 10 , and for CRP bound between Site 1 and Site 2 , is short circuited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Therefore , with CRP bound at the promoter , the dependence on the B-C loop of Fis appears to be alleviated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>When Lrp was ® rst bound to the promoter , the addition of CRP followed by an incubation of 20 minutes had no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Cyclic AMP-CRP binds to the ompB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In lac , CRP binds between - 74 and - 50 ; in ara between - 106 and - 82 ; and in gal , cat and in promoter P4 on the plasmid pBR322 it binds around -40 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>In these cases , CRP binds to the-35 region of the respective promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>Models of the Fis-aCTD interaction and the proP P2 promoter architecture under Fis and CRP co-activating conditions are presented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Of the 188 operons activated by CRP , the promoter-proximal gene was activated in the vast majority ( 176 operons ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Ligand fishing shows that ArcA as well as CRP specifically binds to the promoter regions of the pts operon and the ptsG gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>cspA promoter , not known to be regulated by CRP ( our unpublished results ) , is used as a control in these assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>This lack of a difference in promoter activities supports the notion that the loss of activity due to G126C is CRP dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Here we present evidence that CRP represses the xseA promoter while activating the guaBA promoter in vivo following downshift from rich to minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>These experiments demonstrate that CRP represses the xseA promoter and activates the guaBA promoter in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>IHF can bind simultaneously to the acs promoter region with either CRP or FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Our ﬁndings suggest that FNR and CRP bind to the same sequence in the clyA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Potentially , CRP could both repress and activate the malI promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>However , note that CRP could restore activity to the debilitated promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>This is in contrast to the glpE promoter , which responds weakly to CRP w2x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>FIS had essentially no action on this promoter in LB when CRP was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>CRP and MalT act alone or in combination , depending on the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Binding of CRP to the gntT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Binding of CRP to promoter fragments in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Future efforts will be directed to understanding the synergy between MelR and CRP , and how melibiose affects the MelR ± RNAP interaction at this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Model of fis promoter regulation by FIS , CRP , and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Therefore , the contributions of a-CTD ± UP-element interactions and a-CTD ± CRP interactions to promoter activity were suggested to be independent and additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>C. b-Galactosidase activity from promoter fragments pacs444 , pacs236 , pacs205 , pacs155 or pacs236 variants defective for CRP I , IHF III or CRP II fused to lacZYA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Two dimers of FNR bound in tandem can activate a mutant derivative of ansB Pe in a co-dependent manner A mutant derivative of ansB Pe in which the upstream site had been changed from a CRP to an FNR site , i.e. FF-FF , was functional ( Table 3 ) and , as expected , CRP-independent in contrast to the native promoter , CC-FF ( Jennings and Beacham , 1993 ; see Fig. 1B and Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>With these strains , the level of lacZ expression was low irrespective of the form of CRP introduced ( Table 5 ) , thus showing that the partial induction of lacZ expression seen in cells harboring the wild-type promoter in cis to the inactivated CRP site 1 was dependent on FucR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>At the araFGH promoter , however , the orientation of the sites is reversed as compared with those of the other ara promoters ; the CRP site is adjacent to that for RNA polymerase , and AraC binds upstream of CRP ( Fig. 1 ) ( 29 ) , whereas at araBAD AraC binds adjacent to polymerase and CRP binds upstream ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>These ﬁndings supported a model in which the clyA promoter is dependent on CRP for efﬁcient expression , and where the predicted DNA site for CRP is important for this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>It is more complicated to investigate the DNA site for CRP located 41-bp upstream of the melR transcript startpoint ( CRP2 ) because its inactivation leads to loss of melR promoter activity ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>Here we show that CRP binds to the putative CRP site located upstream of P , whereupon it serves to decrease promoter guaB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Active CRP clears the promoter and binds to its recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>Thus , coactivation by Fis and CRP is absolutely dependent on the helical phasing between CRP and Fis bound at site I within the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Ushida , C. , and Aiba , H. ( 1990 ) Helical phase dependent action of CRP : effect of the distance between the CRP site and the - 3 5 region on promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Moving the CRP site to position -61.5 ( aer211 ) results in a twofold decrease in promoter activity , but , while the residual promoter activity is dependent on CRP , it is independent of s28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The results showing lower occupancy of FIS site II in the absence of CRP both in vitro and in vivo indicate that CRP acts synergistically with FIS in repressing fis promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The results illustrated in Fig. 4 show that , when the DNA site for CRP is moved to position -41.5 , aer promoter activity in the CRP + FliA + strain is reduced to a similar level to that observed in the absence of CRP or s28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Both the deletion , which moved the DNA site for CRP to position -44.5 ( aer226 ) , and the insertion , which moved the DNA site for CRP to position -54.5 ( aer227 ) , resulted in a reduction in promoter activity in the CRP + FliA + strain to the basal level observed in the absence of CRP ( Fig. 4 , lower two panels ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Indeed , the presenceof a large number of MalT molecules in the overproducing cells would be expected to result in the substitution by MalT of the CRP molecules bound to the malEp-malKp regulatory region , leading to the formation of the less active nucleoprotein complex containing only MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>Furthermore , no transcripts were observed for malEp and malKp even when active complex formation was allowed to proceed for 30 minutes or , as tested for malKp , when MalT and CRP concentrations were simultaneously increased four - and 2 + fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>These curves represent , true kine-tics of active complex formation and not slow steps preceding this process , since a longer preincubation of MalT and CRP with the template did not change the rate at which active complexes appeared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>More attractively , this phenomenon may be directly related to a crucial role that supercoiling might play in the formation and stability of a functional pre-initiation complex involving MalT , CRP and malEp-malKp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>CRP may repress in this case by cooperating with the gal repressor which binds immediately adjacent to form a repression complex ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Regulation of the fatty acid pathways is complex and involves transcriptional-regulators ( including FadR , FarR , IclR , and CRP ) and post transcriptional mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>CRP complex acts as a positive control element for P1 and as a negative factor for P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>Three positive effects were detected , however ; at malT , CRP increases the residence time and therefore the stability of the open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>This further indicates that the study of complex regulons , such as CRP , in vivo has many limitations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>This reduced cooperativity would diminish the stability of the ternary CRP -- RNAP -- acsP2 complex and , thus , provide an opportunity for unbound RNAP to-bind to free acsP2A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Thus , the CRP -- RNAP complex can successfully compete with this concentration of IHF for access to the acsP2 promoter , while RNAP alone can not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>It appears that CRP can stabilize the closed complex enough to compete successfully with IHF and , thus , form a TOC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>A response to glucose repression only under anaerobic conditions is not expected for typical CRP-cyclic AMP-dependent regulation , and the signiﬁcance of the CRP site has to be tested experimentally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>Inactivation of the crp or cyaA gene or ablation/translocation of the CRP site relieves repression by CRP and results in a loss of GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Several of the mutations in the CRP site reduce activity to levels below those observed for glucose repression and only slightly above the uninduced basal level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>Our results therefore showed , except for the yad operon , that the different operons studied are positively regulated by CRP and are subjected to CRP-mediated carbon catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>These results demonstrate that propionate metabolism is subject to catabolite repression by the global transcriptional-regulator CRP and that this regulation is effected through control of both the regulator gene prpR and the prpBCDE operon itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In addition , the CRP effect is dependent on a functional gcvA gene and its role may be to antagonize GcvA 's repression of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In contrast to the situation in Pseudomonas and other Gramnegative bacteria where the mechanism of catabolite repression is not yet understood ( 34 , 35 ) , the effect of glucose in E. coli is mediated through the action of CRP ( 23 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>These lacked both the P , and CRP sites and did not show increased CRP-CAMP-independent lac expression when tested under a variety of growth conditions ( Vu and Reznikoff , 1985 ) , and the experiments were interpreted as negating the P , repression model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>During exponential phase , repression by Lrp , CRP , or IHF can not account for the low level of expression because similar low expression was found in mutants deficient for these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>One of these , distal to tsp , is responsible for the sensitivity to catabolite repression as indicated by the reduced level of expression of strain JA141 , which lacks part of the enhancing CRP element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8757746" target="_blank">8757746</a>]<br></li>
<li>Since CRP plays a major role in the glucose repression of many genes , we also tested whether CRP was required for the expression of the P3P2 -- lacZ fusion especially in an mnmE mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>When E. coli is growing in glucose minimal medium , nanA expression is barely detectable due to inactivation of CRP and repression by NanR ( 4 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Because the CRP is known to be a key mediator of catabolite repression in E. coli , we further investigated its potential role in sfm regulation by monitoring expression of sfm in a Dcrp strain and a double mutant strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Assuming that P-galactosidase activity tmly reflects AcnA synthesis , it would appear that a c n A is a typical citric acid cycle gene , subject to catabolite and anaerobic repression mediated by the corresponding regulators , CRP and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>At the bgl operon , which is controlled by HNS , StpA , CRP , and LeuO , BglJ -- RcsB and CRP presumably act synergistically to overcome repression by HNS ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Previously , Charpentier & Branlant ( 1994 ) identiﬁed four in vivo promoters , of which gapA P2 is recognized by RpoH sigma and gapA P3 is subject to catabolite repression by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Physiological experiments are consistent with clearing activity even during permanent catabolite repression since even under these conditions substantial active CRP is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>A number of strategies exist for breaking glucose catabolite repression , by using , for example , a constitutively active CRP allele ( 8 ) or by disrupting the glucose PTS ( 16 , 19 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>These effects may be indirect , and it is concluded that neither CRP nor FruR is responsible for the observed repression by glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Northern blot analysis revealed a single , monocistronic dctA transcript and conﬁrmed that dctA is subject to regulation by catabolite repression and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Repression was also mediated via CRP , as evidenced by the lack of repression in a strain carrying a crp lesion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>CRP is a global regulator of catabolite repression in E. coli ( reviewed in reference 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Thus , in a sense , the tandem CRP sites can be formally considered part of the repression apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Note that repression of ﬁs can be effected by FIS alone or by FIS and CRP acting synergistically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>This repression appears to be independent of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>Moreover , most members of these regulons seem to be under catabolite repression mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>CRP repression of azuC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>More often , CRP acts in conjunction with other factors to bring about repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Since repression by GcvA , GcvR , and Lrp is poorly understood , further elucidation of the roles of these proteins in the regulatory mechanism is necessary for further investigation into CRP 's role in controlling the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Conversely , if CRP 's role is to interfere with repression by GcvA , then CRP would be expected to regulate normally when the repressor function is intact but would have no role when GcvAF31A can not repress ( in a gcvR strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>However , the dependence of the CRP effect on the repressor function of GcvA suggests that its role is to antagonize GcvA-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>ACKNOWLEDGMENTS We appreciate the generosity of the following investigators for providing information and materials which were valuable to this work : Arnold Revzin ( Michigan State University , East Lansing ) for CRP ; Herbert Weissbach , Nathan Brot , and Tim Coleman ( Roche Institute of Molecular Biology , Nutley , N.J. ) for biochemical re-agents for the dipeptide assays ; and Sydney Kustu and John Keener ( University of California , Berkeley ) for ntrA , ntrC , and pJES40 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>In the absence of arabinose and CRP , AraC binds to aralT in an unlooped state on minicircles containing a deletion of ara0 , , and dissociates from the DNA with a half-time of two minutes ( Fig. 3 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>When CRP is added to looped complexes in the absence of cyclic AMP , only AraC-DNA complexes are observed , and these dissociate with a half-time , 40 minutes , characteristic of the looped state ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>As expected , the wild-type lacP1 DNA was distorted weakly at positions 1 , 11 , and 12 , whereas the lacP2 DNA was strongly distorted at its 10 , 11 , and 12 positions ( corresponding to 31 , 32 , and 33 of lacP1 ) in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>No bandshifts were observed in the absence of CRP or with negative control hofB DNA ( Figure 6 , lanes 1 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Effects of different crp alleles on the activities of crp1 and IS5-crp1 promoters in the presence or absence of inducer b-Galactosidase activity ( Miller units ) with : crp1 promotera IS5-crp1 promoterb Plasmid-encoded form of CRP CRP1 K52N 0.7 1.5 1.0 2.0 0.8 1.7 1.1 1.9 0.9 2.6 2.1 4.6 33 18 79 31 CRP H159L CRP K52N , H159L CRP TABLE 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>The activity of the rpsF and rpmI promoters decreased in the absence of CRP ( Fig. 5C and D ) , indicating the involvement of CRP as an activator of these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Except for the talA and fbaB promoters ( shown under gray background ) , the activity of all other promoters decreased in the absence of activator CRP , indicating the involvement of CRP as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Conversely and surprisingly , FIS has a negative effect on both hupB-P2 and hupB-P3 mRNA synthesis since the level of these two RNAs increased at least twofold in the absence of FIS when active CRP is not present Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Furthermore , in the absence of NarL , dcuB is repressed fourfold by nitrate in a manner that appears to be NarP independent , and , in addition , dcuB expression is reduced by glucose in a fashion that is mostly CRP independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Similarly , restoring CRP did not enhance hlyE expression , despite the absence of glucose ( Fig. 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In the absence of CRP , very little open complex was formed , and the modifications of the photoreactivity around 210 were too weak to determine the rate of open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>In the absence of melibiose , MelR and CRP bind to form a complex , but site 2 is not occupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>In stopped-flow kinetics , no obvious time-dependent increase in 2-AP fluorescence change was obtained in the absence of CRP ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>In a ﬁs + strain , active CRP helps to increase HUb to the level of HUa , whereas in its absence the surplus o Figure 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>In the absence of either MalT or CRP , less than 10 % and 5 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>In the absence of CRP , AraC dissociates from the minicircles with a half-time of 40 minutes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>In the absence of CRP , P0 is occluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>In the absence of CRP , however , most of these signals are lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Overall , the SFI levels were signiﬁcantly low in the absence of CRP during stationary phase , indicating positive regulation of PcspD by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>As evidenced by the lysis of erythrocytes in agar ( Table 3 ) , the absence of CRP , but not of FNR , reduced the level of clyA derepression in anaerobically grown hns strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Expression of dcuB is also repressed by glucose , is slightly repressed by TMAO , and in the absence of glucose is threefold induced by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>CRP is activated in the absence of glucose ( see below ) , leading to activate Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>It is well established that , in the absence of CRP , RNAP initiates transcripts at P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>Both templates are transcribed the to same extent in the absence of CRP ( lanes 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>with a relatively low afinity in the absence of cAMI ' - CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>However , the b subunit is more affected by the absence of active CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Mutant MelR that is active in the absence of melibiose has less need for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>PguaB activities were determined from the transcript abundance under each condition and are presented in the bar graph as mean activities ( with standard deviations ) relative to that for PguaB in the absence of CRP ( activity 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>malEp and malKp are inactive in the absence of CRP , but their activity can be partially restored by overproducing MalT ( Chapon , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>In the absence of the repressor site araO2 , the effects of CRP are reduced , suggesting at least some role for AraC in breaking repression ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>We compared the effects of the 3-base-pair deletion and 3-base-pair substitution in the CRP site to determine whether the deletion relieved araBAD repression in the absence of L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>In the two remaining cases of CRP repression , it is less easy to predict , from knowledge of position alone , whether CRP represses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>In the absence of CRP ( LB glu ) , both P2 and P3 are affected but the catabolite repression of P3 considerably lowers the level of P3 RNA , making the derepression less spectacular .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>We note , for example , that there are 39 Fis-regulated genes or operons listed in Table 2 that have been shown to be regulated by CRP , H-NS or GadE , of which 25 exhibit Fis-regulatory effects during the late-exponential-growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Although the data presented here confirmed the existence of a negative control of cyatranscription by CAMP-CRP , they do not exclude the possibility that the activity of adenylate cyclase is negatively regulated by CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>These confirmed that the acnA gene resembles other citric acid cycle genes in being positively regulated by the CAMP-CRP complex and negatively regulated by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>Although many more promoters are positively controlled by the CAMP-CRP system , several genes are also negatively regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3540312" target="_blank">3540312</a>]<br></li>
<li>These genes are induced by many different stress responses ( K.L. Jones and K.E. Rudd , unpublished ) , and several are regulated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Nevertheless , there are many examples of genes regulated by both FIS and CRP ( 13 , 18 , 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>In contrast , strong responses to anaerobiosis and glucose ( a source of pyruvate ) mediated by global regulators ( ArcA , FNR and CRP ) are likely to be less beneficial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>For ArcA and FNR , the cultures were grown anaerobically to A650 = 0.4 in L broth + glucose ( 11 mfui ) , and for CRP , cultures were grown aerobically to ASM = 0.8 In L broth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Thus , CRP enhances hlyE expression in response to glucose-starvation ( 36 ) and FNR enhances hlyE expression in response to oxygen starvation ( 10 , 11 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In summary , the data presented here suggest that FNR and CRP are positive regulators of hlyE expression in liquid culture in response to oxygen and glucose-starvation , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In addition to being repressed by CRP , AzuC-SPA levels were repressed under low-oxygen conditions , moderately induced during heat-shock , oxidative-stress , and thiol stress , and substantially elevated during growth in acidic medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>In contrast , CRP could positively regulate ykgR , for which lower protein levels were detected in glucose-grown cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>In contrast to wild-type cells , those lacking CRP did not induce Acs protein synthesis when shifted from glucose to acetate ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>In this respect it is noteworthy that the CRP protein activates most efficiently when tt binds at -- 41.5 and -- 61.5 whereas only weaker effects are observed at -51.5 and -52.5 [ 73 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>In addition to the strict control exerted by H-NS , the clyA locus appeared to be controlled by the global regulatory protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In Xanthomonadaceae , the CRP ortholog Clp ( CAP-like protein ) regulates the synthesis of extracellular enzymes , pigment and xanthum gum ( 46,47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In M182 crp cells harboring crp on multicopy plasmid , the SFI levels were higher compared to the cells with plasmid control thereby conﬁrming that CRP positively regulates cspD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In contrast , the effects of a crp mutation on operon expression were still observed when an upstream putative CRP box was deleted ( Results ; Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>In turn CRP can either activate ﬁs expression by itself or synergistically enhance the autorepression by FIS ( W. Nasser and G. Muskhelish-vili , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>In contrast to the situation when CRP was present ( Figs 3 and 4 ) , elimination of H170 had a reducing effect on csiD expression in the Dcrp background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>In contrast , acnB expression is activated by CRP and repressed by ArcA , FruR and Fis from PacnBC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>In the case of glucose , this regulation is thought to involve CRP ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>In this study , we identified some ABC-family transporter for carbon sources such as ugpB for glycerol-3-phosphate transport [ 56 ] and xylF for xylose transport [ 56 ] as the novel regulation targets of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In the context of pacs236 , therefore , CRP II has the opposite effect , acting negatively instead of positively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In contrast , IHF exerted no substantial effect on the affinity of CRP to either CRP I or CRP II ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In any case , CRP was found to regulate not only the genes for carbon utilization but also some ribosome-related genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In addition to CRP and NAC , feaR may also be repressed by PhoB ( 20 ) and ArcA ( P. J. Kiley , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>In the Zac operon , CRP binds co-opcrativel ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>In most promoters , CRP has a stronger affinity to the distal operator OCRPD [ 30 ] , with an exception being cddP , where CRP binds stronger to its proximal operator OCRPP [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>In a complementary experiment , we asked whether artiﬁcially increasing the intracellular concentration of CRP can inﬂuence GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In addition , overall geometry of the HTH α-helices signiﬁcantly deviates from that in the active CRP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Keywords small RNA ; Hfq ; CRP ; curli ; flagella tion uc trod In * For correspondence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>In contrast , inhibition can occur by overstabilization of the TOC by CRP as in the context of pacs236 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In contrast , the mechanism whereby MalT and CRP coactivate malEp remains elusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>In this study , 24 of these operons were shown in the ROMA experiments to be regulated by CRP and , therefore , were further analysed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In summary , these results dem-onstrate that gntT is positively regulated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>In the 17 CRP-activated operons , 9 operons are known members of the CRP regulon and 7 operons were activated by CRP in the ROMA experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In both group-B1 and group -- B2 cases , we tentatively estimated that CRP regulates one of the divergently transcribed genes or operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In three cases including caiTABCDE-fixABCX , malI-malXY and rhaBAD-rhaSR , however , the divergent promoters are regulated by different CRP molecules ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In addition , deletion upstream of BCRP , as in probe Pr-47 , showed the same afﬁnity for CRP ( Fig. 5C , lanes 4 to 6 ; Pr-47 ) as in Pr-46 ( Fig. 5C , lanes 1 to 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>In addition to being activated by CRP , YkgR-SPA accumulation was strongly elevated upon heat-shock ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Strain M182Acrp JRG1728 M182 Plasmid ( Acrp fnr + ) ( crp + Afnr ) ( crp + fnr + ) Introduction of CRP containing the substitutions HL159 ( or TA158 , data not shown ) restores very little expression , indicating the involvement of AR1 in transcriptional activation when CRP is bound at - 9 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>No signiﬁcant change of the transcriptional level of CRP was observed ( Figure 5C ) , suggesting that the induction of CRP-dependent genes might be mainly due to the activation of CRP rather than the increase in CRP expression ( 47 -- 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>For example , induction of the CFA/I operon by temperature requires the positive regulator CfaD [ 70 ] expression of the pap operon requires CRP and PapB [ 71 ] , the virB gene of Shi-gella is activated by VirF [ 49 ] and Fis plays a role in activation of the rrnB gene [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Since our previous ﬁnding on the anaerobic activation of mdtEF occurred in M9 glucose minimal medium in which the effect of CRP on gene expression is minimal , we focused on the effect of other regulators in the current study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>However , we found no stimulatory effect of CRP on metE-lacZ expression ; in fact , a crp deletion mutation in GS162 results in a threefold increase in metE-lacZ expression compared with the crp1 lysogen ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836282" target="_blank">7836282</a>]<br></li>
<li>These findings suggest that CRP functions in certain contexts to regulate gene expression in response to osmotic changes , in addition to its role in catabolite control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In addition , GcvA is known to require the gcvR gene product for its role as a repressor ( 12 ) , raising the possibilities that CRP also regulates expression of gcvR and indirectly alters gcvT-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In addition to this role , CRP also regulates gene expression in response to osmotic changes [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li>
<li>As CRP is not known to regulate hns expression or H-NS production , this suggests that CRP might exert its effect indirectly through another regulator that may act as a potential competitor of H-NS in regulation of expression of these ﬁmbriae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>To determine if one or more of CRP 's three activating regions may be involved in the regulation of gcv , four CRP mutants were tested for their ability to complement the crp deletion and restore expression of gcvT-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>To explore the origin of the indirect effect , we tested whether CRP was involved in regulation of rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Previous studies with fnr ' strains indicated that CRP and ArcA have no direct effect on aerobic or anaerobic ndh expression , and that NtrA ( a ^ ) enhances ndh expression about two - to threefold , but only under aerobic conditions ( Spiro et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>At intermediate osmotic upshifts , CRP can still function at some level to reduce P1 expression , but little effect of CRP is observed on initial P1 induction rates under severe osmotic upshift conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>However , clear glucose and CRP effects on dcuB expression were observed in the studies reported here , although the CRP effect was relatively weak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>To test this hypothesis , we added pHA7 * to the malT-lacZ fusion strains to see if excess active CRP levels would restore the levels of MalT expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>These data suggested that hlyE expression is activated by FNR and CRP , repressed by H-NS , and that H-NS-mediated transcriptional silencing is relieved by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Thus , in complete contrast to the observed catabolite sensitivity of ODH synthesis , suc expression appears to be repressed rather than activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Thus , the majority of the effects of CRP and FIS on gene expression are probably exerted under different growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Such cross-talk between FIS and CRP could provide selective advantage by allowing flexibility of transcriptional control and also increasing the precision of the coordination of cellular gene expression with changing growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Activation by CRP is absolutely dependent on AR1 since expression is abolished by CRP HL159 , but is restored by CRP HL159/KN52 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>These ﬁndings suggested that comC expression is modulated by the global regulators RpoE and CRP , in addition to the copper-dependent regulation by ComR shown here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22089859" target="_blank">22089859</a>]<br></li>
<li>However , when the downstream activator is CRP , then expression of FF-CC is observed with CRP alone ; that is , it is not co-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>It is necessary to have both MalT and the mutated form of CRP ( CRP * ) to induce mal gene expression in sfsl-dependent systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>However , CRP was shown to appear to act as a repressor [ 21 ] in the expression of the adenylate cyclase gene , cya .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>Thus , CRP and FNR positively control cai expression and H-NS exerts a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Thus , we suggest that the formation of different H-NS -- PhlyE complexes offers the opportunity for H-NS , in combination with FNR and CRP , to control hlyE expression both positively and negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Evidently , CRP , and not FNR , was required for the expression of ClyA in hns mutants under anoxic conditions , although it appeared that FNR could partly complement CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Involvement of both CRP and FNR in regulation of clyA expression during anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Thus , we concluded that CRP , FNR , and H-NS all contribute towards the regulation of hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>However in the M 182dcrp background , no melibiose-induced stimulation of expression is found , confirming that the CAMP-CRP factor is required .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>After induction of CRP expression by adding various concentrations of arabinose , b-galactosidase activity was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>It is possible that expression of aer and trg is required only in response to conditions that induce CRP activity and not in response to other signals that induce the ﬂagellar cascade .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Alternatively , expression of aer and trg may need to be upregulated to a greater extent than other genes when CRP activity is induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>It has been shown that the three proteins ChbR , CRP and NagC regulate the expression of the chb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>A possible explanation for CRP 's dependence on GcvA is that CRP regulates expression of the gcvA gene and indirectly affects expression of gcvT-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>CRP requires the repressor function of GcvA to regulate gcvT-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>This feature accounts for the unusual effectiveness of CRP to activate expression of cytidine deaminase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This suggested that CRP might repress gadA and gadBC indirectly by regulating gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Second , CRP could indirectly regulate gyrA expression by means of a positive regulator directly controlled by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Finally , CRP could regulate the expression of other topoisomerase genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Effects of CRP , ArcA and FNR on acnA-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>CRP appears to negatively regulate gadX expression by virtue of its effects on RpoS production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Induction of acs expression absolutely required CRP I , while optimal expression required both CRP I and CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>CRP alone activates ﬁs expression Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>RhaS activates rhaSR expression , but CRP coactivation is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>CRP also represses gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>sfm expression is under catabolite expression mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>CRP regulates expression of gcvA but does not regulate gcvR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Effect of CRP on P1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Effects of CRP , SoxRS and Fur on acnB-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Effects of CRP and KdgR on setA and sgrS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>This plasmid , pCM1 17-314 , encodes a MelR derivative * To whom correspondence should be addressed + Present address : CSIC Estacion Experimental del Zaidin , C. Prof. Albareda 1 , 18080 , Granada , Spai ( MelR314 ) that has the sequence : Met-Ala-Ala-Asp-Thr-Asp - CRP t ¬ Thr-Phe-Gly-His-Leu added to the N-terminus of MelR303 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7784205" target="_blank">7784205</a>]<br></li>
<li>This regulation depends on the binding of CAMP-CRP to malTp and results in the pH , modulation of basal and induced activity of MalTMalTJMalT , equilibrium and pH,-dependent dependent promoters ( malKp , malEp , malPp and expression of malK malSp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>This affords an opportunity to study CRP-dependent regulation at a range of naturally occurring promoters , and to uncover novel mechanisms of regulation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>This difference by unrelated DNA sequences without changing the CAMP - corresponds to the five nucleotides difference observed by in CRP ' - dependent regulation of gal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>After Genomic SELEX screening of Cra , originally identified as the repressor FruR but later renamed to Cra ( catabolite repressor and activator ) [ 31 ] , its regulation targets suddenly increased more than 8-fold to 178 [ 30 ] , the number as many as that of CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>If the rest of 103 CRP targets that are listed in Regulon DB but has not been identified by the Genomic SELEX are added , the total number of CRP targets should increase to at least 378 promoters , leading to about 2-fold sudden increase in the regulation target of CRP ( 378/195 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>As we have demonstrated.that FIS regulation only occurs at those promoters that respond to CAMP-CRP , FIS might modulate the CAMP-CRP activationas well as repression of these promoters , depending on the amount of FIS pre-sent in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Whether negative regulation , in these instances , is a consequence of CAMP-CRP interacting directly with the promoters of these genes , or is mediated by a CAMP-induced repressor or ` super repressor ' is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>If , however , negative regulation was mediated by a repressor induced by CAMP-CRP , CAMP would have no effect on the amount of ODC synthesized in minicells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>One unique regulation mode is that a single and the same CRP molecule bound within intergenic regions often regulates both of divergently transcribed operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>However , we noticed such a unique feature of CRP regulation that CRP bound within an intergenic region often regulates both of the divergently transcribed genes or operons ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Thus , while tynA and feaB are coordinately regulated by FeaR , their expression is also ﬁne-tuned by other regulators ( CRP and NAC ) and by the regulation of feaR expression in order to optimize enzyme activities according to the nutritional environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>For some target genes , FNR is the only regulator , whereas most are regulated coordinately by FNR and the regulators ArcA , NarL , NarP ( nitrate/nitrite regulation ) , Fur ( Fe regulator ) , SoxRS and Q ( regulation by O2̇ ¯ ) , LexA ( SOS regulation ) , or CRP ( glucose regulation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Thus , factors other than FNR , NarL , and CRP also appear to contribute to the regulation of dcuB in response to oxygen , nitrate , and glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>For these conditions , we predicted that the glucose-responsive effects of CRP would be retained , whereas the positive regulation by FNR would be abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>We are currently examining possible interactions between Lrp and CRP as part of an analysis of the mechanism of Lrp regulation of the major dad promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Together with CRP , it is involved in the ( generally ) positive regulation of a number of catabolic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Lower panel : Most of the genes for the enzymes involved in glycolysis are controlled by Cra [ 30 ] while only three genes , fbaA , gapA and pgk , had been identified as the regulation targets of CRP ( shown in while under blue background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>We conclude that translation of cya is not regulated by CRP and that the posttranscriptional regulation of cyclase by CRP occurs at the level of enzyme activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Thus , the major posttranscriptional regulation of cyclase by CRP is exerted at the level of the enzyme activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Thus , BglJ -- RcsB is a pleio-tropic transcriptional-activator that coordinates regulation with global regulators including CRP , LeuO and HNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Therefore , the variability of CRP regulation at different operons makes it difﬁcult to design an in vivo microarray experiment to distinguish direct effects from indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>To determine the putative role of CRP and IHF in paa regulation , we have used both genetic and biochemical approaches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>However , induction of comC by copper via ComR is much more pronounced than the effects described for RpoE and CRP and we assume that copper-induction of comC is the principal regulation of this gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22089859" target="_blank">22089859</a>]<br></li>
<li>We therefore tested the effects of CRP and KdgR on regulation of sgrS and setA in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Therefore , sdhE does not appear to be subject to transcriptional regulation via ArcAB , FNR or CRP when it is co-transcribed with YgfX in the Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>We studied ﬁrst the regulation by FIS when CRP is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Osmotic regulation of CRP activity is not limited to proP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Transcriptional regulation of lsrR by LsrR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>Anaerobic regulation of clyA involves both CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>This provides for consistent CRP regulation of all feeder pathways leading to the ED pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>This regulation occurs , in part , through the actions of the transcriptional-regulators CRP , FNR , IclR , and FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Lane 1 , no protein ; lanes 2 to 6 , 5 , 10 , 25 , 50 , and 100 nM CRP dimer , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>CRP was stored in 1 M-KC1 and the final salt concentration in all the samples was adjusted to remain constant at 50 mm Cyclic AMP , L-arabinose and n-fucose ( when present ) were added to final concentrations of 200 PM , 100 mM and 100 mM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6319708" target="_blank">6319708</a>]<br></li>
<li>DNase I footprintingreactions were carriedout using the following concentrations of CRP dimers : lunes 1 , 0 nM ; 2 , 50 nM ; 3 , 100 nM ; 4 , 150 nM ; 5 , 150 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Lane 1 , 50 nM CRP ; lane 2 , 100 nM CRP ; lane 3 , 500 nM CRP ; lane 4 , 1 mM CRP ; lane 5 , no CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>When present in the assay , CRP was used at a final concentration of 500 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>As the CRP concentration was increased to 6 nM , protection of both regions became more clear from -- 71 to -- 100 and from +34 to +46 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>We provide evidence that CRP breaks the nraO,-aral , repression loop ill vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Results ( a. ) PM - ` - induced loop breaking in vitro We studied the ef-l'ect of CRP on the ara0 , - ara / 1 loop formed on supercoiled minicircles by examining the effect of CRP on the occupancy of aral , in tht : loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Thus , the conclusions from both in vivo and in vitro experiments on the effect of the CRP mutation on araC regulation are similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>Despite the effect of the carbon source on the expression of astCADBE , we could not demonstrate a clear involvement of CRP in vivo or in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>However , whereas FIS is primarily involved in the reorganisation of cellular metabolism on transition to exponential-growth in rich medium , CRP is most active under the conditions of exhaustion of glucose and its analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>Guiso and Blazy measured the amount of CRP under various growth conditions and found that the synthesis of CRP was not affected by the growth condition or by the carbon source ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>CRP is present predominantly in the cAMPcomplexed , active-conformation under non-catabolite-repressed conditions ( e.g. , growth with glycerol as the carbon source ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Genes involved in denitriﬁcation , such as nir , nor , and nos , are mainly regulated by two NO-responsive transcriptional-activators from the FNR/CRP family , NnrR in a-proteobacteria and DNR in b-proteobac-teria , and Pseudomonas spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>It was unexpected that the lsr and lsrRK operons , which are divergently transcribed , are controlled by the same transcriptional-regulators : CRP and LsrR ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>Many of the transcriptional changes observed were brought about by global TFs such as ArcA , FNR , CRP , IHF , pdhR and fur in ATPase and NOX ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>Notice that FIS bound to FIS II at the lowest concentration tested ( 0.28 mM ) without displacing CRP from either CRP I or CRP II ( Fig. 3A , lane 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>E , EcoRI ; H , H/fTdlll ; Hp , Hpal ; M , MM ; R , Rsal ; T , Taql divergent RNA ( 153 nt ) CRP ( nM ) 0 15 45 135 227 ¬ 181 ¬ - `` divergent HNA -- crp RNA 83 -- Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Further analysis using the 92-bp fragment ( 151 to 60 ) revealed that both the retarded bands showed an increase in intensity as the CRP concentration increased , while the intensity of the upper retarded band was always higher than the lower retarded band at all of the CRP concentrations ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Upon increase in CRP level by adding increasing concentrations of arabinose , the level of CRP increased as detected by immunoblotting ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>With increasing CRP concentration ( Fig. 3 , lanes 3 and 4 ) , protection appeared between positions 60 and 83 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Characterization of CRP 1 and CRP 2 As shown in Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>As shown in Fig. 7 , CRP concentration gradually increases with time , peaks at 175 min , and then decreases during the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>However , CRP induces a Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Asp700 ^ ACG + + CRP : - + FruR : ¬ ~ + + 1 2 3 4 5 6 ATG CRP -35 -10 CRP O pckA 300 343 Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>Increasing concentrations of IHF exerted little effect ( Fig. 5 , lanes 8 -- 10 ) , suggesting that IHF and CRP can bind simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Remarkably , total DNA was retarded with saturating concentrations of either CaiF or CRP ( Fig. 4A , lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Two divergent sets of transcripts originating around -87 and 1 + could therefore be observed ( +1 refers to the origin of the CRP activated transcript corresponding to the malT gene ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>As expected , transcripts of many genes were increased by the addition of CRP and transcripts of very few genes were reduced , which is consistent with the view that CRP activates most of the genes that it regulates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>( Note that the decrease in CC-CC is presumably due to wild-type CRP being present in lower amounts from crp in single copy , in the crp + host , rather than from the multicopy plasmid pDCRP ; e.g. see JerlstrSm et aL , 1989 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4245">
Operons that are not activated by HL159-CRP or regulated in a reduced level ( outliers ) are indicated by ` # ' , and operons that are activated by HL159-CRP at the same level as by wild-type CRP indicated by ` -- ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="act_similarity_4245", type="checkbox">
<div class="ACT_similar">
<li>Operons that are not activated by KE101-CRP or regulated in a reduced level ( outliers ) are indicated by ` # ' , and operons that are activated by KE101-CRP at the same level as by wild-type CRP indicated by ` -- ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li></div>
<li>It should be pointed out that , although both promoters function only in the presenceof CRP in a wild-type strain , they are partially active in a crpbackground provided that MalT is overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4248">
F , crp1 ; ■ , Dcrp39 ; å , Dcrp39 ( wild-type CRP ) ; ç , Dcrp39 ( CRP [ T158A ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></label>
<input id="act_similarity_4248", type="checkbox">
<div class="ACT_similar">
<li>E , crp1 ; h , Dcrp39 ; Ç , Dcrp39 ( wild-type CRP ) ; É , Dcrp39 ( CRP [ T158A ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_4250">
Lane 1 , wild-type gal , no CRP ; lane 2 , wildtype gal , wild-type CRP ; lane 3 , wild-type gal , [ Leu159 Glu101 ] CRP ; lane 4 , wild-type gal , [ Leu159 Glu101 Asn52 ] CRP ; lane 5 , G1A sequencing ladder ; lane 6 , galp19T , no CRP ; lane 7 , galp19T , wild-type CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></label>
<input id="act_similarity_4250", type="checkbox">
<div class="ACT_similar">
<li>Lane 1 , no CRP ; lane 2 , wild-1 type CRP ; lane 3 , [ Leu159 Glu101 ] CRP ; lane 4 , [ Leu159 Glu101 Asn52 ] CRP ; lane 5 , G1A sequencing ladder .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li></div>
<li>) When CRP EV181/HL159 is introduced , there is a sixfold decrease in activity in the case of XX-CC , as compared to CC-CC which is decreased by only 1.5-fold ( Table 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Weak protection is also evident in the region flanked by these sites as a function of increasing CRP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>However , because CRP does not bind to the mutated sites , hence there is no competition , this effect is not coupled to the interactions with the remaining operator sites and so need not be included in the model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus FNR could be converted such as to-bind specifically to CRP sites , CRP to-bind to FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Consistent with this proposal , we showed that CRP binds two sites , CRP I and CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Here we show that , like MelR303 , when MelR173 binds to sites 1 and 2 it recruits CRP to-bind between these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>As a result , we found that ArcA as well as CRP specifically binds to the two promoters ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>As references , some known CRP target promoters such as ptsG , dgsA , ptsH and glpF ( shown under white background ) were examined in parallel under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>By using the LacZ reporter assay , some of these novel CRP target promoters were confirmed to be under the control of CRP in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>These are CRP , repressing several promoters , and the activators FNR , MetR , PapB , and IlvY , repressing their own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>CRP is known to repress several promoters ( Mallick and Herrlich , 1979 ) and thus can not be considered solely as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2664421" target="_blank">2664421</a>]<br></li>
<li>Therefore , osmY remains repressed by the concerted action of CRP , Lrp , and IHF until the growth rate is further reduced below a certain limit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Whereas rs , Lrp , CRP , and IHF are clearly involved in stationary-phase induction , none of these regulators is essential for osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>However , CRP , Lrp , and IHF are not essential for osmotic induction of osmY ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>araBAD , araC , araE , and papB promoters are also activated by CRP ; narG is also activated by nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Interestingly , a few known FNR-regulated operons , adhE , sdhC , sodA , cyoA and focA , were also activated by CRP in the ROMA experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Furthermore the atpIBFHAGDC operon coding for the ATPase F1 and F0 complexes was also found to be under the control of CRP ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Stationary-phase-specific sigma RpoS is also under the control of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>It is consistent with CRP , of which A144T mutation also leads to the CRP phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>It is also probable that sequence searching generates many false negatives due to the ability of CRP to-bind variable sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>It has been also evidenced that only two molecules of cGMP , as its maximum , in the anti-conformation can bind to a CRP dimer [ 32,38,39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>One of these , mapped by the SF73 substitution , is possibly the functional equivalent of AR1 in CRP since , like T158 and H159 , it is required at -61.5 and ( to a lesser extent ) at -41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Therefore , a functional CRP is required for negative transcriptional control of the speC gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>This positive effect of H-NS was not observed under anaerobic conditions , presumably because FNR and not CRP occupies PhlyE under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>CRP and FixK can act like FNR as positive and negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Both FNR and CRP make positive contributions , whereas H-NS acts negatively under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>This template binds polymerase rapidly ( Stefano and Gralla , 1980 ) and has low CRP affinity ( Fried and Crothers , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>This upregulation was not surprising since gltA is known to be regulated by several regulators such as CAMP-CRP , acrA and fir ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16233096" target="_blank">16233096</a>]<br></li>
<li>This concerted conformational change of CRP dimer appears to require breaking/reforming of the salt E72 -- R82 bridge to facilitate productive CRP inter-domain communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>This result suggested that CRP is required for optimal induction of sgrS under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>This shows that MelR carrying these substitutions has a reduced requirement for CRP to activate pmelAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>This may be due to the use of multicopy pHA7E , which increased the concentration of CRP above that in W3110 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Structural and kinetic studies using this set of deletions are in progress in our laboratory to try to define the arrangement of polymerase and CRP in this zone during the formation of open complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>Lanes 2 to 8 represent runoff transcripts from samples taken at increasing times after the addition of CRP-CAMP It is clear that after the addition of active CRP the P , signal builds at the expense of the P , signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>However , since there does not appear to be a smooth inverse relationship between CRP abundance and growth rate , it is unlikely that the CRP concentration functions as a sensor of changes in the cellular growth rate for PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Sub sequent events , however , are dependent in part on the activity of FIS [ 28 , 29 ] , CRP and of other abundant nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4288">
This effect appears to be both CRP and 54 dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></label>
<input id="act_similarity_4288", type="checkbox">
<div class="ACT_similar">
<li>This effect appears to be both CRP and 54 dependent ( Fig. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li></div>
<li>GalS is itself autogenously regulated and CRP dependent , while the gene for GalR is constitutive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Only one run-off transcript was found to be dependent upon both AraC and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Experiments are in progress to determine whether CRP can function in an osmolarity-dependent fashion in other contexts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>CRP appears to control whether or not the σS-dependent induction mechanism will operate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>We have detected an upstream signal at -94 by UV-laser footprinting that is visible only when CRP is present ( P.Eichenberger , S.Dethiollaz , N.Fujita , A.Ishihama and J.Geiselmann , in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>We thank C. Park , B. Wanner , D. LaPorte , S. Maloy , J. Cronan , Jr. , R. W. Simons , and H. Aiba for strains ; R. W. Simons for plasmids and phage ; H. Wing for puriﬁed FNR DA154 ; N. J. Savery for puriﬁed CRP ; G. S. Lloyd for help performing the CRP gel shifts ; and H. M. Wols for construction of strain AJW1884 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>ACKNOWLEDGMENTS We are indebted to W. Ross , T. Gaal , H. Murray , and R. L. Gourse ( University of Wisconsin-Madison ) and to D. Browning , D. Grainger , and D. Lee ( University of Birmingham ) for puriﬁed CRP , strains , plasmids , and helpful discussions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>ACKNOWLEDGMENTS We are grateful to J. Krakow for puriﬁed CRP , J. Calvo for puriﬁed Lrp , and J. Fuchs and M. Lobocka for bacterial strains and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Acknowledgements We thank J. Krakow for puri ® ed CRP , J. Calvo for puri ® ed Lrp , and J. Fuchs and M. Lobocka for bacterial strains and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Other details are as for A Acknowledgements We are grateful to J. Krakow for puri ® ed CRP , and to J. Fuchs and M. Lobocka for bacterial strains and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Lanes A and F , molecular weight standards ; lanes B and E , cell extracts of strains M15 ( pNP-41 ) and M15 ( pNP-52 ) after IPTG induction ; lanes C and D , puriﬁed GntR and CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Some minor changes of the digestion pattern are , however , induced by CRP ( lanes 11 and 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Mutations in the 287 determinant disrupt cooperativity between CRP and a ; hence , it is proposed that these residues are required for interactions between CRP and - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Both mechanisms depend upon speciﬁc interactions be - tween CRP and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Thus , it is unlikely that direct interactions between CRP and RNAP play a role in GRDC of P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>the afinitv of & MT ' -- CRP for CRP-I and CRP-2 is increased ` 1OO-fold and IO-fold , respectively ( Pedersen it ( 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Relative to F ( rhaS-lacZ ) D128 , the synergism between RhaR and CRP increased at F ( rhaS-lacZ ) D216 ( to 36 - or 37-fold ) , and the synergism between CRP and a-CTD decreased ( to 3.2 - or 3.3-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>MetR MetR metH NR ( I ) , CRP ginApl ... ¬ -4 - II * NR ( I ) NR ( I ) a II I gInL NR ( I ) NFR ( l ) , ' OmpR OmpR O OmO pC pR momp R II OmpR OmpR OmpR ompF I OxyR OxyR ahpC  ' OxyROxyR   OxyR OxyR katG a a ~ ~ OxyR oxyR ' PapB I a PapB , CRP aPapB PapB papB , ' , +92 +125 PhoB phoA PhoB PhoB I Pho B PhoB PhoB phOB 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Likewise CRP was found to regulate NADH-ubiquinone oxidoreductase I encoded by the nuoABCDEFGHIJKLMN operon , NADH-ubiquinone oxidoreductase II encoded by ndh , and the cytochrome bo terminal oxidase encoded by the cyoABCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Using ROMA , we identified 176 operons that were activated by CRP in vitro and 16 operons that were repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Sequence analysis of the operons that were repressed by CRP revealed different mechanisms for CRP inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>For example , it will be of interest to consider whether or not CRP may act in direct cooperation with other factors under som conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Zn vivo , whereas there is ain crease in galPl and a decrease in galP2 in thepresence of CRP , there isa substantial amountof bothgalPl andP2 under eitherCRP + or CRP - conditions ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>Recently , we addressed this question by determining the intracellular CRP levels under various conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>A ﬁnal possibility is that CRP activity is altered under acid and heat-shock stress conditions and that this is responsible for the changes in AzuC and YkgR levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Addition of 1 pg of purified CRP restored the ability of CAMP to inhibit ODC synthesis in these reaction mixtures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>CRP further stimulated the abortive RNA synthesis in both ( Fig. 4 , lanes 2 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Its synthesis is stimulated by glycerol ( lanes 3 and 6 ) and inhibited by CAMP-CRP ( lanes 4 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6271763" target="_blank">6271763</a>]<br></li>
<li>CAMP -- CRP activates P1galE and represses P2galE ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Whether FNR binds at the vicinity of CRP and acts in competition with CRP or on its own will require a more thorough investigation of the 59 regulatory region of caiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>CRP binds in the cspD upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>CRP binds to the gcv control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>When both AraC and CRP were present , the entire region fl ` om - 2 9 to - 1 0 0 was protected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Binding of UlaR , CRP , and IHF to the ulaG-ulaA intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>any model must , take into account t , hat CAMI ' - ( XI and C'ytK bind on the same face of t , he helix , that ( * A > IP-CRP and C'ytR in & act direct ~ ly ( Scqaard-Anderson rt nl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>However , this substitution is analogous to G75C , the most severe mutation generated in CRP I. Therefore , we predict that it severely impacted the ability of CRP to-bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>GaIR and CRP bind independently to the ga/operon ( Dalma-Weiszhausz and Brenowitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>It is most likely , therefore , that CRP breaks the loop , and forces AraC to-bind to nra / in the unlooped state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>One-half of the reaction was then added to active CRP , and the other half to active CRP and excess competitor template ( see Figure 6 legend for the concentrations of components ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>For acsP2 , the most important signals appear to be those represented by the concentrations of CRP , IHF and FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>CRP concentrations were : solid circles , 75 nM ; open squares , 125 nM ; solid squares , 375 nM ; open triangles , 500 nM ; solid triangles , 2000 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>Determination of intracellular concentrations of AscG , CRP , and PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Final CRP concentrations : lanes 1 -- 3 , no CRP added ; lanes 4 -- 6 , 5310 -- 8 M CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CRP and cAMPwere added to final concentrations of 1 ng/pl and 1 PM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Therefore , we concluded that CRP prefers to-bind to CRP box-1 at low CRP concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>We then concluded that CRP binds to both CRP box-1 and box-2 at high concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Binding of CRP to s28-dependent promoters in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>guaB Effect of CRP on P activity in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Effect of CRP on PguaB activity in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP was added where indicated , and the occupancy of AraC at nral was examined as a function of t , he dissociation time by t , he restriction-cutting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>On the other hand , araBAD is only moderately regulated by CRP and can be substantially induced by arabinose-AraC alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>AraC , CRP I I I araBAD AraC AraC AraC I I AraC , CRP araC J ¬ I +100 I AraC AraC , CRP araE I l rI - t - Ag I I 3 ~ ~ ~ AA ~ rgR I argCBH ArgR I I !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>AB LE T Op Pr Reg Act Ada ada 0 0 0 1,2 126 AraC araBAD 1 1 0 1,2 59 , 70 AraC araC 1 1 0 1,2 48 AraC araE 0 1 0 2 145 CRP cat 0 0 0 2 72 CRP ccd 0 1 p 2,4 165 CRP colElpl 1 1 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>-6 O - -6 O - -5 O - -40 ¬ -3 o - -20 ¬ -20 -13 .25 IO-60 60 6060 - -13 .25 1.0 u ) 20 2.020 - - - - - -6 306.0 ( a ) CRP CYTR CRP - 50 50-CYTR - - 6-075 -505 - o - CYTR 150 ¬ - - 6.0 75 ( b ) Lc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>These data are thus in favour of a mechanism involving the synergistic action of CRP and CaiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>To conclude , it is important to stress the complexity of the mechanism by which MalT and CRP coactivate malEp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>CRP regulates the majority of genes for three pathways of sugar transport ( MFS , PTS and ABC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>FNR is a CRP homologue and regulates genes during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>[ 102 ] Egland , P.G. and Harwood , C.S. ( 2000 ) HbaR , a 4-hydroxyben-zoate sensor and FNR-CRP superfamily member , regulates anaerobic 4-hydroxybenzoate degradation by Rhodopseudomonas palustris .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>Conserved relative positioning of XylR and CRP in gamma Proteobacteria suggests close functional interaction between these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>gyrA-lacZ response to nutritional upshift and the effect of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Strikingly , the effect at P2DOWN was non-linear ( Figs 4B and 5 ) : modiﬁcations increased up to 12 nM CRP , above which they decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Schematic of allosteric conformational change suggested As a conclusion , our concept on the allosteric conformational change of CRP is schematically modeled in Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Schematic illustration of the allosteric conformational change of CRP , modeled in this paper .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>GatY is a reversible enzyme that catalyzes the synthesis of tagatose-1 ,6 phosphate from DHAP and GA3P ; it is regulated by CRP , ArcA and GatR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Firstly , CRP deficient cells which could not metabolize sugars such as lactose , arabinose , and maltose can be transformed by the constructed plasmids to use these sugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>However , when these mutants were combined with CRP , transcript levels were considerably increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>However , it is worth noting that a 20 minute preincubation of MalT , CRP and the template was necessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>However , CprK is distinguished from CRP in that it is deactivated under aerobic conditions , by an intramolecular disulﬁde bond formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>CRP , when the UTP concentration was lowered from 30 to 3 MM , contrary to that found previously for the trinucleotide synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>the amount ot ' the CRP st , imulation is affected hy the concentration of AraC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>, then altering the concentration of active CRP should change the rate of displacement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>CRP is present in lanes 8-11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>CRP is present in lanes 2 , 4 , 6 , 8 , 10 , and 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>CRP was added to the reactions shown in lanes 2 , 4 , 6 , and 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>CRP proteins are 0 , 0.2 , 0.8 , and 3.2 3 1029 M in lanes 1 and 5 , 2 and 6 , 3 and 7 , and 4 and 8 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>CRP was added at 0 , 0.25 , 0.5 , 1 , 1.5 , and 2 , uM in lanes 1 through 6 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Overexpression and puriﬁcation of GntR and CRP , demonstrated by SDS-PAGE of the puriﬁed His-tagged proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Regulator proteins such as CRP and ArcA were added to the reaction as described under `` Results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>Genes under light green background represent newly identified CRP targets while those under pale orange background represent the CRP targets listed in Regulon DB [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>ykgR is activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4371">
Operons that are still significantly activated by HL159-CRP ( outliers ) are indicated in boldface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="act_similarity_4371", type="checkbox">
<div class="ACT_similar">
<li>Operons that are still significantly activated by KE101-CRP ( outliers ) are indicated in boldface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li></div>
<li>Reactions contained 1 , ug of pOP12 and 1 , ug of exogenous CRP in a 35 - , lI final volume and were conducted as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>After Genomic SELEX , we found that CRP activates tktAB and talAB encoding the last step enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>RESULTS CRP both directly and indirectly activates rhaBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>CRP activates proP P2 through AR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>CRP activates P2 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>It will be interesting to explore the role of CRP , if any , in coupling cellular metabolic status and persistence phenomenon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Lanes : 1 and 6 , no addition ; 2 and 7 , 0.25 , uM CRP ; 3 and 8 , 0.5 , uM CRP ; 4 and 9 , 1 FM CRP ; 5 and 10 , 1.5 , uM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>CRP 2 was titrated with CRP in the range 10-90 ng while CRP 1was titrated only when the CRP range exceeded 100 ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>-35 -20 -10 +1 CRP ... , , , , , , , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>CRP was overproduced in JM109 containing pDCRP grown aerobically in LB medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>/ l ~ < vneri ; CRP St , Salmonella typhimurium ; CydR Av , Azotobacter vinelandii ; CysR Sy3 , Synechococcus species ; CysR Sy7 , Synechococcus species ; DNR Pa , Pseudomonas aeruginosa ; DNR Syn sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Folding pattern prediction software estimated no signiWcant diVerences between WT CRP and the R82Q , E72D and E72Q forms of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>CRP was added to a final concentration of 1 nglul .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CRP , when added , was at 150 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>These observations are consistent with the prediction that MaeIII should destroy CRP 3 and CRP 4 but not CRP 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>XMI ' - CRP antagonist , modu-Iattbs a I ) SA bend induc ~ rd by CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Role of CRP and its activating regions in GRDC of P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>¬ CRP was an increase in the maximal velocity of the reaction or an improvement of the Michaelis constants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Effects of ArcA and CRP on incorporation of [ 1 - C ] oleic acid into membrane phospholipid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>MalT MalT , CRP malK 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>and malKlam & maEM operons are controlled by both MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Protein bands corresponding to CRP were detected by autoradiography , and their intensities were quantiﬁed densitometrically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>b. CRP , and mutantderivatives , were introducedon pDCRP-derived piasmidsand expressiondeterminedin M182Acrphost cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>After 3 h of induction , cells were harvested and subjected to CRP purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>S 1 2 345 CRP-2 CRP-1 +1 b ) uranyl e ) DNaseI CRP - + + + RNAP - - + ¬ CRP + + + - + + + RNAP - - + - - + -0 ¬ CRP-2 FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>-70 1 -80 -80 3 - + + + - + + + -- + - -- + - + - - - + -- + ¬ CRP CypRWT - + + + -- + - CRP CYTRWT CYTRA9-4 ¬ - + C ' ~ TRA ~ -48 Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>U - 30K - CRP -21 K I - 14.3 K Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>B 2 0 1 CRP Gly CRP Glv ¬ E h Amp W Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>Thus , CRP can not act on its own in this system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>CRP was the kind gift of A. Revzin ( Department of Biochemistry , Michigan State University , East Lansing ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Purified CRP was the generous gift of James Harmon -LCB- Texas Tech University , Lubbock , Texas ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>on ara CRP Represses spf , cya , ompA , galp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Comparison between HI and CRP Complex Binding to the CRP BindingSite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>Binding of puriﬁed CRP to clyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>CRP was purified according to the procedure of Ghosaini etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>CRP Purification-Purification of CRP was performed according to Ghosaini etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>S ~ `` 0 $ + AraC + CRP - 0.25 3 9 + AraC - CRP 3 9 27 2 s !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>AraC AraC CRP araFG2 araFGl FIG. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>CRP stimulates the former and inhibits the latter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>CRP enhances initiation at SI and inhibits starts at S2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>RESULTS CRP positively regulates PcspE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>CRP regulates gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Involvement of CRP in GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Effect of CRP overproduction on GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Cells were grown using CRP media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>azuC is repressed by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Purified CRP was kindly provided by Dr. A. Kolb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Puriﬁed CRP was kindly provided by A.Kolb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>CRP was kindly provided by J. Krakow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>CRP was kindly donated by J. Krakow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>CRP Press Inc. , Boca Raton , FL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>Isolation of GntR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4425">
CRP as if it were S2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></label>
<input id="act_similarity_4425", type="checkbox">
<div class="ACT_similar">
<li>CRP , and that at S2 is inhibited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li></div>
<li>These genes were not previously known to be under the control of CRP , and in line with earlier studies , more genes were observed to be upregulated rather than downregulated by CRP ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>These more robust upstream regulatory analyses can be used to assign function to unknown genes , as illustrated here with the ArgR , CRP and PurR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11897025" target="_blank">11897025</a>]<br></li>
<li>Taken together , these results strongl suggest that CRP-S sites mediate induction of natural competence in V.cholerae by CRP and Sxy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>These results support the idea that , part , of the CRP stimulation of pRAD is derived from hindering the AraGDNA looping that , represses peAn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>These results indicated that CRP regulates P , and this regu-cspE lation is proportional to the crp copy number .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>These results indicated that overexpression of Sfsl could induce amylomaltase in a CRP * and MalT + host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>In these promoters , as has been shown for the mal promoters ( 115 ) , CRP may activate in concert with additional proteins bound to proximal positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Because inhibition depends upon both CRP bound at CRP II and an intact AR2 , we predicted that AR2 would have no effect in the absence of CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Analyses of the five CBSs revealed that CRP2 , 3 and 5 bound the CAMP-CRP complex with highest affinity , that CRP 4 bound activated CRP with slightly lower affinity , and that CRP 1 exhibited much lower affinity for activated CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>If so , then this complex probably consists of FIS and CRP bound at FIS II , CRP I and CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Step 1 , in the absence of FIS and CRP , two polymerase molecules are bound at the fis and divergent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>DNase I footprint analysis of CRP and FIS bound to the hupA and hupB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Binding of CRP results in two retarded complexes ( lanes 3 and 4 ) , which most likely correspond to one and two molecules of CRP bound to rpoH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>This argues strongly that the role of MelR bound at Site 1 and Site 10 , and of CRP bound between Site 1 and Site 2 , is primarily to ensure occupancy of Site 20 by MelR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>As mentioned previously , an alternative model is that Fis and CRP bound to their respective sites must ﬁrst interact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Thus , the PrfA 145 in Fig. 1 is regarded as a maturely active form comparable to the ligand-bound , active state of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>When bound at this position , CRP uses AR1 of the upstream subunit of CRP to contact determinant 287 of the - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>CRP is not bound adjacent to RNAP at rhaBAD ; hence , it is difﬁcult to imagine an identical interaction between CRP AR2 and a-NTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>No signiWcant diVerence in secondary structure content was observed between the apo and ligand bound forms of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>According to the above arguments , it is unlikely that CRP bound there could easily touch the polymerase directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>CRP then provides additional stabilizing contacts that increase the probability that polymerase will be bound during approach .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>Thus , tandem-bound MelR recruits CRP to the MelR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Dimethylsulphate footprints of MalT and CRP bound to malEp-wdKp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>DNase I footprints of CRP bound to the mutated malEpmalKp regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>Cooperat-ivity is pairwise when only one CRP is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Therefore , we suggest that when environmental conditions dictate a low intra cellular concentration of H-NS , the H-NS I region of PhlyE is occupied to modulate the activity of the upstream activator ( FNR or CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>To determine whether the acetate-dependent increase in acs transcription depends on CRP , we constructed a crp derivative of the lCB7 lysogen AJW1786 , grew the resultant strain and its parent in TB , and monitored their growth and b-galactosidase activity ( Fig. 4A and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>To guaB conﬁrm that GRDC of P requires CRP , the activity of guaB P ( 133 to 36 ) in relation to the growth rate was mea-guaB sured in a crp strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Thus , in a crp or cya mutant strain or a strain in which the PguaB CRP site has been inactivated , PguaB exhibits a marked increase in activity at low growth rates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>A protein band of the predicted size for the Fur ' - ' LacZ protein ( 118 kDa ) was apparent in the catabolite-activation-proficient strain E. coli JK935 and in the Acya mutant E. coli JK936 in the presence of cAMP or cAMP plus the iron chelator 2,2 ' - bipyridyl , while the same band is barely detectable in Acrp and Acya mutants ( E. coli JK937 and E. coli JK936 ) without additions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2836193" target="_blank">2836193</a>]<br></li>
<li>Plasmid pST1021 was co-transformed with pKU100 , pKU101 or pKU102 , respectively , into the cya mutant TP2006 and the cya crp double mutant TP2339 -- 1 containing pLG339CRP separately , and b-galactosidase activities were measured in the presence or absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In the presence of cAMP , glnA expression was much lower than in the absence of cAMP in the cya mutant TP2006 and the cya crp double mutant TP2339 -- 1 containing pLG339CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>pLG -- rpoN was cotransformed into the cya crp double mutant TP2339 -- 1 together with pKU101 and pDCRP , and glnAp2 expression was monitored under nitrogen deﬁciency in the presence or absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>pMJ220 was co-transformed into the cya crp double mutant TP2339 -- 1 together with pKU101 and pLG339CRP , and glnAp2 expression was monitored in LB 1 medium supplemented with 40 mM NH4 in the presence or absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The specific activity of , - galactosidase in the Acrp strain ( see Table I ) was the same when cAMP was present in the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6325162" target="_blank">6325162</a>]<br></li>
<li>The expression offl-galactosidase in strains MC1061 ( Alae ) , SB0 ( Alac ) , SB2 ( Alac , Acrp ) and TP2006 ( Alac , Acya ) , containing pAA-X or p N M - X plasmid fusions , was analysed when cells were grown in the presence of glycerol , glucose or glucose + cAMP ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>In both Acrp and Acya mutants , the derepression of , - galactosidase synthesis in the absence of glucose did not EC1631 cya + crp + 820 236 EC2098 + cya Acrp 185 214 EC2126 Acya crp + 213 152 a Bacteria for enzyme assays were grown for 4.5 h in Luria-Bertani medium supplemented with 0.5 % glucose , when indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>Strains UP1000 ( ara + ) and LA2001 ( ACRP ) exhibited catabolite repression in that growth in the presence of glucose decreased enzyme levels 70-80o when compared with cells grown in the presence of glycerol ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>Using NagC extracts from the Acrp strain , in the presence or absence of CAMP , results in the same pattern of protection and hypersensitive DNase I bands as observed with the JMlOl extract minus CAMP ( Fig. 6 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>In M182 cells , harboring either pDU9 or pDCRP , the SFI levels followed similar pattern as in the parent cells ( M182 only ) except that the levels were marginally lower in the presence of pDCRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In line with the earlier results , cspD expression was greater in the presence of pXZCRP than in the presence of the plasmid control ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>But , in the presence of its activator , cAMPWCRP , the abortive RNA synthesis is reduced while the full-length RNA synthesis is enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760580" target="_blank">10760580</a>]<br></li>
<li>Strains containing plasmids derived from pLG339 ( i.e. , pLG339 BS , pLG339CRP , pLG339CRP159L , and pLG339CRP 101E ) were grown in the presence of 25 g/ml kanamycin , and strains containing plasmids derived from pBR322 ( i.e. , pDU9 , pDCRP , and pHA7 ) were grown in the presence of 100 g/ml ampicillin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">CRP protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>In strain LA923 , the three bases deleted by ACRP were substituted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>While the alignment of the nupC upstream regions of five closest relatives of E. coli contains conserved regions , they have very low OCRP scores .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Second , there might be a specific three protein , CRPzCytRzCRP complex that is asymmetric with respect to its interactions with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In such case , the cAMPWCRP may recruit the RNA polymerase into a productive form selectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760580" target="_blank">10760580</a>]<br></li>
<li>Methods Sample preparation The His6-tagged 1-138 domain ( CAPN ) was amplified with the polymerase chain reaction ( PCR ) , using as a template the pAKCRP plasmid that encodes for wild-type E. coli CAP ( residues 1-209 ) , and transformed in BL21 ( DE3 ) cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16906160" target="_blank">16906160</a>]<br></li>
<li>subunit : involvement of ttie C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476160" target="_blank">7476160</a>]<br></li>
<li>Igarashi , K. , and Ishihama , A. ( 1991 ) Bipartite functional map of the E. coliRNA polymeraseCL subunit : involvement of the C-terminal region in transcription-activation by CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_8">
Bipartite functional map of the E. coli polymerase a subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8206845" target="_blank">8206845</a>]<br></label>
<input id="dom_similarity_8", type="checkbox">
<div class="DOM_similar">
<li>Bipartite functional map of the E. coli RNA polymerase alpha subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9440505" target="_blank">9440505</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990294" target="_blank">8990294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512708" target="_blank">9512708</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9737918" target="_blank">9737918</a>]<br></li>
<li>Bipartite functional map of the Escherichia coli RNA polymerase a subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Bipartite functional map of the E. coli RNA polymerase a subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10515932" target="_blank">10515932</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209042" target="_blank">9209042</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10715008" target="_blank">10715008</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836282" target="_blank">7836282</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209035" target="_blank">9209035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751307" target="_blank">7751307</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171412" target="_blank">9171412</a>]<br></li>
<li>Bipartite functional map of the E. coli RNA polymerase subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Igarashi K & Ishihama A ( 1991 ) Bipartite functional map of the E. coli RMA polymerase alpha subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Igarashi , K. & Ishihama , A. ( 1991 ) Bipartite functional map of the E. coli RNA polymerase α subunit : involvement of the Cterminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Igarashi , K. , and Ishihama , A. ( 1990 ) Bipartite functional map of the E. coil RNA polymerase ~ subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8825088" target="_blank">8825088</a>]<br></li>
<li>Igarashi , K. , and Ishihama , A. ( 1991 ) Bipartite functional map of the E. coli RNA polymerase Q subunit : Involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Igarashi K & Ishihama A ( 1991 ) Bipartite functional map of the E. coli RNA polymerase alpha subunit : involvement of the C ¬ terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>[ 25 ] Igarashi , K. and Ishihama , A. ( 1991 ) Bipartite functional map of the E. coli RNA polymerase K subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>Igarashi , K. and Ishihama , A. ( 1991 ) Bipartite functional map of the E.coli RNA polymerase a subunit : involvement of the C-terminal region in transcription-activation by cAMP ± CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Igarashi , K. , and Ishihama , A. ( 1991 ) Bipartite functional map of the E. coli RNA polymerase α subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207706" target="_blank">12207706</a>]<br></li>
<li>Igarashi , K. , and Ishihama , A. ( 1991 ) Bipartite functional map of the E. coli RNA polymerase a subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11555297" target="_blank">11555297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Igarashi , K. , and Ishihama , A. ( 1991 ) Bipartite functional map of the E. coli RNA polymerase a-subunit : involvement of the C-terminal region in transcription-activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9791185" target="_blank">9791185</a>]<br></li></div>
<li>Thus , the extended -10 motif of ParaF is not sufficient for constitutive activity ; AraC , and possibly CRP , must still provide some structure for efficient polymerase recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>Even though the protection pattern between positions -40 and -54 is altered slightly when CRP is added to the binary complex containing the truncated version of RNA polymerase , the two hyperreactive bands at -54 and -65 no longer appear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The delayed formation of the upstream contact with RNA polymerase is seen also at malT ,24 where it is formed uniquely in the presence of CRP , and occurs subsequent to open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054813" target="_blank">12054813</a>]<br></li>
<li>Furthermore , potassium permanganate probing shows that the conformation of the open complex in the presence of CRP appears qualitatively and quantitatively different from that in the absence of CRP , suggesting that contact with RNA polymerase is maintained throughout the transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>This refers to interactions between CRP and the N-terminal domain of the R-subunit ( RNTD ) of the polymerase that increase the rate of open complex formation ( 14 , 15 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>( b ) Activation factors of protein mediated DNA ( b ) Template Activation factor DNA-binding protein H159L CRP Supercoiled CC-55 .5 Supercoiled IC-55 Linear CC-55 .5 Linear IC-55 1.6 3.8 IHF CRPH159L bending in vitro , on supercoiled or linear templates with aCTD ( C-terminal domain of the a subunit of RNA polymerase ) - truncated Es54 in the absence or presence of CRP or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>First , both CRP1-bound CRP and CRP2-bound CRP contact the C-terminal domain of the R-subunit of the polymerase ( RCTD ) to facilitate its binding to the intervening DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Location of RNA polymerase a C-terminal domains at the aer promoter Activation by CRP at both Class I and Class II s70-dependent promoters requires a contact between CRP activating region 1 ( AR1 ) and aCTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>CRP binds at position -41.5 and overlaps what in many promoters would be the -35 contact region for RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>At Class I promoters , CRP binds upstream of the promoter 35 element and a single activation region ( AR1 ) makes contact with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Given the promoter geometry and a wealth of genetic and biochemical data ( 16 -- 20 ) , the 260 signal most likely arises from a direct contact between the downstream subunit of CRP and RNA polymerase ( 21 ) , more specifically between activating region 1 of CRP and the carboxyl-terminal domain of the a-subunit of RNA polymerase ( 22 -- 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>At Class I promoters , CRP binds upstream of the promoter -35 element , at a site centred at position -61.5 ( i.e. between base pairs 61 and 62 upstream from the transcript start ) , or further upstream , and an activating region ( AR1 ) in the downstream subunit of the CRP dimer makes contact with the C-terminal domain of one of the two RNA polymerase a subunits ( aCTD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>AR1 in the upstream subunit of the CRP dimer interacts with aCTD , while a second activating region ( AR2 ) in the downstream subunit interacts with the N-terminal domain of one of the two RNA polymerase a subunits ( aNTD ) ( Busby and Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>AR1 in CRP is mapped by amino acid residues T158 and H159 and is required at all Class I promoters and , to varying extents , at Class II promoters ( Bell et al. , 1990b ; West et aL , 1993 ; Zhou et aL , 1993a , 1994b ) ; it interacts with a contact site in the C-terminal domain of the alpha subunit of RNA polymerase ( see Ishihama , 1992 ; Ebright , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Likewise , mutants of RNA polymerase subunits have been used to establish that the C-terminal domain of the alpha ( a ) - subunit is the target for activation from class I promoters and both the oc - and sigma ( a ) - subunits interact with CRP at class II promoters ( Zou et al. , 1992 ; Ishihama , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>According to the model , one of the CTD monomers mediates protein -- protein contacts between a surface activation region ( AR1 ) on CRP and the subunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>1 The abbreviations used are : RNAP , RNA polymerase ; CRP , cyclic AMP receptor protein ; CTD , the carboxyl terminal domain of the subunit ; CTD , the 1 -- 235 mutant RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The C-terminal domain of the R-subunit of the RNA polymerase makes direct contacts with a specific region of CRP at certain promoters ( Ishihama , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>In support of this hypothesis , we demonstrated that acs transcription requires the surfaces of CRP and the C-terminal domain of the subunit of RNA polymerase holoenzyme ( - CTD ) , which is known to participate in class I interactions : activating region 1 of CRP and the 287 , 265 , and 261 determinants of the - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>In vitro transcription assays indicate that the carboxy-terminal domain of the alpha subunit ( - CTD ) of RNA polymerase is at least partially dispensable for RhaR-dependent activation but that the - CTD is required for CRP activation of rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>CRP activation at the rhaSR operon was shown to require the RNA polymerase ( RNAP ) - subunit C-terminal domain ( - CTD ) ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>To address the mechanisms of activation by CRP and the RNA polymerase a-subunit C-terminal domain ( a-CTD ) at rhaSR , we tested the effects of alanine substitutions in CRP activating regions 1 and 2 , overexpression of a truncated version of a ( a-D235 ) , and alanine substitutions throughout a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Here , we deﬁned the minimal system required for this IHF-dependent inhibition , showing it requires the promoter-distal CRP site and an amino acid residue located within activation region 2 ( AR2 ) , a surface determinant of CRP that interacts-with-RNA-polymerase ( RNAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_46">
McLeod , S.M. , Aiyar , S.E. , Gourse , R.L. and Johnson , R.C. ( 2002 ) The C-terminal domains of the RNA polymerase alpha subunits : contact site with Fis and localization during co-activation with CRP at the Escherichia coli proP P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></label>
<input id="dom_similarity_46", type="checkbox">
<div class="DOM_similar">
<li>The C-terminal domains of the RNA polymerase alpha subunits : contact site with Fis and localization during co-activation with CRP at the Escherichia coli proP P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885445" target="_blank">16885445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The C-terminal domains of the RNA polymerase a subunits : contact site with Fis and localization during co-activation with CRP at the Escherichia coli proP P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866514" target="_blank">11866514</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li></div>
<li>How does RNA polymerase interact with the promoter ( s ) in the presence or absence of PhoB and/or cAMP-CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The P1 open complex must contain both CRP and RNA polymerase , which , together , can protect the promoter region as far as - 7 0 from DNAase attack .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>DNase I footprinting analysis of the interaction of CRP and RNA polymerase with the Pa -- Pr promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Our recent in vitro studies using mutant RNA polymerases consisting of C-terminal truncated a subunits provided furtiier evidence for direct contact between RNA polymerase and CRP ( Igarashi and Ishihama , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Thus , the surfaces of MelR adjacent to the C-terminally located HTH 2 are likely to be involved in any contact with the RNA polymerase s subunit , whereas surfaces adjacent to HTH 1 are placed so that they could interact with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Both require direct contact between CRP and the ~ - subunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>( 1991 ) , thereby enabling one or more CRP dimer ( s ) to contact RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>First , certain amino acids of CRP have been shown to be part of three activation domains ( AR1 , AR2 , and AR3 ) that interact with RNA polymerase ( RNAP ) to facilitate transcription initiation ( 3 , 4 , 7 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Partial disappearance of the bands in this area could also be Involvement of CRP activating regions AR1 and AR3 in cai and ® x transcription To determine further whether direct contact between CRP and RNA polymerase is involved in transcription-activation of both cai and ® x promoters , the effect of amino acid substitutions in CRP was investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Conclusions Here we have described the ﬁrst examples of direct activation by CRP of promoters served by RNA polymerase holoenzyme containing the ﬂagellar sigma factor , s28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The direct contact between CRP and RNA polymerase therefore does not appear to contribute to transcriptional activation at the malT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>Thus , the nucleoprotein complex responsible for activation from -63 appears to be distinct from the activation complex of the -60.5 promoter ( possibly the opposite face of CRP making contact with the RNA polymerase , see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>This resembles the position of a cAMP-CRP binding region of the E. coli lac promoter , - 61.5 ( 47 , 51 ) , which was found to be one of two ( the other being at - 41.5 ) that are optimal for the contact of CRP with RNA polymerase that is required for CRP-mediated transcription-activation ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>The interaction between CRP and RNA polymerase has been shown to play a critical role in the activation of transcription in CRP-dependent promoters ( 6 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>The orientation in which MelR binds to site 2 and site 2 cents has important consequences , as it deﬁnes the surfaces of MelR that can make contact with the RNA polymerase s subunit and CRP during transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>P1 activation depends on protein-protein contact between the RNA polymerase a subunit and CRP ( 13,19 , also reviewed in 20 ) , while P2 transcription is repressed by CRP binding ( Fig. 6 , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>Protein-protein interactions involving opposing negatively and positively charged residues have frequently been reported and have recently been described for CRP and RNA polymerase ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>A very long stretch of amino acids in a completely extended conformation would be necessary to explain almost equal activation factors when CRP is placed at -60 or -80 solely on the basis of identical protein-protein interactions between CRP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_67">
At CRP-dependent promoters , CRP activates transcription by making direct protein -- protein contacts with RNA polymerase ( RNAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="dom_similarity_67", type="checkbox">
<div class="DOM_similar">
<li>It is known that at CRP-dependent promoters CRP activates transcription by making direct protein -- protein contacts with the RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li></div>
<li>At the malT promoter , closed complex formation is efficient and CRP primarily stimulates promoter escape through backside contacts with RNA polymerase and/or by destabilizing RNA polymerase-promoter interactions that impede escape ( e.g. interactions between the ct-subunit and DNA just upstream of -35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>These could either be contacts to RNA polymer-ase itself or CRP induced changes in DNA structure providing an improved contact surface for polymerase to bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>This protein should bend the DNA to a similar degree as does CRP ( Kosturko et al. , 1988 ) without making the exact same proteinprotein contacts with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Because the only detectable interactions between CRP and RNA polymerase at this stage of the reaction are direct contacts , we propose that these contacts improve the rate of isomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>At class II promoters , transcriptional activation is due almost exclusively to favour-able protein-protein contacts between different RNA polymerase subunits and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Once bound , the predominant method of activation by CRP appears to be through protein contacts with RNA polymerase ( RNAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>At some locations the mechanism of activation involves direct protein-protein contacts between CRP and the RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Alternatively , CRP could act through specific protein-protein contacts with either MalT or the RNA polymerase holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>We conclude that the protein-protein contacts between CRP and RNA polymerase are rather rigid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Direct protein -- protein contacts between CRP and the RNA polymerase appeared very rapidly , followed by DNA melting around the 210 hexamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>Once bound to DNA , CRP makes protein -- protein contacts with RNA polymerase and recruits it to promoters to initiate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Although we have argued that some element in the sequence from - 4 5 to - 5 6 is necessary for stable CRP binding , we can not exclude the possibility that RNA polymerase also makes contacts in this zone that are essential to stabilize the open complex at P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>This demonstrates that , in both cases , AR1 in CRP is responsible for direct interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>In this case it is possible that the system that we have used , where the repression loop can not be destabilised by CRP or RNA polymerase binding , allows such a weak interaction to be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>In the case of CRP , this suggests that this activation involves neither an a-CTD interaction nor cooperative binding with RhaR , while in the case of RhaR , this suggests the likelihood of direct interactions with core RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>It is currently not ` known whether activation requires a protein-protein interaction between the CRP and RNA polymerase or cAMP-CRP acts at a distance by changing the structure of the DNA near its binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>The mechanism of transcription-activation appears to rely entirely on contacts between the DNA upstream of the CRP binding site and the back of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>Such protein-protein contacts depend on a precise spatial alignment of the components ; generally , we find strong peaks of activation whenever the CRP binding site is in the correct helical phase with respect to RNA polymerase ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>In araFGH and several other systems , the CRP binding site appears to be positioned so that CRP could make direct contact with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>At PFGH , CRP bends the DNA and , based on the overlap in binding sites , direct contact with RNA polymerase may occur within or near to the DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>On the other hand AraC protein could easily make direct contact with polymerase at araE and araBAD , but CRP might obstruct contact in the araFGH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>RNA-Polymerase-CPR-PromoterComplex-In terms of protein DNA backbone phosphate contacts , the RNA-polym-erase-CRP-promoter initiation complex shows a striking resemblance to the CytR-CRP repression complex in the -30 to -65 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Without this DNA bend , AraC protein alone can not contact polymerase ; however , , the combined bending caused by CRP and AraC protein ( unpublished results ) may allow AraC directly to contact a second region of polymerase and further stimulate its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>AraC contacts polymerase at a position similar to that of CRP at PFGH , and a DNA bend is produced that allows CRP to contact polymerase at the second region to provide additional stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>For example , the glycerol reaction contained three times as much Amp RNA as the CAMP-CRP reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>It is most likely that CAMP-CRP and RNA polymerase make direct contact in open ii Assays were performed as described in the legend to Table I b O.OSY ~ / m , , elibiose ( Sigma ) was included in the medium complexes and mutually tighten binding ( Ponnambalam et al. , 1987 ; Busby and But , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Thymine residues located in the ÿ10 region of the csiD promoter are reactive only with EsS with or without CRP , whereas those at positions ÿ93 , ÿ92 , ÿ88 , ÿ85 and ÿ86 react only with Es70 in the absence of CRP csiD promoter binding by RNA polymerase containing either sS or s70 Our in vivo data ( Figure 1 ) indicated that csiD expression is nearly completely dependent on sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Our data indicate that CRP or RNA-binding to the loop are not essential for GalR dimer cooperation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>In this case , activating region 1 ( AR1 ) of the downstream monomer of CRP contacts the RNAP - CTD to recruit RNAP to the promoter and activate transcription ( reviewed in reference 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In this situation , contacts are made between a second activating region on CRP , AR2 , and the N-terminal domain of a ( a-NTD ) ( 21 , 27 , 29 ; reviewed in references 5 and 6 ) , as well as between CRP AR1 and a-CTD ( 32 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>To activate class II promoters , where CRP binds immediately adjacent to RNAP , CRP AR2 is necessary for positive interactions with the amino-terminal domain of the a subunit ( 21 , 27 , 30 ) ( reviewed in reference 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>CRP AR2 is proposed to interact with the a subunit Nterminal domain at class II CRP-dependent promoters ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_101">
The interaction at class II CRP-dependent promoters also involve interaction between a second region called AR2 of CRP ( H19 , H21 , E96 , and K101 ) and N-terminal domain ( - NTD ) of the RNAP - subunit ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="dom_similarity_101", type="checkbox">
<div class="DOM_similar">
<li>The interaction at class II CRP-dependent promoters also involves interaction between a second region , called AR2 , of CRP ( H19 , H21 , E96 , and K101 ) and the N-terminal domain ( - NTD ) of the RNAP subunit ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_103">
Activating region 1 ( AR1 ) of CRP , composed of amino acid residues 156 to 164 , interacts with speciﬁc surface determinant , 287 , of C-terminal of alpha subunit ( - CTD ) of RNAP in both class I and class II promoters ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="dom_similarity_103", type="checkbox">
<div class="DOM_similar">
<li>Activating region 1 ( AR1 ) of CRP , composed of amino acid residues 156 to 164 , interacts with a speciﬁc surface determinant , 287 , from the C-terminal domain of the alpha subunit ( - CTD ) of RNAP in both class I and class II promoters ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>At class I CRP-dependent promoters , homodimeric CRP binds DNA sites centered near positions 61.5 , 71.5 , 82.5 , and 92.5 with respect to the transcription start site , and the downstream CRP monomer stimulates transcription by contacting the C-ter-minal domain of the RNAP subunit ( CTD ) through a surface-exposed loop known as activating region 1 ( AR1 ; residues 156 to 164 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>At these promoters , CRP activates transcription by interacting with the C-terminal domain of the RNAP a subunit ( aCTD ) via a surface-exposed patch , known as activating region 1 ( AR1 ) ( residues 156 -- 164 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>At these promoters , CRP makes multiple interactions with RNAP , including between AR1 and aCTD , the activating region 2 ( AR2 ) ( residues 19 , 21 and 101 ) and the N-terminal domain of a subunit ( aNTD ) and between AR3 ( residues 52 -- 58 ) and the RNAP s70 subunit region 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP activation at class I promoters involves proteinprotein contacts between a surface-exposed loop on CRP activating region 1 ( AR1 ) , and the carboxyl-terminal domain of the subunit ( - CTD ) of RNAP ( 31 , 35 , 36 ; reviewed in a a reference 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>CRP AR1 is necessary for activation at class I CRP-depen-dent promoters , where CRP binds upstream and not adjacent to RNAP , and interacts with a deﬁned set of amino acids on the carboxyl-terminal domain of the RNAP a subunit ( a-CTD ) ( 33 , 38 , 39 ) ( reviewed in reference 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>When CRP binds to these sites , its activating region 1 ( AR1 ) contacts RNAP 's a subunit C-terminal domain ( aCTD ) to recruit RNAP to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>A class I mechanism refers to interactions between CRP and the C-terminal domain of the R-subunit ( RCTD ) of RNAP that serve to recruit RNAP to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_112">
CRP bound at any of these positions uses a deﬁned surface ( activating region 1 [ AR1 ] ) in the downstream subunit of CRP to contact a speciﬁc surface determinant , 287 , of the C-terminal domain of the subunit of RNA polymerase ( - CTD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></label>
<input id="dom_similarity_112", type="checkbox">
<div class="DOM_similar">
<li>CRP bound at any of these positions uses a deﬁned surface ( activating region 1 [ AR1 ] ) in the downstream subunit of CRP to contact a speciﬁc surface determinant ( 287 ) of the C-terminal domain of the - subunit of RNAP ( - CTD ; gray double oval ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li></div>
<li>When bound at this position , CRP uses AR1 of the upstream subunit of CRP to contact the 287 determinant of the - CTD and a second sur - face ( AR2 ) to contact the 162 to 165 determinant of the N - terminal domain of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>CRP uses a second surface ( AR2 ) of the downstream subunit to contact the 162-165 determinant of the N-terminal domain of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>When bound at a class I promoter ( at some distance upstream of the promoter 235 region , as for the lac promoter ) , the CRP dimer makes a single contact with RNAP by using residues in a solvent-ex-posed loop known as activating region 1 ( AR1 ) in the promot-er-proximal subunit to contact the C-terminal domain of the a subunit ( a-CTD ) ( Fig. 3 ) ( reviewed in reference 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>When bound at a class II promoter , wild-type CRP makes two contacts with RNAP by using AR1 in the promoter-distal subunit to contact the a-CTD and AR2 ( small black rectangle ) in the promoter-proximal subunit to contact the a-NTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>However , when bound at a class II promoter ( immediately adjacent to the promoter 235 region , as for the gal promoter ) , wild-type CRP makes two productive contacts with RNAP ( 17 ) ( Fig. 3 ) : ( i ) it uses a second activating region ( AR2 ) in the promoterproximal subunit to contact the N-terminal domain of the a subunit ( a-NTD ) and ( ii ) it uses AR1 in the promoter-distal subunit to contact the a-CTD ( which in turn associates with the DNA upstream of the DNA-bound CRP dimer ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>To deter mine if CRP is contacting the a-CTD at proP , we mapped the binding position on the proP promoter of a truncated a subunit containing only the last 85 amino acids comprising the Cterminal domain ( 20 ) by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>This subunit consists of an N-terminal domain ( a-NTD ) containing determinants for interaction with the remainder of RNP and with CRP , a C-terminal domain ( a-CTD ) containing determinants for interaction-with-DNA and with various activators , and an unstructured / ¯ exible linker connecting the two subdomains ( Blatter et al. , 1994 ; Negishi et al. , 1995 ; Jeon et al. , 1997 ; Meng et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>A recent model proposed that a flexible linker connecting the C-terminal domain and the N-terminal domain of the oc-subunit could extend to maintain the interaction with the activator when the CRP binding site is moved further upstream ( Zhou et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Second , allosteric interactions between the CRP1-bound CRP and the R N-terminal domain accelerates initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Our studies show that both domains of MelR are involved in cooperative interactions ; the C-terminal domain in interactions with CRP , while the N-terminal domain is required for interactions between MelR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Current understanding on the allosteric process of the Escherichia coli CRP activation can be summarized into a rigid-body movement that involves subunit realignment and domain rearrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>It is known that interaction between the C-terminal domain of the subunit of RNAP , CTD , and CRP is the sole basis of activation at lacP1 ( 10 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Direct activation arises from a second CRP : ~ z-subunit contact , this time with the N-terminal domain , O ~ NTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Similar experiments indicate that the C-terminal domain of the σ-subunit is involved in activation by CRP at the gal promoter ( Kumar et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Each of these DNA sites permits a CRP dimer to recruit RNAP to the acsP2 promoter by contacting the C-terminal domain of each of the two subunits of RNAP ( - CTD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>CRP is proposed to activate transcription by making direct protein-protein contacts with the C-terminal domain of the a-subunit of RNAP , thereby directing interactions between RNAP and otherwise nonspecific DNA sequences ( 64 ) , substituting for the UP promoter element found in some particularly active genes ( cf. Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At these promoters , CRP interacts with the C-terminal domain of the RNAP subunit ( CTD ) , facilitating the binding of CTD to the DNA segment adjacent to the CRP-binding site ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>CRP enhances the transcription of lacP1 , which is an intrinsically weak 35 element-containing promoter , by increasing both RNAP binding and isomerization , both of which involve a specific contact between CRP and the carboxyl-terminal domain of the subunit ( CTD ) of RNAP ( 10 , 12 , 14 , 15 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Full activation by RhaR requires contact with 70 ( RhaR D276 contacts 70 R599 ) , and a rhaSR promoter containing a RhaR binding site , but not a cyclic AMP ( cAMP ) receptor protein ( CRP ) binding site , requires the carboxy-terminal domain of the alpha subunit ( - CTD ) of RNAP for full activation ( 15 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>CooA is distinguished from CRP in that its NTD contains two additional parts : a heme molecule as a prosthetic group to sense CO and an N-terminal extension of which a part provides an internal ligand to the heme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>It seems plausible since CRP does not have additional regions stabilizing interdomain interaction , such as the N-terminal tail in CooA and CprK and the C-terminal helices in CprK and PrfA ( refer to Section 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>We reasoned that alanine substitution of a-CTD residues that were involved in speciﬁc interactions with CRP would be defective compared to wild-type at F ( rhaB-lacZ ) D110 but would probably not be defective at F ( rhaB-lacZ ) D84 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>These residues ( positions 278 , 279 , 315 , 321 , 322 , and 323 ) were only important at the promoter which was activated by RhaS in the absence of CRP [ F ( rhaB-lacZ ) D84 ] ( Fig. 2 and Table 6 ) and are located within helix 2 and the C-terminal loop of a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Using a library of alanine substitutions in a-CTD , we found strong evidence for an a-CTD interaction-with-DNA , suggestive evidence for a possible interaction between a-CTD and RhaS , and no evidence for an a-CTD -- CRP interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Since there is no evidence for an interaction between the 261 determinant and CRP , we propose that residue 255 might interact with RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>In vivo , acs transcription required AR1 , a surface patch of CRP residues that contacts its - CTD counterpart , the 287 determinant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Other surface-exposed residues in the - CTD contributed to acs transcription , suggesting that the - CTD may interact with at least one protein other than CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Intriguingly , several - CTD residues not within the 287 , 265 , and 261 determinants also affected acs transcription , suggesting that the - CTD may interact with at least one factor other than CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Although our results suggested that the 287 determinant was not important at rhaBAD , it was also possible that different residues were required for this a-CTD -- CRP interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Four of these residues ( 283 , 284 , 320 , and 322 ) are very near the 287 determinant of a-CTD ( reviewed in reference 6 ) and therefore might deﬁne an interaction with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>We identiﬁed only one ( residue 315 ) of the eight res-idues that make up the a-CTD 287 determinant ( 6 ) , which is proposed to be the site of interaction with CRP AR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Both the 287 determinant of - CTD , required for - CTD interaction with AR1 domain of CRP ( 24 , 27 ) , and the 265 determinant , important for - CTD-UP element interaction , were found to be important for PcspD transcription ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>The 287 determinant , which is critical for - CTD interaction with the AR1 domain of CRP ( 6 , 38 ) , was found to be important for PcspE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>We reasoned that if a CRP-induced bend is responsible for loop breaking , then CRP might ' break the loop from other positions between ara0 , and aral .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>We provide evidence that the CAMP-CRP complex is involved in pH,-dependent malTp activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>No signiWcant diVerence in secondary structure content was observed between the apo and ligand bound forms of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In Fig. 6 , it is not clear whether the DBDs in apo-CRP forms a dimer as observed in the DhCprK [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>In addition , overall geometry of the HTH α-helices signiﬁcantly deviates from that in the active CRP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>In the CRP crystal structure , AR3 is located within a surface-exposed loop ( Weber and Steitz , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>DISCUSSION In the past few years , genetic and biochemical experiments have identified a surface-exposed loop of CRP ( ARI ) that is involved in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Therefore , the presence of a CRP motif at position 241.5 in the promoter of caiF strongly suggests that it is of physiological signiﬁcance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>A replacement of GC by AT at position - 4 5 resulted in the creation of a cAMP-CRP-independent promoter around position - 3 5 , whereas a deoP2 promoter containing three transitions at positions - 3 6 , - 8 9 and - 9 8 exhibited the same low activity as deoP2-wt in a strain that does not contain the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>We hypothesize that by stabilizing the corresponding transition states , the CRP - CTD interaction ( s ) decreases the energetic barrier to closed , open , and initiating complex formation and helps increased RNAP binding , open complex formation , and abortive RNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>This promoter would overlap with the cAMP-CRP binding site , and open complex formation in vitro was indeed completely abolished in the presence of cAMP-CRP ( Figure 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>EsS was unable to form an open complex in this putative promoter region , both in the absence and presence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The filled and open circles mark the G residues whose methylation by DMS is diminished in the presence of low ( 10 nM ) and high ( 100 nM ) concentrations of cAMP-CRP complex , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>In the presence of CRP , RNAP binds with similar affinity to acsP2 , and no longer forms an open complex at acsP2A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The presence of either WT CRP or KE101 allows RNAP to form an open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In the CRP : cAMP2 crystal , ionic interaction between the R82 guanidinium group and the negatively charged oxygen of the cAMP phosphate was perturbed by introducing a polar group ( Q ) to position 82 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>At a higher CRP concentration two retarded bands appear ( lane 4 ) ; the uppermost complex presumably contains two cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Since CRP exhibits functional inter-conversion depending on the presence or absence of specific effector cAMP [ 11 ] , we carried out Genomic SELEX in the presence and absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Free CRP dimers have three quaternary conformational states corresponding to the three cAMP ligation states .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Where do PhoB and cAMP-CRP bind ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>A competent B-C loop region is necessary to achieve the full level of transcription seen with CRP and wild-type Fis , implying that the B-C loop is still playing a role in coactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>When we used the puriﬁed CRP for in vitro transcription , we discovered that it contained an RNase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>A gel of in vitro transcription reactions showing coactivation by CRP and representative Fis mutants containing substitutions at residue 71 and 72 is shown in Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Single-round in vitro transcription reactions were performed with CRP and Fis mutants containing substitutions in the B-C loop region that is required for P2 activation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>S1 nuclease mapping of m R N A - D N A hybrids from the araJ promoter located the transcription start point to within + 3 bp , and demonstrates that the promoter is dependent upon AraC protein and CRP in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>This is a likely possibility since cAMP ± CRP activation of a CRP II-dependent promoter should require precise interaction between those amino acid determinants on CRP and a-NTD , and therefore a subtle change in protein conformation could interfere with RNP and core promoter interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>We found that alanine substitution of some residues within both AR1 and AR2 of CRP resulted in small defects in rhaBAD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>However , it is possible that amino acids in or near AR2 interact with other proteins involved in rhaBAD activation or , as mentioned above , that these substitutions have a small effect on the stability or overall structure of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>None of the AR1 nor AR2 mutants were signiﬁcantly defective for activation at F ( rhaS-lacZ ) D216 ( Table 6 ) , suggesting that the surface resi-dues on CRP that are most important for activation at simple CRP-dependent promoters may not have a signiﬁcant role in this context .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>This was further substantiated by our data demonstrating the involvement of the AR1 domain of CRP in PcspE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>In vivo , PcspE activity required only the AR1 domain of CRP ( Fig. 4B ) , and the improved AR3 domain did not compensate for the defective AR1 domain for PcspE activation ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>These results indicate that the AR1 domain of CRP is critical for PcspD activation , whereas AR2 domain of CRP is not necessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>These results indicate that the AR1 domain of CRP , not the AR2 domain , is critical for PcspE activation , while an improved AR3 domain did not contribute to the activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Site-directed mutagenesis experiments indicated that previously reported perfect CRP motif 1 , centered at 241.5 of the cai transcriptional start site , plays a direct role in the sole cai activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Further evidence that transcriptional activation does not rely solely on protein-protein interactions comes from experiments using pc-mutants of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Our results suggest that DNA topology and direct protein-protein contacts contribute to a comparable degree to the process of transcriptional activation by CRP at the malT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Even though direct protein-protein contacts certainly participate in transcriptional activation by CRP , DNA bending may provide a major contribution at certain promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>This region of s is located in the C-terminal 70 domain ( CTD ; amino acids 590 ± 613 ) , which also contains residues required for transcription-activation by Ada ( Landini and Busby , 1999 ) , lcI ( Kuldell and Hochschild , 1994 ; Li et al. , 1994 ) , AraC ( Hu and Gross , 1985 ) , RhaS ( Bhende and Egan , 2000 ) and the cyclic AMP receptor protein ( CRP ) ( Lonetto et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>In particular , the same s residues that affect FNR-dependent transcrip-70 tion activation ( K593 , R596 and K597 ) are also required for transcription-activation by the related transcription factor CRP , under conditions that specifically use activating region 3 ( AR3 ) , one of three potential CRP activating regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>With CRP , two interactions affect different steps in transcription initiation , namely an anti-inhibition component that overcomes the inhibitory effects of an untethered ~ CTD and a direct activation component that facilitates the transition from closed to open complexes ( Busby and Ebright , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The CRP - CTD contact may further increase the strength of one or more of the RNAP-promoter contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>In our model for synergistic activation of proP P2 , Fis minimally contacts one a-CTD domain , and the second a-CTD domain is bound to the CRP subunit oriented proximal to the promoter ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Depending upon the architecture of the promoter , one or two activation regions on the surface of CRP ( AR1 and AR2 ) are involved in positive contacts with the a subunit of RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>To test whether the - CTD was required for full activation by CRP or RhaR , we reconstituted RNAP from its individual subunits in order to obtain RNAP preparations in which both copies of the - CTD were deleted for all residues after amino acid 235 ( - 235 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>This is similar to the ﬁnding at several other class III CRP-dependent promoters that interactions with - CTD and cooperative binding do not account for a activation by CRP ( 23 , 25 , 26 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>As mentioned above , previous in vivo experiments from this lab did not provide conclusive evidence for or against an interaction between CRP and - CTD ; therefore , we used in vitro transcription assays to more directly measure the role of - CTD in transcription-activation at the rhaSR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Our ﬁnding that several point mutations in - CTD also decreased CRP-dependent in vitro activation of the rhaSR promoter further supports our conclusion that CRP activation of rhaSR requires interaction with - CTD ( unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Residue 315 is a part of the `` 287 determinant '' of a-CTD that is believed to contact CRP ( 6 , 29 ) and is also important for activation by FNR ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>We wished to identify residues in a-CTD involved in this activation and to investigate whether a-CTD activation depends on interactions with DNA and/or CRP and / or RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Overall , our results suggest that CRP interaction with a-CTD may not be required for rhaBAD activation ; however , a-CTD does contribute to full activation , probably through interactions with DNA and possibly RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>In addition to the possible interaction between a-CTD and CRP , the alanine scan results indicate that DNA contacts by a-CTD are important for its activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>There is also a second component to CRP activation of rhaSR that appears to function through synergism with RhaR and a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>CRP activation of rhaBAD expression appears to occur by a mechanism other than interaction with a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>If an interaction between CRP and a-CTD does occur at rhaSR , it appears to make a relatively small contribution to activation in the wild-type context .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>The small defect upon substitution of CRP G162 leaves open the possibility that CRP activation of rhaBAD could involve AR1 contacts with a-CTD ; however , the evidence for such an interaction is not strong .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>We have some evidence that a-CTD activation , at least in the absence of CRP , may involve contacts with RhaS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>This increased activation by CRP and a-CTD in the absence of the other is the opposite of what would be expected if CRP activation required contacts with a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>We expected that CRP activation would involve contacts with - CTD ; hence , - CTD would be required a a for full activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Does activation by a-CTD involve contacts with CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>In addition , the 10 and extended 10 elements are important for lacP ; single base 1-6S1 pair substitutions at the two elements decreased the promoter activity and made the promoter regain CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>The lack of defect from 70 K593A , R599A , or R608A at the ( rhaS-lacZ ) 216 promoter is similar to previous ﬁndings with RhaS and AraC that substitutions at 70 some residues were defective only in the absence of CRP activation ( 4 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>It is possible that these residues are also important for RhaS activation in the full-length promoter , but for reasons described in the Discussion , such as redundancy , they did not show any detectable defect in the presence of CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Further , if these s70 residues are important for activation by RhaS in the absence of CRP , it is possible that other AraC / XylS family proteins that activate transcription without the aid of a second activator , such as CRP , may also require these residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Figure 6B shows that the CRP mutants T158A and H159L ( 52 ) , which contain mutations in AR1 , had similar levels of activity regardless of the presence of the CRP binding site , indicating that they were unable to stimulate P2 transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Both H159L and H159L K101E CRP mutants signiﬁcantly decreased ( 3.5-fold ) PcspD activity , indicating that the H159 residue is critical for PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>We have recently demonstrated that several of the four CRP binding sites are involved in the activation of both malEp a.nd malKp in , rh .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Some elements of this structure , namely the CRP site 1 , located at − 76.5 , and the distal MalT sites , seem to play a direct role in malEp activation besides their participation in the assembly of the higher-order structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The mechanism of activation by CRP is clearly different from that occurring at the divergent malEp-malKp promoters , which are simultaneously controlled by an array of three binding sites for CRP , all located at distal positions in between ( 36 ) , and forming a high-order structure responsible for activation , assisting direct activation by the MalT primary regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The binding of MalT and CRP to this region is known to result in the formation of a higher-order structure that is responsible for malKp activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>It is noteworthy that other transcription activators , such as CRP , Fnr , and the Mor protein , whose binding site is centered around 241 , also interact simultaneously with the a and s70 subunits and contact similar activation targets at the amino acid level ( 2 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Furthermore , in the promoter , the putative cAMP -- CRP binding site is located at a proper distance for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>The promoter region contains only one potential CRP binding site , centred at a position ( -37.5 ) that is normally associated with repression rather than activation ( Fig. 4b ; Busby & Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>The next activation peak is observed when the CRP binding site is moved one helical turn further upstream to position -81.5 , with an activation factor almost as strong as for the more promoter-proximal positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Instead , the presence of C rather than T at position 6 of the CRP half-site appears to make Sxy essential for CRP-DNA-binding and transcription-activation ( 13,14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>These NMR experiments indicate a possible role of E72 and R82 in cAMP-binding and subsequent cAMP-mediated activation of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>These experiments argue that optimal activation of s28-dependent transcription requires CRP binding at position -49.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The more recent ﬁnding that other s70 substitutions at positions near R596 , especially K593A , signiﬁcantly reduce activation by AraC in the absence of CRP supports the hypothesis that wild-type AraC and s70 make an interaction that contributes to transcription-activation ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Discussion The multiple roles of CRP during activation of acs transcription Previously , we reported that in vitro RNAP alone could form an open complex ( TOC ) at both acsP2A and acsP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_225">
S , ( 1991 ) The role of two surface exposed loops in transcription-activation by the Escherichia co / / CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></label>
<input id="dom_similarity_225", type="checkbox">
<div class="DOM_similar">
<li>The role of two surface exposed loops in transcription-activation by the Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9440505" target="_blank">9440505</a>]<br></li></div>
<li>Therefore , even though CRP and FNR share the same structural framework , there are major differences in the way they interact with CZCTD , such that different contacts are used to modulate the extent of transcription-activation at specific promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>In vitro transcription studies effectively demonstrated this , whereby no CRP-dependent transcription-activation was observed on a linear template ; however , use of a supercoiled template permitted transcription-activation to occur , presumably through providing sufficient helical twist in the DNA to align cAMP ± CRP and RNAP correctly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Taken together , these results confirm the existence of three functional CRP sites 364 We have distinguished two functional regions within cddP that are involved in cAMP-CRP activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Biochemical and biophysical results revealed that regulation of transcription-activation depends on alignment of CRP tertiary structure through inter-domain communication and it was concluded that positions 72 and 82 are essential in the activation of CRP by cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>The larger , N-terminal domain ( NTD ) , which is predominantly β-stranded , is basically responsible for CRP dimerization ( see Fig. 1 ) , and the cAMP-binding to this domain results in the functional activation of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>We can deduce that the former group of deletions cause a reduction in both P1 and P2 expression , whilst the latter group prevents the activation of P1 by CRP and the switch from P2 to P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>In order to test the involvement of different domains of CRP in PcspD activation , PcspD activity was studied in M182 crp cells complemented with different activation region mutant alleles of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>To test the involvement of different domains of CRP in PcspE activation , plasmids pDCRPHL159 ( with the FIG. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_235">
FIG 4 Role of different functional domains of CRP in PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="dom_similarity_235", type="checkbox">
<div class="DOM_similar">
<li>Role of different functional domains of CRP in PcspE activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>This set of experiments determined the roles of key residues at CRP activation mechanism and aVect on CRP biological function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>CRP might alter the geometry of the 70 transcription-activation complex such that RhaS and s are no longer in precisely the correct position to interact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>This presumably facilitates activation of transcription in the presence of cAMP ± CRP and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Inhibition and activation of ompB transcription in vitro by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>These results may indicate that CRP has an inﬂuence on transcription-activation of rhaBAD , rhaT , and araBAD that is redundant with 70 the role of these s residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Therefore , stimulation of CRP transcription-activation was examined by perturbing these residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In conclusion , the experiments reported here revealed the importance of key residues for CRP activation as a transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>It will be interesting to see to what extent the latter is related to CRP function in the activation of other promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Recall that CRP activation sites are mostly restricted to three phased position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Positions 83 , 127 and 128 are important but not essential for CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>What physical interactions lead to transcriptional activation by CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>PrfA PrfA is distinguished from CRP in that it has no known effector for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Therefore , it can be concluded that CRP site-I plays an important functional role in CRP-mediated PcspD activation , while CRP site-II plays a minor role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_250">
PcspD with CRP site-I ( 83.5 ) alone also requires activation region I for CRP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="dom_similarity_250", type="checkbox">
<div class="DOM_similar">
<li>Therefore , similar to the wild-type promoter , PcspD with CRP site-I ( 83.5 ) alone also requires activation region 1 for CRP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li></div>
<li>CRP interacts with promoter-bound s - RNAP and blocks tran-54 scriptional activation of the dctA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>This region probably provides a contact site for CRP in the case of transcription-activation of the lacP1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>This appears to occur in CRP activation of the lacP1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Role of activation by cAMP-CRP at the csiD promoter Despite the general importance of the K173/C ( 213 ) interaction at s - controlled promoters , some specific S promoters do not feature a C ( 213 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>In vivo analysis and DNase I footprinting indicated that the promoter speciﬁc for agaZ also requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>In this context , there is a core motif ( TTGAT ) for a potential FNR half-site at about -55 in the putative ttdABpromoter region ( Nesin et al. , 1987 ) , and an analogous CRP half-site motif ( GTGTA ) at - 130 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8371115" target="_blank">8371115</a>]<br></li>
<li>The csiD promoter region also contains a distal UP-element half-site located downstream of the CRP box , as demonstrated by mutational studies , in which this element was either eliminated or completed to a full UP-element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_259">
Role of an UP-element half-site located downstream of the cAMP-CRP binding site in the csiD promoter region As concluded above , the csiD promoter seems to have some structural feature that does not allow Es to interact 70 productively with cAMP-CRP located at 272.5 , i.e. at a position that would be productive for activation in the context of the lac or CC promoters ( Gaston et al. , 1990 ; Ushida and Aiba , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></label>
<input id="dom_similarity_259", type="checkbox">
<div class="DOM_similar">
<li>S 70 In addition , the csiD promoter seems to have some structural feature that does not allow Es to interact 70 productively with cAMP-CRP located at 272.5 , i.e. at a position that would be productive for activation in the context of the lac or CC promoters ( Gaston et al. , 1990 ; Ushida and Aiba , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li></div>
<li>Second , [ Leu159 Glu101 Asn52 ] CRP carries an additional substitution , Lys52Asn , that improves the function of a third activating region that is known to interact directly with s70 region 4 ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>Since the four sites are almost aligned on the same face of the DNA helix , it is tempting to speculate that the intergenic region is tightly wrapped around a core of four CRP mole-cules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>CRP clearly stabilized complexes that contain an open region of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9256428" target="_blank">9256428</a>]<br></li>
<li>in i CRP is located within 0.5 kb of the coding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>In the NMR-derived secondary structure of apo-CRP [ 25 ] , an ambiguity in the L39 -- V43 region is indicated by dotted lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Rearrangement of hinge and interdomain interaction As noted above , the secondary structure change in the T127 -- L137 region in CRP can be represented as a rearrangement of hinge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>In contrast , certain mutation that obstructs the reconstitution of interdomain interaction , even if it is located outside the cAMPbinding pocket and the hinge region , should have impaired the activity of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Pseudomonadaceae and Xanthomonadaceae orthologs lack conserved regulatory motifs Although none of the Pseudomonadaceae and Xanthomonadaceae genomes listed in Figure 1 contained sxy orthologs , CRP orthologs are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>The 13 residue ¯ exible linker between the C-terminal and N-terminal domains of the a subunit57 ± 59 could enable the aCTD to adopt multiple con ® gurations at various Fis-activated promoters .30 Materials and Methods Proteins Fis protein was puri ® ed as described .53 CRP was obtained from J. Krakow , Hunter College .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The 3-D crystallographic structures of proteins involved in transcriptional regulation of E. coli and its phages ( e.g. , TrpR , CRP , CI repressor and Cro protein from phages 2 and 434 ) have a common structural motif , the ` helix-turn-helix ' motif ( Dodd and Egan , 1987 ; Brennan and Matthews , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1995431" target="_blank">1995431</a>]<br></li>
<li>The mutations introduced in pcspDgfpmut1 are in CRP site-I ( 90 [ G-C ] , 89 [ T-C ] , 87 [ A-T ] , in the ﬁrst core motif and 80 [ T-C ] , 79 [ C-G ] , 78 [ A-T ] , 76 [ A-T ] in the second core motif ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Mutations introduced in pcspDgfpmut2 are in CRP site-II ( 119 [ G-T ] , 117 [ G-C ] , 116 [ A-G ] in the ﬁrst core motif and 109 [ T-G ] , 108 [ C-G ] , 107 [ A-T ] , 106 [ C-T ] , 105 [ A-T ] in the second core motif ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Although the presence of both core motifs was found to be necessary for CRP to bind to the target site in PcspE ( Fig. 3A ) , an intact second core motif was shown to be more important for CRP-PcspE interaction ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Sequence integrity of the second core motif is more important for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Taken together , these data demonstrated that the changes are tolerated relatively more poorly in the second core motif than in the ﬁrst core motif , suggesting that the presence of an intact second core motif is more important for CRP binding to PcspE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Based on these predictions , CRP and FNR homologues have been classified according to the amino-acid-sequence motifs in their putative DNA-recognition helices , where those containing the FNR motif ( E -- SR ) are regarded as FNR-like proteins and those containing the CRP motif ( RE -- R ) are regarded as CRP-like proteins ( Spiro , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Examination of multiple alignment of the CRP/FNR-like proteins revealed one speciﬁc amino acid ( R180 ) in the HTH motif involved in DNA-recognition , which has changed from arginine ( like in E. coli CRP and Desulfovibrio HcpR ) to Ser or Pro in other dproteobacteria ( see the HcpR1 branch of the phylogenetic tree in Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>T Arg180 G 4 ~ % Va1208 e r 2 ~ T S G l y l ~ ~ 8 '' ~ T T Lys188 '' -LCB- 9 1 4 ~ '' Gly216 '' - / 9 ... Arg185 ~ '' Met189 Arg213 - `` T - ' ~ '' Arg217 T w Rgure4 Diagrams showing the amino acid side chains in the DNA-recognition helices of FNR and CRP that are thought to interact with specific base-pairs in the core motifs of the corresponding half-site consensus sequences in FNR-and CRP-bindingsites1 ~ ,19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The DNA-recognition helix of FNR possesses no equivalent to Arg180 of CRP ; instead , a serine residue ( Ser212 ) makes contact with the first T-A pair of the FNR-site ( Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>( a ) The DNA-recognition helices of CRP , FNR and FLP are shown in hall-helical wheel format numbered from the first residue of the helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The DNA-recognition motifs in the relevant α-helices ( αF ) of CRP and FNR are shown in bold and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>21 % identical to CRP and FNR , and it shares many features with the Lactococcus lactis proteins , including an FNR-like DNA-recognition helix ( even though it does not bind to the canonical FNR site , see below ) and only two cysteine residues ( Plate 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Mutagenesis studies with FNR ( Spiro and Guest , 1987 ; Spiro et al. , 1990 ) have indicated that as in CRP , the amino acid residues contacting the FNR target site are located in the putative DNA-recognition helix ( c ~ F ) of FNR ( Plate 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>( b ) Predicted structure of a FNR monomer based on CRP ( Schultz el al. , 1991 ) showing the N and C-termini , -LCB- , - helices ( cylinders A F ) , ] ~ - strands ( arrows 1-12 ) , and the four essential cysteine residues -LCB- C20 , C23 , C29 and C122 ) m  CRP FL ~ ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Secondary structural elements in CRP ( Schultz et al. , 1991 ) are indicated above the alignment , and conserved glycine and proline residues that ﬂank β-strands are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The cysteine residues in YeiL were modelled at the corresponding positions in the X-ray structure of the cAMP -- CRP -- DNA complex ( Schultz et al. , 1991 ) as shown in Fig. 3 ( a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The alignment shows that the glycine and proline residues ﬂanking some of the β-strands of the β-roll in CRP are conserved in YeiL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The residues involved in cAMPbinding by CRP and the cysteine residues in YeiL and FNR are shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>By the N-terminal extension and a further essential cysteine residue close to the center ( Cys122 ) , FNR and FNR homologs are clearly differentiated from CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>FNR is predicted to contain ail of the structural elements of CRP but it differs from CRP in being monomeric rather than dimeric , containing up to one atom of iron per monomer , and having an A / - terminal extension that includes three of the four essential cysteine residues ( Green ef ai , 1991 ; Sharrocks ef ai , 1990 ; Melville and Gunsalus , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>, two Synechocystis species ; NtcA Sy3 , Svnechococcus species ; NtcA Sy7 , Synechococcus species ; PrfA Li , Listeria ivanovii ; PflA Lm , Listeria mono ~ wtogenes ; PtrA Sy , ~ vnechococcus species ; Vfr Pa , Pseudomonas aeruk , inosa ; Ycf Pp , Porphyra purpurea : YeiL Ec , Escherichia coli ( 3 ) have a predicted DNA-binding specificity corresponding to that of FNR rather than CRP ; ( 4 ) fail to respond to cAMP ; ( 5 ) respond to oxygen limitation ; and ( 6 ) retain all or most of the characteristic ` cysteine signature ' of FNR , which comprises three essential N-terminal cysteine residues and a fourth essential cysteine within the [ 3-roll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Based on sequence homology with CRP ( 8 ) , FNR consists of a C-terminal DNA-binding domain , which recognizes specific sequences within FNR-controlled promoters ( 9 ) , and an Nterminal sensory-domain containing 5 cysteine residues , 4 of which , Cys-20 , 23 , 29 , and 122 , but not Cys-16 , are essential for function ( 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>However , FIpA and FIpB were designated FNR-like because : they are more similar to FNR than CRP when conservative amino acid substitutions are included in the comparison ; they retain the FNR-like DNA-recognition motif ( see below ) and lack the amino acid residues required for cAMP-binding ; and they retain one N-terminal cysteine residue ( Cysl5 ) and a single internal residue ( Cys112 ) corresponding to two of the four essential cysteine residues of FNR ( Plate 3a ; Gostick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>It differs from CRP in being monomeric , in possessing a cysteine ~ rich amino-terminal segment that contains three of the four essential cysteine residues in FNR , and in binding equimolar amounts of iron4 ,6 , t2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>FNR differs from CRP in three amino acid residues ( Val-208 , Ser-212 and Gly-216 in FNR ) which are known to be critical for specific binding of CRP to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>If cytidine does not affect CytR-CRP association per se , then does it affect simultaneous interaction of CytR with two CRP dimers to form a protein-protein bridge ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This obviates potential concerns that the structure of the Core domain and its interactions with promoter-bound CRP might be sensitive to the particular N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>2002 ) with the EsA.-fork junction DNA structure ( Murakami et al. , 2002 ) and , as expected , we found that there is a clash between the predicted locations of the aCTD downstream of CRP and domain 4 of s , which contacts the promoter -35 element ( K. Hollands and D.J. Lee , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Configuration 5 pertains to the complex formed by association of CytR Core domain in which ∆ G represents the free energy change for bridging CytR-CRP bridge interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Such an increase indicates cooperativity between Core domain and CRP for association with deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The free energy change for the association of CytR Core domain binding with the CRP2-promoter complex ( ∆ Gbridge ) was determined from paired experiments in which the promoter was titrated with CRP in the presence or absence of a fixed concentration of CytR Core domain ( 0.5-15 µM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Control experiments confirmed that this protection requires the presence of CRP ; no protection was observed in the absence of CRP , even at a concentration of Core domain of 48 µM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These were obtained by global analysis of matched CRP titrations conducted in the presence and in the absence of fixed concentrations of Core domain dimer , as shown in Figure 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Panels A and B represent paired titrations to show the effect of 4.0 µM CytR ∆ 2-63 Core domain dimer on CRP binding to udpP ; panels C and D represent paired titrations to show the effect of 13.1 µM CytR ∆ 2-63 Core domain dimer on CRP binding to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To address this question experimentally , matched CRP binding titrations of both deoP2 and udpP were conducted with and without CytR Core domain , as before ( Figure 6 ) , but now also comparing the presence and absence of 1 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Individual-site binding isotherms were generated by footprint titration of CRP binding in the presence ( panels B and D ) and absence ( panels A and C ) of fixed concentrations of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>First , mutations have been located on the surface of CRP in a putative protein interacting domain that interfere with cooperativity with CytR and with CytR-mediated repression ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These changed residues are positioned in close proximity on the outer surface of the large domain of CRP ( ie .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>CRP is composed of two identical subunits comprising 209 amino acids ( Aiba et al. , 1982 ; Cossart and GicquelSanzey , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>The CRP * protein contains an amino acid substitution ( G141D ) , making it independent of cAMP for its activity ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The CRP concentration was estimated based on an extinction coefficient of 18,400 M 21 at 280 nm , calculated from the average extinction coefficients for amino acid residues in a protein ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>RESULTS The yeiL gene encodes a paralogue of CRP and FNR The YeiL protein contains 219 amino acid residues , and has a total molecular mass of 25 294 Da and a predicted pI of 8 ± 23 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>This reading frame leads to the synthesis of a protein consisting of 209 amino acids , starting with valine as number one , which corresponds well with the CRP subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The contact occurs between AR2 ( residues His l9 , His21 , Lys101 ; Plates 2 and 3 ) of the downstream subunit of the CRP dimer ( Williams et al. , 1996 ) and a site in C ~ N , rDconsisting of amino acids 162-165 ( Nui et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The amino acids of the CRP AR2 are all positively charged and they interact with four negatively charged amino acids in O ~ NTD ( Glu162 , Glu163 , Asp164 , Glu165 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The structural gene of CRP consists of 627 nucleotides which corresponds to 209 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The CRP subunit contains 209 amino acids and its molecular weight is calculated to be 23,619 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The combined binding of CAMP-CRP and CytR-wt , additionally , resulted in diminished reactivity of three G residues and weakly enhanced reactivity of three A residues and one G residue ( Figure 5A , lane 9 ; Figure 58 , lane 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The guanine residues protected by CAMP-CRP and CytR in the DMS footprints are indicated by open arrows , and the residues with enhanced reactivity in the CytR-binding site are indicated by closed arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The extents of the regions protected by CAMP-CRP and CytR in DNAase I footprints are indicated above the helices ; the deoxyriboses protected from attack by the hydroxyl radical are indicated by closed circles ( CAMP-CRP ) and closed triangles ( CytR ) ; the residues with decreased reactivity in the DMS footprint are indicated by open arrows , and residues with enhanced reactivity are indicated by closed arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The five strong down mutations are in the two highly conserved pentanucle-otides that make base-specific contacts with CRP ( Fig. 6 ) ( 13 , 57 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>The ligand-free RrCooA structure showed an asymmetry , with one subunit closed and the other open , as observed in the cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The existence of two distinct structural regions has been shown also in the crystal structure of CRP ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The syncAMP-binding was also evident from the crystal structure of a mutant ( T127L/S128A ) CRP , where two anti- and one syn-cAMP bound [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The two conserved regions are arranged around a center of symmetry ( Figure 5 ) - suggesting that the symmetric elements may correspond to symmetric features of the CRP dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>CRP is a dimer of identical subunits , each of which consists of two domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>The four proteins with this domain architecture are CRP and FNR , which are both global regulators controlling a large number of genes , and two hypothetical proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>The in vivo b-galactosidase assays were performed using two sets of plasmids containing CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The IICBGlc -- CRP fusion proteins containing more than two transmembrane domains were localized at the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>The on-state ChCooA LL assumes the domain orientation well matched to that of the active CRP ( Figs. 1 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The ﬁnal preparation was 90 % pure , did not contain any detectable RNase activity , and was active in a mobility shift assay using a 40-bp oligonucleotide containing the CRP consensus binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>The CRP binding site contains a consensus binding domain consisting of two symmetrically related 5-bp segments at distances of 4-8 bp from the dyad axis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>The binding and speci cents city of CRP was con cents rmed using fragments derived from pCC ( carrying two consensus core CRP-binding motifs ) and pFF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The thymine residue at 2 appeared to be specifically reactive when CRP was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>The star indicates the position of the G residue that is protected when CRP binding site 4 is occupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>The conserved core motifs in the CRP site are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Yet , cAMP-CRP appears to improve binding by Es70 and therefore it seems unlikely that differences in binding ef ® - ciency of the two holoenzymes alone account for the exclusive sS dependency of csiD expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>CRP binding sites contain a palindromic sequence in which two conserved motifs , TGTGA and TCACA , are separated by a spacer ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>There are therefore two cAMP-binding sites per CRP dimer and , by virtue of making contacts with both monomers , each cAMP molecule contributes to the dimer-interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A second fusion plasmid , pJH3 , was constructed not to contain a plausible cAMP-CRP binding site at the two steps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>It contains two binding sites for the cAMP-CRP complex , CRP-1 and CRP-2 , which are centred around positions - 4 0 and - 9 3 , respectively ( Fig. 1 ) ( Valentin-Hansen 1982 , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>This is consistent with the fact that apo-CRP contains two cAMP-binding sites that exhibit negative cooperativity in binding cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Promoter regions controlled by these two proteins characteristically contain tandem cAMP-CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In this study , in vivo experiments were performed with wild-type ZK126 cells carrying pLW12 , which contains the identical lsr promoter region as pLW11 but with the four base substitutions in the CRP binding site as used in the gel shift assay ( Fig. 7A , seq4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>ZK126 ( WT * ) carries plasmid pLW12 ( lacZ fusion containing mutated lsrA promoter region with base substitutions in CRP binding motif ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Transcription of the gltA and sdhCDAB gene appears to diverge from a single intergenic region that contains two pairs of potential promoter sequences and two putative CRP ( cyclic AMP receptor protein ) - binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6383359" target="_blank">6383359</a>]<br></li>
<li>They used gel retardation experiments to show that the dad promoter region probably contains two CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Transcripts Tr6 and Tr7 were made in vivo but not detected in vitro Localization of the two cAMP-CRP binding sites c Fig. 4 DNase I footprinting analysis of CRP binding to the dad promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Two structural regions of each subunit have been defined by proteolytic analyses of CRP ( 22,33,36 ) : the amino proximal region containing both the cAMP-binding and subunit-subunit interaction sites ; and the carboxyl proximal region which contains the DNA-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>Among the four promoters , only the P2 region includes putative CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8208244" target="_blank">8208244</a>]<br></li>
<li>Plasmid pVH1451 contains both CRP binding sites , whereas pVH1424 contains only an intact CRP-1 region , since Sau3A cuts within the CRP-2 region ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>This region contains a putative CRP binding site centered around ÿ68 .5 ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Underlined residues at positions 7 and 16 at certain sites were mutated to prevent CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>This involved deleting two adjacent cytosine residues just downstream of the CRP binding site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>This observation suggests that residues between positions 133 and 118 , which include the CRP binding site , contribute in some way to GRDC of PguaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In terms of signal-recognition , only one of the six residues involved in cAMP-binding in CRP is conserved in YeiL ( Fig. 2 ) , suggesting that cAMP is not a coeffector for YeiL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Interestingly , the amino acid substitutions at position 82 and 72 had little if any eVect on Ka2 suggesting that these residues play little role in the binding of the second molecule of cAMP to CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>CRP requires coupling of cAMP-binding with the necessary conformational changes through cAMP interaction with residues 72 and 82 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Since this site is far upstream of RNAP 's binding site , it seems unlikely that CRP is contacting RNAP to activate transcription of gcv unless DNA looping occurs at the gcv promoter , allowing contact between one of CRP 's activating regions and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>a-CTD residue 301 appears to be located within the DNA-binding determinant of the a-CTD and therefore is not likely to deﬁne an a-CTD -- CRP interaction site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>When the fragments contain a specific CRP binding site , the D N A is displaced in a well-defined band .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>It is interesting to note that this upstream interaction is conserved , whereas the interaction downstream of the CRP binding site is lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>This result indicates that direct contacts to ribose moiety and sequence determinants for secondary structural formation are conserved to exhibit a tertiary structure similar to CRP cAMP-binding pocket [ 2,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>This half-site allows Es to use the stimulatory S potential of one aCTD ( which probably contacts the UP-element half-site as well as cAMP-CRP ) without Es 70 gaining a signiﬁcant advantage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>CRP is a homodimeric protein and each monomer consists of two domains ( Fig. 1 , 1G6N ) [ 10 -- 12 ] : the N-terminal cAMP-binding domain ( residues 1 -- 137 ) and the C-terminal DNA-binding domain ( residues Abbreviations : DBD , DNA-binding domain ; NTD , N-terminal domain ; apo-CRP , ligand-free CRP ; cAMP -- CRP , cAMP-bound CRP ; CRP LA , constitutively active T127L / S128A mutant of CRP ; RrCooA , ligand-free CooA from Rhodospirillum rubrum ; ChCooA , CooA from Carboxydothermus hydrogenoformans ; ChCooA LL , constitutively active N127L / S128A mutant of ChCooA ; Im-ChCooA , imidazole-bound ChCooA ; DdCprK , reduced CprK from Desulﬁtobacterium dehalogenans ; DhCprKox , oxidized CprK from Desulﬁtobacterium hafniense ; CHPA , 3-chloro-4-hydroxyphenylacetate ; CHPA - CprKox DhCprKox ; PrfA 145 , constitutively active G145S mutant of PrfA from Listeria monocytogenes ; NMR , nuclear magnetic resonance ; SANS , small angle neutron scattering Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>Assignment of residues into the DNA-binding category was based on the residues identiﬁed as part of the DNA-binding `` 265 determinant '' based on studies with CRP and UP elements ( 6 , 11 , 19 ) , on the ﬂuorescence characterization of a-CTD binding to a factor-independent promoter and CRP and UP elementdependent promoters ( 24 ) , and on the predicted position of the residues on the structure of a-CTD ( accession no. 1COO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>There are well-known precedents for variable spacing of DNA-binding motifs , in particular the 2-bp range of spacing of 210 and 235 regions of promoters ( 5 , 20 ) , and one can imagine CytR-CRP complexes being quite ﬂexible relative to other well-studied DNA-binding proteins in prokaryotes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>In the structure of CRP , these amin acids ( 156-162 ) form a surface-exposed loop between the psheets 9 and 10 in the small DNA-binding domain on the minor domain of CRPand most likely this loop can be involved in a direct protein-protein contaw ctith RNA polym-erase at both lac andgal promoters ( Busby , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>In CRP this domain is responsible for specific binding of the effector cyclic AMP ( Weber & Steitz 1987 ; Shaw et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>These are different in each case but they represent major structural features such as the sensory-domain ( β-roll ) and dimer-interface ( α ) of CRP C and the DNA-recognition helix ( α ) of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Thus it is predicted that , as illustrated by the CRP model in Plate 1 , the subunits of all members possess a long helix ( o cents c , yellow ) contributing to the dimer-interface , an N-terminal sensory or allosteric domain , and a C-ter-minal DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>In each case the protein subunits are predicted to resemble CRP monomers by containing a C-terminal DNA-binding domain with a helix -- turn -- helix motif for sitespeciﬁc DNA-recognition , a long α-helix that forms a dimer-interface , and an N-terminal sensory-domain based on a series of β-strands ( Schultz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>In contrast , at the lowest Core domain concentration at which bridge complex formation was detectable by its effect on CRP binding ( 0.5 µM Core domain ) , protection of the DNA between CRP1 and CRP2 could not be discerned above the background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To investigate the chemical and structural basis for these effects , we investigated how cytidine binding affects association between CytR and CRP in solution and how it affects the binding of CytR deletion mutants lacking the DNA-binding HTH domain , with tandem CRP dimers bound to either udpP or deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To address this question requires CytR Core domain in which the interaction between CytR and DNA is eliminated but other CytR functions ( i.e. , dimerization , cytidine binding , and CRP association ) are unperturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This suggests that there are contacts between the DNA-binding domain of CytR ( the ` toes ' of CytR ; Figure 6B and C ) and CRP near the DNA , besides the interaction between the ligandbinding/dimerization domain of CytR and CRP demonstrated previously ( Søgaard-Andersen and ValentinHansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CytR Core domain associates when CRP is bound to both CRP1 and CRP2 as a result of favorably oriented protein-protein contacts , despite no interaction between Core domain and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CRP is a homodimer of a 209 amino acid-long subunit ( Aiba et al. , 1982 ) , each consisting of two domains , N-terminal dimerization and cAMP-binding domain and C-terminal DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The acceleration of Fur proteolysis in the presence of metal is analogous to the behavior of the E. coli CRP protein : cyclic-AMP-binding to the N-terminal domain of CRP induces protease sensitivity in the C-terminal DNA-binding domain ( Heyduk & Lee , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>The crystal structure of CRP from E. coli shows that it is 40 % helical and 29 % β-sheet , constituting an N-terminal cAMP-binding domain and a Cterminal `` winged helix '' DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li>
<li>If so , elimination of CytR-CRP interactions in response to cytidine binding might allow the CRP-RNAP interaction necessary for full activation even in the presence of DNA-bound CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_382">
Effect of defective DNA-binding domain of CRP on PcspD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="dom_similarity_382", type="checkbox">
<div class="DOM_similar">
<li>Effect of defective DNA-binding domain of CRP on PcspE activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>CRP 's DNA-binding domain is contained within 50 C-terminal amino acids ( aligned in Figure 8 ) ; six of these are shared only within the EPV clade , as expected of residues that might mediate interactions with CRP-S sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Additionally , the data suggest that CytR has a modular structure with an NHr terminal DNA-binding domain and a COOH-terminal domain engaged in CAMP-CRP interactions , inducer-binding , and oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>But observations made several years ago using truncation mutants of CytR that lack the N-terminal DNA-binding domain suggest that CytR-CRP and CytRCytO interactions may not combine quite so favorably as this .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CRP binding in the absence of Core domain produced the expected pattern of mobility-shifted bands corresponding to species with either one or two CRP dimers bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The region in CAMP-CRP that contacts CytR has been identified and consists of two surface-exposed loops that are in close proximity and physically separated from the DNA-binding domain in CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Many repressor proteins , such as LacR , TrpR , and CRP , have been found to have a two-domain structure , in which one domain is involved in DNA-binding while the other is involved in ligand binding ( Steitz , 1990 ; Pabo & Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1868094" target="_blank">1868094</a>]<br></li>
<li>So even though independent DNA-binding of CytRA9-48 can not be detected , CytRA9-48 may be brought into such close contact with the DNA in the presence of CAMP-CRP that interactions with the backbone contribute to stabilizing the repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The protein makes direct contact with base pairs G : C5 , G : C7 and A : T8 in the highly conserved core motif T4G5T6G7A8 , and binding induces a localized kink of 43 between positions 6 and 7 , wrapping the DNA around CRP and strengthening the association ( 5,6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>However , because the core motif and two residues in the DNA-binding motif ( E -- R ; Glu 184 an Arg188 ) are conserved relative to CRP and FNR , it is predicted that FLP will make the same contacts with its target , as shown in Fig. 4a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The core interactions between the Glu ( position 2 ) and Arg ( position 6 ) residues and the conserved C-G and T-A base pairs at the FNR , FLP and CRP DNA-binding sites , arc indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The conserved G-C and A-T pairs of the TGA half-site core motif are contacted by glutamate and arginine residues ( Glu209 and Arg213 ) , which are predicted to occupy similar positions within the DNA-binding helix as Glu 181 and Arg 185 of CRP ( Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Genetic analysis and in vitro chemical protection studies have localized the initial binding site of cAMP - CRP around position -40 on the DNA , with the G residue at position -35 probably making contact with the protein ( 5 , 6 , 27 ; A. Majumdar and S. Adhya , manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>Based on a comparison with the cAMP receptor protein ( CRP ) and its recognition sequence on the DNA , Spiro and Guest ( 1990 ) have proposed that a glutamic acid residue in the DNA-binding domain of FNR makes an important contact with the GC base pair in the DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Cyclic AMP binding to CRP induces conformational changes that reorient the DNA-binding domains and exposes CRP surface residues important to CRP interaction with RNA E-mail addresses : ytutar@cumhuriyet.edu.tr, ytutar@yahoo.com ( Y. Tutar ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>To explain these observations , we propose that the DNA-binding domains of CytR are tethered to the rest of the protein through a flexible linker that allows the repressor to interact with half-operators separated by different spacings , and to interact with cAMP-CRP complexes at various distances from the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>CRP DNA-binding domains are aligned with the closely related P.aeruginosa DNR and the distantly related E.coli FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Therefore , it is probable that it is the CytR core protein , not the DNA-binding domains of CytR , that interacts with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>These putative domains are approximately the same size as : the XCro protein ( 66 residues ) and the DNA-binding domains of many transcriptional regulators , e.g. CRP ( 70 residues ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2060763" target="_blank">2060763</a>]<br></li>
<li>Its car-boxyl-terminal fragment contains an a-helix-turn-a-helix secondary structure ; the F-helix of each CRP monomer interacts with two consecutive major grooves of DNA ( Weber & Steitz , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>Chemical protection studies of the ara and.lac sites as well as mutational data suggest that cAMP/CRP recognizes two regions on the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>When activated by cAMP , CRP recognizes speciﬁc DNA sites via the C-terminal F-helix ( αF ) which forms a typical helix -- turn -- helix ( HTH ) motif , together with the neighboring E-helix ( αE ) ( Fig. 1 , 1G6N ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The two c ~ v helices in the CRP - ( cAMP ) 2 complex each occupy one half of the CRP target site , where the individual CRP half-sites are located in adjacent major grooves separated from each other by one complete turn of the double helix , such that the C R P : D N A interactions occur on only one lace of the helix ( Plate 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Binding cAMP induces a conformational change in the CRP dimer that aligns the helix-turn-helix motifs of the DNA-binding domains such that they are configured to penetrate adjacent channels of the major groove on one face of the DNA helix ( reviewed by Kolb el al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The cAMP-CRP complex contacts DNA via a helix-turn-helix motif in which the second helix ( recognition helix ) fits into the major groove of the DNA and specific Summary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Nucleo-tide alterations introduced in CRP binding sites are indicate contacts are made between amino acids in this helix and bases in the DNA ( McKay et al. 1982 ; Ebright et al. 1984 ; Pabo and Sauer 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Based on genetic and biochemical analyses of cAMP-CRP/DNA interactions in the lac promoter , the second amino acid in the recognition helix of CRP ( glutamate at position 181 ) has been proposed to make contacts to the GC basepair located at position + 4 and the CG bp at position - 4 in the consensus CRP binding site , 5 ' - TGTGAN6TCACA-Y ( both positions underlined ) ( Ebright et al. 1984 , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Three amino acids in the E.coli CRP helix -- turn -- helix confer CRP-N site recognition through base contacts ( R180 , E181 and R185 ) ; Figure 8 shows that they are conserved in all ﬁve families .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Amino acids in E.coli CRP that make base contacts are highlighted black , those making contact with phosphates in the DNA backbone are highlighted grey , those shared only by the EPV clade are outlined in red ; amino acid numbering is according to E.coli CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>All three residues are buried in the CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>In CRP glycine residues ( m ) important for the 13-rollstructure and residues ( ~ ) specifically interacting with cyclo AMP are marked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>At the bottom of the figure , specific residues in CRP that are known to interact with particular bases of the DNA are also indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Glutamic acid residues ( Glu-181 ) of the CRP dimer are known to make direct contacts with these bases ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>and 6 of the second helix of the CRP protein are the residues which make specific contacts with bases 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>To provide evidence that CRP binding at 2121.5 directly contacts Es38 at proP P2 , CRP mutants containing substitutions at residues that disrupt poly-merase-CRP interactions at either class I or class II promoters were tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Finally , it is also possible that the 70 proposed contacts between RhaS and s only occur in the absence of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Much less is known about promoters where the CRP binding site is located further upstream and direct protein-protein contacts may be more difficult to establish .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>In order directly to investigate this possibility and to minimize possible contributions of direct protein-protein contacts , we reconstructed promoters that replace the CRP binding sites with recognition sites for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>It is likely that CRP coactivation also involves contacts with - CTD at the rhaBAD and rhaT promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In fact , RNAP has been reported to make minor groove contacts in this region , dependent on CRP binding ( 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>guaB An alternative possibility is that the missing factor contacts RNAP at PguaB in a manner that requires CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP has the potential to make at least three different contacts with RNAP ( reviewed in reference 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>CRP contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>In thecase of X - repressor rather distinct points of contact may beidentified and assigned to themajor groove ( Nielsen et al. , 1988 ) , whereas the contacts to CRP clearly are assigned across the minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>CRP targets forms three contacts with the DNA backbone , Probing of OpenComplex-In order to characterize the each spanning the minor groove and all on the same side of initiation open complex further , probing with uranyl citrate , the helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Most noteworthy between two DNA-bound CAMP-CRP complexes ( Sbgaard-are minor groove contacts in the -60 region ascribed Andersen et al. , 1991a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Hence , CytRA9-48 seems to form a protein bridge between the CAMP-CRP complexes without making se-quence-specific contacts to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CytR simultaneously makes specific DNA contacts as well as protein-protein contacts to cAMP-CRP in the repression complexes ( Pedersen et ai , 1991 ; Sogaard-Andersen and ValentinHansen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Thus , we conclude that the -60 contact can DISCUSSION be ascribed to theRNA poymerase-CRP transcription initia - CRP-Operator Complex-CRP belongs to theclass of helixtion complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Both aCTD domains were found to be required for synergistic co-acti-vation , suggesting that Fis and CRP individually contact each aCTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Hence CRP makes active contacts via AR1 and AR2 , whereas FNR uses an extended AR 1 contact and AR3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Our results show that the two aCTDs of Es28 contact DNA both upstream and downstream of CRP , although note that we can not prove that both contacts occur simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>For sterical hindrance considerations , CRP would contact DNA on the opposite face of the helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>These amino acids lie in close proximity on the surface of the CRP tertiary structure in a portion of the protein that is not in contact with the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>The amino acid substitutions in these mutant CRP proteins lie in two different regions that are in close proximity in the CRP tertiary structure and are exposed on the surface in a region that is not in contact with the DNA ( Sogaard-Andersen etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Nucleotides where the CRP homodimer makes contact are in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>Similarly , we speculate that the surface of the other HTH ( HTH 1 ) may have evolved to contact CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>A surface loop of CRP that is involved in the protein-protein contact has been identified ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>A defect in the AR1 domain of CRP fails to activate both class I and II promoters , while an improved AR3 domain reverses the effect of a defective AR1 domain and enhances the activity for class II promoters only ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>( C ) Complementation with plasmids carrying no CRP ( pDU9 ) or WT CRP ( pDCRP ) and CRP mutants with an improved AR3 domain ( pDCRPKN52 ) and an improved AR3 domain with defective AR1/AR2 domains ( pDCRPHL159KE101KN52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The interaction with RNAP at class II promoters is complemented by NTD of CRP , through the AR2 ( residues 19 , 21 , 96 , and 101 ) and the AR3 ( residues 52 -- 55 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_444">
A third surface patch , AR3 of CRP ( amino acid residues 52 to 55 and 58 ) , has recently been shown to be important in CRP-dependent class II promoter activity ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="dom_similarity_444", type="checkbox">
<div class="DOM_similar">
<li>A third surface patch , AR3 , of CRP ( amino acid residues 52 to 55 and 58 ) recently has been shown to be important in CRP-dependent class II promoter activity ( 7 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>From the amino acid residues of CRP which specifically interact with cyclo AMP ( Gly-71 , Glu-72 , Arg-82 , Ser-83 , Thr-127 , Ser-128 ) only Gly-71 is conserved , which is also a structural element of the [ 3-roll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Thus , the amino acid substitutions studied here had negligible aVect on CRP structure stability and on secondary structure content .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>A previous study showed that amino acid substitution at positions 72 or 82 aVects CRP eYcacy in stimulating lacP activity in cells that contain the mutant forms of CRP [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">CRP evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Likewise , at least 11 or 12 members of a total of 16 MFS ( major facilitator superfamily ) transporters [ 57,58 ] were found to belong to the CRP regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_2">
Pedersen H & Valentin-Hansen P ( 1997 ) Protein-induced ﬁt : The CRP activator protein changes sequence-speciﬁc DNA recognition by the CytR repressor , a highly ﬂexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></label>
<input id="evo_similarity_2", type="checkbox">
<div class="EVO_similar">
<li>Pedersen , H. , and Valentin-Hansen , P. ( 1997 ) Protein-induced ﬁt : the CRP activator protein changes sequencespeciﬁc DNA recognition by the CytR repressor , a highly ﬂexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Pedersen , H. , and Valentin-Hansen , P. ( 1997 ) Protein-induced ﬁt : CRP activator protein changes sequencespeciﬁc DNA recognition by the CytR repressor , a highly ﬂexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>The identity of CRP protein was conﬁrmed by both amino-terminal peptide sequencing and immunoblotting with anti-CRP sera ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Comparison of CRP primary structure with six type I and type II cAMP-dependent protein kinase primary sequences showed only 36 -- 41 % homology when both identical or similar amino acids were considered [ 2,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Phylogeneticrelatedness of all known members of the CRP/FNRfamilyoftranscriptionalregulators , derivedfroman alignment similarto that shown in Figure 3 , but including 10 members of the cyclic AMP receptor ( CRP ) family of proteins : CRPs from E. coli ( Eco , AccessionNumberP03020 ) , Salmonellatyphimurium ( Sty , P06170 ) , Shigellaflexneri ( Sfl , M13772 ) , Klebsiellaaerogenes ( Kae , P29282 ) and Haemophilusinfluenzae ( Hin , 1 ' 29281 ) ; CLP , a CRP-like protein from Xanthomonascampestris ( Xca , P22260 ) , the nitrogen regulatory protein NtcA from Synechococcus ( Sy ) strains PCC 7942 ( X60197 ) and PCC 6803 ( X71607 ) and from Anabaena ( An ) strainPCC7120 ( X71608 ) and the sulphurregulatory protein CysR from Synechococcusstrain PCC 7942 ( P27369 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>[ 102 ] Egland , P.G. and Harwood , C.S. ( 2000 ) HbaR , a 4-hydroxyben-zoate sensor and FNR-CRP superfamily member , regulates anaerobic 4-hydroxybenzoate degradation by Rhodopseudomonas palustris .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>Maximum Likelihood Phylogenetic Tree of the FNR/CRP Family of Transcriptional Regulators The third column contains sequences of helix-turn-helix motifs in the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The regions of greatest sequence similarity ( P % 0 ± 001 ) identiﬁed by pairwise comparisons ( Staden , 1982 ) between YeiL and either CRP or FNR are also indicated in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>[ or oxidative stress FlpA and FlpB are 61 % identical to each other and 24 % identical to both CRP and FNR at the amino acid sequence level ( Plate 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The overall similarity of FNR to CRP , Orf240 and FixK amounts to 43 % , 48 % and 42 % identical and similar amino acid residues at equivalent positions of the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>A analysis showed that YeiL is more closely related to members of the CRP -- FNR family ( 16 -- 22 % identical , 42 -- 45 % similar ) than to other known proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Sequence comparisons with representative members of each subgroup of the CRP -- FNR family showed that YeiL is : 22 % identical ( 42 % similar ) to the FNR of E. coli ; 19 % identical ( 45 % similar ) to the FLP of L. casei ; 19 % identical ( 44 % similar ) to the FixK of Rhizobium meliloti ; and 17 % identical ( 45 % similar ) to the CRP of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>In addition to the speciﬁc regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e.g. , the CRP in the PrhaBAD promoter ) , or histone-like proteins that act as negative regulators ( e.g. , YomA in the VirF-regulated Pyop promoters ) ( 52 , 124 , 242 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Optimal expression of the melAB promoter requires Sites 10 , Site 1 , Site 2 and Site 20 ; CRP acts as a ` bridge ' between MelR bound at Sites 10 and 1 and at Sites 2 and 20 , increasing MelR is a melibiose-triggered transcription activator that belongs to the AraC family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Clearly , the dsdXA promoter belongs to the more complex class III CRP-dependent promoters , in which a regulon-speciﬁc activator protein in addition to CRP is required for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>Our ﬁnding that the cAMP-CRP complex activates a 54-dependent promoter in concert with a member of the 54-dependent activator family suggests that the CRP family might interact with the 54-dependent expression system regardless of nitrogen metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Such peculiar synergistic transcription activation has not yet been observed in other aromatic cat-abolic pathways , and the MhpR activator becomes the first member of the IclR family of transcriptional regulators that is indispensable for recruiting CRP to the target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>Although this is highly speculative , we can not disregard the possibility that there is a family of activator proteins ( class II in Fig. 2 ) , typified by Cl and CRP , which are ancestral repressors that have been remodelled to become both activator and repressor proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2664421" target="_blank">2664421</a>]<br></li>
<li>From our studies and those reported by other authors ( 16 , 21 , 30 ) , we may conclude that activation by cAMP-CRP and the repressor effect caused by binding to several operator sites are general features of the genetic systems under the control of repressors belonging to the DeoR family in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Expression of mhp catabolic genes depends on the transcriptional activator MhpR belonging to the IclR family of transcriptional regulators ( Fig. 1A ) comprising more 2 The abbreviations used are : 3HPP , 3-hydroxyphenylpropionate ; CRP , cAMP receptor protein ; DHPP , 3 - ( 2,3-dihydroxyphenyl ) propionic acid ; PA , phe-nylacetic acid ; 3HPA , 3-hydroxyphenylacetic acid ; PP , phenylpropionic acid ; CI , cinnamic acid ; 3HCI , 3-hydroxycinnamic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>Additionally , the YhcN family , which consists of nine , small , mostly uncharacterized proteins ( Rudd et al. 1998 ) including BhsA , whose promoter also has a CRP binding site ( Zhang et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>As far as we are aware , this is the first report of a direct interaction between CRP and an AraC family member ( although there may be a parallel with the interaction between CRP and CaiF reported by Buchet et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Conclusions Members of the LacI -- GalR family of transcriptional repressors are usually functional as dimers , although in some cases , repression depends on the dimerization of dimers or interactions with other proteins , such as CRP ( Weickert & Adhya , 1992 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>Opposing activation and repression , mediated by CRP and a LacI family member , respectively , is one of the most common ways in which transcription of catabolic genes is regulated in bacteria to meet the cell 's metabolic needs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Perspectives The four examples of oxygen/redox responsive members of the CRP family discussed above illustrate the great versatility of the basic CRP fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Microbiology ( 2000 ) , 146 , 3157 -- 3170 Printed in Great Britain YeiL , the third member of the CRP -- FNR family in Escherichia coli Muna F. Anjum , Jeffrey Green and John R. Guest Author for correspondence : John R. Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The E. coli CRP is the only member for which a three-dimensional ( X-ray ) structure has been defined , but there is strong evidence to support the view that each family member possesses an analogou structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>This work has laid a foundation for future investigations aimed at deﬁning the precise role of the third member of the CRP family in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The complete genome of Escherichia coli ( Blattner et al. , 1997 ) was recently shown to encode a third member of the CRP -- FNR family ( YeiL ) when searched with a sequence proﬁle derived from a multiple alignment of ﬁve members of the family : CRP and FNR from E. coli ; FLP from L. casei ; and FlpA and FlpB from Lactococcus lactis ( Gostick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>A phylogenetic tree derived from 38 members of the CRP -- FNR family indicated that YeiL is a distant relative of the CRP group ( Gostick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Keywords : CRP -- FNR protein superfamily , iron -- sulphur proteins , transcriptional regulation , nitrogen starvation , stationary phase INTRODUCTION The CRP -- FNR family of transcription regulators contains at least 70 members that respond to a variety of metabolic and environmental signals to control a diverse range of physiological functions in an equally diverse range of bacteria ( Spiro , 1994 ; Guest et al. , 1996 ; Vollack et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>& control in bacterial denitriﬁcation by members of the FNR-CRP regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The L. casei FLP protein groups with the CRP rather than the FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>FNR is predicted to contain all the structural elements of CRP , and both regulators constitute a family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>DISCUSSION The yeiL gene of E. coli was predicted to encode a new and distinct member of the CRP -- FNR family of transcriptional regulators that might have an iron -- sulphur centre and a reversible intra - or inter-molecular disulphide bond .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>A member of the CRP family of regulatory proteins , Fnr differs by having a cysteine rich N-terminal segment which binds iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>The structural framework of the founder member of the family , the Escherichia coli cAMP receptor protein ( CRP ) , provides a versatile system for sensing and transducing environmental or metabolic signals into a physiological response ( 2 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The cents rst member of the family was the cyclic aden-osine monophosphate ( cAMP ) - binding protein , CRP ( cAMP receptor protein ) , also called CAP for catabolite activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>It is a member of an expanding family of transcriptional regulators that are structurally related to CRP , the cyclic AMP receptor protein ( Irvine and Guest , 1993 ; Kolb ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>In studies of sotA , a SET family member in D. dadantii , the global regulators CRP and KdgR were implicated in transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Whereas the LH region of the PaaX binding motif is rather conserved in the operator sequences of other bacterial regulators such as GntR ( 6 ) and PdhR ( 27 ) ( which belong to the GntR family ) and , surprisingly , CRP ( 23 , 25 ) and LacI ( 26 ) ( which belong to the GalR-LacI family ) , the RH region differs from that of other operators ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>CRP , which is not a member of the GalR-LacI family of regulators , also functions as a symmetric dimer by binding to DNA sites with dyad symmetry located upstream of the RNA polymerase in the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Its hyphenated dyad symmetry resembles that of the binding sites for other regulators with helix-turn-helix motifs ( e.g , CRP and FNR ) but it differs from the known sites for other members of the GntR family which exhibit uninterrupted dyad symmetry , i.e. GntR -LCB- ATACnGTATACAAGTAT : Fujita and Miwa , 1989 ) and HutCpp -LCB- TTGJAATAtAc ; Hu et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>Whereas inducer or co-repressor binding to other family members controls a switch between DNA-binding protein and non-DNA-binding protein conformations , cytidine binding to CytR affects only cooperative interactions between CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR is unique among LacI/GalR family members in that binding of its regulatory ligand , cytidine , is linked not to DNA-binding affinity but instead to the cooperative interactions between CytR bound at CytO and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>© 1997 Elsevier Science B.V. Keywords : GntR ; cAMP-CRP ; GntP family 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>Unlike the other members of this protein family , CytR binds with only modest afﬁnity to its operators and transcription repression thus relies on the formation of nucleoprotein complexes with the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Condusions Proteins structurally homologous to FNR ( and CRP ) of E. coli are widespread amongst Gram-negative bacteria , and there is at least one example from a Grampositive organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Sequence analysis of thefir gene indicates that FNR is structurally homologous to CAMP receptor protein ( CRP ) , the transcriptional regulator of catabolite-sensitive genes [ 2,7,8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>FNR shows sequence and functional similarity to CRP , the cyclic AMP receptor protein ( BELLet al. 1989 ; SPIROand GUEST1987 ) , although the mechanism by which FNR senses ana-erobiosis is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>The protein shows a close homology to the transcriptional regulator CRP ( cyclo AMP receptor protein ) of E. coli , and the gene products of the hlyX gene of Actino-bacilluspleuropneumoniae , the orf240 gene of Rhizobium leguminosarum and the fixK gene of R. meliloti ( Fig. 1 ) ( De Crombrugghe et al. 1984 ; Shaw et al. 1983 ; Maclnnes et al. 1990 ; Colonna-Romano et al. 1990 ; Batut et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Three amino acids in the E.coli CRP helix -- turn -- helix confer CRP-N site recognition through base contacts ( R180 , E181 and R185 ) ; Figure 8 shows that they are conserved in all ﬁve families .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In some of these proteins the presence of extensive sequence similarity has allowed specific families with common ancestries and similar modes of action to be recognised , e.g. the XCro , LysR and CRP families [ 2-5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2060763" target="_blank">2060763</a>]<br></li>
<li>Given the 30 % amino acid sequence identity and 62 % similarity between RhaS and RhaR , we attempted to identify the difference between RhaS and RhaR that dictates their difference in optimal CRP binding site position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by CRP</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>ACKNOWLEDGEMENTS We thank Tove Atlung , Sankar Adhya and Steve Busby for supplying bacterial strains and Steve Busby for supplying plasmid pDCRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Growth of control strain MC4100 ( pMGL25 ) and strain SBS688 ( pMGL25 ) carrying plasmids pBR322 and pYZCRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>In order to avoid the isolation of fusions to non-starvation-specificcAMPICRP-dependent genes or operons , strains were usedfor hplacMuinfection that were either deficient for adenylate cyclase ( Acya ) or even carried a Acya Acrp double mutation ( R01 and DW1 , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>B. E. coli cya crp double mutant strains TP2339 -- 1 containing pKU101 were cotransformed with a low copy number plasmid carrying either the wild-type crp gene ( pLG339CRP ) or a deleted crp gene ( pLG339DRS ) as control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>CSklaCZexpression in minimal media in response-to-starvation for carbon , nitrogen and phosphate sources and the roles of global regulators such as the alternative sigma factor ( encoded by 0 ' rpoS ) , cAMPICRP and the relA gene product were investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>The transcriptional start sites and translational-initiation sites ( bold ) , the -35 and -10 hexamers of putative promoters ( boxed ) , alternative hexamers for P2 , , , A ( dashed overlined ) , ribosome binding sites ( RBS ) , and potential binding sites for FNR ( black box ) , CRP ( bold overlined ) , ArcA ( underlined ) and Fis ( dashed underlined ) , are shown for ( a ) acnA ( co-ordinates 701-1250 ; Prodromou et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Expression analysis of ( ulaG-lacZ ) and ( ulaA-lacZ ) transcriptional fusions in different genetic backgrounds under anaerobic conditions - Galactosidase expressiona ( Miller units ) Relevant phenotype Strain ( ulaG-lacZ ) ( ulaA-lacZ ) CAA L-Ascorbate CAA L-Ascorbate 40 130 JA134 Wild type JA184 IHF JA185 IHF JA188 RpoS JA211 YjfQ JA216 YjfQ CRP JA182 FNR a ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>CRP is involved in gcvT-lacZ expression b-Galactosidase activityc of cells grown in GM medium with : Straina Relevant genotypeb ( plasmid ) No Glycine Inosine addition 12 12 -- -- -- -- -- -- -- -- -- 854 564 -- -- -- -- -- -- -- -- -- 134 34 33 GS162 Wild type GS1068 Dcrp : : Cmr GS1068 ( pGS294 ) Dcrp : : Cmr ( vector ) GS1068 ( pGS396 ) Dcrp : : Cmr ( crp1 ) GS1068 ( pYZcrp ) Dcrp : : Cmr ( crp111 ) GS1068 ( p158A ) Dcrp : : Cmr ( crpAR1111 ) GS1068 ( p162C ) Dcrp : : Cmr ( crpAR1111 ) GS1068 ( p19A ) Dcrp : : Cmr ( crpAR2111 ) GS1068 ( p52N ) Dcrp : : Cmr ( crpAR3111 ) GS1079 D ( cyaA1400 ) : : Knr r r GS1080 D ( cyaA1400 ) : : Kn Dcrp : : Cm a All strains carry the lgcvT-lacZD2341 fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Genes induced at the onset of stationary phase with s - dependent expression as identiﬁed by microarray analyses Functional Mean WT d Ratio classa valueb ( WT/rpoS ) Regulation SDc Gene Gene product Starvation and stress response genes dpsS DNA-protection protein , bacterial ferritin IHFS , OxyRS , rssBTC , MMC , PQ , NaSal , Lrp , BG , DHCP , OS FisS , NaSal , Lrp , OS CRPS , LrpS , IHFS , H-NSS , arcBTC , MMC , Lrp , Rcs , BG , DHCP ompR/envZTC , CD uvrYTC , arcBTC SSR 1,597 2.5 0.4 osmES osmYS Stress-inducible outer membrane lipoprotein Hyperosmotically inducible periplasmic protein Induced by phosphate starvation Superoxide dismulase precursor ( Cu-Zn ) SSR SSR 1,728 424 9.6 4.4 2.1 1.3 656 459 psiF sodCS 4.8 2.7 0.4 0.5 SSR Protein synthesis rpsVS infA Amino acid transport and metabolism artP artI 5.2 2.8 FISS , IHFS , ppGppS 2.7 0.8 30S ribosomal subunit protein S22 Protein chain initiation factor IF-1 3,466 1,011 PS PS Arginine transport , ATP-binding component Arginine transport , periplasmic binding protein L-Asparagine permease Dihydroxyacid dehydratase Tryptophanase AATM AATM 714 346 2.6 0.7 PQ , NaSal , Lrp 3.8 1.9 PQ , Lrp ansP ilvD tnaA AATM AATM AATM 572 364 DHCP ompR/envZTC 2.5 4.8 0.4 2.1 4.3 phoQTC , ompR/envZTC , CRP , uvrYTC , arcATC , arcBTC , rcsBTC , citABTC , MMC , PQ ( ilvG : Lrp ) NaSal , Lrp , BG , DHCP , OS 13.7 7,808 wrbAS Flavodoxin-like protein ; binds the trp repressor SSR 364 5.0 2.0 Iron uptake and storage bfr entD Carbohydrate metabolism crr Bacterioferrin , iron storage homoprotein Enterochelin synthetase , component D IUS IUS 905 508 5.1 2.1 DHCP , ryhB RNA 4.2 0.5 acrBTC Phosphotransferase system , glucose-speciﬁc IIA component Fructose 1,6-bisphosphate aldolase ( b2097 ) Glyoxylate-induced protein Putative oxidase , glycolate utilization ( b2977 ) Glucosamine-6-phosphate deaminase Phosphofructokinase-2 Transaldolase A CHM 443 2.7 0.6 CD 1,264 1,097 CHM 4.6 3.6 3.0 2.6 2.5 2.9 1.4 1.0 1.7 0.6 2 1.8 MMC , Lrp kdpABCDETC fbaB gip uvrYTC , arcATC , arcBTC ompR/envZTC , arcBTC MMC , CD , DHCP rssBTC , MMC , DHCP , CreBC CHM CHM CHM CHM glcG nagB pfkB talA Miscellaneous function or unknown function deoD moeB msyB yafN yahOS ybgA 3.0 1.2 4.0 0.1 2.2 1.7 4.8 3.4 6.1 2.6 4.5 4.5 349 Purine nucleoside phosphorylase Molybdopterin-biosynthesis NM CB 1,100 2,038 CD , Lrp , DHCP MMC , GadX Lrp Membrane protein , protein export ( b1051 ) Hypothetical ( adherence ) protein ( b0232 ) Hypothetical protein ( b0329 ) Hypothetical ( inner membrane ) protein ( b0707 ) Putative enzyme ( b0865 ) Hypothetical protein , bioﬁlm dependent ( b1481 ) Hypothetical protein ( b1724 ) Hypothetical protein ( b2427 ) Putative dehydrogenase protein ( b2660 ) Putative immunogenic protein ( b2922 ) Putative transport periplasmic protein ( b3020 ) Hypothetical ( outer membrane ) protein ( b3024 ) Hypothetical ( regulatory ) protein ( b3555 ) Hypothetical protein ( b4045 ) Hypothetical ( membrane ) protein ( b1582 ) Hypothetical ( periplasmic ) protein ( b3097 ) Hypothetical protein ( b4217 ) MF ( MF ) rssBTC , baeSRTC , atoSCTC , Lrp ybjP yddX 2.5 1.4 4.2 4.2 1,317 376 ( EF ) rssBTC , OS ompR/envZTC ydiZ yfeT ygaF yggE ygiS 1.7 0.9 0.5 1.7 0.8 3.1 3.5 2.8 4.8 2.7 436 rssBTC , NarXLTC rssBTC , MMC ; Rcs , phoQP ( EF ) ( MF ) rssBTC , citABTC , arcATC , NaSal ( MF ) ygiW 1.9 4.0 892 rssBTC , ntrBCTC , NaSal yiaG yjbJ ynfA yqjC ytfK 0.5 1.2 1.1 1.3 3.0 2.7 4.5 3.5 5.9 3.9 1,227 601 3,334 1,121 556 ( MF ) PQ , Fur , BG a SSR , starvation and stress response ; PS , protein-synthesis ; AATM , amino-acid-transport and metabolism ; IUS , iron uptake and storage ; CHM , carbohydrate-metabolism ; NM , nucleoside-metabolism ; CB , cofactor-biosynthesis ; MF , membrane function ; EF , extracellular function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>Continued Gene product b Genes and promoter regions a Unknown and miscellaneous functions yccT Unknown Induced in stationary phase ( 76 ) b0964 b2447 b3679 b1586 b2602 b0255 b2625 1.51 1.52 1.53 1.54 1.55 1.56 1.57 yffP Predicted protein , prophage yidK Putative membrane transporter ynfD Predicted protein yﬁL Putative lipoprotein yi91a Unknown , in CP4-6 prophage sequence yfjL Unknown , possible prophage gene yagL Unknown , prophage protein sS ( 9 ) sS-Dependent in bioﬁlm-grow cells ( 18 ) eutH , in eutHA operon , is sS-dependent ( 9 ) sS ( 9 ) b0278 1.59 ing Reactivating factor for ethanolamine ammonia lyase Predicted outer membrane protein Hypothetical protein Unknown Unknown , mutant affecting swarming motility Phantom gene b2451 1.59 eutA b3524 b0625 b1964 b2510 b2596 1.61 1.66 1.69 1.74 1.79 yhjG ybeH yedS yfgJ G7353 Upstream of a ribosome modulation factor induced in stationary phase ( yﬁA ) yqiG Unknown , interrupted by IS element Intergenic region including ycbX promoter Unknown Unknown Intergenic region downstream of yfjH Unknown b3046 N.A. b0947 b2623 N.A. b1228 1.81 2.26 2.05 2.30 2.01 3.21 ycbX yfjH ychS Non-coding RNAs ryeE ( cyaR ) CRP , sE ( 77,78 ) b4438 Small RNA , promotes degradation of ompX and nadE RNA Small RNA , inhibits ptsG translation/SgrT protein sibD/sibE non coding RNA - ibsD/ibsE toxic peptides 1.50 sgrS/sgrT 1.51 b4577 SgrR protein ECs3934 1.62 b4447 b4664 Complex locus including two non coding RNAs overlapped by two small ORFs encoding a putative toxin/antitoxin system micA small RNA ( downregulates ompA expression ) not annotated in MG1655 Unknown , annotated in CFT073 Intergenic region between syd ( predicted protein ) and csrB ( ncRNA ) , antisense Unknown , annotated in CFT073 , downstream of malM Between fabB and ycfJ , antisense Unknown , annotated in O157 : H7 SAKAI Intergenic region , upstream of predicted helicase ygcB Unknown , annotated in CFT073 ( antisense of atpC ) Intergenic region upstream of gadA , antisense Unknown , annotated in CFT073 , upstream of aqpZ Unknown , annotated in CFT073 , upstream of dacA , transcribed in opposite direction Intergenic region upstream emrKY ( TolC-multidrug efﬂux pump ) Intergenic region upstream of yohJK ( inner membrane proteins ) Intergenic region between yehI-yehK , antisense Unknown , annotated in CFT073 ( antisense of menB ) Unknown , annotated in CFT073 , upstream of perM gene , transcribed in opposite direction Intergenic region , upstream of yhjB Intergenic region upstream of yqcE Unknown , annotated in CFT073 ( antisense of aspA ) Intergenic region upstream of rrsH ribosomal operon ( transcribed in opposite direction ) b4442 2.00 micA Intergenic regions , genes c3878 1.50 1.50 N.A. N.A. 1.50 N.A. c5008 ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Regulatory regions studied in this work Identiﬁcation ‰ Predicted best DNA site for CRP Position Score ROMA ChIP-Chip w z Gene Gene function ATCTGTGAGTGATTTCACAGTA 259 6.654 Yes No mglB Galactose-binding transport protein , receptor for galactose taxis gatY Tagatose-bisphosphate aldolase 1 TTTTGTGATCGTTATCTCGATA aer Aerotaxis sensor receptor AATTGCGATCTAAATCAAATTA 7.895 Yes 4.054 No No Yes 64 93 sdaC Serine transporter ATTTGAGATCAAGATCACTGAT ychH Hypothetical protein AATTGTGATCACGCCCGCACAT 7.618 Yes 6.771 Yes Yes No 168 94 malX PTS-system , maltose and glucose-speciﬁc IIABC component TAATGTGATTTATGCCTCACTA yffQ Hypothetical protein TTTTGTGATGCAGATCGCTTTT 7.892 Yes 4.098 Yes Yes Yes 83 145 ycdZ Hypothetical protein AAGTGTGATCTACGTCACTCAT fdoG Formate dehydrogenase-O major subunit ACCTGTGATATTTGTCACATTT 5.461 Yes 6.281 No No Yes 44 237 ribB Riboﬂavin biosynthesis ; 3,4-dihydroxy-2-butanone 4-phosphate synthase Yes No yidW Regulator protein for dgo operon TTTTGTGATCTAAATTGTAGTA 144 7.852 Yes No slp Outer membrane protein induced after carbon starvation AAAATTAACTGAGTTCACCTAA 100 11.96 No Yes Sites as predicted by Robison et al. ( 1998 ) ( ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Genes transcribed more efﬁciently in the presence of EsS Genes and Gene product b promoter regions a Transcription levels ( EsS/Es70 ratio ) d Stress response yieF sE ( 59 ) ; sS ( 19 ) sS , RcsBA ( 60 ) sS , RcsBA ( 60 ) Chromate reductase b3713 N.A. 1.50 1.51 osmB Intergenic region including osmB promoter Osmolarity-inducible lipoprotein b1283 b2688 1.51 2.21 gshA g-glutamate -- cysteine ligase ( glutathione-biosynthesis ) ahpF Alkyl-peroxidase reductase pphA Protein phosphatase macA Macrolide resistance efﬂux pump DNA and RNA-metabolism and modiﬁcation ppk Polyphosphate kinase , part of RNA degradosome , Intergenic region including xapA promoter xapA Xanthosine phosphorylase , nucleotide-synthesis/degradation gyrB B subunit of DNA gyrase recT Rac prophage , recombinase , in recET-lar-ydaCQ operon lar Rac prophage , restriction alleviation and modiﬁcation enhancement , in recET-lar-ydaCQ operon ydaC Rac prophage , in recET-lar-ydaCQ operon ydaQ Rac prophage , possible recombinase , in recET-lar-ydaCQ operon Polyamine-metabolism puuB g-glutamyl -- putrescine oxidase gabD Succinate semialdehyde dehydrogenase speB Agmatinase ( in arginine/putrescine-degradation pathway ) Transcription regulation Intergenic region including appY promoter appY Regulator of hyaABCDEF operon yciT Putative deoR-type transcription regulator ilvY Regulator of ilvC operon rhaS Regulator of rhamnose-transport Sugar metabolism treF Trehalase OxyR ( 61 ) sH ( 62,63 ) b0606 b1838 b0878 1.52 1.54 1.54 Regulator of intracellular sS concentrations ( 3 ) XapR ( 64 ) XapR ( 64 ) b2501 1.50 N.A. b2407 1.62 1.56 b3699 b1349 1.65 1.93 Fis ( 65 ) sS-Dependent in bioﬁlm cells ( 18 ) - growing b1348 1.93 b1347 b1346 1.74 1.66 as recT as recT sS ( 19 ) b1301 b2661 b2937 1.61 1.65 1.66 sS , Nac ( 9,66 ) 1.69 1.51 1.56 1.75 1.81 N.A. b0564 b1284 b3773 b3905 H-NS ( 67 ) H-NS ( 67 ) CRP ( 29 ) ; RhaS ( 68 ) sS ( 9,19 ) sS ( 19 ) sS ( 7 ) 1.53 1.55 1.56 1.57 b3519 b4230 b1897 b1901 ytfT Putative galactose ABC transporter otsB Trehalose-6-phosphate phosphatise araF Arabinose transporter , component of an ABC transport system glgP Glycogen phosphorylase , part of glgCAP operon gpmM Putative 2,3-bisphosphoglycerate mutase glvC Sugar transport phosphotransferase Other metabolic functions Intergenic region including hyaABCDEF promoter hyaA Hydrogenase small subunit ; in hyaABCDEF operon ( 69 ) ; sS ( 19 ) CRP , AraC CRP ( 70 ) ; sS controls glgCAP operon ( 19 ) 1.60 b3428 1.61 1.79 b3612 b3683 2.02 N.A. sS , AppY , anaerobic regulation by ArcA and NarP/NarL ( 20,71,72 ) as hyaA 1.68 b0972 Hydrogenase large subunit ; in hyaABCDEF operon Hydrogenase subunit ( nickel-binding protein ) ; in hyaABCDEF operon SecY-interacting protein Acetyl-muramic acid permease Transcriptional repressor of murQP Trans-aconitate methylatransferase Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon hyaB 1.66 b0973 as hyaA hyaF 1.50 b0977 syd 1.59 1.59 1.62 1.60 b2793 b2429 b2427 b1519 murP murR tam CRP , MurR ( 73 ) sS ( 74 ) ydhY FNR , NarL ( 75 ) 1.80 b1674 FNR , NarL ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Gene characteristics Gene Predicted Band no.a Gene function ( reference ) Regulation of corresponding gene ( reference ) productb molecular mass ( Da ) c PﬂB * 85,357 GadA * 52,685 1 Pyruvate formate lyase I ( 56 ) Glutamate decarboxylase ( 64 ) Induced by ArcA and FNR ( 61 ) 2 Positively regulated by rpoS , CRP , gadE , gadW , and gadX ; negatively regulated by H-NS ( 21 , 66 ) Positively activated by rpoS and by CRP ( 33 , 74 ) Positively activated by CRP ( 74 ) ; repressed by ArcA ( 60 ) Same transcription unit as gatZ Positively regulated by rpoS ( 33 , 46 , 71 ) Positively regulated by rpoS ( 2 , 33 , 46 , 71 ) 52,773 TnaA † 3 Tryptophan deaminase ( tryptophanase ) ( 22 ) 47,109 GatZ † 3 Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Quinone reductase ; response-to-oxidative-stress ( 45 ) Bacterial ferritin , DNA-protection during stationary phase ( 39 , 73 ) Transcriptional regulator ( 28 ) 30,812 20,846 18,695 GatY † WrbA * Dps * 4 5 6 , 7 , and 11 24,935 CsgD * 8 Positively regulated by rpoS , hns , and ompR ( 25 , 58 ) Same transcription unit as csgD Positively regulated by FNR ( 18 ) Outer membrane protein ; curli transport component ( 38 ) Outer membrane protein ; receptor for colicin S4 ( 49 ) 30,557 22,928 CsgG * OmpW * 9 10 a As in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate-biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>ﬂuoride or Pefabloc SC ; BCA , bicinchoninic acid ; delta Cn , correlation value ( difference between the ﬁrst hit and the second hit ) ; DTE , dithioerythritol ; DTT , dithiothreitol ; iTRAQ , a multiplexed set of isobaric reagents that yield amine-derivatized peptides ( iTRAQ reagents ; Applied Biosystems , CA ) ( 253 ) ; PMSF , phenylmethylsulfonyl ﬂuoride ; RSp , rank preliminary score ; SELDI-TOF-MS , surface-enhanced laser desorption ionization -- time of ﬂight mass spectrometry ; TCA , trichloroacetic acid ; Xcorr , cross-correlation ( measures how close the spectrum ﬁts to the ideal spectrum ) Biotin carboxyl carrier protein of acetyl-CoA carboxylase AccB P0ABD8 179 , 286 , 287 , 323 4.66 / 16,687.21 4.57 / 22,029 4.74 / 18,338 ( 4 -- 5 ) 5.01 / 18,459 ( 4 -- 5 ) 4.78 / 15,809 ( 4 -- 5 ) Biotin carboxylase 6.65 / 49,320.74 7.58 / 33,321.89 P24182 P0A9Q5 AccC AccD 179 179 Acetyl-CoA carboxylase carboxyl transferase subunit beta Isocitrate lyase Protein function and expressione Acetyl-CoA carboxylase is a heterodecamer of four copies of biotin carboxyl carrier protein ( AccB ) , two copies of biotin carboxylase ( AccC ) , and two copies each of the two subunits of carboxyl transferase ( AccA and AccD ) ; AccC catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA See AccA ; induced by high pH during anaerobic growth 4.57 / 22,029 4.74 / 18,338 ( 4 -- 5 ) 5.01 / 18,459 ( 4 -- 5 ) 4.78 / 15,809 ( 4 -- 5 ) See AccA See AccA 6.65 / 49,320.74 7.58 / 33,321.89 P24182 P0A9Q5 AccC AccD 179 179 Involved in glyoxylate-bypass ; activated by phosphorylation ( on histidine ) and inhibited by phosphoenolpyruvate ( PEP ) , 3-phosphoglycerate , and succinate ; induced by pH change ; increases in the physiological short-term adaptation to glucose limitation The pyruvate dehydrogenase complex contains multiple copies of three enzymatic components : pyruvate dehydrogenase ( E1 ) , dihydrolipoamide acetyltransferase ( E2 ) , and lipoamide dehydrogenase ( E3 ) ; catalyzes the overall conversion of pyruvate to acetyl-CoA and CO2 ; increases during aerobic growth and after benzoic acid treatment See AceE ; increases during aerobic growth , during the low temp at 10 ° C , and after benzoic acid treatment Involved in the activation of acetate to acetyl-CoA and the secretion of acetate ; involved in the synthesis of most of the ATP formed catabolically during anaerobic growth ; induced by low pH during anaerobic growth Aconitate hydratases ( AcnA and AcnB ) serve as a protective buffer against the basal level of oxidative stress that accompanies aerobic growth by acting as a sink for reactive-oxygen-species and by modulating translation of the sodA transcript ; AcnA enhances the stability of the sodA transcript , whereas AcnB lowers its stability ; it is induced by iron and oxidative stress and may have an ironresponsive regulatory function See AcnA AceA 27 , 179 , 287 , 311 5.16 / 47,521.57 P0A9G6 5.19 / 44,130 ( 4.5 -- 5.5 ) 5.12 / 44,130 ( 4.5 -- 5.5 ) 5.01 / 33,761 ( 4.5 -- 5.5 ) 4.99 / 33,624 ( 4.5 -- 5.5 ) 5.01 / 51,702 ( 5 -- 6 ) AceE 5.40 / 99,289 5.46 / 99,537.29 P0AFG8 Pyruvate dehydrogenase E1 component 179 , 228 , 270 , 294 , 321 5.39 / 94,195 ( DIGE 4.5 -- 6.5 ) AceF Dihydrolipoyllysine residue acetyltransferase component of pyruvate dehydrogenase complex Acetate kinase P06959 133 , 179 , 228 , 270 , 294 , 321 5.01 / 77,450 5.09 / 65,964.87 5.09 / 78,105 ( DIGE 4.5 -- 6.5 ) AckA 5.84 / 44,001 5.85 / 43,290.45 P0A6A3 179 , 228 , 294 , 287 , 321 , 323 5.79 / 41,417 ( DIGE 4.5 -- 6.5 ) 5.72 / 31,846 ( 5 -- 6 ) 5.59 / 97,516.00 Aconitate hydratase 1 P25516 AcnA 151 , 278 AcnB 151 , 179 , 278 , 287 , 321 Aconitate hydratase 2 P36683 5.30 / 95,000 ( DIGE 4.5 -- 6.5 ) 5.24 / 75,907 ( 4.5 -- 5.5 ) 5.24 / 93,498.11 Continued on facing pag TABLE 2 -- Continued pI/MW Descriptionb Protein function and expressione Experimentald Theoreticalc AcpP P0A6A8 Acyl carrier protein 3.98 / 8,508.33 A carrier of the growing fatty acid chain in fatty-acid-biosynthesis ; even though AcpP is an abundant protein ( ca. 60,000 copies per cell ) , its small size ( 8.5 kDa ) makes it difﬁcult to detect by typical 2-DE ; it can be detected after subcellular fractionation AcrAB is a drug efﬂux protein with broad substrate speciﬁcity 1 Acriﬂavine resistance protein A Adenosine deaminase Adenine deaminase AcrA P0AE06 179 6.08 / 39,723.72 Add P22333 Ade P31441 ( YicP ) 5.36 / 36,397.46 5.63 / 34,723.76 179 321 5.22 / 61,539 ( DIGE 4.5 -- 6.5 ) Repressed by H-NS but activated by insertion of a variety of insertion elements into a region extending from 145 to 13 relative to the transcription start site ; decreases after benzoic acid treatment AdhE has three activities : alcohol dehydrogenase , acetaldehyde dehydrogenase , and PFL deactivase ; PFL deactivase activity catalyzes the quenching of the pyruvate-formate-lyase catalyst in an iron - , NAD - , and CoA-dependent reaction ; in aerobic conditions , it acts as a hydrogen peroxide scavenger ; induced under anaerobic conditions in the absence of nitrate Catalyzes the reversible transfer of the terminal phosphate group between ATP and AMP ; essential for maintenance and cell-growth ; increases during phosphate limitation Absolutely required for the growth of E. coli in a high-phosphate medium containing glucose-1-phosphate as the sole carbon source ; positively controlled by cAMP-CRP Directly reduces organic hydroperoxides in its reduced dithiol form ; may be directly degraded by ClpXP or modulated by a protease-dependent mechanism ; induced by oxidative stress but repressed by sulfate or cysteine ; increases during high-cell-density cultivation and acid or propionate condition during aerobic growth 6.33 / 95,996.05 Aldehyde-alcohol dehydrogenase P0A9Q7 AdhE 179 Adk 5.49 / 28,491 5.60 / 22,470 5.55 / 23,586.02 Adenylate kinase P69441 179 , 228 , 287 , 293 5.00 / 9,929 ( 4 -- 5 ) 4.99 / 9,991 ( 4.5 -- 5.5 ) 5.48 / 31,661 ( 5 -- 6 ) 5.38 / 43,560.36 Glucose-1-phosphatase P19926 Agp 79 1 AhpC Alkyl hydroperoxide reductase subunit C P0AE08 5.01 / 21,554 5.03 / 20,630.25 27 , 179 , 228 , 266 , 287 , 294 , 308 , 321 , 325 5.09 / 23,123 ( DIGE 4.5 -- 6.5 ) 5.08 / 21,705 ( 4.5 -- 5.5 ) 5.03 / 22,439 ( 4.5 -- 5.5 ) AhpF Serves to protect the cell against DNA damage by alkyl hydroperoxides ; can use either NADH or NADPH as electron donor for direct reduction of redox dyes or of alkyl hydroperoxides when combined with the AhpC protein ; induced by oxidative stress and after benzoic acid treatment 5.44 / 53,485 ( DIGE 4.5 -- 6.5 ) 5.20 / 41,503 ( 5 -- 6 ) 5.47 / 56,177.11 Alkyl hydroperoxide reductase subunit F P35340 179 , 203 , 266 , 287 , 321 Alanyl -- tRNA synthetase Aldehyde dehydrogenase A AlaS 29 , 179 , 228 , 294 5.53 / 96,032.40 5.50 / 96,123 P00957 AldA Acts on lactaldehyde as well as other aldehydes ; induced by growth on fucose , rhamnose , arabinose , and amino acids such as glutamate ; increases in the physiological shortterm adaptation to glucose limitation 5.11 / 52,363 ( DIGE 4.5 -- 6.5 ) 5.29 / 42,547 ( 4.5 -- 5.5 ) 5.53 / 38,913 ( 4.5 -- 5.5 ) 5.43 / 50,034 ( 5 -- 6 ) 5.07 / 52,141.60 P25553 179 , 287 , 308 , 311 , 321 , 325 Continued on following pag Involved in regulation of AMP concentrations ; allosterically activated by Mg-ATP but inactivated by inorganic phosphate ; induced by cAMP at limiting phosphate concentrations Serine beta-lactamase with substrate speciﬁcity for cephalosporins AnsA converts L-asparagine into L-aspartate ; there are two L-asparaginase isoenzymes : L-asparaginase I ( AnsA ) , a low-afﬁnity enzyme located in the cytoplasm , and L-asparaginase II ( AnsB ) , a high-afﬁnity secreted enzyme See AnsA ; induced by cAMP and anaerobiosis It is induced during entry into the stationary phase ; its synthesis is triggered by carbon starvation , phosphate starvation , osmotic shift , or a shift from aerobic to anaerobic conditions ; controlled by S and AppY Catalyzes a purine-salvage reaction , resulting in the formation of AMP that is energically less costly than de novo synthesis Involved in L-arabinose-catabolism 293 Amn P0AE12 AMP nucleosidase 5.90 / 53,994.91 6.38 / 47,693 87 Beta-lactamase AmpC P00811 8.78 / 39,551.14 9.06 / 43,647 ( 6 -- 11 ) 2 179 AnsA P0A962 L-Asparaginase I 5.52 / 37,127.24 AnsB P00805 5.66 / 34,593.94 L-Asparaginase II 179 6.11 / 44,689.86 5.42 / 46,694 Periplasmic AppA protein AppA P07102 13 , 228 , 293 Apt 5.30 / 23,792 ( D 5.26 / 19,858.88 Adenine P69503 IG E 4.5 -- 6.5 ) 179 , 321 phosphoribosyltransferase 5.73 / 25,518.91 6.06 / 29,165 L-Ribulose-5-phosphate 4-epimerase L-Arabinose-binding periplasmic protein AraD P08203 294 Involved in high-afﬁnity L-arabinose membrane transport system ; binds with high afﬁnity to arabinose but can also bind D-galactose ( approx 2-fold reduction ) and D-fucose ( approx 40-fold reduction ) ArcA/ArcB two-component regulatory system represses a wide variety of aerobic enzymes under anaerobic conditions , controls the resistance of E. coli to dyes , and may be involved in the osmoregulation of envelope proteins ; when activated by arcB , it negatively regulates the expression of genes of aerobic function ; it activates the transcription of the plfB operon by binding to its promoter Involved in both the arginine and lysine biosynthetic pathways 5.61 / 33,210.05 AraF P02924 179 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 5.21 / 27,292.02 P0A9Q1 Aerobic respiration control protein ArcA ArcA 125 , 321 Acetylornithine / ArgD P18335 287 , 321 5.75 / 41,241 ( DIGE 4.5 -- 6.5 ) 5.19 / 33,559 ( 5 -- 6 ) 5.80 / 43,635.78 succinyldiaminopimelate aminotransferase Ornithine carbamoyltransferase chain F Ornithine carbamoyltransferase consists of ArgF and ArgI ; involved in ﬁrst step of arginine biosynthetic pathway Involved in seventh step of arginine biosynthetic pathway 5.73 / 30,486 ( 5 -- 6 ) 5.86 / 29,835 ( 5.5 -- 6.7 ) 5.63 / 36,695.95 P06960 ArgF 287 ArgG 179 , 287 , 321 5.27 / 46,667 ( DIGE 4.5 -- 6.5 ) 5.28 / 46,209 ( 4.5 -- 5.5 ) 5.08 / 51,667 ( 5 -- 6 ) 5.43 / 39,776 5.23 / 49,767.20 P0A6E4 Argininosuccinate synthase See ArgF ; repressed during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases its level with increasing growth rate Involved in an arginine-transport system ; increases in the physiological short-term adaptation to glucose limitation ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism P0A6E4 Argininosuccinate synthase 5.46 / 36,775.92 Ornithine carbamoyltransferase chain I Arginyl-tRNA synthetase 228 , 293 P04391 ArgI P11875 ArgS 29 , 109 , 228 , 230 , 294 5.32 / 60,405 5.32 / 64,682.96 ArgT 5.15 / 26,055 ( DIGE 4.5 -- 6.5 ) 5.17 / 26,881 ( 4.5 -- 5.5 ) 5.22 / 25,784.99 Lysine-arginine-ornithine-binding periplasmic protein P09551 179 , 287 , 308 , 311 , 321 Continued on facing pag Involved in sixth step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP See AroA : seventh ( ﬁnal ) step See AroA : third step Stereospeciﬁc condensation of PEP and D-erythrose-4-phosphate , giving rise to 3-deoxy-D-arabino - heptulosonate-7-phosphate ( DAHP ) ; involved in ﬁrst step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP See AroF : ﬁrst step AroA P0A6D3 3-Phosphoshikimate 1-carboxyvinyltransferase 5.37 / 46,095.78 5.34 / 53,476 ( 5 -- 6 ) 5.23 / 50,990 ( 5 -- 6 ) Chorismate synthase P12008 P05194 P00888 AroC AroD AroF 179 5.82 / 39,006.25 5.19 / 27,466.65 5.42 / 38,803.96 3-Dehydroquinate dehydratase Phospho-2-dehydro-3-deoxyheptonate aldolase , Tyr sensitive 179 , 287 321 5.31 / 27,332 ( 4.5 -- 5.5 ) 5.38 / 37,715 ( DIGE 4.5 -- 6.5 ) AroG 179 , 286 Phospho-2-dehydro-3 - P0AB91 6.14 / 38,009.52 6.12 / 39,462 6.13 / 38,005 deoxyheptonate aldolase , Phe sensitive Shikimate kinase I 179 , 274 , 286 , 287 , 321 Involved in ﬁfth step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP ; induced by high pH Involved in the arginine periplasmic transport system ( artPIQMJ gene products ) ; increases after benzoic acid treatment See ArtI ; decreases after benzoic acid treatment See ArtI ; probably responsible for energy coupling to the transport system Involved in amino-acid-biosynthetic pathway : Lys , Met , and Thr Involved in asparagine-biosynthesis ; decreases after benzoic acid treatment Involved in asparagine-biosynthesis AroK 5.26 / 19,406.85 P0A6D7 5.30 / 17,959 5.30 / 19,078 ( DIGE 4.5 -- 6.5 ) 5.18 / 21,250 ( 5 -- 6 ) ArtI 5.32 / 25,042.20 Arginine-binding periplasmic protein 1 P30859 179 , 286 , 287 , 321 5.24 / 26,625 5.27 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.07 / 31,478 ( 5 -- 6 ) 5.94 / 25,009 ( DIGE 4.5 -- 6.5 ) 5.59 / 24,831 ( DIGE 4.5 -- 6.5 ) ArtJ 5.97 / 24,908.11 Arginine-binding periplasmic protein 2 Arginine transport ATP-binding protein ArtP P30860 321 6.17 / 27,022.05 P0AAF6 ArtP 179 Asd Aspartate-semialdehyde dehydrogenase 5.37 / 40,017.88 5.20 / 50,339 ( 5 -- 6 ) 179 , 287 P0A9Q9 AsnA 5.45 / 36,650.55 Aspartate-ammonia ligase P00963 179 , 321 5.45 / 37,876 ( DIGE 4.5 -- 6.5 ) 5.38 / 37,715 ( DIGE 4.5 -- 6.5 ) 5.55 / 62,527.82 AsnB P22106 Asparagine synthetase B ( glutamine hydrolyzing ) P0A8M0 Asparaginyl-tRNA synthetase 179 5.64 / 92,858 5.17 / 52,439.25 AsnS 228 , 236 , 287 , 294 , 321 4.69 / 50,800 ( DIGE 4.5 -- 6.5 ) 5.01 / 38,992 ( 4.5 -- 5.5 ) 5.19 / 52,356.13 5.54 / 43,573.36 Aspartate ammonia-lyase Aspartate aminotransferase P0AC38 P00509 AspA AspC 179 5.40 / 41,960 5.52 / 41,877 179 , 228 , 236 , 287 , 294 5.53 / 50,441 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.42 / 61,614 Changes very little at normal temp range ( 23 -- 37 ° C ) Involved in short-chain-fatty-acid-metabolism ; induced during phosphate limitation and at phosphonate growth F-type ATPases consist of the two complex components CF ( 0 ) , the membrane proton channel , and CF ( 1 ) , the catalytic core ; CF ( 1 ) has ﬁve subunits : alpha ( 3 ) , beta ( 3 ) , gamma ( 1 ) ; delta ( 1 ) , and epsilon ( 1 ) ; CF ( 0 ) has three main subunits : a , b , and c ; produces ATP from ADP in the presence of a proton gradient across the membrane ; increases in the physiological short-term adaptation to glucose limitation See AtpA 5.53 / 50,441 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.42 / 61,614 5.47 / 65,913.45 29 , 109 , 228 , 294 Aspartyl-tRNA synthetase P21889 AspS Acetate CoA-transferase beta subunit P76459 AtoA 5.65 / 22,959.65 5.41 / 22,852 293 AtpA 5.98 / 43,053 5.84 / 53,108 5.80 / 55,222.08 P0ABB0 ATP synthase alpha chain 179 , 228 , 286 , 287 , 294 , 311 , 321 5.82 / 52,204 ( DIGE 4.5 -- 6.5 ) 5.15 / 39,861 ( DIGE 4.5 -- 6.5 ) 5.81 / 36,727 ( 5 -- 6 ) 6.11 / 57,637 ( 6 -- 11 ) 6.02 / 30,657 ( 6 -- 11 ) AtpC 179 , 228 , 287 , 294 5.46 / 14,937.07 5.48 / 14,811 5.34 / 10,379 ( 4.5 -- 5.5 ) 5.45 / 13,773 ( 5 -- 6 ) ATP synthase epsilon chain P0A6E6 Continued on following pag See AtpA ; decreases during phosphate limitation AtpD P0ABB4 ATP synthase beta chain 4.90 / 50,194.23 4.90 / 47,721 4.96 / 48,103 ( DIGE 4.5 -- 6.5 ) 4.99 / 48,103 ( DIGE 4.5 -- 6.5 ) 4.95 / 43,643 ( 4 -- 5 ) 4.95 / 47,782 ( 4 -- 5 ) 4.55 / 43,745 ( 4 -- 5 ) 5.00 / 45,494 ( 4.5 -- 5.5 ) 4.96 / 42,894 ( 4.5 -- 5.5 ) 5.53 / 19,289 ( DIGE 4.5 -- 6.5 ) 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) AtpF P0ABA0 ATP synthase B chain 5.99 / 17,263.96 See AtpA ; decreases after benzoic acid treatment See AtpA See AtpA 179 , 321 P0ABA6 ATP synthase gamma chain P0ABA4 ATP synthase delta chain P0AE52 Putative peroxiredoxin Bcp AtpG AtpH Bcp 8.84 / 31,577.42 4.94 / 19,332.22 5.03 / 17,633.94 179 179 5.02 / 15,872 4.92 / 16,060 179 , 228 , 286 , 294 , 321 5.10 / 17,526 ( DIGE 4.5 -- 6.5 ) 5.34 / 16,350 May perform functions in iron detoxiﬁcation and storage analogous to those of animal ferritins P0ABD3 Bacterioferritin Bfr 4.69 / 18,495.03 294 5.77 / 81,406.5 5.32 / 38,648.09 Periplasmic beta-glucosidase Biotin synthase P33363 P12996 BglX BioB 179 179 Involved in ﬁnal step of biotin-biosynthesis : biotin from 6-carboxyhexanoyl-CoA See BioB : third step Involved in the active translocation of vitamin B12 ( cyanocobalamin ) across the outer membrane to the periplasmic space ; derives its energy for transport by interacting with the transperiplasmic membrane protein TonB May be an auxiliary component of the transport system Involved in amino acid biosynthesis , L-arginine-biosynthesis , and synthesis of carbamoyl phosphate from HCO3 ; decreases during phosphate limitation early See CarA ; decreases signiﬁcantly during phosphate limitation Scavenges exogenous and endogenous cytidine and 2-deoxycytidine for UMP-synthesis ; increases during high-cell-density cultivation and overexpression of recombinant protein in large scale Involved in the transmission of sensory signals from the chemoreceptors to the ﬂagellar motors ; autophosphorylated ; can transfer its phosphate group to either CheB or CheY ; a histidine kinase See CheA ; the active CheY ( phosphorylated or acetylated form ) exhibits enhanced binding to a switch component , FliM , at the ﬂagellar motor , which induces a change from counterclockwise to clockwise ﬂagellar rotation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>, probably by increasing the reduction rate of tellurite to metallic tellurium within the bacterium Hydrolyzes only long-chain acyl thioesters ( C12 -- C18 ) ; has speciﬁcity similar to that of chymotrypsin Involved in thiamine-pyrophosphate-biosynthesis ; catalyzes the phosphorylation of 4-amino-5-hydroxymethyl pyrimidine monophosphate to HMP-pyrophosphate ( HMP-PP ) Involved in thiamine-pyrophosphate-biosynthesis ; condenses 4-methyl-5 - ( beta-hydroxyethyl ) - thiazole monophosphate and HMP-PP to form thiamine monophosphate ( TMP ) Involved in thiamine-biosynthesis ; required for the synthesis of the thiazole moiety of thiamine P69425 TatB 179 P07913 Tdh 5.94 / 37,239.04 -- / 38,800 61 6.84 / 22,530.73 Tellurite resistance protein P25397 TehB 179 P0ADA1 Acyl-CoA thioesterase I TesA 5.97 / 19,936 ( DIGE 4.5 -- 6.5 ) 5.91 / 20,470.33 321 ThiD 179 Phosphomethylpyrimidine kinase 5.73 / 28,633.61 P76422 ThiE Thiamine-phosphate pyrophosphorylase P30137 179 , 321 5.51 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.54 / 23,015.28 ThiG 79 5.36 / 26,896.1 Thiazole biosynthesis protein P30139 1 179 179 Protein Bifunctional Q46948 P00561 5.24 / 20,777.09 5.47 / 89,120.24 ThiJ ThrA Involved in amino acid oserine aspartokinase/hom dehydrogenase Homoserine kinase biosynthesis : Lys , Met , or Thr ThrB Involved in threonine 5.38 / 32,000 ( DIGE 4.5 -- 6.5 ) 5.33 / 37,266 ( 5 -- 6 ) 5.45 / 33,623.65 P00547 179 , 287 , 321 biosynthesis from aspirate : step 4 ; decreases after benzoic acid treatment Continued on facing pag TABLE 2 -- Continued ThrC P00934 Threonine synthase 5.24 / 47,113.84 5.30 / 48,579 See ThrB : step 5 ( ﬁnal ) 5.27 / 46,667 ( DIGE 4.5 -- 6.5 ) 6.00 / 28,285 ThyA P0A884 Thymidylate synthase 5.62 / 30,479.69 Provides the sole de novo source of dTMP for DNA-biosynthesis ; binds to its mRNA , thus repressing its own translation Involved in protein export and acts as a chaperone by maintaining the newly synthesized protein in an open conformation ; increases by high pH during anaerobic growth and overproduction of recombinant protein but decreases during phosphate limitation Catalyzes the NADPH-dependent reductions of 2,5-diketo-D-gluconate to 5-keto-D-gluconate , 2-keto-D-gluconate to D-gluconate , and 2-keto-L-gulonate to L-idonate Suppresses the inhibitory activity of the carbon storage regulator csrA Involved in tryptophan-catabolism ; increases by high pH during aerobic or anaerobic growth but decreases after benzoic acid treatment Involved in the TonB-independent uptake of group A colicins ( colicins A , E1 , E2 , E3 , and K ) ; necessary for the colicins to reach their respective targets after initial binding to the bacteria Required for proper expression of outer membrane protein genes such as ompF and nmpC and those encoding protein 2 , hemolysin , colicin V , and colicin E1 ; induced by low pH during anaerobic growth Plays an important role in several metabolic pathways ; decreases after benzoic acid treatment Has antioxidant activity ; could remove peroxides or H2O2 within the catalase - and peroxidase-deﬁcient periplasmic space ; induced by low or high pH ; decreases after benzoic acid treatment Provides the cells with the ability to utilize trehalose at high osmolarity by splitting it into glucose molecules that can subsequently be taken up by the phosphotransferasemediated uptake system ; induced by growth at high osmolarity or by entry into stationary phase ; regulated by cAMP-CRP Catalyzes the formation of 5-methyl-uridine at position 54 ( M-5-U54 ) in all tRNA ; induced during growth rate-dependent regulation of transcription 294 Tig P0A850 Trigger factor 4.83 / 48,192.67 4.83 / 51,045 138 , 179 , 228 , 287 , 293 , 294 , 321 , 323 4.94 / 52,681 ( DIGE 4.5 -- 6.5 ) 4.96 / 49,707 ( 4 -- 5 ) 4.58 / 49,476 ( 4 -- 5 ) 4.97 / 47,893 ( 4 -- 5 ) 4.87 / 45,401 ( 4 -- 5 ) 4.93 / 48,928 ( 4.5 -- 5.5 ) 4.97 / 48,171 ( 4.5 -- 5.5 ) 4.95 / 47,057 ( 4.5 -- 5.5 ) 5.5 / 35,395.5 TkrA 79 P37666 2-Ketogluconate reductase 1 4.93 / 51,364.09 P0AGG8 Protein TldD 179 TnaA 5.81 / 48,395 ( DIGE 4.5 -- 6.5 ) 5.88 / 52,773.46 P0A853 Tryptophanase 27 , 179 , 274 , 321 , 323 TolB 6.14 / 43,601.64 5.98 / 43,053 5.91 / 44,088 7.05 / 35,331 ( 6 -- 11 ) P0A855 Protein 179 , 228 , 286 , 287 , 294 TolC 5.16 / 50,036 ( DIGE 4.5 -- 6.5 ) 5.23 / 51,468.93 Outer membrane protein P02930 179 , 321 , 323 TpiA 5.57 / 26,972 5.64 / 26,971.81 179 , 228 , 287 , 294 , 321 P0A858 Triosephosphate isomerase 5.51 / 25,962 ( DIGE 4.5 -- 6.5 ) 5.01 / 11,079 ( 4 -- 5 ) Tpx 4.90 / 18,723 4.75 / 17,704.12 P0A862 Thiol peroxidase 179 , 228 , 274 , 287 , 321 4.99 / 19,656 ( DIGE 4.5 -- 6.5 ) 5.02 / 15,046 ( 4 -- 5 ) 5.00 / 17,510 ( 4.5 -- 5.5 ) 5.36 / 60,463.80 P13482 Periplasmic trehalase TreA 103 , 244 P23003 tRNA ( uracil-5 - ) - methyltransferase 79 TrmA 1 5.71 / 41,966.95 Continued on following pag TABLE 2 -- Continued Accession pI/MW no.b Theoreticalc Experimentald Protein nameb TrpA P0A877 Responsible for the aldol cleavage of indoleglycerol phosphate to indole and glyceraldehyde 3-phosphate in tryptophan biosynthetic pathway Responsible for the synthesis of Ltryptophan from indole and L-serine in tryptophan biosynthetic pathway Involved in tryptophan-biosynthesis Tryptophan synthase alpha chain 5.31 / 28,724.16 5.30 / 28,724 5.38 / 28,867 ( 4.5 -- 5.5 ) 29 , 179 , 228 , 287 , 294 5.36 / 27,856 ( 4.5 -- 5.5 ) 5.18 / 32,691 ( 5 -- 6 ) 5.75 / 41,241 ( DIGE 4.5 -- 6.5 ) 5.03 / 33,559 ( 5 -- 6 ) TrpB Tryptophan synthase beta chain P0A879 5.71 / 42,851.82 179 , 287 , 321 TrpD Anthranilate synthase component II Tryptophanyl-tRNA synthetase Thioredoxin 1 P00904 6.05 / 56,738.75 6.08 / 55,915 179 , 228 , 294 P00954 TrpS 179 6.27 / 37,437.82 TrxA P0AA25 Participates in various redox reactions through the reversible oxidation of its active center dithiol to a disulﬁde and catalyzes dithiol-disulﬁde exchange reactions May be directly degraded by ClpXP or modulated by a protease-dependent mechanism ; increases in the acnA and / or acnB mutants but decreases late during phosphate limitation Efﬁcient electron donor for the essential enzyme ribonucleotide reductase Associates with the EF-Tu -- GDP complex and induces the exchange of GDP to GTP ; changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases with increasing growth rate , after benzoic acid treatment , and at low pH during anaerobic growth ; decreases during phosphate limitation Receptor for the attractant L-serine and related amino acids ; responsible for chemotaxis away from a wide range of repellents , including leucine , indole , and weak acids Constitutes the receptor for colicin K and phage T6 and functions as substratespeciﬁc channel for nucleosides and deoxynucleosides Promotes the GTP-dependent binding of aminoacyl-tRNA to the A site of ribosomes during protein-biosynthesis ; may play an important regulatory role in cell-growth and in the bacterial response-to-nutrient deprivation ; changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate and after benzoic acid treatment 4.67 / 11,675.43 4.67 / 11,576 29 , 179 , 228 , 287 , 294 4.80 / 9,648 ( 4 -- 5 ) 4.96 / 9,467 ( 4.5 -- 5.5 ) TrxB P0A9P4 Thioredoxin reductase 5.30 / 34,491.84 5.30 / 34,419 29 , 179 , 228 , 255 , 278 , 293 , 294 , 308 5/15 ,554.77 P0AGG4 Thioredoxin 2 TrxC 179 Tsf 5.15 / 33,623 5.06 / 33,695 5.22 / 30,291.79 P0A6P1 EF-Ts 29 , 109 , 179 , 228 , 230 , 286 , 287 , 293 , 294 , 321 , 323 5.14 / 34,596 ( DIGE 4.5 -- 6.5 ) 5.20 / 34,415 ( DIGE 4.5 -- 6.5 ) 5.08 / 34,633 ( DIGE 4.5 -- 6.5 ) 5.22 / 32,484 ( 4.5 -- 5.5 ) Methyl-accepting chemotaxis protein I P02942 Tsr 4.88 / 59,442.99 179 Nucleoside-speciﬁc channel-forming protein P0A927 Tsx 4.87 / 31,413.41 179 TufA 5.30 / 43,182.39 P0A6N1 EF-Tu 5.32 / 44,615 5.38 / 41,860 29 , 109 , 179 , 228 , 230 , 287 , 294 , 321 5.07 / 32,442 ( DIGE 4.5 -- 6.5 ) 5.69 / 25,098 ( DIGE 4.5 -- 6.5 ) 5.40 / 42,307 ( DIGE 4.5 -- 6.5 ) 5.58 / 45,310 ( 4.5 -- 5.5 ) 5.14 / 30,073 ( 4.5 -- 5.5 ) 5.39 / 51,249 ( 5 -- 6 ) 5.20 / 51,059 ( 5 -- 6 ) 5.32 / 50,990 ( 5 -- 6 ) 5.35 / 50,921 ( 5 -- 6 ) 5.34 / 50,034 ( 5 -- 6 ) 5.31 / 48,144 ( 5 -- 6 ) 5.33 / 47,448 ( 5 -- 6 ) 179 Aromatic amino acid aminotransferase Transcriptional regulatory protein Involved in amino acid biosynthesis of Phe , Tyr , Asp , and Leu Involved in transcriptional regulation of aromatic-amino-acid biosynthesis and transport ; modulates the expression of at least eight unlinked operons ; in most cases , causes negative regulation but has positive effects on the TyrP gene at high phenylalanine concentrations Decreases late during phosphate limitation Involved in menaquinone and ubiquinone-biosynthesis P04693 TyrB 5.32 / 43,537.81 5.54 / 57,656.14 P07604 TyrR 179 P0AGJ9 228 , 236 , 293 , 294 Tyrosyl-tRNA synthetase Ubiquinone / menaquinone-biosynthesis methyltransferase Oxidoreductase Uridine kinase 5.59 / 47,395.78 5.59 / 44,527 TyrS P0A887 179 7.77 / 28,073.21 UbiE 5.13 / 27,849.97 6.39 / 24,353.11 UcpA Udk P37440 P0A8F4 179 Involved in pyrimidine-salvage pathway 179 Continued on facing pag Udp catalyzes the reversible phosphorylytic cleavage of uridine and deoxyuridine to uracil and ribose - or deoxyribose-1-phosphate ; the produced molecules are then utilized as carbon and energy sources or in the rescue of pyrimidine bases for nucleotide-synthesis ; increases during high-cell-density cultivation SN-Glycerol-3-phosphate and glycerophosphoryl diester-binding protein interacts with the binding-protein-dependent transport system UgpACE ; increases in the physiological short-term adaptation to glucose limitation Involved in pyrimidine-salvage pathway ; induced by pyrimidine starvation Uridine phosphorylase 179 , 228 , 287 , 294 , 325 Udp P12758 5.81 / 27,027.89 5.86 / 27,960 5.66 / 31,754 ( 5 -- 6 ) 5.71 / 30,842 ( 5.5 -- 6.7 ) UgpB 311 P0AG80 Glycerol-3-phosphate-binding periplasmic protein 5.98 / 46,123.95 5.98 / 46,123.95 Upp P0A8F0 Uracil phosphoribosyltransferase 5.32 / 22,533.26 5.29 / 23,815 179 , 228 , 294 4.38 / 36,364.13 5.4 / 58,208.8 USG-1 protein Protein P08390 P07024 Usg Degradation of external UDP-glucose to UMP and glucose-1-phosphate , which can then be used by the cell Required for resistance to DNA-damaging agents ; induced during growth inhibition caused by the exhaustion of any of a variety of nutrients ( carbon , nitrogen , phosphate , sulfate , required amino acid ) or by the presence of a variety of toxic agents ; positively regulated by guanosine 3 ,5 - bisphosphate ( ppGpp ) and by a recA/ftsK-dependent regulatory pathway but negatively regulated by FadR ; also regulated by CspC and CspE ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and phosphonate growth and at high pH during anaerobic growth See UspA UshA UspA 5.12 / 15,935.18 5.14 / 15,108 5.20 / 11,633 ( 4.5 -- 5.5 ) P0AED0 Universal stress protein A 71 , 179 , 228 , 287 , 293 , 294 , 323 286 228 5.30 / 34,419 7.40 / 30,751 5.16 / 35,575.69 5.60 / 16,016.53 6.03 / 15,935.18 P0AAC0 Universal stress protein E P37903 Universal stress protein F P39177 Universal stress protein G UspE UspF UspG Interacts with GroEL ; induced by starvation and heat shock and in response to some toxic agents Seems to enhance the formation and/or stability of noncovalent complexes between the Trp repressor protein and operatorbearing DNA ; alone does not interact with the operatorbearing DNA ; major target species probably the TrpR/TrpO complex ; may function as an accessory element in blocking TrpR-speciﬁc transcriptional processes that might be physiologically disadvantageous in the stationary phase of the bacterial life cycle ; induced by entry into stationary phase Confers a modal distribution of chain length on the O-antigen component of lipopoly-saccharide ; gives rise to a reduced no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>179 , 287 Flagellar hook-associated protein 3 RNA polymerase sigma factor for ﬂagellar operon FlgL P29744 Sigma factors are initiation factors that promote the attachment of RNA polymerase to speciﬁc initiation sites and are then released ; speciﬁc for class 3 ﬂagellar operons Polymerizes to form the ﬁlaments of bacterial ﬂagella 179 P0AEM6 FliA FliC Flagellin P04949 4.58 / 45,401 ( 4 -- 5 ) 4.63 / 45,613 ( 4 -- 5 ) 4.55 / 45,507 ( 4 -- 5 ) 4.98 / 45,932 ( 4 -- 5 ) 4.61 / 45,507 ( 4 -- 5 ) 4.54 / 45,507 ( 4 -- 5 ) 5.00 / 44,877 ( 4 -- 5 ) 4.96 / 42,894 ( 4.5 -- 5.5 ) 179 , 287 4.50 / 51,163.80 Required for the morphogenesis and for the elongation of the ﬂagellar ﬁlament by facilitating polymerization of the ﬂagellin monomers at the tip of growing ﬁlament Three proteins ( FliG , FliN , FliM ) form a switch complex that is proposed to be located at the base of the basal body ; interacts with the CheY and CheZ chemotaxis proteins , in addition to contacting components of the motor that determine the direction of ﬂagellar rotation Needed for ﬂagellar regrowth and assembly See FliG Part of a binding-protein-dependent transport system for cystine ; repressed by sulfate or cysteine FliD 4.82 / 48,325.26 Flagellar hook-associated protein 2 P24216 179 , 287 4.69 / 36,775.98 Flagellar motor switch protein P31067 FliG 179 4.62 / 25,050.16 Flagellar assembly protein P31068 FliH 179 5.47 / 37,849.16 5.29 / 26,068.48 Flagellar motor switch protein Cystine-binding periplasmic protein P06974 P39174 FliM FliY 179 179 , 286 , 287 , 321 5.01 / 26,213 5.11 / 25,809 5.17 / 25,686 ( DIGE 4.5 -- 6.5 ) 5.19 / 26,500 ( 4.5 -- 5.5 ) 5.23 / 28,594 ( 4.5 -- 5.5 ) Controls colony form variation and autoaggregation ; may function as an adhesin Modiﬁes the free amino group of the aminoacyl moiety of methionyl-tRNA ( fMet ) ; the formyl group appears to play a dual role in the initiator identity of N-formylmethionyl-tRNA by promoting its recognition by IF2 and impairing its binding to EF-Tu -- GTP Increases during phosphate limitation and phosphonate growth but decreases after benzoic acid treatment Two distinct membrane-bound FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion ; the fumarate reductase is used in anaerobic growth , and the succinate dehydrogenase is used in aerobic growth See FrdA Flu 5.37 / 101,278.83 Antigen 43 P39180 1 Fmt 5.56 / 34,037.27 5.49 / 35,885 Methionyl-tRNA formyltransferase P23882 179 , 228 , 236 , 294 FolA P0ABQ4 Dihydrofolate reductase 179 , 228 , 293 , 294 , 321 5.01 / 19,989 4.84 / 17,999.38 5.02 / 21,096 ( DIGE 4.5 -- 6.5 ) Fumarate reductase ﬂavoprotein 5.87 / 65,840.41 subunit P00363 FrdA 179 179 P0AC47 Fumarate reductase iron-sulfur 6.09 / 26,991.87 protein P0A805 Ribosome recycling factor 6.44 / 20,638.57 6.16 / 21,725 FrdB Responsible for the release of ribosomes from mRNA at the termination of protein-biosynthesis ; may increase the efﬁciency of translation by recycling ribosomes from one round of translation to another FrdB Frr 179 , 228 , 294 The phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in fructose-transport ; induced by fructose ; decreases after benzoic acid treatment May be involved in anomalous ﬁlament growth ; may be a component of the septum and interact with FtsZ ; induced at the stationary phase Essential to the cell-division process ; seems to assemble into a dynamic ring on the inner surface of the cytoplasmic membrane at the place where division will occur , with the formation of the ring being the signal for septation to begin ; induced at the stationary phase Involved in second step of fucose-metabolism Involved in fourth step of fucose-metabolism Functions in the generation of fumarate for use as an anaerobic electron acceptor Acts as a global negative controlling element , employing Fe2 as a cofactor to bind the operator of the repressed genes ; regulates the expression of several outer membrane proteins , including the iron-transport operon ; induced during phosphate limitation FruB P69811 M ultiphosphoryl transfer protein 4.77 / 39,647.83 4.85 / 38,938 ( D IG E 4.5 -- 6.5 ) 179 , 321 P0ABH0 Cell division protein FtsA 103 5.84 / 45,329.97 FtsZ 4.65 / 40,323.92 4.63 / 40,491 4.83 / 39,220 ( 4 -- 5 ) 4.77 / 39,129 ( 4 -- 5 ) P0A9A6 Cell division protein 103 , 179 , 228 , 287 , 294 P11553 Fuck L-Fuculokinase 5.38 / 53,235.87 4.73 / 55,145 286 Lactaldehyde reductase 5.1 / 40,644.56 P0A9S1 FucO 179 Fumarate hydratase class 5.88 / 60,105.32 I , anaerobic P14407 FumB 179 Fur 179 , 228 , 287 , 293 , 294 P0A9A9 Ferric uptake regulation 5.68 / 16,794.85 5.79 / 17,433 protein 5.78 / 17,897 ( 5 -- 6 ) 5.78 / 17,233 ( 5 -- 6 ) Promotes the GTP-dependent translocation of the nascent protein chain from the A site to the P site of the ribosome ; decreases during phosphate limitation early GadA converts glutamate to gamma-aminobutyrate , consuming one intracellular proton in the reaction ; the Gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions ; the ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria ; induced by high pH during anaerobic growth Involved in third step of galactose-metabolism ; increases after benzoic acid treatment Involved in lipopolysaccharide-biosynthesis Converts alpha-aldose to the beta-anomer ; active on D-glucose , L-arabinose , D-xylose , D-galactose , maltose , and lactose May play a role in stationary-phase survival FumB 179 Fur 179 , 228 , 287 , 293 , 294 P0A9A9 Ferric uptake regulation 5.68 / 16,794.85 5.79 / 17,433 protein 5.78 / 17,897 ( 5 -- 6 ) 5.78 / 17,233 ( 5 -- 6 ) FusA P0A6M8 Elongation factor G 179 , 287 , 293 , 321 5.29 / 85,046 ( DIGE 4.5 -- 6.5 ) 5.20 / 84,325 ( DIGE 4.5 -- 6.5 ) 5.51 / 46,525 ( DIGE 4.5 -- 6.5 ) 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.78 / 32,101 ( DIGE 4.5 -- 6.5 ) 5.21 / 75,514 ( 4.5 -- 5.5 ) 5.23 / 50,424 ( 5 -- 6 ) 5.24 / 77,450.11 5.22 / 52,685.16 Glutamate decarboxylase alpha P69908 GadA 27 GalE P09147 179 , 321 5.91 / 35,787 ( DIGE 4.5 -- 6.5 ) 5.89 / 37,265.11 UDP-glucose 4-epimerase 5.73 / 32,829.23 P0AAB6 UTP-glucose-1-phosphate uridylyltransferase Aldose-1-epimerase 179 GalF GalM 179 , 287 4.84 / 38,190.46 P0A9C3 4.97 / 39,311 ( 4 -- 5 ) 4.83 / 39,220 ( 4 -- 5 ) 1 P0AEP3 UTP-glucose-1-phosphate 5.11 / 32,811.07 uridylyltransferase 79 GalU Continued on following pag TABLE 2 -- Continued GapA 6.58 / 35,401.30 6.93 / 23,085 Involved in the pathway of glycolysis ; 6.28 / 36,748 induced by high pH during anaerobic 6.58 / 36,386 growth 5.93 / 22,606 ( DIGE 4.5 -- 6.5 ) 5.94 / 36,106 ( DIGE 4.5 -- 6.5 ) 5.32 / 9,085 ( 4.5 -- 5.5 ) 5.03 / 10,612 ( 4.5 -- 5.5 ) 5.74 / 40,430 ( 5 -- 6 ) 6.11 / 34,907 ( 6 -- 11 ) 5.87 / 30,811.93 179 , 228 , 286 , 287 , 294 , 321 , 323 P0A9B2 Glyceraldehyde 3-phosphate dehydrogenase A P37192 Catalyzes the reversible condensation of dihydroxyacetone phosphate with 27 , 311 , 323 Tagatose-1 ,6 - bisphosphate aldolase GatY glyceraldehyde 3-phosphate to produce tagatose-1 ,6 - bisphosphate ; induced by low pH during aerobic or anaerobic growth ; increases in the physiological short-term adaptation to glucose limitation Probably involved in energy conservation rather than in sugar metabolism 5.36 / 40,015.52 The glycine-cleavage-system catalyzes the degradation of glycine Gcd 179 Quinoprotein glucose dehydrogenase 5.4 / 86,747.35 P15877 Aminomethyltransferase ; glycine-cleavage-system T protein NADP-speciﬁc glutamate dehydrogenase P27248 GcvT 179 GdhA 5.98 / 48,581.37 5.96 / 45,327 Decreases during phosphate limitation ; 5.81 / 46,385 ( DIGE 4.5 -- 6.5 ) may be directly degraded by ClpAP or 5.94 / 44,524 ( DIGE 4.5 -- 6.5 ) ClpXP , respectively , or be modulated by a protease-dependent mechanism P00370 4.81 / 38,712.2 P0A9S5 Glycerol dehydrogenase GldA Induced by hydroxyacetone and stationary-phase growth conditions 179 Involved in the conversion of UDP-galactose proton symporter ( GalP ) into UDP-GalF through a 2-keto intermediate for lipopolysaccharide-biosynthesis Involved in glycogen-synthesis ; induced strongly at the early stationary phase and also less osmotic induction ; regulated by S and cAMP-CRP Catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate Responsible for the acetylation of glucosamine-1-phosphate ( Glc-N-1-P ) to give N-acetylglucosamine-1-phosphate ( GlcNAc-1-P ) and the synthesis of UDP-GlcNAc 5.26 / 51,772.57 5.25 / 53,849 Controlled by adenylation under 5.18 / 53,743 conditions of abundant glutamine ; 5.39 / 36,675 decreases during phosphate limitation 5.30 / 53,162 ( DIGE 4.5 -- 6.5 ) and after benzoic acid treatment ; 5.00 / 26,795 ( 4 -- 5 ) regulated positively by Lrp ; may be directly degraded by ClpAP or ClpXP , respectively , or be modulated by a protease-dependent mechanism 6.62 / 42,965.88 P37747 UDP-galactopyranose mutase Glf 179 Glycogen synthesis protein 5.38 / 7,891.88 P26649 GlgS 103 , 105 P31120 Phosphoglucosamine mutase GlmM 5.71 / 47,412.38 P0ACC7 Bifunctional protein 6.09 / 49,190.08 GlmU 179 GlnA P0A9C5 Glutamine synthetase 61 , 179 , 228 , 286 , 287 , 293 , 294 , 308 , 321 P-II indirectly controls the transcription of the glutamine synthetase gene ( glnA ) ; prevents nitrogen regulatory ( NR ) - II-catalyzed conversion of NR-I to NR-I-phosphate , the transcriptional activator of glnA ; when P-II is uridylylated to P-II-UMP , these events are reversed ; when the ratio of Gln to 2-ketoglutarate decreases , P-II is uridylylated to P-II-UMP , which causes the deadenylylation of glutamine synthetase by GlnE , thus activating the enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The phosphoryl group from PEP is transferred to the phosphoryl carrier protein HPr by enzyme I ( PtsI ) ; phosphoHPr then transfers it to the permease ( enzymes II/III ) ; HPr is common to all PTS ; regulated by the cAMP-CRP complex and also by growth on glucose ; induced by low pH and phosphonate growth See PtsH ; decreases after benzoic acid treatment but increases at phosphonate growth Plays an important role in de novo pathway of purine-nucleotide-biosynthesis : AMP-biosynthesis ; increases after benzoic acid treatment Involved in IMP-biosynthesis : 5-amino-1 - ( 5-phospho-D-ribosyl ) imidazole-4 - carboxamide from N ( 2 ) - formyl-N ( 1 ) - ( 5-phospho - dribosyl ) glycinamide ( step 5 ) See PurB : step 4 ; increases after benzoic acid treatment See PurB : step 2 ; increases after benzoic acid treatment See PurB : step 3 PstS P0AG82 Phosphate-binding periplasmic 6.92 / 34,421.76 6.85 / 45,258 ( 6 -- 11 ) protein Pta P0A9M8 Phosphate acetyltransferase 5.28 / 77,040.90 5.29 / 85,417 179 , 228 , 274 , 293 , 294 PtrA 179 Protease III P05458 5.66 / 105,113.29 5.65 / 9,119.37 5.65 / 10,366 P0AA04 Phosphocarrier protein HPr PtsH 27 , 228 , 293 , 294 PtsI 4.78 / 59,810 Phosphoenolpyruvate-protein phosphotransferase P08839 56 , 179 , 228 , 293 , 294 , 321 4.78 / 63,561.84 4.91 / 63,671 ( DIGE 4.5 -- 6.5 ) PurA P0A7D4 Adenylosuccinate synthetase 179 , 321 5.30 / 44,524 ( DIGE 4.5 -- 6.5 ) 5.30 / 43,494 ( DIGE 4.5 -- 6.5 ) 5.32 / 47,213.76 PurB 179 5.68 / 51,542.81 P0AB89 Adenylosuccinate lyase PurC 5.09 / 26,148 ( DIGE 4.5 -- 6.5 ) 5.07 / 26,995.00 P0A7D7 Phosphoribosylaminoimidazolesuccinocarboxamide synthase Phosphoribosylamine-glycine ligase Phosphoribosylaminoimidazole carboxylase catalytic subunit Bifunctional purine biosynthesis protein Phosphoribosylaminoimidazole carboxylase ATPase subunit 179 , 321 PurD 179 , 321 5.06 / 47,667 ( DIGE 4.5 -- 6.5 ) 4.96 / 45,940.36 P15640 PurE 179 , 321 6.05 / 18,899 ( DIGE 4.5 -- 6.5 ) 6.03 / 17,649.13 P0AG18 See PurB : ﬁnal step ; increases after benzoic acid treatment See PurB : step 3 ; possesses an ATPase activity that is dependent on the presence of AIR ( aminoimidazole ribonucleotide ) ; the association of PurK and PurE produces an enzyme complex capable of converting AIR to CAIR efﬁciently under physiological condition See PurB : step 2 PurH 179 , 321 5.50 / 57,000 ( DIGE 4.5 -- 6.5 ) 5.53 / 57,329.21 P15639 5.75 / 28,100 ( 5 -- 6 ) 5.60 / 39,461.12 P09029 PurK 287 179 P08178 Phosphoribosylformylgly-cinamidine cyclo-ligase HTH-type transcriptional repressor 4.83 / 36,722.85 PurM Repressor that binds to the purF operator and coregulates other genes for de novo purine-nucleotide-synthesis ; involved in regulation of purB , purC , purEK , purHD , purL , purMN , and guaBA expression ; binds hypoxanthine and guanine as inducers 4.83 / 36,722.85 PurM P0ACP7 6.3 / 38,043.62 PurR 179 Oxidizes proline to glutamate for use as a carbon and nitrogen source and also functions as a transcriptional repressor of the put operon ; induced by proline Involved in ﬁnal step of glycolysis Involved in UMP-biosynthesis : UMP from HCO3 ( step 2 ) See PyrB : step 3 See PyrB : step 4 See PyrB : step 6 ( ﬁnal ) Bifunctional protein 5.69 / 143,815.16 PykF Pyruvate kinase I P0AD61 5.77 / 50,729.42 5.65 / 56,143 ( DIGE 4.5 -- 6.5 ) 179 , 321 PyrB Aspartate carbamoyltransferase catalytic chain Dihydroorotase Dihydroorotate dehydrogenase Orotidine 5 -- phosphate decarboxylase CTP synthase P0A786 6.13 / 34,296.17 5.70 / 93,664 6.13 / 35,321 179 , 212 , 228 , 294 P05020 P0A7E1 P08244 PyrC 5.77 / 38,696.19 7.66 / 36,774.52 5.81 / 26,350.24 179 179 , 287 PyrD PyrF 7.28 / 47,786 ( 6 -- 11 ) 5.76 / 56,695 179 , 228 , 294 , 321 5.83 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.61 / 58,475 ( DIGE 4.5 -- 6.5 ) Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen ; allosterically activated by GTP when glutamine is the substrate and also activated by magnesium but inhibited by CTP and by divalent metal ions such as copper and zinc Catalyzes the phosphorylation of UMP to UDP , with ATP as the preferred donor Involved in allosteric regulation of aspartate carbamoyltransferase Essential for efﬁcient processing of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; cold shock protein essential for E. coli cells to adapt to low temp PyrG P0A7E5 5.63 / 60,242.96 179 , 321 7.07 / 25,839.07 P0A7E9 Uridylate kinase PyrH PyrI P0A7F3 Aspartate carbamoyltransferase 6.84 / 16,989.44 6.85 / 17,023 regulatory chain 7.01 / 17,642 ( 6 -- 11 ) 179 , 212 , 228 , 287 , 294 RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 Involved in the high-afﬁnity D-ribose membrane transport system and also serves as the primary chemoreceptor for chemotaxis ; repressed in the presence of glucose but induced in the physiological short-term adaptation to glucose limitation Involved in ribose metabolism Catalyzes the hydrolysis of ATP in the presence of single-stranded DNA , the ATP-dependent uptake of single-stranded DNA by duplex DNA , and the ATP-dependent hybridization of homologous single-stranded DNAs ; interacts with LexA , causing its activation and leading to its autocatalytic cleavage ; induced by low temp at 10 ° C as well as by cadmium chloride , hydrogen peroxide , and ACDQ Counteracts the effect of relE by means of direct proteinprotein interaction , enabling the reversion of translation inhibition ; acts as an autorepressor of relBE transcription ; increased transcription rate of relBE and activation of relE is consistent with a lower level of relB in starved cells due to degradation of relB by protease lon ; induced by amino acid starvation RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 RbsB P02925 6.85 / 30,900 5.99 / 28,474.47 D-Ribose-binding periplasmic protein 99 , 179 , 311 , 321 E 4.5 -- 6.5 ) IG 5.92 / 29,066 ( D 4.99 / 32,290.52 P0A9J6 Ribokinase P0A7G6 Protein RecA RbsK RecA 179 5.09 / 37,842.18 5.08 / 41,137 133 , 179 , 228 , 294 , 297 4.81 / 9,071.48 4.81 / 9,100 Antitoxin P0C079 RelB 99 Continued on facing pag Catalyzes the interconversion between ADP-D-glycero-beta-D-manno-heptose and ADP-L-glycero-beta-D-manno-heptose via an epimerization at carbon 6 of the heptose ; completely inhibited by ADP and ADP-glucose and partially inhibited by ATP and NADH ; induced by heat shock Involved in carbohydrate-biosynthesis and dTDP-L-rhamnose-biosynthesis See RfbB RfaD P67910 ADP-L-glycero-D-manno-heptose-6-epimerase 4.80 / 34,893.17 4.85 / 36,820 4.94 / 36,106 ( DIGE 4.5 -- 6.5 ) 4.98 / 35,245 ( 4 -- 5 ) P37759 RfbB 179 dTDP-glucose 4,6-dehydratase 5.47 / 40,558.33 RfbC P37745 dTDP-4-dehydrorhamnose 3,5 - 5.48 / 21,270.11 epimerase P0AG30 Transcription termination factor 6.75 / 47,004.21 179 Facilitates transcription Rho termination by a mechanism that involves rho binding to the nascent RNA , activation of rho 's RNA-dependent ATPase activity , and release of the mRNA from the DNA template Involved in riboﬂavin biosynthesis ; repressed by heat shock but induced by low pH Involved in ﬁnal steps of riboﬂavin synthesis ; riboﬂavin synthase is a bifunctional enzyme complex catalyzing the formation of riboﬂavin from 5-amino-6 - ( 1 - D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione and L-3 ,4 - dihydrohy-2-butanone-4 - phosphate via 6,7-dimethyl-8-lumazine ; the alpha subunit catalyzes the dismutation of 6,7-dimethyl-8-lumazine to riboﬂavin and 5-amino-6 - ( 1-D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione See RibE ; decreases after benzoic acid treatment Acetylates the N-terminal serine of ribosomal protein L7/L12 ; increases during phosphate limitation and phosphonate growth Essential for efﬁcient processing of 16S rRNA ; probably part of the 30S subunit prior to or during the ﬁnal step in the processing of 16S free 30S ribosomal subunits ; could be some accessory protein needed for efﬁcient assembly of the 30S subunit ; needed in a step prior to RbfA during the maturation of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; decreases after benzoic acid treatment P0A7J0 3,4-Dihydroxy-2-butanone 4-phosphate synthase 4.90 / 23,353.47 RibB 274 RibE P0AFU8 Riboﬂavin synthase alpha chain 5.64 / 23,444.90 5.67 / 26,821 ( 5 -- 6 ) 5.73 / 25,731 ( 5.5 -- 6.7 ) 179 , 287 RibH 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.19 / 12,875 ( 4.5 -- 5.5 ) 5.42 / 40,651 5.15 / 16,156.51 P61714 6,7-Dimethyl-8-ribityllumazine synthase Ribosomal protein-serine acetyltransferase 179 , 287 , 321 5.86 / 20,680.55 P13857 RimL 228 , 293 , 294 4.68 / 24,436 ( DIGE 4.5 -- 6.5 ) 4.61 / 20,605.43 P0A7X6 16S rRNA-processing protein RimM 321 179 321 5.09 / 35,712.71 5.39 / 32,693.56 Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA Catalyzes the formation of dTDP-glucose from dTTP and glucose-1-phosphate as well as its pyrophosphorolysis ; decreases after benzoic acid treatment Digests double-stranded RNA ; involved in the processing of rRNA precursors and of some mRNAs Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA 5.41 / 32,544 ( DIGE 4.5 -- 6.5 ) RmlA1 6.4 / 25,550.04 P0A7Y0 RNase III Rnc 179 Matures 5S rRNA from its precursors from all the rRNA genes ; cleaves RNA I , a molecule that controls the replication of colE1 plasmid DNA ; the major endoribonuclease participating in mRNA turnover in E. coli May be part of a membrane complex involved in electron transport Responsible for the end turnover of tRNA : speciﬁcally removes the terminal AMP residue from uncharged tRNA ( tRNA-C-C-A ) ; involved in tRNA-biosynthesis , especially in strains lacking other exoribonucleases P21513 Rne RNase E 5.48 / 118,196.73 Electron transport complex protein P77285 RnfG 179 6.59 / 21,911.91 RNase T P30014 Rnt 179 5.19 / 23,522.7 5.13 / 24,554.25 Ribulose-phosphate 3-epimerase Ribose-5-phosphate isomerase A P0AG07 Rpe 179 Involved in nonoxidative branch of the pentose-phosphate-pathway ; base-induced protein One of the primary rRNA-binding proteins ; binds very close to the 3 end of the 23S rRNA and forms part of the L1 stalk ; translational repressor protein ; controls the translation of the L11 operon by binding to its mRNA See RplA ; increases at phosphonate growth RpiA 5.20 / 22,860.40 5.06 / 25,943 P0A7Z0 179 , 274 , 286 RplA 50S ribosomal protein 9.64 / 24,598.44 8.24 / 25,577 ( 6 -- 11 ) L1 P0A7L0 179 , 287 RplI 6.20 / 19,831 6.17 / 15,769 6.17 / 15,769.06 50S ribosomal protein L9 P0A7R1 29 , 179 , 228 , 286 , 287 , 293 , 294 6.71 / 10,366 5.10 / 8,830 ( 4.5 -- 5.5 ) 6.36 / 15,124 ( 6 -- 11 ) 9.04 / 17,580.4 Translational repressor protein ; controls the translation of the rplJL-rpoBC operon by binding to its mRNA Seems to be the binding site for several of the factors involved in protein-synthesis and appears to be essential for accurate translation ; decreases during phosphate limitation Binds the 5S rRNA ; required for the late stages of subunit assembly Binds to 23S rRNA in the presence of protein L20 One of the proteins that binds to the 5S RNA in the ribosome , where it forms part of the central protuberance ; binds to the 5S rRNA independently of L5 and L18 ; not required for binding of the 5S rRNA/L5/L18 subcomplex to 23S rRNA May be induced under zinc-limiting conditions ; may be repressed in that case by the zinc uptake regulation protein Zur DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Operons activated by wild-type CRP in ROMA experiment Operon name ROMA CRP-binding site Gene product a WT HL HL/wt KE KE/wt Position Site sequence Score b c d e f g h Gene function # acrE -- 1.86 1.35 1.89 289 AATCGTTAAATAAATAATATAT 7.2 Transmembrane protein affects septum formation Coa-linked acetaldehyde dehydrogenase and iron-dependent alcohol dehydrogenase Negative response regulator of genes in aerobic pathways Membrane ; inner membrane # # adhE 2.49 1.13 1.02 241 AAATTTGATTTGGATCACGTAA 18.72 Enzyme ; energy metabolism , carbon : Fermentation 1.84 1.11 # 0.99 # arcA 295 GAACTTGATATATGTCAACGAA 6.78 Regulator ; global regulatory functions 358 33 TAATGTGACGAAAGCTAGCATT TAATGTTGTATCAACCACCATA 5.12 6.42 argR 1.86 1.58 -- 2.05 -- Repressor of arg regulon Regulator ; amino acid biosynthesis : Arginine 193 335 CAATTTGATAACAATTAATTTA CATCGTGACCAGGATGACATTA 5.65 8.37 1.98 0.91 # b0846 2.42 -- Putative DEOR-type transcriptional regulator Putative regulator ; not classified 40 TTGTGTACATTTGTACACAATT TATATTGATACTTAAAACATTT 7.94 6.94 2.3 1.17 # b1329 2 -- 101 Putative transport periplasmic protein Putative transport ; not classified 38 TTCTTTAAGGCGAAACAAATAA GGATGTGAAATTAATCACAGTA 5.8 1.94 1.72 -- 4.56 1.2 # b1431 1.59 -- 1.56 # Orf ; unknown 216 183 14.71 13.9 Orf , hypothetical protein Putative ATP-binding component of a transport system Orf , hypothetical protein Orf , hypothetical protein Putative transport ; not classified ATCTGTGATGGCAACCACAGTT 1.44 # 1.04 # 2.23 1.9 -- 2.52 1.17 # 29 TATTATAATCCTATTCAATTAT TTTTGTGATGCAGATCGCTTTT 7.03 16.65 5.4 16 9.14 13.48 13.2 Orf ; unknown Orf ; unknown 156 4 69 229 31 257 AACTGTGAAGGAGGACGCCATG TTTTGTGATCTGCGTCAATATT 1.44 # b2450 2.35 1.16 # Orf , hypothetical protein Orf ; unknown CAATGAAACTGAGTTCAAACTT CTTAGTGATCTACCTCACCTTT # # # # # # # b2463 1.18 0.99 1.25 1.62 1.4 1.07 0.93 1.02 2.02 1.51 2.03 2.72 1.7 3.31 2.13 Orf , hypothetical protein Putative multimodular enzyme Orf , hypothetical protein Putative flavodoxin Putative dehydrogenase Orf ; unknown ATGAGTGCGTTAATTCACACTT Putative enzyme ; not classified Orf ; unknown Putative enzyme ; not classified Putative enzyme ; not classified # # -- 90 90 51 TAGAGTAAAAACAATCAGATAA TTATGTGAATCAGATCACCATA 7.05 18.82 5.14 TTATATTCACAATATCAAACAA 1.26 # 1.41 # b2740 2.26 1.58 -- 3.24 1.03 # Putative transport protein Orf , hypothetical protein Putative transport ; not classified Orf ; unknown 53 379 TTTTGATACTTGTATCAAGAAT GATATTGCTCTGGTTCGCATAA 7.57 6.7 7.2 6.4 5.15 1.84 -- 2.47 1.96 -- TATCATGATATCGATAACCATA TTTTGCGATTGAGGCAAAATAT Putative hydrogenase subunit Putative enzyme ; not classified # # # # # # b3001 1.42 1.2 1.03 1.63 1.42 0.88 0.88 0.92 2.62 1.98 1.65 2.05 ATACGTGATGTACTCAGCAACA Putative reductase Putative enzyme ; not classified Orf ; unknown Phenotype ; not classified Enzyme ; transport of small molecules : carbohydrates , organic acids , alcohols Putative regulator ; not classified Orf , hypothetical protein Detox protein PEP-dependent phosphotransferase enzyme IV for cellobiose Putative 2-component transcriptional regulator for second curli operon # -- 201 AAATGTGAAGAGGGTCATAACC 11.91 1.12 # 1.78 1.17 # 8.81 201 csgDEFG TTTAGTTACATGTTTAACACTT 5.86 5.25 9.95 35 AGGTGTGCGATCAATAAAAAAA TTATTAGTAAGTTATCACCATT 329 245 2.13 1.23 # Cytochrome o ubiquinol oxidase subunit II 2.46 -- cyoABCDE Enzyme ; energy metabolism , carbon : aerobic respiratio ATAATTGTTTTATTTCACATTG Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Continued Gene function ROMA CRP-binding site Gene product WT HL HL/wt KE KE/wt Position Site sequence Score b c d e f g h 1.88 0.82 # 2.01 1.09 # 2.95 0.84 # 2.09 1.33 # 7.82 1.52 # 1.27 # 1.34 # 1.03 # 1.26 # 2.75 # AACGGTGCTGGGATTTACGCTT 5.95 IS186 hypothetical protein IS186 and IS421 hypothetical protein ATGTGCGCTATAAGGCAAATCT 5.7 Putative transport protein ATTCGTGATCGCTTTCATGCTT 10.02 Putative transcriptional regulator TTTTGTGATCTAAATTGTAGTA 10.24 Regulator protein for dgo operon AAACGCGAACGTCATCACGCTG 8.31 TTTTGCAACCGTAATCACACTT 12.09 Orf , hypothetical protein TAATGTGCCATAAAACAAGCAA 8.42 AGTCGTGGTATGAATCACTTCT 8.59 Putative oxidoreductase AAAATTGACAGCCGTCACTTTT 11.71 Putative kinase Orf , hypothetical protein Putative transcriptional regulator ATATTTGATATCATCCAGGTAT 6.84 Putative transport ATTTGCGAACGTCTTCACCATC 8.45 Orf , hypothetical protein GTTTGTGATTTTCAAAACGCAT 6.6 AAATGAGCGGCAGATTAAAAAA 9.06 Putative ligase TAGAGTGAAATATGTTAAGAAG 5.44 Putative transport protein Orf , hypothetical protein AACCGCGAGAGAGATCAAATAA 10.71 Orf , hypothetical protein TTTTGTGAAAACACACGCATAA 8.94 Orf , hypothetical protein Orf , hypothetical protein TTATTTGAGATTATTAATATAT 8.26 Putative transport protein Putative kinase Putative transport protein Orf , hypothetical protein Orf , hypothetical protein AACGTCGCTGAAATTCACATTT 6.03 Orf , hypothetical protein AAGTGTGATGTAACGCAATCTG 8.65 Putative LACI-type transcriptional regulator Putative ATP-binding component of a transport system AGATGTGAATGATGAAACCATA 6.28 Cell division protein involved in ftsz ring TAGTGTAGAGCAGAAAACAAAA 5.88 185 IS , phage , Tn ; transposon-related functions IS , phage , Tn ; transposon-related functions Putative transport ; not classified Putative regulator ; not classified Putative regulator ; not classified yidE yidP 58 185 370 70 1.85 0.81 # yiePO 1.52 -- Orf ; unknown 2.51 1.15 # 2.17 1.87 -- 2.09 1.42 # 1.82 1.4 # 2.35 0.93 # 1.87 0.93 # yigC 1.89 -- Putative enzyme ; not classified Putative enzyme ; not classified Orf ; unknown Putative regulator ; not classified Putative transport ; not classified Orf ; unknown 1.03 # 1.38 # 2.1 -- 1.31 # 1.24 # yii yiiM yjaE yjeM yjeNO D 75 217 283 34 109 1.45 # 1.26 # 4.23 -- 1.38 # 1.35 # 1.08 # 1.22 # 0.97 # 1.6 -- 3.49 -- 1.44 # 1.29 # 2.03 # 1.58 -- 1.12 # 2.64 1.46 # 1.92 1.19 # 3.42 1.05 # 3.04 1.37 # 2.81 1.27 # 1.9 0.84 # 2.12 1.32 # 1.93 0.95 # 2.37 0.81 # 2.97 1.25 # 2.35 1.28 # 1.89 1.24 # 3.38 1.21 # 2.1 0.96 # 2.16 1.69 -- yjfG yjhBC Putative enzyme ; not classified Putative transport ; not classified Orf ; unknown Orf ; unknown Orf ; unknown Orf ; unknown Putative transport ; not classified Putative enzyme ; not classified Putative transport ; not classified Orf ; unknown Orf ; unknown Orf ; unknown Putative regulator ; not classified Putative transport ; not classified Membrane ; cell-division yjhQP yjiMLKJ yjjM yjjY ynfM yqeA yqgA yqgB yqhA ytfQ zipA 70 173 94 51 98 250 313 a Gene or genes of an operon that are activated by wild-type CRP in ROMA experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The rpoSencoded alternative sigma factor GS , CAMPICRP and , recently for S. fyphimurium , the gene product have been shown to be involved in starvation Or stationary phase-specific gene control ( Hengge-Aronis , 1993a ; Matin , 1991 ; Spector and Cubitt , 1992 ) ; gS and/or CAMP were indeedfound to be involved in the regulation of several csi genes , whereas re/A does not appear to playa major role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>Here we identified novel targets , Glk , YggF , PpsA , and Pck , indicating that CRP plays a major role in switch control between glycolysis and gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>One of the newly identi-ﬁed targets of CRP is the pck gene coding for phosphoenolpyruvate carboxykinase ( Pck ) , a key enzyme for switch control between glycolysis and gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Strain SBS688 ( pMGL25 ) was transformed with plasmid pBR322 ( as a negative control ) , pYZCRP encoding wild-type CRP , and pYZCRP carrying variant CRPpc ( T158A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>To ascertain the role of CRP in negative transcriptional-control of the speC promoter , the Tc resistance of isogenic strains , LS853 ( dcya ) and LS854 .1 ( dcya , dcrp ) carrying either plasmid pCOD or pTET was determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>To examine the contribution of CRP as well as known transcriptional regulators of the E. coli responses to acidic conditions , we assayed AzuC-SPA levels in wild-type , crp , rpoS , gadXW , gadE , and cadC cells exposed to pH 5.5 ( Fig. 5C ) FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>( C to E ) Plots of speciﬁc ﬂuorescence intensity ( SFI ) , expressed in arbitrary units ( AU ) , against time for M182 and M182 crp cells harboring pEUcspEgfp312 , the transcriptional fusion ( C ) , M182 crp/pEUcspEgfp312 cells harboring pDU9 ( control ) and pDCRP ( WT CRP ) multicopy plasmids ( D ) , and M182 , M182 crp , M182 crp/pDU9 , and M182 crp/pDCRP cells harboring pEUcspEgfp380 , the translational fusion ( E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Gene 31:165 -- 171 Colland F , Barth M , Hengge-Aronis R , Kolb A ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_30">
177 , 3166 -- [ 30 ] Colland , F. , Barth , M. , Hengge-Aronis , R. and Kolb , A. ( 2000 ) r factor selectivity of Escherichia coli RNA polymerase , role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15251200" target="_blank">15251200</a>]<br></label>
<input id="rp_similarity_30", type="checkbox">
<div class="RP_similar">
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. , and Kolb , A. ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymerase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. , and Kolb , A. ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymer-ase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_33">
Factor selectivity of Escherichia coli RNA polymerase : Role for CRP , IHF and lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></label>
<input id="rp_similarity_33", type="checkbox">
<div class="RP_similar">
<li>s factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>]<br></li>
<li>Sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621818" target="_blank">16621818</a>]<br></li>
<li>Sigma factor selectivity of Escherichia coli RNA polymerase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li></div>
<li>CRP Modulates fis Transcription by Alternate Formation of Activating and Repressing Nucleoprotein Complexes * Received for publication , January 23 , 2001 , and in revised form , February 22 , 2001 Published , JBC Papers in Press , February 23 , 2001 , DOI 10.1074 / jbc.M100632200 William Nasser ‡ , Robert Schneider ‡ § , Andrew Travers ¶ , and Georgi Muskhelishvili ‡ i From the ‡ Institut für Genetik und Mikrobiologie , Ludwig-Maximilians-Univesitaet , Maria-Ward-Strasse 1a , 80638 München , Germany and the ¶ Medical Research Council Laboratory of Molecular Biology , Hills Road , Cambridge CB2 2QH , United Kingdom The DNA architectural proteins FIS and CRP are global regulators of transcription in Escherichia coli involved in the adjustment of cellular-metabolism to varying growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>43 2 Biochemistry 2010 , 49 , 432 -- 442 DOI : 10.1021 / bi901583n An Unusual Pattern of CytR and CRP Binding Energetics at Escherichia coli cddP Suggests a Unique Blend of Class I and Class II Mediated Activation † Allison K. Holt and Donald F. Senear * Department of Molecular Biology and Biochemistry , University of California , Irvine , California 92697 Received September 9 , 2009 ; Revised Manuscript Received November 4 , 2009 ABSTRACT : Two transcription factors , CRP and CytR , mediate positive and negative control of nine cistrons involved in nucleoside-catabolism and recycling in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Our predictions show a large , 18-gene overlap with the CRP regulon and point to an even broader regulatory role for Spot42 in primary-metabolism involving the citrate cycle and acetyl-CoA -- dependent processes ( Table 2 , Tables S4 and S5 , and SI Appendix , Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>10 % of E. coli strains grown in rich media at low pH ( Applebaum et al. , 1975 ) where it plays a role in regulating intracellular pH. Previous studies from our laboratories have demonstrated that the E. coli speC gene , encoding biosynthetic ODC , is negatively regulated either directly or indirectly at the transcriptional level by CAMP and CRP ( Wright et al. , 1982 ; 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7828866" target="_blank">7828866</a>]<br></li>
<li>Ap , ampicillin ; Cm , chloramphenicol ; Ec , E. coli ; Km , kanamycin ; Kp , K. pneumoniae ; dnaKp , dnaK promoter ; lacp , lac promoter ; mCRP , mutated CRP-binding site ; Tc , tetracycline ; US , upstream sequences ; , resistance ; D , deletion ; : : , novel joint ; lacDX74 , complete deletion of the lac operon ; R glp * is mutation near the argH gene that allows growth of this strain on glycerol ( Roy et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_42">
Marschall C , Labrousse V , Kreimer M , Weichart D , Kolb A & Hengge-Aronis R ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigma s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></label>
<input id="rp_similarity_42", type="checkbox">
<div class="RP_similar">
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigma S and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sS and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731280" target="_blank">14731280</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigmaS and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and s requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and S requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Marschall , C. , Labrousse , V. , Kreimer , M. , Weichart , D. , Kolb , A. , and Hengge-Aronis , R. ( 1998 ) Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_49">
Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on sigma s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26020590" target="_blank">26020590</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24981318" target="_blank">24981318</a>]<br></label>
<input id="rp_similarity_49", type="checkbox">
<div class="RP_similar">
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on S and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621818" target="_blank">16621818</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and requires activation by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Molecular analysis of the regulation of csiD , a carbon starvation-inducible gene in Escherichia coli that is exclusively dependent on s and requires activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730174" target="_blank">12730174</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li></div>
<li>R. ( 1993 ) Complex transcriptional-control of the a ^ - dependent stationary phase-induced osmY -LCB- csi-S ) gene suggests novel roles for Lrp , cAMP-CRP and IHF in the stationary phase response of Escherichia coli J Bacterioi MS : 7910-7917 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997184" target="_blank">7997184</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_54">
Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible gene under the control of sS and the cAMP-CRP complex in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></label>
<input id="rp_similarity_54", type="checkbox">
<div class="RP_similar">
<li>Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible gene under the control of sigma S and the cAMP-CRP complex in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_56">
Marschall , C. , and Hengge-Aronis , R. ( 1995 ) Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible s - dependent gene under positive control S of cAMP-CRP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></label>
<input id="rp_similarity_56", type="checkbox">
<div class="RP_similar">
<li>Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible S-dependent gene under positive control of cAMP-CRP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible sS-dependent gene under positive control of cAMP-CRP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li></div>
<li>6 ( 1992 ) 3257 -- 3265 [ 31 ] Xu J. , Johnson R.C. , aldB , an RpoS-dependent gene in Escheri-chia coli encoding an aldehyde dehydrogenase that is repressed by FIS and activated by CRP , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_60">
Xu , J. and Johnson , R.C. ( 1995b ) aldB an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by FIS and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></label>
<input id="rp_similarity_60", type="checkbox">
<div class="RP_similar">
<li>aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Xu , J.M. , and Johnson , R.C. ( 1995 ) aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Xu , J.M. , and Johnson , R.C. ( 1995a ) aldB , an RpoSdependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li></div>
<li>aldB , an RpoS-depen-dent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_65">
Proteins CRP was puri cents ed from an E. coli strain carrying crp gene on a multicopy plasmid , pHA5 , by fast protein liquid chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760580" target="_blank">10760580</a>]<br></label>
<input id="rp_similarity_65", type="checkbox">
<div class="RP_similar">
<li>Proteins CRP was puri ® ed from an E.coli strain carrying the crp + gene on a multicopy plasmid , pHA5 , by fast protein liquid chromatography ( S.Ryu , Seoul National University , Korea ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>Natl. Aiba , H. : Autoregulation of the Escher & u coli crp gene : CRP is a transcriptional repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>References Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>( b ) In vivo effect of CRP on Pr promoter activity in isogenic E. coli AFMCRL ( CRP + , black bars ) and AFSBRL ( CRP '' , grey bars ) cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Key words : Catabolite repression - Genetics - Malate dehydrogenase - Molecular cloning - Sequence - CRP binding site - Escherichia coli Malate dehydrogenase ( EC 1.1.1.37 ) has been investigated in many organisms , as an important enzyme of the TCA-cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , CRP and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></li>
<li>Cameron , A.D. , and Redﬁeld , R.J. ( 2006 ) Non-canonical CRP sites control competence regulons in Escherichia coli and many other gamma-proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>After Genomic SELEX screening , some additional PTS-system genes for sugar-transport were found to belong the CRP regulon , including fruBKA for fructose [ 51,52 ] , manXYZ for mannose-transport [ 47 ] , ptsG for glucose-transport [ 47 ] , mtlAD for mannitol-transport [ 47 ] and ptsH encoding HPr component of PTS [ 47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>For example , genes encoding the glutamine metabolic enzymes ( Prusiner et al. , 1972 ) , adenylate cyclase ( Majerfeld et al. , 1981 ) , isocitrate dehydrogenase ( Calcott , 1982 ) rmB operon ( Glaser et al. , 1980 ) and spot 42 RNA production ( Sahagan and Dahlberg , 1979 ) are negatively controlled by CAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>Regulation of the gcv operon by GcvA and GcvR is part of a growing number of prokaryotic regulatory systems involving an activator -RCB- antiactivator complex , including TraR -RCB- TraM regulation of conjugal transfer of the Ti plasmid in Agrobacterium tumefaciens ( Hwang et al. , 1999 ) , NifA -RCB- NifL regulation of nitrogen ﬁxation in Klebsiella pneumoniae ( Narberhaus et al. , 1995 ) , ComK -RCB- MecA -RCB- ClpC regulation of competence in Ba-cillus subtilis ( Turgay et al. , 1998 ) and CRP -RCB- CytR regulation of operons encoding the nucleoside-transport and biosynthesis enzymes ( Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>These include cAMP-CRP ( 119 , 133 ) , temperature control ( 1 , 105 ) , heat shock proteins DnaK , DnaJ , and GrpE ( 116 ) , high concentrations of either inorganic salts , carbohydrates , or alcohols ( 74 , 115 ) , DNA supercoiling ( 11 , 63 , 74 , 115 ) , growth phase ( 96 ) , surface-liquid transition ( 33 ) , phosphatidylethanolamine and phosphatidylglycerol-synthesis ( 91 , 114 ) , and cell-cycle control ( 90 , 92 , 103 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>This promoter is controlled by a number of global regulators such as CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , heat shock proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) and H-NS ( Bertin et al. , 1994 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) , as well as by several environmental signals such as temperature control ( Qualding and Stocker , 1962 ; Adler and Templeton , 1967 ) , high concentrations of inorganic salts , carbohydrates or alcohols ( Li et al. , 1993 ; Shi et al. , 1993 ) , growth phase ( Nishimura and Hirota , 1989 ) , DNA supercoiling ( Li et al. , 1993 ; Shi et al. , 1993 ; Kutsukake , 1997 ; Das et al. , 1998 ) , phosphatidylethanolamine and phosphatidylglycerol-synthesis ( Shi et al. , 1993 ; Mizushima et al. , 1995 ) , cell-cycle control ( Morgan et al. , 1995 ; Prüß and Matsumura , 1996 ; Mizushima et al. , 1997 ) and quorum-sensing ( Sperandio et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>]<br></li>
<li>The synthesis and expression of these ﬂagella and motility genes are regulated by several environmental cues besides quorum-sensing , including temperature ( Adler and Templeton , 1967 ) , osmolarity ( Shin and Park , 1995 ) , cAMP-CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , cell-cycle control ( Nishimura and Hirota , 1989 ) , RcsCDB ( Francez-Charlot et al. , 2003 ) , H-NS ( Bertin et al. , 1994 ) and integration host factor ( IHF ) ( Yona-Nadler et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>INVOLVEMENT OF DIFFERENT REGULATORS IN THE ANAEROBIC EXPRESSION OF mdtEF FROM THAT OF AEROBIC CONDITION Previous studies focusing on the expression of gadE-mdtEF operon during acid resistance have revealed that several regulators , such as EvgA , YdeO , MnmE , GadE , GadX , CRP , and H-NS , are involved in the regulation of gadE-mdtEF operon ( Gong et al. , 2004 ; Ma et al. , 2004 ; Hirakawa et al. , 2006 ; Tramonti et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>2010 SGM Printed in Great Britain In concert with the complex mode of regulation for biofilm-formation , the expression of csgD has been proposed to be under the control of various stress-sensing regulatory proteins such as OmpR ( Romling et al. , 1998 ; Vidal et al. , 1998 ; Prigent-Combaret et al. , 1999 , 2000 , 2001 ; Gerstel & Romling , 2001 ; Gerstel et al. , 2003 ) , CpxR ( PrigentCombaret et al. , 2001 ; Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ; Robbe-Saule et al. , 2006 ) , CRP ( Zheng et al. , 2004 ) , H-NS ( Arnqvist et al. , 1994 ; Gerstel et al. , 2003 ; Olsen et al. , 1993 ) , IHF ( Gerstel et al. , 2003 ; Gerstel & Romling , 2001 ) , MlrA ( Brown et al. , 2001 ) and RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Furthermore , a number of transcription factors are involved in the regulation of the csgD promoter , including CpxR ( Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ) , H-NS ( Gerstel et al. , 2003 ) , IHF ( Gerstel et al. , 2003 , 2006 ) , OmpR ( Vidal et al. , 1998 ; Gerstel et al. , 2003 , 2006 ) , RstA ( Ogasawara et al. , 2007a ) , MlrA ( Brown et al. , 2001 ) , RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) and CRP ( Zheng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Similarly , CRP is a global transcription factor regulating transcription at over 200 promoters ( Grainger et al. 2005 ) including those for bhsA , galS , ychH and yhnC ( Brown and Callan 2004 ) identiﬁed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>The latter circuit is also the most complex because apart from GadX and GadW , it also involves the global regulators H-NS and CRP , which act as transcriptional repressors , and RpoS , which positively affects gadA and gadBC transcription ( De Biase et al. , 1999 ; Castanie-Cornet and Foster , 2001 ; Hommais et al. , 2001 ; Tramonti et al. , 2002 ; Giangrossi et al. , 2005 ; Weber et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The stationary phase global regulator RpoS ( S ) ( Olsén et al. 1993 ; Römling et al. 1998 ) and other global regulators including H-NS ( Arnqvist et al. 1994 ; Olsén et al. 1993 ) , Crl ( Arnqvist et al. 1992 ) , OmpR ( Römling et al. 1998 ; Vidal et al. 1998 ) , MlrA ( Brown et al. 2001 ) and CRP ( Zheng et al. 2004 ) can aVect curli synthesis by promoting or suppressing expression of the curli-encoding genes by regulating the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Fis can also affect DNA supercoiling indirectly , because of its roles in repressing transcription of gyrA and gyrB to reduce the DNA gyrase activity ( Schneider et al. , 1999 ) , increasing topA transcription to enhance topoisomerase I activity ( WeinsteinFischer et al. , 2000 ) , and regulating the expression of other DNA-binding proteins such as H-NS , HU and CRP ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) that also affect DNA supercoiling directly or indirectly ( Broyles & Pettijohn , 1986 ; Gomez-Gomez et al. , 1996 ; Tupper et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Accessory components are acetyl phosphate , which contributes to RssB and perhaps also to ArcA phosphorylation ( Bouché et al. 1998 ; Pena-Sandoval et al. 2005 ) , as well as cAMP-CRP , which in log-phase cells has an indirect inhibitory effect on rpoS transcription ( Lange and Hengge-Aronis 1994 ; F. Mika and R. Hengge , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16291649" target="_blank">16291649</a>]<br></li>
<li>For example , cAMP-CRP , GlcIIa , BarA , polyphosphate , homoserine lactone , acetate , and the NADH/NAD ratio have been shown to affect rpoS transcription , whereas DnaK , DksA , Hfq , HU , StpA , LeuO , and several small regulatory RNAs control rpoS translation ( Hengge-Aro-nis 2002 ; Repoila et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17403784" target="_blank">17403784</a>]<br></li>
<li>Sialic acid transport and catabolism are cooperatively regulated by SiaR and CRP in nontypeable Haemophilus inﬂuenzae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>CspD is reported to be regulated by multiple factors ( MqsR / MqsA toxin-antitoxin pair [ 4 , 11 ] , ppGpp [ 3 ] , CRP [ present study ] , etc. ) as a function of cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In order to examine whether CRP is involved in this glucose-repression , we constructed crp and cya mutants ( where cya adenylate cyclase gene ) , and we measured their PGA activities during growth in M9 minimal medium containing glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>However , whereas FIS is primarily involved in the reorganisation of cellular-metabolism on transition to exponential growth in rich medium , CRP is most active under the conditions of exhaustion of glucose and its analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated Glycolysis , citric-acid-cycle , and respiratory chain genes times following the introduction of air 5 min 10 min 15 min 60 min 6.0 6.7 6.9 4.7 3.6 5.7 6.1 3.2 Relevant regulatory proteins aceE pyruvate dehydrogenase ( decarboxylase component ) aceF pyruvate dehydrogenase ( dihydrolipoyltransacetylase component ) acnA aconitase A acnB aconitase B cyoA cytochrome bo terminal oxidase subunit II cyoB cytochrome bo terminal oxidase subunit I cyoC cytochrome bo terminal oxidase subunit III cyoD cytochrome bo terminal oxidase subunit IV cyoE heme o synthase gltA citrate synthase gpmA phosphoglycerate mutase icdA isocitrate dehydrogenase ( NADP ) lpdA lipoamide dehydrogenase ( E3 monomer ) mdh malate dehydrogenase mqo malate dehydrogenase ( quinone ) ndh NADH dehydrogenase II pdhR pyruvate-responsive transcription regulator poxB pyruvate oxidase sdhA succinate dehydrogenase flavoprotein sdhB succinate dehydrogenase iron-sulfur protein PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) ArcA ( ) FNR ( ) Cra ( ) CRP ( ) SoxRS ( ) ArcA ( ) Cra ( ) CRP ( ) Fis ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA * ( ) Fur ( ) P1 , ArcA * ( ) P2 , Cra * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) Plpd-ArcA * ( ) FNR ( ) CRP ( ) PdhR ( ) Fis ( ) ArcA ( ) CRP ( ) FlhD ( ) ArcA ( ) FNR * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) SoxRS ( ) MarR ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2-CRP ( ) P1 - ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) 2.0 3.1 2.6 2.9 6.8 8.6 9.9 7.3 5.2 2.7 1.9 5.0 3.3 3.2 7.4 3.2 5.8 7.3 9.1 11.7 7.8 6.5 4.7 2.1 6.1 4.9 10.4 12.9 9.2 5.7 4.4 1.5 4.8 8.6 10.0 7.9 6.3 4.0 2.1 5.3 3.0 2.4 4.8 2.9 3.5 3.9 3.4 6.9 4.2 4.6 3.5 8.6 5.0 5.0 2.2 3.5 2.4 6.6 3.1 3.0 4.7 2.1 3.7 6.8 9.9 5.5 9.1 5.8 8.7 4.3 2.9 5.8 2.8 sdhC succinate dehydrogenase membrane protein sdhD succinate dehydrogenase membrane protein sucA 2-oxoglutarate decarboxylase 16.6 4.6 3.2 3.6 7.8 1.9 2.0 2.1 2.1 3.2 3.6 3.1 sucB dihydrolipoamide succinate transferase sucC succinyl-CoA synthetase , subunit sucD succinyl-CoA synthetase , subunit 3.5 4.7 2.7 TABLE 2 Transcripts encoding proteins involved in the oxidative stress response that are present in increased abundance after the introduction of air to anaerobic cultures of E. coli MG1655 Fold regulation ( at least 2-fold at one or more time point ) is the ratio of transcript levels at the indicated time after the introduction of air to the transcript levels at t 0 ( the initial anaerobic state ; p 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 4.8 5.2 4.6 3.1 3.1 3.5 2.1 2.0 4.2 4.2 5.1 4.6 Oxidative stress and iron-sulfur-cluster-assembly genes Relevant regulatory proteins ahpC alkyl hydroperoxide reductase small subunit ahpF alkyl hydroperoxide reductase large subunit dadA D-amino acid dehydrogenase dadX alanine racemase OxyR * ( ) MetJ ( ) OxyR * ( ) MetJ ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) IscR * ( ) SoxRS * ( ) OxyR * ( ) OxyR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) OxyR * ( ) P1 , H-NS ( ) Lrp * ( ) NhaR ( ) RcsB ( P2 , H-NS ( ) Lrp * ( ) IscR * ( ) OxyR ( ) IHF ( ) Fur SoxR ( ) SoxS ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) 3.3 2.1 3.4 1.9 3.2 1.3 7.2 1.6 2.2 2.8 6.3 6.1 3.4 3.0 2.1 5.3 2.1 2.8 3.0 9.1 5.8 4.1 fdx ferredoxin fur ferric uptake regulator 2.1 3.4 2.4 2.8 3.3 10.1 7.0 5.1 2.0 1.9 1.8 2.7 1.8 1.8 2.8 2.5 11.2 5.7 4.4 1.8 1.1 3.6 grxA glutaredoxin 1 hscA co-chaperone involved in IscU iron-sulfur-cluster-assembly hscB co-chaperone involved in IscU iron-sulfur-cluster-assembly iscA iron-sulfur-cluster-assembly protein iscR transcriptional regulator iscS cysteine desulfurase iscU scaffold protein involved in iron-sulfur-cluster-assembly iscX possible role in iron-sulfur-cluster-biosynthesis katG bifunctional hydroperoxidase I osmC organic hydroperoxidase Within 5 min of air exposure the abundance of the soxS transcript is increased 8-fold ( Table 2 ) , indicating activation of the sensor SoxR , which enhances soxS expression , which in turn controls the SoxRS regulon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Identiﬁcation of the effectors involved in GadX activation and further investigation of the interactions between regulatory components of the gad system ( H-NS , RpoS , CRP , and GadX ) may provide further insight into the molecular circuitry underlying the glutamic acid-based acid stress response of E. coli A. Tramonti and P. Visca contributed equally to this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Examination of multiple alignment of the CRP/FNR-like proteins revealed one speciﬁc amino acid ( R180 ) in the HTH motif involved in DNA recognition , which has changed from arginine ( like in E. coli CRP and Desulfovibrio HcpR ) to Ser or Pro in other dproteobacteria ( see the HcpR1 branch of the phylogenetic tree in Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Genes involved in denitriﬁcation , such as nir , nor , and nos , are mainly regulated by two NO-responsive transcriptional activators from the FNR/CRP family , NnrR in a-proteobacteria and DNR in b-proteobac-teria , and Pseudomonas spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>On this view FIS , and possibly also CRP , act to ﬁne tune a fundamental mode of transcriptional regulation , not only by their direct effects on DNA topology but also from the indirect effects of their transcriptional-control of other nucleoid-associated proteins , such as HU and H-NS [ 3 , 4 ] , which also act as determinants of chromosomal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>Regulation of the fatty acid pathways is complex and involves transcriptional regulators ( including FadR , FarR , IclR , and CRP ) and post transcriptional mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Whereas rs , Lrp , CRP , and IHF are clearly involved in stationary-phase induction , none of these regulators is essential for osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Thus , BglJ -- RcsB is a pleio-tropic transcriptional activator that coordinates regulation with global regulators including CRP , LeuO and HNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Role of RpoS and Crl in bgl operon regulation The bgl operon is regulated by various pleiotropic regulators including H-NS and CRP , as essential factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
<li>Negative autoreguiation seems to play a role in preventing unnecessary production of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The proximal CRP target site plays an important role in CRP-mediated regulation of PcspD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Since CRP site-II seems to play a minor role in controlling CRP-mediated cspD regulation , PcspD with CRP site-I target site alone ( CRP site-II deletion or mutation ) was analyzed for AR domain preferences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>This suggests the occurrence either of processing of CRP after translation or of proteolytic degradation of CRP during purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>w Location of DNA site for CRP relative to translation start as shown in ColiBASE ( http://colibase.bham.ac.uk/ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>The observations reported here have many notable features : in the deﬁned MOPS minimum medium with different carbon sources , CRP is activated in poor quality ( deﬁned by growth rate ) carbon sources relative to rich quality ones ; this activated CRP associates with the ﬂhDC operon encoding the master regulator with roles in ﬂagellar regulation and development ; the activation of CRP is correlated with the increase of expression of sF as well as the increase of cell-motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>While it is known that carbon-catabolite-repression affects the ﬂagellar synthesis and the CRP activation might be involved in alleviating this repression ( 41,42,44,45 ) , much less is known regarding the role of CRP in motility regulation under a range of carbon source conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>Carbon source effects on cell-motility through activation of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>In addition , the synchronized pattern of increasing CRP activity causing increasing FlhDC expression with decreasing carbon source quality , together with the apparent coupling of motility activity with the activation of motility and chemotaxis genes in poor quality carbon sources , highlights the importance of CRP activation in allowing E. coli to devote progressively more of its limited reserves to search out better conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The correlation between the activity of CRP and the motility of cells indicates that activation of CRP plays an important role in triggering E. coli ` foraging ' - like behavior ( 20 -- 22 ) to actively search for better conditions as the quality of the available substrate ( s ) decreases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The upregulation of motility genes correlated with increasing motility of cells in poor quality carbon sources , together with the synchronized pattern of increasing CRP activity causing increasing FlhDC transcription , further suggests CRP is a factor ( or at least one of multiple factors ) to play a positive role in triggering E. coli ` foraging ' - like behavior to actively search for better conditions as the quality of the available substrate ( s ) decreases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>0 5-minute induction of σF 5 A 0 2 = σF β ' Motility control by carbon source : CRP binds to and scalably activates FlhDC operon in alternative carbon sources Recently , Liu and coworkers ( 20 ) proposed a new model for carbon source foraging strategy by E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>Laboratoire de Microbiologie et Genetique Mokculaire , Centre de Gknetique Moleculaire et Cellulaire , UMR CNRS 5534 , Universite Claude Bernard , 69622ViIleurbanne Cedex , France Keywords : Escherichia coli , transcription , CAMP-CRP , maltose regulon , pH The maltose regulon of E. coli consists of numerous genes whose products are involved in the transport and catabolism of maltose and maltodextrins ( Schwartz , 1987 ; Boos et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>CRP activates transcription from over 150 different promoters in E. coli ( Kolb et al. , 1993 ; Zheng et al. , 2004 ) and this CRP regulon includes several operons mainly involved in catabolism of carbon sources other than glucose ( Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Results Two global regulators , CRP and Cra , of E. coli recognize and regulate as many as 378 and 164 promoters , respectively , of the genes for carbon source utilization ( Shimada et al. , 2005 , 2011a , 2011b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Expression of the E. coli K-12 melAB genes , which encode products essential for melibiose-metabolism , depends on transcription activation by MelR and CRP , and is triggered by melibiose ( 7,9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>CRP was the first purified [ 5 ] , the first crystallized [ 6 ] and the best studied transcription activator from Escherichia coli [ 7 -- 13 ] , and plays a key role in activation of a number of the genes for utilization of carbon sources other than glucose ( for reviews see 7 , [ 14 -- 16 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>CRP activates transcription of about 200 genes in E. coli ( 9 ) in response to the carbon source available .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CRP controls transcription of a large number of Escher-ichia coli genes in response to carbon source limitation ( for a review , see Kolb et al. , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The CRP regulon deﬁned in this study includes several operons that are involved in carbohydrate-transport and metabolism , such as fucPI for fucose , kdgT for gluconate and fucAO for fuculose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Transcription depends upon CRP , the anaerobically triggered transcription activator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>This analysis demonstrates that the majority of genes for the transport and utilization of sugar acids are under the regulation of CRP ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Results Search for the regulation targets of CRP by using Genomic SELEX screening : SELEX-clos analysis In order to get insight into the genome regulation by CRP for control of carbon source utilization in E. coli , we first tried to identify the whole set of target promoters , genes and operons under the control of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In reﬂecting the physiological role of GapA as a real GAPDH , the promoter organization is very unique , being controlled by two global regulators , CRP and Cra , for a large set of genes for carbohydrate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Besides CRP , another transcription factor , Cra ( catabolite repressor activator ) , has been indicated to play a role in the global regulation of the genes for carbon metabolism ( 16 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Moreover , the differences in the regulation targets and regulatory roles of two global regulators , CRP and Cra , in the control of a number of genes for carbon source utilization are totally left undetermined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>This finding raises the need for reevaluation of the differential roles of two global regulators , CRP and Cra , in the overall regulation network of carbon source metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Cra is involved in regulation of the whole set of enzymes in central carbon metabolism ( 43 ) , while CRP regulates the genes for carbon source utilization outside the central metab-olism , including transport of various carbon sources , glycolysisgluconeogenesis switching , the tricarboxylic acid ( TCA ) cycle , the pyruvate-dehydrogenase-pathway , and aerobic-respiration ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Based on the functions of novel target genes , we conclude that CRP plays a key regulatory role in the whole processes from the selective transport of carbon sources , the glycolysis-gluconeogenesis switching to the metabolisms downstream of glycolysis , including tricarboxylic acid ( TCA ) cycle , pyruvate dehydrogenase ( PDH ) pathway and aerobic-respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>More recent genomic studies have shown that CRP also activates transcription of genes that encode enzymes involved in central carbon metabolism and transporters of various alternative carbon sources ( 26 , 29 , 85 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Regulatory roles of CRP : Regulation of the genes for transport of carbon sources Figure 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Regulatory roles of CRP : Regulation of the genes for carbon metabolism Regulatory roles of CRP : Switching control between glycolysis and gluconeogenesis Several step reactions of the central carbon metabolism are apparently irreversible , different enzymes being involved in catalysis of forward and backward reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Among the enzymes involved in these four steps reactions for entry to and exist from the central glycolysis pathway , only one enzyme , GlpX , had been identified as a regulation target of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Interestingly , a large number of genes involved in the nutrient transport and metabolic pathways are regulated by CRP ( 18 ) and also have been shown to be differentially regulated during a downshift in temperature ( 16 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Together with CRP , it is involved in the ( generally ) positive regulation of a number of catabolic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Lower panel : Most of the genes for the enzymes involved in glycolysis are controlled by Cra [ 30 ] while only three genes , fbaA , gapA and pgk , had been identified as the regulation targets of CRP ( shown in while under blue background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>A limited number of the genes encoding the enzymes involved in glycolysis had been found to be regulated by CRP , including fabA ( fructose-1 ,6 - bisphosphate aldolase ) , gapA ( glyceraldehyde-3-phosphate dehydrogenase ) and pgk ( phosphoglycerate kinase ) ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Furthermore a number of the genes involved in the metabolism downstream of glycolysis including PDH pathway and aerobic-respiration were found to be the targets of CRP regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Regulatory roles of CRP : Regulation of the genes for aerobic respiratory CRP was found to regulate a number of the genes coding for enzymes involved in the metabolism downstream of glycolysis , including pyruvate dehydrogenase ( PDH ) complex , and most of the TCA-cycle enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>While the cyclic AMP-CRP complex primarily controls the initiation of carbon source utilization , Cra primarily influences the direction of carbon metabolic flux within the central pathways of carbon metabolism by regulating the levels of key enzymes in these pathways [ 26 , 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>g002 Location of the CRP-binding sites CRP has been identified to regulate some of the enzymes for central carbon metabolism , including fbaA , gapA , and pgk .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>A total of 28 CRP targets including CRP itself are the specific regulators of carbon metabolism ( shown under green background in Fig. 6 ) , while 13 are the regulators of nitrogen metabolism ( orange background in Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank Barry Wanner for providing plasmid pKD46 and E. coli strains BW25113 and BW25993 , Richard Gourse for puriﬁed CRP protein and strain DM0068 , and Marcin Filutowicz for puriﬁed IHF and strain DBP102 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Acknowledgements We thank Dr Hiroji Aiba for providing pHA7 and anti-CRP antibody , Dr Hideji Yoshida for providing anti-L2 antibody , and NBRP : E.coli ( NIG , Japan ) for providing JW3253 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21062370" target="_blank">21062370</a>]<br></li>
<li>We also thank H. Aiba ( Nagoya University ) for anti-CRP antiserum , strains , and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>acs transcription requires CRP I , while optimal transcription also involves CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>We also show that CRP directly activates transcription from a second s28-dependent promoter that has a similar organization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>As observed for the MhpR regulator , CRP also regulates expression of other transcription factors in E. coli such as MelR ( Webster et al. , 1988 ) , MalT ( Chapon & Kolb , 1983 ) , RhaS ( Wickstrum et al. , 2005 ) , MaoB ( Yamashita et al. , 1996 ) , HcaR ( Turlin et al. , 2001 ) , GalS ( Weickert & Adhya , 1993 ) , RpoH ( Kallipolitis & Valentin-Hansen , 1998 ) , BlgG ( Gulati & Mahadevan , 2000 ) , PrpR ( Lee et al. , 2005 ) and Fis ( Nasser et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>This simultaneous CRP control is not unique to the mhp pathway , since it has been also described for the genes for propionate-catabolism in E. coli and Salmonella enterica ( Lee et al. , 2005 ) and for some regulons from E. coli such as those for L-rhamnose ( Holcroft & Egan , 2000 ; Wickstrum et al. , 2005 ) , melibiose ( Webster et al. , 1988 ; Belyaeva et al. , 2000 ) and maltose ( Chapon & Kolb , 1983 ; Richet & Sogaard-Andersen , 1994 ; Richet , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Expression of FlhDC is tightly regulated by multiple transcription factors , including CRP , H-NS and OmpR , and hence the genes of the ﬂagellar cascade are regulated in response to many different environmental inputs ( Shin and Park , 1995 ; Sout-ourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>FruR ( Cra ) is a transcriptional regulator which , together with CRP , controls the expression of genes involved in C metabolism , particularly with respect to the direction of metabolic flow , glycolysis being repressed whereas gluconeogenesis is activated ( Ramseier et al. , 1993 ; Saier & Ramseier , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>For example , the oxidative acid resistance system is regulated by rs and CRP ( Foster 2004 ) , and rs regulates both gene-expression for osmotic stress ( Hengge-Aronis 1996 ) and regulates many acid resistance genes including the uncharacterized ygiW and yodD genes ( Weber et al. 2005 ) identiﬁed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>CRP is known to be essential for the expression of many genes involved in the transport and catabolism of different substrates ( Kolb et al. , 1993 ) , and it is unsurprising to ﬁnd that the expression of the malXY operon is regulated by a completely CRP-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>With increasing acetate-formation , expression of many genes known to be differentially expressed in glucose-grown and acetate-grown cells ( Oh et al. 2002 ) or in cells lacking CRP ( Gosset et al. 2004 ) decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Transcription factor networks under the control of CRP ( TIF ) Promoters for the genes encoding the key metabolic enzymes and the essential cell architectures are under the control of multiple regulators , each monitoring different environmental conditions or metabolic states [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Given the lack of probes for these genes on most microarrays , the genes also were not assayed in whole-genome expression surveys to characterize the regulons of well-studied global transcriptional regulators such as CRP ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>This raises the question of why CRP should directly regulate transcription of aer and trg , particularly when CRP indirectly regulates transcription of all genes in the ﬂagellar cascade by controlling expression of the master regulator , FlhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Thus , we propose a complex regulatory cascade in which global regulators ( RpoS , CRP , HN-S , EvgA , etc. ) inﬂuence the expression levels and/or activities of GadX and GadW , which in turn activate the expression and/or activities of transcription activators ( e.g. , YdeO and YhiE ) that directly activate subsets of target genes involved in AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>It has long been established that in the absence of exogenous glucose , CRP activates the expression of a large number of genes required for catabolism of alternative carbon sources by E. coli ( for a review , see reference 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP modulates positively or negatively the expression of a large number of genes and operons in response to glucose depletion ( 9 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Analysis of these operons supports the hypothesis that CRP is not only a regulator of genes required for catabolism of sugars other than glucose , but also regulates the expression of a large number of other genes in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP has been identiﬁed as a regulator of genes required for catabolism of sugars other than glucose and a large number of other genes [ reviewed in ( 1 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Figure 1 dernonstrates the effects of null mutations in the genes encoding os , CRP and adenylate cyclase ( rpoS359 : : Tn10 , Acrp96 and Acya851 , respectively ) on the expression of the chrornosornal csiE : : lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>As a consequence of this interaction , CRP recruit A ) Transposon mutagenesis ( Tn917-1acZ-fusion ) : ~ a-dependent expression of lacZ : 8 genes ro ) P B teomics : rsbX = etpt5 ~ 4 IF ' , o ° ' - ~ ~ , RsbW 26 kDa ~ ' ~ , = ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>We presume that in crp cells the expression of numerous genes that are under the control of both FIS and CRP will be perturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>CRP also represses gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>As a control we also assayed fadH expression and found that expression decreased in the absence of CRP ( Fig. 6E and F ) ( Fig. 7D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Since CRP plays a major role in the glucose-repression of many genes , we also tested whether CRP was required for the expression of the P3P2 -- lacZ fusion especially in an mnmE mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Evidence is presented that CRP is directly involved in the expression of the cai and ﬁx genes and plays a crucial role in the activation of carnitine-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>These mutants identified the genes for catabolite activator protein ( CAP , also known as CRP ) and adenylate cyclase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929525" target="_blank">11929525</a>]<br></li>
<li>Likewise , CRP may serve as a repressor in genes for adenylate cyclase ( 10,11 ) , glutamine-metabolism enzymes ( 12 ) and the outer membrane protein III ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>Furthermore , cAMP -- CRP is reported to be a repressor of its own transcription ( Aiba , 1983 ; Cossart and Gicquel-Sanzey , 1985 ; Okamoto and Freundlich , 1986 ; Hanamura and Aiba , 1991 ) , of genes for adenylate cyclase ( Majerfeld et al. , 1981 ; Mori and Aiba , 1985 ) , of genes encoding enzymes participating in glutamine-metabolism ( Prusiner et al. , 1972 ) and of genes for the outer membrane protein III ( Mallick and Herrlich , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>The CytR anti-activator and CRP act in concert to repress transcription from genes encoding transport proteins and enzymes necessary for uptake and catabolism of ribonucleosides and deoxyribonucleosides ( reviewed by Hammer-Jespersen , 1983 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The CRP regulon includes the genes encoding the transporters and the catabolic enzymes of non-glucose sugars [ 9,11 -- 13,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Upper panel : The genes under the control of CRP are indicated under either purple background ( the genes for sugar-transport systems ) or blue background ( the genes for metabolic enzymes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In any case , CRP was found to regulate not only the genes for carbon-utilization but also some ribosome-related genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In this case , CRP seems to act as a repressor of the mdtEF genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>CRP also regulates transcription of genes required for energy production , amino-acid-metabolism , nucleotide-metabolism and ion-transport systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>FNR is a CRP homologue and regulates genes during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP regulates the majority of genes for three pathways of sugar-transport ( MFS , PTS and ABC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>CRP has also been shown to repress transcription of some genes , including the crp gene itself and cyaA , encoding adenylate cyclase ( 2 , 25 , 26 , 52 , 85 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Also , evidence has accumulated to indicate that CAMP-CRP complex represses the transcription of as many genes as it activates ( Mallick and Herrlich , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>In addition , CRP can regulate transcription of other transcription factors , such as MelR , RpoH , BlgG , Fis and PdhR , which could further regulate transcription of genes involved in the above processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>A number of genes for uncharacterized transcription factors are under the control of CRP ( black background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>After Genomic SELEX screening , we discovered that a number of genes encoding transcription factors are under the control of CRP ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>CRP complex acts as a positive control element for P1 and as a negative factor for P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>Regulation of this operon is complex , involving negative control by attenuation ( 2,3 ) and positive control by a number of factors ( 4,5 ) including cAMP-CRP ( 6,2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Well studied regulatory cases include OxyR regulation of OxyS RNA ( oxidative stress ) ( 4 ) , Fur regulation of RyhB RNA ( iron limitation ) ( 5 ) , cAMP-CRP regulation of CyaR and Spot 42 RNA ( glucose limitation ) ( 6 -- 9 ) , OmpR regulation of MicF , OmrA , and OmrB RNAs ( osmotic shock ) ( 10 , 11 ) , E-mediated transcription of MicA and RybB RNAs ( envelope stress ) ( 2 , 12 ) , PhoPQ regulation of MgrR ( Mg2 / Ca2 transport and virulence ) ( 13 ) , and LuxO control of Qrr1 -- 4 ( quorum-sensing in Vibrio ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>A number of small mol-ecules , RNAs , and proteins belong to the complex regulatory network that controls S expression ; ppGpp ( the stringent control signal ) ( 11 ) , homoserine lactone ( 12 ) , inorganic phosphate ( 13 ) , UDP-glucose ( 14 ) , and cAMP-CRP ( 8 ) have been reported to influence the transcription of rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>The CRP regulators represent a fairly homogeneous group of proteins that control the response to glucose starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>This hypotShesis however remains to bv tested A number of regulatory proteins that control transcription in E. coli and its phages ( c.g , TrpR ' , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447784" target="_blank">1447784</a>]<br></li>
<li>( iii ) Most of the sugar-transport operons , whose transcription is activated by CRP , are also under the control of repression by other transcriptional regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>Thus , CRP enhances hlyE expression in response to glucose starvation ( 36 ) and FNR enhances hlyE expression in response to oxygen starvation ( 10 , 11 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>RESULTS Effects of the global transcription factors FNR , CRP , and H-NS on hlyE expression in liquid culture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , CRP and FNR positively control cai expression and H-NS exerts a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Expression of the caiF gene is dependent on the same general regulatory proteins , H-NS , CRP , FNR , and transcription factor RpoN ( 11 ) , which control cai and ﬁx expression ( Table 2 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>For example , induction of the CFA/I operon by temperature requires the positive regulator CfaD [ 70 ] expression of the pap operon requires CRP and PapB [ 71 ] , the virB gene of Shi-gella is activated by VirF [ 49 ] and Fis plays a role in activation of the rrnB gene [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>In ( A ) , CRP , and mutant derivatives , were introduced on pDCRP-derived ptasmids and expression determined in Acrp host cells ( M182Acrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>These findings suggest that CRP functions in certain contexts to regulate gene-expression in response to osmotic changes , in addition to its role in catabolite control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>In addition , GcvA is known to require the gcvR gene product for its role as a repressor ( 12 ) , raising the possibilities that CRP also regulates expression of gcvR and indirectly alters gcvT-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In addition to this role , CRP also regulates gene-expression in response to osmotic changes [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li>
<li>CRP regulates gene-expression from a large set of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A possible explanation for CRP 's dependence on GcvA is that CRP regulates expression of the gcvA gene and indirectly affects expression of gcvT-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>While CRP activates the expression of both genes , the FIS protein positively regulates the hupA gene but negatively regulates the tandem P2-P3 promoters of hupB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>When CRP-mediated regulation is dependent on a second protein , CRP often regulates expression of the gene encoding the second regulatory protein ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A CRP binding-site mutation or the absence of a functional crp gene leads to high constitutive expression of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>However , CRP was shown to appear to act as a repressor [ 21 ] in the expression of the adenylate cyclase gene , cya .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>Indicated also are the UP element and discriminator sequence ( involved in growth-rate-dependent regulation and stringent control ) , sites identi cents ed for the binding of PurR and DnaA and the putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>These sites are quite distant from the proposed transcriptional initiation site , although similar separations have been observed for the deo-2 and araBAD CRP binding sites ( 30,31 ) , where additional regulatory proteins are involved in the control of A potential promoter for the unidentified gene , g48 , is indicated in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6328431" target="_blank">6328431</a>]<br></li>
<li>DISCUSSION In the past few years , genetic and biochemical experiments have identified a surface-exposed loop of CRP ( ARI ) that is involved in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>CRP is also involved in activation with regulon-speciﬁc proteins at several pairs of divergent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Global regulatory proteins CRP , FNR , and H-NS are involved in the synthesis of carnitine dehydratase activity ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>location of bases involved in discrimination between FNR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>In the promoter region , there are several sequences of hyphenated dyad symmetries , two of which overlap the potential CRP binding site and may thus be involved in controlling mdh gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>Based on the expression analysis presented here a number of proteins could be involved in the transcriptional regulation of the astCADBE operon : the alternative sigma factors 54 and S , the 54-dependent activator NRI , and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>Our data suggest that additional regulatory proteins , i.e. cAMP-CRP and probably Lrp and H-NS , are crucially involved in generating this speci ® c pattern of csiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>There are at least two regulatory systems involved : rpoS , which controls gIgS ; and cAMP/CRP , which controls the expression of gIgCAP ( Romeo and Preiss .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1324388" target="_blank">1324388</a>]<br></li>
<li>These ﬁndings support a model where CRP is involved in expression of ClyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>To explore the origin of the indirect effect , we tested whether CRP was involved in regulation of rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Titration of the effects of cAMP , CRP , and ppGpp on the synthesis of glycogen-biosynthesis enzymes in S-30 extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Effects of cAMP , CRP , and ppGpp on the synthesis of glycogen-biosynthesis enzymes in S-30 extracts Presence ( + ) or absence ( - ) of : Expression '' of : cAMP CRP ppGpp IA gg gg ( 100 , uM ) ( 28.6 , ug/ml ) ( 250 FM ) lC ggA gIgB - - - 1.0 1.0 1.0 + - - 2.5 1.3 1.0 + + - 6.3 2.2 1.0 - - + 1.6 1.1 1.1 + - + 6.0 2.4 1.0 + + + 18.8 4.8 1.3 `` Results are expressed as the relative incorporation ( in counts per minute ) of [ 35Slmethionine for each gene product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>In contrast , abolition of CRP production or of cAMP-biosynthesis ( crp or cya mutant strains ) strongly reduced promoter gapB P0 activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>Mutants deficient in adenyl cyclase ( cya - ) , the enzyme which synthesizes cAMP , or CRP ( crp - ) are unable to utilize a variety of carbohydrates , including L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Genetic studies have identi ® ed cAMP-CRP , Lrp , integration host factor ( IHF ) and H-NS as such modulators of sS-dependent genes ( see Hengge-Aronis ( 1996b ) for review , and references cited therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In contrast to the speC gene , the nupG gene encoding a nucleoside-transport system is subject to positive regulation by the CAMP-CRP complex ( Mygind et al. , 1972 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7828866" target="_blank">7828866</a>]<br></li>
<li>, 1990 ) and the regulation of lacZYA operon ( encoding a lactose-utilizingsystem ) by Lac1 ( a specific repressor ) and CAMP-CRP complex ( a global modulator ) in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>We emphasize , however , that the regulation of ® s expression is a complex phenomenon subject to growth phase-dependent and stringent control , and involving in addition to IHF the effects of other factors such as FIS itself , CRP , the superhelical density of DNA and , perhaps , NTP pools ( this study ; Ball et al. , 1992 ; Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>The dual regulation by GntR and GntH in addition to that by cAMP-CRP is proposed to be important for expressional control of the GntII genes and also for switching of the GntI to GntII systems along with cell-growth as demonstrated [ Tsunedomi et al. , 2003 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>It has been shown that the regulation of the ¯ hDC operon of E. coli and S. typhimurium is affected by increasing concentrations of glucose ( via cAMP-CRP ) , NaCl , and ethanol , and also by temperature increase ( Macnab , 1996 ; Kutsukake , 1997 ; Shi et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>This ﬁnding reveals that , ﬁrst , the predicted cAMP-CRP binding region of the P3hfq promoter is essential for glucose response , and second , the underlying mechanisms involved in the negative regulation by glucose of ompA mRNA stability is associated with the activatio of the hfq gene caused by the loss of the repressive effects of the decreased cAMP level ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>-80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC DISCUSSION osmY ( csi-5 ) is one of many stationary-phase-induced and osmotically regulated genes in E. coli that require us , the gene product of rpoS , for expression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>BELL , A. I. , K. L. GASTON , J. A. COLE and S. J. W. BUSBY , 1989Cloning of binding sequences forthe Escherichia coli transcription activators , FNR and CRP : location of bases involved in discrimination between FNR and CRPN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Thus it would appear that several genes involved in nucleoside-metabolism are located in the nfo -- fruA region of the E. coli chromosome together with a novel paralogue of CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>CRP activated expression of sotA ( 4 ) , while KdgR , which represses transcription of many genes involved in transport and catabolism of 2-keto-3-deoxy-gluconate in E. coli ( 22 , 27 ) , was shown to weakly repress the expression of sotA in D. dadantii ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Our working model for AI-2 synthesis and transport in E. coli ( Fig. 8 ) is that cAMP-CRP acts as a global controller , while LsrR functions as a speciﬁc controller .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>AI-2 AHL Glucose AI-2 LsrK AHL LsrR sdiA Phospho-AI-2 lsrK lsrR lsrACDBFG AHL SdiA X cAMP CRP ydiV YdiV Figure 5 Conceptual model of signaling-pathway interactions in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>Microbiology ( 2011 ) , 157 , 593 -- 600 DOI 10.1099 / mic.0.043620-0 Escherichia coli mhpR gene-expression is regulated by catabolite repression mediated cAMP -- CRP complex by the I. Manso , J. L. Garcıa and B. Galán Correspondence B. Galán Departamento de Biologıa Medioambiental , Centro de Investigaciones Biológicas , Consejo Superior de Investigaciones Cientıficas ( CSIC ) , Ramiro de Maeztu 9 , 28040 Madrid , Spain bgalan@cib.csic.es The expression of the mhp genes involved in the degradation of the aromatic compound 3 - ( 3-hydroxyphenyl ) propionic acid ( 3HPP ) in Escherichia coli is dependent on the MhpR transcriptional activator at the Pa promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Nucleotide sequence and characterization of the sfs1 gene : sfs1 is involved in CRP * - dependent mal gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808940" target="_blank">8808940</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>]<br></li>
<li>Cold Spring Harbor Laboratory Press , Cold Spring Harbor , New York Martinussen J , Mollegaard NE , Holst B , Douthwaite SR , ValentinHansen P ( 1989 ) A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Martinussen , J , , Mollegaard , N.E. , Hoist , B. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1988 ) A new version of negative controt , DNA sequences involved in expression and regulation of CytR and cAMP/CRP controtled genes in Escherichia coli In DNA-protein interactions in Transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Valentin-Hansen , P. ( 1985 ) DNA sequences invoived in expression and regulation of DeoR , CytP and cAMP-CRP controlled genes in E coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>DNA sequences involved in expression and regulation of DeoR , CytR and cAMI ' : CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_245">
and Valentin-Hansen , P. ( 1989 ) A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP/CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></label>
<input id="rp_similarity_245", type="checkbox">
<div class="RP_similar">
<li>A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_247">
Valentin-Hansen , P. ( 1985 ) DNA sequences involved In expression and regulation of DeoR , CytR and cAMP-CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></label>
<input id="rp_similarity_247", type="checkbox">
<div class="RP_similar">
<li>DNA sequences involved in expression and regulation of DeoR , CytR , and cAMP-CRP controlled genes in Escherichia coli , p. 273-288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_249">
DNA sequences involved in expression and regulation of CYTR and cAMP/CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></label>
<input id="rp_similarity_249", type="checkbox">
<div class="RP_similar">
<li>DNA sequences involved in expression and regulation of CytR and cAMP/CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li></div>
<li>EMBO J 1:1049 -1054 Valentin-Hansen P ( 1985 ) DNA sequences involved in expression and regulation of DooR , CytR and cAMP-CRP controlled genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP -- cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Interactions of elements of negative ( CytR ) and positive ( cAMP-CRP ) regulation in the promoter region of the uridinephosphorylase ( udp ) gene in Escherichia coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_256">
nbt .2014.03.002 Castaño-Cerezo S , Bernal V , Blanco-Catalá J , Iborra JL , Cánovas M ( 2011 ) cAMP-CRP co-ordinates the expression of the protein acet-ylation pathway with central metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25524697" target="_blank">25524697</a>]<br></label>
<input id="rp_similarity_256", type="checkbox">
<div class="RP_similar">
<li>Microb Cell Factories 8:54 Castaño-Cerezo S , Bernal V , Blanco-Catalá J , Iborra JL , Cánovas M ( 2011 ) cAMP-CRP co-ordinates the expression of the protein acet-ylation pathway with central metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li></div>
<li>CsrA and cAMP-CRP both activate motility in E. coli through effects on ﬂhDC expression ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>J Bacteriol 161 : 641 -- 649 Kawamukai M , Kishimoto J , Utsumi R , Hineno M , Komano T , Aiba H ( 1985 ) Negative regulation of adenylate cyclase gene ( cya ) expression by cAMP-CRP in Escherichia coli : studies with cya-lac protein and operon fusion plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Moreover , further transcriptome analysis conducted in different E. coli strains also failed to include glg genes in the CRP regulon [ 22,23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>CRP regulator modulates multidrug-resistance of Escherichia coli by repressing the mdtEF multidrug efﬂux genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>CRP -- cAMP activation of lacP and binding to galP1D4 is osmotically sensitive CRP is a positive and negative regulator of many different genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The elucidation of cAMP-CRP involvement in E. coli autoinduction impacts many areas , including the growth of E. coli in fermentation processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>cAMP -- CRP complex is estimated to control more than 100 genes in E. coli , including ptsH and crr genes ( Titgemeyer and Hillen , 2002 ; Reuse and Danchin , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
<li>INTRODUCTION The structural genes encoding the ~ ystems for nucleoside uptake and metabolism in E. coli are organized in a regulon that is controlled positively by the cAMP/CRP activator complex and negatively by the DeoR and CytR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>The cAMP-CRP complex regulates many E. coli genes positively and/or negatively at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6344011" target="_blank">6344011</a>]<br></li>
<li>Our results also point to a direct role of S in response-to-nutrient starvation , suggesting that the S regulon overlaps with other nutrient-speciﬁc regulatory pathways such as cAMP/CRP , NtrB / 54 NrtC / , and PhoB/PhoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>Since nitrogen assimilation consumes energy and intermediates of central metabolism ( 35 , 38 ) , the direct action of the cAMP-CRP complex as a modulator of E 54 could provide a regulatory link between carbon and nitrogen metabolism by the dual regulatory role of PTS components and the cAMP-CRP complex ( the activation of 70-dependent promoters involved in carbon metabolism and the downregulation of 54-dependent promoters involved in nitrogen assimilation ) ( 5 , 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>The authors proposed a regulatory role for the cAMP-CRP complex as a switch balancing carbon metabolism and nitrogen assimilation in E. coli by cAMP-dependent repression of a 54-dependent promoter via CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>As NtrC protein levels are controlled by the glnAp2 promoter , the dual regulatory role of the CRP -- cAMP complex ( i.e. the activation of s - dependent promoters involved in carbon 70 metabolism and the downregulation of s - dependent 54 promoters , in particular the glnAp2 promoter , involved in nitrogen assimilation ) might provide a novel and general regulatory linkage between carbon metabolism and nitrogen assimilation in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Fig. 3 shows the activity of 12 representative promoters that were predicted to be under the control cAMP-CRP after SELEX-chip analysis , together with 4 known promoters , ptsG ( glucose-specific PTS ) , dgsA ( global regulator of carbon metabo-lism ) , ptsH ( PTS Hpr component ) and glpF ( glycerol facilitator ) , that are activated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The cAMP/CRP complex is a positive transcriptional regulator of a number of catabolic operons , and as such , plays a role in catabolite repression , whereby secondary carbon sources are not catabolized in the presence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>Based on bioinformatics , the promoter region of comC is predicted to contain a binding site for the cAMP response protein ( CRP ) , which is a global regulator involved in catabolite repression ( Brown and Callan 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22089859" target="_blank">22089859</a>]<br></li>
<li>Indeed , a putative FIS binding site ( TTGCA-TATTTTTAACACAA , 236 to 18 ) lies within the fadBA Expression of fadL and fadD is directly activated by the CRP-cAMP complex The global regulator cAMP-CRP complex [ 28 ] can act as either an activator [ 29 ] or a repressor [ 30 ] in expression of genes involved in many E. coli metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The cAMP-CRP complex has been shown to be a global regulator of numerous cellular mechanisms and gene-expressions , either as an activator or as a repressor ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023186" target="_blank">9023186</a>]<br></li>
<li>O cAMP-CRP is a dual regulator , acting as an activator or a repressor [ 13,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>A number of the genes for both glycolysis and gluconeogenesis are under the control of catabolite repressor activator Cra , but independent of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Including these newly identified targets , all the ABC family transporters for carbon source transport in E. coli are now recognized to be all under the control of cAMP-CRP ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Besides these genes for carbon-utilization , we found that a small set of genes apparently unrelated to carbon metabolism is under the control of cAMP-CRP , including some ribosome-re-lated genes ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>As a result , almost all the genes for transport of carbon sources are now identified to be under the direct control of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The CRP group mainly contains transcription factors that regulate catabolite-sensitive genes in response to glucose starvation as signalled by cAMP ( Busby & Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Commonly , cAMP-CRP is an activator of the 70-dependent transcription of genes coding for catabolism of alternative carbon sources ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>cAMP-bound active form of CRP is then regulates transcription of a number of genes that are involved in the utilization of non-glucose carbon sources in the absence of glucose [ 9,11 -- 13 ] , and thus it is an antagonist of catabolite repression [ 9,12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>PTS not only transports specific sugars but also in the absence of its substrates , stimulates , through adenylate cyclase ( Cya ) , the production of cAMP which in turn activates the transcription of many cAMP-CRP dependent catabolic genes , including those involved in glycerol utilization [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Various genes , including rpoS and several sigma S-dependent genes , such as bolA , are negatively controlled by cAMP-CRP ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>In addition , the genes encoding RMF ( ribosome modulation factor ) ( Fig. 1A ) and RaiA ( stationary-phase translation inhibitor and ribosome stability factor ) ( Fig. 1B ) were identiﬁed as the predicted regulation targets of cAMP-CRP ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Key words Adenylate cyclase · cAMP-synthesis · CRP · Transcriptional regulation · Posttranscriptional regulation Introduction Cyclic AMP is an important regulatory molecule involved in various cellular activities in many organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>In addition to cAMP-CRP , the leucine-responsive regulatory protein ( Lrp ) is involved in the regulation of csiD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>DemonSlration o / ¡ he method 011 sites cAMP protein , CRP , is involved in the transcription regulation of several E.coll genes , either as an activa tor or as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>In fact , the FadR system appears to form a global control network as described by Gottesman for the cAMP-CRP system , the SOS system and the proteins involved in the heat shock response ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2843809" target="_blank">2843809</a>]<br></li>
<li>- Sigma-S and the cAMP-CRP complex are global regulatory factors involved i nstationary-phase induction of large groups of genes in Escherichiacoli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>In this regard the fad genes are part of the cAMP-CRP global regulatory network like many other catabolic genes involved in carbon assimilation and utilization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Future work should shed further light on the regulation of the expression and catabolite repression of the mdh gene , the role of CRP action and possible co-regulations of tricarboxylic-acid-cycle genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>CRP plays a complex role in the regulation of the expression of rpoS itself ( 34 ; F. Scheller and R. Hengge , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The expression of gapA , encoding GAPDH , could be related to CRP -- cAMP ( Charpentier and Branlant , 1994 ) , a regulatory complex indicating that energy is in short supply , and orchestrating gene regulation cascades in response to the growth conditions and environmental cues ( Domka et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22176949" target="_blank">22176949</a>]<br></li>
<li>We also examined their role in the regulation of the malT gene , which encodes MalT and which is controlled by cAMP-CRP ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>It has been well documented that the cAMP-CRP complex plays an important role in E. coli gene regulation and mediates catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Discussion The cAMP-CRP complex plays an important role in the regulation of many promoters in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In order to demonstrate the contribution of cAMP-CRP to the glucose-effect on ula regulon transcription , we analyzed the expression of ( ulaG-lacZ ) and ( ulaA-lacZ ) in the genetic background of a crp mutant , strain JA216 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Key words : autoregulation/catabolite repression/CRP - cAMP/crp expression/transcriptional activation The inhibition of gene-expression by glucose in culture medium has been a paradigm for understanding how genes are regulated in response-to-nutrient status in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The presence of glucose interferes with the activation of membrane-bound ad-enylate cyclase ( Cya ) , decreases cAMP levels in the cell , and represses the expression of CRP ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16597943" target="_blank">16597943</a>]<br></li>
<li>About two-thirds of the genes encoding these carbon starvation proteins ( the cst genes ) require the cAMP-cAMP recepter protein ( cAMP-CRP ) complex for induction during starvation ( 31 ) , and it is likely that the increase in intracellular cAMP levels that occurs at the onset of carbon starvation ( 9 , 24 , 28 ) plays a role in this induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2163388" target="_blank">2163388</a>]<br></li>
<li>m e role OfcAMPin the expression ofcsi : : lacZ fusions Many carbon starvation-induced proteins identified on two-dimensional gels or by the analysis of lac2 gene fusions are not expressed in cya or crp mutants and cAMP/CRP has been implicated as a major regulatory system for starvation-induced genes ( Matin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>The expression of most of the FIS-regulated genes involved in the catabolism of sugars and nucleic acids is also dependent on the cAMP-CRP complex ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>All of these genes encode enzymes or transport proteins involved in the catabolism of sugars or nuc-leic acids , and their expression is also dependent on the CAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>The galactose utilization and transport genes have a system of regulatory control and coordination which includes two repressors as well as the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>The ompA and ompF genes encoding membrane proteins and the deo gene involved in nucleic acid metabo-lism are also under the control of the cAMP-CRP complex [ 1-3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>The genes encoding enzymes involved in glycogen metabo-lism ( glycogen synthase and glycogen phosphorylase ) are linked on a single operon that is regulated by the cAMP CRP z complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9334229" target="_blank">9334229</a>]<br></li>
<li>Little is known about the effects of acid and heat stress on cAMP levels and CRP activity , but it has been shown that the expression levels of other acid stress genes ( gadA and gadBC ) ( 5 ) and heat shock genes ( hslS , hslT , and htpG ) ( 16 ) are altered in crp mutant strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>In addition , CyaR transcription depends on the cAMP/CRP complex and the sRNA affects the expression of non-OMP genes as well ( Johansen et al. , 2008 ; Papenfort et al. , 2008 ; De Lay and Gottesman , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400782" target="_blank">19400782</a>]<br></li>
<li>In order to conﬁrm the role of CRP on the expression of cobB , patZ and acs genes , cAMP response experiments were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>In rich LB medium , cAMP-CRP plays the role of a transition state repressor for the expression of osmY ( and perhaps other stationary-phase-inducible genes as well ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>cAMP/CRP can also play a negative role in controlling the expression of stationary phase-induced genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>Whereas Mlc is responsible for induction of ptsG ( and several other genes ) in the presence of glucose , the catabolite activator complex cAMP-CRP is also absolutely necessary for ptsG expression ( 15 , 26 , 32 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855233" target="_blank">16855233</a>]<br></li>
<li>A possible link between these two global regulatory processes ( CRP and DNA supercoiling ) might be that the cAMP-CRP complex regulates the expression of genes encoding the DNA topoisomerases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>As a result , the cAMP-CRP complex regulates the expression of hundreds of genes ( 10 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>This negative influence of cAMP can be attributed to the presenceof os , which plays a positive role in csiE expression and which is itself under negative control by cAMP-CRP ( Lange and Hengge-Aronis , 1994a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>This supports the conclusion that cAMP-CRP plays a positive role in the production of functional RMF ( Fig. 8D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Using a combination of genetic , biochemical , and gene fusion techniques on single-copy plasmids , we have analyzed one such promoter , deoCp2 , to define the DNA sequences involved in positive and negative control by the cAMP-CRP complex and the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>specific promoter ( P5 ) which is sensitive to glucose-repression and which is regulated by cAMP and CRP is involved in the control of rpoH transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>From these results , we concluded that pCMl carries the csiE : : lacZ fusion as well as the entire upstream region involved in the control of csiE expression by os and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>These findings suggest that an unidentified third component besides os and cAMP-CRP is involved in the control of csiE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>cAMP-CRP and LsrR are involved in transcriptional-control of lsr and lsrRK operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>It is possible that the CRP -- cAMP complex may activate transcription of another gene ( such as dnaK ) which , in turn , acts as an anti-s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Structural genes : araA , araB , and araD ; regulator gene : araC ; controlling sites : operator ( araO ) ; initiator ( aral ) ; RNA polymerase initiation site ( araP ) ; site ofaction of the cAMP-CRP complex [ ara ( CRP ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>ac.jp Moreover , the data presented indicate the occurrence of a unique regulation of bacterial gene-expression in the following points : ( 1 ) 2 specific ( not global ) regulators involved in control of a single metabolism system , ( 2 ) 1 specific regulator acting as both positive and negative regulators , and ( 3 ) a possible interaction between the specific regulator and cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>This suggests that cAMP may act , via CRP , as a positive regulator of acrAB expression during carbon limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11886757" target="_blank">11886757</a>]<br></li>
<li>This work represents an initial contribution to the elucidation of the role of cobB and yﬁQ gene products in the control of the metabolism of E. coli , describing for the ﬁrst time how their expression is regulated and co-ordinated by central metabolism regulator CRP and the concentration of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>Thus , cAMP-CRP may have a global regulatory role that goes beyond mediating catabolite control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>It is possible that cAMP-CRP may control another transcriptional regulator ( s ) that modulates luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Although more experimental data are necessary to understand the precise mechanism ( s ) by which the cAMP-CRP complex controls gyrA expression as well as the physiological role of this control , this work offers new perspectives of interest for further studies of CRP-mediated gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>It was shown that expression of the fumA gene resembles that of the succinate dehydrogenase operon and other ` citric-acid-cycle ' genes in being repressed by glucose ( i.e. , dependent on the cAMP-CRP complex ) and anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3282546" target="_blank">3282546</a>]<br></li>
<li>On the other hand , a negative role for cAMP-CRP was shown in the control of the rpoS gene ( 24 ) , of some pex genes ( 29 ) , and of several urs-dependent genes ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We examined the gene-expression of glpD and glpE under the control of cAMP-CRP using glpD-iac and `` glpE-lac fusino genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Abbreviations : CRP , CAMPreceptor protein ; FNR , fumarate and nitrate reduction regulator ; IHF , integration host factor ; ODH , 2-oxoglutarate dehydrogenase ; PDH , pyruvate dehydrogenase ; SCS , succinyl-CoA synthetase ; SDH , succinate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R Involvement of cAMP-CRP in transcription activation and repression of the pck gene encoding PEP carboxykinase , the key enzyme of gluconeogenesis Masahiro Nakano1 , Hiroshi Ogasawara1 ,2 , Tomohiro Shimada1 ,3 , Kaneyoshi Yamamoto1 & Akira Ishihama1 1 2 Department of Frontier Bioscience and Research Center for Micro-Nano Technology , Hosei University , Koganei , Tokyo , Japan ; Research Center for Human and Environmental Sciences , Shinshu University , Ueda , Nagano , Japan ; and Chemical Resources Laboratory , Tokyo Institute of 3 Technology , Nagatsuda , Yokohama , Japan Correspondence : Akira Ishihama , MicroNano Technology Research Center , Hosei Abstract cAMP receptor protein ( CRP ) is the best characterized global regulator of Esc-herichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_337">
Nakano , M. , Ogasawara , H. , Shimada , T. , Yamamoto , K. & Ishihama , A. ( 2014 ) Involvement of cAMP-CRP in transcription activation and repression of the pck gene encoding PEP carboxykinase , the key enzyme of gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></label>
<input id="rp_similarity_337", type="checkbox">
<div class="RP_similar">
<li>Involvement of cAMP-CRP in transcription activation and repression of the pck gene encoding PEP carboxykinase , the key enzyme of gluconeogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li></div>
<li>Putrescine is a biosynthetic precursor of another poly ¬ C , L-arginine P , adenosine Abbreviations : ODC , L-ornithine decarboxylase ; AD decarbo × ylase ; CRP , cAMP receptor protein ; cAM -3 ' :5 ' - cyclicmonophosphate Offprint requests to : S.M. Boyle 0026-8925/82 / 0186/0482 / $ 01.2 We assayed ODC and ADC activities in extracts from cells grown under conditions known to modulate intracellular levels of cAMP : by varying the major carbon source ( Buettner et al. 1973 ; Pastan and Adhya 1976 ) , addition of cAMP to growth medium ( Perlman and Pastan 1968 ) , and in bacterial strains carrying mutations in the adenylate cyclase ( EC 4.6.1.1 ) structural gene ( cya ) or in the crp structural gene encoding the cAMP receptor protein ( Brickman et al. 1973 ; Fraser and Yamazaki 1978 , 1980 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6290846" target="_blank">6290846</a>]<br></li>
<li>In this study , we identified some ABC-family transporter for carbon sources such as ugpB for glycerol-3-phosphate-transport [ 56 ] and xylF for xylose transport [ 56 ] as the novel regulation targets of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Novel roles of cAMP receptor protein ( CRP ) in regulation-of-transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752906" target="_blank">22752906</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25790494" target="_blank">25790494</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Cold Spring Harbor Laboratory , NY , pp 1.119 -- 111.122 Shimada T , Fujita N , Yamamoto K , Ishihama A ( 2011 ) Novel roles of cAMP receptor protein ( CRP ) in regulation-of-transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_343">
Shimada T , Fujita N , Yamamoto K & Ishihama A ( 2011b ) Novel roles of cAMP receptor protein ( CRP ) in regulation-of-transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></label>
<input id="rp_similarity_343", type="checkbox">
<div class="RP_similar">
<li>Shimada , T. , Fujita , N. , Yamamoto , K. & Ishihama , A. ( 2011b ) Novel roles of cAMP receptor protein ( CRP ) in regulation-of-transport and metabolism of carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23301696" target="_blank">23301696</a>]<br></li></div>
<li>Whereas a large group of these proteins requires cAMP/cAMP receptor protein ( CRP ) for induction and apparently is specific for carbon starvation ( Schultz eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>For the cAMP receptor protein ( CRP ) - regulated sRNA CyaR ( 18 , 19 ) , we detected as yet unidentiﬁed targets in primary-metabolism ( sdhA ) and the phosphotransferase system ( ptsI ) , constituting previously unreported links of the CyaR regulon to carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Novel Roles of cAMP Receptor Protein ( CRP ) in Regulation of Transport and Metabolism of Carbon Sources Tomohiro Shimada , Nobuyuki Fujita , Kaneyoshi Yamamoto , Akira Ishihama * 1,2 3 1,2 1,2 1 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo , Japan , 2 Micro-Nanotechnology Research Center , Hosei University , Koganei , Tokyo , Japan , 3 Department of Biotechnology , National Institute of Technology and Evaluation , Shibuya-ku , Tokyo , Japan Abstract CRP ( cAMP receptor protein ) , the global regulator of genes for carbon source utilization in the absence of glucose , is the best-studied prokaryotic transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Citation : Shimada T , Fujita N , Yamamoto K , Ishihama A ( 2011 ) Novel Roles of cAMP Receptor Protein ( CRP ) in Regulation of Transport and Metabolism of Carbon Sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In Pseudomonadaceae , the CRP ortholog Vfr ( virulence factor regulator ) regulates quorum-sensing , protein-secretion , motility and adherence ( 42 -- 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In Xanthomonadaceae , the CRP ortholog Clp ( CAP-like protein ) regulates the synthesis of extracellular enzymes , pigment and xanthum gum ( 46,47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Abbreviations : amp , / I-lactamase gene ( ApR ) ; Ap , ampicillin ; bp , base pair ( s ) ; BLA , / 3-lactamase ; CAMP , cyclic adenosine-3 ' : 5 ' - monophosphate ; Cm , chloramphenicol ; crp , CRP-coding gene ; CRP , CAMP receptor protein ; cya , adenylcyclase gene ; d , deletion ; EtdBr , ethidium bromide ; kb , kilobases ; IPTG , isopropyl-B-thio-D-galactoside ; MOPS , 3 - ( N-morpholino ) - pro-panesulfonic acid ; ODC , ornithine decarboxylase ; PA , polyacrylamide ; R , resistance ; SDS , sodium dodecyl sulfate ; speC , gene coding for ODC ; Tc , tetracycline ; ret , Tc resistance gene ( TcR ) ; [ 1 , designates the plasmid-carrier state ; : : , novel joint .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; cAMP , cyclic AMP ; CRP , catabolite activator protein ; cytR , gene encoding CytR ; CytR and DeoR repressor proteins ; A , deletion ; deoR , gene encoding DeoR ; kb , is part of this regulon ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>ExuR , another Abbreviations : Ap , ampicillin ; bp , base pair ( s ) ; CAMP , cyclic AMP ; Cm , chloramphenicol ; CRP , CAMP receptor protein ; kb , repressor involved in the regulation of the hexuronate system genes , has no effect on the expression of uidA ( Ritzenthaler et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; Ap , ampicillin ; aroN , gene encoding DAHP synthase ( Trp ) ; bp , base pair ( s ) ; CAMP , cyclic AMP ; CRP , CAMP-receptor protein ; DAHP , 3-deoxy-D-arabinoheptulosonate-7-phosphate ; kb , kilobase pair ( s ) or 1000 bp ; nt , nucleotide ( s ) ; P , promoter ; ss , single strand ( ed ) ; asp , transcription start point ( s ) ; URF , The aroH gene from Escherichia coli encodes 3-deoxy-D-arabinoheptulosonate-7-phosphate ( DAHP ) synthase ( Trp ) , one of three isoenzymes which catalyse the first committed step in the biosynthesis of aromatic-amino-acids and vitamins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1677907" target="_blank">1677907</a>]<br></li>
<li>Glutamate and glutamine not only feed directly into protein-synthesis , but also Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; CAMP , cyclic AMP ; CRP , CAMP-receptor protein ; GOGAT , glutamate synthase ; kb , kilobases or 1000 bp ; NN , no nitrogen minimal medium ; nt , nucleotide ( s ) ; ORF , open reading frame ; PA , polyserve as amino-group donors for a wide range of other nitrogen-containing cellular components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326786" target="_blank">3326786</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; Ap , ampicillin ; bp , base pair ( s ) ; CAMP , adenosine 3 ' ,5 ' - cyclic monophosphate ; CRP , CAMP receptor protein ; kb , 1000 bp ; Km , kanamycin ; nt , nucle-otide ( s ) ; on ` , origin of DNA-replication ; ORF , open reading Hanatani et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2830169" target="_blank">2830169</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; Ap , ampicillin ; bp , base pair ( s ) ; BSA , bovine serum albumin ; CAMP , adenosine 3 ' ,5 ' - cyclic monophosphate ; CRP , CAMP receptor protein ; gp , gene product ; GDH , glutamate dehydrogenase ; GOGAT , glutamate 2-oxoglutarate + NH , ' + NADPH + L-glutamate + NADP ' + H , O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2465204" target="_blank">2465204</a>]<br></li>
<li>In addition , a number of global regulators , including cyclic AMP-CRP ( Sutton and Freun-dlich 1980 ; Friden et al. 1984a ) and guanosine tetra-phosphate ( Freundlich 1977 ) influence ilvBN expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine-metabolism ) was controlled by protein regulators such as FNR ( transcriptional regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine-metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>The BglII restriction enzyme site , the transcription initiation site , and a translation-initiation codon ( Met ) are indicated We thank H. Aiba for providing CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The production of L-asparaginase II in E. coli is known to be subject to positive regulation by two pleiotrophic regulatory proteins : the FNR protein ( 20 ) , which activates a number of genes during anaerobiosis ( 18 , 39 , 44 ) ; and CRP ( 8 , 35 ) , which controls the initiation of transcription of genes in various catabolite pathways ( 5 , 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The 3-D crystallographic structures of proteins involved in transcriptional regulation of E. coli and its phages ( e.g. , TrpR , CRP , CI repressor and Cro protein from phages 2 and 434 ) have a common structural motif , the ` helix-turn-helix ' motif ( Dodd and Egan , 1987 ; Brennan and Matthews , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1995431" target="_blank">1995431</a>]<br></li>
<li>Interestingly , the carbon-catabolite-repression protein ( CRP ) has been shown to regulate several types of CU ﬁmbriae expression including type 1 and Pap ﬁmbriae [ 10 , 91 , 92 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21344461" target="_blank">21344461</a>]<br></li>
<li>Full expression of ﬂhCD and thus of the chemotaxis and ﬂagellum genes depends on cAMP-CRP ( 37 , 63 ) ; the carbon storage regulator Csr ( 70 ) ; the nucleoid-associated protein H-NS ( 9 , 26 , 64 ) ; the heat shock proteins DnaK , DnaJ , and GrpE ( 59 ) ; the regulators of phosphatidylethanolamine-biosynthesis -- Pss and Psd ( 58 ) ; polyphosphate ( 54 ) ; and the quorum-sensing regulator QseBC ( 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>Mutations that affect transcription and cyclic AMP-CRP regulation of the adenylate cyclase gene ( cya ) of Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Keywords : Fis ; a subunit of RNA polymerase ; CRP ; proP ; transcription activation * Corresponding author Introduction The Escherichia coli proP P2 promoter , which controls the expression of an integral membrane transporter of proline , glycine betaine , and other osmoprotecting compounds , is induced just prior to entry into stationary phase .1 ± 5 Transcription from the proP P2 promoter is strongly dependent on the Fis protein as well as the stationary phase sigma factor , s38 , encoded by rpoS , which results Abbreviations used : aCTD , C-terminal domain of RNA polymerase a subunit , FIS , factor for inversion stimulation ; CRP , cyclic AMP receptor protein ; BABE , r-bromoacetamidobenzyl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The expression of other stationary phase proteins in E. coli involves a number of additional regulons ( or modulons ) including the cyclic AMP/CRP system , the heat shock regulon and the ArcAB regulon [ 1,2,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
<li>com / locate / bbapap Review Structural overview on the allosteric activation of cyclic A MP receptor protein Hyung-Sik Won a , Yoo-Sup Lee a , Sung-Hee Lee a , Bong-Jin Lee b , a Department of Biotechnology , College of Biomedical and Health Science , Konkuk University , Chungju , Chungbuk 380-701 , Republic of Korea b Research Institute of Pharmaceutical Sciences , College of Pharmacy , Seoul National University , Seoul 151-742 , Republic of Korea abstract article info Article history : Received 15 February 2009 Cyclic AMP receptor protein ( CRP ) is a prokaryotic global transcription regulator that controls the expression of nearly 200 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>The global regulator CRP , like the nucleoid protein IHF , has many low-affinity sites on the chromosome ( Grainger et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The protein shows a close homology to the transcriptional regulator CRP ( cyclo AMP receptor protein ) of E. coli , and the gene products of the hlyX gene of Actino-bacilluspleuropneumoniae , the orf240 gene of Rhizobium leguminosarum and the fixK gene of R. meliloti ( Fig. 1 ) ( De Crombrugghe et al. 1984 ; Shaw et al. 1983 ; Maclnnes et al. 1990 ; Colonna-Romano et al. 1990 ; Batut et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Introduction The enterobacterial cyclic AMP receptor protein ( CRP ) is a global transcription factor that plays a major role in controlling bacterial metabolism ( Kolb et al. , 1993 ; Barrett et al. , 2005 ) , and whole genome studies in Escherichia coli K-12 , using a variety of approaches , have now identiﬁed nearly 200 target promoters ( Gosset et al. , 2004 ; Zheng et al. , 2004 ; Grainger et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_373">
Introduction The cyclic AMP receptor protein ( CRP , also known as the catabolite activator protein , CAP ) is a global transcription factor , which plays a central role in the control of metabo-lism in Escherichia coli and other enteric bacteria ( Kolb et al. , 1993 ; Barrett et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></label>
<input id="rp_similarity_373", type="checkbox">
<div class="RP_similar">
<li>Introduction The cyclic AMP receptor protein ( CRP ) is a global transcription factor that plays a key role in controlling metabo-lism in enteric bacteria ( Kolb et al. , 1993 ; Barrett et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li></div>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>For example , cyclic AMP receptor protein ( CRP ) and IHF are also known to play similar roles in gene regulation in other promoters ( Lobell and Schleif , 1990 ; Søgaard-Andersen et al. , 1991 ; Richet , 1996 ; Yang et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Factors required for motility and full flhDC expression include cyclic AMP ( cAMP ) and cAMP receptor protein ( CRP ; Kutsukake , 1997 ; Soutourina et al. , 1999 ) , histone-like nucleoid-structuring ( H-NS ) protein ( Bertin et al. , 1994 ) , the heat shock response network proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) , Pss and Psd , which are involved in the synthesis of phosphatidy-lethanolamine in cell membranes ( Shi et al. , 1993 ) , and inorganic polyphosphate ( Rashid et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Flagellar synthesis and expression has been shown to be regulated by several environmental cues , including cyclic-AMP receptor protein ( cAMP-CRP ) ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , temperature ( Adler and Templeton , 1967 ) , osmolarity ( Shin and Park , 1995 ) , cell-cycle control ( Nishimura and Hirota , 1989 ) and bacterial cell density ( QS ) ( Chilcott and Hughes , 2000 ; Sperandio et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Cra mediates catabolite repression or catabolite activation depending on the target gene era controls gene-expression in vivo Expression of the negatively Cra-regulated pykF gene and the positively Cra-regulated ppsA gene was recently studied as a function of carbon source Beta-galactosidase activity measurement of singlecopy , chromosomal fusions of these genes to lacZ in E. coli strains wild-type for or lacking the Cra protein showed that Cra controls the response to carbon source independently of the cAMP-cAMP receptor protein ( CRP ) complex ( Bledig et al. , 1996 ; T.M. Ramseier and M.H. Saier , Jr. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9084760" target="_blank">9084760</a>]<br></li>
<li>In the absence of glucose and/or other PTS sugars , the phosphorylated form of Crr may be accumulated in the cell , and , in connection , it is suggested that the phosphorylated Crr activates adenylate cyclase to raise the synthesis rate of cAMP ( Postma et al. , 1993 ) , which is needed for CRP to activate the transcription of many other transport protein genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>These include the transcription factor FNR that mediates anaerobic regulation ( Unden and Schirawski , 1997 ) , the cAMP receptor protein ( CRP ) , which activates transcription of catabolic operons ( Kolb et al. , 1993 ) , and the nucleoid-associated protein H-NS ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) - AMPc complex activates transcription of many gluconeogenic genes such as acs , actP , aceBAK and maeB [ Gosset et al. , 2004 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18679018" target="_blank">18679018</a>]<br></li>
<li>Interestingly , AraC-dependent transcription initiation at the araBAD promoter is increased by the cyclic AMP receptor protein ( CRP ) , a well-characterized global regulator , controlled by cyclic AMP ( cAMP ) , that activates the expression of a number of genes in response to a variety of stresses , including glucose starvation ( reviewed by Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>In enteric bacteria , it acts as an allosteric effector of the cAMP receptor protein ( CRP ) and is essential for the regulation of the transcription of a large number of genes ( reviewed by Botsford and Harman 1992 ; Kolb et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>PItYSIOLOGY VOL 44 ISBN 0-12-027744 - I Copyright © 2001 Academic Press All rights of reproduction in any tiwm reserve ABBREVIATIONS cAMP CAP CRP FLP FNR Adenosine 3 ' ,5 ' - monophosphate Catabo -RCB- ite gene activator protein cAMP receptor protein FNR-like protein fumarate and nitrate reduction regulator 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Abbreviations : CRP , cAMP receptor protein ; FNR , fumarate and nitrate reduction regulator ; MALDI-TOF , matrix-associated laser desorption ionization-time of ﬂight ; MBP , maltose-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>A search revealed several potential binding sites for CRP and FNR ( the cyclic AMP receptor protein or catabolite gene activator protein and the regulator of fumarate and nitrate reduction or anaerobic gene activator protein , respectively ; not shown ) , two putative sites for Fur ( the ferric-uptake regulatory protein ; GATAATGATA/T , half-site concensus ; Cardayre and Neilands , 1991 ) and several putative stem-loop structures , including a potential rho-independent terminator for cysB , AG = -80 kJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1541275" target="_blank">1541275</a>]<br></li>
<li>Its expression is also dependent on the global regulatory proteins mediating catabolic repression via cyclic AMP receptor protein ( CRP ) , or anaerobiosis via fumarate nitrate reduction ( FNR ) transcriptional regulator , and the nucleoid-associated DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Moreover , it is subject to global control by the same regulatory proteins which are known to modulate expression of carnitine-metabolism , i.e. , the cyclic AMP ( cAMP ) receptor protein ( CRP ) which mediates the activation of catabolic operons , the transcriptional regulator FNR that is responsible for anaerobic induction , and the DNA-binding protein H-NS ( 19 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>In this report , we show that both AI-2 synthesis and uptake in Escherichia coli are subject to catabolite repression through the cyclic AMP ( cAMP ) - CRP complex , which directly stimulates transcription of the lsr ( for `` luxS regulated '' ) operon and indirectly represses luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In this study , we show that catabolite repression inﬂu-ences AI-2 accumulation through the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which directly stimulates transcription of the lsr operon and indirectly represses luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>This indicates that the cyclic AMP ( cAMP ) - CRP complex weakly activates dcuB expression in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Our results show that the expression of sxy in E. coli is positively autoregulated via a mechanism dependent on CRP , cyclic AMP ( cAMP ) and a newly identiﬁed CRP-S site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>The cAMP-cAMP receptor protein ( CRP ) complex has a dual role at these promoters : it acts as a conventional activator and as a co-repressor for CytR ( Sogaard-Andersen efal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>In CytR regulated promoters , including the udpP promoter , the cAMP -- CRP complex acts as a transcriptional activator as well as a co-repressor for the CytR protein [ 5,23,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Known regulators involved in the control of acs expression in E. coli are cAMP receptor protein ( CRP ) , factor for inversion stimulation ( Fis ) , integration host factor ( IHF ) ( Wolfe 2005 ) and possibly the dual transcriptional regulator for an-oxic redox control ( ArcA ) ( Covert et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li>
<li>We had shown previously that the Pa and Pz promoters of the two PA catabolic operons in E. coli were under control of global regulators such as the cyclic adenosine monophosphate ( cAMP ) receptor protein ( CRP ) , that accounts for the catabolite repression control mediated by glucose , and the integration host factor ( IHF ) ( Ferrández et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Superimposed in the specific PaaX-mediated repression , two global regulators , the cAMP receptor protein ( CRP ) and the integration host factor protein ( IHF ) , act as activators of the gene-expression driven by Pa and Pz promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>We show here that in addition to crS , the global regulators Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex ( cAMP-CRP ) , and integration host factor ( IHF ) are involved in the control of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In recent studies we have shown that in addition to sS , the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex , Lrp , and integration host factor ( IHF ) are involved in modulating osmY transcription ( 33 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>In this system , the global activator protein cAMP receptor protein ( CRP ; also referred to as the catabolite gene activator protein , CAP ) , guides another DNA-binding protein , the CytR repressor , to a number of binding sites that share minimal sequence homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In this paper , we report that a sequence-specific binding protein , the cAMP receptor protein ( CRP ) , whose primary function is to control transcription as a function of catabolite conditions ( de Crombrugghe et al. 1984 ; Kolb et al. 1993 ) , can also function as an osmoregulator of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The crp * l gene encodes an altered cAMP receptor protein ( CRP * ) which is functional in the expression of some sugar fermentation genes , such as lac , ara , and man , in the absence of cAMP ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670911" target="_blank">2670911</a>]<br></li>
<li>In contrast , expression of the malKpromoter of the maltose regulon requires two activator proteins , MalT and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In addition to regulation by its own protein product , araC gene-expression is under positive control of the cAMP receptor protein ( CRP ) ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>In Haemophilus inﬂuenzae , competence gene-expression also requires the cAMP receptor protein ( CRP ) , a global regulator involved in the sugar starvation response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) also plays a role in this RpoS-GadX pathway by tempering expression of RpoS during growth under conditions that result in high cAMP levels ( Ma et al. , 2003b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) has recently been shown to play a positive role in controlling gcvT : : lacZ expression ( Wonderling & Stauffer , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>]<br></li>
<li>This is particularly severe for nagE whose expression is very low in the absence of a functional cAMP/CAP complex ( CAP = catabolite activating protein , also known as CRP = cAMP receptor protein ) ( Ptumbridge , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>We recently demonstrated that , in vivo , FIS regulates the expression of several genes needed for catabolism of sugars and nucleic acids , a majority of which are also transcriptionally regulated by cAMP -- cAMP-receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>A second regulator , cAMP/CRP , activates P2 , an effect that is largely inhibited by Lrp , such that catabolite repressor protein ( Crp ) and Lrp are rival activators of serA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>In addition to the control by the local transcriptional regulators , the pentose-transport and feeder metabolic pathways are modulated by the global transcription regulator CRP ( cAMP receptor protein ) [ 1 ^ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Three trans-acting proteins , GalR ( the gal repressor ) , cAMP receptor protein ( CRP ) - cAMP complex , and the histone-like protein HU , control transcription from these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Like other cytR-regulated transcriptionalunits , the synthesis is also activated by the cAMP receptor protein ( CRP ) control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>The factors involved include σS , the cAMP receptor protein ( CRP ) , σ ( !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>We show that this regulation occurs through reduction in transcription and depends on both cAMP and the cAMP receptor protein ( CRP ) but is independent of de novo protein-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>Arguably , the best-studied simple transcription factor is CRP ( cAMP receptor protein , also known as CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Cyclic AMP ( cAMP ) receptor protein ( CRP ) , also known as catabolite activator protein ( CAP ) is one of the global regulatory factors involved in station-ary-phase induction of large groups of genes in E. coli ( reviewed in reference 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>receptor protein ( CRP ) , also known as catabolite activator protein ( CAP ) , is responsible for the regulation of genes involved in various metabolic pathways in Escherichia coli ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Key words : CRP targets/DNA sequence/footprinting/gene fusions / gene regulation Introduction In bacteria such as Escherichia coli and Salmonella typhi-murium , the complex of adenosine 3 ' ,5 ' - monophosphate ( cAMP ) and the cAMP receptor protein ( CRP ) activates the expression of certain genes concerned with transport and utilization of energy sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>In Escherichia coli and related bacteria the cAMP receptor protein ( CRP or CAP ) is a DNA-binding protein that acts at promoters to regulate operon expression ( 1-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Superimposed in the specific MhpR-dependent regulation of the mhp catabolic genes from E. coli , the cAMP receptor protein ( CRP ) acts as a man-datory activator that tightly adjusts the expression of the cat-abolic genes to the overall growth status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>Transcription of these genes is activated in response to intracellular cAMP levels by the cAMP receptor protein ( CRP ) , a transcription factor that activates expression of over 100 E. coli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Sequence and Characterization of the sfsl G Is Involved in CRP * - Dependent mal Gene Expression in Escherichia coli Cyclic AMP receptor protein ( CRP ) binds to cAMP and regulates the expression of many genes negatively or positively in Escherichia coli ( 1 , 9 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>Nucleic Acids Research Volume 14 Number 5 1986 Nucleotide sequence of the CytR regulatory gene of E. coli K-12 To increase our knowledge of how the cyclic AMP/cyclic AMP receptor protein complex ( cAMP/CRP ) affects transcriptional initiation and how the interplay takes place between a set of regulatory proteins and RNA polymerase , our studies have focused on the identification and characterization of genes involved in uptake and catabolism of nucleosides and deoxyribonucleosides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>In E. coli , the cyclic AMP receptor protein ( CRP ) is a global transcription factor that regulates numerous genes by binding to a 22-bp DNA sequence ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>INTRODUCTION The Escherichia coli cyclic AMP receptor protein ( CRP ) is an important transcription factor that regulates transcription initiation for more than 100 genes mainly involved in catabo-lism of carbon sources other than glucose ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>INTRODUCTION The cyclic AMP receptor protein ( CRP or CAP ) of Escherichia coli is a sequence-specific DNA binding protein which regulates transcription of a number of genes in response to the carbon source ( 1-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>In particular , the transcription of genes needed for utilization of nucleosides in Escherichia coli is regulated by a repressor protein , CytR , in concert with the cyclic AMP ( cAMP ) - activated form of cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>An Escherichia coli global regulatory protein , cyclic AMP ( cAMP ) receptor protein ( CRP ) , is involved in the regulation of transcription , either positively or negatively , in many genes involved in carbon metabolism ( 4 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>In the course of a systematic screening of regulation targets of DNA-bind-ing transcription factors from E. coli , we realized that CRP ( cyclic AMP [ cAMP ] receptor protein ) , the global regulator for carbon source utilization , participates in regulation of some ribosomal protein genes , including the rmf gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Two QS systems have been identified in Escherichia coli K-12 , including an intact QS system 2 that is stimulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and a partial QS system 1 that consists of SdiA ( suppressor of cell-division inhibitor ) responding to signals generated by other microbial species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>CI , National Institutes of Health , The cyclic AMP receptor protein ( CRP ) , an important para-digm of the positive control of gene-expression , activates transcription at more than 100 promoters in Escherichia coli ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>Keywords : Transcription activation ; DNA footprinting ; Operon ; Filter binding ; CD ; FTIR The Escherichia coli cyclic AMP ( cAMP ) 1 receptor protein ( CRP ) is a 47,000 Da DNA-binding protein responsible for regulating the expression of over 100 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Regulation of glpD and glpE gene-expression by a cyclic AMP-cAMP receptor protein ( cAMP-CRP ) complex in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>The cyclic AMP ( cAMP ) receptor protein ( CRP ) regulates the expression of more than 100 promoters in Escherichia coli ( for a review , see reference 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Alone or in synergy with CRP ( cAMP receptor protein ) , MalT activates all the operons comprising the Escherichia coli maltose regulon , which encodes proteins involved in the uptake and degradation of maltodextrins ( Boos and Shuman , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692154" target="_blank">10692154</a>]<br></li>
<li>The cAMP receptor protein ( CRP ) complexed to cAMP binds at specific sites in promoters to regulate initiation of transcription of many operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_441">
The Escherichia coli cAMP receptor protein ( CRP ) represses the Rhizobium meliloti dctA promoter in a cAMP-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></label>
<input id="rp_similarity_441", type="checkbox">
<div class="RP_similar">
<li>Wang , Y.P. , Giblin , L. , Boesten , B. , and O'Gara , F. ( 1993 ) The Escherichia coli cAMP receptor protein ( CRP ) represses the Rhizobium meliloti dctA promoter in a cAMP-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li></div>
<li>The E. coli cAMP receptor protein ( CRP ) is a global transcriptional regulator that can serve as an activator , repressor , coactivator , and corepressor ( Kallipolitis et al. 1997 ; Busby and Ebright 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>Introduction cAMP receptor protein ( CRP ) is the best characterized global transcriptional regulator in Escherichia coli ( Kolb et al. , 1993 ; Busby & Ebright , 1999 ; Harman , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>One well-characterized mechanism in Escherichia coli involves the global regulator cAMP-receptor protein CRP ( or catabolite gene activator protein CAP ) , which controls hundreds of the genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16115199" target="_blank">16115199</a>]<br></li>
<li>Expression regulation of the genes coding for these enzymes has been studied in details using the model prokaryote Escherichia coli ( for reviews see [ 2 -- 4 ] ) , raising the concept that CRP ( cAMP receptor protein ) or CAP ( catabolite gene activator protein ) is the global regulator controlling transcription of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Enhancing E. coli tolerance towards oxidative stress via engineering its global regulator cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24038693" target="_blank">24038693</a>]<br></li>
<li>In Escherichia coli , cyclic AMP ( cAMP ) receptor protein ( CRP , also referred to as CAP ) is a global transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>The cyclic AMP receptor protein ( CRP ; also referred to as catabolite gene activator protein , CAP ) from Escherichia coli , a classic transcription regulator in prokaryotes , is also allosterically modulated by the effector molecule cAMP [ reviewed in 5 -- 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>References Aiba , H , ( 1983 ) Autoreguiation of the Escherichia coli crp gene : CRP is a transcriptional repressor for its own gene , Ceii 32 : 141-149 , Aiba , H , , and Krakow , J.S , ( 1981 ) Isolation and characterization of the amino and carboxyl proximal fragments of the adenosine cyclic 3 ' , 5 * - phosphate receptor protein of Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>In E. coli , the nanATEK-yhcH operon is positively controlled by cyclic AMP receptor protein ( CRP ) and negatively regulated by the upstream nanR-encoded transcriptional repressor , NanR ( 6 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>The CytR regulon in Escherichia coli constitutes a combinatorial regulatory system in which the cyclic AMP receptor protein ( CRP ) functions both as an independent transcriptional activator and as an ` adaptor ' for a specific repressor , CytR ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ; ( © ) Oxford University Press S0gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>1 Abbreviations : bistris , 2 - [ bis ( 2-hydroxyethyl ) amino ] -2 - ( hydroxymethyl ) - propane-1 ,3 - diol ; CRP , Escherichia coli cyclic AMP receptor protein ; CytR , Escherichia coli cytidine repressor protein ; DNase I , deoxyribonuclease I ; RNAP , bacterial RNA polymerase , Tris , tris ( hydroxymethyl ) amino-methane ; EDTA , ethylenediaminetetraacetic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Multiple regulation of nucleoside catabolizing enzymes in Escherichia coli : effects of 3 ' 5 ` cyclic AMP and CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Summary Activation of the two divergent Escherichia coli cai and ® x operons involved in anaerobic carnitine meta-bolism is co-dependent on the cyclic AMP receptor protein ( CRP ) and on CaiF , the speci ® c carnitine ¬ sensitive transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Transcription of the gal regulon promoters is regulated in different ways by Gal repressor ( GalR ) and Gal isorepressor ( GalS ) and a global regulator , adenosine cyclic-3 ′ ,5 ′ - monophosphate ( cAMP ) receptor protein ( CRP ) , responding to two signals , D-galactose and cAMP respectively ( Weickert and Adhya , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Recently , several promoters of r-protein operons ( e.g. , rpmI-rplT , rplM-rpsI , rpsF-priB-rpsR-rplI ) have been reported to be under the control of cAMP-CRP ( cyclic AMP receptor protein ) , the effector usually implicated in modulating carbon source utilization ( Shimada et al. 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25749694" target="_blank">25749694</a>]<br></li>
<li>We also have found that the timing and/or magnitude of this induction depends , in part , on the carbon regulator cyclic AMP ( cAMP ) receptor protein ( CRP ) , the oxygen regulator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>The 50S L20 operonb rplT < rpmI infC bamD > raiA Translation inhibitor set of genes that are involved in carbon-utilization for energy production is controlled by two global regulators , CRP ( cAMP receptor protein or catabolite activation protein ) and Cra ( catabolite repressor and activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>One well-characterized regulator of the genes for carbon source utilization is CRP ( cyclic AMP [ cAMP ] receptor protein ) ( sometimes called catabolite activator protein [ CAP ] ) , which is converted into the functional form after binding cAMP that is synthesized in the absence of glucose ( 10 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>For instance , the cAMP-cAMP receptor protein ( CRP ) complex has been implicated as an activator of carbon starva-tion-responsive genes ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>36 , No. 8 2667 -- 2676 doi :10.1093 / nar/gkn119 MelR is a member of the AraC-XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the melAB operon that encodes products essential for melibiose-metabolism ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>Majerfeld , I.H. , Miller , D. , Spitz , D. and Rickenberg , H.V. : Regulation ofthe synthesis ofadenylate cyclase in Escherichiu coli by CAMP-CRP receptor protein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>CRP , CAMP receptor protein ; Xgal , 5-bromo-4-chloro-3-indolyl - ~ - ~ - galactopyranosidc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>However , we also identiﬁed novel regulation , such as Zur repression of ykgMO , cyclic AMP response protein ( CRP ) repression of azuC , and CRP activation of ykgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>We have previously shown that yjiY expression is also dependent on cyclic AMP/cyclic AMP receptor protein ( CRP ) regulation ( 2 ) ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659770" target="_blank">24659770</a>]<br></li>
<li>The dcuB gene is strongly activated anaerobically by FNR , repressed in the presence of nitrate by NarL , and subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The in vivo rate of glycogen-biosynthesis has been shown to be affected by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) system ( 17 , 18 , 37 ) and by the relA gene , which is required for the synthesis of guanosine 5 ' - diphos-phate 3 ' - diphosphate ( ppGpp ) during the stringent-response ( 10 , 38 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>in vitro and in vivo experiIn this report , we show by ments , including primer extension analysis and site-directed mutagenesis , that cyclic AMP and the cyclic AMP receptor protein ( CRP ) directly regulate the expression of the ompB The overall positive effect by cyclic AMP ompB on operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>The data obtained indicate that the expression of gyrA is growth phase dependent and under the positive control of the cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>We discuss some mechanisms regulating the operon expression and present evidence for direct involvement of a cyclic AMP-cyclic AMP receptor protein ( cAMP-CRP ) complex in the control of dad transcription , MATERUILS AND METHODS Bacterial strains , plasmids , and phages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>tre operon expression is under the control of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex as well as that of TreR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>As expected , gut operon expression is under the control of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The NagC repressor together with the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex controls expression of the nag regulon ( Plumbridge , 1991 ; Plumbridge , and Kolb , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The cyclicAMP receptor protein ( CRP ) complexed with cyclic AMP regulates expression of many genesby binding to sitesat EXPERIMENTAL PROCEDURES or near manypromoters ofE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>171 , No. 2 Effect of Mutations in the Cyclic AMP Receptor Protein-Bindi Site on araBAD and araC Expression LORI STOLTZFUSt AND GARY WILCOXt * Department of Microbiology , University of California at Los Angeles , Los Angeles , California 90024 ng Received 27 July 1988/Accepted 16 November 1988 Maximum expression of the adjacent but divergently transcribed araBAD operon and araC gene requires the presence of cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>KEINOSUKE OKAMOTO AND MARTIN FREUNDLICH Department of Biochemistry , State University of New York , Stony Brook , NY 11794 ABSTRACT Expression of the crp gene is negatively autoregulated by the complex of cyclic AMP and its receptor protein ( cAMP-CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>In enteric bacteria , expression of the pts operon increases about threefold during growth on PTS substrates and requires an intact cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) system ( 72 , 280 , 390 , 471 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Cyclic AMP ( cAMP ) also negatively regulates speA gene-expression either directly or indirectly via binding to cAMP receptor protein ( CRP ) ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2198270" target="_blank">2198270</a>]<br></li>
<li>In addition , maximum expression of the araBAD op-eron and araC gene requires the presence of cyclic AMP ( cAMP ) and the pleiotropic regulator cAMP receptor protein ( CRP ) ( 1 , 6 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Introduction In Escherichiacoli and related bacteria , cAMP receptor protein ( CRP ) forms a complex with cyclic AMP ( cAMP ) , and regulates gene-expression by binding to CRP-binding sites on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Next , we assessed the role of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex in arb gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11994151" target="_blank">11994151</a>]<br></li>
<li>Choi , S. Kawase , M. Kawamukai , H. Sakai , T. Komano , Regulation of glpD and glpE gene-expression by a cyclic AMP-cAMP receptor protein ŽcAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>The ula regulon also seems to be controlled by the global regulators cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and fumarate nitrate reductase regu - lator ( FNR ) , since mutations in the corresponding genes abol - ished or delayed , respectively , growth on L-ascorbate ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>For example , ptsG transcription is activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) ( 2 ) , and negative control is exerted by the repressor Mlc , which inhibits ptsG transcription in the absence of glucose ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873911" target="_blank">23873911</a>]<br></li>
<li>The more extensively studied system is dependent on cyclic AMP ( cAMP ) and its receptor protein , CRP , which activates the transcription of many catabolic operons ( Busby and Kolb 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>The second major regulator of ptsG transcription is the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855233" target="_blank">16855233</a>]<br></li>
<li>The cyclic AMP receptor protein ( CRP ) is a cyclic AMP ( cAMP ) - dependent global transcription regulator ( for reviews , see references 11 and 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>69 FLAGELLAR PROMOTERS Class 1 Master Operon The early genes are transcribed from a class 1 ﬂagellar promoter in a single operon , ﬂhDC , which is sensitive to environmental and cell state sensors such as cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) ( 119 , 133 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>The observed changes in protein levels could reﬂect regulation by cyclic AMP ( cAMP ) receptor protein ( CRP ) , a transcription factor that modulates the expression of hundreds of genes , depending on glucose availability ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>The regulation of these genes is complex and involves the transcription factors RpoS ( 3 , 14 ) , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) ( 3 ) , HN-S ( 11 ) , and EvgA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>At the onset of glucose starvation , cyclic AMP and its receptor protein ( cAMP-CRP ) were found to play important roles in the expression of a number of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Much of the expression of nucleoside utilization genes is coordinated by a repressor protein , CytR , acting in concert with the cyclic AMP ( cAMP ) - activated form of the cAMP receptor protein ( CRP ) ( 2 , 9 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Transcription of tnaA is under the control of cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) ( 78 ) ; hence , catabolite repression is an important regulatory mechanism involved in indole-synthesis and bioﬁlm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>Several operons encoding b-oxidation cycle enzymes are also activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) -- cAMP complex and therefore subjected to carbon-catabolite-repression in the presence of glucose11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21832992" target="_blank">21832992</a>]<br></li>
<li>The feaR gene is upregulated by carbon or nitrogen limitation , which we propose reﬂects regulation of feaR by the cyclic AMP receptor protein ( CRP ) and the nitrogen assimilation control protein ( NAC ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>The carbonscavenging regulon includes cyclic AMP ( cAMP ) / cAMP receptor protein ( CRP ) , which mediates the use of alternative carbon sources ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23222720" target="_blank">23222720</a>]<br></li>
<li>Sequence analysis of the DNA fragments ﬂanking the mini-Tn10kan insertion showed that both plasmids contained the same target gene , cra , involved in cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) - independent global modulation of carbon ﬂow ( 22 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371462" target="_blank">9371462</a>]<br></li>
<li>A number of molecules participate in this regulation , including the cyclic AMP receptor protein ( CRP ) , adenylate cyclase , cyclic AMP ( cAMP ) , and EIIA from the phosphoenolpyruvate : glucose phosphotransfer-ase system ( PTS ) ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>In the course of identiﬁcation of regulation targets by cyclic AMP ( cAMP ) receptor protein ( CRP ) , we realized that the cAMP-CRP complex is involved in regulation of some ribosomal protein genes , including the rmf gene ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>For glucose-metabolism , E. coli has developed pleiotropic regulators for the transcriptional regulation , such as cyclic AMP ( cAMP ) and its receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>It was shown recently that the functions of the E. coli lsr operon and its regulators , LsrR and LsrK , are similar to those in S. enterica serovar Typhimurium , and cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) are involved in regulation of the lsr operon ( 50 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In this study , the cyclic AMP ( cAMP ) receptor protein ( CRP ) is shown to be involved in positive regulation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>TCA , tricarboxylic-acid-cycle ; MDH , malate dehydrogenase ; kbp , kilopase-pairs ; CRP , cyclic adenosinemonophosphate receptor protein Offprint requests to : K.-D .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>Here , we show that cspD is positively regulated by cyclic AMP receptor protein ( CRP ) , a global metabolic regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Here , we show that the cyclic AMP receptor protein ( CRP ) , a global regulator involved in sugar metabolism , upregulates cspE in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>This mechanism will activate the cyclic AMP receptor protein ( CRP ) ( 43 ) , a global transcriptional factor that positively regulates most carbon catabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The global regulator cyclic AMP receptor protein ( CRP ) administers additional control over GadX production , tying it to the carbon energy state of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>The known regulators of th regulon are gaIR , galS and Crp , which encode Gal repressor ( GaIR ) , Gal isorepressor ( GalS ) and cyclic AMP receptor protein ( CRP ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Here we identify this crp mutation , which results in a single amino acid substitution ( K52N ) that has been proposed previously to uncover a cryptic activating region in the cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>Here , we show that cyclic AMP receptor protein ( CRP ) upregulates cspD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>KD , apparent KD ; CRP , cyclic AMP receptor protein ; DcuR-P , phosphoryl DcuR ; TBE , Tris/borate/EDTA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073297" target="_blank">15073297</a>]<br></li>
<li>3812 Biochemistry 2003 , 42 , 3812-3825 Role of Protein-Protein Bridging Interactions on Cooperative Assembly of DNA-Bound CRP-CytR-CRP Complex and Regulation of the Escherichia Coli CytR Regulon † Mayy Chahla , ‡ John Wooll , § Thomas M. Laue , § Nghia Nguyen , ‡ and Donald F. Senear * , ‡ Department of Molecular Biology and Biochemistry , UniVersity of California , IrVine , California 92697 , and Department of Biochemistry , UniVersity of New Hampshire , Durham , New Hampshire 03824 ReceiVed NoVember 5 , 2002 ; ReVised Manuscript ReceiVed February 10 , 2003 ABSTRACT : The unlinked operons that comprise the Escherichia coli CytR regulon are controlled coordinately through interactions between two gene regulatory proteins , the cAMP receptor protein ( CRP ) and the cytidine repressor ( CytR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>126 , No. 3 Printed in U.S.A. Mutations Affecting Catabolite Repression of the L-Arabinose Regulon in Escherichia coli B/r LAUREL HEFFERNAN , ' * RICHARD BASS , AND ELLIS ENGLESBERG Section in Biochemistry and Molecular Biology , Department ofBiological Sciences , University ofCalifornia , Santa Barbara , California 93106 Received for publication 2 January 1976 Expression of the L-arabinose regulon in Escherichia coli B/r requires , among other things , cyclic adenosine-3 ' ,5 ' - monophosphate ( cAMP ) and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Volume 10 Number 4 1982 Molecular cloning and nucleotide sequencing of the gene for E. coli cAMP receptor protein Hiroji Aiba , Shinji Fujimoto and Norihito Ozaki Radioisotope Laboratory , Faculty of Medicine , and * Department of Biophysics , Faculty of Science , Kyoto University , Kyoto 606 , Japan Received 23 December 1981 ; Accepted 27 January 1982 ABSTRACT The crp gene of E. coli , which codes for cAMP receptor protein ( CRP ) , has been cloned in the plasmid pBR322 on the basis of a genetic complementation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>Memorial University of Newfoundland , St. John 's , Newfoundland , CANADA AlB 3V6 Tel. 709-737-6885 ( Received May 15th , 1985 ) ( Revision received November 1lth , 1985 ) ( Accepted February 3rd , 1986 ) SUMMARY The speC gene encoding ornithine decarboxylase ( ODC ) in Escherichia coli is negatively regulated by CAMP and the CAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>( 1988 ) 203 , 564-583 Positive and Negative Regulators for Glucitol ( gut ) Operon Expression in Escherichia coli Mamoru Yamadat and Milton H. Saier Jr Department of Biology The John Muir College University of California at San Diego La Jolla , CA 92093 , U.S.A. ( Received 20 August 1987 , and in revised form 16 February 1988 ) Expression of the glucitol ( gut ) operon in Escherichia coli is regulated by an unusual , complex system which consists of an activator ( encoded by the gutM gene ) and a repressor ( encoded by the gutR gene ) in addition to the cAMP-CRP complex ( CRP , CAMP receptor protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3062173" target="_blank">3062173</a>]<br></li>
<li>278 , No. 41 , Issue of October 10 , pp. 39755 -- 39761 , 2003 Printed in U.S.A. Kinetics of Transcription Initiation at lacP1 MULTIPLE ROLES OF CYCLIC AMP RECEPTOR PROTEIN * Received for publication , June 6 , 2003 , and in revised form , July 21 , 2003 Published , JBC Papers in Press , July 23 , 2003 , DOI 10.1074 / jbc.M305995200 Mofang Liu ‡ § , Geeta Gupte ‡ , Siddhartha Roy ‡ ¶ , Rajiv P. Bandwar , Smita S. Patel , and Susan Garges ‡ From the ‡ Laboratory of Molecular Biology , Center for Cancer Research , NCI , National Institutes of Health , Bethesda , Maryland 20892-4264 , the ¶ Department of Biophysics , Bose Institute , P-1 / 12 , C.I.T. Scheme VII M , Calcutta 700 054 , India , and the Department of Biochemistry , Robert Wood Johnson Medical School , Piscataway , New Jersey 08854 The cyclic AMP receptor protein ( CRP ) acts as a transcription activator at many promoters of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Cyclic AMP Receptor Protein-Dependent Activation of the Escherichia coli acsP2 Promoter by a Synergistic Class III Mechanism Christine M. Beatty ,1 Douglas F. Browning ,1,2 Stephen J. W. Busby ,2 and Alan J. Wolfe1 * Department of Microbiology and Immunology , Stritch School of Medicine , Loyola University Chicago , Maywood , Illinois 60153,1 and School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom2 Received 12 February 2003/Accepted 16 June 2003 The cyclic AMP receptor protein ( CRP ) activates transcription of the Escherichia coli acs gene , which encodes an acetate-scavenging enzyme required for ﬁtness during periods of carbon starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>32 , No. 19 doi :10.1093 / nar/gkh908 Identification of the CRP regulon using in vitro and in vivo transcriptional profiling Dongling Zheng , Chrystala Constantinidou * , Jon L. Hobman and Stephen D. Minchin School of Biosciences , University of Birmingham , Birmingham B15 2TT , UK Received July 2 , 2004 ; Revised September 3 , 2004 ; Accepted October 7 , 2004 ABSTRACT The Escherichia coli cyclic AMP receptor protein ( CRP ) is a global regulator that controls transcription initiation from more than 100 promoters by binding to a specific DNA sequence within cognate promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>New targets for the cyclic AMP receptor protein in the Escherichia coli K-12 genom e Kerry Hollands , Stephen J.W. Busby & Georgina S. Lloyd School of Biosciences , University of Birmingham , Birmingham , UK Abstract Correspondence : Stephen Busby , School of Biosciences , University of Birmingham , Recent genomic studies with Escherichia coli K-12 have suggested scores of previously unexplored targets for the cyclic AMP receptor protein ( CRP ) global transcription regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>The CRP protein is another global regulator that , on exhaustion of glucose or its analogues , switches the cellular metabo-lism to the utilization of alternative carbon sources , such as lactose , maltose , and arabinose ( 24 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Their activation depends on the synergistic action of MalT , the activator of the maltose regulon ( Schwartz , 1987 ) , and CRP , a global regulator controlling the utilization of carbon sources ( Kolb et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>Thus , our results argue that CRP , not H-NS , is a master regulator of glutamate-dependent acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Our results suggest for the ﬁrst time that CRP , a global regulator , plays an important role in cold adaptation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>These results demonstrate that propionate-metabolism is subject to catabolite repression by the global transcriptional regulator CRP and that this regulation is effected through control of both the regulator gene prpR and the prpBCDE operon itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Of the two tandem promoters driving the expression of the crp gene coding for the global transcriptional regulator , CRP , only one requires DNA negative supercoiling and is subject to stringent control ( GonzaÂlez-Gil et al. , 1998 ; Johansson et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Cyclic AMP-CRP is a positive or negative regulator of transcription of numerous operons in E. coli ( 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Ohki R , Kawamata T , Katoh Y et al ( 1992 ) Escherichia coli dnaJ deletion mutation results in loss of stability of positive regulator , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21221969" target="_blank">21221969</a>]<br></li>
<li>tractor.lncc.br / ) , making CRP the global regulator of the cell 's response to carbon and energy shortage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>CRP was the main global regulator ( Fig. 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>In this report , we show that transcriptional activation of the E. coli prpBCDE operon and the divergent prpR gene depends on the function of the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_533">
Overall , our ﬁndings show that CRP , a global metabolic regulator in E. coli , activates a gene potentially related to persistence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="rp_similarity_533", type="checkbox">
<div class="RP_similar">
<li>Our ﬁndings show that CRP , a global metabolic regulator in E. coli , activates a gene potentially related to persistence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li></div>
<li>Here , we show that CRP , a global regulator involved in sugar me-tabolism , regulates cspE transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>CRP is a global regulator of catabolite repression in E. coli ( reviewed in reference 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>In E. coli , CRP is the global regulator of catabolite repression and previously was shown to negatively regulate Spot 42 RNA and positively regulate CyaR RNA expression ( De Lay & Gottesman , 2009 , Johansen et al. , 2008 , Polayes et al. , 1988 , Papenfort et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>Mgl-BAC is involved in the galactose-transport , GalR and GalS negatively regulate MglBAC , and GalR and GalS are regulated by the global regulator CRP ( Semsey et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>In contrast to other maltose operon promoters , which are regulated only by MalT : malEp and malKp are also positively controlled by CRP , a pleiotropic regulator involved in the control of carbon source utilization ( De Crombrugghe et al. , 1984 ; Chapon , 1982 ; Richet & Raibaud , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>A balanced aerobic growth on glycerol depends on three global regulators : cAMP-CRP as the principal inducer of the glycerol catabolic regulon ( including glpF , glpK and glpD ) ; Cra ( FruR ) as regulator of some gluconeogenic genes , and ArcA as regulator of several central metabolic genes including the TCA-cycle and others involved in respiration [ 1,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>In summary , we found families involved in the regulation of processes that require unique regulators in bacteria and probably different from those in archaea , such as the global regulator protein , CRP , the positive regulator of the cytochrome c552 , DniR , or the DNA-repair system regulated by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>In contrast , strong responses to anaerobiosis and glucose ( a source of pyruvate ) mediated by global regulators ( ArcA , FNR and CRP ) are likely to be less beneficial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>These seemingly include direct interactions by CRP and FNR to activate transcription initiation and indirect effects by acetate metabolic enzymes and transcription factors that control carbon ﬂux .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Our set of 11 global regulators are transcription factors involved in carbon degradation ( mlc and lrp ) , redox status sensing ( arcA , narL and FNR ) , ion-transport regulation ( fur ) , environmental sensors ( cspA and CRP ) , a regulator of structural proteins ( hns ) and two general enhancers ( IHF and FIS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Thus the global regulators are proteins that control responses to changing food levels and carbon degradation ( mlc , lrp , CRP ) and transcription factors that respond to changes in redox status or ion levels of the cell ( arcA , narL , FNR , fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">CRP binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The HindIII-XbaI fragment carrying the mutated sequence was cloned between the HindIII and XbaI sites of plasmid pPBK500 to create plasmid pPBK500mutCRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>Fusion construct -02 , + NO3 +02 , + NO3 and alleiea Glc Srl Glc Srl WT , fnr ' crp + 580 700 UD UD WT , fnr : : TnlO crp + UD UD UD UD WT , fnr : : TnlO Acrp UD UD UD UD CC , fnr + crp + 36 50 1 6 CC , fnr : : TnlO crp + 2 15 1 5 frr : : TnlO CC , Acrp UD UD UD UD a Carries WT or CC sequence at the FNR1 site ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Helix 8 I N-term Helix 1 Helix 3 Helix 2 Helix 5 Helix 4 Helix 6 Helix 7 6 > 34 1 > 33 1 > 35 7 > 35 4 > 24 34 > 57 Helix 10 HYfB HYfD HYfF Hyf C HYfE FocB P 41 > 61 38 > 53 39 > 58 69 > 96 30 > 50 68 > 89 Helix 11 71 > 99 70 > 98 70 > 98 98 > 122 56 > 75 109 > 133 Helix 12 109 > 132 111 > 132 110 > 133 134 > 157 92 > 119 164 > 184 Helix 13 134 > 155 134 > 157 135 > 158 179 > 203 123 > 149 198 > 222 Helix 14 161 > 185 167 > 195 167 > 194 226 > 254 156 > 176 249 > 273 Helix 15 203 > 233 201 > 226 212 > 240 261 > 283 180 > 200 240 > 268 241 > 263 253 > 280 297 > 315 P P P P C Helix 9 Helix 16 HyfB HyfD Hyff 273 > 301 308 > 328 334 > 361 382 > 405 421 > 452 472 > 501 527 > 555 654 > 672 265 > 293 301 > 318 325 > 353 367 > 390 407 > 435 457 > 479 284 > 312 319 > 332 333 > 356 381 > 407 410 > 443 462 > 486 In a previous report , two putative 07 ' - dependent promoters and potential FNR , CRP and Fur sites were detected upstream of the hyfcoding regions ( Andrews et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>Matrix Consensus ( IUPAC ) order sites Total IC IC per FPR50 % ( LOO ) FPR90 % ( LOO ) FPR100 % ( LOO ) FPR50 % FPR90 % FPR100 % ( matrix sites ) LOO/Matrix sites ( FPR 50 % ) ( matrix sites ) ( matrix sites ) colum n 1 0 1 1 e e e e mem mem mem mem TrpR CRP CysB NanR 24 24 30 20 6 tyGtACtmGykaACtaGTaCratr aaawwtgtGayryagaTCACawww gGAavGrrrtaAyKrmwtrmcarakykm ktATAmMwGktataMmrGww 13.62 7.29 10.67 10.28 0.57 0.30 0.36 0.51 6.67 E 1.38 E 9.29 E 1.53 E + + + + 03 00 03 05 1.30 E-10 6.40 E-242 1900 8.30 E-07 1.00 E 6.20 E 2.60 E 2.30 E -08 -05 -04 -06 7.70 E-06 3.70 E-03 1.40 E-01 1.60 E-03 7.70 E-06 2.80 E-02 1.40 E-01 1.60 E-03 1.50 E 4.50 E 2.80 E 1.50 E -12 -05 -08 -11 1.50 E 2.70 E 3.20 E 6.40 E -09 -03 -05 -07 1.50 E 2.10 E 3.20 E 6.40 E -09 -02 -05 -07 198 8 6 kt 1 1 0 meme meme LexA HipB FNR 21 4 72 22 18 20 wwtrCTGtayatamammCAGya ATCCsskagmGCGGATAA tdywwwtTGaTwwmraTCaa 11.65 10.95 6.94 0.53 0.61 0.35 1.33 E 3.33 E 1.71 E + + + 01 03 0 2.60 E-46 0.00022 2.09 E-71 2.00 E 5.00 E 1.30 E -08 -08 -04 4.60 E-04 4.30 E-06 5.30 E-03 3.10 E-03 4.30 E-06 1.80 E-02 1.50 E 1.50 E 7.60 E -09 -11 -05 1.50 E 3.10 E 2.90 E -05 -09 -03 1.50 E 3.10 E 9.80 E -04 -09 -03 consensus B C Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>!? 4l -80 L -60 -40 -20 t20 t40 t60 mg ( -80 L -60 -40 -20 + I t220 t240 t260 CRP 0 g -- c ( a ) Gene Operator Strand Sequence 0 00 Consensus * GTGKAANC E Top GTGTAAAC Bottom GTGGAATC O go/E OX Top Bottom TOP Bottom 0 , mglB ( b ) Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>568 A 567 - - + -- + 1 23 4 TAAKA -340 Glucose N2 cA `` P - + - + - - + + -260 GATACGCGCTTAATCTCC TCTAACCATATGATTTARAAACA -180 AATCACTTCGCACGGOCAGGTTTTGOCACC CAGTAAAACTA TTCATTCGCG -100 P2 - P1 ¬ AIITCTIrTTrIATAAAAICTCGIAT ~ rTMTGAIVATCR ) IMYATTXS-A g A ATTGCTACTAAAATATTATGA I ----------- i 1 CAMP-CRP ATTGAAATGGTGTCYCTTTAI'GGCAATAACGATTACAGGCGCGACGATTACTCGTCACGCTAACAGCCCTTTTTGCAG KetAlaIleAsnAsnThrGlySerArgArgLouLauValThrL * uThrAlaLouPhaAla -20 P2 w ¬ 61 B P1 s v ESVD III pUCGCDI pGCDAPI pGCDAP3 SOO bp FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419307" target="_blank">8419307</a>]<br></li>
<li>bp , Base pairs CACATCOGAACTG CAGGCTTCAAAGCOGCOGTCAATGCTATAGACCACATCGTAGCC 75 CTGTTG ICAG IAITIOOCGO cTTTGCTGCTATGCTAAAACAACCOSTTAA 150 ,225 AGTA 300 ACCGGTC9CG & TATCB&CC & 0 & 3Sa 51X 1 TGCS AsuIl 375 CRP -35 Binfl & T T TA-TA - C TAC6-M - & U T 1 T AI A 2TG TT-A-A-AG-T-T-A -- T !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>establishes A mt1A mt1D 1 164bp mtlR 1914 bp 588 bp I 3 rk1 r -35 / -10 k , II ken r pML 1 B v CRP 1 TGACGAAGG AAT TA CAGGTGAACGCCGTAAGAA 1 CRP2 AAT71TCTTGTGATTCAGATCACASTATTCAACAAACCATCAAAACA7O I CRP3 ATUTGXCA ; CAUA ± T kAAiTGCCATTTTTAGCGAAAATCGCCGCCT I CRPA4 TGTTGCTTTTTTACACAAGCqTTTTGTGATG : AACGTCACG ^ CAATTACCT CRP5 -35 CTCTACCCCCTATATTATITUA ` l' ` l'bT ` lAT ` AUAGAgAAGGCCGTCGACT -10 GGACAGTTAACCGATTCAGTGCCAGATTTCGCAGTATCTACAAGGTCCGG MtlR1 box CTACCTCTGCCGCCACATTAACAAAAAACCTTCGG CGTCCtCTGCGCG MtlR2 box ACAGCAAACATAAGtAGGGGTGTTTTTATGTCATCCq ^ TATTAAGATCAA pMLI mtlA 51 101 151 201 251 301 351 FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300537" target="_blank">8300537</a>]<br></li>
<li>Identification of binding sites forCRP in the glpFK control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>An important requirement is that at least one CRP-operator has to be situated at a distance not exceeding 5 nucleotides to the corresponding CytRoperator [ 20 ] , with the position of the OCYTR operator being not symmetric relative to the flanking OCRP operators [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Both CRP and FNR are 50-kDa homodimers that bind related inverted-repeats with a TGANNNNNNTCA core motif ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>634 r Mini-F transcriptional lacZ fusion vector , Cb Gene replacement vector , Cmr Cloning vector , Tcr Kmr Cloning vector , Cbr 3.5-kb subclone of the clyA locus in pUC18 1.6-kb subclone of the clyA locus in pUC18 Same as plasmid pYMZ80 , but with the 1.6-kb clyA locus in the opposite orientation Transcriptional clyA : : lacZ fusion in pFZY1 clyA : : luxAB in pCH257 pYMZ81 with altered CRP binding site : 59-TTGTTTGATATAGATCACATTT-3 pYMZ81 with altered CRP binding site : 59-AAATGTGATCTAGATCACATTT-39 pYMZ81 with altered CRP binding site : 59-TTGTGTGATATTTATCATATTA-39 pYMZ81 with altered CRP binding site : 59-TTGTTTAATATTTATCATATTA-39 pYMZ81 with altered CRP binding site : 59-TTGTCTAATATTTATCATATTA-39 pYMZ81 with altered clyA 210 sequence : TATGAAT3CACGAAC Subclone of the clyA locus from position 2349 to 250 in pGEM-T Easy Subclone of the clyA locus from position 2250 to 150 in pGEM-T Easy Subclone of the clyA locus from position 2150 to 1150 in pGEM-T Easy Subclone of the clyA locus from position 250 to 1250 in pGEM-T Easy Subclone of the clyA locus from position 150 to 1349 in pGEM-T Easy Subclone of the clyA locus from position 1150 to 1449 in pGEM-T Easy pJON78 with altered CRP binding site : 59-AAATGTGATCTAGATCACATTT-39 2.1-kb subclone of the slyAEC locus in pACYC177 3.5-kb subclone of the clyA locus in pKO3 slyA in-frame deletion in pKO3 crp gene in pLG339 , formerly known as pLG339/CRP fnr gene in pBR322 FNR-V208R ; S212G ; G216K FNR-V208R ; E209D 31 54 7 This laboratory This laboratory This laboratory MATERIALS AND METHODS Bacterial strains and culture media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Plasmids used in this work Plasmid Relevant characteristics Reference or source r Suicide vector , Cm pCH257 pFZY1 pKO3 pLG339 pSL1180 pYMZ62 pYMZ80 pYMZ81 17 28 r Mini-F transcriptional lacZ fusion vector , Cb Gene replacement vector , Cmr Cloning vector , Tcr Kmr Cloning vector , Cbr 3.5-kb subclone of the clyA locus in pUC18 1.6-kb subclone of the clyA locus in pUC18 Same as plasmid pYMZ80 , but with the 1.6-kb clyA locus in the opposite orientation Transcriptional clyA : : lacZ fusion in pFZY1 clyA : : luxAB in pCH257 pYMZ81 with altered CRP binding site : 59-TTGTTTGATATAGATCACATTT-3 pYMZ81 with altered CRP binding site : 59-AAATGTGATCTAGATCACATTT-39 pYMZ81 with altered CRP binding site : 59-TTGTGTGATATTTATCATATTA-39 pYMZ81 with altered CRP binding site : 59-TTGTTTAATATTTATCATATTA-39 pYMZ81 with altered CRP binding site : 59-TTGTCTAATATTTATCATATTA-39 pYMZ81 with altered clyA 210 sequence : TATGAAT3CACGAAC Subclone of the clyA locus from position 2349 to 250 in pGEM-T Easy Subclone of the clyA locus from position 2250 to 150 in pGEM-T Easy Subclone of the clyA locus from position 2150 to 1150 in pGEM-T Easy Subclone of the clyA locus from position 250 to 1250 in pGEM-T Easy Subclone of the clyA locus from position 150 to 1349 in pGEM-T Easy Subclone of the clyA locus from position 1150 to 1449 in pGEM-T Easy pJON78 with altered CRP binding site : 59-AAATGTGATCTAGATCACATTT-39 2.1-kb subclone of the slyAEC locus in pACYC177 3.5-kb subclone of the clyA locus in pKO3 slyA in-frame deletion in pKO3 crp gene in pLG339 , formerly known as pLG339/CRP fnr gene in pBR322 FNR-V208R ; S212G ; G216K FNR-V208R ; E209D 31 54 7 This laboratory This laboratory This laboratory pYMZ83 pMWK4 pMWK9 This work This work This work pMWK10 This work pMWK24 This work pMWK28 This work This work pMWK29 This work pMWK31 This work pMWK32 This work pMWK33 This work pMWK34 This work pMWK35 This work pMWK36 This work pMWK37 This work pMWK45 pJON22 This laboratory This laboratory This work 2 , 60 pJON78 pMOJ2 pDW300 50 53 52 pGS24 pGS215 pGS297 Gel mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>This argues strongly that the role of MelR bound at Site 1 and Site 10 , and of CRP bound between Site 1 and Site 2 , is primarily to ensure occupancy of Site 20 by MelR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>With the JK19 fragment , the changes in Site 20 result in a promoter where the requirement for MelR bound at Site 1 and Site 10 , and for CRP bound between Site 1 and Site 2 , is short circuited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>'' Lanes 1 -- 4 , the sequences of the lacP1 promoter ; lanes 5 , 8 , 11 , 14 , and 17 , KMnO treatment in the absence of RNAP ; lanes 6 , 9 , 12 , 15 , and 18 , 4 KMnO treatment in the presence of RNAP ; lanes 7 , 10 , 13 , 16 , and 4 19 , KMnO4 treatment in the presence of RNAP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>-9.6 ( 0.5 -9.7 ( 0.2 -11.5 ( 0.2 -12.8 ( 0.1 -13.3 ( 0.1 -13.7 ( 0.2 -14.1 ( 0.1 -13.2 ( 0.3 -13.4 ( 0.4 -13.3 ( 0.3 -12.2 ( 0.3 -12.1 ( 0.0 -14.3 ( 0.3 -14.5 ( 0.03 Table 2 : Apparent Cooperativity As Assessed by Differences in Indivia dual Site Loading Free Energy Changes Described in Table 1 a Table 3 : Configurations and Free Energy States for CRP and CytR Binding to the cdd Promoter binding sites a CRP1 free energy contributions free energy state b CRP2 CytR at CRP1 CytO CytR at CRP2 O O O 1 O O reference state ΔGs ,1 2 O O O CRP O ΔG1 ΔGs ,2 3 CRP O O O O ΔG2 ΔGs ,3 4 O O CytR O O ΔG3 ΔGs ,4 5 O O O O CytR ΔG4 ΔGs ,5 6 O CytR O O O ΔG5 ΔGs ,6 7 CRP O O CRP O ΔG1 þ ΔG2 ΔGs ,7 8 O O CytR CRP O ΔG1 þ ΔG3 þ ΔG13 ΔGs ,8 9 O CytR O CRP O ΔG1 þ ΔG5 ΔGs ,9 10 CRP O CytR O O ΔG2 þ ΔG3 þ ΔG23 ΔGs ,10 11 CRP O O O CytR ΔG2 þ ΔG4 ΔGs ,11 12 O O CytR O CytR ΔG3 þ ΔG4 ΔGs ,12 13 O CytR CytR O O ΔG3 þ ΔG5 ΔGs ,13 14 O CytR O O CytR ΔG4 þ ΔG5 ΔGs ,14 15 CRP O CytR CRP O ΔG1 þ ΔG2 þ ΔG3 þ ΔG123 ΔGs ,15 16 CRP O CytR O CytR ΔG2 þ ΔG3 þ ΔG4 þ ΔG23 ΔGs ,16 17 O CytR CytR O CytR ΔG3 þ ΔG4 þ ΔG5 ΔGs ,17 18 O CytR CytR CRP O ΔG1 þ ΔG3 þ ΔG5 þ ΔG13 ΔGs ,18 a Binding sites are listed in columns 2-6 corresponding to the order , from promoter distal to promoter proximal , on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>DNase I footprinting ( Fig. 8 ) revealed that this region ( CRP1 ) and a region immediately downstream ( CRP2 ) were protected by 50 n ~ CRP ( nucleotides 60 to 100 ; Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>FNR and CRP are ~ 50 kDa proteins that recognize related inverted-repeat sequences ( Scott et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>On the contrary , in vitro and in vivo studies with FLP have shown that it recognizes sequences conforming to a CCTGA N4 TCAGG consensus , which is related to but not congruent with either of the FNR or CRP consensus sequences ( Gostick et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Promoter fragment pacs155 was constructed by digestion of pCB63 ( Browning et al. , 2004 ) , followed by religation to form a promoter fragment that lacked sequences upstream of CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In the other , the putative half UP-element present in the wild-type promoter was optimized and completed to a full UP-element ( which has a sequence identical to a full UP-element that is known to function with Es and a CRP box located at 269.5 ; Law et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>FNR is structurally related to CRP and , like CRP , binds to a 22bp sequence that accommodates two FNR subunits ( Green et al. , 1991 ; Sharrocks et aL , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>LuxS was previously shown by Wang et al. ( 65 ) to be negatively regulated by crp , but it lacked a sequence upstream of the coding sequence for luxS that was able to bind CRP in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18978044" target="_blank">18978044</a>]<br></li>
<li>The role of IHF has remained less clear : while in vitro it is known to bind simultaneously with CRP and to cause inhibition of acsP2 transcription , in vivo the upstream sequence containing the three IHF sites has a positive effect on transcription ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>To ﬁnd out whether coregulation by S-containing RNA polymerase and cAMP-CRP is a more general phenomenon , the upstream regions ( 200 bp ) of all 481 positively S-controlled genes identiﬁed here were screened for putative cAMP-CRP boxes , de-ﬁned as TGTGA ( N6 ) TCACA , with a maximum of three mismatches allowed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>This indicates that the minimal csiE promoter consisting of just the -35 and - 10 regions can be recognized by os-containing RNA polymerase holoenzyme , whereas transcriptional initiation by 070-containingRNA polymerase requires activation directly or indirectly by cAMP-CRP mediated by a cis-acting sequence element situated at least partially upstream of position -38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Alternatively , Shanblatt & Revzin ( 1983 ) have recently proposed that RNA polymerase bound at P induces the fixation of a second CRP molecule , which binds to the sequence upstream from - 5 0 and is responsible for the protection as far as - 7 0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>Together with CRP , puriﬁed RNA polymerase gives a clear footprint that runs from position 115 to upstream of +1 +10 +20 CRP bound at Site 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>Class i and II of activators con-espond to activators which interact with direct repeat sequences or with palindromic sequences , respectively B REPRES5ORS and ACTIVATORS ( n ) ACTIVATORSd ) Positive regulation by the Cl protein ofthe bacteriophage \ and by the CRP protein of # coli are often considered as the paradigms of positive regulation in prokaryotes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2664421" target="_blank">2664421</a>]<br></li>
<li>( B ) The sequences represent those protected by cAMP-CRP within the ascG-ascF and prpR-prpB spacer regions ( see Fig. 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Therefore , it was decided to investigate whether sequences upstream of glnAp2 are essential for CRP -- cAMP-mediated inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>These results indicate that upstream sequences of glnAp2 are not essential for CRP -- cAMP-mediated inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>These sequences are recognized by at least three regulators , Gal repressor ( GaIR ) , Gal isorepressor ( GalS ) and cAMP receptor protein ( CRP ) , which modulate transcription from these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>A total of about 70 promoters carry two CRP box sequences , some of which have been experimentally identiﬁed to bind two molecules of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Specific sequences to which the CRP and FNR proteins bind , to stimulate transcription , have been proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>CRP - , CaiF - and RNAP-protected sequences of the ® x regulatory region are boxed or circled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Non-template strand sequences of promoters repressed by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Regions protected by the CRP are marked , and the sequences are numbered ar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>It has been shown recently that when CytR and CRP are used together to select DNA sequences that are preferred for formation of the three-protein complex , the sequences most commonly selected are CytR recognition motifs separated by 10 -- 13 base pairs and almost centered between CRP1 and CRP2 ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>For screening of DNA sequences with affinity to CRP , purified His-tagged CRP was mixed with a collection of E. coli genome fragments of 200 -- 300 bp in length , and CRP-bound DNA fragments were affinity-purified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>For identification of the sequences of DNA fragments bound to CRP , we first performed SELEX-clos ( cloning-sequencing ) analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>This implies that CytR and CRP recognize very similar DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>On the basis of these results , we first proposed a simple competition model for the action of CytR in which the repressor and the cAMP-CRP complex recognize very similar or overlapping DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>© 2003 Blackwell Publishing Lt Results FIS binds the acs regulatory region at multiple locations Beatty et al. ( 2003 ) showed that acs transcription from the major promoter acsP2 depends upon CRP and that CRP binds to tandem sites centred at positions -69.5 and -122.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Discussion Beatty et al. ( 2003 ) showed that CRP binds tandem sites located upstream of the major promoter acsP2 to activate its transcription synergistically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Most notably , in E.coli CRP binds to and stimulates transcription at novel CRP sites with a distinct consensus ( CRP-S ) that makes transcription activation dependent on an additional protein factor , Sxy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In promoters where CRP acts as a sole transcription activator or synergistic activator , it binds to highaffinity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Transcription from the melR promoter is dependent on CAMP-CRP : specific nucleotide sequences downstream of bp -59 with respect to the me & transcription start are sufficient for full promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Predicted FNR and cyclic AMP receptor protein ( CRP ) sites at suitable distances from putative promoter sequences are located immediately upstream of aspA and dcuB ( 36 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In all cases , these studies have implicated interactions of CRP and CytR with sequences located in the 80 -- 100 bp immediately upstream of the various transcription start sites and interactions between the proteins as the basis for positive and negative gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However , at the metK , ybiS and yjcB predicted promoter regions , the CRP sites are located 3 bp upstream of the 35 hexamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>We then cloned sequences of about 500 bp in length upstream from the initiation codons of the ascF , prpR , and prpB genes ( see Fig. 3 and 8 for the gene organization ) , each including transcription initiation sites and recognition sequences for a transcription factor , i.e. , AscG ( ascF ) , CRP ( ascF and prpB ) , or PrpR ( prpB ) , and inserted them into pGRP to construct the promoter assay vectors pGRK259 ( pGRPascF ) , pGRK559 ( pGRPascG ) , and pGRN024 ( pGRPprpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>A CRP sitea Sequence Consensus galE T C C fl T G T T G C T T G T G A T C G A T T G T G A T G A T T gals mglB B operator sites Gene Opetjtor Stiand galB ga2P gals Bottom To ; Botcom Top aoCCOm wglB Similarities among ga/regulon promoters Top T G T A A A C The promoter regions of the gal and mgl operons , and of the galP , galU , gaIR and galS genes have been sequenoed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>The ﬁgure highlights the locations of the divergent malI and malX translation start codons ( bold underlined ) , the two DNA sites for CRP ( doubly underlined ) and the proposed malX promoter 10 hexamer element ( boxed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>Searching the reverse DNA strand or for symmetrical motifs with CONSENSUS and Gibbs did not identify any additional high-conﬁdence sites To score putative CRP sites in the Pasteurellaceae , three weight matrices were generated as described previously ( 13,14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>When activated by cAMP , CRP recognizes speciﬁc DNA sites via the C-terminal F-helix ( αF ) which forms a typical helix -- turn -- helix ( HTH ) motif , together with the neighboring E-helix ( αE ) ( Fig. 1 , 1G6N ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19439203" target="_blank">19439203</a>]<br></li>
<li>CRP binds as a dimer to speciﬁc DNA sites and induces a DNA bend of approximately 90 degrees in the presence of its ligand , cAMP ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_59">
The active cAMP-CRP complex binds to speciﬁc DNA sites located at or upstream of CRP-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></label>
<input id="sit_similarity_59", type="checkbox">
<div class="SIT_similar">
<li>As a consequence , the active cAMP-CRP complex binds to speciﬁc DNA sites located at or upstream of CRP-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li></div>
<li>We carried out DNase I protection experiments with a wild-type malEpmalKp DNA fragment to examine whether CRP binds to these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>On the basis of these observations , we propose that P2 is a synergistic class III promoter that consists of tandem DNA sites for CRP , each lo cated at a class I position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Two CRP homodimers bind in tandem at DNA sites centred at the Class I positions -69.5 and -122.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>AraC , one of the members in Group 2 , is suggested to bind to at least two sites around the promoter region and to form a double stranded DNA loop of more than 200 bp in the absence of arabinose , cAMP or CRP , while the loop could open in the presence of both arabinose and cAMP-CRP ( Dunn et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>Homodimeric CRP forms a complex with cAMP , which is able to bind DNA at specific sites near the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li>
<li>However , comparison of the deo sites with the other known CRP sites ( Figure 5 ) suggests that the cAMP-CRP complex recognizes not only the consensus described by Ebright but two specific 8 bp regions separated by a region of 6 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>To test whether a cAMP-CRP complex might regulate the operon transcription by direct interaction with its putative recognition sites in the regulatory region of dad , we isolated 309-bpAvaII-SspI and 152-bp SspI-BstEII fragments of the dad regulatory region containing each of the two identified sites separately .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>These contained synthetic consensus CRP sites inserted into deoP2 promoters in which CRP-2 and various parts of the spacer region had been deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>On the other hand , CytR regulation is dependent on CRP-1 and CRP-2 and on the length and the sequence of the region between the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Comparison of the spacer regions separating the two CRP sites in deoP2 and tsxP2 does not , however , reveal any clearly homologous sequence that could be recognized by CytR , This does not , of course , rule out the possibility that CytR interacts directly with a sequence in the spacer region ( see below ) and this is presently being tested by engineering point mutations here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Consistent with this proposal , we showed that CRP binds two sites , CRP I and CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Upstream of acsP2 , CRP binds to two sites ( CRP I and CRP II ) , each located in class I positions ( Fig. 11B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>cAfVlP-CRP binds to two sites in the rot regulatoiy region 1 2 3 4 5 6 7 8 1314 9 10 11 12 B 1 2 3 4 5 6 7 -120 • -100 • -80 • PI +1 ' Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>CRP binds at two sites , centered at positions 112.5 and 83.5 , in the cspD upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In line with this , results from EMSAs revealed that CRP binds at two sites in the cspD upstream region spanning positions 151 to 60 ( Fig. 2 ) , which overlaps with the positions of the putative target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Gel shift assays and footprinting experiments suggest that CaiF and CRP bind co-operatively to the F1 / CRP2 and F2/CRP3 sites of the intergenic cai ± ® x region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>In contrast to cAMP-CRP , Lrp did not bind to a single region , but protected at multiple sites extending at least between positions ÿ102 and ÿ14 ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>FIS and CRP bind overlapping sites within the acs regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>CRP binds speciﬁcally to two sites within the acs regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Activated CRP ( i.e. cAMP-CRP , herein called CRP ) binds two sites within the acs regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The CytR operator in udpPEc was located between the two CRP sites and consisted of an imperfect inverted-repeat ( Fig. 3 ) [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The upstream region of the xylKL operon of S. typhi contains the CRP and XylR sites with an unusually long spacer of 21 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The center-to-center distance of 53 bp between the two CRP sites is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>The horizontal shaded boxes denote 22 bp DNA sites for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>A third allele ( dup262 ) consisted of a duplication of the entire udpP region , and a fourth ( dup361 ) contained a partial duplication of CRP2 that also increased the spacing between the CRP sites by 8 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>We note , however , that the spacing between these pentamers is 5 bp in udpP , versus 3 bp in cddP and 2 bp in deoP2 , and that the pentamers are positioned more asymmetrically with respect to the two CRP sites in udpP than in deoP2 and cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>at rhaBAD are separated by 3 bp , while the CRP and RhaR sites at rhaSR are separated by 21 bp ( 12 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>All of these promoters contain tandem CRP sites , at least one of which is in a proximal position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Notably , such a spacing between tandem CRP sites is a signature for CytR / CRP-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>CRP and CytR designated filled sites according to the transcription factor bound ; O designates empty sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CAMP-CRP is indicated by two overlapping circles bound at the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>As mentioned previously , an alternative model is that Fis and CRP bound to their respective sites must ﬁrst interact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>These observations may reflect a weak cooperative interaction between CRP dimers bound at sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Consensus and potential CRP recognition sites are shown in capital letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>These results show that the CRP regulons are subdivided by CRP-S and CRP-N sites in all sequenced Pasteurellaceae genomes Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Here we show that , like MelR303 , when MelR173 binds to sites 1 and 2 it recruits CRP to bind between these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>( A ) Alignment of CRP sites ; arrows highlight positions 6 and 17 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Evidence for functional CRP sites at rhaSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>It remains to be established which ones of these candidate sites actually bind CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553671" target="_blank">2553671</a>]<br></li>
<li>Thus FNR could be converted such as to bind specifically to CRP sites , CRP to bind to FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>In particular , the four sites exhibited enhanced reactivity in the TGTGA motif , indicating an enlargment of the minor groove due to the bending of the DNA by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>We chose to introduce a triple mutation in the most conserved motif , 5 ' - TGTGA , to ensure that the CRP sites would be completely destroyed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>Similarly , a ﬂexible linker allows 34,35 CytR , a 2-fold symmetry dimer , to recognize both directly or inversely repeated half-operator sites in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491603" target="_blank">15491603</a>]<br></li>
<li>Presumably , either the spacing between the CytR and CRP sites or something inherent in the CytR site , such as the spacing between recognition motifs , is responsible for controlling whether interactions between CytR and CRP are still possible when cytidine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Interaction of CytR with these sites provides a complex pattern of both competitive and cooperative interactions between CytR and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP sites ( CRP1 and CRP2 ) and the CytR operator ( CytO ) are delineated by open boxes ; CytR recognition motifs are shaded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>DNAase I , DMS , and Hydroxyl Radical Footprints of the Binding of CytR-wt , CytRA9-48 , and CAMP-CRP to deof2 ( A ) The `` P-end-labeled fragment used is a 250 bp Xbal-Pvull fragment isolated from ~ 13-152 ( Rasmussen et al. , 1993 ) containing deoP2 sequences framm-116 to + I8 and labeled at the upper strand in the Xbal site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The CRP dimer is known to bind to a well-defined 22 bp palindrome the DNA ( Barber on & Zhurkin , 1990 ; see also Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The 311 bp HindIII-EcoRI DNA fragment , at a ﬁnal concentration of 2 nM , were incubated ﬁrst with Mlc for 5 min and then with CRP for 5 min at 37 8C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10469172" target="_blank">10469172</a>]<br></li>
<li>These three bands presumably correspond to the 187 bp DNA fragment with 1,2 and 3 CRP dimers bound , respectively ( Ramseier e t a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Thus , Characterization of CRP 3 as shown in Figs l ( d ) and 5 ( a ) , treatment of the 276 bp DNA fragment with EcoRV gave a 168 bp DNA fragment In order to unequivocally establish that CRP binds to and a 108 bp DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>CytR binds to its cognate promoters in concert with cAMP-CRP in a highly co-operative fashion torming repression complex that generally consists of two mol-ecules of cAMP-CRP separated by 52 or 53 bp and CytR ioound in between ( Sogaard-Andersen et ai , 1991b ; Gerlach etai , 1991 ; Holsi etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>In this complex , cAMP-CRP interacts with tandem CRP targets and CytR interacts with sequences located between these targets ( Fig. 1 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In the first model , cAMP-CRP still binds to CRP-1 and CytR occupies the sequences on both sides of CRP-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>DMS footprint of cAMP-CRP and CytR bound to nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>CytR and cAMP-CRP bind co-operatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>As the pattern of the 50 bp combined footprint corresponds to the sum of the independent footprints , this complex most likely consists of cAMP-CRP bound to CRP2 , and CytR to the CytR target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to a cddP derivative containing 2 bp inserted between CRP-1 and CRP-2 / 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the presence of CytR , however , the two regulators bind cooperatively to cddP forming a nucleoprotein complex in which cAMP - CRP binds to CRP-1 and CRP-3 and CytR occupies the sequence between these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Speciﬁcally , CytR has a relaxed spacing requirement between operator half-sites and can , in concert with cAMP -- CRP , repress transcription in vivo from octamer boxes separated with centre distances up to three helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The hydroxyl radical footprint of CRP bound to the intergenic region consisted of a clearly-defined series of protected regions extending over about 130 bp ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>CRP bound and protected the DNA region between 275 and 249 bp from DNase I cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Model building has suggested that the extended protected region of CRP bound to a single site is a consequence of the bending of the DNA toward the protein ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>At present , we can not decide whether cAMP-CRP , Lrp , and IHF directly bind to the osmY regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Furthermore , CytR binds at the deoP2 regulatory region co-operatively with cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Sequence-specific binding of cAMP-CRP to a 243-base-pair restriction fragment from the region upstream from glgC was observed by virtue of the altered electrophoretic mobility of the bound DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>In the absence of cAMP-CRP polymerase binds mainly upstream of the -35 region In the vicinity of the Rsal and Sau3A restriction sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>( ii ) cAMP-CRP selectively binds to a 243-bp restriction fragment from the region immediately upstream from gIgC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>cAMP-CRP binds to an upstream region of the lsr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Localization of a region upstream from glgC which binds to cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>We show that the cAMP-CRP activator complex recognizes two sites in tsx-p2 that are separated by 33 bp : a high-affinity site ( CRP-1 ) overlaps the -35 region , and a low-affinity site ( CRP-2 ) is centered around position -74 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In combination , the cAMP-CRP and CytR proteins bind cooperatively to tsx-p2 , and the nucleoprotein complex formed covers a region of 78 bp extending from the CRP-2 site close to the -10 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the galPi promoter , this CRP site is centered at -41.5 , while for galS it is centered at -42.5 , but the spacing between the -10 region and the center of the CRP site is 29 bp in each case , a distance optimal for cAMP-CRP-activated promoters in E. coli ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Our results with the tsx-p2 promoter demonstrate that the tandem targets for the cAMP-CRP activator complex are only separated by 33 bp and that the CytR repressor binds independently to a DNA segment that includes the sequences between the CRP targets and overlaps with the CRP-1 site ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Gel mobility shift assay results ( Fig. 4B ) demonstrated that cAMP-CRP binds to a 46-bp DNA fragment in the intergenic region containing this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>We conclude that positive regulation of the araC gene by the cAMP-CRP complex occurs when the complex binds to a region of DNA 38 to 71 bp upstream from its transcriptional initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>For example , in the DNase I footprints , the C1 sequences ( C13-20 and C13-40 ) exhibit a 3 bp unprotected region in the middle of the CytR operator , and C09-20 ( R -- 1 -- R ) and C13-26 ( L -- 1 -- L ) contain unprotected regions to the left or right of the operator , regardless of the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The ACRP mutation deletes 3 bp of DNA in a region that binds the cAMP-CRP complex in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>The presence of both CytR and cAMP-CRP results in protection of a region of approximately 80 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Co-operative binding of cAMP-CRP and CytR Next we tested binding of CytR and cAMP-CRP to a 160 bp fragment containing nupG sequences from - 1 1 8 to - 1 0 in the DNase I footprinting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Addition of CytR protein to binding reactions containing cAMP-CRP resulted in protection of an 84 bp region spanning from CRP-1 to around position - 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Addition of the CytR extract to binding reactions containing cAMP-CRP results in protection of an 80 bp region spanning from CRP-1 to CRP-2 ( lanes 7-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>However , the location and spacing of two CRP targets within the udpP promoter region were conserved : they were separated by 36 bp and the distance between the CRP1 target and the − 10 region in all four cases was identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>In the absence of cAMP -- CRP , the CytR repressor binds two such octamer motifs in inverted or direct repeat arrangement , preferably separated by 2 bp ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>( A ) In the absence of cAMP -- CRP , the CytR repressor preferentially binds octamer repeats in direct or inverted orientation , separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Thus , cAMP -- CRP preferentially stabilizes a set of CytR conformations that ﬁt operators composed of inverted-repeats with wide spacing ( 10 -- 13 bp ) , or direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the presence of cAMP -- CRP , CytR recognizes ( B ) inverted octamer repeats separated by 10 -- 13 bp or ( C ) direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The latter was described as an inverted pentameric repeat TGCAA-N2 -- 3-TTGCA [ 36 ] , ( where N denoted the number of nucleotides ) , a palindrome TTGCAA [ 38 ] , or a pair of inverted octameric repeats ( either 59-AATGYCAAC-GC-GTTGCATT-39 or 59-AYGTGCAAC-Nx-GTTRCATT-39 , where Y = T or C , R = A or G , and x = 10 , 11 , 12 or 13 ) which are the optimal CytR-binding sites in the absence of CRP , or direct repeats of octamers in either orientation with a 1 bp spacer [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>In the absence of CRP , the optimal binding site for CytR consists of two octamer repeats , in direct or inverted orientation , and separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The study revealed that CytR , when co-binding with two molecules of CRP , preferentially recognizes inverted-repeats separated by 10 -- 13 bp , or direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>However , when co-binding with cAMP -- CRP , CytR instead recognizes inverted-repeats separated by 10 -- 13 bp , or direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>When co-binding with tandem cAMP -- CRP complexes , CytR preferentially recognizes inverted octamer repeats separated by 10 -- 13 bp or direct octamer repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Finally , CRP 3 and CRP 4 are separated from each other by 73 bp , or about seven helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The CytR operator of the cdd promoter contains R -- 1 -- R direct repeats , and the repeats are separated by 5 and 14 bp , respectively , from the ﬂanking CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The apparent explanation for our observation is that CRP makes a direct interaction with MelR bound to Site 1 and Site 2 and this recruits the CRP dimer to the 20 bp gap between the two bound MelR subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The CytR/cAMP-CRP repression complex in rot is similar in structure to those formed at deoP2 and cddP and consists of two cAMP-CRP molecules separated by 53 bp , with CytR bound in between .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The majority of the nucleoprotein complexes consist of a single CytR molecule bound between two CRP molecules separated by 52 or 53 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>plasmid 14 51 pAM1956 pEUcspEgfp312 4.2-kb EcoRI fragment in pEU720 replaced with an 1-kb EcoRI cspE-gfp312 fragment comprising the 306 to 6 cspE region fused to translatable gfp ( see Materials and Methods ) 4.2-kb EcoRI fragment in pEU720 replaced with an 1-kb EcoRI cspE-gfp380 fragment comprising the 306 to 72 cspE region fused to the gfp ORF ( see Materials and Methods ) Same as pEUcspEgfp312 , except ﬁrst core motif ( 69 to 65 region ) was deleted from CRP target site in the cspE upstream region ( see Materials and Methods ) Same as pEUcspEgfp312 but with cspE having three point mutations in the ﬁrst core motif GC ( 68 ) , GC ( 66 ) , and AC ( 65 ) of the CRP target site ( see Materials and Methods ) colE1 origin ; Cmr ; 5.24 kb ; lacIq Cmr ; lacIq ; pCA24N PT5-lac : : crp colEI origin ; Amr ; wild-type crp gene ( and derivatives ) cloned into pBR322 with crp coding sequence on HindIII-EcoRI fragment colEI origin ; Amr ; pDCRP with HindIII-EcoRI fragment carrying crp replaced by M13mp8 polylinker pSC101 origin ; Kanr , Tcs ; 6.2 kb ; low-copy-no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The CRP boxconsensus sequence , 59-TGTGA-N6-TCACA-39 , agrees well with the hitherto identified [ 38 ] , but indicates clearly that G at position 4 and C at position 13 of the CRP-box are highly conserved in good agreement of their key roles to exhibit the high affinity to CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>CRP recognizes a 22-bp target sequence and , at many promoters , activates transcription by making one or more direct contacts with RNA polymerase ( Busby & Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>The consensus DNA target for CRP binding is the 22 bp inverted-repeat 50-AAATGTGATCTAGATCACATTT-30 and , at most previously studied targets , CRP activates transcription by making one or more direct contacts with RNA polymerase ( Belyaeva et al. , 1998 ; Busby & Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>Alternatively , since the strong CRP2 target is expected to span ~ 30 bp ( Liu-Johnson et al. , 1986 ) , the sequence at the edges of the CytR operator may optimize both cAMP -- CRP and CytR interactions with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>When bound with cAMP , CRP is converted into a functional form and recognizes a spe-ciﬁc DNA sequence , referred to CRP box , with the consensus sequence of a 22-bp inverted-repeat sequence , 50-AAATGTGATCTAGATCACATTT-30 [ underlines are highly conserved ] ( De Crombrugghe et al. , 1984 ; Berg & von Hippel , 1988 ; Gunaskerra et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In addition to the sequence of the intervening DNA , an obvious difference between these situations is that at proP the Fis protein is bound at 241 instead of a second CRP molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>To attempt to show that CAMP-CRP is bound to this sequence we performed gel binding assays with purified CRP and labelled DNA fragments from pKK60D75 , exactly as described by Kolb et al. ( 1983 , a and b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Similar inverted-repeats tiave been found in the nucleotide sequences near the fnr promoter and at a number of FNR-dependent promoters ( Shaw and -73 -60 -1,0 -20 H p nir B CATTAA6GAGTATATAAAGGTGAATTTGATTTACATCAATAAGCGGQGTTGCT6AATCQTTAAGGTAGQCGGTA -73 p gat PI TTCTTGTGTAAACGATTCCACTAATTTATTCCATGTCAC ACTTTTCGC ATCTTTTTTArGCTATGGTTATTTCA • • X - ^ CRP consensus AA-TGTGA TCACA-TT - V. ^ ̂ -30 -27 +1 ,1,5 Q ^ , TATGGT , TATTTCA -- GCCAAGCTT Bam HI ' Sphl t W/nd 111 p nir B - gat Pt - p nir B - p gat PI ¬ Fig , 3 , A. Comparison of the n/rS promoter nucleotide sequence from -73 to +1 with the sequence of ga/PJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>For instance , the most abundant 59 clones carried the intergenic sequence between the divergently transcribed yecJ and yecR , of which one is counted as a CRP target of sites would have to be sequenced in order to identify the many weaker low abundant sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Three putative CRP sites in partial agreement with the consensus sequence described by de Crombmgghe etai ( 1984 ) were centred in positions -368 , - 7 3 and - 3 1 ( Fig. 2 and Btasband et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Each of these CRP sites contains the highly conserved half of the consensus CRP sequence and , separated by 6 base pairs , a less conserved inverted-repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Some of these previously known CRP sites were predicted after in silico searching for the known CRP-box consensus sequence without experimental confirmation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Residues matching the corresponding consensus sequence for the CRP , 235 , 210 , 11 , and Shine-Dalgarno ( S-D ) sites are in bold and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Residues matching the corresponding consensus sequence residues for the FNR and CRP sites are in boldface type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In the tot42 to tot63 derivatives , the entire sequence between the CRP sites has been substituted ( see Figure 1C for exact sequences ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>It is interesting to note that the FNR proteins of Bacillus subtilis and Bacillus lichenformis are predicted to recognize CRP sites on the basis of the amino acid sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Two synthetic , partially selt-complementary oligonucleotides of 18 and 22 nucleotides ( 5 ' - AATTGTGAGTTAACTCAC-3 ' and 5 ' - AATTATGTGATCCGGATCACAT-3 ' , respectively ) encoding consensus CRP sites were cloned in these EcoRI sites in ttie deletion derivatives pJELl381 ( -78 ) , pJEL1362 ( -75 ) , pJEL1363 ( -74 ) , pJEL1364a ( -72 ) , pJEL1364 ( -66 ) and pJEL1367 ( -63 ) ( numbers in parentheses indicate upstream endpoints of the deo sequence in the deletion derivatives ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Above the sequence , the upstream endpoint ofthe ACRP-2 deletion ( pJEL136 ) , the deletions at positions -68 and -69 ( ATT ) , and the three point mutations in the CRP sites ( G -- Â at -45 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Inspection of the DNA sequence of the promoter region revealed the presence of a possible s70 promoter with a 210 box ( CATTAT ) and the previously observed consensus sequence for CRP ( TGTGA-6 bp-TCACA ) centered at position 241.5 ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The downstream promoter has multiple copies of the pho box , the consensus sequence shared by the pho promoters ; the upstream promoter has a consensus sequence for the promoters regulated by cyclic AMP and its receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Inspection of the DNA sequence around the -35 region of the upstream promoter found an inverted-repeat , TGTcAtctttc TgACA , which is similar to the consensus sequence of the promoter regulated by cAMP and CRP ( cAMP receptor protein ) , TGTGAN6TCACA ( 3 , 20 ) , and overlaps the pho box 3 ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Introduction of additional nucleotides in the sequence does , of course , change the photoreactivity pattern in the region where the substitution has occurred , but the presence of the RNA polymerase and/or CRP does not further modify the basal reactivity noted for the DNA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Since H1 presence of either CRP or HI suggests that the two proteins exhibits a very mild sequence specificity , this provides the recognize a similar DNA structure in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>This region includes the sequence between the two CRP targets and overlaps the CRP-1 target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>A sequence similar to that recognized by the cAMP-cAMP receptor protein ( CRP ) complex ( Zhang and Ebright , 1990 ; Busby et al. , 1995 ) is present in the agaR-agaZ intergenic region ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>CRP binds to gal DNA at the -30 to -52 region , which contains the 5 ' - AANTGTGANNT consensus sequence developed by Ebright et al. ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>As expected from the fact that the cstA gene is positively regulated by the CAMP-CRP complex ( Schultz et al. , 1988 ) , we identified by computer analysis CRP consensus sequence ( AGTGAS , T : AACA ) in the regulatory region ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>( ii ) There are multiple other sequences with ﬁve out of ten matches to the CRP consensus sequence in the rhaSR-rhaBAD intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>DNase I footprint analysis of the agaR-agaZ intergenic region revealed that the cAMP-CRP complex speciﬁcally protects the DNA sequence resembling the CRP consensus ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>B. Comparison of the sequence footprinted by cAMP -- CRP in the agaR-agaZ intergenic region [ 1 ] and the consensus sequence recognized by cAMP-CRP [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>A sequence in PhlyE ( -69 TTTGATATTTATCATA-54 ) more closely resembles the FNR consensus ( NTTGATNNNNATCAAN ) than the CRP consensus ( TGTGANNNNNNTCACA ) , and in vivo transcription studies revealed that although FNR occupies PhlyE more frequently than CRP , the latter is a more efficient activator of hlyE expression ( Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Upstream of the possible promoter , there is a sequence ( AA-TTTGA - - T - - - TCA-ACT at pos-itions 1496 to 1516 ) homologous to the cAMP-CRP consensus sequence ( AA-TGTGA - - T - - - TCA-ATA , Ebright et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>D. CRP consensus sequence ( Ebright et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Because this ` site ' bears only a vague resemblance ( CTGCA n6 TCAAA ) to the consensus sequence ( TGTGA n6 TCACA , Ebright et al. , 1989 ; Gaston et al. , 1989 ) , and because we did not observe this site when using the linearized pacs444 template ( Beatty et al. , 2003 ; Browning et al. , 2004 ) , we suspect that supercoiling might increase the affinity of the template for CRP at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>One half-site recognition sequence in each of CRP2 and CRP1 is underlined ; the CRP consensus sequence is TGTGA-N6-TCACA ( de Crombrugghe et al. , 1984 ; Ebright et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Also the CRP target of the ompA promoter shows this sequence homology ( Movva et al. , 1981 ) , whereas the right hand site of the cat target contains four changes from the consensus ( Le Grice et al. , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Arrows indicate regions of homology with the consensus CRP site 5 ' - TGTGAN6TCACA ( de Crombrugghe et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>To explain the results , we suggest that , in the zone containing the consensus sequence , CRP makes a small number of strong interactions , whereas around - 4 5 the CRP-DNA contact consists of a large number of weak interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>In the model system of the galP1 promoter , the shift of the sequence elements by 1 bp downward from the consensus 241.5 made CRP only a moderate activator on the promoter ( Lavigne et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Site-directed mutagenesis experiments indicated that the perfect consensus sequence with high af ® nity for CRP , centred at ÿ41 .5 bp relative to the cai promoter , plays a direct role in the sole cai activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Another putative half consensus sequence for CRP was detected at 4 bp from the Shine-Dalgarno sequence of caiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Ebright has proposed a 10-bp consensus : 5 ' - AANTGTGANNTNNNNCA-3 ' , as the recognition sequence for the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>In this way we may identify , utilizing the sequence determined by GarciaMartin e t al. ( 1992 ) , putative -10 and -35 boxes , an inverted-repeat likely to interact with RhaS and a CRP consensus inverted-repeat ( see Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8757746" target="_blank">8757746</a>]<br></li>
<li>In udpPYp , the sequence of the second CRP target ( CRP2 ) perfectly matched the consensus ( TGTGA-N6-TCACA ) in contrast to udpPEc and udpPSt that both contained one mismatch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>From nucleotide 637 to 642 a perfectly conserved CRP consensus Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7815937" target="_blank">7815937</a>]<br></li>
<li>After separate analysis of CRP-box sequence for two groups of CRP targets from Regulon DB , it turned clear that the consensus CRP-box sequence is highly conserved for the CRP targets identified by Genomic SELEX ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Just two exceptions are observed : CRP 1 differs from the consensus sequence in three conserved positions while CRP 4 differs from the consensus sequence in one conserved position ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The region protected by CRP contains a sequence that matches the CRP consensus at 14 out of 22 positions including eight out of the ten most highly conserved bases ( Berg & v o n Hippel , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>The consensus sequence for cAMP-CRP IS AATGTGANr ^ TCACATT [ Kolb ei aL. 1993 ) and for CytR TGCAAN3 .3 TTGCA ( Rasmussen ef aL. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The de cents nition of the consensus sequence for the HTH motif varies slightly and results in di ¡ erent assemblies of Crp-Fnr regulators in public databases : the SMART HTH-CRP signature SM00419 of EMBL comprises 229 proteins as of the end of 2002 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>The CRP recognition sequence has been studied in a large number of genes , and the consensus sequence 5 ' - AANTGTGANNtNgNTCAaAt at-3 ' ( 5 ) has been confirmed experimentally ( 5 , 11 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The ACRP mutation removed the central 3 base pairs of the CRP consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>The open boxes represent the positions of the sequences homologous to the CRP consensus recognition sequence ( see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>t002 Consensus recognition sequences of CRP The consensus recognition sequences of CRP have been proposed after sequence analysis of the hitherto identified targets ( for a review see [ 38 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>( b ) The ( c ) ( b ) sequence agreement of CRP box-1 with the consensus sequence is 90 % ( nine of 10 nucleotides ) while that of CRP box-2 is 80 % ( eight of 10 nucleotides ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>In concert with this ﬁnding , the sequence agreement of CRP box-1 with the CRP consensus sequence is 90 % ( nine of 10 nucleotides ) while that of box-2 is 80 % ( eight of 10 nucleotides ) ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Within the protected regions , a sequence similar to the consensus palindromic CRP box sequence , TGTGA-TCACA , was identiﬁed ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>JX15 contains a G-to-C base substitution at 241 as underlined in the TGTGA motif of the proposed CRP core consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>CRP consensus sequence of the promoter P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>Alignment of possible sequence ( complementary strand ) homology with other promoter CRP consensus sequence are marked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>A sequence resembling the CRP consensus sequence was also found ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>All ten agreed with the consensus sequence in CRP 2 , CRP 3 and CRP 5 ; all but one of the ten agreed with the consensus sequence in CRP4 , and only seven of the ten agreed with the consensus sequence in CRP 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Using DNase I footprinting we show that CAMP-CRP and RNA polymerase together bind tightly to the melR promoter sequence , creating a strong footprint from bp -70 to + 20 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>CaiF and CRP recruit RNA polymerase to the ® x promoter To extend the above observation made on the sole F1 / CRP2 region , occupancy of the ® x promoter was examined using the 316 bp sequence from 20 to ÿ296 that includes the two F1/CRP2 and F2/CRP3 regions ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Identification of the nucleotide sequence recognized by the CAMP-CRP dependent CytR repressor protein in the deoP2 promoter in E. co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Identiﬁcation of the nucleotide sequence recognized by the cAMP-CRP dependent CytR repressor protein in the deoP2 promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Rasmussen , P.B. , Sogaard-Andersen , L. , and Valentin Hansen , P. ( 1993 ) Identilication 01 the nucleotide sequence recognized by the cAMP-CRP dependent CytR repressor protein in the deoP2 promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>Our ﬁndings suggest that FNR and CRP bind to the same sequence in the clyA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>and Aiba , H , -LCB- 1988 ) Regulatory effect of a synthetic CRP recognition sequence placed downstream of a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_228">
Regulatory effects of a synthetic CRP recognition sequence placed downstream of a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631685" target="_blank">8631685</a>]<br></label>
<input id="sit_similarity_228", type="checkbox">
<div class="SIT_similar">
<li>Regulatory effect of a synthetic CRP recognition sequence placed downstream of a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18978044" target="_blank">18978044</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li></div>
<li>The cAMP-CRP complex can bind in vitro to 200 - to 300-base-pair DNA fragments of the spf gene containing the spf genes with 120 , 116 , and 103 base pairs of upstream sequence but not to a fragment containing only 77 base pairs of upstream sequence ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>The cAMP-CRP complex thus seems to bind to the putative sequence and to enhance gntT transcription ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>The region protected by partially purified CylR in the presence of cAMP-CRP is indicated by a bracket below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>As a reference , a cAMP.CRP footprint employing ng/pl of 20 CRP and 10 p ~ cAMP is displayed in Lane F.Lane A is the A + G sequence of the endlabeled fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Third , the protection extends close to the -10 sequence , i.e. , beyond the protection obtained with cAMP-CRP or CytR individually .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the absence of cAMP-CRP , the CytR repressor interacts with a sequence extending from CRP-2 into CRP-1 ( 317 to 337 bp [ Fig. 1B ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>When fragment 2 , which reduces the downstream sequence of fragment 1 by about 20 bp , was -100 -50 transcribed , cAMP-CRP activated the formation of the same 75-nucleotide RNA ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>In support of this model , we show that improvement of the base sequence of Site 20 removes the requirement for Site 10 and Site 1 , and short circuits the effects of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>A search for sequence similarity between the glycerol facilitator and other proteins revealed similarities between GlpF Sau31 '' '' '' '' '' '' '' '' '' '' GATCTAACAAACATGCATCATGTACAATCAGATGGAATAAATGGCGCGATAACGCTCATT 6 0 OFZ/CRP OF3KRP -35 OF4 I '' '' `` L '' '' '' '' '' '' '' '' '' '' ~ '' EEISISLE '' LP '' `` I === I ==== - '' '' '' '' '' '' '' '' '' '' '' TTATGACCAGCCACACACATTTTI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Therefore , we can not discard the possibility that the mentioned sequence ( with an A1T-rich N8 spacer ) might be the target for CRP in the regulation of gyrA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>CRP actually bound to the sequence between positions 54 and 14 from the transcriptional start ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>At saturating concentrations of CaiF and CRP , the footprint pattern covered a very large sequence from ÿ45 to ÿ147 ( Fig. 5A , lane 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Table 1 Strains and plasmids Source or reference Relevant characteristics Strains TG2 supE hsdD5 thi D ( lac-proAB ) D ( srl-recA ) 306 : : Tn10 ( tetr ) F cents [ traD36 proAB + lacIq lacZD5M15 ] TG2 containing plasmid pELM490 TG2 containing plasmid pELM264 TG2 containing plasmid pELM272 TG2 containing plasmid pELM301 TG2 containing plasmid pELM244 TG2 containing plasmid pELM295 Sambrook et al. ( 1989 ) ZM490 ZM264 ZM272 ZM301 ZM244 ZM295 Plasmids pBdad1 This study This study This study This study This study This study ClaI-BstEII dad promoter fragment cloned into the SmaI site of pBluescript II KS , the insert orientation ) is EcoRI - ( ClaI-BstEI dad fragment ) - BamHI Plasmid designed for in vitro transcription EcoRI-BamHI dad promoter fragment from pBdad1 subcloned into pLSZ13 pELM490 with the ) 35 region mutation of promoter 2 pELM490 with the ) 10 region mutation of promoter 1 pELM490 with the ) 10 region mutation of promoter 3 pELM490 with the CRP site 1 mutation pELM490 with the CRP site 2 mutation Mathew et al. ( 1996 ) pLSZ13 pELM490 Sun and Fuchs ( 1994 ) Mathew et al. ( 1996 ) 295 pELM pELM pELM pELM pELM This study This study This study This study This study Table 2 Primers used for PCR ¬ Sequence ( 5 cents ® 3 cents ) a based site-directed mutagenesis Primer Purpose TCATGCGGCTTTTTTATG CGAATAATATTTTCAACTGAG GTTATTCAACATGCGGCT GTTTTCAACTGAGTTATCAAG ATCACATCTTGATAACTCA CAGATTATTATTCTTTTACTG TGGCTCACATCTCCCT GCTGTCCATAAAAAAGC TTCATTATAACTCAGTTG CTGATTAGATTATTATTC PM264 PM265 ) 35 region mutation of promoter 2 ) 10 region mutation of promoter 1 ) 10 region mutation of promoter 3 CRP binding site 1 mutation CRP binding site 2 mutation PM272 PM273 P300 PM301 P240 PM241 C PM295 PM296 a Bases that di $ er from wild type are in italic Identi ® cation of three major dad promoters We used site-directed mutagenesis to investigate whether Tr1 ± 3 , the major dad transcripts made in vitro , are expressed from distinct promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>At Class II promoters , CRP binds at a target that overlaps the promoter -35 element and is usually centred at position -41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>At class II promoters , a CRP dimer binds to DNA at a site centered near position 41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_245">
At class I promoters , a CRP dimer binds to DNA at a site centered near position 61.5 , 71.5 , 82.5 , or 92.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></label>
<input id="sit_similarity_245", type="checkbox">
<div class="SIT_similar">
<li>At class I promoters , a CRP dimer ( cross-hatched double oval ) binds to DNA at a site centered near position 61.5 , 71.5 , 82.5 , or 92.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li></div>
<li>DNase I footprint analysis showed that CRP binds and protects two sites centered at bp 2313 ( site 1 ) and bp 2140 ( site 2 ) relative to the transcription initiation site , but a mutational analysis demonstrated that only site 1 is required for CRP-mediated regulation of gcvT-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>20 No. 19 pp. 5392 ± 5399 , 2001 Repression of deoP2 in Escherichia coli by C conversion of a transcription activator into a repressor ytR : Minsang Shin , Soim Kang , Seok-Jin Hyun , Nobuyuki Fujita , Akira Ishihama , 1 1 Poul Valentin-Hansen and Hyon E.Choy 2 3 Department of Microbiology , Chonnam University Medical College , 5 Hakdong , Dongku , Gwangju , South Korea 501-714 , Department of 1 Molecular Genetics , National Institutes of Genetics , Mishima , Shizuoka 411 , Japan and Department of Molecular Biology , 2 Odense University , Campusvej 55 , DK-5230 Odense M , Denmark 3 Corresponding author e-mail : hyonchoy@chonnam.ac.kr In the deoP2 promoter of Escherichia coli , a transcription activator , cAMP ± CRP , binds at two sites , centered at ± 41.5 and ± 93.5 from the start site of transcription , while a repressor , CytR , binds to a space between the two cAMP ± CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In contrast , for the P2 promoter , which functions in the absence of cAMP-CRP , the crucial sequence extends to only 39 base-pairs upstream from the transcription start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>CRP , a homodimer , undergoes a conformational change when complexed with its allosteric effector cAMP and binds to a speciﬁc sequence located near or within target promoters to regulate transcription ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>At low cAMP levels , transcription initiates from the galP2 promoter but as cAMP levels increase , the cAMP receptor protein ( CRP ) - cAMP complex binds to its recognition sequence preventing transcription from galP2 and activating transcription from the galP1 promoter , which is located 5 bp downstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li>
<li>In this case of autogenous regulation , the CAMP-CRP complex binds to a site downstream from the transcription start point ; thus bound the complex presumably prevents RNA polymerase from transcribing distal sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>In fact we showed previously that CRP-cAMP bound to the consensus CRP site placed far downstream of the transcription start site of test promoter only a weakly blocks enlongating RNA polymerase , while CRP-cAMP bound to the CRP site placed close to the promoter strongly inhibits the transcription by excluding RNA polymerase from the promoter ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>It partially overlaps the -10 sequence of the dad promoter and the transcription initiation region , implying that interaction of the complex with this site might interfere with transcription as in the case of some promoters repressed by cAMP-CRP ( for a review , see reference 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>Presumably the cAMP-CRP complex would bind to this site and interfere with the initiation of transcription of the speA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2198270" target="_blank">2198270</a>]<br></li>
<li>to the proposed model , the cAMP-CRP complex binds to this single site in the ara regulatory region to activate transcription of both the araC gene and araBAD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>In the presence of cAMP , the cAMP-CRP complex ( hereinafter called CRP ) will bind to a 22 bp 2-fold-symmetric recognition site located within or near target promoters ( Berg and von Hippel , 1988 ) , and thereby activate transcription from these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Based on in vitro studies of the cloned crp gene , we found previously that the transcription of crp is negatively regulated by CRP-cAMP that binds to a specific site -LCB- CRP site I ) located about 40 bp downstream from the transcription start site -LCB- Aiba , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Molecular mechanism of negative autoregulation of ABSTRACT Transcription of the Eschenchia coli crp gene encoding cAMP receptor protein ( CRP ) is negatively regulated by CRP-cAMP complex that binds to a specific site located downstream from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>Thus , MalI-dependent repression is marginally less efﬁcient compared with the malX promoter , the dependence on CRP is relaxed by the DNA site for CRP being located at position 43.5 , and the promoter carries seven repeats of a 50-TAN8-30 motif , to facilitate RNA polymerase recruitment ( Lloyd et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>The position of the Arr site ( -83 ) relative to the FNR site ( -41.5 ) of the ansB promoter suggests that Arr and FNR may co-regulate anaerobic ansB expression in a manner analogous to the co-activation of this promoter by tandemly bound FNR and CAMPreceptor protein ( CRP ) ( Scott et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>These CRP variants contain leucine ( CRP181 EL ) or valine ( CRPI 81EV ) substitutions at position 181 , they fail to discriminate between different bp at positions +4 and bind to both the intact CRP site in the lac promoter and to sites altered at positions +4 with an affinity 10-fold lower than that with which CRP-wt protein binds to the wildtype lac promoter target ( Ebright et al. 1984 , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Interestingly , the CRP sites at the galP1 and melR promoters ( de Crombrugghe et al. , 1984 ; Webster et al. , 1988 ) are located between positions 252 and 231 from the transcription start site and also share 14 and 12 positions , respectively , with the consensus sequence ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>Interestingly the CRP site at the gafP1 promoter is also located from bp -52 to -31 and is also homologous to th consensus at ten positions ( de Crombrugghe et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Based on genetic and biochemical analyses of cAMP-CRP/DNA interactions in the lac promoter , the second amino acid in the recognition helix of CRP ( glutamate at position 181 ) has been proposed to make contacts to the GC basepair located at position + 4 and the CG bp at position - 4 in the consensus CRP binding site , 5 ' - TGTGAN6TCACA-Y ( both positions underlined ) ( Ebright et al. 1984 , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In the crystal structure of the CRP-aCTD-DNA complex , aCTD contacts approximately 6 bp of DNA spanning a minor groove , centred 18 -- 19 bp from the centre of the DNA site for CRP ( Benoff et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In accordance with this prediction , the CRP target has been found to vary for several genes , not only with respect to the DNA sequences and symmetry characters but also with respect to the distance between the binding site and the start of transcription ( Reznikoff and Abelson , 1978 ; Simpson 1980 ; Taniguchi et al. , 1979 ; Ogden et al. , 1980 ; Lee et al. , 1981 ; Queen and Rosenberg , 1981 ; Le Grice et al. , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>In the case of class II CRP-dependent promoters the activation is mainly due to CRP binding to a 22 bp target site centred at or near position 241.5 from the transcription start site , overlapping and replacing the 235 determinant for binding of RNAP ( Zheng et al. , 2004 ; Busby & Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>It is intriguing that the centre of the DNA binding site for CRP is located at different distances from the transcription start site , e.g. -41.5 , -61.5 and -70.5 bp , for the gal , lac and malT promoters , respectively ( Taniguchi et al. , 1979 ; Schmitz , 1981 ; Chapon and Kolb , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>This demonstrated that there are two CRP target sites in the cspD upstream region with one distal to the promoter ( centered at 112.5 ) and the other proximal ( centered at position FIG 1 ( A ) Nucleotide sequence of cspD upstream region showing the two CRP target sites , PcspD and the transcription start ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Sequence analysis of the cspD upstream region revealed two tandem CRP target sites , CRP site-I ( the proximal site centered at 83.5 with respect to the transcription start ) and CRP site-II ( the distal site centered at 112.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Sequence analysis of cspD upstream region revealed two putative CRP target sites : CRP site-I ( centered at position 83.5 , spanning positions 94 to 73 , proximal to the promoter ) and CRP site-II ( centered at position 112.5 , spanning positions 123 to 102 , promoter distal ) ( Fig. 1A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_273">
Nucleotide positions 9 to 14 is termed the spacer region in the CRP target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></label>
<input id="sit_similarity_273", type="checkbox">
<div class="SIT_similar">
<li>Nucleotide positions 9 to 14 are termed as spacer region in the CRP target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>The sequence analysis of the cspE upstream region revealed a putative CRP target site , spanning the region of 72 to 51 , centered at 61.5 relative to the transcription start site ( TSS ) ( Fig. 1A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>( A ) Nucleotide sequence of cspE upstream region showing the CRP target site , PcspE , and the transcription start ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Sequence analysis of the cspD upstream region revealed a putative CRP target site ( http://www .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Sequence analysis of the cspE upstream region revealed a putative CRP target site centered at 61.5 relative to the transcription start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>Deletions of either of the two core motifs as deﬁned in Fig. 1B , or both of them together , abolished CRP binding ( Fig. 3A , lane 3 to 8 ) , thereby validating that CRP binds to the putative target site sequence , which is centered at 61.5 ( relative to the TSS ) in the cspE upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The regulatory region also contains a single putative IdnR/GntR binding site centered between the two promoters , a CRP binding site upstream of idnD , and an UP element upstream of idnK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>Above the sequence are shown the various P1 elements : the 22-bp DNA site for CRP and the 235 and extended 210 elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781573" target="_blank">10781573</a>]<br></li>
<li>For example , in the aer211 fragment , a 6 bp sequence , 5 ′ - TAAAGA-3 ′ , is located 32 bp downstream of the DNA site for CRP and this may well generate a weak Class II CRP-dependent promoter served by Es70 ( recall that the consensus -10 hexamer for Es70 is 5 ′ - TATAAT-3 ′ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Previous studies have found that for Class II CRP-activated promoters , the distance between the centre of the DNA site for CRP and the 10 hexamer element is usually 32 bp ( Busby & Ebright , 1999 ) and , thus , we predict that the malX 10 hexamer sequence is 50-TATCTT-30 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>Binding of CRP to the P1 promoter region within a 330-bp DNA fragment containing the DNA sequence from 2113 to 1196 with respect to P1 transcription initiation site was assessed by gel mobility shift and DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>The experiment was then repeated with M182 cells carrying pRW50 with the malX400 promoter fragment , in which the malX promoter sequence upstream of the DNA site for CRP had been removed ( illustrated in Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>To check whether the DNA site for MalI at the malX promoter plays any role in this repression , the experiment was repeated with pRW50 carrying the malI375 promoter fragment , in which the malI promoter sequence upstream of the DNA site for CRP had been removed ( illustrated in Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>A well-predicted FNR site and a moderately predicted CRP site are centered at bp 103.5 ( 45.5 ) and bp 43.5 2 bp ( 2106.5 ) , respectively ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>As the CRP concentration was increased from 5 to 100 nM , two regions were protected from DNase I cleavage ; one site centered near bp 2313 as expected and the other site centered at bp 2140 ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>For the proximal , higher-afﬁnity site CRP I , we generated substitutions in the upstream half-site ( G75C , T76A , and G77C ) or inserted 1 bp into the spacer ( 1A ) ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>In both cases , the CRP site is upstream of the XylR site , either immediately adjacent or separated by 10 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Two groups of bands , separated by 10 bp , are observed at the ends of both site 1 and site 2 that abut the CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>To test whether the putative CRP site centered at bp 2313 can be bound by CRP in vitro , gel mobility shift assays were performed with puriﬁed CRP and two different DNA templates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The CRP dimer was added at a concentration of either 10 nM ( 1 ) or 100 nM ( 11 ) 759-bp EcoRI-BamHI fragment carrying wild-type gcv DNA extending from bp 2466 to 1293 relative to the gcv transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Consistent with this , CRP has been shown to bind to an 300-bp DNA fragment that extends to position 253 with respect to the PguaB transcription start site ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>As a prerequisite to activating transcription , CRP must bind to a 22-bp twofold-symmetric recognition site on DNA ( 2 , 9 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Except nirB and pncB , promoter regions are predicted by searching the 10 and 35 sequences upstream of 10 down-regulated genes with a CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>There are no apparent further CRP sequences immediately upstream from this site , indicating that the alteration of the helix phase produced hy the deletion was unlikely to be responsible for the lack of activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>DNA sequences are numbered with the transcription start sites labelled as 11 and upstream and downstream sequences are assigned negative and positive Derivatives of the malI100 fragment carrying G for C substitutions at either position 39 ( p39G ) or position 122 ( l22G ) , with respect to the malI transcript start , that inactivate , respectively , CRP site 1 or CRP site 2 , were constructed as described by Hollands et al. ( 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>HindIII primer : 5 ′ - AGCAAGCTTCGCTGCTGCACATAAA-3 ′ EcoRI primer : 5 ′ - GCAGAATTCCCTCATGATACTCGGA-3 ′ This resulted in an EcoRI -- HindIII fragment carrying pmelR with the EcoRI site upstream of the pmelR DNA site for CRP and the HindIII site downstream of the melR translation initiation codon ( -- 289 to -- 136 with respect to the pmelAB initiation start site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>RP ' complex , and thepromoters analyzed in detail ( deoP2 , cdd , tsx , and cytR ) have been shown to possess tandembinding sites for cAMP.CRP upstream from the start site for transcription ( Valentin-Hansen , 1982 ; Valentin-Hansen et al. , 1989 ; Martinussen et al. , 1989 ; Gerlach et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>In 70 addition , csiD expression is activated by cAMP-CRP , which binds to a site centred around 268.5 upstream from the transcriptional start site ( Weichart et al. , 1993 ; Marschall et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>The S essential activator cAMP-CRP binds to a site centred around 268.5 nucleotides upstream of the transcriptional start site and can activate csiD transcription in vitro in conjunction with Es as well as with Es , although overall S 70 transcriptional activity is much less with the latter ( Marschall et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>data presented here show that the putative CRP site is about 80 nucleotides upstream from the most proximal transcriptional start site , which is unusual for Ea7 ' - activated promoters ( Gralla , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>In the present study , we repositioned the CRP box to locations between 280.5 and 260.5 ( in 2 bp increments ) with respect to the csiD transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>However , 121 bp upstream of the transcriptional start site is an unusually large distance for direct activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>We conclude that 72 bp upstream of the transcriptional start site are sufficient for the control of csiE by os and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>A 5 ' - deletionanalysis revealed that despite the complex regulation of csiE , only 72 bp upstream of the single transcriptional start site are sufficient for normal control by os and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>two targets for the cAMP/CRP complex , located around -41 ( CRP-1 ) and - 9 3 ( CRP-2 ) upstream from the transcriptional initiation site ( Valentin-Hansen , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>TAACGCCTTT ) , located at 257.5 and 255.5 from the transcriptional start site of glnAp2 , respectively , might bind CRP -- cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>By DNase I protection experiments , a cAMP-CRP binding site centered at ÿ68 .5 nucleotides upstream of the csiD transcriptional start site was identi ® ed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>A quite different result was obtained when the cAMP-CRP was shifted to 260.5 nucleotides upstream of the csiD transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The start site determined in the present study agrees particularly well with the location of an extended ¹ 10 box ( TGATAACCT ) and with the idea that CRP activates transcription from P5 : the putative CRP-1 target is centred around 41 bp upstream of the start site , a distance that has proven to be characteristic for activation from Class II-dependent CRP promoters ( Busby and Ebright , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Thus , the measured activities of both promoters are unchanged by the introduction in trans of higher copy number plasmids carrying the malX -- malI intergenic region The malX promoter is a ` textbook ' Class II CRP-depen-dent promoter , with the 10 element positioned 32 bp downstream of CRP site 1 , which is centred at position 41.5 upstream from the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>The modesty of the activation is partly due to the activator-independent strength of the malI promoter , but principally it is because CRP site 1 is located 34 bp upstream of the 10 hexamer element , which is 2 bases upstream of the optimal position for Class II activation ( Busby & Ebright , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>CRP binds to a site from approximately bp ® 303 to ® 324 relative to the transcription start site ( Fig. 1 ) and is required for a fourfold activation of the gcv operon in glucose minimal ( GM ) medium ( Wonderling & Stauffer , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>When present , CRP binds to three sites over 100 base pairs upstream of the malKp transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>At class I CRP-dependent promoters CRP binds upstream but not adjacent to RNAP , with sites for CRP usually centered at positions 262.5 , 272.5 , or 292.5 relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Our results ( Fig. 4A ) demonstrated that both the CRP A and CRP B transcripts result from initiation upstream of the normal pSR start site and that CRP A transcription initiated at either position 19 or 18 with respect to the normal pSR transcription start site and CRP B transcription initiated at position 32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Although CRP sites appear from different distances upstream to cognate promoters , maximal activation is obtained when CRP is located at bp 261.5 or 241.5 upstream of the transcription start site ( reference 25 and citations therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>c Although derived from PguaB ( 133 to 36 ) , upstream displacement of the CRP site in this promoter derivative means that native PguaB sequences extend to position 117 and the upstream boundary of the CRP site is located 139 bp upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In most cases , CRP activates transcription from a point about 40 bp upstream of the start of transcription , but in others it interacts from a point at least 6 turns of DNA helix upstream from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>There are no known examples where CRP , acting alone or in concert with an UP element , modulates transcription from a site located more than 100 bp upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP was first identified as a transcription activator of the lac operon , where CRP binds at a site located 61.5 bp upstream of the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>For class I promoters , the DNA target site for CRP is located upstream of the DNA site for RNAP , at 61.5 or at various distances upstream from the transcription start point , provided that the DNA target site for CRP and the DNA site for RNAP are on the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>From this position , or a site located one helical turn upstream , CRP can activate s38 - , but not s70-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Two members of the CRP family of transcription factors control the expression of hlyE in E. coli K-12 by binding at the same site centered at 61.5 bp upstream of the hlyE transcriptional start ( 10 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Footprinting studies have shown that FNR and CRP activate hlyE expression from the same site centered 61.5 bp upstream of the SlyA-associated transcription start site ( 10 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Based on the results of Busby and coworkers ( 3 , 22 ) , we anticipated that the CRPK52N protein likely exerted its effect when bound to a CRP recognition site centered ; 41.5 bp upstream of the start point of fucPIK transcription for both the wild-type and IS5-disrupted promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>Type II promoters , typified by the galPl promoter ( 19 ) , have the CRP site centered at 41.5 ± 1 bp from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>Each contains both a proximal CRP site ( CRP1 ) that overlaps the -35 - promoter element and a distal site ( CRP2 ) located at -91 to -93 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Next , we used the same system to monitor the effects of making a 5 bp deletion or insertion between the DNA site for CRP and the -35 element at the aer promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>As the DNA site for CRP at the aer promoter is located 12 bp downstream , it appears that there can not be sufficient space for aCTD to ﬁt between the CRP dimer and the promoter-bound sigma factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In the tsr regulatory region , the predicted CRP site is located 132.5 bp upstream of the s28-dependent tsr promoter , so it is unlikely that CRP makes direct contact with bound Es28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In addition to stimulation of proP P2 transcription by Fis , CRP co-activates this promoter when bound to a remote site upstream from the promoter ( centered at ÿ121 .5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Binding of CRP at a site centered 121.5 bp upstream of the P2 promoter gives little ( ,2 - fold ) activation by itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>This 1-bp deletion at the 279 position created a new promoter 64 bp upstream of the original 11 site and made udp transcription independent of CRP and CytR ( Table 3 ; Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>On the basis of these assays , we conclude that CRP binds to a single site located in the 39 portion of the acs-nrfA intergenic region proximal to the putative acs promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Thus , the IS5 element separates the fucPIK core promoter and CRP site 1 from putative regulatory sequences located further upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>Here we show that CRP binds to the putative CRP site located upstream of P , whereupon it serves to decrease promoter guaB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>8 1 4 5 We report here the results of DNase and restriction enzyme site protection studies which indicate that CRP binds to the ilvB promoter at approximately position -44 to -82 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Gel shift assay and AFM observation indicate that CRP binds to both the promoter-distal site ( CRP box-1 ) at 90.5 and the site ( CRP box-2 ) at +13.5 downstream of transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Whether CRP binds to either of the two promoter regions and stimulates transcription from each site is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>Fragment EP was also retarded , suggesting that CRP binds to a site near the gua promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Our results support the model in which repression normally involves the binding of CytR to cAMP-CRP to form a complex which binds to speciﬁc sites in the udp promoter , without direct interaction between CytR protein and a speciﬁc operator DNA sequence , and in which induction by speciﬁc inducer cytidine involves dissociation of CytR from cAMP-CRP and then RNA polymerase interaction with cAMP-CRP bound to a site upstream of the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>At class I promoters ( with the lac promoter as the prototype ) , cAMP-CRP binds to a position ( the ` CRP box ' ) centred around 261.5 nucleotides upstream of the transcriptional start site and interacts with the C-terminal domain of the RNA polymerase a-subunit ( aCTD ) , which is positioned on the DNA between cAMP-CRP and the promoter-binding s - subunit of RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Expression from the melR promoter is completely dependent on the cyclic AMP receptor protein ( CRP ) , which binds to a DNA site centered at position 241.5 upstream from the melR transcript start point ( Webster et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760179" target="_blank">10760179</a>]<br></li>
<li>CRP binds to a single DNA site upstream of I1 and I2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>AraC-dependent transcription initiation at the araBAD promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>To activate transcription , CRP binds to a specific DNA site located upstream from a promoter and interacts with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Further studies showed that the melR promoter is a typical Class II CRP-dependent promoter , activated by a single dimer of CRP , that binds to a 22 bp DNA site centred at position 241.5 with respect to the transcription start point ( reviewed by Busby and Ebright , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The activity of the malT promoter is regulated by the activator protein CRP which binds to a specific recognition site centred at -70.5 bp from the start site of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The upper panel shows a schematic diagram of the CRP-Es70-DNA ternary complex at the Class I CRP-activated lac promoter , where CRP binds at a site centred 61.5 bp upstream of the transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The lower 28 panel shows a model for the organization of the CRP-Es - DNA ternary complex at the aer promoter , where CRP binds at a site centred 49.5 bp upstream from the transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>G85 is located in a region in FNR homo-logous to that in CRP identified by substitutions at K52 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>In this context , there is a core motif ( TTGAT ) for a potential FNR half-site at about -55 in the putative ttdABpromoter region ( Nesin et al. , 1987 ) , and an analogous CRP half-site motif ( GTGTA ) at - 130 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8371115" target="_blank">8371115</a>]<br></li>
<li>Strain characteristics endA1 hsdR17 ( r m 1 K K ) supE44 thi-1 recA1 gyrA96 relA1 D ( lacZYA-argF ) U169 [ f80 ( DlacZ ) M15 ] SY327 ( lpir ) D ( lac-pro ) argE ( Am ) rif nalA recA56 ( lpir ) MC1061 araD139 D ( ara , leu ) 7697 DlacX74 galU galK hsr hsm1 strA BEU616 MC1061 hns : : cat ( Cmr ) M182 D ( lacIPOZY ) X74 galK galU strA M182 crp M182 Dcrp M182 fnr M182 fnr : : tet ( Tcr ) M182 crp fnr M182 Dcrp fnr : : tet ( Tcr ) JON31 M182 hns : : cat ( Cmr ) r JON32 M182 Dcrp hns : : cat ( Cm ) r r BEU701 M182 fnr : : tet hns : : cat ( Tc Cm ) r r BEU705 M182 Dcrp fnr : : tet hns : : cat ( Tc Cm ) MC4100 araD139 D ( lac ) U169 strA thi BSN26 MC4100 trp : : tet ( Tcr ) BSN27 MC4100 trp : : tet Dhns ( Tcr ) JON33 BSN26 clyA : : luxAB JON34 BSN27 clyA : : luxAB MWK6 BSN27 DslyA MWK10 MWK6 clyA : : luxAB MWK11 MC4100 with a consensus CRP binding site in the clyA promoter regiona a 59-AAATGTGATCTAGATCACATTT-39 DH5a 23 38 11 This work 11 9 22 2 This work This work This work This work 10 26 26 This work This work This work This work This work TABLE 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus CRP binding site , ApR pBR322 carrying the complete crp gene and crp promoter , ApR pBR322 carrying the crp promoter and the pUC9 polylinker in place of the crp gene , ApR pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus FNR binding site , ApR pGEX-KG containing the fnrDA154 mutation , ApR Promoter-probe vector with promoter-less cat gene , ApR pUC18 carrying the 310-bp AluI fragment ( xseP ^ guaP ) of pMT104 , ApR pUC18 carrying the 310-bp AluI fragment ( guaP ^ xseP ) of pMT104 , ApR pKK232-8 carrying an approximately 500-bp xseP ^ guaP fragment in front of the promoter-less cat gene , ApR CmR pKK232-8 carrying an approximately 500-bp guaP ^ xseP fragment in front of the promoter-less cat gene , ApR CmR pPV-33H carrying a 1.03-kb guaP BP fragment , ApR small , high copy number vector carrying a 54-bp polylinker in the lacZ N-terminal sequence and the ColE1 origin of replication , Apr [ 26 ] [ 27 ] [ 27 ] [ 26 ] [ 24 ] [ 28 ] this work this work this work this work [ 1 ] [ 23 ] 2.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Description Bacterial strains JM101 JRG3462 M182 M182vcrp MH1002 MH1003 MH1004 MH1005 MH1009 MH1010 MH1011 MH1012 MH1015 MH1016 MT1024 Plasmids pCC pDCRP pDU9 pFF pGS771 pKK232-8 pMH100 pMH101 pMH102 pMH103 pMT104 pUC18 15 proAB ] v ( lac-proAB ) thi supE [ FP traD36 laclq v ( lacZ ) M CAG627 ( pGS771 ) v ( lacIPOZY ) X74 galK galU rpsL M182 vcrp JM101 ( pMH100 ) JM101 ( pMH101 ) M182 vcrp ( pDU9 ) M182 vcrp ( pDCRP ) JM101 ( pMH102 ) JM101 ( pMH103 ) M182 ( pMH102 ) M182 vcrp ( pMH102 ) M182 ( pMH103 ) M182 vcrp ( pMH103 ) W3110 ( pMT104 ) [ 23 ] [ 24 ] [ 25 ] [ 25 ] this work this work this work this work this work this work this work this work this work this work [ 1 ] pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus CRP binding site , ApR pBR322 carrying the complete crp gene and crp promoter , ApR pBR322 carrying the crp promoter and the pUC9 polylinker in place of the crp gene , ApR pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus FNR binding site , ApR pGEX-KG containing the fnrDA154 mutation , ApR Promoter-probe vector with promoter-less cat gene , ApR pUC18 carrying the 310-bp AluI fragment ( xseP ^ guaP ) of pMT104 , ApR pUC18 carrying the 310-bp AluI fragment ( guaP ^ xseP ) of pMT104 , ApR pKK232-8 carrying an approximately 500-bp xseP ^ guaP fragment in front of the promoter-less cat gene , ApR CmR pKK232-8 carrying an approximately 500-bp guaP ^ xseP fragment in front of the promoter-less cat gene , ApR CmR pPV-33H carrying a 1.03-kb guaP BP fragment , ApR small , high copy number vector carrying a 54-bp polylinker in the lacZ N-terminal sequence and the ColE1 origin of replication , Apr [ 26 ] [ 27 ] [ 27 ] [ 26 ] [ 24 ] [ 28 ] this work this work this work this work [ 1 ] [ 23 ] 2.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Whereas the LH region of the PaaX binding motif is rather conserved in the operator sequences of other bacterial regulators such as GntR ( 6 ) and PdhR ( 27 ) ( which belong to the GntR family ) and , surprisingly , CRP ( 23 , 25 ) and LacI ( 26 ) ( which belong to the GalR-LacI family ) , the RH region differs from that of other operators ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Putative GntR - ( positions 1568 to 1585 ) and cAMP-CRP binding sequences ( positions 1496 to 1516 ) are represented by bold characters and an underline , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>Overlines and underlines represent putative ~ ° - dependent promoter sequences and cAMP-CRP binding sequences , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8647276" target="_blank">8647276</a>]<br></li>
<li>Perhaps more important is the fact that the region around -35 in the P2 and P3 sequences is nearly identical ( 10 out of 13 bp ) although this region is also involved in binding of the cAMP/CRP complex in the P2 promoter and is within a translated region in the P3 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6087276" target="_blank">6087276</a>]<br></li>
<li>First , CRP action may be mediated by CRP binding to speciﬁc sequences located at the promoter region of the gyrA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Under low-glu-cose and LP conditions , where both proteins are abundant , PhoB and cAMP-CRP would compete for binding to the psiE promoter region , which contains overlapping sequences recognized by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_365">
Since sequences very similar to the CRP-2 target have also been found in the regulatory region of the cdd gene , and to which CRP binds , we suggest that several of the other CytR binding sites may contain sequences which also will be recognized by the cAMP/CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></label>
<input id="sit_similarity_365", type="checkbox">
<div class="SIT_similar">
<li>Since sequences very similar to the CRP-2 target have also been found in the regulatory region of the cdd gene ( manuscript in pre-paration ) , and to which CRP binds , we suggest that several of the other CytR binding sites may contain sequences which also will be recognized by the cAMP/CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li></div>
<li>In the third model , CytR interacts with the same DNA sequences in the combined footprint as in the independent footprint , and cAMP-CRP , rather than binding to CRP-1 , binds to the region located immediately downstream of the CytR-binding sequence ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Finally , the speciﬁcity of the binding region was conﬁrmed because the cAMP-CRP complex could not bind to the other synthetic fragment containing the mutated sequences , ACGCTAGGGGTAGAGT ( Fig. 3B , top ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>In the intergenic region between the coding regions of gntV and idnD , there are 2 sequences , 5 ) - TGTGATGAA-GATCACG-3 ) ( cCI ) and 5 ) - TGTAATTCTTATCAGT-3 ) ( cCII ) , similar to the canonical cAMP-CRP complex binding element , which are located with HRI side by side and overlapping with HRIII , respectively ( fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>In vitro binding of cAMP-CRP to fragments of the dad regulatory region containing either of its recognition sequences separately suggests that both identified sequences are involved in the cAMP-CRP-mediated regulation of dad transcription in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>We estimate that our search included representatives of 15 to 25 % of the total number of different regulatory binding proteins in E. coli , including sites that are recognized by global regulators of transcription ( for example , sites bound by the cyclic AMP receptor protein , CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9278503" target="_blank">9278503</a>]<br></li>
<li>In this genetic context , i.e. in the absenceof CRP , transcription activation showsthe samedependence on the presence of the MalT-binding sites located 200 to 240 bp upstream from the transcription startpoint and , at least for malKp , shows the same requirement for a correct phasing between the distal MalT-binding sites and proximal elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>DNase I footprinting experiments revealed that ArcA-P binds to three sites upstream of the ptsG P1 promoter , two of which overlap the CRP-binding sites , and the ArcA-P binding decreases the CRP binding that is essential for the ptsG P1 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>Sequence analysis indicates that their CRP-binding sites overlap the binding sites for RNAP ( Figure 3 ) and it is therefore probable that CRP represses transcription at the nirB and pncB promoters by either blocking the interaction between RNAP and the promoter DNA , or blocking transcription from an alternative upstream promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP activates transcription by binding to speciﬁc sites located upstream of promoters and contacting RNAP ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Previously , we reported that cAMP receptor protein ( CRP ) activates acs transcription by binding tandem DNA sites located upstream of the major acsP2 promoter and that the nucleoid protein IHF ( integration host factor ) binds three speciﬁc sites located just upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>CRP activates acs transcription by binding tandem sites located upstream of the major promoter , acsP2 , whereas , both IHF and Fis each act at different times during growth as anti-activators of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>CRP activates acs transcription by binding tandem DNA sites located upstream of the major promoter , acsP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Binding in vitro of CRP to sequences proximal to the gua promoter Sequences upstream of the 335 region of the gua promoter show homology to the consensus binding sites for several transcription regulatory proteins including those for CRP and Fis ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>We show that , like hupA , the hupB gene has binding sites for CRP and FIS upstream of the core promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>DNA sequences of the CRP binding sites in the crp promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>Two locations within the promoter region contain DNA sequences similar to CRP DNA binding sites ( 26 ) , one immediately upstream of the 35 hexamer and the other overlapping with the 10 hexamer ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Nevertheless , there are sequences upstream of the pdh promoter which match the core consensuses of the FNR iJJGAT ' -- ATCAA ) and CRP ( TGTGA TCACA ) binding sites at 9 out of 10 and 8 out of 10 positions , respectively ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Thus , sequences upstream of position 209 , which include the acsP2 promoter and the Fis III and CRP I binding sites , do not inﬂuence expression from pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Alignment was done with the nucleotide sequences of the binding sites for the PaaX repressor in Pa ( PaaX ( Pa ) ) and Pz ( PaaX ( Pz ) ) promoters , the CRP activator in lac promoter ( CRP - ( Plac ) ) ( 25 ) , the LacI repressor ( 26 ) , the GntR repressor ( 6 ) , and the PdhR repressor ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>FIS prevents crp transcription initiation FIS and CRP compete for binding to overlapping binding sites at the crp promoter To investigate the mechanism by which FIS represses crp transcription , we mapped by DNase I footprinting the FIS - and CRP-binding sites on a crp promoter fragment encompassing sequences from -- 126 to 1119 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Sequence analysis also showed four putative CRP binding sites , one centered at 110 with respect to the ulaG gene and another centered at 61 with respect to ulaA , and two additional sites overlapping the 10 or 35 promoter sequences of the ulaA gene ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Transcription of the gltA and sdhCDAB gene appears to diverge from a single intergenic region that contains two pairs of potential promoter sequences and two putative CRP ( cyclic AMP receptor protein ) - binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6383359" target="_blank">6383359</a>]<br></li>
<li>We show the locations of the two strongest DNA binding sites for phosphorylated NtrC activator ( NtrC-P ) ( bold and shadowed ) , the consensus DNA sites for cAMP receptor protein ( CRP ) ( bold and double-underlined ) , the consensus DNA sites for integration host factor ( IHF ) ( bold and dot-underlined ) , the centers of protein-mediated DNA bending ( shadowed , bold and underlined ) , the 24 / 12 s54 binding boxes ( bold and underlined ) , the transcription start point ( 11 ) and the point mutation of the core promoter of glnAp2 in CC-55 .5 - M ( shadowed and underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Comparison of putative CRP-binding sites in the promoteroperator region of the dad operon [ dadR ( 1 ) and dadR ( 2 ) , the sequences underlined and labeled cAMP-CRP ( 1 ) and cAMP-CRP ( 2 ) , respectively , in Fig. 3 ] with the consensus sequence of CRP-binding sites and regions interacting with cAMP-CRP complex found in other operons ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>In a region preceding the transcription start sites of the dad operon , we found two sequences highly homologous to the consensus for cAMP-CRP binding sites ( Fig. 3 and 7 ) ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>Computer analyses revealed several readily recognizable consensus sequences upstream of the cstC ORF ( Fig. 3 ) : two each for cyclic AMP-cyclic AMP receptor protein complex ( cAMP-CRP ) and NRI binding sites , one for an integration host factor binding site , and consensus sequences for three promoters , s70 , sS , and s54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696780" target="_blank">9696780</a>]<br></li>
<li>We identiﬁed 263 S-controlled genes ( i.e. , 55 % ) featuring putative cAMP-CRP binding sites ( with one , two , or three mismatches from the consensus ) upstream of their coding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Since these sites matched the consensus sequences only poorly , we checked their ability to bind CRP in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>This region contains several repeats of AAAAATT/G and TTTTCA segments ( overscored in Figure 2 ) , and these segments are flanking regions which show high homology to the consensus sequences of CRP binding sites ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>Furthermore , the core consensus sequences for the FNR and CRP protein binding sites are quite similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>In only a few such cases has the direct binding of CRP to these consensus sequences been demonstrated in vitro , and in still fewer cases has the physiological significance of two or more binding sites been established in vim .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Even though operators 0 ~ 1 - 0 ~ d4o not match the consensus extremely well , the apparentcooperative binding of repressor to these sites ensures tight controlp , robably because interaction between cAMP.CRP and DNA and/or between RNA polymerase and DNA are efficiently blocked by bound repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Although the sites in the glpFK promoter do not match the consensus very well , cooperative binding of CRP to the tandemly repeated sites may be at least part of the reason this promoter is relatively sensitive to catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Model for Cooperative and Competitive Binding of CytR and cAMP-CRP -- To determine whether such interactions of CytR with DNA sequences overlapping and thereby occluding or competing with CRP binding to CRP1 and CRP2 can account quantitatively for both the loading free energy changes in Table I and apparent free energy changes in Fig. 6 , two models that incorporate two additional CytR binding sites and which constitute opposing possibilities were evaluated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Here too , CRP bound to either CRP1 or CRP2 or both is a net recruiter for CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>By eliminating cooperativity between CytR and CRP bound to flanking sites , cytidine should make CytR into a simple competitor for CRP binding to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Additionally , while cytidine binding to CytR nearly eliminates the pairwise interactions between CytR and CRP bound at CRP1 , it has little effect on pairwise cooperativity between CytR and CRP bound at CRP2 or as a consequence on the overall cooperativity of the three-protein complex in which CRP is bound to both sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In this model , CRP and CytR compete for binding to mutually exclusive sites at CRP2 , but CRP alone binds to CRP1 with no competition from CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>But this would also be partly offset by unfavorable competition between CytR and CRP for binding to overlapping sites at CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These data indicate strong cooperativity in CytR-CRP action and direct CRP binding to its recognition sites at udpP , as at other CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>While the majority of CytR-binding sites are imperfect inverted-repeats situated between binding sites for another transcription factor , CRP , other architectures were observed , in particular , direct repeats .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>deoO is the sites recognized by DeoR ; CRP and CRP2 the cAMP/CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>In the case of cAMP-CRP , the binding sites are mostly located in intergenic spacer regions between divergently transcribed genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In the case of group-B1 ( 50 CRP-binding sites ) , CRP binds to non-coding intergenic regions while in the case of group-B2 ( 14 CRP-binding sites ) , CRP binds on to open reading frames near the boundary with intergenic spacers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>cAMP-CRP binding sites ( Zhi et al. , 1998 ) are shown by wavy lines under the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>In addition , there are two binding sites for cAMP-CRP complex and another for GntR or IdnR ( Izu et al. , 1997 , Tsunedomi et al. , 2003b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
<li>( i ) CytR and cAMP-CRP compete for the same DNA sequences , with CytR binding cooperatively to the tandem CRP sites , i.e. , a model which would resemble the cI/Cro system of bacteriophage lambda , in which the cI and Cro proteins bind the DNA and each excludes to same sequences the binding of the other ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Here we demonstrate the presence of two binding sites for the cAMP-CRP complex , CRP-1 and CRP-2 , and show that cAMP-CRP and CytR bind cooperatively to tSx-p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Activation of these three promoters occurs by binding of cAMP-CRP to two sites separated by 53 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Several promoter regions with tandem binding sites for cAMP-CRP have been identified ( Gerlach et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Three such promoters have been sequenced ( deoCp2 , cdd , and tsx-p2 ) , and all possess tandem cAMP-CRP binding sites ( 14 , 29 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Red boxes represent GalR/GalS operators ; green boxes represent CRP binding sites ; and arrows represent transcription start points .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Thus , the competition between FIS and CRP for binding to their overlapping and tandem sites helps to keep acsP2 transcription low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>These results support the hypothesis that FIS-dependent anti-activation of CRP-dependent acsP2 transcription involves both FIS II and the competition between FIS and CRP for binding to their overlapping sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>FNR , CRP , and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Potential 70 10 and 35 sequences are indicated in bold , and the predicted CRP binding sites are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Possible ribosome-binding sites , putative -- 10 and -- 35 elements and putative CRP sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871335" target="_blank">9871335</a>]<br></li>
<li>Discussion One approach to extracting the relevant information of CRP binding sites is to compare the different targets for conserved sequences and adjust the obtained information with known mutations and enzymatic and chemical protection studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Speciﬁc sequences containing subsets of the putative CRP binding sites were PCR ampli-ﬁed and cloned ( Fig. 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>For that aim , the nucleotide sequences of 311 CRP binding sites from E. coli promoters annotated in RegulonDB were utilized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>Red and green sequences indicate GalR/GalS and CRP binding sites respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Distances between the 5 ' - TGTGA-3 ' sequences of the CRP binding sites ( closed rectangles ) and the 5 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>Despite the lack of similarity in location , CRP binding sites do appear to have sequences in common .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Using restrictionalanalysis to separate , delineate and destroy the various putative CRP bindingsites , all five sites were shown to be functional for CRP binding in witno .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Taken together , these data show that RNAP , CRP and IHF can bind simultaneously at saturating concentrations for their respective speciﬁc binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Tandem CRP binding sites in the deo operon of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_433">
Tandem CRP binding sites in the deo operon of Eseherichia coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></label>
<input id="sit_similarity_433", type="checkbox">
<div class="SIT_similar">
<li>Tandem CRP binding sites in the deo operon of Escherichia coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6315676" target="_blank">6315676</a>]<br></li>
<li>Tandem CRP binding sites in the deo operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li></div>
<li>Three of these sites also correspond to specific binding sites for CRP ( -47 ) and RNA polymerase ( -2 and -34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>In order to test this possibility , we built matrices by sampling random subsets of binding sites for CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>A possible CRP binding of the melibiose promoter is aligned for binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>Underlined residues at positions 7 and 16 at certain sites were mutated to prevent CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>( A ) Sequences of the CRP binding sites in wild-type and mutant clyA clones .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Suggested binding sites for CRP , NAC , PhoB , and NsrR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>A. Sequence analyses of the CRP binding sites of fadL , fadD and fadH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>CRP titration and mutational analysis on secB CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>To locate the binding sites of cAMP-CRP , we performed DNase I protection experiments using puritied CRP protein and a ' ^ ^ P-end-iabeiled DNA fragment that contains rof sequences from - 3 2 to -225 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>By gel retardation and DNasIefootprinting experiments it is shown unambiguously that CytRis a DNA-binding protein that interactsspecifically with sequences located betweenthe two CRP sites in deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>We investigated H1 binding within the lac and gal control regions ; in these regions binding sites for H1 generally overlap the specific DNA binding sites of CRP , RNA polymerase , and lac repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>These data strongly suggest that two separate regions on the DNA helix are recognized by CRP , and this assumption is further supported when the known CRP binding sites are compared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>CRP -- DNA complexes : inducing the A-like form in the binding sites with an extended central spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Here , FIS binds to sites that overlap tandem DNA sites for CRP , probably hindering the binding of the activator during exponential growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The case of Cl and CRP proteins Conclusion In this article , I have described a class of regulatory proteins which are endowed with the ability to ` manipulate ' large regions of DNA by oligomerizing , probably via weak heterologous interactions between protomers , onto DNA binding sites made up of several small asymmetric sequences in direct repeats .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2664421" target="_blank">2664421</a>]<br></li>
<li>The -10 region is boxed , the cAMP-CRP binding sites are indicated by horizontal arrows in opposite directions , and the initiation start sites are boldfaced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>The nucleotide sequences of different cAMP-CRP binding sites show a great deal of homology although their locations with respect to the transcriptional initiation sites are quite variable ( 32 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>The putative binding sites for cAMP-CRP and the experimentally verified binding sites for ArcA within PsdhC are absent from E. coli MG1655ΔPsdh : : Ptet .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>The boxed sequences indicate the binding sites tor CytR and cAMP-CRP as defined in DNase I tootprinting experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The boxed sequences indicate putative binding sites for the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>The sequences of each of the glp regulatory regions and comparison of the positions and sequences of the CRP binding sites are shown in Fig. 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Empirical score distributions distinguish global from speciﬁc TFs The global TFs CRP and FNR have several hundred annotated functional binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>The DNA sequences of the potential CRP binding sites are highlighted by white boxes with the down arrows indicating the base pair substitutions in each of the down mutants used in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The potential ribosome binding sequences ( indicated by SD ) and crp sites ( indicated by CRP ) are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553671" target="_blank">2553671</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553671" target="_blank">2553671</a>]<br></li>
<li>The common regulatory region of the malEp and malKp promoters consists of a compact array of binding sites for both activator proteins , MalT and CRP ( 9 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>The malEp-malKp region is represented to scale with the Pribnow boxes ( hatched boxes ) , the MalT binding sites ( filled arrowed boxes ) and the CRP binding sites ( open boxes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Second , in ours [ 17 ] and Bassler 's previous report [ 16 ] there is significant interplay between CRP and lsr expression , and binding sites in the intergenic region were revealed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21980272" target="_blank">21980272</a>]<br></li>
<li>Footprinting experiments show that ' this region encompassesa seriesof effective binding sites for both MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>( B ) nrf-acs intergenic region , showing the location of binding sites for CRP , FIS , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>( A ) Location of the putative IHF and CRP binding sites in the intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The mutations were mapped upstream of dadA in a region corresponding approximately to the dadR locus of E. coli , i.e. , exactly where two cAMP-CRP binding sites are located .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>CRP functions in the presence of cAMP by binding to speciWc sites located in the control region of CRP-depen-dent promoters [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>DNase I footprint oi the binding sites for CylR and cAMP-CRP in fhe rol regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>A. Putative cAMP-CRP binding sites in the 5 ′ upstream region of patZ gene in E. coli and other Enterobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>The single mismatch operator motif , cATTGGTtatACC / \ / l7T , in the cyo / ^ SCDE operon that encodes the cyfochrome / ̂ r̂ ns-o complex of the aerobic respiratory chain ( Minagawa ef al. , 1990 ) , is situated at -189 to -160 , downstream of a putative ArcA-binding site in a region flaniod by two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>This region comprises two series of MalT binding sites separated by one series of four CRP binding sites ( see Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Although the DNase I footprint analysis identiﬁed two binding sites for CRP , the results from the gel mobility shift assay suggested that CRP binds to a single site in the gcv control region ( Fig. 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Clearly , the necessity for interaction between the c ~ subunit of RNA polymerase , DNA and upstream CRP or FNR dimers at ansB Pe could be influenced by either the nature of the downstream activator , and the ` quality ' of its activating region , and/or the particular DNA sequence between the activator binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Thus , two ` high ' affinity CRP binding sites seem to be pre-sent in the deoP-2 regulatory region , and the targets show considerable homology with other known CRP sites , both with respect to sequence and position , as will be discussed below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>As discussed above , the four CRP binding sites in the rhaSR-rhaBAD intergenic region were predicted on the basis of their DNA sequence and position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>There is also a perfect match in this region between the extent of protection towards the 5 ' end of the upper DNA strand and the register of CRP induced DNasel enhanced cutting sites , suggesting the importance of this DNA sequence for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Lane 2 in figure 3 B shows the ` footprint ' of purified lac repressor in the operator region of the Concomitant binding of CRP to the two binding sites The experiments described above were repeated with DNA fragment A , which contains both CRP interaction sites on one DNA molecule ( fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The contact stabilizes aCTD binding to the DNA segment between the CRP and the rest of RNA polymerase binding sites at the lac promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>The function of H-NS at the csiD promoter would be to counteract this ﬂexibility and thereby improve the precision of activation from the optimal CRP box position , maybe by binding to several sites along this region as a ` local DNA scaffold ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The binding sites of activator CRP are located upstream of the respective promoters , showing a clear periodical pattern of about 10 bp spacing with peak positions starting from 240 ( data not shown ) , in agreement with the direct protein-protein contact model of upstream bound CRP and RNA polymerase on the same DNA surface [ 40 -- 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Restriction sites were found within CRP 4 and CRP 5 , contained within the 276 bp DraI DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>To check the function of the predicted DNA sites for CRP at the gatY , aer , sdaC , ychH and malX regulatory regions , a derivative of each fragment was constructed , carrying substitutions at positions 7 and 16 of the 22 bp CRP-binding sequence ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>In order to identify the putative CRP binding sites , the DNA regions comprising 400 bp upstream and 200 bp downstream the ATG of each gene were scanned , using PATSER .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>l ( c ) , a 168 bp DraI DNA fragment contains CRP 1 and CRP 2 but lacks all other recognized protein binding sites in the mtl operon control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>CRP binding to these sites bends the DNA and causes MalT to shift 3 bp downstream to binding sites from which MalT can activate transcription ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Control DNA fragments were obtained by digesting pFF and pCC with EcoRI and HindIII to release fragments of V300 bp carrying FNR and CRP binding sites , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_488">
BMR assays using CRP and the isolated 276 bp Dral DNA fragment digested with restriction enzymes to destroy the CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></label>
<input id="sit_similarity_488", type="checkbox">
<div class="SIT_similar">
<li>BMR assays using CRP and the isolated 322 bp Sau3Al DNA fragment digested with restriction enzymes to destroy the CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li></div>
<li>We found six putative CRP binding sites , each one located at -244 , -188 , -122 , -117 , -77 , -52 and +22 bp from the ATG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>However , these motifs differ from canonical CRP sites at two key positions ( T > C bp 6 and A > G bp 17 ) , which results in a 100-fold reduction in the binding affinity of CRP ( Cameron and Redﬁeld , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>Based on these results , we first constructed a mutant deoP2 promoter by replacement of the consensus bp at positions 4 in the two CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Regarding this sites we note that its location is suboptimal for CRP-mediated transcription activation ( Gaston et al. , 1990 ) and , furthermore , it contains suboptimal ﬂanking sequences for bending , and thus for binding of CRP ( Gar-tenberg and Crothers , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>Molecular analyses and genetic studies have shown that the cai and ® x promoters share a 289 bp regulatory region , which is located between their transcription start sites and comprises three binding sites with different af ® nities for CRP ( Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Moreover , the putative CRP binding sites found are located at distances from the ATG start codon that are compatible with a regulatory role ( Saier et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>Sequenceanalysis of the promoter region revealed that a 29-bp DNA fragment ( from -48 to -76 ) has 8 of 12 nucleotides identical with the highly conserved base pairs of CAMP-CRP binding sites ( de Crombrugghe et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1712230" target="_blank">1712230</a>]<br></li>
<li>Genes in the competence regulon contain promoters preceded by 22 bp motifs that strongly resemble CRP binding sites ( Redﬁeld et al. , 2005 ; Cameron and Redﬁeld , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>Generality of the isolated DNA sequences One complication in devising a general selection scheme for cAMP -- CRP-dependent CytR-binding sites was that cooperative binding of cAMP -- CRP and CytR had been observed on DNA templates with 52 -- 54 bp separations between the centers of the two CRP targets ( Søgaard-Andersen et al. , 1990 ; Pedersen et al. , 1991 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>This structural complexity , together with the fact that CytR repression at deoP2 has a strict requirement for a spacing of 52 or 53 bp between the two CRP binding sites ( S0gaard-Andersen et al. , 1990b , 1991b ) , has hampered the proposal of a unifying binding model for CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>For thefumA gene of E. colicoding for fumarase and also subject to catabolite repression , similar CRP binding sequences were described , about 100 bp upstream of transcription ( Miles and Guest 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>In addition , DNA fragments having deletions removing sequences upstream of position 109 ( Fig. 2A , lanes 7 and 8 , 11 and 12 , and 13 and 14 ) or sequences downstream of position 60 ( lanes 3 and 4 , 7 and 8 , 11 and 12 , and 13 and 14 ) showed binding to CRP , whereas a deletion upstream of position 74 ( lanes 9 and 10 ) or downstream of position 105 ( lanes 5 and 6 ) resulted in the loss of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>We note , however , that bendability of the ﬂanking sequences are critical for CRP binding ( optimal if the sequences are AT-rich at nucleotides 10/11 and GC-rich at nucleotides 15/16 from the dyad axis ) , and only targets that possess such sequences will show strong interaction to CRP ( Gartenberg and Crothers , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Quantitative analysis of cyclic-AMP induced binding of CRP from Escherchia coli to non-specific and specific DNA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Genomic SELEX screening of CRP-binding sequences was performed by using puriﬁed His-tagged CRP and a collection of E. coli genome DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Using equally stringent search conditions for each factor , we screened the E. coli MG1655 genome for DNA sequences that resembled the DNA-binding sites of RutR , LexA , Fur , FruR , FNR , CRP , NarL and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>Sequence integrity of the second core motif is more important for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>These heptameric sequences are centered at 2109 , 245 , 11 , 119 , and 144 and are suitably positioned to interfere with the binding of FNR , CRP , and RNA polymerase and/or transcript extension .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>BELL , A. I. , K. L. GASTON , J. A. COLE and S. J. W. BUSBY , 1989Cloning of binding sequences forthe Escherichia coli transcription activators , FNR and CRP : location of bases involved in discrimination between FNR and CRPN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Cloning of binding sequences for the E'slberdchia cob transcription activators , FNR and CRP : location of bascs involved in discrimination between FNR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Interestingly , the location and the orientation of the putative CRP binding sequences in the cya gene are in good agreement with those of the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6344011" target="_blank">6344011</a>]<br></li>
<li>Putative IHF and CRP binding sequences are shown by brackets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Based on sequence homology with CRP ( 8 ) , FNR consists of a C-terminal DNA-binding domain , which recognizes specific sequences within FNR-controlled promoters ( 9 ) , and an Nterminal sensory domain containing 5 cysteine residues , 4 of which , Cys-20 , 23 , 29 , and 122 , but not Cys-16 , are essential for function ( 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The heptamers frequently are arranged in tandem orientation with variable spacing rather than as palindromic pairs with fixed spacing which is characteristic for dimeric helix-turn-helix DNA binding proteins such as FNR or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Oligonucleotides used in this study Oligonucleotide Sequence ( 5 -3 ) a Use 744 CGCGGATCCCCACTGGATGCGCCGAGATCG Amplify rha promoter region for subcloning 896 CGCGGATCCTCTATCGCCACGGACGCGTT Amplify pSR or rha promoter region for subcloning 1170 CCGGAATTCTTGTGGTGATGTGATGCTCAC Amplify pSR for subcloning 2168 AGCAAATTCACAACATCATCAC CRP site 1 mutagenic oligonucleotide 2184 CGCCAGGGTTTTCCCAGTCACGA Sequencing lacZ fusions ( IRD41 labeled ) 2198 TTCAGGAAATGCGGTGAGCATCACAATGCCACAATTCAGC CRP site 4 mutagenic oligonucleotide 2239 ACAGCGTGAATTTTCAGGAAATGTGATGAGCATCACATC CRP site 3 up mutagenic oligonucleotide 2240 GCAAATTGTGAACATCATCACATTCATCTTTC CRP site 1 up mutagenic oligonucleotide 2249 GCGCCGCGCATATGGCTTTCTGCAATAACG Downstream oligonucleotide to amplify rhaR for cloning into pTYB1 2251 CTCGTCGCTCTTCGGCAGGCATCTTTCTGCGAATTGAG Upstream oligonucleotide to amplify rhaR for cloning into pTYB1 2371 CGCGGATCCCCACTGGATGCGCCGAGATCG PCR for DNase footprinting template 2409 GGTAAGATCTAAAAAAATCCACACTATGTAATACGGTCAT PCR for DNase footprinting template a Regions of oligonucleotides not complementary to wild-type DNA sequence ( encode restriction endonuclease sites and ﬂanking DNA or mutations ) are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The CRP-2 site , the spacer region between the two CRP targets , and CRP-1 were deleted stepwise in pJEL134 ( Fig. 3 ) to define the sequence information required for regulated expression of deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_516">
The CRP target site is a palindromic sequence , where two conserved core motifs , having the 5-bp sequence TGTGA , are separated by a 6-bp spacer ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></label>
<input id="sit_similarity_516", type="checkbox">
<div class="SIT_similar">
<li>The CRP target site is a palindromic sequence , where two conserved core motifs , having the 5-bp sequence TGTGA , are separated by a 6-bp spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>The CRP target site is a palindromic sequence in which two conserved core motifs , having the 5-bp sequence TGTGA , are separated by a 6-bp spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li></div>
<li>The originally randomized portion is indicated by thick underlining ; the ﬂanking half-site recognition motifs of the CRP targets are indicated by thin underlining ( the centers of CRP2 and CRP1 are 3 bp upstream and downstream , respectively , of the sequences shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The centers of the ﬂanking CRP targets are 3 bp upstream and downstream from the shown sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Based on sequence similarities to the deo P2 CRP sites , two putative CRP sites may be identified in the nupG gene , at -40 bp and at -90 bp , respectively ( underlined in Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>In addition , it should be noted that four other putative half sites for FNR were detected more upstream in the sequence , as was one half site for CRP at 4 bp from the Shine-Dalgarno sequence of caiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>In these derivatives , CRP-2 and parts of the region between the two CRP sites have been deleted and in all the plasmids the deo sequence is flanked on the upstream side by the sequence 5 ' - GAATTCCC-3 ' coding for a unique EcoRI site and by a BamHI site on the downstream side .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Close inspection of the DNA sequence around the CRP-2 site reveals that this region might contain two overlapping CRP sites , CRP-2 and CRP-3 , ( Figure lB and C ) that are displaced by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The particular mutations introduced into the upstream site in XX-CC allow the targeting of CRP containing an EV181 change to this site , since CRP EV181 , but not wild-type CRP , is able to bind at such sites ( Ebright et aL , 1987 ; Gunasekera et aL , 1992 ; and see Zhou et aL , FF-CC ansB Pe - ¬ Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The tandem CRP sites in different promoters are separated by DNA of variable length and sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The Eco RI 203 base pair DNA fragment is shown on top with the relevant restriction enzyme cutting sites , the location of CRP site 1 and the lac operator sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The fourth column includes sequence logos for presumably homogeneous and large site sets and sequence consensi for small sets of DNA sites and for well-established motifs of other factors ( FNR , CRP , CooA , NtcA , ArcR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Oligonucleotide primers Name Sequence ( sites 1 and 1 and sites 2 and 2 ) separated by a DNA site for CRP ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>E.coli CRP binds as a homodimer , speciﬁcally to symme-trical 22 bp DNA sites with the consensus half site 50-A1A2A3T4G5T6G7A8T9C10T11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>The sites where the aCTDs are proposed to contact the DNA ( 6 bp sequences centred 18 -- 19 bp from the centre of the DNA site for CRP ( Benoff et al. , 2002 ) ) are indicated by the solid boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The horizontal shaded boxes denote 22 bp DNA sites for CRP : one site centred at position 2195.5 is responsible for the activation of the melR promoter , while the other , centred at position 281.5 , is discussed in this paper .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The shaded boxes denote 22 bp DNA sites for CRP : one site centered at position 241.5 is responsible for the activation of the melR promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760179" target="_blank">10760179</a>]<br></li>
<li>0002-8080 G 2005 SGM Printed in Great Britain Silencing involves two sites ( Dole et al. , 2004b ) : H-NS represses transcription initiation at the cAMP receptor protein ( CRP ) - dependent promoter by binding to an AT-rich and presumably bent upstream silencer sequence ( Schnetz , 1995 ; Singh et al. , 1995 ; Schnetz & Wang , 1996 ; Mukerji & Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>To ascertain whether the FF-FF promoter was co-dependent on FNR dimers binding at both the upstream and downstream sites ( as in the case of CRP and FNR in CC-FF ) or whether the upstream dimer acted synergistically ( as in CC-CC in ansB Ps ) , the FNR binding sites in FF-FF were modified by the introduction of mutations at nucleotide positions 7 and 16 in the 22 bp binding sequence to give the YY-FF and FF-YY promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Transfer of the target sequence for FNR to a CRP - dependent promoter Recent studies by Guest and coworkers have clearly shown that the DNA binding domains of the FNR protein and CRP are homologous and , therefore , likely to recognize similar sites ( Shaw et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>MelR-dependent activation of the melAB promoter requires the binding of MelR subunits to four 18 bp sites and the binding of a single CRP dimer to a low-afﬁnity target sequence ( Belyaeva et al. , 2000 ; Wade et al. , 2001 ; see Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Q 1998 Blackwell Science Lt Results The rpoH promoter region contains putative binding targets for CytR and CRP By inspection of the rpoH promoter region , a second , putative CRP target ( CRP-2 ) is found 53 bp upstream the CRP binding target suggested by Nagai et al. ( 1990 ) ( CRP-1 , Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>CaiF and CRP bind co-operatively to the ® x promoter The identi ® ed F1 and F2 binding sites of CaiF ( Fig. 2B ) overlap the previously reported low-af ® nity CRP2 and CRP3 binding sites of CRP ( Buchet et al. , 1998 ) ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>( B ) Sequence of the proP P1 promoter with the match to the CRP consensus noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>CRP functions as a dimer in the form of a CRP -- cAMP complex , and regulates transcription initiation by binding to a symmetrical DNA sequence ( consensus sequence 50-AAATGTGATCTAGATCACATTT-30 ) , located near or within the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP modulates transcription initiation by RNA polymerase ( RNAP ) by binding to a 22-base consensus target DNA sequence contained in the gene promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>Since the clones responded to multiple stresses ( Table 2 ) , sequences homologous to the o `` 32 promoter [ 19 ] , cAMP-CRP binding [ 36 ] , cr54 promoter sequence [ 37 ] , gearbox promoter [ 8 ] , or -10 hexamer ( TATACT ) , possible ~ - binding consensus sequence [ 31 ] , were also explored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8647276" target="_blank">8647276</a>]<br></li>
<li>Meanwhile , our ﬁndings that the three iscS-dependent genes commonly contain the CRP-binding consensus sequences in their promoter regions provide a potential link between iscS and gene regulation via CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>Other consensus sequences for CRP binding have been suggested ( 18,19 ) and these sequences , differing by three nucleotides or less , appear in the proposed CRP binding region in the ilvB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Although we have argued that some element in the sequence from - 4 5 to - 5 6 is necessary for stable CRP binding , we can not exclude the possibility that RNA polymerase also makes contacts in this zone that are essential to stabilize the open complex at P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>Genes that are subject to catabolite repression are activated by the complex formed between cyclic AMP and its receptor protein ( CRP ) , which interacts with a conserved sequence of nucleotides close to the corresponding RNA polymerase binding site ( Valentin-Hansen , 1982 ; Chapon & Kolb , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6383359" target="_blank">6383359</a>]<br></li>
<li>We then examined the binding of the cAMP-CRP complex with four sequential fragments spanning the P2hfq and P3hfq promoter region ( sequence 429 to 68 ) ( Fig. 3A , top ) and found that the sequence from 151 to 32 was sufﬁcient for the binding ( Fig. 3A , bottom ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>( A ) ( Top ) Four DNA fragments of the P3hfq promoter region ranging from sequence position 429 to 68 were used to narrow down the potential binding region of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>A fragment ( 115 to 78 ) was synthesized within the proposed sequence ( 104 to 89 ) for EMSA to test the binding of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>The identiﬁed bound region ( sequence 104 to 89 ) for the cAMP-CRP complex appears to be speciﬁc and essential , as mutation of this region abolishes the binding of the complex in an EMSA and DNase I footprinting assay ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>The minimal deoP2 region which still causes derepression encodes an 83bp fragment extending from position -34 to -116 ( p13-134AT ; Fig. 1 ) and thus encompasses both binding sites for the cAMP-CRP complex ( the right-hand side of CRP-1 is substituted with vector sequence but the core binding site for CRP -LCB- ATTGTGA-A/e-TCGA ) is unchanged ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>From the cAMP-CRP-binding sequences identiﬁed by using a number of known and novel CRP target promoters , we identiﬁed the palindromic sequence TGTGA-N6-TCACA as the CRP box ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Likewise , the collection of next abundant 38 clones carried the intergenic sequence between yibI and mltA , both being located downstream of the cAMP-CRP binding sequence , of which the mtlA operon encoding mannitol-specific PTS permease is known to be activated by cAMP-CRP [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>A putative cAMP-CRP binding sequence ( TAATATGACCAACCTCTCATA at positions 157 -- 177 ) homologous to the canonical sequence ( Ebright et al. , 1989 ) was detected 24 bp upstream of the − 35 sequence of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>These results are consistent with the existence of the putative cAMP-CRP binding sequence upstream from P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>CRP binding sites contain a palindromic sequence in which two conserved motifs , TGTGA and TCACA , are separated by a spacer ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>( g ) Presenceof four CRP-binding sites in the intergenic region The CRP protein is presumed to activate transcription initiation at malEp and malKp by binding to one or several of the four putative CRP-binding sites revealed by sequence comparison ( see Fig. 1 ) ( Bedouelle , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>The filled arrowed and open boxes , which indicate the positions of the MalT and CRP binding sites , respectively , are numbered as shown on the schematic representation of the wdEp-malKp region ( for the sequence , see Raibaud et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>The positions of the transcriptional initiation point ( 11 ) and putative regulatory elements , i.e. , binding sites for CRP and FNR , Shine-Dalgarno ( S.D. ) sequence , and 210 and 235 boxes are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The deoP2 promoter sequence ( ± 105 to ± 25 ) is 5 cents - AATTATTTGAACCAGATCGCATTA-CAGTGATGCAACTTGTAAGTAGATTTCCTTAATTGTGATGTG - TATCGAAGTGTGTTG-3 cents , in which the two cAMP ± CRP sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The 16-bp sequence in the tsx-pl promoter showing homology to known DeoR operator sites and two regions in the tsX-p2 promoter with homology to cAMP-CRP DNA-binding sites are indicated ; lines extending from the cAMP-CRP targets indicates those sequences which are protected by cAMP-CRP against DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The deo-p2 , cdd , and cytR promoter regions have been shown directly to contain two binding sites for cAMP-CRP ( 13 , 36 , 38 ) , and sequence analyses suggest that also the tSx-p2 , udp , and nupG promoters contain tandem targets for the activator ( 4 , 20 , 23 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The mode of action of CytR has been puzzling because the only obvious sequence homology between the different promoter regions is the presence of tandem binding sites for the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The inverted-repeat sequence known to be important for CRP binding is in capital letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The sequence homologous to the CRP binding motif is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>The binding sites for the cAMP/CRP compiex and the Pribnow box-like sequence are underiined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>The wavy underline shows the sequence for cAMP - CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>The − 10 promoter sequence and the cAMP -- CRP binding motif are blocked in gray .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>The presence of a 210 promoter box ( TATGA AT ) centered at 29 , a 235 sequence ( TTGACG ) centered at 232.5 , and binding sites for CRP and FNR centered at 261.5 suggested that this clyA promoter is a class I promoter that could be transcriptionally activated by CRP or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The PATSER software also identiﬁes the CRP binding sites in the sequence of the major promoter of acs ( acsP2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>The three promoters that have been analyzed most extensively ( deoP2 , cddP , and tsxP ) each contain two binding sites for CRP ( CRP1 and CRP2 ) but are otherwise dissimilar in sequence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The minimal CRP-responsive promoter sequence was identiﬁed and consists of two proximal CRP binding sites ( CRP I and CRP II ) located at optimal distance from the -10 box ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>CRP-binding sites identiﬁed within all the r-protein genes include the CRP box sequence ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The AscG-binding sites identiﬁed herein are similar to the CRP box sequence ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Although the nucleotide sequence of the -35 region ( GCTTTTCA ) was not very typical , it was very similar to that of sequence for CRP binding proposed by O'Neill - e-t al. ( 15 ) is shown in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>Each protected region contains a sequence ( Fig. 2 ) which is similar to the consensus cAMP-CRP binding sequence ( Kolb et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Scru-tiny of the sequence upstream from the -35 sequence revealed the presence of a sequence ( nucleotides 392 through 412 , Fig. 2 ) that is similar to the consensus sequence ( 7 ) for binding of CRP : AA-TGTGA ------ TCACA-TT ( consensus ) , TAATGTTATACATATCACTCTA ( nucleotides 392 through 413 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>Moreover , a CRP consensus sequence was identiﬁed in the upstream regulatory region ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>Experimental data for binding of CRP to DNA in vitro has shown that a correlation exists between the measured ratio of specific to nonspecific binding and the calculated XE value for a sequence ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>A. CRP binding to fragment C ( fig. 1 ) ; B. CRP binding to fragment E ; C. DNasel digestion and Maxam-Gilbert sequencing reaction products ( 6 ) run in parallel to obtain the sequence assignments shown in B ( this fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>A motif sequence identiﬁed in seven IGRs was similar to the binding sequence of CRP ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>Binding of CAMP-CRP to the cyu Regulatory Region-In the previous work ( 16 ) , we demonstrated that thereexist two sites containing the consensus CRP binding sequence 5 ' - AA-TGTGA -- T -- TCA-ATT-3 ' ( 12,27 ) in the cya regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>We also looked for the presence of possible cAMP / - cAMP receptor protein ( CRP ) binding sites on the basis of the consensus sequence AAATGTGATCTA GATCACATTT proposed for the most frequently observed cAMP-CRP binding sites ( 3 ) ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>( C ) Comparison of the DNA sequence of the gapA P3 promoter with that of the gal P1 promoter ( 33 ) and with the consensus sequence established for the cAMP-CRP binding sites ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>Indeed , there is a sequence of hyphenated dyad symmetry centered at y59 .5 upstream of Pgl pG that weakly resembles the CRP or FNR consensus binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>A similar number of potential binding sites was noted in the control region of the glucitol @it ) operon although deviation from the CRP consensus sequence was more pronounced ( Yamada & Saier , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Two potential CRP binding sites are present ( positions 166-195 ) , based on a consensus sequence proposed by Chapon and Kolb ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6328431" target="_blank">6328431</a>]<br></li>
<li>The effects of 11 mutations within the DNA region thought to bind the cyclic AMP receptor protein correlate well with the CRP consensus binding sequence and confirm that this region is responsible for cyclic AMP regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>The deletion analysis , described here , shows that a region upstream from -45 , where there is little correspondence with any consensus sequence , is also important for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>The protected region contains a sequence , 5 ' - CGTGA-N6-CAACA-3 ' , where N6 indicates a stretch of six nucleotides , which is similar to the consensus cyclic AMP-CRP binding sequence derived by a number of investigators ( 9 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Transcription of both genes might be regulated by CRP , as glucose effectively inactivated transcription and there was a sequence similar to the consensus for CRP binding in the intergenic region of nagE and nagB ; nagA is probably co-transcribed with nagB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3284790" target="_blank">3284790</a>]<br></li>
<li>DNase I protectionexperiments revealed thatCAMP-CRP binds to the cya DNA region between +11 and -20 , in which a consensus CRP binding sequence is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>The high degree of sequence homo-logy strongly suggests an evolutionary relationship between deoCRP-1 deoCRP-2 T T A A T T G T G A T G T G T A T C G A A G T G T G A A T T A T T T G A A C C A G AT C G C A T T A C A cytR -149 , -124 SyLtR -130 , -105 SjtR - 98 , -73 C A C G A T G C G A G G C G G A T C G A A A A A T T A A A A A T T C G A T A T T C A T C A C A C T T T T A A T T C T G T A A C C G T T T T C A C G C G C T A N6 Consensus T G T G A T C A C A Fig. 4 DNA sequences in front of the cytR structural gene which shows homology to the consensus sequences of CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The FNR protein binding sequence is quite similar to the core consensus for CRP protein binding , TGTGA-N6-TCACA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The nucleotide sequences of CRP sites I and II are shown along with the consensus CRP-binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The consensus sequence for CRP binding sites reported by Ebright et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>The CRP binding sequence agrees well with the proposed CRP consensus sequences ( Ebright , 1982 ; Queen & Rosenberg , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6319708" target="_blank">6319708</a>]<br></li>
<li>The effects of the mutations are consistent with the consensus sequence for CRP binding ( 5 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>These include a typical -10 sequence , a -35 sequence , and a sequence similar to the consensus sequence for binding of the CRP ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_601">
CRP binding consensus sequence comparison and DNA fragments for CRP binding assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></label>
<input id="sit_similarity_601", type="checkbox">
<div class="SIT_similar">
<li>( A ) CRP consensus sequence and DNA fragments used for the CRP binding assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li></div>
<li>It must be noted , however , that this sequence departs from the consensus at a position known to be important for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>A perfect consensus sequence for the binding of CRP is shown by an oval .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Moreover , the consensus sequence ( TGTGA ) near 244 bp does not contribute or function , even in the absence of the real CRP binding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Shown above the main sequence are the consensus sequences for CRP binding [ 2 ] , the UP element [ 3 ] , PurR [ 4 ] , DnaA [ 5 ] and the discriminator [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>This result is in agreement with the observation , noted in Fig. 2 , that CRP 4 and CRP 5 exhibit sequences which closely approximate the CRP binding consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>This consensus sequence includes and extends the model for the CRP binding site proposed by O'Neill et al. ( 1981 ) in which it is postulated that the two subunits of the CRP dimer recognize a core of two inverted-repeats , which overlap each other ; the core 5 ' - TGTCACA-3 ' 3 ' - ACAGTGT-5 ' The two CRP binding sites in deo are in nearly perfect agreement with the 10-bp consensus proposed by Ebright , i.e. , a fit of 9 out of 10 for the proximal site ( from - 51 to - 35 ) and 8 out of 10 for the distal site ( from -83 to -99 ) ( see Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>The consensus sequence for FNR binding is very similar to that for CRP , differing at just two symmetrically related positions in the inverted-repeat ; appropriate changes at these positions can result in interconversion of CRP and FNR sites ( Bell et aL , 1989 ; Zhang and Ebright , 1990 ; Jennings and Beacham , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Consistently , the protected glycines are all located in the consensus part of the CRP sites at positions that are important for binding of CAMP-CRP ( see Figure 1B ) ( de Crombrugghe et al. , 1984 ; Schultz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In their survey of 15 CRP binding sites de Crombrugghe et al. ( 1984 ) showed that the nucle-otide sequence 5 ' TGTGA3 ' is the most conserved part ofthe consensus and that most sites contain this sequence element : indeed Weber and Steitz ( 1984 ) and Ebright et al. ( 1984 ) have shown that this sequence contains the bases that make the strongest contacts with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Thirteen independently isolated mutations that stopped CRP binding all fell in the sequence 5 ' T-C-A-C-A 3 ' between - 3 8 and -34 , a sequence that matches well with the consensus sequences proposed for CRP binding sites ( O'Neill et al. , 1981 ; Queen & Rosenberg , 1981b ; Ebright , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>Sequences sharing homology to consensus binding targets for CytR and CRP ( underlined by arrows ) are indicated in bold letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The binding sequences of cAMP-CRP were subjected to the Logo analysis for determinatin of the consensus sequences for the following samples : ( A ) the whole set of CRP targets ( total 323 sequences ) identified by Genomic SELEX screening in this study ; ( B ) the set of CRP targets ( 165 sequences ) that are listed in Regulon DB and identified by Genomic SELEX ; ( C ) the set of CRP targets ( 129 sequences ) that are listed in Regulon DB but not identified by Genomic SELEX ; ( D ) the whole set of CRP targets ( 294 sequences ) listed in Regulon DB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The inverted arrows denote the conserved CRP binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>The numbers of nucleotides conserved within the five CRP binding sequences are indicated on the right ( bold figures , conserved at positions 4-8 ; normal figures , conserved a t positions 1-3 and 9-1 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The consensus sequences proposed for CRP binding sites and Fnr binding sites respectively differ by only two nucleotides ( Bell et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>The consensus sequences for CRP and Es28 binding ( Busby and Ebright , 1999 ; Koo et al. , 2009 ) are shown below the alignment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The boxed sequences indicate the twoCRP sites , and the arrows below the boxes indicate the sequences sharing homology with the consensus CRP sites ( deCrombruggheet al. , 1984 ) ; thetwoarrowsin theCytR-binding site indicate the CytR recognition sequence ( Rasmussen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The boxed sequences indicate the two CRP sites and underlined bold letters indicate the sequences sharing homology with the consensus CytR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The consensus sequences of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The binding proﬁle of CRP was constructed based on similarity to the consensus sequences inferred from known CRP binding sites using RegulonDB , version 6.4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>The consensus of the candidate HcpR binding sites , wTGTGAnnnnnnTCACAw , is similar to the CRP consensus of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The CRP sites presenw t ithin the control regions of the glp operons differ with respect to their positions and degree of similarity with the consensus CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>The estimated CRP binding afﬁnity of secB P2 ( 0.8 3 1029 M is also comparable to that of lacP1 ( 0.328 3 1029 M [ 45 ] ) , even though the secB site deviated more than lacP1 from the well-deﬁned consensus sequences of CRP binding sites ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>Interestingly , many of these putative CRP binding sites were also found in the 5 ′ regions of the yﬁQ gene of many other Enterobacteria species , although those of Salmonella and , especially , Shigella resembled more the ones found in E. coli ( both in terms of sequence conservation and position relative to the ATG start codon ) ( Supporting information ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>P3 TCAACT B. Changes in the CRP binding sites Sequence and mutagenesis AA-TGTGAÐÐTCACA-TT Consensus sequence Site 1 GT - AGATGTGAGCCAGCTCACCATA AC - - Site 2 AGATGTGATTAGATTATTATTC Fig. 5A , B E $ ect of mutations in the CRP1 and CRP2 sites on dad expression in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Oligonucleotide primers used to construct probes for CRP binding assays Primer Probe ( s ) a Relative descriptionb Sequence of oligonucleotide ( 59 to 39 ) c ( 1174 ) TGGAAAACGTGCGGCGCGTTCGGC ( 2134 ) ATGGCAACGCCGCCAAGCGTGAAG ( 289 ) GCACCACGGTTCCCCAGATTTTTATT ( 1296 ) ACACAGGAACCCGGGTTCTTCGCCCGGC ( 258 ) ACAGCACATTGGCGGCTGTGATGACTTGTA ( 271 ) GATCTTTCTTGACGCACGGCGCGTTGGCGGCTG ( 276 ) CTAGAGATTTTTATTGACGCACAGCACAT wt secB TS Pr-46 , -47 , -102 , -78 , -80 Pr-46 Pr-47 Pr-60 Pr-60 Pr-BCRP-M Pr-BCRP-W 42 46 47 56 60 78 80 wt secB upstream CS wt secB upstream CS wt secB TS wt secB upstream CS BCRP mutation CS BCRP wt CS a W , wild type ; M , mutations ; BCRP , secB P2 CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>These results indicated that cAMP-CRP binds speciﬁcally to the site centered at 271 but not to the hypothetical site at 213 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>To obtain binding equations for the two reduced-valence operators , CRP1 and CRP2 , configurations in which CRP ( cAMP ) 1 is -- bound to a mutated site were excluded from the summations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_632">
For reduced valence , operators CRP12 and CRP22 , configurations in which CRP ( cAMP ) 1 is bound to the mutated site were excluded from the summation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="sit_similarity_632", type="checkbox">
<div class="SIT_similar">
<li>For reduced valency , CRP12 and CRP22 operators , configurations in which CRP ( cAMP ) 1 is bound to the mutated site were excluded from the summation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li></div>
<li>CRP is bound at ÿ121 .5 with the aICTD ( b-associ-ated ) bound to the DNA on the downstream side of CRP .29 Fis is bound to a site at ÿ41 , overlapping the ÿ35 DNA recognition element for s38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Additionally , as in araCBdD , CRP binds adjacent to the promoterproximal operator site within the DNA loop both in lac and dro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>These results suggest that CRP 1 , CRP 2 and CRP 3 must be localized to one side of the DNA helix , each separated from the preceding or following site by close to four turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_637">
Mutagenic oligonucleotides , 30 nucleotides in length , produced symmetric transitions of G ( indicted by underline ) to A in both TGTGA CRP recognition motifs of the mutated site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="sit_similarity_637", type="checkbox">
<div class="SIT_similar">
<li>Mutagenic oligonucleotides , 30 nucleotides in length , were designed to produce symmetric G to A transitions in both TGTGA , CRP recognition motifs of the mutated site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li></div>
<li>We have previously shown that sequences located between positions 133 and 100 , which include the putative CRP site , are required for GRDC of PguaB ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In each case , wild-type PguaB sequences ﬂanking the inactivated half-site and the repositioned CRP site were not altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>DNase I footprinting was employed to determine whether the putative CRP site located at position 117.5 is bound by puriﬁed CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In addition , cleavage of a number of nucleotides within the site was enhanced with added cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>TTAGTGAGGCATAAATCACATTACGCAACGATAATAGCGGGTATAAGATAAATAAAAGGTAAAACGTTTTATCTGTCACATAATCAGGGA CRP site 1 -- 35 -- 10 malI +1 CRP site 2 p46T p40G p11G +40 +70 +100 ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>Tn the lac system , for instance , the CRP site is approximately 60 nucleotides from the start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>CRP bound to this duplicate site seemed not to interact with CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>GATTATGTGACAGATAAAACGTTTTACCTTTTATTTATCTTATACCCGCTATTATCGTTGCGTAATGTGATTTATGCCTCACTAAAATTT CRP site 2 CRP site 1 1 Ecocyc + -- 20 +1 +20 +40 HindIII ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>CRP site Pos .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>hc AGGAAACAGCTATGACC -- 3 ' CRP b & g site operator riq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>IHF and CRP Behave as Transcriptional Activators of the Pa and Pz Promoters -- Analysis of the paaZ-paaA intergenic region revealed the existence of two sequences , TATCAACTACTAA and GAGTGCGAAATCTGTCACAGTT , from positions 2125 to 2113 and 273 to 252 relative to the major transcription start site of paaA ( Fig. 4 ) , that significantly match the consensus sequences WATCAANNNNTTR ( where W is A or T , R is A or G , and N is any of the four bases ) and AANTGT ¬ FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The next-best matches to the CRP consensus sequence are two overlapping sequences with eight and seven consensus base pairs centered at 2111.5 and 2116.5 relative to the rhaSR transcription start site , respectively ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Again a stretch of 24 bases is protected against DNasel attack , defining a CRP interaction site covering the lac operator sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>A strictly cAMP-dependent CRP interaction site was discovered , which covers the lac operator sequence ( fig. 2 B , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>These experiments reveal a second CRP interaction site extending over the lac operator sequence in addition to the CRP interaction site in the -60 `` base pair region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Two distinct , cAMP-dependent CRP interaction sites were thus defined , one in the -60 base pair region ( CRP site 1 ) , and a second one covering the lac operator sequence ( CRP site 2 , fig. 2 A , B ; 3 A , B and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>( iii ) There is not a potential CRP site at the position of the CRP site 2 in Salmonella enterica serovar Typhimurium ( 20 ) ( although there are CRP sites at the positions of sites 1 and 3 , each with the same number of matches to the CRP consensus sequence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>An E. coli site with signiﬁcant homology to the CRP consensus sequence can be found at the same position as the S. enterica serovar Typhimurium site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>Interestingly , the mutations creating a consensus sequence at CRP site 3 had a twofold negative effect on rhaBAD expression , while the CRP site 3 down mutation had a small positive effect on rhaBAD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>CRP site 2 is a very weak match to the CRP consensus sequence that we would not have considered had it not been positioned symmetrically with CRP site 1 at rhaBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>While the substitutions changing CRP site 1 to the consensus sequence at the 10 most conserved positions resulted in a small increase in rhaBAD expression , a similar increase in expression was observed in the absence of CRP , suggesting that the increase in expression may be due to the DNA sequence change and not a change in CRP occupancy of the site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>These results suggest that the degree of CRP occupancy of the wild-type site 1 , which matches the consensus sequence at 9 out of 10 positions , does not limit rhaBAD expression , whereas the degree of occupancy of CRP site 3 does normally limit rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>These results support the existence of a site ( CRP I ) centered at 69.5 , as predicted by sequence analysis ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Sequence inspection revealed a site centered at position 122.5 that resembled the consensus CRP sequence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>d Sequences corresponding to the original CRP upstream half-site in PguaB ( CRP 106.5 ) ( positions 125 to 121 ) or the original CRP downstream half-site in PguaB ( CRP 128.5 ) ( positions 114 to 110 ) were inactivated by substitution with the sequence 5 - GACTG-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Bold face letters indicate matches between the sequence of the CRP site 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>For the footprints around the CRP site , we used a primer with the sequence 5 ′ - GCGCTTCGTAATTAATGGTT-3 ′ hybridizing to positions +15 to -4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The 309-bp AvaII-Sspl fragment , containing the site which best matches the consensus sequence , bound the complex at a CRP concentration of as low as 1 Binding of the complex to the 152-bp Sspl-BstEII fragment containing the site less similar to the consensus sequence was readily detectable at a concentration of 25 nM CRP or higher .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>The consensus sequence for FNR is similar to that for CRP consensus , and previous studies have shown that FNR activates transcription initiation in a similar way to CRP and that both proteins can bind to the DNA site for the other protein ( 41,42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The corresponding sequence is present in CRP site 1 from base pair -73 to -63 and as an inverted-repeat from base pair -50 to -60 ( fig. 6 A , matches are shown in bold face letters ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>Taniguchi and coworkers ( 22 ) derived a consensus sequence by comparing the CRP interaction sequences of the lac ( CRP site 1 ) > gal ( 22 ) and the ara ( 23 ) operons : 5 ` A-A-A-G-T-G-T-G-A-C-A 31 The underlined region of the consensus sequence above is thought to be the most important for the CRP interactions ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>( B ) The 22-base consensus CRP target site sequence and the two CRP target sites in PcspD are aligned against each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>The homologous sequence in CRP site 2 is : 5 ' g-A-A-t-T-G-T-G-A-g-c 3 ' The capital letters represent base matches with the consensus sequence shown above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>( B ) The 22-base consensus CRP target site sequence and the CRP target site in Plac and PcspE are aligned against each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The DNA sequence of this region revealed that the structural gene of CRP starts just after the Hind III site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>The underlined portion of this sequence , which is thought to be most important for CRP-DNA interactions , appears without mismatches in the IlvB CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>The CRP site at position -84 partially overlapped the inverted-repeat sequence between positions -80 and -68 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>The sequence of the CRP site is indicated , and above it the arrowheads indicate the nucleotides changed by site-directed mutagenesis in the CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>A variety of weaker matches to the CRP consensus site can also be found in the rhaBAD-rhaSR intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>We constructed a longer rhaS-lacZ fusion that included the entire rhaSR-rhaBAD intergenic region [ F ( rhaS-lacZ ) D309 ] and used site-directed mutagenesis to change three consensus base pairs in each CRP site 2 and CRP site 3 to nonconsensus base pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>At the hlyE promoter ( PhlyE ) , this site ( TTTGATATTTATC ATA ) most closely resembles an FNR site ( 9 of 10 nucleotides match the FNR consensus , TTGATNNNNATCAA , compared to 8 of 10 matches to the CRP consensus , TGTGANNNN NNTCACA ; discriminatory bases are underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Each of these CRP site `` down '' mutations consisted of three simultaneous base pair substitutions that changed consensus base pairs to nonconsensus base pairs ( 5 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The yjcB and ycfR operons contain a CRP site almost identical to the consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The construct pMWK10 contains an altered CRP site ( 59-AAATGTG ATCTAGATCACATTT-39 ) that perfectly matches the consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>This possibility was tested in a set of constructs in which the distance between CRP-1 and a synthetic consensus CRP site was varied systematically between 37 and 54bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>This site contains only a few bases in common with a consensus CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>P2 lacks a good consensus UP element proximal to the CRP I site ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Nucleotides conserved with respect to the CRP consensus site are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>At least for cAMP-CRP , this seems highly likely since potential CRP boxes with strong homology to the consensus ( 4 ) overlap with the - 10 region and the transcriptional start site , a location indicating a repressor function ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The transcriptional start site ( indicated by xxx in the upper part of the figure and by the heavy arrow in the lower part ) and the most likely - 10 region are indicated , as well as regions with homology to the Lrp , CRP , and IHF consensus sites and a motif with the characteristics of curved DNA and palindromic symmetry ( thin arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The -10 region , ttie MalT sites and the CRP site are symbolizecJ by an open box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7830577" target="_blank">7830577</a>]<br></li>
<li>The region protected by CAMP-CRPcorresponds well to thepredicted CRP site A mentioned above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>The centre of CRP 5 is 58.5 bp from the transcriptional start site or 23.5 bp from the end of the -35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_692">
g The position of 50 end of a CRP site relative to corresponding translation start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="sit_similarity_692", type="checkbox">
<div class="SIT_similar">
<li>The position of 50 end of a CRP site relative to corresponding translation start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li></div>
<li>The aCTD binds to its target upstream of the CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The conserved core motifs in the CRP site are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>The CAP ( = CRP = cAMP receptor protein = catabolite activator protein ) binds , in the presence of cAMP , to a site adjacent to the distal NagC site ( Fig. 2 , Lanes 2 and 11 ) centred at -83.5 from the transcriptional start site although protection extends back as far as -100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The regions protected by cAMP-CRP are marked , and the sequences are numbered with respect to the deoCp2 RNA start site ( +1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>The promoter structure of uidA resembles that of a promoter activated by cAMP receptor protein ( CRP ) ; putative control regions are located at positions - 1 0 and - 3 5 ( relative to the transcription start site ) , are separated by more than 17 bp and exhibit poor homology with the normally recognized consensus sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>The following points should be noted : ( i ) neither the integrity of the CRP target , the CytR operator halfsites nor the sequence separating the CRP target and the promoter downstream CytR operator half-site have been altered in the generated promoters ; ( ii ) signiﬁcant similarity to CytR operator half-sites was avoided in the inserted spacer sequences ; ( iii ) primer extension revealed a single start site for transcription initiation ( þ1 ) and all the derived promoters exhibited a similar activation by the cAMP -- CRP complex ( < 100-fold ) ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>DNase I footprint of cAMP-CRP boundto nupG.The final concentrationsof CRP are indicatedbelow each lane in ~ gml-1 ; the regionsprotectedby cAMP-CRP are indicated.The sequence is numberedwith respectto the start site of transcription ( +1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The promoter region of csiD between the cAMP-CRP box ( ﬁrst boxed sequence ) and the transcriptional start site ( 11 , shown in bold ) is shown for constructs carrying the cAMP-CRP box at different locations ( for details of construction , see Experimental procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>287 Flanking genesa Locationb Sequencec nadC and ampD 118698 -- 118719 ATATGTGGTGCTAATACCCGGT yaeP and yaeQ 214196 -- 214217 GAGTGTGGTATAGTCACCTTGC ybiS and ybiT 855062 -- 855083 AAATGTGATTTCGTACACATCT ycfQ and bhsA 1168209 -- 1168230 GTATGTGATCCAGATCACATCT yeiW and yeiP 2263384 -- 2263405 GCAAGTGGTATTCGCACTTTTG aegA and narQ 2583588 -- 2583609 ATGAGTGTTGTTATTACCCGAC aaeX and aaeR 3387505 -- 3387526 AAAAGTGATTTAGATCACATAA Consensus AAATGTGATCTAGATCACATTT a Consensus CRP binding site ( 3 ) is also shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>Particularly from the fact that both the ompA and gal Pz promoters have the same octanucleotide sequence , T-C-A-C-A-C-T-T in their RNA polymerase recognition site ( -35 region ) , it became relevant to find out if the CAMP receptor protein ( CRP ) binds to this region of the ompA gene to influence the cellfree transcription of the ompA gene [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7021177" target="_blank">7021177</a>]<br></li>
<li>The mechanisms underlying the activation of transcription initiation at specific promoters are far from being elucidated , not only in eukaryotes but also in a simple organism such as Escherichia coli For prokaryotes , a simple model has emerged from extensive studies of transcription activation by the E. coli CRP protein ( CAMP receptor protein ) ( De Crombrugghe rt al. , 1984 ) and the lambda CI protein ( Ptashne , 1986 : ( 1 ) the activator , a dimeric protein , binds to a recognition sequence charact.erized by a dyad symmetry and located slightly upstream from the - 35 region of the RNA polymerase binding site , and ( 2 ) t ' hrough protein-protein contacts or/and a DNA conformational change , the activator facilitates one or several steps in the initiation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>In an in vitro system in the absence of CAMP-CRP , RNA polymerase binds preferentially to a site located 22 bp before the principal ( Pl ) binding site ( Malan & McClure , 1984 Malan et al. , 1984 ) and under certain reaction conditions can result in initiation of a transcript from this position in vitro ( $ - U-U-C-C-G - ; Reznikoff et al. , 1982 ; Malan & McClure , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>In analogy with the preceding examples , the RNA polymerase binding site for the -52 transcript could compete directly for cAMP-CRP binding at the -50 to - 70 binding site for cAMP-CRP ( Dickson et al. , 1975 ) , while the transcript itself could interfere with initiation downstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>In others , such as lac ( 23 ) , araBAD ( 22 ) and malT ( 24 ) , CRP binds in a region outside of the RNA polymerase binding site ( around bp -62 , -90 and -70 respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>whereas in tnost ' operons CRP binds at a site adjacent to RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>Moreover , DNase I footprinting studies indicate that the CAMP-CRP complex binds to a site which overlaps extensively with the RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>Relevant nucleotides ( transcriptional and translational start sites , ÿ10 and ÿ35 positions of the csiD promoter and the divergent upstream promo-ter , and highly conserved nucleo-tides in the CRP binding site consensus ) are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Nucleotides identical to the CRP binding site consensus are indicated by vertical lines In vitro analysis of binding sites for cAMP-CRP and Lrp in the csiD promoter region In gel retardation assays , puri ® ed CRP protein ( in the presence of cAMP ) was found to reduce mobility of a digoxigenin-labeled PCR fragment containing 475 nucleotides upstream and 111 nucleotides downstream of the csiD transcriptional start site ( data not shown ) , suggesting that CRP regulation of csiD is probably due to direct binding of CRP in the regulatory region of csiD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Binding to the upstream site was detected at 5 nM CRP , while at least 40 nM was required for detectable binding at the downstream sit Table 3 Site-directed mutagenesis ) 10 region ) 35 region A. Changes in the dad promoters Consensus sequence TTGACA TATAAT CG - - AATATT P1 ATAAAA AC - - TTGAAT P2 TATCAA C - TAGATT E $ ect of mutagenesis of the CRP binding sites on dad expression To assess the role of the two CRP binding sites on the e $ ects of cyclic AMP-CRP on dad transcription we altered the sites by mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>By sequence ana-lyses the presence of two CRP-binding sites , centred at - 92.5 ( CRP2 ) and - 42.5 ( CRP 1 ) relative to the transcription initiation site , have been predicted ( Munch-Petersen and Jensen , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Analyses of the E. coli sequence showed three different conserved sites upstream of the predicted transcriptional start site , which would be binding sites for the FadR repressor , the ArcA repressor , and the CRP activator ( Fig. 2D ) , the ﬁrst two of which had been predicted by others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The transcription initiation site ( +1 ) , the promoter 10 and 35 regions [ 17 ] , and the GcvA [ 12 ] , Lrp [ 3 ] , PurR [ 30 ] and CRP [ 15 ] binding sites are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>The transcription start site ( 1 ) , the promoter ® 10 and ® 35 regions ( Stauffer et al. , 1993 ) , and the GcvA ( Wilson et al. , 1995 ) , Lrp ( Stauffer & Stauffer , 1994 ) , PurR ( Wilson et al. , 1993a ) and CRP ( Wonderling & Stauffer , 1999 ) binding sites are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>( C ) Nucleotide sequence in the rpoSp1 region with − 35 and − 10 regions , the transcriptional start site , binding sites for cAMP-CRP ( centered at positions − 61.5 and +56.5 with respect to the transcriptional start site ; underlined ) , and binding sites for ArcA-P ( boxed ) indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16291649" target="_blank">16291649</a>]<br></li>
<li>The putative cleavage site for the signal sequence peptidase is indicated by a solid upward arrowhead.The positions of the tsx P1 and 1 ' 2 promoters ( -10 and -35 ) are marked below the nt sequence ; two putative cAMP/CRP binding sites near the tsx 1 ' 2 promoter are indicated ; the tsp for the tsx mRNA are marked by arrows below the nt sequence ; the potential ribosome-bindingsite ( rbs ) is underlined.The REP sequences followingtsx are underlinedwith facingarrows as is a region which shows dyad symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Recently , we showed that CytR binds specifically to multiple sites in the E. coli deoP promoter , thereby providing competition for CRP binding to CRP operator site 1 ( CRP1 ) and CRP2 as well as cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Figure 1A illustrates the complex architecture of the melAB promoter with four sites for MelR binding and a DNA site for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Although there are very few known examples of E. coli proteins binding to noncanonical DNA sites , CRP binds to a noncanonical site in the melAB promoter in cooperation with MelR bound to adjacent DNA sites ( Wade et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Comparative analysis of the udpP promoter region from Y. pseudotuberculosis , V. cholerae , S. typhimurium and E. coli revealed that location , spacing and orientation of putative binding sites for CRP protein are highly conserved , whereas CytR protein recognition sequences of udpPYp and udpPV c deviate markedly from the E. coli and S. typhimurium CytR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Two promoter sequences , at least six strong factor for inversion stimulation ( Fis ) - binding sites , and a cAMP receptor protein ( CRP ) - binding site have been localized in this AT-rich region , and the DNA exhibits inherent curvature ( 8 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226664" target="_blank">8226664</a>]<br></li>
<li>CRP , cAMP receptor protein binding site ; I to VI , Fis binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226664" target="_blank">8226664</a>]<br></li>
<li>The five CRP binding sites are indicated by rectangular boxes ( 22 bp ) which are numbered 1 through 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>( A ) The transcription initiation sites of ascG and ascF and the binding sites for AscG and CRP are shown along the 256-bp-long ascG-ascF spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>The nucleotide sequences of the CRP binding sites centered at bp 2313 and 2140 relative to the transcription initiation site are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>These restriction enzyme cleavage sites are located within the proposed cAMP-CRP binding site centered at bp -78 and -67 respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>The inability of cAMP-CRP to inhibit digestion by Alul at a site centered around bp -12 ( Fig. 7 , lane c ) suggests that CRP protection of restriction enzyme sites is specific for the proposed CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Binding site mutations were incorporated into the CRP and FNR 22 bp binding site sequences at positions 7 and 16 , and are designated X or Y in the case of CRP and FNR binding sites , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>By site-directed mutagenesis , positions +4 in the two CRP binding sites were altered to AT and TA bp , respectively , in different combinations ( positions - 3 6 and - 4 5 in CRP-1 and positions - 8 9 and - 9 8 in CRP-2 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>The open boxes indicate the two CAMP-CRP binding sites , CRP-I and CRP-2 , and the hatched box indicates the CyiR-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The CRP site 1 down mutation resulted in a loss of detectable binding to CRP site 1 , but no change in binding to the regio of CRP sites 2 , 3 , and 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Interestingly , the spacer for this binding site is 8 bp , while most of the known CRP binding sites contain a conventional 6-bp spacer ( 3 , 4 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>The CRP binding site is indicated as well as the 2 Hpal sites that were used to construct a 17 bp deletion of half of the CRP binding site ( pAMC27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>DNase I footprinting conﬁrms that CRP binds to site 3 ; CRP binding to the other potential CRP sites at rhaSR was not detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Putative FIS sites I -- IV , FIS sites 1 -- 3 , a putative CRP binding site centred at position '' 117.5 and binding sites for PurR and DnaA are indicated ( Hutchings & Drabble , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>The PCR products were cloned into the pBluscript SK HindIII -- Eco RI and Eco RI -- Bam HI sites , respectively , and subsequent sequence analysis conﬁrmed that the new construct contained 50-GAACCGAATTCCGCTGC-30 mutated sequence instead of the classic CRP binding site 50-TGTG ATCGCTTTCACGG-30 , located between 2194 and 2178 of glnAp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The consensus tandem DNA-binding site for CRP dimer has been identiﬁed by in silico analysis as shown in red color which is approximately palindromic and provides two very similar recognition sites , one for each subunit of the dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>Four potential CRP binding sites located between the two promoters were initially predicted on the basis of position and similarity with the consensus CRP binding site ( 5 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>This analysis found two highly conserved motifs : the expected one matching the CRP sites found in the Enterobacteriaceae and Pasteurellaceae ( Figure 2F ) , and one matching the PurR repressor binding site consensus ( Figure 5 ) ; the genes and sites are listed in Supplementary Tables 6 and 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In order to precisely localize the CRP binding site , DNase I footprint experiments were performed with a radiolabeled PCR fragment that extended from positions ÿ172 to 44 with respect to the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In this communication we report the results of in vitro CRP-DNA interaction studies which establish that four of the five putative binding sequences bind CRP with high affinity , and that the fifth ( the most distal one relative to the transcriptional start site ) binds CRP with lower avidity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Analysis of the nupC promoter region revealed the presence of at least two putative CRP binding sites , centred at - 40 bp and - 89 bp , which probably flank a CytR-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>Additionally , when carefully comparing the positioning of these putative CRP binding sites in the patZ promoter region , the two highest scoring sites were located at -41.5 ( score 2.60 ) and -81.5 ( score 2.66 ) bp from the RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>This conﬁrms that the two highest scoring putative CRP binding sites ( located at -41.5 and -81.5 bp from the RNA Pol binding site , from now on named CRP I and CRP II ) are essential for the complete induction of patZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>In addition , initial searches with Nucleotide Subsequence Search , MacVector , version 6.0 ( Oxford Molecular , Campbell , Calif. ) , revealed two other potential candidates for CRP binding sites , one located at the 2119-bp region and the other at the 244-bp region of the secB gene ( respective to transcriptional start site 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>In contrast to the CRP footprints , which are localized to the 22-bp CRP binding sites , CytR binding protects an extended region from about bp 225 to 2110 bp relative to the start site for transcription ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP binding to the gcv control region protected two sites from DNase I digestion , one centered near bp 2313 ( site 1 ) and the other centered at bp 2140 ( site 2 ) relative to the transcriptional start site ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>All of the start sites remained within 44.5 bp of the center of the CRP site , and they were 1 to 3 bases downstream relative to the unaltered promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>The promoter region contains two CRP sites , one at 60.5 upstream from the promoter and another at +12.5 downstream of transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The ansB promoter in Escherichia coil is co-activated in a co-dependent manner by one dimer each of CRP and FNR protein whose binding sites are at around -91 and -41 bp , respectively , from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>A recent study , which evaluated the distribution of CRP binding sites along the E. coli chromosome , identiﬁed a potential CRP binding site within the fnr promoter region centered at bp 145.5 relative to the transcription start site ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>However , two CRP-binding sites exist within the pck promoter region : The promoter-distal CRP box ( CRP box-1 ) is located at the position 90.5 bp of transcription start site , while the second CRP box ( CRP box-2 ) is located downstream from the transcription initiation site , centered at +13.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The putative CRP binding site on the gcv control region is centered around bp 2313 relative to the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The Fis protein binds to a site centered 81 bp downstream of the CRP binding site in the proP regulatory region and functions as a positive activator of P2 promoter transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Results and discussion Identification of the functional MalI-binding target at the malX promoter Figure 1 shows a diagram illustrating the malX-malI intergenic region with the transcription start sites for the malX and malI promoters , the corresponding 10 elements , and the DNA site for CRP that is located at position 41.5 with respect to the malX transcription start and position 43.5 with respect to the malI transcription start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>The two Fis-binding sites centered at ÿ41 ( site I ) and ÿ81 ( site II ) plus the CRP binding site at ÿ121 .5 relative to the start of transcription from P2 ( 1 ) are shown .4,5,28 The start of the proP coding region and the transcription initiation points for the P1 and P2 promoters are noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>The CytR-regulated promoters usually contain two DNA sites for CRP , centred at positions 41.5 and 93.5 with respect to the transcription start point , and a DNA site for CytR located between the two CRP-binding sites ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The CRP binding site is centered at position -41.5 , and two AraC binding sites , araFGl and araFG2 , are centered at positions -80 and -155 , respectively , from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>CRP is known to activate transcription by binding to sites located at various positions upstream of the 235 site ( from 241.5 to 2103 ) ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>A single CRP binding site is located 2115.5 and 2215.5 bp upstream of the divergent transcription start sites for the papI and papB genes , respectively , and CRP positively regulates expression of these genes by inhibiting binding of the repressing H-NS protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In deoCp2 , the cAMP-CRP binding sites are located around positions -40 ( CRP-1 ) and -93 ( CRP-2 ) preceding the start site for transcription ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>At class I promoters , such as lac , CRP binding sites are located near 62 , 72 , 83 or 93 relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>In agreement with the involvement of cAMP and CRP in transcription from P5 , a putative CRP binding sequence ( TGTGGATAAAATCACG ) was found at 38 to 39 bp upstream of the transcription start sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>In the case of fur , a proposed cAMP-CRP binding site is situated about 60 bp upstream from the transcriptional start sites ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>However , unlike the situation in the genes described above , in which the cAMP-CRP complex binds in the region upstream of the promoters , the proposed binding site in cir lies far downstream ( over 100 bp ) from the start sites of transcription for promoters P1 and P2 ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>In vitro binding experiments and a mutational analysis suggest that CRP binds to a site centered at bp 2313 relative to the transcriptional start site for gcv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>One possible example for transcriptional activation at a distance was the araBAD operon because it was postulated that cAMP-CRP binding for the araBAD operon was centered > 90 bp from the transcriptional start site ( 4 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>With respect to the araC transcriptional initiation site , cAMP-CRP binding is centered 54 bp upstream , a distance that is intermediate to those found in the lacZ and gal P1 promoters ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>Summary Previous work with semi-synthetic promoters containing a single CRP binding site centred at 41.5 bp from the transcription start site has demonstrated enhanced transcription ( synergism ) when a second binding site , for CRP or FNR , is placed upstream at around -91 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This places the CRP binding site at 243.5 bp upstream of the transcription start site rather than at 241.5 bp ( Fig. 1 ) , which is the usual location for the activator binding site at class II CRP-dependent promoters ( Busby and Ebright , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Concerning the mechanism of the negative autoreguiation , we have shown that CRP-cAMP bound at CRP site I stimulates the binding of RNA poiymerase to the overlapping divergent promoter and then this binding of RNA polymerase interferes with the occupation of RNA poiymerase at the crp promoter ( iHanamura and Aiba , 1991 ) , in this paper , we report a previously unknown aspect of the crp reguiation which is concerned with the role of the second CRP-binding site -LCB- CRP site ii ) iocated about 60 bp upstream from the transcription start site of crp -LCB- Aiba , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The negative regulation of crp transcription by CRP-cAMP is mediated by the CRP binding site I located 40 bp downstream from the crp start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>In particular , rotP2 , rotP3 and rotP4 eaoh contain a cAMP-CRP-binding site at a distance from the start site of transcription that would allow activation by cAMP-CRP : ihe centre of CRP-1 is located 42.5 bp upstream of the rotP2 siart site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>We also identified another CRP site located binding about 60 bp upstream from the crp start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>Although CytR binds to an additional site located just downstream from the transcription start site as noted above , there is no interaction between CytR binding to this site and CytR/CRP binding to CRP2 , CytO , and CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>More recently , I have shown that the CAMP-CRP complex inhibits the transcriptioonf the crp gene by binding to a unique site located downstream from the transcriptional start point ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>Deletion of an apparent CRP binding target located 80 nt upstream from the 5 - most transcriptional start site eliminated expression from all three of these putative promoters ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>The CRP binding site ( T / CC/TTTTTTGAATCCCATCACAAA/T ) is located upstream of the transcriptional start site and is centered at position 61 , which ﬁts the paradigm of class I CRP-dependent promoters ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>In class I CRP-dependent promoters , CRP is known to activate transcription from a DNA site located upstream of the DNA-binding site for E 70 , with the site for CRP usually centered at position 62.5 , 72.5 , or 92.5 relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>The previous observation that maximum guaB promoter activity can be achieved in the absence of sequences located upstream of position 59 is inconsistent with a mechanism whereby CRP occludes an adjacent or overlapping binding site for a transcription factor that activates PguaB ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Sequence analysis predicted a potential GalR and a potential CRP binding site near the transcription start site of the PgalR promoter ( Weickert and Adhya , 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Examination of the CRP binding site of the tdc promoter revealed that it was located 43.5 bp upstream of the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>We monitor these signals in promoter variants that place the CRP binding site at different distances upstream of the start site of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>However , much less is known about promoters where the CRP binding site is located further upstream of the transcription start site , such as the malT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The CRP binding site in the promoter is centered at 234.5 and therefore would overlap the RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>The most intensely studied promoter of this type is the lac promoter with a CRP binding site centered at position -61.5 with respect to the start site of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The CRP binding site identified in this promoter region is conserved across 36 E. coli and Shigella strains examined , yet the distance between the CRP binding site and start of mcaS transcription is sub-optimal , falling between that normally found for class I and class II CRP-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>This putative CRP binding site is adjacent to the promoter region and occupies a position analogous to that of the CRP binding site found in the lac operon relative to the transcription start site ; i.e. , at approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6397402" target="_blank">6397402</a>]<br></li>
<li>The cyclic AMP receptor protein ( CRP ) mediates catabolite repression at rhaBAD by binding to a site immediately upstream of RhaS that is centered at position 292.5 relative to the rhaBAD transcription start site ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>The third binding site detected for CRP displayed the lowest afﬁnity and was the most distal upstream from the cai and ﬁx transcription start points , at positions 2162.5 and 2126.5 , respectively ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>A putative binding site for CRP has been identiﬁed centered at position 117.5 relative to the start site for guaB transcription ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Higashitani and coworkers [ 4 ] predicted a fadL CRP binding site ( Fig. 2B ) centered at 57 bp downstream of the fadL transcription start site ( termed fadL2 ) that is covered by our fadL probe 2 , Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>6 glycerol ( no KC1 ) , as judged from the extent of band supression in the ` footprints ' , i.e. A. CRP binding to CRP site 1 r I 5 ' - AACGCAAT TAATGTGAG  T T A G C T C A C T C A T T A G G C A TTGCGTTAATTAlCACTC A [ A T  C  GAGTG  AGTAAT CC GT -70 t B. CRP binding to CRP site 2 GTGT GGAATTGTGAGCGGATAACAATTTCACACAGG CACACCTTAACACTCGCCTAT TGTTAAAGTGTGTCC-51 1 • 10 • 20 30 C. repressor binding to operator ' 1 ' 5-GTGTGGAATTGTGAGC6GATAACAATTTCACACAG6 CACACCTTAACACTCGCCTATTGTTAAAGTGTGTCC-5 ' it 10 20 I 30 Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>The nucleotide identical with the one found most frequently in the corresponding position of the other known cAMP-CRP binding site ( de Crombruggheet al. , 1984 ) is overlined , and those identical with very highly conserved nucleotides are marked by thick lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1712230" target="_blank">1712230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1712230" target="_blank">1712230</a>]<br></li>
<li>The CRP preparation used was partially inactive , as judged from its ability to remain bound to a lac binding site in a gel retardation assay ( Kolb et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>The cAMP-CRP binding site , the S CRP box , is centred at 268.5 ( Weichart et al. , 1993 ; Marschall et al. , 1998 ) , i.e. at a position quite different compared with typical class I promoters ( e.g. the lac promoter where CRP binds at 261.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The site bound by cAMP -- CRP determined by DNase I footprint analysis is centred at -60.5 relative to the transcription start site for agaZp , indicative of a class I CRP binding site ( Zhou et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Also shown are sites identi ® ed by a previous study ( Limon et al. 1997 ) : a URS ( upstream regulatory sequence ) indicated by the convergent arrows ; the CRP ( catabolite repression protein ) binding site indicated by double underlines ; the ) 35 region and the ) 10 region ( Pribnow box ) are indicated by a single underline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10071223" target="_blank">10071223</a>]<br></li>
<li>CRP binding upstream of aer was ﬁrst detected by Grainger et al. ( 2005 ) in a whole genome chromatin immunoprecipitation analysis , and it was subsequently shown that CRP activates transcription by binding to a single DNA site in the aer regulatory region ( Hollands et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Although the location of the CRPbinding sitesin all four genes described above are different with respect to the transcriptional start site , they all contain a DNA site homologous to the canonical CRP binding sequence 5 ' - AA-TGTGA -- T -- TCA-ATT-3 ' ( 12 , 27 ) found in operons where CAMP-CRP acts as a positive effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>We have indicated the position of the start of the gale gene product ( AUG ) , the Shine & Dalgarno ( 1974 ) sequence ( SD ) , and the CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>The positions of the predicted CRP binding site ( 272 to 251 ) , the 235 sequence ( 235 to 230 ) , the 210 sequence ( 212 to 26 ) , and the transcriptional start point ( 11 ) are indicated ACKNOWLEDGMENTS We thank Monica Persson for skillful technical assistance and J. R. Guest for kindly providing fnr plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In order to restore the doublet of thymines that is present in the wt sequence 8.5 base pairs downstream of the 2-fold axis of the CRP binding site , a single site substitution was introduced by a mutagenic primer using the Chameleon double-stranded site-directed mutagenesis kit ( Stratagene ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The second repeat , which overlaps with the first one , is 6 bp long and exhibits a strong similarity to the consensus cAMP receptor protein ( CRP ) - binding site ( Berg and von Hippel , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>CRP exhibits a high degree of speciﬁcity for a G base at position 7 ( corresponding to a C at position 16 ) of the 22-bp consensus CRP site ( aaaTGTGAtctagaTCACAttt ) ( the core binding site is shown in upper case , and positions 7 and 16 are underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The CRP binding site contains a consensus binding domain consisting of two symmetrically related 5-bp segments at distances of 4-8 bp from the dyad axis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>This protected region contains 14 bp that match base pairs in the 22-bp CRP consensus binding site ( Fig. 1 ) ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The second CRP-protected site , designated site 2 , extends from about bp 2131 to 2158 and contains 12 bp that match base pairs in the 22-bp consensus CRP binding site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Because this site also shows a close match to the FNR consensus sequence , matching at all cents ve positions in the second half and at three of cents ve positions in the cents rst half , band shift assays were used to study binding of both CRP and FNR proteins to a DNA fragment ( fragment EP ) carrying this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>However , the left half-site of such a binding motif poorly matches with the consensus sequence ( AANTGTGANNTNNNTCACANTT ) for CRP-binding ( 47 ) , which could explain the lack of interaction of CRP with the Pa promoter and the observation that MhpR is strictly necessary for binding of CRP to its target DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12748194" target="_blank">12748194</a>]<br></li>
<li>The alignment revealed conservation of three sequence motifs that contain good matches to the σ70 ( `` housekeeping '' sigma factor in E. coli ) − 10 box and the consensus binding site for the cAMP -- CRP complex ( TGTGA-N6-TCACA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>The 5 ' end of the operon is clearly defined by the primer extension studies , the initiation of mRNA synthesis occurring at the first available polymerase recognition site 3 ' to a consensus CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1719366" target="_blank">1719366</a>]<br></li>
<li>The 312 CRP promoter has point mutations in three consensus positions of the rhaSR promoter CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>The fusion of CRP1 and CRP2 in del264 created a site ( TGTGAN6TCACG ) almost identical to the consensus CRP binding site ( TGTGAN6TCACA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The inverted-repeat constituting a good catabolite repression protein ( CRP ) consensus binding site is indicated by heavy underlining .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8396120" target="_blank">8396120</a>]<br></li>
<li>The consensus CRP binding site is indicated for comparison .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The palindromic consensus DNA-binding site for CRP dimer is shown in red .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>The putative CRP binding site is boxed with the residues identical to the consensus shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>The consensus CRP binding site ( see reference 16 and references therein ) and positions showing identity to the consensus are shown in boldface type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The cAMP-CRP binding site is at -60 ; the consensus sequence is shown directly below ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3902796" target="_blank">3902796</a>]<br></li>
<li>The CRP binding consensus sequence , a perfect palindrome with a half-site of 11 nucleotides , is shown at the top .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>The CRP site matches the CRP consensus binding sequence in one half and the FNR consensus binding sequence in the other half .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The underlined sequence shows where a putative CRP-binding site conforms to the CRP consensus sequence ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The consensus sequence shows sequence similarity to the CRP binding site ŽTGTGA-N6-TCACA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The score is a prediction of the afﬁnity of the site for CRP , based on its similarity to the consensus sequence for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>The DNA sequence of the consensus CRP binding site is boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>( B ) The DNA sequence around the CRP binding site of gadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>The similarity at the DNA level includes a canonical and symmetrical CRP binding site , but no clear promoter consensus sequence in front of nagE , as well as a sequence corresponding to a Rho independent terminator between nagE and the immediately adjacent glnS sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693951" target="_blank">2693951</a>]<br></li>
<li>The DNA sequence in the vicinity of the putative promoter was inspected for inverted-repeats resembling the for the CRP binding site consensus ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>The sequence of CRP site 1 ( hatched rectangle ) is shown , as are the three base pair substitutions that were introduced to inactivate this CRP-binding site ( vertical arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>The FeaR binding sites are highlighted by lines above the sequence , and the predicted CRP binding site is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>The sequence of the predicted CRP binding site is indicated by bold type and is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18978044" target="_blank">18978044</a>]<br></li>
<li>The CRP-binding site in the glpD promoter does not correspond as closely to the consensus sequence ( 7 ) for CRP binding when compared with the analogous site in other catabolic operons such as lac : AA-TGTGA ------ TCACA-TT ( consensus ) , TAATGjTTATACA TATCACTCTA ( glpD ; 10 of 14 matches ) , and TAATGT GAGTTAGCTCACTCAT ( lac ; 11 of 14 matches ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>The CRP site matches the consensus at 17/22 positions , including 9/10 positions in the core binding motifs that are critical for CRP binding ( 9 , 30 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The predicted FNR site matches the consensus FNR binding site at 7 of the 10 conserved positions , and the predicted CRP site matches the CRP binding site consensus sequence at 9 of a possible 12 positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The sequence has a close match to the consensus CRP binding site ( matching at nine out of 10 positions at the core binding motifs ) and is positioned such that binding of CRP would inhibit xseA transcription while possibly activating guaBA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Based on comparison of the lac , gal and ara CRP binding sites , Taniguchi et jil̂ ( 21 ) derived the following 11 base consensus sequence for CRP binding : AAAGTGTGACA • This sequence ( with four mismatches ) appears within the ilvB CRP binding site between bp -75 and -65 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>We have not yet obtained evidence that cAMP-CRP is able to bind to these regions , and the low degree of sequence similarity to the cAMP-CRP binding site consensus ( Berg and von Hippel , 1988 ) makes it doubtful that these regions really act as cAMP-CRP-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Additionally , `` up '' mutations were made in CRP sites 1 and 3 such that these sites perfectly matched the consensus CRP binding site sequence at the 10 most conserved positions ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>( C ) Comparison of the putative CRP binding sites within the rhaSR-rhaBAD intergenic region and the CRP consensus binding site sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Boxes , pho boxes and the 210 region of psiE ; brackets , regions protected from DNase I digestion in the presence of PhoB and cAMP-CRP ; asterisks , positions identical with the consensus sequence for the cAMP-CRP binding site ( 7 ) ; thick line above pho box 1 , putative 235 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The cAMP-CRP complex bound to the region from 211 to 239 , which includes a sequence similar to the consensus for the cAMP-CRP binding site ( Fig. 4C and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>This region contained a sequence that both conformed to the consensus binding site for cAMP-CRP in positively regulated promoters and acted as a cAMP-CRP binding site in a gel retardation assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>Furthermore , a sequence found in the regulatory region of the E. coli serC-aroA operon [ 13 ] , 5 ' - TGTGATGCAAGCCACA-3 ' , appears to be very similar to the cAMP-CRP binding site consensus sequence , 5 ' - TGTGANATNNNTCANA-3 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>A sequence found in the regulatory region of the E. coli trxA gene [ 17 ] , 5 * - TGTGTTTTGCTCATA-3 * , appears to be very similar to the cAMP-CRP binding site consensus sequence [ 23 ] , 5 * - TGTGANATNNNTCANA-3 * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>A cAMP-CRP binding site of the glpD gene was identified by comparison with the consensus sequence , gel shifting assay , and DNase footprinting experiments [ 4-7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>It mapped to a site downstream of a 210 consensus sequence and a likely binding site for cAMP-activated CRP ( CRP1 ) in accordance with predictions based on DNA sequence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>This change , in the 5 ' - TGTGA-3 ' CRP consensus sequence ( 9 , 15 ) , abolished CRP binding to the ompB promoter region in vitro ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>In the promoter region of the positive regulator for the monoamine regulon , moaR from K. aerogenes , we also found a similarity in the consensus sequence 59-tt-TGTaA ------ TCACA-tg-39 for the binding site of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631685" target="_blank">8631685</a>]<br></li>
<li>Effect of alterations in the CRP binding site in the clyA promoter region on the chromosome to match the consensus sequence ( 59-AAATGTGA TCTAGATCACATTT-39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>e control of Since expression of the melibiose operon is under t h site of CRP catabolite repression ( 5 ) , a consensus sequence for the binding was expected to be present upstream of the -35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>Although this site is not precee-ded by a consensus -35 sequence , it is close to a region with strong homology ( 85 % ) with the CRP binding site ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3031590" target="_blank">3031590</a>]<br></li>
<li>CRP binding to site 2 observed in vitro is likely due to the sequence similarity between the region and the consensus CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A c.sfit -0 RF2 operou is suggested by t ' he fact , that ) the two genes overlap by one nucleotlde and that ' no apparent - 10 sequence or CRP binding site is found immediately upst ) ream from ORF2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>A CRP recognition sequence ( 6 ) centered at 241.5 is preceded by a half site binding motif for the anaerobic regulator FNR ( 13 ) at position 257 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>A CRP box-like sequence was identiﬁed within this sequence ( Fig. 4 ) , which overlaps partially with the AscG-binding site ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>A potential 70 10 sequence is indicated in bold , and the predicted CRP binding site is boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>ttie sequence element 5 ` TTGAT3 ' centred beween bases - 4 1 and - 4 2 ( -4172 ) : ttiis is also ttie centre ot ttie CRP binding site at gaiP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>homologous sequence in CRP site 2 is : 5 ' g-A-A-t-T-G-T-G-A-g-c 3 ' CRP seems to bind with similar affinities to site 1 and 2 in binding buffer containing 25 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6259624" target="_blank">6259624</a>]<br></li>
<li>A sequence which is very similar to the complementary sequence of the very similar to that of the binding site for CRP proposed by O'Neill - e-t al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>, 1987 is shown below the gal sequence and is aligned so as to show the location of the CRP binding site at gaiP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>This makes the sequence a likely CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>We conclude from these experiments that the TGTGA sequence in frr does not function as a CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1860827" target="_blank">1860827</a>]<br></li>
<li>Moreover , mutations at the 244-bp consensus sequence , TGTGA , did not have any effect on CRP binding ( data not shown ) , and this site is not involved in CRP binding , even when the real BCRP site is altered or deleted ( Fig. 5B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>These include a 6 bp consensus sequence , CTCACT , within or partially overlapping the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>A potential CRP binding site , based on a consensus sequence proposed by Chapon and Kolb ( 1983 ) , was found between - 5 0 and - 7 2 ( position 528-550 in Scheme 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>A potential CRP binding site is found at coordinates 293-3 14 ( CRP , in Fig. 2 ) , based on a ` consensus ' sequence ( AA-TGTGA-TA-TCAZATTT ; Chapon and Kolb , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6397402" target="_blank">6397402</a>]<br></li>
<li>Third , the sequence from position 229 , T-A-T-T-G-T-T-A-A-T-T-T-C-T-T-C-A , resembles the consensus sequence ( Adya & Gottesman , 1982 ; Ebright , 1982 ) for the catabolite repression protein ( CRP ) binding site , A-A-T-G-T-G-A - - T ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6310120" target="_blank">6310120</a>]<br></li>
<li>In the consensus sequence , those bases presented in bold print ( positions 4-8 in each half site ) are of greatest importance for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Strikingly , this site did not exhibit any homology to the consensus sequence for CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>An excellent candidate for the CRP binding site ( GGTG ACGCAGTTCACC ) was found in the vicinity of its promoter , while the consensus sequence of the CRP binding site is TGTGANNTANNTCACA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>To further test this hypothesis , we assayed a fusion with an upstream endpoint at 312 and carrying point mutations at three consensus positions in the CRP binding site ( Fig. 1 ) that we previously found greatly reduced CRP coactivation of rhaSR ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>In addition , we substituted four positions in the predicted CRP binding site , resulting in the plasmid pMWK9 , having an altered CRP site ( 59-TTGTTTGATATAGATCACATTT-39 ) which matches the consensus in 17 out of 22 bases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In udpPSt and udpPYp the CRP2 binding site is also more similar to the consensus CRP site , whereas udpPV c has three mismatches in CRP2 but contains a more perfect CRP1 target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>guaB In a complementary experiment , an A-to-C point mutation was introduced at position 18 in the PguaB CRP site [ PguaB ( 133 to 36 , A-111C ) ] , generating a consensus core CRP binding site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Thus , we introduced a series of changes at the FNRl site that converted it into the core consensus FNR protein binding site ( FF construct ) or into the core consensus CRP protein binding site ( CC construct [ Fig. 4A ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>We used site-specific mutagenesis to change this putative FNR protein binding site into the CRP protein binding site core consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Changing the FNR1 site into the CRP protein binding site core consensus drastically lowered anaerobic expression did CRP and confer weak protein-dependent expression ( Tables 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>117 * * * 3 CTATGGCATAGCAAAGTGTGACGCCGTGCAMAATMCMATGTGGAC1I , , , C 3 consensus CRP binding site AA-TGTGA - - T -- TCA-ATA T ATAGCAAAGT : ACGCCGTGC A CRP deletion S3 mutation ATAGCAAAGT = ACGCCGTGC S6 mutation ATAGCAAAGTLIA CCGTGC FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Also , the number of base pair matches out of 10 with the consensus CRP binding site is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>From these studies a consensus CRP binding site containing the imperfect palindrome , T G T G A N ~ T C A C A , was established ( Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>This site possesses 9 of the 16 consensus nucleotides associated with CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>In vitro DNase I footprinting shown in Fig. 5A determined the CRP binding site to be at the core recognition sequence ( 224 to 245 ) described above and demonstrated the dependence on cAMP for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Further , CRP : cAMP complex was treated with a speciWc DNA sequence containing the lac CRP binding site along with RNA polymerase to mimic in vivo conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16934214" target="_blank">16934214</a>]<br></li>
<li>Th sequence may then be a CRP binding site for regulation of transcription in both directions , mediated by the CRP/cAMP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3284790" target="_blank">3284790</a>]<br></li>
<li>By searching the consensus sequence for the CRP binding site and by using lacZ gene fusions , we dem-onstrated that the effect of glucose on the maoB transcription in vivo was mediated by the cAMP-CRP complex but on maoA it was not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631685" target="_blank">8631685</a>]<br></li>
<li>To address whether the genomic context of the CRP binding site within the proP P1 promoter was causing CRP -- cAMP binding to be osmotically sensitive , we measured binding to a synthetically derived sequence representing the proP P1 CRP site plus the flanking 13 bp ( − 55 to − 14 in Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>For site P3 , the sequence TAAAAT was found as a potential -10 promoter sequence of the RNA polymerase Eu70 , and instead of a - 35 sequence we observed the presence of a potential cAMP-CRP binding site ( AAATCGTGAT GAAAATCACAlTIl ) ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>A - 10 box was suggested with the sequence TAGAAT ( Fig. 2 ) and the -35 regíon was postulated to overlap wlth a putative binding site for the cAMP-CRP protein , centred around position -40 ( from the +1 site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>A sequence with 17/22 match with the conscrisus binding site for CRP ( Kolb e i d. 1,993 ) lies at -117 with respect to itiiriation of gun transcription , but downstream of the ciilergent promoter for x s t A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>The P1 promoter contains a sequence centered at 234.5 with respect to the transcription initiation site that has an exact match to the consensus CRP binding site , TGTGAN6TCACA ( 11 ) ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>The E. coli mcaS gene also contains a near consensus -35 promoter element and a putative binding site for the transcription regulator CRP ( centred at -52 ) , but these sequence motifs are less well conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>( SequenceB ) : The consensus nucleotide sequence for cAMP-CRP binding sites is compared to the melR promoter sequence from nt -52 to -31 and the known CAMP-CRP binding site at gulPI , which is also located from -52 to -31 with respect to the gulPI transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>( b ) The consensus nucleotide sequence for the cAMP -- CRP binding site ( Busby & Ebright , 1999 ) is compared to the Pr promoter sequence from nt '' 51 to '' 30 and the known cAMP -- CRP binding sites at pgalP1 and pmelR , which are located from '' 52 to '' 31 with respect to its transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>The positions of transcription start , -10 hexamer , sequence with -35 similarity , and CRP binding site are identified above the promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>The nucleotide sequence around the cya P2 promoter is given below pIT228 ; the transcription start site is indicated as # 1 , and the CRP binding site is doubly underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>Moreover , this protected region corresponds to the binding of CRP on the perfect consensus sequence situated at position 241.5 of the transcription start site for cai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The previously identiﬁed CRP site centered at 292.5 relative to the rhaBAD transcription start site ( 9 ) matches the CRP-binding-site consensus sequence ( 2 ) at 9 of 10 positions and appears to be the strongest CRP site in the rhaSR-rhaBAD intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>Occupancy of the perfect consensus sequence with the highest afﬁnity at position 241.5 relative to the cai transcription start point ( Fig. 5 ) corresponds to a class II promoter in which the DNA-binding site for CRP overlaps the DNA-binding site for RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Within this region , a binding site for the cAMP -- CRP complex similar to the consensus sequence ( Busby & Ebright , 1999 ) was identified ( Fig. 2b ) , centred at position 240.5 with respect to the transcription start point , which suggests that CRP promotes transcription from the Pr promoter by a class II promoter mechanism ( Busby & Ebright , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>However , the interaction of polymerase with CRP and/or the sequence elements within the polymerase binding site are apparently not sufficient , since a second , upstream activator , AraC protein , is also required AT T TT CTTC TAA A AT GCTGGT TA A T T T T T T TC T T CAT TTGACA ¬ 16-18 - t g-TATAAT Consensus Sequence FIG. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>It should be noted that site 2 is totally within the region shown previously to be protected from DNase I digestion by the Lrp protein ( 41 ) , and we believe this site is probably not accessible to CRP in vivo due to Lrp binding to this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The constructions derived from the Al deletion ( CRP binding site located around -40 ) remove the -35 region and show very little promoter activity as expected for a truncated promoter ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The presence of a sequence with only partial homology with the FNR protein-binding site and of a nonoptimal CRP site in the ansB 5 ' sequence do not preclude direct interaction of these proteins with this promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The nucleotide sequence at the gatPI CRP binding site iocated from - 3 1 to - 5 2 displays some similarities with the same region at the nirB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Role of the DNA sequence upstream of the CRP binding site The most direct way demonstrate to that upstream sequences participate in contacts with RNA polymerase is to remove the sequences completely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>At Class I promoters , CRP binds to a DNA sequence upstream of the RNAP-binding site and makes direct protein -- protein contact to aCTD vi Table 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>At class II promoters , where the CRP binding site overlaps the 35 region and RNA polymerase makes DNA contacts upstream and downstream of CRP , there is generally a poor match to the consensus 35 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Finally , using malT promoter variants where the DNA sequence upstream of the CRP binding site was deleted , we assessed the importance of contacts between this DNA region and the back-side of RNA polymerase in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>This is confirmed by an experiment using a promoter in which the upstream CRP binding site was inactivated by mutations at positions 6 and 17 in the 22bp consensus sequence ( see Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>A sequence that bears signiﬁcant similarity to the CRP consensus binding site ( 5 , 9 ) resides about 70 bp upstream of the proposed acs promoter ( 3 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>DNA sequence analysis of the IS5-disrupted promoter localized the site of the 1.3-kb insertion to position 256.5 , just upstream of this CRP-binding site ( hereafter designated CRP site 1 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>Four promoter variants were used : a wild-type promoter ( wt ) which carries the upstream AT-rich silencer sequence , a promoter allele which lacks the upstream silencer ( D ) , an allele with an improved CRP binding site ( CRP + ) , and a double mutant which both lacks the upstream silencer and carries the improved CRP binding site ( D , CRP + ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Analysis of the nucleotide sequence [ 13 ] of the regulatory region of this operon revealed a site 90 bp upstream from the transcriptional start site which is very similar to the consensus sequence proposed for recognition and binding of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>These two promoters differ in the sequence from -49 to -62 and in the sequence upstream of the CRP binding site , suggesting the involvement of either one or both regions in the mechanism of transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Further evidence supporting the model that CRP is involved in the expression of ClyA was obtained by altering the sequence of the potential CRP binding site in the clyA upstream region , both located on the plasmid and on the chromosome ( Fig. 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The sequence analysis of the cspE upstream region showed a putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>I , ArcA-P binding region I ; II , ArcA-P binding region II ; CRP , CRP binding site ; and URS , upstream regulatory sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10071223" target="_blank">10071223</a>]<br></li>
<li>Furthermore , we were unable to identify by examination of the DNA sequence a binding site for CRP in the upstream regulatory region of the hyb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Analysis of the upstream sequence indicated that it might contain a cAMP-CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>Promoters derived from the A2 deletion possess the same sequence between -40 and -49 but they differ widely in the sequence further upstream ( due to different length spacers , insertion of a CRP or IHF binding site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>In each case , the original CRP site was disrupted by point mutations in the core region that are unfavorable for CRP binding ( Table 1 ) , and the repositioned CRP site retained the 22-bp sequence of the original .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The CRP binding sequence is centered 41 `` 5 bp upstream from the araFGH transcriptional start , which places the right one-half of the site in the - 3 5 region of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>The CRP binding site was located adjacent to this sequence and was centered at bp 292.5 ( 63 , 243 , 244 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>The CRP 2 and CRP 3 constructs replaced the native CRP site ( 20 bp upstream of the RhaR binding site ) with the same sequence 2 or 3 bp upstream as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>The sequence of the CRP binding site of malT starting 11 bp upstream of the first pentamer and ending with the downstream pentamer is cloned just upstream of the linkers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Accordingly , a perfectly conserved binding site for the global transcriptional regulator CRP was identified 65 nucleotides upstream ( Fig. 2 ) ( de Crombrugghe et ai , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7815937" target="_blank">7815937</a>]<br></li>
<li>Indeed , the site of CRP binding in target DNA was first deduced by comparing the sequences of CRP-dependent promoters and then confirmed by footprint analysis of C R P - ( c A M P ) 2 : D N A complexes ( de Crombrugghe et al. , 1984 ; Berg and yon Hippel , 1988 ; Busby and Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Additionally , both FIS and CRP bind to highly degenerate sequences ( Ebright et al. , 1984 ; Hübner and Arber , 1989 ) , suggesting that not only sequence speciﬁcity , but other features like DNA topology and the nucleoprotein context in which a binding site is located ( Pan et al. , 1996 ) determine their interaction with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>A consensus cAMP receptor protein ( CRP ) binding site ( Ebright et al. 1987 ) exists between residues 93 and 114 of the determined sequence , confirming the catabolite repressible nature of the operon , and located 35 nucleotides 5 ' of the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1719366" target="_blank">1719366</a>]<br></li>
<li>The only site presenting some homology with the consensus sequence of the CRP binding site ( De Crombrugghe et al. 1984 ) was located downstream from the promoter sequence just before the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>As , in most cases , CAMP-CRP acts by binding at a specific site upstream of the transcription start point ( de Crombrugghe et al. , 1984 ) , we compared the meZR sequence from -59 to + 1 with the suggested consensus sequence for CRP binding sites ( Ebright et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Searches using the CRP consensus sequence of Zheng et al. [ 27 ] produced another fadL candidate binding site upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Of interest is the sequence adjacent to the predicted -35 region ( Agg-TGTGAcacaccTCgcAcT ) which is identical in ten ( uppercase letters ) out of 14 positions to the consensus sequenc AANTGTGANNTNNNTCANATW ( where W is A or T , and N is A , C , G or T ) for the DNA binding site of CRP complexed with CAMP ( Ebright et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1677907" target="_blank">1677907</a>]<br></li>
<li>Secondly , CRP-wt protein exhibits a very high specificity for GC and CG basepairs at positions + 4 , respectively , in the consensus CRP binding site ( Ebright et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>A putative binding site for the catabolite repression protein ( CRP ) ( Kolb et al. , 1993 ) is present just upstream from the Shine ± Dalgarno sequence ( encircled in Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>Nevertheless , such cross-talk is likely to be limited because FNR can not recognize a CRP-site consensus sequence in vitro ( Green et al. , 1996b ; Sawers et al. , 1997 ) and the affinity of CRP for a FNR-site consensus is 50-fold lower than for its natural binding site ( Sawers et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>These changes were made in the CRP binding site consensus sequence ( Kolb et al. 1993 ) and are shown in Table 3B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>, 1987 ) , and aligned it with the CRP binding site in the galP1 sequence running from bp - 5 2 to - 3 1 ( de Crombrugghe et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Mutations within the external gnt operator sequence and the CRP binding site were generated by a three-step recombinant PCR method ﬁrst described by Higuchi et al. ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Regions protected by H-NS and SlyA in footprint assays ( H-NS I , H-NS II , SlyA I and SlyA II ) , and the FNR/CRP binding site described by Westermark et al. ( 2000 ) are indicated by lines above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The shaded box indicates the lacZ region derived from pMC1403 Determination of cAMP levels by a gel mobility shift assay We developed a simple method for cAMP determination based on the ability of CRP to form a stable complex , in the presence of cAMP , with a DNA fragment containing the consensus CRP-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>The band shift assays show that CRP binds speci cents cally to the DNA fragment carrying the guaBA and xseA promoters and a putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The results , illustrated in Fig. 6 , show that CRP binds to a single site in the aer200 fragment , but binding of CRP to speciﬁc targets in the tsr and trg regulatory regions was also detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In agreement with this , the intergenic region between the divergent cai and ﬁx operons revealed important regulatory features , namely , potential rec-ognition sequences for CRP and H-NS , but no binding site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The intergenic region between the divergent cai and ® x operons contains potential recognition sequences for CRP and H-NS , but no binding site for FNR ( Eichler et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Binding of cAMP-CRP gives rise to protection ot two regions that extend from - 9 1 to -115 ( CRP-1 ) and from - 1 4 4 to - 1 6 8 ( CRP-2 ) ( Fig , 3 , lanes 3-6 ) , Consisientiy , these two regions contain sequences that show homoiogy to the consensus cAMP-CRP-binding site ( Fig , IB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Upstream sequences of glnAp2 are not essential for CRP -- cAMP-mediated inhibition Results already described ( Fig. 3 ) indicate that the CRP-binding site at 270.5 upstream of glnAp1 is not essential for CRP -- cAMP-mediated inhibition on glnAp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Inspection of this region revealed two sequences matching the CRP binding site consensus ( 24 ) , one ( site I ) centered at position 262.5 ( 59-TTTGAtccatcTCAGA-39 ) and another ( site II ) at position 2103.5 ( 59-TATGAgtaattaTCGCA-39 ) ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>The mtlOP region previously shown to contain five wellwas conserved for CRP binding , each in consensus sequences proximity to the others and all upstream of the mtlA translational start site ( Fig. 3B ) ( 10 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300537" target="_blank">8300537</a>]<br></li>
<li>Examination of the DNA sequence within the P1 promoter region revealed an almost perfect binding site for the cyclic AMP ( cAMP ) receptor protein ( CRP ; also known as catabo-lite-activating protein ) centered at 234.5 with respect to the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>We showed the presence of a unique transcription promoter , gapB P0 , with a cyclic AMP ( cAMP ) receptor protein ( CRP ) - cAMP binding site centered 70.5 bp upstream of the start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>A sequence that is a good match to the cyclic AMP ( cAMP ) receptor protein ( CRP ) binding site is centered at 71.5 bp upstream of the P promoter , which is suggestive of a class I type activation mech-1 anism by CRP-cAMP ( 30 -- 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>OmpR binds within the IHF-C site ( 5 ) , and the cyclic AMP-binding protein ( cyclic AMP-CRP ) binds at a site located within the IHF-B binding region and activates at least one additional promoter ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The proP regulatory region also contains a highaf ® nity binding site for the cyclic AMP ( cAMP ) receptor protein ( CRP ) located at ÿ121 .5 relative to the P2 promoter start site ( Figure 1 ) .28 In the absence of Fis , CRP-cAMP provides little activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Interestingly , both the dsdX and cstA ( Cy5 : Cy3 ratios are 0.43 and 0.44 ) promoters are positively regulated by the cAMP -- CRP complex ( Schultz and Matin , 1991 ; Nørregaard-Madsen et al. , 1995 ) , and a consensus CRP binding site ( TGTGAN6TCACA ) ( Kolb et al. , 1993 ) is also found in the mglB ( Hogg et al. , 1991 ) and yjiY promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123448" target="_blank">12123448</a>]<br></li>
<li>CRP -- cAMP binding to the consensus site located at − 37.5 in the galP1D4 promoter ( Bell et al. 1990 ) was assayed by in vivo DMS footprinting after an osmotic upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>No consensus binding site for the CRP protein could be detected in the upstream region of sfm , conﬁrming that CRP probably does not bind to the promoter region of sfm ( Grainger et al. , 2005 ) and does not directly compete with H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>A binding site for cAMP-CRP in the osmY promoter region ( Colland et al. , 2000 ) is shown in italics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Indeed , the cAMP/CRP-induced DNA bend at base-pair -62 in the ZacPl promoter region has been shown to be asymmetrical with a larger degree of bending on the 5 ' - side than on the S / - side of the CRP protein binding site ( Wu & Crothers , 1984 ; Lui-Johnson et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>In addition , we have recently found that cyclic AMP ( cAMP ) receptor protein ( CRP ) - cAMP complex has a strong inhibitory effect on proP1 , perhaps by binding to the almost perfect consensus CRP binding site located within its 235 region ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545153" target="_blank">7545153</a>]<br></li>
<li>Abbreviations for the putative sites : CAP/CRP , cAMP-CRP consensus sequence ( 4 ) ; IHF , integration host factor ( 7 ) ; NRI , nitrogen regulatory protein binding site ( 11 ) ; RpoN , s54 promoter region ( 17 ) ; RpoS , 210 region of the sS promoter region ( 24 ) ; 210 and 235 , Es70 promoter sequences ( 18 ) ; S.D. , Shine-Dalgarno sequence ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696780" target="_blank">9696780</a>]<br></li>
<li>Apart from a cyclic AMP receptor protein ( CRP ) half-site , TGTGA ( 1088 to 1093 ) , no sequences resembling the CRP-binding site consensus were detected in a comparable search of the fumB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>In the putative promoter region , there is also a high degree of similarity to the consensus sequence 59-Ag-TGTGA ------ cCgCA-AT-39 ( where a hyphen indicates any nucleotide ) for the binding site of cyclic AMP ( cAMP ) receptor protein ( CRP ) ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631685" target="_blank">8631685</a>]<br></li>
<li>Accordingly , two putative targets for the cAMP-CRP complex , CRP-1 and CRP-2 , with good match to the consensus CRP-binding site ( TGTGAN6TCACA [ 14 ] ) , are present in the dsdXA promoter region ( Fig. 1C ) ; CRP-1 is centered around 287.5 , and CRP-2 is centered around 2117.5 , two positions at which the cAMP-CRP complex is unable to activate transcription as the sole activator ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>Two potential binding sites for the cAMP-CRP complex within the promoter region of gntT were identiﬁed on the basis of similarity to the consensus binding site , positioned at 271 ( TATGAccaaccTC TCA ) and 213 ( TGTTAcccgtaTCATT ) with respect to the transcriptional start site ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>A potential binding site for the cAMP-CRP complex within the promoter region of prpRBCDE was identiﬁed on the basis of similarity to the consensus binding site ( 11 ) ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>To examine the effect of the putative cAMP-CRP binding site on prpBCDE expression , six nucleotide substitutions were made at the consensus CRP-binding site ( AAACGTTAACT 3 TTACCCtGACC [ changed nucleotides are underlined ] ) in the promoter region of the PprpBCDE-rfp fusion ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Despite the presence of a perfect consensus CRP binding site in the galP cis-regulatory region , the PgalP promoter activity reached only about 50 % of the activity of the PmglB , PgalS and P1galE promoters in the presence of cAMP -- CRP in our assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>We propose that CRP , binding to a near-consensus site centred at 3117.5 , activates transcription from guaP and obstructs transcription from the xseA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Although footprinting failed to identify the exact binding site of CRP at the guaBA promoter , the most likely site at which CRP could bind to repress xseP and activate guaP is the near-consensus binding site centred at 3117.5 with respect to guaP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>CRP and FIS Compete with RNAP for Binding at the fis Promoter -- The fis promoter regulatory region contains sequences matching the CRP binding site consensus ( 11 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Analysis of the clyA promoter region revealed a potential CRP binding site ( 59-TTGTTTGATATTTATCATATTA-39 ) that matched the consensus in 13 out of 22 bases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>At an Es70-dependent promoter with no ÿ35 consensus region , an optimal location for a CRP binding site is centered at ÿ72 .5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We also constructed a strain with the CRP consensus DNA binding site in the clyA promoter region on the chromosome of the hns1 strain MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Examination of the upstream region of the serovar Typhimurium lsr operon reveals one potential CRP binding site ( spacer shown in lowercase letters ) : 5 - TGAGAg tttttTGACC-3 ( 32 to 47 relative to the predicted transcriptional start site of the lsr operon ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>There is a potential , albeit weak , CRP binding site within the 20 bp conserved control region located between ® 52 bp and ® 73 bp from the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>Introduction of small insertions and deletions that move the cAMP-CRP binding site to locations between 280.5 and 260.5 nucleotides upstream of the csiD transcriptional start site Site-directed mutagenesis of the UP-element half-site in the csiD promoter region For mutagenesis of the distal UP-element half-site in the csiD promoter region ( Fig. 4 ) , a four-primer/two-step PCR procedure was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Since the protection by CRP is completely cAMP-dependent , our footprinting experiments show that the cdd promoter contains at least two classes of binding sites for CRP : a strong primary site overlapping the -35 region and a weaker secondary upstream target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>In accordance with our primer extension analysis , this result suggests that occupation of secondary binding sites in the upstream region of the crp promoter at high FIS concentrations may both displace cAMP -- CRP from binding to site II and prevent transcription initiation from the crp1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>At the araBAD promoter , CRP binds to a site upstream of AraC Sites I1 and I2 , and this binding is independent of the binding of AraC ( Zhang and Schleif , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Speciﬁcally , cAMP-CRP is shown to bind to a CRP binding site located in the upstream region of the lsr promoter and works with the LsrR repressor to regulate AI-2 uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>These results demonstrate that cAMP-CRP stimulates transcription of the lsrACDBFG operon by directly binding to the upstream region of the lsr promoter and that mutation in the CRP binding site abolishes this stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Addition of cAMP-CRP caused binding to , and transcription from , the primary ( + 1 ) promoter and these authors proposed that binding at the upstream site interferes with binding at the usual promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2582140" target="_blank">2582140</a>]<br></li>
<li>For the IS5-disrupted promoter , which retains a single CRP-binding site , the additional activating region of the mutant CRP evidently compensates for the lack of upstream regulatory sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>]<br></li>
<li>Signals within the core promoter region , as well as those located just upstream of the -35 hexamer , are unaffected by the position of the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>in the lactose promoter region stem-loop structure located seems to be the upstream of the -35 region binding site for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>Sites protected by H1 are situated at the CRP binding site ( -55 , -70 ) , at the repressor binding site ( +20 ) , within the Pribnow box ( - lo ) , and in the -35 region , all of which are also binding sites for specific proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>Under these conditions polymerase appears to bind at a site upstream from the -35 region , which encompasses the Rsal and Sau3A sites , close to or including the region for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Since the a-CTD preferentially binds to A/T-rich regions in the minor groove ( 18 ) , the binding site for the a-CTD may overlap the highly A/T-rich segment present at the downstream end of the CRP protected region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>This region of 247 nucleotides extending from the CRP binding site to the initiation codon is probably the operator promoter region of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1719366" target="_blank">1719366</a>]<br></li>
<li>To mutagenize the CRP binding site at the proP P1 promoter region , primer JX15 ( 59-CGAAATCCATGTCTGAAGTTGATCACA-39 , from nt 253 to 227 ) and primer JX8 were used to initiate the ﬁrst-round PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>For one half of the CRP dimer binding site ( region -90 D - 8 0 ~ I -- 8 0 ~ -7 OD -70 , - 6 0 ~ -60 , ¬ w ¬ -5 0 ~ -50 D FIGURE6 : Binding of H1 and CRP proteins at the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>It was noteworthy that the CRP binding site in the crp regulatory region is clearly separated from the RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>CRP binding site is centered at 61.5 in the cspE upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>This region contains a putative CRP binding site centered around ÿ68 .5 ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>b Genomic region encompassing the putative CRP binding site in the corresponding intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>Sequence analysis indicated a CRP binding site in the gntP regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Crucial role for the spacing between the CRP binding site and the - 10 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>Identical nucleotides to the Sequences other than that of the melibiose this region is the binding site for CRP , as shown below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>RNA poly ¬ Abbreviations : bp , base pair ( s ) ; CRP , cAMP receptor protein ; AACT , 3-bp deletion within the araC activator binding site ; ACRP , 3-bp deletion within the cAMP-CRP binding site of the ara regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>Site-directed mutagenesis of the cAMP-CRP binding site and insertion of 5 or 10 bp in the fragment containing the full-length ulaG and ulaA promoter region were done by crossover PCR ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Deletion of sequences between positions -77 and -60 greatly reduced the level of transcription in the presence or absence of cAMP-CRP , indicating that at least part of this region is a binding site for a positive-acting transcription factor ( or RNA polymerase itself ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>) Since cAMP-CRP binding required sequences between positions -103 and -77 , we conclude that the CRP site centered around position -84 is required for spf gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>Hence , due to the presence of a CRP consensus binding site at a functional distance from the gapB P0 start site , we had to test for the involvement of the CRP-cAMP complex on gapB P0 activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851989" target="_blank">9851989</a>]<br></li>
<li>In many cases , the cAMP-CRP complex binding site is positioned at a relatively long distance from the transcription start site ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>In addition , in view of the fact that the PaaX operator ( from 126 to 77 ) completely overlaps the weaker CRP-binding site ( site I from 119 to 100 ) , we would anticipate that competition between PaaX and cAMP-CRP for binding to the pga upstream region would influence pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In contrast , at class II promoters , the cAMP-CRP binding site is located around 241.5 and compensates for a missing 235 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Recently we have discovered that the cAMP-CRP complex acts as a repressor for the crp transcription , by binding to the unique site located in the crp leader region ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6344011" target="_blank">6344011</a>]<br></li>
<li>These data conﬁrm that cAMP-CRP speciﬁcally recognizes and binds to the CRP binding site located in site 1 , and not site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Next , we used a DNase I footprinting assay to verify the cAMP-CRP complex binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>Mutations with a putative cAMP-CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Deletion of a cAMP-CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>Interestingly , the cAMP-CRP binding site covers the putative 235 region and overlaps the downstream pho box ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Relocating the cAMP-CRP binding site in the csiD promoter region to positions between 280.5 and 260.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>We then analyzed the binding site of cAMP-CRP within the ascG-ascF spacer region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Identiﬁcation of the cAMP -- CRP binding site in the agaR-agaZ intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>We conclude that at least one of the R-subunits of RNA polymerase binds to DNA upstream of the -35 hexamer and that this interaction is unaffected by the position of the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Nevertheless , at least one of the R-subunits remains bound to the DNA upstream of the -35 hexamer for all positions of the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>Alternatively , both R-subunits could remain bound to the DNA upstream of the -35 region irrespective of the location of the CRP binding site ( Figure 10c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>On the other hand , when we introduce CRP - induced DNA bends at H159L different precise location between upstream enhancer sequences and the core promoter of glnAp2 promoter ( e.g. CRP binding site centered at 50.5 , 55.5 , 61.5 , ( a ) Supercoiled CC-55 .5 Supercoiled IC-55 Linear CC-55 .5 Linear IC-55 CRPH159L − + − − − + − − IHF − − − + − − − + Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Ratio of Activity + MelR / -- Mel Optimal MelR-dependent repression of the melR promoter depends on upstream sequences 1 Transcription initiation of the melR promoter is dependent on the binding of CRP to a 22 bp DNA site centered at position 241.5 , but sequences further upstream are not essential ( Webster et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760179" target="_blank">10760179</a>]<br></li>
<li>Nonetheless , the nucleo-tide positions at 5 , 7 , and 8 in the ﬁrst half and the symmetrically related positions ( 15 , 16 , 18 ) in the second half , reported to be critical for CRP binding ( 28 ) , are conserved in the target site in PcspE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>A comparison of the deoP-2 CytR target , which most likely overlaps the second CRP binding site around -90 , with that of EalOE shows indeed a close similarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The sequences of the primers used were as follows : proP P1 nucleotides − 103 to − 85 for detection of the proP crp site in pRJ4039 , lacZ nucleo-tides − 117 to − 99 for detection of Lac repressor binding in pRZ4004 , and pBR322 nucleotides 4333 -- 4351 for detection of CRP binding in pRJ1662 ( proP ) and pRJ1663 ( galP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>The cAMP CRP binding site is centered at position ÿ61 .5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>The cAMP ± CRP binding site is shown as a shaded box and CytR as an open box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The CRP binding site is denoted by a dotted line between the top and bottom DNA strands , with solid lines at the nucleotides most important for CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The underlined sequences indicate locations of the CRP binding site ( CRP I and CRP II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>The CRP binding site partly overlapped with a previously identiﬁed FNR binding site ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>As with PBAt ~ and PE , a CRP binding site , albeit one showing weak binding , was found upstream fl ` om and adjacent to an AraC protein binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>Much less is known about promoters where the CRP binding site is located further upstream and direct protein-protein contacts may be more difficult to establish .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>A triple mutation ( 2306T2307G2308T ) was created in CRP binding site 1 , in the downstream half of the inverted-repeat known to be important for CRP binding ( 11 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>( 2139A2152T ) was created in CRP binding site 2 , with one change being in each half of the inverted-repeat ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>( B ) DNA fragments used in the CRP binding assay and their relative positions with respect to the secB P2 CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>No potential FNR or CRP binding site sequences were detected directly upstream of the dcuA promoter , which is consistent with the expression studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Comparison between upstream sequences of mdh with that offumA ( Miles and Guest 1984 ) revealed a common CRP binding site which gave support that both enzymes are coregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3322223" target="_blank">3322223</a>]<br></li>
<li>Thus , these deletions show that sequences upstream of - 4 5 are essential to the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6408264" target="_blank">6408264</a>]<br></li>
<li>Open triangles show the position of the centre of the 5 ' - TGTGA-3 ' sequences contacted by CRP in each CRP-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2538630" target="_blank">2538630</a>]<br></li>
<li>Consensus sequences for the CRP binding site are indicated by lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329200" target="_blank">6329200</a>]<br></li>
<li>A putative CRP binding site , centered at position 117.5 , and the UP element are shown in bold and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In the case of hupA and hupB shown in Figure 6B , both have one binding site for CRP and four for FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>We attempted to convert the FNR1 site into a CRP protein binding site as a way to demonstrate the importance of the FNR1 site for anaerobic fdn expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>We examined this directly by removing the downstream CRP binding site by cutting with Rsa I at position +24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>For comparison we have included the most recently proposed CRP concsr.cjc binding site , which covers 22bp ( Ebright etat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Fis binding at site I plus CRP binding at 2121.5 coactivate transcription at P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>A weakly predicted FNR site and a strongly predicted CRP binding site are centered at 240.5 and 289.5 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Transcriptional activation by FNR and CRP : reciprocity of binding site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>DISCUSSION CRP activates rhaSR from at least one newly identiﬁed binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>CRP-activated promoters with a CRP binding site centered at -50.5 have not been observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>There is a CRP binding site centered at type , Acrp and Acya strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106445" target="_blank">8106445</a>]<br></li>
<li>We are currently improving the proposed melR CRP binding site with a view to testing this hypothesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>No specific binding of CRP to either mutated site was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>No specific CRP binding site was found in this fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>Restriction site protection of the Sau3A site within the ilvB CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Pr , probe ; BCRP , secB P2 CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>This periodicity of CRP binding site positioning suggests functionality .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>Panels , from left to right , show transcripts obtained from a wild-type ( WT ) DNA template , from a template with a mutation that greatly reduces CRP binding , and from a template with a 15 bp insertion at Fis site II between the CRP site and the P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>The locations of the speciﬁc DNA cleavages at the aer promoter are consistent with binding of the aCTDs at sites centred 18.5 bp both upstream and downstream of the DNA site for CRP ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Two CRP binding sites at the deoP2 promoter are centered -40 bp ( CRP-1 ) and 90 bp ( CRP-2 ) upstream from the P2 transcriptional initiation site ( Valentin-Hansen , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>The ﬁnding that the CRP binding site to the secB P2 promoter is centered at about 261.5 bp upstream of the 11 transcriptional initiation site indicates that it may be a class I CRP-dependent promoter like the lacP1 promoter ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>We compared the expression from the rhaS-lacZ fusions with CRP sites 2 and 3 bp upstream of the RhaR binding site to expression from a fusion with no CRP binding site [ ( rhaS-lacZ ) 85 ] and a fusion with a CRP site at the wild-type position [ ( rhaS-lacZ ) 128 ] ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>This ﬁnding indicates that CRP can function as a coactivator from a site 21 bp , but not 2 or 3 bp , upstream of RhaR , and is consistent with the hypothesis that RhaS and RhaR differ in the position of the CRP binding site that is optimal for coactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Oligonucleotides 2867 ( 5 - CACGAATTCTGTGATGCTCACCGCAGTATCT TGAAAAATCGACG-3 ) and 2868 ( 5 - CACGAATTCTGTGATGCTCACCG CATGTATCTTGAAAAATCGACG-3 ) were used with oligonucleotide 896 , which placed the CRP binding site ( underlined ) 2 bp and 3 bp upstream of the RhaR binding site , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>However , the CRP site 3 bp upstream of RhaS ( bound at the RhaR binding site ) enabled a 3.5-fold increase in expression relative to no CRP binding site [ ( rhaS-lacZ ) 85 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>This placed the CRP site 2 bp upstream of the RhaR binding site [ ( rhaS-lacZ ) CRP 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>Since the CRP site at rhaBAD is 3 bp upstream of the RhaS binding site , we placed the CRP site in the second rhaS-lacZ fusion 3 bp upstream of the RhaR binding site [ at 93.5 , ( rhaS-lacZ ) CRP 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>At the top , a schematic view of the bgl locus is given , including the positions of H-NS binding regions upstream of the promoter , the catabolite regulator protein ( CRP ) binding site , and 600 to 700 bp downstream mapping within bglG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Further more , CytR binding at the cddP promoter has been reported to result in repositioning of the upstream CRP binding site by 2 bp ( Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>A smaller , secondary activation peak is observed when the centre of the CRP binding site is at -63 , 2.5 bp upstream of the activation peak corresponding to a lactype promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The 750 bp EcoRI-BamHI fragment bearing the uidA regulatory region was also examined for a CRP binding site using the method of Fried and Crothers ( 1983 ) and by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>This 606-bp fragment lacks half of the potential CRP binding site centered near bp 2313 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_1075">
A 121 bp D N A fragment containing the specific CRP binding site for the uxu operon was used as a control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></label>
<input id="sit_similarity_1075", type="checkbox">
<div class="SIT_similar">
<li>As a contro we used a 121 bp D N A fragment containing the specific CRP binding site for the uxu operon ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li></div>
<li>The llvB operon falls within the latter group with its CRP binding site centered at bp -63 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>The decrease in transcriptional activation by CRP upon shifting the CRP binding site I bp away from the optimal position is indeed of this magnitude .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>EpKpx corresponds to a 3 bp substitution that inactivates the CRP binding site x ( Vidal-Ingigliardi & Raibaud , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>Base substitutions within the predicted CRP binding site were made via site-directed mutagenesis of pPK8278 to create pPK8429 ( bp 214 to 25 of Pfnr with a mutation in the CRP binding site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In addition , CRP could directly activate the fis promoter by binding at putative site I centered at 262.5 , which is close to the location typical for the CRP-activated class I promoters ( 49 ) , thereby displacing the polymerase molecule from the divergent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>In the case of the CRP site in the lac promoter , the G at position 7 and the C located symmetrically at position 16 are very important for binding of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045087" target="_blank">3045087</a>]<br></li>
<li>Nucleotide sequence homologies suggest that the CAMP-CRP binding site is located from bp -52 to -31 , in exactly the same position as at the gulp1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Analysis of the glpEGR sequence revealed several potential FIS binding sites , with one of them overlapping the CRP site for the glpE promoter and several other sites located further downstream ŽFig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>This inactivity might result from improper positioning of the a-CTD as observed at the malT promoter , whose binding site for a-CTD ( i.e. UP element ) and single binding site for CRP resemble CRP I of acsP2 and the DNA sequence located immediately downstream ( Tagami and Aiba , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Additionally , although CRP is unable to bind to the melAB promoter in the absence of MelR , in the presence of MelR , it binds to a site located between MelR binding Received 19 November , 1999 ; revised 21 January , 2000 ; accepted 24 January , 2000 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Ratios of variances were also separately calculated considering the data obtained for each binding site and set of effectors , e.g. , for binding of CytR to CytO of the wild-type promoter in the presence of CRP and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In this regard , we note that co-operative binding of CytR is considerably lower at a ` single CRP site ' cdd promoter than at an intact cdd promoter carrying tandem CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>A 135 bp BamHI-HpaII fragment , containing a consensus CRP-binding site derived from pPxSL04 ( Morita et al. 1988 ) , was end-labeled , mixed with 100 ng of CRP and an aliquot o sample in 10 ll of binding buffer ( 10 mM TRIS-HCl pH 7.9 , 50 mM KCl , 0.5 mM EDTA , 2 % glycerol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>A Equal amounts of a 32P end-labeled DNA fragment containing the consensus CRP binding site were incubated with various amounts ( 0 fmol , 45 fmol , 90 fmol , 180 fmol , 360 fmol , 720 fmol , 1440 fmol , and 2880 fmol , respectively ) of cAMP in 10 ll of binding buffer containing 100 ng of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9003304" target="_blank">9003304</a>]<br></li>
<li>34 , No. 20 6001 -- 6014 doi :10.1093 / nar/gkl734 Escherichia coli 's cAMP receptor protein ( CRP ) , the archetypal bacterial transcription factor , regulates over a hundred promoters by binding 22 bp symmetrical sites with the consensus core half-site TGTGA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Introduction Positive transcriptional activation by cyclic AMP receptor protein ( CRP ) involves binding of dimeric CRP to a 22 bp sequence at target promoters , with each subunit of the CRP dimer binding to an 11 bp half site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>In E. coli , the cyclic AMP receptor protein ( CRP ) is a global transcription factor that regulates numerous genes by binding to a 22-bp DNA sequence ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>The 187 bp DNA fragment contains the three CRP binding sequences CRP 3 , CRP 4 and CRP 5 ( see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>CRP did not bind the identical DNA fragment with substitutions in 4 bp of one of the CRP-binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>A shorter DNA fragment containing only site 1 was bound by cAMP-CRP ( Fig. 7C ) , and there was no binding of cAMP-CRP to a DNA fragment that contained only site 1 with substitutions in 4 bp in one of the CRP binding motifs ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Digoxigenin-labeled DNA fragments which contained CRP II ( the dotted sequence ) or changed CRP II ( with substitutions in 4 bp in the left CRP binding motif ) ( see Materials and Methods ) were incubated with 0 to 40 nM of puriﬁed CRP , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>Therefore , we next tested binding of CRP and CaiF to the 129 bp PF1 fragment , which contains the ® x sequence from 20 to ÿ107 covering the F1/CRP2 region ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>RESULTS Fig. 2 shows the CRP binding consensus sequence ( top ) as well as the five putative CBSs found in the mtl operon 322 bp Suu3AI DNA fragmentconblnlngCRP bindingaltoa 9.4 and6 : 162 180 I 3 6 - 82 204 5 EcoRV I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>An analysis of the DNA sequence in this region identiﬁed a possible CRP binding site , with 14 of 22 bp matching the consensus binding sequence ( 11 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A computer-assisted ` analysis of the DNA sequence of the HaeIII-generated restriction fragment shown in Fig. 3 revealed only one site which was significantly similar ( 16 out of 22 bp ; 73 % similarity ) to a proposed consensus sequence for recognition and binding of cAMP-CRP complex ( Fig. 3 , bottom ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>Although the presence of both core motifs was found to be necessary for CRP to bind to the target site in PcspE ( Fig. 3A ) , an intact second core motif was shown to be more important for CRP-PcspE interaction ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21926233" target="_blank">21926233</a>]<br></li>
<li>In this complex , CytR binds to an operator site ( usually referred to as CytO ) that is flanked by tandem CRP operators , CRP1 and CRP2 , as are found in most CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CRP binds separately to CRP1 and to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Four CRP target mutations in lac and gal have been sequenced ( the bases changed are boxed in Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>RESULTS CRP Binds with High Affinity to Three Sites in the Intergenic maLEp-malKp Region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1824723" target="_blank">1824723</a>]<br></li>
<li>Sequence LOGOs of the CRP , CytR-distal , CytR-proximal operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>C. Sequence of the/ac CRP target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>A new target for CRP action at 21 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>B. Sequences of CRP targets in nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>CRP Site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>1 2 3 ~ 4 5 6 7 8 ~ 9 1 0 1 1 # 1 1 1 0 9 ~ 8 7 6 5 4 ~ 3 2 1 A A A l T G T G A l T C T # A G A I T C A C A I T T T Conserved Consensus : 8 CRP 1 CRP 2 CRP 3 T A A l C A T G C l T G T # A G A I T C A C A l T C A 7 4 T C T I T G T G A I T T C # A G A I T C A C A I A A G A A A I T G T G A I C A C # T A C I T C A C A l T T T T T T I T G T G A I T G A # A C G I T C A C G I T C A T T A l T G T G A l T T G # A T A I T C A C A l C A A 10 6 1 0 3 c R P 4 CRP 5 9 4 1 0 A A A A A A Arg GluArg ArgGlu Arg 181 180 ... , ... , ... , ... , ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></li>
<li>-45 -35 TCTGCGATGTGATATTGCTCTCCTATG - ¬ CT AG AG AT T TT aaaTGTGAtCTagaTCACAttt CRP Consensus FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>C ( + I ) -10 BOX PROMOTER gapR P3 cAMP/CRP BINDINGJ SITE nnLI ri uu Inu ge/lP1 CTRATTTTTCCRTGTACRCRTTT consensus RROOGTUTCTRGRTRCSTTT FIG. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300536" target="_blank">8300536</a>]<br></li>
<li>De Crombrugghe et al. ( 1984 ) and Raibaud and Schwartz ( 1984 ) have noted that , in general , positively controlled promoters , or CRP activated ones , have particularly poor homology with consensus sequences , and also that the - 1 0 and - 3 5 regions are more than 17 bp apart .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2823062" target="_blank">2823062</a>]<br></li>
<li>A CRP dimer has been shown to recognize two 5-bp sequences ( consensus , TGTGA and TCACA ) that are related by a twofold rotational symmetry and that have a 6 - or 8-bp spacer region in between ( 2 -- 4 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>CRP , which is functional as a homodimer , recognizes 22 bp target sequences , with the consensus 5 ′ - AAATGTGATCTAGATCACATTT-3 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Between the coding regions of fucA and fucP , there are five stretches that match to various degrees the consensus cyclic AMP receptor protein ( CRP ) - binding sequences ( 15 ) - all with a core TGTGA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553671" target="_blank">2553671</a>]<br></li>
<li>We therefore scanned the regulatory regions of the remaining operons for sequences with a lower match to the consensus ( 5 as a cut-off score ) and found an additional 70 operons that contain CRP sites with a score between 5 and 10 ( Tables 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The two targets match the CRP core consensus ( TGTGA-N6-TCACA ) in 8 ( CRP-1 ) and 6 ( CRP-2 ) out of 10 possible positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The FLP consensus retains the same TGA/TCA core motif in each half-site , but differs from the FNR and CRP sequences in having four rather than six intervening nucleotides ( TGA N4 TCA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The following consensus sequences are indicated by the following symbols : Shine-Dalgarno ( GAAA ) , - 35 ( TTGACC ) and - 10 sequences ( TATTCT ) , CRP ( TGTGCN , CACA ) and pyridoxal phosphate ( w ) consensus sequences , PstI ( CTGCAG ) , Hind111 ( AAGCTT ) , Rho-independent terminator ( - * - ) ; aa determined by sequencing ( * ) , translation stop site of nupG ( I - ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7828866" target="_blank">7828866</a>]<br></li>
<li>The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF-lacZ fusion by these two proteins ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>The five putative CRP consensus sequences are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300537" target="_blank">8300537</a>]<br></li>
<li>The putative consensus sequences for the CRP , FNR , Fur , and FhlA/HyfR proteins are enclosed by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Boldface letters indicate putative CRP and FNR half consensus sequences in addition to presumptive recognition sequences of the s54 factor of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>T Arg180 G 4 ~ % Va1208 e r 2 ~ T S G l y l ~ ~ 8 '' ~ T T Lys188 '' -LCB- 9 1 4 ~ '' Gly216 '' - / 9 ... Arg185 ~ '' Met189 Arg213 - `` T - ' ~ '' Arg217 T w Rgure4 Diagrams showing the amino acid side chains in the DNA-recognition helices of FNR and CRP that are thought to interact with specific base-pairs in the core motifs of the corresponding half-site consensus sequences in FNR-and CRP-bindingsites1 ~ ,19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>CRP site Score Operon Site The AraC regulon E. coli CG D araBAD/araC 3.17 7.90 8.13 8.31 7.19 7.00 6.26 3.48 7.66 8.11 7.44 7.00 2.82 7.90 8.35 7.75 6.81 7.50 6.22 3.14 3.97 7.72 6.97 gAaaccAATtgTCCATA D TaaCAaAAgtgTCtATA - ( 4 ) - cgGCAGAAaAgTCCAcA TAGCAttTTtATCCATA - ( 4 ) - TAGCgGATcctaCCtgA cAaGGATTtccagGCTA - ( 4 ) - TATGGATTAATCTGCTg TATGtcTTtTcCcGCTA - ( 4 ) - TATGcAcgtTctcaCTg D ttATtTGcaCggcgTCACAcTT 3131 4.07 7 araFGH 7 cgATGTGATaTtGcTCtCcTaT AatTGgaATaTccATCACATaa tAtccTGcaagctATCACtTTa 3.67 4.25 3.21 D araE araJ cAGCAatTTAATCCATA - ( 4 ) - TgctgttTccgaCCtgA cAGCAGgATAATgaATA - ( 4 ) - ggGgcGAATtATCtcTt D CG D araBAD/araC S. typhi gAaacaAATtgTCCATA TAaCAGAAgtgTCtATA - ( 4 ) - TgGCtGgAatgTCCAcA TAGCAttTTtgTCCATA - ( 4 ) - TAGCgGATcctgCCtgA D D AtATtTGcaCagcgTCACAcTT AAgTGTaATaTccATCACATaT 3131 3131 4.05 4.78 D araE araJ TAGCcatTTAATCCATA - ( 4 ) - TgcCgttTccAgCCtgA TAGCAtAATAATgaATA - ( 4 ) - aAGacGAATAAggaATg ggcaccAATtgTCCATA TAaCAaAAgtgTCtATA - ( 4 ) - cgGCAGAAaAgTCCAcA cAGCAaAATAATCCATA - ( 4 ) - TAGCgaATccggCCtgA D D araBAD/araC K. pneumoniae UG D D AtATtTGcaCggcgTCACAcTT 3130 4.08 7 araFGH cAaGGATaAgcCTGCTg - ( 4 ) - TATaGATgATcCTGCTA TATGtcaTtTTtTGCTA - ( 4 ) - TATGtAcgcAatTaCTg cAGCAatATAgTCCATA - ( 4 ) - TgGCttATcctgCCtgA cATGaAcgAATCaGgct - ( 4 ) - TATGGAggAAaCTGCcg TAGCtGAATgtgaCATA 7 cgATGTGATaatGcTCtCgTaT AagTtTaATgTccATCACAaaa 3161 3131 3.63 4.41 D araE araJ 7 CG D araBA/araFGHC Y. pestis D TAGttaATTtATgCATA D cAGtttATTtATCCATA - ( 4 ) - cAGCAaATTtAaCCtcg aAGCAGAAaAgTgCATA - ( 4 ) - cAGCtatTggcTCCtcA D ActTtTGActgAcATCACAaaa 3146 4.17 yAGCakaWtwrTCCATA The AraC half-site consensus The XylR regulon E. coli CG D xylAB/xylFGHR xylFGHR/xylAB xylR xylE GTGAAtTATCtcAATaG - ( 4 ) - GTGAAATAACaTAATTG 3112 accAAAaATCGTAATcG - ( 4 ) - aTaAAAatctGTAATTG 3131 4.13 3.73 3.51 3.59 4.17 3.41 4.05 2.97 4.49 3.06 4.16 3.28 3.79 4.07 3.85 4.75 3.12 3.05 3.85 10.21 9.51 tttTGcGAgCgAGcgCACAcTT AAgTaaGATCTcGgTCAtAaaT gttctTGATtTtGATaAaAaTT tttcGTGcTCTgagTCACggca AttTtgGATaattATCACAaTT D 7 tttTTACGTTATTTgtt - ( 4 ) - CAcTTACGATAaTTCtC 3219 aaGAcATtACGTAAacG - ( 4 ) - GTaAAAaATgaTAATTG 3107 8.52 8.94 D tAtcacaATtaAGATCACAgaa tAAcGctAcCacGATCACATaa AtATaTcgTtggcgTCACAaaa tttTGaGAgCcAGAgCACATTT AAAaGcGATCgcGATCgaATcc D 7.04 8.21 9.55 8.36 yicJI yagGH aacAAgaATCaTAAaTt - ( 4 ) - accAgATATCGgAATat GcaAAATAACGTAATTc - ( 4 ) - aTaAgATATgaccATTG GgGgAtTcTCtTAATaG - ( 4 ) - GTGAAATAACGTAATTG acaAAAaAcCGTAATat - ( 4 ) - aTaAgAaATgacAATTG D CG D xylAB/xylR xylR/xylAB S. typhi D tgcTGTGATgaAttTCgCATaa tggcGctAcCctGATCACAgaa AAcTGTGAcCacctTagCAaaT AAAcGTGcgCcAGcTCgCAaaa AAATacGATCgccgTCAtAaTT D 8.00 7.63 9.04 7.84 9.06 9.34 7.44 7.42 yicJI xylKL aacAAgaATCGTAATTt - ( 4 ) - accAAATAACGgAATat actAAtaAACGcAATcc - ( 4 ) - GTaAgATtTaaTAATTG CAATTACGTTATTTCAC - ( 4 ) - CcATTgaGAgAaTcCAC acGAAATtTCaTAAcgG - ( 4 ) - aTaAgAaccgGTAATTG GaGcAtTtTCtcAATcG - ( 4 ) - GTGAAATAACGTAATTG D 7 K. pneumoniae UG xylFGHR xylFGHR xylA2/xynTB xynTB/xylA2 yicJI xylMN D D AttTGcGAgaTgGcTCACATTT cttTGcttcgagGATCACAgaa AttccctAcCgcGATCACAgaa AtgTGTGAcaacttTCtCAgTT D GTaAAAaAACaTAATTa - ( 4 ) - aTttAAaAACGTAATaG aacAAgaAACGTAATct - ( 4 ) - acGAgATATCGcAATTc aacAgAaAACaTAAacc - ( 4 ) - GcGAAAaATgacAATaG D D Table 2 ( Continued ) Genome Operon Site Pos .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>continued Operons Position Score Binding site ( s ) * S f R f pelD 279 2126 5.21 5.17 ggAcaAAAtggcGTTTCATTT AAATGAAACgcaGgTTtATTT pemB E. coli kduID 132 6.16 5.34 6.01 5.93 5.11 AAATGAAACATTGTTTtATTT AAAcGAAACAgTGTTTCAcTa AAATGAAACgTTGTTTtAaTT AAATaAAACAgcGTTTCAaTT tAATGgAACAcTGTTTtAaTa yjgK kdgT kdgK S. typhi kduID 6.15 5.2 5.62 4.59 AAATGAAACgTTGTTTtATTT AAtcaAAACAgTGTTTtgaTT AAATaAAACgcTGTTTtAacT tAATGgAcCgATGTTTtAaTa yjgK kdgK K. pneumoniae kduD/ogl 230 5.64 4.95 5.3 5.18 5.59 AAATGAAACATcGTTTtAaaT AAAcGgAACtcTGTTTtATTT AAtTaAAACggTGTTTtATaa tAATGgAACgATGTTTtAaTa AAATGAAAtgcTGTTTtATaT ogtABCD kdgK yjgK Y. pestis kduID pelW-ogtABCD 5.92 4.65 5 5.81 5.84 4.94 5.5 tAATaAAACATcaTTTCATTT AAtcaAAACAAcGTTcCgacT gAtTGAnACgATGTTTCtaTT AAATGAAACATTGTTTCtaTa tAATaAAACAgcaTTTCATTT AttTaAAACAccGTTTtAaTc AAATGgAAtggcGTTTCATTT AAATGAAACAnTGTTTCATTT ogl yjgK kdgK pmeA Consensus D. CRP regulation E. coli uxaB uidAB kduID uxaCA/exuT AAccaTGATCcgcgcCACAcTT tAATGcGATCTAtATCACgcTg AttcGTGATCgAcActgCAcTT AAAgGTGAgagccATCACAaaT tttatTGATCTAacTCACgaaa tAATaTGAcCaAccTCtCATaa AAATtTGAagTAGcTCACAcTT tttTGTGATCaAttTCAaAaTa tgATGTGgTtTtGATCACtTTT AttTGTGATgaAGATCACgTca tAAcGTGATgTgccTtgtAaTT AttgGTtAaCcAcATCACAaga ggATGTGAcaTtcATCgCAaca AAATtTGATtaAccgCACcTaa ± ± 112 2158 ± 3.61 4.21 3.4 4.15 3.9 3.85 4.73 4.62 3.95 4.44 3.55 3.58 3.56 3.78 g gntT S g gntKU kdgT R gntV/idnDOTR g gntP/uxuAB S UxuR S. typhi AttgGctAgCcAaATCACAaaT ttATtTGATCTgcgTCAatTTT AAcTtTGAagTAGcTCACAcTT AtATGTtAcCcgtATCAttcTT 3.95 ± ExuT/uxuABC 4.05 4.48 3.75 gntKU gntT Y. pestis uxuB/uxuA uxuR uxaCBA gntT/gntK 3.8 ttggtTGATCcActTCACgaaT AAcTGTtAcCTAccTCtaATaT AAAgtTGAgtgAcATCACAaaa AgATGTGActTttATCACAaca tAtcaTGATaccGgTaACAaaT tAtaGTGATCTgcATCACAgTT AAtcaTGAcgTgccTCAaAaaa tAtTGTGAcaTtcAaCtCAaaa AAATtTGATtTttATCAtAgag ± 2193 3.56 4.04 4.18 3.71 4.59 3.71 3.66 3.77 idnO-gntV kduID pelW-ogtABCD , pmeA E. chrysanthemi kduID g 4.23 3.49 3.81 4.43 3.45 2.95 4.34 ttgTGTGAaCaAGgTaACAcaa cAtTtTtATtgAatTCACATcT ttATGTGAaCggttaCACcaTT tttTGTGATCgAaggCACAaTa ttgTtTGcaagcGATCACtTTT AAATaaaAcCacGATCACggaa AAtTtTGAgCctGgTCgCAaaa ± S pelW g kdgT ogl outT S g S Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>At that , we gradually increased the number of aligned sequences , starting with closest E. coli relatives and then including more distant ones , in the order given by the phylogenetic tree of the CytR proteins ( Fig. 2 ) , while the OCRP sites could be reliably aligned and the distance between them remained approximately constant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>It contains OCRP sites flanking direct repeats of OCYTR [ 8,29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>To identify new candidate CytR regulon members , we used three recognition rules to select regions for construction and manual analysis of alignments , requiring either ( 1 ) two candidate OCRP sites at a distance 10 -- 40 bp , or ( 2 ) two candidate OCYTR sites at a distance not exceeding 20 bp , or ( 3 ) all four sites in the configuration OCRPD -- N ( 210 ) -- 20 -- OCYTRD -- N0 -- 20 -- OCYTRP -- N ( 210 ) -- 20 -- OCRPP , with negative numbers denoting overlapping sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>While the distance between the OCRP sites flanking OCytR is mostly conserved for the tsx orthologs in the Enterobacteriales , about 14 bp , the score of the 4-box cassette even in close relatives of E. coli is rather low , due to low scores of the OCRP sites ( about 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>coli is preceded by a cassette formed by two OCRP sites at a conserved distance ( 29 -- 31 bp ) and OCYTR sites in the correct arrangement , and this cassette is conserved in 17 related genomes , that is , in almost all Enterobacteriales and Vibrionales .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>The distance between the OCRP sites in the udp promoter is conserved ( 30 or 31 bp ) in almost all Enterobacteriales and Vibrionales ; the only exceptions with non-conserved intersite distances and an overall low score of the cassette are Photobacterium profundum , Photorhabdus luminescens and Vibrio fischeri .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>The SWAS plot has two pronounced peaks corresponding to the OCYTRD and OCYTRP sites with the conserved distance of 9 bp between them and an overlap between the latter and the proximal OCRP site ( Fig. 8 and [ 39 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by CRP</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>CSklaCZexpression in minimal media in response to starvation for carbon , nitrogen and phosphate sources and the roles of global regulators such as the alternative sigma factor ( encoded by 0 ' rpoS ) , cAMPICRP and the relA gene product were investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>Moreover , the chromosomal mafT gene can not be expressed in the Acrp background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>In both Acrp and Acya mutants , the derepression of , - galactosidase synthesis in the absence of glucose did not EC1631 cya + crp + 820 236 EC2098 + cya Acrp 185 214 EC2126 Acya crp + 213 152 a Bacteria for enzyme assays were grown for 4.5 h in Luria-Bertani medium supplemented with 0.5 % glucose , when indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>The ( malTpl malTpZ0-lacZ ' ) 1 operon fusion was constructed by allelic exchange between the chromosome of pop4127 ( malTpZ mafTpZO Acrp A h ) and the plasmid-borne ( mafT ' - lacZ +1 ) operon fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>The strain constructed by Miyada et al. ( 34 ) contained the ACRP mutation within the wild-type ara regulatory region , cis to both the araBAD operon and araC gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Genes most highly repressed upon IbsC , ShoB , LdrD and TisB overexpression.a Average fold repression dusB/yhdG ﬁs yeeF apt efeU ( ycdN ) CRP , IHF , Fis tRNA dihydrouridine synthase B Transcriptional activator Putative transporter Adenine phosphoribosyl transferase Inactive ferrous iron permease 6.5 7.7 0.9 0.9 0.8 0.9 1.2 1.1 1.1 1.2 1.2 2.1 4.3 3.8 4.7 2.3 1.2 17.8 6.1 21.2 CpxR Inactive ferrous iron transporter Inactive ferrous iron transporter Porin Unknown efeO ( ycdO ) efeB ( ycdB ) ompF yagU 14.3 2.6 6.1 26.7 1.0 1.8 1.4 1.7 1.2 1.8 0.9 2.5 1.2 1.2 0.5 1.2 OmpR , EnvYd a. Genes and operons which are repressed 5-fold in three independent experiments are listed with the average fold repression for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Natl. Aiba , H. : Autoregulation of the Escher & u coli crp gene : CRP is a transcriptional repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_8">
[ 15 ] Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></label>
<input id="tu_similarity_8", type="checkbox">
<div class="TU_similar">
<li>Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11555297" target="_blank">11555297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>References Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>References Aiba , H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>H. ( 1983 ) Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126461" target="_blank">15126461</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523115" target="_blank">14523115</a>]<br></li>
<li>Autoregulation of the Escherichia coli crp gene : CRP is a transcriptional repressor for its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2999070" target="_blank">2999070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2180916" target="_blank">2180916</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6193097" target="_blank">6193097</a>]<br></li></div>
<li>6 ( 1992 ) 3257 -- 3265 [ 31 ] Xu J. , Johnson R.C. , aldB , an RpoS-dependent gene in Escheri-chia coli encoding an aldehyde dehydrogenase that is repressed by FIS and activated by CRP , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>Xu , J.M. , and Johnson , R.C. ( 1995a ) aldB , an RpoSdependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>aldB , an RpoS-depen-dent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_19">
Xu , J. and Johnson , R.C. ( 1995b ) aldB an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by FIS and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></label>
<input id="tu_similarity_19", type="checkbox">
<div class="TU_similar">
<li>aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Xu , J.M. , and Johnson , R.C. ( 1995 ) aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li></div>
<li>Continued Gene product b Genes and promoter regions a Unknown and miscellaneous functions yccT Unknown Induced in stationary phase ( 76 ) b0964 b2447 b3679 b1586 b2602 b0255 b2625 1.51 1.52 1.53 1.54 1.55 1.56 1.57 yffP Predicted protein , prophage yidK Putative membrane transporter ynfD Predicted protein yﬁL Putative lipoprotein yi91a Unknown , in CP4-6 prophage sequence yfjL Unknown , possible prophage gene yagL Unknown , prophage protein sS ( 9 ) sS-Dependent in bioﬁlm-grow cells ( 18 ) eutH , in eutHA operon , is sS-dependent ( 9 ) sS ( 9 ) b0278 1.59 ing Reactivating factor for ethanolamine ammonia lyase Predicted outer membrane protein Hypothetical protein Unknown Unknown , mutant affecting swarming motility Phantom gene b2451 1.59 eutA b3524 b0625 b1964 b2510 b2596 1.61 1.66 1.69 1.74 1.79 yhjG ybeH yedS yfgJ G7353 Upstream of a ribosome modulation factor induced in stationary phase ( yﬁA ) yqiG Unknown , interrupted by IS element Intergenic region including ycbX promoter Unknown Unknown Intergenic region downstream of yfjH Unknown b3046 N.A. b0947 b2623 N.A. b1228 1.81 2.26 2.05 2.30 2.01 3.21 ycbX yfjH ychS Non-coding RNAs ryeE ( cyaR ) CRP , sE ( 77,78 ) b4438 Small RNA , promotes degradation of ompX and nadE RNA Small RNA , inhibits ptsG translation/SgrT protein sibD/sibE non coding RNA - ibsD/ibsE toxic peptides 1.50 sgrS/sgrT 1.51 b4577 SgrR protein ECs3934 1.62 b4447 b4664 Complex locus including two non coding RNAs overlapped by two small ORFs encoding a putative toxin/antitoxin system micA small RNA ( downregulates ompA expression ) not annotated in MG1655 Unknown , annotated in CFT073 Intergenic region between syd ( predicted protein ) and csrB ( ncRNA ) , antisense Unknown , annotated in CFT073 , downstream of malM Between fabB and ycfJ , antisense Unknown , annotated in O157 : H7 SAKAI Intergenic region , upstream of predicted helicase ygcB Unknown , annotated in CFT073 ( antisense of atpC ) Intergenic region upstream of gadA , antisense Unknown , annotated in CFT073 , upstream of aqpZ Unknown , annotated in CFT073 , upstream of dacA , transcribed in opposite direction Intergenic region upstream emrKY ( TolC-multidrug efﬂux pump ) Intergenic region upstream of yohJK ( inner membrane proteins ) Intergenic region between yehI-yehK , antisense Unknown , annotated in CFT073 ( antisense of menB ) Unknown , annotated in CFT073 , upstream of perM gene , transcribed in opposite direction Intergenic region , upstream of yhjB Intergenic region upstream of yqcE Unknown , annotated in CFT073 ( antisense of aspA ) Intergenic region upstream of rrsH ribosomal operon ( transcribed in opposite direction ) b4442 2.00 micA Intergenic regions , genes c3878 1.50 1.50 N.A. N.A. 1.50 N.A. c5008 ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_23">
CRP modulates fis transcription by alternate formation of activating and repressing nucleoprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></label>
<input id="tu_similarity_23", type="checkbox">
<div class="TU_similar">
<li>CRP modulates ﬁs transcription by alternate formation of activating and repressing complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12888342" target="_blank">12888342</a>]<br></li>
<li>CRP modulates ﬁs transcription by alternate formation of activating and repressing nucleoprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679232" target="_blank">14679232</a>]<br></li></div>
<li>In the Acya Acrp double mutant ( TP2339 ) , expression of ptsH and ptsl genes was no longer sensitive to cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2457575" target="_blank">2457575</a>]<br></li>
<li>the activation of t , he lac promoter and arnC ' B. 4 I ) 1 ) ~ CRP is similar in that CRT ' shows the S ~ IIIP orientation tlependcnce in both operons : as in nrtc ( ` H ~ ~ lO , CRT ' can partially activate the lnc promoter when its distance from the promoter is increased by 11 base-pairs , but not by five base-pairs , both in uiuo ( Mandecki $ Carut ' hers , 1984 ) and in r * itro ( St.raney et n / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>( 1999 ) Positive co-regulation of the Escherichia coli carnitine pathway cai and fix operons by CRP and the CaiF activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Upstream activating sequences that are shared by two divergently transcibed operons mediate cAMP-CRP regulation of pilus-adhesin in Eschenichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_30">
Goransson , M. , Forsman , P. , Nilsson , P. , and Uhlin , B.E. ( 1989 ) Upstream activating sequences that are shared by two divergently transcribed operons mediate cAMP-CRP regulation of pilus-adhesin in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8861216" target="_blank">8861216</a>]<br></label>
<input id="tu_similarity_30", type="checkbox">
<div class="TU_similar">
<li>Upstream activating sequences that are shared by two divergently transcribed operons mediate cAMP-CRP regulation of pilus-adhesin in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>and Uhlin , B , E , ( 1989 ) Upstream activating sequences which are shared by two divergently transcribed operons mediate cAMP-CRP regulation of pilus-adhesin in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1357528" target="_blank">1357528</a>]<br></li>
<li>B.E. ( 1989 ) Upstream activating sequences that are shared by two divergently transcribed operons mediate cAMP-CRP regulation of pilus-adhesin in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8231804" target="_blank">8231804</a>]<br></li></div>
<li>Within a total of 195 hitherto identified CRP targets in Regulon DB , the binding sites of 63 targets are located within intergenic regions of divergently transcribed operons ( Table 2 , goup-C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>It was unexpected that the lsr and lsrRK operons , which are divergently transcribed , are controlled by the same transcriptional regulators : CRP and LsrR ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>and malKlam & maEM operons are controlled by both MalT and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2016744" target="_blank">2016744</a>]<br></li>
<li>In both group-B1 and group -- B2 cases , we tentatively estimated that CRP regulates one of the divergently transcribed genes or operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Thus , it can not be excluded yet that for some of group-B1 and -- B2 cases , both of the divergently transcribed operons are regulated by cAMP-CRP bound within intergenic regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In addition to being repressed by CRP , AzuC-SPA levels were repressed under low-oxygen conditions , moderately induced during heat shock , oxidative stress , and thiol stress , and substantially elevated during growth in acidic medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>It should be noted that in the presence of glucose the levels of CRP and crp mRNA in cya - cells are than those in cells significantly higher wild-type ( Figures 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Expression of ClyA as evidenced by lysisa of erythrocytes in agar ClyA expression under the following conditions Relevant E. coli strain characteristic ( s ) Aerobic Anaerobic BSN26 Wild type BSN27 hns 2 2 11 11 11 ( 1 ) 11 ( 1 ) 11 ( 1 ) 11 11 11 11 11 11 11 ( 1 ) 11 ( 1 ) 11 ( 1 ) 11 11 11 11 MWK6 hns slyA JON31 hns JON32 hns crp BEU701 hns fnr BEU705 hns crp fnr JON32/pDW300 hns crp/p ( CRP ) JON32/pLG339 hns crp BEU701/pDW300 hns fnr/p ( CRP ) BEU701/pLG339 hns fnr BEU701/pGS24 hns fnr/p ( FNR ) BEU701/pGS215 hns fnr/p ( FNR-V208R ; S212G ; G216K ) BEU701/pGS297 hns fnr/p ( FNR-V208R ; E209D ) BEU701/pBR322 hns fnr BEU705/pDW300 hns crp fnr/p ( CRP ) BEU705/pLG339 hns crp fnr BEU705/pGS24 hns crp fnr/p ( FNR ) BEU705/pGS215 hns crp fnr/p ( FNR-V208R ; S212G ; G216K ) BEU705/pGS297 hns crp fnr/p ( FNR-V208R ; E209D ) BEU705/pBR322 hns crp fnr M182/pJON22 Wild type/p ( SlyA ) M182/pACYC177 Wild type M182 crp/pJON22 crp/p ( SlyA ) M182 crp/pACYC177 crp JON31/pJON22 hns/p ( SlyA ) JON31/pACYC177 hns MWK6/pJON22 hns slyA/p ( SlyA ) MWK6/pACYC177 hns slyA 11 11 11 ( 1 ) ( 1 ) ( 1 ) 11 11 11 ( 1 ) 1 1 ( 1 ) 1 ( 1 ) ( 1 ) 11 2 ( 1 ) 2 11 11 11 11 11 2 ( 1 ) 2 11 11 11 11 FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>As references , some known CRP target promoters such as ptsG , dgsA , ptsH and glpF ( shown under white background ) were examined in parallel under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Finally , introduction of the crp mutation in the other ycb , yfc , ybg , yra and yeh reporter strains revealed that CRP was also a positive regulator of these different operons ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>As additional negative controls , under the same test conditions , adding 0.5 mM IPTG to induce protein CRP or Crl did not significantly affect the value of the specific GFP fluorescence of the strain MG1655/pPyncC-gfp/pCA24N-crp or MG1655 / pPyncC-gfp/pCA24N-crl ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18309357" target="_blank">18309357</a>]<br></li>
<li>IHF can thus play an analogous role to that of CRP in facilitating NagC-dependent DNA loop formation at the divergent nagE-BACD operons ( Plumbridge and Kolb , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Transcription was reduced in the presence of CRP containing HL159 and this reduction was suppressed by KN52 confirming that AR1 is required at the downstream site ( Table 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>S Also , the distal UP-element half-site located downstream of the CRP box seems to have its share in Es S selectivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Although the glpD operators match the consensus verywell , the glpD operon maybe less sensitive to repressor ( relative to theglpFK operon ) because the operator sites are situated downstream of the CRP binding site and the -35 region of the promoter ( -10 to +30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>CRP was first identified as a transcription activator of the lac operon , where CRP binds at a site located 61.5 bp upstream of the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>CRP activates acs transcription by binding tandem sites located upstream of the major promoter , acsP2 , whereas , both IHF and Fis each act at different times during growth as anti-activators of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>CRP activates acs transcription by binding tandem DNA sites located upstream of the major promoter , acsP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>CRP activates transcription by binding to speciﬁc sites located upstream of promoters and contacting RNAP ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP is known to activate transcription by binding to sites located at various positions upstream of the 235 site ( from 241.5 to 2103 ) ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>This site is located immediately upstream of the RhaR-binding site and in the same relative position as the functional CRP site at rhaBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>When the CRP-binding sequences are located upstream or near promoters of the flanking genes , these genes are predicted to be under the control of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>When the CRP-binding site is located upstream of a single gene or operon ( and downstream of another neighboring gene ) , we predicted these downstream genes or operons as regulation targets of CRP ( Table 2 , group-A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>When CRP represses at the ompA and cya promoters , it is bound outside of its activation position and within the common zone of repression ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>These results are consistent with the cooperation of FIS and CRP in repressing the function of both the fis and divergent promoters observed in vitro ( Figs. 3B and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Furthermore the in vivo reporter assay indicated that both pgi ( glucosephosphate isomerase ) and gpmM ( phosphoglycero mutase III ) promoters were activated by CRP while the fbaB ( class-I fructose-bisphosphate aldolase ) promoter was repressed by CRP ( Fig. 5 ; for experimental details see Fig. 3 ) even though these genes were not detected by Genomic SELEX screening under the experimental conditions and the cut-off level of SELEX-chip pattern herein employed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Prominent examples are the bacterial nitrogen uptake regulating transcription factor , NtrC , which is phosphoryiated in its active form by NtrB ( Austin and Dixon , 1992 ; Sanders etai , 1992 ) , or the catabolite regulatory protein , CRP , which is activated by oAMP ( Botsford and Harman , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>One-third of operons subject to CRP-dependent glucose repression ( 13/39 ) in Gosset et al. were activated by CRP in ROMA experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_62">
Operons that are still significantly activated by HL159-CRP ( outliers ) are indicated in boldface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="tu_similarity_62", type="checkbox">
<div class="TU_similar">
<li>Operons that are still significantly activated by KE101-CRP ( outliers ) are indicated in boldface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li></div>
<li>Therefore only 14 % ( 25/176 ) of operons were activated by HL159-CRP to the same level as by wild-type CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_65">
Operons that are not activated by HL159-CRP or regulated in a reduced level ( outliers ) are indicated by ` # ' , and operons that are activated by HL159-CRP at the same level as by wild-type CRP indicated by ` -- ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="tu_similarity_65", type="checkbox">
<div class="TU_similar">
<li>Operons that are not activated by KE101-CRP or regulated in a reduced level ( outliers ) are indicated by ` # ' , and operons that are activated by KE101-CRP at the same level as by wild-type CRP indicated by ` -- ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li></div>
<li>Interestingly , a few known FNR-regulated operons , adhE , sdhC , sodA , cyoA and focA , were also activated by CRP in the ROMA experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Using ROMA , we identified 176 operons that were activated by CRP in vitro and 16 operons that were repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In the 17 CRP-activated operons , 9 operons are known members of the CRP regulon and 7 operons were activated by CRP in the ROMA experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Of the 188 operons activated by CRP , the promoter-proximal gene was activated in the vast majority ( 176 operons ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The deoC , nupG and tsx operons that are under CytR regulation were activated by CRP in our study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>araBAD , araC , araE , and papB promoters are also activated by CRP ; narG is also activated by nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Transcription is activated by CRP and repressed by a multi-protein CRPzCytRzCRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We found that transcription of the ykgMO operon is repressed by Zur and that azuC and ykgR transcription is repressed and activated by CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>azuC is repressed by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>ykgR is activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Expressionfrom the suc promoter was repressed by IHF and partially activated by d8but it was not regulated by ArcA , FNR , CRP , FruR or Fis , and not repressed by glucose or anaerobiosis , indicating that the well-established catabolite and anaerobic repression of ODH synthesis is imposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>This showed that the acnB gene is subject to repression by ArcA , Fis and FruR and activation by CRP , whereas the a c n A gene is repressed by ArcA and FNR , activated by FruR , CRP , Fur and SoxRS , and initiated by a38in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Further studies with acn-lac2 gene fusions and a wider range of transcription regulators indicated that acnA expression is initiated by G ~ ~ from PIacnAa , nd from PZacnA it is activated directly or indirectly by CRP , FruR , Fur and SoxRS , and repressed by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Regulation of acnB gene transcription Apart from being activated by CRP and repressed by ArcA , the acnB gene exhibited a very different pattern of regulation to that of acnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>For example , induction of the CFA/I operon by temperature requires the positive regulator CfaD [ 70 ] expression of the pap operon requires CRP and PapB [ 71 ] , the virB gene of Shi-gella is activated by VirF [ 49 ] and Fis plays a role in activation of the rrnB gene [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>This contrasts with fumA expression , which has recently been shown to be activated by the cyclic AMP-CRP complex and repressed anaerobically by the arcA ( dye ) - encoded regulator ( J. R. Guest and R. S. Buxton , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>These data suggested that hlyE expression is activated by FNR and CRP , repressed by H-NS , and that H-NS-mediated transcriptional silencing is relieved by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In contrast , acnB expression is activated by CRP and repressed by ArcA , FruR and Fis from PacnBC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>An additional negative control , CRP and the gadA promoter ( which lacks a CRP binding site ) , did not show CRP binding under the same conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>Expression was completely CRP dependent , as expected ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The negative control of speC by CAMP in a cell-free system was shown to be dependent upon CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>The developing model for the control circuit is that , when E. coli is actively growing under conditions where cAMP levels are high ( e.g. , exponential growth in media like LB medium ) , CRP will repress gadX by inhibiting RpoS production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The results indicate that CRP repression of gadX occurs primarily , but not completely , through its control over rpoS ( Fig. 7B , compare EF528 and EF678 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The data , in sum , suggest that the majority of CRP control over gadA and gadBC occurs through its control over RpoS , which , in turn , is needed to maximally express gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The CRP-McaS-csgD circuit forms an incoherent feedforward loop where CRP and McaS act in opposition ( CRP activates csgD and McaS while McaS represses csgD ) , while the regulatory circuit formed by CRP-McaS - flhD is a coherent feedforward loop ( CRP activates both mcaS and flhD while McaS activates flhD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>The stimulating effect of Lrp is not sigma factor speciﬁc , and it can only be observed when the CRP box is optimally located ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The changes induced by binding of CRP to its recognition site map to the boundaries of the CRP binding site ( Figure 2a , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8952481" target="_blank">8952481</a>]<br></li>
<li>The Hfq expression plasmid ( pSP73-Phfq-hfq ) containing the same P3hfq promoter region , and a putative CRP binding sequence , was constructed by insertion of the hfq gene ( including a 309-bp coding region and 25-bp terminator from the genomic DNA of E. coli wild-type BW25113 ) into a pSP73 vector , in which the T7 promoter was deleted by HpaI and BamHI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>The full-length rhaT promoter ( including both the CRP and RhaS-binding sites ) was ampliﬁed by PCR using primers 2097 and 2096 and whole cells of E. coli ECL116 as the source of template DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>The activity of PguaB ( 133 to 36 ) was measured in a crp strain containing a plasmid that did not encode CRP ( pDU9 ) ( A ) or a plasmid expressing crp under the control of the crp promoter ( pDCRP ) ( B ) or the bla promoter ( pHA7 ) ( C ) and growing in CRP media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>The plasmids that encode the mutant proteins ( CRP KE101 and CRP HL159 ) and the crp mutant cells were a kind gift of Nigel Savery ( University of Bristol , UK ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The results in Fig. 2B show that CRP alone is unable to activate the melAB promoter ( although it does induce the appearance of another shorter transcript ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>The number of target 0.5 represents such a particular case as fbaB , CRP.yegT ( see Table S3 ) , in which one ( fbaB ) of the divergent promoters is known under the control of CRP but possible regulation of the opposite promoter ( yegT ) by CRP can not be ruled out .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>The only examples of multiple promoter regions coregulated by CRP that do not have remote operators are ginApi and putA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The glnApi promoter is not a true exception since the other promoter is actually transcribed under the control of another sigma factor , sigma 54 , leaving the putA case as the only apparent current exception to the rule that multiple promoter regions coregulated by CRP must have operator duplications ( see Addendum in Proof ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Gene , 76 ( 1989 ) 255-269 Elsevier GEN 02886 Physical identification of an internal promoter , ilvAp , in the distal portion of the iZvGA4EDAoperon ( CAMP ; CRP ; recombinant DNA ; non-coordinate expression ; oxygen ; a-ketobutyrate ; alarmone ; E. coli K-12 ; S. typhimurium ; galK reporter gene ; transcriptional start point ) John M. Lopes * and Robert P. Lawther Biology Department , Universityof South Carolina , Columbia , SC 29208 ( U.S.A. ) Received by G. Wilcox : 10 June 1988 Revised : 5 October 1988 Accepted : 6 October 1988 SUMMARY It has been previously demonstrated that the ilvGMEDA operon is expressed in vivo from the promoters iZvGp2and iZvEp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2473940" target="_blank">2473940</a>]<br></li>
<li>Expression of the araJ gene encoding arabinose efﬂux transporter is under the control of an arabinose-inducible AraC and two molecules of CRP , one ( 90.5 ) upstream of AraC and another ( +44.5 ) downstream of the promoter ( Hendrickson et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>The main conclusions derived from this study are that 1 ) acid and stationary phase induction of gadA and gadB expression occurs at the transcriptional level ; 2 ) a single promoter drives the expression of each gene regardless of the inducing condition ; 3 ) the sigma factor σS is required for stationary phase induction , but not acid induction , of gad expression ; 4 ) CRP is a negative regulator of σS-dependent gad expression ; and 5 ) a conserved 20 bp sequence located between ® 52 and ® 73 bp in the gadA and gadBC promoters is essential for both acid and stationary phase induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_105">
Marschall , C. , and Hengge-Aronis , R. ( 1995 ) Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible s - dependent gene under positive control S of cAMP-CRP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></label>
<input id="tu_similarity_105", type="checkbox">
<div class="TU_similar">
<li>Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible S-dependent gene under positive control of cAMP-CRP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Regulatory characteristics and promoter analysis of csiE , a stationary phase-inducible sS-dependent gene under positive control of cAMP-CRP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li></div>
<li>The amino acid sequences used in the comparison were taken from the European Molecular Biology Laboratory ( EMBL ) data base and included those of factors acting positively on initiation of transcription such as CRP , Fnr , and OmpR of E. coli ; the spoOA , spoOF , spoOB , and spoOH gene products from B. subtilis ; and the sigma factors encoded by rpoD and rpoH of E. coli , rpoN of Klebsiella pneumoniae , and sigE of B. subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>We show that the hupA gene transcribed from a single promoter is positively regulated by both CRP and FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The actP gene , which was also overexpressed is part of the acs operon , is positively regulated by CRP and can be transcribed by 54 , 70 and 38 ( RpoS ) sigma factors [ 12,20,24,55 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The ansB gene is positively regulated by both anaerobiosis , through the FNR protein ( 20 ) , and by CRP ( 8 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The acnA gene is positively regulated by Crp , SoxRS , FNR and repressed by ArcA [ 59 ] , whereas acnB is positively regulated by CRP and negatively regulated by ArcA , FruR and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The phoA and p & E promoters are negatively controlled by the phoCT product : p & E and psi0 are repressed by t.he CAMP-CRP system ; and psi0 may be negatively controlled by a pho gene product that is required for the full induct ' ion of BAPase synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3540312" target="_blank">3540312</a>]<br></li>
<li>Therefore , two of the four types of regulation found for the uidA gene also apply to the uidR gene since this regulatory gene is repressed by its own product and is positively controlled by the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>The phenotype was complemented by both the wild-type crp gene and four crp alleles that encode proteins with amino acid substitutions in known activating regions of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>We conclude that the autoregulatory circuit of the crp gene plays a key role in the down-regulation of CRP by glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The downregulation of CRP by glucose would be a consequence of this dual autoregulatory mechanism of the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>It was concluded that the DNA and amino acid sequences presented here are those of the crp gene and its product , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>One of the documented mechanisms for negative regulation of a gene by CAMP and CRP is that reported for the crp gene ( Aiba , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>OD1 and OD2 are the operator sites for the glpD gene , and CRP is the interaction site for CRP ( dashed underlining ) ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>The hofB ( b0109 ; pilB homolog ) gene is adjacent t ppdD but does not contain any CRP site ; it and cloning-vector DNA ( data not shown ) served as negative controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>As evidenced by the lysis of erythrocytes in agar ( Table 3 ) , the absence of CRP , but not of FNR , reduced the level of clyA derepression in anaerobically grown hns strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Expression of dcuB is also repressed by glucose , is slightly repressed by TMAO , and in the absence of glucose is threefold induced by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Furthermore , in the absence of NarL , dcuB is repressed fourfold by nitrate in a manner that appears to be NarP independent , and , in addition , dcuB expression is reduced by glucose in a fashion that is mostly CRP independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>It appears that a CRP siteoverlapping the -35 region ( as in the glpTQ , glpFK , and glpACB promoters ) renders the operon more sensitive to catabolite repression than a situation with the CRPsite located furtherupstream ( glpD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>CRP prevents formatIon of the repression DNA loop in the araBAD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848302" target="_blank">1848302</a>]<br></li>
<li>These results demonstrate that propionate metabolism is subject to catabolite repression by the global transcriptional regulator CRP and that this regulation is effected through control of both the regulator gene prpR and the prpBCDE operon itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In addition , the CRP effect is dependent on a functional gcvA gene and its role may be to antagonize GcvA 's repression of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In the proposed network ( Fig. 6C ) , RpoS slightly represses the transcription of CRP itself known to activate the ﬂhDC operon ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>In electrophoretic mobi-lity-shift assays as shown in Figure 5A , the upstream DNA fragment of ﬂhDC operon can be shifted by puriﬁed CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>Likewise CRP was found to regulate NADH-ubiquinone oxidoreductase I encoded by the nuoABCDEFGHIJKLMN operon , NADH-ubiquinone oxidoreductase II encoded by ndh , and the cytochrome bo terminal oxidase encoded by the cyoABCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Furthermore the atpIBFHAGDC operon coding for the ATPase F1 and F0 complexes was also found to be under the control of CRP ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>We also dem-onstrate the role of CRP in the direct activation of the ulaAB-CDEF operon and show that ula regulon expression was not affected in an FNR mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>We do not know if this mechanism is also involved in CRP activation of the ilvB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>TcbC does not functionally replace any of the tdc operon regulators Expression of the tdc operon is controlled directly by a number of positive transcriptional regulators , including CRP , IHF , TdcA and TdcR ( Wu et al. , 1992 ; Hagewood et al. , 1994 ) , as well as being controlled indirectly by FNR ( Chattopadhyay et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The inability of CRP to activate malEp on its own is most likely due to the fact that none of the CRP sites is located at a permissive position ( Figure 1 ; Gaston et al. , 1990 ; Ushida & Aiba , 1990 ; Valentin-Hansen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>The metY-yhbC-nusA-infB-rbsA-truB-rpsO-pnp operon including the rpsO gene ( S15 ) was reported previously to be under the control of CRP ( 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>The results suggest that the glpTQ operon is most responsive to the glp operons to CAMP.CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>The glgCAP operon , encoding glycogen biosynthesis and degradation , is known to be regulated by CRP ( 2 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>, 1990 ) , and the mannitol operon repressor , which negatively regulates mannitol operon expression ( mtlR ; Figge et al. , 1994 ) , The regulatory region ot this operon is rather unusual in that it exhibits five putative binding sites for CRP ( Lee and Qaier .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577250" target="_blank">8577250</a>]<br></li>
<li>The activity of the CRP -RCB- cAMP-dependent bgl promoter is also enhanced and expression of the operon is activated by spontaneous mutations that disrupt a silencer sequence located upstream of the promoter , and by point mutations that improve the CRP-binding site ( Reynolds et al. , 1981 , 1986 ; Schnetz & Rak , 1992 ; Singh et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_142">
The Salmonella flhDC operon is also known to be activated by the cAMP -- CRP complex ( Kutsukake et al. , 1990 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></label>
<input id="tu_similarity_142", type="checkbox">
<div class="TU_similar">
<li>The flhDC operon of Salmonella is also positively regulated by cAMP -- CRP ( Kutsukake et al. , 1990 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li></div>
<li>The divergent glpTQ-glpACB operons are controlled by five operator sites , four within the glpACB promoter and one coinciding with the CRP site for the glpTQ operon ( Larson et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>The promoter of the pdhR-aceEF-lpd operon is repressed by pyruvate-sensing PdhR ( 33 ) and Cra ( this study ) and is activated by CRP ( 55 ; T. Shimada et al. , manuscript in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>The csgDEFG operon is part of the CRP regulon ( Zheng et al. 2004 ) and thus is activated in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23824318" target="_blank">23824318</a>]<br></li>
<li>Regulation studies with a dctA-lacZ ( f428-lacZ ) transcriptional fusion showed that dctA is subject to cyclic AMP receptor protein ( CRP ) - dependent catabolite repression and ArcA-mediated anaerobic repression and is weakly induced by the DcuS-DcuR system in response to C4-dicarboxylates and citrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Transcription of the glp operons is subject to multiple controls , including catabolite repression mediated by cyclic AMP-CRP and respiratory control mediated by the FNR and ArcA/ArcB systems ( 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>We are currently testing this hypothesis by measunng the amounts of crp mRNA and CRP protein in cells under various nutritional and growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>crp and cya genes encode the CRP protein and adenylate cyclase , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>As expected , there was no change in SecB when cAMP was added in the cya crp double-mutant RH77 , which lacks the ability to synthesize endogenous cAMP and its functional mediator , cAMP receptor protein ( CRP ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9023186" target="_blank">9023186</a>]<br></li>
<li>To determine whether the expression of the serC-aroA operon was subject to catabolite repression , lacZ expression was examined in cya and crp mutants deficient in adenylate cyclase and cAMP receptor protein ( CRP ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8018734" target="_blank">8018734</a>]<br></li>
<li>This positive and negative control by cyclic AMP-CRP is apparent in both the IHF + and IHF - strains ( Fig. 2A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>At least two regulatory classes of these genes exist : those whose induction is positively regulated by adenosine 3 ' ,5 ' - cyclic monophosphate ( cAMPf ) and its receptor protein ( CRP ) ( the cst genes ) and those independent of this control ( the pex genes ) ( Schultz et al. , 1988 ; Schultz & Matin , 1988 ; Blum et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>Expression of the gal operon is regulated by cellular levels of CAMP and the CAMP receptor protein ( CRP ) through the mechanism of catabolite repression ( for review , see de Crumbrugghe and Pastan , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7023696" target="_blank">7023696</a>]<br></li>
<li>Expression in rich media is largely dependent on the alternate sigma factor σS and is repressed by the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11238978" target="_blank">11238978</a>]<br></li>
<li>Expression from the galS promoter was stimulated by the addition of D-fucose , repressed by glucose , and dependent on cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>The 110-nucleotide Rep RNA is not regulated by the cAMP-CRP complex ( 32 ) , and it was not regulated by araC protein and thus served as an internal control in these experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>The induction proﬁle raised the possibility that the yjiY gene might also be regulated by cAMP and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>Upregulation in E. coli is mediated via activation of the flhDC operon by cAMP -- CRP ( cAMP receptor protein ) ( Yokota & Gots , 1970 ; Silverman & Simon , 1974 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>( A ) The organization of the operon and its regulation by the positive regulator cAMP receptor protein ( CRP ) and the negative regulator GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>The cAMP receptor protein , CRP , is required for activation of ptsG transcription ( De Reuse and Danchin , 1988 ; Fox et al. , 1992 ) , whereas the negative regulatory protein Mlc represses ptsG transcription in the absence of glucose ( Kimata et al. , 1998 ; Plumbridge , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>The acsyjcHactP and aceBAK operons are activated by the cAMP receptor protein ( CRP ) -- cAMP complex ; additionally , transcription of the former operon occurs from the acsP2 promoter , which is dependent on CRP ( Wolfe , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375081" target="_blank">24375081</a>]<br></li>
<li>This suggests that the transcription of the hya operon is not activated by the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>For example , ptsG transcription is activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) ( 2 ) , and negative control is exerted by the repressor Mlc , which inhibits ptsG transcription in the absence of glucose ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873911" target="_blank">23873911</a>]<br></li>
<li>The more extensively studied system is dependent on cyclic AMP ( cAMP ) and its receptor protein , CRP , which activates the transcription of many catabolic operons ( Busby and Kolb 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Clearly , acs transcription responds to catabolite repression through cAMP and its receptor protein , CRP ( 249 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In these operons , cAMP and its receptor protein On ( CRP ) act aspositive effector for transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>ftnA -- lacZ expression is dependent on the growth phase , iron and Fur The cAMP-receptor protein ( CRP ) does not control ftnA or fur expression It was previously reported that the fur gene is induced approximately fourfold during post-exponential growth in rich medium in a manner that is largely dependent on the cAMP -- CRP ( cAMP-receptor protein ) complex ( de Lorenzo et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Choi , S. Kawase , M. Kawamukai , H. Sakai , T. Komano , Regulation of glpD and glpE gene expression by a cyclic AMP-cAMP receptor protein ŽcAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Regulation of glpD and glpE gene expression by a cyclic AMP-cAMP receptor protein ( cAMP-CRP ) complex in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Unlike phoA and appA , agp expression in vivo is not influenced by inorganic-phosphate in the medium , and opposite to appA , it is positively controlled by cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2153660" target="_blank">2153660</a>]<br></li>
<li>A number of genes induced in stationary phase require cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) for their expression ( for a review , see reference 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>At the onset of glucose starvation , cyclic AMP and its receptor protein ( cAMP-CRP ) were found to play important roles in the expression of a number of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Expression of Pa promoter is also influenced by the cAMP receptor protein ( CRP ) , which allows expression of the mhp catabolic genes when the preferred carbon source ( glucose ) is not available but 3HPP is present in the medium ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19520845" target="_blank">19520845</a>]<br></li>
<li>This indicates that the cyclic AMP ( cAMP ) - CRP complex weakly activates dcuB expression in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>This is particularly severe for nagE whose expression is very low in the absence of a functional cAMP/CAP complex ( CAP = catabolite activating protein , also known as CRP = cAMP receptor protein ) ( Ptumbridge , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>With the exception of fadL each is also positively controlled by cyclic adenosine monophosphate ( cAMP ) and cAMP-receptor protein ( CRP ) [ 88 ± 90 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>We show here that gntP is induced by fructuronate , regulated by UxuR , and requires activation by cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>We present evidence that galS is ( i ) negatively autoregulated and ( ii ) positively regulated by cyclic AMP ( cAMP ) receptor protein ( CRP ) and that galR is constitutively expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>171 , No. 2 Effect of Mutations in the Cyclic AMP Receptor Protein-Bindi Site on araBAD and araC Expression LORI STOLTZFUSt AND GARY WILCOXt * Department of Microbiology , University of California at Los Angeles , Los Angeles , California 90024 ng Received 27 July 1988/Accepted 16 November 1988 Maximum expression of the adjacent but divergently transcribed araBAD operon and araC gene requires the presence of cyclic AMP ( cAMP ) and the cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>KEINOSUKE OKAMOTO AND MARTIN FREUNDLICH Department of Biochemistry , State University of New York , Stony Brook , NY 11794 ABSTRACT Expression of the crp gene is negatively autoregulated by the complex of cyclic AMP and its receptor protein ( cAMP-CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The dadAX operon is expressed by multiple promoters that are repressed by leucine-responsive regulatory protein ( Lrp ) and activated by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_185">
The dcuB gene is strongly induced anaerobically by FNR , repressed in the presence of nitrate by NarL , and is subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="tu_similarity_185", type="checkbox">
<div class="TU_similar">
<li>The dcuB gene is strongly activated anaerobically by FNR , repressed in the presence of nitrate by NarL , and subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li></div>
<li>unusual , since only in rare cases does cyclic AMP-CRP both activate and inhibit expression from an individual operon ( 1 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Effect of altering the putative upstream promoter on ilvB expression DISCUSSION Transcription of the ilvB operon is positively activated by cyclic AMP and its receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>One operon , araFGH , codes for the well-studied AraC protein and by cyclic AMP ( cAMPt ) receptor protein ( CRP ) ( Englesberg et al. , 1965 ; Greenblatt & Schleif , 1971 ; Wilcox et al. , 1974 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2231717" target="_blank">2231717</a>]<br></li>
<li>The data obtained indicate that the expression of gyrA is growth phase dependent and under the positive control of the cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>The operon is also positively controlled by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex and negatively by the catabolite repressor / activator ( Cra ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>This operon has also been reported to be positively regulated by cyclic AMP-cyclic AMP receptor protein complex ( cyclic AMP-CRP ) ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>tre operon expression is under the control of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex as well as that of TreR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>As expected , gut operon expression is under the control of the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>ptsG but not crr is subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_196">
This operon is also subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></label>
<input id="tu_similarity_196", type="checkbox">
<div class="TU_similar">
<li>The operon is subject to positive control by cyclic AMP and the cyclic AMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The manXYZ operon is also subject to positive control by the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li></div>
<li>In E. coli , the nanATEK-yhcH operon is positively controlled by cyclic AMP receptor protein ( CRP ) and negatively regulated by the upstream nanR-encoded transcriptional repressor , NanR ( 6 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Taken together , these data strongly suggest that the ilvB biosynthetic operon is negatively controlled by multivalent transcription termination and is positively regulated ` by cyclic AMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>A number of in vivo experiments suggest that the ilvB operon is positively controlled by cyclic AMP and CRP ( 4 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>In a previous report on the stringent and growth-rate-dependent control of the gua operon [ 8 ] , the presence was noted of a cyclic AMP receptor protein ( CRP ) site centred at position 3117.5 with respect to the guaBA transcription start site and overlapping the translational start site of the divergent xseA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>In vitro transcription from the ilvB promoter was greatly increased by cyclic AMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>Regulation of transcription of the araBAD and araFGH operons depends upon promoter occupancy by cyclic AMP receptor protein ( CRP ) ' and AraC protein ( 6 , 8 , 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447222" target="_blank">1447222</a>]<br></li>
<li>( b ) In strain S764 , the bgl -- lacZ fusion carries the same activated bgl promoter allele , bgl-CRP + , as is present at the bgl operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>It has been reported that even though the main role of the CAMP-CRP complex is to activate transcription , it can also , in some cases repress it , as it does in the case of the P2 promoter in the gal operon , and the ompA and clp promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2465204" target="_blank">2465204</a>]<br></li>
<li>These examples with single operators upstream of -45 are the promoters of the CytR regulon , the spf promoter ( see CRP in Tables 1 and 2 ) , and perhaps the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>However , in the absence of CRP , transcription of the gal operon can initiate from the upstream galP2 promoter ( 26 ) , which will mask the weak transcription from galP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Activationof the E. coil ansB promoter,CC-FF , and its derivative FF-FF , by mutant FNR or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Expressionof the E. coil ansB promoter,CC-FF , in the presenceand absence of CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The transcription from both hupA and hupB-P3 promoters is reproducibly decreased by a factor of around 2 in the absence of active CRP , while the hupB-P2 promoter is unaffected ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>At the rhaSR promoter , the 70 L595A derivative was unique in that it was defective in both the presence and absence of the second activator , CRP , but it 's role in RhaR-mediated transcription activation is not yet known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Expression from the melAB promoter in the absence or presence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>In vitro transcription from the rhaSR promoter in the presence ( ) and absence ( ) of CRP , RhaR , and L-rhamnose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>In the absence of CRP , Ea70 transcribes mainly from the P2 promoter , while in the presence of CRP and cAMP , EaY70 transcribes efficiently from the P1 promoter , but P2 transcription is repressed ( Fig. 6 , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>Expression from deoP2 , the cdd promoter ( cddf ) and the udp promoter ( udpf ) is enhanced by the CAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>These experiments demonstrate that CRP represses the xseA promoter and activates the guaBA promoter in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>One can imagine scenarios in which CytR remains bound to DNA , if not to CRP , under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>proP P2 transcription was activated approximately 70-fold in the presence of both Fis and CRP ( Figure 6 , lanes 4 and 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
<li>Coactivation of proP P2 by Fis mutants and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>Model of coactivation of proP P2 by Fis and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>There are changes detected by the potassium permanganate probing in between the 10 and the 35 seen in the presence of CRP that are not seen in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12881519" target="_blank">12881519</a>]<br></li>
<li>Regulation of the divergently transcribed promoters by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>-10 sequence is efficiently transcribed by EaY38 even in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>Both templates are transcribed the to same extent in the absence of CRP ( lanes 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>on ara CRP Represses spf , cya , ompA , galp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Thus , CRP can not act on its own in this system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592420" target="_blank">7592420</a>]<br></li>
<li>For example , after a 0.4 M upshift in LB about 30 % of the CRP site becomes accessible to DMS modification ( Fig. 4 ) , and proP P1 transcription is induced to 70 % of its maximal value as assayed by LacZ fusions ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>araFGH is highly sensitive to catabolite repression and is not capable of significant transcription in the absence of CRP either in vivo or in vitro ( 29 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400237" target="_blank">1400237</a>]<br></li>
<li>In these conditions , recruitment of Es28 by CRP may ensure that the aer and trg transcription units are prefer-28 entially expressed compared with other s - dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In these cases , it is not ruled out that CRP regulates transcription toward opposite directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Regulation of divergently transcribed genes by CRP In E. coli , it is rare that a single and the same regulator controls divergently aligned promoters , which direct transcription toward opposite directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>This successive appearance of HU mRNAs is independent of FIS which regulates the hup genes FIS and CRP regulate the transcription of both hup genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>This raises the question of why CRP should directly regulate transcription of aer and trg , particularly when CRP indirectly regulates transcription of all genes in the ﬂagellar cascade by controlling expression of the master regulator , FlhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Expression of the E. coli K-12 melAB genes , which encode products essential for melibiose metabolism , depends on transcription activation by MelR and CRP , and is triggered by melibiose ( 7,9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>A total of 70 genes for transcription factors are organized under the control of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>A number of genes for uncharacterized transcription factors are under the control of CRP ( black background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>the other hand , it has been shown recently that CAMP-CRP ' represses transcription of another set of genes ( 4-6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2982847" target="_blank">2982847</a>]<br></li>
<li>Also , evidence has accumulated to indicate that CAMP-CRP complex represses the transcription of as many genes as it activates ( Mallick and Herrlich , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>Thus it was concluded that the CAMP-CRP complex is required to activate transcription of both the a c n A and acnB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>These results support the hypothesis that CRP activates acs transcription by means of class I interactions and not via class II interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>Since FIS represses crp transcription ( 155 ) , it can preclude acs transcription by restricting the availability of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>A. Schematic showing acs transcription ( acs , TRXN ) , growth ( OD ) and the relative concentrations of active CRP , FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Similarly , the effect of CRP on the induced level of setA and sgrS transcription was mediated through SgrR-dependent signaling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>First , some operons require other factors for CRP to activate transcription , e.g. AraC at the araBAD promoter ( 25 ) , which are missing in the ROMA system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>CRP is required to activate guaBA transcription and repress xseA transcription following downshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>If IHF can activate transcription by bending the DNA it is quite likely that DNA-bending by CRP has the same effect , namely transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Cyclic AMP-CRP is a positive or negative regulator of transcription of numerous operons in E. coli ( 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>( iii ) Most of the sugar transport operons , whose transcription is activated by CRP , are also under the control of repression by other transcriptional regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>CRP has also been shown to repress transcription of some genes , including the crp gene itself and cyaA , encoding adenylate cyclase ( 2 , 25 , 26 , 52 , 85 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>When the melR gene is expressed from a CAMP-CRP-independent promoter , melibiose-dependent transcription from the mel4B promoter becomes independent of CAMP-CRP , showing that the meIR promoter is the primary site of control by CAMP-CRP in the me1 regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Expression of the melR gene depends on transcription activation by CRP , whose activity depends on cyclic AMP ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li>
<li>The araC gene is regulated by CRP activation of transcription and by autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>It has also been shown that FNR induces an 82-92 ° bend in target DNA ( Ziegelhoffer and Kiley , 1995 ) but , as with CRP , factor-medi-ated DNA bending may not be essential tbr transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Thus , these data suggest that under such conditions , FNR and CRP are in competition for binding at PhlyE , and that the simplest explanation of these observations is that FNR is more efﬁciently bound than CRP at PhlyE but is not as effective as CRP in activating hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Nevertheless , the regulatory analysis of csi-18 : : / acZw , hich is a single-copy fusion to glgA at its natural positionin the chromosome , seems to indicate that at least under the conditions tested here neither as nor cAMP/CRP nor relA play a major role in the transcriptional regulationof the operon of which g/gA is a part .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>Because of its cAMP-CRP dependency , csiDp is activated exclusively under carbon starvation conditions and stationary phase in LB ( Weichart et al. , 1993 ; Marschall et al. , 1998 ; Germer et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731280" target="_blank">14731280</a>]<br></li>
<li>Expression of the plas-mid-encoded fusion showed the same dependency on sS and cAMP-CRP as the original csiD : : lacZ fusion , indicating that the entire relevant regulat-ory region is present on pCM3 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Likewise , the collection of next abundant 38 clones carried the intergenic sequence between yibI and mltA , both being located downstream of the cAMP-CRP binding sequence , of which the mtlA operon encoding mannitol-specific PTS permease is known to be activated by cAMP-CRP [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>In addition , since this is the first operon shown to be controlled by attenuation and cAMP-CRP , it was feasible to investigate © IRL Press Limited , Oxford , England .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>In summary , the data in the present report show that the dadAX operon is expressed from at least three major promoters , each positively controlled by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>11 when Spot42 sRNA is expressed at full strength ( not repressed by cAMP -- CRP ) and the gal operon is transcribed at the maximum rate possible in the absence of cAMP -- CRP and GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Transcription of the gal operon is initiated from two overlapping promoters , P1 , in the presence of the activating complex cAMP/CRP , and P2 , in its absence ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>In such conditions , the transcription of crp may be moderately repressed by the binding of CRP-cAMP to the downstream CRP binding site I. Upon depletion of glucose , the cAMP level would increase resulting in the increased level of CRP-cAMP complex to allow the occupation of CRP-cAMP at the CRP binding site II , and the transcription of crp is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>Discussion It is well established that transcription of crp is negatively autoreguiated by the CRP-cAMP that binds to downstream CRP site I ( Aiba , 1983 ; Okamoto and Freundlich , 1986 ; Hanamura and Aiba , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>It was shown conclusively , by using cya-lac fusion strains , that one of the cya promoters , P2 , is negatively regulated by the cAMP-CRP complex and that the decrease in cya transcription under repressing conditions is about fourfold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Taking these results together , we conclude that crp transcription is regulated both negatively and positively by CRP-cAMP depending on the concentrations of CRP and cAMP in vivo as well as in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The major finding is that transcription of crp is not only negatively but also positively regulated by CRP-cAMP , and upstream CRP site II is responsible for this positive autoreguiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The data also suggest that transcription from this start site is positively regulated by PhoB and negatively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>At P1 and P2 transcription is repressed by the DeoR repressor and induced by deoxyribo-se-5-P ; at P2 transcription is also repressed by the CytR repressor ( the inducer being cytidine or adenosine ) and depends on cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Three promoter/operator regions have been demonstrated : at P01 and P02 , transcription is repressed by the deoR repressor and induced by deoxyribose-5-P ; at P02 , is also the transcription repressed by cytR repressor , induced by cytidine or adenosine , and depends on cAMP and CRP ; at P03 , transciption is controlled by unknown protein factor ( s ) and induced by inosine or guanosine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16453417" target="_blank">16453417</a>]<br></li>
<li>At these D-galactose concentrations the level of active GalS would depend on both the cAMP -- CRP concentration and the active GalS level itself , which represses galS transcription ( Weickert and Adhya , 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>specific promoter ( P5 ) which is sensitive to glucose repression and which is regulated by cAMP and CRP is involved in the control of rpoH transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>It was therefore of interest to determine directly if CRP binds to the ilvB promoter and to further investigate the effect of cAMP-CRP on iji vitro transcription in this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>In vitro transcription from the ilvB promoter was markedly increased by the addition of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>In contrast to lac and tna , ilvB transcription is not completely dependent lji vivo ( 6 ) and iji vitro ( 2 ; Fig. 2 ) on cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Transcription from the gal PI promoter ( equivalent to ompA P2 ) is stimulated by the cAMP/CRP complex whereas transcription from gal P2 is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6298577" target="_blank">6298577</a>]<br></li>
<li>Regulation of the P2galE promoter , located 5 bp upstream of P1galE , is opposite ; transcription from this promoter is further enhanced by OE-bound GalR and repressed by cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>As expected , transcription from P1galE was enhanced and that from P2galE repressed by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>With the galP35 mutant which does not bind cAMP - CRP , the P2 transcription in the Acya background ( 45 % ) was the same as that observed with galP + Acya ( 44 % ) ( Table 2 , line 5 versus line 3 ) cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537823" target="_blank">2537823</a>]<br></li>
<li>A common feature of these FIS-regulated operons was that their transcription is activated by cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Surprisingly , we also observed Es70-mediated and cAMP-CRP / Lrp-stimulated csiD transcription when supercoiled templates were used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>To investigate whether or not transcription of fadL was stimulated by CRP and cAMP , these compounds were also used in the transcription reactions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>We showed that cAMP and CRP negatively regulate luxS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>cAMP and CRP requirement of negative regulation of spf transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>cAMP-CRP stimulation of ilvB transcription ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>The CRP group mainly contains transcription factors that regulate catabolite-sensitive genes in response to glucose starvation as signalled by cAMP ( Busby & Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>In the presence of catabolite-repressing sugars such as D-glucose and D-gluconate , cAMP and CRP levels are low and transcription of the idn genes is not induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>PTS not only transports specific sugars but also in the absence of its substrates , stimulates , through adenylate cyclase ( Cya ) , the production of cAMP which in turn activates the transcription of many cAMP-CRP dependent catabolic genes , including those involved in glycerol utilization [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Importantly , cAMP-CRP also activates the transcription of acs ( Kumari et al. , 2000a ) , which means that both genes are upregulated at the same time .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>The CRP encoded by the crp * GQJ41 allele is relatively independent of cAMP in activating cAMP-CRP-dependent promoters ( 12 ; J. Kim , S. Garges , and S. Adhya , unpublished data ) , and the Acya mutation eliminates changes in cAMP concentration in response to starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2163388" target="_blank">2163388</a>]<br></li>
<li>In addition , the data from these experiments are completely consistent with the in vitro results : negative autoregulation of crp transcription ( Fig. 4B ) and positive activation of the divergent transcript ( Fig. 4A ) by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>In contrast to the normal fragment , the addition of cAMP-CRP did not activate the divergent transcript when the altered template was transcribed and no inhibition of crp transcription was found ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>When CRP site II was mutated , the transcription of crp was strongly inhibited by increasing concentrations of cAMP and the activation was no longer observed ( Fig , 8A , lanes 6-9 ) , The analysis of divergent RNA showed that the mutation in CRP site II did not affect the activation of divergent RNA ( Fig. 8B , lanes 6-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328816" target="_blank">1328816</a>]<br></li>
<li>The data in Fig. 2B show that crp transcription was not blocked by cAMP-CRP when the template lacked the downstream binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The osmotic sensitivity of CRP binding to DNA is not mediated by changes in cAMP binding The presence of cAMP is required for repression by wildtype CRP as proP P1 transcription is constitutively expressed in a Dcya crp + mutant ( Fig. 7B ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>Autoregulation of sxy transcription depends on CRP , cAMP and a CRP-S site As Sxy itself is not sufficient to induce expression from Sxy-dependent promoters but also requires CRP and cAMP for transcriptional activation ( Cameron and Redﬁeld , 2006 ) , induction of the sxy : : lacZ fusions by Sxy overexpression was re-examined in crp and cyaA strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>Since the recipient is cya - crp - , enzyme production must also occur in the absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=181362" target="_blank">181362</a>]<br></li>
<li>Although the down-regulation of crp expression by glucose can be explained by the autoregulatory loop , we also noticed that glucose slightly but consistently reduces the level of CRP independently of cAMP or the crp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>cAMP-CRP Activates an RNA Transcribed in the Opposite Direction to the crp Gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>cAMP-CRP does not block crp expression if the new promoter is altered so that divergent RNA can not be made .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The divergent RNA requires cAMP-CRP for expression ; it strongly blocks crp transcription ; and , if this RNA is not made , cAMP-CRP inhibition of crp expression is abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>We have shown that this divergent RNA strongly blocks crp transcription and that a functional divergent promoter is necessary for cAMP-CRP inhibition of crp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The activation of the divergent promoter by cAMP-CRP and its proximity the suggested that the divergent to crp promoter promoter might play a role in crp autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>These data suggest that the negative autoregulation of the crp gene is a direct consequence of the activation of the divergent promoter by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>These results indicate that the negative autoregulation of the crp gene is a direct result of the activation of the divergent promoter by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Taken together , the data suggest that the RNA activated by the presence of cAMP-CRP is not transcribed by the crp promoter but by a different promoter that allows transcription in the opposite direction to the crp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>This altered template retained crp promoter function but crp transcription was no longer inhibited by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>Since CRP is activated by cAMP that is produced in the absence of glucose , detailed analysis was focused on CRP-dependent regulation of the genes for the ribosome modulation factors RMF and RaiA , which are induced during growth transition from exponential to stationary phase ( 19 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Expression of these genes is activated in the absence of glucose through functional conversion of CRP into an active form after interaction with an effector cAMP [ 17 ] , which is synthesized by the membrane-bound adenylate cyclase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>Again , whereas induction of ppdD , yrfD and sbmC absolutely required both CRP and Sxy , mglB was induced by exogenous cAMP to the same levels in the presence or absence of Sxy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>While in the absence of cAMP-CRP , EsS is superior in binding to the csiD promoter ( and its binding is not further improved by the presence of cAMP-CRP ) , cAMP-CRP stimulates the relatively weak binding of Es70 ( Figure 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Transcription levels were normalized to the divergent p82 promoter , which does not respond to RhaR , CRP , L-rhamnose , or cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>When fragment 2 , which reduces the downstream sequence of fragment 1 by about 20 bp , was -100 -50 transcribed , cAMP-CRP activated the formation of the same 75-nucleotide RNA ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425359" target="_blank">2425359</a>]<br></li>
<li>The synthesis of most of these proteins is also affected strongly by positive regulation through the cAMP -- CRP complex and the enzymes are highly expressed under appropriate growth conditions ( i.e. conditions in which rpoH-P5 is activated ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The results also showed that P2 is the major promoter of crr and is not activated by the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The nupG promoter is activated 50-fold by cAMP-CRP , and repressed fourfold by CytR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The P1 promoter is activated by the positive regulator CRP in the presence of cAMP ; the P2 promoter is repressed by CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>Transcription is activated in response to intracellular cAMP levels by CRP , repressed by CytR , and induced by cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The cst genes including cstA are activated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>ppGpp activated the synthesis of glgC independently of cAMP and CRP ; however , its effects were more pronounced in the presence of these factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Therefore , it can be activated by cAMP -- CRP even in the absence of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The 3-bp deletion in the cAMP-CRP binding site did not reduce expression or affect catabolite repression of the araBAD operon and hence experiments were focused on the regulation of the araC gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>Our data are not consistent with the proposal that the ara-BAD operon and the araC gene share a common site for cAMP-CRP-mediated regulation ( 4 ) or that the site deleted by ACRP is involved only in araBAD regulation ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>This suggests that glucose repression of araBAD is not mediated by cAMP-CRP bound to this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>This suggested that the hyb operon is subject to catabolite repression mediated by the cAMP-CRP system ( Busby & Kolb , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Indeed , the homology between FNR and CRP is such that their DNA-binding specificities can be interconverted by appropriate substitutions in the recognition helices , such that catabolite repression can be lifted in response to anoxia and anaerobic gene activation can be induced by glucose starvation ( cAMP ) ] 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The cAMP/CRP complex is a positive transcriptional regulator of a number of catabolic operons , and as such , plays a role in catabolite repression , whereby secondary carbon sources are not catabolized in the presence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>The sdaB gene is expressed only in rich medium in the absence of glucose , under the control of the cAMP/CRP catabolite repression system [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>There is evidence for a c & acting cAMP/CRP regulatory sequence upstream from the possible promoter nucleotide sequences , and preliminary results also indicate that the gene for glucosamine-6-phosphate deaminase , nagB , is transcribed in the opposite orientation to nagE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3284790" target="_blank">3284790</a>]<br></li>
<li>The ompA and ompF genes encoding membrane proteins and the deo gene involved in nucleic acid metabo-lism are also under the control of the cAMP-CRP complex [ 1-3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>( III ) The rotA ( ppiA ) gene is also controlled by the CytR / CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The expression of the gut operon is known to be regulated by a repressor , the product of the gutR gene which is located downstream from thegut operon3and an activator , the CAMP-CRP complex ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3553176" target="_blank">3553176</a>]<br></li>
<li>Expression of the operon is thought to be regulated by a repressor , the product of the gutR gene which is located downstream from the operon ( Csonka C Clark , 1979 ; Yamada & Saier , 19878 ) , and the cAMP-CRP complex ( Lengeler & Yamaguchi School of Medicine , Ube , Yamaguchi , Japan 755 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3062173" target="_blank">3062173</a>]<br></li>
<li>The RNA is transcribed from the spf ( spot forty-two ) gene , whose expression is negatively regulated by the cAMP -- CRP complex ( Joyce and Grindley 1982 ; Rice and Dahlberg 1982 ; Polayes et al. 1988b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></li>
<li>The gntR gene is constitutively expressed , whereas the gntKU genes are positively regulated by the cAMP -- CRP complex and negatively by GntR , and gntU expression is further repressed by attenuation , which occurs after gntK ( Izu et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9358057" target="_blank">9358057</a>]<br></li>
<li>gene system , the exwession of g/pD and g/pE was found to be positively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>CAT fusion analysis revealed that the operon has a promoter for gntR and another for gntKU , and that the gntR gene is constitutively expressed , while that of gntKU is regulated positively by the cAMP-CRP complex and negatively by GntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>( ii ) Both the araBAD operon and the araC gene are positively regulated via binding of the cAMP-CRP complex to a single site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>The cAMP-CRP complex positively regulates the malT gene and the malEGF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>( i ) The in vitro synthesis of the first dipeptide of glgC is enhanced by cAMP and CRP when the gene is present on an HpaI restriction fragment which contains 0.5 kb of DNA upstream from the glgC initiation codon and the glgC and gigA coding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The cAMP receptor protein CRP can function as an osmoregulator of transcription in Escherichia coli Lenore Landis ,1,4 Jimin Xu ,3,4 and Reid C. Johnson1 ,2,5 1Department of Biological Chemistry , School of Medicine , University of California , Los Angeles , Los Angeles , California 90095-1737 USA ; 2Molecular Biology Institute , University of California , Los Angeles , Los Angeles , California 90095 USA Transcription of the P1 promoter of the Escherichia coli proP gene , which encodes a transporter of osmoprotectants , is strongly induced by a shift to hyperosmotic media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>de Quimica Fisica ` Rocasolano ' , Serrano 119 , Madrid 6 , Spain Communicated by H.Buc In Escherichia coli , the transcription of the malT gene is activated by the complex formed between cAMP and its receptor protein , CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832158" target="_blank">2832158</a>]<br></li>
<li>Molecular mechanism of negative autoregulation of ABSTRACT Transcription of the Eschenchia coli crp gene encoding cAMP receptor protein ( CRP ) is negatively regulated by CRP-cAMP complex that binds to a specific site located downstream from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>We also examined their role in the regulation of the malT gene , which encodes MalT and which is controlled by cAMP-CRP ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The csiD gene provides a prototype of such a promoter ( csiDp ) , as it is nearly completely dependent on cAMP-CRP and Es in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In this phase , CRP is inactive due to the absence of cAMP , and acetyl-CoA synthetase gene is not expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25524697" target="_blank">25524697</a>]<br></li>
<li>( a ) Transcription of the cyaR gene is dependent on cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li>
<li>Transcription of the spf gene encoding the Spot42 sRNA is reduced 3 - to 5-fold in the presence of cAMP-CRP ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>cAMP-regulatory protein ( CRP ) Many cellular signalling pathways operate through the production of cAMP , which can regulate DNA transcription by binding to cAMP-regulatory protein ( CRP ; also known as catabolite gene activator protein , CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16814929" target="_blank">16814929</a>]<br></li>
<li>It is interesting to notice that the transcription level of the acsyjc-HactP operon and maeB gene , both activated by CRP , was downregulated in strains PB11 and PB12 as compared to JM101 strain ; again , this effect could be associated with the reduced cAMP concentrations in the PTS -- strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18679018" target="_blank">18679018</a>]<br></li>
<li>Lee et al. ( 13 ) proposed the involvement of cAMP-CRP in transcription activation of both the divergently transcribed prpR gene and the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>It is possible that the CRP -- cAMP complex may activate transcription of another gene ( such as dnaK ) which , in turn , acts as an anti-s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>to the proposed model , the cAMP-CRP complex binds to this single site in the ara regulatory region to activate transcription of both the araC gene and araBAD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>In this study , we identiﬁed , for the ﬁrst time , that cAMP-CRP bound downstream initiation represses transcription of the pck gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>For example , in the galactose utilization network of E. coli , transcription of both the spf gene and the gal operon is simultaneously regulated by cAMP -- CRP ( 11 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>One region is for the efficient transcription necessary of the spf and the other participates in the control by gene , cAMP-CRP in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>DISCUSSION We have shown here that spf gene transcription is nega - tively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2454912" target="_blank">2454912</a>]<br></li>
<li>The cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex stimulates the expression of lsrR and the lsr operon , which includes lsrACDBFG , while LsrR represses their expression and is located adjacent to , while being transcribed divergently from , the lsr operon [ 14-16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>In this report , we show that both AI-2 synthesis and uptake in Escherichia coli are subject to catabolite repression through the cyclic AMP ( cAMP ) - CRP complex , which directly stimulates transcription of the lsr ( for `` luxS regulated '' ) operon and indirectly represses luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In this study , we show that catabolite repression inﬂu-ences AI-2 accumulation through the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which directly stimulates transcription of the lsr operon and indirectly represses luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Regulation by cAMP-CRP Complex The effects of the cAMP-CRP complex on the expression of the GntII genes were examined by using singlecopy lacZ operon fusions of the gntV and idnD genes , which were constructed on the E. coli genome in the cents cya background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>Expression of lsrR and the lsr operon is inhibited by LsrR and stimulated by the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>cAMP-CRP stimulates expression of the lsr operon by directly binding to the upstream region of its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In E. coli , the cAMP-CRP complex stimulates the expression of both lsrR and the lsr operon by binding to the promoter regions , while LsrR represses their expression [ 14 , 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>The cAMP-CRP complex stimulates the expression of lsrR and the lsr operon , while LsrR represses their expression [ 14 , 15 ] , suggesting that cAMP , the CRP , or both may be affected by sdiA-ydiV double mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>The cAMP-CRP complex stimulates expression of both lsrR and the lsr operon of E. coli , which are divergently transcribed , by directly binding to the two different CRP binding sites in the intergenic region [ 14 , 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18560382" target="_blank">18560382</a>]<br></li>
<li>The repressing effect of cAMP-CRP may be responsible for the low increase of osmY expression in stationary-phase cells starved for glucose in minimal medium ( which is in pronounced contrast to a more than 30-fold induction in LB medium ) , because the at onset of starvation the synthesis of cAMP is drastically stimulated under these conditions ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The ; 30-fold postexponential repression of dctA expression by glucose in L broth observed under aerobic conditions was largely reversed by the addition of cAMP ( from 0.17 to 2.9 mmol/min/mg ) , which provides a strong indication that dctA is activated by the cAMP-CRP complex ( Fig. 8C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>When D-galactose is present , cAMP-CRP can activate P1galE and repress P2galE ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>We found that transcription of galR is independent of cAMP -- CRP concentrations ; therefore , its input function resembles a TRUE gate with different levels of expression in the presence and absence of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Expression of the enzymes encoded by the galETKM operon can be described by the sum of the P1galE and P2galE input functions , which results in a ` D-galactose ' gate , with lower level and dis-coordinated expression of the proteins in the absence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Expression of the psiE9-9cat fusions ( pKE135 , pKE136 , and pKE137 ) was reduced in DphoB strain transformants when transformants were grown in the presence of cAMP in both Tris-glucose LP and HP media ( Table 1 ) , suggesting that expression of these constructs is also negatively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>DNA sequences involved in expression and regulation of CYTR and cAMP/CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>m e role OfcAMPin the expression ofcsi : : lacZ fusions Many carbon starvation-induced proteins identified on two-dimensional gels or by the analysis of lac2 gene fusions are not expressed in cya or crp mutants and cAMP/CRP has been implicated as a major regulatory system for starvation-induced genes ( Matin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>If so , we would expect that GalS will be induced at lower galactose concentrations than GalR , since the mgl operon is the more efficient transport system at low galactose concentrations , ( ii ) The galS gene is autoregulated and its expression is CRP cAMP dependent , while gaIR expression appears to be constitutive -LCB- Weickert and Adhya , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>In the maltose system , the lamB , malK , and malEFG operons encoding the transport system are more dependent for their expression on the cAMP-CRP system than the malPQ operon encoding amylomaltase and maltodextrin phosphorylase ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2834341" target="_blank">2834341</a>]<br></li>
<li>For example , the stationary phase expression of cgsA ( encoding the curlin subunit gene ) is modulated by both a ® and HN-S ( Olsen etai , 1993 ) , and osm / ( encoding a periplasmic protein ) is activated by a ^ and repressed by Lrp , cAMP-CRP and IHF in stationary phase cells ( Lange et ai , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Transcription of these genes is activated in response to intracellular cAMP levels by the cAMP receptor protein ( CRP ) , a transcription factor that activates expression of over 100 E. coli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Interestingly , expression of both genes is activated by cAMP-CRP and repressed by CsrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>Furthermore , the gal regulon genes are differentially regulated by cAMP-CRP and D-galactose , and in the absence of D-galactose the expression of the mglBAC operon can be activated by cAMP-CRP , whereas the transcription of galP gene is strongly repressed [ 38,39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>These in vitro results conﬁrm the previously reported in vivo observations that expression of these three genes is highly dependent upon cAMP and CRP ( Weickert and Adhya , 1993b ; Geanacopoulos and Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>These results indicate that , unlike prpBCDE expression , expression of prpR is not regulated by propionate or its catabolism but is regulated by glucose or glycerol availability , suggesting a classic catabolite repression mechanism mediated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Our results suggest that the cAMP-CRP complex acts as a transcriptional factor to repress the expression of hfq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>These results suggested that the glucose effect on ulaG expression is mediated by a mechanism independent of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Since it is known that the amount of the cAMP-CRP complex increases under glucose starvation conditions such as Tris-glycerol medium , these results suggested that expression of psiE is negatively regulated by cAMP-CRP independent of the phosphate concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Our results show that the expression of sxy in E. coli is positively autoregulated via a mechanism dependent on CRP , cyclic AMP ( cAMP ) and a newly identiﬁed CRP-S site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>On the basis of in vitro and in vivo experimental data , Preiss and co-workers have proposed that expression of E. coli glgCAP genes is positively regulated at the transcriptional level by the cAMP/CRP complex [ 19,21,45 ] ( reviewed in [ 11 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>These results and the occurrence of a putative cAMP-CRP binding site suggest that the gntK gene expression is positively regulated by cAMP-CRP at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>These results indicated that cAMP-CRP positively regulated the expression of gipD and glpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>These results suggest that the expression of the luxS gene is negatively regulated by cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>These results , together with measurements of the in vivo psiE promoter activity under various conditions , show that expression of the psiE gene is under direct positive and negative control by PhoB and cAMP-CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Taken together , these results demonstrate that csiE is under positive control of both as and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Under these conditions any gene under positive cAMP/CRP control should be at least weakly induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>It is well known that the growth-dependent expression of some genes is under the control of the cAMP-CRP complex ( 19 , 25 , 26 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>It is possible that cAMP-CRP may control another transcriptional regulator ( s ) that modulates luxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>This negative influence of cAMP can be attributed to the presenceof os , which plays a positive role in csiE expression and which is itself under negative control by cAMP-CRP ( Lange and Hengge-Aronis , 1994a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>This work represents an initial contribution to the elucidation of the role of cobB and yﬁQ gene products in the control of the metabolism of E. coli , describing for the ﬁrst time how their expression is regulated and co-ordinated by central metabolism regulator CRP and the concentration of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>Discussion We have shown here that expression of the stationary phase-induced gene csiE is under the control of the os subunit of RNA-polymerase and the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>We examined the gene expression of glpD and glpE under the control of cAMP-CRP using glpD-iac and `` glpE-lac fusino genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>In addition to regulation by its own protein product , araC gene expression is under positive control of the cAMP receptor protein ( CRP ) ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6377308" target="_blank">6377308</a>]<br></li>
<li>This indicated that arb gene expression is indeed under cAMP-CRP complex-mediated positive control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11994151" target="_blank">11994151</a>]<br></li>
<li>This variation is further extended by the fmding that cAMP/CRP can act not only as a positive effector but also as a negative regulator of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Because Sxy is thought not to bind DNA , we also examined gene expression in cyaA cells to test whether Sxy might act by overriding-CRP 's dependence on its allosteric effector cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>This could be elucidated by the facts that the trxA-lac fusion of plasmid pJH3 did not contain the cAMP-CRP binding site , indicating that the presumed cAMP-CRP binding site is in vivo functioning negatively in the expression of the trxA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9175752" target="_blank">9175752</a>]<br></li>
<li>Although , it is generally believed that CRP is synthesized constitutively and its function is controled by the level of cAMP ( 11 ) , little is known about the expression of the crp gene , which codes for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6280140" target="_blank">6280140</a>]<br></li>
<li>CRP * mutants which can actively affect gene expression without binding cAMP were isolated by many groups ( 3 , 10 , 11 , 13-15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>In the gal operon , one of two overlapping promoters is activated ; the other is inhibited by the complex ( Adhya and Miller , 1979 ) , and expression of the ompA gene in vitro is likewise inhibited by cAMP/CRP ( Mowa et al. , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Thus , if the level of cAMP is high in cells cultured in the presence of 0.04 % glucose , negative regulation by the cAMP-CRP complex probably occurs in the puuR gene expression ( Shimada et al. 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>It was shown that expression of the fumA gene resembles that of the succinate dehydrogenase operon and other ` citric acid cycle ' genes in being repressed by glucose ( i.e. , dependent on the cAMP-CRP complex ) and anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3282546" target="_blank">3282546</a>]<br></li>
<li>It was reported that expression of the pts operon and the ptsG gene increases a few fold during growth on glucose or other PTS substrates ( 1 ) , and the functional cAMP CRP complex is essential for the expression of the ptsG gene ( 1 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15252051" target="_blank">15252051</a>]<br></li>
<li>Glucose is known to affect gene expression through the cAMP-CRP complex , and the presence of glucose in the growth medium results in decreased levels of cAMP and CRP ( 26 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Taken together , our data reveal that glucose can destabilize the ompA mRNA by reducing the cAMP-CRP complex-medi-ated activation of hfq gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>Although Wang et al ( 2005 ) observed a negative regulation of luxS expression by cAMP and CRP , an alternative explanation is MicA involvement as it is up-regulated in a cya-deficient background [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20976191" target="_blank">20976191</a>]<br></li>
<li>As an example , expression of the cts group , which is induced in the stationary phase or under carbon starvation conditions , is dependent on the cAMP-CRP complex ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Expression of slp during carbon starvation and stationary phase was dependent on neither cAMP/CRP nor cS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
<li>Interestingly , although the effect of cAMP was already observed in the exponential phase of the cultures , the highest changes occurred upon entry into stationary phase , suggesting that CRP is not the only factor necessary for gene expression ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>In stationary phase wild-type cells , the inhibitory effect of cAMP-CRP on the expression of os and therefore of csiE is overcome by several stimulatory effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>While it is obvious that cAMP-CRP is required for full expressionin stationary phase , crp and cya mutantsexhibited increased levels of expression in exponential phase ( Figs 1 , 2 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Induction of lrp expression in cells entering stationary phase does not require cAMP and CRP since cya and crp mutants have lrp expression levels that are the same as or higher than those of the wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>These data indicate that cAMP-CRP represses the expression of osmY and that this repression plays a role during an early period of the transition into stationary phase when the growth rate is just beginning to slow down .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>cAMP/CRP can also play a negative role in controlling the expression of stationary phase-induced genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>cAMP-CRP thus determines the time of induction of osmY , but it is apparently not involved in determining the final level of osmY expression in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In summary , expression of yjiY is induced at the transition of the E. coli culture into the stationary-growth phase , and induction requires cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>Second , the remainingexpression in the A ( -38 ) construct was entirely dependent on as.Finally , in the Acrp background , expression of the A ( -72 ) and A ( -38 ) constructs was very similar , suggesting that the cis-activatingelement located between position -72 and -38 - mediatesthe activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Shifting the CRP box further upstream or downstream ( up to 276.5 and 262.5 respectively ) reduced csiD expression 20-fold and more ( the basal expression obtained with non-optimal positions was no longer cAMP-CRP dependent ; Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The expression of yﬁQ occurs from its own promoter ; it is upregulated in the stationary phase and in the presence of non-PTS carbon sources and is positively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>Figure S2A in the supplemental material shows that , under high-glucose conditions , the cpxA24 mutation still represses cyaR-lux expression , while the cpxR mutation no longer has an effect , suggesting that the cpxR mutation affects cyaR expression via cAMP/CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>1981 ) , Four different transcripts originate in its upstream regutatory region , and it has been shown that in vitro expression of gIgC is stimulated by cAMP/CRP and weakly stimulated by ppGpp ( Romeo and Preiss , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1324388" target="_blank">1324388</a>]<br></li>
<li>In the presence of CRP , expression of glpD and glpE increased 5 - and 3-fold upon the addition of cAMP , respectively ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846566" target="_blank">1846566</a>]<br></li>
<li>The maximal stimulation of expression by cAMP and CRP observed in the experiment shown in Fig. 2 was 26-fold for glgC and 10-fold for gigA ; ppGpp increased the expression 3.6 - and 1.8-fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Of particular interest were mutations that resulted in CytR-dependent constitutive udp expression when intracellular cAMP-CRP concentrations were limiting ( glucose medium ) ( del224 , del364 , dup1 , and dup361 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Since cstA expression is dependent on cAMP-CRP ( 42 , 43 ) , lower expression was observed throughout the growth curve in both the wild-type and csrA mutant strains under this growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867454" target="_blank">12867454</a>]<br></li>
<li>Thus , the observed expression patterns for pckA , tnaA , aspA and others under conditions of aerobic acetate formation can be explained by cAMP-CRP dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Altogether , these data suggest that a reduced level of cAMP-CRP in hns and hns-stpA strains may be one of the reasons why malE and lamB expression is reduced in hns and hns-stpA strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>This indicated that ulaG is not affected by CRP and further supports the participation of another mechanism independent of cAMP-CRP in the expression of both operons of the ula regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>It has been proposed that the expression of rpoS is not only negatively controlled by cAMP-CRP high levels but also inversely correlated with growth rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The expression of MBP and LamB is also activated by the CRP-cAMP complex ( 31 ) , whereas the expression of OmpA is constitutive and thus affected little by CRP ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>The ability of both holoenzymes to be activated by cAMP-CRP from the 260.5 position makes it difﬁcult to distinguish how much of the csiD expression in the strain that contains both holoenzymes results from each of them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The 235 site is relatively poor , and this would explain why dctA expression is weak in the absence of cAMP-CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>The expression of tnaA is under cAMP and CRP regulation ( 78 ) and is also inducible by high pH ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293424" target="_blank">17293424</a>]<br></li>
<li>csiD expression is also strongly dependent on cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The mel locus is induced by several a-D-galactosides ( 2,3 ) and its expression is controlled by the cAMP-CRP regulatory circuit ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3031590" target="_blank">3031590</a>]<br></li>
<li>The activity of CA8306 ( cya ) harboring these clones was higher in the presence of cAMP , suggesting that gntK expression is controlled by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The expression of fMet-Gln , the amino-terminal dipeptide of gigA was enhanced 2.4 - to 2.8-fold by cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>The expression of the protein acetyltransferase PatZ is regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>In an rpoS mutant , csiE expression is significantly but not totally reduced , and can still be stimulated by cAMP-CRP ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>There are at least two regulatory systems involved : rpoS , which controls gIgS ; and cAMP/CRP , which controls the expression of gIgCAP ( Romeo and Preiss .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1324388" target="_blank">1324388</a>]<br></li>
<li>DNA helix phase dependency of cAMP-CRP activation of csiD expression mediated by Es or Es .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Changes in cAMP-CRP levels affected only the expression of the ulaABCDEF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>At least three models can be postulated to describe the mechanism by which cAMP and CRP affect expression ` of cir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160948" target="_blank">2160948</a>]<br></li>
<li>cAMP and CRP negatively regulate the expression of luxS but not pfs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Elimination of the cAMP-CRP site in -180 did not change the pattern of expression under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Both cAMP and CRP are required for expression of glgS and PTS ( phosphotransferase system ) - related genes required for normal glycogen production [ 22,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21029047" target="_blank">21029047</a>]<br></li>
<li>Both genes are regulated by CRP , which activates their expression in response to the elevation of cAMP levels upon glucose starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25524697" target="_blank">25524697</a>]<br></li>
<li>In order to conﬁrm the role of CRP on the expression of cobB , patZ and acs genes , cAMP response experiments were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li>
<li>cAMP-CRP , a positive regulator , stimulates expression of both operons in the absence of glucose , while LsrR prevents their expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In the absence of glucose , cAMP-CRP is needed to stimulate lsr expression , while LsrR represses its expression in the absence of the inducer phospho -- AI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>Interestingly , we found that the expression of pfs was reduced by the presence of glucose , but not through the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743955" target="_blank">15743955</a>]<br></li>
<li>In the case of Pr , when glucose is not available , the cAMP -- CRP complex induces mhpR expression as occurs in typical class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>The roles of additional regulatory proteins ( Lrp , H-NS ) in the control of csiD expression It seems that with the elucidation of the crucial role of cAMP-CRP and a perhaps modulatory role of negative DNA supercoiling , our understanding of the regulation of csiD is not yet complete .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>This result suggested that the putative CRP box does not play a direct role in serC ( pdxF ) - aroA regulation and that the negative regulatory effects of CRP-cAMP are indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Under aerobic conditions with excess glucose , control of sdhCDAB -- b0725 -- sucABCD transcription by Arc , FNR , FruR and cAMP-CRP will play a minor role and , thus , reduced expression of this operon during glucose overflow metab-olism is likely due to the control by the cAMP-CRP independent catabolite control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>cAMP-CRP did not alter the extent of transcription termination within the IlvB leader suggesting that this regulatory system may be independent of the attenuation mechanism involved in the negative control of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Regulation of this operon is complex , involving negative control by attenuation ( 2,3 ) and positive control by a number of factors ( 4,5 ) including cAMP-CRP ( 6,2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6095192" target="_blank">6095192</a>]<br></li>
<li>Positive control by cAMP-CRP is mediated by a cic-activating elernent located between positions -72 and -38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>region missing an active cytR operator and partially defective in its cAMP/CRP control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>cAMP-CRP and LsrR are involved in transcriptional control of lsr and lsrRK operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In addition , we found that three r-protein operons , rpmI-rplT-pheM , rplM-rpsI , and rpsF-priB-rpsR-rplI , are under the control of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Besides the two representative genes for ribosome modulation , rmf and raiA , at least four r-protein operons were found to be under the control of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>At present , it is not known why only these r-protein operons are under the control of cAMP-CRP , but we predict the following .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In addition , several previously identiﬁed S-dependent genes ( e.g. , csiD and osmY ) are also under direct positive or negative control by cAMP-CRP ( 13 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>deoPl deoR by the Initiation from is under negative control repressor , whereas initiation from deoP2 is dependent upon the cAMP/CRP complex and under the joint negative control of the deoR and cytR repressors , respectively ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2653814" target="_blank">2653814</a>]<br></li>
<li>-80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC DISCUSSION osmY ( csi-5 ) is one of many stationary-phase-induced and osmotically regulated genes in E. coli that require us , the gene product of rpoS , for expression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The E. coli BL21 pCRP transformant was grown in LB broth in the presence of 50 g/ml ampicillin , and expression of CRP was induced in mid-log phase by the addition of 1 mM isopropyl - - D-thiogalactopyranoside ( IPTG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>Purification of CRP protein E. coli BL21 ( DE3 ) transformant with the CRP expression plasmid pCRP encoding His-tagged CRP was grown in LB broth , and protein expression was induced by adding 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21673794" target="_blank">21673794</a>]<br></li>
<li>A similar mechanism may account for the selective expression of osmY by E 38 : Its expression by E 70 is efficiently repressed by CRP , IHF , and lrp transcription factors ( Colland et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Both cai and ® x operons require the presence of FNR and CRP for their expression , while they are partially repressed by H-NS ( Eichler et al. , 1995 ; Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Recently , it has been reported that maeB is activated by CRP while sfcA has apparently a constitutive expression [ Gosset et al. , 2004 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18679018" target="_blank">18679018</a>]<br></li>
<li>fis expression is also controlled by DNA supercoiling ( Schneider et al. , 2000 ; Walker et al. , 2004 ) , IHF ( Pratt et al. , 1997 ) and CRP ( Nasser et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The expression of acs encoding acetyl-coenzyme A synthetase is controlled by the regulator proteins CRP , Fis and IHF ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>For example , the oxidative acid resistance system is regulated by rs and CRP ( Foster 2004 ) , and rs regulates both gene expression for osmotic stress ( Hengge-Aronis 1996 ) and regulates many acid resistance genes including the uncharacterized ygiW and yodD genes ( Weber et al. 2005 ) identiﬁed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>CRP is known to be essential for the expression of many genes involved in the transport and catabolism of different substrates ( Kolb et al. , 1993 ) , and it is unsurprising to ﬁnd that the expression of the malXY operon is regulated by a completely CRP-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>As the tdc operon is induced anaerobically in the pre-sence of L-threonine and cAMP-CRP ( Hobert and Datta , 1983 ; Wu et al. , 1992 ) , we compared expression of the tdcB , tdcE and tdcG genes in the pﬂ strain RM202 containing pD1 ( Fig. 2 ) in medium with and without Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484901" target="_blank">9484901</a>]<br></li>
<li>These lacked both the P , and CRP sites and did not show increased CRP-CAMP-independent lac expression when tested under a variety of growth conditions ( Vu and Reznikoff , 1985 ) , and the experiments were interpreted as negating the P , repression model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3907860" target="_blank">3907860</a>]<br></li>
<li>These results suggest that CRP negatively regulates proP expression and point to the Fis-independent P1 promoter as the target of repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>These results suggested that lsrR expression was subject to catabolite repression , and CRP was needed for stimulation of lsrR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>These results imply that ( i ) the DNA upstream of -120 and Mlc do not contribute to the repression of malT expression during growth in basal M M at low pH , and ( ii ) the CAMP-CRP complex is required for the pH , modulation of malT transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9884223" target="_blank">9884223</a>]<br></li>
<li>Thus , it was proposed that during exponential growth , Fis represses acs expression indirectly by repressing crp transcription and directly by binding to sites that overlap the binding sites of CRP and the -10 hexamer of RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>If this were the case , the disruption of the CRP binding site II , which is located upstream of the crp promoter and responsible for the activation of crp transcription by CRP-cAMP ( Hanamura and Aiba , 1992 ) , should lower the expression of crp in the absence of glucose and eliminate the down-regulation of crp by glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>The promoter driving lacZ expression in this strain extends to − 84 ( in all cases , relative to the transcription start site ) and does not include the CRP binding site upstream of rhaBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li>
<li>It is unlikely that CRP binding to site 4 contributes directly to an increase in rhaSR expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>A single CRP binding site is located 2115.5 and 2215.5 bp upstream of the divergent transcription start sites for the papI and papB genes , respectively , and CRP positively regulates expression of these genes by inhibiting binding of the repressing H-NS protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>Two members of the CRP family of transcription factors control the expression of hlyE in E. coli K-12 by binding at the same site centered at 61.5 bp upstream of the hlyE transcriptional start ( 10 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Previous work has shown that the global transcription factors FNR and CRP positively regulate hlyE expression by binding at the same site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Footprinting studies have shown that FNR and CRP activate hlyE expression from the same site centered 61.5 bp upstream of the SlyA-associated transcription start site ( 10 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Since cAMP-CRP is a negative transcription factor for Spot 42 production ( 30 ) , the data from this study suggest that any external or internal changes that evoke cAMP increase would not only promote the well-established notion of gal transcription initiation but also exert effects on expression of galK only through mK2 production independently from transcription initiation and from natural polarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24794565" target="_blank">24794565</a>]<br></li>
<li>Our studies suggest that neither CRP nor ArcA is this transcription factor even though a previous study indicated that expression of fnr is slightly higher ( approximately twofold ) in ArcA cells under microaerobic growth conditions ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Tobin and Schleif ﬁrst demonstrated RhaR activation of rhaSR transcription in vitro ( 31 ) ; however , at that time it was not known that CRP also regulates rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>Interestingly , the transcription of glpF and glpK is regulated by CRP and GlpR , whereas the expression of glpD is regulated by CRP , GlpR and ArcA , as proposed elsewhere [ 1,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>We then measured the transcriptional levels of two well-known CRP positively controlled genes ( cstA and cpdB ) ( to represent the level of active CRP in the cell ) as well as the expression level of ﬂhDC operon by the quantitative RT-PCR approaches ( Figure 5B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17576668" target="_blank">17576668</a>]<br></li>
<li>While CRP activates the expression of both genes , the FIS protein positively regulates the hupA gene but negatively regulates the tandem P2-P3 promoters of hupB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>CRP modulates positively or negatively the expression of a large number of genes and operons in response to glucose depletion ( 9 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655515" target="_blank">8655515</a>]<br></li>
<li>Thus , CRP enhances hlyE expression in response to glucose starvation ( 36 ) and FNR enhances hlyE expression in response to oxygen starvation ( 10 , 11 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In summary , the data presented here suggest that FNR and CRP are positive regulators of hlyE expression in liquid culture in response to oxygen and glucose starvation , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The data revealed that both FNR and CRP made signiﬁcant positive contributions to hlyE expression , whereas the hns mutation resulted in slightly reduced hlyE transcription under these conditions ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The ﬁndings that SlyA does not enhance hlyE expression in hns cultures grown in liquid medium under anaerobic conditions and that SlyA appears unable to activate hlyE expression in the absence of FNR and CRP and the determination of the location of the region of PhlyE occupied by SlyA ( Fig. 6 ) suggest that SlyA activates hlyE expression by antagonizing the negative action of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Previous in vivo experiments also indicate that CRP is not capable of activating rhaBAD expression in the absence of RhaS ( 11 ) , perhaps suggesting that RhaS must bend the DNA to allow CRP to activate , although other possibilities exist as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>We found that CRP alone was not capable of activating rhaBAD expression , similar to previous ﬁndings in vivo , and that His6-RhaS-CTD alone substantially activated rhaBAD expression ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>This is in striking contrast to the action of CRP , which plays a direct role in the promoters of the cai and ﬁx operons ( Fig. 4 and 5 ) , in addition to modulating caiF expression ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>As reported previously , expression of gadX is principally driven by the RpoS sigma factor ( Fig. 7D , stationary phase ) , and it is known that CRP represses rpoS expression ( Fig. 7C , exponential phase ) ( 10 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>This ﬁnding led us to investigate the reported growth phase and CRP dependence of fur expression by Western blotting ( Fig. 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In conclusion , our observations strongly support the notion that CRP and FIS cooperate in the shut-off of fis expression during the exponential growth phase ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Interestingly , the inhibitory effect of glucose on crp expression in cells harboring pHA7 was no longer observed However , experiments revealed that glucose reproducibly lowers the CRP level by - 10 % even in pHA7 transformants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>To explore the role of CRP itself in this regulation , the effect of glucose on the expression of crp mRNA in wild-type ( PP6 ) , cya - ( PP48 ) and crp - ( PP47 ) cells was examined by S 1 assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7518773" target="_blank">7518773</a>]<br></li>
<li>We presume that in crp cells the expression of numerous genes that are under the control of both FIS and CRP will be perturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>Then expression was induced by 250 mM IPTG for 1 h , and Western blotting using anti-CRP polyclonal antibody and Northern blotting using crp probe and 5S rRNA probe were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21062370" target="_blank">21062370</a>]<br></li>
<li>Expression of the caiF gene is dependent on the same general regulatory proteins , H-NS , CRP , FNR , and transcription factor RpoN ( 11 ) , which control cai and ﬁx expression ( Table 2 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Like rpoH , the rotA gene contains multiple start sites for transcription , is under multiple control and its expression is ﬁne tuned by CytR/CRP ( Nørregaard-Madsen et al. , 1994 ; Pogliano et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Not all CRP-bound sites control gene expression ( 9 ) , and it ha been proposed that CRP can act as a chromosome-compacting protein due to its ability to bend DNA ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21460078" target="_blank">21460078</a>]<br></li>
<li>A CRP binding-site mutation or the absence of a functional crp gene leads to high constitutive expression of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>None of the substitutions in the CRP sites had signiﬁcant effects on reporter gene expression in crp deletion strains , demonstrating that these DNA sequence changes did not alter expression in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>The crp * l gene encodes an altered cAMP receptor protein ( CRP * ) which is functional in the expression of some sugar fermentation genes , such as lac , ara , and man , in the absence of cAMP ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670911" target="_blank">2670911</a>]<br></li>
<li>For expression and puriﬁcation of CRP protein , the 633-bp crp gene was ampliﬁed in our laboratory using primers crpF and crpR by PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21840983" target="_blank">21840983</a>]<br></li>
<li>In the absence of FIS , CRP activates fis tra-nscription , thereby displacing the polymerase from the divergent promoter , whereas together FIS and CRP synergistically repress fis gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11279109" target="_blank">11279109</a>]<br></li>
<li>In addition , the mutation also results in elevated gcvA-lacZ expression in the presence and absence of CRP ( Table 6 ) and presumably of the gcvA gene itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A possible explanation for CRP 's dependence on GcvA is that CRP regulates expression of the gcvA gene and indirectly affects expression of gcvT-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In addition , GcvA is known to require the gcvR gene product for its role as a repressor ( 12 ) , raising the possibilities that CRP also regulates expression of gcvR and indirectly alters gcvT-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The fact that hupB-P2 expression is not affected by CRP certainly explains why FIS expression of hupB-P2 is effective in the presence or absence of CRP ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>In the absence of added sugars , CRP indeed appeared to be an activator of sfm both in the presence and in the absence of H-NS , as shown by decreased sfm expression in crp mutants ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>The results using lacZ fusions showed no signiﬁcant difference in expression between crp + and crp constructs , indicating that CRP is not involved in the expression of gadE during growth on glucose ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>The positive-control CRP mutants T158A , H159L ( 49 ) ( Table 2 ) , and G162C ( 15 ) ( data not shown ) were also active in repressing P1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>H159L As control , the expression of native glnAp2 was not affected by the presence of either CRP or IHF ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>A crp mutation increases the basal level of setA - lacZ activity ( Fig. 4A and B ) , but not the activity of sgrS - lacZ , suggesting there may be another promoter controlling setA expression that is repressed by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Finally , the third line of data shows an important control ; expression in every case is minimal in the absence of MelR and CRP alone is unable to activate the melAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Results illustrated in Fig. 7B indicate that expression from the trg regulatory region , like the aer promoter , is dependent on s28 and is activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>In each case , expression is reduced by the HL159 substitution in CRP , while the KE101 substitution reduces expression from the mglB , gatY , sdaC , ychH and malX promoters but not the aer promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17608696" target="_blank">17608696</a>]<br></li>
<li>In this case melR is expressed from the CAMP-CRP-independent gulp2 promoter and meli-biose-dependent expression from the melAB promoter in pKK33 appears in both crp + and Acrp cells ( Table II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>Measurements of the expression of this fusion in strain M182 , and in its Dcrp derivative , show that , in contrast to the malX promoter , the malI promoter is active in the absence of CRP , with a modest 1.7-fold stimulation by CRP ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054084" target="_blank">19054084</a>]<br></li>
<li>At the pmelR promoter , expression is dependent on CRP ( 3 ) , so lacZ cloned under pmelR is expressed in the ∆ melR strain in the absence of MelR303 and MelR173 ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>The expression of the galS promoter is absolutely dependent upon CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Mutations in CRP sites 1 and 4 did not signiﬁcantly alter expression from the rhaSR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>It is now clear that CAMP and CRP regulate ODC expression as a function of the speC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>The RhaS autoregulation mechanism is unusual in that a transcriptional activator protein functions to decrease expression of an operon by activating transcription , but not allowing a second activator protein -- in this case CRP -- to coactivate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>The expression of the LamB protein in hns and hns-stpA strains can be restored by CRP * at 37 ° C but not at 30 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The expression of the MalT protein is not restored by CRP * in hns and hns-stpA mutant strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>It has been shown that the three proteins ChbR , CRP and NagC regulate the expression of the chb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>This is similar to the situation at the m & A region where expression of the malPQ operon is under the control of maltose and muIT which , in turn , is regulated by CAMP-CRP ( Raibaud and Schwartz , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851497" target="_blank">2851497</a>]<br></li>
<li>This suggests that CRP may control expression of the operon indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>In addition , the expression level of the lsrACDBFG operon is much higher in wild-type cells than in the luxS mutant , as CRP is required for its activation ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321939" target="_blank">16321939</a>]<br></li>
<li>In contrast , the effects of a crp mutation on operon expression were still observed when an upstream putative CRP box was deleted ( Results ; Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>When E. coli is growing in glucose minimal medium , nanA expression is barely detectable due to inactivation of CRP and repression by NanR ( 4 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Evidently , CRP , and not FNR , was required for the expression of ClyA in hns mutants under anoxic conditions , although it appeared that FNR could partly complement CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>It is possible that expression of aer and trg is required only in response to conditions that induce CRP activity and not in response to other signals that induce the ﬂagellar cascade .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>However , when the downstream activator is CRP , then expression of FF-CC is observed with CRP alone ; that is , it is not co-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>It is necessary to have both MalT and the mutated form of CRP ( CRP * ) to induce mal gene expression in sfsl-dependent systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2013578" target="_blank">2013578</a>]<br></li>
<li>The activation of FlhDC expression by CRP is reduced by repression of crp by RpoS , a pathway independent of the csgD and cpxR genes ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>An lrp mutation reduced csiD expression only when the CRP box was located at the positions highly productive for activation ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In the cya mutant strain , CRP would not be bound to its site , and AraC would be the only activator of araBAD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>By altering the DNA site for CRP to reduce its similarity to the consensus , ClyA expression was signiﬁcantly lowered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Why do spacing changes at -203 but not spacing changes in the CRP site affect araBAD expression ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Although the single CRP site is involved in activation of both araBAD expression and araC expression , different mutations in the site did not affect expression to the same extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2521619" target="_blank">2521619</a>]<br></li>
<li>Since there is no detectable rhaBAD activation by CRP in the absence of RhaS , it was not possible to test the effect of a-D235 expression on CRP activation independent of RhaS activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Similarly , restoring CRP did not enhance hlyE expression , despite the absence of glucose ( Fig. 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>As a control we also assayed fadH expression and found that expression decreased in the absence of CRP ( Fig. 6E and F ) ( Fig. 7D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Western blot analysis conﬁrms that , in this system , expression of s28 does not require CRP ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>The Northern blot in Fig. 4 ( EF528 ) and the Western blot in Fig. 7B indicate that CRP does repress gadX expression at the TABLE 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>CRP represses the system largely by inhibiting the production of RpoS , the sigma factor primarily responsible for gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>CRP appears to negatively regulate gadX expression by virtue of its effects on RpoS production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>This conﬁrms that both CRP and s28 are essential for expression from the aer regulatory region in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>This suggested that CRP might repress gadA and gadBC indirectly by regulating gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Regulation of expression of the ulaG and ulaA divergent promoters by CRP , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>We also tested whether DNA bending by CRP inﬂuenced rhaSR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>]<br></li>
<li>CRP also represses gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Therefore , contrary to expectation , CRP does not appear to be involved in regulating ftnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>RhaS activates rhaSR expression , but CRP coactivation is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>CRP regulates expression of gcvA but does not regulate gcvR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>sfm expression is under catabolite expression mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Role of CRP AR1 and AR2 in rhaBAD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Various genes , including rpoS and several sigma S-dependent genes , such as bolA , are negatively controlled by cAMP-CRP ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Expression of the deoCA genes from the P-2 promoter is known to be controlled positively by cAMP/CRP and negatively by the cytR repressor , the inducer being cytidine ( J0rgensen et al. , 1977 ; Valentin-Hansen et al. , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329724" target="_blank">6329724</a>]<br></li>
<li>Two other E. coli loci are controlled negatively by cytR and deoR and positively by the CAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2115441" target="_blank">2115441</a>]<br></li>
<li>Initiations from P2 are both negatively controlled by the deoR repressor and the cytR repressor ( the inducer being cytidine ) ( 5-6 ) and positively controlled by the cAMP/CRP complex ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6087276" target="_blank">6087276</a>]<br></li>
<li>These genes are induced by many different stress responses ( K.L. Jones and K.E. Rudd , unpublished ) , and several are regulated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Although many more promoters are positively controlled by the CAMP-CRP system , several genes are also negatively regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3540312" target="_blank">3540312</a>]<br></li>
<li>These confirmed that the acnA gene resembles other citric acid cycle genes in being positively regulated by the CAMP-CRP complex and negatively regulated by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>This growth-phase-dependent pattern resembles that observed for CRP-regulated genes such as the cst genes , mcc , and glgCAP ( 22 , 31 ) , suggesting that dctA is also regulated by CRP ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>This suggests that csiE and poxB are regulated by a similar mechanism.Since the poxB promoter region does not contain a CRP box , it seems likelythat poxB and also csiEare regulated by an unknown intermediate activator that is itself under the control of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>cspA promoter , not known to be regulated by CRP ( our unpublished results ) , is used as a control in these assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24509317" target="_blank">24509317</a>]<br></li>
<li>In summary , these results dem-onstrate that gntT is positively regulated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>This upregulation was not surprising since gltA is known to be regulated by several regulators such as CAMP-CRP , acrA and fir ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16233096" target="_blank">16233096</a>]<br></li>
<li>This compound is known , to increase - in vitro transcription of operons - regulated by CRP ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292893" target="_blank">6292893</a>]<br></li>
<li>Finally , many operons in these pathways are regulated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine metabolism ) was controlled by protein regulators such as FNR ( transcriptional regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Transcriptional monocopy lacZ fusion studies showed that both operons are coexpressed during anaerobic growth in the presence of carnitine , respond to common environmental stimuli ( like glucose and nitrate ) , and are modulated positively by the same general regulators , CRP and FNR , and negatively by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Like cai and ﬁx operons , the caiF gene appeared to be positively controlled by CRP and FNR and negatively controlled by H-NS and the NarL ( plus nitrate ) regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>( 1991 ) 218 , 129-140 Molecular and Functional Characterization of a Carbon Starvation Gene of Escherichia coli J. E. Schultz and A. Matint Department of Microbiology and Immunology Sherman Fairchild Science Building Stanford University School of Medicine Stanford , CA 94305-5402 , U.S.A. ( Received 3 May 1990 ; accepted 16 October 1990 ) Escherichia coli induces the synthesis of at least 30 proteins at the onset of carbon starvation , two-thirds of which are positively regulated by the cyclic AMP ( CAMP ) and CAMP receptor protein ( CRP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848300" target="_blank">1848300</a>]<br></li>
<li>Memorial University of Newfoundland , St. John 's , Newfoundland , CANADA AlB 3V6 Tel. 709-737-6885 ( Received May 15th , 1985 ) ( Revision received November 1lth , 1985 ) ( Accepted February 3rd , 1986 ) SUMMARY The speC gene encoding ornithine decarboxylase ( ODC ) in Escherichia coli is negatively regulated by CAMP and the CAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3021588" target="_blank">3021588</a>]<br></li>
<li>( 1988 ) 203 , 564-583 Positive and Negative Regulators for Glucitol ( gut ) Operon Expression in Escherichia coli Mamoru Yamadat and Milton H. Saier Jr Department of Biology The John Muir College University of California at San Diego La Jolla , CA 92093 , U.S.A. ( Received 20 August 1987 , and in revised form 16 February 1988 ) Expression of the glucitol ( gut ) operon in Escherichia coli is regulated by an unusual , complex system which consists of an activator ( encoded by the gutM gene ) and a repressor ( encoded by the gutR gene ) in addition to the cAMP-CRP complex ( CRP , CAMP receptor protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3062173" target="_blank">3062173</a>]<br></li>
<li>, 1990 ) and the regulation of lacZYA operon ( encoding a lactose-utilizingsystem ) by Lac1 ( a specific repressor ) and CAMP-CRP complex ( a global modulator ) in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Such is the case for complex II in E. coli , with the complex II bio-synthetic operon ( sdhCDAB ) being negatively regulated by ArcAB and FNR in response to oxygen conditions ( i.e. anaerobic ) and positively regulated by CRP in the absence of glucose [ 108 -- 110 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>When crp was supplied by plasmid pLG339CRP , a poor carbon source induced glnA expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>When osmotic regulation of osmY in minimal medium was assayed in a Acya Acrp background ( RO82 ) , csi-5 : : lacZ expression was already increased in exponential phase and could be further stimulated by adding 0.3 M NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The increased exponential phase expression of both fusion constructs in the Acrp background is probably due to the strongly elevated levels of os in cya and crp mutants ( Lange and Hengge-Aronis , 1991b ; Lange and Hengge-Aronis , 1994a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>