<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - CusR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">CusR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">CusR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The RR-bound 32P was stably retained for the A1 group RRs such as pArcA , pCpxR , pCusR , pKdpE , and pNarL but was rapidly released for the A2 group RRs , including pBaeR , pCheB , pCheY , pCreB , pDcuR , pNarP , pNtrC , pTorR , pUhpA , pYehT , and pYfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>After 5 min , cells were harvested and the whole cell lysates were subjected to quantitative immuno-blot analysis of CusR ( A , short exposure ) and CueR ( B , short exposure ) proteins as described in Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Before hybridization , DNA chip was incubated at 65 ∞ C for 1 h in 20 ml of prehybridization buffer ( 6 ¥ SSC , 0.2 % SDS , Puriﬁcation of CusR and CueR To construct pCueR and pCusR for overproduction of CueR and CusR , respectively , DNA fragments containing the CueR - or CusR-coding regions were ampliﬁed by PCR with E. coli W3110 genome DNA as a template and a pair of primers , in which the NdeI and NotI sites were included ( see sequences in Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Thus , changes in oxygen availability ( sensed directly by FNR , and indirectly by ArcA ) affect the redox state of the system , leading to consequences for metal ion homeostasis ( copper , molybdate , iron ; sensed by CusR , ModE and Fur ) , iron -- sulphur cluster turnover ( sensed by IscR ) , ribonucleotide reductase activity 20 0 31 0 10 aerobiosis ( % ) time ( min ) Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>This core network was extended to include local regulators ( CpxR , CusR , FhlA , Fur , IscR , ModE , NarL , NrdR and PdhR ) that exhibited responses in the stable environments o steady-state cultures across the aerobiosis scale and in the unstable environments of the transitions ( figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>In the putative promoter ( 200 bp upstream of the pitA start codon ) , no DNA-binding sites for the following transcription factors are evident : MerR , CueR , ZntR ( all MerR family ) , Zur , NikR , CusR , ZraR , CpxR and ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054109" target="_blank">19054109</a>]<br></li>
<li>Details are shown in Supplementary Table S2 , but briefly this consisted of 184 genes that could be divided into the following categories : repressed by Fur , 63 ; activated by Fur , 15 ; repressed by CpxR , 21 ; activated by CpxR , 37 ; repressed by SoxS , 1 ; activated by SoxS , 23 ; repressed by OxyR , 2 ; activated by OxyR , 16 ; activated by CusR , 6 ; activated by BaeR , 9 ; repressed by Zur , 4 ; repressed by ArsR , 3 ; activated by ZraR , 3 ; activated by ZntR , 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_10">
Statistical modelling of the transcriptomic data generated from the Zn shift focused on the role of ten key regulators ; ArsR , BaeR , CpxR , CusR , Fur , OxyR , SoxS , ZntR , ZraR and Zur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></label>
<input id="act_similarity_10", type="checkbox">
<div class="ACT_similar">
<li>The analysis of the transcriptomic data generated from the Zn shift focused on the role of ten key regulators : ArsR , BaeR , CpxR , CusR , Fur , OxyR , SoxS , ZntR , ZraR and Zur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li></div>
<li>While the profiles of the ` monotonic ' regulators BaeR , CpxR , Zur and CusR are compatible with the predicted profile from the dynamic time-course dataset , neither Fur nor SoxS exhibits the transient behaviour predicted from the dynamic data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>Inferred activity profiles of regulators BaeR , CpxR , CusR , Fur , SoxS and Zur during adaptation from Zn deprivation to Zn excess : TFinfer software predictions on the temporally resolved data collected upon addition of 200 mM Zn to Zn-depleted continuous cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>Inferred activity profiles of regulators BaeR , CpxR , CusR , Fur , SoxS and Zur at steady states with different Zn concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_15">
The amounts of CusR added for lanes 1 -- 6 were 0 , 12.5 , 25 , 50 , 100 and 200 pmol , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></label>
<input id="act_similarity_15", type="checkbox">
<div class="ACT_similar">
<li>The amounts of CusR added for lanes 1 -- 5 were 0 , 12.5 , 25 , 50 , 100 and 200 pmol , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li></div>
<li>When the culture of wild-type E. coli W3110 was exposed to 0.5 mM CuSO4 for 5 min in the middle of exponential-growth , no signiﬁcant change was observed in the level of CusR ( Fig. 4A , short exposure ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>E. coli BL21 transformants were grown in LB broth in the presence of 100 mg ampicillin ml , and expression of His-tagged YedW and CusR was 21 induced in the mid-exponential phase by adding 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>In the presence of increasing concentrations of phosphorylated CusR , the P-labelled hiuH promoter 32 probe was converted to form the CusR complex ( Fig. 2a , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>FITC-labelled hiuH promoter probe was mixed with increasing amounts of purified CusR and subjected to PAGE after incubation at 37 6C for 30 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>( Left panel ) The FITC-labelled hiuH promoter segment ( 1 pmol ) was incubated in the absence or presence of increasing concentrations of purified CusR and then subjected to DNase I footprinting analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>In vitro DNase I footprinting has shown that CopR from P. syringae , a CusR homolog , binds to the copper box upstream of PcopA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004187" target="_blank">11004187</a>]<br></li>
<li>Two DNase hypersensitive sites appeared in the presence of CusR ( Fig. 1B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Interestingly , the yedVp was not induced by copper in cusR null mutant , even though it is not under the direct control of CusR ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>CusR was phosphorylated by three non-cog-nate HKs ( pBarA , pUhpB , and pYedV ) , whereas YfhA was phosphorylated by four non-cognate HK ( pBaeS , pEnvZ , pRstB , and pUhpB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Most interestingly CheY , YfhA , and CusR , which showed high levels of phosphorylation ( see `` Results '' ) , were phosphorylated by two , four , and three HKs , respectively ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>On the other hand , nine species of RR ( AtoC , CheY , CusR , HydG , KdpE , NarL , NarP , NtrC , and YfhA ) were phosphoryl-ated by non-cognate HKs besides their cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Among the RRs tested , five species of RR ( CitB , CpxR , PhoB , RssB , and YhjB ) enhanced dephosphoryl-ation of more than two species of HK , and four species ( CusR , HydG , PhoP , and YfhA ) enhanced dephosphorylation of one non-cognate HK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The trans-phos-phorylation was experimentally demonstrated in vitro , for the first time , for 9 pairs , CusS-CusR , RstB-RstA , YedV-YedW , BaeS-BaeR , YehU-YehT , YfhK-YfhA , HydH-HydG , BasSBasR , and CreC-CreB ( see Fig. 3 ) , supporting the predicted HK-RR pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>A total of 30 sensor HKs , including as yet uncharacterized puta - tive HKs ( BaeS , BasS , CreC , CusS , HydH , RstB , YedV , and YfhK ) , and a total of 34 RRs , including putative RRs ( BaeR , BasR , CreB , CusR , HydG , RstA , YedW , YfhA , YgeK , and YhjB ) , have been suggested to exist in Esche - richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Trans-phosphorylation of presumed cog - nate RRs by HKs was detected , for the first time , for eight pairs , BaeS-BaeR , BasS-BasR , CreC-CreB , CusS-CusR , HydH-HydG , RstB-RstA , YedV-YedW , and YfhK-YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Taken together , we conclude that the same CusR molecule activates transcription towards both directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Transcription of the divergent operons , cusCFB and cusRS , was found to be activated by CusR , which bound to a CusR box between the cusC and cusR promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>For example , PhoB regulates cusR and both PhoB and CusR regulators regulate cusC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>In the absence of its cognate histidine kinase , CusR may be gratuitously activated by another histidine kinase , as has been reported for other two-component systems ( 1 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004187" target="_blank">11004187</a>]<br></li>
<li>The low-level induction of the CusSR-regulated genes , cusC and cusR , suggests that the experimental conditions are not suitable for self-phosphorylation of the CusS sensor kinase and/or trans-phosphorylation of CusR by phosphorylated CusS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Activation of the CusSR-regu-lated genes depends on the phosphorylation of CusR by phosphorylated CusS ( Yamamoto et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Recently , we found that the YedV kinase transfers phosphate not only to YedW but also CusR ( Yamamoto et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Results will show that besides the known CusR and CueR regulons , two additional cellular systems , CpxR and YedW regulons , are involved for defence against increased levels of external copper .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Taken together we conclude that the copper-responsive genes are organized into a hierarchy of the regulation network , forming at least four regulons , i.e. CueR , CusR , CpxR and YedW regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>The copper stimulon in Escherichia coli consists of four regulons , the CueR - , CusS/CusR - , CpxA/CpxR - , and YedV/YedW regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861804" target="_blank">16861804</a>]<br></li>
<li>Gene Function Intracellular levels of CusR and CueR transcription factors Possible inﬂuence of the increase in external copper on the intracellular level of two copper-responsive transcription factors , CusR and CueR , was examined by quantitative immunoblot assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Intracellular copper levels are controlled by a pair of transcriptional-regulators , CueR ( Outten et al. , 2000 ; Petersen and Moller , 2000 ) and CusR ( Munson et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Along this line , it could be possible that YedV responds to yet unidentiﬁed external signal ( s ) , leading to activation of not only YedW but also CusR via signal cross-talk .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>The sequence of the response-regulator YedW is 51 % similar to that of CusR and the sensor YedV phosphorylates not only YedW , but also CusR ( Yamamoto & Ishihama , 2005 ) , suggesting cross-talk between YedVW and CusSR TCSs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>This ﬁnding indicates that this putative two-com-ponent system is under the control of CusR , and regulates another set of genes , tentatively designated as YedW regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>This finding indicated that two TCS regulators , YedW and CusR , bound to the same sequence and regulated the same target hiuH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>YedW and CusR were found to-bind to the same recognition sequences and regulate the same set of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>Here , we identified that YedW and CusR recognize and bind to the same sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>Thus , the activation of YedV by copper-shock may play a role in the activation of CusR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Thus , once E. coli is 2 + exposed to either Cu or H2O2 , activated CusR or YedW may participate in the activation of common targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>Furthermore , the activated forms of YedW and CusR might collaborate for enhanced expression of the same targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">CusR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Thus , the activation of YedV by copper-shock may play a role in the activation of CusR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">CusR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>CusR has 83 % identity to SilR and 61 % identity to both PcoR and CopR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004187" target="_blank">11004187</a>]<br></li>
<li>The sequence of the response regulator YedW is 51 % similar to that of CusR and the sensor YedV phosphorylates not only YedW , but also CusR ( Yamamoto & Ishihama , 2005 ) , suggesting cross-talk between YedVW and CusSR TCSs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by CusR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>To search for the genes under the control of CusR , we next performed the DNA microarray analysis using a CusR-defective mutant JD20810 [ KP7600 cusR : : Km ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>This ﬁnding indicates that this putative two-com-ponent system is under the control of CusR , and regulates another set of genes , tentatively designated as YedW regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Regulon gene CusR cusB * copper/silver transporter cusC * copper/silver transporter cusF * periplamic copper-binding protein yedV putative sensor kinase yedW putative response regulator CueR copA * copper exporter cueO * periplasmic multicopper oxidase moaB molybdenum cofactor-synthesis protein B moaC molybdenum cofactor-synthesis protein C moaD molybdopterin [ MPT ] converting factor subunit 1 moaE molybdopterin [ MPT ] converting factor subunit 2 CpxR cpxP periplasmic protein ; extracytplasmic stress response ebr small multidrug exporter ; ethidium bromide resistance spy periplasmic protein related to spheroplast formation yccA membrane-associated putative carrier/transporter yecI ferritin-like protein ybaJ unidentiﬁed ycfS unidentiﬁed ydeH unidentiﬁed yebE unidentiﬁed yqjA unidentiﬁed JW1832 unidentiﬁed Unclassiﬁed genes aldH aldehyde dehydrogenase aphA ** acid phosphatase mutL methyl-directed mismatch-repair enzyme tesA acyl-CoA thioesterase I htpX heat shock-induced membrane protease soxS transcription factor for super-oxide response regulon yciW putative oxidoreductase yeeF putative amino acid/amine-transporter yncJ unidentiﬁed This list shows the genes , which are more than threefold upregulated by copper , including hitherto identiﬁed copper-response genes ( * ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The closest homologs of CusR and CusS are plasmid-borne two-component systems that are also involved in metal responsive gene regulation : PcoR and PcoS from the pco operon of E. coli ; CopR and CopS from the cop operon , which provides copper resistance to Pseudomonas syringae ; and SilR and SilS from the sil locus , which provides silver ion resistance to Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004187" target="_blank">11004187</a>]<br></li>
<li>Two regulatory systems , the CueR and CusR systems , have been identiﬁed to be involved in transcription regulation of the genes for copper-homeostasis ( Rensing et al. , 2000 ; Rensing and Grass , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Regulation of the copper-responsive genes by CusS-CusR two-component system Two transcription factors , CusR and CueR , have been identiﬁed to be involved in copper-response transcription regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">CusR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Before hybridization , DNA chip was incubated at 65 ∞ C for 1 h in 20 ml of prehybridization buffer ( 6 ¥ SSC , 0.2 % SDS , Puriﬁcation of CusR and CueR To construct pCueR and pCusR for overproduction of CueR and CusR , respectively , DNA fragments containing the CueR - or CusR-coding regions were ampliﬁed by PCR with E. coli W3110 genome DNA as a template and a pair of primers , in which the NdeI and NotI sites were included ( see sequences in Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>This finding indicated that two TCS regulators , YedW and CusR , bound to the same sequence and regulated the same target hiuH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>YedW and CusR were found to bind to the same recognition sequences and regulate the same set of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>Here , we identified that YedW and CusR recognize and bind to the same sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>To identify the sequence recognized by CusR , we next performed DNase I footprinting analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>A sequence of 33 bp was protected by phosphorylated CusR from DNase I digestion ( Fig. 2c , left panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>These CusR-binding sequences include the inverted-repeat CusR box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>The protected region by CusR were -42 to -67 on top strand and -44 to -70 on bottom strand [ the numbers represent the distance from the initiation site of cusC transcription ] ( for sequence see Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Results , shown in Fig. 1B , indicate that CusR binds at a site between the cusC and cusR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>This CusR-binding sequence between the cusC and cusR promoters ( Fig. 1C ) includes the inverted-repeat , AAAAT GACAANNTTGTCATTTT , which has been proposed to constitute the CusR box ( Munson et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Consensus sequence ( c ) CusR box yedW -- hiuH cusR -- cusC cyoABCDE Consensus sequence Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by CusR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>E. coli BL21 transformants were grown in LB broth in the presence of 100 mg ampicillin ml , and expression of His-tagged YedW and CusR was 21 induced in the mid-exponential phase by adding 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>Transcription of the divergent operons , cusCFB and cusRS , was found to be activated by CusR , which bound to a CusR box between the cusC and cusR promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>The results of microarray assay agree well with the ﬁnding that the CusR box is located only in the intergenic region between the cusRS and cusCFB operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>The low-level induction of the CusSR-regulated genes , cusC and cusR , suggests that the experimental conditions are not suitable for self-phosphorylation of the CusS sensor kinase and/or trans-phosphorylation of CusR by phosphorylated CusS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Interestingly , the yedVp was not induced by copper in cusR null mutant , even though it is not under the direct control of CusR ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>In an E. coli mutant lacking CusR , transcription of not only cusS but also cusR was not detected as analysed by S1 mapping , but transcription of both copA and cueO was detected ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
<li>Transcription regulation of the cusS and cusR genes by CusR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15773991" target="_blank">15773991</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>