<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - CynR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">CynR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">CynR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Lane A , protein markers with molecular sizes in kilodaltons on the left side ; lane B , 500 ng of purified CynR ; lane C , I pLg ; lane D , 2.5 p.g ; lane E , 5 , ug .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>In this respect , low cellular levels of CynR protein could be achieved not only through negative transcriptional autoregulation but also by the use of a GTG codon as translation start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>In all reactions , the respective DNAs were used at the final concentration of 4 X 101o M. CynR protein was used in the assay at the following concentrations : no protein , lanes 1 , 5 , and 9 ; 1.3 X 10-8 M , lanes 2 , 6 , and 10 ; 0 65 X 10-8 M , lanes 3 , 7 , and 11 ; and 0.32 X 10-8 M , lanes 4 , 8 , and 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>This binding competition assay shows that at a low concentration , CynR recognizes and binds specifically to the cynR-TSX intergenic region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>JOURNAL OF BACrERIOLOGY , Dec. 1993 , p. / 93/247990 -10 $ 02.00 / 0 Copyright © 1993 , American Society for Microbiology Gene Expression and Purification of the cynR Regulatory Product : CynR Is a DNA-Binding Protein ANNE-FRANCOISE J. LAMBLIN AND JAMES A. FUCHS * Department of Biochemistry , University of Minnesota , 1479 Gortner Avenue , St. Paul , Minnesota 55108 Received 19 May 1993/Accepted 14 October 1993 The CynR protein , a member of the LysR family , positively regulates the Escherichia coli cyn operon and negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>However , when cyanate binds to the CynR protein , we predict that it will shift the conformational equilibrium to the activated form , which will stimulate the transcription of the cyn structural genes while still negatively autoregulating its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>One interpretation of this result is that CynR binding changes the conformation of the DNA between the cynR -10 and cynTSX -35 sequences to a conformation more susceptible to DNase I endonuclease activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The fraction of the CynR bound to DNA was plotted as the logarithmic function of the CynR concentration in the presence ( - ) or absence ( asterisks ) of 10 mM azide in the assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The CynR concentration that results in an equal amount of bound and free DNA is 3 X 10-9 to 4 X 10-9 M and does not change in the presence of 10 mM sodium azide in the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The CynR concentration that results in equal amounts of bound and free DNA agrees with the results presented in Fig. 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The bend induced by CynR to the cynTSX-cynR intergenic DNA fragment was reduced from an approximate 750C to 650C when cyanate was added to the reaction mixture ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>CynR protein was also found to induce bending of the R1 DNA fragment ( Fig. 7 ) , and the bend was mapped near position 1275 * [ 75 ] in the absence of cyanate and to position 1281 * [ 69 ] in its presence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>Circular permutation assays indicated that CynR induces bending of the DNA upon binding and that this bending decreases in the presence of cyanate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The ability of the CynR protein to induce DNA bending upon binding was examined by circular permutation analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>Strong DNase I hypersensitive sites were detected in the presence of the CynR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>DNase I footprinting of CynR on the cynTSX and cynR coding strands , both in the presence and the absence of cyanate , was conducted as described by Galas and Schmitz ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The DNase I protection observed with CynR was identical in the presence or absence of cyanate on either promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>CynR negatively regulates its own synthesis independently of cyanate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The determination of the CynR DNA binding site in the intergenic region and how cyanate affects this pattern should shed some light on the dual mechanism of activation and repression exhibited by the CynR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>The cyn operon promoter in the R2 fragment ( pAL55Z ) failed to respond to induction of transcriptional activation with both CynR , provided in trans ( strain JM101 ) , and cyanate present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The transcriptional activation of the cynTSX polycistronic message is under the positive control of the CynR protein and cyanate ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The promoter of this operon is positively regulated by cyanate and the CynR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The CynR protein functions not only as a transcriptional-activator of the cyn operon in th presence of cyanate but also as a transcriptional autorepressor independently of cyanate ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">CynR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Twenty amino acids constitute the consensus helix-turn-helix motif for the LysR-family , and CynR contains 12 residues that are identical ; of the remaining 8 positions , 4 are conservative mismatches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>Comparisons of amino acid similarities of domains of cynR with the consensus sequence of the LysR-family and selected members of the LysR-family ' cynTsx +1 -10 1 , TrTrCTAT -35 CCATTT CTCTGGAATGAGAGGCAGACTACTCGCCG -35 GGTAAAAGTCTCATTT -10 Domainb ( aa residues ) CynR Comparison protein Protein D '' S GAGTTCCATTGGATTTAcTTATAGGTTGcGAa-r .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>Plasmid pAL52 is a similar construct that contains only the R1 region of the CynR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>A protein such as CynR might be divided into a binding domain , a potential dimerization domain , and an activation domain with a variable degree of domain interdependence for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>The CynR-LacZ chimeric protein encoded in pAL54Z contains the CynR helix-turn-helix DNA-binding domain and the LacZ multimerization domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>We previously demonstrated that CynR is a DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>We first determined the DNA-binding activity of the CynR protein ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>Because the members of this family are DNA-binding proteins , CynR was assayed for potential DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>Gel shift assays confirmed that CynR is a DNA-binding protein like the other members of the LysR-family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>The significant sequence homology between the N-terminal domain of CynR and those of members of the LysR-family led Sung and Fuchs ( 40 ) to postulate that CynR may be a DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">CynR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Twenty amino acids constitute the consensus helix-turn-helix motif for the LysR family , and CynR contains 12 residues that are identical ; of the remaining 8 positions , 4 are conservative mismatches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>To understand the molecular details of the dual activation-autorepression process , we focused our attention on one member of the LysR family , the CynR regulatory protein of the cyn operon recently characterized in our laboratory ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>DNA modulation has been reported for many of the regulators belonging to the LysR family ( AmpR [ 4 ] , CynR [ 21 ] , CatR [ 33 ] , and OccR [ 41 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626295" target="_blank">8626295</a>]<br></li>
<li>to of the The following characteristics shared by most are members of the LysR family and led to the classification of CynR member of this as a quickly expanding family : ( i ) expression of the activator pro- tein is negatively autoregulated , ( ii ) transcriptional activation of the structural gene ( s ) requires both inducer molecule an and the activator protein , ( iii ) the respective regulatory and structural gene promoters direct transcription in opposite directions and occasionally overlap , ( iv ) the regulatory proteins have an average size of 300 amino acids , ( v ) a strong amino acid sequence homology is present in the N-terminal region of the activator proteins , and ( vi ) all the members are DNA - binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>The similarities of the CynR sequence to the LysR consensus sequences as well as to selected members of the family are given in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>Comparisons of amino acid similarities of domains of cynR with the consensus sequence of the LysR family and selected members of the LysR family ' cynTsx +1 -10 1 , TrTrCTAT -35 CCATTT CTCTGGAATGAGAGGCAGACTACTCGCCG -35 GGTAAAAGTCTCATTT -10 Domainb ( aa residues ) CynR Comparison protein Protein D '' S GAGTTCCATTGGATTTAcTTATAGGTTGcGAa-r .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>CynR has almost as much sequence similarity to the LysR family as LysR ( data not shown ) and has greater homology to some members than LysR has .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>The CynR protein is classified as a member of the LysR family ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>Homology of CynR protein to the LysR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>Gel shift assays confirmed that CynR is a DNA-binding protein like the other members of the LysR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>The significant sequence homology between the N-terminal domain of CynR and those of members of the LysR family led Sung and Fuchs ( 40 ) to postulate that CynR may be a DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>CynR , a member of the LysR family of regulatory proteins , binds specifically to a 136-bp DNA fragment containing both the cynR and the cynTSX promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>Because the members of this family are DNA-binding proteins , CynR was assayed for potential DNA binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by CynR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>JOURNAL OF BACrERIOLOGY , Dec. 1993 , p. / 93/247990 -10 $ 02.00 / 0 Copyright © 1993 , American Society for Microbiology Gene Expression and Purification of the cynR Regulatory Product : CynR Is a DNA-Binding Protein ANNE-FRANCOISE J. LAMBLIN AND JAMES A. FUCHS * Department of Biochemistry , University of Minnesota , 1479 Gortner Avenue , St. Paul , Minnesota 55108 Received 19 May 1993/Accepted 14 October 1993 The CynR protein , a member of the LysR family , positively regulates the Escherichia coli cyn operon and negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>We identified both xenobiotic-metabolism ( AhpC , CpxA , and CynR ) and stress based ( ArcA and OxyR ) regulatory proteins as being over-represented amongst the MMS-toxicity modulating proteins , which was expected and served as another validation for our algorithm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>The CynR translation-initiation codon was identified by oligonucleotide-directed mutagenesis , and the CynR coding sequence was cloned under the control of a T7 phage promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">CynR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Examination of the pAL5 plasmid sequence between the EcoRI-HincII ligation point and the identified cynR start of translation revealed the presence of three othe g AGT TCC ATT GGA TTT ACT TAT AGG TTG CGA atg rbs Met CA TTat a -10 3 ¬ cynR ctc tct cga cat atc Leu Ser Arg His Ile B Site directed NdeI restoring the CynR ATG methionine codon ) T -.7 ¬ 5 ' 31 atG CTC TCT CGA cynR promoter ... tttaaggagatatacat rbs pT7-7 pAL5 mI pAL11 m - + + ¬ C 0.5 mM IPTG Rifampicin 200 pg/ml + - + + ¬ + ¬ 69 46 30m 2 3 5 4 6 7 site ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>DNase I footprinting analysis was also conducted on larger fragments extending 205 bp into the CynR coding sequence ( up to the BglI site at position 1053 * [ 296 ] ) and 73 bp into the CynT sequence ( up to the PvuII site at position 1441 * [ 194 ¶ ] ) on both coding strands ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>On the cynTSX coding strand , the CynR binding site was found to extend from -75 to inside the spacer region of the cynTSX promoter at -21 ( positions are given relative to the cynTSX start site of transcription ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>This binding competition assay shows that at a low concentration , CynR recognizes and binds specifically to the cynR-TSX intergenic region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>CynR recognizes a 136-bp DNA fragment encompassing the cynR-cynTSX intergenic region ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>CynR binds specifically to a 136-bp DNA fragment containing both the cynR and cynTSX promoter sequences ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>By gel shift assay with purified CynR protein , we showed that CynR recognizes and specifically binds 136-bp a DNA fragment containing the divergent cynR-cynTSX promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>We propose that CynR binds to R1 as a dimer and that simultaneously the second site of the dimer binds to R2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>If this sequence is the CynR recognition sequence , the two-base mismatch of TA versus GT could account for the difference in CynR binding affinity of the two sites when separated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>- ... o c 0.5 ¬ e0 0.45 ¬ 0 m 4 n0 s r0 - ¬ -- 2 9 9 2 a - - ( 14 C14 Position ( nt ) ( 14 ) , IlvY ( 37 ) , and CynR ( this study ) bind to sites overlapping the RNA polymerase recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>Because of the small size of the DNA probe used in this assay , the previously reported binding sites of other members of the LysR family ( 4 , 17 , 19 , 24 , 32 , 44 , 50 ) , and the functions of these proteins , one can expect the CynR protein binding site to overlap some of the RNA polymerase recognition sequence and/or binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>A potential CynR recognition sequence is indicated by the thin line outside the frame .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The similarities of the CynR sequence to the LysR consensus sequences as well as to selected members of the family are given in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>DNA footprints were delimited by a nonprotected band in the -10 CynR promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The cynTSX -10 sequence is not protected by CynR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The CynR sequence is in uppercase letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>Complexes in which the full intergenic region ( Fig. 5 ) is located at or near the center of the fragment ( Fig. 5 , lane f ) exhibited the lowest level of mobility , whereas complexes with the CynR binding sequence at the end of the fragment ( Fig. 5 , lane a ) showed the highest level of mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>While CynR formed a complex with the R1 fragment ( Fig. 4 , lanes 6 to 8 ) , it did not bind the R2 sequence in the range of protein concentrations used in this assay ( Fig. 4 , lanes 10 to 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>A circular permutation assay ( 38 , 40 ) was used to test the ability of CynR to induce bending upon binding of DNA fragments containing either the full intergenic region ( both R1 and R2 sequences present ) or only the R1 DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>If , as hypothesized , the R1 sequence ATAAGTAAA is the CynR DNA-binding site , an exact copy of the former sequence in the R2 fragment would promote tighter CynR binding to the R2 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by CynR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>CynR regulates expression of the c ) 'n operon by a mechanism that appears to exactly parallel that by which lysR regulates expression of iysA , which encodes diaminopimelate deearboxylase [ 28 ] , and the predicted amino acid sequences of cynR and iysR proteins have a high degree of homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2094285" target="_blank">2094285</a>]<br></li>
<li>JOURNAL OF BACrERIOLOGY , Dec. 1993 , p. / 93/247990 -10 $ 02.00 / 0 Copyright © 1993 , American Society for Microbiology Gene Expression and Purification of the cynR Regulatory Product : CynR Is a DNA-Binding Protein ANNE-FRANCOISE J. LAMBLIN AND JAMES A. FUCHS * Department of Biochemistry , University of Minnesota , 1479 Gortner Avenue , St. Paul , Minnesota 55108 Received 19 May 1993/Accepted 14 October 1993 The CynR protein , a member of the LysR family , positively regulates the Escherichia coli cyn operon and negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>Expression and purification of the cynR regulatory gene product : CynR is a DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>Without cyanate , we predict that CynR will bind to the regulatory site to prevent the transcription of the cynR gene without activating cyn operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>However , when cyanate binds to the CynR protein , we predict that it will shift the conformational equilibrium to the activated form , which will stimulate the transcription of the cyn structural genes while still negatively autoregulating its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
<li>CynR negatively regulates its own synthesis independently of cyanate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>CynR binding was not affected by cyanate or its structural homolog azide , a gratuitous inducer of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>In the absence of cyanate and CynR , the cynTSX operon promoter in the R2 fragment ( pAL55Z ) directed transcription to the same extent as the cynTSX promoter in the nonaltered intergenic region ( pAL56Z ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The CynR gene is located upstream of the cynTSX operon and is transcribed in the opposite direction of the cyn operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>upstream was sequenced and found to encode the regulatory protein CynR , which is transcribed in the direction opposite that cyn operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253686" target="_blank">8253686</a>]<br></li>
<li>The cynTSX -10 sequence is not protected by CynR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The transcriptional activation of the cynTSX polycistronic message is under the positive control of the CynR protein and cyanate ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
<li>The promoter of this operon is positively regulated by cyanate and the CynR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961413" target="_blank">7961413</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>