<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - CytR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">CytR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">CytR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>171 , No. 9 MalI , a Novel Protein Involved in Regulation of the Maltose System of Escherichia coli , Is Highly Homologous to the Repressor Proteins GalR , CytR , and Lacl JOACHIM REIDL , KARIN ROMISCH , MICHAEL EHRMANN , AND WINFRIED BOOS * Department of Biology , University of Konstanz , D-7750 Konstanz , Federal Republic of Germany Received 13 March 1989/Accepted 10 June 1989 The maltose regulon of Escherichia coli comprises several operons that are under common regulatory control of the MalT activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>ReidI , J. , Romisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) Mail , a novel protein involved in the maltose system of Eschertchta cott is highly homotogous to the repressor proteins GaIR , CytR and Lact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>ReidI , J. , Romisch , K. , Ehrmann , M , , and Boos , W. ( 1989 ) Mall , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor ptoteins GaIR , CytR , and Lad .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651130" target="_blank">7651130</a>]<br></li>
<li>Mall , anove protein involved in regulation of the maltose system of Escherichia coli is highlyhomologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to therepressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_9">
Reidl , J. , Römisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></label>
<input id="act_similarity_9", type="checkbox">
<div class="ACT_similar">
<li>Reidl , J. , RoÈmisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692154" target="_blank">10692154</a>]<br></li>
<li>Reidl J , Romisch K , Ehrmann M & Boos W ( 1989 ) MalI , a novel protein involved in the regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003949" target="_blank">12003949</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618441" target="_blank">12618441</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GaIR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1856179" target="_blank">1856179</a>]<br></li></div>
<li>As first demonstrated by Krieger-Brauer and Braun ( 19 ) , expression of the tsx gene is under double negative control of the DeoR and CytR proteins , which are known to regulate the synthesis of two nucleoside transport systems ( NupC and NupG ) and several nucleoside-catabolizing enzymes ( for reviews , see references 14 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>To anatyse the effect of CytR on cytR-lacZ * expression under more physiologicat conditions , we constructed a plasmid , pGP18 , that carried the cytR * gene in the pSCIOI-derived , tow - copy-number vector , pt-ISG576 , which is present in the cetl in approximatety six copies per chromosome ( Tat < eshita et ai , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>1978 ) ; therefore , introduction of a high copy-number plasmid -LCB- 50-100 copies/cell ) encoding a CytR target will decrease the concentration of free CytR represser and consequently result in derepression of CytR-regulated promoters on the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_18">
A , DNase I footprint titration conducted under standard conditions shows increasing protection as a function of CytR concentration in lanes 1 -- 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_18", type="checkbox">
<div class="ACT_similar">
<li>( A ) DNase I footprint titration conducted under standard conditions shows increasing protection as a function of CytR concentration in lanes 1-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>However , in the light of the DNase I footprinting results discussed above , it seems likely that formation of the repression complex in nupG is impaired to such an Structural implications It was previously shown that CytR recognizes operator half-sites separated by 2 or 3bp ( Pedersen et al. , 1991 ; Rasmussen et aL , 1993 ; Hoist et aL , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Q 1998 Blackwell Science Ltd Results Effect of spacing changes on repression in vivo To examine the effects of changing the spacing between CytR operator half-sites on complex formation and on in vivo repression we employed a low-copy-number plasmid carrying a truncated cdd promoter ( cddPD79 ) joined to lacZYA ( carries cdd sequence information from ¹ 79 to þ31 ; Valentin-Hansen et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>RESULTS A Simple Three-StepPurification of CytR-As the intracellular concentration of CytR is relatively low , the expression of cytR is autoregulated , and the Shine-Dalgarnsoequence of cytR is rather poor ( Valentin-Hansen et al. , 1978 ; Gerlach et al. , 1990 ; Valentin-Hansen et al. , 1986 ) , aplasmid allowing production of large amounts of CytR was constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Most likely , the modest synthesis of CytR was due to the presence of the authentic cytR Shine-Dalgarno sequence , which shows low efficiency , in p19-13-C ( Valentin-Hansen et al. 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Regulation of the gcv operon by GcvA and GcvR is part of a growing number of prokaryotic regulatory systems involving an activator -RCB- antiactivator complex , including TraR -RCB- TraM regulation of conjugal transfer of the Ti plasmid in Agrobacterium tumefaciens ( Hwang et al. , 1999 ) , NifA -RCB- NifL regulation of nitrogen ﬁxation in Klebsiella pneumoniae ( Narberhaus et al. , 1995 ) , ComK -RCB- MecA -RCB- ClpC regulation of competence in Ba-cillus subtilis ( Turgay et al. , 1998 ) and CRP -RCB- CytR regulation of operons encoding the nucleoside transport and biosynthesis enzymes ( Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>Moreover , the CytR repressor , which acts by antagonizing CAP activation ( Martinussen et al. , 1989 ; Valentin-Hansen et al. ; 1989 ) , is now believed to int , eract directly with the CAMP-CAP complex ( Sogaard-Andersen et al. , 1990 ; L. Sogaard-Andersen , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>It was therefore proposed that CytR acts as an anti-activator , especially since CytR binding centered at around ± 65 is too far from the promoter to interfere with RNP directly ( Sùgaard Andersen et al. , 1990a , b ; Mùllegaard et al. , 1993 ; Valentin-Hansen et al. , 1996 ; Kristensen et al. , 1997 ; Meibom et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In particular , residue 54 of PurR ( corresponding to resi-dues 56 and 62 in LacI and CytR respectively ; Fig. 6 ) is crucial to proper function and most likely has to be a leu-cine or methionine ( Choi and Zalkin , 1994 ; Schumacher et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In a previous study , speciﬁc cleavage of PurR was observed after R52 , whereas neighbouring residues K41 and K55 were inaccessible ( corresponding to CytR residues R60 , R49 and K63 ; see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In support of this notion , all LacI members , but not CytR , carry a leucine at the position corresponding to Leu54 of the PurR repressor ( Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In support of this view , all LacI members carry a leucine or methionine at the position corresponding to leucine 54 of PurR , with the lone exception of CytR ( for an alignment of LacI members , see Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The amino acids involved in dimerization of CytR have been determined genetically ( Barbier and Short , 1993 ) , and correspond well with-amino-acids involved in dimerization of other LacI members ( Schumacher et al. , 1994 ; Lewis et al. , 1996 ; Friedman et al. , 1995 ; for review , see Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Although autoregulation of several transcriptional-regulators has been observed previously , e.g. CytR ( Pedersen et al. , 1992 ) , TyrR ( Pittard and Davidson , 1991 ) and GalS ( Weickert and Adhya , 1993 ) , in this case , the positive transcriptional-regulator of the maltose regulon , MalT , is subject to negative regulation by a second regulator , Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Proteolysis and NMR studies revealed that the interdomain linker of two of CytR 's closest relatives , the PurR and LacI repressors , is disordered in the absence of operator DNA ( Wade-Jard-etzky et al. , 1979 ; Choi and Zalkin , 1994 ; Nagadoi et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Indeed , many regulatory proteins , such as LuxR , CytR , and PurR are auto-regulated , indicating that tight control of global regulators is necessary for normal cellular functions ( Roy et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>A more salient observation is that this difference is constant for DG1 , M , DG2 , M , and DG3 , M. Thus , the relative distributions of deoP2 species with 1 , 2 , or 3 CytR dimers bound as assessed by gel mobility shift titration is identical to what is predicted when the footprint titration data are analyzed according to the Specific Additional Sites model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Molecular Microbiology ( 1994 ) 14 ( 5 ) , 989-997 T e ene nc he h g oding t periplasmic cyciophiiin homoiogo Piase A , in Escherichia coli , is expresse from four moters , three of which are activated by cAI\/IP-CR mpiex nd egatively reguiated by CytR e e , P pro P c d the o a n the repressor Mads Norregaard-Madsen , ^ Bente Mygind , ^ Rolf Pedersen , ^ Poui Valentin-Hansen ' * and Lotte Sogaard-Andersen ' '' '' ^ Department of Motecuiar Biology , Odense University .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Genetic data suggest that the synthesis of the larger mRNA species is directed by a weak promoter ( t ` I ) that is controlled by the DeoR repressor , whereas the smaller mRNA species is directed by the main promoter 1 ' 2 , which is negatively controlled by the CytR repressor and positively affected by the cyclic AMP/catabolite activator protein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>The Tsx protein , which enhances nucleoside permeation across the outer membrane , is subject to dual control by the DeoR and CytR repressors in addition to being positively controlled by the cyclic AMP-catabolite gene activator protein complex ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644187" target="_blank">2644187</a>]<br></li>
<li>It has been shown that the nupG promoter is activated 50-fold by the cyclic AMP-cyclic AMP receptor protein complex and repressed fourfold by CytR and DeoR and that it has binding sites for these proteins ( 29 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>regulated negatively by the two repressors DeoR and CytR and is activated by the cyclic AMP ( cAMP ) receptor protein-cAMP complex ; however , even under completely repressed conditions , a low basal level of purine nucleoside phosphorylase is maintained ( for a review , see reference 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559336" target="_blank">7559336</a>]<br></li>
<li>In light of these data , it seems reasonable to propose that the `` truncated '' CytR protein of V. cholerae is able to repress both udpPV c and udpPEc , while the more sitespeciﬁc `` full-size '' CytR protein of E. coli exhibits reduced ability to-bind the heterologous V. cholerae udpP promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Mstlegaard , N.E. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1990b ) Tandem DNA-bound CAMP - < ; RP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor In Escherichia coli Mol Microbiol 4 : 1595-1601 , Sogaard-Andersen , L. , Mironov , A.S. , Pedersen , H. , Sukhodelets , V.V. , and Valentin-Hansen , P. ( 1991 ) Single amino acid substitutions in the cyclic-AMP receptor protein specifically abolish regulation by the CytR repressor in Escherichia cofi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_43">
Single amino acid substitutions in the cAMP receptor protein specifically abolish regulation by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></label>
<input id="act_similarity_43", type="checkbox">
<div class="ACT_similar">
<li>Single amino acid substitutions in the cAMP receptor protein speciﬁcally abolish regulation by the CytR repressor in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li></div>
<li>Sukho-delets , V.V , , and Valentin-Hansen , P. ( 1991a ) Single amino acid substitutions in the cAMP receptor protein specifically abolish reguiation by the CytR repressor in Escherichia coii Proc Nati Acad Sd USA 88 : 4921-4925 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>S ~ gaard-Andersen , L. , Mironov , A.S. , Pedersen , H. , Sukho-delets , V.V. , and Valentin-Hansen , P. ( 1991a ) Single amino acid substitutions in the cAMP receptor protein specifically abolish regulation by the CytR repressor in Escherichia coil Proc Nat/Acad Sci USA 88 '' 4921-4925 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_47">
Sogaard-Andersen , L. , Mironov , A. S. , Pedersen , H. , Sukhodelets , V. V. , and Valentin-Hansen , P. ( 1991 ) Single amino acid substitutions in the cAMP receptor protein specifically abolish regulation by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></label>
<input id="act_similarity_47", type="checkbox">
<div class="ACT_similar">
<li>Sùgaard-Andersen , L. , Mironov , A. , Pedersen , H. , Sukhodelets , V. and Valentin-Hansen , P. ( 1991a ) Single amino acid substitutions in the cAMP receptor protein speci ® cally abolish regulation by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Søgaard-Andersen , L. , Mironov , A.S. , Pedersen , H. , Sukhodelets , V.V. and Valentin-Hansen , P. ( 1991a ) Single amino acid substitutions in the cAMP receptor protein speciﬁcally abolish regulation by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Søgaard-Andersen , L. , Mironov , A. S. , Pedersen , H. , Sukho-delets , V.V. , and Valentin-Hansen , P. ( 1991 ) Single amino acid substitutions in the cAMP receptor protein speciﬁcally abolish regulation by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
<li>The CytR repressor antagonizes cyclic AMP-cyclic AMP receptor protein activation of the deo & promoter of Escherichia co/i K-l 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_52">
The CytR repressor antagonizes cyclic AMP cyclic AMP receptor protein activation of the deoCP2 promoter of Esche-richia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></label>
<input id="act_similarity_52", type="checkbox">
<div class="ACT_similar">
<li>The CytR repressor antagonizes cyclic AMP-cyclic AMP receptor protein activation of the deoC , , promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>The CytR repressor antagonizes cyclic AMP-cyclic AMP receptor protein activation of the deoCp2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The CytR repressor antagonizes cyclic AMP -- cyclic AMP receptor protein activation of the deoCp2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></li></div>
<li>Søgaard-Andersen L , Martinussen J , Møllegaard NE , Douthwaite SR , ValentinHansen P ( 1990a ) The CytR repressor antagonizes cyclic AMP-dependent AMP receptor protein activation of the deoCp2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Sùgaard-Andersen , L. , Martinussen , J. , Mollegaard , N.E. , Douthwaithe , S.R. and Valentin-Hansen , P. ( 1990a ) The CytR repressor antagonizes cyclic AMP ± cyclic AMP receptor protein activation of the deoCp2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Structural implications : the CytR repressor protein CytR is a dimer in solution and when bound to the deoP2 promoter ( H-H .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>' The CytR protein was expressed from the tat promoter in pC19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Holt AK & Senear DF ( 2013 ) The cooperative binding energetics of CytR and cAMP receptor protein support a quantitative model of differential activation and repression of CytR-regulated class III Escherichia coli promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>CAP enhances the binding of the transcriptional-repressor CytR to the deoP2 promoter ( Mollegaard et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>However , when overexpressed , a mutant CytR protein lacking its DNA-binding domain can repress the deoP2 promoter in vivo ( Søgaard-Andersen and Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_63">
Multiple speciﬁc CytR binding sites at the Escherichia coli deoP2 promoter mediate both cooperative and competitive interactions between CytR and cAMP 35 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></label>
<input id="act_similarity_63", type="checkbox">
<div class="ACT_similar">
<li>Multiple speciﬁc CytR binding sites at the Escherichia coli deoP2 promoter mediate both cooperative and competitive interactions between CytR and cAMP receptor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Perini , L. T. , Doherty , E. A. , Werner , E. , and Senear , D. F. ( 1996 ) Multiple specific CytR binding sites at the Escherichia coli deoP2 promoter mediate both cooperative and competitive interactions between CytR and cAMP receptor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Pedersen , H. , Sogaard-Andersen , L. , Hoist , B. , and ValentinHansen , P. ( 1991 ) Heterologous cooperativity in Escher-ichia co/i : the CytR repressor both contacts DNA and the cAMP receptor protein when binding to the deeP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_67">
Pedersen , H. , Søgaard-Andersen , L. , Holst , B. and Valentin-Hansen , P. ( 1991 ) Heterologous cooperativity in Escherichia coli : the CytR repressor both contacts DNA and the cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_67", type="checkbox">
<div class="ACT_similar">
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , and ValentinHansen , P. ( 1991 ) Heterologous cooperativity in Escheri-chia coli : the CytR repressor both contacts DNA and the cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , and ValentinHansen , P. ( 1991 ) Heterologous co-operativity in Escheri-chia coli : the CytR repressor both contacts DNA and the cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>The CytR repressor both contacts DNA and the CAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_71">
The CytR repressor both contactsDNA and the cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7551052" target="_blank">7551052</a>]<br></label>
<input id="act_similarity_71", type="checkbox">
<div class="ACT_similar">
<li>The CytR repressor both contacts DNA and the cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8388874" target="_blank">8388874</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Heterologous co-operativity in Escherichia coli : the CytR repressor both contacts DNA and the CRP protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_74">
Heterologous cooperativity in Escherichia co/i : the CytR repressor both contacts DNA and the CAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></label>
<input id="act_similarity_74", type="checkbox">
<div class="ACT_similar">
<li>Heterologous cooperativity in Escherichia coli : the CytR repressor both contacts DNA and the cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Heterologous cooperativity in Escherichia coli : the CytR repressor both contacts DNA and cAMP receptor protein when binding to the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li></div>
<li>[ 30 ] L.E. Thomsen , M. Pedersen , M. Nørregard-Madsen , P. ValentinHansen , B.H. Kallipolitis , Protein -- ligand interaction : Grafting of the uridine-speciﬁc determinants from the CytR regulator of Salmonella typhimurium to Escherichia coli CytR , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Since then , nine Escherichia coli genes were experimentally shown to be regulated by CytR and form the CytR regulon : deoC [ 4,5 ] , cytR itself [ 6 ] , tsx [ 7 ] , cdd [ 8 ] , ppiA [ 9 ] , nupC [ 10 ] , nupG [ 11 ] , udp [ 12 ] , rpoH [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Consist-ent with this view , we show here that although CRP andRNA polymerase interact cooperatively at deoP2 an even higher synergismbetween CRP andCytR in binding to this promoter preventstranscriptioninitiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>This pattern of cooperative effects adds to the diversity of cytidine-mediated allosteric effects in CytR and provides a ready explanation for the different extents of induction of gene expression as are observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This repressive effect of CytR was abolished by the addition of cytidine , an allosteric-effector for CytR ( Figure 5 , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>This effect is specific for cytidine , since the addition of uridine , which functions as an inducer for CytR only in Salmonella typhimurium ( 15 , 24 ) , does not change the footprint ( Fig. 3B , compare lanes 5 and 7 with lanes 6 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Specificity of CytR repression was verified as complete derepression in the presence of the inducer cytidineandtetrahydrouridine , an inhibitor of cytidine deaminase ( Valentin-Hansen ea t l. , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Furthermore , in strains that slightly overproduce CytR , transcription from P3 , P4 and P5 is reduced even further ( < 10-fold ) , and repression can be fully neutralized by the addition of the inducer cytidine to the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Finally , addition of the inducer cytidine to the growth medium fully neutralizes CytR repression ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Repressor of cytR Repressor of deoR UdP Conversion of uridine to uracil plus CytR ribose-1-phosphate Conversion of deoxyribose-5-phosphate CytR , DeoR to glyceraldehyde-3-phosphate deoCABD mflADR rbsACB CVtR deoR Transport and assimilation of o-mannitol MtlR Transport and assimilation of o-ribose RbsR Transcriptional repressor Transcriptional repressor Q 1996 Blackwell Science Ltd , Molecular Microbiobgy ,22 , 21-2 Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Taken together , the data shown in Tabte 2 demonstrate that the expression of the chromosomal ` i > -LCB- cytR-lacZ * -RCB- ^ operon fusion is specifioatly reduced by the CytR repressor , strongly suggesting that cytR trans-cription is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>However , CytR-mediated repression of tSx-p2 iS impaired to such an extent that the mutants no longer respond to the CytR repressor when expressed from the chromosomal cytR + gene ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>However , unlike a typical bacterial repressor , CytR binds with only modest afﬁnity to its operators and can not repress its cognate promoters independently in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Their structurat genes form a reguton controtled by the DeoR and CytR repressor proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Under these conditions of low CytR concentration , CytR might be an ineffective repressor of the latter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>1unit of CytR repressor activity is defined as the amountof CytR repressor that represses the @ - galactosidase expressionby 50 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Cytidine and adenosine are the inducers for the CytR repressor in vivo ( 15 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Summary Unlike cfassicat bacterial repressors , the CytR repressor of Escherichia coli can not independently regulate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_95">
Amino acid substitutions in the CytR repressor which alter itscapacityto regulate gene expression , J. Bacterial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></label>
<input id="act_similarity_95", type="checkbox">
<div class="ACT_similar">
<li>Amino acid substitutions in the CytR repressor which alter its capacity to regulate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li></div>
<li>Both cdd and udp are members of the CytR regulon , i.e. , the expression of these genes is negatively regulated by the CytR repressor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Transcriptionof cytR by Northernanalysiswas followed in order to find out whether the increased expression of cdd and udp in CSH50 , by comparison with CSH50 Afis , was due to transcriptionalregulationof the CytR repressor by FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Still , at the physiological concentration of CytR , the CytR-DNA interactions are absolutely necessary for the repressor complex to be formed [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>The CytR repression site extends from -30 to -100 , but CytR repressor can not bind this DNA on its own .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The amino terminus pur repressor is not known , but thepurR 5 ' sequence suggests in GalR , LacI , and CytR has been implicated in DNA recog - an interesting possibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058704" target="_blank">3058704</a>]<br></li>
<li>The presence of pDK7-CytR enhances repression of CytR-regulated promoters ( e.g. cddP or deoP2 ) three - to fourfold relative to chromosomal encoded repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The mechanism of transcriptional repression by the CytR repressor is clearly different from that of the classical bacterial repressors such as Lad , Cl , Cro and DeoR -LCB- Ptashne , 1986 ; for a review see Adhya , 1989 ) , A priori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>The CytR repressor regulates initiation of transcription from several operons whose products are involved in the uptake and catabolism of nucleosides and deoxynucleosides in E. coli ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>The mechanism by which CytR repressor stimulates the udp expression in dup361 , del224 , and del364 mutants is more difﬁcult to explain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The protection observed is specific for the CytR repressor , as no protection is detected when a CytR - extract is used ( see Fig. 5 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The following oligonucleotides were used : eno , 5 ' - TGACGGAGCAGCTGCCATACCGACGA-3 ; glpQ , 5-TC-CCATGACTATAGTGCTCATCATG-3 ; glpR , 5 ' - GCAGCGA-GCCGTCGCTATCGAT-3 ; Cdd , 5 ` - CAATGGCGTACGCGC-ACAGGCCGCCGCCAG-3 ; Udp , 5 ' - CAGGTAGTGAATTCG-CGGTGAGATGCCA-3 ; deoC , S'CTGATGACACAGGGCG - GATCACTCTC-3 ` ; deoR , 5 ' TCAGATCGCGACGAATCG-TCATCTCCG-3 ; CytR , 5-ACCCGAlTACGGGTGGCCTGG-GAGACT-3 ` ; mt/D , 5-GCATCAAGTACCACCTGAlTGACA-TCG-3 ; rbs8 , B-GTTAAGCGTGGAGACCACCAGCGCGAT-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_108">
The promoter regions of these genes can be classified in three groups regulated negatively by ( i ) CytR , or ( ii ) DeoR alone , or ( iii ) by both CytR and DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></label>
<input id="act_similarity_108", type="checkbox">
<div class="ACT_similar">
<li>The promoter regions for these genes can be classified in three regulatory groups : i ) DeoR regulated , ii ) CytR regulated and iii ) CytR and DeoR regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>The promoter regions can be classified in three regulatory groups , i ) CytR regulated , ii ) DeoR regulated and iii ) CytR and DeoR regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li></div>
<li>The nupC and nupG genes also differ in their modes 01 regulation and expression ; nupC has been shown to be subject to repressíon by the CytR protein whilst nupG is repressed by both CytR and DeoR ( Munch-Petersen and Mygind 1976 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>The cyfR ' - encoded gene product exerts negative controt on the transcription initiating from both the witd-type tsx P2 promoter and the CytR operator mutant , O ' ^ - I , tocated in this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>The DeoR and CytR regulatory proteins repress tsx transcription to different extents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The DeoR protein exerts only a weak negative effect on Tsx synthesis , whereas CytR represses tsx expression more strongly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The expression of the tsx gene is increased in the presence of nucleosides through the action of the DeoR and CytR repressors ( 98 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>The open symbols show systematic decreases in θ0 and θ1 and a systematic increase in θ2 when CytR Core domain dimer is added at ca. 3.5 and 10 µM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>-140 CAP - too - 8 0 -120 -- CAP - 6 0 - 4 0 The CytR * mutation results in increased CytR synthesis The cytR '' mutation we have previously described ( Bremer et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>The gel retardation in Figure 46 shows that CytR binds to all the pGEM-4 fragmentswhen present in a high concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The results we present here for udp confirm that CytR binds specifically and with relatively high affinity to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The independent afﬁnity of CytR for the C-sequences is considerably lower than for the A-sequences ( 2 - to 5-fold lower ; Figure 3 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The activity was partially constitutive and was only regulated two - to threefold by CytR under cya + conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>The putative turnresidues in PurR and CytR are consistent with thciosnclusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058704" target="_blank">3058704</a>]<br></li>
<li>Expression of the fused genes increased in the following order : wild-type < deoR < cytR < deoR cytR double mutant , where the absence of the DeoR repressor increased tsx-lacZ expression about 1.5 - to 2-fold and the absence of CytR resulted in 6-fold-enhanced , B-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>This shows that in the isolated mutant the repression of tsx-lacZ expression by CytR was strongly reduced but not completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>By taking advantage of the strong repression of tsx-lacZ expression in cells carrying the cloned cytR + gene , we have been able to isolate a mutant that shows strongly decreased repression by CytR but no change in regulation by DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>As shown in the top half of Table 2 , the Lys90-Gly substitution results in a more efficient repressor , whereas the substitutions of Thr-23 and Glu-24 result in a decrease in CytR repressor activity , with the Ala-24 + Glu substitution having the most prominent effect ( only 2-fold regulation ) and the Met-31 - + Lys substitution not affecting repressor activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>As shown in Table 1 , 3-galactosidase activities remained unaffected by CytR and were reduced 30-fold relative to the level in a cytR cya + strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>We studied this regulation by selecting and characterizing spontaneous constitutive mutations in the promoter of the udp ( uridine phosphorylase ) gene , one of the genes most strongly regulated by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>It is still distinguish between too soon to these possibilities ; however , the recent isolation of a crp mutant characterized by specifically inhibiting CytR regulation of the udp promoter ( 18 ) points to the second model as the most likely mechanism for CytR action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>It is still premature tc distinguish between the remaining two possibilities ; however , the recent isolation of crp mutants that exhibit unaltered promoter activation but specifically abolish CytR regulation -LCB- Mironov et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>CytR is also negatively autoregulated by twofold when CytR is supplied on a low-copy-number plasmid to cells containing a fusion of the cytR promoter and the lacZ gene ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Effect of altered half-site spacing in the CytR operator on b-galactosidase activitiesa of promoter-lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Schematic representation of the fold of CytR regulation in promoter derivatives with altered half-site spacing in the CytR operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The complexity of addition two new minor groove contacts , ascribed to negative regulation was first shown by the fact that CytR phosphate-CytR interactions , are observed at -60 be - regulation was missing in cells devoid of the CAMP-CPR tween theCRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>The results of these experiments are shown in Table 4 and a clear correlation is observed between CytR regulation of heterlogous promoters in udpP -- lacZ fusions ( Table 3 ) and their competence to-bind CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The results of this experiment are shown in Table II and a clear correlation is observed between CytR regulation of the truncated promoters ( Table IA ) and their competence to-bind CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The absence of consensus sequences for the CytR protein within the udp regulatory region of Y. pseudotuberculosis and V. cholerae ( Fig. 3 ) suggests that reduced CytR regulation of udpPYp and udpPV c was due to the reduced ability of the E. coli CytR protein to-bind heterlogous promoters Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The absence of this motif in udpPYp and udpPV c may be the main reason for abolished CytR regulation of these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The importance of the CRP2 target for CytR regulation is further emphasized by the fact that deletion of sequences upstream of - 8 0 eliminates CytR repression in vivo ( Munch-Petersen and Jensen , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The results reported here show that transcription of rpoH is regulated in vivo by the CytR anti-activator , thereby adding another layer of complexity to the regulation of the cellular levels of s32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Therefore , while we can not predict the specific effects of these different arrays of CytR sites on regulation of other promoters , we can reasonably speculate that the different balance of similar effects is critically involved in the mechanism of their differential regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Thus , udpPV c which was characterized by maximal reduction of CytR regulation and titration in vivo , also exhibited the lowest ability to-bind CytR in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>In this test , the competence of the cddP derivatives to-bind CytR is measured by their ability to alleviate CytR regulation when cloned in a high copy number plasmid , pUC 13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Deletion of sequences upstream of - 8 0 decreases transcription initiation more than 10-fold and abolishes CytR regulation ( Munch-Petersen and Jensen , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>CytR is often described as playing a dual role in regulation ( 30 ) because it can activate transcription under some circumstances , whereas in others , it represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In addition to this , the differential regulation exhibited by the CytR regulon presents a useful paradigm for more widespread mechanisms of coordinate regulation in both prokaryotes and eukaryotes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Next , we asked whether CytR regulation could be reestablished in the presence of CRP181EL or CRPI81EV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Keywords : Uridine phosphorylase structure ; Gamma-proteobacteria ; Heterologous expression ; CytR regulation 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>bThe fold of CytR regulation was calculated as the activity of i3-galactosidase in a CytR - strain divided by the activity in a CytR + strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR regulation was indeed reestablished by the increased concentration of CytR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>These duplications appear to be related to a highly specialized aspect of CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Negative regulation by CytR is , however , clearly enhanced in both cases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Hence , each square indicates the endpoint of the deletion in deoCp2 and the fold of CytR regulation in that particular construct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>RESULTS CytR regulation in deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>( Inset ) CytR regulation in deoCp2-lacZ fusions as a function of the endpoint of the deletion in deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Hence , CytR regulation of the mutant tSx-p2 promoters is strongly reduced but not completely eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Isolation of cddP mutants with reduced CytR regulation To define more precisely the sequences involved in CytR regulation , mutants of cddP with a reduced regulatory response to CytR were isolated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>To identify DNA sequences in tSX-p2 that are important for CytR regulation , we isolated mutants of tSx-p2 that show a reduced response to CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>To analyze the regulation of the mutated fusion by the chromosomally encoded DeoR and CytR repressors , we transduced this fusion into deoR and cytR strains and subsequently measured , B-galac-tosidase activity in glycerol - and glucose-grown cultures ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Therefore , the results presented in this study point to important differences in the regulation of tsx and the deo operon by CytR and DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>From these data we conclude that Tsx synthesis is subject to double negative control by the DeoR and CytR repressors , but is most strongly affected by positive regulation through the cAMPCAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Our data strongly indicate that such a cooperation between DeoR and CytR repressors does not play any role in tsx regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The periodicity is outlined in Fig. 2 where repression is depicted as a function of spacing between the centres of the CytR operator half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Strikingly , we observed in gel retardation assays that CytR 's afﬁnity for the different settings of operator half-sites parallels the in vivo repression of the spacing variants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>However , a key feature of the CytR regulon is that extents of activation , repression , and induction vary substantially among the different transcription units ( cf. Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The CytR regulon provides an ideal model to study combinatorial gene regulation because the number of transcription factors involved is quite small , only two , yet the extents of activation , repression , and induction vary substantially between the different promoters whose architectures are very similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>To determine whether the repression mechanism of CytR has been conserved , we commenced comparative studies of the arrangement of CytR regulated genes in several gamma-proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Furthermore , deo repression by CytR and DeoR is cooperative rather than additive ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>We propose that this structural adaptability is essential for CytR repression of promoters with diverse architectures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Evidently , dissociation of CytR is not necessary for induction of udpP nor is association sufficient for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Although it binds to only one primary operator , CytR appears to direct more than one mechanism of repression ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The mechanism of the CytR action is anti-activation rather than direct repression [ 17,31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Therefore , structural and functional rules governing the formation of repression and activation complexes in one promoter may not be valid for other promoters of the CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Thus the competence of a cloned target to titrate CytR can be measured by its ability to alleviate repression of a chromosomal , CytR-regulated promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>These results suggest that CytR repression also involves interference with interactions between RNP and the core promoter ( ` direct repression ' ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>To increase the level of understanding of the mechanism used by CytR to mediate repression of transcription , we have initiated an analysis of the protein-DNA interactions at a substrate-the tSx-p2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In nupG CytR binds to a region of approximately 30bp that includes the sequence 5 ' - TGCAA-AI-rGGGAATGT-TTGCA-3 ' , in which the proposed halfsite consensus sequences have been underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In other species , CytR was shown to regulate its own gene , cytR , in Salmonella typhimurium [ 14 ] and udp in Salmonella typhimurium , Yersinia pestis and Vibrio cholerae [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Gel retardation experiments performed in the presence of increasing concentrations of CytR clearly showed that at low concentrations ( < 0.4 nM ) of CytR protein , a single retarded complex was observed ( Fig. 5 , single arrow ) ; however , at higher concentrations ( > 0.5 nM ) an additional retarded complex was observed ( Fig. 5 , double arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Moreover , additional genes , which do not belong to the CytR regulon , are strongly induced in the presence of nucleosides ; for example , add and guaC , which encode adenosine deaminase and GMP reductase respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Finally , A12-29 has an afﬁnity for CytR similar to that of other sequences obtained after 12 selection rounds , despite the presence of an extra octamer repeat in this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In these derivatives , translation occurs from the native cyrR Shine-Dalgarno sequence , and , therefore , CytR is only synthesized at a low level in the presence of isopropyl B-o-thio-galactopyranoside ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In the presence of CytR alone , a specific DNase I footprint is detected in all four mutant promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In contrast to this , the xanthosine phosphorylase activity can be measured only when cells are grown in the presence of xanthosine , and the xanthosine phosphorylase activity is not inﬂuenced by DeoR or CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559336" target="_blank">7559336</a>]<br></li>
<li>' Fold of repression is expressed as the enzyme level in the absence over the level in the presence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>As an additional control , CytR titrations were conducted in the presence and absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Even in the presence of a high concentration of CytR in the cell , the level of expression from deoCp2 remains unaffected in a Acya background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Use of pJEL134 facilitated the screening for mutants with an elevated lacZ expression in the presence of the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Unusual duplications and deletions that resulted in constitutive udp expression that depended on the presence of CytR were also found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>For a centrosymmetric three-protein complex to form as envisioned in the model , CytR would have to dissociate from CytO and instead bind more widely spaced and symmetrically arranged DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , the additional interaction between CytR and CytO in the cooperative complex must place additional constraints on the conformation of the promoter DNA , constraints that would contribute to the large positive ∆ Goconnect for this process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In addition to the unliganded udp DNA band , at least three distinct lower mobility bands are observed as a function of increasing CytR concentration in these experiments ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>A , Phosphor storage plate image of the electrophoretic pattern showing unliganded DNA and a ladder of mobility-shifted bands that appear as a function of increasing CytR concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Even at this concentration , CT-DNA still had no affect on the CytR concentration dependence of first three of the lower mobility bands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We interpret this additional protection as reflecting bridged complex formation , which offers the underlying DNA partial protection form DNase I mediated cleavage even in the absence of direct physical contact between CytR Core domain and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Furthermore , the protection of this DNA segment is enhanced in the presence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the presence of operator DNA at concentrations that caused full complex formation of CytR , the cleavage pattern at R60 and the adjacent R/K-residues was unchanged , whereas R41 / R43 and R28 became inaccessible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>At physiological CytR concentrations , the CytR -- DNA interaction is absolutely required for formation of repression complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>DNA was isolated from the band containing CytR -- DNA complexes and ampliﬁed by error-prone PCR , in order to increase the DNA sequence diversity before the next round of selection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Notably , CytR-P2-His binds in vitro to DNA and represses transcription in vivo in a manner similar to wild-type CytR ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>A key to how CytR binds to different promoters is its relatively broad DNA sequence specificity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Specifically , CytR binds to multiple closely spaced or even overlapping sites on DNA with differing affinities and not to just a single operator as previously presumed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Isolates obtained after 12 rounds of selection bind with almost indistinguishable afﬁnities to CytR , and have 30-fold higher afﬁnity for the repressor than the deoP2-wt promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>More recently , it was shown that tactsare almost identical to thsoeseen in the repressionCRPand CytR bind cooperatively to the deoP2 promoter complex and stronR g NA polymerase contacts are seen forming a repression complex in which CytR is sandwiched in the -10 and in the +10 regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>In particular , CytR ∆ 2-63 results in net binding o represent the predicted populations of intermediates in the presence of 3 or 10 µM Core domain if ∆ G ≈ -7 kcal/mol for association of Core domain with CRP2-deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Protection is nearly eliminated at this concentration of CytR ∆ 2-63 , indicating significantly lower affinity for association with CRP2-deoP2 to form the bridged complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CRPcomplex the affinity of CytR for deoP2 is greatly increased , and the two proteins bind cooperatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Thus , these results clearly show that in the conditions used in this experiment , CytR can not independently bind to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>b-Galactosidase Expression from deoP2-kc2 Gene Fusions in Which the CytR-Binding Site Has Seen Substituted the Absence and Presence of CytR CytR Proteir ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In addition , CytR only represses transcription from deoP2 inthe presence of CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>First , CytR still represses transcription from deoP2 in the absence of a CytR operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Moreover , CytR inhibits deoP2 transcription conducted by RNP reconstituted with a-subunits lacking a-CTD ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>As an alternative , CRP2-deoP2 was titrated directly by CytR ∆ 2-63 as described earlier but in the presence of 1.0 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>( B ) deoP2 RNAs were quanti ® ed with a b-scanner and the total c.p.m. in the presence of CytR relative to that in the absence of CytR ( 100 % ) were plotted as a function of CytR concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>DNase I footprint of deoP2 in the presence of CytR extract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Electrophoretic Mobility Shift Analysis of CytR Binding to the udp Promoter -- To assess whether the array of CytR recognition motifs in the udp promoter constitutes separate , specific sites to which CytR can bind simultaneously , we first conducted titrations of CytR binding alone using electrophoretic mobility shift analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Gavigan , S. A. , Nguyen , T. , Nguyen , N. , and Senear , D. F. ( 1999 ) Role of multiple CytR binding sites on cooperativity , competition , and induction at the Escherichia coli udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In vitro , CytR only binds with a low affinity to specific binding sites ( Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Substitutions in these loops do not affect DNA binding or bending of the mutant proteins , but decrease their ability to function as adaptors for CytR ( Segaard-Andersen et al. , 1991 a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>In this context , it should be noted that the TATA box-binding protein binds the DNA in the minor groove ( Lee et al. , 1991 ; Starr and Hawley , 1991 ) , whereas CytR binds the DNA in the major groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Gel Mobility Shift Analysis of CytR Binding to deoP2 -- The parameter values in Table IV predict significant populations of deoP2 species with 0 , 1 , 2 , and 3 CytR dimers bound at intermediate CytR concentration , since the binding affinities of CytR for its three thermodynamically defined deoP2 binding sites differ by only 20-fold from highest to lowest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The results of this analysis ( Fig. 2B ) indicate that the first two CytR dimers that bind , bind to sites that have similar CytR binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The dashed curves are the isotherms from Figure 3 for binding of apo-CytR to these same sites under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>As a consequence , as detailed below , CytR does not bind only to CytO but also binds to additional binding sites whose number and arrangement appear to differ among the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In addition , we note that the mutations used to create the reduced-valence CRP1 and CRP2 promoters resulted in a 2 - - -3 - fold increase in CytR binding affinity for the competing sites in both cases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In this alternative view , CytR binding to CRP1 and to CRP2 would be nonspecific but would be nucleated by cooperative interactions with CytR bound to CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The possibilities are 1 ) that the additional CytR sites that overlap CRP1 and CRP2 are specific binding sites to which CytR binds noncooperatively and with defined affinity ; and 2 ) that the additional binding is nonspecific but cooperative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Because the affinities of CytR for CytO and for CRP2 differ by only 2 - to 3-fold in udp , the binding curves for these sites overlap extensively ; hence , CytR binding to CRP2 is necessarily of physiological relevance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Intrinsic binding of CytR was assumed to be unaffected by base pair substitutions in CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Symbols denote the different operator sites : , CRP2 ; Œ , CRP1 ; f , CytR ; ● , CytR binding to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>That is , if binding of CytR to sites flanking CytO is nonspecific , then protection of CRP1 and protection of CRP2 should have the same CytR concentration dependence , which they do not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This is because the favorable effect due to cooperativity with CytR binding to CytO is offset by the unfavorable effect of CytR binding to and competing for CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This is significant in relation to the CytR binding observed at CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>It therefore appears that the maximal reduction of CytR regulation observed in the case of udpPV c is most likely a con sequence of the combined effect of the absence of a CytR consensus sequence and the presence of an improved CRP1 binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The importance of this motif is strongly emphasized in this analysis as two of the mutations that result in decreased CytR regulation of cddP both map in the 5 ' - TGCAA box and in both cases , independent DNA binding by CytR is perturbed in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In this report we examined the effect of changing the spacing between CytR half-operators on CytR regulation in vivo and on CytR binding in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_237">
Thus , these data strongly suggest that the reduced CytR regulation of udpPYp and udpPV c was due to reduced CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></label>
<input id="act_similarity_237", type="checkbox">
<div class="ACT_similar">
<li>Thus , these data strongly suggest that the reduced CytR regulation is due to reduced CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li></div>
<li>Upon formation , both of the repression complexes are expected to repress nupG , as binding of the RNA polymerase most likely is excluded when the CytR target is occupied ( Mallegaard et aL , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Effect of spacing changes on CytR target occupancy To gain insight into the qualitative interactions of CytR with the DNA in repression complexes we performed DNase I and dimethyl sulphate ( DMS ) footprinting , which serves to delineate the binding sites and purines in close contact with protein respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Hence , the binding studies visualized that CytR made speciﬁc interactions to its half-operators in the repression complexes formed at the various promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Thus , the periodicity observed for repression seems merely to be a consequence of CytR 's binding preference for particular operator conﬁgurations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Therefore , it was most unexpected that the CytR operator as well as the DNA-binding domain of CytR was dispensable for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>It had been speculated that CytR binding to speciﬁc DNA sequences would also be important in repression , since CytR exhibits considerable homology to other repressors ( 21 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>It could result from increased synthesis of the wild-type CytR protein , since in the operator mutant CytR binding is not completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The importance of this cooperativity is highlighted by the fact that expression is induced because the cooperativity is lost when CytR binds cytidine ; cytidine binding to CytR has no effect on the intrinsic DNA binding of CytR ( 4 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The crucial role that cooperativity plays in repression is highlighted by the fact that when CytR binds cytidine , induction occurs as a result of the loss of cooperativity and despite no effect of cytidine binding to CytR on intrinsic binding of CytR to DNA ( 6 ) .2 The complexity of these regulatory properties has generated confusion about the nature of key molecular interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Cytidine binding to CytR results in a decrease in cooperativity but has little or no effect on the intrinsic affinity of CytR for binding to CytO ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Since cooperativity contributes over 2 orders of magnitude to the effective affinity for CytR binding , the effect is simply analogous to a decrease in intrinsic DNA-binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Because CytR couples ligand binding not to DNA binding but to cooperativity , allostery must have a different structural basis than that in other LacR/PurR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Unlike all other E. coli paralogues , for which binding of the regulatory ligand , whether inducer or corepressor , is linked to DNA-binding affinity , the intrinsic affinity of CytR for CytO is unaffected by cytidine binding to CytR ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>While regulatory ligand binding to most LacI family proteins is coupled intrinsic DNA-binding affinity , in CytR , cytidine binding is coupled to protein-protein cooperativity and not to intrinsic protein-DNA-binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus , CytR is a very adaptable DNA-binding protein that retains a high degree of ﬂexibility even in the presence of its co-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Surprisingly , the addition of cytidine or adenosine , the inducers for the E. coli CytR repressor ( 15 , 24 ) , did not prevent binding of CytR to the DNA ( Fig. 3A , lanes 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Summary of the Repression of deof2 Derivatives by Two CytR Proteins DNA-Binding Properties of Mutant CytR Repressors To test this hypothesis , it was essential to verify the proposed DNA binding features of the mutant CytR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CRPdepends on a DNA bend induced by binding of this complex , then CytR might act as ananti-bender , i.e.changing or eliminating the CAMPC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>To explain how CytR uncouples ligand binding from DNA binding , CytR is thought to connect the helix-turn-helix motifs to the globular core domain via peptide linkers in an extended conformation , rather than via hinge helices ( 3 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To investigate whether nonspecific CytR binding is responsible for these observations , the concentration of carrier , CT-DNA , was systematically varied from 0 to 3.2 mg/ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This extended protection reflects specific binding of CytR to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This requires nonspecific binding of CytR adjacent to the specific CytR site to be less probable at a given CytR concentration than elsewhere on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In fact , when we investigated CytR binding to the deoP2 promoter , we found that CytR does bind specifically to multiple sites ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>When we analyzed the deoP2 binding data according to this alternative model , we found that only negative cooperativity between CytR binding to CytO and CytR binding to flanking , nonspecific sites could account for the data ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Previously , we have shown that this spacing affects the CytR binding mode , i.e. , generating what we interpret to be a rigid conformation bound to CytO at deoP2 to a dynamically flexible conformation bound to CytO at nupG ( 25 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Footprint Titration Analysis of CytR Binding -- CytR binding to deoP2 was also investigated by footprint titration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Independent binding of CytR to the selected sequences thus produces a hierarchy of afﬁnities , A-operators C-operators deoP2-wt-oper - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>To test the importance of this distance for CytR binding a series of deoP2 derivatives with decreasing distances between the targets was constructed in pUC13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>For the independent binding of CytR , this corresponded to 7310 -- 8 M and 7310 -- 10 M CytR in the ﬁrst and ﬁnal selection round , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>For CytR homologues LacR and PurR , the conformational changes that accompany ligand binding are understood from X-ray crystallography ( 14 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Consistent with the interpretation that this reflects saturable binding of CytR Core domain , the extent of protection decreases with decreasing concentration of Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Presumably , binding to this particular setting of half-sites provokes a conformational change in the large C-terminal domain of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In the second model , CytR bound to the high affinity site 3 nucleates nonspecific binding on either side .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In these ways , site-specific CytR binding , cooperativity , and competition are inextricably linked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast , cytidine binding to CytR is not linked to CytR binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Dashed curves show cooperative binding of CytR in the absence of cytidine to ( from left to right ) wild type , CRP12 and CRP22 udp promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>DG13 , DG12 , and DG123 were set equal to 0 to mimic the loss of cooperativity that results from cytidine binding by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Effect of cytidine binding to CytR on cooperative binding of CytR to the udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>FIGURE 4 : Effect of inducer cytidine on cooperative binding of CytR to cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The Nonspecific Additional Sites model results in a free energy change for pairwise cooperative interaction between CytR dimers bound to adjacent sites that is positive , thus indicating negative cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>3 ) where DGi , M is the free energy change for binding of i CytR dimers to unliganded deoP2 ( i.e. related to the macroscopic product association equilibrium constant by K 5 i , M 2DGi , M/RT e ) and Z is the binding polynomial ( 51 ) equal to This formulation interprets each successive retarded band in the gel shift as representing one additional CytR dimer bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>One way to compare these results to those obtained from mobility shift analysis of CytR binding is to use these individual site binding free energy changes to calculate free energy changes corresponding to the macroscopic product association constants for binding of one , two , and three CytR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>i DGi , M is the free energy change for binding of i CytR dimers to the unliganded udp promoter ( i.e. related to the macroscopic product association equilibrium constant by Ki , M 5 2DGi , M/RT e The value of N in QN was treated as an adjustable parameter in the analysis , representing the average CytR stoichiometry of the higher order complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The band distribution in Fig. 2B was analyzed according to Equation 3 to estimate the free energy changes corresponding to macroscopic product equilibrium association constants for binding of one or more CytR dimers to udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Summing ∆ Gobridge ( -7.8 kcal/mol ) and the free energy change for intrinsic binding of CytR to CytO of either deoP2 ( -10.4 kcal/mol ; ref 9 ) or udpP ( -10.7 kcal/mol ; ref 11 ) and subtracting the free energy change for cooperative binding of CytR to these promoters ( -13.1 and -12.9 kcal / mol , respectively ) yields ∆ Goconnect ≈ +5 kcal/mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Therefore , the apparent free energy changes obtained in these analyses are intrinsic free energy changes for binding of CytR to the local sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>∆ G3 denotes the free energy change for intrinsic binding of CytR to CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Therefore , each loading free energy change so obtained is equivalent to the intrinsic free energy change for CytR binding to the local site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_287">
The total Gibbs free energy of each configuration relative to the unliganded reference state is given as a sum of contributions from four free energy changes for intrinsic binding of CRP and CytR ( DGk ) and three free energy changes for cooperative interaction between liganded sites ( DGij ( k ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_287", type="checkbox">
<div class="ACT_similar">
<li>The total Gibbs free energy of each configuration relative to the unliganded reference state is given as a sum of contributions from b five free energy changes for intrinsic binding of CRP and CytR and three free energy changes for cooperative interaction between liganded sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Binding of either CRP or CytR alone was analyzed to obtain the Gibbs free energy change corresponding to ki in Equation 1 ( DGi 5 2RT lnki ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>DGCytR , the loading free energy change for CytR binding alone ( no CRP ) is equal to the intrinsic free energy change , DG3 ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_291">
For simple binding of either CRP or CytR alone , Equation 1 gives the intrinsic free energy change for local binding , DGi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_291", type="checkbox">
<div class="ACT_similar">
<li>For simple binding of either CRP or CytR alone , eq 1 gives the intrinsic free energy change for local binding , ∆ G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li></div>
<li>For binding experiments in which both CytR and CRP are present , analysis according to Equation 1 provides an accurate estimate of the individual site loading free energy change , DGl , i ( 25 ) , and its confidence limits ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Fitting to Equation 1 was also used to estimate individual site loading free energy changes DGl , i ( 38 ) for binding experiments in which both CytR and CRP were present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At the highest CytR concentrations , deoP2 migrates as a single band whose mobility continuously decreases with increasing CytR concentration ( lanes 18-20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_296">
Final CytR concentrations : lanes 1 , no CytR ; lanes 2 , 5310 -- 8 M CytR ; lanes 3 , 10 -- 8 M CytR ; lanes 4 , no CytR ; lanes 5 , 5310 -- 9 M CytR ; lanes 6 , 5310 -- 10 M CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_296", type="checkbox">
<div class="ACT_similar">
<li>Final CytR concentrations ; lanes 2 -- 3 and 7 -- 8 : no CytR added ; lanes 4 , 5 and 9 : 10 ¹ 7 M CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>A ` '' P-end labeled 334-bp fragmentencodingtheentire deoP2 was employed astemplate.The CytR ' extractorcontrol extract was added to the final concentrations in nanograms/pl indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>The CytR proteins were added to the final concentrations indicated below each lane in nanograms per microliter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CytR proteins were added to the following final concentrations , CytR-wt : lane 3 , 0.3 nglul ; lane 4 , 0.03 nglul ; lane 5 , 0.003 nglul ; lane 6 , 0.0003 nglul .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>When present , CytR operator fragments were added to a ﬁnal concentration of 100 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The concentration of CytR in each lane for all three panels was as follows : 0 , 5 , 10 , 15 and 20 nM from left to right .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The CytR concentration used in each lane was as follows : 0 for lanes 1 , 5 and 9 ; 5 nM for lanes 2 , 6 and 10 ; 10 nM for lanes 3 , 7 and 11 ; 20 nM for lanes 4 , 8 and 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>But for simplicity , the fraction U4 was treated as representing exactly 4 CytR dimers bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Perhaps CytR differs from those by not forming stable structures even when bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Sufficient concentration of CytR ∆ 2-63 to evaluate the lower bound in this manner is currently not technically achievable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Therefore , at the standard state of 1 M , the effective concentration of CytR bound to CytO is 4 orders of magnitude less than free CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Repositioning might provide a configuration in which RNAP makes favorable contacts with upstream DNA sequences even while CytR is still bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>To evaluate the possibility of cooperative interactions between CytR bound to deoP2 and CytR bound to the upstream and/or P1 site ( s ) , the P2 regulatory region and the upstream and P1 regions were isolated on separate DNA fragments as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This result indicates no cooperative interaction between CytR bound to P2 and CytR bound to the upstream site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Identical residues in CytR and Lacl are indicated by vertical lilies and conservative changes by dotted lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>This operon is expressed from two differently controlled promoters , P1 and P2 , which are separated by 599 base pairs ; the former is regulated by the DeoR repressor , and the latter is regulated by both the CytR and DeoR repressors ( 10 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Expression of cytX is negatively regulated by CytR and the start site for transcription of cytX is only separated from the rot start codon by 220bp ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>These observations led us to consider whether transcription of the rpoH gene is negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Surprisingly , however , CytR still represses transcription 3.5 - to 4.2-fold in all of these promoters ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_316">
In these strains the expression of the lac genes is controlled by CytR regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></label>
<input id="act_similarity_316", type="checkbox">
<div class="ACT_similar">
<li>In these strains lac expression is controlled by CytR regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li></div>
<li>Most surprisingly , the three upstream rot promoters are negatively regulated by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>It should be emphasized that the five promoters were collected randomly and not selected as responding to CytR during the construction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Characteristically , CytR repressed promoters are activated by the CAMPC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>The difference in shape between the isotherms for CytR ∆ 2-62 and CytR ∆ 2-63 reflects different protein concentrations in the two experiments and the resulting effect on free vs total cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These ﬁndings suggest that additional assistance in protein folding is required under conditions in which CytR is induced ( i.e. in the presence of nucleosides ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The fact that protein chaperones are required for various protein transactions during normal growth raises the possibility that additional assistance in protein folding is required under conditions in which CytR regulation is induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Induction of CytR Next , we examined the response of the various D N A-protein complexes to cytidine , the inducer of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>No direct interaction between CytR protein and DNA was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Alternatively , an unknown surface on CytR protein might interact directly with RNA polymerase and increase its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CytR protein is also dimeric [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>To test whether synthesis of CytR is increased , we expressed the parent cytR * ptasmid pCB008and its cy ( R ' derivative , pGP1 , in minicetts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Effect of Cytidine Binding to CytR on Cooperativity -- CytR has a unique mechanism of induction among LacI family repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Its affinity to its operators is rather weak [ 22 ] and because of that , in contrast to most prokaryotic repressors , CytR alone is not capable to repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Transcriptionof udp and cdd is downregulated by CytR , while deoC is repres-sed by both DeoR and CytR ( Hammer-Jespersen and Munch-Petersen , 1975 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>An interesting consequence of this pattern of effects is that cooperativity is still complementary pairwise when CytR is liganded by cytidine , just as in the absence of cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Even in the absence of cytidine , the CytR conformation must not be optimal , as the relatively weak cooperativity attests .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR binds to cl ~ ol' ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>First , CytR binds with only modest afﬁnity to its operators and can not act on its own in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>However , much of this adaptability appears to be lost when CytR binds cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Expression is induced when CytR binds an effector ligand , cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Loss of cooperativity is apparent in the rightward shift of the curve when CytR binds cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>When CytR binds cytidine , cooperativity is reduced resulting in increased transcriptional activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Under these circumstances CytR typically does not completely reduce the activated level of expression to the basal level ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR regulates expression of genes for the transport and catabo - ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>]<br></li>
<li>CytR regulates expression from different cistrons of enzymes and transport proteins involved in both catabolism and recycling of nucleosides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Lane 1 , SØ929 þpBK4026 ; lane 2 , SØ928 þ pBK4026 ; lane 3 , SØ929 þpPS3514 ; lane 4 , SØ928 þpPS3514 ; lane 5 , SØ929 þpBK4026 þpDK7 ; lane 6 , SØ929 þpBK4026 þ pDK7-CytR ; lane 7 , SØ929 þpPS3514 þpDK7 ; lane 8 , SØ929 þ pPS3514 þpDK7-CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Lane 1 shows the cleavage pattern in the absence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Induction would be easily understood to be the result of decreased occupancy of CytO by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Transcription is repressed by CytR , a * This work was supported by NSF Grant MCB-9513661 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This would proportionally diminish the effectiveness of CytR as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These repressors , except for PurR and CytR , regulate genes for the catabolism of sugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>]<br></li>
<li>These tabulated values confirm no difference in bridging behavior between CytR ∆ 2-62 and CytR ∆ 2-63 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Therefore , CytR appears to ` ﬁne tune ' rpoH transcription under most conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Protein prepared in this manner is at least 95 % CytR dimer under native conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Consequently , CytR remains associated to similar extents under inducing and repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>deof7 is regulated by DeoR , while deoP2 is regulated by both DeoR and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>NupC is also known to be regulated by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>How CytR exerts its effect still remains to be elucidated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Gly a t position 20 is replaced by lysine and asparagine in CytR and PurR , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058704" target="_blank">3058704</a>]<br></li>
<li>R , RbsR ; G , GalR ; L , Lad ; M , MalI ; C , CytR ; P , PurR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>Two of the other members of this family , PurR and CytR , are negatively autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>the affinity of CytR for rlroPB is increased lOWfold and , similarly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>CytR differs from such repressors in that it only inhibits factor-dependent initiations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Binding of CytR to wt and point mutation derivatives of ( ddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Cytidine is an allosteric inducer of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Effect of excess CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Effect of Cytidine Binding to CytR on CytR Bridging .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Effect of Cytidine Binding to CytR on Cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Accordingly , CytR is induced by oytidine or adenosine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>E. CytR target in nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>M. Smith , dues in LacI and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058704" target="_blank">3058704</a>]<br></li>
<li>Thus , the heterogeneous nature of natural CytR operators , which appeared at ﬁrst rather puzzling , can now be understood in detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Thus , a reduced intrinsic afﬁnity for its operator may be the price CytR has to pay for increased structural ﬂexibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>43 2 Biochemistry 2010 , 49 , 432 -- 442 DOI : 10.1021 / bi901583n An Unusual Pattern of CytR and CRP Binding Energetics at Escherichia coli cddP Suggests a Unique Blend of Class I and Class II Mediated Activation † Allison K. Holt and Donald F. Senear * Department of Molecular Biology and Biochemistry , University of California , Irvine , California 92697 Received September 9 , 2009 ; Revised Manuscript Received November 4 , 2009 ABSTRACT : Two transcription factors , CRP and CytR , mediate positive and negative control of nine cistrons involved in nucleoside catabolism and recycling in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Biochemical and genetic studies have shown that CytR relies on interactions with the cAMP * CRPcomplex in order to-bind promoterDNAand repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>This effect is most probably due to the strongly reduced CytR binding , emphasizing the dual control of the tsx P2 promoter by CytR and the cAMP-CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>A precedent is the CytR repressor , which has poor afﬁnity for its operator but whose binding is enhanced 100 - to 1,000-fold when the CAP-cAMP complex is present ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>This proposal is supported by mutant and in vitro anatysis of the deo P2 , tsx P2 and cdd promoters , alt of which are affected by CytR and cAMP/CAP and contain tandem cAMP/CAP binding sites ( Vatentin-tHansen , 1982 ; Martinussen etai , 1988 ; Bremer etai , 1988b ; Vatentin-Hansen etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Such an overtap , and the presence of two olosety spaced cAMPCAP binding sites , appears to be typicat for promoters regutated by both CytR and cAMP/CAP and seems to be essential for CytR-mediated repression ( Martinussen et ai , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>This is different from the CytR repressor which only binds to its operators in the presence of cAMP/CAP ( Gertach et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>the levels of found in Hence , high , B-galactosidase activity to the activation of be due glycerol-grown cytR strains must tsx transcription by the cAMP-CAP complex , which is prevented when the CytR repressor is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>This implies that the presence of the CytR repressor prevents activation of transcription initiation at P2 by the cAMP-CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Thus , while repression , which we view as modulation of activation , depends on a complex interplay between cAMP , CytR , and cytidine concentrations , the initial activation is controlled only by cAMP concentration and is largely independent of CytR and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In both the nag and cyt cases the activator protein , cAMP/CAP complex , has been recruited to a second role in repression , essential in the CytR case and modulating in the NagC case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>tn view of the modet for the CytR-mediated repression of gene expression , it has been speculated that transcription of cytR is subjected to regutation by CytR and the cAMP/CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>All known promoters repressible by CytR are activated by the cAMP-CAP complex , while transcription occurring from DeoR-controlled promoters is independent of this activator complex ( 14 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>is activated by the cAMP-CAP complex and is negatively controlled by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>We show that cytR expression is negatively controlled by the CytR protein and positively affected by the cAMP/CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Our results suggest that tsx expression is controlled at two separate and differently regulated promoters : the weaker promoter ( P1 ) is repressible by DeoR , while the stronger promoter ( P2 ) is subject to negative and positive control by the CytR repressor and the cAMP-CAP complex , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The weaker promoter ( P1 ) is repressible by DeoR , While the main promoter ( P2 ) is subject to negative and positive control by CytR and the cAMP-CAP complex , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The use of tsx-lacZ fusions has allowed us to study the interrelationship of the DeoR , CytR , and cAMP-CAP regulatory proteins and to quantitate their contribution to the regulation of tsx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>However , the interrelationship of the double negative ( DeoR ; CytR ) and positive ( cAMP-CAP ) control in tsx expression has not been analyzed to any great extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>A mutant was isolated that showed unaltered tsx regulation by DeoR and the cAMP-CAP complex but strongly reduced repression by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Thus , the positive regulation of tsx expression mediated by the cAMP-CAP complex has a stronger effect than the negative regulation by the DeoR and CytR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>In thisreport we further investigate the combinatorial binding of cAMP - CRP andCytR in order to gain an understanding of the interactions involved in the formation of this nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_393">
Brikun , I. , Suziedelis , K. , Stemmann , O. , Zhong , R. , Alikhanian , L. , Linkova , E. , Mironov , A. , and Berg , D.E. ( 1996 ) Analysis of CRP-CytR interactions at the Escheri-chia coli udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></label>
<input id="act_similarity_393", type="checkbox">
<div class="ACT_similar">
<li>Brikun , I. , Suziedelis , K. , Stemmann , O. , Zhong , R. , Alikhanian , L. , Linkova , E. , Mironov , A. and Berg , D.E. ( 1996 ) Analysis of CRP -- CytR interactions at the Escherichia coli udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>[ 5 ] I. Brikun , K. Suziedelis , O. Stemmann , R. Zhong , L. Alikhanian , E. Linkova , A. Mironov , D.E. Berg , Analysis of CRP -- CytR interactions at the Escherichia coli udp promoter , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>[ 38 ] I. Brikun , K. Suziedelis , O. Stemmann , R. Zhong , L. Alikhanian , E. Linkova , A. Mironov , D.E. Berg , Analysis of CRP -- CytR interactions at the Escherichia coli udp promoter , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li></div>
<li>75 , 557-566 , November 5 , 1993 , Copyright 0 1993 by Cell Press Protein-Protein Interactions in Gene Regulation : The CAMP-CRP Complex Sets the Specificity of a Second DNA-Binding Protein , the CytR Repressor Lotte Segaard-Andersen and Poul Valentin-Hansen Department of Molecular Biology Odense University DK-5230 Odense M Denmark Summary Maximal repression by the CytR protein depends on the formation of nucleoprotein complexes in which CytR interacts with DNA and with CAMP-CAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_398">
Kallipolitis , B.H. , Norregaard-Madsen , M. and Valentin-Hansen , P. ( 1997 ) Protein ± protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></label>
<input id="act_similarity_398", type="checkbox">
<div class="ACT_similar">
<li>Kallipolitis , B.H. , Nørregaard-Madsen , N. , Valentin-Hansen , P. ( 1997 ) Protein-protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Kallipolitis , B.H. , Nørregaard-Madsen , M. , and ValentinHansen , P. ( 1997 ) Protein-protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_401">
Protein -- protein communication : Structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101127" target="_blank">12101127</a>]<br></label>
<input id="act_similarity_401", type="checkbox">
<div class="ACT_similar">
<li>Protein -- protein communication : structural model of the repression complex formed by CytR and the global regulator CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li></div>
<li>Unlike the other members of this protein family , CytR binds with only modest afﬁnity to its operators and transcription repression thus relies on the formation of nucleoprotein complexes with the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>These results indicate that CytR is highly adaptable to different arrangements of CytR and CRP sites in the absence of cytidine in order to form the three-protein repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Consistent with the expectation that cooperativity should enhance-the-activity of CytR as an RCTD antagonist , the three protein , CRP-CytR-CRP complex is required for repression ; CytR alone does not repress basal level transcription ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>As previously shown CytR and CAMP-CRP bind simultaneously to deoP2 in a highly cooperative manner and the protection pattern of the repressor complex is consistent with a model in which CytR is trapped between r -93 40 Uranyl Photofootprinting CRP-2 CRP-CytCRomplex-Uranyl photofootprinting reveals contacts between a DNA-bound ligand and thephosphates of the DNA backbone ( Nielsen et al. , 1988 ; Jeppesen and Niel-sen , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>These observations are not easily explained by current models for the action of pro-karyotic repressors ; instead we favour a model in which the interaction of CytR with deoP2 depends on the presence of tandem DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Hence , CytR relies on interactions with DNA-bound cAMP-CRP complexes in order to-bind strongly to deoP2 , and cAMP-CRP may be viewed as an adaptor that allows CytR to associate with deoP2 ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In this paper , we show that the cAMP-CRP complex is indeed a crucial component of fhe CytR repression apparatus , and that CyfR relies on interactions with DNA-bound activator proteins in order to-bind at deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Summary Repression by CytR depends on the formation of nucleoprotein complexes in which the CytR repressor and the cAMP-CRP activator complex bind cooperatively to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Detailed investigation of the deo operon has shown that cAMP-CRP and CytR bind cooperatively to the deoP2 promoter forming a nucleoprotein complex in which CytR is sandwiched between two DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR simultaneously makes specific DNA contacts as well as protein-protein contacts to cAMP-CRP in the repression complexes ( Pedersen et ai , 1991 ; Sogaard-Andersen and ValentinHansen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>It has consistently been shown that CytR and cAMP-CRP bind cooperatively to deoP2 forming a nucleoprotein complex in which the repressor protein bridges the tandem DNA-bound cAMP-CRP ( Pedersen et al. , et complexes 1991 ; S0gaard-Andersen al. , The 1991a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>These results are similar to those reported for deoP2 ( Pedersen et al. , 1991 ) and the in vitro analyses strongly indicate that the same type of repression complex , consisting of CytR sandwiched between tandem DNA-bound cAMP-CRP complexes , is involved in negative regulation of these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Taken together , these data point to a strong coupling between D N A binding of cAMP-CRP and the extent of CytR regulation in deoP2 , and , thus , strongly corroborate the conclusion obtained from in vitro experiments that CytR relies on interaction with tandem DNA-bound cAMP-CRP complexes in order to-bind to deoP2 D N A ( Sogaard-Andersen et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>The mechanism of repression by CytR proposed in this study is that the cAMP ± CRP bound at ± 41.5 undergoes an allosteric change upon direct interaction with CytR such that it no longer maintains a productive interaction with the N-terminal domain of a , but instead acts as a repressor to interfere with RNA polymerase acting on deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_417">
Tandem DNA-bound CAMP-CRP complexes are required for transcriptional repression of the deof2 promoter by the CytR repressor in Escherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></label>
<input id="act_similarity_417", type="checkbox">
<div class="ACT_similar">
<li>Tandem DNA-bound CAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Tandem DNA bound CAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li></div>
<li>J Bacteriol 172:5706 -5713 Sogaard-Andersen L , Mollegaard NE , Douthwaite SR , ValentinHanson P ( 1990b ) Tandem DNA-bound cAMP-CRP complexes are required for tanscriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_423">
Søgaard-Andersen , L. , Møllegaard , N.E. , Douthwaite , S.R. and ValentinHansen , P. ( 1990 ) Tandem DNA-bound cAMP -- CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_423", type="checkbox">
<div class="ACT_similar">
<li>Sùgaard-Andersen , L. , Mùllegaard , N.E. , Douthwaite , S.R. and ValentinHansen , P. ( 1990b ) Tandem DNA-bound cAMP ± CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>Segaard-Andersen , L. , Mollegaard , N.E. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1990 ) Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deeP2 promoter by the CytR repressor in Escherichia coil Mo / Microbiol 4 : 1595-1601 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Sogaard-Andersen , L. , Mollegaard , N. E. , Douthwaite , S. R. , and Valentin-Hansen , P. ( 1990 ) Tandem DNA-bound cAMP-CRP complexes are required for transcriptional repression of the deoP2 promoter by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In conclusion , the structure of the cdd promoter supports and extends the idea that CytR and CRP are rivals and , consequently , that bound CytR repressor inhibits RNA synthesis by antagonizing CRP activation -LCB- Martinussen et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Hence , CytR and RNA polymerase donot form co-complexes at deoP2 , and thepresence of even low concentrations of the repressor prevents the formation of the transcription initiation complex ( lane 15 ; the cleavage patterns seen in lane 14 and 15 are identical to those obtained with CRP and CytR alone , lanes 16 and 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>This plasmid is a derivative of the well characterized transcription vector pSA508 ( Choy and Adhya , 1993 ) , and carries all the deoP2 sequence inform-ation ( from ± 116 to +25 ) required for cAMP ± CRP activation and CytR repression , followed by a strong factor-independent transcription terminator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_430">
Mollegaard , N.E. , Rasmussen , P.B. , Valentin-Hansen , P. , and Nielsen , P.E. ( 1993 ) Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymerase for activation of transcription on the Escherichia coil deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_430", type="checkbox">
<div class="ACT_similar">
<li>Mùllegaard , N.E. , Rasmussen , P.B. , Valentin-Hansen , P. and Nielsen , P.E. ( 1993 ) Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymerase for activation of transcription on the Escherichia coli deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymer ¬ ase for activation of transcription on the E. co/i deoP2 promoter , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>and Nielsen , P.E , ( 1993 ) Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymerase for activation of transcription on the Escherichia coti deoP2 promoler , J Biol Chem 26B : 17471-17477 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Characterization of promoter recognition complexes formed by CRP and CytR for repression and by CRP and RNA polymer-ase for activation of transcription on the Escherichia coli deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>However , the ® nding that the formation of ( CRP ) ± CytR ± promoter complexes resulted 2 in a complete repression of deoP2 transcription ( i.e. to a promoter activity that was considerably lower than that in the absence of cAMP ± CRP ) was unexpected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In this paper , we address how the formation of CytR ± CRP promoter complexes at deoP2 might result in transcription repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>A prediction of the ® rst model is that the formation of CytR ± CRP promoter complexes should not result in repression if transcription is conducted by RNP derivatives lacking a-CTD ( Igarashi and Ishihama , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>DISCUSSION The most significant aspect of the CytR regulon is that a complex pattern of coordinate and differential regulation of a large number of unlinked transcriptional units is provided by interactions between CRP , CytR , and the DNA sequences of the various promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_439">
Role of protein-protein bridging interactions on cooperative assembly of DNA-bound CRP-CytR-CRP complex and regulation of the Escherichia coli CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></label>
<input id="act_similarity_439", type="checkbox">
<div class="ACT_similar">
<li>Chahla , M. , Wooll , J. , Laue , T. M. , Nguyen , N. , and Senear , D. F. ( 2003 ) Role of protein-protein bridging interactions on cooperative assembly of DNA-bound CRP-CytR-CRP complex and regulation of the Escherichia coli CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Therefore , it is tempting to hypothesize that CRP and/or CytR will bind to one or more of the putative CRP targets present in the DNA sequence in front of cytR and that this region is involved in regulation of expression of the repressor gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>Martinussen , J , , Mollegaard , N.E. , Hoist , B. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1988 ) A new version of negative controt , DNA sequences involved in expression and regulation of CytR and cAMP/CRP controtled genes in Escherichia coli In DNA-protein interactions in Transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Cold Spring Harbor Laboratory Press , Cold Spring Harbor , New York Martinussen J , Mollegaard NE , Holst B , Douthwaite SR , ValentinHansen P ( 1989 ) A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_444">
DNA sequences involved in expression and regulation of DeoR , CytR and cAMI ' : CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></label>
<input id="act_similarity_444", type="checkbox">
<div class="ACT_similar">
<li>Valentin-Hansen , P. ( 1985 ) DNA sequences involved In expression and regulation of DeoR , CytR and cAMP-CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li></div>
<li>DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_448">
DNA sequences involved in expression and regulation of DeoR , CytR , and cAMP-CRP controlled genes in Escherichia coli , p. 273-288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></label>
<input id="act_similarity_448", type="checkbox">
<div class="ACT_similar">
<li>DNA sequences involved in expression and regulation of CytR and cAMP/CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li></div>
<li>20 No. 19 pp. 5392 ± 5399 , 2001 Repression of deoP2 in Escherichia coli by C conversion of a transcription activator into a repressor ytR : Minsang Shin , Soim Kang , Seok-Jin Hyun , Nobuyuki Fujita , Akira Ishihama , 1 1 Poul Valentin-Hansen and Hyon E.Choy 2 3 Department of Microbiology , Chonnam University Medical College , 5 Hakdong , Dongku , Gwangju , South Korea 501-714 , Department of 1 Molecular Genetics , National Institutes of Genetics , Mishima , Shizuoka 411 , Japan and Department of Molecular Biology , 2 Odense University , Campusvej 55 , DK-5230 Odense M , Denmark 3 Corresponding author e-mail : hyonchoy@chonnam.ac.kr In the deoP2 promoter of Escherichia coli , a transcription activator , cAMP ± CRP , binds at two sites , centered at ± 41.5 and ± 93.5 from the start site of transcription , while a repressor , CytR , binds to a space between the two cAMP ± CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The CytR regulon in Escherichia coli constitutes a combinatorial regulatory system in which the cyclic AMP receptor protein ( CRP ) functions both as an independent transcriptional-activator and as an ` adaptor ' for a specific repressor , CytR ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ; ( © ) Oxford University Press S0gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>172 , No. 10 The CytR Repressor Antagonizes Cyclic AMP-Cyclic AMP Protein Activation of the deoCp2 Promoter of Escherichia coli K-12 Receptor LOTTE S0GAARD-ANDERSEN , JAN MARTINUSSEN , NIELS ERIK M0LLEGAARD , t STEPHEN R. DOUTHWAITE , AND POUL VALENTIN-HANSEN * Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark Received 19 April 1990/Accepted 7 July 1990 We have investigated the regulation of the Escherichia coli deoCp2 promoter by the CytR repressor and the cyclic AMP ( cAMP ) receptor protein ( CRP ) complexed to cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>173 , No. 17 The Cyclic AMP ( cAMP ) - cAMP Receptor Protein Complex Functions Both as an Activator and as a Corepressor at the tSx-p2 Promoter of Escherichia coli K-12 PETRA GERLACH , ' LOTTE S0GAARD-ANDERSEN ,2 HENRIK PEDERSEN ,2 JAN MARTINUSSEN ,2 POUL VALENTIN-HANSEN ,2 AND ERHARD BREMER ' * Department of Biology , University of Konstanz , P.O. Box 5560 , D-7750 Konstanz , Germany , ' and Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark2 Received 6 March 1991/Accepted 2 July 1991 The tsx-p2 promoter is one of at least seven Escherichia coli promoters that are activated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The EMBO Journal Vol .16 No. 8 pp. 2108 -- 2118 , 1997 Protein-induced ﬁt : the CRP activator protein changes sequence-speciﬁc DNA recognition by CytR repressor , a highly ﬂexible LacI member the Henrik Pedersen1 and Poul Valentin-Hansen2 Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark 1Present address : Department of Chemistry , University of California , Berkeley , Berkeley , CA 94720 , USA 2Corresponding author The CytR repressor and the cAMP receptor protein ( CRP ) bind cooperatively to several promoters in Escherichia coli to repress transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>3812 Biochemistry 2003 , 42 , 3812-3825 Role of Protein-Protein Bridging Interactions on Cooperative Assembly of DNA-Bound CRP-CytR-CRP Complex and Regulation of the Escherichia Coli CytR Regulon † Mayy Chahla , ‡ John Wooll , § Thomas M. Laue , § Nghia Nguyen , ‡ and Donald F. Senear * , ‡ Department of Molecular Biology and Biochemistry , UniVersity of California , IrVine , California 92697 , and Department of Biochemistry , UniVersity of New Hampshire , Durham , New Hampshire 03824 ReceiVed NoVember 5 , 2002 ; ReVised Manuscript ReceiVed February 10 , 2003 ABSTRACT : The unlinked operons that comprise the Escherichia coli CytR regulon are controlled coordinately through interactions between two gene regulatory proteins , the cAMP receptor protein ( CRP ) and the cytidine repressor ( CytR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>1 Abbreviations : bistris , 2 - [ bis ( 2-hydroxyethyl ) amino ] -2 - ( hydroxymethyl ) - propane-1 ,3 - diol ; CRP , Escherichia coli cyclic AMP receptor protein ; CytR , Escherichia coli cytidine repressor protein ; DNase I , deoxyribonuclease I ; RNAP , bacterial RNA polymerase , Tris , tris ( hydroxymethyl ) amino-methane ; EDTA , ethylenediaminetetraacetic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; cAMP , cyclic AMP ; CRP , catabolite activator protein ; cytR , gene encoding CytR ; CytR and DeoR repressor proteins ; A , deletion ; deoR , gene encoding DeoR ; kb , is part of this regulon ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Multiple Specific CytR Binding Sites at the Escherichia coli deoP2 Promoter Mediate Both Cooperative and Competitive Interactions between CytR and cAMP Receptor Protein * Binding of cAMP receptor protein ( CRP ) and CytR mediates both positive and negative control of transcription from Escherichia coli deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In particular , the transcription of genes needed for utilization of nucleosides in Escherichia coli is regulated by a repressor protein , CytR , in concert with the cyclic AMP ( cAMP ) - activated form of cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Much of the expression of nucleoside utilization genes is coordinated by a repressor protein , CytR , acting in concert with the cyclic AMP ( cAMP ) - activated form of the cAMP receptor protein ( CRP ) ( 2 , 9 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>In this system , the global activator protein cAMP receptor protein ( CRP ; also referred to as the catabolite gene activator protein , CAP ) , guides another DNA-binding protein , the CytR repressor , to a number of binding sites that share minimal sequence homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Single amino acid substitutions in the cAMP receptor prote ABSTRACT Promoters in Escherichia coli that are negatively regulated by the CytR repressor are also activated by the cAMP receptor protein ( CRP ) complexed to cAMP ; as a characteristic , these promoters encode tandem binding sites for cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Heterologous Cooperativityin Escherichia coli Promoters in Escherichiacoli that arenegatively regulated by the CytR repressor are alsoactivated by the cAMP receptor protein ( CRP ) complexedto CAMP ; as a characteristic , these promotersencodetwo binding sites for the cAMP * CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>) 6 106 P Activator Complex and the CytR Repressor Bind Co-operatively to the cytRP Promoter in Escherichia coli and CytR Antagonizes the CAMP-CRP-induced DNA bend CAMP-CR Protein Henrik Pedersen ' , Lotte Ssgaard-Andersen ' , Bjrarn Hoist ' , Petra Gerlach ' , Erhard Bremer2 and Poul Valentin-Hansenl-f - Initiation of tranwriyt ~ ion from t ) he cjyftll ' promotw in E , schPric/lio fwli is acativateti k ) y t htt t ~ AMPp ( IRT ' c ~ ~ mplex and negatively regulated by the C'yt K reprwsor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Pedersen H , Valentin-Hansen P. Protein-induced fit : the CRP activator protein changes sequencespecific DNA recognition by the CytR repressor , ahighly flexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144778" target="_blank">12144778</a>]<br></li>
<li>CAMP-CRP activator complex and the CytR repressor protein bind co-operatively to the cytRPpromoter in Escherichia co/i and CytR antagonizes the CAMP-CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_467">
Pedersen , H. , Segaard-Andersen , L. , Hoist , B. , Gerlach , P. , Bremer , E. , and Valentin-Hansen , P. ( 1992 ) cAMP-CRP activator complex and the CytR repressor protein bind cooperatively to the cytRP promoter in Escherichia coil and CytR antagonizes the cAMP-CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_467", type="checkbox">
<div class="ACT_similar">
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , Gerlach , P. , Bremer , E. , and Valentin-Hansen , P. ( 1992 ) cAMP-CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP-CRP-induced DNA bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , Gerlach , P. , Bremer , E. , and Valentin-Hansen , P. ( 1992 ) cAMP -- CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP -- CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Pedersen , H. , Søgaard-Andersen , L. , Holst , B. , Gerlach , P. , Bremer , E. and Valentin-Hansen , P. ( 1992 ) cAMP -- CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP -- CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_471">
cAMP -- CRP activator complex and the Cyt repressor protein bind co-operatively to the cytRP promoter in Escherichia coli and CytR antagonizes the cAMP -- CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18619465" target="_blank">18619465</a>]<br></label>
<input id="act_similarity_471", type="checkbox">
<div class="ACT_similar">
<li>E , , and Valentin-Hansen , P , ( 1992 ) cAMP-CRP activator complex and the CytR repressor protein bind co-operatively to the cytRP promoter in Escherichia cot and GytR antagonizes the cAMP-CRP-induced DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li></div>
<li>1989 ; L. Sogaard-Andersen and P. Valentin-Hansen , in prepar-ation ) , strongly indicate that the structure recognized by CytR includes tandem DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>1989 ; L. Segaard-Andersen and P. Valentin-Hansen , in preparation ) strongly suggests the involvement of specific interactions between cAMP-CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Meibom , K.L. , Kalliipolitis , B.H. , Ebright , R. and Valentin-Hansen , P. ( 2000 ) Identi ® cation of the subunit of cAMP receptor protein ( CRP ) that functionally interacts with CytR in CRP ± CytR-mediated transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>[ 32 ] P. Valentin-Hansen , L. Søgaard-Andersen , H. Pedersen , A ﬂexible partnership : The CytR anti-activator and the cAMP -- CRP activator protein , comrades in transcription control , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_477">
Valentin-Hansen , P. , Sùgaard-Andersen , L. and Pedersen , H. ( 1996 ) A ¯ exible partnership : the CytR anti-activator and the cAMP ± CRP activator protein , comrades in transcription control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></label>
<input id="act_similarity_477", type="checkbox">
<div class="ACT_similar">
<li>Valentin-Hansen , P. , Søgaard-Andersen , L. and Pedersen , H. ( 1996 ) A ﬂexible partnership : the CytR anti-activator and the cAMP -- CRP activator protein , comrades in transcriptional control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Valentin-Hansen , P. , Søgaard-Andersen , L. , and Pedersen , H. ( 1996 ) A ﬂexible partnership : the CytR anti-activator and the cAMP -- CRP activator protein , comrades in transcriptional control Mol Microbiol 20 : 461 -- 466 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_480">
Pedersen , H. , Dall , J. , Dandanell , G. and Valentin-Hansen , P. ( 1995 ) Gene regulatory modules in Escherichia coli : nucleoprotein complexes formed by cAMP -- CRP and CytR at the nupG promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_480", type="checkbox">
<div class="ACT_similar">
<li>Pedersen , H. , Dall , J. , Dandanell , G. , and Valentin-Hansen , P. ( 1995 ) Gene-regulatory modules in Escherichia coli : nucleoprotein complexes formed by cAMP -- CRP and CytR at the nupG promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_482">
, Valentin-Hansen , P. , and Søgaard-Andersen , L. ( 1997 ) Design of CytR regulated , cAMP -- CRP dependent class II promoters in Escherichia coli : RNA polymerase-promoter interactions modulate the efﬁciency of CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></label>
<input id="act_similarity_482", type="checkbox">
<div class="ACT_similar">
<li>Kristensen , H.H. , Valentin-Hansen , P. and Sùgaard-Andersen , L. ( 1997 ) Design of CytR regulated , cAMP ± CRP dependent class II promoters in Escherichia coli : RNA polymerase ± promoter interactions modulate the ef ® ciency of CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Kristensen , H. H. , Valentin-Hansen , P. , and Sogaard-Andersen , L. ( 1997 ) Design of CytR regulated , cAMP-CRP dependent class II promoters in Escherichia coli : RNA polymerase-promoter interactions modulate the efficiency of CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>EMBCJ5 : 2015-2021 , Valentin-Hansen , P. , Hoist , B. , Josephsen , J , , Hammer , K , , and Albrechtsen , B , ( 1989 ) CRP/cAMP - and CytR - regulated promoters in Escherichia co / / K12 : the cddpromoter , MolMtcrobtoi 3 : 1385-1390 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Valentin-Hansen , P. , Holst , B. , Josephsen , J. , Hammer , K. , and Albrechtsen , B. ( 1989 ) CRP/cAMP - and CytR regulated promoters in Escherichia coli K12 : the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>It was shown recently that to achieve maximal repression , the CytR repressor has to interact with DNA and the cAMP-CRP complex to speciﬁcally repress the deoP2 promoter ( Sogaard-Andersen & Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>For example , to-bind to and achieve repression of the deoP2 , cdd and udp promoters , the CytR repressor has to interact with cAMP -- CRP ( Holst et al. , 1992 ; Sogaard-Andersen and Valentin-Hansen , 1993 ; Brikun et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>Rasmussen , P.B. , Søgaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Identiﬁcation of the nucleotide sequence recognized by the cAMP -- CRP dependent CytR repressor protein in the deoP2 promoter in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Gene 51:255 -267 Sogaard-Andersen L , Martinussen J , Mollegaard NE , Douthwaite SR , Valentin-Hansen P ( 1990a ) The CytR repressor antagonizes cAMP-CRP activation in the deoP2 promoter of Escherichia coli-K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Sagaard-Andersen , L. , Martinussen , J. , Mallegaard , N.E. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1990a ) The CytR repressor antagonizes cAMP-CRP activation in the deoP2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_492">
Søgaard-Andersen , L. , Pedersen , H. , Holst , B. and Valentin-Hansen , P. ( 1991b ) A novel function of the cAMP -- CRP complex in Escherichia coli : cAMP -- CRP functions as an adaptor for the CytR repressor in the deo operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_492", type="checkbox">
<div class="ACT_similar">
<li>Sùgaard-Andersen , L. , Pedersen , H. , Holst , B. and Valentin-Hansen , P. ( 1991b ) A novel function of the cAMP ± CRP complex in Escherichia coli : cAMP ± CRP functions as an adaptor for the CytR repressor in the deo operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li></div>
<li>Moreover , it was reported that CytR repressor binds to deop2 of the deo gene as a large complex sandwiched by cAMP-CRP complexes [ Søgaard-Andersen and Valentin-Hansen , 1993 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_495">
, Valentin-Hansen , P. and Søgaard-Andersen , L. ( 1996 ) CytR/cAMP -- CRP nucleoprotein formation in E.coli : the CytR repressor binds its operator as a stable dimer in a ternary complex with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="act_similarity_495", type="checkbox">
<div class="ACT_similar">
<li>Kristensen , P. Valentin-Hansen , L. Søgaard-Andersen , CytR/cAMP -- CRP nucleoprotein formation in E. coli : The CytR repressor binds its operator as a stable dimer in a ternary complex with cAMP -- CRP , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li></div>
<li>At all CytR operators CytR binding is absolutely dependent on cAMP/CAP binding to two sites surrounding the CytR binding site and in this case it is primarily protein -- protein interactions between CAP and the CytR repressor that permit CytR binding to DNA ( Sogaard-Andersen & Valentin-Hansen , 1993 , and references cited within ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_498">
[ 28 ] L. Søgaard-Andersen , P. Valentin-Hansen , Protein -- protein interactions in gene regulation : The cAMP -- CRP complex sets the speciﬁcity of a second DNA-binding protein , the CytR repressor , Cell 75 ( 1993 ) 557 -- 568 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></label>
<input id="act_similarity_498", type="checkbox">
<div class="ACT_similar">
<li>Søgaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Protein -- protein interactions in gene regulation : the cAMP -- CRP complex sets the speciﬁcity of a second DNA-binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Sùgaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Protein ± protein interactions in gene regulation : the cAMP ± CRP complex sets the speci ® city of a second DNA-binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Sogaard-Andersen , L. , and Valentin-Hansen , P. ( 1993 ) Protein ± protein interactions in gene regulation : the cAMP ± CRP complex sets the speci ® city of a second DNA-binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Sogaard-Andersen , L. and Valentin-Hansen , P. ( 1993 ) Protein -- protein interactions in gene regulation : the cAMP -- CRP complex sets the speciﬁcity for a second DNA binding protein , the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li></div>
<li>As the cooperative binding of cAMP-CRP and CytR relies on both protein-DNA and protein-protein interactions and is dramatically reduced by mutations that affect these parameters ( Sogaard-Andersen al. , 1990a , b , et 1991a ; S0gaard-Andersen and Valentin-Hansen , 1991 ) , this regulatory difference might be explained by the relative affinity of cAMP-CRP for its binding sites in the two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Protein-protein interactions between CytR and cAMP-CRP mediate this co-operative binding ( S ~ gaard-Andersen et aL , 1991a ; Pedersen et aL , 1991 ; S ~ gaard-Andersen and Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_505">
The co-operative binding of CytR and CRP relies on direct protein -- protein interactions ( Søgaard-Andersen and Valentin-Hansen , 1993 ; Kallipolitis et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></label>
<input id="act_similarity_505", type="checkbox">
<div class="ACT_similar">
<li>The co-operative binding of CytR and CRP relies on direct protein -- protein interactions ( Søgaard-Andersen and Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>Mol Gen Genet ( 1991 ) 231:76 -80 OIGG © Springer-Verlag 1991 Restored DNA-binding of the cAMP-CRP activator complex reestablishes negative regulation by the CytR repressor in the deoP2 promoter in Escherichia coli Lotte Sogaard-Andersen and Poul Valcntin-Hanscn Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark ReceivedMay 13 , 1991 July 25 , 1991 1990a , b ) whereas cAMP-CRP activation only requires an intact CRP-1 site ( Valentin-Hansen et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>1 1 no. 10 pp. 3635 - 3643 , 1992 The cAMP - CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP - CRP activator complex are involved in combinatorial regulation of the cdd promoter B.Holst , L.Sogaard-Andersen , H.Pedersen and P.Valentin-Hansen1 Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark Communicated by H.Buc ` Corresponding author Transcription initiation at CytR regulated promoters in Escherichia coli is controlled by a combinatorial regulatory system in which the cAMP receptor protein ( CRP ) functions as both an activator and a co-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_509">
Holst , B. , Søgaard-Andersen , L. , Pedersen , H. , and ValentinHansen , P. ( 1992 ) The cAMP -- CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP -- CRP activator complex are involved in combinational regulation of the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></label>
<input id="act_similarity_509", type="checkbox">
<div class="ACT_similar">
<li>Holst , B. , Søgaard-Andersen , L. , Pedersen , H. and Valentin-Hansen , P. ( 1992 ) The cAMP -- CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP -- CRP activator complex are involved in combinatorial regulation of the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Holst , B. , Sogaard-Andersen , L. , Pedersen , H. and Valentin-Hansen , P. ( 1992 ) The cAMP -- CRP/CytR nucleoprotein complex in Escherichia coli : two pairs of closely linked binding sites for the cAMP -- CRP activator complex are involved in combinatorial regulation of the cdd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li></div>
<li>Initiation of transcription from deoP2 is negatively controlled by two repressors , DeoR and CytR , and is also dependent on the cAMP/CRP complex Location of transcription terminators in the 3 ' - end of the deo operon In order to locate termination signals , we inserted restriction fragments from the 3 ' - end of the deo operon between the deoP2 promoter and the lacZ gene of pJEL144 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3299264" target="_blank">3299264</a>]<br></li>
<li>edtsx mRNA are indicated by arrows Genetic experiments have previously indicated that tsx expression is mediated by two differently regulated promoters : a minor promoter , PI , which is negatively controlled by DeoR and a main promoter , P2 , which is repressi-ble by CytR and activated by the cAMP/CRP complex ( Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Taken together , the results of the S 1 mapping experiment mad the regulatory features of tsx gene expression ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) strongly suggest that transcription initiating from ~ he weak tsx P1 promoter is repressible by DeoR , whereas the activity of the main tsx promoter , 1 ' 2 , is stimulated by the cAMP/CRP complex and negatively controlled by the CytR repre ~ sor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_515">
Nørregaard-Madsen , M. , Mygind , B. , Pedersen , R. , ValentinHansen , P. , and Søgaard-Andersen , L. ( 1994 ) The gene encoding the periplasmic cyclophilin homologue , PPIase A , in Escherichia coli , is expressed from four promoters , three of which are activated by the cAMP -- CRP complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></label>
<input id="act_similarity_515", type="checkbox">
<div class="ACT_similar">
<li>Nørregaard-Madsen , M , Mygind , B. , Pedersen , R. , Valentin-Hansen , P. and Søgaard-Andersen , L. ( 1994 ) The gene encoding the periplasmic cyclophilin homologue , PPIaseA , in Escherichia coli , is expressed from four promoters , three of which are activated by the cAMP -- CRP complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Nørregaard-Madsen M , Mygind B , Pedersen R , Valentin-Hansen P , Søgaard-Andersen L ( 1994 ) The gene encoding the periplasmic cyclophilin homologue , PPIase A , in Escherichia coli , is expressed from four promoters , three of which are activated by the cAMP-CRP complex and negatively regulated by the CytR References 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li></div>
<li>The three most upstream rot promoters are activated by the global gene regulatory cAMP-CRP complex and negatively regulated by the CytR repressor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The gene encoding the periplasmic cyclophilin homoloque , PPIase A , in Esche-richia coli , is expressed from four promoters , three of which are activated by the cAMP-CRP complex and negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9159398" target="_blank">9159398</a>]<br></li>
<li>Initiations from P1 have been shown to be controlled negatively by the DeoR repressor , whereas initiations from P2 are dependent on the cAMP/CRP complex and are negatively controlled by both the CytR and DeoR repressor ( Valentin-Hansen , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>Moreover , the expression of udp and cdd is positively controlled by the CRP-cAMP complex and negatively controlled by the CytR repressor [ 37 -- 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>INTRODUCTION The structural genes encoding the ~ ystems for nucleoside uptake and metabolism in E. coli are organized in a regulon that is controlled positively by the cAMP/CRP activator complex and negatively by the DeoR and CytR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_523">
Kallipolitis , B.H. and Valentin-Hansen , P. ( 1998 ) Transcription of rpoH , encoding the Escherichia coli heat-shock regulator s32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="act_similarity_523", type="checkbox">
<div class="ACT_similar">
<li>Transcription of rpoH , encoding the Escherichia coli heat-shock regulator 32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952953" target="_blank">16952953</a>]<br></li></div>
<li>The cAMP -- CRP complex and the CytR anti-activator bind co-operatively to the promoter region forming a repression complex that overlaps the sE-dependent P3 promoter and the s70-dependent P4 and P5 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The putative repression complex formed by CytR and CRP overlaps promoter elements for P3 , P4 and P5 , suggesting that CytR may affect these promoters in particular ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Therefore , the cAMP-CRP complex strengthens binding of CytR to its operators up to 1000-fold ( Pedersen et aL , 1991 ) , increasing the affinity of the repressor for the promoters to a level adequate for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Thus , CytR does not regulate transcription from deoP2 in cells depleted for the cAMP-CRP complex ( Acya or Acrp strains ) , even when the level of the repressor in the cells is high ( Sogaard-Andersen et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>H Discussion Recent studies have indicated that repression of the deoP2 promoter by the CytR repressor requires the cAMP-CRP complex and that there is a coupling between cAMP-CRP binding to tandem CRP targets and the interaction of the CytR repressor ( Sogaard-Andersen et aL , 1990a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In CytR regulated promoters , including the udpP promoter , the cAMP -- CRP complex acts as a transcriptional-activator as well as a co-repressor for the CytR protein [ 5,23,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>These data indicate that the CytR repressor relies on the presence of the cAMP-CRP activator complex to regulate tsr-p2 promoter activity and that the formation of an active repression complex requires the combined interactions of cAMP-CRP and CytR at tsX-p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Cooperative binding of the CytR repressor and the extent in the presence of cytidine that the promoter no cAMP-CRP activator complex longer responds to the CytR repressor in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_533">
To this end , we titrated in parallel the wild-type and mutant promoters with the cAMP-CRP complex and the CytR repressor in the presence of 50 pRM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></label>
<input id="act_similarity_533", type="checkbox">
<div class="ACT_similar">
<li>To this end we titrated , in parallel , the wild-type and mutant promoters with the cAMP-CRP complex in the absence or the presence of the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li></div>
<li>Next , we performed footprinting experiments in the presence of both the CytR repressor and the cAMP-CRP activator complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Combined DNase I footprint of the wild-type tsx-p2 promoter in the presence of both the CytR repressor and the cAMP-CRP activator complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The role of the tandem CRP sites in negative and positive control has been studied most intensively in the deoP2 promoter , in vt/hich transcriptional initiation is modulated 10-foid by CytR and 30-fold by the cAMP-CRP complex -LCB- Valentin-Hansen , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Using a combination of genetic , biochemical , and gene fusion techniques on single-copy plasmids , we have analyzed one such promoter , deoCp2 , to define the DNA sequences involved in positive and negative control by the cAMP-CRP complex and the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In deoCp1 , promoter control is mediated by the DeoR repressor , whereas deoCp2 is subject to negative regulation by the DeoR and CytR repressors and activation by the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex ( 29 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>The weak tsx-pl promoter is negatively regulated by the DeoR repressor , whereas the main promoter , tSx-p2 , is subjected to both negative control by CytR and activation by the cAMP-CRP complex ( 3 , 4 , 17 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The minor tsx-p1 promoter is negatively controlled by the DeoR repressor , whereas the main promoter , tsX-p2 , is subjected to positive regulation by the cAMP-CRP activator complex and negative control by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>It seems fairly evident that this difference , accounting for only about a 3-fold effect on affinity , is insufficient to account for induction simply on the basis of reduced CytO occupancy Our cddP data demonstrate yet a third pattern of effects on cooperativity , in which the pairwise interaction between CytR and CRP bound to CRP1 is completely unaffected while the pairwise interaction between CytR and CRP bound to CRP1 is greatly reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The total effect of cooperativity with both CRP sites on CytR affinity is 100-1000-fold in different promoters and conditions ( cf. Table I and Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The effective affinity of CytR for CytO ( in the presence of CRP ) is only 1.2 times higher than cytidine-liganded CytR , yet induction is fairly robust , accounting for a roughly 20-fold increase in transcription according to reporter gene assays ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Under conditions in which CytR is inactive ( i.e. in the presence of cytidine ) , CRP interacts with RNP to form transcriptionally active CRP ± RNP ± promoter complexes , whereas under conditions in which CytR is active ( in the absence of cytidine ) , two CRPs interact with CytR to form transcriptionally inactive ( CRP ) 2 ± CytR ± deoP2 complexes ( Sùgaard-Andersen et al. , 1990a , b , 1991a , b ; Sùgaard-Andersen and Valentin-Hansen , 1993 ; Meibom et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>CAMP-CRP activation takes place from CRP-1 ( Valentin-Hansen , 1985 ) , whereas CytR regulation is dependent onthe presence of both CRP targets awsell as the distance and the sequence between these targets ( Sld-gaard-Andersen et al. , 1990a , 1990b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>The CytR anti-activator and CRP act in concert to repress transcription from genes encoding transport proteins and enzymes necessary for uptake and catabolism of ribonucleosides and deoxyribonucleosides ( reviewed by Hammer-Jespersen , 1983 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Transcription of the rpoH gene from these promoters is regulated by CRP -- cAMP , CytR ( Ramirez-Santos et al. , 2001 ; Kallipolitis & Valentin-Hansen , 1998 ) and DnaA ( Wang & Kaguni , 1989b ; Messer & Weigel , 1997 ) proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>The deo operon is controlled by two proteins , CytR and DeoR , in addition to CRP-cAMP and is expressed from two promoters ( ValentinHansen et al. 1978 ; Valentin-Hansen 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The control of deo operon is complex involving two other repressors , DeoR and CytR as well as CRP-cAMP ( Valentin-Hansen et al. 1978 ; Valentin-Hansen 1982 ; Short and Singer 1984 ; Shin et al. 2001 ) , and the absence of these proteins in the assay could account for the higher than expected amount of ModE required for this DNA mobility shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>In the well-characterized case of the CytR regulon , CRP expresses positive or negative inﬂuences in collaboration with CytR ( Pedersen & Valentin-Hansen , 1997 ; Holt & Senear , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Discussion Previous results have shown that CytR can interact and bind co-operatively with CRP from a number of different settings of operator half-sites ( Valentin-Hansen et al. , 1996 ; Pedersen and Valentin-Hansen , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Recently , it was shown by DNase I footprinting in vitro that therepression complex most likely consists of one molecule CytR and two molecules of CRP , i.e. that the activatorprotein CRP is also required for repression ( S ~ gaard-Andersen eatl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>We have previously shown that this band corresponds to the repression complex formed at deof2 in which CytR is sandwiched between tandem CAMP-CRP complexes ( Pedersen et al. , 1991 ; Segaard-Andersen et al. , 1991a ; see also the footprinting experiments below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CRPa-CytR-Promotor Complex-It has previously been shown that both the CRP-1 and tC heRP-2 sites arneecessary for CytR repression of the deoP2 promoter , and DNase I footprintinghas indicated that bothsitesare occupiedby CRP in the repression complex with CytR bound in between the CRP sites ( Sbgaard-Andersen et al. , 199Ob ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>The CRP2 -- CytR -- DNA repression complex covers ~ 80 bp on one face of the DNA helix , and is held together by CRP -- DNA , CytR -- DNA and CRP -- CytR interactions ( Pedersen et al. , 1991 ; Søgaard-Andersen et al. , 1991a , b ; Rasmussen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The second repressor of the system , CytR , binds in concert with the CAMP-CRP complex in a highly cooperative fashion , forming a repression complex in which the DNA is wrapped around CytR and one or two molecules of CAMP-CRP ( Pedersen et al. , 1991 , 1992 ; Gerlach et al. , 1991 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CytR binds to its cognate promoters in concert with cAMP-CRP in a highly co-operative fashion torming repression complex that generally consists of two mol-ecules of cAMP-CRP separated by 52 or 53 bp and CytR ioound in between ( Sogaard-Andersen et ai , 1991b ; Gerlach etai , 1991 ; Holsi etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Nielsens From the Research Center for Medical Biotechnology , Department of Biochemistry B , The Panum Institute , University of Copenhagen , Blegdamsuej3 , DK-2200 Copenhagen N and the $ Department of Molecular Biology , University of Odense , Campusuej55 , DK-5230 Odense , Denmark Thestructureofthe CAMP-CRP-CytR repression observed with CytR-regulated promoters ( SBgaard-Andersen complex and the cAMP CRP-RNA polymerase initia - et al. , 1991a , 1991b ; Gerlach et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>The inducer of CytR , cytidine : perturbs the cooperative t ) KA binding of CAMP-CRP and CytR by disrupting the protein-protein contacts between t , hr two proteins ( Pedersen et al. , 1991 ; Gertach rf nl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>In this context , it should be noted that the inducers of CytR ( cytidine and adenosine ) exert their effect on repression by interrupting the protein-protein interactions between CytR and CAMP-CRP ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CRP and CytR were purified as described ( Ghosaini et al. , 1988 ; Petersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR and CRP proteins were puriﬁed as described ( Ghosaini et al. , 1988 ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_564">
Proteins CytR and CRP proteins were purified according to Pedersen et al. ( 1991 ) and Ghosaini et aL ( 1988 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_564", type="checkbox">
<div class="ACT_similar">
<li>Proteins CytR and CRP proteins were puriﬁed according to Pedersen et al. ( 1991 ) and Ghosaini et al. ( 1988 ) respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>In the presence of CAMP-CRP , however , binding of CytR is enhanced more than lOOO-fold ( Holst et al. , 1992 ; Gerlath et al. , 1991 ; Pedersen et al. , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Binding studies have shown that CytR binds to deoP2 with a dissociation constant of approximately 1Om7M and that CAMP-CRP stimulates CytR binding approximately lOOO-fold ( Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The synergy in the system is prominent ; depending on which promoter is examined , the cAMP -- CRP activator complex strengthens binding of CytR from 100 - to several thousandfold in vitro ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 , 1992 , 1995 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CytR and CRP turn off transcription by binding in a highly co-operative fashion to speciﬁc targets in the promoter regions ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ; 1992 ; 1995 ; Søgaard-Andersen et al. , 1991 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>This model also offers a plausible explanation for the recent findings that CytR does not regulate transcription from deoP2 in Icya or Acrp strains , although transcription occurs from the same site in deoP2 in the absence and presence of cAMP-CRP ( L. Sogaard-Andersen et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Thus , in the presence of both cAMP-CRP and CytR , a nucleoprotein complex is formed that most probably contains cAMP-CRP at each CRP site and CytR bound in the intervening region ( Sogaard-Ander-sen et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>The open bar indicates the operator region protected by CytR in the presence of cAMP-CRP ( Sogaard-Andersen et al. 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>However , a direct analysis of the effect of mutations in CRP-1 on CytR regulation is complicated as such mutations also affect promoter structure and cAMP-CRP activation ( L. Sogaard-Andersen et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Firstly , cAMP-CRP activation of deoP2 is mediated by the CRP-1 site , whereas proper regulation of this promoter by CytR requires tandem intact CRP binding sites ( Martinussen et al. 1989 ; Sogaard-Andersen et al. 1990a , b ; Valentin-Hansen 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In CytR regulated promoters , cAMP-CRP constitutes an essential part of the repression apparatus and the available data suggest that two appropriately positioned CRP binding sites are a prerequisite for negative regulation by CytR ( S0gaard-Andersen et al. , 1990a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Based on these observations , it has been suggested that CytR and RNP are rivals that compete for binding with CRP to deoP2 , and that CytR acts as an anti-activator , nullifying the activation by cAMP ± CRP ( Sùgaard-Andersen et al. , 1990a , b ; Mùllegaard et al. , 1993 ; Kristensen et al. , 1997 ; Meibom et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Repression of deoP2 Repression of deoP2 by CytR was only observed in the presence of cAMP ± CRP , in agreement with the in vivo observation that CytR binding requires cAMP ± CRP ( Sùgaard-Andersen et al. , 1990a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In vitro , CRP-3 is only occupied in the presence of CytR and is located at a position relative to CRP-1 that has been shown to be optimal for the cooperative binding of cAMP-CRP and CytR in deoP2 ( S0gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In vitro , CytR exhibits a low affinity for the sequence located between the two CRP sites ; however , its binding to deoP2 can be stimulated as much as 1000-fold by cAMP-CRP ( Sogaard-Andersen et al. , 1991b ; Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>This ability of cAMP - CRP to stimulate the binding of CytR cooperatively requires a set of amino acids that are located in a specific region on the surface of cAMP-CRP ( Pedersen et al. , 1991 ; Sogaard-Andersen et al. , 1991a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Whereas -7 kcal/mol is a large free energy change as compared to the heterologous cooperativity exhibited when full-length CytR binds to CRP - deoP2 ( ∆ G ≈ -2.5 to -3 kcal/mol ( 9 , 11 ) ) , 2 it indicates a very low affinity for analysis by gel-shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Second , it may also be significant that the three-way cooperativity , i.e. with both CRP sites filled and CytR bound in between , equals the additive sum of free energy changes for these pairwise interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This suggests that the separate interactions between CytR and each CRP dimer are additive , yielding a connection free energy of zero for bridging .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>On the other hand , the free energy changes for cooperativity between CytR and CRP are independent of the particular model and so appear to provide accurate descriptions of these interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The free energy change we obtain for this interaction , ∆ Gobridge ) -7.8 kcal/mol , is essentially double what we estimate for solution assembly of CytR and CRP to form a heterotetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Configuration 5 pertains to the complex formed by association of CytR Core domain in which ∆ G represents the free energy change for bridging CytR-CRP bridge interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The results indicate only weak association between free CytR and CRP to form a hetero-tetrameric complex in solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>On the basis of ∆ Gbridge ≈ -8 kcal/mol , ∆ G of the order -3 to -4 kcal/mol might be anticipated for a 1:1 association between CytR and CRP to form a hetero-tetrameric complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Second , while mutant deoP2 with only CRP1 functional ( i.e. CRP22 ) displays nearly the same extent of CRP activation ( presumably class II-mediated ) as wild type , it is not effectively repressed by CytR ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At the high CRP ( cAMP ) 1 concentration required to significantly load CRP1 , the net effect of cooperativity and competition produces only a slight difference in CytR occupancy whether deoP2 is wild type or CRP22 ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>When 0.2 mM cytidine-liganded CytR was subsequently added , CRP affinity for CRP1 and CRP2 dropped by over 2-fold , giving DGCRP1 5 211.3 6 0.1 kcal/mol and DGCRP2 5 212.9 6 0.2 kcal/mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>of Operator site experimentsb CytR at CRP2 CytR CRP1 CRP2 213.5 6 0.3 -- All CRP None 4 -- 212.0 6 0.1 All CytR None 13 210.7 6 0.4 -- WT/CRP12 CytR None 8 -- CRP22 CytR None 3 -- Wild type CRP CytR 2 213.9 6 0.1 -- 213.5 6 0.1 CRP12 CRP CytR 2 213.7 6 0.4 -- -- CRP22 CRP CytR 2 -- -- 212.9 6 0.1 Wild type CytR CRP 5 -- 212.8 6 0.2 -- CRP12 CytR CRP 3 -- 212.4 6 0.3 -- CRP22 CytR CRP 4 -- 212.7 6 0.2 -- Wild type CytR CRP , cytidine 3 -- 212.2 6 0.4 -- CRP12 CytR CRP , cytidine 2 -- 211.9 6 0.3 -- CRP22 CytR CRP , cytidine 2 -- 211.2 6 0.1 -- a Effector concentrations : CRP , 0.1 mM ( total dimer ) ; cAMP , 150 mM ( present in all experiments ) ; CytR , 75 nM ( dimer ) ; cytidine , 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>of expts b c CytR at CRP2 CytR at CRP1 CytO CRP2 CRP1 wild type CRP1 CRP2 CRP none 7 - ¬ -12.4 ( 0.4 -13.8 ( 0.3 wild type CRP1 CRP2 CytR none 10 -- wild type CRP CytR 2 CRP1 CRP CytR 2-CRP2 CRP CytR 3-wild type CytR CRP 4 CRP1 CytR CRP 2-CRP2 CytR CRP 3-wild type CRP CytR , cytidine 3 CRP1 CRP CytR , cytidine 2-CRP2 CRP CytR , cytidine 2-wild type CytR CRP , cytidine 5 -12.9 ( 0.4 CRP1 CytR CRP , cytidine 2 - -12.3 ( 0.2 CRP2 CytR CRP , cytidine 2 - -12.8 ( 0.1 a Free energy changes for saturation of cdd operators with either CRP ( cAMP ) 1 or CytR in the presence or absence of effector ligands are as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>-9.6 ( 0.5 -9.7 ( 0.2 -11.5 ( 0.2 -12.8 ( 0.1 -13.3 ( 0.1 -13.7 ( 0.2 -14.1 ( 0.1 -13.2 ( 0.3 -13.4 ( 0.4 -13.3 ( 0.3 -12.2 ( 0.3 -12.1 ( 0.0 -14.3 ( 0.3 -14.5 ( 0.03 Table 2 : Apparent Cooperativity As Assessed by Differences in Indivia dual Site Loading Free Energy Changes Described in Table 1 a Table 3 : Configurations and Free Energy States for CRP and CytR Binding to the cdd Promoter binding sites a CRP1 free energy contributions free energy state b CRP2 CytR at CRP1 CytO CytR at CRP2 O O O 1 O O reference state ΔGs ,1 2 O O O CRP O ΔG1 ΔGs ,2 3 CRP O O O O ΔG2 ΔGs ,3 4 O O CytR O O ΔG3 ΔGs ,4 5 O O O O CytR ΔG4 ΔGs ,5 6 O CytR O O O ΔG5 ΔGs ,6 7 CRP O O CRP O ΔG1 þ ΔG2 ΔGs ,7 8 O O CytR CRP O ΔG1 þ ΔG3 þ ΔG13 ΔGs ,8 9 O CytR O CRP O ΔG1 þ ΔG5 ΔGs ,9 10 CRP O CytR O O ΔG2 þ ΔG3 þ ΔG23 ΔGs ,10 11 CRP O O O CytR ΔG2 þ ΔG4 ΔGs ,11 12 O O CytR O CytR ΔG3 þ ΔG4 ΔGs ,12 13 O CytR CytR O O ΔG3 þ ΔG5 ΔGs ,13 14 O CytR O O CytR ΔG4 þ ΔG5 ΔGs ,14 15 CRP O CytR CRP O ΔG1 þ ΔG2 þ ΔG3 þ ΔG123 ΔGs ,15 16 CRP O CytR O CytR ΔG2 þ ΔG3 þ ΔG4 þ ΔG23 ΔGs ,16 17 O CytR CytR O CytR ΔG3 þ ΔG4 þ ΔG5 ΔGs ,17 18 O CytR CytR CRP O ΔG1 þ ΔG3 þ ΔG5 þ ΔG13 ΔGs ,18 a Binding sites are listed in columns 2-6 corresponding to the order , from promoter distal to promoter proximal , on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Table 5 : Global Analysis of Individual Site Titration Data Including Those with Cytidine Present As Represented in Table 1 according to the Model Described in Table 3 parameter value a - þ-0 .2 ( 0.2 , 0.3 ) þ1 .1 ( 0.4 þ0 .2 ( 0.4 ΔΔG13 ΔΔG23 ΔΔG123 s b 0.054 a Consequently , the subsequent binding of CRP to CRP2 should contribute only a relatively modest increment to the overall activation , and even this would be muted to a substantial extent as a result of concomitant recruitment of CytR to CytO that results from the strong three-way cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Cooperative and Competitive Interactions of CytR -- The initial indication and one functional consequence of these interactions is that while saturation of CRP1 and CRP2 greatly increases the apparent affinity of CytR for its putative operator located between these , saturation by CytR has a much lesser effect on the apparent affinity of CRP for CRP1 and CRP2 ( Table I ) ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This effect occurs first at CRP2 due to the higher affinity of CytR for its competing site ( s ) , thus helping to facilitate the switch from class I to class II CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this complex , CytR binds to an operator site ( usually referred to as CytO ) that is flanked by tandem CRP operators , CRP1 and CRP2 , as are found in most CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>A second surprise uncovered by our analysis of udpP is that the inducer cytidine does not eliminate all CytR-CRP cooperativity , as it does at deoP2 , but instead eliminates only the pairwise interaction between CRP bound to CRP1 and CytR bound to CytO ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Whereas the effect of cytidine on pairwise cooperativity between CytR and CRP bound to CRP2 is modest , similar to that for wild type udp , pairwise cooperativity between CytR and CRP bound to CRP1 is nearly eliminated , similar to that for deoP2 ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Second , the free energy changes for pairwise cooperativity between CytR and CRP are the same whether CRP is bound to CRP1 or to CRP2 and are the same for udp as for deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In activation , one consequence of the competition between CytR and CRP is to facilitate configurations in which CRP is bound either to CRP1 or to CRP2 , but not to both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast , it has a negligible effect on pairwise cooperativity between CytR and CRP bound to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This model assumes no significant association of CytR Core domain when CRP is bound to either , but not both , CRP1 or CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The pairwise interaction between CytR and CRP bound to CRP1 is greatly reduced , but because CRP2 is the weaker site and as such is rarely occupied by itself , the reduction in pairwise CytR-CRP2 has a negligible effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This suggests that CytR bound to the intervening site simultaneously contacts CRP bound to both CRP1 and CRP2 and with no influence of either pairwise interaction on the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At udpP , however , the pairwise interaction between CytR and CRP bound to CRP2 is essentially unaffected ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>'' How can CytR can make the same interaction with CRP bound to CRP1 and CRP2 given spacers of 15 and 8 bps ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At cddP , cooperativity between CytR and CRP bound to CRP1 is largely retained , while cooperativity between CytR and CRP bound to CRP2 is largely eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>An important result of our study is that whereas cytidine is an effector of CytR interactions with CRP bound to udp-CRP1 , cytidine is not an effector of CytR interactions with CRP bound to udp-CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Thus , it appears that CytR not only occludes interactions involving RNAP RCTD as a result of bridging interactions but also modulates the activity of CRP1-bound CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Since CytR and CRP bind in a cooperative manner , the separate interactions between CytR and CytO , and between CytR and both CRP1-bound and CRP2-bound CRP , must combine to yield a net free energy change that is more favorable than for any of these individual interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Interaction of CytR with these sites provides a complex pattern of both competitive and cooperative interactions between CytR and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However as CRP ( cAMP ) 1 subsequently binds CRP1 , the additional pairwise cooperative interaction should now cause CytR to compete effectively for CRP2-RNAP interactions and so tend to shut down CRP2-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At high CytR concentration , favorable interactions with CytR would increase the apparent affinity of CRP ( cAMP ) 1 for CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>By contrast , at high CRP concentration , favorable interactions with CRP bound to CRP1 and CRP2 would increase apparent affinity of CytR for its intervening site , and this would not be offset by unfavorable competitive interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The current view is that transcription from these promoters involves CRP binding to CRP1 and that repression involves the binding of CytR protein to CRPs that are , in turn , bound to both CRP1 and CRP2 in the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>These results also a Free energy changes for intrinsic binding of CRP to operator sites CRP1 and CRP2 of the E. coli udp and deoP2 promoters and for subsequent binding of CytR Core domain to form the DNA-bound CRP-Core domain-CRP bridged complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>If direct protein-protein interactions provide the driving force for cooperativity , as has been presumed ( 23 58 ) , then CytR bound to the intervening site must make substantially identical interactions with CRP , whether bound to CRP1 or to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR Core domain associates when CRP is bound to both CRP1 and CRP2 as a result of favorably oriented protein-protein contacts , despite no interaction between Core domain and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Even with its DNA-binding domain deleted , CytR will associate to form a protein bridge between CRP dimers bound to CRP1 and to CRP2 , thus indicating that direct interaction between the DNA-bound proteins is the mechanism for the cooperativity ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>As a result of this cooperativity CytR is recruited to CytO when CRP binds to the flanking sites , CRP1 and CRP2 , to yield a three-protein , DNA-bound complex , CRP-CytR-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Additionally , while cytidine binding to CytR nearly eliminates the pairwise interactions between CytR and CRP bound at CRP1 , it has little effect on pairwise cooperativity between CytR and CRP bound at CRP2 or as a consequence on the overall cooperativity of the three-protein complex in which CRP is bound to both sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Whereas at deoP2 cytidine binding to CytR essentially eliminates cooperativity in the three-protein complex , at udpP it serves to eliminate cooperativity only between CytR and CRP bound to CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Repression by the cooperative CRP-CytR-CRP complex results from competition between CytR and RNAP , both of which are recruited by CRP to-bind to the DNA sequence flanked by CRP1 and CRP2 ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>A critical component of the regulation is that CytR and CRP bind DNA cooperatively , meaning that CRP binding to CRP1 and CRP2 enhanced the affinity of CytR for CytO and vice versa ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In addition , it is only by including competition between CytR and CRP for binding to CRP2 in the global analysis of the footprint data that we can accurately account for the experimentally determined loading free energy changes ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Therefore , the CytR-mediated differences in loading free energy changes for CRP ( cAMP ) 1 binding to CRP1 and CRP2 are summed to yield DG123 5 ( DGCytR CRP CRP1 2 DGCRP1 ) 1 ( DGCytR2 2 DGCRP2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Model for Cooperative and Competitive Binding of CytR and cAMP-CRP -- To determine whether such interactions of CytR with DNA sequences overlapping and thereby occluding or competing with CRP binding to CRP1 and CRP2 can account quantitatively for both the loading free energy changes in Table I and apparent free energy changes in Fig. 6 , two models that incorporate two additional CytR binding sites and which constitute opposing possibilities were evaluated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Analysis of Cooperative and Competitive Binding of CytR and CRP -- Cooperative interactions between CytR and CRP and competition between CytR and CRP for binding to CRP2 were evaluated by considering the thermodynamic cycles for simultaneous binding of both proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To assess these effects together with competition between CytR and CRP for binding CRP2 , we extended the model defined in Table II to include terms to describe the effects of cytidine on cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast , the interaction between CytR and CRP bound to CRP2 of udpP is virtually unaffected by cytidine ( 11 ) resulting in only a small decrease in overall CytR binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR is unique among LacI/GalR family members in that binding of its regulatory ligand , cytidine , is linked not to DNA-binding affinity but instead to the cooperative interactions between CytR bound at CytO and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Finally , how does pairwise cooperativity between CytR binding to CytO and CRP binding to CRP2 of udpP survive cytidine binding , particularly when pairwise cooperative interactions with CRP bound to CRP1 of udpP or either CRP site of deoP2 does not ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>DG13 and DG23 , which pertain to pairwise cooperative interactions between CytR binding and CRP bound to either CRP1 or CRP2 , were similarly evaluated using reduced valency mutants in which specific binding to either CRP1 ( CRP12 ) or CRP2 ( CRP22 ) was eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>One effect of competition between CytR and CRP for binding to CRP1 and CRP2 of deoP2 is that whereas CRP , whether bound to CRP1 alone , to CRP2 alone , or to both together , is a net recruiter of CytR to CytO ; the reverse is not so .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Here too , CRP bound to either CRP1 or CRP2 or both is a net recruiter for CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Cytidine binding to CytR largely eliminates pairwise cooperativity between CytR and CRP bound to CRP1 of udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>By eliminating cooperativity between CytR and CRP bound to flanking sites , cytidine should make CytR into a simple competitor for CRP binding to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Recently , we showed that CytR binds specifically to multiple sites in the E. coli deoP promoter , thereby providing competition for CRP binding to CRP operator site 1 ( CRP1 ) and CRP2 as well as cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>When we analyzed CytR and CRP binding to the deoP2 promoter ( 5 ) , we also considered an alternative explanation for the competition between CytR and CRP for CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>One surprise uncovered by our analysis of deoP2 is that , in addition to its site-specific binding to CytO , CytR also binds with weak specificity to satellite sites and that this binding competes with CRP for binding to both CRP1 and CRP2 ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Consistent with the suggestion that the additional CytR-DNA interactions provide differential modulation of CRP-mediated activation , our results indicate that CytR competes for CRP binding to udp-CRP2 but does not compete for CRP binding to CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , competition between CytR and CRP for binding to CRP1 and CRP2 provides a second mode of CytR-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>We proposed that different patterns of competition between CytR and CRP for binding to CRP1 and CRP2 at different promoters could provide differential modulation of CRP-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>We suggest that cooperative CytR binding has very different effects on class I CRP activation mediated by CRP2 versus class II activation mediated by CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The molecular model defined by the promoter configurations listed in Table 3 accounts for CRP binding to CRP1 and to CRP2 and for CytR binding to CytO , with both pairwise and three-way CytR-CRP cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The values of ΔG1 , ΔG2 , and ΔG3 are statistically indistinguishable from the ΔGLoad , i values listed in Table 1 for CytR binding alone to CytO and for CRP binding alone to both CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The reason for this is that competition between CRP for binding to CRP1 and CRP2 and CytR for binding to satellite sites that occlude CRP1 and CRP2 complicates this simple analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The results also confirm the expectation that CytR and CRP compete for binding to CRP2 but not for binding to CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The net effect on CytR binding is substantial while the net effect on CRP binding to CRP2 is slight .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In principle , the same value should be obtained by summing the Δ ( ΔGLoad , i ) values for CRP binding to CRP1 and to CRP2 in the presence versus absence of saturating CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Competition contributes differently to ΔG1 and ΔG2 for CRP binding to CRP1 and to CRP2 in the presence of fixed CytR concentration than it does to ΔG3 for CytR binding to CytO in the presence of fixed CRP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In addition , concomitant with CRP binding to CRP1 , CytR will tend binding to CytO , due to its recruitment by the strong threeway cooperative interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>But this would also be partly offset by unfavorable competition between CytR and CRP for binding to overlapping sites at CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this situation , CytR competes directly for CRP binding to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast , whereas CytR is a net recruiter for CRP binding to CRP1 , it is not a net recruiter for CRP binding to CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_658">
In this model , CRP and CytR compete for binding to mutually exclusive sites at CRP2 , but CRP alone binds to CRP1 with no competition from CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></label>
<input id="act_similarity_658", type="checkbox">
<div class="ACT_similar">
<li>In this model , binding of CytR at the sites overlapping CRP1 and CRP2 and of CRP binding to CRP1 and CRP2 is mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li></div>
<li>Although CytR binds to an additional site located just downstream from the transcription start site as noted above , there is no interaction between CytR binding to this site and CytR/CRP binding to CRP2 , CytO , and CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>First , the relative affinities for CRP binding to CRP1 and CRP2 and for CytR binding to CytO are very similar in udp and deoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Symbols denote the -- different binding interactions : 1 , CRP binding to CRP1 ; 2 , CRP binding to CRP2 ; 9 CytR binding to CytO ; [ , CytR binding to the CytR site located at CRP1 ; b , CytR binding to the CytR site located at CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In the case of CRP -RCB- CytR , the antiactivator binds DNA between activator-binding sites , forming a nucleoprotein complex which prevents activation ( ValentinHansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>Only a small set of CRP-regulated promoters are repressed by CytR , however , as at least two properly spaced CRP binding sites seem to be required for efficient binding of this repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Altogether the binding experiments demonstrate that CytR and CRP bind synergistically to rpoH forming a nucleoprotein complex , which in analogy to the repression complexes formed at most CytR-regulated promoters , appears to consist of one molecule of CytR bound in between two molecules of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>This band was interpreted to contain a bound complex of CytR and CRP bound to one of the two CRP sites at a low CRP concentration ( 0.25 ng/ml ) ; no band attributable to the binding of only CRP was seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The free energy change for the association of CytR Core domain binding with the CRP2-promoter complex ( ∆ Gbridge ) was determined from paired experiments in which the promoter was titrated with CRP in the presence or absence of a fixed concentration of CytR Core domain ( 0.5-15 µM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Since this association favors promoter complexes with two CRP dimers bound , thus mimicking cooperative CRP binding , ∆ Gbridge can be evaluated by comparing CRP binding in the presence and absence of fixed concentrations of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Ratios of variances were also separately calculated considering the data obtained for each binding site and set of effectors , e.g. , for binding of CytR to CytO of the wild-type promoter in the presence of CRP and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Individual-site binding isotherms were generated by footprint titration of CRP binding in the presence ( panels B and D ) and absence ( panels A and C ) of fixed concentrations of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To address this question experimentally , matched CRP binding titrations of both deoP2 and udpP were conducted with and without CytR Core domain , as before ( Figure 6 ) , but now also comparing the presence and absence of 1 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Figure 6 depicts CRP binding to deoP2 in the presence and absence of 4.0 µM CytR ∆ 2-62 dimer ( left-hand panels ) and 13.1 µM CytR ∆ 2-63 dimer ( right-hand panels ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Parallel experiments of individual-site binding of CRP to the deoP2 promoter in the presence ( panels B and D ) and absence ( panels A and C ) of 13.1 µM CytR ∆ 2-63 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>However , with so little bridged complex formed at these CytR ∆ 2-63 concentrations ( 13-15 µM ) , the stabilizing effect on CRP binding is small , and a lower affinity limit for CytR ∆ 2-63 association is not precisely determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Under the conditions used in the gel mobility shift assay , CRP stimulates binding of CytR < 20-fold ( compare the intensities of the bands containing CytR in lanes 2 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>To address this question requires CytR Core domain in which the interaction between CytR and DNA is eliminated but other CytR functions ( i.e. , dimerization , cytidine binding , and CRP association ) are unperturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To investigate the chemical and structural basis for these effects , we investigated how cytidine binding affects association between CytR and CRP in solution and how it affects the binding of CytR deletion mutants lacking the DNA binding HTH domain , with tandem CRP dimers bound to either udpP or deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To assess these effects and their linkage to cytidine binding , we investigated the assembly of CytR and CRP in solution and compared this to the association of the Core domain dimer with tandem , promoter-bound CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Inhibition of CRP Binding by Cytidine-liganded CytR -- A second testable prediction of our model is based on the unusual mechanism of induction in which cytidine is an effector of heterologous cooperative interaction between CytR and CRP ( 6 ) 2 rather than of intrinsic binding of CytR to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Analysis of Heterologous Cooperative Interaction between deoP2-bound CRP and CytR -- To evaluate cooperative interactions between different proteins binding to DNA , we considered the thermodynamic cycle for their simultaneous binding ( cf. Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Perhaps the simplest explanation to-tie together these various effects is to suggest that cytidine binding results in conformational changes that reorient the interactions between CytR , CRP , and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The challenge then is to explain both the very broad specificity of CytR for DNA binding and for CytR-CRP cooperativity when cytidine is not bound , together with the control of cooperative interactions when cytidine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>We speculate that these different conformational modes restrict interactions between bound DNA-bound CytR and DNA-bound CRP in ways that are reflected in the different effects of cytidine binding to CytR in CytR-CRP cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>If so , elimination of CytR-CRP interactions in response to cytidine binding might allow the CRP-RNAP interaction necessary for full activation even in the presence of DNA-bound CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This indicates that cytidine binding to CytR and bridging of CytR between pairs of DNA-bound CRP dimers are thermodynamically couple processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Titrant DGapp app app 123 DG13 DG23 CytR 22.7 6 0.4 21.9 6 0.7 22.0 6 0.4 CRP 21.8 6 0.6 20.6 6 0.4 20.5 6 0.4 In particular , pairwise binding of CytR and CRP to either site appears to be essentially noncooperative when evaluated from CRP titrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this view , CytR is presumed to be an ineffective inhibitor of transcription in the absence of CRP binding only because its free concentration is insufficient to fill the site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In addition , it was recently observed ( 11 , 21 ) that the apparent cooperativity , when assessed by the effect of saturating CRP concentration on CytR affinity , is substantially greater than when cooperativity is assessed by the effect of saturating CytR concentration on CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>b In order to analyze the effect of inducer binding to CytR on CytR-CRP cooperativity , a second set of experiments was conducted with cytidine-liganded CytR and global analysis performed as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus , defining these sites thermodynamically , by their effect on th coupled energetics for binding of CytR and CRP to their primary operators , but without regard to any particular sequence of DNA , generates parameter values that are indistinguishable from those obtained from the protection actually observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Subsequently , global analysis of the individual site CRP and CytR binding data was conducted using equations that describe cooperative and competitive binding of CRP and CytR according to the model defined by the promoter configurations specified in Table II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>But when CRP is added to this same CytR concentration , cooperative binding of CytR and CRP produces a footprint that extends beyond CRP1 to the 210 region of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The central role of cooperativity in regulation is highlighted by the fact that binding of the inducer , cyt-idine , to CytR is coupled to CytR-CRP cooperativity ; this underlies the mechanism for induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The significance of CytR-CRP cooperativity to this competition is underscored by the fact that binding of cytidine ( the inducer ligand ) to CytR is coupled to CytR-CRP cooperativity ( 19 , 21 ) , a feature that is unique among LacI family repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The guanine residues protected by CAMP-CRP and CytR in the DMS footprints are indicated by open arrows , and the residues with enhanced reactivity in the CytR-binding site are indicated by closed arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The values in Table I demonstrate a similar behavior , although with a different pattern of effects , for CytR and CRP binding to udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Finally , the occurrence of tandem CRP binding sites in all CytRregulated promoters studied so far ( 14 , 32 ; unpublished data ) suggests that this is a structural requirement for CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>We propose that competition and cooperativity in CytR and CRP binding are important to differential regulation of these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Our previous analysis of the binding of CytR and CRP to the deoP2 and udp promoters has highlighted unique aspects of these interactions that appear to be related to differential regulation of the CytRregulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>To test these hypotheses and to assess how linkage might be involved in differential regulation of transcription , we have investigated cooperative and competitive CytR and CRP binding to the udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Furthermore , insertion of a synthetic CRP-binding site at the same position as that of CRP-2 , in a tfeoP2 promoter deleted for CRP-2 , re-establishes CytR regulation -LCB- Martinussen ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Yet , regulation of cytRP , in which there is only one CRP site , also depends on cooperative CytR and CRP binding ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP-1 also seems to be involved in CytR binding as a single point mutation at position -36 has been shown to reduce CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Thus , differential and coordinate gene regulation must depend on different dispositions of CRP and CytR binding sites , different protein binding affinities , and/or different levels of site-site interaction or cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Because titration studies were not conducted it is n't possible to assess either individual affinities for CytR or the mutual effects of CRP binding and of CytR binding to different sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Also , point mutations in either CRP site reduce or abolish CytR titration ; however , no co-operativity is observed in the interaction of CytR with the two CRP binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Previously , we showed that the conformational change in CytR that eliminates its cooperative binding to CRP-liganded deoP2 occurs when the first of the two subunits of the CytR dimer binds cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To ascertain whether this result of the conformational change is widespread or unique to deoP2 , we also assessed the effect of cytidine binding to CytR on CytR-CRP cooperativity in binding to udp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>FIGURE 6 : Effect of CytR Core domain on binding of CRP to the deoP2 and udp promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Panels A and B represent paired titrations to show the effect of 4.0 µM CytR ∆ 2-63 Core domain dimer on CRP binding to udpP ; panels C and D represent paired titrations to show the effect of 13.1 µM CytR ∆ 2-63 Core domain dimer on CRP binding to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Equations that describe the cooperative binding of CRP and CytR were derived by considering the relative probability of each of the deoP2 configurations ( Table III ) as specified by the Specific , and Nonspecific Additional Sites models that are introduced in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>This result confirms that competition between CRP and CytR for binding CRP2 is a significant feature of the interaction of these proteins with udp , just as it is for deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Do differences between CytR binding sites in udp and deoP2 result in different patterns of competition between CytR and CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>We first discovered competition between CytR and CRP while investigating their cooperative binding to deoP2 ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Furthermore , CytR proteins deleted for the DNA-binding domain repress deoP2 in vivo and interact with deoP2 in vitro in a strictly CAMP-CRP-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Two nanomolar DNA was used for binding with CRP and CytR proteins in concentrations shown in the ﬁgure legends , using the same as for gel retardation binding buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The cooperative binding of CytR and CAMP-CRP is mediated by protein-protein interactions between the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The CytR protein stabilized binding of CRP to the CRP sites in udpP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Thus , CytR is a modular regulator that undergoes induced ﬁt when co-binding with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Second , CytR and CRP are reported to mutually antagonize each other 's protein binding induced bends in the cytRP sequence ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP , CytR and DnaA protein binding region is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>One possible explanation for that , discussed in the literature , is that the binding of CytR to DNA has very low specificity , and the regulation is based on the formation of multimetric CRP-CytR complexes stabilized by the CytR-DNA interaction [ 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>This special mode of protein-DNA interaction in which CytR can either enhance CRP binding or compete for CRP binding affects both repression and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>It has been suggested that the significant competition underlying CytR-mediated repression is between CytR and RNAP for making protein-protein contacts with CRP , rather than competition between CytR and RNAP for binding DNA ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>What then is the significance of multiple CytR-DNA interactions and competition between CytR and CRP , and presumably also between CytR and RNAP , for binding DNA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Alternatively , CytR , CRP , and/or the DNA would have to be distorted from the symmetric arrangement proposed in the model to accommodate off-center binding by CytR to CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Therefore , it is probable that it is the CytR core protein , not the DNA-binding domains of CytR , that interacts with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>However , one can not rule out possible CytR interaction with the DNA after binding to CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>whereas CytR is able to stimulate binding of ( CAMP - ( `` RI ' under the conditions of the get retardation assa ; v. LV ' e are ( turrrntly pursuing a more detailed analysis of thr co-operativta binding of CAMP-CRP and ( $ tR to rytR1 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>CRP directly interacts with CytR , thereby shifting its binding mode for activation or repression of transcription of its targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24814025" target="_blank">24814025</a>]<br></li>
<li>Cooperative binding of CytR and CRP is a critical component of both repression and induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR has no effect on basal level transcription but instead requires CRP binding to mediate repression ( 16 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We start with the premise that CytR is a modulator of CRP activation rather than a repressor per se , a view consistent with the fact that CytR is ineffective in the absence of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Cytidine binding to CytR releases the latter from the complex , hence the activation by CRP resumes and the gene is derepressed [ 33,34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Experimental resolution of CRP binding is limited at higher CytR concentrations ; substantial protection from DNase I cleavage by CytR binding alone obscures any changes when CytR is displaced by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As expected , the stabilization resulting from the association of CytR Core domain is more acute for CRP1 as a consequence of its lower intrinsic affinity for CRP binding ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Third , by binding to the CytR operator with high affinity , CytR acts as an antiactivator at lower CytR concentrations and recruits CRP at lower CRP concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Panel A : CRP binding in the absence of CytR Core domain ( CytR ∆ 2-63 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This reflects an increase in apparent affinity for CRP binding when CytR ∆ 2-63 is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This is accomplished by partially decoupling CRP binding from CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>On the basis of the view that the thermodynamic driving force for cooperativity is CytR-CRP association , the presumption was that cytidine binding induces a concerted change in the conformation of the CytR dimer , the result of which is to decrease greatly its affinity for association with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Third , under inducing conditions in which binding of cytidine to CytR eliminates positive cooperativity between CRP and CytR ( 6 ) ,2 CytR was found to be a competitive inhibitor of CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Although not explicitly stated , our expectation was that cytidine binding would produce a CytR conformation that would not interact with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Figure 4 compares results for binding of unliganded versus cytidine-liganded CytR to CRP-liganded cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In the presence of the CAMP-CRPcomplex the two protein species bind cooperatively to deoP2 , forming a complex in which CytR occupies the sequence between the two DNA bound cAMP * CRPcomplexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>In addition to its role in activation , cAMP-CRP also plays a crucial role in negative regulation of cdd ] P. cAMP-CRP and CytR bind cooperatively to cddP forming a nucleoprotein complex that covers 84 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the presence of cAMP-CRP , however , the affinity of CytR for this sequence is greatly increased ( 100-fold ) and , moreover , the two proteins bind cooperatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In the presence of CytR , however , the two regulators bind cooperatively to cddP forming a nucleoprotein complex in which cAMP - CRP binds to CRP-1 and CRP-3 and CytR occupies the sequence between these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the presence of cAMP-CRP , however , the affinity of CytR for this sequence is increased dramatically ( approximately 1,000-fold ) , and CytR and cAMP-CRP bind cooperatively to deo-p2 , forming a nucleoprotein complex in which CytR is sandwiched between two DNA-bound cAMP-CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>As shown ( Fig. 3 , lane 9 ) , CytR aione binds weakiy to the DNA and results in protection against digestion by DNase I in the region between the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>DNase I footprinting experiments performed in the presence of increasing concentrations of CytR ( Figure 3B , lanes 12 - 13 ) revealed that CytR interacts with the region located between the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Implicit in this model is that CytR binds co-operatively to tandem , properly spaced CRP targets , since only a small set of CRP-activated promoters are repressed by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>RESULTS Our goal is to understand how CytR and CRP mediate differential regulation of CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Senear , D. F. , Perini , L. T. , and Gavigan , S. A. ( 1998 ) Analysis of interactions between CytR and CRP at CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>For example , the galE promoter is repressed by GalR , the lacZ promoter repressed by LacI and many CRP-activated promoters are repressed by CytR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Removal of CRP-2 resulted in a further reduction of promoter activity ; however , these promoters are still regulated 10-fold by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Cooperative interactions between CytR , when bound to an operator ( CytO ) located upstream of a CytR-regulated promoter , and CRP , when bound to flanking tandem promoters , are critical to the regulatory role of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Analysis of cooperative interactions between CytR and CRP at the deoP2 and udpP promoters has implicated the importance of promoter architecture in controlling repression and induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank Steven A. Short for the gift of puriﬁed CytR protein , Sankar Adhya for sending us the CRP overproducer strain SA500 , and Kalpana Srivastava for technical assistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>At some promoters , transcription activation by CRP can be repressed by a transcriptional-regulator protein , CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>This promoter is weakly activated by CRP-wt protein , but no longer regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>This is unlike the protein -- protein contact-mediated repression by CytR , which involves antiactivation by contact 39 between CytR repressor and CRP activator at promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Additionally , puriﬁed E. coli CytR and CRP proteins were used in in vitro assays ( gel retardation ) to determine their ability to-bind heterologous promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>For example , CRP depends on direct protein -- protein interactions with MelR and CytR to-bind low-afﬁnity CRP sites in the melA and deoC promoters , respectively ( 58 -- 60 ) Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>An important feature of the regulation mediated by CytR and CRP is that extents of activation , repression , and induction vary among the promoters , even among those with a similar arrangement of regulatory elements and similar binding affinities for the two proteins ( 4-6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>If the unusually high affinity for CRP imparts both high sensitivity to cAMP concentration and also contributes to high levels of CRP-mediated activation , the high affinity for CytR binding to CytO counteracts this with high sensitivity repression , thus making cddP both the first to turn on and the first to shut off of the CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus repression does not occur if CRP-2 is deleted , and single point mutations in CRP-2 reduce CytR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>These findings indicate that both CRP targets are required for efficient CytR repression of deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>These results clearly show the importance of the tandem CRP targets for CytR repression of deoCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>CytR controls the balance between CRP-mediated recruitment and activation of RNA polymerase and transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This is achieved by nesting levels of local repression , mediated by DeoR and CytR , on a more global regulation mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Coordinate gene regulation is achieved by nesting local repression by the LacI family member , CytR , on the global activation mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>We found that constitutive CRP-independent udp expression in many of the isolated mutants was caused by loss of part or all of CRP1 , indicating that it is important in CRP activation and CytR repression of udp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The unusually high affinity CytR site occluding the cdd transcription start site presumably confers another level of repression of the very strong CRP-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>This was done by comparing the energetics of CytR-CRP association to the energetics of cooperativity when CytR and CRP bind to deoP2 and to udpP both in the presence and in the absence of cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Gel Retardation Analysis of the Binding of CytR-wt , CytR-Ala-24-Glu , and CytRA9-48 to deoP2 and Random DNA in the Absence and Presence of CAMP-CRP ( A ) Binding of the three CytR proteins to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>( C ) Binding of the three CytR proteins to deoP2 in the presence of CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Indeed , the effect of competition at deoP2 is to largely eliminate recruitment of CRP by CytR , thereby negating the role of CytR as a co-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Furthermore , the CytR operators in nupG , cdd and deoP2 are not symmetrically positioned with respect to the flanking CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>a , interaction of CRP and CytR with the deoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Figure 3 presents typical results of experiments in which deoP2 DNA was first titrated with CRP , and subsequently , CytR ∆ 2-63 was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Figure 5 presents a footprint autoradiogram from a representative experiment in which deoP2 was titrated with CRP in the presence of 4.0 µM CytR ∆ 2-62 dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Lanes 1-12 show promoter DNA being titrated with increasing concentrations of CRP dimer in the presence of 4.0 µM CytR ∆ 2-63 Core domain dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The apparent afﬁnity of CytR for selected DNA sequences and deoP2-wt , -- 9 in the absence ( black bars ) or presence ( gray bars ) of 6310 M CRP , was determined by the mobility shift assay ( see Materials and methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The interaction of CytR with udpP DNA ( full protection ) required the presence of CRP and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The formation of these complexes involves both CytR ± DNA interaction and CytR ± CRP interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Because cytidine is an effector of CytR-CRP cooperativity , its effect should also be linked to how CytR binds at the various promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>These results support our hypothesis that cytidine is an allosteric-effector of CytR conformations that constrain its simultaneous association with CytO and the flanking CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>We observed in this study that CytR could block transcription in vitro to a level that is signi ® cantly lower than that observed in the absence of CRP ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of both CytR and CRP , a third retarded complex with a lower mobility is observed ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>In the presence of cytidine , CytR is inactive , and thus CRP can activate transcription in a Class III dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In the absence of cytidine , CytR is active and binds between the two CRP dimers , repressing CRP-dependent transcription [ reviewed in ( 42 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Presumably , either the spacing between the CytR and CRP sites or something inherent in the CytR site , such as the spacing between recognition motifs , is responsible for controlling whether interactions between CytR and CRP are still possible when cytidine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CRP alone activates transcription , whereas the CRP-CytR cooperatively bound to OCRP and OCYTR , respectively , represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Our results support and extend a model in which transcription is repressed when CRP moieties are bound to two appropriately positioned sites in the udpP and are themselves bound and stabilized by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CRP and CytR designated filled sites according to the transcription factor bound ; O designates empty sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Upon repression , the CytR repressor binds between the two bound CRP molecules , inactivating them and causing repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>A summary of the protection experiments ( Figures 5C and 5D ) demonstrates that the overall structure of the two repression complexes is identical ; i.e. , CAMP-CRP is bound at the tandem CRP sites , and CytR is located in between .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Binding of CytR and CRP to the rpoH promoter region tested by gel shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Binding in vitro of CytR and CRP to spacing variants The relative afﬁnity of the promoter variants for CytR and CRP , alone and in combination , was measured in a simple competition assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The presence of a promoter-external CRP target has not been observed in other CRP/CytR regulated operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The mutations introduced into the CRP-I and CRP-2 sites resulted in a strong reduction in promoter activity in presence of CRP-wt , and at the same time , CytR regulation was completely absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Figure 3 also shows the effect when CytR ∆ 2-63 was added to reaction mixtures containing CRP dimer at a concentration of 13 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>However , we do observe an additional few-fold decrease in apparent CRP affinity when the CytR concentration is increased to 0.4 mM , consistent with a simple competitive model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We evaluated the cooperative interactions and competition between CytR and CRP as described previously ( 19 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The structure of nupG is an inversion of deoP2 , and similar interactions between CytR and CRP , no doubt , explain the apparent one-way stimulation ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The latter is stabilized by cooperative interactions between CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Role of Competition and Cooperativity in deoP2 Regulation -- The important question remaining is how these multiple competitive and cooperative interactions between CytR and CRP regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The net result is that CytR is similarly effective at reversing the robust CRP-mediated activation at cddP as it is at reversing the much weaker activation of other promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The deoC , nupG and tsx operons that are under CytR regulation were activated by CRP in our study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The central importance of CAMP-CRP in CytR repression is illustrated by several observations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The ∆ G obtained for this reaction accounts for only a small fraction of CytR and CRP dimers associated as a tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The stimulation by CytR also required CRP in the cases of dup1 and dup361 but apparently not in the cases of del224 and del364 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>But simple recruitment of this sort makes little functional sense , as it would yield only two significantly populated states , i.e. CRP and CytR sites empty or CRP and CytR sites filled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Simple cooperativity is of course a two-way street , and as such , CytR would be a net recruiter of CRP as well as the reverse .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Binding sites for regulatory proteins CRP , CytR and DnaA are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>There seem to be rather strict rules for the position of the L and R CytR half-sites relative to the ﬂanking CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Alternatively , CytR may adopt a conformation at these sites that is incompatible with positive CRP -- CytR interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CRP requires cooperativity with MelR and CytR to-bind to low affinity sites in the melAB and deoCABD promoters , among others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>CytR and CRP bind cooperatively when CytR binds CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Only when CRP occupies the two sites can CytR bind and repress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>It is ironic , given the results we report here , that until relatively recently CRP and CytR were believed to compete to-bind to the same sites ( cf. Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Third , the truncated CytR proteins still have the ability to interact with CAMP-CRP , to respond to inducer , and to oligomerize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Second , the region between the CRP targets is strongly protected in the presence of both proteins , which suggests that this region is occupied by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>A direct way to evaluate the linkage would be to compare association of operator-oligonucleotide-bound CRP and CytR to association of free CRP and CytR ( 26 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Sedimentation equilibrium analysis was employed to investigate the association between free CytR and free CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>∆ G values as obtained from this analysis are tightly bounded , both for the association of CRP with free CytR and with cytidine-liganded CytR ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CRP also acts as a co-repressor in the CytR regulon ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15385551" target="_blank">15385551</a>]<br></li>
<li>However , cooperativity also means that CytR can recruit CRP and , by doing so , can act as a co-activa-tor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Given the relatively low solubility limit for CytR , a high CRP concentration was used to drive the assembly and maximize sensitivity to hetero-oligomers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>As practical approximations to the limit of saturating concentrations , 0.1 mM CRP ( total dimer ) and 75 nM CytR were used here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>As a practical limit to saturating concentration , 0.1 μM dimer was used for both CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>With this information , the distributions of ligation states of each of the promoters under physiologically relevant concentrations of CRP and CytR are easily computed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CytR and CRP proteins were added to the following ﬁnal concentrations : Lanes 1 , no proteins ; 2 , 1.3 nM CytR ; 3 , 0.4 nM CRP ; 4 , 4.2 nM CRP ; 5 , 0.65 nM CytR and 0.2 nM CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Cells were loaded with 1:1 , 1:2 , 1:4 , and 1:8 dilutions from initial concentrations of ∼ 2-3 mg/mL for CRP ; ∼ 0.4 mg/mL for CytR ; and 1:1 , 1:3 , and 1:9 dilutions of 2.7 mg/mL for Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Mixtures of CRP and CytR were 1:1 and 1:3 dilutions of these initial concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The final concentrations of CRP and CytR in ng/41 are indicated below each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_838">
Final concentrations of CRP and CytR are indicated in ng/pl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></label>
<input id="act_similarity_838", type="checkbox">
<div class="ACT_similar">
<li>Final concentrations of CRP and CytR are indicated in ng/4l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Final concentrations of CRP and CytR are indicated in ng/1ul .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li></div>
<li>This indicates that the model is a fair approximation to the heteromeric association of CytR and CRP to form tetramers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In this model , the association of the z-average molecular weight monomer corresponds to assocation of CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This entirely new phenomenon is consistent with the interpretation that CytR functions as a modulator of CRP-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As such , CytR would act initially as a switch from one kinetic mode of CRP activation to another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this view the state of CRP activation is as important as occupancy by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Instead , CytR modulates CRP-medi-ated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Of course , this necessarily means that CytR functions only under conditions of CRP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Udp activity in this strain was as high as in constitutive mutants with larger deletions and was completely CytR and CRP independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>ACKNOWLEDGMENT We thank Dr. Vira Tretyachenko-Ladokhina for the CytR and CRP used in this work as well as for discussion and insights .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>DNase I footprinting of complexes between WT , del252 , del264 , and del18 udpPs , CRP , and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Results of sedimentation equilibrium experiments conducted on a mixture of CytR and CRP are summarized in Table 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Lane 14,42 nM CRP + 280 nM RNAP + 260 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Lane 16,42 nM CRP + 260 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Lane 17 , 42 nM CRP + 26 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Lune 11:135 nM CRP + 2.6 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_856">
Lane 9 , 135 nM CRP + 220 nM RNAP + 66 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></label>
<input id="act_similarity_856", type="checkbox">
<div class="ACT_similar">
<li>Lane 10,135 nM CRP + 66 nM CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li></div>
<li>Analysis of Cooperative and Competitive Binding of CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Regions protected by CytR and CRP are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Gel Retardation in the Presence of CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Solution Assembly of CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CRP and CytR Purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In this context , the suggestion that the primary competition between CytR and RNAP is for interaction with CRP ( 31 ) is attractive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This phenomenology is consistent with recent reports of `` unidirectional stimulation '' in CRP and CytR interaction with nupG ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP concentrations were correcte configuration ( fs ) as given by Table 1 : Configurations and Free Energy States for CRP and CytR Core Domain Association with deoP2 and udpP Promoter Configurationa free energy free energy CRP2 CytR CRP1 contributionsb state - ∆ Gs/RT i j e [ CRP ( cAMP ) ] [ Core ] fs ) 1 ∑ - ∆ Gs/RT 1 s , ie [ CRP ( cAMP ) ] i [ Core ] j ( 2 ) 1 O O O reference state ∆ Gs ,1 2 O O CRP ∆ G1 ∆ Gs ,2 3 CRP O O ∆ G2 ∆ Gs ,3 4 CRP O CRP ∆ G + ∆ G ∆ G 1 2 s ,4 5 CRP Core domain CRP ∆ G1 + ∆ G2 + ∆ Gbridge ∆ Gs ,5 6 O CytR O ∆ G3 ∆ Gs ,6 7 O CytR CRP ∆ G1 + ∆ G3 + ∆ G13 ∆ Gs ,7 8 CRP CytR O ∆ G2 + ∆ G3 + ∆ G23 ∆ Gs ,8 9 CRP CytR CRP ∆ G + ∆ G + ∆ G + ∆ G 1 2 3 s ,9 ∆ G123 a Promoter configurations with sites denoted as filled ( CRP , Core domain , or CytR ) or empty ( O ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>DG is the sum of free energy contributions for configuration s ( Table III ) ; i and j are the stoichiometries of bound CRP ( cAMP ) 1 complexes and CytR dimers in configuration s. Summation of the relative probabilities for all configurations in which protein is bound to any given site derives the binding equation for that site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The probabilities are given by where DGs is the sum of free energy contributions for configuration s ( Table III ) , and where i and j are the stoichiometries of bound CRP - ( cAMP ) 1 complexes and CytR dimers in configuration s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Crystalline cAMP ( > 99 % pure as 99 % pure as free acid ) were purchase ΔGs is the sum of all free energy contributions for a given configuration , s ( Table 3 ) ; i and j are the stoichiometries of bound CRP ( cAMP ) 1 and CytR in configuration s , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Neither CRP in the absence of cAMP , nor the CytR extract changes the cleavage pattern of DNase I ( lanes 2-3 ) relative to that in the control in which no protein is added ( lane 1 ) , Th ` footprint ' produced by cAMP-CRP alone is displayed in lanes 4-6 ; cAMP-CRP binds to two targets , with high and low affinity located around positions - 9 3 ( CRP-2 ) and - 4 0 ( CRP-1 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In order to analyze the possibility that CytR afﬁnity to different promoters was mediated by their ability to-bind cAMP -- CRP , gel retardation experiments with the same 32P-end-labeled DNA fragments were performed in the presence of puriﬁed E. coli CRP protein ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Binding of cAMP - CRP and CytR to wild-type and mutant promoters In order to investigate the interaction of cAMP-CRP and CytR with cddP and to define the role of the different protein binding sites , we performed DNase I footprinting experiments on 32P-end-labelled fragments in the presence of purified proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In Fig. 2B the binding of the two CRP proteins is compared in the presence of various amounts of partially purified CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Despite this role of CytR as a functional antagonist of CRP , the two proteins interact cooperatively , resulting in substantially increased CytR binding af-finity when CRP is present ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Fig. 7 demonstrates how the Specific Additional Sites model rationalizes both the apparent high cooperativity of CytR binding in the presence of CRP ( cAMP ) 1 at fixed , saturating concentration and apparent weak cooperativity of CRP ( cAMP ) 1 binding in the presence of CytR at fixed , saturating concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR binding to wild type , [ squlf ] ) , CRP12 ( [ trif ] ) , and CRP22 ( [ itrif ] ) udp promotere in the presence of saturating CRP ( cAMP ) ( 64 nM ) and 1 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Fig. 5 shows binding of CytR to CRP-liganded udp in the presence of a saturating concentration ( 3 ) of cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To test this prediction quantitatively , we conducted parallel titrations of CRP binding in the presence of saturating cytidine , with and without fixed but now subsaturating CytR ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>For a system in which only intrinsic binding and cooperativity contribute , the difference between ΔGLoad , i 's obtained for binding of either protein ( CytR or CRP ) in the presence versus absence of the other yields the cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In the first type of binding experiment , each protein , either CRP or CytR , is ti-trated alone ; in the second , each protein , either CRP or CytR , is titrated in the presence of a constant , near the saturating concentration of the other protein ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Second , binding of each protein ( CytR and CRP ) to each promoter is analyzed in the presence of a fixed and saturating concentration of the other protein ( CRP and CytR , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>We have taken this approach one step further , to deﬁne the optimal binding site of the CytR repressor , alone as well as in the presence of its helper protein , cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>B -- D , CytR and CRP ( cAMP ) 1 binding together , i.e. the concentration of one protein is varied in presence of a fixed concentration of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast to the strong CytR stimulation of cAMP-CRP binding to CRP-1 and CRP-2 in the wild-type promoter , in conjunction with protection of the spacer region ( compare lanes 3 and 6 , right panel ) , there is only a weak effect of CytR in the promoter fragment carrying the single mutation at - 4 5 at the lowest cAMP concentration ( lanes 3 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The loading free energy change for CytR binding to deoP2 that is saturated by CRP ( cAMP ) 1 ( DGCRP CytR ) includes contributions from both intrinsic binding and cooperativity , i.e. DG3 1 DG123 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>TABLE I Loading free energy changes for binding of CRP and CytR to the udp regulatory region Free energy changes for saturation of udp operators with either CRP ( cAMP ) 1 or CytR in the presence or absence of effector ligands are as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To evaluate the cooperative free energy change in this manner , loading free energy changes were determined for binding of CytR and CRP ( cAMP ) 1 to deoP2 , each in the presence and absence of a fixed concentration of the other ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>At 150 mM , cAMP had no effect on intrinsic CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Moreover , the results show that ( i ) , the co-operative binding of cAMP-CRP and CytR to nupG stimulates binding of CytR drastically , by a factor of more than 3000 ; ( ii ) , CytR triggers a repositioning of the cAMP-CRP complex in the - 3 5 region ; and ( iii ) , the inducer of CytR , cytidine , does not completely prevent the co-operative formation of an intermediate repression complex consisting of CytR and one cAMP-CRP molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In contrast , the presence of cytidine eliminates the full repression complex ( lanes 5 and 11 ) ; it does not , however , abolish co-operative binding of CytR and one cAMP-CRP complex ( Fig. 4 , compare lanes 2 , 9 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In the intermediate CRP-CytR-nupG complex , cAMP-CRP bound at CRP2 also strongly stimulates binding of CytR ( Fig. 2 , compare lanes 3 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The stimulation of udp expression by CytR in certain mutants may reﬂect its stabilization of cAMP-CRP binding to target DNA and illustrates that only modest evolutionary changes could allow particular multiprotein complexes to serve as either repressors or transcriptional-activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>By introducing point mutations in the CRP sites , which strongly reduce binding of the cAMP-CRP activator complex , activation of transcription was severely reduced and CytR regulation was completely abolished , even when CytR was overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>First , the effect of the mutations in the CRP sites on CytR regulation is an indirect effect mediated through reduced binding of the cAMP-CRP complex , rather than a direct effect on CytR-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>These findings support a general model for CytR binding and CytR regulation , in which ( i ) CytR and the CRP/cAMP complex bind to similar or Identical targets , ( ii ) two or more targets are necessary for proper binding of CytR to a promoter region , and ( iii ) CytR represses transcription by antagonizing cAMP/CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Here we demonstrate the presence of two binding sites for the cAMP-CRP complex , CRP-1 and CRP-2 , and show that cAMP-CRP and CytR bind cooperatively to tSx-p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In this report , we have tested this model in vivo by assessing the relationship between binding of the cAMP-CRP complex to the two CRP binding sites and the extent of CytR regulation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>It should be emphasized that this analysiswas not designedto provide quantitative estimations of promoter activity Discussion Here we have analysed in vivo how CytR regulation is related to-binding of the cAMP-CRP complex to the two CRP sites in deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Here evidence is presented to show that rot is expressed from four promoters , three of whioh are activated by cAMP-CRP and repressed by the co-operative binding of CytR and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>cddPD79 bears a single target for CytR and cAMP -- CRP , respectively , and is repressed ﬁvefold by the co-operative binding of the two regulators ( see Fig. 1 and Table 1 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>A model for the mode of action of CytR must account for the observations in this study that CytR only inhibits cAMP-CRP-dependent initiation of transcription and that tandem , intact CRP binding sites are required for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In the absence of CytR , cAMP-CRP interacts at one set of sites ( CRP-1 and CRP-2 ) and both of these binding sites are required for full promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Furthermore , we propose that different combinations of these CRP sites are used during activation and repression : activation of cddP involves CRP - 1 and CRP-2 whereas cAMP-CRP binding in the presence of CytR involves CRP-1 and CRP-3 , i.e. the cooperative binding of cAMP-CRP and CytR induces a 2 bp repositioning of from to cAMP-CRP CRP-2 CRP-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>We could not ﬁnd A-sequences for which it would make sense to test for cooperative binding , since most of the A-fragments contain mutations in the CRP targets , or because the binding sites are too close to allow simultan-eous binding of cAMP -- CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>This implies that it is the conﬁguration of the CytR half-sites and the position of these half-sites relative to the ﬂanking CRP targets , and not the strength of the CytR -- DNA interaction , that are important for cooperative binding of CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>A few simple possibilities are likely : binding of CytR to A-sites may deform the DNA in a way that impedes wrapping the DNA helix around the proteins in the CRP2 -- CytR -- DNA complex , or it may induce DNA conformational changes in the ﬂanking CRP targets , unfavorable for binding of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>( i ) CytR and cAMP-CRP compete for the same DNA sequences , with CytR binding cooperatively to the tandem CRP sites , i.e. , a model which would resemble the cI/Cro system of bacteriophage lambda , in which the cI and Cro proteins bind the DNA and each excludes to same sequences the binding of the other ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>So tt is not enough to have two DNA-bound cAMP-CRP complexes to obtain simultaneous binding of CRP and CytR ; the structure of the spacer region or the spatial orientation ot the tandem CRP targets is also critical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Structural ﬂexibility and induced ﬁt While it is not surprising that transcription factors can exhibit promiscuous DNA-binding characteristics , or undergo induced ﬁt when interacting with other molecules , it is remarkable that CytR can assume multiple conformations of about equal stability when co-binding with cAMP -- CRP ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>These results strongly indicate that cooperative DNA binding of CytR and cAMP-CRP is achieved through protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Two mechanisms can be envisaged to participate in this heterologous cooperativity : changes in DNA conformation induced by DNA binding of the two proteins and/or protein-protein interactions between CytR and cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>( ii ) CytR could interact with deoCp2 without precluding cAMP-CRP binding to the DNA and antagonize the activating effect of cAMP-CRP either by inducing a conformational change in the DNA or by directly contacting CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Furthermore , the inducer ( cytidine ) does not affect independ-ent DNA binding of CytR , rather the CytR/cAMP * CRP cooperativity is perturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Although CytR is a sequence-specific DNA-binding protein , it requires cAMP-CRP for effective binding to its cognate promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Here , in vitro selection experiments have been used to examine the DNA-binding characteristics of CytR , by itself and when co-binding with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Rather , cAMP -- CRP stabilizes alternative DNA-binding modes of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Taken together , the CytR ( 20 nM ) binding signi ® cantly enhanced the cAMP ± CRP binding , particularly at the downstream site , suggesting that a protein ± protein interaction between two proteins resulted in increased cAMP ± CRP binding to the DNA sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Consistently , it was noted that cAMP ± CRP binding was greatly enhanced in the presence of CytR , especially at the CRP1 site , suggesting a protein ± protein interaction between two proteins ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Binding of CytR to its target excludes binding of cAMP-CRP to CRP1 , and promotes binding of cAMP-CRP to CRP0 , therefore allowing co-operative binding of the two protein species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Taken together , the observation that neither of the CRP sites by itself titrates CytR , and the lack of co-operativity in CytR interaction with the two CRP sites , seem to ruie out the possibility that CytR can compete with cAMP-CRP for binding to the two CRP sites , as hypothesized above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>The consequences of this fact on the distribution of CRP-bound states ( assuming no CytR binding ) are plotted as function of CRP-cAMP concentration in Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The combined footprint is the sum of the individual cAMP-CRP and CytR footprints , suggesting that tbis footprint is obtained by binding of two oAMP-CRP molecules and the CytR protein bound in belween .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The equation for binding of a protein , CRP ( cAMP ) 1 or CytR , to an individual site is obtained by summing fs over all configurations in which that protein is bound to that individual site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The intensities of the bands containing CytR ( lanes 2 and 8 ) show that cAMP-CRP stimulates binding of CytR more than 30-fold in the mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Isolation of the optimal CytR-binding site The course of the selection for CytR operators ( in the absence of cAMP -- CRP ) was evaluated by mobility shift assays ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The cAMP ± CRP binding site is shown as a shaded box and CytR as an open box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The cooperative binding of CytR and cAMP -- CRP to this sequence is very inefﬁcient , even though the independent CytR afﬁnity is similar to that of other C-sequences ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The CytR half-operator consensus for cooperative binding with cAMP -- CRP is slightly different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Therefore , the reduced repression by CytR observed for the tsx-503 allele is likely to be a consequence of reduced binding of the cAMP-CRP complex to the CRP-2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>These results indicate that strong CytR-udpP interaction depends on binding by the cAMP-CRP complex and , conversely , that CytR itself enhances CRP binding to the udpP region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Efficient binding of CytR to c/eoP2requires interactions of the cAMP-CRP complex at two intact CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Hence , binding of CytR to deoP2 is strongly dependent on proper binding of the cAMP-CRP complex to both CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>While the cAMP-CRP complex is able to-bind independently to the tandem targets in deoP2 , no detect-able binding of CytR could be observed in the absence of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Strong synergy between CytR and CRP in binding to deoP2 is particularly obvious in experiments where the cAMP-CRP comlex is present at low concentrations ( Fig. 2 and Fig. 4 , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>17808 DNA Binding of the C tion pattern of CytR and cAMP.CRP with deoP2 , strongly indicate that the cooperative binding of CytR and CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Specifically , we demonstrate that point mutations reducing cAMP-CRP binding result in drastic reductions in CytR interaction with deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Hence , CytR regulation of deoP2 can be modulated by simply varying DNA binding of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Second , the efficiency of CytR regulation of deoP2 can be varied by changing the D N A binding strength of cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Binding of cAMP-CRP to the mutated CRP-2 site is strongly reduced ; however , in the presence of CytR , binding of cAMP-CRP is reestablished ( Fig. 7B , lanes 10 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>For the cooperative binding with cAMP -- CRP , the corresponding CytR concentrations were 7310 -- 10 M and 2310 -- 11 M. CRP and cAMP were employed at concentrations of 2310 -- 7 M and 50 µM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CytR was in .100 - fold excess to the binding site ( cAMP -- CRP absent ) ; in the presence of cAMP -- CRP , both proteins were in at least 5-fold excess to the binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Finally , in the combined footprint , cAMP-CRP binding to CRP-2 is enhanced , and in the presence of cAMP-CRP , binding of CytR is enhanced ( Fig. 3A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>DG123 can also be evaluated by comparing CRP ( cAMP ) 1 binding in the presence versus the absence of saturating CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As there appear to be more than three CytR binding sites , the saturating CytR stoichiometry , whether in the presence or absence of CRP ( cAMP ) 1 , is n't obvious .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Co-operative binding of cAMP-CRP and CytR Next we tested binding of CytR and cAMP-CRP to a 160 bp fragment containing nupG sequences from - 1 1 8 to - 1 0 in the DNase I footprinting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>This approach revealed that CytR uses different settings of half-operators when binding independently and in concert with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Many different CytR-binding site conﬁgurations seem to allow cooperative interaction of CytR with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Furthermore , the reduction in CytR titration parallels the reduction in binding of cAMP-CRP to the mutated CRP sites in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Independent binding of CytR and cAMP-CRP to the mutant tSX-p2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Obviously , binding of CytR is strongly stimulated by cAMP-CRP ( compare lanes 5 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>A , CytR and CRP ( cAMP ) 1 binding alone , each in the absence of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>( B - - - D ) Simultaneous binding of CRP ( cAMP ) 1 and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Combined binding of cAMP-CRP and CytR to the mutant tsx-p .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Individual site binding of CytR and CRP ( cAMP ) 1 to the udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Global analysis of individual site CRP ( cAMP ) 1 and CytR binding was conducted essentially as described ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>FIGURE 3 : Individual site isotherms for binding of CytR and CRP ( cAMP ) 1 to cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>For example , most of the in vitro studies have been conducted as combinations of plus/minus the various components ( CRP , CytR , cAMP , cytidine , and promoter elements ) with insufficient understanding of the mutual effects of interactions between these molecules to know what concentrations are necessary to achieve particular effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>combined footprint of the tsx-503 allele is that the cAMP-CRP complex can interact strongly with the CRP-2 target in the presence of CytR ( Fig. 7B , lane 11 ) , although the affinity of cAMP-CRP for the mutated target is severely reduced ( Fig. 6 , lanes 31 to 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Binding of CytR to nupG is stimulated more than 3000-fold by cAMP-CRP ( Fig. 2 , compare lanes 3 and 14 ) , and also , the presence of CytR increases protection in the downstream region considerably ( Fig. 2 , lanes 6 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Binding of cAMP-CRP is , however , enhanced by the CytR extract ( Fig. 2 , right panel ; compare lanes 4-6 with lanes 7-9 ; see also Fig. 4 , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The cleavage pattern in the CRP2 region is identical to that obtained in the presence of cAMP-CRP alone ( Fig. 2 , lane 5 ; Fig. 3 , lane 6 ) , indicating that cAMP-CRP also occupies this target in the presence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Firstly , the cleavage pattern in CRP-l is identical to that obtained in the presence of saturating concentrations of cAMP-CRP alone ( Figure 3A , compare lanes 3-4 with 5-6 and 9-10 with 11-12 ) indicating that cAMP - CRP also occupies this target in the presence of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>First , pairwise cooperativity between CytR and CRP ( cAMP ) 1 bound to CRP2 yields about a 10-fold increase in effective CytR affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>When the combined footprints ( lanes 7 and 8 ) are compared with the individual footprints of cAMP-CRP and CytR ( lanes 3-6 ) it appears that the positive interactions between CytR and cAMP-CRP bound at CRP2 is not fully abolished by the inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>As the pattern of the 50 bp combined footprint corresponds to the sum of the independent footprints , this complex most likely consists of cAMP-CRP bound to CRP2 , and CytR to the CytR target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>At P1 and P2 transcription is repressed by the DeoR repressor and induced by deoxyribo-se-5-P ; at P2 transcription is also repressed by the CytR repressor ( the inducer being cytidine or adenosine ) and depends on cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>In this context it should be noted that the dependence of CytR on cAMP-CRP is in striking contrast to the action of the classical bacterial repressors , which appear to regulate promoter activity in the absence of additional regulator proteins ( for reviews , see Adhya 1989 ; Gralla 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In the absence of the CytR repressor , cAMP-CRP binds to two targets ( CRP-1 and CRP-2 ; Figure 1 ) and it is evident from our analysis of deletion mutants that the high affinity site , CRP-1 , is the essential site for promoter activation ( Table IA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In the absence of cAMP -- CRP , the CytR repressor binds two such octamer motifs in inverted or direct repeat arrangement , preferably separated by 2 bp ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>( A ) In the absence of cAMP -- CRP , the CytR repressor preferentially binds octamer repeats in direct or inverted orientation , separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the first model , cAMP-CRP still binds to CRP-1 and CytR occupies the sequences on both sides of CRP-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the third model , CytR interacts with the same DNA sequences in the combined footprint as in the independent footprint , and cAMP-CRP , rather than binding to CRP-1 , binds to the region located immediately downstream of the CytR-binding sequence ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The dilutions of the CytR ' extract used are indicated , cAMP , purified CRP protein , and each of the nucleosides cytidine , adenosine , and uridine were added to the reaction mixture to final concentrations of 50 [ LM , 13 p.M , and 10 mM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The concentrations o * cAMP in the binding reactions are indicated above lanes 4-9 , and the amounts of purified CRP and CytR extract used are 0.5 ng and 0.1 ( ig per reaction , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The concentrations of both CytR and CRP ( cAMP ) 1 , whether used as titrant or as held constant , were the independent variables .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The region protected from DNase I by cAMP -- CRP and CytR is indicated to the left of each panel by a thin and thick line respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Transcription is repressed by a three-protein complex ( cAMP receptor protein ( CRP ) - CytR-CRP ) that is stabilized by cooperative interactions between CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In this complex , the presence of CytR forces cAMP-CRP to interact at CRP-3 by means of protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>B-E : Underlined letters represent base pairs at which the G is protected from DMS methylation by cAMP-CRP or CytR ; letters in bold indicate increased reactivity of adenine or guanine towards DMS methylation in the presence of protein ( this work ; Reznikoff and Abelson , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Moreover , the efﬁciency of E. coli CytR protein to-bind udpPV c increased markedly in the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>In this second model , protein-protein interactions between CytR and DNA-bound CRP could be a prerequisite for the interaction of CytR with deoCp2 , with CytR forming a bridge between the two DNA-bound cAMP-CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In most cases , these complexes consist of a DNA-bound CytR protein , sandwiched between two DNA-bound cAMP -- CRP complexes [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>These experiments clearly demonstrate that E. coli CytR protein bound to heterologous promoters in a highly cooperative fashion in concert with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Hydroxyl-radicalfootprint of cAMP-CRP and CytR bound to nupG.CRP targets are indicated ; final protein concentrations are in pg m1-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>These results imply that a CytR protein bound to C-sequences does not change shape upon addition of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Speciﬁcally , CytR has a relaxed spacing requirement between operator half-sites and can , in concert with cAMP -- CRP , repress transcription in vivo from octamer boxes separated with centre distances up to three helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>The DMS footprints show that CytR simultaneously contacts guanines in both half-sites when bound co-operatively with cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In DMS footprinting , CytR invariably protected the central guanine of each half-operator , whereas cAMP -- CRP , alone and in combination with CytR , protected two guanines in the CRP-target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>CytR ± CRP-mediated repression at deoP2 To assess CytR regulation at deoP2 in vitro , we ® rst examined transcription with intact RNP in the presence of CytR , CytR and cAMP ± CRP , and CytR , cAMP ± CRP and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In an attempt to determine the role of a-CTD in CytR / CRP-mediated repression at deoP2 , we next investigated transcription by the reconstituted RNP derivatives in the presence of a constant concentration of cAMP ± CRP and varying amounts of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>DNase I footprint of deoP2 in thepresence of purified CytR protein , cAMP * CRP , and inducer ( cytidine ) A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>As a result , the cAMP -- CRP-dependent afﬁnity of CytR for the C-sequences is only 2-fold higher than for deoP2-wt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Thus , the high independent afﬁnity of CytR for the C-sequences , relative to deoP2-wt , has not resulted in a corresponding increase in cAMP -- CRP-dependent afﬁnity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the absence of cAMP ± CRP , addition of CytR had no effect on deoP2 transcription ( Figure 5 , left panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Furthermore , CytR only regulates initiations of transcription from deoP2 in the presence of cAMP-CRP ( 7-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Thus , it appears that CytR ± CRP communication at deoP2 also affected the cAMP ± CRP-independent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of CRP ( Pro-110 - + Ser ) , addition of the CytR extract results in formation of both cAMP-CRP/CytR/deoP2 complexes , but , again , the amounts of complexes formed are less than in the presence of CRP-wt ( compare lanes 3 and 8 , curved arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Protection of the wild-type deoP2 promoter ( right panel ) and a deoP2 promoter deleted tor CRP-2 ( left panel ) from DNase I digestion by CRP alone ( lane 2 ) , by CRP and CytR ( lane 3 ) , by CRP in the presence ot cAMP ( lanes 4 , 5 and 6 ) or by a mixture of CRP , cAMP and CytR extract ( lanes 7 , 8 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Ihe DNase 1 digestion pattern in the wild-type ( wl ) and three mutant derivatives of deoP2 in the presence of CRP , cAMP and CytR extract , as indicated above each lane , are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>This difference in the mRNA le-vel and the b-galactosidase activity could be a result of the complex control of the deo operon by other control proteins such as DeoR , CytR and CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Release of CytR from the nucleoprotein complex in response to cytidine leaves the cAMP-CRP activator complex bound to tSx-p2 , which will allow immediate stimulation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>As shown in Table 1 , initiation of transcription is regulated strongly both by the CytR repressor ( 40-fold ) and by the cAMP / CRP complex ( 100-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Recently , we have shown in vitro that CytR interacts strongly with deoP2 only in presence of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In the presence of the cAMP-CRP complex , initiation of transcription from deoCp2 was repressed 10-fold by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Consistently , CytR represses transcription in vivo only in the presence of the cAMP-CRP complex ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Negative regulation by CytR depends on both CRP-1 and CRP-2 and is confined to cells containing the cAMP-CRP complex ( Sogaard-Andersen etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Thus , in the full repression complex cAMP-CRP occupies CRP2 , and CytR a target immediately downstream of CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Most strikingly , CytR can adopt widely different conformations that are equally favored energetically for complex formation with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Gel retardation of 32P-end-labeled udpP-containing DNA fragments , using puriﬁed CytR and CRP in the presence of 50 mM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Two nanomolar DNA was incubated with CytR and CRP in concentrations shown above the gel in the conditions described in Materials and Methods in the presence of 50 mM cAMP , and the mixtures were treated with 0.5 U of DNase I ( Promega ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Isolation of DNA sequences that support CRP2 -- CytR -- DNA complex formation We next performed the selection in the presence of both CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>To investigate whether cAMP -- CRP has any effect on the DNA recognition speciﬁcity of CytR , we repeated the selection/ampli ﬁcation experiments in the presence of both CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>It thus appears that selection in the presence of cAMP -- CRP has produced DNA targets to which CytR binds in a conformation that is designated to interaction with cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The region protected from DNase I by formation of the CRP -- CytR -- DNA complex corresponds to the combined footprint of cAMP -- CRP and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Therefore , in the mobility shift assay , CytR and one cAMP-CRP complex still bind co-operatively to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>We can now consider why CytR requires cAMP -- CRP to-bind DNA efﬁciently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>cAMP-CRP is tentatively showntoiniroduce t * io significant bends in CRP-1 and CRP-2 and CytR is shown not to deform the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>In the presence of CytR ( B ) , the cAMP ± CRP , especially that bound at the promoter-proximal site , undergoes an allosteric change such that it no longer interacts with a-NTD positively , but instead negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of both cAMP-CRP and CytR , however , this footprint remains unchanged , in contrast to the strong effects observed in the wild-type promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The DNase I digestion pattem in the presence of CRP and various concentrations of cAMP ( shown above each lane -RCB- ( lanes 2 , 3 and 4 ) and in the presence ot CRP , cAMP and CytR extract ( lanes 5 , 6 and 7 ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The footprint obtained with cAMP-CRP clearly indicated that the same targets ( CRP-1 and CRP-3 ) were occupied both in the presence and in the absence of CytR ( Figure 8 , compare lanes 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The presence of both CytR and cAMP-CRP results in protection of a region of approximately 80 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The action of CytR in vivo is absolutely dependent on the presence of cAMP ± CRP , and therefore CRP interacts alternatively , and mutually exclusively , with CytR or RNP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In the presence of cAMP -- CRP , CytR binds the C1 , C2 and C3 sequences with very similar afﬁnity ( Figure 3 , hatched bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Two sequences from each of groups C1 , C2 and C3 were footprinted by DNase I and DMS in the presence of CytR and cAMP -- CRP ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the presence of both CytR and cAMP-CRP , guanine positions - 99 , - 74 , - 57 and - 47 on the ( + ) strand ( lane 4 ) and - 8 8 , - 8 6 , - 7 3 and - 5 6 on the ( - ) strand ( lane 8 ) are protected from DMS methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Notably , the footprint patterns in the CytR region are relatively independent of the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the presence of both cAMP-CRP and CytR , a combined footprint is obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Second , the remaining parts of the footprint strongly deviate from those obtained in the presence of cAMP-CRP or CytR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>CytR titration in the absence ( left panel ) and presence ( middle and right panels ) of cAMP ± CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Figure 7B shows the cAMP ± CRP footprint at 0.1 mM CRP and 0.1 mM cAMP in the absence ( lane 2 ) or presence of CytR ( 20 nM ) ( lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>( B ) cAMP ( 0.1 mM ) and CRP ( 0.1 mM ) were reacted with pdeoP2 in the presence or absence of CytR ( 15 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>When both CytR and cAMP-CRP are added , at the same concentrations as in the experiments of lanes 2 and 3 , two new retarded bands appear ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Samples of lane 2 contained CRP ( 30 nM ) and cAMP ( 100p ~ ) sa , mples of lanes 3contained CRP ( 30 nM ) , RNAP ( 250 nM ) and cAMP ( 100 pM ) , and samples of lanes 4contained CRP ( 30 nM ) , cAMP ( 100 pM ) , and CytR ( 26 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>These changes in the digestion pattern are detected in nearly all gels ( see Fig. 2 , 3 , 5 , 6 , and 7 ) , irrespective of whether cAMP-CRP or CytR proteins were present in the assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In this context , cAMP ± CRP , an activator , is converted through an interaction with CytR to a repressor that interferes with RNP acting at deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In order to gain further insight into the mechanism by which the CytR repressor functions , we define here the interaction of cAMP-CRP and CytR with decP2 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Furthermore , our results reveal that widely different conformations of the CytR repressor can cooperate with cAMP -- CRP to form nucleoprotein complexes of equal stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Thus , cAMP -- CRP induces conformational changes in CytR so that the repressor ﬁts the natural targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>cAMP -- CRP-induced conformational changes of the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Transcription at the various promoters is activated by cAMP-CRP , and repressed by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>A subset of these promoters is activated by the cAMP-CRP complex and repressed by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Moreover , mutant cdd promoters in which CRP-2 and CRP-3 have been deleted are partially regulated by CytR , and cAMP-CRP and CytR still bind cooperatively to these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>These findings provide clues to an understanding of how cAMP - CRP and CytR interact at a structurally diverse set of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>CytR and cAMP-CRP bind co-operatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Hence , cAMP-CRP and CytR bind cooperatively to cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>cAMP-CRP and CytR bind cooperatively to this promoter and the combined footprint is the sum of the independent footprints .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Hence , cAMP-CRP and CytR bind cooperatively to the tSx-p2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The nupG promoter is activated 50-fold by cAMP-CRP , and repressed fourfold by CytR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The deoP2 promoter is activated 30-fold by cAMP-CRP and repressed 10-fold by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Moreover , this promoter was only weakly activated by cAMP-CRP ( 3-4 fold ) and not regulated by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>CytR and cAMP CRP to thdeeoP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>RESULTS Binding of cAMP-CRP and CytR to the tsx-p2 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>However , when couched in quantitative terms as in Table II , it is evident that this phenomenology reflects the failure of Fig. 5 to account for all configurations of bound CytR and/or cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Addition of puriﬁed CytR to reactions containing cAMP -- CRP gave rise to a single new band with lower mobility ( Fig. 6A -- D , lines 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Addition of increasing amounts of CytR to transcription assays containing cAMP ± CRP gradually decreased transcription ( Figure 5 , middle panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>We previously showed that deoP1 is regulated neither by CytR nor by cAMP-CRP ( Vaientin-Hansen etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Thus , CytR probably inhibits RNA synthesis by antagonizing cAMP / CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>Here we present evidence that activation by cAMP-CRP and repression by CytR is coupled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Transcription is activated in response to intracellular cAMP levels by CRP , repressed by CytR , and induced by cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>bEffector concentrations : CRP , 0.1 μM ( total dimer ) ; cAMP , 150 μM ; CytR , 0.1 μM ( dimer ) ; cytidine , 2 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>was to A B cAMP CytR CytR CRP 1 80 ¬ 1 1 1 1 00 CytR4 ¬ 1 1 1 ¬ cytidine urid ine cytidine - - + ¬ - + - - + - + adenoslne * + - - + - + ¬ AMML - - - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>It is not obvious from the DNase I footprint whether CytR and/or cAMP-CRP binds in the downstream part .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Subsequently , we examined the DNase I footprint of cAMP ± CRP and CytR bound to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1064">
DMS footprint of cAMP-CRP and CytR bound to nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_1064", type="checkbox">
<div class="ACT_similar">
<li>DNase I footprint of cAMP-CRP and CytR bound to nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li></div>
<li>Purified CRP proteins , CytR extract , and cAMP were added as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>These include strict spacing requirements for the interacting sets of proteins , CytR and cAMP -- CRP or cAMP -- CRP and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Arrows denotes guanines protected from DMS methylation ; ( ← ) DMS protection by cAMP -- CRP ; ( ⇐ ) additional DMS protection by cAMP -- CRP þ CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>cAMP-CRP and CytR targets , as determined in the DNase I footprints , are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>DNase I and DMS footprints of CytR and cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Evidence for CytR induced repositioning of cAMP - CRP Several predictions follow from this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>On the other hand , cAMP-CRP occupies CRP-2 in the absence of CytR in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to a cddP derivative with CRP2/3 deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>( A ) Binding of either CRP ( cAMP ) 1 alone or of CytR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to wt cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Binding of CytR and cAMP -- CRP to spacing variants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>cAMP -- CRP complexes are hatched ; CytR is in black .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>cAMP -- CRP does , however , induce drastic changes in CytR 's quaternary structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to deoP2 deletion derivatives Next we probed cAMP-ORP and CytR binding to two deoP2 derivatives in which CytR regulation in vivo is abolished ( Sogaard-Andersen ef ai , 1990b ; our unpu-blished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to deoP2 derivatives with point mutations in the CRP sites As mutations in each of the two CRP targets of deoP2 decrease CytR regulation in vivo , we were interested to see whether this effect could be demonstrated at the molecular level in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>rot expression from pNM91 is regulated 5.7-fold by CylR and slrongly activated by cAMP-CRP whereas rot expression from pNlV195 is unaffected by CytR and exhibits a 35 % decrease in activity in a Crp strain ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Expression of heterlogous udp promoters in E. coli cells To verify the level of CytR regulation and cAMP -- CRP activation of heterologous promoters we constructed a series of plasmids carrying the udpP -- lacZ transcriptional fusions ( plasmids pMZ32 , pMZ37 , pMZ39 and pMZ43 , see Materials and methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>As shown in Table 3 , CytR regulation was reduced from approximately 8-fold in udpPEc and udpPSt to approximately 3-fold in udpPYp , and to 1.6-fold in udpPV c whereas cAMP -- CRP activation was approximately the same in all promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>cAMP-CRP activation and CytR repression of the mutant promoters were determined by measuring the activity of/Lgalacto-sidase in isogenic strains carrying mutations in cytR or crp ( Table 1 and Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>b Extent of regulation by CytR protein was estimated as the ratio of activities in cytR1 versus cytR strains ; regulation by cAMP-CRP was estimated as the ratio of activities in crp1 cytR cells grown in minimal medium with glycerol ( fully derepressed udp expression ) versus crp cells grown in the same medium FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>As it could be argued that the lack of CytR regulation on cAMP-CRP-independent initiations is caused by a decreased expression of cytR in the Acya background , we introduced the CytR-overproducing plasmid p46-213 ( in which cytR is expressed from the tac promoter ; see Materials and Methods ) into cya + and Acya strains harboring pJEL134 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>As all strains used in this work are deoR + and the DeoR operator is not perturbed in any of the plasmid constructs , P-galactosidase expression from pJEL134 reflects only cAMP-CRP activation and CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>CytR and cAMP -- CRP regulation of rpoH -- lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>In this report we have addressed the question of how cAMP-CRP and CytR interact at deoP2 by isolating CRP mutants that severely decrease CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Results Binding of cAMP-CRP and CytR to wild-type deoP2 In order to test the importance of cAMP-CRP in negative regulation by CytR , we used nuclease protection experiments to probe specific interactions between these proteins and deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>We have undertaken a detailed study of the in vivo activity of the deoCp2 promoter to clarify the mechanism of CytR regulation and cAMP-CRP activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In order to test cAMP-CRP activation and CytR regulation in this mutant deoP2 promoter i '' Enzymelevels are measured during exponential-growth at 30 ° C in AB minimal medium using glycerol ( crp ÷ ) or glucose ( Acrp ) as carbon source as described by Miller ( Miller 1972 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>1 The abbreviations used are : CRP , cAMP receptor protein ; CRP1 and CRP2 , CRP operator sites 1 and 2 , respectively ; RNAP , E. coli RNA member of the LacI family of bacterial repressors , and is induced when CytR binds cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Transcription is repressed by the cytidine repressor protein ( CytR ) , a regulon-specific member of the 1 Abbreviations : RCTD , R-subunit of E. coli RNA polymerase ; bp , base pair ; cAMP , cyclic-5 ′ ,3 ′ - adenosine monophosphate ; CRP , E. coli cAMP receptor protein ; CRP1 and CRP2 , proximal and distal CRP operators , respectively ; CytO , CytR operator site ; CytR , E. coli cytidine repressor protein ; Core domain , CytR mutants with N-terminal , DNA binding domain deleted ; deoP2 , E. coli proximal deo promoter ; dNTP , 5 ′ - deoxyribonucleotide triphosphate ; udpP , E. coli uridine dephosphorylase promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Our results support the model in which repression normally involves the binding of CytR to cAMP-CRP to form a complex which binds to speciﬁc sites in the udp promoter , without direct interaction between CytR protein and a speciﬁc operator DNA sequence , and in which induction by speciﬁc inducer cytidine involves dissociation of CytR from cAMP-CRP and then RNA polymerase interaction with cAMP-CRP bound to a site upstream of the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Taking together the results presented in this study , we propose a simple mechanism to account for the repression by CytR : the cAMP ± CRP complex at the CRP1 site , upon interaction with CytR , undergoes allosteric changes such that its binding to the DNA site is improved but its interaction with a-NTD is no longer productive , but rather destructive , leading to transcription repression ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>These results strongly indicate that CytR can be directed to deoP2 in the absence of specific CytR-DNA interactions and , by implication , that the protein-protein interactions between CytR and CAMP-CRP play a key role in specifying the site of action of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The results of these experiments ( summarized in Figure 3 ) strongly point to the conclusions that CytRAS-48-mediated repression does not rely on the exact sequence in the CytR-binding site and that the specificity of CytR can be achieved solely through the protein-protein interactions to CAMP-CRP , whereas the CytR-DNA interactions may primarily serve to increase the stability of the repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>These data thus provide a picture of the CytRlcAMP-CRP regulatory system in which the site of action of CytR can be specified by the protein-protein interactions to CAMP-CRP and in which the CytR-DNA interactions may primarily contribute to CytR binding and repression by stabilizing the CytRl CAMP-CRPIDNA repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Moreover , the observation that CAMP-CRP recruits CytR to a deoP2 promoter with no CytR operator rules out the possibility that CAMP-CRP only acts by increasing the functional DNA-binding specificity of CytR as was recently shown to be the way the MCMl protein assists a2 binding in the MCMl-a2 system ( Smith and Johnson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>By introducing CRP proteins that exhibit an altered DNA binding specificity into a strain containing a mutant deoP2 promoter in which cAMP-CRP activation was decreased and CytR regulation completely abolished , we show that CytR regulation of this promoter can be reestablished by restored the DNA binding of the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Restored DNA-binding of the CAMP-CRP activator complex reestablishes negative regulation by the CytR repressor in the deoP2 promoter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>We have investigated in vivo the coupling between CytR regulation of the deoP2 promoter in Escherichia coli and the DNA-binding specificity of the cAMP-CRP ( cAMP receptor protein ) complex in order to obtain a more detailed picture of the role played by cAMP-CRP in CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>In one such promoter , deoP2 , CytR binds to the region between the tandem CRP binding sites with a relatively low affinity ; in the presence of cAMP-CRP , however , the repressor and activator bind cooperatively to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>The CytR repressor inhibits expression of the deoP2 promoter by binding cooperatively to deoP2 together with cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Here we show that CytR binding to the deoP2 promoter relies on interaction with the master gene regulatory protein , CRP , and , furthermore , that cAMP-CRP and CytR bind co-opera-tively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>three possibilities can be envisaged for how CytR interacts with deoP2 : ( i ) CytR binds to the DNA and directly contacts cAMP-CRP , i.e. the dependence of CytR on cAMP-CRP is mediated through protein-protein interactions ; ( ii ) cAMP-CRP binding to DNA induces a conformational change in the DNA which is specifically recognized by CytR , i.e. the dependence of CytR on cAMP-CRP is mediated through changes in DNA structure/conformation ; ( iii ) CytR does not contact DNA but sits on top of cAMP-CRP , i.e. a system closely resembling the Gal4/Gal80 system in yeast -LCB- Johnston etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Whereas inducer or co-repressor binding to other family members controls a switch between DNA-binding protein and non-DNA-binding protein conformations , cytidine binding to CytR affects only cooperative interactions between CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR is a co-activator when it recruits CRP to its DNA binding site ( s ) via cooperative binding to DNA , and it is a repressor when it is part of a three-protein complex that is stabilized by CytR-CRP cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The unusual location of the CytR operator in the deoP-2 regulatory region suggests that this repressor inhibits transcription by preventing the binding of the CRP protein or by changing the DNA conformation in such a way that CRP fails to function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The cAMP.CRP / CytR regulatory case has a remarkable functional relationship to several eukaryotic systems , e.g. in yeast the activator protein GRM is required for efficient DNA binding of the repressor protein a2 , and GRM can by itself bind toDNA and activate transcription ( Benderand Sprague , 1987 ; Keleher etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>The inducer of CytR perturbs protein-protein interactions to cAMP-CRP Cytidine , the inducer of the CytR regulon , does not affect individual CytR-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The inducer for the CytR repressor , cytidine , does not prevent in vitro DNA binding of CytR , but releases the repressor from the nucleoprotein complex and leaves the cAMP-CRP activator bound to its two DNA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The physiological importance of the cooperative DNA binding of cAMP-CRP and CytR to tSx-p2 is emphasized by the observation that cytidine , the inducer of CytR , does not interfere with independent DNA binding of the repressor protein ; rather , the cooperative binding of cAMP-CRP and CytR is interrupted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Furthermore , E. coli CytR proteins deleted from the DNA binding domain are able to repress transcription from CytR-regulated promoters , implying that the protein -- protein interactions between CytR and cAMP -- CRP play a key role in specifying the site of action of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Summary The Escherichia coli CytR regulator belongs to the LacI family of sequence-speciﬁc DNA-binding proteins and prevents CRP-mediated transcription in the CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In this model , induction of expression by cytidine was shown to involve dissociation of CytR from CRP complexes , which results in transcription activation , possibly through effects on DNA topology or direct interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>In this model , CytR ± CRP and CytR ± DNA interactions block transcription activation by CRP , by sterically blocking the functional AR1 of CRP and by sterically preventing a-CTD from interacting with the DNA segment immediately upstream of the CRP dimer at the CRP1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>Moreover , Thus , a model is suggestedinwhichthe DNA is the synergistic action of CRPand CytR relies on direct wrappedaround a complex of RNA polymerase and protein-protein interaction on the DNA helix as single amino one CRP.Finally , the resultsshow thatCytR and RNA acid substitutions intwo surface exposed loops of CRP abolish polymerase are rivals that compete for binding with regulation by CytR without altering CAMP-CRP activation CRP at deoP2 and that CytR functions as an antiactivator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Thus , DNA-bound CRP molecules engaged in nucleoprotein formation by CytR are no longer accessible for RNA polymerase , andsterichindrance could account for negative regulation of deoP2 by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>This indicates an interaction between CytR ∆ 2-63 and both DNA-bound CRP dimers , which is consistent with formation of a bridged complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>For CytR to bridge the gap between promoter-bound CRP dimers and juxtapose the interacting surfaces of CytR and CRP as delineated by mutagenesis experiments ( 39 ) , the promoter DNA must be bent substantially .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>One can imagine scenarios in which CytR remains bound to DNA , if not to CRP , under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CRP bound to this duplicate site seemed not to interact with CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>This would suggest that CytR is primarily a protein-protein bridge , dependent on its interactions with CRP bound to the flanking CRP sites for association with its operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR forms a protein bridge between the bound CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>But a three way cooperative complex also forms in which CytR forms a protein bridge between tandem , DNA-bound CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Thus , protein-protein communication between DNA-bound CytR and CRP underlies repression , co-activation , and induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This was interpreted to indicate that CytR mutants associate via bridging protein-protein interactions alone to form a functional , DNA-bound CRP-CytR-CRP repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Either competition between CytR and RNAP for interaction with CRP is the main mechanism of repression , with RNAP-DNA interactions being essentially ancillary , or cytidine-liganded CytR remains bound to the DNA in a conformation which prevents CytR from acting as an allosteric antiactivator of either or both of the flanking CRP molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus , repositioning of CytR could have a profound effect on RNAP-CRP-DNA interactions and therefore on the effectiveness of CytR-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>We study the interactions between DNA , CytR , and CRP proteins in the udpP region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>These results suggest strong interactions between CytR and CRP rather than between CytR and any particular DNA sequence within udpP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>This simplification has a small but still observable effect on the accuracy of the fits to the titrations when CytR and CRP bind DNA together , with one protein in the presence of a near saturating concentration of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>First , CytR and CRP bind cooperatively to form a three-protein complex on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Together , CytR and CRP bind co-opera-tively to their respective targets , and a nucleoprotein complex with protein stoichiometries of 1:2 is formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>n i induced DNA bends in the circular permutation restriction nuclease cleavag CAMP-CRP causes the largest variations in complex mobilities and , therefore , induces a greater DNA bend than CytR ( Thompson & Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>In the CRP-CytR repression complex , no change in the CRP-DNA contacts are evident neither at CRP-1 nor at CRP-2 as revealed byuranyl photofootprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8394345" target="_blank">8394345</a>]<br></li>
<li>Despite only weak association between CytR and CRP in solution , our results demonstrate the formation of a relatively stable complex in which the Core domain forms a protein bridge between tandem CRP dimers when bound to either udpP or deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In contrast to the low affinity for solution assembly of CytR and CRP , these two CytR Core domain constructs show appreciable affinity for bridging between tandem , DNA-bound CRP dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>On the basis of this value for ∆ Gobridge , the connection free energy that pertains to the cooperatively bound complex in which CytR both bridges between tandem DNA-bound CRP dimers and also binds CytO is easily calculated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Here we show that ( i ) CytR selectively regulated cAMP-CRP-dependent initiations , although transcription started from the same site in deoCp2 in the absence or presence of cAMP-CRP ; ( ii ) deletion of the uppermost cAMP-CRP target ( CRP-2 ) resulted in loss of CytR regulation , but had only a minor effect on positive control by the cAMP-CRP complex ; ( iui ) introduction of point mutations in either CRP target resulted in loss of CytR regulation ; and ( iv ) regulation by CytR of deletion mutants lacking CRP-2 could be specifically reestablished by increasing the intracellular concentration of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>In vivo analyses have shown that the uppermost CRP target ( CRP-2 ) is dispensable for cAMP-CRP activation , whereas a deletion of this site completely abolishes negative control by CytR , and point mutations in either CRP-1 or CRP-2 result in a partial loss of CytR regulation ( Valentin-Hansen , 1985 , Sogaard-Andersenefa / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1144">
This work and previous reports regarding CRP and CytR regulation show that E. coil uses activating and repressing modules in different stoichiometric and structural arrangements to differentially adjust the expression of genes belonging to the same regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></label>
<input id="act_similarity_1144", type="checkbox">
<div class="ACT_similar">
<li>In summary , this work conﬁrms previous reports on CRP and CytR regulation which show that bacteria use activating and repressing modules in different stoichiometric and structural arrangements to differentially adjust the expression of genes belonging to the same regulon [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li></div>
<li>Table 2 summarizes the expression data and show that the ability of the different CRP proteins to activate deoP2-wt varied in parallel with the efficiency of regulation by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Most importantly , our results clearly showed that CytR regulation of the mutant deoP2 promoter was reestablished in the presence of either of the altered CRP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>These results can be explained in a rigorous manner in light of our recent findings concerning regulation of the deoP2 promoter by cAMP-CRP and CytR and by taking the ability of the various CRP proteins to interact at the different CRP targets into consideration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Increasing the distance between CRP - 1 and CRP-2 by 2 bp in the wt promoter and in the promoter containing the point mutation at position -60 would be expected to result in decreased activation and increased CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Therefore , in cells containing the mutant deoP2 promoter , CRP-wt protein can only interact weakly with CRP-1 giving rise to weak activation , and no CytR regulation is obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>However , in strains containing altered CRP proteins that bind to the mutated CRP sites with a higher affinity than CRP-wt protein , CytR regulation was reestabl-ished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Thus , the strong reduction in promoter activity observed upon replacement of CRP-wt protein with either of the altered CRP proteins was parallelled by a reduction in the degree of CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>This decrease in efficiency of CytR regulation is not simply a consequence of the reduced promoter activ ity , as the promoter activity in the cytR ÷ strain was higher ( 1.0 ) in presence of the altered CRP proteins than in presence of CRP-wt protein ( 0.6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Strikingly , the effect on CytR regulation varies in the four promoters tested ( Table 2 ) : all CRP mutants almost completely abolish CytR regulation in tsx , whereas a low residual CytR regulation is still present in deoP2 and cdd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>These results , therefore , confirm our location of the cdd-promoter and suggest that all sequences necessary for proper regulation of this promoter by CytR and CRP are present in the Hga \ fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>To further characterize the function of CRP-1 in CytR regulation , we tested whether it was possible to reestablish CytR regulation in a deoCp2 promoter lacking CRP-2 by increasing the intracellular concentration of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Introduction of a point mutation in CRP-1 results in partially constitutive expression of deoCp2 ; furthermore , CytR regulation can be reestablished in decCp2 derivatives lacking CRP-2 by increasing the intracellular concentration of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Although we can not yet account for the effect of this amino acid substitution on activation , the salient point here is that amino acid substitutions in CRP can abolish CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>A direct connection between CytR regulation and the presence of intact tandem CRP targets in deoCp2 ( Fig. 1 ) was evident from analyses of deletion and point mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>i.e. deletion of the CRP-2 site , abolishes CytR regulation and point mutations in this target results in partial loss of CytR regulation ( ValentinHansen , 1985 ; Martinussen ef al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>In contrast , the tsx-500 mutation , which is located outside of the two CRP targets , affects only CytR regulation of tSx-p2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>A key to CytR-mediated regulation is that despite roles as functional antagonists , CytR and CRP bind with positive cooperativity to their respective operators ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To analyze the mechanism ( s ) involved in this combinatorial regulation , we endeavored to isolate CRP mutants that could not support CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Hence , both CRP targets are involved in and required for efficient CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Central to this process and to the differential regulation of expression are the interactions between CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>These studies have also identified competition between CytR and CRP as an additional contributor to differential regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Thus , in addition to the tandem CRP targets , the distance between the two targets seems to be an important factor for CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Mutations in the CRP-1 and CRP-2 targets Transition mutations at positions - 8 9 and - 9 8 in CRP-2 result in a five-fold decrease in CytR regulation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Here this cooperativity isolating investigated by we have mutants of the CRP protein that abolish CytR regulation without exhib-iting a concomitant loss in their ability to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>To test this idea , we have investigated CRP and CytR regulation of transcription initiation at deoP2 with reconstituted wild-type RNP and RNP derivatives lacking a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>In all cases , these studies have implicated interactions of CRP and CytR with sequences located in the 80 -- 100 bp immediately upstream of the various transcription start sites and interactions between the proteins as the basis for positive and negative gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Arrows in panel C indicate sites of IS insertions and the new transcriptional start in ins1 ( position 264 , the same as in del18 ) ; newly created 235 and 210 regions are underlined DISCUSSION We used biochemical and mutational tests to better understand how CRP and CytR repressor interact with each other and target DNA to result in the regulation of transcription of the udp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>This interpretation implies an affinity for CytR bridging that is much higher than what might be expected based on the cooperativity when full-length CytR binds CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>This comparison suggests a high affinity for CytR bridging alone , which must be counterbalanced by another , thermodynamically unfavorable process when CytR both bridges and binds CytO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR should function as an RNAP antagonist by competing with this At low CytR concentration , increasing cAMP concentration directs a two-stage activation : first , of class I as CRP2 fills and second , of class II as CRP1 subsequently fills .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The primary function of CytR may be differential modulation of CRP1-mediated versus CRP2-medi-ated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Based on these findings we propose that CytR functions as a differential modulator of CRP1 versus CRP2-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The effect of competition for CRP2 is to negate the role of CytR as a co-activator for CRP2-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , CytR should function as a co-activator of CRP1-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>However , CytR also binds to separate , specific sites at deoP2 , one of which overlaps CRP1 , and another that overlaps CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The results indicate that CytR binds specifically to multiple sites at deoP2 , including both the well recognized CytR site flanked by CRP1 and CRP2 and also sites coincident with CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The resulting parameters do not describe the data well for CRP2 and CRP1 in the presence and absence of CytR , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The CytR protein binds with low affinity to the DNA between the CRP1 and CRP2 targets , protecting the region from - 8 0 to - 5 0 ( Figs 2 and 3 , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The salient features of this model are that CytR binds separately to CytO and to a site at CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Addition of either 3 or 10 µM CytR ∆ 2-63 increased the fraction of CRP2-deoP2 and decreased both CRP1-deoP2 and to a lesser extent , unliganded deoP2 ( Figure 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR titrations of CRP12 and CRP22 were conducted to evaluate whether there was any effect of mutating either CRP1 or CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In either case , the new RNAP-DNA interaction would be in the sequence between CRP2 and CRP1 , precisely where CytR binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>First , estimated affinities for the hypothesized additional CytR sites match the CytR concentration dependence of the DNase I protection in CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>It is evident by inspection that protection occurs first , i.e. at the lowest CytR concentrations , in the intervening DNA where CytO is located ; only at higher CytR concentration is comparable protection observed at CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Direct observation contradicts this ; protection by CytR of DNA sequences overlapping CRP1 and CRP2 occurs at lower CytR concentration than elsewhere on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">CytR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>LmRNA-P `` R.B. Val Lys ... The CytR repressor contains , near its amino terminus , the helix-turn-hetix motif found in many regutatory proteins ( Vatentin-Hansen et ai , 1986a ; Brennan and Matthews , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Moreover , Lys-20 was changed to glycine because it might be argued that this substitution would improve CytR : one of the most conserved residues in HTH motifs is the glycine in the first position in the turn between the helices ( Brennan and Mat-thews , 1989 ) , and , most likely , this residue participate in maintaining the correct angular orientation of the two heli-ces in the motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>As can be seen , two of the residues ( Ala-7 and Val-17 ) are also present in CytR , whereas this protein contains lysine instead of glycine at position 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The amino acids involved in dimerization of CytR have been determined genetically ( Barbier and Short , 1993 ) , and correspond well with amino acids involved in dimerization of other LacI members ( Schumacher et al. , 1994 ; Lewis et al. , 1996 ; Friedman et al. , 1995 ; for review , see Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Furthermore , the high sequence homology among the HTH motifs of LacI members ( 60 % amino acid identity among PurR and CytR HTHs ) , as well as between their DNA operator half-sites ( Schumacher et al. , 1994 ) , strongly suggests that the HTH motifs take very similar structures , and that they interact with their half-operators in very similar ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CytR has two features that are not observed in other LacI family members and that appear to be important to its role as a differential transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>( IV ) Finally , the CytR regulon may contain additional , yet undiscovered members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>This mutant retains the lysine-rich N-terminal nine residues of CytR , capping a 14-residue hinge peptide that is thought to be largely unstructured ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The basis for this prediction is the observation that CytR has the helix breakers proline ( Pro57 ) and glycine ( Gly59 ) , whereas the other family members have conserved helix maker alanines in the hinge helix ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR shares a conserved lysine residue in the hinge peptide that constitutes the preferred trypsin cleavage site for these other LacR family members ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Interaction of CytR with the octamer motifs AATGTAAC and GTTGCATT invariably protects the central guanine from DMS methylation , consistent with the well conserved G at this position ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>On the one hand , this architecture provides the HTH motifs of CytR with great freedom of movement and allows them to interact with half-operators in different orientations and with different lengths of spacer regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>For example , while extended protection patterns in nupG ( both DNase I and chemical modification reagents ) have been interpreted as resulting from CytR binding to two perfect TGCAA motifs separated by an 11-bp spacer ( 11 ) , this overlooks several additional CytR recognition motifs , including one located in the middle of the proposed 11-bp spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>First , whereas both promoters contain CytR recognition motifs that overlap CRP2 , the udp promoter lacks CytR recognition motifs overlapping CRP1 and downstream to the 210 promoter element , which are present in deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR-regulated promoters differ from one another both in spacing between CytR half recognition motifs and in the location of CytO with respect to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR is also unusually tolerant of variation in spacing between recognition motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR recognition motifs are shown as shaded boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Trypsinolysis of CytR results almost exclusively in cleavage at this residue , K63 , and the neighboring residue , R64 ; N-terminal sequencing indicated approximately an equimolar mixture of the two .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Taken together , these results clearly demonstrate the importance of two distinct domains in cddP for CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Thus , all the substitutions support the notion that the NH,-terminal end of CytR contains a functional HTH motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Using repressor protein speciﬁcally labelled with 32P in the N-terminal end , limited proteolysis has been used to dissect CytR into protein fragments corresponding to domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>It seems likely , therefore , that the interdomain linker serves a different function in CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>This suggests that CytR 's linker arm is not completely devoid of secondary structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In addition , it is possible that the interdomain linkers of the dimeric CytR may be loosely tethered to one another or to the core domain when interacting to adjacent half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>First , mutations have been located on the surface of CRP in a putative protein interacting domain that interfere with cooperativity with CytR and with CytR-mediated repression ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR simultaneously makes specific DNA contacts as well as protein-protein contacts to cAMP-CRP in the repression complexes ( Pedersen et ai , 1991 ; Sogaard-Andersen and ValentinHansen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Based on these results , we propose that CytR in the CytR/cAMP - CRP/deoP2 complex is engaged both in specific DNA contacts andprotein-protein contacts to CAMPC.RP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_28">
What structural information allows CytR to interact with DNA in such a relaxed manner ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></label>
<input id="dom_similarity_28", type="checkbox">
<div class="DOM_similar">
<li>Thus , what structural information allows CytR to interact with DNA in such a relaxed manner ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>Although the ribose binding protein RbsB also shows similarity with the signal -- receptor domains of the regulatory proteins in the regulatory Group 4 such as MalI , PurR and CytR , they are not directly in a regulator -- regulon relationship .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>The present study addressed three related problems : structure of CytR-binding sites and motifs , their evolutionary conservation , and identification of new members of the CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>As expected , based on this quarternary structure , CytR binding sites contain tandem recognition motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Do these motifs in udp constitute separate , specific CytR binding sites ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Most prominently , the udp promoter contains no putative CytR binding site that occludes CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>A single CytR recognition motif within CRP2 might also constitute a weak binding site , analogous to the situation in deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Mutational analysis of E. coli udpP promoter revealed that the pentameric motif TGCA played a key role in binding the CytR repressor to udpPEc [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Firstly , from the analysis of udpP it has ′ Ec been proposed that the pentamer 5 - TGCAA plays an important role in sequence-speciﬁc DNA-binding of CytR [ 5,38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Third , a truncated CytR which is lacking the DNA-binding helix-turn-helix motif was reported to bind to 2 C. S. Barbier , S. A. Short , and D. F. Senear , submitted for publication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>A comparison of the GntR primar structure with those of the repressors in the CytR family revealed that GntR also has a helix-turn-helix motif at its N terminus , which may be responsible for DNA-binding and it may also have domains related to dimerization and inducer-binding at its C terminus ( Weickert & Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The CytR repressor belongs to the LacI family of regulators , and possesses , like the other members , an N-terminal helix -- turn -- helix ( HTH ) DNA-binding motif ( Valentin-Hansen et al. , 1986 ; Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Structural studies of many DNA-binding proteins have revealed several DNA-binding motifs ( for review see Steitz , 1990 ) ; however , CytR only shares homology with the a helix-turn-a helix ( HTH ) motif ( Valentin-Hansen et al. , 1986 ; for review see Brennan and Matthews , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>CytR has extensive homology with known DNA-binding proteins like LacI , GalR , PurR , and MalI ( 22 , 24 , 33 ) , and CytR contains a region exhibiting a high level of homology to the a-helix-turn-a-helix motif found in the DNA-binding domain of several DNA-binding proteins ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>CytR shares extensive amino acid sequence homology with known DNA-binding proteins like GalR , PurR , MalI , and the lac repressor ( Valentin-Hansen et al. , 1986 ; Rolfes and Zalkin , 1988 ; Reid1 et al. , 1989 ) , and specifically , CytR contains a region exhibiting a high level of homology to the helix-turn-helix motif found in the DNA-binding domain of several DNA-binding proteins ( Pabo and Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Specifically , the amino-terminal region of CytR is predic-ted to form an ( ( - helix-turn-a-helix motif charactenstic of some DNA-binding proteins ( Pabo and Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Thus , despite the fact that the CytR repressor exhibits a high level of homology to other bacterial repressors and probably possesses , near its amino terminus , a helix-turn-helix DNA-binding motif ( Pabo and Sauer , 1984 ; Valentin-Hansenefa / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Nevertheless , a structural model has been proposed in which the DNA is wrapped smoothly around the three-protein complex , and the two CytR DNA-binding domains ( one per subunit ) bind to DNA sequences that are arranged centrosymmetrically and are separated by 11 base pairs ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>This would afford the DNA-binding domains of CytR with considerable freedom of movement , and allow them to interact with half-operators separated by as little as 1 bp or as much as 13 bp , contact half-operators on opposite faces of the DNA helix , or ﬂip 180 ° in order to interact with repeats in either orientation ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>It has even been suggested that the helix-turn-helix , DNA-binding feet of CytR might be able to turn 180 ° to accommodate binding to recognition motifs arranged as either tandem or inverted repeats ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>As illustrated in Figure 6B and C , these DNA arrangements are expected to bind either a roughly symmetrical conformation of the CytR dimer in which the DNA-binding domains are held in an inverted orientation with their recognition centers approximately two DNA helical turns apart , or an asymmetric conformation in which the DNA-binding motifs are in direct orientation , with their centers approximately one helical turn apart .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Clearly , CytR can not bind the C-sequences ( or the natural targets ) in a similar binding mode , since the hinge helices would be pulled too far apart when the HTH motifs interact with direct repeats , or inverted repeats separated by one DNA helical turn ( Figure 6B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Based on the PurR -- DNA co-crystal structure ( Schumacher et al. , 1994 ) , and three crystal structures of LacI ( as free protein and in complex with operator DNA or inducer ; Lewis et al. , 1996 ) , CytR is expected to consist of an N-terminal DNA-binding domain of ~ 60 amino acids , connected via a hinge region to a ~ 270 amino acid C-terminal domain that mediates dimerization and ligand binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Based on the PurR -- DNA co-crystal structure ( Schumacher et al. , 1994 ) and the crystal structures of LacI , as free protein and in complex with operator DNA ( Lewis et al. , 1996 ) , CytR is expec-ted to consist of two functional domains : an N-terminal DNA-binding domain ( residues 1 -- 51 ) connected via an interdomain linker ( residues 52 -- 68 ) to a larger C-terminal domain ( residues 69 -- 341 ) that mediates dimerization and effector-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>It has not bee shown directly how these substitutions interfere with CytR activity ; however , they all involve residues that would be predicted either to have important functions in maintaining the structure of the DNA-binding domain ( Val-15 , Thr40 , and Tyr-53 ) or to be engaged in contacts to the DNA ( Ala-24 and Ser-27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>This highly flexible connection between the DNA-binding domains and the ligand-binding core dimer is thought to explain how CytR can tolerate widely variable spacing between DNA-recognition motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>These results are discussed in relation to the current structural model of CytR in which the core , inducer-binding domain is tethered to the he-lix-turn-helix , DNA-binding domain via flexible peptide linkers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To explain how CytR uncouples ligand binding from DNA-binding , CytR is thought to connect the helix-turn-helix motifs to the globular core domain via peptide linkers in an extended conformation , rather than via hinge helices ( 3 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>If so , elimination of CytR-CRP interactions in response to cytidine binding might allow the CRP-RNAP interaction necessary for full activation even in the presence of DNA-bound CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Presumably , binding to this particular setting of half-sites provokes a conformational change in the large C-terminal domain of CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>It has been argued previously that these processes are uncoupled in CytR because the DNA-binding and allosteric Core domains are connected by an unstructured peptide , providing a flexible tether that isolates the DNA-binding domains from structural changes that might occur in the cytidine-binding Core domain ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To evaluate the energetics of the protein bridging reactions in the absence of CytR-DNA interaction , we expressed and purified CytR deletion mutants that lack the entire DNA-binding HTH domain but that , as we show here , retain the functions of the allosteric , inducer-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The fact that CytR Core domain associates to form a bridged complex confirms that such protein-protein interactions underlie cooperative DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR Core domain associates when CRP is bound to both CRP1 and CRP2 as a result of favorably oriented protein-protein contacts , despite no interaction between Core domain and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Our results establish that neither the CytR-binding site nor the DNA-binding domain of CytR is required to obtain repression in vivo or CytRlcAMP-CRPIDNA nucleoprotein complex formation in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Our results show that CytR possesses , in the free state as well as in the DNA-bound state , a protease-hypersensitive interdomain linker that joins the N-terminal DNA-binding domain to the C-terminal domain mediating effector-binding and oli-gomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>This suggests that there are contacts between the DNA-binding domain of CytR ( the ` toes ' of CytR ; Figure 6B and C ) and CRP near the DNA , besides the interaction between the ligandbinding/dimerization domain of CytR and CRP demonstrated previously ( Søgaard-Andersen and ValentinHansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The region in CAMP-CRP that contacts CytR has been identified and consists of two surface-exposed loops that are in close proximity and physically separated from the DNA-binding domain in CAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>These considerations together with a possible steric clash between DNA and protein may also explain why CytR-Ala-24 + Glu represses deoP2 less efficiently than CytR proteins deleted for the DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>However , when overexpressed , a mutant CytR protein lacking its DNA-binding domain can repress the deoP2 promoter in vivo ( Søgaard-Andersen and Valentin-Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Furthermore , CytR proteins deleted for the DNA-binding domain repress deoP2 in vivo and interact with deoP2 in vitro in a strictly CAMP-CRP-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Second , CytR proteins deleted for the DNA-binding domain are able to repress transcription from deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>First , CytR might have substantial structural flexibility as might result if its DNA-binding head and inducer-binding domain are only loosely tethered to one another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Additionally , the data suggest that CytR has a modular structure with an NHr terminal DNA-binding domain and a COOH-terminal domain engaged in CAMP-CRP interactions , inducer-binding , and oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>But observations made several years ago using truncation mutants of CytR that lack the N-terminal DNA-binding domain suggest that CytR-CRP and CytRCytO interactions may not combine quite so favorably as this .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Deletions of the N-terminal , DNA-binding domain of CytR were chosen to mimic the functional , ligand-binding Core domains that result from limited trypsinolysis of LacR ( 33 ) and PurR ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Next , we constructed a set of CytR derivatives in which v #arious parts of the DNA-binding domain have been de-leted ( Figure 26 ; Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>As CytR has a 6 residue NHn-terminal extension , the DNA-binding domain of CytR includes the NHP-terminal 57 residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>It was shown that the DNA-binding domain of the E. coli CytR protein includes the N-terminal 57 residues [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Alternatively , CRP-induced distortion of the DNA helix at the edges of the CRP-binding sites may present an optimal target for a CytR DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Thus , there appears to be no special feature in the DNA-binding domain of CytR that could account for CytR 's DNA-binding ﬂexibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Therefore , it was most unexpected that the CytR operator as well as the DNA-binding domain of CytR was dispensable for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>This supports the idea that residues other than glycine might be accomodated at this important position and that amino acids 10-32 of CytR most likely are involved in the DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>One segment located in the N-terminal part of CytR resembles protein-DNA contact points conserved in other known DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>However , this region of CytR is unusual in that it contains lysine instead of glycine in one of three highly conserved positions among DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>Only one region in CytR starting 10 residues from the N-terminal , shows strong homology to the DNA -- binding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>To explain these observations , we propose that the DNA-binding domains of CytR are tethered to the rest of the protein through a flexible linker that allows the repressor to interact with half-operators separated by different spacings , and to interact with cAMP-CRP complexes at various distances from the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>DNA-binding characteristics of the Escher-ichia coli CytR regulator : a relaxed spacing requirement between operator half-sites is provided by a ﬂexible , unstructured interdomain linker .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491603" target="_blank">15491603</a>]<br></li>
<li>Furthermore , limited proteolysis of radiolabelled CytR protein showed that the interdomain linker connecting the DNA-binding domains and the core part of CytR does not become structured upon DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>This implies that CytR has a disordered and mobile interdomain linker that effectively uncouples the DNA-binding domains from constraints imposed by dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In conclusion , our data support an architecture of CytR in which a loosely structured and ﬂexible interdomain linker effectively uncouples the DNA-binding domains from constraints imposed by dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>44 Alternatively , as has been shown for other transcription regulators , like CytR , TyrR and AraC,44-48 FlhC could exhibit particular flexibility in the region between its potential DNA-binding and dimerisation domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144778" target="_blank">12144778</a>]<br></li>
<li>It thus seems unlikely that CytR should possess alternative dimerization interfaces between which the subunits could slide to allow the DNA-binding domains to recognize differently spaced half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Since the putative DNA-binding domains of CytR and GalR share extensive homology and the subunits of the two repressors are of equal size , it seemed of interest to examine the two poly-peptides for further homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The apparent homology between the putative DNA-binding domains of GalR and CytR suggests that their sites of action may be related .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The individual interactions will combine favorably when the conformation of the DNA-binding and allosteric domains of CytR matche the arrangement or operators at a particular promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>We also examined the CytR polypeptide for regions similar to the concensus sequence for the DNA-binding domains of DNA -- binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>Therefore , it is probable that it is the CytR core protein , not the DNA-binding domains of CytR , that interacts with CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Thus , the cooperative binding of the two DNA-binding domains of CytR is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>If cytidine does not affect CytR-CRP association per se , then does it affect simultaneous interaction of CytR with two CRP dimers to form a protein-protein bridge ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR repressor-cytidine interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>However , it was observed that CytR functions with the RNA polymerase devoid of the C-terminal domain of the a-subunit as well as with intact RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The similarit between what was observed for Core domain and full-length CytR suggests as another possibility to explain some of the difference between observed and expected molecular weights that the calculation of Vj from the amino acid composition might not be accurate in this case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>These results suggest that CytR Core domain is homogeneous dimer in the µM concentration range , but do not eliminate the possibility of a very small fraction of lower molecular weight material ( 27 ) Cytidine Binding by Full-Length CytR and Core Domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Since the nitrocellulose filters retain proteins with significantly less than 100 % efficiency , this method can provide only relative estimates of CytR or Core domain fractional activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>We interpret this additional protection as reflecting bridged complex formation , which offers the underlying DNA partial protection form DNase I mediated cleavage even in the absence of direct physical contact between CytR Core domain and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The open symbols show systematic decreases in θ0 and θ1 and a systematic increase in θ2 when CytR Core domain dimer is added at ca. 3.5 and 10 µM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The dashed lines are calculated to show the effect of addition of 3.5 µM ( short dashes ) and 10 µM ( long dashes ) CytR Core domain , assuming ∆ G ) -7 kcal/mol for association of a Core domain dimer with the CRP2-deoP2 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Construction of Gene Encoding CytR Core Domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Expression and Purification of CytR Core Domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR Core Domain ActiVity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>We first employed a gel mobility shift assay to evaluate the interactions between the CytR Core domain constructs and the CRP2-promoter complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Configuration 5 pertains to the complex formed by association of CytR Core domain in which ∆ G represents the free energy change for bridging CytR-CRP bridge interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The solid curves fit the points obtained in the absence of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Core domain containing fractions were pooled , and the NaCl was reduced to 0.15 M by dilution with salt-free CytR Buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Core domain loaded in CytR Buffer at 0.075 M NaCl elutes in the void volume .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR Core domain was expressed using E. coli strain SS6142 transformed with either pNN001 or pNN002 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Binding reaction mixtures contained between 0.25 and 0.4 µM of either full-length CytR or Core domain dimer and 3-3 .5 µM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>CytR Core domain dimer concentrations used in individual experiments ranged from 0.5 to 14.5 µM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>A crystallographic study of the effect of cytidine binding on CytR core domain conformation should provide a direct approach to address these issues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To address this question experimentally , matched CRP binding titrations of both deoP2 and udpP were conducted with and without CytR Core domain , as before ( Figure 6 ) , but now also comparing the presence and absence of 1 mM cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>To address this question requires CytR Core domain in which the interaction between CytR and DNA is eliminated but other CytR functions ( i.e. , dimerization , cytidine binding , and CRP association ) are unperturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Consistent with the interpretation that this reflects saturable binding of CytR Core domain , the extent of protection decreases with decreasing concentration of Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Filterbinding was also used for equilibrium measurements of cytidine binding to CytR and Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>FIGURE 2 : Filter binding assay comparing cytidine binding to wildtype CytR and CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Second , cytidine binding by Core domain and full-length CytR is similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>The free energy change for the association of CytR Core domain binding with the CRP2-promoter complex ( ∆ Gbridge ) was determined from paired experiments in which the promoter was titrated with CRP in the presence or absence of a fixed concentration of CytR Core domain ( 0.5-15 µM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Individual-site binding isotherms were generated by footprint titration of CRP binding in the presence ( panels B and D ) and absence ( panels A and C ) of fixed concentrations of CytR Core domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>DNase I footprint analysis was used to investigate the binding of CytR Core domain proteins to CRP2-deoP2 and to CRP2-udpP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>establish that ∆ Gbridge is the same for association of CytR Core domain with udpP as with deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>In particular , CytR ∆ 2-63 results in net binding o represent the predicted populations of intermediates in the presence of 3 or 10 µM Core domain if ∆ G ≈ -7 kcal/mol for association of Core domain with CRP2-deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Panels A and B represent paired titrations to show the effect of 4.0 µM CytR ∆ 2-63 Core domain dimer on CRP binding to udpP ; panels C and D represent paired titrations to show the effect of 13.1 µM CytR ∆ 2-63 Core domain dimer on CRP binding to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Residues in CytR which are found in identical positions in the other repressors are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>As shown in Fig. 3 , the corresponding region of udpPSt contained a very similar putative CytR binding site , while neither udpPYp nor udpPV c contained inverted repeats or a TGCA motif between positions − 71 / − 67 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The mutation , located in the protected region downstream from CRP1 ( Fig. 10 ) , is in a CytR recognition motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In a previous study , speciﬁc cleavage of PurR was observed after R52 , whereas neighbouring residues K41 and K55 were inaccessible ( corresponding to CytR residues R60 , R49 and K63 ; see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>* In order to save space , the N-terminal four residues of MalI and six residues of CytR are omitted as are the C-terminal seven residues of LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>In neither role does CytR function alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In the presence of CytR alone , a specific DNase I footprint is detected in all four mutant promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The putative HTH motif in CytR extends from residues 12-32 ( Figure 2A ) , and the putative recognition helix has five potentially surface-exposed resi-dues ( Thr-23 , Ala-24 , Ser-27 , Arg-28 , and Met-31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The guanine residues protected by CAMP-CRP and CytR in the DMS footprints are indicated by open arrows , and the residues with enhanced reactivity in the CytR-binding site are indicated by closed arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The extents of the regions protected by CAMP-CRP and CytR in DNAase I footprints are indicated above the helices ; the deoxyriboses protected from attack by the hydroxyl radical are indicated by closed circles ( CAMP-CRP ) and closed triangles ( CytR ) ; the residues with decreased reactivity in the DMS footprint are indicated by open arrows , and residues with enhanced reactivity are indicated by closed arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>All these residues map in or immediately outside the CytR operator ( Figure 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The coordinates are with respect to the first amino acid residue in CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The first and the last residues deleted are indicated for the truncated CytR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The subunit of CytR is predicted to be 341 or 343 residues in length .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The subunits of CytR are thus predicted to consist of 341 or 343 residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">CytR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Citation : Sernova NV , Gelfand MS ( 2012 ) Comparative Genomics of CytR , an Unusual Member of the LacI Family of Transcription Factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>It should be noted that V. cholerae CytR exhibits a high level of homology throughout the protein ( 81 % similarity and 65 % identity ) with the CytR protein of E. coli but contains deletion of the ﬁrst 8 amino acids in the N-terminal region of protein [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>When the amino acid sequences were appropriately aligned , MalI showed 28 % identity to GalR , 21 % to CytR , and 24 % to LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>When small gaps were introduced for optimal alignment , the MalI amino acid sequence was identical to that of GalR at 89 positions ( 28 % ) , CytR at 69 positions ( 21 % ) , Lacd at 78 positions ( 24 % ) , PurR at 61 positions ( 19 % ) , and RbtR at 62 positions ( 20 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>It can be seen that CytR in this region is more closely related to GalR , DeoR and LacR than to the other proteins , showing the highest homology to GalR ( 70 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>is highly homologous to the repressor proteins GaIR , CytR and Lad .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>The deduced amino acid sequence of MalI is highly homologous to that of the classical E. coli repressor proteins Lacd , GalR , and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>MalI exhibited high homology to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Sequence homology of MalI to the classical repressor proteins GaIR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Furthermore , the high sequence homology among the HTH motifs of LacI members ( 60 % amino acid identity among PurR and CytR HTHs ) , as well as between their DNA operator half-sites ( Schumacher et al. , 1994 ) , strongly suggests that the HTH motifs take very similar structures , and that they interact with their half-operators in very similar ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CytR shares extensive amino acid sequence homology with known DNA-binding proteins like GalR , PurR , MalI , and the lac repressor ( Valentin-Hansen et al. , 1986 ; Rolfes and Zalkin , 1988 ; Reid1 et al. , 1989 ) , and specifically , CytR contains a region exhibiting a high level of homology to the helix-turn-helix motif found in the DNA-binding domain of several DNA-binding proteins ( Pabo and Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Thus , despite the fact that the CytR repressor exhibits a high level of homology to other bacterial repressors and probably possesses , near its amino-terminus , a helix-turn-helix DNA-binding motif ( Pabo and Sauer , 1984 ; Valentin-Hansenefa / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Approximately 30 % of total protein in the extract is CytR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_14">
Pedersen , H. , and Valentin-Hansen , P. ( 1997 ) Protein-induced ﬁt : the CRP activator protein changes sequencespeciﬁc DNA recognition by the CytR repressor , a highly ﬂexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></label>
<input id="evo_similarity_14", type="checkbox">
<div class="EVO_similar">
<li>Pedersen , H. , and Valentin-Hansen , P. ( 1997 ) Protein-induced ﬁt : CRP activator protein changes sequencespeciﬁc DNA recognition by the CytR repressor , a highly ﬂexible LacI member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li></div>
<li>ORF1 encodes a protein homologous to repressors including CytR , GalR , and LacI ( Weickert & Adhya , 1992 ) as shown in Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>Valentin-Hansen P , Søgaard-Andersen L & Pedersen H ( 1996 ) A ﬂexible partnership : the CytR anti-activator and the cAMP ¬ Weickert M & Adhya S ( 1992 ) A family of bacterial regulators homologous to Gal and Lac repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>The basis for this prediction is the observation that CytR has the helix breakers proline ( Pro57 ) and glycine ( Gly59 ) , whereas the other family members have conserved helix maker alanines in the hinge helix ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CytR shares a conserved lysine residue in the hinge peptide that constitutes the preferred trypsin cleavage site for these other LacR family members ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Unlike the other members of this protein family , CytR lacks the ability to locate its operators with a sufficiently high affinity to repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Unlike the other members of this protein family , CytR binds with only modest afﬁnity to its operators and transcription repression thus relies on the formation of nucleoprotein complexes with the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>CytR has two features that are not observed in other LacI family members and that appear to be important to its role as a differential transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>There are , however , a number of features that set CytR apart from the other members of this family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Two of the other members of this family , PurR and CytR , are negatively autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>However , analysis under denaturing conditions on SDS-polyacrylamide gels indicates variable proteolysis of the CytR by endogenous proteases ( 5 ) , similar to what has been observed for other members of the LacI repressor family ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Limited proteolysis of radiolabelled CytR protein The different members of the LacI family show strong sequence similarity and are expected to take up a very similar three-dimensional structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>RbsR is a member of a repressor family including GalR , Lacl , Mali , CytR , and PurR [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>PurR is a member of the lac repressor family , which includes LacI ( 4 , 9 ) , GalR ( 46 ) , RbsR ( 25 ) , MalI ( 36 ) , RafR ( 3 ) , RbtR ( 50 ) , CytR ( 44 ) , EbgR ( 43 ) , CcpA ( 13 ) , and FruR ( 16 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li>
<li>CytR belongs to the LacI repressor family and exhibits extensive amino acid sequence homology to several other members ( e.g. PurR , LacI and GalR ; Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>First , CytR is member of the LacI/GalR family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Summary The Escherichia coli CytR regulator belongs to the LacI family of sequence-speciﬁc DNA-binding proteins and prevents CRP-mediated transcription in the CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Whereas inducer or co-repressor binding to other family members controls a switch between DNA-binding protein and non-DNA-binding protein conformations , cytidine binding to CytR affects only cooperative interactions between CytR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Second , CytR exhibits lower DNA binding specificity than other LacI family members and most other bacterial repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Like other LacI family members , the basic DNA binding unit of CytR is a homodimer ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>While regulatory ligand binding to most LacI family proteins is coupled intrinsic DNA-binding affinity , in CytR , cytidine binding is coupled to protein-protein cooperativity and not to intrinsic protein-DNA-binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CytR is unique among LacI/GalR family members in that binding of its regulatory ligand , cytidine , is linked not to DNA-binding affinity but instead to the cooperative interactions between CytR bound at CytO and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>A comparison of the GntR primar structure with those of the repressors in the CytR family revealed that GntR also has a helix-turn-helix motif at its N-terminus , which may be responsible for DNA binding and it may also have domains related to dimerization and inducer binding at its C-terminus ( Weickert & Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The CytR repressor belongs to the LacI family of regulators , and possesses , like the other members , an N-terminal helix -- turn -- helix ( HTH ) DNA-binding motif ( Valentin-Hansen et al. , 1986 ; Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>CytR is a sequence-specific DNA-binding protein and a member of the LacllGalR family of sequence-specific DNA-binding proteins ( Pedersen et al. , 1991 ; ValentinHansen et al. , 1986 ; for review see Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by CytR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Gene/operon Function Regulator FIS effect Repressor of glpR Repressor of g/pR g W K glpTQ Transport and assimilation of glycerol GlpR Transport and assimilation of GlpR phospholipids Deamination of cytidine to uridine CytR cdd Repressor of cyfR Activator of cdd Repressor of cflR Activator of udp Repressor of cytR Repressor of deoR Activator of deoCABD Repressor of mtlADR Activator of rbsR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Indeed , many regulatory proteins , such as LuxR , CytR , and PurR are auto-regulated , indicating that tight control of global regulators is necessary for normal cellular functions ( Roy et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Comparative Genomics of CytR , an Unusual Member of the LacI Family of Transcription Factors Natalia V. Sernova , Mikhail S. Gelfand * 1 1,2 1 A.A.Kharkevich Institute for Information Transmission Problems , Russian Academy of Sciences ( IITP RAS ) , Moscow , Russia , 2 Faculty of Bioengineering and Bioinformatics , M.V.Lomonosov Moscow State University , Moscow , Russia Abstract CytR is a transcription regulator from the LacI family , present in some gamma-proteobacteria including Escherichia coli and known not only for its cellular role , control of transport and utilization of nucleosides , but for a number of unusual structural properties .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>* E-mail : gelfand@iitp.ru Introduction CytR , the regulator of transport and utilization of nucleosides , was first mentioned in 1975 [ 1 ] and identified in 1985 [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>Regulation of the gcv operon by GcvA and GcvR is part of a growing number of prokaryotic regulatory systems involving an activator -RCB- antiactivator complex , including TraR -RCB- TraM regulation of conjugal transfer of the Ti plasmid in Agrobacterium tumefaciens ( Hwang et al. , 1999 ) , NifA -RCB- NifL regulation of nitrogen ﬁxation in Klebsiella pneumoniae ( Narberhaus et al. , 1995 ) , ComK -RCB- MecA -RCB- ClpC regulation of competence in Ba-cillus subtilis ( Turgay et al. , 1998 ) and CRP -RCB- CytR regulation of operons encoding the nucleoside-transport and biosynthesis enzymes ( Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>To test whether the CytR protein influences the expression of its own structural gene , we transformed the multicopy ptasmid pCBQ08 ( cytR ^ ) and the vector pBR322 into strain GP103 carrying the chromosomat ' ^ ( cytR-iacZ ^ ) ^ operon fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Others that have been investigated include tsx ( 7 ) , encoding an outer membrane protein , cdd ( 8 ) , encoding cytidine deaminase , udp ( 9 ) encoding uridine phosphorylase , cytr ( 10 ) encoding CytR , and most recently nupG ( 11 ) encoding a membrane nucleoside-transport protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The Tsx protein , which enhances nucleoside permeation across the outer membrane , is subject to dual control by the DeoR and CytR repressors in addition to being positively controlled by the cyclic AMP-catabolite gene activator protein complex ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644187" target="_blank">2644187</a>]<br></li>
<li>At present , CytR has been shown to ` ﬁne tune ' transcription of two genes ( rpoH and ppiA ) that are connected with protein-folding activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>In addition , CytR only represses transcription from deoP2 inthe presence of CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>[ 10 ] A.J. Haugo , P.I. Watnick , Vibrio cholerae CytR is a repressor of bioﬁlm development , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>These results expand the CytR regulon to include a gene whose product has no known function in ribo - and deoxyribonucleoside-catabolism or transport , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>These repressors , except for PurR and CytR , regulate genes for the catabolism of sugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>]<br></li>
<li>As first demonstrated by Krieger-Brauer and Braun ( 19 ) , expression of the tsx gene is under double negative control of the DeoR and CytR proteins , which are known to regulate the synthesis of two nucleoside-transport systems ( NupC and NupG ) and several nucleoside-catabolizing enzymes ( for reviews , see references 14 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>In E. coli , the structural gene udp encoding UPase is a member of the CytR regulon which comprises at least nine transcriptional units that encode proteins involved in nucleoside uptake and metabolism [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The CytR regulon comprises at least nine transcriptional units that encode proteins involved in nucleoside uptake and metabolism ( for a review , see Hammer-Jespersen , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Introduction The CytR repressor is part of a regulatory circuit that controls at least eight transcriptional units encoding proteins involved in uptake and metabolism of deoxy - and ribonucleosides ( for review , see Hammer-Jespersen , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>The CytR anti-activator and CRP act in concert to repress transcription from genes encoding transport proteins and enzymes necessary for uptake and catabolism of ribonucleosides and deoxyribonucleosides ( reviewed by Hammer-Jespersen , 1983 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Similar interactions at the different promoters of the CytR regulon coordinate expression of the transport proteins and enzymes required for nucleoside-catabolism but also provide differential expression of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Similar interactions at the other transcriptional units of the CytR regulon coordinate expression of the transport proteins and enzymes required for nucleoside-catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The Escherichia coli CytR regulon comprises at least nine unlinked transcriptional units that encode enzymes and transport proteins required for nucleoside-catabolism and recycling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In Escherichia coli , the enzymes and transport proteins required for nucleoside-catabolism and recycling are encoded by genes belonging to the CytR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR regulates expression of genes for the transport and catabo - ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>]<br></li>
<li>The CytR protein represses a subset of the promoters engaged in the expression of genes that code for proteins involved in transport or catabotism of ribo - and deoxyrtbo-nucleosides ( for review , see Hammer-Jespersen , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>CytR regulates expression from different cistrons of enzymes and transport proteins involved in both catabolism and recycling of nucleosides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR regulates initiation of transcription from at least seven promoters that are engaged in expression of genes encoding proteins involved in transport and catabolism of nucleosides and deoxynucleosides ( 15 , 24 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>43 2 Biochemistry 2010 , 49 , 432 -- 442 DOI : 10.1021 / bi901583n An Unusual Pattern of CytR and CRP Binding Energetics at Escherichia coli cddP Suggests a Unique Blend of Class I and Class II Mediated Activation † Allison K. Holt and Donald F. Senear * Department of Molecular Biology and Biochemistry , University of California , Irvine , California 92697 Received September 9 , 2009 ; Revised Manuscript Received November 4 , 2009 ABSTRACT : Two transcription factors , CRP and CytR , mediate positive and negative control of nine cistrons involved in nucleoside-catabolism and recycling in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>171 , No. 9 MalI , a Novel Protein Involved in Regulation of the Maltose System of Escherichia coli , Is Highly Homologous to the Repressor Proteins GalR , CytR , and Lacl JOACHIM REIDL , KARIN ROMISCH , MICHAEL EHRMANN , AND WINFRIED BOOS * Department of Biology , University of Konstanz , D-7750 Konstanz , Federal Republic of Germany Received 13 March 1989/Accepted 10 June 1989 The maltose regulon of Escherichia coli comprises several operons that are under common regulatory control of the MalT activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>ORF1 encodes a protein homologous to repressors including CytR , GalR , and LacI ( Weickert & Adhya , 1992 ) as shown in Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The mall gene ( at 36 min on the linkage map ) codes for a typical repressor protein that is homologous to the Escherichia coli Lacl , GaIR , ' or CytR repressor ( J. Reidl , K. Romisch , M. Ehrmann , and W. Boos , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1856179" target="_blank">1856179</a>]<br></li>
<li>ReidI , J. , Romisch , K. , Ehrmann , M , , and Boos , W. ( 1989 ) Mall , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor ptoteins GaIR , CytR , and Lad .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651130" target="_blank">7651130</a>]<br></li>
<li>Mall , anove protein involved in regulation of the maltose system of Escherichia coli is highlyhomologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_33">
Reidl , J. , Römisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></label>
<input id="rp_similarity_33", type="checkbox">
<div class="RP_similar">
<li>Reidl , J. , RoÈmisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692154" target="_blank">10692154</a>]<br></li>
<li>Reidl J , Romisch K , Ehrmann M & Boos W ( 1989 ) MalI , a novel protein involved in the regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_36">
MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GaIR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></label>
<input id="rp_similarity_36", type="checkbox">
<div class="RP_similar">
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1856179" target="_blank">1856179</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to therepressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003949" target="_blank">12003949</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618441" target="_blank">12618441</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li></div>
<li>We also identified orthologs of genes known to be regulated by CytR in E. coli ( Table S1 ) ( see Data and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>The plasmid pJEL134 , encoding a protein fusion between deoCp2-deoC ' and lacZ ' , contains all the sequence information required for CytR - and cAMP-CRP-regulated expression of deoCp2 ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>CRP-1 also seems to be involved in CytR binding as a single point mutation at position -36 has been shown to reduce CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>INTRODUCTION The structural genes encoding the ~ ystems for nucleoside uptake and metabolism in E. coli are organized in a regulon that is controlled positively by the cAMP/CRP activator complex and negatively by the DeoR and CytR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_44">
Kallipolitis , B.H. and Valentin-Hansen , P. ( 1998 ) Transcription of rpoH , encoding the Escherichia coli heat-shock regulator s32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="rp_similarity_44", type="checkbox">
<div class="RP_similar">
<li>Kallipolitis BH & Valentin-Hansen P ( 1998 ) Transcription of rpoH , encoding the Escherichia coli heat shock regulator s32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li></div>
<li>These findings show that the bacterium uses positive and negative regulatory modules to differentially control the expression of CytR - and cAMP-CRP-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Using a combination of genetic , biochemical , and gene fusion techniques on single-copy plasmids , we have analyzed one such promoter , deoCp2 , to define the DNA sequences involved in positive and negative control by the cAMP-CRP complex and the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Interactions of elements of negative ( CytR ) and positive ( cAMP-CRP ) regulation in the promoter region of the uridinephosphorylase ( udp ) gene in Escherichia coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>Cold Spring Harbor Laboratory Press , Cold Spring Harbor , New York Martinussen J , Mollegaard NE , Holst B , Douthwaite SR , ValentinHansen P ( 1989 ) A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></li>
<li>Martinussen , J , , Mollegaard , N.E. , Hoist , B. , Douthwaite , S.R. , and Valentin-Hansen , P. ( 1988 ) A new version of negative controt , DNA sequences involved in expression and regulation of CytR and cAMP/CRP controtled genes in Escherichia coli In DNA-protein interactions in Transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_51">
and Valentin-Hansen , P. ( 1989 ) A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP/CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></label>
<input id="rp_similarity_51", type="checkbox">
<div class="RP_similar">
<li>A new version of negative control : DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_53">
DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></label>
<input id="rp_similarity_53", type="checkbox">
<div class="RP_similar">
<li>DNA sequences involved in expression and regulation of CytR and cAMP-CRP controlled genes in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>DNA sequences involved in expression and regulation of CytR and cAMP/CRP controlled genes in Escherichia coli , p. 31-41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>DNA sequences involved in expression and regulation of DeoR , CytR and cAMI ' : CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>DNA sequences involved in expression and regulation of DeoR , CytR , and cAMP-CRP controlled genes in Escherichia coli , p. 273-288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_58">
EMBO J 1:1049 -1054 Valentin-Hansen P ( 1985 ) DNA sequences involved in expression and regulation of DooR , CytR and cAMP-CRP controlled genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1661372" target="_blank">1661372</a>]<br></label>
<input id="rp_similarity_58", type="checkbox">
<div class="RP_similar">
<li>Valentin-Hansen , P. ( 1985 ) DNA sequences involved In expression and regulation of DeoR , CytR and cAMP-CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li></div>
<li>3812 Biochemistry 2003 , 42 , 3812-3825 Role of Protein-Protein Bridging Interactions on Cooperative Assembly of DNA-Bound CRP-CytR-CRP Complex and Regulation of the Escherichia Coli CytR Regulon † Mayy Chahla , ‡ John Wooll , § Thomas M. Laue , § Nghia Nguyen , ‡ and Donald F. Senear * , ‡ Department of Molecular Biology and Biochemistry , UniVersity of California , IrVine , California 92697 , and Department of Biochemistry , UniVersity of New Hampshire , Durham , New Hampshire 03824 ReceiVed NoVember 5 , 2002 ; ReVised Manuscript ReceiVed February 10 , 2003 ABSTRACT : The unlinked operons that comprise the Escherichia coli CytR regulon are controlled coordinately through interactions between two gene regulatory proteins , the cAMP receptor protein ( CRP ) and the cytidine repressor ( CytR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Summary The Escherichia coii cytR-encoded repressor protein ( CytR ) controls the expression of several genes involved in nucleoside and deoxynucleoside uptake and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>The CytR repressor regulates initiation of transcription from several operons whose products are involved in the uptake and catabolism of nucleosides and deoxynucleosides in E. coli ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>The CytR repressor protein is part of a regulatory circuit controlling uptake and catabolism of nucleosides and deoxy-nucleosides in Escherichia coli ( for review see Hammer-Jes-persen , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>The Escherichia coli CytR regulon presents a simple para-1 digm for coordinate control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In Escherichia coli negative control of gene-expression by the CytR repressor illustrates such a regulatory case ( 26 , 30 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Sùgaard-Andersen , L. , Mironov , A. , Pedersen , H. , Sukhodelets , V. and Valentin-Hansen , P. ( 1991a ) Single amino acid substitutions in the cAMP receptor protein speci ® cally abolish regulation by the CytR repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>, Fujita , N. , Ishihama , A. , Valentin-Hansen , P. , and Choy , H.E. ( 2001 ) Repression of deoP2 in Escherichia coli by CytR ; conversion of a transcription activator into a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Repression of deoP2 in Escherichia coli by CytR : Conversion of a transcription activator into a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; cAMP , cyclic AMP ; CRP , catabolite activator protein ; cytR , gene encoding CytR ; CytR and DeoR repressor proteins ; A , deletion ; deoR , gene encoding DeoR ; kb , is part of this regulon ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>In this system , the global activator protein cAMP receptor protein ( CRP ; also referred to as the catabolite gene activator protein , CAP ) , guides another DNA-binding protein , the CytR repressor , to a number of binding sites that share minimal sequence homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The cAMP-cAMP receptor protein ( CRP ) complex has a dual role at these promoters : it acts as a conventional activator and as a co-repressor for CytR ( Sogaard-Andersen efal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>In CytR regulated promoters , including the udpP promoter , the cAMP -- CRP complex acts as a transcriptional activator as well as a co-repressor for the CytR protein [ 5,23,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>1 Abbreviations : bistris , 2 - [ bis ( 2-hydroxyethyl ) amino ] -2 - ( hydroxymethyl ) - propane-1 ,3 - diol ; CRP , Escherichia coli cyclic AMP receptor protein ; CytR , Escherichia coli cytidine repressor protein ; DNase I , deoxyribonuclease I ; RNAP , bacterial RNA polymerase , Tris , tris ( hydroxymethyl ) amino-methane ; EDTA , ethylenediaminetetraacetic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Søgaard-Andersen L , Martinussen J , Møllegaard NE , Douthwaite SR , ValentinHansen P ( 1990a ) The CytR repressor antagonizes cyclic AMP-dependent AMP receptor protein activation of the deoCp2 promoter of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>The CytR regulon in Escherichia coli constitutes a combinatorial regulatory system in which the cyclic AMP receptor protein ( CRP ) functions both as an independent transcriptional activator and as an ` adaptor ' for a specific repressor , CytR ( Gerlach et al. , 1991 ; Pedersen et al. , 1991 ; ( © ) Oxford University Press S0gaard-Andersen et al. , 1991b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Nucleic Acids Research Volume 14 Number 5 1986 Nucleotide sequence of the CytR regulatory gene of E. coli K-12 To increase our knowledge of how the cyclic AMP/cyclic AMP receptor protein complex ( cAMP/CRP ) affects transcriptional initiation and how the interplay takes place between a set of regulatory proteins and RNA polymerase , our studies have focused on the identification and characterization of genes involved in uptake and catabolism of nucleosides and deoxyribonucleosides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>In particular , the transcription of genes needed for utilization of nucleosides in Escherichia coli is regulated by a repressor protein , CytR , in concert with the cyclic AMP ( cAMP ) - activated form of cAMP receptor protein ( CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Much of the expression of nucleoside utilization genes is coordinated by a repressor protein , CytR , acting in concert with the cyclic AMP ( cAMP ) - activated form of the cAMP receptor protein ( CRP ) ( 2 , 9 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Sequences related to PurR-binding sites in genes not directly related to purine-nucleotide-synthesis Region and gene Gene product or function Nucleotide sequencea Reference 5 ' flanking glnB cytR dksA speA ascF gshII ppiA gitS gitS prsA PII protein , nitrogen regulation CytR repressor DnaK suppressor protein Arginine decarboxylase Phosphotransferase system Glutathione synthetase Peptidyl-prolyl cis-trans-isomerase Na + / glutamate symport carrier Na + / glutamate symport carrier PRPP synthetase ATGCAAAC GATTTCAA CTGTAACC GTTTTCAC GTGAAATC GGTTACGT AAGAAACC GGITGCGC GTGAAACC GGTTTCTT TCGTAACC GGGTGCCG TTGCAATC GG ¶ TGCTA GCGCAACC GGTTGCGC AAGCAAAC GTTTGCCA AAGAAAAC GTTTTCGC 39 44 21 28 11 10 13 9 9 4 Coding kdpD Regulatory protein of kdp operon Regulatory protein of tdc operon CGGGAAAC GTTTGCTG ATGCAATC GTTTTCCA CAGCAAAC GATTTCTT ATGAACAC GATTTCTT TTGAAACC GTTTTCGA CTGAACCC GGTTTCGA CTGAAACC GTGTACGT CCGAAACC GTGTACCG AGGCAAAC GGGTTCGC TGGCAACC GGTTCCTA ACGAAACC GGTTACTC 46 36 tdcR LT-IIa rimI metH tenC3 rmB rnnA gitA nirB gap Heat-labile enterotoxin type Ila Enzyme acetylating ribosomal protein S18 B12-dependent methionine synthase Replication terminus gene 23S rRNA 23S rRNA Citrate synthase NADH-dependent nitrite reductase Glyceraldehyde-3-phosphate dehydrogenase 31 50 2 17 6 37 19 3 5 3 ' flanking 20 recQ Recombination CCGTAAAC GTTTCCCC rshA DNA duplication TTGAACCC GATTTCTG fimA Type 1 fimbrial subunit CAGTAACC GGTTTCAT metK S-adenosylmethionine synthetase CTGAAACC GATTACAC aacC2 Aminoglycoside - ( 3 ) - N-acetyltransferase AGGAAATC GGGTTCGT a Positions that match the perfect palindromicpur regulon operator shown in Table 1 are in boldface type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8388874" target="_blank">8388874</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">CytR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>While the distance between the OCRP sites flanking OCytR is mostly conserved for the tsx orthologs in the Enterobacteriales , about 14 bp , the score of the 4-box cassette even in close relatives of E. coli is rather low , due to low scores of the OCRP sites ( about 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>A computer-aided search for protein sequence homology with the Fasta program of Pearson and Lipman ( 40 ) revealed that MalI was homologous to GalR ( 60 ) , CytR ( 57 ) , and Lacd ( 3 , 20 ) ( Fig. 8 ) as well as the RbtR protein ( 61 ) from Klebsiella aerogenes and the PurR protein of E. coli ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>The Quik-Change PCR-directed mutagensis kit from Strat-agene was used to convert the sequence immediately upstream of the K63 or R64 codon of the wild-type CytR gene carried by plasmid pCB091 to an NdeI site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Nucleotide sequence of the CytR regulatory gene of E. co/i K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_5">
Nucleotide sequence of the CytR regulatory gene of E. coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></label>
<input id="sit_similarity_5", type="checkbox">
<div class="SIT_similar">
<li>Nucleotide sequence of the CytR regulatory gene of Escherichia coli K 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Nucleotide sequence of the CytR regulatory gene of E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501076" target="_blank">8501076</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>Nucleotide sequence of CytR regulatory gene of E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li></div>
<li>Global analysis of individual site binding data from DNase I footprint titrations Parameter values were obtained from global analysis of the individual site protection data represented in Table I according to the model formulated in Table II.a Parameter Valueb DG1 212.0 6 0.2 DG2 213.7 6 0.2 DG 210.9 6 0.2 3 DG4 210.5 6 0.4 DG13 21.4 6 0.2 DG23 21.3 6 0.3 DG123 22.3 6 0.3 DDG 11.0 6 0.3 13 DDG23 10.3 6 0.3 DDG123 10.8 6 0.4 s 0.058 c a CytR binding data for only the previously recognized operator ( designated site 3 in the models ) included in the analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>To identify new candidate CytR regulon members , we used three recognition rules to select regions for construction and manual analysis of alignments , requiring either ( 1 ) two candidate OCRP sites at a distance 10 -- 40 bp , or ( 2 ) two candidate OCYTR sites at a distance not exceeding 20 bp , or ( 3 ) all four sites in the configuration OCRPD -- N ( 210 ) -- 20 -- OCYTRD -- N0 -- 20 -- OCYTRP -- N ( 210 ) -- 20 -- OCRPP , with negative numbers denoting overlapping sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>20 No. 19 pp. 5392 ± 5399 , 2001 Repression of deoP2 in Escherichia coli by C conversion of a transcription activator into a repressor ytR : Minsang Shin , Soim Kang , Seok-Jin Hyun , Nobuyuki Fujita , Akira Ishihama , 1 1 Poul Valentin-Hansen and Hyon E.Choy 2 3 Department of Microbiology , Chonnam University Medical College , 5 Hakdong , Dongku , Gwangju , South Korea 501-714 , Department of 1 Molecular Genetics , National Institutes of Genetics , Mishima , Shizuoka 411 , Japan and Department of Molecular Biology , 2 Odense University , Campusvej 55 , DK-5230 Odense M , Denmark 3 Corresponding author e-mail : hyonchoy@chonnam.ac.kr In the deoP2 promoter of Escherichia coli , a transcription activator , cAMP ± CRP , binds at two sites , centered at ± 41.5 and ± 93.5 from the start site of transcription , while a repressor , CytR , binds to a space between the two cAMP ± CRP complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>For example , while extended protection patterns in nupG ( both DNase I and chemical modification reagents ) have been interpreted as resulting from CytR binding to two perfect TGCAA motifs separated by an 11-bp spacer ( 11 ) , this overlooks several additional CytR recognition motifs , including one located in the middle of the proposed 11-bp spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR recognition motifs are shown as shaded boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Such flexibility could also be invoked to explain CytR 's ability to bind pairs recognition motifs with variable spacing of 2 -- 5 bp in different operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>nupG is a representat-ive of group C1 sequences , and the arrangement of the three CytR repeats in the divergent cytX-rot promoter region corresponds to the group C3 sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>It was recently suggested that CytR recognizes operators with sequences 5 ' - ' ( Rasmussen eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Moreover , inspection of the recognition sequences reveals the conservation of the central CpG dinucleotide in all LacI family operators ( Schumacher et al. , 1994 ) , again with the exception of CytR operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Repositioning might provide a configuration in which RNAP makes favorable contacts with upstream DNA sequences even while CytR is still bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>It has been suggested that CytR and the cAMP/CAP complex recognize simitar DNA sequences ( Vatentin-Hansen , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>This implies that CytR and CRP recognize very similar DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>On the basis of these results , we first proposed a simple competition model for the action of CytR in which the repressor and the cAMP-CRP complex recognize very similar or overlapping DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>It has been shown recently that when CytR and CRP are used together to select DNA sequences that are preferred for formation of the three-protein complex , the sequences most commonly selected are CytR recognition motifs separated by 10 -- 13 base pairs and almost centered between CRP1 and CRP2 ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Nevertheless , a structural model has been proposed in which the DNA is wrapped smoothly around the three-protein complex , and the two CytR DNA binding domains ( one per subunit ) bind to DNA sequences that are arranged centrosymmetrically and are separated by 11 base pairs ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>For a centrosymmetric three-protein complex to form as envisioned in the model , CytR would have to dissociate from CytO and instead bind more widely spaced and symmetrically arranged DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>From the analysis of CytR interaction with the deo-p2 promoter , we have recently proposed that the sequence motif 5 ' - tTTNCa-3 ' ( N = G , A , T , or C ) might constitute an important part of the sequence recognized by CytR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the presence of the CAMP-CRPcomplex the two protein species bind cooperatively to deoP2 , forming a complex in which CytR occupies the sequence between the two DNA bound cAMP * CRPcomplexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>Identification of the nucleotide sequence recognized by the CAMP-CRP dependent CytR repressor protein in the deoP2 promoter in E. co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_28">
Identiﬁcation of the nucleotide sequence recognised by the cAMP-CAP dependent CytR repressor protein in the deoP2 promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></label>
<input id="sit_similarity_28", type="checkbox">
<div class="SIT_similar">
<li>Identiﬁcation of the nucleotide sequence recognized by the cAMP-CRP dependent CytR repressor protein in the deoP2 promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li></div>
<li>In Figure 6 the amino acid sequence of CytR is aligned with that of GalR in a manner to maximize the homology of the NH2-terminal regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>According to the primary amino acid sequence of CytR ( Valentin-Hansen et al. , 1986 ) the exact site of clea-vage could be assigned for most of the smaller fragments ( Fig. 5 , lanes 2 -- 8 and Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>This fact is consistent with the sequence of udp , which , like other CytR-regulated promoters , contains multiple CytR rec-ognition motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>An imperfect inverted-repeat motif ( 59-TGCAAN2 -- 3 TTGCA ) is present in each of these promoters and is considered a candidate CytR recognition sequence ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>A family of pentamer motifs matching or nearly matching the sequence 59-TGCAA has been found as inverted-repeats in all CytR-regulated promoters studied to date , including udpP ( Fig. 1 ) , and had been postulated to constitute a CytR binding site ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>In the three upper diagrams , the repression by CytR-wl of deoP2-wf and of deoP2 derivatives with a single base pair substitution or complete substitutions of the CytR recognition sequence is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Here we demonstrate that CytR regulates transcription from deoP2 promoters in which the entire CytR recognition sequence has been eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>Based on where CytR binds specifically to the CytRregulated deoP1 and deoP2 promoters , we proposed that TTGCAA , a symmetric variant of these sequences , is the recognition sequence ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In nupG CytR binds to a region of approximately 30bp that includes the sequence 5 ' - TGCAA-AI-rGGGAATGT-TTGCA-3 ' , in which the proposed halfsite consensus sequences have been underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The absence of consensus sequences for the CytR protein within the udp regulatory region of Y. pseudotuberculosis and V. cholerae ( Fig. 3 ) suggests that reduced CytR regulation of udpPYp and udpPV c was due to the reduced ability of the E. coli CytR protein to bind heterlogous promoters Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Consistently , this region contains a sequence with homology to the CytR consensus recognition sequence ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>We have compared these sequences to the regulatory region of deoP2 and other CytR-regulated promoters ( Fig. 10 ) to further refine the consensus CytR recognition motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Based on literature reports that CytR binds to a single specific site flanked by CRP1 and CRP2 , routine analysis of fractional protection included only this sequence , i.e. approximately bp 255 to 280 ( see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Multiple , specific CytR sites are arranged over an extended region of DNA that includes both CRP1 and CRP2 as well as the sequence they flank .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In either case , the new RNAP-DNA interaction would be in the sequence between CRP2 and CRP1 , precisely where CytR binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>A key to how CytR binds to different promoters is its relatively broad DNA sequence specificity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In this way , the additional CytR site is defined thermodynamically but is not defined in relation to any particular sequence of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>It therefore appears that the maximal reduction of CytR regulation observed in the case of udpPV c is most likely a con sequence of the combined effect of the absence of a CytR consensus sequence and the presence of an improved CRP1 binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>CRP1 has been proposed to substitute for the lack of a 235 consensus promoter sequence ( 14 ) , whereas CRP2 is proposed to be necessary only for CytR to bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>F. CytR consensus sequence , x = 2 , 3 , or 11 bp ( Rasmussen et al. , 1993 ; this work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>However , if based on the criterion of most probable nucleotide at each position in these putative CytR operators , then the 6-bp sequence , TTGCAA , defines the consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR is reported to protect only a 20-bp sequence ( 7 ) corresponding to the putative CytR operator that overlaps CRP1 ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Two copies of this motif are present in the tsx-p2 sequence protected by CytR ( 315 to 320 bp and 332 to 337 bp [ Fig. 1B ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>From threonine in position 7 to leucine in position 26 of MalI ( Fig. 8 ) , the sequence was highly homologous to regions in GalR , Ladl , and CytR that have been implicated to bind operator sequences as helix-turn-helix motifs ( 51 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Finally , A12-29 has an afﬁnity for CytR similar to that of other sequences obtained after 12 selection rounds , despite the presence of an extra octamer repeat in this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The sequence A08-21 ( L -- 1 -- R ) with a spacing of 1 bp between the octamer motifs has a 2-fold lower afﬁnity for CytR than the L -- 2 -- R or R -- 2 -- R sequences ; DNA fragments containing a single octamer repeat bind CytR half as efﬁciently as the deoP2-wt promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Similar specific interactions of CytR with extended DNA sequences have been identified at deoP1 and also near the middle of the approximately 600-bp sequence separating P1 and P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Finally , the CytR operator site of the wild-type deoP2 promoter deviates somewhat from the consensus sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The UP-element overlaps with the DNA site for CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The udpP structure is similar to that of cddP in the spacing between CRP1 and CRP2 ( 52 bp ) and in the spacing between CRP1 and the 210 site ( 29 bp ) ( Fig. 1 ) ( 31 ) , and it contains the putative CytR operator ( 59-TGCAA and TTGCG inverted-repeat motif ) but with 5 bp between the half-sites ( in contrast to 2 bp in deoP2 and tsxP2 and 3 bp in cddP and cytRP ) ( 16 , 24 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>As the spacing between the half-sites is 11 bp , as opposed to 2 or 3 bp in the consensus previously proposed , these results suggest that CytR can bind operators with rather different half-site spacings .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The present work allows us to deﬁne an 8 bp half-operator consensus ( ATTGT/CAAC ) for CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The consensus is best matched by the site previously identified as the CytR operator , centered at 270.5 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In this report we show that the CytR operator half-sites of nupG are separated by 11 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The DNase I footprint covers ~ 20 bp at both sequences , and , as expected , CytR interacts speciﬁcally with both operator half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Similarly , a ﬂexible linker allows 34,35 CytR , a 2-fold symmetry dimer , to recognize both directly or inversely repeated half-operator sites in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491603" target="_blank">15491603</a>]<br></li>
<li>Speciﬁcally , CytR has a relaxed spacing requirement between operator half-sites and can , in concert with cAMP -- CRP , repress transcription in vivo from octamer boxes separated with centre distances up to three helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>CytR binds to its cognate promoters in concert with cAMP-CRP in a highly co-operative fashion torming repression complex that generally consists of two mol-ecules of cAMP-CRP separated by 52 or 53 bp and CytR ioound in between ( Sogaard-Andersen et ai , 1991b ; Gerlach etai , 1991 ; Holsi etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The CytR/cAMP-CRP repression complex in rot is similar in structure to those formed at deoP2 and cddP and consists of two cAMP-CRP molecules separated by 53 bp , with CytR bound in between .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>The majority of the nucleoprotein complexes consist of a single CytR molecule bound between two CRP molecules separated by 52 or 53 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>For example , in the DNase I footprints , the C1 sequences ( C13-20 and C13-40 ) exhibit a 3 bp unprotected region in the middle of the CytR operator , and C09-20 ( R -- 1 -- R ) and C13-26 ( L -- 1 -- L ) contain unprotected regions to the left or right of the operator , regardless of the presence of cAMP -- CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>As the pattern of the 50 bp combined footprint corresponds to the sum of the independent footprints , this complex most likely consists of cAMP-CRP bound to CRP2 , and CytR to the CytR target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Alternatively , since the strong CRP2 target is expected to span ~ 30 bp ( Liu-Johnson et al. , 1986 ) , the sequence at the edges of the CytR operator may optimize both cAMP -- CRP and CytR interactions with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The consensus sequence for cAMP-CRP IS AATGTGANr ^ TCACATT [ Kolb ei aL. 1993 ) and for CytR TGCAAN3 .3 TTGCA ( Rasmussen ef aL. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Third , the protection extends close to the -10 sequence , i.e. , beyond the protection obtained with cAMP-CRP or CytR individually .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The second model hypothesizes a complex in which CytR alone occupies the region from CRP-2 close to the -10 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Our results with the tsx-p2 promoter demonstrate that the tandem targets for the cAMP-CRP activator complex are only separated by 33 bp and that the CytR repressor binds independently to a DNA segment that includes the sequences between the CRP targets and overlaps with the CRP-1 site ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the first model , cAMP-CRP still binds to CRP-1 and CytR occupies the sequences on both sides of CRP-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In this complex , cAMP-CRP interacts with tandem CRP targets and CytR interacts with sequences located between these targets ( Fig. 1 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>In the third model , CytR interacts with the same DNA sequences in the combined footprint as in the independent footprint , and cAMP-CRP , rather than binding to CRP-1 , binds to the region located immediately downstream of the CytR-binding sequence ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In combination , the cAMP-CRP and CytR proteins bind cooperatively to tsx-p2 , and the nucleoprotein complex formed covers a region of 78 bp extending from the CRP-2 site close to the -10 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the absence of cAMP-CRP , the CytR repressor interacts with a sequence extending from CRP-2 into CRP-1 ( 317 to 337 bp [ Fig. 1B ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>Co-operative binding of cAMP-CRP and CytR Next we tested binding of CytR and cAMP-CRP to a 160 bp fragment containing nupG sequences from - 1 1 8 to - 1 0 in the DNase I footprinting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Addition of CytR protein to binding reactions containing cAMP-CRP resulted in protection of an 84 bp region spanning from CRP-1 to around position - 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Addition of the CytR extract to binding reactions containing cAMP-CRP results in protection of an 80 bp region spanning from CRP-1 to CRP-2 ( lanes 7-9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Furthermore , CytR binds at the deoP2 regulatory region co-operatively with cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>CytR and cAMP-CRP bind co-operatively to deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>DMS footprint of cAMP-CRP and CytR bound to nupG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>Hence , cAMP-CRP and CytR bind cooperatively to cddP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Binding of cAMP-CRP and CytR to a cddP derivative containing 2 bp inserted between CRP-1 and CRP-2 / 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The deletion that ends at position -103 and removes the upstream flanking sequences of CRP-2 exhibited a 30 % reduction in both promoter activity and CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>In all cases , these studies have implicated interactions of CRP and CytR with sequences located in the 80 -- 100 bp immediately upstream of the various transcription start sites and interactions between the proteins as the basis for positive and negative gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The following points should be noted : ( i ) neither the integrity of the CRP target , the CytR operator halfsites nor the sequence separating the CRP target and the promoter downstream CytR operator half-site have been altered in the generated promoters ; ( ii ) signiﬁcant similarity to CytR operator half-sites was avoided in the inserted spacer sequences ; ( iii ) primer extension revealed a single start site for transcription initiation ( þ1 ) and all the derived promoters exhibited a similar activation by the cAMP -- CRP complex ( < 100-fold ) ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>CAMP-CRF ' and CytR bind simultaneously and co-operatively to r ; ytRP covering a region of 5i t ' o 69 bp that ctorresponds to the two individual pro-tein-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>Presumably , either the spacing between the CytR and CRP sites or something inherent in the CytR site , such as the spacing between recognition motifs , is responsible for controlling whether interactions between CytR and CRP are still possible when cytidine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>CRP bound to this duplicate site seemed not to interact with CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CRP and CytR designated filled sites according to the transcription factor bound ; O designates empty sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Ratios of variances were also separately calculated considering the data obtained for each binding site and set of effectors , e.g. , for binding of CytR to CytO of the wild-type promoter in the presence of CRP and cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In this regard , we note that co-operative binding of CytR is considerably lower at a ` single CRP site ' cdd promoter than at an intact cdd promoter carrying tandem CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Q 1998 Blackwell Science Lt Results The rpoH promoter region contains putative binding targets for CytR and CRP By inspection of the rpoH promoter region , a second , putative CRP target ( CRP-2 ) is found 53 bp upstream the CRP binding target suggested by Nagai et al. ( 1990 ) ( CRP-1 , Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Further more , CytR binding at the cddP promoter has been reported to result in repositioning of the upstream CRP binding site by 2 bp ( Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>Generality of the isolated DNA sequences One complication in devising a general selection scheme for cAMP -- CRP-dependent CytR-binding sites was that cooperative binding of cAMP -- CRP and CytR had been observed on DNA templates with 52 -- 54 bp separations between the centers of the two CRP targets ( Søgaard-Andersen et al. , 1990 ; Pedersen et al. , 1991 ; Holst et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>This structural complexity , together with the fact that CytR repression at deoP2 has a strict requirement for a spacing of 52 or 53 bp between the two CRP binding sites ( S0gaard-Andersen et al. , 1990b , 1991b ) , has hampered the proposal of a unifying binding model for CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_103">
Since sequences very similar to the CRP-2 target have also been found in the regulatory region of the cdd gene , and to which CRP binds , we suggest that several of the other CytR binding sites may contain sequences which also will be recognized by the cAMP/CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></label>
<input id="sit_similarity_103", type="checkbox">
<div class="SIT_similar">
<li>Since sequences very similar to the CRP-2 target have also been found in the regulatory region of the cdd gene ( manuscript in pre-paration ) , and to which CRP binds , we suggest that several of the other CytR binding sites may contain sequences which also will be recognized by the cAMP/CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li></div>
<li>The CytR-regulated promoters usually contain two DNA sites for CRP , centred at positions 41.5 and 93.5 with respect to the transcription start point , and a DNA site for CytR located between the two CRP-binding sites ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Our results support the model in which repression normally involves the binding of CytR to cAMP-CRP to form a complex which binds to speciﬁc sites in the udp promoter , without direct interaction between CytR protein and a speciﬁc operator DNA sequence , and in which induction by speciﬁc inducer cytidine involves dissociation of CytR from cAMP-CRP and then RNA polymerase interaction with cAMP-CRP bound to a site upstream of the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>CytR binds to two 50-TTGCAA-30 motifs between two CRP-binding sites separated by 10 -- 30 bp ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In vitro CytR interacts weakly with a sequence located between the two cAMP-CRP-binding sites separated by 53 bp ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>In the presence of CytR , however , the two regulators bind cooperatively to cddP forming a nucleoprotein complex in which cAMP - CRP binds to CRP-1 and CRP-3 and CytR occupies the sequence between these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>Here we demonstrate the presence of two binding sites for the cAMP-CRP complex , CRP-1 and CRP-2 , and show that cAMP-CRP and CytR bind cooperatively to tSx-p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The cAMP ± CRP binding site is shown as a shaded box and CytR as an open box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></li>
<li>The mode of action of CytR has been puzzling because the only obvious sequence homology between the different promoter regions is the presence of tandem binding sites for the cAMP-CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>( i ) CytR and cAMP-CRP compete for the same DNA sequences , with CytR binding cooperatively to the tandem CRP sites , i.e. , a model which would resemble the cI/Cro system of bacteriophage lambda , in which the cI and Cro proteins bind the DNA and each excludes to same sequences the binding of the other ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>By mobility shift and footprinting experiments one CytR and four CRP-binding sites were identified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>However , the relative positions of the CytR - and CRP-binding sites resemble those of group C2 sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>The boxed sequences indicate the binding sites tor CytR and cAMP-CRP as defined in DNase I tootprinting experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Comparative analysis of the udpP promoter region from Y. pseudotuberculosis , V. cholerae , S. typhimurium and E. coli revealed that location , spacing and orientation of putative binding sites for CRP protein are highly conserved , whereas CytR protein recognition sequences of udpPYp and udpPV c deviate markedly from the E. coli and S. typhimurium CytR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The putative flanking sites in both directions match the consensus less well , consistent with the footprints which indicate decreasing affinity for CytR binding to these sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Sequences sharing homology to consensus binding targets for CytR and CRP ( underlined by arrows ) are indicated in bold letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The boxed sequences indicate the twoCRP sites , and the arrows below the boxes indicate the sequences sharing homology with the consensus CRP sites ( deCrombruggheet al. , 1984 ) ; thetwoarrowsin theCytR-binding site indicate the CytR recognition sequence ( Rasmussen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The boxed sequences indicate the two CRP sites and underlined bold letters indicate the sequences sharing homology with the consensus CytR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Comparison of the spacer regions separating the two CRP sites in deoP2 and tsxP2 does not , however , reveal any clearly homologous sequence that could be recognized by CytR , This does not , of course , rule out the possibility that CytR interacts directly with a sequence in the spacer region ( see below ) and this is presently being tested by engineering point mutations here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>On the other hand , CytR regulation is dependent on CRP-1 and CRP-2 and on the length and the sequence of the region between the two CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1647022" target="_blank">1647022</a>]<br></li>
<li>Here we demonstrate , by means of in vivo titration , that CytR interaction with deoP2 depends not only on CRP-2 , but also on CRP-1 and the length and possibly the sequence separating these two sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Degenerate versions 01 the pentameric sequence 5 ' - TGCAA-3 ' in direct or inverted-repeat by 2-3 bases ) which are gener ally flanked by one or more CRP-binding sites have been suggested as a mínimal consensus for CytR binding , and related motifs have been reported for deoP2 , cddP , cytRP and ( Rasmussen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>Results Elimination of the CytR Operator Has Little Effect on Repression The CytR operator in deof2 is located between the two CRP sites and consists of an imperfect inverted-repeat ( Figures 1A and 1B ) ( Rasmussen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The CytR operator in udpPEc was located between the two CRP sites and consisted of an imperfect inverted-repeat ( Fig. 3 ) [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>Specifically , CytR binds to multiple closely spaced or even overlapping sites on DNA with differing affinities and not to just a single operator as previously presumed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The results we present here demonstrate that CytR binds specifically to multiple sites at the udp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The results we present here for udp confirm that CytR binds specifically and with relatively high affinity to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Electrophoretic Mobility Shift Analysis of CytR Binding to the udp Promoter -- To assess whether the array of CytR recognition motifs in the udp promoter constitutes separate , specific sites to which CytR can bind simultaneously , we first conducted titrations of CytR binding alone using electrophoretic mobility shift analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>A single CytR recognition motif within CRP2 might also constitute a weak binding site , analogous to the situation in deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Consistently , examination of the udpPYp and udpPV c sequences reveals potential CytR binding sites with sequences : AATGTATT-10-CTTTGCG and TATGAAG-10-ATTGCTGT , respectively ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The sequences protected from DNase I cleavage by CytR binding to the upstream and P1 sites are also coincident with appropriately spaced pairs of motifs ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Therefore , these DNA sequences constitute specific CytR recognition sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Comparison of the DNA sequences of these sites further clarifies the CytR binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR binding to a single operator does not simply nucleate nonspecific binding to adjacent DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Taken together , these results indicate CytR binding as consistent with the CytR recognition elements pres-ent in the udp regulatory sequence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Based on footprinting analysis of purified CytR binding to deoP2 and other promoters , this intervening sequence is now thought to contain the CytR binding site ( 6 , 8 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However , DNA-binding studies with purified CytR are required to assess whether CytR actuatty recognizes this DNA sequence motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Evaluation of the DNA sequence requirement for specific CytR binding suggests that a limited array of contiguous and overlapping CytR sites exists at deoP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>A possible CytR binding sequence between these sites is shown in lower case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>The sequence responsible for the sequence specific binding of CytR has not been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>The sequence motifs that may be involved in sequence specific binding of CytR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1327747" target="_blank">1327747</a>]<br></li>
<li>The CytR concentration dependence of separate regions of the extended CytR footprint was evaluated systematically to determine whether the entire footprint represents a single binding event or multiple binding of CytR to separate sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These might constitute a specific CytR binding site centered at þ13 bp , albeit with unusual spacing of five base pairs between recognition half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>In vitro , CytR only binds with a low affinity to specific binding sites ( Pedersen et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The distribution of such motifs , both at deoP2 and at other CytR-regulated promoters , define arrays of contiguous and overlapping CytR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>As expected , based on this quarternary structure , CytR binding sites contain tandem recognition motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Do these motifs in udp constitute separate , specific CytR binding sites ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Pairs of motifs with appropriate spacing to constitute additional CytR binding sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Also since adjacent sites overlap , CytR binding to these should be self-competitive , a feature we did not incorporate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>t ~ AMT ' - ( ` RJP and CytR have overlapping binding sites on hot.h st , rands ( see Pig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>These are specific sites for CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>These sites both appear to comprise pairs of overlapping CytR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR Binding Sites Upstream from deoP2 Define a Recognition Motif and Demonstrate an Array of Overlapping deoP2 Sites -- In addition to these specific CytR binding sites in the P2 regulatory region , we have identified two other CytR binding loci .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>In fact , when we investigated CytR binding to the deoP2 promoter , we found that CytR does bind specifically to multiple sites ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Multiple speciﬁc CytR binding sites at the Escherichia coli deoP2 promoter mediate both cooperative and competitive interactions between CytR and cAMP 35 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074084" target="_blank">10074084</a>]<br></li>
<li>-9.6 ( 0.5 -9.7 ( 0.2 -11.5 ( 0.2 -12.8 ( 0.1 -13.3 ( 0.1 -13.7 ( 0.2 -14.1 ( 0.1 -13.2 ( 0.3 -13.4 ( 0.4 -13.3 ( 0.3 -12.2 ( 0.3 -12.1 ( 0.0 -14.3 ( 0.3 -14.5 ( 0.03 Table 2 : Apparent Cooperativity As Assessed by Differences in Indivia dual Site Loading Free Energy Changes Described in Table 1 a Table 3 : Configurations and Free Energy States for CRP and CytR Binding to the cdd Promoter binding sites a CRP1 free energy contributions free energy state b CRP2 CytR at CRP1 CytO CytR at CRP2 O O O 1 O O reference state ΔGs ,1 2 O O O CRP O ΔG1 ΔGs ,2 3 CRP O O O O ΔG2 ΔGs ,3 4 O O CytR O O ΔG3 ΔGs ,4 5 O O O O CytR ΔG4 ΔGs ,5 6 O CytR O O O ΔG5 ΔGs ,6 7 CRP O O CRP O ΔG1 þ ΔG2 ΔGs ,7 8 O O CytR CRP O ΔG1 þ ΔG3 þ ΔG13 ΔGs ,8 9 O CytR O CRP O ΔG1 þ ΔG5 ΔGs ,9 10 CRP O CytR O O ΔG2 þ ΔG3 þ ΔG23 ΔGs ,10 11 CRP O O O CytR ΔG2 þ ΔG4 ΔGs ,11 12 O O CytR O CytR ΔG3 þ ΔG4 ΔGs ,12 13 O CytR CytR O O ΔG3 þ ΔG5 ΔGs ,13 14 O CytR O O CytR ΔG4 þ ΔG5 ΔGs ,14 15 CRP O CytR CRP O ΔG1 þ ΔG2 þ ΔG3 þ ΔG123 ΔGs ,15 16 CRP O CytR O CytR ΔG2 þ ΔG3 þ ΔG4 þ ΔG23 ΔGs ,16 17 O CytR CytR O CytR ΔG3 þ ΔG4 þ ΔG5 ΔGs ,17 18 O CytR CytR CRP O ΔG1 þ ΔG3 þ ΔG5 þ ΔG13 ΔGs ,18 a Binding sites are listed in columns 2-6 corresponding to the order , from promoter distal to promoter proximal , on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>The salient features of this model are that CytR binds separately to CytO and to a site at CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In this complex , CytR binds to an operator site ( usually referred to as CytO ) that is flanked by tandem CRP operators , CRP1 and CRP2 , as are found in most CytR-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>Although CytR binds to an additional site located just downstream from the transcription start site as noted above , there is no interaction between CytR binding to this site and CytR/CRP binding to CRP2 , CytO , and CRP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>CytR binds specifically to a site overlapping CRP2 and with an affinity similar to that for CytO , whereas the affinity for CytR binding to sites overlapping CRP1 is similar to that for nonspecific DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The possibilities are 1 ) that the additional CytR sites that overlap CRP1 and CRP2 are specific binding sites to which CytR binds noncooperatively and with defined affinity ; and 2 ) that the additional binding is nonspecific but cooperative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>However , CytR also binds to separate , specific sites at deoP2 , one of which overlaps CRP1 , and another that overlaps CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The results indicate that CytR binds specifically to multiple sites at deoP2 , including both the well recognized CytR site flanked by CRP1 and CRP2 and also sites coincident with CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Model for Cooperative and Competitive Binding of CytR and cAMP-CRP -- To determine whether such interactions of CytR with DNA sequences overlapping and thereby occluding or competing with CRP binding to CRP1 and CRP2 can account quantitatively for both the loading free energy changes in Table I and apparent free energy changes in Fig. 6 , two models that incorporate two additional CytR binding sites and which constitute opposing possibilities were evaluated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CRP sites ( CRP1 and CRP2 ) and the CytR operator ( CytO ) are delineated by open boxes ; CytR recognition motifs are shaded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Interaction of CytR with these sites provides a complex pattern of both competitive and cooperative interactions between CytR and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Here too , CRP bound to either CRP1 or CRP2 or both is a net recruiter for CytR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>By eliminating cooperativity between CytR and CRP bound to flanking sites , cytidine should make CytR into a simple competitor for CRP binding to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Additionally , while cytidine binding to CytR nearly eliminates the pairwise interactions between CytR and CRP bound at CRP1 , it has little effect on pairwise cooperativity between CytR and CRP bound at CRP2 or as a consequence on the overall cooperativity of the three-protein complex in which CRP is bound to both sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Recently , we showed that CytR binds specifically to multiple sites in the E. coli deoP promoter , thereby providing competition for CRP binding to CRP operator site 1 ( CRP1 ) and CRP2 as well as cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In this model , CRP and CytR compete for binding to mutually exclusive sites at CRP2 , but CRP alone binds to CRP1 with no competition from CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>But this would also be partly offset by unfavorable competition between CytR and CRP for binding to overlapping sites at CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>To evaluate the possibility of cooperative interactions between CytR bound to deoP2 and CytR bound to the upstream and/or P1 site ( s ) , the P2 regulatory region and the upstream and P1 regions were isolated on separate DNA fragments as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>The transcriptionat initiation site of the cytR mRNA was mapped 53 bp upstream of the presumed GTG trans-tationat start codon for CytR ( Fig. 1 ) , Eight bp upstream of this site there is a hexamer ( AAAAAT ) that shows homotogy to the consensus Pribnow box of E. coli promoters ( Hartey and Reynolds , 1987 ) ; however , an appropriately spaced - 3 5 region is not readity apparent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>As shown in Fig. 3 , the corresponding region of udpPSt contained a very similar putative CytR binding site , while neither udpPYp nor udpPV c contained inverted-repeats or a TGCA motif between positions − 71 / − 67 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>How CytR prevents transcriptional initiation is still un ¬ -70 -60 GAL 5 - T T G T G T A A A C G A T T C C A c T A -3 3 - A A C A C A T T T A A G G T G A T -5 DEO-5 - T T A T T T G A A C C A G A T C G C A T T A -3 3 - A A T A A A C T T G GXT C T A G c G T A A T -5 -100 -90 Fig. 7 Comparison of the putative CytR-binding site in the deoP-2 region with that of qalOE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>An additional region of protection suggests a CytR site located just downstream from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>However , we note a pair of sequences with close matches to the CytR half-site consensus sequence located just downstream from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Two additional CytR footprints were observed , one centered at about 2235 bp and another at about 2592 bp from the P2 transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>When CytR binds to udp , a region extending from about 25 to 110 bp upstream from the start site for transcription is protected from cleavage ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>In contrast to the CRP footprints , which are localized to the 22-bp CRP binding sites , CytR binding protects an extended region from about bp 225 to 2110 bp relative to the start site for transcription ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>CytR binding protects an extended region of the promoter , from about bp to -120 to þ10 , measured relative to the trans-cription start site ( Figure 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Alternatively , the interaction of the CRP-CytR complex with the deleted DNA made the DNA in this region partially inaccessible to DNase I. Moreover , even in del264 , which lacks the putative CytR operator region , there was some CRP-dependent protection by CytR a few nucleotides immediately downstream of the chi-meric CRP site created by the deletion ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>We found that the 50-upstream region of yeiT contains a 5-base inverted-repeat sequence that matches with the consensus sequence [ 42,43 ] , namely , 50-NNNTGTGANNNNNNTCACANNN-30 , for the target DNA of CRP-binding ( Fig. 4B ) and has no binding site for CytR [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>We characterized four tsx-p2 mutants exhibiting a reduced response to CytR ; three carried mutations in the CRP-2 site , and one carried a mutation in the region between CRP-1 and the -10 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>A comparison of the cytR regutatory region with other proposed CytR targets ( Fig. 5 ) reveated a region of homotogy in cytR , around position -136 bp , that overlaps one of the regions recognized by the cAMP/CAP comptex ( -145 bp to -129 bp ; see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>A putative recognition motif has been identified ( 6 , 17 ) , and pairs of such motifs , arranged as either direct or inverted-repeats , have been implicated as the CytR operator ( 9 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Pairs of inverted and/or direct repeats of this motif with variable spacing ( 2 -- 5 bp ) are thought to constitute CytR operators in the different CytR-regulated promoters ( cf. Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>Clearly , CytR can not bind the C-sequences ( or the natural targets ) in a similar binding mode , since the hinge helices would be pulled too far apart when the HTH motifs interact with direct repeats , or inverted-repeats separated by one DNA helical turn ( Figure 6B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>However , as shown in a SELEX experiment with the deo operator , both inverted-repeats of OCYTR boxes with a large spacer ( 10 to 13 bp ) and direct repeats in either direction ( OCYTRD or OCYTRP ) with a short spacer ( 1 bp ) may be bound by CytR [ 37 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>A potential CytR target situated between CRP-1 and CRP-2 in the rpoH promoter region matches the consensus for inverted octamer motifs separated by 11 bp [ ( A / T ) ( G/C ) TGCAAC-N11-GTT ( G/A ) CATT ] at 12 out of 16 possible positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The CytR operator of the cdd promoter contains R -- 1 -- R direct repeats , and the repeats are separated by 5 and 14 bp , respectively , from the ﬂanking CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>Henrik Pedersen1 and Poul Valentin-Hansen2 © Oxford University Pres In one mode , CytR recognizes inverted octamer repeats separated by 10 -- 13 bp ; in another the repressor binds direct octamer repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In one mode , CytR recognizes inverted octamer repeats separated by 10 -- 13 bp ( inverted-repeats , L-10 / 13-R site : ( A/T ) ( G/C ) TGCAAC-10 / 13-GTT ( G/A ) CATT ) ; in another the repressor binds direct optamer repeats separated by 1 bp ( R-1R : GTTGCATT-1-GTTGCAC ) [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>The study revealed that CytR , when co-binding with two molecules of CRP , preferentially recognizes inverted-repeats separated by 10 -- 13 bp , or direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>However , when co-binding with cAMP -- CRP , CytR instead recognizes inverted-repeats separated by 10 -- 13 bp , or direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>When co-binding with tandem cAMP -- CRP complexes , CytR preferentially recognizes inverted octamer repeats separated by 10 -- 13 bp or direct octamer repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>Thus , cAMP -- CRP preferentially stabilizes a set of CytR conformations that ﬁt operators composed of inverted-repeats with wide spacing ( 10 -- 13 bp ) , or direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the presence of cAMP -- CRP , CytR recognizes ( B ) inverted octamer repeats separated by 10 -- 13 bp or ( C ) direct repeats separated by 1 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>We ﬁnd that CytR by itself preferentially binds two octamer repeats , in direct or inverted orientation , and separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>In the absence of CRP , the optimal binding site for CytR consists of two octamer repeats , in direct or inverted orientation , and separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
<li>In the absence of cAMP -- CRP , the CytR repressor binds two such octamer motifs in inverted or direct repeat arrangement , preferably separated by 2 bp ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>( A ) In the absence of cAMP -- CRP , the CytR repressor preferentially binds octamer repeats in direct or inverted orientation , separated by 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by CytR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Moreover , additional genes , which do not belong to the CytR regulon , are strongly induced in the presence of nucleosides ; for example , add and guaC , which encode adenosine deaminase and GMP reductase respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>CytR is often described as playing a dual role in regulation ( 30 ) because it can activate transcription under some circumstances , whereas in others , it represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>When quantifying the levels of rpoH transcripts produced from pBK4026 and pPS3514 in cytR ¹ ( SØ929 and SØ929 þ pDK7 ) and cytR þ ( SØ928 and SØ929 þ pDK7-CytR ) strains , it is observed that transcription from P3 , P4 and P5 is reduced ,2.5 - fold in SØ928 and ,10 - fold in SØ929 carrying pDK7-CytR , whereas CytR does not seem to interfere markedly with the progress of the RNA polymerase transcribing from the P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>MicroCorrespondence Kallipolitis and Valentin-Hansen ( 1998 , Mol Microbiol 29 : 1091 -- 1099 ) recently reported that transcription of rpoH ( which encodes the heat shock sigma factor in Escheri-chia coli ) is controlled by the cAMP -- CRP/CytR complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987139" target="_blank">9987139</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_5">
Kallipolitis , B.H. and Valentin-Hansen , P. ( 1998 ) Transcription of rpoH , encoding the Escherichia coli heat-shock regulator s32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></label>
<input id="tu_similarity_5", type="checkbox">
<div class="TU_similar">
<li>Kallipolitis BH & Valentin-Hansen P ( 1998 ) Transcription of rpoH , encoding the Escherichia coli heat shock regulator s32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>Transcription of rpoH , encoding the Escherichia coli heat-shock regulator 32 , is negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952953" target="_blank">16952953</a>]<br></li>
<li>Transcription of rpoH , encoding the Escherichia coli heat-shock regulator s , is 54 negatively controlled by the cAMP-CRP/CytR nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li></div>
<li>These examples with single operators upstream of -45 are the promoters of the CytR regulon , the spf promoter ( see CRP in Tables 1 and 2 ) , and perhaps the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>One can imagine scenarios in which CytR remains bound to DNA , if not to CRP , under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>For example , the galE promoter is repressed by GalR , the lacZ promoter repressed by LacI and many CRP-activated promoters are repressed by CytR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Indeed , when P1 is omitted , rpoH -- lacZ expression from the remaining promoters is repressed 2.8-fold by CytR ( Table 1 , plasmid pBK4026 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>At P1 and P2 transcription is repressed by the DeoR repressor and induced by deoxyribo-se-5-P ; at P2 transcription is also repressed by the CytR repressor ( the inducer being cytidine or adenosine ) and depends on cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Transcription is activated in response to intracellular cAMP levels by CRP , repressed by CytR , and induced by cytidine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>( III ) The rotA ( ppiA ) gene is also controlled by the CytR / CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The nupG promoter is activated 50-fold by cAMP-CRP , and repressed fourfold by CytR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>The deoC , nupG and tsx operons that are under CytR regulation were activated by CRP in our study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_18">
In these strains the expression of the lac genes is controlled by CytR regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></label>
<input id="tu_similarity_18", type="checkbox">
<div class="TU_similar">
<li>In these strains lac expression is controlled by CytR regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li></div>
<li>tn view of the modet for the CytR-mediated repression of gene expression , it has been speculated that transcription of cytR is subjected to regutation by CytR and the cAMP/CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Both cdd and udp are members of the CytR regulon , i.e. , the expression of these genes is negatively regulated by the CytR repressor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14499937" target="_blank">14499937</a>]<br></li>
<li>We show that cytR expression is negatively controlled by the CytR protein and positively affected by the cAMP/CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Moreover , the expression of udp and cdd is positively controlled by the CRP-cAMP complex and negatively controlled by the CytR repressor [ 37 -- 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>Two of the other members of this family , PurR and CytR , are negatively autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Expression of cytX is negatively regulated by CytR and the start site for transcription of cytX is only separated from the rot start codon by 220bp ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>These observations led us to consider whether transcription of the rpoH gene is negatively regulated by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Transcription of rpoH is regulated by CytR in vivo To investigate rpoH expression , a 342 bp EcoRV -- EcoRV fragment , which contains all the rpoH promoters ( Fig. 1A ) , was inserted upstream from the lacZ gene in the trans-criptional fusion vector pOU254 , creating pPS3514 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>To test whether the CytR protein influences the expression of its own structural gene , we transformed the multicopy ptasmid pCBQ08 ( cytR ^ ) and the vector pBR322 into strain GP103 carrying the chromosomat ' ^ ( cytR-iacZ ^ ) ^ operon fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Taken together , the data shown in Tabte 2 demonstrate that the expression of the chromosomal ` i > -LCB- cytR-lacZ * -RCB- ^ operon fusion is specifioatly reduced by the CytR repressor , strongly suggesting that cytR trans-cription is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>CytR is also negatively autoregulated by twofold when CytR is supplied on a low-copy-number plasmid to cells containing a fusion of the cytR promoter and the lacZ gene ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>The ctoned cytR '' gene specificatly affected cytR-iacZ ^ transcriptio since the expression of an ompF-lacZ ^ operon fusion was not inftuenced by CytR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>19B8a ) into cytR ^ disrupts the integrity of this gene and consequently no CytR protein is synthesized In ^ -LCB- cytR-lacZ * ) ^ operon fusion strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>S0 ^ 3 ^ 6/F ` tacl ' ^ is cytR ' and encodes on the chromosome an operon-fusion between the cdd promoter , which is highly -LCB- 55-fold ) regulated by CytR , and the lacZ gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Unusual duplications and deletions that resulted in constitutive udp expression that depended on the presence of CytR were also found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626289" target="_blank">8626289</a>]<br></li>
<li>The nupC and nupG genes also differ in their modes 01 regulation and expression ; nupC has been shown to be subject to repressíon by the CytR protein whilst nupG is repressed by both CytR and DeoR ( Munch-Petersen and Mygind 1976 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>As S0744 is deficient for both repressors ( DeoR and CytR ) , this result indicated that FIS not only affected transcriptionof deoRand cytR , but also directly influenced the expression of cdd , udpand deoC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Plasmid pCB008 carrying the cytR ' gene ( 3 ) strongly repressed the expression of the D ( tsx-lacZ ) 1 ( Hyb ) protein fusion regardless of the presence of the chromosomally encoded DeoR and CytR repressors ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The expression of the tsx gene is increased in the presence of nucleosides through the action of the DeoR and CytR repressors ( 98 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>The cyfR ' - encoded gene product exerts negative controt on the transcription initiating from both the witd-type tsx P2 promoter and the CytR operator mutant , O ' ^ - I , tocated in this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>As first demonstrated by Krieger-Brauer and Braun ( 19 ) , expression of the tsx gene is under double negative control of the DeoR and CytR proteins , which are known to regulate the synthesis of two nucleoside transport systems ( NupC and NupG ) and several nucleoside-catabolizing enzymes ( for reviews , see references 14 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>However , the interrelationship of the double negative ( DeoR ; CytR ) and positive ( cAMP-CAP ) control in tsx expression has not been analyzed to any great extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>For the deoCABD operon it has been shown that transcription is initiated from two promoters that are differentially regulated by DeoR and CytR ( Valentin-Hansen et 01996 Blackwell Science Ltd , Molecular Microb/o/ogy ,22 , 21-2 a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>RESULTS A Simple Three-StepPurification of CytR-As the intracellular concentration of CytR is relatively low , the expression of cytR is autoregulated , and the Shine-Dalgarnsoequence of cytR is rather poor ( Valentin-Hansen et al. , 1978 ; Gerlach et al. , 1990 ; Valentin-Hansen et al. , 1986 ) , aplasmid allowing production of large amounts of CytR was constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>It is evident by inspection that protection occurs first , i.e. at the lowest CytR concentrations , in the intervening DNA where CytO is located ; only at higher CytR concentration is comparable protection observed at CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10347150" target="_blank">10347150</a>]<br></li>
<li>The operator for CytR ( CytO ) is located between CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12667072" target="_blank">12667072</a>]<br></li>
<li>Another significant observation of the cddP footprints is the unusually high affinity for CytR binding , both to CytO ( Table 4 ) and also to the site located just downstream from the transcription start site ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Negative autoregulation of the expression of CytR , another member of this family , requires only one cytR operator located upstream of the transcription start site ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Its affinity to its operators is rather weak [ 22 ] and because of that , in contrast to most prokaryotic repressors , CytR alone is not capable to repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028500" target="_blank">23028500</a>]<br></li>
<li>First , CytR binds with only modest afﬁnity to its operators and can not act on its own in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466254" target="_blank">9466254</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>