<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Dan</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Dan</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Dan transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Acknowledgments We would like to thank Doug Rees , Olaf Andersen , Pierre Sens , Sergei Sukharev , Nily Dan , Jennifer Stockdill , and Ned Wingreen for their thoughtful comments on the manuscript , Chris Gandhi for his input into possible experiments , Ben Freund for useful discussion , and the useful comments of our anonymous reviewers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17480116" target="_blank">17480116</a>]<br></li>
<li>Acknowledgments We thank Jay Shendure , Rhonda Harrison , Dan Janse , Edward Sekinger , Joseph Geisberg , Zarmik Moqtaderi , Heather Hirsch , Marc Schwabish , Jason Lieb , and Xiao Liu for helpful discussions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>aA JkspR $ 182 - - - - - - - - - - - - - - - - - - - - - V AlrLli P Q Q L AS & & $ 100 SfIi L L N L P R ND0 Q C V D r D D 7 7 T$ i M V 9 9Q L : C8 * SSa ( lQ 9 E Lyman 220 aA L D n - - - - -- - - - - - - - - - - - R aX -- sL a R aspas 201 9Uun S JDnjri9Yv * Cr a Dan L a n ag-oar !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2129559" target="_blank">2129559</a>]<br></li>
<li>W1 and M1 represent ; W2 and M2 , Indirect immuno-blot against anti-Dan ; W3 and M3 , DAPI-staining ; W4 and M4 , merged images of DAPI and immuno-stained patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Dan Herschlag , Stanford University , is gratefully acknowledged for setting up the ` ribosome connection ' and for his advice and encouragement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10675343" target="_blank">10675343</a>]<br></li>
<li>In fact , Dan was proposed to regulate the neighboring ttdA-ttdB-ygiE operon ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Oshima and Biville ( 18 ) predicted that YgiP , here renamed to Dan , is involved in regulation of the adjacent ttdA-ttdB-ygiE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Along this line , Dan may be involved in the regulation of genome transcription under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Along this line , besides the ttdA-ttdB-ygiE operon , a set of genes must exist , which are induced in the presence of Dan altogether contribute to the E. coli survival under the anaerobic conditions such as within animals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Our ﬁndings extend this concept and further indicate that Dan is a bifunctional nucleoid protein , playing dual roles both in structuring the nucleoid under anaerobic-growth conditions and in regulation of a set of genes needed for growth under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>These observations altogether indicate that Dan is needed for full growth under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The growth of mutants lacking Dan is retarded under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>We then propose that YgiP is a novel nucleoid protein of E. coli under anaerobiosis and propose to rename it Dan ( DNA-binding protein under anaerobic conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Considering all these results , we conclude that Dan is a nucleoid protein speciﬁc in cells growing under anaerobic conditions , and thus belongs to the family of nucleoid proteins that are produced only at speciﬁc growth-phases or under speciﬁc growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Growth of Dan-defective E. coli mutant under anaerobic conditions Dan was identiﬁed as a nucleoid protein in cells growing under hypoxic or anaerobic conditions , implying that Dan plays a role in cell survival under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>None of these culture conditions induced the synthesis of Dan protein , supporting the notion that Dan is speciﬁcally induced under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>After testing various culture conditions , we found that Dan is highly expressed in cells grown under hypoxic or anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>With respect to the number of DNA-binding sites , Dan is different from the well-characterized global regulators so far examined by the genomic SELEX , including CRP , Cra , RstA , PdhR , RutR , TyrR , NemR , AllR and CitB ( 7 -- 10,12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The contour length of Dan -- DNA complexes gradually decreased 2-fold from 2.5 -- 2.7 mm to 0.8 -- 1.2 mm concomitant with the increase in Dan concentration ( Figure 4E ) , supporting the notion that Dan binding induces DNA compaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The width of peak position increased 1.5 -- 2-fold reaching to around 45 nm concomitant with the increase in Dan concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The height of DNA-bound Dan is rather constant independent of the protein concentrations , but the width increased with the increase in Dan concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The width of Dan -- DNA complexes at the site of Dan binding slightly increased upon increase in Dan concentration ( Figure 4D and Supplementary Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The height of naked DNA was around 0.5 -- 0.6 nm , but upon binding of Dan , increased almost 10-fold to around 5 nm ( Supplementary Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Accordingly , the longest group-III complexes disappeared upon further increase in Dan concentration , and instead the proportion of intermediate-sized group-II increased linearly ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>At low Dan concentrations , Dan binds to various positions on plasmid DNA in non-speciﬁc manner ( Form-I ) and upon increase in the Dan concentration , more Da molecules binds through protein-protein interactions , ultimately forming the Dan-saturated complexes ( Form-II ) ( Figure 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Dan binds to DNA in a polymerization or stiffening mode , leading to form Dan -- DNA rods at saturation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>At high Dan concentration , the increased number of Dan molecules bound to DNA strand , leading to extension of Dan oligomers along the DNA strand and ultimately covering the entire DNA surface ( Form 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>( B ) A 2 model of local conformation of DNA-bound Dan dots .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>( A ) Fluorescent-labeled SELEX segment ( 1.0 pmol ) from the dan-ttdA spacer was incubated in the absence or presence of increasing concentrations of puriﬁed Dan ( 10 , 20 and 40 pmol from left to right ) and then subjected to DNase-I foot-printing assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_30">
Cell lysates were prepared at various growth-phases ( lanes 1 and 7 , log-phase at 0.3 OD600 ; lanes 2 and 8 , log-phase at 0.6 OD600 ; lanes 3 and 9 , late log-phase , 1.0 OD600 ; lanes 4 and 10 , 24-h stationary-phase ; lane 5 and 11 , 48-h stationary-phase ; lanes 6 and 12 , 72-h stationary-phase ) and subjected to quantitative western blots analysis against anti-Dan and anti-RpoA antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></label>
<input id="act_similarity_30", type="checkbox">
<div class="ACT_similar">
<li>Cell lysates were prepared at various growth-phases ( lanes 1 and 7 , log-phase at 0.3 OD600 ; lanes 2 and 8 , log-phase at 0.6 OD600 ; lanes 3 and 9 , late log-phase , 1.0 OD600 ; lanes 4 and 10 , 24-h stationary-phase ; lane 5 and 11 , 48-h stationary-phase ; lanes 6 and 12 , 72-h stationary-phase ) and subjected to western blots analysis against anti-Dan and anti-RpoA antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li></div>
<li>An Unusual Twin-His Arrangement in the Pore of Am monia Channels Is Essential for Substrate Conductance * □ S Received for publication , August 31 , 2006 Published , JBC Papers in Press , October 12 , 2006 , DOI 10.1074 / jbc.M608325200 Arnaud Javelle ‡ § , Domenico Lupo § , Lei Zheng § , Xiao-Dan Li § , Fritz K. Winkler § 1 , and Mike Merrick ‡ 2 From the ‡ Department of Molecular Microbiology , John Innes Centre , Colney Lane , Norwich , Norfolk , NR4 7UH , United Kingdom and the § Biomolecular Research , Paul Scherrer Institut , CH-5232 Villigen , Switzerland Amt proteins constitute a class of ubiquitous integral membrane proteins that mediate movement of ammonium across cell membranes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17040913" target="_blank">17040913</a>]<br></li>
<li>dPeak : High Resolution Identification of Transcription Factor Binding Sites from PET and SET ChIP-Seq Data Dongjun Chung , Dan Park , Kevin Myers , Jeffrey Grass , Patricia Kiley , Robert Landick , 1 $ 2 2 3,4 2,4 3,4,5 Sündüz Keleş * 1,6 1 Department of Statistics , University of Wisconsin , Madison , Wisconsin , United States of America , 2 Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , United States of America , 3 Department of Biochemistry , University of Wisconsin , Madison , Wisconsin , United States of America , 4 Great Lakes Bioenergy Research Center , University of Wisconsin , Madison , Wisconsin , United States of America , 5 Department of Bacteriology , University of Wisconsin , Madison , Wisconsin , United States of America , 6 Department of Biostatistics and Medical Informatics , University of Wisconsin , Madison , Wisconsin , United States of America Abstract Chromatin immunoprecipitation followed by high throughput sequencing ( ChIP-Seq ) has been successfully used for genome-wide profiling of transcription factor binding sites , histone modifications , and nucleosome occupancy in many model organisms and humans .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24146601" target="_blank">24146601</a>]<br></li>
<li>47 , No. 9 Mechanisms of Fosfomycin R Escherichia coli Biological Costs and esistance in Annika I. Nilsson ,1,2 Otto G. Berg ,3 Olle Aspevall ,4 Gunnar Kahlmeter ,5 and Dan I. Andersson1 ,2 * Department of Bacteriology , Swedish Institute for Infectious Disease Control , S-171 82 Stockholm1 ; Microbiology and Tumour Biology Center , Karolinska Institute , S-171 77 Stockholm2 ; Department of Molecular Evolution , Uppsala University , S-752 36 Uppsala3 ; Division of Clinical Bacteriology , Karolinska Institute , Huddinge University Hospital , S-141 86 Stockholm4 ; and Central Hospital Clinical Microbiology , S-351 85 Växjö ,5 Sweden Received 10 December 2002/Returned for modiﬁcation 26 May 2003/Accepted 19 June 2003 Fosfomycin is a cell wall inhibitor used mainly for the treatment of uncomplicated lower urinary tract infections .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12936984" target="_blank">12936984</a>]<br></li>
<li>Gross chromosomal rearrangement mediated by DNA replication in stressed cells : evidence from Escherichia coli J. M. Moore1 , Hallie Wimberly2 ,3 , P. C. Thornton2 , Susan M. Rosenberg1 ,2,4,5 , and P. J. Hastings2 1Department of Biochemistry and Molecular Biology , Baylor College of Medicine , Houston , Texas 2Department of Molecular and Human Genetics , Baylor College of Medicine , Houston , Texas 3Department of Pathology , Yale University , New Haven , Connecticut 4Department of Molecular Virology and Microbiology , Baylor College of Medicine , Houston , Texas 5Dan L. Duncan Cancer Center , Baylor College of Medicine , Houston , Texas Abstract Gross chromosomal rearrangements ( GCRs ) , or changes in chromosome structure , play central roles in evolution and are central to cancer formation and progression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22954223" target="_blank">22954223</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Dan protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>This motif is also involved in metal chelating , suggesting a role of Dan in resistance of a metal , as does Dps against iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Dan Ferber in this laboratory recently determined that the stationary-phase cultures of the virulent strain Salmonella typhimurium ATCC 14028 contains 60 U of cyanide-inhibitable SOD activity per mg in osmotic shockates , showing that it , too , synthesizes a periplasmic CuZnSOD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626323" target="_blank">8626323</a>]<br></li>
<li>Dan contains two sets of Cys -- Cys pair : one consisting of Cys and Cys , and another of Cys and Cys .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">Dan evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Here we discovered Dan ( DNA-binding protein under anaerobic conditions ) as the third member of the GNP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>We then conclude that YgiP is a hitherto unidentiﬁed nucleoid protein in E. coli growing under anaerobic conditions , and propose to rename YgiP ( or TtdR ) to Dan ( DNA-binding protein under anaerobic conditions ) , which belongs to the same family of growth phase-speciﬁc nucleoid proteins as Fis in exponential growth phase ( 22,23 ) and Dps in stationary phase ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Considering all these results , we conclude that Dan is a nucleoid protein speciﬁc in cells growing under anaerobic conditions , and thus belongs to the family of nucleoid proteins that are produced only at speciﬁc growth phases or under speciﬁc growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Instead Dan is rather similar to those of major nucleoid proteins such as HU , IHF , H-NS and Fis ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Under the hypoxic conditions , Dan was markedly induced ( Figure 6B ) , and formed highly stained dots ( Figure 7B , W1 -- W4 ; Figure 6D , W ) , indicated that Dan belongs to the group-II nucleoid proteins , including Fis , Rob ( CbpB ) , CbpA and IciA , which show irregular distribution within the nucleoid ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Dan</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>To overcome this problem and identify the genes under the direct control of a test regulator , we have developed the genomic SELEX screening system for search of the recognition and binding sequences by use of a test transcription factor ( 43 ) , and we have successfully employed this system for identiﬁcation of whole sets of regulation targets by a number of transcription factors , such as PdhR ( 35 ) , RutR ( YcdC ) ( 44 ) , and Dan ( YgiP ) ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21421764" target="_blank">21421764</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_2">
In fact , the regulatory roles of uncharacterized transcription factors YcdC , YdhM , YgiP and YbjK have been identiﬁed to be RutR ( regulator of pyrimidine synthesis and degradation ) ( Shimada et al. 2007 ) , NemR ( N-ethylmalei-mide reductase repressor ) ( Umezawa et al. 2008 ) , Dan ( DNA-binding protein under anaerobic conditions ) ( Teramoto et al. 2010 ) and RcdR ( regulator of csgD encoding the master regulator of bioﬁlm formation ) ( Shimada et al. 2012 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23301696" target="_blank">23301696</a>]<br></label>
<input id="rp_similarity_2", type="checkbox">
<div class="RP_similar">
<li>For instance , the regulatory roles of uncharacterized transcription factors YcdC , YdhM , and YgiP have been identiﬁed to be RutR ( regulator of pyrimidine synthesis and degradation ) ( Shimada et al. 2007 ) , NemR ( N-ethylmaleimide reductase repressor ) ( Umezawa et al. 2008 ) , and Dan ( DNA-binding protein under anaerobic conditions ) ( Teramoto et al. 2010 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li></div>
<li>This ﬁnding suggests that Dan plays a role in either maintenance of the nucleoid architecture and/or expression of the nucleoid function under the anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Growth of Dan-defective E. coli mutant under anaerobic conditions Dan was identiﬁed as a nucleoid protein in cells growing under hypoxic or anaerobic conditions , implying that Dan plays a role in cell survival under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Along this line , Dan may be involved in the regulation of genome transcription under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Our ﬁndings extend this concept and further indicate that Dan is a bifunctional nucleoid protein , playing dual roles both in structuring the nucleoid under anaerobic growth conditions and in regulation of a set of genes needed for growth under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>One possible function of Dan is the autogenous control of its own expression , but Dan may also play a regulatory role in expression of the neighboring and divergently transcribed ttdA-ttdB-ygjE operon as predicted based on the gene organization ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Likewise , Dan may play dual roles , i.e. an architectural role and a global regulator of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Each Dan may play a role of seeding for further assembly of additional Dan molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>This motif is also involved in metal chelating , suggesting a role of Dan in resistance of a metal , as does Dps against iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Thus , the reversible disulﬁde bond formation may also be involved in structural and functional control of the Dan protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>For protein expression , the pDan plasmid was transformed into E. coli BL21 ( DE3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>ACKNOWLEDGEMENTS We thank Dr. Takenori Miki for providing the E. coli mutant lacking the dan gene , Naoki Kobayashi and Minami Naruse for the initial SELEX screening , and Ayako Kori and Kayoko Yamada for expression and puri-ﬁcation of Dan , and Tomohiro Shimda , Hiroshi Ogasawara and Kaneyoshi Yamamoto for discussion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Dan binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Gene , 35 ( 1985 ) 199-207 Elsevier GENE 1277 Analysis of the ptsH-ptsl-err region in Escherichiu coli K-12 : nucleotide sequence of the ptsH gene ( Recombinant DNA ; phosphoenolpyruvate-dependent operon ; Sl mapping ; CAP site ) Hilde De Reuse , Anne Roy and Antoine Dan & in * UnitPde Biochimie des R & ulations Cellulaires , InstitutPasteur , 28 , Rue du Docteur Roux , 7.5724 Paris Cedex 15 ( France ) Tel. 16 ( l ) 30619 19 ( 20.55 ) ( Received August 14th , 1984 ) ( Revision received January 22nd , 1985 ) ( Accepted January 28th , 1985 ) SUMMARY The nucleotide sequence of an Escherkhia coli DNA segment containing the ptsH gene and the first 162 nucleotides of theptd gene encoding , respectively , Hpr and enzyme I of the phosphoenolpyruvate-dependent glycose phosphotransferase system ( PTS ) , was determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2411636" target="_blank">2411636</a>]<br></li>
<li>We are also grateful to Gary Olsen for assistance with the phylogenetic analysis of sodC , John Mayﬁeld and Louisa Tabatabai for communicating the DNA sequence of sodC from B. abortus , and Howard Steinman and Dan Ferber for their helpful suggestions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626323" target="_blank">8626323</a>]<br></li>
<li>We also thank Dan Fajardo for assisting us with DNA sequence graphics , Neng-Vao and Georgia Smith for Northern analysis , and Scott Bingham for oligonucleotide synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476172" target="_blank">7476172</a>]<br></li>
<li>The number of SELEX isolates by a test transcription factor correlates with its afinity to the target sequence ( 7 -- 16 ) , we predicted that Dan has a strong afinity to this dan-ttdA spacer region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Identiﬁcation of the Dan box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>After brief treatment with DNase I , at least four protected regions were identiﬁed ( Figure 3A ) , of which three ( Dan-I , Dan-II and Dan-III ) covered 14-bp-long sequences and one ( Dan-IV ) covered 29-bp sequence , suggesting that two molecules of Dan are associated with Dan-IV site , designated as Dan-IV-1 and Dan-IV-2 ( Figure 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>All the protected regions by Dan against DNase-I digestion included GTTNATT consensus sequence ( Figure 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Dan binds to DNA in a polymerization or stiffening mode , leading to form Dan -- DNA rods at saturation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>In brief , 5 pmol of a mixture of genome DNA fragments of 150 -- 300 bp in length and 10 pmol His-tagged Dan were mixed in a binding buffer ( 10 mM Tris -- HCl , pH 8.0 at 4 C , 3 mM Mg-acetate , 150 mM NaCl and 0.1 mM EDTA ) and incubated for 30 min at 37 C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The slight decrease in DNA length after binding Dan may be explained by local folding of DNA at the site of Dan binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Puriﬁcation of Dan protein For construction of plasmid pDan for Dan expression , a DNA fragment corresponding to the Dan-coding region was ampliﬁed by PCR using E. coli W3110 genome DNA as a template and a pair of primers , which were designed so as to hybridize upstream or downstream of the Dancoding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>In brief , probes were generated by PCR ampliﬁcation of Dan-binding sequences in SELEX using a pair of primers , 50 ﬂuorescein isothiocyanate ( FITC ) - labeled T7-F primer ( 50-TAATACGACTACTATAGGG-30 ) and T7-R primer ( 50-GGTTTTCCCAGTCACACGACG-30 ) , the genomic SELEX plasmids containing the respective Dan recognition sequences as templates , and Ex Taq DNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>RESULTS Genomic SELEX search for Dan-binding sequences For the identiﬁcation of DNA sequences that are recognized by E. coli Dan , we employed the ` Genomic SELEX ' system ( 7 ) , in which a mixture of genome DNA fragments was used instead of mixtures of oligonucleotides with all possible sequences as used in the original SELEX ( systematic evolution of ligands by exponential enrichment ) ( 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>For the conﬁrmation of Dan binding to the DNA sequences identiﬁed by the Genomic SELEX screening , we performed the gel mobility shift assay for ﬁve representative DNA probes from SELEX-clos and SELEX-chip library : dan-ttdA ( Figure 2A ) , maoC-paaA ( Figure 2B ) , yfaZ-yfaO ( Figure 2C ) and intZ-yffL ( Figure 2D ) from group A , and ygaD - [ mltB ] - srlA ( Figure 2E ) and perR - [ insN ] - ykfC ( Figure 2F ) from group B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by Dan</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Expression of Dan was induced by adding IPTG ( isopropyl-b-D-1-thiogalactopyranoside ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>None of these culture conditions induced the synthesis of Dan protein , supporting the notion that Dan is speciﬁcally induced under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Along this line , besides the ttdA-ttdB-ygiE operon , a set of genes must exist , which are induced in the presence of Dan altogether contribute to the E. coli survival under the anaerobic conditions such as within animals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>If Dan is such a speciﬁc DNA-binding transcription factor , the regulation targets may be the genes , which are located downstream of these Dan-binding sites ( shown by arrows in Supplementary Table 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>This sequence with strong afinity to Dan is located upstream of its own gene and upstream of the divergently transcribed ttdAB genes encoding L-tartrate dehydratase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>One possible function of Dan is the autogenous control of its own expression , but Dan may also play a regulatory role in expression of the neighboring and divergently transcribed ttdA-ttdB-ygjE operon as predicted based on the gene organization ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>