<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - DeoR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">DeoR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">DeoR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Loop formation in viro and in vin ' o has been demonstrated in natural and artificial situations for a number of prokaryotic regulatory pro-reins including the Lacl , GalR , AraC , DeoR , ~ cI and P22 repressors ( reviewed in [ 49 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Elgrably-Weiss , M. , Schlosser-Silverman , E. , Rosenshine , I. , and Altuvia , S. ( 2006 ) DeoR , a DeoR-type transcriptional-regulator of multiple target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>In many cases ( DeoR , GlpR , LacR , AccR and FruR ) , the inducer is a phosphorylated sugar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>In many cases , the effector-molecule that interacts with the transcriptional-regulator in order to achieve induction is a phosphorylated sugar ( DeoR , GlpR , LacR , and AccR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>However , this is not a general rule , the octameric DeoR repressor can bind to three operator sites without significant distance effects [ 33 , 46 , 67 , 681 ~ attl whereas the formation of DNA loops by the tetrameric Lac repressor was shown to be dependent on the correct phasing on linear DNA fragments , no such dependency was observed on supercoiled DNA [ 69 , 701 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Examples of carbohydrate catabolic systems that are regulated by repressors belonging to the DeoR family include those of the glycerol-P ( GlpR ) , glucitol ( GutR ) , fucose ( FucR ) , L-ascorbate ( UlaR ) and deoxyribonucleoside ( DeoR ) systems of E. coli , as well as those of the lactose ( LacR ) , fructose ( FruR ) and inositol ( IolR ) systems of Gram-positive bacteria and agrocinopine ( AccR ) of A. tumefaciens ( Valentin-Hansen et al. , 1985 ; Yamada & Saier , 1988 ; Lu & Lin , 1989 ; van Rooijen & de Vos , 1990 ; Beck von Bodman et al. , 1992 ; Yoshida et al. , 1995 ; Zeng et al. , 1996 ; Campos et al. , 2004 ; Barriere et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>DeoR and GlpR of E. coli regulate the expression of genes involved in the metabolism of deoxyribonucleosides and sn-glycerol 3-phosphate respectively ( Mortensen et al. , 1989 ; Zeng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>One member of the DeoR family , FucR , is known to activate transcription of genes involved in metabolism of fucose ( Chen et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Although GlpR and DeoR exhibit signiﬁcant similarities in the recognition helices and corresponding operators sites , wild-type GlpR , or its var-iants , failed to-bind the deoR operator sites ( Zeng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>GlpR ( Larson , AgaR repressor and dominant repressor variants 815 et al. , 1987 ) , DeoR ( Valentin-Hansen et al. , 1986 ; Mortensen et al. , 1989 ) and possibly all members of this family of proteins are active as oligomers of identical subunits that bind to multiple palindromic operator sites on DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>The tetrameric nature of AgaR and the fact that each of the promoter regions contains multiple operators suggest that , like GlpR and DeoR , AgaR binds several operators simultaneously , resulting in DNA looping around the site of transcription initiation ( Dandanell and Hammer , 1985 ; Hammer and Dandanell , 1989 ; Larson et al. , 1992 ; Weissenborn et al. , 1992 ; Yang and Larson , 1996 ; Yang et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>As we have observed that in the absence of DeoR udp and cdd transcription was no longer influenced by FIS ( see Fig. 4A ) , udp and cdd appear to be regulated by DeoR in a similar manner to deoCABD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>DeoR works by binding co-operatively to three widely spaced operators forming DNA loops between DNA-bound repressors ( Dandanell and Hammer , 1985 ; Dandanell etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>Both DeoR and GlpR form higher-order oligomers under native conditions ( octamers and tetramers , respectively ) and achieve repression by simultaneous binding of repressor to widely separated operator sites , with the intervening DNA forming a loop ( 1 , 11 , 12 , 16 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Therefore , it was surprising that the operator binding speciﬁcity of GlpR was not changed to that of DeoR when these two residues of GlpR were changed to match those of DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Thus , the decO , operator will increase repression of the P2 promoter by DeoR when introduced at a position between 200 and 1000 bp upstream of this promoter ( Dandanell and Hammer , 1985 ) and it is also able to enhance repression although inserted 4000 bp downstream of P2 ( our unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>DNA - protein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli deoCABD promoters by the DeoR repressor P.Valentin-Hansen , B.Albrechtsent and J.E.Love Larsen Department of Molecular Biology , Odense University , Campusvej 55 , DK-5230 Odense M , Denmark ` Present address : Department of Biological Sciences , Columbia University , NY 10027 , USA Communicated by N.O.Kjeldgaard The sequences required for full repression of the Escherichia coli deoPl and P2 promoters by the deoR repressor ( DeoR ) have been analyzed in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>DSA-protein recognition : Demonstration of three genetically separated elements t , hat are required for repression of the Escherichia coli deoCABD promot , er by the DeoR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1640456" target="_blank">1640456</a>]<br></li>
<li>G ( 1992 ) DeoR repression at-a-distance only weakly respondsIO changesin intempemtorseparationand DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_23">
DeoR repression at-a-distance only weakly responds to changes in interoperator separation and DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></label>
<input id="act_similarity_23", type="checkbox">
<div class="ACT_similar">
<li>DeoR repression at a distance only weakly responds to changes in interoperator separation and DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li></div>
<li>x Editor : Marco Soria Keywords yciT ; DeoR ; transcription control ; operon ; repression ; global regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>Since all four fusions produce the same messenger , and the derepressed levels are equal , these results suggest that sequences between -270 and -360 are involved in DeoR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>Our results clearly show , however , that the presence of all three targets is necessary for full repression of the P1 and P2 promoters by DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>Increased repression by DeoR is observed in fusions harbouring two operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>DeoR repression requires the presence of three widely spaced operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></li>
<li>However , there are some examples of DeoR regulators involved in cell responses to environmental stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>Repressor of cytR Repressor of deoR UdP Conversion of uridine to uracil plus CytR ribose-1-phosphate Conversion of deoxyribose-5-phosphate CytR , DeoR to glyceraldehyde-3-phosphate deoCABD mflADR rbsACB CVtR deoR Transport and assimilation of o-mannitol MtlR Transport and assimilation of o-ribose RbsR Transcriptional repressor Transcriptional repressor Q 1996 Blackwell Science Ltd , Molecular Microbiobgy ,22 , 21-2 Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Its primary sequence is highty homotogous to GaIR , DeoR , Lad , PurR , and Mall repressor proteins ( Vatentin-Hansen etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>In addition , the DeoR repressor regulates nupG fourfold ( Munch-Petersen and Jensen , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596434" target="_blank">8596434</a>]<br></li>
<li>In accordance with previous results these data show that initiation from P1 is negatively controlled by the DeoR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>DeoR , a classical bacterial repressor , acts alone and regulates transcription initiation negatively from a set of the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The similarity values for identical residues in the headpiece were 38 for LacI , 36 for GalR , 34 for PurR , 28 for DeoR and 35 for RbtR ( the lac , gal , put and deo repressors from E. coli , and the rbt repressor from Klebsiella pneumoniae respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1851954" target="_blank">1851954</a>]<br></li>
<li>The classical repressors of transcriptional initiation in E. coli , such as Lacd and DeoR , inhibit the initiation of both factor-dependent and independent transcriptions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>The mechanism of transcriptional repression by the CytR repressor is clearly different from that of the classical bacterial repressors such as Lad , Cl , Cro and DeoR -LCB- Ptashne , 1986 ; for a review see Adhya , 1989 ) , A priori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></li>
<li>Valentin-Hansen , P. , Svenningsen , I , , Munch-Petersen , A. , and Hammer-Jespersen , K. ( 1978 ) Regulation of the deo operon in Escherichia coti : the double negative oontrol of the deo opei `` on by the GytR and DeoR repressors in a DNA directed in vitro system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715459" target="_blank">7715459</a>]<br></li>
<li>Transcriptionof udp and cdd is downregulated by CytR , while deoC is repres-sed by both DeoR and CytR ( Hammer-Jespersen and Munch-Petersen , 1975 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>We strongly suggest that the regulation of both P1 and P2 expression by DeoR is responsible for this property of deo regulation , since 1 ) the deoP2 promoter seems to be much stronger than deoPl , 2 ) strong repression of P2 by DeoR is observed in fusions between the Pl-P2 region and lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>However , our studies of gene fusions have also shown that in absence of the P1 region DeoR regulation of the P2 promoter is weak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>To analyze the regulation of the mutated fusion by the chromosomally encoded DeoR and CytR repressors , we transduced this fusion into deoR and cytR strains and subsequently measured , B-galac-tosidase activity in glycerol - and glucose-grown cultures ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>In contrast to this , the xanthosine phosphorylase activity can be measured only when cells are grown in the presence of xanthosine , and the xanthosine phosphorylase activity is not inﬂuenced by DeoR or CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559336" target="_blank">7559336</a>]<br></li>
<li>The regulon is negatively regulated by a repressor of the DeoR family which is encoded by the constitutive gene ulaR located downstream of ulaG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>deof7 is regulated by DeoR , while deoP2 is regulated by both DeoR and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_47">
The promoter regions of these genes can be classified in three groups regulated negatively by ( i ) CytR , or ( ii ) DeoR alone , or ( iii ) by both CytR and DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2575702" target="_blank">2575702</a>]<br></label>
<input id="act_similarity_47", type="checkbox">
<div class="ACT_similar">
<li>The promoter regions for these genes can be classified in three regulatory groups : i ) DeoR regulated , ii ) CytR regulated and iii ) CytR and DeoR regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>The promoter regions can be classified in three regulatory groups , i ) CytR regulated , ii ) DeoR regulated and iii ) CytR and DeoR regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li></div>
<li>deoO is the sites recognized by DeoR ; CRP and CRP2 the cAMP/CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>As all strains used in this work are deoR + and the DeoR operator is not perturbed in any of the plasmid constructs , P-galactosidase expression from pJEL134 reflects only cAMP-CRP activation and CytR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_52">
DNA sequences involved in expression and regulation of DeoR , CytR and cAMI ' : CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></label>
<input id="act_similarity_52", type="checkbox">
<div class="ACT_similar">
<li>DNA sequences involved in expression and regulation of DeoR , CytR , and cAMP-CRP controlled genes in Escherichia coli , p. 273-288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_54">
Valentin-Hansen , P. ( 1985 ) DNA sequences invoived in expression and regulation of DeoR , CytP and cAMP-CRP controlled genes in E coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></label>
<input id="act_similarity_54", type="checkbox">
<div class="ACT_similar">
<li>Valentin-Hansen , P. ( 1985 ) DNA sequences involved In expression and regulation of DeoR , CytR and cAMP-CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li></div>
<li>As first demonstrated by Krieger-Brauer and Braun ( 19 ) , expression of the tsx gene is under double negative control of the DeoR and CytR proteins , which are known to regulate the synthesis of two nucleoside transport systems ( NupC and NupG ) and several nucleoside-catabolizing enzymes ( for reviews , see references 14 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The nupC and nupG genes also differ in their modes 01 regulation and expression ; nupC has been shown to be subject to repressíon by the CytR protein whilst nupG is repressed by both CytR and DeoR ( Munch-Petersen and Mygind 1976 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>Expression of the fused genes increased in the following order : wild-type < deoR < cytR < deoR cytR double mutant , where the absence of the DeoR repressor increased tsx-lacZ expression about 1.5 - to 2-fold and the absence of CytR resulted in 6-fold-enhanced , B-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Furthermore , deo repression by CytR and DeoR is cooperative rather than additive ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>This is achieved by nesting levels of local repression , mediated by DeoR and CytR , on a more global regulation mediated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969182" target="_blank">8969182</a>]<br></li>
<li>By taking advantage of the strong repression of tsx-lacZ expression in cells carrying the cloned cytR + gene , we have been able to isolate a mutant that shows strongly decreased repression by CytR but no change in regulation by DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Therefore , the results presented in this study point to important differences in the regulation of tsx and the deo operon by CytR and DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>From these data we conclude that Tsx synthesis is subject to double negative control by the DeoR and CytR repressors , but is most strongly affected by positive regulation through the cAMPCAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Our data strongly indicate that such a cooperation between DeoR and CytR repressors does not play any role in tsx regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>It is well established that tsx expression is subject to catabolite repression ( 1 , 20 , 24 ) , but it is unknown how this positive regulatory mechanism interacts with DeoR - and CytR-mediated regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Thus , the extent of the DeoR - and CytR-mediated repression of tsx expression appears to be additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>A mutant was isolated that showed unaltered tsx regulation by DeoR and the cAMP-CAP complex but strongly reduced repression by CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>However , the interrelationship of the double negative ( DeoR ; CytR ) and positive ( cAMP-CAP ) control in tsx expression has not been analyzed to any great extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Thus , the positive regulation of tsx expression mediated by the cAMP-CAP complex has a stronger effect than the negative regulation by the DeoR and CytR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The use of tsx-lacZ fusions has allowed us to study the interrelationship of the DeoR , CytR , and cAMP-CAP regulatory proteins and to quantitate their contribution to the regulation of tsx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The DeoR and CytR regulatory proteins repress tsx transcription to different extents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The DeoR protein exerts only a weak negative effect on Tsx synthesis , whereas CytR represses tsx expression more strongly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The expression of the tsx gene is increased in the presence of nucleosides through the action of the DeoR and CytR repressors ( 98 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>This operon is expressed from two differently controlled promoters , P1 and P2 , which are separated by 599 base pairs ; the former is regulated by the DeoR repressor , and the latter is regulated by both the CytR and DeoR repressors ( 10 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>This clearly demonstrates that the DeoR and CytR control of Tsx synthesis can be genetically separated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>It has been shown that the nupG promoter is activated 50-fold by the cyclic AMP-cyclic AMP receptor protein complex and repressed fourfold by CytR and DeoR and that it has binding sites for these proteins ( 29 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>regulated negatively by the two repressors DeoR and CytR and is activated by the cyclic AMP ( cAMP ) receptor protein-cAMP complex ; however , even under completely repressed conditions , a low basal level of purine nucleoside phosphorylase is maintained ( for a review , see reference 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559336" target="_blank">7559336</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; cAMP , cyclic AMP ; CRP , catabolite activator protein ; cytR , gene encoding CytR ; CytR and DeoR repressor proteins ; A , deletion ; deoR , gene encoding DeoR ; kb , is part of this regulon ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>This difference in the mRNA le-vel and the b-galactosidase activity could be a result of the complex control of the deo operon by other control proteins such as DeoR , CytR and CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The deo operon is controlled by two proteins , CytR and DeoR , in addition to CRP-cAMP and is expressed from two promoters ( ValentinHansen et al. 1978 ; Valentin-Hansen 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The control of deo operon is complex involving two other repressors , DeoR and CytR as well as CRP-cAMP ( Valentin-Hansen et al. 1978 ; Valentin-Hansen 1982 ; Short and Singer 1984 ; Shin et al. 2001 ) , and the absence of these proteins in the assay could account for the higher than expected amount of ModE required for this DNA mobility shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Genetic data suggest that the synthesis of the larger mRNA species is directed by a weak promoter ( t ` I ) that is controlled by the DeoR repressor , whereas the smaller mRNA species is directed by the main promoter 1 ' 2 , which is negatively controlled by the CytR repressor and positively affected by the cyclic AMP/catabolite activator protein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>The Tsx protein , which enhances nucleoside permeation across the outer membrane , is subject to dual control by the DeoR and CytR repressors in addition to being positively controlled by the cyclic AMP-catabolite gene activator protein complex ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644187" target="_blank">2644187</a>]<br></li>
<li>Initiation of transcription from deoP2 is negatively controlled by two repressors , DeoR and CytR , and is also dependent on the cAMP/CRP complex Location of transcription terminators in the 3 ' - end of the deo operon In order to locate termination signals , we inserted restriction fragments from the 3 ' - end of the deo operon between the deoP2 promoter and the lacZ gene of pJEL144 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3299264" target="_blank">3299264</a>]<br></li>
<li>INTRODUCTION The structural genes encoding the ~ ystems for nucleoside uptake and metabolism in E. coli are organized in a regulon that is controlled positively by the cAMP/CRP activator complex and negatively by the DeoR and CytR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Initiations from P1 have been shown to be controlled negatively by the DeoR repressor , whereas initiations from P2 are dependent on the cAMP/CRP complex and are negatively controlled by both the CytR and DeoR repressor ( Valentin-Hansen , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>At P1 and P2 transcription is repressed by the DeoR repressor and induced by deoxyribo-se-5-P ; at P2 transcription is also repressed by the CytR repressor ( the inducer being cytidine or adenosine ) and depends on cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>edtsx mRNA are indicated by arrows Genetic experiments have previously indicated that tsx expression is mediated by two differently regulated promoters : a minor promoter , PI , which is negatively controlled by DeoR and a main promoter , P2 , which is repressi-ble by CytR and activated by the cAMP/CRP complex ( Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Taken together , the results of the S 1 mapping experiment mad the regulatory features of tsx gene expression ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) strongly suggest that transcription initiating from ~ he weak tsx P1 promoter is repressible by DeoR , whereas the activity of the main tsx promoter , 1 ' 2 , is stimulated by the cAMP/CRP complex and negatively controlled by the CytR repre ~ sor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>Our results suggest that tsx expression is controlled at two separate and differently regulated promoters : the weaker promoter ( P1 ) is repressible by DeoR , while the stronger promoter ( P2 ) is subject to negative and positive control by the CytR repressor and the cAMP-CAP complex , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The weaker promoter ( P1 ) is repressible by DeoR , While the main promoter ( P2 ) is subject to negative and positive control by CytR and the cAMP-CAP complex , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>In deoCp1 , promoter control is mediated by the DeoR repressor , whereas deoCp2 is subject to negative regulation by the DeoR and CytR repressors and activation by the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex ( 29 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>The weak tsx-pl promoter is negatively regulated by the DeoR repressor , whereas the main promoter , tSx-p2 , is subjected to both negative control by CytR and activation by the cAMP-CRP complex ( 3 , 4 , 17 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
<li>The minor tsx-p1 promoter is negatively controlled by the DeoR repressor , whereas the main promoter , tsX-p2 , is subjected to positive regulation by the cAMP-CRP activator complex and negative control by the CytR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">DeoR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Fusion of the deoPl-P2 region to lacZ the interaction of with the ` intact ' deoPl-P2 To study DeoR promoter region we constructed three types of protein fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>Residues in DeoR which are found in identical positions in the other repressors are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>16 residues in this region of DeoR can be found in positions common to one or more of the other five repressors ( underlined in Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Binary comparison of AgaR ( ORF f269 ) with protein members of the DeoR family The FASTA program ( Pearson & Lipman , 1988 ) , was used to assess similarities for segments having the indicated number of compared residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>An arginine residue located within the putative recognition helix of this motif is absolutely conserved among members of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>An analysis of the C-terminal domain of DcuR using the MOTIF program ( http://www.genome.ad.jp/SIT/SIT.html ) revealed a probable DNA-binding helix-turn-helix motif ( res-idues 177 to 218 ) resembling that of the GntR and DeoR families of bacterial gene regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
<li>Putative Ligand-binding Site -- LsrR is a repressor in the large SorC/DeoR family of prokaryotic transcriptional regulators , and it contains an N-terminal DBD and a C-terminal LBD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23589368" target="_blank">23589368</a>]<br></li>
<li>The carboxy-terminal region of proteins in the DeoR / GlpR family is responsible for protein oligomerization and inducer-binding ( van Rooijen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Computer-assisted analysis of the nucleotide sequence strongly suggests that the DNA-binding domain of the DeoR repressor is located in the N-terminal part of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>In DeoR only a single region , starting 21 residues from the N-terminal , shows strong homology to the DNA-binding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>The strong homology and similar secondary structure predictions suggest that rpsidues number 22-44 in DeoR are involved in binding to the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>The alignment of the N-terminal helix-turn-helix DNA-binding regions of proteins comprising the DeoR family is shown in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>Helix-turn-helix motifs of the 14 homologous DNA-binding proteins of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>The ﬁrst two residues of the recognition helix are not conserved among the DeoR family members and are therefore thought to confer DNA sequence speciﬁcity for repressor binding to the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>These results suggest that not only is the recognition helix important for determining binding speciﬁcity for the DeoR family members but that the supporting structure also must play a role in DNA sequence binding speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">DeoR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Other common regulator families are ArsR ( 175 ) , AsnC ( 140 ) , Crp ( 233 ) , DeoR ( 17 ) , GntR ( 29 , 104 , 213 ) , IclR ( 214 ) , LacI ( 253 ) , LuxR/UphA ( 112 ) , LysR ( 111 ) , MarR ( 56 ) , MerR ( 107 ) , NtrC ( 189 ) , TetR ( 198 ) , YedF/YeeD/YhhP ( 14 ) , and YhdG/YjbN/YohI ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Members Group function Ec1 So2 Bs3 Group function LuxR/UhpA or OmpR family transcriptional response regulator of two-component regulatory system LysR family transcriptional regulator GntR or DeoR family transcriptional regulator sensory histidine kinase in two-component regulatory system sigma54 activator protein , enhancer binding protein AraC/XylS family transcriptional regulator ROK family transcriptional regulator/sugar kinase IclR family transcriptional regulator methyl-accepting chemotaxis protein MerR family transcriptional regulator DNA-binding regulatory protein AraC/XylS family transcriptional regulator MarR family transcriptional reguator AsnC family transcriptional regulator 46 20 18 18 13 13 11 10 9 18 14 13 13 11 16 13 8 1 oxidoreductase , Fe-S-binding 31 10 11 13 6 oxidoreductase , NAD ( P ) - binding oxidoreductase4 , NAD ( P ) - binding aldehyde dehydrogenase , NAD ( P ) - binding oxidoreductase , FAD/NAD ( P ) - binding sugar kinase respiratory reductase , alpha subunit aldo-keto reductase , NAD ( P ) - binding phosphatase nucleoside diphosphate ( Nudix ) hydrolase 43 30 22 5 14 14 7 8 8 7 5 8 8 2 7 5 1No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>It can be seen that CytR in this region is more closely related to GalR , DeoR and LacR than to the other proteins , showing the highest homology to GalR ( 70 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>An analysis of the C-terminal domain of DcuR using the MOTIF program ( http://www.genome.ad.jp/SIT/SIT.html ) revealed a probable DNA-binding helix-turn-helix motif ( res-idues 177 to 218 ) resembling that of the GntR and DeoR families of bacterial gene regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
<li>A grouping of Rhizobiales common to plant symbioses had a conserved set of adjacent , low homology elements and nonscoring elements that consisted of unidirectionally expressed genes annotated as : ABC-type transporters ( ribose , putative , putative ) , a DeoR family repressor ( LsrR synonym ) , aldo-keto reductase ( non-scoring ) , glycerol-3-phosphate dehydrogenase ( nonscoring LsrF synonym ) , fructose-biphosphate aldolase ( LsrF synonym ) , and xylulose kinase ( LsrK synonym ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22916001" target="_blank">22916001</a>]<br></li>
<li>Introduction Members of the DeoR family of transcriptional regulators are present in a variety of bacterial organisms , ranging from Gram-positive bacteria ( e.g. Lactococcus lactis and Streptococcus mutans ) to Gram-negative bacteria ( e.g. Agrobacter-ium tumefaciens and Pseudomonas aeruginosa ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>GlpR ( Larson , AgaR repressor and dominant repressor variants 815 et al. , 1987 ) , DeoR ( Valentin-Hansen et al. , 1986 ; Mortensen et al. , 1989 ) and possibly all members of this family of proteins are active as oligomers of identical subunits that bind to multiple palindromic operator sites on DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Examples of carbohydrate catabolic systems that are regulated by repressors belonging to the DeoR family include those of the glycerol-P ( GlpR ) , glucitol ( GutR ) , fucose ( FucR ) , L-ascorbate ( UlaR ) and deoxyribonucleoside ( DeoR ) systems of E. coli , as well as those of the lactose ( LacR ) , fructose ( FruR ) and inositol ( IolR ) systems of Gram-positive bacteria and agrocinopine ( AccR ) of A. tumefaciens ( Valentin-Hansen et al. , 1985 ; Yamada & Saier , 1988 ; Lu & Lin , 1989 ; van Rooijen & de Vos , 1990 ; Beck von Bodman et al. , 1992 ; Yoshida et al. , 1995 ; Zeng et al. , 1996 ; Campos et al. , 2004 ; Barriere et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>Taken together , our data show that unlike other members of the DeoR family that act as repressors of one target regulon only ( Valentin-Hansen et al. , 1985 ; Yamada & Saier , 1988 ; Lu & Lin , 1989 ; van Rooijen & de Vos , 1990 ; Beck von Bodman et al. , 1992 ; Yoshida et al. , 1995 ; Zeng et al. , 1996 ; Campos et al. , 2004 ; Barriere et al. , 2005 ) , DeoT acts as a regulator capable of affecting the expression of multiple unrelated genes of different regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>The ula regulon is under the control of the UlaR repressor ( Campos et al. , 2004 ) , which belongs to the DeoR repressor family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>Proteins of the DeoR family exhibit several common features ( van Rooijen & de Vos , 1990 ; Reizer et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>Features common to the members of the DeoR family have been pointed out by van Rooijen and de Vos ( 50 ) and by Reizer et al. ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>This is also the case in the DeoR repressor family , which includes , among others , the deo operon and the sn-glycerol-3-P regulon of E. coli ( 8 , 16 , 21 , 30 ) and the iol operon of B. subtilis ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>From our studies and those reported by other authors ( 16 , 21 , 30 ) , we may conclude that activation by cAMP-CRP and the repressor effect caused by binding to several operator sites are general features of the genetic systems under the control of repressors belonging to the DeoR family in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The FruR protein contains a typical helix-turn-helix motif , common to regulatory proteins , and shows similarity to the Lacd , GalR , PurR , RbtR , and DeoR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The genes within this cluster include : ( 1 ) five genes encoding homologues of the E. coli mannose permease of the phosphotransferase system ( IIB , IIB ' , IIC , IIC ' and IID ) ; ( 2 ) genes encoding a putative N-acetylgalactosamine 6-phosphate metabolic pathway including ( a ) a deacetylase , ( b ) an isomerizing deaminase , ( c ) a putative carbohydrate kinase , and ( d ) an aldolase ; and ( 3 ) a transcriptional regulatory protein homologous to members of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>AgaR ( ORF f269 ) The agaR gene , divergently transcribed from the other genes in the aga gene cluster ( Fig. 1 ) ` encodes a homologue of a family of transcriptional regulatory proteins known as the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>Binary comparison of AgaR ( ORF f269 ) with protein members of the DeoR family The FASTA program ( Pearson & Lipman , 1988 ) , was used to assess similarities for segments having the indicated number of compared residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>While this work was in progress , an additional protein member of the DeoR family was identified in B. mbtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>Most of the DeoR family regulators already characterized are involved in sugar metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>One member of the DeoR family , FucR , is known to activate transcription of genes involved in metabolism of fucose ( Chen et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>However , the binding speciﬁcity of the DeoR family members to their operators seems to be affected by multiple determinants , as indicated by the result obtained with the repressors GlpR and DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>The alignment of the N-terminal helix-turn-helix DNA-binding regions of proteins comprising the DeoR family is shown in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>Helix-turn-helix motifs of the 14 homologous DNA-binding proteins of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>Such bending may facilitate DNA looping that seems to be a common theme among the DeoR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>The ﬁrst two residues of the recognition helix are not conserved among the DeoR family members and are therefore thought to confer DNA sequence speciﬁcity for repressor binding to the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>An arginine residue located within the putative recognition helix of this motif is absolutely conserved among members of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>The Escherichia coli family of DeoR regulators contains at least 14 members , of which seven have deﬁned functions and the rest are deﬁned based on sequence similarities ( Perez-Rueda & Collado-Vides , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>Putative Ligand-binding Site -- LsrR is a repressor in the large SorC/DeoR family of prokaryotic transcriptional regulators , and it contains an N-terminal DBD and a C-terminal LBD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23589368" target="_blank">23589368</a>]<br></li>
<li>Ray , W.K. ( 1999 ) Characterization of AgaR and YihW , members of the DeoR family of transcriptional regulators , and GlpE , a rhodanese belonging to the GlpR regulon , also a member of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931310" target="_blank">10931310</a>]<br></li>
<li>Arginine is highly conserved at this position in the DeoR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>The glp repressor is a member of the DeoR family of transcriptional regulators , which at present contains 19 members on the basis of similarity searches of the databases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>The corresponding amino acid sequence ( P39299 ) was used as the query sequence for a BLAST P search , and a high degree of similarity was found between this sequence and regulator proteins of the DeoR repressor family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12374842" target="_blank">12374842</a>]<br></li>
<li>A signature sequence of the DeoR family has been derived ( unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by DeoR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Gene NaSal-activated 2.3 2.3 b4014 b1276 b1241 b2252 b0864 b0485 b0881 b1112 b1164 b1165 b1200 b1450 b1452 b1643 b1795 b2174 b2266 b2672 b3004 b3024 b3238 b3242 b1661 b3806 b2414 b4382 b4383 b0014 b0812 b0593 b3289 b1611 b3517 b1493 b1779 b2094 b2093 b2092 b2091 b2090 b2095 b0720 b3212 b2947 b3510 b3509 b2237 b1732 b3604 b3603 b4129 b1817 b1818 b1531 b1532 b1530 b3028 b3601 b0578 b1482 b4376 b1897 b0932 b0903 b1014 b3506 b3284 b3908 b2703 aceB acnA Malate synthase A Aconitase A 2.2 3.8 2.5 2.7 2.1 9.0 3.4 3.2 5.9 2.3 3.0 2.1 2.9 2.4 2.9 2.6 2.5 2.3 27.6 6.9 3.7 2.4 2.4 2.4 2.1 2.6 7.0 2.2 2.4 8.4 2.7 2.7 2.6 5.3 5.2 2.5 3.8 2.3 2.8 2.6 2.8 3.0 9.9 9.2 3.9 2.1 4.9 2.7 2.1 2.3 2.4 11.3 9.0 20.7 2.2 2.5 3.4 4.4 2.9 2.2 2.1 2.1 3.2 2.1 2.1 2.3 2.7 adhE ais artP b0485 b0881 b1112 b1164 b1165 b1200 b1450 b1452 b1643 b1795 b2174 b2266 b2672 b3004 b3024 b3238 b3242 cfa cyaA cysK deoA deoB dnaK dps entC fmu fumC gadA gadB gapA gatA gatB gatC gatD gatZ gltA gltB gshB hdeA hdeB inaA katE lctR lldP lysU manX manY marA marB marR mdaB mtlR nfnB osmC osmY otsB pepN pﬂB putA slp smg sodA Acetaldehyde-coenzyme A dehydrogenase Aluminum-inducible protein ; function unknown Arg periplasmic transport system Putative glutaminase Function unknown Function unknown Function unknown Function unknown Putative dihydroxyacetone kinase Putative transcriptional regulator ( GntR family ) Function unknown Function unknown Function unknown Putative permease Function unknown Function unknown Function unknown Function unknown Function unknown Function unknown Cyclopropane fatty acid synthase Adenylate cyclase Cysteine synthase Thymidine phosphorylase Deoxyribouratase , phosphopentomutase HSP-70-type molecular chaperone Stress response DNA-binding protein Isochorismate synthetase 16S rRNA m5C967 methyltransferase , S-adenosyl-L-methionine dependent Fumarase C , aerobic ; member of soxRS regulon Glutamate decarboxylase Glutamate decarboxylase GAPDH A Galactitol-speciﬁc enzyme IIA of PTSb Galactitol-speciﬁc enzyme IIB of PTS Galactitol-speciﬁc enzyme IIC of PTS Galactitol-1-phosphate dehydrogenase Split transcriptional repressor of galactitol utilization , fragment 2 ( DeoR family ) Tagatose 6-phosphate aldolase 2 , subunit with GatY Citrate synthase Glutamate synthase , large subunit Glutathione synthetase Periplasmic , unknown function , has sigma S-dependent promoter Periplasmic , unknown function , has sigma S-dependent promoter Function unknown Catalase hydroperoxidase III Regulatory gene for lld operon L-Lactate permease Lysyl tRNA synthetase , inducible PTS family , mannose-speciﬁc enzyme IIA component Mannose PTS , EIIC component Transcription activator of multiple-antibiotic-resistance system Regulatory gene for mar Repressor of mar operon Modulator of drug activity Mannitol repressor Resistance to nitrofurantoin ; a nitroreductase Osmotically inducible protein C ; nonessential gene Periplasmic , sigma S-dependent protein ( stationary phase ) Trehalose phosphate phosphatase Aminopeptidase N Pyruvate formate lyase I ; induced anaerobically Proline dehydrogenase C starvation and stationary phase-inducible ; outer membrane lipoprotein Function unknown Member of soxRS regulon ; superoxide dismutase , Mn PTS family , glucitol/sorbitol-speci ﬁc enzyme IIB component Continued on following page TABLE 2 -- Ratio 2.9 2.1 2.1 2.2 2.7 2.6 2.3 3.4 2.0 2.9 2.3 3.3 3.9 2.2 2.8 Continued Gene type No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>DISCUSSION Transcriptional control of metabolic pathways for carbohydrate utilization in bacteria is mediated by a variety of TFs from different protein families ( e.g. AraC , LacI , GntR , DeoR , and ROK ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>x Editor : Marco Soria Keywords yciT ; DeoR ; transcription control ; operon ; repression ; global regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>DeoR , a classical bacterial repressor , acts alone and regulates transcription initiation negatively from a set of the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>This is also the case in the DeoR repressor family , which includes , among others , the deo operon and the sn-glycerol-3-P regulon of E. coli ( 8 , 16 , 21 , 30 ) and the iol operon of B. subtilis ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Elgrably-Weiss , M. , Schlosser-Silverman , E. , Rosenshine , I. , and Altuvia , S. ( 2006 ) DeoR , a DeoR-type transcriptional regulator of multiple target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>The genes within this cluster include : ( 1 ) five genes encoding homologues of the E. coli mannose permease of the phosphotransferase system ( IIB , IIB ' , IIC , IIC ' and IID ) ; ( 2 ) genes encoding a putative N-acetylgalactosamine 6-phosphate metabolic pathway including ( a ) a deacetylase , ( b ) an isomerizing deaminase , ( c ) a putative carbohydrate kinase , and ( d ) an aldolase ; and ( 3 ) a transcriptional regulatory protein homologous to members of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>AgaR ( ORF f269 ) The agaR gene , divergently transcribed from the other genes in the aga gene cluster ( Fig. 1 ) ` encodes a homologue of a family of transcriptional regulatory proteins known as the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>Abbreviations : aa , amino acid ( s ) ; bp , base pair ( s ) ; cAMP , cyclic AMP ; CRP , catabolite activator protein ; cytR , gene encoding CytR ; CytR and DeoR repressor proteins ; A , deletion ; deoR , gene encoding DeoR ; kb , is part of this regulon ( Krieger-Brauer and Braun , 1980 ; Bremer et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
<li>The Tsx protein , which enhances nucleoside permeation across the outer membrane , is subject to dual control by the DeoR and CytR repressors in addition to being positively controlled by the cyclic AMP-catabolite gene activator protein complex ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644187" target="_blank">2644187</a>]<br></li>
<li>As first demonstrated by Krieger-Brauer and Braun ( 19 ) , expression of the tsx gene is under double negative control of the DeoR and CytR proteins , which are known to regulate the synthesis of two nucleoside-transport systems ( NupC and NupG ) and several nucleoside-catabolizing enzymes ( for reviews , see references 14 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The DeoR repressor is known to regulate the deoCABD genes , encoding deoxy - and ribonucleoside catabolizing enzymes ( 3-5 ) , and the nupG and tsx genes which encode a transport protein and a pore-forming protein , respectively ( 6,7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Most of the DeoR family regulators already characterized are involved in sugar metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>However , there are some examples of DeoR regulators involved in cell responses to environmental stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>INTRODUCTION The structural genes encoding the ~ ystems for nucleoside uptake and metabolism in E. coli are organized in a regulon that is controlled positively by the cAMP/CRP activator complex and negatively by the DeoR and CytR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2265760" target="_blank">2265760</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
Valentin-Hansen , P. ( 1985 ) DNA sequences invoived in expression and regulation of DeoR , CytP and cAMP-CRP controlled genes in E coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1962841" target="_blank">1962841</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>Valentin-Hansen , P. ( 1985 ) DNA sequences involved In expression and regulation of DeoR , CytR and cAMP-CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1649947" target="_blank">1649947</a>]<br></li>
<li>DNA sequences involved in expression and regulation of DeoR , CytR and cAMI ' : CRP controlled genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328649" target="_blank">1328649</a>]<br></li>
<li>DNA sequences involved in expression and regulation of DeoR , CytR , and cAMP-CRP controlled genes in Escherichia coli , p. 273-288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li></div>
<li>mutans ( Rosey & Stewart , 1992 ) , respectively ; DeoR ( Eco ) , repressor of deoCABD that encodes nucleoside-catabolizing enzymes in E. coli ( ValentinHansen et al. , 1985 ) ; FucR ( Eco ) , transcriptional regulator ( activator ) of L-fucose-metabolism in E. coli ( Lu & Lin , 1989 ) ; SgcR ( Eco ) , putative transcriptional regulator in the sgc gene cluster of E. coli ( GenBank accession no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>DeoR and GlpR of E. coli regulate the expression of genes involved in the metabolism of deoxyribonucleosides and sn-glycerol 3-phosphate respectively ( Mortensen et al. , 1989 ; Zeng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>One member of the DeoR family , FucR , is known to activate transcription of genes involved in metabolism of fucose ( Chen et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Its primary sequence is highty homotogous to GaIR , DeoR , Lad , PurR , and Mall repressor proteins ( Vatentin-Hansen etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>The amino acid sequence of the glp repressor was similar to several repressors of carbohydrate catabolic systems , including those of the glucitol ( GutR ) , fucose ( FucR ) , and deoxyribonucleoside ( DeoR ) systems of E. coli , as well as those of the lactose ( LacR ) and inositol ( IolR ) systems of gram-positive bacteria and agrocinopine ( AccR ) system of Agrobacterium tumefaciens .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Catabolite response Deoxirribose assimilation D 87 135 Crp w 27 Pérez - Rueda and Collado-Vides ( 2000 ) 220 -- R 235 10 DeoR DicA 35 w Pérez - Rueda and Collado-Vides ( 2000 ) -- n/d 0 -- -- n/d HLH R 4 DicC DnaA 0 Transcription and replication regulation Positive regulator of cytochrome c552 77 0 -- Pérez - Rueda and Collado-Vides ( 2000 ) 41 -- n/d DniR 0 Kajie et al. ( 1991 ) 209 8 215 -- A Ebp FhlC 77 m Pérez - Rueda and Collado-Vides ( 2000 ) Pérez - Rueda and Collado-Vides ( 2000 ) n/d 0 -- 7 -- Pérez - Rueda and Collado-Vides ( 2000 ) Schnetz ( 2002 ) n/d R FhlD FrvR 0 -- Phosphoenolpyruvate sugar phosphotransferase system operon regulation Iron assimilation 8 0 -- -- 128a Fur R w Achenbach and Yang ( 1997 ) 107 460 35 Carbon sources assimilation Nguyen and Saier ( 1995 ) ; Weickert and Adhya ( 1992 ) 388 GalR/LacI 55 R c -- 13 GcvR GntR Rigali et al. ( 2002 ) Glycine cleavage operon repression 0 R 735 775 Ghrist and Stauffer ( 1998 ) 96 R w Yamada and Saier ( 1988 ) Activator of the glucitol operon in E. coli High-frequency persistence of the lethal effects of inhibition of either DNA or peptidoglycan-synthesis regulation ( HipAB ) -- 8 GutM 0 A -- Black et al. ( 1991 ) -- 1 HipA 1 0 n/d -- n/d HipB HtgA Black et al. ( 1991 ) Missiakas et al. ( 1993 ) 87 0 -- -- -- n/d 7 Heat shock regulation 0 -- HycA HypF 6 -- n/d ZnF A 0 -- 0 Bouhouche et al. ( 2000 ) Gui et al. ( 1996 ) 27 -- Glyoxylate bypass operon Integration host factor 31 106 161 w R -- IclR Ihf -- 15 8 Pérez - Rueda and Collado-Vides ( 2000 ) Campoy et al. ( 2002 ) -- D 43 -- DNA-repair 54 LexA R w 102 279 Miller and Bassler ( 2001 ) Schell ( 1993 541 LuxR/UhpA m A Quorum sensing 1548 LysR 1271 Amino acid biosynthesis regulation D w 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">DeoR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Accordingly , a helix-turn-helix motif was found in the amino-terminal end , matching the signature sequence of the DeoR family proposed by Reizer et al. ( 10 ) , RX3 [ LIVM ] X3 [ LIVM ] X20T [ LIVMA ] R [ RKAN ] D [ LIVMF ] ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12374842" target="_blank">12374842</a>]<br></li>
<li>Mutant anatysis and binding studies with the purified DeoR repressor have shown that DeoR recognizes a 16-bp patindromic sequence ( Dandanetl and Hammer 1985 ; Mortensen etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162467" target="_blank">2162467</a>]<br></li>
<li>Recently , Oc mutants in the P1 target have been isolated , which strongly indicates that the DeoR recognition sites encompass the Pribnow box sequence of each of the two promoters ( Dandanell and Hammer , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>We have recently shown that both P1 and P2 are regulated by the DeoR repressor and oc mutations as well as sequence homology strongly indicate that the DeoR recognition sites overlap the Pribnow box sequences of each of the two promoters ( Valentin-Hansen et al. , 1982a ; Valentin-Hansen , 1985 ; Dandanell and Hammer , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>Interestingly , a region located just downstream from the start of deoR ( from bp 38 to 53 ) shows a striking homology with these two targets for DeoR ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>deoO is the sites recognized by DeoR ; CRP and CRP2 the cAMP/CRP targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Due to the sequence homology we have suggested that these repeats are the operator sites for DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3019678" target="_blank">3019678</a>]<br></li>
<li>The 16-bp sequence in the tsx-pl promoter showing homology to known DeoR operator sites and two regions in the tsX-p2 promoter with homology to cAMP-CRP DNA-binding sites are indicated ; lines extending from the cAMP-CRP targets indicates those sequences which are protected by cAMP-CRP against DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715855" target="_blank">1715855</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by DeoR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>AgaR ( ORF f269 ) The agaR gene , divergently transcribed from the other genes in the aga gene cluster ( Fig. 1 ) ` encodes a homologue of a family of transcriptional regulatory proteins known as the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>The UlaR repressor belongs to the DeoR family of bacterial The ula regulon , responsible for the utilization of L-ascorbate in Escherichia coli , is formed by two divergently transcribed operons , ulaG and ulaABCDEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The regulon is negatively regulated by a repressor of the DeoR family which is encoded by the constitutive gene ulaR located downstream of ulaG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>For the deoCABD operon it has been shown that transcription is initiated from two promoters that are differentially regulated by DeoR and CytR ( Valentin-Hansen et 01996 Blackwell Science Ltd , Molecular Microb/o/ogy ,22 , 21-2 a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>At P1 and P2 transcription is repressed by the DeoR repressor and induced by deoxyribo-se-5-P ; at P2 transcription is also repressed by the CytR repressor ( the inducer being cytidine or adenosine ) and depends on cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2994018" target="_blank">2994018</a>]<br></li>
<li>Their expression should , consequently , not have been affected by DeoR , i.e. expression patterns of cdd and udp in S0303 ( deoR - ) and CHS50 ( deoR ' ) should have been identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Therefore , our result suggests that expression of cdd and udp might also be regulated by DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>As S0744 is deficient for both repressors ( DeoR and CytR ) , this result indicated that FIS not only affected transcriptionof deoRand cytR , but also directly influenced the expression of cdd , udpand deoC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>The nupC and nupG genes also differ in their modes 01 regulation and expression ; nupC has been shown to be subject to repressíon by the CytR protein whilst nupG is repressed by both CytR and DeoR ( Munch-Petersen and Mygind 1976 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022285" target="_blank">8022285</a>]<br></li>
<li>As first demonstrated by Krieger-Brauer and Braun ( 19 ) , expression of the tsx gene is under double negative control of the DeoR and CytR proteins , which are known to regulate the synthesis of two nucleoside transport systems ( NupC and NupG ) and several nucleoside-catabolizing enzymes ( for reviews , see references 14 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Plasmid pCB008 carrying the cytR ' gene ( 3 ) strongly repressed the expression of the D ( tsx-lacZ ) 1 ( Hyb ) protein fusion regardless of the presence of the chromosomally encoded DeoR and CytR repressors ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>The expression of the tsx gene is increased in the presence of nucleosides through the action of the DeoR and CytR repressors ( 98 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>It is of interest to compare the regulatory pattern of tsx expression with those of other DeoR - and CytR-controlled genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>However , the interrelationship of the double negative ( DeoR ; CytR ) and positive ( cAMP-CAP ) control in tsx expression has not been analyzed to any great extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
<li>Thus , the extent of the DeoR - and CytR-mediated repression of tsx expression appears to be additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2826388" target="_blank">2826388</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>