<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - DhaR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">DhaR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">DhaR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The protein was eluted with 200 mM imidazole in buffer A , and DhaR and DhaR ( N ) containing fractions were pooled and dialyzed against buffer B ( 20 mM Tris -- HCl pH 8.0 , Table II Strains and plasmids Strain/plasmid Relevant genotype or structures Source/reference Bacterial strains MC4100 F - , araD139 , D ( argF-lac ) U169 , prsL150 , relA1 , deoC1 , rbsR , fthD5301 , fruA25 , l-as MC4100 but DptsI F - , l - , ls , Dlac ( MS265 ) , mel , NalAr , supF58 ( 1/4 su III + ) MC4100 attBlHPdhaK E. coli-lacZ-bla TH074 attBlHPdhaK E. coli-lacZ-bla As CBZ but DdhaR As CBZ but DdhaKL As CBZDI but DdhaKL As CBZ but DdhaKLM As CBZDI but DdhaKLM As CBZ but DdhaM As CBZ but DrpoN As CBZDI but DdhaR As CBZDI but DdhaM As MC4100 but DdhaKLM F - , l - , end A1 , hsdR17 , hsdM + , supE44 , thi1 , recA96 , relA1 D ( argF lacZYA ) U169 , F80d , D ( lacZ ) M15 , tetR + Lab stock TH074 DHB6521 Hesterkamp and Erni ( 1999 ) Boyd et al ( 2000 ) This work This work This work This work This work This work This work This work This work This work This work This work Lutz et al ( 1997 ) CBZ CBZDI CBZDR CBZDKL CBZDIDKL CBZDKLM CBZDIDKLM CBZDM CBZDrpoN CBZDIDR CBZDIDM CBDKLM DH5aZ Plasmids pBR322 bla , tetRA , oriR ColE1 PT7 , lacI , kan , oriR ColE1 , His tag Lab stock Novagen q Plac/ara -1 , oriR SC101 * , kan PLtetO-1 , oriR colE1 , kan luc under control of PLtetO-1 ( oriR , p15A , cat ) dhaK ( H230K ) L under control of Ptac ( oriR ColE1 , bla ) dhaK with C-terminal H6 tag under control of Ptac ( oriR ColE1 , bla ) dhaL with C-terminal H6 tag under control of Ptac ( oriR ColE1 , bla ) dhaM with C-terminal H6 tag under control of Ptac ( oriR ColE1 , bla ) dhaM ( H9A , H169A , H430A ) under control of Ptac ( oriR ColE1 , bla ) pET28 pZS * 24-MCS-1 pZE21-MCS-1 pZA31-Luc pJF K ( H230K ) L pJF K H6 pJF L H6 pJF M H6 pJFM ( H9A , H169A , H430A ) pJFRH6 pAC R Ec cterm pAC K pAC M pAC M ( H9A ) pAC M ( H169A ) pAC M ( H430A ) pAC ptsI pAC R Ec pAC R Cf pZStetR MCS pZS KLM pZS KL pZS L pZS K ( H230K ) L pZS KL cf pET28 R ( N ) pBR PdhaK-lacZ pBR PdhaR-lacZ Lutz et al ( 1997 ) Lutz et al ( 1997 ) Lutz et al ( 1997 ) This work This work This work This work This work dhaR with C-terminal H6 tag under control of Ptac ( oriR ColE1 , bla ) dhaR ( aa 312 -- 642 ) under control of ParaB , ( oriR pACYC177 , kan ) dhaK under control of ParaB , ( oriR pACYC177 , kan ) dhaM under control of ParaB , ( oriR pACYC177 , kan ) dhaM H9A under control of ParaB , ( oriR pACYC177 , kan ) dhaM H169A under control of ParaB , ( oriR pACYC177 , kan ) dhaM H430A under control of ParaB , ( oriR pACYC177 , kan ) ptsI under control of ParaB , ( oriR pACYC177 , kan ) dhaR E. coli under control of ParaB , ( oriR pACYC177 , kan ) dhaR C. freundii under control of ParaB , ( oriR pACYC177 , kan ) PLtetO-1 , ( MCS , tetR , oriR SC101 * , cat ) dhaKLM under control of PLtetO-1 , ( tetR , oriR SC101 * , cat ) dhaKL under control of PLtetO-1 , ( tetR , oriR SC101 * , cat ) dhaL under control of PLtetO-1 , ( tetR , oriR SC101 * , cat ) dhaK ( H230 K ) L under control of PLtetO-1 , ( tetR , oriR SC101 * , cat ) dhaKL of C. freundii under control of PLtetO-1 , ( tetR , oriR SC101 * , cat ) dhaR ( aa 1 -- 318 ) under control of PT7 ( lacI , oriR pBR322 , kan ) q PdhaK E. coli-lacZ , ( oriR ColE1 , kan ) PdhaR E. coli-lacZ , ( oriR ColE1 , kan ) This w This w This w This w This w This w This w This w This w This w This w This w This w This w This w This w This w This w This w ork ork ork ork ork ork ork ork ork ork ork ork ork ork ork ork ork ork ork 300 mM NaCl , 1 mM DTT , 10 % glycerol , 2 mM EDTA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Dihydroxyacetone ( DHA ) is also a reactive intracellular metabolite , whose role is recently illuminated by the presence of devoted PTS systems , consisting of DhaKLM and DhaR in E. coli ( Bachler et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17846588" target="_blank">17846588</a>]<br></li>
<li>Taken together , this indicates that DhaR like TyrR of E. coli and NtrC of Rhodobacter capsulatus ( Studholme and Dixon , 2003 ) and LevR of Lactobacillus casei ( Maze et al , 2004 ) activates an Es70 rather than a Es54 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Protein mixtures were preincubated , and complexes containing hexahistidine-tagged DhaR or DhaR ( N ) 2 þ were afﬁnity captured with Ni - NTA-beads , analyzed by denaturing polyacrylamide gel electrophoresis and visualized by silver staining .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the presence of inactive DhaM ( H9A , H169A , H430A ) , DhaLHADP was not phosphorylated and the DhaLHDhaR ( N ) complex remained stable ( lanes 9 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>This organism lacks DhaR , clearly suggesting that DhaH regulates DhaK1/K2 and not DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>EI and DhaM thus are negative regulators , and DhaR is a positive regulator ( activator ) of dha operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The dha operon is constitutively active in the absence of a functional DhaM or EI , and inactive in the absence of DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaR remains autorepressing after deletion of the dhaKLM genes as well as in the presence of the activating DhaL subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaR with a signalling domain is necessary for induction by Dha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the presence of DhaR ( N ) , DhaL eluted together with DhaR ( N ) in the relative elution volume corresponding to 125 kDa , a molecular weight compatible with DhaR ( N ) 2DhaL1 stoichio-metry ( Figure 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the absence of Dha , DhaL eluted late as monomer and far behind the 466 kDa DhaR ( N ) HDhaK complex ( Figure 9F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the presence of Dha ( Figure 9G ) , DhaL eluted earlier than the monomer but still behind the forward edge of the DhaR ( N ) 2HDhaK2 protomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In brief , DhaLHADP is the coactivator of DhaR while DhaLHATP is transcription inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the absence of Dha , DhaLHADP is converted by the PTS to DhaLHATP , which does not bind to DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In summary , the genetic analyses of dhaKLM and dhaR transcription show the following : ( i ) DhaR is an autonomous autorepressor of its own synthesis , ( ii ) DhaR and DhaL are positive regulators of the dhaKLM operon , ( iii ) phosphorylated DhaM and DhaK are negative regulators , ( iv ) Dha is the inducer and ( v ) there is no evidence for DhaR being controlled by protein phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In vivo , DhaL is expected to be in the DhaLHATP form ( if EI and DhaM are active and Dha is absent ) and thus without afﬁnity for DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the pulldown experiment , DhaL and DhaR ( N ) are present in equi-molar amounts and DhaL is kept in the phosphorylated form by DhaM-mediated rephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The small proportion of DhaLHADP , which might be present in the steady state ( due to the intrinsic ATPase activity of DhaL ; Bächler et al , to be published ) and which might bind to DhaR ( N ) , is below the limit of gel electrophoretic detection ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The concentrations were as follows : DhaR ( full length ) 100 nM , DhaR ( N ) 100 nM , DhaK 300 nM , DhaLHADP 400 nM , apo-DhaL 400 nM , ADP 1 mM , AMP 1 mM , ADPNP 10 mM , DhaM 5 nM , PEP 1 mM , and EI and HPr 5 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The addition of Dha had no effect on the interaction between DhaK and DhaR as long as Dha could To further elucidate the function of ADP , the nucleotide was removed by gel ﬁltration of DhaL in the presence of EDTA , and apo-DhaL was then reconstituted with AMP and ADPNP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Lanes 11 -- 16 , DhaL apoenzyme ( without ADP ) and apoenzyme complemented with AMP or ADPPNP do not bind to DhaR ( shown only with DhaR ( N ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Neither apo-DhaL nor DhaL supplemented with AMP and ADPNP ( adenylylimidodiphosphonate ) bound to DhaR ( N ) ( Figure 7B , lanes 11 -- 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The catalytic DhaL and DhaK subunits are coactivators and corepressors of DhaR -- controlled gene expression The dhaK , dhaL and dhaM genes were interrupted , and the mutants complemented with plasmids coding for DhaK , DhaL and DhaM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Addition of Dha reduced the estimated mass of the complex to 200 kDa , while DhaR ( N ) and DhaK remained associated ( Figure 9C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>This resistance of the complex between DhaR ( N ) and DhaK was also noticed in pull-down assays performed in the presence of Dha ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaK binds to the sensing domain of DhaR ( Figure 10B ) and thereby keeps DhaR in a transcription inactive state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Two complexes of low electrophoretic mobility formed when DhaR ( N ) and DhaK were present in a 1:1 ratio .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>This and the DhaK to DhaR ( N ) ratio point to a dimer [ DhaR ( N ) 2HDhaK2 ] 2 of DhaR ( N ) 2HDhaK2 proto-mers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The DhaR ( N ) HDhaK complex could be dissociated with less than a four-fold molar excess of DhaLHADP ( Figure 8B ) , conﬁrming that binding of DhaLHADP and DhaK to DhaR ( N ) are mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>It will be shown below that DhaK and DhaL bind to DhaR in a mutually exclusive manner and that DhaLHADP is inactivated by DhaM-mediated phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In conclusion , DhaLHADP and DhaK bind to the sensing domain of DhaR in a mutually exclusive manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Binding of DhaK and DhalHADP to DhaR is mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The pull-down experiments and the genetic analysis ( above ) suggest that binding of DhaL and DhaK to DhaR ( N ) are mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>This indicates that heterotrimeric complexes between the three subunits do not form and is consistent with mutually exclusive binding of DhaL and DhaK to DhaR ( N ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Dha is the inducer that by binding to DhaK reduces the afﬁnity of DhaK for DhaR , and by dephosphor-ylation of DhaLHATP increases the afﬁnity of DhaLHADP for DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In the presence of Dha , DhaL is dephosphorylated and DhaLHADP displaces DhaK and stimulates DhaR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Dephosphorylated DhaLHADP displaces DhaK and activates DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Lane 1 , pull-down of puriﬁed DhaR with background ; lanes 2 and 3 , binary complexes of DhaK and DhaLHADP with DhaR ; lane 4 , competition between DhaK and DhaLHADP for DhaR ; lanes 5 -- 10 , the PTS-dependent phosphorylation of DhaLHADP reduces DhaL afﬁnity for DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Full-length DhaR and DhaR ( N ) captured both DhaK and DhaLHADP with comparable afﬁnity ( Figure 7 , lanes 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The most salient feature of Figure 9F -- H is that DhaK and DhaL never occur together in fractions containing high-molecular-weight complexes between DhaR ( N ) and DhaK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>This indicates that DhaL formed a DhaR ( N ) 2DhaL1 complex ( as in Figure 9A ) by displacing DhaK from DhaR ( N ) but did not bind to the DhaR ( N ) 2HDhaK2 protomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Lanes R , K , L , 5 mM DhaR ( N ) , 10 mM DhaK , 20 mM DhaL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Complexes between DhaR ( N ) , DhaK and DhaL were formed as described in Afﬁnity puriﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In vivo , however , DhaL is present in a large excess over DhaR , and even a small proportion of DhaLHADP might sufﬁce to saturate DhaR in the absence of competing DhaK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Similar to the conditions in vivo ( Table I ) , DhaL and DhaK were present in molar excess over DhaR ( N ) , all other conditions being identical with those of Figure 9C -- E .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Conversely , PdhaR-lacZ reporter gene activity , which is low when DhaR is produced , became constitutively high after disruption of dhaR ( Figure 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Hence , DhaR activity is assumed to be controlled by protein -- protein interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">DhaR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>This indicates that heterotrimeric complexes between the three subunits do not form and is consistent with mutually exclusive binding of DhaL and DhaK to DhaR ( N ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>However , the fact that a DhaR mutant without a sensor domain has reduced autorepressor activity ( Figure 3B ) suggests that sensor domain-dependent dimerization is decisive for strong binding to the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>This indicates that the sensing-domain is an activator of the AAA þ domain and suggests that DhaR activity is not controlled by protein phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Hence , DhaR activity is assumed to be controlled by protein -- protein interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>( B ) Domain structure of DhaR with the N-terminal sensing , the central AAA þ ( ATPase ) and the C-terminal helix -- turn -- helix DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaR with a signalling domain is necessary for induction by Dha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The N-terminal sensing-domain of DhaR features conserved aspartyl ( Asp-79 ) and histidyl ( His-135 ) residues ( Figure 4 ) , which , in principle , could be phosphorylated by a sensor histidine kinase or a component of the PTS , for instance by DhaM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaK and DhaL form protein complexes with the DhaR sensing-domain Full-length DhaR with a carboxy-terminal histidine tag and the sensing-domain DhaR ( N ) ( residues 1 -- 318 ) with an amino-terminal histidine tag were puriﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Therefore , the better soluble N-terminal sensing-domain DhaR ( N ) was used for the experiments shown in Figures 7B and 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>E. coli BL21 ( DE3 ) transformed with plasmid pET28 R ( N ) was used to overproduce the N-terminal sensing-domain DhaR ( N ) with an N-terminal hexahistidine tag .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The AAA þ domain of DhaR indeed does not contain two sequence motives ( ESELF and GAFTGA ) , which are conserved in the EBPs that interact with s54 ( Figure 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Therefore , it can not be excluded that the full-length DhaR in complex with the operator sequence might display additional oligomerization states that are not observable with the sensor domain alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">DhaR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>DhaR also features sequence similarity ( o30 % identity ) to AcoR , the regulator of acetoin metabolism in Thermoanaerobacter tengcongensis and Clostridium magnum ( Figure 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaR belongs to the family of bacterial enhancer binding proteins ( EBP ; Buck et al , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_3">
DhaR is a transcription activator belonging to the AAA þ family of enhancer binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></label>
<input id="evo_similarity_3", type="checkbox">
<div class="EVO_similar">
<li>DhaR is a transcription activator from the family of enhancer binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by DhaR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Involved in accelerating the dephosphorylation of phosphorylated CheY ; involved in generating a regulating signal for bacterial ﬂagellar rotation ; increases at phosphonate growth 179 179 5.56 / 24,008.4 5.1 / 66,325.63 P13000 Dethiobiotin synthetase Q93SE0 Vitamin B12 transporter BtuB BioD BtuB 179 179 4.81 / 20,469.56 Vitamin B12 transport P06610 BtuE 179 periplasmic protein BtuE Carbamoyl-phosphate synthase small chain CarA 5.91 / 44,088 5.94 / 43,827 5.91 / 41,431.01 P0A6F1 5.88 / 42,128 ( DIGE 4.5 -- 6.5 ) 5.82 / 42,128 ( DIGE 4.5 -- 6.5 ) 5.79 / 41,417 ( DIGE 4.5 -- 6.5 ) 5.49 / 127,208 5.22 / 117,710.53 Carbamoyl-phosphate synthase large chain P00968 CarB 293 5.36 / 32,716 ( DIGE 4.5 -- 6.5 ) 5.42 / 31,539.87 Cytidine deaminase P0ABF6 Cdd 97 , 321 , 325 4.78 / 71,382.39 Chemotaxis protein P07363 CheA CheY 4.89 / 13,966.17 4.95 / 9,698 ( 4 -- 5 ) P0AE67 Chemotaxis protein 179 , 287 CheZ 4.44 / 23,976.03 4.51 / 28,668 P0A9H9 Chemotaxis protein 228 , 293 , 294 Continued on facing pag Participates in iron-transport ; induced by iron and cAMP and increases after benzoic acid treatment Part of a stress-induced multichaperone system ; involved in the recovery of the cell from heat-induced damage in cooperation with DnaK , DnaJ , and GrpE ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and after benzoic acid treatment ; induced by heat shock , phosphate limitation , phosphonate growth , and other environmental stresses ; controlled by RpoH Cleaves peptides in various proteins in a process that requires ATP-hydrolysis ; has a chymotrypsin-like activity ; plays a major role in the degradation of misfolded proteins ; may play the role of a master protease which is attracted to different substrates by different speciﬁcity factors such as ClpA or ClpX ; induced by heat shock ; controlled by RpoH Involved in the modulation of the speciﬁcity of the ClpAPmediated ATP-dependent protein-degradation ATP-dependent speciﬁcity component of the Clp protease ; directs the protease to speciﬁc substrates ; may bind to the lambda O substrate protein and present it to the ClpP protease in a form that can be recognized and readily hydrolyzed by ClpP ; can perform chaperone functions in the absence of ClpP ; induced by heat shock Colicin I receptor 5.03 / 71,149.34 5.06 / 73,587 ( DIGE 4.5 -- 6.5 ) ClpB Chaperone P63284 71 , 179 , 228 , 249 , 287 , 293 , 294 , 321 5.37 / 95,585.02 5.30 / 88,421 5.38 / 73,900 5.33 / 88,745 ( DIGE 4.5 -- 6.5 ) 5.44 / 34,853 ( 5 -- 6 ) ClpP 5.52 / 23,186.65 5.60 / 24,224 ( 5 -- 6 ) P0A6G7 ATP-dependent Clp protease proteolytic subunit 179 , 249 , 287 4.94 / 12,179.06 5.40 / 10,645 ( 4.5 -- 5.5 ) P0A8Q6 ATP-dependent Clp protease adaptor protein ClpS 287 P0A6H1 ATP-dependent Clp protease ATP-binding subunit ClpX 179 5.24 / 46,224.82 179 179 5.56 / 24,746.34 7.04 / 43,306.95 P0A6I0 Cytidylate kinase Cmk CoaBC catalyzes two steps in the biosynthesis of CoA ; in the ﬁrst step , cysteine is conjugated to 4-phosphopantothenate to form 4-phosphopantothenoylcysteine ; in the second , compound is decarboxylated to form 4-phosphopantotheine Involved in CoA-biosynthesis ; catalyzes the phosphorylation of the 3 - hydroxyl group of dephospho-CoA to form CoA Catalyzes two consecutive reactions converting 2 ,3 - cyclic nucleotide to 3-nucleotide and then 3-nucleotide to nucleic acid and phosphate Cmk P0ABQ0 CoA-biosynthesis bifunctional protein CoaBC 79 5.77 / 22,621.71 Dephospho-CoA kinase CoaE P0A6I9 ( YacE ) 1 5.38 / 68,914.86 2 ,3 - Cyclic-nucleotide 2-phosphodiesterase CpdB 9 P08331 17 Continued on following pag The phosphoenolpyruvate-dependent-sugar-phosphotransferase-system ( PTS ) , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in glucose-transport ; plays an important role not only in the transcriptional-control but also in the translational control of rpoS expression Induced by low temp ; cold shock protein Induced by low temp ; cold shock protein Induced during a stationary phase and starvation Inhibits DNA-replication at both initiation and elongation steps , most probably by binding to the opened , single-stranded regions at replication forks ; plays a regulatory role in chromosomal replication in nutrient-depleted cells ; induced by stationary phase and starvation 4.73 / 18,119.88 4.68 / 18,985 4.57 / 20,069 phosphotransferase enzyme IIA component 4.95 / 14,761 ( 4 -- 5 ) 4.86 / 14,800 ( 4 -- 5 ) 4.95 / 13,865 ( 4 -- 5 ) 4.95 / 18,479 ( 4.5 -- 5.5 ) P0A9X9 Cold shock protein CspA 5.57 / 7,272.09 133 , 163 6.54 / 7,716.72 P36995 Cold shock-like protein CspB 163 6.82 / 7,271.17 6.71 / 10,366 P0A9Y6 Cold shock-like protein CspC 54 , 163 , 228 , 293 , 294 163 , 320 5.81 / 7,968.97 P0A968 Cold shock-like protein CspD 8.06 / 7,332.26 5.64 / 7,780.73 P0A972 Cold shock-like protein P0A978 Cold shock-like protein CspE CspG 179 207 Induced by low temp ; cold shock protein Involved in cysteine-biosynthesis ; repressed by sulfate or cysteine ( protein induced by sulfate starvation ) ; induced by high pH CysK 5.83 / 34,358.46 5.81 / 36,027 P0ABK5 Cysteine synthase A 5.83 / 34,342 ( DIGE 4.5 -- 6.5 ) 4.95 / 24,032 ( 4 -- 5 ) 5.01 / 26,270 ( 4 -- 5 ) 4.95 / 25,586 ( 4 -- 5 ) 4.93 / 22,204 ( 4 -- 5 ) 5.06 / 29,410 ( 4.5 -- 5.5 ) 5.32 / 32,815 ( 4.5 -- 5.5 ) 5.04 / 28,256 ( 4.5 -- 5.5 ) 4.95 / 28,122 ( 4.5 -- 5.5 ) 5.78 / 42,978 ( 5 -- 6 ) 5.10 / 41,867 ( 5 -- 6 ) 5.47 / 31,383 ( 4.5 -- 5.5 ) Like a CysK , catalyzes the same reaction of cysteine-biosynthesis ; can also use thiosulfate in place of sulﬁde to give cysteine thiosulfonate as a product Part of the ABC transporter complex cysAWTP , involved in sulfate/thiosulfate import ; speciﬁcally binds thiosulfate and is involved in its transmembrane transport Could help control the pool of 3 - phosphoadenoside-5-phosphosulfate or its use in sulﬁte synthesis Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors Not involved in transpeptidation but in peptidoglycan-synthesis Involved in oxidative deamination of D-amino acids such as alanine-catabolism Involved in L-lysine-biosynthesis-via diaminopimelate-pathway : tetrahydrodipicolinate from L-aspartate ( third step ) ; sensitive to lysine inhibition 5.81 / 36,027 P0ABK5 Cysteine synthase A 5.83 / 34,342 ( DIGE 4.5 -- 6.5 ) 4.95 / 24,032 ( 4 -- 5 ) 5.01 / 26,270 ( 4 -- 5 ) 4.95 / 25,586 ( 4 -- 5 ) 4.93 / 22,204 ( 4 -- 5 ) 5.06 / 29,410 ( 4.5 -- 5.5 ) 5.32 / 32,815 ( 4.5 -- 5.5 ) 5.04 / 28,256 ( 4.5 -- 5.5 ) 4.95 / 28,122 ( 4.5 -- 5.5 ) 5.78 / 42,978 ( 5 -- 6 ) 5.10 / 41,867 ( 5 -- 6 ) 5.47 / 31,383 ( 4.5 -- 5.5 ) Cysteine synthase B P16703 CysM 5.42 / 32,664.16 287 6.43 / 35,057.44 6.58 / 39,151 ( 6 -- 11 ) Thiosulfate-binding protein P16700 CysP 5.59 / 27,175.86 CysQ protein CysQ 79 P22255 1 6.68 / 41,337.2 P0AEB2 Penicillin-binding protein 5 DacA 179 179 8.73 / 49,568.63 P24228 Penicillin-binding protein 4 DacB DadA 6.17 / 47,607.29 P0A6J5 D-Amino acid dehydrogenase small subunit 179 DapA P0A6L2 Dihydrodipicolinate synthase 5.98 / 31,269.97 6.10 / 32,361 5.94 / 32,916 179 , 228 , 286 , 294 Continued on facing pag See DapA : fourth step ; its activity is repressed by lysine ; it decreases after benzoic acid treatment Involved in L-lysine-biosynthesis-via diaminopimelate-pathway : dl-diaminopimelate from lldiaminopimelate See DapD Required for phage C1 adsorption ; increases after benzoic acid treatment Required for cell-wall-formation Reassigned CsdA ; plays a key role in optimal cell-growth at low temp and required for normal cell-division ; exclusively localized in the ribosomal fraction and becomes a major ribosomal-associated protein in cells grown at low temp ; decreases after benzoic acid treatment but induced by cold shock or acid condition Serine protease ; required at high temp and involved in the degradation of damaged proteins ; induced by heat shock Protease with a shared speciﬁcity with DegP ; induced by high pH during anaerobic growth Phosphotransfer between the C1 and C5 carbon atoms of pentose Involved in nucleotide and deoxyribonucleotide-catabolism ; increases during high-cell-density cultivation Cleaves guanosine or inosine to respective bases and sugar-1-phosphate molecules ; decreases after benzoic acid treatment Involved in glycerol utilization ; dihydroxyacetone-binding subunit of the dihydroxyacetone kinase , which is responsible for phosphorylating dihydroxyacetone ; binds covalently dihydroxyacetone in hemiaminal linkage ; acts also as a corepressor of DhaR by binding to its sensor domain in the absence of dihydroxyacetone ; induced by high pH during anaerobic growth Involved in glycerol utilization ; ADP-binding subunit of the dihydroxyacetone kinase , which is responsible for phosphorylating dihydroxyacetone ; DhaL-ADP receives a phosphoryl group from DhaM and transmits it to dihydroxyacetone ; DhaLADP acts also as a coactivator by binding to the sensor domain of dhaR ; DhaL-ATP is inactive ; induced by high pH during anaerobic growth TABLE 2 -- Continued Dihydrodipicolinate reductase 5.45 / 28,756.61 5.37 / 27,969 ( DIGE 5.11 / 37,922 ( 5 -- 6 ) 5.33 / 31,023 ( 5 -- 6 ) 4.5 -- 6.5 ) DapD 2,3,4,5-Tetrahydropyridine-2 ,6 - dicarboxylate Nsuccinyltransferase P0A9D8 5.56 / 29,892.10 5.51 / 31,680 179 , 228 , 294 , 321 5.49 / 32,476 ( DIGE 4.5 -- 6.5 ) Diaminopimelate epimerase Protein P0A6K1 P37620 DapF DcrB 179 5.86 / 30,208.56 4.76 / 17,786.96 179 , 321 4.50 / 18,143 ( DIGE 4.5 -- 6.5 ) DdlA DeaD P0A6J8 D-Alanine-D-alanine ligase A P0A9P6 Cold shock DEAD-box protein A 5.02 / 39,315.81 8.76 / 70,414.96 179 5.41 / 37,079 ( DIGE 4.5 -- 6.5 ) 132 , 274 , 321 DegP 7.87 / 46,829.14 8.01 / 66,440 ( 6 -- 11 ) P0C0V0 Protease Do 179 , 287 DegQ 5.39 / 44,445.75 5.34 / 46,509 Protease P39099 179 , 286 , 323 5.11 / 44,369.96 P0A6K6 Phosphopentomutase DeoB 179 P0A6L0 Deoxyribose-phosphate aldolase 5.50 / 27,733.80 5.52 / 33,657 ( 5 -- 6 ) DeoC 70 , 287 , 325 4.5 -- 6.5 ) 5.40 / 25,098 ( DIGE 5.42 / 25,818.72 Purine nucleoside phosphorylas DeoD P0ABP8 e 321 PTS-dependent DhaK P76015 ( YcgT ) 4.93 / 39,494.75 323 dihydroxyacetone kinase , dihydroxyacetone-binding subunit PTS-dependent DhaL P76014 ( YcgS ) 5.31 / 22,631.75 323 dihydroxyacetone kinase , ADP-binding subunit Continued on following pag DhaM P37349 ( YcgC ) PTS-dependent See DhaL ; serves as the 4.61 / 51,579.81 4.69 / 50,800 ( DIGE 4.5 -- 6.5 ) 179 , 321 , 323 dihydroxyacetone kinase , phosphotransferase subunit phosphoryl donor ; induced by high pH during anaerobic growth Dosage-dependent suppressor of a dnaK deletion mutation ; suppressed not only the temp-sensitive growth but also the ﬁlamentous phenotype of the dnaK deletion strain , while the defect of lambda growth is not suppressed ; induced by high pH Participates in initiation and elongation during chromosome replication ; exhibits DNA-dependent ATPase activity and contains distinct active sites for ATP binding , DNA binding , and interaction with DnaC protein , primase , and other prepriming proteins Interacts with DnaK and GrpE to disassemble a protein complex at the origins of replication of phage lambda and several plasmids ; participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stressdenatured proteins and by disaggregating proteins , also in an autonomous , DnaK-independent fashion ; induced by heat shock under the control of the htpR regulatory protein Involved in chromosomal DNA-replication , possibly through an analogous interaction with the DnaA protein , and participates actively in the response to hyperosmotic shock ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and after benzoic acid treatment ; induced by temp , ethanol , nalidixic acid , puromycin , and overexpression of bioproducts DksA P0ABS1 DnaK suppressor protein 5.06 / 17,527.75 4.90 / 18,723 5.01 / 17,853 179 , 274 , 286 , 321 5.08 / 19,656 ( DIGE 4.5 -- 6.5 ) DnaB P0ACB0 Replicative DNA helicase 4.95 / 52,390.08 4.94 / 49,355 179 , 228 , 294 8.03 / 40,969.14 DnaJ 79 Chaperone protein P08622 1 DnaK 4.81 / 69,647 4.83 / 68,983.76 P0A6Y8 Chaperone protein 71 , 97 , 138 , 179 , 212 , 228 , 287 , 294 , 297 , 321 4.92 / 74,532 ( DIGE 4.5 -- 6.5 ) 4.94 / 73,587 ( DIGE 4.5 -- 6.5 ) 4.91 / 74,850 ( DIGE 4.5 -- 6.5 ) 4.95 / 60,559 ( 4 -- 5 ) 4.98 / 67,563 ( 4 -- 5 ) 4.98 / 66,075 ( 4 -- 5 ) 4.96 / 61,267 ( 4 -- 5 ) 4.97 / 61,125 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.48 / 59,582 ( 4 -- 5 ) 5.00 / 44,877 ( 4 -- 5 ) 4.93 / 52,435 ( 4 -- 5 ) 4.96 / 60,841 ( 4 -- 5 ) 4.96 / 60,234 ( 4.5 -- 5.5 ) 4.97 / 59,766 ( 4.5 -- 5.5 ) 4.95 / 53,032 ( 4.5 -- 5.5 ) 4.97 / 41,400 DNA polymerase III is a complex , multichain enzyme responsible for most of the replicative synthesis in bacteria ; exhibits 3 - to-5 exonuclease activity ; required for initiation of replication once it is clamped onto DNA See DnaN ; contains the editing function and is a proofreading 3 - to-5 exonuclease Dipeptide-binding protein of a transport system that can be subject to osmotic shock and is also required for peptide chemotaxis ; increases in the physiological short-term adaptation to glucose limitation P0A6Y8 Chaperone protein 71 , 97 , 138 , 179 , 212 , 228 , 287 , 294 , 297 , 321 4.92 / 74,532 ( DIGE 4.5 -- 6.5 ) 4.94 / 73,587 ( DIGE 4.5 -- 6.5 ) 4.91 / 74,850 ( DIGE 4.5 -- 6.5 ) 4.95 / 60,559 ( 4 -- 5 ) 4.98 / 67,563 ( 4 -- 5 ) 4.98 / 66,075 ( 4 -- 5 ) 4.96 / 61,267 ( 4 -- 5 ) 4.97 / 61,125 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.48 / 59,582 ( 4 -- 5 ) 5.00 / 44,877 ( 4 -- 5 ) 4.93 / 52,435 ( 4 -- 5 ) 4.96 / 60,841 ( 4 -- 5 ) 4.96 / 60,234 ( 4.5 -- 5.5 ) 4.97 / 59,766 ( 4.5 -- 5.5 ) 4.95 / 53,032 ( 4.5 -- 5.5 ) 4.97 / 41,400 P0A988 DNA polymerase III beta subunit DnaN 294 5.25 / 40,586.60 DNA polymerase III epsilon subunit 5.54 / 27,098.93 5.65 / 26,832 P03007 DnaQ 228 , 294 DppA 5.47 / 52,481 5.69 / 52,170 5.75 / 57,407.06 Periplasmic dipeptide-transport protein P23847 179 , 286 , 287 , 311 5.71 / 52,000 ( 5 -- 6 ) 5.71 / 50,118 ( 5 -- 6 ) 5.81 / 52,000 ( 5.5 -- 6.7 ) Continued on facing pag Unspeciﬁcally binds and Dps P0ABT2 DNA-protection during starvation protein 5.72 / 18,564.11 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) 5.78 / 17,897 ( 5 -- 6 ) 5.27 / 16,787 ( 5 -- 6 ) 5.21 / 16,164 ( 5 -- 6 ) 5.94 / 17,273 ( 5.5 -- 6.7 ) protects DNA from oxidative damage mediated by hydrogen peroxide ; induced by prolonged starvation , stationary phase , and acetic acid accumulation ; controlled by ClpXP and ClpAP proteases , which affect logphase stability and stationary-phase synthesis of Dps , respectively Transfers its disulﬁde bond to other proteins and is reduced in the process and then reoxidized by DsbB ; induced by high pH during aerobic or anaerobic growth See DsbA ; required for disulﬁde bond formation in some periplasmic proteins Involved in de novo synthesis of thymidylate ; produces dUMP , the immediate precursor of thymidine nucleotides , and decreases the intracellular concn of dUTP so that uracil can not be incorporated into DNA General inhibitor of pancreatic serine proteases ; inhibits chymotrypsin , trypsin , elastases , factor X , and kallikrein as well as a variety of other proteases Key enzyme in the EntnerDoudoroff pathway and participates in the regulation of the intracellular level of glyoxylate ; induced at the stationary phase and in the presence of gluconate or hexuronic acids ; increases after benzoic acid treatment , phosphate limitation , and phosphonate growth Involved in peptide bond synthesis ; stimulates efﬁcient translation and peptide bond synthesis on native or reconstituted 70S ribosomes in vitro ; probably functions acceptors for peptidyl transferase May be involved in the early steps of isoprenoid-biosynthesis GTP-dependent nucleic acidbinding protein which may act as a translation factor Involved in the pathway of glycolysis ; also involved in the RNA degradosome , a multienzyme complex important in RNA-processing and mRNA-degradation ; increases during anaerobic growth 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) 5.78 / 17,897 ( 5 -- 6 ) 5.27 / 16,787 ( 5 -- 6 ) 5.21 / 16,164 ( 5 -- 6 ) 5.94 / 17,273 ( 5.5 -- 6.7 ) DsbA P0AEG4 Thiol : disulﬁde interchange protein 5.42 / 21,132.07 5.34 / 22,426 5.31 / 21,942 179 , 228 , 274 , 294 , 321 , 323 5.34 / 22,606 ( DIGE 4.5 -- 6.5 ) P0AEG6 Thiol : disulﬁde interchange protein 5.86 / 23,460 DsbC 179 Dut 5.03 / 16,155.47 Deoxyuridine 5 - triphosphate nucleotidohydrolase P06968 5.03 / 16,145 179 , 228 , 294 , 321 5.10 / 17,526 ( DIGE 4.5 -- 6.5 ) 5.94 / 16,099.52 Ecotin P23827 Eco 179 Eda P0A955 2-Keto-4-hydroxyglutarate / 2-keto-3-deoxy-6 - phosphogluconate aldolase 99 , 179 , 287 , 293 , 321 4.93 / 23,247 ( DIGE 4.5 -- 6.5 ) 5.43 / 27,937 ( 5 -- 6 ) 5.57 / 22,284.03 Efp 179 4.9 / 20,460.12 P0A6N4 Elongation factor P 179 , 287 ElbB P0ABU5 Enhancing lycopene biosynthesis 4.68 / 22,981.55 4.92 / 25,756 ( 4 -- 5 ) ( YhbL ) protein 2 5.03 / 42,217 ( DIGE 4.5 -- 6.5 ) 5.02 / 42,547 ( 4.5 -- 5.5 ) 4.87 / 39,536.13 EngD P0ABU2 GTP-dependent nucleic acidbinding protein 287 , 321 Eno 5.34 / 46,509 5.29 / 46,234 5.32 / 45,523.75 P0A6P9 Enolase 29 , 179 , 228 , 271 , 286 , 287 , 294 , 321 5.35 / 43,401 ( DIGE 4.5 -- 6.5 ) 4.74 / 22,872 ( 4 -- 5 ) 5.46 / 45,494 ( 4.5 -- 5.5 ) 5.20 / 40,116 ( 4.5 -- 5.5 ) 5.16 / 31,510 ( 4.5 -- 5.5 ) 5.29 / 51,580 ( 5 -- 6 ) 5.27 / 51,301 ( 5 -- 6 ) 5.16 / 51,319 ( 5 -- 6 ) 5.00 / 50,698 ( 5 -- 6 ) 5.02 / 50,647 ( 5 -- 6 ) 5.26 / 50,000 ( 5 -- 6 ) 6.95 / 24,005 ( 6 -- 11 ) 6.72 / 25,314 ( 6 -- 9 ) Continued on following pag 2,3-Dihydro-2 ,3 - EntA P15047 4.97 / 26,249.66 179 dihydroxybenzoate ( 2,3-DHB ) dehydrogenase Isochorismatase Involved in siderophore-biosynthesis and enterobactin-biosynthesis Required for production of 2,3-DHB and serves as an aryl carrier protein ; plays a role in enterobactin assembly : proteins EntB , EntD , EntE , and EntF form a multienzyme complex called enterobactin-synthetase Member of the two-component regulatory system EvgS/EvgA ; regulates the expression of emrKY operon and yfdX ; also seems to control expression of at least one other multidrug efﬂux operon Catalyzes the condensation reaction of fatty-acid-synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP Involved in fatty-acid-biosynthesis 4.97 / 26,249.66 179 P0ADI4 EntB 21 5.08 / 34,633 ( DIGE 4.5 -- 6.5 ) 5.05 / 32,554.34 3 EvgA 79 P0ACZ4 Positive transcription regulator 6.83 / 22,690.2 1 P0A953 3-Oxoacyl - acyl-carrier-protein 5.35 / 42,613.32 5.76 / 44,039 synthase I FabB 294 FabD 4.95 / 32,286.01 5.37 / 44,881 4.95 / 33,409 P0AAI9 Malonyl CoA-acyl carrier protein transacylase 179 , 228 , 287 , 294 , 321 5.03 / 33,449 ( DIGE 4.5 -- 6.5 ) 5.03 / 32,221 ( 4.5 -- 5.5 ) See FabB ; has a preference for short chain acid substrates and may function to supply the octanoic substrates for lipoic-acid-biosynthesis Involved in fatty-acid-biosynthesis pathway See FabB ; catalyzes the ﬁrst condensation reaction , which initiates fatty-acid-synthesis and may therefore play a role in governing the total rate of fatty acid production ; possesses both acetoacetyl-ACP synthase and acetyl transacylase activities Involved in second reduction step of fatty-acid-biosynthesis ; decreases after benzoic acid treatment Involved in saturated fatty-acid-biosynthesis Catalyzes the ﬁnal step of fatty-acid-oxidation in which acetyl-CoA is released and the CoA ester of a fatty acid two carbons shorter is formed ; involved in the aerobic and anaerobic degradation of long-chain-fatty-acids ; induced in the presence of fatty acids and is under the control of the fadR repressor Catalyzes the formation of a hydroxyacyl-CoA by addition of water on enoyl-CoA ; also exhibits 3-hydroxyacyl-CoA epimerase and 3-hydroxyacyl-CoA dehydrogenase activities ; involved in the aerobic and anaerobic degradation of long-chain-fatty-acids ; induced in the presence of fatty acids and is under the control of the fadR repressor Involved in translocation of long-chain-fatty-acids across the outer membrane ; induced in the presence of long-chain-fatty-acids and under the control of the fadR repressor 5.03 / 33,449 ( DIGE 4.5 -- 6.5 ) 5.03 / 32,221 ( 4.5 -- 5.5 ) 5.71 / 42,914.57 n i e t o r p FabF ¬ r e i r r a c acyl-II xoacylsynthase 3 - O 5 I A A 0 P 179 6.76 / 25,560.29 P0AEK2 3-Oxoacyl - acyl-carrier-protein reductase 3-Oxoacyl - acyl-carrier-protein synthase III FabG 179 5.08 / 33,515.12 P0A6R0 FabH 179 FabI 5.60 / 32,613 ( DIGE 4.5 -- 6.5 ) 5.40 / 32,959 ( DIGE 4.5 -- 6.5 ) 5.37 / 38,366 ( 5 -- 6 ) 5.33 / 37,266 ( 5 -- 6 ) 5.58 / 27,732.75 P0AEK4 Enoyl - acyl-carrier-protein reductase ( NADH ) 179 , 287 , 321 6.84 / 17,032.95 FabZ P0A6Q6 ( 3R ) - hydroxymyristoyl - acyl carrier protein dehydratase P21151 3-Ketoacyl-CoA thiolase 179 6.31 / 40,876.20 FadA 46 5.84 / 79,593.91 Fatty oxidation complex alpha subunit P21177 FadB 4 6 FadL 26 Long-chain fatty-acid-transport 4.91 / 45,906.47 protein P10384 Continued on facing pag Involved in the pathway of glycolysis ; increases in poly ( 3-hydroxybutyrate ) - accumulating cells FbaA P0AB71 Fructose-bisphosphate aldolase class II 5.52 / 39,016.07 5.55 / 40,651 5.43 / 39,776 5.49 / 39,104 ( DIGE 5.56 / 50,220 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.68 / 32,568 4.5 -- 6.5 ) Necessary for growth on Fbp P0A993 Fructose-1 ,6 - bisphosphatase 179 , 228 , 294 5.67 / 36,833.93 substances such as glycerol , succinate , and acetate ; inhibited by AMP , which affects the turnover of bound substrate and not the afﬁnity for substrate Outer membrane receptor protein in the Fe3 dicitrate-transport system ; for induction , the TonB and the ExbB proteins have to be active Involved in the initial step of iron uptake by binding ferrienterobactin , an iron chelatin siderophore that allows E. coli to extract iron from the environment ; acts as a receptor for colicins B and D ; increases after benzoic acid treatment Binds the ferrichrome-iron ligand ; interacts with the TonB protein , which is responsible for energy coupling of the ferrichromepromoted iron-transport system ; acts as a receptor for bacteriophage T5 as well as for T1 , phi80 , and colicin M ; can also transport the antibiotic albomycin Activates rRNA-transcription ; plays a direct role in upstream activation of rRNA promoters ; binds to a recombinational enhancer sequence that is required to stimulate Hinmediated DNA inversion ; prevents initiation of DNA-replication from OriC PPIases accelerate the folding of proteins ; catalyzes the cistrans isomerization of proline imidic peptide bonds in oligopeptides ; strongly inhibited by FK506 See FklB Iron ( III ) dicitrate-transport protein P13036 FecA 79 5.36 / 81,707.21 1 5.23 / 79,771.07 Ferrienterobactin receptor P05825 FepA 321 5.24 / 79,447 ( DIGE 4.5 -- 6.5 ) 5.13 / 78,742.19 Ferrichrome-iron receptor P06971 FhuA 179 9.34 / 11,239.93 P0A6R3 DNA-binding protein Fis 79 1 FklB 4.5 -- 6.5 ) 4.84 / 24,436 ( DIGE 4.85 / 22,085.00 P0A9L3 FKBP-type 22-kDa peptidylprolyl cis-trans isomerase 179 , 321 P45523 FKBP-type peptidyl-prolyl cis - 6.73 / 26,223.63 7.08 / 33,268 trans isomerase Flavodoxin 1 4.21 / 19,605.75 4.65 / 21,200 FkpA 179 , 286 Low-potential electron donor to a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>( Proposed ) Function aga ( gam ) operon AgaB AgaC AgaD AgaF Man Man Man Man P42909 P42910 Galactosamine IIB Galactosamine IIC Galactosamine IID Galactosamine IIA IIB IIC IID IIA P42911 gi8895750 aga operon AgaV AgaW AgaE AgaF Man Man Man Man P42904 N-Acetylgalactosamine IIB N-Acetylgalactosamine IIC N-Acetylgalactosamine IID N-Acetylgalactosamine IIA IIB IIC IID IIA gi8895748 gi8895749 gi8895750 asc operon AscF 485 Glc IIBIICIIA P24241 Arbutin/Cellobiose/Salicin IIABC bgl operon BglF 625 IIBIICIIA Glc P08722 Beta-Glucoside IIABC chb operon ChbA ChbB ChbC cmt operon CmtA CmtB IIA IIB IIC Lac Lac Lac P17335 P17409 P17334 Di-N-acetylchitobiose/Cellobiose IIA Di-N-acetylchitobiose/Cellobiose IIB Di-N-acetylchitobiose/Cellobiose IIC P32059 P32058 IIBC IIA 462 147 Fru Fru Mannitol ( cryptic ) IIBC Mannitol ( cryptic ) IIA dha operon DhaH DhaK1 DhaK2 DhaR P37349 P76015 P76014 P76016 RH ( DPr ) ( I ADI ; YcgC HPr , E I Dihydroxyacetone kinase regulator ( putative ) Dihydroxyacetone kinase subunit I Dihydroxyacetone kinase subunit II Dihydroxyacetone operon regulator ( putative ) fru operon FruA FruB Fructose IIB ' BC Fructose IIA/FPr IIB ' BC P20966 IIAMH ( FPr ) P24217 563 376 Fru DTP ; FPr Fru , HPr frv operon FrvA FrvB Fructose-like IIA Fructose-like IIBC P32155 P32154 IIA 148 485 Fru Fru IIBC frw operon FrwA - I-IIA IIB IIC IIB ' H ijH TP ; Y YijK YijJ YijM T r , I , HP Fru Fru Fru Fru E Enzyme I , HPr , Fructose-like IIA Fructose-like IIB Fructose-like IIC Fructose-like IIB ' P32670 P32673 P32672 P32676 FrwB FrwC FrwD frx operon Frx 658 IIABC HsrA Fructose-like IIABC P54745 Fru fry operon FryA Enzyme I , HPr , Fructose-like IIA Fructose-like IIB Fructose-like IIC P77439 P77579 P76525 - I-IIA IIB IIC H TTP ; YpdD YpdH YpdG E I , HPr , Fru Fru Fru FryB FryC gat operon GatA GatB GatC glv operon GlvB GlvC Galactitol IIA Galactitol IIB Galactitol IIC P37187 P37188 P37189 IIA IIB IIC 150 94 451 Gat Gat Gat Arbutin IIB Arbutin IIC P31451 P31452 IIB IIC 161 368 Glc Glc ( Probably uses IIAGlc ) gut operon GutA GutB GutE Glucitol IIC1 Glucitol IIA Glucitol IIB-IIC IIC1 P56580 ( gi1789055 ) IIA P05706 IIB-IIC2 P56580 ( gi3915813 ) SrlA SrlB SrlE Gut Gut Gut mal operon MalX Glc 530 IICB P19642 Maltose/Glucose IIBC man operon ManX ManY ManZ mtl operon MtlA nag operon NagE ptsG operon PtsG pts operon PtsH PtsI Crr Man Man Man IIAB IIC IID P08186 P08187 P08188 Mannose/Glucose/Glucosamine / Fructose IIAB Mannose/Glucose/Glucosamine / Fructose IIC Mannose/Glucose/Glucosamine / Fructose IID Fru 637 IIC-IIB-IIA P00550 Mannitol IIABC Glc 648 IIC-IIB-IIA P09323 Acetylglucosamine IIABC Glc 477 IIC-IIB P05053 Glucose IIBC 85 HPr E I HPr E I Glc H ( HPr ) I IIA P07006 P08839 P08837 Sugar transport/phosphorylation Sugar transport/phosphorylation Glucose IIA 575 169 IIAGlc rpoN operon PtsN NPr Fru HPr II ANtr IIA 163 90 P31222 P33996 Nitrogen regulation Nitrogen regulation H ( NPr ) ptsP operon PtsP 748 P37177 NifA-I E I INtr Nitrogen regulation sga operon SgaA SgaB ( SgaT ) P39303 P39302 P39301 IIA IIB Fru ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>DhaR also features sequence similarity ( o30 % identity ) to AcoR , the regulator of acetoin-metabolism in Thermoanaerobacter tengcongensis and Clostridium magnum ( Figure 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Escherichia coli dihydroxyacetone kinase controls gene-expression by binding to transcription factor DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17846588" target="_blank">17846588</a>]<br></li>
<li>A fourth divergently transcribed gene , dhaR , codes for the transcription regulator DhaR ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>a 13 Figure 3 DhaR is the activator of dhaKLM operon ( A ) and the repressor of dhaR gene transcription ( B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaR ( P76016 ) encodes the transcription regulator DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>EI and DhaM thus are negative regulators , and DhaR is a positive regulator ( activator ) of dha operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">DhaR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Figure 4 Conserved sequence motives from sensing domains of DhaR homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by DhaR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Finally , a divergently transcribed E. coli gene b1201 ( ycgU ; here designated dhaR ; see Fig. 1 ) was found to encode a putative transcriptional activator , homologous to DhaR , the putative regulator of dhaK expression in Citrobacter freundii ( 2 ) , as well as the NtrC response regulator of the nitrogen two-component system ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11021910" target="_blank">11021910</a>]<br></li>
<li>However , disruption of rpoN , the gene encoding s54 , did not impair induction of the dha operon by DhaR ( Figure 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Escherichia coli dihydroxyacetone kinase controls gene expression by binding to transcription factor DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17846588" target="_blank">17846588</a>]<br></li>
<li>Repression of the dhaR gene by DhaR is not affected by Dha and the Dha kinase subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>A fourth divergently transcribed gene , dhaR , codes for the transcription regulator DhaR ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>a 13 Figure 3 DhaR is the activator of dhaKLM operon ( A ) and the repressor of dhaR gene transcription ( B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>DhaR is an ( auto ) repressor of the dhaR gene and an activator of the dhaKLM operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>Results and discussion DhaR is an activator of the dhaKLM operon and a repressor of its own synthesis Induction of the dha operon with Dha E. coli MC4100 containing the PdhaK promoter and lacZ in the chromosomal l attachment site was used to measure dha operon expression ( Boyd et al , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In summary , the genetic analyses of dhaKLM and dhaR transcription show the following : ( i ) DhaR is an autonomous autorepressor of its own synthesis , ( ii ) DhaR and DhaL are positive regulators of the dhaKLM operon , ( iii ) phosphorylated DhaM and DhaK are negative regulators , ( iv ) Dha is the inducer and ( v ) there is no evidence for DhaR being controlled by protein phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>EI and DhaM thus are negative regulators , and DhaR is a positive regulator ( activator ) of dha operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>The dha operon is constitutively active in the absence of a functional DhaM or EI , and inactive in the absence of DhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>