<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - DnaA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">DnaA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">DnaA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Pull-down assay of HobA in the absence or the presence of His6-HpDnaA and His6-Nterm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Addition of HobA rapidly increases HpDnaA interaction with HporiC , which suggests that binding of HobA triggers formation of an active orisome and allows replication to start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Additionally , we could not detect binding of DiaA to the N-terminus of HpDnaA ( domains I-II , HpDnaAI-II ) , which interacts strongly with HobA ( Fig. 2 , sectors 5 and 6 , respectively ) .16 We conclude that the absence of protein interaction between HobA and EcDnaA and between DiaA and HpDnaA is a key explanation for the inability to replace one protein with its structural counterpart during orisome assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>DiaA stimulation resulted in both faster association of the hybrid protein and an approximately 2.5-fold increase of EcIHpII-IVDnaA binding to oriC ( Fig. 5c ; Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>However , the slope of the sensogram representing binding of EcDnaA in the presence of DiaA indicated that prolonged incubation would result in an overall two - to threefold increase in binding of EcDnaA to its oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>HobA enhances and accelerates HpDnaA binding to oriC , whereas DiaA increases but decelerates EcDnaA binding with oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>This confirmed that HobA enhances HpDnaA binding to oriC and stimulates orisome assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>DnaA that is bound to recognition 1Department of Biochemistry and Molecular Biology , Georgetown University Medical Center , 3900 Reservoir Road NW , Washington , District of Columbia 20007 , USA 2Molecular and Cell Biology , University of California , Berkeley , 16 Barker Hall , Berkeley , California 94720 , USA 3NCI-DBS , Frederick Cancer Research and Development Center , Box B , Building 539/223 , Frederick , Maryland 21702 , USA wPresent address : Department of Biology , Massachusetts Institute of Technology , 77 Massachusetts Ave 68-530 , Cambridge , Massachusetts 02139 , USA * These authors contributed equally to this work + Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>Beta-GalactosidaseActivity ( Miller units ) pACYCl84 pHK27 ( dnaA + ) - IPTG + IPTG - IPTG + IPTG Gene fusion Plasmid polA ' - ` lacZ polA ' - ` lacZ orf 310 ' - ` lacZ pNH3 pNP4 pNP21 dnaA ' - ` lacZ dna3 ' - ` lacZ pAZ2 pBZ2 pzz7 pDZ1 660 994 84 810 651 887 91 822 680 965 87 869 276 979 95 852 dnaZ - ` lacZ uvrD ' - ` lacZ polA ' - ` lacZ DnaA box deleted pNH3-1 Beta-galactosidaseactivities of polA ' - ` lacZ and several control fusions were measured in the stationary-phase cells from IPTG-treated ( + ) and - untreated ( - ) cultures of strain KAlO harbouring the respective fusion plasmid and the compatible DnaA overproducer vector pHK27 or pACYC184 as control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>( 2003 ) 329 , 59 -- 68 Interplay Between DnaA and SeqA Proteins During egulation o Bacteriop Prom ter Activity R f hage l p o R Monika Sl ⁄ omińska1 , Grażyna Konopa1 , Sylwia Barańska1 Grzegorz Węgrzyn1 ,2 and Alicja Węgrzyn3 * 1Department of Molecular Biology , University of Gdańsk Kl ⁄ adki 24 , 80-822 Gdańsk Poland 2Institute of Oceanology Polish Academy of Sciences Św .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>94 , 61 -- 71 , July 10 , 1998 , Copyright 1998 by Cell Press The Initiator Function of DnaA Protein gatively Is Ne Regulated by the Sliding Clamp eplicase of the E. coli Chromosomal R Tsutomu Katayama , * † ‡ Toshio Kubota , * Kenji Kurokawa , * § Elliott Crooke , † and Kazuhisa Sekimizu * * Department of Microbiology Kyushu University Faculty of Pharmaceutical Sciences Fukuoka 812-8582 Japan † Department of Biochemistry and Molecular Biology Georgetown University Medical Center Washington , D.C. 20007 Summary The b subunit of DNA polymerase III is essential for negative regulation of the initiator protein , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_15">
Mol Microbiol 41 : 9 -- 17 Katayama T , Kubota T , Kurokawa K , Crooke E , Sekimizu K ( 1998 ) The initiator function of DnaA protein is negatively regulated by the sliding clamp of the E. coli chromosomal replicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></label>
<input id="act_similarity_15", type="checkbox">
<div class="ACT_similar">
<li>Katayama , T. , Kubota , T. , Kurokawa , K. , Crooke , E. , and Sekimizu , K. ( 1998 ) The initiator function of DnaA protein is negatively regulated by the sliding clamp of the E. coli chromosomal replicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Katayama , T. , Kubota , T. , Kurokawa , K. , Crooke , E. & Sekimizu , K. ( 1998 ) The initiator function of DnaA protein is negatively regulated by the sliding clamp of the E. coli chromosomal replicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>Sekimizu , K. ( 1998 ) The initiator function of DnaA protein is negatively regulated by the sliding clamp of the E. coli chromosomal replicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>The initiator function of DnaA protein is negatively regulated by the sliding clamp of the E. coli chromosomal replicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23994470" target="_blank">23994470</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></li></div>
<li>Activation of fabA , fabB and fabI by FadR results in high levels of long-chain fatty acid biosynthesis while activity of the fatty acid transport system including FadL and acyl-CoA synthetase , additional fad structural genes and uspA are low due to the combined e $ ects of repression by FadR , low cAMP levels and OmpR-and DnaA - mediated repression of fadL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>All 12 DNA-binding proteins , CbpA , CbpB ( or Rob ) , DnaA , Dps , Fis , Hfq , H-NS , HU , IciA , IHF , Lrp and StpA , were over-expressed using pCU60 , pMK19 , pSY567 , pDPS1 , pRJ1077 , pHFQ607 , pHOP11 , pLhupAhupB , pICS1 , pSA5hiphimA , pMWD1 and pT7stpA , respectively , and puri ® ed to apparent homogeneity as previously described ( Talukder & Ishihama 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>We attempted to assess the in vivo nucleotide forms of DnaA in hda-185 cells by 32P labeling and immunoprecipitation as we have done previously ( 27 , 29 , 36 ) ; however , growth of the hda-185 cells was severely inhibited in a phosphate-limited minimum medium which is required for 32P labeling ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Similarly , the Asp-269 residue might directly contact an adjacent protomer during complex formation , causing stabilization of the complexes and a conformational change of the DnaA AAA + domain that decreases its affinity for nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>To explore the model that the chromosomal level of DnaAcos , or the increased abundance of DnaA increases initiation frequency by respectively escaping or overcoming pathways that control initiation , we developed a genetic selection and identified seqA , datA , dnaN and hda , which function in pathways that either act at oriC , or modulate DnaA activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Testing the model that DnaAcos overinitiates because it fails to respond to one or more regulatory mechanisms , we show that dnaAcos is unresponsive to hda and dnaN , which encodes the β clamp , and also datA , a locus proposed to titer excess DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>DnaAcos fails to respond to Hda and the β clamp , which leads to excessive initiation Because overexpression of wild type DnaA interferes with viability in recA and recB strains , and multicopy plasmids bearing hda , dnaN , seqA or datA suppress this lethal effect , we hypothesized that DnaAcos initiates excessively because it fails to respond to one or more of these regulatory pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Plasmids bearing hda , dnaN , datA , or seqA can counteract the lethal effect of DnaA overproduction ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>By comparison , an elevated level of DnaA also induces extra initiations , but lethality occurs only in strains defective in repairing double strand breaks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Previously , we showed that an increased level of DnaA causes more frequent initiations and an increase in the abundance of double-strand breaks ( DSBs ) that are toxic in a recA or recB mutant that is defective in DSB repair ( 84 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>As shown in Table 2 , the frequency of colony formation of E. coli HMS174 ( relevant genotype , recA1 DE3 ) induced to express T7 RNA polymerase or XL1-Blue ( recA1 ) was substantially lower when the level of DnaA was elevated via the respective dnaA plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Our study shows that the colony formation of strains bearing mutations in rep , recG , and rcbA , like recA and recB mutants , is inhibited by an oversupply of DnaA and that a multicopy plasmid carrying rcbA neutralizes this inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>As an elevated copy number of rcbA suppresses the inhibitory effect caused by surplus DnaA in recG , rep , recA , and recB mutants , the rcbA product does not seem to act with the respective proteins in pathways to resurrect stalled and collapsed replication forks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Proteins DnaA and SeqA proteins were puriﬁed as described ,14,45 and were provided by Walter Messer ( Max-Planck-Institute for Molecular Genetics , Berlin , Germany ) and Kirsten Skarstad ( Institute for Cancer Research , Oslo , Norway ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>If the DnaA complex at datA normally resides in a specific location , it is possible that a joint DnaA -- SeqA structure at this location contributes to the increased number of total SeqA structures per cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Activity of pR is controlled both positively and negatively by variety of regulators , including the stringent response alarmon ppGpp , phage-encoded cI and Cro proteins and host-en-coded DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Interestingly , it seems that at relatively high amounts of SeqA , DnaA may stimulate pR at its optimal concentration ( Figure 3 ( a ) ) , though this phenomenon was observed only at one DnaA concentration , and thus , its physiological relevance is not obvious .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Plasraids used in the galactokinase experiments ( Table 4 ) contain little or no promoter activity in this region in the presence of wild-type DnaA protein ; thus , activation of transcription activity via decrease in DnaA concentration if DnaA protein acts as a transcription terminator is difficult to explain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>The use of the optimal DnaA concentration precluded further activation of pR by SeqA , even at optimal amount of this protein ( Figure 3 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Since SeqA can still stimulate pR at high DnaA levels , which are normally inhibitory for transcription from this promoter , it appears that the level of activation of pR achieved at optimal DnaA concentration is maximal , thus no further stimulation by SeqA is possible , even at optimal concentration of this protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>As demonstrated previously ,16,22 when present separately in the reaction mixture , both DnaA and SeqA activate pR at relatively low concentrations , whereas these proteins act as repressors at higher concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Thus , one might speculate that under normal growth conditions DnaA occurs in E. coli cells at levels sub-optimal for activation of pR as effects on in vivo activity of this promoter were more dramatic in a double dnaA46 DseqA < Tn10 mutant than in a single dnaA46 mutant ( Figure 2 ) , suggesting that the SeqA-mediated activation still operates in the letter case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Therefore , we investigated the activity of the pR promoter in vivo and in vitro under conditions of combinations of the presence and absence of DnaA and SeqA functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Interestingly , for regulation of pR activity , binding to sequences located downstream of the promoter is necessary in the case of both DnaA and SeqA ,16,22 which is unusual in prokaryo-tic transcription control systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_44">
Interplay between DnaA and SeqA proteins during regulation of bacteriophage lambda pR promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></label>
<input id="act_similarity_44", type="checkbox">
<div class="ACT_similar">
<li>Interplay between DnaA and SeqA proteins during regulation of bacteriophage l pR promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>Interplay between DnaA and SeqA proteins during regulation of bacteriophage k pR promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li></div>
<li>Both in vivo and in vitro studies revealed that DnaA and SeqA can activate the pR promoter independently and separately rather than in co-operation , however , increased concentrations of one of these proteins negatively inﬂuenced the transcription stimulation mediated by the second regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Functions of dnaA and seqA in the activation of pR in vivo To investigate possible functional interactions between DnaA and SeqA during stimulation of transcription from the pR promoter , we measured activity of the pR -- lacZ fusion in various mutant strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Combinations of DnaA and SeqA functions during activation of pR in vitro To investigate a possible interplay between DnaA and SeqA during activation of pR in more detail , we performed in vitro transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>The most intriguing result is that in the presence of high SeqA amounts ( but not of moderate SeqA amounts , optimal for the transcription stimulation ) , DnaA could still activate the pR-promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Bacteriophage l pR promoter is stimulated by both DnaA and SeqA .16,22 At concentrations higher than optimal for the promoter activation , DnaA represses pR and SeqA is no longer a stimulator of transcription from this promoter , and becomes a weak repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>When SeqA was pre-incu-bated at a concentration equal to that optimal for pR activation , no signiﬁcant further stimulation of this promoter by DnaA was observed ( Figure 3 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>On the other hand , low DnaA concentrations do not affect stimulation of pR by SeqA , whereas at concentration of DnaA that is optimal for activation of pR further activation of this promoter by SeqA does not occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Furthermore , Table 1 shows that similar results without the enhancer effect were also obtained in other control experiments using plasmid pZZ7 ( dnaZ ' - ` lacZ ) with a DnaA box in the promoter region , and plasmids pBZ2 ( dnaS ' - ` lacZ ; Kleinsteuber and Quiiiones , 1995 ) , and pDZ1 ( uvrD ' - ` lacZ ; Neumann et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Interestingly , as seen in Table 1 , the stimulation effect by DnaA overproduction was not observed in KAlO cells harbouring the plasmid pNP21 with the orf370 ' - ` lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Moreover , in a positive control experiment we observed the expected repression by DnaA overproduction on the dnaA ' - ` lacZ fusion of pAZ2 ( Quifiones et at , 1991 ) due to the well-known autogenous regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>As detailed above , these are predicted to be a consequence of cumulative e $ ects caused by ppGpp on phospholipid biosynthesis [ 220 , 304 , 307 ] and fatty acid biosynthesis [ 407 ] , by FadR-dependent control of the fad , iclR , uspA and fab genes , [ 108 ] by growth rate-dependent alterations in acc expression [ 294 ] , and by inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>We are accumulating evidence that 110operon is regulated multivalently and apart from ` he $ ation by DnaA , is also under stringent control me - ` Oled by guanosine bis-diphosphate ( Davies and Drabdi3 in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736096" target="_blank">1736096</a>]<br></li>
<li>Transcription from guaP is subject to stringent control and growth-rate-dependent regulation , and to repression by DnaA and PurR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Indicated also are the UP element and discriminator sequence ( involved in growth-rate-dependent regulation and stringent control ) , sites identi cents ed for the binding of PurR and DnaA and the putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>HobA and DiaA are functional homologs , but assist orisome assembly in a species-specific manner We showed that HobA and DiaA do not bind heterologous DnaA proteins and , therefore , they can not stimulate assembly of the foreign orisomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>We found , however , that neither of the proteins can interact with their foreign DnaA partner ( Fig. 2 ) and thus neither can stimulate assembly of heterologous orisomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In the HobA/DnaA complex , DnaA binds via its helices α2 and α3 ( yellow ribbon ) to two cavities that are very different in size/shape and potential in DiaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In HobA , the cavity that binds to DnaA α3 is predominantly hydrophobic and negative ( Leu100 , Ala101 , Glu104 ) , whereas the same area of DiaA is positively charged with residues K101 and R104 ( Fig. 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In particular , the N-terminal portion of helix α3 of DnaA binds to a hydrophobic/negatively charged area in HobA , whereas the corresponding surface is positively charged in DiaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>As controls , we determined that levels of DnaA1 and DnaAD96-120 were comparable in the prebound extracts ( Figure 5 , lanes B and C ) ; no signal was apparent for DiaA beads when the His6-DnaA extracts were omitted ( Figure 5 , lane D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DnaA did not detectably bind mock-treated beads with no loaded DiaA ( Figure 5 , lane H ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>( 2011 ) 408 , 238 -- 251 Contents lists available at www.sciencedirect.com Journal of Molecular Biology journa l homepage : http : / / ees.elsevier.com.jmb DiaA/HobA an DnaA : A Pair of Proteins Co-evolved to perate Bacterial Orisom A s mbly d uring D Coo e s e Anna Zawilak-Pawlik1 , Rafał Donczew1 , Szymon Paweł Mackiewicz2 , Laurent Terradot3 and Jolanta Zakrzewska-Czerwińska1 ,4 Szafrański1 , icrobiology , Received 18 November 2010 ; received in revised form 8 February 2011 ; accepted 17 February 2011 Available online 25 February 2011 Edited by M. Gottesman Keywords : HobA/DiaA ; oriC ; initiation complex ; initiation of chromoso replication ; Helicobacter pylori 1Institute of Immunology and Experimental Therapy , Polish Academy of Sciences , Department of M Weigla 12 , 53-114 Wrocław , Poland 2University of Wrocław , Faculty of Biotechnology , Przybyszewskiego 63/77 , 51-148 Wrocław , Poland 3Université de Lyon , Lyon , France ; Université Lyon 1 ; IFR 128 ; CNRS , UMR 5086 ; IBCP , Institut de Biologie et Chimie des Protéines , 7 passage du Vercors , Lyon , F-69367 , France 4University of Wrocław , Faculty of Biotechnology , Tamka 2 , 50-138 Wrocław , Poland Replication of the bacterial chromosome is initiated by binding the DnaA protein to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Multiple pathways regulating DnaA function in Escherichia coli : Distinct roles for DnaA titration by the datA locus and the regulatory inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_70">
175 , 6537 -- 6545 Katayama , T. , Fujimitsu , K. and Ogawa , T. ( 2001 ) Multiple pathways regulating DnaA function in Escherichia coli : distinct roles for DnaA titration by the datA locus and the regulatory inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></label>
<input id="act_similarity_70", type="checkbox">
<div class="ACT_similar">
<li>Katayama , T. , Fujimitsu , K. , and Ogawa , T. ( 2001 ) Multiple pathways regulating DnaA function in Escherichia coli : distinct roles for DnaA titration by the datA locus and the regulatory inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li></div>
<li>Bacterial two-hybrid analysis of interactions between DiaA , HobA , and H. pylori and E. coli DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Augustin , L. , Jacobson , B. , and Fuchs , J. ( 1994 ) Escherichia coli FIS and DnaA proteins bind specifically to the nrd promoter and affect expression of a nrd-lac fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Escherichia coli prereplication complex assembly is regulated by dynamic interplay among Fis , IHF and DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Escherichia coli cells with increased levels of DnaA and deﬁcient in recombinational repair have decreased viability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>The rcbA Gene Product Reduces Spontaneous and Induced Chromosome Breaks in Escherichia coli Magdalena M. Felczak and Jon M. Kaguni Department of Biochemistry and Molecular Biology , Michigan State University , East Lansing , Michigan , USA Elevated levels of DnaA cause excessive initiation , which leads to an increased level of double-strand breaks that are proposed to arise when newly formed replication forks collide from behind with stalled or collapsed forks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Department of Biology , B-022 , and the Center for Molecular Genetics , University of California , San Diego , La Jolla , CA 92093 , USA Key words : Escherichia coli - DNA replication - GATC methylation - dam gene expression - DnaA sites * Permanent address : Institute of Biochemistry and Biophysics , Polish Academy of Sciences , Rakowiecka 36 , PL-02 532 , Warsaw , Poland Offprint requeststo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>sscciieennttiiffiic report cre p or t Hda inactivation of DnaA is the predominant mechanism preventing hyperinitiation of Escherichia coli DNA replication Johanna E. Camara1 * , Adam M. Breier 2 * w , Therese Brendler 3 , Stuart Austin3 , Nicholas R. Cozzarelli2 & Elliott Crooke1 + 1Department of Biochemistry and Molecular Biology , Georgetown University Medical Center , Washington , District of Columbia , USA , 2Molecular and Cell Biology , University of California , Berkeley , Berkeley , California , USA , and 3NCI-DBS , Frederick Cancer Research and Development Center , Frederick , Maryland , USA Initiation of DNA replication from the Escherichia coli chromosomal origin is highly regulated , assuring that replication occurs precisely once per cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>Camara , J. E. , A. M. Breier , T. Brendler , S. Austin , N. R. Cozzarelli et al. , 2005 Hda inactivation of DnaA is the predominant mechanism preventing hyperinitiation of Escherichia coli DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Hda inactivation of DnaA is the predominant mechanism preventing hyperinitiation of Escherichia coli DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>[ PubMed : 12730188 ] Camara JE , Breier AM , Brendler T , Austin S , Cozzarelli NR , Crooke E. Hda inactivation of DnaA is the predominant mechanism preventing hyperinitiation of Escherichia coli DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Camara , J.E. , Breier , A.M. , Brendler , T. , Austin , S. , Cozza-relli , N.R. , and Crooke , E. ( 2005 ) Hda inactivation of DnaA is the predominant mechanism preventing hyperinitiation of Escherichia coli DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18761688" target="_blank">18761688</a>]<br></li>
<li>Molecular Microbiology ( 1994 ) 14 ( 4 ) , 763-772 SeqA limits DnaA activity in replication oriC in Escherichia coii Ulrik von Freiesleben , * ̂ Knud V. Rasmussen ^ and Moseiio Schaechter ^ '' Department of Molecular Biology and Microbiology , Tufts University Medical School .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_84">
von Freiesleben , U. , K. V. Rasmussen and M. Schaechter , 1994 SeqA limits DnaA activity in replication from oriC in Escher-ichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></label>
<input id="act_similarity_84", type="checkbox">
<div class="ACT_similar">
<li>von Freiesleben , U. , Rasmussen , K.V. and Schaechter , M. ( 1994 ) SeqA limits DnaA activity in replication from oriC in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>von Freiesleben , U. , Rasmussen , K.V. , and Schaechter , M. ( 1994 ) SeqA limits DnaA activity in replication from oriC in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>von Freiesleben , U. , Rasmussen , K.V. & Schaechter , M. ( 1994 ) SeqA limits DnaA activity in replication from oriC in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>[ 4 ] U. Von Freiesleben , K.V. Rasmussen , M. Schaechter , SeqA limits DnaA activity in replication from oriC in Escherichia coli , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li></div>
<li>[ PubMed : 15752198 ] von Freiesleben U , Rasmussen KV , Schaechter M. SeqA limits DnaA activity in replication from oriC in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_90">
SeqA limits DnaA activity in replication from oriC in Escherichia coll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></label>
<input id="act_similarity_90", type="checkbox">
<div class="ACT_similar">
<li>SeqA limits DnaA activity in replication from oriC in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981994" target="_blank">8981994</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004192" target="_blank">11004192</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20875449" target="_blank">20875449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li></div>
<li>It was demonstrated in vivo that SeqA limits DnaA activity in replication from the chromosomal origin oriC [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_93">
It has been demonstrated that , in vivo , SeqA limits DnaA activity in replication from oriC ( von Freiesleben et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></label>
<input id="act_similarity_93", type="checkbox">
<div class="ACT_similar">
<li>It has been demonstrated that in vivo SeqA limits DnaA activity in replication from oriC ( von Freiesleben et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li></div>
<li>DNA initiation increase in parallel in synchronized exponentially growing cells , and DnaA appears to be involved in the regulation of both , it is surprising that the DnaA-binding boxes do not appear to be required for increased nrd expression during DNA inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>These results allow us to propose a model for nrdAB gene regulation by which DnaA can inﬂuence the total amount of RNR expression but not its timing as a function of DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>In addition to its role in initiating DNA replication , DnaA is a transcriptional-regulator that controls positively or negatively the expression of nrdAB by binding to DnaA boxes immediately upstream of the nrdAB operon ( 32 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>A recent report , where this phenomenon has been studied in more detail ( Atlung and Hansen , 1993 ) , shows that at very high levels of DnaA overproduction origin to mass ratios increase and cells do not finish replication in presence of rifampicin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>Overproduction of DnaA does lead to overinitiation of replication , but initiations remain synchronous ( Skarstad ef aL. 1989 ; Atlung and Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>Fine tuning of the gua operon , which is required as the demand for dNTPs fluctuates over the DNA replication cycle [ 48 ] , could be provided by control through DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Thus , trans-criptional activity of thegzaa operon is sensitive to DnaA concentration and is thereby coupled to the DNA replicatiun cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>The transcriptional activity of the gua operon , by being sensitive to DnaA concentration , is therefore coupled to the DNA replication cycle [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Thus , transcriptional activity of the gua operon is sensitive to DnaA concentration and is thereby coupled to the DNA replication cycle [ 20 , 211 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>0006-291W97 $ 25.00 Copyright O 1997 by Academic Press All righu of reproductionin any lorm reserved Zn vitro complementation assay of DnaA activity for minichromosome replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>After the first or second stage , replication activity of DnaA was assessed in a minichromosomal replication system as described in Figure 1E .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Competition between the replication initiator DnaA and the sequestration factor SeqA for binding to the hemimethylated chromosomal origin of E. coli in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20640526" target="_blank">20640526</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20875449" target="_blank">20875449</a>]<br></li>
<li>In vivostudiesof DnaA binding to the origin of replication of fscherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>The speciﬁc binding of SeqA to these sequences that are abundant in oriC is thought to sequester the replication origin from DnaA and other replication proteins ( 6 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>In Escherichia coli , the initiation of DNA replication requires the recognition of speciﬁc sequences in its single replication origin ( oriC ) by DnaA bound to ATP , which leads to the stepwise assembly of the molecular machinery at each replication fork ( reviewed in references 46 , 54 , and 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>DnaA and oriC , the bacterial mode to initiate DNA replication , FEMS Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15865419" target="_blank">15865419</a>]<br></li>
<li>pacity and , as nontagged DnaA , is able to rescue a recA2dnaA temperature-sensitive mutant at its nonpermissive temperature even in the absence of inducer ( data not shown ) , we next examined its ability to initiate DNA replication in vitro from oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>DnaA binds to several high-afﬁnity binding sites , known as DnaA boxes , near the origin of replication , oriC , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Therefore the pBR322 origin of replication is the only active origin , but most of the oriC sequence , including the four DnaA boxes , and the mioC DnaA box is still present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>DnaA binds to four DnaA boxes within the unique 245-base pair origin of replication where it facilitates DNA strand separation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>One of these is dnaA , the product of which binds to sites ( DnaA boxes ) in the origin of replication , oriC , to facilitate the opening of the DNA duplex to start replication ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981994" target="_blank">8981994</a>]<br></li>
<li>Interaction of DpiA with rep-lication origins competes with binding of the replication proteins DnaA and DnaB : When overexpressed , DpiA can interrupt DNA replication and induce the SOS response ( 4 ) , thereby inhibiting cell division ( 4 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15308764" target="_blank">15308764</a>]<br></li>
<li>For instance , chromosome replication and cell constriction are not initiated until the required threshold levels of active DnaA and FtsZ proteins , respectively , have assembled ( 26 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981994" target="_blank">8981994</a>]<br></li>
<li>Impaired replication of l plasmids in dnaA ( ts ) and seqA strains and in double dnaA ( ts ) seqA mutants at 43 8C can be explained by abolition of DnaA and SeqA-mediated stimulation of pR under these conditions ( compare Figure 2 with Figures 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>At this temperature , stimulation of the pR promoter by DnaA ( ts ) is only partially impaired .32 Therefore , apart from moderately decreased pR activity , which results in impairment of replication initiation at oril , a second mechanism operates leading to death of a dnaA mutant host cell bearing wild-type l plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>This control result is seen in Fig. 3A , In the seqA2 mutant , initiations were asynchro-nous ( Fig. 3D ) as seen by the appearance of discrete peaks corresponding to 3 , 4 , 5 and 6 chromosomes Genetic evidence for interaction between SeqA and DnaA The above results indicate that seqA plays a role in the control of initiation of replication , probably by controlling the ability of DnaA to function in initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>Activation of the alternative replication origins in the presence of the intact oriC and dnaA did not suppress the inhibition of colony formation caused by DARS2 oversupply ( YT411 in Fig. 4F ) , which excludes the possibility that the colony formation inhibition could result from inhibition of the oriC function by titration of replication initiation factors such as DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>mioC transcription is shut off prior to initiation of chromosome replication , due to the binding of DnaA upstream of its promoter , and remains off for several minutes after initiation of replication ( 2 , 29 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981994" target="_blank">8981994</a>]<br></li>
<li>The mode of DnaA association with oriC in the presence of HobA might confirm in vivo observations concerning the necessity of HobA in initiation of H. pylori chromosome replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>This would explain why HobA is absolutely required for initiation of H. pylori chromosome replication .11,12 DiaA , in contrast to HobA , slows DnaA binding to oriC ( Fig. 4d ) but probably makes the orisome structure more organized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Further characterization of the biochemical properties of this interesting DnaA mutant should clarify its genetic effects and the role of domain II in the regulation of replication initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>This could be because neither deletion of DnaA domain II nor of DiaA fully negates the activation of DnaA for replication initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Alternatively , both DiaA and DnaA domain II could be independently required for a step in the activation of DnaA for replication initiation ; for exam ple , both could promote DnaB recruitment by formation of speciﬁc DnaA complexes at oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DiaA was shown to-bind the domains I and II of DnaA and was shown to timely regulate the initiation of replication in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Keywords : replication ; DnaA ; oriC ; Hda ; initiation EMBO reports ( 2005 ) 6 , 736 -- 741 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>( G ) Reactivation of DnaA in replication initiation activity by DARS2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This argues that , although DnaA concentration may be an important parameter in the control of initiation of chromosome replication , it can not be the sole determinant under both the above growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Initiation of replication in response to the lowering of hyperstructure diversity may result partly from the dissociation of hyperstructures that previously contained DnaA so releasing it to join a DnaA-initiation hyperstructure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>As a specific example , SeqA might allosteritally impede the transition of DnaA from a conformation that functions early in the replication initiation process to another that is required for later steps , thereby making the required transition sharply dependent upon either the concentration of DnaA itself or the concentration of some other critical factor ( s ) , as yet unidentified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>Control of replication initiation is important for survival of DNA damage In the absence of SeqA , the origin fails to be sequestered and cells initiate new rounds of replication in a manner dependent on the initiation factor , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Those directly involved in the cell cycle include initiation , replication and division hyperstructures based on DnaA , SeqA and the 2-minute cluster , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>Keywords DnaA ; Hda ; β clamp ; SeqA ; datA ; initiation Introduction Chromosomal DNA replication in free-living organisms is a highly regulated event that is coordinated with the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>The results suggest that SeqA , datA , and Hda with the clamp inhibit initiation under conditions that increase the cellular abundance of DnaA ( 29 ) , such as DNA damage by methyl methanesulfonate , which leads to blocked replication forks ( 76 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Specific Roles of SeqA , DnaA , and GATC Methylation during Replication Initiation If the genetic interactions observed here reflect direct physical interactions during replication initiation , they imply that DnaA activity directly senses the methylation status of GATC sites , while SeqA interacts negatively with DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>Specific Roles of SeqA , DnaA , and GATC Methylation during Sequestration It is attractive to suppose that the interactions between SeqA and DnaA and between DnaA and GATC methylation suggested above for replication initiation might also apply during sequestration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>SeqA and DnaA proteins might be involved in the formation of such a structure , which would control replication initiation from oril by both facilitating stabilization of the heritable replication complex and regulating pR-dependent transcriptional activation of the origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>Activity of pR plays a crucial role in the regulation of l DNA replication initiation .23 -- 25 Therefore , DnaA and SeqA-mediated regulation of pR was also studied in the light of its role in the control of replication of plasmids derived from bacteriophage l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>This experimental system appeared especially interesting , as functions of DnaA and SeqA in the control of replication initiation from oriC are opposite ,8 while both these proteins have similar effects on the activity of pR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Therefore , DnaA and SeqA-mediated regulation of pR activity seems to be very important for the control of replication initiation at oril .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>The physiological signiﬁcance of this DnaA and SeqA-mediated regulation of pR is demonstrated by studies on l plasmid DNA replication in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>tween DnaA and other proteins involved in its regulation and chromosomal replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Inactivation of the seqA gene has been shown to impose high negative superheli-city on the E. coli chromosome .32 In order to separ-ate the contributions of the altered nucleoid superhelicity in initiation control from that caused by delayed re-methylation of the oriC region in seqA mutants , we have included in this study a topA mutant and a topAgyrB mutant known to have a higher than normal negative superhelicity of the chromosome .33,34 Mutations in DnaA and topoisomerases are known to cause asynchronous initiation12 ,35 in presence of SeqA and Dam functions ; the eclipse periods in these strains are as yet unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></li>
<li>Recently , it was demonstrated that even stable maintenance of oriC-plasmids is dependent upon active sequestration functions .46 DnaA function and nucleoid structure have active roles in maintaining the eclipse period and synchrony We have shown in a previous work that loss of sequestration ( seqA and dam mutations ) reduces but does not abolish the eclipse .14 The results presented above ( Table 1 and Figure 3B ) show that the eclipse period can be reduced and synchrony of initiation abolished even in the presence of SeqA and Dam functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></li>
<li>However , loss of the SeqA function reduces the eclipse period to 0.12 -- 0.16 generation time14 ( see Table 1 ) in the presence of DnaA activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></li>
<li>We did observe a 1.8-fold increase in DnaA content in the seqA2 mutant , but this could be a consequence of the increase in gene dosage of the dnaA gene resufting from the increase in initiation and the proximity of fhe dnaA gene to oriC-When DnaA is overproduced , the origin to mass ratio as weli as DNA to mass ratio increases ( Atlung et ai , 1987 ; Atlung and Hansen , 1993 ; Skarstad ef aL , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>We did observe a 1.8-fold increase in DnaA content in the seqA2 mutant , but this is probably a consequence of the increase in gene dosage of the dnaA gene caused by the increase in initiation and the proximity of the dnaA gene to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>Therefore , all three analyzed orisome components , oriC , DnaA and HobA/DiaA , presumably act in concert to form an active orisome and therefore are not interchangeable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>This locus can bind to a considerable number of DnaA molecules , which therefore reduces the number of DnaA molecules that are accessible to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>tion of oriC requires the proteins DnaA and SeqA , which bind at and sequester hemimethylated oriC ( 1 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>An oriC plasmid , which is inert for initiation because the 13-mer region has been deleted and thus can not be replicated ( Bramhill and Kornberg , 1988 ) , was unable to stimulate DnaA inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>In the detachment model , presented here , cyclical changes in the phospholipid composition of the cytoplasmic membrane activate initiator proteins such as Dn ~ A protein and cause origins to detach ; this detachment allows torsional stresses to open 13mer sequences in oriC ; DnaA assists in the serial opening of these sequences and guides the entry of the helicase to form a pre-priming complex and trigger initiation ; the greater affinity of hemi-methylated origin for membrane is re-interpreted as a mechanism for preventing re-initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>The detachment model has the following requirements : a cyclical involvement of phospholipids in initiation ; an affinity between oriC and membrane ; an increase in this affinity when oriC is hemi-methylated ; a relationship between phospholipids and the activation of proteins essential to initiation such as DnaA ; an unwinding of as a result of oriC detachment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>Strain CM987 with the DnaA overproducing plasmid pLSK5 in the uninduced state showed a termination pattern intermediate between the plasmid-free strain and the fully induced situation , due to the incomplete repression of the tac promoter in this plasmid , pLSK5 is , for example , able to suppress dnaA ( ts ) mutations without induction ( Schauzu et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>Transcription of the rpoH gene from these promoters is regulated by CRP -- cAMP , CytR ( Ramirez-Santos et al. , 2001 ; Kallipolitis & Valentin-Hansen , 1998 ) and DnaA ( Wang & Kaguni , 1989b ; Messer & Weigel , 1997 ) proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>In order to investigate a possible interaction between SeqA and DnaA , we performed ELISA with purified SeqA and DnaA proteins as described by Choda-varapu et al. , but could not demonstrate specific interaction between the two proteins .39 We found that both SeqA and DnaA proteins easily stick to other proteins in vitro ( for instance BSA ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Discussion Interactions of E. coli DnaA and SeqA proteins with cellular membranes have been reported previously ( Sekimizu and Kornberg 1988 ; Sekimizu et al. 1988 ; Yung and Kornberg 1988 ; Slater et al. 1995 ; Garner and Crooke 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>However , although regulation of the activities of these proteins by membranes or their components was reported ( Sekimizu and Kornberg 1988 ; Yung and Kornberg 1988 ; Garner and Crooke 1996 ) or at least suggested ( Slater et al. 1995 ) , little is known about the in ¯ uence of DnaA and SeqA on the properties of cell membranes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>Although recent studies have suggested the pre-sence of protein factors whicb regulate the activity of the DnaA protein ( Lu et al. , .7994 ; von Freiesleben et al. , 1994 ; Katayama and Crooke , 1995 ) , many of the mechan-isms underlying the regulationof the DnaA activity in connection with the cell cycle require elucidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>At least three systems enforce the once-per-cell-cycle rule : the sequestration of the newly replicated oriC by SeqA protein ( von Freiesleben et al. , 1994 ; Lu et al. , 1994 ; Slater et al. , 1995 ; Brendler and Austin , 1999 ) , the titration of free initiator protein , DnaA , by the datA locus ( Kitagawa et al. , 1996 ; 1998 ; Ogawa et al. , 2002 ) , and the regulatory inactivation of DnaA protein ( RIDA ) ( Katayama et al. , 1998 ; 2001 ; Kurokawa et al. , 1999 ; Kato and Katayama , 2001 ; Su'etsugu et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>RIDA functions independently of the temporal inactivation of oriC by SeqA and the reduction of oriC-accessible DnaA molecules by DnaA titration on the datA locus ( Kurokawa et al. 1999 ; Katayama et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Apo-DnaA multimers on DARS1 are unstable Several AAA family proteins form multimers , and + nucleotides bound to these multimers mediate interactions between protomers , leading to an overall conformational change of the multimers ( Gai et al. 2004 ; Iyer et al. 2004 ; Enemark and Joshua-Tor 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Because of the apparent discrepancy between our results , which show that all three pathways are important in regulating initiation caused by an elevated level of DnaA , and those cited above ( Camara et al. , 2005 ) , we examined the effect of induced dnaA expression on the frequency of colony formation in isogenic repair-proficient strains lacking hda , seqA , or datA as well as a strain carrying null mutations in both seqA and datA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Loci that are bound by DnaA with high affinity hda , dnaN , seqA , and datA each affect the frequency of initiation at oriC when dnaA expression is elevated One study compared the frequency of initiation in strains lacking hda , seqA or datA ( Camara et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>One is that their increased gene dosage compensates for the chromosomally encoded level of the β clamp ( Burgers et al. , 1981 ; Leu et al. , 2000 ; Sutton et al. , 2005 ) , or Hda ( Su'etsugu et al. , 2005 ) , which is insufficient to reduce initiations caused by the increased level of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Indeed , the cellular DnaA level is not significantly increased even in hda mutant cells ( Fujimitsu et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Although the level of ATP-DnaA is reduced by the ygfZ disruption , we can not rule out the possibility that other inhibitory effects on initiation induced by the ygfZ disruption may be responsible for the suppression , as the cs growth of the dnaAcos mutant , which is defective in RIDA , is suppressed by the inhibition of initiation caused by the introduction of the seqA plasmid ( Lu et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>At 37 ° C , although initiation by DnaAcos relative to wild type DnaA was only slightly more frequent , this mutant DnaA promotes appreciably more double strand breaks to account for its toxicity ( Simmons et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Supporting the second possibility , DnaAcos is insensitive to IdaA , a partially purified cellular factor later identified as the β clamp , which acts with Hda to inactivate DnaA + in initiation ( Katayama and Kornberg , 1994 ; Katayama and Crooke , 1995 ; Katayama et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Because DnaA complexed to ATP represses dnaA expression , and Hda with the β clamp stimulates the hydrolysis of ATP bound to DnaA ( Katayama et al. , 1998 ) , the authors of the second study suggested that the effect of Hda on dnaA expression , coordinated with its role in modulating the nucleotide bound state of DnaA , regulates initiation ( Riber et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>This would result in the amount of RNR produced being proportional to the amount of free DnaA-ATP , consistent with the results of Beckwith and coworkers on the effect of DnaA mutations in vivo ( Gon et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA R334A , which has a substitution in the arginine residue of sensor 2 , is defective in DnaA-ATP hydrolysis in RIDA-derived complexes , although its affinity for ATP / ADP is intact ( Nishida et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Dialysis of DnaA According to the method of Speck et al. ( 1999 ) , DnaA was dialysed against ATP or ADP in order to obtain an ` exclusive ' nucleotide state of DnaA ( DnaA-ATP - or DnaA-ADP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Previously , we observed the ADP-to-ATP regeneration of DnaA in vivo under conditions in which RIDA and de novo DnaA synthesis were inhibited ( Kurokawa et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>During this period , DnaA is bound only to its high-afﬁnity sites in oriC ( sites that bind DnaA in both ATP and ADP forms ) and is precluded from binding to lower afﬁnity sites ( bound only by ATP-DnaA ) , whose occupancy is necessary for origin ﬁring ( Nievera et al. 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Domains III and IV , which bind ATP / ADP and DnaA box ( Roth and Messer 1995 ; Messer 2002 ; Fujikawa et al. 2003 ) , are intact in this allele , which is consistent with the possibility that initiation activity of DnaA508 might be supported by ATP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Moreover , DnaA functions as a transcription repressor , and this activity is also stimulated by ATP binding ( Messer and Weigel 1997 ; Gon et al. 2006 ) ; e.g. , ATP-DnaA represses dnaA transcription more tightly than ADP-DnaA ( Speck et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Mutants in hda are somewhat inviable and show over-replication , particularly when DnaA levels are elevated ( Kato and Katayama 2001 ; Camara et al. 2005 ; Riber et al. 2006 ; Fujimitsu et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>This DnaA-inactivation system ( RIDA , regulatory inactivation of DnaA ) decreases the level of cellular ATP-DnaA in a replication-coordinated manner ( Katayama et al. 1998 ; Kurokawa et al. 1999 ; Kato and Katayama 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>RIDA-defective cells have significantly elevated levels of ATP-DnaA and overinitiate chromosomal replication ( Kato and Katayama 2001 ; Nishida et al. 2002 ; Fujimitsu et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Lack of Hda increases the cellular ATP-DnaA level , causing overinitiation of replication and inhibition of colony formation ( Kato and Katayama 2001 ; Gon et al. 2006 ; Fujimitsu et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Given the positive and negative roles that DnaA ¬ ATP has on transcription regulation at the nrdAB promoter , one could propose that the timing of nrdAB expression could result from the oscillation in DnaA-ATP to DnaA-ADP ratio during the cell cycle [ ( Fujimitsu et al. , 2009 ) and reviewed by ( Katayama , 2001 ; Kaguni , 2006 ) ] and thus mirror DnaA 's interactions with the origin of DNA replication , oriC ( Gon et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Although the affinity of these mutant proteins for ATP / ADP is intact , they are defective in specific inter-DnaA interactions within the initiation complexes , and are thus unable to activate replication initiation ( Felczak and Kaguni 2004 ; Kawakami et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>The fusion gene is negatively controlled by DnaA by its binding to the DnaA box at the promoter region , and therefore a change in cellular DnaA level can be detected by measuring the β-galactosidase activity ( Hansen et al. , 1987 ; Kitagawa et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Multicopy plasmids carrying oriC and the adjacent mioC gene mitigate the toxicity caused by extra initiations DnaA transcriptionally regulates the mioC gene , which is located to the right of oriC on the genetic map , by binding to specific sites in the mioC promoter region ( Bogan and Helmstetter , 1996 ; Hansen et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>SeqA and DnaA exert their direct function in the regulation of pR activity by binding to unusually positioned sites -- downstream of the transcription start ( Szalewska-Pałasz , et al. , 1998 ; Słomińska et al. , 2001 , 2003a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>First , DnaA R399A and DnaA T435M , DnaA domain IV mutants lacking activity of overall DNA binding and DnaA box binding , respectively ( Sutton and Kaguni 1997 ; Blaesing et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This binding prevents remethylation of the N6-adenine residue ( Campbell and Kleckner 1990 ) , and sterically inhibits binding of DnaA for reinitiation ( Boye et al. 2000 ; Torheim and Skarstad 1999 ; Waldminghaus and Skarstad 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20640526" target="_blank">20640526</a>]<br></li>
<li>Here are represented binding sites for IciA , Fis , DnaA and NrdR ( Augustin et al. , 1994 ; Han et al. , 1998 ; Rodionov and Gelfand , 2005 ; Torrents et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>It has been shown that one DnaA monomer bound to a DnaA box induces a bend of 42 ° ; this distortion of the double helix in turn results in the increased accessibility to DNaseI cleavage observed here and at the dnaA promoter ( Schaper and Messer , 1995 ; Speck et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Recently , the presence of DnaA bound to the upstream sites of the dnaA P2 promoter has been shown to stabilize the activity of RNAP in the presence of the transcription inhibitor rifampicin ( Flåtten et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>This gene encodes NAD-linked aldehyde dehydrogenase with a broad substrate spectrum and is under control of several transcriptional-regulators that bind to sites within or near the region bound by DnaA ( Limon et al. , 1997 ; Pellicer et al. , 1999 ) ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>It is known that a 9 bp sequence , which binds DnaA when located in certain surrounding sequences does not bind it in other places ( Fuller et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Messer et al. [ 7 ] proposed that when DnaA binds to a second DnaA box , within the dnaAcoding region , it acts as a transcriptional terminator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Basically , DnaA binds to DnaA boxes even at 0 ° C ( Obita et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>However , a roughly twofold increase in DnaA content was observed in DseqA cells , consistent with previous observations ( von Freiesleben et al , 1994 , 2000a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>The discrepancy between our result and the mutational study ( Augustin et al. 1994 ) could be explained , if the amount of DnaA in the non-overproducing cell is already saturating for nrd regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>The activity of DnaA is dependent on its nucleotide bound state ( Speck et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Determination of DnaA nucleotide form and cellular DnaA content , and flow cytometry The nucleotide forms of cellular DnaA were determined as described previously ( Kurokawa et al. 1999 ; Fujimitsu et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>In addition to its role in initiation of replication , DnaA has been shown to autoregulate its own transcription ( Sekimizu and Kornberg , 1988 ) and to control either positively or negatively the expression of several other genes ( van den berg et al. , 1985 ; Wang and Kaguni , 1989 ; Zhou and Syvanen , 1990 ; Mizushima et al. , 1994 ; Skarstad and Boye , 1994 ; Kitagawa et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>DiaA forms a tetramer and directly interacts with multiple DnaA molecules and in vitro recruits DnaA to sites in oriC to stimulate open complex formation ( Keyamura et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Alternatively , the increased level of SeqA may inhibit re-initiation by prolonging the sequestered hemi-methylated state of oriC ( Bach et al. , 2003 ) , and by interfering with the binding of DnaA or DnaAcos to low affinity sites within oriC , which is necessary for initiation ( Nievera et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Although SeqA inhibits replication from oriC at high concentrations of DnaA in vitro , it stimulates this process at low DnaA concentrations ( Wold et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>However , it has been proposed that the direct effect of SeqA on replication initiation is rather the result of binding to hemimethylated oriC and releasing DnaA molecules from this region ( Taghbalout et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>Hybridization temperature was 51 ° C. 1 , 838 bp probe ; 2 , RNA from strain CM987 ; 3 , RNA from strain CM987/pLSK5 , not induced ; 4 , RNA from strain CM987/pLSK5 , 90 min induced with 1 mM IPTG ; 5 , ~ bX174/HaeIII marker Analysis of transcripts in the in vitro replication system The oriC and DnaA dependent in vitro replication system ( Fuller et al. 1981 ) constitutes a cell extract which is optimized for replication , not for transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>These proteins cooperate in loading the MCM complex to the origin of replication site analogously to the action of DnaA and DnaC in loading DnaB in bacteria ( Lee and Bell , 2000 ; Liu et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>DiaA rescues colony formation in a mutant strain that over-initiates chromosomal DNA replication and also stimulates minichromosome replication when the level of DnaA is low ( Ishida et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>It is a negative regulator of the onset of chromosome replication in vivo ( Slater et al. 1995 ; Boye et al. 1996 ) , and at high concentrations of DnaA in vitro ( Wold et al. 1998 ) , but it may stimulate replication at lo Communicated by A. Kondorosi A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>This replication-cycle-co-ordinated interchange of active and inactive forms of DnaA has been conﬁrmed in vivo ( Kurokawa et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>This is consistent with the expectation that more DnaA must be produced to-bind the DnaA boxes in the extra datA copies [ 22,24 ] , and could be achieved because the dnaA promoter is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>On the basis that several hundred DnaA molecules can apparently bind at this locus and that the deletion of this site leads to extra initiations , datA was proposed to titrate DnaA when in excess to avert extra initiations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Several lines of evidence argue for important interactions between DnaA and the membrane ( reviewed by Kornberg and Baker , 1991 ; Skarstad and Boye , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>DnaA is also a transcription factor that negatively regulates the expression of several other genes by binding to DnaA boxes located in their promoter regions ( reviewed by Skarstad and Boye , 1994 ; Messer and Weigel , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DnaA also negatively regulates several genes including dnaA , mioC , rpoH , guaBA , and uvrB through interaction with DnaA boxes in each of the respective promoters [ 357 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The M45-47box1 mutation results in a decrease of the / 2-value , 1.64 0.06 , which however is still higher than the control lac promoter , 1.37 0.10 , indicating that activation by DnaA may inﬂu-ence the cooperativity of induction as well as the overall expression level of the nrdAB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The effect of these mutations on the GFP ratio observed here conﬁrms their results and suggests that the presence of DnaA at the high-affinity sites inﬂu-ences the cooperativity and synchrony of activation by insuring that all the copies of the nrdAB promoter are occupied by RNAP at the time of induction ( Fig. 8C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The authors of these reports suggested that since DnaA is known to be activated by phospholipids , chemicals that interact with phospholipids and a $ ect the bilayer structure of the cytoplasmic membrane might negatively in ¯ uence the activation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>The reason for the increased DnaA concentration in the absence of SeqA might be an effect of the increased dnaA gene dosage during rapid growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>The DnaA concentration in the dnaA204 mutant was similar to the wild type , but higher in the DseqA and dnaA204DseqA strains in glucose ± CAA medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>DnaA concentrations under different growth conditions The DnaA concentration in wild-type cells grown in batch culture was found to be the same during rapid and slow growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>The level of DnaA induction by this plasmid was about 10-fold as measured by an immunoblotting experiment ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>The presence of DnaA in the samples was determined by Western blotting with His6 antibody .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>The presence of excessive numbers of DnaB molecules in the hda-185 mutant might abnormally interact with DnaA , resulting in the inhibition of overinitiation ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Other genes for which there is direct or circumstantial evidence for regulation by DnaA are mioC [ 8 ] , uvrB [ 9 ] , dam [ lo ] , rpoH [ ll ] , ftsQA [ 12 ] , pros ( drpA ) [ 13 ] , and nrdAB [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Alternatively , DnaA and SeqA could cooperate in the regulation of expression of the same genes involved in production of membrane components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>The roles of DnaA and SeqA proteins in ensuring that membrane structure is correctly maintained may be direct or indirect , and it is likely that several di $ erent molecular mechanisms are involved in the regulation of the properties of cellular membranes by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>Under aerobic conditions , the DnaA , Fis , and IciA proteins bind to speciﬁc sites in the promoter region to regulate nrdAB expression , while several cis-acting sites and an AT-rich region are important in coupling transcription , in a complex way , to the cell cycle ( 18 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>The transcriptional regulation of these genes requires several transcription factors ( including DnaA , IciA , FIS [ factor for inversion stimulation ] , Fnr , Fur , and NrdR ) , depending on the RNR class ; however , the factors that dictate the expression of some RNR genes in response to different environmental conditions are not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The aim of this work is to test this hypothesis and to elucidate the role of DnaA in the regulation of nrdAB expression and its possible effect on the timing of expression as a function of the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The nrdAB operon is transcriptionally activated by FIS , IciA , and DnaA and is cell cycle regulated ; furthermore , its expression is not dependent on Arc or Fnr ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The binding of proteins , Fis and DnaA , was found not to be essential for cell cycle regulation of the nrd operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>These results suggest that some sites ( Fis and DnaA binding sites ) , although not required for cell cycle regulation , may contribute to cell cycle regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>No significant difference was detected in the cellular content of DnaA in Dhda or DdatA cells when compared with wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>No increase in overall cellular DnaA content was associated with this overinitiation ( Fig 3 ; Table 1 ) , which suggests , at least for rapidly growing cells , that the nucleotide-bound state has a central role in suppressing re-initiation and that cells do not seem to downregulate expression of the initiator to compensate for increased initiation potential .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>Gon and coworkers have recently proposed a model in which the nucleotide bound state of E. coli DnaA regulates transcription of the nrdAB genes ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Gon and coworkers have proposed an additional control mechanism to regulate NrdAB activity in which an excess of dNTPs leads to-binding of dNTP by DnaA and repression of nrdAB expression ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>In order to test whether DnaA activation and repression of the nrdAB promoter inﬂuences its timing of expression we have measured the effect of relieving DnaA-dependent repression or activation on the pattern of gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA repression thus plays a role in limiting the amount of transcript produced at the nrdAB operon but not its timing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Whereas surplus DnaA should also repress the expression of the nrdAB-yfaE operon , this has no detectable effect on the cellular abundance of ribonucleotide reductase ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>When the intracellular concentration of DnaA is increased ( by induction of a multicopy plasmid carrying the dnaA gene fused to a tac promoter ) , expression of the gua operon is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Lowering the concentration of DnaA , either by sequestration with an plasmid or by placing a temperatureoriC sensitive d n d A mutant at the restrictive temperature , results in increased expression of gua .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Apparently following induced expression , excess DnaA oligomerizes at datA in the multicopy plasmid to reduce the frequency of initiations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>We also showed that a multicopy plasmid carrying the oriC-mioC region neutralized the lethal effect caused by induced dnaA expression , and suggest that DnaA when in excess binds to this DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>These results suggest that the activation by DnaA and the basal polA gene expressionare , at least partially , under stringent control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_248">
These results suggest that DnaA protein negatively regulates expression of the aldA gene under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></label>
<input id="act_similarity_248", type="checkbox">
<div class="ACT_similar">
<li>These results suggest that DnaA negatively regulates the aldA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>In an attempt to study polA gene expression under reduced levels of functional DnaA we measured p-galac-tosidase activity from pNH3 in the isogenic strain GW2 carrying the dnaA46 allele .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Since the dnaA gene is autoregulated , aseqA mutation could increase dnaA expression independent of sequestration if SeqA affects DnaA activity per se ( below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>Thus , decrease in DnaA concentration , but not DnaC concentration , appears to increase transcription activity into the dam gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>In M9 medium containing rhamnose in place of glucose and supplemented with Casamino acids , or in L-broth , where expression of the gene is induced , neither increase nor decrease of cellular DnaA concentration affected the level of β-galactosidase activity ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>In exponentially growing cultures we did not observe significant changes in polA expressionfollowing DnaA induction by IPTG ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Therefore , we performedcomparative S1 analyses to localize the transcription-initiationsites of polA and to demonstrate that the increased polA gene expression by DnaA overproduction is due to enhanced transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>In exponentially growing cells poIA expression is maximally induced ; therefore we did not observe increased po/A expression by DnaA overproduction at this stage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>However , in stationary cells where polA expression is normally decreased , artificial overproduction of DnaA can reveal its role in polA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>In addition to these , a number of R-type DnaA boxes of lower affinity have recently been identified .3 Thus , the presence of numerous weak DnaA boxes , in addition to the two high-affinity boxes , may explain the large amounts of DnaA bound at datA in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>13 , 676 -- 683 Duderstadt , K. E. , Mott , M. L. , Crisona , N. J. , Chuang , K. , Yang , H. and Berger , J. M. ( 2010 ) Origin remodeling and opening in bacteria rely on distinct assembly states of the DnaA initiator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Miller , D. , Grimwade , J. , Betteridge , T. , Rozgaja , T. , Torgue , J. , and Leonard , A. ( 2009 ) Bacterial origin recognition complexes direct assembly of higher-order DnaA oligomeric structures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>This observation can be explained by a loss of negative autoregulation by DnaA , as indicated by the loss of the effect of Mbox1 on the activity of dnaAP1 ( Figure 6D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>This increase , however , does not sufﬁce to maintain negative autoregulation of dnaAP1 which requires extended oligomerization of DnaA -- ATP , in agreement with the in vitro footprinting results showing a loss of dnaAP1 protection as the temperature is lowered ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Nucleotides bound to DnaA were recovered and analyzed as described in the Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Cells were grown in a medium containing ortho - [ P ] , nucleotide-bound DnaA was isolated , and 32 the bound nucleotides were analyzed using thin-layer chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Nucleo-tides bound to DnaA were recovered on nitrocellulose filters and analyzed using thin-layer chromatography and BAS2500 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>C , control using pre-immune serum ; αA , anti-DnaA antiserum.The total cellular levels of the nucleotides bound to DnaA were not signiﬁcantly affected by the ygfZ mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The effect of DARS1 and DARS2 on the stimulation of the exchange of the nucleotide bound to DnaA is like that of anionic phospholipids ( 13 , 82 , 98 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>We suggest that the formation of stable DnaA ± DNA complexes is enhanced in the absence of reefold relative to wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Indeed , formation of DARS complexes containing three or more DnaA molecules was inhibited for apo-DnaA , even when high levels of DnaA were input , and the remaining free DNA levels were similar between ADP-DnaA and apo-DnaA ( Fig. 7E , F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Thus , the stimulation of DnaA inactivation is evidently dependent on DNA chain elongation and not solely on the act of initiating at oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>( B ) Following the DnaA inactivation reactions , portions ( 5 ml ) were transferred to in vitro DNA synthesis complementation reactions ( 25 ml , 20 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Reactivation of DnaA by DNA sequence-specific nucleotide exchange in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>It is possible that the SeqA -- DNA structure may displace some of the bound DnaA when the DNA becomes hemi-methylated [ 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>In the presence of excess competitor DNA poly ( dI ) poly ( dC ) , DnaA bound only to the 251-bp fragment containing the DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>a ΔM of a complex , an increment of the molecular mass of the DnaA -- HobA/DiaA complex bound to DNA , calculated for one DnaA molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>enting recombination between the plasmid and chro - n Vitro Replication of oriC Minichromosomes A single colony of transformed BLR ( DE3 ) pLysS was Frozen cells were thawed on ice , incubated ( 15 min , Recombinant His10-DnaA protein was isolated using NA polymerase III * ; 25 ng of DNA polymerase III ine nucleotides tightly bound to His10-DnaA , and con - NA Binding naA and the resin were removed by treatment ( 30 ith HhaI methylase , as described ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>The 2 mM ATP ; 0.5 mM each of GTP , CTP , and UTP ; 0.1 crude cell lysate was clariﬁed by centrifugation mM each of dATP , dGTP , dCTP , and [ a-32P ] dTTP ( 100,000 g , 30 min , 4 ° C ) , and the supernatant fraction ( 50 -- 150 cpm/pmol ) ; 200 ng ( 600 pmol as nucleotide ) which contained the soluble recombinant DnaA protein pBSoriC ; 450 ng E. coli SSB ; 10 ng protein HU ; 400 ng was retained for further puriﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Lane 1 , whole extract from uninduced BLR ( DE3 ) pLysS cells ; lane 2 , whole extract from BLR ( DE3 ) pLysS cells after 1.5 h of induced expression ; lane 3 , soluble cell lysate from induced cells ; lane 4 , proteins not retained by resin ; lane 5 , nonspeciﬁcally bound proteins removed by 100 mM imidazole wash ; lane 6 , contaminant proteins removed by treatment with 7 M urea ; lane 7 , pool of proteins eluted with 1M imidazole ; lane 8 , eluted proteins equilibrated with buffer HD ; lane 9 , high-speed supernatant of eluted proteins equil-ibrated in Buffer HD ; and lane 10 , nontagged DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Influence of DnaA protein on the termination of the asnC transcript A B M. o b M c d e M gidA rnio [ osn [ M Coordinate Percentage of total transcripts found '' r CM 987 CM987 + CM987 + pLSK5 pLSK5 not 90 rain induced induced Ratio - - 872 CM987 + pLSK5 induced / CM987 - - 603 ¬ ¬ ¬ ¬ ¬ ¬ 310 281 271 5.8 11.4 4.0 11.9 9.3 6.0 1.4 4.9 14.9 3.3 0.4 11.2 17.2 2.8 0.5 0.5 3.6 1.9 0.3 0.5 2 % 11.3 6.5 4.0 7.3 2.6 9.4 14.9 6.1 5.7 12.3 6.3 3.4 1.6 3.0 2.1 4.2 7.9 7.2 15.7 18.4 6.9 4.1 6.6 6.4 8.9 2.6 3.3 0.2 0.4 1.3 5.1 3.2 1.2 2.1 10.2 0.6 0.4 3.2 5.2 6.6 - ¬ - - 194 11 - ¬ 2 ) - ¬ - - 118 a Densitometer tracings of tracks 2 , 3 and 4 in Fig. 4 Fig. 5A , B. Transcripts synthesized in the in vitro replication system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>Formation of the b sliding clamp by the g complex of pol III and inclusion of partially purified IdaB protein fraction are necessary for RIDA activity , and concomitant DNA synthesis stimulates this IdaB-dependent inactivation of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>This agrees with the small but significant activity of g complex DNA Replication Further Stimulates the Inactivation of DnaA Protein We found that inactivation of DnaA protein is further enhanced by the concomitant DNA synthesis action of pol III in an oriC plasmid replication system ( Figures 4A and 4B ) reconstituted with highly purified protein Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>of pathways that inhibit re-initiation n ariso p m o C If seqA , hda , or datA acts to regulate the frequency of initiation , the chromosomally encoded level of DnaA protein may initiate DNA replication in an unregulated manner in the respective null mutants , leading to the accumulation of lethal double strand breaks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>However , numerous species , such as Helicobacter pylori , lack most of them5 or encode species-specific factors such as Bacillus subtilis YabA .7 In general , both oriC and DnaA are regulated by multiple mechanisms that control the timing of initiation complex formation and coordinate replication with the cell cycle .5,6,8 These mechanisms control the assembly or disassembly of a multimeric protein -- oriC complex ( i.e. orisome ) and regulate transitions between inactive orisome and an active initiation complex able to unwind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In Enterobacteriaceae , SeqA protein seems to be the main negative regulator of the chromosome replication initiation .4 -- 8 It was proposed that SeqA may limit DnaA activity in DNA replication from oriC .5 Moreover , SeqA is essential for sequestration , which affects oriC in the newly replicated , hemimethylated state .4,6 SeqA also DnaA and SeqA proteins are main regulators ( positive and negative , respectively ) of the chromosome replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>If the steps in replication initiation at oriC in vivo correspond to those defined in vitro , a reasonable biochemical representation of the observed genetic relationships would be that DnaA protein is sensitive to GATC methylation at early steps ( e.g. , binding of DnaA to DNA and initial interactions among DnaA monomers ) , while SeqA protein is inhibitory to DnaA action at latter stages in the process ( e.g. , during formation of an open complex ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>Interaction of DnaA with its bindings sites at oriC , the origin of bacterial chromosome replication , leads to formation of the replication complex responsible for duplication of the genetic material .1 -- 3 To regulate initiation of DNA replication precisely , apart from a positive regulator ( DnaA protein in eubacteria ) , a negative control is also necessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Several lines of evidence suggest that the increased initiation from oriC seen in the seqA mutant is the result of enhanced activity of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>It found that the level of DnaA protein was increased 2-fold in the seqA mutant , which was overinitiating replication at oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>We expect that the copy number of any replicon that requires DnaA protein for replication will increase in the seqA mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>In-vitro helix opening of M. tuberculosis oriC by DnaA occurs at precise location and is inhibited by IciA like protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>Inactivation of DnaA Protein In Vitro DnaA protein was inactivated for replication of oriC plasmids as previously described ( Katayama and Crooke , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>DnaA protein , the product of the dnaA gene , is essential for initiation of replication from oriC both in vivo [ l ] and in vitro [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>In this paper , we describe a chromosomal locus which has a profound effect on the regulation of free DnaA protein c o x e m o n , and thus may contribute to the control of initiation of replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Cumulatively , while these two means of regulating initiation ( the b clamp -- dependent inactivation of DnaA protein and the methylation control of oriC ) appear to function in an independent manner , both seem necessary for precise control of the on-off switch for chromosomal replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Similarly , a minichromosome possessing an intact oriC sequence , which can complete the initiation process but can not be replicated in the absence of dNTPs , also failed to stimulate inactivation of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>This mechanism is in agreement with the observation that de novo protein synthesis is necessary for replication to be initiated at oriC ; without de novo synthesis , the cellular content of ATP-DnaA molecules is rapidly depleted by efficient inactivation to levels too low to support initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Once chromosomal replication is completed , this negative regulation is lost as disassembly of pol III occurs , and the capacity for DnaA protein to initiate DN synthesis can then increase in preparation for a subsequent round of replication ( for detailed discussion , see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>J. ( 2013 ) 449 , 333 -- 341 ( Printed in Great Britain ) doi :10.1042 / BJ20120876 Temperature-dependence of the DnaA -- DNA interaction and its effect on the autoregulation of dnaA expression Chiara SAGGIORO , Anne OLLIVER and Bianca SCLAVI1 LBPA ( Laboratoire de biologie et pharmacologie appliquée ) , CNRS , ENS Cachan , 94235 Cachan , France The DnaA protein is a key factor for the regulation of the timing and synchrony of initiation of bacterial DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>[ Keywords : DnaA reactivation ; nucleotide exchange ; initiation regulation ; functional DNA sequence ; protein complex ; replication cycle ] Supplemental material is available at http://www.genesdev.org .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>While negative auto-regulation is lost at dnaAP1 , the effects of both positive and INTRODUCTION The DnaA protein is a key regulator of the initiation of DNA replication in most bacterial species , affecting both the timing and synchrony of origin activation [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The primary function of DnaA is in the initiation of DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The transcriptional mechanisms include the presence of multiple promoters , which are regulated by temperature , state of supercoiling , catabolite state , and the DnaA DNA replication initiation protein of E. coli ( 3 , 39-43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1565647" target="_blank">1565647</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_303">
Goranov , A. , Katz , L. , Breier , A. , Burge , C. , and Grossman , A. ( 2005 ) A transcriptional response to replication status mediated by the conserved bacterial replication protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></label>
<input id="act_similarity_303", type="checkbox">
<div class="ACT_similar">
<li>Goranov , A.I. , Katz , L. , Breier , A.M. , Burge , C.B. , and Grossman , A.D. ( 2005 ) A transcriptional response to replication status mediated by the conserved bacterial replication protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li></div>
<li>As the origin is initiated the DnaA protein is released .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>As replication commences at the first origin within a single cell , the bound DnaA protein is assumed to be released .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>Fujikawa , N. , Kurumizaka , H. , Nureki , O. , Terada , T. , Shir-ouzu , M. , Katayama , T. , and Yokoyama , S. ( 2003 ) Structural basis of replication origin recognition by the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Typically , this required the addition of 0.5 lal of DnaA protein antiserum to the replication assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2546039" target="_blank">2546039</a>]<br></li>
<li>Initiation of replication is proposed to occur when site R3 binds DnaA protein as well ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>Discussion Replication in E. coli is controlled by the initiator protein , DnaA , which binds to multiple sites near the origin of replication , unwinds DNA and recruits the replicative helicase , DnaB to establish bidirectional replication forks ( Leonard and Grimwade , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Lobner-Olesen , A. , Atiung , T , , von Meyenburg , K , , and Boye , E. ( 1989 ) Initiation of DNA replication in Escherichia coii after overproduction of the DnaA protein , Mol Gen Genet 218 : 50-56 , Smith , D.W , , Garland .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>In the case of pBR322 dependent DNA replication , the ﬁnal concentration of the assay mixture was adjusted to 100 mM KCl and the DnaA protein fraction was omitted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20875449" target="_blank">20875449</a>]<br></li>
<li>The polyhistidine-tagged DnaA protein was comparable to nontagged DnaA protein for initiating in vitro DNA replication , binding to oriC DNA , binding of allo-steric effector adenine nucleotides , and interaction with membrane acidic phospholipids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>DnaA is the key protein in the initiation of bacterial DNA replication , binding with high affinity to the four DnaA boxes in oriC ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>1997 DnaA protein binds to the chromosomalorigin ( oriC ) to initiate DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Zawilak , A. , Durrant , M.C. , Jakimowicz , P. , Backert , S. , and Zakrzewska-Czerwinska , J. ( 2003 ) DNA binding speciﬁcity of the replication initiator protein , DnaA from Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>DNA binding specificity of the replication initiator protein , DnaA from Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>This inactivating action is targeted to DnaA protein ; other proteins involved in DNA replication are not affected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Transcription from the guaBA promoter is sensitive , through repression by the DnaA protein , to the demands for guanine nucleotides for DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Purified DnaAcos mutant protein is active for minichromosome replication in vitro , but unlike wildtype DnaA protein , it is defective for the high affinity binding of ATP and ADP ( Katayama , 1994 ) and insensitive to the DnaA-inactivating factor ( RIDA , Regulatory Inactivation of DnaA ; formerly named IdaA ) ( Katayama and Crooke , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>While searching for a novel DnaA-reactivating factor , we found previously that a specific DNA sequence can catalytically promote the release of ADP from ADP-DnaA , followed by ATP binding and reactivation of replication initiation in vitro ( Fujimitsu and Katayama 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Nevertheless we tried to `` supershift '' our hemimethylated specific complexes by using an anti-DnaA polyclonal antiserum ( 33 , 34 ) in our binding reaction , which at 1:100 dilution has already been shown to completely inhibit DnaA protein-dependent in vitro oriC plasmid replication ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>Significantly , the formation of a DnaA-ATP complex and the subsequent replication of an oriC plasmid can be achieved in vitro by the specific binding of DnaA protein to the acidic phospholipids PG and DPG in membrane vesicles of which the fluidity is critical ( Yung & Kornberg , 1988 ) ; a cyclical change in membrane composition and structure , as proposed in the detachment model , is entirely compatible with such a change also activating DnaA protein as a prerequisite for successful initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>As recently shown by Bramhill and Kornberg ( 1988 ) , DnaA protein acts by opening the DNA at an adjacent region in oriC , a process requiring the presence of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2524384" target="_blank">2524384</a>]<br></li>
<li>Assisted by protein HU or IHF , cooperative binding of 20 -- 40 monomers of ATP-bound DnaA protein per oriC stimulates duplex opening at the AT-rich 13-mer repeats in oriC , where several replication proteins are loaded to initiate DNA replication ( Kornberg & Baker , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>While both forms are capable of binding oriC , only ATP-DnaA protein is active in opening the duplex DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>ATP-bound DnaA binds to multiple sequences at the origin of replication , oriC , unwinding the DNA and promoting the binding of DnaB helicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>It is primarily regulated at the first step ; i.e. initiation .1 Bacterial chromosome replication is initiated by cooperative binding of the DnaA -- ATP protein to multiple sequences within oriC followed by DNA unwinding within the DUE region.2-4 After DNA unwinding , a set of replication fork-forming proteins is loaded ( e.g. DnaB helicase , DnaG primase and DNA III polymerase holoenzyme ) and then DNA replication starts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Although DnaA levels appear not to ﬂuctuate during the cell cycle , the onset of replication is regulated by the ATP-bound state of the protein or its interaction with binding sites in oriC ( reviewed in Messer 2002 ; Leonard and Grimwade 2005 ; Kaguni 2006 ; Katayama 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DnaA protein purified by this procedure showed an indistinguishable affinity for ATP , and activity for i n vitro replication of oriC plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Taken together , binding of ATP apparently leads to a conformational change required for the replication activity of DnaA protein , and stability of the structure is thus enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Since isolated replication intermediates generated during oriC plasmid synthesis caused , in the presence of IdaB , elevated hydrolysis of DnaA-bound ATP ( unpublished results ) , a fully assembled replisome may have enhanced interaction with DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Whereas the extract of cells having functional b subunit inactivated DnaA protein for minichromosomal replication ( Figure 2B ) and stimulated the hydrolysis of DnaA-bound ATP ( Figure 2C ) , the extract from cells cultured at the nonpermissive temperature was much less efficient for both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Purified DNA polymerase III b subunit or fraction V ( 100 units each , as b-subunit activity for ssDNA replication ) were examined for their ability to promote hydrolysis of ATP bound to DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Copyright 2009 by the Genetics Society of America DOI : 10.1534 / genetics .109.104760 A Role for Nonessential Domain II of Initiator Protein , DnaA , in Replication Control Kathryn L. Molt , Vincent A. Sutera , Jr. , Kathryn K. Moore1 and Susan T. Lovett2 ent of Biology an Departm d R osenstiel B asic M altham , Massachusetts 02454-9110 edical Sciences Research Center , Brandeis U niversity , W Manuscript received May 5 , 2009 Accepted for publication June 16 , 200 9 ABSTRACT The initiation of replication in bacteria is regulated via the initiator protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Messer , W. , Blaesing , F. , Jakimowicz , D. , Krause , M. , Majka , J. , Nardmann , J. , Schaper , S. , Seitz , H. , Speck , C. , Weigel , C. , Wegrzyn , G. , Welzeck , M. , and Zakrzewska-Czerwinska , J. ( 2001 ) Bacterial replication initiator DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>Wegrzyn , G. , Slalewska-Palasz , A , , Wegrzyn , A , , Obuchow-ski , M. , and Taylor , K. ( 1995 ) Transcriptional activation of the origin of coliphage A DNA replication is regulated by the host DnaA initiator function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DnaA inactivation in the presence of IdaB and DNA polymerase III is further stimulated by DNA synthesis , indicating close linkage between initiator inactivation and replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The chromosomal replication activities of His10-DnaA prepared by this method and nontagged DnaA isolated through an established , but time-consuming puriﬁcation scheme are comparable , indicating that the afﬁn-ity puriﬁed protein will be useful for elucidating key structural aspects of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>The domain swapping experiment showed clearly that DiaA and HobA are functional homologs and upon binding to DnaA they exert the same effect on the replication initiator protein ( Figs. 4b and d and 5c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>PicoGreen staining histograms for cultures of the following strains were grown at 37 in LB me-dium : wild-type MG1655 , seqAD STL7222 , dnaAD96-120 STL12692 , dnaAD96-120 seqAD STL12697 , diaAD STL12846 , diaAD seqAD STL12848 , dia-AD dnaAD96-120 STL12851 , and diaAD seqAD STL12854 DISCUSSION DnaA and replication control : The DnaA initiator protein controls the timing of bacterial DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_342">
Messer , W. , 2002 The bacterial replication initiator DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></label>
<input id="act_similarity_342", type="checkbox">
<div class="ACT_similar">
<li>The bacterial replication initiator DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_344">
23 , 1221 -- 1233 Kurokawa , K. , Nishida , S. , Emoto , A. , Sekimizu , K. and Katayama , T. ( 1999 ) Replication cycle-coordinated change of the adenine nucleotide-bound forms of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></label>
<input id="act_similarity_344", type="checkbox">
<div class="ACT_similar">
<li>Kurokawa , K. , Nishida , S. , Emoto , A. , Sekimizu , K. , and Katayama , T. ( 1999 ) Replication cycle-coordinated change of the adenine nucleotide-bound forms of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_346">
Three distinct chromosome replication states are induced by increasing concentrations of DnaA protein in Escherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></label>
<input id="act_similarity_346", type="checkbox">
<div class="ACT_similar">
<li>Atiung , T. , and Hansen , F.G. ( 1993 ) Three distinct chromosome replication states are induced by increasing concentrations of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>181 , 2683 -- 2688 Atlung , T. and Hansen , F. G. ( 1993 ) Three distinct chromosome replication states are induced by increasing concentrations of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>References Atlung , T. , and Hansen , F.G. ( 1993 ) Three distinct chromosome replication states are induced by increasing concentrations of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Atlung , T. , and Hansen , F.G. ( 1993 ) Three distinct chromosome replication states are induced by increasing concentrations of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12068816" target="_blank">12068816</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12067334" target="_blank">12067334</a>]<br></li>
<li>Atlung , T. and Hansen , F.G. ( 1993 ) Three distinct chromosome replication states are induced by increasing concentrations of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li></div>
<li>Wojciecha 5 , 81-347 Gdynia , Poland 3Laboratory of Molecular Biology , Institute of Biochemistry and Biophysics Polish Academy of Sciences University of Gdańsk Kl ⁄ adki 24 , 80-822 Gdańsk Poland Keywords : transcription activation ; protein -- DNA interactions ; SeqA ; DnaA ; lambda plasmid DNA replication * Corresponding author Introduction The DnaA protein of Escherichia coli is the initiator of chromosome replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>241 , 411-416 ( 1996 ) 0FEBS 1996 Specific binding of DnaA protein to a DnaA box in the guaB gene of Escherichia coli K12 Fisehaye TESFA-SELASE and William T. DRABBLE Department of Biochemistry , University of Southampton , England ( Received 19 April/30 July 1996 ) - EJB Expression of the guaBA operon of Escherichia coli is regulated by the DNA replication-initiating protein , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Crooke , E. , Thresher , R. , Hwang , D. S. , Grifﬁth , J. , and Kornberg , A. ( 1993 ) Replicatively active complexes of DnaA protein and the Escherichia coli chromosomal origin observed in the electron microscope .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_355">
Kitagawa , R. , Ozaki , T. , Moriya , S. , and Ogawa , T. ( 1998 ) Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional afﬁnity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></label>
<input id="act_similarity_355", type="checkbox">
<div class="ACT_similar">
<li>Kitagawa , R. , Ozaki , T. , Moriya , S. , and Ogawa , T. ( 1998 ) Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional affinity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_357">
Atlung , T. , Løbner-Olesen , A. & Hansen , F.G. ( 1987 ) Overproduction of DnaA protein stimulates initiation of chromosome and minichromosome replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></label>
<input id="act_similarity_357", type="checkbox">
<div class="ACT_similar">
<li>Atlung , T. , L0bner-Olesen , A. and Hansen , F.G. ( 1987 ) Overproduction of DnaA protein stimulates initiation of chromosome and minichromosome replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_359">
[ PubMed : 8175680 ] Katayama T , Crooke E. DnaA protein is sensitive to a soluble factor and is specifically inactivated for initiation of in vitro replication of the Escherichia coli minichromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></label>
<input id="act_similarity_359", type="checkbox">
<div class="ACT_similar">
<li>DnaA protein is sensitive to a soluble factor and is specifically inactivated for initiation of in vitro replication of the Escherichia coli minichromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>DnaA protein is sensitive to a soluble factor and is speciﬁcally inactivated for initiation of in vitro replication of the Escherichia coli minichromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li></div>
<li>Chromosome replication in E. coli is controlled at the level of initiation , and this stepseems to be regulated by the DnaA protein ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2841344" target="_blank">2841344</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_363">
[ 15 ] K. Sekimizu , A. Kornberg , Cardiolipin activation of DnaA protein , the initiation protein of replication in Escherichia coli , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></label>
<input id="act_similarity_363", type="checkbox">
<div class="ACT_similar">
<li>Sekimizu , K. , and Kornberg , A. ( 1988 ) Cardiolipin activation of DnaA protein , the initiation protein of replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li></div>
<li>D N A Replication in Escherichia coli is Initiated by Membrane Detachment of oriC An adequate model for the initiation of chromosome replication in Escherichia coli should explain why the introduction of multiple copies of the chromosomal origin of replication , oriC , does not perturb cells seriously and why such multiple origins are replicated synchronously ; it should explain why the key initiator protein , DnaA , is activated in vitro by binding specifically to acidic phospholipids and why the Dam methyltransferase is essential for the correct timing of initiation ; it should explain why phospholipid synthesis and fluidity are necessary for initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>In Escherichia coli , a round of replication commences with the binding of the initiator protein , DnaA , to the unique chromosomal origin ( oriC ) ( Kornberg and Baker , 1992 ; Skarstad and Boye , 1994 ; Messer and Weigel , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>[ 17 ] M. Makise , S. Mima , T. Katsu , T. Tsuchiya , T. Mizushima , Acidic phospholipids inhibit the DNA-binding activity of DnaA protein , the initiator of chromosomal DNA replication in Escherichia coli , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>The timing and synchrony of DNA replication initiation in Escherichia coli is controlled in part by the binding of the DnaA protein to the origin of replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_369">
Marszalek , J. , and J. M. Kaguni , 1994 DnaA protein directs the binding of DnaB protein in initiation of DNA replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10224240" target="_blank">10224240</a>]<br></label>
<input id="act_similarity_369", type="checkbox">
<div class="ACT_similar">
<li>Marszalek , J. , and Kaguni , J.M. ( 1994 ) DnaA protein directs the binding of DnaB protein in initiation of DNA replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li></div>
<li>It is an inhibitor of the onset of E. coli chromosome replication in vivo [ 6,7 ] and , at high concentrations , of the replication initiator protein , DnaA , in vitro , but it may stimulate replication at low DnaA concentrations in vitro [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>A novel role for cAMP in the control of the activity of the E. coli chromosome replication initiator protein , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8396119" target="_blank">8396119</a>]<br></li>
<li>In Escherichia coli , the activity of the initiator protein , DnaA , is the focal point of this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>5 , 201 Smith , R. W. , McAteer , S. and Masters , M. ( 1997 ) Autoregulation of the Escherichia coli replication initiator protein , DnaA , is indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Interaction of the initiator protein DnaA of Escherichia coli with its DnaA target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>A stimulation factor for hydrolysis of ATP bound to DnaA protein , the initiator of chromosomal DNA replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>[ 16 ] T. Mizushima , S. Nishida , K. Kurokawa , T. Katayama , T. Miki , K. Sekimizu , Negative control of DNA replication by hydrolysis of ATP bound to DnaA protein , the initiator of chromosomal DNA replication in Escherichia coli , EMBO J. 16 ( 1997 ) 3724 -- 3730 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>Conformational transition of DnaA protein by ATP : structural analysis of DnaA protein , the initiator of the Escherichia coli chromosome replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>ATP activates DnaA protein in initiating replication of plasmids bearing the origin of the E. coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_380">
Riber , L. , J. A. Olsson , R. B. Jensen , O. Skovgaard , S. Dasgupta et al. , 2006 Hda-mediated inactivation of the DnaA protein and dnaA gene autoregulation act in concert to ensure homeostatic maintenance of the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></label>
<input id="act_similarity_380", type="checkbox">
<div class="ACT_similar">
<li>Riber , L. , Olsson , J.A. , Jensen , R.B. , Skovgaard , O. , Dasgupta , S. , Marinus , M.G. & Løbner-Olesen , A. ( 2006 ) Hda-mediated inactivation of the DnaA protein and dnaA gene autoregulation act in concert to ensure homeostatic maintenance of the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li></div>
<li>[ PubMed : 10209749 ] Riber L , Olsson JA , Jensen RB , Skovgaard O , Dasgupta S , Marinus MG , Lobner-Olesen A. Hdamediated inactivation of the DnaA protein and dnaA gene autoregulation act in concert to ensure homeostatic maintenance of the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_383">
Hansen , F.G. , Atlung , T. , Braun , R.E. , Wright , A. , Hughes , P. and Kohiyama , M. ( 1991b ) Initiator ( DnaA ) protein concentration as a function of growth rate in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></label>
<input id="act_similarity_383", type="checkbox">
<div class="ACT_similar">
<li>Hansen , F.G. , Atlung , T. , Braun , R.E. , Wright , A. , Hughes , P. , and Kohiyama , M. ( 1991b ) Initiator ( DnaA ) protein concentration as a function of growth rate in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_385">
Hansen , F.G. , Koefoed , S. , S0rensen , L. and Atlung , T. ( 1987 ) Titration of DnaA protein by oriC DnaA-boxes increases dnaA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></label>
<input id="act_similarity_385", type="checkbox">
<div class="ACT_similar">
<li>Titration of DnaA protein by oriC DnaA-boxes increases dnaA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>Molecular Microbiology ( 1997 ) 23 ( 6 ) ,1193 -1202 DnaA protein stimulates polA gene expression in Escherichia coli Ariel Quiiiones , ' * Gudrun Wandt , ' + Sabine Kleinstauber ' * and Walter Messer ' ` lnstitut fur Genetik , Martin-Luther-Universitst , Domplatz 1 , D-06108 Hale , Germany .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DnaA protein stimulates polA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Quinones , A. , Wandt , G. , Kleinstauber , S. , and Messer , W. ( 1997 ) DnaA protein stimulates polA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411720" target="_blank">10411720</a>]<br></li>
<li>The gzta operon encoding two enzymes specificfor GMP biosynthesis is repressed , in addition , by DnaA protein ( Tesfa-Selase & Drabble , 1992,1996 ) and is regulated by stringent and growth-rate-dependent controls ( Davies & Drabble , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969519" target="_blank">8969519</a>]<br></li>
<li>In this bacterium , the expression of RNR genes can be positively or negatively regulated at the transcriptional level ; nrdAB expression is repressed by the NrdR protein , whereas the DnaA , IciA , and FIS ( factor for inversion stimulation ) transcriptional-regulators induce the expression of this operon ( for a review , see reference 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>REPLICATION initiation in bacteria is controlled by a number of factors that regulate the activity of the the AAA1 DnaA protein ( reviewed in Messer 2002 ; Leonard and Grimwade 2005 ; Kaguni 2006 ; Katayama 2008 ) , the functional and structural equivalent of eukaryotic Cdc6/Orc proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>A possible indication of an in vivo DnaA -- SeqA interaction has , however , been reported : The SeqA protein destabilized the mutant DnaA204 protein , suggesting that SeqA and DnaA migh interact , and that the resulting change in the conformation of DnaA increased the instability of DnaA204 .40,41 In these reports , it was also suggested , however , that the SeqA-directed conformational change in DnaA might alternatively be caused indirectly by changes in DNA topology performed by SeqA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>This negative autoregulation , combined with the growth-rate-dependence of dnaA expression and DnaA titration on the chromosome , has been proposed to contribute to maintaining a constant concentration of DnaA protein in the cell as a function of growth rate and temperature [ 21 -- 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>In the absence of autoregulation the intrinsic activity of the promoter could in theory sufﬁce to maintain the required average amount of DnaA protein per cell [ 25,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>In that study it was proposed that more DnaA per cell was required in these conditions because the protein may be less active , resulting in decreased autorepression and thus an increased protein concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>he dam-mediated repression and derepression of the dnaA gene ( Campbell and Kleckner , 1990 ) , duplication of multiple DnaA boxes which titrate free DnaA protein should contribute to this oscillation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>The enhancer effect by DnaA protein was specific for the pofA gene , because it could not be observed in several other genes independently of the presence or absence of DnaA boxes in their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>It has been shown previously that DnaA protein negatively regulates the expression of the rpoH gene , which is consistent with the presence of two DnaA boxes in close vicinity to the p3rpoH promoter region ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8478327" target="_blank">8478327</a>]<br></li>
<li>In dnaA mutants of E. coli , the expression of ,3 - galactosidase in a drpA ' - lacZ protein fusion is eight times higher than its expression in a cell that overproduces DnaA protein ( T. Luby , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1688424" target="_blank">1688424</a>]<br></li>
<li>In cells growing in minimal medium containing glucose , expression of β-galactosidase activity from an aldA -- lacZ fusion gene was suppressed by oversupply of DnaA protein and was enhanced by reducing the free DnaA level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Additionally , in a following study it was demonstrated that CAMP and the CAP protein repress spf transcription of Stimulation polA expression by DnaA is Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>To determine the influence of increased levels of DnaA protein on polA gene expression , bacteria harbouring pNH3 and pHK27 or pACYC184 as control were grown to an ODeo0of approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>In these previous experiments , the intracellular level of DnaA protein was increased by using an external inducer ( IPTG ) to act on a DnaA overproducing plasmid , or reduced either by sequestration with a multicopy plasmid carrying oriC or by placing a d n d - t s mutant under non-permissive conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>It seems , therefore , that malfunction of DnaA protein results in changes in the biological properties of membranes which could lead directly to higher sensitivity of the mutant cells to detergents , organic solvents and other reagents that interact with phospholipids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>The position of OnaA is indicated by the arrow ¬ DnaA oontent is increased in the seqA mutant By using anti-DnaA antibodies , we looked at the level of DnaA protein in protein lysates from exponentially growing cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>The yield of DnaA protein was increased at least 6-fold compared to an available method being used , and over 22 mg of the protein were obtained from only 100 g of cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Our results suggest that seqA affects the activity of the DnaA protein itself ( increased origin to mass ratio and and increased copy number of tninichromosomes as well as pBR322 derivatives ) rather than its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>These results suggest that the seqA2 n ^ utant has an increased DnaA protein activity , and titration of DnaA protein by introducing extra DnaA-boxes therefore increases the growth rate of the mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>5 4 3 Overinitiation in the seqA mutant The increase in growth rate seen after introduction of extra DnaA-boxes suggests that the activity or level of the DnaA protein has increased in the mutant to a levei where it impedes oeli growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>In the absence of methyiation , a free unmethylated origin will immediately be able to-bind DnaA ( no sequestration ) and since the number of DnaA-boxes has been doubled , the ratio of DnaA protein to DnaA-boxes will go down and consequently prevent the cascade .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>To determine whether DnaA binds to these DnaA boxes we performed in vitro DNase I footprinting experiments with purified DnaA protein and the same 5 ' - end-labelledDNA fragment used for S1 mapping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DnaA protein binds to numerous dnaA boxes around the chromosome [ 22,23 ] , but whether it is capable of forming higher-order structures at sites other than oriC is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>Detection of nucleotide-bound forms of cellular DnaA protein The hda null mutant , in which oriC is deleted and the alternative origins are activated by the rnhA mutation , was used ( Kato and Katayama , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The binding assay consisted of immobilization of BBD-DiaA on steptavidin beads , followed by application of extracts of His6-DnaA - or His6-DnaAD96-120-expressing cells ; bound protein was detected by Western blotting using an antibody to the His-tag moiety .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>In the absence of DnaA protein , all the Hinfl-BamHI [ 32P ] dATP-labeled restriction fragments of pOC42 ( Figure 4C ) disappear in proportion to the ddTTP concentration and their respective sizes-an indication that initiation is not favored within any particular place on the DNA template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>In contrast , when DnaA protein and rifampicin are added to the assays , essentially only DNA restriction fragments corresponding to OK and its immediate vicinity ( Figures 48 and 4C ) are seen at high concentrations of ddTTP The effect is all the more striking when one considers that these fragments are among the smallest and therefore should be the least radioactive if initiation were occurring at nonspecific sites on the DNA template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>The release of bound nucleotides from DnaA protein erspective romoted by acidic phospholipids in a ﬂuid bilayer , a sed in the puriﬁcation of numerous proteins , was rocess that may play a crucial role in cell cycle regu - xploited here for the isolation of an initiator protein , ation of chromosomal replication ( 6,29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>To address this issue , mechanisms to activate the initiator DnaA protein during the cell cycle have to be revealed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>When the initiation mass is reached , site R3 of one of the origins binds DnaA protein and initiation occurs ( B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>When DnaA protein accumulation approaches the level critical for initiation , all origins within the cell will have DnaA protein bound to sites RI , R2 and R4 ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>The presence of plasmids pFHC1425 and pALO237 in Dam - cells , is therefore not expected to result in a sudden sink in DnaA protein concentration at the precise time of chromosome initiation at a given oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>The methylation control acts to initially prevent reinitiations , and by the time oriC is no longer sequestered , presumably the initiation potential has dropped to levels too low to support initiation because of the inactivation of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The necessity of DnaA protein activated by ATP or cyclic AMP for the initiation of replication has been shown in vitro ( 6 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8396119" target="_blank">8396119</a>]<br></li>
<li>The cyclical occurrence of such bindings has yet to be shown but , if DnaA protein is activated cyclically , identification of the agents and processes responsible should make our understanding of initiation clearer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>The time of initiation appears to be controlled by the availability of active DnaA protein in the cell , iys -- 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>he level of free DnaA protein appears to oscillate , the time of initiation being !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>ated with anionic phosphatidylglycerol , but not when The initiation activity of DnaA protein is strongly ixed with zwitterionic phosphatidylcholine ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>How the initiation activity of the DnaA protein is regulated in the bacterial cell cycle remains to be solved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8396119" target="_blank">8396119</a>]<br></li>
<li>It is only a fraction of the origins that are initiated in the dam - cells , thus the decrease in DnaA/origin ratio will not be as marked as in the wild-type cells , leading to a shorter time period for the build-up of initiation potential ( accumulation of DnaA protein ) for the next initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>The basic postulate of the dam - cell cycle model is that the absence of the initiation cascade in dam - cells is caused by the missing sequestration of newly initiated dam-origins to the membrane , and thereby the rapid rebinding of DnaA protein to the DnaA binding sites in the origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>The major origin-recognition protein is DnaA , while DnaC protein recruits DnaB helicase to the origin ( Baker and Wickner , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>The authors suggest that normally sequestration of the hemimethylated origin prevents rebinding of DnaA , thereby increasing the rafio of DnaA protein to DnaA-boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>The result suggests that titration of DnaA molecules -- and thereby reduction of the availability of DnaA -- has a negative effect on sequestration and indicates that the DnaA protein contri-butes to keeping oriC and other sites hemimethylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>The key inactivation step appears to be membrane InhibItion of DnaA initiator protein bindlng to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_436">
The initiator protein DnaA contributes to keeping new origins inactivated by promoting the presence of hemimethylated DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></label>
<input id="act_similarity_436", type="checkbox">
<div class="ACT_similar">
<li>Bach , T. , Morigen & Skarstad , K. ( 2008 ) The initiator protein DnaA contributes to keeping new origins inactivated by promoting the presence of hemimethylated DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>and Skarstad , K. ( 2008 ) The initiator protein DnaA contributes to keeping new origins inactivated by promoting the presence of hemimethylated DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22373925" target="_blank">22373925</a>]<br></li></div>
<li>The band intensities were normalized by the intensity of a reference band that does not change as a function of DnaA concentration and relative to the intensity of the band in the absence of protein [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The gua operon is also regulated by the DNA replica-tion-initiating protein DnaA [ 14,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The authors claim that the concentration of the DnaA protein is normal in their experiment , but this means that the ratio DnaA : DNA is drastically elevated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12067334" target="_blank">12067334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12068816" target="_blank">12068816</a>]<br></li>
<li>Using a ﬁlter retention assay , His10-DnaA and DnaA protein were found to-bind oriC DNA located on a supercoiled plasmid to the same extent of approximately 15 -- 20 DnaA protomers per oriC-plasmid ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>In fact , homologues of this protein were found in all eubacteria analysed so far .1 -- 3 DnaA binds to speciﬁc DNA sequences , called DnaA boxes , and all functions performed by it require these protein -- DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>In this respect , DnaA protein has been shown to-bind preferentially to negatively supercoiled origin DNA ( Fuller and Kornberg , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>That is , the DnaA protein could directly bind DNA at the diagnostic restriction site and block remethylation by Dam methylase , or the DnaA could act indirectly by recruiting other proteins to the DNA at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>b subunit may undergo a conformational change when loaded onto DNA and in conjunction with IdaB protein , interact with DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Inclusion in the incubation of a tenfold excess of unlabelled DNA resulted in the same pattern of bands as the control incubation lacking DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Concomitant DNA synthesis further stimulates inactivation of DnaA protein ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_449">
Schaefer , C. , and Messer , W. ( 1991 ) DnaA protein/DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></label>
<input id="act_similarity_449", type="checkbox">
<div class="ACT_similar">
<li>[ 5 ] Schaefer , C. and Messer , W. ( 1991 ) DnaA protein/DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li></div>
<li>The rate at which DnaA protein is inactivated in an IdaB-dependent manner increases significantly with concomitant DNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>This was taken as the quantity of DNA synthesis which took place when all DnaA protein dependent rep-lication was inhibited by DnaA protein antiserum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2546039" target="_blank">2546039</a>]<br></li>
<li>Functional Comparison of Nontagged and Polyhistidine-Tagged DnaA Protein functional studies , its behavior in initiating DNA rep-lication from oriC and its response to regulation b For His10-DnaA protein to be useful in structural -- adenine nucleotides and membrane phospholipids must be the same as those for nontagged DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>However , although the mutant has no hemimethylated DNA , there may be weaker interactions between unmethyl-ated oriC sequences and , for example , the DnaA protein bound to the inner membrane surface ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1551854" target="_blank">1551854</a>]<br></li>
<li>Figure 6 shows that this is the most likely step to be inhibited by the outer membrane fraction , as the addition of even high levels of outer membrane fraction after prior preincubation of the hemimethylated oriC template with DnaA protein failed to inhibit subsequent DNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>Here , it was found that a GATC cluster inserted at the datA site , which binds large amounts of DnaA in vivo , did not become remethylated at all , unless the availability of the DnaA protein was severely reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>It is possible that the influence of the DnaA -- protein complex bound to GATC ∷ datA explains the somewhat larger increase in SeqA foci numbers seen in GATC ∷ datA-containing cells compared to GATC ∷ t-naA-containing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Binding of DnaA to the diagnostic restriction site in GATC ∷ datA would explain its protection also in the absence of SeqA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Perhaps sequestration is accomplished by the combined effects of SeqA protein interacting with the component GATC site in its hemimethylated form and DnaA protein interacting with the 9-mer and 13-mer sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>DnaA determination Samples for determination of DnaA protein content were taken from steady-state exponentially growing cells , haivested by centrifugation and redissolved in standard loading buffer for SDS-PAGE and boiied for 5 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>As a control , we verified that DnaA protein was induced to similar levels relative to the abundance of ribosomal protein L2 in portions of the same cultures that were analyzed by quantitative PCR ( Figure 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>It is also possible that conformational changes in the DnaA protein , which depend on the integrity of domain II , are also associated with this active conﬁguration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>This suppression can be attributed to the conversion from the accumulated active form to the inactive form of the initiator protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>In contrast to detailed characterizations of the functions , structural analysis of this protein remains largely unexplored , mainly because of difficulty in acquiring a large-scale supply of DnaA protein in the active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>In addition , we have data that y-phos-phate of ATP bound to DnaA protein is close to Lys-415 located in a putative a-helix in the C-terminal region ( Katayama , T. , Kubota , T. , Ito , K. , Tagaya , M. , Mizu-shima , T. , and Sekimizu , K. submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>In a subsequent step , the DnaA protein bound to DNA recruits DnaB helicase in the DnaB-DnaC complex to form a pre-priming complex ( Marszalekand Kaguni , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Under the same conditions , IciA bound to the 251 - and 195-bp fragment with much higher a nity compared to the binding of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>It is assumed that the bound DnaA protein is released from the binding sites in this process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>Cells were metabolically labeled with 32P-orthophosphate , shifted to an elevated temperature ( 428C ) for a period , lysed , and DnaA protein with tightly bound nucleotide was recovered by immunoprecipitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>DnaA protein was detected in the DiaA-bound fraction equally for DnaA1 and for DnaAD96-120 ( Figure 5 , lanes E and G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Reactions ( 25 ml ) were incubated for 20 min at 308C , and nucleotides bound to DnaA protein were assessed ( see Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_472">
After incubation at 308C for 20 min , nucleotides bound to DnaA protein were determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></label>
<input id="act_similarity_472", type="checkbox">
<div class="ACT_similar">
<li>Following the second incubation ( 308C , 20 min ) nucleotides bound to DnaA protein were determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li></div>
<li>Agarose gel electrophoresis of DNA bound to DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>DNA bound by DnaA protein was measured by retention on a nitrocellulose membrane ( see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>DnaA protein in lysates was isolated by immunoprecipitation , and nucleo-tides bound to DnaA protein were assayed by TLC ( see Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>However , it is not clear if these sites are actually bound by the DnaA protein , inasmuch - , '' * h. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Triangles at the top indicate increase of the bound DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>About 20 clones bound DnaA protein with varied affinities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>DnaA protein bound to the two DnaA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>From the ﬁgure , it is evident that DnaA protein bound the insert fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>( C ) Hydrolysis of [ a-32P ] ATP bound to DnaA protein in 5 ml portions was measured following the inactivation reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Each DnaA sample ( 1.9 pmol of the protein ) was incubated with various concentrations of [ `` PIATP a t $ C for 15 min and then [ 32P ] ATP bound to DnaA was measured by nitrocellulose filter binding assay ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Results DnaA protein 's capacity to initiate minichromosomal rep-lication can be inactivated by an activity termed RIDA , which may act by stimulating the hydrolysis of DnaA-bound ATP ( Katayama and Crooke , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>These mutants either result in a decrease in the activity of the RIDA process or in a DnaA protein that is always in an ATP-bound-like conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>One unit ( U ) of activity corresponds to a 2-fold stimulation in the hydrolysis of ATP bound to DnaA protein ( 1 pmol ) when assayed for 20 min at 308C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>In vitro , the b clamp -- dependent inactivation of DnaA protein does not require SeqA protein ( Katayama and Crooke , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The strong genetic interactions between seqA and dnaA have led us to consider the possibility that the primary effector of sequestration is in fact DnaA protein with its activity suitably modified by SeqA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>Thus , it has been proposed that phospholipids regulate the activity of DnaA protein in cells and in vitro [ 15,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>dnaA transcription is autoregulated by DnaA protein , and the nucleotide forms of DnaA affect its repressor activity in vitro ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>In this work , we show that the YgfZ protein is involved in the regulation of DnaA initiator activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Interaction of DnaA protein with the b clamp and IdaB may stimulate this intrinsic ATPase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>As expected frorn the inhibition of transcription from Px by DnaA protein , the activities decreased upon induction of the DnaA protein 1142 R. Kitagawa , H. Mitsuki , T. Okazaki and T. Ogawa from the plasmid pROA200 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Moreover , the P3 and P4 promoters may be repressed by DnaA protein ( Wang & Kaguni , 1989b ; Messer & Weigel , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>However , in the dnaA204 mutant , the DnaA protein was unstable in both glucose ± CAA and alanine medium ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>DnaA protein concentration in cells grown in glucose ± CAA or alanine media , or in a chemostat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Absence of SeqA protein stabilized the DnaA204 protein , increased the DnaA protein concentration and normalized the initiation mass in the dnaA204 mutant cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>In slowly growing cells , this results in a lower concentration of DnaA protein and a dramatic increase in the mass at initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>This effect was due to an increased termination of the asnC transcript in the intergenic region mediated by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>Apparently , the increased DnaA protein synthesis compensated for the protein degradation under these growth conditions , in which the doubling time was of the same order of magnitude as the half-life of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Discrete bands were resolved in the gel and slowly migrating species increased at higher DnaA protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Neither higher concentrations of IPTG nor prolonged periods of induction increased the synthesis of the recombinant His10-DnaA protein ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Replication levels have been corrected for background non-specific D N A synthesis which was taken as the residual synthesis in the presence of antibodies raised against purified DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2546039" target="_blank">2546039</a>]<br></li>
<li>DnaA protein was mixed with Tug1 fragments of pFT103 in the presence of poly ( d1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>DNase I footprints of DnaA box sequences in the EcoRI-Hind111 fragment of pFT103 in the presence of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>These data suggest that DnaA protein increases UFA biosynthesis ; the mechanism of this apparent activation is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>To monitor the stability of the DnaA proteins , we measured the ratio of DnaA to total protein after inhibition of protein synthesis with-chloramphenicol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>When over 40 ng of DnaA protein was added to the reactions , more slowly migrating complexes appeared , presumably caused by oligomerization of the DnaA protein ( Lee and Hwang 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>When added , DnaA protein was at a final concentration of 3 kg/ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>This is in contrast to the inhibition observed when DnaA protein is added after the outer membrane fraction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>Thus , it seems unlikely that the DnaA protein is present in these complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>Therefore , it seems unlikely that DnaA protein contributes here to outer membrane fraction-hemimethylated OriC interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>Both promoters are subject to repression by DnaA protein , albeit to a different extent ( 13-15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2841344" target="_blank">2841344</a>]<br></li>
<li>Termination of asnC transcripts at elevated levels of DnaA protein ; autoradiogram of Sl-protected fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>It has been reported that the dam gene is negatively regulated by the DnaA protein ( Jonczyk ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630320" target="_blank">1630320</a>]<br></li>
<li>for guanine nucleotide biosynthesis , is regulated by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Recent evidence indicates that excess DnaA protein inhibits rpoH transcription from P3 and P4 both in vivo and in vitro ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>This result suggests that binding of the DnaA protein to a dnaA box inhibits transcription through this complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>DnaA protein binds specifically to the consensus DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Indeed , dnaA transcription is negatively autoregulated by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Active orientation of the DnaA protein/DnaA box complex in transcription termination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Specifically , it appeared likely that the DnaA protein inhibits transcription from P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>This procedure should be widely applicable to the isolation of novel mutant forms of the protein and valuable in better characterization of the biochemical features and structural -- functional relationships of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>This supports the idea that autoregulation of the dna .4 gene is responsible for the coupling of the DnaA protein synthesis to cell mass increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8045420" target="_blank">8045420</a>]<br></li>
<li>This increase in DnaA activity could either be an increased ievel of DnaA in the cell or lack of some controlling element that renders the DnaA protein more active , or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>This increase in dam expression is dependent on the dnaA mutant allele used to decrease the intracellular concentration of `` active '' DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>Twofold increase in the amount ot DnaA protein in the seqA2 mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>DnaA protein was shown to-bind specifically only to the fragment carrying the consensus sequence DnaA box , and to protect this box from deoxyribonuclease I. Transcription termination resulting from the binding of DnaA to this box within the guaB gene explains repression by DnaA of the gua operon in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>We were interested in determining whether DnaA protein would specifically bind to one or both of these sites and thus explain the repressive action of the protein on gua expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Another in vitro system , one that is reconstituted with the puriﬁed proteins , DnaA , DnaB helicase , DnaC , DnaG primase , single-strand binding protein SSB , histone-like protein HU , gyrase , and DNA polymer-ase III holoenzyme , also efﬁciently replicates oriC plasmids ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>The DiaA protein , though increasing the amount of DnaA binding to oriC by 20 % ∼ 30 % ( assuming the EcDnaA to DiaA ratio in the orisome complex to be 1.6:110 to 2.5:113 ) , simultaneously decelerated DnaA binding to DNA ( Fig. 4d ; Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In Figure 1B we have examined under similar conditions the binding affinity of purified DnaA protein for methylated , unmethylated , and hemi-methylated oriC DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>In summary , our analyses showed that HobA and DiaA originate from a common ancestor and are functional homologs that stimulate and organize binding of the DnaA protein to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>The wild-type protein does not seem to be susceptible to degradation , but the enhancement of degradation of DnaA204 by SeqA may indicate that a conformational change of the DnaA protein is induced by interaction with SeqA -- DNA complexes , and may suggest a direct interaction between DnaA and SeqA , not only at oriC , but also at other binding sites around the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>Since preparations made from dnaA : : TnlO cells bind hemimethylated DNA , DnaA protein is not required either directly for oriC binding activity in these membrane preparations or indirectly for localization of SeqA protein to these fractions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>Thus , the activity of DnaA protein on dam gene expression may be somewhat different from its activities in the initiation process , yet retaining dependence on the presence of one or more DnaA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>The measurement of DnaA binding to the lower afﬁnity sites is less precise owing to the higher amounts of protein required and the subsequent non-speciﬁc interactions that take place on the DNA at higher DnaA concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>In order to quantify the titration levels of DnaA protein by the binding sitesdirectly , the amountsof DnaAprotein accumulated in cells were estimated by immunoblot analysis The binding of DnaA protejri ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>The expression of the DnaA protein is autogeneously regulated by binding to its own dnaA box between the two Offprint requeststo : A. Quifiones 1984 ; Armengod and Lambies 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851700" target="_blank">2851700</a>]<br></li>
<li>Interaction of DnaA protein with acidic phospholipids results in a decrease in the affinity of DnaA protein for adenine nucleotide , and thus may contrib-ute toward regulation of the nucleotide binding of the protein ( 4-6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>A 33-kDa protein that binds specifically to the right border oforiC , next to the rightmost of the binding sitesof the initiator protein , DnaA , was identified and purified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>Whether the affinity of the DnaA protein to its binding site has changed in the seqA2 mutant is difficult to determine , but no dramatic effect is seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>In such reactions , His10-DnaA protein Since His10-DnaA retained its origin binding ca ¬ FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Regarding DnaA , a second binding site for this protein , located in the guaB coding region , is also required for DnaAmediated repression of P ( 73 , 74 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>CRP , CytR and DnaA protein binding region is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>A much less intense band of 53 kDa , detected in samples from both uninduced and induced cultures ( Fig. 2B , lanes 1 and Expression was induced with IPTG , and during a ( Fig. 1 ) permitted synthesis of DnaA protein with an amino-terminal polyhistidine tag ( termed His10-DnaA ) , which facilitated its easy isolation by immobilized met-al-ion afﬁnity chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>This applies also to bacteriophages and their hosts , including one of the most intensively studied models in molecular biology , phage l. Examples of the processes described above are the use of E. coli LamB protein , normally allowing uptake of maltose and maltodextrins , for phage adsorption on the cell surface ( Vinga et al. , 2006 ) , employment of heat-shock proteins in l DNA replication and morphogenesis ( Polissi et al. , 1995 ; Taylor & Węgrzyn , 1995 ) , and involvement of the replication initiator protein DnaA in the regulation of transcription from the l pR promoter ( Szalewska-Pa asz et al. , 1998 ; Glinkowska et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>The DnaA protein is clearly of major importance in the control of initiation in E. coli ( Masters , 1989 ) : it binds co-operatively to several sites within oriC , where it is involved in an early stage of initiation , the successive opening of the 13mer sequences in the origin ( Bramhill & Kornberg , 1988 ) ; it also assists the entry of the DnaB-DnaC complex , which extends this opening of the duplex ( Bramhill & Kornberg , 1988 ) ; mutations in dnaA often give rise to an initiation defective phenotype ( Kohiyama et al. , 1966 ; von Meyenburg & Hansen , 1987 ) ; considerable asynch-rony has been observed in cytofluorimetric analyses of numerous alleles of dnaA ( Skarstad et al. , 1988 ) ; the overproduction of DnaA protein can lead to an increase in origin concentrations and minichromosome numbers ( Atlung et al. , 1987 ; Xu & Bremer , 1988 ; Skarstad et al. , 1989 ; Pierucci et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>DnaA protein was isolated by immunoprecipitation from cleared lysates and the nucle-otides bound to the recovered DnaA were analysed by polyethyleneimine-cellulose thin-layer chromatography , as described ( Katayama et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>We show that three conditions must be met to observe switch off in vitro : a hemimethylated oriC plasmid template , the addition of a previously defined outer membrane fraction with specific binding affinity for hemimethylated oriC DNA ( Ogden et al. , 1988 ) and an oriC region free of bound DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>Since the two newly formed origins are hemimethylated and membrane bound ( Ogden et al. , 1988 ) and therefore inaccessible to the DnaA protein ( Landoulsi et al. , 1990 ) , the ratio of free DnaA protein to the accessible origins will be momentarily increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>Since the newly replicated origins are transiently hemi-methylated and membrane-bound ( Ogden et al. , 1988 ) , they are transiently inaccessible to the DnaA protein ( Landoulsi et al. , 1990 ; Figure 6C boxed region ) , and the ratio of free DnaA protein to accessible origins is momentarily increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>DnaA protein 's loss of initiation activity is apparently specific for replication from oriC , since RIDA-treated DnaA protein is still able to assemble and function in an ABC primosome in which DnaA protein bound to a hairpin structure interacts with DnaB helicase to initiate replication of ssDNA ( Masai et al. , 1990 ; Marszalek and Kaguni , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>When the cell extract was fractionated by either anion exchange ( DE52 ) chromatography or gel filtration , the ability to inactivate DnaA protein for oriC replication and an activity that promotes the hydrolysis of DnaA-bound ATP coeluted with similar increases in specific activities ( Kurokawa et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>In order to examine the role of the IciA protein , which binds to an AT-rich sequence and counteracts the e $ ect of DnaA on oriC DNA replication ( Hwang and Kornberg 1990b ) and on dnaA gene expression ( Lee et al. 1996 , 1997b ) , in nrd expression , a gel-shift assay was performed with puri ® ed DnaA and IciA proteins ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>In this study , we have investigated whether the SeqA protein has a negative influence on the timing of initiation of replication under conditions in which the DnaA activity is reduced and thus already limiting the initiation frequency ( Boye et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Apart from its main function in the initiation of DNA replication from oriC , the DnaA protein is involved in the control of the replication of several plasmids ( Helinski et al. 1996 ) , and in the regulation of transcription ( Messer and Weigel 1997 ; Szalewska-Paøasz et al. 1998b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>Replication under these conditions was shown to be dependent on DnaA protein , specific for oriC plasmids and initiation began at oriC ( Hughes et al. 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2546039" target="_blank">2546039</a>]<br></li>
<li>Although the original replication assay described by Fuller et al. ( 1981 ) was almost completely inhibited by rifampicin , more recent studies using purified replication proteins have shown that efficient initiation at oriC can occur in the absence of RNA polymerase under conditions that are optimal for the unwinding of oriC DNA by DnaA protein ( Ogawa et al. , 1985 ; Baker and Kornberg , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>Based on the observation that DnaA boxes R1 , R2 , and R4 , but not R3 , are boundby DnaA protein throughout rnost of the cell cycle , it is suggested that further aggregation of DnaA molecules to cover the DnaA 1138 R. Kitagawa , H. Mitsuki , T. Okazaki and T. Ogawa box R3 may be necessary to effect initiation ( Samitt et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Kim , P.D. , Banack , T. , Lerman , D.M. , Tracy , J.C. , Camara , J.E. , Crooke , E. , et al. ( 2003 ) Identiﬁcation of a novel membrane-associated gene product that suppresses toxicity of a TrfA peptide from plasmid RK2 and its relationship to the DnaA host initiation protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Recent models for replication control have favored triggering of replication by accumulation of positive factor ( s ) , usually DnaA protein ( e.g. , Hansen et al. , 1991 ; Atlung and Hansen , 1993 ; Mahaffy and Zyskind , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>The factor controlling the frequency of initiation from oriC seems to be the availability of active DnaA protein , since a good correlation between the dnaA gene expression and the time of initiation in the cell cycle has been established ( L0bner-Olesen et al. , 1989 ; Atlung and Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>Loss of SeqA protein function therefore resembles the situation of DnaA protein-overproduction , which also leads to an elevated number of origins per cell ( Atlung et al. , 1987 ; L0bner-Olesen et al. , 1989 ; Atlung and Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>When DnaA protein is overproduced , initiation occurs earlier in the cell cycle ( Atlung et al. 1987 ; Løbner-Olesen et al. 1989 ; Skarstad et al. 1989 ) , and the C period is extended .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>In this model accumulation of a critical amount of DnaA protein leads to an initiation event ( Atlung et al. , 1987 ; Pierucci et al. , 1987 ; L0bner-Olesen et al. , 1989 ; Skarstad et al. , 1989 ; Atlung and Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>This situation is similar to the forced initiation observed when DnaA protein synthesis was suddenly induced ( Atlung et al. , 1987 ; Pierucci et al. , 1987 ; L0bner-Olesen et al. , 1989 ; Skarstad et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>Here , we first compared cells in which the availability of the DnaA initiator protein was limited by the presence of extra datA DnaA binding sites ( Morigen et al. 2003 ) , to wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>The first and most crucial stage appears to be DnaA protein binding to the 9 bp DnaA recognition sites ( Fuller et al. , 1984 ) and the simultaneous formation of a initial complex where the DNA is wrapped around the DnaA protein core ( Funnell et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>The level of free DnaA protein results from a balance between de novo DnaA protein synthesis and accumulation of chromosomal DnaA binding sites ( DnaA boxes ) that serve as negative elements for replication ( Kitagawa et al. , 1998 ; Christensen et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Rob was shown to-bind next to the rightmost binding sites for the replication initiator protein , DnaA ( Skarstad et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476863" target="_blank">7476863</a>]<br></li>
<li>DNA replication is initiated after binding of the initiator protein , DnaA , to five sites , DnaA boxes , in oriC ( Fuller et al. , 1984 ; Matsui et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>These upstream interactions induce DNA wrapping around the enzyme ( Cellai et al. , 2007 ) and DNA bending by DnaA may further increase the stability of the nucleo-protein complex by decreasing the energy required by RNAP to wrap the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Alternatively , passage of the origin through the inner cell membrane could result in the dissociation of the DnaA protein complex by hydrophobic interactions between DnaA and inner membrane phospholipids such as cardiolipin ( Sekimizu et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>This difference could reflect negative transcriptional regulation of seqA expression by DnaA protein ( Lu et al. , 1994 ) or destabilization of SeqA protein by interaction with DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>The P5 promoter is totally dependent on the cAMP receptor protein ( CRP ) , whereas the P3 and P4 promoters have been reported to be negatively controlled by the DnaA protein ( Wang and Kaguni , 1989b ; Nagai et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>The activation of DnaA protein has also been achieved with cAMP , which has a central role in the cell 's response to the nutritional state of the medium -LCB- Hughes et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>The mioC promoter is negatively regulated by DnaA protein and depends on dam methylation for maximal activity ( Lother et al. 1985 ; Rokeach and Zyskind 1986 ; Schauzu et al. 1987 ; Lobner-Olesen et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>Binding of DnaA protein results in repression of P , , ioc ( Lother et al. 1985 ; Rokeach et al. 1986 ; Stuitje et al. 1986 ; Lobner-Olesen et al. 1987 ; Schauzu et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>This is in accordance with the notion that DnaA protein is synthesized in proportion to bulk protein ( Hansen et al. , 1991b ) and is not growth rate regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Two transcripts start from the two dnaA promoters which are negatively controlled by the DnaA protein ( Atlund et al. 1985 ; Braun et al. 1985 ; Kiicherer et al. 1986 ) and these seem to be extended through dnaN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851700" target="_blank">2851700</a>]<br></li>
<li>Up to 30 monomers of DnaA protein bind to a dnaA box ( Fuller et al. 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>Moreover , the change in autoregulation as a function of temperature can help to explain the cold-sensitive phenotype of certain DnaA and Hda mutants [ 52 -- 54 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>pACYC177 bearing DnaA box-deleted DARS derivatives did not stimulate initiation ( pOA76 [ DARS1TDDnaA boxes ] and pOA71 [ DARS2TDDnaA boxes ] in Fig. 4C , D ) , consistent with the in vitro data ( Figs. 1A , 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>As a result , only one origin will be initiated and after the cell has grown and increased the DnaA to origin ratio to the right level , another initiation event can take place .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>Excessive interaction between DnaA and DnaB might inhibit the actions of DnaA in the initiation processes , which might result in repression of overinitiation ( Fig. 6A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>To assess each pathway 's relative effectiveness , we used genetically inactivated strains , and quantified initiation frequency after elevating the level of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>In these strains , the overproduction of DnaA led to comparable increased levels of initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>DnaA is involved in the expression of many genes [ 28 ] and , in our hypothesis , in the build-up to initiation , DnaA accumulates due to the disappearance of hyperstructures that contained it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>This together with the observed increase in DnaA content would add to the over-initiation phenotype of seqA2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>This results in a drop in the diversity of hyperstructures , some of which release DnaA as they dissociate , and a DnaA-initiation hyperstructure forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>We propose that this is due to the absence of an initiation cascade caused by liberated DnaA molecules , and that this cascade normally synchronizes initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>Therefore , following b-subunit loading , which serves as a hallmark of replisome formation , DnaA is inactivated for re-initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>Replication initiation is tightly controlled by the AAA + ATPase , DnaA [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079575" target="_blank">19079575</a>]<br></li>
<li>Thus , generation of ATP-DnaA is prerequisite for replicational initiation and its regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Model for the DnaA cycle in regulation of the replicational initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Initiation requires the polymerisation of DnaA [ 23 ] and a membrane with a particular phospholipid composition [ 24 ] and domain structure to activate DnaA [ 21,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>It has been shown that the level of DnaA is decreased twofold in dam mutants ( Braun and Wright , 1986 ; Kucherer etai , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>As shown ( Fig. 4A ) , addition of increasing amounts of the nontagged DnaA and His10-DnaA proteins to the extract exhibited virtually identical enzymatic activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Aggregation and activation of DnaA are prevented by phospholipase and DnaK indicating the importance of phospholipids in assembly and activity of DnaA [ 354 , 365 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Induction of DnaA from pROA200 by the addition of IPTG resulted in repression of the β-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Concentrations of DnaA ( nM ) in reaction mixtures for lanes 1 -- 7 were 0 , 19 , 38 , 76 , 114 , 152 and 190 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>This includes DnaA , FarR , GlpR , MalT , Lrp , MetR , NarL , TyrR , TorR and TreR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>RIDA-inactivated DnaA is also reactivated by DARSs in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>We thus investigated if DARS1 could reactivate DnaA that was inactivated by RIDA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>His10-DnaA was eluted from the matrix by increasing the imidazole concentration to 1 M. Immediate dialysis was performed to remove the imidazole .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>These factors appear to respond to nutritional changes ( FIS ) ( 30 ) or the DnaA concentration itself ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Binding sites for regulatory proteins CRP , CytR and DnaA are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Ong of DnaA ; 6 , 5ng of DnaA ; 7 , l o n g of DnaA ; 8 , 25 ng of DnaA ; 9 , 50 ng of DnaA ; 10 , 100ng of DnaA ; 11,200 ng of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>One player in the coordinate control of macromolecular synthesis is DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Various factors control the ability of DnaA to-bind and unwind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Schaefer , C. , and Messer , W. ( 1989 ) Directionality of DnaA proteinlDNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Speciﬁc DnaA -- DnaA and DnaA -- DnaG primase interactions are also likely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Samples were taken at 0 , 0.5 , 1 , 2 , 3.5 , 5 and 24 h for determination of DnaA concentrations as described below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>is a target for DnaA regulation of IMP and AMP biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969519" target="_blank">8969519</a>]<br></li>
<li>Thus , the missing factor is unlikely to be PurR or DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>No cross-reacting band the size of DnaA was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>As expected , pMAPCdSG10 did not bind DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>His10-DnaA pro- oaked in Buffer DB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>DARS can catalytically regenerate DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Footprinting experiments demonstrated that DnaA binds only to the proximal DnaA box near the polA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>As noted above , dATP also binds DnaA , and the nrdAB operon contains two tandem DnaA boxes proximal to NrdR box 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>In addition , we show that the binding of DnaA on the promoter can either activate or repress transcription as a function of its concentration and its nucleotide-bound state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>In some cases in which the DnaA boxes are located in , or close to , the promoter sequence , transcription is inhibited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Binding of DnaA to the promoter region mostly inhibits but in some cases stimulates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>On the other hand , DnaA is reported to activate the transcription of nrd , although the mechanism of this activation is not yet known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>Repression of transcription has been demonstrated for dnaA itself [ 4,15 , 16 , 271 although at normal physiological concentrations DnaA may activate transcription initiation and repress at the dnuA promoter only when overproduced [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Nonetheless , it is possible that multiple mechanisms of cell cycle control , such as the DnaA levels and the regulatory inactivation of DnaA ( RIDA ) , cooperate to control cell cycle in response to ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079575" target="_blank">19079575</a>]<br></li>
<li>If Hda is absolutely essential for RIDA in vivo , there may be another DnaA inactivation system that is repressed by YgfZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The regulation inactivation of DnaA ( RIDA ) is one , by which the DnaA hydrolyses ATP , producing replicatively inactive ADP-DnaA ( Kato and Katayama , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>The third system is the functional regulation of DnaA termed RIDA ( for regulatory inactivation of DnaA ) ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>In contrast , DnaAcos is considerably more active than DnaA + at this temperature and at 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>In wildtype strains , YgfZ may stimulate this regeneration , thereby maintaining a critical level of the active form of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>However , in the ygfZ mutant , the level of the active form of DnaA may be reduced , leading to suppression of the hda mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>To identify other factors involved in the conversion between the active and inactive forms of DnaA , suppressor mutants were isolated in an hdats background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The active form of DnaA is regenerated from the inactive form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The inactive form of DnaA can be converted to the active form in vitro and this conversion is enhanced by phospholipid ( Sekimizu and Kornberg , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>In these complexes , a heat-dependent confor - + mational change of the AAA domain in DnaA occurs , resulting in reduction in ADP affinity , production of apo-DnaA , release of apo-DnaA from the complexes , and further binding of new ADP-DnaA , which thereby allows a catalytic reaction to occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Given that cells contain ; 3 mM ATP , ; 10-fold that of ADP ( Bochner and Ames 1982 ) , release of ADP from ADP-DnaA could be directly followed by ATP binding , which would reactivate DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>When apo-DnaA was included in experiments , 125 mM potassium chloride was included in the DARS1-binding reaction , ADP was excluded in the DARS1-binding reaction and electrophoresis was performed at 4 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Various amounts ( 0 , 60 , 150 , 300 , 600 , and 1000 fmol ) of ADP-DnaA ( ADPform ) or apo-DnaA ( Apo ) were incubated for 15 min on ice with 32 P-labeled FK7-7 ( 30 fmol ) in the presence or absence of 2 mM ADP , followed by EMSA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>ADP-DnaA was prepared by incubation of apo-DnaA with 2 mM ADP for 15 min at 0 ° C , with the exception of DnaA D269N , which was incubated with 2 mM ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>EMSA was performed in the presence of 2 mM ADP to detect the complex containing ADP-DnaA and DARS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>When ADP-DnaA was incubated at 30 ° C in the presence of FK7-7 and 1.5 mM ATP , ATP-DnaA was formed by nucleotide exchange ( Fig. 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>When ADP-bound DnaA D269N was incubated in the presence of DARS1 and 2 mM ATP , ADP was not substantially released ( Fig. 7B ) , indicating that Asp-269 is required for DARS1 function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Each DnaA box within DARS1 as well as the DnaA box cluster was required for ADP-releasing activity ( cf. FK7-7 and FK7-21 , FK7-8 , FK7-9 , FK7-13 in Fig. 1A -- C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This is consistent with the hypothesis that apo-DnaA , a product of ADP release , efficiently dissociates from DARS complexes , allowing further association of ADP-DnaA and enhancing cyclic reactivation of DnaA ( Supplemental Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA R281A and DnaA R285A were able to undergo DARS1-mediated ADP release ( Fig. 7B ) , but DnaA R285A showed a moderate decrease in ADP release .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA sensor 1 Asp-269 is required for DARS1-dependent ADP release and formation of highly ordered complexes on DARS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA molecules form specific homomultimers on DARS1 , which causes specific interactions among the protomers , reducing their affinity for ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Our analyses suggest that specific DnaA homomultimers form on DARS1 , and that specific interactions occur between their AAA domains , thereby reducing + their affinity for ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>In vivo , DnaA predominantly takes on the ADP form in a b subunit -- dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>In the first stage , the level of ATP-DnaA activity was reduced to that of ADP-DnaA in a RIDA-dependent manner ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Taken together , these results support the idea that the hda-185 mutant causes overinitiation due to an elevated level of DnaA activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Alternatively , the increase in the ADP-DnaA level attributed to the ygfZ mutation could be repressed in the presence of the hda gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>As shown in Fig. 5 , although the level of the ATP-bound form of DnaA increased to 64 % of the total DnaA in the hda mutant , the level in the hda ygfZ double mutant was reduced to 26 % of the total , comparable to that of the control strain ( 27 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The accumulation of the active form of DnaA , ATP-DnaA , in the hda mutant is suppressed by the disruption of the ygfZ gene , indicating that YgfZ is involved in regulating the level of ATP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The cellular ATP-DnaA level was increased in an hda-185-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>In dnaN59 cells bearing pOA21 ( pACYC177-DARS1 ) , the ATP-DnaA level increased at 42 ° C in the presence of chloramphenicol ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>When DARS was not oversupplied in dnaN ( Ts ) cells , the ATP-DnaA level was not elevated ( Fig. 4 ) , suggesting the presence of a tight regulatory system for DARS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>These results indicate that hda-185 is dysfunctional at 25 ° C and suggest that overinitiation is caused in the presence of the wild-type diaA by the elevated level of ATP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>This might be caused by the elevated ATP-DnaA level and transcriptional repression of the chromosomal nrdAB genes in the hda-185 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>The sum of positive and negative autoregulation at dnaAP2 , therefore , can result in the synthesis of the amount of DnaA -- ATP required for the adaptation of the cell to changes in growth or in response to stress to take place within a smaller fraction of the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The presence of multiple pathways to provide ATP-DnaA might be advantageous in cell cycle controls responding to various growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Coordinated regulation of RIDA and DARS activities could cause the level of ATP-DnaA to oscillate during the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Before initiation both dnaA gene expression and the recycling of DnaA -- ADP into DnaA -- ATP [ 13 ] contribute to the increase in the amount of DnaA -- ATP in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>These results are consistent with an increase in the amount of active DnaA -- ATP throughout the cell cycle as the temperature is lowered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The loss of negative autoregulation of dnaAP1 as the temperature is decreased leads to a higher increase in the proportion of DnaA -- ATP to total DnaA in these mutants compared with the wildtype , and thus in overinitiation of DNA replication and stalled replication forks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The amount of DnaA -- ATP available for DNA binding thus oscillates during the cell cycle to peak at the time of initiation , and the rate at which the amount of DnaA -- ATP increases during the cell cycle can affect the timing of initiation [ 1,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The observation of the effects of both positive and negative transcription regulation by DnaA in vivo is consistent with the oscillation in the amount of DnaA -- ATP available as a function of the DNA replication cycle ( reviewed in [ 36 ] ) that could result in activation and repression of the reporter gene constructs to take place at different times during the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The timing of expression of nrdAB with replication initiation is only partly affected by DnaA activation and independent of DnaA repression In order to determine the effects of DnaA activation and repression on the cell-cycle-dependent expression of nrdAB we carried out ﬂow cytometry measurements of the amount of GFP per cell as a function of DNA content ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>In their model , ATP-DnaA acts as a repressor of nrdAB transcription ; after initiation of DNA replication , ATP DnaA is converted to ADP DnaA , resulting in increased nrdAB expression and thereby coordinating DNA replication and RNR synthesis during the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Keyamura , K. , N. Fujikawa , T. Ishida , S. Ozaki , M. Su'etsugu et al. , 2007 The interaction of DiaA and DnaA regulates the replication cycle in E. coli by directly promoting ATP DnaA-speciﬁc initiation complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_674">
The interaction of DiaA and DnaA regulates the replication cycle in E. coli by directly promoting ATP DnaA-speciﬁc initiation complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></label>
<input id="act_similarity_674", type="checkbox">
<div class="ACT_similar">
<li>The interaction of DiaA and DnaA regulates the replication cycle in E. coli by directly promoting ATP DnaA-specific initiation complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li></div>
<li>Alternatively , when SeqA is pre-sent , only the ATP form of DnaA works in replication whereas in the absence of SeqA any form of DhaA will function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>Indeed , in vivo experiments using replication cycle-synchronized cells demonstrate that the dnaA promoter activity is tightly repressed in the postinitiation period in which the ATP-DnaA level is decreased and the ADP-DnaA level is elevated by RIDA ( 6 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>When the level of ATP-DnaA increases prior to initiation in cells lacking either DARS1 or DARS2 , the remaining DARS may activate DnaA , which would slightly increase the amount of time needed for ATP-DnaA levels to reach the threshold needed for initiating replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This uncoupling of effects suggests that whereas the level of ATP-DnaA controls initiation , increases in overall DnaA concentration ( combined ADP-DnaA and ATP-DnaA ) have negative effects on replication fork elongation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>Thus , cellular ATP-DnaA is converted to ADP-DnaA in a manner that is coordinated with replication and represses untimely extra initiation events .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>We further demonstrate that DARS1 and DARS2 can enhance the level of cellular ATP-DnaA and the initiation of chromosomal replication in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Oversupply of DARSs results in increase in the ATP-DnaA level and enhancement of replication initiation , which can inhibit cell growth in an oriC-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Deletion of DARSs results in decrease in the ATP-DnaA level and inhibition of replication initiation , which can cause synthetic lethality with a temperature-sensitive mutant dnaA and suppression of overinitiation by the lack of seqA or datA , negative regulators for initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This pathway was ﬁrst suggested by an in vitro experiment showing that the conversion of ADP-DnaA to ATP-DnaA is facilitated by acidic phospholipids in a ﬂuid bilayer in the presence of oriC DNA ( Sekimizu and Kornberg , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Acidic phospholipids , such as phosphatidylglycerol and cardiolipin , were reported previously to reactivate ADP-DnaA in vitro by nucleotide exchange in the presence of ATP and oriC ( Sekimizu and Kornberg 1988 ; Crooke et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Hda in a complex with the β clamp stimulates the hydrolysis of ATP bound to DnaA in a process named the `` regulatory inactivation of DnaA '' ( reviewed in Katayama , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>The separate isolation of the hda and dnaN plasmids suggests that the respective proteins act independently , which contrasts with biochemical studies showing that both Hda and the β clamp are required to stimulate the hydrolysis of ATP bound to DnaA ( Kato and Katayama , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>The second possibility is that Hda and the β clamp act independently , which contrasts with findings from biochemical studies that Hda and the β clamp are both required to stimulate the hydrolysis of ATP bound to DnaA ( Kato and Katayama , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>At the post-initiation stage , DnaA-ATP is hydrolyzed in a DNA-loaded b sliding clamp-dependent and ADP-Hda protein-dependent manner , yielding inactive ADP-DnaA ( Katayama et al. 1998 ; Kato and Katayama 2001 ; Su'etsugu et al. 2005 , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Hda , a protein related in amino acid sequence to DnaA itself ( Kato and Katayama , 2001 ) , in conjunction with the replication processivity clamp β ( Katayama et al. , 1998 ) , binds and inactivates DnaA in vitro by promoting ATP hydrolysis , a mechanism termed ` RIDA ' , for regulatory inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>A protein homologous to DnaA ( Hda , for homolog of DnaA ) binds the processivity clamp , b , and DnaA , promoting hydrolysis of ATP ( Kato and Katayama 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Following initiation of DNA replication two main processes contribute to a decrease in DnaA activity and reduce the amount of protein available for replication initiation in the cell : ( i ) the increase in the rate of DnaA-bound ATP hydrolysis mediated by the interaction with the Hda and β-clamp proteins [ RIDA ( regulatory inactivation of DnaA ) process ] in a manner dependent on the activity of the replication forks and ( ii ) the titration of the free protein on sites of different afﬁnities found all along the genome , such as the datA locus [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Su'etsugu , M. , Kawakami , H. , Kurokawa , K. , Kubota , T. , Takata , M. , and Katayama , T. ( 2001 ) DNA replication-cou-pled inactivation of DnaA protein in vitro : a role for DnaA arginine-334 of the AAA + Box VIII motif in ATP hydrolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Kawakami , H. , Su'etsugu , M. , and Katayama , T. ( 2006 ) An isolated Had -- clamp complex is functional in the regulatory inactivation of DnaA and DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>A major obstacle has been the difﬁculty in obtaining abundant quantities of pure , active , nucleotide-free DnaA protein ; overproduction and puriﬁcation of DnaA protein is troublesome in that the protein tends to accumulate in inclusion bodies , and isolation of large amounts of relatively concentrated DnaA without the use of ATP to increase the protein 's solubility has proven trouble-Recent studies implicate important associations be-A variety of assays for monitoring the activities of While much is known about the critical functions some ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>His10-DnaA is replicatively active in vivo and in ith tightly bound ATP responded the same as non - itro and displays properties very similar , if not indisagged ATP-DnaA when treated with acidic ( Fig. 5A ) inguishable , to those of nontagged DnaA for binding to nd neutral ( Fig. 5B ) phospholipid vesicles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>CHEMICAL AND BIOPHYSICAL RESEARCH COMMUMCATIONS 232 , 130 - 135 ( 1997 ) NO RC976244 3 ' ' 1 ' I ' p -2 Conformational Transition of DnaA Protein by ATP : Structural Analysis of DnaA Protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>KDand the stoichiometric ratio for ATP of the DnaA sample prepared by the new method were 96 nM , and 0.42 ATP molecules per DnaA molecule , respectively ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>DnaA -- ATP at the following concentrations , 14 , 27 , 55 , 82 , 137 and 205 nM , was used for each titration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>DnaA appears to be active as an ATP -- DnaA complex ( Katayama , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>Therefore , affinities for ATP of the DnaA samples purified by the new and previous methods were much the same .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Released apo-DnaA binds ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Newly synthesized DnaA would provide ATP-DnaA , which assists the increase of the ATP-DnaA level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>We inhibited de novo synthesis of DnaA using chloramphenicol , as newly translated DnaA molecules would predominantly bind to ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>De novo synthesized DnaA would predominantly bind ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Thus , there seems to be three means of generating the ADP form of DnaA protein from ATP-DnaA : ( i ) the weak , DNA-dependent intrinsic ATPase activity of DnaA protein ( Sekimizu et al. , 1987 ) ; ( ii ) the reaction that depends on b subunit in the configuration of a sliding clamp and the IdaB protein ( a protein postulated to be present in the IdaB fraction and is necessary for DnaA inactivation ) ( Figure 6A ) ; and ( iii ) the enhanced conversion that occurs during concomitant DNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The initiator titration model ( Hansen et al. , 1991 ) seems to be able to explain synchronous initiation of replication as resulting from variations in the concentration of free DnaA protein in the ATP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12067334" target="_blank">12067334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12068816" target="_blank">12068816</a>]<br></li>
<li>The DnaA protein is active only in its ATP-bound for al. ; í987j-J The ADP-bound inactive form full activity by phospholipids in DNA ( Sekimizu and Kornberg , 1988 ; Castuma et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>In vitro , ADP bound to DnaA protein can exchange with free ATP through interaction of DnaA protein with acidic phospholipids ( Sekimizu and Kornberg , 1988 ; Crooke et al. , 1992 ; Garner and Crooke , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>A simple specific mechanism for SeqA action consistent with existing biochemical observations would be that SeqA blocks the allosteric transition of DnaA from an inactive form ( potentially the ADP form ) to an active ( ATP ) form The ATP form of DnaA is required for open complex formation in vitro ( Sekimizu et al. , 1987 ) , and purified DnaAcos protein is defective in ADP binding ( T. Katayama and A. Kornberg , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>The DnaA protein may be in the active ATP form or the inactive ADP form ( Sekimizu et al. , 1987 ) and , thus , regulation of the conversion between the ATP and ADP forms may also contribute to regulation of the initiation event ( Katayama et al. , 1998 ) The SeqA protein appears to be a negative regulator of initiation of replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Assessing Hydrolysis of ATP Bound to DnaA Protein by Cell-Free Extracts or Purified Proteins DnaA protein typically was incubated with 1.5 mM [ a-32P ] - ATP ( 105 cpm/pmol ) on ice according to the method by Sekimizu et al. ( 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The dnaN59 gene product degrades rapidly at restrictive temperatures in vivo ( Burgers et al. , 1981 ) ( unpublished results ) , and concurrently the content of ATP-bound DnaA protein increases rapidly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The first of these , DnaA protein 's own ATPase activity , one which does not require a specific DNA sequence , millimolar levels of ATP , or other proteins ( Sekimizu et al. , 1987 ) , may be integral to the other two mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The regeneration of ATP-DnaA in the presence of overexpressed DnaA protein also has been shown directly ( Kurokawa et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The activation of DnaA protein by its binding of ATP enables it to participate in initiation ( Sekimizu et al. , 1987 ; Hwang & Kaguni , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>Several factors have been shown to influence the ability of DnaA protein to open the strands ( 3-6 ) : ( i ) binding of ATP or ADP to DnaA ; ( ii ) the level of negative superhelicity ; ( iii ) the temperature ; ( iv ) the presence of structure-modifying proteins ( e.g. HU and IHF ) ; ( v ) thepresence of theinhibitory IciA protein ; and ( vi ) transcriptional activationI l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>COntentof the p-structure of DnaA protein seems to increase interactiOn with ATP * an event perhaps caused by a structural transition of the P-loop StruCture ( the ATP-binding motif A ) t h a t is present in DnaA protein ( 3 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Analysis of circular dichroism revealed that DnaA protein is rich in a-helices , and that ATP-bindingleads to a significant transition of protein conformation in that the content of a-helices is decreased.This is the first evidence indicating that ATP-binding profoundly affects conformation of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Measurements of intrinsic fluorescence indicated that ATP stabilizes the conformation of the protein against heat , while circular dichroism measurement revealed an alteration of the secondary structure of DnaA protein by nucleotide binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>DARS1 or DARS2 multicopies increase the level of ATP-DnaA independently of de novo protein synthesis To assess in vivo function , we tested whether an increase in the copy number of DARS1 or DARS2 would stimulate the regeneration of ATP-DnaA from ADP-DnaA in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>b Subunit Is Required for Inactivation of DnaA Protein In Vivo To determine whether the b subunit is involved in the inactivation of DnaA protein in vivo , the levels of ADP-DnaA versus ATP-DnaA protein in lysates prepared from cells with differing levels of functional b subunit were measured ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>DnaA Inactivation In Vivo Immunoprecipitation of intracellular DnaA revealed that ca. 80 % of the protein was in the inactive , ADP-bound form , indicating that the inactivation of DnaA protein in vivo is efficient ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>[ 30 ] T. Mizushima , Y. Ishikawa , E. Obana , M. Hase , T. Kubota , T. Katayama , Inﬂuence of cluster formation of acidic phospholipids on decrease in the afﬁnity for ATP of DnaA protein , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>Both ATP-DnaA and ADP-DnaA bind to 9mer DnaA boxes , TTA/TTNCACA , but only ATP-DnaA protein binds in addition to a 6mer site , AGATCT [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>Nucleotide Binding of Purified DnaA Protein DnaA protein binds ATP and ADP with a high affinity ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>DnaA protein plays a central role in the initiation of DnaA protein binds ATP and ADP with high afﬁnity ( KD of 30 and 100 nM , respectively ) ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>DnaA protein tightly binds ATP and ADP with a KD of 30 and 100 nM , respectively ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Nontagged DnaA or His10-DnaA protein was incubated with 32P ] ATP or [ 3H ] ADP ( 0.01 to 1.2 mM ) for 15 min at 0 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_730">
A and B. DnaA protein were incubated in the ( A ) absence or ( B ) presence of 10 pM ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></label>
<input id="act_similarity_730", type="checkbox">
<div class="ACT_similar">
<li>C. D.and E , DnaA protein ( 0.57 pM ) were incubated in the absence ( C ) or presence ( D ) of 6.7 pM ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li></div>
<li>Values of 100 % correspond to e ATP retained by each form of DnaA protein in the absence of phospholipids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Affinity of purified DnaA protein for ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Additionally , we find that b subunit in vivo functions to control the cellular distribution of DnaA protein between the ATP and ADP forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_735">
Speck , C. , Weigel , C. , and Messer , W. ( 1999 ) ATP - and ADP-DnaA protein , a molecular switch in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></label>
<input id="act_similarity_735", type="checkbox">
<div class="ACT_similar">
<li>ATP - and ADP-DnaA protein , a molecular switch in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>The naA and His10-DnaA proteins also responded equally fﬁnities His10-DnaA and nontagged DnaA protein had n releasing bound nucleotides when exposed to vesi-or ADP and ATP are indistinguishable ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>The process of denaturation of DnaA protein , which was blocked by ATP , was monitored by intrinsic fluorescence and circular dichroism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>The decrease in helical content was small but significant , thereby suggesting that the interaction of ATP slightly decreased the population of the helical structure of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>These results indicate that b subunit is necessary for inactivating DnaA protein by generating the ADP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>However , when dNTPs were included so that DNA synthesis was possible , greater than 80 % of the DnaA protein was converted into the replicatively inert ADP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Experiments with puriﬁed wild-type DnaA protein showed that this protein was not degraded by the ClpXP protease [ 29 ] , irrespective of whether the protein was in the active ATP - , or inactive ADP-bound form or whether supercoiled oriC plasmid and SeqA protein were present ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>Acidic phospholipids in a ﬂuid bilayer promote the release of bound ADP and ATP from DnaA protein ( 6 -- 8 ) , and in the presence of oriC DNA , facilitate an exchange of ATP for ADP , thus rejuvenating ADP-DnaA protein to its active ATP form ( 6,9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Since newly synthesized DnaA protein will likely be in the ATP form , given that ATP is present at much higher cellular concentrations than ADP , these results suggest that the ATP-bound form of DnaA protein is rapidly converted to ADP-DnaA in a manner dependent on the dnaN gene product , the b subunit of pol III holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Since the cellular concentration of ATP ( ca. 3 mM ) is much higher than that of ADP ( ca. 250 mM ) ( Bochner and Ames , 1982 ) , newly synthesized DnaA protein will bind ATP predominantly and be in the active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Thus , RIDA may hydrolyze ATP bound to DnaA protein , or enhance the weak intrinsic ATPase of DnaA protein , to yield replicatively inactive ADP-DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Control of the cellular amount of the ATP form of DnaA protein is possible by modulating the b-clamp function and regulating regeneration of the ADP-bound form of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>When DNA was omitted , only 8 % of the ATP-DnaA was converted into the ADP form ( line 7 ) , confirming the necessity for the interaction of DnaA protein or b subunit ( or both ) with DNA for DnaA inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The DNA-loaded form of the clamp subunit of the polymerase binds the Hda protein , which promotes ATP-DnaA hydrolysis , yielding inactive ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>The ATP form of DnaA protein seems to be slowly regenerated from the ADP form in vivo ( unpublished results ) ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Discussion Regulatory Inactivation of DnaA Protein Involves the b Clamp of Pol III : Interaction of the Initiator and Replicase The replicatively active ATP-bound form of DnaA protein is converted to inactive ADP-DnaA by the b subunit of pol III and IdaB fraction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Pol III b Subunit Is Necessary for DnaA Inactivation ( A ) Conversion of ATP-DnaA protein to the ADP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Newly synthesized DnaA protein adopts the active ATP form but then is rapidly changed to the inactive , ADP form in a b subunit -- dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The defective DnaA46 protein might also affect the eclipse period by bypassing regulatory inhibition of DnaA ( RIDA ) ; the regulation of the hydrolysis of the ATP bound to DnaA by linking it to the b-subunit `` sliding clamp '' of the pol III holoenzyme ( DnaN ) .24,25 The consequences of different dnaA mutations need to be clariﬁed for any meaningful speculation along these lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></li>
<li>The non -- DNA-bound form of b subunit does not support inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Pol III * Promotes the b Subunit for DnaA Inactivation Purified pol III * and fraction IV-500 were assayed for their capacity to stimulate hydrolysis ATP bound to DnaA protein in the presence of M13E10 RFI DNA ( 200 ng ) ( see Experimental procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The extracts were added to DnaA protein inactivation reaction mixtures ( 10 l ) containing 2.0 pmol of m unlabled DnaA protein ( B ) or [ a-32P ] ATP-bound DnaA protein ( C ) , and incubated at 308C for 25 min in the presence of M13mp19 RFI DNA ( 400 ng ) ( see Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>However , when either ATP , g complex , or b subunit was omitted prior to the gel filtration step , and then the missing component was added back to the void-volume fraction , accelerated hydrolysis of DnaA-bound ATP was not observed ; only 30 % of DnaA protein was found in the ADP form ( lines 2 -- 4 ) , the level of ADP-DnaA protein produced by the slow DNA-dependent , intrinsic ATPase of DnaA protein ( lines 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>When bound to DNA , this complex stimulates the hydrolysis of ATP bound to DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>DnaA binds to five 9mer DnaA boxes in oriC : strongly to three ( R1 , R2 , and R4 ) boxes that adhere to the consensus sequence TTWTNCACA , and more weakly to two ( M and R3 ) boxes that have one or two mismatches to the consensus sequence .1 -- 3 Also other low-affinity sites ( three I-sites and several 6mer ATP-DnaA boxes ) have been identified .4,5 The latter two types of low-affinity sites are exclusively bound by the active ATP-bound form of DnaA , whereas 9mer R-type DnaA boxes are bound by both ATP - and ADP-form DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Two sites , named DARS1 and DARS2 , for DnaA-reactivating sequence ( DARS ) , contain DnaA box sequences like datA , but the DARS sites stimulate the dissociation of ADP bound to DnaA to permit DnaA to-bind to ATP , which is more abundant than ADP in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Here we demonstrate that both DnaA-ADP and DnaA-ATP can bind to the nrdAB promoter at two activating sites , and that the DnaA protein in its ATP-bound form can form oligomers at higher concentrations that result in repression of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA bound at the high-affinity sites ( grey ovals ) , in either its ATP or ADP-bound form , binds to the upstream , high-affinity , sites ( a ) and stabilizes RNAP binding ( white oval ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA bound to the high-affinity sites , in its ATP or ADP-bound form , stabilizes RNAP binding at the nrdAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DARSs probably facilitate a specific inter-DnaA interaction that leads to the release of bound ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Because DnaA complexed to ATP is active in initiation , but DnaA bound to ADP is relatively inactive , the change in the nucleotide-bound state of DnaA by Hda and the β clamp is suggested to control DnaA activity in initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Because DnaA complexed with ATP is active in initiation , whereas DnaA-ADP is feeble , the interaction of the Hda - clamp complex regulates the frequency of initiation by affecting the activity of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>ADP-DnaA complexes are dissociated by these DARSs in vitro , and initiation-competent ATP-DnaA is generated in the presence of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Evidence consistent with a DnaA-initiation hyperstructure In E. coli , the ATP-bound form of DnaA is required for initiation in vitro whilst the ADP-bound form is inactive [ 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>Discussion In this study , we demonstrated that specific genomic sequences , DARS1 and DARS2 , reactivate DnaA for rep-lication initiation by exchanging bound ADP with ATP in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Levels of active ATP-bound DnaA appear to be a primary determinant of initiation capacity ( reviewed in Donachie and Blakely , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Three mechanisms for regulation of replication initiation have been proposed : titration of free DnaA initiator protein by the datA locus , sequestration of newly replicated origins by SeqA protein and regulatory inactivation of DnaA ( RIDA ) , in which active ATP-DnaA is converted to the inactive ADP-bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>In addition to concentration , the activity of DnaA protein should be controlled for regulation of initiation of replication by some mechanism ( s ) which presumably includes conversion between its ATP - and ADP-bound forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Bound ATP is hydrolyzed slowly to ADP , which remains tightly associated and renders DnaA protein inactive for initiation ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Levels of active ATP-bound DnaA protein therefore are key determinants of initiation capacity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Binding of adenine nucleotide is important for functions of DnaA protein ; the ATP , but not the ADP , - bound form of the protein is active for initiation of replication of minichromosomes in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>YgfZ may participate in the repression of hydrolysis of ATP bound to DnaA in RIDA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>One such mechanism , termed regulatory inactivation of DnaA ( RIDA ) , is a process that accelerates the hydrolysis of ATP bound to DnaA , thereby generating replicatively inactive ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>RIDA stimulates the hydrolysis of DnaA-bound ATP , yielding the inactive ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>RIDA mediates the conversion of active ATP-DnaA to inactive ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Regeneration of the ADP form to the ATP form is slower than theconversion of ATP-DnaA into the ADP form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>However , efficient hydrolysis of the bound ATP will rapidly convert DnaA into the inactive ADP form ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>When DnaA is produced it is under the ATP-bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The ygfZ-dependent suppression of the replication defect of the hda mutant could be explained either by the conversion of the accumulated ATP-bound form of DnaA to the ADP-bound form or by the correction of abnormal replication despite the accumulation of the active form of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The ATP-bound form of DnaA appears to be Ihe active form required for in vitro replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>The ADP-bound form of DnaA , generated from the ATP form by the intrinsic ATPase activity , is inactive for oriC replication ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The activity of DnaA is regulated by adenine nucleotides ; the ATP bound-form is active in the initiation of replication while the ADP-bound form is inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>DnaA in the cell is found bound either to ATP or ADP , both forms contributing to the timely formation of an activated origin leading to the recruitment of the DNA replication factors [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>To determine whether suppression of the replication deﬁciency resulted from the inactivation of DnaA , the ratios of the active ( ATP-bound ) and inactive ( ADP-bound ) forms were examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>DnaA inactivation occurs through accelerated hydrolysis of ATP bound to DnaA ; the resulting ADP-DnaA fails to initiate replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>To determine whether DnaA protein bound to these sequences in the nrd promoter , a 32P-end-labeled nrd promoter region fragment ( fragment 229f , - 165 to +85 relative to the start of nrd transcription ) was mixed with increasing amounts of DnaA and subjected to Cu-phenanth-roline footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>To determine the effect of the mutations on DnaA binding , the 261-bp BamHI-EcoRI nrd promoter fragment 229f , containing the DnaA box mutations , was mixed with increasing amounts of DnaA and subjected to Cu-phenanthroline footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>These results show that DnaA is not necessary to recruit RNAP to the promoter in vitro ; however , the close proximity of the DnaA-binding sites to the -35 region suggests that in vivo it may play a stabilizing role in the presence of possible transcription repressors , such as NrdR , that can compete with RNAP for promoter binding ( Torrents et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>These results clearly show that DnaA binding upstream of the promoter is required for transcription activation while its oligomerization to the downstream sites results in transcription repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>In this genetic background the M12-14GG mutation still has a signiﬁcant positive effect on the level of transcription , conﬁrming that the loss of repression is due to the decrease in binding affinity for DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA binding to the high-affinity sites is required for activation of nrdAB transcription in vivo , while binding to the downstream low-affinity site represses its transcription In order to determine the effects on nrdAB expression in vivo of DnaA binding to its different sites we measured the rate of change in the cellular level of the GFP protein ( ( dGFP/dt ) / OD ) expressed under control of the nrdAB promoter and its mutants in the low copy pKK plasmid by growing the cells in a 96-well ﬂuorimeter ( Zaslaver et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Mutation of DnaA sites in the dnaAP2 promoter region affects DnaA binding to both high - and low-afﬁnity sites In order to characterize the effect of DnaA binding to its own promoter 's activity we ﬁrst measured the effects of speciﬁc mutations of its sites in vitro and subsequently carried out in vivo experiments to determine whether these effects found a correlation in the level of promoter activity in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Quantitative DNase I footprinting of DnaA binding shows decreased affinity for mutated DnaA sites on the promoter In order to study the effect of DnaA binding on the activity of the promoter in vivo we carried out directed mutagenesis to its different binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The increased level of nrdAB expression in mutants M12-14GG and M12-14AG can be considered to be due to the loss of the low-affinity DnaA-ATP boxDown repressing site , whereas the decreased level of nrdAB expression in the mutants M40-42box2 , M45-47box1 and M49box1 can be considered to be due to the loss of the high-affinity DnaA box1 - and box2-activating sites consistent with the in vitro results in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>At low concentrations , DnaA-ATP binds upstream of the -35 region to two DnaA high-affinity sites , box1 and box2 , and enhances nrdAB expression , as shown by the in vivo GFP expression results in agreement with previous work ( Augustin et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The ﬁnal concentrations of DnaA -- ATP were ◦ 0 , 14 , 27 , 41 , 82 , 137 and 205 nM at 37 C ; and 0 , 14 , 27 , 41 , 55 , 82 , 137 and 205 nM at 30 and 20 ◦ C. ( B ) Change in the dissociation constant ( K ) of DnaA to the protected regions ( box d 1 , box 2 , box 4 and box b ) corresponding to the higher afﬁnity sites as a function of temperature ( the error bars represent the S.E.M. obtained from the average of at least three independent experiments ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>As the ◦ temperature is decreased from 37 to 30 and 25 C , approximately the same amount of DnaA -- ATP is required to-bind the highafﬁnity sites and activate its own transcription at dnaAP2 , whereas more protein is needed in order to form the oligomer , occupy the low-afﬁnity sites and repress transcription at dnaAP1 ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>( B ) As the amount of DnaA -- ATP in the cell increases , oligomerization of DnaA -- ATP on the dnaAP2 promoter region results , at ﬁrst , in repression of transcription of dnaAP2 and then in repression of dnaAP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>This process has been shown to take place in sequential stages : the binding of DnaA -- ATP or -- ADP to high-afﬁnity sites provides a scaffold for formation of a higher-order structure by the ﬁlling of the lower afﬁnity sites when the amount of DnaA -- ATP in the cell increases ( reviewed in [ 7 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>An increase in the amount of DnaA -- ATP in the cell at lower temperatures may be required in order to counteract the steep temperature-dependence of the stability of the melted complex at the origin compared with the initial complex [ 41 ] , which is consistent with the temperature-dependence of DnaA -- DNA interactions and DnaA co-operativity shown in the present Figure 7 Summary of the mechanism of autoregulation of dnaA expression ( A ) At low concentrations of DnaA -- ATP ( grey ovals ) , DnaA binding to the high-afﬁnity sites box 1 and box 2 ( ﬁlled black boxes ) results in activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The quantitative approach used in the present study has permitted for the ﬁrst time characterization of the changes in DnaA -- ATP DNA-binding activity in vitro as a function of temperature and to obtain a direct measurement of the effects that the temperature-dependence of this interaction has on both positive and negative autoregulation of promoter activity in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Figure 3 Quantiﬁcation of DnaA -- ATP binding to the DnaA sites on the dnaA promoter region at 37 ◦ C The graphs represent the fractional saturation of the DnaA sites as a function of DnaA -- ATP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The observation that the effect of the mutations of the activating DnaA sites on promoter activity in vivo reﬂects the changes in binding affinity measured in vitro ( Figs 5 and 6 ) indicates that the concentration of DnaA-ATP in the cell is within the range of its affinity constant for these upstream sites , at least for the period during which RNAP is recruited to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The 2 bp substitutions to box 1 ( Figure 1B ) strongly affect the protection by DnaA of the other binding sites , and addition of the box 2 mutation , in the context of the mutated box 1 ( ) , results in a further decrease in afﬁnity for box 1 compared with Mbox1 , in agreement with the presence of co-operativity in DnaA binding stabilizing the interactions of DnaA -- ATP with the weaker sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The high-afﬁnity sites speciﬁc for both DnaA -- ATP and DnaA -- ADP are indicated by boxes with a black edge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>( C ) The broken line indicates that the extent of dnaAP1 repression depends on the temperature , reﬂecting the temperature-dependence of DnaA -- ATP binding to the lower afﬁnity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The mutation altering the DnaA box2 ( M40-42box2 ) decreases , as expected , the affinity of DnaA-ATP for DnaA box2 ( by a factor of 2 ) , and to a less extent DnaA binding to box1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Since the GATC cluster is situated adjacent to datA , it is possible that binding of a DnaA complex to datA could promote further binding of DnaA to putative ATP-DnaA boxes , overlapping the GATC sites in the GATC cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>In dnaA , as in oriC , binding of DnaA to ATP-DnaA boxes first requires DnaA binding to DnaA box R1 .4,38 Thus , it is possible that the prolonged sequestration found at the dnaA promoter partly is mediated by DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>In the presence of ATP , DnaA binding extends to several other sites by cooperative interactions , ultimately leading to the melting of an AT-rich region in oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>These observations can be explained by the results obtained in vitro showing a difference in the temperature-dependence of DnaA -- ATP binding to its highand low-afﬁnity sites , resulting in a decrease in DnaA -- ATP oligomerization at lower temperatures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Quantiﬁcation of these experiments showed that , as the temperature is decreased , the binding afﬁnity of DnaA -- ATP for box 1 , box 2 and box 4 sites remains more or less constant between 37 and 30 ◦ C and then decreases between 30 and 20 ◦ C ( Figure 2B and Table 1 , and Supplementary Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>These results are in agreement with previous studies [ 17,20 ] and show that , as the DnaA -- ATP concentration is increased , binding to the box 1 site is followed by the co-operative formation of a higher-order oligomeric structure spanning the adjacent sites and , at higher concentration , by the protection of the core promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>RESULTS Temperature-dependence of DnaA -- ATP binding to its own promoter We used a quantitative DNase I footprinting assay in order to measure the binding afﬁnity of DnaA -- ATP to the dnaAP2 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Several DnaA-binding sites are found both upstream and overlapping the main promoter elements , resulting in negative autoregulation of transcription by DnaA -- ATP [ 16 -- 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>NrdR , like DnaA , binds ATP and dATP and controls the transcription of RNR genes by binding to speciﬁc target sites i the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>At a DnaA concentration of 10 , ug/ml , protection of bases -55 to -35 was observed ( wild-type footprint , Fig. 4 ) , demonstrating in vitro binding of DnaA immediately upstream of the nrd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>This is similar to what had been observed by affinity chromatography .10 Unlike HobA , DiaA stimulated EcDnaA binding only within an optimal EcDnaA : DiaA ratio of 4:1 -- 1:1 ( 8 -- 32 nM DiaA ) , similar to earlier results ; 10 a further increase in the concentration of DiaA slightly decreased DnaA binding ( compare Fig. 4d and b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Experiments presented betow ( Table 4 ) show that this is not due to presence of the 76 bp pBR322 fragment , but to the presence of E. coli sequences , most likely including the 8/9 DnaA binding site found just upstream of the dam gene ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>Although it was mentioned previously by others14 that complexes formed by DnaA and SeqA with DNA and observed under an electron microscope could be distinguished , we were not able to ﬁnd sufﬁciently high differences between such DnaA and SeqA complexes at the pR region to analyse binding of one of these proteins in the presence of another ( Figure 1 ( c ) and ( d ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Results and Discussion Binding of DnaA and SeqA to the pR promoter region It was found previously that both DnaA and SeqA proteins bind to the replication region of bacteriophage l DNA .16,27 However , DnaA boxes necessary for DnaA-mediated activation of p bind R DnaA weakly , as they differ from the consensus DnaA box sequence in a few positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>It was found that DnaA binding to this DNA region is inhibited in the presence of SeqA , and that SeqA canno bind efﬁciently to the DNA fragments complexed with DnaA .14,19 Moreover , under certain conditions , DnaA molecules could be released from DNA by addition of SeqA .15 Although there are some differences between these observations and the results of our experiments , it seems that the same type of interplay between DnaA and SeqA may regulate both initiation of chromosomal DNA replication and initiation of transcription from the pR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>The result shows that not only SeqA binding but also DnaA binding to newly replicated origins contributes to keeping them hemimethylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>It is also possible that part of the contribution of DnaA to sequestration of the origin is indirect , and that a DnaA complex bound at R1 , R2 , and R4 , for instance , interacts with SeqA and prolongs its binding to the hemimethylated GATC sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>In the presence of SeqA , it is also a possibility that DnaA binding to the DnaA boxes in datA causes changes in the DNA conformation that are favorable for SeqA binding/complex building in the adjacent GATC cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Thus far , three proteins have been identified with specific functions and binding sites in oriC ( i ) DnaA , the initiator protein , which binds four 9-mers and then interacts with a contiguous set of three 13-mersto open the duplex ; ( ii ) the IciA protein which binds these 13-mers to prevent DnaA protein from opening this region and thus inhibits initiation ; and ( iii ) IHF , which binds a sequence that facilitates the bending of oriC around an aggregate of DnaA proteins ; HU protein serves the same function as IHF buw t ithout an identified binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>When oligonucleotide templates were used in studies on SeqA -- DNA interactions , it was demonstrated that SeqA binds speciﬁcally to hemimethyl-ated GATC sequences , but not to fully methylated and unmethylated GATC regions .12,13 However , experiments with longer DNA fragments indicated that SeqA can also interact speciﬁcally with fully methylated oriC6 ,14,15 and other DNA regions .16,17 At the oriC region , SeqA binds speciﬁcally to two GATC sites , one on each side of the DnaA box R1 .14 It was proposed that this protein spreads to adjacent regions , which may result in competition between SeqA and DnaA for binding to DNA .14,15,18,19 Quite surprisingly , recent studies indicated that DnaA and SeqA , the main positive and negative replication regulators , are also speciﬁc transcription factors .16,17,20 -- 22 Both proteins can either activate or repress certain promoters , whereas activities of other promoters are not affected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>This discrepancy may arise from the fact that , in contrast to the results presented here , interactions of DnaA with boxes of the consensus sequence , and of SeqA with a DNA region rich in GATC sequences ( the oriC region ) were investigated previously .14 Therefore , we decided to compar binding patterns of DnaA and SeqA on the basis of experiments with each protein added separately to DNA samples , rather than to analyse protein -- DNA interactions in the presence of both DnaA and SeqA in the same reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Oligomerization via the N-terminus of DnaA is also required for the initiation reaction .6 Initiation of replication occurs once per cell cycle and , during rapid growth , at all origins in the cell simultaneously .7 DnaA binding to the low-affinity sites , as well as binding of architectural elements ( Fis , IHF , and HU ) , may possibly contribute to the synchrony of initiation .8 -- 10 Two main mechanisms prevent initiation from reoccurring at already replicated origins and involve either inactivation of the newly replicated origin ( by sequestration ) or inactivation of the DnaA protein ( by hydrolysis of ATP-DnaA to ADP-DnaA ) .11,12 Specific inactivation of new origins occurs because new origins are hemimethylated ( i.e. , adenine nucleotides of GATC sites are methylated on only one strand ) .13,14 The SeqA protein binds to the hemimethylated GATC sites and prevents remethylation by Dam methylase .15 Regions on the chromosome that have high numbers of GATC sites ( oriC and dnaA promoter ) remain hemimethylated considerably longer ( for up to one-third of a cell cycle ) than regions with few GATC sites .14,16 Also a region with an artificially high number of GATC sites ( an oriC-like distribution of GATC sites placed in the tnaA gene ) was found to be sequestered as long as oriC , showing that sequestration depends on the concentration and distribution of GATC sites .17 During sequestration of newly replicated origins , ATP-DnaA is hydrolyzed to ADP-DnaA in a process that requires the Hda protein and the DNA-loaded β-clamp subunit of the DNA polymerase III holoenzyme .18 This process contributes to lowering the initiation potential by lowering the proportion of ATP-form DnaA .19 The total amount of free DnaA is also lowered because new DnaA binding sites are generated by replication .20,21 After sequestration , the initiation potential is low , and initiation does not occur until a generation has passed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>During most of the cell cycle , only the three highest-affinity 9mer DnaA boxes ( R1 , R2 , and R4 ) are bound by DnaA .22,23 The SeqA protein binds to newly replicated hemimethylated DNA24 -- 27 and forms large complexes that can be visualized by immunofluorescence microscopy .28,29 Each SeqA focus may contain several replication forks , although the exact composition of forks in SeqA foci may vary with growth conditions .29,30 SeqA complex building behind the forks may function to organize newly replicated DNA24 ,31,32 and may also have a role in chromosome segregation .33 In oriC , some of the low-affinity DnaA boxes ( I2 , I3 , and R5M ) also contain GATC sequences .34 SeqA prevents DnaA from binding to these boxes on hemimethylated oriC , but does not hinder binding to the three highest-affinity DnaA boxes during sequestration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Analysis of DnaA binding to oriC in the presence of DiaA/HobA Protein/concentration ( nM ) DnaA : HobA or DnaA : DiaA ΔM of a complexa RU b c max ΔRU DnaA bindingd HpDnaA/32 -- -- 1.40 -- 1:112 1 4.54 50 HpDnaA/32 , HobA/128 EcDnaA/32 nM EcDnaA/32 , DiaA/32 EcIHpII-IVDnaA/32 EcIHpII-IVDnaA/32 , DiaA/128 6.36 -- 1.51 -- 3.46 318 79 121 85 294 -- 1.28 - 1.18 -- 1.29 -- 1.6:110 -- 2.5:113 -- ed as in EcDnaA + DiaA 1 1.20 - 1.30 1 2.68 Assum In all calculations the values of the SPR analysis , which reflect the maximal increment of protein -- DNA interactions in the presence of HobA/DiaA , were used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>It is therefore tempting to speculate that by binding to this N-terminal part of DnaA , HobA and DiaA could regulate the self-oligomerization status of DnaA or its interaction with DnaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>This proved that DiaA and HobA are functional homologs and upon binding to DnaA they exert a similar effect on orisome formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>The DnaA interaction surfaces involved in HobA -- HpDnaAI-II interactions were similar to those identified in the DiaA -- EcDnaAI-II complex , suggesting a conserved mechanism of binding of DiaA and HobA to DnaA .12,13 In this study , we asked whether DiaA and HobA are functional homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>The mode of E. coli orisome formation assisted by DiaA might confirm that the function of DiaA consists of modulation and remodeling of DnaA binding with oriC in order to direct DnaA molecules to low-affinity DnaA sites ( i.e. ADLAS boxes ) , rather than activation of robust and highly increased DnaA binding with oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Because of the ability of DiaA to-bind multiple DnaA molecules , it has been assumed to promote DnaA binding to oriC by a recruitment mechanism , whereby occupancy of low-afﬁnity sites , and subsequent activation of initiation , is enhanced by DiaA-bridged interaction to DnaA-bound high-afﬁnity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DiaA binds to multiple DnaA subunits and is thought to promote cooperative DnaA binding to weak afﬁnity DNA sites through interactions with DnaA in domains I and/or II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DnaAD96-120 mutants still bind DiaA : DiaA binds multivalently to DnaA and is believed to promote cooperative binding of DnaA to lower afﬁnity DNA sites by recruitment of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>When DnaA and RNAP were used in the same binding reaction , DnaA was added ﬁrst and incubated for 15 min , and then RNAP was added to the reaction mixture for an additional 10 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Since the promoters guaB used for assessment of GRDC in the current study contained only the upstream DnaA binding site , which does not serve to repress P in the absence of the downstream site , this argues guaB against a role for DnaA in GRDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>In several experiments and under differing DnaA concentrations we always observed DnaA protection only at the second DnaA box , but we never observed binding to the first one .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>In this process , DnaA promotes the reaction by directly binding to DnaB ( 1 , 40 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>This sequestration establishes an `` eclipse '' period , a time at which binding of DnaA and reinitiation is actively prevented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Speck , C. , and Messer , W. ( 2001 ) Mechanism of origin unwinding : sequential binding of DnaA to double - and single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Rules for DnaA binding and roles of DnaA in origin unwinding and helicase loading .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>Speciﬁc binding was demonstrated by in vitro assays including ﬁlter binding as well as mobility shift in a polyacrylamide gel of the DnaA -- DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>This indicates that DnaA bends the DNA while binding to the nrdAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Summary of results of binding of DnaA and SeqA to DNA is shown in ( a ) , where DnaA-binding pattern is presented as a thick line , and SeqA-binding pattern as a thin line .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Inhibition of the colony formation of cells bearing the intact oriC and dnaA depended on the DnaA box cluster on DARS2 ( pOA77 in Fig. 4F ) , which is consistent with the idea that DnaA binding to DARS2 is prerequisite for the inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Hence by preferentially binding the hemimethylated sequences in this region of the origin , reinitiation by DnaA might be inhibited until some mechanism of desequestration succeeded in removing the hemimethylated oriC binding factor , allowing the Dam methylase to remethylate the region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>However , EcIHpII-IVDnaA was more sensitive to DiaA stimulation than HpDnaA to HobA : less DiaA was required to obtain maximal binding of the fusion DnaA to oriC than for maximal stimulation of HpDnaA -- oriC interactions by HobA ( equimolar DiaA -- EcIHpII-IVDnaA and 4:1 HobA : HpDnaA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Published data together with our SPR results ( Fig. 4 ) demonstrated that both proteins , in spite of the particular differences discussed below , enhance binding of their own DnaA with oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>cation during steady-state growth may be set by the binding of a ﬁxed quantity of active DnaA to oriC ( 16 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981994" target="_blank">8981994</a>]<br></li>
<li>They were found to interact with domain I of DnaA and stimulate DnaA binding to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">DnaA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Secondary structure elements are shown for HpDnaA ( based on the HpDnaA crystal structure ) and the conserved ERP loop is indicated by broken-line box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>This was done by moving EcDnaA slightly so that the interactions identified by NMR by Keyamura and co-workers ,13 i.e. DiaA residues Arg71 , Pro72 , Leu190 and Phe191 interacting with EcDnaA residues Glu21 , Trp50 , Phe46 and Trp25 , were respected ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>This is due to the different properties of some key residues of the DnaAI α3 helix involved in the interface Arg45 , Asp49 , Asp53 for EcDnaA that correspond to Gln52 , Thr56 , Ala60 in HpDnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In addition , the size of the side chains of some HpDnaA residues involved in the binding is different in EcDnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Moreover , by using a library of fragments of HpDnaA , a putative interacting domain ( 12 -- 98 ) was determined as a minimal interacting region with HobA ( Rain et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Finally , we show that HobA speciﬁcally interacts with the N-terminal part of HpDnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>We also show that HobA binds the N-terminal region of HpDnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>These results , together with previous studies clearly indicate that HobA interacts directly with HpDnaA and that the N-terminal domains I and II are sufficient for this binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Additionally , we could not detect binding of DiaA to the N-terminus of HpDnaA ( domains I-II , HpDnaAI-II ) , which interacts strongly with HobA ( Fig. 2 , sectors 5 and 6 , respectively ) .16 We conclude that the absence of protein interaction between HobA and EcDnaA and between DiaA and HpDnaA is a key explanation for the inability to replace one protein with its structural counterpart during orisome assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In eukaryotes , the proteasomal ATPase family has undergone a massive proliferation : 6 orthologous lineages , namely S4 , S6a , S6b , S7 , S8 , S10a , and S10b , can be traced back to the common ancestor of all crown group eukaryotes Table 1 Clamp loader clade Pre-sensor 1 hairpin superclade SR [ CAT ] motif associated with Arginine ﬁnger b-hairpin inserted before Sensor 1 motif * HolB/DNAX family SFIII helicase clade [ viruses ] Zn cluster between helix-1 and strand 2 ** Hol B subfamily [ B ] ** DNAX subfamily [ B , Pl ] * N * P * O CLDV D5 ATPase family ositive strand RNA virus helicase fam ther viral helicases ily * RFC family ** MTH241/RFC2 subfamily [ AE ] HslU/ClpX/Lon / ClpAB-C clade Extended loop after strand 2 * HslU/ClpX family ( In eukaryotes includes RFC2 , RFC3 , RFC4 , RFC5 ) ** MTH240/RFC1 subfamily [ AE ] ( In eukaryotes includes RFC1 , Chl12 , RAD24 , Yor144c ) * WHIP family [ B , E ] ** HslU subfamily [ B , Kin , Api ] ** ClpX subfamily [ B , E ] * Lon family Fused to a C-terminal WC domain Fused to a Lon-protease at the C-terminus ** Bacterial lon protease subfamily [ B , E ] Fused to a LAN domain at the N-terminus ** Archaeal lon protease subfamily [ A > B ] * ClpAB-C family NRhD motif associated with Arginine ﬁnger ** ClpAB-C subfamily [ B , E ] ** Torsin subfamily [ E ] Torsin has a distinct a helical N-terminal domain RuvB family [ B , E ] Has C-terminal wHTH domain ; Helical segment between strand 5 and C-terminal bundle DnaA/ORC clade Two helices of approximately equal length after strand 2 * DnaA-family ** DnaA subfamily [ B ] ** DnaC subfamily [ B ] * CDC6/ORC family [ A , E ] ** CDC6/ORC [ A ] ** Orc1p , Orc4p , Orc5p , and Cdc6p subfamilies [ E ] Classical AAA clade and its divergent relatives Helix-2 insert clade Small helix between strand-2 and helix-2 , [ GN ] R motif associated with arginine ﬁnger * FtsH family [ B , E ] Fused to a metalloprotease at the C-terminus * CDC48 family [ A , E ] DPBB at N-terminus , 2 AAA + domains * Proteasomal ATPase family [ A , E ] ** S4 , S6a , S6b , S7 , S8 , S10a and S10b subfamilies [ E ] b -- a -- b insert in helix-2 NtrC/MCM group Have [ AG ] [ FL ] T motif in helix-2 before b -- a -- b insert * NtrC family [ B ] * MCM family [ A , E ] Fused to a Zn ribbon at N-terminus and a wHTH at the C-terminus * Chelatase / YifB group ** Chelatase family [ A , B , plants ] Associated with a vWA domain ** YifB family [ B ] Fused to a Lon Protease at its N-terminus , insert of Zn cluster after strand 4 * McrB family [ B > A , E ] * Mox R group ** MoxR family [ A , B ] ** Dynein/Midasin family * Katanin p60/Fidgetin family [ E ] * NSF1 family [ E ] * Pex1/6 subfamily [ E ] * Bcs1p subfamily [ E ] ( Possible relatives of classical AAA clade ) * AFG1 family [ B , E ] * ClpAB-N family [ B > A , E ] * TIP49 family [ A , E ] N-terminal module with characteristic H between Helix-2 and Strand 3 [ ST ] H motif , Predicted b barrel 6 Tandem AAA + domains ** Dynein subfamily [ E ] ** Midasin subfamily [ E ] Abbreviations : A , Archaea ; B , Bacteria ; E , Eukaryotes ; Api , Apicomplexans ; Kin , Kinetoplastids ; and Pl , Plants `` > '' indicates sporadic lateral transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>Domain II is var-iable in length and even absent among bacterial DnaAs ( Messer et al. 1999 ; Erzberger et al. 2002 ) and is composed of residues 87 -- 134 for E. coli DnaA , with our spontaneous deletion spanning amino acids 96 -- 120 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>This work was supported by National Institutes of Health grants RO1 GM051753 and GM079510 and a Brandeis University Undergraduate Research Program fellowship to K.M. LITERATURE CITED Abe , Y. , T. Jo , Y. Matsuda , C. Matsunaga , T. Katayama et al. , 2007 Structure and function of DnaA N-terminal domains : spe-ciﬁc sites and mechanisms in inter-DnaA interaction and in DnaB helicase loading on oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_13">
Structure and function of DnaA N-terminal domains : speciﬁc sites and mechanisms in inter-DnaA interaction and in DnaB helicase loading on oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></label>
<input id="dom_similarity_13", type="checkbox">
<div class="DOM_similar">
<li>Structure and function of DnaA N-terminal domains : Specific sites and mechanisms in inter-DnaA interaction and in DnaB helicase loading on oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li></div>
<li>After SDS-polyacrylamide gel electrophoresis and electrophoretic transfer , the membranes ( Protran , Schleicher & Schuell ) were incubated with rabbit antiserum that specifically recognizes one or more epitopes within amino acids 370 to 467 of the C-terminal region of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>DnaA with the amino terminal polyhistidine tag was isolated using immobilized metal-ion afﬁnity chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>An amino-terminal tag of 10 consecutive histidine residues has been fused to DnaA protein ( termed His10-DnaA ) , facilitating rapid isolation of high yields of soluble protein through tight binding to nickel immobilized on a resin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>In this study , we demonstrate that the E. coli genome contains two intergenic regions , herein called DARS1 and DARS2 , capable of reactivating ADP-DnaA in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>However , it is important to point out that the p3rpoH-lacZ fusion does not contain any of the other promoters of the rpoH gene , but it does contain the two DnaA boxes which are located upstream of the rpoH p3 promoter ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8478327" target="_blank">8478327</a>]<br></li>
<li>The p3rpoH-lacZ fusion construct used in the present study indeed contains both DnaA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8478327" target="_blank">8478327</a>]<br></li>
<li>Fragment J carries two putative DnaA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Notably , each of these two regions contains three similarly arranged DnaA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>There are no R-type DnaA boxes within the GATC cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Since the GATC cluster is situated adjacent to datA , it is possible that binding of a DnaA complex to datA could promote further binding of DnaA to putative ATP-DnaA boxes , overlapping the GATC sites in the GATC cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Plasraids used in the galactokinase experiments ( Table 4 ) contain little or no promoter activity in this region in the presence of wild-type DnaA protein ; thus , activation of transcription activity via decrease in DnaA concentration if DnaA protein acts as a transcription terminator is difficult to explain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>All rights reserved Results The GATC cluster is extensively sequestered when inserted near datA We showed previously that a GATC cluster , containing 10 GATC sites distributed as in oriC , caused full sequestration at an otherwise nonsequestered chromosomal site .17 Here , we studied the effects of inserting the same GATC cluster next to the datA region , which binds a large amount of DnaA protein in vivo .20 The datA region contains five 9mer R-type DnaA boxes that bind ADP-DnaA and ATP-DnaA3 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>First , DnaA protein forms a unique DnaA-oriC complex by recognizing 9-mer motifs termed DnaA boxes , other sequences named I-sites , and possibly the AT-rich region carrying three 13-mer motifs near the left boundary of oriC ( reviewed in Leonard and Grimwade , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>First , DnaA protein forms a unique DnaA-oriC complex by recognizing 9-mer motifs termed DnaA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Here , we report the crystal structure of HobA ( HP1230 ) at 1.7 Å , a recently discovered protein that speciﬁcally interacts with DnaA protein from Helico-bacter pylori ( HpDnaA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>HobA HobA Phosphoheptose-isomerase from P. aeruginosa Phosphoheptose-isomerase from C. jejuni HobA binds to the N-terminal region of DnaA HobA was found to bind HpDnaA in a high throughput yeast two-hybrid study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>To determine whether HobA , like DiaA , binds to the N-terminal region of DnaA , we puriﬁed a His6-tagged HpDnaA ( His6-HpDnaA ) and a N-terminal fragment ( His6-Nterm ) encompassing residues 1 -- 112 ( domain I and II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>In the HobA/DnaA complex , DnaA binds via its helices α2 and α3 ( yellow ribbon ) to two cavities that are very different in size/shape and potential in DiaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In particular , the N-terminal portion of helix α3 of DnaA binds to a hydrophobic/negatively charged area in HobA , whereas the corresponding surface is positively charged in DiaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In the HobA/DnaAI-II crystal structure , Val53 of HpDnaA is inserted into a small hydrophobic cavity , and the equivalent residue is Phe46 , which is crucial for DnaA interaction with DiaA .13 Analysis of the putative complex EcDnaA/HobA shows that Phe46 can not be inserted into the same cavity as Val53 without steric clashes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>The DnaA interaction surfaces involved in HobA -- HpDnaAI-II interactions were similar to those identified in the DiaA -- EcDnaAI-II complex , suggesting a conserved mechanism of binding of DiaA and HobA to DnaA .12,13 In this study , we asked whether DiaA and HobA are functional homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Both complexes seemed to be formed by binding helices α2 and α3 of DnaA to two cavities located at the dimer -- dimer-interface and involved a conserved loop ( the ERP loop ) of DiaA and HobA .12,13 The structure of DiaA was superimposed on that of HobA and the structure of EcDnaAI was superimposed on that of HpDnaAI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>ThepwB N-terminal coding region also contains a DnaA box at position 1382 ( Fig. 3 ) on the non-transcribed strand which differs from the consensus sequence ( TTATC , CAC , A ) at one position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969519" target="_blank">8969519</a>]<br></li>
<li>When A and C residues at the third and the fifth position of this DnaA box , respectively , were deleted simultaneously in pROAKK-EX , the derepression activity dropped by about 30 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>In other proteobacterial species , a DnaA box cluster occurs in various intergenic regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DARS1 and DARS2 share the DnaA box cluster essential for DARS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>The region corresponding to FK7-4 ( Fig. 1A ) and the DnaA box cluster Figure 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>The flanking region of the DnaA box cluster promotes DARS activity , possibly by facilitating DnaA multimer formation or apo-DnaA dissociation by nonspecific binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This box has a non-permitted G residue at the eighth position that , according to the analysis of Schaefer and Messer [ 34 ] , would preclude binding of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Domains III and IV , which bind ATP / ADP and DnaA box ( Roth and Messer 1995 ; Messer 2002 ; Fujikawa et al. 2003 ) , are intact in this allele , which is consistent with the possibility that initiation activity of DnaA508 might be supported by ATP-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>First , DnaA R399A and DnaA T435M , DnaA domain IV mutants lacking activity of overall DNA-binding and DnaA box binding , respectively ( Sutton and Kaguni 1997 ; Blaesing et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>A model based on the nuclear magnetic resonance structure of domain I suggests that head-to-head dimerization of DnaA domain I could reveal a surface to unite DnaB-binding sites in domain I and III ( Abe et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Regions involved in DnaB binding include not only domain I of DnaA , most likely also the site of DiaA binding ( see above ) , but also regions of domain III , aa 135 -- 148 , adjacent to the domain II linker ( Seitz et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>The site of DiaA binding in DnaA has been deduced to be in domain I and/or domain II ( Ishida et al. 2004 ) , the linker region affected by DnaAD96-120 ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Previous studies had implicated either domain I or II of DnaA as the site of DiaA binding ( Ishida et al. 2004 ; Keyamura et al. 2007 ) ; our results narrow this to domain I and/or to regions at amino acids 87 -- 95 or 120 -- 134 of domain II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>It was also shown that DiaA formed a speciﬁc complex with the N-terminal domains I and II of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>This could be because neither deletion of DnaA domain II nor of DiaA fully negates the activation of DnaA for replication initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Alternatively , both DiaA and DnaA domain II could be independently required for a step in the activation of DnaA for replication initiation ; for exam ple , both could promote DnaB recruitment by formation of speciﬁc DnaA complexes at oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DiaA regulation of DnaA : The similarity of DiaA and DnaA domain II defects raises two possibilities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Domain II could be required for DiaA 's function in the regulation of DnaA , although it does not appear to be required for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DiaA binds to multiple DnaA subunits and is thought to promote cooperative DnaA binding to weak afﬁnity DNA sites through interactions with DnaA in domains I and/or II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DnaB binding to domain I of DnaA may also function to modulate its interactions with DiaA , potentially releasing DiaA in a hand-off mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>HobA and DiaA are adjusted to domain I of their cognate DnaA Earlier studies suggested that HobA and DiaA residues involved in interactions with DnaA are similar12 and indicated a common mode of binding DiaA/HobA to DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Among bacterial initiation regulators , E. coli DiaA and H. pylori HobA have been characterized recently as factors that stimulate the formation of the initiation complex.9-12 Both proteins interact with orisomes via domain I of the DnaA proteins and promote DnaA oligomerization around oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Because DiaA and HobA interact with orisomes via domain I of DnaA proteins , we speculated that the inability of DiaA and HobA to stimulate assembly of heterologous orisomes might originate from the lack of DiaA -- HpDnaA and HobA -- EcDnaA complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>DiaA and HobA are phylogenetically related and abundant in bacterial species Because DiaA and HobA show a high degree of structural similarity and both influence orisome i the same way ( by interaction with DnaA domain I and promotion of DnaA oligomerization around oriC ) , we examined their phylogenetic relationship .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Consistently , by circumventing species-specific interactions , we restored activation of a hybrid orisome assembly in vitro , which indicates that HobA and DiaA exert a similar influence on DnaA during binding to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Further , the experiment confirmed a modular function of DnaA ; i.e. the fact that the N-terminus is responsible for species-specific interactions with other proteins including DnaA itself and DiaA / HobA , while domain IV is responsible for interactions with the cognate oriC and it emphasized that the crosstalk between the domains of the initiator protein is required for its proper function during orisome formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>It is possible that the self-interacting properties of the N-terminal domain from EcDnaA increased the oligomeri-zation potential of the fusion EcIHpII-IVDnaA when compared to HpDnaA , for which no dimerization of domain I has been shown .12 Therefore , for maximal stimulation of DnaA binding , fewer DiaA mole-cules per EcIHpII-IVDnaA or EcDnaA molecule were required than HobA molecules per HpDnaA molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>When domain I of H. pylori DnaA was exchanged for domain I of E. coli DnaA and enabled DiaA to interact with hybrid EcIHpII-IVDnaA , DiaA stimulated H. pylori orisome assembly in a manner very similar to HobA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>In order to determine whether DiaA and HobA are functional homologs we created a hybrid orisome consisting of DiaA , HporiC and the fusion protein EcIHpII-IVDnaA ( domain I of E. coli DnaA fused to domains II -- IV of H. pylori DnaA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>pAZP28 obtained allows for overproduction of the untagged fusion EcIHpII-IVDnaA , representing domain I of E. coli DnaA ( residues 1 -- 81 ) fused to domains II -- IV of H. pylori DnaA ( residues 93 -- 457 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Our results therefore suggest that a common , structure-based regulation mechanism may exist during the replication initiation that requires a SIS-domain fold and involves interaction with the N-terminal region of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Initiation requires the polymerisation of DnaA [ 23 ] and a membrane with a particular phospholipid composition [ 24 ] and domain structure to activate DnaA [ 21,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>The crystal structure of the Aquifex aeolicus DnaA AAA domain shows that the residue ( Arg-246 ) corresponding to the Hda Lys-185 residue resides on an - helix of the C-terminal subdomain ( domain IIIb ) and its basic side chain is located inside the protein structure ( 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>These mutations reside in the AAA domain which is shared by DnaA and Hda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>The mutation site ( K185C ) within the hda-185 allele is located in the AAA domain that Hda and DnaA share ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Second , domain I -- II-truncated DnaA ( denoted DnaA III -- IV ) fully sustained the DARS1-dependent ADP-releasing activity ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Rotation around the domain II linker would be required for this domain I interaction on a scaffold of head-to-tail DnaA complexes mediated through domain III interactions , as is suggested by the crystal structure of domains III and IV ( Erzberger et al. 2002 , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DnaA domains III and IV are required for DARS function Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA has four functional domains ( I to IV ) ; domain III ( AAA ) has IIIa and IIIb subdomains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>( III -- IV ) Truncated DnaA carrying only domain III ( AAA ) and IV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>( DnaC ) 6 is positioned at oriC through interactions of the N-terminal domain of DnaA with a central region of DnaB , and of DnaA residues 135 -- 148 ( in domain 3 ; Fig. 3 ) with the N-terminus of DnaB ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>Indeed , Hda carries the same residues or residues that are chemically similar to those in DnaA ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>) residues in DnaA and Hda are also shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>In the presence of ATP , DnaA binding extends to several other sites by cooperative interactions , ultimately leading to the melting of an AT-rich region in oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>In the absence of a clear mode of action for both proteins , our analysis suggests that they could carry out similar functions possibly via similar protein/protein interactions with DnaA at the conserved dimer-interface or via a region of accessible residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>A 12-kDa protein that binds thsequence which contains membranous material , cell wall , and nucleic containing the DnaA proteinboxes R2 and R3 andincludes acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>For DnaA , the asterisks mark a second area of contact with DnaB ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>The DnaA initiator protein binds separate domains in the replication origin of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>The interaction domains of the DnaA and DnaB replication proteins of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_86">
20 : 2121 -- 2134 Seitz , H. , C. Weigel and W. Messer , 2000 The interaction domains of the DnaA and DnaB replication proteins of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></label>
<input id="dom_similarity_86", type="checkbox">
<div class="DOM_similar">
<li>Seitz , H. , Weigel , C. , and Messer , W. ( 2000 ) The interaction domains of the DnaA and DnaB replication proteins of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li></div>
<li>DnaA protein of Escherichia coli : oligomerization at the E. coli chromosomal origin is required for initiation and involves specific N-terminal amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_89">
K. M. , Cherry , S. , and Kaguni , J. M. ( 1996 ) Domains of DnaA protein involved in interaction with DnaB protein , and in unwinding the Escherichia coli chromosomal origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></label>
<input id="dom_similarity_89", type="checkbox">
<div class="DOM_similar">
<li>Domains of DnaA protein involved in interaction with DnaB protein , and in unwinding the Escherichia coli chromosomal origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li></div>
<li>Since isolated replication intermediates generated during oriC plasmid synthesis caused , in the presence of IdaB , elevated hydrolysis of DnaA-bound ATP ( unpublished results ) , a fully assembled replisome may have enhanced interaction with DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>In the experiment shown in Fig. 1 , various amounts of DnaA protein were added to reaction mixtures containing three labelled linear DNA fragments : vector pUC119 , the insert fragment containing the aldA regulatory region , and pMAPCdSG10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>We noticed that the Ruv protein showed substantial homology to the proteins with ATP-binding activity , including the DnaA and RecA proteins , and various ATPases of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>Measurements of intrinsic fluorescence indicated that ATP stabilizes the conformation of the protein against heat , while circular dichroism measurement revealed an alteration of the secondary structure of DnaA protein by nucleotide-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>There are putative a-helices in the region that has a n ATP-binding motif of DnaA protein ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>COntentof the p-structure of DnaA protein seems to increase interactiOn with ATP * an event perhaps caused by a structural transition of the P-loop StruCture ( the ATP-binding motif A ) t h a t is present in DnaA protein ( 3 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>DNA sequence analysis of four independent isolates revealed an identical deletion of the DnaA-coding region at a repeated hexanucleotide sequence , causing a loss of 25 amino acids in domain II of the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>This deletion was in-frame , such that 25 residues from amino acids 96 to 120 would be removed from the protein in nonessential domain II of the DnaA , with other regions intact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>In this study , we characterize these suppressor mutations and show them to be caused by a single nonlethal deletion in domain II of the replication initiator protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>It is also possible that conformational changes in the DnaA protein , which depend on the integrity of domain II , are also associated with this active conﬁguration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>There are 12 families of enzyme domains , of which at least three are certainly catalytically active : the signal peptidase domain in LexA , the methyl-DNA protein methyltransferase as in Ada and the P-loop nucleotide triphosphate hydrolase in DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Finally , several independent fusions of the AAA + domain with different versions of the DNA-binding HTH domain are seen in the NtrC , MCM , Cdc6 , DnaA , and RuvB families ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>Biochemical and structural studies of the DnaA protein have shown that the DBD ( DNA-binding domain ) can exist under at least two different conformations depending , in part , on the formation of the DnaA oligomer structure in the absence of the high-afﬁnity sites of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Abbreviations used : DBD , DNA-binding domain ; GFP , green ﬂuorescent protein ; RIDA , regulatory inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_105">
Roth , A. , and Messer , W. ( 1995 ) The DNA-binding domain of the initiator protien DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></label>
<input id="dom_similarity_105", type="checkbox">
<div class="DOM_similar">
<li>Roth , A. , and Messer , W. ( 1995 ) The DNA-binding domain of the initiator protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>The DNA-binding domain of the initiator protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li></div>
<li>Analysis of the DNA-binding domain of Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Determination of secondary structure in solution of Escherichia coli DnaA DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>They were found to interact with domain I of DnaA and stimulate DnaA binding to oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>We revealed particular structural differences impeding formation of heterologous complexes and , consistently , we restored DiaA-enhanced oriC binding by the hybrid EcI-HpII-IVDnaA protein ; i.e. H. pylori DnaA in which domain I was exchanged with that of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Synthesis of approximately 260 residues of RNA occurred on template A and was inhi-bited by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>In addition to the DnaA protein , several other factors are important in facilitating open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>The level of free DnaA protein appears to be an important factor that determines the time of initiation of replication ( Skarstad and Boye , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>Interaction of DnaA protein with the b clamp and IdaB may stimulate this intrinsic ATPase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The study of oriC plasmid replication in vitro by Baker and Kornberg ( 7 ) has revealed that transcription by RNA polymerase preceding the initiation of replication from oriC assists DnaA protein to open the DNA duplex and that the reaction does not need interaction between the two proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836308" target="_blank">7836308</a>]<br></li>
<li>DnaA protein -RCB- DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_118">
Directionality of DnaA protein/DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7691799" target="_blank">7691799</a>]<br></label>
<input id="dom_similarity_118", type="checkbox">
<div class="DOM_similar">
<li>DnaA protein/DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li></div>
<li>The process of replicative DNA synthesis requires large cellular machinery , which in E. coli consists of DNA-polymerase-III holoenzyme ( containing at least 10 subunits ) and other essential proteins , including DnaA initiator , DnaB helicase , DnaC protein ( which delivers DnaB to the oriC region ) , gyrase and DnaG primase [ 2,6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22198407" target="_blank">22198407</a>]<br></li>
<li>Previous work has established no function for this region of protein , and deletions in the region , unlike other domains of the DnaA protein , do not produce lethality .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>The role of domain II in DnaA function : Previous work has established the nonessential nature of domain II of DnaA , believed to be a ﬂexible linker region between its oligomerization and DnaB-binding domain I and ATPase domain III of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>The strong genetic interactions between seqA and dnaA have led us to consider the possibility that the primary effector of sequestration is in fact DnaA protein with its activity suitably modified by SeqA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>The Trp residue of DnaA protein was excited using a UV light a t 290 nm and intensity of the emission was scanned over a range of 300-500 nm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Although the essential elements are conserved ( oriC , HpDnaA and HpDnaB ) , understanding the function of these unknown putative DnaA-binding proteins might help to deﬁne how H. pylori controls the replication process and the network of interactions that connect it to other fundamental cell events .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>The linkage between DnaA activity , membrane structure and fatty acids appears to be more complex than simply a requirement for UFA side chains in PG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>SeqA and DnaA proteins might be involved in the formation of such a structure , which would control replication initiation from oril by both facilitating stabilization of the heritable replication complex and regulating pR-dependent transcriptional activation of the origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>The substitution disrupts recognition of the DnaA box motif and impairs DnaA function in initiation of an oriC plasmid ( Sutton and Kaguni , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>At oriC , DnaA forms an initiation structure which includes approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>DnaA structure , function , and dynamics in the initiation at the chromosomal origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Asp-269 , the AAA sensor 1 motif , is required for DARS + function and formation of high-order complexes First , we analyzed a DnaA AAA sensor 1 motif mutant , + DnaA D269N .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Similarly , the Asp-269 residue might directly contact an adjacent protomer during complex formation , causing stabilization of the complexes and a conformational change of the DnaA AAA + domain that decreases its affinity for nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>In terms of structure , and like several other replication proteins , DnaA is a member of the ubiquitous AAA + family of ATPases ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>AAA family proteins , which also include DnaA , share unique motifs associated with functions in nucleotide-binding and hydrolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>The other DnaA proteins are full length , with + amino acid substitutions in the AAA motifs indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA proteins are highly conserved among bacteria and are composed of four domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>The arginine ﬁnger appears to be displaced relative to its position in other AAA + domains in the DnaA and Orc / Cdc6 families ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>DnaA , ORC , and Cdc6 : similarity beyond the domains of life and diversity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Additionally , we find that b subunit in vivo functions to control the cellular distribution of DnaA protein between the ATP and ADP forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>The DNA-loaded form of the clamp subunit of the polymerase binds the Hda protein , which promotes ATP-DnaA hydrolysis , yielding inactive ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>RIDA stimulates the hydrolysis of DnaA-bound ATP , yielding the inactive ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Formation of an ATP-DnaA-specific initiation complex requires DnaA arginine 285 , a conserved motif in the AAA protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>The Hda Arg-168 residue in the AAA box VII arginine ﬁnger motif is required for ATP-DnaA hydrolysis during the RIDA reaction ( 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_144">
The exceptionally tight afﬁnity of DnaA for ATP/ADP requires a unique aspartic acid residue in the AAA sensor 1 motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></label>
<input id="dom_similarity_144", type="checkbox">
<div class="DOM_similar">
<li>The exceptionally tight affinity of DnaA for ATP/ADP requires a unique aspartic acid residue in the AAA sensor 1 motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li></div>
<li>ADP-DnaA molecules assemble on DARS1 in a unique manner DnaA R281A and DnaA R285A have substitutions in the arginine residues of the Box VII motif arginine fingers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>EMSA was performed in the presence of 2 mM ADP to detect the complex containing ADP-DnaA and DARS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Our analyses suggest that specific DnaA homomultimers form on DARS1 , and that specific interactions occur between their AAA domains , thereby reducing + their affinity for ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>ADP-DnaA molecules assemble on DARS1 in a manner dependent on both DnaA box binding and nucleotide-mediated interaction of AAA + domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>These results suggest that the conformation of the AAA + domain is important for the inter-DnaA interaction in DARS complexes and for modulating the interaction with ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DARSs probably facilitate a specific inter-DnaA interaction that leads to the release of bound ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Due to domain swapping , EcIHpII-IVDnaA gained the ability to interact with DiaA ( Fig. 5a and c ) and did not bind HobA ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>The interactions of HpDnaA and EcIHpII-IVDnaA with HporiC binding were very similar ( Figs. 4a and 5b ) , which indicates that the overall oriC-binding properties were retained by the fusion protein and were not changed significantly by domain I of EcDnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>Domain I of EcDnaA and DiaA are optimally adjusted for interaction ; thus , the stoichi-ometry and the sensitivity of their interaction applies also to the fusion protein EcIHpII-IVDnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">DnaA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Therefore , we concluded that both proteins are in fact functional homologs , but the low-level sequence conservation ( 15 % identity and 31 % similarity ) and co-evolution with domain I of the cognate DnaA preclude interchangeability between DiaA and HobA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>90-min induction period , His10-DnaA protein accumulated to approximately 30 % of total cellular protein ( Fig. 2A , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>A protein homologous to DnaA ( Hda , for homolog of DnaA ) binds the processivity clamp , b , and DnaA , promoting hydrolysis of ATP ( Kato and Katayama 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>Furthermore , the -10 region of P1 is located within the sequence highly homologous to the DnaA protein binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>In terms of structure , and like several other replication proteins , DnaA is a member of the ubiquitous AAA + family of ATPases ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>AAA family proteins , which also include DnaA , share unique motifs associated with functions in nucleotide binding and hydrolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Formation of an ATP-DnaA-specific initiation complex requires DnaA arginine 285 , a conserved motif in the AAA protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>The proportions ( % ) of ATP-DnaA in the total ATP/ADP-DnaA are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The DnaA protein , the initiator of E. coli chromosomal replication , is a member of the AAA ATPase family and + has an exceptionally high affinity ( Kd of 10 -- 100 nM ) for ATP/ADP ( Sekimizu et al. 1987 ; Neuwald et al. 1999 ; 1 Present address : Otsuka Pharmaceutical Co. , Ltd. , Tokushima 771-0192 , Japan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>A similar distribution was observed in the DnaA , ArgR , and Ihf families ( where a high proportion in small genomes was observed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>There are 12 families of enzyme domains , of which at least three are certainly catalytically active : the signal peptidase domain in LexA , the methyl-DNA protein methyltransferase as in Ada and the P-loop nucleotide triphosphate hydrolase in DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>The arginine ﬁnger appears to be displaced relative to its position in other AAA + domains in the DnaA and Orc / Cdc6 families ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>DnaA , ORC , and Cdc6 : similarity beyond the domains of life and diversity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by DnaA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Measurement of DnaA protein 's nucleotide binding Strain , plasmid , and phage Genotype characteristics or Source or reference A. Strains DH5a F2 supE44 DlacU169 ( f80 lacZDM15 ) dR17 recA1 endA1 gyrA96 thi-1 relA1 l 2mcrA mcrB nalR F2 ompT hsdS ( r2 2 - rec B BmB ) gal dcm D ( srl A ) 306 < Tn10 ( TetR ) l ( DE3 ) [ pLysS ( CamR ) ] endA1 , hsdR17 , recA1 , dnaA5 B/r arg-28 deoB23 dnaA204 gal-11 h 7 hsdS-K12 lac-11 leu-19 mal met-55 pro-19 rbs rpsL56 ( StrR ) sul-1 thyA5 rp-25 hs is-4 9 t Laboratory stock BLR ( DE3 ) pLysS MM294 WM433 Novagen Laboratory stock Laboratory stock ( 20 ) B. Plasmids & Phages pKA211 pET19b AmpR ; dnaA1 Laboratory stocka Novagen AmpR ; expression vector , T7lac prom r and translation signal ; T7 terminator ; multiple cloning site ; pBR322 origin AmpR ; dnaA1 in pET19b AmpR ; HincII-PstI ( 2189 to 1448 bp agment oriC in pBluescript AmpR ; HincII-PstI ( 2189 to 1448 bp agment oriC in M13mp8 ote ) fr ) fr pZL411 pBSoriC This work Laboratory stock ( 21 ) Laboratory stock ( 22 ) M13mpRE85 Note .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Involved in accelerating the dephosphorylation of phosphorylated CheY ; involved in generating a regulating signal for bacterial ﬂagellar rotation ; increases at phosphonate growth 179 179 5.56 / 24,008.4 5.1 / 66,325.63 P13000 Dethiobiotin synthetase Q93SE0 Vitamin B12 transporter BtuB BioD BtuB 179 179 4.81 / 20,469.56 Vitamin B12 transport P06610 BtuE 179 periplasmic protein BtuE Carbamoyl-phosphate synthase small chain CarA 5.91 / 44,088 5.94 / 43,827 5.91 / 41,431.01 P0A6F1 5.88 / 42,128 ( DIGE 4.5 -- 6.5 ) 5.82 / 42,128 ( DIGE 4.5 -- 6.5 ) 5.79 / 41,417 ( DIGE 4.5 -- 6.5 ) 5.49 / 127,208 5.22 / 117,710.53 Carbamoyl-phosphate synthase large chain P00968 CarB 293 5.36 / 32,716 ( DIGE 4.5 -- 6.5 ) 5.42 / 31,539.87 Cytidine deaminase P0ABF6 Cdd 97 , 321 , 325 4.78 / 71,382.39 Chemotaxis protein P07363 CheA CheY 4.89 / 13,966.17 4.95 / 9,698 ( 4 -- 5 ) P0AE67 Chemotaxis protein 179 , 287 CheZ 4.44 / 23,976.03 4.51 / 28,668 P0A9H9 Chemotaxis protein 228 , 293 , 294 Continued on facing pag Participates in iron-transport ; induced by iron and cAMP and increases after benzoic acid treatment Part of a stress-induced multichaperone system ; involved in the recovery of the cell from heat-induced damage in cooperation with DnaK , DnaJ , and GrpE ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and after benzoic acid treatment ; induced by heat shock , phosphate limitation , phosphonate growth , and other environmental stresses ; controlled by RpoH Cleaves peptides in various proteins in a process that requires ATP-hydrolysis ; has a chymotrypsin-like activity ; plays a major role in the degradation of misfolded proteins ; may play the role of a master protease which is attracted to different substrates by different speciﬁcity factors such as ClpA or ClpX ; induced by heat shock ; controlled by RpoH Involved in the modulation of the speciﬁcity of the ClpAPmediated ATP-dependent protein-degradation ATP-dependent speciﬁcity component of the Clp protease ; directs the protease to speciﬁc substrates ; may bind to the lambda O substrate protein and present it to the ClpP protease in a form that can be recognized and readily hydrolyzed by ClpP ; can perform chaperone functions in the absence of ClpP ; induced by heat shock Colicin I receptor 5.03 / 71,149.34 5.06 / 73,587 ( DIGE 4.5 -- 6.5 ) ClpB Chaperone P63284 71 , 179 , 228 , 249 , 287 , 293 , 294 , 321 5.37 / 95,585.02 5.30 / 88,421 5.38 / 73,900 5.33 / 88,745 ( DIGE 4.5 -- 6.5 ) 5.44 / 34,853 ( 5 -- 6 ) ClpP 5.52 / 23,186.65 5.60 / 24,224 ( 5 -- 6 ) P0A6G7 ATP-dependent Clp protease proteolytic subunit 179 , 249 , 287 4.94 / 12,179.06 5.40 / 10,645 ( 4.5 -- 5.5 ) P0A8Q6 ATP-dependent Clp protease adaptor protein ClpS 287 P0A6H1 ATP-dependent Clp protease ATP-binding subunit ClpX 179 5.24 / 46,224.82 179 179 5.56 / 24,746.34 7.04 / 43,306.95 P0A6I0 Cytidylate kinase Cmk CoaBC catalyzes two steps in the biosynthesis of CoA ; in the ﬁrst step , cysteine is conjugated to 4-phosphopantothenate to form 4-phosphopantothenoylcysteine ; in the second , compound is decarboxylated to form 4-phosphopantotheine Involved in CoA-biosynthesis ; catalyzes the phosphorylation of the 3 - hydroxyl group of dephospho-CoA to form CoA Catalyzes two consecutive reactions converting 2 ,3 - cyclic nucleotide to 3-nucleotide and then 3-nucleotide to nucleic acid and phosphate Cmk P0ABQ0 CoA-biosynthesis bifunctional protein CoaBC 79 5.77 / 22,621.71 Dephospho-CoA kinase CoaE P0A6I9 ( YacE ) 1 5.38 / 68,914.86 2 ,3 - Cyclic-nucleotide 2-phosphodiesterase CpdB 9 P08331 17 Continued on following pag The phosphoenolpyruvate-dependent-sugar-phosphotransferase-system ( PTS ) , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in glucose-transport ; plays an important role not only in the transcriptional-control but also in the translational control of rpoS expression Induced by low temp ; cold shock protein Induced by low temp ; cold shock protein Induced during a stationary phase and starvation Inhibits DNA-replication at both initiation and elongation steps , most probably by binding to the opened , single-stranded regions at replication forks ; plays a regulatory role in chromosomal replication in nutrient-depleted cells ; induced by stationary phase and starvation 4.73 / 18,119.88 4.68 / 18,985 4.57 / 20,069 phosphotransferase enzyme IIA component 4.95 / 14,761 ( 4 -- 5 ) 4.86 / 14,800 ( 4 -- 5 ) 4.95 / 13,865 ( 4 -- 5 ) 4.95 / 18,479 ( 4.5 -- 5.5 ) P0A9X9 Cold shock protein CspA 5.57 / 7,272.09 133 , 163 6.54 / 7,716.72 P36995 Cold shock-like protein CspB 163 6.82 / 7,271.17 6.71 / 10,366 P0A9Y6 Cold shock-like protein CspC 54 , 163 , 228 , 293 , 294 163 , 320 5.81 / 7,968.97 P0A968 Cold shock-like protein CspD 8.06 / 7,332.26 5.64 / 7,780.73 P0A972 Cold shock-like protein P0A978 Cold shock-like protein CspE CspG 179 207 Induced by low temp ; cold shock protein Involved in cysteine-biosynthesis ; repressed by sulfate or cysteine ( protein induced by sulfate starvation ) ; induced by high pH CysK 5.83 / 34,358.46 5.81 / 36,027 P0ABK5 Cysteine synthase A 5.83 / 34,342 ( DIGE 4.5 -- 6.5 ) 4.95 / 24,032 ( 4 -- 5 ) 5.01 / 26,270 ( 4 -- 5 ) 4.95 / 25,586 ( 4 -- 5 ) 4.93 / 22,204 ( 4 -- 5 ) 5.06 / 29,410 ( 4.5 -- 5.5 ) 5.32 / 32,815 ( 4.5 -- 5.5 ) 5.04 / 28,256 ( 4.5 -- 5.5 ) 4.95 / 28,122 ( 4.5 -- 5.5 ) 5.78 / 42,978 ( 5 -- 6 ) 5.10 / 41,867 ( 5 -- 6 ) 5.47 / 31,383 ( 4.5 -- 5.5 ) Like a CysK , catalyzes the same reaction of cysteine-biosynthesis ; can also use thiosulfate in place of sulﬁde to give cysteine thiosulfonate as a product Part of the ABC transporter complex cysAWTP , involved in sulfate/thiosulfate import ; speciﬁcally binds thiosulfate and is involved in its transmembrane transport Could help control the pool of 3 - phosphoadenoside-5-phosphosulfate or its use in sulﬁte synthesis Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors Not involved in transpeptidation but in peptidoglycan-synthesis Involved in oxidative deamination of D-amino acids such as alanine-catabolism Involved in L-lysine-biosynthesis-via diaminopimelate-pathway : tetrahydrodipicolinate from L-aspartate ( third step ) ; sensitive to lysine inhibition 5.81 / 36,027 P0ABK5 Cysteine synthase A 5.83 / 34,342 ( DIGE 4.5 -- 6.5 ) 4.95 / 24,032 ( 4 -- 5 ) 5.01 / 26,270 ( 4 -- 5 ) 4.95 / 25,586 ( 4 -- 5 ) 4.93 / 22,204 ( 4 -- 5 ) 5.06 / 29,410 ( 4.5 -- 5.5 ) 5.32 / 32,815 ( 4.5 -- 5.5 ) 5.04 / 28,256 ( 4.5 -- 5.5 ) 4.95 / 28,122 ( 4.5 -- 5.5 ) 5.78 / 42,978 ( 5 -- 6 ) 5.10 / 41,867 ( 5 -- 6 ) 5.47 / 31,383 ( 4.5 -- 5.5 ) Cysteine synthase B P16703 CysM 5.42 / 32,664.16 287 6.43 / 35,057.44 6.58 / 39,151 ( 6 -- 11 ) Thiosulfate-binding protein P16700 CysP 5.59 / 27,175.86 CysQ protein CysQ 79 P22255 1 6.68 / 41,337.2 P0AEB2 Penicillin-binding protein 5 DacA 179 179 8.73 / 49,568.63 P24228 Penicillin-binding protein 4 DacB DadA 6.17 / 47,607.29 P0A6J5 D-Amino acid dehydrogenase small subunit 179 DapA P0A6L2 Dihydrodipicolinate synthase 5.98 / 31,269.97 6.10 / 32,361 5.94 / 32,916 179 , 228 , 286 , 294 Continued on facing pag See DapA : fourth step ; its activity is repressed by lysine ; it decreases after benzoic acid treatment Involved in L-lysine-biosynthesis-via diaminopimelate-pathway : dl-diaminopimelate from lldiaminopimelate See DapD Required for phage C1 adsorption ; increases after benzoic acid treatment Required for cell-wall-formation Reassigned CsdA ; plays a key role in optimal cell-growth at low temp and required for normal cell-division ; exclusively localized in the ribosomal fraction and becomes a major ribosomal-associated protein in cells grown at low temp ; decreases after benzoic acid treatment but induced by cold shock or acid condition Serine protease ; required at high temp and involved in the degradation of damaged proteins ; induced by heat shock Protease with a shared speciﬁcity with DegP ; induced by high pH during anaerobic growth Phosphotransfer between the C1 and C5 carbon atoms of pentose Involved in nucleotide and deoxyribonucleotide-catabolism ; increases during high-cell-density cultivation Cleaves guanosine or inosine to respective bases and sugar-1-phosphate molecules ; decreases after benzoic acid treatment Involved in glycerol utilization ; dihydroxyacetone-binding subunit of the dihydroxyacetone kinase , which is responsible for phosphorylating dihydroxyacetone ; binds covalently dihydroxyacetone in hemiaminal linkage ; acts also as a corepressor of DhaR by binding to its sensor domain in the absence of dihydroxyacetone ; induced by high pH during anaerobic growth Involved in glycerol utilization ; ADP-binding subunit of the dihydroxyacetone kinase , which is responsible for phosphorylating dihydroxyacetone ; DhaL-ADP receives a phosphoryl group from DhaM and transmits it to dihydroxyacetone ; DhaLADP acts also as a coactivator by binding to the sensor domain of dhaR ; DhaL-ATP is inactive ; induced by high pH during anaerobic growth TABLE 2 -- Continued Dihydrodipicolinate reductase 5.45 / 28,756.61 5.37 / 27,969 ( DIGE 5.11 / 37,922 ( 5 -- 6 ) 5.33 / 31,023 ( 5 -- 6 ) 4.5 -- 6.5 ) DapD 2,3,4,5-Tetrahydropyridine-2 ,6 - dicarboxylate Nsuccinyltransferase P0A9D8 5.56 / 29,892.10 5.51 / 31,680 179 , 228 , 294 , 321 5.49 / 32,476 ( DIGE 4.5 -- 6.5 ) Diaminopimelate epimerase Protein P0A6K1 P37620 DapF DcrB 179 5.86 / 30,208.56 4.76 / 17,786.96 179 , 321 4.50 / 18,143 ( DIGE 4.5 -- 6.5 ) DdlA DeaD P0A6J8 D-Alanine-D-alanine ligase A P0A9P6 Cold shock DEAD-box protein A 5.02 / 39,315.81 8.76 / 70,414.96 179 5.41 / 37,079 ( DIGE 4.5 -- 6.5 ) 132 , 274 , 321 DegP 7.87 / 46,829.14 8.01 / 66,440 ( 6 -- 11 ) P0C0V0 Protease Do 179 , 287 DegQ 5.39 / 44,445.75 5.34 / 46,509 Protease P39099 179 , 286 , 323 5.11 / 44,369.96 P0A6K6 Phosphopentomutase DeoB 179 P0A6L0 Deoxyribose-phosphate aldolase 5.50 / 27,733.80 5.52 / 33,657 ( 5 -- 6 ) DeoC 70 , 287 , 325 4.5 -- 6.5 ) 5.40 / 25,098 ( DIGE 5.42 / 25,818.72 Purine nucleoside phosphorylas DeoD P0ABP8 e 321 PTS-dependent DhaK P76015 ( YcgT ) 4.93 / 39,494.75 323 dihydroxyacetone kinase , dihydroxyacetone-binding subunit PTS-dependent DhaL P76014 ( YcgS ) 5.31 / 22,631.75 323 dihydroxyacetone kinase , ADP-binding subunit Continued on following pag DhaM P37349 ( YcgC ) PTS-dependent See DhaL ; serves as the 4.61 / 51,579.81 4.69 / 50,800 ( DIGE 4.5 -- 6.5 ) 179 , 321 , 323 dihydroxyacetone kinase , phosphotransferase subunit phosphoryl donor ; induced by high pH during anaerobic growth Dosage-dependent suppressor of a dnaK deletion mutation ; suppressed not only the temp-sensitive growth but also the ﬁlamentous phenotype of the dnaK deletion strain , while the defect of lambda growth is not suppressed ; induced by high pH Participates in initiation and elongation during chromosome replication ; exhibits DNA-dependent ATPase activity and contains distinct active sites for ATP binding , DNA binding , and interaction with DnaC protein , primase , and other prepriming proteins Interacts with DnaK and GrpE to disassemble a protein complex at the origins of replication of phage lambda and several plasmids ; participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stressdenatured proteins and by disaggregating proteins , also in an autonomous , DnaK-independent fashion ; induced by heat shock under the control of the htpR regulatory protein Involved in chromosomal DNA-replication , possibly through an analogous interaction with the DnaA protein , and participates actively in the response to hyperosmotic shock ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and after benzoic acid treatment ; induced by temp , ethanol , nalidixic acid , puromycin , and overexpression of bioproducts DksA P0ABS1 DnaK suppressor protein 5.06 / 17,527.75 4.90 / 18,723 5.01 / 17,853 179 , 274 , 286 , 321 5.08 / 19,656 ( DIGE 4.5 -- 6.5 ) DnaB P0ACB0 Replicative DNA helicase 4.95 / 52,390.08 4.94 / 49,355 179 , 228 , 294 8.03 / 40,969.14 DnaJ 79 Chaperone protein P08622 1 DnaK 4.81 / 69,647 4.83 / 68,983.76 P0A6Y8 Chaperone protein 71 , 97 , 138 , 179 , 212 , 228 , 287 , 294 , 297 , 321 4.92 / 74,532 ( DIGE 4.5 -- 6.5 ) 4.94 / 73,587 ( DIGE 4.5 -- 6.5 ) 4.91 / 74,850 ( DIGE 4.5 -- 6.5 ) 4.95 / 60,559 ( 4 -- 5 ) 4.98 / 67,563 ( 4 -- 5 ) 4.98 / 66,075 ( 4 -- 5 ) 4.96 / 61,267 ( 4 -- 5 ) 4.97 / 61,125 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.48 / 59,582 ( 4 -- 5 ) 5.00 / 44,877 ( 4 -- 5 ) 4.93 / 52,435 ( 4 -- 5 ) 4.96 / 60,841 ( 4 -- 5 ) 4.96 / 60,234 ( 4.5 -- 5.5 ) 4.97 / 59,766 ( 4.5 -- 5.5 ) 4.95 / 53,032 ( 4.5 -- 5.5 ) 4.97 / 41,400 DNA polymerase III is a complex , multichain enzyme responsible for most of the replicative synthesis in bacteria ; exhibits 3 - to-5 exonuclease activity ; required for initiation of replication once it is clamped onto DNA See DnaN ; contains the editing function and is a proofreading 3 - to-5 exonuclease Dipeptide-binding protein of a transport system that can be subject to osmotic shock and is also required for peptide chemotaxis ; increases in the physiological short-term adaptation to glucose limitation P0A6Y8 Chaperone protein 71 , 97 , 138 , 179 , 212 , 228 , 287 , 294 , 297 , 321 4.92 / 74,532 ( DIGE 4.5 -- 6.5 ) 4.94 / 73,587 ( DIGE 4.5 -- 6.5 ) 4.91 / 74,850 ( DIGE 4.5 -- 6.5 ) 4.95 / 60,559 ( 4 -- 5 ) 4.98 / 67,563 ( 4 -- 5 ) 4.98 / 66,075 ( 4 -- 5 ) 4.96 / 61,267 ( 4 -- 5 ) 4.97 / 61,125 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.95 / 60,700 ( 4 -- 5 ) 4.48 / 59,582 ( 4 -- 5 ) 5.00 / 44,877 ( 4 -- 5 ) 4.93 / 52,435 ( 4 -- 5 ) 4.96 / 60,841 ( 4 -- 5 ) 4.96 / 60,234 ( 4.5 -- 5.5 ) 4.97 / 59,766 ( 4.5 -- 5.5 ) 4.95 / 53,032 ( 4.5 -- 5.5 ) 4.97 / 41,400 P0A988 DNA polymerase III beta subunit DnaN 294 5.25 / 40,586.60 DNA polymerase III epsilon subunit 5.54 / 27,098.93 5.65 / 26,832 P03007 DnaQ 228 , 294 DppA 5.47 / 52,481 5.69 / 52,170 5.75 / 57,407.06 Periplasmic dipeptide-transport protein P23847 179 , 286 , 287 , 311 5.71 / 52,000 ( 5 -- 6 ) 5.71 / 50,118 ( 5 -- 6 ) 5.81 / 52,000 ( 5.5 -- 6.7 ) Continued on facing pag Unspeciﬁcally binds and Dps P0ABT2 DNA-protection during starvation protein 5.72 / 18,564.11 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) 5.78 / 17,897 ( 5 -- 6 ) 5.27 / 16,787 ( 5 -- 6 ) 5.21 / 16,164 ( 5 -- 6 ) 5.94 / 17,273 ( 5.5 -- 6.7 ) protects DNA from oxidative damage mediated by hydrogen peroxide ; induced by prolonged starvation , stationary phase , and acetic acid accumulation ; controlled by ClpXP and ClpAP proteases , which affect logphase stability and stationary-phase synthesis of Dps , respectively Transfers its disulﬁde bond to other proteins and is reduced in the process and then reoxidized by DsbB ; induced by high pH during aerobic or anaerobic growth See DsbA ; required for disulﬁde bond formation in some periplasmic proteins Involved in de novo synthesis of thymidylate ; produces dUMP , the immediate precursor of thymidine nucleotides , and decreases the intracellular concn of dUTP so that uracil can not be incorporated into DNA General inhibitor of pancreatic serine proteases ; inhibits chymotrypsin , trypsin , elastases , factor X , and kallikrein as well as a variety of other proteases Key enzyme in the EntnerDoudoroff pathway and participates in the regulation of the intracellular level of glyoxylate ; induced at the stationary phase and in the presence of gluconate or hexuronic acids ; increases after benzoic acid treatment , phosphate limitation , and phosphonate growth Involved in peptide bond synthesis ; stimulates efﬁcient translation and peptide bond synthesis on native or reconstituted 70S ribosomes in vitro ; probably functions acceptors for peptidyl transferase May be involved in the early steps of isoprenoid-biosynthesis GTP-dependent nucleic acidbinding protein which may act as a translation factor Involved in the pathway of glycolysis ; also involved in the RNA degradosome , a multienzyme complex important in RNA-processing and mRNA-degradation ; increases during anaerobic growth 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) 5.78 / 17,897 ( 5 -- 6 ) 5.27 / 16,787 ( 5 -- 6 ) 5.21 / 16,164 ( 5 -- 6 ) 5.94 / 17,273 ( 5.5 -- 6.7 ) DsbA P0AEG4 Thiol : disulﬁde interchange protein 5.42 / 21,132.07 5.34 / 22,426 5.31 / 21,942 179 , 228 , 274 , 294 , 321 , 323 5.34 / 22,606 ( DIGE 4.5 -- 6.5 ) P0AEG6 Thiol : disulﬁde interchange protein 5.86 / 23,460 DsbC 179 Dut 5.03 / 16,155.47 Deoxyuridine 5 - triphosphate nucleotidohydrolase P06968 5.03 / 16,145 179 , 228 , 294 , 321 5.10 / 17,526 ( DIGE 4.5 -- 6.5 ) 5.94 / 16,099.52 Ecotin P23827 Eco 179 Eda P0A955 2-Keto-4-hydroxyglutarate / 2-keto-3-deoxy-6 - phosphogluconate aldolase 99 , 179 , 287 , 293 , 321 4.93 / 23,247 ( DIGE 4.5 -- 6.5 ) 5.43 / 27,937 ( 5 -- 6 ) 5.57 / 22,284.03 Efp 179 4.9 / 20,460.12 P0A6N4 Elongation factor P 179 , 287 ElbB P0ABU5 Enhancing lycopene biosynthesis 4.68 / 22,981.55 4.92 / 25,756 ( 4 -- 5 ) ( YhbL ) protein 2 5.03 / 42,217 ( DIGE 4.5 -- 6.5 ) 5.02 / 42,547 ( 4.5 -- 5.5 ) 4.87 / 39,536.13 EngD P0ABU2 GTP-dependent nucleic acidbinding protein 287 , 321 Eno 5.34 / 46,509 5.29 / 46,234 5.32 / 45,523.75 P0A6P9 Enolase 29 , 179 , 228 , 271 , 286 , 287 , 294 , 321 5.35 / 43,401 ( DIGE 4.5 -- 6.5 ) 4.74 / 22,872 ( 4 -- 5 ) 5.46 / 45,494 ( 4.5 -- 5.5 ) 5.20 / 40,116 ( 4.5 -- 5.5 ) 5.16 / 31,510 ( 4.5 -- 5.5 ) 5.29 / 51,580 ( 5 -- 6 ) 5.27 / 51,301 ( 5 -- 6 ) 5.16 / 51,319 ( 5 -- 6 ) 5.00 / 50,698 ( 5 -- 6 ) 5.02 / 50,647 ( 5 -- 6 ) 5.26 / 50,000 ( 5 -- 6 ) 6.95 / 24,005 ( 6 -- 11 ) 6.72 / 25,314 ( 6 -- 9 ) Continued on following pag 2,3-Dihydro-2 ,3 - EntA P15047 4.97 / 26,249.66 179 dihydroxybenzoate ( 2,3-DHB ) dehydrogenase Isochorismatase Involved in siderophore-biosynthesis and enterobactin-biosynthesis Required for production of 2,3-DHB and serves as an aryl carrier protein ; plays a role in enterobactin assembly : proteins EntB , EntD , EntE , and EntF form a multienzyme complex called enterobactin-synthetase Member of the two-component regulatory system EvgS/EvgA ; regulates the expression of emrKY operon and yfdX ; also seems to control expression of at least one other multidrug efﬂux operon Catalyzes the condensation reaction of fatty-acid-synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP Involved in fatty-acid-biosynthesis 4.97 / 26,249.66 179 P0ADI4 EntB 21 5.08 / 34,633 ( DIGE 4.5 -- 6.5 ) 5.05 / 32,554.34 3 EvgA 79 P0ACZ4 Positive transcription regulator 6.83 / 22,690.2 1 P0A953 3-Oxoacyl - acyl-carrier-protein 5.35 / 42,613.32 5.76 / 44,039 synthase I FabB 294 FabD 4.95 / 32,286.01 5.37 / 44,881 4.95 / 33,409 P0AAI9 Malonyl CoA-acyl carrier protein transacylase 179 , 228 , 287 , 294 , 321 5.03 / 33,449 ( DIGE 4.5 -- 6.5 ) 5.03 / 32,221 ( 4.5 -- 5.5 ) See FabB ; has a preference for short chain acid substrates and may function to supply the octanoic substrates for lipoic-acid-biosynthesis Involved in fatty-acid-biosynthesis pathway See FabB ; catalyzes the ﬁrst condensation reaction , which initiates fatty-acid-synthesis and may therefore play a role in governing the total rate of fatty acid production ; possesses both acetoacetyl-ACP synthase and acetyl transacylase activities Involved in second reduction step of fatty-acid-biosynthesis ; decreases after benzoic acid treatment Involved in saturated fatty-acid-biosynthesis Catalyzes the ﬁnal step of fatty-acid-oxidation in which acetyl-CoA is released and the CoA ester of a fatty acid two carbons shorter is formed ; involved in the aerobic and anaerobic degradation of long-chain-fatty-acids ; induced in the presence of fatty acids and is under the control of the fadR repressor Catalyzes the formation of a hydroxyacyl-CoA by addition of water on enoyl-CoA ; also exhibits 3-hydroxyacyl-CoA epimerase and 3-hydroxyacyl-CoA dehydrogenase activities ; involved in the aerobic and anaerobic degradation of long-chain-fatty-acids ; induced in the presence of fatty acids and is under the control of the fadR repressor Involved in translocation of long-chain-fatty-acids across the outer membrane ; induced in the presence of long-chain-fatty-acids and under the control of the fadR repressor 5.03 / 33,449 ( DIGE 4.5 -- 6.5 ) 5.03 / 32,221 ( 4.5 -- 5.5 ) 5.71 / 42,914.57 n i e t o r p FabF ¬ r e i r r a c acyl-II xoacylsynthase 3 - O 5 I A A 0 P 179 6.76 / 25,560.29 P0AEK2 3-Oxoacyl - acyl-carrier-protein reductase 3-Oxoacyl - acyl-carrier-protein synthase III FabG 179 5.08 / 33,515.12 P0A6R0 FabH 179 FabI 5.60 / 32,613 ( DIGE 4.5 -- 6.5 ) 5.40 / 32,959 ( DIGE 4.5 -- 6.5 ) 5.37 / 38,366 ( 5 -- 6 ) 5.33 / 37,266 ( 5 -- 6 ) 5.58 / 27,732.75 P0AEK4 Enoyl - acyl-carrier-protein reductase ( NADH ) 179 , 287 , 321 6.84 / 17,032.95 FabZ P0A6Q6 ( 3R ) - hydroxymyristoyl - acyl carrier protein dehydratase P21151 3-Ketoacyl-CoA thiolase 179 6.31 / 40,876.20 FadA 46 5.84 / 79,593.91 Fatty oxidation complex alpha subunit P21177 FadB 4 6 FadL 26 Long-chain fatty-acid-transport 4.91 / 45,906.47 protein P10384 Continued on facing pag Involved in the pathway of glycolysis ; increases in poly ( 3-hydroxybutyrate ) - accumulating cells FbaA P0AB71 Fructose-bisphosphate aldolase class II 5.52 / 39,016.07 5.55 / 40,651 5.43 / 39,776 5.49 / 39,104 ( DIGE 5.56 / 50,220 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.68 / 32,568 4.5 -- 6.5 ) Necessary for growth on Fbp P0A993 Fructose-1 ,6 - bisphosphatase 179 , 228 , 294 5.67 / 36,833.93 substances such as glycerol , succinate , and acetate ; inhibited by AMP , which affects the turnover of bound substrate and not the afﬁnity for substrate Outer membrane receptor protein in the Fe3 dicitrate-transport system ; for induction , the TonB and the ExbB proteins have to be active Involved in the initial step of iron uptake by binding ferrienterobactin , an iron chelatin siderophore that allows E. coli to extract iron from the environment ; acts as a receptor for colicins B and D ; increases after benzoic acid treatment Binds the ferrichrome-iron ligand ; interacts with the TonB protein , which is responsible for energy coupling of the ferrichromepromoted iron-transport system ; acts as a receptor for bacteriophage T5 as well as for T1 , phi80 , and colicin M ; can also transport the antibiotic albomycin Activates rRNA-transcription ; plays a direct role in upstream activation of rRNA promoters ; binds to a recombinational enhancer sequence that is required to stimulate Hinmediated DNA inversion ; prevents initiation of DNA-replication from OriC PPIases accelerate the folding of proteins ; catalyzes the cistrans isomerization of proline imidic peptide bonds in oligopeptides ; strongly inhibited by FK506 See FklB Iron ( III ) dicitrate-transport protein P13036 FecA 79 5.36 / 81,707.21 1 5.23 / 79,771.07 Ferrienterobactin receptor P05825 FepA 321 5.24 / 79,447 ( DIGE 4.5 -- 6.5 ) 5.13 / 78,742.19 Ferrichrome-iron receptor P06971 FhuA 179 9.34 / 11,239.93 P0A6R3 DNA-binding protein Fis 79 1 FklB 4.5 -- 6.5 ) 4.84 / 24,436 ( DIGE 4.85 / 22,085.00 P0A9L3 FKBP-type 22-kDa peptidylprolyl cis-trans isomerase 179 , 321 P45523 FKBP-type peptidyl-prolyl cis - 6.73 / 26,223.63 7.08 / 33,268 trans isomerase Flavodoxin 1 4.21 / 19,605.75 4.65 / 21,200 FkpA 179 , 286 Low-potential electron donor to a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Catabolite response Deoxirribose assimilation D 87 135 Crp w 27 Pérez - Rueda and Collado-Vides ( 2000 ) 220 -- R 235 10 DeoR DicA 35 w Pérez - Rueda and Collado-Vides ( 2000 ) -- n/d 0 -- -- n/d HLH R 4 DicC DnaA 0 Transcription and replication regulation Positive regulator of cytochrome c552 77 0 -- Pérez - Rueda and Collado-Vides ( 2000 ) 41 -- n/d DniR 0 Kajie et al. ( 1991 ) 209 8 215 -- A Ebp FhlC 77 m Pérez - Rueda and Collado-Vides ( 2000 ) Pérez - Rueda and Collado-Vides ( 2000 ) n/d 0 -- 7 -- Pérez - Rueda and Collado-Vides ( 2000 ) Schnetz ( 2002 ) n/d R FhlD FrvR 0 -- Phosphoenolpyruvate sugar phosphotransferase system operon regulation Iron assimilation 8 0 -- -- 128a Fur R w Achenbach and Yang ( 1997 ) 107 460 35 Carbon sources assimilation Nguyen and Saier ( 1995 ) ; Weickert and Adhya ( 1992 ) 388 GalR/LacI 55 R c -- 13 GcvR GntR Rigali et al. ( 2002 ) Glycine cleavage operon repression 0 R 735 775 Ghrist and Stauffer ( 1998 ) 96 R w Yamada and Saier ( 1988 ) Activator of the glucitol operon in E. coli High-frequency persistence of the lethal effects of inhibition of either DNA or peptidoglycan-synthesis regulation ( HipAB ) -- 8 GutM 0 A -- Black et al. ( 1991 ) -- 1 HipA 1 0 n/d -- n/d HipB HtgA Black et al. ( 1991 ) Missiakas et al. ( 1993 ) 87 0 -- -- -- n/d 7 Heat shock regulation 0 -- HycA HypF 6 -- n/d ZnF A 0 -- 0 Bouhouche et al. ( 2000 ) Gui et al. ( 1996 ) 27 -- Glyoxylate bypass operon Integration host factor 31 106 161 w R -- IclR Ihf -- 15 8 Pérez - Rueda and Collado-Vides ( 2000 ) Campoy et al. ( 2002 ) -- D 43 -- DNA-repair 54 LexA R w 102 279 Miller and Bassler ( 2001 ) Schell ( 1993 541 LuxR/UhpA m A Quorum sensing 1548 LysR 1271 Amino acid biosynthesis regulation D w 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Activity of pR is controlled both positively and negatively by variety of regulators , including the stringent-response alarmon ppGpp , phage-encoded cI and Cro proteins and host-en-coded DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Another study ( 46 ) also suggests that SeqA plays a role in negative modulation of DnaA activity independent of the methylation status of the origin , and hence in a manner not directly related to oriC sequestration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>( E ) Immunoblot analysis using anti-DnaA antiserum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA standards and total cellular protein from MG1655 , JC125 ( MG1655DhdaHtetR ) , JC126 ( MG1655DdatAHkanR ) and JC326 ( MG1655DseqAHtetR ) cells were probed with anti-DnaA antiserum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
<li>However , the polA gene-expression and the enhancer effect by DnaA were modulated by ppGpp which plays an important role in the regulationof cell-growth and cell-division ( Schreiber et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>The aim of this work is to test this hypothesis and to elucidate the role of DnaA in the regulation of nrdAB expression and its possible effect on the timing of expression as a function of the cell-cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Nonetheless , it is possible that multiple mechanisms of cell-cycle control , such as the DnaA levels and the regulatory inactivation of DnaA ( RIDA ) , cooperate to control cell-cycle in response to ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079575" target="_blank">19079575</a>]<br></li>
<li>The presence of multiple pathways to provide ATP-DnaA might be advantageous in cell-cycle controls responding to various growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Table 1 Correspondence among replication proteins from Escherichia coli , Bacillus subtilis , and bacteriophages T7 and T4 phage T4 phage T7 B. subtilis E. coli Function PolC , DnaEBS b2 t5 RTP2 SSB4 DnaA , B , D DnaC6 DnaI DnaG polymerase/exonuclease sliding clamp clamp loader terminator SSB initiator replicative helicase helicase loader primase aey b2 gp43 gp453 gp5 Trx ( E. coli ) -- -- gp2 .52 T7 RNAP gp4 -- gp4 dgt2d ' cw Tus gp444gp62 -- gp32 -- gp41 gp59 gp6 SSB4 DnaA , IHF , HU , FIS DnaB6 DnaC DnaG Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>Approximately around the same time , a detailed computational analysis of various cellular and viral proteins involved in nucleic-acid-metabolism , such as DnaA , the MCM proteins , NtrC-type transcription factors , and helicases of various RNA and DNA viruses comprising the helicase * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004192" target="_blank">11004192</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20875449" target="_blank">20875449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20640526" target="_blank">20640526</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Mol Gen Genet 197:125 ± 128 Boye E , Stokke T , Kleckner N , Skarstad K ( 1996 ) Coordinating DNA-replication-initiation with cell-growth : di $ erential roles for DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
EMBO Rep 1 : 479 -- 483 Boye E , Stokke T , Kleckner N , Skarstad K ( 1996 ) Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>[ 7 ] E. Boye , T. Stokke , N. Kleckner , K. Skarstad , Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA and SeqA proteins , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>Boye , E. , Stokke , T. , Kleckner , N. , and Skarstad , K. ( 1996 ) Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12068816" target="_blank">12068816</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12067334" target="_blank">12067334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Boye , E. , Stokke , T. , Kleckner , N. and Skarstad , K. ( 1996 ) Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22373925" target="_blank">22373925</a>]<br></li>
<li>Boye , E. , Stokke , T. , Kleckner , N. , and Skarstad , K. ( 1996 ) Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Boye , E. , Stokke , T. , Kleckner , N. & Skarstad , K. ( 1996 ) Coordinating DNA-replication-initiation with cell-growth : differential roles for DnaA and SeqA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li></div>
<li>Also , IciA transcriptionally activates expression of the dnaA and nrd genes , which encode the chromosomal replication initiator DnaA and ribonucleoside diphosphate reductase , respectively ( Lee et al. 1996 ; Han et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>The release of bound nucleotides from DnaA protein erspective romoted by acidic phospholipids in a ﬂuid bilayer , a sed in the puriﬁcation of numerous proteins , was rocess that may play a crucial role in cell-cycle regu - xploited here for the isolation of an initiator protein , ation of chromosomal replication ( 6,29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>This applies also to bacteriophages and their hosts , including one of the most intensively studied models in molecular biology , phage l. Examples of the processes described above are the use of E. coli LamB protein , normally allowing uptake of maltose and maltodextrins , for phage adsorption on the cell surface ( Vinga et al. , 2006 ) , employment of heat-shock proteins in l DNA-replication and morphogenesis ( Polissi et al. , 1995 ; Taylor & Węgrzyn , 1995 ) , and involvement of the replication initiator protein DnaA in the regulation of transcription from the l pR promoter ( Szalewska-Pa asz et al. , 1998 ; Glinkowska et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>We also show genetically that hda functions with dnaN , which encodes the β clamp of DNA polymerase III holoenzyme , to control the frequency of initiation dependent on wild type DnaA , and that this pathway appears to be more critical in regulating initiation frequency than those involving datA or the sequestration of hemi-methylated oriC by SeqA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>In addition DnaA also acts as a transcription factor for a large set of genes , possibly acting as a co-ordinator of the gene-expression and DNA-replication programmes in response to perturbations in DNA-synthesis ( [ 4,5 ] and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>In most cases , DnaA acts as a repressor of transcriptional initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>In addition to being required for replication initiation , DnaA also acts as a repressor and/or terminator of transcription for several genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>In addition to its role in initiation of replication , DnaA has been shown to autoregulate its own transcription ( Sekimizu and Kornberg , 1988 ) and to control either positively or negatively the expression of several other genes ( van den berg et al. , 1985 ; Wang and Kaguni , 1989 ; Zhou and Syvanen , 1990 ; Mizushima et al. , 1994 ; Skarstad and Boye , 1994 ; Kitagawa et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>Finally , DnaA 's double role , at the origin and as a transcription factor of several genes , may allow it to act as a general mediator in the coupling of DNA-replication-initiation and elongation ( Messer and Weigel , 1997 ; Goranov et al. , 2005 ; Ishikawa et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Mutations in DiaA also suppress SeqA : DiaA has been shown to be a positive regulator of DnaA function in replication initiation ( Ishida et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>An indication of the importance of DnaA protein ( the dnaA gene product ) as a molecular switch for initiation of chromosome replication is the behavior of cells bearing the mutant dnaAcos allele ; at nonpermissive temperatures overinitiation at oriC occurs ( Kellenberger-Gujer et al. , 1978 ; Katayama , 1994 ; Katayama and Kornberg , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Expression analysis of the polA gene in relA-spoT double mutants and further physiological and genetic studies will help to identify the factors involved in the growth-rate regulation of DNA polymerase I. Consideringthat DnaA protein and DNA-replication are also under stringent control ( Chiara-mello and Zyskind , 1990 ; Guzman et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>The sequence in this region ( GTTATACTCTT ) has reasonable homology with the DnaA protein binding site consensus found in the regulatory region of genes involved in DNA-replication and repair ( van den Berg et al. 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3550388" target="_blank">3550388</a>]<br></li>
<li>Apart from its main function in the initiation of DNA-replication from oriC , the DnaA protein is involved in the control of the replication of several plasmids ( Helinski et al. 1996 ) , and in the regulation of transcription ( Messer and Weigel 1997 ; Szalewska-Paøasz et al. 1998b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>This gene encodes the DnaA protein , a molecule involved in the initiation of oriC replication in E. coli ( Lee et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10600368" target="_blank">10600368</a>]<br></li>
<li>~ gulatory role in gene-expression , the DnaA boxes outside oriC may have a pivotalrole in controllingthe frequency of initiation of replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>In addition to its role in initiating DNA-replication , DnaA is a transcriptional regulator that controls positively or negatively the expression of nrdAB by binding to DnaA boxes immediately upstream of the nrdAB operon ( 32 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>tween DnaA and other proteins involved in its regulation and chromosomal replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>SeqA and DnaA proteins might be involved in the formation of such a structure , which would control replication initiation from oril by both facilitating stabilization of the heritable replication complex and regulating pR-dependent transcriptional activation of the origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_41">
References DnaA and oriC , the bacterial mode to initiate DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></label>
<input id="rp_similarity_41", type="checkbox">
<div class="RP_similar">
<li>DnaA and oriC , the bacterial mode to initiate DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li></div>
<li>Interaction of DnaA with its bindings sites at oriC , the origin of bacterial chromosome replication , leads to formation of the replication complex responsible for duplication of the genetic material .1 -- 3 To regulate initiation of DNA-replication precisely , apart from a positive regulator ( DnaA protein in eubacteria ) , a negative control is also necessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Structure of a DnaA / HobA complex from Helicobacter pylori : insights into regulation-of-DNA-replication in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>]<br></li>
<li>( E ) Reactivation of DnaA by DARS1 for replication initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Katayama , T. , 2008 Roles for the AAA1 motifs of DnaA in the initiation of DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>The primary function of DnaA is in the initiation of DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_48">
DnaA : Controlling the initiation of bacterial DNA-replication and more .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></label>
<input id="rp_similarity_48", type="checkbox">
<div class="RP_similar">
<li>DnaA : controlling the initiation of bacterial DNA-replication and more .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21354425" target="_blank">21354425</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li></div>
<li>cAMP interacts directly with the DnaA protein and plays a role in the re-activation of DnaA , which is an essential element for initiation of DNA-replication from the chromosome origin , oriC ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8810311" target="_blank">8810311</a>]<br></li>
<li>DnaA is essential for initiation of chromosomal DNA-replication at oriC in E. coli ( Lother et al. 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>In addition to being a transcription factor for several genes , the DnaA protein is best known for its key regulatory role in the initiation of DNA-replication ( Kaguni , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Second , the only negative control known to be exerted on initiation of rpoH transcription is by DnaA , the key regulatory protein for DNA-replication ( Wang and Kaguni , 1989b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>This specificity for the form of b subunit loaded onto DNA may be a determinant that couples negative control of DnaA protein with DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>This inactivating action is targeted to DnaA protein ; other proteins involved in DNA-replication are not affected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Mechanism of DnaA protein-dependent pBR322 DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Keywords DnaA ; Hda ; β clamp ; SeqA ; datA ; initiation Introduction Chromosomal DNA-replication in free-living organisms is a highly regulated event that is coordinated with the cell-cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Those directly involved in the cell-cycle include initiation , replication and division hyperstructures based on DnaA , SeqA and the 2-minute cluster , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>In their model , ATP-DnaA acts as a repressor of nrdAB transcription ; after initiation of DNA-replication , ATP DnaA is converted to ADP DnaA , resulting in increased nrdAB expression and thereby coordinating DNA-replication and RNR synthesis during the cell-cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Collier , J. , Murray , S.R. , and Shapiro , L. ( 2006 ) DnaA couples DNA-replication and the expression of two cell-cycle master regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Results from several independent investigations have shown the E. coli cell-cycle to be a composite of several autonomous cycles ( replication cycle , growth , and division cycle , etc. ) co-ordinated through signals and checkpoints .3 -- 8 The existence of a tight co-ordination between chromosome replication and cellular-growth is implied by : ( i ) the constancy of initiation mass over a wide range of growth rates ,9,10 ( ii ) mainten-ance of `` once and only once '' rule for initiation from oriC every generation11 ; ( iii ) synchrony of initiation from all the oriCs in every cell ( even for multicopy oriC plasmids ) during exponential growth ; 12,13 and ( iv ) the growth rate-dependent eclipse period .14 Three molecular mechanisms have so far been identiﬁed contributing to these co-ordination characteristics in E. coli : ( a ) transcriptional-control of the gene for the initiator protein , dnaA , through autoregulation ; 15 -- 20 ( b ) transition between the ATP and ADP forms of the initiator protein DnaA and titration of DnaA by the highafﬁnity chromosomal locus datA ; 21 -- 25 and ( c ) control of the re-methylation rate of oriC and dnaA gene by sequestration which leads to inhibition of the re-initiation from newly replicated oriC .25 -- 29 Inactivation of SeqA and perturbation of the meth-ylation rates result in asynchronous initiation along with a reduced eclipse period .14,30,31 The stable maintenance of the E. coli cell-cycle is clearly a result of the complex interplay among these functions and possibly several others yet to be discovered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></li>
<li>Degradation of mutant DnaA protein was suppressed in the absence of the SeqA protein An alternative explanation for the low DnaA concentration in the dnaA204 mutant is protein-degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Strain Half-life of DnaA protein ( h ) CM735 wild type CM746 ( dnaA204 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>: Initiator ( DnaA ) protein concentratiun as a function of growth riite in Eschericliia coli and Srtliiioitell ~ c r ~ ~ pliiiriirrirrrti .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8045420" target="_blank">8045420</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_65">
Hansen , F.G. , Atlung , T. , Braun , R.E. , Wright , A. , Hughes , P. and Kohiyama , M. ( 1991b ) Initiator ( DnaA ) protein concentration as a function of growth rate in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></label>
<input id="rp_similarity_65", type="checkbox">
<div class="RP_similar">
<li>Hansen , F.G. , Atlung , T. , Braun , R.E. , Wright , A. , Hughes , P. , and Kohiyama , M. ( 1991b ) Initiator ( DnaA ) protein concentration as a function of growth rate in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li></div>
<li>Initiator ( DnaA ) protein concentration as a function of growth rate in Eschenchia coli and Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li>
<li>Initiator ( DnaA ) protein concentration as a function of growth rate in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_69">
172 , 2013 -- 2019 Polaczek , P. and Wright , A. ( 1990 ) Regulation of expression of the dnaA gene in Escherichia coli : role of the two promoters and the DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></label>
<input id="rp_similarity_69", type="checkbox">
<div class="RP_similar">
<li>Polaczek , P. , and Wright , A. ( 1990 ) Regulation of expression of the dnaA gene in Escherichia coli : role of the two promoters and the DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li></div>
<li>[ PubMed : 10209749 ] Riber L , Olsson JA , Jensen RB , Skovgaard O , Dasgupta S , Marinus MG , Lobner-Olesen A. Hdamediated inactivation of the DnaA protein and dnaA gene autoregulation act in concert to ensure homeostatic maintenance of the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_72">
Hansen , F.G. , Koefoed , S. , S0rensen , L. and Atlung , T. ( 1987 ) Titration of DnaA protein by oriC DnaA-boxes increases dnaA gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></label>
<input id="rp_similarity_72", type="checkbox">
<div class="RP_similar">
<li>Titration of DnaA protein by oriC DnaA-boxes increases dnaA gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>Molecular Microbiology ( 1997 ) 23 ( 6 ) ,1193 -1202 DnaA protein stimulates polA gene-expression in Escherichia coli Ariel Quiiiones , ' * Gudrun Wandt , ' + Sabine Kleinstauber ' * and Walter Messer ' ` lnstitut fur Genetik , Martin-Luther-Universitst , Domplatz 1 , D-06108 Hale , Germany .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DnaA protein stimulates polA gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>American Society for Microbiology , Washington DC , pp 580 ± 599 Quinones A , Wandt G , Kleinstauber S , Messer W ( 1997 ) DnaA protein stimulates polA gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>Quinones , A. , Wandt , G. , Kleinstauber , S. , and Messer , W. ( 1997 ) DnaA protein stimulates polA gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411720" target="_blank">10411720</a>]<br></li>
<li>J Bacteriol 171 : Wang Q & Kaguni JM ( 1989b ) DnaA protein regulates transcription of the rpoH gene of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_79">
K. M. , Cherry , S. , and Kaguni , J. M. ( 1996 ) Domains of DnaA protein involved in interaction with DnaB protein , and in unwinding the Escherichia coli chromosomal origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></label>
<input id="rp_similarity_79", type="checkbox">
<div class="RP_similar">
<li>Domains of DnaA protein involved in interaction with DnaB protein , and in unwinding the Escherichia coli chromosomal origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li></div>
<li>Keywords : guaBA operon ; transcriptional-control ; DnaA protein ; Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Novel heat shock protein HspQ stimulates the degradation of mutant DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Department of Biology , B-022 , and the Center for Molecular Genetics , University of California , San Diego , La Jolla , CA 92093 , USA Key words : Escherichia coli - DNA-replication - GATC methylation - dam gene-expression - DnaA sites * Permanent address : Institute of Biochemistry and Biophysics , Polish Academy of Sciences , Rakowiecka 36 , PL-02 532 , Warsaw , Poland Offprint requeststo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>The process of replicative DNA-synthesis requires large cellular machinery , which in E. coli consists of DNA polymerase III holoenzyme ( containing at least 10 subunits ) and other essential proteins , including DnaA initiator , DnaB helicase , DnaC protein ( which delivers DnaB to the oriC region ) , gyrase and DnaG primase [ 2,6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22198407" target="_blank">22198407</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_85">
DnaA and SeqA proteins of Escherichia coli as regulators of bacteriophage l development .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></label>
<input id="rp_similarity_85", type="checkbox">
<div class="RP_similar">
<li>DnaA and SeqA proteins of Escherichia coli as regulators of bacteriophage k development .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_87">
Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional afﬁ-nity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643657" target="_blank">14643657</a>]<br></label>
<input id="rp_similarity_87", type="checkbox">
<div class="RP_similar">
<li>Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional afinity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional afﬁnity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li></div>
<li>Mol Microbiol 19 : 1137 -- 1147 Kitagawa R , Ozaki T , Moriya S , Ogawa T ( 1998 ) Negative control of replication initiation by a novel chromosomal locus exhibiting affinity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_91">
[ PubMed : 8830270 ] Kitagawa R , Ozaki T , Moriya S , Ogawa T. Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional affinity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></label>
<input id="rp_similarity_91", type="checkbox">
<div class="RP_similar">
<li>Kitagawa , R. , Ozaki , T. , Moriya , S. , and Ogawa , T. ( 1998 ) Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional afﬁnity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Kitagawa , R. , Ozaki , T. , Moriya , S. , and Ogawa , T. ( 1998 ) Negative control of replication initiation by a novel chromosomal locus exhibiting exceptional affinity for Escherichia coli DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_94">
Titration of the Escherichia coli DnaA protein to excess datA sites causes destabilization of replication forks , delayed replication initiation and delayed cell-division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></label>
<input id="rp_similarity_94", type="checkbox">
<div class="RP_similar">
<li>Morigen , Løbner-Olesen , A. & Skarstad , K. ( 2003 ) Titration of the Escherichia coli DnaA protein to excess datA sites causes destabilization of replication forks , delayed replication initiation and delayed cell-division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>Cold Spring Harbor , NY , USA : Cold Spring Harbor Laboratory Morigen , Lobner-Olesen A , Skarstad K ( 2003 ) Titration of the Escherichia coli DnaA protein to excess datA sites causes destabilization of replication forks , delayed replication initiation and delayed cell-division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li></div>
<li>Lobner-Olesen , A. , Atiung , T , , von Meyenburg , K , , and Boye , E. ( 1989 ) Initiation of DNA-replication in Escherichia coii after overproduction of the DnaA protein , Mol Gen Genet 218 : 50-56 , Smith , D.W , , Garland .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_98">
13 , 766 -- 772 Skarstad , K. , Lobner-Olesen , A. , Atlung , T. , von Meyenburg , K. and Boye , E. ( 1989 ) Initiation of DNA-replication in Escherichia coli after overproduction of the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></label>
<input id="rp_similarity_98", type="checkbox">
<div class="RP_similar">
<li>Skarstad , K. , Løbner-Olesen , A. , Atlung , T. , von Meyenburg , K. & Boye , E. ( 1989 ) Initiation of DNA-replication in Escherichia coli after overproduction of the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></li>
<li>Skarstad , K. , Lùbner-Olesen , A. , Atlung , T. , von Meyenburg , K. , and Boye , E. ( 1989 ) Initiation of DNA-replication in Escherichia coli after overproduction of the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li></div>
<li>Camara , J.E. , Breier , A.M. , Brendler , T. , Austin , S. , Cozza-relli , N.R. , and Crooke , E. ( 2005 ) Hda inactivation of DnaA is the predominant mechanism preventing hyperinitiation of Escherichia coli DNA-replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18761688" target="_blank">18761688</a>]<br></li>
<li>In E. coli , YgfZ , a folate derivative-binding protein was shown to be involved in chromosomal replication Summary In prokaryotes , DNA-replication is initiated by the binding of DnaA to the oriC region of the chromosome to load the primosome machinery and start a new replication round .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>[ 4 ] U. Von Freiesleben , K.V. Rasmussen , M. Schaechter , SeqA limits DnaA activity in replication from oriC in Escherichia coli , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>J Mol Biol 176:155 -159 Hughes P , Landoulsi A , Kohiyama M ( 1988 ) A novel role for cAMP in the control of the E. coli DNA-replication initiator protein DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2546039" target="_blank">2546039</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_105">
A novel role for cAMP in the control of the activity of the E. coli chromosome replication initiator protein , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8396119" target="_blank">8396119</a>]<br></label>
<input id="rp_similarity_105", type="checkbox">
<div class="RP_similar">
<li>A novel role for CAMP in the control of the activity of the E. coli chromosome replication initiator protein , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li></div>
<li>In Escherichia coli , initiation of bidirectional replication at the origin ( oriC ) occurs through the concerted actions of the DnaA replication initiator protein , the hexameric DnaB helicase , the DnaC helicase loading partner and the DnaG primase , leading to establishment of two replication forks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>[ 16 ] T. Mizushima , S. Nishida , K. Kurokawa , T. Katayama , T. Miki , K. Sekimizu , Negative control of DNA-replication by hydrolysis of ATP bound to DnaA protein , the initiator of chromosomal DNA-replication in Escherichia coli , EMBO J. 16 ( 1997 ) 3724 -- 3730 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16644499" target="_blank">16644499</a>]<br></li>
<li>Interestingly , in many respects , MalT rather resembles the DnaA protein , the activator of initiation of DNA-replication in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2524384" target="_blank">2524384</a>]<br></li>
<li>The DnaA protein is the key protein in the initiation of DNA-replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_111">
Marszalek , J. , and J. M. Kaguni , 1994 DnaA protein directs the binding of DnaB protein in initiation of DNA-replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10224240" target="_blank">10224240</a>]<br></label>
<input id="rp_similarity_111", type="checkbox">
<div class="RP_similar">
<li>DnaA protein directs the binding of DnaB protein in initiation of DNA-replication in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li></div>
<li>DnaA is the initiator protein for DNA-replication in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Initiation of chromosomal DNA-replication which is stimulated without oversupply of DnaA protein in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Initiation of DNA-replication in Escherichia coli after overproduction of the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Summary The Escherichia coli hda gene codes for a DnaA-related protein that is essential for the regulatory inactivation of DnaA ( RIDA ) , a system that controls the initiation of chromosomal replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>An essential tryptophan of Escherichia coli DnaA protein functions in oligomerization at the E. coli replication origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Nontagged DnaA protein was puriﬁed ( 32 ) from BL21 ( DE3 ) pLysS / pKA211 , a transformed strain in which inducible expression of the dnaA gene is under the control of a bacteriophage T7 RNA polymerase promoter ( T. Kata-yama , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Pull-down assays for DnaA and DiaA protein binding : Plasmid pSTL376 , carrying BBD fusion to DiaA under control of the T7 promoter , was transformed into BL21 ( DE3 ) for protein expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>It is therefore conceivable that there is a coupling between the expression of the cell-division operon and the heat shock response regulator , and that the DnaA protein is involved in that coupling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3550388" target="_blank">3550388</a>]<br></li>
<li>We are currently altering the DNA sequence of the dnaA box by site-di-rected mutagenesis in order to evaluate further the effect of DnaA protein on gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1581360" target="_blank">1581360</a>]<br></li>
<li>In dnaA mutants of E. coli , the expression of ,3 - galactosidase in a drpA ' - lacZ protein fusion is eight times higher than its expression in a cell that overproduces DnaA protein ( T. Luby , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1688424" target="_blank">1688424</a>]<br></li>
<li>Such studies would contribute to understanding the role of DnaA protein in the global control of gene-expression and the cell-cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Longer term inhibition might rely on transcriptional arrest of the dnaA gene and dilution out of free DnaA protein by cell-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>We constructed several overproducing systems of DnaA protein using various expression vectors and host strains , and tested various conditions for overproduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>Influence of DnaA overproduction on polA gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_127">
These results suggest that DnaA protein negatively regulates expression of the aldA gene under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></label>
<input id="rp_similarity_127", type="checkbox">
<div class="RP_similar">
<li>These results suggest that DnaA negatively regulates the aldA gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>Discussion DnaA is both an activator and a repressor of RNR expression In 2006 Gon et al. showed that DnaA represses transcription from the nrdAB promoter controlling expression of the major aerobic ribonucleotide reductase gene in E. coli ( Gon et al. , 2006 ) while earlier the Fuchs laboratory had shown an activating role for DnaA at the same promoter ( Augustin et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The gene dnaA , which encodes E. coli DnaA transcription dual regulator and is not subjected t variable expression , i.e. abundant expression at relatively constant rate in most cells , was used as an internal control for the RT-PCR determinations [ 68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
<li>Reduction of intracellular DnaA concentrations using E. coli dnaA mutants is shown to increase transcription into the dam gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>Functional Comparison of Nontagged and Polyhistidine-Tagged DnaA Protein functional studies , its behavior in initiating DNA rep-lication from oriC and its response to regulation b For His10-DnaA protein to be useful in structural -- adenine nucleotides and membrane phospholipids must be the same as those for nontagged DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_133">
Speck , C. , Weigel , C. , and Messer , W. ( 1999 ) ATP - and ADP-DnaA protein , a molecular switch in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></label>
<input id="rp_similarity_133", type="checkbox">
<div class="RP_similar">
<li>ATP - and ADP-DnaA protein , a molecular switch in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>is a target for DnaA regulation of IMP and AMP-biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969519" target="_blank">8969519</a>]<br></li>
<li>Several genes involved in modifying DNA architecture ( e.g. , those encoding FIS and DnaA ) have been shown to be involved in the transcriptional regulation of E. coli nrdAB genes ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>In this work , we show that the YgfZ protein is involved in the regulation of DnaA initiator activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>Initiator Proteins DnaA protein is involved in the regulation of numerous genes via at least 300 ( and probably many more ) DnaA boxes scattered throughout the E. coli chromosome ( Schaefer & Messer , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>We therefore measured the ( i-gaiactosidase activity cf pALO163 and pALO219 in strain ALO454 , which carTies the dnaA46 allele encoding a temperature-sensitive DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630320" target="_blank">1630320</a>]<br></li>
<li>( A , B ) Analysis of DARS1-dependent ADP-release activity of DnaA 3 mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>This link between DnaA , fatty acid import and biosynthesis clearly requires further exploration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>As detailed above , these are predicted to be a consequence of cumulative e $ ects caused by ppGpp on phospholipid-biosynthesis [ 220 , 304 , 307 ] and fatty-acid-biosynthesis [ 407 ] , by FadR-dependent control of the fad , iclR , uspA and fab genes , [ 108 ] by growth rate-dependent alterations in acc expression [ 294 ] , and by inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>We are accumulating evidence that 110operon is regulated multivalently and apart from ` he $ ation by DnaA , is also under stringent control me - ` Oled by guanosine bis-diphosphate ( Davies and Drabdi3 in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736096" target="_blank">1736096</a>]<br></li>
<li>Indicated also are the UP element and discriminator sequence ( involved in growth-rate-dependent regulation and stringent control ) , sites identi cents ed for the binding of PurR and DnaA and the putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>In : Neidhardt FC , Curtiss III R , Ingraham J , Lin ECC , Low KB , Magasanik B , Rezniko $ WS , Riley M , Schaechter M , Umbarger H ( eds ) Escherichia coli and Salmonella : molecular and cellular biology , American Society for Microbiology , Washington , DC , pp 1579 ± 1601 Messer W , Weigel C ( 1997 ) DnaA initiator ± also a transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>DnaA also negatively regulates several genes including dnaA , mioC , rpoH , guaBA , and uvrB through interaction with DnaA boxes in each of the respective promoters [ 357 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>DnaA boxes have also been noted at or near other genes involved in nucleotide-biosynthesis [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>DnaA also controls the transcription of fadL , fabF and other genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>DnaA initiator -- also a transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>The transcriptional regulation of these genes requires several transcription factors ( including DnaA , IciA , FIS [ factor for inversion stimulation ] , Fnr , Fur , and NrdR ) , depending on the RNR class ; however , the factors that dictate the expression of some RNR genes in response to different environmental conditions are not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>DnaA is known to be a transcription factor ( Messer and Weigel 1997 ) ; thus , one might assume that the expression of genes coding for components of the membranes or for proteins involved in synthesis of such components might be impaired in dnaA mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>Apart from DnaA , expression of other genes are a $ ected by alterations in phospholipids [ 370 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Alternatively , DnaA and SeqA could cooperate in the regulation of expression of the same genes involved in production of membrane components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>DnaA is involved in the expression of many genes [ 28 ] and , in our hypothesis , in the build-up to initiation , DnaA accumulates due to the disappearance of hyperstructures that contained it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">DnaA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>insrted cytidines remove FIS binding site C C I I AACCTATTATGCCGTTCAAGAAATCGCCGAACAGTTATTTTTAACAAATTTTTCTCTTCC -170 j -150 J -130 -120 229 300 deletion removes 325-deletion 307 ~ deletion deletion removes soond positive regulatory dite positive regulatory site CATTGACTTTCCCGGACACCTTGTCTGACCTAAGGTGCGCGAAAGCCACTTII I ICCTTCC - 10 -100 -90 -80 -70 -60 start of tiSce4pion Consensus Pribnow box AGGGTCATTTTCACACmAIJT GCAGTGA I I I -20 -10 +1 DnaA box -35 box TGAGfTATCCACAAAGTTATGCACLtQ-QCA -- I I -50 -40 -30 R.B.S. ATCCCAAACATACCCCCTATATATAGTGTTCTMGCAGCTTCCCGTACTACAGGTAGTCT III III +10 +20 +30 +40 +50 +60 419 deoltion Leader peptide sart codon nrdA start codon EcoRl R.B.S. GCATGAAACTATTGCGGAAGAATTCCAAAAACAGGTACGACATACATGA * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>A1 UI gtaaagtaccagtgaccggszggttgcgtgaaa ttagaaatttcgccgctgatccaa 10 20 30 40 50 60 acctgtcccatctcatgctcaapcagcagacgaacgaaccgtttgatkcaggcgactaacgg ~ a 70 120 CRP/FNR + I I dseA aaaattgcaggggattgagaagg : aacatGTGAGCGAGATCAAATTCTAAATCAGCAGGT 8 FIS FIS AT-r1ch 1 4 ~ idseA ) TATTCAG ' TCGATliGTAACCCGCCCTTCGGGGATAOCMGCATTTTTTGCAAAAAGGGGTA 230 240 AAGTTAACTTTGC + ¬ Par box ijnaA box ~ rscfr P1 ) ' ` a SStIIf ` * * * GATWAATCGGTTACGCTCTGTATMTGCCGCQGCAATATTTATT-CTC ~ CGA 250 300 * guaa + * Al UI GATATTGC catgrtacgt-atcgctaaagaagctctgacgt ttgacgacgt tctcctcgt 3FO 3 2 0 33 0 370 A t a t ~ c q t ~ ' ; ~ a t a t c c ~ ~ ~ ~ g c t t t c c g c a g c a a t g g a t a c c g ~ ~ a c g g a a g c g c g c c t -3 DnaA box ggctattgctctggctcaggaaggcggtatcqgctttatccac ~ ~ aaacatgtccattg ~ 490 5CO 51C Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Cells were grown in a medium containing ortho - [ P ] , nucleotide-bound DnaA was isolated , and 32 the bound nucleotides were analyzed using thin-layer chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Specific genomic sequences of E. coli promote replicational initiation by directly reactivating ADP-DnaA Kazuyuki Fujimitsu , Takayuki Senriuchi , and Tsutomu Katayama 1 2 Department of Molecular Biology , Graduate School of Pharmaceutical Sciences , Kyushu University , Fukuoka 812-8582 , Japan In Escherichia coli , ATP-DnaA , unlike ADP-DnaA , can initiate chromosomal replication at oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Speciﬁc genomic sequences of E. coli promote replicational initiation by directly reactivating ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_6">
Biochimie 83 , 13 -- 17 Fujimitsu , K. , Senriuchi , T. and Katayama , T. ( 2009 ) Speciﬁc genomic sequences of E. coli promote replicational initiation by directly reactivating ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></label>
<input id="sit_similarity_6", type="checkbox">
<div class="SIT_similar">
<li>Fujimitsu , K. , Senriuchi , T. , and Katayama , T. ( 2009 ) Speciﬁc genomic sequences of E. coli promote replicational initiation by directly reactivating ADP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li></div>
<li>In Escherichia coli , the initiation of DNA replication requires the recognition of speciﬁc sequences in its single replication origin ( oriC ) by DnaA bound to ATP , which leads to the stepwise assembly of the molecular machinery at each replication fork ( reviewed in references 46 , 54 , and 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Two sites , named DARS1 and DARS2 , for DnaA-reactivating sequence ( DARS ) , contain DnaA box sequences like datA , but the DARS sites stimulate the dissociation of ADP bound to DnaA to permit DnaA to bind to ATP , which is more abundant than ADP in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Discussion In this study , we demonstrated that specific genomic sequences , DARS1 and DARS2 , reactivate DnaA for rep-lication initiation by exchanging bound ADP with ATP in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Reactions ( 25 ml ) were incubated for 20 min at 308C , and nucleotides bound to DnaA protein were assessed ( see Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>After incubation at 308C for 20 min , nucleotides bound to DnaA protein were determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>In the detachment model , the detachment of the origin from the membrane allows torsional stresses to open the 13mer sequences ; DnaA protein ensures that these sequences open successively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2204708" target="_blank">2204708</a>]<br></li>
<li>It would appear that the DnaA protein-OS complex is no longer in a form recognizable by outer membrane fractions or both DnaA and an outer membrane protein compete for common or overlapping recognition sequences at 0riC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>e-mail : h44851a!nucc.cc.nagoya - u.ac.jp Division of Biological Science , Graduate School of Science , Nagoya University , Chikusa-ku , Nagoya 464-8602 , Japan A binding site for DnaA protein was identiﬁed in the regulatory region of the aldA gene of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>A 33-kDa protein that binds specifically to the right border oforiC , next to the rightmost of the binding sitesof the initiator protein , DnaA , was identified and purified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>First , DnaA protein forms a unique DnaA-oriC complex by recognizing 9-mer motifs termed DnaA boxes , other sequences named I-sites , and possibly the AT-rich region carrying three 13-mer motifs near the left boundary of oriC ( reviewed in Leonard and Grimwade , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>First , DnaA protein forms a unique DnaA-oriC complex by recognizing 9-mer motifs termed DnaA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Results Intergenic regions upstream of uvrB and mutH can dissociate ADP-DnaA complexes in vitro To identify genomic sites capable of promoting ADP-DnaA dissociation , we searched for sequences carrying two DnaA boxes ( with the typical 9-mer TTATNCACA sequence or with only one mismatch ) that are closely located and oppositely orientated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Theoretically predicted weak DnaA boxes are shown by ticks below the line , while ticks located above the line represent GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Binding sites of DnaA and SeqA at the pR promo-ter region roughly corresponded to predicted DnaA boxes and GATC sequences ( Figure 1 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>The strain devoid of both ORF2 .1 and ORF8 .6 on the chromosome lacks sequences which include the third and fourth DnaA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>The stability of DnaA204 was not affected by providing extra datA sites Figure 1 Stability of the DnaA protein in the strains DnaA protein binds to 9-mer binding sites ( dnaA boxes ) [ 23 ] , whereas SeqA binds to two or more hemimethylated GATC sequences if situated with a correct spacing in the DNA helix [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>The datA sequence contains several dnaA boxes binding DnaA especially strongly [ 22 ] , but contains few GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>The chromosomal locus datA contains ﬁve DnaA boxes which are 9-mer DnaA-binding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Both ATP-DnaA and ADP-DnaA bind to 9mer DnaA boxes , TTA/TTNCACA , but only ATP-DnaA protein binds in addition to a 6mer site , AGATCT [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>Released apo-DnaA binds ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>DnaA binds to five 9mer DnaA boxes in oriC : strongly to three ( R1 , R2 , and R4 ) boxes that adhere to the consensus sequence TTWTNCACA , and more weakly to two ( M and R3 ) boxes that have one or two mismatches to the consensus sequence .1 -- 3 Also other low-affinity sites ( three I-sites and several 6mer ATP-DnaA boxes ) have been identified .4,5 The latter two types of low-affinity sites are exclusively bound by the active ATP-bound form of DnaA , whereas 9mer R-type DnaA boxes are bound by both ATP - and ADP-form DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>The high-afﬁnity sites speciﬁc for both DnaA -- ATP and DnaA -- ADP are indicated by boxes with a black edge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Thus far , three proteins have been identified with specific functions and binding sites in oriC ( i ) DnaA , the initiator protein , which binds four 9-mers and then interacts with a contiguous set of three 13-mersto open the duplex ; ( ii ) the IciA protein which binds these 13-mers to prevent DnaA protein from opening this region and thus inhibits initiation ; and ( iii ) IHF , which binds a sequence that facilitates the bending of oriC around an aggregate of DnaA proteins ; HU protein serves the same function as IHF buw t ithout an identified binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>IciA is a DNA-binding protein with the recognition speci ® city of three repeats of a 13 nucleotide-long sequence , located near the replication origin ( oriC ) , and inhibits the initiation of DNA replication in vitro by blocking the opening of the oriC region driven by the initiator DnaA protein ( Balke & Gralla 1987 ; Hwang & Kornberg 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>The nucleotide sequences of these recognition sites were compared to the sequences of the DNA fragments containing the three 13 nucleotide repeats in the origin of the minichromosome oriC , to which the IciA protein binds ( Hwang & Kornberg , 1990 ) , and to the sequence of the 1P promoter of the dnaA gene of E. coli , which encodes the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10600368" target="_blank">10600368</a>]<br></li>
<li>It binds to the four 9-bp consensus sequences ( so-called DnaA boxes ) within the replication origin ( oriC ) , leading to a complex consisting of origin DNA wrapped around a cluster of approximately 25 DnaA protomers ( 3,4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>DnaA binds to oriC specifically and cooperatively at 9-bp repeats ( DnaA boxes of consensus sequence TTAT ( C , ) CA ( C , ) A [ 3 ] ) present four times within the minimal origin region of 245 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>In fact , homologues of this protein were found in all eubacteria analysed so far .1 -- 3 DnaA binds to speciﬁc DNA sequences , called DnaA boxes , and all functions performed by it require these protein -- DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>In the most studied system , Escherichia coli , the DnaA protein , speciﬁcally binds to DNA sequences ( DnaA boxes ) located at the oriC region and unwinds the adjacent A-T-rich region ( Kaguni , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17683397" target="_blank">17683397</a>]<br></li>
<li>Besides the four repeats in oriC , DnaA boxes are distributed at many sites on the E. coligenome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>DnaA protein binds to numerous dnaA boxes around the chromosome [ 22,23 ] , but whether it is capable of forming higher-order structures at sites other than oriC is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12479794" target="_blank">12479794</a>]<br></li>
<li>ATP-bound DnaA binds to multiple sequences at the origin of replication , oriC , unwinding the DNA and promoting the binding of DnaB helicase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>DnaA binds to the ﬁve binding sites ( DnaA boxes ) within oriC and opens the two strands of DNA in an AT-rich region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>DnaA binds to several high-afﬁnity binding sites , known as DnaA boxes , near the origin of replication , oriC , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19546317" target="_blank">19546317</a>]<br></li>
<li>One of these is dnaA , the product of which binds to sites ( DnaA boxes ) in the origin of replication , oriC , to facilitate the opening of the DNA duplex to start replication ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981994" target="_blank">8981994</a>]<br></li>
<li>When oligonucleotide templates were used in studies on SeqA -- DNA interactions , it was demonstrated that SeqA binds speciﬁcally to hemimethyl-ated GATC sequences , but not to fully methylated and unmethylated GATC regions .12,13 However , experiments with longer DNA fragments indicated that SeqA can also interact speciﬁcally with fully methylated oriC6 ,14,15 and other DNA regions .16,17 At the oriC region , SeqA binds speciﬁcally to two GATC sites , one on each side of the DnaA box R1 .14 It was proposed that this protein spreads to adjacent regions , which may result in competition between SeqA and DnaA for binding to DNA .14,15,18,19 Quite surprisingly , recent studies indicated that DnaA and SeqA , the main positive and negative replication regulators , are also speciﬁc transcription factors .16,17,20 -- 22 Both proteins can either activate or repress certain promoters , whereas activities of other promoters are not affected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Transcription affecting replication of oriC plasmids has been shown to be modulated by the dnaA product , which binds to DnaA box sequences within or around oriC ( 1 , 10 , 11 , 13 , 20 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836308" target="_blank">7836308</a>]<br></li>
<li>The transcription affecting oriC plasmid replication is modulated by the dnaA product , which binds to DnaA box sequences within or around the oriC region ( 1 , 14 , 15 , 22 , 28 , 35 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631698" target="_blank">8631698</a>]<br></li>
<li>To determine whether DnaA protein bound to these sequences in the nrd promoter , a 32P-end-labeled nrd promoter region fragment ( fragment 229f , - 165 to +85 relative to the start of nrd transcription ) was mixed with increasing amounts of DnaA and subjected to Cu-phenanth-roline footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Interestingly , for regulation of pR activity , binding to sequences located downstream of the promoter is necessary in the case of both DnaA and SeqA ,16,22 which is unusual in prokaryo-tic transcription control systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Plasmids pLBAR2UD , pLBAR2DD , and pLBAR2D2 are derivatives of pLBAR2 with mutations in the DnaA binding box consensus sequences located between - 52 and - 32 relative to the start of nrd transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Sequences that exhibit a reasonable match to suggested consensus binding sequences for FIS ( Finkel and Johnson , 1992 ) , leucine-responsive protein ( Rex et al. , 1991 ) , IHF ( Goodrich et al. , 1990 ) , and DnaA proteins ( Schaefer and Messer , 1991 ) are indicated by solid underlining , dashed underlining , dotted overlining , and wavy overlining , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>After duplication of oriC , which has an unusual abundance of 11 GATC sequences recognized by DNA adenine methylase , the binding of SeqA to the resultant hemi-methylated DNA is thought to occlude oriC from DnaA and other proteins needed for replisome assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Hence by preferentially binding the hemimethylated sequences in this region of the origin , reinitiation by DnaA might be inhibited until some mechanism of desequestration succeeded in removing the hemimethylated oriC binding factor , allowing the Dam methylase to remethylate the region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>One of these involves titration of DnaA to the datA ( ` DnaA titration ' ) region , rich in DnaA-binding sequences ( Ogawa et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Nevertheless , it has been proposed that the main role of SeqA is sequestration of newly replicated origin sequences , as this protein interacts with hemimethylated GATC motifs , abundant in the oriC region , and prevents DnaA binding to this region ( Lu et al. , 1994 ; Slater et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464080" target="_blank">17464080</a>]<br></li>
<li>Such weak DnaA -- DNA interactions could not be detected by footprinting analysis and were found to be hardly detectable in an electrophoretic mobility shift assay .27 On the other hand , complexes of DnaA with weak DnaA boxes could be effectively analysed using electron microscopic techniques .27,28 Electron microscopy was also chosen as the most appropriate technique for studies on interactions of SeqA with relatively long DNA fragments .14 Therefore , we decided to use electron microscopy pictures to compare binding sites for DnaA and SeqA at the pR promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Thick lines indicate a region protected by DnaA and stars indicate hypersensitive sites exposed by protein binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Thick lines indicate regions protected by DnaA and stars indicate hypersensitive sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>At least ﬁve upstream sites appear to positively regulate nrd expression including a Fis binding site , a DnaA binding site , an AT-rich region , an inverted-repeat and a 10 bp site between the AT-rich region and the inverted-repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>Wild-type 15 kb upstream of nrd start site Wild-type 165 bp upstream of nrd start site Fis binding site deleted Fis binding site and part of AT-rich region deleted Fis binding site and AT-rich region deleted pLBAR307 with a 8 bp AT-rich SspI linker replacing nrd AT-rich region pLBAR307 with two 8 bp AT-rich SspI linkers replacing nrd AT-rich region Site-directed Fis binding site mutant Mutant defective in both DnaA binding sites pLBAR2 except six G/C bp in AT-rich region pLBAR2 except 16 G/C bp in AT-rich region pLBARF except six G/C bp in AT-rich region pLBARF except 16 G/C bp in AT-rich region pLBAR2 except sequence altered from ¹ 114 to ¹ 123 pLBAR2 except downstream half of inverted-repeat altered pLBARF except downstream half of inverted-repeat altered pLBARF except mutations altering both DnaA binding sites pLBAR2 pLBAR229 pLBAR325 pLBAR310 1 , 2 , 3 1 1 1 pLBAR307 pBJ307S 1 1 pBJ307SS 1 2 , 3 3 2 2 2 2 2 pLBARF pLBAR2D2 pBJ10 pBJ11 pBJ12 pBJ13 pBJ14 3 pBJRS 3 pBJ15 3 pBJ16 Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>Multicopy plasmids carrying oriC and the adjacent mioC gene mitigate the toxicity caused by extra initiations DnaA transcriptionally regulates the mioC gene , which is located to the right of oriC on the genetic map , by binding to specific sites in the mioC promoter region ( Bogan and Helmstetter , 1996 ; Hansen et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Fuchs and coworkers identiﬁed two binding sites for the DnaA protein upstream of the -35 region of the nrdAB promoter ( Augustin et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Three potential DnaA binding sites , each with homology of 8 of 9 bp , are present , two in the aroB promoter region and one just upstream of the dam gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>Three sites of 8/9 homology to the 9 bp site [ TTAT ( C/A ) CA ( C/A ) A ] required for DnaA protein binding ( Fuller et al. 1984 ) are present , two in the aroB promoter region and one just upstream of the dam gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>Here are represented binding sites for IciA , Fis , DnaA and NrdR ( Augustin et al. , 1994 ; Han et al. , 1998 ; Rodionov and Gelfand , 2005 ; Torrents et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>SeqA and DnaA exert their direct function in the regulation of pR activity by binding to unusually positioned sites -- downstream of the transcription start ( Szalewska-Pałasz , et al. , 1998 ; Słomińska et al. , 2001 , 2003a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Indicated also are the UP element and discriminator sequence ( involved in growth-rate-dependent regulation and stringent control ) , sites identi cents ed for the binding of PurR and DnaA and the putative CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Putative FIS sites I -- IV , FIS sites 1 -- 3 , a putative CRP binding site centred at position '' 117.5 and binding sites for PurR and DnaA are indicated ( Hutchings & Drabble , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>In addition , binding of DnaA to some but not all binding sites located within genes terminates transcription ( Messer & Weigel , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>DnaA is known to act as a repressor for various genes by binding to DnaA boxes located within their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Positions of single base changes pertinent to this study are indicated by an X below the substituted base , with the introduced base shown below the X. For clarity , binding sites for FIS and DnaA are not shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>Alternatively , SeqA could compete with DnaA for binding sites or interfere directly with the building of the DNA-bound DnaA complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>It might be significant that potential DnaA binding sites ( i.e. , sequences homologous to the DnaA binding sites in oriC with 7/9 or 8/9 matching scores ( 4 ) ) were identified in the vicinity of these GATC clusters ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054812" target="_blank">3054812</a>]<br></li>
<li>Part of oriC sequestration is caused by DnaA OriC has many binding sites for DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>DnaA protein binding sites outside oriC are not indicated in the figure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>NrdR , like DnaA , binds ATP and dATP and controls the transcription of RNR genes by binding to speciﬁc target sites i the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>DnaA bound at the high-affinity sites ( grey ovals ) , in either its ATP or ADP-bound form , binds to the upstream , high-affinity , sites ( a ) and stabilizes RNAP binding ( white oval ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA bound to the high-affinity sites , in its ATP or ADP-bound form , stabilizes RNAP binding at the nrdAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA binding to its sites on this promoter results in protection from nucleotides -58 to -48 ( DnaA box1 ) and from nucleotides -40 to -30 ( DnaA box2 ) ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Quantitative Figure 1 Sequence of the E. coli dnaA promoter region ( dnaAp ) and speciﬁc mutations of the DnaA sites ( A ) dnaAp contains multiple DnaA-binding sites [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The nrd promoter region contains binding sites for DnaA and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>Black arrowheads show the positions of the polA promoter and the off370 promoter , respectively.The positions of the DnaA - and LexA-binding sites are symbolized by dotted boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>A search for FIS boxes revealed that there are two putative binding sites for FIS upstream of the polA promoter near the functional DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>These results suggest that DnaA binds cooperatively to two sites immediately upstream of the - 35 RNA polymerase-binding site and that this binding results in increased nrd transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>These results are in agreement with previous studies [ 17,20 ] and show that , as the DnaA -- ATP concentration is increased , binding to the box 1 site is followed by the co-operative formation of a higher-order oligomeric structure spanning the adjacent sites and , at higher concentration , by the protection of the core promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>These sites are found adjacent to the DnaA binding box , R1 , and overlap the integration host factor binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>This is in agreement with previous results obtained with the same plasmid containing the nrdAB promoter sequence with a similar number of DnaA sites [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Cultures containing an nrd-lac fusion in which these sites are mutated so that DnaA no longer binds have decreased lac expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>We report here that DnaA binds to these two sites in the nrd promoter region and that mutations in these sites reduce the expression of an nrd-lac fusion gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Here , we report that DnaA binds cooperatively to two sites adjacent to the nrd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Both forms of DnaA show protection and hypersensitive sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA was shown to bind co-operatively to these two sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>DnaA box complex serves a second function by acting as a transcription terminator in the intergenic region between asnC and mioC [ 41 , 421 , upstream to dam [ lo ] and within dnaA [ 18 , 431 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Hda , a protein related in amino acid sequence to DnaA itself ( Kato and Katayama , 2001 ) , in conjunction with the replication processivity clamp β ( Katayama et al. , 1998 ) , binds and inactivates DnaA in vitro by promoting ATP hydrolysis , a mechanism termed ` RIDA ' , for regulatory inactivation of DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Messer et al. [ 7 ] proposed that when DnaA binds to a second DnaA box , within the dnaAcoding region , it acts as a transcriptional terminator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Moreover , promoterftsZlp ( Table II ) contains a sequence similar to the consensus DnaA box ( Fuller et al. , 1984 ) and shows a very high homology particularly to the DnaA box responsible for autoregulation in the dnaA promoter region ( Braun et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1698623" target="_blank">1698623</a>]<br></li>
<li>The potential boxA sequence has been identified before ( Landick et al. 1984 ) as has the putative DnaA box ( Gill et al. 1986 ) although the functional significance , if any , of such structures is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3550388" target="_blank">3550388</a>]<br></li>
<li>Weak interaction to sites deviating from this sequence has been observed ( Fuller et al. , 1984 ; Yung & Kornberg , 1989 ; Schaefer & Messer , 1991 ) and a relaxed DnaA box sequence proposed ( Schaefer & Messer , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Box 4 was deﬁned by Hansen et al. [ 32 ] and overlaps with the box a DnaA -- ATP site as deﬁned by Speck et al. [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>® A199 ) - ® CSH50 ( pFAL1 , pROA200 ) - AKK1 ) CSH50 ( pFAL1 , pRO CSH50 ( pFAL1 , pRO ® ® AKK-EX ) DISCUSSION DnaA protein usually recognizes a 9 bp sequence ( DnaA box ) , TTAT ( C -RCB- A ) CA ( C -RCB- A ) A ( Fuller et al. , 1984 ) or TT ( A -RCB- T ) TNCACA ( Schaper & Messer , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Up to 30 monomers of DnaA protein bind to a dnaA box ( Fuller et al. 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>Hybridization temperature was 51 ° C. 1 , 838 bp probe ; 2 , RNA from strain CM987 ; 3 , RNA from strain CM987/pLSK5 , not induced ; 4 , RNA from strain CM987/pLSK5 , 90 min induced with 1 mM IPTG ; 5 , ~ bX174/HaeIII marker Influence of DnaA protein on the termination of the asnC transcript A binding site for DnaA protein , dnaA box , is located between P , , ioC and T , ~ nc ( Fig. 6 , Hansen et al. / 982 ; Fuller et al. 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>, U/C C.G step asn C u. ~ I I A.U I I The influence of DnaA protein on the termination of the asnC transcript Between T ° s , c and PmioC , a binding site for DnaA protein , dnaA box ( Hansen et al. 1982 ; Fuller et al. 1984 ) , is located ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>DnaA protein binds to two DnaA boxes located between positions ) 35 and ) 55 ( Augustin et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>Basically , DnaA binds to DnaA boxes even at 0 ° C ( Obita et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>For this reason , we usedfragment-retentionassays ( Fuller et al. , 1984 ) to w y an aligned E. coli DNA library ( Kohara et al. , 1987 ) for DNA sequences in which DnaA proteinbinds in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>It is known that a 9 bp sequence , which binds DnaA when located in certain surrounding sequences does not bind it in other places ( Fuller et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Keywords : DnaA , aldA , transcription , replication , Escherichia coli INTRODUCTION The DnaA protein of Escherichia coli is the initiator protein of chromosomal replication and binds to the ﬁve 9 bp repeats termed DnaA boxes , TTAT ( C -RCB- A ) CA - ( C -RCB- A ) A ( Fuller et al. , 1984 ) or TT ( A -RCB- T ) TNCACA ( Schaper & Messer , 1995 ) , in the replication origin , oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>At oriC , 20 -- 40 DnaA monomers bind to ﬁve asymmetric 9 bp consensus sequences [ TTAT ( C/A ) CA ( C/A ) A ] , referred to as DnaA-binding boxes , to open the DNA duplex at the 13-mer AT-rich sites to form the open complex ( Fuller et al. , 1984 ; Langer et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>The first step in the initation process is the binding of the initiator protein DnaA to its four recognition sequences ( DnaA boxes ) within oriC ( Fuller et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>The first and most crucial stage appears to be DnaA protein binding to the 9 bp DnaA recognition sites ( Fuller et al. , 1984 ) and the simultaneous formation of a initial complex where the DNA is wrapped around the DnaA protein core ( Funnell et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2257623" target="_blank">2257623</a>]<br></li>
<li>Usually 2 µg of an active HDM protects the probe at both 13-mer regions and R1 DnaA box [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572297" target="_blank">10572297</a>]<br></li>
<li>As seen in Fig. 3 , DnaA was only able to protect a region of = 222-231 bp from the 5 ' - end-labelled Hindlll site ( blunted Mlul of polA into the Smal site of pNM480 +27 bp up to the 5 ' - end-labelled G of the Hindlll site ) that is equivalent to the position of the second DNA box ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>As the box that is recognized by DnaA lies 66bp upstream from the promoter defined above , these results strongly suggest that DnaA may stimulate polA transcriptionby bindingto this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>The DNase I footprinting experiments show that DnaA -- ATP binds to the dnaA promoter protecting the DNA from − 73 to − 25 of dnaAP2 , and resulting in one strong hypersensitive site at the upstream edge of box 1 , indicative of DNA bending at this site ( Figure 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The box near nt 390 encloses the proposed DnaA recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3074007" target="_blank">3074007</a>]<br></li>
<li>Experiments presented betow ( Table 4 ) show that this is not due to presence of the 76 bp pBR322 fragment , but to the presence of E. coli sequences , most likely including the 8/9 DnaA binding site found just upstream of the dam gene ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>Transcription in a dnaA46 strain It has been reported that the dam gene is negatively regulated by the DnaA protein [ 28 ] presumably through DNA binding of the protein at a putative DnaA box located about 30 nucleotides upstream of the ATG codon ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1581360" target="_blank">1581360</a>]<br></li>
<li>The binding of such clusters of DnaA protein may explain the observation that termination is influenced at a distance of up to 80 bp away from the dnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2464126" target="_blank">2464126</a>]<br></li>
<li>In addition , we identify a DnaA box ( binding site for DnaA protein ) , a potential pur operator , and a site with dyad symmetry which may be a candidate p z operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>DNase I footprints of DnaA box sequences in the EcoRI-Hind111 fragment of pFT103 in the presence of DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Results DnaA protein binding by a DNA , fragment originating from 94.7 min on the E. coli genetic map Computer searches of the database have revealed many DnaA-box consensus sequences on !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>The presence of two DnaA-binding sequences is required for an eficient interaction of the Escherichia coli DnaA protein with each particular weak DnaA box region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Additionally , to examine whether the DnaA box is responsible for the enhancer effect we constructed a deletion in pNH3 removing all sequences upstream of the Bglll site ( pNH3-1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DARS-like DnaA box clusters are seen in genomic sequences of many eubacterial species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Underlined sequences indicate a putative DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1698623" target="_blank">1698623</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1698623" target="_blank">1698623</a>]<br></li>
<li>Assuming the same affi-nity of DnaA protein for the sequence on the plasmid and that on the chromosorne , about 60 % and 8 % of the total DnaA protein in a plasmid-free cell are bound to this site at 94.7 min on the genetic map and the oriC-PmioC region , respectively.We also observed that DNA contents in cells carving plasmids which titrate high levels of DnaA protein were 50 to 60 % of the normal level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>It is possible that imperfect ATP-DnaA boxes may be found at most GATC sites , since much of the sequence requirement is made up by GATC itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>DnaA was found to bind at both boxes in a cooperative manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>In some cases in which the DnaA boxes are located in , or close to , the promoter sequence , transcription is inhibited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>~ the DNA region containinb four DnaA boxes inhibited transcription which starts in this / area .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>DnaA binds to the DnaA box near the polA promoter Two putative DnaA boxes exist in the extended promoter region of polA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Results and Discussion Binding of DnaA and SeqA to the pR promoter region It was found previously that both DnaA and SeqA proteins bind to the replication region of bacteriophage l DNA .16,27 However , DnaA boxes necessary for DnaA-mediated activation of p bind R DnaA weakly , as they differ from the consensus DnaA box sequence in a few positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Upstream of the DnaA boxes ( ¹ 57 to ¹ 68 ) is an 11 bp direct repeat ( with a 1 bp deletion ) of a sequence that occurs 60 bp upstream ( ¹ 117 to ¹ 128 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>This specific sequence ( termed DARS , for DnaA-reactivating sequence ) lies in the ColE1 plasmid replication origin and contains three DnaA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>Between Ap and Xp are three potential DnaA boxes ( each with seven of nine matches to the consensus sequence ) ; two of these were pointed out by Flower and McHenry ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7691799" target="_blank">7691799</a>]<br></li>
<li>DnaA boxes that perfectly match the proposed consensus sequence have been identiﬁed on a B. aphidicola plasmid carrying trpEG ( 36 ) , indicating that this element is potentially conserved in Buchnera .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9244264" target="_blank">9244264</a>]<br></li>
<li>The region protected by DnaA from cleavage by DNase I ± the nucleotide sequence from ) 35 to ) 55 ± overlapped with that of RNA polymerase , which extends from approximately +1 to ) 50 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>Removal of this sequence from minichromosomes is known to result in a lowered copy number ( L0bner-Olesen et al. , 1987 ) , probably resulting from a less efficient binding of DnaA protein to oriC ( Hansen et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>A potential DnaA protein binding site , of homology 8/9 to the TTAT ( C/A ) CA ( C/A ) A consensus sequence ( Fuller et al. 1984 ) , is present immediately 5 ' to the dam gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2549371" target="_blank">2549371</a>]<br></li>
<li>In this region a possible DnaA-binding site is identiﬁed using the Virtual Footprint software ( Münch et al. , 2005 ) according to the DnaA consensus binding sequence ( TTA/TTNCACA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>The sequence in this region ( GTTATACTCTT ) has reasonable homology with the DnaA protein binding site consensus found in the regulatory region of genes involved in DNA replication and repair ( van den Berg et al. 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3550388" target="_blank">3550388</a>]<br></li>
<li>This sequence has homology with the proposed DnaA binding site ( DnaA-box ) found in replication origins , and in the regulatory region of genes known to be important in replication and repair of D N A ( van den Berg et al. 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3025556" target="_blank">3025556</a>]<br></li>
<li>Other regions of potential regulatory significance ( ribosome binding site , boxA sequence , DnaA binding site , 5 S-23 S spacer region homology ) are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3550388" target="_blank">3550388</a>]<br></li>
<li>Furthermore , the -10 region of P1 is located within the sequence highly homologous to the DnaA protein binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>This discrepancy may arise from the fact that , in contrast to the results presented here , interactions of DnaA with boxes of the consensus sequence , and of SeqA with a DNA region rich in GATC sequences ( the oriC region ) were investigated previously .14 Therefore , we decided to compar binding patterns of DnaA and SeqA on the basis of experiments with each protein added separately to DNA samples , rather than to analyse protein -- DNA interactions in the presence of both DnaA and SeqA in the same reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>DNA fragments specifically bound by DnaA protein share a 9 bp consensus sequence ( DnaA Dox ) , TTAT - ( C/A ) CA ( C/A ) A , which is postulated to be recognized by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>DnaA protein/DNA interactions , Modulation of the recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>This sequence has since been shown to be the consensus binding sequence for the bacterial DNA replication initiation protein DnaA ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>protein , recognizing a 9-mer consensus sequence ( 59-DnaA protein is a sequence-speciﬁc DNA binding TTAT ( C/A ) CA ( C/A ) A ) , ﬁve of which are located in the chromosomal origin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>We introduced speciﬁc mutations at most of these sites in order to substitute the bases known to deﬁne the DnaA-binding consensus sequence ( Figure 1B ) and measured the effect that these substitutions have on DnaA -- ATP binding in vitro by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Direct protection by DnaA protein is likely to involve ATP-DnaA boxes , since these sites contain both the GATC sequence and a DnaA binding site ( with consensus AGatct4 ,38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>Gray rectangles represent DnaA boxes with up to one mismatch to the consensus sequence TTATNCACA , while a white rectangle represents the Fis binding site ( 4 ) , and black rectangles represent FNR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>The two nine-base sequences with homology to the DnaA consensus binding sequence , box1 and box2 , are indicated as open boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>Two nine-base sequences with homology to the DnaA consensus binding sequence are located immediately upstream of the nrd putative -35 RNA polymerase binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Schaefer and Messer ( 39 ) showed this to be one of the stronger binding permutations of the DnaA consensus binding sequence ( T/C ) ( T/C ) ( A/T/C ) T ( A/C ) C ( A/G ) ( A/C/T ) ( A/C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Shown above the main sequence are the consensus sequences for CRP binding [ 2 ] , the UP element [ 3 ] , PurR [ 4 ] , DnaA [ 5 ] and the discriminator [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>A box with high affinity for DnaA may direct the protein to adjacent sites which in themselves are not active in binding , but cooperative effects of this type are probably dependent on the exact distance between the sites [ 35 ] and may not extend over the 0.25 kb separating the two boxes in the gua sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>This may be due to variations in DNA sequence between certain boxes and/or to competition between boxes for DnaA binding and inﬂuence of neighbouring DNA regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>The location of these sites relative to the DnaA box , R1 , and the IHF binding sequence coincides with a potential blocking of the initiation of DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>Previous footprinting studies [ 3 , 61 have indicated that DnaA can bind to regions of DNA containing a single copy of the 9-bp box sequence to generate an extensive pattern of cleavage protection and enhancement with the box lying centrally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>DnaA binds to a sequence overlapping the core promoter region and also downregulates transcription from PguaB ( Tesfa-Selase & Drabble , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>Thegzla promoter region has a DnaA box , the binding site for DnaA , with 8/9 match to the consensus , although a consensus sequence box within thegtlaB coding region is also implicated in this regulation ( F. Tesfa-Selase & W. T. Drabble , unpublished results ) The p a promoter lies in the back-to-back arrangement with the promoter for sse ,4 [ encoding the mismatch repair enzyme csonuclease I ` Ilj ( Chase e t a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>DnaA protein was shown to bind specifically only to the fragment carrying the consensus sequence DnaA box , and to protect this box from deoxyribonuclease I. Transcription termination resulting from the binding of DnaA to this box within the guaB gene explains repression by DnaA of the gua operon in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>In this paper we have shown , using the filter retention technique , that purified DnaA protein binds to a 0.31-kb fragment of gua DNA carrying the consensus box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>In this paper , we demonstrate by a combination of specific binding assays and by deoxyribunuclease I ( DNase I ) footprinting , that the consensus sequence DnaA box within the guaB coding region specifically binds DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>DnaA protein binds specifically to the consensus DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Protection from DNase I cleavage is observed within the region of the consensus DnaA box in the guaB coding sequence and shows a pattern of protection from , and enhancement of , cleavage similar to those reported for the DnaA boxes in oriC and dnuA [ 3 , 61 , and in rpoH [ Ill. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>ThepwB N-terminal coding region also contains a DnaA box at position 1382 ( Fig. 3 ) on the non-transcribed strand which differs from the consensus sequence ( TTATC , CAC , A ) at one position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969519" target="_blank">8969519</a>]<br></li>
<li>( A ) Footprint of the upper strand in the region of the consensus sequence DnaA box in guaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>One box ( with 819 match to the DnaA box consensus sequence ) is at the gua promoter ; the other box , which has a consensus sequence , is on the non-transcribed strand within the guaB coding region approximately 200 bp downstream of the initiation codon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>mismatch to the consensus sequence DnaA box , is located at the gua promoter and the other , which conforms to the consensus sequence , is within the 5 ' region of guaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>The protection i s in the region of the consensus sequence DnaA box and was observed at levels of DnaA protein sufficient for specific retention on nitrocellulose filters of fragments containing this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>( A ) The DNA sequence of the mioC regulatory region is shown , including PmioC and the DnaA box ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>The DnaA box sequence is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>This fragment carries the consensus sequence DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>The consensus sequence DnaA box in the guaB gene has this ` transcription termination ' orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Plasmid pALO237 carries a functional but truncated ( by deletion of the mioC promoter and the mioC DnaA box ) oriC sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></li>
<li>DnaA protein afforded no protection to the 8/9 match box sequence at the gua promoter ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>One box overlaps the gua promoter and has a 8/9 match to the DnaA box consensus sequence of TTAT ( Z ) CA ( C , ) A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>This site has the sequence AGATCA and could thus possibly function as an ATP-DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>It has been shown [ 3 , 111 that binding of DnaA influences the footprinting pattern over a region of up to 0.03-0 .04 kb either side of the 9-bp box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>, 1986 ) by binding to a 9 bp consensus sequence motif called a DnaA box ( Fuller et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Despite fairly strong binding , the bound DNA fragment had no consensus 9 bp DnaA binding sequence ( DnaA box ) , and anomalous binding to an AT-rich sequence located close to the transcription start site was revealed by a footprinting experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Characterization of the binding site by a foootprinting experiment Despite the clear binding , the 261 bp TaqI fragment has no canonical 9 bp DnaA box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Approximately 200 bp downstream from the translation initiation codon for guaB is a consensus DnaA binding site that has been shown to bind DnaA in vitro and is required for DnaA-mediated repression of guaB transcription in vivo ( 73 , 74 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633076" target="_blank">19633076</a>]<br></li>
<li>A footprinting experiment suggested that DnaA initially interacts with an AT-rich site between the ® 10 promoter sequence and the transcription start point , leading to cooperative binding to a region of about 100 bp around the site at higher DnaA concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>It seems that DnaA bound ﬁrst at a site between the transcription start site and the ® 10 sequence on the upper strand triggered the cooperative binding upon increase of the number of bound molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>In D2 , both DnaA boxes were mutated by six transversions ( shown in lowercase letters ) , ( - 52 ) 1TAaCgtCAAAcaaATgCACt ( -32 ) ; in UD , the upstream DnaA box was mutated by a single T-to-A transversion at position - 49 , ( - 52 ) TTAaCCACA ( -44 ) ; and in DD , the downstream DnaA box was mutated by a T-to-A transversion at position -40 and a G-to-C transversion at position -41 , ( -41 ) caTATgCACt ( -32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>Black-and gray-filled arrowheads represent DnaA boxes that completely match the 9-mer consensus and that differ from the consensus by a single base , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by DnaA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>RESULTS AND DISCUSSION Construction and Expression of Polyhistidine-Tagged DnaA Protein ( Fig. 1 ) permitted synthesis of DnaA protein with an amino-terminal polyhistidine tag ( termed His10-DnaA ) , which facilitated its easy isolation by immobilized met-al-ion afﬁnity chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>A much less intense band of 53 kDa , detected in samples from both uninduced and induced cultures ( Fig. 2B , lanes 1 and Expression was induced with IPTG , and during a ( Fig. 1 ) permitted synthesis of DnaA protein with an amino-terminal polyhistidine tag ( termed His10-DnaA ) , which facilitated its easy isolation by immobilized met-al-ion afﬁnity chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Black arrowheads show the positions of the polA promoter and the off370 promoter , respectively.The positions of the DnaA - and LexA-binding sites are symbolized by dotted boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>This argues that , although DnaA concentration may be an important parameter in the control of initiation of chromosome replication , it can not be the sole determinant under both the above growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Cellular DnaA content was not significantly affected by the plasmids ( Fig. 4E ) , which supports the hypothesis that DARS1 and DARS2 can promote replication initiation by stimulating DnaA reactivation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>In exponentially growing cultures we did not observe significant changes in polA expressionfollowing DnaA induction by IPTG ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>The accumulation of the active form of DnaA , ATP-DnaA , in the hda mutant is suppressed by the disruption of the ygfZ gene , indicating that YgfZ is involved in regulating the level of ATP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>C , control using pre-immune serum ; αA , anti-DnaA antiserum.The total cellular levels of the nucleotides bound to DnaA were not signiﬁcantly affected by the ygfZ mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>The cellular level of DnaA is not affected signiﬁcantly in the ygfZ mutant , as estimated by Western blotting ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359333" target="_blank">16359333</a>]<br></li>
<li>, 1995 ) and the nrd operon , where FIS ( factor for inversionstimulation ) and DnaA positively influence nrd transcription ( Augustin eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Because of the similarity to the situation in the nrd operon that is modulated by FIS and DnaA together ( Augustin eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Termination of transcription was shown to be orientation dependent [ 18 ] ; a DnaA box located on the non-transcribed strand , which is orientated in the direction of transcription , is necessary for transcription termination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>An additional implication of these results is that dnaAP1 transcription is not repressed by DnaA -- ATP bound on the sites downstream of its initiation site and overlapping the dnaAP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Interestingly , for regulation of pR activity , binding to sequences located downstream of the promoter is necessary in the case of both DnaA and SeqA ,16,22 which is unusual in prokaryo-tic transcription control systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>( j ) minichromosome was not included ; ( r ) DNA synthesis by 1 pmol of ATP-DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>172 : 1392 -- 1399 42 Smith R. W. P. , McAteer S. and Masters M. ( 1996 ) The coupling between ftsZ transcription and initiation of DNA replication is not mediated by the DnaA-boxes upstream of ftsZ or by DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9614967" target="_blank">9614967</a>]<br></li>
<li>As a control , we verified that DnaA protein was induced to similar levels relative to the abundance of ribosomal protein L2 in portions of the same cultures that were analyzed by quantitative PCR ( Figure 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432804" target="_blank">19432804</a>]<br></li>
<li>Relative to an internal control of the ribosomal protein L2 , the abundances of DnaA encoded by a dnaA plasmid ( pDS596 ) after 1 h of induced expression were comparable in various strains ( XL1-Blue [ recA1 ] , MG1655 , MF1341 [ rcbA : : kan ] , or JJC315 [ recB268 : : Tn10 ] ) , regardless of whether they carried an rcbA plasmid ( pMMF-D4 ) , pACYC184 , or no other plasmid ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>The transcriptional regulation of these genes requires several transcription factors ( including DnaA , IciA , FIS [ factor for inversion stimulation ] , Fnr , Fur , and NrdR ) , depending on the RNR class ; however , the factors that dictate the expression of some RNR genes in response to different environmental conditions are not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>In this bacterium , the expression of RNR genes can be positively or negatively regulated at the transcriptional level ; nrdAB expression is repressed by the NrdR protein , whereas the DnaA , IciA , and FIS ( factor for inversion stimulation ) transcriptional regulators induce the expression of this operon ( for a review , see reference 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Whereas surplus DnaA should also repress the expression of the nrdAB-yfaE operon , this has no detectable effect on the cellular abundance of ribonucleotide reductase ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>The nrdAB operon is transcribed from two promoters ( 7 ) , and transcription is activated by Fis ( 4 ) and IciA ( 12 ) and probably also by DnaA , because mutations in the two DnaA binding sites in the promoter region reduce expression ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>The nrdAB operon is transcriptionally activated by FIS , IciA , and DnaA and is cell cycle regulated ; furthermore , its expression is not dependent on Arc or Fnr ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Together these results show that the coupling of nrdAB expression with the time of replication initiation is not determined by DnaA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>However , transcription regulation by DnaA does not signiﬁcantly affect the timing of expression of RNR from the nrdAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>These results allow us to propose a model for nrdAB gene regulation by which DnaA can inﬂuence the total amount of RNR expression but not its timing as a function of DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>In addition to its role in initiating DNA replication , DnaA is a transcriptional regulator that controls positively or negatively the expression of nrdAB by binding to DnaA boxes immediately upstream of the nrdAB operon ( 32 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Modulation of expression of the gua operon by DnaA may be regarded as one of these controls serving to coordinate guanine nucleotide biosynthesis with the DNA replication cycle [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Also , IciA transcriptionally activates expression of the dnaA and nrd genes , which encode the chromosomal replication initiator DnaA and ribonucleoside diphosphate reductase , respectively ( Lee et al. 1996 ; Han et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Discussion DnaA is both an activator and a repressor of RNR expression In 2006 Gon et al. showed that DnaA represses transcription from the nrdAB promoter controlling expression of the major aerobic ribonucleotide reductase gene in E. coli ( Gon et al. , 2006 ) while earlier the Fuchs laboratory had shown an activating role for DnaA at the same promoter ( Augustin et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>In addition to its role in initiation of replication , DnaA has been shown to autoregulate its own transcription ( Sekimizu and Kornberg , 1988 ) and to control either positively or negatively the expression of several other genes ( van den berg et al. , 1985 ; Wang and Kaguni , 1989 ; Zhou and Syvanen , 1990 ; Mizushima et al. , 1994 ; Skarstad and Boye , 1994 ; Kitagawa et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>DnaA is also a transcription factor that negatively regulates the expression of several other genes by binding to DnaA boxes located in their promoter regions ( reviewed by Skarstad and Boye , 1994 ; Messer and Weigel , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DnaA is known to act as a repressor for various genes by binding to DnaA boxes located within their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Apart from DnaA , expression of other genes are a $ ected by alterations in phospholipids [ 370 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>DnaA is involved in the expression of many genes [ 28 ] and , in our hypothesis , in the build-up to initiation , DnaA accumulates due to the disappearance of hyperstructures that contained it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12106913" target="_blank">12106913</a>]<br></li>
<li>DnaA is known to be a transcription factor ( Messer and Weigel 1997 ) ; thus , one might assume that the expression of genes coding for components of the membranes or for proteins involved in synthesis of such components might be impaired in dnaA mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10394913" target="_blank">10394913</a>]<br></li>
<li>In addition DnaA also acts as a transcription factor for a large set of genes , possibly acting as a co-ordinator of the gene expression and DNA replication programmes in response to perturbations in DNA synthesis ( [ 4,5 ] and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>DnaA also controls the transcription of fadL , fabF and other genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In exponentially growing cells poIA expression is maximally induced ; therefore we did not observe increased po/A expression by DnaA overproduction at this stage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>DNA initiation increase in parallel in synchronized exponentially growing cells , and DnaA appears to be involved in the regulation of both , it is surprising that the DnaA-binding boxes do not appear to be required for increased nrd expression during DNA inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8288532" target="_blank">8288532</a>]<br></li>
<li>With a dnaA allele ( T435M ) that replaces threonine 435 with methionine and is defective in both the recognition of the DnaA boxes in oriC and initiation ( 90 ) , its expression did not reduce viability ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>A reduction of = 50 % in polA expression was observed in the dnaA mutant suggesting that DnaA also contributes to polA expressionin growing cells ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Previous work had shown that DnaA -- ATP represses expression from both dnaAP1 and dnaAP2 [ 17,20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The promoter region also contains DnaA boxes that are recognized by DnaA-ATP to control dnaA expression by autoregulation ( 3 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22343303" target="_blank">22343303</a>]<br></li>
<li>Two transcripts start from the two dnaA promoters which are negatively controlled by the DnaA protein ( Atlund et al. 1985 ; Braun et al. 1985 ; Kiicherer et al. 1986 ) and these seem to be extended through dnaN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851700" target="_blank">2851700</a>]<br></li>
<li>When the intracellular concentration of DnaA is increased ( by induction of a multicopy plasmid carrying the dnaA gene fused to a tac promoter ) , expression of the gua operon is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>High intracellular concentrations of DnaA ( achieved by inducing a multi-copy plasmid carrying the dnaA gene fused to the t a p ~ romoter ) reduce expression of the p a operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Since the dnaA gene is autoregulated , aseqA mutation could increase dnaA expression independent of sequestration if SeqA affects DnaA activity per se ( below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>Before initiation both dnaA gene expression and the recycling of DnaA -- ADP into DnaA -- ATP [ 13 ] contribute to the increase in the amount of DnaA -- ATP in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>Overexpression of IciA protein enhances expression of the nrd gene , while overexpression of DnaA protein does not The recombinant plasmid pDS596 contains the dnaA ORF under control of the inducible araB promoter of vector pING1 ( Johnston et al. 1985 ; Hwang and Kaguni 1988 ; Hwang et al. 1990a ; Lee et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9819053" target="_blank">9819053</a>]<br></li>
<li>The factor controlling the frequency of initiation from oriC seems to be the availability of active DnaA protein , since a good correlation between the dnaA gene expression and the time of initiation in the cell cycle has been established ( L0bner-Olesen et al. , 1989 ; Atlung and Hansen , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>In dnaA mutants of E. coli , the expression of ,3 - galactosidase in a drpA ' - lacZ protein fusion is eight times higher than its expression in a cell that overproduces DnaA protein ( T. Luby , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1688424" target="_blank">1688424</a>]<br></li>
<li>We are currently altering the DNA sequence of the dnaA box by site-di-rected mutagenesis in order to evaluate further the effect of DnaA protein on gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1581360" target="_blank">1581360</a>]<br></li>
<li>It has been reported that the DnaA protein can modulate gene expression by blocking transcribing RNA polymerase at dnaA boxes [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1581360" target="_blank">1581360</a>]<br></li>
<li>Nontagged DnaA protein was puriﬁed ( 32 ) from BL21 ( DE3 ) pLysS / pKA211 , a transformed strain in which inducible expression of the dnaA gene is under the control of a bacteriophage T7 RNA polymerase promoter ( T. Kata-yama , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>Results DnaA overproduction leads to enhanced polA gene expression The gene expressionof polA was monitoredby two different translationalfusions of polA to lacZ ( pNH3 and pNP4 ; Fig. 1 ) using the vectors developed by Minton ( 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Negative control of aldA gene expression by DnaA In order to examine the role of DnaA in aldA gene expression , the regulatory region and N-terminal three codons of the gene ( ® 184 to 52 ) was fused in-frame to the lacZ gene on a mini-F plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Negative control of aldA gene expression by DnaA was suggested from the derepression assay ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>These results suggest that the activation by DnaA and the basal polA gene expressionare , at least partially , under stringent control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>These results suggest that upstream sequences beyond the Bglll site are not only essential for the DnaA effect , but also to maintain the correct level of polA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_61">
These results suggest that DnaA protein negatively regulates expression of the aldA gene under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></label>
<input id="tu_similarity_61", type="checkbox">
<div class="TU_similar">
<li>These results suggest that DnaA negatively regulates the aldA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>However , the polA gene expression and the enhancer effect by DnaA were modulated by ppGpp which plays an important role in the regulationof cell growth and cell division ( Schreiber et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Here , we describe studies of polA gene expression that suggest a positive regulatory role of the initiator protein DnaA in the expression of Pol I by interaction with a DnaA box located upstream of the polA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>To determine the influence of increased levels of DnaA protein on polA gene expression , bacteria harbouring pNH3 and pHK27 or pACYC184 as control were grown to an ODeo0of approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Therefore , we performedcomparative S1 analyses to localize the transcription-initiationsites of polA and to demonstrate that the increased polA gene expression by DnaA overproduction is due to enhanced transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Influence of DnaA overproduction on polA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Additionally , in a following study it was demonstrated that CAMP and the CAP protein repress spf transcription of Stimulation polA expression by DnaA is Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9106210" target="_blank">9106210</a>]<br></li>
<li>Keywords : guaBA operon ; transcriptional control ; DnaA protein ; Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Regulation of the gan operon of hcherichia coii by the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>This suggests the possible coupling between rpoH transcription and expression of the cell division operon mediated by the DnaA protein ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2139650" target="_blank">2139650</a>]<br></li>
<li>DISCUSSION We have previously shown [ 19 ] that expression of the gua operon in vivo is modulated by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Therefore it is unlikely that the overall regulation of the dam gene is affected by the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630320" target="_blank">1630320</a>]<br></li>
<li>Transcription in a dnaA46 strain It has been reported that the dam gene is negatively regulated by the DnaA protein [ 28 ] presumably through DNA binding of the protein at a putative DnaA box located about 30 nucleotides upstream of the ATG codon ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1581360" target="_blank">1581360</a>]<br></li>
<li>It has been reported that the dam gene is negatively regulated by the DnaA protein ( Jonczyk ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630320" target="_blank">1630320</a>]<br></li>
<li>van den berg , E.A. , Geerse , R.H. , Bovenberg , R.A.L. , Magnee , F.A. , and van de putte , P. ( 1985 ) Analysis of regulatory sequences upstream of the E. coli uvrB gene ; involvement of the DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>DnaA protein regulates transcription of the rpoH gene of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_78">
J Bacteriol 171 : Wang Q & Kaguni JM ( 1989b ) DnaA protein regulates transcription of the rpoH gene of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></label>
<input id="tu_similarity_78", type="checkbox">
<div class="TU_similar">
<li>Wang , Q.P. , and Kaguni , J.M. ( 1989b ) DnaA protein regulates transcription of the rpoH gene of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Wang , Q. , and Kaguni , J.M. ( 1989 ) DnaA protein regulates transcription of the rpoH gene of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li></div>
<li>Polaczek , P. , and Wright , A. ( 1990 ) Regulation of expression of the dnaA gene in Escherichia coli : role of the two promoters and the DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>DnaA protein stimulates polA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12742018" target="_blank">12742018</a>]<br></li>
<li>Quinones , A. , Wandt , G. , Kleinstauber , S. , and Messer , W. ( 1997 ) DnaA protein stimulates polA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411720" target="_blank">10411720</a>]<br></li>
<li>Transcriptional repression of the dnaA gene of Escherichia coli by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10600368" target="_blank">10600368</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_85">
Hansen , F.G. , Koefoed , S. , S0rensen , L. and Atlung , T. ( 1987 ) Titration of DnaA protein by oriC DnaA-boxes increases dnaA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8918477" target="_blank">8918477</a>]<br></label>
<input id="tu_similarity_85", type="checkbox">
<div class="TU_similar">
<li>Titration of DnaA protein by oriC DnaA-boxes increases dnaA gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li></div>
<li>Transcription of the dnaA gene is autoregulated , and ATP-DnaA represses about twofold more tightly dnaA transcription than ADP-DnaA in vitro ( Speck et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19401329" target="_blank">19401329</a>]<br></li>
<li>dnaA transcription is autoregulated by DnaA protein , and the nucleotide forms of DnaA affect its repressor activity in vitro ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>Indeed , dnaA transcription is negatively autoregulated by DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>The expression of the DnaA protein is autogeneously regulated by binding to its own dnaA box between the two Offprint requeststo : A. Quifiones 1984 ; Armengod and Lambies 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2851700" target="_blank">2851700</a>]<br></li>
<li>The dnaA gene is auto-regulated by the binding of DnaA to a DnaA box located between its two promoters [ 6 , 15-17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>The gene dnaA , which encodes E. coli DnaA transcription dual regulator and is not subjected t variable expression , i.e. abundant expression at relatively constant rate in most cells , was used as an internal control for the RT-PCR determinations [ 68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
<li>Pérez - Roger , I. , Macián , F. and Armengod , M.E. ( 1995 ) Transcription termination in the E.coli dnaA gene is not mediated by the internal DnaA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9501104" target="_blank">9501104</a>]<br></li>
<li>It has been demonstrated that a DnaA box in the direction of transcription can attenuate dnaA transcription as well as that of a downstream gene ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7691799" target="_blank">7691799</a>]<br></li>
<li>The consensus sequence DnaA box in the guaB gene has this ` transcription termination ' orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Transcription repression by DnaA has also been demonstrated for the promoters of mioC [ 8,22,37-401 , rpoH [ 11 ] , and the uvrB gene [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>However , it is important to point out that the p3rpoH-lacZ fusion does not contain any of the other promoters of the rpoH gene , but it does contain the two DnaA boxes which are located upstream of the rpoH p3 promoter ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8478327" target="_blank">8478327</a>]<br></li>
<li>It has been shown previously that DnaA protein negatively regulates the expression of the rpoH gene , which is consistent with the presence of two DnaA boxes in close vicinity to the p3rpoH promoter region ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8478327" target="_blank">8478327</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_99">
Since high-level expression of wild-type DnaA protein is toxic to cells ( data not shown ) , the `` leakiness '' of the T7lac promoter was controlled by using as a host strain , BLR ( DE3 ) pLysS Insertion of the dnaA gene into the pET-19b vector His10-DnaA protein was expressed with a bacterio ¬ Other Methods ing assay ( 31 ) , using BSA as a standard .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></label>
<input id="tu_similarity_99", type="checkbox">
<div class="TU_similar">
<li>Since high-level expression of wild-type DnaA protein is toxic to cells ( data not shown ) , the `` leakiness '' of the T7lac promoter was controlled by using as a host strain , BLR ( DE3 ) pLysS Insertion of the dnaA gene into the pET-19b vector His10-DnaA protein was expressed with a bacterio ¬ 2 ) , represents the bacterial chromosome-encoded DnaA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li></div>
<li>In cells growing in minimal medium containing glucose , expression of β-galactosidase activity from an aldA -- lacZ fusion gene was suppressed by oversupply of DnaA protein and was enhanced by reducing the free DnaA level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>In M9 medium containing rhamnose in place of glucose and supplemented with Casamino acids , or in L-broth , where expression of the gene is induced , neither increase nor decrease of cellular DnaA concentration affected the level of β-galactosidase activity ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>When SeqA is defective , the otherwise a. dnaA / 6/was grown at 30 C and 42 C since it was not temperature sensitive af 41 C. non-functional DnaA ( Ts ) protein is able to initiate replication at42 `` C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891562" target="_blank">7891562</a>]<br></li>
<li>\ Ne found that the chromosomal locus between the glyV and amiB-mutL operons has unusually high affinity for DnaA protein in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8830270" target="_blank">8830270</a>]<br></li>
<li>The putative DnaA box ( with 8/9 match ) overlapping the gua promoter does not appear to interact with DnaA protein even when it is located on the same fragment as the consensus sequence box , as in the footprinting experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Expression was induced with 1 % arabinose , and during 1 h incubation a t 37OC , DnaA protein accumulated to about 15 % of the entire cell protein ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9125116" target="_blank">9125116</a>]<br></li>
<li>The sequence encoding for His10-DnaA protein in the ﬁnal construct of pZL411 was conﬁrmed by DNA sequencing ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>241 , 411-416 ( 1996 ) 0FEBS 1996 Specific binding of DnaA protein to a DnaA box in the guaB gene of Escherichia coli K12 Fisehaye TESFA-SELASE and William T. DRABBLE Department of Biochemistry , University of Southampton , England ( Received 19 April/30 July 1996 ) - EJB Expression of the guaBA operon of Escherichia coli is regulated by the DNA replication-initiating protein , DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8917437" target="_blank">8917437</a>]<br></li>
<li>Second , the only negative control known to be exerted on initiation of rpoH transcription is by DnaA , the key regulatory protein for DNA replication ( Wang and Kaguni , 1989b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767576" target="_blank">9767576</a>]<br></li>
<li>Transcription from the guaBA promoter is sensitive , through repression by the DnaA protein , to the demands for guanine nucleotides for DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The band intensities were normalized by the intensity of a reference band that does not change as a function of DnaA concentration and relative to the intensity of the band in the absence of protein [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23092251" target="_blank">23092251</a>]<br></li>
<li>The gua operon is also regulated by the DNA replica-tion-initiating protein DnaA [ 14,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>The gzta operon encoding two enzymes specificfor GMP biosynthesis is repressed , in addition , by DnaA protein ( Tesfa-Selase & Drabble , 1992,1996 ) and is regulated by stringent and growth-rate-dependent controls ( Davies & Drabble , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8969519" target="_blank">8969519</a>]<br></li>
<li>have reported that thegzacrBA operon is also regulated We by DnaA protein [ Tesfa-Selase & Drabble , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Expression of the operon is sensitive to changes in cellular DnaA protein concentration ( Tesfa-Selase & Drabble , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>