<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - ExuR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">ExuR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">ExuR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>mparative Genomic Analysis of the Hexuronate Metabolism G and Their Regulation in Gammaproteobacteria † Co enes Inna A. Suvorova ,1 Maria N. Tutukina ,2 Dmitry A. Ravcheev ,1,3 Dmitry A. Rodionov ,1,3 Olga N. Ozoline ,2 * and Mikhail S. Gelfand1 ,4 * Institute for Information Transmission Problems RAS ( The Kharkevich Institute ) , Moscow 127994 , Russia1 ; Institute of Cell Biophysics RAS , Pushchino , Moscow Region 142290 , Russia2 ; Sanford-Burnham Medical Research Institute , La Jolla , California 920373 ; and Faculty of Bioengineering and Bioinformatics , Moscow State University , Moscow 119991 , Russia4 Received 25 February 2011/Accepted 13 May 2011 The hexuronate metabolism in Escherichia coli is regulated by two related transcription factors from the FadR subfamily of the GntR family , UxuR and ExuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Teleologically , the ExuR repressor protein should be synthesized in the presence of a better substrate ( glucose ) to ensure effective repression of the hexuronate degradative pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6808145" target="_blank">6808145</a>]<br></li>
<li>ExuR , another Abbreviations : Ap , ampicillin ; bp , base pair ( s ) ; CAMP , cyclic AMP ; Cm , chloramphenicol ; CRP , CAMP receptor protein ; kb , repressor involved in the regulation of the hexuronate system genes , has no effect on the expression of uidA ( Ritzenthaler et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>As was mentioned earlier , the typical distance between GT and AC in most FADR-subfamily TF-binding sites is 3 nt ( DgoR , ExuR , FadR , GlcC , LldR , PdhR etc ) , although in several orthologous groups it is 2 nt ( e.g. , GntR , HpxS , HypR , MdcY , PrpR , UxuR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>The isolation of negative dominant mutations affecting the exuR regulatory gene indicates that the active form of the ExuR repressor is multimeric like that of UxuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>It seems that overproduction of negative dominant UxuR monomers can interact with the repressor activity of the regulatory mole-cule encoded by the chromosomal exuR gene and vice versa for the negative dominant ExuR monomers relative to the UxuR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>The uxuB derepression is about twice that observed in a chromosomal exuR negative mutant strain and therefore , can not be explained solely by the inactivation of the ExuR repressor , resulting from the aggregation of wild-type ExuR and negative dominant ExuR subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>The greater synthesis of the negative dominant U x u R subunits lead to an increased concentration of UxuR-ExuR hybrid molecules ; the titration of the wild-type ExuR monomer was in this case strong enough to lead to weak derepression of the exu operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>ExuR regulates expression of the uxaCA , uxaB and exuT operons involved in galacturonate catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Repressor : b-glucuronide induction b-Glucuronide hydrolase b-Glucuronide transport kdgT eda exuR exuT uxaCA ExuR uxaB 4552.6 4549.5 4551.2 UxuR 58 58 58 uxuR uxuA uxuB uxuR uxuAB gntP uidR 4548.3 1694.8 1693.2 1691.6 38 11 11 11 gntP uidR uidA uidB UidR uidAB OTHER PROPOSED ROLES FOR THE ED ENZYMES A growing number of studies have shown that eda expression is regulated in response to several conditions in addition to its role in sugar acid catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>ExuR negatively controls most of * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Even though a detailed molecular mechanisms underlying UxuR / ExuR regulatory action remains to be elucidated , it seems likely that yjjM and yjjN are regulated by at least UxuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>We have previously shown that the UxuR repressor was unable to repress the exu genes even in the presence of a large amount of repressor , probably because the exu operator sites are only recognized specifically by the ExuR repressor ( Ritzenthaler et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>In a wild-type exuR + strain carrying plasmid pRU12 ( uxuR + ) , the large amount of UxuR + repressor subunits caused an increase in the hybrid ExuR-UxuR molecule concentration and therefore a decrease in the concentration of the active ExuR repressor , but a sufficient amount of the homogeneous ExuR repressor remained in the cell to repress totally the expression of the exu regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>In contrast , the UxuR and ExuR repressors failed to repress the expression of the hybrid 1acZgene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>This is probably also the case for the repressor domains implied in the binding of the inducer since mannonic amide , a gratuitous inducer , inactivated ExuR as well as UxuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>The complicated case of cross-talking UxuR and ExuR factors is preserved only in Y. pestis , but the latter has lost the GntR regulation of the ED pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The ExuR and UxuR repressors are antagonized by glucuronate and mannonic amide resulting in the derepression of the uxuAB , uxaCA , e x u T and uxaB operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>The two UxuR and ExuR repressors are cornpletely interchangeable with respect to u x u A B control provided that UxuR or ExuR is overproduced in the cell ; however ExuR can not substitute for UxuR in the regulation of the exu regulon operons ( Ritzenthaler et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_21">
The repressors ExuR and UxuR apparently act together to mediate regulation of the UxuR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></label>
<input id="act_similarity_21", type="checkbox">
<div class="ACT_similar">
<li>The ExuR and UxuR repressors apparently act together to mediate regulation of the UxuR regulon by the true inducer , fructuronate ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li></div>
<li>CRP regulation There are experimental data showing that the expression of gntKU and gntT from the GntR regulon and uxuAB from the UxuR/ExuR regulon is under catabolite repression by CRP ( Izu et al. , 1997 ; Peekhaus and Conway , 1998b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>It includes all known and predicted members of the GntR , UxuR/ExuR , KdgR , UidR and IdnR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The last column ( * ) represents experimental data about regulation : S , known binding sites ; R , known regulation , binding site not known.Regulons : a. GntR ; b. IdnR ; c. UxuR/ExuR ; d. ExuR ; e. UidR ; f. KdgR ; g. CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Glucuronate catabolism requires ﬁrst the ExuR regulon , which controls transport of glucuronate and galactu-ronate and their conversion to fructuronate and tagaturonate , respectively , and then the UxuR and KdgR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Glucuronate , because it proceeds through a fructuronate intermediate , must induce the UxuR regulon and must also induce the ExuR regulon , which encodes the glucuronate transporter , ExuT , and the ﬁrst step in its catabolism , UxaC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Thus , we conclude that the physiological conditions yielding the hexuronate metabolism also increase the expression of regulators UxuR and ExuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Thus , hexuronate catabolism in E. coli requires both the ExuR and UxuR regulons , while fructuronate catabolism requires only the UxuR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Thus , hexuronate catabolism in E. coli involves three distinct functional modules encoded by the ExuR , UxuR , and KdgR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>There are no obvious differences between the sites upstream of genes regulated by both UxuR and ExuR and those upstream of genes regulated by ExuR only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>UxuR controls the D-glucuronate metabolism , while ExuR represses genes involved in the metabolism of all hexuronates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_33">
Transport and catabolism of galacturonate and glucuronate are mediated by genes regulated by UxuR and ExuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></label>
<input id="act_similarity_33", type="checkbox">
<div class="ACT_similar">
<li>UxuR/ExuR regulon Transport and catabolism of galacturonate and glucuronate are mediated by genes from the UxuR/ExuR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li></div>
<li>Colors : red , UxuR ; blue , ExuR ; outgroups , black .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>, both the UxuR and the ExuR regulators are present and control the D-glucu-ronate and D-galacturonate catabolic pathways , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>, where it was always found under the control of either UxuR or ExuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Previous studies showed that synthesis of enzymes encoded by uxuAB is principally regulated by UxuR and partially regulated by ExuR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Among the Enterobacteriales , UxuR mostly represses the uxuABR and gntP genes , whereas ExuR mostly represses the exuR , exuT , and uxaABC genes , as well as the uxuAB genes in some genomes that lack UxuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">ExuR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The behaviour of the exuR and uxuR negative dominant plasmids suggests that the altered subunits are still able to form multimers and the hypothesis given here implies that the domain of the UxuR monomer involved in the UxuR subunit aggregation probably presents some homo-logies with that of the ExuR monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>Previous in silico analysis of the UxuR/ExuR regulon revealed a common DNA-binding motif with consensus AAATTGGTATACCAATTT in upstream regions of many known UxuR/ExuR-regulated genes from E. coli and their orthologs in three other gammaproteobacteria ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>The ExuR motif is not strictly palindromic in positions 6/14 , 7/13 , and 9/11 ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>The ExuR binding motif seems to have evolved from the UxuR one , and now we observe early stages of the TFs ' diversiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">ExuR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Two transcription factors of this regulon , ExuR and UxuR , are highly similar ( 49 % identity ) and capable of cross-talk ( Ritzenthaler et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The hexuronate metabolism in E. coli is regulated by two related transcription factors ( TFs ) from the FadR subfamily of the GntR family , UxuR and ExuR , whose amino acid sequences are 46 % identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by ExuR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>ExuR , another Abbreviations : Ap , ampicillin ; bp , base pair ( s ) ; CAMP , cyclic AMP ; Cm , chloramphenicol ; CRP , CAMP receptor protein ; kb , repressor involved in the regulation of the hexuronate system genes , has no effect on the expression of uidA ( Ritzenthaler et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3934044" target="_blank">3934044</a>]<br></li>
<li>Repressor : b-glucuronide induction b-Glucuronide hydrolase b-Glucuronide transport kdgT eda exuR exuT uxaCA ExuR uxaB 4552.6 4549.5 4551.2 UxuR 58 58 58 uxuR uxuA uxuB uxuR uxuAB gntP uidR 4548.3 1694.8 1693.2 1691.6 38 11 11 11 gntP uidR uidA uidB UidR uidAB OTHER PROPOSED ROLES FOR THE ED ENZYMES A growing number of studies have shown that eda expression is regulated in response to several conditions in addition to its role in sugar acid catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>The ExuR training set included known ExuR-regulated genes from E. coli ( exuR , exuT , uxaCA , and uxaB ) and their orthologs from exuR-containing genomes ( CKO , ENT , ECA , KPN , SPE , and YPK ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>ExuR regulates expression of the uxaCA , uxaB and exuT operons involved in galacturonate-catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The ExuR regulon ( 36 ) governs expression of the genes involved in ga-lacturonate catabolism ( Table 2 ) , although ExuT ( 46 ) and UxaC ( 36 ) allow for the uptake of both galacturonate and glucuronate and their isomerization to fructuronate and taga-turonate , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>There is also an additional GntR family TF in Salmonella enterica serovar Typhimurium strain LT2 , related to UxuR and ExuR , but not close enough to be considered an orthologous repressor ( STM3084.S , named uxuR * in Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>ExuR negatively controls most of * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>strain 638 , Klebsiella pneumoniae , and Serratia proteamaculans have only ExuR TFs that regulate genes of both catabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>We have also identiﬁed new candidate members of the UxuR / ExuR regulons , a novel predicted TRAP transporter for hexu-ronates ( uxuPQM ) , hypothetical carbohydrate catabolic genes ( lfaR/lfaTA ) , and the L-galactonate utilization gene cluster ( ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>As in the wild-type uxu operon , the expression of the uxulac fusion on plasmid pNM-U1 was thus under the dual control of the UxuR and ExuR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3083215" target="_blank">3083215</a>]<br></li>
<li>Thus , we conclude that the physiological conditions yielding the hexuronate metabolism also increase the expression of regulators UxuR and ExuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>UxuR/ExuR regulon organization in various gammaproteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>UxuR controls the D-glucuronate-metabolism , while ExuR represses genes involved in the metabolism of all hexuronates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Several bacteria ( Mannheimia succiniciproducens , A. succinogenes , and P. profundum ) have two or more sets of uxuPQM genes in the UxuR/ExuR regulons , suggesting that some of these paralogs are involved in the D galacturonate uptake , in accordance with their colocalization with the D-galacturonate catabolic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>The obtained data revealed a complex response of yjjN on the induced production of UxuR , as well as unexpected , positive correlations in the expression pattern of genes encoding repressors UxuR/ExuR and members of the corresponding regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Mol Gen Genet ( 1985 ) 199:507 -511 © Springer-Verlag1985 Genetic analysis of uxuR and exuR genes : evidence for ExuR and UxuR monomer repressors interactions Paul Ritzenthaler , Carlos Blanco , and MireiHe Mata-Gilsinger Summary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>Genetic analysis of uxuR and exuR genes : evidence for ExuR and UxuR monomer repressors interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>, , C. BLANCOand M. MATA-GILSINGE1R9 ,85 Genetic analysis of uxuR and exuR genes : evidence for ExuR and UxuR monomer repressors interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3079718" target="_blank">3079718</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">ExuR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Operons Position Score Binding site ( s ) * A. GntR regulon E. coli gntKU R a 2211 11 2116 5.84 5.3 5.69 4.42 6 4.65 4.82 cAATGTTACCGaTAACAgTT AAcaGTTACCcGTAACATTT AgATGTTACCcGTAtCATTc TgAcGTTACCcaTAACAaaT TtATtTTACCGGTAACATgA AAtTGTTACgcGTAACAaaT TcAcGTTAtgcGTAACATag S a S a R a gntT edd-eda R b idnDOTR/gntV S. typhi gntKU 2213 9 5.84 4.62 5.78 4.36 5.94 4.48 cAATGTTACCGaTAACAgTT AAcaGTTACCcGTAACAaaT AtATGTTACCcGTAtCATTc TgAcGTTACCcaTAACAaag gtATtTTACCGGTAACATgA AttTGTTACgcGTAACAcag gntT edd-eda Y. pestis gntT/gntK 6.05 5.48 4.39 5.72 ccATGTTACCGGTAtCATgA TcATGaTACCGGTAACAaaT AtATaTTAtCGcTAtCAggA TcAcGTTACCGGTAACATgT idnO-gntV sdsX V. cholerae gntU-eda / gntK-edd 5.8 ± 81 TcATGTTAtgGGTAACATgT 5.57 267 AcATGTTACCGGTAACtTcA saATGTTACCGGTAACATtw Consensus B. UxuR/ExuR regulon E. coli exuR uxaB uxuAB/gntP R c R c R d 5.26 5.24 5.24 4.5 5.06 4.28 3.82 5.19 4.14 261 AAAgTGGTATAaCAAaTa AAtgTGtTcTACCAcTTT gAtgTGGTtaACCAATTT AAATTGGTcaACCAtTgT tAtTTGGTtgACCAgTTT AAAgTtGTATgaCAAgTT AatTTttTtaACtAcgTT ctATTGGTtaACCAATTT gAATTGGTtaACtAATca R d R c uxuR exuT/uxaCA uidR uidAB R , R e c S. typhi exuT/uxuABC AAATTGGctagCCAAaTc tgATTGGTcaACCAATTT ttgTTGGTtgACCAgTTT AtATTGGTAaACCAATaT 4.42 5.07 4.57 5.25 uxuR Y. pestis uxuB uxuA uxuR exuR gntP AAAcTGGTcTAaCAAcTT AAATcGGTATAaCAAcaT AAgTTGtTtgACCAcaTg tAAgTGGTATAaCAAaTc AAcTTGGTtaACCAATac cAtTTGGTtatCCAcTTa AAcTTGtcATACagATTT 4.99 4.44 4.66 4.93 4.78 4.54 3.72 uxaCBA/exuT H. influenzae uxuRA AAATTGGaATACCAATTT tttTTGGaATACCAAaTa AAATTGGTATACCAATTT 5.47 4.39 223 273 Consensus C. KdgR regulon E. chrysanthemi ogl f AAATGAAAgAATGTTTtATaa AAATGAAACgTTGTTTCtaca AAATaAAACATTaTTTCATTT AAtcaAAACAATGTTTCtaTT AAATaAAACATcGTTTCATcg AAAaGAAACATTGTTTCATTT tAATGAAACggTGTTTtATTa AttTaAAACATcGTTTCATTa tAATGAAAtggcaTTTCAaTT tAATGAAAttAcGTTTCAacT AgATGAAAtggTaTTTCgTTT S S S R S S S S S S S 5.15 5.63 6.05 5.42 5.64 6.07 5.83 5.52 5.72 5.44 5.59 f f kduID pelW kdgK kdgT outT pelA pelB pelC pelE f f f f f f f f Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The ExuR motif is not strictly palindromic in positions 6/14 , 7/13 , and 9/11 ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>We found two distinct motifs ( Fig. 3 ) , differing in the 3 - box , ACCA for UxuR and AYAA for ExuR , and in the length of the spacer between the boxes ( 2 and 3 bp , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>This resulted in the identification of palindromic sites with consensus AAATTGGTATACCAATTT and construction of the profile that is likely to represent the UxuR/ExuR signal ( Table 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Previous in silico analysis of the UxuR/ExuR regulon revealed a common DNA-binding motif with consensus AAATTGGTATACCAATTT in upstream regions of many known UxuR/ExuR-regulated genes from E. coli and their orthologs in three other gammaproteobacteria ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Although previous studies assumed that candidate UxuR - and ExuR-binding sites both match the consensus sequence AAATTGGTATACCAATTT and are too similar to be distinguished by sequence analysis ( 30 ) , we observed that if the larger set of sites from different related genomes is used and sites are attributed to the TFs based on the function of the regulated genes , it is possible to separate UxuR and ExuR binding motifs and detect subtle differences between the regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>The UxuR binding motif is a 18-bp palindromic sequence ; the 19-bp palindromic ExuR binding motif is rather similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Sequence logo for the UxuR and ExuR binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>No binding sites for either UxuR or ExuR have been mapped experimentally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>We have distinguished UxuR and ExuR candidate binding sites by positional and sequence analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by ExuR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Glucuronate , because it proceeds through a fructuronate intermediate , must induce the UxuR regulon and must also induce the ExuR regulon , which encodes the glucuronate transporter , ExuT , and the ﬁrst step in its catabolism , UxaC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>Expression of the uxuAB operon is controlled primarily by the uxuR gene product and secondarily by the ExuR repressor ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6309752" target="_blank">6309752</a>]<br></li>
<li>The expression of the uxuAB operon is primarily controlled by the uxuR gene product and second-arily by the ExuR repressor ( Robert-Baudouy et al. 1981 ; Ritzenthaler et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3083215" target="_blank">3083215</a>]<br></li>
<li>There are candidate UxuR/ExuR boxes upstream of the uidA and uidR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Among the Enterobacteriales , UxuR mostly represses the uxuABR and gntP genes , whereas ExuR mostly represses the exuR , exuT , and uxaABC genes , as well as the uxuAB genes in some genomes that lack UxuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>ExuR regulates expression of the uxaCA , uxaB and exuT operons involved in galacturonate catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The ExuR and UxuR repressors are antagonized by glucuronate and mannonic amide resulting in the derepression of the uxuAB , uxaCA , e x u T and uxaB operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>As in the wild-type uxu operon , the expression of the uxulac fusion on plasmid pNM-U1 was thus under the dual control of the UxuR and ExuR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3083215" target="_blank">3083215</a>]<br></li>
<li>The operon structures of the UxuR/ExuR regulon genes are shown in Fig. 2B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The u x u A B operon is under the dual control of uxuR - and exuR-encoded repressors whereas the exu re-gulon genes are regulated by the sole ExuR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
<li>The ExuR repressor is the sole regulatory molecule controlling the expression of the exu regulon genes ( uxaCA , e x u T and uxaB ) whereas the u x u A B and uidA operons are each controlled by two different repressors ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3929016" target="_blank">3929016</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>