<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - FabR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">FabR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">FabR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>This was confirmed by quantitative RT-PCR , which showed that the degree of regulation was modest ; deletion of FabR resulted in about an 80 % increase in P2 transcripts , whereas P1 transcripts showed about a 2-fold increase ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>This approach not only avoided the solubility problems , but also allowed the use of an untagged protein and precluded the possibility of binding of endogenous ligands by FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>This raised the question of whether or not DNA binding by FabR necessarily inhibits transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Transcription of the Escherichia coli Fatty Acid Synthesis Operon fabHDG Is Directly Activated by FadR and Inhibited by ppGpp Laetitia My , a Brian Rekoske , b Justin J. Lemke , b Julie P. Viala , a Richard L. Gourse , b Emmanuelle Bouvereta LISM , CNRS , Aix-Marseille University , Marseille , Francea ; Department of Bacteriology , University of Wisconsin -- Madison , Madison , Wisconsin , USAb In Escherichia coli , FadR and FabR are transcriptional-regulators that control the expression of fatty acid degradation and unsaturated fatty acid synthesis genes , depending on the availability of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>In those experiments the same batch of FabR was used for parallel gel shift experiments done in the presence or absence of acylCoAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>In agreement with the observations of Zhu and coworkers ( Zhu et al. , 2009 ) , addition of oleoyl-ACP ( Fig. 10A & B ) or palmitoleoyl-CoA ( or oleoyl-CoA , Fig. 10C & D ) gave increased binding of FabR to its targets although in our hands binding was also seen in the absence of these ligands ( the super-shifted bands sometimes seen in our assays may be attributable to the minor polymeric forms of FabR observed by the size exclusion chromatography ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Moreover , the presence of oleoyl-CoA did not shift the binding curve to lower FabR concentrations , although the super-shift seen in the absence of acyl-CoA was no longer present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>In contrast , the presence of palmitoyl-CoA shifted the binding curve to higher FabR concentrations ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>As seen for the E. coli FabR DNA binding was obtained in the absence of any regulatory ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Our in vivo data plus the prior in vitro data outlined in the Introduction argued that FabR binds the fabA and fabB promoters in the absence of an unsaturated thioester ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>FabR bound both B2A probes in the absence of an unsaturated ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>The location of the predicted binding sequence and the data discussed above suggests that FabR regulates by repression and antagonizes the action of the FadR activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Regulation of UFA content by the FabR repressor requires the presence of the FadR activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>In our experiments we used ΔfadR strains lacking the FadR activator protein in order to limit regulation to FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Panel A , real time quantitative RT-PCR assays of fabA expression normalized to the mRNA level expressed from the fabA gene of fadR strain K113 in the presence or absence of FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>The fact that the UFA content is not increased in a fadR fabR strain compared with a fadR strain illustrates that the level of fabA expression , supported by the transcriptional activation by FadR , is required for UFA regulation by FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>The increased repression of fabB expression engendered by oleate addition was not seen in the ΔfabR ΔfadR strain indicating that the FadR-independent oleate sensing required FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>1 The abbreviations used are : ACP , acyl carrier protein ; FabA , - hy-droxydecanoyl-ACP dehydratase ; FabB , - ketoacyl-ACP synthase I ; FabF , - ketoacyl-ACP synthase II ; SFA , saturated fatty acid ; UFA , unsaturated fatty acid ; FadR , fatty acid degradation regulator ; and FabR , fatty acid biosynthesis regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Transcriptional fusions containing the fabA , fabB , and fadE promoters , previously reported as regulated by FadR or FabR , were used as positive controls ( 15 , 16 , 54 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Expression of fabA and fabB is repressed by FabR , which binds a site overlapping their promoters , and is activated by FadR , which binds a consensus site located around the 40 position ( 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>fabA and fabB genes are repressed by FabR and activated by FadR ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In contrast to Gram-positive bacteria , it was thought previously that not all the fab genes in E. coli were controlled by fatty acid availability ; rather , it was thought that regulation was restricted to unsaturated fatty acid biosynthesis by FadR and FabR regulation of only the fabA and fabB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The FadR factor controls fabA and fabB expression in response to long-chain acyl-CoA derived from exogenous fatty acids ( 9 , 17 ) , and it is tempting to speculate that FabR is controlled by a ligand that is an integral component of endogenous de novo fatty acid biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>FadR activates transcription of fabA and fabB when it binds in its apo form ( 15 , 16 ) , whereas FabR represses transcription of fabA and fabB in the presence of unsaturated fatty acids ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>FabR represses expression of the two genes , fabA and fabB , required for unsaturated fatty acid synthesis and has been reported to require the presence of an unsaturated thioester ( of either acyl carrier protein or CoA ) in order to-bind the fabA and fabB promoters in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Thus , our in vivo data indicate that because FabR does not sense unsaturated acyl-ACP thioesters in vivo , the primary function of FabR is to report the presence of exogenous UFA ( as their CoA esters ) to the fabB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Our data argue that the function of FabR is to compensate for the weak FadR regulation of fabB expression in the presence of exogenous UFA ( the converse argument , that FadR functions to compensate for the weak FabR regulation of fabA expression by UFA , also applies ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>In the model of Zhu and coworkers ( Zhu et al. , 2009 ) FabR bound to an unsaturated acyl-ACP was the form of FabR responsible for repression of fabB expression in the absence of exogenous UFAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Recently Zhu et al. ( Zhu et al. , 2009 ) reported that presence of a CoA or ACP thioester of a long chain UFA was required for binding of FabR to the fabA and fabB promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>C & D. Effect of long chain unsaturated - acyl CoAs on the binding of FabR to fabA and fabB probes , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Taken together the DNAC plus qRT-PCR and lacZ transcriptional fusion data indicated that only fabB expression was subject to significant FabR regulation and only exogenous UFAs ( imported as acyl-CoAs ) affected the severity of the regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Finally , FabR was found not to regulate yqfA expression and to not be subject to autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Indeed , although FabR binds the yqfA promoter , the absence of FadR has no effect on yqfA expression in vivo ( Fig. 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">FabR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The two FabR proteins behaved similarly in that both gave three peaks indicating monomeric , dimeric and polymeric forms ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>FabR is a member of the TetR superfamily ( Pfam00440 ( 37 ) ) of bacterial regulatory proteins , and , by analogy to the known mechanisms used by these factors , it seems reasonable to postulate that a regulatory ligand binds to the carboxyl-terminal domain of the protein to release the amino-terminal DNA-binding domain from its target sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Finally , these workers ( Zhu et al. , 2009 ) tagged the FabR Nterminus which raised a possible caveat because the predicted DNA-binding domain is located adjacent to the N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>E. coli FabR was synthesized in both native and N-terminal hexahistidine-tagged forms to test if the tag interfered with DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">FabR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>We have now shown that the V. cholerae FabR which is 73 % identical to the E. coli protein has in vivo and in vitro properties very similar to those of E. coli FabR ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by FabR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In the absence of a long chain acyl-CoA or acyl-ACP , FabR binding shifted the B+D probes ( which acted as positive controls ) whereas the D1 probes showed no shifted bands ( negative controls ) ( Fig. 9A -- C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>There are important questions that remain concerning the role of the FabR repressor in cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>We ﬁrst checked whether the transcription regulator FadR or FabR regulated the fab-acpP locus by measuring expression from the 4 promoters yceDp1 , yceDp2 , fabH , and acpP in fadR and fabR mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Another part of the FabR story that deserves attention is defining what mechanisms control the expression of fabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>FabR plays no detectable role in yqfA expression Based on the results of phylogenetic footprinting , McCue et al. ( McCue et al. , 2001 ) suggested that E. coli yqfA , encoding a membrane protein of unknown function was a third FabR regulated gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>TABLE IV Fatty acid metabolic genes regulated by FadR versus FabR Fold changea Gene fadR wt fadR fabR wt Gene description fabR wt Fatty acid uptake fadD fadL -- b -- 13 13 Fatty acyl-CoA synthetase 13 14 Outer membrane long-chain-fatty-acid-transporter Fatty acid degradation fadA 17 113 18 111 - Ketoacyl thiolase I -- -- -- -- -- -- Fatty acid oxidation complex Ic Acyl-CoA dehydrogenase 2,4-Dienoyl-CoA reductase Fatty acid oxidation complex IId - Ketoacyl-CoA thiolase II fadB fadE ( yafH ) fadH ( ygjL ) yfcX ( b2341 ) yfcY ( b2342 ) Acetate metabolism aceA 14 13 Isocitrate lyase -- aceB 14 -- 14 Malate synthase aceK 13 -- 16 Isocitrate dehydrogenase kinase/phosphatase iclR 23 -- -- Repressor of aceBAK operon a The expression levels of metabolic genes were monitored in E. coli strain PDJ14 ( fadR ) , strain UB1005 ( wt ) , and in the isogenic fabR ( ANS8 ) and fadR fabR ( ANS4 ) strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Our data argue that the function of FabR is to compensate for the weak FadR regulation of fabB expression in the presence of exogenous UFA ( the converse argument , that FadR functions to compensate for the weak FabR regulation of fabA expression by UFA , also applies ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>The FabR ( YijC ) Transcription Factor Regulates Unsaturated Fatty Acid Biosynthesis in Escherichia coli * Received for publication , February 11 , 2002 , and in revised form , February 20 , 2002 Published , JBC Papers in Press , February 21 , 2002 , DOI 10.1074 / jbc.M201399200 Yong-Mei Zhang ‡ § , Hedia Marrakchi ‡ § , and Charles O. Rock ‡ ¶ From the ‡ Department of Infectious Diseases , Protein Science Division , St. Jude Children 's Research Hospital , Memphis , Tennessee 38105 and the ¶ Department of Molecular Biosciences , University of Tennessee Health Science Center , Memphis , Tennessee 38163 Unsaturated fatty-acid-biosynthesis is a vital facet of Escherichia coli physiology and requires the expression of two genes , fabA and fabB , in the type II fatty acid synthase system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Transcription of the Escherichia coli Fatty Acid Synthesis Operon fabHDG Is Directly Activated by FadR and Inhibited by ppGpp Laetitia My , a Brian Rekoske , b Justin J. Lemke , b Julie P. Viala , a Richard L. Gourse , b Emmanuelle Bouvereta LISM , CNRS , Aix-Marseille University , Marseille , Francea ; Department of Bacteriology , University of Wisconsin -- Madison , Madison , Wisconsin , USAb In Escherichia coli , FadR and FabR are transcriptional regulators that control the expression of fatty-acid-degradation and unsaturated-fatty-acid-synthesis genes , depending on the availability of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>FabR represses expression of the two genes , fabA and fabB , required for unsaturated-fatty-acid-synthesis and has been reported to require the presence of an unsaturated thioester ( of either acyl carrier protein or CoA ) in order to bind the fabA and fabB promoters in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>In E. coli , the balance between unsaturated and saturated FA is sensed by the FabR repressor , which controls the expression of fabA and fabB , which are speciﬁcally required for the synthesis of unsaturated FA ( 25 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Thus , FabR functions as a repressor that potently controls the expression of the fabB gene , which in turn , modulates the physical properties of the membrane by altering the level of unsaturated fatty acid production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>These compositional data strongly support the conclusion that FabR acts as a repressor of fabB , which in turn , modifies the membrane fatty acid composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Based on these data we propose renaming this repressor FabR ( fatty-acid-biosynthesis regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Complementation , RT-qPCR and fatty acid composition analyses demonstrated that V. cholerae FabR is a functional repressor of unsaturated-fatty-acid-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>1 The abbreviations used are : ACP , acyl carrier protein ; FabA , - hy-droxydecanoyl-ACP dehydratase ; FabB , - ketoacyl-ACP synthase I ; FabF , - ketoacyl-ACP synthase II ; SFA , saturated fatty acid ; UFA , unsaturated fatty acid ; FadR , fatty-acid-degradation regulator ; and FabR , fatty-acid-biosynthesis regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>The regulation of fabB is the most important function of FabR in determining fatty acid composition , and this discovery is a unique example of a transcription factor that exclusively regulates the expression of the enzymes of a type II fatty acid synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>The goal of this study was to determine whether FabR ( YijC ) regulates unsaturated-fatty-acid-formation by examining the effects of deleting this gene on fatty-acid-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>This study links the FabR ( YijC ) transcription factor to the regulation of unsaturated fatty acid content through the regulation of fabB gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>The FabR ( YijC ) transcription factor regulates unsaturated-fatty-acid-biosynthesis in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>In contrast to Gram-positive bacteria , it was thought previously that not all the fab genes in E. coli were controlled by fatty acid availability ; rather , it was thought that regulation was restricted to unsaturated-fatty-acid-biosynthesis by FadR and FabR regulation of only the fabA and fabB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The FadR factor controls fabA and fabB expression in response to long-chain acyl-CoA derived from exogenous fatty acids ( 9 , 17 ) , and it is tempting to speculate that FabR is controlled by a ligand that is an integral component of endogenous de novo fatty-acid-biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">FabR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Binding of FabR to its cognate promoters does not require unsaturated acyl-CoA or acyl-ACP thioesters 1In a personal communication with the corresponding author of that paper we learned that the probe was a 46 bp fabB probe of sequence 5 ′ - GGCTGATCGGACTTGTTCGGCGTACAAGTGTACGCTATTGTGCATT-3 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Since the FabR binding site is a degenerate palindrome and bacterial transcription factors generally bind to the target DNA sites as symmetric structures ( Collado-Vides et al. , 1991 ; McCue et al. , 2001 ) ( e.g. , dimer or tetramer ) , it seemed likely that the functional forms of both FabRs were the dimeric species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Although we could not find a plausible match for the consensus FabR binding site upstream of the coding sequences of the E. coli and V. cholerae fabR genes , binding sites involved in autoregulation often deviate from the consensus binding site ( Collado-Vides et al. , 1991 ; Gunsalus and Yanofsky , 1980 ) because weak regulation is required to allow sufficient levels of the regulatory protein to accumulate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>ot regulate V. cholerae fabB expression n s e o d R b a F To test if bound the fabB promoter region by a sequence we had failed to recognize , a set of gel shift assays were performed in which all four promoter regions ( i.e. , the fabA and fabB regions of both bacteria ) were tested with both FabR proteins ( Fig. 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>To examine FabR binding to this region two probes covering the putative binding site ( 43 bp and 32 bp ) ( Fig. 11B ) were used in gel shift assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>The V. cholerae fabA probe covering a predicted FabR binding site was PCR amplified with two primers , plus , and a Electrophoretic mobility shift assays DNA fragment containing fabB promoter region ( 257 bp ) was obtained with a pair of primers , plus ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Thus in common with our 56 bp B+D fabB probe the Zhu et al probe contained both the FadR and FabR binding sites , although it lacked the extra sequences we placed upstream of the FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>However , the differential binding scenario is consistent with the finding that FabR does not specifically require the fabB B2 probe FadR binding site sequences for efficient binding in vitro , a random DNA sequence suffices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Moreover , the 10 region of the P2 promoter was located within the FabR binding site suggesting that P2 promoter function was repressed by FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>Expression of fabA and fabB is repressed by FabR , which binds a site overlapping their promoters , and is activated by FadR , which binds a consensus site located around the 40 position ( 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Native FabR binds the fabA promoter region provided that the canonical FabR binding site is extended by inclusion of flanking sequences that overlap the neighboring FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>We report that FabR binds to both the fabA and fabB promoter regions in the absence of unsaturated fatty acid thioesters provided that the DNA sequences required for FabR binding are longer than the canonical binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Indeed , no FadR binding site is found upstream of fabB in any of the currently available Vibrio genome sequences , although a FabR binding site is invariably found within the fabA promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>FabR binds to a sequence found in the promoter regions of the fabA and fabB genes from E. coli ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>C. & D. V. cholerae FabR protein binds to degenerate palindromic sequences within the promoter regions of E. coli fabA and fabB , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>This raised the question of whether the extra bp of the B2 probe provided specific or nonspecific augmentation of FabR binding relative to the B1 probe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Although the shorter B1 version of the FabR-specific probes contained the full-length FabR-binding site , FabR protein failed to bind this probe whereas the slightly longer B2 probes were bound indicating that the additional DNA sequences adjacent to the FabR-binding site allowed binding ( Fig. 9C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>As seen above for fabA ( Fig. 11C ) , the longer yqfA probe bound FabR protein ( Fig. 11C ) , although the binding was weak compared to the fabA sequence assayed in parallel whereas the shorter probe was shifted by FabR only upon addition of palmitoleoyl-CoA or oleoyl-CoA ( Fig. 11D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by FabR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Finally , FabR was found not to regulate yqfA expression and to not be subject to autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>The increased repression of fabB expression engendered by oleate addition was not seen in the ΔfabR ΔfadR strain indicating that the FadR-independent oleate sensing required FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>fabA and fabB genes are repressed by FabR and activated by FadR ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>