<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - FadR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">FadR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">FadR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Subdivision of the Helix-Turn-Helix GntR Family of Bacterial Regulators in the FadR , HutC , MocR , and YtrA Subfamilies * Received for publication , November 15 , 2001 , and in revised form , December 20 , 2001 Published , JBC Papers in Press , December 27 , 2001 , DOI 10.1074 / jbc.M110968200 Sébastien Rigali ‡ , Adeline Derouaux , Fabrizio Giannotta , and Jean Dusart From the Centre d'Inge ́nierie des Protéines , Université de Liège , Institut de Chimie B6 , Sart-Tilman , B-4000 , Liège , Belgium Haydon and Guest ( Haydon , D. J , and Guest , J. R. ( 1991 ) FEMS Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>DNA Sequences Surrounding the mcbR ( yncC ) Gene in Escherichia coli ( MG1655 ) other names mnaT , b1448 curA , b1449 gene yncA gene description Methionine N-acyltransferase ; - amino acid NL acyltransferase Curcumin/dihydrocurcumin reductase , NADPH-dependent Predicted iron outer membrane transporter ATP-binding protein , periplasmic , function unknown L-asparagine transporter Glutathione S-transferase homologue Conserved protein , function unknown yncB b1451 b1452 yncD yncE yncF yncG yncH ansP , b1453 b1454 b1455 ■ ABBREVIATIONS ASU , asymmetric unit ; CD , circular dichroism ; EPS , exopolysaccharide ; EDTA , ethylenediaminetetraacetic acid ; EMSA , electrophoretic mobility shift assay ; FCD , FadR Cterminal domain ; IPTG , isopropyl-β-D-thiogalactopyranoside ; MPD , 2-methyl-2 ,4 - pentanediol ; PDB , Protein Data Bank ; SEC , size-exclusion chromatography ; SeMet , selenomethionine ; TCEP , tris ( 2-carboxyethyl ) phosphine ; TEV , Tobacco Etch Virus ; wHTH , winged helix-turn-helix ; WT , wild-type ■ REFERENCES ( 1 ) Lewis , K. ( 2001 ) Riddle of biofilm resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>mparative Genomic Analysis of the Hexuronate Metabolism G and Their Regulation in Gammaproteobacteria † Co enes Inna A. Suvorova ,1 Maria N. Tutukina ,2 Dmitry A. Ravcheev ,1,3 Dmitry A. Rodionov ,1,3 Olga N. Ozoline ,2 * and Mikhail S. Gelfand1 ,4 * Institute for Information Transmission Problems RAS ( The Kharkevich Institute ) , Moscow 127994 , Russia1 ; Institute of Cell Biophysics RAS , Pushchino , Moscow Region 142290 , Russia2 ; Sanford-Burnham Medical Research Institute , La Jolla , California 920373 ; and Faculty of Bioengineering and Bioinformatics , Moscow State University , Moscow 119991 , Russia4 Received 25 February 2011/Accepted 13 May 2011 The hexuronate metabolism in Escherichia coli is regulated by two related transcription factors from the FadR subfamily of the GntR family , UxuR and ExuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Aug. 1996 , p. 4704 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Regulated Expression of a Repressor Protein : FadR Activates iclR LIZHEN GUI , ALDEN SUNNARBORG , AND DAVID C. LAPORTE * Department of Biochemistry , University of Minnesota , Minneapolis , Minnesota 55455 Received 23 February 1996/Accepted 21 May 1996 The control of the glyoxylate bypass operon ( aceBAK ) of Escherichia coli is mediated by two regulatory proteins , IclR and FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Nov. 1996 , p. 6443 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Role of the Escherichia coli FadR Regulator in Stasis Survival and Growth Phase-Dependent Expression of the uspA , fad , and fab Genes ANNE FAREWELL ,1 ALFREDO A. DIEZ ,1 CONCETTA C. DIRUSSO ,2 † AND THOMAS NYSTRÖM1 * Department of Microbiology , Lund University , Lund , and Department of General and Marine Microbiology , Lundberg Laboratory , Göteborg University , Göteborg , Sweden ,1 and Department of Biochemistry , College of Medicine , The University of Tennessee , Memphis , Tennessee 381632 Received 13 May 1996/Accepted 30 August 1996 The increased expression of the uspA gene of Escherichia coli is an essential part of the cell 's response to growth arrest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>* -35 CCATGGCCATTACGTTGGCTGAACTGGTTTATTCGGAACTGATCGGACTTGTTCAGCGTA -10 +1 CACGTGTTAGCTATCCTGCGTGCTTCAATAAAATAAGGCTTACAGAGAACATG - 3 5 * - 1 0 AGATCATATTTGAAAAAAGATAGAAAGATAGTTGCAACATTCCAGGTGGTCCGACCTATA + 1 * Specific Activities ' ^ Relevant genotype Strain / Plasmid ^ G M rowth edium '' ase ( J-oxidatior1 l ^ - galactosid LSI345 \ <t> fadB-lacZ TB 0.71 ± 0.2 3.65 +0.1 ND 3.11 ± 0,2 3.53 ± 0,6 ND 0.47 '' ND ND 4.88 ° ND ND 0.6610.2 2.55 ± 0,2 ND 2.5010.4 2.50 ± 0.1 ND 320187 Regulation o ^ fadL and fabA transcription in vitro To compare the role of FadR in regulating the transcription of fadL and fabA , we performed in vitro transcription experiments using purified FadR , CRP and RNA poiy-merase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>S2 support the overall importance of the Nan box for NanR binding and repression of the sialoregulon using a repressor in the FadR family of helix-turn-helix ( HTH ) regulators with , however , atypical responses to inducer , its oligomerization state , and binding mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Typically , bacterial transcriptional-regulators use a helix-turn-helix ( HTH ) motif to-bind DNA and commonly form homodimers ( e.g. , cI , cro , trp repressor , CRP , NtrC , MarR , and FadR ) ( 2 , 22 , 36 , 40 , 42 , 43 , 49 , 60 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>This family includes GntR , HutC , KorA , and five regulatory proteins of E. coli : PdhR , FadR , LctR , PhnF , and MngR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645248" target="_blank">14645248</a>]<br></li>
<li>Rigali , S. , Derouaux , A. , Giannotta , F. , and Dusart , J. ( 2002 ) Subdivision of the helix-turn-helix GntR family of bacterial regulators in the FadR , HutC , MocR , and YtrA subfamilies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Though not shown in Fig. 5 , the C-terminal NanR effector binding domain is predicted to be composed of either turns or - helical secondary structural units ( 6 ) , which is topologically identical to FadR and clearly places NanR in the FadR subfamily of bacterial HTH regulators despite the fact that NanR uses overlapping binding sites instead of wings for speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>As was mentioned earlier , the typical distance between GT and AC in most FADR-subfamily TF-binding sites is 3 nt ( DgoR , ExuR , FadR , GlcC , LldR , PdhR etc ) , although in several orthologous groups it is 2 nt ( e.g. , GntR , HpxS , HypR , MdcY , PrpR , UxuR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>Soluble FadR protein was obtained following induction with 0.2 mM isopropyl - - D-1-thiogalactopyranoside at 30 ° C for 3 h. Following lysis by French pressure cell treatment and centrifugation , the clariﬁed bacterial supernatant was loaded onto a nickel-ion afﬁnity column ( Qiagen ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>in the presence of acetate and is regulated by cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and FNR as well as by IclR and its activator , FadR ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>We also have found that the timing and/or magnitude of this induction depends , in part , on the carbon regulator cyclic AMP ( cAMP ) receptor protein ( CRP ) , the oxygen regulator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Transcription depends upon CRP , the anaerobically triggered transcription activator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>It also depends , probably indirectly , upon the glyoxylate shunt repressor IclR , its activator FadR , and many enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>FadR is unusual in that it regulates two directly opposing pathways , is the ﬁrst positive activator shown to release DNA upon binding of the regulatory ligand ( Henry and Cronan , 1991 ; 1992 ) ( hence the ﬁrst example of repression mediated by positive control ) and , unlike many DNA binding proteins , does not autoregulate its synthesis ( Henry and Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The survival and growth of bacteria under such changing environments largely depend on the adjustments in the expression proﬁles of several global regulatory proteins , such as RpoD , RpoS , SoxRS , Cra , FadR and IclR ( Aronis 2002a , b ; Cortay et al. 1991 ; Jordan et al. 1999 ; Liochev and Fridovich 1992 ; Lu et al. 2003 ; Moat 2002 ; Tang et al. 2002 ; Varghese et al. 2003 ; Perrenoud and Sauer 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>It should be noted that Farewell et al. ( 1996 ) reported that the transcription of uspA , a gene involved in response to a variety of environmental stresses , is repressed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>These results are consistent with previous studies showing that both fadJ and fadI are negatively regulated by FadR ( Campbell et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Furthermore , it has been reported that the anaerobic b-oxidation pathway is mediated by fadJ , fadI and fadK genes , whose transcriptional regulation is independent of FadR control ( Campbell et al. , 2003 ; Morgan-Kiss & Cronan , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>It is known that the transcription of fadJ and fadI is controlled by FadR in the presence of oxygen , whereas that of fadK is independent of FadR control ( Campbell et al. , 2003 ; Morgan-Kiss & Cronan , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Lane 1 , no additions ; iane2 , 10 nM FadR ; lane 3 , 45 nM CRP and 10 ( jm cAMP ; lane 4 , 10 nM FadR , 45nM CRP and lO ^ McAMP Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_28">
100 nM FadR ; lane 3 , 40 nM FadR ; lane 4 , 20 nM FadR ; lane 5 , no FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></label>
<input id="act_similarity_28", type="checkbox">
<div class="ACT_similar">
<li>Lane 1,200 nM FadR ; lane 2.100 nM FadR ; lane 3 , 40 nM FadR ; lane 4 , 20 nM FadR ; lane 5 , no FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li></div>
<li>Current evidence indicates that very low levels of LCACoA are required to inactivate FadR ; estimates of Kd are in the 10 ± 100 nanomolar range for C16 ± C18 coenzyme A thioesters [ 119 , 120 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The biochemical evidence above support the conclusion that LCACoA thioesters are the natural ligands which bind to and inactivate FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>First , accumulation LCACoA in vivo results in inactivation of FadR and second , LCACoA must accumulate from an endogenous source to inactivate FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>When the same strains were used to evaluate the activity of the promoters fused to IacZ by assaying p-galactosi-dase activity , the Individual patterns of regulation emerged ( Table 1 ) , The strains carrying the fadB-lacZ operon fusion were negatively controlled by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>No effects on accBC operon expression were observed under the conditions we examined , including accumulation of fatty acidbiosynthetic intermediates by shifting of cultures of fabA ( Ts ) , fabB ( Ts ) , or fabD ( Ts ) mutants to nonpermissive temperatures ; supplementation of cultures of long-chain saturated and unsaturated fatty acids ; or overproduction of BCCP , biotin carboxylase , carboxyltransferase , biotin li-gase , acyl carrier protein , FadR , integration host factor , or the crp or cya gene product ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7678242" target="_blank">7678242</a>]<br></li>
<li>As fadR mutants were easily isolated without mutagenesis , they seemed likely to be loss-of-function mutants , suggesting to Over-ath et al. ( 1969 ) that FadR negatively regulated the fad regulon and that FadR repressor action might be neutralized upon binding of fatty acids ( or a related metabolite ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The model for induction is analogous to the lac paradigm : fatty acid ( or a derived metabolite ) binds to FadR , causing disassociation from the fad operators ( Nunn , 1987 ; Hughes et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>) Dimeric FadR is also consistent with the known trans-dominance of fadR super-repressor mutants ( Hughes et al. , 1988 ; Raman and DiRusso , 1995 ) ( iii ) FadR should be released from the operators upon binding of fatty acid or a fatty acid-derived metabolite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>These results indicate that speciﬁc changes in membrane composition , resulting from overproduction of spe-ciﬁc membrane proteins or of the FadR regulator involved in fatty acid metabolism ( Cronan and Subrahmanyam , 1998 ) , lead to signals that stimulate EnvZ activity , in turn inducing expression of OmrA and OmrB sRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>RESULTS ArcA downregulates the transcription of fatty acid transport genes under anaerobic conditions The expression of fadL , which encodes one of the outer-membrane-bound proteins involved in the uptake of fatty acids , is regulated by at least three transcription factors : Crp , OmpR and FadR ( Berg et al. , 2004 ; Busch & Saier , 2002 ; Clark & Cronan , 1996 ; DiRusso et al. , 1992 ; Cronan & Subrahmanyam , 1998 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>None of the transcription factors involved in fatty acid regulation ( Crp , OmpR and FadR ) is oxygen sensing , though , recently , the ArcBA two-component signal transduction system , which senses oxygen indirectly through a change in redox potential , has been implicated in the negative regulation of aerobic fatty acid metabolism ( Georgellis et al. , 2001 ; Clark & Cronan , 1996 ; Malpica et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The model presented here is supported by the following evidence : ( i ) FadR repression coincides with ArcA repression for the fatty acid metabolism under anaerobic conditions ; ( ii ) ArcA-P is a DNA-binding protein speciﬁc to the upstream of fad regulon genes involved in fatty acid metabolism of E. coli ; ( iii ) under anaerobic conditions , transcription of fad regulon genes is increased upon the inactivation of ArcA , whereas , under aerobic conditions , transcription of those genes is not affected by the inactivation of ArcA ; ( iv ) the fadR deletion affects both aerobic and anaerobic transcription of fad regulon genes ; and ( v ) under anaerobic conditions , the transcript levels of fad regulon genes are synergistically increased upon the deletion of both arcA and fadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The apparent surprise is the stringent repression by ArcA seen under anaerobic conditions even in the absence of FadR because E. coli has a pathway to degrade fatty acids under anaerobic conditions [ 34 ] and low expression of FadD and FadL would seem likely to compromise function of the anaerobic pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Gene expression proﬁles of deletion mutants ( DarcA , DfadR and DarcA/DfadR ) indicated that ( i ) ArcA is a major transcription factor for the transcriptional regulation of fatty acid metabolism in the absence of oxygen , and ( ii ) ArcA and FadR cooperatively regulate the fad regulon under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>When no fatty acids are present , FadR remains active under anaerobic conditions , and co-regulates the fad regulon , along with ArcA-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The fatty acid acid-mediated repression of OLE1 requires the activity of fatty acyl-CoA synthetase ( FAA1 or FAA4 [ 158 ± 160 ] ) suggesting that LCACoA mediates regulatio perhaps either by direct modi ® cation of a transcription factor as is seen for E. coli FadR or by modifying an additional protein that in turn regulates gene expression [ 153 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>This considerable increase in unsaturated fatty acid production is consistent with the fact that FadR activates the expression of fabA and fabB , whose enzyme products catalyze the biosynthesis of unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Here , our results also indicate that FadR expression led to increased FabB levels , which increased fatty acid yields , especially unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Gui et al. , ( 1996b ) proposed that FadR , which is known as a repressor for the transcription of the fatty acid deg-radation operon , regulates the expression of iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>Instead , FadR appears to prevent an increase in the expression of iclR during growth on fatty acids by activating the transcription of this gene on all other carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>This transcription pattern shows that FadR negatively regulates yafH and that induction of yafH expression upon supplementation of cultures with fatty acids is mediated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The activation of fabH expression by FadR and its inhibition by oleate suggest that FadR activates the fabH promoter in the absence of long-chain fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The expression of fadL is under the control of the FadR repressor and is induced in the presence of long-chain fatty acids ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987139" target="_blank">1987139</a>]<br></li>
<li>In the absence of fatty acid , FadR binds to the downstream promoter and activates transcription ( thick sawtooth lines ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>In the presence of fatty acid , FadR is released from the downstream promoter , and expression from that promoter decreases by 20-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>It should be noted that the rationale for FadR regulation of uspA is not obvious , as uspA null mutants retain the ability to grow on fatty acids as sole carbon source and have extremely pleiotropic effects on growth and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Our results on fadSA regulation are in general agreement with those previously obtained in this laboratory by Clark ( 1981 ) , except that we found that unsaturated fatty acids were more potent in antagonizing FadR regulation than were saturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>While this seems an attractive proposal , it still begs the question of why the synthesis of unsaturated acids in particular , as opposed to that of saturated fatty acids , is regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Why does FadR regulate only the synthesis of unsaturated fatty acids in response to exogenous supplementation ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Apparently , in the absence of FabF and FadR , even at the permissive temperature the synthetic demand placed on the decreased level of the FabB ( Ts ) enzyme exceeded its capacity to produce sufﬁcient fatty acids to support growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>During growth in TB , which contains no fatty acids , FadR probably operates through IclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>FadR may act speciﬁcally to prevent an increase in IclR during growth on fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>This ﬁnding is consistent with the observation that FadR is inactivated during growth on fatty acids ( 5 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>However , repression by FadR would not solve this problem for iclR since FadR is inactivated during growth on fatty acids ( references 5 and 10 and this study ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>A FadR Activation of fabA Transcription Is Repressed by Fatty Acids In Vivo Repression of the P-oxidation regulon by FadR is relieved by addition of fatty acids to the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>As shown in Figure 5 , the addition of fatty acids to the medium decreased fabA gene expression by approximately lo-fold , to levels of expression approaching those seen in strains lacking FadR ( Henry and Cronan , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>We proposed that FadR activation of fabA transcription is a mechanism to limit fatty acid synthesis when exogenous fatty acids are available ( Henry and Cronan , 1991 ) and , thus , we predicted that fabA transcription would be inhibited upon addition of exogenous fatty acids ( owing to release of FadR from its activating site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Transcription of fabA was decreased by 10 - to 12-fold upon either the addition of fatty acids or the inactivation of FadR , whereas fadBA transcription increased by 10 - to 12-fold ( Henry and Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>A decrease in FadR levels during stationary phase might also contribute to derepression of fad genes that are normally repressed by FadR , e.g. , those needed for degradation of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>If so , the fad genes and operons , derepressed by fatty acids and repressed by FadR , should also be increasingly expressed in stationary phase while the expression of the fatty acid biosynthetic genes activated by FadR should be reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>The genes of the fad regulon are repressed by FadR in the absence of long-chain fatty acids ( LCFAs ) ( Campbell et al. , 2003 ) , which bind FadR , eliminating its activity ( DiRusso et al. , 1992 ; Xu et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>In the presence of fatty acids , FadR becomes inactive , and is released from promoter regions , whereas ArcA-P continues to repress the transcription of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>genes have no signiﬁcant transcriptional response to the presence of fatty acids or FadR , a conclusion consistent with the sequence of this genomic segment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>In the absence of fatty acids , FadR represses the transcription of fad genes required for fatty acid degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>All genes in this pathway are repressed by FadR in the absence of fatty acids ( Henry & Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The transcription of fadL is induced by Crp in response to cAMP levels , and repressed by FadR in the absence of exogenous fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Conversely , transcription of fabA decreased in the absence of FadR or in the presence of long-chain fatty acids ( 29 , 30 ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Q 1998 Blackwell Science Lt The model ( Fig. 1 ) In the absence of an exogenous source of long-chain fatty acids , FadR is bound to all of its cognate operators acting both to repress transcription of the fad genes and to activate transcription of the fabA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>A protein bound specifically and with varying affinity to all three of the sites , fabA , fabB and yqfA ( Fig. 1 , lanes 1 -- 3 ) , that did not bind to affinity columns containing binding sites for FadR , a transcriptional-regulator of fatty acid metabolism genes ( 22,23 ) , or PurR , which negatively regulates genes involved in purine nucleotide biosynthesis ( 24 ; Fig. 1 , lane 4 , and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>This is the result expected if upon binding a fatty acid derivative , FadR is released from all specific DNA-binding sites resulting in repression of fabA and induction of fadSA ( Henry and Cronan , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Amino acid residues D74 and R75 have the same relative positions in GntR as do residues H65 and G66 in FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Here we report mapping of the fadH promoter and document its complex regulation by three independent regulators , the fatty acid degradation FadR repressor , the oxygen-responsive ArcA-ArcB two-component system , and the cyclic AMP receptor pro-tein-cyclic AMP ( CRP-cAMP ) complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Regulation of the fatty acid pathways is complex and involves transcriptional-regulators ( including FadR , FarR , IclR , and CRP ) and post transcriptional mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>DiRusso , C.C , Heimert , T.L. , and Metzger , A.K. ( 1992 -RCB- Characterization of FadR , a global transcriptional-regulator of fatty acid metabolism in Escherichia coii : interaction with the fadB promoter is prevented by long chain fatty acyl coenzyme As .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Mierobiol Rev 55:371 394 DiRusso CC , Heimert TL , Metzger AK ( 1992 ) Characterization of FadR , a global transcriptional-regulator of fatty acid metabol-ism in Eseherichia eoli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>DiRusso , C.C. , Heimert , T.L. , and Metzger , A.K. ( 1992 ) Characterization of FadR , a global transcriptional-regulator of fatty acid metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Under conditions where fatty acids are the predominant nutrient , FadR is inactivated and fatty acid oxidation increases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Thus , through FadR , E. coli regulates its fatty acid metabolism , catabolism , and transport according to the availability of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Summary FadR is an Escherichia coli transcriptional-regulator that optimizes fatty acid metabolism in response to exogenously added fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Reassessment of the genetic regulation of fatty acid synthesis in Escherichia coli : global positive control by the functional dual regulator FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Transcription of the Escherichia coli fatty acid synthesis operon fabHDG is directly activated by FadR and inhibited by ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>n E scherichia coli FadR Positively Regulates Transcriptio of the fabB Fatty Acid Biosynthetic Gene JOHN W. CAMPBELL1 AND JOHN E. CRONAN , JR. 1,2 * Departments of Microbiology1and Biochemistry ,2 University of Illinois at Urbana-Champaign , Urbana , Illinois 61801 Received 2 May 2001/Accepted 24 July 2001 In Escherichia coli expression of the genes of fatty acid degradation ( fad ) is negatively regulated at the transcriptional level by FadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Transcription of the Escherichia coli Fatty Acid Synthesis Operon fabHDG Is Directly Activated by FadR and Inhibited by ppGpp Laetitia My , a Brian Rekoske , b Justin J. Lemke , b Julie P. Viala , a Richard L. Gourse , b Emmanuelle Bouvereta LISM , CNRS , Aix-Marseille University , Marseille , Francea ; Department of Bacteriology , University of Wisconsin -- Madison , Madison , Wisconsin , USAb In Escherichia coli , FadR and FabR are transcriptional-regulators that control the expression of fatty acid degradation and unsaturated fatty acid synthesis genes , depending on the availability of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Reassessment of the Genetic Regulation of Fatty Acid Synthesis in Escherichia coli : Global Positive Control by the Functional Dual Regulator FadR L. My , N. Ghandour Achkar , J. P. Viala , E. Bouveret LISM CNRS , Aix-Marseille University , Marseille , France ABSTRACT In Escherichia coli , the FadR transcriptional-regulator represses the expression of fatty acid degradation ( fad ) genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Assuming that the regulation of FarR by fatty acids is of physiological signi ® cance , these data seem to indicate that the intracellular concentrations of the fatty acid signaling molecules required to release repression of the TCA cycle genes controlled by FarR is much higher than that to release repression of the FAO structural genes by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Due to identi ® cation of FadR , FadL and FACS , we expect that the regulatory link between fatty acid transport , activation and regulation of FadR is operative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Based on the known regulation , overexpression of FadR is expected to slow down fatty acid degradation and enhance unsaturated fatty acid biosynthesis , thus increasing the total fatty acid yield .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Fatty acid biosynthesis and FadR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>These data suggest that the role of FadR in the regulation of fatty acid biosynthesis may not be limited to transcriptional control of fabA , fabB and fabI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In contrast the unsaturated fatty acid biosynthetic gene , fabA , is positively regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>In contrast to Gram-positive bacteria , it was thought previously that not all the fab genes in E. coli were controlled by fatty acid availability ; rather , it was thought that regulation was restricted to unsaturated fatty acid biosynthesis by FadR and FabR regulation of only the fabA and fabB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Experimental evidence that both genes involved in unsaturated fatty acid biosynthesis are regulated similarly would discount the possibility that FadR regulation of fabA is merely fortuitous or vestigial in nature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Other mechanisms that regulate lipid biosynthesis during the E. coli life cycle The mechanisms which regulate the fatty acid biosythetic genes and enzyme activities , apart from FadR , are complex and just beginning to be unraveled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The coeffector specificity of FarR resembles that of the FadR protein of E. coli , which represses fatty acid degradation genes , activates a fatty acid biosynthesis gene @bA ) , and is thought to have a role in regulating the glyoxylate cycle operon , aceBAK [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7805834" target="_blank">7805834</a>]<br></li>
<li>The work in E. coli on the FadR protein has established that this transcription factor negatively regulates genes encoding the enzymes of fatty acid catabolism and activates at least fabA and fabB , the two genes essential for the de novo synthesis of unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Departments of Microbiology and Biochemistry University of Illinois Urbana , Illinois 61601 The FadR protein of E. coli activates transcription of the fabA gene , a key enzyme of fatty acid synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Speciﬁcally , FadR down-regulates several genes in the fatty acid degradation pathway , including fadE , fadBA , fadH , and fadIJ ( Fujita et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>FadR represses the fatty acid degradation ( fad ) system and activates the unsaturated fatty acid synthetic pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>FadR repression and oleate induction of fatty acid transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Perhaps , the primary function of FadR in this pathogen is to activate the fatty acid biosynthetic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR was discovered through the analysis of a mutation that results in the constitutive induction of the - oxidation enzymes ( 7 ) , and led to its characterization as a repressor of fatty acid - oxidation genes ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>The effects of fadR mutations on uspA expression suggest that UspA may have a role in fatty acid metabo-lism ; all genes demonstrated to be regulated by FadR so far have assignments in fatty acid uptake , activation , or metabo-lism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Therefore , FadR appears to be a trans - acting protein which controls fatty acid degradation and acetate degradation by decreasing the expression of the fad and ace structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2843809" target="_blank">2843809</a>]<br></li>
<li>RESULTS FadR activates the global expression of fatty acid synthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , in this paper we reassessed the regulation of fatty acid synthesis gene expression by the functional dual regulator FadR in E. coli and showed that FadR activates all fatty acid synthesis genes that we tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Our results suggest that FadR , as well as controlling fatty acid degradation , could directly activate much of the fatty acid synthesis pathway under at least some conditions through its effects on the fabH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Lastly , we found that overexpression of V. cholerae FadR could also increase fatty acid yield , but to a lesser extent as compared with overexpressing E. coli FadR , ( Fig. 3a and b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Finally , it should be noted that UspA , encoded upstream of uspB , is also implicated in membrane function during stationary phase because it is regulated , in part , by FadR , the global regulator of fatty acid synthesis and degradation ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>To quantify the relative FadR levels in the presence of various arabinose concentrations , we replaced fadR with a red ﬂuorescent protein gene ( rfp ) and measured the relative ﬂuorescence under the same condition as used for fatty acid production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>The ﬁrst indications that fabA was positively regulated by FadR were the discoveries that fadR strains contained abnormally low levels of cellular unsaturated fatty acid and , more deﬁnitively , that fabA ( Ts ) fadR double mutants were unable to grow even at low temperature unless unsatur-ated fatty acids were provided ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The explanation for this ﬁnding is that in the presence of functional FadR , sufﬁcient mutant FabA is produced to provide a level of isomerase activity that satisﬁes cellular unsaturated fatty acid requirements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>1 The abbreviations used are : ACP , acyl carrier protein ; FabA , - hy-droxydecanoyl-ACP dehydratase ; FabB , - ketoacyl-ACP synthase I ; FabF , - ketoacyl-ACP synthase II ; SFA , saturated fatty acid ; UFA , unsaturated fatty acid ; FadR , fatty acid degradation regulator ; and FabR , fatty acid biosynthesis regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Palmitoyl-CoA concentrations as low as 200 nM decreased FadR interactions below detectable levels ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Palmitoyl-CoA Inhibition of FadR DNA Binding Is Reversible Binding reactions were set up as described in Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Finally , the interaction of acyl-CoAs with speciﬁc eukaryotic binding proteins is well understood ( Færgeman and Knudsen , 1997 ) , and it seems likely that FadR uses an analogous mechanism for ligand binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The mechanism by which FadR-dependent regulation in stationary phase occurs will also be a subject of future study ; preliminary work indicates that FadR may not be responding to acyl-CoA thioesters synthesized by fadD , thus either FadR is itself repressed or is responding to a signal molecule synthesized by a novel fadD-independent mechanism in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>This may in turn result in an increase in acyl-CoA which inactivates FadR and would result in a cascade of changes in FadR-dependent gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Thus , the FadR repression may be compromised by the increased cellular acyl-CoAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>As detailed above , the emerging information for E. coli requires that any molecular model seeking to explain the coordinate regulation of lipid metabolism during growth , stress and stasis should include : ( 1 ) control by speci ® c transcription factors including FadR ; ( 2 ) growth rate - and growth-phase-dependent regulation of speci ® c genes and operons which are still largely unde ® ned ; and ( 3 ) regulation by small molecule e $ ectors including ppGpp , cAMP , long-chain acyl-CoA , long-chain acyl-ACP and ( possibly ) acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>For all three systems , NagC , FadR and FruR , release of the bound regulatory protein is achieved by the presence of the inducing metabolite in the cell , GlcNAc-6-phosphate for NagC , long chain acyl-CoAs in the case of FadR and fructose-l-phosphate for FruR ( Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>Linkage of fatty acid-mediated changes in gene regulation to transport and activation In the previous section , the regulation of FadR by long-chain acyl CoA was described in detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In E. coli , expression of the fatty acid degradation ( fad ) genes is inhibited by the transcriptional-repressor FadR in its apo form , but when long-chain fatty acids are available , acyl-CoA binds to FadR , releasing the repressor from the promoter and inducing fatty acid degradation ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Here , we report that FadR also activates the expression of fabHDG genes , which code for enzymes involved in saturated fatty acid synthesis , and that this activation is reversed by the binding of long-chain fatty acyl-CoA ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Perhaps , the fatty acids released by phospholipase actvity are also imported and activated by the bacterial fad transport system resulting in an increase in long-chain acyl-CoA and subsequent inactivation of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Finally , FadM , which is a long-chain acyl-CoA thioesterase involved in the - oxidation of oleic acid , also is repressed by FadR but is not depicted here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Activation of fabA , fabB and fabI by FadR results in high levels of long-chain fatty acid biosynthesis while activity of the fatty acid transport system including FadL and acyl-CoA synthetase , additional fad structural genes and uspA are low due to the combined e $ ects of repression by FadR , low cAMP levels and OmpR-and DnaA - mediated repression of fadL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>When fatty acyl CoA concentrations are low , FadR binds to sites within the promoters of the fad genes to prevent transcription and FadR binds adjacent to the promoters of genes required for fatty acid biosynthesis to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>This induction appears to occur by inactivation of the FadR repressor by long-chain fatty acyl-coenzyme A thioesters ( Sirnons et al. 1980a , b ; Sallus et al. 1983 ; DiRusso et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Recent work from Raman and DiRusso ( 1995 ) characterized the FadR proteins encoded by fadR super-repressor mutants ; mutants refractory to induction of the fad regulon by long-chain fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The resulting protein fusion retained the DNA binding speci ® city of LexA and was inducible by long-chain fatty acids demonstrating the ligand binding function contributed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>) Further , we were the first to show that DNA binding by purified FadR is specifically inhibited by long-chain acyi CoA compounds but not medium-chain acyi CoA compounds or fatty acids of any chain length ( DiRusso et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>FadR is inactivated by long-chain fatty acids after their conversion to acyl-CoA by the acyl-CoA synthase FadD upon import into the cell , releasing ligand-bound FadR from the DNA ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Low concentrations of long-chain acyl-CoAs were readily found to release FadR from operator DNA in vitro , while fatty acids were inactive ( DiRusso et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>For these reasons , acyl-CoAs rather than fatty acids were thought likely to be the ligand that regulates FadR action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>This source of fatty acids was found to induce the fad regulon efﬁciently , and a functional fadD gene was required for this route of induction , indicating that acyl-CoA is the small molecule ligand that regulates DNA binding by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>In vivo data combining transcriptional lacZ fusion and real-time quantitative PCR ( qPCR ) analyses indicated that fadH is strongly repressed by FadR , in agreement with induction of fadH by long-chain fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>It is noteworthy , however , that three amino acid residues when changed result in a supperrepressor phenotype ( not inducible by long-chain fatty acids ) for E. coli FadR that are identical in H. in ¯ uenzae FadR : Y194 , S219 and W 223 [ 120 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In summary , our results in vitro ( Fig. 4 and 5 ) and in vivo ( Fig. 2 and 3 ) demonstrate that FadR directly activates transcription from the fabH promoter and that activation is regulated by long-chain acyl-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Based on our work , FadR is the first transcription factor shown to be directly regulated by acyl CoA compounds ( DiRusso et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The release of FadR from DNA did depend on the length of the acyl chain attached to the CoA ( decanoylCoA was inactive , consistent with the pattern of induction in vivo ) , but this result was compromised by the known direct dependence of detergent strength on acyl chain length ; decanoyl-CoA is a weaker detergent than acylCoAs of longer chain length .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Binding of acyl-CoAs to FadR triggers a conformational change in FadR and releases FadR from its cognate DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>From the LacI paradigm , such mutant FadR proteins should either be defective in acyl-CoA binding or should bind the ligand but fail to undergo the conformational change that releases the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The acyl-CoA binds to FadR causing a conformational change that results in the dissociation of FadR from all operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The physiological ligands that antagonize FadR binding to its cognate operators are LCFA CoA thioesters synthesized by the cytosolic acyl-CoA synthetase , FadD [ 7,10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The mechanism by which FadR is inactivated is unknown but may include accumulation of long-chain fatty acyl-CoA , protein degradation , or other as yet unidenti ® ed transcriptional and/or post-transcriptional mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR Is Released from DNA by Specific Acyl-CoAs The strongest candidate for the physiological ligand modulating Fad & DNA interaction are fatty acyl-CoA thioesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The fatty acid is converted to acyl-CoA , which binds to FadR and releases the operator DNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>However , binding of FadR to its operator is prevented by the binding of either saturated or unsaturated fatty acyl-CoA indistinctively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Long-chain acyi CoA binds to FadR to prevent DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The fadH promoter binds FadR , and binding is speciﬁcally reversed by long-chain acyl-CoAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_160">
Electrophoretic mobility shift assays demonstrated that FadR binds to the fadH promoter region and that this binding can be speciﬁcally reversed by long-chain acyl-coenzyme A ( CoA ) thioesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></label>
<input id="act_similarity_160", type="checkbox">
<div class="ACT_similar">
<li>FadR binds the fadH promoter region , and binding is speciﬁcally reversed by long-chain acyl-CoA thioesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li></div>
<li>These FadR alleles , called superrepressors , are able to-bind to DNA to repress transcription of the fadB gene but were not inactivated by long-chain acyl-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Indeed , FadR recognizes and binds its operator in its apo form , without ligand , while the ﬁxation of long-chain fatty acyl-CoA on FadR triggers its dissociation ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FadR binds DNA as an apoprotein , and it is released when it binds long-chain acyl-CoA ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Therefore , we do not expect that V. alginolyticus FadR binds long-chain acyl-CoA and must be regulated in a di $ erent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>DISCUSSION FadR activates fabH when long-chain acyl-CoA concentrations are low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>In the presence of long-chain acyl-CoA , FadR dissociates from its operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Binding of FadR to the fadBA operator was inhibited specifically by long-chain acyi CoA compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>This was probably due to residual FadR DNA binding activity in the presence of long-chain acyl-CoAs , versus the complete absence of FadR in strain JWC264 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>This is complicated by the fact that the DNA-binding activity of FadR is antagonized by fatty acyl-CoAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>This mutated form of FadR has lost its ability to-bind the effector-molecule acyl-CoA but remains capable of binding DNA , repressing fad genes , and activating fab genes ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>DNA binding activity of FadR ( and hence transcriptional regulation of FadR-responsive genes ) is negatively controlled by long-chain acyl-CoA thioesters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Upon binding long chain fatty acyl-CoA thioesters , FadR dissociates from its binding DNA sites resulting in decreased transcription of fabA and fabB and increased transcription of the fad regulon genes ( Cronan , 1997 ; Henry and Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>It was later demonstrated that FadR directly activates the transcription of fabA and fabB genes by binding its consensus sequence located 30 bp from the transcription start sites ( 2 , 3 ) , and that FadR binding to DNA is prevented by the binding of long-chain fatty acyl-CoA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>However , in vivo it is more di cult to prove that acyl-CoA thioesters accumulate in su cient quantities to inactivate FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR directly binds to the promoters of FA synthesis genes , which is dissociable in an acyl-CoA-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Each sample was assayed four times , and the data are expressed in mean S.D. Panel B , - ga-lactosidase activities of fusion constructs in which the fabA gene was expressed from either P1 ( in the presence of FadR , strain Ymel ) or P2 ( in the absence of FadR , strain K113 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>( B ) EMSAs were performed using puriﬁed 6His-Tev-FadR and PCR products containing the accB , accD , or fabI promoters and in the presence or absence of oleyl-CoA ( C18 :1 - CoA ) or octanoyl-CoA ( C8 :0 - CoA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In most experiments , little or no protection was observed in the presence of FadR alone ( Fig. 4C , compare lanes 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>No fabH-speciﬁc transcript was detected in the absence of FadR ( Fig. 5 , lane 1 ) , but a transcript of the expected length ( 200 nucleotides ) was obtained in the presence of FadR ( lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Proteomic analysis of relative FadR levels showed a 22-fold increase in the presence of 1 % arabinose as compared to the no arabinose control , close to the value measured by RFP ﬂuorescence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Apart from AccD and FabA , we could detect an increase of all the proteins when FadR was overproduced using the pBADFadR plasmid , with the strongest effect observed for AccA and AccC ( 3 - to 4-fold ) ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Hence , the high relative activity seen in cells grown on oleate could be due to a combination of release of FadR repression and gain of transcriptional activation by the CRP-cAMP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>In the absence of the CRP-cAMP complex , repression of fadH expression by both FadR and ArcA-ArcB was very weak , suggesting a possible interplay with other DNA binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>In the absence of the cAMP receptor protein ( CRP ) complex ( cAMP-CRP ) , FadR and ArcA-ArcB repression of fadL and fadD expression was relatively weak ( Fig. 8 ) and similar to that seen with fadBA and fadH [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Repression of fadH transcription by the ArcA-ArcB system is additive to FadR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Thus it seems that regulation by both FadR and ArcA-ArcB rely on the cAMP-CRP complex to activate transcription such that it can be further modulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The corresponding regulatory proteins ( in gray rounded rectangles ) are FadR , the CRP-cAMP complex , ArcA-P , and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>This raised the question of regulatory interactions among the multiple regulators , FadR , ArcA and CRP-cAMP when bound to the respective promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>DadA and FadR phenotypes were checked on TTC media as described under Media and growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3920477" target="_blank">3920477</a>]<br></li>
<li>For example , fadB and fadE are both repressed by FadR in the absence of long-chain FAs and by ArcA-P under anaerobic-growth conditions ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Recent mRNA level measurements of cultures grown under anaerobic conditions indicated that a strain carrying deletions of both arcA and fadR had the same levels of fadH mRNA as a strain that lacked only arcA ( 8 ) , suggesting that FadR binding required ArcA-P binding under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Indeed , a putative FIS binding site ( TTGCA-TATTTTTAACACAA , 236 to 18 ) lies within the fadBA Expression of fadL and fadD are additively repressed by FadR and ArcA under anaerobic conditions The weak effects of FadR inactivation on expression of fadL and fadD suggested that other regulatory proteins may be involved and thus we tested the effects of deletion of the genes encoding the other proteins proposed to-bind these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Cho and coworkers [ 12 ] initially used quantitative RT-PCR to assay regulation of fadL and fadD by FadR and ArcA under anaerobic conditions and reported that deletion of either arcA , fadR alone , or both arcA and fadR resulted in increased fadL expression by 34 - , 4 - or 69-fold , respectively , whereas fadD transcription increased 69 - , 4.5 - , and 84-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>However , our data differed from those of Cho and coworkers in the effects reported on FadR regulation of fadL transcription under anaerobic conditions when ArcA is functional ( we see no effect versus their reported 4-fold increase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Deletion of FadR resulted in only modest derepression of the two LCFA transport system genes ( 2 to 2.5 fold for fadL and 2 to 3-fold for fadD ) in medium with acetate as sole carbon source ( Fig. 4A and B ) whereas expression of fadBA and fadH increased by 5 to10-fold ( Fig. 4C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Expression was also sensitive to the carbon source in strains which retained the ability to regulate iclR : : lacZ with either IclR or FadR , although the range was less pronounced ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Although FadR and IclR mediate the responses of a variety of genes to the carbon source , they also appear to play the paradoxical roles of ensuring that the transcription of iclR does not change under these same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR prevented this increase by activating iclR transcription on carbon sources such as glucose and acetate but not on oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Regulation of expression of the universal shock protein A , fad and fab genes by FadR is growth-phase dependent The ® rst indication FadR was an important player in the adaptive response to growth arrest came from the discovery that the universal stress protein A ( UspA ) was negatively regulated by FadR in a growth-phase-dependent manner [ 113 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>It is expected that expression of other genes regulated by FadR , including the glyoxylate shunt repressor IclR , are also altered in a growth-phase dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>It is worth noting that the FadR transcriptional-repressor , which is regulated by acyl-CoA compounds and shows local similarity to PaaX , is also controlling the expression of genes involved in b-oxidation mechanisms ( 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The expression of the aceBAK operon , which codes for the key enzymes of the glyoxylate cycle , isocitrate lyase and malate synthase , and for isocitrate dehydrogenase kinase/phosphatase , is activated by FruR and IHF and is repressed by IclR , which in turn is activated by FadR ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>The expression of the uspA gene is , however , repressed by FadR during exponential-growth and expression increases as the cells enter stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The results in the preceding sections indicate that the FadR repressor is inactivated or is itself repressed during entry of cells into stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Possibly , the fabR gene is controlled by another factor that ties UFA synthesis to other aspects of bacterial physiology in a fashion analogous to the control of acetate metabolism by FadR by virtue of its ability to control the expression of iclR , a transcriptional-regulator of the glyoxylate bypass ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>To address this issue , we inactivated the FadR binding site of iclR by mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Disruption of fadR or inactivation of the FadR binding site of iclR decreased the expression of an iclR : : lacZ operon fusion , indicating that FadR activates the expression of iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>The extent to which FadR activates iclR can be estimated from the effect of inactivation of the FadR binding site on the expression of iclR : : lacZ ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Based on the position of the putative FadR binding site , it seemed likely that FadR and ArcA negatively regulate fadH expression , as previously reported ( 8 ) , whereas CRP activates fadH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>FadR is known to negatively regulate expression of fadE ( 6 , 15 ) , and thus a FadR binding site should be located upstream of the yafH/fadE coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The gene expressions of fadR and iclR were also higher , where FadR activates iclR , and IclR is known to repress aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19055729" target="_blank">19055729</a>]<br></li>
<li>GS , glyoxylate shunt ; TCA , tricarboxylic acid cycle ; PoxB , pyruvate oxidase ; ICL , aceA gene product isocitrate lyase ; IclR , repressor of the glyoxylate shunt operon aceBAK ; FadR , regulator of fatty acid metabolism that also activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>By extending to the FadR repressor the type of analysis presented here for IcIR and by investigating the respective role and mode of action of each repressor , one can expect to get more insight soon into the molecular mechanism that regulates the expression of the ace operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2001680" target="_blank">2001680</a>]<br></li>
<li>FadR action is straightforward , it acts as a classical LacI-type repressor and only weakly represses fadD and fadL expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Although FadR activates iclR expression about ﬁvefold during growth-on-glucose , it has only a slight effect during growth on oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>In this report , we demonstrate that FadR activates the expression of iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>These results are consistent with our model in which FadR activates iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR activates iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>These observations conﬁrm our conclusion that FadR activates iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Therefore , loss of the protein ( FadR ) that activates expression of the real repressor ( IclR ) mimicked direct repression of the ace operon by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>FadR also activates the expression of iclR , the gene which encodes the repressor protein of the glyoxylate bypass operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_224">
Regulated expression of a repressor protein : FadR activates IclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></label>
<input id="act_similarity_224", type="checkbox">
<div class="ACT_similar">
<li>Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19563847" target="_blank">19563847</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19574213" target="_blank">19574213</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li></div>
<li>Gui , L. , Sunnarborg , A. , Pan , B. , and Laporte , D.C. ( 1996b ) Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>Gui , L. , Sunnarborg , A. , and LaPorte , D.C. ( 1996 ) Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>By regulating the expression of iclR , FadR indirectly regulates not only the aceBAK operon but also acs encoding acetyl coenzyme A synthetase which is also regulated by IclR [ 100 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>To determine whether FadR regulates iclR expression , we constructed an iclR : : lacZ operon fusion and transferred it to the lac locus on the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>This was expected , as there already was not a very strong decrease of the transcriptional fusions in the fadR mutant ( Fig. 2A ) , and even for FabA-SPA , whose expression is strongly dependent on FadR ( 25 ) , the decrease was only 2-fold ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , we predicted that arti-ﬁcial overproduction of FadR from an arabinose-inducible plasmid construct , pBAD-fadR ( pEB1210 ) , might increase expression of promoters activated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Oleate induced fadH expression to essentially the same level as that seen in a FadR strain , indicating that there was little or no residual repression ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>In reverse , in a strain where FadR was overproduced using the pBAD-FadR plasmid , all constructs displayed a drastic increase of expression ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In this study , we used DarcA , DfadR and DarcA/DfadR mutants to analyse the roles played by ArcA and FadR on gene expression of the fad regulon in response to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Thus , at any given time in a population of cells , fadH expression would be regulated by FadR in one cell subpopulation and ArcA in another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Loss of FadR increased fadH expression 7 - to 8-fold , in general agreement with the data obtained by the RT-qPCR assay ( Fig. 4 ) , whereas loss of ArcA increased expression 4 - to 5-fold ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>In contrast , FadR and ArcA seem to act synergistically in fadBA expression under anaerobic conditions in that loss of either repressor results in only a small ( 3 - to 4-fold ) increase in expression whereas loss of both repressors results in a 12-fold increase in expression ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Moreover , fadH expression was increased 8 - to 10-fold under anaerobic conditions upon deletion of both the fadR and the arcA gene , indicating that anaerobic expression is additively repressed by FadR and ArcA-ArcB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>We ﬁrst estimated the relative FadR expression levels under our experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Interestingly , under anaerobic conditions , the expression of fadK is independent of FadR and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Our genomic array experiments indicated that fabB was positively regulated by FadR and that the level of expression was depressed in cultures of the fadR mutant strain , JWC264 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>First , both the decreased expression and the decrease in protein amounts in the fadR mutant compared to those of the wild-type strain show that under wild-type conditions , FadR does activate the expression of the FA synthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>However , as direct FadR regulation of fabF or acc gene expression has not been reported , increases in fabF and accA mRNA levels may be indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>FIG 2 Global activation of FA synthesis gene expression by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>However , several results of our experiments clearly demonstrate the global regulation of FA synthesis gene expression by FadR under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>The aceBAK operon is under negative regulation by IclR , FadR ( indirectly by regulation of the iclR expression ( 62 ) , Crp ( 52 ) , and ArcA ( 63 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17426033" target="_blank">17426033</a>]<br></li>
<li>However , it appears that FadR regulation and IclR regulation combine to maintain a constant level of expression of iclR under the conditions which we have examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>These lines of evidence indicate that FadR is not involved in the regulation of ACP expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655562" target="_blank">8655562</a>]<br></li>
<li>FadR regulates fadM expression only weakly , whereas FadR regulation of fadH expression is quite robust ( Fig. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>This result suggests that FadR regulates aceBAK indirectly by altering the expression of IclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>sion : loss of FadR should result in decreased expression of IclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>This result strongly suggests that FadR positively activates expression of fabB , as is the case with fabA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>This was a surprising result since FadR activates the expression of this construct during growth-on-glucose and acetate but not during growth on oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>The FadR expression level in FRT-DfadE was controlled by varying the concentration of the inducer , arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Finally , it is essential that we elucidate signals and factors which regulate the expression and activity of FadR itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>To test if the effect of FadR on uspA expression was direct , we analyzed the binding of FadR to uspA by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Our results demonstrate that the effect of FadR on expression of fabHDG is direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>FadR , which , in part , regulates uspA expression ( 15 ) , has no effect on uspB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>This suggested a positive effect of FadR on fabA expression , and it was soon dem-onstrated that FadR is a transcriptional-activator that binds to the 40 region of the fabA gene , a site common for activators of 70-responsive promoters ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>They suggested that FadR exerts its effect on phospholipid metabolism by activating , in an undefined manner , fabA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The FadR-independent promoter is thought to ensure continued expression of fabA under conditions where FadR activation is compromised .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>Promoter P2 may provide the needed fail-safe mechanism by ensuring some fabA expression in the absence of FadR activation ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>The fact that the UFA content is not increased in a fadR fabR strain compared with a fadR strain illustrates that the level of fabA expression , supported by the transcriptional activation by FadR , is required for UFA regulation by FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>On the other hand , FadR activation is most important for the regulation of fabA expression , but also contributes to fabB regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Our data argue that the function of FabR is to compensate for the weak FadR regulation of fabB expression in the presence of exogenous UFA ( the converse argument , that FadR functions to compensate for the weak FabR regulation of fabA expression by UFA , also applies ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Numerous genes identi ® ed as induced during infection of a host , involve central metabolic processes and global regulators [ 198 , 274 , 275 ] Therefore , it is expected that the FadR regulon will be induced in the course of infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Further analysis of the results from genome-wide transcriptome analysis of the FadR regulon ( 16 , 54 ) will be needed to identify the full complement of genes that are directly activated or inhibited by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Therefore , FadR seemed to be LacI revisited , except that the fad regulon genes are scattered around the genome and hence FadR must bind several operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Therefore , the cell responds quickly to very low levels of LCACoA to inactivate FadR , derepress the fad genes and decrease expression of the fab genes and iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR is active in DNA binding and the fad genes including fadD and fadL are synthesized at low levels while several fab genes and iclR are synthesized at high levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Despite the presence of dual binding sites FadR only modestly regulates expression of these genes , indicating that the number of binding sites does not determine regulatory strength .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The method of 2 CT ( 34 ) was utilized to measure the relative expression levels of three FadR regulated genes , fadH , fabA ( 16 ) , and fadM ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The other sites are from genes that are negatively regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The weak binding might be explained by a weak afﬁn-ity of FadR for its operators in activated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Repression of these fad genes depends on the available concentration of apo-FadR , which was dynamically regulated by the cellular concentration of acyl-CoAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Transcription of the acc genes dose not appear to be regulated by Crp , IHF or FadR [ 294 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>However , the overall magnitude of the differential transcription of FadR regulated genes was generally lower .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Evidence of a role for FadR in lipid synthesis Several observations led to the proposal that FadR had a role in lipid metabolism apart from repression of the fad genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Sequences corresponding to the genes activated by FadR were aligned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FA degradation enzymes , colored in red , are coded by genes repressed by FadR ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FA synthesis enzymes colored in green are coded by genes activated by FadR ( 8 ; this paper ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>When LCFAs are present in growth medium , the FadR protein is released from the promoter regions , resulting in transcriptional induction of the fad genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Under aerobic conditions , the absence of LCFAs induces FadR to repress transcription of the fad regulon genes by direct binding to their promoter ( Henry & Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Expression of FadR under our conditions increased the transcription of genes for phage shock proteins PspA and PspG by 6.2 - and 8.5-fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>For example , all the fad genes repressed by FadR are also repressed by the ArcA transcription factor ( encoded by arcA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Our experiments do not address stationary phase , but the new information in this report expands the repertoire of - oxidation genes subject to FadR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>However , all genes showed increased transcript levels in the DfadR and DarcA/DfadR mutants relative to the wild type ( Fig. 3 ) , and this is consistent with the current understanding of FadR regulation of these genes ( Henry & Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>In addition to the regulation by FadR and ArcA under aerobic and anaerobic conditions , many fad genes are positively regulated by the Crp regulatory protein in response to chan-ging cyclic AMP levels ( Clark & Cronan , 1996 ; Pauli et al. , 1974 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Indeed , the reported intracellular FadR levels ( Taniguchi et al. , 2010 ) seem too low to account for the observed efficient regulation the FadR controlled genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>It was also reported recently that FadR inhibits a new target involved in phospholipid synthesis in Vibrio cholerae ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The FadL and FadD proteins involved in the uptake and activation of exogenous FA also are repressed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>We also had evidence for a weaker promoter that did not depend on FadR activation and would function when activation by FadR was not available ( see Introduction ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>It would be interesting to check whether other repressor proteins , namely FadR , could also be purified by this efficient procedure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2001680" target="_blank">2001680</a>]<br></li>
<li>5 ' - ATCTGGTGTGGGATGGC-3 ' would also bind FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Residues identi ® ed in FadR which when changed result in a dominant negative phenotype are in bold and are marked with a double underline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>As mentioned above , a FadR homologue is also identi ® ed , although by comparison to the H. in ¯ uenza , this gene is more divergent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In the absence of FadR activation , we conclude that FabA activity is limiting for UFA synthesis , whereas in the presence of FadR activation , the level of FabB activity controls UFA production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>In this context , we do not expect to observe any effect on the ﬂux of FA synthesis if this global activation by FadR would be missing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Indeed , given that the ﬂux magnitude is controlled mainly by allosteric enzyme regulation ( 1 ) , the decrease in enzyme amounts observed in the absence of FadR ( Fig. 7 ) certainly could not impact FA synthesis activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>For the reasons just explained , because we do not expect to see an effect of FadR directly on FA synthesis activity , it might be difﬁcult to demonstrate the importance of this regulation on the physiology of the bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FadR regulation provides a means to decrease UFA synthesis when UFAs are present in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>FIG 7 Impact of FadR regulation on the abundance of FA synthesis enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Is regulation of FadR , its sole function or do LCACoAs serve preferentially as precursors to phospholipids when they are available from the host environment ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The major remaining question is how global is FadR regulation , both in E. coli and among bacteria in general .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Therefore , FadR regulation may be broadly distributed in Gram-negative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Thus , oleic acid uptake is reduced by high osmolarity , at least in the absence of functional FadR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>FadR can act as both a transcriptional-repressor and an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Regulation of UFA content by the FabR repressor requires the presence of the FadR activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>FadR FadR was ﬁrst thought to be only a straightforward and rather boring repressor that regulated the b-oxidation pathway of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_310">
Overlapping repressor binding sites result in additive regulation of Escherichia coli fadH by FadR and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></label>
<input id="act_similarity_310", type="checkbox">
<div class="ACT_similar">
<li>Overlapping repressor binding sites result in additive regulation of Escherichia coli FadH by FadR and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li></div>
<li>A transcriptional-regulator such as FadR , having both positive and negative regulatory roles , provides a good test of this experimental approach .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>This regulation occurs , in part , through the actions of the transcriptional-regulators CRP , FNR , IclR , and FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Therefore , transcriptional control of acetate metabolism by FadR and IclR is predicted to be complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2843809" target="_blank">2843809</a>]<br></li>
<li>Additionally , physical interactions between Crp and/or Fur with other regulators such as IHF , Cra , and FadR could be important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659676" target="_blank">15659676</a>]<br></li>
<li>Activation by FadR is dependent on the carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Lanes : 1 , template alone ; 2 , 1 M FadR ; 3 , 15 nM RNAP ; 4 , 1 M FadR plus 15 nM RNAP ; 5 , template alone ; 6 , 500 nM FadR ; 7 , 1 M FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Lanes 1 through 6 resulted from reactions containing 0 , 0.2 , 1.0 , 1.2 , 6.0 , and 0 mg of puriﬁed FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>However , despite these unconventional attributes , FadR seems to perform its duties in a conventional manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>This simple scheme seems to readily explain FadR action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Given this latter role it seems surprising that FadR fails to repress its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>In this section we review the role of FadR and other mechanisms of regulation that modulate lipid composition and metabolism under conditions that lead to growth arrest and stasis In prokaryotes , conditions that result in growth arrest are characterized by a generalized cessation of DNA , RNA , stable protein and phospholipid biosynthesis [ 209 , 215 , 216 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Future work will also include examination of the importance of alterations in membrane composition under the diverse stresses which are known to induce UspA and whether FadR regulation of UspA is sufﬁcient to induce the protein in all these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>We found no significant differences between strains that lacked FadR and those containing normal levels of the protein ( data not shown ) ; thus , we conclude that FadR is not autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Interaction between a regulatory ligand and FadR should be readily reversible , whereas renaturation of a denaturated protein is a slow and inefficient process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>FadR protein was puriﬁed by nickel chelate chromatography of a polyhistidine-tagged fusion protein known to be fully functional in vivo ( 5 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The FadR protein was used at concentrations of 0 ( ) and 0.2 , 0.5 , 1.0 , 2.0 , 5.0 , and 7.5 M , depicted by the triangle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>( B ) Transcriptional fusion activity when FadR protein is overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>These data indicate that the growth-phase-dependent regulation of the universal shock protein is in part dependent upon FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In our experiments we used ΔfadR strains lacking the FadR activator protein in order to limit regulation to FabR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Binding of FadR to OA ( found at -47 through -31 ) may facilitate interaction between FadR and RNA poiymerase bound to the fabA promoter as is typical for activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>However , in contrast to accA , the accBp2 promoter always appeared to be preferred to the accBp1 promoter , even in the absence of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>The second promoter is constitutive and has an activity equivalent to that seen for the proximal promoter in the absence of FadR activation ( thus , the overall range of FadR regulation of fabA is 10-to 12-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>This promoter has an activity equivalent to that of the main promoter in the absence of FadR activation ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>We proposed that FadR activation of fabA * Present address : Department of Molecular Biology , Princeton University , Princeton , New Jersey 06544-1014 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>This di $ erence in binding a nity is re ¯ ected in limited activation of the fabB promoter by FadR compared with activation of the fabA promoter as noted above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Therefore , researchers have always wondered why FadR would be involved speciﬁcally and only for fabA and fabB activation ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Transcriptional fusions containing the fabA , fabB , and fadE promoters , previously reported as regulated by FadR or FabR , were used as positive controls ( 15 , 16 , 54 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Clark ( 6 ) used strains carrying this allele to show that fadE transcription is negatively regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>Mass spectrometry analysis identified the 26 kDa protein bound specifically to the fabA , fabB and yqfA columns as YijC and the protein bound to the fadB column as FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>FadR , whose principal role ﬁrst was discovered to be a repressor of FA degradation genes in E. coli , later was shown to activate the expression of fabA and fabB genes involved in unsaturated FA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , although it long has been postulated that FadR only activates the fabA and fabB genes , we suspected that FadR is the missing global regulator of FA synthesis genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>A reduced amount of FadR protein might then reduce the expression of other genes activated by FadR , such as fabA , fabB , and iclR , in addition to fabH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Our initial ﬁnding that the fabH promoter was directly activated by FadR ( 8 ) , the global increase in the expression of FA synthesis genes in a strain overproducing FadR ( 9 ) , and the potential presence of a consensus sequence for FadR binding in the promoters of FA synthesis genes other than fabA and fabB were strong indications that FadR activates the transcription of genes additional to those previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Expression of fabA and fabB is repressed by FabR , which binds a site overlapping their promoters , and is activated by FadR , which binds a consensus site located around the 40 position ( 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In addition to the proximal promoter of fabH and the promoters of fabA and fabB already described , this suggested that accA , accBC , accD , fabI , and acpP-fabF genes also were activated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>This was already well shown before by quantitative measurements of FadR afﬁnity for fabA , fabB , and fabI promoters , listed here by decreasing afﬁnity , well behind the fad genes ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>The remaining three genes , fabA , fabB , and fabI are activated by FadR ( [ 108 ] and DiRusso , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>fabA and fabB genes are repressed by FabR and activated by FadR ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Single-round in vitro transcription from the fabH plasmid template in the presence or absence of 2 M FadR , together with octanoyl-CoA or palmitoyl-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>( B ) Single-round in vitro transcription from the fabH and fabA plasmid templates in the presence or absence of 2 M FadR , 2 M DksA , and 100 mM ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Exogenous UFAs are converted to CoA esters that bind FadR and release it from the P1 promoter resulting in decreased fabA transcription ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>FadR was required for fabH and fabA promoter activity , as shown above ( Fig. 5 ) , and transcription from the FadR-activated fabH and fabA promoters was inhibited by ppGpp/DksA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Furthermore , we show that transcription in vitro of fabA is activated by FadR while transcription of fadL is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The binding site for fabA activation is centered at -40 , while the fadSA - and fadL-binding sites ( where FadR represses transcription ) are centered at +9 and -17 , respectively ( Figures 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The FadR -- acylCoA thioester complex is unable to-bind the operator sequence , resulting in transcription of the fad genes and the loss of activation of fabA transcription ( 5 -- 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>B ATGC F Discussion The regulation of fabA transcription by FadR and acylCoAs is the first reported example of repression mediated by a positive activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The remaining attractive hypothesis was that FadR positively regulates fabA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The absence of an increase in FabA-SPA level is consistent with our previous observation that overproducing FadR does not increase fabA transcription very much ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FadR activates transcription of fabA and fabB when it binds in its apo form ( 15 , 16 ) , whereas FabR represses transcription of fabA and fabB in the presence of unsaturated fatty acids ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The fadR gene ( not shown ) produces FadR , which binds to the various operators , repressing transcription of fadBA and fadE , while activating transcription ( sawtooth lines ) from the fabA and iclR promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>( iii ) FadR activates the transcription of at least four genes , fabA , fabB , fabI and iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR positively regulates transcription of fabA , fabB , and iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>In contrast , FadR activates transcription when it binds just upstream of the promoters of fabA and iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>In contrast , fabA transcription is stimulated by the addition of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Third , LCACoAs bind the transcription factor FadR and prevent DNA binding and transcriptional activation or repression [ 108 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Overall , these experiments demonstrated that the affinity of FadR binding for DNA containing the fadB , fadL and fabA promoters was Og > OLI , OL2 > OA - We could not distinguish separate binding affinities for 0 ^ or OL2 - We demonstrated repression of fadL transcription and activation of fabA transcription in vitro using run-off transcription assays containing purified FadR and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Finally , DiRusso et al. ( 1992 ) demonstrated that FadR represses the transcription of fadBA and activates the transcription of fabA in a deﬁned in vitro system ( ii ) FadR is a DNA binding protein speciﬁc for sequences found upstream of fabA and the fad genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>We show that FadR binds to a region of DNA adjacent to the fabA promoter as is typical of activators of o ^ ° - RNA poiymerase holoenzyme ( Collado-Vides et ai , 1991 ) and that purified FadR activates fabA transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>When FadR was produced by induction of the pBAD promoter , the amounts of the FabH - and FabD-tagged proteins increased more than 3-fold , although the FabG - and FabF-tagged proteins increased by only 40 % and 70 % , respectively ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>FadR also increased the activity of the acpP promoter in vivo , although to a lesser extent than that for fabH ( Fig. 2 ) , but further studies will be needed to determine whether the effect of FadR on this promoter is direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>We recently showed that FadR also activates the promoter upstream of the fabH gene ( L. My , B. Rekoske , J. J. Lemke , J. P. Viala , R. L. Gourse , and E. Bouveret , J Bacteriol 195:3784 -- 3795 , 2013 , doi :10.1128 / JB.00384-13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>S2B and its legend in the supplemental material ) , probably because fabH was expressed by read-through transcription from the relatively strong yceDp1 and yceDp2 promoters ( see also references 35 and 36 ) , FadR expression did not increase FabH-SPA production in this construct ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>To conﬁrm that the increase in the FabH-SPA protein was the direct consequence of activation of the fabH promoter by FadR , we deleted the plsX gene in the FabH-SPA strain , thereby eliminating the fabH promoter ( which is located in the plsX open reading frame ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>This is consistent with the interpretation that the increase in FabH protein produced in the context of the wild-type locus derived from activation of the fabH promoter by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>These results suggested that FadR activates transcription of fabH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The results described above were strong evidence that FadR directly activates all of the studied promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Then , we demonstrate that the promoter driving expression of the fabHDG operon ( the fabH promoter ) is in fact activated directly by FadR and inhibited directly by ppGpp / DksA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Indeed , we previously showed that FadR also activates the promoter just upstream of fabH , thereby contributing to the increase in fabHDG-acpP-fabF expression ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>This enabled us to show that the accBp2 promoter was active and activated by FadR ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FIG 5 The fabH promoter is activated by FadR in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The promoter located adjacent to the coding sequence requires FadR for optimal expression and is activated about 20-fold upon Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Our experience with the fabH promoter suggests that the FadR-DNA complex may be short lived , but interactions between RNAP and FadR increase its longevity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Mutations in the FadR site upstream of the fabH promoter reduced transcription to the level observed in a fadR mutant , in the presence or absence of FadR ( Fig. 4B ) , consistent with the model that FadR activates the fabH promoter directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>However , in the presence of both FadR and RNAP , strong protection was observed in the region typically recognized by RNAP FIG 2 FadR activates the fabH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>ppGpp also inhibits the fadR promoter , thereby reducing transcription activation of fabH by FadR indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>In order to address whether FadR is necessary and also sufﬁcient for activation of transcription , we performed in vitro transcription assays on the fabH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>FadR binding to the fabH promoter and stimulation of transcription initiation in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Taken together , the position of the FadR binding site with respect to the binding site for RNAP suggests a model in which FadR activates transcription directly by binding to the DNA cooperatively with RNAP , helping to recruit the transcription complex to the fabH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Likewise , when the 950-bp fragment was digested with SphI only one of two resulting fragments was bound by FadR , as shown in lanes 3 and 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>If the equilibrium binding constant is defined as that concentration of FadR which caused a 50 % shift of the unbound fraction to the the first band representing a single occupied site the result was approximately 1-5 x 10 '' ' ° M. Above 1 x 10 '' ^ M no unbound DNA was detected and the second bound fraction was detected In addition to the first bound fraction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>For each of the promoters , PL and PA , binding ol FadR was inhibited by the addition of oleoyl-CoA ( 50 nM was tested ; data not shown ) as has been demonstrated with the fadB promoter ( DiRusso etai , , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>In the presence of FadR , the 605-bp fragment was shifted , indicating that FadR speciﬁcally binds to that DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>In the presence of FadR , only the 704-bp fragment was shifted ( lane 5 ) , indicating that FadR speciﬁcally bound to sequences present only on that DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>When FadR binds acyl-CoA , the ability of the protein to-bind DNA is greatly decreased and it no longer functions as an effective transcriptional-repressor or activator ( 11 , 19 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Our results demonstrate that FadR binds to regions within fabA and fadL DNA with high affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>To determine whether or not the DNA upstream of the fabB coding sequence binds FadR , an in vitro gel shift analysis was 33 conducted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>( A ) In its apo form , FadR binds its operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>The FadR protein binds DNA as a homodimer [ 118 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The LC Acyl-CoA binds to FadR and changes the conformation of the protein so that it no longer binds DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Puri ® ed FadR binds to DNA with high a nity [ 111 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>, FadR ) binds to the DNA enabling RNA polymerase to act .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>FadR acts as a repressor when it binds to a site which overlaps the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Puriﬁed FadR binds to DNA sequences upstream of the fabB 10 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>FadR protein binds upstream of the uspA coding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>FadR binds upstream of fadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>To address whether the effect of FadR on the fabH promoter was a direct effect of FadR binding to the predicted site , we puri-ﬁed FadR as a 6His-tagged recombinant protein and DNase I footprint assays were performed ( see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>One possibility is that each protein sterically blocks binding of the other such that at a given promoter either FadR or ArcA is bound in a stochastic manner ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>To localize and characterize amino acids in FadR required for binding of LCACoA , non-inducible mutations in the FadR gene were selected after chemical mutagenesis of plasmid DNA encoding FadR [ 120 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_410">
2 and indicate , in panel a , residues implicated in effector binding and dimerization of the FadR protein ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></label>
<input id="act_similarity_410", type="checkbox">
<div class="ACT_similar">
<li>2 and indicate , in FadR , residues implicated in DNA binding and dimerization ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li></div>
<li>It is not clear whether this protein is a FadR functional homologue with regards to DNA binding speci ® city and regulation by LCACoA or if it is merely a structural homologue as many other GntR/FadR H ± T ± H family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR is predicted to be a helix -- turn -- helix DNA binding protein of the GntR family ( Haydon and Guest , 1991 ) , and thus probably binds DNA in a conventional manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Control sites : FadR site upstream of fadB , 5 ′ - GAACGACTC-ATCTGGTACGACCAGATCACCTAA-3 ′ ; PurR site upstream of purH , 5 ′ - GAACGCATTGTAACGAAAACGTT-TGCGCAACG-3 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>The data set produced by comparing the global transcriptional activities in a wild-type FadR strain in the presence and absence of oleic acid was similar to that observed in the FadR mutant versus wild type experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The sensitivity of the transcriptional fusions to the presence of FadR strongly suggested the direct activation of all of these promoters by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>The rate of [ i-oxldatlon was unaffected by the presence of the / acZ fusion and demonstrated control of p-oxldatlon by FadR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_418">
The presence or absence of FadR in the binding assay is indicated by or at the top of each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></label>
<input id="act_similarity_418", type="checkbox">
<div class="ACT_similar">
<li>The presence or absence of FadR in the binding assay is denoted by or , respectively , at the top of each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li></div>
<li>The assay tests the ability of FadR to speciﬁcally bind and retard the electrophoretic mobility of the fabB-associated putative FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The experiments test the ability of FadR to speciﬁcally bind and retard the mobility of a 924-bp 33P-labeled DNA fragment containing the yafH/fadE-associated , putative FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The difference in apparent affinity of FadR for binding to each of the sites within fadL and fadD may be the result of the inherent affinity of each site for FadR or may be due to co-operative interactions between FadR molecules bound to operator pairs or to a combination of inherent affinity and co-opera-tivify .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The two fadL FadR sites bound FadR with equivalent affinities ( essentially complete binding of the probes by 50 nM FadR , Fig. 3B and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The binding conditions are given in the text , and the ratios of FadR dimer to target DNA are essentially equimolar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The CRP , FadR , and ArcA binding sites are indicated with blue , gray , and black rectangles , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The regulatory effects of FadR and ArcA are additive , which seems surprising given that their binding sites overlap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The location of the predicted binding sequence and the data discussed above suggests that FabR regulates by repression and antagonizes the action of the FadR activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>The effect of inactivation of the FadR binding site was slightly greater than that observed when fadR was disrupted in an iclR1 background ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>The straightforward explanation would be direct repression of the ace operon by FadR , but no FadR binding site could be found within the aceAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>As expected , the mutation in the binding site of FadR prevented both the repression of accBp1 by FadR ( compare accBP1 * to accBP1 in Fig. 5D ) and the activation of accBp2 by FadR ( compare accBP2 * to accBP2 in Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>However , we wanted to unambiguously demonstrate the direct binding of FadR to the identiﬁed motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>For the assessment of relative binding affinity of FadR in the range of 1 to 10 x 10 '' ' ° M to OQ , OLI , OL2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>FadR Does Not Regulate Its Own Synthesis Many transcriptional-activators autoregulate transcription of their structural gene ( Raibaud and Schwartz , 1984 ) and therefore we tested whether FadR is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>In contrast , gene fusions of the fabI promoter to lacZ are weakly attenuated ( approximately 2-fold reduction ) when FadR is mutated leading to speculation that FadR regulation of this gene is not of major signi ® cance in vivo ( DiRusso , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Derepression of fadF appears to be the result of the combined e $ ects of inactivation of FadR , cAMP ± CRP dependent gene activation and , to a lesser extent , accumulation of the alarmone ppGpp [ 262 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>We demonstrate that stationary-phase activation of the uspA promoter is in part dependent on growth-phase-dependent inactivation or repression of the FadR regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>In this study , we demonstrate that stationary-phase induction of uspA is dependent on the status of the fadR gene and that the FadR regulon and uspA are induced concomitantly in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>We could also observe induction of omrA-lacZ by pFadR on MacConkey Lactose Ampicillin plates ( Table 1 and data not shown ) , but the corresponding β-galactosidase activity is too low to be detected in liquid assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Therefore , the activation of the omr-lacZ fusions by pYqjA-YqjB , pYgdD-GcvA or pFadR requires EnvZ-OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>In the wild-type strain , pYqjA-YqjB , pYgdD-GcvA and pOmpR activate both fusions and pFadR activates omrB-lacZ ( Fig. 3 ) , with the strongest signal from pYgdD-GcvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>In this context , pYqjA-YqjB , pOmpR and pFadR activate both fusions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">FadR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Recently , our laboratory has purified FadR and characterized its interaction with the promoter of the fadBA operon which encodes two proteins which form a multi-enzyme complex that carries out at least five enzymatic reactions of [ i-oxidation ( DiRusso etai , , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>In the absence of FadR activation , we conclude that FabA activity is limiting for UFA synthesis , whereas in the presence of FadR activation , the level of FabB activity controls UFA production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Because mutating residues in the interior of a protein can also lead to protein unfolding , we generated two distinct mutants : a double mutant in which Glu93 and Arg139 were substituted with Ser and Phe , respectively , the structurally homologous residues in FadR ( the residue structurally homologous to McbR Arg89 is also an Arg in FadR ) and a triple mutant in which all three residues were mutated to alanines ( Arg89Ala , Glu93Ala , and Arg139Ala ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>After removal of contaminant proteins with washing buffer containing 20 mM imidazole , the His-tagged FadR protein was eluted in elution buffer containing 100 mM imidazole .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>YncC belongs to the GntR subfamily of FadR ( pfam007729 ) , which binds-as-a-dimer-to-its-operator site via its winged helix domains ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In addition , FadR uses a winged-helix ( WH ) motif to contact the adjacent minor grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>The other two mutations are outside of the helix-turn-helix motif and do not correspond to residues identi ® ed as negative transdominant for FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Three-dimensional modeling of the NanR N terminus with the equivalent FadR domain indicates topological similarity between the predicted structure of NanR and the corresponding structural elements of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>This region in NanR was previously hypothesized to be the recognition domain on the basis of its similarity to FadR ( 7 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Though not shown in Fig. 5 , the C-terminal NanR effector-binding domain is predicted to be composed of either turns or - helical secondary structural units ( 6 ) , which is topologically identical to FadR and clearly places NanR in the FadR subfamily of bacterial HTH regulators despite the fact that NanR uses overlapping binding sites instead of wings for speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>In this subfamily , the proteins consist of an all-helical C-terminal domain ( Fig. 2a ) with seven or six - helices for the FadR and VanR subgroups , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>The crystal structure of the C-terminal domain of FadR ( Protein Data Bank code 1EX2 ) has been determined ( 52 , 53 ) and , according to our comparative study , its relative threedimensional data could be used as a scaffold to orient studies to the entire subfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>The fusion between the D-b domain and the E-b/O domain should have occurred separately for the FadR , HutC , and MocR/YtrA subfamilies , and none of the four subfamilies has emerged from one of the three others by internal molecular rearrangements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>The C-terminal domain of NanR has predicted secondary structure most similar to FadR , placing it in the FadR subfamily of the GntR superfamily ( 6 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Detailed bioinformatics studies have led to the deﬁnition of at least 7 families ( AraR , DevA , FCD , HutC , MocR , PlmA , and YtrA ) , which are classiﬁed by the effector-binding domain topology and secondary structure ( Figure 1A ) .7 The majority of GntR regulators belong to the FadR C-terminal domain family ( FCD , Pfam PF07729 ; Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>The secondary structure elements and topology of the C-terminal domain places McbR in the FadR C-terminal domain ( FCD ) family of GntR transcriptional regulators ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>Our comparative study first focused on the C-terminal E-b/O domains and next on DNA-binding domains and palindromic operator sequences , has classified the GntR members into four subfamilies that we called FadR , HutC , MocR , and YtrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>DISCUSSION The structural , phylogenetic , and functional analysis of about 270 members of the bacterial HTH GntR-family led us to limit the C-terminal E-b/O domain heterogeneity to four major subfamilies that we called FadR , HutC , MocR , and YtrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>According to this analysis , the GntR-family has been divided into four major subfamilies , FadR , HutC , MocR , and YtrA , where each subfamily is categorized by a specific type of the C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16672238" target="_blank">16672238</a>]<br></li>
<li>However , other major groups within the GntR-family di ¡ er from FadR other than in their C-terminal domains [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12757941" target="_blank">12757941</a>]<br></li>
<li>FadR has a carboxyl-terminal domain that is homologous to the regulatory domains of the TetR family members ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>S2 support the overall importance of the Nan box for NanR binding and repression of the sialoregulon using a repressor in the FadR family of helix-turn-helix ( HTH ) regulators with , however , atypical responses to inducer , its oligomerization state , and binding mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>This structure demonstrates an unusual interaction of FadR with its 17-bp pseudopalindromic DNA-binding site , in which the rec-ognition helices of both HTH motifs of the FadR homodimer interact with the same major groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>In contrast , the bacterial wHTH regulator FadR binds as a dimer after undergoing a structural change caused by effector-binding to the C terminus such that both recognition helices of the homodimer interact with a single major groove , with one wing of each monomer contacting the ﬂanking minor grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>We are presently conducting saturation mutagenesis of OB to determine which residues are critical for FadR binding and to evaluate the contribution of each nucleo-tide pair to binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Third , LCACoAs bind the transcription factor FadR and prevent DNA-binding and transcriptional activation or repression [ 108 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Fatty acyl-CoA binding domain of the transcription factor FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Crystal structure of FadR , a fatty acid-responsive transcription factor with a novel acyl coenzyme A-binding fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16672238" target="_blank">16672238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18263722" target="_blank">18263722</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_29">
Analysis of acyl coenzyme A binding to the transcription factor FadR and identiﬁcation of amino acid residues in the carboxyl terminus required for ligand binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></label>
<input id="dom_similarity_29", type="checkbox">
<div class="DOM_similar">
<li>Raman , N. , and DiRusso , C.C. ( 1995 ) Analysis of acyl coenzyme A binding to the transcription factor FadR and identiﬁcation of amino acid residues in the carboxyl terminus required for ligand binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li></div>
<li>Within each sequence the FadR binding sites identified in Figs 5A , 5B , 6A and 6B are underlined and the central residue is marked by an asterisk ( * ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>( After all , one would not be surprised to ﬁnd that SDS disrupted the FadR -- DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>In this report , we evaluate FadR interaction-with-DNA containing the fabA and fadL promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>FadR shares amino acid identities in this region with the GntR-family subclass of H ± T ± H proteins [ 121 ] ( PROSITES domain PDOC00042 [ 122 , 123 ] and ProDom domain number ID 586/ProDom36 [ 121 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The speci ® city of binding to the cognate recognition site ( s ) is generally dependent upon one or more amino acids in the regulator that contact one or more unique bases in the recognition site ( or , as is the case for FadR , a series of recognition sites ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The italicized amino acids 34 ± 54 in FadR are predicted to form a helix-turn-helix domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The amino terminal sequence of the labeled peptide was SLALGFYHK which corresponds to amino acids 187 ± 195 in FadR [ 119 ] 3.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The average C-terminal length of the FadR and VanR subgroups is , respectively , about 170 and 150 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>This could imply that NanR makes contacts with DNA similar to those made by FadR , with the nonidentical amino acids conferring speciﬁcity on the NanR operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>The solid arrows indicate amino acids that are identical in FadR and NanR , while the dotted arrows indicate those amino acids without homology to NanR that could be responsible for speciﬁc recognition , as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>These are also both members of the FadR subfamily , but they have an additional helix between the wHTH and the FCD ligand binding domains ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>Residues from helix α2 often contribute to DNA-recognition , as has been observed for the FCD transcription factor FadR .9 Second , the two domains in McbR differ not only in their relative orientations to the FCD-domain but also in conformation , with an rmsd of 1.2 Å ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>Due to the alteration in binding a nity of the puri ® ed FadRS219N protein , the non-inducible phenotype of several proteins carrying alanine substitutions and similarities to CODH , it is likely this region of FadR forms part of the acyl-CoA binding domain and in particular is required for binding the CoA moiety .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The acyl-CoA binding domain of FadR was further localized by a nity labeling of the full length protein and an amino terminal deletion derivative , FadRD1 ± 167 , with a palmitoyl-CoA analogue , 9-p-azidophenoxy [ 9 - H ] nonanoic acid-CoA ( [ H ] APNA-CoA ) [ 119 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Localization and characterization of the long-chain fatty Acyl-CoA binding domain of FadR 143 3.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Localization and characterization of the long-chain fatty Acyl-CoA binding domain of FadR Isothermal titration calorimetry was used to estimate a nity of the wild-type full length FadR , a His-tagged derivative and FadRD1 ± 167 for acyl-CoA [ 119 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Finally , the interaction of acyl-CoAs with speciﬁc eukaryotic binding proteins is well understood ( Færgeman and Knudsen , 1997 ) , and it seems likely that FadR uses an analogous mechanism for ligand binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>This was probably due to residual FadR DNA-binding activity in the presence of long-chain acyl-CoAs , versus the complete absence of FadR in strain JWC264 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>In addition , the DNA-binding activity of FadR is speciﬁcally antagonized by acyl-CoAs ( Cronan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>A third probe , in which the FadR binding site was centrally located , was also constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The last line shows the putative FadR binding site located upstream of yafH/fadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The GntR and FadR binding sites within the respective promoters contain a region of dyad symmetry , which is located very close to the transcription initiation sites ( 71 -- 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The FadR binding sites upstream of fabA and iclR are located in the ¹ 40 region and , thus , transcriptional activation probably results from interaction of a FadR surface with the alpha subunits of RNA polymerase ( Gralla , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Moreover , amino acids in position 39 of the HTH domain , also involved in binding to DNA by FadR and AraR ( Table 1 ) , are correlated with central nucleotides in positions 9/11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>Correlations are also observed for nucleotides 16-17/25 -26 and amino acids 37 and 39 , and that conforms well to the structural data for FadR , where these positions are important for the interaction-with-DNA ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>This indicates the importance of the ® rst two residues in the FadR binding site in protein-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_57">
2 and indicate , in panel a , residues implicated in effector-binding and dimerization of the FadR protein ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></label>
<input id="dom_similarity_57", type="checkbox">
<div class="DOM_similar">
<li>2 and indicate , in FadR , residues implicated in DNA-binding and dimerization ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li></div>
<li>However , we note that 9 of the 13 FadR amino acid residues lying within hydro-gen-bonding distance of the DNA target are potentially shared with the NanR N terminus ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>The arrows indicate FadR residues within hydrogenbonding distance of DNA targets in the cocrystal ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>The results of these analyses con ® rmed the importance of R35 , H65 , and G66 and identi ® ed only one additional residue , R49 , as conferring a negative transdominant phenotype , suggesting that these amino acid side chains form contacts with the FadR ± DNA-binding site within fadB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Figure 5A shows the FadR amino acid residues within contact distance of the DNA-binding site observed in the DNA-FadR cocrystal ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>All amino acid residues involved in DNA-binding identi ® ed by mutational analysis of E. coli FadR are identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR is predicted to be a helix -- turn -- helix DNA-binding protein of the GntR-family ( Haydon and Guest , 1991 ) , and thus probably binds DNA in a conventional manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The PaaX protein ( 316-amino acid length ) contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif predicted to be important for DNA rec-ognition and binding in other transcriptional repressors such as GntR ( 71 ) and FadR ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein with a stretch of 25 amino acids ( residues 39 -- 64 ) that shares signiﬁcant similarity with the helix-turn-helix motif for DNA-recognition and binding of transcriptional regulators from the GntR-family such as FadR ( DiRusso et al. , 1992 ) and GntR ( Haydon and Guest , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein that shows 41.4 % amino acid sequence identity with its ortholog PhaN ( recently renamed as PaaN ( 4 ) ) from the PA degradation pathway of P. putida U ( 1 ) and contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif for DNA-recognition and binding of transcriptional repressors from the GntR-family such as FadR ( 5 ) and GntR ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>For Vibrio alginolyticus , the open reading frame encoding the putative FadR homologue ( D83728 ) is only 158 amino acids and this protein is essentially missing the carboxyl terminal ligand binding domain ( sequencing error not withstanding ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The identities and similarities are more limited within the C-terminus that contains the FadR LCACoA ligand binding domain ( Fig. 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The structure of FadR ( PDB : 1H9G ) revealed a C-termi-nal ligand-binding domain that assumed a novel K-helical fold with seven helices [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12757941" target="_blank">12757941</a>]<br></li>
<li>This observation suggested that this domain was unrelated to the entirely K-helical C-terminal ligand-binding domain of the FadR subfamily [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12757941" target="_blank">12757941</a>]<br></li>
<li>The resulting protein fusion retained the DNA-binding speci ® city of LexA and was inducible by long-chain fatty acids demonstrating the ligand binding function contributed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The discrepancy between the carboxy termini is intriguing in that it may indicate the two transcription factors bind DNA with the same speci ® city but the V. alginolyticus FadR is missing the ligand binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Taken together , these ® ndings for FadR and GntR indicate that amino acids required for DNA-binding are not limited to the 20 amino acids that make up the putative H ± T ± H domain but include amino acid residues more amino terminal ( e.g. , A9 in FadR ) and more C-terminal ( e.g. , residues D74 and R75 in GntR and residues H65 and G66 in FadR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>At least 35 proteins have been placed in the GntR-FadR family of transcriptional regulators due to amino acid identities and similarities in this amino terminal DNA-binding domain ( ProDom Domain 586 [ 124 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The data suggest that amino acids in the homologous region of the GntR/FadR family of H ± T ± H proteins are required for DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR homologues and relatives The most prominent sequence similarities between FadR and other proteins in the databases are within the amino terminal DNA-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>This was determined to be the case by phenotypic analyses of protein fusions between the DNA-binding domain of LexA ( amino acids 1 ± 87 ) and amino acids 102 ± 239 of FadR [ 118 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR is released from its DNA-binding sites by long-chain acyl-CoAs ( 9 -- 12 ) , which bind to the carboxyl terminus of the protein and release the amino-terminal winged helix domain from the DNA ( 13 -- 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>At least two functional domains for FadR are distinguished by molecular genetic and biochemical analyses , an amino terminal DNA-binding domain and a C-terminal ligand binding domain [ 118 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Comparison of the proteins indicates that identities within the amino terminal half of FadR which includes the DNA-binding domain are very extensive an the two proteins most likely bind the same recognition sites in DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>According to FadR structural data , we can consider that the N-terminal DNA-binding domain of all GntR-like members contains a small - sheet core and three - helices , the HTH motif being formed by helices 2 and 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>According to our comparative study , the structural data relative to the FadR protein ( 52 , 53 ) should be regarded as a reference for the whole GntR-family concerning the DNA-binding domain but must be limited to the FadR subfamily concerning the E-b/O domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>Escherichia coli FadR and Bacillus subtitlis GntR have been placed together in a helix-turn-helix DNA-binding domain subfamily ( Prosites PS00043 [ 17 ] and ProDom 13918 [ 124 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The N-terminal region contains a conserved helix-turn-helix DNA-binding domain shared by several bacterial transcription factors , including protein A ( a hypothetical transcription factor ) , P30 ( a 30-kDa hypothetical transcription factor ) , FadR ( a regulator for fatty acid metabolism ) , and PhnF ( a hypothetical transcription factor ) of E. coli ; HutC ( histidine utilization repressor ) of K aerogenes and P. putida ; and GntR ( a repressor protein ) of B. subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8407843" target="_blank">8407843</a>]<br></li>
<li>1 The abbreviations used are : HTH , helix-turn-helix ; E-b/o domain , effector-binding and oligomerization domain ; D-b domain , DNA-binding domain ; PLP , pyridoxal 5 - phosphate ; FadR , fatty acid-responsive regulator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>Identi ® cation and characterization of the DNA-binding domain of FadR Classical and molecular genetic techniques have been used to characterize FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The conserved putative FadR DNA-binding motif , , is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>The C terminus of regulators of the GntR/FadR family contains an effector-binding and/or oligomerisation domain that influences the DNA-binding properties of the regulator ( 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The DNA-binding domain of FadR was ® rst predicted from analysis of the open reading frame derived from the DNA sequence [ 110 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Multiple sequence alignment of PhnF against other members of the HutC family and FadR DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16672238" target="_blank">16672238</a>]<br></li>
<li>Correlation analysis shows that , despite significant differences in the structure of TFs from different subfamilies , main predicted contacts ( Arg-G , Asn-A , Asp-C etc ) are quite similar and conform well to the DNA-protein contacts known for FadR from E. coli and AraR from B. subtilis , as well as to the interaction trends described in literature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>We identify pairs of positions with high correlation between amino acids and nucleotides for FADR , HUTC , and YTRA subfamilies and show that the most predicted DNA-protein interactions are quite similar in all subfamilies and conform well to the experimentally identified contacts formed by FadR from E. coli and AraR from B. subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>Our data shows that predicted protein-DNA interactions for all three analyzed subfamilies of the GNTR family correspond well to known nucleotide-amino acid contacts of FadR and AraR [ 40 , 41 , 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>Amino acid residues are numbered from the N terminus of FadR ( A ) or NanR ( B ) and are shown below the known or predicted regions of - helical , - helical , or random helical structures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Amino acid residues D74 and R75 have the same relative positions in GntR as do residues H65 and G66 in FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Amino acid residues in positions 40 and 59 , which are known to be important for the FadR and AraR interactions with DNA ( Table 1 ) , are also correlated with nucleotide in positions 6 / 14 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>For example , all amino acid residues that , when altered , conferred a transdominant negative phenotype over wild-type FadR are identical between the two proteins [ 118 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Haemophilus inﬂu-enzae encodes a FadR-like protein that is 47 % identical to E. coli FadR , and amino acid residues known to play important roles in the function of the E. coli protein ( Raman and DiRusso , 1995 ; Raman et al. , 1997 ) are conserved in the H. inﬂuenzae protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>It is noteworthy , however , that three amino acid residues when changed result in a supperrepressor phenotype ( not inducible by long-chain fatty acids ) for E. coli FadR that are identical in H. in ¯ uenzae FadR : Y194 , S219 and W 223 [ 120 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>This protein shares 34 % amino acid identities with E. coli FadR in the amino terminus ( amino acids 13 ± 104 E. coli FadR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Amino acid residues in this region of FadR that ful ® ll the criterion to form this structure include : ( 1 ) Gly at position 42 which is critical to form the turn ; ( 2 ) hydrophobic residues L7 , I41 , V43 and L48 ; ( 3 ) hydrophilic residues E34 , R35 , E36 , E39 , T44 , R45 , T46 , T47 , R49 and E50 which might provide DNA contacts , and ; ( 4 ) the absence of proline which would disrupt domain architecture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In the FadR − DNA complex , Arg35 ( helix α2 ) , Arg45 ( helix α3 ) , Thr46 ( helix α3 ) , and His65 ( β2 − β3 wing ) mediate base-speciﬁc contacts with the bound DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>Asterisks mark the residues in FadR that make base-speciﬁc contacts with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>Residues making base-speciﬁc contacts in FadR and the structurally overlapping residues in McbR are shown as sticks and labeled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>This glycine residue is likely to have a function similar to that of FadR G42 ( Fig. 5A ) in orienting the 2 and 3 helices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>This search revealed three protein s ( FadR , KorA and PhnF ) which together with those used to construct the matrix , form a distinct group possessing high similarity scores .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2060763" target="_blank">2060763</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">FadR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>FCD Family Members and Their Structural Similarity to McbRa FCD only full-length name PDB Z rmsd ( Å ) Z rmsd ( Å ) Id ( % ) subfamily metal binding FCD ligandsb McbR 4P9F VanR no UNK PS5454 3C7J 18.2 1.9 18.3 1.9 23 VanR yes Ni CGL2915 2DI3 15.5 2.6 15.5 4.0 20 FadR yes Zn RO03477 2HS5 15.4 2.6 19.7 3.1 19 VanR no Act 3IHU 15.4 2.5 20.3 2.6 16 VanR no TM0439 3FMS 5 14.8 2.4 4 16.6 3.1 22 VanR yes Act , Ni FadR 1H9G 6 12.8 2.6 6 12.6 6.2 14 FadR no CoA-Myr a R , Z-score rank .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>The most similar is the Haemophilus in ¯ uenza FadR which has 47 % identical amino acids and 65 % with conservative substitutions [ 500 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>This protein shares 34 % amino acid identities with E. coli FadR in the amino-terminus ( amino acids 13 ± 104 E. coli FadR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Haemophilus inﬂu-enzae encodes a FadR-like protein that is 47 % identical to E. coli FadR , and amino acid residues known to play important roles in the function of the E. coli protein ( Raman and DiRusso , 1995 ; Raman et al. , 1997 ) are conserved in the H. inﬂuenzae protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Given the high similarity between the members of the FadR subfamily and the characteristics of their recognition sequences , a model for protein binding has been proposed for this group ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18263722" target="_blank">18263722</a>]<br></li>
<li>Subdivision of the Helix-Turn-Helix GntR Family of Bacterial Regulators in the FadR , HutC , MocR , and YtrA Subfamilies * Received for publication , November 15 , 2001 , and in revised form , December 20 , 2001 Published , JBC Papers in Press , December 27 , 2001 , DOI 10.1074 / jbc.M110968200 Sébastien Rigali ‡ , Adeline Derouaux , Fabrizio Giannotta , and Jean Dusart From the Centre d'Inge ́nierie des Protéines , Université de Liège , Institut de Chimie B6 , Sart-Tilman , B-4000 , Liège , Belgium Haydon and Guest ( Haydon , D. J , and Guest , J. R. ( 1991 ) FEMS Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>Our data shows that predicted protein-DNA interactions for all three analyzed subfamilies of the GNTR family correspond well to known nucleotide-amino acid contacts of FadR and AraR [ 40 , 41 , 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>YncC belongs to the GntR subfamily of FadR ( pfam007729 ) , which binds as a dimer to its operator site via its winged helix domains ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>S2 support the overall importance of the Nan box for NanR binding and repression of the sialoregulon using a repressor in the FadR family of helix-turn-helix ( HTH ) regulators with , however , atypical responses to inducer , its oligomerization state , and binding mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_10">
Rigali , S. , Derouaux , A. , Giannotta , F. , and Dusart , J. ( 2002 ) Subdivision of the helix-turn-helix GntR family of bacterial regulators in the FadR , HutC , MocR , and YtrA subfamilies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></label>
<input id="evo_similarity_10", type="checkbox">
<div class="EVO_similar">
<li>[ 8 ] Rigali , S. , Derouaux , A. , Giannotta , F. and Dusart , J. ( 2002 ) Subdivision of the helix-turn-helix GntR family of bacterial regulators in the FadR , HutC , MocR , and YtrA subfamilies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12757941" target="_blank">12757941</a>]<br></li>
<li>Rigali , S. , Derouaux , A. , Giannotta , F. , and Dusart , J. ( 2002 ) Subdivision of the helix -- turn -- helix GntR family of bacterial regulators in the FadR , HutC , MocR , and YtrA subfamilies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890032" target="_blank">12890032</a>]<br></li>
<li>of the helix-turn-helix GntR family of bacterial regulators in the FadR , HutC , MocR , and YtrA subfamilies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Subdivision of the helix-turn-helix GntR family of bacterial regulators in the FadR , HutC , MocR , and YtrA subfamilies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16672238" target="_blank">16672238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18263722" target="_blank">18263722</a>]<br></li></div>
<li>The hexuronate metabolism in E. coli is regulated by two related transcription factors ( TFs ) from the FadR subfamily of the GntR family , UxuR and ExuR , whose amino acid sequences are 46 % identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>As FadR is the prototype of a major group in the GntR family , this structure threw light on several features speci cents c to this group , as well as some generic features of GntR transcription factors [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12757941" target="_blank">12757941</a>]<br></li>
<li>FadR , which belongs to the GntR family of transcription factors , acts as an activator of fabA and fabB where it acts as a repressor of the β-oxidation ( fad ) regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>FadR is predicted to be a helix -- turn -- helix DNA binding protein of the GntR family ( Haydon and Guest , 1991 ) , and thus probably binds DNA in a conventional manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein with a stretch of 25 amino acids ( residues 39 -- 64 ) that shares signiﬁcant similarity with the helix-turn-helix motif for DNA recognition and binding of transcriptional regulators from the GntR family such as FadR ( DiRusso et al. , 1992 ) and GntR ( Haydon and Guest , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The PaaX protein ( 316-amino acid length ) contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif predicted to be important for DNA rec-ognition and binding in other transcriptional repressors such as GntR ( 71 ) and FadR ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein that shows 41.4 % amino acid sequence identity with its ortholog PhaN ( recently renamed as PaaN ( 4 ) ) from the PA degradation pathway of P. putida U ( 1 ) and contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif for DNA recognition and binding of transcriptional repressors from the GntR family such as FadR ( 5 ) and GntR ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>It is not clear whether this protein is a FadR functional homologue with regards to DNA binding speci ® city and regulation by LCACoA or if it is merely a structural homologue as many other GntR/FadR H ± T ± H family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In addition , the PhnF protein may be a member of a new family of regulatory proteins together with the FadR , GntR , HutC , KorA , GenA , and P30 proteins ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8388873" target="_blank">8388873</a>]<br></li>
<li>At least 35 proteins have been placed in the GntR-FadR family of transcriptional regulators due to amino acid identities and similarities in this amino-terminal DNA-binding domain ( ProDom Domain 586 [ 124 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR shares amino acid identities in this region with the GntR family subclass of H ± T ± H proteins [ 121 ] ( PROSITES domain PDOC00042 [ 122 , 123 ] and ProDom domain number ID 586/ProDom36 [ 121 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FarR is a member of the FadR/GntR family of H ± T ± H proteins and shares amino acid identities and similarities with FadR and other members of this protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Multiple sequence alignment of PhnF against other members of the HutC family and FadR DNA binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16672238" target="_blank">16672238</a>]<br></li>
<li>The data suggest that amino acids in the homologous region of the GntR/FadR family of H ± T ± H proteins are required for DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The C-terminus of regulators of the GntR/FadR family contains an effector-binding and/or oligomerisation domain that influences the DNA-binding properties of the regulator ( 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>However , other major groups within the GntR family di ¡ er from FadR other than in their C-terminal domains [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12757941" target="_blank">12757941</a>]<br></li>
<li>FadR has a carboxyl-terminal domain that is homologous to the regulatory domains of the TetR family members ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>LldR belongs to the ﬁrst subfamily , called FadR , which groups 40 % of the GntR regulators ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18263722" target="_blank">18263722</a>]<br></li>
<li>GabC is a member of the FadR subfamily of the GntR family of bacterial regulators , which are generally repressors ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>]<br></li>
<li>GabC is in the FadR subfamily of the GntR family of transcriptional regulators ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>]<br></li>
<li>DISCUSSION The similarities in sequence , molecular size , and predicted secondary structure , combined with the known regulatory functions of the GntR , FadR , HutC and KorA proteins suggest that the eight proteins represent a family of transcriptional regulators , the GntR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2060763" target="_blank">2060763</a>]<br></li>
<li>Detailed bioinformatics studies have led to the deﬁnition of at least 7 families ( AraR , DevA , FCD , HutC , MocR , PlmA , and YtrA ) , which are classiﬁed by the effector binding domain topology and secondary structure ( Figure 1A ) .7 The majority of GntR regulators belong to the FadR C-terminal domain family ( FCD , Pfam PF07729 ; Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>The structure reveals McbR to be a member of the FadR C-terminal domain ( FCD ) family of the GntR superfamily of transcriptional regulators ( this family was named after the ﬁrst identiﬁed member , GntR , a transcriptional repressor of the gluconate operon of Bacillus subtilis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>The secondary structure elements and topology of the C-terminal domain places McbR in the FadR C-terminal domain ( FCD ) family of GntR transcriptional regulators ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by FadR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>TABLE 1 Plasmidsd Lab codea Name Descriptionb Ampr Kanr , TAP tag cassette Promoter endpointsc Reference 45 44 44 47 49 40 This work This work 41 This work 39 This work This work This work 39 This work This work 39 39 This work This work 39 This work 69 This work This work This work This work pEB0793 pEB0269 pEB0267 pEB0266 pMSB1 pEB0227 pEB1210 pEB1502 pEB1188 pEB1209 pEB0898 pEB1301 pEB1299 pEB1183 pJL72 pKD4 pKD46 pCP20 Ampr Kanr ts Ampr , lambda Red genes ts Cmr Ampr , FLP recombinase gene Cloning vector for lambda recombination Ampr , ColE1 replication origin , PBAD promoter PCR ebm598/599 EcoRI/XhoI in pBAD24 ( EcoRI/SalI ) PCR ebm598/968 on EB734 EcoRI/XhoI in pBAD24 ( EcoRI/SalI ) Ampr , T7 promoter , N-terminal 6His-Tev PCR ebm598/599 EcoRI/XhoI in pEB1188 Kanr p15A ori , transcriptional fusion gfp PCR ebm569/689 XhoI/BamHI in pUA66 PCR ebm690/570 XhoI/BamHI in pUA66 PCR ebm567/568 XhoI/BamHI in pUA66 pBAD24 pBAD-FadR pBAD-FadR-SPA pET-6His-Tev pET-6His-Tev-FadR pUA66 pUA-yceDp1 pUA-yceDp2 pUA-rpmF pUA-plsX pUA-fabH pUA-fabHmut pUA-acpP pUA-fabF pUA-fabA pUA-fabB pUA-fadE pUA-fadR 86 / 26 129 / 4 PCR ebm553/554 XhoI/BamHI in pUA66 PCR mutagenesis ebm625/626 on pEB1179 pEB1179 pEB1298 230 / 52 250 / 104 PCR ebm632/633 XhoI/BamHI in pUA66 PCR ebm638/635 XhoI/BamHI in pUA66 pEB1235 pEB1386 265 / 100 188 / 67 176 / 141 * 256 / 75 PCR ebm630/631 XhoI/BamHI in pUA66 Transcription vector pRLG770 containing the fabH promoter pRLG770 containing the fabA promoter pRLG770 containing the fabB promoter pRLG770 containing the fadR promoter pEB1234 pRLG770 pRLG9890 pRLG9885 pRLG9886 pRLG10026 100 / 50 113 / 50 100 / 50 100 / 50 a Lab codes correspond to our stock numbering .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>, probably by increasing the reduction rate of tellurite to metallic tellurium within the bacterium Hydrolyzes only long-chain acyl thioesters ( C12 -- C18 ) ; has speciﬁcity similar to that of chymotrypsin Involved in thiamine-pyrophosphate-biosynthesis ; catalyzes the phosphorylation of 4-amino-5-hydroxymethyl pyrimidine monophosphate to HMP-pyrophosphate ( HMP-PP ) Involved in thiamine-pyrophosphate-biosynthesis ; condenses 4-methyl-5 - ( beta-hydroxyethyl ) - thiazole monophosphate and HMP-PP to form thiamine monophosphate ( TMP ) Involved in thiamine-biosynthesis ; required for the synthesis of the thiazole moiety of thiamine P69425 TatB 179 P07913 Tdh 5.94 / 37,239.04 -- / 38,800 61 6.84 / 22,530.73 Tellurite resistance protein P25397 TehB 179 P0ADA1 Acyl-CoA thioesterase I TesA 5.97 / 19,936 ( DIGE 4.5 -- 6.5 ) 5.91 / 20,470.33 321 ThiD 179 Phosphomethylpyrimidine kinase 5.73 / 28,633.61 P76422 ThiE Thiamine-phosphate pyrophosphorylase P30137 179 , 321 5.51 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.54 / 23,015.28 ThiG 79 5.36 / 26,896.1 Thiazole biosynthesis protein P30139 1 179 179 Protein Bifunctional Q46948 P00561 5.24 / 20,777.09 5.47 / 89,120.24 ThiJ ThrA Involved in amino acid oserine aspartokinase/hom dehydrogenase Homoserine kinase biosynthesis : Lys , Met , or Thr ThrB Involved in threonine 5.38 / 32,000 ( DIGE 4.5 -- 6.5 ) 5.33 / 37,266 ( 5 -- 6 ) 5.45 / 33,623.65 P00547 179 , 287 , 321 biosynthesis from aspirate : step 4 ; decreases after benzoic acid treatment Continued on facing pag TABLE 2 -- Continued ThrC P00934 Threonine synthase 5.24 / 47,113.84 5.30 / 48,579 See ThrB : step 5 ( ﬁnal ) 5.27 / 46,667 ( DIGE 4.5 -- 6.5 ) 6.00 / 28,285 ThyA P0A884 Thymidylate synthase 5.62 / 30,479.69 Provides the sole de novo source of dTMP for DNA-biosynthesis ; binds to its mRNA , thus repressing its own translation Involved in protein export and acts as a chaperone by maintaining the newly synthesized protein in an open conformation ; increases by high pH during anaerobic growth and overproduction of recombinant protein but decreases during phosphate limitation Catalyzes the NADPH-dependent reductions of 2,5-diketo-D-gluconate to 5-keto-D-gluconate , 2-keto-D-gluconate to D-gluconate , and 2-keto-L-gulonate to L-idonate Suppresses the inhibitory activity of the carbon storage regulator csrA Involved in tryptophan-catabolism ; increases by high pH during aerobic or anaerobic growth but decreases after benzoic acid treatment Involved in the TonB-independent uptake of group A colicins ( colicins A , E1 , E2 , E3 , and K ) ; necessary for the colicins to reach their respective targets after initial binding to the bacteria Required for proper expression of outer membrane protein genes such as ompF and nmpC and those encoding protein 2 , hemolysin , colicin V , and colicin E1 ; induced by low pH during anaerobic growth Plays an important role in several metabolic pathways ; decreases after benzoic acid treatment Has antioxidant activity ; could remove peroxides or H2O2 within the catalase - and peroxidase-deﬁcient periplasmic space ; induced by low or high pH ; decreases after benzoic acid treatment Provides the cells with the ability to utilize trehalose at high osmolarity by splitting it into glucose molecules that can subsequently be taken up by the phosphotransferasemediated uptake system ; induced by growth at high osmolarity or by entry into stationary phase ; regulated by cAMP-CRP Catalyzes the formation of 5-methyl-uridine at position 54 ( M-5-U54 ) in all tRNA ; induced during growth rate-dependent regulation of transcription 294 Tig P0A850 Trigger factor 4.83 / 48,192.67 4.83 / 51,045 138 , 179 , 228 , 287 , 293 , 294 , 321 , 323 4.94 / 52,681 ( DIGE 4.5 -- 6.5 ) 4.96 / 49,707 ( 4 -- 5 ) 4.58 / 49,476 ( 4 -- 5 ) 4.97 / 47,893 ( 4 -- 5 ) 4.87 / 45,401 ( 4 -- 5 ) 4.93 / 48,928 ( 4.5 -- 5.5 ) 4.97 / 48,171 ( 4.5 -- 5.5 ) 4.95 / 47,057 ( 4.5 -- 5.5 ) 5.5 / 35,395.5 TkrA 79 P37666 2-Ketogluconate reductase 1 4.93 / 51,364.09 P0AGG8 Protein TldD 179 TnaA 5.81 / 48,395 ( DIGE 4.5 -- 6.5 ) 5.88 / 52,773.46 P0A853 Tryptophanase 27 , 179 , 274 , 321 , 323 TolB 6.14 / 43,601.64 5.98 / 43,053 5.91 / 44,088 7.05 / 35,331 ( 6 -- 11 ) P0A855 Protein 179 , 228 , 286 , 287 , 294 TolC 5.16 / 50,036 ( DIGE 4.5 -- 6.5 ) 5.23 / 51,468.93 Outer membrane protein P02930 179 , 321 , 323 TpiA 5.57 / 26,972 5.64 / 26,971.81 179 , 228 , 287 , 294 , 321 P0A858 Triosephosphate isomerase 5.51 / 25,962 ( DIGE 4.5 -- 6.5 ) 5.01 / 11,079 ( 4 -- 5 ) Tpx 4.90 / 18,723 4.75 / 17,704.12 P0A862 Thiol peroxidase 179 , 228 , 274 , 287 , 321 4.99 / 19,656 ( DIGE 4.5 -- 6.5 ) 5.02 / 15,046 ( 4 -- 5 ) 5.00 / 17,510 ( 4.5 -- 5.5 ) 5.36 / 60,463.80 P13482 Periplasmic trehalase TreA 103 , 244 P23003 tRNA ( uracil-5 - ) - methyltransferase 79 TrmA 1 5.71 / 41,966.95 Continued on following pag TABLE 2 -- Continued Accession pI/MW no.b Theoreticalc Experimentald Protein nameb TrpA P0A877 Responsible for the aldol cleavage of indoleglycerol phosphate to indole and glyceraldehyde 3-phosphate in tryptophan biosynthetic pathway Responsible for the synthesis of Ltryptophan from indole and L-serine in tryptophan biosynthetic pathway Involved in tryptophan-biosynthesis Tryptophan synthase alpha chain 5.31 / 28,724.16 5.30 / 28,724 5.38 / 28,867 ( 4.5 -- 5.5 ) 29 , 179 , 228 , 287 , 294 5.36 / 27,856 ( 4.5 -- 5.5 ) 5.18 / 32,691 ( 5 -- 6 ) 5.75 / 41,241 ( DIGE 4.5 -- 6.5 ) 5.03 / 33,559 ( 5 -- 6 ) TrpB Tryptophan synthase beta chain P0A879 5.71 / 42,851.82 179 , 287 , 321 TrpD Anthranilate synthase component II Tryptophanyl-tRNA synthetase Thioredoxin 1 P00904 6.05 / 56,738.75 6.08 / 55,915 179 , 228 , 294 P00954 TrpS 179 6.27 / 37,437.82 TrxA P0AA25 Participates in various redox reactions through the reversible oxidation of its active center dithiol to a disulﬁde and catalyzes dithiol-disulﬁde exchange reactions May be directly degraded by ClpXP or modulated by a protease-dependent mechanism ; increases in the acnA and / or acnB mutants but decreases late during phosphate limitation Efﬁcient electron donor for the essential enzyme ribonucleotide reductase Associates with the EF-Tu -- GDP complex and induces the exchange of GDP to GTP ; changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases with increasing growth rate , after benzoic acid treatment , and at low pH during anaerobic growth ; decreases during phosphate limitation Receptor for the attractant L-serine and related amino acids ; responsible for chemotaxis away from a wide range of repellents , including leucine , indole , and weak acids Constitutes the receptor for colicin K and phage T6 and functions as substratespeciﬁc channel for nucleosides and deoxynucleosides Promotes the GTP-dependent binding of aminoacyl-tRNA to the A site of ribosomes during protein-biosynthesis ; may play an important regulatory role in cell-growth and in the bacterial response-to-nutrient deprivation ; changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate and after benzoic acid treatment 4.67 / 11,675.43 4.67 / 11,576 29 , 179 , 228 , 287 , 294 4.80 / 9,648 ( 4 -- 5 ) 4.96 / 9,467 ( 4.5 -- 5.5 ) TrxB P0A9P4 Thioredoxin reductase 5.30 / 34,491.84 5.30 / 34,419 29 , 179 , 228 , 255 , 278 , 293 , 294 , 308 5/15 ,554.77 P0AGG4 Thioredoxin 2 TrxC 179 Tsf 5.15 / 33,623 5.06 / 33,695 5.22 / 30,291.79 P0A6P1 EF-Ts 29 , 109 , 179 , 228 , 230 , 286 , 287 , 293 , 294 , 321 , 323 5.14 / 34,596 ( DIGE 4.5 -- 6.5 ) 5.20 / 34,415 ( DIGE 4.5 -- 6.5 ) 5.08 / 34,633 ( DIGE 4.5 -- 6.5 ) 5.22 / 32,484 ( 4.5 -- 5.5 ) Methyl-accepting chemotaxis protein I P02942 Tsr 4.88 / 59,442.99 179 Nucleoside-speciﬁc channel-forming protein P0A927 Tsx 4.87 / 31,413.41 179 TufA 5.30 / 43,182.39 P0A6N1 EF-Tu 5.32 / 44,615 5.38 / 41,860 29 , 109 , 179 , 228 , 230 , 287 , 294 , 321 5.07 / 32,442 ( DIGE 4.5 -- 6.5 ) 5.69 / 25,098 ( DIGE 4.5 -- 6.5 ) 5.40 / 42,307 ( DIGE 4.5 -- 6.5 ) 5.58 / 45,310 ( 4.5 -- 5.5 ) 5.14 / 30,073 ( 4.5 -- 5.5 ) 5.39 / 51,249 ( 5 -- 6 ) 5.20 / 51,059 ( 5 -- 6 ) 5.32 / 50,990 ( 5 -- 6 ) 5.35 / 50,921 ( 5 -- 6 ) 5.34 / 50,034 ( 5 -- 6 ) 5.31 / 48,144 ( 5 -- 6 ) 5.33 / 47,448 ( 5 -- 6 ) 179 Aromatic amino acid aminotransferase Transcriptional regulatory protein Involved in amino acid biosynthesis of Phe , Tyr , Asp , and Leu Involved in transcriptional regulation of aromatic-amino-acid biosynthesis and transport ; modulates the expression of at least eight unlinked operons ; in most cases , causes negative regulation but has positive effects on the TyrP gene at high phenylalanine concentrations Decreases late during phosphate limitation Involved in menaquinone and ubiquinone-biosynthesis P04693 TyrB 5.32 / 43,537.81 5.54 / 57,656.14 P07604 TyrR 179 P0AGJ9 228 , 236 , 293 , 294 Tyrosyl-tRNA synthetase Ubiquinone / menaquinone-biosynthesis methyltransferase Oxidoreductase Uridine kinase 5.59 / 47,395.78 5.59 / 44,527 TyrS P0A887 179 7.77 / 28,073.21 UbiE 5.13 / 27,849.97 6.39 / 24,353.11 UcpA Udk P37440 P0A8F4 179 Involved in pyrimidine-salvage pathway 179 Continued on facing pag Udp catalyzes the reversible phosphorylytic cleavage of uridine and deoxyuridine to uracil and ribose - or deoxyribose-1-phosphate ; the produced molecules are then utilized as carbon and energy sources or in the rescue of pyrimidine bases for nucleotide-synthesis ; increases during high-cell-density cultivation SN-Glycerol-3-phosphate and glycerophosphoryl diester-binding protein interacts with the binding-protein-dependent transport system UgpACE ; increases in the physiological short-term adaptation to glucose limitation Involved in pyrimidine-salvage pathway ; induced by pyrimidine starvation Uridine phosphorylase 179 , 228 , 287 , 294 , 325 Udp P12758 5.81 / 27,027.89 5.86 / 27,960 5.66 / 31,754 ( 5 -- 6 ) 5.71 / 30,842 ( 5.5 -- 6.7 ) UgpB 311 P0AG80 Glycerol-3-phosphate-binding periplasmic protein 5.98 / 46,123.95 5.98 / 46,123.95 Upp P0A8F0 Uracil phosphoribosyltransferase 5.32 / 22,533.26 5.29 / 23,815 179 , 228 , 294 4.38 / 36,364.13 5.4 / 58,208.8 USG-1 protein Protein P08390 P07024 Usg Degradation of external UDP-glucose to UMP and glucose-1-phosphate , which can then be used by the cell Required for resistance to DNA-damaging agents ; induced during growth inhibition caused by the exhaustion of any of a variety of nutrients ( carbon , nitrogen , phosphate , sulfate , required amino acid ) or by the presence of a variety of toxic agents ; positively regulated by guanosine 3 ,5 - bisphosphate ( ppGpp ) and by a recA/ftsK-dependent regulatory pathway but negatively regulated by FadR ; also regulated by CspC and CspE ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and phosphonate growth and at high pH during anaerobic growth See UspA UshA UspA 5.12 / 15,935.18 5.14 / 15,108 5.20 / 11,633 ( 4.5 -- 5.5 ) P0AED0 Universal stress protein A 71 , 179 , 228 , 287 , 293 , 294 , 323 286 228 5.30 / 34,419 7.40 / 30,751 5.16 / 35,575.69 5.60 / 16,016.53 6.03 / 15,935.18 P0AAC0 Universal stress protein E P37903 Universal stress protein F P39177 Universal stress protein G UspE UspF UspG Interacts with GroEL ; induced by starvation and heat shock and in response to some toxic agents Seems to enhance the formation and/or stability of noncovalent complexes between the Trp repressor protein and operatorbearing DNA ; alone does not interact with the operatorbearing DNA ; major target species probably the TrpR/TrpO complex ; may function as an accessory element in blocking TrpR-speciﬁc transcriptional processes that might be physiologically disadvantageous in the stationary phase of the bacterial life cycle ; induced by entry into stationary phase Confers a modal distribution of chain length on the O-antigen component of lipopoly-saccharide ; gives rise to a reduced no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>DNA Sequences Surrounding the mcbR ( yncC ) Gene in Escherichia coli ( MG1655 ) other names mnaT , b1448 curA , b1449 gene yncA gene description Methionine N-acyltransferase ; - amino acid NL acyltransferase Curcumin/dihydrocurcumin reductase , NADPH-dependent Predicted iron outer membrane transporter ATP-binding protein , periplasmic , function unknown L-asparagine-transporter Glutathione S-transferase homologue Conserved protein , function unknown yncB b1451 b1452 yncD yncE yncF yncG yncH ansP , b1453 b1454 b1455 ■ ABBREVIATIONS ASU , asymmetric unit ; CD , circular dichroism ; EPS , exopolysaccharide ; EDTA , ethylenediaminetetraacetic acid ; EMSA , electrophoretic mobility shift assay ; FCD , FadR Cterminal domain ; IPTG , isopropyl-β-D-thiogalactopyranoside ; MPD , 2-methyl-2 ,4 - pentanediol ; PDB , Protein Data Bank ; SEC , size-exclusion chromatography ; SeMet , selenomethionine ; TCEP , tris ( 2-carboxyethyl ) phosphine ; TEV , Tobacco Etch Virus ; wHTH , winged helix-turn-helix ; WT , wild-type ■ REFERENCES ( 1 ) Lewis , K. ( 2001 ) Riddle of biofilm resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Aug. 1996 , p. 4704 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Regulated Expression of a Repressor Protein : FadR Activates iclR LIZHEN GUI , ALDEN SUNNARBORG , AND DAVID C. LAPORTE * Department of Biochemistry , University of Minnesota , Minneapolis , Minnesota 55455 Received 23 February 1996/Accepted 21 May 1996 The control of the glyoxylate-bypass operon ( aceBAK ) of Escherichia coli is mediated by two regulatory proteins , IclR and FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Nov. 1996 , p. 6443 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Role of the Escherichia coli FadR Regulator in Stasis Survival and Growth Phase-Dependent Expression of the uspA , fad , and fab Genes ANNE FAREWELL ,1 ALFREDO A. DIEZ ,1 CONCETTA C. DIRUSSO ,2 † AND THOMAS NYSTRÖM1 * Department of Microbiology , Lund University , Lund , and Department of General and Marine Microbiology , Lundberg Laboratory , Göteborg University , Göteborg , Sweden ,1 and Department of Biochemistry , College of Medicine , The University of Tennessee , Memphis , Tennessee 381632 Received 13 May 1996/Accepted 30 August 1996 The increased expression of the uspA gene of Escherichia coli is an essential part of the cell 's response to growth arrest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>A transcriptional regulator such as FadR , having both positive and negative regulatory roles , provides a good test of this experimental approach .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>J Bacteriol 175:1484 -- 1492 Cronan JE Jr , Subrahmanyam S ( 1998 ) FadR , transcriptional coordination of metabolic expediency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>FadR , transcriptional coordination of metabolic expediency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>FadR , transcriptional coordination of metabolic expendiency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>[ PubMed : 9045847 ] Cronan JE Jr , Subrahmanyam S. FadR , transcriptional co-ordination of metabolic expediency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>FadR , transcriptional co-ordination of metabolic expediency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The survival and growth of bacteria under such changing environments largely depend on the adjustments in the expression proﬁles of several global regulatory proteins , such as RpoD , RpoS , SoxRS , Cra , FadR and IclR ( Aronis 2002a , b ; Cortay et al. 1991 ; Jordan et al. 1999 ; Liochev and Fridovich 1992 ; Lu et al. 2003 ; Moat 2002 ; Tang et al. 2002 ; Varghese et al. 2003 ; Perrenoud and Sauer 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>We also have found that the timing and/or magnitude of this induction depends , in part , on the carbon regulator cyclic AMP ( cAMP ) receptor protein ( CRP ) , the oxygen regulator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Transcription depends upon CRP , the anaerobically triggered transcription activator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>It also depends , probably indirectly , upon the glyoxylate shunt repressor IclR , its activator FadR , and many enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>FadR action is straightforward , it acts as a classical LacI-type repressor and only weakly represses fadD and fadL expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Disruption of fadR resulted in a threefold decrease in iclR : : lacZ expression during growth on glucose or acetate ( Table 3 ) , indicating that FadR activates expression of iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Therefore , loss of the protein ( FadR ) that activates expression of the real repressor ( IclR ) mimicked direct repression of the ace operon by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>FadR activates iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR also activates the expression of iclR , the gene which encodes the repressor protein of the glyoxylate-bypass operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_22">
Regulated expression of a repressor protein : FadR activates IclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></label>
<input id="rp_similarity_22", type="checkbox">
<div class="RP_similar">
<li>Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19563847" target="_blank">19563847</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19574213" target="_blank">19574213</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li></div>
<li>Gui , L. , Sunnarborg , A. , Pan , B. , and Laporte , D.C. ( 1996b ) Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_25">
Biochem J 242:661 -- 665 Gui L , Sunnarborg A , LaPorte DC ( 1996 ) Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></label>
<input id="rp_similarity_25", type="checkbox">
<div class="RP_similar">
<li>Gui , L. , Sunnarborg , A. , and LaPorte , D.C. ( 1996 ) Regulated expression of a repressor protein : FadR activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li></div>
<li>No effects on accBC operon expression were observed under the conditions we examined , including accumulation of fatty acidbiosynthetic intermediates by shifting of cultures of fabA ( Ts ) , fabB ( Ts ) , or fabD ( Ts ) mutants to nonpermissive temperatures ; supplementation of cultures of long-chain saturated and unsaturated fatty acids ; or overproduction of BCCP , biotin carboxylase , carboxyltransferase , biotin li-gase , acyl carrier protein , FadR , integration host factor , or the crp or cya gene product ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7678242" target="_blank">7678242</a>]<br></li>
<li>Our data argue that the function of FabR is to compensate for the weak FadR regulation of fabB expression in the presence of exogenous UFA ( the converse argument , that FadR functions to compensate for the weak FabR regulation of fabA expression by UFA , also applies ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>FadR , which belongs to the GntR family of transcription factors , acts as an activator of fabA and fabB where it acts as a repressor of the β-oxidation ( fad ) regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Work from this laboratory and others showed that fabA transcription is positively regulated by FadR , a protein that also functions as the repressor of the - oxidation regulon ( 5 -- 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li>
<li>A reduced amount of FadR protein might then reduce the expression of other genes activated by FadR , such as fabA , fabB , and iclR , in addition to fabH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>( iii ) FadR activates the transcription of at least four genes , fabA , fabB , fabI and iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR positively regulates transcription of fabA , fabB , and iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>Gene expression data indicate a slight decrease in the expression of rpoD and some of the rpoD-dependent genes , such as pgi and serA , at the later phases of pykF : puruvate kinase F , aspA : aspartate ammonia-lyase , serA : D-3-phosphoglycerate dehydrogenase , rpoS : sigma S factor , tktB : transketolase B , talA : transaldolase A , acnA : aconitase A , sucA : 2-oxoglutarate decarboxylase , fumC : fumarase C , acs : acetyl-CoA synthetase , sodC : superoxide dismutase C , soxR : SoxR transcriptional dual regulator , soxS : SoxS transcriptional dual regulator , zwf : glucose-6P dehydrogenase , sodA : superoxide dismutase A , fadR : FadR transcriptional dual regulator , iclR : IclR transcriptional repressor , aceA : isocitrate lyase , aceB : malate synthase , cra : CRA transcriptional dual regulator , pck : phosphoenolpyruvate kinase , ppsA : phosphoenolpyruvate synthase , icd : isocitrate dehydrogenase , arcA : ArcA transcriptional dual regulator , gltA : citrate synthase , acnB : aconitase B , fumA : fumarase A growth ( Fig. 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li>
<li>The expression of the aceBAK operon , which codes for the key enzymes of the glyoxylate-cycle , isocitrate lyase and malate synthase , and for isocitrate dehydrogenase kinase/phosphatase , is activated by FruR and IHF and is repressed by IclR , which in turn is activated by FadR ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>In fact , the FadR system appears to form a global control network as described by Gottesman for the cAMP-CRP system , the SOS system and the proteins involved in the heat shock response ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2843809" target="_blank">2843809</a>]<br></li>
<li>As detailed above , the emerging information for E. coli requires that any molecular model seeking to explain the coordinate regulation-of-lipid-metabolism during growth , stress and stasis should include : ( 1 ) control by speci ® c transcription factors including FadR ; ( 2 ) growth rate - and growth phase-dependent regulation of speci ® c genes and operons which are still largely unde ® ned ; and ( 3 ) regulation by small molecule e $ ectors including ppGpp , cAMP , long-chain acyl-CoA , long-chain acyl-ACP and ( possibly ) acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Regulation of expression of the universal shock protein A , fad and fab genes by FadR is growth phase dependent The ® rst indication FadR was an important player in the adaptive response to growth arrest came from the discovery that the universal stress protein A ( UspA ) was negatively regulated by FadR in a growth phase-dependent manner [ 113 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_39">
Farewell A , Diez AA , DiRusso CC & Nystrom T ( 1996 ) Role of the Escherichia coli FadR regulator in stasis survival and growth phase-dependent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></label>
<input id="rp_similarity_39", type="checkbox">
<div class="RP_similar">
<li>Farewell , A. , Diez , A.A. , DiRusso , C.C. & Nyström , T. ( 1996 ) Role of the Escherichia coli FadR regulator in stasis survival and growth phase-dependent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12071968" target="_blank">12071968</a>]<br></li>
<li>Farewell , A. , Diez , A.A. , DiRusso , C.C. , and Nystrom , T. ( 1996 ) Role of the Escherichia coli FadR regulator in stasis survival and growth phase-dependent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_42">
Role of the Esch-erichia coli FadR regulator in stasis survival and growth phase-dependent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></label>
<input id="rp_similarity_42", type="checkbox">
<div class="RP_similar">
<li>Role of the Escherichia coli FadR regulator in stasis survival and growth-phase dependent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>Role of the Escherichia coli FadR regulator in stasis survival and growth phase-depen-dent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Role of Escherichia coli FadR regulator in stasis survival and growth phase-depen-dent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633102" target="_blank">10633102</a>]<br></li></div>
<li>It should be noted that Farewell et al. ( 1996 ) reported that the transcription of uspA , a gene involved in response to a variety of environmental stresses , is repressed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>It was also reported recently that FadR inhibits a new target involved in phospholipid-synthesis in Vibrio cholerae ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The FadL and FadD proteins involved in the uptake and activation of exogenous FA also are repressed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Furthermore , it has been reported that the anaerobic b-oxidation pathway is mediated by fadJ , fadI and fadK genes , whose transcriptional regulation is independent of FadR control ( Campbell et al. , 2003 ; Morgan-Kiss & Cronan , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>rlapping Repressor Binding Sites Result in Additive R ve O egulation of Escherichia coli FadH by FadR and ArcA Youjun Feng1 and John E. Cronan1 ,2 * Departments of Microbiology1 and Biochemistry ,2 University of Illinois , Urbana , Illinois 61801 Received 5 May 2010/Accepted 25 June 2010 Escherichia coli fadH encodes a 2,4-dienoyl reductase that plays an auxiliary role in - oxidation of certain unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Crosstalk of Escherichia coli FadR with Global Regulators in Expression of Fatty Acid Transport Genes Youjun Feng , John E. Cronan * 1 1,2 1 Department of Microbiology , University of Illinois , Urbana , Illinois , United States of America , 2 Department of Biochemistry , University of Illinois , Urbana , Illinois , United States of America Abstract Escherichia coli FadR plays two regulatory roles in fatty-acid-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Citation : Feng Y , Cronan JE ( 2012 ) Crosstalk of Escherichia coli FadR with Global Regulators in Expression of Fatty Acid Transport Genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Expression of FadR under our conditions increased the transcription of genes for phage-shock proteins PspA and PspG by 6.2 - and 8.5-fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Haemophilus inﬂu-enzae encodes a FadR-like protein that is 47 % identical to E. coli FadR , and amino acid residues known to play important roles in the function of the E. coli protein ( Raman and DiRusso , 1995 ; Raman et al. , 1997 ) are conserved in the H. inﬂuenzae protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>FadR is the more thoroughly studied and functions as a global regulator of both UFA synthesis and the β-oxidative utilization of fatty acids as carbon sources ( Cronan and Subrahmanyam , 1998 ; Henry and Cronan , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>These predictions remain to b tested to determine ( 1 ) if acyl-CoA and acyl-ACP levels are modulated in response to changing environmental conditions , during progression though the growth cycle and in response to changes in growth rate ; ( 2 ) whether or not acetyl-phosphate levels play a role in the regulation of these lipid-related genes and pathways ; and ( 3 ) if trans-acting factors in addition to FadR play a role in modulating fatty acid and phospholipid-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The N-terminal region contains a conserved helix-turn-helix DNA binding domain shared by several bacterial transcription factors , including protein A ( a hypothetical transcription factor ) , P30 ( a 30-kDa hypothetical transcription factor ) , FadR ( a regulator for fatty-acid-metabolism ) , and PhnF ( a hypothetical transcription factor ) of E. coli ; HutC ( histidine utilization repressor ) of K aerogenes and P. putida ; and GntR ( a repressor protein ) of B. subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8407843" target="_blank">8407843</a>]<br></li>
<li>The family also includes the HutC regulators of histidine utilisation in Pseudomonas putida and Klebsiella aerogenes , the KorA regulator of conjugal plasmid transfer in Streptomyces , and five regulatory proteins of Escherichia coli : PdhR ( formerly GenA ) , the repressor of the pyruvate dehydrogenase operon ( pdhR-aceEF-lpd ) ; FadR , the repressor of fatty-acid-degradation ; LctR , a potential regulator of L-lactate utilisation encoded by the 1ctPRLI operon ; PhnF , a potential regulator of alkylphosphonate uptake and metabo-lism ; and FarR , an uncharacterised regulator that was formerly designated P30 [ 1,3,4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7805834" target="_blank">7805834</a>]<br></li>
<li>The family contains the repressor of the Bacillus subtills gluconate operon ( GntR ) , the regulators for histidine utilization in Pseudomonas putida ( HutCep ) and Klebsiella aerogenes ( HutCKa ) , the repressor ( FadR ) of fatty-acid-degradation in Escherichia toll , a regulator involved in the conjugal transfer of the broad host range plasmid pIJ101 ( KorA ) , and three proteins of unidentified function in E. coli ( GenA , P30 and PhnF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2060763" target="_blank">2060763</a>]<br></li>
<li>RESULTS ArcA downregulates the transcription of fatty-acid-transport genes under anaerobic conditions The expression of fadL , which encodes one of the outer-membrane-bound proteins involved in the uptake of fatty acids , is regulated by at least three transcription factors : Crp , OmpR and FadR ( Berg et al. , 2004 ; Busch & Saier , 2002 ; Clark & Cronan , 1996 ; DiRusso et al. , 1992 ; Cronan & Subrahmanyam , 1998 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>None of the transcription factors involved in fatty acid regulation ( Crp , OmpR and FadR ) is oxygen sensing , though , recently , the ArcBA two-component signal-transduction-system , which senses oxygen indirectly through a change in redox potential , has been implicated in the negative regulation of aerobic fatty-acid-metabolism ( Georgellis et al. , 2001 ; Clark & Cronan , 1996 ; Malpica et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Instead , FadR appears to prevent an increase in the expression of iclR during growth on fatty acids by activating the transcription of this gene on all other carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>This transcription pattern shows that FadR negatively regulates yafH and that induction of yafH expression upon supplementation of cultures with fatty acids is mediated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>Gui et al. , ( 1996b ) proposed that FadR , which is known as a repressor for the transcription of the fatty acid deg-radation operon , regulates the expression of iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>GS , glyoxylate shunt ; TCA , tricarboxylic-acid-cycle ; PoxB , pyruvate oxidase ; ICL , aceA gene product isocitrate lyase ; IclR , repressor of the glyoxylate shunt operon aceBAK ; FadR , regulator of fatty-acid-metabolism that also activates iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Furthermore , the regulator of fatty-acid-metabolism ( FadR ) , encoded by fadR , activates the expression of the ICL repressor , which inhibits the expression of the glyoxylate operon ( Phue et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19452501" target="_blank">19452501</a>]<br></li>
<li>These results indicate that speciﬁc changes in membrane composition , resulting from overproduction of spe-ciﬁc membrane proteins or of the FadR regulator involved in fatty-acid-metabolism ( Cronan and Subrahmanyam , 1998 ) , lead to signals that stimulate EnvZ activity , in turn inducing expression of OmrA and OmrB sRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Our results support the conclusion that FadR is a global regulator of lipid metab-olism , activating multiple operons responsible for fatty-acid-synthesis in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Reassessment of the genetic regulation-of-fatty-acid-synthesis in Escherichia coli : global positive control by the functional dual regulator FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Mierobiol Rev 55:371 394 DiRusso CC , Heimert TL , Metzger AK ( 1992 ) Characterization of FadR , a global transcriptional regulator of fatty acid metabol-ism in Eseherichia eoli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_71">
Characterization of FadR , a global transcriptional regulator of fatty-acid-metabolism : interaction with the fadB promoter is prevented by long chain fatty acyl coenzyme As .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></label>
<input id="rp_similarity_71", type="checkbox">
<div class="RP_similar">
<li>DiRusso , C.C , Heimert , T.L. , and Metzger , A.K. ( 1992 -RCB- Characterization of FadR , a global transcriptional regulator of fatty-acid-metabolism in Escherichia coii : interaction with the fadB promoter is prevented by long chain fatty acyl coenzyme As .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li></div>
<li>Summary FadR is an Escherichia coli transcriptional regulator that optimizes fatty-acid-metabolism in response to exogenously added fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_74">
DiRusso , C.C. , Heimert , T.L. , and Metzger , A.K. ( 1992 ) Characterization of FadR , a global transcriptional regulator of fatty-acid-metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></label>
<input id="rp_similarity_74", type="checkbox">
<div class="RP_similar">
<li>DiRusso , C.C. , Heimert , T.L. and Metzger , A.K. ( 1992 ) Characterization of FadR , a global transcriptional regulator of fatty-acid-metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Characterization of FadR , a global transcriptional regulator of fatty-acid-metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Characterization of FadR , a global transcriptional regulator of fatty-acid-metabolism in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li></div>
<li>Conclusions Transcriptional regulation of fatty-acid-transport in E. coli involves three distinct regulatory systems , the specialized FadR system and the two global systems , ArcA and CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Control of fatly acid metabolism in E. coli by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Thus , through FadR , E. coli regulates its fatty-acid-metabolism , catabolism , and transport according to the availability of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>E. coli FadR coordinates the catabolic and anabolic fatty acid pathways ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Therefore , FadR plays an important role in maintaining the balance between saturated and unsaturated fatty acids in the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Why does FadR regulate only the synthesis of unsaturated fatty acids in response to exogenous supplementation ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_84">
As FadR responds to CoA thioesters of both unsaturated and saturated fatty acids , E. coli could shut down unsaturated-fatty-acid-synthesis in response to saturated fatty acids , a potential disaster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></label>
<input id="rp_similarity_84", type="checkbox">
<div class="RP_similar">
<li>Because FadR responds to CoA thioesters of both unsaturated and saturated fatty acids , E. coli could shut down UFA synthesis in response to saturated fatty acid ( SFA ) availability , a potential disaster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19679654" target="_blank">19679654</a>]<br></li></div>
<li>The fatty acid acid-mediated repression of OLE1 requires the activity of fatty acyl-CoA synthetase ( FAA1 or FAA4 [ 158 ± 160 ] ) suggesting that LCACoA mediates regulatio perhaps either by direct modi ® cation of a transcription factor as is seen for E. coli FadR or by modifying an additional protein that in turn regulates gene-expression [ 153 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Transport and activation are directly coupled to transcriptional-control of the structural gene of the fatty-acid-metabolic pathways through the fatty acyl-CoA-responsive transcription factor FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Linkage of fatty acid-mediated changes in gene regulation to transport and activation In the previous section , the regulation of FadR by long-chain acyl CoA was described in detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Our study shows that both ppGpp and FadR have direct roles in the control of fatty acid promoters , linking expression in response to both translation activity and fatty acid availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Our data demonstrate that the fabH promoter , and fatty acid promoter activity more generally , is subject to the interplay of multiple regulators , including FadR and ppGpp/DksA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>To quantify the relative FadR levels in the presence of various arabinose concentrations , we replaced fadR with a red ﬂuorescent protein gene ( rfp ) and measured the relative ﬂuorescence under the same condition as used for fatty acid production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Three separate genomic differential transcriptional analyses were used to measure the effects of various fatty acid and FadR conditions on global gene-expression patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>In this study , we overexpressed FadR in an engineered fatty acid-producing E. coli strain and examined its effects on both fatty acid production and gene-expression using both cDNA micro-arrays and targeted proteomics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>n E scherichia coli FadR Positively Regulates Transcriptio of the fabB Fatty Acid Biosynthetic Gene JOHN W. CAMPBELL1 AND JOHN E. CRONAN , JR. 1,2 * Departments of Microbiology1and Biochemistry ,2 University of Illinois at Urbana-Champaign , Urbana , Illinois 61801 Received 2 May 2001/Accepted 24 July 2001 In Escherichia coli expression of the genes of fatty-acid-degradation ( fad ) is negatively regulated at the transcriptional level by FadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Transcription of the Escherichia coli Fatty Acid Synthesis Operon fabHDG Is Directly Activated by FadR and Inhibited by ppGpp Laetitia My , a Brian Rekoske , b Justin J. Lemke , b Julie P. Viala , a Richard L. Gourse , b Emmanuelle Bouvereta LISM , CNRS , Aix-Marseille University , Marseille , Francea ; Department of Bacteriology , University of Wisconsin -- Madison , Madison , Wisconsin , USAb In Escherichia coli , FadR and FabR are transcriptional regulators that control the expression of fatty-acid-degradation and unsaturated-fatty-acid-synthesis genes , depending on the availability of fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Reassessment of the Genetic Regulation of Fatty Acid Synthesis in Escherichia coli : Global Positive Control by the Functional Dual Regulator FadR L. My , N. Ghandour Achkar , J. P. Viala , E. Bouveret LISM CNRS , Aix-Marseille University , Marseille , France ABSTRACT In Escherichia coli , the FadR transcriptional regulator represses the expression of fatty-acid-degradation ( fad ) genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Nud Acids Res 12 : 387-395 , DiRusso , C.C , Metzger , A.K , , and Heimert , T-L ( 1993 ) Regulation of transcription of genes required for fatty-acid-transport and unsafurated fatty-acid-biosynthesis in Escherichia co / ( by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_98">
DiRusso , C.C. , Metzger , A.K. , and Heimert , T.L. ( 1993 ) Regulation of transcription of genes required for fatty-acid-transport and unsaturated-fatty-acid-biosynthesis in Escherichia coli by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></label>
<input id="rp_similarity_98", type="checkbox">
<div class="RP_similar">
<li>Regulation of transcription of genes required for fatty-acid-transport and unsaturated-fatty-acid-biosynthesis in Escherichia coli by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li></div>
<li>The Escherichia coli FadR is a transcription factor that regulates several processes in fatty-acid-biosynthesis , degradation , and membrane transport .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Summary Fatty acid biosynthesis and fatty-acid-degradation in Escherichia coli are co-ordinateiy reguiated at the level of transcription by the product of the fadR gene , FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_102">
Escherichia coli FadR positively regulates transcription of the fabB fatty-acid-biosynthetic gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></label>
<input id="rp_similarity_102", type="checkbox">
<div class="RP_similar">
<li>Campbell JW , Cronan JE ( 2001 ) Escherichia coli FadR positively regulates transcription of the fabB fatty-acid-biosynthetic gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20080587" target="_blank">20080587</a>]<br></li>
<li>J Bacteriol 178:5121 -- 5129 Campbell JW , Cronan JE Jr ( 2001 ) Escherichia coli FadR positively regulates transcription of the fabB fatty-acid-biosynthetic gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li></div>
<li>In contrast the unsaturated fatty-acid-biosynthetic gene , fabA , is positively regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>We report that fabB , a second unsaturated fatty-acid-biosynthetic gene , is also positively regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Gene expression proﬁles of deletion mutants ( DarcA , DfadR and DarcA/DfadR ) indicated that ( i ) ArcA is a major transcription factor for the transcriptional regulation of fatty acid metabolism in the absence of oxygen , and ( ii ) ArcA and FadR cooperatively regulate the fad regulon under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The model presented here is supported by the following evidence : ( i ) FadR repression coincides with ArcA repression for the fatty-acid-metabolism under anaerobic conditions ; ( ii ) ArcA-P is a DNA-binding protein speciﬁc to the upstream of fad regulon genes involved in fatty-acid-metabolism of E. coli ; ( iii ) under anaerobic conditions , transcription of fad regulon genes is increased upon the inactivation of ArcA , whereas , under aerobic conditions , transcription of those genes is not affected by the inactivation of ArcA ; ( iv ) the fadR deletion affects both aerobic and anaerobic transcription of fad regulon genes ; and ( v ) under anaerobic conditions , the transcript levels of fad regulon genes are synergistically increased upon the deletion of both arcA and fadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Strains defective in FadR have constitutive levels of fatty-acid-degradation ( Fad ) enzymes and thus acquire the ability to grow on medium chain length fatty acids ( e.g. , decanoic acid ) that fail to induce the Fad enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The explanation for this ﬁnding is that in the presence of functional FadR , sufﬁcient mutant FabA is produced to provide a level of isomerase activity that satisﬁes cellular unsaturated fatty acid requirements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>1 The abbreviations used are : ACP , acyl carrier protein ; FabA , - hy-droxydecanoyl-ACP dehydratase ; FabB , - ketoacyl-ACP synthase I ; FabF , - ketoacyl-ACP synthase II ; SFA , saturated fatty acid ; UFA , unsaturated fatty acid ; FadR , fatty-acid-degradation regulator ; and FabR , fatty-acid-biosynthesis regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>In this study , we show that FadR directly activates another fatty-acid-synthesis promoter , PfabH , which transcribes the fabHDG operon , indicating that FadR is a global regulator of both fatty-acid-degradation and fatty-acid-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Finally , it should be noted that UspA , encoded upstream of uspB , is also implicated in membrane function during stationary phase because it is regulated , in part , by FadR , the global regulator of fatty-acid-synthesis and degradation ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>The FadR factor controls fabA and fabB expression in response to long-chain acyl-CoA derived from exogenous fatty acids ( 9 , 17 ) , and it is tempting to speculate that FabR is controlled by a ligand that is an integral component of endogenous de novo fatty-acid-biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>This considerable increase in unsaturated fatty acid production is consistent with the fact that FadR activates the expression of fabA and fabB , whose enzyme products catalyze the biosynthesis of unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Here , our results also indicate that FadR expression led to increased FabB levels , which increased fatty acid yields , especially unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>A protein bound specifically and with varying affinity to all three of the sites , fabA , fabB and yqfA ( Fig. 1 , lanes 1 -- 3 ) , that did not bind to affinity columns containing binding sites for FadR , a transcriptional regulator of fatty-acid-metabolism genes ( 22,23 ) , or PurR , which negatively regulates genes involved in purine-nucleotide-biosynthesis ( 24 ; Fig. 1 , lane 4 , and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>The Vibrio cholerae fatty acid regulatory protein , FadR , represses transcription of plsB , the gene encoding the ﬁrst enzyme of membrane phospholipid-biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The key player in this regulatory gambit is FadR , a transcription factor that acts both as a repressor of the fatty-acid-degradation and as an activator of fatty-acid-biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Enhancing fatty acid production by the expression of the regulatory transcription factor FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In addition , the intracellular levels of these compounds govern the DNA binding activity of the transcription factor FadR which in turn regulates the di $ erential expression of genes involved in fatty-acid-degradation and biosynthesis ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Here we focused on a global regulator of the fatty acid pathways , FadR , which is a transcription factor that binds to speciﬁc DNA sequences and controls the expression of several genes involved in fatty-acid-biosynthesis , degradation , and transport through the membrane ( Lutke-Eversloh and Stephanopoulos , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>The work in E. coli on the FadR protein has established that this transcription factor negatively regulates genes encoding the enzymes of fatty-acid-catabolism and activates at least fabA and fabB , the two genes essential for the de novo synthesis of unsaturated fatty acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Departments of Microbiology and Biochemistry University of Illinois Urbana , Illinois 61601 The FadR protein of E. coli activates transcription of the fabA gene , a key enzyme of fatty-acid-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The coeffector specificity of FarR resembles that of the FadR protein of E. coli , which represses fatty-acid-degradation genes , activates a fatty-acid-biosynthesis gene @bA ) , and is thought to have a role in regulating the glyoxylate-cycle operon , aceBAK [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7805834" target="_blank">7805834</a>]<br></li>
<li>Other mechanisms that regulate lipid-biosynthesis during the E. coli life cycle The mechanisms which regulate the fatty acid biosythetic genes and enzyme activities , apart from FadR , are complex and just beginning to be unraveled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The effects of fadR mutations on uspA expression suggest that UspA may have a role in fatty acid metabo-lism ; all genes demonstrated to be regulated by FadR so far have assignments in fatty acid uptake , activation , or metabo-lism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Here , we report that FadR also activates the expression of fabHDG genes , which code for enzymes involved in saturated fatty-acid-synthesis , and that this activation is reversed by the binding of long-chain fatty acyl-CoA ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>If so , the fad genes and operons , derepressed by fatty acids and repressed by FadR , should also be increasingly expressed in stationary phase while the expression of the fatty-acid-biosynthetic genes activated by FadR should be reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Moreover , Zhang and colleagues reported that overexpression of FadR increased expression of many genes in the fatty-acid-synthesis pathway , including fabH ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>FadR overexpression also altered expression of several genes in the fatty-acid-biosynthetic pathways , which increased the fatty acid yield .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Furthermore , recent transcriptomic and proteomic data suggested that FadR activates the majority of fatty acid ( FA ) synthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>RESULTS FadR activates the global expression of fatty-acid-synthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , in this paper we reassessed the regulation-of-fatty-acid-synthesis gene-expression by the functional dual regulator FadR in E. coli and showed that FadR activates all fatty-acid-synthesis genes that we tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , FadR appears to be a trans - acting protein which controls fatty-acid-degradation and acetate-degradation by decreasing the expression of the fad and ace structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2843809" target="_blank">2843809</a>]<br></li>
<li>The FadR regulatory protein negatively controls expression of the genes of the fatty-acid-degradation pathway ( 33 , 40 ) and also functions as a positive regulator of unsatur-ated fatty-acid-synthesis ( 19 , 29 , 30 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>To date , the E. coli FadR protein , which controls the expression of many genes involved in fatty-acid-synthesis and degradation , was the only regulator for which there is convincing evidence that interaction of alkyl-CoA compounds with the protein prevents DNA binding ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The fadD gene , like the fadBA , fadEa , nd fadL genes , is partof the fatty acid degradative regulon under the control of the transcriptional regulator FadR ( 11-13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1460045" target="_blank">1460045</a>]<br></li>
<li>Previous work has cataloged a substantial list of genes controlled by FadR that are either related to fatty-acid-metabolism or survival in stationary phase ( 20 , 39 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>In the present work we investigate FadR interaction with two genes which have important roles in fatty-acid-metabolism in E. coli , namely fadL and fabA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>However , the consensus sequence derived from our work provides a tool for examining other genes involved in fatty acid and phospholipid-metabolism which may be part of the FadR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Experimental evidence that both genes involved in unsaturated-fatty-acid-biosynthesis are regulated similarly would discount the possibility that FadR regulation of fabA is merely fortuitous or vestigial in nature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>In addition to its role as a repressor of the fatty-acid-degradation genes in E. coli , FadR was shown previously to activate the fabA and fabB genes required for biosynthesis of unsaturated fatty acids ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>In this dual role FadR acts as a repressor for the entire set of fad regulon genes and also functions as an activator for unsaturated-fatty-acid-biosynthesis pathway by increasing transcription of both fabA [ 7,8 ] and fabB [ 9 ] ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Thus , FadR acts as a repressor of - oxidation genes and an activator of the two genes required for unsaturated-fatty-acid-synthesis ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>However , FadR is also an activator , stimulating transcription of the products of the fabA and fabB genes responsible for unsaturated-fatty-acid-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>However , FadR is also an activator of the expression of fabA and fabB , two genes involved in unsaturated-fatty-acid-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>The FadR regulator has a dual function in repressing the enzymes of fatty acid - oxidation and activating the transcription of the fabA and fabB genes of fatty-acid-biosynthesis ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645248" target="_blank">14645248</a>]<br></li>
<li>Inhibition of FadR causes an attenuation of fatty-acid-biosynthesis through a decrease in transcription of fabA , fabB and fabI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR is the dominant regulator of the fatty acid degradative ( fad ) genes , iclR , uspA , fabA and fabB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>We extended this observation to FadR , which activates iclR transcription directly ( 13 ) in addition to its role as the regulator of fatty-acid-metabolism genes ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Previous studies indicated that transcription of genes encoding fatty acid degradative ( fad ) enzymes ( includingfadL ) is regulated by the FadR repressor ( Simons et al. 1980a , b ; Sallus et al. 1983 ; DiRusso et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The compiex control of fatty-acid-metabolism by FadR appears unique in that genes encoding enzymes required for both the catabolic [ fad structural genes ) and anabolic pathways -LCB- fab structural genes ) are co-ordinately regulated at the level of transcription by the same factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>the repressor of fatty-acid degradation ( FadR ) and three unidentified proteins ( P30 , PhnF , and LctR ) of E. coii , and KorA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>Early work by Overath and coworkers established a role for FadR as a repressor of several genes and operons encoding enzymes required for fatty-acid-degradation ( Ovarath ef ai , 1969 ; Klein et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Speciﬁcally , FadR down-regulates several genes in the fatty-acid-degradation pathway , including fadE , fadBA , fadH , and fadIJ ( Fujita et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>FadR was previously identiﬁed as a repressor for several genes in the fatty-acid-degradation pathway ( fadBA , fadD , fadIJ , and fadL ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Regulators analogous to FadR that repress fatty-acid-degradation genes have also been described in other bacteria , such as PsrA in the Pseudomonadales ( 6 ) and YsiA ( FadR ) in Bacillus subtilis ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>In the absence of fatty acids , FadR represses the transcription of fad genes required for fatty-acid-degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The role of FadR in fatty-acid-biosynthesis is less well defined than its established role in the control of transcription of the fad and ace genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2843809" target="_blank">2843809</a>]<br></li>
<li>Perhaps , the primary function of FadR in this pathogen is to activate the fatty-acid-biosynthetic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The genes encoding the fatty acid oxidative enzymes ( Fig. 2 ) are negatively controlled by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR was initially identiﬁed because it represses the genes encoding the enzymes of fatty-acid-degradation ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR was discovered through the analysis of a mutation that results in the constitutive induction of the - oxidation enzymes ( 7 ) , and led to its characterization as a repressor of fatty acid - oxidation genes ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>FadR is a repressor/activator protein which participates in control of the genes of fatty-acid-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Asterisks show positions that were mutated in the FadR binding site of iclR : : lacZ DISCUSSION FadR was originally identiﬁed as a repressor which controls the genes of fatty-acid-metabolism ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR represses the fatty-acid-degradation ( fad ) system and activates the unsaturated fatty acid synthetic pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The FadR protein is a repressor for the fad genes involved in fatty-acid-degradation but is an activator for the fab genes of the biosynthetic pathway ( Henry and Cronan , 1991a , b ; reviewed in Black and DiRusso , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>We recently reported that the FadR protein of E. coii acts both as a repressor of the genes of fatty-acid-degradation ( fad ) pathway and as a transcriptional activator of the fabA gene of the fatty-acid-biosynthetic pathway ( Henry and Cronan , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>It has been shown that FadR acts both as a repressor of the genes of the fatty-acid-degradation ( fad ) pathway , and as a transcriptional activator of fatty-acid-biosynthetic ( fab ) pathway ( Henry & Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>FadR is the repressor of the fatty-acid-degradation genes and is also the activator of the fatty acid synthetic gene , fabA ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655562" target="_blank">8655562</a>]<br></li>
<li>FadR is a known repressor of fatty-acid-degradation ( fad ) genes and an activator of fatty-acid-biosynthesis ( fab ) genes ( 23,26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Unexpected functional diversity among FadR fatty acid transcriptional regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Regulation of the fatty acid pathways is complex and involves transcriptional regulators ( including FadR , FarR , IclR , and CRP ) and post transcriptional mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Overexpression of FadR led to transcriptional changes for many genes , including genes involved in fatty acid pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Based on the known regulation , overexpression of FadR is expected to slow down fatty-acid-degradation and enhance unsaturated-fatty-acid-biosynthesis , thus increasing the total fatty acid yield .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Fatty acid biosynthesis and FadR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>FadR functions during periods of rapid growth to ensure adequate and balanced fatty acid precursors for membrane-lipid-synthesis and other lipid-derived cellular macromolecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>It is clear from the data summarized in the previous sections of this article that FadR has an important role in maintaining a balance between fatty-acid-synthesis and fatty acid and acetate-degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Thus , FadR is an important mediator in the maintenance of balanced levels of fatty-acid-synthesis and degradation ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>FadR repression and oleate induction of fatty-acid-transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Regulation by FadR affects the amounts of fatty-acid-synthesis enzymes in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>TABLE IV Fatty acid metabolic genes regulated by FadR versus FabR Fold changea Gene fadR wt fadR fabR wt Gene description fabR wt Fatty acid uptake fadD fadL -- b -- 13 13 Fatty acyl-CoA synthetase 13 14 Outer membrane long-chain-fatty-acid-transporter Fatty acid degradation fadA 17 113 18 111 - Ketoacyl thiolase I -- -- -- -- -- -- Fatty acid oxidation complex Ic Acyl-CoA dehydrogenase 2,4-Dienoyl-CoA reductase Fatty acid oxidation complex IId - Ketoacyl-CoA thiolase II fadB fadE ( yafH ) fadH ( ygjL ) yfcX ( b2341 ) yfcY ( b2342 ) Acetate metabolism aceA 14 13 Isocitrate lyase -- aceB 14 -- 14 Malate synthase aceK 13 -- 16 Isocitrate dehydrogenase kinase/phosphatase iclR 23 -- -- Repressor of aceBAK operon a The expression levels of metabolic genes were monitored in E. coli strain PDJ14 ( fadR ) , strain UB1005 ( wt ) , and in the isogenic fabR ( ANS8 ) and fadR fabR ( ANS4 ) strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Finally , FadM , which is a long-chain acyl-CoA thioesterase involved in the - oxidation of oleic acid , also is repressed by FadR but is not depicted here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>( A ) FadR regulation in cultures grown on various carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Therefore , we predicted that arti-ﬁcial overproduction of FadR from an arabinose-inducible plasmid construct , pBAD-fadR ( pEB1210 ) , might increase expression of promoters activated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Because FadR also is the repressor of FA degradation , FadR acts both as a repressor and an activator of the two opposite pathways of FA degradation and synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Evidence of a role for FadR in lipid-synthesis 138 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Evidence of a role for FadR in lipid-synthesis Several observations led to the proposal that FadR had a role in lipid-metabolism apart from repression of the fad genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Our results show that FadR is a global regulator of FA metabolism in E. coli , acting both as a repressor of catabolism and an activator of anabolism , two directly opposing pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Our work identiﬁes the FadR functional dual regulator as a global activator of almost all FA synthesis genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , although it long has been postulated that FadR only activates the fabA and fabB genes , we suspected that FadR is the missing global regulator of FA synthesis genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , a global regulation of FA synthesis by FadR also would be important to tune and coordinate the protein amounts of the FA synthesis machinery with growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , in E. coli , FadR alone is responsible for controlling the expression of the two opposite pathways of FA degradation and FA synthesis ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FIG 2 Global activation of FA synthesis gene-expression by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>However , several results of our experiments clearly demonstrate the global regulation of FA synthesis gene-expression by FadR under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Our initial ﬁnding that the fabH promoter was directly activated by FadR ( 8 ) , the global increase in the expression of FA synthesis genes in a strain overproducing FadR ( 9 ) , and the potential presence of a consensus sequence for FadR binding in the promoters of FA synthesis genes other than fabA and fabB were strong indications that FadR activates the transcription of genes additional to those previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FadR , whose principal role ﬁrst was discovered to be a repressor of FA degradation genes in E. coli , later was shown to activate the expression of fabA and fabB genes involved in unsaturated FA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Therefore , we decided to screen the effect of fadR deletion or FadR overproduction on the transcription of all of the genes involved in FA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In the present study , we tested the role of FadR in the expression of all genes involved in FA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In this study , we screened systematically the effect of fadR de-letion or FadR overproduction on the expression of all of the genes involved in FA synthesis except for fabZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Furthermore , a global transcriptome study of a strain overproducing FadR evidenced a global increase in the expression of FA synthesis genes ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Because we observed a global effect of FadR on the expression of FA synthesis genes , we also decided to test the effect of a fabR deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In this study , we used DarcA , DfadR and DarcA/DfadR mutants to analyse the roles played by ArcA and FadR on gene-expression of the fad regulon in response to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Other members of the FadR regulon including fadL , fadD , fadBA and fabA also show growth-phase dependent alterations in gene-expression [ 113 ] ( illustrated in Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Increased expression of genes fadL and fadD in the DfadR strain was also seen when the carbon source was either glucose or glycerol ( Fig. 4A and B ) , These observations are generally consistent with those obtained upon oleate induction of strains carrying a functional FadR ( Fig. 4E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>We ﬁrst checked whether the transcription regulator FadR or FabR regulated the fab-acpP locus by measuring expression from the 4 promoters yceDp1 , yceDp2 , fabH , and acpP in fadR and fabR mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Possibly , the fabR gene is controlled by another factor that ties UFA synthesis to other aspects of bacterial physiology in a fashion analogous to the control of acetate-metabolism by FadR by virtue of its ability to control the expression of iclR , a transcriptional regulator of the glyoxylate-bypass ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>By regulating the expression of iclR , FadR indirectly regulates not only the aceBAK operon but also acs encoding acetyl coenzyme A synthetase which is also regulated by IclR [ 100 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The regulation of uspA expression by FadR may suggest that UspA also has a role in lipid-metabolism which is protective during the growth-arrested state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>It is expected that expression of other genes regulated by FadR , including the glyoxylate shunt repressor IclR , are also altered in a growth-phase dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>It is worth noting that the FadR transcriptional repressor , which is regulated by acyl-CoA compounds and shows local similarity to PaaX , is also controlling the expression of genes involved in b-oxidation mechanisms ( 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>FadR is a transcriptional regulator that modulates the expression of the fabB and fabA genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>Table 5 Selected genes that display signiﬁcant transcriptional up-regulation during expression of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>FadR affects expression of aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR , which , in part , regulates uspA expression ( 15 ) , has no effect on uspB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">FadR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>In the presence of FadR , only the 704-bp fragment was shifted ( lane 5 ) , indicating that FadR speciﬁcally bound to sequences present only on that DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>The fadD promoter FadR site 2 and the ArcA site overlap by a single bp ( Fig. 2C ) as also seen in fadH [ 17 ] whereas in the fadL promoter the ArcA and FadR site 1 sequences To test for cross-talk strains carrying arcA and/or fadR null mutations plus the chromosomal fadL-lacZ ( or fadD-lacZ ) transcriptional fusions were constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>The observation that a FadR consensus pseudopalindromic operator can be identiﬁed overlapping the ﬁrst two GGTATA repeats ( Fig. 3 ) could argue that this pal-indrome is the biologically relevant motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>These sequences have 12 and 10 nucleotides , respectively , cut of 17 in common with the proposed FadR consensus sequenc ( underlined ; DlRusso ef ai , 1992 ) , There was no evidence from these experiments that one site was occupied before the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>, 1989 ) , or no obvious symmetry , i.e , FadR ( ACCTGGTCAGACGTGTG consensus ; DiRusso et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>However , the sites were located at distances ranging from 32 to 41 nucleotides relative to the transcription start sites ( 1 ) for the genes acpP , fabI , and accD , which is in agreement with the action of FadR as an activator , with a distance similar to what has been described for fabA and fabB genes ( 3 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Members of the FadR subfamily are dimers in solution ( 16 , 24 ) and bind as dimers to speciﬁc palindromic operator sites , with each monomer recognizing a half-site ( 25 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18263722" target="_blank">18263722</a>]<br></li>
<li>The FadR bind-ing-site consensus , ACCTGGTCAGACGTGTG [ 14 ] , contains a repeated AC-TG , but bears no resemblance to the FarR-site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7805834" target="_blank">7805834</a>]<br></li>
<li>5 ' - ATCTGGTGTGGGATGGC-3 ' would also bind FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Together with the DNase I results , we conclude that FadR does not bind to the upstream site a T C 1 2 3 4 5 -26 Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Computer-assisted searches for consensus FadR recognition sites within the E. coli genome identify fabB as a potential target of FadR regulation ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Lanes 4 and 5 , competition with 10 and 50 ng of unlabeled probe ; lanes 6 and 7 , competition with 10 and 50 ng of a fragment of iclR ( 2128 to 266 ) which does not contain the predicted FadR binding site ; lanes 8 and 9 , competition with 10 and 50 ng of sequences upstream of aceBAK ( 2118 to 19 relative to the start of transcription ) ; lane 10 , competition with 500 ng of salmon sperm DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Binding of FadR to sequences upstream of iclR was tested by gel shift analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>To determine whether or not the DNA upstream of the fabB coding sequence binds FadR , an in vitro gel shift analysis was 33 conducted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>( 6 ) Overlapping restriction fragments representing 240 bp of DNA upstream of the fabA-coding sequence were assayed for FadR binding by mobility shift analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>As shown in Fig. 6 , FadR clearly binds upstream of the uspA coding region at a position 188 to 1104 relative to the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>FadR binds speciﬁc operator sites upstream of the fatty acid degradative ( fad ) gene coding sequences to turn off transcription ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Puriﬁed FadR binds to DNA sequences upstream of the fabB 10 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>FadR binds upstream of fadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>We introduced mutations at the distal and less conserved part of the FadR binding motifs that we identiﬁed and farther upstream from the 35 position in order to avoid the complete destruction of the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>In contrast , induction of the tightly regulated nan operon depends on freeing multiple overlapping sites composed of three copies of the loose palindrome recognized by members of the GntR superfamily , with the ﬁrst site conforming almost exactly to the superfamily consensus sequence and to that of the FadR subfamily ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>We report that FadR binds to a DNA sequence positioned at -40 relative to the start site of the FadR-regu-lated fabA transcript ( the location favored by positive activators ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Expression of fabA and fabB is repressed by FabR , which binds a site overlapping their promoters , and is activated by FadR , which binds a consensus site located around the 40 position ( 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Finally , sequences matching the FadR binding consensus had been spotted before in acpP and fabI promoters , and they are even located at a position compatible with activation by FadR ( 4 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Binding of acyl-CoAs to FadR triggers a conformational change in FadR and releases FadR from its cognate DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>Currently , E. coli FadR is the only member of the GntR family whose DNA-bound structure has been determined ,8,9 revealing that FadR binds the short palindromic consensus sequence 5 ′ - TGGNNNNNCCA-3 ′ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>The consensus sequence which we had determined in vitro ( Table 2 ) suggested that conversion of CT at 256 and 255 to GG should greatly reduce the afﬁnity of this site for FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>of occurrences in 44 clones cleotide A T G 0 9 14 3 41 44 3 11 8 1 0 0 12 33 2 0 32 7 0 0 * * 6 3 0 0 41 3 0 0 0 0 0 2 0 0 1 1 6 2 0 32 3 6 3 21 8 12 3 22 6 10 37 1 0 36 0 5 37 7 2 42 0 42 44 0 41 1 11 31 C Consensus a a c T G G T c g A C C A G T t a The consensus sequence of the FadR binding site was determined by oligonucleotide selection ( see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Note that no evidence exists for a direct effect by either IclR or FadR : although in vitro DNA binding experiments have not been performed , sequence analysis reveals no sequence with similarity to the consensus for either transcription factor ( Wolfe , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Amino acid sequence of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Neither is a FadR homologue obvious based on sequence comparisons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Nucleotides below the box indicate the overlapping FadR consensus sequence ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>This sequence has 9 nucleotides out of 17 in common with the proposed FadR consensus sequence ( underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>This method uses the gel shift technique to select sequences capable of binding to FadR from a pool of random sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Given these precedents together with the weak dyad symmetry of the natural sites and the ﬁnding that the aligned natural FadR binding sites and 44 FadR binding sequences isolated from a random pool ( Gui et al. , 1996 ) show only one strictly conserved base , we doubt that sequence inspection will be an effective means of ﬁnding FadR-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>A synthesized 37 bp double-stranded DNA fragment was assayed for FadR binding by mobility shift assay as described in Experimental Procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The assay tests the ability of FadR to speciﬁcally bind and retard the electrophoretic mobility of the fabB-associated putative FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The experiments test the ability of FadR to speciﬁcally bind and retard the mobility of a 924-bp 33P-labeled DNA fragment containing the yafH/fadE-associated , putative FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>Surprisingly , computer-assisted sequence inspection ( 17 ) failed to detect this sequence , and it is not listed as a potential FadR binding site on the DNA Motifs web page ( http://arep.med.harvard.edu/ / fadR.html ) , although a very recent search using different criteria detected the site ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>Unlike with fadH , where the sites overlap , the FadR and ArcA-P binding sites of the fadBA promoter are separated by 42 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>In the fadL promoter the FadR sites are separated by only 8 bp whereas in the fadD promoter the two FadR sites are separated by 68 bp ( Fig. 2B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Since we can ﬁnd no sequences within the acs-nrfA intergenic region that closely resemble consensus binding sites for either IclR ( 28 ) or FadR ( 8 ) , we believe it most likely that IclR operates upon acs transcription indirectly through its ability to control the synthesis of the glyoxylate shunt enzymes ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Examination of sequence logos for the two sets of FadR sites ( Figure 5 ( a ) ) reveals that positions 5 through 8 of the half-site could easily specify a Dam site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>Taken together , the position of the FadR binding site with respect to the binding site for RNAP suggests a model in which FadR activates transcription directly by binding to the DNA cooperatively with RNAP , helping to recruit the transcription complex to the fabH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The boxed region denotes the FadR binding sites de ® ned using DNaseI protection ; +1 de ® nes the transcriptional start sites ; RBS denotes the presumptive ribosome binding sites ; and the initiation codons for both genes are underlined with their positions noted relative to the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The 235 and 210 regions of the uspA promoter , the transcription start site ( small arrow ) , the FadR binding sites ( OU1 and OU2 ) , the Shine-Dalgarno sequence ( SD ) , and the ﬁrst codon of uspA ( large arrow ) are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>The promoter and untranslated leader within fadL had two binding sites for FadR , OLI at -25 to -9 and OL2 at - 1 to +16 reiative to the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The GntR and FadR binding sites within the respective promoters contain a region of dyad symmetry , which is located very close to the transcription initiation sites ( 71 -- 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The DNA sequence for the region around the FadR-binding sites are given on the left for [ A ) and ( Bj , The regions footprinted by FadR within each sequence are marked by lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The solid box indicates the location of the FadR binding sequence shown in panel A. Open boxes indicate the 210 and 235 promoter elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>It was later demonstrated that FadR directly activates the transcription of fabA and fabB genes by binding its consensus sequence located 30 bp from the transcription start sites ( 2 , 3 ) , and that FadR binding to DNA is prevented by the binding of long-chain fatty acyl-CoA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Based on the consensus sequence for FadR binding , we hypothesized a second region found upstream of the start point of fabA transcription at -177 through -160 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>For all of the genes activated by FadR , we were able to spot a sequence matching the FadR binding consensus sequence , including the ones already mentioned for fabI and acpP ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Shaded letters in the alignment indicate a match to the computed consensus sequence motif described before for FadR binding sites in Enterobacteriales , which is shown at the top in grayscale ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Indeed , one of the uspA FadR binding sites detected in vitro ( Farewell et al. , 1996 ) shows only a very minimal match to the consensus FadR binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>For example , a sequence with an excellent match to the consensus for FadR binding is located close to ¹ 40 of the promoter of the fabI gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>However , aceBAK does not have a sequence which matches the consensus for FadR binding sites and does not appear to bind FadR in vitro ( reference 5 and this study ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>We determined the consensus sequence which is required for binding FadR by oligonucleotide rescue in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>( B ) FadR DNA binding to a synthetic consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Determination of the consensus sequence of the FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Determination of the consensus sequence of the FadR binding sitea Nu No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>DiRusso et al. ( 1992 ) proposed two potential FadR binding sites in the fadL promoter by homology search based on the F a d R binding sequence in thefadB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The binding sites are quite divergent in sequence and there is a close correlation between the a nity of the protein for the recognition sites with the level of regulation by FadR as measured using promoter fusions to lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Within each sequence the FadR binding sites identified in Figs 5A , 5B , 6A and 6B are underlined and the central residue is marked by an asterisk ( * ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Given the E. coli genomic sequence , one could hope to ﬁnd FadR binding sites by inspection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Examination of the sequence shows that the region where FadR binds has sequence similarity to other FadR binding sites ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>Alignment of this sequence with those of the known FadR binding sites ( listed at http://arep.med.harvard .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>A. sequence alignments of several known FadR binding sites from E. coli fad regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Position and sequence of FadR binding sites in responsive promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>( A ) Sequence alignments of the FadR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>FadR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>A potential FadR binding sequence upstream of acpP was reported previously ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>A sequence just upstream of the iclR promoter bears a striking resemblance to FadR binding sites found in the fatty acid metabolic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Examination of the region upstream of iclR revealed a sequence which is similar to the binding sites for FadR that are found in a variety of genes ( 5 , 10 ) ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>( A ) The sequence of a region upstream of iclR ( 33 ) is compared with reported binding sites for FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Thus in common with our 56 bp B+D fabB probe the Zhu et al probe contained both the FadR and FabR binding sites , although it lacked the extra sequences we placed upstream of the FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Native FabR binds the fabA promoter region provided that the canonical FabR binding site is extended by inclusion of flanking sequences that overlap the neighboring FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Indeed , no FadR binding site is found upstream of fabB in any of the currently available Vibrio genome sequences , although a FabR binding site is invariably found within the fabA promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>Visual inspection of the fabH promoter sequence identiﬁed a weak match of the DNA region between positions 30 and 46 to FadR binding sites upstream of the fabA , fabB , and iclR promoters , three other promoters activated by FadR ( 15 , 16 , 57 ) ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Scrutiny of the sequences in the promoter region of fabB showed that a region about 70 bp upstream of the fabB start codon had a good match to known FadR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>( A ) Consensus alignments within the FadR binding region are boxed , and sequences conserved in fabB are shaded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>Identification of a FadR DNA-Binding Consensus Sequence ( A ) Comparison of the FadR-binding region upstream of the fabA gene with the upstream sequences from two fadgenes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>In both cases , we deﬁned a new 1 site about 35 nucleotides downstream of the FadR potential binding site ( noted as P2 in Fig. 4 and 5 ; also see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Lane 2 is somewhat underloaded , but normalization of the proﬁles conﬁrmed that there was little protection by FadR alone in the region corresponding to the position of the predicted FadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Although these gel shifts suffer from nonspeciﬁc trapping of DNA , they localize the fabB-associated FadR binding site to an approximately 230-bp region at the beginning of the fabB gene consistent with the site detected by sequence inspection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>We therefore made probes ( called B2A ) for the E. coli fabA and fabB promoters in which the probes were of the same length as B2 , but the FadR binding site sequence was replaced with a random sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>However , the differential binding scenario is consistent with the finding that FabR does not specifically require the fabB B2 probe FadR binding site sequences for efficient binding in vitro , a random DNA sequence suffices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276098" target="_blank">21276098</a>]<br></li>
<li>We identified , by DNase I footprinting , a single binding site for FadR around the transcriptional start site for fadBA at +1 to + ^ 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>We were unable to distinguish affinity of FadR for binding tc site OLI from 0 ^ 2 by these experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Analyses of the E. coli sequence showed three different conserved sites upstream of the predicted transcriptional start site , which would be binding sites for the FadR repressor , the ArcA repressor , and the CRP activator ( Fig. 2D ) , the ﬁrst two of which had been predicted by others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The putative FadR binding site ( AACTCATCCGACC/TACAT ) overlaps the transcriptional start site and the downstream ArcA binding site ( TAACAAT/ATATTTTA ) ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>( A ) The putative FadR binding site in the fabH promoter is aligned with the sequences of the FadR binding site in fabA , fabB , and iclR promoters described previously ( 15 , 16 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The fabB promoter contains a putative FadR binding that shares 15 of the 17 bp of the known fabA activator binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Consensus alignment of known FadR binding sites and the putative fabB-associated site ( alignment data from reference 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The sequence of the putative fabB FadR binding site most closely matches those of the two positively regulated genes iclR and fabA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>The FadR binding site is not the only region of sequence similarity in the fabB and fabA promoters , and their alignment is shown in Fig. 1 with the sites of transcription initiation indicated by the arrows for both fabA ( 18 ) and fabB ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>In fact , FadR binds to several sites that bear little resemblance to the reported consensus binding site ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>The ﬁrst line of nucleotides below the intergenic region gives the FadR consensus pseudopalindrome , and the second line gives the FadR subfamily consensus binding site ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>The consensus binding site for FadR includes 17 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>A putative FadR binding site is located upstream of the coding sequence , and strains lacking FadR have shown a 10 - to 12-fold increase in expression as measured by mRNA levels ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>FadR is known to negatively regulate expression of fadE ( 6 , 15 ) , and thus a FadR binding site should be located upstream of the yafH/fadE coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>Within the fadE promoter region ( 300 bp upstream ) , a putative ArcA-binding site was located at the 280 position ( GTTTACAATATAATT ) , upstream of a FadR binding site located at the 225 position ( AAGTGGTCAGACCTCCT ) ( Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Indeed , a candidate sequence centered 31 bp upstream of the yafH/fadE initiation codon closely matched the FadR consensus binding site ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>This consensus is in good agreement with the naturally occurring binding sites of FadR and with the possible site which we identiﬁed upstream of iclR ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>In other studies , a sequence upstream of the putative acpP promoter ( at 240 ) was found to strongly resemble the FadR binding site consensus sequence ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655562" target="_blank">8655562</a>]<br></li>
<li>FadR accomplishes this task by binding to a site which is just upstream of the iclR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>When this putative FadR-binding sequence is aligned with the 17-bp FadR-binding sequence from the fadBA promoter ( the strongest known binding site for FadR ) , 11 out of the 17 bases are identical ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>S2 in the supplemental material ) and conﬁrmed that the FadR binding sequence was at the 40 position relative to this 1 site ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>( A ) Identiﬁcation of the FadR binding site within uspA and the sequence of the uspA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>A region with weak homology to the FadR consensus binding site is predicted in the putative promoter region ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>This region corresponds to the proposed FadR binding site presented in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by FadR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>( B ) EMSAs were performed using puriﬁed 6His-Tev-FadR and PCR products containing the accB , accD , or fabI promoters and in the presence or absence of oleyl-CoA ( C18 :1 - CoA ) or octanoyl-CoA ( C8 :0 - CoA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FadR is unusual in that it regulates two directly opposing pathways , is the ﬁrst positive activator shown to release DNA upon binding of the regulatory ligand ( Henry and Cronan , 1991 ; 1992 ) ( hence the ﬁrst example of repression mediated by positive control ) and , unlike many DNA binding proteins , does not autoregulate its synthesis ( Henry and Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Indeed , the accBp2 promoter clearly was activated when FadR was overproduced ( Fig. 5D ) , yet it was not particularly affected in the fadR mutant ( Fig. 5C ) , and we were not able to detect the 1 position from the accBp1 promoter by 5 = - RACE even in the fadR mutant ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>Furthermore , it has been reported that the anaerobic b-oxidation pathway is mediated by fadJ , fadI and fadK genes , whose transcriptional regulation is independent of FadR control ( Campbell et al. , 2003 ; Morgan-Kiss & Cronan , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>It is known that the transcription of fadJ and fadI is controlled by FadR in the presence of oxygen , whereas that of fadK is independent of FadR control ( Campbell et al. , 2003 ; Morgan-Kiss & Cronan , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>These FadR alleles , called superrepressors , are able to bind to DNA to repress transcription of the fadB gene but were not inactivated by long-chain acyl-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>FadR Does Not Regulate Its Own Synthesis Many transcriptional activators autoregulate transcription of their structural gene ( Raibaud and Schwartz , 1984 ) and therefore we tested whether FadR is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Given this latter role it seems surprising that FadR fails to repress its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>Nunn and coworkers suggested that this might be due to repression of the aceBAK operon by FadR [ 93 ± 95 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Finally , it should be noted that UspA , encoded upstream of uspB , is also implicated in membrane function during stationary phase because it is regulated , in part , by FadR , the global regulator of fatty acid synthesis and degradation ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>Dual control of fabH by FadR and ppGpp/DksA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Then , we demonstrate that the promoter driving expression of the fabHDG operon ( the fabH promoter ) is in fact activated directly by FadR and inhibited directly by ppGpp / DksA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Transcription of the Escherichia coli fatty acid synthesis operon fabHDG is directly activated by FadR and inhibited by ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>FadR prevented this increase by activating iclR transcription on carbon sources such as glucose and acetate but not on oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>The fadR gene ( not shown ) produces FadR , which binds to the various operators , repressing transcription of fadBA and fadE , while activating transcription ( sawtooth lines ) from the fabA and iclR promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>They suggested that FadR exerts its effect on phospholipid metabolism by activating , in an undefined manner , fabA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The promoter located further upstream gives weak constitutive expression of the gene , whereas the second promoter gives strong expression only when FadR is present ( Henry and Cronan , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1505031" target="_blank">1505031</a>]<br></li>
<li>The promoter located adjacent to the coding sequence requires FadR for optimal expression and is activated about 20-fold upon Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Both of these genes have putative FadR binding sites located in regions downstream of the predicted transcriptional start site , consistent with a role for FadR in repressing their expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11859088" target="_blank">11859088</a>]<br></li>
<li>A putative FadR binding site is located upstream of the coding sequence , and strains lacking FadR have shown a 10 - to 12-fold increase in expression as measured by mRNA levels ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>FadR is known to negatively regulate expression of fadE ( 6 , 15 ) , and thus a FadR binding site should be located upstream of the yafH/fadE coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>However , as direct FadR regulation of fabF or acc gene expression has not been reported , increases in fabF and accA mRNA levels may be indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>The expression of the aceBAK operon , which codes for the key enzymes of the glyoxylate cycle , isocitrate lyase and malate synthase , and for isocitrate dehydrogenase kinase/phosphatase , is activated by FruR and IHF and is repressed by IclR , which in turn is activated by FadR ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>This was a surprising result since FadR activates the expression of this construct during growth on glucose and acetate but not during growth on oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Gui et al. , ( 1996b ) proposed that FadR , which is known as a repressor for the transcription of the fatty acid deg-radation operon , regulates the expression of iclR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>Furthermore , the regulator of fatty acid metabolism ( FadR ) , encoded by fadR , activates the expression of the ICL repressor , which inhibits the expression of the glyoxylate operon ( Phue et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19452501" target="_blank">19452501</a>]<br></li>
<li>The gene expressions of fadR and iclR were also higher , where FadR activates iclR , and IclR is known to repress aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19055729" target="_blank">19055729</a>]<br></li>
<li>The aceBAK operon is under negative regulation by IclR , FadR ( indirectly by regulation of the iclR expression ( 62 ) , Crp ( 52 ) , and ArcA ( 63 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17426033" target="_blank">17426033</a>]<br></li>
<li>By regulating the expression of iclR , FadR indirectly regulates not only the aceBAK operon but also acs encoding acetyl coenzyme A synthetase which is also regulated by IclR [ 100 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>These experiments demonstrate that expression of fabB is positively regulated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566998" target="_blank">11566998</a>]<br></li>
<li>FadR affects expression of aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR also activates the expression of iclR , the gene which encodes the repressor protein of the glyoxylate bypass operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR activates iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>These observations conﬁrm our conclusion that FadR activates iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>FadR action is straightforward , it acts as a classical LacI-type repressor and only weakly represses fadD and fadL expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Oleate induced fadH expression to essentially the same level as that seen in a FadR strain , indicating that there was little or no residual repression ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>FadR regulates fadM expression only weakly , whereas FadR regulation of fadH expression is quite robust ( Fig. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>As mentioned above , there is an apparent H. in ¯ uenzae FadR homologue and there are FadL and FACS homologues suggesting that it can aquire fatty-acids from its environment , but no other members of the fatty acid degradative genes are identi ® ed by sequence analysis of the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The genes encoding the fatty acid oxidative enzymes ( Fig. 2 ) are negatively controlled by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The compiex control of fatty acid metabolism by FadR appears unique in that genes encoding enzymes required for both the catabolic [ fad structural genes ) and anabolic pathways -LCB- fab structural genes ) are co-ordinately regulated at the level of transcription by the same factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>The Vibrio cholerae fatty acid regulatory protein , FadR , represses transcription of plsB , the gene encoding the ﬁrst enzyme of membrane phospholipid biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The coeffector specificity of FarR resembles that of the FadR protein of E. coli , which represses fatty acid degradation genes , activates a fatty acid biosynthesis gene @bA ) , and is thought to have a role in regulating the glyoxylate cycle operon , aceBAK [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7805834" target="_blank">7805834</a>]<br></li>
<li>This transcription pattern shows that FadR negatively regulates yafH and that induction of yafH expression upon supplementation of cultures with fatty-acids is mediated by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057976" target="_blank">12057976</a>]<br></li>
<li>In E. coli , expression of the fatty acid degradation ( fad ) genes is inhibited by the transcriptional repressor FadR in its apo form , but when long-chain-fatty-acids are available , acyl-CoA binds to FadR , releasing the repressor from the promoter and inducing fatty acid degradation ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The activation of fabH expression by FadR and its inhibition by oleate suggest that FadR activates the fabH promoter in the absence of long-chain-fatty-acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>The expression of fadL is under the control of the FadR repressor and is induced in the presence of long-chain-fatty-acids ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987139" target="_blank">1987139</a>]<br></li>
<li>The transcription of fadL is induced by Crp in response to cAMP levels , and repressed by FadR in the absence of exogenous fatty-acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Q 1998 Blackwell Science Lt The model ( Fig. 1 ) In the absence of an exogenous source of long-chain-fatty-acids , FadR is bound to all of its cognate operators acting both to repress transcription of the fad genes and to activate transcription of the fabA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>RESULTS ArcA downregulates the transcription of fatty acid transport genes under anaerobic conditions The expression of fadL , which encodes one of the outer-membrane-bound proteins involved in the uptake of fatty-acids , is regulated by at least three transcription factors : Crp , OmpR and FadR ( Berg et al. , 2004 ; Busch & Saier , 2002 ; Clark & Cronan , 1996 ; DiRusso et al. , 1992 ; Cronan & Subrahmanyam , 1998 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>All genes in this pathway are repressed by FadR in the absence of fatty-acids ( Henry & Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Under aerobic conditions , the absence of LCFAs induces FadR to repress transcription of the fad regulon genes by direct binding to their promoter ( Henry & Cronan , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Other members of the FadR regulon including fadL , fadD , fadBA and fabA also show growth-phase dependent alterations in gene expression [ 113 ] ( illustrated in Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Role of the Esch-erichia coli FadR regulator in stasis survival and growth phase-dependent expression of the uspA , fad , and fab genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>Regulation of expression of the universal shock protein A , fad and fab genes by FadR is growth phase dependent 150 4.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Regulation of expression of the universal shock protein A , fad and fab genes by FadR is growth phase dependent The ® rst indication FadR was an important player in the adaptive response to growth arrest came from the discovery that the universal stress protein A ( UspA ) was negatively regulated by FadR in a growth phase-dependent manner [ 113 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>If so , the fad genes and operons , derepressed by fatty-acids and repressed by FadR , should also be increasingly expressed in stationary phase while the expression of the fatty acid biosynthetic genes activated by FadR should be reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>The expression of the uspA gene is , however , repressed by FadR during exponential growth and expression increases as the cells enter stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The genes negatively regulated by FadR are derepressed upon entry into stationary phase and fabA gene expression is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Expression of fabA decreases upon entry into stationary phase , an e $ ect which is dependent upon FadR ( [ 113 ] and discussed above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In this study , we demonstrate that stationary-phase induction of uspA is dependent on the status of the fadR gene and that the FadR regulon and uspA are induced concomitantly in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932299" target="_blank">8932299</a>]<br></li>
<li>If FadR is needed to activate the fabH promoter , why do fadR mutants not display the same growth defects as do strains deleted for the genes that it controls ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>( ii ) FadR negatively controls ( or represses ) at least seven genes and operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>It also restored the FadR + phenotype of this strain , indicating that genes constituting the dad operon as well as the closely linked fadR gene ( 59 ) were cloned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7906689" target="_blank">7906689</a>]<br></li>
<li>Overexpression of any of these genes alone did not result in a high yield comparable to fadR expression , indicating that FadR enhanced fatty acid production globally by tuning the expression levels of many genes to optimal levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>First , both the decreased expression and the decrease in protein amounts in the fadR mutant compared to those of the wild-type strain show that under wild-type conditions , FadR does activate the expression of the FA synthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
<li>S2B and its legend in the supplemental material ) , probably because fabH was expressed by read-through transcription from the relatively strong yceDp1 and yceDp2 promoters ( see also references 35 and 36 ) , FadR expression did not increase FabH-SPA production in this construct ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>We suggest that the fabH promoter may not be the only contributor to fabH expression under all conditions , that other activators might compensate to activate fabH expression when FadR is absent , and/or that basal expression of the fabHDG genes ( either from unactivated fabH core promoter activity or from read-through transcription from the upstream promoters , yceDp1 and - p2 ) might be sufﬁcient for cells to tolerate the absence of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23772072" target="_blank">23772072</a>]<br></li>
<li>For example , fadB and fadE are both repressed by FadR in the absence of long-chain FAs and by ArcA-P under anaerobic growth conditions ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Cho and coworkers [ 12 ] initially used quantitative RT-PCR to assay regulation of fadL and fadD by FadR and ArcA under anaerobic conditions and reported that deletion of either arcA , fadR alone , or both arcA and fadR resulted in increased fadL expression by 34 - , 4 - or 69-fold , respectively , whereas fadD transcription increased 69 - , 4.5 - , and 84-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23029459" target="_blank">23029459</a>]<br></li>
<li>Moreover , fadH expression was increased 8 - to 10-fold under anaerobic conditions upon deletion of both the fadR and the arcA gene , indicating that anaerobic expression is additively repressed by FadR and ArcA-ArcB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Interestingly , under anaerobic conditions , the expression of fadK is independent of FadR and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>Although FadR and IclR mediate the responses of a variety of genes to the carbon source , they also appear to play the paradoxical roles of ensuring that the transcription of iclR does not change under these same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>Expression of FadR under our conditions increased the transcription of genes for phage shock proteins PspA and PspG by 6.2 - and 8.5-fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>It should be noted that Farewell et al. ( 1996 ) reported that the transcription of uspA , a gene involved in response to a variety of environmental stresses , is repressed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767562" target="_blank">9767562</a>]<br></li>
<li>Repression of fadH transcription by the ArcA-ArcB system is additive to FadR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Repression of fadH transcription by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>Furthermore , we show that transcription in vitro of fabA is activated by FadR while transcription of fadL is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>Our in vifro transcription assays show that transcription of fadL by E. coii RNA poiymerase and o ^ ° is repressed by the addition of FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8446033" target="_blank">8446033</a>]<br></li>
<li>For example , all the fad genes repressed by FadR are also repressed by the ArcA transcription factor ( encoded by arcA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>fabA and fabB genes are repressed by FabR and activated by FadR ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25802297" target="_blank">25802297</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>