<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - FliZ</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">FliZ</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">FliZ transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( http://creativecommons.org/licenses/ by-nc/3 .0 ) , which permits unrestricted non-commercial use , distribution , and reproduction in any medium , provided the original work is properly cited MATERIALS AND METHODS Standard molecular biology and biochemical techniques For the puriﬁcation of FliZ protein , SDS-PAGE , immunoblot analysis , electrophoretic mobility shift assays ( EMSA ) , DNaseI footprint analysis and the determination of b-galactosidase activity in cellular extracts , standard procedures were applied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D. and Slauch , J.M. ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The activity and stability of FlhDC are regulated by FliZ , FliT , and FliD , the downstream gene products in the ﬂagellar regulon , by YdiV , an EAL-like protein acting as an anti-FlhD4C2 factor , by ClpXP , an ATP-dependent protease , and by DnaK , a protein chaperone ( 1 , 45 , 46 , 58 , 59 , 62 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>The FlhD2C2 complex activates transcription of the ﬂiAZY operon , and the s28 ( FliA ) and FliZ protein products act to increase ﬂagellar gene expression : s28 will transcribe early , middle , and late genes , and FliZ enhances the function of the FlhD2C2 complex on class 2 promoter transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>Being itself under positive FlhDC control , FliZ thus can exert a negative feedback control in motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Their timing of activation is precisely controlled by FliZ ( Fig. 2 ) and by Crl ( Supplemental Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Many of these genes are activated by s and repressed by FliZ ( including the curli genes and other sScontrolled genes , i.e. gadBC , gadE , hdeAB and yjbJ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Moreover , in contrast to ` directional ' promoter binding by EsS , FliZ could bind to the same or the opposite DNA strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ clearly bound to DNA containing promoters of genes , which , like mlrA and yciR , are expressed from sS-dependent promoters and showed negative regulation by FliZ in the microarray experiments ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Overall , FliZ seems to negatively regulate sS-dependent genes by directly binding to sS-controlled promoter regions , while other modes of regulation by FliZ are likely to be indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>As expected for a FlhDC-activated gene product , FliZ levels increased during the post-exponential phase of a culture growing in LB , reached a peak at an OD578 of about 2 , then steadily declined and could hardly be detected anymore in over-night cells ( the limit of detection under our experimental conditions corresponded to approximately 5 % of the maximal FliZ levels measured in the post-exponential phase ; see Figure S9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>When this plasmid was introduced into MC4100 , which due to its flhDC mutation does not express any other flagellar gene product besides this plasmid-encoded FliZ , it completely inhibited curli expression during entry into stationary phase even in the absence of inducer ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>The mutant protein FliZ-R108A was indeed strongly reduced in its ability to-bind to a series of target promoter fragments ( Figure 4B , with data quantiﬁed in Supplementary Figure S6 ) ; this is due specifically to the loss of R108 , since overall stability of the FliZ-R108A protein is similar to that of wild-type FliZ , as indicated by similar digestion patterns and kinetics in limited proteolysis experiments ( Supplementary Figure S7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Since this resulted in higher FliZ levels than in the wild-type , the difference in target gene repression was best seen with a target with moderate afﬁnity for FliZ such as yciR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>While plasmid-encoded wild-type FliZ strongly repressed yciR expression , yciR repression was relieved with FliZ-R108A ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">FliZ protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Kutsukake et al. ( 68 ) pointed out that FliZ protein is predicted to contain a coiled-coil domain , which is believed to be involved in protein-protein interactions ( 83 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>Thus , it appears unlikely that FliZ provides this function for FliY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550423" target="_blank">8550423</a>]<br></li>
<li>For example , mutating residues of a CAAAACTG motif that is found in the spacers of both the mlrA and yciR promoters did not affect FliZ binding to the mlrA promoter ( Figure 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>R108 and the two neighboring amino acids in FliZ are ﬂanked by strongly conserved residues ( Supplementary Figure S2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Within this element , R108 in FliZ corresponds to K173 in pS , which contacts a conserved cytosine at the 13 promoter position that is specific for pS-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>In this putative a-helix , FliZ also features a positively charged amino acid ( R108 ) at the position corresponding to K173 in sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Similar as discussed above for sS , FliZ could also make additional speciﬁc contacts to DNA that differ from the ones made by sS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Overall , the DNA-binding helices in FliZ and in domain 3 of sS show the same pattern of conserved residues interspersed with variable residues used for speciﬁc DNA sequence rec-ognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Such contacts may account for FliZ binding to additional sites besides the extended 10 ( - like ) elements as can be observed in the footprint experiments ( Figure 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The DNA-binding element in FliZ is similar to the a-helix 3.0 in pS that is crucial for promoter binding in the extended 10 region The C-terminal part of FliZ contains a region of 76 amino acids ( of a total of 183 amino acids of FliZ ) with similarity to the DNA-binding domain of the phage integrase family ( Supplementary Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Similarities in DNA-recognition by FliZ , domain 3 of p factors and phage integrase family members The alignment of FliZ with the core-binding domains of several phage intergrase family members reveals that the putative a-helix in FliZ , which contains R108 and looks like a mimic of the extended 10 recognition helix 3.0 in sS , corresponds to the DNA-binding a-helix 2 in phage integrase family members ( Supplementary Figure S2A ) ( 21,33 -- 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The C-terminal region of FliZ indeed contains a predicted a-helix that is S similar to helix 3.0 of s ( Figure 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>( A ) Alignment of a putative a-helix in the C-terminal region of FliZ with S the extended 10 recognition helix 3.0 of s ( RpoS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>This resemblance to the extended 10 recognition element in sS prompted us to analyze a potential role of R108 in the interaction of FliZ with sS-dependent promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Moreover , FliZ binds to DNA using a structural element that shows similarity to the DNA-binding element in sS , which allows sS ( within the RNAP holo-enzyme complex ) to recognize the extended 10 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Taken together , our in vivo and in vitro data indicate that ( i ) FliZ antagonizes expression of a series of sS-de-pendent genes by directly binding within their promoter regions ; ( ii ) the FliZ binding site on the DNA is similar ( though not necessarily identical ) to the extended 10 promoter region and in particular includes a cytosine , S which in an active s - dependent promoter corresponds to C ( 13 ) and ( iii ) FliZ uses a structural element ( containing R108 ) for DNA-binding that is similar to the extended 10 recognition helix in sS , i.e. region 3.0 , which contains the key residue K173 that interacts with C ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Moreover , FliZ contains a region with similarity to certain phage integrases , which is also present in XerD , a DNA-binding protein and site-specific recombinase involved in chromosome decatenation ( Subramanya et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Moreover , the residue corresponding to E109 in FliZ plays a role in determining DNA-binding speciﬁcity in lambda integrase ( 36,37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ and s use a similar element for DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>This suggests that FliZ , domain 3 of s factors and proteins of the phage integrase family may all use a similar mechanism of DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">FliZ evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Overall , these two a-helices in FliZ and sS show an identity of 31.6 % and a similarity of 42.1 % ( for comparison : the respective a-helices in FliZ and XerD , which is the phage integrase family member that shows strongest similarity to FliZ ( Supplementary Figure S2A ) , display 26.3 % identity and 36.8 % similarity , while the identity and similarity betwee the respective a-helices of RpoS and the closely related vegetative s factor RpoD are 21.1 % and 63.2 % , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>This suggests that FliZ , domain 3 of s factors and proteins of the phage integrase family may all use a similar mechanism of DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The DNA-binding element in FliZ is similar to the a-helix 3.0 in pS that is crucial for promoter binding in the extended 10 region The C-terminal part of FliZ contains a region of 76 amino acids ( of a total of 183 amino acids of FliZ ) with similarity to the DNA-binding domain of the phage integrase family ( Supplementary Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Similarities in DNA recognition by FliZ , domain 3 of p factors and phage integrase family members The alignment of FliZ with the core-binding domains of several phage intergrase family members reveals that the putative a-helix in FliZ , which contains R108 and looks like a mimic of the extended 10 recognition helix 3.0 in sS , corresponds to the DNA-binding a-helix 2 in phage integrase family members ( Supplementary Figure S2A ) ( 21,33 -- 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by FliZ</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Along the curli control cascade , there are many potential targets and , as a first step , we tested whether the presence of the inhibitory plasmids pFlhDC and pFliZ affected the cellular levels of the essential curli activators S and CsgD ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Wada , T. , Tanabe , Y. and Kutsukake , K. ( 2011 ) FliZ Acts as a Repressor of the ydiV Gene , Which Encodes an Anti-FlhD4C2 Factor of the Flagellar Regulon in Salmonella enterica Serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Moreover , FliZ contains a region with similarity to certain phage integrases , which is also present in XerD , a DNA-binding protein and site-specific recombinase involved in chromosome decatenation ( Subramanya et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Using single-copy lacZ reporter fusions , we then found that pFliZ reduced the expression of MlrA , YdaM , and YciR ( Fig. 1B ) , which together and highly specifically control csgD transcription ( Weber et al. 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Indeed , the expression of Scontrolled genes not associated with curli expression was also reduced by pFliZ : While osmY , a directly E Scontrolled gene , was partially affected ( Fig. 1B ) , the expression of gadB -- i.e. , a gene involved in acid resistance , which is under multiple feedforward control of S ( Weber et al. 2005 ) in a manner analogous to csgB -- was also completely eliminated ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Here , we show that FliZ is a potent inhibitor for the S-controlled general stress response ( Fig. 1 ; Supplemental Table S1 ) , with those genes being most affected that are under multiple S control in feedforward loops ( csgB , gadB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Our results may also shed some light on the mechanisms underlying the reported impact of FliZ in the regulation of virulence and motility in other bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Since this shut-down is at least in part a consequence of the competition of s70 , sFliA and sS for RNAP core enzyme ( 10 ) , and FliZ also globally affects sS-driven gene-expression as discussed above , FliZ could be a key player in setting thresholds in the s factor competition balance that controls switching between the motile ` foraging ' life-style and a life-style characterized by slow or no growth , high stress resistance and cellular adhesion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Published online 9 February 2012 The global repressor FliZ antagonizes gene pS-c expression by ontaining RNA polymerase due to overlapping DNA binding specificity Christina Pesavento and Regine Hengge * Institut für Biologie -- Mikrobiologie , Freie Universität Berlin , Königin - Luise-Strasse 12-16 , 14195 Berlin , Germany Received August 4 , 2011 ; Revised January 5 , 2012 ; Accepted January 16 , 2012 ABSTRACT FliZ , a global regulatory protein under the control of the flagellar master regulator FlhDC , was shown to antagonize pS-dependent gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Being itself under positive FlhDC control , FliZ thus can exert a negative feedback control in motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Lanois , A. , Jubelin , G. and Givaudan , A. ( 2008 ) FliZ , a ﬂagellar regulator , is at the crossroads between motility , haemolysin expression and virulence in the insect pathogenic bacterium Xenorhabdus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ , a flagellar regulator , is at the crossroads between motility , haemolysin expression and virulence in the insect pathogenic bacterium Xenorhabdus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li>
<li>A ﬂiZ mutant showed a subtle but reproducible increase in swimming motility as compared to the wild-type , while FliZ expression from the low-copy plasmid reduced motility ( Figure 6A ) , indicating that in E. coli , FliZ acts as a weak negative regulator of motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Thus , FliZ acts as a positive regulator of motility in Salmonella ( 31 ) and X. nematophila ( 23 ) , while we show here that in E. coli FliZ represses motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>terferes FliZ , a protein under flagellar control , in S with activity and curli formation Down-regulation of motility by a S-controlled c-di-GMP module comprising the GGDEF proteins YegE/YedQ/YeaJ and the EAL protein YhjH With FliZ , the flagellar system contains a factor that during the post-exponential growth phase interferes with the general stress response and therefore also with adhesive curli fimbriae expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>The FlhDC/motility control system contains at least three components that down-regulate the S/curli control system ( Fig. 7 , red arrows ) : ( 1 ) FliZ interferes with S activity , ( 2 ) FliA ( F ) competes with S for RNAP core enzyme , and ( 3 ) the EAL protein and phosphodiesterase YhjH eliminates c-di-GMP that otherwise ( via YcgR ) inhibits motility and ( via a second still unknown effector component ) activates csgD transcription and therefore curli expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Many of these genes are activated by s and repressed by FliZ ( including the curli genes and other sScontrolled genes , i.e. gadBC , gadE , hdeAB and yjbJ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>This would not only affect ﬂagellar components but would also result in higher expression of two regulatory factors that downregulate CsgD and the bioﬁlm matrix components curli and cellulose : ( i ) PdeH ( YhjH , see above ) , which keeps c-di-GMP levels low and thereby interferes with the expression of CsgD , and ( ii ) FliZ , a histone-like protein that downregulates many RpoS-dependent genes ( including those involved in activating CsgD expression ) ( 15 , 70 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26303830" target="_blank">26303830</a>]<br></li>
<li>The genes thus identified by microarray analysis , however , represent those indirectly affected because the SutR regulon includes the TF FliZ , which controls a set of genes involved in curli expression and motility ( Pesavento and Hengge , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li>
<li>In E. coli K12 , FliZ acts by indirectly causing downregulation of genes involved in the expression of curli ﬁmbriae , which are required for surface attachment ( Pesavento et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>( B ) FliZ reduces the expression of S-controlled genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>FliZ acts as a timing factor for the expression of certain S-controlled genes , including those involved in curli expression ( also thereby antagonizing Crl ) , as illustrated by the mlrA and ydaM genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>However , a recent publication demonstrates that FliZ indirectly activates ﬂagellar gene-expression in Salmonella by directly binding to and repressing transcription from the nlpC promoter that also controls expression of the ydiV gene , which encodes the anti-FlhDC factor YdiV ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ acts as a repressor of the ydiV gene , which encodes an anti-FlhD C factor of the flagellar 4 2 regulon in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>Similarly , the yciR gene , which speciﬁes a cyclic di-GMP-speciﬁc PDE involved in the regulation of csgD expression ( 18 ) , was previously shown to be repressed by FliZ ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The FlhD2C2 complex activates transcription of the ﬂiAZY operon , and the s28 ( FliA ) and FliZ protein products act to increase ﬂagellar gene-expression : s28 will transcribe early , middle , and late genes , and FliZ enhances the function of the FlhD2C2 complex on class 2 promoter transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>In Salmonella , FliZ was suggested to post-transcriptionally regulate ﬂagellar gene-expression ( 31 ) , the expression of pathogenicity island 1 ( SPI1 ) genes ( 39 -- 41 ) as well as type 1 ﬁmbrial genes ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>These binding properties confer to FliZ a broad role in the global control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ is a posttranslational activator of FlhD4C2-dependent ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_31">
Saini , S. , Brown , J.D. , Aldridge , P.D. , and Rao , C.V. ( 2008 ) FliZ Is a posttranslational activator of FlhD4C2-dependent ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></label>
<input id="rp_similarity_31", type="checkbox">
<div class="RP_similar">
<li>Saini , S. , Brown , J.D. , Aldridge , P.D. and Rao , C.V. ( 2008 ) FliZ Is a posttranslational activator of FlhD4C2-dependent ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li></div>
<li>FliZ induces a kinetic switch in ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>FliZ antagonizes sSactivity during post-exponential growth , when ﬂagellar gene-expression and motility peak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The other mode of antagonistic regulation of motility and adhesion involves FliZ protein , which is under control of the ﬂagellar gene hierarchy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Switching from motility to adhesion requires down-regulation of flagellar gene-expression and proteolysis of the flagellar master regulator FlhDC With FliZ giving priority to motility , down-regulation of the flagellar system is required to allow the activation of S-dependent genes , including the control cascade for the expression of the adhesive curli fimbriae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>As a result , FliZ gives priority to motility and the planktonic lifestyle over sS-dependent gene-expression ( including curli ﬁmbriae expression ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>DISCUSSION During the post-exponential growth phase of E. coli , where FliZ levels reach a maximum , FliZ globally interferes with sS-dependent gene-expression and thereby gives priority to motility over sS-mediated functions which include curli ﬁmbriae-mediated adhesion ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">FliZ binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Promoter region nucleotides involved in FliZ binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>In the yciR promoter sequence as proposed by Cairrao et al. ( 24 ) , FliZ bound to the spacer region , the 35 element and further upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Together , these results indicate that FliZ and s bind DNA with overlapping sequence speciﬁcity , with a cytosine being a key nucleotide for interaction with both factors , which in an active sS-dependent promote sequence corresponds to C ( 13 ) in the extended 10 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Overall , this determination of binding sites for FliZ in sS-dependent promoter regions and/or sS promoter-like sequences suggests that FliZ may have overlapping binding speciﬁcity with sS-containing RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>It should also be noted that apart from the presence of sequences that constitute or resemble a sSdependent 10 promoter element , we were unable to detect any other common features in the FliZ binding sites identiﬁed in this report .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>A mlrA 0 yciR 0 FliZ mlrA-TR FliZ 0 rpoS 0 The C ( 13 ) element , which provides pS selectivity to a promoter , also contributes to FliZ binding With FliZ speciﬁcally recognizing sequence elements that correspond to or resemble sS-controlled promoters , the question arose how FliZ is able to distinguish between the very similar s70 - and sS-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Here , we show that FliZ is an abundant DNA-binding protein that antagonizes expression of many sS-dependent genes by recognizing DNA sequences that ressemble the extended 10 promoter regions of these genes which conform to the consensus sequence TCTATACTTAA ( with the core 10 hexamer in italics ) ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>In conclusion , the speciﬁc FliZ binding site in the mlrA promoter region includes the extended 10 region , consisting of the 10 hexamer and the adjacent TC ( 14 / 13 ) element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>We conclude that FliZ interferes with ﬂhDC expression by binding to a sequence right downstream of the transcriptional start site , which resembles the 10 element of a sSdependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ is an abundant DNA binding protein which resembles pS in a specific DNA binding region and the DNA sequence recognized FliZ directly binds to DNA regions that either correspond to known extended 10 promoter regions ( consisting of nucleotides at positions 14 to 7 ) of sS-dependent genes such as mlrA and gadE ( Figure 2 ) or to sequences that resemble such sS-dependent promoter elements but are not active as such , as analyzed here for the binding site right downstream of the ﬂhDC promoter ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ-binding is not restricted to active pS-dependent promoters In the ﬂhDC promoter FliZ binds to a sequence , which is similar to -- but not functional as a 10 element of a sS-dependent promoter and which is located right downstream of the known vegetative ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The hdeA promoter does not contain a C ( 13 ) ; however , the opposite strand features a sequence with similarity to a sS-dependent 10 promoter element ( including an appropriate cytosine ) , which overlaps with the FliZ binding site ( underlined in red in Figure 2B ) and might therefore be recognized by the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Interestingly , this FliZ binding site in the yciR promoter contains a sequence that resembles the sS-dependent consensus 10 promoter element ( located 6 bp upstream of the proposed 10 region ; Figure 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>In the yciR and hdeA promoters , FliZ seems to bind to sequences that also resemble the sS-dependent consensus 10 element but that overlap only with the actual promoter ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by FliZ</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Expression patterns of the S-dependent genes csgB , mlrA , ydaM , yciR , osmY , and gadB were recorded using appropriate single-copy reporter fusions in strain MC4100 carrying either pFliZ ( squares ) or the vector ( pCAB18 ; circles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>FliZ-binding is not restricted to active pS-dependent promoters In the ﬂhDC promoter FliZ binds to a sequence , which is similar to -- but not functional as a 10 element of a sS-dependent promoter and which is located right downstream of the known vegetative ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>By contrast , FliZ did not bind to a control fragment comprising part of the translated region of the mlrA gene , nor to a region 70 including the s - dependent promoter of the rpoS gene , which encodes sS ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ clearly bound to DNA containing promoters of genes , which , like mlrA and yciR , are expressed from sS-dependent promoters and showed negative regulation by FliZ in the microarray experiments ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>When this plasmid was introduced into MC4100 , which due to its flhDC mutation does not express any other flagellar gene product besides this plasmid-encoded FliZ , it completely inhibited curli expression during entry into stationary phase even in the absence of inducer ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Indeed , the expression of Scontrolled genes not associated with curli expression was also reduced by pFliZ : While osmY , a directly E Scontrolled gene , was partially affected ( Fig. 1B ) , the expression of gadB -- i.e. , a gene involved in acid resistance , which is under multiple feedforward control of S ( Weber et al. 2005 ) in a manner analogous to csgB -- was also completely eliminated ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Similarly , the yciR gene , which speciﬁes a cyclic di-GMP-speciﬁc PDE involved in the regulation of csgD expression ( 18 ) , was previously shown to be repressed by FliZ ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ directly affects motility by repressing ﬂhDC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>FliZ induces a kinetic switch in ﬂagellar gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>Here , we dem-onstrate that FliZ down-regulates ﬂagellar gene expression by directly binding to a regulatory region of the ﬂhDC operon which encodes the ﬂagellar master regulator FlhDC ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>However , a recent publication demonstrates that FliZ indirectly activates ﬂagellar gene expression in Salmonella by directly binding to and repressing transcription from the nlpC promoter that also controls expression of the ydiV gene , which encodes the anti-FlhDC factor YdiV ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>The activity of sF was also suggested to be under the control of FliZ and FliY , encoded by genes organized in the same operon with rpoF ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209042" target="_blank">9209042</a>]<br></li>
<li>Many of these genes are activated by s and repressed by FliZ ( including the curli genes and other sScontrolled genes , i.e. gadBC , gadE , hdeAB and yjbJ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>( B ) FliZ reduces the expression of S-controlled genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>FliZ acts as a timing factor for the expression of certain S-controlled genes , including those involved in curli expression ( also thereby antagonizing Crl ) , as illustrated by the mlrA and ydaM genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>