<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - FNR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">FNR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">FNR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The presence of apoFNR in aerobically and anaerobically grown E. coli was analyzed in situ using thiol reagents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The presence of one or two accessible residues in anaerobic cells suggests that the bacteria contain some apoFNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>) 1,2 139 CRP cytR 0 1 p 1,2 142 CRP deop2 1 1 p 1,2 142 , 162 CRP galpi 1 1 0 1,2 60 CRP glpD 0 1 0 2,4 174 CRP glnApl 1 1 0 4 119 CRP ilvB 0 0 0 4 39 CRP lac 0 1 0 1,2 87 CRP malT 0 0 0 1,2 88 CRP melR 0 0 0 1,2 169 CRP nagB 1 0 0 4 108 CRP nagE 1 0 0 4 108 CRP pBRp4 0 0 0 2,4 113 CRP putA 1 1 0 4 47 CRP rpoHpS 1 0 0 4 99 CRP tnaA 0 0 0 2,4 27 CRP tsx 0 1 p 4 142 , 165 FNR dmsA 0 0 0 4 35 FNR fdrA 0 0 0 1,4 35 , 61 FNR , Y gipA 0 0 0 4 35 FNR , Y narG 0 0 0 1,4 35 , 75 FNR nirB 0 1 0 1,4 35 , 61 , 62 FNR PFLp6 1 0 0 4 133 FNR PFLp7 1 0 0 4 75 , 133 IlvY ilvC 1 0 p 3 170 MalT malE 1 1 R 1,2 114 , 115 MalT malK 1 1 R 1,2 114 , 115 MalT malPQ 0 0 0 1,4 138 MetR metE 1 1 0 1,3 161 MetR metH 0 0 0 4 160 , 161 OmpR ompC 0 0 p 1,2 80 , 81 , 154 OmpR ompF 0 0 p 1,2 38 , 116 , 154 OxyR ahpC 0 0 0 3 146 OxyR katG 0 0 0 3 146 , 149 PapB papB 1 1 R 1,2 37 , 41 PhoB phoA 0 0 0 4 84 PhoB phoB 0 0 0 1,4 84 , 154 PhoB phoE 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>, Svnechocvstis species ; DnrD Ps , Pseudomonas stutzeri ; DnrE Ps , Pseudomonas stutzeri : DnrS Ps , Pseudomonas stutzeri ; EtrA Sp , Shewanella putreficians ; FixK A ~ , Azorhicobium caulinodans : FixK Bj , Bradvrhizobium japonicum ; FixK2 Bj , Bradyrhizobium japonicum : FixK Re , Rhizobium etli ; FixK RI , Rhiz ~ obium leguminosarum ; FixK Rm , Rhizobium meliloti : FLP Lc , Lactobacillus casei ; FIpA Ll , Lactococcus lactis ; FlpB LI , Lactococcus lactis ; FNR Aa , Aquifex aeolicus ; FNR BI , Bacillus licheniJ ~ mnis ; FNR Bs , Bacillus subtilis : FNR Ec , Escherichia coli ; FNR Hi , Haemophilus infiuenzae ; FNR St , Sahnonella o'phimurium ; FNR T / / ~ Thiobacillus ferrooxidans ; FNR VJl Vibrio fischeri : FnrA P. ~ , Pseudomonas stutzeri ; FnrL Rs , Rhodobacter sphaeroides ; FnrN Pd , Paracoccus denitH [ ~ icans ; FnrN RI , Rhizobium le , ~ uminosarum : FnrN2 RI , Rhizobium leguminosarum : FnrNC Re , Rhizobium etli ; FnrND Re , Rhizobium etli ; FnrP Pd , Paracoccus denitrificans ; FnrP Ph , Pasteurella haemolvtica ; HlyX Ap , Actinobacillus pleuropneumoniae ; LcrE Rh sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Genes induced by NO treatment Functional Group Blattner ID Aminoacid biosynthesis b3774 Gene name Fold Name [ FNR/Fur regulation ] ilvC 5.1 Acetohydroxy acid isomeroreductase Biosynthesis of cofactors , prosthetic groups and cofactors b0782 moaB 2.4 b0783 moaC 3.0 b0785 moaE 2.4 b1680 sufS 2.2 b1683 sufB 2.2 b1684 sufA 6.2 b2527 hscB 1.9 b2528 iscA 2.6 Molybdopterin biosynthesis , protein B Molybdopterin biosynthesis , protein C Molybdopterin converting factor , subunit 2 Cysteine desulphurase , Fe-S cluster formation/repair protein Putative Fe-S cluster formation/repair protein IscA homologue [ Fur ] Cell envelope Putative fimbral-like protein 5.0 2.6 b0938 b2010 ycbQ dacD D-carboxypeptidase , penicillin binding protein 6b Cellular processes yajR fsr tehA tehB ydeR ydhK ydhP yeaR fimG Putative transport protein , drug resistance translocase family Fosmidomycin resistance-protein Tellurite resistance , component A Tellurite resistance , component B Putative fimbrial-like protein Putative fusaric acid resistance-protein 3.5 3.8 3.0 2.5 2.3 2.3 b0427 b0479 b1429 b1430 b1503 b1645 Putative transport protein , drug resistance translocase family Orf , hypothetical protein Fimbrial morphology 2.9 2.9 8.4 b1657 b1797 b4319 ediary metabolism b0342 Central interm lacA Thiogalactoside acetyltransferase Alkanesulfonate monooxygenase 2.2 4.0 5.5 1.7 6.7 ssuD ordL nemA nanE b0935 b1301 b1650 b3223 Probable FAD-dependent NAD-binding oxidoreductase N-ethylmaleimide reductase Putative N-acetylmannosamine-6-phosphate 2-epimerase DNA metabolism SOS mutagenesis and repair ( 6 ) , [ FNR ] Endonuclease Cho 2.9 2.6 b1184 b1741 umuC cho Energy metabolism Putative ferredoxin , carnitine metabolism Pyruvate dehydrogenase , decarboxylase component 3.7 5.5 3.7 3.1 b0044 b0114 b0115 b0116 fixX aceE aceF lpdA Pyruvate dehydrogenase , dihydrolipoyltransacetylase component Pyruvate dehydrogenase , 2-oxodehydrogenase complex : lipoam dehydrogenase component Phosphoglyceromutase 1 [ Fur , FNR ] ide 3.0 3.0 b0755 b0895 gpmA dmsB Anaerobic dimethyl sulfoxide reductase subunit B [ FNR b1007 b1109 b1630 b2552 b2710 b2711 ycdH ndh 3.7 5.7 Putative NADH : flavin oxidoreductase Respiratory type-II NADH dehydrogenase [ FNR ] Subunit of Na + - translocating NADH-ubiquinone oxidoreductase Flavohemoglobin [ FNR/Fur ] Flavorubredoxin NO reductase [ FNR/Fur ] NADH : flavorubredoxin oxidoreductase rnfD hmpA ygaK/norV ygbD / norW ygcQ 2.2 13.8 9.9 4.5 b2769 2.6 Putative electron transfer flavoprotein subunit Hypothetical proteins ykgI ylbE ybeD yceP ydbD yncH bdm ydiV b0303 b0519 b0631 b1060 b1407 5.3 4.2 Orf , hypothetical protein Orf , hypothetical protein Orf , hypothetical protein Orf , hypothetical protein Orf , hypothetical protein Putative transferase 3.1 2.4 7.6 16.1 b1455 b1481 b1707 2.4 2.0 Putative biofilm-dependent modulation protein Orf , hypothetical protein , putative diguanylate Cyclase/phosphodiesterase domain Orf , hypothetical protein Orf , hypothetical protein Putative protein of the SWIM zinc finger family ydjO yedN yehQ yehR yjcF yjeP ytfE b1730 b1933 b2122 b2123 b4066 b4159 2.4 2.6 3.3 Orf , hypothetical protein Orf , hypothetical protein 3.3 3.9 Putative periplasmic binding protein Orf , hypothetical protein 4.4 54.5 b4209 Protein fate b0718 b2070 b3328 ybgQ ygeI hofG 2.7 3.1 2.2 Putative outer membrane protein , fimbrial export usher family Putative chaperonine Putative general protein secretion protein G Protein synthesis Suppression of transcription of USHA b0481 b4171 b4371 ybaK miaA rsmC 9.6 2.0 6.5 Delta ( 2 ) - isopentenylpyrophosphate tRNA-adenosine transferase Putative ribosomal RNA small subunit methyltransferase C Purines , pyrimidines , nucleosides and nucleotides b0425 panE b2673 nrdH 2-dehydropantoate reductase 4.7 4.8 Putative component of ribonucleotide reductase 2 ; glutaredoxin-like protein [ Fur ] Putative component of ribonucleotide reductase Ribonucleotide reductase , alpha subunit Ribonucleotide reductase , beta chai 4.4 2.8 1.9 b2674 b2675 b2676 nrdI nrdE nrdF Regulatory functions glnK sdiA b0450 b1916 b2117 b2420 b2531 b3711 4.8 2.7 2.5 3.4 3.9 5.6 Nitrogen regulatory protein P-II 2 Transcriptional regulator of ftsQAZ gene cluster Molybdate metabolism regulator , third fragment Putative molybdate metabolism regulator Transcriptional regulator of the isc operon Putative LYSR-type HTH transcriptional-regulator yfeS iscR yidZ soxS adiY b4062 b4116 16.0 Transcriptional activator of superoxide response regulon 3.8 Putative ARAC-type regulatory protein Transport and binding proteins b0451 b1186 b1472 b1592 b1801 amtB nhaB 2.2 2.4 3.7 1.6 3.2 4.4 Ammonium transporter Na + / H + antiporter , pH-independent yddL clcB yeaV mntH kup/trkD yjbB phnC Putative outer membrane porin protein Putative chloride channel Putative betaine/carnitine/choline transporter Manganese transport protein , Nramp1 - like protein b2392 b3747 b4020 b4106 Low affinity potassium transport system Putative Na + / Pi cotransporter ATP-binding component of phosphonate transport 2.4 3.3 5.0 Viral functions rzpR ydfU yfjZ pinH yjgW IS30 transposase 2.7 6.4 3.1 2.4 2.5 7.4 b0256 b1362 b1560 b2645 b2648 b4274 Putative Rac prophage endopeptidase Orf , hypothetical protein Orf , hypothetical protein Putative DNA-invertase from prophage CP4-44 KLDE2-phage like element 4-acetoamido-4 ,6 - dideoxy-D-galactose transferas 2.2 b4404 b2750 b2751 b2762 b2764 b3136 ATP-sulphurylase , subunit 1 -2.8 -3.4 -2.7 -2.7 3 ' - phosphoadenosine 5 ' - phosphosulfate reductase Sulfite reductase [ NADPH ] flavoprotein alpha component Putative tagatose-6-phosphate aldose/ketose isomerase DNA metabolism b3066 b3261 b3832 dnaG fis rmuC DNA primase -3.4 -2.3 -2.6 Fis transcriptional-dual-regulator DNA recombination protein rmuC [ FNR ] etabolism Energy m acnB citD narG cybB rnfA ynjF pykA nuoE ygbE ansB nifE glpG yibO atpC Aconitase Citrate lyase acyl carrier protein , gamma chain b0118 b0617 b1224 b1418 b1627 -2.4 -3.7 -3.2 -3.8 -4.0 -3.8 Ñitrate reductase 1 , alpha subunit [ Fur ] Cytochrome b561 Putative Na + - translocating NADH-ubiquinone oxidoreductase subunit Putative cytochrome oxidase REFERENCES 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>Mutationalchanges , with respect to the parentalsequence , are double underlined.I I I Wild-typeCRP bindingsite ; ~ m , JH , Derivative of CC-CC ansB Ps containingtwo symmetry-relatedpoint mutationsthat change each CRP half site ; ~ FNR bindingsite ; Derivativeof FF-FF anaBPecontainingtwo symmetry-relatedpoint mutationsthat change each FNR half site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The question arises as to the involvement of the different activating regions in CRP and FNR at promoters in which both CRP and FNR function in tandem in various combinations , such as in the ansB promoters of Salmonella and Escherichia coil ( Jennings and Beacham , 1993 ; Jennings et aL , 1993 ; see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_9">
Scott , S. , Busby , S. , and Beacham , I. ( 1995 ) Transcriptional coactivation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></label>
<input id="act_similarity_9", type="checkbox">
<div class="ACT_similar">
<li>[ 20 ] Scott , S. , Busby , S. and Beacham , I. ( 1995 ) Transcriptional co-acti-vation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>[ 149 ] Scott , S. , Busby , S. and Beacham , I. ( 1995 ) Transcriptional co-acti-vation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>Scott , S. , Busby , S. , and Beacham , I. ( 1995 ) Transcriptional co-activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Transcriptional co-activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10715008" target="_blank">10715008</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16166533" target="_blank">16166533</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601201" target="_blank">10601201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li></div>
<li>Two dimers of FNR bound in tandem can activate a mutant derivative of ansB Pe in a co-dependent manner A mutant derivative of ansB Pe in which the upstream site had been changed from a CRP to an FNR site , i.e. FF-FF , was functional ( Table 3 ) and , as expected , CRP-independent in contrast to the native promoter , CC-FF ( Jennings and Beacham , 1993 ; see Fig. 1B and Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>We conclude that , at the ansB promoters , AR1 is required when CRP binds upstream ( around -- 91 ) , whether functioning together with downstream FNR in a co-dependent manner , or with CRP downstream in a synergistic manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Consistent with other studies ( Busby et aL , 1994 ; 1995 ) , AR1 in CRP is required at both upstream and downstream binding sites in all ` versions ' of the ansB promoters with CRP sites ; the CC-FF and FF-CC promoters specifically demonstrate a requirement upstream and downstream , respectively , when binding in tandem with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Ferredoxin ( ﬂavodoxin ) : NADP oxidoreductase ( FNR ) was discovered in þ E. coli as an NADPH-dependent reductant of Fld1 that facilitated generation of active methionine synthase in vitro ; FNR and Fld1 will also supply electrons for the reductive cleavage of AdoMet essential for generating protein or substrate radicals in pyruvate formate-lyase , class III ribonucleotide reductase , biotin synthase , and , potentially , lipoyl synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Spectral changes are normalized to facilitate Discussion In E. coli , FNR plays important roles in enzyme activation , providing electrons from the cellular NADPH pool for reductive methylation of cobalamin in methionine synthase [ 1 ] , for glycyl radical generation in class III ribonucleotide reductase [ 18 ] , and pyruvate formate-lyase [ 16 ] , and for substrate radical generation in biotin synthase [ 4 ] and probably the homologous enzyme lipoyl synthase [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>174 , No. 11 Anaerobic Induction of Pyruvate Formate-Lyase Gene E Is Mediated by the ArcA and FNR Proteins xpression GARY SAWERS * AND BERNHARD SUPPMANN Lehrstuhlftir Mikrobiologie der Universitat Munchen , Maria-Ward-Strasse la , 8000 Munich 19 , Gernany Received 16 December 1991/Accepted 26 March 1992 The pyruvate formate-lyase ( pfl ) gene of Escherichia coli is transcribed from seven promoters which are coordinately induced 12 - to 15-fold by anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>However , it is further repressed and inhibited during fermentative growth and fumarate respiration , when the conversion of pyruvate to acelyl-CoA increasingly depends on pyruvate formate-lyase , the pyruvate-inducible product of the FNR - and ArcA-dependent pfl gene ( Sawers and Suppmann , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>Cruz-Ramos , H. , Hughes , M.N. and Poole , R.K. ( 2000 ) Nitric oxide is an effector in bacterial gene expression mediated by FNR and FNR-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Sawers , G , , and Bock , A. ( 1989 ) Novel transcriptional control of the pyruvate formate-lyase gene : upstream regulatoty sequences and multiple promoters regulate anaerobic expression , J Bacterioi 171 : 2485-2498 , Sawers , G , , and Suppmann , B. ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by lhe ArcA and FNR proteins , J Bacterio !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_23">
Anaerobic induction of pyruvate for-mate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12486057" target="_blank">12486057</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665524" target="_blank">7665524</a>]<br></label>
<input id="act_similarity_23", type="checkbox">
<div class="ACT_similar">
<li>Anaerobic induction of pyruvate for-mate-lyase gene expression is mediated by ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Anaerobic induction of pyruvate-for-mate lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7773398" target="_blank">7773398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li></div>
<li>Mol Microbiol 10:737 -- 747 Sawers G , Suppmann B ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_28">
Savtfers , G. , and Suppmann , B. ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></label>
<input id="act_similarity_28", type="checkbox">
<div class="ACT_similar">
<li>Sawers , G , , and Suppmann , B , ( 1992 ) Anaerobic induction of pyruvate formale-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>Sawers , G. , and Suppmann , B. ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022271" target="_blank">8022271</a>]<br></li>
<li>Sawers G , Suppmann B ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_32">
A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K-12 adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></label>
<input id="act_similarity_32", type="checkbox">
<div class="ACT_similar">
<li>A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19767843" target="_blank">19767843</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li></div>
<li>Constantindou , C. , Hobman , J.L. , Grifﬁths , L. , Patel , M.D. , Penn , C.W. , Cole , J.A. & Overton , T.W. ( 2006 ) A reas-sessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escheri ¬ chia coli K12 adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>References Constantinidou C , Hobman JL , Grifﬁths L , Patel MD , Penn CW , Cole JA & Overton TW ( 2006 ) A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_36">
Constantinidou , C. , Hobman , J.L. , Griffiths , L. , Patel , M.D. , Penn , C.W. , Cole , J.A. , and Overton , T.W. ( 2006 ) A reas-sessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></label>
<input id="act_similarity_36", type="checkbox">
<div class="ACT_similar">
<li>[ 2 ] C. Constantinidou , J.L. Hobman , L. Grifﬁths , M.D. Patel , C.W. Penn , J.A. Cole , T.W. Overton , A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic-growth , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>2 , 57 -- 65 2 Constantinidou , C. , Hobman , J. L. , Grifﬁths , L. , Patel , M. D. , Penn , C. W. , Cole , J. A. and Overton , T. W. ( 2006 ) A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Constantinidou , C. , Hobman , J.L. , Griffiths , L. , Patel , M.D. , Penn , C.W. , Cole , J.A. and Overton , T.W. ( 2006 ) A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li></div>
<li>Global Gene Expression Profiling in Escherichia coli K12 THE EFFECTS OF OXYGEN AVAILABILITY AND FNR * □ S Received for publication , December 20 , 2002 , and in revised form , May 15 , 2003 Published , JBC Papers in Press , May 15 , 2003 , DOI 10.1074 / jbc.M213060200 Kirsty Salmon ‡ § , She-pin Hung § ¶ , Kathy Mekjian ‡ , Pierre Baldi ** ‡ ‡ , G. Wesley Hatfield ¶ § § , and Robert P. Gunsalus ‡ ¶ ¶ From the ‡ Department of Microbiology , Immunology , and Molecular Genetics , and the Molecular Biology Institute , UCLA , Los Angeles , California 90095-1489 , the ¶ Department of Microbiology and Molecular Genetics , the ** School of Information and Computer Science , the ‡ ‡ Department of Biological Chemistry , and the Institute for Genomics and Bioinformatics , University of California , Irvine , California 92697 The work presented here is a first step toward a long term goal of systems biology , the complete elucidation of the gene regulatory networks of a living organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Printed in the U.S.A. mic Reassessment of the FNR Regulon and Transcripto Analysis of the Effects of Nitrate , Nitrite , NarXL , and NarQP as Escherichia coli K12 Adapts from Aerobic to Anaerobic Growth * □ S Received for publication , November 16 , 2005 , and in revised form , December 20 , 2005 Published , JBC Papers in Press , December 22 , 2005 , DOI 10.1074 / jbc.M512312200 Chrystala Constantinidou , Jon L. Hobman , Lesley Griffiths , Mala D. Patel , Charles W. Penn , Jeffrey A. Cole , and Tim W. Overton1 From the School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom A which the complete genome sequence is available , and an isogenic fnr mutant aerobically and anaerobically in a minimal salts medium , and compared their transcriptome data with those from the previous study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>A 7000 6000 2000 1500 1000 0 pSch2 - pSch2 - pSch3 - pSch5 - pSch7 - pSch7-ybhK moaA dmsA ynfE napF eco 1000 ββ-Galactosidase activity ( Miller unit 500 0 pSch5 - pSch6 - pSch9-anti - anti- anti-ynfD cutC yghX B-300 -200 -100 +1 +100 +200 +300 ModE-activating promoter ybhKp moaAp1 moaAp2 ModE FNR moaAp2 ybhKp FNR ybhK moaAp1 moaA ModE dmsA2 dmsAp1 dmsA serS FNR ModE ynfEp1 ynfE FNR ModE napFp2 napFp1 napF napFp3 yojO ecop ModE F napFp3 NR napFp1 napFp2 ecop eco yojO ModE FNR ModE-repressing promoter modAp modA ModE morAp ybhT yghX yghW ModE FIG 2 Promoters activated and repressed by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Clusterb Strain '' Protein ( metal ion ) ( 16 ) - C-X2-H - C-X2 - C-Xs - C - ( 17 ) - C-X2 - C-X3 - It-X3 - C - ( 86 ) - H-H-H-X - I-I-Xz - C-Xz - C - ( 133 ) - C-X4 - C-X4 - H-X - tt - ( 29 ) - C-X3 - C-X2 - C-H-X7 - I -- I - ( ll ) - C-X2 - C-Xl3 - C-X2 - C - ( 6 ) - C-X2 - C-X29-C-X2 - C - ( 440 ) - C-Xu - C-X19-C-X4 - C - ( 184 ) - C-X2 - C ¬ FNR Orf240 coli leg coli ... E R E Fur ( Fe2 + ) Activase ( Fe2 + ) Gal4 ( Zn2 + ) E. coli S. cer Rubredoxin ( Fe2 + / Fe 3 + ) NifA NifL C. past B. jap K. pneum `` R. leg = Rhizobium leguminosarum ; S. cer = Saccharomyces cerevisiae ; C. past = Clostridium pasteurianum ; B. jap = Bradyrhizobium japonicum ; K. pneum = Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>21 No. 13 pp. 3235 ± 3244 , 2002 NO sensing by FNR : regulation of the Escherichia coli NO-detoxifying ¯ avohaemoglobin , Hmp Hugo Cruz-Ramos , Jason Crack1 , Guanghui Wu , Martin N.Hughes , 2 Colin Scott , Andrew J.Thomson , 1 Jeffrey Green and Robert K.Poole3 The Krebs Institute for Biomolecular Research , Department of Molecular Biology and Biotechnology , The University of Shef ® eld , Shef ® eld S10 2TN , Centre for Metalloprotein Spectroscopy and 1 Biology , School of Chemical Sciences , University of East Anglia , Norwich NR4 7TJ and Chemistry Department , King 's College 2 London , Strand , London WC2R 2LS , UK 3 Corresponding author e-mail : r.poole@shef®eld.ac.uk Nitric oxide ( NO ) is a signalling and defence molecule of major importance in biology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_45">
[ 23 ] Cruz-Ramos , H. , Crack , J. , Wu , G. , Hughes , M.N. , Scott , C. , Thomson , A.J. , Green , J. and Poole , R.K. ( 2002 ) NO sensing by FNR : regulation of the Escherichia coli NO-detoxifying # avohaemoglobin , Hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></label>
<input id="act_similarity_45", type="checkbox">
<div class="ACT_similar">
<li>277 , 8166 -- 8171 5 Cruz-Ramos , H. , Crack , J. , Wu , G. , Hughes , M.N. , Scott , C. , Thomson , A.J. , Green , J. and Poole , R.K. ( 2002 ) NO sensing by FNR : regulation of the Escherichia coli NO-detoxifying ﬂavohaemoglobin , Hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21265775" target="_blank">21265775</a>]<br></li>
<li>[ 169 ] Cruz-Ramos , H. , Crack , J. , Wu , G. , Hughes , M.N. , Scott , C. , Thomson , A.J. , Green , J. and Poole , R.K. ( 2002 ) NO sensing by FNR : regulation of the Escherichia coli NO-detoxifying # avohae-moglobin , Hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>Cruz-Ramos , H. , Crack , J. , Wu , G. , Hughes , M. N. , Scott , C. , Thomson , A. J. , Green , J. , and Poole , R. K. ( 2002 ) NO sensing by FNR : regulation of the Escherichia coli NO-detoxifying flavohaemoglobin , Hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25193663" target="_blank">25193663</a>]<br></li></div>
<li>[ 5 ] D.M. Bates , C.V. Popescu , N. Khoroshilova , K. Vogt , H. Beinert , E. Munck , P.J. Kiley , Substitution of leucine 28 with histidine in the Escherichia coli transcription factor FNR results in increased stability of the [ 4Fe -- 4S ] ( 2 + ) cluster to oxygen , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_50">
Substitution of leucine 28 with histidine in the Esch-erichia coli transcription factor FNR results in increased stability of the 2 [ 4Fe-4S ] cluster to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></label>
<input id="act_similarity_50", type="checkbox">
<div class="ACT_similar">
<li>Substitution of leucine 28 with histidine in the Escherichia coli transcription factor FNR results in increased stability of the [ 4Fe-4S ] 2 + cluster to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Substitution of leucine 28 with histidine in Escherichia coli transcription factor FNR results in increased stability of the [ 4Fe -- 4S ] cluster to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>]<br></li>
<li>Bates DM , et al. ( 2000 ) Substitution of leucine 28 with histidine in the Escherichia coli transcription factor FNR results in increased stability of the [ 4Fe-4S ] 2 cluster to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Bates , D.M. , Popescu , C.V. , Khoroshilova , N. et al. ( 2000 ) Substitution of leucine 28 with histidine in the Escherichia coli transcription factor FNR results in increased stability of the [ 4Fe4S ] 2 + cluster to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li></div>
<li>Biochemistry 2004 , 43 , 791-798 791 Superoxide Destroys the [ 2Fe-2S ] 2 + Cluster of FNR from Escherichia coli † Victoria R. Sutton , ‡ ,  Audria Stubna , § Thomas Patschkowski ,  Eckard Münck , § Helmut Beinert , ⊥ and Patricia J. Kiley * ,  Program in Cellular & Molecular Biology , UniVersity of Wisconsin , and Department of Biomolecular Chemistry , Medical School , UniVersity of Wisconsin , 1300 UniVersity AVenue , Madison , Wisconsin 53706 , Department of Chemistry , Carnegie Mellon UniVersity , 4400 Fifth AVenue , Pittsburgh , PennsylVania 15213 , and Institute for Enzyme Research and Department of Biochemistry , College of Agriculture and Life Sciences , UniVersity of Wisconsin , Madison , Wisconsin 53705 ReceiVed September 21 , 2003 ; ReVised Manuscript ReceiVed NoVember 12 , 2003 ABSTRACT : The oxygen sensing ability of the transcription factor FNR depends on the presence of a [ 4Fe-4S ] 2 + cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Iron-sulfur cluster disassembly in the FNR protein of Escherichia coli by O2 : [ 4Fe-4S ] to [ 2Fe-2S ] conversion with loss of biological activity ABSTRACT The transcription factor FNR ( fumarate nitrate reduction ) requires the presence of an iron-sulfur ( Fe-S ) cluster for its function as a global transcription regulator in Escherichia coli when oxygen becomes scarce .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_57">
Khoroshilova , N. , Popescu , C , MUnck , E. , Beinert , H. , and Kiley , P.J. ( 1997 ) Iron-sulfur cluster disassembly in the FNR protein of Escherichia coli by O , : [ 4Fe-4S ] to [ 2Fe-2S ] conversion with loss of biological activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10544286" target="_blank">10544286</a>]<br></label>
<input id="act_similarity_57", type="checkbox">
<div class="ACT_similar">
<li>[ 165 ] Khoroshilova , N. , Popescu , C. , Mu `` nck , E. , Beinert , H. and Kiley , P.J. ( 1997 ) Iron-sulfur cluster disassembly in the FNR protein of Escherichia coli by O2 : [ 4Fe-4S ] to [ 2Fe-2S ] conversion with loss of biological activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>[ 24 ] Khoroshilova , N. , Popescu , C. , Muënck , E. , Beinert , H. and Kiley , P. ( 1997 ) Iron-sulfur cluster disassembly in the FNR protein of Escherichia coli by O2 : [ 4Fe-4S ] to [ 2Fe-2S ] conversion with loss of biological activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li></div>
<li>Lazazzera , B.A. , Beinert , H. , Khoroshilova , N. et al. ( 1996 ) DNA-bindingand dimerization of the Fe-S containing FNR protein Escherichia eoli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_61">
DNA binding and dimerization of the Fe -- Scontaining FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></label>
<input id="act_similarity_61", type="checkbox">
<div class="ACT_similar">
<li>DNA binding and dimerization of the Fe-S-containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>DNA-binding and dimerization of the Fe-S containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>DNA binding and dimerization of the Fe-S containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Lazzazera , B.A. , Beinert , H. , Khoroshilova , N. , Kennedy , C. and Kiley , P.J. ( 1996 ) DNA binding and dimerization of the Fe ± S-containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Lazazzera BA , Beinert H , Khoroshilova N , Kennedy MC , Kiley PJ ( 1996 ) DNA binding and dimerization of the Fe-S-containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>[ 16 ] Lazazzera , B.A. , Beinert , H. , Khoroshilova , N. , Kennedy , M.C. and Kiley , P.J. ( 1996 ) DNA binding and dimerization of the Fe-S containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Lazazzera , B.A. , Beinert , H. , Khoroshilova , N. , Kennedy , M.C. , and Kiley , P.J. ( 1996 ) DNA binding and dimerization of the Fe-S containing FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Lazazzera , B.A. , Beinert , H. , Khoroshilova , N. , Kennedy , M.C. , and Kiley , P.J. ( 1996 ) DNA-binding and dimerization of the Fe-S containing FNR protein Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>[ 164 ] Lazazzera , B.A. , Beinert , H. , Khoroshilova , N. , Kennedy , M.C. and Kiley , P.J. ( 1996 ) DNA binding and dimerization of the Fe-S-con-taining FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>Lazazzara , B.A. , Beinert , H. , Khoroshilova , N. , Kennedy , M.C. and Kiley , P.J. ( 1996 ) DNA binding and dimerization of the Fe ± Scontaining FNR protein from Escherichia coli are regulated by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li></div>
<li>, two Synechocystis species ; NtcA Sy3 , Svnechococcus species ; NtcA Sy7 , Synechococcus species ; PrfA Li , Listeria ivanovii ; PflA Lm , Listeria mono ~ wtogenes ; PtrA Sy , ~ vnechococcus species ; Vfr Pa , Pseudomonas aeruk , inosa ; Ycf Pp , Porphyra purpurea : YeiL Ec , Escherichia coli ( 3 ) have a predicted DNA-binding specificity corresponding to that of FNR rather than CRP ; ( 4 ) fail to respond to cAMP ; ( 5 ) respond to oxygen limitation ; and ( 6 ) retain all or most of the characteristic ` cysteine signature ' of FNR , which comprises three essential N-terminal cysteine residues and a fourth essential cysteine within the [ 3-roll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A protein which is related to both FNR and CRP may be assigned to the FNR family on the basis of : - an ability to complement anfnr mutation in E. coli , or its equivalent in another organism ; - a greater degree of sequence identity to FNR than to CRP ; - h a v i n g the predicted DNA-binding specificity of FNR rather than that of CRP ; - failing to respond to cAMP , or lacking amino acids which are involved in liganding cAMP ; - responding to the same physiological signal as FNR , namely oxygen limitation or - retaining some or all of the cysteine residues of FNR which are thought to be involved in its response to oxygen limitation , and which are absent from CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>J Mol Biol 166:241 -- 247 Six S , Trageser M , Kojro E , Fahrenholz F , Unden G ( 1995 ) Reactivity of the N-terminal cysteine residues in active and inactive forms of FNR , an O2-responsive , Fe-containing transcriptional-regulator of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_75">
Engel , P. , Trageser , M. , and Unden , G , ( 1991 ) Reversible interconversion of the functional state of the gene regulator FNR from Eschehchia coli in vivo by O2 and iron availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></label>
<input id="act_similarity_75", type="checkbox">
<div class="ACT_similar">
<li>[ 35 ] Engel , P. , Trageser , M. and Unden , G. ( 1991 ) Reversible in ¬ terconversion of the functional state of the gene regulator FNR from Escherichia coli in vivo by O2 and iron availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li></div>
<li>The ability of Escherichia coli to efﬁciently sense and respond to O2 is primarily controlled by the global regulatory protein FNR ( 22 , 42 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Control of FNR Function of Escherichia coli by O and Reducing Conditions 2 Abstract G. Unden , * S. Achebach , G. Holighaus , H.-Q .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>[ 10 ] Unden , G. , Achebach , S. , Holighaus , G. , Tran , H.G. , Wackwitz , B. and Zeuner , Y. ( 2002 ) Control of FNR function of Escherichia coli by O2 and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Control of FNR function of Escherichia coli by O2 and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Effect of oxygen on the Escherichia coli ArcA and FNR regulation systems and metabolic responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20707605" target="_blank">20707605</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20707865" target="_blank">20707865</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16291649" target="_blank">16291649</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_82">
Shalel-Levanon S , San KY , Bennett GN : Effect of oxygen on the Escherichia coli ArcA and FNR regulation systems and metabolic responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></label>
<input id="act_similarity_82", type="checkbox">
<div class="ACT_similar">
<li>J Bacteriol 186:7186 -- 7195 Levanon S , San KY , Bennett GN ( 2005 ) Effect of oxygen on the Escherichia coli ArcA and FNR regulation systems and metabolic responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18175070" target="_blank">18175070</a>]<br></li></div>
<li>The requirement of ArcA and FNR for peak expression of the cyd operon in Escherichia coli under microaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>J Bacteriol 189 : 1150 -- 1153 Shalel-Levanon S , San KY , Bennett GN ( 2005 ) Effect of ArcA and FNR on the expression of genes related to the oxygen regulation and the glycolysis pathway in Escherichia coli under microaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20683441" target="_blank">20683441</a>]<br></li>
<li>Effect of ArcA and FNR on the expression of genes related to the oxygen regulation and the glycolysis pathway in Escherichia coli under microaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752906" target="_blank">22752906</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23563322" target="_blank">23563322</a>]<br></li>
<li>Introduction FNR is a transcriptional-regulator which controls the expression of target genes in response to anoxia in Escherichia coti ( Spiro and Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Summary FNR Is a transcriptional-regulator that controls gene expression in response to oxygen limitation in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Despite the vast abundance of genes that are regulated by the FNR and ArcAB systems , it appears that they do not represent the sole means by which gene expression in E. coli is regulated in response to oxygen levels ( Sawers , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_90">
Spiro , S. , and Guest , J. ( 1990 ) FNR and its role in oxygen regulated gene expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></label>
<input id="act_similarity_90", type="checkbox">
<div class="ACT_similar">
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygen regulated gene expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li></div>
<li>However , NO sensing by E.coli FNR has special physiological signi ® cance since FNR regulates expression of Hmp , which protects against NO and nitrosative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>This unusual use of FNR and OxyR to activate expression suggests that yhjA plays a speciﬁc role in countering oxidative-stress in E. coli , possibly in particular micro-aerobic environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>35 , No. 1 269 -- 278 doi :10.1093 / nar/gkl1023 Transcription factor distribution in Escherichia coli : studies with FNR protein David C. Grainger * , Hirofumi Aiba , Douglas Hurd , Douglas F. Browning and 1 2 Stephen J. W. Busby School of Biosciences , University of Birmingham , Edgbaston , Birmingham B15 2TT , UK , Laboratory of 1 Molecular Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464-8601 , Japan and 2 Oxford Gene Technology , Begbroke Science Park , Sandy Lane , Yarnton , Oxford OX5 1PF , UK Received August 25 , 2006 ; Revised November 8 , 2006 ; Accepted November 14 , 2006 ABSTRACT Using chromatin immunoprecipitation ( ChIP ) and high-density microarrays , we have measured the distribution of the global transcription regulator protein , FNR , across the entire Escherichia coli chromosome in exponentially growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>y Binding n-Regulation of the Escherichia coli K-12 nrf Promoter b of the NsrR Nitric Oxide-Sensing Transcription Repressor to an Upstream Site Douglas F. Browning ,1 * David J. Lee ,1 Stephen Spiro ,2 and Stephen J. W. Busby1 School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom ,1 and Department of Molecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 750802 ow D Received 1 March 2010/Accepted 9 May 2010 FNR-dependent activation of the Escherichia coli K-12 nrf promoter is downregulated by the nitric oxide-sensitive NsrR protein together with the nucleoid-associated protein IHF , which bind to overlapping targets adjacent to the DNA site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The nrf Escherichia coli K-12 NarL and NarP Proteins Insulate the Promoter from the Effects of Integration Host Factor Douglas F. Browning ,1 * David J. Lee ,1 Alan J. Wolfe ,2 Jeffrey A. Cole ,1 and Stephen J. W. Busby1 School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom ,1 and Department of Microbiology and Immunology , Stritch School of Medicine , Loyola University Chicago , Maywood , Illinois 601532 Received 5 July 2006/Accepted 16 August 2006 The Escherichia coli K-12 nrf operon promoter can be activated fully by the FNR protein ( regulator of fumarate and nitrate reduction ) binding to a site centered at position 41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Regulation by Nucleoid-Associated Proteins at the Escherichia coli nir Operon Promoter Douglas F. Browning , * Jeffrey A. Cole , and Stephen J. W. Busby School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom Received 23 July 2008/Accepted 22 August 2008 The Escherichia coli K-12 nir operon promoter can be fully activated by binding of the regulator of fumarate and nitrate reduction ( FNR ) to a site centered at position 41.5 upstream of the transcript start , and this activation is modulated by upstream binding of the integration host factor ( IHF ) and Fis ( factor for inversion stimulation ) proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Popescu , C.V. , Bates , D.M. , Beinert , H. , Munck , E. and Kiley , P.J. ( 1998 ) Mossbauer spectroscopy as a tool for the study of activation / inactivation of the transcription regulator FNR in whole cells of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_99">
Williams SM , Wing HJ , Busby SJW ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></label>
<input id="act_similarity_99", type="checkbox">
<div class="ACT_similar">
<li>Williams , S.M. , Wing , H.J. , and Busby , S.J. ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li>
<li>Williams , S.M. , Wing , H.J. and Busby , S.J.W. ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Williams , S.M. , Wing , H.J. , and Busby , S.J.W. ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_103">
Takahashi , K. , Hattori , T. , Nakanishi , T. , Nohno , T. , Fujita , N. , Ishihama , A. , et al. ( 1994 ) Repression of in vitro transcription of the Escherichia coli fnr and narX genes by FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></label>
<input id="act_similarity_103", type="checkbox">
<div class="ACT_similar">
<li>Takahashi , K. , Hattori , T. , Nakianishi , T. , Nohno , T. , Fujita , N. , Ishihama , A. , and Taniguchi , S. ( 1994 ) Repressionof in vitro transcription of the Escherichia coli fnr and narX genes by FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Takahashi , K. , Hattori , T. , Nakanishi , T. , Nohno , T. , Fujita , N , Ishihama , A. , and Taniguchi , S. ( 1994 ) Repression of in vitro transcription of the Escherichia coil fnr and narX genes by FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_106">
Li , B. , Wing , H. , Lee , D. et al. ( 1998 ) Transcription activation by Escherichia coli FNR protein : similarities to and differences from , the CRP paradigm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></label>
<input id="act_similarity_106", type="checkbox">
<div class="ACT_similar">
<li>Li , B. , Wing , H. , Lee , D. , Wu , H. , and Busby , S. ( 1998 ) Transcription activation by Escherichia coli FNR protein : similarities to , and differences from , the CRP paradigm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li></div>
<li>Transcription activation by Escherichia coli FNR protein : similarities to , and differences from , the CRP paradigm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Williams , R , , Bell , A. , Sims , G. , and Busby , S. ( 1991 ) The role of two surface exposed loops in transcriptional activation by the Escherichia cofi CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>6:1447 -1456 WilliamsR , Bell A , Sims G & Busby S ( 1991 ) The role oftwo surface exposed loops in transcription activation by the Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_111">
Williams , R. , Bell , A. , Sima , G. and Busby , S. ( 1991 ) The role of two surface exposed loops in transcription activation by the Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11574471" target="_blank">11574471</a>]<br></label>
<input id="act_similarity_111", type="checkbox">
<div class="ACT_similar">
<li>Williams , R. , Bell , A. , Sims , G. and Busby , S. ( 1991 ) The role of two surface exposed loops in transcription activation by the Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Williams , R. , Bell , A. , Sims , G. , and Busby , S. ( 1991 ) The role of two surface exposed loops in transcription activation by Escherichia coli CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li></div>
<li>Re-assessment of the FNR regulon of E. coli Transcription of many operons regulated by FNR are co-dependent on other transcription factors [ 5,6 ] that respond to changes in carbon source , growth rate or nutrient availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_115">
The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligo-meric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550494" target="_blank">8550494</a>]<br></label>
<input id="act_similarity_115", type="checkbox">
<div class="ACT_similar">
<li>The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The activity of the Escherichia coli transcription factor FNR is regulated by a change in the oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li></div>
<li>Lazazzera , B.A. , Bates , D.M. , and Kiley , P.J. ( 1993 ) The activity of the Escherichia coil transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_119">
J Gen Microbiol 97:145 -- 160 Lazazzera BA , Bates DM , Kiley PJ ( 1993 ) The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="act_similarity_119", type="checkbox">
<div class="ACT_similar">
<li>Lazazzera BA , Bates DM , Kiley PJ ( 1993 ) The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>, and Kiley , P.J ( 1993 ) The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>[ 8 ] B.A. Lazazzera , D.M. Bates , P.J. Kiley , The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state , Genes Dev .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>[ 20 ] Lazazzera , B.A. , Bates , D.M. and Kiley , P.J. ( 1993 ) The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Lazazzera , B.A. , Bates , D. and Kiley , P.J. ( 1993 ) The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Lazazzera , B.A. , Bates , D.M. , and Kiley , P.J. ( 1993 ) The activity of the Escherichia coli transcription factor FNR is regulated by a change in oligomeric state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li></div>
<li>TABLE 2 Previously identified FNR-regulated operons Ratiosa Operon Product of operon O O Ø NO NO NO NO 2 2 2 3 3 2 2 WT fnr fnr WT fnr WT fnr A. Previously reported FNR-activated operons that appear to be activated by FNR in these experiments caiF Transcriptional regulator of 0.19 0.16 0.25 0.21 0.16 0.51 0.14 caiTABCDE operon dcuC Dicarboxylate transporter 0.32 0.47 0.40 0.42 0.29 0.72 0.18 Position of FNR siteb Evidence of FNR regulation Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Gene characteristics Gene Predicted Band no.a Gene function ( reference ) Regulation of corresponding gene ( reference ) productb molecular mass ( Da ) c PﬂB * 85,357 GadA * 52,685 1 Pyruvate formate lyase I ( 56 ) Glutamate decarboxylase ( 64 ) Induced by ArcA and FNR ( 61 ) 2 Positively regulated by rpoS , CRP , gadE , gadW , and gadX ; negatively regulated by H-NS ( 21 , 66 ) Positively activated by rpoS and by CRP ( 33 , 74 ) Positively activated by CRP ( 74 ) ; repressed by ArcA ( 60 ) Same transcription unit as gatZ Positively regulated by rpoS ( 33 , 46 , 71 ) Positively regulated by rpoS ( 2 , 33 , 46 , 71 ) 52,773 TnaA † 3 Tryptophan deaminase ( tryptophanase ) ( 22 ) 47,109 GatZ † 3 Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Quinone reductase ; response to oxidative-stress ( 45 ) Bacterial ferritin , DNA protection during stationary phase ( 39 , 73 ) Transcriptional regulator ( 28 ) 30,812 20,846 18,695 GatY † WrbA * Dps * 4 5 6 , 7 , and 11 24,935 CsgD * 8 Positively regulated by rpoS , hns , and ompR ( 25 , 58 ) Same transcription unit as csgD Positively regulated by FNR ( 18 ) Outer membrane protein ; curli transport component ( 38 ) Outer membrane protein ; receptor for colicin S4 ( 49 ) 30,557 22,928 CsgG * OmpW * 9 10 a As in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>FNR-binding sitesc Other reported regulation Y/N Y/Y Y/N CRP-activated , NarLrepressed 46 47 48 Half-sites 63.5 , and 39.5 - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant and promoter truncations - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and EMSAd - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis , EMSA , and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis , and EMSA ArcA - and DcuSR-regulated ModE-repressed 33.5 , 113.5 moaABCDE Molybdopterin biosynthesis 0.50 1.06 0.34 1.52 0.43 0.97 0.59 hcp-hcr dmsABC Hybrid cluster protein and NADPH oxidoreductase Anaerobic dimethyl sulfoxide reductase 73.5 41.5 Y/Y Y/Y NarL - and NarP-regulated ModE-activated , NarLrepressed 0.72 1.12 0.66 30.26 1.60 47.81 2.54 0.12 0.27 0.10 0.10 0.07 0.70 0.07 49 50 -- 52 focA-pflB Formate transporter and pyruvate-formate lyase 0.24 0.30 0.34 0.35 0.22 0.87 0.31 41.5 Y/Y ArcA-activated 53 -- 55 narK Nitrate/nitrite transport protein 0.20 0.25 0.14 14.30 0.08 12.07 0.12 41.5 Y/Y NarL-activated 56 , 57 41.5 Y/Y narGHJI 0.07 0.07 0.11 33.98 0.14 3.91 0.14 Nitrate reductase A NarL-activated 58 , 59 Y/Y fdnGHI 0.33 0.40 0.20 15.19 0.15 2.16 0.13 Formate dehydrogenase N 42.5 and 97.5 NarL-activated 55 , 60 fliZY Putative cell density - 1.08 0.72 0.23 1.34 0.32 0.62 0.08 responsive regulator of 28 napFDAGHBC Periplasmic nitrate reductase 0.51 0.47 0.39 0.95 0.27 4.04 0.26 No predicted ts 64.5 FlhDC-regulated NarL-repressed , N activated Y/Y Y/N 61 30 arP ¬ glpABC yfiD 0.19 0.16 0.16 0.55 0.21 0.71 0.18 0.07 0.02 0.04 0.66 0.06 1.22 0.08 Glycerol-3-phosphate dehydrogenase Putative formate acetyltransferase 41.5 N/N Y/Y GlpR-repressed , CRP-activated PdhR-repressed 62 40.5 and 93.5 63 , 64 65 N/N hypBCDE 0.20 0.15 0.21 0.59 0.22 0.67 0.19 Hydrogenase maturation Half-sites ansB 41.5 Y/Y Periplasmic L-asparaginase II 0.01 0.02 0.01 0.25 0.01 0.63 0.01 CRP-activated 55 , 66 41.5 Y/Y nirBDC 0.74 0.48 0.38 9.99 0.23 8.80 0.38 Nitrite reductase NarL/NarP-activated , IHF - , Fis - , and H-NS-regulated 10 , 67 0.50 0.29 0.19 0.03 0.80 0.63 0.65 4.73 0.40 0.24 0.10 0.03 1.11 0.29 0.33 0.32 0.53 0.31 0.14 0.05 0.20 0.78 0.22 0.05 0.28 1.38 0.11 0.05 Ferrous iron transport Nickel transporter Putative cytochrome c peroxidase Periplasmic nitrite reductase No reported ts 41.5 61.5 41.5 Y/N Y/N Y/Y Y/Y Fur-repressed NikR-repressed feoAB 68 69 nikABCDE yhjA nrfABCDEFG 61 10 , 70 NarP-activated and NarLregulated TABLE 2 -- continued Ratiosa Position of FNR siteb FNR-binding Other reported regulation Product of operon Operon Evidence of FNR regulation Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>, B-Galactosidase specific activity ( 28 ) from a pfl-lacZ fusion was determined for cells grown aerobically ( solid bars ) , anaerobically ( open bars ) , or anaerobically with pyruvate ( stippled bars ) in rich media with the following strains : wt ( wild type ) , parent , RM123 ; himD , IHF - , RM444 ; frzr , FNR - , RM135 ; arcA , ArcA - , RM320 ; fnr himD , FNR - IHF - , RM324 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_130">
Speciﬁc transcriptional requirements for positive regulation of the anaerobically inducible pﬂB operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></label>
<input id="act_similarity_130", type="checkbox">
<div class="ACT_similar">
<li>Speciﬁc transcriptional requirements for positive regulation of the anaerobically inducible pﬂ operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>Specific transcriptional requirements for positive regulation of the anaerobically inducible pfl operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>Specific transcriptional requirements for positive regulation of the anaerobically inducible pfE operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li></div>
<li>Sawers , G. ( 1993 ) Speciﬁc transcriptional requirements for positive regulation of the anaerobically inducible pﬂ operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484901" target="_blank">9484901</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_135">
J Bacteriol 175:7938 -- 7944 Sawers G ( 1993 ) Specific transcriptional requirements for positive regulation of the anaerobically inducible pfl operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="act_similarity_135", type="checkbox">
<div class="ACT_similar">
<li>Sawers , G. ( 1993 ) Specific transcriptional requirements for positive regulation of the anaerobically inducible pfi operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>Sawers , G. ( 1993 ) Specific transcriptional requirements for positive regulation of the anaerobically inducible pfl operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li></div>
<li>Although FNR is a direct O2 sensor , ArcA primarily responds to the redox status of the quinone pool and the accumulation of fermentation products , such as NADH and D-lactate ( 31 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Kinetic modelling indicated that ArcA and FNR behaviour could not explain the ndh transcript profile , leading to the identification of another TF , PdhR , as the source of the asymmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>1982 ; Li and Stewart , 1992 ) or ArcA and FNR are known ( Sawers and Suppmann , 1992 ; Hassan and Sun , 1992 : Cotter and Gunsalus , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>To check against this , and to assess the robustness of our predictions , we repeated our analysis including three further regulators , FNR , ArcA and IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>Evans , M.R. ( 2008 ) Global effects of the transcriptional-regulators ArcA and FNR in anaerobically grown Salmonella enterica sv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19732340" target="_blank">19732340</a>]<br></li>
<li>Lee , H.S. , Lee , Y.S. , Kim , H.S. , Choi , J.Y.H. , Hassan , M. , and Chung , M.H. ( 1998 ) Mechanism of regulation of 8-hydroxyguanine endonuclease by oxidative-stress : roles of FNR , ArcA , and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Mechanism of regulation of 8-hydroxyguanine endonuclease by oxidative-stress : roles of FNR , ArcA , and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>What is the physiological significance of the pfl operon being under the dual control of ArcA and FNR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Expression of the operon is also induced anaerobically , although this regulation does not rely on direct control by either FNR or ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>When FNR activity was assayed over a range of oxygen concentrations in vivo [ 41,42 ] , the amount of oxygen required to inactivate half of the FNR in these cultures was in the range of 1 ^ 10 WM. At this external O2 concentration , Unden and colleagues have estimated that O2 di ¡ usion into cells exceeds the rate of cellular consumption thus providing su $ cient O2 to inactivate FNR which is present at a concentration of approximately 1 WM. Only when the external oxygen concentration drops below 1 WM is the oxygen concentration inside the cells predicted to be lower than the external oxygen concentration thus providing an environment where FNR will be no longer inactivated [ 42,43 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>FNR can be inactivated in vivo and in vitro by iron-depletion , and active and inactive forms can be interconverted reversibly by Fe availability ( Spiro et al. 1989 ; Trageser and Unden 1989 ; Engel et al. 1991 ; Green and Guest 1993a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>1989 ) , from in vivo and in vitro KMnO4 footprinting studies ( Green and Guest , 1993a ) , and from the recent reactivation of iron-depleted FNR by Fe ^ ' and p-mercaptoethanol ( Green and Guest , 1993b ) , Furthermore , in vivo studies have shown that the active ( anaerobic ) and inactive ( aerobic ) forms of FNR are interconvertible depending on the availability of iron and oxygen ( Engel ef ai , 1991 ) and that FNR responds to redox state rather than oxygen per se ( Unden ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>That FNR binds to this proposed FNR-binding site and activates transcription has been demonstrated in vitro ( Green ef al. , 1991 ; Sharrocks ef al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_151">
J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></label>
<input id="act_similarity_151", type="checkbox">
<div class="ACT_similar">
<li>FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li></div>
<li>Sharrocks , A.D. , Green , J. , and Guest , J.R. ( 1991 ) FNR activates and repressestranscription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>[ 16 ] A.D. Sharrocks , J. Green , J.R. Guest , FNR activates and represses transcription in vitro , Proc Biol Sci 245 ( 1991 ) 219 -- 226 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>J Bacteriol 174 : 3474-3478-Sharrocks , A.D , , Green , J , , and Guest , J.R , ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_156">
J eacter/o/176 : 4985-4992 , Sharrocks , A.D. , Green , J , , and Guest , J.R , ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></label>
<input id="act_similarity_156", type="checkbox">
<div class="ACT_similar">
<li>Sharrocks , A.D. , Green , J. , and Guest , J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Sharrocks , A. , Green , J. , and Guest , J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022271" target="_blank">8022271</a>]<br></li>
<li>Sharrocks , A. , Green , J. , and Guest , J. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Sharrocks , A.D. , Green , J. and Guest , J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li></div>
<li>In this study , we analyzed the in vivo and in vitro interactions of these boxes with the FNR transcriptional-regulator , in an attempt to elucidate their roles in nrdDG activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>, Green , J. , and Guest , J.R. ( 1990 ) In vivoand in vitro mutants of FNR , the anaerobic transcriptional-regulator of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Sharrocks , A.D. , Green , J. , and Guest , J.R. ( 1990 ) In wVoand in vitro mutants of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Cell cultures ( 6 L ) for Mössbauer analysis were prepared as described previously ( 11 ) , except that the time for sparging cell cultures with argon after IPTG induction ( to allow for accumulation of 4Fe-FNR ) was increased to 17 h and the initial centrifugation step was decreased to 15 min by using a Beckman JLA 10.500 rotor , run at 8000 rpm in a Beckman Avanti J-25 centrifuge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>FEMS Microbiology Reviews 22 ( 1999 ) 341 ^ 352 Oxygen sensing by the global regulator , FNR : the role of the iron-sulfur cluster Patricia J. Kiley a ; * , Helmut Beinert b a Department of Biomolecular Chemistry , 574 Medical Science Center , 1300 University Avenue , University of Wisconsin Medical School , Madison , WI 53706 , USA b Institute for Enzyme Research , Graduate School , and Department of Biochemistry , University of Wisconsin-Madison , Madison , WI 53706 , USA Received 2 June 1998 ; received in revised form 7 September 1998 ; accepted 7 September 1998 Abstract FNR is a global regulator that controls transcription of genes whose functions facilitate adaptation to growth under O2 limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Contributions of [ 4Fe-4S ] - FNR and Integration Host Factor to fnr Transcriptional Regulation Erin L. Mettert and Patricia J. Kiley * Department of Biomolecular Chemistry , University of Wisconsin -- Madison , 1300 University Avenue , 574 MSC , Madison , Wisconsin 53706 Received 10 January 2007/Accepted 5 February 2007 Maintaining appropriate levels of the global regulator FNR is critical to its function as an O2 sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Nevertheless , while the conversion of 4Fe-FNR to 2Fe-FNR Victoria R. Sutton ,1,2 Erin L. Mettert ,2 Helmut Beinert ,3 and Patricia J. Kiley2 * Program in Cellular & Molecular Biology ,1 Department of Biomolecular Chemistry ,2 and Institute for Enzyme Research , Department of Biochemistry ,3 University of Wisconsin , Madison , Wisconsin * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>To simplify comparisons of isolated mutant and wild-type ( WT ) proteins , we constructed FNR-AR mutations in the context of FNR-D154A , a constitutively active mutant form of FNR that retains activity in the presence of O2 ( Kiley and Reznikoff , 1991 ; Ziegelhoffer and Kiley , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Indeed , there is evidence for the coexistence of oxidized and reduced forms both in isolated samples of FNR and in vivo , where the proportion of the oxidized form increases in aerobic organisms , and for the existence of a covalently modified species of FNR with an unknown substituent ( Green etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>If this is the case , then there must be a mechanism for NO induction of hmpA in aerobic cultures that involves neither MetR nor FNR ( which is inactive in the presence of oxygen ) and that mechanism is presumably the one described here involving YjeB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>However , no speci cents c spectral features were observed by electronic or EPR spectroscopy nor was any labile sul - cents de found , which excluded the presence of Fe-S clusters in the aerobic FNR preparations [ 16,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Even though little activity could be attributed to the presence of Fe in aerobic FNR preparations , it was nevertheless considered , at that time , that a mono-nuclear Fe thiolate of the rubredoxin type ( FeCys4 ) [ 23 ] was present , which might be oxidized in inactive FNR and reduced in active FNR ( discussed in [ 4,5 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The net effect of exposing anaerobic FNR to oxygen is the rapid conversion of dimeric , DNA-binding , [ 4Fe-4S ] ~ + - containing FNR protein to an inactive [ 2Fe-2S ] 2 + form that subsequently decays to the monomeric apo-protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The transition from active FNR containing cubic [ 4Fe-4S ] 2 + clusters to inactive forms , having planar [ 2Fe-2S ] 2 + clusters or no clusters , presumably involves substantial rearrangement of the essential cysteine ligands , which in turn could drive the dimer-monomer transition and thus inactivate the transcriptionally active DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The [ 2Fe-2S ] 2 form of FNR ( called 2Fe-FNR ) is monomeric in solution and is inactive for DNA binding and transcriptional regulation ( 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>ApoFNR , like [ 2Fe-2S ] FNR , is inactive in DNA binding and gene regulation ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The [ 2Fe-2S ] 21 cluster form of FNR was much more stable to oxygen , but was unable to sustain biological activity ( e.g. , DNA binding ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>In the presence of O2 , FNR is converted into a monomeric , non-DNA-binding form containing a [ 2Fe-2S ] 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Nevertheless , this [ 2Fe-2S ] 2 + cluster species of FNR was inactive for site-specific DNA binding and appeared to be in a monomeric form ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>FNR , another Fe -- S transcription factor in E. coli , only forms dimers and binds DNA in the presence of its oxygen-labile [ 4Fe-4S ] cluster .8,31 SoxR , a [ 2Fe -- 2S ] - containing transcription activator in the MerR family , can bind its single target DNA site in the presence or absence of its Fe -- S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The effect of DNA binding on the rate of cluster conversion was determined by measuring O-2 induced Fe2 release from 4Fe-FNR in the presence of 34-bp dsDNA containing the consensus FNR-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>The presence of the Fe-S cluster in the anaerobically purified form of FNR is correlated with an increase in dimerization and specific DNA binding ( 3 ) , compared with the aerobically purified form that lacks an Fe-S cluster ( 2 , 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>We report here the unexpected observation that the [ 4Fe-4S ] cluster of FNR is converted in less than 5 min to a [ 2Fe-2S ] cluster in about 60 % yield as judged from Mössbauer spectra and this conversion results in a reduction in DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>While it is known that the [ 4Fe-4S ] 2 cluster of FNR is required for DNA binding , we are not aware of any experiments that have addressed whether DNA binding inﬂuences the rate of O - induced [ 4Fe-4S ] 2 cluster conversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>To provide further evidence that the decrease in specific DNA binding is a consequence of a loss of the [ 4Fe-4S ] cluster , a time course of the oxygen-induced inactivation of FNR was first established by measuring the amount of specific DNA binding as a function of time of FNR incubation in air ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>As described below , the association of a [ 4Fe-4S ] cluster in FNR increases site-speci cents c DNA binding of FNR by increasing the ability of FNR to dimerize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Rather it appears that the form of FNR as isolated ( [ 4Fe-4S ] 2 cluster ) is the relevant form and the presence of this cluster stabilizes a conformation of FNR that is able to dimerize and bind DNA site-speci cents cally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The [ 4Fe ± 4S ] 2 + cluster of FNR is oxygen labile and controls protein dimerization and site-speci ® c DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The loss of this cluster as well as the loss of specific DNA binding upon exposure of FNR to oxygen suggested that the [ 4Fe-4S ] cluster , through its intrinsic instability , serves as an oxygen sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The presence of the [ 4Fe-4S ] cluster increases dimerization of FNR which is correlated with an increase in site-specific DNA binding of FNR , a property expected of transcription factors of the FNR/CRP family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The ability of FNR to function as a transcription factor depends on the integrity of a [ 4Fe-4S ] 2 cluster , which promotes a conformation that is necessary for FNR dimerization , site-speciﬁc DNA binding , and transcriptional regulation ( 12 , 16 , 19 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Assembly of the [ 4Fe -- 4S ] 2 + cluster results in simultaneous FNR dimerization and activity of speciﬁc DNA binding and transcription regulation [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>This property of IscR was surprising since other known Fe -- S transcription factors ( e.g. , FNR , SoxR ) have only one type of site that either does ( FNR ) or does not ( SoxR ) require the Fe -- S cluster for DNA binding .8,9 The proposed Fe -- S ligands for IscR ( Cys92 , Cys98 , and Cys104 ) are distal to the predicted DNA-binding site ( Fig. 1 ) , and upon anaerobic isolation , the cluster is partially reduced ( [ 2Fe -- 2S ] 1 + - IscR ) but is readily oxidized in air to [ 2Fe -- 2S ] 2 + - IscR .1 Because IscR appears to require its Fe -- S cluster for regulation of PiscR and not for PsufA , this raises the question of whether IscR requires its [ 2Fe -- 2S ] cluster for regulation of other Type 2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Nevertheless , if the acquisition of the iron -- sulphur is not simply fortuitous or for purely structural purposes , cluster assembly and the potential for reversible disulphide and dimer formation could provide functional switches for DNA-binding and transcription regulation analogous to those operating in FNR and FLP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>As the dimerization and DNA-binding functions of FNR require a functional [ 4Fe-4S ] iron-sulfur center that is destroyed by exposure to oxygen ( Khoroshilova et al. , 1997 ) , both of the pathways for nitrate reduction to ammonia are expressed only during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Under anaerobic conditions , acquisition of 1 [ 4Fe-4S ] 2 cluster per FNR protomer causes dimerization and enhances site-specific DNA binding ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Previously , we observed that brief exposure of anaerobic FNR preparations to air caused a decrease in both specific DNA binding and the absorption spectrum characteristic of a [ 4Fe-4S ] cluster ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Despite lacking two of the N-terminal cysteine residues that are essential in FNR , FlpA was found to acquire [ 4Fe-4S ] clusters under anaerobic conditions and their incorporation was accompanied by the loss of the DNA-binding activity ( Scott et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>However , under anaerobic conditions a dimeric form of FNR with two [ 4Fe -- 4S ] clusters per dimer that is capable of enhanced site-specific DNA binding is predominant ( Jervis & Green , 2007 ; Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_200">
Under anaerobic conditions FNR contains a [ 4Fe -- 4S ] cluster that promotes dimerization of the protein and enhances site-specific DNA binding ( Jervis & Green , 2007 ; Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="act_similarity_200", type="checkbox">
<div class="ACT_similar">
<li>Under anaerobic conditions FNR contains a [ 4Fe -- 4S ] cluster that promotes dimerization of the protein and enhances site-speciﬁc DNA binding ( Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>In contrast to WT-FNR , FNR-D154A does not require a [ 4Fe24S ] cluster 21 for dimerization and DNA binding under aerobic conditions ( Ziegelhoffer and Kiley , 1995 ; Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Therefore , aerobically purified FNR-D154A-AR derivatives provide a dimerization-competent form of FNR , which avoids the complication of obtaining uniform Fe ± S cluster occupancy in each FNR-AR mutant protein that would otherwise be necessary for the comparison of mutant phenotypes under anaerobic conditions ( Lazaz-zera et al. , 1993 ; Ziegelhoffer and Kiley , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>To correlate DNA-binding ability with the amount of the Fe-S cluster present in FNR , we examined changes in the Fe-S cluster content after exposure of the protein to air by UV-visible , EPR , and Mössbauer spectros-copies and by analyses for Fe , inorganic sulfide , and protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Interestingly , the A3 subdomain of HyfR contains a cysteine-rich segment ( - C-X , - H-C-X-C-X-P-X-C-X-P - ; Fig. 4 ) which could serve as a binding site for a metal cofactor or iron-sulphur cluster , endowing the HyfR protein with the ability to respond to changes in redox and/or oxygen status as for the FNR , IRP and SoxR transcriptional-regulator proteins ( Hidalgo et al. , 1995 ; Rouault & Klausner , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>DNA binding by the ( mutated ) FNR * protein is enhanced when the Fe-S center is reduced ( Khoroshilova et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>FNR senses anaerobiosis through the acquisition of an oxy-gen-labile [ 4Fe -- 4S ] cluster , which promotes dimerization of the protein and enhances site-speciﬁc DNA binding ( Lazzazera et al. , 1996 ; Jordan et al. , 1997 ; Popescu et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>The maximal amount of target DNA that was specifically bound by anaerobically purified FNR using these solution conditions was typically between 50 and 70 % ; higher protein concentrations led to the formation of nonspecific protein -- DNA complexes ( 15 ) , which were not quantified in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The DA154 substitution in this FNR protein has been shown to enhance dimer stability , thus providing a sufficiently active form of FNR for use in gel retardation analysis under oxic conditions ; it has been widely used to simplify the in vitro analysis of FNR -- DNA interactions [ 16,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>FNR was cross-linked to its DNA-binding sites in cultures of bacteria during exponential-growth under different conditions , and the precipitated protein-DNA complex was probed for the presence of target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Although it is not obvious why these two proteins should have different properties , the reconstituted protein was reported to be unstable in the presence of ferrous and ferric chelators , in contrast to native 4Fe-FNR , which is stable under anaerobic conditions to either the Fe3 - speciﬁc chelator Tiron ( data not shown ) or the Fe2 - speciﬁc chelator ferene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>The attenuated lytic activity of BEU705 could be restored both with and without the presence of oxygen by the introduction of CRP on a plasmid ( pDW300 ) , and to some extent , under anaerobic conditions only , by the introduction of FNR ( plasmid pGS24 ) or altered FNR proteins having CRP binding speciﬁcities ( plasmids pGS215 and pGS297 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Further mutagenic and structural studies will be needed to support this prediction and to identify other residues that contact DNA and define the DNA-binding specificity of FLE A major difference that might explain the divergent DNA-binding specificities of FNR and FLP is the presence of proline as the first residue of the DNA-recognition helix in FLP ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>In particular , substitution to Ala at position 154 increases dimerization of apoFNR ( FNR lacking any Fe or S23 ) and thus allows FNR to function in the presence of oxygen [ 20,26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>b-galactosidase assay of FNR dependent PydfZ-lacZ showed that the tandem FNR dimer exhibit 3-fold higher transcription activity than that of single copy ( Fig. 2 ) , suggesting that covalently linking of two FNR molecules can improve its activity in the presence of oxygen presumably because of enhanced protein dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>To screen for variant with further enhanced activity , we introduced D154A substitution which has been demonstrated to enhance dimerization between two FNR monomers in the presence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>In the presence of oxygen , the cluster disintegrates , and the protein no longer specifically binds DNA2 ~ Thus , FNR provides a clear example in which a reactive iron-sulfur cluster is used to sense concentrations of the destabilizing agent , oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>A mutation , in the putative dimerisation domain of FNR , results in a greater proportion of the FNR protein being puri cents ed as a dimer that is better able to-bind DNA in the presence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>However , it is also important to note that developing the methodology for studying the WT protein was greatly aided by earlier studies on proteins from FNR * mutants that alter the oxygen sensitivity of FNR [ 13,20,25 ] Of great interest is the mechanism by which the presence of the [ 4Fe-4S ] 2 cluster senses oxygen availability and controls FNR dimerization and DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>FNR senses the presence of oxygen by disassembly of its oxygen-sensitive iron-sulfur cluster , converting the protein to a non ¬ DNA-binding monomer ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>SDS-PAGE of purified FNR proteins , A , FNR samples ( 1 pg of protein ) prepared in the absence of & • mercaptoethanol , 8 , Equivalent samples of FNR proteins prepared in the presence of  J-mercaptoethano !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Furthermore , in the presence of a DNA fragment lacking the downstream FNR-1 site , FNR * rendered a single shifted band , whereas no retardation was observed at the tested protein concentrations in the absence of FNR-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>The purified material contained four separable components ( Fig. 3 ) Volume concentration ( pmol ndb DNA retarded Step ( ml ) ( rngml - ' ) per mg of protein ) a. Nbp activity could not be estimated accurately in crude extracts owing to the presence of nucleases and other DNA-binding proteins , e.g. RNAP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Advantage was taken of the FNR * protein ( FNR-D154A ) which retains some activity in the presence of oxygen and has been widely used to simplify in vitro analyses of FNR : DNA interactions [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Direct evidence of FNR binding to the yfgF promoter was obtained by EMSA with an FNR protein ( FNR-D154A , designated FNR * ) that retains some DNA-binding activity in the presence of air ( Ziegelhoffer & Kiley , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_226">
To simplify the experiments , an FNR protein ( FNR-D154A ; referred to as FNR * ) that retains some DNA binding activity in the presence of air was used in the gel retardation assays ( Ziegelhoffer & Kiley , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="act_similarity_226", type="checkbox">
<div class="ACT_similar">
<li>To simplify the experiments an FNR protein ( FNR-D154A , referred to as FNR * ) that retains some DNA-binding activity in the presence of air was used ( Ziegelhoffer & Kiley , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>In the presence of increasing amounts of [ 4Fe-4S ] - FNR protein ( 0 to 500 nM ) , the amount of fnr transcript decreased in a concentration-dependent manner , whereas no effect on the amount of RNA-1 transcript was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_229">
FNR protein concentrations are given from left to right in terms of nM total protein : [ 4Fe-4S ] - FNR ( 4Fe-FNR ) , 200 and 400 ; apo-FNR , 200 and 400 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></label>
<input id="act_similarity_229", type="checkbox">
<div class="ACT_similar">
<li>[ 4Fe-4S ] - FNR ( 4Fe-FNR ) concentrations are given from left to right in terms of nM total protein : 100 , 200 , and 400 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li></div>
<li>By quantifying the amount of [ 2Fe-2S ] 2 + cluster in 2Fe-FNR in vitro in the presence of various reductants and oxidants ( GSH , DTT , cysteine , O2 , hydrogen peroxide , and superoxide ) , we found that superoxide , a byproduct of aerobic metabolism , significantly destabilized the [ 2Fe-2S ] 2 + cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>However , it is not obvious why super-oxide-mediated destruction of the [ 2Fe-2S ] 2 + cluster of FNR is necessary since 2Fe-FNR is already inactive for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>This opens up the question of whether the 2Fe cluster of FNR is as stable in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Nonetheless , whether it is the 2Fe form or apoprotein ( or some combination of both ) that is the substrate in vivo , activation of FNR will likely require additional proteins that facilitate Fe-S cluster assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Thus , since both sulfur and Fe are released from the cluster upon reaction with superoxide , we conclude that the attack by superoxide on the [ 2Fe-2S ] 2 + cluster of FNR is likely to be more complex than that of the hydrolyase clusters from which no sulfur is released .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>In spinach chloroplasts , photosystem I reduces the 1/2 2Fe -- 2S 2þ cluster in Fd , and reduced Fd shuttles this electron to FNR [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>It is impossible from the present experiments to ascertain whether this band is due to changes in the FAD cofactor in FNR or in the 1/2 2Fe -- 2S 2þ cluster in Fd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>No repression of fnr was observed in vivo or in vitro with the apoprotein form of FNR , indicating that repression requires the dimeric , Fe-S cluster-containing protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>DISCUSSION FNR requires a Fe-S cluster for its regulatory functions as a transcription factor and in vitro is inactivated within a few minutes when exposed to air .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>However , the domain containing the L-roll structure is involved in regulation of FNR activity , since it contains Cys122 , and this Cys appears to be one of the ligands for the Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>However , it is unknown whether the FNR that was activated following the shift arose from apo-FNR , which was generated by cluster degradation , or from newly synthesized FNR , which had yet to acquire a Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>( E ) When the NO concentration is reduced , putative Fe ± S cluster-repairing mechan-isms can reconstitute the NO sensor , FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>By oxidation of the FeS cluster , FNR is converted to the inactive state in a reversible process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>FNR possesses a cysteine cluster to which iron ( Fe ( H ) ) binds only in the active form [ 2,6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>To define the oxidation state and type of Fe-S cluster present in the active form of FNR , we have studied anaerobically purified FNR with Mössbauer spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>For specific binding of FNR to DNA sites in gel retardation or in vitro footprinting dimeric FNR is required w60 ,70,73,74,121 x. FNR is also inactivated in vivo by chelating agents or after loss of the FeS cluster upon isolation of FNR Žstep II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>We also found that there was no detectable difference in the rate of O2-induced cluster conversion for FNR free in solution compared to its DNA-bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>If FNR was inactivated by O2 in vivo using this same mechanism ( 12 , 16 , 19 ) , then we expect that the rate of inactivation in cells should be similar to the in vitro rate of cluster conversion taking into account the other kinetic steps involved in in vivo measurements of FNR activity , such as the time required for O2 diffusion into cells , FNR dissociation from ndh promoter DNA , transcription and translation of lacZ , and folding and assembly of - galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In addition , the rate of FNR inactivation was monitored in vivo by measuring the rate at which transcriptional regulation by FNR is lost upon the exposure of cells to O2 ; a comparison of the in vitro and in vivo rates of conversion suggests that O2-induced cluster conversion is sufﬁcient to explain FNR inactivation in cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Determining the in vitro rates of cluster conversion also allowed us to assess whether these values were compatible with in vivo measurements of FNR inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>It has been shown , however , that ferricyanide is able in vivo and in vitro to promote inactivation of FNR function or [ 4Fe -- 4S ] FNR destruction in an artiﬁcial reaction ( Unden et al. , 1990 ; Lazazzea et al. , 1996 ) , indicating that inactivation of FNR can also be achieved by oxidizing agents without direct need for O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>The cluster ligands for the [ 4Fe-4S ] cluster of FNR are presumed to be Cys20 , Cys23 , Cys29 and Cys122 since mutational analysis of these residues has shown that they are required for FNR function in vivo [ 10,12 ] and these mutant proteins lack iron and sul - cents de upon puri cents cation ( Khoroshilova et al. , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The Fe was suggested to be present as a non-heme , non-sulfur cofactor , but recently in a mutant FNR protein a [ 3Fe-4S ] + cluster was detected ( Green et al. 1991 ; Khoroshilova et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The [ 4Fe -- 4S ] cluster is essential for O 2 2 + sensing , and is converted to a [ 2Fe -- 2S ] cluster ( [ 2Fe -- 2S ] FNR ) in the presence of O2 ( Kiley and Beinert , 1999 ; Khoroshilova et al. , 1995 , 1997 ; Green et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>In FNR prepared from aerobic cultures ( Green et al. , 1991 ) , the presence of small quantities of iron , as well as absorbance bands associated with iron-sulfur clusters , suggests that the 14Fe-4S ] 2 + cluster may be continuously assembled in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The total cellular content of FNR is similar in aerobic and anaerobic bacteria , but the apo-form is by far the most abundant aerobi species whereas the [ 4Fe-4S ] 2 + form predominates anaerobically ( Engel et al. , 1991 ; Green et al. , 1991 ; Lazazzera et al. , 1996 ; Popescu et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>To address this possibility , the effects of NO on the [ 4Fe ± 4S ] 2 + clusters of FNR were investigated in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Throughout the paper , reported 4Fe-FNR concentrations refer to the concentration of FNR molecules containing [ 4Fe-4S ] clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>These data indicate that [ 4Fe-4S ] - FNR binds only to the downstream site under the in vitro solution conditions used in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Effect of [ 4Fe-4S ] - FNR on Pfnr transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>( B ) Quantiﬁed data showing the amounts of in vitro transcription from Pfnr in the presence of increasing concentrations of [ 4Fe-4S ] - FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Through DNase I footprinting and in vitro transcription experiments , we observed that direct binding of [ 4Fe-4S ] - FNR to the predicted downstream FNR binding site is sufﬁcient for repression of the fnr promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Since the in vitro transcription assays were carried out under reaction conditions similar to those for the DNase I footprinting experiments , these data suggest that binding of [ 4Fe-4S ] - FNR to the downstream site is sufﬁcient to repress Pfnr transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>[ 4 ] E.T. Ralph , C. Scott , P.A. Jordan , A.J. Thomson , J.R. Guest , J. Green , Anaerobic acquisition of [ 4FE 4S ] clusters by the inactive FNR ( C20S ) variant and restoration of activity by second-site amino acid substitutions , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_265">
[ 34 ] Green , J. , Bennett , B. , Jordan , P. , Ralph , E. , Thomson , A. and Guest , J. ( 1996 ) Reconstitution of the [ 4Fe-4S ] cluster in FNR and demonstration of the aerobic-anaerobic switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></label>
<input id="act_similarity_265", type="checkbox">
<div class="ACT_similar">
<li>Green , J. , Bennett , B. , Jordan , P. , Ralph , E.T. , Thomson , A.J. , and Guest , J.R. ( 1996 ) Reconstitution of the [ 4Fe-4S ] cluster in FNR and demonstration of the aerobic-anaerobic switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Green , J. , Bennett , B. , Jordan , P. , Ralph , E.T. , Thomson , A.J. and Guest , J.R. ( 1996 ) Reconstitution of the [ 4Fe ± 4S ] cluster in FNR and demonstration of the aerobic ± anaerobic transcription switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Green , J. , Bennett , B. , Jordan , P. , Ralph , E.T. , Thomson , A.J. , and Guest , J.R. ( 1996a ) Reconstitution of the [ 4Fe-4S ] cluster in FNR and demonstration of the aerobic-anaerobic transcription switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li></div>
<li>If this is the case , then superoxide-mediated conversion of 2Fe-FNR to apo-FNR may make the inactivation of 4Fe-FNR irreversible under aerobic-growth conditions , preventing any transcriptional regulation by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Therefore , we proposed that 4Fe-FNR is synthesized under aerobic-growth conditions but is then rapidly inactivated by O2-induced cluster conversion ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The absence of any measurable amount of Fe-S cluster within FNR in aerobically grown cells implies that both 4Fe-FNR and 2Fe-FNR are relatively unstable under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>To address this question , we have developed a method for purifying 2Fe-FNR from whole cells , which has allowed us to examine whether significant amounts of 2Fe-FNR are present in aerobically grown cells and whether the [ 2Fe-2S ] 2 + cluster is stable to oxidants found under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Moreover , if superoxide were the major contributor to the destruction of the [ 2Fe-2S ] 2 + cluster of FNR under aerobic-growth conditions , the superoxide concentration would determine the levels of 2Fe-FNR that would be available for reactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>This lability may explain the absence of the [ 2Fe-2S ] 2 + cluster form of FNR under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>One prediction of this hypothesis is that the [ 2Fe-2S ] 2 + cluster form of FNR should accumulate under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Our earlier studies had suggested to us that the [ 2Fe-2S ] 2 + cluster form of FNR might accumulate under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>In vivo , only indirect evidence for the formation of apoFNR has been provided : the [ 4Fe-4S ] 2 and [ 2Fe-2S ] 2 clusters of FNR are lacking in aerobically grown cells under conditions in which the FNR protein is still present * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The conversion of [ 4Fe-4S ] FNR to [ 2Fe-2S ] FNR was followed under pseudo-ﬁrst-order conditions ( with O2 in excess ) by measuring absor-bance changes at 420 nm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>As a ﬁrst step , we developed an assay to monitor the [ 4Fe-4S ] 2 cluster to [ 2Fe-2S ] 2 cluster conversion under conditions where O2 could be added in excess relative to 4Fe-FNR and where product formation could be easily monitored in real time .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In the presence of O2 , conversion of the [ 4Fe-4S ] 2 + cluster to a [ 2Fe-2S ] 2 + cluster inactivates FNR , but the fate of the [ 2Fe-2S ] 2 + cluster in cells grown under aerobic conditions is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The present study shows that the predominant form of FNR in aerobic cells is apo-FNR ( cluster-less FNR ) indicating that the [ 2Fe-2S ] 2 + cluster , like the [ 4Fe-4S ] 2 + cluster , is not stable under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>These results imply that the [ 2Fe-2S ] cluster of FNR , like the [ 4Fe-4S ] cluster of FNR ( 4 , 10 , 11 ) , is unstable under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Furthermore , because the [ 4Fe-4S ] cluster of FNR is not stable under aerobic conditions ( Khoroshilova et al. , 1997 ) , we also overexpressed the mutant protein FNR-L28H , whose [ 4Fe-4S ] cluster is more resistant to destruction by oxygen ( Bates et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Under conditions of very low oxygen tension , most of FNR will be in the dimeric state and will contain the 4Fe cluster , with the remainder of FNR being in the monomeric state as apoprotein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>To obtain oxygen stable FNR variants for in vitro characterizations , in this study , we utilized the approach of covalently linking two fnr alleles in tandem to promote their in situ dimerization in the absence of the [ 4Fe -- 4S ] 2 + cluster under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Thus it seems likely that the anaerobic activation of FNR requires the de novo incorporation of a [ 4Fe-4S ] 2 + cluster into apo-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Oxygen is sensed by the interconversion of the [ 4Fe-4S ] x + ( active anaerobic dimeric ) and apo ( inactive aerobic monomeric ) forms of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>FNR responds to anaerobiosis and attains its active state via the anaerobic incorporation of a [ 4Fe-4S ] centre ( Khoroshilova et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Taken together , negative autoregulation appears to provide an optimal balance of FNR protein levels , directing a new rate of synthesis of the FNR regulon under anaerobic conditions and allowing the efﬁcient inactivation of [ 4Fe-4S ] - FNR upon exposure to O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>For comparison , culture aliquots were also prepared under conditions known to accumulate only 4Fe-FNR ( anaerobic ) or 2Fe-FNR ( O2 treated ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Under anaerobic conditions , one mechanism that controls fnr expression is negative autoregulation by the active [ 4Fe-4S ] form of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Thus , we hypothesize that under anaerobic conditions , negative autoregulation also prevents [ 4Fe-4S ] - FNR from exceeding a critical level beyond which it can be efﬁciently inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>The presence of [ 2Fe-2S ] FNR instead of apoFN could possibly allow a rapid return to the [ 4Fe-4S ] FNR state upon reversal to anoxic conditions and thereby rapid restoration of anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Since the overnight sparging step was omitted , we overproduced FNR under analogous anaerobic-growth conditions to establish that , during the 1 h induction with IPTG , sufficient 4Fe-FNR was formed to allow detection of cluster from the purified protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>To further enrich for the dimeric , cluster-containing form of FNR , the [ 4Fe-4S ] - FNR preparation was subject to size exclusion chromatography under anaerobic conditions as previously described ( 40 ) and the pooled dimeric FNR protein fraction was analyzed for protein , iron , and sulﬁde content ( 2 , 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>The oxygen lability of FNR protein is attributed to its oxygen sensing [ 4Fe -- 4S ] 2 + cluster , which is assembled into FNR during its de novo biosynthesis under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Because the Isc requirement for biogenesis of the [ 4Fe-4S ] cluster of FNR is greater under aerobic than anaerobic conditions ( Mettert et al. , 2008 ) , we considered the possibility that acquisition of the [ 2Fe-2S ] cluster by IscR is reduced under aerobic conditions because of the competition with other substrates for the Isc machinery .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>As described in the following sections , a simple change in the redox state of the [ 4Fe-4S ] 2 cluster to the expected 1 + oxidation state is not the physiologically relevant redox reaction for FNR ; the cluster is very di $ cult to reduce to the +1 oxidation state in vitro and no physiological electron donors or acceptors have been identi cents ed that would maintain the cluster in the +1 oxidation state under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>No change in absorption at 593 nm was observed in a reaction of 4Fe-FNR with excess ferene under anaerobic conditions , demonstrating that the [ 4Fe-4S ] 2 cluster was stable to ferene under these conditions and that the release of Fe2 was O dependent , as ex-2 pected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Nevertheless , a variant of FNR ( FNRL28H ) with an O2-stable [ 4Fe-4S ] 2 + cluster accumulates intracellularly as the [ 4Fe-4S ] 2 + cluster form under both aerobic and anaerobic conditions , suggesting that de novo assembly of clusters into wild-type FNR can occur under aerobic conditions ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Reduction of the [ 4Fe-4S ] 2 + cluster FNR puri cents ed under anaerobic conditions has been shown by Mo ëssbauer spectroscopy to contain a [ 4Fe-4S ] cluster in the 2 + form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>At the low cell densities and the sparging conditions used in these studies , the dissolved O2 in the culture media ( 25 ° C ) was present in excess ( 220 M ) of the cellular [ FNR ] of 3.7 M ; accordingly , under these conditions , FNR inactivation rates should follow pseudo-ﬁrst order kinetics and can be extrapolated from Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>These data also imply that at O2 levels in the range of microaerobic culture conditions , the rate of cluster conversion will be even slower , indicating that the amount of FNR active in cells at a particular dissolved [ O2 ] should be largely determined by its rate of inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>This process has been observed in vivo , but also in vitro after prolonged exposure of FNR to O2 w74 ,75,77 x. Mere oxidation of FNR might be the normal reaction to inactivate FNR in the aerobicranaerobic switch , but under some conditions loss of the FeS cluster could occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>While the O2-induced conversion of 4Fe-FNR to 2Fe-FNR is sufficient to explain how FNR in anaerobically grown E. coli is inactivated upon O2 exposure , less is known about how FNR in aerobically grown cells is kept in an inactive state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Crack JC , Green J , Cheesman MR , Le Brun NE , Thomson AJ ( 2007 ) Superoxide-mediated ampliﬁcation of the oxygen-induced switch from [ 4Fe-4S ] to [ 2Fe-2S ] clusters in the transcriptional-regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>In view of the ease , rapid rate and high yield of interconversion between the 4Fe and 2Fe clusters , the possibility that the 2Fe form has the same ligands as 4Fe-FNR seems most likely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Thus , in the microaerobic state , there is likely to be an equilibrium between three species , 4Fe , 2Fe and apo-FNR , with the 4Fe species being produced by biosynthesis pathways from apo and eventually being converted to 2Fe by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>2 Because the extinction coefﬁcient for the Fe - ferene complex 2 2 is much higher than that of either the [ 4Fe-4S ] or [ 2Fe-2S ] cluster , using Fe2 - ferene to monitor reaction progress offered the advantage that [ O ] could be varied over a wide range 2 while still maintaining O in excess of 4Fe-FNR , which was not 2 possible when monitoring changes in Fe-S cluster absorption in the visible region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Both systems become activated in response to anaerobiosis , but whereas FNR is a direct O2 sensor via an oxygen sensitive [ 4Fe-4S ] 2 cluster , ArcB senses O2 availability indirectly by monitoring the redox state of the quinone pool ( 28 -- 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Although information is accumulating about the mechanism of [ 4Fe-4S ] > cluster disassembly in response to oxygen ( both in vivo and in vitro ) , tar less is known about the assembly of the iron-sulfur clusters in FNR during anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Reversal of cluster conversion When oxygen-treated FNR was made anaerobic and reduced by dithionite , Moëssbauer spectroscopy showed that the [ 4Fe-4S ] 2 cluster was regenerated in 90 % yield on the basis of total cluster iron , that is , two 2Fe clusters apparently reformed one 4Fe cluster [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>In addition to providing a means for rapidly reaching steady-state levels of [ 4Fe-4S ] - FNR , the amount of [ 4Fe-4S ] - FNR protein produced by the negative autoregulation mechanism also seems optimal for the efﬁcient inactivation of FNR by O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In the absence of O2 , FNR binds a [ 4Fe-4S ] 2 cluster ( ligated by Cys-20 , 23 , 29 , 122 ) promoting the formation of a transcriptionally active dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>In addition to the synthesis of the FNR protein , such steps include the acquisition of the [ 4Fe-4S ] 2 cluster , superoxide-induced conversion of 2Fe-FNR to apo-FNR , reconversion of 2Fe-FNR to 4Fe-FNR , and the possible recycling of the apo-FNR form to regenerate active 4Fe-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>We have thus further supported our proposed model for the function of FNR , namely that the active form of the FNR protein contains a [ 4Fe-4S ] 21 cluster , and that this cluster serves as the O2 sensor by virtue of its extreme sensitivity toward O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The primary mechanism of regulation is the direct inactivation of FNR via the O2-dependent destruction of its [ 4Fe-4S ] cluster , which is required for its activity ( 20 , 28 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>To test whether the anaerobically functional dimer of ( FNRD154A ) 2 was still dependent on the [ 4Fe -- 4S ] 2 + cluster and thus retains the O2 sensing capability , we introduced C122A , which has been shown to abolish the [ 4Fe -- 4S ] 2 + cluster binding and transcription activity upon introduction to native FNR [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>[ 14 ] E.L. Mettert , P.J. Kiley , ClpXP-dependent proteolysis of FNR upon loss of its O2-sensing [ 4Fe-4S ] cluster , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Finally , we consistently ﬁnd , through numerous experiments of variable design , that the rate of the conversion of 4Fe-FNR to 2Fe-FNR , and the consequent release of Fe2 , is dependent on the concentration of O2 ; this ﬁnding is inconsistent with the proposal ( 7 ) that the rate-determining step in cluster conversion does not involve O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>To induce the conversion of 4Fe-FNR to 2Fe-FNR , 10 mL of cells were diluted in 390 mL of buffer M and sparged with 100 % O2 for 15 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Thus , at lower O2 concentrations that are still sufficient to inactivate 4Fe-FNR ( 30 ) , we propose that superoxide is unlikely to accumulate to levels sufficient to destroy 2Fe-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>1 Abbreviations : 2Fe-FNR , [ 2Fe-2S ] 2 + cluster containing FNR ; 4Fe-2 + FNR , [ 4Fe-4S ] cluster containing FNR ; apo-FNR , cluster-less FNR ; DTT , dithiothreitol ; FNR , fumarate nitrate reduction ; GSH , glutathione ; IPTG , isopropyl-thio-â-D-galactoside .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Direct reactivation of [ 4Fe -- 4S ] FNR from [ 2Fe -- 2S ] FNR has been demonstrated so far neither in vitro nor in vivo , and it is not known whether this reaction takes place .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Our current understanding is that FNR is continually supplied with [ 4Fe-4S ] 2 clusters by the iron -- sulfur cluster biosynthetic machinery ( 25 -- 27 ) , but O2 converts the [ 4Fe-4S ] 2 cluster to a [ 3Fe-4S ] cluster that spontaneously disassembles to yield a [ 2Fe-2S ] 2 cluster with concomitant inactivation of FNR ( 12 , 15 , 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>When O2 is supplied only brieﬂy or at low concentrations , inactivation of [ 4Fe-4S ] FNR might stop at the level of [ 2Fe-2S ] FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Upon exposure to O2 , FNR is inactivated by conversion of the [ 4Fe-4S ] 2 + cluster to a [ 2Fe-2S ] 2 + cluster , which produces the 2Fe-FNR form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>DISCUSSION The kinetic data presented in this study provide key support for the model that [ 4Fe-4S ] 2 to [ 2Fe-2S ] 2 cluster conversion is sufﬁcient to explain how FNR is inactivated by O2 in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>The O2 sensitivity of FNR is also mediated by the [ 4Fe-4S ] 2 cluster ; in the presence of O , the [ 4Fe-4S ] 2 2 cluster is converted to a [ 2Fe-2S ] 2 cluster both in vitro and in vivo ( 15 , 16 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>The properties of [ 4Fe-4S ] FNR and [ 2Fe-2S ] FNR , their presence in aerobically and anaerobically growing E. coli in vivo and in vitro , and their signiﬁcance in O2 sensing are well documented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In the presence of O2 , the [ 4Fe-4S ] 2 cluster is converted to a [ 2Fe-2S ] 2 cluster , which inactivates FNR as a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In the presence of O2 , [ 4Fe-4S ] FNR is converted to monomeric [ 2Fe-2S ] FNR , which is no longer active in gene regulation ( 6 , 7 , 12 , 18 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>[ 7 ] A.J. Jervis , J.C. Crack , G. White , P.J. Artymiuk , M.R. Cheesman , A.J. Thomson , N.E. Le Brun , J. Green , The O2 sensitivity of the transcription factor FNR is controlled by Ser24 modulating the kinetics of [ 4Fe -- 4S ] to [ 2Fe -- 2S ] conversion , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>The transition from activeto inactiveprotein upon exposure to oxygen is a bipbasic process , in which oxygen catalytically drives the redox balanced conversion of the [ 4Fe-4S ] 2 + form to the inactive [ 2Fe-2S ] 2 + form , which in turn degrades to apo-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>nalysis of the Oxidative Conversion of the [ 4Fe-4S ] 2 of FNR to a [ 2Fe-2S ] 2 Cluster Kinetic A Cluster Received 20 May 2004/Accepted 31 August 2004 The ability of FNR to sense and respond to cellular O levels depends on its [ 4Fe-4S ] 2 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Thus , apoFNR , which was reduced by DTT , and reconstituted [ 4Fe-4S ] FNR showed the same Mr of 34,000 after labeling by AMS , suggesting that AMS replaces the FeS cluster and labels FNR at up to ﬁve Cys residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Thus , these data suggest that the FNR iron -- sulfur cluster is stabilized by the presence of Phe at position 24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Thus , the presence of Phe at position 24 does not impair the ability of FNR to-bind an iron -- sulfur cluster with properties very similar to the [ 4Fe-4S ] cluster acquired by wild-type FNR under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The presence of the [ 4Fe-4S ] cluster was required for FNR to-bind to the downstream site , since no enhancements or regions of protection were detected when equivalent amounts of apo-FNR were used in the assay ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>The stoichiometry of iron and labile sulfide relative to the number of cysteine ligands of anaerobically purified FNR is most compatible with the presence of a [ 4Fe-4S ] cluster ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The [ 4Fe-4S ] 21 cluster of FNR was not reduced by dithionite , even at pH 9 and in the presence of methylviologen , indicating a very low redox potential .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Previous analysis of an FNR mutant protein ( 2 ) suggests that sensitivity toward O2 is primarily determined by subtle details of protein structure , since substitution of Leu-28 for His allowed for purification in the presence of oxygen of FNR containing a [ 4Fe-4S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>However , it is not understood how the presence of the [ 4Fe-4S ] 2 cluster increases dimerization of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>If one accepts the structure of the CAP protein as model for that of FNR , then it seems that the presence of the [ 4Fe-4S ] cluster causes FNR to adopt a speci cents c conformation which is favorable for dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>In the presence of a range of initial [ O2 ] from 80 to 440 M , an average of 4.33 0.06 M Fe2 ions were reproducibly released from 2 M [ 4Fe-4S ] 2-FNR ( 8 M with respect to cluster sulﬁde and 9.84 M with respect to the initial cluster-bound Fe ) within 2 min in the presence of 100 M ferene ( Fig. 1 ; also data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>RESULTS In vivo negative autoregulation requires the presence of [ 4Fe-4S ] - FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>These data support th notion that negative autoregulation in vivo requires the presence of active [ 4Fe-4S ] - FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_348">
Conclusions : Perturbation of the global regulator FNR and PurR functions as well as increase in purine nucleotide availability could affect the oxidative damage cell death pathway initiated by topoisomerase cleavage complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></label>
<input id="act_similarity_348", type="checkbox">
<div class="ACT_similar">
<li>Our study showed that perturbation of the global regulator FNR and PurR function as well as increase in purine nucleotide availability , could affect the oxidative damage cell death pathway initiated by topoisomerase cleavage complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li></div>
<li>For example , only residual Fe-S cluster-dependent activities were detected for IscR , fumarate nitrate regulator ( FNR ) , NsrR , glutamate synthase , isopropylmalate isomerase , NADH dehydrogenase I , and succinate dehydrogenase in strains lacking a functional Isc pathway ( 10 , 20 , 23 , 29 , 42 -- 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>For þþpyﬁD , FNR I was fully occupied when FNR or HlyX were added to a ﬁnal concentration of 5 nM ( not shown ) whereas FNR II became saturated when the concentration of either regulator was $ 10 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>It is therefore possible that Fe2 § which is required for functioning of FNR and Fur either stems from the same ` pool ' or can be oxidized and reduced in a similar mode Presence of a redox sensitive cofactor at FNR opens the possibility that sensing of 02 ( or positive redox potential ) occurs by FNR itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>In those bacteria that possess more than one member of the DNR/FLP subgroup , problems due to cross-talk may not arise because the regulators appear to be expressed in response to different environmental stimuli ( Vollack et al. , 1999 ) and their active forms may therefore never coexist Despite possessing the same structural framework , differences in primary structure between CRP and FNR within , and probably outside , the DNA-recognition helix may sufficiently alter DNA-target recognition such that CRP and FNR are able to regulate distinct modulons whilst sharing the same cytoplasm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>β-galactosidase assays For the FNR overexpression experiment , strains were grown in M9 minimal medium ( described in the above paragragh ) containing 50 µg ml − 1 Ap , and 20 µg ml − 1 Cm under lo b tern es W t analysis Strains were grown as for β-galactosidase assays , and IscR , SufD , or FNR levels were measured by Western blots as described previously ( Nesbit et al. , 2009 , Mettert et al. , 2008 , Sutton et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In vivo modulation of FNR activity by NO Regulation of hmp transcription by NO and RNS appears complex , and at least one regulatory mechanism involving MetR is known ( Membrillo-HernaÂndez et al. , 1998 ; Poole and Hughes , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>FNR -41.5 IHF bound to the IHF II site activates FNR-dependent transcription Previously , we demonstrated that IHF binding to the IHF I site represses FNR-dependent transcription at pnir ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>± 6700 ± 2200 pnir7150 pnir7150 p146A 1.0 1.0 0.8 0.8 2.5 1.9 1.1 1.6 pnir7150 p130G pnir7150 p123G pnir7150 p123G/p130G IHF binds to the nir promoter and represses FNR - dependent transcription Wu et al. ( 1998 ) demonstrated that , apart from Fis , at least one other protein factor binds to the upstream region of pnir and inhibits FNR-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>This results from its dependence on FNR that binds to a site at position -41.5 , and NarL or NarP that bind to a 7-2-7 element centred at position -69.5 ( Fig. 10 ; Jayaraman et al. , 1989 ; Tyson et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Transcription control of acs -- yjcH -- actP is complex and the intergenic region between acs and nrfA is bound by CRP , FNR , IHF , FIS and NarL or NarP and is folded into an ordered nucleoprotein structure that permits the two divergent promoters to be regulated independently ( Browning et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>It has recently been reported that FNR negatively regulates several other genes in E. coli ( Sawers et al. , 1988 ) and it is anticipated that , in contrast to genes activated by FNR , the corres-ponding regulatory regions will contain FNR-recognition sites 3 ' to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>FNR is structurally related to CRP and , like CRP , binds to a 22bp sequence that accommodates two FNR subunits ( Green et al. , 1991 ; Sharrocks et aL , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>In addition , PhlyE was characterized as a class I promoter that could be activated by the oxygenresponsive regulator FNR and/or the glucose-responsive regulator CRP ( Green and Baldwin , 1997 ; Ralph et al. , 1998 ; Westermark et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In addition , both FNR and CRP contribute to regulation of hlyE expression ( Green and Baldwin , 1997 ; Ralph et al. , 1998 ; Westermark et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>It has been shown that relief of H-NS-mediated hlyE silencing by SlyA is much less efficient in the absence of CRP as the hlyE promoter normally needs CRP or FNR to be activated to higher levels ( Westermark et al. , 2000 ; Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Transcription from the main promoter , PsdhC , is thought to be activated by cAMP-CRP and was shown to be activated by reduced FNR and phosphorylated ArcA ( Park et al. 1997 ; Shen and Gunsalus 1997 ; Cunningham and Guest 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>The third feature is that yﬁD expression is enhanced during growth at acid pH. It is well known that the effects of pH on gene expression overlap with other environmental factors and particularly with oxygen availability ( Cotter et al. , 1990 ; Meng & Bennett , 1992 ; Neely et al. , 1994 ; Slonczewski & Foster , 1996 ; Blankenhorn et al. , 1999 ) and it now appears that yﬁD expression responds to both oxygen starvation and acidic pH via FNR , ArcA and an as yet unknown acid-responsive component .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>The available evidence therefore supports the notion that HIyX activates the expression of a latent haemolysin gene in E. coli , and that the failure of FNR to activate the expression of that gene is a consequence , in part , of subtle differences between the properties of HlyX and FNR ( Green et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The FnrN protein of R. leguminosarum is functionally similar to FixK of R. meliloti in that it activates expression of fixN , and fnrN mutants have reduced nitrogenase activity ( Colonna-Romano et al. 1990 ) However , FnrN appears to be sensitive to oxygen , indeed when expressed in E. coli , FnrN behaves rather like the FNR protein itself ( Schlfiter et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>M o r e recently , studies with lac2 fusions have shown that sdh gene expression is anaerobically repressed some 70-fold by the combined effects of ArcA and FNR , a n d repressed up to threefold by glucose in a CRP - ( and FruR - ) independent manner ( Iuchi et al. , 1994 ; Park et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Both cai and ® x operons require the presence of FNR and CRP for their expression , while they are partially repressed by H-NS ( Eichler et al. , 1995 ; Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>This is presumably due to the increased dependence of glpACB expression on FNR duringanaerobiosis ( Iuchi et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1372899" target="_blank">1372899</a>]<br></li>
<li>Furthermore , the FNR homologue , Anr , activates CCP expression in Pseudomonas aeruginosa ( Zimmermann et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>We previously demonstrated that NO acts as an effector in FNR regulation of Azotobacter vinelandii gene expression ( Cruz-Ramos et al. , 2000 ; Wu et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>As a read-out , we used a ydfZ-lacZ gene fusion , the expression of which is activated by the holoform of FNR ( Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Anaerobically , the global regulator FNR ( Kiley and Beinert , 1999 ; Green et al. , 2001 ) is also involved in the regulation of hmp since an fnr mutation enhances hmp-lacZ expression ( Poole et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>As an example , the global regulator FNR , whose expression is partially under the control of RpoS , represses catabolic genes involved in carbohydrate metabolism ( Dong et al. , 2008 ; Patten et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23867204" target="_blank">23867204</a>]<br></li>
<li>Expression of yﬁD is activated by FNR acting at ® 40 ± 5 , a conventional activation position , but is downregulated by a second FNR dimer at ® 93 ± 5 , which leads to maximal yﬁD expression during microaerobic-growth ( Green et al. , 1998 ; Marshall et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>The simplest explanation for this data , based on our knowledge of FNR-mediated regulation of yﬁD expression ( Green et al. , 1998 ; Green & Marshall , 1999 ; Marshall et al. , 2001 ) , is that ArcA relieves the repression mediated by the far upstream FNR dimer by competing for occupation of the ® 93 region of the yﬁD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>The ndh gene belongs to a small group of genes that are anaerobically repressed by FNR , others being cyo , sodA , fnr and possibly cyd ( Spiro et al. 1989 ; luchi and Lin , 1991 ; Cotter and Gunsalus , 1992 ; Hassan and Sun .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Many studies have been done to examine the effect of ArcA and FNR on regulation of the electron transfer chain genes and the TCA cycle genes ( e.g. Cotter et al. , 1990 ; Iuchi et al. , 1990 ; Cotter and Gunsalus , 1992 ; Chao et al. , 1997 ; Cook et al. , 1997 ; Park et al. ,1997 ; Cunningham and Guest , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>]<br></li>
<li>The relief of ArcA - and FNR-mediated repression accounts for the increased abundance of CAC and aerobic respiratory chain transcripts upon introduction of air ( Table 1 ) ( reviewed by Lynch and Lin ( 29 ) and Guest et al. ( 1 ) ) , and the temporal changes observed here are consistent with the kinetics of FNR inactivation by O2 ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>The effect of ArcA and Fnr on the TCA cycle genes and the electron transfer chain genes under microaerobic conditions is interesting since it was previously found that ArcA and FNR regulators affect the succinate ﬂuxes as well as the redox potential ( as measured by the NADH/NAD ratio ) , under those + conditions ( Shalel-Levanon et al. , 2005a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>]<br></li>
<li>The cyd and cyo operons encoding the quinol oxidases Cyo and Cyd which operate under aerobic or microaerobic conditions are subject to O2 regulation by ArcA and FNR ( Tseng et al. , 1996 ; Fu et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_384">
In the presence of oxygen , disassembly of the FNR iron -- sulphur clusters is accompanied by the formation of FNR monomers and inhibition of DNA binding , thereby switching off FNR-activated genes ( Kiley & Beinert , 2003 ; Lazazzera et al. , 1996 ; Sutton et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="act_similarity_384", type="checkbox">
<div class="ACT_similar">
<li>In the presence of oxygen , the iron -- sulphur clusters disassemble , leading to the formation of FNR monomers and subsequent inhibition of DNA binding , thereby switching off FNR-activated genes ( Crack et al. , 2004 , 2006 ; Kiley & Beinert , 2003 ; Lazazzera et al. , 1996 ; Sutton et al. , 2004a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>In Alcaligenes species , Paracoccus denitrificans , Vitreoscilla , and Azorhizobium , the presence of FNR homologs has also been suggested ( Nees et al. 1988 ; Hoitink et al. 1990 ; Spiro 1992 ; Hoeren et al. 1993 ; Nishiyama et al. 1993 ; Cramm et al. 1994 ; Joshi and Dikshit 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>FNR DA154 , a constitutively active mutant of FNR that dimerizes stably in the presence of oxygen ( 48 ) , was puriﬁed as a His6-tagged protein from strain M15 by a new method described by Wing et al. ( H. Wing , J. Green , J. Guest , and S. Busby , submitted for publication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Green et al. ( 9 ) reported that the wild-type FNR protein , purified in the presence of 0.1 mM Fe2 + salts , contained 1.1 mol of Fe per mol of FNR monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Genetic and biochemical analyses of FNR suggest that the Fe is bound in a tetrahedral conformation by four cysteine residues ( Trageser and Unden 1989 ; Melville and Gunsalus 1990 ; Sharrocks et al. 1990 ; Unden et al. 1990 ; Six et al. 1995 ) , similar to the situation in rubredoxin ( Jensen 1974 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>However , only mutations in Cys122 , but not in Cys20 , Cys23 , and Cys129 strongly decreased the Fe-content in isolated FNR , indicating that Cys122 is of special importance for Fe binding ( Green et al. 1993 ; Six et al. 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Of the five cysteine residues in FNR , Cys-122 appears to be the most important in that it is absolutely required for disulphide bond formation , iron binding and covalent modification ( Green et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>FNR-D154A-AR mutants do not impair DNA binding When FNR-AR mutants were initially isolated , in vivo FNR-dependent repression of Pndh was used as a reporter for FNR ± DNA binding ( Williams et al. , 1991 ; Bell and Busby , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The cofactors of FNR essential for specific binding to DNA were not studied so far in vitro , since isolated FNR does not bind sequence specific to DNA ( Green et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The increase in absorbance at 330 nm is frequently seen during the degradation ( and acquisition ) of the iron -- sulphur clusters of FNR and related proteins , e.g. the FNR of Azotobacter vinelandii ( Wu et al. , 2000 ) , but the reason is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The figure shows the β-galactosidase activity expressed from frdA ' - ' lacZ in Miller units ( Miller 1972 ) FNR as a Fe-dependent gene regulator FNR is isolated as monomer and contains loosely bound Fe ( Unden and Guest 1985 ; Green et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The iron was bound very loosely and the amount of bound iron was inversely related to the amount of cysteines ( max 4/mol FNR ) reacting with DTNB ( Green et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The requirements for Lrp and IHF ( especially in the absence of FNR ) are consistent with their greater abundance in slow-growing or stationary-phase organisms ( Landgraf et al. , 1996 ; Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The results of these experiments also challenged claims that FNR can function as a physiologically relevant sensor of NO ( Cruz-Ramos et al. , 2002 ; Corker & Roole , 2003 ; Pullan et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>The putative cAMP-CRP binding sequence resembles that of canonical FNR ( Scott et al. , 1995 ) , but it appears that FNR is not involved in the regulation of gntK expression , because the gluconate Transcriptional attenuation within the beginning of gntU The CAT activity from pGNTKU-CAT7 also depended on cAMP , like pGNTK-CAT1 but it was three to sixfold lower under all conditions tested ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>The presence of potential FNR-binding sites suggests that aconitase may be negatively regulated by both FNR and ArcA , like cytochrome o oxidase ( Iuchi , et al. , 1990 ; Cotter et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1541275" target="_blank">1541275</a>]<br></li>
<li>The consensus sequence for FNR binding is very similar to that for CRP , differing at just two symmetrically related positions in the inverted repeat ; appropriate changes at these positions can result in interconversion of CRP and FNR sites ( Bell et aL , 1989 ; Zhang and Ebright , 1990 ; Jennings and Beacham , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Recent works suggest that such negative regulation is indirect and involves trans-regu-lators such as FliA or FNR or diguanylate cyclases tha produce the signal molecule c di-GMP ( Dong et al. , 2008 ; Patten et al. , 2004 ; Pesavento et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23867204" target="_blank">23867204</a>]<br></li>
<li>Keywords : CRP -- FNR protein superfamily , iron -- sulphur proteins , transcriptional regulation , nitrogen starvation , stationary phase INTRODUCTION The CRP -- FNR family of transcription regulators contains at least 70 members that respond to a variety of metabolic and environmental signals to control a diverse range of physiological functions in an equally diverse range of bacteria ( Spiro , 1994 ; Guest et al. , 1996 ; Vollack et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>These include the transcription factor FNR that mediates anaerobic regulation ( Unden and Schirawski , 1997 ) , the cAMP receptor protein ( CRP ) , which activates transcription of catabolic operons ( Kolb et al. , 1993 ) , and the nucleoid-associated protein H-NS ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The FNR protein primarily coordinates gene regulation in remodelling central metabolism in response to changes in oxygen availability in many bacteria ( Guest et al. , 1996 ; Korner et al. , 2003 ) , but it also exerts effects over other systems , and new members of the FNR regulon are still being uncovered ( Covert et al. , 2004 ; Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>IHF is also a positive effector of 070 promoters , some of which , for example hemA , narGHJZ , narK , pfl and sodA , are regulated in response to oxygen by FNR ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Furthermore , the ACS enzyme is positively regulated by CRP ( generating acetyl-CoA ) , high levels of AMP , and FNR ( at low oxygen partial pressure ) , whereas it is negatively regulated by the j2 factor ( Kumari et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Control of signal transmission to the ArcA transcription factor is complex ( Alexeeva et al. , 2000 ; Kwon et al. , 2000 ; Matsubara and Mizuno , 2000 ) , and it is clear that the Arc system and FNR represent two quite different means of monitoring the presence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>These experiments are in agreement with the observations of Jordan et al. [ 30 ] , who could not detect signi cents cant amounts of ferric iron by EPR when FNR was oxidized by oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>β-galactosidase values were expressed as fold repression for the level of IscR activity , and as fold activation for the level of FNR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>The FNR transcriptional-activator activity was expressed in fold activation by dividing the β-galactosidase activity measured for each strain by the β-galactosidase activity present in the strain lacking FNR ( PK8203 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>The changes in ndb-lacZexpression further suggest that a regulatorynetwork is operatingat the ndhpromoter , in which Arr ( amino acid specific activation ) and Nbp ( growth-phase-dependent repression ) may serve to modulate the repression mediated by the primary regulator , FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>The current microarray data confirmed 32 of these 43 assignments , but alone did not confirm FNR activation of five operons ( adhE , glpTQ , cydDC , hlyE and arcA ) or FNR repression of six operons ( hemA , narXL , tpx , yeiL , norVW and ubiCA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Note , however , that these microarray data alone did not confirm FNR activation of five of these operons ( adhE , glpTQ , cydDC , hlyE , and arcA ; Table 2 , part B ) or FNR repression of the seven operons listed in Table 2 , part D. Comparison with Previous Microarray Studies of the Effects of Oxygen and FNR on the Transcription of Previously Identified FNR-dependent Operons -- The first three columns of ratios in Table 2 provide a basis for comparison with previous microarray studies of the effects of oxygen and FNR on transcription of operons known from independent studies to be FNR-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Although synthesis of FNR protein is indeed subject to catabolite repression ( 39 ) , fnr ' - ` lacZ fusions contained in crp and cya mutant cells can still express 60 % of the B-galactosidase activity measured in the wild-type strain ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>In strains containing lacZ fused to Pndh , inactivation of FNR should lead to a rapid loss of Pndh repression and a corresponding increase in - galactosidase activity per milliliter of culture , reﬂecting a new rate of synthesis ; since - galactosidase is a stable protein , the rate of its accumulation should reﬂect the rate of - galactosidase synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In contrast , repression of Pfnr was approximately ﬁve - to sixfold less efﬁcient in strains expressing the FNR-CA23 and FNR-CA122 mutants than Pfnr repression by plasmid-derived wild-type FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>For example , repression of Pfnr was increased approximately fourfold in vivo when FNR was expressed from a multicopy plasmid and was larger ( approximately ﬁvefold ) in vitro than in vivo ( approximately twofold ) when FNR was expressed from the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>However , a study of the NarL regulon based on transcriptome ana-lysis [ 11 ] identiﬁed some genes that appeared to be activated by NarL without FNR , and a thorough genetic analysis of the yeaR-yoaG operon regulatory region by Valley Stewart and co-workers proved that NarL activates the yeaR promoter independently of FNR [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_420">
Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></label>
<input id="act_similarity_420", type="checkbox">
<div class="ACT_similar">
<li>Regulatory pattern III ( anaerobically repressed , but not affected by ArcA ) and pattern IV ( anaerobically activated , but not affected by ArcA ) are most easily explained as genes controlled by the FNR protein or by an as yet unidentified global regulator such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li></div>
<li>As at the nrf promoter , the nir promoter can be activated by FNR alone but FNR-dependent activation is repressed by IHF and Fis ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Another example is the promoter of the nrfA operon that is activated by FNR and repressed by Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>These data clearly indicate that the hcp promoter is activated by FNR and repressed by NsrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Consequently , protection against damage by NO must also be activated by NarL , FNR or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Only five operons were induced by nitrate , NarXL and NarP , two of which were also activated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Nitrate-stimulated NarP activated expression of 14 operons ; all of those most strongly activated by NarP were part of the FNR regulon and had been characterized independ-ently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Recent genomic studies of the FNR and NarL regulons revealed that ynfEFGHI expression is also activated by FNR and repressed by NarL ( 4 , 8 ) , but the different promoter elements were not deﬁned experimentally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>These results suggest that we might expect the total number of genes directly activated or repressed by FNR to be in the range of 200 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Consistent with the inactivation of FNR by NO ( 34 ) , it was generally observed that FNR-repressed genes ( ndh , hmpA , gpmA , and lpdA ) were induced , whereas FNR-activated genes were repressed ( narG , pyrD , yjiH , aroP , and rmuC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>The combined data , summarized in Fig. 2 , have established that at least 103 operons are regulated by FNR , 68 of which are activated by FNR and 35 are repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>On this basis , we predict that relatively few operons that are strongly activated or repressed by FNR were not detected in the current experiments , and the biochemical relevance of any small effects of FNR missed in the current experiments is likely to be minimal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_433">
A total of 44 operons not previously known to be included in the FNR regulon were activated by FNR and a further 28 operons appeared to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></label>
<input id="act_similarity_433", type="checkbox">
<div class="ACT_similar">
<li>Forty four operons not previously known to be included in the FNR regulon were activated by FNR , and a further 28 operons appeared to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li></div>
<li>Expression in fnr + strain Expression in fnr m utant + Nitrate + Nitrite + Nitrate + Nitrite Example Regulatory subgroup 150 175 0.4 2.5 0.03 60.4 7.8 76 29.9 38.5 8.8 47.8 4.7 3.8 1.7 20.4 51.4 15.5 42.9 0.3 87.5 5.2 hmpA ytfE Repressed by FNR ; induced by both nitrate and nitrite 129 0.2 1.6 0.03 14.6 9.5 Activated by FNR ; induced by both nitrate and nitrite nirB hcp nrfA yeaR-yoaG yibIH Activated by FNR ; repressed by NarL but activated by NarP Induced by nitrate ; strongly induced by nitrite in an fnr mutant In summary , we have exploited at least five critical factors that limited previous attempts to define the FNR regulon of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Loss of ability of an fnr or narXL mutant to reduce nitrate will result in more reduction of the ubiquinone pool , and hence repression of components of the Arc regulon such as cyoA , so FNR should indirectly activate rather than repress cyoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Identiﬁcation and Characterization of a Two-Component Sensor-Kinase and Response-Regulator System ( DcuS-DcuR ) Controlling Gene Expression in Response to C4-Dicarboxylates in Escherichia coli PAUL GOLBY ,1 SUZANNE DAVIES ,2 DAVID J. KELLY ,2 JOHN R. GUEST ,2 AND SIMON C. ANDREWS1 * The School of Animal and Microbial Sciences , University of Reading , Whiteknights , Reading RG6 6AJ ,1 and The Krebs Institute for Biomolecular Research , Department of Molecular Biology and Biotechnology , University of Shefﬁeld , Western Bank , Shefﬁeld S10 2TN ,2 United Kingdom Received 30 September 1998/Accepted 8 December 1998 The dcuB gene of Escherichia coli encodes an anaerobic C4-dicarboxylate transporter that is induced anaerobically by FNR , activated by the cyclic AMP receptor protein , and repressed in the presence of nitrate by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_438">
The dcuB gene is strongly induced anaerobically by FNR , repressed in the presence of nitrate by NarL , and is subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="act_similarity_438", type="checkbox">
<div class="ACT_similar">
<li>The dcuB gene is strongly activated anaerobically by FNR , repressed in the presence of nitrate by NarL , and subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li></div>
<li>The frd ( Con ) mutants such as ECL511 synthesize fumarate reductase aerobically and no longer respond to FNR or nitrate repression mediatedvia NarL , whereas frd ( 0xr ) mutants , ECL515 and 516 , are partially derepressed aerobically , strongly induced by anaerobiosis but still subject to the nitrate effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The GA85 substitution suppresses nitrite-dependent stimulation of pnrf , suggesting again that nitrite-triggered NarL or NarP Interacts in some way with FNR , and that this interaction can be disrupted by an alteration in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>However , the effects , if any , of FNR , nitrate , nitrite , NarXL , and NarP on torCAD and frdABCD transcription , and also the responses of the large majority of operons known to be subject to FNR control , were entirely consistent with previous data from other studies , so we conclude that this was not a problem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Integration of three signals at the Escherichia coli nrf ole f Summary Expression from the Escherichia coli nrf operon promoter is activated by the anaerobically triggered transcription factor , FNR , and by the nitrate/nitrite ioncontrolled response-regulators , NarL or NarP , but is repressed by the IHF and Fis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>These problems are especially acute for studies of global regulators such as FNR at promoters where FNR regulation is co-de-pendent upon another factor , for example NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR , NarL and NarP regulation of # rd / # rd-red transcription 3.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>the transcription activator FNR ( fumarate-nitrate regulation ) in response to anaerobiosis , and by transcription regulators NarL and NarP in response to nitrite and nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>NarL and NarP activate FNR-dependent transcription by counteracting the repressive effects of IHF and Fis ( Fig. 1B ) , and thus the role of IHF and Fis is to set the co-dependence of pnir on both FNR and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Thus , our measurements reﬂect the ability of FNR to activate pnir without the aid of NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Operons in this group include frdA and dmsA , both of which are known to be subject to hierarchical control by FNR and NarL in response to oxygen and nitrate ( 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Appropriately , both are anaerobically induced by FNR and repressed by NarL in response to nitrate ( 14 , 17 ) , thus ensuring that oxygen and nitrate are utilized in preference to fumarate as terminal electron acceptors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>This result shows that NarL is ` active ' in the absence of extracellular nitrate or nitrite and suggests that FNR needs a ` helper ' when in the context of the nirB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>results conﬁrm that the ogt promoter is induced by NarL in response to nitrate ions , without help from FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Transcription initiation at the nrfA promoter is activated by FNR in the absence of oxygen and induced further by NarL and NarP in response to low concentrations of nitrate or to nitrite ( 35 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Transcription of a further group of genes , including yeaR , yoaG , yibH , and yibI encoding proteins of unknown function , is partially FNR-dependent and strongly induced by nitrate but also strongly induced by nitrite in an FNR mutant , suggesting that multiple regulatory circuits regulate the response of these genes to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The YtfE protein was recently implicated in the NO stress response [ 14 ] : in parallel with hmpA , ytfE transcription is strongly induced by nitrate , repressed by FNR and optimally expressed in an fnr mutant during growth in the presence of nitrite ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Conversely , transcription of both the nap and nrf operons is totally dependent on FNR , repressed by nitrate but induced by nitrite , effectively providing protection against NO generated in the anaerobic gastrointestinal tract where nitrate is a scarce but valuable electron acceptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>All of the seven operons that were both previously reported and confirmed in the current experiments to be repressed by FNR were induced by nitrate in the fnr mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Transcription from the hmp and ytfE promoters appeared to be repressed by FNR but strongly induced by both nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Microarray analysis of gene regulation by oxygen , nitrate , nitrite , FNR , NarL and NarP during anaerobic-growth of Escherichia coli : new insights into microbial physiology T.W. Overton , L. Grifﬁths , M.D. Patel , J.L. Hobman , C.W. Penn , J.A. Cole1 and C. Constantinidou School of Biosciences , University of Birmingham , Birmingham B15 2TT , U.K. Abstract RNA was isolated from cultures of Escherichia coli strain MG1655 and derivatives defective in fnr , narXL , or narXL with narP , during aerobic-growth , or anaerobic-growth in the presence or absence of nitrate or nitrite , in non-repressing media in which both strain MG1655 and an fnr deletion mutant grew at similar rates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Summary Expression from both the Escherichia coli nir and nrf promoters is dependent on anaerobic induction by FNR but is further reguiated by NarL and NarP in response to the presence of nitrite and nitrate in the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>In our recent microarray study of the FNR , NarL , and NarP regulons of E. coli , members of the NsrR regulon were up-regulated in cultures growing anaerobically in the presence of nitrate and nitrite , presumably as a consequence of NO formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>DISCUSSION Induction of the E. coli nir operon is codependent on two environmental stimuli , the absence of oxygen and the presence of nitrate or nitrite ions , and these stimuli are signaled by FNR and NarL or NarP , respectively ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>To examinewhether the nitrate block , like oxygen repression , could be lifted by increased amounts of FNR a slot blot analysis was performed on mRNA samples extracted from organisms grown anaerobically in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Mutation of the 242 heptamer ( overlapping the FNR site ) impaired repression in the presence of nitrate , but not nitrite ( Table 4 ) , despite the absence of clear protection of the 242 heptamer by phospho-NarL in the footprinting assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Regulation of acnA gene expression by CRP , ArcA and FNR The 0-galactosidase activity expressed from a single-copy A.acnA-lacZ fusion prophage in the crp ' parental strain was reduced 3-5-foldin the crp deletion strain but restored by the presence of a multicopy crp + plasmid ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>Regulatory patterns I ( anaerobically repressed , decreased expression in the presence of FNR ) and II ( anaerobically activated , increased expression in the presence of FNR ) where FNR acts as a repressor ( Table I , Fig. 4 , and Fig. 6 ) or activator ( Table II , Fig. 4 , and Fig. 7 ) are most easily reconciled with previous reports .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The expression and transport data are consistent with each other insofar as Dcu transport activity and the combined dcuA-and dcuB-lacZ activities are activated anaerobically by FNR , repressed anaerobically in the presence of nitrate by NarL , increased by fumarate or succinate , and weakly affected by ArcA ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>To compare the activity of different promoters : lacZ fusions in the presence and absence of FNR , we used E.coli JCB387 and the fnr derivative JRG1728 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>To determine whether FNR directly effected the expression of talA , encoding transaldolase A of the nonoxidative branch of the pentose phosphate pathway , we constructed a talA - lacZ fusion and measured its activity in both the presence and the absence o fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Thus , while our analysis did not indicate a role for FNR in focApﬂB expression , potential changes in transcript levels in the absence of FNR may have been masked by the presence of multiple promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>In addition , Pfnr expression was further decreased approximately fourfold in the presence of plasmidderived wild-type FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>and broken arrows ) enhances yfiD expression and intracellular YﬁD content in acid cultures in the presence and absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Anaerobic expression from the wild-type ndh promoter in the absence of FNR was 30-fold higher than that observed in the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Differential Gene Expression in the Absence of Oxygen in the Presence and Absence of the FNR Global Regulatory Protein -- A comparison of the gene expression levels between cells grown in the absence of oxygen and in the presence or absence of FNR revealed 2,402 genes that exhibited expression levels above background for all replicates of Experiments 2 and 3 ( O2 , FNR versus O2 , FNR ; Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Furthermore , the spatial separ-ation of oxygen consumption and FNR sensing of oxygen could provide an explanation for the presence of membraneassociated ( ArcBA ) and cytoplasmic ( FNR ) TFs to coordinate global gene expression in response to oxygen availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>A yeiL -- lacZ transcriptional fusion showed that aerobic yeiL expression increases at least sixfold during stationary phase , requires RpoS , and is positively autoregulated by YeiL , positively activated by Lrp ( and IHF in the absence of FNR ) and negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Attempts to detect the effects of FNR inactivation by measuring the levels of RNA produced from Pndh were unsuccessful because of low expression levels ( data not shown ) ; therefore , we investigated the effect of FNR inactivation on the production of - galactosidase from constructs in which Pndh is fused to lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Because O2 inactivates FNR and prevents Pnik expression , the nickel ions required for RcnR inactivation must enter the cells in the absence of NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>O2-induced FNR inactivation should rapidly shut off FNR-activated expression of lacZ , preventing any further increase in - galactosidase activity per milliliter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_480">
This loss of repression of rpoS expression by FNR may contribute to the increase in transcript levels of the RpoS-dependent genes in group II in the strain lacking FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></label>
<input id="act_similarity_480", type="checkbox">
<div class="ACT_similar">
<li>As with group II genes , the loss of repression of rpoS expression by FNR may contribute to the increase in transcript levels of rpoS-dependent genes in group V or may be an indirect effect of regulation of RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li></div>
<li>NifA , which activates expression of the N2-fixation genes of B. japonicum , apparently acts similarly to FNR in O2-dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Six of the eight groups showed regulation by FNR ; while some expression groups represent genes that are simply activated or repressed by FNR , others , such as those encoding functions for chemotaxis and motility , showed a more complex pattern of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Furthermore , the FNR-independent aerobic increase in YﬁD protein content at low pH correlated with enhanced expression of yﬁD : : lac in these cultures ( 110 Miller U for cultures without added benzoate , compared to 460 Miller U for those with benzoate ) , suggesting that although FNR may contribute to increasing yﬁD expression during growth under acidic conditions it is not essential for acid induction of yﬁD expression and that another factor is mediating this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>In the absence of the FNR protein , the aerobic exponential expression of nrdD was slightly reduced ( Table 2 ) and the stationary-phase induction was virtually absent , indicating that the induction during the deceleration phase of growth is caused by oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Regulation of nikABCDE expression is positively controlled by FNR ( 44 ) and negatively controlled by NikR ( 9 ) , in both cases by direct protein binding to the nikABCDE promoter ( PnikABCDE or Pnik ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>FNR was expressed and pu-riﬁed as previously described [ 8,43 ] , except that protein expression was extended for 4 -- 5 days to allow increased cytosolic accumulation of the recombinant enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>expression is controlled by the FNR protein ( Page et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934939" target="_blank">7934939</a>]<br></li>
<li>Therefore , because yﬁD expression is regulated by FNR and the YﬁD gene product is acid-inducible , we have investigated the effects of anaerobiosis , pH and growth-phase on yﬁD expression and activation of the YﬁD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Oxygen regulated gene expression in Escherichia coil : Control of anaerobic respiration by the FNR protein Gottfried Unden * & Martin Trageser Institut far Biochemie , Gebtiude 26.42 , Heinrich-Heine-Universitiit , Universitiitsstr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The role of FNR II in the control of yﬁD expression It is clear that enhanced yﬁD expression in HlyX-expres-sing cultures is not due to the superior ability of HlyX ( compared with FNR ) to activate transcription from Class I promoters but rather that HlyX occupation of FNR II has a neutral/positive effect on yﬁD expression , whereas FNR has a negative effect at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>The ability of FNR , located in a typical class II position , to only weakly activate expression of yhjA in the absence of OxyR suggests that the role of OxyR might be to counteract the effects of an as-yet-unidentiﬁed regulator that inhibits the action of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The effect of the fnr mutation may well be indirect in this case , since FNR probably controls the expression of genes that are involved in the reduction of nitrite to NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>The FNR * mutation D154A increases aerobic expression from the narGHJI promoter , probably by ` freezing ' FNR in an active-conformation ( Kiley and Reznikoff , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>The introduction of FNR on a plasmid increases expression of FF-CC by approximately twofold in the Afnr crp + host , JRG1728 , indicating a low level of synergy due to the upstream activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The expression of fnr , ndhII , cyo and possibly cyd is negatively controlled by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The requirements for YeiL , RpoS and Lrp for optimal yeiL expression were retained in the absence of FNR ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Notably , in GFP-based target fusion ( Fig. 6D ) led us to propose that the independent work by Durand and Stortz ( 55 ) , 9 of the latter anaerobic repression of sodA expression by FNR is mediated transcripts were also found to be down-regulated after FnrS post-transcriptionally by FnrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>As noted previously ( 33 ) , pyruvate induction is also completely dependent on FNR , the major activator of anaerobic pfl expression ; however , pyruvate induction was only mar-ginally affected ( 1.5-fold induction compared with 2.3-fold for the wild type ) by an arcA mutation ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The evidenceis ofthreesorts : F , thepresenceofFNRboxesinpromoters which might reasonablybe expected to be subject to regulation by an FNR homologue ; H , positive hybridisationto the clonedfnr homologueof another organism ;P , a knownFNR sensitivepromoter from another organism shows a pattern of expression ( i.e. anaerobic activation ) that is consistentwiththe presenceofan FNR homologue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The anaerobic transcriptional-regulator FNR ( Spiro and Guest , 1990 ) was shown to mediate a major proportion of the anaerobic regulation of the pfl operon and this FNR dependence correiated exactly with anaerobic induction when expression of the various 5 ' deletion derivatives was examined ( Sawers and Bock .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>One mechanism involves the transcriptional-activator and repressor FNR , the major role of which is to regulate the expression of genes involved in anaerobic respiration and pyruvate metabolism ( Spiro & Guest 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>In sharp contrast , b-galactosidase expression from PyfiD 21 ( pGS1067 ; PyfiD with an inactive 293.5 FNR site TTTATG TAAAAAAA ) was maximal between anaerobic and 10 mM oxygen ( that is , it was not repressed at very low oxygen concentrations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Although the array data presented in Table VII indicate that nirB expression is decreased during anaerobic-growth , a lacZ transcriptional fusion showed that expression of the nirBDC operon is activated by FNR during anaerobic cell growth ( 44 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>However , in the absence of FNR , ndh expression is activated by the amino acid response-regulator ( Arr ) during anaerobic-growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Anaerobic ndh expression is repressed by FNR but enhanced in fnr mutants during anaerobic-growth in rich medium ( Spiroet al. , 1989 ; Green & Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>During anaerobic-growth , transcription of the hmp gene is repressed by FNR , but severe exposure to NO stress results in inactivation of FNR and hence de-repression of hmp expression [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>FNR primarily functions to regulate anaerobic metabolism in E. coli and other bacteria by co-ordinating changes in gene expression that optimize growth potential in different niches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The Arc-activated cydAB ( 1 ) , and Arc-repressed sodA ( 13 ) and lpdA ( 26 ) , are also under the direct negative control of FNR , a regulator that co-mediates gene expression in response to the redox conditions of growth ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14711822" target="_blank">14711822</a>]<br></li>
<li>Fis FNR act either individually or in concert to regulate ndh expression in response to growth-phase and anaerobiosis , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>The need for the secondary FNR site ( - 94.5 ) for efficient repression was demonstrated by in vitro transcription analysis and with the ndh-lacZ fusions , where disrupting this site limits both aerobic expression and anaerobic repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>This suggests that FNR represses specific roles of the regulators in trancyd expression under anaerobic con - scriptional activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>All the FNR variants displayed enhanced anaerobic activation of yﬁD expression , indicating that their ability to activate transcription has not been compromised but their ability to downregulate yﬁD expression from FNR II is impaired ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Introduction The expression of genes required for anaerobic-growth of Escherichia coli is tightly controlled , and many such genes are reguiated by the FNR protein , a transcription factor that is activated by the absence of oxygen in the medium ( reviewed by Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>The transcriptional-regulator FNR of E. coli controls the expression of many genes required for anaerobic metabo-lism in response to O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The expression of anaerobic respiration and of related functions of E. coli is controlled by the positive gene regulator FNR , which activates transcription in the absence of 02 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The direct oxygen sensor FNR activates ` anaerobic ' gene expression in the absence of oxygen [ 31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The FNR protein is required for anaerobic expression , while the NARL protein mediates nitrate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The resuits in Tabie 7 confirm that expression from pnrf was totally FNR dependent and induced during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The requirement of CRP and FNR for anaerobic clyA expression was subsequently investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The reason why residual twofold anaerobic induction of , B-galactosidase activity is observed in a fir mutant is unclear , but we can not rule out the possibility that ArcA has some influence on anaerobic expression in the absence of a functional FNR protein when the region of DNA including promoter 7 is present on this construct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>The novel nature of fdhF gene expression has further been emphasized by the fact that anaerobic induction is independent of the FNR protein ( Birkmann ef al. , 1987b ) and transcription from the promoter is dependent upon the presence of a functional NtrA protein ( synonyms GInF , RpoN , ( T̂ ' * ) -LCB- Birkmann etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181234" target="_blank">2181234</a>]<br></li>
<li>Moreover , they suggest that FNR-dependent activation of hyp expression is completely dependent on the downstream half-site , and that both FNR half-sites are required for maximum an-aerobic induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>As predicted , exchanging the FNR sites so that activation depended on a low-affinity site and repression on a high-affinity site caused repression of yfiD : : lac expression both anaerobically and microaerobically ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Aerobically , FNR is largely inactive , neither FNR site is occupied and yfiD expression is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Microaerobically , a low level of active FNR appears in the bacteria , and the high afinity activating FNR site ( FNR I ) becomes occupied , allowing yﬁD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Microaerobically , active FNR begins to appear in the bacterial cytoplasm and the high afﬁnity activating FNR site ( FNR I ) becomes occupied , allowing yfiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Under these conditions there is insuficient active FNR to occupy the low afinity upstream repressing FNR site ( FNR II ) , and the data presented here suggests that ArcA binding to this region ensures that this is so , thereby facilitating maximal yﬁD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Therefore , the enhanced expression of yfiD under microaerobic conditions is dependent on the presence of a high-affinity activating FNR site ( 240.5 ) and a lower affinity repressing FNR site ( 293.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>The expression of dcuC * 9-9lacZ was increased by a factor of about 2.2 compared to that in the wild type , but the responses to oxygen , nitrate , and the regulators FNR , NarL , and NarP were comparable to those in the wild type ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>Effect of Arc and FNR on moeA-lacZ expression Although nitrate increased the level of moe-lacZ expression , oxygen had a negative effect on moe expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>FNR-independent expression is minimal , and the simplest explanation for the results is that NarL interacts with FNR : we suppose that the G85A change in FNR alters the conformation ol FNR to interfere with this interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>, ArcA and FNR , or NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Response of ttdA9 -- 9lacZ expression to the presence of O2 , nitrate and the regulators FNR , ArcA and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>Regulation of pﬂ expression by nitrate was dependent on the NarL , NarP , NarQ , and NarX proteins but independent of FNR , ArcA , and integration host factor , which are additional regulators of pﬂ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>It is now generally accepted that FNR , in conjunction with an unknown cofactor , functions in a manner analogous to CAP to regulate expression of its cognate target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>It is of interest that whereas the dependence of 'D [ glpAJ ( Oxr ) - lac ] expression on FNR was decreased , the dependence on CAP appears to be increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2403539" target="_blank">2403539</a>]<br></li>
<li>Thus , further analysis is necessary to determine whether FNR directly regulates the expression of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>FNR expression was induced by adding a final concentration of 100 µM IPTG at OD600 of 0.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>When the expression of plasmid-en-coded fnr was not induced , neither the ﬂuorescent band ( Fig. 1A ) nor the band responding to anti-FNR in the immunoblot was produced ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>By binding to the FNR consensus in front of the fnr gene , FNR represses its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Whether the NarP phosphate bound to DNA is unable to productively interact with FNR or RNA polymerase to induce fdnG gene expression is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>In contrast to NarP , when NarL phosphate is bound to DNA , it can successfully interact with FNR and/or RNA polymerase to switch on fdnG gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>This oould be caused by repression of nuo by FNR or , indirectly , through the effects of FNR on arcA expression ( Compan and Touati , 1994 ) , Mutations in bimO did not significantly affect expression of nuo277 under both condilions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>No rationale can be offered at present for the positive autoregulatory effect of YeiL on yeiL gene expression , nor for the aerobic repression by FNR , be it direct or indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>J.B. ( 1989 ) FNR-dependent repression of the ndh gene of Esdierichia coti and mefal ion requirement for FNR regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>[ 22 ] Green , J. and Marshall , F.A. ( 1999 ) Identi cents cation of a surface of FNR overlapping activating region 1 that is required for repression of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Green , J. , and Marshall , F.A. ( 1999 ) Identification of a surface of FNR overlapping AR1 that is required for repression of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Anaerobic induction of each of these promoters requires FNR and ArcA ( 35 ) , and anaerobic expression is further increased by growth in media containing pyruvate ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Here it has been shown that the anaerobic transcription factor FNR is the major activator of yﬁD expression , but that the extent of FNR-mediated activation can be modulated by the indirect oxygen sensor ArcA and the pyruvate sensor PdhR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>These results indicate that the anaerobic repression of dctA is mediated primarily by ArcA , with a minor contribution from FNR that is likely to be related to the activation of arcA expression by FNR ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Despite the activation of the repressors ArcA and FNR by the withdrawal of oxygen in the aerobic -- anaerobic transition , ndh expression did not decrease ( figure 6b ) , consistent with the microarray data ( figure 4c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>3474 RESULTS AND DISCUSSION ArcA and FNR are required for complete anaerobic pfl operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>However , this does not seem to be the case because FNR and ArcA are both essential for the anaerobic activation of pfl expression ( Sawers an 00 M. A. Quail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>This is phenotypicaily similar to the effect of an arcA mutation on anaerobic pfl-lacZ expression ( see Fig. 4C ) ArcA and FNR regulation of pfi 739 740 G Sawers Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Although FNR and ArcA are clearly implicated in the anaerobic regulation of hya , the level of anaerobic expression observed even in the arcA fnr double mutant was still greater than that found for aerobic-growth ( Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Also , the absence of a fumB transcript in the aerobic sample is consistent with previous studies showing anaerobic induction of fumB-lacZ fusions of up to 5-fold and 3 - to 10-fold reductions in anaerobic expression in the absence of FNR and/or ArcA ( 41 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Se-cond , multiple copies of the fnr ' gene lowered aerobic and anaerobic expression to 68 % and 58 % of the respective parental levels , suggesting that in the absence of ArcA , amplified FNR can interact negatively at the acnB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Previous transcriptional fusion studies have reported a 3-fold increase in the expression level of this gene during aerobic-growth , and this decreased expression during anaerobiosis is independent of FNR , as well as ArcA ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Gene expression suggests that ArcA is active in aerobic ECOM4LA , while FNR is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Many of these catabolic pathways are controlled in response to the availability of oxygen and alternative electron acceptors at the level of gene expression Moreover , many of the metabolic transitions between aerobic and anaerobic states are controlled at the transcription level , primarily by the activities of two anaerobically activated gene regulatory systems composed of FNR and ArcA/B ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Mean gene expression data ( fold change ) , acquired from the WT and ECOM4LA strains grown in triplicate , for all genes that are known to be regulated by the anaerobic regulator FNR or ArcA according to RegulonDB were compared to the reported function of the regulator ( activator or repressor ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Hence , the membrane-bound sensor of respiratory activity and anaerobic metabolism ArcB [ 28 -- 30 ] mediates changes in gene expression through its cognate regulator ArcA when the supply of oxygen is insufficient to fully penetrate the cytoplasm and inactivate the direct oxygen sensor FNR [ 31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The nuoE gene also appeared to be subject to FNR regulation in our previous work ( 1 ) , but the effect of FNR may be indirect as a consequence of its role in regulating ArcA expression ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Anaerobic activation of yﬁD expression by FNR , HlyX and hybrid proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>This pattern of expression results from ArcA activation as O2 becomes limited , while FNR represses cydAB as O2 is depleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>In con - regulated gene expression in E. coil , trast , the cytoehrome d oxidase has a ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Nevertheless , it is clear that FNR and ArcA work together to coordinate gene expression in response to the availability of oxygen ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>Transcription of ndh is repressed by FNR [ 38 ] and gene expression profiling suggests that it is activated by ArcA [ 39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>In addition to regulation by ArcA and FNR , ndh expression is repressed by the pyruvate-responsive TF PdhR ; repression is relieved by pyruvate [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>When both arcA and fnr were mutated , the basal level of moeA-lacZ expression was increased by the removal of the repressor , FNR protein , although the enhancing effect of ArcA-P was absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Consistency of differential expression with ArcA and FNR activity in the respective regulons was determined by comparing differential expression ( up - or downregulation ) with increased ArcA or FNR activity ( activator or repressor ) as reported by RegulonDB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Effects of ArcA and FNR on acnA-lacZ and acnB-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_574">
Effects of ArcA , FNR , cAMP , and CRP on dctA-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></label>
<input id="act_similarity_574", type="checkbox">
<div class="ACT_similar">
<li>Effects of CRP , ArcA and FNR on acnA-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li></div>
<li>To investigate the effects of ArcA , RpoS , and FNR on dctA expression , arcA , rpoS , and fnr deletions were transferred from the respective donor strains , ECL585 , RH90 , and JRG1728 , by P1vir-mediated transduction , to the dctA-lacZ fusion strain , JRG3351 , generating strains JRG4011 , SCA345 , and JRG4013 , respectively ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Deletion of arcA had no signiﬁcant effect on FNR-mediated expression from FF-41 .5 at both pH 7 ± 2 and pH 6 ± 3 , and thus the ArcAmediated enhancement of yﬁD expression ( Table 2 ) is probably mediated by ArcA competing with FNR for occupancy of the ® 93 region of the yﬁD promoter , where FNR can act negatively and ArcA positively , and that the positive effects of ArcA are magniﬁed by the presence of arcA in multiple copies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>On the sole basis of insignificant induction during anaerobic-growth compared with aerobic-growth ( Table 2 , part A , 1st column , which lists the ratio of aerobic to anaerobic transcript levels for the parental strain ) , the hcp-hcr , fliZY , napF , nirB , nikA , aspA , and nrdD operons could not be assigned to the FNR regulon , despite independent evidence to the contrary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Although previous studies demonstrated that FNR is inactivated by the presence of O2 , expression of glnK and yedL decreased under aerobic-growth conditions in an fnr mutant compared to the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>In contrast , transcript levels of aegA , phnA , and ycfH were increased under aerobic conditions in an fnr mutant strain compared to the wild type , although this aerobic effect of FNR was not observed with an aegA - lacZ fusion ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It is suggested FixK in R. meliloti , and the azurin genes that the cyd operon is activated by ( azu ) of Alcaligenes faecalis and A. deni - ArcA under microaerobic conditions trificans23 , which encode the blue-cop - and repressed by FNR under anaerobic per proteins thought to be involved in conditions , so that this dual control nitrite respiration and therefore likely allows cytochrome d oxidase synthesis to be induced anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>However , different regulators are used in each case : nuoA-N uses the ArcBA sensor-regulator system for anaerobic regulation whereas ndh uses FNR ; nuoA-N responds to C,-dicarboxylic acids whereas ndh responds to amino acids , although it is conceivable that the C , and amino acid responses are mediated by the same or related regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Genes and regulators responding to O2 and other acceptors FNR and ArcA , as shown earlier for NarL and NarP , control different sets of target genes Žreviewed in w4 ,5,9,11,12,44 x. FNR acts as the major regulator of anaerobic metabolism and ArcA as the regulator of aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>TorC autoregulation is different from tor anaerobic control The torCAD operon is fully induced in anaerobiosis but , in contrast to all other known anaerobic respiratory systems , this control is mediated neither by FNR , a pleiotropic anae-robic regulator , nor by ArcA , a response-regulator involved in redox control of numerous genes ( Simon et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411745" target="_blank">10411745</a>]<br></li>
<li>Abbreviations used : ArcA , anaerobic respiratory control ; FNR , fumarate and nitrate reductase regulator ; HMP , ﬂavohaemoglobin ; TMAO , trimethylamine N-oxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Remarkably , anaerobic control is mediated neither by FNR , the general anaerobic regulator , nor by ArcA , another anaerobic global regulator ( luchi and Lin , 1988 ; Pascal et aL , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>Thus , under aerobic conditions , an increase in the Fe-S cluster demand via overexpression of FNR relieves repression of the isc operon , presumably due to the competition between FNR and IscR for acquiring Fe-S clusters from the Isc machinery .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>As these growth conditions promote maximal activity of PnarG as a result of the combined function of NarL , IHF and FNR , these results suggest that the Accepted 30 August , 2000 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Therefore it is only when FNR and Arr are inactive ( owing to aerobiosis or nutritional conditions ) that the repressive effects of Fis can be observed ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Under both conditions , therefore , sufficient O2 should be within the cells to serve as the signal for FNR and direct its transfer to the inactive state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Fd binds to FNR with high afinity ðKd 6 0:5 lMÞ and is reduced under single-turnover conditions with k 1/4 2:3 s 1 obs and under steady state conditions with kcat 1/4 0:15 s 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>FNR activity under steady-state conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , under oxic conditions , [ 2Fe-2S ] FNR appears to be an intermediate in the formation of apoFNR , and apoFNR represents the inactive form of FNR in aerobically growing E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>However , since this apo-protein had been isolated prior to the discovery of 2Fe-FNR , it was unknown whether 2Fe-FNR was stable under the purificatio conditions employed in these earlier studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>For all conditions , 2Fe-FNR was incubated until time 0 , when the appropriate reagent was introduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>However , it is currently unknown what form of FNR ( e.g. the 2Fe form or apoprotein ) is activated under these shift conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>( FNRD154A ) 2 displayed signiﬁcantly enhanced DNA binding afﬁnities and transcriptional-regulatory activities to various FNR dependent promoters under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>We show that the covalently linked tandem FNR provides as an oxygen stable FNR variant that not only displays high binding afﬁnities to FNR dependent promoters under aerobic conditions but also retains its promoter speciﬁcity and conformational stability , thus can be used for various in vitro characterizations of FNR as well as its regulation of target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>To discover whether other substitutions at position 24 could influence the reactivity of the FNR iron -- sulfur cluster , 19 variants were created and tested for the ability to activate an FNR-dependent promoter under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Even though the promoter activities under aerobic conditions were lower compared to the longer promoter used initially , once again , there was no indication of aerobic activation in the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>iHowever , under aerobic conditions both promoters were completely inactivated by multicopy fnr * ( data not shown ) , suggesting that FNR may indeed repress the pdh promoter in the absence of competing signais .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>It was also concluded that any involvement of FNR must be indirect ( or operational only in the absence of ArcA ) , because a c n A expression was not enhanced in an fnr mutant , but it was partially repressed under aerobic conditions by fnr + amplification in an arcA + strain ( Gruer & Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Expression of the aspA gene is repressed by glucose ( ; 10-fold ) under aerobic conditions and enhanced anaerobically ( ; 10-fold ) in a manner which is partially ( ; 2-fold ) FNR dependent ( 16 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>To test if tandem construct ( FNRD154A ) 2 also activates other FNR dependent promoters under aerobic conditions , we measured b-gal activity of FNR activated PnarG-lacZ , PdmsA-lacZ , and FNR repressed Pndh-lacZ ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>FNR is a transcriptional-regulator whose activity is regulated directly by oxygen ( 25 ) ; therefore , it is expected that under aerobic conditions , the FNR regulon would not be signiﬁcantly differentially expressed between the WT and ECOM4LA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Here , we show that replacement of Ser-24 by Arg , His , Phe , Trp , or Tyr enhances aerobic activity of FNR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Replacement of Ser-24 by Arg , His , Phe , Trp , or Tyr enhanced FNR activity 2-fold under aerobic conditions ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Furthermore , b-galactosidase activity from the single-copy ydhY -- lacZ fusion under aerobic conditions in the absence of FNR was increased approximately twofold compared with the parent , confirming the observations made with the multi-copy plasmid-based reporter ( Fig. 1d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>This is supported by the data in Fig. 1 in which increasing FNR concentration by expression from a plasmid ( filled bar labeled FNR ) leads to an 10-fold increase in FNR activity under aerobic conditions compared with chromosomally encoded FNR ( filled bar labeled cFNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The likely explanation for this is that most of the genes controlled by these promoters encode products that play a role in anaerobic respiration of nitrate and nitrite , and thus their expression is fully induced only in both the absence of oxygen ( signalled via FNR ) and the presence of nitrate or nitrite ( signalled by NarL ) [ 4,29,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>The genes for fumarate respiration and related enzymes ( dcuB , frdABCD , fumB , ttdAB , and ttdT ) are subject to anaerobic induction by the oxygen sensor FNR and to nitrate repression by the NarX-NarL two-component system ( 15 , 16 , 37 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>Discussion The ndh gene is an unusual member of the FNR reguion because it is repressed rather than being activated under anaerobic conditions , and KMnO4 footprinting studies have now shown that RNA polymerase can not form an open complex at the ndh promoter in the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The results show that the dcuB gene is expressed exclusively under anaerobic conditions in a manner that is FNR dependent and that it is repressed by NarL in the presence of nitrate and is subject to CRP-mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The dcuB gene is expressed exclusively under anaerobic conditions in a manner that is largely FNR dependent , is repressed by nitrate through a mechanism that is mostly NarL mediated , and is strongly induced by C4-dicar-boxylates anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Maximal expression of PnarG occurs under anaerobic-growth conditions in the presence of nitrate and , in addition to FNR , requires the DNA-binding proteins NarL and IHF , which bind to DNA sequences located upstream of the FNR site ( Rabin et al. , 1992 ; Schroder et al. , 1993 ) ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Transcript-profiling experiments have shown that the abundance of the ydhY and downstream transcripts is enhanced under anaerobic conditions in the presence of FNR ( Constantinidou et al. , 2006 ; Kang et al. , 2005 ) , and that the transcripts are less abundant in the presence of nitrate ( Constantinidou et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Thirteen of these operons were also identified in the study of Kang et al. ( 2 ) , but anaerobic-growth in the presence of nitrate or nitrite was essential for revealing the FNR depend-ence of most of the remaining operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>However , repression in the presence of oxygen or nitrate was signiﬁcantly reduced with respect to the monocopy fusion , which suggests that an increase in the copy number of caiF renders transcription of the gene less dependent on the anaerobic regulators FNR and NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Other targets for future research Synthesis of three further proteins is strongly up-regu-lated by nitrate-activated NarL during anaerobic-growth in the presence of nitrate but is not dependent on activation by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>As NarG is a major catalyst for the formation of NO in the cytoplasm , protection mechanisms become essential when NarL and FNR are both active , which is during anaerobic-growth in the presence of a high concentration of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>FNR repression was often most apparent during anaerobic-growth in the presence of nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Activity of this promoter is totaiiy dependent on FNR and partialiy dependent on NarL : expression is induced by anaerobic-growth conditions but optimai activity depends on the inciusion of nitrite or nitrate in the growth medium ( Griffiths and Coie , 1987 ; Page et ai , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>A mutation in FNR that affects induction o/pnirB Since expression from pnirB is dependent on FNR in all growth conditions , it is possible that additional proteins , responsible for nitrite and nitrate induction , exert their effect by interacting with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Transcripts that are less abundant ( repressed ) in the reference pool relative to the test sample include those that are induced directly or indirectly during aerobic-growth , transcripts that are induced during anaerobic-growth by nitrate or nitrite , or repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Conversely , transcripts that are more abundant in the reference pool than in the test sample include genes that , directly or indirectly , are activated by FNR or induced during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Microarray data supplemented with bioinformatic data revealed that the FNR ( fumarate and nitrate reductase regulator ) regulon includes at least 104 , and possibly as many as 115 , operons , 68 of which are activated and 36 are repressed during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>The ynfEFGHI operon is induced under anaerobic conditions in a fumarate nitrate reduction regulator ( FNR ) - dependent manner ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Under anaerobic conditions ndh is repressed by the regulator of fumarate and nitrate reduction ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Under anaerobic conditions , the expression of ndh is repressed by FNR ( regulator of fumarate and nitrate reduction ) ( 13 , 20 , 21 ) , while the expression of cyoABCDE is repressed by FNR under anaerobic-growth conditions ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513468" target="_blank">17513468</a>]<br></li>
<li>Under microaerobic conditions , the pyruvate dehydrogenase complex is rapidly inactivated by the increasing NADH/NAD + ratio ( 10 ) ; rapid transcriptional and posttranslational activation of the PFL system would result in an efficient shift to anaerobic metab-olism , and under these conditions , FNR is probably the principle activator of pfl transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>In this study , we found that , although both FNR-AR1 and FNR-AR3 are necessary for transcription activation of PnarG and PdmsA under conditions in which FNR is the only known activator , the requirements for FNR-AR1 and FNR-AR3 are changed in the presence of additional activators at PnarG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Under conditions in which all these factors are functional , we found that the contributions of FNR-AR1 and FNR-AR3 to transcription activation were greatly reduced compared with when FNR functioned alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Although FNRD154A is able to activate gene transcription under aerobic conditions , it does not exist exclusively as dimeric form under this condition and the ratio between dimer and monomer varies under different conditions and concentrations , rendering difﬁculties to characterize the binding of FNR to its regulated promoters [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Using reverse-transcription polymerase chain reaction we show that although the flavorubredoxin gene ( flrd ) is transcribed in both aerobic and anaerobic conditions , anaerobiosis induced transcription up to 12-fold , under fermentative conditions ; a 28-fold stimulation was observed in an E. coli fnr mutant strain , showing that the flavorubredoxin gene is negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>The FNRsystem The product of the fnr gene ( FNR ) is now known to be involved in the anaerobic activation or repression of a variety of enzymes , and it may also repress its own transcription under anaerobic conditions ( Fig. 3 ) 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The proposed interaction of FNR , the product of the fnr gene , and RNA polymer-ase ( RNP ) under conditions in which frd transcription is induced or not induced is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3298218" target="_blank">3298218</a>]<br></li>
<li>This operon is therefore another candidate for encoding proteins that protect E. coli against RNS , especially in anaerobic environments where FNR might be inactivated by severe NO damage ( 11 , 12 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Such is the case for complex II in E. coli , with the complex II bio-synthetic operon ( sdhCDAB ) being negatively regulated by ArcAB and FNR in response to oxygen conditions ( i.e. anaerobic ) and positively regulated by CRP in the absence of glucose [ 108 -- 110 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>FNR is known to induce the transcription of the E. coli nrfABCDEFG operon during anaerobic cell growth [ 22 ] , and to negatively regulate the transcription of E. coli hmp [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>Expression of yﬁD in E. coli is known to be controlled by the transcription regulator FNR ( 7 , 8 ) and in response to pH-stress conditions ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11444864" target="_blank">11444864</a>]<br></li>
<li>In E. coli , the NO-induction of the hmpA gene was reported to be dependent on MetR , a transcriptional-regulator of the methionine biosynthetic pathway , under aerobic conditions ( 23 ) and on FNR , an oxygen-responsive regulator , under anaerobic conditions ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>In Escherichia coli , the shift between aerobic and anaerobic metabolism , which is controlled primarily by the global transcriptional-regulator FNR ( 13 , 23 , 32 ) , is known to involve the repression and activation of hundreds of genes ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In the Gram negative bacterium Escherichia coli , this process is primarily controlled by the global transcription regulator FNR ( fumarate and nitrate reduction ) which is activated under anaerobic conditions and upon activation , regulates the expression of more than one hundred genes in E. coli genome [ 1,2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>In the absence of oxygen , FNR binds to promoters of the genes of anaerobic metabolism and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>As synthesis of both of these nitrite reductases is dependent on transcription activation by the FNR protein , they are active only during anaerobic-growth and fulfil a dissimilatory rather than an assimilatory role ( Cole , 1978 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1435259" target="_blank">1435259</a>]<br></li>
<li>This induction is dependent on the regulatory protein , FNR , a factor essential for the transcription of a number of genes expressed during anaerobic-growth ( Cole and Ward , 1973 ; Lambden and Guest , 1976 ; Griffiths and Cole , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Finally , under anaerobic conditions in the presence of FNR , active FNR bound at the -50.5 and -94.5 sites will prevent RNA polymerase contacting the - 58 to - 1 3 2 region and this , presumably accompanied by negatively acting contacts between the bound FNR and RNA poiymerase , results in the FNR-dependent repression of ndh transcription , as illustrated in Fig. 8C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Unexplained is why the promoter for yeaR transcription is far more active during anaerobic-growth than during aerobic-growth , and more active in the absence of FNR than in its presence , despite the absence of an FNR site in the regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>During anaerobic-growth , the intracellular pool of active FNR increases such that the low-affinity upstream FNR site is also occupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Note , however , that this group also includes sodA ( 6.7-fold induction ) , which is regulated by FNR and other transcription factors , and cyoA , which is known to be repressed by ArcA during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>However , the factors controlling the anaerobic transcription of hya and hyb have not been completely established , since neither FNR nor ArcA could completely account for the anaerobic induction of these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The results indicated that FNR is not directly responsible for the residual anaerobic repression , although it could still act in conjunction with ArcA , or function as an anaerobic repressor solely in the absence of ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>FNR appeared not to function as an independent anaerobic repressor , but it could serve as an anaerobic repressor in the absence of ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>ArcA has no signiﬁcant role in dcuB regulation , although it appears that a factor other than FNR also contributes to anaerobic induction of dcuB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Surprisingly , when these strains can no longer synthesize a functional FNR protein , ArcA-dependent anaerobic regulation is restored ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>d Negative anaerobic FNR regulation in any of Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>ArcB has no function in the regulation by FNR and no sensory mutants in the regulation of anaerobic respiration were detected so far in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Furthermore , it has been shown that the alkylation sensitivity of FNR sulphydryl groups is increased by the addition of chelating agents to anaerobic cultures and by aerobiosis ~ s , and that the sulphydryl and iron contents of Glucose Pyruvate ~ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>When samples of wild-type FNR were exposed to air , then made anaerobic and reduced by photoreduction as above , no EPR signals were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>To figure out the effect of FNR in the anaerobic condition , we made a fnr - strain isogenic to KM1007 , and pMW22206 was introduced into the strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>Collectively , our results provide insight into the mechanism by which FNR negatively regulates genes such as sodA , sodB , cydDC , and metE , thereby demonstrating that adaptation to anaerobic-growth involves the action of a small regulatory RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Twenty-eight operons not previously known to be regulated by FNR appeared to be repressed by FNR during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR repressed slightly , and possibly indirectly , during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>FNR is a CRP homologue and regulates genes during anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Therefore , we propose that some CRP-regulated operons identiﬁed in this study are actually regulated by FNR in vivo and contribute to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>In further control experiments , RNA was isolated from the same anaerobic cultures at different stages of the growth cycle , and concentrations of RNA from genes that are known to be activated by FNR were monitored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR is required for anaerobic induction of narGHJI , fdnGHI , and frdABCD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>In contrast , pﬂ is under partial anaerobic control of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>This explanation of the asymmetry of the ndh transcript profiles in the transitions implies that the ndh transcript response in a pdhR mutant should resembl the wild-type for the anaerobic -- aerobic transition ( ndh repression is still relieved upon inactivation of ArcA and FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Woods and Guest ( 46 ) demonstrated that the fumB gene is elevated 4-fold during anaerobic fermentative growth , and Tseng ( 47 ) showed that both ArcA and FNR are responsible for this anaerobic activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The anaerobic fumarase , encoded by fumB , is known to be activated during anaerobic fermentative growth ( 50 , 51 ) , and Tseng ( 51 ) showed that both ArcA and FNR are responsible for this anaerobic activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Consistent with our hypothesis , we observed activation of the ArcA regulator under oxic conditions and consequent activation of fermentative metabolism during aerobic-growth , while the regulatory action of the FNR regulator remained similar to that for the wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Thus , the slight increase in cydAB transcripts observed under anaerobic-growth conditions in the microarray data can be explained by ArcA activation , and the increase in cydAB transcripts in the absence of FNR is due to the loss of FNR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>C y s ~ Inactive Active Rgure3 Speculative representation of the anaerobic conversion of FNR into its active DNA-binding form , and a selection of functions that are activated ( + ) or repressed ( - ) by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The expression of fumB is activated under anaerobic conditions by the anaerobic transcriptional-regulator , FNR ( 37 ) , and potential FNR-binding sites were sought by using a 12-bp symmetrical consensus , TGATA - - TATCAA , based on that proposed by Spiro and Guest ( 25 ; S. Spiro , M. H. Todd , P. J. Artynfliuk , and J. R. Guest , Biochem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>In conclusion , it is shown that ndh expression is driven from a Class I Fis-activated promoter and that Fis-mediated activation is inhibited under anaerobic conditions by FNR binding to two sites immediately ﬂanking the region occupied by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>As IscR protected a region of the hyaA promoter that overlaps the − 35 promoter element and site ( s ) where an activator may bind , one mechanism to explain the lack of IscR repression of the hyaA and hybO promoters under anaerobic conditions would be the occlusion of IscR binding by anaerobic regulators ( ArcA , AppY or FNR ) and/or RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Nevertheless , the much reduced anaerobic activities observed in the absence of FNR , are consistent with inactivation of an Arr-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Although our studies indicate that the predicted upstream site is not an FNR binding site , we found that this sequence binds IHF and increases expression of fnr under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Although the improved FNR binding site in frd-5 explains the increased promoter activity under both aerobic and anaerobic conditions it does not account for thefive-fold higher levelsof fumarate reductase produced in the fnr mutant ECL522 compared to the frd + fnrstrain , ECL520 ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Under aerobic conditions ( i.e. , after exposure to air ) , the binding affinity of purified FNR152 for the synthetic FNR consensus site was decreased by about 8-fold compared to the binding under anaerobic conditions ( Figs. 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>One mutant protein , FNR152 , showed specific binding at the FNR consensus site under anaerobic conditions ( Fig. 1B ) , thus permitting purification of this protein in an active form to near homogeneity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>( A ) Holo-FNR ± DNA binding and repression of hmp under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>For DNA-binding assays , reconstituted FNR was separated from the reaction mixture by passage through a spun column ( Sephadex G-25 ) under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>For long-term growth under oxic conditions , on the other hand , a stable repression of genes encoding enzymes of anaerobic metabolism is appropriate ; the presence of apoFNR instead of [ 2Fe-2S ] FNR might suppress frequent shifts to anaerobic metabolism during short-term lack of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Despite this , it has been shown that even small increases in FNR protein result in increased FNR activity under aerobic-growth conditions , indicating that excess FNR is not efﬁciently inactivated , presumably as a result of insuf-ﬁcient time to inactivate the additional protein ( 4 , 37 , 38 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>promoter is derepressed in response to increased Fe-S demand cR is e h T In contrast , derepression of PiscR was not observed when FNR was overexpressed under anaerobic conditions ( Fig. 7B ) , despite the accumulation of FNR protein to similar levels as under aerobic conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Furthermore , the anaerobic transcription of fnrS was abolished in a fnr strain , thus demonstrating that the global regulatory protein FNR induces FnrS transcription in response to anoxic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Under aerobic-growth conditions , proteolysis decreases FNR protein levels ( 12 , 38 ) , while under anaerobic conditions , FNR represses its own transcription ( 24 , 41 , 44 , 47 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>A previous study indicated that FNR present in aerobic cells can be activated upon a shift to anaerobic conditions in the presence of a protein synthesis inhibitor ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Transport studies suggest that the Dcu systems are expressed exclusively under an-aerobic conditions , activated by the anaerobic activator protein FNR , and repressed in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Glutathione and the Formation of Active FNR Formation of Active FNR : FNR protein is present in E. coli in roughly the same concentrations under aerobic and anaerobic-growth conditions ( Spiro and Guest , 1987 ; Unden and Duchêne , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>FNR protein levels were also compared for cells grown under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>This conclusion was based on the observation that no new protein synthesis was required to observe FNR activation in vivo , when aerobic cultures were shifted to anaerobic-growth conditions [ 35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>To determine the length of time it took for Pndh to establish a new rate of synthesis following O2-mediated inactivation of FNR , - galactosidase activity per milliliter of culture was measured from cells following a shift from anaerobic to aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>To compare the time of FNR inactivation obtained in vivo with the rate of FNR cluster conversion determined from the Fe2 - ferene experiments , the number of FNR molecules present per cell was established for both aerobic and anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Under such conditions , any differences in the transcriptomes of an fnr mutant and its parental strain would be due to both direct effects of FNR and to differences in growth rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Therefore , we grew PK872 cells at 37 ° C under aerobic conditions in a minimal glucose medium and induced overexpression of FNR with IPTG for 1 h , after which the cells were immediately processed for purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Because FNR is presumed to be inactive under aerobic ( O2 ) conditions , we did not perform experiments comparing fnr genotypes under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Even though under standard aerobic-growth conditions the O2 concentration in the medium can never exceed 220 M at 37 ° C , it is in excess relative to the cellular concentration of FNR ( 7 M ) ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>If the second explanation is correct , repression of Hmp synthesis by FNR implies that , under normal growth conditions , Hmp is primarily formed to protect bacteria during aerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>As expected ( 24 , 41 , 44 , 47 , 54 ) , no repression of Pfnr by FNR was observed under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In this case , transcript levels were higher under anaerobic conditions than under aerobic-growth conditions in the wild-type strain , and this increase in transcript levels was dependent on FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Furthermore , in a strain lacking FNR , transcript levels were increased , although not always to the same level found in aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_705">
However , it is interesting that these newly discovered genes that are regulated by FNR under anaerobic conditions fall into to the same functional classes as the genes regulated by the leucine-responsive regulatory protein ( Lrp ) under aerobic conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></label>
<input id="act_similarity_705", type="checkbox">
<div class="ACT_similar">
<li>These newly discovered genes fall into the same functional classes as the genes regulated by the leucine-responsive regulatory protein Lrp under aerobic conditions ( 25 ) and FNR under anaerobic conditions ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li></div>
<li>More interestingly , of the remaining 27 genes of this expression group , 20 were also found to be regulated by FNR under an-aerobic conditions ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>In fact , of the 31 remaining genes of this expression group regulated by FNR under anaerobic-growth conditions , 10 are regulated by Lrp under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In agreement with the suggestion that there is a significant interaction between the Lrp and FNR regulatory networks , 23 of the 58 genes of this group are regulated by Lrp under aerobic-growth conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In order to test whether FNR speciﬁcally represses genes in the rpoS regulon or whether FNR acts on rpoS directly , we measured the - Gal activity from fnr and fnr derivatives of strains containing an rpoS - lacZ fusion grown under anaerobic conditions ( 80 ) ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Under anaerobic-growth conditions , transcript levels of 39 genes , representing 35 operons ( group II ) ( Fig. 2 ) , were increased in the strain lacking FNR relative to wild-type strain MG1655 , indicating that FNR repressed their transcription under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Expression of the 12 genes in group VII was decreased under anaerobic conditions in both wild-type and FNR mutant strains , indicating that the regulation of these genes is independent of FNR ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Fifty-ﬁve genes showed a decrease in expression in a strain lacking FNR , although surprisingly , their expression in a wild-type strain was only slightly increased under anaerobic conditions ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Levels of transcripts of three genes , ymdA , yqjH , and yjdA , were unchanged in the wild-type strain under the experimental conditions tested but were decreased in a strain lacking FNR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>To identify genes controlled by FNR , we used Affymetrix Antisense GeneChips to compare global gene expression proﬁles from isogenic MG1655 wild-type and fnr strains grown in glucose minimal media under aerobic or anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>2 To identify which genes of E. coli K-12 are differentially expressed under anaerobic conditions and to determine the role of FNR in this process , we compared changes in transcript proﬁles of an FNR mutant with its wild-type parent , MG1655 , grown under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It is tempting to speculate that in the wild-type cell , low aerobic expression of gipA largely reflects activation by CAP , whereas the high anaerobic expression of the operon largely reflects activation by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2403539" target="_blank">2403539</a>]<br></li>
<li>yﬁD expression was anaerobically induced in the wild-type strain , while in the absence of FNR , both the aerobic and anaerobic expression levels were decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The aim of the work described here was to further characterise the role of FNR in regulating hyp expression under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The expression of hmp is highly induced by NO under aerobic and anaerobic conditions , through a complex regulation that involves at least three regulators , namely , FNR , MetR , and the NO-sensitive repressor NsrR ( 3 , 9 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>The ndh gene is normally expressed under aerobic conditions when FNR is inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>The FNR strain had half as much - Gal activity as the strain lacking FNR , suggesting that FNR does repress rpoS expression under an-aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It has long been appreciated that the activity of FNR must be regulated by O2 availability , since FNR dependent gene expression is observed in vivo only under anaerobic conditions , while similar levels of this protein are present in both aerobic and anaerobic grown cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>However , for both strains , the shift from aerobic to anaerobic conditions clearly causes gene expression changes in the FNR regulon consistent with known FNR activity of activation or repression ( Fig. 2A ) , suggesting that FNR activity changes in the aerobic-anaerobic shift but not between these two strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Since FNR has also been reported to enhance arcA expression under anaerobic conditions ( although this result was not observed in our studies ) , it raises the question of whether the effects of FNR observed with this group are all direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Studies with an ndh-lacZ fusion have shown that ndh expression is enhanced two - to threefold under anaerobic conditions in the absence of FNR , rather than reverting to the FNR-independent aerobic level ( Spiro et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Under anaerobic conditions it is likely that the opposing activities of FNR ( - ) and Arr ( + ) compete in order to ` fine-tune ' ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Thus , under some anaerobic-growth conditions Arr may compete with FNR and IHF for occupation of the ndh promoter and , by acting positively , Arr may counteract the negative effects of FNR and IHF to ` finetune ' ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>An in vivo study using a fnr-IucZ translational fusion suggested that fnr is expressed under both aerobic and anaerobic-growth conditions but that its expression is repressed by glucose or hyper-expression of FNR particularly during anaerobic-growth [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>As nikABCDE expression is very low in the absence of FNR and rcnA activation is identical under aerobic and anaerobic-growth conditions , nickel ions are probably entering the cell via another transport pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Involvement of both CRP and FNR in regulation of clyA expression during anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>When the gatPI sequence from - 3 0 to - 5 4 was replaced by the corresponding nirB sequence , expression from gatPI became inducibie by FNR under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>However , expression of these genes was upregulated under anaerobic-growth conditions in the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>However , the data for the parental and fnr cultures suggested that even under aerobic-growth conditions , some active FNR was pres-ent in the bacteria , and that even this small amount of active protein ( relative to anaerobic conditions ) was sufﬁcient to interfere with CRP-mediated hlyE expression ( Fig. 1c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Under anaerobic conditions FNR is a dimeric iron-sulfur protein that binds at specific DNA sequences to activate the expression of `` anaerobic genes '' and to repress the expression of `` aerobic genes '' ( 7 , 8 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>Under anaerobic conditions in the absence of nitrate the FNR protein induces fdnG operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>The suggestion that the acid-responsive yﬁD expression is independent of FNR was supported by Western blot analyses of cell-free extracts of MC1000 ( pGS1000 ) ( fnr + ) and JRG1728 ( pGS1000 ) ( ∆ fnr ) prepared after aerobic and anaerobic-growth at pH 7 ± 2 in the presence or absence of benzoate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>To identify the global changes and adjustments of gene expression patterns that facilitate a transition from aerobic to anaerobic-growth conditions , and to determine the effects of genotype on these gene expression patterns , we analyzed E. coli gene expression profiles obtained from cells cultured under aerobic or anaerobic-growth conditions ( O2 or O2 ) and under anaerobic-growth conditions in the presence or absence of the global regulatory protein for anaerobic metabolism , FNR ( O2 , FNR or O2 , FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>To identify the genes controlled by FNR , we analyzed gene expression profiles of cells cultured under anaerobic-growth conditions in the presence or absence of FNR ( O , 2 FNR or O , FNR ) in otherwise isogenic strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>To verify that fnrS transcription is indeed controlled by FNR , we analyzed by Northern blot the expression profile of this sRNA in cells cultured under aerobic or anaerobic-growth conditions ( O2 or O2 ) and under anaerobic-growth conditions in the presence or absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>aeg-46 .5 is known to respond to the presence of FNR for anaerobic expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>Expression of yecI , yjhB and smf was not affected by the presence or absence of FNR under anaerobic conditions ( Table 2 ) , although expression of both yecI and smf was apparently lower in the absence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Genome-wide expression analysis has previously shown that FNR contributes to the repression of a number of genes induced by oxidative-stress conditions [ 16,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>It is suggested that tandem FNR half-sites may be a feature of other genes that are active under anaerobic conditions and use both c54 - and c70-dependent promoters for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Group III : genes whose expression is decreased under an-aerobic conditions and whose repression is partially FNR dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group I : genes whose expression is increased under anaerobic conditions and whose activation is FNR dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group VIII : genes whose expression is increased under an-aerobic conditions and whose activation is FNR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Simply avoiding the use of the repressing carbon source , glucose , allowed the FNR dependence of the nrf and nap promoters to be demonstrated , even under growth conditions that were suboptimal for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>We ﬁnd that the major regulator of yﬁD expression is FNR , but that yﬁD expression is induced at the level of transcription during growth under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>It was therefore proposed that one or both of these novel poly-peptides might be involved in stimulating ndh promoter activity in bacteria grown in rich medium , and might also normally compete with FNR under anaerobic conditions in order to fine-tune ndh expression to match prevailing requirements for cellular energy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Effect of progressive 59 deletions on FNR - and Fismediated regulation of ndh promoter activity under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>Results from experiments using a low-copy plasmid-borne clyA promoter-lacZ fusion were consistent with the idea that both CRP and FNR are involved in the transcriptional regulation of clyA under anaerobic-growth conditions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Because the function of the PDH complex is adopted by pyruvate formate-lyase ( PFL ) under anaerobic conditions , it was anticipated that the two operons might exhibit reciprocal responses to the redox-responsive regulators , ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Thus , ndh expression was rapidly and strongly enhanced in the anaerobic to aerobic transition , which could be explained by ArcA , FNR and PdhR all repressing ndh expression under the initial anaerobic conditions followed by rapid de-repression upon introduction of oxygen , which inactivates ArcA and FNR , and causes pyruvate accumulation , by inhibition of pyruvate for-mate-lyase [ 20 ] , to inactivate PdhR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>In looking at the top 500 genes from each group , 48 genes were identified as being subject solely to ArcA regulation and 57 solely to FNR regulation under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Moreover , the 378 genes in the ArcA grouping and the 369 genes in the FNR grouping that do not show regulation by oxygen , but that are regulated by each of these proteins ( or co-regulated ) under anaerobic conditions , suggest that these two proteins may also be important for adaptation to the anaerobic environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>These pattems were essentiaiiy the same under aerobic and anaerobic conditions and they suggest that ArcA represses P/pa but not Ppat , or Pace - FNR appeared to activate Pp ^ h and repress P ^ ce under anaerobic conditions , but these obsen/ations are difficult to interpret since the effects of the fnr mutation were B Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>ArcA and FNR activities during transitions between aerobic and anaerobic conditions , and repression of ndh transcription by ArcA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>FNR and ArcA are involved in sensing the redox status of the cell to regulate genes involved in respiration under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>This further supports our previous suggestion ( 1 ) that the FNR , Lrp , and now ArcA regulons reveal overlapping functions under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>To explain the peak of cyd expression observed under microaerobic conditions , it was hypothesized that the two regulatory proteins act in competition at the cyd promoter and that under strictly anaerobic conditions the relatively weakly acting FNR protein out-competes the ArcA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Induction was dependent on the transcriptional-regulator ArcA , with a second transcriptional-regulator , FNR , playing a negative role speciﬁcally under anaerobic conditions ( S. Darie and R. P. Gunsalus , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550494" target="_blank">8550494</a>]<br></li>
<li>Other than its effect on the rpoS regulon , CsgD expression appears to regulate a subset of anaerobiosis-dependent genes , such as ompW and pﬂB , both induced in anaerobic conditions by FNR , either alone or in concert with ArcA ( 18 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>FnrS is involved in gene regulation after the shift from aerobic to anaerobic conditions , and its expression is activated by the transcription factors FNR and ArcA ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>In this same report , FNR and IHF were reported to have roles as weak repressors under anaerobic conditions , and it was suggested that the effect of FNR may be indirect as a consequence of its role in regulating ArcA expression ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>For example , it was reported recently that ArcA and FNR repressed the expression of puuA when E. coli was grown under anaerobic conditions and that there are putative binding motifs of ArcA and FNR upstream of the open reading frame of puuA ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18495664" target="_blank">18495664</a>]<br></li>
<li>Rather , hya expression in S. enterica is regulated not by IscR or ArcA but , rather , by FNR under anaerobic conditions ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>is repressed by FNR and stimulated by ArcA under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Expression of cyd is thus stimulated by ArcA under microaerobic conditions and repressed by FNR unde anaerobic conditions , resulting in a maximal expression under microaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>FNR appeared not to be responsible for the weak anaerobic repression of the PDH complex under normal conditions , but the severe repression exerted aerobically in the multicopy situation suggests that it is capable of exerting a negative regulatory effect at the FNR site in the pdh promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Surprisingly , repression of fnr : : lacZ was shown to be further enhanced when fnr was expressed from a multicopy plasmid ( 44 , 54 ) , suggesting that repression is limited by FNR levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Furthermore , it is conﬁrmed that both FNR sites contribute to FNR-mediated repression under anae-robic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>Under anaerobic conditions the effects of FNR and Arr predominate and mask any Fis-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>These investigations conﬁrmed that under anaerobic conditions both FNR sites contribute to FNR-mediated repression ( Fig. 2d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>Third , under anaerobic conditions both FNR sites within the ndh promoter contribute to FNR-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>FNR activates transcription of many genes in response to anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>FNR acts as a dual transcription regulator that activates certain genes required for anaerobic-growth and represses many genes required for aerobic-growth [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>FNR activates transcription of a number of genes coding for enzymes required under anaerobic-growth conditions and represses transcription of some genes for aerobic functions ( 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Although much progress has been achieved in understanding what limits FNR function to anaerobic-growth conditions ( Kiley and Beinert , 1999 ) , the mechanism by which FNR regulates transcription initiation of its target genes is not as well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>FNR is selectively active as a transcription factor under anaerobic-growth conditions , where it has been shown to control the transcription of hundreds of genes , many of which are necessary for adaptation to O2-limiting growth conditions ( 10 , 16 , 25 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In vivo transcription studies using promoter fusions with 59 deletions conﬁrmed that both FNR sites are required for maximum repression under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>The transcription of hlyE in the hns strain under anaerobic conditions in the presence of glucose is presumably driven by FNR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In anaerobic conditions , FNR binds to its target site , but transcription activation is repressed by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Under anaerobic conditions , FNR binds to a site centred between positions 241 and 242 , activating transcription of the nir operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Here it is shown that the oxygen-responsive transcription factor FNR regulates PhypA under anaerobic conditions only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Yet , FNR is only functional as a transcription factor under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Under anaerobic conditions FNR activated hyp transcription by approximately seven-fold , and v29FNR had no e ¡ ect ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Moreover , transcription from the c70-dependent promoter is activated by FNR under anaerobic conditions [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Measurement of aerobic activation from pFF-41 .5 in the presence of chromosomally encoded FNR suggests that FNR is on average 2.5 % active under aerobic conditions [ 150 8.6 Miller units ( MU ) under aerobic conditions and 6,177 343 MU under anaerobic conditions ; Fig. 1 , labeled cFNR ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>This might apply in particular under anaerobic conditions when slow activation by the artiﬁcial reaction might be sufﬁcient to retain FNR in the active state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>It has recently been shown that , under anaerobic-growth conditions , Fpg activity increases in strains deﬁcient in the global regulators Fur , FNR , and ArcA ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>It should also be pointed out that it has not yet been determined how much FNR needs to be activated under anaerobic-growth conditions to see an increase in FNR activity , i.e. , what fraction of the FNR pool actually becomes activated under these shift conditions is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The reduced uptake rates in an fnr background and under aerobic conditions suggest that the synthesis is under ( direct or indirect ) transcriptional control of FNR , the transcriptional-regulator of anaerobic respiratoxy genes ( reviewed in references 25 and 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></li>
<li>ldhA is one of the shortest fermentation genes ( 991 nucleotides ) , it is not under control of FNR , and its product is present both under aerobic and anaerobic conditions ( Linch and Lin , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>However , in anaerobically grown strains lacking FNR , expression of these genes was increased , suggesting that anaerobic expression of genes in group V is repressed by FNR and that the increase in expression under anaerobic conditions must be from another factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>A large variety of FNR regulated genes are known which are transcriptionally activated or repressed under anaerobic conditions ŽTable 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Group II : genes whose expression appears to be repressed by FNR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group V : genes that are slightly upregulated under anaerobic conditions but repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group VII : genes that are repressed under anaerobic conditions independently of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Genes required for anaerobic metabolism are regulated positively by FNR under anoxic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Previous approaches have revealed about 70 genes that are regulated by FNR or ArcAB under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Group IV : genes that are upregulated by FNR under anaerobic conditions but are only somewhat O2 regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>This is in agreement with the active role of FNR in gene regulation under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>The NADH dehydrogenase II gene -LCB- ndh ) Is repressed by FNR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Expression of the ndh gene , which encodes NADH dh II , is repressed by FNR under anaerobic conditions ( 15 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730262" target="_blank">7730262</a>]<br></li>
<li>Expression of the nuoE gene ( Fig. 6 ) is 3.8-fold lower under anaerobic conditions and elevated 4.5-fold in the FNR mutant ( Table IX ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Expression of the fumB gene is activated under anaerobic conditions by FNR ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>This suggests that the Isc machinery can meet an increase in Fe-S demand under anaerobic conditions such that no competition between FNR and IscR was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This suggests a reactivation of O2-inactivated FNR in E. coli under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Purification of FNR from cell extracts was carried out under anaerobic conditions as described ( 3 ) except that DTT was added to the purification buffer to a final concentration of 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>However , no specific retardation of DNA by FNR oouki be observed under aerobic or anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>It is tempting to speculate that these results reveal common functions between FNR and Lrp under aerobic or anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>FNR regulates transition between anaerobic and aerobic conditions [ 14,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Aerobic or anaerobic culture conditions were achieved by sparging cells as described in `` Assay of FNR activity in cells '' above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>DcuS is constitutively expressed but induced 2-fold under anaerobic conditions by FNR ( 10 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22101843" target="_blank">22101843</a>]<br></li>
<li>A primary role for FNR is to coordinate carbon and energy metabolism during growth under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>B. Under anaerobic conditions in rich medium and in the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>FNR and IHF acted as weak repressors under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>FNR and NifA act as positive regulators under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>When the terminal electron acceptor , O2 , is no longer available , FNR represses the synthesis of enzymes involved in aerobic respiration and increases the synth-esis of enzymes required for anaerobic respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The active ( anaerobic ) and inactive ( aerobic ) forms of FNR are readily interconverted in vivo depending on oxygen and iron availability ( Engel et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The inactive form from aerobically growing cells is converted reversibly to the active ( anaerobic ) FNR in the absence of O2 and vice versa ( Engel et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>In response to O2 availability , FNR is converted reversibly from the aerobic ( inactive ) state to the anaerobic ( active ) state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Therefore , it is assumed that by the alkylation assay two functionally different forms of FNR are recognized , an aerobic , or metal depleted , inactive form and an anaerobic active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Anoxic conditions FNR → FNR inactive ← active Oxic conditions In vivo , FNR can be inactivated not only by O2 , but also by a positive redox potential ( Unden et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>As shown below , the properties of O2 diffusion to the cytoplasm are consistent with the presence of O2 in the cytoplasm under aerobic and microaerobic conditions , which is a prerequisite for direct interaction between FNR and O2 ( Becker et al. , 1996 ; Unden et al. , 1997 ; Tseng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Overexpression and puriﬁcation of FNR protein , containing the DA154 substitution , which renders FNR active under aerobic conditions , was as described in Wing et al. ( 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Mutants of this type produce FNR protein that is partially active also under oxic conditions ( Kiley and Reznikoff 1991 ; Lazazzera et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>This can be concluded from the fact that iron limitation of the cells is not obligatory for the switch and that Fe availability under physiological conditions is not related to the functional state of FNR ( Niehaus et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Therefore FNR function has to be controlled by changes in the functional state , and it has been shown that inactive FNR from aerobically grown E. coli can be converted under anaerobic conditions in vivo to the functional state without the need for protein synthesis ( Engel et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>In contrast , for the hyaA and hybO promoters , IscR repressed the previously identiﬁed O2-regulated transcription start sites ; under anaerobic-growth conditions , AppY and the ArcA/B two-component regulatory system have been implicated in the activation of transcription of the hya operon ( Brondsted and Atlung , 1994 ; Richard et al. , 1999 ) , whereas FNR appears to contribute to activation of the hybO promoter ( Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>During aerobic-growth , transcription from the nir and nrf promoters is repressed , but expression is induced during anaerobiosis due to activation by the anaerobically regulated transcription factor FNR ( Jayaraman et al. 1989 ; Bell et al. 1990 ; Page et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>FNR has been known to respond to th intracellular [ Fe / Fe ] ratio under strictly anaerobic 2 + 3 + condition , whereas Arc system has more signiﬁcant effect on the cell growth in microaerobic condition ( Drapal and Sawers , 1995 ; Gunsalus , 1992 ; Iuchi et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
<li>Strains unable to synthesize the global transcriptional-regulator FNR ( Guest et al. , 1996 ) were also defective in the synthesis of both hydrogenase isoenzymes ( Sawers et al. , 1985 ) , strongly suggesting that FNR was responsible for anaerobic induction of enzyme synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Introduction The anaerobic respiratory systems of tschenchia coii are only expressed under anaerobtc conditions - The anaerobic dercpression of the corresponding genes is mediated by the FNR protein , which ts thought to function as an anaerobic transcriptional-activator ( Lambden and Guest , 1976 , Newman and Colo , 1978 : Chippaux et al. , 1978 ; Shaw ef al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>The mod operon is transcribed under both oxic and anoxic conditions and does no appear to be subject to control by FNR protein or Arc proteins , global regulators of anaerobic respiratory pathways ( Lynch and Lin 1996a ; Rosentel et al. 1995 ; Spiro and Guest 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The promoter , which has an FNR box in its vicinity , is activated some 20-fold by anaerobic-growth ( Fig. 2 ; Arai et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The uncertainty arises because hmp expression is repressed by FNR , and this repression is relieved during anaerobic-growth under conditions of severe nitrosative stress ( Table 1 ; Cruz-Ramos et al. , 2002 ; Corker & Poole , 2003 ; Pullan et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>The repression of ndhll by FNR occurs only under anaerobic conditions ( Spiro et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>FNR shows no homology to regulators of two-component-regulatory systems and no phosphorylation , neither under aerobic nor anaerobic conditions , could be demonstrated ( Green et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Other factors , such as ArcAB ( regulator of anaerobic and aerobic metabolism inhibiting the Krebs cycle , the electron transport chain and PDH activity in anaerobic conditions ) , activate the PFL enzyme ( De Graef et al. , 1999 ) , which is also regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>We supposed that FNR was responsible for the effect in the anaerobic condition , because FNR was the global transcriptional-regulator that controlled gene expression in response to the transition between aerobic and anaerobic-growth ( Green et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_843">
During prep-aration of this paper , Salmon et al. ( 73 ) reported results of global gene expression proﬁles of E. coli K-12 strain MC4100 grown under aerobic and anaerobic conditions and the effect of FNR on these global gene expression patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></label>
<input id="act_similarity_843", type="checkbox">
<div class="ACT_similar">
<li>During preparation of this paper , Salmon et al. ( 73 ) reported results of global gene expression proﬁles of E. coli K-12 strain MC4100 grown under aerobic and anaerobic conditions and the effect of FNR on these global gene expression patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li></div>
<li>Thus , expression of ttdA requires transcriptional activation by FNR , which is in the active state only under anaerobic conditions ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine metabolism ) was controlled by protein regulators such as FNR ( transcriptional-regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>However , in the absence of FNR , ndh expressionis enhancedby anaerobicgrowth in mediacontaining amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>were associated with the rich medium - or amino acid-dependent anaerobic activation of the ndh promoter in the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>In the absence of both FNR and NsrR , ytfE promoter activity was signiﬁcantly higher in anaerobic cultures , but the underlying mechanism is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>It has also been suggested that the anaerobic activation of FNR could involve reduction of the bound iron or the anaerobic incorporation of iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>A considerable amount remains to be learnt about the mechanism of anaerobic activation of FNR by anoxia in E. coli , and in non-enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Expression of a caiF-lacZ operon fusion was subject to FNR regulator-mediated anaerobic induction and cyclic AMP receptor protein activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>receptor protein ( CRP ) or catabolite gene activator protein ( CAP ) of E. coil , and the homology suggested that FNR performs a role in anaerobic gene activation analogous to that of CRP in catabolite gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The importance of the FNR protein as a transcriptional-regulator has been further extended to include possible rotes in both aerobic and anaerobic gene activation and repression , as a result of a detailed two-dirnensionai gel electrophoretic analysis of the proteins of aerobtcally and anaerobicady grown fnr ' and fnr strains ( Sawers ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>A mutation in the fnr gene , however , only reduces anaerobic induction fivefold , indicating that FNR is not the only factor involved in the anaerobic activation process ( Sawers and Bock , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Here , we report an anaerobic regulatory mechanism of hmp induction directly mediated by NO , involving nitrosylation of the [ 4Fe ± 4S ] 2 + clusters in the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The FNR protein that regulates transitions between aerobic and anaerobic-growth is inactivated as a consequence of nitrosative damage , but we challenge the idea that FNR is a physiologically relevant sensor of NO , except under the most severe nitrosative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>The EtrA protein is functionally as well as structurally similar to FNR , given that its major role appears to be in the regulation of anaerobic respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Further studies with acn-lac2 gene fusions and a wider range of transcription regulators indicated that acnA expression is initiated by G ~ ~ from PIacnAa , nd from PZacnA it is activated directly or indirectly by CRP , FruR , Fur and SoxRS , and repressed by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>This showed that the acnB gene is subject to repression by ArcA , Fis and FruR and activation by CRP , whereas the a c n A gene is repressed by ArcA and FNR , activated by FruR , CRP , Fur and SoxRS , and initiated by a38in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Both genes are subject to ArcA-mediated anaerobicrepression and CRP-mediated catabolite repression , but they differ insofar as a c n A gene transcription is activated by Fur , SoxRS and FruR , can be initiated by t ~ a ~ d ~ is r , epressed by FNR , whereas the acnB gene is repressed by Fis and FruR but unaffected by the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Expressionfrom the suc promoter was repressed by IHF and partially activated by d8but it was not regulated by ArcA , FNR , CRP , FruR or Fis , and not repressed by glucose or anaerobiosis , indicating that the well-established catabolite and anaerobic repression of ODH synthesis is imposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Incontrast , the / pel promoter was repressed by both glucose ( via a CRP-independentmechanism ) and anaerobiosis ( mediated by ArcA ) , and activated by Fis , but it was not regulated by FNR , FruR , IHF or These observations support the view that transcription of the sucABCD genes is primarily initiated and regulated at the upstream sdh promoter , and that the Ipd promoter is independently co-regulated with Pdh ( primarily by ArcAmediated repression ) rather than with PsucDirect evidence for co-transcription of the entire sdhCDAksucABCD region from PHNw t as obtained by detecting a 10 kb transcript inm c and me mutants , but not inthe parental strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>In contrast , inactivation of the fnr gene had no significant effect on expression of the ZpdA-lac2 fusion , but expression was repressed by the multicopy fnr + plasmid , suggesting that excess FNR can either invade the promoter and impair transcription or alternatively function indirectly by activating ArcA synthesis ( Compan & Touati , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>It is now apparent from studies with the corresponding l a c 2 fusions that the fumA promoter resembles PacnBin being anaerobically repressed by ArcA , whereas the fumC promoter resembles P2acnAin its dual regulation by ArcA and FNR , although there is evidence that fumC transcription is partially facilitated by readthrough from the upstream fumA gene promoter ( Woods & Guest , 1987 ; Guest & Russell , 1992 ; Gruer & Guest , 1994 ; Park & Gunsalus , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The effects of the global regulators ( ArcA , FNR and CRP ) on the pdh , ace and Ipd promoters were investigated in wiid-type and mutant strains containing singie-copy X. / acZ-fusion prophages in the presence and absence of the corresponding multicopy regulator piasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>i219 ) was generated by cloning the 1.3 kb Pst \ - Xho \ \ fragment of pGS81 into pGS1406 , For testing the effects of global regulators ( ArcA , FNR and CRP ) , monoiysogenic derivatives of the corresponding Alac parental , mutant , and plasm id-transformed mutant , containing each of the pdhR - , aceE - and / pd - / acZfusion phages , were constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Thus , ArcA and FNR regulate respiration in response to oxygen availability ( the variable in these experiments ) , CRP and Lrp sense nutritional state via cAMP and L-leucine concentrations , and Fis , H-NS and IHF modify DNA topology , which is itself dependent on energy status [ 7,25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , it is concluded that ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp are truly global regulators , at least as far as the response to oxygen availability is concerned , in the sense that their outputs create the transcriptional framework upon which local regulators operate in response to specific unstable environments Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The influence of CRP on the transcriptome progressively increased with increased aerobiosis , whereas the activities of ArcA , FNR , IHF and Lrp decreased as aerobiosis increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Many of the transcriptional changes observed were brought about by global TFs such as ArcA , FNR , CRP , IHF , pdhR and fur in ATPase and NOX ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>By these criteria , ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp were designated as global TFs in E. coli K-12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Transcription of the glp operons is subject to multiple controls , including catabolite repression mediated by cyclic AMP-CRP and respiratory control mediated by the FNR and ArcA/ArcB systems ( 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>In contrast , strong responses to anaerobiosis and glucose ( a source of pyruvate ) mediated by global regulators ( ArcA , FNR and CRP ) are likely to be less beneficial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>For ArcA and FNR , the cultures were grown anaerobically to A650 = 0.4 in L broth + glucose ( 11 mfui ) , and for CRP , cultures were grown aerobically to ASM = 0.8 In L broth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>This situation exactly parallels that of fumarate reductase and succinate dehydrogenase , in which the corresponding frdABCD and sdhCDAB operons are subject to the same differential regulation by FNR in one case and the cyclic AMP-CRP complex plus the anaerobic repressor in the other , and the enzyme complexes are specifically adapted to accept electrons from menaquinione in fumarate respiration ( fumarate reductase ) or to donate electrons to ubiquinone in aerobic respiration ( succinate dehydrogenase ) ( 11 , 17 , 24 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>Transcriptional monocopy lacZ fusion studies showed that both operons are coexpressed during anaerobic-growth in the presence of carnitine , respond to common environmental stimuli ( like glucose and nitrate ) , and are modulated positively by the same general regulators , CRP and FNR , and negatively by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Like cai and ﬁx operons , the caiF gene appeared to be positively controlled by CRP and FNR and negatively controlled by H-NS and the NarL ( plus nitrate ) regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>We thank C. Park , B. Wanner , D. LaPorte , S. Maloy , J. Cronan , Jr. , R. W. Simons , and H. Aiba for strains ; R. W. Simons for plasmids and phage ; H. Wing for puriﬁed FNR DA154 ; N. J. Savery for puriﬁed CRP ; G. S. Lloyd for help performing the CRP gel shifts ; and H. M. Wols for construction of strain AJW1884 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Furthermore , no effect of CRP was found in vivo by either mutation of the CRP binding site or use of strains that lacked CRP ( data not shown ) , indicating that under the conditions tested , CRP does not regulate the FNR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Thus , these data suggest that under such conditions , FNR and CRP are in competition for binding at PhlyE , and that the simplest explanation of these observations is that FNR is more efﬁciently bound than CRP at PhlyE but is not as effective as CRP in activating hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>This suggests that during anaerobic-growth in liquid cultures in the absence of glucose , FNR is the major regulator of hlyE expression , and that CRP-mediated hlyE expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The ﬁndings that SlyA does not enhance hlyE expression in hns cultures grown in liquid medium under anaerobic conditions and that SlyA appears unable to activate hlyE expression in the absence of FNR and CRP and the determination of the location of the region of PhlyE occupied by SlyA ( Fig. 6 ) suggest that SlyA activates hlyE expression by antagonizing the negative action of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Evidently , CRP , and not FNR , was required for the expression of ClyA in hns mutants under anoxic conditions , although it appeared that FNR could partly complement CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Under aerobic conditions with excess glucose , control of sdhCDAB -- b0725 -- sucABCD transcription by Arc , FNR , FruR and cAMP-CRP will play a minor role and , thus , reduced expression of this operon during glucose overflow metab-olism is likely due to the control by the cAMP-CRP independent catabolite control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Therefore , we suggest that when environmental conditions dictate a low intra cellular concentration of H-NS , the H-NS I region of PhlyE is occupied to modulate the activity of the upstream activator ( FNR or CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>This positive effect of H-NS was not observed under anaerobic conditions , presumably because FNR and not CRP occupies PhlyE under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Both FNR and CRP make positive contributions , whereas H-NS acts negatively under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>For these conditions , we predicted that the glucose-responsive effects of CRP would be retained , whereas the positive regulation by FNR would be abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>However , rather than the approximately eightfold enhancement in hlyE expression observed in an hns mutant on solid medium ( 36 ) , in liquid culture H-NS has a positive effect on CRP-driven hlyE expression , a negative effect on FNR-driven hlyE expression , and little intrinsic regulatory activity in the absence of FNR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , CRP enhances hlyE expression in response to glucose-starvation ( 36 ) and FNR enhances hlyE expression in response to oxygen starvation ( 10 , 11 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In summary , the data presented here suggest that FNR and CRP are positive regulators of hlyE expression in liquid culture in response to oxygen and glucose-starvation , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>These data suggested that hlyE expression is activated by FNR and CRP , repressed by H-NS , and that H-NS-mediated transcriptional silencing is relieved by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>By competing with H-NS for the region of PhlyE downstream of the FNR/CRP site , SlyA may prevent the formation of the negatively acting H-NS -- PhlyE complexes by blocking a primary interaction between H-NS and PhlyE , allowing FNR and CRP to continue to activate hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , we concluded that CRP , FNR , and H-NS all contribute towards the regulation of hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , CRP and FNR positively control cai expression and H-NS exerts a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Thus , we suggest that the formation of different H-NS -- PhlyE complexes offers the opportunity for H-NS , in combination with FNR and CRP , to control hlyE expression both positively and negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>These observations may be explained by a mechanism in which FNR recognizes PhlyE efﬁciently but acts only as a relatively poor activator of hlyE transcription ( 10 , 11 ) , whereas , conversely , the recognition of PhlyE by CRP is poor , but once CRP is bound , CRP-mediated activation of hlyE expression is efﬁcient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>FNR is also required to mediate narK activation in response to nitrate as evidenced by the degree of narK-lacZ expression in aftnr strain ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>We hypothesized that the residual FNR protein-indepen-dent expression of the CC fusion construct was due to activation by the CRP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>We also dem-onstrate the role of CRP in the direct activation of the ulaAB-CDEF operon and show that ula regulon expression was not affected in an FNR mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Whether FNR binds at the vicinity of CRP and acts in competition with CRP or on its own will require a more thorough investigation of the 59 regulatory region of caiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>These are CRP , repressing several promoters , and the activators FNR , MetR , PapB , and IlvY , repressing their own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Our ﬁndings suggest that FNR and CRP bind to the same sequence in the clyA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Thus FNR could be converted such as to-bind specifically to CRP sites , CRP to-bind to FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The roles of the global transcriptional-regulators , ArcA , FNR , CRP , and RpoS , in the regulation of dctA transcription were investigated by studying the appropriate regulatory mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>CRP and FixK can act like FNR as positive and negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Therefore , sdhE does not appear to be subject to transcriptional regulation via ArcAB , FNR or CRP when it is co-transcribed with YgfX in the Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>This regulation occurs , in part , through the actions of the transcriptional-regulators CRP , FNR , IclR , and FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Anaerobic regulation of clyA involves both CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>As evidenced by the lysis of erythrocytes in agar ( Table 3 ) , the absence of CRP , but not of FNR , reduced the level of clyA derepression in anaerobically grown hns strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>This is consistent with our observation that in the absence of CRP and FNR , addition of SlyA only leads to partial derepression of the hlyE promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Table 6 shows however that , surprisingly , this promoter is not co-dependent as are CC-FF and FF-FF : in the absence of FNR substantial CRP-dependent expression is found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Expressionof the E. coil ansB promoter,CC-FF , in the presenceand absence of CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The use of similar sensing mechanisms by structurally unrelated transcription factors , such as OxyR and SoxR , suggests that there may be a limited number of biologically effective solutions to the problem of setting transcriptional switches within specific redox windows The conformational changes induced in CRP , FNR and FNR-like proteins upon perception of the appropriate signal ` switch ' the regulators from their inactive to active forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>activate transcription ; in particular , two FNR dimers are able to function co-dependently in tandem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This is surprising because it was expected that tandem-bound FNR dimers would act synergistically to activate transcription , as has been reported for the closely related transcription factor CRP ( Savery et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Thus the global regulators are proteins that control responses to changing food levels and carbon degradation ( mlc , lrp , CRP ) and transcription factors that respond to changes in redox status or ion levels of the cell ( arcA , narL , FNR , fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Therefore , even though CRP and FNR share the same structural framework , there are major differences in the way they interact with CZCTD , such that different contacts are used to modulate the extent of transcription activation at specific promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>However , we propose that , although NarP and NarL bind as dimers , transcription activation requires activity in just one of the subunits of the dimer ( this has been shown for both CRP and FNR : Zhou etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Transcription from galP1 is stimulated by CRP , ynfE is FNR dependent and yeaR transcription requires NarL [ 25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23341772" target="_blank">23341772</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535077" target="_blank">12535077</a>]<br></li>
<li>Transcription depends upon CRP , the anaerobically triggered transcription activator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>These seemingly include direct interactions by CRP and FNR to activate transcription initiation and indirect effects by acetate metabolic enzymes and transcription factors that control carbon ﬂux .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Specific sequences to which the CRP and FNR proteins bind , to stimulate transcription , have been proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>694 J. K. Lithgow , F. Haider , I. S. Roberts and J. Green effect on hlyE transcription in vivo in the absence of H-NS , and active CRP and FNR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Effects of FNR , CRP , and H-NS on in vivo transcription of hlyE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Can primary and secondary consequences of their formation be distinguished , for example , at the level of transcription control by proteins such as the NO-sensitive repressor , NsrR , or the regulator of fumarate and nitrite reduction , FNR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>In this work , binding of the pleiotropic FNR ( fumarate and nitrate reduction ) transcriptional-regulator to the nrdDG promoter region and the effects of binding on transcription were investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>NarL controls the expression of many proteins involved in anaerobic respiration and , at most of its known promoter targets , it co-regulates transcription together with another global transcription factor , protein FNR , the ` master ' regulator of anaerobic adaptation [ 2,8 -- 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Transcription of this operon is driven from a single promoter ( pnir ) and is induced by the fumarate and nitrate reduction ( FNR ) protein , a global transcription regulator that activates the expression of many genes in response to oxygen starvation ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Crack JC , Le Brun NE , Thomson AJ , Green J , Jervis AJ ( 2008 ) Reactions of nitric oxide and oxygen with FNR ( regulator of fumarate and nitrate reduction ) , a global transcriptional-regulator , during anaerobic-growth of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Abbreviations : dO2 , dissolved oxygen tension ; FNR , fumarate and nitrate reduction regulator ; GST , glutathione S-transferase ; PFL , pyruvate transcription factors is FNR ( fumarate and nitrate reduction regulator ) , a direct oxygen sensor that functions mainly to activate the expression of genes required during anaerobic-growth ( Kiley & Beinert , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>To determine whether the effects of FNR and NarL on hcp transcription are direct or indirect ( for example , due to inactivation of NsrR by nitrite or nitric oxide generated during anaerobic-growth ) , the hcp promoter fragment from pNF383 was incubated with different concentrations of puriﬁed FNR and NarL proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>To determine whether the effects of FNR on cusC and csgD were direct , we constructed lacZ fusions to the region upstream of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>In this context it may be relevant that several genes , including sodA ( manganese superoxide dismutase ) , sdb ( succinate dehydrogenase ) , yo ( cytochrome o ) , y d ( cytochrome d ) and pfl ( pyruvate formate lyase ) , are regulated by both FNR and ArcA , and this dual action has recently been ascribed to the activation of a r c A transcription by FNR for sodA , sdb , and yo , but not for y d or pfl ( Compan & Touati , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8000525" target="_blank">8000525</a>]<br></li>
<li>Anaerobically , the intracellular concentration of active FNR becomes suficient to allow occupation of the FNRII site by displacing ArcA , and thereby inhibit the transcription activation mediated by the FNR dimer bound at FNR I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Anaerobically , the intracellular concentration of active FNR is now sufﬁcient to-bind at the FNR II site by displacing ArcA and thereby inhibit the transcription activation mediated by the FNR dimer bound at FNR I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>It was therefore possible that the lack of derepression of the hcp promoter by externally added NO was due to compensating effects of NO-acti-vated derepression by NsrR and loss of activation by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>Three of the obvious possible explanations for the minimal response of the hcp promoter to external NO are that derepression of NsrR was counter-balanced by loss of transcription activation by FNR ; that derepression of the NsrR regulon resulted in sufﬁcient capacity to repair nitrosative damage to FNR as rapidly as it occurred or that the capacity of the bacteria to reduce NO was sufﬁcient to prevent its cytoplasmic accumulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>In marked contrast , the corresponding enhancements were only five-fold ( relative to the wildtype ) for both altered promoters in the absence of FNR , indicating that Arr-mediated activation is severely impaired .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Repression by FNR has not been as well studied as activation and , as a consequence , not as many genes were known to be repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Expression of the genes for all of these proteins is dependent upon FNR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Several genes are known to be subject to dual FNR - and CRP-dependent activation ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In E. coli , both CAP and FNR are required for its full activation , and in Salmonella , two dimers of CAP are required .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10715008" target="_blank">10715008</a>]<br></li>
<li>[ 151 ] Blake , T. , Barnard , A. , Busby , S.J.W. and Green , J. ( 2002 ) Transcription activation by FNR : evidence for a functional activating region 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>As a result , modification with the unknown substituent could be related to the redox-dependent activation and inactivation of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Inactive FNR is monomeric and becomes dimeric after reductive activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>[ 2Fe -- 2S ] FNR is no longer active in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_950">
[ 23 ] Marshall , F.A. , Messenger , S.L. , Wyborn , N.R. , Guest , J.R. , Wing , H. , Busby , S.J.W. and Green , J. ( 2001 ) A novel promoter architecture for microaerobic activation by the anaerobic transcription factor FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></label>
<input id="act_similarity_950", type="checkbox">
<div class="ACT_similar">
<li>Marshall , F.A. , Messenger , S.L. , Wyborn , N.R. , Guest , J.R. , Wing , H. , Busby , S.J.W. and Green , J. ( 2001 ) A novel promoter architecture for microaerobic activation by the anaerobic transcription factor FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Marshall , F. A. , Messenger , S. L. , Wyborn , N. R. , Guest , J. R. , Wing , H. , Busby , S. J. W. , and Green , J. ( 2001 ) A novel promoter architecture for microaerobic activation by the anaerobic transcription factor FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11444864" target="_blank">11444864</a>]<br></li></div>
<li>If these palindromic boxes in the overlapping promoter are functioning in transcription in the narX direction , the activation effect of FNR should be also observed for transcription of narX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>It is therefore very likely that transcription from the nrdD promoter is stimulated directly by FNR upon its activation by a shift to anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Repression of narX transcription by FNR Analysis of the effect of FNR on transcriptional activation and repression was extended to the regulatory region for two other anaerobic operons , narX and narK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Mutations in the putative NarL-binding site at the nirB promoter decrease FNR-dependent anaerobic induction , suggesting that NarL acts as a helper to FNR during transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>These data indicate that both ArcA and FNR are essential for anaerobic activation of promoter 7 transcription , which suggests functional interaction between these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>A decrease in oxygen tension results in activation of FNR and upregulation of nikABCDE expression , while the presence of excess nickel activates NikR , which overrides the action of FNR and results in repression of nikABCDE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Since transcription activation by AraC or FNR requires sCTD only in the absence of their respective coactivators , CAP or NarL ( 25 ) , we tested the effect of some alanine substitutions in sCTD on uhpT-lacZ expression in the Dcrp strain TM37 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The interaction of FNR-AR1 and FNR-AR3 with RNAP is apparently altered by NarL and/or IHF Although some FNR-dependent promoters appear to use only FNR for transcription activation , other promoters , such as PnarG , require the additional factors NarL and IHF for maximal expression ( Rabin et al. , 1992 ; Stewart , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Thus , these data suggest that the regions of FNR disrupted by the FNR-AR1 and FNR-AR3 mutants can account for the majority of the transcription activation of PnarG and PdmsA in the absence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>However , it appears that the role of NarL at PnarG is more direct , as transcription activation of PnarG can not be detected in vitro , unless FNR , IHF and NarL are all present ( E. C. Ziegelhoffer and P. J. Kiley , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The molecular mechanism by which NarL , IHF and FNR function together to stimulate transcription activation at PnarG is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>In the absence of oxygen , the oxygen-sensitive global regulators ArcA and FNR mediate the repression of many TCA promoters , but most dramatically the sdh-suc operon , which encodes SDH , KDGH , and the SCSC ( 104 , 105 , 170 , 293 , 335 , 339 , 406 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Furthermore , the regulatory outcome is essentially reciprocal in that FNR usually activates transcription , whereas ArcA generally acts as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Other studies have shown that ArcA functions to activate cy-dAB transcription as oxygen becomes limiting ( 19 , 20 ) , whereas FNR is required for repression when the oxygen tension is decreased further ( 18 , 21 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Subsequent studies have shown that ArcA functions to anti-repress cydAB transcription when oxygen is limiting ( 46 ) , whereas FNR is required for repression when the oxygen tension is decreased further ( 14 , 17 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Although the roles of FNR and ArcA in mediating the changes in transcript profiles necessary for adaptation to growth at different oxygen availabilities are well established ( see above ) , the consequent changes in physiology are expected to modulate the activities of other transcription factors that respond to specific metabolic signals that are affected by oxygen availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>Example of interactions between direct and indirect effects on transcription of narG ( encoding the membrane-associated nitrate reductase ) and cyo ( encoding a cytochrome oxidase ) because of ArcA , FNR , nitrate , oxygen , and changes in the redox state of the ubiquinone : ubiquinol pool .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>There are no clear FNR or ArcA sites in front of dmlA , supporting the view that the effects of FNR , ArcA , and oxygen are indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>FNR SF73 has a relatively small effect , reducing activity by 1.5-fold ; FNR GA85 is far less effective in promoting transcription , expression being reduced by 3.7-fold relative to that with wild-type FNR These results are very similar to those obtained when transcription is activated by a single FNR dimer positioned at - 4 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Expression of yﬁD as monitored by the yﬁD : : lac reporter fusion pGS1000 in the presence or absence of the transcription factors FNR , ArcA and PdhR , and the effect of acidity on the activity of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>When iron becomes scarce , Fur becomes inactive , resulting in derepression of sodA transcription ; however , only under conditions in which FNR and ArcA are inactive ( i.e. in presence of O2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Consistent with this hypothesis , growth in the presence of pyruvate or under microaerobic conditions elevates the level of YﬁD ( 50 , 290 ) , while the pyruvate-sensitive regulator PdhR and the ox-ygen-sensitive global regulators FNR and ArcA control yﬁD transcription ( 290 , 476 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In vivo transcription studies using a ydhY -- lacZ fusion have shown that ydhY expression is FNR-dependent , and in vitro transcription reactions in the presence of FNR yield a product with a size consistent with the presence of a class II FNR-dependent promoter upstream of ydhY ( Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Tandemly linked FNR constructs display elevated transcriptional activity in the presence of O2 Following successful expression of the covalently linked tandem FNR , we measured in vivo transcription activity of this variant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Previous studies using an fdhF : : lacZ gene fusion confirm that its transcription is activated anaerobically in the presence of FNR ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>hmp gene expression in the presence of NOz is independent of the SoxS , OxyR , and FNR transcription factors and instead relies on inactivation of the iron-de-pendent Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>However , our data showing that NO does not regulate # rd suggest that the mechanism of repression exerted by the transcription factor FNR on the # rd operon is clearly di ¡ erent from that on the hmp gene , which is proposed to be upregulated by NO through FNR inhibition [ 17,23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>Furthermore , upon the addition of FNR , transcription of the narX promoter was significantly repressed ( Fig. 4A and B ) , 250 nM FNR exhibiting approximately 90 % repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>In addition to its transcriptional-activator function , FNR has been shown to repress thefir and n & r genes in vivo [ 2 ] , and Guest and colleagues [ 9 ] have reported repression of in vitro transcription of ndh in the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Transcripts of sufA or sufD were poorly detected in the in vitro transcription system , precluding a test of FNR in their repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>FNR also activates the transcription of the small non-coding RNA FnrS which negatively regulates the expression of multiple genes , including many that encode enzymes with functions linked to oxi-dative stress [ 26,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Residue K593 of 70 has been found to be important for several transcription activators , including AraC , UhpA , cI , FNR , Ada , RhaR ( this study ) , and RhaS ( 4 , 24 , 27 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Furthermore , the fnr mutation caused a 27-fold increase in the # rd / # rd-red transcription ( Table 2 , Fig. 1C ) , indicating that the transcription of # rd / # rd-red is negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>FNR is structurally and functionally hornologous t o CAP , the catabolite gene activator protein , and increased FNR production strongly stirnulates transcription of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>One of these is the FNR protein , which activates transcription of many anaerobically regulated genes ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>In this study , we examined the mechanisms that control transcription of fnr to increase our understanding of how FNR protein levels are regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In addition to the specific repression by the FNR protein , the significant decrease in transcription of the fnr and narX genes was also noticed even in the absence of FNR by truncation of respective upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>At low nickel concentrations in the absence of nitrate ( curves and diagrams I and II ) , FNR upregulates transcription from PnikABCDE and some component of the hydrogenase assembly pathway blocks NikR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>This effect is compounded by the fact that arcA transcription is directly activated by FNR ( 5 ) , which in turn is essential for oxidation of ubiquinol by the cytoplasmic or periplasmic nitrate reducta-ses , NarG or Nap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>In some of these cases , however , apparent regulation was based on promoter fusion data and might therefore be indirect , for example , due to an effect of FNR on arcA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>As concluded previously by Gruer & Guest ( 1994 ) , the effect of FNR is not simply due to its known ability to activate arcA transcription ( Compan & Touati , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Thus , nitrosylation of iron atoms , especially ironsulfur clusters , in enzymes such as aconitase and fumarase or in the transcription factors FNR , Fur , SoxR , OxyR , and NsrR inactivates their functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Whereas the effect on Fur and FNR caused transcription alterations in the genes under their control , that of SoxR was unexpected ; although a large increase upon the transcription of soxS occurred , a corresponding increase on the SoxRS regulon could not be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>Indeed , the homology between FNR and CRP is such that their DNA-binding specificities can be interconverted by appropriate substitutions in the recognition helices , such that catabolite repression can be lifted in response to anoxia and anaerobic gene activation can be induced by glucose-starvation ( cAMP ) ] 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional-regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Its expression is also dependent on the global regulatory proteins mediating catabolic repression via cyclic AMP receptor protein ( CRP ) , or anaerobiosis via fumarate nitrate reduction ( FNR ) transcriptional-regulator , and the nucleoid-associated DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Moreover , it is subject to global control by the same regulatory proteins which are known to modulate expression of carnitine metabolism , i.e. , the cyclic AMP ( cAMP ) receptor protein ( CRP ) which mediates the activation of catabolic operons , the transcriptional-regulator FNR that is responsible for anaerobic induction , and the DNA-binding protein H-NS ( 19 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The ula regulon also seems to be controlled by the global regulators cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and fumarate nitrate reductase regu - lator ( FNR ) , since mutations in the corresponding genes abol - ished or delayed , respectively , growth on L-ascorbate ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>PItYSIOLOGY VOL 44 ISBN 0-12-027744 - I Copyright © 2001 Academic Press All rights of reproduction in any tiwm reserve ABBREVIATIONS cAMP CAP CRP FLP FNR Adenosine 3 ' ,5 ' - monophosphate Catabo -RCB- ite gene activator protein cAMP receptor protein FNR-like protein fumarate and nitrate reduction regulator 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Abbreviations : CRP , cyclic AMP receptor protein ; Fis , factor for inversion stimulation ; FNR , transcription factor for fumarate-nitrate reduction anced production of GMP with that of AMP and with the pyrimidine nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Abbreviations : CRP , CAMPreceptor protein ; FNR , fumarate and nitrate reduction regulator ; IHF , integration host factor ; ODH , 2-oxoglutarate dehydrogenase ; PDH , pyruvate dehydrogenase ; SCS , succinyl-CoA synthetase ; SDH , succinate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Abbreviations : CRP , cAMP receptor protein ; FNR , fumarate and nitrate reduction regulator ; MALDI-TOF , matrix-associated laser desorption ionization-time of ﬂight ; MBP , maltose-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Here , we present evidence that this switch occurs primarily through the induction of acs and that the timing and magnitude of this induction depend , in part , on the direct action of the carbon regulator cyclic AMP receptor protein ( CRP ) and the oxygen regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>We also have found that the timing and/or magnitude of this induction depends , in part , on the carbon regulator cyclic AMP ( cAMP ) receptor protein ( CRP ) , the oxygen regulator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>FNR shows sequence and functional similarity to CRP , the cyclic AMP receptor protein ( BELLet al. 1989 ; SPIROand GUEST1987 ) , although the mechanism by which FNR senses ana-erobiosis is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>in the presence of acetate and is regulated by cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and FNR as well as by IclR and its activator , FadR ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>Regulation by cyclic AMP receptor protein and anerobiosis ( FNR protein ) was confirmed , and the presence of nucleotide sequence motifs , with homology to cyclic AMP receptor protein and FNR protein-binding sites , investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>It is structurally-related to CRP ( the cyclic AMP receptor protein ) except for the presence of a cysteine-rich / V-terminat extension , which may form part of an iron-binding , redox-sensing domain in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>2 The abbreviations used are : sRNA , small regulatory RNA ; ArcA , regulator of aerobic respiratory control ; CAP , the catabolite gene activator protein ; FNR , regulator of fumerate and nitrate reduction ; Fur , ferric uptake regulator ; TIR , translational initiation region ; IGR , intergenic regions ; SOD , superoxide dismutase ; GFP , green fluorescent protein ; fw , forward ; rv , reverse ; DTT , dithiothreitol ; Bis-Tris , 2 - [ bis ( 2-hydroxyethyl ) amino ] -2 - ( hydroxy-methyl ) propane-1 ,3 - diol ; CHAPS , 3 - [ ( 3-cholamidopropyl ) dimethylam-been identified , and a major class consists of trans-encoded gene regulatory RNAs that act by an antisense mechanism to activate or , more frequently , to repress translation of target mRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Anaerobic induction depended mainly on the transcriptional-regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>The induction was completely dependent on the transcriptional-activator FNR and was independent of the ArcA-ArcB two-component response-regulator system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>The same phenotype was also observed in an IHF + strain that lacked FNR , suggesting that IHF and FNR may work in consort to mediate pyruvate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The regulation of the nar operon has some aspects analogous to that reported here for the pfl operon , and it is tempting to speculate that pyruvate induction , like nitrate induction of the nar operon , is also mediated by a specific DNA-binding protein and that IHF functions as a coactivator facilitating interaction between the activator and either the FNR protein or RNA polymerase or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>To check directly for any involvement of FNR in regulating the ogt promoter , nitrate-dependent induction of the ogt : : lac fusion in strain JCB387 was compared with induction in a fnr derivative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Anaerobic induction is mediated by the FNR protein ( for review , reference a see 23 ) , while nitrate induction is mediated by the NARL protein ( 4 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Anaerobic induction was totally dependent upon a functional FNR protein , partially dependent on the nitrate-sensing two-component regulatory system NarXL , and independent of the alternative nitrate response protein NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>However , further nitrite - or nitrate-dependent induction is almost totally suppressed , suggesting that NarL-depen-dent activation by nitrite or nitrate is mediated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Transcription activation ofthenitratereductase structural gene operon ( narGHJI ) requires the FNR protein for induction by anaerobiosis and the NarL proteinforinduction by nitrate ( STEWART1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Common mechanismsfor regulatingfdnGHZ and narGHJZ expression : Previous work has shown that narGHJI operon expression requires FNR for induction by anaerobiosis , and NarL for induction by nitrate ( STEWART1982 ; LI and DEMOS 1988 ; reviewed by STEWART 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Although the intensity of qBBr ﬂuorescence was lower than that of mBBr ﬂuorescence , aerobically grown cells again showed a twofold increase in labeling of FNR compared to anaerobically grown bacteria ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>It now becomes apparent that the increase in the number of accessible Cys residues in FNR of aerobically growing E. coli is functionally relevant , but it has to be attrib-uted to the release of the FeS clusters and the formation of apoFNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>To calculate the speciﬁc content of labeled Cys residues in FNR , the amounts of bound mBBr and of FNR were determined by quantifying the ﬂuo-rescence and immunostaining in Western blots , as in Fig. 1 ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Labeling of the Cys residues of FNR by mBBr was a rapid process , and after 1 min of labeling , no further increase was observed ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>It can be assumed that this form of FNR is labeled at ﬁve Cys residues , indicating an approximate increase in Mr of 1,000 per molecule of bound AMS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Transcription of the NikABCDE nickel transporter , which is required for NiFe hydrogenase synthesis , was previously shown to be upregulated by FNR ( fumarate-nit rate regulator ) in the absence of oxygen and repressed by the NikR repressor in the presence of high extracellular nickel levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>This enhancement was independent of FNR and dissolved oxygen tension , because even in the fnr strain , JRG1728 , grown aerobically in the presence of benzoate , the intracellular content of YﬁD increased C fourfold compared to cultures grown in the absence of benzoate ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Thus , changes in oxygen availability ( sensed directly by FNR , and indirectly by ArcA ) affect the redox state of the system , leading to consequences for metal ion homeostasis ( copper , molybdate , iron ; sensed by CusR , ModE and Fur ) , iron -- sulphur cluster turnover ( sensed by IscR ) , ribonucleotide reductase activity 20 0 31 0 10 aerobiosis ( % ) time ( min ) Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>These include SoxR , OxyR , FNR , MetR , and Fur ( 5 , 6 , 10 , 12 , 17 ) , though in each case the principal function of the regulator is to sense another signal ( superoxide , hydrogen peroxide , oxygen , homocysteine , and iron , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>In the example of FNR , cluster disassembly results from exposure to oxygen , and although iron sufficiency is required in sensing , the level of oxygen in the cell is the primary stimulus to which the regulatory response is oriented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>It is important to recall that we tested IscR , a transcriptional-regulator with an oxygen sensitive Fe-S cluster , an oxygen-resistant variant of FNR , another transcriptional-regulator , and the Nuo and Sdh complexes that both contain buried oxygen resistant Fe-S clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>We showed that the protective effect of the Δfnr mutation on cell death following topoisomerase cleavage complex accumulation was more prominent under low oxygen condition , consistent with the increased activity of FNR expected when oxygen is limiting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>It has been estimated that intracellular Salmonella can be exposed to up to 4 lM NO , which has Nitrate reductase encoded by narGHJI is the major contributor to nitrosation Minimal medium Metheringham & Cole ( 1997 ) Glycerol Fumarate FNR can respond to NO as well as oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>However , it also appears to respond to other environmental factors , including low oxygen partial pressure via FNR ( 249 ) , depleted media ( 410 ) , and elevated temperature ( Beatty and Wolfe , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Thus , there is no direct evidence that another component interacts directly with FNR to communicate the presence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>However , it is difficult to extrapolate from FNR to AadR , since it is not known whether or not AadR is directly responsive to oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Besides this speciﬁc molybdate-dependent regulation , transcription of the mod operon was enhanced by the presence of oxygen ( Table 9 ) , probably by the inactivation of FNR protein , which controls the transcription of oxygen-regulated enzyme systems ( 44 ) either directly or indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665461" target="_blank">7665461</a>]<br></li>
<li>However , the mechanism by which the FNR protein senses and responds to changes in oxygen availability remains rather poorly understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>However , owing to the strict oxygen lability of native FNR protein , studies of its 3D-structure and its regulation of target genes in vitro have been limited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>To compare the loss of the visible absorption features of 2Fe-FNR with the release of Fe2 + under the reaction conditions used , 5 µM 2Fe-FNR was exposed to the xanthine oxidase system in the presence and absence of ferene , and the absorption from 350 to 700 nm was monitored over time .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The release of Fe2 + from the [ 2Fe-2S ] 2 + cluster was measured in triplicate by monitoring A593 during the reaction of 3 µM 2Fe-FNR with the xanthine oxidase system in the presence of 1 mM ferene ( Sigma ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The release of Fe2 + from 2Fe-FNR upon reaction with superoxide was monitored in the presence of ferene , an Fe2 + - specific iron chelator ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The reaction of 5 µM 2Fe-FNR with the xanthine oxidase system was monitored in the presence and absence of ferene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>These data were ﬁt to a quadratic binding isotherm assuming a 1:1 binding stoichiometry , 1/4 ð dÞ DA = DAtotal E þ L þ KdÞ þ ððE þ L þ K 2 4ELÞ0 :5 = 2E ; where DA is the change in absorbance , Atotal is the maximal absorbance change , E is the FNR concentration as estimated from the initial absorbance , L is the concentration of the added second protein , and Kd is the dissociation constant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>These observations led the authors to conclude that the active form of FNR is the dimeric form and that the FNR-DA154 mutant has a FNR * phenotype due to an increased dimerization constant relative to the wild-type protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>We were unable to purify the wild-type FNR protein in an active form by using the protocols described here , and thus , could not test its DNA binding properties in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The concentrations of FNR were as follows : lane 3 , 0.13 M ; lanes 4 , 8 , and 9 , 0.25 M ; and lane 5 , 0.5 M. NarL and RNAP were used at concentrations of 0.2 M and 50 nM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>B. End-labelled AatII -- BamHI fragment was incubated with FNR DA154 in combination with Fis or IHF and treated with DNase I.The concentration of FNR was : lanes 2 , 5 and 8 , no protein ; lanes 3 , 6 and 9 , 0.25 mM ; lanes 4 , 7 and 10 , 0.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The concentration of Fis in each reaction mixture was as follows : lanes 1 to 4 , no protein ; lane 5 , 447 nM ; lane 6 , 894 nM ; lane 7 , 1.79 M. ( B ) End-labeled pnir7150 AatII-HindIII fragment was incubated with RNA polymerase ( RNAP ) , FNR DA154 , IHF , and Fis and subjected to potassium permanganate footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>FNR was added as indicated : lanes 1 and 3 , no protein ; lanes 2 and 4 , 80 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Lane 1 , no protein ; lanes 2 and 3 , FNR * ; lanes 4 and 5 , OxyR ; lane 6 , FNR * and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1053">
The concentration of FNR was : lanes 2 -- 9 , no protein ; lane 10 , 0.25 mM ; lane 11 , 0.5 mM ; lane 12 , 1.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></label>
<input id="act_similarity_1053", type="checkbox">
<div class="ACT_similar">
<li>The concentration of FNR DA154 was : 1 , 5 and 9 , no protein ; lanes 2 , 6 and 10 , 0.1 mM ; lane 3 , 7 and 11 , 0.3 mM ; lanes 4 , 8 and 12 , 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li></div>
<li>The concentration of FNR was : lane 1 , no protein ; lane 2 , 0.5 μM ; lane 3 , 1.0 μM ; lane 4 , 2.0 μM ; and lane 5 , 4.0 μM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>The concentration of FNR in each reaction was as follows : lanes 2 -- 13 , no protein ; lanes 14 -- 17 , 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1057">
The concentration of FNR in the reaction mixture was as follows : lanes 1 , 2 , 3 , and 8 , no protein ; lanes 4 to 7 , 1.5 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></label>
<input id="act_similarity_1057", type="checkbox">
<div class="ACT_similar">
<li>The concentration of FNR was as follows : lanes 1 to 3 and 8 , no protein ; lanes 4 to 7 , 1.5 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_1059">
The concentration of FNR in each reaction was : lanes 1 -- 3 , no protein ; lanes 4 -- 15 , 0.125 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></label>
<input id="act_similarity_1059", type="checkbox">
<div class="ACT_similar">
<li>The concentration of FNR was : lanes 1 -- 7 , no protein ; lanes 8 -- 14 , 0.125 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The concentration of FNR was : lanes 2 -- 4 and 6 , no protein ; lanes 5 and 7 -- 9 , 0.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The concentration of FNR was : lanes 2 , 3 , 5 and 7 , no protein ; lanes 4 , 6 , 8 and 9 , 0.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li></div>
<li>The concentrations of FNR DA154 were as follows : lanes 1 and 6 to 9 , no protein ; lane 2 , 0.5 M ; lane 3 , 1 M ; lane 4 , 2 M ; and lane 5 , 3 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1064">
The concentration of FNR DA154 was as follows : lanes 1 -- 4 , no protein ; lane 5 , 100 nM ; lane 6 , 250 nM ; lanes 7 -- 10 , 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></label>
<input id="act_similarity_1064", type="checkbox">
<div class="ACT_similar">
<li>The concentration of FNR was as follows : lane 1 , no protein ; lanes 2 to 7 , 250 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The concentration of FNR was as follows : lane 1 , no protein ; lanes 2 to 9 , 250 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li></div>
<li>The presence of Cys-29 HI addition to Cys-16 , Gys-20 and Cvs-23 was confirmed by deferminifig fhe A / - terminal protein seguence of carboxy-mefhylated FNR The revised molar mass for the 250-resi-due / n ; - gone product is 2/965 , It should be noted thaf fhe overlapping Taql fTCGA ) and Sm/3A ( GATC ) restriction sifes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>The initial biochemical studies of FNR utilized protein preparations that were puri cents ed in the presence of air which , in retrospect , has given us important information about one of the inactive forms of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Transcription of nikABCDE is up-regulated by the fumarate and nitrate regulatory protein ( FNR ) in response to low O2 levels and down-regulated by NikR in response to high intracellular nickel concentration9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>In the absence of nitrate , the affinity of FNR protein for the nar URS may not be high , due to the absence of NarL-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Mechanisms to minimize damage caused when FNR is active include nitrite reduction by the cytoplasmic nitrite reductase , NirBD , nitrite expulsion by the nitrate and nitrite transporter , NarK ( Jia et al. , 2009 ) , and possibly repair by the hybrid cluster protein , Hcp , and its reductase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>When there is a need for sensing of oxygen levels , it appears that in at least one situation , the Iumarate nitrate reduction protein ( FNR ) of E. coli , the loss of integrity of the cluster in the presence of oxygen is the key to sensing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>Abbreviations used : FNR , fumarate-nitrate regulation ; HCP , hybrid-cluster protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>A 24-mer oligonucleotide corresponding to the putative FNR-binding site could , when located appropriately , piace other promoters under positive regulation by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>The ansB 5 ' - untrans-lated sequence required for regulation by FNR protein and CRP , as defined by deletion analysis , was examined for the presence of the regulatory elements described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>It should be noted that regulation of L-asparaginase II by CRP is not via the FNR protein ( i.e. , regulation of FNR by CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>Thus , the physiological role of the FNR regulatory protein on hydrogen metabolism seems to be confined to its involvement in the regulation of the nik locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934931" target="_blank">7934931</a>]<br></li>
<li>Speci ® cally , NO inactivation of FNR results in upregulation of synthesis of Hmp , a protein with key roles in NO resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>It has been proposed that upregulation of hmpA by NO involves inactivation of MetR by nitrosation of its corepressor , homocysteine ( 25 ) , and inactivation of FNR by reaction of NO with the protein-bound [ Fe-S ] cluster ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>Jennings , M.P.and Beacham , I.R. : Ana!ysisof the # scherichiacoligene encodingL-asparaginaseII , ansB , and its regulationby cyclicAMP receptor and FNR protein.J .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2144836" target="_blank">2144836</a>]<br></li>
<li>FNR binds iron as a cofactor which most likely is involved in the O2-sensing by the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>FNR not only binds to the DNA but is also probably involved in several protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>At this point , the amount of active FNR is maximal such that the protein binds to both FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>It is important to note that there is greater need for Mo-cofac-tor when the FNR protein is also active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Trageser , M. , Six , S , , Unden , G. , and Guest , J , R. ( 1991 ) Characterisation of the FNR protein of Eschericim coii , an iron-binding transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>It is something of a paradox that substitution of the cysteine residues of FNR renders the protein incompetent for transcriptional activation , whereas naturally occurring proteins which lack the cysteines are functional .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The activities of these proteins are controlled in some manner by a small mol-ecule : FNR ( positive ) is inactivated by O2 , NarL ( negative ) is phosphorylated by NarX in the presence of NO3 , NikR ( negative ) is activated by Ni2 , and NikR function is inhibited by one or more formate-inducible hydrogenase assembly components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The reduced flavin bound to SsuD activates molecular oxygen to form a putative 4a ¬ Abbreviations : SsuE , alkanesulfonate FMN reductase ; SsuD , alkanesulfo-nate monooxygenase ; Fre , NAD ( P ) H : flavin reductase ; CT , charge -- transfer complex ; MC , Michaelis complex ; FNR , ferredoxin-NADP + reductase Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
<li>Perspectives The transcription of nik genes is stimulated by the global regulator FNR under anoxic conditions , and negatively controlled by a new repressor protein , NikR , responding specifically to Ni .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>The purpose of this genome-based study is to identify additional genes differentially expressed in response to oxygen availability and to define further the network of genes controlled by the global regulatory protein , FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The aerobic requirement for FNR is surprising because the iron ^ sulfur-con-taining holo-FNR protein activates the expression of an-aerobic genes in response to oxygen-starvation , but does not bind at these target promoters when oxygen is available ( reviewed in [ 8 ^ 11 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The FNR protein induces expression anaerobically , and evidence indicates that it probably binds to a specific recognition sequence located around position -40 bp relative to the transcription initiation site ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The consensus sequence for FNR is similar to that for CRP consensus , and previous studies have shown that FNR activates transcription initiation in a similar way to CRP and that both proteins can bind to the DNA site for the other protein ( 41,42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The unbound FNR protein in the cell can bind to lower affinity targets including moe DNA and repress its transcription when the demand for Mo-cofactor and Moe proteins is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The possibility that the binding of heterodimers has an effect on transcription regulation was judged unlikely following another control experiment in which heterodimers of FNR-E209V and wild-type FNR were co-expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>The regulation of yhjA expression is interesting , because transcription is dependent on two transcription factors , FNR and OxyR , that respond to two seemingly incompatible signals , i.e. oxygen starvation and peroxide stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The ability of FNR to activate transcription was observed to be impaired by iron deprivation , and the relationship between nutritional iron status and oxygen sensing led to speculation that an iron co-factor was involved in sensing of oxygen levels 19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>The FNR protein activates transcription in response to anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>1988 ) , The mechanism by which FNR controls transcription in response to the availability ot oxygen is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>The importance of the upstream site ( - 94.5 ) was later confirmed by the deficiency in FNR-mediated repression and the lowering of promoter activity in the absence of FNR that accompany deletion of this site ( Green and Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>The complete dependence of promoter 7 activity on both functional FNR and ArcA proteins was demonstrated in a primer extension experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>The modelled behaviours of ArcA and FNR were validated by direct measurements of the abundance of phosphorylated ArcA ( figure 5a , diamonds ) and by quantifying the transcript generated from activation of the synthetic FNR-dependent promoter FF-41 .5 ( figure 5b , dashed lines ) during the transitions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The ChIP experiments established that FNR binds directly at the cyoA promoter , so the effects of FNR and nitrate were confirmed to be both direct and indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The yabN promoter , to which FNR does not bind , was used as a negative control to normalize the data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The data in Table 4 show that , in the presence of wild-type FNR , which does not appreciably bind to YY sites , FF-FF is a very active promoter , but both of its derivatives , YY-FF and FF-YY , are inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The ability to sense and respond to oxygen limitation may be correlated with the presence of cysteine residues which , in the case of FNR , are thought to be involved in oxygen or redox sensing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The presence of an intermolecular disulphide bond seems unlikely because a covalently linked dimer would appear much larger , not smaller , than the FNR ( 30 ) form in SDS-PAGE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The highly reactive form of FNR can , aside from the addition of 02 , also be generated by growth of the bacteria in the presence of chelators such as o-phenanthroline ( Trageser & Unden 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The observed increase in the rate of Fld1 reduction in the presence of Fd could be due to electron transfer into Fld from either Fd or Fd : FNR complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>The suboptimal labeling in aerobic bacteria ( labeling of four compared to ﬁve Cys labels ) could be due either to incomplete labeling and accessibility or to the presence of some residual FNR with bound FeS cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The role of FNR-AR3 On the basis of amino acid similarity , the overall structure of FNR is predicted to be similar to CRP ( Fig. 2 ) , suggesting that , like its counterpart in CRP , FNR-AR3 is in a solvent-exposed loop that could be in close proximity to s - CTD in an RNAP ± FNR ± DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF-lacZ fusion by these two proteins ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Results Properties of witd-type and mutant FNR proteins The major structural difference between FNR and CRP is the presence of an W-terminal cysteine cluster in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The residues involved in cAMPbinding by CRP and the cysteine residues in YeiL and FNR are shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The putative consensus sequences for the CRP , FNR , Fur , and FhlA/HyfR proteins are enclosed by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The conformational changes required for transformation to the active state could involve a monomer/dimer transition of FNR , as suggested by the stabilization of FNR dimers in fnr * mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The Fe-S clusters are bound by four cysteine residues ( Cys20 , Cys23 , Cys29 , and Cys122 ) of the ﬁve Cys residues of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The effects of GA85 and SF73 substitutions in FNR , in each case , demonstrate the importance of G85 in FNR bound downstream and $ 73 in FNR bound upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The inhibitory action of FNR bound at the most upstream site appears to be mediated through protein -- protein contacts involving the face of FNR that includes AR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>The proteins were then transferred to a nitrocellulose membrane by Western transfer , and FNR levels were detected using - FNR primary antibodies and ﬂuorescein isothiocyanatelabeled anti-rabbit secondary antibodies ( BD Pharmingen ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>The nature of the response of FNR to anoxia is ` uncertain , but if it involves a conformationaI change , the FNR * proteins may be locked into the active-conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The change in the labeling behavior was then attributed to a thiol/disul ﬁde change and to metal ion binding by FNR in response to O2 availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>A ) ridh promoters was assayed with increasing concentrations of FNR , The degree of repression ( B ) was deduced from the synthesis of ndh transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The response to NO at Phmp was therefore due to partial relief of NsrR repression rather than relief of FNR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>The FNR site of ndh which is negatively regulated by FNR is located at position y50 .5 w60x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The [ NO ] / [ FNR ] ratio increases from 0 , 0.2 , 0.3 , 0.6 , 1.2 , 2.2 , 2.9 , 3.7 to 5.2 , giving rise to dose-dependent absorbance increases at 360 nm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The increased labeling was speciﬁc for FNR ; the ﬂuo-rescence of other labeled bands did not increase in the same way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The majority of the other known cellular nucleotides are not suitable as effectors of FNR for various reasons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The electron donor for the reverse reaction , i.e. , for the reductive activation of FNR , is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Conversely , although the arcA and adhE promoters include almost consensus FNR sites located appropriately to activate transcription , regulation by FNR is masked by other more significant factors that also regulate transcription positively or negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>This appears to be a recurring theme in the regulation of this promoter since the positively acting Arr and the negatively acting FNR also compete for overlapping sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>For that reason oxygen or reaction products of oxygen such as H20 2 or 02 - are not required for aerobic repression and the regulation by FNR appears to include a redox reaction , either at FNR or at a connected sensory component .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>bC = Cys , H = His , X = variant amino acid residues Role of positive redox potential ( Eh > + 400 mV ) in the regulation by FNR The effect of ` aerobic repression ' of anaerobic re-spiration can be brought about also in the absence of 02 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The situation seems to be different in regulation by FNR since functional aerobic respiration is not essential for aerobic repression ( Trageser & Unden , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Partridge , J.D. , Browning , D.F. , Xu , M. , Newnham , L.J. , Scott , C. , Roberts , R.E. , Poole , R.K. and Green , J. ( 2008 ) Characterization of the Escherichia coli K-12 ydhYVWXUT operon : regulation by FNR , NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Hence , the E. coli class III nrdDG operon appears to be under positive regulation by FNR and negative regulation by NrdR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>To date , inactivation of FNR by NO has only been shown to control E. coli hmpA regulation ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>A striking feature of the regulation of these genes is that nitrite induction is most apparent only in the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Gostick , D.O. , Green , J. , Gasson , M.J. and Guest , J.R. ( 1997 ) Redox-mediated regulation by an FNR homologuc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Keywords : FNR ; Oxygen sensing ; Transcription regulation ; Hydrogenase ; Sigma factor ; Carnitine metabolism ; CaiF 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Studies to examine this and other aspects of FNR regulation are ongoing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The target genes of FNR are mostly under multiple regulation by FNR and other regulators that respond to O2 , nitrate , or glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Regulation by O2 is effected by the O2 sensorregulators FNR and ArcArB w4 ,9,61 -- 63x , regulation by nitrate by the sensor-regulators NarXrL and NarPrQ w6 ,8 x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Thus , FNR can respond to not only O2 , but also NO , with major implications for global gene regulation in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>An iron cofactor ( Fe2 § of FNR appears to be essential in the process of O2-sensing and O2-dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>For some target genes , FNR is the only regulator , whereas most are regulated coordinately by FNR and the regulators ArcA , NarL , NarP ( nitrate/nitrite regulation ) , Fur ( Fe regulator ) , SoxRS and Q ( regulation by O2̇ ¯ ) , LexA ( SOS regulation ) , or CRP ( glucose regulation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>However , open complex formation in the presence of RNA polymerase , which is an indication for active FNR and transcriptional activation , is stimulated by Fe ( II ) Most of the FNR-dependent promoters are regulated by FNR in cooperation with other regulators such as NarL , NarP ( nitrate regulation ) , ArcA , Fur ( ferric uptake regulation ) , or CRP ( cyclic AMP receptor protein , glucose repression ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The mechanism of regulation of ydhY -- T expression by FNR , in response to oxygen availability , and by the NarXL and NarPQ systems , in response to nitrate and nitrite availability , revealed that NarL represses ydhY -- T expression by binding to sites in the promoter region that overlap those occupied by FNR and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Overview of metabolic regulation in response to oxygen and nitrate mediated by FNR , NarXL , and NarQP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Gene regulation by FNR is responsive to low level of oxygen [ 22,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>The glucose , oxygen , and nitrate repression of dcuB expression suggests that dcuB could be subject to regulation by the global transcriptional-regulatory proteins FNR , CRP , FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Sensitivity of the DcuS-PAS domain to oxygen , nitrate , redox status , or metabolic status might partly explain the FNR - , NarL - , and CRP-independent regulation of dcuB expression by oxygen , nitrate , and glucose , respectively ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
<li>Thus , factors other than FNR , NarL , and CRP also appear to contribute to the regulation of dcuB in response to oxygen , nitrate , and glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In two denitrifying Pseudomonas species , P. stutzeri and P. aeruginosa , expression of the nir , nor , and nos genes is regulated by the NO-responsive FNR-like transcriptional-activator DNR that binds to a DNA motif similar to the consensus FNR box , TTGATnnnnATCAA [ 34,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>It would further appear that the presence of iron is not essentiai for site-specific DNA bindin Introduction FNR is a transcriptional-regulator that controls the expression of target genes in response to anaerobiosis in such a Received 30 September , 1993 : revised and accepted 1 February , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>© 1995BlackwellScienceLt FNR is a transcriptional-activator which regulates a number of genes in response to anaerobiosis ( reviewed by Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Thus it is envisaged that in response to anoxia , FNR is transformed by a conformational change into a transcriptional-activator or repressor , which recognizes and binds to specific DNA sequences in the promoters of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Application of CopraRNA to previously characterized sRNAs proposed and partially veriﬁed additional targets and functions for the sRNAs cyclic AMP activated sRNA ( CyaR ) , FNR regulated sRNA ( FnrS ) , RybB , RyhB , sugar transport-related sRNA ( SgrS ) , and Spot42 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Establishing an anaerobic puri cents cation scheme for FNR provided the key breakthrough for investigation as to how WT FNR activity is regulated by O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Furthermore , the yceJI operon encodes a cytochrome b561 homolog that was reported to be negatively regulated by FNR ( 32 -- 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>What is significant is that this operon , like pfl , is also regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Groups VII and VIII , in contrast , appeared to be regulated in a manner that was independent of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Transcripts that appeared to be regulated directly by FNR are marked ( F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>No significant activity of IscR and FNR was predicted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>How is FNR activity regulated in vivo ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Of these 36 genes , 5 have been reported to be regulated directly by FNR ( 8 ) , and 31 are newly discovered genes that are either directly or indirectly regulated by this global regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>in the medium which cause half-maximal induction or repression of FNR regulated genes w12 ,120 x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The pO0 .5 values are in the range of 1 -- 5 mbar for most FNR regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The O2 tension corresponding to half-maximal induction or repression ( pO0 .5 ) is in the range of 1 to 5 mM O2 for most FNR regulated genes ( equivalent to approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>For example , the genes of patterns I , V , and VI are negatively regulated by FNR ; and those of pattern II , VII , and VIII are positively regulated by FNR ( Figs. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>For example , many genes involved in the biosynthesis of the O-antigen and capsular polysaccharides are epistatically regulated , as are several genes also under FNR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>However , this classification is not strict , and some of the FNR regulated genes code for enzymes of aerobically required enzymes and vice versa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Our ﬁnding suggest that as many as 465 genes of MG1655 are regulated by changes in environmental O2 and/or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Ninety-one genes with no presently deﬁned function were also altered in expression , including seven of the most highly anaerobically induced genes , six of which we found to be directly regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Consistently , anaerobically sodA as well as metE expression was known to be negatively regulated by FNR though an undetermined mechanism ( 32 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>The majority of other ArcA or FNR regulated genes studied so far are either repressed anaerobically by ArcA or activated anaerobically by FNR , although three examples are known whose expression is repressed by FNR anaerobically ( 14 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>For example , most of the genes in group III are repressed by the O2-sensing system mediated by ArcA ( 49 , 70 ) , and although not directly demonstrated , the data in this paper suggest that these genes can also be repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Fig. 3 illustrates one possible scenario by which indirect regulation might occur , in this case why nitrate activates but FNR represses transcription of cyoA ; similar regulatory circuits explain why Kang et al. ( 2 ) included many ArcA-regulated operons in their lists of genes activated or repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Although the transcription factor MarA is not directly regulated by FNR or ArcA , four genes that are activated by MarA ( acnA , fumC , sodA , zwf ) are repressed by ArcA and/or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Therefore , to determine whether the observed asymmetry of the ndh response could be accounted for by changes in the activities of ArcA and FNR , a dynamical model was developed based on high-resolution RT-PC data for a training set of genes that were either ArcA - ( hypB and icd ) or FNR - ( dmsB , hybO ) regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Three other promoters , P2 , P3 , and P4 , are coordinately regulated with P1 in response to oxygen , ArcA , and FNR ; however , a fifth promoter , P5 , is not ( Fig. 6 ) ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Interestingly , 303 genes were found to be regulated by both ArcA and FNR , and 74 of these genes showed additional regulation by oxygen ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Recent lacZ transcriptional fusion experiments in our laboratory confirm these results and further demonstrate that these genes are regulated by ArcA but , 3 as reported here , not by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Oxygen limitation regulates numerous genes such as those of the FNR and ArcA regulons [ 6,7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>B , Venn diagram for ArcA - , FNR - , and Lrp-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>To illustrate the overlap between genes regulated by ArcA , FNR , and Lrp , a Venn diagram was constructed ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Plpd is anaerobically repressed by ArcA , whereas the Ppdh promoter was not strongly regulated by FNR or ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>FNR regulatory protein and repressed or inhibited by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10486127" target="_blank">10486127</a>]<br></li>
<li>The contiguous fumB and dcuB genes are also in this group and were already known to be regulated by FNR , NarL , and ArcA ( 31 ) ; thus , at this promoter , regulation by NarL might be both direct and indirect ( via the ubiquinol pool and ArcA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The latter form of regulation involves thefnr gene product ( FNR protein ) ( 20 ) , which also activates a number of other anaerobically regulated genes ( 18 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The hypothesis that NsrR mediates regulation of denitriﬁcation genes in Neisseria is further supported by the observation that in N. gonorrhoeae , the norB gene is strongly induced by NO independently of FNR and NarP [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>These results indicated that excess putrescine is appropriately catabolized by products of the puu gene cluster that are regulated by ArcA , FNR , and PuuR to decrease the toxicity of putrescine and to utilize putrescine as both nitrogen and carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18495664" target="_blank">18495664</a>]<br></li>
<li>The acnA gene is positively regulated by Crp , SoxRS , FNR and repressed by ArcA [ 59 ] , whereas acnB is positively regulated by CRP and negatively regulated by ArcA , FruR and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The fumB gene is positively regulated by FNR , ArcA , Crp and Fur and negatively by Fis and NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The single gene of this cluster previously known to be regulated by oxygen and not affected by FNR is formate dehydro-genase-O , the product of the fdoH gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Green , J. , Bennett , B. , Jordan , P. , Ralph , E.T. , Thomson , Guest , J. , Green , J. , Irvine , A. , and Spiro , S. ( 1996 ) The FNR modulon and FNR-regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>1996 The FNR modulon and FNR regulated gene expression , p. 317 -- 342 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>NO is a damaging agent of iron-sulfur clusters , and apart from the wide variety of iron-sulfur-containing proteins involved in relevant pathways , the formation of these clusters is essential at least for the transcriptionally active form of regulators such as FNR , IscR , and SoxR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>Enteric bacteria appear to have solved this problem by evolving multiple repair pathways , some that function when FNR ( and Fur , etc. ) are active , and others that deal with the greater threat when damage is so severe that FNR is itself inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Tran , Q.H. , Arras , T. , Becker , S. , Holighaus , G. , Ohlberger , G. , and Unden , G. ( 2000 ) Role of glutathione in the formation of the active form of the oxygen sensor FNR ( [ 4Fe4S ] FNR ) and in the control of FNR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Due to the lability of active ( Fe-containing ) FNR , no structural and spectroscopic data are available for wildtype FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Further work is now needed to test this hypothesis and establish the relative contributions of the alternative terminal oxidases in shielding transcriptionally active FNR from inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The importance of controlling the concentration of FNR by negative autoregulation and proteolysis ( 31 ) is evident ; the model predicts that a 3-fold increase in FNR concentration results in 3 M active FNR at 30 M O2 , an O2 concentration at which FNR is normally 10 % active ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The O2-triggered transformation between active and inactive FNR presumably is due to a redox reaction at the Fe cofactor , but other modes of interaction can not be excluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The reverse reaction , the formation of active FNR is assumed to be slow and to represent the sum of de novo synthesis of FNR and of the reactivation of O2-inactivated FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>The in vivo and in vitro experiments therefore strongly suggest the functioning of iron as the cofactor of FNR for gene activation and O2-sensing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The rate of inactivation of FNR by O2 in cells has also been measured in order to analyze the relevance of the in vitro reaction to the in vivo process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>The reaction is part of de novo formation of [ 4Fe-4S ] FNR , but also of the reactivation of O2 inactivated apoFNR ( demonstrated 2 + in vitro ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>These results indicate that for inactivation of FNR oxidation is essential , whereas mere binding of O2 Žor O2 analogs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>FNR samples inactivated by either O2 or NO bind speci ® cally to Phmp , but with lower af ® nity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>DNA binding by holo-FNR , O2 - and NO-treated FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The DNA-binding activity of FNR was affected by either O2 or NO treatment , resulting in a lower Phmp af ® nity and speci ® city .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Our current model for cellular O2 sensing by FNR is based upon the critical discovery that FNR is directly inactivated by O2 in vitro ( 12 , 15 , 16 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In vivo demonstration of FNR dimers in response to lower O2 availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>This suggests that in vivo , the rate of FNR inactivation by O2 is the same for both the DNA-bound and - unbound forms of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>FNR can be inactivated in vivo by oxidants like hexacyanoferrate without involvement of O2 Žstep I. w126x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Therefore it is assumed that the inactivation of FNR by O2 essentially is responsible for the control of the functional state of FNR , or the ratio of active and inactive FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>1 this can be explained assuming that the inactivation of FNR by O is rapid compared to the formation of active FNR , and that the inactivation varies largely in response to the O2 concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Therefore at O2 concentrations as low as 1 mM in the medium , sufﬁcient O2 should be present in the cytoplasm for FNR inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Therefore in vitro FNR can be inactivated by O2 and reactivated in a cyclic process as shown in Figure 1 , and in this way O2 inactivation of FNR is reversible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>According to this model the functional state of FNR is determined by a ( rapid ) inactivation of FNR by O , and a slow ( constant ) 2 reactivation with glutathione as the reducing agent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Biochemically and functionally different forms of FNR and their interconversion induced by O2 as oxidizing agent and glutathione as reducing agent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>In addition , FNR function was decreased upon treatment of anaerobically growing cells with metal chelators and activity was restored in vivo speci cents cally upon addition of Fe2 [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>In cyt b , reductase it is a haem , in nitric oxide synthase it is FMN , and in spinach FNR and bacterial dioxygenases it is a soluble iron-sulphur-containing ferredoxin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>In vivo , assuming relatively high concentrations of Fd and low concentrations of FNR , Fd may serve as a redox mediator of electron ﬂow from FNR to Fld .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>In summary , the data presented in this study support the hypothesis that the FNR cluster conversion pathway provides a facile system for responding to ﬂuctuating levels of O2 in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In particular , we found that sufficient salt , low temperature , and relatively rapid purification were important for the successful purification of 2Fe-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>2Fe-FNR Is Stable in Both the Presence and Absence of O2 In Vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>In this way , 2Fe-FNR could serve an important adaptive function as a superoxide-sensitive intermediate in FNR regulation , by responding to fluctuating O2 levels in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Both positive and negative controls were included in these assays : note , however , that although enrichment of a promoter in the immunoprecipitate is strong evidence that FNR binds to the targeted promoter , nothing should be concluded from a negative ChIP result , which might arise for many different reasons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>More than 2-fold enrichment of the positive control , pdcuB , was obtained in almost every experiment , and the cspA promoter was always enriched , confirming that FNR binds to pcspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Our results show that NarL and FNR bind to their targets independently , but NarL is unable to activate the nrf promoter in the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Our results conﬁrm that Fis binds independently of FNR and IHF and we suppose that Fis blocks access of RNAP to the nrf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , activation involving FNR , NarL and IHF is overridden by Fis when it binds to its target at position -15 and blocks access to the nrf promoter -10 element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Our results show that open complex formation is completely dependent on FNR and is enhanced by NarL , but is repressed by IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Rather , a Fis -- IHF -- FNR -- pnir complex forms in which pnir activity is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1237">
NarL can bind simultaneously to pnir with FNR , Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></label>
<input id="act_similarity_1237", type="checkbox">
<div class="ACT_similar">
<li>FNR , Fis and IHF can bind to pnir simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li></div>
<li>Figure 2 illustrates such an experiment , in which the pnrf97 fragment was incubated with RNAP in the presence of puriﬁed Fis , IHF , FNR and NarL , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , in the presence of all three proteins , Fis binds at its target at position -15 , but the FNR and IHF footprints remain incomplete .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>To investigate the mechanism by which NarL counteracts IHF - and Fis-mediated repression at pnir , we used gel retardation assays and DNase I footprinting to examine the binding of NarL , FNR , Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Thus , although Fis and IHF repress FNR-dependent activation of pnir , they do not prevent FNR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Thus , we selected the p22C and p42T substitutions for further study and we used electromobility shift assays to measure the effects of these changes on the binding of Fis , FNR and IHF to the pnrf97 fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Previously , using electromobility shift assays , we investigated the pairwise binding of FNR , NarL , Fis and IHF to the pnrf97 fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Fis , IHF and FNR can bind together to the nir promoter Further DNase I footprinting experiments were performed to investigate the binding of puriﬁed Fis , IHF and FNR protein in different combinations at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The third minor Fis -- NarL -- DNA complex , seen in Fig. 6B ( lanes 10 -- 13 ) , is likely NarL , FNR and Fis can bind to pnir together , but NarL interferes with the binding of IHF Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>A. End-labelled pnir7150 EcoRI -- HindIII fragment was incubated with increasing concentrations of puriﬁed phospho-NarL and FNR DA154 proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>End-labelled hcp383-1 AatII-HindIII fragment was incubated with increasing concentrations of Ala154 FNR and subjected to DNase I footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Hence , Fis protein is a major factor responsible for catabolite repression at the nrf promoter , and Fis can override activation by FNR and NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>These results conﬁrm that FNR and NarL , respectively , are responsible for activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>Based upon this proposition , the dual function of FNR in the overlapping promoter could be interpreted to exhibit repression toward narX but activation toward narK depending on the arrangement of these three anaero-boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>By this mechanism , promoter regions that display the full range of FNR regulation , from synergistic activation to full repression , could be generated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>It was anticipated that nitrate repression would not show a dependence on the presence of ArcA , FNR , or IHF , which proved to be the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Nitrate repression is independent of IHF , FNR , and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>These results indicate that repression by nitrate is independent of the three transcription factors ArcA , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>These observations suggest that the surface of FNR that is required for repression mediates an interaction between the upstream and downstream FNR dimers that is inhibitory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>It is possible that FNR binds to the sequence TTCAT in the moe operator DNA to facilitate repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>It will be interesting to elucidate the mechanistic relationship between FNR - and YjeB-mediated repression of the hmpA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>Thus , in part A , one subunit of the heterodimer is repression defective , whereas in ( B ) , all the FNR subunits are repression competent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Here , both FNR subunits are repression competent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Here , both adding a solution of NO gas and by maintenance of NO in cultures using NOC compounds , we demonstrate that FNR can sense pathophysiological levels of NO ( 5 ± 10 mM ) in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Membrane permeable labels are supposed to allow labeling of cytoplasmic proteins in situ without cell disruption and with minimal perturbation of cellular metabolism and functional state of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>This review will deal mainly with the question of how FNR regulates in response to 02 and by which means the functional state of FNR is controlled by 02 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Therefore it is not clear so far whether in vivo the functional state of FNR is controlled only by oxidation and reductio Fig. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>room temperature caused decreases in absorbance at 420 nm and increases 550 nm , as reported for FNR ( Fig. 2 A and ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Data in Fig. 2 ( inset ) were normalized by dividing the change in absorbance at 460 nm by the maximal change in absorbance and by dividing the concentration of Fd or Fld added by the initial concentration of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>induction of FNR synthesis , cell cultures were sparged with argon at 4 ° C for 14 hr , which appeared to improve recovery of FNR as compared with our previous results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>( E ) Cell extracts of indicated strains , cultured in LB medium , were subjected to immunoblot analysis using antibodies raised against IscR , FNR , NuoF and NuoC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>FNR -- DNA complex Free DNA when cells were grown in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Both proteins apparently recognize the same DNA sequence as FNR but the physiological signal to which they respond is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>This increase is also evident with FNR GA85 , but is abolished by FNR SF73 , confirming that $ 73 , but not G85 , is required upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This model for FNR induction is likely to apply to dcuB also .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>This induction was tentatively attributed to deactivation of FNR , which requires iron for function ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>Binding of FNR to the ygbA , cyoA , and ytfE promoters was confirmed , but only negative results were obtained for the yeaR , gltB , and ogt promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>( B ) Increasing magnitudes of EPR signals eli-cited by addition of aliquots of NO solution ( NO and FNR concentrations were as indicated ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>x Editor : Simon Silver Keywords nitric oxide ; nitrosative stress ; NsrR ; NO sensing by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>[ FNR ' FNR pHHlOO FNR + FNR-pHH200 Aerobic Anoerobic + ^ J0 ^ Anaerobic Anaerobic + NO , Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057835" target="_blank">8057835</a>]<br></li>
<li>Clearly , even if FNR , Fld1 , and methionine synthase are at roughly equivalent concentrations , FNR is not capable of ful-ﬁlling this requirement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Results Replacement of Ser-24 by Phe Enhances FNR Activity Under Aerobic Conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>FNR senses Received 19 October , 1997 ; revised 1 June , 1998 ; accepted 5 June , 1998 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Contribution of CyaY to the IscR , FNR , Nuo and Sdh maturation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>IscR , FNR , Nuo and Sdh maturation in ΔcyaY , ΔiscX and ΔcyaYΔiscX mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Microbiol Binding of FNR * to the nrdDG promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>FNR bound at promoter 6 appears to be required for this to occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Lane G , calibration G track ; lane 1 , no addition ; lane 2 , Fis ; lane 3 , FNR ; lane 4 , FNR and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>For many years it was unclear how this of FNR were unchanged .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>FNR are fumB , lysU , nikA , and narX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Lane 1 , PhypA no FNR * ; 2 , PhypA ; 3 , PhypA3 + ; 4 , PhypA +3 ; 5 , PhypA33 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Hence , and contrary to SoxRS , Fur and FNR seem to be NO-sensitive regulators resulting in a finely tuned control of their regulons , i.e. they act as multi-responsive regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>Are these proteins also part of a defence mechanism that , like HMP , would be most important physiologically under extreme reactive nitrogen stress , when NO is sufficiently abundant to inactivate FNR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>This would be most important physiologically under extreme RNS stress , when NO is sufficiently abundant to inactivate FNR ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>However , because the levels of b-galactosidase were lower than observed with pGS1422 in the presence of FNR-G74C and FNR-E209K , it is suggested that all the FNR subunits can contribute to downregulation , but only when the two inwardfacing subunits of the tandem FNR dimers are repression competent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>However , in the presence of FNR the altered promoters were about three-fold more active than wildtype , which may in part be due to weaker anaerobic repression by Fis , IHF and FNR , because position -66 forms part of the Fis I1 and IHF I11 sites , and the -55 substitution removes a key FNR : DNA interaction at the FNR I site ( Figs 1 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Nevertheless , our results strongly suggest that another factor , whose nature remains to be elucidated , is playing the major control , although in competition with FNR and NarL/NarP , and its action is mainly triggered by the presence of nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>aeg-46 .5 is regulated by NarL and possibly by FNR ; thus , the presence of sequences that may be responsible for NarL or FNR control were sought in the upstream sequences by using the search program Find , allowing some mismatches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>Thus it would appear that RNA polymerase binds to the - 35 and - 1 0 regions of the upper strand of the ndh promoter in the presence of FNR , but the contact extending up to -132 on the lower strand is restricted when FNR is bound to both of its target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Puriﬁed promoter fragments were end-labelled with [ g-32P ] ATP and 0.5 ng of each fragment was incubated with varying amounts of puriﬁed FNR D154A , which allows binding in the presence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>C. Anaerobically in Ihe presence of FNR , transcription from the rjdh promoter is repressed because FNR binds to the - 50.5 and 94.5 sites , preventing activator binding and the upstream RNA polymer-ase contacts , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Thus , the architecture of the ydhY -- T promoter appears to be somewhat different from that of previously characterized FNR - , NarL - and NarP-regulated promoters , having evolved to be repressed in the presence of nitrate and nitrite by using NarP/NarL to inhibit RNA polymerase binding , and NarL to inhibit FNR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>RNAP binding and that , in the presence of NarL , a new highly retarded complex appears that , most likely , contains NarL , FNR , and RNAP bound together to the ynfE promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>Earlier approaches to identify accessible Cys residues of FNR in vivo ( 10 ) were performed when the presence of FeS clusters in FNR was not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>With this finding , earlier observations are confirmed which showed the presence of only FNR of M , 30000 in bactena ( Unden and Duchene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>In particular , there were no signals equivalent to FNR labeled at only one or three Cys residues which would be characteristic for the presence of two or one intramolecular disulﬁde bonds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>In the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Furthermore , these transcripts ( except dadX ) are regulated either by ArcA , FNR , or OxyR , possibly accounting for the increased abundance of these transcripts in the presence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>FNR from bacteria with ( moderately ) overproduced FNR is supposed to show normal response to the presence of O2 ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>We conclude that this novel variant of FNR can be widely used in various biochemical and structural studies of FNR in the presence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>For example , Paracoccus denitrificans has two paralogous FNR-like proteins , FnrP and NNR , which are each predicted to recognize FNR sites by the presence of E -- SR motifs in their DNA-recognition helices ( Fig. 4b ) and by the presence of potential FNR sites in FnrP - and NNR-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>There was no appreciable difference in the rate of O - induced Fe2 release 2 from 4Fe-FNR in the presence and absence of DNA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Moreover , in gel retardation assays in which both FNR * and OxyR were present , the migration of the labelled yhjA promoter was slower than in the presence of either FNR * or OxyR alone , suggesting that both proteins occupy the yhjA promoter simultaneously ( Fig. 1 , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Transcription from the c54-dependent promoter was una ¡ ected by the presence or absence of FNR ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>See text for details Fig. 5 DNase I hydrolysis pattern of moe operator/promoter DNA in the presence of FNR * - DA154 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>We found in carefully controlled experiments that addition of Fd , Fld1 , or Fld2 to FNR results in 10 -- 20 % increases in the extinction coeficient of the FAD cofactor at 460 nm and other minor shifts in absorption bands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Determination of the afinity of Fld1 , Fld2 , and Fd for FNR To determine the afinity of Fd , Fld1 , and Fld2 for FNR , we examined UV -- visible spectra of FNR as increasing amounts of each protein was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Using the protein concentrations determined above and known extinction coeficients , the predicted contribution due to Fd , Fld1 , or Fld2 was subtracted , and the remaining increase in absorbance at 460 nm was attributed to changes in the FAD cofactor of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>We examined binding by following changes in the spectrum of the FAD cofactor in FNR as increasing concentrations of Fld1 , Fld2 , or Fd were added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Overall , these results suggest that while FNR can bind each potential electron transfer partner , binding to Fd is thermodynamically favored over Fld1 and Fld2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Total observed spectral changes are normalized to facilitate plotting on the same scale and are ﬁt with rate constants of Fd , 8.4 s 1 ; Fld1 , 7.5 and 25 s 1 ; and Fld2 , 1.2 and 8.7 s 1 Fd facilitates electron transfer into Fld1 and Fld2 Since Fd binds tightly with FNR , we considered the possibility that Fd might inhibit binding of Fld to FNR , and further slow the rate of Fld reduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Since Fd was prereduced to equilibrium with FNR and NADPH and is rapidly rereduced following electron transfer , Fd contributes little to the observed absorbance changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>It is intended that the structure-function comparisons of FNR and FLP will highlight the way in which a diversity of functional specificities , at the sensory , DNA-recognition and RNA-polymerase-interaction levels , can be ` mixed and matched ' to create regulators that are adapted for particular physiological roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>In N-oxide sensing and denitrification transcription factor ( DNR ; effector NO ) , a member of the CAP/FNR superfamily for which a structure of the isolated effector-binding domain in the absence of effector is available ( 27 ) , the C-helix and the intersubunit C-helix/C - helix coiled coil seem to be partly unstructured in the absence of effector and to extend only to the residue corresponding to Gln-125 in CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19359484" target="_blank">19359484</a>]<br></li>
<li>It has also been shown that FNR induces an 82-92 ° bend in target DNA ( Ziegelhoffer and Kiley , 1995 ) but , as with CRP , factor-medi-ated DNA bending may not be essential tbr transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>It was also shown that a Ci 22A substitution in the FNR protein prevents site-specific DNA-binding and in vitro transcription activation and repression ( Sharrocks et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Current models of how FNR activates transcription ( based mainly on in vitro studies with the cyclic AMP receptor protein , GRP , a related activator ) envisage that the DNA wraps around the FNR , resulting in a contact between RNA polymerase and sequences upstream of bound FNR ( Busby , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Results : Analysis of the intergenic region upstream of purM demonstrated a novel mechanism of resistance to the covalent protein-DNA cleavage complex through titration of the cellular transcription regulators FNR and PurR responsible for oxygen sensing and repression of purine nucleotide synthesis respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>As the pO2 decreases ( microaerophilia ) , the amount of FNR able to-bind DNA increases proportionally , yielding amounts of active FNR in the cell that are sufficient to allow binding to the high-affinity FNR-2 site ( which also displays greater similarity to the FNR consensus sequence ) , but not to the FNR-1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>This may indicate an FNR152 monomer can bind to a single half-site , but only when it is in the proper context , since no DNAse I protections were observed at any other potential FNR halfsites present in the DNA fragments used in this study ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Two of the five reportedly `` FNR activated '' operons , cydDC and glpTQ , lack even a remotely credible consensus FNR-binding site , and the data implicating FNR in their regulation is weak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1330">
Changes in expression levels of operons known from a range of genetic , biochemical and in vitro data to be dependent on FNR activation were not detected in these previous studies , and the nirB promoter in which there is a consensus binding site for FNR activation even appeared to be repressed by FNR [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></label>
<input id="act_similarity_1330", type="checkbox">
<div class="ACT_similar">
<li>Consequently , changes in expression levels of operons known from a range of genetic , biochemical , and in vitro data to be dependent upon FNR activation were not detected in the previous studies , and the nirB promoter in which there is a consensus binding site for FNR activation even appeared to be repressed by FNR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li></div>
<li>An FNR dependent promoter from E. coli is activated anaerobically in Paracoccus denitrificans , and thefixK gene of Rhizobium meliloti hybridises to genomic DNA from P. denitrificans ( Spiro 1992 ; A. Hinsley & S. Spiro , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>It was previously reported that overproduction of the FNR homolog HlyX of A. pleuropneumoniae in anaerobically grown E. coli K-12 results in binding to the FNR site and activation of clyA expression , while overproduction of the E. coli FNR protein results in less efﬁcient activation ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>It was shown previously that when expressed in anaerobically grown E. coli K-12 , the FNR homolog HlyX of A. pleuropneumoniae and , although much less effective , FNR , are able to activate ClyA expression , presumably by binding to the FNR binding site in the clyA upstream region ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Thus , it is suggested that Fis enhances ndh expression during periods of rapid growth , by acting as Received 28 October 2003 Revised 19 November 2003 a Class I activator , and that the binding of tandem FNR dimers represses ndh expression by preventing interaction of the RNA polymerase a-subunit with DNA and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>It is currently assumed that the redox status of the cell is `` sensed '' by iron which in its reduced form induces a conformational change in FNR that promotes DNA binding ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Thus it would seem that FNR activation may be controlled by a redox-sensitive iron cofactor in the iron-FNR-DNA complex which senses and responds to the redox state of the celt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>In previous work ( 3 -- 6 ) , we showed that the DNA target for FNR at pnrf is centered at position 41.5 ( i.e. , between 41 and 42 bp upstream from the transcript start point ) , that anaerobically induced FNR-dependent activation of pnrf is downregulated by the binding of the nucleoid-associated protein IHF to a target ( IHF I ) centered at position 54 , and that this repression is relieved by the binding of either NarL or NarP to a DNA target centered at position 74.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>This event would , conceivably , position the upstream NarL-binding site and bound NarL protein close to the FNR and RNA polymerase proteins to provide the necessary conditions for narK transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>Deletion of either the FNR or PurR binding site from pInter decreased the protective effect , demonstrating that titration of these transcription factors could result in increased resistance to norfloxacin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>These contacts were prevented by FNR binding to the secondary site , leading to inhibition of open complex formation and repression of ndh transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Thus , the consequence of NarL binding to pnir is to remodel the nucleoprotein complex such that FNR activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>FNR binding at FNR II impairs the ability of FNR I to activate transcription anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Presumably repression is the result of nitrate-triggered NarL binding to heptamers - 5 0 and - 2 2 and blocking FNR binding or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Thus , in this case FNR binding leads to repression , probably via the usual mechanism of touching polymerase or occluding its binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Misplacing the NarL-binding site by one base pair stops induction and leads to NarL-dependent repression , presumably as the misplaced NarL interferes with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Hypersensitive sites , induced by FNR and Fis binding , are indicated by stars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>bThe presence of FNR-binding sites upstream of genes encoding FNR homologues is presumed to be indicative of autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>This would be consistent with the preferred binding of FNR * to the downstream half-site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>If this is a functional NARL-binding site , it is possible that the binding of NARL , in the presence of nitrate , interferes by steric hindrance with the binding of FNR to its site nearby .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934939" target="_blank">7934939</a>]<br></li>
<li>Hence it can be widely used in in vitro DNA binding assay to study FNR regulated genes and to characterize the structural details of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>The proposal that dimerization is su $ cient to increase site-speci cents c DNA binding of FNR is further supported by the property of mutants that have substitutions of Asp154 ( located in the putative dimeri-zation helix , Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Specific binding of purified FNR to the anaero-box was not detected in gel retardation assays [ 6 ] , although Guest and colleagues recently succeeded in detecting the DNA-binding activity by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>As the DNA binding site for FNR overlaps the acsP1-35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Overall nrf promoter activity is increased by both substitutions , with a greater increase associated with tightening of FNR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>These results for CRP activation at proP through AR1 are similar to those found where CRP is acting as an upstream activator alone or in conjunction with either another molecule of CRP , FNR , or the l cI protein bound in an analogous position to Fis site I at proP P2 ( 11 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li>
<li>In Bacteria , an abrupt slope in the NWD curve reveals the presence of a handful of high-scoring binding sites for highly speciﬁc TFs ( e.g. TrpR , LexA ) , whereas a progressive increase of the NWD slope is indicative of global factors , such as CRP , FNR ( Figure 3 ) , Fur , FruR , IHF and FIS ( Supplementary Data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>55 different TFs binding sites were detected with high confidence , of which the most frequently found were those for CRP , Fis , PhoB , RhaS , PurR and FNR ( Table S4 and Figure S6 ) , of which , CRP , Fis and FNR are general TFs that regulate many genes in E. coli ( Table S5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838305" target="_blank">19838305</a>]<br></li>
<li>Consistent with our observations at PhlyE , it has been suggested that SlyA binding to such sites remodels the local nucleoprotein structure , counteracting the effect of negative regulators such as H-NS , and clearing the way for classical transcriptional-activators such as PhoP , or as in the case of hlyE , FNR and CRP ( Ellison and Miller , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>However , a degree of reciprocity of binding-site recognition can be inferred from the observation that FNR can activate transcription from a promoter containing a consensus CRP site , but only in a crp mutant , and CRP can likewise activate an FNR-dependent promoter in the absence of FNR ( Sawers et aL , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Similarly , the hlyE promoter includes a 9 of 10 fit to the core FNR-binding site , but this is also a binding site for CRP , the cAMP receptor protein , and H-HNS ( heat-stable nucleoid-structuring protein ) interferes with FNR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Seemingly , CRP uses different mechanisms of transcriptional activation depending on the position of its binding site , and FNR may be less versatile in this respect TM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1363">
Sawers , G. , Kaiser , M. , Sirko , A. and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding-siterecognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></label>
<input id="act_similarity_1363", type="checkbox">
<div class="ACT_similar">
<li>Sawers , G. , Kaiser , M. , Sirko , A. , and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding-site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Sawers , G. , Kaiser , M. , Sirko , A. , and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li></div>
<li>In this study , we found that the addition of EDTA and ~ - phenanthroline inhibited the repression activity of FNR ( see Fig. 2C or D ) , Despite the sequence homology in the DNA binding domain between FNR and CRP , the two regulatory proteins respond to different signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>We repeated the analysis , using equally stringent search criteria , for other global DNA-binding proteins ( LexA , Fur , FruR , FNR , CRP , NarL and Fis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>FNR differs from CRP in three amino acid residues ( Val-208 , Ser-212 and Gly-216 in FNR ) which are known to be critical for specific binding of CRP to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>This form of FNR is therefore the equivalent , in terms of binding preferences , to CRP EV181 ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>, 1989 ; Jennings & Beacham , 1993 ) that hybrid binding sites of this type respond to both CRP and FNR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>Here , MS analysis revealed a major signal at a mass of 28,398 Da corresponding to apoFNR alkylated at ﬁve Cys residues by acetate ( increase in molecular mass by 58 Da/acetate ) which conﬁrms the presence of apoFNR with ﬁve reduced thiol residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>After prolonged exposure to air in vitro , apoFNR lacking a Fe-S cluster is formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>ApoFNR from aerobic bacteria contained no disulﬁde , in contrast to apoFNR formed in vitro by air inactivation , and all Cys residues were in the thiol form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">FNR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Puriﬁed ApoFNR was reconstituted in vitro using Fe ( II ) , cysteine , and cysteine desulfurase NifSAV as described previously ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>ApoFNR with one nonreactive Cys residue could be due to temporary binding of a monothiolate or a metal ion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>After prolonged exposure to air in vitro , apoFNR lacking a Fe-S cluster is formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>By reaction with O the cluster is converted 2 to a [ 2Fe -- 2S ] cluster and ﬁnally to apoFNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>The presence of one or two accessible residues in anaerobic cells suggests that the bacteria contain some apoFNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The labeling pattern of apoFNR from aerobically grown E. coli suggested that all Cys residues are accessible to labeling by AMS ( Fig. 4 ) , indicating that no intramolecular disulﬁdes are formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The labeling methods used here allowed direct dem-onstration of apoFNR , and in addition , of the redox state of the Cys residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>ApoFNR from aerobic bacteria contained no disulﬁde , in contrast to apoFNR formed in vitro by air inactivation , and all Cys residues were in the thiol form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In puriﬁed apoFNR , deﬁned numbers of Cys residues were blocked by reaction with deﬁned amounts of NEM ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>For a more quantitative analysis of the redox state of the cysteine residues , apoFNR from aerobically growing E. coli was labeled in vivo in nonpermeabilized bacteria with NEM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In the in vitro reconstitution glutathione serves as the reducing agent for disulﬁde residues present in apoFNR and for the formation of HS from cysteine by the cysteine desulfurase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>ApoFNR can be differentiated from Fe-S-containing forms by the accessibility of the ﬁve Cys thiol residues , four of which serve as ligands for the Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Here , MS analysis revealed a major signal at a mass of 28,398 Da corresponding to apoFNR alkylated at ﬁve Cys residues by acetate ( increase in molecular mass by 58 Da/acetate ) which conﬁrms the presence of apoFNR with ﬁve reduced thiol residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>This residue is in the so-called positive patch of FNR that is predicted to interact with RNA polymerase ( Williams et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Transcription activation is facilitated by the formation of speciﬁc protein -- protein contacts between the activating regions ( AR1 , AR2 and AR3 ) of FNR and regions of RNA polymerase ( Bell & Busby , 1994 ; Blake et al. , 2002 ; Lamberg et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>2007/012146 G 2008 SGM Printed in Great Britai This architecture permits the formation of only one protein -- protein contact , between FNR and the C-terminal domain of the RNA polymerase a-subunit ( Browning et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>This arrangement allows the formation of multiple protein -- protein contacts between RNA polymerase and FNR ( Browning et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Previous studies have identified a small region in the s subunit of RNAP that is required for FNR-dependent 70 expression of PnarG and PdmsA ( Lonetto et al. , 1998 ) and thus is one candidate for a protein ± protein contact with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Class I promoters have FNR sites located close to 261.5 , 271.5 , 282.5 or 292.5 , relative to the transcript start ( Wing et al. , 1995 ) , which allows only one protein -- protein contact between the C-terminal domain of the RNAP a-subunit and activating region 1 ( AR1 ) of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>Class I promoters possess FNR sites located close to 261.5 , 271.5 , 282.5 or 292.5 ( Wing et al. , 1995 ) that permit the formation of only one protein -- protein contact , between AR1 of FNR and the C-terminal domain of the RNA polymerase a-subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Since the FNR protein is thought to be a dimer like CAP ( Shaw et al. , 1983 ; Spiro and G$ $ 1988 ) it should also contact both half-sites in a similar manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Model building studies have suggested that a hydrophobic residue at the position corresponding to Val-208 in FNR also contributes to specificity ( Cherfils et al. 1989 ) , although the phenotype of an FNR protein substituted at this position is not consistent with this suggestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>FNR activation of both the class I and class II promoters is achieved through direct contact between FNR and the RNA polymerase σ subunit ( RNAP σ ) [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_24">
Williams , S.M. , Savery , N.J. , Busby , S.J.W. , and Wing , H. ( 1997 ) transcription-activation at Class I FNR-dependent promoters : identiﬁcation of the activating surface of FNR and the corresponding contact site in the C-terminal domain of the RNA polymerase a subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></label>
<input id="dom_similarity_24", type="checkbox">
<div class="DOM_similar">
<li>Williams , S.M. , Savery , N.J. , Busby , S.J.W. and Wing , H.J. ( 1997 ) Transcription activation at Class 1FNR-dependcnt promoters : identification of the activating surface of FNR and the corresponding contact site in the C-terminal domain of the RNA polymerase o ~ subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Williams , S.M. , Savery , N.J. , Busby , S.J.W. , and Wing , H.J. ( 1997 ) Transcription activation at class I FNR-dependent promoters : identification of the activating surface of FNR and the corresponding contact site in the C-terminal domain of the RNA polymerase a subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Williams , S.M. , Savery , N.J. , Busby , S.J.W. , and Wing , H.J. ( 1997 ) Transcription activation at Class I FNR-dependent promoters : identification of the activating surface of FNR and the corresponding contact site in the C-terminal domain of the RNA polymerase alpha subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_28">
Transcription activation at class I FNR-dependent promoters : identiﬁcation of the activating surface of FNR and the corresponding contact site in the Cterminal domain of the RNA polymerase alpha subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></label>
<input id="dom_similarity_28", type="checkbox">
<div class="DOM_similar">
<li>Transcription activation at class I FNR-dependent promoters : identiﬁcation of the activating surface of FNR and the corresponding contact site in the Cterminal domain of the RNA polymerase a subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Transcription activation at class I FNR-dependent promoters : identiﬁcation of the activating surface of FNR and the corresponding contact site in the C-terminal domain of the RNA polymerase a subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Transcription activation at Class I FNR-dependent promoters : identiﬁcation of the activating surface of FNR and the corresponding contact site in the C-terminal domain of the RNA polymerase a subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>Therefore , even though CRP and FNR share the same structural framework , there are major differences in the way they interact with CZCTD , such that different contacts are used to modulate the extent of transcription-activation at specific promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Lastly , we provided genetic evidence that transcription-activation by FNR is likely to 70 require an interaction with the RNAP s subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Thus , it is likely that interactions between the tandem FNR dimers conspire to inhibit ndh transcription at several levels , i.e. by preventing interaction between the a-C-terminal domains of polymerase with DNA and with Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>Thus , for FNR-mediated transcription of a Class II promoter , AR3 of the downstream subunit of the FNR dimer contacts region 4 of ( y70to effect direc activation , and the extended AR 1 of FNR contacts O ~ CTD to overcome the otherwise inhibitory efl ` ects of an untethered o ~ - subunit so as to promote open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Single amino acid substitutions ( G74 → C , F92 → S , A95 → P , R184 → P , P188 → A or L193 → P ) in the surface of FNR that contains activating region 1 ( AR1 contacts the a-subunit of RNA polymerase to promote transcription-activation ) reduced the inhibitory effect of FNR at FNR II , indicating that this region of the protein may have a role in repression as well as activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>All the substitutions recovered were located in the face of FNR that contains activating region 1 ( AR1 ) , a surface-exposed region of FNR that has a role in transcription-activation by making direct contacts with the a-subunit of RNA Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Therefore , by occupying sites located immediately upstream and downstream of the Fis II site , FNR acts as a physical barrier , preventing the formation of the Fis : RNA polymerase contacts necessary for Fis-mediated transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>To reveal the molecular mechanisms of transcription-activation and repression by FNR , it is also a prerequisite to determine the contact site ( s ) for FNR on RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>This promoter architecture must also prevent the formation of contacts between FNR and RNA polymerase that result in activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>Bases substituted are shown in bold lower case letters ; the core motifs of the two FNR-sites are highlighted in bold : þþpyﬁD ( wild-type ) ; þ ¹ pyﬁD ( substitution of the key GC base pair in each FNR I core motif ) ; ¹ þpyﬁD ( substitution of the key GC base pair in each FNR II core motif ) ; [ þ / ¹ ] þpyﬁD ( substitution of the key GC base pair of the promoter proximal FNR II core motif only ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>As in CRR the ARI of FNR functions solely in the downstream subunit of the dimer where it contacts a region of C ~ CTDdefined by substitutions at Leu300 , Glu302 , Asp305 , Ala308 , Gly315 and Arg317 , which represents two distinct ` subregions ' rather than a single discrete patch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>This promoter contains two regions of FNR protection centred at -50.5 and -94.5 , and DNase I footprints indicate that RNA polymerase protection extends at least as far as -132 , including contacts in the region of the secondary FNR site ( -94.5 ) but not the primary FNR site -LCB- -50.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>A similar situation is found in the dcuC promoter in E. coli , which contains two FNR activating regions centered over positions − 34 and − 65 [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>- Fold change relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 2.1 2.0 3.7 2.3 2.2 Position of potential FNR - and/or Arc-binding sites Gene Predicted function/conserved domains None None yacL ygbF yhjG yzgL ybdN ycgW ybdJ yaeI yciO ydeJ Conserved domain of unknown function Conserved domain of unknown function Homology to AsmA family Conserved domain of unknown function Phosphoadenosine phosphosulfate reductase domain Conserved domain of unknown function Conserved domain of unknown function Function unknown , phosphodiesterase activity Possesses a domain shown to bind double-stranded RNA Function unknown , domain similarity to CinA None None ArcA , 53.5 ( 2 ) ; FNR , 143.5 ( 2 ) , 420.5 ( 1 ) ArcA , 103.5 ( 1 ) ; 257.5 ( 2 ) ; FNR , 157.5 ( 2 ) None None FNR , 111.5 ( 2 ) ; 321.5 ( 2 ) FNR , 489.5 ( 1 ) 2.1 4.3 5.0 2.0 2.1 3.7 4.6 3 Y. Tang and J. Green , unpublished data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>In carbon monoxide oxidation activator ( CooA ; effector CO ) and o-chlorophenol reductive dehalogenase transcription regulator ( CprK ) ( effector ortho-chlorophenolacetic acid ) , 2 members of the CAP/FNR superfamily for which structural data are available ( 24 -- 26 ) , the C-helix and the intersubunit C-helix/C - helix coiled coil seem to be fully structured in the absence of effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19359484" target="_blank">19359484</a>]<br></li>
<li>In N-oxide sensing and denitrification transcription factor ( DNR ; effector NO ) , a member of the CAP/FNR superfamily for which a structure of the isolated effector-binding domain in the absence of effector is available ( 27 ) , the C-helix and the intersubunit C-helix/C - helix coiled coil seem to be partly unstructured in the absence of effector and to extend only to the residue corresponding to Gln-125 in CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19359484" target="_blank">19359484</a>]<br></li>
<li>This is followed by an intra-protein electron transfer from the reductase domain ( or FNR , ferredoxin-NADP reductase-like domain ) to the N-terminal heme domain in an NAD ( P ) H-dependent reaction via a non-covalently bound FAD allowing the reconversion of the oxidized heme ( ferric , Fe ( III ) ) to the reduced ( ferrous , ( FeII ) ) form and consequent continuation of catalytic-activity ( 26 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25193663" target="_blank">25193663</a>]<br></li>
<li>It contains three phylogenetically distinct subgroups designated FnrN -RCB- AadR , FixK and DNR -RCB- FLP in addition to the FNR subgroup , which contains the archetypal FNR proteins that regulate oxygen-dependent gene expression via the anaerobic acquisition of oxygen-labile [ 4Fe -- 4S ] clusters 0002-4345 # 2000 SGM ( Kiley & Beinert , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>HlyX , the FNR homologue of Actinoba-cillus pleuropneumoniae , is a [ 4Fe-4S ] - containing oxygen-responsive transcription regulator that anaerobically activates FNR-dependent class I promoters via an enhanced AR1 contact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>A further modification of the CRP-FNR structural framework is found in Bacillus subtilts ( Cruz-Ramos et al. , 1995 ) and Bacillus licheniformis ( Klinger et al. , 1998 ) , where the putative [ 4Fe-4S ] sensory clusters are located within the Cterminal domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>From these kinetic curves of Fe2 release with FNR , it appears that one Fe is released very rapidly in the ferrous form , and in air a total of two Fe2 are released in a cents rst phase , after which apparently a slow destruction of the cluster continues for a longer period ( D.M. Bates and P.J. Kiley , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>While we were preparing this report , Crack et al. ( 7 ) reported that H2O2 was produced from the reaction of FNR with O2 , which led to the proposal of a reaction mechanism where two cluster irons are oxidized to Fe3 during O2-induced cluster conversion to provide the two electrons to produce H2O2 from O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>This result also suggests that although covalently linking two FNR monomers enhances dimerization by locating the two monomers in close proximity , it does not suppress the dimerization induced by the assembly of the [ 4Fe -- 4S ] 2 + cluste which is spontaneous in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>To investigate the effect of superoxide on 2Fe-FNR turnover in vivo , we were limited to whole-cell methods that monitor the status of the Fe-S cluster ( since 2Fe-FNR has no known in vivo activity ) , and all such available methods require the overexpression of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>While this cluster is reactive with superoxide , we have no evidence that FNR plays any role in the well-known adaptive response to superoxide stress , since 2Fe-FNR has no known transcription activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Isolated 2Fe-FNR Contains a [ 2Fe-2S ] 2 + Cluster and Is Monomeric in Solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>[ 2Fe -- 2S ] FNR is no longer active in transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>However , the destabilization of the [ 2Fe-2S ] cluster of FNR with superoxide is not a general property of [ 2Fe-2S ] proteins since imposing a superoxide stress on SoxR results only in a change in cluster oxidation state ( from [ 2Fe-2S ] 1 + to [ 2Fe-2S ] 2 + ) , which converts SoxR into an active transcription factor ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>However , the [ 2Fe-2S ] 2 + cluster of FNR does not contain any iron lacking a cysteinyl ligand like that of the hydrolyase clusters , and thus the decomposition of 2Fe-FNR is likely to show some differences from that of the hydrolyase clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>By quantifying the amount of [ 2Fe-2S ] 2 + cluster in 2Fe-FNR in vitro in the presence of various reductants and oxidants ( GSH , DTT , cysteine , O2 , hydrogen peroxide , and superoxide ) , we found that superoxide , a byproduct of aerobic metabolism , significantly destabilized the [ 2Fe-2S ] 2 + cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The loss of the [ 2Fe-2S ] cluster from purified 2Fe-FNR was monitored as a decrease in absorption at 550 nm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>To further characterize the destruction of the [ 2Fe-2S ] cluster of FNR , the Fe3 + - specific chelator Tiron was used to monitor release of Fe3 + from the [ 2Fe-2S ] 2 + cluster during the reaction with superoxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>It can be seen that the doublet representing 2Fe-FNR has essentially disappeared , i.e. , the [ 2Fe-2S ] 2 + cluster has been destroyed by the formation of superoxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>We conclude that the [ 2Fe-2S ] 2 + cluster of FNR is labile to superoxide both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Oxidation of FNR by oxygen in vitro : formation of the [ 2Fe-2S ] cluster ... 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Determining the in vitro rates of cluster conversion also allowed us to assess whether these values were compatible with in vivo measurements of FNR inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>It has yet to be determined whether the form of FNR containing the [ 2Fe-2S ] 21 cluster has any biological significance , e.g. , as an in vivo intermediate that is more rapidly converted to the active form than the apoprotein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Such information will be important in determining whether the 2Fe cluster form of FNR is a physiologically relevant intermediate in the activation/deactivation of FNR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Nonetheless , whether it is the 2Fe form or apoprotein ( or some combination of both ) that is the substrate in vivo , activation of FNR will likely require additional proteins that facilitate Fe-S cluster assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>With FNR we now have a 4Fe 3 2Fe conversion occurring simply with a pure protein exposed to air in an aqueous solution at neutral pH. Since the 2Fe form appears relatively stable in the presence of air ( half-life at least 12 h at 0 ° C ) and this cluster ( or its components ) could be used to regenerate the 4Fe form in vitro , it is possible that the 2Fe form could have some physiological relevance if the 2Fe form was more readily reactivated in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>As described in the following sections , a simple change in the redox state of the [ 4Fe-4S ] 2 cluster to the expected 1 + oxidation state is not the physiologically relevant redox reaction for FNR ; the cluster is very di $ cult to reduce to the +1 oxidation state in vitro and no physiological electron donors or acceptors have been identi cents ed that would maintain the cluster in the +1 oxidation state under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>FNR , another Fe -- S transcription factor in E. coli , only forms dimers and binds DNA in the presence of its oxygen-labile [ 4Fe-4S ] cluster .8,31 SoxR , a [ 2Fe -- 2S ] - containing transcription activator in the MerR-family , can bind its single target DNA site in the presence or absence of its Fe -- S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Molecular modeling suggests a mechanism by which Phe at position 24 protects the FNR [ 4Fe-4S ] 2 cluster and that Cys-23 or the iron bound to it is a primary site of O2 interaction with the [ 4Fe-4S ] 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>In summary , we have demonstrated a 2-step mechanism for the FNR [ 4Fe-4S ] 2 to [ 2Fe-2S ] 2 cluster conversion via a [ 3Fe-4S ] intermediate , shown that the residue at position 24 in FNR modulates the reaction of the [ 4Fe-4S ] 2 cluster with O2 , and formulated a plausible molecular model that suggests that 2 O2 interacts with the FNR [ 4Fe-4S ] cluster in the vicinity of Cys-23 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Reversal of cluster conversion When oxygen-treated FNR was made anaerobic and reduced by dithionite , Moëssbauer spectroscopy showed that the [ 4Fe-4S ] 2 cluster was regenerated in 90 % yield on the basis of total cluster iron , that is , two 2Fe clusters apparently reformed one 4Fe cluster [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>We report here the unexpected observation that the [ 4Fe-4S ] cluster of FNR is converted in less than 5 min to a [ 2Fe-2S ] cluster in about 60 % yield as judged from Mössbauer spectra and this conversion results in a reduction in DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>We also found that , on reaction of FNR with air , the [ 4Fe-4S ] 21 cluster decayed and a new cluster species was formed in high yield ( 60 -- 65 % ) , which , according to UVyVIS and Mössbauer spectroscopies , exhibited properties of [ 2Fe-2S ] 21 clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Our current understanding is that FNR is continually supplied with [ 4Fe-4S ] 2 clusters by the iron -- sulfur cluster biosynthetic machinery ( 25 -- 27 ) , but O2 converts the [ 4Fe-4S ] 2 cluster to a [ 3Fe-4S ] cluster that spontaneously disassembles to yield a [ 2Fe-2S ] 2 cluster with concomitant inactivation of FNR ( 12 , 15 , 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The present study shows that the predominant form of FNR in aerobic cells is apo-FNR ( cluster-less FNR ) indicating that the [ 2Fe-2S ] 2 + cluster , like the [ 4Fe-4S ] 2 + cluster , is not stable under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Khoroshilova N , Popescu C , Münck E , Beinert H & Kiley PJ ( 1997 ) Iron-sulfur cluster disassembly in the FNR protein of Escherichia coli by O2 : [ 4Fe-4S ] to [ 2Fe-2S ] conversion with loss of biological activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Apparently , [ 2Fe-2S ] - and [ 4Fe-4S ] - contain-ing FNR showed similar sensitivities to the displacement of the FeS cluster , and both forms were not differentiated in the labeling studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The O2 sensitivity of FNR is also mediated by the [ 4Fe-4S ] 2 cluster ; in the presence of O , the [ 4Fe-4S ] 2 2 cluster is converted to a [ 2Fe-2S ] 2 cluster both in vitro and in vivo ( 15 , 16 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Thus , the lack of 2Fe-FNR in aerobic cells is difficult to explain by lability of the [ 2Fe-2S ] cluster to O2 as has been previously observed with the [ 4Fe-4S ] cluster ( 4 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>In the presence of O2 , conversion of the [ 4Fe-4S ] 2 + cluster to a [ 2Fe-2S ] 2 + cluster inactivates FNR , but the fate of the [ 2Fe-2S ] 2 + cluster in cells grown under aerobic conditions is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>In the presence of O2 , the [ 4Fe-4S ] 2 cluster is converted to a [ 2Fe-2S ] 2 cluster , which inactivates FNR as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>As a ﬁrst step , we developed an assay to monitor the [ 4Fe-4S ] 2 cluster to [ 2Fe-2S ] 2 cluster conversion under conditions where O2 could be added in excess relative to 4Fe-FNR and where product formation could be easily monitored in real time .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Upon exposure to O2 , FNR is inactivated by conversion of the [ 4Fe-4S ] 2 + cluster to a [ 2Fe-2S ] 2 + cluster , which produces the 2Fe-FNR form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_89">
Kinetic analysis of the oxidative conversion of the [ 4Fe -- 4S ] cluster of FNR 2 + to a [ 2Fe -- S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="dom_similarity_89", type="checkbox">
<div class="DOM_similar">
<li>Kinetic analysis of the oxidative conversion of the [ 4Fe -- 4S ] cluster of FNR 2 + to a [ 2Fe -- 2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Kinetic analysis of the oxidative conversion of the [ 4Fe-4S ] 2 cluster of FNR to a [ 2Fe-2S ] 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Kinetic analysis of oxidative conversion of the [ 4Fe-4S ] 2 cluster of FNR to a [ 2Fe-2S ] 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Kinetic analysis of the oxidative conversion of the [ 4Fe-4S ] 2 + cluster of FNR to a [ 2Fe-2S ] 2 + cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li></div>
<li>Since our previous Mössbauer analyses of the cluster conversion process 2 suggested that Fe is one product of the reaction of 4Fe-FNR with O2 ( 16 , 24 ) , we examined the effectiveness of using the Fe2 - speciﬁc chelator ferene to monitor reaction progress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>In our previous analysis of the cluster conversion process , we found that only 70 % of the sulﬁde originally in the cluster was detectable following the treatment of 4Fe-FNR with O2 ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_96">
O2-dependent cluster conversion of 4Fe-FNR can reasonably explain the loss of FNR activity by O2 in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></label>
<input id="dom_similarity_96", type="checkbox">
<div class="DOM_similar">
<li>Acknowledging the stated assumptions , we conclude that O2-dependent cluster conversion of 4Fe-FNR can reasonably explain the loss of FNR activity by O2 in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li></div>
<li>For FNR prepared with naturally abundant iron , a maximal [ 4Fe-4S ] cluster content of 0.50 -- 0.58 mol per mol of FNR monomer was obtained , whereas the samples labeled with 57Fe contained 0.76 and 0.59 mol [ 4Fe-4S ] cluster per mol of FNR monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>To test whether the anaerobically functional dimer of ( FNRD154A ) 2 was still dependent on the [ 4Fe -- 4S ] 2 + cluster and thus retains the O2 sensing capability , we introduced C122A , which has been shown to abolish the [ 4Fe -- 4S ] 2 + cluster binding and transcription activity upon introduction to native FNR [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>The ability of FNR to function as a transcription factor depends on the integrity of a [ 4Fe-4S ] 2 cluster , which promotes a conformation that is necessary for FNR dimerization , site-speciﬁc DNA-binding , and transcriptional regulation ( 12 , 16 , 19 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Assembly of the [ 4Fe -- 4S ] 2 + cluster results in simultaneous FNR dimerization and activity of speciﬁc DNA-binding and transcription regulation [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>The [ 4Fe ± 4S ] 2 + cluster of FNR is oxygen labile and controls protein dimerization and site-speci ® c DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>FNR has an O2-sensitive [ 4Fe-4S ] 2 cluster that directly senses O2 and regulates site-speciﬁc DNA-binding ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>By removal of the cluster with a Fe chelator , Lazazzera et al. also showed that the [ 4Fe-4S ] cluster was responsible for the increased dimerization and site-spe-ci cents c DNA-binding that was observed with anaerobically puri cents ed FNR [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Under anaerobic conditions , acquisition of 1 [ 4Fe-4S ] 2 cluster per FNR protomer causes dimerization and enhances site-specific DNA-binding ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_106">
Under anaerobic conditions FNR contains a [ 4Fe -- 4S ] cluster that promotes dimerization of the protein and enhances site-specific DNA-binding ( Jervis & Green , 2007 ; Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="dom_similarity_106", type="checkbox">
<div class="DOM_similar">
<li>Under anaerobic conditions FNR contains a [ 4Fe -- 4S ] cluster that promotes dimerization of the protein and enhances site-speciﬁc DNA-binding ( Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>We have investigated whether DNA-binding and the spectral characteristics of the protein containing the [ 4Fe-4S ] cluster could be restored to the air-oxidized ( 10 min ) FNR by reconstitution ( see above , Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Previously , we observed that brief exposure of anaerobic FNR preparations to air caused a decrease in both specific DNA-binding and the absorption spectrum characteristic of a [ 4Fe-4S ] cluster ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>While it is known that the [ 4Fe-4S ] 2 cluster of FNR is required for DNA-binding , we are not aware of any experiments that have addressed whether DNA-binding inﬂuences the rate of O - induced [ 4Fe-4S ] 2 cluster conversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Therefore , we conclude that the interaction of DNA with 4Fe-FNR does not substantially inﬂuence the susceptibility of the [ 4Fe-4S ] 2 cluster to O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>To obtain oxygen stable FNR variants for in vitro characterizations , in this study , we utilized the approach of covalently linking two fnr alleles in tandem to promote their in situ dimerization in the absence of the [ 4Fe -- 4S ] 2 + cluster under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Replacing aspartic acid with alanine in FNRD154A eliminates charge repulsions between the two FNR monomers and thus improves its dimerization in the absence of the [ 4Fe -- 4S ] 2 + cluster under aerobic conditions [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Under conditions of very low oxygen tension , most of FNR will be in the dimeric state and will contain the 4Fe cluster , with the remainder of FNR being in the monomeric state as apoprotein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>This review will describe these recent cents ndings : namely , what led to the discovery of a [ 4Fe-4S ] cluster in FNR and how this metal cluster functions in oxygen sensing by FNR both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Furthermore , we found that the [ 4Fe-4S ] 21 cluster in wild-type FNR was very difficult to reduce , suggesting that its redox potential is rather low and that such a reduced cluster ( 11 oxidation state ) would not occur in vivo ( see above and ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Attempts to reduce the [ 4Fe-4S ] 2 cluster of FNR with dithionite indicated that the redox potential of the +1 / +2 couple is 9 3650 mV and that the [ 4Fe-4S ] cluster form is , therefore , not likely to occur in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>We have thus further supported our proposed model for the function of FNR , namely that the active form of the FNR protein contains a [ 4Fe-4S ] 21 cluster , and that this cluster serves as the O2 sensor by virtue of its extreme sensitivity toward O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Previous analysis of an FNR mutant protein ( 2 ) suggests that sensitivity toward O2 is primarily determined by subtle details of protein structure , since substitution of Leu-28 for His allowed for purification in the presence of oxygen of FNR containing a [ 4Fe-4S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The predominant absorption ( 17 % of total Fe and outlined by the solid line ) of the central doublet in Figure 7A belongs to the [ 4Fe-4S ] 2 + cluster of 4Fe-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>al form of FNR contains a [ 4Fe-4S ] cluster ... ction nal d ction odu ctio fun Intr Fun The ... 1 2 3 * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The functional form of FNR contains a [ 4Fe-4S ] cluster 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The [ 4Fe-4S ] 21 cluster of FNR was not reduced by dithionite , even at pH 9 and in the presence of methylviologen , indicating a very low redox potential .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The electronic spectra obtained upon addition of dithionite to FNR containing the [ 4Fe-4S ] 2 cluster indicate at least partial reduction of the cluster to the 1 + state [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Our data showed that this form of FNR contained a [ 4Fe-4S ] 21 cluster ( d 5 0.45 mmys ; DEQ 5 1.22 mmys ) and that the [ 4Fe-4S ] 21 cluster was rapidly destroyed on exposure of FNR to air .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Recent data suggest that FNR contains a [ 4Fe-4S ] cluster ( 2 -- 4 ) and that this cluster apparently mediates the sensitivity of this transcription factor to oxygen , thus limiting FNR activity to anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>FNR is located in the cytoplasm and contains a [ 4Fe -- 4S ] cluster in the active ( anaerobic ) state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Thus , our data show that anaerobically purified FNR contains a [ 4Fe-4S ] 21 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Recent progress in this field has shown that anaerobically purified FNR contains a [ 4Fe-4S ] 2 cluster and that this [ 4Fe-4S ] 2 cluster is sufficiently unstable toward O2 to make it suitable as an O2 sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Thus it seems likely that the anaerobic activation of FNR requires the de novo incorporation of a [ 4Fe-4S ] 2 + cluster into apo-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Reduction of the [ 4Fe-4S ] 2 + cluster FNR puri cents ed under anaerobic conditions has been shown by Mo ëssbauer spectroscopy to contain a [ 4Fe-4S ] cluster in the 2 + form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>FNR is a homodimeric DNA-bind-ing protein containing approximately one [ 4Fe-4S ] 2 cluster per subunit ( called 4Fe-FNR ) ( 3 , 16 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>Under anaerobic conditions , FNR exists in a transcriptionally active form that consists of a homodimeric complex containing one [ 4Fe-4S ] 2 + cluster per subunit ( herein called 4Fe-FNR ) 1 ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Role of the Fe-S cluster in dimerization FNR puri cents ed from anaerobic cells is a dimer containing one [ 4Fe-4S ] 2 cluster per subunit [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Rather it appears that the form of FNR as isolated ( [ 4Fe-4S ] 2 cluster ) is the relevant form and the presence of this cluster stabilizes a conformation of FNR that is able to dimerize and bind DNA site-speci cents cally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>However , it is not understood how the presence of the [ 4Fe-4S ] 2 cluster increases dimerization of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>If one accepts the structure of the CAP protein as model for that of FNR , then it seems that the presence of the [ 4Fe-4S ] cluster causes FNR to adopt a speci cents c conformation which is favorable for dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>From the stoichiometries of iron and sulfide relative to the number of cysteine residues of anaerobically purified FNR ( 3 ) , we had previously concluded that FNR contains one [ 4Fe-4S ] rather than two [ 2Fe-2S ] clusters per monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The stoichiometry of iron and labile sulfide relative to the number of cysteine ligands of anaerobically purified FNR is most compatible with the presence of a [ 4Fe-4S ] cluster ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Previous determinations of Fe and S22 stoichiometries of anaerobically purified FNR ( 3 ) relative to the number of cysteine residues have suggested that the most likely type of cluster present in anaerobic preparations of FNR is a [ 4Fe-4S ] 21 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Under anoxic conditions , FNR is a dimer and carries a [ 4Fe -- 4S ] 2 + cluster per monomer ( [ 4Fe -- 4S ] FNR ) which is bound by 4 cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Here we studied the formation and presence of apoFNR in aerobically and anaerobically growing E. coli in situ by differentiating apoFNR from [ 2Fe-2S ] FNR and [ 4Fe-4S ] FNR using a method for measuring the accessibility of the Cys residues to thiol reagents ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Up to ﬁve Cys residues are accessible to thiol reagents in apoFNR , whereas only Cys16 is accessible in [ 4Fe-4S ] FNR and [ 2Fe-2S ] FNR ; the other four Cys residues are ligands of the Fe-S clusters in [ 4Fe-4S ] FNR and [ 2Fe-2S ] FNR and are thereby protected from labeling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Since [ 4Fe-4S ] FNR and [ 2Fe-2S ] FNR require the same number of Cys residues ( four ) for liganding the FeS cluster , the difference should indicate increased amounts of apoFNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In vitro , air inactivation of [ 4Fe-4S ] FNR yields apoFNR with 1.5 Cys-disulﬁde residues ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Before the presence and function of the [ 4Fe-4S ] cluster in FNR was known , the Cys residues of aerobically growing E. coli were shown to be more readily accessible to alkylation than those from anaerobically growing E. coli ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Thus , we modeled residues Cys-20 -- Cys-29 of FNR with a [ 4Fe-4S ] cluster bound by using SWISS-Model ( 38 -- 42 ) and endonuclease III as a template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Thus , apoFNR , which was reduced by DTT , and reconstituted [ 4Fe-4S ] FNR showed the same Mr of 34,000 after labeling by AMS , suggesting that AMS replaces the FeS cluster and labels FNR at up to ﬁve Cys residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>23 and 29 ) , together with the single internal cysteine ( Cys122 ) , represent the ` cysteine signature ' that is essential for FNR function ( Melville and Gunsalus , 1990 ; Sharrocks et al. , 1990 ) , presumably by providing the ligands tk ) r assembly of the [ 4Fe-4S ] cluster assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The cluster ligands for the [ 4Fe-4S ] cluster of FNR are presumed to be Cys20 , Cys23 , Cys29 and Cys122 since mutational analysis of these residues has shown that they are required for FNR function in vivo [ 10,12 ] and these mutant proteins lack iron and sul - cents de upon puri cents cation ( Khoroshilova et al. , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The E. coli strain JRG1728 ( vlac , vfnr ) containing pGS1483 was then transformed with either pGS196 , an fnr expression plasmid ; pGS198 , an equivalent plasmid encoding an FNR protein that is unable to sense and respond to oxygen-starvation , because it lacks three of the four cysteine residues that ligate the [ 4Fe-4S ] cluster ; or the vector pBR322t ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The exact molecular basis of the enhanced oxygen sensitivity of the [ 4Fe-4S ] cluster and sensory capacity of CydR is unknown but it could be due to differences in amino sequence of the N-terminal region of CydR relative to E. coli FNR , where substitutions in non-liganding residues are known to affect cluster stability ( see section 2.3.2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>So , it would appear that histidine residues in the N-terminal region of FNR improve the stability of the [ 4Fe-4SI cluster , and that the presence of three histidine residues in this region of FIpA may have a similar stabilizing effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>At the amino acid level , CydR is 50 % identical to E. coli FNR and it retains the DNA-binding specificity of FNR and the four cysteine ligands involved in [ 4Fe-4S ] cluster acquisition by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A number of FNR homologues have been identiﬁed and are characterized by the presence of four essential cysteine residues that act as ligands for the [ 4Fe -- 4S ] 2þ cluster , and the amino acid sequence E -- SR in the DNA-binding helix , which confers speciﬁcity for the FNR ope-rator sequence TTGAT -- ATCAA ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Despite lacking two of the N-terminal cysteine residues that are essential in FNR , FlpA was found to acquire [ 4Fe-4S ] clusters under anaerobic conditions and their incorporation was accompanied by the loss of the DNA-binding activity ( Scott et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>It is easy to envisage the conversion of an anaerobic transcription factor like FNR to an FlpA-like factor simply by replacing two of the N-terminal cysteine residues by histidine to generate a more stable [ 4Fe-4S ] protein ( and vice versa ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>It is interesting to note that the molar extinction coefficients of the [ 2Fe-2S ] Rieske iron-sulfur proteins ( which have two cysteine and two histidine ligands ) are lower than those of proteins with ` all-cysteinyl ' [ 2Fe-2S ] clusters ( Fee et al. , 1984 ) , because the molar extinction coefficient of the [ 4Fe-4S ] form of FIpA is also low compared with that of reconstituted FNR , which would be consistent with the use of Rieske-like liganding in FlpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The transition from active FNR containing cubic [ 4Fe-4S ] 2 + clusters to inactive forms , having planar [ 2Fe-2S ] 2 + clusters or no clusters , presumably involves substantial rearrangement of the essential cysteine ligands , which in turn could drive the dimer-monomer transition and thus inactivate the transcriptionally active DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The accompanying rearrangement of the cysteine ligands in moving from a cubic [ 4Fe-4S ] z + cluster to a planar [ 2Fe-2S ] 2 + cluster drives the dimer to monomer transition and consequent loss of biological activity.Apo-FNR [ 4SH ] ( excludingthe non-essential Cys16 sulfhydryl ) can be reversibly oxidized to a disulfide form [ 2SH , S-S ] , suggesting that FNR may be able to sense oxidative stress as well as normoxia .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Nevertheless , if the acquisition of the iron -- sulphur is not simply fortuitous or for purely structural purposes , cluster assembly and the potential for reversible disulphide and dimer formation could provide functional switches for DNA-binding and transcription regulation analogous to those operating in FNR and FLP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>By analogy with FNR and FLP ( the FNR-like protein of Lactobacillus casei ) it was suggested that the transcriptionregulatory activity of YeiL might be modulated by a sensory iron -- sulphur cluster and/or by reversible disulphide bond formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>By analogy with FNR and FLP the DNA-binding form of YeiL might be dimeric and its activity might likewise be modulated by a sensory iron -- sulphur cluster , by reversible dithiol -- disulphide formation , or by both mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>To correlate DNA-binding ability with the amount of the Fe-S cluster present in FNR , we examined changes in the Fe-S cluster content after exposure of the protein to air by UV-visible , EPR , and Mössbauer spectros-copies and by analyses for Fe , inorganic sulfide , and protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>In these latter two cases , DNA-binding is restored , thus indicating that these two forms of FNR appear to be similar and , at least by this criterion , no additional factors are required for FNR activity except for those required for cluster assembly ( e.g. , NifS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>WT-FNR requires the 21 [ 4Fe24S ] cluster for dimerization and DNA-binding , whereas the 21 mutant , FNR-D154A ( open circle ) , which is used in our in vitro experiments , does not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>In contrast to WT-FNR , FNR-D154A does not require a [ 4Fe24S ] cluster 21 for dimerization and DNA-binding under aerobic conditions ( Ziegelhoffer and Kiley , 1995 ; Lazazzera et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The presence of the Fe-S cluster in the anaerobically purified form of FNR is correlated with an increase in dimerization and specific DNA-binding ( 3 ) , compared with the aerobically purified form that lacks an Fe-S cluster ( 2 , 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>In the presence of O2 , FNR is converted into a monomeric , non-DNA-binding form containing a [ 2Fe-2S ] 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Nevertheless , this [ 2Fe-2S ] 2 + cluster species of FNR was inactive for site-specific DNA-binding and appeared to be in a monomeric form ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Here , we report the reactivity of the iron -- sulfur cluster of an FNR protein with a single amino acid substitution , S24F , adjacent to a Cys cluster ligand and show that the replacement inhibits both the O2-dependent and O2-independent steps of cluster conversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>We used a mutated version of FNR , lacking amino acid residues 2 to 15 ( FNRΔ2 -- 15 ) , whose cluster is more stable to O2 than the wt protein ( Yan and Kiley , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>The cluster is destroyed within seconds upon exposure to oxygen and an apoprotein devoid of the cluster is the product of the reaction2 ~ Interestingly , the discovery that an iron-sulfur cluster was the key to function did not occur until several mutants of FNR were analysed22 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>Thus , these data suggest that the FNR iron -- sulfur cluster is stabilized by the presence of Phe at position 24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Thus , the presence of Phe at position 24 does not impair the ability of FNR to bind an iron -- sulfur cluster with properties very similar to the [ 4Fe-4S ] cluster acquired by wild-type FNR under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>In the presence of oxygen , the cluster disintegrates , and the protein no longer specifically binds DNA2 ~ Thus , FNR provides a clear example in which a reactive iron-sulfur cluster is used to sense concentrations of the destabilizing agent , oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>FNR senses the presence of oxygen by disassembly of its oxygen-sensitive iron-sulfur cluster , converting the protein to a non ¬ DNA-binding monomer ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>FNR senses oxygen via an N-terminal iron -- sulfur cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>Thus , our work provides insight into the mechanism of the iron -- sulfur cluster rearrangements that switch FNR between active and inactive forms and implies that residues equivalent to position 24 in E. coli FNR , inter alia , play a key role in determining the sensitivity of the switch , thereby matching FNR activity to a range of O2 concentrations that require a response in the environmental niche inhabited by the bacterium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>In vivo transcription experiments indicate that other amino acids with bulky side chains at position 24 also protect the FNR iron -- sulfur cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>These data support the proposal that a bulky amino acid side chain at position 24 protects the FNR iron -- sulfur cluster from O2 and also suggest that enhanced FNR activity is detrimental during aerobic growth of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Thus , although an FNR dimer located at 250 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>Isolated FNR contains three Cys thiols ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>However , the domain containing the L-roll structure is involved in regulation of FNR activity , since it contains Cys122 , and this Cys appears to be one of the ligands for the Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>FNR is located in the cytoplasm and contains a 4Fe4S cluster in the sensory-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Concerning the cluster ligands , we have found no indication that the 2Fe cluster might use the single alternative cysteine ligand available in FNR by studying the Cys-16 to Ala FNR mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Of those FNR homologues which have an N-terminal cysteine cluster , AadR is the only one which does not have a Cys-X-X-Cys motif , the corresponding sequence being Cys-E-T-H-Cys in AadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Furthermore , until the precise role of the cysteine cluster in FNR has been elucidated , it is difficult to draw firm conclusions about the equivalent region in FNR homologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The FeS cluster can be reconstituted in vivo and in vitro w75 ,122,123 x In FNR from aerobically grown bacteria , a similar number of cysteine residues Žabout 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The N-terminal 30 amino acids of FNR are also involved in regulation and contain a cluster of cysteine residues ( 16Cx3Cx2Cx5C29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The amino-terminal cysteine cluster of FNR is partly conserved in FnrN but absent from FixK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The cysteine cluster of FNR is partially conserved in ANR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Cysteine cluster in FNR : Homology with metal-binding proteins The third functionally important domain in FNR is represented by a cluster of cysteine residues close to the N-terminus ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The N-terminal cysteine cluster of FNR shows homology to metal-binding domains of the type of metal proteins ( X = any amino acid ) ( Berg 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>However , FNR and FNR-like sensor proteins contain an N-terminal extension with three or four cysteine residues , three of which are part of a cluster of four conserved and essential cycteine residues in the protein w67 -- 69,71 x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>FNR contains 5 cysteine residues , four of which are arranged in a cluster close to the N-terminal end .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The ORF4 protein is the predicted product of o ~ of Rh & obium strain IC3342 ( Rhi ) The cysteine cluster The E. coli FNR protein has an N-terminal cluster of four cysteine residues , and a single internal cysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The molecular identity of the redox-sensitive element is unknown ; however , three cysteine residues located in cluster the a at N terminus and a fourth cysteine near the center of FNR are essential for the anaerobic response ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>For the function of FNR as a O2-dependent regulator three of the cysteine residues in the cluster and the residue outside the cluster are essential .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The cysteine residues serve as ligands for the O2-responsive w4Fe 4Sx cluster of FNR w70 ,72 -- 77x 3.2.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Anvno aad sequence homologies The possibility that the cysteine cluster of FNR might be a metal-binding site has been discussed previously ( Spiro ef a !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>The spacing of the cysteine res-idues in FNR ( three in the N-terminal tail and the fourth , Cys122 , in the regulatory domain ) are characteristic of a ferredoxin-type sequence indicating that FNR might contain an Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>A 4Fe4S cluster was also reconstituted in FNR in vitro with ferrous ions , cysteine and the NifS protein of Azotobacter Õinelandii w75x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>These properties indicate that the first cysteine residue in the A / - terminal cluster ( C16 ) is not essential for iron-binding , nor is it essential for the formation of FNR ,27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>YeiL also lacks the cysteine-rich N-terminal extension that provides three of the four ligands for the sensory iron -- sulphur cluster of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>because FNR function is abolished by genetic deletion of the cysteine cluster ( Spiro and Guest , 1988 ) and because the iron and reactive sulphydry !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>NO reacts anaerobically with the Fe ± S cluster of puri ® ed FNR , generating spectral changes consistent with formation of a dinitrosyl-iron ± cysteine complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>FNR possesses a cysteine cluster to which iron ( Fe ( H ) ) binds only in the active form [ 2,6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Iron and sulfide analyses and CD spectra of FNR preparations derived from reconstitution of a cluster into apoprotein also supports this cluster assignment ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>The results showing up to three NO per Fe ± S cluster to complete the overall reaction in FNR suggest a multistep process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The FNR protein , previously suggested to contain an Fe ion , is now known to carry a FeS cluster Žpresumably 4Fe4S .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>By oxidation of the FeS cluster , FNR is converted to the inactive state in a reversible process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Such a redox reaction would be conceivable for a supposed Cys4 · Fe rubredoxin-type cluster or an Fe-S center in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Cysteine/Histidine cluster in FNR and ( supposed ) metal proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Since the two cluster types of FNR utilize the same four cysteinyl ligands ( 4 ) , studies of FNR provide an unusual opportunity to compare the properties of two types of Fe-S clusters within the same protein environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Because IscR contained a series of Cys residues , we reasoned that it might contain a Fe-S cluster ; therefore , the protein was purified under anaerobic conditions by using the overexpression scheme previously shown to allow isolation of the FNR protein , which contains an oxygen-sensitive Fe-S cluster ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>It is important to recall that we tested IscR , a transcriptional regulator with an oxygen sensitive Fe-S cluster , an oxygen-resistant variant of FNR , another transcriptional regulator , and the Nuo and Sdh complexes that both contain buried oxygen resistant Fe-S clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>J Mol Biol 166:241 -- 247 Six S , Trageser M , Kojro E , Fahrenholz F , Unden G ( 1995 ) Reactivity of the N-terminal cysteine residues in active and inactive forms of FNR , an O2-responsive , Fe-containing transcriptional regulator of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>J Bactenoi 170 : 3650-3654 , Trageser , M , , and Unden , G ( 1989 ) Role ol cysteine residues anci of metal ions tntlie leguiatory kinctioning of FNR , thetranscriptional legulator of anaerobic respiration m Esctienchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>Trageser M & Unden G ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulatorof anaerobicrespiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_221">
Reviews 63:341 -358 Trageser M & Unden G ( 1989 ) Role of cysteine residues and metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic respiration in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></label>
<input id="dom_similarity_221", type="checkbox">
<div class="DOM_similar">
<li>[ 22 ] Trageser , M. and Unden , G. ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic respiration in Escheri-chia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li></div>
<li>FEBS Lett 340:59 -- 64 Trageser M , Unden G ( 1989 ) Role of cysteine residues and metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_224">
J Bacteriol 170:1589 -1597 Trageser M , Unden G ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1851949" target="_blank">1851949</a>]<br></label>
<input id="dom_similarity_224", type="checkbox">
<div class="DOM_similar">
<li>Trageser , M. , and Unden , G. ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , th transcriptional regulator of anaerobic respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Role of the cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of amaerobic respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Role of cysteine residues and metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic respiration in Escherichia coli 29 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></li>
<li>Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>]<br></li></div>
<li>Note that bacteria having multiple FNR homologues usually contain a typical FNR with a E -- SR motif and at least one other homulogue with a PE -- SR motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>These observations suggested that a fraction of purified FNR contains an internal disulphide bridge or at least one cysteine residue that is modified by a substituent that can be removed reductively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>However , the PE -- SR motif allows FLP and FIpA to recognize FLP ( CCTGA ) and FNR ( TTGAT ) sites , indicating that residues outside the recognition motifs and recognition helices can profoundly affect the recognition specificities of this type of regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>It has been suggested that side chains of these residues make interactions with the T-A and G-C base pairs at positions 1 and 3 of the TTGAT motif of the FNR box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>At the sensory level , the basic structural fold has been adapted in FNR and FLP by the acquisition in the N-terminal region of different combinations of cysteine or other residues , which bestow oxygen/redox sensing mechanisms that are poised according to the oxidative stress thresholds affecting the metabolism of specific bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>These observations suggest that the cysteine residues participate in iron-binding and that the interconversion of the ` aerobic ' and ` anaerobic ' species of FNR is a redox reaction with a protein-bound metal ion as the redox-sensitive site ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Role of cysteine residues and iron for 02-sensing by FNR In bacteria two forms of FNR , an aerobic and an anaerobic , can be differentiated ( Trageser & Unden 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The existence of an FNR-like protein having only two cysteine residues , in an organism that probably lacks iron , led to the speculation that FLP might operate the same transcriptional switch as FNR but at a later stage in the signal transduction pathway , thus by-passing the need for iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Indeed , the properties of FNR ( 27 , confirm that a high iron content is compatible with the presence of an intramolecular disulphide ( Table 1 , Fig. 1 ) and this implies that residues other than cysteine serve as Ngands for iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The FNR protein exists in two forms separable by non-reducing SOS-PAGE , and in studies with altered FNR proteins containing single substitutions at each of the five cysteine residues it was concluded that the fastermigrating form ( FNR ( 27 ) ) , possesses an intramolecular disulphide bond linking C122 to one of the cysteines near the Mterminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The presence of the two forms of FNR was investigated in at least two independent preparations of five mutant proteins , each containing a substitution in one of the five cysteine residues , together with studies on their ability to bind iron and the reactivity of their suiphydryi groups ( Table 1 ; Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Since this could involve disulphide formation or some other modification which might be related to redox-sensing , five FNR proteins with single substitutions In each of the cysteine residues were purified and characterized with respect to the presence of the two forms , the relationship between the iron and reactive sulphydryl contents , and the ability to interact with target DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>It was hoped that a clear link between iron-binding and specific sulphydryl ligands would emerge from studies with FNR proteins containing single substitutions at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Furthermore , the cysteinyl ligands of apo-FNR are sufficiently reactive to generate intramolecular disulfide bonds between Cys122 and one of the Nterminal cysteine residues ( Green et al. , 1993 ) , with the production of an oxidized or disulfide form of apo-FNR , designated [ 2SH , S-S ] in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Of the five cysteine residues in FNR , Cys-122 appears to be the most important in that it is absolutely required for disulphide bond formation , iron-binding and covalent modification ( Green et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Such a disulphide bridge might have been expected to affect the electrophoretic behaviour of all but two of the cysteine-substituted FNR proteins ( i.e. the two in which the residues participating in bridge formation are replaced ) , rather than all but one ( FNR-385 , C122A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Footprint studies indicated that FNR proteins lacking essential cysteine residues are impaired in their ability to protect FNR sites in the ndh promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>In the proteins from closely related bacteria , > 74 % of the amino acids are identical to those of FNR , and each of the five cysteine residues of FNR is conserved ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_247">
Properties of FNR proteins substituted at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9019397" target="_blank">9019397</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>]<br></label>
<input id="dom_similarity_247", type="checkbox">
<div class="DOM_similar">
<li>Properties of FNR proteins substituted at each of the ﬁve cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>1993 Properties of FNR proteins substituted at each of the ﬁve cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li></div>
<li>Proc R Soc Lond [ Biol ] 244:137 -- 144 Green J , Sharrocks AD , Green B , Geisow M , Guest JR ( 1993 ) Properties of FNR proteins substituted at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Green , J. , Green , B. , Geisow , M. and Guest , J.R. ( 1993 ) Properties of FNR proteins substituled at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_252">
B248 :79 -84 Green J , Sharrocks AD , Green B , Geisow M & Guest JR ( 1993 ) Properties of FNR proteins substituted at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></label>
<input id="dom_similarity_252", type="checkbox">
<div class="DOM_similar">
<li>Green , J. , Sharrocks , A.D. , Green , B. , Geisow , M. , and Guest , J.R. ( 1993 ) Properties of FNR proteins substituted at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Sharrocks , A.D , , Green , B. , Geisow , M. , and Guest , J , R , ( 1993 ) Properties of FNR proteins substituted at each of the five cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li></div>
<li>Of those proteins which have the conserved cysteine residues , ANR , HlyX , FnrN and FixK ( Bradyrhizobium japonicum ) are known to respond directly to oxygen in a manner that is , at least superficially , similar to the FNR protein itself ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>FNR proteins purified from JRG1728 lack nine W-termi-nal residues ( Trageser etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>It is something of a paradox that substitution of the cysteine residues of FNR renders the protein incompetent for transcriptional activation , whereas naturally occurring proteins which lack the cysteines are functional .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>This consideration brings up the question as to whether the 2Fe cluster uses the same four of the total of five cysteine residues present in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>This indicates that Fe of isolated FNR is liganded , similar to in vivo conditions , by cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>and the total cysteine content becomes five residues per mole of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>which indicates that FNR ( 27 ) is not formed by the conversion of one or more of its cysteine residues to cysteic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The residues involved in cAMPbinding by CRP and the cysteine residues in YeiL and FNR are shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The ability to sense and respond to oxygen limitation may be correlated with the presence of cysteine residues which , in the case of FNR , are thought to be involved in oxygen or redox sensing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The FixK proteins of R. meliloti and A. caulinodans do have internal cysteine residues , although only in the case of A. caulinodans is one of these near to a position corresponding to Cys-122 of the E. coli FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The cysteine ( Cys ) residues refer to the number of cysteine residues present at the N-terminal extension corresponding to Cys16 , 20 , 23 , 29 , plus the central Cys residue ( Cys122 ) of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The Fe-S clusters are bound by four cysteine residues ( Cys20 , Cys23 , Cys29 , and Cys122 ) of the ﬁve Cys residues of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>It is apparent that a clear distinction should be drawn between those FNR homologues which contain cysteine residues in the N-terminal region , and those ( the FixK proteins of R. meliloti and A. caulinodans and the orf4 gene product of Rhizobium strain IC3342 ) which do not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>However , upon exposure to air , FLP resembles FNR in forming intramolecular disuffide bonds between the N-terminal and internal cysteine residues ( Cys5 and Cysl02 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Recent experiments have shown that FNR is ( nactrvafed by deletion of a 28-residue segment ( residues 3-30 inclusive ) in the / V-terminal region , and thai at least one cysteine residue within this segment is essentiai for the O ; .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>( b ) Predicted structure of a FNR monomer based on CRP ( Schultz el al. , 1991 ) showing the N and C-termini , -LCB- , - helices ( cylinders A F ) , ] ~ - strands ( arrows 1-12 ) , and the four essential cysteine residues -LCB- C20 , C23 , C29 and C122 ) m  CRP FL ~ ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Site-directed substitution has previously shown that four of the cysteine residues -LCB- C20 , C23 , C29 and C122 ) are essential for FNR function , whereas the fifth ( C16 ) is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>In previous studies , site-directed mutagenesis was used to show that four of the five cysteine residues in FNR ( C20 , C23 , C29 and C122 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Disulphide bonding might also play a role in regulating the function of the FNR protein , since an / V-terminal segment containing three cysteine residues is essential for activating the target frd operon encoding fumarate reductase , an enzyme for anaerobic respiration ( Spiro and Guest , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li>
<li>Site-directed mutagenesis has demonstrated that the internal cysteine , and three of the four cysteines in the N-terminal cluster are essential for the normal function of FNR ( Spiro & Guest 1988 ; Sharrocks et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>By the N-terminal extension and a further essential cysteine residue close to the center ( Cys122 ) , FNR and FNR homologs are clearly differentiated from CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>It differs from CRP in being monomeric , in possessing a cysteine ~ rich amino-terminal segment that contains three of the four essential cysteine residues in FNR , and in binding equimolar amounts of iron4 ,6 , t2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>FNR carries an N-terminal extension of 29 amino acid residues containing four cysteine residues , three of which are essential ( Melville and Gunsalus 1990 ; Sharrocks et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>FNR is predicted to contain ail of the structural elements of CRP but it differs from CRP in being monomeric rather than dimeric , containing up to one atom of iron per monomer , and having an A / - terminal extension that includes three of the four essential cysteine residues ( Green ef ai , 1991 ; Sharrocks ef ai , 1990 ; Melville and Gunsalus , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>bC = Cys , H = His , X = variant amino acid residues Role of positive redox potential ( Eh > + 400 mV ) in the regulation by FNR The effect of ` aerobic repression ' of anaerobic re-spiration can be brought about also in the absence of 02 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The genetic evidence suggests that three other residues of FNR are involved in positive control : Ile-81 , Thr-82 , Asp ¬ 86 and Glu-87 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Although E. coli FNR has little sequence identity with Anabaena FNR , speciﬁc charged residues implicated in protein : protein contacts are somewhat conserved , particularly Arg237 and Arg238 ( both Lys in the Anabaena enzyme ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>When isolated from most strains including JRG1728 ( used here ) , nine W-terminal residues are cleaved from the FNR protein ( Trageser etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Functional domains inferred from the primary sequence of FNR FNR is a protein of 250 residues ( MrV30 000 [ 7 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Isolation of intact FNR fM , 30000 ) Previously the only available form of FNR was the protein of M. 29000 wfiich ( acks nine / V-terminaf amino acid residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>RESULTS The yeiL gene encodes a paralogue of CRP and FNR The YeiL protein contains 219 amino acid residues , and has a total molecular mass of 25 294 Da and a predicted pI of 8 ± 23 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Here , individual replacement of amino acid residues flanking the essential cysteine residues of FNR by Phe residues confirmed that FNR-S24F has greater aerobic activity than FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>It is notable that for both E. coli ( see above ) and gonococcal ( 24 ) FNR proteins , the presence of amino acid residues with bulky side chains ( Arg , His , Phe , Trp , or Tyr ) results in significant aerobic activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>FNR and its close relatives EtrA , ANR , FnrA and HIyX have acidic residues at these positions , whereas there is a preponderance of basic amino acids , particularly arginine , in the rhizobial proteins and AadR at the equivalent positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>While the residues indicative of the L-roll structure and an K-helix are conserved in FNR , the residues that contact cAMP are not conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Residues identical with FNR are printed in capitals and in bold face , similar residues in capitals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>48 % of the 50 amino acid residues are identical or conservatively substituted in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>HIyX shows a striking homology to FNR , 86 % of 237 equivalent residues are identical or conservatively substituted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>In endonuclease III and thymine-DNA glycosylase , the position equivalent to Ser-24 in FNR is occupied by nonpolar residues , Ile and Gly , respectively ( 34 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>The speciﬁc amount of ﬂuorescence per FNR protein showed a linear relationship to the number of accessible Cys residues ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The speciﬁc labeling varied by a factor of approximately 5 when FNR with one and ﬁve accessible Cys residues was labeled , conﬁrming that by this method up to ﬁve Cys residues per FNR become labeled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Up to ﬁve Cys residues per FNR become labeled by mBBr , and the number of reactive residues varies between one and two and about four Cys residues in vivo per FNR depending on the presence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Before the shift , about 20 % of the Cys residues ( corresponding to about one labeled Cys residue/FNR ) were accessible to mBBr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In particular , there were no signals equivalent to FNR labeled at only one or three Cys residues which would be characteristic for the presence of two or one intramolecular disulﬁde bonds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>mBBr binding was quantitatively calibrated using FNR with one , two , three , and ﬁve Cys residues in the thiol state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>In this way , FNR with one , two , three , or ﬁve Cys residues in the thiol state was available , which then reacted with mBBr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>FNR with an Mr of 32,000 is presumably labeled at three ( or two ) Cys residues by AMS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Again , there were no signiﬁcant signals with masses corresponding to that of unlabeled FNR or to the masses of forms of FNR with only one , two , or three alkylated Cys residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Alkylation of FNR in aerobic bacteria and counting the labeled residues by mass spectrometry showed a form of FNR with ﬁve accessible Cys residues , corresponding to apoFNR with all Cys residues in the thiol state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Labeling of FNR in vivo in aerobic bacteria with NEM showed indeed the theoretical value of ﬁve accessible residues predicted for apoFNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>ApoFNR can be differentiated from FNR containing Fe-S clusters by measuring the accessibility of the Cys residues ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Labeling reagents like mBBr which selectively label accessible Cys residues without removing the labile FeS cluster allowed identiﬁcation of apoFNR and differentiation from FeS-containing FNR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Earlier approaches to identify accessible Cys residues of FNR in vivo ( 10 ) were performed when the presence of FeS clusters in FNR was not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>By a combination of similarity search and phylogenetic analysis of the CRP/FNR protein family ( Figure 3 ) , we identiﬁed DNR orthologs in the genomes of various denitrifying b-proteobacteria , including three Ralstonia and two Burkholderia species , C. violaceum and Thiobacillus denitriﬁcans ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>lrvine , A.S. and Guest , J.R. ( 1993 ) Laetobacilhls casei contains a member of the CRP-FNR limfily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Mol Microbiol 12:153 -- 16 Irvine AS , Guest JR ( 1993 ) Lactobacillus casei contains a member of the CRP-FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>[ 84 ] Irvine , A.S. and Guest , J.R. ( 1993 ) Lactobacillus casei contains a member of the CRP-FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>In the presence of FNR , signals due to open complex formation at acsP1 are lost and signals appear between positions -11 and +3 , indicating open complex formation at the nrf promoter ( Fig. 2A , lane 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>FNR alone appears sufﬁcient for this , but NarL can enhance FNR-dependent open complex formation without assistance from any other factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Bulky Amino Acid Side Chains at Position 24 Stabilize the FNR Iron -- Sulfur Cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>We conclude that co-operative transcriptional activation by an FNR dimer binding upstream at -91.5 , in conjunction with one at -41.5 , occurs , and in a co-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>FNR heterodimers , in which one subunit carried a defective repression surface , revealed that the upstream subunit of the 240.5 dimer and the downstream subunit of the 293.5 dimer were most important for downregulating yfiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>[ 19 ] Green , J. and Guest , J.R. ( 1993 ) A role for iron in transcriptional activation by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>It has been proposed that contacts between two FNR dimers are important in the anaerobic activation of pﬂ expression ( Sawers , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Residue 315 is a part of the `` 287 determinant '' of a-CTD that is believed to contact CRP ( 6 , 29 ) and is also important for activation by FNR ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Previous experiments in studying FNR activation in vivo suggested that it was a reversible process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>This process could also be important for the reductive activation of FNR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>In vitro gel retardation analysis suggested that the primary interaction occurs at the downstream FNR half-site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>In this study , we analyzed the in vivo and in vitro interactions of these boxes with the FNR transcriptional regulator , in an attempt to elucidate their roles in nrdDG activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>In vivo and in vitro studies on the function of FNR as an O2 sensor-regulator suggest two potential steps of FNR inactivation by O2 , oxidation Žstep I. and destruction Žstep II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Fold activation was determined by dividing the β-galactosidase activity measured in each strain by the β-galactosidase activity present in the strain lacking FNR ( PK8203 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>In vivo b-galactosidase activity was measured from three independent isolates of strains containing pACYC184 plasmid-encoded mutant or WT-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>ment of any one of these four cysteines with a serine completely inactivates FNR activity , as measured by in vivo reporter gene assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>How is FNR activity regulated in vivo ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Studies using PhypA : : lacZ fusions carrying lesions in one or both FNR half-sites indicated that although some residual anaerobic activity was retained by the promoter containing only the downstream FNR half-site , both half-sites are required for maximal PhypA activity in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>However , it is shown that two FNR half-site motifs located upstream of the c70-de-pendent promoter are necessary for FNR-mediated hypB-E expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Structural considerations An alignment of the amino acid sequences of thirteen characterised members of the FNR group is shown in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Interestingly , an FNR mutant which has a single amino acid insertion within the Cys-X-X-Cys motif is severely impaired for both activation and repression ( Melville & Gunsalus 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Hence CRP makes active contacts via AR1 and AR2 , whereas FNR uses an extended AR 1 contact and AR3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>In this context , there is a core motif ( TTGAT ) for a potential FNR half-site at about -55 in the putative ttdABpromoter region ( Nesin et al. , 1987 ) , and an analogous CRP half-site motif ( GTGTA ) at - 130 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8371115" target="_blank">8371115</a>]<br></li>
<li>In FNR , the region analogous to CRP-AR3 , FNR-AR3 , appears to have a similar transcription-activation function , as amino acid substitutions within this region decrease transcription-activation of two class II promoters ( Fig. 2 ) ( West et al. , 1993 ; Bell and Busby , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>FNR-AR1 includes not only the region analogous to CRP-AR1 ( referred to as the 181 ± 191 loop in FNR ) , but also two other surfaceexposed patches ( amino acids 71 ± 75 and 116 ± 121 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Additionally , some of the fnr mutations that alter oxygen regulation are located at positions in FNR analogous to those found in the L-roll region and the dimerization helix of CAP [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>To screen for variant with further enhanced activity , we introduced D154A substitution which has been demonstrated to enhance dimerization between two FNR monomers in the presence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Several observations indicate that although a large portion of the AR1 surface is involved in FNR II mediated downregulation , the protein -- protein interactions that are involved are speciﬁc and easily perturbed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Several amino acids in both region 4.2 and in the basic cluster have been found to be important for interaction with transcription activators , such as PhoB , bacteriophage l cI protein , and FNR ( 29 , 30 ) .2 A set of substitutions between amino acids 570 and 580 of s70 severely impairs transcription-activation by PhoB at the pstS promoter ( 31 ) and also affects activation by cI at l PRM promoter ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>The C-terminal domain of s70 ( sCTD ) contains a putative helix-turn-helix motif ( region 4.2 , residues 570 to 613 ) which recognizes the 235 promoter element ( 15 ) and some of the transcription activators that bind near the 235 region , including l cI , PhoB , Ada , cyclic AMP ( cAMP ) receptor protein ( CAP ) -- activating region-3 ( AR3 ) , FNR , and phage Mu Mor proteins ( 1 , 21 , 23 -- 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Speciﬁc surfaces of the RNAP components RpoD ( 70 ) or the N-terminal domain of RpoA are sites of functional contact by some transcription factors that bind near the 35 promoter region , such as FNR , PhoB , or cI protein ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>In recent studies , a class of fnr mutants were identified that contain amino acid replacements located in the N-terminal region of the FNR protein ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>] VHb has a truncated C-terminal domain compared with the other two , and it is possible that a separate protein fulfils the FNR role [ XI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>To investigate this process , a fir mutant that encodes an altered FNR protein with three amino acid substitutions in the N-terminal domain was constructed by site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>This FNR * protein contains three mutations in the N-terminal domain : two amino acid substitutions , Asp-22 -- > Ser and Gln-27 - > Arg , plus the insertion of a serine after position 17 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>It is possible that a serine residue at this position allows HIyX and BTR to activate transcription at a promoter at which FNR is unable so to do .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>In this case , the FNR domain would be acting like mammalian cyt 6 , reductase , to ensure that the haemoglobin moiety remains in the oxy-form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>In contrast , the FNRDA154 protein contains an Asp - > Ala substitution in the region of FNR that corresponds to the dimerization domain of the highly homologous CAP protein ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Characterization of the dimerization domain in the FNR transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>FNR is believed to contain a redox/02-sensitive element for detecting the anaerobic state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>FNR is believed to contain a redox / 02-sensitive domain that senses the anaerobic state and converts the protein into an active conformation for DNA-binding ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Thus it would seem that FNR activation may be controlled by a redox-sensitive iron cofactor in the iron-FNR-DNA complex which senses and responds to the redox state of the celt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Thus it has been proposed that the A / - terminal cysteine cluster forms an iron-binding redox-sensing-domain and that FNR activation involves an allosteric transition initiated by a redox-sensitive iron cofactor ( Green ef ai , 1991 ; Green and Guest , 1993a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>T Arg180 G 4 ~ % Va1208 e r 2 ~ T S G l y l ~ ~ 8 '' ~ T T Lys188 '' -LCB- 9 1 4 ~ '' Gly216 '' - / 9 ... Arg185 ~ '' Met189 Arg213 - `` T - ' ~ '' Arg217 T w Rgure4 Diagrams showing the amino acid side chains in the DNA-recognition helices of FNR and CRP that are thought to interact with specific base-pairs in the core motifs of the corresponding half-site consensus sequences in FNR-and CRP-bindingsites1 ~ ,19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>For example , Paracoccus denitrificans has two paralogous FNR-like proteins , FnrP and NNR , which are each predicted to recognize FNR sites by the presence of E -- SR motifs in their DNA-recognition helices ( Fig. 4b ) and by the presence of potential FNR sites in FnrP - and NNR-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Based on these predictions , CRP and FNR homologues have been classified according to the amino-acid-sequence motifs in their putative DNA-recognition helices , where those containing the FNR motif ( E -- SR ) are regarded as FNR-like proteins and those containing the CRP motif ( RE -- R ) are regarded as CRP-like proteins ( Spiro , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Within the FNR family of regulators , we observed a correlation of two specificity-determining amino acids and contacting bases in corresponding DNA-recognition motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>These contacts were prevented by FNR binding to the secondary site , leading to inhibition of open complex formation and repression of ndh transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Anaerobic ndh repression may thus involve the binding of two pairs of FNR mono-mers upstream of the - 3 5 region , which prevents essentiai RNA poiymerase-DNA contacts in the upstream region as well as inhibiting RNA piolymer-ase function by direct FNR interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>It would thus appear that FNR mediates repression by altering the formation of DNA-RNA polymerase contacts in the - 5 8 to - 1 3 2 region and through inhibitory FNR-RNA polymerase contacts , rather than by directly blocking RNA polymerase binding at the promoter -LCB- promoter occlusion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The effects of mutations in the activator proteins and DNase footprinting suggest that the C-terminal domain of the alpha subunit of RNA polymerase may contact both CAP and FNR and simultaneously come close to the DNA between the two activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10715008" target="_blank">10715008</a>]<br></li>
<li>Arr positioned at -83 would be on the same face of the DNA helix as FNR at -41.5 and thus both regulators could contact RNA polymerase to activate transcription synergistically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Based on a comparison with the cAMP receptor protein ( CRP ) and its recognition sequence on the DNA , Spiro and Guest ( 1990 ) have proposed that a glutamic acid residue in the DNA-binding domain of FNR makes an important contact with the GC base pair in the DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>FNR differs from CRP in three amino acid residues ( Val-208 , Ser-212 and Gly-216 in FNR ) which are known to be critical for specific binding of CRP to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>As originally pointed out by Guest and co-workers ( Shaw et al. , 1983 ) , FNR is structurally similar to CAP , and the substitution of three amino acid residues in its DNA-binding domain is sufficient to allow anaerobic recognition of the CAP site in the lac promoter by the altered FNR protein ( Spiro and Guest , 1987b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Mutagenesis studies with FNR ( Spiro and Guest , 1987 ; Spiro et al. , 1990 ) have indicated that as in CRP , the amino acid residues contacting the FNR target site are located in the putative DNA-recognition helix ( c ~ F ) of FNR ( Plate 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Nevertheless , FLP is regarded as an FNR homologue because of its likely DNA-binding specificity and its possession of two conserved cysteine residues ( Irvine & Guest , 1993 ; see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The core interactions between the Glu ( position 2 ) and Arg ( position 6 ) residues and the conserved C-G and T-A base pairs at the FNR , FLP and CRP DNA-binding sites , arc indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>However , because the core motif and two residues in the DNA-binding motif ( E -- R ; Glu 184 an Arg188 ) are conserved relative to CRP and FNR , it is predicted that FLP will make the same contacts with its target , as shown in Fig. 4a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>However , FIpA and FIpB were designated FNR-like because : they are more similar to FNR than CRP when conservative amino acid substitutions are included in the comparison ; they retain the FNR-like DNA-recognition motif ( see below ) and lack the amino acid residues required for cAMP-binding ; and they retain one N-terminal cysteine residue ( Cysl5 ) and a single internal residue ( Cys112 ) corresponding to two of the four essential cysteine residues of FNR ( Plate 3a ; Gostick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>21 % identical to CRP and FNR , and it shares many features with the Lactococcus lactis proteins , including an FNR-like DNA-recognition helix ( even though it does not bind to the canonical FNR site , see below ) and only two cysteine residues ( Plate 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Based on sequence homology with CRP ( 8 ) , FNR consists of a C-terminal DNA-binding domain , which recognizes specific sequences within FNR-controlled promoters ( 9 ) , and an Nterminal sensory-domain containing 5 cysteine residues , 4 of which , Cys-20 , 23 , 29 , and 122 , but not Cys-16 , are essential for function ( 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>, two Synechocystis species ; NtcA Sy3 , Svnechococcus species ; NtcA Sy7 , Synechococcus species ; PrfA Li , Listeria ivanovii ; PflA Lm , Listeria mono ~ wtogenes ; PtrA Sy , ~ vnechococcus species ; Vfr Pa , Pseudomonas aeruk , inosa ; Ycf Pp , Porphyra purpurea : YeiL Ec , Escherichia coli ( 3 ) have a predicted DNA-binding specificity corresponding to that of FNR rather than CRP ; ( 4 ) fail to respond to cAMP ; ( 5 ) respond to oxygen limitation ; and ( 6 ) retain all or most of the characteristic ` cysteine signature ' of FNR , which comprises three essential N-terminal cysteine residues and a fourth essential cysteine within the [ 3-roll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Specific binding of purified FNR to the anaero-box was not detected in gel retardation assays [ 6 ] , although Guest and colleagues recently succeeded in detecting the DNA-binding activity by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>It was also shown that a Ci 22A substitution in the FNR protein prevents site-specific DNA-binding and in vitro transcription-activation and repression ( Sharrocks et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>In addition , the closely spaced FNR sites might also allow protein -- protein contacts between FNR dimers , such that cooperative binding and activation can not be excluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>Point mutations of the central GC pairs within the FNR box were suf ® cient to abolish protein ± DNA interaction , demonstrating the presence of a single FNR-binding site in Phmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>To further study the function of the putative FNR protein-binding site centered at position -42.5 , we constructed a series of site-specific mutations in this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>J Bacteriol 175:612 -- 619 Khoroshilova N , Beinert H , Kiley PJ ( 1995 ) Association of a poly-nuclear iron-sulfur center with a mutant FNR protein enhances DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>We were unable to purify the wild-type FNR protein in an active form by using the protocols described here , and thus , could not test its DNA-binding properties in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Nevertheless , some residual DNA-binding ability was observed after treatment of the FNR protein with either O2 or NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Whether this difference in DNA-binding relates to a different FNR structure due to nitrosylation requires further studies since the structure of the FNR protein has not yet been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The conservation of a potential nucleotide-binding site in FNR , HIyX and Orf240 stresses the significance of this domain in the O2-dependent regulation and a possible involvement of a nucleo-tide as a cofactor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The proposal that dimerization is su $ cient to increase site-speci cents c DNA-binding of FNR is further supported by the property of mutants that have substitutions of Asp154 ( located in the putative dimeri-zation helix , Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>From this similarity , it was hypothesized that FNR has both a regulatory ( resi-dues V30 ^ 160 ) and DNA-binding domain ( residues V170 ^ 250 ) , which has since largely been con cents rmed experimentally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>A supposed nucleotide-binding domain in FNR implies the participation of a nucleotide as an additional regulatory component .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>FNR is a cytoplasmic O2 responsive regulator with a sensory and a regulatory DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The p ~ dicted DNA-binding domain is underlined , and the ~ llowing residues of the K coH protein am highlighted : ( + ) , cysmine ~ sidues which are essenti ~ ~ r FNR activity ; ( , ) , residues which are implied to interact with RNA polymerase and ( =) , residues within the DNA-recognition helix which are required ~ r the sequence specific interaction-with-DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The FNR protein in addition contains a potential nucleotide-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The G residue has been proposed to form an important contact with a glutamic acid residue in the helix-turn-helix DNA-binding domain of the FNR protein ( Belief a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>The DNA-binding domain The DNA-binding domain of the E. coli FNR protein is well defined , both from genetic studies and on the basis of its similarity to the helix-turn-helix units of related DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>From the sequence it is further predicted that FNR contains a DNA-binding domain with a helix-turn-helix motif and a nucleotide-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Three characteristic regions of homology are present , a helix-turn-helix DNA-binding motif at the C-terminus ( residues 196 to 219 in FNR ) , a nucleotide cofactor binding domain ( residues 54 to 104 in FNR ) and a cysteine cluster ( residues 16 to 29 in FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The C-terminal domain of FNR is involved in sitespeci cents c DNA-binding via a putative helix-turn-helix motif ( residues 195 ^ 215 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The fourth subgroup within the major FNR group is designated DNR/FLP to denote a distinct group of regulators having sensory functions as well as potential FNR-like DNA-binding activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The DNA-binding domains are the most highly conserved regions of the FNR family of proteins as illustrated in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>CRP DNA-binding domains are aligned with the closely related P.aeruginosa DNR and the distantly related E.coli FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17068078" target="_blank">17068078</a>]<br></li>
<li>Each is predicted to have the same , or a very similar , DNA-binding specificity as that of FNR , as judged by the structures of their DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>One feature used in classifying the FLP of Lactobacillus casei as an FNR homologue was the presence of the FNR-like DNA-binding motif ( E -- SR ) in the putative recognition helix , c ~ r ( Plate 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Further mutagenic and structural studies will be needed to support this prediction and to identify other residues that contact DNA and define the DNA-binding specificity of FLE A major difference that might explain the divergent DNA-binding specificities of FNR and FLP is the presence of proline as the first residue of the DNA-recognition helix in FLP ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>( a ) The DNA-recognition helices of CRP , FNR and FLP are shown in hall-helical wheel format numbered from the first residue of the helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The DNA-recognition helix of FNR possesses no equivalent to Arg180 of CRP ; instead , a serine residue ( Ser212 ) makes contact with the first T-A pair of the FNR-site ( Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>These are different in each case but they represent major structural features such as the sensory-domain ( β-roll ) and dimer-interface ( α ) of CRP C and the DNA-recognition helix ( α ) of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>For the regions of FNR and FixK corresponding to the DNA-binding domain of CRP , a helix-turn-helix structure similar to that of CRP can be modelled ( Shaw et al. 1983 ; Cherfils et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>DNR and NnrR Core Regulons Co-Evolution of Regulators of the CRP/FNR Family and Their Recognition Motifs The HcpR recognition motifs identiﬁed in several bacteria demonstrated some diversity , which could be correlated with changes in the regulator DNA-binding helix-turn-helix domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>RNAP binding and that , in the presence of NarL , a new highly retarded complex appears that , most likely , contains NarL , FNR , and RNAP bound together to the ynfE promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>ScanAce analysis revealed that only yejG contains a putative FNR binding site , supporting the hypothesis that genes in group VII are regulated in an FNR-indepen-dent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>This site is centred at -94.5 and contains only one FNR half-site motif , which could make it a more sensitive reporter of impaired binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>This fragment contains a near consensus FNR binding site , with exact matches at the two core motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>This demonstrates the presence an function of the supposed helix-turn-helix motif in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The central motif TTGAT ( Spiro and Guest , 1990 ) of the half site of this potential FNR box is particularly well conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934931" target="_blank">7934931</a>]<br></li>
<li>The motif clearly matches the known FNR consensus binding motif of TTGAT ( n ) 4ATCAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>The rapid formation of the two inactive forms , [ 2Fe-2S ] FNR and apoFNR , suggests that both play a role in the bacterial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The Fe-S clusters in FNR can be detected by Mössbauer spectroscopy , which has been essential for characterizing the Fe-S-containing forms in vitro and in vivo ( 18 , 27 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The formation of active FNR is supposed to require essentially the same components identiﬁed in vitro , i.e. glutathione as the reducing agent , cysteine , Fe ions , IscS and possibly some other factors ( Eq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>The polylinker domain containing the FNR site insert was then isolated after digestion with BamHI and HindlIl and cloned into the large BamHI-HindIII fragment of pBR322 ( 18 ) to give plasmid pFNRCS ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The binding afinity of Fd and Fld for FNR is similar ( Kd 4 lM [ 32,33 ] ) , and recent crystal structures [ 34,35 ] and site-directed mutagenesis data [ 36 ] suggest that both proteins share a common binding interface on FNR that consists of two separate patches of positively charged residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>The point of contact between FNR and ( yT0has been mapped to a positively charged region of ( yT0 , residues 590 to 603 ( Lonetto et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The positively charged nature of this region is consistent with the observation that negatively charged residues are often replaced by positively charged residues ( e.g. Asp86Asn , Glu87Lys ) in AR3-defective FNR variants ( Bell and Busby , 1994 : Ralph et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The FNR protein also employs two contacts for initiating transcription from Class II promoters , but although they fulfil the same roles as those of CRR namely anti-inhibition and direct activation , they are fundamentally different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The two forms of FNR observed in unreduced samples are subscripted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The FNR group is currently the largest group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The four proteins with this domain architecture are CRP and FNR , which are both global regulators controlling a large number of genes , and two hypothetical proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>The genes fall into four subgroups , and transcription of the first two groups is dependent upon FNR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>S , ( 1991 ) The role of two surface exposed loops in transcription-activation by the Escherichia co / / CRP and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>The initial biochemical studies of FNR utilized protein preparations that were puri cents ed in the presence of air which , in retrospect , has given us important information about one of the inactive forms of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The region beyond the N-terminal 30 residues of FNR is similar to that of the well studied catabolite activator protein ( CAP ) [ 8 ] for which a crystal structure is known [ 9 ] ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>The occurrence of multiple protein -- protein contacts between a transcriptional regulator and RNAP has been most thoroughly characterized in the case of catabolite activator protein ( CAP , CAP/FNR family ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>The FNR protein was purified as a monomer of Mr 28,000 that contained nonheme iron at 2.05 ± 0.34 mol of Fe per FNR monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The protein ± DNA dissociation constants ( Kd ) for the three FNR forms were determined by gel retardation and repressor ± operator interaction analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The HIyX protein is identical to FNR in this respect , and ANR contains the three essential cysteines , with the same spacing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Rapid reaction kinetic analyses have shown that proteins in the FNR family with a tightly bound flavin prosthetic group form a characteristic charge -- transfer complex during flavin reduction [ 19,27 -- 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
<li>Since a symmetrical binding motif is typical of dimeric proteins , this similarity suggested that FNR might also be dimeric .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>All FNR proteins contain NAD ( P ) and FAD binding sites and it is here where the greatest sequence similarity among family members is seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>These enzymes are members of the ferredoxin-NADP1 reductase ( FNR ) family and they contain a FNR-like domain consisting of a FMN ( FAD ) - and a NAD ( P ) - binding region ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>Invariant residues defining a fingerprint for the FAD and NAD domains in proteins belonging to the FNR family [ 30,31 ] are conserved in the E. coli HCR , as indicated in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>In addition the yeast and the E. coli proteins contain a C-terminal domain homologous to the ferredoxin NAr ) P + reductase ( FNR ) family , FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>The E. coli globin-like protein HMP [ 6 ] is a soluble protein with two structural domains : the Nterminal portion shows striking similarity to VGB and leghaemoglobin , whilst the C-terminal portion is related to proteins in the ferredoxin-NADP + reductase ( FNR ) family [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7875569" target="_blank">7875569</a>]<br></li>
<li>The conserved motif found in SsuE suggests that the mechanism of flavin reduction may be similar to other members of the FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
<li>Despite being structurally very similar , members of the FNR family have disparate regulatory roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Therefore , the [ 3-roll structure is likely to be retained in FNR and in the other proteins of this group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Therefore , it seems likely that similar FNR ± FNR interactions may be responsible for the downregulation of all three promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Two genes of unknown function clustered into this group ( ybeD and ygjD ) and also clustered into the same group in our FNR study ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>All four members of this expression pattern clustered into the same group in our FNR study ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>It is likely that other bacterial groups also contain FNR homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>We anticipate that other members of the CAP/FNR superfamily also use this mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19359484" target="_blank">19359484</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">FNR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>While the second batch had a better 4Fe-FNR/background ratio ( ∼ 30 % of the Fe belonged to FNR ) , a larger fraction of the background produced a `` background doublet '' at 1.5 K .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Gene 48:33 -40 Green J , Trageser M , Six S , Unden G & Guest JR ( 1991 ) Characterization of the FNR protein of E. coli , an ironbinding transcriptional regulator ( submitted ) Gross R , Arico B & Rappuoli R ( 1989 ) Families of bacterial signal-transducing proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_3">
R.G. ( 1991 ) Identification and molecular characterization of a transcriptional regulator from Pseudorr ^ onas aeruginosa PA01 exhibiting structural and functional similarity to the FNR protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></label>
<input id="evo_similarity_3", type="checkbox">
<div class="EVO_similar">
<li>R.G. ( 1991 ) Identification and molecular characterisation of a transcriptional regulator from Pseudomonas aeruginos PA01 exhibiting structural and functional similarity to the FNR protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_5">
J Bacteriol 174:3474 -- 3478 Sawers RG ( 1991 ) Identification and molecular characterization of a transcriptional regulator from Pseudomonas aeruginosa PA01 exhibiting structural and functional similarity to the FNR protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="evo_similarity_5", type="checkbox">
<div class="EVO_similar">
<li>175:7938 -7944 Sawers RG ( 1991 ) Identification and molecular characterization of a transcriptional regulator from Pseudomonas aeruginosa PAO1 exhibiting structural and functional similarity to the FNR protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li></div>
<li>Structural considerations An alignment of the amino acid sequences of thirteen characterised members of the FNR group is shown in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>[ or oxidative stress FlpA and FlpB are 61 % identical to each other and 24 % identical to both CRP and FNR at the amino acid sequence level ( Plate 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>HlyX has 73 % identical amino acid sequence compared with the oxygenresponsive transcriptional regulator , FNR , which binds to a putative FNR binding site in the clyA upstream region ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The N-terminal cysteine cluster of FNR shows homology to metal binding domains of the type of metal proteins ( X = any amino acid ) ( Berg 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>In the proteins from closely related bacteria , > 74 % of the amino acids are identical to those of FNR , and each of the five cysteine residues of FNR is conserved ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The overall similarity of FNR to CRP , Orf240 and FixK amounts to 43 % , 48 % and 42 % identical and similar amino acid residues at equivalent positions of the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>48 % of the 50 amino acid residues are identical or conservatively substituted in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>HIyX shows a striking homology to FNR , 86 % of 237 equivalent residues are identical or conservatively substituted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The average activity is shown as a percentage of WT-FNR ( normalized to 100 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The predominant absorption ( 17 % of total Fe and outlined by the solid line ) of the central doublet in Figure 7A belongs to the [ 4Fe-4S ] 2 + cluster of 4Fe-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>The latter more closely resembles the Aquifex aeolicus FNR protein ( FNR Aa ) , which lacks the cysteine-rich sensory motif of typical FNR proteins and the CooA protein of Rhodospirillum rubrum ( CooA Rr ) , neither of which belongs to the major FNR group ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The ndh gene belongs to a small group of genes that are anaerobically repressed by FNR , others being cyo , sodA , fnr and possibly cyd ( Spiro et al. 1989 ; luchi and Lin , 1991 ; Cotter and Gunsalus , 1992 ; Hassan and Sun .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The primary structure of HlyX is 73 % identical to that of FNR ( MacInnes et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>One of the genes of this cluster ( off4 ) predicts a protein which is 29 % identical to FNR ( Fig. 3 ; Upadhyaya e t al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Vollack , K.U. , Hartig , E. , Korner , H. and Zumft , W.G. ( 1999 ) Multiple transcription factors of the FNR family in denitrifying Pxeudotnona , s stut ; eri : characterization of fourfi ~ r-like genes , regulatory responses and cognate metabolic processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>, Ha `` rtig , E. , Ko `` rner , H. and Zumft , W.G. ( 1999 ) Multiple transcription factors of the FNR family in denitrifying Pseudomonas stutzeri : characterization of four fnr-like genes , regulatory responses and cognate metabolic processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>In contrast , the FNRDA154 protein contains an Asp - > Ala substitution in the region of FNR that corresponds to the dimerization domain of the highly homologous CAP protein ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Proteins which are homologous to FNR but are clearly not analogous ( ie CRP and its relatives ) are not discussed , but are included in the phyiogenetic analysis ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Condusions Proteins structurally homologous to FNR ( and CRP ) of E. coli are widespread amongst Gram-negative bacteria , and there is at least one example from a Grampositive organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Sequence analysis of thefir gene indicates that FNR is structurally homologous to CAMP receptor protein ( CRP ) , the transcriptional regulator of catabolite-sensitive genes [ 2,7,8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Table 3 FNR homologous regulators and fnr-like genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>FNR is homologous to [ 3-strands 2 to 6 of this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Although NarL appears to be a typical transcription activator belonging to the response-regulator family , at most of its previously studied target promoters [ 4 ] , it co-activates promoter activity in conjunction with FNR , the master regulator for the adaptation of E. coli to growth in the absence of oxygen [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>[ 93 ] van Spanning , R.J.M. , de Boer , A.P.N. , Reijnders , W.N.M. , Spiro , S. , Westerho ¡ , H.V. , Stouthamer , A.H. and van der Oost , J. ( 1995 ) Nitrite and nitric oxide reduction in Paracoccus denitri cents cans is under the control of NNR , a regulatory protein that belongs to the FNR family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>[ 47 ] van Spanning , R.J.M. , de Boer , A.P.N. , Reijnders , W.N.M. , Wes-terho ¡ , H.V. , Stouthamer , A.H. and van der Oost , J. ( 1997 ) FnrP and NNR of Paracoccus denitri cents cans are both members of the FNR family of transcriptional activators but have distinct roles in respiratory adaptation in response to oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>( 1997 ) FnrP and NNR of P ~ lmroc cents ` u.s ' denitrific ~ ms are both members of the FNR family of transcriptional activa tots but have distinct roles in respiratory adaptation in response to oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>FnrP and NNR of Paracoccus denitriﬁcans are both members of the FNR family of transcriptional activators but have distinct roles in respiratory adaptation in response to oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>FNR shows sequence and functional similarity to CRP , the cyclic AMP receptor protein ( BELLet al. 1989 ; SPIROand GUEST1987 ) , although the mechanism by which FNR senses ana-erobiosis is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>The regions of greatest sequence similarity ( P % 0 ± 001 ) identiﬁed by pairwise comparisons ( Staden , 1982 ) between YeiL and either CRP or FNR are also indicated in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Fumarate and nitrate reduction regulatory ( FNR ) proteins are members of a superfamily of structurally related bacterial transcriptional factors ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Thus the designated FNR of Thiobacillusferrooxidans ( FNR Tf ) is most closely related to the DNR proteins and rightly belongs in the DNR/FLP subgroup .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>~ / \ -3 '' ~ ) Figure 1 Unrooted phylogenetic tree for 64 members of the CRP-FNR superfamily showing the positions of the FNR and FLP regulators within and relative to the major groups ( CRR FNR and Ntc ) and subgroups ( FNR , DNR/FLE FixK and FnrA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>) of members of the FNR and DNR/FLP subgroups .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Other members of the extended FNR family are the reductase components of the methane , alkene , phenol , and toluene diiron monooxygenases ( 59 -- 63 ) , a group of bacterial hydrocarbon oxidation enzymes that com ¬ FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>REFERENCES Anjum , M.F. , Green , J. and Guest , J.R. ( 2000 ) YieL , the third member of the CRP-FNR family of transcriptional regulators in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Microbiology ( 2000 ) , 146 , 3157 -- 3170 Printed in Great Britain YeiL , the third member of the CRP -- FNR family in Escherichia coli Muna F. Anjum , Jeffrey Green and John R. Guest Author for correspondence : John R. Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_43">
[ 90 ] Anjum , M.F. , Green , J. and Guest , J.R. ( 2000 ) YeiL , the third member of the CRP-FNR family in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></label>
<input id="evo_similarity_43", type="checkbox">
<div class="EVO_similar">
<li>[ 22 ] M.F. Anjum , J. Green , J.R. Guest , YeiL , the third member of the CRP-FNR family in Escherichia coli , Microbiology 146 ( 2000 ) 3157 -- 3170 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li></div>
<li>Indeed , FLP was the first member of the CRP-FNR family to be found in a Gram-positive organism ( Bowie et al. , 1991 : Irvine and Guest , 1993 : Gostick et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>lrvine , A.S. and Guest , J.R. ( 1993 ) Laetobacilhls casei contains a member of the CRP-FNR limfily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_47">
[ 84 ] Irvine , A.S. and Guest , J.R. ( 1993 ) Lactobacillus casei contains a member of the CRP-FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></label>
<input id="evo_similarity_47", type="checkbox">
<div class="EVO_similar">
<li>A.S , , and Guest , J , R , ( 1993 ) Lactobaciltus casei contains a member of the CRP-FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li></div>
<li>Mol Microbiol 12:153 -- 16 Irvine AS , Guest JR ( 1993 ) Lactobacillus casei contains a member of the CRP-FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Gene 90:15 -20 Irvine AS & Guest JR ( 1993 ) Lactobacillus casei contains a member of the CRP-FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>A analysis showed that YeiL is more closely related to members of the CRP -- FNR family ( 16 -- 22 % identical , 42 -- 45 % similar ) than to other known proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Sequence comparisons with representative members of each subgroup of the CRP -- FNR family showed that YeiL is : 22 % identical ( 42 % similar ) to the FNR of E. coli ; 19 % identical ( 45 % similar ) to the FLP of L. casei ; 19 % identical ( 44 % similar ) to the FixK of Rhizobium meliloti ; and 17 % identical ( 45 % similar ) to the CRP of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>A recent paper by Elvers et al. [ 52 ] describes a new nitrosative stress-responsive regulon in e-proteobacterium C. jejuni regulated by a member of the CRP/FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The complete genome of Escherichia coli ( Blattner et al. , 1997 ) was recently shown to encode a third member of the CRP -- FNR family ( YeiL ) when searched with a sequence proﬁle derived from a multiple alignment of ﬁve members of the family : CRP and FNR from E. coli ; FLP from L. casei ; and FlpA and FlpB from Lactococcus lactis ( Gostick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>A phylogenetic tree derived from 38 members of the CRP -- FNR family indicated that YeiL is a distant relative of the CRP group ( Gostick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Synopsis Results HcpR : Recognition Motifs and Core Regulon A member of the CRP/FNR family of transcription factors , HcpR has been initially identiﬁed as the regulator of the hcp gene encoding the hybrid cluster protein and the frdX encoding a ferredoxin-like protein in the Desulfovibrio species , anaerobic metal-reducing d-proteobacteria [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Site-directed deletion suggests that the extr tqL , ure 3 Schematic representations of the oxygen/redox-sensing switches operated by three members of the CRP-FNR super family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Various transcription factors of the FNR family have been described as NO-sensing regulators of denitriﬁcation : DNR in Pseudomonas species , NNR in Paracoccus denitriﬁcans , and NnrR in Rhodobacter sphaeroides and Bradyrhizobium japonicum [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Its hyphenated dyad symmetry resembles that of the binding sites for other regulators with helix-turn-helix motifs ( e.g , CRP and FNR ) but it differs from the known sites for other members of the GntR family which exhibit uninterrupted dyad symmetry , i.e. GntR -LCB- ATACnGTATACAAGTAT : Fujita and Miwa , 1989 ) and HutCpp -LCB- TTGJAATAtAc ; Hu et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>The FNR family includes such diverse proteins as mammalian nitric oxide synthase and cytochrome ( cyt ) b , reductase , spinach FNR and bacterial toluate and benzoate dioxygenases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>Invariant residues defining a fingerprint for the FAD and NAD domains in proteins belonging to the FNR family [ 30,31 ] are conserved in the E. coli HCR , as indicated in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>The DNA-binding domains are the most highly conserved regions of the FNR family of proteins as illustrated in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>All FNR proteins contain NAD ( P ) and FAD binding sites and it is here where the greatest sequence similarity among family members is seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>These enzymes are members of the ferredoxin-NADP1 reductase ( FNR ) family and they contain a FNR-like domain consisting of a FMN ( FAD ) - and a NAD ( P ) - binding region ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>PDR belongs to the ferredoxin ± NADP reductase ( FNR ) family of flavoproteins + [ 29,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>In addition the yeast and the E. coli proteins contain a C-terminal domain homologous to the ferredoxin NAr ) P + reductase ( FNR ) family , FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7821669" target="_blank">7821669</a>]<br></li>
<li>The E. coli globin-like protein HMP [ 6 ] is a soluble protein with two structural domains : the Nterminal portion shows striking similarity to VGB and leghaemoglobin , whilst the C-terminal portion is related to proteins in the ferredoxin-NADP + reductase ( FNR ) family [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7875569" target="_blank">7875569</a>]<br></li>
<li>[ 9 ] Green , J. , Scott , C. and Guest , J.R. ( 2001 ) Functional versatility in CRP-FNR family of transcription factors : FNR and FLP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Introduction The FNR protein of Escherichia coli is an oxygen-respon-sive member of the structurally related CRP/FNR family of transcription regulators ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>The L. casei FLP protein groups with the CRP rather than the FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Keywords : CRP -- FNR protein superfamily , iron -- sulphur proteins , transcriptional regulation , nitrogen starvation , stationary phase INTRODUCTION The CRP -- FNR family of transcription regulators contains at least 70 members that respond to a variety of metabolic and environmental signals to control a diverse range of physiological functions in an equally diverse range of bacteria ( Spiro , 1994 ; Guest et al. , 1996 ; Vollack et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>DISCUSSION The yeiL gene of E. coli was predicted to encode a new and distinct member of the CRP -- FNR family of transcriptional regulators that might have an iron -- sulphur centre and a reversible intra - or inter-molecular disulphide bond .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>FNR is predicted to contain all the structural elements of CRP , and both regulators constitute a family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Spiro , S. ( 1994 ) The FNR family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The FNR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_76">
[ 39 ] Spiro , S. ( 1994 ) The FNR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></label>
<input id="evo_similarity_76", type="checkbox">
<div class="EVO_similar">
<li>[ 3 ] Spiro , S. ( 1994 ) The FNR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li></div>
<li>The FNR protein is highly homologous to CAP ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2403539" target="_blank">2403539</a>]<br></li>
<li>The occurrence of multiple protein -- protein contacts between a transcriptional regulator and RNAP has been most thoroughly characterized in the case of catabolite activator protein ( CAP , CAP/FNR family ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>The only family members that diverge flom this arrangement are the FNR proteins of Bacillus suhtilis and Bacillus lichenifbnnis , where the sensory domains are located primarily at the C-termini ( Cruz-Ramos et al. , 1995 ; Klinger et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>However , there may be subtle differences in the DNA-binding specificities in at least some members of the FNR family ( Galimand et al. 1991 ; Sawers 1991 ; Green et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The DNR/NNR and NnrR proteins cluster phylogenetically in separate subgroups , separately from other family members including FNR , a global regulator of anaerobiosis in facultative anaerobic bacteria [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Some members of this family , like FNR itself , appear to function as sensors of oxygen availability , whereas others do not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The conserved motif found in SsuE suggests that the mechanism of flavin reduction may be similar to other members of the FNR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
<li>Despite being structurally very similar , members of the FNR family have disparate regulatory roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>It is of interest to examine the sequence alignment of FNR , HlyX and BTR ( Fig. 3 ) in search of lea bacterium belonging to the family Pasteurellaceae an + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>However , Gln-84 of FNR is replaced by serine in both HlyX and BTR , but in no other member of this family ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Recently , a general flavin reductase , Fre , was also shown to belong to the FNR family based on the mechanism of flavin reduction [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
<li>However , an amino acid sequence ( RXXS ) conserved within the spinach ferredoxin-NADP + reductase family of enzymes ( FNR family ) has also been identified in SsuE [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
<li>As mentioned , the conserved RXXS sequence found in the FNR family has also been identified in SsuE [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289450" target="_blank">17289450</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by FNR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Genes induced by NO treatment Functional Group Blattner ID Aminoacid biosynthesis b3774 Gene name Fold Name [ FNR/Fur regulation ] ilvC 5.1 Acetohydroxy acid isomeroreductase Biosynthesis of cofactors , prosthetic groups and cofactors b0782 moaB 2.4 b0783 moaC 3.0 b0785 moaE 2.4 b1680 sufS 2.2 b1683 sufB 2.2 b1684 sufA 6.2 b2527 hscB 1.9 b2528 iscA 2.6 Molybdopterin-biosynthesis , protein B Molybdopterin-biosynthesis , protein C Molybdopterin converting factor , subunit 2 Cysteine desulphurase , Fe-S cluster formation/repair protein Putative Fe-S cluster formation/repair protein IscA homologue [ Fur ] Cell envelope Putative fimbral-like protein 5.0 2.6 b0938 b2010 ycbQ dacD D-carboxypeptidase , penicillin binding protein 6b Cellular processes yajR fsr tehA tehB ydeR ydhK ydhP yeaR fimG Putative transport protein , drug-resistance translocase family Fosmidomycin resistance protein Tellurite resistance , component A Tellurite resistance , component B Putative fimbrial-like protein Putative fusaric acid resistance protein 3.5 3.8 3.0 2.5 2.3 2.3 b0427 b0479 b1429 b1430 b1503 b1645 Putative transport protein , drug-resistance translocase family Orf , hypothetical protein Fimbrial morphology 2.9 2.9 8.4 b1657 b1797 b4319 ediary metabolism b0342 Central interm lacA Thiogalactoside acetyltransferase Alkanesulfonate monooxygenase 2.2 4.0 5.5 1.7 6.7 ssuD ordL nemA nanE b0935 b1301 b1650 b3223 Probable FAD-dependent NAD-binding oxidoreductase N-ethylmaleimide reductase Putative N-acetylmannosamine-6-phosphate 2-epimerase DNA-metabolism SOS mutagenesis and repair ( 6 ) , [ FNR ] Endonuclease Cho 2.9 2.6 b1184 b1741 umuC cho Energy metabolism Putative ferredoxin , carnitine-metabolism Pyruvate dehydrogenase , decarboxylase component 3.7 5.5 3.7 3.1 b0044 b0114 b0115 b0116 fixX aceE aceF lpdA Pyruvate dehydrogenase , dihydrolipoyltransacetylase component Pyruvate dehydrogenase , 2-oxodehydrogenase complex : lipoam dehydrogenase component Phosphoglyceromutase 1 [ Fur , FNR ] ide 3.0 3.0 b0755 b0895 gpmA dmsB Anaerobic dimethyl sulfoxide reductase subunit B [ FNR b1007 b1109 b1630 b2552 b2710 b2711 ycdH ndh 3.7 5.7 Putative NADH : flavin oxidoreductase Respiratory type-II NADH dehydrogenase [ FNR ] Subunit of Na + - translocating NADH-ubiquinone oxidoreductase Flavohemoglobin [ FNR/Fur ] Flavorubredoxin NO reductase [ FNR/Fur ] NADH : flavorubredoxin oxidoreductase rnfD hmpA ygaK/norV ygbD / norW ygcQ 2.2 13.8 9.9 4.5 b2769 2.6 Putative electron-transfer flavoprotein subunit Hypothetical proteins ykgI ylbE ybeD yceP ydbD yncH bdm ydiV b0303 b0519 b0631 b1060 b1407 5.3 4.2 Orf , hypothetical protein Orf , hypothetical protein Orf , hypothetical protein Orf , hypothetical protein Orf , hypothetical protein Putative transferase 3.1 2.4 7.6 16.1 b1455 b1481 b1707 2.4 2.0 Putative biofilm-dependent modulation protein Orf , hypothetical protein , putative diguanylate Cyclase/phosphodiesterase domain Orf , hypothetical protein Orf , hypothetical protein Putative protein of the SWIM zinc finger family ydjO yedN yehQ yehR yjcF yjeP ytfE b1730 b1933 b2122 b2123 b4066 b4159 2.4 2.6 3.3 Orf , hypothetical protein Orf , hypothetical protein 3.3 3.9 Putative periplasmic binding protein Orf , hypothetical protein 4.4 54.5 b4209 Protein fate b0718 b2070 b3328 ybgQ ygeI hofG 2.7 3.1 2.2 Putative outer membrane protein , fimbrial export usher family Putative chaperonine Putative general protein-secretion protein G Protein synthesis Suppression of transcription of USHA b0481 b4171 b4371 ybaK miaA rsmC 9.6 2.0 6.5 Delta ( 2 ) - isopentenylpyrophosphate tRNA-adenosine transferase Putative ribosomal RNA small subunit methyltransferase C Purines , pyrimidines , nucleosides and nucleotides b0425 panE b2673 nrdH 2-dehydropantoate reductase 4.7 4.8 Putative component of ribonucleotide reductase 2 ; glutaredoxin-like protein [ Fur ] Putative component of ribonucleotide reductase Ribonucleotide reductase , alpha subunit Ribonucleotide reductase , beta chai 4.4 2.8 1.9 b2674 b2675 b2676 nrdI nrdE nrdF Regulatory functions glnK sdiA b0450 b1916 b2117 b2420 b2531 b3711 4.8 2.7 2.5 3.4 3.9 5.6 Nitrogen regulatory protein P-II 2 Transcriptional regulator of ftsQAZ gene cluster Molybdate metabolism regulator , third fragment Putative molybdate metabolism regulator Transcriptional regulator of the isc operon Putative LYSR-type HTH transcriptional regulator yfeS iscR yidZ soxS adiY b4062 b4116 16.0 Transcriptional activator of superoxide response regulon 3.8 Putative ARAC-type regulatory protein Transport and binding proteins b0451 b1186 b1472 b1592 b1801 amtB nhaB 2.2 2.4 3.7 1.6 3.2 4.4 Ammonium transporter Na + / H + antiporter , pH-independent yddL clcB yeaV mntH kup/trkD yjbB phnC Putative outer membrane porin protein Putative chloride channel Putative betaine/carnitine/choline transporter Manganese transport protein , Nramp1 - like protein b2392 b3747 b4020 b4106 Low affinity potassium-transport system Putative Na + / Pi cotransporter ATP-binding component of phosphonate transport 2.4 3.3 5.0 Viral functions rzpR ydfU yfjZ pinH yjgW IS30 transposase 2.7 6.4 3.1 2.4 2.5 7.4 b0256 b1362 b1560 b2645 b2648 b4274 Putative Rac prophage endopeptidase Orf , hypothetical protein Orf , hypothetical protein Putative DNA-invertase from prophage CP4-44 KLDE2-phage like element 4-acetoamido-4 ,6 - dideoxy-D-galactose transferas 2.2 b4404 b2750 b2751 b2762 b2764 b3136 ATP-sulphurylase , subunit 1 -2.8 -3.4 -2.7 -2.7 3 ' - phosphoadenosine 5 ' - phosphosulfate reductase Sulfite reductase [ NADPH ] flavoprotein alpha component Putative tagatose-6-phosphate aldose/ketose isomerase DNA-metabolism b3066 b3261 b3832 dnaG fis rmuC DNA primase -3.4 -2.3 -2.6 Fis transcriptional dual regulator DNA-recombination protein rmuC [ FNR ] etabolism Energy m acnB citD narG cybB rnfA ynjF pykA nuoE ygbE ansB nifE glpG yibO atpC Aconitase Citrate lyase acyl carrier protein , gamma chain b0118 b0617 b1224 b1418 b1627 -2.4 -3.7 -3.2 -3.8 -4.0 -3.8 Ñitrate reductase 1 , alpha subunit [ Fur ] Cytochrome b561 Putative Na + - translocating NADH-ubiquinone oxidoreductase subunit Putative cytochrome oxidase REFERENCES 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>TABLE VIII Regulatory pattern VIII : genes that exhibit similar levels during aerobic and anaerobic growth but decreased levels in an fnr-deficient strain p value PPDE ( p ) ( O2 , FNR vs. O2 , FNR ) 0.9999420 0.9996743 0.9995954 0.9995151 0.9994941 0.9994898 0.9994572 0.9994334 0.9993988 0.9993902 0.9993727 0.9993519 0.9991966 0.9991456 0.9991158 0.9990277 0.9990207 0.9989210 0.9987423 0.9986922 0.9986516 0.9986372 0.9986435 0.9985813 0.9985671 0.9985299 0.9984495 0.9982947 0.9982677 0.9981554 0.9981017 0.9980332 0.9979519 p value PPDE ( p ) ( O2 , FNR vs. O2 , FNR ) 0.7119911 0.6586042 0.7045299 0.7733437 0.7797786 0.6925384 0.7666014 0.6533494 0.7163046 0.6944700 0.6652543 0.6553178 0.7212742 0.6549812 0.7534451 0.7755093 0.7211014 0.7181570 0.7108355 0.7707909 0.7117933 0.6602961 0.7067209 0.7406371 0.6854618 0.7552905 0.7701812 0.7217079 0.7496791 0.7509828 0.6559196 0.7761592 0.6879241 Fold Fold ( O2 , FNR vs. O2 , FNR ) 7.62E-01 9.76E-01 7.90E-01 5.50E-01 5.29E-01 8.36E-01 5.72E-01 9.99E-01 7.46E-01 8.29E-01 9.48E-01 9.91E-01 7.28E-01 9.92E-01 6.15E-01 5.43E-01 7.28E-01 7.39E-01 7.66E-01 5.58E-01 7.62E-01 9.69E-01 7.82E-01 6.59E-01 8.64E-01 6.09E-01 5.60E-01 7.26E-01 6.28E-01 6.23E-01 9.88E-01 5.41E-01 8.54E-01 ( O2 , FNR vs. O2 , FNR ) 5.89E-06 6.93E-05 9.46E-05 1.23E-04 1.30E-04 1.32E-04 1.45E-04 1.53E-04 1.67E-04 1.70E-04 1.78E-04 1.86E-04 2.53E-04 2.84E-04 2.92E-04 3.32E-04 3.36E-04 3.86E-04 4.81E-04 5.08E-04 5.31E-04 5.37E-04 5.38E-04 5.71E-04 5.80E-04 6.01E-04 6.49E-04 7.46E-04 7.61E-04 8.32E-04 8.64E-04 9.12E-04 9.67E-04 ( O2 , FNR vs. O2 , FNR ) 1.04 1.01 1.04 1.07 1.10 1.03 1.10 1.00 1.03 1.02 1.01 1.00 1.04 1.00 1.11 1.08 1.03 1.04 1.03 1.13 1.04 1.01 1.05 1.04 1.02 1.06 1.07 1.05 1.10 1.04 1.00 1.05 1.02 ( O2 , FNR vs. O2 , FNR ) 5.43 6.53 4.34 2.42 4.13 4.05 4.15 1.95 1.89 2.18 2.69 2.35 2.55 3.26 5.24 2.43 2.26 2.40 1.94 5.55 2.43 5.31 3.20 1.82 2.10 2.18 2.25 2.91 4.94 1.67 4.58 1.73 1.87 Gene nikA nadC ycbO narX yjiL emrK ycfT yaaU yi41 ilvY lysU yqiG vjcP ybhK parC yiaY yfbS sdaB fhuC yjeT b1012 b2438 gcvT fumB yceL dfp umuC aroK dadX aroP aat yaaJ yigN FIG. 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Genes transcribed more efﬁciently in the presence of EsS Genes and Gene product b promoter regions a Transcription levels ( EsS/Es70 ratio ) d Stress response yieF sE ( 59 ) ; sS ( 19 ) sS , RcsBA ( 60 ) sS , RcsBA ( 60 ) Chromate reductase b3713 N.A. 1.50 1.51 osmB Intergenic region including osmB promoter Osmolarity-inducible lipoprotein b1283 b2688 1.51 2.21 gshA g-glutamate -- cysteine ligase ( glutathione-biosynthesis ) ahpF Alkyl-peroxidase reductase pphA Protein phosphatase macA Macrolide resistance efﬂux pump DNA and RNA-metabolism and modiﬁcation ppk Polyphosphate kinase , part of RNA degradosome , Intergenic region including xapA promoter xapA Xanthosine phosphorylase , nucleotide-synthesis/degradation gyrB B subunit of DNA gyrase recT Rac prophage , recombinase , in recET-lar-ydaCQ operon lar Rac prophage , restriction alleviation and modiﬁcation enhancement , in recET-lar-ydaCQ operon ydaC Rac prophage , in recET-lar-ydaCQ operon ydaQ Rac prophage , possible recombinase , in recET-lar-ydaCQ operon Polyamine-metabolism puuB g-glutamyl -- putrescine oxidase gabD Succinate semialdehyde dehydrogenase speB Agmatinase ( in arginine/putrescine-degradation pathway ) Transcription regulation Intergenic region including appY promoter appY Regulator of hyaABCDEF operon yciT Putative deoR-type transcription regulator ilvY Regulator of ilvC operon rhaS Regulator of rhamnose-transport Sugar metabolism treF Trehalase OxyR ( 61 ) sH ( 62,63 ) b0606 b1838 b0878 1.52 1.54 1.54 Regulator of intracellular sS concentrations ( 3 ) XapR ( 64 ) XapR ( 64 ) b2501 1.50 N.A. b2407 1.62 1.56 b3699 b1349 1.65 1.93 Fis ( 65 ) sS-Dependent in bioﬁlm cells ( 18 ) - growing b1348 1.93 b1347 b1346 1.74 1.66 as recT as recT sS ( 19 ) b1301 b2661 b2937 1.61 1.65 1.66 sS , Nac ( 9,66 ) 1.69 1.51 1.56 1.75 1.81 N.A. b0564 b1284 b3773 b3905 H-NS ( 67 ) H-NS ( 67 ) CRP ( 29 ) ; RhaS ( 68 ) sS ( 9,19 ) sS ( 19 ) sS ( 7 ) 1.53 1.55 1.56 1.57 b3519 b4230 b1897 b1901 ytfT Putative galactose ABC transporter otsB Trehalose-6-phosphate phosphatise araF Arabinose transporter , component of an ABC transport system glgP Glycogen phosphorylase , part of glgCAP operon gpmM Putative 2,3-bisphosphoglycerate mutase glvC Sugar transport phosphotransferase Other metabolic functions Intergenic region including hyaABCDEF promoter hyaA Hydrogenase small subunit ; in hyaABCDEF operon ( 69 ) ; sS ( 19 ) CRP , AraC CRP ( 70 ) ; sS controls glgCAP operon ( 19 ) 1.60 b3428 1.61 1.79 b3612 b3683 2.02 N.A. sS , AppY , anaerobic regulation by ArcA and NarP/NarL ( 20,71,72 ) as hyaA 1.68 b0972 Hydrogenase large subunit ; in hyaABCDEF operon Hydrogenase subunit ( nickel-binding protein ) ; in hyaABCDEF operon SecY-interacting protein Acetyl-muramic acid permease Transcriptional repressor of murQP Trans-aconitate methylatransferase Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon hyaB 1.66 b0973 as hyaA hyaF 1.50 b0977 syd 1.59 1.59 1.62 1.60 b2793 b2429 b2427 b1519 murP murR tam CRP , MurR ( 73 ) sS ( 74 ) ydhY FNR , NarL ( 75 ) 1.80 b1674 FNR , NarL ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Gene characteristics Gene Predicted Band no.a Gene function ( reference ) Regulation of corresponding gene ( reference ) productb molecular mass ( Da ) c PﬂB * 85,357 GadA * 52,685 1 Pyruvate formate lyase I ( 56 ) Glutamate decarboxylase ( 64 ) Induced by ArcA and FNR ( 61 ) 2 Positively regulated by rpoS , CRP , gadE , gadW , and gadX ; negatively regulated by H-NS ( 21 , 66 ) Positively activated by rpoS and by CRP ( 33 , 74 ) Positively activated by CRP ( 74 ) ; repressed by ArcA ( 60 ) Same transcription unit as gatZ Positively regulated by rpoS ( 33 , 46 , 71 ) Positively regulated by rpoS ( 2 , 33 , 46 , 71 ) 52,773 TnaA † 3 Tryptophan deaminase ( tryptophanase ) ( 22 ) 47,109 GatZ † 3 Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Quinone reductase ; response-to-oxidative-stress ( 45 ) Bacterial ferritin , DNA-protection during stationary phase ( 39 , 73 ) Transcriptional regulator ( 28 ) 30,812 20,846 18,695 GatY † WrbA * Dps * 4 5 6 , 7 , and 11 24,935 CsgD * 8 Positively regulated by rpoS , hns , and ompR ( 25 , 58 ) Same transcription unit as csgD Positively regulated by FNR ( 18 ) Outer membrane protein ; curli transport component ( 38 ) Outer membrane protein ; receptor for colicin S4 ( 49 ) 30,557 22,928 CsgG * OmpW * 9 10 a As in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>TABLE 3 -- continued Ratiosa Location of putative TS FNR Operon Function of operon O O 2 2 Ø2 NO3 NO3 ] NO2 NO2 FNR site ( s ) b determined site IDsc WT fnr fnr WT fnr WT ycaDM Putative transporter 1.63 2.18 2.06 1.11 1.35 1.44 1.35 3.5 Predicted Y/Y aldA NAD-linked aldehyde 8.04 7.02 4.67 1.90 5.36 1.12 4.95 10.5 Mapped Y/Y dehydrogenase cfa Cyclopropane fatty - 0.46 0.49 2.35 0.75 1.92 0.82 2.52 75.5 ( p1 ) Mapped Y/N acid synthase yeaU Putative tartrate 0.65 1.17 1.72 0.96 2.23 1.10 2.27 17.5 Predicted Y/N dehydrogenase yobA-yebZY Hypothetical proteins 2.08 1.31 2.99 2.03 3.10 1.29 2.93 26.5 Predicted Y/N mqo Malate dehydrogenase 13.52 3.10 4.19 1.79 4.67 0.84 2.82 11.5 Predicted Y/N kgtP - Ketoglutarate 8.22 5.47 4.61 3.27 4.13 0.81 2.87 10.5 Predicted Y/Y permease ygbA Hypothetical protein 0.98 1.20 1.39 10.14 17.44 6.68 13.36 67.5 Predicted Y/Y ygiS Putative transport 5.68 5.50 3.83 6.36 4.84 1.51 4.13 122.5 Predicted Y/N periplasmic protein ygiV Hypothetical protein 0.65 0.74 2.20 0.80 2.04 1.24 2.74 -116.5 Predicted Y/N yhbW Putative enzyme 9.40 8.53 2.96 2.93 3.99 1.17 4.13 30.5 Predicted Y/N gltBD Glutamate synthase 2.40 2.46 2.15 1.58 2.00 1.08 1.47 26.5 Mapped Y/N ychO Hypothetical protein 1.70 2.02 1.87 1.01 2.05 1.09 2.24 4.5 Predicted Y/N gadW Acid stress regulatory 1.37 1.22 2.61 0.96 1.81 0.89 2.31 116.5 Predicted Y/Y protein gadX Acid stress regulatory 0.11 0.13 2.19 0.37 1.38 0.64 2.15 159.5 Mapped Y/Y protein gadA Glutamate 0.01 0.02 2.13 0.11 1.27 0.78 3.02 11.5 Mapped Y/N decarboxylase dppABCDF Dipeptide-transporter 0.56 0.65 2.83 0.60 3.04 0.36 2.38 162.5 Mapped Y/N cspA Cold shock protein 12.08 2.70 2.39 2.13 2.22 1.31 1.50 5.5 wrt p1 , Mapped Y/N 56.5 wrt p2 yibIH Hypothetical protein 0.95 1.23 1.36 5.13 9.51 1.74 7.75 145.5 Predicted Y/Y ysgA Putative enzyme 1.88 2.29 2.99 1.16 4.20 1.29 4.64 88.5 Predicted Y/N a The reported ratios represent the ratio of transcript abundance in the RNA extracted from the stated growth condition ( aerobic , anaerobic , anaerobic plus nitrate , or nitrite ) and strain ( wild type or fnr ) compared with transcript abundance in the pool RNA ( anaerobically grown parental strain ) for the first gene in the operon displaying FNR dependence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR-binding sitesc Other reported regulation Y/N Y/Y Y/N CRP-activated , NarLrepressed 46 47 48 Half-sites 63.5 , and 39.5 - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant and promoter truncations - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and EMSAd - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis , EMSA , and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis , and EMSA ArcA - and DcuSR-regulated ModE-repressed 33.5 , 113.5 moaABCDE Molybdopterin-biosynthesis 0.50 1.06 0.34 1.52 0.43 0.97 0.59 hcp-hcr dmsABC Hybrid cluster protein and NADPH oxidoreductase Anaerobic dimethyl sulfoxide reductase 73.5 41.5 Y/Y Y/Y NarL - and NarP-regulated ModE-activated , NarLrepressed 0.72 1.12 0.66 30.26 1.60 47.81 2.54 0.12 0.27 0.10 0.10 0.07 0.70 0.07 49 50 -- 52 focA-pflB Formate transporter and pyruvate-formate lyase 0.24 0.30 0.34 0.35 0.22 0.87 0.31 41.5 Y/Y ArcA-activated 53 -- 55 narK Nitrate/nitrite-transport protein 0.20 0.25 0.14 14.30 0.08 12.07 0.12 41.5 Y/Y NarL-activated 56 , 57 41.5 Y/Y narGHJI 0.07 0.07 0.11 33.98 0.14 3.91 0.14 Nitrate reductase A NarL-activated 58 , 59 Y/Y fdnGHI 0.33 0.40 0.20 15.19 0.15 2.16 0.13 Formate dehydrogenase N 42.5 and 97.5 NarL-activated 55 , 60 fliZY Putative cell density - 1.08 0.72 0.23 1.34 0.32 0.62 0.08 responsive regulator of 28 napFDAGHBC Periplasmic nitrate reductase 0.51 0.47 0.39 0.95 0.27 4.04 0.26 No predicted ts 64.5 FlhDC-regulated NarL-repressed , N activated Y/Y Y/N 61 30 arP ¬ glpABC yfiD 0.19 0.16 0.16 0.55 0.21 0.71 0.18 0.07 0.02 0.04 0.66 0.06 1.22 0.08 Glycerol-3-phosphate dehydrogenase Putative formate acetyltransferase 41.5 N/N Y/Y GlpR-repressed , CRP-activated PdhR-repressed 62 40.5 and 93.5 63 , 64 65 N/N hypBCDE 0.20 0.15 0.21 0.59 0.22 0.67 0.19 Hydrogenase maturation Half-sites ansB 41.5 Y/Y Periplasmic L-asparaginase II 0.01 0.02 0.01 0.25 0.01 0.63 0.01 CRP-activated 55 , 66 41.5 Y/Y nirBDC 0.74 0.48 0.38 9.99 0.23 8.80 0.38 Nitrite reductase NarL/NarP-activated , IHF - , Fis - , and H-NS-regulated 10 , 67 0.50 0.29 0.19 0.03 0.80 0.63 0.65 4.73 0.40 0.24 0.10 0.03 1.11 0.29 0.33 0.32 0.53 0.31 0.14 0.05 0.20 0.78 0.22 0.05 0.28 1.38 0.11 0.05 Ferrous iron-transport Nickel transporter Putative cytochrome c peroxidase Periplasmic nitrite reductase No reported ts 41.5 61.5 41.5 Y/N Y/N Y/Y Y/Y Fur-repressed NikR-repressed feoAB 68 69 nikABCDE yhjA nrfABCDEFG 61 10 , 70 NarP-activated and NarLregulated TABLE 2 -- continued Ratiosa Position of FNR siteb FNR-binding Other reported regulation Product of operon Operon Evidence of FNR regulation Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>91.5 78 N/N glpTQ 0.46 0.28 0.30 0.64 0.35 0.68 0.34 GlpR-repressed , CRP-activated 82.5 5 Y/Y 79 N/N Y/Y ArcA - and NarL-activated ArcA-activated 80 ndh 81 Y/Y 0.30 0.99 6.60 1.21 6.16 1.18 5.80 ArcA-regulated acnA nrdAB hmpA Aconitate hydrase 1 Y/Y N/N Y/Y CRP-activated , ArcA-repressed Proposed anti-activation in stationary phase 2.58 4.16 75.91 1.43 2.23 20.42 2.51 3.79 3.02 1.36 5.99 1.68 2.02 6.32 2.04 1.18 1.73 29.89 2.44 3.04 149.7 40.5 82 75 Ribonucleoside diphosphate reductase 1 Flavohemoglobin H alf-site 5.5 37 , 85 Half-sites Fur-repressed 86 N/N IHF - and ArcA-activated ModE-activated ArcA-repressed N/N Y/Y N/N 57 , 87 83 88 83 , 84 Y/Y 89 Y/N IHF - , Lrp - , and YeiL-activated 22 N/N NorR-activated , NarL/NarP-repressed 73 N/Y IHF-repressed nitrite ) and strain ( wild type or fnr ) compared with transcript TABLE 3 Operons that display FNR regulation and that have putative FNR-binding sites upstream Ratiosa O2 O Ø 2 2 NO3 NO3 ] NO2 NO2 WT fnr fnr WT fnr WT fnr Location of putative TS FNR FNR site ( s ) b determined site IDsc Operon Function of operon Putative FNR-activated operons caiTABCDE Carnitine metabolism fixABCX Carnitine metabolism allB Allantoinase ybcW Hypothetical protein fes-entF-fepE Ferric enterobactin-synthesis and transport Citrate lyase Hypothetical protein Hypothetical protein Putative peptidase T Hypothetical proteins Outer membrane protein ; colicin S4 receptor ydcA Hypothetical protein ydfZ Hypothetical protein ynfEFGHI Me2SO reductase paralogue ynfK Hypothetical protein ydhYVWXU Putative oxidoreductase ydjXYZynjABCDE Hypothetical proteins pykA Pyruvate kinase II tar tap Chemotaxis cheRBYZ yecR Hypothetical protein yedM Hypothetical protein ackA Acetate kinase yfcC Putative S-transferase upp-uraA Uracil salvage yqjI Hypothetical protein aer Aerotaxis sensor receptor uxaCA Glucuronate degradation tdcABCDEFG Threonine degradation garPLRK Glucarate degradation yhbTS Hypothetical proteins yhbUV Hypothetical protein and putative collagenase Hypothetical protein Maltose degradation Low affinity phosphate-transport Putative 2-component regulator PTS-system Catalase Hypothetical proteins Hypothetical protein Transcriptional activator of cad operon fimBEAICDFGH Type 1 fimbriae yjiD Hypothetical protein tsr Chemotaxis protein yjjIW Hypothetical protein Putative FNR-repressed operons pdhR-aceEF-lpdA Pyruvate dehydrogenase and regulator Glucose dehydrogenase Hypothetical protein Cytochrome o ubiquinol oxidase Rhs element protein / hypotheticals Hypothetical protein Outer membrane protein X Putative cysteine hydrolase 41.5 and 90.5 197.5 185.5 142.5 and 21.5 143.5 Mapped Mapped Predicted Predicted Mapped Y/N Y/N Y/N Y/N Y/N 0.79 1.35 1.16 0.29 1.39 0.94 1.31 0.77 0.29 1.32 0.51 0.40 0.57 0.44 0.30 0.82 0.82 1.20 0.66 0.21 0.39 0.56 0.56 0.28 0.16 0.74 0.81 0.84 0.89 0.29 0.60 0.64 0.41 0.33 0.14 32.5 30.5 Predicted Predicted Y/Y Y/N Y/Y Y/N Y/Y Y/Y citCDEFXG ycbJ yccM pepT ymfED ompW 0.19 0.23 0.27 0.33 0.46 0.09 0.19 0.30 0.42 0.59 0.21 0.06 0.37 0.41 0.43 0.39 0.69 0.27 0.49 0.39 1.64 0.66 1.49 0.03 0.30 0.33 0.44 0.35 0.60 0.02 0.48 0.74 2.80 0.85 0.91 0.53 0.34 0.36 0.36 0.36 0.68 0.02 25.5 83.5 124.5 2 putative sites Predicted Predicted Predicted No predicted ts 0.46 0.01 0.06 0.21 0.42 0.49 0.02 0.05 0.15 0.14 43.5 35.5 64.5 29.5 167.5 Predicted Predicted Predicted Predicted Predicted Y/N N/Y Y/Y N/Y Y/Y 0.40 0.01 0.03 0.11 0.21 0.66 0.52 0.03 0.50 0.23 0.39 0.01 0.02 0.09 0.25 0.84 0.73 0.25 0.77 0.31 0.45 0.01 0.03 0.09 0.16 2.81 0.82 0.96 1.23 1.19 0.54 0.48 0.48 0.51 1.78 0.48 0.39 0.18 0.42 0.39 0.33 0.25 0.78 0.39 0.44 0.57 0.42 0.91 1.63 0.55 0.24 0.29 0.10 0.61 1.15 0.88 0.26 0.80 1.39 0.80 0.31 0.23 0.34 0.71 1.91 79.5 117.5 Predicted Predicted Y/N Y/N Y/N Y/Y Y/N Y/N Y/N Y/Y Y/N Y/N 0.53 0.42 0.28 0.55 0.42 0.28 0.16 0.42 0.40 0.62 0.63 1.01 0.53 0.67 0.91 0.73 0.84 0.93 1.38 0.64 0.46 0.43 0.05 0.17 0.29 0.27 0.23 0.31 0.47 0.27 160.5 113.5 14.5 1 putative site 18.5 61.5 150.5 and 200.5 51.5 Predicted Predicted Predicted No predicted ts Predicted Mapped Predicted Mapped 0.57 0.44 0.60 0.37 0.24 1.12 0.81 1.30 0.80 0.97 0.42 0.28 0.46 0.35 0.34 0.64 0.60 0.59 0.58 0.88 0.47 0.45 0.49 0.34 0.56 1.14 1.04 1.67 0.63 0.71 0.85 0.75 0.99 0.53 0.46 58.5 Predicted Mapped Predicted Predicted Predicted Y/N Y/Y Y/N Y/N Y/N 10.5 and 148.5 154.5 76.5 45.5 yhcC 0.22 0.62 0.37 0.62 0.84 0.85 0.93 0.82 0.21 0.54 0.34 0.71 0.68 0.73 0.57 0.75 0.32 0.49 0.48 0.45 0.38 4.29 0.70 0.67 0.28 1.54 0.75 0.50 Predicted Mapped Predicted Predicted 70.5 127.5 66.5 37.5 Y/N Y/N Y/Y Y/N malPQ pitA yieNM frvABXR katG yjdAZ yjdK cadC 0.34 0.27 0.04 0.20 0.15 0.43 0.94 0.50 0.65 0.25 0.37 0.23 0.22 0.16 0.11 0.39 0.73 1.36 0.78 0.29 0.32 0.21 0.15 0.31 0.14 1.45 0.32 0.76 0.08 1.54 1.82 0.22 1.19 0.09 1.98 Predicted Mapped Predicted Predicted Mapped 11.5 Y/N Y/Y Y/N Y/Y Y/Y 96.5 and 112.5 50.5 and 56.5 14.5 156.5 0.40 0.59 0.05 0.06 0.86 1.06 0.62 0.70 0.70 0.37 0.19 0.04 1.06 0.67 1.22 0.62 0.93 0.50 0.16 0.06 0.35 0.71 0.78 0.11 0.50 0.44 1.55 0.11 Y/Y Y/Y Y/N Y/N Mapped Predicted Predicted Predicted 71.5 148.5 80.5 56.5 3.30 0.81 2.97 1.05 2.64 0.74 0.63 Y/Y Mapped 50.5 Y/N Y/N Y/Y gcd yaiB 1.52 1.96 43.04 0.75 1.09 1.12 1.85 2.04 61.33 0.91 1.04 13.07 1.07 2.30 60.68 5.27 2.48 33.75 10.29 1.54 74.26 Mapped Predicted Mapped 31.5 wrt p1 , 15.5 wrt p2 62.5 192.5 E cyoABCD rhsDybbCylbH-ybbDb0501 ybiJ ompX 2.05 2.17 1.98 0.90 0.42 1.44 1.13 2.46 1.94 0.55 1.05 1.61 1.52 1.44 2.15 0.72 1.68 1.70 0.49 2.84 1.81 Predicted Predicted Predicted 61.5 and 128.5 2.5 and 62.5 3.5 Y/Y Y/N Y/N ycaC Y/Y 134.5 Predicted 0.79 3.01 1.73 2.54 1.41 2.99 1.17 2 , part B , be confirmed ; these were cydDC , hlyE , adhE , glpTQ , and arcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Links to RegulonDB and related resources Main page of RegulonDB Summary of data History of data Data overviews Evidences TractorDB Promoter analysis tools Textpresso search sample http://regulondb.ccg.unam.mx http://regulondb.ccg.unam.mx/chartForm.jsp http://regulondb.ccg.unam.mx/evidenceClassiﬁcation.jsp http://regulondb.ccg.unam.mx/html/TextpressoSample.jsp ( ` FNR ' as keyword and biological-process , ` regulation ' and ` pathway ' as speciﬁc categories ) biocomputo/Web/Regulon / GIs/16131824 Gene context tool Transcription initiation TSS experimentally mapped Predicted promoters TF Cell sensing classes RNA regulatory elements sRNAs Riboswitches Attenuators http://regulondb.ccg.unam.mx/data/HighTroughputTSSs.txt http://regulondb.ccg.unam.mx/LicenseRegulonDBp.jsp http://regulondb.ccg.unam.mx/CellSensing.jsp http://regulondb.ccg.unam.mx/data/sRNADataSet.txt http://regulondb.ccg.unam.mx/data/RiboswitchesPrediction.txt http://regulondb.ccg.unam.mx/data/AttenuatorsPrediction.txt Interestingly , for only two of them , additional TSSs had been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18158297" target="_blank">18158297</a>]<br></li>
<li>The transcriptional start sites and translational-initiation sites ( bold ) , the -35 and -10 hexamers of putative promoters ( boxed ) , alternative hexamers for P2 , , , A ( dashed overlined ) , ribosome binding sites ( RBS ) , and potential binding sites for FNR ( black box ) , CRP ( bold overlined ) , ArcA ( underlined ) and Fis ( dashed underlined ) , are shown for ( a ) acnA ( co-ordinates 701-1250 ; Prodromou et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Transcription depends upon CRP , the anaerobically triggered transcription activator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>We also have found that the timing and/or magnitude of this induction depends , in part , on the carbon regulator cyclic AMP ( cAMP ) receptor protein ( CRP ) , the oxygen regulator FNR , the glyoxylate shunt repressor IclR and its activator FadR , and several enzymes involved in acetate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>These seemingly include direct interactions by CRP and FNR to activate transcription initiation and indirect effects by acetate metabolic enzymes and transcription factors that control carbon ﬂux .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>Our set of 11 global regulators are transcription factors involved in carbon degradation ( mlc and lrp ) , redox status sensing ( arcA , narL and FNR ) , ion-transport regulation ( fur ) , environmental sensors ( cspA and CRP ) , a regulator of structural proteins ( hns ) and two general enhancers ( IHF and FIS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Thus the global regulators are proteins that control responses to changing food levels and carbon degradation ( mlc , lrp , CRP ) and transcription factors that respond to changes in redox status or ion levels of the cell ( arcA , narL , FNR , fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
Conclusions : Perturbation of the global regulator FNR and PurR functions as well as increase in purine nucleotide availability could affect the oxidative damage cell-death pathway initiated by topoisomerase cleavage complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>Our study showed that perturbation of the global regulator FNR and PurR function as well as increase in purine nucleotide availability , could affect the oxidative damage cell-death pathway initiated by topoisomerase cleavage complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li></div>
<li>FEMS Microbiology Reviews 22 ( 1999 ) 341 ^ 352 Oxygen sensing by the global regulator , FNR : the role of the iron-sulfur cluster Patricia J. Kiley a ; * , Helmut Beinert b a Department of Biomolecular Chemistry , 574 Medical Science Center , 1300 University Avenue , University of Wisconsin Medical School , Madison , WI 53706 , USA b Institute for Enzyme Research , Graduate School , and Department of Biochemistry , University of Wisconsin-Madison , Madison , WI 53706 , USA Received 2 June 1998 ; received in revised form 7 September 1998 ; accepted 7 September 1998 Abstract FNR is a global regulator that controls transcription of genes whose functions facilitate adaptation to growth under O2 limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_19">
Oxygen sensing by the global regulator , FNR : the role of the iron -- sulfur cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></label>
<input id="rp_similarity_19", type="checkbox">
<div class="RP_similar">
<li>Kiley , P.J. and Beinert , H. ( 1999 ) Oxygen sensing by the global regulator , FNR , the role of the iron ± sulfur cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Kiley PJ , Beinert H : Oxygen sensing by the global regulator , FNR : the role of the iron-sulfur cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Kiley PJ , Beinert H ( 1998 ) Oxygen sensing by the global regulator , FNR : the role of the iron -- sulfur cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li></div>
<li>Expression analysis of ( ulaG-lacZ ) and ( ulaA-lacZ ) transcriptional fusions in different genetic backgrounds under anaerobic conditions - Galactosidase expressiona ( Miller units ) Relevant phenotype Strain ( ulaG-lacZ ) ( ulaA-lacZ ) CAA L-Ascorbate CAA L-Ascorbate 40 130 JA134 Wild type JA184 IHF JA185 IHF JA188 RpoS JA211 YjfQ JA216 YjfQ CRP JA182 FNR a ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Application of CopraRNA to previously characterized sRNAs proposed and partially veriﬁed additional targets and functions for the sRNAs cyclic AMP activated sRNA ( CyaR ) , FNR regulated sRNA ( FnrS ) , RybB , RyhB , sugar-transport-related sRNA ( SgrS ) , and Spot42 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>© 1995BlackwellScienceLt FNR is a transcriptional activator which regulates a number of genes in response to anaerobiosis ( reviewed by Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Data Sources for the Escherichia coli Microarray Compendium Publication Title Present study : Large-scale mapping and validation of Escherichia coli transcriptional regulation from a compendium of expression profiles Integrating high-throughput and computational data elucidates bacterial networks Genome-scale analysis of the uses of the Escherichia coli genome : Model-driven analysis of heterogeneous data sets Transcriptome profiles for high-cell-density recombinant and wild-type Escherichia coli Amino acid content of recombinant proteins influences the metabolic burden response pH regulates genes for flagellar motility , catabolism , and oxidative stress in Escherichia coli K-12 Genome-wide analysis of lipoprotein expression in Escherichia coli MG1655 Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function Global transcriptional effects of a suppressor tRNA and the inactivation of the regulator frmR Global transcriptional programs reveal a carbon source foraging strategy by Escherichia coli estimates may be biased by the incomplete nature of this dataset .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></li>
<li>Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24982304" target="_blank">24982304</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804186" target="_blank">16804186</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_28">
Kang Y , Weber KD , Qiu Y , Kiley PJ , Blattner FR ( 2005 ) Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></label>
<input id="rp_similarity_28", type="checkbox">
<div class="RP_similar">
<li>Kang Y , Weber KD , Qiu Y , Kiley PJ , Blattner FR : Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>References [ 1 ] Y. Kang , K.D. Weber , Y. Qiu , P.J. Kiley , F.R. Blattner , Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Kang , Y. , Weber , K.D. , Qiu , Y. , Kiley , P.J. , and Blattner , F.R. ( 2005 ) Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Kang , Y. , Weber , K.D. , Qiu , Y. , Kiley , P.J. and Blattner , F.R. ( 2005 ) Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>Kang , Y. , Weber , K. D. , Qiu , Y. , Kiley , P. J. , and Blattner , F. R. ( 2005 ) Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169565" target="_blank">21169565</a>]<br></li></div>
<li>2 The abbreviations used are : sRNA , small regulatory RNA ; ArcA , regulator of aerobic respiratory control ; CAP , the catabolite gene activator protein ; FNR , regulator of fumerate and nitrate reduction ; Fur , ferric uptake regulator ; TIR , translational-initiation region ; IGR , intergenic regions ; SOD , superoxide dismutase ; GFP , green fluorescent protein ; fw , forward ; rv , reverse ; DTT , dithiothreitol ; Bis-Tris , 2 - [ bis ( 2-hydroxyethyl ) amino ] -2 - ( hydroxy-methyl ) propane-1 ,3 - diol ; CHAPS , 3 - [ ( 3-cholamidopropyl ) dimethylam-been identified , and a major class consists of trans-encoded gene regulatory RNAs that act by an antisense mechanism to activate or , more frequently , to repress translation of target mRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Significant electron flow is directed through both enzymes during glucose-limited aerobic growth ( Calhoun et al. , 1993 ) and recently it has been shown that NdhII is the Abbreviations : Arr , amino acid response regulator ; Fis , factor for inversion stimulation ; FNR , regulator of fumarate and nitrate reduction ; IHF , integrationhost factor ; NbpB ( NbpC ) , heat-stablefraction 1 ( 2 ) ; Ndhl ( Ndhll ) , NADH dehydrogenaseI ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_36">
A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K-12 adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20418396" target="_blank">20418396</a>]<br></label>
<input id="rp_similarity_36", type="checkbox">
<div class="RP_similar">
<li>A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL and NarPQ as Escherichia coli adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19767843" target="_blank">19767843</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_39">
Constantinidou , C. , Hobman , J.L. , Griffiths , L. , Patel , M.D. , Penn , C.W. , Cole , J.A. , and Overton , T.W. ( 2006 ) A reas-sessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></label>
<input id="rp_similarity_39", type="checkbox">
<div class="RP_similar">
<li>[ 2 ] C. Constantinidou , J.L. Hobman , L. Grifﬁths , M.D. Patel , C.W. Penn , J.A. Cole , T.W. Overton , A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli K12 adapts from aerobic to anaerobic growth , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_41">
CW , Cole JA & Overton TW ( 2006 ) A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></label>
<input id="rp_similarity_41", type="checkbox">
<div class="RP_similar">
<li>References Constantinidou C , Hobman JL , Grifﬁths L , Patel MD , Penn CW , Cole JA & Overton TW ( 2006 ) A reassessment of the FNR regulon and transcriptomic analysis of the effects of nitrate , nitrite , NarXL , and NarQP as Escherichia coli adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li></div>
<li>For example , FNR has been implicated in the peroxide stress response as an anaerobic activator of cytochrome c peroxidase ( CCP ) in a number of bacteria including Neisseria gonorrhoeae , Paracoccus denitriﬁcans and Pseudomonas stutzeri ( Turner et al. , 2003 ; van Spanning et al. , 1997 ; Vollack et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated Glycolysis , citric-acid-cycle , and respiratory chain genes times following the introduction of air 5 min 10 min 15 min 60 min 6.0 6.7 6.9 4.7 3.6 5.7 6.1 3.2 Relevant regulatory proteins aceE pyruvate dehydrogenase ( decarboxylase component ) aceF pyruvate dehydrogenase ( dihydrolipoyltransacetylase component ) acnA aconitase A acnB aconitase B cyoA cytochrome bo terminal oxidase subunit II cyoB cytochrome bo terminal oxidase subunit I cyoC cytochrome bo terminal oxidase subunit III cyoD cytochrome bo terminal oxidase subunit IV cyoE heme o synthase gltA citrate synthase gpmA phosphoglycerate mutase icdA isocitrate dehydrogenase ( NADP ) lpdA lipoamide dehydrogenase ( E3 monomer ) mdh malate dehydrogenase mqo malate dehydrogenase ( quinone ) ndh NADH dehydrogenase II pdhR pyruvate-responsive transcription regulator poxB pyruvate oxidase sdhA succinate dehydrogenase flavoprotein sdhB succinate dehydrogenase iron-sulfur protein PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) ArcA ( ) FNR ( ) Cra ( ) CRP ( ) SoxRS ( ) ArcA ( ) Cra ( ) CRP ( ) Fis ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA * ( ) Fur ( ) P1 , ArcA * ( ) P2 , Cra * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) Plpd-ArcA * ( ) FNR ( ) CRP ( ) PdhR ( ) Fis ( ) ArcA ( ) CRP ( ) FlhD ( ) ArcA ( ) FNR * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) SoxRS ( ) MarR ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2-CRP ( ) P1 - ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) 2.0 3.1 2.6 2.9 6.8 8.6 9.9 7.3 5.2 2.7 1.9 5.0 3.3 3.2 7.4 3.2 5.8 7.3 9.1 11.7 7.8 6.5 4.7 2.1 6.1 4.9 10.4 12.9 9.2 5.7 4.4 1.5 4.8 8.6 10.0 7.9 6.3 4.0 2.1 5.3 3.0 2.4 4.8 2.9 3.5 3.9 3.4 6.9 4.2 4.6 3.5 8.6 5.0 5.0 2.2 3.5 2.4 6.6 3.1 3.0 4.7 2.1 3.7 6.8 9.9 5.5 9.1 5.8 8.7 4.3 2.9 5.8 2.8 sdhC succinate dehydrogenase membrane protein sdhD succinate dehydrogenase membrane protein sucA 2-oxoglutarate decarboxylase 16.6 4.6 3.2 3.6 7.8 1.9 2.0 2.1 2.1 3.2 3.6 3.1 sucB dihydrolipoamide succinate transferase sucC succinyl-CoA synthetase , subunit sucD succinyl-CoA synthetase , subunit 3.5 4.7 2.7 TABLE 2 Transcripts encoding proteins involved in the oxidative stress response that are present in increased abundance after the introduction of air to anaerobic cultures of E. coli MG1655 Fold regulation ( at least 2-fold at one or more time point ) is the ratio of transcript levels at the indicated time after the introduction of air to the transcript levels at t 0 ( the initial anaerobic state ; p 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Other Gram-negative bacteria In a number of other species of Gram-negative bacteria there is indirect evidence for the presence of FNR homologues , usually from the observation of FNR boxes in the upstream regions of genes involved in denitrification ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Vollack , K.U. , Hartig , E. , Korner , H. and Zumft , W.G. ( 1999 ) Multiple transcription factors of the FNR family in denitrifying Pxeudotnona , s stut ; eri : characterization of fourfi ~ r-like genes , regulatory responses and cognate metabolic-processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Transcription of a further group of genes , including yeaR , yoaG , yibH , and yibI encoding proteins of unknown function , is partially FNR-dependent and strongly induced by nitrate but also strongly induced by nitrite in an FNR mutant , suggesting that multiple regulatory circuits regulate the response of these genes to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Dominy CN , Deane SM , Rawlings DE ( 1997 ) A geographically widespread plasmid from Thiobacillus ferrooxidans has genes for ferredoxin - , FNR - , prismane - and NADH-oxidoreductase-like proteins which are also located on the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Green , J. , Sharrocks , A.D. , MacInnes , J.I. , and Guest , J.R. ( 1992 ) Puriﬁcation of HlyX , a potential regulator of haemo-lysin synthesis , and properties of HlyX : FNR hybrids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_50">
Green , J. and Baldwin , M.L. ( 1997a ) HIyX , the FNR homologue ot Actinobacillus pleurolmeumoniae , is a [ 4Fe 4S ] - containing oxygen-responsive transcription regulator that anaerobically activates FNR-dependent Class i promoters via an enhanced ARl-contact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></label>
<input id="rp_similarity_50", type="checkbox">
<div class="RP_similar">
<li>Green , J. , and Baldwin , M.L. ( 1997a ) HlyX , the FNR homologue of Actinobacillus pleuropneumoniae , is a [ 4Fe-4S ] - containing oxygen-responsive transcription regulator that anaerobically activates FNR-dependent Class I promoters via an enhanced AR1 contact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li></div>
<li>To test if tandem construct ( FNRD154A ) 2 also activates other FNR dependent promoters under aerobic conditions , we measured b-gal activity of FNR activated PnarG-lacZ , PdmsA-lacZ , and FNR repressed Pndh-lacZ ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>The recent description of mutant proteins ( FNR * ) that are active under aerobic growth conditions is a potentially important development ~ 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The protective effect of Δfnr mutation was greater under low oxygen conditions The genes suppressed by FNR directly or indirectly via the sRNA FnrS [ 26,27 ] include many aerobic metabolic genes as well as genes involved in removal of reactive-oxygen-species such as katE , sodA and sodB [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>The product of the aadR gene is structurally related to FNR , and is proposed to be involved in regulating the biosynthesis of benzoate-CoA ligase and aromatic acid-CoA ligase II , which are required for benzoate and 4-hydroxybenzoate-catabolism respectively ( Dispensa et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The identification of an FNR box in the regulatory region of the lux regulon supports the notion that FNR is involved in the regulation of luminescence ( Fig. 2 ; U. Winkler , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The genetic evidence suggests that three other residues of FNR are involved in positive control : Ile-81 , Thr-82 , Asp ¬ 86 and Glu-87 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Serratia Genes Product or function Effect of FNR Additional regulators involved Regulatory ( anaerobically ) ( positive or negative effect ) signals Reference 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Proteins related to FNR are involved in regulating rather disparate aspects of bacterial physiology ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Global regulatory proteins CRP , FNR , and H-NS are involved in the synthesis of carnitine dehydratase activity ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>location of bases involved in discrimination between FNR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>NO is a damaging agent of iron-sulfur clusters , and apart from the wide variety of iron-sulfur-containing proteins involved in relevant pathways , the formation of these clusters is essential at least for the transcriptionally active form of regulators such as FNR , IscR , and SoxR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>NO is known to affect several regulons , namely those that involve iron proteins as transcriptional regulators ( Fur , FNR and SoxR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>In contrast , strong responses to anaerobiosis and glucose ( a source of pyruvate ) mediated by global regulators ( ArcA , FNR and CRP ) are likely to be less beneficial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Mechanism of regulation of 8-hydroxyguanine endonuclease by oxidative stress : roles of FNR , ArcA , and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>Gene regulators ArcA , FNR , NarX Toxins Colicin E1 production Žcea kil .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Three exceptions were the ferric enterobactin-synthesis and transport operon , fes-entF-fepE , ompW encoding an outer membrane protein and colicin S4 receptor , and cadC encoding the transcriptional activator of the cad operon ( but why was the cad operon itself apparently not FNR regulated ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Of the unknown genes that are highly regulated by FNR , ynfE is part of an operon that encodes a putative DMSO reductase ( 54 ) , ydhY is predicted to be an Fe-S-containing oxidoreductase , ydfZ is postulated to be involved in selenium delivery ( 44 ) , and ydjX is a putative membrane protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>FNR also plays an important role in the cellular response-to-oxidative-stress , as demonstrated by membership in the SoxR regulon and by the ability of FNR to confer methyl viologen resistance [ 54 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>SDS/PAGE , Protein , Fe Content , and Amino Acid Analysis of Purified FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The production of L-asparaginase II in E. coli is known to be subject to positive regulation by two pleiotrophic regulatory proteins : the FNR protein ( 20 ) , which activates a number of genes during anaerobiosis ( 18 , 39 , 44 ) ; and CRP ( 8 , 35 ) , which controls the initiation of transcription of genes in various catabolite pathways ( 5 , 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The FNR protein activates transcription in response to anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Perspectives The transcription of nik genes is stimulated by the global regulator FNR under anoxic conditions , and negatively controlled by a new repressor protein , NikR , responding specifically to Ni .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>the transcription activator FNR ( fumarate-nitrate regulation ) in response to anaerobiosis , and by transcription regulators NarL and NarP in response to nitrite and nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>The ula regulon also seems to be controlled by the global regulators cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) and fumarate nitrate reductase regu - lator ( FNR ) , since mutations in the corresponding genes abol - ished or delayed , respectively , growth on L-ascorbate ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>The DNR/NNR and NnrR proteins cluster phylogenetically in separate subgroups , separately from other family members including FNR , a global regulator of anaerobiosis in facultative anaerobic bacteria [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Synopsis Results HcpR : Recognition Motifs and Core Regulon A member of the CRP/FNR family of transcription factors , HcpR has been initially identiﬁed as the regulator of the hcp gene encoding the hybrid cluster protein and the frdX encoding a ferredoxin-like protein in the Desulfovibrio species , anaerobic metal-reducing d-proteobacteria [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Abbreviations : CRP , CAMPreceptor protein ; FNR , fumarate and nitrate reduction regulator ; IHF , integration host factor ; ODH , 2-oxoglutarate dehydrogenase ; PDH , pyruvate dehydrogenase ; SCS , succinyl-CoA synthetase ; SDH , succinate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Abbreviations : CRP , cAMP receptor protein ; FNR , fumarate and nitrate reduction regulator ; MALDI-TOF , matrix-associated laser desorption ionization-time of ﬂight ; MBP , maltose-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>PItYSIOLOGY VOL 44 ISBN 0-12-027744 - I Copyright © 2001 Academic Press All rights of reproduction in any tiwm reserve ABBREVIATIONS cAMP CAP CRP FLP FNR Adenosine 3 ' ,5 ' - monophosphate Catabo -RCB- ite gene activator protein cAMP receptor protein FNR-like protein fumarate and nitrate reduction regulator 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>A search revealed several potential binding sites for CRP and FNR ( the cyclic AMP receptor protein or catabolite gene activator protein and the regulator of fumarate and nitrate reduction or anaerobic gene activator protein , respectively ; not shown ) , two putative sites for Fur ( the ferric-uptake regulatory protein ; GATAATGATA/T , half-site concensus ; Cardayre and Neilands , 1991 ) and several putative stem-loop structures , including a potential rho-independent terminator for cysB , AG = -80 kJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1541275" target="_blank">1541275</a>]<br></li>
<li>These include the transcription factor FNR that mediates anaerobic regulation ( Unden and Schirawski , 1997 ) , the cAMP receptor protein ( CRP ) , which activates transcription of catabolic operons ( Kolb et al. , 1993 ) , and the nucleoid-associated protein H-NS ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Expression of a caiF-lacZ operon fusion was subject to FNR regulator-mediated anaerobic induction and cyclic AMP receptor protein activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Its expression is also dependent on the global regulatory proteins mediating catabolic repression via cyclic AMP receptor protein ( CRP ) , or anaerobiosis via fumarate nitrate reduction ( FNR ) transcriptional regulator , and the nucleoid-associated DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Moreover , it is subject to global control by the same regulatory proteins which are known to modulate expression of carnitine-metabolism , i.e. , the cyclic AMP ( cAMP ) receptor protein ( CRP ) which mediates the activation of catabolic operons , the transcriptional regulator FNR that is responsible for anaerobic induction , and the DNA-binding protein H-NS ( 19 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine-metabolism ) was controlled by protein regulators such as FNR ( transcriptional regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine-metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>However , arcB ( that codes for the autopho-sphorylating signal membrane enzyme that senses oxygen availability ) and fnr ( coding for FNR protein , the main aerobic-respiration regulator ) showed a slight decrease in their transcription levels compared to aerobic cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>Speci ® cally , NO inactivation of FNR results in upregulation-of-synthesis of Hmp , a protein with key roles in NO resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Deﬁning the mechanisms that control FNR protein levels is important in understanding the global response to O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>FNR binds iron as a cofactor which most likely is involved in the O2-sensing by the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>MATERIALS AND METHODS FNR protein puriﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>We thank P. Kiley ( University of Wisconsin , Madison , USA ) for providing FNR * - DA154 protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Contributions of [ 4Fe-4S ] - FNR and integration host factor to fnr transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Key words FNR · Anaerobic gene regulation · Oxygen · Facultative anaerobic metabolism · Anaerobic-respiration Biological role of facultative anaerobic metabolism Facultatively anaerobic metabolism is wide-spread among bacteria and archaea , including purple phototrophic ( pro-teo - ) and gram-positive bacteria and halophilic or thermophilic archaea ( see review by Unden et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Disulphide bonding might also play a role in regulating the function of the FNR protein , since an / V-terminal segment containing three cysteine residues is essential for activating the target frd operon encoding fumarate reductase , an enzyme for anaerobic-respiration ( Spiro and Guest , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li>
<li>The isolation of mutants with pleiotropic defects in anaerobic-respiration led to the identification of the fnr gene , also known as nirA or nirR ( Lambden and Guest , 1976 ; Newman and Cole , 1978 ; Chippaux et al. , 1978 ) encoding a 28000 Dalton polypeptide , FNR ( Shaw and Guest , 1982a , b ) , which is homologous to the catabolite gene activator protein , CAP ( Shaw et al. , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The regulators of anaerobic-respiration ( FNR , fumarate-nitrate-regulator ) and of nitrogen-fixation in Klebsiella ( NifA/NifL ) are known ( Lambden & Guest 1976 ; Newman & Cole 1978 ; Chippaux et al. 1978 ; Filser et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Those from Shewanella putrefaciens , Pseudomonas aeruginosa , Pseudomonas stutzeri and Rhodopseudomonas palustris are functionally similar to FNR in that they regulate anaerobic-respiration or carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Finally , with the emergence of aerobic metabolism , all the operators involved in anaerobic-respiration would eventually have to have been endowed with a site for the FNR regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3035558" target="_blank">3035558</a>]<br></li>
<li>It is tempting to speculate that other members of the purple nonsulphur bacteria may have FNR homologues , which are involved not in the regulation-of-photosynthesis , but in other aspects of anaerobic metabolism such as nitrate respiration and degradation of aromatic compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>It is suggested FixK in R. meliloti , and the azurin genes that the cyd operon is activated by ( azu ) of Alcaligenes faecalis and A. deni - ArcA under microaerobic conditions trificans23 , which encode the blue-cop - and repressed by FNR under anaerobic per proteins thought to be involved in conditions , so that this dual control nitrite respiration and therefore likely allows cytochrome d oxidase synthesis to be induced anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>When the terminal electron acceptor , O2 , is no longer available , FNR represses the synthesis of enzymes involved in aerobic-respiration and increases the synth-esis of enzymes required for anaerobic-respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>FNR also represses synthesis of enzymes required for aerobic-respiration , such as NADH dehydrogenase II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Most genes involved in an-aerobic-respiration fall into group I , which are upregulated by FNR anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>FNR and ArcA are involved in sensing the redox status of the cell to regulate genes involved in respiration under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Under the same conditions FNR represses some of the genes of aerobic-respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The anaerobic repression of ndhH and the genes of th terminal oxidases cyo ( and cyd ) indicates that FNR also functions for some genes as a repressor of aerobic-respiration during anoxia .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The transcription factor FNR is known to activate genes involved with anaerobic metabolism while ArcB governs the expression of aerobic-respiration-related genes ( Salmon et al. , 2003 ; Shalel-Levanon et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23563322" target="_blank">23563322</a>]<br></li>
<li>It has recently been shown that , under anaerobic growth conditions , Fpg activity increases in strains deﬁcient in the global regulators Fur , FNR , and ArcA ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>'' Formate Dehydrcgenase '' '' MATERIALS AND METHODS FNR ( anaerobic ) Nomenclature : Previously identified genes required for formate dehydrogenaseactivity have been designated `` jdh .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>For long-term growth under oxic conditions , on the other hand , a stable repression of genes encoding enzymes of anaerobic metabolism is appropriate ; the presence of apoFNR instead of [ 2Fe-2S ] FNR might suppress frequent shifts to anaerobic metabolism during short-term lack of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>FNR is a CRP homologue and regulates genes during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Collectively , our results provide insight into the mechanism by which FNR negatively regulates genes such as sodA , sodB , cydDC , and metE , thereby demonstrating that adaptation to anaerobic growth involves the action of a small regulatory RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Furthermore , the anaerobic transcription of fnrS was abolished in a fnr strain , thus demonstrating that the global regulatory protein FNR induces FnrS transcription in response to anoxic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Under aerobic growth conditions , proteolysis decreases FNR protein levels ( 12 , 38 ) , while under anaerobic conditions , FNR represses its own transcription ( 24 , 41 , 44 , 47 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>This induction is dependent on the regulatory protein , FNR , a factor essential for the transcription of a number of genes expressed during anaerobic growth ( Cole and Ward , 1973 ; Lambden and Guest , 1976 ; Griffiths and Cole , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>FNR is selectively active as a transcription factor under anaerobic growth conditions , where it has been shown to control the transcription of hundreds of genes , many of which are necessary for adaptation to O2-limiting growth conditions ( 10 , 16 , 25 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>FNR activates transcription of many genes in response to anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>FNR acts as a dual transcription regulator that activates certain genes required for anaerobic growth and represses many genes required for aerobic growth [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>FNR activates transcription of a number of genes coding for enzymes required under anaerobic growth conditions and represses transcription of some genes for aerobic functions ( 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Other factors , such as ArcAB ( regulator of anaerobic and aerobic metabolism inhibiting the Krebs-cycle , the electron-transport-chain and PDH activity in anaerobic conditions ) , activate the PFL enzyme ( De Graef et al. , 1999 ) , which is also regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Thus , moe operon expression is modulated at three levels : ( 1 ) an aerobic low level , ( 2 ) an anaerobic intermediate level controlled by ArcA-P and FNR ( supporting FHL and DMSO reductase ) , and ( 3 ) by ArcA-phosphate and NarL-phosphate , which , during nitrate respiration , control the highest level of moe operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>In fact , the rplM operon is under the control of FNR , which is involved in the metabolic switch from aerobic to anaerobic-respiration ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>In contrast , pﬂ is under partial anaerobic control of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>Genes and regulators responding to O2 and other acceptors FNR and ArcA , as shown earlier for NarL and NarP , control different sets of target genes Žreviewed in w4 ,5,9,11,12,44 x. FNR acts as the major regulator of anaerobic metabolism and ArcA as the regulator of aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The reduced uptake rates in an fnr background and under aerobic conditions suggest that the synthesis is under ( direct or indirect ) transcriptional-control of FNR , the transcriptional regulator of anaerobic respiratoxy genes ( reviewed in references 25 and 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></li>
<li>The transcriptional-control of anaerobic-respiration by the transcriptional regulator FNR takes place in a completely different mode .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Therefore it appears that FNR is the specific regulator of anaerobic-respiration and related functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>FNR is the regulator of genes required for anaerobic-respiration and related pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Remarkably , anaerobic control is mediated neither by FNR , the general anaerobic regulator , nor by ArcA , another anaerobic global regulator ( luchi and Lin , 1988 ; Pascal et aL , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>TorC autoregulation is different from tor anaerobic control The torCAD operon is fully induced in anaerobiosis but , in contrast to all other known anaerobic respiratory systems , this control is mediated neither by FNR , a pleiotropic anae-robic regulator , nor by ArcA , a response regulator involved in redox control of numerous genes ( Simon et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411745" target="_blank">10411745</a>]<br></li>
<li>The genes for fumarate respiration and related enzymes ( dcuB , frdABCD , fumB , ttdAB , and ttdT ) are subject to anaerobic induction by the oxygen sensor FNR and to nitrate repression by the NarX-NarL two-component system ( 15 , 16 , 37 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>However , Key words : fumarate and nitrate reductase regulon ( FNR regulon ) , MG1655 , microarray , NarXL regulon , nitrate , regulation of anaerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Crack JC , Le Brun NE , Thomson AJ , Green J , Jervis AJ ( 2008 ) Reactions of nitric oxide and oxygen with FNR ( regulator of fumarate and nitrate reduction ) , a global transcriptional regulator , during anaerobic growth of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Adaptation to anaerobic growth is regulated by two global transcription factors , FNR ( fumarate and nitrate reductase regulator ) and ArcA ( anaerobic respiratory control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Abbreviations used : ArcA , anaerobic respiratory control ; FNR , fumarate and nitrate reductase regulator ; HMP , ﬂavohaemoglobin ; TMAO , trimethylamine N-oxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Evidence for the existence of two other proteins which may be involved in enhancing anaerobic ndh expression in rich media in the absence of FNR was also obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>It was therefore proposed that one or both of these novel poly-peptides might be involved in stimulating ndh promoter activity in bacteria grown in rich medium , and might also normally compete with FNR under anaerobic conditions in order to fine-tune ndh expression to match prevailing requirements for cellular energy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Arr responds to anaerobic amino-acid-metabolism and may compete with FNR for overlapping binding sites in order to fine tune ndh expression to cellular energy requirements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>The effect of anaerobiosis thought be to is mediated positive-through FNR ( the fnr gene product ) , a acting transcription factor required for the synthesis of several anaerobic Stewart , in respiratory enzymes ( 26 , 43 ; press ) including nitrate reductase ( 14 , 50 ) and fumarate reductase physiological signal for FNR-medi - ( 18 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2832370" target="_blank">2832370</a>]<br></li>
<li>The target genes of FNR encode redox or electron-transfer enzymes , transmembrane carriers for anaerobic catabolites , enzymes of central anaerobic pathways or fermentation , enzymes of anaerobic biosynthetic pathways , gene regulators , and toxins or detoxifying agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Anaerobic regulation is mediated by the fnr gene product , FNR , which is a positive activator required for the transcription of a number of anaerobic genes ( reviewed in references 31 and 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>A mutation in the fnr gene , however , only reduces anaerobic induction fivefold , indicating that FNR is not the only factor involved in the anaerobic activation process ( Sawers and Bock , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>The FNRsystem The product of the fnr gene ( FNR ) is now known to be involved in the anaerobic activation or repression of a variety of enzymes , and it may also repress its own transcription under anaerobic conditions ( Fig. 3 ) 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The importance of the FNR protein as a transcriptional regulator has been further extended to include possible rotes in both aerobic and anaerobic gene activation and repression , as a result of a detailed two-dirnensionai gel electrophoretic analysis of the proteins of aerobtcally and anaerobicady grown fnr ' and fnr strains ( Sawers ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>The FNR protein plays a major role in the anaerobic control of this system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>enome-Wide Expression Analysis Indicates that FNR G of Escherichia coli K-12 Regulates a Large Number of Genes of Unknown Function † Yisheng Kang ,1 K. Derek Weber ,2 Yu Qiu ,1 Patricia J. Kiley ,2 and Frederick R. Blattner1 * Department of Genetics1 and Department of Biomolecular Chemistry ,2 University of Wisconsin , Madison , Wisconsin Received 10 June 2004/Accepted 11 October 2004 The major regulator controlling the physiological switch between aerobic and anaerobic growth conditions in Escherichia coli is the DNA binding protein FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Accepted 3 June 2010 INTRODUCTION The Escherichia coli FNR ( regulator of fumarate and nitrate reduction ) protein is an oxygen-responsive global transcription factor ( Constantinidou et al. , 2006 ; Green et al. , 2009 ; Green & Paget , 2004 ; Kang et al. , 2005 ; Partridge et al. , 2006 , 2007b ; Salmon et al. , 2003 ; Unden et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The E.coli FNR protein ( regulator of fumarate and nitrate reduction ) is the global transcription factor that manages the distribution of RNA polymerase in response to oxygen starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>The FNR transcription factor of E. col ~ an oxygen sensor In response to lowered oxygen tension , the metabolism of E. coli switches from use of oxygen to use of alternative terminal electron acceptors such as fumarate and nitrate , a change that requires the simultaneous transcriptional activation of over 50 genes of anaerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8871401" target="_blank">8871401</a>]<br></li>
<li>Hence , absence of oxygen triggers the activity of FNR ( fumarate and nitrate reduction regulator ) , a well-characterized transcription factor that is the master regulator of E. coli adaptation to growth in anaerobic conditions [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>The transcription factor FNR , the regulator of fumarate and nitrate reduction , regulates major changes as Escherichia coli adapts from aerobic to anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The global transcription factor FNR controls the adaptation of Escherichia coli to growth under anaerobic conditions ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>Genes of Global Regulators of Aerobic/Anaerobic Metabolism Genes coding for global regulators of aerobic ( ArcAB ) and anaerobic ( FNR ) metabolism in E. coli were also tested and their transcription levels are shown in Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>FNR is known to induce the transcription of the E. coli nrfABCDEFG operon during anaerobic cell-growth [ 22 ] , and to negatively regulate the transcription of E. coli hmp [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>FNR , a transcription factor known to positively regulate a flavohemoglobin homolog in the bacterium Vitreoscilla as well as many denitrification enzymes in other species , actually represses anaerobic expression of the E. coli hmp ( 14 , 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>A primary role for FNR is to coordinate carbon and energy metabolism during growth under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In E. coli , the NO-induction of the hmpA gene was reported to be dependent on MetR , a transcriptional regulator of the methionine biosynthetic pathway , under aerobic conditions ( 23 ) and on FNR , an oxygen-responsive regulator , under anaerobic conditions ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Induction was dependent on the transcriptional regulator ArcA , with a second transcriptional regulator , FNR , playing a negative role speciﬁcally under anaerobic conditions ( S. Darie and R. P. Gunsalus , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550494" target="_blank">8550494</a>]<br></li>
<li>Second , characterization of regulatory mutations that caused moderate anaerobic derepression of sodA : : lacZ protein fusions identified FNR as an additional negative regulator of sodA expression that increases the downregulation further ( 6-fold ) ( 56 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>These results suggest that the FNR protein acts as a negative regulator of moe operon in controlling the basal level of expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>We also report the identification of a subset of these genes that are regulated by a global regulatory protein for anaerobic metabolism , FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In contrast , the expression of fumB resembles that of the fumarate reductase operon and some other ` anaerobic ' genes in being depressed anaerobically by a mechanism that involves FNR , the CRP-like anaerobic transcriptional activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3282546" target="_blank">3282546</a>]<br></li>
<li>On the other hand , converting this putative FNR protein binding site into the FNR protein binding site core consensus resulted in elevated anaerobic induction of thefdn operon and also caused weak aerobic expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Under anaerobic conditions FNR is a dimeric iron-sulfur protein that binds at specific DNA sequences to activate the expression of `` anaerobic genes '' and to repress the expression of `` aerobic genes '' ( 7 , 8 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>To incorporate a role for IscR , we would suggest that for the napF promoter , anaerobic activation of the downstream promoter ( s ) via FNR , a bona ﬁde anaerobic global regulator , and NarP , a nitrate-dependent activator ( Darwin et al. , 1998 ; Stewart et al. , 2003 ) , masks detection of IscR-dependent changes in expression from the upstream promoter ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>This suggests that during anaerobic growth in liquid cultures in the absence of glucose , FNR is the major regulator of hlyE expression , and that CRP-mediated hlyE expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The fnr mutant , lacking the O2-responsive regulator FNR , was completely devoid of dcuC9-9lacZ expression during aerobic and anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>We supposed that FNR was responsible for the effect in the anaerobic condition , because FNR was the global transcriptional regulator that controlled gene-expression in response to the transition between aerobic and anaerobic growth ( Green et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>Results FNR is an NO sensor The starting point for this work was the observation that the oxygen-responsive global transcription factor FNR represses expression of the NO-detoxifying protein Hmp during anaerobic growth ( Poole et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>NarL controls the expression of many proteins involved in anaerobic-respiration and , at most of its known promoter targets , it co-regulates transcription together with another global transcription factor , protein FNR , the ` master ' regulator of anaerobic adaptation [ 2,8 -- 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Finally , FNR has been shown to have an indirect effect on anaerobic induction of tdc expression , possibly by controlling levels of an important metabolite ( Chattopadhyay et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The likely explanation for this is that most of the genes controlled by these promoters encode products that play a role in anaerobic-respiration of nitrate and nitrite , and thus their expression is fully induced only in both the absence of oxygen ( signalled via FNR ) and the presence of nitrate or nitrite ( signalled by NarL ) [ 4,29,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Our studies have focused on understanding the role of FNR in activating expression of the genes encoding the anaerobic terminal reductases for nitrate , narGHJI ( Stewart and Rabin , 1995 ) , and dimethyl sulphoxide , dmsABC ( Cotter and Gunsalus , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>A primary role for FNR is to coordinate carbon and energy metabolism under anaerobic growth conditions , and results from global gene-expression profiling revealed that a very large number of genes ( 200 operons ) directly or indirectly are controlled by FNR ( 32 -- 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>To identify the global changes and adjustments of gene-expression patterns that facilitate a transition from aerobic to anaerobic growth conditions , and to determine the effects of genotype on these gene-expression patterns , we analyzed E. coli gene-expression profiles obtained from cells cultured under aerobic or anaerobic growth conditions ( O2 or O2 ) and under anaerobic growth conditions in the presence or absence of the global regulatory protein for anaerobic metabolism , FNR ( O2 , FNR or O2 , FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>To identify the genes controlled by FNR , we analyzed gene-expression profiles of cells cultured under anaerobic growth conditions in the presence or absence of FNR ( O , 2 FNR or O , FNR ) in otherwise isogenic strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>To identify genes controlled by FNR , we used Affymetrix Antisense GeneChips to compare global gene-expression proﬁles from isogenic MG1655 wild-type and fnr strains grown in glucose minimal media under aerobic or anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The lpd ( lpdA ) gene , encoding lipoamide dehydrogenase , a member of the phdR-aceEF-lpdA operon ( Fig. 6 ) , shows a 4.8-fold decrease in expression when grown under anaerobic conditions and a 11.4-fold increase in the FNR deletion strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>FnrS is involved in gene regulation after the shift from aerobic to anaerobic conditions , and its expression is activated by the transcription factors FNR and ArcA ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>The transcription factor FNR ( for fumarate and nitrate reduction ) plays a major role in altering gene-expression between aerobic and anaerobic conditions to facilitate such changes in energy metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Thefnr gene product , FNR , activates the anaerobic expression of nitrate reductase ( 23 ) , dimethyl sulfoxide reductase ( 6 ) , and fumarate reductase ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>During anaerobic growth the fumarate and nitrate reduction regulator ( FNR ) was important for ClyA expression , and the clyA gene could be activated by overexpression of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Surnrnary In enteric bacteria , the expression of rnany genes encoding various anaerobic electron-transfer functions is controlled by FNR , the product of the auto-regulated fnrgene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Mean gene-expression data ( fold change ) , acquired from the WT and ECOM4LA strains grown in triplicate , for all genes that are known to be regulated by the anaerobic regulator FNR or ArcA according to RegulonDB were compared to the reported function of the regulator ( activator or repressor ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Sawers , G , , and Bock , A. ( 1989 ) Novel transcriptional-control of the pyruvate formate-lyase gene : upstream regulatoty sequences and multiple promoters regulate anaerobic expression , J Bacterioi 171 : 2485-2498 , Sawers , G , , and Suppmann , B. ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene-expression is mediated by lhe ArcA and FNR proteins , J Bacterio !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>Two transcriptional regulators , FNR and ArcA , control this anaerobic induction of gene-expression ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The expression of ndb is mainly regulated by FNR ( anaerobic repression ) and by unknown , pulative amino acid response regulators ( Green and Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>This suggests that FNR represses specific roles of the regulators in trancyd expression under anaerobic con - scriptional activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>FNR is an activator of anaerobic respiratory gene-expression and also represses ndh gene-expression ( Green and Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10486127" target="_blank">10486127</a>]<br></li>
<li>FNR is essential for anaerobic induction of narK expression ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>It is shown that FNR acts an anaerobic activator of hyp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Harborne , N , , Wootton , J , , Nicolson , R. , and Cole , J. ( 1990a ) Nucleotide sequence , organisation and structural analysis of the products of genes in the nirB-cysG region of the Escherichia coti K-12 chromosome , Eur J Biochem 191 : 315-323 , Peakman , T , , Busby , S , , and Cole , J , ( 1990b ) Transcriptional control of the cysG gene of Escherichia coli K-12 during aerobic and anaerobic growth , Euz-JS/ochem 191 : 325-331 , Spiro , S , , and Guest , J , ( 1988 ) Inactivation of the FNR protein of Escherichia coii by targeted mutagenesis in the W-termi-nal region , Molf ^ icrobiol2 : 701-707 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1435259" target="_blank">1435259</a>]<br></li>
<li>Physiological role of FNR : target genes FNR , originally identified as the transcriptional regulator of anaerobic-respiration in E. coli ( Lambden and Guest 1976 ; Newman and Cole 1978 ; Chippaux et al. 1978 ) , has now been recognized as a general regulator of anaerobic metabolism that affects many genes of diverse functions ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>One mechanism involves the transcriptional activator and repressor FNR , the major role of which is to regulate the expression of genes involved in anaerobic-respiration and pyruvate-metabolism ( Spiro & Guest 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>In E. coli , the expressions of the genes encoding the enzymes catalyzing the fermentation and respiration are under the control of some global regulators such as FNR and ArcA/B system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_198">
E. coli has two different regulators that control expression of genes involved in the aerobic-anaerobic shift , the ArcB/ArcA two-component system and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></label>
<input id="rp_similarity_198", type="checkbox">
<div class="RP_similar">
<li>E. coli has two distinct regulators that control expression of the many genes involved in the aerobic-anaerobic shift : the ArcB/ArcA two-component system and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li></div>
<li>Repression of in vitro transcription of the Escherichia coli fnr and narX genes by R protein The expression of respiratory genes in E. coli is controlled by two trandscriptional regulatory systems : one is the ArcB/A system for aerobic genes [ l ] ; and the other is the FNR system for anaerobic genes [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Introduction The expression of genes required for anaerobic growth of Escherichia coli is tightly controlled , and many such genes are reguiated by the FNR protein , a transcription factor that is activated by the absence of oxygen in the medium ( reviewed by Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Introduction In Escherichia coli , the expression of many genes required for anaerobic-respiration is controlled by the FNR protein , an anaerobically triggered transcription activator ( reviewed by Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Isolation of an oxygen-sensitive FNR protein of Escherichia coli : Interaction at activator and repressor sites of FNR-controlled genes ABSTRACT The Escherichia colifnr gene product , FNR , is a DNA binding protein that regulates a large family of genes involved in cellular-respiration and carbon metabolism during conditions of anaerobic cell-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>[ 6 ] S.B. Melville , R.P. Gunsalus , Isolation of an oxygen-sensitive FNR protein of Escherichia coli : interaction at activator and repressor sites of FNR-controlled genes , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>In Azotobacter vinelandii , expression of the ﬂhDC genes ( which encode the master regulator of motility ) is negatively regulated by the oxygen-sensor CydR , an orthologue of the E. coli FNR protein ( León and Espín , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>Summary FNR is a transcriptional regulator controlling the expression of a number of Escherichia coli genes in response to anoxia .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Introduction FNR is a transcriptional regulator which controls the expression of target genes in response to anoxia in Escherichia coti ( Spiro and Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Introduction Escherichia coli FNR protein is a global transcription regulator that activates the expression of many genes in Accepted 24 February , 2004 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Introduction The FNR ( fumarate and nitrate reduction regulator ) protein of Escherichia coli is a global transcriptional regulator controlling the expression of more than 70 genes in response to anaerobiosis ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Department of Microbiology and Molecular Genetics , and the Molecular Biology Institute , 1602 Molecular Sciences Building , University of California , Los Angeles , CA 90095 The Escherichia colifnr gene product , FNR , is a global transcriptional regulator that under anaerobic conditions , activates the expression of a number of anaerobic electron transport-related genes , including nitrate reductase ( narGHJI ) , fumarate reductase ( frdABCD ) , trimethylamine-N-oxide/dimethyl sulfoxide reductase ( dmsABC ) , nitrite reductase ( nirB ) , nitrite export ( narK ) , and formate dehydrogenase ( fdnGHI ) ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>In Escherichia coli , the shift between aerobic and anaerobic metabolism , which is controlled primarily by the global transcriptional regulator FNR ( 13 , 23 , 32 ) , is known to involve the repression and activation of hundreds of genes ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>FNR has been shown to be a global regulator of anaerobic metabolism in E. coli , controlling the synthesis of up to 125 genes [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>In an attempt to further our understanding of the metabolic potential of E. coli under anaerobic growth conditions , we sought to identify additional genes that are controlled by the global regulator FNR through genome-wide expression studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Summary In Escherichia coli , the anaerobic expression of genes encoding the nitrate ( narGHJI ) and dimethyl sulphoxide ( dmsABC ) terminal reductases is stimulated by the global anaerobic regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>In the Gram negative bacterium Escherichia coli , this process is primarily controlled by the global transcription regulator FNR ( fumarate and nitrate reduction ) which is activated under anaerobic conditions and upon activation , regulates the expression of more than one hundred genes in E. coli genome [ 1,2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>However the data do provide an interesting insight inasmuch as FNR is an oxygen responsive global transcriptional regulator that controls expression of a large number of genes involved in anaerobic functions ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169565" target="_blank">21169565</a>]<br></li>
<li>The transcriptional regulator FNR of E. coli controls the expression of many genes required for anaerobic metabo-lism in response to O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Re-evaluation of expression studies using ttdA9 -- 9lacZ reporter gene fusions shows that FNR is the main regulator in response to O , similar to the regulation of other genes of 2 anaerobic-respiration and fermentation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>It would further appear that the presence of iron is not essentiai for site-specific DNA bindin Introduction FNR is a transcriptional regulator that controls the expression of target genes in response to anaerobiosis in such a Received 30 September , 1993 : revised and accepted 1 February , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Anaerobically , the global regulator FNR ( Kiley and Beinert , 1999 ; Green et al. , 2001 ) is also involved in the regulation of hmp since an fnr mutation enhances hmp-lacZ expression ( Poole et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Salmon et al. ( 1 ) compared RNA isolated from cultures of strain MC4100 that had been grown aerobically or anaerobically and also from an anaerobic culture of an fnr mutant that lacks FNR ,2 the regulator of fumarate and nitrate reduction , which is a global regulator of many oxygen-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Abbreviations : dO2 , dissolved oxygen tension ; FNR , fumarate and nitrate reduction regulator ; GST , glutathione S-transferase ; PFL , pyruvate transcription factors is FNR ( fumarate and nitrate reduction regulator ) , a direct oxygen sensor that functions mainly to activate the expression of genes required during anaerobic growth ( Kiley & Beinert , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Transcription of this operon is driven from a single promoter ( pnrf ) , and expression is induced in the absence of oxygen by the FNR protein ( regulator of fumarate and nitrate reduction ) , a global transcription activator that controls the expression of many genes required for anaerobic-respiration ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Transcription of this operon is driven from a single promoter ( pnir ) and is induced by the fumarate and nitrate reduction ( FNR ) protein , a global transcription regulator that activates the expression of many genes in response to oxygen starvation ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>This example underlines the difficulty of identifying genes regulated by a global regulator such as FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR is a global regulator for the response of many genes to oxygen level [ 22,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>For instance , the global regulator FNR ( fumarate and nitrate reductase ) , whose expression is partially induced by RpoS , represses genes of the tricarboxylic-acid-cycle ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>As an example , the global regulator FNR , whose expression is partially under the control of RpoS , represses catabolic genes involved in carbohydrate-metabolism ( Dong et al. , 2008 ; Patten et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23867204" target="_blank">23867204</a>]<br></li>
<li>The latter form of regulation involves thefnr gene product ( FNR protein ) ( 20 ) , which also activates a number of other anaerobically regulated genes ( 18 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>One of these is the FNR protein , which activates transcription of many anaerobically regulated genes ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Of the remaining genes clustered into this expression pattern , two genes , mrcA ( penicillin-binding protein 1A ) and rarD ( involved in chloramphenicol resistance ) , were also observed to be regulated by FNR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>For example , many genes involved in the biosynthesis of the O-antigen and capsular polysaccharides are epistatically regulated , as are several genes also under FNR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>B , Venn diagram for ArcA - , FNR - , and Lrp-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>The effect of the fnr mutation may well be indirect in this case , since FNR probably controls the expression of genes that are involved in the reduction of nitrite to NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>There is evidence for a similar interaction between the a subunit of RNA polymerase and FNR , a transcriptional activator that controls the expression of certain anaerobically induced genes in S. typhimurium ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836282" target="_blank">7836282</a>]<br></li>
<li>Six of the eight groups showed regulation by FNR ; while some expression groups represent genes that are simply activated or repressed by FNR , others , such as those encoding functions for chemotaxis and motility , showed a more complex pattern of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>NifA , which activates expression of the N2-fixation genes of B. japonicum , apparently acts similarly to FNR in O2-dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>In any case , insertion of an IS5 element is able to increase FNR-dependent expression or to place genes under FNR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>This article reviews the properties of two transcriptional regulators , ArcA and FNR , which control the expression of networks of genes in response to oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Thus , these data indicate an interesting but complex relationship between FNR and ArcA in controlling the expression of genes in group III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>FNR also activates the transcription of the small non-coding RNA FnrS which negatively regulates the expression of multiple genes , including many that encode enzymes with functions linked to oxi-dative stress [ 26,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>FNR can also repress the expression of genes associated with the aerobic electron transport pathways including cytochrome o oxidase ( cyoABCDE ) , cytochrome d oxidase ( cydAB ) , glutamyl-tRNA dehydrogenase for 6-aminolevulinate biosynthesis ( hemA ) , and soluble NADH dehydrogenase ( ndh ) ( 1-4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Furthermore , FNR represses the expression of genes that are needed for aerobic metabolism ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>FNR also represses the expression of genes that are needed for aerobic metabolism , such as the respiratory enzymes NADH dehydrogenase II and cytochrome oxidases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Thus it would appear that several genes involved in nucleoside-metabolism are located in the nfo -- fruA region of the E. coli chromosome together with a novel paralogue of CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>An E. coli FNR-regulated promoter remains anaerobically inducible in the fnrA mutant of P. stutzeri , which raises the possibility that a second FNR homologue is involved in the regulation of denitrification genes ( Cuypers & Zumft 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Some FNR-dependent promoters from E. coli are active and anaerobically inducible in P. aeruginosa , and FNR boxes are associated with the P. aeruginosa genes encoding the arginine-deiminase-pathway enzymes ( arcDABC ) and azurin ( azu ) , aredox protein associated with denitrification ( Fig. 2 ; Lodge et al. 1990 ; Galimand et al. 1991 ; Hoitink et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>There are other examples of FNR boxes being associated with genes involved in denitrification , for example the gene encoding the copper-containing nitrite reductase ofAlcaligenesfaecalis ( Nishiyama et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>As part of ongoing efforts to understand the various redox pathways that will support AdoMet-dependent radical enzymes in E. coli , we have examined the relative speciﬁcity of E. coli FNR for Fd , Fld1 , and Fld2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Using steady state and transient kinetics , we have investigated the rates and speciﬁcity of electron-transfer from E. coli FNR to Fd , Fld1 , and Fld2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Transient kinetic analysis of electron-transfer from reduced FNR to oxidized Fd , Fld1 , or Fld2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>Identiﬁcation and Characterization of a Two-Component Sensor-Kinase and Response-Regulator System ( DcuS-DcuR ) Controlling Gene Expression in Response to C4-Dicarboxylates in Escherichia coli PAUL GOLBY ,1 SUZANNE DAVIES ,2 DAVID J. KELLY ,2 JOHN R. GUEST ,2 AND SIMON C. ANDREWS1 * The School of Animal and Microbial Sciences , University of Reading , Whiteknights , Reading RG6 6AJ ,1 and The Krebs Institute for Biomolecular Research , Department of Molecular Biology and Biotechnology , University of Shefﬁeld , Western Bank , Shefﬁeld S10 2TN ,2 United Kingdom Received 30 September 1998/Accepted 8 December 1998 The dcuB gene of Escherichia coli encodes an anaerobic C4-dicarboxylate-transporter that is induced anaerobically by FNR , activated by the cyclic AMP receptor protein , and repressed in the presence of nitrate by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
<li>The dcuB gene is strongly activated anaerobically by FNR , repressed in the presence of nitrate by NarL , and subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>R E S E A R C H A R T I C L E Open Access Resistance to topoisomerase cleavage complex induced lethality in Escherichia coli via titration of transcription regulators PurR and FNR I-Fen Liu , Sandra Aedo and Yuk-Ching Tse-Dinh * Abstract Background : Accumulation of gyrase cleavage complex in Escherichia coli from the action of quinolone antibiotics induces an oxidative damage cell-death pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Regulation of the Puu pathway by PuuR seems to be a simple , primary regulatory system that maintains the putrescine concentration in the cell , although there is a possibility of other regulatory mechanisms such as an aeration shift by FNR or ArcA for expression of the Puu pathway ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>The FNR protein primarily coordinates gene regulation in remodelling central metabolism in response to changes in oxygen availability in many bacteria ( Guest et al. , 1996 ; Korner et al. , 2003 ) , but it also exerts effects over other systems , and new members of the FNR regulon are still being uncovered ( Covert et al. , 2004 ; Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The FNR homologue from P. aeruginosa ( ANR ) regulates arginine fermentation , nitrate respiration and cyanide-synthesis , and anr ( formerly nirD ) mutants are defective in these processes ° The ANR protein is oxygen sensitive , is structurally very similar to FNR , and the cloned anr gene complements an E. coli fnr mutant ( Zimmermann et al. 1991 ; Sawers 1991 ; Galimand et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Microbiology ( 2007 ) , 153 , 1499 -- 1507 DOI 10.1099 / mic .0.2006 / 004838-0 The Escherichia coli yhjA gene , encoding a predicted cytochrome c peroxidase , is regulated FNR and OxyR by Jonathan D. Partridge , Robert K. Poole and Jeffrey Green Department of Molecular Biology and Biotechnology , The University of Shefﬁeld , Shefﬁeld S10 2TN , UK Correspondence Jeffrey Green jeff.green@shefﬁeld.ac.uk The Escherichia coli FNR protein is an oxygen-responsive global transcription factor , and OxyR is a key regulator of the peroxide stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Furthermore , it was recently shown that altered FNR proteins , similarly to HlyX , could activate the expression of clyA ( 43 ) ; i.e. , minor alterations in a gene encoding a global regulator have a profound effect on the production of cytotoxic factors like ClyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Fis FNR act either individually or in concert to regulate ndh expression in response to growth phase and anaerobiosis , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Thus , CRP enhances hlyE expression in response to glucose starvation ( 36 ) and FNR enhances hlyE expression in response to oxygen starvation ( 10 , 11 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>RESULTS Effects of the global transcription factors FNR , CRP , and H-NS on hlyE expression in liquid culture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , CRP and FNR positively control cai expression and H-NS exerts a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Expression of the caiF gene is dependent on the same general regulatory proteins , H-NS , CRP , FNR , and transcription factor RpoN ( 11 ) , which control cai and ﬁx expression ( Table 2 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Mol Microbiol 3:593 -- 599 Trageser M , Spiro S , Duchêne A , Kojro E , Fahrenholtz F , Guest JR , Unden G ( 1990 ) Isolation of intact FNR protein ( Mr 30 000 ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>SoxRS ( Escherichia coli ) OxyR ( Escherichia coli ) Oxidative stress ( O2̇ ¯ ) Oxidative stress ( H2O2 ) Redox reaction Fe-S -- Redox reaction at cysteine residu Table 2 FNR regulated genes , regulatory signals , and additional regulators acting at the relevant promoters in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Shalel-Levanon S , San KY , Bennett GN : Effect of oxygen on the Escherichia coli ArcA and FNR regulation systems and metabolic responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
<li>Effect of oxygen , and ArcA and FNR regulators on the expression of genes related to the electron-transfer chain and the TCA-cycle in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752906" target="_blank">22752906</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513468" target="_blank">17513468</a>]<br></li>
<li>J Bacteriol 189 : 1150 -- 1153 Shalel-Levanon S , San KY , Bennett GN ( 2005 ) Effect of ArcA and FNR on the expression of genes related to the oxygen regulation and the glycolysis pathway in Escherichia coli under microaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20683441" target="_blank">20683441</a>]<br></li>
<li>Effect of ArcA and FNR on the expression of genes related to the oxygen regulation and the glycolysis pathway in Escherichia coli under microaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752906" target="_blank">22752906</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23563322" target="_blank">23563322</a>]<br></li>
<li>BELL , A. I. , K. L. GASTON , J. A. COLE and S. J. W. BUSBY , 1989Cloning of binding sequences forthe Escherichia coli transcription activators , FNR and CRP : location of bases involved in discrimination between FNR and CRPN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>J Bacteriol 171:3817 -- 3823 Darwin AJ , Stewart V ( 1998 ) FNR , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>In the facultative anaerobe , Escherichia coli , the transcription factor FNR ( fumarate nitrate reduction ) regulates a network of genes that facilitates adaptation to oxygen deprivation by providing alternative pathways for energy generation ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9177174" target="_blank">9177174</a>]<br></li>
<li>Characterization of FNR * mutant proteins indicates two distinct mechanisms for altering oxygen regulation of Escherichia coli transcription factor FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>]<br></li>
<li>The nosR gene which encodes a protein required for the expression of an active nitrous oxide reductase also contains an FNR box in its promoter region ( Fig. 2 ; Cuypers et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The novel nature of fdhF gene-expression has further been emphasized by the fact that anaerobic induction is independent of the FNR protein ( Birkmann ef al. , 1987b ) and transcription from the promoter is dependent upon the presence of a functional NtrA protein ( synonyms GInF , RpoN , ( T̂ ' * ) -LCB- Birkmann etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181234" target="_blank">2181234</a>]<br></li>
<li>Furthermore , the spatial separ-ation of oxygen consumption and FNR sensing of oxygen could provide an explanation for the presence of membraneassociated ( ArcBA ) and cytoplasmic ( FNR ) TFs to coordinate global gene-expression in response to oxygen availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The data also suggest that the Arc response pathway , which translates into a net global downscaling of gene-expression , overlaps partly with the FNR regulatory network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14711822" target="_blank">14711822</a>]<br></li>
<li>The Arc-activated cydAB ( 1 ) , and Arc-repressed sodA ( 13 ) and lpdA ( 26 ) , are also under the direct negative control of FNR , a regulator that co-mediates gene-expression in response to the redox conditions of growth ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14711822" target="_blank">14711822</a>]<br></li>
<li>hmp gene-expression in the presence of NOz is independent of the SoxS , OxyR , and FNR transcription factors and instead relies on inactivation of the iron-de-pendent Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>Thus , locating sensors in the membrane ( ArcB ) and the cytoplasm ( FNR ) allows optimal coordination of gene-expression in the ` micro-aerobic ' range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_282">
The FNR modulon and FNR regulates gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></label>
<input id="rp_similarity_282", type="checkbox">
<div class="RP_similar">
<li>The FNR modulon and FNR-regulated gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li></div>
<li>Identiﬁcation of a surface of FNR overlapping AR1 that is required for repression of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>We also chose a gene , yfcZ , on whose expression FNR showed a small effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>In summary , we have shown that direct modi ® cation of the FNR [ 4Fe ± 4S ] 2 + clusters to form protein-bound dinitrosyl-iron ± thiol complexes serve as a functional switch in NO-responsive control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>In this study we found that the expression of a glpAl ( Oxr ) ( oxygen-resistant ) mutant operon , selected for increased aerobic expression , became less dependent on the FNR protein but more dependent on the cyclic AMP-catabolite gene activator protein complex mediating catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2403539" target="_blank">2403539</a>]<br></li>
<li>2 ( 4 ) : 527-530 Jennings MP & Beacham IR ( 1990 ) Analysis of the E. coli gene encoding L-asparaginase II , ansB , and its regulation by cyclic AMP receptor and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>This unusual use of FNR and OxyR to activate expression suggests that yhjA plays a speciﬁc role in countering oxidative stress in E. coli , possibly in particular micro-aerobic environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>FNR is a transcription regulator that controls gene-expression in response to oxygen limitation in E. coli ( Spiro and Guest , 1990 ; Unden et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Summary FNR Is a transcriptional regulator that controls gene-expression in response to oxygen limitation in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Regulation of succinate dehydrogenase ( sdhCDAB ) operon expression in Escherichia coli in response to carbon supply and anaerobiosis : role of ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9765574" target="_blank">9765574</a>]<br></li>
<li>The role of the FNR protein in the global control of E. coli gene-expression has been profiled in response to anaerobiosis ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Somewhat surprisingly , the Dfnr mutation caused a 2.4-fold reduction in aerobic dctA expression ( Fig. 8B ) , which suggests a minor aerobic role for FNR in the aerobic induction of dctA expression , as has been observed previously for eight unidentiﬁed E. coli gene products ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>]<br></li>
<li>Regulatory studies along with genome-wide expression approaches have shown that FNR , directly or indirectly , controls the expression of numerous gene products in E. coli ( 200 ) ( 32 -- 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>TTGAT ... ATCAAg -- tures which might allow HlyX and BTR to activate the expression of an E. coli gene which is not activated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>MIcrobtot Rev 56:195 -228 , Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Eschertchia coti FEMS Microbiol Rev 75 : 399-428 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934931" target="_blank">7934931</a>]<br></li>
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coil FEMS Microbiol Rev 75 : 399-428 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li>
<li>FNR-dependent repression of the ndh gene of Escherichia coli and metal ion requirement for FNR regulated gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_300">
Global gene-expression profiling in Escherichia coli K12 : the effects of oxygen availability and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></label>
<input id="rp_similarity_300", type="checkbox">
<div class="RP_similar">
<li>Global gene-expression proﬁling in Escherichia coli K12 : the effects of oxygen availability and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>FNR and its role in oxygen-related gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_303">
2:701 -707 -- ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></label>
<input id="rp_similarity_303", type="checkbox">
<div class="RP_similar">
<li>FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665461" target="_blank">7665461</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665524" target="_blank">7665524</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576043" target="_blank">8576043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209047" target="_blank">9209047</a>]<br></li></div>
<li>Spiro , S. and Guest , J.R. ( 1990 ) FNR and its role in oxygen-regulated gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Mol Gen Genet 228:125 -128 Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>J Gen Microbiol 133:3279 -- 3288 Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_308">
Mol Microbiol 6:3452 -- 3454 Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></label>
<input id="rp_similarity_308", type="checkbox">
<div class="RP_similar">
<li>Spiro S , Guest JR : FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>7 , 5 4 Spiro , S. and Guest , J. R. ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Spiro , S. & Guest , J.R. ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>Spiro , S. , and Guest , J. R. ( 1990 ) FNR and its role in oxygen-regulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10794600" target="_blank">10794600</a>]<br></li></div>
<li>Spiro , S. , and Guest , J. ( 1990 ) FNR and its role in oxygen-reg-ulated gene-expression in Escherichia coti .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_315">
Spiro , S. , and Guest , J. ( 1990 ) FNR and its role in oxygen regulated gene-expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></label>
<input id="rp_similarity_315", type="checkbox">
<div class="RP_similar">
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygen regulated gene-expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_317">
Spiro , S. , and Guest , J. ( 1990 ) FNR and its role on oxygenregulated gene-expression in Escherichia coll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></label>
<input id="rp_similarity_317", type="checkbox">
<div class="RP_similar">
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022286" target="_blank">8022286</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_319">
S. , and Guest , JR. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></label>
<input id="rp_similarity_319", type="checkbox">
<div class="RP_similar">
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022271" target="_blank">8022271</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23506073" target="_blank">23506073</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022270" target="_blank">8022270</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564515" target="_blank">10564515</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>[ 7 ] Spiro , S. and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Spiro , G. , and Guest , J. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li></div>
<li>J.R. ( 1990 ) FNR and its role in oxygenregulated gene-expression in Escherichia coli FEMS Microbiol Rev 75 : 399-428 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_324">
FNR and its role in oxygenregulated gene-expression in Eschenchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></label>
<input id="rp_similarity_324", type="checkbox">
<div class="RP_similar">
<li>FNR and its role in oxygenregulated gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961411" target="_blank">7961411</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071201" target="_blank">8071201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1744060" target="_blank">1744060</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>]<br></li></div>
<li>Recently , the E. coli K-12 yfgF gene has been identified as an FNR target ( Partridge et al. , 2007a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The available evidence therefore supports the notion that HIyX activates the expression of a latent haemolysin gene in E. coli , and that the failure of FNR to activate the expression of that gene is a consequence , in part , of subtle differences between the properties of HlyX and FNR ( Green et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>However , NO sensing by E.coli FNR has special physiological signi ® cance since FNR regulates expression of Hmp , which protects against NO and nitrosative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Stewart , V. ( 1982 ) Requirement of FNR and NARL functions for nitrate reductase expression in Esciierictiia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668685" target="_blank">2668685</a>]<br></li>
<li>Anaerobic growth and cyanide-synthesis of Pseudomonas aeruginosa depend on anr , a regulatory gene homologous with FNR of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Roles of SpoT and FNR in NH4 assimilation and osmoregulation in GOGAT ( glutamate synthase ) - deﬁcient mutants of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150242" target="_blank">15150242</a>]<br></li>
<li>Oxygen and nitrate-dependent regulation of dmsABC operon expression in Escherichia coli : sites for FNR and NarL protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Isolation of intact FNR protein ( Mr30000 ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1735706" target="_blank">1735706</a>]<br></li>
<li>The intermediate level of expression , seen in fermenting E. coli , is achieved by both ArcA protein acting as an activator and FNR protein serving as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The Escherichia coli K-12 nrf operon encodes the NrfA periplasmic nitrite reductase ( 9 ) , and its expression is controlled by a single promoter ( pnrf ) whose activity is completely dependent on FNR , the principal global transcription regulator responsible for anaerobic adaptation ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The expression of anaerobic-respiration and of related functions of E. coli is controlled by the positive gene regulator FNR , which activates transcription in the absence of 02 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The studies described here reveal the global changes in gene-expression of E. coli K-12 strain MG1655 that are dependent on environmental O2 and/or the anaerobic regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Introduction In Escherichia coli , the transcription factor FNR facilitates adaptation to anaerobic growth conditions by regulating the expression of gene products that are involved in anaerobic energy metabolism ( Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>FNR primarily functions to regulate anaerobic metabolism in E. coli and other bacteria by co-ordinating changes in gene-expression that optimize growth potential in different niches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The ansB gene encodes the anaerobic asparaginase I1 of E. and coli requires FNR and anaerobic growth with 17 amino acids for maximum expression ( Jennings & Beacham , 1993 ; Cedar & Schwartz , 1968 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>1 , 4000 Diisseldorf , Germany ( * request for offprints ) Received 15 October 1990 ; accepted 15 October 1990 Key words : anaerobic-respiration , FNR protein , oxygen regulation , gene-expression , E. coli Abstract Molecular oxygen is an important regulatory signal in facultative anaerobic bacteria and controles the expression of a great variety of genes positively or negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The Escherichia coli yhjA gene , encoding a predicted cytochrome c peroxidase , is regulated by FNR and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>introduction The expression of many of the gene products needed for anaerobic growth of Escherichia coli is tightly regulated and dependent on the FNR protein , a transcription factor Received 4 January , 1994 ; revised 31 May , 1994 ; accepted 2 June , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Oxygen regulated gene-expression in Escherichia coil : Control of anaerobic-respiration by the FNR protein Gottfried Unden * & Martin Trageser Institut far Biochemie , Gebtiude 26.42 , Heinrich-Heine-Universitiit , Universitiitsstr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>J Bacteriol 172:4197 -4205 Unden G , Trageser M ( 1991 ) Oxygen regulated gene-expression in Escherichia coli : control of anaerobic-respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>Eur J Biochem 146:193 -- 199 Unden G , Trageser M ( 1991 ) Oxygen regulated gene-expression in Escherichia coli : control of anaerobic-respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_347">
Oxygen regulated gene-expression in Escherichia coli : control of anaerobic-respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576043" target="_blank">8576043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></label>
<input id="rp_similarity_347", type="checkbox">
<div class="RP_similar">
<li>M. ( 1991 ) Oxygen regulated gene-expression in Escherichia coli : Control of anaerobic-respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>Oxygen regulated gene-expression in Escherichia coli : Control of anaerobic-respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li></div>
<li>35 , No. 1 269 -- 278 doi :10.1093 / nar/gkl1023 Transcription factor distribution in Escherichia coli : studies with FNR protein David C. Grainger * , Hirofumi Aiba , Douglas Hurd , Douglas F. Browning and 1 2 Stephen J. W. Busby School of Biosciences , University of Birmingham , Edgbaston , Birmingham B15 2TT , UK , Laboratory of 1 Molecular Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464-8601 , Japan and 2 Oxford Gene Technology , Begbroke Science Park , Sandy Lane , Yarnton , Oxford OX5 1PF , UK Received August 25 , 2006 ; Revised November 8 , 2006 ; Accepted November 14 , 2006 ABSTRACT Using chromatin immunoprecipitation ( ChIP ) and high-density microarrays , we have measured the distribution of the global transcription regulator protein , FNR , across the entire Escherichia coli chromosome in exponentially growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>Contents lists available at SciVerse ScienceDirect Biochemical and Biophysical Research Communications journal homepage : www.elsevier.com/locate/ybbrc Covalently linking the Escherichia coli global anaerobic regulator FNR in tandem allows it to function as an oxygen stable dimer Yue Shan a , Qing Pan a , Junyan Liu a , Feijuang Huang b , Hongzhe Sun b , Kunihiko Nishino c , Aixin Yan a , ⇑ a School of Biological Sciences , The University of Hong Kong , Hong Kong , China b Department of Chemistry , The University of Hong Kong , Hong Kong , China c Institute of Scientiﬁc and Industrial Research , Osaka University , Osaka 567-0047 , Japan a r t i c l e i n f o a b s t r a c t The anaerobic global regulator FNR from Escherichia coli is a [ 4Fe -- 4S ] 2 + cluster containing , O2 labile dimer that plays an important role in adapting the bacterium to its anaerobic lifestyle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_352">
Mössbauer spectroscopy as a tool for the study of activation/inactivation of the transcription regulator FNR in whole cells of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></label>
<input id="rp_similarity_352", type="checkbox">
<div class="RP_similar">
<li>Mösbauer spectroscopy as a tool for the study of activation / inactivation of the transcription regulator FNR in whole cells of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_354">
Popescu CV , Bates DM , Beinert H , Munck E , Kiley PJ ( 1998 ) Mössbauer spectroscopy as a tool for the study of activation/inactivation of the transcription regulator FNR in whole cells of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></label>
<input id="rp_similarity_354", type="checkbox">
<div class="RP_similar">
<li>Popescu , C.V. , Bates , D.M. , Beinert , H. , Munck , E. and Kiley , P.J. ( 1998 ) Mossbauer spectroscopy as a tool for the study of activation / inactivation of the transcription regulator FNR in whole cells of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li></div>
<li>J Bactenoi 170 : 3650-3654 , Trageser , M , , and Unden , G ( 1989 ) Role ol cysteine residues anci of metal ions tntlie leguiatory kinctioning of FNR , thetranscriptional legulator of anaerobic-respiration m Esctienchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_357">
Reviews 63:341 -358 Trageser M & Unden G ( 1989 ) Role of cysteine residues and metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic-respiration in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></label>
<input id="rp_similarity_357", type="checkbox">
<div class="RP_similar">
<li>[ 22 ] Trageser , M. and Unden , G. ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic-respiration in Escheri-chia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_359">
Role of cysteine residues and metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic-respiration in Escherichia coli 29 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></label>
<input id="rp_similarity_359", type="checkbox">
<div class="RP_similar">
<li>Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic-respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>]<br></li></div>
<li>FEBS Lett 340:59 -- 64 Trageser M , Unden G ( 1989 ) Role of cysteine residues and metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic-respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_362">
J Bacteriol 170:1589 -1597 Trageser M , Unden G ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , the transcriptional regulator of anaerobic-respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1851949" target="_blank">1851949</a>]<br></label>
<input id="rp_similarity_362", type="checkbox">
<div class="RP_similar">
<li>Trageser , M. , and Unden , G. ( 1989 ) Role of cysteine residues and of metal ions in the regulatory functioning of FNR , th transcriptional regulator of anaerobic-respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li></div>
<li>[ 33 ] Unden , G. and Schirawski , J. ( 1997 ) The oxygen-responsive transcriptional regulator FNR of Escherichia coli : search for signals and reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></li>
<li>'' Denitrification , FNR , FNR homologues , nitrogen-fixation Abstract Homologues of the transcriptional regulator FNR from Escherichia coli have been identified in a variety of taxonomically diverse bacterial species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_366">
170:321 -325 -- ( 1985 ) Isolation and characterization of the FNR protein , the transcriptional regulator of anaerobic electron transport in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></label>
<input id="rp_similarity_366", type="checkbox">
<div class="RP_similar">
<li>Isolation and characterization of the FNR protein , the transcriptional regulator of anaerobic electron transport in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1735706" target="_blank">1735706</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_368">
Arch Microbiol 147:195 -- 200 Unden G , Guest JR ( 1985 ) Isolation and characterization of the FNR protein , the transcriptional regulator of anaerobic electron transport in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="rp_similarity_368", type="checkbox">
<div class="RP_similar">
<li>[ 30 ] Unden , G. and Guest , J.R. ( 1985 ) Isolation and characterization of the FNR protein , the transcriptional regulator of anaerobic electron transport in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></li>
<li>Unden , G. , and Guest , J.R. ( 1985 ) Isolation and characterization of the FNR protein , the transcriptional regulator of anaerobic electron transport in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2041473" target="_blank">2041473</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160397" target="_blank">2160397</a>]<br></li></div>
<li>Summary FNR , the activator of anaerobic respiratory genes of Escherichia coli , has previously only been isolated as a protein of M , 29000 , which lacks nine W-terminal amino acid residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181237" target="_blank">2181237</a>]<br></li>
<li>Homology between CAP and FNR , a regulator of anaerobic-respiration in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1735706" target="_blank">1735706</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">FNR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>TABLE 1 Primers used in this work Primer name Sequence ( 5 to 3 ) a K12-FNR1 AAATTGACAAATATCAATTACGGCTTGA GCAGACCTTGTAGGCTGGAGCTGCTTCG K12-FNR2 TGATATGACAGAAGGATAGTGAGTTATG CGGAAAAACATATGAATATCCTCCTTAG FNR primer A TGGTTATTGCGCCATGAAGG FNR primer B TGGTTGGTCGTCCTGGTTAG NarX-P1 AGGTTATTGCTCATTTAAAGCCTGAAGGAA GAGGTTTACTGTGTAGGCTGGAGCTGCTTC NarL-P2 GATGCATTGTCAAACGACGAACTGCGCTG GGAACCGTAACATATGAATATCCTCCTTAG XL1 ACGCAGGATCTTCCGG XL2 GCAGCGAAATAATCTACCG yabN-189F CAGAATTGCGGTCATCCCACT yabN-318R TGCCCCATGCGTCAGTTTT yeaR-111F TGGTGCTTTTGTCACGCAGT yeaR-397R TTTCCCTCCTTCGGTGAATGC ogt-79F TCCGCAATCTGGTCGATTCT ogt-204R TACCGGCGCTAAAGGCAATAC gltB-279F TCCGCTGGAAGCTTTCTGGAT gltB-387R ATTCGCCCCCTTGTTGTCCTT dcuB-181F ACCCATCGCCAGAGAGTCTTTC dcuB-297R GGTCGTTACCGGCTTTAGCAAA cspA-215F CGCCAATGCGTGGCTTAAT cspA-376R TGTGCCTTTCGGCGATATG ytfE-145F TGGCAGGTTTGCTGTACACCT ytfE-260R TTCTGGTTTTGCTCTGACCGC cyoA-376F TGGTTATACCAATTGCCCGCC cyoA-495R TGGCTGCCAGGTCACATATGT ygbA-184F GCGCACATGATACCGGAGTTA ygbA-285R CATCACGCAAAAGCTGCAAA a Underlined nucleotides are identical to the regions of the E. coli chromosome flanking the fnr gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR Ec DNR Pa DnrD Ps DnrE Ps DnrS Ps FLP Lc FIpA LI FIpB LI FnrL Rs FnrP Pd MIPEKRIIRRIQSGGCAIHCQDCSISQLCIPFTLNE MEFQRVHQQLLQSHHLFEPLSPVQLQELLASS MVLHRVHHQILRSHHLFEPLNEEQMEELLNAS MAMLTGSAVLNTLRRHHLFSGLAEAALQDIAAHT MLTEKTLVAELCRLHLFSRLPEAALQEVCASA MAHSCTAVVPLFKNLNDEARA M-KIKDFDEHLSDHHCIQLVPLFSDLSPDELA MSKYK-HHFSHHEHHCVQLVPLFGLLSESELV MTLHEVPTILHRCGDCPIRHRAVCARCDSEEL MPHDAA-LCATQQPCEICPIRYRAVCAHCEG Figure 6 Comparison of the N-terminal sequences of FNR and FNR homologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>G211 ) , A/ac 434 J. Green and J. R. Guest -220 -240 -200 -160 -160 EcoRl SECONDARY FNR SITE TCACTCTATATCACTCTCACAAATTCGCTCAAATAATAAACAATAAACTCTCfiTTTTTGATCTCACCCGGTAA / ^ TCGCCTATCTTTTCAGCAACA -140 -120 -100 Hpall - GO PRIMARY FNR SITE - 3 5 - 1 0 PJACTTGATTAACATCAATTTIJGGTATGACCAATGCACCATTCATGTTATTCTCAATAGCGAAGAACATTTTCATTGCTGTAACCTGTTGTTAATTAAGAG -60 - Ttr -- -20 +1 +20 +40 Met C ^ ATGTTAATAACCATTAATTAAfilAATTGGTTAATAAATTTAAGGGGGTCACGTTGACTACGCCATTGAAAAAGATTGTGATTGTCGGCGGCGGTGCTGGT +60 +60 +100 +120 +140 GGQCtGGM > ATGSCPACACf < GGGGGATCCTCTAGAGTCGACCTGCAGGCATGCAAGCTT +160 PvuII Windlll / Sma I Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Sequences resembling FNR boxes are located in the promoter regions of the nirS and nirM genes encoding a cytochrome cdl nitrite reductase and cytochrome c551 respectively , the norC gene encoding a subunit of the nitric oxide reductase and a possible regulatory gene , off8 ( Fig. 2 ; Jtingst et al. 1991 ; Cuypers & Zumft 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>This implies that more Fis binds to the nrf promoter region p23C p10T p8T p7T p22C / p22A p21A p42T NarL / NarP pnrf FNR TAACTCTAAAGTGGTATTTTACATGCACTTACAATTGATTAAAGACAACATTCACAGTGTGGTTATTTGTTACACATAGGGGcGAGCA WWWWWWnnnnnnnnWATCAAnnnnTTR GntcAaaTttTganC +1 IHF Consensus tg tAa c Fis Consensus Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>To test whether the FNR sites upstream of yecI , yjhB , yfgF , yhjA and smf contributed to regulation of these genes , the corresponding promoter regions ( e.g. MC4100 oxyR-Kan Zheng et al. ( 1999 ) Lab collection JRG1728 JRG3462 MC1000 D ( tyrR-fnr-rac-trg ) 17 zdd-230-Tn9 CAG627 pGS771 Meng et al. ( 1998 ) This work MC1000/lRZ5 ( PyhjA -- lacZ ) from pGS1634 JRG5066 JRG5067 JRG1728/lRZ5 ( PyhjA -- lacZ ) from pGS1634 W3110 DyhjA This work JRG5354 JRG5393 MC1000 oxyR-Kan This work This work JRG5393/lRZ5 ( PyhjA -- lacZ ) from pGS1634 JRG5609 JRG5641 MC1000/lRZ5 ( PyhjA0 -- lacZ ) from pGS2018 MC1000/lRZ5 ( PyhjA9 -- lacZ ) from pGS1829 This work This work JRG5694 MC1000 D ( lacIPOZYA ) X74 galU galK D ( araABC-leu ) XL-1 Blue pMV107 F , l 2 2 Lab collection MV247 W3110 Haagmans & van der Woude ( 2000 ) Lab collection Plasmids pACYC184 Source of cat cassette for linear transformation PyfgF -- lacZ fusion in pRS415 ( 192 bp promoter fragment , 2 +84 relative to the transcript start ) , Ap R Martinez et al. ( 1988 ) This work pGS1633 108 to pGS1634 This work PyhjA -- lacZ fusion in pRS415 ( 187 bp promoter fragment , 2127 to +60 relative to the transcript start ) , Ap R PyecI -- lacZ fusion in pRS415 ( 399 bp promoter fragment ) , Ap R Psmf -- lacZ fusion in pRS415 ( 383 bp promoter fragment ) , Ap R PyjhB -- lacZ fusion in pRS415 ( 450 bp promoter fragment ) , Ap R PyhjA -- lacZ fusion in pRS415 with the FNR-binding site altered from TTGAT-N4-ATCAA to ATCAT-N4-ATGAT , Ap R PyhjA : : lacZ fusion in pRS415 with the OxyR-binding site altered from ATAGgcacaggCTATcttattgATAGtTtatAttcAT to TATCgcacaggGATActtattgTATCtAtatTttcTA , Ap R pGS1740 pGS1740a This work This work pGS1828 pGS1829 This work This work pGS2018 This work Haagmans & van der Woude ( 2000 ) Simons et al. ( 1987 ) pMV107 pRS415 pZE24 with a 929 bp fragment encoding OxyR lacYZA-based promoter-fusion vector , Ap R pTP223 lam , bet and exo genes of phage l , under the control of the lac promoter , Tet R Poteete & Fenton ( 1984 ) Oligonucleotides JDP1 This work This work TTTTGAATCAACTGTAGGTGGGTATC TTTTGGATCCGTGTTATTCTAATATAGCCC JDP2 JDP3 This work This work TTTTGAATTCTGTAATAATGATGAATTAACATG TTTTGGATCCGCAGAATCCCCCACATCC JDP4 JDP5 This work This work TTTGAATTCCCGCAATCATGATATGCC TTTGGATCCCGGTGGTGAGTTATTGTG JDP6 JDP7 This work This work CGTAATCCATCAACTGTTTCG GCCTAAAACCTTATTATTTTCAC JDP8 FyecI TTTTGAATCTTAATTATCACTGGGCCCATTAATTAACAG TTTTGGATCCAAGCATTCCAGCGGATCCCATACCTAATAT RyecI Fsmf TTTTGAATCTCTAAAAGCTCTGAATTCARTTAACACTAGC TTTTGGATCCTTTCTGTATCGAGGATCCTTATCTCCCTGC Rsmf Cultures of MC1000 ( lac ) and JRG1728 ( lac fnr ) transformed with plasmids pGS1828 ( PyjhB -- lacZ ) , pGS1633 ( PyfgF -- lacZ ) , pGS1634 ( PyhjA -- lacZ ) , pGS1740 ( PyecI -- lacZ ) , pGS1740a ( Psmf -- lacZ ) , were grown under aerobic and anaerobic conditions at 37 uC in L broth to OD600 0.4 -- 0.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>This work This work TTTTGAATCTTAATTATCACTGGGCCCATTAATTAACAG TTTTGGATCCAAGCATTCCAGCGGATCCCATACCTAATAT RyecI Fsmf This work This work TTTTGAATCTCTAAAAGCTCTGAATTCARTTAACACTAGC TTTTGGATCCTTTCTGTATCGAGGATCCTTATCTCCCTGC Rsmf Of the 12 transcriptional units associated with consensus FNR sites , seven ( cydA , ndh , narK , narG , nirB , ydhY and yﬁD ) are known to be FNR-regulated ( Green et al. , 1998 ; Guest et al. , 1996 ; Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>FNR samples inactivated by either O2 or NO bind speci ® cally to Phmp , but with lower af ® nity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Both CRP and FNR are 50-kDa homodimers that bind related inverted-repeats with a TGANNNNNNTCA core motif ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Conserved positions in the inveriedrepeat thought to be recognizedby FNR are shown in bold face and the probable ' -10 ' sequences recognized by RNA polymeraseare overlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>A 26 bp region overlapping the FNR box and located between ± 6 and +20 in Phmp was protected by the addition of 0.2 mM holo-FNR ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>By itself , RNAP bound to a 64 bp DNA region ( ± 44 to +20 ) containing the ± 10 and ± 35 hexamers , as well as the FNR box ( Figure 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_13">
Radiolabelled ydhY promoter fragments [ native promoter , lanes 1 and 2 ; promoter with the FNR site mutated ( TTGATAACGATCAAAATCATAACGATGAT ) , lanes 3 and 4 ] were incubated with ( + ) or without ( '' ) 8 mM FNR * ( FNR-D154A ) for 10 min before separating FNR * -- DNA complexes ( C ) from free DNA ( F ) in TBE-buffered polyacrylamide gels and subsequent visualization by autoradiography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="sit_similarity_13", type="checkbox">
<div class="SIT_similar">
<li>Radiolabelled yfgF promoter fragments [ native promoter , lanes 1 and 2 ; promoter with the FNR site mutated ( TTGATATATATCAAAATCATATAT-ATGAT ) , lanes 3 and 4 ] were incubated with ( + ) or without ( '' ) 8 mM FNR * ( FNR-D154A ) for 10 min before separating yfgF -- FNR * complexes from free ( yfgF ) DNA in Tris/borate / EDTA-buffered polyacrylamide gels and visualization by autoradiography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li></div>
<li>pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus CRP binding site , ApR pBR322 carrying the complete crp gene and crp promoter , ApR pBR322 carrying the crp promoter and the pUC9 polylinker in place of the crp gene , ApR pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus FNR binding site , ApR pGEX-KG containing the fnrDA154 mutation , ApR Promoter-probe vector with promoter-less cat gene , ApR pUC18 carrying the 310-bp AluI fragment ( xseP ^ guaP ) of pMT104 , ApR pUC18 carrying the 310-bp AluI fragment ( guaP ^ xseP ) of pMT104 , ApR pKK232-8 carrying an approximately 500-bp xseP ^ guaP fragment in front of the promoter-less cat gene , ApR CmR pKK232-8 carrying an approximately 500-bp guaP ^ xseP fragment in front of the promoter-less cat gene , ApR CmR pPV-33H carrying a 1.03-kb guaP BP fragment , ApR small , high copy number vector carrying a 54-bp polylinker in the lacZ N-terminal sequence and the ColE1 origin of replication , Apr [ 26 ] [ 27 ] [ 27 ] [ 26 ] [ 24 ] [ 28 ] this work this work this work this work [ 1 ] [ 23 ] 2.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Description Bacterial strains JM101 JRG3462 M182 M182vcrp MH1002 MH1003 MH1004 MH1005 MH1009 MH1010 MH1011 MH1012 MH1015 MH1016 MT1024 Plasmids pCC pDCRP pDU9 pFF pGS771 pKK232-8 pMH100 pMH101 pMH102 pMH103 pMT104 pUC18 15 proAB ] v ( lac-proAB ) thi supE [ FP traD36 laclq v ( lacZ ) M CAG627 ( pGS771 ) v ( lacIPOZY ) X74 galK galU rpsL M182 vcrp JM101 ( pMH100 ) JM101 ( pMH101 ) M182 vcrp ( pDU9 ) M182 vcrp ( pDCRP ) JM101 ( pMH102 ) JM101 ( pMH103 ) M182 ( pMH102 ) M182 vcrp ( pMH102 ) M182 ( pMH103 ) M182 vcrp ( pMH103 ) W3110 ( pMT104 ) [ 23 ] [ 24 ] [ 25 ] [ 25 ] this work this work this work this work this work this work this work this work this work this work [ 1 ] pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus CRP binding site , ApR pBR322 carrying the complete crp gene and crp promoter , ApR pBR322 carrying the crp promoter and the pUC9 polylinker in place of the crp gene , ApR pAA182 carrying an approximately 300-bp EcoRI ^ HindIII fragment with a near consensus FNR binding site , ApR pGEX-KG containing the fnrDA154 mutation , ApR Promoter-probe vector with promoter-less cat gene , ApR pUC18 carrying the 310-bp AluI fragment ( xseP ^ guaP ) of pMT104 , ApR pUC18 carrying the 310-bp AluI fragment ( guaP ^ xseP ) of pMT104 , ApR pKK232-8 carrying an approximately 500-bp xseP ^ guaP fragment in front of the promoter-less cat gene , ApR CmR pKK232-8 carrying an approximately 500-bp guaP ^ xseP fragment in front of the promoter-less cat gene , ApR CmR pPV-33H carrying a 1.03-kb guaP BP fragment , ApR small , high copy number vector carrying a 54-bp polylinker in the lacZ N-terminal sequence and the ColE1 origin of replication , Apr [ 26 ] [ 27 ] [ 27 ] [ 26 ] [ 24 ] [ 28 ] this work this work this work this work [ 1 ] [ 23 ] 2.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>Page et al. ( 1990 ) Wu et al. ( 1998 ) Tyson et al. ( 1994 ) Wu et al. ( 1998 ) Wing et al. ( 1995 ) JCB387 narL narP253 : : Tn10d ( Cm ) JCB3884 ﬁs985 ( str/spcR ) Dfnr General cloning vector for EcoRI -- HindIII ragments derived from pBR322 lac fusion vector for cloning test promoters on EcoRI -- HindIII fragments : contains colE1 origin of replication and encodes AmpR pBR322 derivative containing two l oop transcription terminators Broad-host-range lac fusion vector for cloning promoters on EcoRI -- HindIII fragments : contains the RK2 origin of replication and encodes TcR Kelsall et al. ( 1985 ) pAA182 Jayaraman et al. ( 1987 ) pLSR El-Robh and Busby ( 2002 ) pRW50 Lodge et al. ( 1992 ) Promoters ( all on EcoRI -- HindIII fragments ) pnrf97 E. coli nrf promoter fragment carrying nucleotide sequences from -87 to +10 E. coli nrf promoter fragment carrying nucleotide sequences from -209 to + 131 E. coli nrf promoter fragment carrying nucleotide sequences from -87 to +131 E. coli nrf promoter fragment carrying nucleotide sequences from -66 to +131 E. coli nrf promoter fragment carrying nucleotide sequences from -56 to +131 FNR-dependent promoter with FNR binding site centred at position -41.5 of the melR promoter Browning et al. ( 2002 ) pnrf53 Tyson et al. ( 1994 ) pnrf53 D87 Darwin et al. ( 1997 ) pnrf53 D66 K.L. Tyson ( unpublished ) pnrf53 D56 Browning et al. ( 2002 ) FF ( -41.5 ) Wing et al. ( 1995 ) Elements responsible for catabolite repression of the nrf promoter The nrf promoter is maximally active in minimal medium and repressed in rich medium ( Tyson et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Strains E. coli K-12 strains nir lac JCB387 narL narP253 : : Tn10dCm JCB387 JCB3884 JCB38841 JCB38849 JCB38849S JRG1728 E. coli E2348/69 S. enterica serovar Typhimurium LT2 18 25 JCB3884 ﬁs985 str/spcr JCB3884 himA452 : : Tn10dTc Tets version of JCB38849 isolated as a fusaric acid-resistant colony fnr lac EPEC Wild type 28 3 This work 26 I. Henderson I. Henderson Plasmids pAA121 pSR pRW50 Cloning vector for EcoRI-HindIII fragments derived from pBR322 pBR322 derivative containing a oop transcription terminator Broad-host-range lacZ fusion vector for cloning promoters on EcoRI-HindIII fragments ; contains the RK2 origin of replication 12 13 14 Promoters pnir7150 E. coli nir promoter fragment carrying nucleotide sequences from 150 to 36 E. coli nir promoter fragment carrying nucleotide sequences from 133 to 36 E. coli nir promoter fragment carrying nucleotide sequences from 106 to 36 E. coli nir promoter fragment carrying nucleotide sequences from 83 to 36 E. coli pnir7150 promoter fragment carrying a T-to-A substitution at position 146 E. coli pnir7150 promoter fragment carrying a C-to-G substitution at position 112 E. coli pnir7150 promoter fragment carrying an A-to-G substitution at position 99 E. coli pnir7150 promoter fragment carrying the p99G and p146A substitutions E. coli pnir7150 promoter fragment carrying the p99G , p112G , and p146A substitutions E. coli pnir7150 promoter fragment carrying 5-bp insertion at position 60 E. coli pnir7150 promoter fragment carrying 5-bp insertion at position 134 E. coli pnir7150 promoter fragment carrying 10-bp insertion at position 134 E. coli pnir7133 promoter fragment carrying 5-bp insertion at position 106 E. coli pnir7133 promoter fragment carrying 10-bp insertion at position 106 E. coli pnir7106 promoter fragment carrying 5-bp insertion at position 60 EPEC nir promoter fragment carrying nucleotide sequences from 150 to 36 S. enterica serovar Typhimurium nir promoter fragment carrying nucleotide sequences from 150 to 36 E. coli nrf promoter fragment carrying nucleotide sequences from 61 to 131 Promoter fragment carrying pnir7150 nucleotide sequences from 150 to 61 and pnrf53 nucleotide sequences from 60 to 131 Semisynthetic FNR-dependent promoter carrying a consensus FNR site at position 41.5 2 pnir7133 pnir7106 pnir7083 pnir7150/p146A pnir7150/p112G pnir7150/p99G This work This work This work 28 5 28 pnir7150/p99Gp146A 28 5 pnir7150/p99Gp112Gp146A pnir7150/H74 .5 pnir7150 / 5 pnir7150 / 10 pnir7133 / 5 pnir7133 / 10 pnir7106 / 5 pnirEPEC pnirSTM 28 This work This work This work This work This work This work This work pnrf53 / 61 pnir-nrf 4 25 FF ( 41.5 ) 26 RESULTS Modulation of the nir promoter from upstream sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>NarL , wiiicin was initiaily discovered as a transcription factor essentiai for the activation of nitrate reductase expression by nitrate ( Stewart , 1982 ) , Recent studies of the nitrate reductase promoter have identified the FNR - and NarLbinding sites around 41 and 200 bp upstream of the transcription start , respectiveiy ( Li and DeMoss , 1988 ; Walker andDeMoss , 1991 ; Dong ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Analysis of the DNA upstream of the yﬁD coding region revealed an unusual architecture for an FNR-activated promoter ( Fig. 1 ) with two FNR sites positioned at ¹ 61.5 and ¹ 114.5 relative to the transcription start for HlyXmediated expression ( Green and Baldwin , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Transcript mapping using RNA isolated from HlyX-expressing stationary-phase cultures indicated that the FNR sites were 61.5 ( FNR I ) and 114.5 ( FNR II ) bp upstream of the transcript start ( Green and Baldwin , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>In two denitrifying Pseudomonas species , P. stutzeri and P. aeruginosa , expression of the nir , nor , and nos genes is regulated by the NO-responsive FNR-like transcriptional activator DNR that binds to a DNA motif similar to the consensus FNR box , TTGATnnnnATCAA [ 34,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>In other cases the evidence is at present only indirect , and involves the identification of DNA sequences potentially recognised by FNR ( subsequently referred to as FNR boxes ) , observations on the behaviour of heterologous promoters that are consistent with the presence of an FNR homologue , or hybridisation to thefnr homologue of another organism ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>e.g. for FNR at position 11 a T which is found five times in a sample of 14 G 6 2 4 1 3 2 3 2 5 3 2 - 7 2 1 6 4 1 2 1 5 2 3 2 2 1 4 1 1 6 7 6 7 5 5 6 - - 1 6 2 - - 1 1 6 9 9 4 7 7 6 3 7 7 3 6 7 7 4 9 9 6 1 1 - - 2 1 6 - - 6 5 5 7 6 7 6 1 1 4 1 2 2 1 2 5 1 2 1 4 1 6 2 7 - 2 3 5 2 3 2 3 1 4 2 6 A T C cons T A T T T A T T T T C A G T G A A A A T A A A T A G K K K C A C T G T T T C Discussion The aim of the present study was to show that FNR actually stimulates transcription , to characterize a group of FNR-dependent promoters , and to detect sequences likely to correspond to FNR bindingsites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The major effect of mutation on anaerobic induction of promoters 6 and 7 is consistent with the existence of putative FNR recognition sequences directly upstream of these promoters ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>To confirm that these two promoters direct transcription and to examine whether these putative FNR-binding sites are required to mediate FNR-dependent anaerobic induction of pfl operon expression , base pair substitutions were introduced with the aim of destroying the recognition capacities of these respective sequences for RNA polymerase or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>The FNR protein probably binds to at least two related DNA sequences in the regulatory region of the pfl operon , is both of which are proximal to promoters ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Thus it is envisaged that in response to anoxia , FNR is transformed by a conformational change into a transcriptional activator or repressor , which recognizes and binds to specific DNA sequences in the promoters of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Under anaerobic conditions FNR is a dimeric iron-sulfur protein that binds at specific DNA sequences to activate the expression of `` anaerobic genes '' and to repress the expression of `` aerobic genes '' ( 7 , 8 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>FNR binds to ∼ 25 bp sequences at target promoters and recruits RNA polymerase , thereby activating transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Hence , in anaerobic conditions , FNR is able to bind to speciﬁc DNA targets at promoters and modulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>y Binding n-Regulation of the Escherichia coli K-12 nrf Promoter b of the NsrR Nitric Oxide-Sensing Transcription Repressor to an Upstream Site Douglas F. Browning ,1 * David J. Lee ,1 Stephen Spiro ,2 and Stephen J. W. Busby1 School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom ,1 and Department of Molecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 750802 ow D Received 1 March 2010/Accepted 9 May 2010 FNR-dependent activation of the Escherichia coli K-12 nrf promoter is downregulated by the nitric oxide-sensitive NsrR protein together with the nucleoid-associated protein IHF , which bind to overlapping targets adjacent to the DNA site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>DNA Binding of FNR152 to a Synthetic Consensus FNR Recognition Site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Consensus FNR A A A * T T G A T **** A T C A A * T T T feo AACCTTGAGCCACA T C A A CATT FIG. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8407793" target="_blank">8407793</a>]<br></li>
<li>A previous study to identify new members of the FNR regulon by searching the E. coli K-12 genome sequence for sites similar to this consensus using the search pattern tool in the Colibri database ( http : / genolist.Pasteur.fr / Colibri ) identified yfgF as a potential target ( Partridge et al. , 2007a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The E. coli K-12 genome sequence was searched for the presence of consensus FNR sites ( TTGATnnnnATCAA ) using the search pattern tool in the Colibri database ( http : / / genolist.Pasteur.fr / Colibri ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Studies with t w o rnutant frd prornoters frorn Escherichia coli , displaying altered regulation and FNR response , provided additional evidence for recognition of this sequence by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Consensus sequences to which the FNR protein may bind have been proposed on the basis of sequence homology in the 5 ' - noncoding regions of FNR-dependent genes ( 18 , 22 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>To explain the repressive effect on narX transcription by FNR as disclosed in the present study , another consensus sequence between + lOl and + ll4 ( box ( l ) of ~ ~ r ~ inTable 1 ) isproposed to be essential as an additional anaero-box in the overlapping narX/K promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>The FNR homolog in S. Typhimurium , oxaR , has a similar but less defined consensus sequence [ 60 ] , which could account for the lack of association with stm57 motif # 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22144874" target="_blank">22144874</a>]<br></li>
<li>The recognition sequence for FNR exhibits twofold symmetry ( Spiro and Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>The fourth column includes sequence logos for presumably homogeneous and large site sets and sequence consensi for small sets of DNA sites and for well-established motifs of other factors ( FNR , CRP , CooA , NtcA , ArcR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The consensus sequence for FNR is similar to that for CRP consensus , and previous studies have shown that FNR activates transcription initiation in a similar way to CRP and that both proteins can bind to the DNA site for the other protein ( 41,42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>The DNA sequences of the mutated FNR site and Nar heptamers are shown below the main sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The DNA sequences protected by ArcA-P or FNR * - DA154 from DNase I hydrolysis and also containing the consensus sequence for the two proteins are bracketed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The conservation of sequences within the DNA-binding domains and FNR boxes from non-enteric bacteria lends support to the idea that there is a conservation of the mechanism of DNA sequence recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The sizes of the DNA footprints are consistent with the binding of two monomers of FNR that protect the symmetrical FNR-recognition sequence TTGAT-nnnnATCAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The palindromic nature of the DNA recognition sequence dictates that FNR binds as a dimer ( Green ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>The tentative conclusion is that the FNR homologues recognise the same , or a very similar DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>( C ) The figure shows a DNA sequence motif present at newly identified FNR targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>See M methods section for details oter DNA in aterials and FNR binding to moe operator/promoter DNA Although FNR behaves as a repressor of moeA-lacZ expression in vivo , an FNR-binding consensus sequence ( TTGAT-N4-ATCAA ; Spiro and Guest 1990 ) is not detectable in the DNA upstream of the moe operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Both proteins apparently recognize the same DNA sequence as FNR but the physiological signal to which they respond is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>FNR protein is a homodimer that contains symmetrical he-lix-turn-helix structures that interact with a highly conserved , dyad consensus DNA sequence ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>A putative FNR consensus sequence ( 44 ) in the hyf upstream DNA can be detected between positions 111 and 98 ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The FNR * - DA154 protein , at a concentration less than 0.5 µM , protected a region of DNA upstream of moe that included the TTCAT sequence in the moe operator DNA ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>It is possible that FNR binds to the sequence TTCAT in the moe operator DNA to facilitate repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>If the moe operator DNA has the full FNR consensus sequence in the operator region , the moe operon would be effectively repressed under conditions of the highest demand for Mocofactor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Our ﬁndings suggest that FNR and CRP bind to the same sequence in the clyA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Since modification of the putative FNR boxes affected FNR binding to the nrdD promoter sequence , we examined whether such modifications also altered in vivo transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>FNR is an iron-dependent , DNA-binding protein which recognizes a specific sequence motif found in the promoter regions of the genes it regulates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Transfer of the target sequence for FNR to a CRP - dependent promoter Recent studies by Guest and coworkers have clearly shown that the DNA binding domains of the FNR protein and CRP are homologous and , therefore , likely to recognize similar sites ( Shaw et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Nevertheless , such cross-talk is likely to be limited because FNR can not recognize a CRP-site consensus sequence in vitro ( Green et al. , 1996b ; Sawers et al. , 1997 ) and the affinity of CRP for a FNR-site consensus is 50-fold lower than for its natural binding site ( Sawers et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The cofactors of FNR essential for specific binding to DNA were not studied so far in vitro , since isolated FNR does not bind sequence specific to DNA ( Green et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Identification of FNR - and NarP Promoter-binding Sites -- Potential FNR - and NarP-binding sites were located using the sequence motif search facility of Colibri ( genolist.pasteur.fr / Colibri/genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Regulation by cyclic AMP receptor protein and anerobiosis ( FNR protein ) was confirmed , and the presence of nucleotide sequence motifs , with homology to cyclic AMP receptor protein and FNR protein-binding sites , investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>Identification and sequence analysis of FNR targets To determine the location of peaks for FNR binding in an unbiased manner , a Cy5/Cy3 cut-off was applied to the ` anaerobic ' dataset .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>At 10 of the 63 targets of FNR binding , we were unable to identify a match to the canonical FNR binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>Whether this is due to competition between FNR and RNA polymerase for binding to Pfnr and/or reﬂects additional sequence requirements for FNR binding in vivo is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Motif # 1 matches the E. coli FNR ( fumarate and nitrate reduction ) binding consensus sequence ( TTGAT N4 ATCAA ) [ 57 ] and eco57 motif # 3 corresponds to the NarP binding sequence [ 58,59 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22144874" target="_blank">22144874</a>]<br></li>
<li>No typical consensus sequence , however , could be found for the binding of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Eco57 motif # 1 matches the known FNR binding sequence and motif # 3 matches the known NarP binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22144874" target="_blank">22144874</a>]<br></li>
<li>PhypA does not possess a canonical FNR recognition sequence , but two FNR half-sites are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>FNR sites with a poorer match to the consensus sequence are present in these promoter regions , so that FNR repression might be lifted when NO deactivates a smaller percentage of the total FNR pool .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR consensus sites [ FNR ( 1 ) to FNR ( 3 ) ] and half-sites ( FNR/2 ) with $ 6 conserved residues ( bold letters ) of the FNR consensus sequence ( TTGAT -- ATCAA ) ( 8 , 11 ) are shaded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>Residues matching the corresponding consensus sequence residues for the FNR and CRP sites are in boldface type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Conserved residues serving as a fingerprint for the FNR family of flavoproteins [ 30 ] are printed reversed and marked with ` 1 ' above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>omains inferred from the primary sequence of FNR ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Similarly , the sequence that FNR recognises ( the FNR box ) is well characterised and substantiated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The FNR box centred at position +5.5 was bound speci ® cally by holo-FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Molecular modeling suggests a mechanism by which Phe at position 24 protects the FNR [ 4Fe-4S ] 2 cluster and that Cys-23 or the iron bound to it is a primary site of O2 interaction with the [ 4Fe-4S ] 2 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>FNR targets identified by ChIP-chip analysis Sequence motif identified by AlignACE Site centre Distance from nearest transcription start site Gene Peak centre Identified by transcriptome analysis ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>Sequence motif identified by AlignACE Site centre Distance from nearest transcription start site Peak centre Gene ( E ) DNA/RNA manipulation 579 727 nohB 1 023 647 yccF/helD 1 407 100 dbpA 1 634 803 nohA/ydfO 4 584 413 hsdR/mrr 5 - TTAAGTTGATGCAGATCAATTAAT-3 0 0 5 - AGTAATTGATTGAAAGGAATAAGG-3 0 0 5 - AAAGTTTGAGCGAAGTCAATAAAC-3 0 0 5 - TTAAGTTGATGCAGATCAATTAAT-3 0 0 5 - CATCATTGTTATTAATCCATTGCT-3 0 0 No No No No No 579 735.5 1 023 811.5 1 407 110.5 1 634 713.5 4 584 337.5 Unknown Unknown Unknown 85.5 / unknown Unknown The table lists the locations of peaks for FNR binding identified using chip and high-density microarray analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>CTTTAAGTGTAATTCGGGGCAGAATTGGTAAAGAGAGTCGTGTAAAATAT CGAGTTC ~ CACATTTTGTTGTCTGATTATTGATTTTTGGCGAAA ~ CATTT GATCATATGACAAGATGTGTATCTACCTTAACTTAATGATTTTGATAAAA f s t AAAAAAATCGATCTCGTCAAATTTCA < 16 > TATACT < 11 > A 1 f z - 4 C < 16 > TATACT < 11 > A FNR C O ~ S AAA-TTGAT -- ATCAA-TTT f 2 - S AAAAAATCGATCTCIGTCAAATTTCA < 16 > TATACT < 11 > A lnformation theory and the FNR site T m m A T C T C G - m G G A A A A C C A T C A T T C A T A T C A A A T T T C C G C A A A A T G A C G C A T G I A A A T C A C G T T T C A C A T C A C T C T T 6 A T C G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>DNase I footprinting Footprinting reactions with the 458 bp ndh promoter fragment ( Fig. 1 ) confirmed that FNR protects over the primary site at - 6 1 to - 4 0 and the secondary site ( -105 to -84 ) when the lower strand was labelled ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The ndh promoter is repressed by FNR under anaerobic conditions ( Spire et al. , 1989 ) and FNR protects a 27 bp region ( -36 to -62 , the primary site ) , which overlaps the predicted FNR-bind-ing site , and a 34 bp region ( -76 to -109 , the secondary site ) , which contains only one FNR half-site motif ( Sharrocks etat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The importance of the upstream secondary FNR site was inferred from DNase I footprinting in which several FNR variants that failed to repress ndh expression in vivo ( Sharrocks et al. , 1990 ) retained the ability to bind at the downstream primary site ( -50.5 ) but were unable to recognize the upstream site ( Green eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>FNR binds at an FNR box within the hmp promoter ( Phmp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Surprisingly , we observed two FNR-dependent shifts , suggesting that FNR binds a second , previously unknown , site within this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>β-Galactosidase activity narL + narP + 14 400 + narL narP 12 500 + narL narP 16 700 narL narP 7300 36 500 36 000 30 700 11 000 32 400 42 500 22 300 10 100 Nitrate - and nitrite-mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate-activated regulators , NarL and NarP ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>( 1990 ) D25X of pmeIR -10 sequence and transcript start on EcoRI-H/ndlll fragment Broad-host range/ac expression vector Lodge ef a / , ( 1992 ) with poiylinker upstream of promoterless / ac genes pnirB cloned into pRW50 giving pnirB - This work / acZ operon fusion pBR322 derivative carrying Corona Spiro ef a / , ( 1990 ) BamHI-H/ndlll fragment pRW50 pKTI pFNR ( Raibaud , 1989 ) , Our current work suggests that a similar structure may form at the nirB promoter : this may explain the large number of polyA tracts in the 150 bp non-coding region upstream of the pnirB NarL - and FNR-binding sites ( Jayaraman e ^ a / , , 1987 ) , Experimentai procedures E. coli strains , promoter fragments and piasmids used in this work are listed in Table 2 , By convention , promoter sequences are numbered with the transcript start site as +1 , Standard methods for recombinant DNA work were used throughout this project , Transformants carrying pRW50 derivatives were grown on media containing 35 ) igmr ^ tetracycline , whilst 80 Mg ml '' ' ampicillin was used to maintain pFNR derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>To test this hypothesis , these binding sites were eliminated individually b A B ACTCTTTCTCCTTTGAGGGGCAGCCACAAAAAAAATCGACGGATTAT FNR ACCTCCTTTCTTCAAGGCGGCAATATTCTTTTCGTTGACTTTAGTCAA AATGATAACGGTTTGAGATAAAGTTATTTTATATTCAGATGGTTATG AAAGAAGATTATTCCATCCGAAAACTAACCTTTACCCTGGCACAAGT CTTCTTTCGCCGCGCGCCTGGGGAAAAGACGTGCAAAAAGGTTGTGT PurR AAAGCAGTCTCGCAAACGTTTGCTTTCCCTGTTAGAATTGCGCCGAA TTTTATTTTTCTACCGCAAGTAACGCGTGGGGACCCAAGCA C Deletion of FNR binding site in pInterD1 Deletion of PurR binding site in pInterD2 Figure 1 E. coli chromosomal DNA insert in high copy plasmid clone pAQ5 and its derivatives ( a ) Clone pAQ5 containing sequence in the upp-purM-purN region was selected from an E. coli genomic DNA plasmid library for resistance to cell killing mediated by mutant topoisomerase I YpTOP1-D117N expressed in BW117N .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>In previous work ( 3 -- 6 ) , we showed that the DNA target for FNR at pnrf is centered at position 41.5 ( i.e. , between 41 and 42 bp upstream from the transcript start point ) , that anaerobically induced FNR-dependent activation of pnrf is downregulated by the binding of the nucleoid-associated protein IHF to a target ( IHF I ) centered at position 54 , and that this repression is relieved by the binding of either NarL or NarP to a DNA target centered at position 74.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Results in Fig. 2 show that FNR and NarL each protect 30 bp , consistent with binding of a dimer of each protein to a single target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>As expected , when NarL or FNR was incubated with the pnir7150 fragment , single shifted complexes were detected ( Fig. 6A , lanes 1 -- 4 and 5 -- 7 ) , conﬁrming that each protein binds to a single site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Using mutational analysis and gel retardation assays , we showed that IHF binds to a site centred at position -54 , overlapping the DNA sites for FNR and NarL/NarP , while Fis binds to a site at position -15 , overlapping the promoter -10 element ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>This results from its dependence on FNR that binds to a site at position -41.5 , and NarL or NarP that bind to a 7-2-7 element centred at position -69.5 ( Fig. 10 ; Jayaraman et al. , 1989 ; Tyson et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>In the case of the six promoters activated by FNR , these always precede the ' -10 ' region by -30 nucleotides , whereas the position of the CAP-recognition site shows considerable variation ( Busby , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>When this was applied to the FNR recognition sequence , taken from the alignment shown in Fig. 6 , each half-site wasfound to havean , R , , = 7.79 bits of information compared to 7.24 for the CAP site : As the Schneider alogorithm is especially powerful in assessing the relative inforrnational content of each position , it was employed to compare the FNR and CAP sites and in particular to discriminate between them at those positions at which both show the same preference ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Thus , the T which is at position 6 in both sites contains almost 1.6 times as much ` information ' for FNR as CAP , while the T at position 9 , although of little value to CAP , i a target for the NarL protein , a possibilitywe can now test by site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>In this context , there is a core motif ( TTGAT ) for a potential FNR half-site at about -55 in the putative ttdABpromoter region ( Nesin et al. , 1987 ) , and an analogous CRP half-site motif ( GTGTA ) at - 130 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8371115" target="_blank">8371115</a>]<br></li>
<li>In agreement with this , the intergenic region between the divergent cai and ﬁx operons revealed important regulatory features , namely , potential rec-ognition sequences for CRP and H-NS , but no binding site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The intergenic region between the divergent cai and ® x operons contains potential recognition sequences for CRP and H-NS , but no binding site for FNR ( Eichler et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Statistical analysis of the sequences of FNR control sites ( Figs 6 and 7 ) led us to propose the nonamer AAA-TGAT as the consensus half-site and its near identity with the corresponding CAP recognition site ( Ebright et al. , 1987 ) is both striking , intriguing and evolutionarily significant.The half-site can be subdivided into the inner T G A T motif , which is very well conserved and has a high information content , and the outer AAA sequencethat seems to be less important ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The central motif TTGAT ( Spiro and Guest , 1990 ) of the half site of this potential FNR box is particularly well conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934931" target="_blank">7934931</a>]<br></li>
<li>Comparison of the FNR152 protected regions reveals that the FNR-activated sites consist of two symmetrical halfsites with dyad symmetry ( TTGAT and ATCAA ) : the FNR repressor site at hemA consists of a single consensus half-site ( TTGAT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>In addition , it should be noted that four other putative half sites for FNR were detected more upstream in the sequence , as was one half site for CRP at 4 bp from the Shine-Dalgarno sequence of caiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>However , Fnr binds to ndh promoter at two sites ; a high-afﬁnity site ( FNR I ) and a low-afﬁnity site ( FNR II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>]<br></li>
<li>In contrast , FNR has been shown to directly bind to two sites within the FNR-repressed ndh promoter ( 19 ) , one of which contains only 6 out of 10 bp that match the consensus site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In dcuC , the transcriptional start site is preceded by two potential FNR consensus sites and one half-site between positions 233.5 and 2115.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>However , two overlapping putative -RCB- ArcA sites ( 7 10 bases matching the consensus centred at ® 97 , AGTTgATgtA , and ® 94 , TGTaAAacAA ; non-consensus bases in lower case ) were detected within ® ± the far upstream ( 93 5 ) FNR site that is responsible for the anaerobic downregulation of yﬁD expression ( Green et al. , 1998 ) , suggesting that ArcA might prevent FNR binding at this location and thereby increase yﬁD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Potential FNR - and ArcA-binding sites were identified by comparison to the consensus sites for each ( TTGATn4ATCAA and nGTTA-ATTAn , respectively ) located within 500 bp of the translational start site identified by the Colibri website .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Eliminating those located within coding regions , and those associated with prophages , left 12 consensus FNR sites , which were all centred within 345 bp of a start codon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The extent of protection associated with the H-NS I site ( 38 bp ) suggests that more than one H-NS dimer is bound , and the formation of different subcomplexes could potentially act to modulate FNR / CRP-driven transcription activation either positively ( occupation of the downstream portion of H-NS I ; thick arrow in Fig. 6 ) or negatively ( occupation of sequences overlapping the FNR/CRP-binding site ; thin arrow in Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The hemA promoter region includes a perfect FNR box which is centred on the transcription start site ( Fig. 2 ; Neidle & Kaplan 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The transcript start ( arrow ) , putative '' 10 box ( italicized ) , FNR site ( boxed ) , and OxyR site ( underlined ) are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The italicized number above the box is the ScanAce quality score for each FNR site ; the number below is the distance from the FNR site to the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The FNR site of ndh which is negatively regulated by FNR is located at position y50 .5 w60x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The yfgF transcript start was mapped by 59 RACE-PCR and was found to be located 41.5 bp downstream of the centre of the FNR site ( Fig. 3b , c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The 39 end of the potential OxyR site is located only 8 bp upstream of the 59 end of the FNR site ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Thus , the FNR binding site is located at a proper distance for activation ( i.e. 42.5 ) , and a potential 70 Pribnow box ( CAATAT ) for this initiation point was detected as well ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>except for a region within the primary FNR site , the strongest protection being between - 6 2 and -132 ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>+10 + 1 -10 -20 '' 10 box ~ J IHF box t ~ FNR site Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9315722" target="_blank">9315722</a>]<br></li>
<li>yhjA FNR yhjA 6 1 2 3 4 5 yhjA yhjA ted yhjA ted ctiva ctiva ite ina a s site in R FNR Oxy yhjA Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>A second aim was to conﬁrm the location of the DNA site for FNR at position − 72.5 and to identify other functional determinants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>DNase I footprinting was then used to provide a biochemical assay to conﬁrm FNR binding at the proposed DNA site , and the result presented in Figure 2 shows that puriﬁed FNR creates a clear single ∼ 25 bp footprint covering the target centred at position − 72.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Although the downstream site contains 9 out of 10 bp which match the consensus FNR binding site , previous ﬁndings ( 44 , 54 ) and data presented in this study suggest that the fnr promoter is not fully saturated by endogenous levels of [ 4Fe-4S ] - FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>c Identification of the FNR site by bioinformatic methods ( pattern search , 8 of 10 match to the consensus TTGATNNNNATCAA/position weight matrix search ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>At the hlyE promoter ( PhlyE ) , this site ( TTTGATATTTATC ATA ) most closely resembles an FNR site ( 9 of 10 nucleotides match the FNR consensus , TTGATNNNNATCAA , compared to 8 of 10 matches to the CRP consensus , TGTGANNNN NNTCACA ; discriminatory bases are underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Site-directed mutagenesis of the consensus FNR site ( TTGAT-N4-ATCAA to ATCAT-N4-ATGAT ) and putative OxyR-binding site ( ATAGgcacaggCTATcttattgATAGtTtatAttcAT to TATCgcacaggGA-TActtattgTATCtAtatTttcTA ) was achieved using PCR and appropriate synthetic oligonucleotides , with all mutations conﬁrmed by DNA sequencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Upstream of the E. coli ydhY gene is a consensus FNR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Site-directed mutagenesis of the consensus FNR site ( TTGATAT-ATATCAA to ATCATATATATGAT ) was achieved using the oligo-nucleotides shown in Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>To test this , we constructed three derivatives of the pnrf53 D87 fragment ( Fig. 9 ) , carrying a consensus -10 element ( -10 6/6 ) , an improved -35 element ( -35 4/6 ) or a consensus DNA site for FNR ( FNR con ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>B. Fragments were incubated with increasing concentrations of puriﬁed FNR DA154 protein : lanes 1 -- 4 , pnrf97 fragment ; lanes 5 -- 8 , pnrf97 p42T fragment ; lanes 9 -- 12 , pnrf97 FNR con fragment , carrying a consensus DNA site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Exposure of the FNR protein or protein-DNA complex to air for even short periods of time ( -5 min ) led to the complete loss of DNA protection at a consensus FNR recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The second , the promoter of the nikABCDE operon , contains a partially conserved FNR box located immediately upstream of an inverted-repeated sequence composed of a 14-base dyad that could serve as a binding site for the NikR regulatory protein ( Navarro et al. 1993 ; V. Desjardin and M.-A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>This hypothesis is supported by the ﬁnding , in the promoter-operator region of the nik operon ( 11 ) , of an inverted-repeated sequence composed of a 14-base dyad , AA TCAGTATGACGA-N10-TCGTCATACTTATT , which may serve as a NikR binding site and of a partially conserved FNR box located just upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>The direct involvement of these sequences in FNR recognition and binding in vitro was further demonstrated by independently mutagenizing FNR-2 and FNR-1 ( see Materials and methods ) , yielding two novel DNA fragments each containing a single unaltered FNR site ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>We , therefore , carried out searches for sequences in IGRs of the E. coli genome that contain a near consensus DNA binding site for FNR followed by a potential termination signal and that are separated by 100 -- 350 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Fe2 release from 4Fe-FNR was monitored in the presence or absence of DNA containing a consensus FNR-binding site ( 5 - CGCAATAAATTTGATGTACA TCAAATTTTAGGCA-3 ; underlined bases represent the consensus site ) ( 18 ) , as the increase in A593 characteristic of the ferene-Fe2 complex ( as described above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>When oxygen is low or absent , the FNR152 protein binds to the DNA recognition site as a monomer , and upon the subsequent binding of a second FNR monomer at a neighboring DNA half-site , assembles into a stable FNR dimer ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Since the protected region was approximately 22 bp and overlapped the dyad-symmetrical FNR-binding site , FNR appears to bind to DNA as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Under anaerobic conditions , FNR152 completely protected a region of -20 bp overlapping the consensus FNR binding site at a concentration of 3.2 x 10 -6 M , assuming the FNR 152 protein bound to the DNA as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>That the protected region overlaps both halves of the FNR consensus binding site supports the hypothesis that the FNR152 protein is bound to the DNA a the predicted FNR site at the dmsA promoter differs by one base from the consensus FNR-binding site ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>FNR binding to a single site , centered at position 41.5 , is sufﬁcient for maximum induction of pnrf ( 3 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Expression from the E. coli K-12 nrf promoter ( pnrf ) is induced by the binding of FNR to a site centered at position 41.5 ( 4 , 6 , 7 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Therefore , dcuC lacks a typical FNR binding site at position 241.5 ( class II site ) ( 8 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>In most FNR-dependent promoters , FNR acts as an activator by binding to a single FNR site centered near position − 41 ( class II promoters , which overlap the − 35 sequence ) or near positions − 61 or − 71 ( class I promoters ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>These observations suggest that the TTGAT sequences are the sites of FNR action and that FNR binding at the downstream half-site ( Fig. 1 ) is the primary interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Binding site mutations were incorporated into the CRP and FNR 22 bp binding site sequences at positions 7 and 16 , and are designated X or Y in the case of CRP and FNR binding sites , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Catabolite repression of nrf promoters carrying improved -10 , -35 and FNR binding site sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Strikingly , six out of seven of these changes fall in the proposed DNA site for FNR and would be expected to reduce FNR binding at this target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>As the DNA binding site for FNR overlaps the acsP1-35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>At pnrfA , FIS represses transcription by binding to a site that overlaps the -10 element , whereas IHF binds to a site that overlaps the DNA site for the activator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Evidence for direct interaction of FNR with the proposed FNR binding site upstream of ydhY was sought using gel retardation assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>A CRP recognition sequence ( 6 ) centered at 241.5 is preceded by a half site binding motif for the anaerobic regulator FNR ( 13 ) at position 257 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Thus , we have looked for the presence of a possible FNR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>Nucleotides TGACTTTAGTCA were deleted from the FNR binding site to result in plasmid pInterD1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>A weakly predicted FNR site and a strongly predicted CRP binding site are centered at 240.5 and 289.5 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The CRP binding site partly overlapped with a previously identiﬁed FNR binding site ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Transcriptional activation by FNR and CRP : reciprocity of binding site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Sawers , G. , Kaiser , M. , Sirko , A. , and Freundlich , M. ( 1997 ) Transcriptional activation by FNR and CRP : reciprocity of binding site recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li>
<li>In this paper , we have shown that an FNR binding site overlaps the fnrtranscriptional startpoint and could thus prevent RNA polymerase from recognizing , or interacting with , the ' -12 ' and ' -10 ' regions of its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>No potential FNR or CRP binding site sequences were detected directly upstream of the dcuA promoter , which is consistent with the expression studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>FNR binds to the hmpA promoter in vitro ( 5 ) and the FNR binding site ( TTGAG -- ATCAA ) overlaps the presumed YjeB binding site ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>Inspection of the hyp promoter region failed to reveal any sequences that closely matched an FNR binding site ( TTGATNNNNATCAA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>f An FNR binding site is located in the promoter region of this gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Because this site also shows a close match to the FNR consensus sequence , matching at all cents ve positions in the second half and at three of cents ve positions in the cents rst half , band shift assays were used to study binding of both CRP and FNR proteins to a DNA fragment ( fragment EP ) carrying this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>More is known about the interaction of FNR with DNA.Comparisons between FNR-regulated promoters revealed a binding-site consensus sequence , which was later confirmed by a variety of in vivo studies 6 and in vitro footprinting ~ 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>As the pO2 decreases ( microaerophilia ) , the amount of FNR able to bind DNA increases proportionally , yielding amounts of active FNR in the cell that are sufficient to allow binding to the high-affinity FNR-2 site ( which also displays greater similarity to the FNR consensus sequence ) , but not to the FNR-1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>Furthermore , a sequence with similarity to the consensus binding site for the anaerobic fumarate and nitrate reduction response regulator ( known as FNR ) , ` TTGAT-N4-ATCAA ' [ 38 ] , is encountered centred about 105 bp upstream the start codon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>Of the 46 genes down-regulated in the presence of FNR ( patterns I , V , and VI , Figs. 4 and 5 ) , 20 contain a documented or predicted FNR-binding site at or near the transcriptional start site with less than a 2-bp mismatch to the 10-bp FNR consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>None of these genes possess a documented FNR-binding site or any site with less than a 2-bp mismatch to the 10-bp FNR consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Sequences were selected on the basis of their similarity to the core of the E. coli FNR binding site consensus ( TI'GAT -- ATCAA ) , and if there is good reason to believe that the promoter concerned might be regulated by an FNR homologue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Clearly , the FNR binding site situated in the E. coli frdA promoter bears rather poor resemblance to the consensus sequence and appears to contain a single half-site ( Figs6 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Two sequence elements that showed similarity to the FNR consensus binding site ( TTGAT-N4-ATCAA ) ( 22 ) were identiﬁed within the fnr promoter region ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>vation was reported for the frdABCD ( fumarate reductase operon ) control region , in which a mutation causing aerobic frd expression changed the FNR protein binding site into a closer match to the core consensus ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>This fragment contains a near consensus FNR binding site , with exact matches at the two core motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>We used site-specific mutagenesis to change this putative FNR protein binding site into the CRP protein binding site core consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Thus , we introduced a series of changes at the FNRl site that converted it into the core consensus FNR protein binding site ( FF construct ) or into the core consensus CRP protein binding site ( CC construct [ Fig. 4A ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>A second site with substantial similarity to the FNR protein binding site core consensus is centered at position -97.5 ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The FNRl site at position -42.5 is not identical to the core consensus sequence for FNR protein binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>A site centered at position -42.5 , which shares sequence similarity with the FNR protein binding site core consensus , was required for anaerobic induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Finally , a site centered at position -97.5 , which also shares sequence similarity with the FNR protein binding site core consensus , might play a negative role infdn operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The cure motif TGTGX of this site is conserved in the left half of the palindrome , but one difference in the right half confers the consensus core motif ( TCAAA ) for binding uf FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>The upstream potential recognition motif resembles the consensus sequence of the FNR-binding site and thus most likely is involved in the anaerobic induction of the hcp-hcr operon by FNR [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The ﬁgure shows the sequence of the core nrf promoter elements in derivatives in which the -35 element , -10 element and the FNR binding site have been mutated towards consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The predicted FNR site matches the consensus FNR binding site at 7 of the 10 conserved positions , and the predicted CRP site matches the CRP binding site consensus sequence at 9 of a possible 12 positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The CRP site matches the CRP consensus binding sequence in one half and the FNR consensus binding sequence in the other half .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>For the FNR binding site search , the pattern sequence TTJTlGAT was used , revealing five possible FNR control sites ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>Two other sites share sequence similarity with the FNR protein binding site core consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Gray rectangles represent DnaA boxes with up to one mismatch to the consensus sequence TTATNCACA , while a white rectangle represents the Fis binding site ( 4 ) , and black rectangles represent FNR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Vertical dashes shown above a consensus FNR binding sequence ( 14 ) indicate sequence identity to the proposed narK FNR-binding site ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>A sequence with homology to the proposed FNR protein-binding site ( 17 ) is indicated with lines above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The second potential DNA motif , an imperfect inverted-repeat with consensus gATGyAT - ( N5 ) - ATrCATc located downstream of the FNR site , is likely the binding site for a regulatory protein that responds to nitrogen oxides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The sequence of the FNR and PurR binding site deleted in pInterD1 and pInterD2 is shown in ( c ) E. coli BW27784 carrying pAYTOP128 was transformed with high copy number plasmid shown in the table .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>The other putative FNR protein binding site , centered at -97.5 , was not involved in anaerobic induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The DNA binding site for the FNR protein has been deduced by sequence comparison and mutational analysis of a number of FNR-dependent promoters ( for a review , see reference 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The presence of a sequence with only partial homology with the FNR protein-binding site and of a nonoptimal CRP site in the ansB 5 ' sequence do not preclude direct interaction of these proteins with this promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The requirement for a functional FNR-binding site at promoter 6 to mediate transcription from promoter 7 suggests that binding of a protein ( probably FNR ) that recognizes this sequence either promotes the formation of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>A sequence with some homology to the putative FNR proteinbinding site was found ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>A predicted FNR site is boxed in the S. enterica sequence ( bottom ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Bases fitting the consensus FNR sequence TTGATNNANATCAA [ 11 ] and the transcription start site [ 4 ] are shown in upper-case letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>The latter FNR site also overlaps with the upstream NrdR box 1 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>The ± 10 and ± 35 hexamers , the ribosome-binding site ( rbs ) , the transcription start site ( +1 ) and the FNR box are indicated on the Phmp sequence shown below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>The FNR binding sequence , TTGACTTTAGTCAA versus the TTGATN4ATCAA consensus sequence [ 18-20 ] , is located 61.5 nucleotides upstream of the upp transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>It is possible that binding of FNR at this site may not have an impact on binding of the smaller constitutive activator FhlA165 but may minimize the ability of the larger HyfR protein to bind at the target sequence , which is only about 25 bases upstream of the predicted FNR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Although our studies indicate that the predicted upstream site is not an FNR binding site , we found that this sequence binds IHF and increases expression of fnr under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>HlyX has 73 % identical amino acid sequence compared with the oxygenresponsive transcriptional regulator , FNR , which binds to a putative FNR binding site in the clyA upstream region ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Examination of this intergenic sequence showed that it includes the binding site sequences of two transcription factors , FNR and PurR ( Figure 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>For newly identified FNR targets ( plain typeface ) the sequence of the FNR binding site identified by AlignACE is given along with its genomic coordinate and , if known , the distance from the nearest transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>Sun et al. ( 44 ) identiﬁed an FNR binding sequence in the upstream region of nrdD , and Boston and Atlung ( 8 ) noted a second site ; one site is centered at position 65 relative to the transcription start site , and the second overlaps the 35 promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>In support of this notion , the results of DNase I footprinting experiments revealed that the region of DNA from bp 117 to 89 relative to the fnr transcription start site was protected in the presence of 0.5 M IHF ( Fig. 7 ) and includes the sequence of DNA that was originally predicted to be the upstream FNR binding site centered at bp 103.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>This is an unusual position for a FNR-binding site , since it has been shown that the majority of FNR-dependent promoters have a recognition sequence for FNR centered around -40 bp with respect to the transcription initiation site ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>FNR is known to stimulate transcription of target gene promoters by binding to a DNA sequence centered between 41 and 42 bp upstream of the transcription site ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>In most cases examined , the FNR protein binding sequence is located approximately 42 bp upstream of the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_216">
The consensus FNR site centred at '' 42.5 is boxed and the core inverted-repeat is highlighted in bold type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="sit_similarity_216", type="checkbox">
<div class="SIT_similar">
<li>The consensus FNR site centred at '' 41.5 is boxed and the core inverted-repeat is highlighted in bold type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li></div>
<li>The FNR152 protein exists as a monomer in solution and binds strongly to the consensus FNR sites at the narK and narG promoters and somewhat weaker to the FNR recognition site at the hemA promoter ( Figs. 2 , 3 , and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The ability of the FNR152 protein to bind at a consensus FNR recognition site and protect it from cleavage by the endonuclease Rsa I was used as the basis for developing an in vitro assay for FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The assay was based on the ability of FNR to recognize and bind a consensus FNR recognition site : this prevents access of Rsa I to its recognition site located between the left and right FNR half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>Interaction of FNR with wild-type and mutant Phmp A putative FNR box , TTGAG -- ATCAA , with a strong resemblance to the consensus FNR-binding sequence is centred at position +5.5 in the Phmp region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Inspection of the region protected by FNR reveals a sequence that corresponds to the consensus for FNR ( 16 ) , centered at position 40.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>To demonstrate that this region of the nirB promoter is indeed the target site for the FNR protein , we have transplanted the nucleotide sequence from - 3 0 to - 5 4 of the nirB promoter into gaiPI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>The regulated genes carry a FNR consensus sequence upstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>1989 ) directly upstream of promoter 6 and a sequence equivalent to a FNR half-site is present upstream of promoter 7 ( Sawers and Bock , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>However , a search of the cyo promoter region does not reveal a site similar to the FNR consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>A site similar to the ArcA consensus sequence has been identified upstream of the 3 cyoA promoter and was also shown to be subject to regulation by FNR in our previous study ( 1 ) , but this is likely indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>The smaller protein appeared not to be the IHF ( Driica and Rouviere-Yaniv , 1989 ) despite its comparable size and heat stability , and the presence of a sequence , '' '' A C A A A A C T T G A T T '' ^ ^ , overlapping the primary FNR site , which resembles the IHF consensus sequences : C/TAANNNNTTGATA/T ( Craig and Nash , 1984 ) ; and TAAAAGCTTTAAA ( Fiiutowicz and Inman , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The putative FNR control site at position 56 which has one half-perfect consensus sequence and two bases matching the other half site is in this expected location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>The consensus sequence for the half site recognized by FNR ( AAA-TTGAT ) is only slightly different frorn that of CAP ( AA-TGTGA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The deduced consensus sequence for an FNR site is shown belowthe analysis where it is compared with the current consensus sequence recognizedby CAP ( Ebrightet al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>For FNR , the consensus sequence TTGAT-NNNNATCAA was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>This site shares fhe TTGAT sequence with the potential FNR consensus around base pair -107 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>While this study was in progress , Jayaraman et al. ( 1988 ) elegantlydissectedthe E. colinirB prornoter and showed that transplantation of the inverted-repeat , located between ' -54 ' and ' -30 ' , to a similar Position upstream of the galPl promoter , resulted in concomitant FNR control.Takentogether , these collected findings unambiguously identify the site of action of the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>One heptamer , centred at 242 ( heptamer 242 ) , matches the consensus ( indicated by underlining ) at five positions ( TAACGAT ) , and is in the middle of the FNR site ( centred at 242.5 ) , and thus is located in a similar position to a NarL heptamer ( centred at 250 ) , which overlaps the FNR site in the nrfA promoter ( Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Note that these effects are speciﬁc to the nir promoter constructs as the promoter activity of the semisynthetic FNR-dependent promoter , FF ( 41.5 ) ( which carries a consensus FNR site at position 41.5 ) , was unaffected by the genetic background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The FNR-dependent dmsA promoter P1 ( referred to here as PdmsA ) has a similar architecture to PnarG in that FNR binds to and activates from a site centred at 241.5 bp ( Fig. 1A ) ( McNicholas et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The position of the Arr site ( -83 ) relative to the FNR site ( -41.5 ) of the ansB promoter suggests that Arr and FNR may co-regulate anaerobic ansB expression in a manner analogous to the co-activation of this promoter by tandemly bound FNR and CAMPreceptor protein ( CRP ) ( Scott et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Consistent with the hypothesis that YdhY -- T has a role in anaerobic metabolism , gel retardation , footprinting , sitedirected mutagenesis and transcript-mapping experiments showed that the oxygen-responsive transcription factor FNR directly activates transcription of ydhY -- T under anaerobic conditions by binding at a site located at position 242.5 relative to the transcript start , confirming and extending earlier work that shows that ydhY is expressed from a class II FNR-dependent promoter ( Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>At most targets , two FNR monomers bind to a 22 bp sequence , resulting in the activation of transcription initiation by RNA poiymerase at a site around 40 bp from the centre of the FNR-binding site ( Spiro and Guest , 1990 ; Lazazzera et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>It is worth noting that , like malEp , the E. coli aeg-46 .5 promoter , which is synergistically controlled by NarP and FNR ( a CRP-related activator ) has a FNR-binding site at a non-permiss-ive position that is separated from the RNA polymerase recognition sequence by two binding sites for a second transcription factor ( Darwin & Stewart , 1995 ; Wing et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>FNR -41.5 IHF bound to the IHF II site activates FNR-dependent transcription Previously , we demonstrated that IHF binding to the IHF I site represses FNR-dependent transcription at pnir ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>That FNR binds to this proposed FNR-binding site and activates transcription has been demonstrated in vitro ( Green ef al. , 1991 ; Sharrocks ef al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Similar conclusions about the identify of the FNR binding site have been re , md by other workers ( Spiroand Guest , 1987a ; ~ ~ ~ á r a m , etal.n 1988 ; Nees et al. , 1988 ) usingdifferent alignmentsand lesswell substantiated promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The location of the FNR binding sites at PdmsA and PnarG are typical of class II activated promoters ( Guest et al. , 1996 ; Rhodius and Busby , 1998 ) , in which the activator sites are centred at approximately 241.5 bp from the start site and overlap the 235 region of the promoter element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>FNR normally acts as a class II activator , binding to a site centred near 240.5 ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>To measure promoter activities , fragments were cloned into the lac expression vec ¬ Primers for the mutagenesis of nrf promoter sequences nrfA42T GGGCAATTGATTTAAGACAACATTCACAG nrfAfnr CCCCAATTGATTAAAATCAACATTCACAGTG nrfA35/4 GGGCAATTGATTAAAGACAACGACAACAGTGTGGTTATTTG nrfA10 GGGCACAGTGTGGTTATTTGTTATAATTAGGGGCGAGC Introduces an A to T substitution at -42 Introduces a consensus FNR site Introduces a 4/6 consensus -35 element Introduces a consensus -10 element Primers used for chromatin immunoprecipitation Sequences downstream of position -209 Sequences upstream of position +131 nrfUp CCCGAATTCCCGGGGATCACGCAAAAGTAGAATTGTGC nrfDown CCCGGATCCCTGAAGATACGGCGTGCG All primers are shown 5 cents to 3 cents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>To determine which of these is actually the promoter , we mapped the transcript start site by primer extension using the nrdDG2 primer , complementary to sequences 160 residues downstream of FNR site 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>It has been established that HlyX prefers Class I ( FNR site at ¹ 61 or beyond ) , whereas FNR prefers Class II ( FNR site close to ¹ 40 ) promoters , and that in stationary-phase hlyX-expressing cultures , yﬁD transcription initiates 61.5 bp downstream of FNR I ( Green and Baldwin , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Spacing requirements To determine whether the spacing of the tandem FNR sites is critical for FNR-mediated downregulation of class II FNR-dependent transcription activation , PyfiD variants in which the upstream FNR site was moved closer to , or farther from , the fixed class II site at 240.5 were constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>The FNR-dependent regulation of yfgF expression under anaerobic conditions thus isolates YfgF activity from that of other E. coli phosphodiesterases , with the possible exception of the EAL domain protein YjcC , which has a plausible FNR site ( TaGAannnnATCAA ; matches to the consensus in upper -- case type ) centred 80 bp upstream of the yjcC start codon , and directly links cyclic di-GMP metabolism to the oxygen-sensing FNR family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>Molecular Microbiology ( 2000 ) 37 ( 5 ) , 1258 ± 1269 S ppression o FNR-de e dent transcription at the cherichia coli nir promoter by Fis , IH H-NS dulation of transcription initiation b ucleo-protein ssem y u f p n activation F a d Es n : mo y a complex n a bl Summary promoters , FNR binds as a dimer to a 22 bp sequence centred near position 241 , upstream of the transcript start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The FNR protein induces expression anaerobically , and evidence indicates that it probably binds to a specific recognition sequence located around position -40 bp relative to the transcription initiation site ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Transcription from the nrf promoter was totally dependent upon the FNR protein which , at other promoters , binds to an inverted-repeat sequence centred about 41 bases upstream of the transcription initiation site ( Jayaraman et ai , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934939" target="_blank">7934939</a>]<br></li>
<li>Under anaerobic conditions , FNR binds to a site centred between positions 241 and 242 , activating transcription of the nir operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>At most target promoters , FNR binds to a site centered near position 41.5 and activates transcription by interacting directly with the RNAP and subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>At pnir , FNR binds to a DNA site centered at position 41.5 and activates transcription directly by interacting with RNA polymerase ( 25 ) ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>This expression depends on transcription activation by FNR ( fumarate and nitrate reduction regulator ) , which binds to a target centred at position − 72.5 upstream of the transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>FNR binds to a single DNA target centred at position -41.5 ( i.e. between base pairs -41 and -42 upstream from the transcription start ) and promoter activity is completely dependent on FNR bound at this site ( Tyson et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>At most target promoters , FNR binds as a dimer to a DNA site centred near position -41 , upstream of the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Galimand et al. ( 1991 ) have drawn attention to a number of other FNR boxes upstream of P. aeruginosa genes , including one around the transcription start site of the plcS gene encoding phospholipase C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Although neither the pdhR nor lpd promoters have been reported to be strongly controlled by FNR using lacZ fusions ( 31 , 32 ) , the results reported here are supported by the observation that the pdh promoter has a previously reported FNR site at bp position 49.5 ( 31 ) , and a previously unidentified putative FNR site ( 2-mismatch ) located upstream of the internal lpd promoter ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>( A ) Assay mixtures contained 2 nM plasmid DNA containing bp 155 to 25 of the fnr promoter region relative to the transcription start site , 50 nM E 70 RNA polymerase , and , where indicated , 200 nM or 400 nM [ 4Fe-4S ] - FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>At the proposed downstream FNR binding site , a clear pattern of protection was observed from bp 14 to 11 relative to the fnr transcription start site by [ 4Fe-4S ] - FNR ( 200 to 400 nM ) ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Given that the downstream FNR binding site is centered at bp 0.5 relative to the transcription start site , these results may suggest that [ 4Fe-4S ] - FNR represses Pfnr by blocking RNA polymerase from binding to Pfnr through steric hindrance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>A major clyA transcript having its 5 * end ( 11 ) located 72 bp upstream of the translational start codon and 61 bp downstream of the CRP-FNR binding site was detected in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Footprinting studies have shown that FNR and CRP activate hlyE expression from the same site centered 61.5 bp upstream of the SlyA-associated transcription start site ( 10 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The predicted upstream FNR binding site ( TTAAG-N4-TTCAA ) is centered at bp 103.5 relative to the transcription start site , whereas the predicted downstream FNR binding site ( TTGAC-N4-ATCAA ; underlined nucleotide match the consensus ) is centered at bp 0.5 and overlaps the transcription start site ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>The typical location of the FNR binding site is at a position centered ca. 41.5 bp upstream of the transcription start site ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Anaerobic expression of the promoter for narGHJI , PnarG , requires FNR binding to a site centred 41.5 bp ( 241.5 ) upstream of the transcription start point ( Walker and DeMoss , 1991 ) ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Summary Previous work with semi-synthetic promoters containing a single CRP binding site centred at 41.5 bp from the transcription start site has demonstrated enhanced transcription ( synergism ) when a second binding site , for CRP or FNR , is placed upstream at around -91 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>This event would , conceivably , position the upstream NarL-binding site and bound NarL protein close to the FNR and RNA polymerase proteins to provide the necessary conditions for narK transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>The ansB promoter in Escherichia coil is co-activated in a co-dependent manner by one dimer each of CRP and FNR protein whose binding sites are at around -91 and -41 bp , respectively , from the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Fig. 1 Organisation of the Escherichia coli nir promoter showing the relative locations of the transcription start ( arrow labelled +1 ) , the -- 10 hexamer , the binding site for the FNR protein centred between bases 41 and 42 upstream of the transcription start ( labelled -- 41.5 ) , the overlapping binding sites for the NarL and NarP proteins centred as an inverted-repeat at -- 69.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Activation requires the binding of NarL/NarP upstream of the FNR-binding site , whilst repression results from binding to sites flanking the FNR site ( at pnrf ) or the transcription start region ( in other cases , see Stewart , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>This site is centered at approximately -115 in relation to the start of transcription of promoter 6 and is located between two putative FNR binding sites ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Thus , the dmsA promoter is activated by FNR binding to a single site centered at position 41.5 ( i.e. , between bp 41 and 42 ) upstream of the transcript start and is repressed by NarL binding to multiple sites ( 1 , 5 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>To analyze the interaction of FNR D154A-His6 to PfnrS , a Cy5 end-labeled PCR product covering 147 bp upstream and 47 bp downstream of the transcription start site of fnrS was generated using the primers fnrS 147 fw and fnrS 47 rv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>In the yobA promoter region the putative DNA site for FNR ( 5 - TTGATgatccTCgAc-3 ) is centered 67 bp upstream of the translation start site , but the transcription initiation site has not yet been examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Similarly , mutation of the - 1 0 region or the putative FNR half-site located at - 5 0 relative to the transcription start site of promoter 7 severely reduced transcription from that promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>However , the mutational analysis of the putative FNR half-site located approximately 50 bp upstream of the transcription initiation site of promoter 7 identifies it as being necessary for promoter 7 activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Only potential FNR - and NarPbinding sites within 200 bp upstream of the transcription start site were considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Two putative FNR boxes upstream of the +1 transcription start site were also proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>RESULTS FNR binds at a site located 42.5 bp upstream of the ydhY transcript start Earlier studies have shown that ydhY expression is FNR-dependent in vivo and in vitro , and that the size of the in vitro transcription product is consistent with the presence of an FNR-dependent class II promoter ( Kang et al. 2005 ) , but the precise transcript start site has not been mapped .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Transcription from the narGHJI promoter ( narGp ) under anaerobic conditions depends on the activation of FNR ( 14 ) , which binds to sequences just upstream ( 242 bp ) from the transcription start site ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>One promoter ( PydjX ) contained an FNR site at a class I spacing ( 60.5 bp upstream of the transcription start site ) , making this the ﬁrst naturally occurring class I FNR-dependent promoter characterized by in vitro transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It is shown that the yhjA gene of E. coli possesses a class II FNR site and an OxyR site upstream of the yhjA transcript start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>When FNR acts as an activator of gene expression , it most often binds to a site ( s ) upstream of and including a site centered at an average distance of 41.5 bp before the transcriptional start site of the affected gene or operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>However , the more common architecture for FNR-activated promoters is that adopted by class II promoters , which have an FNR site centred at or close to 41.5 bp upstream of the transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Three promoters ( PynfE , PydhY , and PydfZ ) had a putative FNR site centered at 41.5 bp upstream of the transcription start site , a position typical of previously characterized FNR class II promoters ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>A well-predicted FNR site and a moderately predicted CRP site are centered at bp 103.5 ( 45.5 ) and bp 43.5 2 bp ( 2106.5 ) , respectively ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>FNR directly binds to the predicted downstream site but not the upstream site in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>PyfiDD begins at 261 and therefore lacks the upstream FNR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>In cydD-P the site ( 5 - TTGATN4AaCgc-3 , where lowercase letters are used to indicate bases that do not match the FNR site consensus ) is centered 123 bp upstream of the transcription initiation site ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>For example , FNR did not bind to the predicted upstream site , which contains 7 out of 10 bp that match the consensus site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>ScanAce revealed putative FNR sites for cfa , hdeA , slp , ompC , wrbA , yccD , yliI , and yhiD , suggesting that FNR may act directly at these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It is well established that narK and narGHJI are organized as two operons transcribed in the same orientation , but with 500 bp of intergenic DNA that includes a rho-independent narK transcription terminator , and binding sites for regulation by the oxygen-responsive transcription factor , FNR , and the nitrate -- nitrite-responsive two-component regulatory system , NarX -- NarL ( Stewart & Parales , 1988 ; Kolesnikow et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804183" target="_blank">16804183</a>]<br></li>
<li>NarP ( Darwin and Stewart , 1995 ) , FNR ( Darwin et al. , 1998 ) and ModE ( McNicholas and Gunsalus , 2002 ) binding sites are over - or underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>FNR * binding to nrdDG was assessed by a gel retardation assay , as described by Lazazzera et al. [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>Class I promoters possess FNR sites located close to 261.5 , 271.5 , 282.5 or 292.5 ( Wing et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Competition has been reported between IHF bound at IHF I and both RNAP and FNR for their overlapping sites , and between IHF bound at IHF II and NarL/P for their overlapping sites ( Browning et al. , 2002 ; 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Similar explanations apply to the six operons that we failed to confirm to be FNR-repressed ; no credible FNR sites are located in the hemA or norVW promoters ( the latter is an RpoN-dependent promoter ) , and putative sites in the ubiCA , yeiL , and tpx promoters diverge from the consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Here we report a deletion analysis of nirB promoter sequences designed to locate the sites for transcription activation by the FNR protein and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Predicted FNR and cyclic AMP receptor protein ( CRP ) sites at suitable distances from putative promoter sequences are located immediately upstream of aspA and dcuB ( 36 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>It is interesting to note that the FNR proteins of Bacillus subtilis and Bacillus lichenformis are predicted to recognize CRP sites on the basis of the amino acid sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Thus FNR could be converted such as to bind specifically to CRP sites , CRP to bind to FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_307">
FNR and NarL/NarP binding sites are represented by inverted arrows , while IHF sites are depicted by boxes , with the positions of binding sites shown in parentheses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></label>
<input id="sit_similarity_307", type="checkbox">
<div class="SIT_similar">
<li>FNR and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>FNR , CRP , and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li></div>
<li>The binding sites for FNR and NarL/NarP , centred at position -41.5 and position -69.5 , respectively , are shown by inverted arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_311">
FNR and NarL/NarP binding sites are represented by inverted arrows , and the IHF I binding site is depicted by a box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></label>
<input id="sit_similarity_311", type="checkbox">
<div class="SIT_similar">
<li>The locations of FNR and NarL / NarP binding sites are represented by inverted arrows , and the IHF binding site is shown by a box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li></div>
<li>) , putative ® 35 and ® 10 hexamers ( underlined or overlined ) ; potential binding sites for FNR , Lrp and IHF ( boxed ) ; regions of partial dyad symmetry ( converging arrows ) ; ribosome-binding sites ( RBS , consensus bases italicized ) , and translation initiators ( fMet ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The transcriptional start sites and translational initiation sites ( bold ) , the -35 and -10 hexamers of putative promoters ( boxed ) , alternative hexamers for P2 , , , A ( dashed overlined ) , ribosome binding sites ( RBS ) , and potential binding sites for FNR ( black box ) , CRP ( bold overlined ) , ArcA ( underlined ) and Fis ( dashed underlined ) , are shown for ( a ) acnA ( co-ordinates 701-1250 ; Prodromou et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Two putative Arr-binding sites were observed ( Fig. 6 ) , Arr I ( -46 to -69 ) and Arr I1 ( -94 to -120 ) , each containing related DNA sequences , -66 - AA CAAC-AACTTGAn-52 and -11 - TA c 7CAA-AACTCTGTTT-1w0h3i , h overlap the previously identified FNR boxes centred at -50.5 and -94.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>DNA sites for FNR and NarL in the ynfE regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>Inspection of the DNA sequence in the upstream region of these promoters with the ScanAce program revealed the presence of putative FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The Figure shows the base sequence from position − 125 to position + 98 with respect to the reported transcript start , and the previously proposed DNA sites for NarL and NarP , FNR and NsrR [ 9 -- 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Consistent with this , comparison of the promoter sequence in several related bacteria show that core elements , including the DNA sites for FNR and NsrR , are better conserved than the upstream ﬂanking sequences that contain the DNA site for NarL and NarP ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>It is noteworthy that the ModE-activated promoters are always associated with FNR-binding sites within a distance of less than 100 bp ( see Fig. 2 ) , suggesting a cooperative function between ModE and FNR for induction of promoters by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>To ascertain whether the FF-FF promoter was co-dependent on FNR dimers binding at both the upstream and downstream sites ( as in the case of CRP and FNR in CC-FF ) or whether the upstream dimer acted synergistically ( as in CC-CC in ansB Ps ) , the FNR binding sites in FF-FF were modified by the introduction of mutations at nucleotide positions 7 and 16 in the 22 bp binding sequence to give the YY-FF and FF-YY promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Clearly , the necessity for interaction between the c ~ subunit of RNA polymerase , DNA and upstream CRP or FNR dimers at ansB Pe could be influenced by either the nature of the downstream activator , and the ` quality ' of its activating region , and/or the particular DNA sequence between the activator binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>A computer search for FNR DNA binding sites within predicted promoter regions identiﬁed 63 new sites for 54 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>To address whether FNR directly binds to both of the predicted FNR binding sites within the fnr promoter region , in vitro DNase I footprinting experiments with [ 4Fe-4S ] - FNR were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Footprinting of FNR at the predicted downstream ( A ) and upstream ( B ) FNR binding sites within the fnr promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>C. Anaerobically in Ihe presence of FNR , transcription from the rjdh promoter is repressed because FNR binds to the - 50.5 and 94.5 sites , preventing activator binding and the upstream RNA polymer-ase contacts , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Nevertheless , there are sequences upstream of the pdh promoter which match the core consensuses of the FNR iJJGAT ' -- ATCAA ) and CRP ( TGTGA TCACA ) binding sites at 9 out of 10 and 8 out of 10 positions , respectively ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Analysis of the sequence between the treC and nrdD genes revealed two good homologies to FNR binding sites ( 20 ) located upstream of two putative 10 promoter sequences ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>The two upstream IHF sites colocalize with binding sites for other transcription factors ( NarL/P and FNR ) and with at least one promoter , acsP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Two putative binding sites for Arr , which overlap the two FNR boxes in the ndh promoter , were identified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>The ScanAce algorithm identiﬁed putative FNR binding sites with a score higher than 9 for talA as well as aceE , ybiJ , yhbW , ybiI , and ycaC ; ycaC possesses two putative FNR sites , consistent with another well-characterized member of this group , ndh .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Control DNA fragments were obtained by digesting pFF and pCC with EcoRI and HindIII to release fragments of V300 bp carrying FNR and CRP binding sites , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856643" target="_blank">10856643</a>]<br></li>
<li>To identify FNR binding sites at the 43 loci not currently listed by Ecocyc , we selected and then combined 500 bp DNA sequences corresponding to the centre of each peak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>The sequences of ATP , ADP , NAD ( H ) , FAD , cyclo AMP or cyclo GMP binding sites of other proteins show no specific or characteristic homology to FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Proposed DNA sites for FNR and NarL are aligned with the cognate consensus binding sequences ( where Y C or T , M A or C , K G or T , and R A or G ) , and the center of each site is indicated ( 6 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>The lower portion of the ﬁgure depicts the location of the binding sites for IHF , the FNR protein ( FNR1 and FNR2 ) , and DNA sequences protected by NarL-phosphate from DNase I attack ( NarL-1 through NarL-4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>The DNA sequences for the predicted FNR binding sites are indicated at the right of each panel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Using equally stringent search conditions for each factor , we screened the E. coli MG1655 genome for DNA sequences that resembled the DNA-binding sites of RutR , LexA , Fur , FruR , FNR , CRP , NarL and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>Bioinformatic analysis has been used to search the E.coli genome for DNA sequences that resemble known FNR binding sites ( 6,7 ) and DNA microarrays have been used to study differences in the transcriptome that arise when the fnr gene is deleted from the genome ( 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17164287" target="_blank">17164287</a>]<br></li>
<li>In order to test this possibility , we built matrices by sampling random subsets of binding sites for CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>Relevant positions are indicated , and the locations of FNR and NarL binding sites are shown by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>Our results show that IHF and Fis bind to multiple sites at pnir but do not interfere with FNR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Compilation of potential binding sites for FNR homologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Hypersensitive sites due to FNR and Fis binding are labelled with stars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Hypersensitive sites , induced by FNR and Fis binding , are indicated by stars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The nrdD transcript start site was mapped to a position immediately downstream of two FNR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>The small open boxes indicate the positions of the putative FNR binding sites , and the small filled box indicates the binding site for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The location of the DNA site for FNR is shown by a box and hypersensitive sites due to FNR binding are indicated with a star .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>The positions of the transcriptional initiation point ( 11 ) and putative regulatory elements , i.e. , binding sites for CRP and FNR , Shine-Dalgarno ( S.D. ) sequence , and 210 and 235 boxes are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The presence of a 210 promoter box ( TATGA AT ) centered at 29 , a 235 sequence ( TTGACG ) centered at 232.5 , and binding sites for CRP and FNR centered at 261.5 suggested that this clyA promoter is a class I promoter that could be transcriptionally activated by CRP or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Empirical score distributions distinguish global from speciﬁc TFs The global TFs CRP and FNR have several hundred annotated functional binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>The FNR binding sites are boxed in the wildtype sequence , and the − 35 and − 10 promoter regions are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>The FNR binding sites are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>The E. coli consensus is derived from a compilation of 14 known FNR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The consensus sequence for FNR binding is very similar to that for CRP , differing at just two symmetrically related positions in the inverted-repeat ; appropriate changes at these positions can result in interconversion of CRP and FNR sites ( Bell et aL , 1989 ; Zhang and Ebright , 1990 ; Jennings and Beacham , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Indeed , there is a sequence of hyphenated dyad symmetry centered at y59 .5 upstream of Pgl pG that weakly resembles the CRP or FNR consensus binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>The FNR protein binding sequence is quite similar to the core consensus for CRP protein binding , TGTGA-N6-TCACA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Furthermore , the core consensus sequences for the FNR and CRP protein binding sites are quite similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>Structures of O2 and nitrate regulated promoters From the FNR binding sites of several FNR regulated promoters a consensus of dyad symmetry ŽTTGAT-N4-ATCAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>An oligonucleotide was synthesized that contained the FNR consensus binding motif , TTGATnnnnATCAA , flanked by Sal I restriction sites with 5 ' overhanging ends ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>These positions correspond to the CG base pairs in the ATCAA and CCCGG motifs of the primary and secondary FNR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The internal hypA promoter ( PhypA ) is unusual in that it does not contain such an inverted-repeat but rather has two FNR half-sites ( TTGAT ) separated by 102 bp ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The nucleotide sequences of both strands of proposedrecognition sites in seven FNR controlledpromoterswere alignedand the frequency of occupancy for each position tabulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The positions and sequences of two FNR halfsite motifs ( F ; TTGAT ) are highlighted along with the sequences of the mutated half-sites ( TTTAT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The significance of the potential FNR sites is difficult to assess , partly because their homologies with the consensus were not particularly strong .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>The IS5 element supplies two FNR consensus sites at positions 292.5 and 2131 which could serve as the FNR regulatory sites of dcuC * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>However , regulatory sites upstream of position 253 , including two of the FNR consensus sites , are replaced by IS5 sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>A consensus sequences derived here for non-enteric FNR boxes is based on the occurrence of a particular base in at least half of the sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>At FNR sites consisting of one or two half-sites , approximately the same range of DNA is protected from DNase I attack ( about 30 bp ) , indicating that FNR is bound as a dimer ( or as a pair of monomers ) under both conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Two further sequences with slightly less homology were located closer to the fumB promoter at the end of the genF coding region , CGGGTT -- TATAAA ( 1133 to 1146 ) and TTCTIA - - AAIGCA ( 1137 to 1150 ) , and there were also two FNR half-sites , CATCAA ( 1052 to 1057 ) and TATCGT ( 1096 to 1101 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>Repression is not simply due to occlusion of the promoter ( - 3 5 and -10 ) region by FNR because adjacent pairs of FNR monomers were found to bind at two sites centred at -50.5 and -94 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Two potential promoters ( P6 and P7 in Fig. 1A ) are also present in this region as are two propose recognition sites for FNR ( termed FNR-1 and FNR-2 In Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Tet R A pRW50 derivative equivalent to pGS1000 but with two high-affinity FNR sites ( TTGATTTAAATCAA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>EMSA data clearly support the existence of two FNR sites within the acs-nrfA intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>There are credible FNR ( ttgataacgagcgg ) and ArcA ( gttcatta ) sites in the puuA -- D intergenic region , suggesting the possibility of de-repression in the presence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>FNR and CRP are ~ 50 kDa proteins that recognize related inverted-repeat sequences ( Scott et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>On the contrary , in vitro and in vivo studies with FLP have shown that it recognizes sequences conforming to a CCTGA N4 TCAGG consensus , which is related to but not congruent with either of the FNR or CRP consensus sequences ( Gostick et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Consensus sequences for FNR and CAP were from Fig. 5 and Ebright et al. ( 1987 ) , as modified by S. T. Cole ( unpublisheddata ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Recognition sequences for Fis , FNR , and IHF ( boxed ) were described previously by Jackson et al. ( 18 ) , Meng et al. ( 21 ) , and Green et al. ( 11 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513468" target="_blank">17513468</a>]<br></li>
<li>A sequence in PhlyE ( -69 TTTGATATTTATCATA-54 ) more closely resembles the FNR consensus ( NTTGATNNNNATCAAN ) than the CRP consensus ( TGTGANNNNNNTCACA ) , and in vivo transcription studies revealed that although FNR occupies PhlyE more frequently than CRP , the latter is a more efficient activator of hlyE expression ( Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>FNR is structurally related to CRP and , like CRP , binds to a 22bp sequence that accommodates two FNR subunits ( Green et al. , 1991 ; Sharrocks et aL , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>In genes which are activated by FNR , the consensus apparently replaces the canonical ' -35 ' region of the standard C ~ Nevertheless , FNR-dependent promoters seem to be recognized by RNA-polymerase containing oTM ( Sawers et al. 1989 ; Jayaraman et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The FNR protein has been shown to be necessary and responsible for the induction of expression of many anaerobic genes ( Lambden and Guest , 1976 ; Stewart , 1982 ; Newman and Cole , 1978 ; Sawers and Bock , 1988 ) while NtrA is an alternate sigma factor species ( Hirschman ef al. , 1985 ; Hunt and Magasanik , 1985 ) enabling RNA polymerase to recognize promoters showing a consensus GG/GC sequence lying at - 2 4 / - 1 2 b p with respect to the initiation site of transcription -LCB- Gussin et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181234" target="_blank">2181234</a>]<br></li>
<li>The key features of this inverted-repeat are the A -- T base pairs at positions +7 and 27 ( underlined ) , which allow discrimination between FNR sites and sites recognized by related proteins , and the G -- C base pairs at +5 and 25 ( bold ) , which form a core interaction with the DNA recognition helices of this family of transcription factors ( Schultz et al. , 1991 ; Spiro et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The -10 6/6 derivative carries a consensus -10 element , the -35 4/6 derivative matches four out of six bases of the -35 consensus element and the FNR con derivative has a consensus DNA site for FNR ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The activation of FNR similarly promotes site-specific DNA-binding , and a consensus FNR recognition motif was identified by comparing the sequences of well-characterized FNR-dependent promoters ( Eiglmeier et al. , 1989 ; Spiro and Guest , 1990 ; Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>DNA binding and gene regulation by FNR Upstream regions of FNR-regulated genes are characterized by FNR consensus sequences of dyad symmetry ( TTGAT -- -- -- -- ATCAA ) ( Bell et al. 1989 ; Eiglmeier et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_388">
The DNA sequence recognized by FNR consists of an inverted-repeat with a consensus sequence of TTGATnnnnATCAA ( Eiglmeier et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></label>
<input id="sit_similarity_388", type="checkbox">
<div class="SIT_similar">
<li>The DNA sequence recognized by FNR consists of an inverted-repeat with a consensus TTGATnnnnATCAA ( Eiglmeier et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li></div>
<li>Dimeric FNR binds at DNA sequences related to the consensus TTGATNNNNATCAA ( Eiglmeier et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>RESULTS Expression of the E. coli K-12 yfgF gene is driven from a class II FNR-dependent promoter The E. coli FNR protein recognizes and binds DNA sequences related to an established consensus ( TTGATNN-NNATCAA ) ( Eiglmeier et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>RESULTS Identiﬁcation of the yhjA gene as a member of the FNR regulon The E. coli FNR protein recognizes DNA sequences related to the FNR consensus ( TTGATnnnnATCAA ) ( Eiglmeier et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Specifically , a simple pattern search at the Colibri Web Server for sequences possessing the underlined part of the FNR consensus binding motif , 5 - TTGATN4ATCTT-3 and ending with a string of seven T residues resulted in a single candidate sRNA gene localized in the ydaN-dbpA IGR ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>We confirmed by electrophoretic mobility shift assays that FNR associates with the near FNR consensus binding motif of fnrS ( data not shown ) , and primer extension analyses of FnrS RNA showed that the 5 end most likely corresponds to bp 1.407.153 of the E. coli chromosome ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>These heptameric sequences are centered at 2109 , 245 , 11 , 119 , and 144 and are suitably positioned to interfere with the binding of FNR , CRP , and RNA polymerase and/or transcript extension .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Specific sequences to which the CRP and FNR proteins bind , to stimulate transcription , have been proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>BELL , A. I. , K. L. GASTON , J. A. COLE and S. J. W. BUSBY , 1989Cloning of binding sequences forthe Escherichia coli transcription activators , FNR and CRP : location of bases involved in discrimination between FNR and CRPN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Cloning of binding sequences for the E'slberdchia cob transcription activators , FNR and CRP : location of bascs involved in discrimination between FNR and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8828209" target="_blank">8828209</a>]<br></li>
<li>The heptamers frequently are arranged in tandem orientation with variable spacing rather than as palindromic pairs with fixed spacing which is characteristic for dimeric helix-turn-helix DNA binding proteins such as FNR or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Based on sequence homology with CRP ( 8 ) , FNR consists of a C-terminal DNA-binding domain , which recognizes specific sequences within FNR-controlled promoters ( 9 ) , and an Nterminal sensory domain containing 5 cysteine residues , 4 of which , Cys-20 , 23 , 29 , and 122 , but not Cys-16 , are essential for function ( 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>All members of the CRP-FNR superfamily recognize , or are predicted to recognize , closely related DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Strains SØ928 fnrS/pNDM220 ( control ) and obvious sequences that resemble the SØ928 fnrS/pNDM220-fnrS were grown aerobically in LB media to an A450 of 0.4 and then induced by addition FNR consensus sequence seemed to of IPTG ( final concentration 1 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>T Arg180 G 4 ~ % Va1208 e r 2 ~ T S G l y l ~ ~ 8 '' ~ T T Lys188 '' -LCB- 9 1 4 ~ '' Gly216 '' - / 9 ... Arg185 ~ '' Met189 Arg213 - `` T - ' ~ '' Arg217 T w Rgure4 Diagrams showing the amino acid side chains in the DNA-recognition helices of FNR and CRP that are thought to interact with specific base-pairs in the core motifs of the corresponding half-site consensus sequences in FNR-and CRP-bindingsites1 ~ ,19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>Boldface letters indicate putative CRP and FNR half consensus sequences in addition to presumptive recognition sequences of the s54 factor of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>The FLP consensus retains the same TGA/TCA core motif in each half-site , but differs from the FNR and CRP sequences in having four rather than six intervening nucleotides ( TGA N4 TCA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF-lacZ fusion by these two proteins ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>The putative consensus sequences for the CRP , FNR , Fur , and FhlA/HyfR proteins are enclosed by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The program Find was used to find sequences that match with -35 and -10 consensus sequences and with NarL , FNR , catabolite activator protein , and integration host factor control sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>In addition to this FNR box , two possible RpoN-specifIc promoter sequences with typical GC and GG consensus elements ( Kustu etat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934931" target="_blank">7934931</a>]<br></li>
<li>In E. coli , FNR-regulated promoters contain DNA sequences related to the consensus FNR box , which consists of an imperfect palindrome ( TTGATNNNNATCAA ) [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Consequently , the sequences of both DNA strands were aligned and used to produce a consensus FNR recognitionsite ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Matches to the FNR and OxyR consensus sequences are highlighted in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Helix 8 I N-term Helix 1 Helix 3 Helix 2 Helix 5 Helix 4 Helix 6 Helix 7 6 > 34 1 > 33 1 > 35 7 > 35 4 > 24 34 > 57 Helix 10 HYfB HYfD HYfF Hyf C HYfE FocB P 41 > 61 38 > 53 39 > 58 69 > 96 30 > 50 68 > 89 Helix 11 71 > 99 70 > 98 70 > 98 98 > 122 56 > 75 109 > 133 Helix 12 109 > 132 111 > 132 110 > 133 134 > 157 92 > 119 164 > 184 Helix 13 134 > 155 134 > 157 135 > 158 179 > 203 123 > 149 198 > 222 Helix 14 161 > 185 167 > 195 167 > 194 226 > 254 156 > 176 249 > 273 Helix 15 203 > 233 201 > 226 212 > 240 261 > 283 180 > 200 240 > 268 241 > 263 253 > 280 297 > 315 P P P P C Helix 9 Helix 16 HyfB HyfD Hyff 273 > 301 308 > 328 334 > 361 382 > 405 421 > 452 472 > 501 527 > 555 654 > 672 265 > 293 301 > 318 325 > 353 367 > 390 407 > 435 457 > 479 284 > 312 319 > 332 333 > 356 381 > 407 410 > 443 462 > 486 In a previous report , two putative 07 ' - dependent promoters and potential FNR , CRP and Fur sites were detected upstream of the hyfcoding regions ( Andrews et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>Matrix Consensus ( IUPAC ) order sites Total IC IC per FPR50 % ( LOO ) FPR90 % ( LOO ) FPR100 % ( LOO ) FPR50 % FPR90 % FPR100 % ( matrix sites ) LOO/Matrix sites ( FPR 50 % ) ( matrix sites ) ( matrix sites ) colum n 1 0 1 1 e e e e mem mem mem mem TrpR CRP CysB NanR 24 24 30 20 6 tyGtACtmGykaACtaGTaCratr aaawwtgtGayryagaTCACawww gGAavGrrrtaAyKrmwtrmcarakykm ktATAmMwGktataMmrGww 13.62 7.29 10.67 10.28 0.57 0.30 0.36 0.51 6.67 E 1.38 E 9.29 E 1.53 E + + + + 03 00 03 05 1.30 E-10 6.40 E-242 1900 8.30 E-07 1.00 E 6.20 E 2.60 E 2.30 E -08 -05 -04 -06 7.70 E-06 3.70 E-03 1.40 E-01 1.60 E-03 7.70 E-06 2.80 E-02 1.40 E-01 1.60 E-03 1.50 E 4.50 E 2.80 E 1.50 E -12 -05 -08 -11 1.50 E 2.70 E 3.20 E 6.40 E -09 -03 -05 -07 1.50 E 2.10 E 3.20 E 6.40 E -09 -02 -05 -07 198 8 6 kt 1 1 0 meme meme LexA HipB FNR 21 4 72 22 18 20 wwtrCTGtayatamammCAGya ATCCsskagmGCGGATAA tdywwwtTGaTwwmraTCaa 11.65 10.95 6.94 0.53 0.61 0.35 1.33 E 3.33 E 1.71 E + + + 01 03 0 2.60 E-46 0.00022 2.09 E-71 2.00 E 5.00 E 1.30 E -08 -08 -04 4.60 E-04 4.30 E-06 5.30 E-03 3.10 E-03 4.30 E-06 1.80 E-02 1.50 E 1.50 E 7.60 E -09 -11 -05 1.50 E 3.10 E 2.90 E -05 -09 -03 1.50 E 3.10 E 9.80 E -04 -09 -03 consensus B C Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by FNR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The relative contribution of both processes to apoFNR formation is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>( 1993 ) PKT53/63/73 nrf operon KT53/KT63/KT73 EcoH \ - Hin6 \ \ \ fragments cloned in pRW50 This work pFNR Derivative of pBR322 carrying fnr or mutant derivatives on a H/ndlll-SamHi fragment ; contains colEI origin of replication and encodes Ap '' Spiro ef al. ( 1990 ) pFNR with H/ndill-SamHI fragment replaced by M13 mp18 H/ndlll -6 amHI poiyiinker sequence Bell and Busby ( 1994 ) pF18 Promoter '' pnir7150 E. coli nir promoter carrying nucleotide sequences from -150 to +36 Bell ef al. ( 1990 ) # coli nrf promoter carrying nucleotide sequences from -209 to +131 This work Derivative of pnrf53 fragment carrying sequences from -55 to +131 This work Derivative of pnrf53 fragment carrying sequences from-209 to +18 This work Hybrid of pnir7t50 and pnrf53 fragments carrying nir sequences upstream of -- 55 and nrf sequences downstream This work ( Fig. 4 ) Hybrid of pnir7150 and pnrf53 fragments carrying nrf sequences upstream of - 55 and nir sequences downstream pnrf53 pnrf63 pnrf73 pnir-nrf pnrf-nir This work ( Fig. 4 ) a. All the promoters were on EcoRI-H/ndlll fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>TcbC does not functionally replace any of the tdc operon regulators Expression of the tdc operon is controlled directly by a number of positive transcriptional regulators , including CRP , IHF , TdcA and TdcR ( Wu et al. , 1992 ; Hagewood et al. , 1994 ) , as well as being controlled indirectly by FNR ( Chattopadhyay et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_4">
Sawers , G. ( 1993 ) Speciﬁc transcriptional requirements for positive regulation of the anaerobically inducible pﬂ operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484901" target="_blank">9484901</a>]<br></label>
<input id="tu_similarity_4", type="checkbox">
<div class="TU_similar">
<li>Speciﬁc transcriptional requirements for positive regulation of the anaerobically inducible pﬂB operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Speciﬁc transcriptional requirements for positive regulation of the anaerobically inducible pﬂ operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>]<br></li>
<li>Specific transcriptional requirements for positive regulation of the anaerobically inducible pfl operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>Specific transcriptional requirements for positive regulation of the anaerobically inducible pfE operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_9">
J Bacteriol 175:7938 -- 7944 Sawers G ( 1993 ) Specific transcriptional requirements for positive regulation of the anaerobically inducible pfl operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="tu_similarity_9", type="checkbox">
<div class="TU_similar">
<li>Sawers , G. ( 1993 ) Specific transcriptional requirements for positive regulation of the anaerobically inducible pfi operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>Sawers , G. ( 1993 ) Specific transcriptional requirements for positive regulation of the anaerobically inducible pfl operon by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li></div>
<li>Expression of the operon is also induced anaerobically , although this regulation does not rely on direct control by either FNR or ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Expression of fumB is also induced anaerobically ( 1.5 - or 5-fold ) in a manner which is dependent on both FNR and ArcA ( 41 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Wild-type fnr and fnrGA85 were then introduced on pFNR and expression from the pnrf - / acZ fusion was measured during growth in different conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_15">
J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></label>
<input id="tu_similarity_15", type="checkbox">
<div class="TU_similar">
<li>FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li></div>
<li>J Bacteriol 174 : 3474-3478-Sharrocks , A.D , , Green , J , , and Guest , J.R , ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_18">
J eacter/o/176 : 4985-4992 , Sharrocks , A.D. , Green , J , , and Guest , J.R , ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></label>
<input id="tu_similarity_18", type="checkbox">
<div class="TU_similar">
<li>Sharrocks , A.D. , Green , J. , and Guest , J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Sharrocks , A. , Green , J. , and Guest , J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022271" target="_blank">8022271</a>]<br></li>
<li>Sharrocks , A. , Green , J. , and Guest , J. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Sharrocks , A.D. , Green , J. and Guest , J.R. ( 1991 ) FNR activates and represses transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li></div>
<li>Green , J. , and Guest , J.R. ( 1994 ) Reguiation of transcription at the ndh promoter of Escherichia coii by FNR and novel factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>As concluded previously by Gruer & Guest ( 1994 ) , the effect of FNR is not simply due to its known ability to activate arcA transcription ( Compan & Touati , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>It is well established that narK and narGHJI are organized as two operons transcribed in the same orientation , but with 500 bp of intergenic DNA that includes a rho-independent narK transcription terminator , and binding sites for regulation by the oxygen-responsive transcription factor , FNR , and the nitrate -- nitrite-responsive two-component regulatory system , NarX -- NarL ( Stewart & Parales , 1988 ; Kolesnikow et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804183" target="_blank">16804183</a>]<br></li>
<li>It is also apparent that this extra regulatory process is not affected by truncating the nd/7 promoter ( Fig , 5 ) , Further evidence supporting the view that the secondary FNR site ( -94.5 ) plays an important role in regulating the ndh promoter came from in vitro transcription studies in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Conversely , although the arcA and adhE promoters include almost consensus FNR sites located appropriately to activate transcription , regulation by FNR is masked by other more significant factors that also regulate transcription positively or negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Whereas the effect on Fur and FNR caused transcription alterations in the genes under their control , that of SoxR was unexpected ; although a large increase upon the transcription of soxS occurred , a corresponding increase on the SoxRS regulon could not be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>It has also been shown that FNR induces an 82-92 ° bend in target DNA ( Ziegelhoffer and Kiley , 1995 ) but , as with CRP , factor-medi-ated DNA bending may not be essential tbr transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Expression of feoB , encoding the iron ( II ) transporter , has previously been shown to be mediated by FNR , as an FNR strain was unable support feoAB transcription ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Accordingly , caiF transcription appears to be positively controlled by FNR ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>These results indicate that repression by nitrate is independent of the three transcription factors ArcA , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Expression from this promoter is totaiiy dependent on the transcription factor , FNR , which binds between positions -52 and -30 upstream of the transcription startsite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Induction of nrdDG transcription is dependent on the FNR system but is independent of the ArcAB two-compo-nent system ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Nitrate-independent transcription of moeA-lacZ was repressed by the FNR protein , which also protected moe operator DNA from DNase I hydrolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The unbound FNR protein in the cell can bind to lower affinity targets including moe DNA and repress its transcription when the demand for Mo-cofactor and Moe proteins is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Furthermore , upon the addition of FNR , transcription of the narX promoter was significantly repressed ( Fig. 4A and B ) , 250 nM FNR exhibiting approximately 90 % repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>In addition to the specific repression by the FNR protein , the significant decrease in transcription of the fnr and narX genes was also noticed even in the absence of FNR by truncation of respective upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>The current microarray data confirmed 32 of these 43 assignments , but alone did not confirm FNR activation of five operons ( adhE , glpTQ , cydDC , hlyE and arcA ) or FNR repression of six operons ( hemA , narXL , tpx , yeiL , norVW and ubiCA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>As evidenced by the lysis of erythrocytes in agar ( Table 3 ) , the absence of CRP , but not of FNR , reduced the level of clyA derepression in anaerobically grown hns strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>It was anticipated that any disruption by NO of FNR might reduce its inducing activity on FF ( ± 71.5 ) : : lacZ and derepress the FFgalD4 : : lacZ promoter fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Activationof the E. coil ansB promoter,CC-FF , and its derivative FF-FF , by mutant FNR or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Expressionof the E. coil ansB promoter,CC-FF , in the presenceand absence of CRP and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>Expression in fnr + strain Expression in fnr m utant + Nitrate + Nitrite + Nitrate + Nitrite Example Regulatory subgroup 150 175 0.4 2.5 0.03 60.4 7.8 76 29.9 38.5 8.8 47.8 4.7 3.8 1.7 20.4 51.4 15.5 42.9 0.3 87.5 5.2 hmpA ytfE Repressed by FNR ; induced by both nitrate and nitrite 129 0.2 1.6 0.03 14.6 9.5 Activated by FNR ; induced by both nitrate and nitrite nirB hcp nrfA yeaR-yoaG yibIH Activated by FNR ; repressed by NarL but activated by NarP Induced by nitrate ; strongly induced by nitrite in an fnr mutant In summary , we have exploited at least five critical factors that limited previous attempts to define the FNR regulon of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Operons in this group include frdA and dmsA , both of which are known to be subject to hierarchical control by FNR and NarL in response to oxygen and nitrate ( 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The YtfE protein was recently implicated in the NO stress response [ 14 ] : in parallel with hmpA , ytfE transcription is strongly induced by nitrate , repressed by FNR and optimally expressed in an fnr mutant during growth in the presence of nitrite ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Conversely , transcription of both the nap and nrf operons is totally dependent on FNR , repressed by nitrate but induced by nitrite , effectively providing protection against NO generated in the anaerobic gastrointestinal tract where nitrate is a scarce but valuable electron acceptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Only five operons were induced by nitrate , NarXL and NarP , two of which were also activated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Transcription initiation at the nrfA promoter is activated by FNR in the absence of oxygen and induced further by NarL and NarP in response to low concentrations of nitrate or to nitrite ( 35 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Transcription from the hmp and ytfE promoters appeared to be repressed by FNR but strongly induced by both nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Appropriately , both are anaerobically induced by FNR and repressed by NarL in response to nitrate ( 14 , 17 ) , thus ensuring that oxygen and nitrate are utilized in preference to fumarate as terminal electron acceptors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Transcription of the four operons encoding nitrate and nitrite reductases in enteric bacteria is totally dependent upon FNR , the regulator of fumarate and nitrate reduction ( Table 3 ; Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Expression of two other operons encoding proteins of unknown function , YeaR-YoaG and YibIH , is induced strongly by nitrite only in the absence of FNR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Transcription of a further group of genes , including yeaR , yoaG , yibH , and yibI encoding proteins of unknown function , is partially FNR-dependent and strongly induced by nitrate but also strongly induced by nitrite in an FNR mutant , suggesting that multiple regulatory circuits regulate the response of these genes to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Furthermore , despite being repressed by FNR and ArcA ( 22 ) , fumA did not meet the criteria set for responsive transcripts at any of the time points .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>It is now apparent from studies with the corresponding l a c 2 fusions that the fumA promoter resembles PacnBin being anaerobically repressed by ArcA , whereas the fumC promoter resembles P2acnAin its dual regulation by ArcA and FNR , although there is evidence that fumC transcription is partially facilitated by readthrough from the upstream fumA gene promoter ( Woods & Guest , 1987 ; Guest & Russell , 1992 ; Gruer & Guest , 1994 ; Park & Gunsalus , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Transcription of the glp operons is subject to multiple controls , including catabolite repression mediated by cyclic AMP-CRP and respiratory control mediated by the FNR and ArcA/ArcB systems ( 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>This showed that the acnB gene is subject to repression by ArcA , Fis and FruR and activation by CRP , whereas the a c n A gene is repressed by ArcA and FNR , activated by FruR , CRP , Fur and SoxRS , and initiated by a38in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Both genes are subject to ArcA-mediated anaerobicrepression and CRP-mediated catabolite repression , but they differ insofar as a c n A gene transcription is activated by Fur , SoxRS and FruR , can be initiated by t ~ a ~ d ~ is r , epressed by FNR , whereas the acnB gene is repressed by Fis and FruR but unaffected by the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Expressionfrom the suc promoter was repressed by IHF and partially activated by d8but it was not regulated by ArcA , FNR , CRP , FruR or Fis , and not repressed by glucose or anaerobiosis , indicating that the well-established catabolite and anaerobic repression of ODH synthesis is imposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Incontrast , the / pel promoter was repressed by both glucose ( via a CRP-independentmechanism ) and anaerobiosis ( mediated by ArcA ) , and activated by Fis , but it was not regulated by FNR , FruR , IHF or These observations support the view that transcription of the sucABCD genes is primarily initiated and regulated at the upstream sdh promoter , and that the Ipd promoter is independently co-regulated with Pdh ( primarily by ArcAmediated repression ) rather than with PsucDirect evidence for co-transcription of the entire sdhCDAksucABCD region from PHNw t as obtained by detecting a 10 kb transcript inm c and me mutants , but not inthe parental strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>As at the nrf promoter , the nir promoter can be activated by FNR alone but FNR-dependent activation is repressed by IHF and Fis ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Table 2 , parts B and D , includes the microarray data for a few additional operons or promoters for which there is independent evidence in the literature that an operon or promoter is activated or repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Another example is the promoter of the nrfA operon that is activated by FNR and repressed by Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>These data clearly indicate that the hcp promoter is activated by FNR and repressed by NsrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Fourteen operons , including fnr itself , were previously considered to be repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>All of the seven operons that were both previously reported and confirmed in the current experiments to be repressed by FNR were induced by nitrate in the fnr mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>A comprehensive literature search revealed that there is previous independent biochemical or genetic evidence that 29 operons are activated and 14 operons are repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_69">
A total of 44 operons not previously known to be included in the FNR regulon were activated by FNR and a further 28 operons appeared to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></label>
<input id="tu_similarity_69", type="checkbox">
<div class="TU_similar">
<li>Forty four operons not previously known to be included in the FNR regulon were activated by FNR , and a further 28 operons appeared to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li></div>
<li>Similar explanations apply to the six operons that we failed to confirm to be FNR-repressed ; no credible FNR sites are located in the hemA or norVW promoters ( the latter is an RpoN-dependent promoter ) , and putative sites in the ubiCA , yeiL , and tpx promoters diverge from the consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_72">
Changes in expression levels of operons known from a range of genetic , biochemical and in vitro data to be dependent on FNR activation were not detected in these previous studies , and the nirB promoter in which there is a consensus binding site for FNR activation even appeared to be repressed by FNR [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></label>
<input id="tu_similarity_72", type="checkbox">
<div class="TU_similar">
<li>Consequently , changes in expression levels of operons known from a range of genetic , biochemical , and in vitro data to be dependent upon FNR activation were not detected in the previous studies , and the nirB promoter in which there is a consensus binding site for FNR activation even appeared to be repressed by FNR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li></div>
<li>Two of these genes , yfiA and yggV , are members of predicted operons , which have putative FNR sites ( 2 mismatch ) in their upstream regions ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Using the ScanAce algorithm , we also found putative FNR sites in the regions upstream of other operons of unknown function ( yjiM and yfeX ) , suggesting that these genes may also be directly regulated by FNR ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>For example , most of the genes in group III are repressed by the O2-sensing system mediated by ArcA ( 49 , 70 ) , and although not directly demonstrated , the data in this paper suggest that these genes can also be repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Fig. 3 illustrates one possible scenario by which indirect regulation might occur , in this case why nitrate activates but FNR represses transcription of cyoA ; similar regulatory circuits explain why Kang et al. ( 2 ) included many ArcA-regulated operons in their lists of genes activated or repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Although the transcription factor MarA is not directly regulated by FNR or ArcA , four genes that are activated by MarA ( acnA , fumC , sodA , zwf ) are repressed by ArcA and/or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Consistent with the inactivation of FNR by NO ( 34 ) , it was generally observed that FNR-repressed genes ( ndh , hmpA , gpmA , and lpdA ) were induced , whereas FNR-activated genes were repressed ( narG , pyrD , yjiH , aroP , and rmuC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>All these genes are activated by FNR bound to the anaero-box ( es ) located near the promoters [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Promoters of genes which are repressed by FNR such as ndhHorfnr are organized in a different way ( compare Fig. 2 for fnr ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Nine of these genes were also found to be repressed in anaerobiosis due to regulation by FNR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Repression by FNR has not been as well studied as activation and , as a consequence , not as many genes were known to be repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_85">
Williams SM , Wing HJ & Busby SJW ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></label>
<input id="tu_similarity_85", type="checkbox">
<div class="TU_similar">
<li>Williams , S.M. , Wing , H.J. and Busby , S.J.W. ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Williams , S.M. , Wing , H.J. , and Busby , S.J.W. ( 1998 ) Repression of transcription initiation by Escherichia coli FNR protein : repression by FNR can be simple .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_88">
Takahashi , K. , Hattori , T. , Nakianishi , T. , Nohno , T. , Fujita , N. , Ishihama , A. , and Taniguchi , S. ( 1994 ) Repressionof in vitro transcription of the Escherichia coli fnr and narX genes by FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></label>
<input id="tu_similarity_88", type="checkbox">
<div class="TU_similar">
<li>Takahashi , K. , Hattori , T. , Nakanishi , T. , Nohno , T. , Fujita , N , Ishihama , A. , and Taniguchi , S. ( 1994 ) Repression of in vitro transcription of the Escherichia coil fnr and narX genes by FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li></div>
<li>Identiﬁcation and Characterization of a Two-Component Sensor-Kinase and Response-Regulator System ( DcuS-DcuR ) Controlling Gene Expression in Response to C4-Dicarboxylates in Escherichia coli PAUL GOLBY ,1 SUZANNE DAVIES ,2 DAVID J. KELLY ,2 JOHN R. GUEST ,2 AND SIMON C. ANDREWS1 * The School of Animal and Microbial Sciences , University of Reading , Whiteknights , Reading RG6 6AJ ,1 and The Krebs Institute for Biomolecular Research , Department of Molecular Biology and Biotechnology , University of Shefﬁeld , Western Bank , Shefﬁeld S10 2TN ,2 United Kingdom Received 30 September 1998/Accepted 8 December 1998 The dcuB gene of Escherichia coli encodes an anaerobic C4-dicarboxylate transporter that is induced anaerobically by FNR , activated by the cyclic AMP receptor protein , and repressed in the presence of nitrate by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
<li>Summary The ndhgene that encodes the non-proton-translocating NADH dehydrogenaseII of Escherichia coli is anaerobically repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Microbiology ( 2008 ) , 154 , 608 -- 618 DOI 10.1099 / mic .0.2007 / 012146-0 Characterization of the Escherichia coli K-12 ydhYVWXUT operon : regulation by FNR , NarL NarP and Jonathan D. Partridge ,1 Douglas F. Browning ,2 Meng Xu ,2 Louise J. Newnham ,1 Colin Scott ,1 Ruth E. Roberts ,1 Robert K. Poole1 and Jeffrey Green1 Correspondence Jeffrey Green 1 Department of Molecular Biology and Biotechnology , The University of Sheffield , Sheffield S10 2TN , UK jeff.green@sheffield.ac.uk 2 The School of Biosciences , The University of Birmingham , Birmingham B15 2TT , UK In Escherichia coli K-12 the expression of many genes is controlled by the oxygen-responsive transcription factor FNR and the nitrate - and nitrite-responsive two-component systems NarXL and NarPQ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Activation of yeaR-yoaG operon transcription by nitrate-responsive regulator NarL is independent of oxygenresponsive regulator FNR in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Integration of three signals at the Escherichia coli nrf ole f Summary Expression from the Escherichia coli nrf operon promoter is activated by the anaerobically triggered transcription factor , FNR , and by the nitrate/nitrite ioncontrolled response regulators , NarL or NarP , but is repressed by the IHF and Fis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Localization of Upstream Sequence Elements Required for Nitrate and Anaerobic Induction of fdn ( Formate Dehydrogenase-N ) Operon Expression in Escherichia coli K-12 Two transcriptional activators , the FNR and NARL proteins , are required for induction of the fdnGHI operon , encoding Escherichia coli formate dehydrogenase-N .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>174 , No. 11 Anaerobic Induction of Pyruvate Formate-Lyase Gene E Is Mediated by the ArcA and FNR Proteins xpression GARY SAWERS * AND BERNHARD SUPPMANN Lehrstuhlftir Mikrobiologie der Universitat Munchen , Maria-Ward-Strasse la , 8000 Munich 19 , Gernany Received 16 December 1991/Accepted 26 March 1992 The pyruvate formate-lyase ( pfl ) gene of Escherichia coli is transcribed from seven promoters which are coordinately induced 12 - to 15-fold by anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>Anaerobic induction of each of these promoters requires FNR and ArcA ( 35 ) , and anaerobic expression is further increased by growth in media containing pyruvate ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_98">
Anaerobic induction of pyruvate for-mate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12486057" target="_blank">12486057</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665524" target="_blank">7665524</a>]<br></label>
<input id="tu_similarity_98", type="checkbox">
<div class="TU_similar">
<li>Anaerobic induction of pyruvate for-mate-lyase gene expression is mediated by ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Anaerobic induction of pyruvate-for-mate lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7773398" target="_blank">7773398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Sawers , G. , and Suppmann , B. ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022271" target="_blank">8022271</a>]<br></li>
<li>Savtfers , G. , and Suppmann , B. ( 1992 ) Anaerobic induction of pyruvate formate-lyase gene expression is mediated by the ArcA and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li></div>
<li>See M methods section for details oter DNA in aterials and FNR binding to moe operator/promoter DNA Although FNR behaves as a repressor of moeA-lacZ expression in vivo , an FNR-binding consensus sequence ( TTGAT-N4-ATCAA ; Spiro and Guest 1990 ) is not detectable in the DNA upstream of the moe operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>This unusual use of FNR and OxyR to activate expression suggests that yhjA plays a speciﬁc role in countering oxidative stress in E. coli , possibly in particular micro-aerobic environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>FNR is a transcription regulator that controls gene expression in response to oxygen limitation in E. coli ( Spiro and Guest , 1990 ; Unden et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>MIcrobtot Rev 56:195 -228 , Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene expression in Eschertchia coti FEMS Microbiol Rev 75 : 399-428 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934931" target="_blank">7934931</a>]<br></li>
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coil FEMS Microbiol Rev 75 : 399-428 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_109">
Spiro , S. , and Guest , J. ( 1990 ) FNR and its role on oxygenregulated gene expression in Escherichia coll .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></label>
<input id="tu_similarity_109", type="checkbox">
<div class="TU_similar">
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022286" target="_blank">8022286</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_111">
S. , and Guest , JR. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></label>
<input id="tu_similarity_111", type="checkbox">
<div class="TU_similar">
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022271" target="_blank">8022271</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23506073" target="_blank">23506073</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022270" target="_blank">8022270</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564515" target="_blank">10564515</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>[ 7 ] Spiro , S. and Guest , J.R. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Spiro , G. , and Guest , J. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li></div>
<li>J.R. ( 1990 ) FNR and its role in oxygenregulated gene expression in Escherichia coli FEMS Microbiol Rev 75 : 399-428 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_116">
FNR and its role in oxygenregulated gene expression in Eschenchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></label>
<input id="tu_similarity_116", type="checkbox">
<div class="TU_similar">
<li>FNR and its role in oxygenregulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8132465" target="_blank">8132465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071201" target="_blank">8071201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071219" target="_blank">8071219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961411" target="_blank">7961411</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1744060" target="_blank">1744060</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>]<br></li>
<li>FNR and its role in oxygen-related gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_119">
2:701 -707 -- ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></label>
<input id="tu_similarity_119", type="checkbox">
<div class="TU_similar">
<li>FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665461" target="_blank">7665461</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665524" target="_blank">7665524</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576043" target="_blank">8576043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209047" target="_blank">9209047</a>]<br></li></div>
<li>Spiro , S. , and Guest , J. ( 1990 ) FNR and its role in oxygen-reg-ulated gene expression in Escherichia coti .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_122">
Spiro , S. and Guest , J.R. ( 1990 ) FNR and its role in oxygen-regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></label>
<input id="tu_similarity_122", type="checkbox">
<div class="TU_similar">
<li>Spiro , S. , and Guest , J. ( 1990 ) FNR and its role in oxygen regulated gene expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Spiro , S. , and Guest , J.R. ( 1990 ) FNR and its role in oxygen regulated gene expression in Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li></div>
<li>Mol Gen Genet 228:125 -128 Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>J Gen Microbiol 133:3279 -- 3288 Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_127">
Mol Microbiol 6:3452 -- 3454 Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></label>
<input id="tu_similarity_127", type="checkbox">
<div class="TU_similar">
<li>Spiro S , Guest JR : FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Spiro S , Guest JR ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824175" target="_blank">8824175</a>]<br></li>
<li>7 , 5 4 Spiro , S. and Guest , J. R. ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Spiro , S. & Guest , J.R. ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>Spiro , S. , and Guest , J. R. ( 1990 ) FNR and its role in oxygen-regulated gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10794600" target="_blank">10794600</a>]<br></li></div>
<li>Upstream of the E. coli ydhY gene is a consensus FNR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>FNR is a dimeric , class II transcription factor , which activates gene expression when E. coli grows anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>TTGAT ... ATCAAg -- tures which might allow HlyX and BTR to activate the expression of an E. coli gene which is not activated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Here both FNR and OxyR are shown to regulate expression of the E. coli yhjA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The Escherichia coli yhjA gene , encoding a predicted cytochrome-c peroxidase , is regulated by FNR and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>Oxygen regulated gene expression in Escherichia coil : Control of anaerobic respiration by the FNR protein Gottfried Unden * & Martin Trageser Institut far Biochemie , Gebtiude 26.42 , Heinrich-Heine-Universitiit , Universitiitsstr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_139">
Oxygen regulated gene expression in Escherichia coli : control of anaerobic respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576043" target="_blank">8576043</a>]<br></label>
<input id="tu_similarity_139", type="checkbox">
<div class="TU_similar">
<li>M. ( 1991 ) Oxygen regulated gene expression in Escherichia coli : Control of anaerobic respiration by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li></div>
<li>J.B. ( 1989 ) FNR-dependent repression of the ndh gene of Esdierichia coti and mefal ion requirement for FNR regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_142">
and Guest , J.R. ( 1989 ) FNR-depen-dent repression of the ndh gene of Escherichia coli and metal ion requirement for FNR regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></label>
<input id="tu_similarity_142", type="checkbox">
<div class="TU_similar">
<li>JR. ( 1989 ) FNR-dependent repression of the ndh gene of Escherichia coli and metal ion requirement for FNR regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>FNR-dependent repression of the ndh gene of Escherichia coli and metal ion requirement for FNR regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li></div>
<li>S.J.W. , Quincey , R.V. , andCole , J.A. ( 1987 ) Locationand sequence of the promoterof the gene for the NADH-dependentnitrite reductaseof Escher-ichia coli and its regulation by oxygen , the FNR protein and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>2 ( 4 ) : 527-530 Jennings MP & Beacham IR ( 1990 ) Analysis of the E. coli gene encoding L-asparaginase II , ansB , and its regulation by cyclic AMP receptor and FNR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Notably , in GFP-based target fusion ( Fig. 6D ) led us to propose that the independent work by Durand and Stortz ( 55 ) , 9 of the latter anaerobic repression of sodA expression by FNR is mediated transcripts were also found to be down-regulated after FnrS post-transcriptionally by FnrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>A yeiL -- lacZ transcriptional fusion showed that aerobic yeiL expression increases at least sixfold during stationary phase , requires RpoS , and is positively autoregulated by YeiL , positively activated by Lrp ( and IHF in the absence of FNR ) and negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>This observation indicates that the synthesis of the fnr transcript is autogenously repressed by its own gene product FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>This is due to transcriptionfrom the FNR - and u7 ' - depend-ent promoterPAlocated within the hypA gene ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>In addition to the above members of the FNR regulatory network , another gene , nuoE , has been described as negatively regulated by FNR ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The same E. coli haemolysin gene has recently also been shown to be positively regulated by HlyX , an Actinobacillus pleuropneumoniae homologue of the E. coli regulatory protein FNR that is not related to SlyA ( Green and Baldwin , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027972" target="_blank">10027972</a>]<br></li>
<li>Experimental Design -- The experiments described here ( Fig. 1 ) were performed at the same time as our previously reported experiments profiling gene expression levels in the presence or absence of oxygen and FNR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>The single gene of this cluster previously known to be regulated by oxygen and not affected by FNR is formate dehydro-genase-O , the product of the fdoH gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Guest , J.R. , Green , J. , hvine , A.S. and Spiro , S. ( 1996 ) The FNR modulon and FNR-regu-lated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_156">
Guest , J. , Green , J. , Irvine , A. , and Spiro , S. ( 1996 ) The FNR modulon and FNR-regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></label>
<input id="tu_similarity_156", type="checkbox">
<div class="TU_similar">
<li>Guest , J.R. , Green , J. , Irvine , A.S. , and Spiro , S. ( 1996 ) The FNR modulon and FNR-regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>[ 5 ] Guest , J.R. , Green , J. , Irvine , A.S. and Spiro , S. ( 1996 ) The FNR modulon and FNR-regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li></div>
<li>The FNR modulon and FNR regulates gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_160">
The FNR modulon and the FNR-regulated gene expression , p. 343 -- 359 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601216" target="_blank">10601216</a>]<br></label>
<input id="tu_similarity_160", type="checkbox">
<div class="TU_similar">
<li>The FNR modulon and FNR-regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The FNR modulon and FNR regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The FNR modulon and FNR-regulated gene expression , p. 317 -- 337 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li>
<li>The FNR modulon and FNR-regulated gene expression , p. 317 -- 342 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9765574" target="_blank">9765574</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10482502" target="_blank">10482502</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li></div>
<li>The responses obtained with the fusion indicated that ArcA functions as an anaerobic repressor acnB acnB of transcription and that the gene is not regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>The pﬂB gene is transcribed from multiple promoters , and promoters 6 and 7 are , like the yﬁD promoter , positively regulated in response to anaerobiosis by both FNR and ArcA ( Sawers , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>The acnA gene is positively regulated by Crp , SoxRS , FNR and repressed by ArcA [ 59 ] , whereas acnB is positively regulated by CRP and negatively regulated by ArcA , FruR and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The fumB gene is positively regulated by FNR , ArcA , Crp and Fur and negatively by Fis and NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The ansB gene is positively regulated by both anaerobiosis , through the FNR protein ( 20 ) , and by CRP ( 8 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2407723" target="_blank">2407723</a>]<br></li>
<li>The direction of the triangles indicates whether the gene product is positively ( ) or negatively ( ƒ ) regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The nosR gene which encodes a protein required for the expression of an active nitrous oxide reductase also contains an FNR box in its promoter region ( Fig. 2 ; Cuypers et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>The novel nature of fdhF gene expression has further been emphasized by the fact that anaerobic induction is independent of the FNR protein ( Birkmann ef al. , 1987b ) and transcription from the promoter is dependent upon the presence of a functional NtrA protein ( synonyms GInF , RpoN , ( T̂ ' * ) -LCB- Birkmann etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2181234" target="_blank">2181234</a>]<br></li>
<li>The expression of ndb is mainly regulated by FNR ( anaerobic repression ) and by unknown , pulative amino acid response regulators ( Green and Guest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>FNR is an activator of anaerobic respiratory gene expression and also represses ndh gene expression ( Green and Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10486127" target="_blank">10486127</a>]<br></li>
<li>The alteredfnr gene was overexpressed in E. coli and the resultant FNR protein was purified to near homoge-neity by using anaerobic chromatography procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>The ansB gene encodes the anaerobic asparaginase I1 of E. and coli requires FNR and anaerobic growth with 17 amino acids for maximum expression ( Jennings & Beacham , 1993 ; Cedar & Schwartz , 1968 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>The expression of anaerobic respiration and of related functions of E. coli is controlled by the positive gene regulator FNR , which activates transcription in the absence of 02 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The direct oxygen sensor FNR activates ` anaerobic ' gene expression in the absence of oxygen [ 31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>It is possible that the same mechanism is responsiblefor the stimulatory effects of amino acid mixtures on expression of the asparaginase I1 gene ( ansB ) , an anaerobically-activatedmember of the FNR modulon ( Cedar and Schwartz , 1968 ; Jennings and Beacham , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>When FNR acts as an activator of gene expression , it most often binds to a site ( s ) upstream of and including a site centered at an average distance of 41.5 bp before the transcriptional start site of the affected gene or operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>hmp gene expression in the presence of NOz is independent of the SoxS , OxyR , and FNR transcription factors and instead relies on inactivation of the iron-de-pendent Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>Cruz-Ramos , H. , Hughes , M.N. and Poole , R.K. ( 2000 ) Nitric oxide is an effector in bacterial gene expression mediated by FNR and FNR-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Thus , control exerted by the RpoN and FNR proteins on cai and ﬁx expression appears to be mediated mainly via the caiF gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>No rationale can be offered at present for the positive autoregulatory effect of YeiL on yeiL gene expression , nor for the aerobic repression by FNR , be it direct or indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Whether the NarP phosphate bound to DNA is unable to productively interact with FNR or RNA polymerase to induce fdnG gene expression is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>However , expression of this gene has not been reported to be down-regulated anaerobically in strains lacking FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>FNR has been reported to repress expression of the alternative NADH dehydrogenase encoded by the ndh gene ( 15 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730262" target="_blank">7730262</a>]<br></li>
<li>By binding to the FNR consensus in front of the fnr gene , FNR represses its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>We also chose a gene , yfcZ , on whose expression FNR showed a small effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>NO modulates gene expression in vivo by reacting with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Regulationof acn gene expression by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Analysis of the ogt promoter A previous transcriptome analysis had predicted that expression of the ogt gene is also activated by NarL independently of FNR [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>As a read-out , we used a ydfZ-lacZ gene fusion , the expression of which is activated by the holoform of FNR ( Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>In addition , the expression of cyoABCDE ( 18 ) , sucABCD , sdhCDAB ( 63 ) , sodA ( 31 ) , tpx ( 41 ) , fumA ( 65 ) , and icdA ( 15 ) was shown to be negatively regulated by both FNR and ArcA under anaerobi conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Consistently , anaerobically sodA as well as metE expression was known to be negatively regulated by FNR though an undetermined mechanism ( 32 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Alternatively , it is conceivable that genF and fumB are both expressed from an FNR-regulated promoter situated upstream of genF or that FNR exerts an indirect effect on fumB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>The results of the gene expression profiling experiments further show that as many as two-thirds of the genes whose expression levels are affected by the ArcA protein are also affected by the FNR protein ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Group VI : genes that are repressed by FNR but that do not show any O2-dependent changes in gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The majority of other ArcA or FNR regulated genes studied so far are either repressed anaerobically by ArcA or activated anaerobically by FNR , although three examples are known whose expression is repressed by FNR anaerobically ( 14 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>The large number of genes whose expression is regulated by changes in O2 and the dramatic reprogramming of metabolic pathways have made the study of FNR and its regulon ideal for a system level approach .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Despite the vast abundance of genes that are regulated by the FNR and ArcAB systems , it appears that they do not represent the sole means by which gene expression in E. coli is regulated in response to oxygen levels ( Sawers , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>In Azotobacter vinelandii , expression of the ﬂhDC genes ( which encode the master regulator of motility ) is negatively regulated by the oxygen-sensor CydR , an orthologue of the E. coli FNR protein ( León and Espín , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>Components of the 02-sensingsystem of FNR Conclusions In E. coli the expression of O2-regulated genes is controlled by different regulatory systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>Ninety-one genes with no presently deﬁned function were also altered in expression , including seven of the most highly anaerobically induced genes , six of which we found to be directly regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Of the remaining genes clustered into this expression pattern , two genes , mrcA ( penicillin-binding protein 1A ) and rarD ( involved in chloramphenicol resistance ) , were also observed to be regulated by FNR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Two other genes , organized in an operon encoding a putative alternative cytochrome oxidase , appCB ( cbdAB ) , were not observed previously to be regulated by FNR ( 1 ) , and xylR ( regulatory gene for the xylose operon ) also clustered into this expression pattern .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>The likely explanation for this is that most of the genes controlled by these promoters encode products that play a role in anaerobic respiration of nitrate and nitrite , and thus their expression is fully induced only in both the absence of oxygen ( signalled via FNR ) and the presence of nitrate or nitrite ( signalled by NarL ) [ 4,29,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Expression Pattern II : Increased Expression during Anaerobiosis and Decreased Expression in an FNR Strain -- The transcription of the 58 genes of expression pattern II ( Table II ) are both induced in the absence of oxygen and positively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The aerobic requirement for FNR is surprising because the iron ^ sulfur-con-taining holo-FNR protein activates the expression of an-aerobic genes in response to oxygen-starvation , but does not bind at these target promoters when oxygen is available ( reviewed in [ 8 ^ 11 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The FNR * mutation D154A increases aerobic expression from the narGHJI promoter , probably by ` freezing ' FNR in an active conformation ( Kiley and Reznikoff , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>The expression of fnr , ndhII , cyo and possibly cyd is negatively controlled by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>The requirements for YeiL , RpoS and Lrp for optimal yeiL expression were retained in the absence of FNR ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The expression of dcuC * 9-9lacZ was increased by a factor of about 2.2 compared to that in the wild type , but the responses to oxygen , nitrate , and the regulators FNR , NarL , and NarP were comparable to those in the wild type ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>The in vivo analysis of a yfcZ-lacZ fusion strain revealed that the expression of yfcZ was not as strongly regulated by FNR , in agreement with the microarray data ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Because hmp gene transcription is repressed by the O2-responsive regulator FNR , we investigated whether FNR also senses NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>However , our data showing that NO does not regulate # rd suggest that the mechanism of repression exerted by the transcription factor FNR on the # rd operon is clearly di ¡ erent from that on the hmp gene , which is proposed to be upregulated by NO through FNR inhibition [ 17,23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>FNR is structurally and functionally hornologous t o CAP , the catabolite gene activator protein , and increased FNR production strongly stirnulates transcription of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Previous studies using an fdhF : : lacZ gene fusion confirm that its transcription is activated anaerobically in the presence of FNR ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Expression of the caiF gene is dependent on the same general regulatory proteins , H-NS , CRP , FNR , and transcription factor RpoN ( 11 ) , which control cai and ﬁx expression ( Table 2 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>For example , H-NS is known to interfere with ss and Lrp in transcription of the osmC gene ( 8 ) and with FNR and CAP in the regulation of cai and ﬁx operons ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Like cai and ﬁx operons , the caiF gene appeared to be positively controlled by CRP and FNR and negatively controlled by H-NS and the NarL ( plus nitrate ) regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Indeed , the homology between FNR and CRP is such that their DNA-binding specificities can be interconverted by appropriate substitutions in the recognition helices , such that catabolite repression can be lifted in response to anoxia and anaerobic gene activation can be induced by glucose starvation ( cAMP ) ] 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_223">
The dcuB gene is strongly induced anaerobically by FNR , repressed in the presence of nitrate by NarL , and is subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="tu_similarity_223", type="checkbox">
<div class="TU_similar">
<li>The dcuB gene is strongly activated anaerobically by FNR , repressed in the presence of nitrate by NarL , and subject to cyclic AMP receptor protein ( CRP ) - mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li></div>
<li>The results show that the dcuB gene is expressed exclusively under anaerobic conditions in a manner that is FNR dependent and that it is repressed by NarL in the presence of nitrate and is subject to CRP-mediated catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>The dcuB gene is expressed exclusively under anaerobic conditions in a manner that is largely FNR dependent , is repressed by nitrate through a mechanism that is mostly NarL mediated , and is strongly induced by C4-dicar-boxylates anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>While our data indicate that [ 4Fe-4S ] - FNR represses its own synthesis by binding to th predicted downstream FNR binding site within the fnr promoter region , the mechanism by which [ 4Fe-4S ] - FNR prevents RNA polymerase from transcribing the fnr gene is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>It is perhaps surprising that FNR boxes are not apparent in the promoter region of the nitrous oxide reductase structural gene , although perfect half sites can be found ( Viebrock & Zumft 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Melville and Gunsalus ( 1996 ) reported that the hemA gene , which is also repressed by FNR , contains only one half-site of FNR consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The moe operon is similar in this regard to the hemA gene , which also contains only a single FNR-pentamer consensus sequence ( Melville and Gunsalus 1996 ) to which the FNR protein was shown to bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>E. coli Fd is also reduced by FNR [ 7 ] , although kinetic data have not been reported for this reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12234497" target="_blank">12234497</a>]<br></li>
<li>The upstream promoter sequences of the E. coli nrdAB and nrdEF genes were also probed for FNR : DNA interactions but no retarded bands were detected ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>Transcriptional monocopy lacZ fusion studies showed that both operons are coexpressed during anaerobic growth in the presence of carnitine , respond to common environmental stimuli ( like glucose and nitrate ) , and are modulated positively by the same general regulators , CRP and FNR , and negatively by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Experimental procedures FNR strain construction RZ8459 was lysogenized with lPC25 ( containing the dmsA0 ± lacZ operon fusion ; Cotter and Gunsalus , 1989 ) to create the reporter strain PK3293 ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Jennings , M.P.and Beacham , I.R. : Ana!ysisof the # scherichiacoligene encodingL-asparaginaseII , ansB , and its regulationby cyclicAMP receptor and FNR protein.J .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2144836" target="_blank">2144836</a>]<br></li>
<li>Hence , the E. coli class III nrdDG operon appears to be under positive regulation by FNR and negative regulation by NrdR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Thus the nrdDG operon is transcribed from a single promoter that has two FNR FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Transcription activation ofthenitratereductase structural gene operon ( narGHJI ) requires the FNR protein for induction by anaerobiosis and the NarL proteinforinduction by nitrate ( STEWART1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Common mechanismsfor regulatingfdnGHZ and narGHJZ expression : Previous work has shown that narGHJI operon expression requires FNR for induction by anaerobiosis , and NarL for induction by nitrate ( STEWART1982 ; LI and DEMOS 1988 ; reviewed by STEWART 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Expression of a caiF-lacZ operon fusion was subject to FNR regulator-mediated anaerobic induction and cyclic AMP receptor protein activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Anaerobic regulation of frdABCD operon is mediated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>Anaerobic induction of the hyb operon was only partially dependent on the FNR protein and , surprisingly , was enhanced by an arcA mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>The yfgF transcript start was mapped by 59 RACE-PCR and was found to be located 41.5 bp downstream of the centre of the FNR site ( Fig. 3b , c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>The upstream potential recognition motif resembles the consensus sequence of the FNR-binding site and thus most likely is involved in the anaerobic induction of the hcp-hcr operon by FNR [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The results underscore the importance of FNR for induction of the hcp-hcr operon promoter and corroborate the deletion analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>The two operon candidate transcripts , cydDC and yobA-yebZY , were previously considered to be repressed by FNR ( 32 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>The activationof the E. coil ansB promoterderivative , FF-CC , by FNR mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8748035" target="_blank">8748035</a>]<br></li>
<li>The increased labeling was speciﬁc for FNR ; the ﬂuo-rescence of other labeled bands did not increase in the same way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>The modified form was not affected by p-mercaptoethanol or dithiothreitol , so it is not FNR ( 27 , per se .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8497198" target="_blank">8497198</a>]<br></li>
<li>The FNR protein and its known relatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Furthermore , the fnr mutation caused a 27-fold increase in the # rd / # rd-red transcription ( Table 2 , Fig. 1C ) , indicating that the transcription of # rd / # rd-red is negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>Three exceptions were the ferric enterobactin synthesis and transport operon , fes-entF-fepE , ompW encoding an outer membrane protein and colicin S4 receptor , and cadC encoding the transcriptional activator of the cad operon ( but why was the cad operon itself apparently not FNR regulated ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Of the unknown genes that are highly regulated by FNR , ynfE is part of an operon that encodes a putative DMSO reductase ( 54 ) , ydhY is predicted to be an Fe-S-containing oxidoreductase , ydfZ is postulated to be involved in selenium delivery ( 44 ) , and ydjX is a putative membrane protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Expression of both enzymes is dependent on FNR and is further regulated by nitrite and nitrate levels ( Page et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>These data ( not shown , but available online ) failed to reveal other operons that might be regulated directly by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The combined data , summarized in Fig. 2 , have established that at least 103 operons are regulated by FNR , 68 of which are activated by FNR and 35 are repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The FNR site of ndh which is negatively regulated by FNR is located at position y50 .5 w60x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Furthermore , the yceJI operon encodes a cytochrome b561 homolog that was reported to be negatively regulated by FNR ( 32 -- 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>What is significant is that this operon , like pfl , is also regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>The frdABCD operon is also positively regulated by FNR and negatively by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>Recent lacZ transcriptional fusion experiments in our laboratory confirm these results and further demonstrate that these genes are regulated by ArcA but , 3 as reported here , not by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>It is noteworthy that the aspartase , asparaginase II , and fumarase B genes resemble the fumarate reductase operon in being positively regulated by FNR ( 12 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>We found that 297 genes contained within 184 operons were regulated by FNR and/or by O2 levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Our ﬁnding suggest that as many as 465 genes of MG1655 are regulated by changes in environmental O2 and/or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Three of these four genes , yajG , yceD , and yfiA , were also observed to be regulated by FNR in anaerobiosis ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>A common feature of this subset of FNR-regulated genes is that they appear to be repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The cyd and cyo operons encoding the quinol oxidases Cyo and Cyd which operate under aerobic or microaerobic conditions are subject to O2 regulation by ArcA and FNR ( Tseng et al. , 1996 ; Fu et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>If the moe operator DNA has the full FNR consensus sequence in the operator region , the moe operon would be effectively repressed under conditions of the highest demand for Mocofactor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>In fact , the rplM operon is under the control of FNR , which is involved in the metabolic switch from aerobic to anaerobic respiration ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>e. defective in aerobic respiration sion of the succinate dehydrogenase ( encoded by the cyo operon ) as the ter - control ) s , and ( 2 ) FNR , with pleiotropic promoter is gradually lifted above the minal oxidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>It is suggested FixK in R. meliloti , and the azurin genes that the cyd operon is activated by ( azu ) of Alcaligenes faecalis and A. deni - ArcA under microaerobic conditions trificans23 , which encode the blue-cop - and repressed by FNR under anaerobic per proteins thought to be involved in conditions , so that this dual control nitrite respiration and therefore likely allows cytochrome d oxidase synthesis to be induced anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>TorC autoregulation is different from tor anaerobic control The torCAD operon is fully induced in anaerobiosis but , in contrast to all other known anaerobic respiratory systems , this control is mediated neither by FNR , a pleiotropic anae-robic regulator , nor by ArcA , a response regulator involved in redox control of numerous genes ( Simon et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411745" target="_blank">10411745</a>]<br></li>
<li>In the anaerobic condition , the β-gal activities of the fnr - strain carrying pMW22206 were almost same as those of KM1007 carrying pMW22206 ( data not shown ) , indicating that the fragment 6 did not contain any DNA sequence related to the regulation by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17618229" target="_blank">17618229</a>]<br></li>
<li>The FNR protein should not be active in well aerated cultures , so data for the fnr mutant during aerobic growth should correlate well with those of the parental strain ; this was confirmed ( Table 2 , part B , 2nd column , aerobic data for the fnr mutant ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>iHowever , under aerobic conditions both promoters were completely inactivated by multicopy fnr * ( data not shown ) , suggesting that FNR may indeed repress the pdh promoter in the absence of competing signais .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>The results of studies with transfonnants of MClOOO -LCB- fnr * ) and JRG1728 -LCB- fnr ~ ) containing the truncated ( pGS602 ) and intact ( pGS603 ) ndh-lacZ fusion plasmids grown under aerobic and anaerobic conditions , showed that even in multiple copies , the intact ndh promo-ter is anaerobically repressed by FNR , whereas the truncated ndh promoter is not ( Fig. 5 ) , Furthermore , the tfuncated promoter is only 75 % as active as the intact promoter under aerobic conditions in the fnr * strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>In E. coli , the expressions of the genes encoding the enzymes catalyzing the fermentation and respiration are under the control of some global regulators such as FNR and ArcA/B system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15781419" target="_blank">15781419</a>]<br></li>
<li>Expression of the genes for all of these proteins is dependent upon FNR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Expression of the nrdDG genes occurs under oxygen-limiting conditions , during anaerobiosis , and , to a lesser extent , in the deceleration phase of aerobically growing cultures and is completely dependent on the global transcription regulator FNR , but not on the ArcA-ArcB two-component system ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17496099" target="_blank">17496099</a>]<br></li>
<li>Expression of yhjA was found to be dependent on this unusual combination of FNR and OxyR under conditions of oxygen starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Therefore , we grew PK872 cells at 37 ° C under aerobic conditions in a minimal glucose medium and induced overexpression of FNR with IPTG for 1 h , after which the cells were immediately processed for purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14730984" target="_blank">14730984</a>]<br></li>
<li>Thus , these data suggest that under such conditions , FNR and CRP are in competition for binding at PhlyE , and that the simplest explanation of these observations is that FNR is more efﬁciently bound than CRP at PhlyE but is not as effective as CRP in activating hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>When iron becomes scarce , Fur becomes inactive , resulting in derepression of sodA transcription ; however , only under conditions in which FNR and ArcA are inactive ( i.e. in presence of O2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Regulation of hcp transcription by FNR , NsrR , and sources of nitric oxidea alactosidase activity under the indicated growth conditions Aerobic with NONO Anaerobic - G Background FNR ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Glutathione and the Formation of Active FNR Formation of Active FNR : FNR protein is present in E. coli in roughly the same concentrations under aerobic and anaerobic growth conditions ( Spiro and Guest , 1987 ; Unden and Duchêne , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>The mod operon is transcribed under both oxic and anoxic conditions and does no appear to be subject to control by FNR protein or Arc proteins , global regulators of anaerobic respiratory pathways ( Lynch and Lin 1996a ; Rosentel et al. 1995 ; Spiro and Guest 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Such is the case for complex II in E. coli , with the complex II bio-synthetic operon ( sdhCDAB ) being negatively regulated by ArcAB and FNR in response to oxygen conditions ( i.e. anaerobic ) and positively regulated by CRP in the absence of glucose [ 108 -- 110 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22985599" target="_blank">22985599</a>]<br></li>
<li>Expression of yﬁD in E. coli is known to be controlled by the transcription regulator FNR ( 7 , 8 ) and in response to pH-stress conditions ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11444864" target="_blank">11444864</a>]<br></li>
<li>FNR is known to induce the transcription of the E. coli nrfABCDEFG operon during anaerobic cell growth [ 22 ] , and to negatively regulate the transcription of E. coli hmp [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>FNR , a transcription factor known to positively regulate a flavohemoglobin homolog in the bacterium Vitreoscilla as well as many denitrification enzymes in other species , actually represses anaerobic expression of the E. coli hmp ( 14 , 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>Restriction enzymesand T4 DNA ligase were from New England Biolabs , Inc. ( Beverly , Massachusetts ) FNR ( anaerobic ) FIGURE1.-Model for regulation offdnCHI and narGHJI transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>Repression of narX transcription by FNR Analysis of the effect of FNR on transcriptional activation and repression was extended to the regulatory region for two other anaerobic operons , narX and narK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Synthesis of formate dehydrogenase-N and nitrate reductase is induced by nitrateduringanaerobic growth.Figure1 illustrates ourcurrent model for regulation of narGHJI transcription by anaerobiosis and nitrate.Anaerobic inductionis mediated by FNR , Genetics 125 : 691-702 ( August , 1990 an activator of anaerobic respiratory genes ( LAMBDEN and GUEST1976 ; NEWMANand COLE1978 ; reviewed by STEWART1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2168848" target="_blank">2168848</a>]<br></li>
<li>This induction is dependent on the regulatory protein , FNR , a factor essential for the transcription of a number of genes expressed during anaerobic growth ( Cole and Ward , 1973 ; Lambden and Guest , 1976 ; Griffiths and Cole , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Transcript-profiling experiments have shown that the abundance of the ydhY and downstream transcripts is enhanced under anaerobic conditions in the presence of FNR ( Constantinidou et al. , 2006 ; Kang et al. , 2005 ) , and that the transcripts are less abundant in the presence of nitrate ( Constantinidou et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>It has been suggested that the anaerobic induction of the ttdAB genes is independent of the O2-sensitive regulator FNR ( fumarate nitrate reductase regulator ) ( Oshima & Biville , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>The ynfEFGHI operon is induced under anaerobic conditions in a fumarate nitrate reduction regulator ( FNR ) - dependent manner ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Under anaerobic conditions ndh is repressed by the regulator of fumarate and nitrate reduction ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Under anaerobic conditions , the expression of ndh is repressed by FNR ( regulator of fumarate and nitrate reduction ) ( 13 , 20 , 21 ) , while the expression of cyoABCDE is repressed by FNR under anaerobic growth conditions ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513468" target="_blank">17513468</a>]<br></li>
<li>Transcripts that are less abundant ( repressed ) in the reference pool relative to the test sample include those that are induced directly or indirectly during aerobic growth , transcripts that are induced during anaerobic growth by nitrate or nitrite , or repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>which is induced by the fumarate and nitrate reductase regulatory protein ( FNR ) only during anaerobic growth [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10548535" target="_blank">10548535</a>]<br></li>
<li>Measurement of aerobic activation from pFF-41 .5 in the presence of chromosomally encoded FNR suggests that FNR is on average 2.5 % active under aerobic conditions [ 150 8.6 Miller units ( MU ) under aerobic conditions and 6,177 343 MU under anaerobic conditions ; Fig. 1 , labeled cFNR ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19261852" target="_blank">19261852</a>]<br></li>
<li>Because FNR is presumed to be inactive under aerobic ( O2 ) conditions , we did not perform experiments comparing fnr genotypes under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>These pattems were essentiaiiy the same under aerobic and anaerobic conditions and they suggest that ArcA represses P/pa but not Ppat , or Pace - FNR appeared to activate Pp ^ h and repress P ^ ce under anaerobic conditions , but these obsen/ations are difficult to interpret since the effects of the fnr mutation were B Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>Supplemental Table S1 lists raw data for 502 genes for which transcript levels were significantly affected by FNR during anaerobic growth in at least one growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Thus , we hypothesize that under anaerobic conditions , negative autoregulation also prevents [ 4Fe-4S ] - FNR from exceeding a critical level beyond which it can be efﬁciently inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>FNR is required for anaerobic induction of narGHJI , fdnGHI , and frdABCD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>FNR is located in the cytoplasm and contains a [ 4Fe -- 4S ] cluster in the active ( anaerobic ) state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>The reduced uptake rates in an fnr background and under aerobic conditions suggest that the synthesis is under ( direct or indirect ) transcriptional control of FNR , the transcriptional regulator of anaerobic respiratoxy genes ( reviewed in references 25 and 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></li>
<li>Under the same conditions FNR represses some of the genes of aerobic respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1854188" target="_blank">1854188</a>]<br></li>
<li>ldhA is one of the shortest fermentation genes ( 991 nucleotides ) , it is not under control of FNR , and its product is present both under aerobic and anaerobic conditions ( Linch and Lin , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16187334" target="_blank">16187334</a>]<br></li>
<li>The transcription of hlyE in the hns strain under anaerobic conditions in the presence of glucose is presumably driven by FNR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Furthermore , the anaerobic transcription of fnrS was abolished in a fnr strain , thus demonstrating that the global regulatory protein FNR induces FnrS transcription in response to anoxic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Under aerobic growth conditions , proteolysis decreases FNR protein levels ( 12 , 38 ) , while under anaerobic conditions , FNR represses its own transcription ( 24 , 41 , 44 , 47 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Although much progress has been achieved in understanding what limits FNR function to anaerobic growth conditions ( Kiley and Beinert , 1999 ) , the mechanism by which FNR regulates transcription initiation of its target genes is not as well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>FNR activates transcription of many genes in response to anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>FNR activates transcription of a number of genes coding for enzymes required under anaerobic growth conditions and represses transcription of some genes for aerobic functions ( 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>FNR is selectively active as a transcription factor under anaerobic growth conditions , where it has been shown to control the transcription of hundreds of genes , many of which are necessary for adaptation to O2-limiting growth conditions ( 10 , 16 , 25 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Here it is shown that the oxygen-responsive transcription factor FNR regulates PhypA under anaerobic conditions only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Yet , FNR is only functional as a transcription factor under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Under anaerobic conditions FNR activated hyp transcription by approximately seven-fold , and v29FNR had no e ¡ ect ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Moreover , transcription from the c70-dependent promoter is activated by FNR under anaerobic conditions [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>In anaerobic conditions , FNR binds to its target site , but transcription activation is repressed by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Under anaerobic conditions , FNR binds to a site centred between positions 241 and 242 , activating transcription of the nir operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Transport studies suggest that the Dcu systems are expressed exclusively under an-aerobic conditions , activated by the anaerobic activator protein FNR , and repressed in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Transcription from the hemA promoter is modestly repressed by the FNR protein during anaerobic cell growth conditions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577745" target="_blank">8577745</a>]<br></li>
<li>FNR protein levels were also compared for cells grown under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15547274" target="_blank">15547274</a>]<br></li>
<li>For the FNR overexpression experiment , strains were grown under aerobic or anaerobic conditions by sparging ( Sutton and Kiley , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>( A ) Holo-FNR ± DNA binding and repression of hmp under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>DcuS is constitutively expressed but induced 2-fold under anaerobic conditions by FNR ( 10 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22101843" target="_blank">22101843</a>]<br></li>
<li>Under anaerobic growth conditions , transcript levels of 39 genes , representing 35 operons ( group II ) ( Fig. 2 ) , were increased in the strain lacking FNR relative to wild-type strain MG1655 , indicating that FNR repressed their transcription under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Expression of the 12 genes in group VII was decreased under anaerobic conditions in both wild-type and FNR mutant strains , indicating that the regulation of these genes is independent of FNR ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Expression Pattern I : Decreased Expression during Anaerobiosis and Increased Expression in an FNR Strain -- Among the 205 genes displayed in the clustering procedures described above , 36 are repressed in anaerobic conditions due to regulation by FNR ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Of the 18 genes of known function ( Table VIII ) , 4 are known to be regulated by oxygen and/or FNR under anaerobic growth conditions , and 8 contain putative FNR-binding sites ( Fig. 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>A large variety of FNR regulated genes are known which are transcriptionally activated or repressed under anaerobic conditions ŽTable 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Group V : genes that are slightly upregulated under anaerobic conditions but repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group VII : genes that are repressed under anaerobic conditions independently of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Genes required for anaerobic metabolism are regulated positively by FNR under anoxic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Previous approaches have revealed about 70 genes that are regulated by FNR or ArcAB under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Group IV : genes that are upregulated by FNR under anaerobic conditions but are only somewhat O2 regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Unexplained is why the promoter for yeaR transcription is far more active during anaerobic growth than during aerobic growth , and more active in the absence of FNR than in its presence , despite the absence of an FNR site in the regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>In the absence of both FNR and NsrR , ytfE promoter activity was signiﬁcantly higher in anaerobic cultures , but the underlying mechanism is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>The anaerobic fumarase , encoded by fumB , is known to be activated during anaerobic fermentative growth ( 50 , 51 ) , and Tseng ( 51 ) showed that both ArcA and FNR are responsible for this anaerobic activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>In contrast , for the hyaA and hybO promoters , IscR repressed the previously identiﬁed O2-regulated transcription start sites ; under anaerobic growth conditions , AppY and the ArcA/B two-component regulatory system have been implicated in the activation of transcription of the hya operon ( Brondsted and Atlung , 1994 ; Richard et al. , 1999 ) , whereas FNR appears to contribute to activation of the hybO promoter ( Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Twenty-eight operons not previously known to be regulated by FNR appeared to be repressed by FNR during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Note , however , that this group also includes sodA ( 6.7-fold induction ) , which is regulated by FNR and other transcription factors , and cyoA , which is known to be repressed by ArcA during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>To determine if negative autoregulation requires the [ 4Fe-4S ] form of FNR , expression of Pfnr - lacZ ( bp 155 to 25 of Pfnr relative to the transcription start site ) was monitored in anaerobically grown strains expressing either wild-type FNR or FNR mutants ( FNR-CA23 and FNRCA122 ) that were previously shown to not contain [ 4Fe-4S ] clusters ( 29 , 32 , 34 , 37 , 50 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>It was shown previously that when expressed in anaerobically grown E. coli K-12 , the FNR homolog HlyX of A. pleuropneumoniae and , although much less effective , FNR , are able to activate ClyA expression , presumably by binding to the FNR binding site in the clyA upstream region ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Biochemically and functionally different forms of FNR and their interconversion induced by O2 as oxidizing agent and glutathione as reducing agent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Western blot Expression of tandem FNR variants was determined by western blot .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>It is therefore unlikely that malPQ is part of the FNR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>FNR are fumB , lysU , nikA , and narX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>FNR was expressed from pGS330 ; HlyX from pGS415 ; and [ þ / ¹ ] þpyﬁD as pGS1052 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Expression of ClyA as evidenced by lysisa of erythrocytes in agar ClyA expression under the following conditions Relevant E. coli strain characteristic ( s ) Aerobic Anaerobic BSN26 Wild type BSN27 hns 2 2 11 11 11 ( 1 ) 11 ( 1 ) 11 ( 1 ) 11 11 11 11 11 11 11 ( 1 ) 11 ( 1 ) 11 ( 1 ) 11 11 11 11 MWK6 hns slyA JON31 hns JON32 hns crp BEU701 hns fnr BEU705 hns crp fnr JON32/pDW300 hns crp/p ( CRP ) JON32/pLG339 hns crp BEU701/pDW300 hns fnr/p ( CRP ) BEU701/pLG339 hns fnr BEU701/pGS24 hns fnr/p ( FNR ) BEU701/pGS215 hns fnr/p ( FNR-V208R ; S212G ; G216K ) BEU701/pGS297 hns fnr/p ( FNR-V208R ; E209D ) BEU701/pBR322 hns fnr BEU705/pDW300 hns crp fnr/p ( CRP ) BEU705/pLG339 hns crp fnr BEU705/pGS24 hns crp fnr/p ( FNR ) BEU705/pGS215 hns crp fnr/p ( FNR-V208R ; S212G ; G216K ) BEU705/pGS297 hns crp fnr/p ( FNR-V208R ; E209D ) BEU705/pBR322 hns crp fnr M182/pJON22 Wild type/p ( SlyA ) M182/pACYC177 Wild type M182 crp/pJON22 crp/p ( SlyA ) M182 crp/pACYC177 crp JON31/pJON22 hns/p ( SlyA ) JON31/pACYC177 hns MWK6/pJON22 hns slyA/p ( SlyA ) MWK6/pACYC177 hns slyA 11 11 11 ( 1 ) ( 1 ) ( 1 ) 11 11 11 ( 1 ) 1 1 ( 1 ) 1 ( 1 ) ( 1 ) 11 2 ( 1 ) 2 11 11 11 11 11 2 ( 1 ) 2 11 11 11 11 FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>However , it is shown that two FNR half-site motifs located upstream of the c70-de-pendent promoter are necessary for FNR-mediated hypB-E expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>However , even in the absence of FNR , we were unable to detect a transcript in our in vitro transcription system using a template that contained the same upstream sequences , suggesting that expression of the talA promoter may require additional factors or that it is a weak promoter ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Previous work has shown that the global transcription factors FNR and CRP positively regulate hlyE expression by binding at the same site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Footprinting studies have shown that FNR and CRP activate hlyE expression from the same site centered 61.5 bp upstream of the SlyA-associated transcription start site ( 10 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>During aerobic growth , transcription from the nir and nrf promoters is repressed , but expression is induced during anaerobiosis due to activation by the anaerobically regulated transcription factor FNR ( Jayaraman et al. 1989 ; Bell et al. 1990 ; Page et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>This pattern of expression results from positive regulation of transcription from the narGHJI operon by three transacting factors , FNR , NARL , and integration host factor ( IHF ) ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>It is clear that E. coii contains a family of anaerobically regulated promoters where expression is dependent on at least two transcription factors , FNR and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The regulation of yhjA expression is interesting , because transcription is dependent on two transcription factors , FNR and OxyR , that respond to two seemingly incompatible signals , i.e. oxygen starvation and peroxide stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>In a number of cases , activation by FNR is modulated by further transcription factors which establish priorities between the expression of different anaerobically expressed operons ( reviewed by luchi and Un , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>It has also been suggested that the oxygenresponsive transcription factor FNR activates expression of ompW ( 7 ) , although direct evidence for FNR interaction with the ompW promoter is lacking .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>Anaerobic expression of yfgF is dependent on the transcription factor FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>In vitro transcription from promoter indicated that FNR alone can dow yfiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>Titration of the FNR and PurR transcription factors by the high copy number clone is expected to increase the expression level of genes normally suppressed by these two regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Thus , while it seems likely that FNR does not directly effect transcription of these genes , the decrease in expression in fnr strains remains unexplained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The FNR protein induces expression anaerobically , and evidence indicates that it probably binds to a specific recognition sequence located around position -40 bp relative to the transcription initiation site ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The FNR box is therefore in a position which is not indicative of anaerobic activation of transcription by an FNR-like protein and understanding of its role must await further study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747934" target="_blank">7747934</a>]<br></li>
<li>Furthermore , the anaerobic expression of FUMB is reduced infnr mutants and amplified by a multicopy fnr plasmid ( 37 ) , which is consistent with FUMB being an anaerobic enzyme requiring the anaerobic transcription factor ( FNR ; 23 , 24 ) for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>The anaerobic transcription factor FNR represses ndh expression by binding at two sites centred at '' 94 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>The transcription factor FNR is known to repress ndh expression under anaerobic conditions by binding at two sites ( FNR I and FNR II ) in the promoter region ( Sharrocks et al. , 1991 ; Green & Guest , 1994 ; Meng et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>It would further appear that the presence of iron is not essentiai for site-specific DNA bindin Introduction FNR is a transcriptional regulator that controls the expression of target genes in response to anaerobiosis in such a Received 30 September , 1993 : revised and accepted 1 February , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>For most genes , changed expression in fnr -- background and presence of FNR consensus sites have been shown Supposed O2-sensor or redox-sensor proteins Table 1 O2 or redox sensor-regulator systems in Proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>FNR-dependent promoters often possess weak 235 sites and good 210 sites , and in such cases the FNR site is compensatory in allowing FNR-dependent expression of otherwise weakly transcribed genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Because of the high statistical significance of the genes of these groups that do not exhibit upstream FNR-binding sites , we can be confident that these genes are members of the FNR regulatory network whose expression levels are indirectly affected by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Furthermore , because the expression levels of the 57 genes of patterns III and IV are not affected by the presence or absence of FNR , they are not expected to possess binding sites for this regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>To determine whether FNR directly effected the expression of talA , encoding transaldolase A of the nonoxidative branch of the pentose phosphate pathway , we constructed a talA - lacZ fusion and measured its activity in both the presence and the absence o fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>acid residues to the inhibitory effect of FNR on yﬁD expression , a library of randomly mutated fnr genes in pBR322 was created by error-prone PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Other plasmids encoding FNR variants were pBR322 derivatives isolated after screening of a mutant fnr library ( generated by error-prone PCR ; Bell and Busby , 1994 ) for enhanced yﬁD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Thus , CRP enhances hlyE expression in response to glucose starvation ( 36 ) and FNR enhances hlyE expression in response to oxygen starvation ( 10 , 11 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In summary , the data presented here suggest that FNR and CRP are positive regulators of hlyE expression in liquid culture in response to oxygen and glucose starvation , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Darwin and Stewart ( 52 ) used a narX-lacZ operon fusion to demonstrate no change in expression with respect to oxygen but a slight repression by FNR in anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In the case of the cydAB operon , it has been postulated that dual ArcA-FNR control optimizes expression when the extracellular oxygen tension decreases below a threshold in which the cytochrome o oxidase no longer can be fully functional owing to its high Km for oxygen ( 1 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>These data suggested that hlyE expression is activated by FNR and CRP , repressed by H-NS , and that H-NS-mediated transcriptional silencing is relieved by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Both cai and ® x operons require the presence of FNR and CRP for their expression , while they are partially repressed by H-NS ( Eichler et al. , 1995 ; Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Further studies with acn-lac2 gene fusions and a wider range of transcription regulators indicated that acnA expression is initiated by G ~ ~ from PIacnAa , nd from PZacnA it is activated directly or indirectly by CRP , FruR , Fur and SoxRS , and repressed by ArcA and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>Transcription of ndh is repressed by FNR [ 38 ] and gene expression profiling suggests that it is activated by ArcA [ 39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>It has also been reported that poxB expression is repressed during anaerobiosis in a manner that is not mediated by ArcAB or FNR ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In addition to regulation by ArcA and FNR , ndh expression is repressed by the pyruvate-responsive TF PdhR ; repression is relieved by pyruvate [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Also regulation of the expression of puuA was not fully reported except that the regulation of puuA occurring in response to O2 was mediated by ArcA and FNR ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18495664" target="_blank">18495664</a>]<br></li>
<li>Gene expression suggests that ArcA is active in aerobic ECOM4LA , while FNR is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>It seems that the expression level of ndh was independent of the ArcA and FNR regulators as the expression levels were similar in all strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140031" target="_blank">16140031</a>]<br></li>
<li>In agreement with this observation , increasing the level of FNR protein in the cell by introducing a plasmid containing a fnr + gene ( pGS24-1 ) into strain AH59 effectively reduced the basal level of expression of moe operon by about 8.5-fold to 250 units of β-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>When both arcA and fnr were mutated , the basal level of moeA-lacZ expression was increased by the removal of the repressor , FNR protein , although the enhancing effect of ArcA-P was absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Moreover , in the absence of Fis and FNR , deletion of the Fis and FNR sites did not signiﬁcantly alter ndh expression , showing that , as expected , deletion of the upstream Fis-binding sites has no effect on ndh expression in a ﬁs background ( Fig. 2e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>It is unlikely that Fis acts as an antirepressor by preventing FNR from fulﬁlling its role as a repressor of ndh expression , because altering the Fis II site lowers ndh expression even in an fnr mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>This oould be caused by repression of nuo by FNR or , indirectly , through the effects of FNR on arcA expression ( Compan and Touati , 1994 ) , Mutations in bimO did not significantly affect expression of nuo277 under both condilions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>These results suggest that the FNR protein acts as a negative regulator of moe operon in controlling the basal level of expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Although FNR represses expression of the moe operon , only a halfsite resembling a FNR-binding site ( TTCAT ; Fig. 4 ) is detected in the moe operator region protected by the FNR * - DA154 protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>We also dem-onstrate the role of CRP in the direct activation of the ulaAB-CDEF operon and show that ula regulon expression was not affected in an FNR mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Nevertheless , the in vivo expression data support the view that FNR represses expression of this operon and also suggest that there is an internal promoter within the operon controlling at least the expression of sufD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>This would also account for the indirect involvement of FNR in controlling operon expression , with the synthesis of the metabolite being under FNR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>This is consistent with FNR being essential for expression from the hcp-hcr operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Furthermore , FNR represses the expression of genes that are needed for aerobic metabolism ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>FNR also represses the expression of genes that are needed for aerobic metabolism , such as the respiratory enzymes NADH dehydrogenase II and cytochrome oxidases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Thus , further analysis is necessary to determine whether FNR directly regulates the expression of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Further analysis is necessary to determine whether FNR directly regulates the expression of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>When the expression of plasmid-en-coded fnr was not induced , neither the ﬂuorescent band ( Fig. 1A ) nor the band responding to anti-FNR in the immunoblot was produced ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055593" target="_blank">18055593</a>]<br></li>
<li>Nitrate-stimulated NarP activated expression of 14 operons ; all of those most strongly activated by NarP were part of the FNR regulon and had been characterized independ-ently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>It is possible that FNR exerts its effect on expression of both operons through altering the levels of intracellular nickel or a regulatory metab-olite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>FNR is also required to mediate narK activation in response to nitrate as evidenced by the degree of narK-lacZ expression in aftnr strain ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>These observations indicate that occupation of FNR II by FNR , but not by HlyX , downregulates yﬁD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Regulation of expression of the ulaG and ulaA divergent promoters by CRP , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>NO derepresses hmp expression in vivo by reacting with FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>At that , NO was found to inactivate FNR anaerobically , restoring the hmp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>FNR expression was induced by adding a final concentration of 100 µM IPTG at OD600 of 0.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>FNR and NarL expression is decreased significantly in the rimO mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169565" target="_blank">21169565</a>]<br></li>
<li>Anaerobic HlyX-driven expression from þþpyﬁD was approximately fourfold higher than that observed with equivalent FNR expressing cultures in both exponential and stationary phases ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>( d ) Anaerobic activation of ydhY expression is abolished by mutation of the consensus FNR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Anaerobic activation of yﬁD expression by FNR , HlyX and hybrid proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>Nevertheless , it can be concluded that acid stress does increase expression from the yﬁD promoter ( Table 1 ) , and that FNR is the major regulator of yﬁD expression ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Anaerobic expression from the wild-type ndh promoter in the absence of FNR was 30-fold higher than that observed in the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>In the absence of the FNR protein , the aerobic exponential expression of nrdD was slightly reduced ( Table 2 ) and the stationary-phase induction was virtually absent , indicating that the induction during the deceleration phase of growth is caused by oxygen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923108" target="_blank">12923108</a>]<br></li>
<li>Since aspA expression is reported to be induced 10 - to 15-fold anaerobically by FNR ( 16 , 44 ) , a good FNR site would be expected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In accord with previous observations , yﬁD ( wild-type promoter , þþpyﬁD ) expression was anaerobically induced 13-fold relative to the aerobic level in exponential cultures expressing FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>yﬁD expression was anaerobically induced in the wild-type strain , while in the absence of FNR , both the aerobic and anaerobic expression levels were decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>A mutation in the fnr gene , however , only reduces anaerobic induction fivefold , indicating that FNR is not the only factor involved in the anaerobic activation process ( Sawers and Bock , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>This i mediated by the anaerobic repressor , CydR , which is an FNR homologue encoded by a gene ( cydR ) located close to the cydAB genes ( Wu el al. , 1997 , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>In a mutant deficient in FNR ( encoded by the fnr gene ) , the transcriptional activator of anaerobic respiratory genes , fumarate uptake was decreased by a factor of 5.3 compared with that of the parental strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512189" target="_blank">1512189</a>]<br></li>
<li>Anaerobic regulation is mediated by the fnr gene product , FNR , which is a positive activator required for the transcription of a number of anaerobic genes ( reviewed in references 31 and 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>Expression of the aspA gene is repressed by glucose ( ; 10-fold ) under aerobic conditions and enhanced anaerobically ( ; 10-fold ) in a manner which is partially ( ; 2-fold ) FNR dependent ( 16 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Expression of the nuoE gene ( Fig. 6 ) is 3.8-fold lower under anaerobic conditions and elevated 4.5-fold in the FNR mutant ( Table IX ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Expression of the fumB gene is activated under anaerobic conditions by FNR ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>Discussion The ndh gene is an unusual member of the FNR reguion because it is repressed rather than being activated under anaerobic conditions , and KMnO4 footprinting studies have now shown that RNA polymerase can not form an open complex at the ndh promoter in the presence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>The NADH dehydrogenase II gene -LCB- ndh ) Is repressed by FNR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>Expression of the ndh gene , which encodes NADH dh II , is repressed by FNR under anaerobic conditions ( 15 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730262" target="_blank">7730262</a>]<br></li>
<li>Using reverse-transcription polymerase chain reaction we show that although the flavorubredoxin gene ( flrd ) is transcribed in both aerobic and anaerobic conditions , anaerobiosis induced transcription up to 12-fold , under fermentative conditions ; a 28-fold stimulation was observed in an E. coli fnr mutant strain , showing that the flavorubredoxin gene is negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>The FNRsystem The product of the fnr gene ( FNR ) is now known to be involved in the anaerobic activation or repression of a variety of enzymes , and it may also repress its own transcription under anaerobic conditions ( Fig. 3 ) 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The proposed interaction of FNR , the product of the fnr gene , and RNA polymer-ase ( RNP ) under conditions in which frd transcription is induced or not induced is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3298218" target="_blank">3298218</a>]<br></li>
<li>Under aerobic conditions with excess glucose , control of sdhCDAB -- b0725 -- sucABCD transcription by Arc , FNR , FruR and cAMP-CRP will play a minor role and , thus , reduced expression of this operon during glucose overflow metab-olism is likely due to the control by the cAMP-CRP independent catabolite control system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>2 To identify which genes of E. coli K-12 are differentially expressed under anaerobic conditions and to determine the role of FNR in this process , we compared changes in transcript proﬁles of an FNR mutant with its wild-type parent , MG1655 , grown under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It has been reported that FNR is able to sense oxygen directly ( 47 ) , while the ArcB/ArcA system responds to the content of the quinone pool ( 4 , 5 , 17 ) and switches on the expression of fermentation genes and represses aerobic pathways when E. coli encounters low-oxy-gen growth conditions ( 21 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20709841" target="_blank">20709841</a>]<br></li>
<li>Analysis of these data demonstrated that the expression of over one-third of the genes expressed during growth under aerobic conditions are altered when E. coli cells transition to an an-aerobic growth state , and that the expression of 712 ( 49 % ) of these genes are either directly or indirectly modulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Based on this analysis of whole genome transcription data , it was estimated that the expression of over one-third of the genes expressed during growth under aerobic conditions are altered when E. coli cells transition to an anaerobic growth state and that the expression of half of these genes is modulated either directly or indirectly by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>It was also concluded that any involvement of FNR must be indirect ( or operational only in the absence of ArcA ) , because a c n A expression was not enhanced in an fnr mutant , but it was partially repressed under aerobic conditions by fnr + amplification in an arcA + strain ( Gruer & Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9421904" target="_blank">9421904</a>]<br></li>
<li>It is conceivable that in addition to FNR the frd promoter may be recognized by another anaerobic activator and in this respect it should be noted that other workers have reported significant anaerobic frd expression in fnr mutants ( Jones and Gunsalus , 1987 ; Spiro and Guest , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>Anaerobic ndh expression is repressed by FNR but enhanced in fnr mutants during anaerobic growth in rich medium ( Spiroet al. , 1989 ; Green & Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine metabolism ) was controlled by protein regulators such as FNR ( transcriptional regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Although the array data presented in Table VII indicate that nirB expression is decreased during anaerobic growth , a lacZ transcriptional fusion showed that expression of the nirBDC operon is activated by FNR during anaerobic cell growth ( 44 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>However , a mutation in fnr did not affect anaerobic expression of an ( cydD-lacZ ) operon fusion , suggesting that FNR repression is indirect and mediated at the post-transcriptional level ( 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>In contrast , the expression of fumB resembles that of the fumarate reductase operon and some other ` anaerobic ' genes in being depressed anaerobically by a mechanism that involves FNR , the CRP-like anaerobic transcriptional activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3282546" target="_blank">3282546</a>]<br></li>
<li>Thus , moe operon expression is modulated at three levels : ( 1 ) an aerobic low level , ( 2 ) an anaerobic intermediate level controlled by ArcA-P and FNR ( supporting FHL and DMSO reductase ) , and ( 3 ) by ArcA-phosphate and NarL-phosphate , which , during nitrate respiration , control the highest level of moe operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>It is tempting to speculate that in the wild-type cell , low aerobic expression of gipA largely reflects activation by CAP , whereas the high anaerobic expression of the operon largely reflects activation by the FNR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2403539" target="_blank">2403539</a>]<br></li>
<li>3474 RESULTS AND DISCUSSION ArcA and FNR are required for complete anaerobic pfl operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>It was clear from previous studies that a functional FNR protein was not the sole factor responsible for the anaerobic induction of the pfl operon expression ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>In ECL515 ( frd-4 ) anaerobic expression is not significantly influenced by FNR whereas ECL516 ( frd-5 ) produced 1.5-fold more enzyme in an fnr ' background unlike the fr & allele which synthesized four-fold more fumarate reducatase in response to FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>The fnr strain , JRG1728 , failed to produce a detectable yﬁD transcript in both exponential ( not shown ) and stationary-phase cultures , indicating that all yﬁD expression is FNR dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767578" target="_blank">9767578</a>]<br></li>
<li>The twofold anaerobic induction of the dcuA-lacZ fusion was unaffected by the fnr deletion ( data not shown ) , revealing that FNR has no role in dcuA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>This increase in anaerobic expression does not appear to be FNR dependent , as shown by the fact that strains lacking FNR showed no differences from the FNR strain in transcript levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_463">
Additionally , all promoters were dependent on FNR since anaerobic expression was negligible in the JRG1728 fnr strain ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></label>
<input id="tu_similarity_463", type="checkbox">
<div class="TU_similar">
<li>In each case , anaerobic induction was dependent on FNR since expression was negligible in the JRG1728 fnr strain ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li></div>
<li>We ﬁnd that the major regulator of yﬁD expression is FNR , but that yﬁD expression is induced at the level of transcription during growth under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>That FNR is the major transcription regulator of yﬁD expression was conﬁrmed by Western blotting , which showed that extracts of an fnr mutant strain ( JRG1728 ) grown anaerobically at pH 7 ± 2 contained very little YﬁD protein compared to the parent strain , indicating that anaerobic expression of yﬁD under these conditions is FNR dependent ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Although previous studies demonstrated that FNR is inactivated by the presence of O2 , expression of glnK and yedL decreased under aerobic growth conditions in an fnr mutant compared to the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The fnr mutant , lacking the O2-responsive regulator FNR , was completely devoid of dcuC9-9lacZ expression during aerobic and anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>The suggestion that the acid-responsive yﬁD expression is independent of FNR was supported by Western blot analyses of cell-free extracts of MC1000 ( pGS1000 ) ( fnr + ) and JRG1728 ( pGS1000 ) ( ∆ fnr ) prepared after aerobic and anaerobic growth at pH 7 ± 2 in the presence or absence of benzoate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>The data revealed that both FNR and CRP made signiﬁcant positive contributions to hlyE expression , whereas the hns mutation resulted in slightly reduced hlyE transcription under these conditions ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The ﬁndings that SlyA does not enhance hlyE expression in hns cultures grown in liquid medium under anaerobic conditions and that SlyA appears unable to activate hlyE expression in the absence of FNR and CRP and the determination of the location of the region of PhlyE occupied by SlyA ( Fig. 6 ) suggest that SlyA activates hlyE expression by antagonizing the negative action of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The Arc-activated cydAB ( 1 ) , and Arc-repressed sodA ( 13 ) and lpdA ( 26 ) , are also under the direct negative control of FNR , a regulator that co-mediates gene expression in response to the redox conditions of growth ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14711822" target="_blank">14711822</a>]<br></li>
<li>The ndh gene is normally expressed under aerobic conditions when FNR is inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14766919" target="_blank">14766919</a>]<br></li>
<li>The results presented in Table 4 show that the residual FNR protein-independent expression of the CC fusion construct was eliminated by the Acrp mutation , irrespective of growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629153" target="_blank">1629153</a>]<br></li>
<li>The lpd ( lpdA ) gene , encoding lipoamide dehydrogenase , a member of the phdR-aceEF-lpdA operon ( Fig. 6 ) , shows a 4.8-fold decrease in expression when grown under anaerobic conditions and a 11.4-fold increase in the FNR deletion strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The FNR strain had half as much - Gal activity as the strain lacking FNR , suggesting that FNR does repress rpoS expression under an-aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The aim of the work described here was to further characterise the role of FNR in regulating hyp expression under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The transcription factor FNR ( for fumarate and nitrate reduction ) plays a major role in altering gene expression between aerobic and anaerobic conditions to facilitate such changes in energy metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>Results The starting point for this work was the report that the c70-dependent hyp operon promoter ( PhypA ) requires the anaerobic transcription factor FNR for expression under aerobic and anaerobic conditions [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>The expression of hmp is highly induced by NO under aerobic and anaerobic conditions , through a complex regulation that involves at least three regulators , namely , FNR , MetR , and the NO-sensitive repressor NsrR ( 3 , 9 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Evidently , CRP , and not FNR , was required for the expression of ClyA in hns mutants under anoxic conditions , although it appeared that FNR could partly complement CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Simply avoiding the use of the repressing carbon source , glucose , allowed the FNR dependence of the nrf and nap promoters to be demonstrated , even under growth conditions that were suboptimal for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Therefore , the enhanced expression of yfiD under microaerobic conditions is dependent on the presence of a high-affinity activating FNR site ( 240.5 ) and a lower affinity repressing FNR site ( 293.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>The expression of fumB is activated under anaerobic conditions by the anaerobic transcriptional regulator , FNR ( 37 ) , and potential FNR-binding sites were sought by using a 12-bp symmetrical consensus , TGATA - - TATCAA , based on that proposed by Spiro and Guest ( 25 ; S. Spiro , M. H. Todd , P. J. Artynfliuk , and J. R. Guest , Biochem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656658" target="_blank">2656658</a>]<br></li>
<li>Thus , expression of ttdA requires transcriptional activation by FNR , which is in the active state only under anaerobic conditions ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>During anaerobic growth , transcription of the hmp gene is repressed by FNR , but severe exposure to NO stress results in inactivation of FNR and hence de-repression of hmp expression [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>The uncertainty arises because hmp expression is repressed by FNR , and this repression is relieved during anaerobic growth under conditions of severe nitrosative stress ( Table 1 ; Cruz-Ramos et al. , 2002 ; Corker & Poole , 2003 ; Pullan et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>This suggests that during anaerobic growth in liquid cultures in the absence of glucose , FNR is the major regulator of hlyE expression , and that CRP-mediated hlyE expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>During anaerobic growth the fumarate and nitrate reduction regulator ( FNR ) was important for ClyA expression , and the clyA gene could be activated by overexpression of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>However , in the absence of FNR , ndh expression is activated by the amino acid response regulator ( Arr ) during anaerobic growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Despite the activation of the repressors ArcA and FNR by the withdrawal of oxygen in the aerobic -- anaerobic transition , ndh expression did not decrease ( figure 6b ) , consistent with the microarray data ( figure 4c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>expression is controlled by the FNR protein ( Page et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934939" target="_blank">7934939</a>]<br></li>
<li>Both types are expressed only during anaerobic growth and , in Escheri-chia coti , expression is dependent on the FNR protein ( Page et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057835" target="_blank">8057835</a>]<br></li>
<li>This suggests that FNR represses specific roles of the regulators in trancyd expression under anaerobic con - scriptional activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1957353" target="_blank">1957353</a>]<br></li>
<li>The resuits in Tabie 7 confirm that expression from pnrf was totally FNR dependent and induced during anaerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The residuai anaerobic expression in such constructs was FNR - , but no longer ArcA-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>FNR is essential for anaerobic induction of narK expression ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>However , the magnitude of anaerobic induction of narK expression by FNR is greater ( 100-fold ) than the induction of narGHJI ( 10-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>Thus , it seems most likely that FNR ± FNR interactions alone are sufficient for anaerobic downregulation of yfiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11169114" target="_blank">11169114</a>]<br></li>
<li>aeg-46 .5 is known to respond to the presence of FNR for anaerobic expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432709" target="_blank">8432709</a>]<br></li>
<li>FNR protein is needed for anaerobic expression of the nar operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Expression Pattern IV : Increased Expression during Anaerobiosis , No Change in an FNR Strain -- The 12 genes of this cluster ( Table IV ) show elevated expression under anaerobic growth conditions but are not affected by the deletion of the FNR allele ; none of these genes contain documented or putative FNR-binding sites near their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Intriguingly , the expression of both genes was shown to be repressed under anaerobic growth conditions in an FNR-de-pendent manner ( 32 , 34 ) ; however , no well conserved FNR consensus binding site was found to be present in the promoter region of gpmA and sodB , leaving the regulation by FNR unexplained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>Genome-wide expression analysis has previously shown that FNR contributes to the repression of a number of genes induced by oxidative stress conditions [ 16,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22152010" target="_blank">22152010</a>]<br></li>
<li>Other than its effect on the rpoS regulon , CsgD expression appears to regulate a subset of anaerobiosis-dependent genes , such as ompW and pﬂB , both induced in anaerobic conditions by FNR , either alone or in concert with ArcA ( 18 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>However , in anaerobically grown strains lacking FNR , expression of these genes was increased , suggesting that anaerobic expression of genes in group V is repressed by FNR and that the increase in expression under anaerobic conditions must be from another factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>It is suggested that tandem FNR half-sites may be a feature of other genes that are active under anaerobic conditions and use both c54 - and c70-dependent promoters for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>Group I : genes whose expression is increased under anaerobic conditions and whose activation is FNR dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group II : genes whose expression appears to be repressed by FNR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group III : genes whose expression is decreased under an-aerobic conditions and whose repression is partially FNR dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Group VIII : genes whose expression is increased under an-aerobic conditions and whose activation is FNR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Surnrnary In enteric bacteria , the expression of rnany genes encoding various anaerobic electron transfer functions is controlled by FNR , the product of the auto-regulated fnrgene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2677602" target="_blank">2677602</a>]<br></li>
<li>In fact , of the 31 remaining genes of this expression group regulated by FNR under anaerobic growth conditions , 10 are regulated by Lrp under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>More interestingly , of the remaining 27 genes of this expression group , 20 were also found to be regulated by FNR under an-aerobic conditions ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>It has long been appreciated that the activity of FNR must be regulated by O2 availability , since FNR dependent gene expression is observed in vivo only under anaerobic conditions , while similar levels of this protein are present in both aerobic and anaerobic grown cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9990723" target="_blank">9990723</a>]<br></li>
<li>To identify the genes controlled by FNR , we analyzed gene expression profiles of cells cultured under anaerobic growth conditions in the presence or absence of FNR ( O , 2 FNR or O , FNR ) in otherwise isogenic strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>To identify genes controlled by FNR , we used Affymetrix Antisense GeneChips to compare global gene expression proﬁles from isogenic MG1655 wild-type and fnr strains grown in glucose minimal media under aerobic or anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>is repressed by FNR and stimulated by ArcA under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Expression of cyd is thus stimulated by ArcA under microaerobic conditions and repressed by FNR unde anaerobic conditions , resulting in a maximal expression under microaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>FnrS is involved in gene regulation after the shift from aerobic to anaerobic conditions , and its expression is activated by the transcription factors FNR and ArcA ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Rather , hya expression in S. enterica is regulated not by IscR or ArcA but , rather , by FNR under anaerobic conditions ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In this same report , FNR and IHF were reported to have roles as weak repressors under anaerobic conditions , and it was suggested that the effect of FNR may be indirect as a consequence of its role in regulating ArcA expression ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>For example , it was reported recently that ArcA and FNR repressed the expression of puuA when E. coli was grown under anaerobic conditions and that there are putative binding motifs of ArcA and FNR upstream of the open reading frame of puuA ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18495664" target="_blank">18495664</a>]<br></li>
<li>In contrast , there was a C 1 ± 9-fold increase in stationary-phase yeiL expression in the fnr mutant ( JRG3886 ) under both aerobic conditions ( Fig. 6 ) and anaerobic fermentative conditions ( data not shown ) , suggesting that yeiL transcription is directly or indirectly repressed by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>An in vivo study using a fnr-IucZ translational fusion suggested that fnr is expressed under both aerobic and anaerobic growth conditions but that its expression is repressed by glucose or hyper-expression of FNR particularly during anaerobic growth [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>However , the data for the parental and fnr cultures suggested that even under aerobic growth conditions , some active FNR was pres-ent in the bacteria , and that even this small amount of active protein ( relative to anaerobic conditions ) was sufﬁcient to interfere with CRP-mediated hlyE expression ( Fig. 1c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Since FNR has also been reported to enhance arcA expression under anaerobic conditions ( although this result was not observed in our studies ) , it raises the question of whether the effects of FNR observed with this group are all direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>Although our studies indicate that the predicted upstream site is not an FNR binding site , we found that this sequence binds IHF and increases expression of fnr under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Expression of yecI , yjhB and smf was not affected by the presence or absence of FNR under anaerobic conditions ( Table 2 ) , although expression of both yecI and smf was apparently lower in the absence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>To verify that fnrS transcription is indeed controlled by FNR , we analyzed by Northern blot the expression profile of this sRNA in cells cultured under aerobic or anaerobic growth conditions ( O2 or O2 ) and under anaerobic growth conditions in the presence or absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>As nikABCDE expression is very low in the absence of FNR and rcnA activation is identical under aerobic and anaerobic growth conditions , nickel ions are probably entering the cell via another transport pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Under anaerobic conditions FNR is a dimeric iron-sulfur protein that binds at specific DNA sequences to activate the expression of `` anaerobic genes '' and to repress the expression of `` aerobic genes '' ( 7 , 8 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>Under anaerobic conditions in the absence of nitrate the FNR protein induces fdnG operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Under anaerobic conditions , one mechanism that controls fnr expression is negative autoregulation by the active [ 4Fe-4S ] form of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>However , expression of these genes was upregulated under anaerobic growth conditions in the absence of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>When the gatPI sequence from - 3 0 to - 5 4 was replaced by the corresponding nirB sequence , expression from gatPI became inducibie by FNR under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845227" target="_blank">2845227</a>]<br></li>
<li>Involvement of both CRP and FNR in regulation of clyA expression during anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Under anaerobic conditions it is likely that the opposing activities of FNR ( - ) and Arr ( + ) compete in order to ` fine-tune ' ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></li>
<li>Thus , under some anaerobic growth conditions Arr may compete with FNR and IHF for occupation of the ndh promoter and , by acting positively , Arr may counteract the negative effects of FNR and IHF to ` finetune ' ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>For instance , the global regulator FNR ( fumarate and nitrate reductase ) , whose expression is partially induced by RpoS , represses genes of the tricarboxylic acid cycle ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>Transcription of this operon is driven from a single promoter ( pnrf ) , and expression is induced in the absence of oxygen by the FNR protein ( regulator of fumarate and nitrate reduction ) , a global transcription activator that controls the expression of many genes required for anaerobic respiration ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Transcription of this operon is driven from a single promoter ( pnir ) and is induced by the fumarate and nitrate reduction ( FNR ) protein , a global transcription regulator that activates the expression of many genes in response to oxygen starvation ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>In the Gram negative bacterium Escherichia coli , this process is primarily controlled by the global transcription regulator FNR ( fumarate and nitrate reduction ) which is activated under anaerobic conditions and upon activation , regulates the expression of more than one hundred genes in E. coli genome [ 1,2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22326866" target="_blank">22326866</a>]<br></li>
<li>Summary In Escherichia coli , the anaerobic expression of genes encoding the nitrate ( narGHJI ) and dimethyl sulphoxide ( dmsABC ) terminal reductases is stimulated by the global anaerobic regulator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Our studies have focused on understanding the role of FNR in activating expression of the genes encoding the anaerobic terminal reductases for nitrate , narGHJI ( Stewart and Rabin , 1995 ) , and dimethyl sulphoxide , dmsABC ( Cotter and Gunsalus , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Introduction The expression of genes required for anaerobic growth of Escherichia coli is tightly controlled , and many such genes are reguiated by the FNR protein , a transcription factor that is activated by the absence of oxygen in the medium ( reviewed by Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437517" target="_blank">8437517</a>]<br></li>
<li>Introduction In Escherichia coli , the transcription factor FNR facilitates adaptation to anaerobic growth conditions by regulating the expression of gene products that are involved in anaerobic energy metabolism ( Spiro and Guest , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The Escherichia coli K-12 nrf operon encodes the NrfA periplasmic nitrite reductase ( 9 ) , and its expression is controlled by a single promoter ( pnrf ) whose activity is completely dependent on FNR , the principal global transcription regulator responsible for anaerobic adaptation ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Repression of in vitro transcription of the Escherichia coli fnr and narX genes by R protein The expression of respiratory genes in E. coli is controlled by two trandscriptional regulatory systems : one is the ArcB/A system for aerobic genes [ l ] ; and the other is the FNR system for anaerobic genes [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8119409" target="_blank">8119409</a>]<br></li>
<li>Here , we extend this view by the identification and characterization of a highly conserved , anaerobically induced small sRNA in E. coli , whose expression is strictly dependent on the anaerobic transcriptional fumarate and nitrate reductase regulator ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>In this work we report on the identification and characterization of a highly conserved , anaerobically induced sRNA in E. coli , whose expression is strictly dependent on FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>The requirement of ArcA and FNR for peak expression of the cyd operon in Escherichia coli under microaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592804" target="_blank">1592804</a>]<br></li>
<li>To examine the effects of FNR and ArcA on expression of the E. coli nrdDG operon under oxic and anoxic conditions , Δfnr and ΔarcA mutations were individually transduced into -- -- strain IG40 to generate strains IG34 ( Fnr ) , IG35 ( ArcA ) , -- -- and IG36 ( Fnr / ArcA ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18683632" target="_blank">18683632</a>]<br></li>
<li>Oxygen and nitrate-dependent regulation of dmsABC operon expression in Escherichia coli : sites for FNR and NarL protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Firstly , as a polypeptide upregulated by expression of the heterologous oxygen-responsive FNR protein , HlyX , in anaerobically grown E. coli ( Green & Baldwin , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>