<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - GadW</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">GadW</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">GadW transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Lanes C show the results for incubation with 0.8 M MalEGadW in the presence of 100 fmol of the speciﬁc unlabeled DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>Their regulation is known to be mediated by transcription factors GadXGadW and EvgA-YdeO , as well as by RpoS , H-NS , and cyclic AMP [ 60,61,74 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>GadX-GadW Regulon in Escherichia coli Genes of the Don L. Tucker ,1 † Nancy Tucker ,1 ‡ Zhuo Ma ,2 John W. Foster ,2 Regina L. Miranda ,3 Paul S. Cohen ,3 and Tyrrell Conway1 * Advanced Center for Genome Technology , The University of Oklahoma , Norman , Oklahoma 730191 ; Department of Microbiology and Immunology , University of South Alabama College of Medicine , Mobile , Alabama 366882 ; and Department of Biochemistry , Microbiology , and Molecular Genetics , University of Rhode Island , Kingston , Rhode Island 028813 Received 13 December 2002/Accepted 4 March 2003 Acid in the stomach is thought to be a barrier to bacterial colonization of the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>hdeB Periplasmic chaperone of acid-denatured proteins hdeA Periplasmic chaperone of acid-denatured proteins hdeD Putative membrane transporter , H-NS repressed gadE Required for stationary phase-induced , pH 5.5 growth medium-induced , and EvgA-induced acid resistance mdtE MdtEF-TolC multidrug resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine mdtF MdtEF-TolC multidrug resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine gadW Positive AraC-type regulator of gadA and gadBC , in absence of gadX ; repressor of gadX gadY sRNA regulates gadX and gadY gadX Positive transcriptional-regulator of gadA and gadBC ; repressed by HNS and GadW ; stimulated by RpoS gadA Glutamate decarboxylase isozyme GadB operon : 1567 -- 1570 kb gadB Glutamate decarboxylase isozyme gadC Glutamate transporter 1.03 0.95 3.17 -1.59 1.58 2.58 -0.98 1.60 2.70 -1.17 1.53 3.17 2.66 -1.38 -1.47 1.80 1.19 4.89 -3.46 1.43 3.21 3.73 -2.50 -2.44 1.29 Table 4 : Drug resistance , cytoskeletal and ion transport proteins showing pH dependence under oxygen limitation Gene Function Log2pH ratio Acid up-regulated pH 5.7 / pH 7.0 pH 7.0 / pH 8.5 pH 5.7 / pH 8.5 0.80 0.60 0.86 1.31 0.44 chaB chaC Regulator of Na + - Ca + / H + antiporter Regulator of Na + - Ca + / H + antiporter -0.38 -0.56 -0.4 copA cueR cusC feoA feoB marR1 mdtE mdtF mdtG1 mdtI1 mdtJ1 mdtL mgtA nhaA nhaB1 nikA nikB nikC nikD nikE prc Copper - , silver-translocating P-type ATPase Activator of copper-responsive regulon genes cueO and copA Silver and copper efflux ferrous iron transport protein A ferrous iron transport protein B multiple antibiotic resistance-protein MdtEF-TolC multidrug resistance efflux transporter MdtEF-TolC multidrug resistance efflux Multidrug resistance efflux transporter multidrug transport protein multidrug transport protein Multidrug resistance efflux protein Mg2 + transport Na + / H + antiporter Na + / H + antiporter periplasmic binding protein for nickel nickel transport nickel transport ATP-binding protein of nickel transport system ATP-binding protein of nickel transport system carboxy-terminal protease for penicillin-binding protein 3 0.91 0.63 1.22 0.53 0.82 0.41 0.31 -1.59 -0.98 3.17 2.58 0.16 1.58 1.60 0.25 1.16 3.02 0.57 0.63 1.00 0.37 0.84 0.56 0.86 0.74 0.49 0.37 1.16 2.91 0.48 1.93 -0.93 0.41 0.7 0.50 0.70 0.60 0.45 0.32 Base up-regulated -0.34 -0.35 -0.69 -0.42 -1.22 -0.52 -0.12 -0.66 -0.66 -0.71 -0.51 -0.66 -0.58 -0.53 -1.1 -0.39 -0.3 -0.65 acrA acrB acridine efflux pump acridine efflux pump -0.93 acrR2 alx2 ampC atpA2 atpC2 atpD2 atpF2 atpG2 atpH2 atpI2 chaA corA cutA dacA dacC dacD emrA1 mepA1 mreB2 mreC napF napG napH acrAB operon repressor Putative metal transporter , homolog of terC ( tellurium resistance ) beta-lactamase ; penicillin resistance ; penicillin-binding protein ATP synthase , F1 alpha-subunit ATP synthase , F1 epsilon-subunit ATP synthase , F1 beta-subunit ATP synthase , F0 subunit b ATP synthase , F1 gamma-subunit ATP synthase , F1 delta-subunit ATP synthase subunit Na + - Ca + / H + antiporter Mg2 + transport , system I divalent cation tolerance protein D-alanyl-D-alanine carboxypeptidase , fraction A ; penicillin-binding protein 5 D-alanyl-D-alanine carboxypeptidase ; penicillin-binding protein 6 penicillin binding protein 6b multidrug resistance secretion protein murein DD-endopeptidase , penicillin-insensitive regulator of ftsI , penicillin binding protein 3 , septation function rod shape-determining protein Fe-S ferredoxin-type protein ferredoxin-type protein ferredoxin-type protein -0.57 -0.24 -0.36 -0.46 -0.35 -0.42 -0.86 0.10 -0.27 -0.58 0.87 -1.22 -0.64 -0.68 -0.53 -0.29 -0.59 -0.39 -0.37 -1.16 -0.90 -0.11 -0.19 -0.44 -0.39 -0.14 -0.20 -0.20 1Acid up-regulated with aeration [ 2 ] 2Base up-regulated with aeration [ 2 ] Real-time PCR expression ratios of selected genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response to drugs , QseB for quorum sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>In a recent report , Zwir et al. ( 43 ) verified that PhoP played a critical role in the control of acid resistance : Transcription of hdeA and gadAX is induced at low Mg2 in a PhoP - and YhiW ( GadW ) - dependent fashion ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17998538" target="_blank">17998538</a>]<br></li>
<li>DISCUSSION In the present study we demonstrated that even in the presence of a functional gadX gene , GadW is an activator of gad structural genes in E. coli K-12 strains MC4100 and MG1655 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>Although the mechanism by which GadW can exert these dual effects is not known , it is possible that the relative intracellular levels of these two regulators could account for activation or repression of the gad system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The increased resistance of the gadW mutant following 2 and 4 h of acid challenge suggests that GadW represses an AR function that is important in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>Altogether , these data demonstrate that acetylation of lysine 154 of RcsB impairs its function , affecting flagella biosynthesis and B A flhDC FlhDC RcsB RcsA fliLMNOPQR fliE flgMN flgKL fliFGHIJK fliC 1 -3 -2 -1 0 2 3 4 flgAMN FlgM CobB flgBCDEFGHIJK RcsB RcsB AcK Acetyl-P motAB cheAW flhBAE Signal transduction Response to external stimulus fliDST FliA fliAZY Chemotaxis Locomotion Ciliary or flagellar motility Cell motility Cellular component organization or biogenesis Cellular component organization Response to chemical stimulus Cell projection organization Response to pH Metabolic process Cellular metabolic process Cellular response to stress Response to stress Response to abiotic stimulus Response to DNA damage stimulus Cellular catabolic process Catabolic process Small molecular metabolic process Carboxylic acid metabolic process Small molecule catabolic process Carboxylic acid catabolic process C GadX RcsB GadE GadX RcsB GadE RcsB GadE gadE mdtEF gadA gadX slp dctR gadBC cadBA GadW gadW hdeAB ydeP hdeD Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25518064" target="_blank">25518064</a>]<br></li>
<li>Induction of NhaR , GadE , and GadW by YdeO Four transcription factors , the LysR-type NhaR , the LuxR-type GadE , and the AraC-type GadW , were found to be under the direct control of YdeO ( see Figs. 1 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>However , we failed to detect NhaR and GadW even in the presence of YdeO expression ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>In addition , gadE-mtdEF belongs to the AFI and was shown to be controlled by GadX and GadW ( Kobayashi et al. , 2006 ; Sayed et al. , 2007 ) , while ybaST is not included in the AFI but is known to be signiﬁcantly induced under acidic conditions and to belong to the GadX -- GadW regulon ( Tucker et al. , 2002 ; 2003 ; Hommais et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>GadW activates gadX expression via the gadAX promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The positive effect of GadW on the gadA promoter was also observed through an increase in gadX expression which was cotranscribed with gadA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>GadW directly represses gadX whereas Crp indirectly represses gadX by lowering RpoS levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>GadW represses gadX but can also directly activate the gadA/BC promoters under different situations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>GadW also represses gadX under this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>These include the EvgA-YdeO-GadE feed forward regulatory loop which is effective in minimal glucose medium under aerobic condition ( Ma et al. , 2004 ) , the conditional activation of the operon in a MnmE dependent manner in LB containing glucose ( LBG ) ( Gong et al. , 2004 ) , as well as the GadX or GadW dependent activation which functions primarily in rich media in stationary growth-phase ( Sayed et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>The gadA/BC genes , activated in response to acid stress and in stationary phase cells , are subjected to complex circuits of regulation involving 70 , S , cAMP receptor protein , H-NS , EvgAS , TorRS , GadE , GadX , GadW , and YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Transcription of hdeAB depends on complex circuits of regulation involving RpoD ( 70 ) , RpoS ( 38 ) , H-NS , cyclic AMP receptor protein , GadE , GadX , GadW , GadY , EvgAS , YdeO , Lrp , MarA , SoxRS , TorRS , and TrmE , among other regulators ( see Fig. 1 and references in the legend ; see Fig. 2 for the DNA binding sites of some of these regulators in the hdeAB promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Thus , we propose a complex regulatory cascade in which global regulators ( RpoS , CRP , HN-S , EvgA , etc. ) inﬂuence the expression levels and/or activities of GadX and GadW , which in turn activate the expression and/or activities of transcription activators ( e.g. , YdeO and YhiE ) that directly activate subsets of target genes involved in AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>A complex acid resistance regulon including the gad system is regulated by transcription factors GadX-GadW and EvgA-YdeO , as well as by RpoS , H-NS , and cyclic AMP ( 11 , 12 , 44 , 79 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601715" target="_blank">15601715</a>]<br></li>
<li>Regulation of gadE expression involves such regulators as EvgA , YdeO , GadE , PhoP , TrmE , GadX , and GadW ( 12 , 21 , 24 , 30 , 33 ) , which makes GadE the center of regulation for the induction of acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>]<br></li>
<li>More than ten transcription regulators including EvgA , YdeO , GadE , TorR , H-NS , PhoP , RpoS , CRP , MnmE , GadX , and GadW have been found to participate in the regulation of gadE during various circumstances of acid stresses through at least ﬁve different regulatory circuits ( Ma et al. , 2004 ; Sayed and Foster , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Regulation of glutamine supply by RutR The supply of glutamine from externally added glutamate was also found to be indirectly regulated by RutR through control of the synthesis of GadX and GadW , the regulators of the Gad system for glutamate transport .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>Many AraC-like regulators bind ligands that alter their function ( 5 ) , but whether or not GadX or GadW binds speciﬁc cytoplasmic ligands remains unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Glucose induction of P3 and P3P2 is independent of YdeO , GadXW , EvgAS and Crp We then asked whether any of the E. coli-speciﬁc regulators YdeO , GadX , GadW or EvgAS were needed for glucose-dependent induction of P3P2 -- lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>GadX and GadW appear to play a role in gadE induction regardless of whether glucose is present in the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>Overall , the absence of GadW resulted in a decrease in gadA and gadBC induction that was more pronounced at an alkaline pH. In particular , transcription of gadA was affected only at pH 8.0 , while transcription of gadBC was reduced at pH 8.0 , as well as in the stationary phase at pH 5.5 ( residual activity , 72 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>GadX and GadW initially appeared to activate and repress the core gad genes , depending on the situation ( 13 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>GadX and GadW have been reported to activate and repress these promoters under different conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>GadX transcriptionally activates GadA and GadB expression , and some studies suggest that GadW acts as a repressor to oppose this action ( 24 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466020" target="_blank">15466020</a>]<br></li>
<li>GadX activates expression from PgadA , while GadW inhibits expression ( in the presence of GadX ) from PgadA and PgadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>This hypothesis is supported by the ﬁnding that in minimal medium under fermentative growth conditions and independent of the pH , GadW repression of gadA and gadBC is not due to inhibition of GadX and the latter protein is responsible for gadW repression ( 20 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>Promoters P3 and P2 can be activated by GadX and GadW during growth to stationary phase , or by YdeO during log-phase growth in minimal medium below pH 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Within the S-dependent circuit during the stationary phase of growth , either in rich or in minimal medium , the increase in gadA and gadBC transcription requires GadX , which directly binds the gadA and gadB regulatory regions and , under some circumstances , acts in combination with GadW ( 8 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The results indicate that GadW mediates control of the gadA and gadBC genes by repressing gadX and , under some situations , by activating gadA and gadBC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The AraC-family regulators GadX and GadW , previously found to activate gadA/BC in vitro , are now shown in vivo to directly activate gadE expression , which , in turn , activates the gadA/BC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>RcsB-dependent regulation of gadABC is not mediated by the regulation of gadEWX genes The expression of gadA and gadBC is regulated by a complex network involving notably the GadE , GadW and GadX regulators ( Ma et al. , 2003 ; see Foster , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>]<br></li>
<li>Another unusual aspect of GadW is seen in Fig. 6A and B. Whereas removing GadW allowed GadX to activate gadA-lacZ expression at both growth pH values , removing GadX allowed W to activate expression only at pH 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Unlike LB , where they were needed to activate gadA/BC , GadX and GadW acted as negative regulators in minimal media ( in exponential phase ) such that removing them allowed additional , acid-induced expression ( Fig. 1E , lanes 1 and 2 versus 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>However , when GadW was removed through mutation , GadX activated gadA and gadBC expression under acid or alkaline conditions ( Fig. 2A , lanes 5 and 6 versus lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW binds to the gadA and gadB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>One model , discussed brieﬂy above , is that GadW might bind gadA and gadBC promoters best when cells are grown at pH 8 , which either prevents GadX from binding or attenuates GadX activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>It seems that , while either GadX or GadW can induce gadA and gadBC , when both AraC-like regulators are present , they inhibit each other 's function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW clearly represses gadX and , in situations where GadX is missing , activates gadA and gadBC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>However , when GadX is missing , GadW could inadvertently activate gadA and gadBC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>An interesting question is why does GadW , in the absence of GadX , activate gadA and gadBC only at pH 8 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadX levels increase and overcome repression of gadA and gadBC by GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Thus , GadX serves as an activator of gadA and gadBC , while GadW can inhibit activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW inhibits activation of gadA and gadBC by GadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Thus , GadW is a negative regulatory protein that inhibits the activation of gadA and gadBC by GadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Results we report elsewhere indicate that GadW inhibits the GadX-dependent activation of gadA and gadBC transcription , yet GadW activates their transcription in the absence of GadX ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>GadX activates expression of gadA and gadBC at any pH , while GadW inhibits GadX-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>In the merodiploid strain , GadX and + GadW stimulate the wild-type gadE allele , and the resulting GadE protein activates the 2360 and 2195 gadE -- lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>GadW , though less effective than GadX , still activated gadE -- lacZ between four - and fivefold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>Moreover , repression of hdeAB by MarA was greater in the absence of GadX or Lrp in exponential phase at pH 7.0 and in the absence of GadW or RpoS in stationary phase at pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>In turn , MarA enhanced repression of hdeAB by H-NS and hindered activation by GadE in stationary phase and also reduced the activity of GadX , GadW , RpoS , and Lrp on hdeAB under some conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>GadE was the main activator of hdeAB in LB medium under all the conditions studied , whereas GadX repressed or activated hdeAB depending on the growth conditions , and GadW repressed hdeAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>The different effects that GadX and GadW had under diverse conditions , sometimes opposite to that of GadE , support the idea that they can regulate hdeAB independently of GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">GadW protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Such bending may bring the activator protein , GadX or GadW at around position 220.5 , into contact either with the RNA polymerase or with an additional factor binding in the nucleotide region close to the 35 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>Therefore , it is possible that speciﬁc nucleotides must be present at the indicated positions in each site to allow for the correct binding of the recognition helix by the relevant HTH motif in each GadX ( GadW ) subunit , and any mutation may affect the overall binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The Crp-RpoS-GadX-GadW loop ( called TABLE 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The length of the binding site suggests that GadW and GadX bind as dimers to the gadA promoter region , with the recognition helix of each of the four helix-turn-helix motifs occupying one of four consecutive DNA major grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>GadW is 60 and 30 % homologous to GadX in the DNA-binding and dimerization domains , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>This observation is in line with a report showing that GadX and GadW , via their N-terminal domains , form not only homodimers but also heterodimers ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>Therefore , we split each binding site into two ; each half-site ( shown in Fig. 11A ) was of sufficient length ( 21 bp ) to allow the binding of the two HTH motifs present in the C-terminal domain of a GadX ( or GadW ) monomer , as in the case of the AraC / XylS-like proteins ( Gallegos et al. , 1997 ; Egan , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Similar to other members of the AraC/XylS-family , GadX and GadW have an N-terminal domain that is composed of approximately 200 amino acids and is involved in dimerization ( Ma et al. , 2002 ; Tramonti et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>GadX and GadW are both members of the AraC/XylS-family ( PS01124 ) , which typically comprises ª300-amino-acid-long transcriptional regulators that bind an unusually extended 17 -- 21 bp sequence at target promoters via two helix -- turn -- helix ( HTH ) motifs localized within the 100-amino-acid-long C-terminal domain ( Gallegos et al. , 1997 ; Egan , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>GadW and GadX are members of the AraC/XylS protein family of transcriptional regulators , which are characterized by the presence in the C-terminal domain of two helix-turn-helix motifs ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">GadW evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The role of GadX , GadW , and YdeO , all of which are members of the AraC-like family of transcriptional regulators , is restricted to spe-ciﬁc circuits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>GadW , like GadX , belongs to the family of AraC-like regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21314918" target="_blank">21314918</a>]<br></li>
<li>Similar to other members of the AraC/XylS family , GadX and GadW have an N-terminal domain that is composed of approximately 200 amino acids and is involved in dimerization ( Ma et al. , 2002 ; Tramonti et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>GadX and GadW are both members of the AraC/XylS family ( PS01124 ) , which typically comprises ª300-amino-acid-long transcriptional regulators that bind an unusually extended 17 -- 21 bp sequence at target promoters via two helix -- turn -- helix ( HTH ) motifs localized within the 100-amino-acid-long C-terminal domain ( Gallegos et al. , 1997 ; Egan , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>GadW and GadX are members of the AraC/XylS protein family of transcriptional regulators , which are characterized by the presence in the C-terminal domain of two helix-turn-helix motifs ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The consensus binding sequences we used were the following : 5 WYAGGMWWWWDYTW YWWW for binding of GadE , GadX , and GadW ( gadbox [ 26 , 43 ] ) ; 5 YAGHA WATTWTDCTR for Lrp ( 11 ) ; and 5 CTGTTCATAT for TorR ( 2 , 40 ) ; where D is A , T , or G ; H is A , C , or T ; M is A or C ; R is A or G ; W is A or T ; and Y is C or T. Only sequences with 70 % or higher identity to the consensus sequences were selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>The association of GadX and GadW in heterodimers is likely to occur because these proteins exhibit certain levels of sequence identity and similarity ( 14 % and 38 % ) in the N-termi-nal dimerization domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The ﬁnding that GadX and GadW recognize the same site is in agreement with previous results ( Tramonti et al. , 2006 ) and is not unexpected because their C-terminal DNA-binding domains show high sequence identity and similarity ( 41 % and 66 % respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>GadX and GadW are DNA binding proteins that form homodimers in vivo and are 42 % homologous to each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW is 60 and 30 % homologous to GadX in the DNA binding and dimerization domains , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by GadW</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>hdeB Periplasmic chaperone of acid-denatured proteins hdeA Periplasmic chaperone of acid-denatured proteins hdeD Putative membrane transporter , H-NS repressed gadE Required for stationary phase-induced , pH 5.5 growth medium-induced , and EvgA-induced acid resistance mdtE MdtEF-TolC multidrug-resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine mdtF MdtEF-TolC multidrug-resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine gadW Positive AraC-type regulator of gadA and gadBC , in absence of gadX ; repressor of gadX gadY sRNA regulates gadX and gadY gadX Positive transcriptional regulator of gadA and gadBC ; repressed by HNS and GadW ; stimulated by RpoS gadA Glutamate decarboxylase isozyme GadB operon : 1567 -- 1570 kb gadB Glutamate decarboxylase isozyme gadC Glutamate transporter 1.03 0.95 3.17 -1.59 1.58 2.58 -0.98 1.60 2.70 -1.17 1.53 3.17 2.66 -1.38 -1.47 1.80 1.19 4.89 -3.46 1.43 3.21 3.73 -2.50 -2.44 1.29 Table 4 : Drug resistance , cytoskeletal and ion-transport proteins showing pH dependence under oxygen limitation Gene Function Log2pH ratio Acid up-regulated pH 5.7 / pH 7.0 pH 7.0 / pH 8.5 pH 5.7 / pH 8.5 0.80 0.60 0.86 1.31 0.44 chaB chaC Regulator of Na + - Ca + / H + antiporter Regulator of Na + - Ca + / H + antiporter -0.38 -0.56 -0.4 copA cueR cusC feoA feoB marR1 mdtE mdtF mdtG1 mdtI1 mdtJ1 mdtL mgtA nhaA nhaB1 nikA nikB nikC nikD nikE prc Copper - , silver-translocating P-type ATPase Activator of copper-responsive regulon genes cueO and copA Silver and copper efflux ferrous-iron-transport protein A ferrous-iron-transport protein B multiple antibiotic resistance protein MdtEF-TolC multidrug-resistance efflux transporter MdtEF-TolC multidrug-resistance efflux Multidrug-resistance efflux transporter multidrug-transport protein multidrug-transport protein Multidrug-resistance efflux protein Mg2 + transport Na + / H + antiporter Na + / H + antiporter periplasmic binding protein for nickel nickel transport nickel transport ATP-binding protein of nickel transport system ATP-binding protein of nickel transport system carboxy-terminal protease for penicillin-binding protein 3 0.91 0.63 1.22 0.53 0.82 0.41 0.31 -1.59 -0.98 3.17 2.58 0.16 1.58 1.60 0.25 1.16 3.02 0.57 0.63 1.00 0.37 0.84 0.56 0.86 0.74 0.49 0.37 1.16 2.91 0.48 1.93 -0.93 0.41 0.7 0.50 0.70 0.60 0.45 0.32 Base up-regulated -0.34 -0.35 -0.69 -0.42 -1.22 -0.52 -0.12 -0.66 -0.66 -0.71 -0.51 -0.66 -0.58 -0.53 -1.1 -0.39 -0.3 -0.65 acrA acrB acridine efflux pump acridine efflux pump -0.93 acrR2 alx2 ampC atpA2 atpC2 atpD2 atpF2 atpG2 atpH2 atpI2 chaA corA cutA dacA dacC dacD emrA1 mepA1 mreB2 mreC napF napG napH acrAB operon repressor Putative metal transporter , homolog of terC ( tellurium resistance ) beta-lactamase ; penicillin resistance ; penicillin-binding protein ATP synthase , F1 alpha-subunit ATP synthase , F1 epsilon-subunit ATP synthase , F1 beta-subunit ATP synthase , F0 subunit b ATP synthase , F1 gamma-subunit ATP synthase , F1 delta-subunit ATP synthase subunit Na + - Ca + / H + antiporter Mg2 + transport , system I divalent cation tolerance protein D-alanyl-D-alanine carboxypeptidase , fraction A ; penicillin-binding protein 5 D-alanyl-D-alanine carboxypeptidase ; penicillin-binding protein 6 penicillin binding protein 6b multidrug-resistance secretion protein murein DD-endopeptidase , penicillin-insensitive regulator of ftsI , penicillin binding protein 3 , septation function rod shape-determining protein Fe-S ferredoxin-type protein ferredoxin-type protein ferredoxin-type protein -0.57 -0.24 -0.36 -0.46 -0.35 -0.42 -0.86 0.10 -0.27 -0.58 0.87 -1.22 -0.64 -0.68 -0.53 -0.29 -0.59 -0.39 -0.37 -1.16 -0.90 -0.11 -0.19 -0.44 -0.39 -0.14 -0.20 -0.20 1Acid up-regulated with aeration [ 2 ] 2Base up-regulated with aeration [ 2 ] Real-time PCR expression ratios of selected genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>Altogether , these data demonstrate that acetylation of lysine 154 of RcsB impairs its function , affecting flagella biosynthesis and B A flhDC FlhDC RcsB RcsA fliLMNOPQR fliE flgMN flgKL fliFGHIJK fliC 1 -3 -2 -1 0 2 3 4 flgAMN FlgM CobB flgBCDEFGHIJK RcsB RcsB AcK Acetyl-P motAB cheAW flhBAE Signal transduction Response to external stimulus fliDST FliA fliAZY Chemotaxis Locomotion Ciliary or flagellar motility Cell motility Cellular component organization or biogenesis Cellular component organization Response to chemical stimulus Cell projection organization Response to pH Metabolic process Cellular metabolic-process Cellular response-to-stress Response to stress Response to abiotic stimulus Response to DNA damage stimulus Cellular catabolic-process Catabolic process Small molecular metabolic-process Carboxylic acid-metabolic-process Small molecule catabolic-process Carboxylic acid-catabolic-process C GadX RcsB GadE GadX RcsB GadE RcsB GadE gadE mdtEF gadA gadX slp dctR gadBC cadBA GadW gadW hdeAB ydeP hdeD Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25518064" target="_blank">25518064</a>]<br></li>
<li>To obtain insight into the role of GadW in controlling transcription of the gad genes in response to growth phase and pH , we constructed a gadW mutant and analyzed transcription of the gad genes in wild-type and mutant strains by performing Northern blot and Gad enzymatic assays ; the results are shown in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>AdiY and GadW are also involved in expression of the low pH response genes for control of intracellular pH by using Arg and Glu , respect-ively ( Ma et al. , 2002 ; Stim-Herndon et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Northern blot analysis further indicates that GadW represses expression of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW represses expression of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Cross lines , negative control ; arrows , positive control This result for the strain lacking GadW but possessing GadX suggests that GadW functions as a repressor of at least some GadX-activated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA-cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>J Bacteriol 181 ( 7 ) :2084 -- 2093 Tucker DL , Tucker N , Ma Z , Foster JW , Miranda RL , Cohen PS , Conway T ( 2003 ) Genes of GadX-GadW regulon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>NhaR is an activator of a sodium/proton antiporter gene [ 29 ] and both GadE and GadW are involved in regulation of the genes for glutamate-dependent acid resistance system [ 8,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>NhaR , GadE , GadW E. coli responds to temporary low pH using the glutamate-dependent acid resistant system , which involves two complex regulatory systems : EvgAS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>In addition to induction by acid and GadX control , the adjacent AraC-like transcription factor GadW plays a role in regulating the glutamate-dependent AR system ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_13">
Ma Z , Richard H , Tucker DL , Conway T , Foster JW : Collaborative regulation of Escherichia coli glutamate-dependent acid resistance by two AraC-like regulators , GadX and GadW ( YhiW ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></label>
<input id="rp_similarity_13", type="checkbox">
<div class="RP_similar">
<li>Collaborative regulation of Escherichia coli glutamate-dependent acid resistance by two AraC-like regulators , GadX and GadW ( YhiW ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21147125" target="_blank">21147125</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19426207" target="_blank">19426207</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867448" target="_blank">12867448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601715" target="_blank">15601715</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594828" target="_blank">14594828</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466020" target="_blank">15466020</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17259322" target="_blank">17259322</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li></div>
<li>Two other genes in this region , GadX and GadW , are involved in modulating gadE expression as well as the expression of gadA and gadBC ( 16 , 32 , 33 , 44 , 48 -- 50 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17259322" target="_blank">17259322</a>]<br></li>
<li>GadX or GadW alone activated this expression ( Fig. 3a , d , fourth bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>The goal of the gene-expression pro-ﬁling experiments was to identify target genes under GadX control and to investigate the role of the associated transcription factor , GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>The latter circuit is also the most complex because apart from GadX and GadW , it also involves the global regulators H-NS and CRP , which act as transcriptional repressors , and RpoS , which positively affects gadA and gadBC transcription ( De Biase et al. , 1999 ; Castanie-Cornet and Foster , 2001 ; Hommais et al. , 2001 ; Tramonti et al. , 2002 ; Giangrossi et al. , 2005 ; Weber et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>RpoS , which is directly involved in transcription from the P1 promoter , activates the P2 and P3 promoters via transcriptional regulators GadX and GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>]<br></li>
<li>Thus , we propose a complex regulatory cascade in which global regulators ( RpoS , CRP , HN-S , EvgA , etc. ) inﬂuence the expression levels and/or activities of GadX and GadW , which in turn activate the expression and/or activities of transcription activators ( e.g. , YdeO and YhiE ) that directly activate subsets of target genes involved in AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>The gadA/BC genes , activated in response to acid stress and in stationary phase cells , are subjected to complex circuits of regulation involving 70 , S , cAMP receptor protein , H-NS , EvgAS , TorRS , GadE , GadX , GadW , and YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Expression of these components is under the control of a complex network of transcription factors , including GadE , GadX , GadW , EvgA , YdeO , and H-NS [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The expression of the gadAXW and gadBC operons varies , depending on the culture medium , growth phase and environmental pH. The balance of three transcription factors , GadE , GadX and GadW , is involved in this complex regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>This system likely includes several regulons , which are under hierarchical control of a number of regulatory proteins : the two-component signal-transduction-system EvgA/EvgS , GadE , YdeO , GadX , GadW as well as the global regulators RpoS and H -- NS [ 18,19,22,23,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>Components involved include a two-component signal-transduction-system ( EvgAS ) , three activators ( GadE , X , W ) , two sigma factors ( sigma S and sigma 70 ) , two global regulators ( Crp , H-NS ) and at least one regulatory loop ( GadX -- GadW ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>In this regulatory process H-NS represses the expression of gadE ( encoding the central activator of the glutamate-dependent acid resistance pathway ) both in a direct and an indirect way , via EvgA , YdeO , GadX and GadW [ 1,7,8 ] , while it decreases rcsD expression , essential to the phosphorylation of RcsB ( the capsular synthesis regulator component ) required for the formation of the regulatory complex with GadE [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Identification of the targets directly controlled by H-NS H-NS modulates the expression of several regulators controlling acid stress resistance including HdfR , RcsD , EvgA , YdeO , YdeP , GadE , GadW , GadX , AdiY and CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response-to-drugs , QseB for quorum-sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Within this YdeO regulon , we identified four stress-response transcription factors , DctR , NhaR , GadE , and GadW and enzymes for anaerobic-respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>In this study , we performed a comprehensive interrogation of YdeO-binding sites in vivo on the E. coli genome using ChIP-chip analysis , and identified a set of YdeO-regulated genes , including four stress-response transcription factors , DctR , NhaR , GadE , and GadW , and several genes involved in respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">GadW binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>GadX and GadW are both members of the AraC/XylS family ( PS01124 ) , which typically comprises ª300-amino-acid-long transcriptional regulators that bind an unusually extended 17 -- 21 bp sequence at target promoters via two helix -- turn -- helix ( HTH ) motifs localized within the 100-amino-acid-long C-terminal domain ( Gallegos et al. , 1997 ; Egan , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Therefore , we split each binding site into two ; each half-site ( shown in Fig. 11A ) was of sufficient length ( 21 bp ) to allow the binding of the two HTH motifs present in the C-terminal domain of a GadX ( or GadW ) monomer , as in the case of the AraC / XylS-like proteins ( Gallegos et al. , 1997 ; Egan , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The consensus binding sequences we used were the following : 5 WYAGGMWWWWDYTW YWWW for binding of GadE , GadX , and GadW ( gadbox [ 26 , 43 ] ) ; 5 YAGHA WATTWTDCTR for Lrp ( 11 ) ; and 5 CTGTTCATAT for TorR ( 2 , 40 ) ; where D is A , T , or G ; H is A , C , or T ; M is A or C ; R is A or G ; W is A or T ; and Y is C or T. Only sequences with 70 % or higher identity to the consensus sequences were selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Therefore , it is possible that speciﬁc nucleotides must be present at the indicated positions in each site to allow for the correct binding of the recognition helix by the relevant HTH motif in each GadX ( GadW ) subunit , and any mutation may affect the overall binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The 42-bp-long GadX/GadW binding site clearly arises from the tandem arrangement of two 21 bp sequences , which were analysed separately in the ﬁnal consensus because we hypothesized that the binding of each monomer in the functional GadX ( and/or GadW ) dimer does not necessarily occur with similar affinity and/or strength at each site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Certainly , the location of the 20-bp sequence element upstream of the mapped gadA and gadB promoters is consistent with binding by an activator , but it may serve as a binding site for yet another regulator , and GadX-GadW may inﬂuence that binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>The newly identiﬁed GadX and GadW binding site , centered at 220.5 bp upstream of the transcription start site of gadBC , is consistent with the hypothesis that stimulation of transcription mediated by these two regulators may occur through a DNA looping mechanism ( for a review , see reference 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The position of the GadX ( GadW ) binding site , centered 110 bp from the gadA transcriptional start site , is compatible with a model in which GadX and GadW contact this promoter as class I activators , i.e. , activators that bind to a site upstream of the promoter 35 element and contact the C-terminal domain of the subunit of RNA polymerase , thereby recruiting the polymerase to the promoter ( 3 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>In this study , we identiﬁed a new 41 bp GadX / GadW binding site that was centred 41.5 bp upstream of the gadY ( 105 ) transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The distance of the GadX/GadW binding site from the transcriptional start site of the target genes varies signiﬁcantly : it can be so close that it overlaps with the -35 hexamer , as in the case of gadY , or it can be as far as 584 bp upstream , as in the case of gadE-mtdEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The length of the binding site suggests that GadW and GadX bind as dimers to the gadA promoter region , with the recognition helix of each of the four helix-turn-helix motifs occupying one of four consecutive DNA major grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The length of the binding site already suggested that GadX ( or GadW ) binds as a dimer ( Tramonti et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>In a previous study ( Tramonti et al. , 2006 ) , we noted that GadX and GadW recognize the same 44 bp binding site in the gadA or gadBC 5 ′ regulatory region and that the two binding sites share a signiﬁcant degree of sequence identity ( 57 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>In addition , based on previous results ( Tramonti et al. , 2006 ) and the identiﬁcation of a GadX/GadW binding site within the gadY -- gadW intergenic region , we have deﬁned a GadX/GadW consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Indeed , we identiﬁed a GadX/GadW binding site in the gadY -- gadW intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>It has been shown that both GadX and GadW bind to the gadA/BC promoter regions and that the 20 bp GAD box enhances GadX binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>By aligning the GadX/GadW binding site on the gadY promoter with the GadX / GadW binding sites previously identiﬁed in the gadA and gadBC 5 regulatory regions , we generated a 42 bp GadX/GadW consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Interestingly , the location of the GadX/GadW binding site on the gadA and gadBC 5 ′ regulatory regions is not within the most conserved region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Identiﬁcation of MalE-GadW and MalE-GadX binding sites in gadA and gadBC regulatory regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_20">
A. Sequence alignment of the GadX and GadW binding sites on the gadA , gadBC and gadY regulatory regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></label>
<input id="sit_similarity_20", type="checkbox">
<div class="SIT_similar">
<li>B. Sequence alignment of the GadX and GadW binding sites in the gadA , gadBC , gadY , hdeAB , slp and gadE regulatory regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li></div>
<li>Alignment of sequences upstream of operons regulated by GadX-GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>We have thoroughly analyzed the putative regulatory regions of the 10 operons regulated by GadX-GadW and identiﬁed no other conserved sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>The availability of three new GadX/GadW binding sites prompted us to align the six available sequences ( Fig. 11B ) to generate a more reliable consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Another important ﬁnding of this study was the identiﬁcation of a GadX/GadW consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_26">
Identiﬁcation of the GadX/GadW consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></label>
<input id="sit_similarity_26", type="checkbox">
<div class="SIT_similar">
<li>Identiﬁcation of a GadX/GadW consensus sequence Fig. 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li></div>
<li>We also showed for the ﬁrst time that GadX and GadW speciﬁcally bind to a 42 bp sequence that is found in ﬁve of the eight 5 ′ regulatory regions of the AFI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>DNase I footprinting analyses conﬁrmed that a 42 bp GadX / GadW binding site , which matched the consensus sequence 5 - WANDNCTDWTWKTRAYATWAWMATG KCTGATNTTTWYNTYAK-3 , is also present in the regulatory region of the slp-yhiF , hdeAB and gadE-mtdEF operons , all of which belong to the AFI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>DNase I footprinting analyses of the 5 ′ regulatory regions of the slp-yhiF , hdeAB and gadE-mtdEF operons corroborated the results from the in silico analysis and helped in improving the GadX/GadW consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by GadW</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Among the genes regulated by EvgA are gadX and ydeO , which encode AraC-like transcription factors similar to GadW , and yhiF and yhiE , which encode LuxR-like transcription factors ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>YdeO-overexpression induced GadE-dependent transcription of the gadW gene but GadW protein was not detected in growing E. coli cell ( Fig. 6 ) , suggesting that stationary phase specific factors are required for GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>( b ) Transcriptional fusions of the b1500-ydeO , gadE , gadBC , and asr gene promoters were constructed in the pLUX vector , and the response of each of these promoters to a shift from pH 7 to pH values ranging between 6.3 and 4.9 ( as well as a mock induction at pH 7 ) was assayed in an E. coli MG1655 wild-type genetic back ¬ GadX and GadW , along with the small regulatory RNA GadY , collaborate with RpoS to relieve H-NS-mediated repression of the system when cultures enter into stationary phase16 but have also been reported to have a role during low pH response .20,29 In addition , GadW is under direct control of the PhoPQ two-component system ,30 and it has been shown that EvgAS can cross-talk to PhoPQ via the connector protein encoded by b1500 ( also known as safA26 ) , which resides directly upstream of and in the same operon as YdeO .18,31 As a first step to building a mathematical description of this complex network , we wished to assess whether a dynamic approach to monitoring gene expression within the network was both feasible and valid and in particular whether such an approach would reveal any novel aspects of the network that had not been detected to date .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>In addition , gadE-mtdEF belongs to the AFI and was shown to be controlled by GadX and GadW ( Kobayashi et al. , 2006 ; Sayed et al. , 2007 ) , while ybaST is not included in the AFI but is known to be signiﬁcantly induced under acidic conditions and to belong to the GadX -- GadW regulon ( Tucker et al. , 2002 ; 2003 ; Hommais et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>GadW was shown to repress gadX expression in rich LB me-dium both at an alkaline pH and at an acidic pH ( 21 ) , and the repression was proposed to occur indirectly because GadW does not bind to the gadX promoter ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>It has been reported that , independent of pH , GadW represses transcription of the gad structural genes indirectly by repressing gadX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>GadW activates gadX expression via the gadAX promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>GadW also represses gadX under this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Northern blot analysis further indicates that GadW represses expression of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW represses expression of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The results indicate that GadW mediates control of the gadA and gadBC genes by repressing gadX and , under some situations , by activating gadA and gadBC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The positive effect of GadW on the gadA promoter was also observed through an increase in gadX expression which was cotranscribed with gadA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The effects of GadW on gadA and gadBC expression are not solely through its impact on gadX expression , however , since GadW can affect gadA and gadBC expression even in the absence o FIG. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>In fact , GadX directly binds to gadW promoter region , where GadX and GadW collaborate to repress gadA/BC gene expressions under alkaline conditions ( Ma et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>In addition to low RpoS levels , log phase expression of gadX is repressed by GadW ( Ma et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>The expression of the gadAXW and gadBC operons varies , depending on the culture medium , growth phase and environmental pH. The balance of three transcription factors , GadE , GadX and GadW , is involved in this complex regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>As GadE is also required for stationary phase expression of gadA/BC We demonstrated previously that either GadX or GadW was required by stationary phase cells to express gadA / BC ( Ma et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>RcsB-dependent regulation of gadABC is not mediated by the regulation of gadEWX genes The expression of gadA and gadBC is regulated by a complex network involving notably the GadE , GadW and GadX regulators ( Ma et al. , 2003 ; see Foster , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_19">
GadX and GadW together participate the regulation of expression of the gadAXW and gadBC operons ( Ma et al. , 2002 ; 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></label>
<input id="tu_similarity_19", type="checkbox">
<div class="TU_similar">
<li>GadX and GadW together participate the regulation of expression of the gadAXW and gadBC operons ( Ma et al. , 2002 ; 2003 ) ( see Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li></div>
<li>Mechanisms of transcription activation exerted by GadX and GadW at the gadA and gadBC gene promoters of the glutamate-based acid resistance system in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17259322" target="_blank">17259322</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>The two AraC-like regulators , GadX and GadW , reside downstream of gadA but are transcribed , for the most part , by independent promoters ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>GadX and GadW have been reported to activate and repress these promoters under different conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>The finding that GadX and GadW do not appear to directly activate gadA/BC promoters in vivo , but do so indirectly by activating gadE , contradicts earlier models , including our own .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>This hypothesis is supported by the ﬁnding that in minimal medium under fermentative growth conditions and independent of the pH , GadW repression of gadA and gadBC is not due to inhibition of GadX and the latter protein is responsible for gadW repression ( 20 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>Under these growth conditions , the GadE activator is primarily implicated in gadA and gadBC transcription ( its overexpression causes GAD production even in cells lacking both GadX and GadW regulators ) , and gadE itself turns out to be acid-induced ( 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Strain EF861 , possessing GadX but lacking GadW , highly expressed the gadA and gadBC messages under both acid and alkaline conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Effects of GadX and GadW on pH-regulated gadA and gadBC expression in exponential - and stationary-phase cells grown in LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadX levels increase and overcome repression of gadA and gadBC by GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>However , when GadW was removed through mutation , GadX activated gadA and gadBC expression under acid or alkaline conditions ( Fig. 2A , lanes 5 and 6 versus lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>However , when GadX is missing , GadW could inadvertently activate gadA and gadBC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>GadW clearly represses gadX and , in situations where GadX is missing , activates gadA and gadBC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Since the gadW gene is located downstream of gadA and gadX , it was necessary to determine if independent promoters drive expression of each gene before analyzing the individual roles of GadX and GadW in controlling gadA and gadBC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Nevertheless , since the in vivo data presented indicate that GadX and GadW activate gadE , and that GadE is essential for gadA and gadBC induction , it seemed possible that the in vivo roles of GadX and GadW in inducing gadA and gadBC may have more to do with their triggering the expression of GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>At least under certain conditions , the regulators GadX and GadW seem to play opposite roles in the expression of gadA , gadBC , and the hde genes , whereas GadE seems to be an essential activator for these genes ( 27 , 43 , 44 , 47 , 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>MarA seems also to compete functionally with GadX , GadW , RpoS , and Lrp under some conditions , which might ensure that hdeAB expression is not repressed more than needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>We also found that neither GadX nor GadW activated expression of the 2360 to +331 fusion , confirming that sequences upstream of 2360 are important for activation by these proteins ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>Alignment of sequences upstream of operons regulated by GadX-GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>RpoS does not appear to activate gadE through a GadXW-independent pathway , since rpoS , DgadXW and rpoS DgadXW mutants are equally acid sensitive , and the overexpression of RpoS does not induce gadE in a DgadXW mutant ( data not shown ) The '' 360 to '' 804 region is required for GadX and GadW activation of gadE The intergenic region controlling gadE expression is large ( 798 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>However , during growth in glucose ( pH 5.5 ) , GadX and GadW activation of gadE was additive ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>However , the fact that RpoS is not needed for gadE expression in glucose , yet GadX and GadW remain important , suggests the presence of an RpoS-independent route for gadX induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>Effects of GadX and GadW on gadE expression in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>However , neither GadX nor GadW is required for gadE expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>Therefore , we decided to further analyse the gadY -- gadW intergenic region because it contains not only the regulatory elements required for gadY transcription but also the 5 ′ regulatory region of the gadW gene , which is transcribed divergently from gadY and encodes another known regulator of the GadX/GadW-dependent circuit of the gad system , i.e. GadW ( Tramonti et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>At present , it is known that the expression of the essential regulator GadE , a LuxR-like protein that is encoded by the gadE gene and located in the AFI ( Fig. 1 ) , is under the control of three different activation circuits ( for review , see Foster , 2004 ) : ( i ) the EvgAS/YdeO-dependent circuit activates gadE during the exponential phase of growth in minimal medium ( pH 5.5 ) containing glucose and relies on the activity of both the EvgAS two-component system and the activator YdeO , a member of the AraC/XylS family of transcriptional regulators , ( ii ) the TrmE-dependent circuit activates gadE in the stationary phase in unbuffered LB medium containing glucose and relies on the activity of TrmE , an Era-like molecular switch GTPase , and ( iii ) the GadX/GadW-dependent circuit is required for gadE activation upon entry into the stationary phase in buffered LB medium and involves the transcriptional regulators GadX and GadW , the genes of which are located downstream of gadA and gadX respectively ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>When using the -222 / +121 gadX : : lacZ construct , which contains the entire gadX 5 ′ regulatory region ( Tramonti et al. , 2002 ; Giangrossi et al. , 2005 ) , we observed that the transcription of the gadX gene from its native promoter increases in the stationary phase but is not appreciably affected by the pH of the growth medium or any of the known regulators of the GadX/GadW-dependent circuit of activation in the gad system , such as GadX , GadW , GadE and RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The data from the gadX : : lacZ transcriptional fusions indicate that transcription of the gadX gene from its own promoter is not signiﬁcantly affected by either the pH of the growth medium or any of the regulators of the gad system , such as GadX itself , GadW , GadE and RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The goal of the gene expression pro-ﬁling experiments was to identify target genes under GadX control and to investigate the role of the associated transcription factor , GadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>Thus , in contrast to other highly low-pH-responsive E. coli genes , asr does not appear to depend on the hierarchical control exerted by the EvgA/S two-component system and the downstream GadE , GadX and GadW regulatory proteins [ 8,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>The expression of glutamate-dependent acid resistance genes is induced at low pH and is subject to regulation by GadE , GadW , GadX , TrmE , HNS , TorR , Crp , YdeO , the EvgSA two-component regulatory system and the sigma factor RpoS ( reviewed in Foster 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>