<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - GalR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">GalR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">GalR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>171 , No. 9 MalI , a Novel Protein Involved in Regulation of the Maltose System of Escherichia coli , Is Highly Homologous to the Repressor Proteins GalR , CytR , and Lacl JOACHIM REIDL , KARIN ROMISCH , MICHAEL EHRMANN , AND WINFRIED BOOS * Department of Biology , University of Konstanz , D-7750 Konstanz , Federal Republic of Germany Received 13 March 1989/Accepted 10 June 1989 The maltose regulon of Escherichia coli comprises several operons that are under common regulatory control of the MalT activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Except for a B. subtilis transacting protein that is involved in catabolite repression of a-amylase and shows similarity to the LacI and GalR repressors of E. coli ( 168 ) , no other gene products have been identified that are responsible for the regulation of various catabolite-repressible genes in nonenteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>Mall , anove protein involved in regulation of the maltose system of Escherichia coli is highlyhomologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_6">
Reidl , J. , Römisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></label>
<input id="act_similarity_6", type="checkbox">
<div class="ACT_similar">
<li>Reidl , J. , RoÈmisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692154" target="_blank">10692154</a>]<br></li>
<li>Reidl J , Romisch K , Ehrmann M & Boos W ( 1989 ) MalI , a novel protein involved in the regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to therepressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003949" target="_blank">12003949</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618441" target="_blank">12618441</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1856179" target="_blank">1856179</a>]<br></li></div>
<li>The third column reports specific activity of the specified gal operator construct in the presence of the repressor deletion plasmid , pGalR ∆ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>Loop formation in viro and in vin ' o has been demonstrated in natural and artificial situations for a number of prokaryotic regulatory pro-reins including the Lacl , GalR , AraC , DeoR , ~ cI and P22 repressors ( reviewed in [ 49 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>β-Galactosidase activities of gal operator constructs GalR plasmid + inducer 363.7 ± 10.3 212.1 ± 15.8 GalR plasmid GalR ∆ plasmid pLacZ Oi → ← Oi ← Oe 325.0 ± 50.4 87.4 ± 15.3 260.6 ± 10.2 387.6 ± 52.4 316.9 ± 14.9 551.8 ± 56.6 597.3 ± 9.1 497.0 ± 20.9 490.0 ± 9.8 402.3 ± 35.2 478.7 ± 21.8 254.1 ± 11.0 539.9 ± 44.3 442.4 ± 62.4 25.2 ± 2.9 Oe → Oe → Oi → ← → OeOi 47.1 ± 3.0 55.1 ± 4.2 66.6 ± 2.0 ± 24.2 0.9 32.2 ± 2.7 39.1 ± 1.0 332.3 ± 45.9 465.7 ± 69.9 570.3 ± 31.0 Oe → ← Oi ← Oe ← Oi 109.3 ± 10.2 The lacz gene is under the control of the indicated combination of operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>β-Galactosidase activities of the gal operator ← Oe102 ← Oi construct with insertions of 8 and 14 bp between operators GalR plasmid + inducer 109.3 ± 10.2 636.2 ± 27.0 GalR plasmid GalR ∆ plasmid ← Oe102 ← ← Oe110 ← ← ← Oe116 570.3 ± 31.0 492.2 ± 90.6 ± 530.0 35.1 39.1 ± 1.0 287.1 ± 8.7 ± 33.7 1.7 i i i O O O ± 53.3 5.7 Designations are as in the legend to Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>Operator preference of wild-type GalR and GalRA16T Bound by wild-type GalR Bound by GalRA16T Bound by wild-type GalR Not bound by GalRA16T GTGTAATCGTTTTCAC GTGTAAACGTTACCAC GTGTAAACGATAACAC GTGTAAGCGGTACCAC GTGTAATCGCTTCCAC GTGTAATCGATTACAC GTGTAAACGCTCTCAC GTGTAACCGTTGTCAC GTGTAATCGTTAACAC GTGTAAACGCTTGCAC GTGAAAGCGTTTCCAC GTGAAAGCGTTACCAC GTGAAAGCGCTAACAC GTGGTACCGGTTCCAC GTGGAAACGTTCTCAC GTGGAAACGGTACCAC GTGTAAACGCTTTCAC GTGTAATCGATTACAC GTGTAACCGTTACCAC GTGTAAACGTTTTCAC GTGTAATCGATTACAC GTGTAAACGTTCTCAC GTGTAACCGTTTACAC GTGGTACCGATACCAC GTGGTACCGTTCCCAC GTGGTAGCGGTACCAT GTGGTACCGCTGCCAC GTGGTAACGTTAACAC GTGGTAGCGTTAACAC GTGGTAACGTTTGCAC GTGGTAGCGTTACCAC GTGGAACCGTTGCCAC GTGGGAGCGGTTTCAC Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>As shown in Figure 3A , the expression of either wild-type GalR or R282L mutant protein genes from a plasmid resulted in strong reduction of the b-glucuronidase activity compared with that in the total absence of GalR ( DgalR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Operator sequences ( GTGNNNNCGNNNNCAC ) , randomized at positions N , bound to the wild-type or mutant GalR proteins were enriched by using the SELEX method ( Tuerk and Gold 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>heat unstable nucleoid protein ( HU ) - mediated DNA looping Giuseppe Lia * † , David Bensimon ‡ , Vincent Croquette ‡ , Jean-Francois Allemand ‡ § , David Dunlap ¶ , Dale E. A. Lewis , Sankar Adhya , and Laura Finzi * , ** * Department of Biology , University of Milan , 20133 Milan , Italy ; ‡ Laboratoire de Physique Statistique and Department de Biologie , and § Departement de Chimie , Ecole Normale Superieure , 75005 Paris , France ; ¶ San Raffaele Scientiﬁc Institute , 20132 Milan , Italy ; and Laboratory of Molecular Biology , National Cancer Institute , Bethesda , MD 20892 Contributed by Sankar Adhya , July 31 , 2003 The overall topology of DNA profoundly inﬂuences the regulation GalR dimers , and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>However , with regard to affinities , the GalR interaction is better compared to less stable interactions , for example to the lacO1 -- lacO3 loop ( in particular see 37 ) or to the lacO2 -- lacO3 interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>Subunit exchange of GalR dimers in solution StrepII-tagged GalRA16T and HIS V15T 6-tagged GalR proteins were purified as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Values for the loops formed by transient interaction of operatorbound GalR dimers were calculated as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In a similar experiment , GalR / GalRV15T , T322R heterodimers repressed P2 promoter effectively only when the orientation of the functional te-tramerization surfaces would allow the AL1 loop formation ( Fig. 3 , cf. lanes 2 and 5,8,11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Roy , S. , Semsey , S. , Liu , M. , Gussin , G.N. , and Adhya , S. ( 2004 ) GalR represses galP1 by inhibiting the rate-determining open complex formation through RNA polymerase contact : a GalR negative control mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Negative control mutants It appears from the data presented above , that GalR speciﬁcally inhibits the last and the slowest step of isomerization at P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>GalR binds to two operators , † These studies are supported by Grants GM39929 and T32-GM07260 from the National Institutes of Health and by the Hirshl Weill-Caulier Trust .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>1 Abbreviations : RNA , ribonucleic acid ; RNAP , RNA polymerase ; LacI , Lac repressor ; GalR , Gal repressor ; CAP , catabolite activator protein ( also known as CRP , cAMP receptor protein ) ; O , `` external '' E operator ; OI , `` internal '' operator ; bp , base pair ; BSA , bovine serum albumin ; DTT , dithiothreitol ; EDTA , ethylenediaminetetraacetic acid ; Bis-tris , bis ( 2-hydroxyethyl ) iminotris ( hydroxymethyl ) methane ; DNase I , deoxyribonuclease I ( EC 3.1.21.1 ) ; DNA , deoxyribonucleic acid ; cAMP , adenosine 3 ′ 5 ′ - cyclic monophosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>( A ) The organization of the operon and its regulation by the positive regulator cAMP receptor protein ( CRP ) and the negative regulator GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>Transcription of the gal regulon promoters is regulated in different ways by Gal repressor ( GalR ) and Gal isorepressor ( GalS ) and a global regulator , adenosine cyclic-3 ′ ,5 ′ - monophosphate ( cAMP ) receptor protein ( CRP ) , responding to two signals , D-galactose and cAMP respectively ( Weickert and Adhya , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Three trans-acting proteins , GalR ( the gal repressor ) , cAMP receptor protein ( CRP ) - cAMP complex , and the histone-like protein HU , control transcription from these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Binding of GalR + , GalRA16T , and GalRV15T , T322R to hybrid operator sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>GalR bound strongly to the previously identiﬁed operator , galOE1 , and also to a site further upstream , labelled galOE5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_33">
For clarity , two sides of a GalR dimer bound to either operator are colored differently , light-gray and light-green .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></label>
<input id="act_similarity_33", type="checkbox">
<div class="ACT_similar">
<li>Two sides of a GalR dimer bound to either operator are colored differently , light-gray and light-green .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li></div>
<li>Only a few of the transporters and regulators in this group bind the same substrates ; galactose ( MglB and GalR ) , ribose ( RbsR and RbsB ) and xylose ( XylF and XylR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>Mutations in nagC , galR and galS increase expression from a galP-lacZ fusion : galR is epistatic to galS To investigate the relative contributions of NagC , GalR and GalS to galP repression , we tested the effect of mutations in nagC , galR and galS in trans on the fusions in the three media ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>GalR-Alexa 488 , at a constant concentration of 20 nM , was mixed with a saturating amount of non-labelled protein , GalS , GalR or NagC , in 20 mM Tris-HCl , pH 7.5 , 300 mM KCl , 2.5 mM MgCl2 , 0.5 mg ml-1 bovine serum albumin and the anisotropy was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Lanes : 6 , no additions ; 7 and 8 , 1 nM GalR in the absence and presence of 10 mM galactose , respectively ; 9 and 10 , 200 nM GalS in the absence and presence of 10 mM galactose , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Lanes : 11 , no additions ; 12 and 13 , 5 nM GalR in the absence and presence of 10 mM galactose , respectively ; and 14 and 15 , 100 nM GalS in the absence and presence of 10 mM galactose , respectively ( cAMP ) receptor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Protection by low concentrations of GalR and GalS seems to be additive at galOE3 ( Fig. 3A , compare lanes 11 and 12 with lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Moreover , the presence of NagC enhances protection of the galOE3 and OE5 operators by low concentrations of GalR and GalS ( Fig. 3A lanes 13 and 14 compared with lanes 11 and 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>When 4 concentrations of NagC , ChbR , GalR or GalS were tested on each DNA ( indicated by a wedge covering 4 lanes ) , the concentrations were 100 , 50 , 25 and 12.5 nM ( calculated for monomeric protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The Gal11-Gal2 DNA fragment , labelled at either end , was mixed with NagC , GalR or GalS at the following concentrations : A. Lanes 2 -- 6 GalR 50 , 25 , 12.5 , 6.2 and 3.1 nM ; lanes 7 -- 10 , GalS 200 , 100 , 50 , 25 nM ; lane 11 , GalS 25 nM and GalR 25 nM ; lane 12 , GalS 25 nM and GalR 12.5 nM ; lanes 13 and 14 , as lanes 11 and 12 with NagC 25 nM ; Lane 15 NagC 25 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>B. Lanes 2 -- 6 NagC 50 , 25 , 12.5 , 6.2 and 3.1 nM , lanes 7 -- 11 GalR 50 , 25 , 12.5 , 6.2 and 3.1 nM ; lanes 12 -- 15 GalS 200 , 100 , 50 , 25 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>We have found that NagC , binds to a single , high-affinity site overlapping the promoter of galP ( galactose permease ) gene and that expression of galP is repressed by a combination of NagC , GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>If so , we would expect that GalS will be induced at lower galactose concentrations than GalR , since the mgl operon is the more efficient transport system at low galactose concentrations , ( ii ) The galS gene is autoregulated and its expression is CRP cAMP dependent , while gaIR expression appears to be constitutive -LCB- Weickert and Adhya , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>As galactose levels increase further , genes under GalR control , the low-afﬁnity galactose transporter , galactose permease , as well as galactose-metabolizing enzymes would be induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>If the primary role of GalS is to control the expression of the high-afﬁnity galactose transporter and GalR controls the low-afﬁnity permease , then it might be predicted that GalS would respond to lower concentrations of galactose than would GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>The increased sensitivity of GalS to galactose relative to that of GalR permits the induction of the mgl operon coding for the high-afﬁnity galactose transporter at galactose concentrations lower than those that would signiﬁcantly induce the gal operon and the low-afﬁnity transporter , galactose permease , which our results indicate are largely under the control of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>However , maintaining a constant concentration of GalR while carefully regulating the concentration of GalS would be an efficient mechanism of differentially modulating the members of the gal regulon to mediate cellular responses to physiological changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Growth on galactose produced values comparable to the wt strain on galactose , while growth on GlcNAc was somewhat higher than the wt strain on GlcNAc , due to the combined effects of the partial loss of GalR binding by mutation and the induction by growth on GlcNAc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>As the mutation nagO2-2 , in the right hand half of the nagO2 site , had only a minor effect on galP expression ( 1.5-fold ) , we presume that the increase in galP-lacZ activity is mostly due to loss of binding of GalR and/or GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Mutations in galS produce partial derepression of galP but the fact that the galS mutation had no effect in the absence of GalR could mean that GalS is incapable of binding/repressing in the absence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>In order to obtain a better understanding of the roles of GalR and GalS in coordinating both galactose metabolism and uptake , we have puriﬁed GalS and studied the interaction of both of these repressors with the regulatory sequences in the regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>DISCUSSION In E. coli two regulatory proteins , GalR and GalS , regulate the uptake and metabolism of galactose by binding to conserved operator sequences in the gal regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR bound most strongly to the external operator , OE , of the low-afﬁnity galactose transporter gene , galP , with a dissociation constant of 0.3 3 1029 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR bound more weakly to OE of mgl , with a Kd of 11 3 1029 M. GalS bound with similar afﬁnity to OE of mgl , with a Kd of 18 3 1029 M ; note that at a saturating concentration , GalS gave more complete protection of the operator site than GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Based on these titrations , GalS was added to OE of mgl ( B ) and GalR was added to OE of galE ( D ) at concentrations at which approximately 85 % of the operators were bound in each case , and the repressor-operator complexes were titrated with increasing concentrations of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>We can not rule out the possibility that the GalR and GalS protein preparations used in the in vitro studies reported here differ in their activity , although two methods of GalS puriﬁcation produced protein preparations with the same mgl binding activity , and the DNA-binding activity of GalR in our preparations agrees closely with results reported previously ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR was bound to the operator DNA , followed by the addition of different amounts of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The percentage of DNA remaining bound to GalR after the addition of different concentrations of D-galactose is shown in Figure 7A and B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The sensitivities of the GalS and GalR DNA complexes to the inducer , galactose , can be compared if the proteins are bound to operators at similar loading free energies and if the bound complexes were titrated with galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>The reduction in DNA-binding afﬁnity indicated in the dissociation curves shown in Fig. 4 ( 11-fold for GalR and 9-fold for GalS ) would be misleading if a fraction of the DNA-bound repressor were inactive with respect to galactose binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Consistent with the fact that GalR is the exclusive regulator of the low-afﬁnity galactose transporter , galactose permease , and that the major role of GalS is in regulating expression of the high-afﬁnity galactose transporter encoded by the mgl operon , we found that the DNA binding of GalS is 15-fold more sensitive than that of GalR to galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Since GalR appeared in vitro to be the sole repressor of the expression of the low-afﬁnity galactose uptake system and since the primary function of GalS is in regulating highafﬁnity galactose transport , we assessed the sensitivity of DNA-bound complexes of these repressors to galactose and found that GalS was 15-fold more sensitive to galactose than was GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>( A and C ) The binding of GalS to OE of mgl and GalR to OE of galE , respectively , is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>The greatest difference in the DNA-binding afﬁnity of GalR and GalS was for OE of the galP gene coding for the low-afﬁnity galactose transporter , galactose permease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Alternatively , in vitro conditions could favor the binding of GalR over that of GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>DNA binding by both GalR and GalS is inhibited in the presence of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>GalR and GalS binding to all sites was sensitive to the inducer galactose as indicated in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Although the afﬁnity change that occurs upon galactose binding is similar for the two proteins , the midpoints of the dissociation curves are different ( 5 3 1026 M galactose for GalS and 100 3 1026 M galactose for GalR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>However , the observed change in the DNA-binding afﬁnity of GalR is that which is predicted by the observation that GalR afﬁnity for galactose changes 15-fold in the presence of saturating operator DNA ( 6a ) , indicating that at least for GalR , this is not the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Its DNA binding is sensitive to galactose , which effects a 12-fold change in its afﬁnity for the mgl operator , similar to the afﬁnity change caused by galactose binding to GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Under these conditions , the dissociation of the repressors from the DNA in response to a given concentration of galactose is a function of the afﬁnity of GalS or GalR for galactose and the change in afﬁnity of the proteins for the operator which is caused by galactose binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>This order of afﬁnities is the reverse of that of GalR , which binds most strongly to the operator of galP and most weakly to that of mgl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>In summary , the order of afﬁnities of the two proteins is reversed : GalR binds most strongly to galP and most weakly to mgl ; the opposite is true for GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>INTRODUCTION The Escherichia coli GalR repressor binds to two operator sites , Oi and Oe , in the galactose operon ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>The apparent association constant , K4 , of galactose for the free repressor was ( 1.25 6 0.4 ) 3 106 M21 for GalS and ( 9.0 6 1.1 ) 3 104 M21 for GalR , indicating that free Gal binds galactose with 14-fold-higher afﬁnity than GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>The relative afﬁnities of GalR and GalS for the operator sequences in gal , mgl , and galP were reﬂected in the effectiveness of the two repressors in inhibiting transcription of these genes in vitro ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Thus , the afﬁnity of GalR appears to be higher than that of GalS for all operators tested , although for the single mgl operator the two proteins had comparable afﬁnities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR had an afﬁnity for this sequence that was 5 - to 10-fold higher than its afﬁnity for the operators of the gal operon , although we could not precisely determine the afﬁnity of GalS for this operator due to nonspeciﬁc binding at high protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalS has a speciﬁc role , as it binds with higher affinity to one of the upstream operators but its effect in vivo is only observed in the presence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Interaction between GalR and GalS The epistatic behaviour of the galR mutation on galS could mean that GalS only binds to the galP operators in the presence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>GalS , like GalR , can be titrated specifically by multicopy gal operators to which the Gal repressor itself binds ( Tokeson ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Our biochemical data conﬁrm the genetic results in that at pH 8.0 , GalR binds much more strongly than GalS to the gal operators OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>In the first point , such a case has been reported in the galETK ( M ) operon concerning galactose utilization , which is regulated by GalR and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A/T at positions 8 of the operators ( Fig. 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>In addition , there is one negative feedback loop involving cAMP , as well as a positive feedback loop between GalR and GalS ( see Supplementary material for details of motif counting ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Results show the combined effect of cAMP -- CRP and GalR ( A ) or GalS ( B ) binding on promoter activities ( right lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Because cAMP -- CRP-mediated P1galE activation is strongly inhibited by GalS , it is unlikely that the differential effect of GalR and GalS on cAMP -- CRP action would be due to the different binding affinities of the two proteins towards the galETKM OE operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>GalR and GalS bind to the same operator sites in the cisregulatory regions of the P1galE , PmglB , PgalP and PgalS promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Both GalR and GalS inhibited CRP-mediated activation of the PgalP and PgalS promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In turn , the intracellular concentrations of the small molecules cAMP and D-galactose control-the-activity of CRP , GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>GalR , GalS , and CRP-cAMP apparently do not play any role in gaiR regulation under the conditions we tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>GalS is itself autogenously regulated and CRP dependent , while the gene for GalR is constitutive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Transcription of both P1galE and P2galE can be regulated by GalR and cAMP -- CRP , while GalS affects only P1galE activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Because three of the promoters do not have detectable intrinsic activity , the effect of GalR and GalS in transcription reactions was studied in the presence of the cAMP -- CRP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Because both GalR and GalS remove cAMP -- CRP-mediated activation , transcription of galS in wild-type cells depends on both cAMP and D-galactose ( AND gate ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Expression of GalP requires the presence of both cAMP and D-galactose , for transcriptional activation ( by cAMP -- CRP ) and inhibition of repression ( by GalR and GalS ) respectively ( AND function ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Indeed , the GalS isorepressor was discovered because , in the absence of GalR , there was some residual repression of the galE gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Because repression of P1galE by GalS is incomplete , we assume that GalR plays the major role in the D-galactose-dependent regulation of the galETKM operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>This ﬁnding suggests that inducer-bound GalR or GalS may still partly repress transcription under conditions where inducer concentrations are high enough to saturate the repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Therefore , unlike GalR , GalS represses P1galE transcription only by contact inhibition , not by DNA looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>GalS is believed to repress transcription , although not efficiently , in a manner analogous to GalR ( Weickert and Adhya , 1992a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Repression of transcription by GalR and GalS in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Regulation of galP by three speciﬁc repressors , NagC , GalR and GalS is unusual in that it involves multiple , speciﬁc regulators from two different areas of metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>By this test there was strong interaction between GalR and GalS , similar to that between GalR with GalR or GalS with GalS ( Fig. 5A ) , implying the possible existence of heterodimers in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The estimated Kd for GalR -- GalS interaction was 400 nM , while the affinity GalR-GalR was 3 -- 4 times weaker .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The affinities of galOE1 , OE3 and OE5 for GalS were similar , albeit somewhat lower than GalR for galOE1 or OE5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>This indicates that GalS acts as a repressor of its own synthesis and that GalR represses galS in a noninducible manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>It is interesting to compare the NagC -- Mlc pair with the isorepressors , GalS and GalR , of the gal regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Binding of GalS and GalR to galactose regulon operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Sensitivity of GalR and GalS operator complexes to galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Unlike GalR , GalS binds with similar affinity to galOE1 , 3 and 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Demonstration of an interaction between GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Regulation by GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>B. Fluorescence anisotropy titration curves for Alexa 488 labelled GalR with GalS , GalR or NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Binding of NagC , ChbR , GalR and GalS to the regulatory regions of E. coli and Salmonella chiP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Regions protected by NagC , ChbR , GalR and GalS are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>We conﬁrmed that GalR does bind to this site by DNase I footprinting ( site called GalR , Fig. 2C and D , lanes 9 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Moreover , no interaction between GalR and NagC was detected by ﬂuorescence anisotropy ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Tryptophan substitution at pos-itions 262 , 263 and 267 in aCTD did not alter GalR regulation in a detectable way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>GalR exerted its speci ® c regulatory effect on each promoter when the other was mutationally inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>However , we detected no effect of GalR on regulation of ytfQ by AraC ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272778" target="_blank">24272778</a>]<br></li>
<li>We tested the effect of GalR concentrations on the regulation of the gal promoters in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>For example , the galE promoter is repressed by GalR , the lacZ promoter repressed by LacI and many CRP-activated promoters are repressed by CytR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>R , RbsR ; G , GalR ; L , Lad ; M , MalI ; C , CytR ; P , PurR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>The fusions carrying mutations in galOI and nagO2 behaved essentially as wt , conﬁrming that these sites , which show negligible binding of GalR or NagC , do not contribute to regulation of galP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Leucine 54 plays a key role in sequence-specific binding of PurR ; this same amino acid is conserved in GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The similarity values for identical residues in the headpiece were 38 for LacI , 36 for GalR , 34 for PurR , 28 for DeoR and 35 for RbtR ( the lac , gal , put and deo repressors from E. coli , and the rbt repressor from Klebsiella pneumoniae respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1851954" target="_blank">1851954</a>]<br></li>
<li>The amino terminus pur repressor is not known , but thepurR 5 ' sequence suggests in GalR , LacI , and CytR has been implicated in DNA recog - an interesting possibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058704" target="_blank">3058704</a>]<br></li>
<li>The operonsregulated by LacI and GalR are also subject to induction by sugars , a situationentirely analogous to that for threbs op-eron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>Specific amino acids of the a-subunit of RNA polymerase are involved in both activation and repression by GalR We previously showed that both activation and repression by OE-bound GalR require an intact aCTD of the RNA polymerase molecule , suggesting the involvement of aCTD region in GalR action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Unlike the Gal repressosome , the OE-bound GalR dimer does not inhibit cAMP -- CRP action at the galETKM promoters ; instead , GalR and cAMP -- CRP cancels each other 's effect ( Lewis , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In one model , GalR tetramerization involves a speciﬁc stacking of the core of the OE and OI bound dimers , OI on top of OE ( Figure 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The top GalR dimer is bound to OI , the bottom one to OE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The inhibition of the CAP-RNAP cooperativity by GalR does not require the synergistic interaction between GalR dimers bound at OE and OI ( Figures 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The separation of these two functions suggests that the mechanisms of repression mediated by the inhibition of CAP-induced activation and by the synergistic action of GalR dimers bound to OE and OI have evolved separately .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Figure 2 shows the typical regulatory effect of GalR bound to OE on transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Isolation and characterization of GalR mutants , which bind to DNA but do not repress transcription , indicated that DNA-bound 3Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>However , since inhibition of GalR binding is not observed on the two-site template ( Figure 5 , lane 8 ) , and RNAP titrations are unaffected by GalR bound at OE ( Dalma-Weiszhausz , 1995 ) , the preponderance of evidence points to RNAP and GalR binding occurring with ∆ ∆ G values close to zero .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>This cooperativity is inhibited by the binding of GalR to the upstream operator , OE , but does not result in the repartitioning of RNAP between the P1 and the P2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>However , the distribution of complexes remains > 99.0 % P1 when both CAP and GalR are simultaneously bound with RNAP , even though the CAP-RNAP cooperativity is inhibited when GalR is bound at OE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>( A ) GalR binding to the 32P-labeled OE operator sequence in the presence of varying amounts of D-galactose in a gel retardation assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>DNA binding of GalR in the absence and presence of inducer The 88 bp EcoRI ± BglII DNA fragment containing the OE operator from pSA907 was labeled with [ a-32P ] dATP by end ® lling , separated by PAGE and puri ® ed using a Qiaex II gel extraction kit ( Qiagen ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The binding of GalR to OE and ∆ ∆ G ) ( ∆ Gintrinsic + ∆ Gcoop ) - ( ∆ Gintrinsic ) or ∆ ∆ G ) ∆ Gcoop OI , or to each operator separately , is independent of RNAP except for the GalR titration of OE on the single-site template conducted in the presence of RNAP ( Figure 5 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The CAP-RNAP cooperativity is completely inhibited by the presence of a saturating concentration of GalR only when this repressor is bound to the OE sequence ( O G E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The following three bars represent the difference in free energy of CAP binding in the presence of RNAP and GalR bound at OE ( third bar ) , OI ( fourth bar ) , and both operators simultaneously ( fifth bar ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Even though the third bar should represent the difference of GalR binding to the same operator in the presence and absence of DNA-bound CAP and RNAP , it is labeled ND , for not determinable , due to the reasons mentioned in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>FIGURE 4 : Histogram of the free energy differences of binding CAP in the presence and absence of RNAP and of GalR bound to one or two of the operator sequences at the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Arrows indicate the RNAP-induced hypersensitive base pair which remains hypersensitive even in the presence of GalR binding to OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Upon RNAP binding , sequences with altered reagent susceptibilities downstream of OI remain visible even in the presence of GalR at OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Because of this protection , the detection of GalR binding to OE in the presence of the CAP-RNAP complex can not be observed by this method ( Figure 3 , lanes 21 and 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>F 5 : Histogram of the free energy differences of GalR binding IGURE in the presence and absence of RNAP and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The second bar represents the difference between GalR binding to OE in the absence and in the presence of RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Bars 7-9 refer to GalR binding to OE in the absence of any other protein , in the presence of RNAP , and in the presence of the RNAP-CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The DNase I footprint titration studies presented in this paper demonstrate independent binding of RNAP and GalR , within experimental error , when the latter binds to each operator separately or to both operators simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The observation that a base pair within OI , which is usually protected upon GalR binding , remains hypersensitive when RNAP is present suggests that GalR at OI and RNAP can bind simultaneously to this DNA , perhaps on opposite sides of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>At least a 90 ° change in the phase of the bound CAP and GalR dimers is required to juxtapose GalR near the `` activation residues '' of CAP , arguing against GalR sterically blocking the RNAP-CAP contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Protein concentrations of GalR and of CAP are expressed in units of dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Nuclease Protection Analysis of the Simultaneous Binding of RNAP , CAP , and GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_162">
Semsey , S. , Geanacopoulos , M. , Lewis , D. E. , and Adhya , S. ( 2002 ) Operator-bound GalR dimers close DNA loops by direct interaction : tetramerization and inducer binding , EMBO J. 21 , 4349-4356 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></label>
<input id="act_similarity_162", type="checkbox">
<div class="ACT_similar">
<li>Semsey , S. , Geanacopoulos , M. , Lewis , D. , and Adhya , S. ( 2002 ) Operator-bound GalR dimers close DNA loops by direct interaction : tetramerization and inducer binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Semsey , S. , Geanacopoulos , M. , Lewis , D.E. , and Adhya , S. ( 2002 ) Operator-bound GalR dimers close DNA loops by direct interaction : tetramerization and inducer binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_165">
Operator-bound GalR dimers close DNA loops by direct interaction : Tetramerization and inducer binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></label>
<input id="act_similarity_165", type="checkbox">
<div class="ACT_similar">
<li>Operator-bound GalR dimers close DNA loops by direct interaction : tetramerization and inducer binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li></div>
<li>The results reported here show that operator-bound GalR forms the DNA loop by direct interaction , and that the inducer may act by targeting the tetramerization step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>These results demonstrate that tetramerization is an intrinsic property of GalR ; it closes the DNA loop by direct GalR ± GalR contact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In the GalR mutant R282L , non-inducibility resulting from strong tetramerization was coupled with a slight increase in the stability of the GalR ± O complex , E strongly suggesting that the R282L mutation acts by stabilizing the DNA-binding conformation and increasing the energy required for the structural rearrangement in the sugar-binding state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Differences in the height of the DNA chain crossovers between the control and DNA -- protein samples were assumed to be caused by the presence of a GalR ( b ) or a LacI ( c ) tetramer bound to the operators on minicircles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Gel retardation assays , as well as quantitative footprinting experiments , have shown that the presence of DNA-bound CAP has no effect upon GalR binding to either or both operators simultaneously ( Dalma-Weiszhausz & Brenowitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The stronger GalR ± GalR interaction also made the gal transcription non-inducible , suggesting that the inducer binding acts by modulating tetramerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Induction of the gal operon requires inducer binding to GalR ; in the presence of a high concentration of inducer , the repressor loses its affinity for DNA ( Majumdar and Adhya , 1984 ; Majumdar etai , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>The multiple effects of the R282L substitution indicate that tetramerization as well as DNA and inducer binding are coupled properties of the GalR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>For GalR , inducer binding results in a loss of repressor affinity for DNA ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>For example , induction of highafﬁnity galactose transport was not affected by a GalRs mutation , which , while not affecting the DNA-binding activity of GalR , makes it insensitive to an inducer ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>CytR is unique among LacI/GalR family members in that binding of its regulatory ligand , cytidine , is linked not to DNA-binding affinity but instead to the cooperative interactions between CytR bound at CytO and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>Multiple binding sites allow some repressors , such as LacI or GalR , to form DNA loops which decrease transcription by restricting the conformational ﬂexibility needed for DNA to melt or wrap around RNAP ( 3 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>In summary , the available structural data support a significant role for upstream DNA contacts by RNAP in the mechanism of CAP-RNAP binding cooperativity and its inhibition by GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Even though GalR binding to O is I energetically independent of RNAP , the conformation of the DNA between these two proteins is altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Because of this discrepancy between the in vivo and in vitro observations , we predicted that efﬁcient and coordinated repression of the two promoters requires another factor that acts by assisting GalR in looping the DNA ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In the Gal repressosome , which represses transcription in the gal operon in Escherichia coli , two GalR dimer proteins bind to two operator elements , OE and OI , and form a DNA loop encompassing a 113 bp DNA segment comprising the gal promoters ( Irani et al. , 1983 ; Majumdar and Adhya , 1984 ; Adhya et al. , 1998 ; Figure 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>GalR represses both promoters in vivo by binding to two operators , OE and OI , which ﬂank the two promoters ( Musso et al. , 1978 ; Adhya and Miller , 1979 ; Aiba et al. , 1981 ; Irani et al. , 1983 ; Kuhnke et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Although the binding of GalR to OE and OI by plasmidborne operator titration experiments has been indicated to be co-operative in vivo ( Irani et al. , 1983 ; Haber and Adhya , 1988 ) , in vitro binding of GalR to the two operators is non-co-operative ( Brenowitz et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Simultaneous binding of GalR to both operators is required for full repression in ViVo ( Haber & Adhya , 1988 ; Choy & Adhya , 1992 ) although no cooperativity has been detected for GalR binding in Vitro ( Brenowitz et al. , 1990 ; Dalma-Weiszhausz & Brenowitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>As mentioned , DNA loopingmediated P2 repression requires the binding of GalR to both gal operators ( Irani et al. , 1983 ; Choy and Adhya , 1992 ; Aki and Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>HU binding requires occupancy of both operators by GalR The repression of the two overlapping gal promoters both in vivo and in vitro requires GalR binding to both gal operators ( Adhya and Miller , 1979 ; Irani et al. , 1983 ; Haber and Adhya , 1988 ; Aki et al. , 1996 ; Figure 1 , lanes 3 , 8 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>GENES & DEVELOPMENT 15:2273 -- 2281 © 2001 by Cold Spring Harbor Laboratory Press ISSN 0890-9369/01 $ 5.00 ; www.genesdev.org The binding of GalR dimers to OE and OI in the absence of HU is noncooperative ( Brenowitz et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The effect of GalR binding to OE , and separately to OI , and the synergy resulting from the binding of GalR to both operators simultaneously are distinguished by analyzing experiments conducted with DNA fragments in which only one of the operators is competent to-bind GalR and comparing those to the results obtained from the DNA fragment where both operators are competent to-bind GalR ( see Methods and Materials ; Ackers et al. , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>GalR binding to OI alone has no detectable effect on transcriptional regulation while the simultaneous binding of GalR to both operators results in a 15-fold repression of transcription ( Adhya & Majumdar , 1987 ; Haber & Adhya , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>GalR-RNAP interactions are thus unlikely to be a major contributor to the repression mechanism mediated by the simultaneous binding of GalR to OE and to OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>In ﬂuorescence kinetics with 2-AP labeled P1 DNA , GalR inhibited the slower of the two-step base-pair distortion process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Repression involves the same DNA looping ; deletion of either OE or OI makes the mutant GalR ineffective in repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>S29R GalR appeared to be effective in repression when DNA looping was allowed in the presence of HU , suggesting that the mutant GalR was functional in DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>These results demonstrate that the repression of P2 by R282L in the absence of HU involves DNA looping by the bipartite operators bound by the mutant GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Lack of inhibition of P2 transcription upon destruction of OI demonstrated that the repression of transcription on the DNA templates with the inserts and two intact operators originated from DNA looping by GalR bound to the operators and were independent on HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In the presence of HU , GalR bound to OE and OI brings the two operators together to form a DNA loop , which simultaneously represses P1 and P2 [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Although HU has no effect on transcription in the absence of GalR , it forms a cooperative complex with GalR bound to the tandem operators , OE and OI , which effectively represses P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>The most effective repression of both P1 and P2 by GalR and HU occurred in the wild-type gal DNA with 113 bp distance between OE and OI , while in the plasmids with 31 -- 105 additional base-pairs the repression was partial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>We report here that in restoring repression of transcription to P1 and P2 by GalR , HU binds site-speciﬁcally between the two gal operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The coordinated repression of P1 and P2 requires the binding of GalR to both gal operators , OI and OE ; mutation of either results in loss of the coordination ( lanes 8 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Results P1 repression and P2 activation are independent events GalR normally represses transcription from the two gal promoters by binding to bipartite operators , OE and OI , located at position ÿ60 .5 and 53.5 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>GalR represses transcription initiation from the two promoters , P1 and P2 , of the gal operon by binding to two spatially separated operators , OE and OI , which encompass the Abbreviations : HU , heat unstable nucleoid protein ; GalR , gal repressor dimer protein ; SSB , single-stranded binding protein ; ssDNA , single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>To ascertain whether the inhibition of P1 and stimulation of P2 by GalR itself is related to the relative angular orientation of bound RNA polymerase and repressor on the face of DNA , we inserted an extra half of a DNA helical turn ( 5 bp ) between OE and the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Results GalR super-repressor mutants In an in vivo system , the DNA looping and DNA binding activities of GalR were monitored by following the repression of the P2 and P1 promoters , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Repression of P1 Since amino acid alterations that reduce or eliminate P2 activation also reduce or eliminate P1 repression , it is possible that the same contac between GalR and/or DNA and the holoenzyme brings about the opposite effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>When cellular conditions do not allow Gal repressosome assembly , GalR binding to the external galETKM operator ( OE ) represses the P1galE and activates the P2galE promoter ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Also , GalR completely repressed P1galE by DNA looping ( Aki et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>This was based on the fact that the repression of P2 is entirely dependent upon the formation of a DNA loop , whereas binding of GalR to the OE operator is suf ® cient for repression of P1 ( Aki et al. , 1996 ; Choy et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>These results are consistent with the in vitro results that , in the absence of HU-mediated DNA looping , the P1 promoter is partially repressed by GalR binding to OE ( Aki et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>P1 could also be repressed partially both in vivo and in vitro by the binding of GalR to the OE operator in the absence of DNA supercoiling ( Kuhnke et al. , 1986 ; Choy and Adhya , 1992 ; Aki et al. , 1996 ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>( i ) If DNA looping is prevented by the use of relaxed DNA or by the omission of HU , GalR binding to OE alone brings about partial repression of P1 ( Kuhnke et al. , 1986 ; Choy and Adhya , 1992 ; Aki et al. , 1996 ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Without DNA looping , GalR binding to OE partially represses P1 and stimulates P 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>When DNA looping is not allowed , GalR binding to OE of the galETKM operon activates P2galE and represses P1galE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In the absence of HU , however , GalR binding to OE shows different effects on P1 and P2 ; it represses the former and activates the latter ( Cho 0022 ± 2836/97/380293 ± 08 $ 25.00 / 0/mb971221 & Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>In the absence of HU , GalR binding to OE partially represses the P1 promoter and activates the P2 promoter ( Choy and Adhya 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In the absence of this complex , P1 but not P 2 is still repressed by GalR binding to OE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>In the absence of HU or DNA 13 supercoiling the P1 promoter is repressed by a looping-independent mechanism by binding of GalR to an operator ( OE ) located upstream of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>DISCUSSION In ViVo , the binding of GalR solely to OE modestly represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>In cell extracts , GalR binding solely to OE can partially repress transcription from P1 in the presence and in the absence of CAP ( Kuhnke et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>GalR ( like GalS ) is a dimer in solution but repression of galEP2 requires tetramerization of GalR , mediated by HU forming the complex called a repressosome ( Aki et al. , 1996 ; Lewis et al. , 1999 ; Geanacopoulos et al. , 2001 ; Kar and Adhya , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Increasing the loop size ( Virnik et al. 2003 ) or strengthening the tetramerization interface of GalR dimers ( Geanacopoulos and Adhya 2002 ; Semsey et al. 2002 ) enhanced nonassisted loop formation , resulting in reduced dependency on the architectural contribution of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The incomplete repression of P1galE by GalS may be because GalS dimers bound to the two galETKM operators do not interact with each other to bring about tight repression by forming a DNA loop in the way GalR does in the galETKM operon ( Geanacopoulos and Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>DNA looping , a consequence of interaction between the operator-bound GalR dimers , engenders inhibition of transcription from both promoters ( Aki and Adhya 1997 ; Geanacopoulos et al. 1999 ; Lewis et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>DNA loop formation by direct interaction of the OE-bound GalR dimer with the OI-bound GalR dimer results in simultaneous repression of both galETKM operon promoters , P1galE and P2galE ( Aki et al. , 1996 ; Geanacopoulos et al. , 1999 ; Semsey et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The results of transcription repression studies reported here support an antiparallel DNA loop in which the GalR dimer bound to OI stacks upon the one bound to OE , consistent with the previous theoretical calculations ( Geanacopoulos et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Discussion In our attempt to de ® ne the structure of the Gal repressosome , we previously described the isolation and characterization of GalR mutants that bind to DNA operators normally but are defective in potential GalR ± GalR or GalR ± HU contacts during DNA loop formation ( Geanacopoulos et al. , 1999 , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The functional signiﬁcance of strong DNA bending was recently exempliﬁed in a magnetic tweezers experiment in which the interaction between two GalR molecules separated by 115 bp distance was shown to be directly stimulated by the presence of HU ( Lia et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>The models for GalR bound to DNA were taken from the crystal structures of PurR -- DNA complex ( Schumacher et al. 1994 ) , and the model for HU bound to DNA was taken from the complex based on the affinity cleavage data ( Lavoie et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The two operator-bound GalR molecules associate in the presence of the histonelike protein HU resulting in the formation of a DNA loop encompassing the promoters ( Aki et al. , 1996 ; Aki & Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>In contrast , dimeric GalR molecules bound to bipartite operators separated by 11 helical turns associate to form DNA loops on supercoiled DNA only in the presence of HU ( Choy et al. 1995 ; Aki et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>For the GalR and OE-DNA combination , the mean molecular mass in the solution was computed to be about 64 kDa , indicating the presence of a GalR dimer-OE complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>We tested whether HU binds to the GalR dimer-OE complex observed above and , if so , at what stoichiometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Energy transfer experiments suggested that the DNA ends and the tryptophan residues in GalR2 are excluded to AEDANS labels in HU , indicating that HU binds to the GalR2-OE binary complex in the manner shown in the cartoon , i.e. , through GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Energy of DNA bending and twisting for the antiparallel loops Etotal Etwist Ebend Linear DNA + GalR AL1 loop AL2 loop 26.0 28.0 6.4 7.6 21.2 22.2 Linear DNA + GalR + HU bound to hbs ( +6.5 ) AL1 loop AL2 loop 9.4 8.7 5.8 25.5 14.9 13.6 Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>To study any critical GalR -- HU interaction , we performed site-directed mutagenesis of the hupA gene to identify any HU mutants that would be specifically defective in the formation of the repressosome while retaining their ability to-bind to DNA with normal affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Although a specific interaction of GalR and HU was shown to be essential for loop formation , the HU-GalR contact is unlikely to be present in the repressosome ; it appears that GalR piggybacks HU to its architecturally critical binding site on DNA ( Kar and Adhya 2001 ; S. Roy , E. Dimitriadis , S. Kar , M. Geanaco-poulos , M.S. Lewis , and S. Adhya , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Besides induction ( loop breakdown ) of the operon by D ( ) - galactose , which acts by inactivating GalR , the loop must break down for expression in response to anabolic needs resulting from changes in DNA superhelicity or HU concentration ( 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>The fac - cause transcriptional repression of the gal operon , it would be tors required for gal DNA looping in single-molecule experiments possible to detect transitions between short ( looped ) and long ( HU , GalR and DNA supercoiling ) correspond exactly to those ( unlooped ) conformations of single , supercoiled gal DNA molnecessary for gal repression observed both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Taken together , these measurements confirm the widely held hypothesis that GalR and HU induce DNA looping in the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>In this study , we report the physiological and biochemical characterization of a GalR mutant protein , which can repress gal transcription and thus form the Gal repressosome in the absence of HU and supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In addition , a possibility of the gal operon repression mediated by GalR in the absence of HU was shown in the new DNA constructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In the absence of GalR , HU did not bind to the 20 bp DNA duplex in any detectable measure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>In this case HU is necessary for formation of a repression loop with GalR ( Aki and Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>DNA , under slight tension , denatured to facilitate GalR HU-medi-ated DNA loop formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>To further investigate the reason for failure of repression in the AL2 loop , we constructed stereochemical models of the transient GalR -- DNA loops formed in the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>GalR binds to the operators with high afﬁnity .10,11 Two conditions are necessary for realization of a stable loop : negatively supercoiled DNA and the presence of the histone-like nucleoid-structural protein HU .12 During DNA looping , the HU heterodimer binds to an architecturally critical site , hbs13 ( Figure 1 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Geometry of loops Gal loops In contrast to the LacI , the GalR tetramer is likely to have a rather rigid V-shaped structure that is formed via stacking interactions of two GalR dimers with relatively weak afﬁnity for each other .18,19 These properties restrict the conditions ( e.g. phasing and spatial separation of operators along DNA , presence or absence of HU , negative superhelicity etc. ) in which the bound GalR dimers are able to form a loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Recruitment of HU by piggyback : a special role of GalR in repressosome assembly Sudeshna Kar and Sankar Adhya1 Department of Molecular Biology , National Cancer Institute , National Institutes of Health , Bethesda , Maryland 20892-4255 , USA In Gal repressosome assembly , a DNA loop is formed by the interaction of two GalR , bound to two distal operators , and the binding of the histone-like protein , HU , to an architecturally critical position on DNA to facilitate the GalR -- GalR interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In vitro , DNA loopingmediated repression involves binding of the Gal repressor ( GalR ) to two gal operators ( OE and OI ) and binding of the histone-like protein HU to a speciﬁc locus ( hbs ) about the midpoint between OE and OI , and supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Formation of the repression loop by GalR absolutely requires the intervention of the histone-like protein , HU , binding between the two GalR operators and stabilising an antiparallel DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>GalR dimers bound to two spatially separated operators , O and O , speci ® cally interact with the his-E I tone-like protein HU and close the loop in supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The repression of the two gal promoters occurs due to repressosome formation when two GalR dimers bound to the operators tetramerize with the help of HU , resulting in a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In model B , the delivery of HU by piggyback to its destination on the DNA may be followed by one of the two scenarios ( Fig. 8B ) : ( 1 ) After HU binds to the specified position on the DNA , it dissociates from GalR , leaving behind a GalR dimer -- dimer contact required for DNA looping ; ( 2 ) HU binds to DNA while remaining GalR bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>We assume that under normal conditions the DNA loop is formed by transient interaction of operator-bound GalR dimers , and HU plays an architectural role by stabilizing the repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>HU binding is considered to be not strictly sequence-speciﬁc ; HU has strong afﬁ-nity for distorted DNA structure .7,13 It shows tri-partite-binding cooperativity with GalR dimer binding to the operators in DNA looping .34 Our results support the model that DNA looping occurs via a direct interaction of two GalR dimers bound to the operators , and is helped by HU protein binding to the architecturally critical position hbs .12 Probably , increasing the distance between OE and OI reduces the persistence of initially short DNA element , its resistance to torsional and bending distortions , and shifts the energy balance in favor of a more stable GalR tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>As an adaptor , HU binds simultaneously to OE - and OI-bound GalR , which are otherwise incapable of interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>As a bender , it binds to a DNA site between OE and OI and stabilizes a transient interaction between two operator-bound GalR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>During the repressosome assembly , a DNA loop is formed by the interaction of two GalR dimers , bound to two spatially separated operators , OE and OI , flanking the gal promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>A model showing the DNA bent around HU bound to hbs with GalR dimers bound to OE and O facing away from each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>HU binding , in turn , results in co-operative binding of GalR to OE and OI , presumably by an interaction between two DNA-bound GalRs as shown below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In this model , co-operat-ive GalR binding is brought about by an interaction between OE - and OI-bound GalR molecules , resulting in the formation of a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>( ii ) The crystal structure of LacI is known .16,17 ( iii ) LacI tetramer forms stable DNA loops and does not require HU and supercoiled DNA although the latter enhances looping efﬁciency .1,33,36 Besides , LacI has higher speciﬁc binding afﬁnity to DNA than GalR .1,10,33 ( iv ) The gal promoters can be negatively regulated via DNA looping by binding of the LacI repressor to the gal operon in which the OE and OI operators were modiﬁed to be recognized by LacI .29 The C-terminal tetramerization domain of Lac repressor is ﬂexible and , therefore , depending on phasing and separation of operators along the DNA , the LacI-mediated DNA complex can potentially adopt various types of loops .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Footprinting experiments using iron -- EDTA-coupled HU show that HU binding to gal DNA is orientation speciﬁc and is speciﬁcally dependent upon binding of GalR to both OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The site-speciﬁc stable binding of HU to gal DNA is totally dependent upon GalR binding to both operators , OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Since HU does not bind to gal DNA in the absence of GalR ( 3 ) , HU binding may be dependent on direct GalR -- GalR cooperation and may stabilise an interaction which is weaker in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>( 3 ) Finally , there is a tripartite cooperativity between GalR and HU in binding to gal DNA ; binding of HU to hbs is absolutely dependent on binding of GalR dimers to both operators and HU binding , in turn , results in increasing the strength of GalR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Since sitespeciﬁc HU binding to gal DNA is strictly dependent upon GalR interaction with both operators , we reinvestigated the co-operative binding of GalR using the ` HU-nuclease ' - dependent DNA cleavage as a diagnosis of GalR cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>binding of HU to hbs is totally dependent upon the binding of GalR to the operators , a protein-protein interaction between GalR and HU may explain the cooperativity in DNA binding ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>In one model , the strong binding of GalR or GalR -- HU complex to the operator sequences precedes HU binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In this model , HU-bound GalR binds to the gal operators , thereby bringing HU to the vicinity of the gal region and facilitating its DNA binding , which in turn stabilizes DNA-bound GalR tetramerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In summary , we propose that direct interaction of op-erator-bound GalR dimers can close the gal DNA loop on its own and create a high-affinity HU-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The GalR -- HU contact that results in cooperative binding of the two proteins to DNA may be transient and absent in the final repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The synergistic binding of GalR and HU , the strength of HU binding , and the specificity of GalR and HU in the repressosome may originate from a functional interaction between GalR and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The simplest interpretation of such co-operative binding is an interaction between two GalR dimers in the presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Repression of P2 requires the formation of the repressosome through cooperative binding of GalR to both OE and OI , and of HU to the interoperator segment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>This and the following results suggest that the binding of GalR to both OE and OI is required to facilitate HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In turn , HU helps co-operative binding of GalR to OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Immunoprecipitation of GalR and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>GalR + HU + OE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Binding of HU was totally dependent upon the binding of GalR to O and O ; conversely , HU E I increased the afﬁnity of GalR to O and O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>GalR dimer is also speciﬁc for inducing sitespeciﬁc binding of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Thus , GalR is speciﬁc in assisting HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The GalR and HU mediated repression of the two gal promoters requires both gal operators and is sensitive to the addition of inducer , D-galactose ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The site-speciﬁc cleavages were attenuated by the addition of inducer , D-galactose ( lanes 5 and 10 ) , clearly indicating that GalR binding to the operators stimulates HU binding to speciﬁc sites in gal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The properties of site-speciﬁc stable binding of HU parallel the properties of HU in transcriptional repression -- the requirement for binding of GalR to both operators , the requirement for supercoiled DNA and the sensitivity to the presence of inducer , D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Conversely , homologous repressors , like GalS and LacI , when provided with their cognate operators , can not replace GalR in bringing about HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>HU binding is entirely dependent upon binding of GalR to the two operators and is sensitive to exogenous signal D-galactose , thus revealing a novel way by which a HU-containing higher-order complex responds to transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>As shown in Figure 5A , lane 2 , LacIadi and HU complex formation was barely detectable under the lowest salt concentration that was used in reactions involving GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>( A ) A fixed concentration of GalR ( 5 nM ) was mixed with increasing concentrations of wild-type and mutant HU , as indicated in the figure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Equimolar amounts of GalR and HU were mixed together in buffer containing varying concentrations of salt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>After induction of GalR , the cellular extracts were analyzed for GalR -- HU complexes by immunoprecipitating with anti-His antibody and immunoblotting with antiserum specific for HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In this paper , we demonstrate and determine the thermodynamic parameters of several of these interactions , GalR dimer-OE , GalR tetramerization , HU-GalR , and HU-GalR-OE interactions , by analytical ultracentrifugation , fluorescence anisotropy , and fluorescence resonance energy transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>In this paper , we report physical interactions between the different interacting partners , including GalR and HU , of the proposed repressosome complex , and determine the energetics of these interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>In this paper , we provide struc-ture-based genetic and biochemical evidence to show a specific and functional GalR -- HU interaction , both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>A GalR -- HU interaction was also demonstrated by sedimentation ultracentrifugation analysis and fluorescence studies ( S. Roy and M. Geanacopoulos , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>11 when Spot42 sRNA is expressed at full strength ( not repressed by cAMP -- CRP ) and the gal operon is transcribed at the maximum rate possible in the absence of cAMP -- CRP and GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Regulation of regulators Autoregulation of GalR allows its low-level expression in the absence of cAMP -- CRP and D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Both promoters are repressed in the absence of galactose by GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Transcription is expressed as the percentage of that which was obtained in the absence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Both GalR and GalS are known to respond to inducers , but the presence of saturating concentrations of inducer does not allow induction of the gal operon to the maximal level ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>In the absence of GalR , the gal operon is induced two - to fourfold by the addition of D-galactose or D-fucose ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>GalR , the ﬁrst of these proteins to be identiﬁed and which has been puriﬁed , functions as the major repressor of the gal operon and is sufﬁcient for repression of its genes in the absence of galactose ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR regulates transcription of the gal operon in the absence of GalS , and GalS regulates transcription of the mgl operon ( 32 ) and the galS gene in the absence of GaIR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>It has lower afﬁnity for the bipartite operators controlling the expression of the gal operon and , like GalR , is incapable , in the absence of other factors , of repressing transcription of the upstream gal operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid-associated protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Transcription of the mglBAC operon is regulated in a similar way , except that it is only partially repressed by GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>On the other hand , GalR increased the af ® nity of RNA polymerase at the P2 promoter ; the half-maximal saturating concentration of RNA polymerase shifted from 60 nM in the absence of GalR to 30 M in the presence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Effect of varying concentrations of RNA polymerase on gal RNA synthesis in the absence and presence of GalR ( 100 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>We investigated a 5 potential kinetic trap by examining the rates of isomerization upon RNA polymerase binding in the absence and in the presence of GalR by the use of 2-AP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>RNA polymerase binding at 4 8C to the P1 P2 template in the presence of 200 nM GalR ( B ) C K and in its absence ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Figure 3 shows the isotherms of RNA polymerase binding to P1 in the absence C and in the presence of saturating concentrations of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>However , in the presence of D-galactose , elevated levels of galR transcription produce higher levels of the GalR -- D-galactose complex , but not active GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The speci ® c effect of GalR at one promoter is independent of the presence of the other promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>The results of the transcription in the presence of GalR only showed that the GalR dimers could interact directly and not through an adaptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The level of P2 transcription was the same both in the absence and presence of GalR ; the addition of HU also did not change the amount of P2 transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The level of P2 transcription in the presence of T19D , for example , was similar to that with GalR alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The rate of ﬂuorescence increase by the wild-type GalR increased about 2.7-fold in the presence of D-galactose similar to the results shown in Figure 4 ( B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>The effect of S29R GalR in the absence and in the presence of the inducer D-galactose on the slowest of the open complex formation steps , as detected by 2-AP ﬂuorescence , was also tested with a template labeled at position K5 , at 37 8C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>The equilibrium dissociation constants in the absence and in the presence of GalR were ( 1.14 G0 .21 ) !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>The ﬂuorescence increase followed similar initial kinetics both in the absence and in the presence of GalR with rate constants 1.59 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>K3 K1 K3 K1 However , the amplitude of the ﬂuorescence appeared to be greater in the presence of GalR than in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Why S29R GalR lowers the rate of ﬂuorescence increase ( in the presence and in the absence of D-galactose ) is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Although weak cleavages by HU-nuclease were observed in the absence or presence of the reducing agent Na-ascorbate , some of which could be artifacts of primer extension assay ( lanes 1 , 3 , 6 and 8 ) , the addition of GalR dramatically enhanced site-speciﬁc cleavages on both gal DNA strands at speciﬁc sites ( lanes 4 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Interestingly , the DNA cleavages caused by the EPD -- Fe ( III ) - derivatized HU pre-bound to the DNA in the presence of GalR were not affected by the subsequent addition of a 5-fold excess of ` cold ' HU before the Na-ascorbate treatment ( lane 4 ) , whereas pre-bound ` cold ' HU inhibited signiﬁcantly the cleavages by a 5-fold excess of derivatized HU added later ( lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The nucleoprotein complex of GalR , HU and gal DNA prevents open complex formation We tested whether RNA polymerase binds to gal promoter in the presence of GalR and HU by DNase I protection analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In other words moderate enlargement ( up to 84 bp ) of the distance between the gal operators was an advantage to the loop formation by GalR both in the absence and presence of HU while further increase more diminished this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The most advantageous length of the insert in terms of loop formation among the insert-contain-ing templates in the presence of GalR and HU proteins was 84 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Results and Discussion DNA molecules that were negatively supercoiled by at least 3 % ( 0.03 ) and stretched with a force ( F ) of 0.88 pN intermittently switched between two conformations in the presence of both GalR and HU ( Figs. 1b and 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Our ecules under moderate tension only in the presence of the GalR single-molecule experiments revealed that negatively supercoiled and HU proteins ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Coimmunoprecipitation using purified GalR and HU with antiserum specific to GalR consistently revealed the presence of HU in these immunocomplexes , as ascertained by Western blotting with HU-specific antiserum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Map of HU-nuclease cut sites observed in the presence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The concentration of GalR and HU was 80 nM as shown on the top .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Concentration of GalR was 80 nM ; concentrations of HU were 40 nM , 80 nM , and 140 nM , as shown at the top .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The footprinting reaction was performed in the presence of GalR with cold HU ( 100 nM ; lane 1 ) or iron -- EDTA-derivatized HU ( ~ 20 nM ; indicated in box in lanes 2 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The assays for each plasmid were performed as described in Materials and Methods and included three transcription reactions : a control reaction without repressor GalR or HU ; a reaction in the presence of 80 nM GalR ; or in the presence of both 80 nM GalR and 80 nM HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The transcription reactions were performed in the absence and presence of 80 nM GalR and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The transcription reactions were performed in the absence and presence of 80 nM GalR and HU with OE -- OI DNA template , and in the presence of 80 nM LacI with OLE -- OL I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In a purified system , P2 transcription was enhanced by 80 nM GalR in the absence of HU , as observed previously , and D-galactose eliminated the activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>However , as was found in vivo , the puri ® ed R282L mutant GalR repressed transcription in the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In the presence of mutant GalR protein , however , transcription from both promoters decreased dramatically , in both the absence and presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>However , the mutant GalR protein , R282L , ef ® ciently repressed transcription from the altered DNA template regardless of the presence or absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>We verified that no telegraphic signals were observed in the absence of GalR and or HU ( data not shown ) , or in the presence of 40 nM D ( ) - galactose , an inducer of gal transcription ( Fig. 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>( a ) Repression of in vitro transcription of the gal operon mediated by GalR in the absence and presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Supercoiled P1 -- P21 gal template was incubated in the presence or absence of GalR , HU or RNA polymerase as indicated under conditions of the in vitro transcription except that dNTPs were omitted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Regulation by the iso-regulatory proteins of the gal regulon , GalR and GalS , was compared in in vitro transcription assays performed on the pRPGSM template in the presence of heat-unstable protein ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In vitro transcription on the resulting plasmid DNAs in the presence of wild-type GalR protein showed that GalR homodimers could form repressosome and repress the gal promoters in the presence of HU ( Fig. 3 , A16T lanes 3,6,9,12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In the presence of DMS , GalR resulted in protein bands with molecular weights of monomers , dimers , trimers and tetramers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Experiments in which one protein was titrated in the presence of a constant concentration of another protein or proteins , 125 nM RNAP , 200 nM CAP ( 100 µM cAMP ) , and 23.5 nM GalR were the constant `` saturating concentrations '' used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The energetic interactions between GalR , CAP , and RNAP were determined by conducting a series of DNase I footprint titration experiments of each protein in the absence and separately in the presence of a saturating concentration of the other protein ( s ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>( 1997 ) 272 , 293 ± 300 Repression and Activation of Pro oter-bound RNA Polym Activity by Gal r m epresso erase R Hyon E. Choy1 , Robert R. Hanger1 , Tsunehiro Aki1 , Michael Mahoney1 Katsuhiko Murakami2 , Akira Ishihama2 and Sankar Adhya1 * 1 Laboratory of Molecular Biology , National Cancer Institute , National Institutes of Health , Bethesda MD 20892-4255 , USA 2 Department of Molecular Genetics , National Institute of Genetics , Mishima Shizuoka 411 , Japan * Corresponding author Introduction By binding to the DNA site OE at position ÿ60 .5 in the gal operon , the GalR protein activates transcription from the P2 promoter located on the opposite face of DNA ( position ÿ5 ) and represses transcription from the P1 promoter located on the same face ( position 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Gal repressor ( GalR ) , which is a dual function regulator , represses transcription from the P1 promoter and enhances transcription from the P2 promoter when it binds to the operator OE in the gal operon of Escherichia coli ( Choy & Adhya , 1992 ; Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>In contrast to the in vivo observation , GalR represses P1 inefﬁciently and activates P2 in vitro with puriﬁed components , and does not form the proposed DNA loop ( Mandal et al. , 1990 ; Choy and Adhya , 1992 ; Goodrich and McClure , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Furthermore , GalR , like LacIadi , represses transcription from P1 somewhat inefﬁciently and activates that from P2 ( Choy and Adhya , 1992 ; Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Results gal repression requires HU In a puriﬁed transcription system , GalR activates the P2 promoter and represses , albeit less efﬁciently , the P1 promoter ( Choy and Adhya , 1992 ; Goodrich and McClure , 1992 ; Choy et al. , 1995a ; Figure 1 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In vivo , however , GalR , like LacI and unlike LacIadi , represses gal transcription from both P1 and P2 ( Haber and Adhya , 1988 ; Mandal et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Because , in the absence of DNA looping , GalR activates transcription from the P2 promoter without the inducer being present ( Choy and Adhya , 1992 ; Aki and Adhya , 1997 ) , induction of the gal operon by this scenario would be considerably more rapid than in the case where the stability of the GalR ± GalR interface is not modulated by inducer binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Thus , although a LacI-mediated looped complex can inhibit RNAP binding and repress transcription at the gal operon ( Haber & Adhya , 1988 ; Choy & Adhya , 1993 ) , GalR does not function by this mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Effect of GalR bound to OE on transcription from gal P1 and P2 promoters using reconstituted RNA polymerases carrying different amino acid substitutions at aCTD Enhancement of P2 The ® nding of GalR enhancement of P2 parallels the activation of transcription of the E. coli lac promoter by cAMP receptor protein ( CRP or CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>As previously shown ( 8 ) , GalR selectively inhibited transcription from P1 of the gal operon in vitro and activated transcription from P2 , although its effect on transcription from P2 is biphasic , with the blockage of elongating transcripts by repressor bound to OI causing repression at higher GalR concentrations ( 8 ) ( Fig. 3A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Like GalR , GalS selectively inhibited transcription from P1 , albeit more weakly , and caused a slight activation of transcription from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Our results also show that GalS , like its homolog , GalR , is a dimeric protein which in binding to the bipartite operators of the gal operon selectively represses its P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Nevertheless , like GalR , GalS selectively represses the downstream promoter , P1 , of the gal operon and slightly activates the upstream promoter , P2 , in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Both GalR and GalS selectively repress the gal operon transcription from the P1 promoter , while transcription from P2 is activated slightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>In the absence of GalR , RNA polymerase formed distinct binary complexes with both P1 ( Figure 5a , Amino acid GalR repression GalR activation substitution in aCTD at galP1 at galP2 WT L260W E261W L262W T263W V264W R265W ÿ ÿ R265A ÿ ÿ R265K ÿ ÿ S266W A267W N268W ÿ ÿ C269W ÿ ÿ L270W ÿ ÿ The results of columns 2 and 3 were obtained from the GalR titration experiments with pSA544 similar to that shown in Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>This is in agreement with the DNase protection results , which showed that open complexes are formed at P2 at lower RNA polymerase concentrations in the presence of GalR ( Choy et al. , 1995a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>By screening of sitedirected substitutions in a de ® ned surface area of GalR previously surmised to participate in a GalR ± GalR interaction , we isolated a GalR mutant , R282L , which showed stronger repression of the P2 promoter relative to that of wild-type GalR without changing P1 repression on indicator plates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>It is clear that 50 nM wild-type GalR repressed P1 efﬁciently , while S29R was incapable of causing any repression even at 100 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Upon addition of GalR , transcription from the P1 promoter in the wild-type gal operon is repressed , while the P2 transcription is enhanced .31 As in the previous cases of enlarged gal DNA , P2 is repressed , not activated , by GalR ( although repression of P1 was weaker ) in the templates with 83 bp , 84 bp and 85 bp inserts ( Figure 2 ( b ) , second lane of panels 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Presence of both GalR and wild-type HU caused simultaneous repression of both P1 and P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The wild-type GalR protein , as expected , enhanced P2 and repressed P1 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>As expected , the wild-type GalR enhanced P2 transcription with a concurrent partial repression of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>However , in the templates containing the inserts , addition of GalR alone showed not only repression of P1 but also partial repression of P2 ( Figure 2 ( a ) , panels 2 -- 9 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>With increasing length of inserts , transcription repression in the presence of GalR virtually did not change for P1 while that for P2 gradually increased reaching a maximum in the 84 bp insert and decreasing thereafter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The presence of GalR alone caused repression of P1 with a concurrent activation of P2 , as expected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Moreover , increasing concentrations of GalR inhibited the P1 RNA synthesis in the absence of an active P2 promoter ( Figure 3b ) and enhanced the P2 transcription in the absence of an active P1 promoter although the P2 enhancement effect was slightly less than that found in the wild-type template ( Figure 3c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Relative to the amounts made in the absence of GalR , P1 and P2 RNA were plotted as a function of repressor concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>The reduction of GalR-mediated stimulation of the P2 promoter is attributed to the fact that P2 RNA synthesis is already higher from the P1ÿP2 DNA as compared to that from the wildtype DNA in the absence of GalR ; the P1 mutation present in the P1ÿP2 DNA has increased P2 RNA synthesis about twofold in an unknown way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>The fact that GalR stimulated P2 in the absence of P1 and repressed P1 in the absence of P2 suggests that GalR regulates each promoter independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>These results indicated the independence of P1 and P2 in responding to the regulatory effect of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>The results of transcription from this DNA template showed clearly that chan-ging the orientation of the two promoters reverse the regulation ; GalR repressed P2 and activated P1 ( Figure 3d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Both GalR mutants resulted in higher P2 promoter activity compared to that in the wild-type GalR in the absence of the HU protein , showing that the tetramerization interface of GalR dimers is involved in the residual repression of the P2 promoter ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>We demonstrated the ability of GalR to repress the P2 promoter by specific interaction of repressor dimers in the absence of HU ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In the presence of GalR and HU , - glucuronidase synthesis is repressed from the P2 ∼ gus fusion ( Lewis et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>To investigate the mechanism and specificity of the weak HU-indepen-dent looping repression in the gal system , we investigated the effect of the GalR mutants GalRT322R and GalRH327R , defective in tetramerization , on the P2 activity in the absence of HU ( Geanacopoulos et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Non-inducibility of GalR R282L The addition of inducer D-galactose to cells expressing the wild-type GalR derepressed P2 transcription , as expected ( Lewis et al. , 1999 ; Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>( B ) Percentage of the OE DNA bound by the wild-type ( squares ) and the R282L mutant ( circles ) GalR proteins in the presence of the indicated amounts of D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Stronger repression of the P1 promoter in the presence of wild-type protein ( Fig. 2E ) can also be an indication of cooperativity between the OE - and OI-bound GalR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>However , the R282L mutant GalR showed a non-inducible phenotype ; the mutant protein maintained repression of P2 in the presence of D-galactose ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Repression of P2 in cells expressing the wild-type and R282L mutant GalR was compared in the presence of coumermycin ( Figure 2C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In contrast to wild-type , the mutant GalR showed repression of P2 in the presence of coumermycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>( B ) Effect of hup deletions on the looping repression of the P2 promoter in the presence of the wild-type GalR ( squares ) and R282L GalR ( circles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The presence of wild-type GalR and HU had the same effect , since the repressosome structure can not be formed on linear template ( Aki et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>( A ) Transcription of galP1 and galP2 promoters in the presence of GalR or wild-type HU and HU mutants , S17P , K18A , or T19D .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The presence of GalR ( wild-type or R282L ) and HU resulted in complete repression of both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The improper helical phasing of the operators inhibited the loopingmediated repression of P2 with wild-type GalR in the presence of HU ( Figure 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The amino acid residues S17 , K18 , and T19 in HU , which when substituted showed defect in repression of gal transcription and in interaction of GalR with HU , are located contiguously in a small turn between the first and second alpha helices ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The repression complex was sensitive to the inducer D-galac-tose , which acts by altering the activity of GalR allosterically and thus deforming the repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The repression observed for aceB transcription by the IclR box I-bound IclR is , however , more drastic than the GalR , f29 p4 and PhoP cases , because the repressor IclR interacts with the aCTD and induces dissociation of the open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>Feasibilities of these trajectories were tested by in vitro transcription repression assays , first by isolating GalR mutants with altered operator specificity and then by constructing four different potential loops with mutant GalR heterodimers bound to specifically designed hybrid operators in such a way as to give rise to only one of the four putative trajectories .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>We isolated a negative control GalR mutant , S29R , which while bound to the operator DNA was incapable of repression of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>A strong prediction of such a model of repression is that it would be possible to isolate a negative control ( nc ) GalR mutant that could bind well to the cognate operator OE , but would be incapable of repressing P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>If GalR mutant R282L resulted in increased af ® nity of the bound dimers to form a tetrameric structure , one may expect to form the DNA loop and thus bring about P2 repression with reduced DNA supercoiling or improper helical phasing of the operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>These data demonstrate that the residual P2 repression is the result of specific interaction of operator-bound GalR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Because helical dependence is a critical factor in activation by MarA ( 19 ) , the transcriptional outcome , repression versus activation , may be dependent on which DNA face the protein binds as is the case for GalR ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>To measure the effect of the mutations on looping-mediated repression by GalR , b-glucuronidase activities were measured from cell extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>However , repression of the PmglB promoter by GalR was less efficient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>As mentioned before , the repression of P1 by GalR is a-CTD-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Deletion of the a-CTD abolished repression by GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Because GalR has low affinity to the mgl operator ( Geanacopoulos and Adhya , 1997 ) , partial repression of the PmglB promoter by GalR can be removed by a small amount of externally added or internally synthesized D-galactose in glucose-starving cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Conclusions Members of the LacI -- GalR family of transcriptional-repressors are usually functional as dimers , although in some cases , repression depends on the dimerization of dimers or interactions with other proteins , such as CRP ( Weickert & Adhya , 1992 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>However , the LacI protein , LacIY17V , Q18A , which carried the GalR recognition sequences , was shown to have DNA-binding specificity of GalR in in vitro DNA-bind-ing assays as well as in in vivo transcription repression ( Lehming et al. 1990 ; Barker et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Similarly , tetramerization of hybrid GalR via the LacI flexible Cterminal heptad repeats results in poor repression compared to dimeric GalR in vivo ( Barker et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The failure of TyrR and RNAP to-bind together on the tyrP fragment lacking the upstream RNAP binding site made it less likely that repression of the tyrP promoter involved an interaction between the TyrR protein and the a subunit of RNAP , as has been seen for the repression of gal and lac promoters by GalR , aceB promoter by IclR and treRP3 promoter by PhoP , where inhibition of transcription initiation occurs following the binding of RNAP to the promoters ( Choy et al. , 1995 ; Yamamoto et al. , 2002 ; Yamamoto and Ishihama , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15049824" target="_blank">15049824</a>]<br></li>
<li>Coumermycin inhibits P2 repression by GalR without affecting the intrinsic strength of the promoter ( Lewis et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>For instance , the GalR protein represses transcription from the galP1 promoter , but its direct contact with the aCTD is needed for this repression ( Choy et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>Moreover , both GalR and PurR are known to bend their cognate sequences ( Schuma-cher et al. , 1994 ; Kuhnke et al. , 1989 ; Zwieb et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Repression is abolished when D-galactose binds to GalR ( Nakanishi et al. , 1973 ; Majumdar and Adhya , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>This seemed to imply another link between galactose and amino sugar metabolism as exempliﬁed by NagC participating with GalR and GalS in the repression of galP in E. coli ( El Qaidi et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Mgl-BAC is involved in the galactose transport , GalR and GalS negatively regulate MglBAC , and GalR and GalS are regulated by the global regulator CRP ( Semsey et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>The binding curves obtained for the independent binding of CAP and GalR were described , as expected , by the Langmuir binding expression ( Dalma-Weiszhausz & Bre-nowitz , 1992 ; Brenowitz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>There is no obvious overlap of the two proteins in the model , consistent with the absence of any detectable interaction between GalR and CAP upon binding to the gal operon ( Dalma-Weiszhausz & Brenowitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The binding of CAP to its specific sequence in the gal operon has no effect on the binding of GalR to either or to both operator sequences ( Goodrich & McClure , 1992 ; Dalma-Weiszhausz & Bre-nowitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Previous studies have shown that CAP and GalR bind to the gal operon independently ( Goodrich & McClure , 1992 ; Dalma-Weiszhausz & Brenowitz , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>GalR and cAMP -- CRP can bind simultaneously to their cognate binding sites ( Adhya , 1989 ; Dalma-Weiszhausz and Brenowitz , 1992 ; Goodrich and McClure , 1992 ) , cancelling each other 's effect , and allowing transcription of both promoters to levels similar to their intrinsic activities ( Lewis , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Several studies have shown that GalR and RNAP can bind simultaneously to the gal regulatory region ( Busby et al. , 1987 ; Kuhnke et al. , 1989 ; Adhya & Majumdar , 1987 ; Goodrich & McClure , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>This is consistent with previous work , which showed that Occ ~ OLL ~ was partially repressed by GalR and OEc OILwas partially repressed by LacI ( Haber and Adhya 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>Because of this non-inducible nature , we tested the effect of the inducer on the OE ± GalR complex in vitro by electrophoretic mobility shift assay ( EMSA ; Majumdar and Adhya , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Interaction Reference Galactose transport by GalP 1 2 Riordan and Kornberg ( 1977 ) Robbins et al. ( 1976 ) Geanacopoulos and Adhya ( 1997 ) Geanacopoulos and Adhya ( 1997 ) This work Geanacopoulos and Adhya ( 1997 ) Weickert and Adhya ( 1992 ) Geanacopoulos and Adhya ( 1997 ) Weickert and Adhya ( 1992 ) Majumdar and Adhya ( 1984 ) ; Majumdar et al. ( 1987 ) Weickert and Adhya ( 1993b ) Weickert and Adhya ( 1993b ) This work This work This work Polayes et al. ( 1988 ) Rice and Dahlberg ( 1982 ) Weickert and Adhya ( 1992 ) Aki et al. ( 1996 ) ; Choy and Adhya ( 1992 ) Choy and Adhya ( 1992 ) Aki et al. ( 1996 ) Choy and Adhya ( 1992 ; 1993 ) Choy and Adhya ( 1992 ; 1993 ) Adhya ( 2003 ) ; Ullmann et al. ( 1979 ) Adhya ( 2003 ) ; Ullmann et al. ( 1979 ) Hucho and Wallenfels ( 1971 ) Møller et al. ( 2002 ) Sherman and Adler ( 1963 ) Kurahashi and Sugimura ( 1960 ) ; Wong et al. ( 1977 ) Wilson and Hogness ( 1964 ) Weissborn et al. ( 1994 ) Joshi and Handler ( 1964 ) Hogema et al. ( 1997 ) Hogema et al. ( 1998 ) Galactose transport by the Mgl system cAMP -- CRP activates PgalP cAMP -- CRP activates PmglB GalS represses PgalP GalR represses PgalP GalS represses PmglB GalR represses PmglB D-galactose binding inactivates GalS D-galactose binding inactivates GalR 3 4 5 6 7 8 9 10 cAMP -- CRP activates PgalS GalS autorepression 11 12 13 14 15 16 17 18 19 GalS represses PgalR GalR represses PgalS GalR autorepression cAMP -- CRP represses spf transcription spf encodes Spot42 RNA GalS represses P1galE GalR represses P1galE by contact inhibition and DNA looping GalR binding to OE activates P2galE under non-looping conditions GalR represses P2galE by DNA looping 20 21 22 23 24 25 26 27 28 29 cAMP -- CRP activates P1galE cAMP -- CRP represses P2galE P1galE makes equimolar expression of the galETKM operon genes P2galE expresses primarily the promoter proximal galE gene ( natural polarity ) GalM catalyses the b-D-galactose ↔ a-D-galactose reaction Spot42 RNA inhibits translation of galK GalK catalyses the a-D-galactose + ATP ↔ galactose-1-phosphate + ADP reaction GalT catalyses the galactose-1-phosphate + UDP-glucose ↔ glucose 1-phosphate + UDP-galactose reaction GalE catalyses the UDP-galactose ↔ UDP-glucose reaction GalU catalyses the UDP-glucose ↔ glucose 1-phosphate reaction Pgm catalyses the glucose 1-phosphate ↔ glucose 6-phosphate reaction Glucose 6-phosphate decreases the cAMP -- CRP level Galactose transport is inhibited by inducer exclusion in the presence of glucose 6-phosphate 30 31 32 33 34 Numbering refers to interactions in Fig. 6A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Regulation of the P2galE promoter , located 5 bp upstream of P1galE , is opposite ; transcription from this promoter is further enhanced by OE-bound GalR and repressed by cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The GalR -- OE complex also prevents P1galE activation by the cAMP -- CRP complex ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Contact inhibition by GalR prevents cAMP -- CRP activation of P1galE , and reduces P1galE activity by about 75 % in the absence of cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">GalR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The interaction model that best fit the data was 2GalR + OE S GalR2OE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Specifically , the helix-turn - tide matches a t positions 96-111 and 184-199 to the purF helix motif ( 34 ) has been ascribed to GalR positions 10-32 ( as numbered in Fig. 10 ) and the correspondingaligned resi - F. J. Schendel , E. Mueller , J. Stubhe , A. Shiau , andJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058704" target="_blank">3058704</a>]<br></li>
<li>Since GalR apparently did not antagonize closed complexes formation at P1 , we studied two different base-pair opening rates using 2-AP substitutions at the corresponding positions in or around the K10 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>However , with regard to affinities , the GalR interaction is better compared to less stable interactions , for example to the lacO1 -- lacO3 loop ( in particular see 37 ) or to the lacO2 -- lacO3 interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>These contacts , observed in stationary cells , might include the chiP GalR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Based on these results and previous demonstration that repression requires the C-terminal domain of the a subunit ( a-CTD ) of RNA polymerase , we propose that GalR establishes contact with a-CTD at the last resolved isomerization intermediate , forming a kinetic trap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>OE-bound GalR represses P1galE transcription by contacting the C-terminal domain of the a subunit of RNA polymerase , and inhibiting open complex formation ( Choy and Adhya , 1992 ; Choy et al. , 1995 ; Roy et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Formation of GalR-OE-RNA polymerase ternary complex DNase protection results showed that GalR enhances open complex formation at P2 and helps RNA polymerase to form a heparin-resistant intermediate between the closed and open forms at P1 ( Choy et al. , 1995a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>We propose that OE-bound GalR enhances open complex formation at P2 by increasing the af ® nity of RNA polymerase and inhibits the conversion of closed to open complex at P1 by directly contacting RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Our demonstration of the inability of GalR to show any regulatory effect when transcription reactions were carried out with RNA polymerases carrying speci ® c mutations strongly supports the notion that a protein-protein contact between OE-bound GalR and the aCTD of RNA polymerase is involved in the enhanced binding of RNA polymerase to P2 ( cooperative binding ) and inactivation of an intermediate complex at P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Our results show that substitutions in the same interval render RNA polymerase poorly responsive or non-responsive to GalR at P2 , suggesting that this interval is involved in aCTD-GalR and/or aCTD-DNA interactions important for GalR-dependent transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Specific amino acids of the a-subunit of RNA polymerase are involved in both activation and repression by GalR We previously showed that both activation and repression by OE-bound GalR require an intact aCTD of the RNA polymerase molecule , suggesting the involvement of aCTD region in GalR action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Both enhancement and repression appear to require the same interaction between RNA polymerase a-subunit and GalR and/or the same interaction between RNA polymerase a-subunit and DNA in the ternary complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Here , we provide evidence that suggests that both repression of P1 and enhancement of P2 by GalR depends upon interactions involving the same amino acid residues in aCTD and we demonstrate the formation of GalR-DNA-RNA polymerase ternary complex at both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>It seems likely that a non-hydrophobic residue at position 282 in wild-type GalR contributes to the reversibility of the system by making the dimer ± dimer interaction weaker .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>GalR contains a single tryptophan residue , and its interaction with galactose has been analyzed by tryptophan ﬂuorescence ; at pH 8.0 , the dissociation constant of GalR for D-galactose is 16 3 1025 M ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR monomer contains a single tryptophan residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>When we translated the 5 ' end of the AS.typhimwrium my1 sequence in the same direction as the my1 genes , we found a potent.ial reading frame in which a stretch of 18 amino acid residues had 15 identical t.o t * he ( ` - t , erminal amino acid residues of Gals , including seven in a row that were also iderl-t ' ical with the GalR ( ' terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>All three amino acids have solvent-exposed side chains and are likely candidates to contact GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>These looping-defective amino acid substitutions in the GalR dimer were located in the DNA-distal domain , which preferably de ® ned a GalR ± GalR rather than a GalR ± HU contact area .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Since the substituted amino acid residue ( R282L ) is located within the genetic-ally de ® ned area of the tetramerization interface ( Geanacopoulos et al. , 2001 ) , and the GalR containing both R282L and another looping-defective substitution failed to form the repressosome ( data not shown ) , we conclude that the R282L change did not create a new GalR ± GalR tetramer interface , but enhanced the strength of the pre-existing interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The 346 amino acid residue Gals protein is shown on top , and the 343 amino acid residue GalR protein below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>These results show that speci ® c amino acid residues in aCTD are involved in both types of GalR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>a substitu tion of a leucine in GalS for isoleucine in GalR at thr eighth position of the recognition helix ( amino acid 22 in the proteins ) along with two other differences in the first helix ( Fig. 4 ) may provide the necessar ) ' st , ruct ' ural differences t ' o confer an abilit ) y to discriminate between gal and my1 operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>Importantly , the overall shape of the GalR mediated loop proved to be indistinguishable from that in the chimerical loop of the same size containing two lac operators ( instead of two gal operators ) and formed by LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>AraR is a hybrid protein that is able to bind operator sites ( AaACTTGT/A/T / ACAAGTaT ) ( 50 ) that presents the typical GntR signature , and its C-terminal domain binds to a carbohydrate effector molecule ( L-arabinose ) as do most of the members of the LacI/GalR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>GalR and GalS were fused to the T25 domain of adenylate cyclase in pKT25 and the T18 domain in pUT18C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Both GalR and GalS lack the leucine-rich C-terminal domain involved in LacI tetramerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>It is suggested that the CAP-RNAP cooperativity is dependent upon contacts made by the complex with the upstream DNA and that GalR binding to OE prevents these contacts from occurring .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>A result of the protein-binding sites being out of phase is that the CAP residues that are implicated in activation at class II promoters are not occluded by the OE-bound GalR from potentially interacting with RNAP ( Figure 8 , greenish yellow residues ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>At least a 90 ° change in the phase of the bound CAP and GalR dimers is required to juxtapose GalR near the `` activation residues '' of CAP , arguing against GalR sterically blocking the RNAP-CAP contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>We can not rule out the possibility that the GalR and GalS protein preparations used in the in vitro studies reported here differ in their activity , although two methods of GalS puriﬁcation produced protein preparations with the same mgl binding activity , and the DNA-binding activity of GalR in our preparations agrees closely with results reported previously ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>All of the proteins in the LacI/GalR family have a conserved leucine residue on the hinge helix , and it is likely that all these proteins pry open the minor groove when bound to operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>GalR and GalS are iso-regulators because of the high similarity in their amino acid sequence , inducer-binding speciﬁcity and DNA-binding site recognition ( Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Leucine 54 plays a key role in sequence-specific binding of PurR ; this same amino acid is conserved in GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>GalR is a member of the GalR-LacI family of gene regulatory proteins .14 In the absence of a crystal structure , GalR monomer was modeled by homologous extension using the X-ray diffraction structures of two related repressors , PurR and LacI ( 31 % and 27 % sequence homology , respectively ) .8,15,16 The GalR dimer was constructed also on the basis of PurR and LacI structures , which have similar pattern of dimerization .8 However , the tetramerization mode of GalR is different from that of LacI , since the former does not have an antiparallel four-helix bundle tetramerization domain found in LacI .16,17 GalR appears to be a dimer in solution at micromolar concentrations .18 Structure-based genetic studies involving hom-ology-scanning mutagenesis identiﬁed amino acid substitutions that are defective in GalR tetrameri-zation but do not affect DNA-binding .19,20 The sensitive amino acids constitute a potential tetramerization interface conﬁned within a `` crescent '' and located on either side of the GalR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The N-terminal 57 resi-dues of RbsR and 58 residues of GalR , homologous to the Lac1 DNA-binding domain , are not shown.The numbers above the alignments refer to the ABP sequence and those below to the RbsR sequence with the last numeral above or below the indicated residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>Since the putative DNA-binding domains of CytR and GalR share extensive homology and the subunits of the two repressors are of equal size , it seemed of interest to examine the two poly-peptides for further homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The apparent homology between the putative DNA-binding domains of GalR and CytR suggests that their sites of action may be related .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>CytR has extensive homology with known DNA-binding proteins like LacI , GalR , PurR , and MalI ( 22 , 24 , 33 ) , and CytR contains a region exhibiting a high level of homology to the a-helix-turn-a-helix motif found in the DNA-binding domain of several DNA-binding proteins ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2170326" target="_blank">2170326</a>]<br></li>
<li>CytR shares extensive amino acid sequence homology with known DNA-binding proteins like GalR , PurR , MalI , and the lac repressor ( Valentin-Hansen et al. , 1986 ; Rolfes and Zalkin , 1988 ; Reid1 et al. , 1989 ) , and specifically , CytR contains a region exhibiting a high level of homology to the helix-turn-helix motif found in the DNA-binding domain of several DNA-binding proteins ( Pabo and Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>To create GalR with altered DNA-binding specificity , we changed the critical first and/or second residues of the recognition helix ( residues 15 and 16 , respectively ) of GalR to the corresponding residues of the homologous LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>PurR belongs to the LacI/GalR family of DNA-binding transcription regulators that show a bipartite structure with an N-terminal DNA-binding domain of the helix-turn-helix ( HTH ) type that is connected through a hinge helix to the larger C-terminal corepressor binding and oligomerization domain .21 E. coli PurR is a 76 kDa homodimer that binds to a 16 bp palindromic operator ( see Figures 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>The proteins in the LacI/GalR family have been shown by either structural studies or sequence similarities to contain headpiece domains with a HTH motif for recognizing an operator and have a core region that is responsible for effector-binding and oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>The DNA-binding domains of the proteins in the LacI/GalR family have strong sequence homology [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>Members of the LacI/GalR family also have signiﬁcant sequence homology throughout the core or effector-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>The protein presents a GntRlike DNA-binding domain and a C-terminal domain that is GntR-like and a C-terminal domain typical of the HTH LacI / GalR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>Since the sequence similarity of the N-terminal DNA-binding domain is higher than that of the C-terminal ligand -- binding domain in the LacI/GalR family , it would appear that the functional divergence of ancestral periplasmic sugar bind ing proteins took place prior to the acquisition of the N-terminal domain [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>Moreover the nag consensus does not agree with conserved central base-pairs ( AA-C , positions 4 to 1 ) found in the LacI-GalR family ( Weickert & Adhya , 1992 ) nor the TGT motif ( positions 8 to 6 ) found in a number of other DNA-binding proteins ( Gicquel-Sanzey & Cossart , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Cra is a member of the LacI-GalR family of transcriptional regulatory proteins with N-terminal helix-turn-helix DNA-binding motifs and C-terminal ligand-binding domains ( Nguyen and Saier 1995 ; Weickert and Adhya 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Although the helix-turn-helix structure are found in other DNA-binding protein families , recognition of the DNA minor groove by the hinge helix is unique to the LacI/GalR family and plays a crucial role in DNA-binding of the repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>( i ) LacI and GalR are closely related members of the paradigm family of the helix-turn-helix DNA-binding proteins35 though they have different stable oligomeric units : LacI tetramer versus GalR dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>TreR of E. coli and the very closely related bacterium Sal-monella typhimurium ( 31 ) is homologous to other repressor proteins of the LacI-GalR family , but significant homology only exists in the amino-terminal part of the protein known to contain the helix-turn-helix motif for DNA-binding ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9148912" target="_blank">9148912</a>]<br></li>
<li>Arrows indicate the RNAP-induced hypersensitive base pair which remains hypersensitive even in the presence of GalR binding to OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Our data indicate that CRP or RNA-binding to the loop are not essential for GalR dimer cooperation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>This additional interaction may be responsible for the tighter binding of RNA polymerase to P1 observed in the absence of GalR when compared to KB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>The amino acid residues S17 , K18 , and T19 in HU , which when substituted showed defect in repression of gal transcription and in interaction of GalR with HU , are located contiguously in a small turn between the first and second alpha helices ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In the modeled structure of the HU dimer , the three residues , S17 , K18 , and T19 , whose replacements resulted in defective transcriptional repression and GalR interaction , are prominently exposed in a peptide turn between two alpha helices on the face of HU near the N-terminal region and farthest from the DNA-binding C-domain and are accessible for interaction with other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>We think that the altered function of the mutant proteins originate from a failed interaction with GalR , a specific interaction essential for the assembly of the repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Although further studies are needed to distinguish between these two models , we note that energetic considerations favor a model in which HU is not in contact with GalR in the final repressosome structure ( Geanacopoulos et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The demonstration of GalR tetramer formation by ultracentrifugation studies ( Table 1 ) , the ability of mutant GalR proteins to interact with each other in the absence of HU using a genetically defined interface ( 23 ) , and the existence of a specific HU-DNA contact between the two operators in repressosome formation ( 13 ) rule out a simple adapter model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>The GalR -- HU contact that results in cooperative binding of the two proteins to DNA may be transient and absent in the final repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The DNA-bound HU could then either maintain or lose contact with GalR in the final repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In principle , either of the two OE-bound GalR dimer-interfaces can interact with any one of the OI-bound dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Tetramerization of two GalR dimers is realized via a V-shaped right-handed stacking interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Looping was presumably mediated by an interaction between the two operator-bound GalR , with HU facilitating the interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The simplest interpretation of such co-operative binding is an interaction between two GalR dimers in the presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Genetic and biochemical studies suggested that the higher order Gal repressosome structure , which governs the transcription of two tandem gal promoters in Escherichia coli , involves sequence-specific binding of GalR repressor dimers to two operators , OE and OI , located 113 bp apart , binding of GalR to the sequence-nonspecific DNA-binding protein HU , interaction of HU with an architecturally critical DNA site between the two operators , and interaction between two DNA-bound GalR dimers generating a loop of the intervening DNA segment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Although a specific interaction of GalR and HU was shown to be essential for loop formation , the HU-GalR contact is unlikely to be present in the repressosome ; it appears that GalR piggybacks HU to its architecturally critical binding site on DNA ( Kar and Adhya 2001 ; S. Roy , E. Dimitriadis , S. Kar , M. Geanaco-poulos , M.S. Lewis , and S. Adhya , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>An adaptor role for HU raised the question of whether DNA-bound GalR directly interacts at all in loop closure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>We assume that under normal conditions the DNA loop is formed by transient interaction of operator-bound GalR dimers , and HU plays an architectural role by stabilizing the repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Hence , the rigid stacking interaction of wild-type GalR dimers may play a key role in creating a high-affinity kinked HU-binding site in the DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Operator-bound GalR dimers close DNA loops by direct interaction : tetramerization and inducer-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">GalR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>GalR is a member of the GalR-LacI family of gene regulatory proteins .14 In the absence of a crystal structure , GalR monomer was modeled by homologous extension using the X-ray diffraction structures of two related repressors , PurR and LacI ( 31 % and 27 % sequence homology , respectively ) .8,15,16 The GalR dimer was constructed also on the basis of PurR and LacI structures , which have similar pattern of dimerization .8 However , the tetramerization mode of GalR is different from that of LacI , since the former does not have an antiparallel four-helix bundle tetramerization domain found in LacI .16,17 GalR appears to be a dimer in solution at micromolar concentrations .18 Structure-based genetic studies involving hom-ology-scanning mutagenesis identiﬁed amino acid substitutions that are defective in GalR tetrameri-zation but do not affect DNA binding .19,20 The sensitive amino acids constitute a potential tetramerization interface conﬁned within a `` crescent '' and located on either side of the GalR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>On the basis of sequence similarities with other bacterial regulatory proteins , FruR has been classi ® ed in the LacI and GalR superfamily of repressors ( Weickert & Adhya , 1992 ) , which contains at least 36 members to date according to the Pro-site dictionary of signatures and potential sites ( Bairoch , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9237914" target="_blank">9237914</a>]<br></li>
<li>ORF1 encodes a protein homologous to repressors including CytR , GalR , and LacI ( Weickert & Adhya , 1992 ) as shown in Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>CytR belongs to the LacI repressor family and exhibits extensive amino acid sequence homology to several other members ( e.g. PurR , LacI and GalR ; Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9155036" target="_blank">9155036</a>]<br></li>
<li>GalR and PurR are members of the `` Gal and Lac repressor '' family of bacterial regulatory proteins and possess amino acid sequences that are 31 % identical and 53 % similar ( Weickert & Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>CytR is a sequence-specific DNA-binding protein and a member of the LacllGalR family of sequence-specific DNA-binding proteins ( Pedersen et al. , 1991 ; ValentinHansen et al. , 1986 ; for review see Weickert and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8221894" target="_blank">8221894</a>]<br></li>
<li>The NagC repressor in fact , shows strong homology to the family of xylose repressors from different Gram-positive species ( Kreuzer et al. , 1989 ; Lokman et al. , 1991 ) , and certain other proteins , notably some sugar kinases ( Angell et al. , 1992 ; Titgemeyer et al. , 1994 ) but not signiﬁcantly to the other major families of prokaryotic repressors , the LacI-GalR family ( Weickert & Adhya , 1992 ) or the DeoR-GlpR family ( van Rooijen & de Vos , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Conclusions Members of the LacI -- GalR family of transcriptional repressors are usually functional as dimers , although in some cases , repression depends on the dimerization of dimers or interactions with other proteins , such as CRP ( Weickert & Adhya , 1992 ; Valentin-Hansen et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>For example , the purine repressor , a member of the LacI/GalR family , recognizes the central portion of the operator by placing a pair of helices in the minor grove of the operator [ 54 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>RbsR is a member of a repressor family including GalR , Lacl , Mali , CytR , and PurR [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>PurR is a member of the lac repressor family , which includes LacI ( 4 , 9 ) , GalR ( 46 ) , RbsR ( 25 ) , MalI ( 36 ) , RafR ( 3 ) , RbtR ( 50 ) , CytR ( 44 ) , EbgR ( 43 ) , CcpA ( 13 ) , and FruR ( 16 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li>
<li>This ubiquitous member ( 253 ) of the LacI-GalR family of transcription factors ( 185 ) binds to conserved cre ( formerly amyO ) sites in the promoter regions of its target genes ( 184 , 207 , 236 , 468 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>J Bacteriol 187:6262 -- 6272 Nguyen CC , Saier MH Jr ( 1995 ) Phylogenetic , structural and functional analyses of the LacI-GalR family of bacterial transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_14">
Phylogenetic , structural and functional analyses of the LacI-GalR family of bacterial transription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9865945" target="_blank">9865945</a>]<br></label>
<input id="evo_similarity_14", type="checkbox">
<div class="EVO_similar">
<li>Phylogenetic , structural and functional analyses of the LacI -- GalR family of bacterial transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_16">
Swint-Kruse , L. , and Matthews , K. S. ( 2009 ) Allostery in the LacI/GalR family : variations on a theme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></label>
<input id="evo_similarity_16", type="checkbox">
<div class="EVO_similar">
<li>Swint-Kruse L & Matthews KS ( 2009 ) Allostery in the LacI/GalR family : variations on a theme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li></div>
<li>PurR belongs to the LacI/GalR family of DNA-binding transcription regulators that show a bipartite structure with an N-terminal DNA-binding domain of the helix-turn-helix ( HTH ) type that is connected through a hinge helix to the larger C-terminal corepressor binding and oligomerization domain .21 E. coli PurR is a 76 kDa homodimer that binds to a 16 bp palindromic operator ( see Figures 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>The protein presents a GntRlike DNA-binding domain and a C-terminal domain that is GntR-like and a C-terminal domain typical of the HTH LacI / GalR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>AraR is a hybrid protein that is able to bind operator sites ( AaACTTGT/A/T / ACAAGTaT ) ( 50 ) that presents the typical GntR signature , and its C-terminal domain binds to a carbohydrate effector molecule ( L-arabinose ) as do most of the members of the LacI/GalR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>CytR is unique among LacI/GalR family members in that binding of its regulatory ligand , cytidine , is linked not to DNA-binding affinity but instead to the cooperative interactions between CytR bound at CytO and CRP bound to CRP1 and CRP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>( i ) LacI and GalR are closely related members of the paradigm family of the helix-turn-helix DNA-binding proteins35 though they have different stable oligomeric units : LacI tetramer versus GalR dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>TreR of E. coli and the very closely related bacterium Sal-monella typhimurium ( 31 ) is homologous to other repressor proteins of the LacI-GalR family , but significant homology only exists in the amino-terminal part of the protein known to contain the helix-turn-helix motif for DNA binding ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9148912" target="_blank">9148912</a>]<br></li>
<li>Cra is a member of the LacI-GalR family of transcriptional regulatory proteins with N-terminal helix-turn-helix DNA-binding motifs and C-terminal ligand-binding domains ( Nguyen and Saier 1995 ; Weickert and Adhya 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Since the sequence similarity of the N-terminal DNA-binding domain is higher than that of the C-terminal ligand -- binding domain in the LacI/GalR family , it would appear that the functional divergence of ancestral periplasmic sugar bind ing proteins took place prior to the acquisition of the N-terminal domain [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>[ 56 ] K. Fukami-Kobayashi , Y. Tateno , K. Nishikawa , Parallel evolution of ligand speciﬁcity between LacI/GalR family repressors and periplasmic sugar-binding proteins , Molecular Biology & Evolution 20 ( 2003 ) 267 -- 277 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>The level obtained is relatively low compared , for instance , with the LacI/GalR HTH family ( 45 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>These proteins , referred to as the LacI/GalR family , appear to have similar structures and regulate transcription in an analogous fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>Cra is a member of the LacI-GalR family ( Feldheim et al. , 1990 ; Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>First , CytR is member of the LacI/GalR family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20000490" target="_blank">20000490</a>]<br></li>
<li>GalS and GalR belong to the GalR-LacI family of regulators ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>MalI exhibited high homology to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>The LacI/GalR family of repressors There are many repressors , which regulate the transcription of inducible genes , and have a high degree of sequence homology with the lactose repressor [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>There is also noticeable homology in the operators of the LacI/GalR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>Members of the LacI/GalR family also have signiﬁcant sequence homology throughout the core or effector binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>The DNA binding domains of the proteins in the LacI/GalR family have strong sequence homology [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>The LacI/GalR family of proteins has sequence and structural homology with the periplasmic sugar binding proteins ( Fig. 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>It can be seen that CytR in this region is more closely related to GalR , DeoR and LacR than to the other proteins , showing the highest homology to GalR ( 70 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3515317" target="_blank">3515317</a>]<br></li>
<li>The FruR protein contains a typical helix-turn-helix motif , common to regulatory proteins , and shows similarity to the Lacd , GalR , PurR , RbtR , and DeoR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>When the amino acid sequences were appropriately aligned , MalI showed 28 % identity to GalR , 21 % to CytR , and 24 % to LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>When small gaps were introduced for optimal alignment , the MalI amino acid sequence was identical to that of GalR at 89 positions ( 28 % ) , CytR at 69 positions ( 21 % ) , Lacd at 78 positions ( 24 % ) , PurR at 61 positions ( 19 % ) , and RbtR at 62 positions ( 20 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Comparison of the deduced amino acid sequence of TreR with that of the LacI ( 27 ) , PurR ( 28 ) , and other repressors of the LacI-GalR subfamily shows a high degree of homology at the NH2-terminal part of the protein sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9148912" target="_blank">9148912</a>]<br></li>
<li>The deduced amino acid sequence of MalI is highly homologous to that of the classical E. coli repressor proteins Lacd , GalR , and CytR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>CytR shares extensive amino acid sequence homology with known DNA-binding proteins like GalR , PurR , MalI , and the lac repressor ( Valentin-Hansen et al. , 1986 ; Rolfes and Zalkin , 1988 ; Reid1 et al. , 1989 ) , and specifically , CytR contains a region exhibiting a high level of homology to the helix-turn-helix motif found in the DNA-binding domain of several DNA-binding proteins ( Pabo and Sauer , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1655726" target="_blank">1655726</a>]<br></li>
<li>The same extracts were also immunoblotted with anti-IHF antibody to check whether GalR coimmunoprecipitated with IHF , a protein that is highly homologous with HU ( Oberto et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The binding of homologous GalS ( an isorepressor of GalR with 86 % amino acid similarities ) to OE and OI did not show the enhanced DNA cleavages by HU-nuclease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The binding of GalR at OE decreases the proportion of complexes at P1 to 29.0 ( 2.0 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by GalR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Gal repressor ( GalR ) , which is a dual function regulator , represses transcription from the P1 promoter and enhances transcription from the P2 promoter when it binds to the operator OE in the gal operon of Escherichia coli ( Choy & Adhya , 1992 ; Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Furthermore , GalR , like LacIadi , represses transcription from P1 somewhat inefﬁciently and activates that from P2 ( Choy and Adhya , 1992 ; Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Transcription of the gal regulon promoters is regulated in different ways by Gal repressor ( GalR ) and Gal isorepressor ( GalS ) and a global regulator , adenosine cyclic-3 ′ ,5 ′ - monophosphate ( cAMP ) receptor protein ( CRP ) , responding to two signals , D-galactose and cAMP respectively ( Weickert and Adhya , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The gal transcription starts from two overlapping promoters , P1 and P2 , which are subject to , among others , negative control by Gal repressor ( GalR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Three trans-acting proteins , GalR ( the gal repressor ) , cAMP receptor protein ( CRP ) - cAMP complex , and the histone-like protein HU , control transcription from these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>171 , No. 9 MalI , a Novel Protein Involved in Regulation of the Maltose System of Escherichia coli , Is Highly Homologous to the Repressor Proteins GalR , CytR , and Lacl JOACHIM REIDL , KARIN ROMISCH , MICHAEL EHRMANN , AND WINFRIED BOOS * Department of Biology , University of Konstanz , D-7750 Konstanz , Federal Republic of Germany Received 13 March 1989/Accepted 10 June 1989 The maltose regulon of Escherichia coli comprises several operons that are under common regulatory control of the MalT activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Except for a B. subtilis transacting protein that is involved in catabolite repression of a-amylase and shows similarity to the LacI and GalR repressors of E. coli ( 168 ) , no other gene products have been identified that are responsible for the regulation of various catabolite-repressible genes in nonenteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>ORF1 encodes a protein homologous to repressors including CytR , GalR , and LacI ( Weickert & Adhya , 1992 ) as shown in Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9135111" target="_blank">9135111</a>]<br></li>
<li>Mall , anove protein involved in regulation of the maltose system of Escherichia coli is highlyhomologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_11">
Reidl , J. , Römisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></label>
<input id="rp_similarity_11", type="checkbox">
<div class="RP_similar">
<li>Reidl , J. , RoÈmisch , K. , Ehrmann , M. , and Boos , W. ( 1989 ) MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692154" target="_blank">10692154</a>]<br></li>
<li>Reidl J , Romisch K , Ehrmann M & Boos W ( 1989 ) MalI , a novel protein involved in the regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20141531" target="_blank">20141531</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and Lacd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2211500" target="_blank">2211500</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1856179" target="_blank">1856179</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to therepressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>MalI , a novel protein involved in regulation of the maltose system of Escherichia coli , is highly homologous to the repressor proteins GalR , CytR , and LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003949" target="_blank">12003949</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618441" target="_blank">12618441</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li></div>
<li>We discuss epistasis of Gals and GalR and possible ways by which the two repressors co-ordinate regulation of galactose-transport and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>In Escherichia coli , the genes responsible for D-galactose-metabolism ( the galactose operon [ galETK ] ) and for high ¬ affinity transport of methylgalactosides , glucose , and galactose ( the mglBAC operon ) are controlled by a repressor , GalR ( 5 , 17 , 18 , 24 ) , and an isorepressor , GalS ( 9 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>The expression of the enzymes involved with the Leloir pathway and galactose-transport activities is regulated by two highly homologous , galac-tose-sensitive repressors , GalR and GalS , encoded by the genes galR and galS , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Regulation of galactose-transport and utilization by GalS and GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>Mgl-BAC is involved in the galactose-transport , GalR and GalS negatively regulate MglBAC , and GalR and GalS are regulated by the global regulator CRP ( Semsey et al. 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>Besides induction ( loop breakdown ) of the operon by D ( ) - galactose , which acts by inactivating GalR , the loop must break down for expression in response to anabolic needs resulting from changes in DNA superhelicity or HU concentration ( 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>In vivo genetic experiments have begun to clarify the functions of GalR and GalS in controlling expression of genes comprising the gal regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>It has lower afﬁnity for the bipartite operators controlling the expression of the gal operon and , like GalR , is incapable , in the absence of other factors , of repressing transcription of the upstream gal operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">GalR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Operator preference of wild-type GalR and GalRA16T Bound by wild-type GalR Bound by GalRA16T Bound by wild-type GalR Not bound by GalRA16T GTGTAATCGTTTTCAC GTGTAAACGTTACCAC GTGTAAACGATAACAC GTGTAAGCGGTACCAC GTGTAATCGCTTCCAC GTGTAATCGATTACAC GTGTAAACGCTCTCAC GTGTAACCGTTGTCAC GTGTAATCGTTAACAC GTGTAAACGCTTGCAC GTGAAAGCGTTTCCAC GTGAAAGCGTTACCAC GTGAAAGCGCTAACAC GTGGTACCGGTTCCAC GTGGAAACGTTCTCAC GTGGAAACGGTACCAC GTGTAAACGCTTTCAC GTGTAATCGATTACAC GTGTAACCGTTACCAC GTGTAAACGTTTTCAC GTGTAATCGATTACAC GTGTAAACGTTCTCAC GTGTAACCGTTTACAC GTGGTACCGATACCAC GTGGTACCGTTCCCAC GTGGTAGCGGTACCAT GTGGTACCGCTGCCAC GTGGTAACGTTAACAC GTGGTAGCGTTAACAC GTGGTAACGTTTGCAC GTGGTAGCGTTACCAC GTGGAACCGTTGCCAC GTGGGAGCGGTTTCAC Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>u birA331 I 35 40 [ Ala Ala Ile Asn LYS His m Gln Gln Thr Val Ser Arg Val Val Asn Val m Asn Gly Ser Asn Leu Phe Asn Ala m His m Gln Tyr Asp Val BirA LacI GalR TrpR CI Cro Lys Asp Val Ala Arg Leu Ar Glu Lys Asn Glu rc $ lu Ser Ala Asp Lys Tyr Lys Thr Ala Lys Asp *** === Ala Ile/Val GUY DNA Backbone Binding Helix DNA Sequence Specificity Helix Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536662" target="_blank">3536662</a>]<br></li>
<li>Moreover , both GalR and PurR are known to bend their cognate sequences ( Schuma-cher et al. , 1994 ; Kuhnke et al. , 1989 ; Zwieb et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>However , the simultaneous occupancy of specific DNA sites by RNA polymerase , CRP-cAMP , and GalR , along with the phased bending of the DNA by each of the bound proteins , and by naturally occurring sequences , does indicate that a complex nuoleoprotein structure exists at the ga/operon promoters under repressed conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Operator sequences ( GTGNNNNCGNNNNCAC ) , randomized at positions N , bound to the wild-type or mutant GalR proteins were enriched by using the SELEX method ( Tuerk and Gold 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>To isolate an operator variant that GalRA16T does not recognize , we performed the same SELEX method of using the random pool of the operator sequences but with wild-type GalR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Sequence analysis predicted a potential GalR and a potential CRP binding site near the transcription start site of the PgalR promoter ( Weickert and Adhya , 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>From threonine in position 7 to leucine in position 26 of MalI ( Fig. 8 ) , the sequence was highly homologous to regions in GalR , Ladl , and CytR that have been implicated to bind operator sequences as helix-turn-helix motifs ( 51 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670898" target="_blank">2670898</a>]<br></li>
<li>Sequences with some homology to the consensus GalR operator can be identi-ﬁed in all three regions galOE2 , 3 and 4 ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>GalR and GalS bind to the same operator sites in the cisregulatory regions of the P1galE , PmglB , PgalP and PgalS promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>GalR bound strongly to the previously identiﬁed operator , galOE1 , and also to a site further upstream , labelled galOE5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>We conﬁrmed that GalR does bind to this site by DNase I footprinting ( site called GalR , Fig. 2C and D , lanes 9 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>A GalR site was found in 6 species ( E. coli , Shigella , Citro-bacter , Chronobacter , Klebsiella and Raoultella ) , which otherwise show a low sequence conservation in the region ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Binding of GalR and GalS proteins in the region upstream from the E. coli chiP promoter Interestingly , the region occupied by ChbR in Salmonella corresponds to a GalR consensus site in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>However , we have no explanation of why these sequences have evolved to bind GalR in some species but ChbR in Salmonella and probably in Serratia where a ChbR consensus site was also found at a similar location but in the opposite orientation relative to Salmonella ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>NagC ( yellow ) , CAP ( bright green ) , ChbR ( cyan ) and GalR ( olive green ) consensus binding sites are indicated by shading .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>S3A ) prompted us to search for GalR consensus binding sites ( using the E. coli galE , galP , galR , galS and mglB GalR binding sites as input for the search matrix ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The sequences of the proposed GalR and NagC operators are given and compared with the GalR consensus ( Weickert and Adhya , 1993a ) and NagC consensus ( Plumbridge , 2001 ; El Qaidi and Plumbridge , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The filled circles above the consensus sequence indicate bases believed to make direct contact with GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>The proteins in the LacI/GalR family have been shown by either structural studies or sequence similarities to contain headpiece domains with a HTH motif for recognizing an operator and have a core region that is responsible for effector binding and oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15950160" target="_blank">15950160</a>]<br></li>
<li>The sequence TGTG AAGTTTTGCAC ( nucleotides 1,670 to 1,684 ) , which immediately precedes the putative rpiB ribosome binding site , conforms well to the consensus operator sequence associated with the binding of regulator proteins such as GalR and the cyclic AMP-catabolite gene activator protein complex ( 12 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576032" target="_blank">8576032</a>]<br></li>
<li>The same relative order is used in bars 10-12 for the binding of GalR to the OI operator sequence in the two-site template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>Interestingly , the length of DNA that separates the GalR binding sites OE and OI ( 113 bp or 38 nm for B-DNA ) is less than the 55 5 nm transition ( z ) observed experimentally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Upon RNAP binding , sequences with altered reagent susceptibilities downstream of OI remain visible even in the presence of GalR at OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>To test this hypothesis , GalR and GalS were added to operator sequences such that binding reached similar degrees of saturation and were then titrated with D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Binding of GalR + , GalRA16T , and GalRV15T , T322R to hybrid operator sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In one model , the strong binding of GalR or GalR -- HU complex to the operator sequences precedes HU binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The length of the sequences protected from DNase I cleavage was similar at all sites for both GalS and GalR , suggesting that they bound as dimers ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>The DNA sequences of each of the known transcriptional regulator binding sites are shown in the line marked GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The base pairs that were changed to alter OE and OI from GalR to LacI recognition sequences are shown in the line marked LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>, , A E C Y C G TT C E gd % ( GaW 0 gal ( + ( GalR ) lac0 E C Z T A U T E A Z A A C A A T T 0 C CT A B C A E A I ~ G C U The EcoRI-labeled fragment , the coding strand for the operon , demonstrates protection at bp +2 to +14 relative to the rbs operontranscriptional start site ( bp 3-15 in the 5 ' direction of the dyad symmetry ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>Reduction of P2 transcription by GalR alone was previously observed with gal DNA under speciﬁed conditions ; 30 in this case GalR bound to OI blocks transcription elongation if OI was preceded by a DNA insert with a speciﬁc sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>There is no evidence of RNAP-dependent protection at the Prib-now-like sequence located 14/15 base pairs downstream of P1 ( P3 , Figure 1 ) , although mutations that render P1 and P2 FIGURE 2 : Digital representation of an autoradiogram showing the DNase I cutting pattern of combinations of RNAP , CAP , and GalR bound to the O L I E / O G sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>When GalR is bound to OI , the RNAP-induced hypersensitivity at that site is obscured by the GalR-induced protection , except for a single base pair at the 5 ′ border of OI ( Figure 2 , insert ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by GalR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>It was shown by gel retardation assay that GalR does not bind to this sequence ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The deduced amino acid sequence of the malI gene product revealed that it is highly similar to the classical repressor proteins , such as LacI or GalR ( 209 ) and must represent a repressor itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A/T at positions 8 of the operators ( Fig. 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>For example , the galE promoter is repressed by GalR , the lacZ promoter repressed by LacI and many CRP-activated promoters are repressed by CytR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Three assumptions are made in its construction : ( 1 ) The structure of the two-protein complex is the sum of the individual structures ; ( 2 ) the DNA bend induced by CAP at its cognate sequence at the gal operon can be modeled as the bent DNA structure in the CAP-DNA cocrystal ; ( 3 ) PurR is an appropriate structural model for GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>The DAP decarboxylase coding sequence is immediately followed by that of the GalR protein , transcribed in the opposite direction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6350601" target="_blank">6350601</a>]<br></li>
<li>( A ) The organization of the operon and its regulation by the positive regulator cAMP receptor protein ( CRP ) and the negative regulator GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2186968" target="_blank">2186968</a>]<br></li>
<li>Regulation of the P2galE promoter , located 5 bp upstream of P1galE , is opposite ; transcription from this promoter is further enhanced by OE-bound GalR and repressed by cAMP -- CRP ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>When DNA looping is not allowed , GalR binding to OE of the galETKM operon activates P2galE and represses P1galE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>When cellular conditions do not allow Gal repressosome assembly , GalR binding to the external galETKM operator ( OE ) represses the P1galE and activates the P2galE promoter ( Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Results P1 repression and P2 activation are independent events GalR normally represses transcription from the two gal promoters by binding to bipartite operators , OE and OI , located at position ÿ60 .5 and 53.5 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>In the absence of HU or DNA 13 supercoiling the P1 promoter is repressed by a looping-independent mechanism by binding of GalR to an operator ( OE ) located upstream of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>As galactose levels increase further , genes under GalR control , the low-afﬁnity galactose transporter , galactose permease , as well as galactose-metabolizing enzymes would be induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>If so , we would expect that GalS will be induced at lower galactose concentrations than GalR , since the mgl operon is the more efficient transport system at low galactose concentrations , ( ii ) The galS gene is autoregulated and its expression is CRP cAMP dependent , while gaIR expression appears to be constitutive -LCB- Weickert and Adhya , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>The increased sensitivity of GalS to galactose relative to that of GalR permits the induction of the mgl operon coding for the high-afﬁnity galactose transporter at galactose concentrations lower than those that would signiﬁcantly induce the gal operon and the low-afﬁnity transporter , galactose permease , which our results indicate are largely under the control of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>We have found that NagC , binds to a single , high-affinity site overlapping the promoter of galP ( galactose permease ) gene and that expression of galP is repressed by a combination of NagC , GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Expression of P-galactosidase from the galR promoter was not significantly affected by the loss of GalR and/or GalS ( Fig. 6 ) , nor was it inducible by D-fucose or D-galactose ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Therefore , unlike GalR , GalS represses P1galE transcription only by contact inhibition , not by DNA looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>This indicates that GalS acts as a repressor of its own synthesis and that GalR represses galS in a noninducible manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>Because ( 1 ) Gals and GalR proteins show extremely high homology in amino acid sequence , ( 2 ) Gals activity can be modulated by the same inducers t.hat ligand with Gal repressor , ( 3 ) Gals also negatively controls transcription of the gal operon , and ( 4 ) Gals acts by binding to the same two gal operators , 0 , and O , , as does the Gal repressor ( Tokeson et al. , 1991 ) , we call Gals ; following the definition of an isozyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1619663" target="_blank">1619663</a>]<br></li>
<li>GalS is believed to repress transcription , although not efficiently , in a manner analogous to GalR ( Weickert and Adhya , 1992a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934815" target="_blank">7934815</a>]<br></li>
<li>Thus , although a LacI-mediated looped complex can inhibit RNAP binding and repress transcription at the gal operon ( Haber & Adhya , 1988 ; Choy & Adhya , 1993 ) , GalR does not function by this mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8619994" target="_blank">8619994</a>]<br></li>
<li>In Escherichia coli , the genes responsible for D-galactose metabolism ( the galactose operon [ galETK ] ) and for high ¬ affinity transport of methylgalactosides , glucose , and galactose ( the mglBAC operon ) are controlled by a repressor , GalR ( 5 , 17 , 18 , 24 ) , and an isorepressor , GalS ( 9 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>In the first point , such a case has been reported in the galETK ( M ) operon concerning galactose utilization , which is regulated by GalR and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14593252" target="_blank">14593252</a>]<br></li>
<li>Because repression of P1galE by GalS is incomplete , we assume that GalR plays the major role in the D-galactose-dependent regulation of the galETKM operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Taken together , these measurements confirm the widely held hypothesis that GalR and HU induce DNA looping in the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid-associated protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Repression by GalR of the galETK operon has been extensively studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>GalR appeared to be the major repressor of the gal operon , because in galS mutants the gal operon is repressed , while mutation of galR leads to a high level of expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Transcription of the gal operon involves two promoters and operators and can be repressed by GalR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>Transcription of the mglBAC operon is regulated in a similar way , except that it is only partially repressed by GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>11 when Spot42 sRNA is expressed at full strength ( not repressed by cAMP -- CRP ) and the gal operon is transcribed at the maximum rate possible in the absence of cAMP -- CRP and GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>Both promoters are repressed in the absence of galactose by GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19541626" target="_blank">19541626</a>]<br></li>
<li>In the absence of GalR , the gal operon is induced two - to fourfold by the addition of D-galactose or D-fucose ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>GalR , the ﬁrst of these proteins to be identiﬁed and which has been puriﬁed , functions as the major repressor of the gal operon and is sufﬁcient for repression of its genes in the absence of galactose ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Both GalR and GalS are known to respond to inducers , but the presence of saturating concentrations of inducer does not allow induction of the gal operon to the maximal level ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>( a ) Repression of in vitro transcription of the gal operon mediated by GalR in the absence and presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>It has lower afﬁnity for the bipartite operators controlling the expression of the gal operon and , like GalR , is incapable , in the absence of other factors , of repressing transcription of the upstream gal operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Gal repressor ( GalR ) , which is a dual function regulator , represses transcription from the P1 promoter and enhances transcription from the P2 promoter when it binds to the operator OE in the gal operon of Escherichia coli ( Choy & Adhya , 1992 ; Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Nevertheless , like GalR , GalS selectively represses the downstream promoter , P1 , of the gal operon and slightly activates the upstream promoter , P2 , in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Both GalR and GalS selectively repress the gal operon transcription from the P1 promoter , while transcription from P2 is activated slightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Repression of PgalR by both GalR and GalS in our in vitro assays ( Fig. 4 ) suggests that the galR gene also belongs to the gal regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>The relative afﬁnities of GalR and GalS for the operator sequences in gal , mgl , and galP were reﬂected in the effectiveness of the two repressors in inhibiting transcription of these genes in vitro ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR wa more potent as an inhibitor of transcription of the galP gene and less effective as an inhibitor of transcription of the mgl operon compared to its potency as a repressor of gal operon transcription from P1 ( Fig. 3C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>GalR regulates transcription of the gal operon in the absence of GalS , and GalS regulates transcription of the mgl operon ( 32 ) and the galS gene in the absence of GaIR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8416900" target="_blank">8416900</a>]<br></li>
<li>We measured the binding of GalR and GalS to the operators of the gal operon as well as those of the mgl operon and the galP gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Similar levels of derepression were observed for both promoters when they were expressed in the absence of GalR , i.e. in cells deleted for the gene galR , which encodes the repressor ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Transcription is expressed as the percentage of that which was obtained in the absence of GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>We isolated and characterized a GalR mutant containing an amino acid substitution ( R282L ) that can repress transcription in the absence of HU and supercoiled DNA both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The level of P2 transcription was the same both in the absence and presence of GalR ; the addition of HU also did not change the amount of P2 transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In the absence of GalR and HU ( ﬁrst lane in each panel ) P2 is transcribed more than P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>