<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - GlpR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">GlpR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">GlpR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This includes DnaA , FarR , GlpR , MalT , Lrp , MetR , NarL , TyrR , TorR and TreR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>In many cases ( DeoR , GlpR , LacR , AccR and FruR ) , the inducer is a phosphorylated sugar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>In many cases , the effector-molecule that interacts with the transcriptional-regulator in order to achieve induction is a phosphorylated sugar ( DeoR , GlpR , LacR , and AccR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Examples of carbohydrate catabolic systems that are regulated by repressors belonging to the DeoR family include those of the glycerol-P ( GlpR ) , glucitol ( GutR ) , fucose ( FucR ) , L-ascorbate ( UlaR ) and deoxyribonucleoside ( DeoR ) systems of E. coli , as well as those of the lactose ( LacR ) , fructose ( FruR ) and inositol ( IolR ) systems of Gram-positive bacteria and agrocinopine ( AccR ) of A. tumefaciens ( Valentin-Hansen et al. , 1985 ; Yamada & Saier , 1988 ; Lu & Lin , 1989 ; van Rooijen & de Vos , 1990 ; Beck von Bodman et al. , 1992 ; Yoshida et al. , 1995 ; Zeng et al. , 1996 ; Campos et al. , 2004 ; Barriere et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>The tetrameric nature of AgaR and the fact that each of the promoter regions contains multiple operators suggest that , like GlpR and DeoR , AgaR binds several operators simultaneously , resulting in DNA looping around the site of transcription initiation ( Dandanell and Hammer , 1985 ; Hammer and Dandanell , 1989 ; Larson et al. , 1992 ; Weissenborn et al. , 1992 ; Yang and Larson , 1996 ; Yang et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>GlpR ( Larson , AgaR repressor and dominant repressor variants 815 et al. , 1987 ) , DeoR ( Valentin-Hansen et al. , 1986 ; Mortensen et al. , 1989 ) and possibly all members of this family of proteins are active as oligomers of identical subunits that bind to multiple palindromic operator sites on DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Although GlpR and DeoR exhibit signiﬁcant similarities in the recognition helices and corresponding operators sites , wild-type GlpR , or its var-iants , failed to-bind the deoR operator sites ( Zeng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>DeoR and GlpR of E. coli regulate the expression of genes involved in the metabolism of deoxyribonucleosides and sn-glycerol 3-phosphate respectively ( Mortensen et al. , 1989 ; Zeng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>IS + versus hs normallzed lo the eno control 2.30 2.23 0.31 4.74 1.99 0.25 1.90 1.18 0.41 Transcriptionalregulation in GlpR and RbsR repressor-deficientstrains As glpK and glpQ expression is regulated by GlpR ( Cozzarelli et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Both DeoR and GlpR form higher-order oligomers under native conditions ( octamers and tetramers , respectively ) and achieve repression by simultaneous binding of repressor to widely separated operator sites , with the intervening DNA forming a loop ( 1 , 11 , 12 , 16 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>, reduces the binding affinity of GlpR for the operator and thus releases repression of the glp genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Therefore , it was surprising that the operator binding speciﬁcity of GlpR was not changed to that of DeoR when these two residues of GlpR were changed to match those of DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_16">
Reactions were carried out with the following concentrations of GlpR tetramers and analyzed : lane 1 , no repressor ; lane 2 , 1 nM GlpR ; lane 3 , 2 nM GlpR ; lane 4 , 3 nM GlpR ; lane 5 , 4 nM GlpR ; and lane 6 , 4 nM GlpR and 2 mM glycerol-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></label>
<input id="act_similarity_16", type="checkbox">
<div class="ACT_similar">
<li>Reactions were carried out with the following concentrations of the glp repressor and analyzed : lane 1 , no repressor ; lane 2 , 2 nM GlpR ; lane 3 , 4 nM GlpR ; lane 4 , 8 nM GlpR ; lane 5 , 20 nM GlpR ; lane 6 , 40 nM GlpR ; and lane 7 , 40 nM GlpR and 3 mM glycerol-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Reactions were carried out with the following concentrations of glp repressor tetramers and analyzed : lane 1 , no repressor ; lane 2 , 2 nM GlpR ; lane 3 , 4 nM GlpR ; lane 4 , 8 nM GlpR ; lane 5 , 20 nM GlpR ; lane 6 , 40 nM GlpR ; and lane 7 , 8 nM GlpR and 3 mM glycerol-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Reactions were carried out with the following concentrations of glp repressor tetramers and analyzed : lane 1 , no repressor ; lane 2 , 1 nM GlpR ; lane 3 , 3 nM GlpR ; lane 4 , 6 nM GlpR ; lane 5 , 9 nM GlpR ; lane 6 , 20 nM GlpR ; lane 7 , 40 nM GlpR ; and lane 8 , 9 nM GlpR and 3 mM glycerol-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li></div>
<li>GlpR negatively controls all the Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>glpD belongs to the glp regulon which is negatively regulated by a repressor ( GlpR ) that is inactivated by the inducer G3P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934831" target="_blank">7934831</a>]<br></li>
<li>The glp genes and operons are negatively controlled by the GlpR repressor ( reviewed in reference 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400248" target="_blank">1400248</a>]<br></li>
<li>Speciﬁcally , the glp regulon is repressed by GlpR in the absence of glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601708" target="_blank">15601708</a>]<br></li>
<li>The afﬁnity of GlpR for each glp operator site is reduced upon the binding of glycerol-P , the inducer for the glp regulon ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>In the presence of 2 mM inducer ( glycerol-P ) , the binding afﬁnity of GlpR was reduced ( Fig. 2A , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The disappearance of OD3 protection when OD4 was mutated provides strong evidence that OD3 and OD4 bind GlpR cooperatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The fact that this plasmid still expresses sufficient GlpR for repression provides strong evidence that the glp GR sequence X Ž1384 -- 2691 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>From the results shown in Table 4 , pSY15 , which harbors the tandem operators OD1 and OD2 , bound the repressor 10 times better than did pDW65 , which harbors only OD1 ( 2,210 versus 190 ) , consistent with the previous discovery that GlpR binds tandem operators cooperatively ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>An emerging general feature of negative control by GlpR is that GlpR binds cooperatively to tandem operators ( OD1 and OD2 or OA2 and OA3 ) located at or near the promoter , with additional contributions to repression provided by remote operators ( OD3 and OD4 or OT1 to OT3 ) , to form a dense nucleoprotein complex with the intervening DNA forming a loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Repressor completely protected the tandem operators OD1 and OD2 at 1 nM GlpR ( Fig. 2A , lanes 1 through 5 ) , showing that GlpR is active and binds with high afﬁnity to these tandem operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>When a truncated labeled DNA fragment without the tandem operators ( OD1 and OD2 ) was used for footprinting ( Fig. 4 ) , the internal glpD operators still bound GlpR starting at 6 nM , a concentration slightly higher than that required for protection in the presence of OD1 and OD2 ( lanes 1 through 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The fact that the internal operators were unable to function alone in vivo even though these operators did bind GlpR in vitro ( the amount of GlpR needed for footprinting of the internal operators was essentially independent of the presence of OD1 and OD2 ) suggested a further cooperative mechanism , with GlpR simultaneously binding the four operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>This means that although the internal operators required OD1 and OD2 for repression in vivo , OD1 and OD2 were not required in order to demonstrate the binding of GlpR to OD3 and OD4 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The results of the present study indicate that the internal operators OD3 and OD4 exert their function in vivo by cooperative binding of GlpR tetramer ( s ) with OD1 and OD2 , since they had no effect in vivo in the absence of OD1 and OD2 but contributed ﬁve - to sevenfold to repression in the presence of OD1 and OD2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Plasmid-borne internal glpD operators OD3 and O n D4 have no apparent afﬁnity for GlpR in the absence of OD1 and OD2 , as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The distance between OD2 and OD3 is 395 bp , a distance suitable for DNA loop formation as GlpR binds to the four glpD operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>This observation suggested that GlpR could directly regulate the lsr operon by regulating lsrR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601708" target="_blank">15601708</a>]<br></li>
<li>Constitutive expression from Pgl pG may be necessary to produce the required amount of GlpR for specific repression during growth-on-glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Therefore , the glpEGR promoters were not subject to negative control by GlpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Potential regulation of the operon promoters by GlpR , catabolite repression , anaerobiosis or by FIS was studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>The glpE promoter was apparently controlled by the cAMP -- CRP complex , but none of the promoters was responsive to specific repression by GlpR , to anaerobiosis or to FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>In the case of glpD , besides regulation by GlpR , the promoter is subject to regulation by cyclic AMP-CRP and by the ArcA / ArcB systems ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Interestingly , the transcription of glpF and glpK is regulated by CRP and GlpR , whereas the expression of glpD is regulated by CRP , GlpR and ArcA , as proposed elsewhere [ 1,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>As shown in Table 6 , a higher level of GlpR alone repressed glpD -- lacZ better than the combined contributions of GlpD and the lower level of GlpR expressed from pBY415 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>The glpK -- lacZ fusion serves as a reporter for the level of plasmid-encoded GlpR ; b-galactosidase activity i expected to be inversely proportional to the level of GlpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Specific repression exerted by GlpR was characterized in vivo using glpD -- lacZ and glpK -- lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>The degree of repression was correlated with the level of GlpR expression , and was inefficient when the glpD-encoded glycerol-P dehydrogenase was absent , presumably due to accumulation of the inducer , glycerol-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">GlpR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The C-terminal resi-due of GlpR is predicted to be Cys , the sole Cys residue in the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>The helix-turn-helix motif of GlpR is in boldface type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>These results taken together suggest that amino acid residues conserved among proteins of the DeoR/GlpR family play similar functional roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>The carboxy-terminal region of proteins in the DeoR / GlpR family is responsible for protein oligomerization and inducer-binding ( van Rooijen et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Previous work with GlpR and other members of the DeoR/GlpR family have identiﬁed highly conserved amino acid residues that function in DNA-binding or response to inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Three of the four residues shown to be important for inducer response in LacR are highly conserved among members of the DeoR/GlpR family [ Lys-80 , Lys-213 and Asp-210 ( van Rooijen et al. , 1993 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Substitution of this invariant arginine with alanine abolishes the DNA-binding activity of GlpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Generation of a GlpR variant with altered DNA sequence binding speciﬁcity would provide direct information concerning amino acid residues that contact the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>Although we were unable to generate a glp repressor variant with deo operator binding speciﬁcity , the fact that all amino acid substitutions within the second helix of the proposed helix-turn-helix motif conferred a negative-dominant phenotype , along with the observed high degree of sequence similarity of this region to the helix-turn-helix motif of other transcriptional regulators , provides the ﬁrst experimental evidence for the assignment of this region of GlpR and other family members as the helix-turn-helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">GlpR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Examples of carbohydrate catabolic systems that are regulated by repressors belonging to the DeoR family include those of the glycerol-P ( GlpR ) , glucitol ( GutR ) , fucose ( FucR ) , L-ascorbate ( UlaR ) and deoxyribonucleoside ( DeoR ) systems of E. coli , as well as those of the lactose ( LacR ) , fructose ( FruR ) and inositol ( IolR ) systems of Gram-positive bacteria and agrocinopine ( AccR ) of A. tumefaciens ( Valentin-Hansen et al. , 1985 ; Yamada & Saier , 1988 ; Lu & Lin , 1989 ; van Rooijen & de Vos , 1990 ; Beck von Bodman et al. , 1992 ; Yoshida et al. , 1995 ; Zeng et al. , 1996 ; Campos et al. , 2004 ; Barriere et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>The NagC repressor in fact , shows strong homology to the family of xylose repressors from different Gram-positive species ( Kreuzer et al. , 1989 ; Lokman et al. , 1991 ) , and certain other proteins , notably some sugar kinases ( Angell et al. , 1992 ; Titgemeyer et al. , 1994 ) but not signiﬁcantly to the other major families of prokaryotic repressors , the LacI-GalR family ( Weickert & Adhya , 1992 ) or the DeoR-GlpR family ( van Rooijen & de Vos , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The genome of H. inﬂuenzae encodes three family members , including those corresponding to GlpR and FucR of E. coli ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>GlpR ( Larson , AgaR repressor and dominant repressor variants 815 et al. , 1987 ) , DeoR ( Valentin-Hansen et al. , 1986 ; Mortensen et al. , 1989 ) and possibly all members of this family of proteins are active as oligomers of identical subunits that bind to multiple palindromic operator sites on DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>However , the binding speciﬁcity of the DeoR family members to their operators seems to be affected by multiple determinants , as indicated by the result obtained with the repressors GlpR and DeoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16451192" target="_blank">16451192</a>]<br></li>
<li>Previous work with GlpR and other members of the DeoR/GlpR family have identiﬁed highly conserved amino acid residues that function in DNA-binding or response to inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Three of the four residues shown to be important for inducer response in LacR are highly conserved among members of the DeoR/GlpR family [ Lys-80 , Lys-213 and Asp-210 ( van Rooijen et al. , 1993 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Although we were unable to generate a glp repressor variant with deo operator binding speciﬁcity , the fact that all amino acid substitutions within the second helix of the proposed helix-turn-helix motif conferred a negative-dominant phenotype , along with the observed high degree of sequence similarity of this region to the helix-turn-helix motif of other transcriptional regulators , provides the ﬁrst experimental evidence for the assignment of this region of GlpR and other family members as the helix-turn-helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>glpD is a member of the glp regulon , which is under transcriptional control of the GlpR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601708" target="_blank">15601708</a>]<br></li>
<li>Ray , W.K. ( 1999 ) Characterization of AgaR and YihW , members of the DeoR family of transcriptional regulators , and GlpE , a rhodanese belonging to the GlpR regulon , also a member of the DeoR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931310" target="_blank">10931310</a>]<br></li>
<li>Divergently transcribed from this gene cluster is the gene agaR encoding a transcriptional repressor belonging to the DeoR/GlpR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>The agaR repressor ( AgaR ) belongs to the DeoR/GlpR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by GlpR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>Involved in a glutamine-transport system GlnHPQ ; induced by lack of glutamine P0A9Z1 Nitrogen regulatory protein 5.17 / 12,425.45 P-II 1 GlnB 1 79 GlnH 6.87 / 24,963.41 7.04 / 25,213 6.93 / 24,504 7.32 / 30,106 ( 6 -- 11 ) 7.13 / 29,388 ( 6 -- 11 ) P0AEQ3 Glutamine-binding periplasmic protein 48 , 179 , 286 , 287 See GlnB P0AC55 179 , 287 Nitrogen regulatory protein P-II 2 Glutamine-transport ATP-binding protein 5.84 / 12,259.22 5.78 / 12,732 ( 5 -- 6 ) 5.94 / 11,914 ( 5.5 -- 6.7 ) 6.08 / 26,730.99 GlnK Part of the binding-protein-dependent transport system for glutamine ; probably responsible for energy coupling to the transport system ; induced by lack of glutamine Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Conversion of glycerol-3-phosphate to dihydroxyacetone ; uses fumarate or nitrate as the electron acceptor Electron transfer protein ; may also function as the membrane anchor for the GlpAB dimer Converts glycerol-3-phosphate to dihydroxyacetone by using molecular oxygen or nitrate as the electron acceptor Key enzyme in the regulation of glycerol uptake and metabolism ; the activity of this regulatory enzyme is affected by several metabolites ; the inhibition by fructose 1,6-biphosphate causes alterations in the quaternary structure of the enzyme ; induced by L-alpha-glycerol-3-phosphate ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol Hydrolyzes deacylated phospholipids to glycerol-3-phosphate and the corresponding alcohols Repressor of the glycerol-3-phosphate regulon Involved in tricarboxylic-acid-cycle and is allosterically inhibited by NADH Involved in nitrogen metabolism and glutamate-biosynthesis ; has low activity in the presence of exogenous leucine ; regulated positively by Lrp Part of the binding-protein-dependent transport system for glutamate and aspartate 179 P10346 GlnQ GlnS Glutaminyl-tRNA synthetase P00962 5.89 / 63,346.70 5.88 / 63,346 29 , 109 , 179 , 228 , 230 , 294 5.91 / 74,850 ( DIGE 4.5 -- 6.5 ) Anaerobic glycerol-3-phosphate 5.75 / 45,357.24 dehydrogenase subunit B P13033 GlpB 179 P0A996 Anaerobic glycerol-3-phosphate 8.78 / 44,108.04 dehydrogenase subunit C GlpC 79 1 GlpD 6.96 / 56,750.54 5.38 / 49,649 Aerobic glycerol-3-phosphate dehydrogenase P13035 179 , 228 , 294 GlpK 5.36 / 56,099.57 5.30 / 50,642 P0A6F3 Glycerol kinase 29 , 71 , 179 , 228 , 296 5.22 / 38,200 GlpQ 179 Glycerophosphoryl diester phosphodiesterase P09394 5.82 / 28,047.84 5.82 / 28,724 GlpR P0ACL0 Glycerol-3-phosphate regulon repressor P0ABH7 Citrate synthase 228 , 294 GltA 6.21 / 48,014.99 6.85 / 45,571 179 , 294 Glutamate synthase ( NADPH ) 5.54 / 51,884.08 5.47 / 52,481 small chain 5.40 / 52,585 5.48 / 53,050 ( 5 -- 6 ) P09832 GltD 61 , 212 , 228 , 286 , 287 , 294 GltI 5.67 / 15,568 ( F ) 7.87 / 36,857 ( 6 -- 11 ) 7.82 / 31,229.49 Glutamate/aspartate periplasmic binding protein P37902 179 , 228 , 287 5.59 / 53,815.73 6.03 / 45,316.59 Glutamyl-tRNA synthetase Serine hydroxymethyltransferase P04805 P0A825 179 GltX GlyA Interconversion of serine and glycine and a key enzyme in the biosynthesis of purines , lipids , hormones , and other components ; decreases during overexpression of recombinant protein in large scale 97 , 179 , 228 , 286 , 294 , 321 6.04 / 45,960 5.94 / 46,142 4.5 -- 6.5 ) 4.5 -- 6.5 ) 4.5 -- 6.5 ) 6.02 / 45,274 ( DIGE 6.00 / 45,000 ( DIGE 6.00 / 43,217 ( DIGE 179 5.29 / 76,681.76 P00961 GlyS Glycyl-tRNA synthetase beta chain Phosphoheptose isomerase 179 Catalyzes the isomerization of sedoheptulose 7-phosphate in D-glycero-D-manno-heptose 7-phosphate Glycyl-tRNA synthetase beta chain Phosphoheptose isomerase 5.97 / 20,814.75 P63224 GmhA 179 Continued on following pag Phosphoglycolate phosphatase Gph P32662 4.58 / 27,389.17 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 179 , 321 2,3-Bisphosphoglycerate - dependent phosphoglycerate mutase GpmA P62707 5.86 / 28,425 5.86 / 28,425.21 5.86 / 27,000 ( DIGE 4.5 -- 6.5 ) 5.82 / 32,032 ( 5 -- 6 ) 2,3-Bisphosphoglycerate-independent phosphoglycerate mutase Xanthine-guanine phosphoribosyltransferase Involved in the hexose monophosphate shunt ; increases during phosphate limitation Negative regulator for the gluconate utilization system GNT-I , the gntUKR operon Could be involved in gluconate metabolism Maintains high levels of reduced glutathione in the cytosol Speciﬁcally catalyzes the dephosphorylation of 2-phosphoglycolate ; involved in the dissimilation of the intracellular 2-phosphoglycolate formed during the DNA-repair of 3 phosphoglycolate ends , a major class of DNA lesions induced by oxidative stress ; constitutively expressed Catalyzes the interconversion of 2-phosphoglycerate and 3-phosphoglycerate ; strongly inhibited by vanadate and regulated by the Fur protein See GpmA ; insensitive to vanadate Protein GntY GntY ( YhgI ) Gor P63020 4.52 / 20,997.70 4.72 / 19,718 ( 4 -- 5 ) 4.96 / 19,460 ( 4 -- 5 ) 5.65 / 48,676 179 , 287 P06715 Glutathione reductase 5.64 / 48,772.51 29 , 179 , 228 , 294 Gph P32662 4.58 / 27,389.17 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 179 , 321 GpmA P62707 5.86 / 28,425 5.86 / 28,425.21 5.86 / 27,000 ( DIGE 4.5 -- 6.5 ) 5.82 / 32,032 ( 5 -- 6 ) P37689 GpmI 5.15 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.11 / 52,363 ( DIGE 4.5 -- 6.5 ) 5.15 / 52,348 ( 4.5 -- 5.5 ) 5.44 / 13,981 ( 4.5 -- 5.5 ) 5.21 / 16,164 ( 5 -- 6 ) 5.14 / 56,193.89 287 , 321 Involved in purine-salvage pathway ; acts on guanine , on xanthine , and to a lesser extent on hypoxanthine Acts as a radical domain for damaged PFL and possibly other radical proteins ; induced by low pH ; a strong candidate for response to internal acidiﬁcation 5.52 / 16,970.59 P0A9M5 Gpt 87 2 GrcA ( YﬁD ) 5.09 / 14,284.19 5.02 / 15,065 5.00 / 12,390 Autonomous glycyl radical cofactor P68066 27 , 179 , 274 , 286 , 287 5.13 / 11,675 5.01 / 11,079 ( 4 -- 5 ) 4.96 / 10,626 ( 4 -- 5 ) 5.05 / 9,834 ( 4.5 -- 5.5 ) 5.21 / 9,497 ( 4.5 -- 5.5 ) 5.09 / 11,523 ( 4.5 -- 5.5 ) 5.23 / 9,710 ( 4.5 -- 5.5 ) 4.68 / 15,984 4.95 / 13,865 ( 4 -- 5 ) Necessary for efﬁcient RNA polymerase transcription elongation past template-encoded arresting sites ; cleavage of the nascent transcript by cleavage factors such as GreA or GreB allows the resumption of elongation from the new 3 terminus Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and overexpression of bioproducts ; induced by temp , ethanol , nalidixic acid , and puromycin Binds to GroL in the presence of Mg-ATP and suppresses the ATPase activity of the latter ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and is induced by temp , ethanol , nalidixic acid , puromycin , and phosphate limitation GreA 179 , 286 , 287 4.71 / 17,640.96 Transcription elongation factor P0A6W5 GroL 4.85 / 57,197.66 P0A6F5 60-kDa chaperonin 71 , 97 , 138 , 179 , 228 , 287 , 294 , 297 , 321 4.85 / 56,695 4.93 / 61,278 ( DIGE 4.5 -- 6.5 ) 4.98 / 60,758 ( DIGE 4.5 -- 6.5 ) 4.96 / 59,987 ( DIGE 4.5 -- 6.5 ) 4.99 / 56,877 ( 4 -- 5 ) 4.95 / 55,185 ( 4 -- 5 ) 4.97 / 61,125 ( 4 -- 5 ) 4.93 / 55,185 ( 4 -- 5 ) 4.95 / 55,185 ( 4 -- 5 ) 4.84 / 52,802 ( 4 -- 5 ) 4.96 / 54,146 ( 4.5 -- 5.5 ) 4.99 / 55,284 ( 4.5 -- 5.5 ) 5.15 / 15,657 5.15 / 10,500 ( DIGE 4.5 -- 6.5 ) 5.22 / 12,998 ( 4.5 -- 5.5 ) GroS P0A6F9 10 -- kDa chaperonin 71 , 179 , 228 , 287 , 293 , 294 , 297 , 321 5.15 / 10,386.95 Continued on facing pag GMP reductase GuaC 179 6.1 / 37,383.67 P60560 5.09 / 96,963.51 5.28 / 100,213 P0AES4 DNA gyrase subunit A GyrA 228 , 294 5.72 / 89,818.72 5.63 / 31,059.27 P0AES6 DNA gyrase subunit B P31658 Chaperone protein GyrB HchA 179 321 5.44 / 33,098 ( DIGE 4.5 -- 6.5 ) HdeA 4.68 / 9,740.91 4.56 / 10,724 4.84 / 9,424 ( 4 -- 5 ) P0AES9 Protein 103 , 179 , 228 , 287 , 294 , 323 HdeB 4.94 / 9,065.24 4.85 / 11,285 5.00 / 9,588 ( 4.5 -- 5.5 ) P0AET2 Protein 103 , 179 , 228 , 278 , 287 , 294 HdhA 5.22 / 26,778.59 5.17 / 25,083 P0AET8 7-Alpha-hydroxysteroid dehydrogenase 179 , 286 321 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.25 / 35,493.60 HemB P0ACB2 Delta-aminolevulinic acid dehydratase P06983 Porphobilinogen deaminase 5.31 / 33,851.83 HemC 179 Uroporphyrinogen decarboxylase Putative uroporphyrin-III C-methyltransferase Cell division protein FtsH Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins in association with DnaK ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol ; induced by temp , ethanol , nalidixic acid , CdCl2 , and puromycin Inhibited by 7,8-dihydrofolate , methotrexate , and trimethoprim Conjugates reduced glutathione to a large no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Gene/operon Function Regulator FIS effect Repressor of glpR Repressor of g/pR g W K glpTQ Transport and assimilation of glycerol GlpR Transport and assimilation of GlpR phospholipids Deamination of cytidine to uridine CytR cdd Repressor of cyfR Activator of cdd Repressor of cflR Activator of udp Repressor of cytR Repressor of deoR Activator of deoCABD Repressor of mtlADR Activator of rbsR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>is a hypothetical repressor protein from E. coli that is closely related to GlpR ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
<li>glpD is a member of the glp regulon , which is under transcriptional-control of the GlpR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601708" target="_blank">15601708</a>]<br></li>
<li>Abbreviations and references to the published sequences are as follows : GutR ( Eco ) , repressor of the glucitol operon of E. coli ( Yamada & Saier , 1988 ) ; AccR ( Atu ) , Agrobacterium ttlmefaciensrepressor of opine catabolism and nopaline Ti plasmid conjugal transfer ( von Bodman et al. , 1992 ) ; GatR ( Eco ) , transcriptional regulator of galactitol utilization of E. coli ( Komoda e t al. , 1991 ; Nobelmann & Lengeler , 1995 ) ; GlpR ( Eco ) , repressor of the glycerol 3-phosphate regulon of E. coli ( Choi e t al. , 1988 ) ; ORF f251 ( Eco ) and ORF 0269 ( Eco ) , two putative transcriptional regulators that are encoded respectively within the 92.8-100 min , GenBank accession no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932697" target="_blank">8932697</a>]<br></li>
<li>DeoR and GlpR of E. coli regulate the expression of genes involved in the metabolism of deoxyribonucleosides and sn-glycerol 3-phosphate respectively ( Mortensen et al. , 1989 ; Zeng et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">GlpR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>( B ) OD3 and OD4 ( in GlpR binds cooperatively to tandem consensus operators ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The distance between OD2 and OD3 is 395 bp , a distance suitable for DNA loop formation as GlpR binds to the four glpD operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>To ﬁnd out if any of them bind GlpR in vivo , plasmids containing different combinations of glpD operator sequences were constructed ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The potential GlpR binding sequences are in boldface type and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Comparison of GlpE , GlpG , and GlpR sequences with other protein sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955387" target="_blank">8955387</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by GlpR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>These results suggest that glpG andror glpR have their own promoters that express GlpGrGlpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>nor the plasmids pBY415 or pBY200 contain the glpE or intact glpG gene , it is clear that repression conferred by multicopy GlpR is not dependent on GlpE or GlpG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>The glp genes and operons are negatively controlled by the GlpR repressor ( reviewed in reference 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400248" target="_blank">1400248</a>]<br></li>
<li>Speciﬁcally , the glp regulon is repressed by GlpR in the absence of glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601708" target="_blank">15601708</a>]<br></li>
<li>Recently , it was found that three operators within the coding region of glpT contributed to about threefold negative control of glpT transcription by GlpR ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Therefore , the glpEGR promoters were not subject to negative control by GlpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9524241" target="_blank">9524241</a>]<br></li>
<li>Interestingly , the transcription of glpF and glpK is regulated by CRP and GlpR , whereas the expression of glpD is regulated by CRP , GlpR and ArcA , as proposed elsewhere [ 1,5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>IS + versus hs normallzed lo the eno control 2.30 2.23 0.31 4.74 1.99 0.25 1.90 1.18 0.41 Transcriptionalregulation in GlpR and RbsR repressor-deficientstrains As glpK and glpQ expression is regulated by GlpR ( Cozzarelli et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Lin and coworkers found that the glpD operon was controlled more tightly by GlpR than was either the glpTQ or glpFKX operon ( 30 - , 26.5 - , and 3.2-fold , respectively [ 12 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>This observation suggested that GlpR could directly regulate the lsr operon by regulating lsrR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601708" target="_blank">15601708</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>