<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - H-NS</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">H-NS</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">H-NS transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul>				<label class="ACT_collapse_similarity" for="act_similarity_1">
Large-scale monitoring of pleiotropic regulation of gene expression by the procaryotic nucleoid-asso-ciated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></label>
<input id="act_similarity_1", type="checkbox">
<div class="ACT_similar">
<li>Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-asso-ciated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670992" target="_blank">12670992</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11872702" target="_blank">11872702</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426343" target="_blank">12426343</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_3">
Hommais , F. , Krin , E. , Laurent-Winter , C. , Soutourina , O. , Malpertuy , A. , Le Caer , J.P. , et al. ( 2001 ) Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731270" target="_blank">14731270</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></label>
<input id="act_similarity_3", type="checkbox">
<div class="ACT_similar">
<li>Hommais , F. , Krin , E. , Laurent-Winter , C. , Soutourina , O. , Malpertuy , A. , Le Caer , J.P. et al. ( 2001 ) Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Hommais , F. , Krin , E. , Laurent-Winter , C. , Soutourina , O. , Malpertuy , A. , Le Caer , J.P. , et al. ( 2001 ) Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Hommais , F. , Krin , E. , Laurent-Winter , C. , Soutourina , O. , Malpertuy , A. , Le Caer , J.P. , Danchin , A. and Bertin , P. ( 2001 ) Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>[ 40 ] Hommais F. , Krin E. , Laurent-Winter C. , Soutourina O. , Malpertuy A. , Le Caer J.P. , Danchin A. , Bertin P. , Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein , H-NS , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li></div>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage shock protein Regulator ; phage shock protein Regulator ; phage shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Genes whose mRNA levels displayed twofold decreases after addition of 0.2 mM ZnSO4 to minimal medium Genea Gene product and/or functionb Fold P value decrease ( 0.05 ) Transport and binding actP 10 3 10 3 10 4 10 5 10 3 10 4 10 2 10 3 10 2 10 2 10 3 10 3 10 4 10 4 Acetate permease Possible assembly or transport protein for curli 3.1 6.2 2.7 5.9 2.5 2.8 2.1 2.8 2.4 2.2 2.4 3.7 2.2 2.3 4.4 2.5 4.3 1.7 4.4 2.4 1.0 1.1 2.9 3.5 5.0 8.1 8.0 6.0 csgF csgG gadC lsrB hdeD lsrD lsrG ompW ugpA ugpE uidB ydcU ygjU Other gene implicated in responses to metals yrbL Genes implicated in response to stresses other than metals csgD Transcriptional regulator for curli operon csgBA csiE Stationary-phase inducible protein ; sigma S-dependent promoter cstA Starvation-induced protein involved in peptide utilization during carbon starvation dps Stress response DNA-binding protein gadE Required for stationary-phase induction ; pH 5.5 growth medium induced ; overexpression confers acid resistance hdeB Acid resistance ; periplasmic protein ; repressed by H-NS ; rpoS regulon osmE Regulated by growth-phase as well as osmotic pressure osmC Osmotically inducible , stress-inducible membrane protein C ; involved in defense ag oxidative compounds psiF pho regulon member , requiring PhoRB system rssB Two-component response-regulator , affecting RpoS regulon sodC Superoxide dismutase ; Cu , Zn , periplasmic uspB Ethanol resistance in stationary phase Periplasmic protein hdeA Periplasmic chaperone of acid-denatured proteins Extracellular protein hlyE Latent hemolysin Intermediary metabolism astA Putative arginine succinyltransferase gadB Glutamate decarboxylase B , vitamin B6 dependent otsB Trehalose phosphate phosphatase ; cold and heat induced talA Transaldolase A ; creBC regulon Membrane structure frdD Fumarate reductase membrane anchor polypeptide ychH Function unknown ; putative membrane protein transcribed divergently from pth ynfA Inner membrane protein ; function unknown Phage-related function sgcC Putative phosphotransferase enzyme IIC sgcX Putative gene in sgc cluster ; function unknown stfR Phage lambda stf gene homolog in prophage Rac Unclassiﬁed elaB Function unknown yagU Function unknown yahO Function unknown ybaY Function unknown ycfH Putative DNAse ; tatD paralog ; no Tat phenotype ycgK Function unknown ydeI Function unknown ydhZ Function unknown yeaQ Function unknown yebV Function unknown yhcO Function unknown yhfG Function unknown yjbJ Function unknown yjfO Function unknown yoaE Function unknown yodD Function unknown ytfR Function unknown ytfT Function unknown ytfR Function unknown ytfT Function unknown a Gene names obtained from http://bmb.med.miami.edu/EcoGene/EcoWeb/ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659689" target="_blank">15659689</a>]<br></li>
<li>of exogenous and endogenous hydrophobic electrophiles Involved in ﬁrst step of GMP biosynthesis from IMP ; induced by low pH during anaerobic-growth Catalyzes the irreversible NADPH-dependent deamination of GMP to IMP ; functions in the conversion of nucleobase , nucleoside , and nucleotide derivatives of G-to-A nucleotides and in maintaining the intracellular balance of A and G nucleotides DNA gyrase negatively supercoils closed circular double-stranded DNA in an ATP-dependent manner and also catalyzes the interconversion of other topological isomers of double-stranded DNA rings , including catenanes and knotted rings See GyrA Uses temp-induced exposure of structured hydrophobic domains to capture and stabilize early unfolding protein intermediates under severe thermal stress ; rapidly releases them once stress has abated and is induced by heat-shock Induced by low pH during anaerobic-growth and at the stationary phase Periplasmic protein that may be involved in acid resistance ; enhanced at the stationary phase and in the acnB mutant but repressed by H-NS 7-Alpha-dehydroxylation of cholic acid , yielding deoxycholic acid and lithocholic acid , respectively ; highest afﬁnity with taurochenodeoxycholic acid Involved in porphyrin biosynthesis HdeA 4.68 / 9,740.91 4.56 / 10,724 4.84 / 9,424 ( 4 -- 5 ) P0AES9 Protein 103 , 179 , 228 , 287 , 294 , 323 HdeB 4.94 / 9,065.24 4.85 / 11,285 5.00 / 9,588 ( 4.5 -- 5.5 ) P0AET2 Protein 103 , 179 , 228 , 278 , 287 , 294 HdhA 5.22 / 26,778.59 5.17 / 25,083 P0AET8 7-Alpha-hydroxysteroid dehydrogenase 179 , 286 321 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.25 / 35,493.60 HemB P0ACB2 Delta-aminolevulinic acid dehydratase P06983 Porphobilinogen deaminase Tetrapolymerization of the monopyrrole porphobilinogen into the hydroxymethylbilane preuroporphyrinogen in several discrete steps Involved in porphyrin biosynthesis Involved in siroheme and cobalamin biosynthesis Seems to act as an ATP-dependent zinc metallopeptidase ; involved in the degradation of 32 5.31 / 33,851.83 HemC 179 179 179 5.88 / 39,248.12 4.68 / 42,963.02 P29680 P09127 HemE HemX 5.91 / 70,708.09 P0AAI3 HﬂB 179 Continued on following pag HﬂC and HﬂK govern the stability of phage lambda cII protein and have been proposed to encode or regulate a cII-speciﬁc protease RNA-binding protein that stimulates the elongation of poly ( A ) tails and targets several mRNAs for degradation , possibly by increasing polyadenylation or by interfering with ribosome binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Gene characteristics Gene Predicted Band no.a Gene function ( reference ) Regulation of corresponding gene ( reference ) productb molecular mass ( Da ) c PﬂB * 85,357 GadA * 52,685 1 Pyruvate formate lyase I ( 56 ) Glutamate decarboxylase ( 64 ) Induced by ArcA and FNR ( 61 ) 2 Positively regulated by rpoS , CRP , gadE , gadW , and gadX ; negatively regulated by H-NS ( 21 , 66 ) Positively activated by rpoS and by CRP ( 33 , 74 ) Positively activated by CRP ( 74 ) ; repressed by ArcA ( 60 ) Same transcription unit as gatZ Positively regulated by rpoS ( 33 , 46 , 71 ) Positively regulated by rpoS ( 2 , 33 , 46 , 71 ) 52,773 TnaA † 3 Tryptophan deaminase ( tryptophanase ) ( 22 ) 47,109 GatZ † 3 Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Quinone reductase ; response to oxidative-stress ( 45 ) Bacterial ferritin , DNA protection during stationary phase ( 39 , 73 ) Transcriptional regulator ( 28 ) 30,812 20,846 18,695 GatY † WrbA * Dps * 4 5 6 , 7 , and 11 24,935 CsgD * 8 Positively regulated by rpoS , hns , and ompR ( 25 , 58 ) Same transcription unit as csgD Positively regulated by FNR ( 18 ) Outer membrane protein ; curli transport component ( 38 ) Outer membrane protein ; receptor for colicin S4 ( 49 ) 30,557 22,928 CsgG * OmpW * 9 10 a As in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>hdeB Periplasmic chaperone of acid-denatured proteins hdeA Periplasmic chaperone of acid-denatured proteins hdeD Putative membrane transporter , H-NS repressed gadE Required for stationary phase-induced , pH 5.5 growth medium-induced , and EvgA-induced acid resistance mdtE MdtEF-TolC multidrug resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine mdtF MdtEF-TolC multidrug resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine gadW Positive AraC-type regulator of gadA and gadBC , in absence of gadX ; repressor of gadX gadY sRNA regulates gadX and gadY gadX Positive transcriptional-regulator of gadA and gadBC ; repressed by HNS and GadW ; stimulated by RpoS gadA Glutamate decarboxylase isozyme GadB operon : 1567 -- 1570 kb gadB Glutamate decarboxylase isozyme gadC Glutamate transporter 1.03 0.95 3.17 -1.59 1.58 2.58 -0.98 1.60 2.70 -1.17 1.53 3.17 2.66 -1.38 -1.47 1.80 1.19 4.89 -3.46 1.43 3.21 3.73 -2.50 -2.44 1.29 Table 4 : Drug resistance , cytoskeletal and ion transport proteins showing pH dependence under oxygen limitation Gene Function Log2pH ratio Acid up-regulated pH 5.7 / pH 7.0 pH 7.0 / pH 8.5 pH 5.7 / pH 8.5 0.80 0.60 0.86 1.31 0.44 chaB chaC Regulator of Na + - Ca + / H + antiporter Regulator of Na + - Ca + / H + antiporter -0.38 -0.56 -0.4 copA cueR cusC feoA feoB marR1 mdtE mdtF mdtG1 mdtI1 mdtJ1 mdtL mgtA nhaA nhaB1 nikA nikB nikC nikD nikE prc Copper - , silver-translocating P-type ATPase Activator of copper-responsive regulon genes cueO and copA Silver and copper efflux ferrous iron transport protein A ferrous iron transport protein B multiple antibiotic resistance-protein MdtEF-TolC multidrug resistance efflux transporter MdtEF-TolC multidrug resistance efflux Multidrug resistance efflux transporter multidrug transport protein multidrug transport protein Multidrug resistance efflux protein Mg2 + transport Na + / H + antiporter Na + / H + antiporter periplasmic binding protein for nickel nickel transport nickel transport ATP-binding protein of nickel transport system ATP-binding protein of nickel transport system carboxy-terminal protease for penicillin-binding protein 3 0.91 0.63 1.22 0.53 0.82 0.41 0.31 -1.59 -0.98 3.17 2.58 0.16 1.58 1.60 0.25 1.16 3.02 0.57 0.63 1.00 0.37 0.84 0.56 0.86 0.74 0.49 0.37 1.16 2.91 0.48 1.93 -0.93 0.41 0.7 0.50 0.70 0.60 0.45 0.32 Base up-regulated -0.34 -0.35 -0.69 -0.42 -1.22 -0.52 -0.12 -0.66 -0.66 -0.71 -0.51 -0.66 -0.58 -0.53 -1.1 -0.39 -0.3 -0.65 acrA acrB acridine efflux pump acridine efflux pump -0.93 acrR2 alx2 ampC atpA2 atpC2 atpD2 atpF2 atpG2 atpH2 atpI2 chaA corA cutA dacA dacC dacD emrA1 mepA1 mreB2 mreC napF napG napH acrAB operon repressor Putative metal transporter , homolog of terC ( tellurium resistance ) beta-lactamase ; penicillin resistance ; penicillin-binding protein ATP synthase , F1 alpha-subunit ATP synthase , F1 epsilon-subunit ATP synthase , F1 beta-subunit ATP synthase , F0 subunit b ATP synthase , F1 gamma-subunit ATP synthase , F1 delta-subunit ATP synthase subunit Na + - Ca + / H + antiporter Mg2 + transport , system I divalent cation tolerance protein D-alanyl-D-alanine carboxypeptidase , fraction A ; penicillin-binding protein 5 D-alanyl-D-alanine carboxypeptidase ; penicillin-binding protein 6 penicillin binding protein 6b multidrug resistance secretion protein murein DD-endopeptidase , penicillin-insensitive regulator of ftsI , penicillin binding protein 3 , septation function rod shape-determining protein Fe-S ferredoxin-type protein ferredoxin-type protein ferredoxin-type protein -0.57 -0.24 -0.36 -0.46 -0.35 -0.42 -0.86 0.10 -0.27 -0.58 0.87 -1.22 -0.64 -0.68 -0.53 -0.29 -0.59 -0.39 -0.37 -1.16 -0.90 -0.11 -0.19 -0.44 -0.39 -0.14 -0.20 -0.20 1Acid up-regulated with aeration [ 2 ] 2Base up-regulated with aeration [ 2 ] Real-time PCR expression ratios of selected genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>More than 30 proteins are altered in the hfq null mutant ; some of these proteins are Sdependent genes Involved in fourth step of Lhistidine from 5-phospho-alpha-D-ribose 1-diphosphate P0ABC3 Protein HﬂC 6.3 / 37,649.88 179 Hfq 204 P0A6X3 Protein 7.20 / 11,035.19 HisA P10371 4.94 / 26,032.65 1 - ( 5-Phosphoribosyl ) -5 - ( 5-phosphoribosylamino ) methylideneamino imidazole-4-carboxamide isomerase Histidine biosynthesis bifunctional protein Histidinol-phosphate aminotransferase Histidinol dehydrogenase 4.99 / 24,831 ( DIGE 4.5 -- 6.5 ) 5.46 / 23,581 ( DIGE 4.5 -- 6.5 ) 179 , 321 5.76 / 40,277.96 5.43 / 35,112 See HisA : steps 6 and 8 286 P06987 HisB See HisA : step 7 ; induced by high pH during anaerobic-growth See HisA : ﬁnal step ; catalyzes the sequential NAD-dependent oxidations of L-histidinol to Lhistidinaldehyde and then to L-histidine ; induced by high pH during anaerobic-growth See HisA : step 5 5.01 / 39,360.14 P06986 HisC 323 HisD P06988 5.06 / 45,979.15 5.07 / 48,837 ( DIGE 4.5 -- 6.5 ) 179 , 321 , 323 P60664 HisF 5.03 / 28,454.45 179 Imidazole glycerol phosphate synthase subunit ATP phosphoribosyltransferase Has a crucial role in the pathway because the rate of histidine biosynthesis seems to be controlled primarily by regulation of hisG enzymatic activity ; feedback inhibited by histidine or inhibited by AMP and N -5 - phosphoribosyl-ATP See HisA : step 1 HisG 5.47 / 33,366.72 P60757 179 , 321 E 4.5 -- 6.5 ) IG 5.40 / 32,959 ( D HisH 287 P60595 5.33 / 21,652.86 5.24 / 27,057 ( 5 -- 6 ) Imidazole glycerol phosphate synthase subunit Histidine biosynthesis bifunctional protein HisIE Histidine-binding periplasmic protein HisI P06989 See HisA : steps 2 and 3 179 5.24 / 22,755.82 Component of the high-afﬁnity histidine permease , a binding-protein-dependent transport system ( HisJ , HisQ , HisM , and HisP ) HisJ 179 , 286 , 287 , 321 5.17 / 26,232.61 5.05 / 28,668 P0AEU0 5.28 / 27,969 ( DIGE 4.5 -- 6.5 ) 5.14 / 27,701 ( DIGE 4.5 -- 6.5 ) 4.99 / 10,461 ( 4 -- 5 ) 5.14 / 28,391 ( 4.5 -- 5.5 ) 5.59 / 50,642 HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Catalyzes the phosphorylation of D-glycero-D-manno-heptose 7-phosphate at the C-1 position to form d,D-heptose-1 ,7 - bisphosphate ; also catalyzes the ADP transfer to D-glycero-D-manno-heptose 1-phosphate , yielding ADP-d , D-heptose Involved in NO detoxiﬁcation in an aerobic process , termed nitric oxide dioxygenase reaction , that utilizes O2 and NAD ( P ) H to convert NO to nitrate , which protects the bacterium from various noxious nitrogen compounds ; induced by nitric oxide NO ( under aerobic conditions ) , nitrite , nitrate ( under anaerobic conditions ) , nitroso compounds , and paraquat ; induced by low pH during anaerobic-growth HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Hmp 5.48 / 43,867.66 Flavohemoprotein P24232 323 Continued on facing pag Has possible histone-like function and may be a global transcriptional-regulator through its ability to-bind to curved DNA sequences , which are found in regions upstream of a certain subset of promoters ; subject to transcriptional autorepression by binding preferentially to the upstream region of its own gene recognizing two segments of DNA on both sides of a bend centered around position 150 ; the hns mutant exhibit derepression not only of csgBA , hdeAB , mcc , and osmY but also of the expression of at least 22 S-controlled genes , including S itself ; increases during phosphate limitation and phosphonate growth Involved in purine salvage pathway and acts exclusively on hypoxanthine Involved in the maturation of iron-sulfur cluster-containing proteins ; seems to help targeting proteins to be folded toward HscA Redox regulated molecular chaperone ; protects both thermally unfolding and oxidatively damaged proteins from irreversible aggregation ; plays an important role in the bacterial defense system toward oxidative-stress ; induced by heat-shock Chaperone subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol Protease subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol Molecular chaperone ; has ATPase activity ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol but decreases during phosphate limitation late Participates in the periplasmic electron-transferring activity of hydrogenase 2 during its catalytic turnover Required for the maturation of the three NiFe hydrogenases ; exhibits a low intrinsic GTPase activity ; affects some aspect of the processing of hydrogenases 1 and 2 , perhaps nickel incorporation into the apoenzymes , since hypB gene lesions can be complemented by high nickel ion concn in the medium TABLE 2 -- Continued Hns P0ACF8 DNA-binding protein H-NS 5.44 / 15,408.44 5.45 / 15,612 5.00 / 12,390 5.38 / 15,657 5.02 / 9,935 Hpt 5.09 / 20,115.24 5.38 / 12,316 ( 4.5 -- 5.5 ) 179 , 287 P0A9M2 Hypoxanthine phosphoribosyltransferase P0A6L9 Cochaperone protein 5.05 / 20,137.71 HscB 79 1 4.35 / 32,534.48 P0A6Y5 33-kDa chaperonin HslO 79 1 HslU 5.24 / 49,593.80 5.30 / 48,579 P0A6H5 ATP-dependent hsl protease ATP-binding subunit 71 , 179 , 228 , 294 5.95 / 18,961.66 5.93 / 22,161 P0A7B8 ATP-dependent protease HslV 71 , 228 , 294 HtpG 5.09 / 71,422.53 5.06 / 65,639 P0A6Z3 Chaperone protein 71 , 179 , 228 , 293 , 294 HybA 79 P0AAJ8 Hydrogenase-2 operon protein 6.38 / 33,263.79 6.38 / 33,263.79 1 P0AAN3 Hydrogenase isoenzymes nickel 5.42 / 31,565.04 6.32 / 32,224 incorporation protein HypB 286 Continued on following pag Required for the formation of all three hydrogenase isoenzymes Associates with aggregated proteins , together with IbpB , to stabilize and protect them from irreversible denaturation and extensive proteolysis during heat-shock and oxidative-stress ; aggregated proteins bound to the IbpAB complex are more efﬁciently refolded and reactivated by the ATP-dependent chaperone systems ClpB and DnaK/DnaJ/GrpE ; its activity is ATP independent ; IbpAB increase in overexpression of heterologous proteins and biopolymers See IbpA ; decreases during phosphate limitation early Inhibition of this enzyme by phosphorylation regulates the branch point between the Krebs cycle and the glyoxylate bypass , which is an alternate route that accumulates carbon for biosynthesis when acetate is the sole carbon source for growth ; increases during high-cell-density cultivation but decreases during phosphate limitation Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the formation of alpha-ketobutyrate from threonine in isoleucine biosynthesis ; isoleucine allosterically inhibits whereas valine allosterically activates this enzyme ; increases during phosphate limitation and phosphonate growth Involved in valine and isoleucine biosynthesis : step 1 See IlvB : step 2 ; induced in the presence of acetohydroxybutyrate and acetolactate ; decreases after benzoic acid treatment See IlvB : step 3 ; increases after benzoic acid treatment See IlvB : step 4 ( ﬁnal ) ; acts on leucine , isoleucine , and valine ; decreases during phosphate limitation See IlvB : step 1 ; sensitive to valine inhibition Determines N-hexane tolerance ; involved in outer membrane permeability ; essential for envelope biogenesis ; could be part of a targeting/usher system for outer membrane components TABLE 2 -- Continued HypD P24192 5.80 / 41,363.38 5.81 / 41,711 Hydrogenase isoenzymes formation protein P0C054 Small heat-shock protein IbpA 5.57 / 15,773.71 5.35 / 15,600 5.68 / 15,600 4 , 97 , 98 , 228 , 287 , 296 5.63 / 15,600 5.63 / 15,384 ( 5 -- 6 ) 5.63 / 14,237 ( 5.5 -- 6.7 ) 5.19 / 16,093.20 P0C058 Small heat-shock protein IbpB 5.02 / 15,600 5.20 / 15,600 4 , 97 , 98 , 295 Icd 5.06 / 46,200 5.15 / 46,200 5.02 / 46,100 5.11 / 44,335 ( DIGE 4.5 -- 6.5 ) 5.15 / 44,053 ( DIGE 4.5 -- 6.5 ) 4.70 / 22,351 ( 4 -- 5 ) 5.15 / 43,951 ( 4.5 -- 5.5 ) 5.00 / 74,518 ( 5 -- 6 ) 5.15 / 45,756.71 P08200 Isocitrate dehydrogenase ( NADP ) 179 , 228 , 236 , 287 , 293 , 294 , 321 , 325 P52643 IdhA 5.29 / 36,534.79 5.59 / 34,100 D-Lactate dehydrogenase 128 , 293 5.57 / 56,195.25 6.16 / 54,999 Threonine dehydratase biosynthetic P04968 IlvA 293 287 5.22 / 56,152 ( 4.5 -- 5.5 ) P08142 5.30 / 60,440.57 I e Acetolactate synthase isozym large subunit Ketol-acid reductoisomerase IlvB 5.26 / 52,066 5.20 / 53,937.83 P05793 IlvC 286 , 321 5.22 / 52,046 ( DIGE 4.5 -- 6.5 ) 5.29 / 51,264 ( DIGE 4.5 -- 6.5 ) 321 5.54 / 62,065 ( DIGE 4.5 -- 6.5 ) 5.59 / 65,400.37 P05791 Dihydroxy-acid dehydratase IlvD 5.43 / 35,112 5.54 / 33,962.46 P0AB80 Branched-chain amino acid aminotransferase IlvE 228 , 236 , 293 , 294 287 8.11 / 15,587 ( 6 -- 11 ) 8.01 / 17,976.76 e Acetolactate synthase isozym III small subunit Organic solvent tolerance protein P00894 IlvH Imp 4.91 / 88,368 ( DIGE 4.5 -- 6.5 ) 4.85 / 87,068.95 P31554 179 , 321 Continued on facing pag IscA P0AAC8 Iron-binding protein ( YfhF ) 4.75 / 11,556.05 IscU 179 Involved in Fe-S biosynthesis 4.82 / 13,848.59 IspA IspG Geranyltranstransferase 5.27 / 32,159.63 5.37 / 26,625 4-Hydroxy-3-methylbut-2-en-1-yl 5.87 / 40,683.61 diphosphate synthase P22939 P62620 179 , 286 179 5.2 / 34,774.56 4-Hydroxy-3-methylbut-2-enyl diphosphate reductase P62623 IspH 179 5.46 / 15,065 5.41 / 14,304 ( 5 -- 6 ) 5.51 / 14,102.90 Inhibitor of vertebrate lysozyme P0AD59 Ivy 228 , 287 5.33 / 12,548 ( 5 -- 6 ) 5.47 / 86,507 ( DIGE 4.5 -- 6.5 ) 103 , 321 5.54 / 84,162.61 Catalase HPII P21179 KatE Peroxidase/catalase HPI P13029 KatG 103 , 287 , 321 5.18 / 77,774 ( DIGE 4.5 -- 6.5 ) 5.16 / 74,539 ( 4.5 -- 5.5 ) 5.14 / 80,023.82 5.37 / 31,293.68 5.98 / 31,100 P0AB74 Tagatose-1 ,6 - bisphosphate aldolase AgaY Kba 99 2-Amino-3-ketobutyrate CoA ligase 2-Dehydro-3-deoxygluconokinase 2-Dehydro-3-deoxyphosphooctonate aldolase Could transfer iron-sulfur clusters to apo-ferredoxin ; multiple cycles of 2Fe2S cluster formation and transfer are observed ; acts catalytically ; recruits intracellular free iron so as to provide iron for the assembly of transient ironsulfur cluster in IscU in the presence of IscS , L-cysteine , and the thioredoxin reductase system TrxA/TrxB Not found in SWISS-PROT / TrEMBL database but available in NCBI database ( GI : 16130454 ) Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate into 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate Converts 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate into isopentenyl diphosphate and dimethylallyl diphosphate ; also involved in penicillin tolerance , control of the stringent response , and isoprenoid biosynthesis Strong inhibitor of lysozyme C Decomposes hydrogen peroxide into water and oxygen ; serves to protect cells from the toxic effects of hydrogen peroxide ; induced by entry into stationary phase and after benzoic acid treatment Bifunctional , exhibiting both catalase and broad-spectrum peroxidase activities ; induced by hydrogen peroxide and by entry into stationary phase but decreases after benzoic acid treatment Catalyzes the reversible condensation of dihydroxyacetone phosphate with glyceraldehyde 3-phosphate to produce tagatose-1 ,6 - bisphosphate ; repressed in the presence of glucose Negatively regulated by Lrp Kba 99 - / 42,200 5.64 / 43,117.04 P0AB77 61 Kbl 4.92 / 33,962.36 6.32 / 30,832.69 P37647 P0A715 179 KdgK KdsA Synthesis of 2-keto-3 - 179 , 287 6.10 / 33,261 ( 6 -- 11 ) deoxyoctulosonic acid 8-phosphate , which is required for lipid A maturation and cellular growth Activates 2-keto-3-deoxyoctulosonic acid for incorporation into bacterial lipopolysaccharide in gramnegative bacteria Involved in the transport of maltose and maltodextrins ; indispensable for translocation of dextrins containing more than three glucosyl moieties ; acts as a receptor for several bacteriophages including lambda ; induced by maltose 5.15 / 27,483.25 5.06 / 33,695 3-Deoxy-manno-octulosonate cytidylyltransferase P04951 KdsB 228 , 294 LamB 79 4.72 / 47,385.03 Maltoporin P02943 1 Continued on following pag Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate ( 2-oxoisovalerate ) to form 3-carboxy-3-hydroxy-4-methylpentanoate ( 2-isopropylmalate ) ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate via the formation of 2-isopropylmaleate LdhA P52643 D-Lactate dehydrogenase 5.29 / 36,534.79 5.59 / 34,100 P09151 2-Isopropylmalate synthase LeuA 308 5.47 / 57,166.71 LeuC 3-Isopropylmalate dehydratase large subunit P0A6A6 5.90 / 49,750.64 5.95 / 51,656 5.77 / 51,451 179 , 286 , 321 5.42 / 44,439 5.93 / 50,340 ( DIGE 4.5 -- 6.5 ) See LeuC ; increases after benzoic acid treatment Increases late during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Represses a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic-growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic-growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_15">
Mukerji , M. , and Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></label>
<input id="act_similarity_15", type="checkbox">
<div class="ACT_similar">
<li>Mukerji , M. & Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li>
<li>Mukerji , M. and Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Mukerji , M. , and Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible role for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : Possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22020646" target="_blank">22020646</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP -- cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_23">
Probing the structure , function , and interactions of the Escherichia coli H-NS and StpA proteins by using dominant negative deriv-atives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></label>
<input id="act_similarity_23", type="checkbox">
<div class="ACT_similar">
<li>Probing the structure , function , and interactions of the Escherichia coli H-NS and StpA proteins by using dominant negative derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>[ 39 ] Williams R.M. , Rimsky S. , Buc H. , Probing the structure , function and interactions of the Escherichia coli H-NS and StpA proteins using dominant negative derivatives , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Williams , R.M. , Rimsky , S. , and Buc , H. ( 1996 ) Probing the structure , function , and interactions of the Escherichia coli H-NS and StpA proteins by using dominant negative deriv-atives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Williams , R.M. , Rimsky , S. , and Buc , H. ( 1996 ) Probing the structure , function and interactions of the Escherichia coli H-NS and StpA proteins by using dominant negative deri-vatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Williams , R.M. , Rimsky , S. , and Buc , H. ( 1996 ) Probing the structure , function , and interactions of the Escherichia coli H-NS and StpA proteins by using dominant negative derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Williams , R.M. , Rimsky , S. , and Buc , H. ( 1996 ) Probing the structure , function and interactions of the Escherichia coli H-NS and StpA proteins using dominant negative derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li></div>
<li>H-NS-like proteins form three obvious clusters in a parsimony tree , which correspond to H-NS of enterobacteria , to H-NS of H. inﬂuenzae , StpA of E. coli and S. typhimurium and VicH of V. cholerae , and to H-NS-like proteins isolated from Rhodobacter and Bordetella genera .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>BglJ is a transcriptional-regulator whose constitutive expression causes activation of the H-NS - and StpA-repressed bgl ( aryl - , D-glucoside ) operon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>[ 32 ] Johansson J. , Dagberg B. , Richet E. , Uhlin B.E. , H-NS and StpA proteins stimulate expression of the maltose regulon in Escheri-chia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_33">
[ 45 ] Johansson , J. , Dagberg , B. , Richet , E. and Uhlin , B.E. ( 1998 ) H-NS and StpA proteins stimulate expression of maltose regulon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></label>
<input id="act_similarity_33", type="checkbox">
<div class="ACT_similar">
<li>Johansson , J. , Dagberg , B. , Richet , E. , and Uhlin , B.E. ( 1998 ) H-NS and StpA proteins stimulate expression of the maltose regulon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_35">
, Wurm , R. , Lux , B. , Meltzer , M. , Menzel , A. , and Wagner , R. ( 2005 ) LRP and H-NS -- cooperative partners for transcription regulation at E. coli rRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></label>
<input id="act_similarity_35", type="checkbox">
<div class="ACT_similar">
<li>, Wurm , R. , Lux , B. , Meltzer , M. , Menzel , A. , and Wagner , R. ( 2005 ) Lrp and H-NS -- cooperative partners for transcription regulation at Escherichia coli rRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li></div>
<li>This seems unlikely , since direct activation of transcription by Fur in E. coli has not been shown conclusively but rather occurs indirectly through the RyhB sRNA or H-NS protein ( 29 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21908668" target="_blank">21908668</a>]<br></li>
<li>It is remarkable that in E. coli K-12 , regulation of yciGFE by YncC and by H-NS are intimately linked , whereas in Salmonella , YncC directly activates transcription , and thus , activation by YncC is , at least partly , disconnected from the H-NS network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>However , transcription of pap is independent of CRP activation in the E. coli K-12 mutant lacking H-NS ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>E. coli chromosomal cas3 is located upstream of casA -- E ( Figure 1A ) and is not controlled by H-NS , although cas3 transcription in Thermus thermophilus is increased 2.5-fold in response to infection by phage ϕYS40 [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21699496" target="_blank">21699496</a>]<br></li>
<li>Transcription of nhaA , the Main Na1/H1 Escherichia coli , Is Regulated b and Growth Phase Antiporter of y Na1 NIR DOVER AND ETANA PADAN * Division of Microbial and Molecular Ecology , Institute of Life Sciences , The Hebrew University of Jerusalem , 91904 Jerusalem , Israel Received 7 August 2000/Accepted 19 October 2000 The transcription of nhaA , encoding the main Na1/H1 antiporter of Escherichia coli , is induced by Na1 , regulated by NhaR , and affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Na - induced transcription of nhaA , which encodes an Na / H anti-porter in Escherichia coli , is positively regulated by NaR and affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867437" target="_blank">12867437</a>]<br></li>
<li>White-Ziegler , C.A. , Villapakkam , A. , Ronaszeki , K. , and Young , S. ( 2000 ) H-NS controls pap and daa ﬁmbrial transcription in Escherichia coli in response to multiple environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , July 1996 , p. 4176 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Effects of H-NS and Potassium Glutamate on sS - and s70-Directed Transcription In Vitro from Osmotically Regulated P1 and P2 Promoters of proU in Escherichia coli K. RAJKUMARI ,1 SHUICHI KUSANO ,2 AKIRA ISHIHAMA ,2 TAKESHI MIZUNO ,3 AND J. GOWRISHANKAR1 * Centre for Cellular & Molecular Biology , Hyderabad 500007 , India ,1 and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka 411,2 and Laboratory of Molecular Biology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464,3 Japan Received 8 December 1995/Accepted 30 April 1996 We have used supercoiled DNA templates in this study to demonstrate that transcription in vitro from the P1 and P2 promoters of the osmoresponsive proU operon of Escherichia coli is preferentially mediated by the sS - and s70-bearing RNA polymerase holoenzymes , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_45">
Effects of H-NS and potassium glutamate on sigmaS - and sigma70-directed transcription in vitro from osmotically regulated P1 and P2 promoters of proU in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></label>
<input id="act_similarity_45", type="checkbox">
<div class="ACT_similar">
<li>Effects of H-NS and potassium glutamate on S - and 70-directed transcription in vitro from osmotically regulated P1 and P2 promoters of proU in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17504942" target="_blank">17504942</a>]<br></li>
<li>Rajkumari , K. , Kusano , S. , Ishihama , A. , Mizuno , T. , and Gowrishankar , J. ( 1996 ) Effects of H-NS and potassium glutamate on s - and s - directed transcription in vitro from S 70 osmotically regulated P1 and P2 promoters of proU in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Rajkumari , K. , Kusano , S. , Ishihama , A. , Mizuno , T. , and Gowrishankar , J. ( 1996 ) Effects of H-NS and potassium glutamate on sS-and sD-directed transcription in vitro from osmotically regulated P1 and P2 promoters of proU in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9791185" target="_blank">9791185</a>]<br></li></div>
<li>Three effects of potassium glutamate on proU transcription in vitro were identiﬁed , namely , ( i ) direct activation of P1 transcription by EsS ( and P2 transcription by Es70 ) , ( ii ) inhibition of P1 tran-70 scription by Es ( both together leading to the increase in selectivity for EsS at P1 ) , and ( iii ) antagonism of inhibition of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Effects of H-NS on P1 and P2 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>This demonstrates that neither FIS nor H-NS seem to effect P2-directed transcription , when isolated from P1 , to a notable extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_52">
Mol Microbiol 11:913 -- 918 Terui Y , Higashi K , Taniguchi S , Shigemasa A , Nishimura K , Yamamoto K , Kashiwagi K , Ishihama A , Igarashi K ( 2007 ) Enhancement of the synthesis of RpoN , Cra , and H-NS by polyamines at the level of translation in Escherichia coli cultured with glucose and glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></label>
<input id="act_similarity_52", type="checkbox">
<div class="ACT_similar">
<li>Terui , Y. , Higashi , K. , Taniguchi , S. , Shigemasa , A. , Nishimura , K. , Yamamoto , K. , Kashiwagi , K. , Ishihama , A. and Igarashi , K. ( 2007 ) Enhancement of the synthesis of RpoN , Cra , and H-NS by polyamines at the level of translation in Escherichia coli cultured with glucose and glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>Barth , M. , Marschall , C. , Muffler , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) A role for the histone-likeprotein H-NS in growth-phase-dependent and osmotic regulation of os and many os-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_55">
Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of ss and many ss dependent genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></label>
<input id="act_similarity_55", type="checkbox">
<div class="ACT_similar">
<li>Barth , M. , Marschall , C. , Muffler , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of s-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of s-depen-dent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role of the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of ss and many ss-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of sS and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role of the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of sS and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_61">
Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. , and Bertin , P. ( 1999 ) Multiple control of ﬂagellum biosynthesis in Escherichia coli : Role of H-NS protein and the cyclic AMP -- catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></label>
<input id="act_similarity_61", type="checkbox">
<div class="ACT_similar">
<li>Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. , and Bertin , P. ( 1999 ) Multiple control of flagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP ± catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li></div>
<li>Sigma S-dependent growthphase induction of the csgBA promoer in Escherichia coli can be achieved in vivo by sigma 70 in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21421764" target="_blank">21421764</a>]<br></li>
<li>Arnqvist , A. , Olsén , A. , and Normark , S. ( 1994 ) sigmaS-dependent growthphase induction of the csgBA promoter in Escherichia coli can be achieved in vivo by sigma 70 in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224606" target="_blank">17224606</a>]<br></li>
<li>Molecular Microbiology ( 1994 ) 13 ( 6 ) , 1021-1032 ( 7 ® - dependent growth-phase induction of the csgBA promoter in Escherichia coii can be achieved in vivo by G ^ ^ in the absence of the nucieoid-associated protein H-NS Anna Arnqvist ^ * , Arne Olsen ^ and Staffan Normark ^ ^ Department of Microbioiogy , Umea University , S-901 87 Umea , Sweden .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_66">
Arnqvist , A. , OIs ~ n , A. , and Normark , S. ( 1994 ) aS-dependent growth-phase induction of the csgBA promoter in Escheri-chia coli can be achieved in vivo by ~ 7o in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8817489" target="_blank">8817489</a>]<br></label>
<input id="act_similarity_66", type="checkbox">
<div class="ACT_similar">
<li>S Arnqvist , A. , Olsén , A. , and Normark , S. ( 1994 ) s - dependent growth-phase induction of the csgBA promoter in Escheri-chia coli can be achieved in vivo by s70 in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li></div>
<li>- dependent growth-phase induction of the csgBA promoter in Escherichia coli can be achieved in vivo by 70 in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12057949" target="_blank">12057949</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_69">
Arnqvist , A. , Olsen , A. , and Normark , S. ( 1994 ) ss-depen-dent growth-phase induction of the csgBA promoter in Escherichia coli can be achieved in vivo by s70 in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></label>
<input id="act_similarity_69", type="checkbox">
<div class="ACT_similar">
<li>References Arnquist , A. , Olsen , A. , and Normark , S. ( 1994 ) ss-depen-dent growth-phase induction of the csgBA promoter in Escherichia coli can be achieved in vivo by s70 in the absence of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_71">
Dersch , P. , Schmidt , K. , and Bremer , E. ( 1993 ) Synthesis of the Escherichia coli K-12 nucleoid-associated DNA-bind-ing protein H-NS is subjected to growth-phase control and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></label>
<input id="act_similarity_71", type="checkbox">
<div class="ACT_similar">
<li>Dersch , P. , Smith , K. , and Bremer , E. ( 1993 ) Synthesis of the Escherichia coli K-12 nucleoid-associated DNA-binding protein H-NS is subjected to growth-phase control and regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_73">
P. , Schmidt , K. , and Bremer , E. ( 1993 ) Synthesis of the Eschericia coli K-12 nucleoid-assooiated DNA-binding protein H-NS is subjected to growth-phase control and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></label>
<input id="act_similarity_73", type="checkbox">
<div class="ACT_similar">
<li>Dersch , P. , Schmidt , K. , and Bremer , E. ( 1993 ) Synthesis of the Escherichia coil K-12 nucleoid-associated DNA-binding protein H-NS is subjected to growth-phase control and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>[ 21 ] Dersch P. , Schmidt K. , Bremer E. , Synthesis of the Escherichia coli K-12 nucleoid-associated DNA-binding protein H-NS is subjected to growth-phase control and autoregulation , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>Dersch , P. , Schmidt , K. , and Brener , E. ( 1993 ) Synthesis of the Escherichia coli K-12 nucleoid-associated DNA-binding protein H-NS is subjected to growth-phase control and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>[ 36 ] Dersch , P. , Schmidt , K. and Bremer , E. ( 1993 ) Synthesis of the Escherichia coli K-12 nucleoid-associated DNA-binding protein H-NS is subjected to growth-phase control and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></li>
<li>Dersch , P. , Schmidt , K. , and Bremer , E. ( 1993 ) Synthesis of the Escherichia coli K-12 nucleoid-associated DNA-binding protein H-NS is subjected to growth-phase control and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li></div>
<li>The CnuK9E H-NS Complex Antagonizes DNA Binding of DicA and Leads to Temperature-Dependent Filamentous Growth in E. coli Sang Hoon Yun , Sang Chun Ji , Heung Jin Jeon , Xun Wang , Si Wouk Kim , Geunu Bak , 1 1 $ 1 1 2 3 Younghoon Lee , Heon M. Lim * 3 1 1 Department of Biology , College of Biological Sciences and Biotechnology , Chungnam National University , Taejon , Republic of Korea , 2 Department of Environmental Engineering , Pioneer Research Center for Controlling of Harmful Algal Blooming , Chosun University , Gwangju , Republic of Korea , 3 Department of Chemistry , KAIST , Daejeon , Republic of Korea Abstract Cnu ( an OriC-binding nucleoid protein ) associates with H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_80">
[ 13 ] Donato G.M. , Kawula T.H. , Enhanced binding of altered H-NS protein to ﬂagellar rotor protein FliG causes increased ﬂagellar rotational speed and hypermotility in Escherichia coli , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></label>
<input id="act_similarity_80", type="checkbox">
<div class="ACT_similar">
<li>Enhanced binding of altered H-NS protein to ﬂagellar rotor protein FliG causes increased ﬂagellar rotational speed and hypermotility in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Enhanced binding of altered H-NS protein to ¯ agellar rotor protein FliG causes increased ¯ agellar rotational speed and hypermotility in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>Enhanced binding of altered H-NS protein to flagellar rotor protein FliG causes increased flagellar rotational speed and hypermotility in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20346719" target="_blank">20346719</a>]<br></li></div>
<li>Btcd , a mouse protein that binds to curved DNA , can substitute in Escherichia coli for H-NS , a bacterial nucleoid protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Shindo , H. , Ohnuki , A. , Ginba , H. , Katoh , E. , Ueguchi , C. , Mizuno , T. , and Yamazaki , T. ( 1999 ) Identiﬁcation of the DNA binding surface of H-NS protein from Escherichia coli by heteronuclear NMR spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>~ 9 GENE 06753 Histone-like proteins are required for cell growth and constraint of supercoils in DNA ( Escherichia coli ; DNA-binding proteins ; hupA , hupB , himA , hz ' rvtDand hns mutations ; hygromycin phosphotransferase ; cryptic bgl operon ; recombinant DNA ; superhelical density ) Kayoko Yasuzawa , Naataka Hayashi , Naoki Goshima , Kyoko K & no , Fumio fmamoto and Yasunobu Kano Received by A. Nakazawa : 30 April 1992 ; Revised/Accepted : 14 June/I3 July 1992 ; Received at publishers : 27 July 1992 SUMMARY The gene hns of Escherichia coii K-12 , which encodes the histone-like protein H-NS , was inactivated by insertion of a DNA ( gene hph ) encoding hygromycin phosphotransferase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Selective repression by Fis and H-NS at the Escherichia coli Introduction The Escherichia coli chromosome is packaged into a structure known as the nucleoid , which is believed to be organized by DNA supercoiling , RNA , transcription and nucleoid-associated proteins ( reviewed in Dame , 2005 ; Travers and Muskhelishvili , 2005 ; Jin and Cabrera , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Yang , J. , Tauschek , M. , Strugnell , R. and Robins-Browne , R.M. ( 2005 ) The H-NS protein represses transcription of the eltAB operon , which encodes heat-labile enterotoxin in enterotoxigenic Escherichia coli , by binding to regions downstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_89">
Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Dan-chin , A. , et al. ( 1994 ) The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></label>
<input id="act_similarity_89", type="checkbox">
<div class="ACT_similar">
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Danchin , A. , and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Danchin , A. and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of ¯ agella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Danchin , A. , and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of flagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E. H. , Lejeune , P. , Colson , C. , Danchin , A. , and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of flagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li></div>
<li>The H-NS protein is a global repressor of all chaperone -- usher ﬁmbriae in E. coli K-12 Our data demonstrated that , although cryptic under laboratory settings , ycb , yfc , yad , yra , sfm and yeh ﬁmbriae are nevertheless functional and potentially subjected to negative regulation alleviated under speciﬁc conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Lac repressor Model for enhanced DNA looping in the absence of H-NS The data in Figure 2 indicate that loss of the H-NS protein enhances the formation of the experimental DNA loop in living E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>A Proteins Stimulate Expression Regulon in Escherichia coli H-NS and Stp of the Maltose JÖRGEN JOHANSSON ,1 BJÖRN DAGBERG ,2 EVELYNE RICHET ,3 AND BERNT ERIC UHLIN1 * Department of Microbiology1 and Department of Physiology ,2 Umeå University , S-90187 Umeå , Sweden , and Unité de Génétique Moléculaire , Institut Pasteur , 75724 Paris Cedex 15 , France3 Received 11 March 1998/Accepted 4 September 1998 The nucleoid-associated protein H-NS is a major component of the chromosome-protein complex , and it is known to inﬂuence the regulation of many genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The smtA gene is not essential for cell growth and its expression is positively regulated by H-NS , an Escherichia coli histone-like protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_98">
The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>]<br></label>
<input id="act_similarity_98", type="checkbox">
<div class="ACT_similar">
<li>References Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional-repressor for expression of the anaerobic and growth-phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>For example , in the case of the expression of ﬂagellin genes , there seems to be stimulation , since the lack of a functional H-NS protein abolishes the motility of Escherichia coli ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>[ 20 ] Donato , G.M. , Lelivelt , M. and Kawula , T.H. ( 1997 ) Pro-moter-speci cents c repression of cents mB expression by the Escherichia coli nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>Promoterspeciﬁc repression of ﬁmB expression by the Escherichia coli nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207912" target="_blank">16207912</a>]<br></li>
<li>Promoter-speci ® c repression of ® mB expression by the Escherichia coli nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_106">
Donato , G.M. , Lelivelt , M.J. , and Kawula , T.H. ( 1997 ) Promoter-speciﬁc repression of ﬁmB expression by the Escherichia coli nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890032" target="_blank">12890032</a>]<br></label>
<input id="act_similarity_106", type="checkbox">
<div class="ACT_similar">
<li>Promoter-speciﬁc repression of ﬁmB expression by the Escherichia coli nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781567" target="_blank">10781567</a>]<br></li></div>
<li>The H-NS protein , in addition to acting as an autorepressor , acts as a transcriptional-repressor of proU , which is involved in osmoregulation ( UEGUCHIandMIZUNO1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
<li>H-NS : A UNIVERSAL REGULATOR The effect of the bacterial heat-stable nucleoid-structuring ( H-NS ) protein on gene expression is overwhelmingly negative and extends throughout the genome , pointing to an almost universal role for this nucleoid-associated protein as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The nucleoid-associated protein H-NS is a global transcriptional-repressor that controls the expression of several environmentally regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_111">
Ueguchi , C , and Mizuno , T. ( 1993 ) The Escherichia ccti nucieoid protein H-NS functions directly as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></label>
<input id="act_similarity_111", type="checkbox">
<div class="ACT_similar">
<li>Ueguchi , C. , and Mizuno , T. ( 1993 ) The Escherichia Coli nucleoid protein H-NS functions directly as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17493135" target="_blank">17493135</a>]<br></li>
<li>C. and Mizuno , T. ( 1993 ) The Escherichia coii nucleoid protein H-NS functions directly as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476160" target="_blank">7476160</a>]<br></li>
<li>The Escherichia coli nucleoid protein H-NS functions directly as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002611" target="_blank">8002611</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626295" target="_blank">8626295</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8244952" target="_blank">8244952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>Ueguchi , C. , and Mizuno , T. ( 1993 ) The Escherichia coli nucleoid protein H-NS functions directly as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li></div>
<li>Francetic , O. , Badaut , C. , Rimsky , S. and Pugsley , A.P. ( 2000 ) The ChiA ( YheB ) protein of Escherichia coli K-12 is an endochitinase whose gene is negatively controlled by the nucleoid-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Ueguchi , C. , Suzuki , T. , Yoshida , T. , Tanaka , K. , and Mizuno , T. ( 1997 ) Clariﬁcation of the dimerization domain and its functional signiﬁcance for the Escherichia coli nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_118">
Yoshida , T. , Yamashino , T. , Ueguchi , C. , and Mizuno , T. ( 1993b ) Expression of the Escherichia coli dimorphic glutamic acid decarboxylases is regulated by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></label>
<input id="act_similarity_118", type="checkbox">
<div class="ACT_similar">
<li>Yoshida , T. , Yamashimo , T. , Ueguchi , C. , and Mizuno , T. ( 1993b ) Expression of the Escherichia coli dimorphic glutamic acid decarboxylases is regulated by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li></div>
<li>For operons involved in use of carbon sources in Escherichia coli , the most frequent global regulator is cAMP/CAP , responsible for catabolite repression , but histone-like proteins , IHF , H-NS , Fis and HU are often involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_121">
Venkatesh , G.R. , Koungni , F.C.K. , Paukner , A. , Stratmann , T. , Blissenbach , B. , and Schnetz , K. ( 2010 ) BglJ-RcsB heterodimers relieve repression of the Escherichia coli bgl operon by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></label>
<input id="act_similarity_121", type="checkbox">
<div class="ACT_similar">
<li>Venkatesh , G.R. , Kembou Koungni , F.C. , Paukner , A. , Stratmann , T. , Blissenbach , B. and Schnetz , K. ( 2010 ) BglJ-RcsB heterodimers relieve repression of the Escherichia coli bgl operon by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li></div>
<li>Modeling feedback loops in the H-NS mediated regulation of the Escherichia coli bgl operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>Radde , N. , Gebert , J. , Faigle , U. , Schrader , R. and Schnetz , K. ( 2008 ) Modeling feedback loops in the H-NS mediated regulation of the Escherichia coli bgl operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Free , A. , Williams , R.M. , and Dorman , C.J. ( 1998 ) The StpA protein functions as a molecular adapter to mediate repression of the bgl operon by truncated H-NS in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_126">
The StpA protein func tions as a molecular adapter to mediate repression of the bgl operon by truncated H-NS in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></label>
<input id="act_similarity_126", type="checkbox">
<div class="ACT_similar">
<li>The StpA protein functions as a molecular adapter to mediate repression of the bgl operon by truncated H-NS in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li></div>
<li>Homers AM , van der Zeijst BAM , Gaastra W. Uh!in BE 11992 ) The positive regulator Clad overcomes the repression mediated by histone-like protein H-NS -LCB- HI ) in the CFA/i fimbrial operon of Escherichia colt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_129">
Jordi , B.J. , Dagberg , B. , de Haan , L.A. , Hamers , A.M. , van der Zeijst , B.A. , Gaastra , W. , and Uhlin , B.E. ( 1992 ) The positive regulator CfaD overcomes the repression mediated by histone-like protein H-NS ( H1 ) in the CFA/I fimbrial operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></label>
<input id="act_similarity_129", type="checkbox">
<div class="ACT_similar">
<li>Jordi , B.J.A.M. , Dagberg , B. , de Haan , L.A.M. , Hamers , A.M. , van der Zeijst , B.A.M. , Gaastra , W. , and Uhlin , B.E. ( 1992 ) The positive regulator CfaD overcomes the repression mediated by histone-like protein H-NS ( H1 ) in the CFA/I ﬁmbrial operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li></div>
<li>JOURNAL OF BACTERIOLOGY , June 1996 , p. 3418 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology e Histone-Like Protein H-NS Acts as a Transcriptional Repressor for Expression of the Anaerobic and Growth Phase Th Activator AppY of Escherichia coli TOVE ATLUNG ,1 * SUSANNE SUND ,2 KIRSTEN OLESEN ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 12 October 1995/Accepted 9 April 1996 The transcriptional-activator AppY is required for anaerobic and stationary-phase induction of the cyx-appA and hya operons of Escherichia coli , and expression of the appY gene itself is induced by these environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Induction of the ferritin gene ( ftnA ) of Escherichia coli by Fe2 - Fur is mediated by reversal of H-NS silencing and is RyhB independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21908668" target="_blank">21908668</a>]<br></li>
<li>The clyA locus in the E. coli K-12 chromosome does not seem to be expressed under most laboratory growth conditions , and our present evidence established that the gene is subject to silencing by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The Cryptic Adenine Deaminase Gene of Escherichia coli SILENCING BY THE NUCLEOID-ASSOCIATED DNA-BINDING PROTEIN , H-NS , AND ACTIVATION BY INSERTION ELEMENTS * Received for publication , May 1 , 2002 , and in revised form , June 10 , 2002 Published , JBC Papers in Press , June 19 , 2002 , DOI 10.1074 / jbc.M204268200 Carsten Petersen ‡ § , Lisbeth Birk Møller ¶ , and Poul Valentin-Hansen From the ‡ Department of Biological Chemistry , Institute of Molecular Biology , University of Copenhagen , Sølvgade 83H , DK1307 Copenhagen K , ¶ John .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>The mechanism of H-NS 's function on specific genes is not quite clear , though it has been suggested to be due to its effects on DNA conformation or its ability to Biodegradative arginine decarboxylase and lysine decarboxylase , encoded by adi and cadA , respectively , are induced to maximal levels when Escherichia coli is grown anaerobically in rich medium at acidic pH. Mutants formed by transposon mutagenesis , namely , GNB725 , GNB729 , GNB88 , GNB824 , and GNB837 , exhibited considerably elevated expression at pH 8.0 compared with the corresponding parental strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381784" target="_blank">8381784</a>]<br></li>
<li>These rest & s , together with our previous Ending that cells deficient in both HU and IHF are viable at 30-37 ° C [ Kane and Imamoto , Gene 89 ( 1990 ) I33-1371 showed that E. cnfi cells deficient in any two of these three histone-like proteins are viable at 30-37 '' C , and suggested that simultaneous deficiency of all three of the proteins is lethal , There were no detectable differences in the levels of superhelicity of the reporter plasmids isolated from cells deficient in either IHF or H-NS , or from wild-type cells , but about 1576 decrease in negative superhehcity was detected for the reporter plasmid isolated from cells lacking HU and lacking both HU and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Since RpoS and H-NS play an important role in globally regulating genes in response to changing environments , it is not surprising that AI-2-related QS networks utilize their uptake regulators ( e.g. , LsrR and LuxP ) , together with DsrA , in a hierarchical modality for mediating prompt responses to environmental stimuli and extracellular stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557827" target="_blank">17557827</a>]<br></li>
<li>Among the derivatives of MS368 containing random insertions , a clone showing derepression of ﬂhDC-lacZ even in the absence of H-NS was selected on indicator agar plates ( Luria-Bertani agar containing 50 mg of 5-bromo-4-chloro-3-indolyl-b-D-galactopyranoside [ X-Gal ] / ml ) and exhibited a consistent increase in LacZ activity ( 14 ) when transferred back to MS368 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Examples of such H-NS antagonism are many and include the action of VirB on icsB , SlyA on clyA and hlyA , UreR on ureR-ureD , RovA on inv , HilC/HilD on hilA , Ler on lpf and LeuO on osmS1 ( Westermark et al. , 2000 ; Bustamante et al. , 2001 ; Poore and Mobley , 2003 ; Cathelyn et al. , 2007 ; de la Cruz et al. , 2007 ; 2009 ; Lithgow et al. , 2007 ; Olekhnovich and Kadner , 2007 ; Torres et al. , 2007 ; Turner and Dorman , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The acid resistant island including the gadE-mdtEF operon indeed is one of the acquired gene clusters in E. coli with different GC content from the rest of E. coli genome ; Third , the interplay between the H-NS mediated repression and ArcA mediated antagonizing of the repression is also observed in the case of another gene cydAB which is also involved in adapting E. coli to the microaerobic and anaerobic-growth ( Govantes et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>More than ten transcription regulators including EvgA , YdeO , GadE , TorR , H-NS , PhoP , RpoS , CRP , MnmE , GadX , and GadW have been found to participate in the regulation of gadE during various circumstances of acid stresses through at least ﬁve different regulatory circuits ( Ma et al. , 2004 ; Sayed and Foster , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Involvement of EvgA , YdeO , H-NS , CRP , GadX , as well as a small RNA DsrA in the regulation of the mdtEF gene in the same operon has also been reported ( Nishino and Yamaguchi , 2002 , 2004 ; Oshima et al. , 2006 ; Nishino et al. , 2008a , b , 2009 , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>INVOLVEMENT OF DIFFERENT REGULATORS IN THE ANAEROBIC EXPRESSION OF mdtEF FROM THAT OF AEROBIC CONDITION Previous studies focusing on the expression of gadE-mdtEF operon during acid resistance have revealed that several regulators , such as EvgA , YdeO , MnmE , GadE , GadX , CRP , and H-NS , are involved in the regulation of gadE-mdtEF operon ( Gong et al. , 2004 ; Ma et al. , 2004 ; Hirakawa et al. , 2006 ; Tramonti et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Subsequently , it was found to be engaged in other site-specific DNA recombination processes ( Ball & Johnson , 1991 ; Thompson et al. , 1987 ; Weinreich & Reznikoff , 1992 ) , to alter DNA supercoiling ( Schneider et al. , 1997 , 2001 ; Skoko et al. , 2005 ; Travers et al. , 2001 ) , to influence the initiation of DNA replication Abbreviations : CRP , cyclic AMP receptor protein ; H-NS , histone-like nucleoid structuring protein ; IHF , integration host factor ; qRT-PCR , reverse transcriptase quantitative PCR ; RNAP , RNA polymerase ; WMW , Wilcoxon -- Mann -- Whitney ; WT , wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Factors required for motility and full flhDC expression include cyclic AMP ( cAMP ) and cAMP receptor protein ( CRP ; Kutsukake , 1997 ; Soutourina et al. , 1999 ) , histone-like nucleoid-structuring ( H-NS ) protein ( Bertin et al. , 1994 ) , the heat-shock response network proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) , Pss and Psd , which are involved in the synthesis of phosphatidy-lethanolamine in cell membranes ( Shi et al. , 1993 ) , and inorganic polyphosphate ( Rashid et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>The histone-like nucleoid-structuring protein ( H-NS ) is also involved in the recombination event ( Donato et al. , 1997 ; O'Gara and Dorman , 2000 ; Olsen & Klemm , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207912" target="_blank">16207912</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_147">
Ussery , D.W. , Hinton , J.C.D. , Jordi , B.J.A.M. , Granum , P.E. , Seirafi , A. , Stephen , R.J. , Tupper , A.E. , Berridge , G. , Sidebotham , J.M. , and Higgins , C.F. ( 1994 ) The chromatin-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></label>
<input id="act_similarity_147", type="checkbox">
<div class="ACT_similar">
<li>Ussery , D.W. , Hinton , J.C. , Jordi , B.J. , Granum , P.E. , Seirafi , A. , Stephen , R.J. , et al. ( 1994 ) The chromatin-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Ussery , D.W. , Hinton , J.C.D. , Jordi , B.J.A.M. , Granum , P.E. , Seiraﬁ , A. , Stephen , R.J. , et al. ( 1994 ) The chromatin-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Ussery , D.W. , Hinton , J.C.D. , Jordi , B. , Granum , P.E. , Seiraﬁ , A. , Stephen , R.J. , et al. ( 1994 ) The chromatin-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li></div>
<li>Interestingly FIS stimulated the activity of the hns promoter when H-NS was absent and alleviated the inhibition of H-NS when the protein was present ( Falconi et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>One of these , the outer membrane porin protein OmpF , was found to be regulated not only by H-NS as described previously ( Graeme-Cook et al. , 1989 ; Suzuki et al. , 1996 ) , but also by the StpA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Another protein believed to be implicated in the organization of bacterial chromatin is the 15.4 kDa protein H-NS ( probably homologous to Bl [ 5 ] and to one of the Hl variants reported by Spassky et al. [ 6 ] ) , which has recently been characterized and A. La Teana * + , M. Falconi * , V. Scarlato '' , M. Lammi and C.L. Pan ' Correspondence address : C.L. Pon , Max-Planck-Institut fu Molekulare Genetik , Abt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2494066" target="_blank">2494066</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_154">
Smyth , C.P. , Lundback , T. , Renzoni , D. , Siligardi , G. , Beavil , R. , Layton , M. , Sidebotham , J.M. , Hinton , J.C. , Driscoll , P.C. et al. ( 2000 ) Oligomerization of the chromatin-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></label>
<input id="act_similarity_154", type="checkbox">
<div class="ACT_similar">
<li>Smyth , C.P. , Lundback , T. , Renzoni , D. , Siligardi , G. , Beavil , R. , Layton , M. , et al. ( 2000 ) Oligomerization of the chromatinstructuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_156">
Smyth , C.P. , Lundbäck , T. , Renzoni , D. , Siligardi , G. , Beavil , R. , Layton , M. , et al. ( 2000 ) Oligomerization of the chro-matin-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></label>
<input id="act_similarity_156", type="checkbox">
<div class="ACT_similar">
<li>Smyth , C.P. , Lundbäck , T. , Renzoni , D. , Siligardi , G. , Beavil , R. , Layton , M. , et al. ( 2000 ) Oligomerization of the chromatin-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Smyth , C.P. , Lundbäck , T. , Renzoni , D. , Siligardi , G. , Beavil , R. , Layton , M. , et al. ( 2000 ) Oligomerization of the chromatin-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li></div>
<li>After leucine-responsive protein , H-NS and σs , NhaR and RcsB are the fourth and ﬁfth regulators of osmC transcription to have been identiﬁed The participation of so many factors in the regulation of osmC illustrates the notion of cooperation of global regulators in the ﬁne-tuning of stress-inducible genes ( Bouvier et al. , 1998 ; Hengge-Aronis , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></li>
<li>These include the transcription factor FNR that mediates anaerobic regulation ( Unden and Schirawski , 1997 ) , the cAMP receptor protein ( CRP ) , which activates transcription of catabolic operons ( Kolb et al. , 1993 ) , and the nucleoid-associated protein H-NS ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Although the transcription factor Fis helps to recruit RNAP to rrn P1 promoters ( Bokal et al. , 1995 ) , and the nucleoid protein H-NS can inhibit rRNA promoter activity ( Afflerbach et al. , 1998 ) , most regulation of rRNA transcription is not a function of the action of classical activator or repressor proteins ( Bartlett and Gourse , 1994 ; Murray et al. , 2003b ; Schneider et al. , 2003b ; Murray and Gourse , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15294157" target="_blank">15294157</a>]<br></li>
<li>During growth in minimal medium , the leucine-responsive protein has been shown to repress the transcription of osmC and to activate that of osmC , p '' p # and the nucleoid-associated protein H-NS represses transcription from both promoters ( Bouvier et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></li>
<li>The other is the nucleoid-associated protein H-NS or StpA , which binds to the promoter region of the ydiV gene and may negatively E regulate its transcription ( Uyar et al. , 2009 ; Kahramanoglou et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>Pleiotropic effects of increased ArcZ expression Pioneering discoveries in the ﬁeld of sRNA-mediated gene regulation were based on fortuitous overexpression of afore unknown sRNA genes in E. coli , e.g. the discoveries of the micF or dsrA sRNA genes through their multicopy effects on OmpF or H-NS protein synthesis ( Mizuno et al. , 1984 ; Sledjeski and Gottesman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19732340" target="_blank">19732340</a>]<br></li>
<li>Although the possibility exists that H-NS interacts directly with the gad promoters by negatively regulating their expression , it can also be hypothesized that this nucleoid protein indirectly affects the expression of gad via post-transcriptional regulation of RpoS ( ss ) expression ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>However , the effect of H-NS on gad expression is likely to be indirect because of the involvement of this protein in the post-transcriptional control of ss expression ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Results sS and H-NS are implicated in hchA regulation In an early inventory of genes regulated by the nucleoidbinding protein H-NS , Yoshida and coworkers identiﬁed four unknown proteins whose production was upregulated in a Dhns mutant ( Yoshida et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Many of the target genes regulated by LeuO are repressed by the pleiotropic H-NS ( heat-stable nucleoid structuring ) protein , and thus LeuO emerged as important H-NS antagonist and global regulator ( Hernandez-Lucas et al. , 2008 ; Shimada et al. , 2009 ; 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>This promoter is controlled by a number of global regulators such as CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , heat-shock proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) and H-NS ( Bertin et al. , 1994 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) , as well as by several environmental signals such as temperature control ( Qualding and Stocker , 1962 ; Adler and Templeton , 1967 ) , high concentrations of inorganic salts , carbohydrates or alcohols ( Li et al. , 1993 ; Shi et al. , 1993 ) , growth-phase ( Nishimura and Hirota , 1989 ) , DNA supercoiling ( Li et al. , 1993 ; Shi et al. , 1993 ; Kutsukake , 1997 ; Das et al. , 1998 ) , phosphatidylethanolamine and phosphatidylglycerol synthesis ( Shi et al. , 1993 ; Mizushima et al. , 1995 ) , cell cycle control ( Morgan et al. , 1995 ; Prüß and Matsumura , 1996 ; Mizushima et al. , 1997 ) and quorum sensing ( Sperandio et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_170">
The 32p-labeled fragment ( 0.1 pmol ) was incubated with purified H-NS in the absence and presence of competitor DNA ( a 233 bp synthetic non-curved DNA , designated S15-12 ; Yamada et al. 1990 ) , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></label>
<input id="act_similarity_170", type="checkbox">
<div class="ACT_similar">
<li>labeled fragments ( 0.1 pmol ) were incubated with purified H-NS in the presence or absence of a competitor DNA ( a 233 bp synthetic non-curved DNA , designated $ 15-12 ; Yamada et al. 1990 ) , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li></div>
<li>During the transition from exponential to stationary phase , when Fis levels are continually declining while elevated levels of the nucleoid-associated proteins Dps , IHF , HU , Hfq and H-NS are maintained ( Ali Azam et al. , 1999 ) , a global redistribution of chromosomal DNA branching can affect the accessibility of RNAP to numerous promoters to indirectly alter gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The synthesis of IHF is indeed growth-phase regulated ( as is the case for the abundant H-NS and Dps proteins ; Almiron et al. , 1992 ) and an elevated concentration in stationary phase could help either to compact DNA , or protect it against chemical or physical attacks ( as was proposed for HU ; Boubrik & Rouviere-Yaniv , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Nucleoid compaction and DNA-associated proteins DNA-associated proteins such as H-NS , HU , IHF , and Fis have often been suggested to be responsible for DNA compaction in E. coli , based among other things on their colocalization with DNA in vivo ( Azam et al. , 2000 ) , on their occurrence in isolated nucleoids that are prepared in the absence of high salt concentrations ( Murphy and Zimmerman , 1997 ; Pettijohn , 1996 ) , and on their ability to bend or compact DNA in vitro ( Dame , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>Also in accord , Brunetti et al. ( 2001 ) found that individual deletions of H-NS , HU , Fis , or IHF did not appear to change the large scale structure of the E. coli chromosome , based upon the lack of changes in the in vivo genomic fragmentation patterns by DNA gyrase in the presence of oxolinic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>This observation is in marked contrast to the effects of other DNA-binding proteins such as H-NS and SASP , whose overexpression in actively growing E.coli leads to a massive DNA condensation and rapid loss of viability ( Setlow et al. , 1991 ; Spurio et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>In exponential phase E. coli cells , the order of intracellular concentration of these DNA-binding proteins is , in descending order , Fis , HU , StpA , H-NS and IHF ( Talukder et al. 1999 ) , among which HU , StpA , H-NS and IHF have been considered to participate in the condensation of genome DNA to form the compact nucleoid ( for reviews see Pettijohn 1982 , 1988 , 1996 ; Drlica & Rouviere-Yaniv 1987 ; Rouviere-Yaniv & Yaniv 1979 ; Murphy & Zimmerman 1997a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>Falconi et al. ( 1991 ) suggested that H-NS potentiates the activity of transcriptional-repressors , Our results are in accordance with these hypotheses and suggest that H-NS binding might stabilise or mould bent DNA and thereby reduce the ability of Lrp to activate R N A polymerase , perhaps by using the elastic energy stored in the induced bend , as suggested by Wang and Calvo ( 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Since H-NS is known to-bind DNA of incoming phage or plasmid directly ( Navarre et al. , 2006 ; Navarre et al. , 2007 ) this might result in redistribution of H-NS ( Doyle et al. , 2007 ; Dillon et al. , 2010 ) , allowing expression of the Cascade genes due to decreased local concentrations of the repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>The target DNA sites in different operons shown to be bound by Lrp or H-NS have features in common and are often naturally bent DNA regions ; several protein molecules may bind to such curved DNA ( Yamada et al. , 1990 , 1991 ; Newman et al. , 1992 ; Owen-Hughes et al. , 1992 ; Calvo and Matthews , 1994 ; Zuber et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Citation : Yun SH , Ji SC , Jeon HJ , Wang X , Kim SW , et al. ( 2012 ) The CnuK9E H-NS Complex Antagonizes DNA Binding of DicA and Leads to TemperatureDependent Filamentous Growth in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>The H-NS is reported to form a dimer and to-bind to DNA nonspecifically , introducing significant compaction of DNA but no appreciable change in the linking number ( Lammi et al. , 1984 ; Falconi et al. , 1988 ; Spassky et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>The active form of H-NS is believed to be dimeric , although larger oligomers have been detected in solution ( Falconi et al. , 1988 ; Smyth et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>At present , it is not understood how recognition and/or activation by Es70 is avoided within the context of the cell , but the concentration of salts ( Ding et al. , 1995 ) , negative supercoiling of the DNA ( Kusano et al. , 1996 ) and protein factors such as H-NS ( Arnqvist et al. , 1994 ; Barth et al. , 1995 ; Hengge-Aronis , 1996a ) have been proposed to be involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Although the RovA protein in Yersinia pseudotuberculosis has been shown to directly activate transcription by stimulating RNA polymerase ( RNAP ) in the absence of other accessory factors , it is thought that SlyA-type proteins generally activate transcription indirectly , by binding to DNA and modulating the activity of other factors such as PhoP or the nucleoid-associated protein H-NS ( Heroven et al. , 2004 ; Navarre et al. , 2005 ; Tran et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>It is known that the binding of FIS , H-NS , LRP and StpA occurs preferentially to curved or flexible DNA ( Hil-lebrand et al. , 2005 ; Pul et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>LRP and H-NS bind to the same DNA sequences , yet they differ in binding mechanisms In comparison with H-NS , which also binds to all seven rRNA P1 promoter upstream regions ( Hillebrand et al. , 2005 ) , LRP complexes were formed at notably lower ( 2-to 20-fold ) protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>In addition , the T7 5.5 protein binds to the H-NS protein and can inhibit its oligo-merization with DNA ( Ali et al. 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22327802" target="_blank">22327802</a>]<br></li>
<li>This result is in agreement with previous data on H-NS , suggesting a co-operative binding of this protein to DNA ( Rimsky and Spassky , 1990 ; Lucht et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>As demonstrated before ( Hillebrand et al. , 2005 ) , H-NS binding occurred cooperatively , and a rather diffuse range of shifted DNA fragments with decreasing electrophoretic mobility is apparent at increasing protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Further , it was proposed that LeuO acts as a positive regulator of the leu operon and leuO gene with activation based on transcription induced changes in DNA supercoiling and delimiting of H-NS spreading by LeuO ( Fang and Wu , 1998b ; Chen et al. , 2003 ; Chen and Wu , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>This event is sensitive to topological changes in the DNA ( Glasgow et al. 1989 ) , and our result that increased osmZ gene dosage reduces the frequency of fimbrial phase variation can be explained by the effect of H-NS on DNA rigidity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2177526" target="_blank">2177526</a>]<br></li>
<li>H-NS can bind to DNA and form higher-order oligomeric H-NS complexes in a concentration-dependent manner , changing DNA topology and interfering with the action of other transcription factors ( Tupper et al. , 1994 ; Esposito et al. , 2002 ; Rimsky , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In vitro the formation of higher order structures of H-NS on the DNA is reduced at elevated temperatures ( 37 ∞ C versus 25 ∞ C ) and by increased salt concentrations ( Amit et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>For instance , the sequence speciﬁc DNA bending proteins IHF and Fis can relieve repression by H-NS at speciﬁc promoters ( Tippner et al. , 1994 ; van Ulsen et al. , 1996 ; Afﬂerbach et al. , 1998 ; Falconi et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>12 C. Madrid , C. Balsalobre , J. García and A. Juárez H-NS multimerization and H-NS-dependent DNA bridging are mediated by H-NS dimers and tetramers ( Dorman , 2004 ; Stella et al. , 2005 ; Dame et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>H-NS bridging between nearby segments of DNA can generate DNA loops or hairpins ( Dame et al. , 2001 ; 2005 ) which effectively inhibit RNA polymerase access and/or passage along the DNA template , resulting in repression ( Schröder and Wagner , 2000 ; Dame et al. , 2002 ; 2005 ; Wyborn et al. , 2004 ; Gralla , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>For instance , one of the peculiar properties of H-NS is to bridge neighbouring DNA strands , which potentially leads to repression structures ( Dame et al. , 2002 ; 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>There is increasing evidence that DNA topology in bacterial cells changes in response to environmental signals such as osmolarity , anaerobicity , temperature and nutrient status , and these alterations involve H-NS activity ( Higgins et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Under physiological conditions in vitro H-NS binds tightly to double-stranded DNA and increases the thermal stability of DNA ( Higgins et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>In other cases , changes in DNA supercoiling have been proposed to mediate H-NS repression ( Hulton et al. , 1990 ; Hinton et al. , 1992 ; Owen-Hughes et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>H-NS regulatory activity depends both on its preferential interaction with intrinsically curved DNA ( Yamada et al. , 1990 ; Owen-Hughes et al. , 1992 ) and on its ability to induce bending of non-curved DNA ( Lucht et al. , 1994 ; Spurio et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>The binding sites of MlrA , OmpR , RstA , CpxR , IHF and H-NS were determined by DNase-1 footprinting assays ( Fig. 2 ; and Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>CRP , CpxR , CsgD , H-NS , IHF , MlrA , OmpR , and RstA on the csgD promoter ( Ogasawara et al. 2010a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>1 NarL and NarP counteract the negative effects of Fis and IHF but not H-NS Wu et al. ( 1998 ) proposed that NarL and NarP function at pnir , in response to nitrite , by counteracting the repressive effects of upstream factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The activity of sS is ﬁne tuned by a number of global regulators , including CRP , Lrp , IHF , H-NS and Crl ( Hengge-Aronis , 1996b ; Loewen et al. , 1998 ; Pratt and Silhavy , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>A number of the rsdependent genes are also regulated through the action of one or several additional global regulators like CRP-cAMP , Fis , IHF , Lrp or H-NS ( see for instance Xu and Johnson 1995a , 1995b , 1995c ; Bouvier et al. 1998 ; Colland et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12395194" target="_blank">12395194</a>]<br></li>
<li>These include activating factors as , e.g. cAMP-CRP , H-NS and CsrA as well as repressing factors such as RcsB/A , OmpR and LrhA ( recently summarized in Prüß et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537210" target="_blank">17537210</a>]<br></li>
<li>Expression of FlhDC is tightly regulated by multiple transcription factors , including CRP , H-NS and OmpR , and hence the genes of the ﬂagellar cascade are regulated in response to many different environmental inputs ( Shin and Park , 1995 ; Sout-ourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>0002-8080 G 2005 SGM Printed in Great Britain Silencing involves two sites ( Dole et al. , 2004b ) : H-NS represses transcription initiation at the cAMP receptor protein ( CRP ) - dependent promoter by binding to an AT-rich and presumably bent upstream silencer sequence ( Schnetz , 1995 ; Singh et al. , 1995 ; Schnetz & Wang , 1996 ; Mukerji & Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Fis can also affect DNA supercoiling indirectly , because of its roles in repressing transcription of gyrA and gyrB to reduce the DNA gyrase activity ( Schneider et al. , 1999 ) , increasing topA transcription to enhance topoisomerase I activity ( WeinsteinFischer et al. , 2000 ) , and regulating the expression of other DNA-binding proteins such as H-NS , HU and CRP ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) that also affect DNA supercoiling directly or indirectly ( Broyles & Pettijohn , 1986 ; Gomez-Gomez et al. , 1996 ; Tupper et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>However , it is possible that the change in transcription activity of PguaB in a fis background occurs as a result of altered regulation of PguaB by a transcription factor other than CRP , for example H-NS or HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ) or by changes in supercoiling ( Schneider et al. , 1997 ; Weinstein-Fischer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>A similar phenomenon is observed at the Shigella flexneri virB gene , where the positive regulator VirB relieves H-NS repression , but also activates transcription directly , as well as in several other systems ( Tobe et al. , 1993 ; Murphree et al. , 1997 ; Coker et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Although a K12DstpA strain showed similar cas gene transcript levels as a wt K12 strain ( Pul et al. , 2010 ) it can not be excluded that StpA-mediated repression of cas gene transcription takes place in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>Another study showed that in this strain the transcription of cas3 , casB , casC and casD is elevated in the absence of H-NS ( Navarre et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>Finally , the effects could be indirect , for instance when overexpression of H-NS leads to the ` complete ' shut down of transcription , resulting in strongly reduced transertion mediated nucleoid expansion and , thus , compact nucle-oids ( Spurio et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Interestingly , Fis and H-NS have both previously been shown to selectively regulate transcription by Es70 and Es38 , although by mechanisms that differ to those presented here ( reviewed in Typas et al. , 2007a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>© 2004 Blackwell Publishing Lt Activation by Fis and repression by H-NS Fis protein has been shown to activate ribosomal RNA transcription both in vitro and in vivo ( Ross et al. , 1990 ; Schneider et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686546" target="_blank">15686546</a>]<br></li>
<li>For instance , transcription of the bgl operon is shut off in presence of H-NS ( Dole et al. , 2004 ) , and LeuO relieves this silencing ( Stratmann et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>For Regulation of rRNA transcription by H-NS and LRP 873 instance , H-NS is involved in the LRP-dependent activation of the ilvIH operon ( Levinthal et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>LRP transcription is , for instance , negatively regulated by H-NS ( Oshima et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Model to explain H-NS-mediated Fe2 + -- Fur activation of ftnA H-NS acts as a global repressor of transcription in E. coli and many other bacteria ( Westermark et al. , 2000 ; Hommais et al. , 2001 ; Grainger et al. , 2006 ; Lucchini et al. , 2006 ; Navarre et al. , 2006 ; Stoebel et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>H-NS antagonizes the FIS-mediated activation of the E. coli rrn-operons in vivo In previous studies , the effect of H-NS as a repressor of rRNA transcription was shown under in vitro conditions ( Tippner et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Later , in vitro and in vivo studies had shown that H-NS acts as a repressor of rRNA transcription and contributes to the rapid shutdown during the onset of stationary phase or under stress conditions ( Tippner et al. , 1994 ; Afﬂerbach et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>In conclusion , the in vitro transcription experiments indicate that H-NS represses ftnA transcription directly , as reported for other H-NS-controlled genes ( Göransson et al. , 1990 ; Tupper et al. , 1994 ) , and that Fur acts directly as an antirepressor , reversing the effect of H-NS transcriptional silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In contrast to its much more general role as a transcriptional-repressor or silencer ( Dorman , 2004 ; Rimsky , 2004 ) , H-NS was previously observed to activate csgD transcription by an unknown mechanism ( Fig. 8 and Gerstel et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Because this effect is abolished in hns null cells and because the levels of H-NS increase under cold-shock conditions ( La Teana et al. , 1991 ) , this phenom-enon is adequately explained by more effective H-NS-mediated negative regulation of hchA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>On the other hand , it is remarkable that most of the reports about the inﬂuence of H-NS on transcription emphasize its role under limiting growth conditions , particularly at increased osmolarity , suboptimal temperature , pH variations or poor availability of carbon sources ( Arnquist et al. , 1994 ; Manna and Gowrishankar , 1994 ; O'Byrne and Dorman , 1994 ; Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Although the low level of micF RNA in an hns wild-type strain prevented us from accurately comparing its half-life in the presence and absence of H-NS , it has been reported previously that H-NS does not affect the stability of micF ( Suzuki et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>There is a class of promoters that is regulated in a similar manner to that of hdeAB and that is transcribed by E 38 , but not by E 70 in the presence of H-NS ( Arnqvist et al. 1994 ; Robbe-Saule et al. 1997 ; Marschall et al. 1998 ; Waterman and Small 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Arnqvist et al. ( 1994 ) reported that H-NS prevents the functioning of E-s70 on this promoter , while E-ss is able to transcribe it , even in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>OmpT-mediated proteolytic cleavage of H-NS has previously been reported ( Goldberg et al. , 1997 ) and most probably accounts for the presence of such protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>No H-NS effect was observed when cells were grown at 37 ° C ( Fig. 2C , ii ) consistent with previous observations that H-NS activity is regulated by temperature ( Ceschini et al. , 2000 ; Ono et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>H-NS , another of the histone-like proteins , is reported to be more abundant in cells grown on complex medium as compared with minimal medium ( Falconi et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>In several cases , FIS has been found to regulate genes that are controlled by H-NS and CAP respectively ( Xu and Johnson , 1995b , 1995c , 1997 ; Claret and RouvieÁre-Yaniv , 1996 ; Falconi et al. , 1996 ; GonzaÂlez-Gil et al. , 1996 , 1998 ; Afflerbach et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>Interestingly , the three genes rcsA , gadE and bglJ , respectively , encoding the three RcsB heterodimerization partners are all repressed by H-NS ( Majdalani and Gottesman , 2005 ; Stratmann et al. , 2008 ; Sayed and Foster , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>This is consistent with observations in other H-NS-repressed operons , in which H-NS is antagonized by different positive regulators ( Forsman et al. , 1992 ; Jordi et al. , 1992 ; Tobe et al. , 1993 ; Tippner et al. , 1994 ; Falconi et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Silencing of the bgl operon by H-NS occurs under all laboratory growth conditions tested ( Prasad & Schaeﬂer , 1974 ; Reynolds et al. , 1981 ; Lopilato & Wright , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Similarly , the bgl operon , encoding the gene products necessary for the uptake and fermentation of aryl-b , D-glucosides is repressed ~ 100-fold by H-NS ( Higgins et al. , 1988 ; Lopilato and Wright , 1990 ; Schnetz , 1995 ; Caramel and Schnetz , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>( iii ) H-NS negatively regulates lysine decarboxylase CadA and cadaverine/lysine antiporter CadB , essential for lysine-dependent system ( Atlung and Ingmer , 1997 ; Hommais et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>By means of sequence-speciﬁc base-pairing , DsrA enhances the stability and translation of RpoS mRNA while blocking H-NS translation and increasing hns mRNA turnover ( Lease et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>We know that DsrA not only stimulates RpoS translation , it downregulates H-NS ( Sledjeski and Gottesman , 1995 ; Lease et al. , 1998 ) , and the regions of dsrA necessary for RpoS stimulation are independent of those regulating H-NS ( Majdalani et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251852" target="_blank">11251852</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_245">
H-NS negatively regulates RpoS translation ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></label>
<input id="act_similarity_245", type="checkbox">
<div class="ACT_similar">
<li>In addition , H-NS negatively regulates the amount of RpoS ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li></div>
<li>Consistent with our results , ChIP-Chip analysis performed by Oshima and collaborators demonstrated that the H-NS protein indeed binds to the ﬁrst gene of the ycb , ybg , yfc , yra , sfm and yeh operons and polymerizes on entire yad operon ( Oshima et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>H-NS also binds , albeit with a much lower affinity , to ssDNA and to various types o f R N A ( Lammi et al. 1984 ; Gualerzi et al. 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2841565" target="_blank">2841565</a>]<br></li>
<li>Instead , Hha binds to H-NS , which binds to speciﬁc regions of regulatory sequences in this operon ( Nieto et al. , 2000 ; Madrid et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>CHIP-chip data of H-NS and ArcA are also consistent with this proposal , as characterized by a broad peak of H-NS binding to PgadE under aerobic conditions and the replacement of this broad peak by that of ArcA under anaerobic conditions ( Park and Kiley et al. , unpublished data , pers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Similarly , HU is proposed to counteract the effects of H-NS , possibly by competing for preferential binding sites ( Dame and Goosen , 2002 ; van Noort et al. , 2004 and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>These results conﬁrm and extend the observations of Westermark et al. ( 36 ) and suggest that , either directly or indirectly , SlyA activates hlyE expression by modulating H-NS binding and ultimately by counteracting the negative effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>RcsB is indirectly affected by H-NS via repression of the regulator of capsule synthesis D , RcsD , that phosphorylates and activates RcsB , and of the accessory factor RcsA , that helps RcsB binding to the regulatory region of the ﬂhDC ﬂagellar master operon to repress its expression ( Table 2 ) ( Francez-Charlot et al. , 2003 ; Sledjeski and Gottesman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Overall , the above results suggest that hchA heat-induction relies primarily on the relief of H-NS silencing upon thermal upshift , possibly resulting from a conformational change in the DNA ( Falconi et al. , 1998 ) , and on a secondary mechanism that may involve direct or indirect activation of its EsD-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>In vivo H-NS is essential for silencing ( Higgins et al. , 1988 ) and activated promoter derivatives are CAP dependent ( Reynolds et al. , 1981 , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>A differential effect of repressors or activators on transcription by Es 70 and Es can lead to a specific induction upon accumulation s of s , as described for the effect of H-NS on the csgBA s promoter ( Arnqvist et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Several lines of evidence support this proposal : ﬁrst , bioinformatics have suggested the presence of an efﬁcient DNA helix locking mechanism by H-NS in the promoter of gadE ( Hommais et al. , 2004 ) ; Second , H-NS mediated repression has been found to be common among acquired gene clusters which is consistent with the functions of these gene clusters in conferring the survival and adaptation of bacteria in various adverse growth environments ( Oshima et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>It has been noted previously that luxAB fusions can substitute for DNA downstream of a promoter involved in repression by H-NS , because of the presence of curved DNA in the luxA gene which binds H-NS ( OwenHughes et aL , 1992 ; Forsberg et aL , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Therefore , the stationary-phase increase in gadX promoter activity may be ascribed to the well-known relief of H-NS repression , which occurs at this stage of growth ( Tramonti et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>During stationary phase , H-NS levels vastly exceed Fis levels ( Ali Azam et al. , 1999 ) , contributing to the repression of ribosomal RNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686546" target="_blank">15686546</a>]<br></li>
<li>This result indicates that the repression of hdeABp by H-NS is not through a direct interaction with E 70 ( Choy and Adhya 1996 ; Roy et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Rather , the region upstream from P1 is required for H-NS repression of cydAB ( F. Govantes et al. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10998176" target="_blank">10998176</a>]<br></li>
<li>Low expression is in agreement with repression of leuO by H-NS ( Chen et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>As activation by ArcA is maximal in microaerobiosis ( Tseng et al. , 1996 ) , we favour the hypothesis that ArcA anti-repression prevents H-NS control in microaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>They suggest that the mutations mapping in cis to the promoter activate the operon by relieving the repression of the promoter by H-NS ( Higgins et al. , 1988 ; Lopilato and Wright , 1990 ; Schnetz and Wang , 1996 ; Mukerji and Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>It has also been shown that LeuO when binding in between an H-NS bound fragment and a promoter can delimit repression by H-NS ( Chen et al. , 2003 ; Chen and Wu , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>As binding of H-NS to areas upstream and downstream of the promoter is needed for repression , these regions were proposed to be bridged to form a loop ( Dole et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>This finding indicates that physiological conditions in the host could induce bgl expression , although so far no culture conditions were defined that relieve silencing of the bgl promoter , and it relates to the essential role of H-NS , which is a regulator of virulence , stress induced and other genes ( reviewed in Ussery et al. , 1994 ) , in bgl silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>Other examples exist where IHF and H-NS interplay to modulate gene expression : in the hilA promoter of the S. enterica , IHF is required to alleviate H-NS-mediated silencing depending upon growth conditions ( Queiroz et al. , 2011 ) ; P. van Ulsen and N. Goosen have reported that , in vivo , in E. coli the extent of IHF-mediated Pe promoter activation differs in the presence of H-NS ( van Ulsen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Received on February 9 , 1995 ; revised on June 10 , 1995 Note added in proof Recently , data on the expression and function of StpA and H-NS have also been reported by Zhang et al. [ Zhang , A. , Rimsky , S. , Reaban , M.E. , Buc , H. and Belfort , M. ( 1996 ) EMBO J. , 15 , 1340-1349 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>In previous work , this porin was found to be reduced in expression in the absence of H-NS ( Graeme-Cook et al. , 1989 ; Suzuki et al. , 1996 ) ; our current study suggests that StpA might collaborate with H-NS in modulating ompF gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Secondly , stpA expression is itself low in rich medium when H-NS is present ( SondeÂn and Uhlin , 1996 ; Zhang et al. , 1996 ; Free and Dorman , 1997 ) , so the StpA protein is not present at high levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>This is consistent with previous analyses showing that StpA is generally subservient to H-NS in controlling gene expression , probably primarily because of its low expression in the presence of its homologue ( Sonde Ân and Uhlin , 1996 ; Zhang et al. , 1996 ; Free and Dorman , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>First , micF expression is low under standard growth conditions in the presence of H-NS ( Figs 5 , 6 and 8 ; Suzuki et al. , 1996 ) , so the substrate for StpA action is not abundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Several ss-dependent genes exhibit enhanced expression in the presence of hns mutations , and H-NS might affect the expression of some genes through its effect on ss ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Both cai and ® x operons require the presence of FNR and CRP for their expression , while they are partially repressed by H-NS ( Eichler et al. , 1995 ; Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Thus , neither Lrp nor IHF has a signiﬁcant effect on ﬁmB expression under the conditions described in this paper , and H-NS represses the expression of the recombinase ( Blomﬁeld et al. , 1993 ; 1997 ; Donato et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890032" target="_blank">12890032</a>]<br></li>
<li>Moreover , lrp expression is regulated autogenously by Lrp , as has been suggested previously ( Lin et al. 1992b ; Nakamura and Ito 1993 ; Wang et al. 1994 ) , and is further controlled by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Many genes where expression is affected by Lrp have actually been shown to be subjected to the effects , either directly or indirectly , of H-NS , e.g. papB , fin7B , ilvIH and lvsU ( Kawula and Omdorff , 1991 ; Van der Woude et al. , 1992 ; Ito et al. , 1994 ; Levinthal et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>These environmental signals are mediated at the gene expression level by a number of regulators , such as OmpR , H-NS , Besides its role as activator of the csgBA operon , CsgD regulates a number of genes involved in biofilm formation and production of cell-surface-associated structures ( Latasa et al. , 2005 ; Gibson et al. , 2006 ) , as well as genes involved in transport , metabolism and gene regulation ( Chirwa & Abbreviation : CV , crystal violet .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18599830" target="_blank">18599830</a>]<br></li>
<li>However , since under laboratory growth conditions the genomic leuO gene itself is repressed by H-NS ( Klauck et al. , 1997 ; Majumder et al. , 2001 ) all LeuO regulation studies make use of plasmid encoded leuO under control of constitutive or inducible promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>This confirms that H-NS dominates over StpA in global gene regulation under these growth conditions , but that StpA can make a significant contribution when H-NS is absent , as found previously ( Zhang et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>In the present study we focused on the genetic control of the structural gene for H-NS and show that hns expression , in addition to its already known cold-shock regulation ( La Teana et al. , 1991 ) , is growth-phase controlled and negatively autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>The expression of leuO is likewise repressed by H-NS , and not detectable during exponential-growth ( Klauck et al. , 1997 ; Chen et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>H-NS silencing of gene expression is relieved in response to a number of environmental stimuli , including growth-phase and osmolarity ( Hulton et al. , 1990 ; Atlung and Ingmer , 1997 ; Laurent-Winter et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>As leuO expression is negatively regulated by H-NS and positively by LeuO itself ( Hommais et al. , 2001 ; Chen et al. , 2005 ) , this would further amplify the activating signal for cas gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>Under acidic condition , cAMP and Crp indirectly affect gadW gene expression by inhibiting the production of RpoS ( Ma et al. 2003 ) , where RpoS is also under control of H-NS ( Jung and Kim 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>The amount of H-NS within the cell is therefore likely to be under tight control , which is presumably achieved by the negative autoregulation of the hns gene ( Dersch et al. , 1993 ; Ueguchi and Mizuno , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The fimB gene is subject to several regulatory inputs - , suggesting that it can be exp ; essed over a range of levels in vivo : It is under positive control by LRP ( Blomfield et al. , 1993 ) , negative control by H-NS ( Dove , 1996 ; Olsen and Klemm , 1994 ) , is stimulated by a relaxation ot DNA supercoiling ( Dove and ~ o r m a n1 , 994 ) and is differentially expressed in liquid and on solid growth media ( Schwan et al. , 1992 ; 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
<li>The increased H-NS levels may be involved in shutting down gene expression during cold-shock , and H-NS appears to be important for the adaptation to low temperature ( Dersch et aL , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>The gadB and gadC genes form an operon and a number of factors , including H-NS , RpoS and CRP-cAMP , are involved in the transcriptional control of gad expression ( Castanie-Cornet & Foster , 2001 ; De Biase et al. , 1999 ; Rowbury , 1997 ; Rowland et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>The OmpR regulator is directly involved in activation of curli expression , in concert with several other regulatory proteins such as RpoS , H-NS , CpxR , and other yet unknown factors ( Romling et al. , 1998b ; Prigent-Combaret et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>Among the known bacterial proteins involved in phase-depend-ent transcriptional regulation ( Azam et al. , 1999 ; Dame , 2005 ) , we have selected H-NS , StpA , LRP and FIS as potential candidates for our studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>Medina-Aparicio , L. , Rebollar-Flores , J.E. , GallegoHernandez , A.L. , Vazquez , A. , Olvera , L. , Gutierrez-Rios , R.M. , et al. ( 2011 ) The CRISPR/Cas immune system is an operon regulated by LeuO , H-NS and LRP in Salmonella enterica serovar Typhi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Synthesis of ppGpp is not regulated by H-NS The major contribution responsible for rRNA synthesis shut-off known so far had been correlated to the concentration of the global effector-molecule guanosine tetraphosphate ( ppGpp ) ( Cashel et al. , 1996 ; see also Zacharias et al. , 1989 ; Krohn and Wagner , 1996 ; Heinemann and Wagner , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>However , in hns mutants , the PgadE-lacZ activities increased more than 10-fold to about 250 Miller units , suggesting H-NS strongly represses the expression of gadE-mdtEF under aerobic condition in the M9 glucose minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>DELETION OF EvgA , YdeO , H-NS DOES NOT HAVE OBVIOUS EFFECT ON THE TRANSCRIPTION FROM INDIVIDUAL PROMOTERS UNDER ANAEROBIC CONDITION Our investigation above showed that ArcA and MnmE are the major activators responsible for the expression of gadE-mdtEF under anaerobic condition , whereas other regulators tested did not show obvious regulatory effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>ArcA ANTAGONIZES THE H-NS MEDIATED REPRESSION OF gadE-mdtEF DURING THE TRANSITION FROM AEROBIC TO ANAEROBIC GROWTH Our results above showed that the expression of gadE-mdtEF is largely repressed by the DNA silencer H-NS under aerobic conditions in M9 glucose medium , whereas under anaerobic condition in the same medium its expression is activated primarily in an ArcA dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Interestingly , under anaerobic condition , while arcA almost completely abolished transcription of gadE-mdtEF , further deletion of hns in arcA mutant restored the transcription of the full-length PgadE-lacZ , and P1 - and P3-lacZ fusions , suggesting an antagonistic effect of ArcA on the H-NS mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>As shown in Figure 6B , transcription of PgadE-lacZ in arcA hns double deletion did display signiﬁcantly higher activity than that in arcA strain , suggesting that the anaerobically up-regulated gadE-mdtEF expression was through the antagonization of H-NS mediated repression by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Thus , in the present study , we ﬁrst examined whether previously identiﬁed regulators , such as EvgA , YdeO , MnmE , GadE , GadX , and H-NS , also participate in the anaerobic activation of mdtEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Interestingly , except for mnmE , deletion of the regulators previously shown to participate in the regulation of mdtEF under aerobic condition , such as EvgA , YdeO , GadE , and GadX , caused no or only a moderate decrease ( 17 -- 40 % ) of the transcription of PgadE-lacZ , and deletion of H-NS caused a moderate increase of PgadE-lacZ activity , suggesting that these regulators are not responsible for the activation of mdtEF under anaerobic condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Consistent with our observations at PhlyE , it has been suggested that SlyA binding to such sites remodels the local nucleoprotein structure , counteracting the effect of negative regulators such as H-NS , and clearing the way for classical transcriptional-activators such as PhoP , or as in the case of hlyE , FNR and CRP ( Ellison and Miller , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Discussion We began this work with the observation that a lacZ transcriptional fusion to dsrB , a small conserved gene lying between rcsA and dsrA , was negatively regulated by H-NS and positively regulated by DsrA , consistent with previous observations on other H-NS-silenced genes ( Sledjeski and Gottesman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>Transcriptional monocopy lacZ fusion studies showed that both operons are coexpressed during anaerobic-growth in the presence of carnitine , respond to common environmental stimuli ( like glucose and nitrate ) , and are modulated positively by the same general regulators , CRP and FNR , and negatively by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Like cai and ﬁx operons , the caiF gene appeared to be positively controlled by CRP and FNR and negatively controlled by H-NS and the NarL ( plus nitrate ) regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Second , pyruvate H-NS helps maintain a rapid degradation of arginine oxidase ( PoxB ) , which may be preferentially used to and glucose produce acetate when bacteria approach the stationary phase , is unlikely to play a key role under Pi-starvation Because H-NS is required for a full induction of the conditions because the expression of poxB is strictly LexA regulon in Pi-starved cells ( Dri & Moreau , 1993 ; controlled by RpoS ( Chang et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Although a number of studies ( Tanaka et aL , 1991b ; Hinton et aL , 1992 ; Yasuzawa et aL , 1992 ; Dersch et aL , 1993 ; Ueguchi et aL , 1993 ) have previously looked at H-NS levels as a function of growth-phase by Western blotting , the results are inconclusive as to whether the protein accumulates in stationary phase or not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>The intracellular level of H-NS has also been reported to increase 3-fold at the stationary phase ( Spassky et al. , 1984 ; Dersch et al. , 1993 ; Ueguchi and Mizuno , 1993 ) , although this finding has not been observed in other reports ( Hinton et al. , 1992 ; Yasuzawa et al. , 1992 ; Free and Dorman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Some studies have reported an increase in H-NS levels as cells enter stationary phase ( Spassky et aL , 1984 ; Dersch et aL , 1993 ; Ueguchi et aL , 1993 ) , although this has not been detected in other reports ( Tanaka et aL , 1991b ; Hinton et aL , 1992 ; Yasuzawa etaL , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Surprisingly , H-NS affects both RpoS mRNA translation and RpoS turnover in logarithmic growth ; it was previously reported that there is a 10-fold increase in the RpoS synthesis rate , as well as a 10-fold increase in RpoS stability , in hns mutants ( 1 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li>
<li>After pair-wise promoter-binding assays , the increase in csgD transcription in the stationary phase was suggested to be due , at least in part , to the increase in IHF level cancelling the silencing effect of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>On the other hand , a marked increase in csgD mRNA level was observed for mutants lacking CpxR and H-NS , particularly in the stationary phase ( Fig. 1b , lanes 3 and 7 ) , indicating that CpxR and H-NS are the negative regulators of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Thus , the silencing of csgD by H-NS should be reduced in stationary phase because of the increase in IHF level ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The present work revealed that the P2 promoter of nhaA is a stationary phase-induced promoter which is activated by sS and neither NhaR nor Na1 or H-NS is involved in this induction ; in the stationary phase , transcription from P2 increased dramatically , and this increase was eliminated in an rpoS mutant strain ( Fig. 6A ) but not in a DnhaR strain ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Our data support this report and demonstrate that the increased content of H-NS in stationary phase cultures is largely caused by the genetic regulation of hns transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Its levels are lowest when rRNA transcription is most active , and H-NS concentrations increase during entry into stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15568992" target="_blank">15568992</a>]<br></li>
<li>It is possible that the observed increase in ppGpp concentration during exponential-growth indicates a compensating mechanism in cells lacking functional H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>One might ask whether H-NS-dependent repression at the end of logarithmic growth under wild-type conditions involves a corresponding increase in the concentration of H-NS relative to exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>In the exponential phase , only a few differences were observed between the wild-type strains and the mutants defective in FIS , H-NS , IHF , OmpR , cAMP , and ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>In contrast , in cells expressing functional H-NS molecules a clear decrease in P1 expression is already apparent during late log phase , whereas at stationary phase the P1-directed expression is almost totally repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>In the case of LRP and H-NS , derepression is discernible during the complete growth curve , while in the fis mutant expression of 6S RNA during exponential phase is not enhanced but rather slightly reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>In summary , Hsp31 expression is highly sensitive to the growth temperature and H-NS plays an important role in modulating Hsp31 concentration in exponential phase cells , silencing hchA transcription more efﬁciently as the temperature decreases while allowing RNA polymerase access to its EsD - and EsS-dependent promoters in times of thermal stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>In the present paper , we report that hns transcription is under autoregulation during the logarithmic growth-phase , and that H-NS functions as a repressor for its own transcription Summary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>In most of the studies , H-NS was found to be active in stationary phase ; however , some reports describe phase-speciﬁc effects during exponential-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>In contrast , the level of H-NS and IHF increases as the cells enter the stationary phase of growth ( Spassky etai , 1984 ; Bushman efai , 1985 ; Ditto etai , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>In this respect , it is worth mentioning that the observed auto-regulatory expression of H-NS is dependent on the growth-phase , i.e. its autoregulatory expression appears to be derepressed at stationary phase , and consequently the level of H-NS in cells appears to increase at this stage of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>In any event , these results support the view that hns transcription depends on the growth-phase to some extend and , as a consequence , the absolute amount of H-NS in cells appears to increase at stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>In summary , we conclude from these results that a measurable , although not dramatic , increase in the concentration of H-NS occurs when cells leave the exponential phase or when growth is retarded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Before cell growth ceases , at late log phase , the pre-sence of H-NS apparently enables a more efﬁcient rRNA synthesis shut-down under conditions in which the FIS concentration is still sufﬁcient to activate rRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Our studies demonstrate that LRP binds in addition to H-NS and FIS to the upstream regulatory sequence region ( UAS ) of rRNA P1 promoters , and thus potentially contributes to the efﬁcient shutdown of rRNA synthesis , which is known to occur at the onset of stationary phase or under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>There is also in vitro evidence that the activities of H-NS ( 43 , 54 ) and of Rho ( 39 , 57 ) are sensitive to the potassium salt concentration , which is known to vary within the cell with changes in osmolarity of the growth medium ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>Osmotic shock as well as changes from exponential to stationary growth are known specifically to decrease the K * ion concentration in the cell , and also to activate H-NS function ( Schultz and Solomon , 1962 ; Higgins etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>We note , for example , that there are 39 Fis-regulated genes or operons listed in Table 2 that have been shown to be regulated by CRP , H-NS or GadE , of which 25 exhibit Fis-regulatory effects during the late-exponential-growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>A common denominator for the genes known to be regulated by H-NS ( e.g. regulation of genes at stationary growth-phase , starvation , changes in high osmolarity , low temperature shock , and altered DNA topology ) seems to be their involvement in different kinds of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Most of the genes belonging to the H-NS regulon are linked to the stress response or to changes in the environmental conditions such as temperature , osmolarity , growth-phase , pH , and availability of oxygen and nutrients ( 10 , 19 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The templates were first incubated with or without H-NS for 20 min : lanes 1 , 2 , 9 , 10 , 17 , and 18 ; no H-NS ; lanes 3 , 4 , 11 , 12 , 19 , and 20 , 150 nM H-NS ; lanes 5 , 6 , 13 , 14 , 21 , and 22 , 250 nM H-NS ; lanes 7 , 8 , 15 , 16 , 23 , and 24 , 350 nM H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The amounts of transcription factors added were : CpxR and IHF , 0 , 0.5 , 1.0 , 1.5 , 2.0 , 2.5 , 3.0 , 3.5 , 4.0 , 4.5 , 5.0 , 5.5 and 6.0 pmol for lanes 1 -- 13 ; H-NS , 0 , 2.5 , 5.0 , 7.5 , 10 , 12.5 , 15 , 17.5 , 20 , 22.5 , 25 , 27.5 and 30 pmol for lanes 1 -- 13 ; OmpR and RstA , 0 , 5 , 10 , 15 , 20 , 25 , 30 , 35 , 40 , 45 , 50 , 55 and 60 pmol for lanes 1 -- 13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The amounts of proteins used were ( from lane 2 to 5 ) : 1.25 , 2.5 , 5 and 10 pmol for IHF ; 2.5 , 5 , 10 and 20 pmol for CpxR ; 5 , 10 , 20 and 40 pmol for OmpR and H-NS in the absence ( H-NS and IHF ) or presence ( CpxR , OmpR and RstA ) of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>All five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , formed binary DNA complexes in vitro in the absence of other factors ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In this report , we describe the regulatory modes of five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , and their interplays in binding in vitro to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In addition , three factors , IHF , CpxR and H-NS , bind to the same upstream region between 2188 and 2159 , designated hot-spot I. Interplay between transcription factors in binding to these two hot-spots should influence the level of csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In principle , YdaM-generated c-di-GMP may modulate the action of any of these regulators ( i.e. OmpR , IHF , MlrA , CpxR , H-NS ) at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>The apparent affinity for the csgD promoter was found to be high for IHF and CpxR , followed by OmpR , H-NS and RstA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The site of BasR binding on the csgD promoter overlaps with those of IHF , CpxR and H-NS ( Fig. 7 ) , implying that the silencing effect of H-NS might be antagonized by the combination of three regulators , BasR , IHF and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>Briefly , E. coli whole-cell lysates were prepared by sonication after lysozyme treatment , and treated with antibodies , which were prepared in rabbits against purified transcription factors , CsgD , RstA , OmpR , CpxR , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>At the times indicated , whole-cell lysates were prepared and subjected to quantitative Western blot analysis using antibodies against IHF , OmpR , H-NS , CpxR and RstA ( upper panels ) , and anti-RpoA ( bottom panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In growing cells , the intracellular concentrations of two nucleoid proteins , IHF and H-NS , are similar and higher than those of the specific regulators , RstA , OmpR and CpxR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Based on the results described herein , we propose a novel regulation model of cooperation between two positive factors ( OmpR and IHF ; and RstA and IHF ) and also between two negative factors ( CpxR and H-NS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In addition to the competition between positive and negative factors , we found unexpected cooperation between two negative factors , CpxR and H-NS , and between two pairs of positive factors , OmpR and IHF , and RstA and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Previously , we classiﬁed three factors ( OmpR , RstA and IHF ) as activators and two factors ( CpxR and H-NS ) as repressors , and found novel modes of their interplay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Indeed , the opposite effect was observed , whereby the increased levels of Rho that were detected in the rho and nusG mutants were partially reversed upon H-NS 64 or YdgT expression ( Fig. 6A , compare lanes 5 and 8 to lanes 2 and 3 ) ; since it is known that Rho is autoregulated ( 31 ) , these results provide yet another line of evidence that H-NS 64 and YdgT increase the efﬁciency of Rho-dependent termination in the mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Since both CpxR and H-NS bind to wide and overlapping regions , 2190 and 215 for CpxR and 2201 and +28 for H-NS , we used the long probe CD6 ( 2335 to +67 ; see Fig. 6 ) for the gel shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In the presence of MlrA overexpression , an increase of csgD mRNA was observed in the mutants lacking Crl ( Fig. 4 , lanes 5 and 6 ) , H-NS ( Fig. 4 , lanes 7 and 8 ) , RstA ( Fig. 4 , lanes 11 and 12 ) , CpxR ( Fig. 4 , lanes 13 and 14 ) and RcsB ( Fig. 4 , lanes 15 and 16 ) , indicating that MlrA functions independent of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Synthesis of the control RNA-I transcript was not inhibited , and was in fact stimulated two - to threefold , by the addition of H-NS ( compare lanes 1 and 3 in Fig. 4B or lanes 1 and 4 in Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Overall , the footprinting data suggested that a Phly - complex ( Fig. 4A , lanes 3 -- 5 ) , but mobility was increased E : SlyA complex is formed when SlyA binds at speciﬁc in the presence of 1.25 and 2.5 mM H-NS ( Fig. 4A , lanes sites in the upstream and downstream regions of the 6 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>] When the amount of H-NS added was increased in the presence of a saturating amount of IHF , H-NS was able to-bind by replacing IHF sites ( Fig. 7a , b , lanes 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>On the other hand , the addition of increasing amounts of IHF in the presence of a high concentration of H-NS to saturate the CD1 and CD3 probes did not change the gel pattern ( Fig. 7a , b , lanes 11 -- 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Among these real or presumed signals there are three direct repeats of uncertain functional signiﬁcance and a fourth one [ 6 ] corresponding to a phoB box ( CTGCTAT TGCACAA CTGAATT ) [ 7 ] which likely accounts for the 2 -- 3-fold H-NS induction observed upon Pi deprivation and when phosphonate is the sole source of phosphate [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>( i ) It directly acts as transcriptional-repressor of gadA and gadX promoter ; ( ii ) it exhibits an additional negative control over gadX expression possibly by reducing its transcription from the gadA promoter ; ( iii ) unlike gadA , gadB does not display any specific H-NS binding , and its promoter activity is not affected in vitro by H-NS ; however , H-NS might indirectly inhibit the GadX-mediated expression of gadBC by repressing gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Through combined in vitro and in vivo assays , we have identiﬁed a gene , ﬁmB , in which H-NS acts as a promoter-speciﬁc transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>This is consistent with H-NS functioning as a repressor for its own promoter , both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Ueguchi and Mizuno ( 1993 ) have recently demonstrated that H-NS functions as a transcriptional-repressor by direct binding to the promoter region of the proVWX operon in E coli It has also been shown that , in the absence of H-NS , promo-ters may become independent of positively acting regulators such as CRP , suggesting that these regulators may act as antirepressors for H-NS ( Forsman et ai , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Recent reports estabiished that in ooriditions of low ostTiolarity H-NS inhibits proL/transcription directly through binding to a site downstream from the promoter ( Ueguchi and Mizuno , 1993 ; Lucht et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li>
<li>In this respect , we recently demonstrated , in vivo and in vitro , that H-NS functions as a repressor of its own transcription by directly binding to the promoter ( Uegu-chi et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Under non-cold-shock conditions , H-NS acts as a direct transcriptional-repressor of the osmotically induced proU operon ( 42 ) as well as of numerous other genes ( reviewed in reference 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>This suggests H-NS functions directly as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>H-NS , which is activated by CspA , likely acts as a cold-shock transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>In contrast , Ueguchi and Mizuno ( 40 ) recently demonstrated that H-NS can selectively inhibit transcription of proU by E. coliRNA polymerase in vitro , which suggests that H-NS functions directly as a transcriptional-repressor affecting early stepsof transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>This view is supportedby the recent report of Ueguchi and Mizuno ( 40 ) , who demonstrate that H-NS can act as a specific , direct repressor ofproU transcrip tion in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>This delay and the lower increase of the protein level compared to that of its mRNA are not typical of the cold-shock proteins ( e.g. , CspA , DeaD and H-NS ) which are synthesized with a much shorter delay due to the efficient utilization of cold-shock mRNAs by the cold-shock-modiﬁed translational apparatus ; the accumulation of CAT activity is typical instead of non-cold-shock proteins ( e.g. , - galactosidase ) whose mRNAs are inefficiently translated during cold-shock [ 18 , 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>CspA and H-NS induction patterns are consistent with the hypothesis that proteins induced early act as regulatory elements while proteins induced late perform functions directly required by the cold-shocked cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>Identiﬁcation of the effectors involved in GadX activation and further investigation of the interactions between regulatory components of the gad system ( H-NS , RpoS , CRP , and GadX ) may provide further insight into the molecular circuitry underlying the glutamic acid-based acid stress response of E. coli A. Tramonti and P. Visca contributed equally to this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Here , we used our series of strains with the CRP box located at different positions in the csiD promoter region in order to test whether this modulation by H-NS and Lrp is coupled to activation by CRP or proceeds by CRP-independent mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Although both H-NS and Lrp stimulate csiD activation by cAMP-CRP , they obviously do so by different mechan-isms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Finally , the constructs with S the relocated CRP box were used to test whether modulation by the histone-like proteins H-NS and Lrp is connected to activation by cAMP-CRP or proceeds independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The histone-like proteins H-NS and Lrp positively modulate csiD expression by differentially affecting activation by cAMP-CRP Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In summary , both histone-like proteins , H-NS and Lrp , modulate csiD expression by inﬂuencing activation by cAMP-CRP , but they obviously do so by different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Thus , the ` topological ' function of H-NS in activation by cAMP-CRP suggested by the data in Fig. 7A seems to be speciﬁc for Es .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Deletion of the sequences corresponding to the second predicted S-L ( D2 , Fig. 1 ) remained partially active for rpoS-lacZ activation but lost anti-H-NS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770508" target="_blank">9770508</a>]<br></li>
<li>GeÂrard , F. , Dri , A.M. and Moreau , P.L. ( 1999 ) Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>Tolerating some redundancy RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Our results linking H-NS to Lrp lend further support to the notion that both regulators are involved in responses to stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , ArcA and FNR regulate respiration in response to oxygen availability ( the variable in these experiments ) , CRP and Lrp sense nutritional state via cAMP and L-leucine concentrations , and Fis , H-NS and IHF modify DNA topology , which is itself dependent on energy status [ 7,25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>By these criteria , ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp were designated as global TFs in E. coli K-12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , it is concluded that ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp are truly global regulators , at least as far as the response to oxygen availability is concerned , in the sense that their outputs create the transcriptional framework upon which local regulators operate in response to specific unstable environments Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>We examined the effects of some global regulators ( factor inversion stimulation [ FIS ] , H-NS , OmpR , cAMP , ppGpp , and IHF ) on the expression of sra by using global regulator-defec-tive mutants lysogenized with lpF13 ( Psra-lacZ ) or carrying pFF6 ( Psra-lacZ ) and by measuring b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Thus , many global regulators , including cAMP , H-NS , IHF , and ppGpp , inﬂuence the expression of sigma S , which in turn may regulate the transcription of sra in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_392">
Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></label>
<input id="act_similarity_392", type="checkbox">
<div class="ACT_similar">
<li>Regulatory pattern III ( anaerobically repressed , but not affected by ArcA ) and pattern IV ( anaerobically activated , but not affected by ArcA ) are most easily explained as genes controlled by the FNR protein or by an as yet unidentified global regulator such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li></div>
<li>This activation is brought about in an RpoS-depen-dent manner in concert with a combination of one or more global regulators such as Lrp , H-NS , and IHF ( 11 , 14 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>]<br></li>
<li>This differentiates it from tlie many stereotypically regulated geiies controlled by IHF , Lrp , H-NS aiid RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108285" target="_blank">9108285</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative-stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>H-NS , IHF and Fis sites do not exhibit spatial clustering Interaction set Genic regions Non-genic regions Clustering Interactions R Clustering Interactions Clustering Interactions R RS RS RS RS S CLS S CLS NAP a Fis Exp SHX Low ** NC Low ** Low ** ND ND ND ND H-NS Exp SHX Low ** Low * Low ** Low ** ND ND ND ND Exp SHX IHF Low ** Low ** ND ND ND ND Low * NC NAP ( 500 bp ) b Exp NC NC Low ** Low ** SHX Low * NC Low *** Low *** Fis , H-NS , IHF coding Fis , H-NS , IHF non-coding Exp Low ** Low * Low ** Low ** SHX Low ** NC Low ** Low ** a The summed interaction or clustering strength for each condition was compared with 1000 random data sets with RS or CLS between each element ( see ` Materials and Methods ' section ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>The results of this study support the proposals made by Wu and colleagues , but show that the repressing factors are more complex than initially supposed : FNR-dependent activation is repressed by three nucleoid-associated factors , Fis , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The major nucleoid proteins , HU , IHF , H-NS and Fis , in exponential-growth-phase E. coli under aerobic conditions are all known to have functional dichotomy ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response to oxidative-stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth-phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Fis and H-NS seem to occupy different planes of th determine the possible functional effects of H-NS we have performed transcriptional analyses in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>These results clearly underline the function of H-NS as a specific and potent transcriptional inhibitor for PI-directed rRNA synthesis , indicating a possible bioiogical significance , namely to antagonize Fis activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Bands identiWed by tandem mass spectrometry : a , partially resolved bands of ( above ) and ( below ) subunits of RNA polymerase ; b and c , - and - subunits of ATP synthase ; d , subunit of RNA polymerase ; e , Dps ; f , H-NS , StpA ; g , egg white lysozyme ; h , IHF - , IHF - , Fis ; i , HU-1 , HU-2 ; j , L4 ; k , S3 , S7 , S9 ; l , L13 , Fis , IHF - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>For routine assays , the antibodies were diluted 200-fold for CbpA , Fis , H-NS , IHF , RpoD , SeqA and L7/L12 , and 500 fold for CbpB , Dps , Hfg , HU , IciA , StpA , RpoA , RpoS , Rsd and RMF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>Nonetheless , our results are supported by earlier analyses of the somewhat differing effects of gene disruptions in the global regulators H-NS [ 121-124 ] , IHF [ 125,126 ] , and Fis [ 127,128 ] in the closely-related genera Escherichia and Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Many of these , notably Fis , IHF and H-NS , have been found to exert specific effects at particular promoters ( reviewed by Friedman , 1988 ; Finkel and Johnson , 1992 ; Williams and Rimsky , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>This upstream region activates when Fis dominates and inhibits when H-NS dominates , as discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686546" target="_blank">15686546</a>]<br></li>
<li>Thus , H-NS seems specifically to antagonize Fis-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Thus , repressive effects of Fis and IHF appear to be additive to the repressive effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Our results clearly show that H-NS represses pnir and that the action of H-NS is independent of the action of Fis , IHF and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>NarL and NarP can relieve both IHF - and Fis-mediated repression , but are unable to counteract H-NS - mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , we conclude that IHF and Fis can repress pnir in the absence of H-NS ( Table 2 ) , and that repression by H-NS can not be reversed by the action of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Figure 3 and Table 3 show the results when plasmids from WT , HU , IHF and H-NS strains were electrophor-esed in the presence of 1 , 2 , 3 , 5 , 7 or 9 mg/ml chloroquine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>These data show that levels of unrestrained negative supercoiling in the WT and HU backgrounds are similar , while negative supercoiling in H-NS and IHF strains was increased relative to WT ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>It is possible that IHF and an additional protein , such as FIS , H-NS or HU , along with CysB may be necessary for full activation of the positively regulated cysJIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>Goshima , N. , Kano , Y. , Tanaka , H. , Kohno , K. , Iwaki , T. , and Imamoto , F. ( 1994 ) IHF suppresses the inhibitory effect of H-NS on HU function in the hin inversion system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>While HU is widely distributed among the eubacteria , H-NS , FIS and IHF are restricted to the proteobacteria ( Oberto and Rouviere-Yaniv , 2001 ; Tendeng and Bertin , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>In the course of our work on the metabolic controls involving the one carbon ( C l ) pool ( ie methylene tetra-hydrofolate and its derivatives : thymine , purines , serine and glycine , methionine , pantothenic acid and N-formylmethionyl tRNA for initiation of translation ) we discovered that H-NS was involved in the regulation of their synthesis , as will be described below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Results Determination of new H-NS targets involved in the regulation of glutamate-dependent acid resistance As H-NS strongly represses the glutamate-dependent acid stress response , there is a very low level of survival after acid stress in the FB8 wild-type context [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>In contrast , several genes involved in adaptation to low pH are strongly induced in a H-NS deﬁcient strain , resulting in an increased resistance to acidic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>In parallel , H-NS decreases acid resistance by repressing genes involved in general acid stress response ( e.g. dps encoding the DNA protection during starvation protein ) or in three amino-acid decarboxylase systems , dependent on glutamate , lysine and arginine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Expression of the leuO gene is repressed by H-NS , and it is upregulated in stationary phase and under amino acid starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>First , the leuO gene itself is repressed by H-NS and is positively autoregulated ( 5 , 6 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>However , while the relevance of LeuO and YjjQ for pathogenicity indicates that their genes are expressed under certain in vivo conditions in the host environment , both the leuO gene and the yjjQ-bglJ operon are repressed by H-NS under laboratory growth conditions ( 6 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>However , expression of leuO and bglJ is repressed by H-NS , at least under laboratory growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>YjjQ was provided plasmidically , as expression of the native yjjQ-bglJ operon is repressed by H-NS during exponential-growth under laboratory conditions ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Transcription of the yjjQ-bglJ operon is repressed by the global repressor H-NS ( heat-stable nucleoid structuring protein ) , and it is activated by the pleiotropic transcriptional-regulator LeuO ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Expression of the yjjQ gene , which is encoded in the yjjQ-bglJ operon , is repressed by the global repressor H-NS and activated by the pleiotropic transcriptional-regulator LeuO , as shown for the laboratory E. coli K-12 strain ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Furthermore , the yjjP gene , encoding a membrane protein of unknown function and located upstream in divergent orientation to the yjjQ-bglJ operon , is likewise repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The yjjP gene located upstream of the yjjQ-bglJ operon and in a divergent orientation is likewise repressed by H-NS but is not activated by LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Taken together , the data conﬁrm the regulation of the yjjQ-bglJ operon by H-NS and LeuO , and they demonstrate the repression of yjjP by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Here we show that yjjQ and bglJ are expressed as an operon , which is repressed by H-NS and activated by LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>RESULTS H-NS represses and LeuO activates the yjjQ-bglJ operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Although the mechanism of regulation in this case is unknown , and present information indicates that neither Fis nor H-NS is responsible , the results described above demonstrate that cells can compensate for mutation even in major components of the regulatory system when there is no competition for survival , and therefore rapid responses to changes in environmental conditions are not crucial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15568992" target="_blank">15568992</a>]<br></li>
<li>Therefore , multiple systems ( EnvZ/OmpR , Cpx , Rcs , and H-NS ) appear to be involved in sensing environmental osmolarity , leading to sophisticated regulation of the curli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>EnvZ/OmpR , H-NS , and the Cpx and Rcs systems all appear to affect bioﬁlm formation by regulating the production of curli ﬁmbriae in response to osmotic conditions ( 16 , 39 , 54 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>Since it was demonstrated that H-NS is an inhibitory component of the sS stationary-phase induction network and also inhibits the expression of sS itself ( 3 ) and since we now know that sS also takes part in nhaA regulation , we examined the possible involvement of H-NS in rpoS-dependent stationary phase induction of nhaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Such an observation is consistent with the function of H-NS at osmotic shock or the regulation of genes expressed under stationary phase ( Higgins etai , 1988 ; Hengge-Aronis , 1993 ) , Thus H-NS may link rRNA expression to situations of rapid osmotic changes or reduced growth at stationary phase or even be effective under conditions of cold-shock ( La Teana etai , 1991 ) , Obviously , acellular process as central as the regulation of rRNA synthesis must be controlled by many different parameters and sensors to be as effective and efficient as observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Osmotic upshift Cold shock Discussion Participation of H-NS in growth rate regulation One of the hallmarks in the regulation of rRNA synthesis , apart from stringent control , is growth rate regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Although the regulation of rRNA synthesis is clearly related to the growth-phase-dependent expression of the activator FIS , a similar relationship to the concentration of H-NS is less clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Repression of these stress response genes is mediated by H-NS either indirectly via its negative regulation of RpoS or directly by binding to the control regions of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>]<br></li>
<li>These results indicate that H-NS is involved in the negative transcriptional regulation of the gad genes and raise the possibility that H-NS would directly silence the gadA and gadB promoters and/or indirectly repress gad expression through the control of trans-acting factors , probably the stationary-phase factor ss .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Bertin , P. , Hommais , F. , Krin , E. , Soutourina , O. , Tendeng , C. , Derzelle , S. , and Danchin , A. ( 2001 ) H-NS and H-NS-like proteins in Gram-negative bacteria and their multiple role in the regulation of bacterial metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Because H-NS is known to-bind preferentially to curved DNA and to have relaxed sequence specificity ( reviewed by Williams and Rimsky , 1997 ) , we investigated whether H-NS could bind to pnir and affect its regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>It is interesting to note that both integration host factor and H-NS , the product of the hns gene , seems to be involved in interactions with curves or bends in DNA , and the role and regulation of this aspect of DNA structure are areas of broad current interest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383109" target="_blank">8383109</a>]<br></li>
<li>One of the most extensively studied properties of H-NS is its ability to-bind DNA , since this interaction is often coupled to gene regulation ( 9 , 38 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The interaction of the Hha -- H-NS complex with DNA differs from those of the individual proteins59 and it has been proposed that the heteromeric complex is the active form that is required for haemolysin gene regulation in response to thermal and osmotic signals60 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>This suggests that the virus needs to regulate H-NS and that improper regulation of H-NS in the absence of YhiM may eVect both viral replication and survival in acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22327802" target="_blank">22327802</a>]<br></li>
<li>This co-regulation of H-NS and RpoS may be important to the cell under the conditions in which DsrA is made .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251852" target="_blank">11251852</a>]<br></li>
<li>Although we were unable to detect any effect of RprA on H-NS ( data not shown ) , there may well be other targets for RprA and , again , the co-regulation of RpoS with these targets may be important for the cell under particular environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251852" target="_blank">11251852</a>]<br></li>
<li>If all of the effects of the singlecopy dsrA gene were by antagonizing H-NS 's negative regulation of RpoS , then we would expect an hns - mutant to be epistatic to a dsrA - mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>If in fact H-NS regulation of this putative gene is negative , this supports the idea that rapid degradation of RpoS may be the default situation , and stabilization under speciﬁc conditions underlies regulated proteolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li>
<li>As CRP is not known to regulate hns expression or H-NS production , this suggests that CRP might exert its effect indirectly through another regulator that may act as a potential competitor of H-NS in regulation of expression of these ﬁmbriae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Hha -- H-NS mediated regulation of htrA is most likely a gradual process and this fact can explain the progressive increase of HtrA expression when the med-ium osmolarity increases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Here we show that the regulation of hlyE expression by H-NS is more complex than was previously thought and that the overproduction of SlyA enhances hlyE expression by antagonizing the negative effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Binding of H-NS to these sequences might explain the negative regulation of hns expression by its own gene product ( 20,21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>What is the physiological function of H-NS for the expression and regulation of proU ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Role of RpoS and Crl in bgl operon regulation The bgl operon is regulated by various pleiotropic regulators including H-NS and CRP , as essential factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
<li>In particular , we studied its influence on the transcriptional regulation of the pap operon and other operons known to be H-NS regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>We show that H-NS is involved in the transcriptional regulation of the kbl/tdh operon , which is also Lrp regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>This same functional role can be argued for Lrp regulation ( see introduction ) , perhaps implying that the H-NS protein is involved in fine tuning this regulatory network in some way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Hence the lack of ability to produce functional H-NS protein precludes down-regulation , indicating that this is a consequence of negative autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Promoters sensitive to negative regulation by H-NS were sensitive to positive regulation by DsrA , while a promoter that was insensitive to H-NS was also unaffected by DsrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>However , some H-NS control ( 2.5-fold ) was still evident in the shortest fusion , lFER6 , suggesting that sequences located further downstream are also required for cydAB regulation by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>It is interesting that the opposing activities of FIS and H-NS are not restricted to the regulation of rrn P1 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Transcriptional regulation or anti-H-NS activity is independent of the first S-L ( S-L1 ) , while translational regulation is absolutely dependent on S-L1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770508" target="_blank">9770508</a>]<br></li>
<li>H-NS does not participate in the temperature-dependent regulation of nrdA , as shown by evaluation of - galactosidase in strain ETS119 at high and low temperatures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Ono , S. , Goldberg , M.D. , Olsson , T. , Esposito , D. , Hinton , J.C. , and Ladbury , J.E. ( 2005 ) H-NS is a part of a thermally controlled mechanism for bacterial gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Poore , C.A. , and Mobley , H.L. ( 2003 ) Differential regulation of the Proteus mirabilis urease gene cluster by UreR and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Nucleotide regulators -- ppGpp and NTPs Changing Fis and H-NS levels by no means accounts for the totality of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686546" target="_blank">15686546</a>]<br></li>
<li>Thus , H-NS and NhaR interact directly or indirectly in the regulation of nhaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Similarly , it may be that H-NS disrupts the regulation of cirA , fes , srlE , and hisJ through indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>An additional aspect of regulation by H-NS is the interdependent regulation of H-NS and its paralogs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>H-NS regulon genes and their regulation by environmental parameters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Hence , regulation by Lrp intersects with several channels of regulation , and H-NS participates , at least in part , in this intersection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>However , it remains to be clarified whether regulation by Lrp always involves H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Whereas FIS is known as transcription factor responsible for growth rate-dependent activation of stable RNA expression , allowing very fast cellular growth rates and rapid change to nutritional upshifts , H-NS can be regarded as a factor that slows down rRNA transcription under limiting conditions and thus enables a better adaptation of the cell to situations when growth is very slow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>As LeuO is an H-NS antagonist with pleiotropic function , activation of leuO expression should occur in response to speciﬁc environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>As it is known for FIS-dependent activation , the inhibitory function of H-NS is speciﬁc for P1 , the ﬁrst of the tandem rRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>H-NS is a global transcriptional-regulator that affects various unlinked genes which include ompF and ompC .53 H-NS represses several genes in environmental conditions that pertain to outside of the intestine but allows for expression in the intes-tinal tract where environmental signals can activate genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>This regulation is different from that of many H-NS controlled genes , where H-NS serves as a transcriptional silencer in response to a variety of environmental cues and deletion of H-NS leads to high-level expression under th normally repressive cue ( reviewed in references 1 and 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>In addition , they are regulated in response to very different environmental signals : H-NS influences the expression of genes regulated by osmolarity , temperature , pH , oxygen availability , and genes under stringent control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>This suggests that overexpression of either GadX or GadE alone , in an FB8 genetic background , is not sufﬁcient to induce expression of genes involved in the bacterial response to low pH when cells are grown at neutral pH. Overexpression of gadX signiﬁcantly induced the expression of 48 genes of the H-NS regulon ( Table 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>However , even if we could not rule out some false positives , our results showed that several genes could be identiﬁed as signiﬁcantly induced at low pH , demonstrating that GadE and GadX act as transcriptional-activators of genes of the H-NS regulon under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_486">
Fig. 8A , B. Binding of H-NS to DNA fragments encompassing the putative promoters whose expression is repressed by H-NS , and mobility analysis of the DNA fragments by two-dimensional PAGE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></label>
<input id="act_similarity_486", type="checkbox">
<div class="ACT_similar">
<li>x20 xlO0 Fig. 8A , B. Binding of H-NS to DNA fragments encompassing the putative promoters whose expression is repressed by H-NS , and mobility analysis of the DNA fragments by two-dimensional PAGE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li></div>
<li>H-NS regulates the expression of proVWX , bgl , appY and ® mB of Korea Advanced Institute of Science and Technology , Yusong-Ku , Taejon 305-710 , South Korea , and dDepartment of Life Science , Division of Molecular and Life Science , Pohang University of Science and Technology , Pohang 790-784 , South Korea ² Dr Lee should also be considered as corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077448" target="_blank">12077448</a>]<br></li>
<li>It is known that s synthesis is derepressed about sixfold in strains lacking H-NS ( 5 , 59 ) , and this could explain the moderate increase in proU P1-lac expression from plasmids pHYD275 ( E. coli P1.a ) and pHYD374 ( S. enterica P1 att ) as well as the increase in osmY-lac expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>and MCI 061 , can be activated for csgBA transcription by mutating the hns gene encoding H-NS ( Olsen et ai , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Both promoters can be activated in anCT ^ - negativebackground when the hns gene encoding H-NS is also inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Transcription of the hns gene encoding the H-NS protein appears to be negatively regulated by H-NS itself both in vitro and in vivo , We have examined the role of this mode of regulation in wild-type cells in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>These results indicated that the lrp gene is not only feedback regulated by Lrp but is also controlled by H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>However , the expression of lrp is not regulated solely by Lrp but is also under the control of H-NS protein , as revealed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>For example , the protein that is encoded by gene 5.5 of bacteriophage T7 inactivates H-NS , resulting in derepression of both host and viral promoters99 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>During the 1980s the protein was characterised extensively by the groups of Buc and Gualerzi , although it was only when the gene encoding H-NS was identified and characterised [ 7-9l , that a rather more general interest in the role of this protein was stimulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The activating insertion elements essentially eliminated the H-NS-mediated silencing , and stimulated ade gene expression 2 -- 3-fold independently of the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>Considering that H-NS modulates gene expression in response to environmental factors , we next analyzed whether this protein modulates the expression of the nrdAB operon in response to speciﬁc environmental stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>In the present paper , we show that hns , the structural gene for the nucleoid protein H-NS ( 8 ) , belongs to the cold-shock regulon , its expression being increased 3 - to 4-fold after a temperature shift to 100C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1961761" target="_blank">1961761</a>]<br></li>
<li>Furthermore , should the in vivo function of H-NS be that of a histone-like protein , it would be interesting to study th regulation of its gene expression which could possibly be geared to the initiation of D N A replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2841565" target="_blank">2841565</a>]<br></li>
<li>In addition to confirming the primary structure of the protein presented here , this should now allow us to apply gene-replacement techniques to construct mutants of H-NS and , ultimately , to clarify its in vivo function and the regulation of its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>Lrp , and H-NS iii coiitrolling f i , ~ expression and the in \ ~ olveineiit of these tliree proteins and RpoS in cross-control of inembers of each of tlie regulons , ule were interested in exploriiig tlie possibility that tlie pliase-variable Ji17iA gene niiglit be iiifluenced by RpoS aild , hence , by growlli pliase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108285" target="_blank">9108285</a>]<br></li>
<li>While H-NS usually acts as a negative regulator of gene expression , hlyE is not the only example of a CRP-regulated gene whose expression is positively regulated by H-NS ; for example , expression of both the malT and csiD genes in E. coli is stimulated by H-NS ( 8 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Oshima , T. , Ito , K. , Kabayama , H. , and Nakamura , Y. ( 1995 ) Regulation of lrp gene expression by H-NS and Lrp proteins in Escherichia coli : dominant negative mutations in lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Direct and indirect effects of H-NS and Fis on global gene expression control in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>When H-NS is inactive , the expression of hdfR gene is increased , resulting in an overproduction of HdfR , which will reduce the transcription of ﬂhDC by binding to its upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>tems in which H-NS represses transcription , appY gene expression in the wild-type strain was not affected by changes in osmolarity or by changes in temperature in the 30 to 428C range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>It is also involved in transcription termination ,20 and can bind and stabilise mRNA .21 As a transcriptional-regulator , H-NS silences gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>This direct activation of ftnA transcription by Fe2 + -- Fur through H-NS antisilencing represents a new mechanism for iron-induced gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Examples of direct activation of gene expression by H-NS are rare .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>By contrast , we found that the expression of the dps gene , directly regulated by H-NS and known to protect cells against multiple stresses [ 29 ] , is essential to lysine - and arginine-dependent responses to acid stress , while its role is less important during the glutamate-dependent response ( Table 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>With respect to temperature , it should be noted that the temperature-dependent modulation of gene expression by H-NS usually occurs in genes or operons that are downregulated at low temperature , which is not the case for the nrdAB operon , which is upregulated at low temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>References Afflerbach , H. , Schroder , O. , and Wagner , R. ( 1999 ) Atlung , T. , and Ingmer , H. ( 1997 ) H-NS : a modulator of environmentally regulated gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_514">
[ 20 ] Atlung T. , Ingmer H. , H-NS : a modulator of environmentally regulated gene expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></label>
<input id="act_similarity_514", type="checkbox">
<div class="ACT_similar">
<li>[ 2 ] Atlung T. , Ingmer H. , H-NS : a modulator of environmentally regulated gene expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li></div>
<li>Furthermore , the expression of the hdfR gene was shown to be negatively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Further , the expression of the orj30 gene , named smtA , was positively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>Expression of the hdfR gene is negatively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>To the best of our knowledge , the smtA gene is the first gene identified to be positively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>Thus , H-NS does not seem to be restricted to a purely structural role but appears to influence gene expression , probably through its effect on DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>In the first scenario , the interaction of H-NS • : , ith DNA would be influenced by specific environmental signals , such that it regulates gene expression in response to those signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>In gene regulation , H-NS acts as a transcriptional-repressor in most cases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In Yersinia , YmoA [ 4 ] and a list of other proteins all appear to share the ability to-bind H-NS and regulate gene expression at the transcriptional level [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>In the present paper , we demonstrate that in vivo H-NS exerts a negative control on the transcriptional expression of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>H-NS binds the chiA promoter region with high affinity The global transcriptional-repressor H-NS has a negative effect on expression of a number of genes , in particular those regulated by different environmental factors ( e.g. temperature , pH or osmolarity ; Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>H-NS is a direct transcriptional-repressor of proU ( 16 , 23 , 40 ) , and expression is phenotypically detectable in strains with proU-lacZ fusions on lactose MacConkey indicator medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The fact that H-NS is a global transcriptional-repressor , and that its loss leads to extremely high expression of many of its targets , might lead to an amplified selection against its loss or inactivation ; this could select for backup by StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>The hns-66 allele present in ZK819-66 results in reduced activity of H-NS , which is strikingly similar to the nature of rpoS819 allele .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>The expression of [ 3-galac-tosidase activity expressed by CU237/pMK1 was affected by tryptophan added to the medium , i.e. the level of [ ~ - galactosidase activity was markedly reduced when production of H-NS was appropriately induced by depletion of tryptophan in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>H-NS and Fur speciﬁcally interact with the ftnA promoter The data presented above indicate that H-NS acts to repress ftnA expression and also mediates the Fe2 + -- Fur induction of ftnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The rcsA promoter itself is H-NS repressed ; its expression is positively autoregulated by RcsA-RcsB and requires induction of the Rcs signaling pathway ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>The expression of hdfR was negatively controlled by H-NS , which explains the negative effect on ﬂagellation of an H-NS mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>The results show , furthermore , that H-NS represses the expression of this yjjQ-bglJ operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The expression directed by the divergent yjjQ and yjjP promoters is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The expression ratios are completely reversed in the pDIA510 strains , which could have an excess of H-NS protein , and the biggest difference is seen at high osmolarity in the absence of leucine where we believe that Lrp activator is rendered limiting by a high concentration of active H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The effect of H-NS on cydA ± lacZ in vivo expression was subsequently determined under aerobic and anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Results The hns-118 mutation affects both TDH enzyme activity and gcv-lacZ expression Under certain growth conditions , the H-NS protein modulates the activation of iivlH expression by Lrp [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The approximately ﬁvefold repression in ftnA -- lacZ expression caused by Dip in the wild type was reduced to approximately twofold by the fur and/or hns mutations suggesting that , at least in part , iron induction of ftnA is dependent on both Fur and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In the osmotically Inducible genes treA and osmC , the absence of H-NS also results in an increased expression at low osmolarity , but with a marked reduction in the induction ratios ( Repoila and Gutieriez , 1991 ; Gutierrez and Dovedjian , 1991 ) , The present results show a similar behaviour for osmE ( Table 2 ) , indicating that repression by H-NS at low osmolarity is probably a major determitiant in the mechanism of osmEregulatioti , as is also the case for treA and osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li>
<li>When the intracellular concentration of SlyA decreases , or the concentration of H-NS increases , H-NS is able to displace both SlyA and RNAP silencing hlyE expression ( Fig. 7 , A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>These data suggested that hlyE expression is activated by FNR and CRP , repressed by H-NS , and that H-NS-mediated transcriptional silencing is relieved by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>By competing with H-NS for the region of PhlyE downstream of the FNR/CRP site , SlyA may prevent the formation of the negatively acting H-NS -- PhlyE complexes by blocking a primary interaction between H-NS and PhlyE , allowing FNR and CRP to continue to activate hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , we concluded that CRP , FNR , and H-NS all contribute towards the regulation of hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , CRP and FNR positively control cai expression and H-NS exerts a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Thus , we suggest that the formation of different H-NS -- PhlyE complexes offers the opportunity for H-NS , in combination with FNR and CRP , to control hlyE expression both positively and negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>This suggests that during anaerobic-growth in liquid cultures in the absence of glucose , FNR is the major regulator of hlyE expression , and that CRP-mediated hlyE expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The ﬁndings that SlyA does not enhance hlyE expression in hns cultures grown in liquid medium under anaerobic conditions and that SlyA appears unable to activate hlyE expression in the absence of FNR and CRP and the determination of the location of the region of PhlyE occupied by SlyA ( Fig. 6 ) suggest that SlyA activates hlyE expression by antagonizing the negative action of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>However , in the presence of high salt ( LB supplemented with 0.3 M NaCl ) there is already a high proU expression ( Gowrishankar and Manna 1996 ) , and so there is no measurable difference between the wt and the himA ( − ) strain ( data not shown ) on the expression of the proU promoter suggesting that IHF is acting only when expression is down regulated by H-NS , in this case at low salt concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>We conclude that , at the csiD promoter , H-NS generates high stringency with respect to the position of the activator cAMP-CRP and thereby contributes to maximal expression in the presence of cAMP-CRP ( when bound to its wild-type position at 268.5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Taken together , these results suggest that both H-NS and LRP may require DNA sequences located several hundred base pairs upstream of the primary stpA promoter in order to regulate its expression correctly and that Lrp may in part act to overcome the negative effect of H-NS at the stpA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Taken together these results conﬁrm that an interplay exists between NhaR and H-NS in the regulation of nhaA expression mediated by the P1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>These authors also examined the expression of stpA in response to H-NS , Lrp , and temperature and the requirement for promoter-distal DNA for regulation , and they obtained results in good agreement with our own .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>When the promoter fragment was extended to include sequences from positions 87 to 26 and 147 to 26 , the expression was repressed by H-NS , suggesting that H-NS binds in between 147 and 47 ( Fig. 5A , PQ 87 to 26 and PQ 147 to 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>In contrast , the expression of the yjjQ promoter-lacZ fusion encompassing position 347 to 26 was activated by LeuO and was repressed more efﬁciently by H-NS than the shorter constructs were ( Fig. 5A , PQ 347 to 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Since ﬁmB expression is increased ( reference 40 and this study ) and ﬁmA promoter inversion rates are rapid in the absence of H-NS ( 28 ) , it is possible that the mechanism by which H-NS regulates ﬁmA promoter inversion is through altering FimB levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>H-NS represses nhaA expression , and by analogy with other H-NS-dependent promoters , H-NS probably binds to a site around the promoter region to maintain repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>( a ) Due to the binding of H-NS to the cas promoter expression of the Cas proteins is supposed to be cryptic under optimal growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Therefore , it was of interest to know whether the positive effect of H-NS on ﬂhDC expression was a direct consequence of its binding to the promoter region of the master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>With respect to the mechanism underlying Hha-mediated induction of H-NS expression , gel retardation assays did not provide any evidence that Hha modiﬁes H-NS afﬁnity for its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Considering that the hns expression is under autoregulatory control , we test in vitro whether Hha could modify the afﬁnity of H-NS for its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Due to the tightness of the interaction between H-NS and the ﬁmB promoter sequences , we surmised that this interaction was the cause of decreased ﬁmB expression in wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>As an example , H-NS acts upstream of its own promoter to autoregulate its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>As the intracellular levels of H-NS increase , H-NS II is occupied and hlyE expression is downregulated by promoter occlusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Considering that NrdAB activity is not required when cells grow an-aerobically , we hypothesized that NrdAB expression could be repressed in anoxically growing cells and that H-NS might play a relevant role silencing nrdAB expression under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>For expression of HA-tagged H-NS , IPTG was added to a ﬁnal concentration of 1 mM , and the growth of cultures was stopped on ice after 2 h of induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>H-NS autoregulates its own expression [ 13-15 ] , and the intraceilular concentration of H-NS does not appear to vary significantly in response to growth conditions , except that it is induced when cells are cold-shocked ( see below ; [ 16 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>On the basis of this association with H-NS and the fact that Hsp70 proteins are typically induced under stress conditions , we examined hscA expression following cold-shock , which induces H-NS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>These include ( i ) their maximal induction at low pH under anaerobic conditions in the presence of excess substrate , ( ii ) the fact that they are major cellular proteins , ( iii ) the use of pyridoxal 5 ' - phosphate as a cofactor , ( iv ) their proposed role in pH homeostasis , and ( v ) the involvement of H-NS , which reduces expression of both systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Because CnuK9E downregulated dicA expression better in the presence of H-NS ( Table 1 ) and CnuK9E can form a protein complex as efficiently as its wild-type Cnu with H-NS ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>Furthermore , FtnA levels in the hns mutant remained high in the presence of Dip , indicating that a functional H-NS protein is required for the iron-dependent regulation of ftnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Taken together , these data indicate that expression of the E. coli slyA gene is maximal during exponential-growth , is temperature ¬ Autoregulation and temperature regulation of slyA expression Shown is expression of a slyA - lacZ fusion ( pDC6 ) in the presence or absence of SlyA or H-NS at 37 and 20 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Aerobic expression from the cydAB P1 , P2 , P3 and P4 promoters was decreased in a DarcA strain , both in the presence ( Fig. 6 , lanes 1 and 3 ) and in the absence of H-NS ( Fig. 6 , lanes 2 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In the absence of added sugars , CRP indeed appeared to be an activator of sfm both in the presence and in the absence of H-NS , as shown by decreased sfm expression in crp mutants ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>However , in the presence of glucose , H-NS appeared to have little effect on hlyE expression ( Fig. 1d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In the absence of glucose , H-NS appeared to have a positive effect on hlyE expression ( Fig. 1c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In the previous report ( 36 ) , H-NS acted as a strong repressor of hlyE expression , whereas here H-NS appears to have a positive effect on hlyE expression in the absence of glucose but an attenuated or negative effect on hlyE expression in the presence of glucose ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>These studies have also demonstrated that the effect of H-NS on rcsA expression is independent of RcsB : a mutation in hns leads to a 10-fold increase in rcsA90 : : lacZ expression in the presence or absence of RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>The hns mutation caused a two - to fourfold increase in ftnA -- lacZ expression in the absence of Dip , and a ﬁve - to eightfold increase in its presence , thus conﬁrming that H-NS represses ftnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The combination of the lrp mutation with hns-203 reduced the level of expression somewhat , particularly in the absence of leucine , but the effect was not as severe as that of the lrp mutation in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The results obtained ( data not shown ) conﬁrmed the ﬁndings obtained with the malT-lacZ fusion , and we conclude that full MalT expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>These data either means that both pcsgBA and pcsgDEFG , in the presence of H-NS , require ~ s or that only pcsgDEFG requires the expression of this sigma factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8817489" target="_blank">8817489</a>]<br></li>
<li>We also showed that nmpC expression is repressed by H-NS ( threefold increased expression in the presence of an hns mutation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Interestingly , these changes in gene expression were observed regardless of the presence of H-NS ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>They proposed the presence of an efﬁcient locking mechanism involving H-NS that prevents inadvertent expression during normal growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>In both cases , the increase in the level of expression caused by hns inactivation was eliminated by the appY mutation , suggesting that the effect of H-NS on the hya and cyx promoters might be indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Results in Table 2 show that , with wild-type pnir , disruption of hns resulted in a sixfold increase in expression : thus , H-NS does repress FNR-dependent expression from pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>When H-NS is absent , as in hns-ll8 , expression levels ( compared to hns + parental controls ) of the ilvIH operon increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>We conclude that the low levels of MalT detected in these strains were due to the absence of H-NS , which appeared to play a positive role in malT gene expression at the level of translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>If this strong H-NS repressive effect is abolished in the absence of a regulator negatively controlled by H-NS , we can conclude that this deleted regulator has positive effect on target gene expression and may be an intermediary actor in H-NS-dependent control for this target , as previously shown [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>garP expression demonstrated a purely negative effect of H-NS on transcription , as an hns651 mutation led to increased expression of garP at both temperatures ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>The increased transcription from these promoters observed in the absence of H-NS correlates with the increased expression of the gspA-lacZ and gspD-lacZ operon fusions ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>If H-NS normally represses ﬁmB transcription , then in the absence of H-NS , ﬁmB expression would increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>These studies demonstrated that Ler directly activates transcription of the LEE genes by competing for binding with H-NS , which represses their expression in the absence of Ler .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12011002" target="_blank">12011002</a>]<br></li>
<li>It is clear that the ability to-bind directly to or near the core region could enable H-NS to repress ﬁmB expression ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Alternatively , H-NS could indirectly repress expression of gad genes by negatively acting on RpoS stability ( 3 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>As shown in Figure 6A , the induction of StpA overproduction caused a significant decrease in the expression of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>At present , it is not known if expression of the mlc gene is affected by H-NS and/or StpA , but evidently Mlc is subject to autoregulation and represses its own expression ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Furthermore , because H-NS and StpA coordinately regulate some genes while independently regulating others , small perturbations in H-NS levels can cause global changes in gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>H-NS is also reported to inhibit gadX expression but appears capable of inhibiting gadA and gadBC expression independently of gadX control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Plasmid-expressed DsrA increased expression of genes normally repressed by H-NS , including rcsA , proU , and papA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
<li>Indeed , we found that ydaM expression is under positive control of H-NS , whereas yciR is repressed by H-NS ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Thus , the physiological role of H-NS appears to be to prevent high expression from the regulated promoters under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Taken together , these results strongly suggest that H-NS negatively regulates cydAB expression under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Effect of ArcA and H-NS on induction of appY expression under oxygen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>In aerobically growing E. coli cells , H-NS represses nrdAB expression under all conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS inhibits csgBA expression mediated by E 70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>Examination of these data revealed that H-NS binds the yciGFEkatN locus and represses its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>This occurred because H-NS represses acrEF , mdtEF , and emrKY expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>H-NS represses expression both in aerobically and in anaerobically growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>One can envision a mechanism in which the silencing of rcsA by H-NS is removed , leading to increased expression of rcsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>Therefore , H-NS appeared to be necessary for maintaining the low basal level of rcsA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
<li>Here we demonstrate that H-NS represses sS expression , that the absence of H-NS results in a failure to osmotically induce sS , and that most proteins overexpressed in an hns mutant are in fact sS dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>In comparison , expression of the chromosomally encoded leuO mRNA was elevated approximately 31-fold in the Dhns stpA mutant , conﬁrming derepression of leuO in absence of H-NS and StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>In the absence of H-NS , the expression of `` epistatically controlled genes '' is repressed by StpA , whereas that of `` unilaterally controlled genes '' is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>In light of our observations , it seemed reasonable to further investigate the impact of Lon , RcsB , and H-NS on rcsA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>In this study , we isolated an insertion enabling cells to enhance ﬂhDC expression even in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>This increase of dicB and dicC expression was considerately less in the absence of H-NS : in MG1655ghns / pCnuK9E , dicB and dicC expression was increased 78 - and 24-fold , respectively , whereas that of dicA was decreased by half ( 0.55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>This suggests that H-NS has the potential to regulate cydAB expression negatively in both aerobiosis and anaerobiosis , but ArcA antagonizes its action in the absence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Two other genes known to be under the control of o ® for expression , boiA and i < atE , remained transcriptionaiiy siient in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>So , in the absence of H-NS , the hdfR gene product is more abundant , leading to decreased expression of the flagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>We concluded that ﬁmB expression was signiﬁcantly induced in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Since SlyA , when overexpressed , activates the expression of ClyA , we wanted to investigate the requirement of SlyA for derepression of clyA in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Changes in growth and environmental conditions are known to correlate with alterations in DNA topology ( 1 , 7 , 27 ) , and it is thus possible that under certain conditions the DNA of the stpA promoter is in a favorable conformation for expression even in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Under conditions of rapid growth and de novo DNA synthesis , the corresponding reduction in the H-NS : DNA ratio would Department of Microbiology , Moyne Institute of Preventive Medicine , University of Dublin , Trinity College , Dublin 2 , Ireland .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>MG1655hns was shown to grow normally in liquid culture and could form single colonies on agar plates even in the presence of CnuK9E , suggesting that CnuK9E requires H-NS to induce filamentous growth at 37uC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>However , we show here that even in a strain containing functional H-NS , stpA is transiently induced during growth of a batch culture in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>We asked how the synthesis of rRNA is affected in the presence or absence of H-NS when the osmolarity of the growth medium is increased or the temperature of the growing culture is rapidly decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>In contrast , when the osmolarity of the growth medium is increased by the addition of 200 mM NaCl the effect on rRNA synthesis rates depends strongly on the presence of H-NS : in wildtype cells synthesis rates are reduced by 50 % , whereas in hns cells a twofold increase in transcriptional activity Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Cells generate new copies even in the absence of horizontal gene transfer , presumably by duplication and genetic drift ( for example , the HU α - and β-subunits , and H-NS and StpA ) and one assumes that those novel arrangements that lead to optimum fitness are selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>However , the marked increase in cellular FimB levels in the absence of H-NS was not the primary cause of the mutant rapid inversion phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>H-NS and YdgT perturbations do not increase endogenous levels of Rho and NusG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>H-NS levels decrease , whereas RpoS ( ss ) levels increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10954740" target="_blank">10954740</a>]<br></li>
<li>Protein bands corresponding to His-YdgT , H-NS and dimeric His-YdgT or H-NS were also present in preparations obtained in the absence of the cross-linking agent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>StpA represses the promote twofold in the absence of H-NS ( Fig. 1C , compare bars d and f to bars c and e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Repression by StpA in the absence of H-NS was twofold ( Fig. 1D , compare bars d and f to bars c and e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Interestingly , an increase in OmpC and a decrease in OmpF level in the absence of OsmZ ( H-NS ) was observed ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>First , even in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Conformational plasticity of Hha was observed even in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>The probability of thymidine formation increased significantly in the absence of IHF in comparison to the wt , while in the absence of H-NS the opposite effect was observed ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Primer extension assays showed that P1 induction by hydrogen peroxide occurs in the absence of Lrp , IHF and H-NS ( Fig. 4A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>Transcription from P2 , and to a lesser extent from P1 , was inhibited by the nucleoid protein H-NS but only in the absence of potassium glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Impaired stability of the StpA protein in the absence of Hha and YdgT It has been shown that the StpA protein is quickly degraded in vivo in the absence of its paralogue H-NS by the action of the Lon protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>M182Dhns was used as a control , showing the increase in StpA protein in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The StpA protein was more unstable in the absence of either H-NS and Hha or H-NS and YdgT than in the absence of only H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>In similar experiments , here we demonstrate that the absence of Hha/YdgT either in the presence or in the absence of H-NS affects StpA stability , presumabily by the interaction between Hha/StpA and YdgT/StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Overproduction of the StpA protein affects expression from at least four hns regulated operons ( the papB , proU , bgl and hns operons ) , in both the presence and absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Taken together , the results demonstrate that StpA can silence four known H-NS-regulated operons , and that it can function as a silencer in both the presence and absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>As stated above , it is known that certain hns mutations are suppressors of rho and nusG ( 27 ) and , since YdgT is an interacting partner of H-NS ( 30 , 33 , 42 ) , we tested whether it is the presence of YdgT in the plasmid derivatives that was responsible for the suppression phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>In addition , the presence of a CRP effect in an hns mutant background also suggests than CRP acts independently of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>That H-NS acts by inﬂuencing activation by cAMP-CRP was also conﬁrmed by the observation that , in cya mutant backgrounds , the presence or absence of H-NS no longer made a difference ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Although it was demonstrated ( Donato & Kawula , 1998 ) that there is an interaction between H-NS and FliG affecting the motor rotational speed , the presence of H-NS on the ¯ agellar basal body is still in question , since FliG could exist freely from the basal body .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>The nhaAp1 promoter is repressed by H-NS , whereas NhaR relieves H-NS-mediated silencing under alkaline pH and in the presence of Na ( 12 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22037401" target="_blank">22037401</a>]<br></li>
<li>The increase up to 4-fold in proU-P2 promoter expression in the presence of IHF occurs at low salt concentrations where the promoter is under H-NS repression showing that IHF was partially alleviating H-NS repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>The repression of hdeABp by H-NS is not due to direct occlusion of E 70 , since formation of an open promoter complex ( RPO ) was not affected in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>The widespread influence that is exerted by H-NS on transcription and the presence of genes that encode its paralogues on mobile genetic elements make H-NS and its close relatives ideal subjects for precisely this type of investigation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The size of H-NS66 , which is already larger than the wild-type H-NS , increased further in the presence of pssrADD ( Fig. 5C , lanes 3 , 4 , and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>The answer to this question can be inferred from Fig. 8 , lanes 8 -- 11 where increasing amounts of the mutant H-NS protein were analysed in the presence of constant amounts of the wild-type H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>The extent of IHF-mediated Pe activation in vivo indeed differs depending on the absence or the presence of H-NS ( P. van Ulsen and N. Goosen , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The analysis was repeated with H-NS-speciﬁc antibodies , but again , no H-NS could be identiﬁed in the complexes formed in the presence of both proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>690 J. K. Lithgow , F. Haider , I. S. Roberts and J. Green The effect of adding SlyA or H-NS to PhlyE in the presence of RNAP was then tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The presence or absence of Rs and H-NS is indicated by + or - symbols below the lane numbers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>The presence or absence of H-NS has almost no effect on this reduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>The lacUV5p activity was unaffected by the presence of H-NS ( Fig. 1B , panel 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>The location of the bands that are only protected in the presence of both YncC and H-NS is marked with an open arrow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The presence of H-NS did not decrease the intensities of the KMnO4 hypersensitive bands created by E 70 or E 38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Although we did not measure the amount of H-NS produced by this plasmid , it is likely that H-NS levels are higher in the presence of the plasmid than in the wild-type haploid situation , autoregulatory mechanisms notwithstanding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>In the presence of increasing concentrations of H-NS , the hdeAB transcript generated by E 70 decreased , while the rna1 transcript from the origin of replication remained constant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>In the left panel transcripts from reactions performed in the presence of increasing concentrations H-NS are presented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>the formation of the typical H-NS complex was inhibited in the presence of Fis and occurred only at much higher H-NS concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>DNase I protection assays of the PhlyE : H-NS complex in the presence of increasing concentrations of SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Initiation complex formation was performed in the presence of increasing concentrations of H-NS or LRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>His6-H-NS was puriﬁed under native conditions while His6-sS , the majority of which is produced in inclusion bodies , was puriﬁed under denaturing conditions in the presence of 6 M guanidine hydrochloride .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>One can ask therefore if the shut-down of ribosomal RNA synthesis under shock conditions might be dependent on the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Transcription of the pre-CRISPR RNA in vivo is not completely cryptic but strongly down-regulated in presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Subsequently , we examined whether or not the presence of H-NS prevented RNP , especially E 70 , from forming an RPO at hdeABp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>These data indicate that E 70 forms an RPO even in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Open or black triangles ( right side ) indicate bands with decreased or enhanced intensity , respectively , in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Interestingly , the presence of the anti-cas transcript depends on cells , which do not express functional H-NS ( PD32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>H-NS is able to repress pnir , even in the presence of nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Most notably , the hdeAB transcript generated by E 38 was not affected by the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>On the other hand , E 70 can transcribe some S-dependent genes in the presence of additional factors or in the absence of speciﬁc repressors such as H-NS ( 6 , 22 , 87 ; reviewed in reference 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>Increasing the concentration of SlyA to 0.55 M in the presence of 0.5 M H-NS resulted in a further reduction in intensity of the H-NS V1 complex with the simultaneous appearance of a new less mobile H-NS V1 SlyA complex that was not detectable at this concentration of SlyA in the absence of H-NS ( Fig. 4 , cf. lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>In the presence of H-NS , 0.72 M SlyA yielded a single low mobility H-NS V1 SlyA complex distinct from that formed in the absence of H-NS ( Fig. 4 , cf. lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Thus , a shorter promoter construct which is not repressed by H-NS is not signiﬁcantly repressed by the absence of Lrp or the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>We used in vivo laser footprinting to compare the local conformation of proU promoters in the presence or absence of IHF or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>An autoradiogram of a polyacrylamide sequencing gel on which DNase I digestion of the dps400 fragment was compared in the absence of H-NS and in the presence of 0.5 mM , 2 mM , 4 mM , 6 mM or 8 mM H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Wild-type H-NS is unaltered in the presence or absence of pssrADD ( Fig. 5C , lanes 1 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Wildtype H-NS was detected in the presence or absence of SsrADD ( lanes 1 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Figure 4 gives a typical example of such a competition experiment , where the concentration of H-NS was varied from 70 to 700 nM in the presence or absence of 130nMFis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Figure 4 shows the KMnO4 reactive bases generated by RNP binding in the absence and presence of H-NS ( 228 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>In mutants lacking H-NS , the secreton genes cloned on a moderate-copy-number plasmid are expressed and promote ef ® cient secretion of the endo-genous , co-regulated endochitinase ChiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>In cells expressing functional H-NS , the expected typical pattern for growth rate-regulated genes is apparent when rRNA synthesis rates increase almost as the square of the increasing growth rates ( Gausing , 1977 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>H-NS does not directly a ¡ ect switching frequency In order to test whether H-NS plays a direct role in inversion of the phase switch , for example by blocking access of proteins required for switching such as IHF or Lrp , in vitro switching experiments were carried out in the absence or presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>In the present study we employ chromosomally located cents m-lacZ cassettes , switch-orientation assays , and in vitro recombination to examine the e ¡ ect of H-NS in regulation of the cents m genes , notably with respect to temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>Control of H-NS expression and activity In addition to regulation by H-NS , StpA and Fis , the hns promoter is stimulated by cold-shock through the CspA protein37 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>In this study , we investigated the involvement of the H-NS and StpA proteins in the regulation of the malKlamBmalM and malEFG operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>In the present study , two-dimensional electrophoresis was used to help identify proteins whose expression varied when the genes coding for H-NS and StpA were inactivated individually or collectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>In this study , we found that LeuO is able to lead the bgl operon to its derepressed state ( i.e. , Bgl1 ) , even in the presence of H-NS , provided that LeuO was overexpressed under certain conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422614" target="_blank">9422614</a>]<br></li>
<li>In the presence of 1 Na , however , NhaR functions as a positive regulator of nhaA ( 39 ) and overcomes the repressive effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>In the presence of H-NS , the homologous protein StpA of E. coli is expressed at a low level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>In contrast we found that H-NS increases MDR in E. coli in the presence of MarA and an intact AcrAB-TolC pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In addition , StpA autoregulates stpA , with threefold repression , in an hns null mutant , i.e. , independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>In addition to RcsA , our experiments establish that H-NS indirectly controlled the RcsB regulator via repression of RcsD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>In contrast to PhlyE , umuD was repressed whenever H-NS was present , showing that SlyA speciﬁcally antagonizes H-NS repression of PhlyE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In an hns1 strain in the absence of Na1 , nhaA was repressed whether or not nhaR was present ( Fig. 2 and 3 ) ( 39 ) , suggesting that the repression by H-NS is NhaR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>In that case , translation of the rpoS mRNA appeared to increase in the absence of H-NS , suggesting the hypothesis that H-NS may act by repressing translational induction of rpoS ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>In our previous investigations of DsrA activity , it appeared that all of its action was to overcome the effect of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>In addition , H-NS does not seem to bring about its repressing effect via a change in the concentration of the global effector ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>In fact , since H-NS is the product of a monocistronic transcript ( 9 ) , it unlikely that the stimulation of its is very production during cold-shock is merely coincidental .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1961761" target="_blank">1961761</a>]<br></li>
<li>In these studies , H-NS was demonstrated to be the product of both the drdX and osmZ loci , which are thus allelic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>In contrast , pap is transcriptionally inactive at 23 ° C , and this thermoregulation requires H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>In contrast to its action on nhaA , NhaR activation of osmCp1 is independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></li>
<li>In contrast , H-NS negative regulation operated only in the absence of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>In vivo however , the magnitude of YncC activation was reduced in the absence of H-NS ( Figs. 2D , E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>CU237 cells carrying these plasmids were cultivated in M9 minimal medium supplemented with a high concentration of tryptophan ( 50 gg/ml ) , and then transferred to medium without-tryptophan to induce the production of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>It has been reported that H-NS affects the degree of supercoiling of reporter plasmids ( 20 , 37 ) , and it has been shown that shift to anaerobic conditions leads to an increase in negative supercoiling ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Therefore , we suggest that when environmental conditions dictate a low intra cellular concentration of H-NS , the H-NS I region of PhlyE is occupied to modulate the activity of the upstream activator ( FNR or CRP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Both FNR and CRP make positive contributions , whereas H-NS acts negatively under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>This positive effect of H-NS was not observed under anaerobic conditions , presumably because FNR and not CRP occupies PhlyE under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>We demonstrate that H-NS can decrease the activity of Lrp in stationary phase and under conditions of high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>It is possible that under certain growth conditions the activity of Rho or of H-NS is reduced or antagonized even in a rho hns strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>Thus , it appears that H-NS modulates PhlyE activity both positively and negatively in response to the prevailing growth and/or environmental conditions , such as growth in liquid and on solid media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Secondary structures in H-NS and related proteins analysed by the GORIV method [ 23 ] Predicted α-helices , - sheets and random coils are coloured in red , blue and green , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Forns , N. , Baños , R.C. , Balsalobre , C. , Juárez , A. , and Madrid , C. ( 2005a ) Temperature-dependent conjugative transfer of R27 : role of chromosome - and plasmid-encoded Hha and H-NS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>García , J. , Cordeiro , T. , Nieto , J.M. , Pons , I. , Juárez , A. , and Pons , M. ( 2005 ) Interactions between the bacterial nucle-oid associated proteins Hha and H-NS involves a conformational change of Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>As the H-NS concentration was increased , the proportion of RNAP : PhlyE complex was reduced and the proportion of H-NS : PhlyE was increased ( Fig. 4D , lanes 2 -- 7 ) , suggesting that H-NS replaces RNAP at PhlyE in a concentration-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>This may be due to increased stability of H-NS : PhlyE complex at lower temperatures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>H-NS most strongly repressed P4 and had a moderate effect on P1 , P3 and P2 , whereas the intensity of the P5 band was only marginally affected by H-NS ( , twofold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>A simple REFERENCES interpretation of our results is thercforc that H-NS may also stimulate , directly or indirectly , the synthesis of Altuvia , S. , Almirdn , M. , Huisman , G. , Kolter , R. & St0rz.C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>There is also no clear evidence that H-NS binds a ligand that might act allosterically to alter its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>It was not possible to completely dissociate the interaction between YdgT and H-NS proteins by using increasing concentrations of KCl in the elution buffer ( up to 2 M ) ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>B. Effect of increasing concentrations of KCl in the interaction between His-YdgT and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Hence , a speciﬁc role for H-NS repressing nrdAB in response to anaerobiosis can not be drawn from the data obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS regulates bacterial response to osmolarity and other stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>H-NS regulates many stress response genes by acting as a major component of chromatin and by responding to changes in osmolarity ( 5 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>Fragments bound preferentially by H-NS are indicated by thick black horizontal bars , and fragments showing no speciﬁc shift are indicated by thin black horizontal bars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Western blot analysis showing the effect of H-NS on FtnA and Fur levels under iron-rich and - poor conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Furthermore , in an H-NS mutant , no further effect of increasing DsrA was detected ( Sledjeski and Gottesman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>act to overcome the negative effect of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Induction of these genes by H-NS deletion might be the secondary effect of the EvgA induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>When the H-NS concentration is ^ i o • * M , tetramers also appear in increasing amounts among the reaction products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>Each of the wedges represents increasing amounts of added H-NS ( 0 , 10 , 20 , and 50 pmol in panel A ; 0 , 10 , and 50 pmol in panel B ) from left to right .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Lanes 1 and 5 , 0 pM H-NS ; lanes 2 and 6 , 97.5 pM H-NS ; lanes 3 and 7 , 195 pM H-NS ; lanes 4 and 8 , 390 pM H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Lanes 1 and 9 show samples without any H-NS added and lanes 2 to 8 show samples with increasing amounts ( 4.8 , 5.6 , 6.4 , 7.2 , 8.0 , 8.8 , and 9.6 mM ) of H-NS added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Again , three retarded bands of decreasing mobility were evident as the H-NS concentration was increased up to 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>This section of the model is reversible , because when the relative concentration of H-NS increases H-NS is able to displace SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>However , upon increasing the concentration of H-NS , a unique footprint was formed at the SlyA-I site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Furthermore , P2 is activated by sS and is neither induced by Na1 nor dependent on NhaR or affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>When H-NS is present , it represses NhaA and a single copy of NhaR is needed for Na1 induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>When H-NS is removed from the cell ( or micF is otherwise induced ) , the level of antisense RNA increases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Binding of H-NS to this site is probably the ultimate cause of transcriptional auto-repres-sion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>It is important to identify proteins that are repressed or induced in response to overproduction of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li>
<li>Speciﬁc competition experiments further support the result that H-NS binds to fragment IIIc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>It was concluded that H-NS speciﬁcally antagonizes the activating function of FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Preliminary data from our lab indicate that it is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>It is repressed by both LRP and H-NS ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>FIG 7 Class Ia RNR is repressed anaerobically by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>It is activated by s70 and affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Reconstituted EsS transcribes hchA in a reaction that is inhibited by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>H-NS , Rho , and xenogeneic silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Likewise , H-NS represses theE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Interrelationship between NhaR and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Binding of puriﬁed H-NS to clyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Although the arginine and lysine-dependent acid resistance pathways are regulated by H-NS [ 1 ] , it is not known whether AdiY and CadC , the specific regulators of these pathways respectively , are controlled by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Complementation of the Dhns strain reversed the reduction in phage sensitivity , demonstrating that CRISPR-based defence is negatively regulated by H-NS ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>This suggests that the targets of the corresponding proteins are also indirectly regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>Moreover , such an effect also occurs on hdeAB , which is regulated by MarA and H-NS both independently and synergistically ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>It is intriguing that unlike its homolog H-NS , StpA is strongly regulated by environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Thus , the level of H-NS in the cells may need to be regulated properly under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>However , it can not be excluded that lrp is tightly regulated by H-NS under some restricted conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>H-NS is itself regulated in interesting and complex ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>While possible modification mechanisms are feasible for modulating H-NS activity , its function may be regulated in a different way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Studies to show if the activity of H-NS is regulated independently from the cellular concentration by modiﬁcation or cofactors have been negative to date .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Such a speciﬁc contribution by H-NS to MarA induction of acrA , but not the other MarA-regulated genes tested , is of interest because H-NS is known to have no effect on regulation of acrA per se ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>To assess the functionality of the H-NS binding motif , we asked whether its occurrence correlated with genes known to be regulated by H-NS in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Thus , RpoS and H-NS that control a large repertoire of environmentally regulated genes are key players in the physiological adaptation to the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Many genes under the negative control of H-NS are regulated in response to changes in the environment ( 1 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>Half of these are not bound by H-NS , in contrast to the up-regulated genes where nearly three-fourths are bound by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Indeed , in this organism , numerous H-NS target genes are regulated by environmental stresses , which are frequently encountered by enterobacteria in the transition to the existence inside their host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>It is possible that other H-NS repressed genes are also regulated by this mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Of 30 genes induced more than eightfold by EvgA overexpression , 18 , including ydePO and yhiE , are also induced by H-NS deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>Alternatively , under anaerobic conditions H-NS can still bind to the gadE promoter , however , its binding is antagonized by the activated ArcA , resulting in the recruitment of RNAP and transcript initiation of the operon under this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>However , if H-NS was not inactivated , rather , it can still bind to the promoter but its binding was antagonized by ArcA , then its binding to the gadE promoter should be resumed when ArcA is absent and an increase of PgadE-lacZ activity should be observed in hns arcA than arcA single deletion strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>It should be noted that our data do not allow us to discern whether H-NS remains bound to the cydAB promoter region in microaerobiosis or whether it still has a repressing role under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>However , H-NS can still repress the cydAB P4 promoter ( albeit weakly ) in an arcA Dfnr background 1 under anaerobic conditions ( Fig. 6 ) , and preliminary results suggest that ArcA and H-NS can simultaneously bind the ArcA-II region in vitro ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>More interestingly , the increased transcription of lacZ fusions in arcA hns double deletion strain than that in arcA single deletion strain was also observed in the case of individual P3 and P1 promoters , suggesting that ArcA antagonization of H-NS takes place at multiple sites in the PgadE promoter , consistent with the notion that both H-NS and ArcA can oligomerize and bind to a broad region in its regulated gene promoters ( Green and Paget , 2004 ; Dorman , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Although Fis and H-NS affect rrn P1 promoter activity both in vitro and in vivo , promoters lacking Fis binding sites ( or cells lacking the ﬁs gene ) are regulated normally ( 11 , 50 , 89 ) , and under most conditions ( delete `` tested '' ) , cells lacking hns still regulate rRNA transcription ( B.J.P. & R.L.G. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15568992" target="_blank">15568992</a>]<br></li>
<li>Thus , genetic and biochemical results ( 40,51,69 ) and thHe-NS binding studies presented here suggest that the strong but finely tuned induction of proU expression under highosmolarity growth conditions is achieved by a combination of osmotically stimulated transcription , mediated by sequences in thevicinity of the proU promoter , and osmotically relieved repression by H-NS , mediatedby the high affinity H-NS binding region at the 5 ' end of proV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Thus , in the proposed regulatory model the relative concentrations of SlyA and H-NS determine the type of nucleoprotein complex formed at PhlyE and when SlyA is dominant it inhibits binding of H-NS and , in combination with the GC-rich spacer region , maintains the topology of the promoter in a conformation that enables RNAP to begin hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>When H-NS is dominant RNAP binding is inhibited and hlyE expression is silenced ; when SlyA is dominant H-NS binding is inhibited allowing RNAP access to the promoter facilitating hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>When H-NS concentrations increase relative to SlyA , SlyA and RNAP binding is inhibited and hlyE transcription is silenced ( A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>HdfR binds to the promoter region of the ﬂhDC operon , and its expression was negatively regulated by H-NS , suggesting that the apparent activation of ﬂhDC transcription by H-NS is mediated through the negative regulator HdfR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Upon the activation of different transcription activators in response to different environmental or physiological conditions that presage acid stresses or efﬂux of certain metabolic by-products , the activators then displace H-NS binding on the promoter , resulting in the activation of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Binding of H-NS upstream of the promoter represses transcription initiation , whilst binding within the coding region is also proposed to repress transcription elongation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>H-NS binds upstream of the promoter and represses transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>One possibility is that H-NS is inactive under anaerobic conditions and it disassembles from the gadE promoter , allowing ArcA which is active under this condition to-bind the promoter of gadE and initiate gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>If H-NS is inactive under anaerobic condition and is disassembled from the gadE promoter , then transcription of PgadE-lacZ in arcA hns should be similar to that of arcA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Nevertheless , in a fis mutant the bgl promoter activity is approximately five-fold increased in the early log phase in rich medium ( unpublished results ) , and we are currently testing whether this is a direct effect of FIS-binding and whether indirect FIS-effects due to alterations of H-NS , CAP and gyrase levels in the fis mutant contribute to the activation of the bgl promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>Indeed , as GadX is synthesized and its concentration rises over a threshold level , it can successfully relieve H-NS repression at the gadA promoter ( as shown in Fig. 7A , a ratio GadX : H-NS ranging between 1:10 and 1:20 is required to alleviate the H-NS-dependent inhibition ) , further causing its own production from the dicistronic gadAX mRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>However , in the case of the yjjP-yjjQ-bglJ locus , H-NS binds to the center of the intergenic region with the highest afﬁnity , while LeuO binds with highest afﬁnity to the yjjP promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>One such gene is nir ( 3 ) where H-NS binds to the upstream region in concert with FIS and IHF to repress promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>However , our data indicate that H-NS binds upstream of the promoter and enhances its activity in a very e $ ective manner , namely only when the switch is in the ON orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>Interestingly , the indirect effect , by counteracting H-NS inhibition , is more productive than the direct stimulation of gadA promoter activity by GadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Therefore , possible antagonistic effects between GadX and H-NS , resulting in activation/repression of gadA gene , have been investigated in vitro by monitoring promoter activity in the presence of both purified proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>This observation is in line with the finding that even at the highest GadX concentration tested , in vitro gadA promoter activity in the presence of H-NS never attains the values obtained with GadX alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In addition , reversal of H-NS repression at the ftnA promoter required both Mn2 + and Fur showing that Fur needs its co-effector to achieve the observed antirepression effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>However , it is unclear how this effect is exerted , and whether or not H-NS and Fur act directly at the ftnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In summary , the above data suggest that both H-NS and Fur act directly upon the ftnA promoter to mediate iron-dependent induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>This type of behavior , in which speciﬁc patterns of protection are replaced by general protection as the concentration of H-NS increases , has been observed for footprints of gal promoter variants ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In the absence of H-NS , the level of csgD mRNA significantly increased ( Fig. 2a , lanes 1 and 5 ) , indicating that H-NS is a repressor of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In order to further localize the clyA promoter active in the absence of H-NS , the clyA transcript was assayed by primer extension analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In fact , the promoter activity directed by pKgadA-69 and pKgadX-67 , was decreased by H-NS to 85 and 52 % of the starting values , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In another mutant lacking H-NS activity , resulting in increased ss level in log-phase , the transcription from promoter P 1 during log phase was increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9126832" target="_blank">9126832</a>]<br></li>
<li>Therefore , we could not test whether BglJ-RcsB further enhances bgl promoter activity in the absence of H-NS and StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>As expected , the P1 promoter activity increases from early to mid-log phase in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>As shown in Fig. 4 , the first fragment to be displaced as the amount of H-NS added was increased was the chiA promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>5 , activation of the pcsgBA promoter at 26 C remains dependent on the UAS positioned between - 5 4 and - 9 8 in the absence of CT ^ and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>It appears that H-NS specificaily antagonizes the Fismediated activation of the PI promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Binding of H-NS to the dps promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Binding of LeuO presumably delimits the spread of H-NS into the promoter and thus causes activation ( 4 , 6 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>We observed that the two forms of RNPs could form an open initiation complex ( RPO ) at hdeABp , but that E 70 failed to initiate transcription in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Inhibition of P1 and P2 transcription by H-NS was greatly reduced in the presence of potassium glutamate ( Fig. 4A , lanes 5 to 8 , and Fig. 4B , lanes 4 to 6 and 10 to 12 ) , and there was some evidence that this relief of inhibition might be speciﬁc for proU ( compare , for example , the degree of general inhibition of background radiolabel incorporation in lanes 3 and 6 in Fig. 4B with the intensities of the P2 transcript in these two lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Densitometric analysis of bands on the autoradiograms depicted in Fig. 4 indicated that the residual extent of P2 transcription following addition of 10 pmol of H-NS ( compared with no H-NS addition ) was 13 and 56 % , respectively , in the absence and presence of potassium glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>We suggest that the key problem in curli synthesis is the repression of csg transcription through multiple repressor systems responding to closely related stimuli ( Fig. 7 ) , such as ( i ) surface sensing , envelope stress , and high salt content for CpxR ; ( ii ) osmolarity , desiccation , and membrane perturbation for RcsB ; and ( iii ) high sucrose content for H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Consistent with our previous results ,13,21 no Pcas-specific cDNA product was detected in wild-type cells , but an efficient transcription could be demonstrated in the hns-deficient or leuO strains , confirming the antagonistic regulation of Pcas C transcription by H-NS and LeuO ( Fig. 1A , lanes 2 , 6 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23392250" target="_blank">23392250</a>]<br></li>
<li>Specifically , the observations that the increase in transcription induced by multicopy slyA was H-NS-dependent and that in an hns mutant multicopy slyA failed to restore the level of transcription to that seen in the wild type at 37 ° C ( Table 3 ) indicate that H-NS is required for maximal SlyA-mediated activation of transcription from PR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>A complex acid resistance regulon including the gad system is regulated by transcription factors GadX-GadW and EvgA-YdeO , as well as by RpoS , H-NS , and cyclic AMP ( 11 , 12 , 44 , 79 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601715" target="_blank">15601715</a>]<br></li>
<li>Their regulation is known to be mediated by transcription factors GadXGadW and EvgA-YdeO , as well as by RpoS , H-NS , and cyclic AMP [ 60,61,74 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>For a more detailed description of the model , see also the Discussion section No transcription 600 D. Tippner , H. Affierbach , C. Bradaczek and R. Wa A model explaining changes from activation to repression g n e r We have summarized the available information on H-NS and Fis binding and their effects on rRNA transcription , and have derived a model explaining changes from aotivation to repression caused by the interdependence of the two proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>As just discussed , a signiﬁcant contribution is also made by the replacement of the activator Fis by the inhib-itor H-NS ; when the nutritional environment changes so as to support new growth , the NTP and Fis concentrations rise and ribosomal transcription is stimulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686546" target="_blank">15686546</a>]<br></li>
<li>Although gadA transcription is stimulated more than 2-fold by GadX alone and strongly repressed ( up to 70 % ) by H-NS alone , as also shown in Figs. 4 and 6 , the effect of simultaneously adding both proteins to the reaction mixture gives rise to the gadA transcription profiles shown in Fig. 7A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The authors observed that the increase in gadA transcription , obtained as an indirect effect of counteracting H-NS inhibition , was greater than the direct stimulation of gadA transcription in the presence of GadX alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>The GadX-dependent stimulation , without and with different amounts of H-NS , is more apparent when data from Fig. 7A are plotted as the relative increase ( - fold ) of gadA transcription versus GadX concentration ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The results can be summarized as follows , ( i ) The addition of Fis to the transcription assay resulted in the expected ( c. eightfold ) increase of PI promoter activity , ( ii ) Addition of H-NS in the absence of Fis caused a moderate but significant repression of Pi-directed transcription , ( iii ) Repression was dependent on the H-NS concentration , with no more PI-directed transcription detectable at2MM H-NS or higher , ( iv ) H-NS-dependent repression was maximal in the presence of Fis , completely antagonizing Fis-dependent activation at concentrations above 1,4 nM ( v ) H-NS almost exclusively affected Pi-directed trans-cription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>As shown in Fig. 4 , transcription from pKgadA-179 and pKgadX-222 , carrying the entire promoter regions of the genes of interest , was inhibited by H-NS in a dose-dependent manner , and in the presence of 1 M H-NS , residual promoter activity of gadA and gadX was 30 and 25 % , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The level of ~ - galactosidase activity expressed and the amount of H-NS produced by the cells exhibited close inverse correlation , suggesting that H-NS may directly repress the transcription initiating at its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Similarly , when cells were grown at 25 ° C , deletion of H-NS resulted in an increase in promoter activity and rapid triggering of dps transcription at a lower OD650 ( Fig. 2C , i ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>For instance , the osmotic induction of StpA may contribute to the induction of some osmotically regulated , H-NS-repressed genes by altering the ability of the H-NS complexes at the promoters of those genes to repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Under standard laboratory growth conditions transcription is repressed by H-NS and/or StpA ( repressed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Many stressresponse genes are both repressed by H-NS and activated by RpoS , and similar kinds of conditions both relieve H-NS silencing and stimulate transcription mediated by RpoS ( 15 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The fact that ArcA still has a regulatory role in the absence of H-NS strongly suggests that , in addition to antagonizing H-NS repression , ArcA also directly activates cydAB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Our results conﬁrm that transcription of leuO is repressed by H-NS , and further show that StpA contrib-utes to this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>The three transcription factors , H-NS , LRP and StpA , cause increasingly strong repression for both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>The results ruled out the possibility of an indirect effect via Mlc repression of malT transcription as the reason for the effects of H-NS and StpA found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The above data suggest that LeuO is an autoregulator that may have a moderate positive autoregulatory effect , but that strongly represses leuO transcription in the absence of H-NS and StpA and that counteracts activation of leuO by RcsB-BglJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>The induced StpA evidently exerts its negative effect on transcription from the proU promoter independently of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The current view of the mechanism of H-NS-mediated repression is that H-NS binds as a dimer to AT-rich and planar-bent DNA sequences and then forms an extended oligomeric complex on the DNA , which represses transcription initiation when located next to a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Moreover , consistently with the failure of several attempts to detect any interaction of H-NS with the gadBC promoter , no H-NS repression was observed when the pKgadB-244 construct was used as DNA template in in vitro transcription assays The above observations were also confirmed by in vivo assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>However , the greatest repression in our in vitro transcription assays was observed when H-NS was added to pYANK1 , a supercoiled template harboring ﬁmB under the control of its natural promoter and upstream 59 repeat sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Fur counteracts H-NS repression of ftnA transcription in vitro To discover whether interaction of Fur and H-NS with the ftnA promoter region directly affects ftnA transcription , in vitro transcription reactions were performed using a PCR-ampliﬁed ftnA promoter fragment specifying a 370 nt ftnA transcript ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The reversal of H-NS-mediated repression by Fur was speciﬁc for the ftnA promoter since Mn2 + -- Fur could not reverse H-NS repression of proU in vitro transcription ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Stoebel , D.M. , Free , A. , and Dorman , C.J. ( 2008 ) Antisilencing : overcoming H-NS mediated repression of transcription in Gram-negative enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Furthermore , H-NS-mediated inhibition of P2 transcription was not observed in the presence of potassium glutamate , suggesting that potassium glutamate relieves the P2 repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>According to this model , ArcA-P plays a central role in cydAB regulation by antagonizing H-NS repression of cydAB transcription when oxygen becomes limiting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>At the capsule nonpermissive temperature , the loss of H-NS resulted in a 2.5-fold increase in transcription from PR1 , confirming a role for H-NS in the repression of capsule gene transcription at 20 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Effects of H'NS binding on in vitro transcription H-NS has been shown to be a bacterial transcription factor , mediating both activation and repression ( e.g. Bertin etai , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Thus , whereas the in vivo results indicate the need for NRE in mediating proU repression by H-NS , the data from this study demonstrate that H-NS represses P2 transcription in vitro even from templates lacking the NRE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>These results are in agreement with the in vitro transcription data ( Fig. 7 ) , and suggest that YncC activation and H-NS repression occur independently at pyciGSTM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In this study it is shown that transcription of hlyE in vitro is repressed by the addition of H-NS alone and it is likely that this is caused by H-NS dimers initially bridging between the H-NS I and H-NS II sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In this study , using in vivo and in vitro analyses the interactions between H-NS , SlyA and RNAP at the PhlyE were examined to elucidate the mechanism whereby SlyA can antagonize H-NS-mediated repression of hlyE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In vivo and in vitro transcription assays suggest that an increase in temperature alone can not relieve repression of hlyE by H-NS ( Table 1 , Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>I reduced transcription Function of H-NS as transcriptional-repressor We have shown here that H-NS acts as a specific repressor for PI-directed ribosomal RNA transcription , antagonizing Fis-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>It is still unclear whether in vivo H-NS could function as a repressor of rRNA transcription by its own , or whether it only counteracts the activation brought about by FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>It is thus tempting to speculate that the major role of the second activator CaiF would be to counteract the repressive effect of H-NS on transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Moreover , in vitro transcription analysis with both RNA polymerase holoenzymes , Es70 and Es38 , demonstrated a direct inhibition of 6S RNA transcription by H-NS , StpA and LRP , while FIS seems to act as a dual regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>An alternative possibility is that the nature of the H-NS scaffold inﬂuences properties of the transcription elongation complex , such as its stability , pausing behavior , or proneness to backtracking and inactivation ( 6 , 40 , 44 , 47 ) , and its consequent susceptibility to the process of Rho-dependent termination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Thus , while inhibition of cat transcription from pKK660 is already strong at low concentrations of H-NS -LCB- 50-100 nM ) , transcription from pKK110 began to be inhibited only at concentrations > 200 nM H-NS dimer ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>These results indicate that transcription from P1 occurs via s70 and is positively regulated by NhaR in a fashion that is modulated by the level of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>These results , considered altogether , fulfil the predictions that follow from the proposal that H-NS reduces the ability of Lrp to activate ilvIH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Expression of the cps-lacZ fusion is dependent on RcsA ; transcription of rcsA is in turn negatively regulated by H-NS ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770508" target="_blank">9770508</a>]<br></li>
<li>H-NS negatively regulates evgA and gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>H-NS negatively regulates its own transcription ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>H-NS regulation of the ﬂhD operon is complex , however ; although an hns mutant was nonmotile , in vitro transcription of the ﬂhD operon by H-NS was negative ( 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>It was clearly demonstrated that H-NS , when added to the reaction mixture , specifically inhibited hns transcription but not tac transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Although it was assumed that H-NS positively affects ﬂhDC transcription , its regulation has not been clearly demonstrated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>As H-NS and Es70 could not act as co-repressors , we reasoned that , in contrast to Fis , H-NS was likely to repress transcription by Es70 via a simple blocking mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Schematic model for the function of Fis and H-NS in activating and repressing rRNA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>IHF and OxyR both function as activators , while Fis and H-NS repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>An in vitro transcription assay demonstrated that P1 is activated by s70-RNA polymerase and both NhaR and H-NS increase the speciﬁcity of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>H-NS further increased the speciﬁcity to P1 , but the most pronounced speciﬁc transcription from P1 was obtained by adding both H-NS and NhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>H-NS binds tightly to dsDNA in physiological buffers , increases its thermal stability and inhibits transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2841565" target="_blank">2841565</a>]<br></li>
<li>In vitro transcription experiments showed that CAP activates ﬂhDC expression while H-NS represses it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>In in vitro reactions , H-NS can inhibit transcription [ 66 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Effects of LRP and H-NS on in vitro transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>H-NS represses transcription of ﬁmB in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Thus , H-NS directly represses the transcription of ftnA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Models for the H-NS - and Fur-mediated induction of ftnA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>C. Effect of increasing amounts of H-NS on cat transcription after 30min incubation with pKKHO ( .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Such interactions can either reduce or increase the ability of H-NS to repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>694 J. K. Lithgow , F. Haider , I. S. Roberts and J. Green effect on hlyE transcription in vivo in the absence of H-NS , and active CRP and FNR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Effects of FNR , CRP , and H-NS on in vivo transcription of hlyE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Furthermore , we demonstrate that an additionai operon , hdeAB , which is also dependent upon a ^ for transcription , can be transcribed by Ed ™ in vivo in the absence of IH-NS by utilizing the phdeAB promo-ter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Furthermore , the negative influence of H-NS is not confined to transcription -- it can also inhibit recombination24 ,25 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>RovA was also shown to activate inv transcription by direct contact with RNAP in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>However , SlyA appears capable of activating transcription to some extent in the absence of H-NS , something that was also recently demonstrated for RovA ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Salmonella enterica response-regulator SsrB relieves H-NS silencing by displacing H-NS bound in polymerization mode and directly activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Two proteins , RimJ and H-NS , are known to be important in the regulation of papBA transcription in response to temperature and other environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>H-NS and RpoS are apparent antagonists that either repress or activate transcription , respectively , in response to various environmental signals ( reviewed in ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>A PCR-ampliﬁed DNA fragment extending from -474 to +222 relative to the hlyE transcript start containing the hlyE promoter and the promoter for the divergently transcribed umuD gene was used in in vitro transcription assays in the presence of increasing concentrations of H-NS as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Also , Fur was only able to displace H-NS in the presence of MnCl2 ( not shown ) , which supports results obtained using in vitro transcription ( provision of Fur without Mn2 + produced no Fur footprints such that only the H-NS binding sites were visible ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In the presence of Fe2 + -- Fur , H-NS sites II -- VI are occupied by multiple Fe2 + -- Fur dimers which prevents cooperative binding of H-NS at other sites ( I , V and VI ) , thus relieving the H-NS-dependent repression of ftnA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>H-NS is still capable of repressing gadA transcription in the presence of low concentrations of GadX ( below 25 nM ) , thus significantly counteracting the GadX-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Furthermore , it is important to notice that the activation of gadA transcription by GadX in the absence of H-NS is not cooperative , whereas it shows a substantial degree of cooperativity in terms of effects in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In the absence of H-NS , transcription from osmCp2 is stimulated but remains totally ss dependent ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>In the stpA hns double mutant , transcription of micF is enhanced as a result of the absence of H-NS , and the micF RNA is more stable in the absence of StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Therefore , it can be seen that the presence of H-NS was required for maximal SlyA promotion of transcription from PR1 , indicating a functional interaction between H-NS and SlyA in the regulation of transcription from PR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Secondly , ArcA activates transcription directly , regardless of the presence of H-NS ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In the presence of increasin amounts of the different bacterial regulators FIS , H-NS , LRP and StpA , the transcription products were affected in a differential way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>In contrast , the presence of 334 nM H-NS in the reaction mixture resulted in a sevenfold decrease in ﬂhDC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>( B ) In vitro transcription in the absence or presence of His-tagged NhaR ( 400 ng ) , H-NS ( 40 ng ) , or both as indicated FIG. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>In vitro transcription analysis in Ihe presence of Fis and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In vitro transcription was carried out with supercoited plasmid DNAs as templates in the presence of purified H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>In vitro transcription assays as described in ( C ) but in the presence of both H-NS ( 2 mM ) and increasing amounts of SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In vitro transcription from the pHYD351 and pHYD355 templates was studied in the presence of increasing concentrations of H-NS ( Fig. 4 ) FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Because the product of the ilvlH gene , AHAS III , is specific for isoleucine biosynthes ; s and thus competes for the threonine pool ( see fig i ) , we wanted to test if H-NS would also regulate the action of the kblitdh operon for which Lrp acts as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The active participation of H-NS in many cellular functions thus provides the cell with additional flexibility to fine-tune th level of gene expressionin response to demands imposed by the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Hha therefore functions to alter the activity of H-NS by forming a complex to regulate a specific set of genes , of which the α-haemolysin gene is the best characterized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Summary The LysR-type transcription factor LeuO is involved in regulation of pathogenicity determinants and stress responses in Enterobacteriaceae , and acts as antagonist of the global repressor H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_919">
Pul , U. , Wurm , R. , and Wagner , R. ( 2007 ) The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></label>
<input id="act_similarity_919", type="checkbox">
<div class="ACT_similar">
<li>, Wurm , R. , and Wagner , R. ( 2007 ) The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromo-lecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li></div>
<li>The study also supports the assumption that H-NS , LRP and StpA directly repress 6S RNA transcription , while effects observed with the fis mutant are consistent with a more complex dual regulation mediated by this regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>The results strongly support the direct negative regulation of the global regulators H-NS , LRP and StpA in the control of 6S RNA transcription and suggest that FIS probably has a dual functional role in ssrS reg ulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>Thus , the data strongly suggest that the combined H-NS - and Fur-mediated regulation of ftnA induction by iron in vivo is the result of direct interaction of these proteins at the ftnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The ability of H-NS to speciﬁcally recognize and retard the ftnA promoter fragment strongly supports a direct role for H-NS in mediating the Fe2 + -- Fur-dependent regulation of ftnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The role of H-NS , therefore , appears to be to place the promoter in an appropriate conﬁguration so that it can show increased sensitivity to regulation by NhaR and Na1 , allowing signiﬁcant induction ratios .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>The binding of H-NS appeared to be co-operative with a small increase in H-NS concentration resulting in complete retardation of the pnir fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The mechanism for the synergistic increase in binding afﬁnity for H-NS and LRP observed in this study is not completely clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>pKK400 and pKK660 were respectively : 5486 , 2963 and 2937 at Agao = 0-15 ; 2142 , 760 and 776 at \ zo = 0-7 ; 3560 , 3585 and 2894 at Aezo = 3-0 The binding of purified H-NS to specific regions of its gene and the effect of H-NS on the transcription of tins-caMusions were then studied in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>The effect of LRP on the binding of RNA polymerase to the rrnB P1 promoter and in vitro transcription experiments indicated that LRP acts as a transcriptional-repressor , thus resembling the activity of H-NS described previously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>The binding of IHF may therefore in ¯ uence the binding of H-NS in such a way that H-NS enhances transcription from the ® mA promoter only in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>The reason why the remodeled nucleoprotein complex is conducive to transcription is unclear , but the simplest interpretations are that the promoter conformation induced by simultaneous SlyA and H-NS binding favors either open complex formation or the binding of a further Class I or Class II transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>The relief of H-NS silencing by SlyA was also much less efﬁcient in the absence of CRP , since only a very weak cytolytic activity could be detected in CRP-deﬁcient strains overexpressing SlyA ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The binding of H-NS to an upstream silencer results in an ~ threefold repression of the catabolite gene regulator protein ( CRP ) dependent bgl promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The activity ot carnitine dehydratase was dependent on the CRP regulatory protein and strongly enhanced in the absence of a functional H-NS protein , in relation to the consensus sequences detected in the promoter region of the cai operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7815937" target="_blank">7815937</a>]<br></li>
<li>The activity of the CRP + promoter increases ~ 1.5-fold in the hns mutant ( from 865 to 1275 units ) to the level of the D , CRP + double mutant promoter , which is not affected by H-NS ( Fig. 2C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The antirepressor mechanism proposed for activation of gadA is unlikely to be responsible for gadBC activation because H-NS does not bind to the gadBC promoter ( 14 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The transcriptional activity in vivo , as estimated by the measurement of - galactosidase activity , indicated that this ( pGS1065 ) low-copy-number reporter containing a minimal hlyE promoter responded in the same way as the longer medium-copy-number hlyE : : lacZ fusion ( pGS1629 ) to the presence or absence of H-NS ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The finding in this study that H-NS can function as a transcriptional-repressor for its own promoter is fully compatible with this hypothesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>The results reveal that the anti-cas promoter , identiﬁed in this study , is active in natural chromosomal context in absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The activity of H-NS is enhanced or diminished by other proteins including its homologue StpA , and Hha and YdgT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>The absence of the H-NS and StpA proteins resulted in a reduction in the levels of MalT to about 20 % the normal level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The lower stability of StpA in the absence of H-NS suggested that the levels of homodimers of StpA in the cells are very low ( Johansson and Uhlin , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>The levels of known cold-shock-induced proteins , such as CspA , CspG , and H-NS , increased upon the shift to the lower temperature ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422595" target="_blank">9422595</a>]<br></li>
<li>The average DsrA-induced decrease in H-NS levels was consistently ' 3-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The precise mechanism by which this variatíon in H-NS content in the cell is achíeved is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>The bound proteins were then probed with the rabbit H-NS antiserum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>The intracellular levels of the major DNA-binding proteins , Fis , H-NS , HU and StpA , in the growing cells decreased in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>The best characterised and most abundant members of this group of proteins are HU , H-NS , IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>The lack of cas transcripts in the stpA - strain proofs that silencing in the cell is mediated by H-NS and not by StpA although StpA binds quite strongly to the Pcas upstream region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The autoregulation presumably keeps the Mlc level rather constant ; this idea argues against the hypothesis that the effect of H-NS deﬁciency is mediated via Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The cadBA operon and the adiA gene , which encode a lysine decarboxylase , an antiporter and an arginine decarboxylase , respectively , have been shown to be regulated by H-NS in rich medium under anaerobic conditions ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>The activity of H-NS appears to be correlated with its cellular concentration It is known that the concentration of FIS is precisely regulated through the cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>The effects of either the dsrA plasmid or an hns mutant were compared for lacZ fusions to two promoters that are negatively regulated by H-NS , proU and papA ( 4 , 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_956">
The majority of genes are negatively regulated by H-NS , although some of them , including the ¯ agellar regulon , are positively regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></label>
<input id="act_similarity_956", type="checkbox">
<div class="ACT_similar">
<li>The majority of affected genes are negatively regulated by H-NS , although some , including the ﬂagellar regulon , are positively regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li></div>
<li>The fact that no signiﬁcant differences could be observed probably reﬂects that , under physiological conditions , the cellular concentration of Hha/H-NS or Ydgt/H-NS complexes is too low to be detected by standard techniques .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>The modulatory properties of the different heteromeric complexes formed ( i.e. Hha -- H-NS , YdgT -- H-NS , Hha -- StpA , etc. ) and their formation in response to speciﬁc modulatory responses ( extreme osmolarity , changing temperature conditions or other stresses ) should be examined , as should the sets of genes regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>The effect of H-NS on rRNA synthesis was further assessed under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>The genes affected by H-NS are dispersed throughout the chromosome and are often regulated in response to changes in environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The ﬁrst set of scanned genes was initially compiled to include genes responding to H-NS removal under different growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>These effects are also clear in the E 0 data ( Figure 2D , lower panel ) where troughs ( maximal repression in the absence of IPTG ; open circles ) are deepened in the absence of H-NS , while peaks ( promoter leakiness with out-of-phase operators ) are unchanged .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Our results indicate that relief of repression by H-NS plays a role in stationary-phase induction as well as in hyperosmotic induction of rpoS translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Our results indicate that relief of inhibition by H-NS is involved in both stationary-phase-associated and hyperosmotic induction of rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>c Results in italics ( P 0.05 ) or underlined ( P 0.01 ) indicate that MarA produced a statistically signiﬁcant change in the activation or repression produced by a regulator under the same conditions ( e.g. , the repression produced by H-NS on hdeAB in stationary phase at pH 5.5 was signiﬁcantly stronger in the presence of MarA than in the absence of MarA ) FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>In turn , MarA enhanced repression of hdeAB by H-NS and hindered activation by GadE in stationary phase and also reduced the activity of GadX , GadW , RpoS , and Lrp on hdeAB under some conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>We found that , in wild-type cells grown in LB medium at pH 7.0 or pH 5.5 , repression of hdeAB by MarA occurred only in stationary phase and was reduced in the absence of H-NS and GadE , the main regulators of hdeAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Conversely , MarA helped H-NS repression and interfered with GadE activation of hdeAB in stationar phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Therefore , both H-NS and GadE were involved in the increased hdeAB repression by MarA that occurred as cells enter stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>MarA enhanced hdeAB repression by H-NS in stationary phase at both pH 7.0 and pH 5.5 , which indicates that MarA assists or collaborates with H-NS to repress hdeAB in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>H-NS and GadE were associated with the increase in hdeAB repression by MarA that occurred in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>In particular , during the logarithmic phase of growth , H-NS dominates , whereas increased levels of S during stationary phase might overcome H-NS repression primarily at the responsive gadX promoter , leading to induction of the whole gad system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>During the stationary phase , the ss level increases and this factor can overcome H-NS repression , probably by more efﬁcient recognition of bent promoter regions required for H-NS binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Although reproducible , the enhancement in the abundance of the hlyE transcript when SlyA was added to the PhlyE : H-NS complex was somewhat disappointing under the conditions used for the experiments in Fig. 5A and B. Using an equivalent supercoiled template in place of the linear-ized DNA used for the experiments in Fig. 5A and B failed to improve SlyA-mediated inhibition of SlyA repression under the same conditions ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>However , at 37 ° C , addition of 1 mM or 12 mM SlyA partially relieved the repression caused by the presence of 0.5 mM H-NS ( Fig. 5B , compare lanes 5 and 6 with lanes 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Rather , it is reminiscent of repression of rrnB P1 by H-NS , in which an open complex , formed in the presence of H-NS , may be too stable to go into the elongation mode ( Schroder and Wagner 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>H-NS binding results in repression of the four regulated cydAB promoters ( P1 , P2 , P3 and P4 ) ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>These results suggested that , when present in high amounts in E. coli , YncC/McbR counters H-NS mediated-repression at pyciGECO ( Fig. 4B ) by modulating H-NS binding and , ultimately , by counteracting the negative effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Thus , StpA complements H-NS proteins defective in DNA binding to repress bgl , while in autoregulation of stpA it acts autonomously , indicating a difference in the mechanisms of repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>To further address the interdependence of StpA and truncated or mutated H-NS proteins in the repression of stpA and bgl , dominant-negative mutants of H-NS defective in dimerization/oligomerization or speciﬁc DNA binding were used ( 2 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>In contrast , repression of bgl by StpA occurs only in the presence of the N-terminal H-NS protein domain or an H-NS-I119T mutant , i.e. , in the presence of mutant H-NS proteins defective in speciﬁc DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Repression of bgl becomes StpA dependent in the presence of H-NS mutants defective in ( speciﬁc ) DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>The cryptic bgl operon is activated by point mutations that improve the CRP binding site of the bgl promoter ( 6 ) , and several other H-NS repressed genes are activated by specific DNA-binding proteins that antagonize H-NS-mediated repression in response to specific stimuli ( 41 -- 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>Current evidence suggests that osmotic induction of proU occurs partly through the potassium-mediated relief of repression by the DNA-binding protein H-NS ( 25 , 29 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002611" target="_blank">8002611</a>]<br></li>
<li>DNA binding is not sufﬁcient for H-NS mediated repression of proU expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Jordi , B.J.A.M. , Fielder , A.E. , Burns , C.M. , Hinton , J.C.D. , Dover , N. , Ussery , D.W. , and Higgins , C.F. ( 1997 ) DNA binding is not sufﬁcient for H-NS mediated repression of proU expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Disruption of H-NS binding or oligomerization by site-specific DNA binding of the lcI or LacI repressors has been shown to cause transcriptional activation by releasing H-NS repression ( Caramel and Schnetz , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Therefore , another mechanism , such as a structural change in the DNA induced by H-NS , would better account for the repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Furthermore , DNA-binding proteins , which bind in the vicinity of H-NS repressed promoters , are frequently observed to alleviate repression ( 6 , 41 -- 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>form composition and the action of H-NS in the processes of piliation and motility , as well as the more frequently studied roles of DNA binding and gene repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>This would seem to represent a strategy for overcoming H-NS repression that relies on competition for common binding sites in DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>This suggested a regulatory mechanism whereby increased intracellular concentrations of SlyA remodel H-NS binding at PhlyE to relieve H-NS-mediated repression by allowing RNAP to access the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Possibly , LeuO has a dual role in the regulation of the yjjP promoter ; it may repress the promoter directly and also counteract its repression by H-NS when the binding site downstream of the yjjP promoter is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>In addition , K gtutamate may act by relieving the transcriptional repression caused by binding of H-NS ( Ueguchi and Mizuno .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476160" target="_blank">7476160</a>]<br></li>
<li>On the other hand , binding of H-NS to Hha-like proteins enhances repression by H-NS , which would potentiate rather than inhibit dimerization and oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Most commonly , repression by H-NS is relieved by the binding of speciﬁc transcriptional-regulators , which compete with H-NS for binding or which restructure the * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>Our attempts to phosphorylate H-NS , and thereby to restore binding or repression activity , were unsuccessful , however .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Therefore , ArcA may preclude H-NS binding to this region , thus preventing repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Phosphorylated ArcA ( ArcA-P ) antagonizes H-NS binding to gadE promoter , resulting in the relief of repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>In hupAhupB mutants proU expression is more strongly repressed , which could be due to to a higher e ¡ ective repressive activity of H-NS , whereas overproduction of HU apparently alleviates the repression by H-NS [ 68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>In other words , as the amount of H-NS increased rapidly , the expression of 13-galactosidase activity concomitantly decreased , suggesting that the apparent repression of hns transcription is most probably a primary consequence of the accumulation of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Our results clearly prove that H-NS regulates gad expression by acting at the transcriptional level because repression of gad transcription during exponential-growth is relieved in the hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Our ﬁndings demonstrating that ArcA activates the expression of the operon through antagonizing the H-NS mediated repression probably provide a common model that underlies the dynamic and coordinated regulation of gadE-mdtEF in response to different environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>This dual role of $ S in the control of the operon , and the inverse correlation , observed in this study , between the $ S level and the magnitude of H-NS repression , might account for the strong sensitivity of yciGFEkatN expression to $ S levels and to activation by Crl ( 21 ) , the $ S - chaperone that increases $ S activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>We report , as well , the role of the nucleoid associated proteins H-NS and Hha in the repression of htrA expression at low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Therefore , it is possible that StpA plays a role in modulating the expression of OmpF in response to environmental signals under conditions in which H-NS repression has been removed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Maximal expression under intermediate ( microaerobic ) oxygen levels is achieved by the counterbalance of activation by phosphorylated ArcA and repression by H-NS under aerobic conditions and repression by Fnr under anaerobicity [ 60 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>ArcA is shown to antagonize H-NS action to render cydAB expression insensitive to H-NS repression in anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>proU expression is mainly under the repression effect of another NAP , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Binding of LeuO and putative concomitant DNA looping could prevent spreading of the H-NS nucleoprotein complex into the yjjQ promoter and thus relieve its repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>By consideration of this model , our data suggest that H-NS may bind upstream and downstream to the stpA promoter to form such a stable nucleoprotein complex , resulting in repression of the stpA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Taken together , within the natural context of the bgl operon the relief of the H-NS mediated silencing of the promoter masks the H-NS mediated repression at the downstream silencer due to increased transcription and translation of bglG , positive autoregulation of BglG , and limitation of the maximum expression level of the operon by the PTS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Mutations characterized so far that relieve silencing ( or repression ) of the bgl operon by H-NS affect only one level of repression , the repression of the bgl promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>This assumption was drawn because of the fact that mutations mapping in cis to the promoter and preventing repression of the promoter by H-NS completely relieve silencing of the operon ( see Introduction ) , which raises the question of why the downstream repression is masked , when repression of the promoter is relieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>However , as in case of proU in © 2004 Blackwell Publishing Lt 590 S. Dole , V. Nagarajavel and K. Schnetz vitro repression of the bgl promoter by H-NS is merely four - to ﬁvefold ( Schnetz and Wang , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>These results support previous data indicating that the upstream silencer is necessary for repression of the promoter by H-NS and that efﬁcient binding of CRP counteracts the H-NS-mediated repression of the bgl promoter ( Schnetz and Wang , 1996 ; Mukerji and Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The data also show that downstream sequences contribute to repression by H-NS and StpA , as repression was 30-fold when lacZ was fused downstream of stpA ( orf ) , while it was only 6 - to 8-fold in the mutants carrying stpA promoter-lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>These data indicate a difference in the mechanisms of repression of stpA and bgl by StpA The speciﬁcity of regulation of stpA by H-NS is high ( 30-fold ) in the presence of a downstream regulatory element ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Taken together , these data show that H-NS is necessary and sufﬁcient for maximal repression of stpA , while StpA moderately represses stpA in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Furthermore , we have shown that StpA together with H-NS-I119T represses bgl more effectively than does wild-type H-NS , although H-NS-I119T is dominant negative in repression of proU ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>During aerobic-growth , the gadE-mdtEF is expressed at a very low level due to the strong repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Thus , the presence of the downstream sequences alone results in a rather low , possibly signiﬁcant level of repression by H-NS and StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>However , cellular levels of StpA are low due to repression of stpA by H-NS and negative autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>The cellular level of StpA is signiﬁcantly lower than that of its abundant homologue , H-NS , most likely due to repression of stpA by H-NS ( 37 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>The extent of transcriptional auto-repression was found to depend on the stage of growth , being maximal in mid-log cells and minimal in late-log when the amount of H-NS in the cell is reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>The reduction in H-NS translation was accompanied by an increase in StpA levels ( Fig. 5B ) , which can be attributed to decreased transcriptional repression of StpA by H-NS ( ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The degree of Irp repression by H-NS in vivo was relatively small under normal growth conditions ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>The same trend was observed in CJD1091 , indicating that this repression was partially independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The magnitude of H-NS repression was not affected by YncC ( Fig. 7 ) , and the magnitude of YncC activation was not affected by H-NS ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Quantitation of ﬁmB and 5S mRNA transcriptional repression depicted in Fig. 5B and C Fold decrease in transcriptiona with H-NS concn ( pM ) of : 195 Plasmid 97.5 390 FIG. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Activation of motility by H-NS has recently be found to involve its repression of a transcriptional-repressor of flhDC , hdfR ( Ko and Park , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Under all the conditions studied , H-NS was the main repressor of hdeAB , although repression by H-NS was about half as great at pH 5.5 as at pH 7.0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Taken together , we conclude that the anaerobic activation of the gadE-mdtEF was primarily mediated by the two-component system ArcBA through antagonizing the H-NS mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>We found that unlike the regulatory patterns occurred during aerobic acid stresses , anaerobic activation of the gadE-mdtEF operon is primarily mediated by ArcA through its antagonization of the H-NS mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Thus , anaerobic activation of gadE-mdtEF through antagonizing the H-NS mediated repression by ArcA provides a regulatory model that is relevant to the dynamic lifestyle of the facultative bacterium E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>This led us to ask how the H-NS mediated repression is replaced by the ArcA dependent activation during the transition of the facultative bacterium E. coli from its aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Two possibilities may explain the transition from H-NS mediated repression to ArcA mediated activation of gadE-mdtEF during the transition from aerobic to anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>FIGURE 6  ArcA antagonizes the H-NS mediated repression of gadE under anaerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>This is in agreement with the proposed competition between activation by specific regulators and repression by H-NS , in several bacterial systems [ 24,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>As ArcA most often acts as a repressor , it is tempting to speculate that ArcA may exert its activation by relieving H-NS repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Is H-NS repression restricted to the counteraction of FIS-dependent activation ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Interestingly , we demonstrated that the ArcA dependent activation of the operon is through its antagonistic effect on the H-NS mediated repression of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>In E. coli , this locus is subject to gene rearrangements and also likely to horizontal gene transfer , consistent with its repression by the xenogeneic silencer H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Hence , it was interesting to evaluate whether H-NS plays a role in nrdAB repression when E. coli cells are growing anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>These results indicated that $ S induces expression of yciGFE in E. coli K-12 only when H-NS mediated repression was relieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Nagarajavel , V. ( 2007 ) H-NS mediated repression of the E. coli bgl and proU operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></li>
<li>Thus , LeuO , which counteracts the H-NS-mediated repression of several loci , indirectly controls H-NS and RpoS synthesis under speciﬁc conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>This finding implies cooperative repression of the csgD promoter by the two negative factors CpxR and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Thus , the repression of hdeABp by H-NS was promoter specific under these in vitro conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>This corroborates the speciﬁc repression of the rrnB P1 promoter by H-NS found in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Our model predicts that H-NS mediated repression of the dps promoter can be overcome by Es38 and that repression by Fis is reduced as cell density increases and Fis levels decrease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Selective repression of the dps promoter by Fis and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Grainger , D.C. , Goldberg , M.D. , Lee , D.J. , and Busby , S.J.W. ( 2008 ) Selective repression by Fis and H-NS at the Escheri-chia coli dps promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>When the translation of bglG was excluded the H-NS mediated repression was more efﬁcient ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Thus the H-NS mediated repression is enhanced when the bglG mRNA can not be translated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Thus , we propose that the partial alleviation of H-NS repression is mediated by the promoter structure modification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>To answer this question , we ﬁrst examined which individual promoter region is responsible for the H-NS mediated repression under aerobic condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Thus , the cellular DnaK protein levels correlate well with the differences in downstream repression of bgl by H-NS ( Fig. 4c ) , i.e. downstream repression is more effective when DnaKJ is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>These data demonstrate that upstream and downstream repression of bgl by H-NS is regulated independently by LeuO and BglJ , as well as DnaKJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Presently , it is not known whether Lon and the DnaKJ chaperone system affect downstream repression of bgl by H-NS directly or indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>In dnaK mutants , downstream repression of bgl by H-NS is speciﬁcally reduced , whilst BglJ and LeuO have no impact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>However , Lon and DnaKJ speciﬁcally modulate downstream repression by H-NS , but not bgl promoter repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Interestingly , in this dnaKJ : : miniTn10 mutant , downstream repression of bgl by H-NS is less effective , whilst upstream repression by H-NS remains unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>This mutant , which is novel in the context of bgl , was found to speciﬁcally affect downstream repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Here we addressed the role of the downstream silencer in repression of the bgl operon by H-NS and how speciﬁcity of repression of the bgl operon by H-NS is achieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>A downstream silencer enhances the repression of bgl by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Expression of functional Cas proteins therefore requires anti-silencing mechanisms abrogating H-NS repression in order to become effective .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>If the model describing proU induction as due to the potassium-induced relief of H-NS repression is correct , then the absence of potassium glutamate induction of proP is not surprising .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002611" target="_blank">8002611</a>]<br></li>
<li>Therefore , H-NS repression of these four promoters is antagonized by ArcA in the absence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>To achieve repression in the absence of FIS , much higher H-NS concentrations had to be applied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Other regulators , such as H-NS and RcsB , are involved in csgD repression in response to closely related stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>This suggests that relief of repression by H-NS plays a role in translational induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>However , repression by H-NS was greatly decreased by deletions that removed sequences upstream of position 268 and position 261 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>How the activity of H-NS is modulated and how high speciﬁcity of repression by H-NS is achieved are largely open questions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1076">
This result shows that H-NS mediates the csgD repression in high-sucrose medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></label>
<input id="act_similarity_1076", type="checkbox">
<div class="ACT_similar">
<li>( B ) H-NS mediates csgD repression in high-sucrose medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li></div>
<li>Many promoters that are subject to repression by H-NS are also controlled by sequence-specific transcriptional-activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>To the best of our knowledge , this is the first report of simultaneous repression of an array of multiple tandem promoters by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Examples of promoters that are subject to repression by H-NS abound , and new examples are constantly being published .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>When H-NS repression is relieved , GadX levels increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Several mechanisms are also possible to explain how ArcA relieves H-NS repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Either of these mechanisms could account for the relief of H-NS repression by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Finally , in contrast to reports using a different background , repression by H-NS was independent of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>LeuO counteracts the repression of yjjQ by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Further , the H-NS paralogue StpA causes repression of leuO in addition to H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>FIG 3 Speciﬁc H-NS repression of NrdA is not inﬂuenced by StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Tobe , T. , Yoshikawa , M. , Mizuno , T. , and Sasakawa , C. ( 1993 ) Transcriptional control of the invasion regulatory gene virB of Shigella flexneri : activation by virF and repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Transcriptional control of the invasion regulatory gene uirB of Shigella j' exneri : activation by VirF and repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Transcriptional control of the invasion regulatory gene virB of Shigella ﬂexneri : activation by virF and repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Likewise , deletion of the arcA gene prevents Fnr regulation and re-establishes H-NS repression in anaerobiosis , i.e. it mimics aerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Similarly , the complex regulation of the acid stress response gene gadA , which is activated by GadE and RcsB , involves repression by H-NS ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>What is the role of the proposed H-NS repression of a gene or genes that , in turn , regulate RpoS stability ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li>
<li>With the aim of identifying the physiological role of the H-NS repression , we tested the effect of inactivation of the hns gene on the induction of appY by different environmental stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>We predict that , as for other H-NS-repressed genes , H-NS repression is overcome by binding of a positive regulator under spe-ciﬁc environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li>
<li>It has been noted that many of the genes repressed by H-NS require activation signals to overcome repression ( Schroder and Wagner 2002 ; Yu and DiRita 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Given the hierarchy of regulation of the whole gad system , a direct activity of H-NS on the gadX promoter can be envisaged , resulting in repression of the gad structural genes during the exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Here , we demonstrate that H-NS plays a negative role on the regulation of the gad system , identifying gadA and gadX as the target genes of such repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Activation of these genes by DsrA therefore is indirect and is mediated through down regulation of H-NS repression ( Fig. 1 ) ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
<li>LeuO antagonizes H-NS and StpA-dependent repression in Salmonella enterica ompS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1101">
de la Cruz , M.A. , Fernández-Mora , M. , Guadarrama , C. , Flores-Valdez , M.A. , Bustamante , V.H. , Vázquez , A. , and Calva , E. ( 2007 ) LeuO antagonizes the H-NS and StpA-dependent repression in Salmonella enterica ompS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></label>
<input id="act_similarity_1101", type="checkbox">
<div class="ACT_similar">
<li>De la Cruz , M.A. , Fernandez-Mora , M. , Guadarrama , C. , Flores-Valdez , M.A. , Bustamante , V.H. , Vazquez , A. , and Calva , E. ( 2007 ) LeuO antagonizes H-NS and StpA-dependent repression in Salmonella enterica ompS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li></div>
<li>In addition to being controlled by H-NS , StpA , LeuO , and BglJ-RcsB , the bgl promoter is CRP dependent and is repressed by FIS ( 4 , 48 , 52 ) DISCUSSION BglJ , a LuxR-type transcriptional-regulator , and LeuO , a LysR-type transcriptional-regulator , relieve repression of the bgl operon by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>In addition to being controlled by H-NS , StpA , LeuO , and BglJ-RcsB , the bgl promoter is CRP dependent and is repressed by FIS ( 4 , 48 , 52 ) ACKNOWLEDGMENTS This work was supported by a grant from the Deutsche Forschungsgemeinschaft ( Schn371/10 -1 ) and by the Cologne Graduate School of Genetics and Functional Genomics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>For example , H-NS and StpA may be involved in repair of double strand DNA breaks ,79 with the former shown to-bind to Holliday junctions in vitro .19 The possible occurrence of such events especially as part of homologous recombination occurring during rapid exponential-growth , might also contribute to the growth defect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Its activity is enhanced or diminished by other proteins including StpA , Hha and YdgT ( Cnu ) .28 ( a ) StpA is a homologue of H-NS , the two proteins sharing nearly 70 % sequence similarity .29,30 A recent in vitro study has shown that StpA can also form DNA filaments similar to those formed by H-NS and thus silence gene expression .31 Recent super-resolution microscopy showed different patterns of loca-lisation for H-NS and StpA .25 This is in curious contrast to ChIP-chip studies , which demonstrated that the genome-wide binding profile of StpA reflects that of H-NS .32 H-NS and StpA repress each other transcriptionally .29 The absence of H-NS not only abolishes StpA binding to two-thirds of its sites ,32 but also makes StpA subject to rapid degradation by the Lon protease .33 Thus , the expression level of StpA -- which contributes to its activity -- is controlled by a balance between its synthesis and degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Though this appears to suggest a form of molecular backup between two homologous proteins , the backup is unequal : whereas the DstpA mutant hardly affects gene expression , a Dhns mutant leads to upregulation of at least 500 genes as agreed upon by several studies .12,13,15,34 Whereas the absence of stpA does not visibly affect in vivo DNA binding of H-NS on a genomic scale , that of hns abolishes binding of StpA to about two-thirds of its wildtype binding regions .32 Though both H-NS and StpA repress each other 's transcription , leading to the up-regulation of one in the absence of the other , the degree of up-regulation conferred on StpA by the absence of H-NS may not be suficient to compensate for the lack of H-NS .30 This in part might be due to the higher susceptibility of StpA to degradation by Lon in the absence of H-NS .33 A complete backup of H-NS activity by StpA might not be desirable as there are conditions , such as a late stationary phase , in which an attenuated H-NS is favoured .36 The close relative of E. coli , Salmonella enterica Typhimurium , encodes three H-NS-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>There are , however , speciﬁc differences in the properties of the two proteins ; StpA can enhance the splicing of a mutant td intron in vivo or the wild-type intron in vitro more efﬁciently than H-NS , probably due to its greater effectiveness in promoting RNA annealing and strand exchange ( 33 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>H-NS not only represses micF but also negatively regulates stpA , and activates ompF .55 Thus , the function of StpA appears to be masked in a hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Polarity relief conferred by rho or nusG was unaffected in a derivative completely deﬁcient for both H-NS and StpA , although the suppression effects of YdgT or the oligomerizationdefective H-NS variants were abolished in this background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>We therefore propose that it is the interruption of the linear polymeric scaffold of H-NS and StpA , caused by the recruitment of protomers of 64 or L26P into the scaffold , that is speciﬁcally required in cis for suppression of transcriptional polarity relief in the rho and nusG mutants , as depicted sche-matically in Fig. 8B and D , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>This difference in regulation of bgl versus stpA and proU by H-NS-I119T and StpA may further imply that growth conditions which induce stpA may allow the maintenance of strong repression of bgl , even if H-NS is less active under such conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Differential Dependence of StpA on H-NS in Autoregulation of stpA and in Regulation of bgl Tinka Wolf , Wiebke Janzen , Corinna Blum , and Karin Schnetz * Institut für Genetik , Universität zu Köln , Zülpicher Str .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1114">
Differential dependence of StpA on H-NS in auto-regulation of stpA and in regulation of bgl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></label>
<input id="act_similarity_1114", type="checkbox">
<div class="ACT_similar">
<li>Differential dependence of StpA on H-NS in autoregulation of stpA and in regulation of bgl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245262" target="_blank">18245262</a>]<br></li></div>
<li>Strain Genotype Plasmida specific activityb a. Plasmids pDIA547 , pYCStpA , pSUH3 .19 and pDIA523 produce H-NS , StpA , BpH 3 and HvrA respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Plasmids pLGH-NS , pDStpA , pSUH3 .19 and pDIA523 produce H-NS , StpA , BpH3 and HvrA respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Our results suggest that the H-NS and StpA proteins stimulate MalT translation and hence play a positive role in the control of the maltose regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>This result argued for a mechanism in which the H-NS and StpA proteins are not absolutely required components in the activation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Indeed , the binding proﬁle of H-NS and StpA to the leuO locus is very similar suggesting that both proteins bind and repress simulta ¬ A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>High levels of such truncated H-NS proteins repress bgl independently of StpA ( 17 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>This suggests that StpA , in the absence of H-NS , binds only to those target sites to which it has very high binding afinities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>H-NS : 0 , 2.0 , 3.0 , 4.0 , 5.0 and 6.0 mM ; StpA : 0.5 , 1.0 , 2.0 , 4.0 and 6.0 mM ; LRP : 0.25 , 0.5 , 0.75 , 1.0 , 1.25 and 1.5 mM ; FIS : 0 , 0.1 , 0.5 , 1.5 and 2.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>As expected , no inﬂuence of leucine on the binding of H-NS , its homologue StpA or FIS can be detected ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1125">
Hulton , C.S. , Seiraﬁ , A. , Hinton , J.C. , Sidebotham , J.M. , Waddell , L. , Pavitt , G.D. , et al. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></label>
<input id="act_similarity_1125", type="checkbox">
<div class="ACT_similar">
<li>Hulton , C.S. , Seirafi , A. , Hinton , J.C. , Sidebotham , J.M. , Waddell , L. , Pavitt , G.D. , et al. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li></div>
<li>Hulton , C.S.J. , Seirafi , A. , Hinton , J.C.D. , Sidebotham , J.M. , Waddell , L , Pavitt , G.D. , Owen-Hughes , T. , Spassy , A. , Buc , H. , and Higgins , C.F. ( 1990 ) Histone-iike protein HI ( H-NS ) , DNA supercoiiing , and gene expression In bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022275" target="_blank">8022275</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1128">
Hulton , C.S.J. , Seifari , A. , Hinton , J.C.D. , Sidebotham , J.M. , Waddell , l. , Pavitt , G.O. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , C.F. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></label>
<input id="act_similarity_1128", type="checkbox">
<div class="ACT_similar">
<li>Hulton C , Seirafi , A. , Hinton , J. , Sidebotham , J. , Waddell , L. , Pavitt , G. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , C. ( 1990 ) Histone-like protein H1 ( H-NS  , DNA supercoiling and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1710760" target="_blank">1710760</a>]<br></li>
<li>, Seirafi , A. , Hinfon , J.CD , , Sidebotham , J , M , , Waddelt , L , Pavitt , G.D. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , CF. ( 1990 ) Histone-like protein HI ( H-NS ) DNA supercoiling and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Waddel , L , Pavitt , G.D. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , CF. ( 1990 ) Histone-like protein HI ( H-NS ) , DNA supercoiling , and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1625583" target="_blank">1625583</a>]<br></li>
<li>J , M , , Waddell , L , , Pavitt , G.D , , Owen-Hughes , T , , Spassky , A , , Buc , H , , and Higgins , C , H , ( 1990 ) Histone-like protein HI ( H-NS ) DNA supercoiling , and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1357528" target="_blank">1357528</a>]<br></li></div>
<li>It has been shown that alterations , particularly increases , in the level of H-NS protein in the cell have severe effects upon gene expression and the stability of the genome ( Lejeune and Danchin , 1990 ; Barr et aL , 1992 ; Spurio etaL , 1992 ; McGovern etaL , 1994 ) , presumably through the resulting effects on nucleoid structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>In support of this explanation , inactivation of the histone-like protein H-NS , which induces negative DNA supercoiling and binds to curved DNA regions , has been shown to abrogate the requirement of the rpoS gene for expression of curli by E. coli K-12 ( Olsén et al. , 1993 ; Arnqvist et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Despite its relatively pleiotropic effects on gene expression , regulation by H-NS is nevertheless confined to a specific subset of genes ( Bertin et aL , 1990 ; Ueguchi and Mizuno , 1993 ) , probably as a result of the ability of the protein to-bind selectively to regions of bent or curved DNA at high affinity ( Yamada et aL , 1991 ; Tanaka et aL , 1991a ; Owen-Hughes et aL , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1136">
Falconi , M. , McGovern , V. , Gualerzi , C. , Hillyard , D. , and Higgins , N.P , ( 1991 ) Mutations altering chromosomal proteins H-NS induce mini-Mu transpbsition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></label>
<input id="act_similarity_1136", type="checkbox">
<div class="ACT_similar">
<li>Falconi , M. , McGovern , V. , Gualerzi , C , Hillyard , D , , and Higgins , N. P. ( 1991 ) Mutations altering chromosomal protein H-NS induce mini-Mu transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Falconi , M. , McGovern , V. , Gualerzi , C. , Hillyard , D. and Higgins , N.P. : Mutations altering chromosomal protein H-NS induce mini-Mu transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_1139">
However , the strongest effect seems to be the repression of stpA by H-NS ; this causes the level of stpA expression in a wild-type cell to be very low and has led to the suggestion that StpA might act as a `` molecular backup '' of H-NS which is expressed only when the latter protein is absent ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></label>
<input id="act_similarity_1139", type="checkbox">
<div class="ACT_similar">
<li>The strongest effect seems to be the repression of stpA by H-NS ; this causes the level of stpA expression in a wild-type cell to be very low and has led to the suggestion that StpA might act as a ` molecular back-up ' of H-NS that is only expressed when the latter protein is absent ( Zhang et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li></div>
<li>On this view FIS , and possibly also CRP , act to ﬁne tune a fundamental mode of transcriptional regulation , not only by their direct effects on DNA topology but also from the indirect effects of their transcriptional control of other nucleoid-associated proteins , such as HU and H-NS [ 3 , 4 ] , which also act as determinants of chromosomal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine metabolism ) was controlled by protein regulators such as FNR ( transcriptional-regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional-regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>Its expression is also dependent on the global regulatory proteins mediating catabolic repression via cyclic AMP receptor protein ( CRP ) , or anaerobiosis via fumarate nitrate reduction ( FNR ) transcriptional-regulator , and the nucleoid-associated DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Moreover , it is subject to global control by the same regulatory proteins which are known to modulate expression of carnitine metabolism , i.e. , the cyclic AMP ( cAMP ) receptor protein ( CRP ) which mediates the activation of catabolic operons , the transcriptional-regulator FNR that is responsible for anaerobic induction , and the DNA-binding protein H-NS ( 19 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>It is also activated by the leucine responsive protein ( LRP ) and repressed by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>In minimal medium , this promoter is also activated by the leucine-responsive protein and is repressed by the nucle-oid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867437" target="_blank">12867437</a>]<br></li>
<li>The leucine-responsive protein Lrp represses osmCp1 and activates osmCp2 , the nucleoid-associated protein H-NS represses both promoters , and the stationary-phase sigma factor s speciﬁcally recognizes osmCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>The expression of gadX is activated by the alternative sigma factor RpoS during the stationary phase of growth [ 15,19,21 ] , but is repressed during the exponential phase by the nucleoid protein H-NS due to its binding to the gadX promoter or its destabilizing effect on RpoS [ 23-25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21314918" target="_blank">21314918</a>]<br></li>
<li>Finally , the nucleoid associated protein , H-NS , preferentially binds to upstream sequences at the nir promoter and represses promo-ter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>However , H-NS binds to this promoter in vitro only at very high concentrations of the protein and no details of the mechanism of direct activation are available26 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Indeed , in the DrcsB mutant expression of BglJ from plasmid pKETS1 did not cause activation of the leuO promoter lacZ fusion and only background expression levels of 3 units were detected Transcription of leuO is repressed by H-NS and StpA Previous studies have shown that transcription of leuO is repressed by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Moreover , we found by gene fusion and Northern blot hybridization that the `` histonelike '' protein , H-NS , bound specifically to a promoter segment of lrp in vitro , and the level of Irp expression increased in the hns null mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Although we demonstrated that H-NS protein binds with very high afﬁnity to the appY promoter , we can not exclude that the increased appY expression during exponential-growth might be indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>ACC , acetyl-CoA carboxylase ; Amp , ampicillin ; CAI , codon adaptation index ; CAT , chloramphenicol acetyltransferase ; CHL , chloramphenicol ; CHL , chloramphenicol resistance ; H-NS protein , histone-like nucleoid structuring protein ; IPTG , isopropyl thio-R b-D-galactoside ; LB , lysogeny broth ; SPEC , spectinomycin ; SPEC , spectinomycin resistance ; STR , streptomycin ; STR , streptomycin resistance ; tAI , tRNA adaptation R R index ; tmRNA , transfer-messenger RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24966114" target="_blank">24966114</a>]<br></li>
<li>Abbreviations : Ap , ampicillin ; H-NS , histone-like nucleoid-structuring protein ; IHF , integration host factor ; IRL , left inverted repeat ; Km , kanamycin ; Lrp , leucine-responsive regulatory protein ; Sm , spectinomycin ; UPEC , uropathogenic E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207912" target="_blank">16207912</a>]<br></li>
<li>Expression of hlyE is repressed by the global regulator H-NS ( histone-like nucleoid structuring protein ) , but can be activated by the regulator SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1158">
Transcription of the Salmonella typhimurium spv virulence locus is regulated negatively by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></label>
<input id="act_similarity_1158", type="checkbox">
<div class="ACT_similar">
<li>O'Byrne , C.P. , and Dorman , C.J. ( 1994 ) Transcription of the Salmonella typhimurium spv virulence locus is regulated negatively by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li></div>
<li>A functional hlyE gene is also present in non-pathogenic E. coli K-12 , 686 J. K. Lithgow , F. Haider , I. S. Roberts and J. Green but under normal laboratory conditions HlyE is not produced at phenotypically detectable levels due to repression by the nucleoid-associated regulatory protein H-NS ( histone-like nucleoid structuring protein ) ( Ludwig et al. , 1999 ; Westermark et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Transcription of hns , the gene encoding the abundant nucleoid-associated protein H-NS , is also subject to negative autoregulation ( Dersch et aL , 1993 ; Falconi et aL , 1993 ; Ueguchi et aL , 1993 ) but is unaffected by RpoS ( Dersch et aL , 1993 ; our unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Falconi , M. , Higgins , N , P. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1993 ) Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional autorepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Biosci Biow ( ' h Biochem 57 , 1568-1569 83 Jordi BJAM , van der Zeijst BAM , Gaastra W 11994 ) Regions of the CFA/I promoter involved in the activation by the transcriptional-activator CfdA and repression by the histone-like H-NS , Boh ` himie 76 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Regions of the CFA/I promoter involved in the activation by the transcriptional-activator CfaD and repression by the histone-like protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>DNA looping-mediated repression by histone-like protein H-NS : specific requirement of E σ70 as a cofactor for looping Minsang Shin , Miryoung Song , Joon Haeng Rhee , et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1166">
, et al. ( 2005 ) DNA looping-mediated repression by histone-like protein H-NS : speciﬁc requirement of Es70 as a co-factor for looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></label>
<input id="act_similarity_1166", type="checkbox">
<div class="ACT_similar">
<li>et al. ( 2005 ) DNA looping-mediated repression by histone-like protein H-NS : speciﬁc requirement of Es70 as a cofactor for looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li></div>
<li>DNA loopingmediated repression by histone-like protein H-NS : Specific requirement of E. 70 as a cofactor for looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17504942" target="_blank">17504942</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1169">
DNA looping-mediated repression by histonelike protein H-NS : speciﬁc requirement of E 70 as a cofactor for looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></label>
<input id="act_similarity_1169", type="checkbox">
<div class="ACT_similar">
<li>DNA looping-mediated repression by histonelike protein H-NS : speciﬁc requirement of Esigma70 as a cofactor for looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li></div>
<li>Mode of repression by H-NS We investigated the mechanism of repression by H-NS by determining whether a direct protein -- protein interaction between H-NS and E 70 or an indirect effect such as a change in DNA topology , i.e. , bending , looping , or supercoiling , was critical for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>If H-NS concentrations were suficient to drive high protein occupation of the repression loop , DNA stiffening might antagonize looping , but this seems unlikely , as exponentially growing cells are thought to contain only about one H-NS dimer per 1400 bp of DNA ( 21,70 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>We showed that their expression is under the negative control of H-NS and , except for yad , subjected to cAMP receptor protein-mediated activation and carbon catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>We showed that these surface ﬁmbriae are all under the negative control of the H-NS repressor therefore explaining their cryptic character and are subjected to cAMP receptor protein ( CRP ) - mediated catabolite repression ( except for yad ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>In our case , this cryptic character was found to be due to the repression exerted by the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>In contrast to acrAB , the expression level of acrEF is very low because of global repression by a histonelike protein , H-NS ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>However , csgD repression in response to high sucrose content is not mediated by CpxR but by the global regulatory protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Without translation of bglG the H-NS mediated repression was ~ sevenfold , while translation of bglG reduced the repression to ~ threefold , suggesting that the access of a protein to the mRNA or the secondary structure of the mRNA is important in this process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>In exponentially growing cells , the nucleoid-binding protein H-NS downregulates Hsp31 synthesis , and hchA thermal induction primarily relies on the relief of H-NS-mediated silencing of EsD-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Intriguingly , Rho-dependent transcription termination is also implicated in promotion of xenogeneic silencing ( 11 ) , and our results suggest that downregulation of H-NS activity , for example , by a protein such as H-NST , may concomitantly lead to an increased efﬁciency of Rho-dependent termination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>When the CaiF protein was overproduced , the pattern of induction of carnitine dehydratase activity was shown to escape from the control of the histone-like protein H-NS and S the alternate sigma transcription factor s and to rely primarily on the stimulatory effect of CaiF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>H-NS is known to be one of the most abundant E. a ) ii histone-like proteins ( approximately 20000 copies per cell ) , it is difficult to envisage , therefore , how a specific regulatory function like the repression of rRNA transcription , which has to be balanced in response to the cell 's demands , can be attributed to this protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Instead , the histone-like nucleoid-associated protein ( H-NS ) mediates the Fur dependence of ftnA through a Fur-antagonized repression mechanism : H-NS was found to directly repress ftnA transcription in vitro and this repression was reversed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>H-NS Functions Directly as Transcriptional Repressor of gadA and gadX -- To determine whether H-NS is directly responsible for gad gene repression , the effect of this histone-like protein on transcription was investigated in a purified in vitro system programmed with pKK232-8 derivatives in which the gadA , gadBC , and gadX DNA fragments analyzed by gel retardation and DNase I footprinting were fused to the promoterless cat gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The level of repression in early-log phase was lower than in mid-log phase cultures , probably because of the presence of high levels of Fis protein , which counteracts the H-NS inhibition by stimulating / lnstranscription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Auto-repression during early log is partly alleviated by th high levels of Fis protein present in these cells ( Finkel and Johnson , 1992 -RCB- , since this protein competes with H-NS for the same binding sites and stimulates hns transcription counteracting the inhibitory effect of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>This fact , together with our ﬁnding that both htrA transcription and HtrA levels increase in an hha hns genetic background , being the effect specially relevant when cells grow in salt-free LB medium , shows that Hha and H-NS protein exhibit a relevant role in htrA osmoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Because H-NS represses its own transcription ( 6 ) , a decrease in H-NS protein should lead to increased hns mRNA levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10954740" target="_blank">10954740</a>]<br></li>
<li>Thus the H-NS protein appears to selectively inhibit csgA transcription by Ea70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7708499" target="_blank">7708499</a>]<br></li>
<li>Our data are in agreement witah direct repression model for proU transcription modulation by the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>The H-NS protein functions to inhibit proU transcription under low-osmolarity conditions ; the repression is relieved as the potassium glutamate levels increase to 0.3 M in vitro ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Although the assay conditions were slightly different , the 45 kDa phosphoprotein we observe may correspond to the protein no longer phosphorylated in vitro in ihe presence of the histone-iike proteins HU and H-NS ( S. T. Sweeney , P , H. Williams and V. Norris , submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783627" target="_blank">7783627</a>]<br></li>
<li>In these cases , the H-NS protein represses transcription speciﬁcally at low temperature and seems to act by antagonizing the transcriptional-activators CfaD , cyclic AMP ( cAMP ) / CRP , and possibly VirF ( 16 , 22 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>In addition to the strict control exerted by H-NS , the clyA locus appeared to be controlled by the global regulatory protein CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>In solution , H-NS acts as a homodimer and can form higher-order oligomers at increased protein concentrations ( 12 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>In this work anti-silencing was demonstrated through the induction of the dominant negative variant G113D H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>In summary , we conclude that the csiD promoter has an intrinsic preference for EsS , but that also protein factors such as cAMP-CRP , Lrp and probably H-NS as well as DNA conformation contribute to its strong EsS selectivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In the presence of the histone-like protein H-NS , E 70 poorly recognizes this promoter and provides a low , basal level of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>In preliminary experiments , however , it has been observed that the preference for EsS appears to be restored in the presence of the histone-like protein H-NS , which seems to strongly interfere with Es70/cAMP-CRP-mediated transcription of csiD ( A. Kolb , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>However , even with supercoiled csiD templates , EsS preference , especially in the presence of cAMP-CRP , was restored in the presence of the histone-like protein H-NS ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We postulate that the group of genes controlled jointly by H-NS and Lrp serves to modulate the supply of methylene-THE We believe that manipulation of the one carbon pool can regulate the rate of protein synthesis by managing the concentration of N-formyl methioninyl tRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The interplay between Lrp and other NAPs , in particular H-NS , in the regulation of several genes is well documented ( 10 , 25 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The global regulatory protein H-NS was shown to be involved in csg genes regulation , and it regulates other genes in response to osmolarity changes ( 2 , 22 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The gadA/BC genes , activated in response to acid stress and in stationary phase cells , are subjected to complex circuits of regulation involving 70 , S , cAMP receptor protein , H-NS , EvgAS , TorRS , GadE , GadX , GadW , and YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The H-NS protein negatively regulates its own gene , and this may help account for the pattern of growth-phase-dependent regulation ( 10 , 15 ) , as may the contribution of the positive regulator FIS ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The transcription of csgA is positively regulated by the RpoS sigma factor and repressed by the histone-like protein H-NS , but the temperature and osmolarity regulation of curli expression involves other factors ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11466275" target="_blank">11466275</a>]<br></li>
<li>The H-NS and StpA proteins negatively autoregulate the transcription of their own genes and each represses the gene that encodes the other protein in trans 6,10 -- 15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Im ( 1984 ) of a characteristic increase in the level of H-NS in late stationary phase , the intracellular level of this protein was observed to be constant throughout growth of the has + strains in the present conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>For this purpose , rRNA synthesis rates were determined using the pulse-labelling technique after rapid reduction in the growth temperature or increase in osmolarity in cells that can or can not express functional H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>These results confirm the theory that H-NS negatively regulates its own gene in order to match protein production to DNA synthesis , and they suggest that it is important for cells to maintain a constant ratio of H-NS protein to DNA during logarithmic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Falconi et aL ( 1993 ) have suggested that autorepression by H-NS may be less efficient in late log and stationary phase ; this might lead to a slight increase in H-NS protein levels as cells enter stationary phase but could not account for active hns transcription when DNA synthesis ceases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Since both the hns-lacZprotein and operon fusion strains show the same pattern of hns expression , the data presented in Fig. 2 and Fig. 3 strongly suggest that synthesis of the H-NS protei is regulated in response to growth-phase at the level of hns transcription , Nuc/eoid-associated ONA-binding protein H-NS 877 A EcuRI + CAATTCCTTACATTCCTCGCT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Thus , at lowered H-NS levels as exhibited in an hns-1 background , there would still be enough protein to repress ﬁmB expression because of the tight binding between H-NS and ﬁmB DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Growth-phase regulation of hns mRNA and H-NS protein levels A prediction of our model for autoregulatory coupling of H-NS synthesis to DNA synthesis is that hns expression should be repressed in stationary phase when DNA synthesis ceases but the protein remains present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>In contrast , induction of MG1655 / pHOP11 , which causes enhanced levels of the wild-type H-NS protein , results in almost a complete CRISPR transcript shut-off ( Fig. 6A , lanes 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>DsrA decreases H-NS protein levels , whereas it increases RpoS levels , with a corresponding decrease in hns mRNA stability in vivo ( .8 - fold ) and an increase in stability of rpoS mRNA , approximately 3-fold ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10954740" target="_blank">10954740</a>]<br></li>
<li>Because H-NS levels predominated ( Fig. 5A , compare lanes 5 and 6 ) and the overall level of protein ( StpA1H-NS ) decreased when DsrA was overproduced ( Fig. 5A , compare lanes 3 and 4 ) , a decrease in H-NS protein was suggested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>H-NS protein levels decreased in response to DsrA overproduction , with a concomitant increase in StpA protein levels ( Fig. 5B , compare M182ypA with M182ypDsrA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The second peculiarity of these H-NS footprints is a kind of all-or-none response , which suggests that the protein binds co-operatively ; in fact , in all the experiments carried out , little effect of H-NS is seen until a critical concentration of protein gives rise to extended effects ( e.g. Fig. 6B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>The HU protein binds the rpoS leader region and stimulates translation ( 2 ) , while H-NS inhib-its RpoS induction ( 3 ) in a manner antagonized by DsrA ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142416" target="_blank">12142416</a>]<br></li>
<li>H-NS also interacts with Hfq ( or HF-1 ) 93 , a protein that binds to RNA and modulates the translation of rpoS mRNA , although the details of this interaction are unknown94 ,95 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>35 , No. 18 doi :10.1093 / nar/gkm712 Published online 18 September 2007 High-affinity DNA binding sites for H-NS provide a molecular basis for selective silencing within proteobacterial genomes Benjamin Lang , Nicolas Blot , Emeline Bouffartigues , Malcolm Buckle , 1 2 3 3 Marcel Geertz , Claudio O. Gualerzi , Ramesh Mavathur , Georgi Muskhelishvili , 2 4 2 2 Cynthia L. Pon , Sylvie Rimsky , Stefano Stella , M. Madan Babu * and Andrew Travers 4 3 4 1 , 1 1 2 Received June 11 , 2007 ; Revised August 22 , 2007 ; Accepted August 24 , 2007 ABSTRACT The global transcriptional-regulator H-NS selectively silences bacterial genes associated with pathogenicity and responses to environmental insults .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>On the other hand , the hns mutant effect on proU P1-lac expression from plasmids pHYD394 and pHYD395 ( which carry more than 1 kb of DNA downstream of P1 including the NRE and inactivated P2 ) is around 50-fold , which is very much more pronounced than can be accounted for by the upstream effect on s synthesis alone ; it is likely , therefore , that in this case H-NS is also acting locally , perhaps by binding to the high-afﬁnity binding sites at the NRE and around P1 ( 30 , 38 , 50 ) , to repress reporter gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>We detected a46 - netic recombination ( 22 , 31-33 ) , affect bacterial virulence ( 9 , base pair high affinity H-NS binding region downstrea25m ) , decrease themotility of the bacterialcell ( 34,35 ) , increase of the proU promoter at t5h ' eend of the proV gene andthe formationof chromosomal deletions ( 361 , and stimulate the a complex array of additional H-NS bindinsg ites which transposition of bacteriophage Mu ( 37,381 , Certain hns alleles suggest the presence of an extended H-NS nucleoproatlesioncause alterationsin the DNA supercoiling of reporter plascomplex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Finally , in addition to the observation that H-NS can constrain supercoiis , other lines of evidence show that H-NS causes distortion of the DNA upon binding : i ) H-NS enhances cleavage of DNA by DNase I in footprinting reactions [ 40 , 48 ] : ii ) - H-NS strongly compacts DNA in vin ' o and in vivo [ 36 , 50 ] ; and iii ) H-NS increases the thermal stability of DNA slightly , although previous reports 15 , 61 have overestimated this due to a strong hypochromic effect , presumably due to compacting or ` clumping ' of the DNA [ 201 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>H-NS exhibits preferential binding to the 59 end of proV that comprises the NRE ( 30 , 42 , 54 ) , and there is genetic evidence that such binding is required for the inhibition by H-NS of proU transcription at low osmolarity ( 8 , 13 , 30 , 42 ; reviewed in reference 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1227">
Nagarajavel , V. , Madhusudan , S. , Dole , S. , Rahmouni , A.R. , and Schnetz , K. ( 2007 ) Repression by binding of H-NS within the transcription unit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></label>
<input id="act_similarity_1227", type="checkbox">
<div class="ACT_similar">
<li>Nagarajavel , V. , Madhusudan , S. , Dole , S. , Rahmouni , A.R. and Schnetz , K. ( 2007 ) Repression by binding of H-NS within the transcription unit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li></div>
<li>Transcription of hdeAB depends on complex circuits of regulation involving RpoD ( 70 ) , RpoS ( 38 ) , H-NS , cyclic AMP receptor protein , GadE , GadX , GadW , GadY , EvgAS , YdeO , Lrp , MarA , SoxRS , TorRS , and TrmE , among other regulators ( see Fig. 1 and references in the legend ; see Fig. 2 for the DNA binding sites of some of these regulators in the hdeAB promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Binding of H-NS to the dps promoter and regulation by H-NS in vitro DNase I footprinting was used to investigate the binding of puriﬁed H-NS to the E. coli K-12 dps promoter ( Fig. 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>H-NS and Fur compete for the same binding region on ftnA promoter To further elucidate the mechanism of ftnA regulation by Fur and H-NS , and to determine whether these two DNA-binding proteins do indeed compete for the same site , H-NS and Fur were added in combination to footprinting reactions ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Such extended protection is commonly observed when H-NS binds to speci ® c sites with high a nity and is probably due to co-operative binding of multiple H-NS molecules ( Atlung and Ingmer 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>These results indicate that the two nucleoid proteins , IHF and H-NS , compete with each other for binding to the respective target sequences , and in the presence of high concentrations of both proteins , only the H-NS complexes are formed because of the higher affinity of H-NS than IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Therefore , the displacement of one of the Fis compiexes at very high H-NS concentrations is probably the result of a conformational change owing to extensive H-NS binding , rather than a direct steric block caused by overlapping radii of the bound proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>To investigate the relationship between H-NS and SlyA , competitive EMSA and DNase I footprinting analysis was performed in the presence of both proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>The presence of both proteins induced a novel digest pattern at the SlyA-II site , although the two DNase I-hypersensitive sites characteristic of H-NS binding were retained , confirming the continued binding of H-NS at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Therefore , in the presence of a high level of H-NS , addition of a low level of SlyA did not affect the binding of H-NS at PhlyE , but when the level of SlyA was increased , a unique pattern of protection was obtained which is distinct from , yet shares some features of , the footprints made by H-NS or SlyA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Both CpxR and H-NS alone formed multiple CD6 complexes , in agreement with the presence of multiple binding sites for both factors ( Fig. 8a , lanes 2 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Two DNase-hypersensitive sites characteristic of H-NS binding in this region persisted in the presence of SlyA , accompanied by the appearance of several hypersensitive sites ( near positions 15 , 78 , and 138 ) unique to the presence of both H-NS and SlyA ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>This suggests the presence of a LeuO binding site in between positions 347 and 147 and also binding of H-NS within that region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Our interpretation of these data is that in the presence of both H-NS and SlyA , H-NS rather than SlyA is bound at the SlyA-II site and that this binding is affected by the binding of SlyA at the SlyA-I site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>In vivo function of an H-NS binding motif In the absence of other data , the binding of H-NS to a site in vivo , or even the presence of a site , is not necessarily indicative of a biologically relevant role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>In lanes 5 -- 9 and 5 ′ -- 9 ′ the binding of Es70 in presence of the indicated amounts of H-NS was analysed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>In lanes 8 -- 11 binding of wild-type H-NS ( 2 μM ) was analysed in the presence of no ( lane 8 ) or 1 , 2 or 4 μM G113D ( lanes 9 -- 11 respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>These results demonstrate that more CnuK9E is required to antagonize DicA binding in the absence of H-NS and imply that the CnuK9E-H-NS complex antagonizes DicA binding to Oc more effectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>These results were recently partially con cents rmed since binding of H-NS to the region upstream of cents mB was demonstrated by non-competitive gel retardation [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>However , this problem is minimized when comparing the binding afﬁnities of two proteins , such as wild-type and T108I H-NS , under the exact same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>However , in contrast to the situation with Lrp-dependent methylation protection , it is not known if methyl-ation of GATC-I and GATC-II affects H-NS binding and thermoregulation of pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>In addition , binding competition experiments with increasing H-NS and Fis concentrations were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In addition , we demonstrate that H-NS binding exerts a specific biological function , namely to antagonize Fis activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In contrast , SlyA did not prevent RNAP binding at PhlyE , but did inhibit H-NS binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In particular , H-NS directly or indirectly increases positional stringency for the CRP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>H-NS binding , therefore , which is obviously very sensitive to changes in salt concentrations , may simply be inactivated by an increase in intracellutar K * io concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Indeed , at low SlyA concentrations , the binding of both H-NS and SlyA to the promoter was enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Cooperative binding of the two repressors CpxR and H-NS to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>( b ) Model of cooperation between CpxR and H-NS in binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>This binding is the mode of operation of H-NS in regard to rrnB P1 inhibition ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>H-NS control is the result of H-NS binding to an under fully anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>is the H-NS repressor function dependent on modification or binding of cofactors ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>SlyA inhibited the binding of H-NS to the SlyA-I site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Obviously , binding of H-NS and Fis can occur simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Competition of Fis binding at increasing H-NS concentralions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>[ 86 ] H. Afﬂerbach , O. Schröder , R. Wagner , Conformational changes of the upstream DNA mediated by H-NS and FIS regulate E. colirrnBP1 promoter activity , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12504278" target="_blank">12504278</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1264">
Afﬂerbach , H. , Schröder , O. , and Wagner , R. ( 1999 ) Conformational changes of the upstream DNA mediated by H-NS and FIS regulate E. coli rrnB P1 promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></label>
<input id="act_similarity_1264", type="checkbox">
<div class="ACT_similar">
<li>Conformational changes of the upstream DNA mediated by H-NS and FIS regulate E. coli rrnB P1 promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10715005" target="_blank">10715005</a>]<br></li></div>
<li>H-NS binds to the hchA promoter region which exhibits characteristics of curved DNA To determine whether H-NS directly or indirectly regulates Hsp31 expression , we ﬁrst tested its ability to-bind to the hchA promoter using a gel mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>The experimental data reported here are consistent with gene silencing models in which the curved DNA to which H-NS initially binds is located upstream or downstream of the promoter that is to be repressed ( Rimsky , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>H-NS binds to no specific target sequence in DNA but instead recognizes intrinsic DNA curva ¬ 3 The abbreviations used are : PR1 , region 1 promoter ; X-gal , 5-bromo-4-chloro-3-indolyl - - D-galactopyranoside ; EMSA , electrophoretic mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Specifically , we have addressed the following items : ( i ) location , number and arrangement of H-NS and Fis binding domains ; ( ii ) involvement of DNA conformation in stable interactions ; ( ili ) results of competition studies for H-NS and Fis ; ( iv ) differences in cellular concentrations or binding activities of the proteins ( see betow ) ; ( v ) the mechanism of Fis-dependent activation via enhanced promoter affinity ; and ( vi ) the sub-optimal spacing of the - 1 0 / - 3 5 regions , which makes promoters especially prone to modulation of their efficiency by oonformational or torsional effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>All 12 DNA-binding proteins , CbpA , CbpB ( or Rob ) , DnaA , Dps , Fis , Hfq , H-NS , HU , IciA , IHF , Lrp and StpA , were over-expressed using pCU60 , pMK19 , pSY567 , pDPS1 , pRJ1077 , pHFQ607 , pHOP11 , pLhupAhupB , pICS1 , pSA5hiphimA , pMWD1 and pT7stpA , respectively , and puri ® ed to apparent homogeneity as previously described ( Talukder & Ishihama 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>These proteins , among which HU , H-NS , FIS ( factor of inversion stimulation ) and IHF ( integration tiosf factor ) are the best characterized , are considered to be histone-like as they are small , relatively abundant and able to bend or wrap the DNA ( reviewed by Schmid , 1990 ) , HU and H-NS bind to DNA non-speoifically , although for Received 21 October , 1994 ; revised 5 December , 1994 : accepted 7 December , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Some sequences produce an intrinsic bend in the DNA or bind proteins such as Fis , H-NS , HU , or IHF that bend the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Some of these proteins bind nonspeci ® cally to DNA , e.g. HU and H-NS , while others are site-speci ® c , e.g. FIS , integration host factor ( IHF ) , and Lrp ( Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>In contrast to HU and H-NS , FIS and IHF are site-specific binding proteins which , upon interaction with their respective recognition sites , induce a significant bend in the DNA with an estimated angle of 90 for Fis and 140 '' for IHF ( Thompson and Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>We show that the nir promoter is repressed by three DNA binding proteins , Fis , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Other global regulators , such as IHF , H-NS , and FIS , are histone-like DNA-binding proteins with a variety of functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>In the ﬁrst model , wild-type levels of H-NS could bind to DNA sequences within the IE , to sequences on either side of the IE , or to 9-bp inverted repeats ﬂanking the IE to hinder ﬂipping or preclude the binding of activators such as integration host factor ( 11 , 14 ) and Lrp ( 4 , 19 ) or the recombinases ( FimB and FimE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In this study , we show that H-NS regulates FimB levels in the cell , but this regulation is not the sole cause for increased DNA ﬂipping observed in hns mutant allele backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In particular , H-NS cells showed strongly enhanced DNA looping but had a level of negative DNA supercoiling that was similar to WT cells ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>It is also important to be aware of the fact that , at high enough concentrations , H-NS will bind to any promoter and inhibit the interaction of RNA polymerase : given the unusual DNA-binding properties of H-NS it is not a trivial problem to perform in vitro studies under conditions which can confidently be considered to reflect in vivo specificity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>By binding to the UR , H-NS could bend the DNA in such a way as to change the 210 and 235 spacing or disrupt the RNA polymerase-DNA promoter interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In the absence of H-NS , NhaR can act as a repressor ( in the absence of Na1 ) , suggesting that it is able to-bind to DNA in the vicinity of the nhaA promoter but that its binding does not activate RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>In preliminary experiments we have tested the binding capacity of H-NS to target DNA after treatment of the protein with phosphatase to remove potential phosphate groups of functional importance and found that the binding capacity of H-NS to the rRNA PI promoter fragment and its in vitro repressor activity is completely lost , while the binding of Fis , treated in the same way , is only slightly affected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>When in vitro transcription experiments were carried out to test this hypothesis , it was found that indeed H-NS inhibits and Fis stimulates hns transcription , while a mixture of both DNA-binding proteins restores a nearly normal transcriptional level ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>We then analysed the existence of Pcas transcripts by primer Evidence that silencing of the Pcas promoter by H-NS is direct To verify that the observed effect of H-NS on transcription of the Pcas promoter in vivo is direct and not the result of other regulators , which are under the control of H-NS and to learn more about the mechanism of this regulation we conducted a detailed binding analysis of H-NS to the IGLB DNA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>In the ﬁrst , H-NS takes an active role by directly binding to DNA and inhibiting transcription ( 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Both , H-NS and LRP , act as repressors for in vitro transcription from rRNA promoters In order to determine the functional effect of LRP binding to the rRNA promoter DNA , we conducted in vitro transcription experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Conversely , H-NS transcription is enhanced coordinately with DNA synthesis and is induced by the cold-shock positive regulator CspA and the FIS protein ( reviewed in ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The H-NS protein also increases the thermal stability of DNA and inhibits transcription In vitro of either phage X - or E. coli DNA ( Gualerzi ef ai , 1986 ; K. Friedrich ef ai , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>The Fis-polymerase interactions are disrupted and transcription is no longer activated by Fis , In ( c ) , high occupation of H-NS sites completely deforms the DNA structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>The association of Fis , IHF and H-NS suggests that nir promoter DNA is sequestrated into a highly ordered nucleo-protein structure that represses FNR-dependent transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>These results demonstrate that both Cnu and H-NS are required to make the host cells resistant to Sm , presumably because the Cnu-H-NS complex binds to the operator DNA ( Ori14 ) , which , in turn , represses transcription of the rpsL gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>Moreover , results obtained in vitro and in vivo demonstrate that H-NS binds to the CRISPR leader DNA and affects crRNA transcription , suggesting a functional role of H-NS in the regulation of the CRISPR system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Thus , H-NS fails to repress E 38-driven transcription initiation because the DNA arms leaving E 38 · hdeABp are not close enough for the upstream bound H-NS to laterally oligo-merize to the downstream DNA -- no DNA looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Apparentiy , H-NS and Fis bind to separated curved DNA segments , with the planes of the curves pointing into different directions , / n vitro transcriptionai analyses demonstrate that H-NS represses rRNA P1 promoterdirected transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>H-NS and YncC can bind DNA simultaneously to regulate in vitro transcription at pyciGSTM ( Figs. 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Whether or not NhaR can bind to DNA in the absence of Na1 in an hns1 strain is unknown , as transcription is already repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Indeed , it is not known wLether or not H-NS remains bound to the DNA when transcription is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Under aerobic condition , transcription of gadE was repressed by the global DNA silencer H-NS in M9 minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>It is also possible that HF-I may exist in two forms in the dell : one form would be involved in translation and would bind to RNA , and another would be compiexed with H-NS and would bind to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984093" target="_blank">7984093</a>]<br></li>
<li>It is not unreasonable to assume that H-NS in the nucleoid is bound to DNA , although as H-NS does bind to RNA ( albeit with lower affinity than to DNA ; [ 22 ] ) , the possibility that some H-NS is associated with RNA can not be e × cluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>b-Galactosidase activity ( Miller units ) b-Galactosidase activity ( Miller units osmCp1 activation by NhaR is independent of H-NS and σs The sequence required for activation by NhaR is immediately adjacent to the osmC w35 region p1 To identify the sequence required for the stimulation of osmC by NhaR , we made use of a set of osmC -- lac p '' p '' fusion strains in which osmC was carried on various p '' DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></li>
<li>) The presence of the two dimerization interfaces allows H-NS to assemble ( either alone or with its paralogous partner StpA [ 53 , 55 , 60 , 65 , 67 ] ) into a helical polymeric scaffold around which DNA is bound ( 3 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>The observations reported here suggest that lowering the GC content of the spacer could increase DNA curvature and H-NS affinity in this region , increasing the stability of the DNA : H-NS complex , and preventing derepression by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The DNA was incubated with increasing amounts of hexahistidine-tagged H-NS or BSA as a control , and mixtures were fractionated on a polyacrylamide gel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>The reverse situation , addition of increasing amounts of H-NS to pre-bound LRP -- DNA , again results in formation of complexes with lower electrophoretic mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Map position Gene recA nusA injB pnp aceE aceF Gene product 58 ' 69 ' 69 ' 69 ' 3 ' 3 ' RecA NusA IF2 ( Iz + ~ ) Polynucleotide phosphorylase Pyruvate dehydrogenase ( lipomide ) Pyruvate dehydrogenase ( dihydrolipoamide acetyltransferase ) CS7 .4 H-NS DNA gyrase ( subuni !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748932" target="_blank">7748932</a>]<br></li>
<li>HU , which shares with H-NS the property of introducing , in vitro , negative supercoiling in relaxed DNA in the presence of topoisomerase I ( 3 , 31 ) has clearly opposite effects on OmpF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>We observed no such hairpin-like configuration with H-NS alone or with the hdeABp DNA lacking the sequence upstream of − 77 in the presence of H-NS and E 70 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>B. Binding of RNA polymerase in the presence of H-NS to the Pcas and anti-Pcas promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>It is suggested that H-NS occupation of the ftnA promoter depends upon the presence of H-NS bound at the upstream sites through an H-NS oligomerization and H-NS -- DNA looping mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>These results suggest that the presence of H-NS primarily acts to destabilize DNA looping through its effect on DNA bending , not twisting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Base-change positions of the linker mutant DNA fragments are indicated by numbersThe presence or absence of H-NS in gel retardation assays is indicated by a + or - , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Template DNA was incubated with either EsS or EsD in the absence ( -- ) or presence ( + ) of 528 ng of His6-H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>AFM images of E 70 bound to hdeABp DNA in the presence or absence of H-NS convincingly suggested such a mechanism of DNA looping ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Figure 6 shows hdeABp DNA bound by RNP , in the presence or absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Atomic force microscopy images of representative RNP ( 20 nM ) bound to hdeABp DNA ( − 216 to +580 ) in the presence or absence of H-NS ( 228 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>These images show thickening of the DNA arms by H-NS binding , which cross-bridged DNA in the presence of E 70 ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>E 70 and E 38 could each bind to hdeABp DNA in the presence of H-NS ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>In the presence of 1 M H-NS , retardation of some of the DNA target was observed as a smear in the gel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>B , C. Gel retardation of H-NS and Fis DNA complexes in the presence of distamycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Protein -- DNA complexes were formed in presence of H-NS and RNA polymerase at the concentrations given above the lanes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>However , when the 400 bp probe CD6 , including both the IHF-1 and the IHF-2 sites , was used , IHF -- H-NS -- DNA ternary complexes were detected ( data not shown ) , implying that IHF was not ejected even in the presence of high concentrations of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Furthermore , unlike the case of NS ( HU ) , which was found to form a larger amount of large cross-linked products in the presence of DNA ( Losso e ( al. , 1986 ) , the cross-linking reaction with H-NS was found to be completely unaffected by the presence of DNA ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>H-NS and Fis were predicted to exhibit more complex activity ( DNA-binding ) profiles than the other global regulators , with H-NS activity being the lowest in the mid-aerobiosis range and Fis activity being greatly increased under conditions of excess oxygen supply ( aerobiosis 115 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>A key aspect of the biological role of these proteins is whether they bind DNA themselves , or if their modulatory activity is absolutely dependent upon interaction with H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>These results demonstrated that H-NST108I is defective in its ability to-bind relevant ﬁmB DNA and corro-borates evidence that the H-NS C terminus is essential for this function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>This function of H-NS is dependent on its ability to-bind DNA and to oligomerize , leading to the bridging together of otherwise distant DNA segments on the chromosome ( reviewed in references 20 , 23 , 39 , and 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Lanes 1 and 8 , DNA only ; lanes 2 and 9 , 1.3 pM H-NS ; lanes 3 and 10 , 13 pM H-NS ; lanes 4 and 11 , 39 pM H-NS ; lanes 5 and 12 , 65 pM H-NS ; lanes 6 and 13 , 91 pM H-NS ; lanes 7 and 14 , 130 pM H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Biophysical data show that the interaction of purified H-NS with DNA responds to physical and chemical signals in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Increases in temperature or the addition of osmolytes , such as NaCl or KCl , inhibit the interaction of H-NS 55 with purified bacteriophage λ DNA , and increased temperature inhibits the in vitro interaction between H-NS and a curved DNA sequence from the proU promoter56 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>It should be noted that H-NS is not itself a topoisomerase , but simply constrains supercoils when it binds to DNA ; this results in a topological change in the DNA which can be recognised by topoisomerases which can then introduce an appropriate compensatory change in linking number [ 8 , 17 , 29 , 44 , 541 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>H-NS and StpA bind to similar target sites ; therefore the above-described semi-quantitative and relative measure of afinity of different DNA sites to H-NS can be readily applied to StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Binding appeared to be cooperative , as suggested by Rimsky and Spassky ( 46 ) , whereby a small change in H-NS concentration led to a drastic difference in DNA mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Although there is ample evidence that H-NS binds preferentially to DNA containing curved regions , we show here that a major basis for this selectivity is the presence of a conserved sequence motif in H-NS target transcriptons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>StpA binds to AT-rich , planar , bent DNA and constrains supercoils , as does its homologue H-NS , but with a higher afﬁnity ( 24 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>It has been known for some time that H-NS binds to DNA in a sequence-nonspecific manner , but it preferentially binds with curved AT-rich DNA ( 45 -- 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>We tested whether H-NS binds to this bent DNA sequence of lrp by DNA mobility shift experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>H-NS binds preferentially to curved DNA and , in fact , gel mobility shift of lrp DNA by H-NS was eliminated by a bent DNA competitor ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>H-NS binds with high affinity but low DNA interactions ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Binding of H-NS to DNA H-NS binds DNA in a relatively non-specific fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Fourth , when bound to preferred sites at which it exerts a biological effect , regions of DNA protected by H-NS in footprinting studies are extended and diffuse [ i 5 , 40 , 48 , 49 [ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Such changes could be mediated , at least in part , by changes in the interaction of H-NS with cellular DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>This implies that several molecules of H-NS are bound to the site , that considerable conformational changes have been introduced into the DNA ( such as if the DNA was wrapped around H-NS molecules ) , or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>First , the number of H-NS molecules which can bind to any DNA molecule depends on the H-NS-DNA ratio , increasing as the H-NS-DNA ratio increases until the DNA molecule is covered from end-to-end in a ` saturated ' complex [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Supercoiled plasmid DNA ( 0.15 pM ) was added to increasing amounts of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Atomic-force microscopic analysis of covalently closed circular plasmid DNA bound by H-NS supports the proposal that H-NS can crosslink DNA in this way79 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>A single shifted band of H-NS bound to DNA was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Fifth , when bound to DNA H-NS can constrain supercoils [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Next , we determined whether H-NS and RNPs could bind simultaneously to hdeABp DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>H-NS may bind to and alter the conformation of DNA without changing its superhelicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>H-NS and LRP bind DNA synergistically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>This binding prefe H-NS binds preferentially to curved DNA At high enough concentrations , H-NS will bind to any double-stranded DNA molecule tested so far , irrespect-ive of its sequence , and can cover the DNA molecule from end-to-end 110 , 401 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Remarkably , DNA looping is strongly enhanced in the absence of H-NS , and an explanatory model is proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Other ordinary H-NS-binding DNA sequences may lead to the binding of H-NS , but in the absence of an H-NS nucleation site , the bound H-NS is not necessarily well organized and hence is not transcriptionally repressive ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>Therefore , 44 Hha-DNA interactions were lost in the absence of H-NS ( Table S3 ) , showing that H-NS confers some DNA binding specificity to Hha by forming heteromeric complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>H-NS thus appears to function as a global DNA-binding-transcriptional-regulator ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8244952" target="_blank">8244952</a>]<br></li>
<li>H-NS may selectively function as a transcriptional-regulator through its ability to recognize changes in DNA curvature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>The arftificial nuclease 1,10-phenanthroline-copper complex , ( ( OP ) 2Cu * '' ) , whose cleavage pattern is sensitive to DNA conformation in a predictable manner ( Sigman , 1990 ; Sigman etai , 1991 ) , was further used to assay for variations in the minor-groove geometry as a consequence of H-NS binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>The decrease in intensity of ﬂuorescence between free and bound DNA ( compare lanes 4 and 5 ) was probably a consequence of H-NS binding and obscuring ethidium bromide intercalating sites on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>The antagonizing effect of CnuK9E with H-NS on DNA-binding activity of DicA was stronger at 37uC , presumably due to the lower operator binding of DicA at 37uC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>The binding of H-NS , StpA , LRP and FIS to the IGLB DNA fragment was analysed by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The binding of H-NS to DNA does , however , appear to be influenced by structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Rather than binding directly to specific DNA sequences in the hly operon , Hha exhibits nonspecific DNA binding ; therefore , it was proposed Hha binds to H-NS , which binds to specific regions of regulatory sequences in the hly operon [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Speciﬁc binding of H-NS to pap DNA in vitro also results in methylation protection of the GATC sequences ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Distamycin , which binds in the minor groove of DNA and ` straightens out ' bends in DNA [ 47 ] has been also shown to abolish H-NS pre-ferentially binding to curved DNA [ 43 , 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Obviously , this property is independent of the capacity of H-NS to-bind to DNA and appears to rest predominantly on the DNA-binding properties of LRP alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>We take this observation as clear evidence that H-NS and LRP stimulate each other in DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>We conclude from this observation that the stimulating effect of H-NS on the binding of LRP to the target DNA must either be of transient nature or occur at such a low H-NS to LRP stoichometry that we were unable to verify it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Hence we were interested to ﬁnd out whether the defect in DNA binding would persist when heteromers between the wildtype H-NS and the G113D proteins are formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Conclusions H-NS differs from typical sequence-specific regulat-ory proteins , and its binding to DNA is very unusual , both in terms of recognition and affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1372">
Dame , R.T. , Wyman , C. , and Goosen , N. ( 2001 ) Structural basis for preferential binding of H-NS to curved DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></label>
<input id="act_similarity_1372", type="checkbox">
<div class="ACT_similar">
<li>Dame , R.T. , Wyman , C. and Goosen , N. ( 2001 ) Structural basis for preferential binding of H-NS to curved DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li></div>
<li>These data suggest binding of H-NS to the leader DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Loss of curvature would be expected to inhibit the high-affinity binding of H-NS to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>In spite of the fact that the H-NS protection pattern is complex and that the borders of the protected regions are not sharp , the effects of H-NS in the presence of excess competitor DNA are specific and allow us to conclude that preferential binding of H-NS occurs on both sides of an experimentally located DNA bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>In a different approach to studying the effect of curvature on H-NS binding we took advantage of the n on-intercalating ONA-binding ligand distamycin , which is known to remove DNA curvature ( Wu and Crothers , 1984 ) by binding to the minor-groove of ( dA ) n stretches ( van Dyke et ai , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In this context it may be relevant that intercalating agents , which can alter the helical pitch of DNA as well as flexibility of the helix , inhibit H-NS binding [ 461 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Silencing by H-NS is presumably regulated gene speciﬁcally , although physiological parameters such as temperature , osmolarity , and DNA supercoiling have been postulated to pleio-tropically modulate H-NS activity ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1380">
References Amit , R. , Oppenheim , A.B. , and Stavans , J. ( 2003 ) Increased bending rigidity of single DNA molecules by H-NS , a temperature and osmolarity sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></label>
<input id="act_similarity_1380", type="checkbox">
<div class="ACT_similar">
<li>Amit , R. , Oppenheim , A.B. , and Stavans , J. ( 2003 ) Increased bending rigidity of single DNA molecules by H-NS , a temperature and osmolarity sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li></div>
<li>Binding of H-NS to DNA results in several structural ( including supercoiling ) alterations that have been variously referred to as bending , bridging , coating , looping , and stiffening of the DNA ( 16 , 20 , 383 23 , 32 , 39 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>, 1996 ) while DNA concentration is increased by about 20 ' / o ( P. L. Moreau , unpublished results ) , which suggests that the ratio of H-NS to DNA and , thus , the activity of H-NS may be significantly increased in Pi-starved cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Moreover , since a significant increased level of carnitine dehydratase activity was observed in the hns mutant , curved DNA sequences characterized as dA and dT stretches located upstream of caiT may function as regulatory regions to which the histone-iike H-NS protein binds in vivo ( Tanaka et ai , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7815937" target="_blank">7815937</a>]<br></li>
<li>Gel shift assays were performed as descnbed in tbe Experimental procedures in tbe presence of different amounts of H-NS ( lane C , no protein ; lanes 1-5 ( left ) or & -10 ( right ) : 50 ng , 100 ng , 200 ng , 400 ng , 800 ng ) using different labelled DNA fragments in the presence of excess unlabelled nonspecific DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Sfh binds to a subset of H-NS binding sites in wildtype cells , but expands its reach in Dhns .63 Sfh is different from StpA however , in the sense that it is a plasmid encoded protein , which serves the role of a stealth agent , as opposed to StpA which is well-ensconced in the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>It is worth noting the observation that very pure H-NS preparations are often weak in their binding affinity , and protein concentrations higher than under standard conditions have to be employed for proper binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>However , the effect was dependent on an intact appY gene , and there was no speciﬁc binding of H-NS to these promoters , indicating that the effect is indirect , being mediated by increased concentrations of AppY protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Surprisingly , a signiﬁcant stimulation in complex formation was observed when the G113D mutant H-NS protein was added to a binding reaction containing 0.1 μM LRP , which by its own merely results in a minute complex band ( see Fig. 8 , lanes 12 -- 14 and 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Data are derived from Figs 8A , 9A and 10A ( sense strand only ) and show the footprints obtained : ( i ) in the absence of any binding protein , ( ii ) in the presence of H-NS , ( iii ) in the presence of Mn2 + -- Fur , and ( iv ) in the presence of both H-NS and Mn2 + -- Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Aliquots of the digests were mixed with increasing concentrations of H-NS protein and binding was visualized by gel retardation ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>As observed in Fig. 4 , when the PnrdA and PnrdD DNA fragments were used in the presence of increasing H-NS concentrations , protein-DNA complexes exhibiting decreased mobility were detected , thus suggesting the progressive occupancy of multiple binding sites by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>This control clearly showed that stimulation of LRP binding does not occur in the presence of arbitrary protein , but is exclusively observed with H-NS , thus supporting the speciﬁcity of the interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Binding of H-NS and LRP was tested in the presence of increasing concentrations of the respective other protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Moreover , H-NS as well as LRP is known to exist in a variety of oligomeric structures and is capable of heterologeous protein -- protein interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Although it is not known whether this is a direct interaction , or whether the Hsp70-1ike protein interacts with wild-type H-NS , this finding raises many in triguing possibilities [ 281 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Thus , it seems that either H-NS competes with Fur as a direct antagonist or Fur and H-NS regulate the same intermediary regulatory protein but in opposing ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>H-NS and related proteins can engage in both homologous and heterologous protein -- protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>be expected to relieve autorepression , allowing synthesis of more H-NS protein to match the increase in target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Effect of increasing amounts of H-NS protein ( lanes 1 and 6 , no protein added ; lanes 2 and 7 , 0.32 lg ; lanes 3 and 8 , 0.48 lg ; lanes 4 and 9 , 0.65 lg ; lanes 5 and 10 , 1.3 lg ) on the electrophoretic mobility of DNA fragments corresponding , respectively , to the promoter regions of htrA ( lanes 1 -- 5 ) and rpoE ( lanes 6 -- 10 ) genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Lane 1 , protein standards with sizes indicated to the left ; lane 2 , ﬂowthrough that did not bind to the dsDNA-cellulose column ; lane 3 , puriﬁed H-NS that was bound and eluted off of the column , pooled , and dialyzed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>B. End-labelled purified pnir7150 EcoRI ± HindIII fragment was incubated with increasing amounts of purified H-NS protein as follows : lane 1 , no protein ; lane 2 , 0.1 mg of H-NS ; lane 3 , 0.2 mg of H-NS ; lanes 4 , 0.3 mg of H-NS ; lane 5 , 0.5 mg of H-NS ; lane 6 , 0.8 mg of H-NS ; lane 7 , 1 mg of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Restricted DNA was incubated with purified H-NS protein as follows : lanes 2 and 8 , no protein ; lanes 3 and 9 , 0.2 mg of H-NS protein ; lanes 4 and 10 , 0.4 mg of H-NS protein ; lanes 5 and 11 , 0.6 mg of H-NS protein ; lane 6 and 12 , 0.8 mg of H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Lane 1 , no protein ; lanes 2 to 8 , H-NS ( 0.04 , 0.08 , 0.12 , 0.25 , 0.5 , 1.0 , and 2.0 M , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>H-NS-dependent stimulation of LRP binding is independent of H-NS DNA interaction The synergism in DNA binding between H-NS and LRP could either be mediated by the formation of heteromers via direct protein -- protein interaction or involve alterations of the DNA structure , rendering it more favourable for the binding of the respective other protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>We wished to know whether the synergism between H-NS and LRP was caused by the formation of stable heteromers or alternatively resulted from a conformational distortion of the DNA , rendering it more adequate for protein interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>It is feasible that H-NS could as well act as a bridging protein , linking LRP -- DNA complexes , which are then formed in a more speciﬁc and stable manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>The results present strong evidence that the DNA-binding protein H-NS , known to repress transcriptional activity from horizontally transferred genetic elements , is involved in downregulation of the Pcas and anti-Pcas promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1409">
Badaut , C. , Williams , R. , Arluison , V. , Bouffartigues , E. , Robert , B. , Buc , H. , and Rimsky , S. ( 2002 ) The degree of oligomerization of the H-NS nucleoid structuring protein is related to speciﬁc binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></label>
<input id="act_similarity_1409", type="checkbox">
<div class="ACT_similar">
<li>Badaut , C. , Williams , R. , Arluison , V. , Bouffartigues , E. , Rob-ert , B. , Buc , H. , and Rimsky , S. ( 2002 ) The degree of oligomerization of the H-NS nucleoid structuring protein is related to speciﬁc binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li></div>
<li>However , many reports [ 6 -- 8 ] have raised the possibility that H-NS binds to DNA in a sequence-specific manner by binding small accessory proteins , resulting in protein complexes involved in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>DNA topology changes in stationary phase leading to more compact nucleoids , which is probably the result of an increased production of the nucieoid-associated DNA-binding protein H-NS ( Spassky etai , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Binding of H-NS in vitro to circular closed plasmid binding protein-dependent transport system for the osmopro-DNA results in a compaction of its substrate withouta strong tectant glycine betaine ( 43 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>This cooperative binding was exempliﬁed when a less than twofold increase in protein concentration changed the DNA substrate proﬁle from 100 % free to mostly H-NS bound ( compare lanes 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1415">
Silencing by the nucleoid-associated DNA-binding protein , H-NS , and activation by insertion elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></label>
<input id="act_similarity_1415", type="checkbox">
<div class="ACT_similar">
<li>Silencing by the nucleoid-asso-ciated DNA-binding protein , H-NS , and activation by insertion elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li></div>
<li>Mutations in DNA gyrase and in the DNA-binding protein H-NS are known to activate the bgl operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1418">
LaTeana , A. , Fatconi , M. , Scarlato , V - , Lammi , M. , and Pon , C ( 1989 ) Characterization of the structural genes for the DNA-binding protein H-NS in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></label>
<input id="act_similarity_1418", type="checkbox">
<div class="ACT_similar">
<li>La Teana , A. , Falconi , M. , Scarlato , V. , Lammi , M. , and Pon , C.L. ( 1989 ) Characterization of the structural genes for the DNA-binding protein H-NS in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li></div>
<li>Signi ® cantly , proteins that are involved directly in bacterial stress responses , including the heat-shock protein DnaK and the DNA-binding protein H-NS , were found to be particularly susceptible to oxidative damage ( Dukan and NystroÈm , 1998 , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>Any cooperative interaction between the two binding proteins H-NS and LRP and the common DNA target site might in principle involve protein -- protein interaction and / or a change in the DNA conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>In most cases , H-NS repression is almost completely reversed in the presence of specific inducers or changes in growth conditions , often acting though a specific activator protein that displaces H-NS from the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>In addition , we could not identify the presence of H-NS in DNA -- protein complexes , which were exclusively formed when both proteins are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>In addition to the well-known protein factors , such as NusG and NusA , another group of proteins that can be oligomerized on DNA to form a scaffold , such as H-NS , Cnu , and Hha , was recently shown to participate in modulation of Rho termination efﬁciency ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24794565" target="_blank">24794565</a>]<br></li>
<li>Dissociation rate determinations To determine the half-lives of H-NS and Fis DNA complexes , changes in the equilibrium concentration of DNA-protein complexes owing to the addition of excess free DNA were followed by gel retardation as described ( Fried , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Keywords : Nucleoid ; Bacterial chromatin ; DNA compaction ; Nucleoid-associated protein ; H-NS ; HU ; IHF ; Fis 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>To this end , DNA fragments corresponding to the entire nrdA , nrdD , and nrdH promoters ( 703 bp , nucleotide positions 575 to 127 ; 261 bp , 244 to 17 ; and 441 bp , 419 to 22 , respectively ) were used for band shift assays in the presence of puriﬁed H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Atomic force microscopic images of hdeABp DNA -- protein complexes Lastly , we employed high-resolution atomic force microscopy ( AFM ) to visualize the two forms of RNPs bound to hdeABp DNA ( − 216 to +580 ) in the presence or absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Five nanograms of 32p-labeled DNA and 40 or 80 pmol of purified H-NS protein were mixed , in the presence or absence of competitor DNAs , and were subjected to 5 % polyacrylamide gel electrophoresis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>No cross-linking between H-NS and NS was found , however , either at low - or at high-ionic strength , either in the presence or absence of DNA at protein concentrations close to 10 • * M ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>B. Lane 1 , size markers as in ( A ) ; lane 2 , H-NS marker sample : lane 3 , sample oJ the void volume from G100 chromatography of a DNA-protein complex ; lane 4 , same fractions of a separation as in lane 3 but in the absence of DNA ; lanes 5 and 6 , silver stain of lanes 3 and 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>This response is dependent upon continuing protein synthesis during the period of DNA synthesis inhibition and upon the presence of a gene producing functional H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>We attribute the presence of H-NS in this fraction to the tact that a portion of the protein sediments with Ihe crude ribosomal pellet either via a loose association with these particles or bound to DNA fragments which are DNase-resistant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>Additional cross-linking experiments were carried out in which H-NS was reacted in the presence of DNA and/or in the presence of the other histone-like protein NS ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>The electrophoretic mobility of this DNA-protein complex was further retarded in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Most notable , though , were the very low amounts of H-NS needed to ﬁrst form a protein-DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Purified H-NS protein binds to pnir DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>This result suggests that H-NS may ﬁrst bind tightly to DNA and then form H-NS -- H-NS interactions as protein concentrations are increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>We presume that this reflects the time required to establish a genome-wide equilibrium of H-NS bound to high-affinity ( curved ) DNA sites under conditions of changing H-NS protein : DNA ratio .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1440">
Lucchini , S. , Rowley , G. , Goldberg , M.D. , Hurd , D. , Harrison , M. , and Hinton , J.C.D. ( 2006 ) H-NS mediates the silencing of laterally acquired genes in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></label>
<input id="act_similarity_1440", type="checkbox">
<div class="ACT_similar">
<li>Lucchini , S. , Rowley , G. , Goldberg , M.D. , Hurd , D. , Harrison , M. , and Hinton , J.C. ( 2006 ) H-NS mediates the silencing of laterally acquired genes in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19732340" target="_blank">19732340</a>]<br></li>
<li>Lucchini , S. , Rowley , G. , Goldberg , M. D. , Hurd , D. , Harrison , M. , and Hinton , J. C. ( 2006 ) H-NS mediates the silencing of laterally acquired genes in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_1443">
Navarre , W. W. , Porwollik , S. , Wang , Y. , McClelland , M. , Rosen , H. , Libby , S. J. , and Fang , F. C. ( 2006 ) Selective silencing of foreign DNA with low GC content by the H-NS protein in .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></label>
<input id="act_similarity_1443", type="checkbox">
<div class="ACT_similar">
<li>Navarre , W.W. , Porwollik , S. , Wang , Y. , McClelland , M. , Rosen , H. , Libby , S.J. , and Fang , F.C. ( 2006 ) Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19732340" target="_blank">19732340</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Navarre , W.W. , Porwollik , S. , Wang , Y. , McClelland , M. , Rosen , H. , Libby , S.J. and Fang , F.C. ( 2006 ) Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">H-NS protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>In light of the fact that H-NS is found in a cross-section of bacterial species ( 43 ) and is part of global regulatory networks involved in virulence , metabolism , and environmental stress , yet contains no known structural peptide motifs ( such as zinc ﬁnger or helix-turn-helix motifs ) , we randomly mutagenized hns to help delineate biologically active H-NS domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The phasing of the hyper-reactive and protected nucieotide positions is compatible with the assumption that H-NS interacts with the major groove on one side of the helix axis , thereby expanding and distorting the adjacent minor groove ( Drew and Travers , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In summary , the results are entirely consistent with the conclusion that H-NS , like Fis , must be a major-groove binder , and that there are more than one ( probably three ) interacting domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>To examine further the interaction of H-NS with the cydAB control region , three additional probes , designated A , B and C , were generated ( Fig. 4 , top ; see Experimental procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>We observed that the two forms of RNPs could form an open initiation complex ( RPO ) at hdeABp , but that E 70 failed to initiate transcription in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>These data suggest that the two forms of RNPs do not compete with H-NS for hdeABp DNA-binding , but form a stable RNP -- promoter complex , presumably an open promoter complex ( see below ) , in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>These data indicate that E 70 forms an RPO even in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>We thus carried out an in vitro transcription experiment with purified R N A polymerase and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>It was also observed that one of the vector DNA fragments in this experiment ( fragment 1 ) shifted in the presence of H-NS , which is consistent with previous ﬁndings of H-NS interaction with the plasmid carried bla promoter region ( 33 , 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>It is also possible that HF-I may exist in two forms in the dell : one form would be involved in translation and would bind to RNA , and another would be compiexed with H-NS and would bind to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984093" target="_blank">7984093</a>]<br></li>
<li>B. H-NS dimers are stacked on top of each other and form a scaffold , from which DNA bending domains extend in two opposite directions allowing direct interaction with two separate tracts of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>As in H-NS , StpA , BpH3 and HvrA , theoretical analysis suggests that the N-terminal domains of SPB , XrvA and VicH are mainly a-helical and able to adopt a coiled-coil conformation , frequently identiﬁed as the dimerization element in several transcription factors ( Lupas , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>For example , the Leu residue in position 26 in H-NS and in StpA corresponds to Leu at positions 20 and 14 in BpH3 and HvrA respectively ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Interestingly , a naturally occurring single amino acid substitution at position 16 ( alanine to valine ) in the truncated H-NSTUPEC results in a much weaker interaction with H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Interestingly , H-NS helix H2 appears to contain key residues for the interaction with its truncated derivatives : arginine at position 12 is essential for the agonistic interaction of Hha with H-NS64 ( García et al. , 2006 ) and alanine at position 16 of H-NSTEPEC for the antagonistic interaction with HNS ( Wil-liamson and Free , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Structural aspects of H-NS -- Hha interaction The three-dimensional structure of the Hha protein consists of four alpha-helical segments separated by loops : helix 1 ( residues 8 -- 16 ) , helix 2 ( residues 21 -- 34 ) , helix 3 ( residues 37 -- 55 ) and helix 4 ( residues 65 -- 69 ) ( Yee et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Studies of the interaction of Hha with H-NS64 ( H-NS residues 1 -- 64 ) showed that strongly perturbed residues are located both in the surface and in the hydrophobic core of the Hha structure , suggesting that Hha undergoes conformational rearrangement after interacting with H-NS ( García et al. , 64 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Several other proteins have been identified that contain the H-NS C terminus fused to an unrelated N terminus16 ,71 , so this might be a relatively common mechanism for attenuating the effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>In panel c , high occupation ot H-NS sites -LCB- A , B and C ) completely changes the activation loop geometry of the UAS region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>As LeuO is an H-NS antagonist with pleiotropic function , activation of leuO expression should occur in response to speciﬁc environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Therefore , S we conclude that , in contrast to csiD activation by H-NS , activation by Lrp is not speciﬁc for either s - or S s - containing RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>It appears that H-NS specificaily antagonizes the Fismediated activation of the PI promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Is H-NS repression restricted to the counteraction of FIS-dependent activation ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Esposito , D. , Petrovic , A. , Harris , R. , Ono , S. , Eccleston , J.F. , Mbabaali , A. , et al. ( 2002 ) H-NS oligomerization domain structure reveals the mechanism for high order self association of the intact protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>H-NS oli-gomerization domain structure reveals the mechanism for high order selfassociation of the intact protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_26">
H-NS oligomerization domain structure reveals th mechanism for high-order self-association of the intact protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></label>
<input id="dom_similarity_26", type="checkbox">
<div class="DOM_similar">
<li>H-NS oligomerization domain structure reveals the mechanism for high order self-association of the intact protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>H-NS oligomerization domain structure reveals the mechanism for high-order self-association of the intact protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li></div>
<li>Systematic mutational analysis revealing the functional domain organization of Escherichia coli nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422614" target="_blank">9422614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Ueguchi , C. , Seto , C. , Suzuki , T. , and Mizuno , T. ( 1997 ) Clari-ﬁcation of the dimerization domain and its functional signiﬁcance for the Escherichia coli nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Clariﬁcation of the dimerization domain and its functional signiﬁcance for the Escherichia coli nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Since the hns : : neo strain used here still retains the N-terminal 37 residues , it is possible that the deletion mutants express a truncated form of H-NS , and may exhibit residual function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>These ® ndings suggest that ycgR and yhjH , together with H-NS , are involved in the motor function and constitute new members of the ¯ agellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>Identical residues in comparison with H-NS are in grey boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>, 1986 ) , the distance between the lysine residues involved in cross-linking H-NS mono-mers is not critical either for yield or for the rate of the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>These results indicate that the Lys residues involved in the cross-linking reaction are not the same as those presumably responsible for the interaction of H-NS with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>The H-NS concentration appears to play a signiﬁcant role in chromosome replication ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The two main regions of H-NS interaction ( labelled I and II ) are shown by dashed lines along the right .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The H3 helices of two H-NS monomers adopt a negatively charged coiled-coil structure Xanked by the H1 and H2 helices , which fold back onto the coiled-coil ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>) Primary structure The primary structure of H-NS was determined as described in Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>Binding studies with LRP and H-NS in combination demonstrated that the two proteins interacted with obvious synergism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Interaction of H-NS wiih other proteins H-NS may interact specifically with other proteins in the cell , permanently or transiently , and this may influence its function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>We also speculate that the N and C termini may interact with each other , and we await determination of the threedimensional crystal structure of H-NS to test this hypothesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Utilizing a random mutagenesis scheme followed by multiple phenotypic analyses , we have been able to aid in deﬁning the relationship between H-NS structure and function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>To obtain information on the structure , function and/or evolution of H-NS , we searched for other H-NS-related proteins in the latest databases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>To get further information on the structure and the function of H-NS , we undertook two different approaches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>However , by characterization of dominant negative mutants and mutational analysis , it has been proposed that H-NS contains at least two distinct domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>H-NS contains two functionally diVerent domains separated by a Xexible linker ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Although some H-NS functions may be isolated to particular domains , it seems likely that all portions of the protein work coordinately and in concert with each other to exert full H-NS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Rectangles represent domains involved in the indicated H-NS functions based on this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>We concluded that the ability of H-NS to control promoter ﬂipping resides in the N-terminal half of the protein and that the C terminus may be needed only for some undeﬁned role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Another model , based on data from the S. typhimurium H-NS protein , suggests a role for the dimeric N terminus in forming head-to-tail oligomers70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>B A Discussion In this report , we bring further evidence about the interaction of members of the H-NS and Hha families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>It is apparent that interaction of H-NS with members of the Hha/YmoA family has a biological role different from the interactions reported above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Since then , interacions of other members of the family ( YmoA , YdgT ) with members of the H-NS family have also been shown [ 33,34 ] , and a mutational analysis of the Hha protein has suggested Hha consisting of a unique protein-binding domain [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Furthermore , this interaction appears to be wide-spread between members of both families : YmoA protein also interacts with Y. enterocolitica H-NS protein ( Nieto et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Residues most affected by the interaction with the H-NS protein are coloured in yellow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Evidence for direct protein-protein interaction between members of the enterobacterial Hha/YmoA and H-NS families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Mutagenesis data imply that the entire Hha protein is involved in the interaction with H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The relationship between members of the Hha and H-NS families is not only supported by protein -- protein interaction data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Indeed , the Hfq protein copurifies with H-NS , suggesting a specific interaction [ 26 ] and genetic data indicate that hfq mutants have phenotypes consistent with a role in modulating H-NS function [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>One proposal , which is based on structural data from the E. coli protein , is that the interaction of the N and C termini of H-NS with DNA imposes a more stable structure on the linker , allowing it to interact with the corresponding portions of other bound H-NS mole-cules and leading to the formation of a complex , crosslinked nucleoprotein structure ( FIG. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>To clarify protein structure and function , we examined H-NS iso ¬ * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Overall , the results suggest that the interaction-with-DNA does not influence the quaternary structure of H-NS and that the function of this protein does not entail a direct interaction with NS ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>The possibility that protein-protein interactions modulate H-NS function remains to be fully explored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The bridging of two tracts coated with H-NS protein stems from interactions between the ﬂexible linker regions in H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>The H-NS protein includes two well-deﬁned domains connected by a ﬂexible linker .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>References Ali SS , Beckett E , Bae SJ , Navarre WW ( 2011 ) The 5.5 protein of phage T7 inhibits H-NS through interactions with the central olig-omerization domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22327802" target="_blank">22327802</a>]<br></li>
<li>The proline-to-alanine substitution at position 116 has also been reported to disrupt the oligomerization of H-NS in vitro and that of a fusion protein in which H-NS replaces the missing oligomerization domain of a truncated bacteriophage λ cI repressor in vivo78 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Heterologous protein -- protein interactions The oligomerization domain of H-NS allows it to interact not only with another copy of itself , but also with heterologous polypeptides that share this domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Based on work with regions of the H-NS protein , it has been suggested that higher-order oligomerization is a function of the central region of the protein ( residues 46 -- 90 ) 69 , which includes the flexible linker domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Interestingly , this island -- which is thought to have been acquired horizontally -- encodes Ler , a protein antagonist of H-NS that shares the same nucleic-acid-binding domain but lacks a homologue of the H-NS oligomerization domain48 ,105 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The dimerization domain is highly conserved among members of the H-NS family of proteins , making it likely that it also facilitates heteromeric interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_74">
The H-NS dimerization domain deWnes a new fold contributing to DNA-recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></label>
<input id="dom_similarity_74", type="checkbox">
<div class="DOM_similar">
<li>The H-NS dimerization domain deﬁnes a new fold contributing to DNA-recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li></div>
<li>The negative effects of hns mutations on bacterial motility were mainly clustered to the H-NS truncations , arguing that the conformation of this domain , rather than individual residues , is relevant to this motility phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>H-NS promotes looped domain formation in the bacterial chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>The former involves the N-terminal domain of H-NS ( residue 2 to approximately residue 47 ) and the latter the intermediate or linker domain ( residues 58 to 84 ) , and polym-erization is achieved by the alternation of site 1-site 1 ( head-to-head ) and site 2-site 2 ( tail-to-tail ) interactions between the adjacent protomers in the scaffold ( Fig. 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Studies of mutant H-NS proteins suggest that the N-term-inal domain plays a crucial role in transcriptional repression of the proU operon and in the oligomerization process ( Ueguchi et al. , 1996 ; Williams et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Although we lack a precise picture of the complexes formed in vivo , we speculate that Hha-like proteins , when interacting with H-NS , affect the oligomerization of H-NS by providing additional sites for interaction or by inﬂuencing the folding of the linker domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>These H-NS fragments encompass amino acids 1 to 91 and 1 to 93 and include the complete dimerization and oligomerization domain ( 14 ) and a region of H-NS ( aa 39 to 60 ) which is necessary for stabilization of StpA ( 3 , 15 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>For repression of bgl , it was shown before that StpA complements truncated H-NS proteins comprising the N-terminal dimerization/oligomerization domain , including amino acids 1 to 91 ( or 93 ) ( 17 , 18 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>The locations of these amino acid substitutions and their effects on various H-NS activities allowed us to construct a model of H-NS domain organization ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Only a few amino acids are conserved in both regions in H-NS , StpA , BpH3 and HvrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>The amino acid composition of the C-domain of StpA ( residues 91 -- 134 ) is 70 % identical to that of H-NS ( residues 91 -- 137 ) , while the C-domains of BpH3 ( residues 82 -- 123 ) and HvrA ( residues 61 -- 102 ) are both 40 % identical to that of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>However , despite a low amino acid conservation in their N-terminal domain , we observed that H-NS , StpA , BpH3 and HvrA decreased , at least in part , the accumulation of ProX in an hns mutant ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>In marked contrast , the bgl operon is not markedly activated by mutations affecting the C-terminal domain of H-NS , including the hns205 : : Tn10 allele used here ( 19 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>García , J. , Madrid , C. , Juárez , A. , and Pons , M. ( 2006 ) New roles for key residues in helices H1 and H2 of the Escheri-chia coli H-NS N-terminal domain : H-NS dimer stabilization and Hha binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>All available data suggest that H-NS has evolved to modify its ability to modulate gene expression by interacting with proteins with at least some features of the N-terminal domain of H-NS itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Other proteins besides the Hha/YmoA family are structurally equivalent to the H-NS N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Recent structural data obtained by our group ( manuscript to be submitted ) support the previous hypothesis : proteins of the Hha family bind the N-terminal domain of members of the H-NS family and are structurally equivalent to truncated H-NS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Structural and functional organization of the C-terminal domain in H-NS and H-NS-like proteins The C-terminal domain is the most conserved region in H-NS-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Altogether , it seems that Hha-like proteins might have evolved to mimic the N-terminal domain of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Heteromeric complexes formed by truncated H-NS proteins ( 93 amino acid resi-dues corresponding to the N-terminal end ) and fulllength StpA may resemble structurally and functionally to complexes formed by Hha ( 72 amino acid residues , structural similarity to the N-terminal domain of H-NS ) and H-NS or StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>A comparison of the amino acid sequences corresponding to both Hha and H-NS families suggests that Hha-like proteins may be specialized homologues of the N-terminal oligomerization domain of H-NS ( Nieto et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Additional data supported the model ( Nieto et al. , 2002 ) : ( i ) the mutational analysis of the Hha protein suggested that Hha represents an unique protein-binding domain , ( ii ) a comparison of the amino acid sequence between members of the H-NS and Hha family of proteins suggested that Hha/YmoA are evolved to mimic the H-NS N-terminal domain and ( iii ) the interaction described between YmoA and H-NS in Y. enterocolitica suggested that such kind of protein complexes are conserved in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Finally , secondary structures were predicted to identify structural and/or functional domain in the H-NS protein of Y. enterocolitica using the GORIV method [ 23 ] The N-terminal domain was clearly predicted as mainly α-helical while the C-terminal domain is a probable mixed α - structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>The N-terminal domain seems to play a crucial role in the oligomerization process of H-NS and StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>These data show that stabilization of StpA and its functional cooperation with a fragment of the N-terminal H-NS dimerization domain can be separated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>It becomes StpA dependent in the presence of a fusion protein encompassing the N-terminal 37 amino acids of H-NS , which comprise the core of the dimerization domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>In addition , autorepression by StpA is enhanced in an hns : : kan mutant carrying a 70-amino-acid fusion protein , H-NSKN ( Fig. 4 ) , encompassing the 37 N-terminal amino acids of H-NS , which include the ﬁrst two alpha helices and half of the third alpha helix that form the core of the dimerization domain ( 3 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Structurally based alignment of the C-terminal domain of H-NS and H-NS-like protein sequences [ 2 , 8 , 16 , 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>H-NS is able to oligomerize , being this property a funcion of the N-terminal domain of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Isolated outside these groups , the BpH3 protein of Bordetella pertussis exhibits a low amino acid conservation with H-NS , particularly in the N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Moreover , we recently identiﬁed VicH ( Vibrio cholerae H-NS-like protein ) , a 135-amino-acid protein ( GenBank accession number AJ010791 ) showing a strong amino acid conservation between its C-terminal domain and that of H-NS and H-NS-like proteins ( unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Further according to this model , overexpressed YdgT would behave like H-NS 64 in serving to interrupt the polymeric scaffold structure by interacting with the N-terminal domain of H-NS , as shown in Fig. 8C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Structural topology of the C-terminal domain is conserved : the hydrophobic core of H-NS that holds this structure packed is built around the interactions of W109 and Y97 , Y99 , F134 and I135 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>( A ) High resolution models of dimeric H-NS based on the structure of the Escherichia coli H-NS C-terminal DNA-binding domain ( PDB : 1HNR -- residues 80 -- 136 ) ( Shindo et al. , 1995 ) and two diVerent structures of the N-terminal dimerization domain ( PDB : 1NI8 -- residues 1 -- 46 ( Escherichia coli ) ( Bloch et al. , 2003 ) and 1LR1 -- residues 1 -- 57 ( Salmonella typhimurium ) ( Esposito et al. , 2002 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>NMR analysis of the N-terminal domain of Sal-monella typhimurium H-NS suggests parallel dimerization ( Esposito et al. , 2002 ) , whereas NMR analysis of the Nterminal domain of E. coli H-NS and X-ray analysis of the same domain in Vibrio cholerae H-NS indicate antiparallel dimerization ( Bloch et al. , 2003 ; Cerdan et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Furthermore , a chimeric protein obtained by replacing the N-terminal domain of H-NS ( residues 1 -- 64 ) by the ﬁrst 60 residues of Hha partially complements some hns-induced phenotypes ( Rodríguez et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>In addition to the N-terminal domain of H-NS , the linker domain appears to be required for tetramerization and oligomerization ( Stella et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>In bold is plotted the STRIDE assignment for H-NS C-terminal domain from NMR structure and for StpA , BpH3 and HvrA C-domains from MODELLER .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Tridimensional structures of the C-terminal domain of BpH3 , HvrA and StpA were calculated by homology with the NMR structure of the C-terminal domain ( 47 amino acids ) of H-NS ( Shindo et al. , 1995 ) using MODELLER ( Sali and Blundell , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Three-dimensional structures were obtained by satisfying spatial restraints derived from the structure of the C-terminal domain of H-NS obtained by NMR experiments ( Shindo et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>In contrast , no information is available concerning H-NS structure , except for the organization of its C-terminal domain that has been resolved by nuclear magnetic resonance ( NMR ; Shindo et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>However , except this C-terminal domain , no information is available concerning the structure of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Structural and functional organization of the N-terminal domain in H-NS and H-NS-like proteins To identify structural and/or functional domains in H-NS , StpA , BpH3 and HvrA , sequences were aligned by MULT-ALIGN and hydrophobic cluster analysis ( HCA ) methods , and secondary structures were predicted ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Taken together , these data reinforce the idea that similarities exist in the functional and structural organization of the N-terminal region in H-NS and H-NS-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>These results are discussed in terms of H-NS domain organization and implications for biological activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>These results suggest that the hydrophobic core of the N-domain is made up of two subdomains separated by a loop ( loop 1 ) in H-NS and H-NS-like proteins ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Intriguingly , the N-terminal 90 amino acids of H-NS exhibit 22-29 % identity to structural proteins such as myosin , keratin and dystrophin ( fig IC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Dimerization of H-NS monomers requires the Wrst 64 N-terminal amino acids and the formation of larger olig-omers involves amino acids 65 -- 91 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Amino ( N ) - terminal sequences were determined for both component A ( H-NS ) and B ( HF-I ) , and compared with that of published N-terminal 5 amino acid residues of HF-I ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2020545" target="_blank">2020545</a>]<br></li>
<li>two amino acids ( Met and Ser ) were detected in the first Edman degradation cycle showing that the N-terminal Met residue is removed from only a fraction of the H-NS protein ( 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>( Amino acid residues in H-NS are numbered here beginning from the N-terminal methionine as 1 [ 65 ] , even though this residue is posttranslationally removed from the mature protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>The H-NS protein is 137 amino acids in length and has three structural components ( in this discussion , the amino-terminal methionine of H-NS is referred to as position 1 ) : an N-terminal domain extending up to residue 65 that contains oligomerization activity ; a car-boxy-terminal domain beginning at residue 90 that has nucleic-acid-binding activity ; and a flexible linker that connects the two16 ,56,69,70 ( FIG. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Thus , these results suggested that the H-NS modiﬁcation site is located within the amino half of the protein , potentially near residues 18 to 26 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Among the conserved residues are most of the basic amino acids and all the aromatic residues ( i.e. three tyrosines , one phenylalanine and the single tryptophan residue , which in E. coli H-NS was found to be buried within the protein structure and sen ¬ Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2494066" target="_blank">2494066</a>]<br></li>
<li>The H-NS protein is a non-basic polypeptide consisting 01 137 amino acid residues , many 01 which are negatively or positively charged ( pi = 5.6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>As judged from the nucleotide sequencing data ( 24 ) , H-NS consists of 136 amino acid residues with a calculated molecular weight of 15,402 , this value being in good agreement with that for the protein purified in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2126011" target="_blank">2126011</a>]<br></li>
<li>This could account for a sensitivity of the promoter to DNA topology , which may affect such a structure , and for its recognition by H-NS , which has a preferential afﬁnity for the intrinsic DNA curvature often associated with long stretches of A and T residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Accordingly , the intergenic common regulatory region shared by cai and ﬁx harbors putative motifs of recognition for the binding of CRP and H-NS ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>It is noteworthy that an in vivo analysis suggests that the 13C element occurs in more than 70 % of putative rpoS-dependent promoters ( 4 ) : this observation would suggest that the 13C element might be needed to improve EsS -- promoter interaction in the bacterial cell , possibly to overcome the negative effects of DNA-binding proteins , such as H-NS , that can modulate RNA polymerase -- promoter interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Indeed , there is evidence that H-NS influences open complex formation rather than RNA polymerase binding [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The latter region is located upstream of the identiﬁed Pcas promoter , suggesting that binding of H-NS might affect RNA polymerase -- promoter interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>It would be interesting to examine in vitro binding of H-NS to the cai ± ® x regulatory region and to study its possible interaction with the two activators and the RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The amino acid sequences of H-NS from various species are shown in figure I. H-NS contains a large number of charged residues although , unlike eukaryotic histones ( and most other DNA-binding proteins ) , these are predominantly acidic rather than basic ( fig 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>A typical DNA-binding motif ( ie helix-loop-helix , zinc-finger ) can not be identified , and it is not known which residues of H-NS contribute to DNA-binding , although there is a stretch of positively charged resi-dues towards the C-terminus which might be involved in DNA-binding ( fig IB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Therefore domain A seems to be arranged in an inverted orientation relative to domains B and C. Thus , possible protein contacts between H-NS molecules and/or other components of the transcription machinery might be different for H-NS bound to domain A or B and C. Another interesting feature of our footprinting results was the partial overlap of Fis and H-NS binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>From the spacing of the central binding domains between the Fis and H-NS contact sites , based on the minor-groove accessibilities , a difference in the angular orientation of the protein centres with respect to the dyad helix axis of 72 '' to 108 ' can be inferred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Moreover , changing the proline at position 116 to alanine abolishes the ability of H-NS to distinguish curved from non-curved DNA , while retaining non-78 specific DNA-binding activity , and changing this residue to serine has a similar effect56 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Caret denotes residue implicated in DNA-binding in this study ; asterisks represent potential sites of H-NS modiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>This region contains two potential H-NS binding sites ( GGGTATATTG and TTGTATAAAT ) correlating with the H-NS footprint .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Such cooperative binding would be achieved by formation of H-NS dimer -- dimer contacts leading to DNA -- H-NS -- DNA bridging and looping ( see two possible mechanisms in Fig. 11B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>This binding pattern is similar to that for the rrnB P1 promoter region , where H-NS has been shown through several different footprinting techniques to interact with three DNA domains ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Variants of H-NS defective in one or the other of its two dimerization domains , but not those defective in DNA-binding alone , also conferred a similar suppression phenotype in rho and nusG mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>A characteristic feature of dimeric H-NS is the presence of two DNA-binding domains that can potentially interact with two DNA duplexes simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Another model to explain H-NS inhibition of DNA looping is based on the observation that H-NS dimers have two DNA-binding domains and can bridge between DNA duplexes ( 21,43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>It oligomerises on the DNA and forms DNA -- protein -- DNA bridges or rod-like structures , which might be structural motifs for strong transcriptional suppression .22 -- 24 Recent evidence has suggested that the 500 binding regions of H-NS collapse together spatially into two foci ,25 which might act as the bacterial heterochroma-tin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Much remains to be discovered about the physiological significance of the H-NS complexes ; NMR structures of the Cnu and Hha proteins are available [ 9,10 ] and the structures of the N-terminal ( dimerization ) and C-terminal ( DNA-binding ) domains of H-NS are known [ 11 -- 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>YdgT and Hha bear structural resemblance to , and also interact with , the N-terminal oligomerization domains of H-NS and StpA ; in this manner , YdgT and Hha are believed to modulate the DNA-binding and nucleoid-organizing properties of H-NS and StpA even though they do not bind DNA by themselves ( 23 , 30 , 33 , 38 , 42 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Using theoretical analysis ( e.g. secondary structure prediction and DNA-binding domain modelling ) of the amino acid sequence of H-NS , StpA ( an H-NS-like protein in E. coli ) , BpH3 and HvrA and by in vivo and in vitro experiments ( e.g. complementation of various H-NS-related phenotypes and competitive gel shift assay ) , we present evidence that these proteins belong to the same class of DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>In the present paper , using an approach involving theor-etical analysis of functional and/or structural protein domains as well as in vivo and in vitro experiments , we provide evidence that , despite a low amino acid conservation , proteins such as H-NS and StpA of E. coli , BpH3 of B. pertussis and HvrA of R. capsulatus belong to the same family of structurally and functionally related DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>It has been suggested that initial binding of H-NS to a preferred site is followed by lateral extension along the DNA , by oligomerization of H-NS through interactions between N-terminal domains of the protein ( Dorman et al. 1999 ; Badaut et al. 2002 ; Schroder and Wagner 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Instead , a ﬂexible loop structure ﬂanked by a β-sheet and a short α-helix within the C-terminal half of H-NS is believed to be responsible for DNA-binding of this protein ( Shindo et al. , 1995 ; 1999 ; Schröder and Wagner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>H-NS-mediated Silencing of the ade Locus -- In the present work we found that mutations truncating the C-terminal DNA-binding domain of H-NS strongly induced ade gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>In the present study , we found that the expression of H-NS variants defective in one or the other of the oligomerization interfaces , but not in the DNA-binding domain alone , was associated with restoration of transcriptional polarity in rho and nusG mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>For example , H-NS proteins with truncation of the C-terminal DNA-binding domain are sufﬁcient to render repression of bgl StpA dependently , indicating that StpA acts as an adapter to target H-NS mutants defective in DNA-binding to bgl ( 17 , 18 , 29 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>In this study we employ an H-NS mutant containing an amino acid substitution within the loop region constituting the DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>These preliminary data suggest the existence in Y. pestis ( http://www.sanger.ac.uk/Projects/ ) of a protein which mainly differs from the H-NS protein in Y. enterocolitica by a three amino acid deletion in the linker domain located between the N-terminal region and the C-terminal DNA-binding domain in H-NS-like proteins [ 6 , 8 , 29 , 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>H-NS and H-NS-like proteins constitute a large class of DNA-binding proteins As the C-terminal domain is highly conserved in the four proteins studied , we looked for similar domains in proteins of the databases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>The strong amino acid conservation in the C-terminal domain of the various H-NS-like proteins prompted us to determine whether the DNA-binding properties of H-NS-related proteins were similar to those of H-NS , i.e. a preferential binding to the bla promoter region ( Lucht et al. , 1994 ; Zuber et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>This prompted us to search for H-NS-like proteins in databases using a 40 amino acid sequence extending from Lys at position 89 to Lys at position 128 in the DNA-binding domain of H-NS ( ﬁgure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Genetic , biophysical , and structural studies have shown that the 137-amino acid H-NS protein possesses two separate dimerization interfaces , between approximately residues 2 and 47 and residues 58 and 84 , respectively , and a third C-terminal DNA-binding domain between residues 92 and 137 ( 3 , 5 , 8 , 22 , 54 , 57 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Based on the binding properties of an H-NS truncation containing only the C-terminal 47 amino acids of the protein ( 33 ) and analysis of H-NS mutants ( 42 ) , it has been established that the C-terminal third of H-NS encompasses the DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Dimerization of the 136 amino acid H-NS protein requires the N-terminal ~ 45 amino acids , whereas the C-terminus encompasses the DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>H-NS H-NS consists of a dimerization ( N-terminal ) and a DNA-binding ( C-terminal ) domain connected by a ﬂexible linker ( Dorman et al. , 1999 ; Rimsky , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>H-NS consists of an Nterminal dimerization domain and a C-terminal DNA-binding domain that are separated by a linker domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS consists of an N-terminal oligomerization domain and a C-terminal DNA-binding domain , and the ability of the protein to condense DNA and repress transcription apparently depends on its ability to oligomerize ( 19 -- 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>Apparently , repression of the bgl operon is retained in such mutants because the truncated H-NS protein forms a complex with a homologous protein , StpA , which may serve as an adapter , providing a DNA-binding domain to the N-terminal oligomerization domain of H-NS ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>In contrast , repression of bgl by StpA occurs only in the presence of the N-terminal H-NS protein domain or an H-NS-I119T mutant , i.e. , in the presence of mutant H-NS proteins defective in speciﬁc DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>That is , a carboxyl-terminally truncated form of H-NS , lacking the entire DNA-binding domain , is still able to fully repress bgl expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422614" target="_blank">9422614</a>]<br></li>
<li>It has been well documented that the DNA-binding domain is located in the C terminus of H-NS ( 33 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The paralog has the same DNA-binding domain as H-NS and so is assumed to bind to the same DNA sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Studies using deletion derivatives of H-NS have assigned the DNA-binding domain to the C terminus73 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>H-NS is one of the most abundant DNA-binding proteins in enterobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Residues conserved in the DNA-binding domain of all proteins of the H-NS family are in boldface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Therefore , binding of H-NS to proteins lacking the linker domain ( i.e. H-NSTEPEC ) would reduce the ability of H-NS to oligo-merize and bridge DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Conversely , alleles encoding single amino acid substitutions in the C-terminal DNA-binding domain of H-NS resulted in signiﬁcantly reduced afﬁnity for DNA yet conferred a wild-type ﬁmA promoter inversion frequency , indicating that the mechanism of H-NS activity in modulating promoter inversion is independent of DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Interestingly , even though residues within the DNA-binding domain of H-NS were not required for proper ﬁmA promoter ﬂipping , the entire C-terminal half of the protein was needed in some capacity , as evidenced by the mutant inversion phenotype exhibited by the H-NS truncations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>We postulate that H-NS regulates inversion without binding DNA , possibly through protein-protein interactions with other mol-ecules present at the ﬁmA invertible element , and that the C-terminal domain may be important for overall structural integrity and protein stabilization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Taken together , the effects of hns mutations and deletions on ﬁmA promoter inversion and bacterial motility suggested that the H-NS C-terminal domain may have multiple functions including involvement in protein-protein interactions as well as interactions with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The formation of this structure , in turn , would al!ow a more stable binding of additional H-NS mo!ecules , through long-range DNA-protein or protein-protein interactions , to the downstream site around -50 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Any cooperative interaction between the two binding proteins H-NS and LRP and the common DNA target site might in principle involve protein -- protein interaction and / or a change in the DNA conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>HF-I copurifies with the H-NS DNA-binding protein , and it has been suggested that the two proteins may interact ( Kajitani and Ishihama .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984093" target="_blank">7984093</a>]<br></li>
<li>Our results suggest that the nucleoid-associated DNA-binding protein H-NS ( Hla ) plays a critical role in gene expression and in determining the structure of the genetic material .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2177526" target="_blank">2177526</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_187">
Solution structure of the DNA-binding domain of a nucleoid-associated protein , H-NS , from Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></label>
<input id="dom_similarity_187", type="checkbox">
<div class="DOM_similar">
<li>Solution structure of the DNA-binding domain of a nucleoid-associated protein , H-NS , from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li></div>
<li>The results are consistent with the known difference of the protein domains , considered to be responsible for DNA-binding , which had been characterized for H-NS and LRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_190">
Silencing by the nucleoid-associated DNA-binding protein , H-NS , and activation by insertion elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></label>
<input id="dom_similarity_190", type="checkbox">
<div class="DOM_similar">
<li>Silencing by the nucleoid-asso-ciated DNA-binding protein , H-NS , and activation by insertion elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li></div>
<li>Since the H-NS protein lacks typical DNA-binding motifs it is not yet clear how it recognizes and interaots with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>DNA-binding activity of altered H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>It should be noted that the mutant H-NS protein has not completely lost DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>These data point to a key role for this region of the protein in binding to intrinsically curved DNA , and a core DNA-binding motif , TWTGXGRXP ( where ` X ' is any amino acid ) -- which lies between residues 108 and 116 -- is highly conserved throughout the H-NS protein family16 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Summary The primary sequence of H-NS ( 136 amino acid residues , Mr = 15402 ) , an abundant Escherichia coli DNA-binding protein , has been elucidated and its quaternary structure has been investigated by proteinprotein cross-Mnking reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>Because the derived motif depended on the assumption that high-afinity binding sites are related in sequence we used the motif to predict three potential H-NS binding sites in the ﬁs promoter from positions 150 to +1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Nearly all of these , with the exception of the second ﬂiC peak and ﬂiD , contain the H-NS binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>First , highly transcribable genes contain a higher density of high afinity H-NS binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Figure 4 shows the KMnO4 reactive bases generated by RNP binding in the absence and presence of H-NS ( 228 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>In addition , we demonstrate that H-NS binding exerts a specific biological function , namely to antagonize Fis activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In vivo function of an H-NS binding motif In the absence of other data , the binding of H-NS to a site in vivo , or even the presence of a site , is not necessarily indicative of a biologically relevant role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">H-NS evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>cReported TBLASTN identity to H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>cReported TBLASTN identity to each NAP 53 HU 57 NC 009140 176473 14067 14333 99 HU 15070 148225 15336 148626 57 61 99 99 H-NS H-NS 111747 111346 64 99 query coverage ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>Intriguingly , the N-terminal 90 amino acids of H-NS exhibit 22-29 % identity to structural proteins such as myosin , keratin and dystrophin ( fig IC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The amino acid composition of the C-domain of StpA ( residues 91 -- 134 ) is 70 % identical to that of H-NS ( residues 91 -- 137 ) , while the C-domains of BpH3 ( residues 82 -- 123 ) and HvrA ( residues 61 -- 102 ) are both 40 % identical to that of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>In the present paper , using an approach involving theor-etical analysis of functional and/or structural protein domains as well as in vivo and in vitro experiments , we provide evidence that , despite a low amino acid conservation , proteins such as H-NS and StpA of E. coli , BpH3 of B. pertussis and HvrA of R. capsulatus belong to the same family of structurally and functionally related DNA binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Plasmids containing the hfq gene , which encodes the host factor HF-1 for bacteriophage Qp , or s t p A , which encodes a protein with more than 60 % identity with the protein sequence of H-NS , can comp-lement hns mutants of a d i A ( Shi & Bennett , 1994a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>As expected , StpA , which is a para-logue of H-NS with 58 % amino acid identity ( Zhang and Belfort , 1992 ) , shows a very similar DNA-binding characteristic as H-NS , producing the same large region of DNase I protection ( Fig. 4A , lanes 1 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Recently , an E coli gene ( stpA ) has been identified whose predicted product shares 58 % sequence identity with H-NS [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The high sequence identity ( 58 % ) between StpA ( 15.3 kDa ) and H-NS ( 15.4 kDa ) suggested that the two proteins could have similar functions ( Zhang & Belfort 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>The 15.3-kDa StpA protein of Escherichia coli is 58 % identical to H-NS , a pleiotropic repressor and nucleoid structuring protein ( 14 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>We found a protein , HvrA , that shows a low ( 30 % ) but signiﬁcant similarity to the whole sequence of E. coli H-NS ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Furthermore , its amino-terminus and amino acid composition match that of a 16 kDa DNA-binding protein ( Hla ; Laine et al. 1984 ) , demonstrating the identity of OsmZ with H-NS and Hla .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2177526" target="_blank">2177526</a>]<br></li>
<li>Another protein , StpA , encoded by stpA , shows high sequence identity ( 58 % ) to H-NS , and it was proposed that StpA plays a role as a molecular backup of H-NS ( 30 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>]<br></li>
<li>Moreover , a protein ( StpA ) showing 57 % identity with H-NS has been characterized in E. coli ( Zhang et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>This protein shows approximately 50 % identity with H-NS and with StpA , its paralogous protein in E. coli [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>The StpA protein has 58 % amino acid identity with H-NS , and the gene was found as a multicopy suppressor of the Td2 phenotype of a splicing-de-fective phage T4 td intron mutant ( 42 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Cnu has 71 amino acids and 39 % amino acid identity with the Hha protein , also known to complex with H-NS to downregulate the hemolysin gene of E. coli [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>The amino acid sequence of these proteins shows only 30 % identity with that of H-NS [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>This protein ( ﬁgure 1 ) showed a strong homology , i.e. , approximately 85 % identity , with the amino acid sequence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Identity and similarity scores were calculated by reference to H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Madrid C , Balsalobre C , Garcıa J , Juárez A ( 2007 ) The novel Hha/YmoA family of nucleoid-associated proteins : use of structural mimicry to modulate the activity of the H-NS family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096603" target="_blank">22096603</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Furthermore , this interaction appears to be wide-spread between members of both families : YmoA protein also interacts with Y. enterocolitica H-NS protein ( Nieto et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Evidence for direct protein-protein interaction between members of the enterobacterial Hha/YmoA and H-NS families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_24">
[ 33 ] Nieto , J.M. , Madrid , C. , Miquelay , E. , Parra , J.L. , Rodrıguez , S. and Juárez , A. ( 2002 ) Evidence for direct protein -- protein interaction between members of the enterobacterial Hha/YmoA and H-NS families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></label>
<input id="evo_similarity_24", type="checkbox">
<div class="EVO_similar">
<li>Nieto , J.M. , Madrid , C. , Miquelay , E. , Parra , J.L. , Rodríguez , S. , and Juárez , A. ( 2002 ) Evidence for direct protein -- protein interaction between members of the enterobacterial Hha/YmoA and H-NS families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Nieto , J.M. , Madrid , C. , Miquelay , E. , Parra , J.L. , Rodríguez , S. , and Juárez , A. ( 2002 ) Evidence for direct protein -- pro ¬ tein interaction between members of the enterobacterial Hha/YmoA and H-NS families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li></div>
<li>B A Discussion In this report , we bring further evidence about the interaction of members of the H-NS and Hha families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>The relationship between members of the Hha and H-NS families is not only supported by protein -- protein interaction data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Since then , interacions of other members of the family ( YmoA , YdgT ) with members of the H-NS family have also been shown [ 33,34 ] , and a mutational analysis of the Hha protein has suggested Hha consisting of a unique protein-binding domain [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Additional data supported the model ( Nieto et al. , 2002 ) : ( i ) the mutational analysis of the Hha protein suggested that Hha represents an unique protein-binding domain , ( ii ) a comparison of the amino acid sequence between members of the H-NS and Hha family of proteins suggested that Hha/YmoA are evolved to mimic the H-NS N-terminal domain and ( iii ) the interaction described between YmoA and H-NS in Y. enterocolitica suggested that such kind of protein complexes are conserved in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>One of the most relevant features of proteins of the Hha / YmoA family is their ability to interact with members of the H-NS family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>Other proteins besides the Hha/YmoA family are structurally equivalent to the H-NS N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Recent structural data obtained by our group ( manuscript to be submitted ) support the previous hypothesis : proteins of the Hha family bind the N-terminal domain of members of the H-NS family and are structurally equivalent to truncated H-NS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>However , the ability of H-NS to form heteromeric complexes with paralogous proteins , like StpA , and with members of the more distinct Hha protein family , might represent a mechanism for functional modulation ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The dimerization domain is highly conserved among members of the H-NS family of proteins , making it likely that it also facilitates heteromeric interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>In addition to interactions with closely related family members , H-NS can also form complexes with more distantly related proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>As yet , no genes positively regulated by members of the LysR family have also been shown to be inﬂuenced by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Some transcriptional regulators bind their target sites as monomers ( e.g. , members of the AraC family and MotA from bacteriophage T4 ) ( 11 , 17 ) , or as homomultimers ( e.g. , LacI , H-NS , and the LysR family ) ( 4 , 6 , 34 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>Interplay between H-NS and three different members of the AraC family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>YdgT ( also called Cnu [ 30 ] ) and Hha belong to the H-NS family of proteins ( 33 ) , whose prototypic member H-NS participates in both regulation of transcription initiation and nucleoid structure ( reviewed in references 16 , 20 , 23 , and 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>We recognize H-NS as a new and important member in the family of factors regulating rRNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Examples of members of this family include HU , IHF , H-NS , and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1459953" target="_blank">1459953</a>]<br></li>
<li>Fis is a member of the general family of nucleoid-associated proteins that includes HU , IHF , H-NS , LRP , and Dps ( 2 , 35 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768815" target="_blank">7768815</a>]<br></li>
<li>For example , H. influenzae has just one gene that encodes a member of the H-NS family , and Gram-positive bacteria have none16 ,92 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>YdgT and Hha belong to the H-NS and StpA family of proteins that repress a large number of genes in Gram-negative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_46">
H-NS in Gram-negative bacteria : a family of multifaceted proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></label>
<input id="evo_similarity_46", type="checkbox">
<div class="EVO_similar">
<li>Tendeng , C. , and Bertin , P.N. ( 2003 ) H-NS in Gram-negative bacteria : a family of multifaceted proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li></div>
<li>Conclusions Proteins of the H-NS family are widespread in Gramnegative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>The MvaT family protein is the functional homolog of H-NS in Pseudomonas bacteria [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>Ko , M. , and Park , C. ( 2000 ) H-NS ± dependent regulation of flagellar synthesis is mediated by a LysR family protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Like CfaD and VirF -- its close relatives in the AraC family -- AppY is a link in a regulatory cascade which involves the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Examples are regulatory proteins in the LysR family ( 34 , 48 ) , H-NS ( 29 , 61 ) , and Lrp ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>The size of such regulatory proteins is relatively small ( LysR family , 30 kDa ; H-NS , 16 kDa ; Lrp , 15 kDa ) compared to the length of their protected DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>The bacterial H-NS protein belongs to a family of small nucle-oid-associated proteins that play an important role in the structuring of the bacterial chromosome , a function that is reminiscent of histones in higher organisms ( 6 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>The synthesis of the Y. enterocolitica H-NS protein in the BE1410 mutant partly reversed the loss of motility on semi-solid medium ( ﬁgure 2 ) , as observed for different proteins belonging to the H-NS family [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>IHF is a small heterodimeric protein that belongs to a class of prokaryotic histone-like proteins , together with HU , H-NS and FIS ( Schmid , 1990 ; Schmid & Johnson , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>B himA : + hip : - f-pDRC158 : ¬ IFI'G : 1 -60 - - 4 0 * - 4 0 * - 8 0 * - 2 0 * Discussion IHF belongs to a group of histone-like proteins which , together with HU , H-NS and Fis , appear to participate in the three-dimensional organization of the bacterial chromosome ( Driica and Rouviere-Yaniv , 1987 ; Schmid , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>H-NS and CAP both belong to a regulatory network to which FIS belongs to as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>Like H-NS it belongs to the NAPs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by H-NS</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Edited by : Kunihiko Nishino , Osaka University , Japan Reviewed by : Yuji Morita , Aichi Gakuin University , Japan Hidetada Hirakawa , Gunma University , Japan * Correspondence : Aixin Yan , School of Biological Sciences , The University of Hong Kong , Pokfulam Road , Hong Kong Special Administrative Region , Hong Kong Island , Hong Kong , China e-mail : ayan8@hku.hk Ziqing Deng , Yue Shan , Qing Pan , Xiang Gao and Aixin Yan * School of Biological Sciences , The University of Hong Kong , Hong Kong , China Keywords : multidrug efﬂux pump , acid resistance , anaerobic adaptation , H-NS , ArcBA two-component system , antagonization INTRODUCTION Drug efﬂux constitutes an important mechanism in bacterial drug-resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage-shock protein Regulator ; phage-shock protein Regulator ; phage-shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Genes transiently induced upon cold shock Wild-type 15 ° C / 37 ° C ratio at 5 h Mutant/wild-type ratio ( 15 ° C , 1 h ) Gene product and/or function Wild-type 15 ° C / 37 ° C ratio at 1 h Gene name and role Genes involved in membrane synthesis/function Aerotaxis receptor Short-chain fatty-acid-transporter aer atoE dctA fabB glnH malE , - F , - K , - M manY manZ nupC rbsA-D sanA trg xylF ybeJ 9.8 6.5 0.14 0.5 1.07 1.1 0.73 0.5 0.28 0.39 0.015 0.8 0.28 0.8 2.1 0.08 4.8 3 0.4 0.5 0.22 0.08 0.1 0 DctA protein 3-Oxoacyl - [ acyl-carrier protein ] synthase I Glutamine-binding protein precursor Maltose transport proteins Phosphotransferase system enzyme II PTS-system , mannose-speciﬁc IID component Nucleoside permease NupC Ribose transport proteins SanA protein Methyl-accepting chemotaxis protein III D-Xylose-binding periplasmic protein precursor Amino acid ABC transporter binding protein 7.8 13 6.1 9.0 -- 10 8.9 6.9 9.3 5.0 -- 10 4.3 9.2 20 6.8 1.75 2.6 1 0.5 -- 2.7 1.9 0.3 1.2 1.5 -- 2 2.1 4.1 3.5 1.8 0.6 0.35 0.1 0.1 0.2 0.01 0.09 0.2 0.5 0.6 0.4 0.5 0.03 -- 0.08 0.01 0.13 0.02 0.14 0.03 0.05 -- 0.09 0.02 0.24 0.03 Genes involved in cell metabolism Alcohol dehydrogenase Glucose-1-phosphatase precursor adhE agp aldA aphA aspA bfr carA cfa cpdB cysK deoA fpr fruB fruK fucU fumA gapA glpK lipA malP malQ malT manX mdh nrdD otsA otsB pgi poxB pykA rbsK srlA 18.8 16 15 16.2 20 5 8.5 8.2 10 7 21.8 4.5 7 6.5 6.3 9.3 13.5 12.8 4.2 20 6.3 12 11.4 3.2 15.2 4.3 2.7 6.1 3.6 7.4 5.1 9.3 0.48 2 1.5 0.44 3 1 0.15 1.16 2 1.27 1.02 0.45 0.16 0.75 0.87 0.79 1.2 0.55 0.02 2.5 0.46 0.72 0.7 0.3 1.58 0.12 0.19 0.15 0.6 0.14 0.34 1.3 3.2 2.9 0.45 0.3 0.01 26 0.8 0.1 0.1 0.9 0.15 0.025 0.33 0.31 0.2 0.2 0.22 0.08 0.3 0.02 0.25 0.75 0.075 0.22 0.03 0 0.22 0.29 0.15 0.15 0.02 0.16 0.37 0 0.21 0.15 0.12 0.13 0.06 0.01 0.005 0.005 Aldehyde dehydrogenase Acid phosphatase Aspartate ammonia-lyase Bacterioferritin Carbamoyl-phosphate synthase small chain Cyclopropane fatty acid synthase Cyclic nucleotide 2 ' phosphodiesterase Cysteine synthase Thymidine phosphorylase Ferredoxin-NADP reductase PTS-system , fructose-speciﬁc IIA/FPR component 1-Phosphofructokinase ( fructose 1-phosphate kinase ) Fucose operon FucU protein Fumarate hydratase Glyceraldehyde-3-phosphate dehydrogenase Glycerol kinase Lipoic acid synthetase ( lip-syn ) Maltodextrin phosphorylase 4-Alpha-glucanotransferase MalT regulatory protein Phosphotransferase system enzyme II Malate dehydrogenase Oxygen-sensitive ribonucleoside-triphosphate reductase Tehalose-6-phosphate synthase Trehalose-phosphatase Glucose-6-phosphate isomerase Pyruvate oxidase Pyruvate kinase Ribokinase Phosphoenolpyruvate-carbohydrate phosphotransferase system , glucitol/sorbitol-speci ﬁc IIBC component Phosphotransferase system enzyme II Sorbitol-6-phosphate 2-dehydrogenase Glucitol operon repressor Phosphotransferase system trehalose permease Trehalose-6-phosphate hydrolase Uridine phosphorylase Pyrimidine-speciﬁc nucleoside hydrolase 0.15 2.1 7.3 1 0.9 3.7 1.6 0.6 3.6 1.6 0.8 0.7 2.7 1 2.3 1 1.7 4 3 4.2 2.4 0.28 1.8 1 0.7 1.7 0.18 0.75 2.9 2.7 0.12 0.03 0.16 0.03 0.18 0.015 0.2 0.01 0.13 0.03 0.24 0.05 0.19 0.25 0.16 0.19 0.025 0.04 0 0.04 srlB srlD srlR treB treC udp ybeK 0.22 0.23 0.06 1 1.1 1.5 0.55 4.4 5.1 0.3 1.5 1.5 0.02 0.4 0.01 0.05 0.02 0.2 2.6 0.7 0.45 0.62 1.6 5.5 6 6 10 10.2 0.08 0 Genes encoding proteins with diverse functions 0.1 0.1 1 1 0.3 0.4 4 9 CspA CspB cspA cspB cspG cspl dps grxB hns hobH htpG kbl mdaA mlc mopA mopB ppiA rimJ sseA 0.07 0.05 0.015 0.06 0.5 0.35 0.4 0.2 0.9 0.67 0.21 0.02 0.24 0.03 0.15 0.8 0.7 0.9 1 2.4 2 1.6 1.8 4 2.5 1.3 0.9 1.7 2.7 1.8 0.5 0.03 1.3 0.4 0.8 2 0.5 0.24 0.06 0.29 0.88 1 0.51 0.74 1.2 6 2 10 4.7 4.8 10 4.5 4.8 11.6 6.8 9.3 8 4.6 6.5 7.6 CspG Cspl DNA-binding protein Dps Glutaredoxin 2 DNA-binding protein H-NS DNA binding protein , replication-origin speciﬁc Heat shock protein C62 .5 2-Amino-3-ketobutyrate coenzyme A ligase Modulator of drug activity A Making large colonies protein GroEL protein GroES protein Peptidyl-prolyl-cis-trans-isomerase A precursor Ribosomal-protein-alanine acetyltransferase Putative thiosulfate sulfurtransferase 0.05 0.01 0.5 0.1 0.15 0.07 0.27 0.005 0.13 0.01 0.18 0.04 0.22 0.06 TABLE 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466053" target="_blank">15466053</a>]<br></li>
<li>Genes transcribed more efﬁciently in the presence of EsS Genes and Gene product b promoter regions a Transcription levels ( EsS/Es70 ratio ) d Stress response yieF sE ( 59 ) ; sS ( 19 ) sS , RcsBA ( 60 ) sS , RcsBA ( 60 ) Chromate reductase b3713 N.A. 1.50 1.51 osmB Intergenic region including osmB promoter Osmolarity-inducible lipoprotein b1283 b2688 1.51 2.21 gshA g-glutamate -- cysteine ligase ( glutathione-biosynthesis ) ahpF Alkyl-peroxidase reductase pphA Protein phosphatase macA Macrolide resistance efﬂux pump DNA and RNA-metabolism and modiﬁcation ppk Polyphosphate kinase , part of RNA degradosome , Intergenic region including xapA promoter xapA Xanthosine phosphorylase , nucleotide-synthesis/degradation gyrB B subunit of DNA gyrase recT Rac prophage , recombinase , in recET-lar-ydaCQ operon lar Rac prophage , restriction alleviation and modiﬁcation enhancement , in recET-lar-ydaCQ operon ydaC Rac prophage , in recET-lar-ydaCQ operon ydaQ Rac prophage , possible recombinase , in recET-lar-ydaCQ operon Polyamine-metabolism puuB g-glutamyl -- putrescine oxidase gabD Succinate semialdehyde dehydrogenase speB Agmatinase ( in arginine/putrescine-degradation pathway ) Transcription regulation Intergenic region including appY promoter appY Regulator of hyaABCDEF operon yciT Putative deoR-type transcription regulator ilvY Regulator of ilvC operon rhaS Regulator of rhamnose-transport Sugar metabolism treF Trehalase OxyR ( 61 ) sH ( 62,63 ) b0606 b1838 b0878 1.52 1.54 1.54 Regulator of intracellular sS concentrations ( 3 ) XapR ( 64 ) XapR ( 64 ) b2501 1.50 N.A. b2407 1.62 1.56 b3699 b1349 1.65 1.93 Fis ( 65 ) sS-Dependent in bioﬁlm cells ( 18 ) - growing b1348 1.93 b1347 b1346 1.74 1.66 as recT as recT sS ( 19 ) b1301 b2661 b2937 1.61 1.65 1.66 sS , Nac ( 9,66 ) 1.69 1.51 1.56 1.75 1.81 N.A. b0564 b1284 b3773 b3905 H-NS ( 67 ) H-NS ( 67 ) CRP ( 29 ) ; RhaS ( 68 ) sS ( 9,19 ) sS ( 19 ) sS ( 7 ) 1.53 1.55 1.56 1.57 b3519 b4230 b1897 b1901 ytfT Putative galactose ABC transporter otsB Trehalose-6-phosphate phosphatise araF Arabinose transporter , component of an ABC transport system glgP Glycogen phosphorylase , part of glgCAP operon gpmM Putative 2,3-bisphosphoglycerate mutase glvC Sugar transport phosphotransferase Other metabolic functions Intergenic region including hyaABCDEF promoter hyaA Hydrogenase small subunit ; in hyaABCDEF operon ( 69 ) ; sS ( 19 ) CRP , AraC CRP ( 70 ) ; sS controls glgCAP operon ( 19 ) 1.60 b3428 1.61 1.79 b3612 b3683 2.02 N.A. sS , AppY , anaerobic regulation by ArcA and NarP/NarL ( 20,71,72 ) as hyaA 1.68 b0972 Hydrogenase large subunit ; in hyaABCDEF operon Hydrogenase subunit ( nickel-binding protein ) ; in hyaABCDEF operon SecY-interacting protein Acetyl-muramic acid permease Transcriptional repressor of murQP Trans-aconitate methylatransferase Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon hyaB 1.66 b0973 as hyaA hyaF 1.50 b0977 syd 1.59 1.59 1.62 1.60 b2793 b2429 b2427 b1519 murP murR tam CRP , MurR ( 73 ) sS ( 74 ) ydhY FNR , NarL ( 75 ) 1.80 b1674 FNR , NarL ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Effect of genes whose inactivation prevented repression of hdeAB by M arA on MarA-mediated MD R Level of MarA-mediated MD a level ) R ( % of parental Locusb Gene Protein/function FOX MIN NOR CHL Transcriptional regulators Cold shock protein CspG cspGpc crp hns ompR b0990 b3357 b1237 b3405 54 21 19 67 91 33 23 77 93 29 20 93 75 38 Cyclic AMP receptor protein DNA-binding protein H-NS OmpR transcriptional dual regulator 35 100 Enzymes Acid phosphatase appA cyaA degP maoC metL pcnB recD treC ynfE b0980 b3806 b0161 b1387 b3940 b0143 b2819 b4239 b1587 68 43 84 41 91 61 91 23 77 46 68 93 46 100 32 100 40 100 Adenylate cyclase Serine protease Do Putative ring-cleavage enzyme of phenylacetate-degradation Aspartate kinase/homoserine dehydrogenase Poly ( A ) polymerase I ; RNA modiﬁcation RecBCD DNA helicase/exonuclease Trehalose-6-phosphate hydrolase Oxidoreductase subunit paralog of DmsA 10 80 94 94 50 75 100 33 50 85 93 Transport acrA acrB mhpT nikD tolC b0463 b0462 b0353 b3479 b3035 16 6 68 68 0 8 8 40 40 0 9 AcrAB-TolC multidrug efﬂux system AcrAB-TolC multidrug efﬂux system Putative propionic acid transporter Subunit of nickel ABC transporter TolC outer membrane channel 100 50 0 100 125 40 91 68 0 Others Putative membrane protein alx b3088 b3388 b2816 b2350 120 40 74 68 68 46 91 91 100 50 75 93 65 30 65 65 Predicted membrane-anchored protein tRNA-methionine CPS-53 ( KpLE1 ) prophage ; bactoprenol-linked glucose translocase ( ﬂippase ) Conserved protein Predicted unconserved protein damX metV yfdG yibLpc yniDpc b3602 b4535 68 94 68 77 50 100 50 125 a Abbreviations : FOX , cefoxitin ; NOR , norﬂoxacin ; CHL , chloramphenicol ; MIN , minocycline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>of exogenous and endogenous hydrophobic electrophiles Involved in ﬁrst step of GMP-biosynthesis from IMP ; induced by low pH during anaerobic growth Catalyzes the irreversible NADPH-dependent deamination of GMP to IMP ; functions in the conversion of nucleobase , nucleoside , and nucleotide derivatives of G-to-A nucleotides and in maintaining the intracellular balance of A and G nucleotides DNA gyrase negatively supercoils closed circular double-stranded DNA in an ATP-dependent manner and also catalyzes the interconversion of other topological isomers of double-stranded DNA rings , including catenanes and knotted rings See GyrA Uses temp-induced exposure of structured hydrophobic domains to capture and stabilize early unfolding protein intermediates under severe thermal stress ; rapidly releases them once stress has abated and is induced by heat shock Induced by low pH during anaerobic growth and at the stationary phase Periplasmic protein that may be involved in acid resistance ; enhanced at the stationary phase and in the acnB mutant but repressed by H-NS 7-Alpha-dehydroxylation of cholic acid , yielding deoxycholic acid and lithocholic acid , respectively ; highest afﬁnity with taurochenodeoxycholic acid Involved in porphyrin-biosynthesis HdeA 4.68 / 9,740.91 4.56 / 10,724 4.84 / 9,424 ( 4 -- 5 ) P0AES9 Protein 103 , 179 , 228 , 287 , 294 , 323 HdeB 4.94 / 9,065.24 4.85 / 11,285 5.00 / 9,588 ( 4.5 -- 5.5 ) P0AET2 Protein 103 , 179 , 228 , 278 , 287 , 294 HdhA 5.22 / 26,778.59 5.17 / 25,083 P0AET8 7-Alpha-hydroxysteroid dehydrogenase 179 , 286 321 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.25 / 35,493.60 HemB P0ACB2 Delta-aminolevulinic acid dehydratase P06983 Porphobilinogen deaminase Tetrapolymerization of the monopyrrole porphobilinogen into the hydroxymethylbilane preuroporphyrinogen in several discrete steps Involved in porphyrin-biosynthesis Involved in siroheme and cobalamin-biosynthesis Seems to act as an ATP-dependent zinc metallopeptidase ; involved in the degradation of 32 5.31 / 33,851.83 HemC 179 179 179 5.88 / 39,248.12 4.68 / 42,963.02 P29680 P09127 HemE HemX 5.91 / 70,708.09 P0AAI3 HﬂB 179 Continued on following pag HﬂC and HﬂK govern the stability of phage lambda cII protein and have been proposed to encode or regulate a cII-speciﬁc protease RNA-binding protein that stimulates the elongation of poly ( A ) tails and targets several mRNAs for degradation , possibly by increasing polyadenylation or by interfering with ribosome binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>hdeB Periplasmic chaperone of acid-denatured proteins hdeA Periplasmic chaperone of acid-denatured proteins hdeD Putative membrane transporter , H-NS repressed gadE Required for stationary phase-induced , pH 5.5 growth medium-induced , and EvgA-induced acid resistance mdtE MdtEF-TolC multidrug-resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine mdtF MdtEF-TolC multidrug-resistance efflux RND-type transporter ; overexpression resistance to erythromycin , deoxycholate , octane and rhodamine gadW Positive AraC-type regulator of gadA and gadBC , in absence of gadX ; repressor of gadX gadY sRNA regulates gadX and gadY gadX Positive transcriptional regulator of gadA and gadBC ; repressed by HNS and GadW ; stimulated by RpoS gadA Glutamate decarboxylase isozyme GadB operon : 1567 -- 1570 kb gadB Glutamate decarboxylase isozyme gadC Glutamate transporter 1.03 0.95 3.17 -1.59 1.58 2.58 -0.98 1.60 2.70 -1.17 1.53 3.17 2.66 -1.38 -1.47 1.80 1.19 4.89 -3.46 1.43 3.21 3.73 -2.50 -2.44 1.29 Table 4 : Drug resistance , cytoskeletal and ion-transport proteins showing pH dependence under oxygen limitation Gene Function Log2pH ratio Acid up-regulated pH 5.7 / pH 7.0 pH 7.0 / pH 8.5 pH 5.7 / pH 8.5 0.80 0.60 0.86 1.31 0.44 chaB chaC Regulator of Na + - Ca + / H + antiporter Regulator of Na + - Ca + / H + antiporter -0.38 -0.56 -0.4 copA cueR cusC feoA feoB marR1 mdtE mdtF mdtG1 mdtI1 mdtJ1 mdtL mgtA nhaA nhaB1 nikA nikB nikC nikD nikE prc Copper - , silver-translocating P-type ATPase Activator of copper-responsive regulon genes cueO and copA Silver and copper efflux ferrous-iron-transport protein A ferrous-iron-transport protein B multiple antibiotic resistance protein MdtEF-TolC multidrug-resistance efflux transporter MdtEF-TolC multidrug-resistance efflux Multidrug-resistance efflux transporter multidrug-transport protein multidrug-transport protein Multidrug-resistance efflux protein Mg2 + transport Na + / H + antiporter Na + / H + antiporter periplasmic binding protein for nickel nickel transport nickel transport ATP-binding protein of nickel transport system ATP-binding protein of nickel transport system carboxy-terminal protease for penicillin-binding protein 3 0.91 0.63 1.22 0.53 0.82 0.41 0.31 -1.59 -0.98 3.17 2.58 0.16 1.58 1.60 0.25 1.16 3.02 0.57 0.63 1.00 0.37 0.84 0.56 0.86 0.74 0.49 0.37 1.16 2.91 0.48 1.93 -0.93 0.41 0.7 0.50 0.70 0.60 0.45 0.32 Base up-regulated -0.34 -0.35 -0.69 -0.42 -1.22 -0.52 -0.12 -0.66 -0.66 -0.71 -0.51 -0.66 -0.58 -0.53 -1.1 -0.39 -0.3 -0.65 acrA acrB acridine efflux pump acridine efflux pump -0.93 acrR2 alx2 ampC atpA2 atpC2 atpD2 atpF2 atpG2 atpH2 atpI2 chaA corA cutA dacA dacC dacD emrA1 mepA1 mreB2 mreC napF napG napH acrAB operon repressor Putative metal transporter , homolog of terC ( tellurium resistance ) beta-lactamase ; penicillin resistance ; penicillin-binding protein ATP synthase , F1 alpha-subunit ATP synthase , F1 epsilon-subunit ATP synthase , F1 beta-subunit ATP synthase , F0 subunit b ATP synthase , F1 gamma-subunit ATP synthase , F1 delta-subunit ATP synthase subunit Na + - Ca + / H + antiporter Mg2 + transport , system I divalent cation tolerance protein D-alanyl-D-alanine carboxypeptidase , fraction A ; penicillin-binding protein 5 D-alanyl-D-alanine carboxypeptidase ; penicillin-binding protein 6 penicillin binding protein 6b multidrug-resistance secretion protein murein DD-endopeptidase , penicillin-insensitive regulator of ftsI , penicillin binding protein 3 , septation function rod shape-determining protein Fe-S ferredoxin-type protein ferredoxin-type protein ferredoxin-type protein -0.57 -0.24 -0.36 -0.46 -0.35 -0.42 -0.86 0.10 -0.27 -0.58 0.87 -1.22 -0.64 -0.68 -0.53 -0.29 -0.59 -0.39 -0.37 -1.16 -0.90 -0.11 -0.19 -0.44 -0.39 -0.14 -0.20 -0.20 1Acid up-regulated with aeration [ 2 ] 2Base up-regulated with aeration [ 2 ] Real-time PCR expression ratios of selected genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate-biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>ﬂuoride or Pefabloc SC ; BCA , bicinchoninic acid ; delta Cn , correlation value ( difference between the ﬁrst hit and the second hit ) ; DTE , dithioerythritol ; DTT , dithiothreitol ; iTRAQ , a multiplexed set of isobaric reagents that yield amine-derivatized peptides ( iTRAQ reagents ; Applied Biosystems , CA ) ( 253 ) ; PMSF , phenylmethylsulfonyl ﬂuoride ; RSp , rank preliminary score ; SELDI-TOF-MS , surface-enhanced laser desorption ionization -- time of ﬂight mass spectrometry ; TCA , trichloroacetic acid ; Xcorr , cross-correlation ( measures how close the spectrum ﬁts to the ideal spectrum ) Biotin carboxyl carrier protein of acetyl-CoA carboxylase AccB P0ABD8 179 , 286 , 287 , 323 4.66 / 16,687.21 4.57 / 22,029 4.74 / 18,338 ( 4 -- 5 ) 5.01 / 18,459 ( 4 -- 5 ) 4.78 / 15,809 ( 4 -- 5 ) Biotin carboxylase 6.65 / 49,320.74 7.58 / 33,321.89 P24182 P0A9Q5 AccC AccD 179 179 Acetyl-CoA carboxylase carboxyl transferase subunit beta Isocitrate lyase Protein function and expressione Acetyl-CoA carboxylase is a heterodecamer of four copies of biotin carboxyl carrier protein ( AccB ) , two copies of biotin carboxylase ( AccC ) , and two copies each of the two subunits of carboxyl transferase ( AccA and AccD ) ; AccC catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA See AccA ; induced by high pH during anaerobic growth 4.57 / 22,029 4.74 / 18,338 ( 4 -- 5 ) 5.01 / 18,459 ( 4 -- 5 ) 4.78 / 15,809 ( 4 -- 5 ) See AccA See AccA 6.65 / 49,320.74 7.58 / 33,321.89 P24182 P0A9Q5 AccC AccD 179 179 Involved in glyoxylate-bypass ; activated by phosphorylation ( on histidine ) and inhibited by phosphoenolpyruvate ( PEP ) , 3-phosphoglycerate , and succinate ; induced by pH change ; increases in the physiological short-term adaptation to glucose limitation The pyruvate dehydrogenase complex contains multiple copies of three enzymatic components : pyruvate dehydrogenase ( E1 ) , dihydrolipoamide acetyltransferase ( E2 ) , and lipoamide dehydrogenase ( E3 ) ; catalyzes the overall conversion of pyruvate to acetyl-CoA and CO2 ; increases during aerobic growth and after benzoic acid treatment See AceE ; increases during aerobic growth , during the low temp at 10 ° C , and after benzoic acid treatment Involved in the activation of acetate to acetyl-CoA and the secretion of acetate ; involved in the synthesis of most of the ATP formed catabolically during anaerobic growth ; induced by low pH during anaerobic growth Aconitate hydratases ( AcnA and AcnB ) serve as a protective buffer against the basal level of oxidative stress that accompanies aerobic growth by acting as a sink for reactive-oxygen-species and by modulating translation of the sodA transcript ; AcnA enhances the stability of the sodA transcript , whereas AcnB lowers its stability ; it is induced by iron and oxidative stress and may have an ironresponsive regulatory function See AcnA AceA 27 , 179 , 287 , 311 5.16 / 47,521.57 P0A9G6 5.19 / 44,130 ( 4.5 -- 5.5 ) 5.12 / 44,130 ( 4.5 -- 5.5 ) 5.01 / 33,761 ( 4.5 -- 5.5 ) 4.99 / 33,624 ( 4.5 -- 5.5 ) 5.01 / 51,702 ( 5 -- 6 ) AceE 5.40 / 99,289 5.46 / 99,537.29 P0AFG8 Pyruvate dehydrogenase E1 component 179 , 228 , 270 , 294 , 321 5.39 / 94,195 ( DIGE 4.5 -- 6.5 ) AceF Dihydrolipoyllysine residue acetyltransferase component of pyruvate dehydrogenase complex Acetate kinase P06959 133 , 179 , 228 , 270 , 294 , 321 5.01 / 77,450 5.09 / 65,964.87 5.09 / 78,105 ( DIGE 4.5 -- 6.5 ) AckA 5.84 / 44,001 5.85 / 43,290.45 P0A6A3 179 , 228 , 294 , 287 , 321 , 323 5.79 / 41,417 ( DIGE 4.5 -- 6.5 ) 5.72 / 31,846 ( 5 -- 6 ) 5.59 / 97,516.00 Aconitate hydratase 1 P25516 AcnA 151 , 278 AcnB 151 , 179 , 278 , 287 , 321 Aconitate hydratase 2 P36683 5.30 / 95,000 ( DIGE 4.5 -- 6.5 ) 5.24 / 75,907 ( 4.5 -- 5.5 ) 5.24 / 93,498.11 Continued on facing pag TABLE 2 -- Continued pI/MW Descriptionb Protein function and expressione Experimentald Theoreticalc AcpP P0A6A8 Acyl carrier protein 3.98 / 8,508.33 A carrier of the growing fatty acid chain in fatty-acid-biosynthesis ; even though AcpP is an abundant protein ( ca. 60,000 copies per cell ) , its small size ( 8.5 kDa ) makes it difﬁcult to detect by typical 2-DE ; it can be detected after subcellular fractionation AcrAB is a drug efﬂux protein with broad substrate speciﬁcity 1 Acriﬂavine resistance protein A Adenosine deaminase Adenine deaminase AcrA P0AE06 179 6.08 / 39,723.72 Add P22333 Ade P31441 ( YicP ) 5.36 / 36,397.46 5.63 / 34,723.76 179 321 5.22 / 61,539 ( DIGE 4.5 -- 6.5 ) Repressed by H-NS but activated by insertion of a variety of insertion elements into a region extending from 145 to 13 relative to the transcription start site ; decreases after benzoic acid treatment AdhE has three activities : alcohol dehydrogenase , acetaldehyde dehydrogenase , and PFL deactivase ; PFL deactivase activity catalyzes the quenching of the pyruvate-formate-lyase catalyst in an iron - , NAD - , and CoA-dependent reaction ; in aerobic conditions , it acts as a hydrogen peroxide scavenger ; induced under anaerobic conditions in the absence of nitrate Catalyzes the reversible transfer of the terminal phosphate group between ATP and AMP ; essential for maintenance and cell-growth ; increases during phosphate limitation Absolutely required for the growth of E. coli in a high-phosphate medium containing glucose-1-phosphate as the sole carbon source ; positively controlled by cAMP-CRP Directly reduces organic hydroperoxides in its reduced dithiol form ; may be directly degraded by ClpXP or modulated by a protease-dependent mechanism ; induced by oxidative stress but repressed by sulfate or cysteine ; increases during high-cell-density cultivation and acid or propionate condition during aerobic growth 6.33 / 95,996.05 Aldehyde-alcohol dehydrogenase P0A9Q7 AdhE 179 Adk 5.49 / 28,491 5.60 / 22,470 5.55 / 23,586.02 Adenylate kinase P69441 179 , 228 , 287 , 293 5.00 / 9,929 ( 4 -- 5 ) 4.99 / 9,991 ( 4.5 -- 5.5 ) 5.48 / 31,661 ( 5 -- 6 ) 5.38 / 43,560.36 Glucose-1-phosphatase P19926 Agp 79 1 AhpC Alkyl hydroperoxide reductase subunit C P0AE08 5.01 / 21,554 5.03 / 20,630.25 27 , 179 , 228 , 266 , 287 , 294 , 308 , 321 , 325 5.09 / 23,123 ( DIGE 4.5 -- 6.5 ) 5.08 / 21,705 ( 4.5 -- 5.5 ) 5.03 / 22,439 ( 4.5 -- 5.5 ) AhpF Serves to protect the cell against DNA damage by alkyl hydroperoxides ; can use either NADH or NADPH as electron donor for direct reduction of redox dyes or of alkyl hydroperoxides when combined with the AhpC protein ; induced by oxidative stress and after benzoic acid treatment 5.44 / 53,485 ( DIGE 4.5 -- 6.5 ) 5.20 / 41,503 ( 5 -- 6 ) 5.47 / 56,177.11 Alkyl hydroperoxide reductase subunit F P35340 179 , 203 , 266 , 287 , 321 Alanyl -- tRNA synthetase Aldehyde dehydrogenase A AlaS 29 , 179 , 228 , 294 5.53 / 96,032.40 5.50 / 96,123 P00957 AldA Acts on lactaldehyde as well as other aldehydes ; induced by growth on fucose , rhamnose , arabinose , and amino acids such as glutamate ; increases in the physiological shortterm adaptation to glucose limitation 5.11 / 52,363 ( DIGE 4.5 -- 6.5 ) 5.29 / 42,547 ( 4.5 -- 5.5 ) 5.53 / 38,913 ( 4.5 -- 5.5 ) 5.43 / 50,034 ( 5 -- 6 ) 5.07 / 52,141.60 P25553 179 , 287 , 308 , 311 , 321 , 325 Continued on following pag Involved in regulation of AMP concentrations ; allosterically activated by Mg-ATP but inactivated by inorganic phosphate ; induced by cAMP at limiting phosphate concentrations Serine beta-lactamase with substrate speciﬁcity for cephalosporins AnsA converts L-asparagine into L-aspartate ; there are two L-asparaginase isoenzymes : L-asparaginase I ( AnsA ) , a low-afﬁnity enzyme located in the cytoplasm , and L-asparaginase II ( AnsB ) , a high-afﬁnity secreted enzyme See AnsA ; induced by cAMP and anaerobiosis It is induced during entry into the stationary phase ; its synthesis is triggered by carbon starvation , phosphate starvation , osmotic shift , or a shift from aerobic to anaerobic conditions ; controlled by S and AppY Catalyzes a purine-salvage reaction , resulting in the formation of AMP that is energically less costly than de novo synthesis Involved in L-arabinose-catabolism 293 Amn P0AE12 AMP nucleosidase 5.90 / 53,994.91 6.38 / 47,693 87 Beta-lactamase AmpC P00811 8.78 / 39,551.14 9.06 / 43,647 ( 6 -- 11 ) 2 179 AnsA P0A962 L-Asparaginase I 5.52 / 37,127.24 AnsB P00805 5.66 / 34,593.94 L-Asparaginase II 179 6.11 / 44,689.86 5.42 / 46,694 Periplasmic AppA protein AppA P07102 13 , 228 , 293 Apt 5.30 / 23,792 ( D 5.26 / 19,858.88 Adenine P69503 IG E 4.5 -- 6.5 ) 179 , 321 phosphoribosyltransferase 5.73 / 25,518.91 6.06 / 29,165 L-Ribulose-5-phosphate 4-epimerase L-Arabinose-binding periplasmic protein AraD P08203 294 Involved in high-afﬁnity L-arabinose membrane transport system ; binds with high afﬁnity to arabinose but can also bind D-galactose ( approx 2-fold reduction ) and D-fucose ( approx 40-fold reduction ) ArcA/ArcB two-component regulatory system represses a wide variety of aerobic enzymes under anaerobic conditions , controls the resistance of E. coli to dyes , and may be involved in the osmoregulation of envelope proteins ; when activated by arcB , it negatively regulates the expression of genes of aerobic function ; it activates the transcription of the plfB operon by binding to its promoter Involved in both the arginine and lysine biosynthetic pathways 5.61 / 33,210.05 AraF P02924 179 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 5.21 / 27,292.02 P0A9Q1 Aerobic respiration control protein ArcA ArcA 125 , 321 Acetylornithine / ArgD P18335 287 , 321 5.75 / 41,241 ( DIGE 4.5 -- 6.5 ) 5.19 / 33,559 ( 5 -- 6 ) 5.80 / 43,635.78 succinyldiaminopimelate aminotransferase Ornithine carbamoyltransferase chain F Ornithine carbamoyltransferase consists of ArgF and ArgI ; involved in ﬁrst step of arginine biosynthetic pathway Involved in seventh step of arginine biosynthetic pathway 5.73 / 30,486 ( 5 -- 6 ) 5.86 / 29,835 ( 5.5 -- 6.7 ) 5.63 / 36,695.95 P06960 ArgF 287 ArgG 179 , 287 , 321 5.27 / 46,667 ( DIGE 4.5 -- 6.5 ) 5.28 / 46,209 ( 4.5 -- 5.5 ) 5.08 / 51,667 ( 5 -- 6 ) 5.43 / 39,776 5.23 / 49,767.20 P0A6E4 Argininosuccinate synthase See ArgF ; repressed during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases its level with increasing growth rate Involved in an arginine-transport system ; increases in the physiological short-term adaptation to glucose limitation ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism P0A6E4 Argininosuccinate synthase 5.46 / 36,775.92 Ornithine carbamoyltransferase chain I Arginyl-tRNA synthetase 228 , 293 P04391 ArgI P11875 ArgS 29 , 109 , 228 , 230 , 294 5.32 / 60,405 5.32 / 64,682.96 ArgT 5.15 / 26,055 ( DIGE 4.5 -- 6.5 ) 5.17 / 26,881 ( 4.5 -- 5.5 ) 5.22 / 25,784.99 Lysine-arginine-ornithine-binding periplasmic protein P09551 179 , 287 , 308 , 311 , 321 Continued on facing pag Involved in sixth step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP See AroA : seventh ( ﬁnal ) step See AroA : third step Stereospeciﬁc condensation of PEP and D-erythrose-4-phosphate , giving rise to 3-deoxy-D-arabino - heptulosonate-7-phosphate ( DAHP ) ; involved in ﬁrst step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP See AroF : ﬁrst step AroA P0A6D3 3-Phosphoshikimate 1-carboxyvinyltransferase 5.37 / 46,095.78 5.34 / 53,476 ( 5 -- 6 ) 5.23 / 50,990 ( 5 -- 6 ) Chorismate synthase P12008 P05194 P00888 AroC AroD AroF 179 5.82 / 39,006.25 5.19 / 27,466.65 5.42 / 38,803.96 3-Dehydroquinate dehydratase Phospho-2-dehydro-3-deoxyheptonate aldolase , Tyr sensitive 179 , 287 321 5.31 / 27,332 ( 4.5 -- 5.5 ) 5.38 / 37,715 ( DIGE 4.5 -- 6.5 ) AroG 179 , 286 Phospho-2-dehydro-3 - P0AB91 6.14 / 38,009.52 6.12 / 39,462 6.13 / 38,005 deoxyheptonate aldolase , Phe sensitive Shikimate kinase I 179 , 274 , 286 , 287 , 321 Involved in ﬁfth step of chorismate-biosynthesis from D-erythrose 4-phosphate and PEP ; induced by high pH Involved in the arginine periplasmic transport system ( artPIQMJ gene products ) ; increases after benzoic acid treatment See ArtI ; decreases after benzoic acid treatment See ArtI ; probably responsible for energy coupling to the transport system Involved in amino-acid-biosynthetic pathway : Lys , Met , and Thr Involved in asparagine-biosynthesis ; decreases after benzoic acid treatment Involved in asparagine-biosynthesis AroK 5.26 / 19,406.85 P0A6D7 5.30 / 17,959 5.30 / 19,078 ( DIGE 4.5 -- 6.5 ) 5.18 / 21,250 ( 5 -- 6 ) ArtI 5.32 / 25,042.20 Arginine-binding periplasmic protein 1 P30859 179 , 286 , 287 , 321 5.24 / 26,625 5.27 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.07 / 31,478 ( 5 -- 6 ) 5.94 / 25,009 ( DIGE 4.5 -- 6.5 ) 5.59 / 24,831 ( DIGE 4.5 -- 6.5 ) ArtJ 5.97 / 24,908.11 Arginine-binding periplasmic protein 2 Arginine transport ATP-binding protein ArtP P30860 321 6.17 / 27,022.05 P0AAF6 ArtP 179 Asd Aspartate-semialdehyde dehydrogenase 5.37 / 40,017.88 5.20 / 50,339 ( 5 -- 6 ) 179 , 287 P0A9Q9 AsnA 5.45 / 36,650.55 Aspartate-ammonia ligase P00963 179 , 321 5.45 / 37,876 ( DIGE 4.5 -- 6.5 ) 5.38 / 37,715 ( DIGE 4.5 -- 6.5 ) 5.55 / 62,527.82 AsnB P22106 Asparagine synthetase B ( glutamine hydrolyzing ) P0A8M0 Asparaginyl-tRNA synthetase 179 5.64 / 92,858 5.17 / 52,439.25 AsnS 228 , 236 , 287 , 294 , 321 4.69 / 50,800 ( DIGE 4.5 -- 6.5 ) 5.01 / 38,992 ( 4.5 -- 5.5 ) 5.19 / 52,356.13 5.54 / 43,573.36 Aspartate ammonia-lyase Aspartate aminotransferase P0AC38 P00509 AspA AspC 179 5.40 / 41,960 5.52 / 41,877 179 , 228 , 236 , 287 , 294 5.53 / 50,441 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.42 / 61,614 Changes very little at normal temp range ( 23 -- 37 ° C ) Involved in short-chain-fatty-acid-metabolism ; induced during phosphate limitation and at phosphonate growth F-type ATPases consist of the two complex components CF ( 0 ) , the membrane proton channel , and CF ( 1 ) , the catalytic core ; CF ( 1 ) has ﬁve subunits : alpha ( 3 ) , beta ( 3 ) , gamma ( 1 ) ; delta ( 1 ) , and epsilon ( 1 ) ; CF ( 0 ) has three main subunits : a , b , and c ; produces ATP from ADP in the presence of a proton gradient across the membrane ; increases in the physiological short-term adaptation to glucose limitation See AtpA 5.53 / 50,441 ( 5 -- 6 ) 5.56 / 49,421 ( 5 -- 6 ) 5.42 / 61,614 5.47 / 65,913.45 29 , 109 , 228 , 294 Aspartyl-tRNA synthetase P21889 AspS Acetate CoA-transferase beta subunit P76459 AtoA 5.65 / 22,959.65 5.41 / 22,852 293 AtpA 5.98 / 43,053 5.84 / 53,108 5.80 / 55,222.08 P0ABB0 ATP synthase alpha chain 179 , 228 , 286 , 287 , 294 , 311 , 321 5.82 / 52,204 ( DIGE 4.5 -- 6.5 ) 5.15 / 39,861 ( DIGE 4.5 -- 6.5 ) 5.81 / 36,727 ( 5 -- 6 ) 6.11 / 57,637 ( 6 -- 11 ) 6.02 / 30,657 ( 6 -- 11 ) AtpC 179 , 228 , 287 , 294 5.46 / 14,937.07 5.48 / 14,811 5.34 / 10,379 ( 4.5 -- 5.5 ) 5.45 / 13,773 ( 5 -- 6 ) ATP synthase epsilon chain P0A6E6 Continued on following pag See AtpA ; decreases during phosphate limitation AtpD P0ABB4 ATP synthase beta chain 4.90 / 50,194.23 4.90 / 47,721 4.96 / 48,103 ( DIGE 4.5 -- 6.5 ) 4.99 / 48,103 ( DIGE 4.5 -- 6.5 ) 4.95 / 43,643 ( 4 -- 5 ) 4.95 / 47,782 ( 4 -- 5 ) 4.55 / 43,745 ( 4 -- 5 ) 5.00 / 45,494 ( 4.5 -- 5.5 ) 4.96 / 42,894 ( 4.5 -- 5.5 ) 5.53 / 19,289 ( DIGE 4.5 -- 6.5 ) 5.53 / 18,722 ( DIGE 4.5 -- 6.5 ) AtpF P0ABA0 ATP synthase B chain 5.99 / 17,263.96 See AtpA ; decreases after benzoic acid treatment See AtpA See AtpA 179 , 321 P0ABA6 ATP synthase gamma chain P0ABA4 ATP synthase delta chain P0AE52 Putative peroxiredoxin Bcp AtpG AtpH Bcp 8.84 / 31,577.42 4.94 / 19,332.22 5.03 / 17,633.94 179 179 5.02 / 15,872 4.92 / 16,060 179 , 228 , 286 , 294 , 321 5.10 / 17,526 ( DIGE 4.5 -- 6.5 ) 5.34 / 16,350 May perform functions in iron detoxiﬁcation and storage analogous to those of animal ferritins P0ABD3 Bacterioferritin Bfr 4.69 / 18,495.03 294 5.77 / 81,406.5 5.32 / 38,648.09 Periplasmic beta-glucosidase Biotin synthase P33363 P12996 BglX BioB 179 179 Involved in ﬁnal step of biotin-biosynthesis : biotin from 6-carboxyhexanoyl-CoA See BioB : third step Involved in the active translocation of vitamin B12 ( cyanocobalamin ) across the outer membrane to the periplasmic space ; derives its energy for transport by interacting with the transperiplasmic membrane protein TonB May be an auxiliary component of the transport system Involved in amino acid biosynthesis , L-arginine-biosynthesis , and synthesis of carbamoyl phosphate from HCO3 ; decreases during phosphate limitation early See CarA ; decreases signiﬁcantly during phosphate limitation Scavenges exogenous and endogenous cytidine and 2-deoxycytidine for UMP-synthesis ; increases during high-cell-density cultivation and overexpression of recombinant protein in large scale Involved in the transmission of sensory signals from the chemoreceptors to the ﬂagellar motors ; autophosphorylated ; can transfer its phosphate group to either CheB or CheY ; a histidine kinase See CheA ; the active CheY ( phosphorylated or acetylated form ) exhibits enhanced binding to a switch component , FliM , at the ﬂagellar motor , which induces a change from counterclockwise to clockwise ﬂagellar rotation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>More than 30 proteins are altered in the hfq null mutant ; some of these proteins are Sdependent genes Involved in fourth step of Lhistidine from 5-phospho-alpha-D-ribose 1-diphosphate P0ABC3 Protein HﬂC 6.3 / 37,649.88 179 Hfq 204 P0A6X3 Protein 7.20 / 11,035.19 HisA P10371 4.94 / 26,032.65 1 - ( 5-Phosphoribosyl ) -5 - ( 5-phosphoribosylamino ) methylideneamino imidazole-4-carboxamide isomerase Histidine biosynthesis bifunctional protein Histidinol-phosphate aminotransferase Histidinol dehydrogenase 4.99 / 24,831 ( DIGE 4.5 -- 6.5 ) 5.46 / 23,581 ( DIGE 4.5 -- 6.5 ) 179 , 321 5.76 / 40,277.96 5.43 / 35,112 See HisA : steps 6 and 8 286 P06987 HisB See HisA : step 7 ; induced by high pH during anaerobic growth See HisA : ﬁnal step ; catalyzes the sequential NAD-dependent oxidations of L-histidinol to Lhistidinaldehyde and then to L-histidine ; induced by high pH during anaerobic growth See HisA : step 5 5.01 / 39,360.14 P06986 HisC 323 HisD P06988 5.06 / 45,979.15 5.07 / 48,837 ( DIGE 4.5 -- 6.5 ) 179 , 321 , 323 P60664 HisF 5.03 / 28,454.45 179 Imidazole glycerol phosphate synthase subunit ATP phosphoribosyltransferase Has a crucial role in the pathway because the rate of histidine-biosynthesis seems to be controlled primarily by regulation of hisG enzymatic activity ; feedback inhibited by histidine or inhibited by AMP and N -5 - phosphoribosyl-ATP See HisA : step 1 HisG 5.47 / 33,366.72 P60757 179 , 321 E 4.5 -- 6.5 ) IG 5.40 / 32,959 ( D HisH 287 P60595 5.33 / 21,652.86 5.24 / 27,057 ( 5 -- 6 ) Imidazole glycerol phosphate synthase subunit Histidine biosynthesis bifunctional protein HisIE Histidine-binding periplasmic protein HisI P06989 See HisA : steps 2 and 3 179 5.24 / 22,755.82 Component of the high-afﬁnity histidine permease , a binding-protein-dependent transport system ( HisJ , HisQ , HisM , and HisP ) HisJ 179 , 286 , 287 , 321 5.17 / 26,232.61 5.05 / 28,668 P0AEU0 5.28 / 27,969 ( DIGE 4.5 -- 6.5 ) 5.14 / 27,701 ( DIGE 4.5 -- 6.5 ) 4.99 / 10,461 ( 4 -- 5 ) 5.14 / 28,391 ( 4.5 -- 5.5 ) 5.59 / 50,642 HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Catalyzes the phosphorylation of D-glycero-D-manno-heptose 7-phosphate at the C-1 position to form d,D-heptose-1 ,7 - bisphosphate ; also catalyzes the ADP transfer to D-glycero-D-manno-heptose 1-phosphate , yielding ADP-d , D-heptose Involved in NO detoxiﬁcation in an aerobic process , termed nitric oxide dioxygenase reaction , that utilizes O2 and NAD ( P ) H to convert NO to nitrate , which protects the bacterium from various noxious nitrogen compounds ; induced by nitric oxide NO ( under aerobic conditions ) , nitrite , nitrate ( under anaerobic conditions ) , nitroso compounds , and paraquat ; induced by low pH during anaerobic growth HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Hmp 5.48 / 43,867.66 Flavohemoprotein P24232 323 Continued on facing pag Has possible histone-like function and may be a global transcriptional regulator through its ability to bind to curved DNA sequences , which are found in regions upstream of a certain subset of promoters ; subject to transcriptional autorepression by binding preferentially to the upstream region of its own gene recognizing two segments of DNA on both sides of a bend centered around position 150 ; the hns mutant exhibit derepression not only of csgBA , hdeAB , mcc , and osmY but also of the expression of at least 22 S-controlled genes , including S itself ; increases during phosphate limitation and phosphonate growth Involved in purine-salvage pathway and acts exclusively on hypoxanthine Involved in the maturation of iron-sulfur cluster-containing proteins ; seems to help targeting proteins to be folded toward HscA Redox regulated molecular chaperone ; protects both thermally unfolding and oxidatively damaged proteins from irreversible aggregation ; plays an important role in the bacterial defense system toward oxidative stress ; induced by heat shock Chaperone subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol Protease subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol Molecular chaperone ; has ATPase activity ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol but decreases during phosphate limitation late Participates in the periplasmic electron-transferring activity of hydrogenase 2 during its catalytic turnover Required for the maturation of the three NiFe hydrogenases ; exhibits a low intrinsic GTPase activity ; affects some aspect of the processing of hydrogenases 1 and 2 , perhaps nickel incorporation into the apoenzymes , since hypB gene lesions can be complemented by high nickel ion concn in the medium TABLE 2 -- Continued Hns P0ACF8 DNA-binding protein H-NS 5.44 / 15,408.44 5.45 / 15,612 5.00 / 12,390 5.38 / 15,657 5.02 / 9,935 Hpt 5.09 / 20,115.24 5.38 / 12,316 ( 4.5 -- 5.5 ) 179 , 287 P0A9M2 Hypoxanthine phosphoribosyltransferase P0A6L9 Cochaperone protein 5.05 / 20,137.71 HscB 79 1 4.35 / 32,534.48 P0A6Y5 33-kDa chaperonin HslO 79 1 HslU 5.24 / 49,593.80 5.30 / 48,579 P0A6H5 ATP-dependent hsl protease ATP-binding subunit 71 , 179 , 228 , 294 5.95 / 18,961.66 5.93 / 22,161 P0A7B8 ATP-dependent protease HslV 71 , 228 , 294 HtpG 5.09 / 71,422.53 5.06 / 65,639 P0A6Z3 Chaperone protein 71 , 179 , 228 , 293 , 294 HybA 79 P0AAJ8 Hydrogenase-2 operon protein 6.38 / 33,263.79 6.38 / 33,263.79 1 P0AAN3 Hydrogenase isoenzymes nickel 5.42 / 31,565.04 6.32 / 32,224 incorporation protein HypB 286 Continued on following pag Required for the formation of all three hydrogenase isoenzymes Associates with aggregated proteins , together with IbpB , to stabilize and protect them from irreversible denaturation and extensive proteolysis during heat shock and oxidative stress ; aggregated proteins bound to the IbpAB complex are more efﬁciently refolded and reactivated by the ATP-dependent chaperone systems ClpB and DnaK/DnaJ/GrpE ; its activity is ATP independent ; IbpAB increase in overexpression of heterologous proteins and biopolymers See IbpA ; decreases during phosphate limitation early Inhibition of this enzyme by phosphorylation regulates the branch point between the Krebs-cycle and the glyoxylate-bypass , which is an alternate route that accumulates carbon for biosynthesis when acetate is the sole carbon source for growth ; increases during high-cell-density cultivation but decreases during phosphate limitation Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the formation of alpha-ketobutyrate from threonine in isoleucine-biosynthesis ; isoleucine allosterically inhibits whereas valine allosterically activates this enzyme ; increases during phosphate limitation and phosphonate growth Involved in valine and isoleucine-biosynthesis : step 1 See IlvB : step 2 ; induced in the presence of acetohydroxybutyrate and acetolactate ; decreases after benzoic acid treatment See IlvB : step 3 ; increases after benzoic acid treatment See IlvB : step 4 ( ﬁnal ) ; acts on leucine , isoleucine , and valine ; decreases during phosphate limitation See IlvB : step 1 ; sensitive to valine inhibition Determines N-hexane tolerance ; involved in outer membrane permeability ; essential for envelope biogenesis ; could be part of a targeting/usher system for outer membrane components TABLE 2 -- Continued HypD P24192 5.80 / 41,363.38 5.81 / 41,711 Hydrogenase isoenzymes formation protein P0C054 Small heat shock protein IbpA 5.57 / 15,773.71 5.35 / 15,600 5.68 / 15,600 4 , 97 , 98 , 228 , 287 , 296 5.63 / 15,600 5.63 / 15,384 ( 5 -- 6 ) 5.63 / 14,237 ( 5.5 -- 6.7 ) 5.19 / 16,093.20 P0C058 Small heat shock protein IbpB 5.02 / 15,600 5.20 / 15,600 4 , 97 , 98 , 295 Icd 5.06 / 46,200 5.15 / 46,200 5.02 / 46,100 5.11 / 44,335 ( DIGE 4.5 -- 6.5 ) 5.15 / 44,053 ( DIGE 4.5 -- 6.5 ) 4.70 / 22,351 ( 4 -- 5 ) 5.15 / 43,951 ( 4.5 -- 5.5 ) 5.00 / 74,518 ( 5 -- 6 ) 5.15 / 45,756.71 P08200 Isocitrate dehydrogenase ( NADP ) 179 , 228 , 236 , 287 , 293 , 294 , 321 , 325 P52643 IdhA 5.29 / 36,534.79 5.59 / 34,100 D-Lactate dehydrogenase 128 , 293 5.57 / 56,195.25 6.16 / 54,999 Threonine dehydratase biosynthetic P04968 IlvA 293 287 5.22 / 56,152 ( 4.5 -- 5.5 ) P08142 5.30 / 60,440.57 I e Acetolactate synthase isozym large subunit Ketol-acid reductoisomerase IlvB 5.26 / 52,066 5.20 / 53,937.83 P05793 IlvC 286 , 321 5.22 / 52,046 ( DIGE 4.5 -- 6.5 ) 5.29 / 51,264 ( DIGE 4.5 -- 6.5 ) 321 5.54 / 62,065 ( DIGE 4.5 -- 6.5 ) 5.59 / 65,400.37 P05791 Dihydroxy-acid dehydratase IlvD 5.43 / 35,112 5.54 / 33,962.46 P0AB80 Branched-chain amino acid aminotransferase IlvE 228 , 236 , 293 , 294 287 8.11 / 15,587 ( 6 -- 11 ) 8.01 / 17,976.76 e Acetolactate synthase isozym III small subunit Organic solvent tolerance protein P00894 IlvH Imp 4.91 / 88,368 ( DIGE 4.5 -- 6.5 ) 4.85 / 87,068.95 P31554 179 , 321 Continued on facing pag IscA P0AAC8 Iron-binding protein ( YfhF ) 4.75 / 11,556.05 IscU 179 Involved in Fe-S biosynthesis 4.82 / 13,848.59 IspA IspG Geranyltranstransferase 5.27 / 32,159.63 5.37 / 26,625 4-Hydroxy-3-methylbut-2-en-1-yl 5.87 / 40,683.61 diphosphate synthase P22939 P62620 179 , 286 179 5.2 / 34,774.56 4-Hydroxy-3-methylbut-2-enyl diphosphate reductase P62623 IspH 179 5.46 / 15,065 5.41 / 14,304 ( 5 -- 6 ) 5.51 / 14,102.90 Inhibitor of vertebrate lysozyme P0AD59 Ivy 228 , 287 5.33 / 12,548 ( 5 -- 6 ) 5.47 / 86,507 ( DIGE 4.5 -- 6.5 ) 103 , 321 5.54 / 84,162.61 Catalase HPII P21179 KatE Peroxidase/catalase HPI P13029 KatG 103 , 287 , 321 5.18 / 77,774 ( DIGE 4.5 -- 6.5 ) 5.16 / 74,539 ( 4.5 -- 5.5 ) 5.14 / 80,023.82 5.37 / 31,293.68 5.98 / 31,100 P0AB74 Tagatose-1 ,6 - bisphosphate aldolase AgaY Kba 99 2-Amino-3-ketobutyrate CoA ligase 2-Dehydro-3-deoxygluconokinase 2-Dehydro-3-deoxyphosphooctonate aldolase Could transfer iron-sulfur clusters to apo-ferredoxin ; multiple cycles of 2Fe2S cluster formation and transfer are observed ; acts catalytically ; recruits intracellular free iron so as to provide iron for the assembly of transient ironsulfur cluster in IscU in the presence of IscS , L-cysteine , and the thioredoxin reductase system TrxA/TrxB Not found in SWISS-PROT / TrEMBL database but available in NCBI database ( GI : 16130454 ) Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate into 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate Converts 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate into isopentenyl diphosphate and dimethylallyl diphosphate ; also involved in penicillin tolerance , control of the stringent-response , and isoprenoid-biosynthesis Strong inhibitor of lysozyme C Decomposes hydrogen peroxide into water and oxygen ; serves to protect cells from the toxic effects of hydrogen peroxide ; induced by entry into stationary phase and after benzoic acid treatment Bifunctional , exhibiting both catalase and broad-spectrum peroxidase activities ; induced by hydrogen peroxide and by entry into stationary phase but decreases after benzoic acid treatment Catalyzes the reversible condensation of dihydroxyacetone phosphate with glyceraldehyde 3-phosphate to produce tagatose-1 ,6 - bisphosphate ; repressed in the presence of glucose Negatively regulated by Lrp Kba 99 - / 42,200 5.64 / 43,117.04 P0AB77 61 Kbl 4.92 / 33,962.36 6.32 / 30,832.69 P37647 P0A715 179 KdgK KdsA Synthesis of 2-keto-3 - 179 , 287 6.10 / 33,261 ( 6 -- 11 ) deoxyoctulosonic acid 8-phosphate , which is required for lipid A maturation and cellular-growth Activates 2-keto-3-deoxyoctulosonic acid for incorporation into bacterial lipopolysaccharide in gramnegative bacteria Involved in the transport of maltose and maltodextrins ; indispensable for translocation of dextrins containing more than three glucosyl moieties ; acts as a receptor for several bacteriophages including lambda ; induced by maltose 5.15 / 27,483.25 5.06 / 33,695 3-Deoxy-manno-octulosonate cytidylyltransferase P04951 KdsB 228 , 294 LamB 79 4.72 / 47,385.03 Maltoporin P02943 1 Continued on following pag Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate ( 2-oxoisovalerate ) to form 3-carboxy-3-hydroxy-4-methylpentanoate ( 2-isopropylmalate ) ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate via the formation of 2-isopropylmaleate LdhA P52643 D-Lactate dehydrogenase 5.29 / 36,534.79 5.59 / 34,100 P09151 2-Isopropylmalate synthase LeuA 308 5.47 / 57,166.71 LeuC 3-Isopropylmalate dehydratase large subunit P0A6A6 5.90 / 49,750.64 5.95 / 51,656 5.77 / 51,451 179 , 286 , 321 5.42 / 44,439 5.93 / 50,340 ( DIGE 4.5 -- 6.5 ) See LeuC ; increases after benzoic acid treatment Increases late during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Represses a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_12">
Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. , and Bertin , P. ( 1999 ) Multiple control of ﬂagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></label>
<input id="rp_similarity_12", type="checkbox">
<div class="RP_similar">
<li>Soutourina O , Kolb A , Krin E , Laurent-Winter C , Rimsky S , Danchin A & Bertin P ( 1999 ) Multiple control of ﬂagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20528937" target="_blank">20528937</a>]<br></li>
<li>Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. and Bertin , P. ( 1999 ) Multiple control of ﬂagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. , and Bertin , P. ( 1999 ) Multiple control of flagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP ± catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Soutourina O , Kolb A , Krin E , Laurent-Winter C , Rimsky S , Danchin A , Bertin P. Multiple control of flagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144778" target="_blank">12144778</a>]<br></li>
<li>[ PubMed : 14550943 ] Soutourina O , Kolb A , Krin E , Laurent-Winter C , Rimsky S , Danchin A , Bertin P. Multiple control of flagellum biosynthesis in Escherichia coli , role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15941987" target="_blank">15941987</a>]<br></li>
<li>[ PubMed : 20457943 ] Soutourina O , Kolb A , Krin E , Laurent-Winter C , Rimsky S , Danchin A , Bertin P. Multiple control of flagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>[ 14 ] Soutourina O. , Kolb A. , Krin E. , Laurent-Winter C. , Rimsky S. , Danchin A. , Bertin P. , Multiple control of ﬂagella biosynthesis in Escherichia coli : Role of H-NS protein and the cyclic AMP / catabolite activator protein complex in transcription of the ﬂhDC master operon , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. , and Bertin , P. ( 1999 ) Multiple control of flagella biosynthesis in Escherichia coli : Role of H-NS protein and the cyclic AMP/catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Soutourina , O. , Kolb , A. , Krin , E. , Laurent-Winter , C. , Rimsky , S. , Danchin , A. , and Bertin , P. ( 1999 ) Multiple control of ﬂagellum biosynthesis in Escherichia coli : Role of H-NS protein and the cyclic AMP -- catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_22">
Multiple control of ﬂagellum biosynthesis in Esche-richia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></label>
<input id="rp_similarity_22", type="checkbox">
<div class="RP_similar">
<li>Multiple control of ﬂagellum biosynthesis in Escherichia coli : Role of H-NS protein and the cyclic AMP-catabolic activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730174" target="_blank">12730174</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_24">
Multiple control of flagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the flhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22422754" target="_blank">22422754</a>]<br></label>
<input id="rp_similarity_24", type="checkbox">
<div class="RP_similar">
<li>Multiple control of ﬂagellum biosynthesis in Escherichia coli : role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the ﬂhDC master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489855" target="_blank">11489855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11872702" target="_blank">11872702</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li></div>
<li>Table 5 ( continued ) hokA 2847732 Toxic polypeptide , small [ b4455 ] ( cell killer ) 2.7 L7091 3683665 Nickase [ L7091 ] ( P ) phnI 1038042 PhnI protein [ ] 2.7 ECs3727 916452 Type III secretion protein EivJ [ ECs3727 ] 4.0 4.1 yciW 945859 insD 1037557 4.2 4.3 Oxidoreductase [ b1287 ] ( P ) 2.8 2.8 959384 Regulator [ Z3284 ] ( P ) ECs1620 913267 Antitermination protein [ ECs1620 ] Transposase insD for insertion element IS2A/D/F / H/I/K [ ] Cytochrome c biogenesis , possible subunit of a heme lyase [ b2197 ] Probable pyruvate formate lyase activating enzyme 2 [ b3952 ] Virulence-related membrane protein ( PagC-like protein ) [ ECs3850 ] ( P ) Permease of iron compound ABC transport [ Z4918 ] ( P ) Exonuclease [ ECs2216 ] ( P ) Transporter subunit ; ABC superfamily [ b0793 ] ( P ) Shiga toxin I subunit A precursor [ ECs2974 ] Formate-dependent nitrate reductase complex [ b4073 ] ccmE 946697 4.4 2.8 Z4969 961069 Fimbrial chaperone [ Z4969 ] ( P ) pflC 948453 ECs3850 916326 4.4 4.5 2.9 2.9 ECs4965 914966 C4-type zinc finger protein [ ECs4965 ] ECs3243 915657 Sucrose-6 phosphate hydrolase [ ECs3243 ] chuU 961162 ECs2216 917419 3.0 3.1 ECs4271 915856 ATP binding protein of ABC transporter [ ECs4271 ] ( P ) flhE 1036868 Flagellar protein flhE precursor [ ] ybhS 945411 ECs2974 916678 5.0 5.1 3.2 3.2 ECs1612 913277 Replication protein P [ ECs1612 ] 1035552 Membrane transport protein [ ] ( P ) nrfD 948580 5.2 5.2 3.2 ECs0804 917548 Membrane protein [ ECs0804 ] ( P ) sseB 946994 Enhanced serine sensitivity [ b2522 ] ybhI 945377 Membrane pump protein [ b0770 ] ( P ) 3.3 3.3 hmuV 1037200 ATP-binding protein of ABC transport system [ ] ( P ) ECs4501 915551 Lipopolysaccharide 1,2-N - acetylglucosaminetransferase [ ECs4501 ] 1037712 Transposase [ ] 5.7 entC 945511 Isochorismate hydroxymutase 2 , enterochelin-biosynthesis [ b0593 ] gutQ 947587 D-arabinose 5-phosphate isomerase [ b2708 ] ECs1220 912871 Terminase large subunit [ ECs1220 ] ( P ) 1036351 Membrane spanning export protein [ ] ( P ) 5.7 3.4 5.7 5.9 3.4 3.5 wcaK 962104 Galactokinase [ Z3209 ] ( P ) etpJ 3683738 Type II secretion protein [ L7039 ] 3.7 3.7 5.9 6.9 ECs0140 913740 Fimbrial protein [ ECs0140 ] ( P ) ECs2180 917376 Terminase small subunit [ ECs2180 ] ( P ) fliI 946457 Flagellum-specific ATP synthase [ b1941 ] fliI 912799 Flagellum-specific ATP synthase [ ECs2680 ] 3.7 Z3494 957220 Antibiotic efflux protein [ Z3494 ] ( P ) hcr 947660 NADH-dependent HCP oxidoreductase [ b0872 ] 3.7 ECs2582 914087 Biotin sulfoxide reductase 2 [ ECs2582 ] 7.2 7.3 1037311 DNA-binding protein H-NS [ ] 3.8 sdiA 946421 ftsQAZ gene cluster transcriptional regulator ; Q-S activator [ b1916 ] 1040091 Pyruvate formate-lyase 2 activating enzyme [ ] 945845 DNA-binding protein [ b0278 ] 946961 Hydrogenase 4 membrane subunit [ b2482 ] 7.9 958282 Cytotoxin [ Z4333 ] ( P ) 1038301 Membrane protein [ ] ( P ) Z4333 3.8 3.9 pflC yagL 8.2 9.5 3.9 hyfB Z4876 961227 Phosphotransferase system enzyme subunit [ Z4876 ] ( P ) 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24952731" target="_blank">24952731</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_27">
H-NS antagonism in Shigella ﬂexneri by VirB , a virulence gene transcription regulator that is closely related to plasmid partition factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></label>
<input id="rp_similarity_27", type="checkbox">
<div class="RP_similar">
<li>Turner , E.C. and Dorman , C.J. ( 2007 ) H-NS antagonism in Shigella ﬂexneri by VirB , a virulence gene transcription regulator that is closely related to plasmid partition factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Turner , E.C. , and Dorman , C.J. ( 2007 ) H-NS antagonism in Shigella ﬂexneri by VirB , a virulence gene transcription regulator that is closely related to plasmid partition factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li></div>
<li>Plasmids described previously included the following ( with salient features in parentheses ) : the vectors pWSK30 ( pSC101 replicon , Amp resistant [ Ampr ] ) ( 62 ) , pBAD18 ( ColE1 replicon , Ampr , for L-arabinose [ Ara ] - induced expression of target genes ) ( 26 ) , pACYC184 ( p15A replicon , Tetr Cmr ) ( 14 ) , and pTrc99A r ( ColE1 replicon , Amp , for IPTG-induced expression of target genes ( 2 ) ; pCP20 ( pSC101-based Ts replicon , Cmr Ampr , for in vivo expression of Flp recombinase ) ( 17 ) ; pHYD272 ( single-copy-number IncW replicon carrying proU-lac transcriptional fusion , Tpr ) ( 18 ) ; and a set of plasmids described by Williams et al. ( 65 ) comprising the vector pLG339 ( pSC101 replicon , Kanr ) and its derivatives carrying hns or one of the following hns alleles encoding the dominantnegative H-NS substitution variants L26P , E53G/T55P , Y97C , P116S , or I119T or the truncation variant referred to as 64 ( an allele which has a frameshifting 1-bp deletion mutation after codon 63 in the gene ) and designated accordingly with a pLG - preﬁx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>To test the effects of H-NS on SlyA-driven hlyE expression , we measured - galactosidase activities from cultures of isogenic parental and hns strains carrying the hlyE : : lacZ plasmid pGS1065 and either a multicopy S. enterica SlyA expression plasmid ( pGS1657 ; Table 1 ) or , as a control , the vector ( pBluescript ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_32">
Antagonist involvement of FIS and H-NS proteins in the transcriptional-control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></label>
<input id="rp_similarity_32", type="checkbox">
<div class="RP_similar">
<li>Antagonistic involvement of FIS and H-NS proteins in the transcriptional-control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885445" target="_blank">16885445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_34">
Falconi , M. , Brandl , A. , La Teana , A. , Gualerzi , C.O. , and Pon , C.L. ( 1996 ) Antagonistic involvement of FIS and H-NS protein in the transcriptional-control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></label>
<input id="rp_similarity_34", type="checkbox">
<div class="RP_similar">
<li>Falconi , M. , Brandi , A. , La Teana , A. , Gualerzi , C.O. , and Pon , C.L. ( 1996 ) Antagonistic involvement of FIS and H-NS proteins in the transcriptional-control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>[ 4 ] Falconi M. , Brandi A. , La Taena A. , Gualerzi C.O. , Pon C.L. , Antagonistic involvement of FIS and H-NS proteins in the transcriptional-control of hns expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>[ 10 ] Falconi M. , Brandi A. , La Teana A. , Gualerzi C.O. , Pon C.L. , Antagonistic involvement of FIS and H-NS proteins in the transcriptional-control of hns expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li></div>
<li>Effect of hns and/or crp mutations on ﬂhDC-cat transcriptional fusion activity a c CAT activity ( U ) in strain ( genotype ) : PS2652 BE1818 ( hns-1001 ) ( crp : : Sm ) Transcriptional fusion [ gene ( regulatory region ) b and plasmid ] BE1819 ( hns crp ) PS2209 ( wild type ) Recently , interactions between H-NS and various regulators have been demonstrated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>It also inhibits H-NS translation by sharply increasing hns mRNA turnover ( 40 , 44 ) and thereby curtails H-NS-me-diated transcriptional silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557827" target="_blank">17557827</a>]<br></li>
<li>Panels A [ H-NS ] and B [ StpA ] : a , W3110 , A-type lineage ; b , BSN27 , hns disruptant ; c , BSN28 , stpA disruptant ; d , ÿ ÿ BSN29 , hns stpA double mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>ACKNOWLEDGEMENTS We thank Erhard Bremer for generously providing strain PD32 ( hns-206 ) and plasmid pPD3 for overproduction of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>There is increasing evidence that DNA topology in bacterial cells changes in response to environmental signals such as osmolarity , anaerobicity , temperature and nutrient status , and these alterations involve H-NS activity ( Higgins et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Recently the phenotypically cryptic csgA gene has been shown to be transcriptionally activated by mutations affecting H-NS ( A , Oisen , A , Arnqvist and S. Normark , unpublished ) , suggesting that the activation of csgA in HBIOI and many other E. co / ; strains may involve local changes in DNA topology -LCB- Hulton et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1357528" target="_blank">1357528</a>]<br></li>
<li>This promoter is controlled by a number of global regulators such as CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , heat shock proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) and H-NS ( Bertin et al. , 1994 ; Kutsukake , 1997 ; Yanagihara et al. , 1999 ) , as well as by several environmental signals such as temperature control ( Qualding and Stocker , 1962 ; Adler and Templeton , 1967 ) , high concentrations of inorganic salts , carbohydrates or alcohols ( Li et al. , 1993 ; Shi et al. , 1993 ) , growth phase ( Nishimura and Hirota , 1989 ) , DNA supercoiling ( Li et al. , 1993 ; Shi et al. , 1993 ; Kutsukake , 1997 ; Das et al. , 1998 ) , phosphatidylethanolamine and phosphatidylglycerol-synthesis ( Shi et al. , 1993 ; Mizushima et al. , 1995 ) , cell-cycle control ( Morgan et al. , 1995 ; Prüß and Matsumura , 1996 ; Mizushima et al. , 1997 ) and quorum-sensing ( Sperandio et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>]<br></li>
<li>Fis can also affect DNA supercoiling indirectly , because of its roles in repressing transcription of gyrA and gyrB to reduce the DNA gyrase activity ( Schneider et al. , 1999 ) , increasing topA transcription to enhance topoisomerase I activity ( WeinsteinFischer et al. , 2000 ) , and regulating the expression of other DNA-binding proteins such as H-NS , HU and CRP ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) that also affect DNA supercoiling directly or indirectly ( Broyles & Pettijohn , 1986 ; Gomez-Gomez et al. , 1996 ; Tupper et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Several DNA-binding proteins , including H-NS ( Suzuki et al. , 1996 ) , HU ( Painbeni et al. , 1997 ) , Lrp ( Ferrario et al. , 1995 ) , Rob ( Ariza et al. , 1995 ) , SoxS ( Chou et al. , 1993 ) and MarA ( Cohen et al. , 1988 ; Jair et al. , 1995 ) , are known to regulate micF promoter activity and , consequently , modulate ompF mRNA translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>In fact , control by Lrp often involves H-NS protein , e.g ompF and ompC ( Graeme-Cook et al. 1989 ; Ernsting et al. 1992 ) , fimB ( Kawula and Orndorff 1991 ; Blomfield et al. 1993 ) , papB ( van der Woude et al. 1992 ) , ilvIH ( Levinthal et al. 1994 ) , and lysU ( Ito et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This hypothesis is consistent with findings regarding H-NS control of the transcription of several other operons , which are also under Lrp control : ompFand ompC ( Graeme-Cook et al. 1989 ; Ernsting et al. 1992 ) , fimB ( Kawula and Orndorff 1991 ; Blomfield et al. 1993 ) , and papB ( van der Woude et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>We also engineered the global regulator H-NS to control biofilm-formation via prophage excision and cell-death ( Hong et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059164" target="_blank">21059164</a>]<br></li>
<li>By discovering BdcA and creating the E50V and E50Q variants , we have also obtained important new tools that along with H-NS K57N ( evolved to decrease biofilm-formation through cell lysis by inducing cryptic prophage Rac ) ( Hong et al. , 2010 ) , SdiA 1E11 ( evolved to decrease biofilm-formation by increasing indole concentrations ) ( Lee et al. , 2009 ) , and SdiA SD10 ( evolved to increase biofilm with the addition of homoserine lactone signals ) ( Lee et al. , 2009 ) , allow the control of biofilm-formation for various applications .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059164" target="_blank">21059164</a>]<br></li>
<li>2010 SGM Printed in Great Britain In concert with the complex mode of regulation for biofilm-formation , the expression of csgD has been proposed to be under the control of various stress-sensing regulatory proteins such as OmpR ( Romling et al. , 1998 ; Vidal et al. , 1998 ; Prigent-Combaret et al. , 1999 , 2000 , 2001 ; Gerstel & Romling , 2001 ; Gerstel et al. , 2003 ) , CpxR ( PrigentCombaret et al. , 2001 ; Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ; Robbe-Saule et al. , 2006 ) , CRP ( Zheng et al. , 2004 ) , H-NS ( Arnqvist et al. , 1994 ; Gerstel et al. , 2003 ; Olsen et al. , 1993 ) , IHF ( Gerstel et al. , 2003 ; Gerstel & Romling , 2001 ) , MlrA ( Brown et al. , 2001 ) and RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Furthermore , a number of transcription factors are involved in the regulation of the csgD promoter , including CpxR ( Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ) , H-NS ( Gerstel et al. , 2003 ) , IHF ( Gerstel et al. , 2003 , 2006 ) , OmpR ( Vidal et al. , 1998 ; Gerstel et al. , 2003 , 2006 ) , RstA ( Ogasawara et al. , 2007a ) , MlrA ( Brown et al. , 2001 ) , RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) and CRP ( Zheng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The latter circuit is also the most complex because apart from GadX and GadW , it also involves the global regulators H-NS and CRP , which act as transcriptional repressors , and RpoS , which positively affects gadA and gadBC transcription ( De Biase et al. , 1999 ; Castanie-Cornet and Foster , 2001 ; Hommais et al. , 2001 ; Tramonti et al. , 2002 ; Giangrossi et al. , 2005 ; Weber et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>In E. coli , several global regulators such as H-NS ( Bertin et al. , 1994 ) , OmpR ( Shin & Park , 1995 ) , CRP-cAMP ( Soutourina et al. , 1999 ) , LrhA ( Lehnen et al. , 2002 ) and RcsAB ( Francez-Charlot et al. , 2003 ) are directly involved in the complex genetic regulatory hierarchy that assures ordered assembly of ﬂagellar components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20528937" target="_blank">20528937</a>]<br></li>
<li>Model to explain H-NS-mediated Fe2 + -- Fur activation of ftnA H-NS acts as a global repressor of transcription in E. coli and many other bacteria ( Westermark et al. , 2000 ; Hommais et al. , 2001 ; Grainger et al. , 2006 ; Lucchini et al. , 2006 ; Navarre et al. , 2006 ; Stoebel et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Recently , we reported that deletion of hns ( encoding the global regulator H-NS ) abolished eDNA production in E. coli ( Sanchez-Torres et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21181144" target="_blank">21181144</a>]<br></li>
<li>( iii ) H-NS negatively regulates lysine decarboxylase CadA and cadaverine/lysine antiporter CadB , essential for lysine-dependent system ( Atlung and Ingmer , 1997 ; Hommais et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Overproduction or an altered regulation of H-NS has been reported to exert some deleterious effects on cell-physiology ( Spurio et al. , 1992 ) , including swarm migration ( Hinton et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>H-NS negatively regulates RpoS translation ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>Indirect action could be via transcriptional regulation of a gene involved in RpoS translational regulation , as suggested by Yamashino and colleagues for the action of H-NS on RpoS ( Yamashino et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>More recently , H-NS had aiso been demonstrated to be a transcriptional repressor involved in the regulation of a number of apparently unrelated genes ( BertJn et ai , 1990 ; Yamada etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>H-NS is involved in numerous cellular-processes , including replication ( Katayama et al. , 1996 ) , recombination ( Goshima et al. , 1994 ) and transcriptional regulation of a large number of genes ( Ussery et al. , 1994 ; Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Among the known bacterial proteins involved in phase-depend-ent transcriptional regulation ( Azam et al. , 1999 ; Dame , 2005 ) , we have selected H-NS , StpA , LRP and FIS as potential candidates for our studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>Among a set of more than 12 characterized proteins , FIS , H-NS , IHF and LRP have been recognized as more speciﬁc regulators , acting as transcriptional activators or repressors for numerous genes ( Azam and Ishihama , 1999 ; Dame et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Intriguing results on the possible function of H-NS were obtained recently in genetic studies of drdX ( G6ransson et al. 1990 ) , a regulatory locus involved in the thermoregulation of a determinant of pilus adhesion , and also in genetic studies on osmZ , which is a mutation that causes altered osmoregulation of the proU operon ( Higgins et al. 1988 ; Hulton et al. 1990 ; May et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Moreover , a possible role of H-NS in many other biological-processes such as replication , recombination and transposition has also been suggested ( Ussery et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>In view of new ﬁndings related to the role of H-NS in silencing foreign DNA ( Lucchini et al. , 2006 ; Navarre et al. , 2006 ) , it would be interesting to determine whether Hha is also involved in such processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>LeuO is another Leu-sensing global regulator that controls * 140 targets , of which most are involved in anti-silencing against the H-NS silencer ( Shimada et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Recent data indicate that LeuO is involved in regulating transcription of many genes , often as an H-NS antagonist ( Stoebel et al. , 2008 ; Shimada et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>For Regulation of rRNA-transcription by H-NS and LRP 873 instance , H-NS is involved in the LRP-dependent activation of the ilvIH operon ( Levinthal et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_71">
Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible role for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></label>
<input id="rp_similarity_71", type="checkbox">
<div class="RP_similar">
<li>Mukerji , M. & Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10792463" target="_blank">10792463</a>]<br></li>
<li>Mukerji , M. , and Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>Mukerji , M. and Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li>
<li>Mukerji , M. , and Mahadevan , S. ( 1997 ) Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : Possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP-cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22020646" target="_blank">22020646</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Characterization of the negative elements involved in silencing the bgl operon of Escherichia coli : possible roles for DNA gyrase , H-NS , and CRP -- cAMP in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_79">
Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867437" target="_blank">12867437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15838058" target="_blank">15838058</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></label>
<input id="rp_similarity_79", type="checkbox">
<div class="RP_similar">
<li>Interplay between global regulators of Escherichia coli : effect of RpoS , H-NS and Lrp on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_81">
[ 6 ] J. Bouvier , S. Gordia , G. Kampmann , R. Lange , R. Hengge-Aronis , C. Gutierrez , Interplay between global regulators of Escherichia coli : Effect of RpoS , Lrp and H-NS on transcription of the gene osmC , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16427772" target="_blank">16427772</a>]<br></label>
<input id="rp_similarity_81", type="checkbox">
<div class="RP_similar">
<li>Mol Microbiol 35:845 -- 853 Bouvier J , Gordia S , Kampmann G , Lange R , Hengge-Aronis R , Gutierrez C ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12395194" target="_blank">12395194</a>]<br></li>
<li>Bouvier J , Gordia S , Kampmann G , Lange R , Hengge-Aronis R , Guti-errez C : Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene OsmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , H-NS and Lrp on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp , and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li></div>
<li>These activities include the translational regulation of RpoS and H-NS , global transcriptional regulators in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_90">
Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></label>
<input id="rp_similarity_90", type="checkbox">
<div class="RP_similar">
<li>GeÂrard , F. , Dri , A.M. and Moreau , P.L. ( 1999 ) Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>J Bacteriol 188 : Gerard F , Dri AM & Moreau PL ( 1999 ) Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li></div>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>Thus , LeuO , which counteracts the H-NS-mediated repression of several loci , indirectly controls H-NS and RpoS synthesis under speciﬁc conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Tolerating some redundancy RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>I Keywords : H-NS , LexA , phosphate starvation , reactive-oxygen-species , RpoS INTRODUCTION respiration , or fermentation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Discussion H-NS regulates directly and indirectly the RcsB-P/GadE complex , that is located at the centre of the acid resistance network as well as control of motility ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Expression of these components is under the control of a complex network of transcription factors , including GadE , GadX , GadW , EvgA , YdeO , and H-NS [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>To find further H-NS-dependent intermediary actors of glutamate-dependent acid resistance , several of the H-NS induced targets , identified in a previous transcriptome analysis [ 1 ] and related either to acid stress resistance or to information pathways , were deleted in an hns-deficient strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Identification of the target genes for major regulators To decipher the regulatory hierarchy in acid stress resistance involving several new H-NS controlled regulators , the mRNA level of target genes was compared between wild-type and hns , hns rcsB , hns gadE , hns hdfR , hns adiY mutant strains , using real-time quantitative RT-PCR ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>To identify targets of the RcsBeGadE complex that are involved in acid stress resistance , the mRNA level of genes previously identiﬁed as H-NS targets were compared between wild-type and hns , hns rcsB and hns gadE mutant strains using real-time quantitative RT-PCR ( Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Indeed , the EvgA-YdeO regulatory pathway plays a secondary modulator role in the glutamate-dependent acid stress response , in comparison to H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>In particular , the putative regulator , YhiX , plays a central role in the H-NS control of genes required in the glutamate-dependent acid stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>In this regulatory process H-NS represses the expression of gadE ( encoding the central activator of the glutamate-dependent acid resistance pathway ) both in a direct and an indirect way , via EvgA , YdeO , GadX and GadW [ 1,7,8 ] , while it decreases rcsD expression , essential to the phosphorylation of RcsB ( the capsular synthesis regulator component ) required for the formation of the regulatory complex with GadE [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>( i ) H-NS represses the gadE gene encoding the central activator of the glutamate-dependent acid resistance pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Conclusions : H-NS lies near the top of the hierarchy orchestrating acid response centred on RcsB-P/GadE regulatory complex , the general direct regulator of glutamate - , arginine - and lysine-dependent acid resistance pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Finally , we found that in addition to H-NS and RcsB , a third regulator , HdfR , inversely controls glutamate-dependent acid resistance pathway and motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Thus , our results argue that CRP , not H-NS , is a master regulator of glutamate-dependent acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Decrypting the H-NS-dependent regulatory Abstract Background : H-NS regulates the acid stress resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>( ii ) H-NS also represses arginine decarboxylase AdiA and arginine : agmatine antiporter AdiC ( YjdE ) , essential for arginine-dependent acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>The gadA/BC genes , activated in response to acid stress and in stationary phase cells , are subjected to complex circuits of regulation involving 70 , S , cAMP receptor protein , H-NS , EvgAS , TorRS , GadE , GadX , GadW , and YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In parallel , H-NS decreases acid resistance by repressing genes involved in general acid stress response ( e.g. dps encoding the DNA-protection during starvation protein ) or in three amino-acid decarboxylase systems , dependent on glutamate , lysine and arginine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Determination of H-NS targets involved in arginine and / or lysine-dependent acid resistance We wondered whether the new genes identified as H-NS-controlled , but unrelated to regulation by the RcsB-P / GadE complex , also play a role in the arginine and lysine-dependent acid resistance pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>This suggests a role of CadC regulator in the H-NS regulation of the lysine-dependent acid stress resistance and a role of AdiY regulator in the arginine - and glutamate-dependent pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Quantitative RT-PCR analysis of H-NS targets involved in acid stress resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Results Determination of new H-NS targets involved in the regulation of glutamate-dependent acid resistance As H-NS strongly represses the glutamate-dependent acid stress response , there is a very low level of survival after acid stress in the FB8 wild-type context [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>The new characterized H-NS targets , CadC and AdiY , have no effect on motility ( data not shown ) and are involved in the H-NS-dependent regulation of lysine and arginine-dependent response to acid stress , respectively ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Second , pyruvate H-NS helps maintain a rapid degradation of arginine oxidase ( PoxB ) , which may be preferentially used to and glucose produce acetate when bacteria approach the stationary phase , is unlikely to play a key role under Pi-starvation Because H-NS is required for a full induction of the conditions because the expression of poxB is strictly LexA regulon in Pi-starved cells ( Dri & Moreau , 1993 ; controlled by RpoS ( Chang et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Our results indicate that relief of repression by H-NS plays a role in stationary-phase induction as well as in hyperosmotic induction of rpoS translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Our results indicate that relief of inhibition by H-NS is involved in both stationary-phase-associated and hyperosmotic induction of rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>The Fur-dependent regulation of ftnA requires H-NS Nucleoid-associated proteins , such as IHF ( integration host factor ) and H-NS , play a major role in modifying DNA conformation and thus controlling the access of certain regulators and/or RNA polymerase to promoters ( Atlung and Ingmer , 1997 ; Dame , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Histone-like proteins , such as IHF , H-NS , HU , and Fis , play a major role in modifying DNA conformation , changing the access of certain regulators and/or RNA polymerase to promoters ( 2 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>Forns , N. , Baños , R.C. , Balsalobre , C. , Juárez , A. , and Madrid , C. ( 2005a ) Temperature-dependent conjugative transfer of R27 : role of chromosome - and plasmid-encoded Hha and H-NS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>H-NS and H-NS-like proteins in Gram-negative bacteria and their multiple role in the regulation of bacterial metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Bertin , P. , Hommais , F. , Krin , E. , Soutourina , O. , Tendeng , C. , Derzelle , S. , and Danchin , A. ( 2001 ) H-NS and H-NS-like proteins in Gram-negative bacteria and their multiple role in the regulation of bacterial metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>For instance , comparative proteome analysis of an H-NS deletion mutant and the wild-type strain revealed that some of the S-dependent proteins or genes , including rpoS itself , were controlled by the H-NS regulon ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Our results suggest that the H-NS and StpA proteins stimulate MalT translation and hence play a positive role in the control of the maltose regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>This demonstrates that at least 22 sS-controlled proteins are overexpressed in a strain that lacks H-NS and that , therefore , H-NS plays a rather global role in the control of genes that belong to the sS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Bacterial strains disrupted in genes encoding nucleoid proteins Strain Relevant genotype Designation Comment Strep derivative of CSH142 [ araD ( gpt-lac ) 5 ] R FW102 hupA hupB FW102 hns FW102 himD FW102 a BL643 BL706 BL707 WT HU U heterodimer Loss of both HU-1 and HU-2 subunits of H Loss of H-NS homodimer Loss of IHFb subunit of IHF heterodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>We have concentrated on a selection of proteins ( FIS , H-NS , LRP and StpA ) whose regulatory properties are either typically linked to changes in the environment and the growth behavior of the cells or to stress adaptation and nutritional quality .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>Since the H-NS , HU , IHF and Fis proteins participate in a wide variety of cellular-processes , careful control of their intracellular amount at a given time during the bacterial growth cycle is obviously of great importance for cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response-to-oxidative-stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Because H-NS often acts in combination with other nucleoid-associated proteins ( NAPs ) we have also analy-sed the binding of related regulatory proteins , such as StpA , LRP and FIS ( Luijsterburg et al. , 2006 ; Dorman , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>It is becoming increasingly evident that nucleoid-associated proteins such as H-NS may play important roles as participants in transcriptional regulatory mechanisms and as modulators and/or architectural components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>On this view FIS , and possibly also CRP , act to ﬁne tune a fundamental mode of transcriptional regulation , not only by their direct effects on DNA topology but also from the indirect effects of their transcriptional-control of other nucleoid-associated proteins , such as HU and H-NS [ 3 , 4 ] , which also act as determinants of chromosomal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>The two proteins FIS and H-NS have previously been described to act as antagonistic transcription factors for rRNA-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>A whole battery of regulatory mechanisms is known to control rRNA-transcription , including the involvement of two small regulatory proteins , FIS and H-NS ( Gralla , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Expression of functional Cas proteins therefore requires anti-silencing mechanisms abrogating H-NS repression in order to become effective .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>They include several known proteins , namely , NusA , IF2 , polynucle-otide phosphorylase , RecA , H-NS , and the a subunit of DNA gyrase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631696" target="_blank">8631696</a>]<br></li>
<li>Some inducible processes , including pH-regulated ones , which show a DNA gyrase requirement , involve both integration host factor , IHF , which is hrmA/hzmD encoded , and H-NS ( Stim and Bennett 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588889" target="_blank">8588889</a>]<br></li>
<li>In addition to its role in structuring the chromosome , H-NS is also known to inﬂu-ence several cellular functions , such as defense against foreign DNA , virulence , general stress response , genome stability , and metabolism ( 1 , 2 , 9 , 17 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Whether H-NS is also involved in this process ( e.g. by binding to AT-rich foreign DNA , marking it as proto-spacer for integration within the CRISPR array ) remains to be examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>It is interesting to note that both integration host factor and H-NS , the product of the hns gene , seems to be involved in interactions with curves or bends in DNA , and the role and regulation of this aspect of DNA structure are areas of broad current interest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383109" target="_blank">8383109</a>]<br></li>
<li>The observation that down-regulation also fails to occur in an hns mutant supports the hypothesis that H-NS is solely ( or at least primarily ) responsible for the regulation of its own promoter in response to changing rates of DNA-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Furthermore , the data support the conclusion that LeuO , whose expression is likewise repressed by H-NS and which is a virulence factor in Salmonella enterica , is a master regulator that among other loci , also controls the yjjQ-bglJ operon and thus indirectly the presumptive targets of YjjQ and BglJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>LeuO , a regulator that is conserved in Enterobacteriaceae , is of pleiotropic function which includes virulence control in Salmonella enterica , and it is considered an antagonist of the global repressor H-NS ( 13 , 15 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Expression profiling of wild-type and H-NS-deficient strains H-NS has been described as a transcriptional regulator ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>H-NS thus appears to function as a global DNA-binding transcriptional regulator ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8244952" target="_blank">8244952</a>]<br></li>
<li>H-NS binds DNA , especially A+T - rich regions including promoter regions or horizontally acquired DNA and acts as a global transcriptional repressor [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>H-NS binds the chiA promoter region with high affinity The global transcriptional repressor H-NS has a negative effect on expression of a number of genes , in particular those regulated by different environmental factors ( e.g. temperature , pH or osmolarity ; Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>The major role of H-NS is as a global transcriptional repressor for a large number of genes ( for a review , see reference 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li>
<li>The results of recent extensive genetic studies on H-NS clearly support the hypothesis that H-NS may function as a global trans-acting transcriptional regulator ( see Introduction ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>3.3 A molecular backup between two global transcriptional repressors The use of homologous proteins ( H-NS and StpA ) acting together to silence expensive genes might confer robustness in the event of a mutation adversely affecting the function of either .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>However , H-NS also acts as a positive regulator for lrp ( 33 ) and ﬂagellum biosynthesis genes ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>As noted above , IHF acts as a positive regulator while H-NS is a potent repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In the virulence regulatory system in S. ﬂexneri , H-NS probably acts only as a repressor for the VirF-activated transcription of VirB , which in turn activates expression of several operons encoding functions required for virulence ( 33 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>H-NS , which is activated by CspA , likely acts as a cold shock transcriptional repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>In addition to its function as a transcriptional activator , ArcA modulates the activities of the aerobic repressor ( H-NS ) and the anaerobic repressor ( Fnr ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In addition , H-NS is a repressor of virB ( Tobe etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Osmotic shock as well as changes from exponential to stationary growth are known specifically to decrease the K * ion concentration in the cell , and also to activate H-NS function ( Schultz and Solomon , 1962 ; Higgins etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>A common denominator for the genes known to be regulated by H-NS ( e.g. regulation of genes at stationary growth phase , starvation , changes in high osmolarity , low temperature shock , and altered DNA topology ) seems to be their involvement in different kinds of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Osmotic upshift Cold shock Discussion Participation of H-NS in growth rate regulation One of the hallmarks in the regulation of rRNA-synthesis , apart from stringent control , is growth rate regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>346 Moreover , the data indicate that H-NS is involved in both the growth phase-associated and osmotic regulation of rpoS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Although the regulation of rRNA-synthesis is clearly related to the growth phase-dependent expression of the activator FIS , a similar relationship to the concentration of H-NS is less clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>A small RNA , DsrA , has been implicated as an antisilencer ofH-NS in rcsA regulation thatcontrols capsule-polysaccharide-synthesis ( SLEDJESKan I d GOT-TESMAN 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
<li>Clearly , DsrA could be important for coordinate regulation of these pathways and those controlled by H-NS and RpoS , thereby acting as a global orchestrator of a broad sweep of biological functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>Since RpoS and H-NS play an important role in globally regulating genes in response to changing environments , it is not surprising that AI-2-related QS networks utilize their uptake regulators ( e.g. , LsrR and LuxP ) , together with DsrA , in a hierarchical modality for mediating prompt responses to environmental stimuli and extracellular stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557827" target="_blank">17557827</a>]<br></li>
<li>H-NS plays a pleiotropic role in bacterial response to environmental stimuli such as temperature , osmolarity , pH , and starvation ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>H-NS regulates bacterial response to osmolarity and other stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>H-NS is a major component of chromatin in enteric bacteria which plays a structural role in organizing the chromosome and inﬂuences DNA rearrangements as well as the expression of many genes involved in the response to environmental signals , such as osmolarity , pH , oxygen availability , nutrient status , and temperature ( reviewed in references 18 and 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>H-NS regulates many stress response genes by acting as a major component of chromatin and by responding to changes in osmolarity ( 5 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>The results strongly support the direct negative regulation of the global regulators H-NS , LRP and StpA in the control of 6S RNA transcription and suggest that FIS probably has a dual functional role in ssrS reg ulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_173">
Pul U , Wurm R , Wagner R : The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></label>
<input id="rp_similarity_173", type="checkbox">
<div class="RP_similar">
<li>Pul , U. , Wurm , R. , and Wagner , R. ( 2007 ) The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>, Wurm , R. , and Wagner , R. ( 2007 ) The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromolecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>The role of LRP and H-NS in transcription regulation : involvement of synergism , allostery and macromo-lecular crowding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li></div>
<li>Summary The LysR-type transcription factor LeuO is involved in regulation of pathogenicity determinants and stress responses in Enterobacteriaceae , and acts as antagonist of the global repressor H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>The interaction of Lrp and H-NS in the regulation of another amino-acid-biosynthetic pathway , strongly encourages the view of a coupling between H-NS fimction and amino acid biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>6A ) and TP2101 , TP2101-1 ( a ^ - ) ( Fig. 7A ) whereas YMel , YMel-42 ( H-NS '' , o ^ '' ) , MC106M ( H-NS ) , MC1061-14 ( H-NS - , CT ^ ) ( Rg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>kan ' ) , YMel-3 ( a ^ - ) , YMel-42 ( H-NS - , cT ^ ) , MC1061 , MC1061-1 andMC1061-14 ( H-NS '' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>H-NS negatively regulates evgA and gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Peak A , component A ( H-NS ) ; peak B , component B ( HF-I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2020545" target="_blank">2020545</a>]<br></li>
<li>6A ) , TP2600 ( H-NS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>o ^ '' ) ( Fig. 10A ) and from TP2101 , TP2600 ( H-NS ) , and TP2600-1 ( H-NS , a ^ ) ( Fig. 10B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>In both strains , there was a rapid repression of stpA transcription following carbon starvation ( Fig. 5B ) , which is obviously therefore H-NS independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>To determine if H-NS affects its own transcription , we constructed an isogenic derivative of CU236 , which carries a hns : : neo ( Kin r ) insertion ( CU237 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Schematic model for the function of Fis and H-NS in activating and repressing rRNA-transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Effects of H-NS and potassium glutamate on in vitro transcription from supercoiled pHYD351 ( A ) and pHYD355 ( B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>Whether H-NS plays an additional positive role in regulating transcription from these promoters in response to specific signals is still unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Because of the role of H-NS in controlling iron utilization gene transcription , we wanted to explore if H-NS played a similar regulatory role for the amino acid and carbohydrate utilization genes in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>In contrast to many other virulence genes in which H-NS plays only a repressive role , the protein plays both a positive and a negative role in papBA transcription ( 67 , 70 , 74 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>H-NS plays a dual role in the CfaD system ( Fig. 9B ) , acting both as a repressor of CfaD transcription and as a CfaD antagonist for expression of the structural genes of CfaI ﬁmbriae ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>This suggests that relief of repression by H-NS plays a role in translational induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Besides RpoS and LexA , show that H-NS also plays we a key role in Pi-starved cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Our rcsults indicate that a role of H-NS is to maintain a rapid metabolism of glucose to acetare in Pi-starved cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>The role of H-NS in one-carbon-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>The analysis of suppressor mutations strongly suggested a major role of RcsB in the motility and acid stress resistance phenotypes related to H-NS control of these processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>In the course of our work on the metabolic controls involving the one carbon ( C l ) pool ( ie methylene tetra-hydrofolate and its derivatives : thymine , purines , serine and glycine , methionine , pantothenic acid and N-formylmethionyl tRNA for initiation of translation ) we discovered that H-NS was involved in the regulation of their synthesis , as will be described below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Components involved include a two-component signal-transduction-system ( EvgAS ) , three activators ( GadE , X , W ) , two sigma factors ( sigma S and sigma 70 ) , two global regulators ( Crp , H-NS ) and at least one regulatory loop ( GadX -- GadW ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12940989" target="_blank">12940989</a>]<br></li>
<li>Identiﬁcation of the effectors involved in GadX activation and further investigation of the interactions between regulatory components of the gad system ( H-NS , RpoS , CRP , and GadX ) may provide further insight into the molecular circuitry underlying the glutamic acid-based acid stress response of E. coli A. Tramonti and P. Visca contributed equally to this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Other regulators , such as H-NS and RcsB , are involved in csgD repression in response to closely related stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Our results linking H-NS to Lrp lend further support to the notion that both regulators are involved in responses to stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>However , H-NS also directly controls specific regulators of these pathways ( e.g. cadC ) and genes involved in general stress resistance ( hdeAB , hdeD , dps , adiY ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>This could also explain why three regulators H-NS , HdfR and RcsB are directly involved in the control of both glutamate-dependent acid stress response and the flagellum biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Conclusion In E. coli , the control of acid stress resistance is achieved by the concerted efforts of multiple regulators and overlapping systems , most of the genes directly involved in acid resistance being both controlled by RcsB-P/GadE complex and by at least one other regulator such as H-NS , HdfR , CadC or AdiY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 4.8 5.2 4.6 3.1 3.1 3.5 2.1 2.0 4.2 4.2 5.1 4.6 Oxidative stress and iron-sulfur-cluster-assembly genes Relevant regulatory proteins ahpC alkyl hydroperoxide reductase small subunit ahpF alkyl hydroperoxide reductase large subunit dadA D-amino acid dehydrogenase dadX alanine racemase OxyR * ( ) MetJ ( ) OxyR * ( ) MetJ ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) IscR * ( ) SoxRS * ( ) OxyR * ( ) OxyR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) OxyR * ( ) P1 , H-NS ( ) Lrp * ( ) NhaR ( ) RcsB ( P2 , H-NS ( ) Lrp * ( ) IscR * ( ) OxyR ( ) IHF ( ) Fur SoxR ( ) SoxS ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) 3.3 2.1 3.4 1.9 3.2 1.3 7.2 1.6 2.2 2.8 6.3 6.1 3.4 3.0 2.1 5.3 2.1 2.8 3.0 9.1 5.8 4.1 fdx ferredoxin fur ferric uptake regulator 2.1 3.4 2.4 2.8 3.3 10.1 7.0 5.1 2.0 1.9 1.8 2.7 1.8 1.8 2.8 2.5 11.2 5.7 4.4 1.8 1.1 3.6 grxA glutaredoxin 1 hscA co-chaperone involved in IscU iron-sulfur-cluster-assembly hscB co-chaperone involved in IscU iron-sulfur-cluster-assembly iscA iron-sulfur-cluster-assembly protein iscR transcriptional regulator iscS cysteine desulfurase iscU scaffold protein involved in iron-sulfur-cluster-assembly iscX possible role in iron-sulfur-cluster-biosynthesis katG bifunctional hydroperoxidase I osmC organic hydroperoxidase Within 5 min of air exposure the abundance of the soxS transcript is increased 8-fold ( Table 2 ) , indicating activation of the sensor SoxR , which enhances soxS expression , which in turn controls the SoxRS regulon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>This study has identiﬁed at least three factors which in an interactive manner may be involved in stimulating expression from this promoter , namely , growth at low temperature and inactivation of the Rho and H-NS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>Our data suggest that additional regulatory proteins , i.e. cAMP-CRP and probably Lrp and H-NS , are crucially involved in generating this speci ® c pattern of csiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Global regulatory proteins CRP , FNR , and H-NS are involved in the synthesis of carnitine dehydratase activity ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>However , the in vitro results support a direct effect for H-NS , LRP and also indicate that StpA may be involved in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>We show that H-NS is involved in the transcriptional regulation of the kbl/tdh operon , which is also Lrp regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>In contrast , H-NS is almost always involved in transcriptional silencing ( for example , see Mukerji and Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Genes under the control of H-NS include those involved in bacterial adaptation to changes in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>This result supports the conclusion that H-NS is involved in silencing the Pcas promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>H-NS is involved in the conden ¬ * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>Genetic studies have identi ® ed cAMP-CRP , Lrp , integration host factor ( IHF ) and H-NS as such modulators of sS-dependent genes ( see Hengge-Aronis ( 1996b ) for review , and references cited therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Lastly , we demonstrate that H-NS controls the amino acid , carbohydrate , and iron utilization genes tested in this study , linking this global regulator to acquisition of these important substances required for growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>This suggests that overexpression of either GadX or GadE alone , in an FB8 genetic background , is not sufﬁcient to induce expression of genes involved in the bacterial response to low pH when cells are grown at neutral pH. Overexpression of gadX signiﬁcantly induced the expression of 48 genes of the H-NS regulon ( Table 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>In contrast , several genes involved in adaptation to low pH are strongly induced in a H-NS deﬁcient strain , resulting in an increased resistance to acidic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>In particular , H-NS was shown , unlike StpA , to control several genes involved in bacterial motility and low pH resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Several environmentally regulated genes are under the negative control of H-NS , including rpoS encoding another global regulator , the stationary-phase sigma factor RpoS ( 2 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>According to our results , the master regulators H-NS and RpoS are hierarchically involved in the transcriptional-control of gad expression : H-NS prevents gad expression during the exponential growth whereas the alternative sigma factor RpoS relieves H-NS repression during the stationary phase , directly or indirectly accounting for transcription of gad genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>These results indicate that H-NS is involved in the negative transcriptional regulation of the gad genes and raise the possibility that H-NS would directly silence the gadA and gadB promoters and/or indirectly repress gad expression through the control of trans-acting factors , probably the stationary-phase factor ss .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Apart from the gadBC operon , the most important genes involved in acid resistance are present within the acid fitness island ( AFI ) , a 15 kb region both repressed by H-NS and under the control of RpoS [ 10,11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>H-NS was first identified as a transcription factor by Jacquet and co-workers [ 2 ] and is involved in many general processes such as control of genes involved in osmoregulation and virulence ( for reviews see [ 3 , 4 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>These included several additional genes , including the hde locus and many genes involved in ﬂagella formation , previously known to be under H-NS control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>H-NS and RpoS are involved in the transcriptional-control of gad genes Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>The previously reported strong derepression of acid resistance genes in hns mutants involves gadX ( 28 ) and most likely reﬂects the strong negative regulation of S by H-NS ( 5 , 69 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>H-NS controls the regulation of carbohydrate and amino acid utilization genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>H-NS regulon genes and their regulation by environmental parameters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Therefore , multiple systems ( EnvZ/OmpR , Cpx , Rcs , and H-NS ) appear to be involved in sensing environmental osmolarity , leading to sophisticated regulation of the curli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Mutations in hns result in various phenotypes , because H-NS is involved in the regulation of a variety of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>This suggests that H-NS plays an indirect role in the regulation of many genes identified in this study and further supports a high position of H-NS in the regulatory hierarchy of bacterial physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Despite possible pleiotropic effects , first of all , we provide strong experimental evidence that H-NS plays a direct role in the regulation of this acid resistance system , identifying gadA and gadX as target genes responsive to H-NS control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Lucchini S , Rowley G , Goldberg MD , Hurd D , Harrison M , Hinton JC : H-NS mediates the silencing of laterally acquired genes in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Similarly , the virulence regulon in V. cholerae , which is a veritable mosaic of ancestral and horizontally acquired genes , uses H-NS as a transcriptional repressor of imported genes52 ,103,104 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>For a number of other genes , H-NS has been shown to act as repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>H-NS acts as a repressor of most of these genes , which include gadX , gadW , and gadE ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>H-NS represses a large number of genes that constitute the rpoS regulon in the exponential phase ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>Discussion H-NS has generally been considered a nonspecific silencer of many genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>H-NS might control many genes by participating in other global networks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Stem-loop 2 decreases the translation of H-NS and also the ability of H-NS to repress transcription of rcsA and other H-NS-repressed genes ( 13 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
<li>The H-NS binding motif is associated with not only the genes encoding the structural genes of ﬂagella but also with the speciﬁc sigma factor ( ﬂiA ) and anti-sigma factor ( ﬂgM ) genes for regulating ﬂagella biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>In contrast , the hns-118 strain TP504 showed no flagella ( Fig. 2B and D ) H-NS afects the transcription of the flagellum biosynthesis The so-called deep-rough of E. coli known genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li>
<li>The limited number of genes which have been studied in detail are derepressed in hns mutants , which has led to H-NS being called a ` silencer ' or ` repressor '' of transcription ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Expression of FlhDC is tightly regulated by multiple transcription factors , including CRP , H-NS and OmpR , and hence the genes of the ﬂagellar cascade are regulated in response to many different environmental inputs ( Shin and Park , 1995 ; Sout-ourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>Phase variation of the fimA promoter as well as transcription of the fim genes is influenced by several global , environmental regulators ; for example , H-NS reduces the frequency of phase variation [ 9 ] and stimulates the transcription of fimA in a locked switched-on mutant [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Additional regulators acting on some of these genes include transcription factors speciﬁc for certain stress response , such as OxyR , and also some more global regulators , such as H-NS or Lrp ( Hengge-Aronis , 1996b and references herein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>After leucine-responsive protein , H-NS and σs , NhaR and RcsB are the fourth and ﬁfth regulators of osmC transcription to have been identiﬁed The participation of so many factors in the regulation of osmC illustrates the notion of cooperation of global regulators in the ﬁne-tuning of stress-inducible genes ( Bouvier et al. , 1998 ; Hengge-Aronis , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></li>
<li>Transcription factors may be global or local in scope depending on various parameters including the numbers of genes that they regulate , their propensity to be involved in combinatorial control , and the numbers of environmental conditions in which they are active .10 One of the global regulators of transcription in E. coli is the protein called H-NS , whose regulatory scope extends to 20 % of the genes encoded by the genome .11 -- 13 H-NS is a small protein of 137 amino acids , which recognises DNA that is A+T - rich in sequence12 ,14,15 and/or specific DNA geometries such as intrinsic bending16 -- 18 and Holliday 19 junctions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>For example , several global regulators , such as cyclic AMP receptor protein and H-NS affect transcription of the cps genes ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763957" target="_blank">8763957</a>]<br></li>
<li>TABLE 2 Primers used in this study Name Gfp-mut3-rev PnrdABamHI-up PnrdASmaI-lw PA-450 ext-lw PA-400 extpho-up PnrdA-50pb-up PnrdA-50pb-lw PnrdA-300pb-lw PnrdA-400pb-up PnrdA-120pb-lw PnrdA-172pb-up PnrdA-235pb-up StpA-up StpA-lw E. coli ﬁs-up E. coli ﬁs-lw PBR-E HNS-3 HNSBDist M13-dir M13-rev kT k1 k2 Sequence ( 5 = ¡ 3 =) GAATTGGGACAACTCCAGTG Application Cloning assessment Cloning , probe ampliﬁcation Cloning Cloning Cloning Probe ampliﬁcation Probe ampliﬁcation Probe ampliﬁcation Probe ampliﬁcation Probe ampliﬁcation Probe ampliﬁcation Probe ampliﬁcation Mutation assessment Mutation assessment Mutation assessment Mutation assessment Cloning assessment Mutation assessment Mutation assessment Cloning assessment Cloning assessment Cloning assessment Cloning assessment Cloning assessment ACCCGGGATCATTTTCTATAAGACGG AGGATCCAGCAGATTCTGATTCATG CGCATGCATTGCACCGGGTTGAACAAT GTACGCTTAAAGTCATGAATAATTTTCTTATAATATAAGG ATCCACAAAGTTATGCACTTGC TAACTCAGGAAGGAAAAAGTGG TTGTTGATGGCGAATGGTTGTT TTAAAGTCATGAATAATTTTCTTATAATATAAG AGAGAAAAATTTGTTAAAAATAACTGTTCG AGGTTAGATAAATTGATATAGATGGC TAGATCAATTTTTGCAATCATTAGCAA GATCGCTTACACTACGCGACG CAGCGACATCCGGCCTC GCGTAAATTCTGACGTACTGAC TTTACGCAGCGTACCACGG ATTATCATGACATTAACC CCACCCCAATATAAGTTTGAG CCGGATCCTAAAAAATCCCGC GTTTTCCCAGTCACGAC CAGGAAACAGCTATGAC CGGCCACAGTCGATGAATCC CAGTCATAGCCGAATAGCCT CGGTGCCCTGAATGAACTGC RESULTS H-NS represses expression of class Ia and III RNRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_257">
Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional auto-repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></label>
<input id="rp_similarity_257", type="checkbox">
<div class="RP_similar">
<li>Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional autorepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_259">
Falconi , M. , Higgins , N.P. , Spurio , R. , Pon , C.L. , and Gual-erzi , C.O. ( 1993 ) Expression of the gene encoding the major bacterial nucleotide protein H-NS is subject to transcriptional auto-repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></label>
<input id="rp_similarity_259", type="checkbox">
<div class="RP_similar">
<li>Falconi , M. , Higgins , N.P. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1993 ) Expression of the gene encoding the major bacterial nucleotide protein H-NS is subject to transcriptional autorepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li></div>
<li>Mol Gen Genet 236 , 17I-i 78 15 Falconi M. Higgins N. Spurio R , Pen C. Gualerzi CO ( 1993 ) Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional auto-repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_262">
[ 9 ] Falconi M. , Higgins N.P. , Spurio R. , Pon C.L. , Gualerzi C.O. , Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional autorepression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></label>
<input id="rp_similarity_262", type="checkbox">
<div class="RP_similar">
<li>Falconi , M. , Higgins , N.P. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1993 ) Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional autorepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Falconi , M. , Higgins , N , P. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1993 ) Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional autorepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li></div>
<li>Mol Microbiol I0 , 273-282 16 La Teana A , Brandi A , Falconi M. Spurio R , Pen CL , Guale ~ i CO ( 1991 ) IdentifiCation of a cold shock transcriptional enhancer of the Escherichia coil gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Agric Biol Chem 55:313 % 3141 Teana AL , Brandi A , Falconi M , Spurio R , Pon CL , Gualerzi CO ( 1991 ) Identification of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_267">
[ 27 ] A. La Teana , A. Brandi , M. Falconi , R. Spurio , C.L. Pon , C.O. Gua-lerzi , Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14527658" target="_blank">14527658</a>]<br></label>
<input id="rp_similarity_267", type="checkbox">
<div class="RP_similar">
<li>[ 6 ] La Teana A. , Brandi A. , Falconi M. , Spurio R. , Pon C.L. , Gualerzi C.O. , Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>[ 38 ] La Teana A. , Brandi A. , Falconi M. , Spurio R. , Pon C.L. , Gualerzi C.O. , Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_270">
J Bacteriol 173:4116 M123 La Teana A , Brandi A , Falconi M , Spurio R , Pon CL , Gualerzi CO ( 1991 ) Identification of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></label>
<input id="rp_similarity_270", type="checkbox">
<div class="RP_similar">
<li>La Teana , A. , Brandi , A. , Falconi , M. , Spurio , R. , Pon , C.L. , and Gualerzi , CO. ( 1991 ) Identification of a cold shook transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984109" target="_blank">7984109</a>]<br></li>
<li>La Teana A , Brandi A , Falconi M , Spurio R , Pon CL , Gualerzi CO : Identification of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23815967" target="_blank">23815967</a>]<br></li>
<li>La Teana , A. , Brandi , A. , Falconi , M. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1991 ) Identification of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298285" target="_blank">11298285</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>La Teana A , Brandi A , Falconi M , Spurio R , Pon CL & Gualerzi CO ( 1991 ) Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177308" target="_blank">18177308</a>]<br></li>
<li>La Teana , A. , Brandi , A. , Falconi , M. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1991 ) Identi ® cation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10476034" target="_blank">10476034</a>]<br></li>
<li>La Teana , A. , Brandi , A. , Falconi , M. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1991 ) Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>[ 9 ] La Teana , A. , Brandi , A. , Falconi , M. , Spurio , R. , Pon , C.L. and Gualerzi , C.O. ( 1991 ) Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16405963" target="_blank">16405963</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_278">
CL - , and Gualerzi , CO. ( 1991 ) Identification of a cold shock transcriptional enhancer of the Escherichia coti gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></label>
<input id="rp_similarity_278", type="checkbox">
<div class="RP_similar">
<li>CO. ( 1991 ) Identification of a cold shook transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Identification of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1325964" target="_blank">1325964</a>]<br></li>
<li>Identiﬁcation of a cold shock transcriptional enhancer of the Escherichia coli gene encoding nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631696" target="_blank">8631696</a>]<br></li></div>
<li>Identification of a cold shock transcriptional enhancer o Escherichia coli gene encoding nucleoid protein H-NS ABSTRACT The hns ( 27 min ) gene encoding the 15.4-kDa nucleoid protein H-NS was shown to belong to the cold shock regulon of Escherichia coli , its expression being enhanced 3 - to 4-fold during the growth lag that follows a shift from 37C to 100C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1961761" target="_blank">1961761</a>]<br></li>
<li>N.P. , Gualerzi , CO. , and Pon , C.L ( 1991 ) Transcriptional control of the expression of the gene for the nucleoid protein H-NS of Escherichia coli Atti Ass Genet Ital37 :123 -124 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>H-NS Is a Novel Transcriptional Modulator of the Ribonucleotide Reductase Genes in Escherichia coli Maria del Mar Cendra , a Antonio Juárez , a , b Cristina Madrid , b Eduard Torrentsa Institute for Bioengineering of Catalonia , Cellular Biotechnology , Barcelona , Spaina ; Departamento de Microbiologia , Facultat de Biologia , Universitat de Barcelona , Barcelona , Spainb Ribonucleotide reductases ( RNRs ) are essential enzymes for DNA-synthesis because they are responsible for the production of the four deoxyribonucleotides ( dNTPs ) from their corresponding ribonucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>[ 9 ] Rowbury , R.J. ( 1997 ) Regulatory components , including integration host factor , CysB and H-NS , that in # uence pH responses in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12892888" target="_blank">12892888</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_286">
Regulatory components , including integration host factor , CysB and H-NS , that inﬂuence pH responses in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14663078" target="_blank">14663078</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10698756" target="_blank">10698756</a>]<br></label>
<input id="rp_similarity_286", type="checkbox">
<div class="RP_similar">
<li>Regulatory components , including integration host factor , CysB , and H-NS , that inﬂuence pH responses in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li></div>
<li>Controlling biofilm-formation , prophage excision and cell-death by rewiring global regulator H-NS of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059164" target="_blank">21059164</a>]<br></li>
<li>H-NS controls metabolism and stress tolerance in Escherichia coli O157 : H7 that inﬂuence mouse passage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>[ 25 ] V. Sanchez-Torres , T. Maeda , T.K. Wood , Global regulator H-NS and lipoprotein NlpI inﬂuence production of extracellular DNA in Escherichia coli , Biochem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23131563" target="_blank">23131563</a>]<br></li>
<li>Global regulator H-NS and lipoprotein NlpI influence production of extracellular DNA in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21181144" target="_blank">21181144</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_292">
H-NS controls pap and daa ﬁmbrial transcription Escherichia coli in response to multiple environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></label>
<input id="rp_similarity_292", type="checkbox">
<div class="RP_similar">
<li>White-Ziegler , C.A. , Villapakkam , A. , Ronaszeki , K. , and Young , S. ( 2000 ) H-NS controls pap and daa ﬁmbrial transcription in Escherichia coli in response to multiple environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li></div>
<li>White-Ziegler , C.A. , Angus Hill , M.L. , Braaten , B.A. , van der Woude , M.W. , and Low , D.A. ( 1998 ) Thermoregulation of Escherichia coli pap transcription : H-NS is a temperaturedependent DNA-methylation blocking factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_295">
Thermoregulation of E. coli pap transcription : H-NS is a temperature-dependent DNA-methylation blocking factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></label>
<input id="rp_similarity_295", type="checkbox">
<div class="RP_similar">
<li>Thermoregulation of Escherichia coli pap transcription : H-NS is a temperature-dependent DNA-methylation blocking factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li></div>
<li>Quantitative control of the stationary phase-specific factor , S , in Esch-erichia coli : Involvement of the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>]<br></li>
<li>Yamashino , T. , Ueguchi , C. , and Mizuno , T. ( 1995 ) Quantitative control of the stationary phase-speciﬁc sigma factor , s s , in Escherichia coli : involvement of the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_299">
Quantitative control of the stationary phase-speci ® c sigma factor , ss , in Escherichia coli : involvement of the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9126832" target="_blank">9126832</a>]<br></label>
<input id="rp_similarity_299", type="checkbox">
<div class="RP_similar">
<li>Quantitative control of the stationary phase-speciﬁc sigma factor , sS , in Escherichia coli : involvement of the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626294" target="_blank">8626294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Quantitative control of the stationary phase-speciﬁc sigma factor , ss , in Escherichia coli : involvement of the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Quantitative control of the stationary phase-speciﬁc sigma factor , S , in Escherichia coli : involvement of the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_303">
The nucleoid protein H-NS is involved in the negative control of gadA and gadBC transcription during the exponential growth phase under oxidative conditions , while the alternative sigma factor RpoS is responsible for gad expression at the stationary phase but not during fermentative growth ( 6 , 7 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></label>
<input id="rp_similarity_303", type="checkbox">
<div class="RP_similar">
<li>The nucleoid protein H-NS is involved in the negative con trol of gadA and gadBC transcription during exponential phase , while the alternative sigma factor RpoS is responsible for Gad expression in stationary phase ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li></div>
<li>DsrA RNA regulates translation of RpoS message by an anti-antisense mechanism , independent of its action as a antisilencer of transcription ABSTRACT DsrA RNA regulates both transcription , by overcoming transcriptional silencing by the nucleoid-associated H-NS protein , and translation , by promoting efficient translation of the stress s factor , RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770508" target="_blank">9770508</a>]<br></li>
<li>Riboregulation in Escherichia interactions at multiple loci ABSTRACT DsrA is an 87-nt untranslated RNA that regulates both the global transcriptional silencer and nucleoid protein H-NS and the stationary phase and stress response sigma factor RpoS ( ss ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>DsrA inhibits the synthesis of the predominant nucleoid protein and global transcriptional silencer H-NS ( 13 ) and enhances the translation of the stationary phase and major stress response sigma factor RpoS ( ss ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_308">
Identification , cloning , nucleotide sequence and chromosomal map location of hns , the structural gene for Eschenichia coli DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381784" target="_blank">8381784</a>]<br></label>
<input id="rp_similarity_308", type="checkbox">
<div class="RP_similar">
<li>Identiﬁcation , cloning , nucleotide sequence and chromosomal map location of hns , the structural gene for Escherichia coli DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11872702" target="_blank">11872702</a>]<br></li>
<li>Identification , cloning , nucleotide sequence and chromosomal map location of hns , the structural gene for Escherichia coli DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1340475" target="_blank">1340475</a>]<br></li>
<li>[ 2 ] Pon C.L. , Calogero R.A. , Gualerzi C.O. , Identiﬁcation , cloning , nucleotide sequence and chromosomal map location of hns , the structural gene for Escherichia coli DNA-binding protein H-NS , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>Pon , C.L. , Calogero , R.A. , and Gualerzi , C.O. ( 1988 ) Identiﬁcation , cloning , nucleotide sequence and chromosomal map location of hns , the stuctural gene for Escherichia coli DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Pon , C L , Calogero , R.A. , and Gualerzi , CO. ( 1988 ) Identification , cloning , nucleotide sequence and chromosomal map location of hns , the structural gene for Escherichia coli DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Pon , C.L. , Calogero , R.A. , and Gualerzi , C.O. ( 1988 ) Identi-ﬁcation , cloning , nucleotide sequence and chromosomal map location of hns , the structural gene for Escherichia coli DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li></div>
<li>( Received forpublication , August 18 , 1993 , and in revised form , November 12 , 1993 ) The Escherichia coli hns gene encodes the abundant cuits control theintracellularconcentration of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>E. ( 1990 ) The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( H1 al , a component ofthe Escherichia co1iK-12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>63 t-642 9 May G , Dersch P. ttaardt M , Middendoff A , Bremer E ( 1990 ) The osnuZ ( bglY ) gene encodes the DNA-binding protein H-NS ( Hia ) , a component of the Escherichia coli K !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( Hla ) , a component of the Escherichia coli K-12 nucle-oid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381784" target="_blank">8381784</a>]<br></li>
<li>May , G. , Dersch , P. , Haardt , M. , Middendorf , A. , and Bremer , E. ( 1990 ) The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( H1a ) , a component of the Escherichia coli K12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_320">
The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( H1a ) , a component of the Escherichia coli K-12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></label>
<input id="rp_similarity_320", type="checkbox">
<div class="RP_similar">
<li>The osmZ ( bg1Y ) gene encodes the DNA-binding protein H-NS ( Hla ) , a component of the Escherichia coli nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715407" target="_blank">1715407</a>]<br></li>
<li>The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( H1a ) , a component of the Escherichia coli K12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( Hla ) , a component of the Escherichia coli K12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8169209" target="_blank">8169209</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li></div>
<li>Cell 32:78 % 797 Jacquet M , Cukier-Kahn R , Pla J , Gros P ( 1971 ) A thermostable protein factor acting on in vitro DNA transcription , Biochem Biophys Res Commun 45:1597 -1607 May G , Dersch P , Haardt M , Middendorf A , Bremer E ( 1990 ) The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( Hla ) , a component of the Eseherichia coli K12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Cell 50 : 495-508 May G , Dersch P , Haardt M , Middendorf A , Bremer E ( 1990 ) The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( H1 a ) , a component of the Eseherichia coli K12 nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Proc Natl Acad Sci USA 85:2820 -2824 May G , Dersch P , Haardt M , Middendorf A , Bremer E ( 1990 ) The osmZ ( bglY ) gene encodes the DNA-binding protein H-NS ( H1 a ) , a component of the Eseherichia coli K12 nucleotid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_327">
The ChiA ( YheB ) protein of Escherichia coli K-12 is an endochitinase whose gene is negatively controlled by the nucleoid-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></label>
<input id="rp_similarity_327", type="checkbox">
<div class="RP_similar">
<li>Francetic , O. , Baudaut , C. , Rimsky , S. , and Pugsley , A. ( 2000a ) The ChiA ( YheB ) protein of Escherichia coli K-12 is an endochitinase whose gene is negatively controlled by the nucleoid-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Francetic , O. , Baudaut , C. , Rimsky , S. , and Pugsley , A. ( 2000 ) The ChiA ( YheB ) protein of Escherichia coli K-12 is an endochitinase whose gene is negatively controlled by the nucleoid-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Francetic , O. , Badaut , C. , Rimsky , S. and Pugsley , A.P. ( 2000 ) The ChiA ( YheB ) protein of Escherichia coli K-12 is an endochitinase whose gene is negatively controlled by the nucleoid-structuring protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></li></div>
<li>Escherichia coli protein analogs StpA and H-NS : Regulatory loops , similar and disparate effects on nucleic acid dynamics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_332">
Atlung , T. & Hansen , F.G. ( 2002 ) Effect of different concentrations of H-NS protein on chromosome replication and the cell-cycle in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19371375" target="_blank">19371375</a>]<br></label>
<input id="rp_similarity_332", type="checkbox">
<div class="RP_similar">
<li>Effect of different concentrations of H-NS protein on chromosome replication and the cell-cycle in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_334">
Katayama , T. , Takata , M. , and Sekimizu , K. ( 1996 ) The nucleoid protein H-NS facilitates chromosome DNA-replication in Escherichia coli dnaA mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></label>
<input id="rp_similarity_334", type="checkbox">
<div class="RP_similar">
<li>The nucleoid protein H-NS facilitates chromosome DNA-replication in Escherichia coli dnaA mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li></div>
<li>H-NS is an approximately 15.4-kDa nucleoid-associated Escherichia coli protein involved in bacterial chromatin condensation ( 12 , 49 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In Escherichia coli , H-NS , a nucleoid-associated protein , is one of the global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Mol Gen Genet ( 1993 ) 236:171 -178 OIGG © Springer-Verlag 1993 Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H - N S functions as a repressor of its own transcription Chiharu Ueguchi , Minoru Kakeda , and Takeshi Mizuno Laboratory of Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya464 , Japan ReceivedApril 22 , 1992 / AcceptedJuly 6 , 1992 properties of H-NS , was made by a different biochemical approach ( Bracco et al. 1989 ; Yamada et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Kakeda , M. , and Mizuno , T , ( 1993 ) Autoregulatory expression of the Eschehctiia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Agric Biol Chem 55:3139 -3141 Ueguchi C , Kakeda M , Mizuno T ( 1992 ) Autoregulatory expression of the Eseherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor for its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_341">
[ 22 ] Ueguchi C. , Kakeda M. , Mizuno T. , Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></label>
<input id="rp_similarity_341", type="checkbox">
<div class="RP_similar">
<li>Ueguchi , C. , Kakeda , M. , and Mizuno , T. ( 1993 ) Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein -- H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Ueguchi , C , Kakeda , M. , and Mizuno , T. ( 1993 ) Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997164" target="_blank">7997164</a>]<br></li>
<li>Ueguchi , C. , Kakeda , M. , and Mizuno , T. ( 1993 ) Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>and Mizuno , T. ( 1993 ) Auto-regulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984093" target="_blank">7984093</a>]<br></li>
<li>Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li></div>
<li>Ueguchi , C , Kakeda , M. , and Mizuno , T , ( 1993 ) Autoregulatory expession of the Esherichia ccli hns gene encoding a nucleoid protein -- H-NS functions as a repressor of its own transcription , Mol Gen Genet 236 : 171 -178 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>EMBO J 12:1039 -1046 Ueguchi C , Kakeda M , Mizuno T ( 1993 ) Autoregulatory expression of the Eseheriehia eoli hns gene encoding a nucleoi protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Kaneda , M. and Mizuno , T. ( I 993 ) Autoregulatory expression of the Eschen ' chia co/i hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_350">
Function of the Escherichia coli nucleoid protein , H-NS : Molecular analysis of a subset of proteins whose expression is enhanced in a hns deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></label>
<input id="rp_similarity_350", type="checkbox">
<div class="RP_similar">
<li>Function of the Escherichia coli nucleoid protein , H-NS : molecular analysis of a subset of proteins whose expression is enhanced in an hns deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>Function of the Escherichia coli nucleoid protein , H-NS : molecular analysis of a subset of proteins whose expression is enhanced in a hns deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8244952" target="_blank">8244952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Yoshida , T. , Ueguchi , C. , Yamada , H. , and Mizuno , T. ( 1993a ) Function of the Escherichia coli nucleoid protein , H-NS : molecular analysis of a subset of proteins whose expression is enhanced in a hns-deletion background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Yoshida , T. , Ueguchi , C. , Yamada , H. , and Mizuno , T. ( 1993a ) Function of the Escherichia coli nucleoid protein , H-NS : molecular analysis of a subset of proteins whose expression is enhanced in a hns deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li></div>
<li>Sardesai , A.A. , and Gowrishankar , J. ( 2001 ) trans-acting mutations in loci other than kdpDE that affect kdp operon regulation in Escherichia coli : effects of cytoplasmic thiol oxidation status and nucleoid protein H-NS on kdp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Yoshida , T. , Ueguchi , C , and Mizuno , T. ( 1993a ) Physicai map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucieoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_357">
Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426343" target="_blank">12426343</a>]<br></label>
<input id="rp_similarity_357", type="checkbox">
<div class="RP_similar">
<li>[ 35 ] Yoshida T. , Uegushi C. , Mizuno T. , Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Yoshida , T. , Ueguchi , C. , and Mizuno , T. ( 1993c ) Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>[ 39 ] Yoshida , T. , Ueguchi , C. and Mizuno , T. ( 1993 ) Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li></div>
<li>For example , in the case of the expression of ﬂagellin genes , there seems to be stimulation , since the lack of a functional H-NS protein abolishes the motility of Escherichia coli ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The H-NS protein inﬂuences the expression of many genes in E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The bgl operon of Escherichia coli , involved in the catabolism of aromatic - glucosides such as salicin and arbutin , is one of the well-studied genetic systems regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>The H-NS protein is involved in the biogenesis of ¯ agella in Escheri-chia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_365">
The H-NS protein is involved in the biogenesis of flagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22422754" target="_blank">22422754</a>]<br></label>
<input id="rp_similarity_365", type="checkbox">
<div class="RP_similar">
<li>The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li></div>
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Dan-chin , A. , et al. ( 1994 ) The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>[ 11 ] Bertin P. , Terao E. , Lee E.H. , Lejeune P. , Colson C. , Danchin A. , Collatz E. , The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_369">
John Wiley and Sons , New York Bertin P , Terao E , Lee EH , Lejeune P , Colson C , Danchin A , Collatz E ( 1994 ) The H-NS protein is involved in the biogenesis of ¯ agella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></label>
<input id="rp_similarity_369", type="checkbox">
<div class="RP_similar">
<li>Bertin P , Terao E , Lee EH , Lejeune P , Colson C , Danchin A & Collatz E ( 1994 ) The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20528937" target="_blank">20528937</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Danchin , A. , and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of ﬂagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Danchin , A. and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of ¯ agella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E.H. , Lejeune , P. , Colson , C. , Danchin , A. , and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of flagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Bertin , P. , Terao , E. , Lee , E. H. , Lejeune , P. , Colson , C. , Danchin , A. , and Collatz , E. ( 1994 ) The H-NS protein is involved in the biogenesis of flagella in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li></div>
<li>Biochimie ( 1994 ) 76 , 1063-1070 © Soci ~ t6franfaise de biocbimie et biologie mol ~ culaire / Elsevier , Paris The role of H-NS in one-carbon-metabolism J R L a n d g r a f a , M L e v i n t h a l a , A D a n c h i n b aDepartment of Biological Sciences , Purdue University , W Lafayette , IN 47907 , USA ; bUnit ~ de Rdgulation de rExpression G # n #tique , lnstitut Pasteur , 75724 Paris Cedex , France Summary m The H-NS protein of Escherichia coli regulates the expression of genes involved in many general processes such as osmoregulation and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>H-NS is a small nucleoid-associated protein that acts as a transcriptional repressor of many E. coli genes involved in diverse functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>A Proteins Stimulate Expression Regulon in Escherichia coli H-NS and Stp of the Maltose JÖRGEN JOHANSSON ,1 BJÖRN DAGBERG ,2 EVELYNE RICHET ,3 AND BERNT ERIC UHLIN1 * Department of Microbiology1 and Department of Physiology ,2 Umeå University , S-90187 Umeå , Sweden , and Unité de Génétique Moléculaire , Institut Pasteur , 75724 Paris Cedex 15 , France3 Received 11 March 1998/Accepted 4 September 1998 The nucleoid-associated protein H-NS is a major component of the chromosome-protein complex , and it is known to inﬂuence the regulation of many genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The Escherichia coli nucleoid protein , H-NS ( or H 1a ) , appears to influence the regulation of a variety of unrelated E. coli genes and operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>The nucleoid-associated protein H-NS is known to inﬂuence the regulation of many genes in E. coli , and it appears that H-NS may cause silencing of many different operons ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>The H-NS protein is a global repressor of all chaperone -- usher ﬁmbriae in E. coli K-12 Our data demonstrated that , although cryptic under laboratory settings , ycb , yfc , yad , yra , sfm and yeh ﬁmbriae are nevertheless functional and potentially subjected to negative regulation alleviated under speciﬁc conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>These results demonstrated that the H-NS protein is a global repressor of E. coli K-12 chaperone -- usher ﬁmbriae operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Possible regulatory mechanisms H-NS acts as a repressor of multiple target genes in E. coli and other bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Silencing of horizontally-acquired genes in the bacterium Escherichia coli is caused by the global transcriptional repressor H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>It has been dem-onstrated that in Escherichia coli transcription of the Cascade genes ( casABCDE ) and to some extent the CRISPR array is repressed by heat-stable nucleoidstructuring ( H-NS ) protein , a global transcriptional repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>For operons involved in use of carbon sources in Escherichia coli , the most frequent global regulator is cAMP/CAP , responsible for catabolite repression , but histone-like proteins , IHF , H-NS , Fis and HU are often involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Moreover , several regulators have been shown to be involved in the control of swarming behavior in E. coli and S. typhimu-ium , in particular , the cyclic AMP ( cAMP ) - catabolite activator protein ( CAP ) complex ( 54 , 61 ) and the histone-like nucleoid-structuring ( H-NS ) protein ( 7 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>The smtA gene is not essential for cell-growth and its expression is positively regulated by H-NS , an Escherichia coli histone-like protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_388">
The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079897" target="_blank">9079897</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></label>
<input id="rp_similarity_388", type="checkbox">
<div class="RP_similar">
<li>Atlung , T. , Sund , S. , Olesen , K. , and Brønsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>References Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. , and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18383615" target="_blank">18383615</a>]<br></li>
<li>Atlung , T. , Sund , S. , Olesen , K. and Brondsted , L. ( 1996 ) The histone-like protein H-NS acts as a transcriptional repressor for expression of the anaerobic and growth phase activator AppY of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>Physical Map Location of a Set of Escherichia coli Genes ( hde ) Whose Expression Is Affected by the Nucleoid Protein H-NS Results of extensive genetic studies on the Escherichia coli nucleoid ( histone-like ) protein H-NS revealed that this protein influences the regulation of a variety of unlinked genes ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8244952" target="_blank">8244952</a>]<br></li>
<li>The H-NS protein of Escherichia coli belongs to the group of small histone-like proteins which function to compact the nucleoid and which inﬂuences a variety of cellular-processes such as transcription , recombination , and replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Summary Speciﬁcity of repression by the histone-like nucleoid structuring protein and pleiotropic regulator , H-NS , is exceptionally high in case of the Escherichia coli bgl ( b-glucoside ) operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>e-mail : schnetz!uni - koeln.de Institut fu $ r Genetik , Silencing of the Escherichia coli bgl operon is mediated by histone-like protein H-NS and affected by other pleiotropic regulators , including sigma factor RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
<li>[ 87 ] D. Tippner , H. Afﬂerbach , C. Bradaczek , R. Wagner , Evidence for a regulatory function of the histone-like E. coli protein H-NS in ribosomal RNA synthesis , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12504278" target="_blank">12504278</a>]<br></li>
<li>Posttranslational modiﬁcation of E. coli histone-like protein H-NS and bovine histones by short-chain poly - ( R ) -3 - hydroxybutyrate ( cPHB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Regulation of virulence-associated genes in Escherichia coli : involvement of the histone-like protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Williamson , H.S. , and Free , A. ( 2005 ) A truncated H-NS-like protein from enteropathogenic Escherichia coli acts as an H-NS antagonist .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Mol Gen Genet 226 : 367-376 Tanaka K , Yamada H , Yoshida T , Mizuno T ( 199lb ) Overproduction and rapid purification of the Escherichia coli histone-like protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Anucleate cell production by Escherichia coli Dhns mutant lacking a histone-like protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_403">
Nishino K , Yamaguchi A. Role of histone-like protein H-NS in multidrug-resistance of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18297445" target="_blank">18297445</a>]<br></label>
<input id="rp_similarity_403", type="checkbox">
<div class="RP_similar">
<li>Role of histone-like protein H-NS in multidrug-resistance of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li></div>
<li>JOURNAL OF BACTERIOLOGY , Feb. 1995 , p. 810 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Effects of Multicopy LeuO on the Expression of the Acid - Inducible Lysine Decarboxylase Gene in Escherichia coli XIAOLU SHI AND GEORGE N. BENNETT * Department of Biochemistry and Cell Biology , Rice University , Houston , Texas 77251 Received 10 June 1994/Accepted 19 November 1994 We previously reported that mutations in hns , the structural gene for the histone-like protein H-NS , cause derepressed expression of cadA , which encodes the acid-inducible lysine decarboxylase at noninducing pH ( pH 8.0 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836317" target="_blank">7836317</a>]<br></li>
<li>Protein H-NS in M Escherichia coli Role of Histone-Like ultidrug Resistance of Kunihiko Nishino1 ,2,3,4 and Akihito Yamaguchi2 ,3,4 * Department of Bacterial Infections , Research Institute for Microbial Diseases ,1 and Faculty of Pharmaceutical Science ,4 Osaka University , Suita , Osaka 565-0871 , and Department of Cell Membrane Biology , Institute of Scientiﬁc and Industrial Research , Osaka University , Ibaraki ,2 and Core Research Evolutional Science and Technology ( CREST ) , Japan Science and Technology Corporation ,3 Osaka 567-0047 , Japan Received 22 August 2003/Accepted 14 November 2003 The histone-like protein H-NS is a major component of the bacterial nucleoid and plays a crucial role in global gene regulation of enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>~ 9 GENE 06753 Histone-like proteins are required for cell-growth and constraint of supercoils in DNA ( Escherichia coli ; DNA-binding proteins ; hupA , hupB , himA , hz ' rvtDand hns mutations ; hygromycin phosphotransferase ; cryptic bgl operon ; recombinant DNA ; superhelical density ) Kayoko Yasuzawa , Naataka Hayashi , Naoki Goshima , Kyoko K & no , Fumio fmamoto and Yasunobu Kano Received by A. Nakazawa : 30 April 1992 ; Revised/Accepted : 14 June/I3 July 1992 ; Received at publishers : 27 July 1992 SUMMARY The gene hns of Escherichia coii K-12 , which encodes the histone-like protein H-NS , was inactivated by insertion of a DNA ( gene hph ) encoding hygromycin phosphotransferase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>T. , Spassky , A. , But , H. and Higgins , C.F. : Histone-like protein Hl ( H-NS ) , DNA supercoiling , and gene Kano , Y. and Imamoto , F. : Requirement of integration host factor ( IHF ) for growth of Escherichia coli deficient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Expression of the hemolysin operon in Escherichia coli is modulated by a nucleoid-protein complex that includes the proteins Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_410">
[ 29 ] Nieto , J.M. , Madrid , C. , Prenafeta , A. , Miquelay , E. , Balsalobre , C. , Carrascal , M. and Juárez , A. ( 2000 ) Expression of the hemolysin operon in Escherichia coli is modulated by a nucleoid -- protein complex that includes the proteins Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></label>
<input id="rp_similarity_410", type="checkbox">
<div class="RP_similar">
<li>Nieto , J.M. , Madrid , C. , Prenafeta , A. , Miquelay , E. , Balsalobre , C. , Carrascal , M. , and Juárez , A. ( 2000 ) Expression of the hemolysin operon in Escherichia coli is modulated by a nucleoid-protein complex that includes the proteins Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li></div>
<li>DISCUSSION The nucleoid-binding proteins of E. coli , assimilated to his-tone-like proteins , include HU , H-NS , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>Heteromeric interactions among nucleoid-associated bacter-ial proteins : localization of StpA-stabilizing regions in H-NS of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Johansson , J. , Eriksson , S. , Sondén , B. , Wai , S.N. , and Uhlin , B.E. ( 2001 ) Heteromeric interactions among nucle-oid-associated bacterial proteins : localization of StpA sta ¬ bilizing regions in H-NS of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>S0300908401012470/FLA H-NS and H-NS-like proteins in Gram-negative bacteria and their multiple role in the regulation of bacterial metabolism Philippe Bertin * , Florence Hommais , Evelyne Krin , Olga Soutourina , Christian Tendeng , Sylviane Derzelle , Antoine Danchin Unité de Régulation de l'Expression Génétique , Institut Pasteur , 28 , rue du Dr.-Roux , 75724 Paris cedex 15 , France ( Received 15 December 2000 ; accepted 19 January 2001 ) Abstract -- In Escherichia coli , the H-NS protein plays an important role in the structure and the functioning of bacterial chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Others play a major role in bacterial physiology , like H-NS in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Mol Microbiol 26:223 -- 236 Soutourina OA , Krin E , Laurent-Winter C , Hommais F , Danchin A , Bertin PN ( 2002 ) Regulation of bacterial motility in response to low pH in Escherichia coli : the role of H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>Regulation of bacterial motility in response to low pH in Escherichia coli : the role of H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601715" target="_blank">15601715</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22422754" target="_blank">22422754</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>Synthesis of H-NS , StpA , BpH3 and HvrA in E. coli clearly showed that all these proteins were able to complement , at least in part , various phenotypes of an hns mutant such as loss of motility ( Fig. 2 and Table 1 ) , b-glucoside utilization , serine susceptibility , mucoidy and reduction in growth yield ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>To confirm that there are multiple isoforms of H-NS , total cellular proteins from an hns + E coli strain and a congenic hns : : TnlO derivative were separated by two-dimen-sional gel electrophoresis ( fig 2A , B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Effect of H-NS and H-NS-like proteins on various H-NS-related phenotypes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>[ 32 ] Johansson J. , Dagberg B. , Richet E. , Uhlin B.E. , H-NS and StpA proteins stimulate expression of the maltose regulon in Escheri-chia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>[ 45 ] Johansson , J. , Dagberg , B. , Richet , E. and Uhlin , B.E. ( 1998 ) H-NS and StpA proteins stimulate expression of maltose regulon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>The EMBO Journal vol .15 no. 18 pp.4970-4980 , 1996 Coordinated and differential expression of histone-like proteins in Escherichia coli : regulation and function of the H-NS analog StpA Berit Sonden and Bernt Eric Uhlin1 Department of Microbiology , Ume'a University , S-901 87 Umea , Sweden ` Corresponding author The histone-like protein H-NS has been shown to influence the regulation-of-gene-expression at the transcriptional level in several Escherichia coli operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Coordinated and differential expression of histone-like protein in Escherichia coli : regulation and function of the H-NS analog StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>SondeÂn , B. , and Uhlin , B.E. ( 1996 ) Coordinated and differential expression of histone-like proteins in Escher-ichia coli : regulation and function of the H-NS analog StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_427">
Coordinated and differential expression of histone-like proteins in Escherichia coli : Regulation and function of the H-NS analog StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></label>
<input id="rp_similarity_427", type="checkbox">
<div class="RP_similar">
<li>Coordinated and differential expression of histone-like proteins in Escherichia coli : regulation and function of the H-NS analog SopA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Coordinated and differential expression of histone-like proteins in Escherichia coli : regulation and function of the H-NS analog StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Sondén , B. , and Uhlin , B.E. ( 1996 ) Coordinated and differential expression of histone-like proteins in Escherichia coli : regulation and function of the H-NS analog StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Sondén , B. , and Uhlin , B.E. ( 1996 ) Coordinated and differential expression of histone-like proteins in Escherichia coli : regulation and function of the H-NS analog StpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li></div>
<li>Regulation of Escherichia coli Hemolysin E Expression by H-NS and Salmonella SlyA Neil R. Wyborn , Melanie R. Stapleton , Valia A. Norte , Ruth E. Roberts , Jamie Grafton , and Jeffrey Green * The Krebs Institute for Biomolecular Research , Department of Molecular Biology and Biotechnology , University of Shefﬁeld , Western Bank , Shefﬁeld S10 2TN , United Kingdom Received 30 June 2003/Accepted 5 December 2003 The Escherichia coli hlyE gene ( also known as clyA or sheA ) codes for a novel pore-forming toxin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1995 , p. 3455 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology R ole for the Histone-Like Protein H-NS in Growth Phase-Dependent and Osmotic Regulation of sS and Many sS-Dependent Genes in Escherichia coli ECHTHILD BARTH , CHRISTOPH MARSCHALL , ANDREA MUFFLER , DANIELA FISCHER , AND REGINE HENGGE-ARONIS * Department of Biology , University of Konstanz , 78434 Konstanz , Germany Received 14 November 1994/Accepted 4 April 1995 M The sS subunit of RNA polymerase ( encoded by the rpoS gene ) is the master regulator in a complex regulatory network that controls stationary-phase induction and osmotic regulation of many genes in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1996 , p. 3418 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology e Histone-Like Protein H-NS Acts as a Transcriptional Repressor for Expression of the Anaerobic and Growth Phase Th Activator AppY of Escherichia coli TOVE ATLUNG ,1 * SUSANNE SUND ,2 KIRSTEN OLESEN ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 12 October 1995/Accepted 9 April 1996 The transcriptional activator AppY is required for anaerobic and stationary-phase induction of the cyx-appA and hya operons of Escherichia coli , and expression of the appY gene itself is induced by these environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>ReseARch pApeR ReseARch pApeR RNA Biology 10:5 , 708 -- 715 ; May 2013 ; © 2013 Landes Bioscience RcsB-BglJ-mediated activation of Cascade operon does not induce the maturation of CRISPR RNAs in E. coli K12 Zihni Arslan ,1 Thomas stratmann ,2 Reinhild Wurm ,1 Rolf Wagner ,1 Karin schnetz2 and Ümit pul1 , * 1Molecular Biology of Bacteria ; heinrich-heine University ; Düsseldorf , Germany ; 2Institute for Genetics ; University of cologne ; cologne , Germany Keywords : CRISPR , Cas protein , transcription regulation , H-NS , LeuO , Cascade prokaryotic immunity against foreign nucleic acids mediated by clustered regularly interspaced short palindromic repeats ( cRIspR ) depends on the expression of the cRIspR-associated ( cas ) proteins and the formation of small cRIspR RNAs ( crRNAs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23392250" target="_blank">23392250</a>]<br></li>
<li>Relationships between H-NS , S , SpvR and growth phase in the control of spvR , the regulatory gene of the Salmonella plasmid virulence operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Expression of the virulence plas-mid-carried apyrase gene ( apy ) of enteroinvasive Escheri-chia coli and Shigella ﬂexneri is under the control of H-NS and the VirF and VirB regulatory cascade .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>A phage infectiondependent regulation of the CRISPR response has been reported to occur in Streptococcus thermophilus or Thermus thermophi-lus ,31,32 and crRNAs are among the most abundant sRNAs in Streptococcus pyogenes .33 In contrast , in E. coli K12 , the crRNA level is nearly undetectable under laboratory growth condition ,12,13 while the type I-F CRISPR system in E. coli LF82 has been reported to be constitutively active and to limit the transformation of target plasmid DNA .34 In E. coli K12 , the transcriptional inhibition of Cascade expression by H-NS has been shown to be responsible for the dormant crRNA maturation .13 Consistently , the transcriptional regulator LeuO , a well-known anti-silencer of H-NS , has been identified as an activator of the CRISPR system , inducing Cascade gene transcription and concomitantly crRNA maturation .21 Thus , the upregulation of the LeuO protein was considered to be one factor triggering the CRISPR defense in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23392250" target="_blank">23392250</a>]<br></li>
<li>DISCUSSION Previously , the presumptive transcriptional regulator YjjQ of E. coli has been implicated in virulence of the avian pathogenic E. coli strain IMT5155 ( 6 ) , and yjjQ expression has been shown to be regulated by H-NS and LeuO ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Expression of the yjjQ gene , which is encoded in the yjjQ-bglJ operon , is repressed by the global repressor H-NS and activated by the pleiotropic transcriptional regulator LeuO , as shown for the laboratory E. coli K-12 strain ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Wyborn , N.R. , Stapleton , M.R. , Norte , V.A. , Roberts , R.E. , Grafton , J. , and Green , J. ( 2004b ) Regulation of Escheri-chia coli hemolysin E expression by H-NS and Salmonella SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_442">
Alternate SlyA and H-NS nucleoprotein complexes control hlyE expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></label>
<input id="rp_similarity_442", type="checkbox">
<div class="RP_similar">
<li>Lithgow , J.K. , Haider , F. , Roberts , I.S. , and Green , J. ( 2007 ) Alternate SlyA and H-NS nucleoprotein complexes control hlyE expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li></div>
<li>BglJ is a transcriptional regulator whose constitutive expression causes activation of the H-NS - and StpA-repressed bgl ( aryl - , D-glucoside ) operon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_445">
Forns , N. , Juárez , A. , and Madrid , C. ( 2005b ) Osmoregulation of the HtrA ( degP ) protease of Escherichia coli : an Hha-H-NS complex represses HtrA expression at low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></label>
<input id="rp_similarity_445", type="checkbox">
<div class="RP_similar">
<li>Osmoregulation of the HtrA ( DegP ) protease of Escherichia coli : an Hha-H-NS complex represses HtrA expression at low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li></div>
<li>K e y w o r d s Lrp • Dominant negative mutation • H-NS • lysU • lrp autoregulation Introduction Lrp ( leucine-responsive regulatory protein ) is a global regulatory protein of Escherichia coli , and controls expression of a number of operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Mol Gen Genet ( 1995 ) 247:521 -528 © Springer-Verlag 1995 Taku Oshima • Koichi Ito • Hiroyuki Kabayama Yoshikazu Nakamura Regulation of Irp gene-expression by H-NS and Lrp proteins in Escherichia coli : dominant negative mutations in Irp Received : 3 October 1994 / Accepted : 17 December 1994 A b s t r a c t Lrp ( leucine-responsive regulatory protein ) is a global transcription factor of Escherichia coli and regulates , negatively or positively , many genes including lysU , which encodes lysyl-tRNA synthetase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Oshima , T. , Ito , K. , Kabayama , H. , and Nakamura , Y. ( 1995 ) Regulation of lrp gene-expression by H-NS and Lrp proteins in Escherichia coli : dominant negative mutations in lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Torres , A.G. , López-Sánchez , G.N. , Milﬂores-Flores , L. , Patel , S.D. , Rojas-López , M. , Martínez de la Peña , C.F. , et al. ( 2007 ) Ler and H-NS , regulators controlling expression of the long polar ﬁmbriae of Escherichia coli O157 : H7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>qPCR analysis of gene-expression Escherichia coli RNAP , LeuO and H-NS were puriﬁed according to published procedures ( Stratmann et al. , 2008 ; Pul et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>While H-NS usually acts as a negative regulator of gene-expression , hlyE is not the only example of a CRP-regulated gene whose expression is positively regulated by H-NS ; for example , expression of both the malT and csiD genes in E. coli is stimulated by H-NS ( 8 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>H-NS is a global regulator able to influence the expression of 5 % of all genes in E. coli ( 13 ) and whose direct influence on transcription is almost universally negative , often regulating gene-expression in response to environmental conditions including temperature and osmolarity ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Discussion In the present work , we have demonstrated that the expression of the gene osmC of E. coli is regulated by a complex s network of several global regulators , including at least s , Lrp and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Discussion In the present work , we have analysed the structural organization and the regulation of the gad gene system in E. coli , and provided evidence for the involvement of H-NS and RpoS in the hierarchical control of gad expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_456">
In addition , we show that expression of the E. coli slyA gene is tem-perature-regulated , positively autoregulated , and independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></label>
<input id="rp_similarity_456", type="checkbox">
<div class="RP_similar">
<li>In addition , we demonstrate that expression of the E. coli slyA gene is temperature-regulated , positively autoregulated , and independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li></div>
<li>H-NS is a global modulator of gene-expression affecting the synthesis , both positively and negatively , of more than 50 E. coli proteins ( 3 , 22 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_459">
Direct and indirect effects of H-NS and Fis on global gene-expression control in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></label>
<input id="rp_similarity_459", type="checkbox">
<div class="RP_similar">
<li>Direct and indirect effects of H-NS and Fis on global gene-expression control in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li></div>
<li>Taken together , these data indicate that expression of the E. coli slyA gene is maximal during exponential growth , is temperature ¬ Autoregulation and temperature regulation of slyA expression Shown is expression of a slyA - lacZ fusion ( pDC6 ) in the presence or absence of SlyA or H-NS at 37 and 20 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Our data showing the lack of role for H-NS in the expression of slyA is in agreement with the absence of SlyA from proteomic and transcriptomic based studies of changes in E. coli gene-expression in an hns mutant ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Together , these studies indicate that temperature is a broadly used cue for regulating gene-expression in E. coli and that H-NS regulates iron , carbohydrate , and amino acid utilization gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Our results indicate that H-NS regulates iron , carbohydrate , and amino acid utilization gene-expression in E. coli , supporting a broad role for this regulator in the adaptation to temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>They also participate in diverse roles such as regulation-of-DNA-replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>DNA-replication that H-NS inhibits the expression of the other arginine triggered by double-stranded breaks in E. coli : dependence o n decarboxylase , encoded by adiA , that is expressed under homologous recombination functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Panels ( A -- D ) show expression data reﬂecting DNA looping in ( A ) WT E. coli , ( B ) cells lacking HU , ( C ) cells lacking IHF and ( D ) cells lacking H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Regulation of the Escherichia coli K5 capsule gene cluster : evidence for the roles of H-NS , BipA , and integration host factor in regulation of group 2 capsule gene clusters in pathogenic E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18820021" target="_blank">18820021</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12654836" target="_blank">12654836</a>]<br></li>
<li>Colonna B , Casalino M , Fradiani PA , Zagaglia C , Naitza S , Leoni L , Prosseda G , Coppo A , Ghelardini P , Nicoletti M : H-NS regulation of virulence gene-expression in enteroinvasive Escherichia coli harboring the virulence plasmid integrated into the host chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>H-NS regulation of virulence gene-expression in enteroinvasive Escherichia coli harboring the virulence plasmid integrated into the host chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Universita $ $ t zu Koln , Weyertal 121 , 50931 Ko $ ln , Germany Keywords : β-glucoside , pleiotropic regulation , H-NS , lac operon region INTRODUCTION The bgl operon , encoding the gene products necessary for the fermentation of aryl-β-glucosides , is silent in wild-type Escherichia coli ( Schaeﬂer & Maas , 1967 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177351" target="_blank">12177351</a>]<br></li>
<li>Molecular Microbiology ( 2012 ) 83 ( 6 ) , 1109 -- 1123 doi :10.1111 / j.1365-2958.2012.07993.x First published online 14 February 2012 RcsB-BglJ activates the Escherichia coli leuO gene , encoding an H-NS antagonist and pleiotropic regulator of virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_473">
, Wurm , R. , Wagner , R. and Schnetz , K. ( 2012 ) RcsB-BglJ activates the Escherichia coli leuO gene , encoding an H-NS antagonist and pleiotropic regulator of virulence determinants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>]<br></label>
<input id="rp_similarity_473", type="checkbox">
<div class="RP_similar">
<li>RcsB-BglJ activates the Escherichia coli leuO gene , encoding an H-NS antagonist and pleiotropic regulator of virulence determinants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li></div>
<li>K5 Capsule Gene Cluster , and Expression of slyA in Escherichia In this paper , we present the first evidence of a role for the transcriptional regulator SlyA in the regulation of transcription of the Escherichia coli K5 capsule gene cluster and demonstrate , using a combination of reporter gene fusions , DNase I footprinting , and electrophoretic mobility shift assays , the dependence of transcription on the functional interplay between H-NS and SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>In summary , we have demonstrated that SlyA and H-NS play an integral and novel role in regulating transcription of the E. coli K5 polysaccharide gene cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>stpA , a putative homolog to the gene encoding the H-NS protein in E.coli , was first identified as a multicopy suppressor of a Td - phenotype in phage T4 ( Zhang and Belfort , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>In addition to the hns gene , the chromosome of E. coli and other enteric bacteria also includes the stpA gene that encodes the StpA protein , an H-NS paralogue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>We also analyzed the growth of pCnuK9E/MG1655gstpA , an E. coli strain with a deletion in the StpA gene that encodes an H-NS homolog [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>In E. coli , this locus is subject to gene rearrangements and also likely to horizontal gene transfer , consistent with its repression by the xenogeneic silencer H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>These data demonstrate that CpxR and H-NS are involved in csgD regulation in response to me-dium osmolarity and that E. coli cells are able to discriminate between high osmolarity resulting from a high salt and that resulting from a high sucrose concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Similarly , H-NS is involved in several cellular functions in E. coli and related organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>H-NS is required for normal thermoregulation of different genes involved in pathogenesis , e.g. , the E. coli * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Consistent with this role , H-NS was found to be a negative transcriptional regulator of micF and regulates ompF expression post-transcription-ally through micF .54,55 On the other hand , H-NS negatively regulate ompC expression transcriptionally .54 H-NS has been implicated in the thermoregulation of genes in E. coli and related bacteria .53,56,57 An intriguing question is whether H-NS is involved in thermoregulation of micF expression No studies have been done with an hns deletion strain to determine if thermoregulation is lost in this genetic background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>For instance , H-NS or H-NS-like proteins , global regulators that down regulate genes responsive to environmental conditions [ 22 ] , have been shown to repress the expression of almost all E. coli cryptic ﬁmbriae and several Cup ﬁmbriae of P. aeruginosa [ 10 , 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21344461" target="_blank">21344461</a>]<br></li>
<li>Most importantly , H-NS is a global regulator of a variety of unrelated genes in E. coli ( 2 , 31 , 58 ) as well as being involved in virulence expression in pathogenic Shigella ( 35 ) , Salmonella ( 21 ) , and enteroinvasive E. coli ( 7 ) strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>H-NS is a global regulator of a variety of unlinked and unrelated genes in E. coli and Salmo-nella typhimurium ( 2 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The expression of virulence genes in the LOCUS OF ENTEROCYTE EFFACEMENT ( LEE ) pathogenicity island of enteropatho-genic E. coli is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>It is known that the expression of a variety of genes is repressed by H-NS , and mutations in hns result in various phenotypes , but the role of H-NS in the drug-resistance of Escherichia coli has not been known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>In this paper , we report that H-NS controls the multidrug-resistance of E. coli by regulating the expression of drug exporter genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>Barth , M. , Marschall , C. , Muffler , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) A role for the histone-likeprotein H-NS in growth phase-dependent and osmotic regulation of os and many os-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sigma S and many sigma S-dependent genes in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_493">
A role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sS and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></label>
<input id="rp_similarity_493", type="checkbox">
<div class="RP_similar">
<li>A role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of S and many S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation in sigma S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sS and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626294" target="_blank">8626294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and S osmotic regulation of s and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631665" target="_blank">8631665</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of ss and many ss-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of S and many S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142416" target="_blank">12142416</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth-phase-dependent and osmotic regulation of S and many S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Role of the histone-like protein H-NS in growth phase-dependent and osmotic regulation of S and many S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of s and many s-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li></div>
<li>Barth , M. , Marschall , C. , Muffler , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of c s and many ~ S-dependent genes in Escherichia coil J Bacteriol 177 : 3455-3464 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8817489" target="_blank">8817489</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_504">
Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of ss and many ss dependent genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></label>
<input id="rp_similarity_504", type="checkbox">
<div class="RP_similar">
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) A role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of s and S many s - dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Barth , M. , Marschall , C. , Muffler , A , , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) A role for the histone-like protein H-NS in growth phase-dependentand osmotic regulation of oSand many os-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Barth , M. , Marschall , C. , Muffler , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) A role for the histone-like protein H-NS in S growth phase-dependent and osmotic regulation of s and S many s - dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Barth , M. , Marshall , C. , Muffler , A. , Fisher , D. , and Hengge-Aronis , R. ( 1995 ) Role of the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sigma S and many sigma S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Barth M , Marschall C , Muffler A , Fischer D , Hengge-Aronis R : Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sigma S and many sigma S-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21314918" target="_blank">21314918</a>]<br></li>
<li>Barth , M. , Marschall , C. , Muffler , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of s-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of s-depen-dent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role of the histone-like protein H-NS in growth phase-dependent and osmotic regulation of ss and many ss-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sS and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Barth , M. , Marschall , C. , Mufﬂer , A. , Fischer , D. , and Hengge-Aronis , R. ( 1995 ) Role of the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sS and many sS-dependent genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_515">
Giangrossi M , Zattoni S , Tramonti A , De Biase D , Falconi M. Antagonistic role of H-NS and GadX in the regulation of the glutamate decarboxylase-dependent acid resistance system in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18297445" target="_blank">18297445</a>]<br></label>
<input id="rp_similarity_515", type="checkbox">
<div class="RP_similar">
<li>Giangrossi , M. , Zattoni , S. , Tramonti , A. , De Biase , D. and Falconi , M. ( 2005 ) Antagonistic role of H-NS and GadX in the regulation of the glutamate decarboxylase-dependent acid resistance system in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20189963" target="_blank">20189963</a>]<br></li>
<li>Giangrossi , M. , Zattoni , S. , Tramonti , A. , De Biase , D. , and Falconi , M. ( 2005 ) Antagonistic role of H-NS and GadX in the regulation of the glutamate decarboxylase-dependent acid resistance system in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li></div>
<li>DISCUSSION In this study , we identiﬁed several perturbations involving the H-NS family of proteins that affect Rho-dependent transcription termination in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>, Wurm , R. , Lux , B. , Meltzer , M. , Menzel , A. , and Wagner , R. ( 2005 ) LRP and H-NS -- cooperative partners for transcription regulation at E. coli rRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_520">
Shimada , T. , Bridier , A. , Briandet , R. & Ishihama , A. ( 2011 ) Novel roles of LeuO in transcription regulation in E. coli : antagonistic interplay with the universal silencer H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></label>
<input id="rp_similarity_520", type="checkbox">
<div class="RP_similar">
<li>Shimada , T. , Bridier , A. , Briandet , R. & Ishihama , A. ( 2011a ) Novel roles of LeuO in transcription regulation in E. coli : antagonistic interplay with the universal silencer H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23301696" target="_blank">23301696</a>]<br></li>
<li>Shimada T , Bridier A , Briandet R & Ishihama A ( 2011a ) Novel roles of LeuO in transcription regulation in E. coli : antagonistic interplay with the universal silencer H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li></div>
<li>Similarly , the bgl operon , encoding the gene products necessary for the uptake and fermentation of aryl-b , D-glucosides is repressed ~ 100-fold by H-NS ( Higgins et al. , 1988 ; Lopilato and Wright , 1990 ; Schnetz , 1995 ; Caramel and Schnetz , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Results Repression of the bgl operon by H-NS H-NS represses the wild-type bgl operon encoding the positive regulator BglG , the b-glucoside speciﬁc permease BglF ( or EIIBgl ) , and the phospho-b , D-glucosidase BglB ( Fig. 1 ) ( Defez and de Felice , 1981 ; Schnetz et al. , 1987 ; Higgins et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The cadBA operon and the adiA gene , which encode a lysine decarboxylase , an antiporter and an arginine decarboxylase , respectively , have been shown to be regulated by H-NS in rich medium under anaerobic conditions ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>The acid resistant island including the gadE-mdtEF operon indeed is one of the acquired gene clusters in E. coli with different GC content from the rest of E. coli genome ; Third , the interplay between the H-NS mediated repression and ArcA mediated antagonizing of the repression is also observed in the case of another gene cydAB which is also involved in adapting E. coli to the microaerobic and anaerobic growth ( Govantes et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Because the product of the ilvlH gene , AHAS III , is specific for isoleucine biosynthes ; s and thus competes for the threonine pool ( see fig i ) , we wanted to test if H-NS would also regulate the action of the kblitdh operon for which Lrp acts as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>x20 xlO0 ( b ) Transcription initiating at the putative promoter ( s ) for the operon encoding 10K-S and 10K-L is also under the control of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>We investigated the interaction between H-NS and ﬁmB , the gene encoding the bidirectional recombinase that catalyzes ﬁmA promoter ﬂipping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Alternatively , a bound transcription factor may form a barrier that prevents spreading of the H-NS nucleoprotein complex into the promoter , as proposed for autoregulation of the H-NS-repressed leuO gene and the divergent leuABCD operon encoding enzymes of the leucine-synthesis pathway ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Genes whose mRNA levels displayed twofold decreases after addition of 0.2 mM ZnSO4 to minimal medium Genea Gene product and/or functionb Fold P value decrease ( 0.05 ) Transport and binding actP 10 3 10 3 10 4 10 5 10 3 10 4 10 2 10 3 10 2 10 2 10 3 10 3 10 4 10 4 Acetate permease Possible assembly or transport protein for curli 3.1 6.2 2.7 5.9 2.5 2.8 2.1 2.8 2.4 2.2 2.4 3.7 2.2 2.3 4.4 2.5 4.3 1.7 4.4 2.4 1.0 1.1 2.9 3.5 5.0 8.1 8.0 6.0 csgF csgG gadC lsrB hdeD lsrD lsrG ompW ugpA ugpE uidB ydcU ygjU Other gene implicated in responses to metals yrbL Genes implicated in response-to-stresses other than metals csgD Transcriptional regulator for curli operon csgBA csiE Stationary-phase inducible protein ; sigma S-dependent promoter cstA Starvation-induced protein involved in peptide utilization during carbon starvation dps Stress response DNA-binding protein gadE Required for stationary-phase induction ; pH 5.5 growth medium induced ; overexpression confers acid resistance hdeB Acid resistance ; periplasmic protein ; repressed by H-NS ; rpoS regulon osmE Regulated by growth phase as well as osmotic pressure osmC Osmotically inducible , stress-inducible membrane protein C ; involved in defense ag oxidative compounds psiF pho regulon member , requiring PhoRB system rssB Two-component response regulator , affecting RpoS regulon sodC Superoxide dismutase ; Cu , Zn , periplasmic uspB Ethanol resistance in stationary phase Periplasmic protein hdeA Periplasmic chaperone of acid-denatured proteins Extracellular protein hlyE Latent hemolysin Intermediary metabolism astA Putative arginine succinyltransferase gadB Glutamate decarboxylase B , vitamin B6 dependent otsB Trehalose phosphate phosphatase ; cold and heat induced talA Transaldolase A ; creBC regulon Membrane structure frdD Fumarate reductase membrane anchor polypeptide ychH Function unknown ; putative membrane protein transcribed divergently from pth ynfA Inner membrane protein ; function unknown Phage-related function sgcC Putative phosphotransferase enzyme IIC sgcX Putative gene in sgc cluster ; function unknown stfR Phage lambda stf gene homolog in prophage Rac Unclassiﬁed elaB Function unknown yagU Function unknown yahO Function unknown ybaY Function unknown ycfH Putative DNAse ; tatD paralog ; no Tat phenotype ycgK Function unknown ydeI Function unknown ydhZ Function unknown yeaQ Function unknown yebV Function unknown yhcO Function unknown yhfG Function unknown yjbJ Function unknown yjfO Function unknown yoaE Function unknown yodD Function unknown ytfR Function unknown ytfT Function unknown ytfR Function unknown ytfT Function unknown a Gene names obtained from http://bmb.med.miami.edu/EcoGene/EcoWeb/ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659689" target="_blank">15659689</a>]<br></li>
<li>Gene characteristics Gene Predicted Band no.a Gene function ( reference ) Regulation of corresponding gene ( reference ) productb molecular mass ( Da ) c PﬂB * 85,357 GadA * 52,685 1 Pyruvate formate lyase I ( 56 ) Glutamate decarboxylase ( 64 ) Induced by ArcA and FNR ( 61 ) 2 Positively regulated by rpoS , CRP , gadE , gadW , and gadX ; negatively regulated by H-NS ( 21 , 66 ) Positively activated by rpoS and by CRP ( 33 , 74 ) Positively activated by CRP ( 74 ) ; repressed by ArcA ( 60 ) Same transcription unit as gatZ Positively regulated by rpoS ( 33 , 46 , 71 ) Positively regulated by rpoS ( 2 , 33 , 46 , 71 ) 52,773 TnaA † 3 Tryptophan deaminase ( tryptophanase ) ( 22 ) 47,109 GatZ † 3 Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Subunit of tagatose-1 ,6 - bisphosphate aldolase 2 ( 42 ) Quinone reductase ; response-to-oxidative-stress ( 45 ) Bacterial ferritin , DNA-protection during stationary phase ( 39 , 73 ) Transcriptional regulator ( 28 ) 30,812 20,846 18,695 GatY † WrbA * Dps * 4 5 6 , 7 , and 11 24,935 CsgD * 8 Positively regulated by rpoS , hns , and ompR ( 25 , 58 ) Same transcription unit as csgD Positively regulated by FNR ( 18 ) Outer membrane protein ; curli transport component ( 38 ) Outer membrane protein ; receptor for colicin S4 ( 49 ) 30,557 22,928 CsgG * OmpW * 9 10 a As in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>Table 1 Up-regulated genes in E. coli MG1655 cra mutant ( fold change > 1.5 ) Gene Gene product Fold change ( log ) 2 6.680 5.842 4.561 3.411 3.313 fruK fruA Fructose-1-phosphate kinase Fused fructose-speciWc PTS enzymes : IIB component/IIC components D-Erythrose 4-phosphate dehydrogenase Conserved protein Sulphur carrier protein epd ymdF thiS prep 2.644 2.518 2.416 2.362 Predicted propionyl-CoA synthetase with ATPase domain 2-Methylcitrate synthase 2-Methylcitrate dehydratase Fused acetaldehyde-CoA dehydrogenase/iron-dependent alcohol dehydrogenase/pyruvate-formate lyase deactivase Nitrate reductase 2 ( NRZ ) , beta subunit Periplasmic protein , combats stress prpC prpD adhE narY cpxP 1.938 1.849 Phosphoglyceromutase III , cofactor-independent Nitrate/nitrite-transporter Fructose-bisphosphate aldolase , class II Predicted phosphate starvation inducible protein 2-Methylisocitrate lyase gpmM narU 1.736 1.706 1.695 1.639 1.592 fbaA psiE prpB ydfN Qin prophage ; predicted side tail Wbre assembly protein Modulator of gene-expression , with H-NS 16S rRNA pseudouridylate 516 synthase Nitrate reductase 2 ( NRZ ) , gamma subunit 1.568 1.558 1.540 1.524 hha rsuA narV Discussion Curli biosynthesis is regulated by Cra Cra is a global plieotropic transcription factor in enteric bacteria that regulates a number of genes involved in carbon and energy metabolism ( Saier 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Part of a system for autoinducer 2 uptake in Salmonella Putative membrane transporter , H-NS repressed Part of a system for autoinducer 2 uptake in Salmonella Function unknown ; part of an operon for autoinducer 2 uptake in Salmonella Outer membrane protein ; colicin S4 receptor sn-Glycerol-3-phosphate-transport system ; integral membrane protein sn-Glycerol-3-P transport system Glucuronide permease Function unknown ; putative ABC transporter permease protein Na / serine ( threonine ) symporter 3.0 10 7 Mg ( II ) starvation-stimulated gene ; function unknown ; phoPQ regulon 7.7 10 3 10 4 10 4 10 3 10 4 2.3 2.7 2.3 2.1 2.7 2.0 5.0 8.3 1.3 1.7 1.7 10 4 3.5 10 4 7.2 10 7 5.8 2.9 2.6 ainst 10 3 10 2 10 5 10 3 2.4 2.2 2.6 2.1 2.9 3.6 2.3 5.5 3.6 10 5 5.4 2.0 10 2 2.9 10 4 10 6 10 3 10 2 3.9 5.3 2.3 2.1 1.1 3.7 5.6 2.1 3.5 10 3 1.0 10 2 8.9 10 4 2.0 2.6 2.5 6.4 10 4 1.3 10 4 4.9 10 2 5.3 3.5 2.0 10 4 10 5 10 3 10 3 10 3 10 3 10 2 102 10 4 10 3 10 2 10 4 10 2 10 2 10 4 10 3 10 4 10 2 10 4 10 2 1.2 4.5 5.3 1.1 5.1 5.0 1.0 4.8 2.6 1.2 3.5 1.9 1.3 3.3 9.8 4.9 4.2 3.7 4.2 3.7 2.4 5.1 3.1 2.2 2.2 5.2 2.2 2.7 2.1 2.4 2.5 2.4 2.0 2.4 2.3 2.1 2.4 3.0 2.4 3.0 ( 2.5-fold decrease ) , which is part of a system for autoinducer 2 uptake in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659689" target="_blank">15659689</a>]<br></li>
<li>E. coli genes whose relative expression levels were increased or decreased by evgA ampliﬁcation Effect of EvgA on gene-expression ( fold change ) Functional classiﬁcationc b no.b Genea Known or predicted function Increased expression hdeAd , e Not known Not known Not known Regulation Metabolism Metabolism Cell structure Metabolism None b3510 b2375 b3509 b2369 b3517 b1501 b4184 b2374 b1051 Protein regulated by H-NS , chaperone Protein regulated by EvgA Protein regulated by H-NS , predicted chaperone Regurator of EvgAS two-component system Glutamate decarboxylase-alpha Putative reductase Putative membrane protein Putative formyl-coenzyme A transferase Acidic protein suppresses mutants lacking function of protein export Stress response DNA-binding protein Putative regulator Putative amidase Acid shock RNA controlled by phoBR Putative transferase Predicted amino-acid-transporter Function unknown Putative cysteine hydrolase Function unknown Putative replication protein Multidrug-transport protein Putative glutaminase Magnesium transporter Putative formyl-coenzyme A transferase 84.3 54.4 42.2 41.8 41.4 15.3 9.2 6.8 6.5 yfdX hdeBd , e evgAe gadAd , e ydePe yjfL yfdWe msyB dpse yhiEe Adaptation ( starvation ) Regulation Metabolism Adaptation ( stress ) Extrachromosomal ( phage ) Transport None Metabolism None Extrachromosomal ( phage ) Transport Metabolism Transport Metabolism b0812 b3512 b0867 b1597 b2359 b1492 b2390 b0897 b0329 b2358 b3514 b0485 b4242 b2371 6.3 6.2 5.5 5.3 5.2 5.2 5.2 5.0 4.7 4.6 4.4 4.3 4.1 4.0 ybjR asr yfdM gadC ( xasA ) d , e ypeC ycaC yahO yfdO yhiV ybaS mgtA yfdE Decreased expression tnaA tnaL glpQ b3708 Metabolism Tryptophan deaminase b3707 Metabolism Tryptophanase leader peptide b2239 Metabolism Glycerophosphodiester phosphodiesterase 7.1 5.0 4.5 a Based on information from the EcoGene database ( http://bmb.med.miami.edu/ecogene/ecoweb/ ) ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670992" target="_blank">12670992</a>]<br></li>
<li>Pavitt GD , Santos DS , Sidebotham JM , Hulton CSJ , Hinton JCD , Higg ~ ns CF ( 1992 ) The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_537">
Owen-Hughes , T.A. , Pavitt , G.D. , Santos , D.S. , Sidebottom , J.M. , Hulton , C.J. , Hinton , J.C.D. , and Higgins , C.F. ( 1992 ) The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></label>
<input id="rp_similarity_537", type="checkbox">
<div class="RP_similar">
<li>Owen-Hughes , T.A. , Pavitt , G.D. , Santos , D.S. , Sidebotham , J.M. , Hulton , C.S.J. , Hinton , J.C.D. , and Higgins , C.F. ( 1992 ) The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Owen-Hughes , T.A. , Pavitt , G.D. , Santos , D.S. , Sidebotham , J.M. , Hulton , C.S. , Hinton , J.C. , and Higgins , C.F. ( 1992 ) The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Owen-Hughes , T. , Pavitt , G.D. , Santos , O.S. , Sidebotham , J.M. , Hulton , C.S.J. , Hinton , J.C.O. , and Higgins , C.F. ( 1992 ) The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_541">
The chromatin-associated protein H-NS interacts with curved DNA to inﬂuence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626294" target="_blank">8626294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></label>
<input id="rp_similarity_541", type="checkbox">
<div class="RP_similar">
<li>The chromatin-associated protein H-NS interacts with curved DNA to inﬂuence topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>]<br></li>
<li>, and Higgins , CF. ( 1992 ) The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>J.M. , Huiton , C.S.J. , Hinton , J.C.D. , and Higgins , C.F. ( 1992 ) The ohromatin-associated protein iH-NS interacts with curved DNA to influence DNA topology and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_545">
Hulton , C.S. , Seirafi , A. , Hinton , J.C. , Sidebotham , J.M. , Waddell , L. , Pavitt , G.D. , et al. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></label>
<input id="rp_similarity_545", type="checkbox">
<div class="RP_similar">
<li>Hulton , C.S.J. , Seiraﬁ , A. , Hinton , J.C.D. , Sidebotham , J.M. , Waddell , L. , Pavitt , G.D. , et al. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Hulton , C.S. , Seiraﬁ , A. , Hinton , J.C. , Sidebotham , J.M. , Waddell , L. , Pavitt , G.D. , et al. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li></div>
<li>Hulton , C.S.J. , Seirafi , A. , Hinton , J.C.D. , Sidebotham , J.M. , Waddell , L , Pavitt , G.D. , Owen-Hughes , T. , Spassy , A. , Buc , H. , and Higgins , C.F. ( 1990 ) Histone-iike protein HI ( H-NS ) , DNA supercoiiing , and gene-expression In bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8022275" target="_blank">8022275</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_549">
, Seirafi , A. , Hinfon , J.CD , , Sidebotham , J , M , , Waddelt , L , Pavitt , G.D. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , CF. ( 1990 ) Histone-like protein HI ( H-NS ) DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></label>
<input id="rp_similarity_549", type="checkbox">
<div class="RP_similar">
<li>Waddel , L , Pavitt , G.D. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , CF. ( 1990 ) Histone-like protein HI ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1625583" target="_blank">1625583</a>]<br></li>
<li>J , M , , Waddell , L , , Pavitt , G.D , , Owen-Hughes , T , , Spassky , A , , Buc , H , , and Higgins , C , H , ( 1990 ) Histone-like protein HI ( H-NS ) DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1357528" target="_blank">1357528</a>]<br></li>
<li>Hulton , C.S.J. , Seifari , A. , Hinton , J.C.D. , Sidebotham , J.M. , Waddell , l. , Pavitt , G.O. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , C.F. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Hulton C , Seirafi , A. , Hinton , J. , Sidebotham , J. , Waddell , L. , Pavitt , G. , Owen-Hughes , T. , Spassky , A. , Buc , H. , and Higgins , C. ( 1990 ) Histone-like protein H1 ( H-NS  , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1710760" target="_blank">1710760</a>]<br></li>
<li>Mol Microbiol 4 : 2007-2012 Hulton CSJ , Seirafi A , Hinton JCD , Sidebotham JM , Waddell L , Pavitt GD , Owen-Hughes T , Spassky A , Buc H , Higgins CF ( 1990 ) Histone-like protein H 1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_555">
J Bacteriol 175 : 1580-1589 Hulton CS , Seirafi A , Hinton JC , Sidebotham JM , Waddell L , Pavitt GD , Owen-Hughes T , Spassky A , Buc H , Higgins CF ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7500958" target="_blank">7500958</a>]<br></label>
<input id="rp_similarity_555", type="checkbox">
<div class="RP_similar">
<li>J Bacteriol 171:2879 -2881 Hulton CS , Seirafi A , Hinton JCD , Sidebotham JM , Waddell L , Pavitt GD , Owen-Hughes T , Spassky A , Buc H , Higgins CF ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Gene 57:8 % 99 Hulton CSJ , Seirafi A , Hinton JCD , Sidebotham JM , Waddell L , Pavitt GD , Owen-Hughes T , Spassky A , Buc H , Higgins CF ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Gene 57 : 89-99 Hulton CSJ , Seirafi A , Hinton JCD , Sidebotham JM , Waddell L , Pavitt GD , Owen-Hughes T , Spassky A , Buc H , Higgins CF ( / 990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li></div>
<li>( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>C.F. ( 1990 ) Histone-iiko protein HI ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_561">
Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381784" target="_blank">8381784</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></label>
<input id="rp_similarity_561", type="checkbox">
<div class="RP_similar">
<li>Histone-like protein H1 ( H-NS ) , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Histone-like protein Hl ( H-NS ) , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715407" target="_blank">1715407</a>]<br></li>
<li>Histone-like protein Hi ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1348054" target="_blank">1348054</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li>
<li>Histone-like protein Hi ( H-NS ) , DNA supercoiling and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_566">
Spassky A , Buc H , Higgins CF ( 1990 ) Histone-like protein HI ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></label>
<input id="rp_similarity_566", type="checkbox">
<div class="RP_similar">
<li>H. , and Higgins , C.F. ( 1990 ) Histone-like protein H1 ( H-NS ) , DNA supercoiling , and gene-expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li></div>
<li>The gene / 7ns ( osmZ ) encodes the histone-like proteitT H-NS ( Hutton etai , 1990 ; May etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li>
<li>The latter view is based on the fact that mutations of the structural gene for H-NS ( hns or osmZ ) have a global effect on chromosomal and plasmid DNA supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>[ 40 ] Hommais F. , Krin E. , Laurent-Winter C. , Soutourina O. , Malpertuy A. , Le Caer J.P. , Danchin A. , Bertin P. , Large-scale monitoring of pleiotropic regulation-of-gene-expression by the prokaryotic nucleoid-associated protein , H-NS , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_571">
Large-scale monitoring of pleiotropic regulation-of-gene-expression by the prokaryotic nucleoid-associated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18297445" target="_blank">18297445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></label>
<input id="rp_similarity_571", type="checkbox">
<div class="RP_similar">
<li>Large-scale monitoring of pleiotropic regulation-of-gene-expression by the procaryotic nucleoid-asso-ciated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>Large-scale monitoring of pleiotropic regulation-of-gene-expression by the prokaryotic nucleoid-asso-ciated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670992" target="_blank">12670992</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11872702" target="_blank">11872702</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426343" target="_blank">12426343</a>]<br></li></div>
<li>Protein subcellular-localization ( column six ) came from the Niceprot database ( http://www.expasy.ch/cgi-bin/niceprot.pI ) or was predicted ( f ) using the PSORT software ( http://psort.ims.u-tokyo.ac.jp/ ) Percentage survival b Growth conditions FB8 ( wild type ) BE1410 ( hns ) A High osmolarity stress c 3 M NaCl B Acid stress d pH 2.5 pH 2.5 1 0.012 % lysine pH 2.5 1 0.012 % arginine pH 2.5 1 0.012 % glutamate 2.6 12.5 # 0.01 0.1 0.2 4.1 0.9 2.9 13.0 38.0 Discussion Proteome and transcriptome analyses demonstrated that the nucleoid-associated protein , H-NS , plays a major role in bacterial physiology by directly or indirectly controlling the expression of many genes and/or the synthesis of their product ( these data are accessable at the web site http://www.pasteur.fr/recherche/unites/RE6/H-NS/regulateur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Mutations altering chromosomal protein H-NS induce mini-Mu transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>J Bacteriol 174:110 % 1118 Falconi M , McGovern V , Guarlezi C , Hillyard D , Higgins NP ( 199l ) Mutations altering chromosomal protein H-NS induce mini-Mu transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_577">
Falconi , M. , McGovern , V. , Gualerzi , C. , Hillyard , D. and Higgins , N.P. : Mutations altering chromosomal protein H-NS induce mini-Mu transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></label>
<input id="rp_similarity_577", type="checkbox">
<div class="RP_similar">
<li>Falconi , M. , McGovern , V. , Gualerzi , C. , Hillyard , D. , and Higgins , N , P. ( 1991 ) Mutations altering chromosomal protein H-NS inducemini-Mu transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li></div>
<li>The histone-like nucleoid-structuring protein ( H-NS ) is also involved in the recombination event ( Donato et al. , 1997 ; O'Gara and Dorman , 2000 ; Olsen & Klemm , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207912" target="_blank">16207912</a>]<br></li>
<li>The role of H-NS in the curli control network is connected to YdaM and YciR The histone-like protein H-NS plays multiple and crucial roles in the sS network ( Barth et al. , 1995 ; Yamashino et al. , 1995 ) , and also affects curli control ( Arnqvist et al. , 1994 ; Gerstel et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>However , nucleoid-associated proteins ( NAPs ) , particularly histone-like proteins , such as histone-like nucleoid structuring ( H-NS ) protein , heat unstable nucleoid protein ( HU ) , factor for inversion stimulation ( Fis ) and integration host factor ( IHF ) , are believed to act like histones and play a signiﬁcant role in the organization of the nucleoid ( 14 -- 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>These proteins include ( in decreasing order of abundance during exponential growth ) : factor for inversion stimulation ( Fis ) , heat-unstable nucleoid protein ( HU ) , integration host factor ( IHF ) , histone-like , nucleoid-structuring protein ( H-NS ) , suppressor of td mutant phenotype A ( StpA ) and DNA-binding protein from starved cells ( Dps ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>ACC , acetyl-CoA carboxylase ; Amp , ampicillin ; CAI , codon adaptation index ; CAT , chloramphenicol acetyltransferase ; CHL , chloramphenicol ; CHL , chloramphenicol resistance ; H-NS protein , histone-like nucleoid structuring protein ; IPTG , isopropyl thio-R b-D-galactoside ; LB , lysogeny broth ; SPEC , spectinomycin ; SPEC , spectinomycin resistance ; STR , streptomycin ; STR , streptomycin resistance ; tAI , tRNA adaptation R R index ; tmRNA , transfer-messenger RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24966114" target="_blank">24966114</a>]<br></li>
<li>The nucleoid-associated protein HU , the histone-like protein H-NS , and the small regulatory RNAs DsrA and OxyS in conjugation with the RNA chaperone Hfq control the translation of rpoS by modulating the secondary structure of the rpoS mRNA ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>The enterobacterial histone-like nucleoid structuring ( H-NS ) protein is an abundant global repressor that affects many genes and cellular-processes ( 16 , 18 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Introduction The histone-like nucleoid structuring protein H-NS , a highly abundant 15.6 kDa protein , is an important pleio-tropic regulator in the adaptation of the cellular physiology of enterobacteriaceae to external signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Signi ® cantly , proteins that are involved directly in bacterial stress responses , including the heat shock protein DnaK and the DNA-binding protein H-NS , were found to be particularly susceptible to oxidative damage ( Dukan and NystroÈm , 1998 , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>As shown in Fig. 4 , these include the following proteins : GyrA , the a-subunit of the topoisomerase DNA gyrase ( 132 , 134 ) ; H-NS , a nucleoid-as-sociated DNA-binding protein required for optimal growth at low temperature ( 157 ) ; Hsc66 , a homolog of Hsp70 ( 165 ) ; NusA , which is involved in both termination and antitermination of transcription ( 132 ) ; PNP , which is an exoribonuclease ( 132 ) ; and RecA , which is involved in recombination and induction of the cold shock response ( 283 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The regulation of nhaA is also affected by H-NS ( 10 ) , a major DNA binding protein and a global regulator involved in salt stress in bacteria ( 8 , 9 , 15 , 22 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Recently we have shown that the NhaR-dependent regulation of nhaA is affected by H-NS ( Dover et al. , 1996 ) , a major DNA-binding protein and a global regulator involved in salt stress in bacteria ( Owen-Hughes et al. , 1992 ; Ussery et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></li>
<li>Our results indicate that the H-NS protein is involved in the integration of the CRISPR-cas system to the regulatory network of E. coli , linking both prokaryotic defence systems against invading DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>This same functional role can be argued for Lrp regulation ( see introduction ) , perhaps implying that the H-NS protein is involved in fine tuning this regulatory network in some way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The DNA-binding protein H-NS , one of the major components of bacterial chromatin , is recognized as a global regulator of bacterial cell-physiology , being involved in the modulation , primarily at transcriptional level , of a fairly large number of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>The H-NS protein , in addition to acting as an autorepressor , acts as a transcriptional repressor of proU , which is involved in osmoregulation ( UEGUCHIandMIZUNO1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
<li>The results present strong evidence that the DNA-binding protein H-NS , known to repress transcriptional activity from horizontally transferred genetic elements , is involved in downregulation of the Pcas and anti-Pcas promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The protein is likely to play an architectural role , but instead of being responsible for DNA compaction it could be involved in antagonising compaction by other proteins such as H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>The global regulatory protein H-NS was shown to be involved in csg genes regulation , and it regulates other genes in response to osmolarity changes ( 2 , 22 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>These results suggest that the H-NS protein acts as a positive regulator of genes involved in the biogenesis of flagella by a mechanism independent of the cAMP-CAP pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li>
<li>Moreover , we demonstrated that H-NS , a protein known to silence laterally acquired genes , is involved in the downregulation of all promoters in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>In recent years it has become apparent that the H-NS protein is involved , directly or indirectly , in a number of cellular functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The expression of several genes involved in flagellum biosynthesis was decreased in the hns mutant strains , which indicates that their transcription requires the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li>
<li>H-NS protein is involved in a number of cellular functions such as the modulated expression of numerous environmentally regulated genes The hns gene is not part of an operon in E. coli ( Ussery et al. , 1994 ; Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>These include dps whose product appears to protect DNA from oxidative damage in the stationary phase ( Bridges and Timms , 1998 ) , stpA , which encodes an H-NS paralogue in E. coli ( Zhang et al. , 1996 ) , and the DNA-binding protein , Lrp , structural gene ( Newman et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>The gene for the protein H-NS , which is probably identical with H1 protein , has recently been cloned from several organisms ( La Teana et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160266" target="_blank">2160266</a>]<br></li>
<li>H-NS itself is able to interact with various other proteins , such as the bacteriophage T7 gene 5.5 protein product , the ﬂagellar rotor protein FliG , or Hfq , a RNA-binding protein that participates in translation of rpoS RNA ( Liu and Richardson , 1993 ; Marykwas et al. , 1996 ; Mufﬂer et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>hns down-regulation requires H-NS protein-synthesis An obvious model for down-regulation of hns transcription upon cessation of DNA-synthesis is that the resulting increase in the H-NS protein : DNA ratio leads to auto-repression of hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>The H-NS protein negatively regulates its own gene , and this may help account for the pattern of growth phase-dependent regulation ( 10 , 15 ) , as may the contribution of the positive regulator FIS ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Comparative and extensive studies on subsets of hns-dependent genes , including those newly identified in this study , should shed light on the complex mechanisms underlying global gene regulation by the nucleoid protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Role of the nucleoid-associated protein H-NS in the synthesis of virulence factors in the phytopathogenic bacterium Erwinia chrysanthemi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Discussion The H-NS protein has been determined to play an important role in enterobacteria as a component of the bacterial nucleoid affecting recombination , transposition and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The nucleoid protein H-NS has also been identiﬁed as a repressor for this system ( 4 , 8 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>The nucleoid-associated protein H-NS also represses osmC transcription from both promo-ters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>The nucleoid-associated protein H-NS is a pleiotropic regulator that generally functions as a repressor ( silencer ) of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>Other regulators ( AppY , Fis , H-NS , StpA , and IHF ) The AppY protein Ž acid poly phosphatase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The H-NS and StpA proteins negatively autoregulate the transcription of their own genes and each represses the gene that encodes the other protein in trans 6,10 -- 15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Regulation of rRNA-transcription by H-NS and LRP 871 protein -- protein contact , and probably not caused by stoichiometric H-NS -- LRP oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Another class of multicopy suppressors , msyA , encodes the H-NS ( or H1 ) histonelike protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li>
<li>Sequence alignment of Hha ( ) , H-NSTEPEC ( AAV88357 ) , and N-terminal end of H-NS protein ( ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Summary Mutations in the structural gene ( hns ) for the Escher íchía coN nucleoid-associated DNA-binding protein H-NS cause highly pleiotropic effects on gene expres sion , site-specific recombination , transposition of phage Mu , the stability 01 the genetic material and the topological state of the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Mutations in hscA , the gene encoding Hsc66 , compensate for some phenotypic effects of a mutation in hns , a gene encoding the cold-inducible , nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>Transcription of hns , the gene encoding the abundant nucleoid-associated protein H-NS , is also subject to negative autoregulation ( Dersch et aL , 1993 ; Falconi et aL , 1993 ; Ueguchi et aL , 1993 ) but is unaffected by RpoS ( Dersch et aL , 1993 ; our unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Summary The structural gene of the H-NS protein , a global regulator of bacterial metabolism , has been identiﬁed in the group of enterobacteria as well as in closely related bacteria , such as Erwinia chrysanthemi and Haemo-philus inﬂuenzae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>First , H-NS is a repressor of hdfR , a gene that codes for a LysR-like protein that negatively regulates the promoter of the flagellar master-regulator operon , flhDC29 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>As in the case of H-NS , the HU protein affects the global pattern of gene transcription by modulating the conformation of genome DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>This response is dependent upon continuing protein-synthesis during the period of DNA-synthesis inhibition and upon the presence of a gene producing functional H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>These results confirm the theory that H-NS negatively regulates its own gene in order to match protein production to DNA-synthesis , and they suggest that it is important for cells to maintain a constant ratio of H-NS protein to DNA during logarithmic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>The H-NS protein , the product of the hns gene , plays a central role in the cellular response of bacteria to environmental stresses such as modification of osmolarity and temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Introduction Research Initiative Center for Behavioral Genetics , Department of Biological Science , The nucleoid-associated protein H-NS , which is encoded by the gene hns , has a central role in the structuring and control of the enteric bacterial chromosome ( Williams , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077448" target="_blank">12077448</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_629">
Beloin , C. and Dorman , C.J. ( 2003 ) An extended role for the nucleoid structuring protein H-NS in the virulence gene regulatory cascade of Shigella ﬂexneri .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></label>
<input id="rp_similarity_629", type="checkbox">
<div class="RP_similar">
<li>Beloin , C. , and Dorman , C.J. ( 2003 ) An extended role for the nucleoid structuring protein H-NS in the virulence gene regulatory cascade of Shigella ﬂexneri .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li></div>
<li>Therefore , the H-NS protein not only serves a role in structuring thechromosomal DNA ( 18,23 ) , but also actively participates in theprocess of gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Indeed , a genetic screening has led to the identification of a histone-like nucleoid structuring protein ( H-NS ) for its role in the gene silencing mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>Involvement of StpA in post-transcriptional-control of micF RNA chaperones are proteins that aid in folding of RNAs and in resolving misfolded structures .84 ± 87 StpA , an RNA chaperone protein which shares 58 % amino acid identity with H-NS , has been shown to promote self splicing .88,89 StpA is also a nucleoid-associated protein that can alter gene-expression , possibly by affecting DNA supercoiling .90 StpA has now been implicated in the control of micF RNA stability .55 In this instance , StpA appears to regulate micF RNA post-transcriptionally by targeting it for degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Phenotypic Analysis of Random hns Mutations Differentiate DNA-Binding Activity from Properties of ﬁmA Promoter Inversion Modulation and Bacterial Motility GINA M. DONATO AND THOMAS H. KAWULA * Department of Microbiology and Immunology , University of North Carolina School of Medicine , Chapel Hill , North Carolina 27599 Received 3 August 1998/Accepted 20 November 1998 H-NS is a major Escherichia coli nucleoid-associated protein involved in bacterial DNA condensation and global modulation of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>70 The E. coli nucleoid-associated proteins including Fis ( factor for inversion stimulation ) , H-NS , IHF ( integration host factor ) and Lrp ( leucine-responsive regulatory protein ) are known to be involved in DNA compaction as well as in transcriptional-control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>An important conceptual question when considering the function of H-NS in the cell is whether H-NS is a true regulatory protein ( ie whether it plays an active role in altering gene-expression in response to specific signals ) , or whether it simply serves a passive , structural role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>A null mutation in stpA , the gene encoding the H-NS-like protein StpA , which is believed to represent a molecular backup of H-NS ( 58 ) , had no effect on proU P1 expression either by itself or in combination with other mutations such as rho and hns ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673423" target="_blank">11673423</a>]<br></li>
<li>Mutations in hns , the gene for the his-tone-like protein H-NS , cause derepressed expression of the gad genes , whereas the rpoS mutation abrogates gad transcription even in the hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Suppression by an hns mutation requires a fimctional tdh gene The H-NS protein is a global regulator of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The model as proposed establishes a new role for the polymer scaffold of H-NS that is unrelated to the protein 's function in silencing of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Considering that H-NS modulates gene-expression in response to environmental factors , we next analyzed whether this protein modulates the expression of the nrdAB operon in response to speciﬁc environmental stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>in a control experiment , the expression of hns ( the gene eoding ior the nueleoid-assoeiated protein H-NS ) was monitored under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7830553" target="_blank">7830553</a>]<br></li>
<li>plain the effects of H-NS , which is clearly not a classic se-quence-specific regulatory protein , on gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>They identified a gene , drdX , which maps at 27 min and affects thermoregulation of pap expression ; this gene encodes the H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2177526" target="_blank">2177526</a>]<br></li>
<li>Doyle , M. and Dorman , C.J. ( 2006 ) Reciprocal transcriptional and posttranscriptional growth-phase-dependent expression of sfh , a gene that encodes a paralog of the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>At the amino acid level , StpA is 58 % identical to the E. coli nucleoid protein H-NS , which is a major component of the bacterial nucleoid and has been identiﬁed as a pleiotropic regulator of gene-expression , recombination , and genome stability ( see reference 30 for a review ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The H-NS protein is a bacterial nucleoid-associated protein and a pleiotropic regulator of gene-expression ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>H-NS : A UNIVERSAL REGULATOR The effect of the bacterial heat-stable nucleoid-structuring ( H-NS ) protein on gene-expression is overwhelmingly negative and extends throughout the genome , pointing to an almost universal role for this nucleoid-associated protein as a transcriptional repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>H-NS plays a dual role , both as an architectural protein that contributes to nucleoid structure and as a global modulator of gene-expression ( see references 13 and 14 for reviews ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS is a nucleoid-associated DNA-binding protein that plays a major role in the organization of the bacterial chromosome and in regulation-of-gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Our results suggest that the nucleoid-associated DNA-binding protein H-NS ( Hla ) plays a critical role in gene-expression and in determining the structure of the genetic material .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2177526" target="_blank">2177526</a>]<br></li>
<li>Olekhnovich , I.N. , and Kadner , R.J. ( 2007 ) Role of nucleoid-associated proteins Hha and H-NS in expression of Sal-monella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Other abundant nucleoid-associated proteins such as IHF , H-NS , and Dps/PexB also have been shown to control the expression of a variety of speciﬁc genes ( 1 , 9 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7860604" target="_blank">7860604</a>]<br></li>
<li>Its expression is also dependent on the global regulatory proteins mediating catabolic repression via cyclic AMP receptor protein ( CRP ) , or anaerobiosis via fumarate nitrate reduction ( FNR ) transcriptional regulator , and the nucleoid-associated DNA-binding protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631699" target="_blank">8631699</a>]<br></li>
<li>Moreover , it is subject to global control by the same regulatory proteins which are known to modulate expression of carnitine-metabolism , i.e. , the cyclic AMP ( cAMP ) receptor protein ( CRP ) which mediates the activation of catabolic operons , the transcriptional regulator FNR that is responsible for anaerobic induction , and the DNA-binding protein H-NS ( 19 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Full expression of ﬂhCD and thus of the chemotaxis and ﬂagellum genes depends on cAMP-CRP ( 37 , 63 ) ; the carbon storage regulator Csr ( 70 ) ; the nucleoid-associated protein H-NS ( 9 , 26 , 64 ) ; the heat shock proteins DnaK , DnaJ , and GrpE ( 59 ) ; the regulators of phosphatidylethanolamine-biosynthesis -- Pss and Psd ( 58 ) ; polyphosphate ( 54 ) ; and the quorum-sensing regulator QseBC ( 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>An extragenic factor , the global regulatory protein H-NS , was identified as a regulator of chiA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>H-NS , a nucleoid-associated protein , is a top level regulator affecting the expression of at least 250 genes , mainly related to the bacterial response to environmental changes [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>DsrA represses either the function or synthesis of the global transcriptional silencer and nucleoid protein H-NS , allowing expression of H-NS-silenced capsular synthesis genes when DsrA is overexpressed ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>The nucleoid-associated protein H-NS is a global transcriptional repressor that controls the expression of several environmentally regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>In support of this explanation , inactivation of the histone-like protein H-NS , which induces negative DNA supercoiling and binds to curved DNA regions , has been shown to abrogate the requirement of the rpoS gene for expression of curli by E. coli K-12 ( Olsén et al. , 1993 ; Arnqvist et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The abundant histone-like DNA-binding protein H-NS compacts DNA ( 24 ) , and mutations in the gene encoding H-NS , variously designated osmZ ( 7 ) , puG ( 11 ) , drdX ( 4 ) , bglY ( 13 ) , and hns ( 18 ) , affect the expression of a number of environmentally sensitive genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381784" target="_blank">8381784</a>]<br></li>
<li>Furthermore , we demonstrate that the DNA-binding protein H-NS is involved in silencing the CRISPR-cas promoters , resulting in cryptic Cas protein expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>As a DNA-binding histone-like protein , H-NS could be involved in the control of expression of any such component .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Mutations in H-NS , a histone-like DNA-binding protein , derepress the expression of both a d i A and cadBA ( Shi etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Role of the histone-like protein H-NS in the curli control network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Here we demonstrate that the histone-like protein H-NS is also a component of this network , in which it functions as a global inhibitor of gene-expression during the exponential phase of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>We show that rcsA is silenced by H-NS , a histone-like protein that silences the expression of a variety of bacterial genes and affects recombination and transposition ( 3-9 ) , in a manner reminiscent of eukaryotic silencing ( 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
<li>H-NS is a histone-like protein able to repress the expression of numerous genes with unrelated functions ( Williams and Rimsky , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>) , encoding the histone-like protein H-NS , which represses expression of many genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>]<br></li>
<li>In addition , the histone-like protein H-NS represses curli synthesis , but seems to have an unexplained positive effect on the expression of the CsgD regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>The histone-like protein H-NS , FnrEc , and a second oxygen responsive regulator , ArcA , control its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>Recently , Sledjeski and Gottesman have identiﬁed H-NS , a histone-like protein , as a negative regulator of rcsA expression , as well as a * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>Sledjeski and Gottesman identiﬁed the histone-like protein H-NS as a regulator of rcsA expression ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>Q 2000 Blackwell Science Lt Results H-NS negatively regulates aerobic cydAB expression To determine whether the histone-like protein H-NS affects in vivo expression of the cydAB operon , an hns mutation was introduced by P1vir transduction into wildtype and DarcA E. coli strains lysogenized with the cydA ± lacZ fusion phage lPC64 ( Cotter et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Genetic and biochemical data showed that an Hha / H-NS protein complex modulates haemolysin expression in E. coli ( Nieto et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>DISCUSSION In the present study , we focused on the molecular mechanism adopted by the nucleoid-associated protein H-NS and the AraC-like activator GadX to cause antagonistic effects , negative and positive , respectively , on the expression of gad genes that play a major role in E. coli acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In addition to the bgl operon and other cryptic genes the H-NS protein also modulates the expression of a large number of active E. coli genes , usually by repressing transcription initiation ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>H-NS , the nucleoid protein ( Varshavsky et al. , 1977 ; Durrenberger et al. , 1991 ) , affects the expression of many unrelated genes , including proVWX , bgl , appY , ompC , and ® mB of E. coli ( Higgins et al. , 1988 ; Atlung et al. , 1996 ; Donato et al. , 1997 ) , and also affects the expression of some virulence genes of Salmonella typhimurium and Abbreviations used : DUF , domain of unknown function ; KRm , kanamycin resistant ; AmpR , ampicillin resistant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>H-NS , a nucleoid protein ( 7 , 25 ) , affects the expression of many unrelated genes , including proVWX , bgl ( 9 ) , appY ( 1 ) , and ﬁmB ( 6 ) of E. coli or Salmonella enterica serovar Typhi-murium , and also affects expression of some virulence genes of Salmonella serovar Typhimurium and Shigella spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>In order to gain a deeper understanding of the genetic control of hns expression , we have constructed a set of chromosomal hns -- lacZ protein and operon fusions and have used these hybrid genes to study transcriptional-control of hns in response to the growth phase and the synthesis of H-NS itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>A role for IHF or H-NS in regulating ftnA expression was tested by Western blot analysis of the corresponding himA ( one of two unlinked genes encoding the heterodimeric IHF protein ) and hns mutants , and parental strains ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Plasmid pHM52 , expressing the dominant negative H-NS mutant protein G113D and plasmid pHOP11 , encoding wild-type H-NS protein were used to transform wild-type ( MG1655 ) and hns - strains ( PD32 ) from which total RNA was analysed before and after the induction of the expression of H-NS or its mutant variant by primer extension .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>To this end , the wild type and an hns : : amp mutant were transformed with a pSC101-derived plasmid encoding a C-termi-nally HA-tagged H-NS protein and a pACYC-derived plasmid for expression of the hnsKN allele , which codes for the H-NSKN fusion peptide with a C-terminal HA tag .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine-metabolism ) was controlled by protein regulators such as FNR ( transcriptional regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine-metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>H-NS was originally isolated as a nucleoid-associated protein ( Jacquet et al. , 1971 ; Cukier-Kahn et al. , 1972 ; Spassky et al. , 1984 ) and repeatedly isolated as a transcriptional regulator of numerous genes ( Goransson et al. , 1990 ; May et al. , 1990 ; Falconi et al. , 1991 ; Kawula and Omdorff , 1991 ; Shi et al. , 1993 ; Bertin et al. , 1994 ; Levinthal et al. , 1994 ; Sledjeski and Gottesman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8670904" target="_blank">8670904</a>]<br></li>
<li>H-NS is a 15 kDa protein that acts as a regulator of more than 200 E. coli genes mostly related to adaptation to environmental stress ( Drlica and Rouviere-Yaniv , 1987 ; Hulton et al. , 1990 ; Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The H-NS protein is thought to play a major role in the condensation of genome DNA into the compact nucleoid because of its curved DNA-binding activity ( Yamada et al. 1991 ) and its abundancy ( 22 000 molecules/cell or 1 H-NS dimer for each 440 bp DNA ) ( Talukder et al. 1999 ) and thereby acts as a global transcription repressor for a number of genes ( for reviews see Drlica & Rouviere-Yaniv 1987 ; Pettijohn 1988 ; Atlung & Ingmer 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>A recent study has demonstrated that in E. coli K12 transcription from the casA and CRISPR I promoters is repressed by heat-stable nucleoid-structuring protein ( H-NS ) ( Pul et al. , 2010 ) , a global repressor of transcription in many Gram-negative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>Laboratory of Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464 , Japan Key words : Protein H-NS ( or H1 ) - Regulation of transcription - E. coli chromosome - Curved DNA Introduction An Escherichia coli protein , H-NS ( or H1 ) , was first identified many years ago and was recently confirmed to be a major component of the E. coli nucleoid ( Varshavsky et al. 1977 ; Durrenberger et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>These include the transcription factor FNR that mediates anaerobic regulation ( Unden and Schirawski , 1997 ) , the cAMP receptor protein ( CRP ) , which activates transcription of catabolic operons ( Kolb et al. , 1993 ) , and the nucleoid-associated protein H-NS ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>During growth in minimal medium , the leucine-responsive protein has been shown to repress the transcription of osmC and to activate that of osmC , p '' p # and the nucleoid-associated protein H-NS represses transcription from both promoters ( Bouvier et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>]<br></li>
<li>Overproduction of the nucleoid protein H-NS also leads to nucleoid condensation and a strong inhibition of transcription and translation , and results in a drastic loss of cell viability ( Spurio et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123448" target="_blank">12123448</a>]<br></li>
<li>The ﬂhD operon regulatory region is a direct target for several regulatory proteins : These are the cyclic AMP-catabo-lite gene activator protein complex , which is an activator in response to an alleviation of catabolite repression ( 60 ) ; OmpR P , which represses at high osmolarity ( 57 ) ; histonelike nucleoid-structuring ( H-NS ) protein , an activator ( 5 , 60 ) ; LrhA , a repressor ( 27 ) ; RcsAB , a repressor ( 17 ) ; and HdfR , a repressor ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>Factors required for motility and full flhDC expression include cyclic AMP ( cAMP ) and cAMP receptor protein ( CRP ; Kutsukake , 1997 ; Soutourina et al. , 1999 ) , histone-like nucleoid-structuring ( H-NS ) protein ( Bertin et al. , 1994 ) , the heat shock response network proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) , Pss and Psd , which are involved in the synthesis of phosphatidy-lethanolamine in cell membranes ( Shi et al. , 1993 ) , and inorganic polyphosphate ( Rashid et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>The factors required for full ﬂhDC expression ( and thus motility ) include cyclic AMP and the cAMP receptor protein CAP ( Kutsukake , 1997 ; Soutourina et al. , 1999 ) , the carbon storage regulator Csr ( Wei et al. , 2001b ) , the nucleoid-structuring protein H-NS ( Bertin et al. , 1994 ; Ko and Park , 2000 ) , the heat shock response proteins DnaK , DnaJ and GrpE ( Shi et al. , 1992 ) , the regulators Pss and Psd of phosphatidyl ethanolamine-synthesis ( Shi et al. , 1993 ) and inorganic polyphosphate ( Rashid et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>We have examined the survivors of prolonged stationary phase to identify loci involved in conferring a growth advantage and show that a mutation in the hns gene causing reduced activity of the global regulator H-NS confers a GASP phenotype under speciﬁc conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>We examined the effects of some global regulators ( factor inversion stimulation [ FIS ] , H-NS , OmpR , cAMP , ppGpp , and IHF ) on the expression of sra by using global regulator-defec-tive mutants lysogenized with lpF13 ( Psra-lacZ ) or carrying pFF6 ( Psra-lacZ ) and by measuring b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>INVOLVEMENT OF DIFFERENT REGULATORS IN THE ANAEROBIC EXPRESSION OF mdtEF FROM THAT OF AEROBIC CONDITION Previous studies focusing on the expression of gadE-mdtEF operon during acid resistance have revealed that several regulators , such as EvgA , YdeO , MnmE , GadE , GadX , CRP , and H-NS , are involved in the regulation of gadE-mdtEF operon ( Gong et al. , 2004 ; Ma et al. , 2004 ; Hirakawa et al. , 2006 ; Tramonti et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Identification of the targets directly controlled by H-NS H-NS modulates the expression of several regulators controlling acid stress resistance including HdfR , RcsD , EvgA , YdeO , YdeP , GadE , GadW , GadX , AdiY and CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>To elucidate the mechanism underlying this higher resistance to acid stress via GadX and/or GadE , we analysed the effect of their overexpression on gene-expression focusing our attention on H-NS gene targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>Results : We combined mutational , phenotypic and gene-expression analyses , to unravel the regulatory hierarchy in acid resistance involving H-NS , RcsB-P/GadE complex , HdfR , CadC , AdiY regulators , and DNA-binding assays to separate direct effects from indirect ones .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>H-NS controls genes related to pathogenicity and stress responses , and H-NS represses genes acquired by horizontal gene transfer , including pathogenicity islands ( 24 , 25 , 36 , 46 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>In this study , we use transcriptome sequencing to show that there is a bi-layered gene silencing system -- involving the homologous H-NS and StpA -- operating on horizontally-acquired genes among others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Moreover , these genes also recruit StpA as a molecular backup for H-NS , thus building a global epistatic gene silencing network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>In the case of the maltose regulon , H-NS has been shown to stimulate MalT translation ( 15 ) , and H-NS plays a positive role in the activation of the traJ gene of pRK100 ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>In gene regulation , H-NS acts as a transcriptional repressor in most cases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In addition , using the pleu-500 activity as a reporter , we have shown genetically that H-NS plays a repressive role in the transcriptional regulation .2 Together , these results suggest a possible involvement of H-NS in AT4-mediated gene silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121424" target="_blank">11121424</a>]<br></li>
<li>If in fact H-NS regulation of this putative gene is negative , this supports the idea that rapid degradation of RpoS may be the default situation , and stabilization under speciﬁc conditions underlies regulated proteolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980505" target="_blank">16980505</a>]<br></li>
<li>Similarly , the complex regulation of the acid stress response gene gadA , which is activated by GadE and RcsB , involves repression by H-NS ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>These include ( i ) their maximal induction at low pH under anaerobic conditions in the presence of excess substrate , ( ii ) the fact that they are major cellular proteins , ( iii ) the use of pyridoxal 5 ' - phosphate as a cofactor , ( iv ) their proposed role in pH homeostasis , and ( v ) the involvement of H-NS , which reduces expression of both systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>H-NS regulates the expression of proVWX , bgl , appY and ® mB of Korea Advanced Institute of Science and Technology , Yusong-Ku , Taejon 305-710 , South Korea , and dDepartment of Life Science , Division of Molecular and Life Science , Pohang University of Science and Technology , Pohang 790-784 , South Korea ² Dr Lee should also be considered as corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077448" target="_blank">12077448</a>]<br></li>
<li>As LeuO is an H-NS antagonist with pleiotropic function , activation of leuO expression should occur in response to speciﬁc environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Our ﬁndings demonstrating that ArcA activates the expression of the operon through antagonizing the H-NS mediated repression probably provide a common model that underlies the dynamic and coordinated regulation of gadE-mdtEF in response to different environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Therefore , it is possible that StpA plays a role in modulating the expression of OmpF in response to environmental signals under conditions in which H-NS repression has been removed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>This suggests that during anaerobic growth in liquid cultures in the absence of glucose , FNR is the major regulator of hlyE expression , and that CRP-mediated hlyE expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Thus , many global regulators , including cAMP , H-NS , IHF , and ppGpp , inﬂuence the expression of sigma S , which in turn may regulate the transcription of sra in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>In summary , S the above results indicate that s is one of the sigma factors involved in hchA transcription , while H-NS mainly acts as a silencer of hchA gene-expression during the exponential phase of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>We show here that premature carbon starvation causes a rapid shutoff of stpA transcription which is independent of H-NS ; this is consistent with the complete absence of stpA expression in stationary-phase cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Thus , expression of any one of several dominant-negative variants of H-NS , including 64 , L26P , and E53G/T55P , restores the efﬁciency of transcription termination ( that is , suppresses polarity relief ) in rho and nusG strains , as does overexpression of YdgT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>Regulators that affect ﬂhDC expression by controlling ﬂhDC transcription ( LrhA , OmpR-P , H-NS , cAMP/CAP ) or mRNA stability and translation ( CsrA ) positively ( + ) or negatively ( -- ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>DsrA is a small untranslated RNA that regulates expression of two global transcription factors , H-NS and RpoS ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
<li>RESULTS Effects of the global transcription factors FNR , CRP , and H-NS on hlyE expression in liquid culture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>It therefore has a dominant negative effect on wild-type H-NS activating the transcription of the Pcas promoter and thus provides a ` proof of principle ' for the expression of the CRISPR-cas system by anti-silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>garP expression demonstrated a purely negative effect of H-NS on transcription , as an hns651 mutation led to increased expression of garP at both temperatures ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Global regulators such as H-NS ( 27 , 28 ) , OmpR ( 30 ) , and IHF ( 36 ) control the expression of the curli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>In addition to these three two-component systems , global regulators such as H-NS ( 1 , 28 , 36 ) and IHF ( 22 ) control the expression of the curli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>In vitro transcription analysis of H-NS - and Fur-dependent control of ftnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Expression of the caiF gene is dependent on the same general regulatory proteins , H-NS , CRP , FNR , and transcription factor RpoN ( 11 ) , which control cai and ﬁx expression ( Table 2 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>We showed that their expression is under the negative control of H-NS and , except for yad , subjected to cAMP receptor protein-mediated activation and carbon-catabolite-repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Thus , CRP and FNR positively control cai expression and H-NS exerts a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Consistent with this hypothesis , H-NS derivatives R93H , T108I , and G111S caused 6 - to 25-fold increases in ﬁmB expression ( Fig. 1B ) while maintaining a wild-type inversion phenotype ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>The expression of H-NS also seems to be coupled to DNA-synthesis ( Free and Dorman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The expression of hdfR was negatively controlled by H-NS , which explains the negative effect on ﬂagellation of an H-NS mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>In addition , H-NS is also involved in regulating expression of virulence-associated genes in pathogenic Shigella spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>We suggest that H-NS acts as a system regulator that is involved in coordinating the expression of the ﬂagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>H-NS is involved in the posttranscriptional-control of sS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Conclusions Several lines of evidence con cents rmed that H-NS is extensively involved in modulating the expression of type 1 cents mbriae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>At low osmolarity the mutation of hns caused a reduction of csgD expression , thus indicating that H-NS acts as an activator of csgD in this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Taken together , these results strongly suggest that H-NS negatively regulates cydAB expression under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>This suggested that H-NS only controls rcsD expression within the Rcs phosphorelay pathway that leads to RcsB phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>They proposed the presence of an efﬁcient locking mechanism involving H-NS that prevents inadvertent expression during normal growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>In the absence of glucose , H-NS appeared to have a positive effect on hlyE expression ( Fig. 1c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>H-NS represses expression both in aerobically and in anaerobically growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>In aerobically growing E. coli cells , H-NS represses nrdAB expression under all conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Thus , PhlyE ` see-saws ' between an H-NS dominated nucleoprotein complex ( A and B ) , which silences expression , and a SlyA dominated complex , which promotes expression ( D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>As shown in Figure 6A , the induction of StpA overproduction caused a significant decrease in the expression of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Moreover , we show that H-NS positively affects curli expression by inversely controlling the expression of ydaM and yciR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>ArcA is shown to antagonize H-NS action to render cydAB expression insensitive to H-NS repression in anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Effect of ArcA and H-NS on induction of appY expression under oxygen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Acknowledgments We thank E. Bremer ( FRG ) for providing plasmids for expression of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Thus it appears thatthe ability of H-NS to modulate expression of íts own structl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>These authors also examined the expression of stpA in response to H-NS , Lrp , and temperature and the requirement for promoter-distal DNA for regulation , and they obtained results in good agreement with our own .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Taking into account the facts that nrd genes are also environmentally regulated and that H-NS inﬂuences DNA architecture and modulates gene-expression in response to environmental factors ( 32 -- 34 ) , we hypothesized that H-NS might participate in the modulation of the expression of some of the nrd operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>while emphasizing the central role of H-NS in the global control of gene-expression in response to various physiological changes , may also explain Ihe different extent of hns auto-repression during various stages of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Therefore , we tested the role of hns and rpoS in the regulation of nhaA and found that H-NS , not sS , plays a most important role in the control of nhaA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Figure 1 demonstrates that exponential-phase expression of the transcriptional csi-5 : : lacZ fusion ( located in the chromosomal copy of the osmY gene ) is derepressed in a strain that carries an insertional inactivation of the hns gene , indicating that H-NS is also involved and plays a negative role in the regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>It is also involved in transcription termination ,20 and can bind and stabilise mRNA .21 As a transcriptional regulator , H-NS silences gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>H-NS , the 14 kDa protein encoded by the hns gene , has been suggested to be involved in the overall control of gene-expression either because it modulates DNA supercoiling in an appropriate fashion [ 3 ] or because it functions as a transcriptional repressor at specific sites involving DNA bending [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>H-NS , encoded by hns , is involved in the transcriptional regulation-of-gene-expression , genetic recombination , and the topological organization of the chromosomal DNA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>]<br></li>
<li>gadX gene is under transcriptional-control of H-NS and RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>In several other cases of thermoregulated gene-expression , the H-NS protein ( and presumably also the StpA protein ) has been found to play a negative role at the transcriptional level ( reviewed in , e.g. , references 2 and 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Alternatively , H-NS and StpA may act at the translational level or act indirectly through negative control of a gene encoding a repressor of the expression of the malT gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>In the present paper , we demonstrate that in vivo H-NS exerts a negative control on the transcriptional expression of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>The expression of over 80 % of these genes was induced in the hns mutant strain , which further supports the role of H-NS as a repressor of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Control by H-NS and related proteins on gene-expression The physiological role of H-NS and related proteins displays important differences between microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>For H-NS , an activating function is quite uncommon , as it usually plays the role of a general repressor or even a silencer of gene-expression ( Williams and Rimsky , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>H-NS plays a predominantly repressive role for transcription of cysP at low temperature , with the hns651 mutation leading to statistically signiﬁcant increased transcription at 23 ° C and a subsequent loss of a thermoregulatory differential in gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Role of H-NS in the Regulation of PR1 -- Previous studies have shown that H-NS plays an unusual role in regulating Group 2 capsule gene-expression , appearing to maximize transcription from PR1 at 37 ° C while repressing transcription at 20 ° C ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Despite the identiﬁcation of two other factors involved in osmC regulation , LRP and H-NS , neither of these factors is related to the osmolarity-dependent expression of the gene , suggesting that other regulators might control osmC expression ( 4 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>Finally , although four regulators involved in the control of osmC expression have now been identiﬁed ( LRP , H-NS , s , and RcsB ) , none could explain the osmotic regulation of this gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>There is frequent evidence for a tight coupling of H-NS and LRP in regulation-of-gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Role of H-NS in anaerobic regulation of appY gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Recently , H-NS 's role in global regulation-of-gene-expression became apparent ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_776">
A role for H-NS in the thermo-osmotic regulation of virulence gene-expression in Shigella ﬂexneri .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></label>
<input id="rp_similarity_776", type="checkbox">
<div class="RP_similar">
<li>A role of H-NS in the thermo-osmotic regulation of virulence gene-expression in Shigella ﬂexneri .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li></div>
<li>Mutations in hns , the gene encoding H-NS , have pleiotropic effects on the cell altering both the expression of a variety of unlinked genes and the inversion rate of the DNA element containing the ﬁmA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Mutations in the gene coding for H-NS , hns , result in elevated expression of several other gene products , and it appears that H-NS may cause silencing of many different op-erons ( reviewed in reference 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>Proc Natl Acad Sci USA 82:5724 ± 5727 Atlung T , Ingmer H ( 1997 ) H-NS : a modulator of environmentally regulated gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>Furthermore , because H-NS and StpA coordinately regulate some genes while independently regulating others , small perturbations in H-NS levels can cause global changes in gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>These alterations in gene-expression indicate that the intracellular amount of H-NS ( Hla ) is a critical determinant of proper expression of a number of chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2177526" target="_blank">2177526</a>]<br></li>
<li>Loss of hns function results in diverse and unrelated phenotypes , indicating that H-NS is a global regulator of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>H-NS is a global inhibitor of gene-expression during the exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>]<br></li>
<li>We were , therefore , interested to examine the effect of the superhelicity of cellular DNA on expression of the bgl gene in mutant cells lacking H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>H-NS interacts largely within the ﬁrst structural gene of the proU operon , proV , to strongly repress operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002611" target="_blank">8002611</a>]<br></li>
<li>H-NS negatively modulates the expression of its own structural gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Discussion DsrA RNA acts as an orchestrator of cellular gene-expression by simultaneously modulating the activities of at least two global transcriptional regulators , H-NS and RpoS ( Fig. 5B ) , with potential for direct interactions with the RNAs of at least three more genes , argR , ilvIH , and rbsD ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10954740" target="_blank">10954740</a>]<br></li>
<li>Such environmental signals are mediated at the gene-expression level by a number of regulators , including , besides CsgD , global regulatory proteins such as OmpR , H-NS , CpxR , and * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>Polyamines , in particular putrescine , play an important role in various cellular-processes and can affect intracellular concentrations of the regulatory proteins sN , Cra and H-NS , thus impacting global gene-expression ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>These data , together with those for evgAS deletion , clearly indicate that the hns deletion-mediated increase in the expression of drug exporter genes is independent of EvgAS - and YdeO-mediated regulation DISCUSSION In this study , we found that H-NS represses the expression of some TolC-dependent multidrug exporter genes and that , as a result , deletion of the hns gene confers multidrug-resistance on the acrAB-deﬁcient strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>H-NS is a global transcriptional regulator that affects various unlinked genes which include ompF and ompC .53 H-NS represses several genes in environmental conditions that pertain to outside of the intestine but allows for expression in the intes-tinal tract where environmental signals can activate genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>This regulation is different from that of many H-NS controlled genes , where H-NS serves as a transcriptional silencer in response to a variety of environmental cues and deletion of H-NS leads to high-level expression under th normally repressive cue ( reviewed in references 1 and 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>We succeeded in cloning and identifying genes encoding such proteins and demonstrate that the expression of these genes is negatively regulated by H-NS at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>H-NS acts as a transcriptional repressor or silencer , with an H-NS-deﬁcient strain showing increased expression of 4100 genes ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Here we report ﬁndings showing that H-NS and the largely homologous protein StpA play a positive role in the expression of genes in the maltose regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>DISCUSSION The present data show that the H-NS and StpA proteins play a positive role in the expression of some genes of the maltose regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>The major role of H-NS is to modulate the expression of a large number of unlinked genes ( Atlung & Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077448" target="_blank">12077448</a>]<br></li>
<li>The effects of iron , Fur and H-NS on the expression of the ftnA -- lacZ transcriptional fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The expression levels of both sets of genes are low , as expected from the role of H-NS as a transcriptional silencer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>The results of such promot-er-probe analyses indicated that H-NS represses the expression of these genes at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Natural selection for buffering of the expression of many genes could produce general regulators like H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>groups to assume that H-NS exerts a negative control over the expression of gad genes during the exponential phase ( 7 , 8 , 10 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>H-NS represses the expression of many genes that are expressed in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
<li>H-NS is a positive regulator of ﬂhCD , the master operon which controls the expression of all ﬂagellar genes required for motility ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Based on these genome-wide analyses we propose that the expression of the type E ( Cse ) cas genes from S. enterica are likely to be regulated by H-NS and LeuO as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>H-NS controls the expression of a number of environmentally controlled virulence genes in several gramnegative genera ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Several mutations that elevate expression of diverse sets of genes proved to be mutations of H-NS ( 11 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li>
<li>Plasmid-expressed DsrA increased expression of genes normally repressed by H-NS , including rcsA , proU , and papA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
<li>Whereas certain sS-dependent genes ( e.g. , osmY ) are only indirectly regulated by H-NS via its role in the control of sS expression , others are also H-NS-regulated in a sS-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>In addition , they are regulated in response to very different environmental signals : H-NS influences the expression of genes regulated by osmolarity , temperature , pH , oxygen availability , and genes under stringent control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The synthesis and expression of these ﬂagella and motility genes are regulated by several environmental cues besides quorum-sensing , including temperature ( Adler and Templeton , 1967 ) , osmolarity ( Shin and Park , 1995 ) , cAMP-CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , cell-cycle control ( Nishimura and Hirota , 1989 ) , RcsCDB ( Francez-Charlot et al. , 2003 ) , H-NS ( Bertin et al. , 1994 ) and integration host factor ( IHF ) ( Yona-Nadler et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>The stationary phase global regulator RpoS ( s ) s ( Olsén et al. , 1993 ; Römling et al. , 1998b ) and other global regulators including H-NS ( Olsén et al. , 1993 ; Arnqvist et al. , 1994 ) , Crl ( Arnqvist et al. , 1992 ) and OmpR ( Römling et al. , 1998b ; Vidal et al. , 1998 ) can effect curli synthesis by promoting or suppressing expression of the curli-encoding genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The stationary phase global regulator RpoS ( S ) ( Olsén et al. 1993 ; Römling et al. 1998 ) and other global regulators including H-NS ( Arnqvist et al. 1994 ; Olsén et al. 1993 ) , Crl ( Arnqvist et al. 1992 ) , OmpR ( Römling et al. 1998 ; Vidal et al. 1998 ) , MlrA ( Brown et al. 2001 ) and CRP ( Zheng et al. 2004 ) can aVect curli synthesis by promoting or suppressing expression of the curli-encoding genes by regulating the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>The gadB and gadC genes form an operon and a number of factors , including H-NS , RpoS and CRP-cAMP , are involved in the transcriptional-control of gad expression ( Castanie-Cornet & Foster , 2001 ; De Biase et al. , 1999 ; Rowbury , 1997 ; Rowland et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>The OmpR regulator is directly involved in activation of curli expression , in concert with several other regulatory proteins such as RpoS , H-NS , CpxR , and other yet unknown factors ( Romling et al. , 1998b ; Prigent-Combaret et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>In fact , mutations altering expression of some unlinked genes , i.e. , bglY ( Defez and DeFelice 1981 ) , cur ( Diderichsen 1980 ) , pilG ( Spassky et al. 1984 ) , osmZ ( Higgings et al. 1988 ) and virR ( Hromockyj and Maurelli 1989 ) , have been shown , or are very likely , to affect H-NS activity ( G6ransson et al. 1990 ; Higgins et al. 1990 ; Hulton et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This finding indicates that physiological conditions in the host could induce bgl expression , although so far no culture conditions were defined that relieve silencing of the bgl promoter , and it relates to the essential role of H-NS , which is a regulator of virulence , stress induced and other genes ( reviewed in Ussery et al. , 1994 ) , in bgl silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>H-NS plays a role in the organization of the bacterial chromosome and , in addition , participates in the modulation of the expression of many bacterial operons , including some that encode virulence factors ( Dagberg and Uhlin , 1992 ; Harrison et al. , 1994 ; Dorman and Por-ter , 1998 ; White-Ziegler et al. , 1998 ; Nye et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>These environmental signals are mediated at the gene-expression level by a number of regulators , such as OmpR , H-NS , Besides its role as activator of the csgBA operon , CsgD regulates a number of genes involved in biofilm-formation and production of cell-surface-associated structures ( Latasa et al. , 2005 ; Gibson et al. , 2006 ) , as well as genes involved in transport , metabolism and gene regulation ( Chirwa & Abbreviation : CV , crystal violet .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18599830" target="_blank">18599830</a>]<br></li>
<li>H-NS silencing of gene-expression is relieved in response to a number of environmental stimuli , including growth phase and osmolarity ( Hulton et al. , 1990 ; Atlung and Ingmer , 1997 ; Laurent-Winter et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>In fact , an Hha/H-NS complex appears as responsible for the modulation of the expression of the toxin a-haemolysin ( Nieto et al. , 2000 ; Madrid et al. , 2002b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>SspA is a transcriptional regulator that affects the expression of H-NS ; however , sspA expression is independent of rpoS ( Niedhardt , 1996 ; Williams et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849790" target="_blank">16849790</a>]<br></li>
<li>H-NS also represses the expression of evgA and gadX ( Hommais et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Interestingly , expression of mutant HU induces activation of a subset of genes that is normally silenced by H-NS ( Kar et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>H-NS is an abundant , approximately 15 kDa nucleoid-associated protein that is present at about 20 000 copies per genome equivalent and is a well-known silencing protein that represses expression of cryptic genes such as the bgl operon involved in b-glucoside utilization ( Dorman and Deighan , 2003 ; Dole et al. , 2004 ; Dorman , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>HvrA is a 103-amino-acid protein ( 137 in H-NS ) known as a trans-acting positive regulator involved in the control of photosystem gene-expression in R. capsulatus ( Buggy et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>This involves the H-NS protein , which has also been shown to be involved in regulating yhiM expression ( Zhang et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22327802" target="_blank">22327802</a>]<br></li>
<li>Previous work has shown that H-NS is required for full expression of OmpF porin protein and that this involves a role for H-NS in repressing the micF promoter ( Suzuki et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>H-NS mutations also affect DNA supercoiling , and a modulation of this parameter through H-NS action might be involved in the control of gene-expression ( Hulton et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>The increased H-NS levels may be involved in shutting down gene-expression during cold-shock , and H-NS appears to be important for the adaptation to low temperature ( Dersch et aL , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Effect of H-NS on DNA-binding protein-encoding genes Several genes that are upregulated in the hns mutant code for proteins with nucleic acid-binding properties Changes in the levels of mRNAs of starvation-induced proteins in the hns mutant strain Some genes induced by oxygen starvation , such as cydAB , which encodes the cytochrome d ubiquinol oxidase complex ( Cotter et al. , 1990 ) , and appB , which encodes a cytochrome bd II oxidase ( Sturr et al. , 1996 ) , showed increased expression in the hns mutant background ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>More recently , FIS has been shown to modulate expression of some RpoS-regulated genes ( Xu and Johnson , 1995a , b , c ) , to regulate the promoters of the hupA and hupB genes encoding HU ( Claret and Rouviere-Yaniv , 1996 ) and of the hns gene encoding H-NS ( Falconi et al. , 1996 ) , and to regulate expression of several operons involved in the catabolism of sugars and nucleic acids ( González - Gil et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">H-NS binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Target DNAs ( 200 to 500 ng ) were incubated with H-NS ( 0 to 6 M ) in 25 mM HEPES , pH 7.6 , containing 0.1 mM EDTA , 10 % ( vol/vol ) glycerol , 5 mM dithiothreitol , 50 mM KCl , and 0.01 mg of poly ( dI-dC ) ml 1 for 15 min at 25 ° C in a total volume of 10 or 20 l. Complexes were separated in 6 % ( wt/vol ) nondenaturing Tris-borate-EDTA-buffered polyacrylamide gels for the long PhlyE fragment or in 12.5 % ( wt/vol ) gels for the dissected PhlyE fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In contrast , the third deletion ( D3 ) , which lies closest to ﬁmB and removes sequences known to bind the repressor H-NS ( Donato et al. , 1997 ) , produced a signiﬁcant ( twofold ) increase in the frequency of ﬁmB-promoted inversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890032" target="_blank">12890032</a>]<br></li>
<li>At low concentrations H-NS speci cents cally bound to a fragment of 1148 bp which spanned the region from 3920 to +206 , relative to the translational start point of cents mB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>Plasmid pHYD2509 was obtained by subcloning an EcoRI fragment encoding H-NS 64 ( genomic coordinates from bp 1291698 to 1292238 [ 48 ] ) from the corresponding pLG plasmid of Williams et al. ( 65 ) into the EcoRI site of pWSK30 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>The sequence of the H-NS region contained the following differences , in addition to those noted by May et al. ( 26 ) , from the sequence reported by Pon et al. ( 32 ) : A at nucleotide 601 ( numbering according to reference 32 ) was missing , and G was inserted between nucleotides 1315 and 1316 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1537791" target="_blank">1537791</a>]<br></li>
<li>Note that this sequence contains an almost perfect match to the recently published high affinity nucleation site for H-NS ( Bouffartigues et al. , 2007 ; Lang et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>A sequence TCGTTTTATT located between -200 and -250 indicated by an arrow and a double bar represents the high affinity nucleation site for H-NS published recently ( Bouffartigues et al. , 2007 ; Lang et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Sequence alignments have shown that Hha and H-NS share significant amino-acid homology in their N-terminal regions , corresponding to the site of the H-NS oligomerization domain60 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Sequence analysis of hns mutations H-NS a No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>B. Sequence of E coli H-NS with charged residues depicted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Binding of H-NS to foreign DNA elements depends on the generally higher AT content of the invading DNA compared with the host , while the CRISPR system recognizes speciﬁc sequences , which have been taken up by the host from previous infections .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Derivation of a DNA binding motif template for H-NS Average AT content ( % Gene Site Fit ( / 10 ) 9 proV 1 proV 2 TCGATATATT TCGATATATT CGGATAAATC TCGTTAAAAT TAGATTAAGT CCGATTAATC TCCTTACATT TGTATACTTT tCGtTAaATt a t GCGATAATTA GCATTTAAAA GCGACTAAAA 9 7 9 7.5 7.5 8 6 gadA gadX virF cspA hns OriC Template ﬁs 1 ﬁs 2 ﬁs 3 7 5.5 5.5 The proV sequences are taken from ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>The rcsA promoter , like others in which H-NS is implicated as a silencer ( 38 , 39 ) , appears to be capable of high levels of transcription initiation , although the sequences at -10 and -35 are not particularly close to consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
<li>The sequences are aligned from tile E coli ATG initiation codon of H-NS t ~ t ; , ' - '' ~ ` a ~ Also stippled are the -10 and -35 promoter elements identified in E coli [ I 3 ] and , when present , the CCAAT cold shock box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748932" target="_blank">7748932</a>]<br></li>
<li>The relevant sequences present in the fragment ( from left to right : the-35 and -10 element of the promoter , the transcriptional startpoint , the CCAAT motif , the Shine-Daigamo sequence , and the H-NS initiation triplet ) are indicated by bold letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748932" target="_blank">7748932</a>]<br></li>
<li>Also , it is reasonable to assume that this site is responsible for transcriptional repression by H-NS , the interaction of which with DNA might occlude the access of RNA polymerase to the 10 and 35 consensus sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>( ii ) Although site I overlaps the 10 and 35 consensus sequences of gadA and gadX promoters and H-NS bound at this site may compete with RNA polymerase , the deletion of distal site II ( i.e. pKgadA-69 and pKgadX-67 ) significantly reduces the ability of H-NS to inhibit gadA and gadX promoter activity both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>As mentioned above , the CS7 .4 protein has been identified as the factor present in extracts of cold shock cells responsible : ; : ~ the transcriptional enhancement of hns recognizing a i i0 bp DNA fragment ( fig I ) containing the - 1 0 and - 3 5 elements of the promoter and a CCAAT motif located between the transcriptional startpoint and the first few codons of H-NS [ 6 ] * Correspondence and reprints In the present paper we investigated whether additional signals for CS7 ,4 might exist upstream of its I 10 bp target and determined the effects of CS7 .4 on the accessibility of this DNA fragment to DNase I. Finally , to determine the relevance of a conserved CCAAT motif in eliciting the cold shock enhancement we investigated the cold inducibility of the hns promoter derived from Proteus vulgaris which , unique among the known Enterobacteriaceae hns genes ( fig I ) , lacks this motif , Our results indicate that no additional cold shock signal is present in the hns promoter , at least within 550 bp upstream the 110 bp fragment constituting the main target and that cold shock induction in vivo can occur also in the absence of a CCAAT sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748932" target="_blank">7748932</a>]<br></li>
<li>S0300908401012330/FLA Mutagenesis of the downstream region of the Escherichia coli hns promoter M. Giangrossi , C.O. Gualerzi * , C.L. Pon Laboratory of Genetics , Department of Biology MCA , University of Camerino , 62032 Camerino ( MC ) , Italy ( Received 21 December 2000 ; accepted 15 January 2001 ) Abstract -- The promoter of hns , the structural gene for the abundant nucleoid-associated protein H-NS of Escherichia coli , contains , downstream of the initiation site , two four bp-long ` CG clamps ' , one of which overlaps the potential target sequence ( CCAAT ) of CspA , the cold-shock transcriptional enhancer of this gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>The 2840 bp band exhibited a mobility shift starting at 1.08 mM H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In addition , for H-NS and StpA , which are known to have a tendency to bind non-specifically to curved DNA , we used a random DNA fragment ( 222 bp AvaII fragment from pUC18 ) as control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>The AT4 DNA sequence was 59-CACAATCATACACCAA - 1 The abbreviations used are : bp , base pair ( s ) ; PCR , polymerase chain reaction ; IPTG , isopropyl-1-thio-b-D-galactopyranoside ; H-NS , histonelike nucleoid structuring protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121424" target="_blank">11121424</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Here we identified that five regulatory proteins ( CpxR , OmpR , RstA , IHF and H-NS ) bind within a narrow region of about 229 bp along the csgD promoter ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Though the data are not deﬁnitive , it also appeared that H-NS may bind to at least two separate sites within the ﬁmB promoter , as illustrated by multiple complexes within one reaction lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In this article , we report that Fis and H-NS bind to adjacent sites within the core dps promoter and selectively shut down transcription by Es70 but not by Es38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>For construction of plasmids for His-tagged transcription factors , pCpxR , pH-NS , pOmpR and pRstA , DNA fragments corresponding to the coding sequences of the respective transcription factors were amplified by PCR using E. coli W3110 genomic DNA as a template and pairs of primers ( Table 1 ) designed to hybridize upstream or downstream of each coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Since the same low H-NS-sensitive level is seen for fusions extending further upstream than -97 , it would seem that rather than a specific recognition sequence for H-NS , these changes have disrupted or restored some aspect of DNA structure which allows H-NS to bind and/or act .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
<li>In either case , H-NS is not acting as a classical repressor since we do not believe that it recognizes a speciﬁc consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Because of the angular distortion of the promoter sequence by bound H-NS molecules , the sub-optimal spacing of the - 1 0 and - 3 5 regions is no longer able to make stable contacts with the polymerase surface ( Travere , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>A. DNase I footprint analysis of the 246 bp ftnA promoter sequence ( both sense and antisense strands ) with and without the addition of 1 mM H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Using hdeABp as a model promoter , we observed with purified components that H-NS , acting at a sequence centered at − 118 , selectively repressed transcription by E 70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Fis that binds to a degenerate consensus sequence was initially discovered as a factor in site-speciﬁc DNA inversion reactions ( Johnson et al. , 1988 ; Koch et al. , 1988 ) and later found to function in transcriptional activation of tRNA and RNA operons ( Nilsson et al. , 1990 ; Ross et al. , 1990 ; Condon et al. , 1992 ) as well as transcription factor H-NS ( Falconi et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>One H-NS dimer binds to each 15 -- 20 bp ( Amit et al. , 2003 ) and bridges adjacent tracts of DNA , resulting in lateral condensation of large regions and formation of large globular structures ( Dame et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Instead , Hha binds to H-NS , which binds to speciﬁc regions of regulatory sequences in this operon ( Nieto et al. , 2000 ; Madrid et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>This finding prompted us to propose a model in which H-NS may function as a transcriptional regulator through its ability to bind to curved DNA sequences , which are found in regions upstream of a certain subset of E. coli promoters ( Tanaka et al. 1991a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>The finding that H-NS binds to both synthetic and naturally occurring curved D N A sequences supports the idea that H-NS may function as a global regulator by recognising local alterations in DNA curvature ( Yamada et al. 1990 ; Tanaka et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>H-NS recognizes DNA with a certain configuration , such as curved DNA generated by stretches of AT-rich sequence ( AT tracts ) rather than a specific sequence ( Rimsky et al. 2001 ; Rimsky 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>H-NS binds DNA without any obvious sequence speciWcity but displays preferential binding to DNA that is curved and/or more Xexible than random DNA ( Dame et al. , 2001 ; Jordi et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>H-NS binds preferentially to sequences that are high in A1T content ( Lucht et al. , 1994 ) and contain bent DNA ( Yamada et al. , 1990 ; Owen-Hughes et al. , 1992 ; Falconi et al. , 1993 ; Zuber et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>H-NS is a highly abundant protein that binds to DNA with a preference for AT-rich sequences ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>H-NS has a preference for binding AT-rich DNA sequences ( Navarre et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>H-NS binds to DNA with a preference for bent and AT-rich sequences ( Ussery et al. , 1994 ; Schröder and Wagner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>It has previously been shown that H-NS binds to PhlyE in regions upstream ( H-NS I ) and downstream ( H-NS II ) of the transcript start site , and that a SlyA binding site overlaps H-NS I ( Westermark et al. , 2000 ; Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Therefore , it is suggested that H-NS ﬁrst binds at AT-rich regions up - and downstream of the hlyE transcript start site ( Wyborn et al. , 2004b ; Fig. 7 , A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Taken together these data suggested that at lower concentrations H-NS may bind at AT-rich speciﬁc nucleation sites , and at higher concentrations , multiple H-NS proteins oligomerize to occupy both the upstream and downstream regions of PhlyE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Overall , the footprinting data suggested that a Phly - complex ( Fig. 4A , lanes 3 -- 5 ) , but mobility was increased E : SlyA complex is formed when SlyA binds at speciﬁc in the presence of 1.25 and 2.5 mM H-NS ( Fig. 4A , lanes sites in the upstream and downstream regions of the 6 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The radiolabelled 471 bp BamHI ± BclI cydAB promoter fragment ( Fig. 1 ) , including upstream sequences from 2712 to 2243 relative to the cydA translational start , was incubated with different amounts of H-NS in the presence of excess unlabelled competitor DNA [ poly - ( dI ± dC ) ; see Experimental procedures ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>We concluded from these experiments ( Fig. 5 and Table 2 ) that H-NS functions as a direct transcriptional repressor of ﬁmB expression speciﬁc for the natural ﬁmB promoter and/or upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In this paper , we demonstrate an essential role for the histone-like protein H-NS in the complex pattern of cydAB extended region within the cydAB promoter element , including sequences upstream from and overlapping the four regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>In Fig. 2 B and C , the sequences of the upstream region relevant to H-NS silencing are shown , as well as the vector sequences which are present upstream in the fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
<li>This result agrees with the gel retardation assays , which showed that H-NS bound preferentially to upstream pnir sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The current view of the mechanism of H-NS-mediated repression is that H-NS binds as a dimer to AT-rich and planar-bent DNA sequences and then forms an extended oligomeric complex on the DNA , which represses transcription initiation when located next to a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>When H-NS acts as a silencer or repressor it binds to AT-rich , curved sequences and thereby blocks transcription of the gene in question ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>H-NS binds preferentially to sequences which can adopt an intrinsic curve [ 40 , 42 -- 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>However , we also do not think that H-NS binds randomly to ﬁmB promoter sequences to exert an effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Finally , the nucleoid associated protein , H-NS , preferentially binds to upstream sequences at the nir promoter and represses promo-ter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>With the HaeIII ± HindIII digest , we observed that H-NS binds preferentially to the 216 bp HaeIII ± HindIII fragment carrying the entire pnir sequence ( lanes 2 ± 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>With the HaeIII ± NsiI digest , we observed that H-NS binds preferentially to the 115 bp HaeIII ± NsiI fragment carrying the upstream pnir sequence , in comparison to the 148 bp HaeIII ± NsiI fragment carrying the downstream pnir sequence ( lanes 8 ± 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Molecular Microbiology ( 2000 ) 37 ( 5 ) , 1258 ± 1269 S ppression o FNR-de e dent transcription at the cherichia coli nir promoter by Fis , IH H-NS dulation of transcription initiation b ucleo-protein ssem y u f p n activation F a d Es n : mo y a complex n a bl Summary promoters , FNR binds as a dimer to a 22 bp sequence centred near position 241 , upstream of the transcript start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>60 ° C 622 bp -- sz7 bp -- f 404 bp -- 309 bp -- 242 bp -- H - N S binds to a D N A fragment encompassin9 the hns promoter sequence The results of the in vitro transcription experiment with purified H-NS suggested that the protein must bind somewhere around the hns promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Note also that the 100 bp DNA marker ladder appears fortuitously to contain a curved DNA segment ( indicated by the stars ) as previously noted by the manufacturer H-NS binds to DNA fragments encompassing the putative hns-dependent promoters Considering the results presented above , it was suspected that H-NS may directly bind to a region somewhere near the putative promoter sequences characterized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>In the case of restriction fragments originating from plasmid pPBO23 ( cents mE-lacZ ) , H-NS exhibited speci cents c binding a $ nity for a fragment of 897 bp ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>Analysis of HincII-digested PCR fragment 0 mixed with a restriction enzyme digest of plasmid pTAC3953 ( Fig. 8B ) showed that there are at least two high-afﬁnity H-NS binding sites , since both resulting fragments were speciﬁcally bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Our studies demonstrate that LRP binds in addition to H-NS and FIS to the upstream regulatory sequence region ( UAS ) of rRNA P1 promoters , and thus potentially contributes to the efﬁcient shutdown of rRNA synthesis , which is known to occur at the onset of stationary phase or under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Extending the yjjP promoter fragment to include the central region ( Fig. 5B , PP 250 to 25 ) or the complete intergenic region ( Fig. 5B , PP 450 to 25 ) resulted in more effective repression by H-NS , in agreement with results described above that suggest that H-NS binds to the central region and to sequences distal to the divergent yjjQ promoter region ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Taken together , these results demonstrate that H-NS binds to the center region between the divergent promoters and to sites located distal to the two promoters ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Mapping of the promoters as well as the H-NS and LeuO binding sites within the 555-bp intergenic region revealed that H-NS binds to the center of the AT-rich regulatory region and distal to the divergent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Mapping of the divergent promoters and regulatory sites revealed that H-NS binds at the center of the extended intergenic region and distal to the divergent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Insertion elements , located from 145 to 13 bp relative to the transcription start site , activated the ade gene as did unlinked mutations in the hns gene , encoding the histone-like protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>Since the preferred in vivo targets of H-NS are generally AT-rich ( 12 ) , we compared the base composition of these two classes of genes for the region from 300 to +100 bp relative to the translation start site ( see Methods section ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>( C ) Average AT content proﬁles of sequences from 300 to +100 bp relative to the start of translation for sets of genes differentially expressed in hns cells but containing or lacking a predicted H-NS binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>We found that of the 57 genes scanned 33 ( or 57.9 % ) contained one or more sequences related to the H-NS binding motif within a region spanning positions 300 to +100 bp relative to the ﬁrst nucleotide of the translation start site ( see Methods section ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>H-NS binds within the coding region of the bglG gene approximately ~ 600 -- 700 base pairs downstream of the transcription initiation site ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>In addition , H-NS binds to a site located 600 -- 700 bp downstream of the transcriptioninitiation site within the coding region of the ﬁrst gene , bglG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>In addition , H-NS binds within the coding region of the ﬁrst gene , approximately 600 -- 700 bp downstream of the transcription initiation site , where it induces a Rho-dependent repression , which is likely to be mediated by stalling transcription elongation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The first site , located in the upstream YncC box AATaTATCcA ( Fig. 3B ) , matches the consensus over 8 bp , whereas the other , AATaTATttt in the downstream YncC box ( Fig. 3B ) , has 6 bp matches and , therefore , is expected to have lower affinity ( note that the bases matching the H-NS consensus site are indicated in capital letters ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The 3 = end of the fragment to which H-NS binds preferentially is located 235 bp upstream of the nrdA transcriptional start site ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The extent of protection associated with the H-NS I site ( 38 bp ) suggests that more than one H-NS dimer is bound , and the formation of different subcomplexes could potentially act to modulate FNR / CRP-driven transcription activation either positively ( occupation of the downstream portion of H-NS I ; thick arrow in Fig. 6 ) or negatively ( occupation of sequences overlapping the FNR/CRP-binding site ; thin arrow in Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>As shown in Fig. 4B , fragments 2 and 4 were shifted very efﬁciently by H-NS ; fragment 2 encompasses the predicted bend region located between the divergent yjjP and yjjQ promoters , while fragment 4 was mapped to a position downstream of the yjjQ promoter ( from positions 7 to 193 relative to that of the yjjQ transcription start site , which was mapped to 158 bp upstream of yjjQ ) ; fragment 1 , which encompasses the yjjP promoter and sequences located downstream to it ( from position 80 to 163 relative to that of the yjjP transcription start at 113 bp upstream of yjjP ) was also shifted speciﬁcally ( Fig. 4B ) ; fragment 3 , which encompasses the yjjQ promoter was not shifted compared to that of the lacZ control fragment , which suggests that in this region no sites or only weak H-NS sites are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Furthermore , the expression level directed by a yjjP promoter-lacZ fusion that encompasses the complete intergenic region ( from position 450 to 182 relative to that of the transcription start site mapping to a position 113 bp upstream of yjjP ) increased from 33 units in the wildtype strain to 263 units and 211 units in the hns and the hns leuO double mutants , respectively ( Fig. 5B ) , demonstrating that in the complete context , H-NS represses the yjjP promoter eightfold and suggesting that H-NS also binds in between the yjjP promoter and the yjjP gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>pETS168 ( PnrdA-lacZ H-NS binding site ) was constructed by ampliﬁcation of the nrdA promoter region from nucleotide position 575 to 450 and from 400 to 127 using primers PnrdABamHI-up and PA-450ext-lw for the upstream region and PA-400extpho-up and PnrdASmaI-lw for the more proximal region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>PCR ampliﬁcation using the PnrdABamHI-up and PnrdASmaI-lw primers generated an nrdA promoter region band ( 652 bp ) without the putative H-NS binding site of approximately 50 bp located at a position centered at 425 nt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Recent reports estabiished that in ooriditions of low ostTiolarity H-NS inhibits proL/transcription directly through binding to a site downstream from the promoter ( Ueguchi and Mizuno , 1993 ; Lucht et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li>
<li>A major clyA transcript having its 5 * end ( 11 ) located 72 bp upstream of the translational start codon and 61 bp downstream of the CRP-FNR binding site was detected in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>A single CRP binding site is located 2115.5 and 2215.5 bp upstream of the divergent transcription start sites for the papI and papB genes , respectively , and CRP positively regulates expression of these genes by inhibiting binding of the repressing H-NS protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In contrast , site V ( -36 to -5 ; 32 bp ) is located within the ftnA promoter where H-NS binding is likely to inhibit RNA polymerase binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>In bgl , one of the H-NS binding sites is located upstream of the promoter , while the second site is located approximately 600 to 700 bp downstream of the transcription start site ( 13 ) ( Fig. 3 , top panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>At the top , a schematic view of the bgl locus is given , including the positions of H-NS binding regions upstream of the promoter , the catabolite regulator protein ( CRP ) binding site , and 600 to 700 bp downstream mapping within bglG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>SlyA binds at two separate sites at the hlyE promoter and inhibits H-NS binding Previous in vitro DNase I protection analysis with PhlyE identiﬁed a SlyA footprint from -70 to -30 relative to the hlyE transcription start site , reﬂecting the presence of two separate palindromic SlyA-recognition sequences in this upstream region , these being SlyA Ia ( -61 TTATCATAT TAA-50 ) and SlyA Ib ( -50 ATAGAAATAAAG-39 ) ( consensusmatching bases in bold ) ( Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Here , at the dps promoter , Fis binds to a site ( FisI ) in the spacer region between the -10 and -35 hexamers to trap Es70 at the promoter , while H-NS prevents promoter binding by Es70 ( see Fig. 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>In this system , H-NS binds to three sites , one directly over the RNA polymerase binding site and two upstream of the core region to block transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>In contrast to the H-NS I and H-NS II binding sites upstream and downstream of the transcript start site , there is no H-NS protection in the region of the GC-rich spacer between the -35 and -10 elements ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The protected region contains a 10-base-pair DNA element , centred at position +2.5 , with a 7/10 match to the recently identiﬁed consensus target for H-NS binding ( Bouffartigues et al. , 2007 ; Lang et al. , 2007 ) , with the three non-matching base pairs corresponding to the second preference at that position ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>The H-NS II site contains a hypersensitive base at position 17 that is separated by several unprotected bases ( 16 to 1 ) from another protected A-T-rich region ( 8 A-T/10 bp ) ( Fig. 4b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>In agreement with this , the intergenic region between the divergent cai and ﬁx operons revealed important regulatory features , namely , potential rec-ognition sequences for CRP and H-NS , but no binding site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>The intergenic region between the divergent cai and ® x operons contains potential recognition sequences for CRP and H-NS , but no binding site for FNR ( Eichler et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>Regions protected by H-NS and SlyA in footprint assays ( H-NS I , H-NS II , SlyA I and SlyA II ) , and the FNR/CRP binding site described by Westermark et al. ( 2000 ) are indicated by lines above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Concomitant with the increase in RcdA concentration , its binding sequence expanded from the initial binding site so as to overlap the binding sequence by the universal silencer H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>As a control , we randomly picked 99 regions of 500 bp in length and asked what fraction contained a high-afinity H-NS binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>The initial binding site I , occupied between 0.4 and 0.6 mM H-NS , spans ~ 50 nucleotides ( positions -180 to -240 , relative to the ﬁrst nucleotide of the CasA start codon ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>The deletion analysis of this fragments shows that the putative H-NS binding site of approximately 50 bp is centered at position 425 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The proV-lac2 fusion A574-BstNI comprises proV sequences from -54 to +936 bp in proV and thus carries the complete extended H-NS binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>The H-NS-I binding site extended outwards from a core region located between positions 187 and 208 upon increasing H-NS concentration and extended 5 at least as far as 285 and thus overlapped the SlyA-I binding site ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>Fourth , when bound to preferred sites at which it exerts a biological effect , regions of DNA protected by H-NS in footprinting studies are extended and diffuse [ i 5 , 40 , 48 , 49 [ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>By including 3 the additional putative H-NS motifs identiﬁed in our sequence screen , we reﬁned the logo for H-NS binding ( Figure 1 ) and observed little change from the input motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Thus , we suggest that the H-NS bound to the upstream site would laterally extend to the downstream sequence 70 joined by DNA wrapping around E that seals off the loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>We propose here that H-NS bound to the AT tract centered at − 118 nt ( nucleation site ) laterally extends by cooperatively recruiting H-NS molecules to the downstream sequence joined through DNA wrapping around E 70 , which results in trapping of E 70 in a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Hence , the gene silencer AT8 ( an H-NS nucleation site ) serves as a crucial driving force for triggering the transcription regulatory effect , but the rest of the DNA sequence in the region determines the distance and the directionality of the effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>H-NS shows the typical extended protection with no sharp borders , centring within the region of the strongest DNA curvature , extending into the core promoter sequence and covering a stretch of nucleotides downstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>In addition to th extended H-NS binding regionw a , e observed number of additional protected sites that vary insize from 9 to 24 bp ( Fig. 5 ) and encompass a weakly protected region centered around the proU -35 sequence ( from-22 to -39 bp ) ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>This is illustrated by the more than 20-fold higher concentration of H-NS required to protect the G + C-rich region around theproU -35 sequence ( -22 to -39 bp ) in comparison to the highly A + T-rich extended H-NS binding region at the beginning of the proV gene ( +64 to + lo9 bp ) ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>We aligned the DNA sequences of the H-NS bindingsites from theproU regulatory region ( Fig. 6 ) to look for common sequence determinants , but found no obvious consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>This sequence motif often matches theH-NS footprints observed in the proU regulatory region ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Furthermore , using the nrdAB operon as a model , we demonstrated the speciﬁc interaction of H-NS with two DNA fragments of the operon , one of which maps upstream of the promoter and the other of which is located within the nrdA coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS can repress cydAB expression from an upstream sequence element H-NS protects DNA regions located both upstream and overlapping the cydAB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>These data are consistent with models of transcriptional silencing in which the curved DNA sequence to which H-NS initially binds is located upstream or downstream of the promoter that is to be repressed81 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>H-NS was found to bind preferentially to a curved DNA sequence upstream of the hns gene in E coil and furthermore this curved region was shown to be necessary for autoregulation of H-NS [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The H-NS protein has been shown to bind preferentially to intrinsically curved DNA ( 38 ) , for which the sequence characteristic is phased A tracts of 4 to 6 bp ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>DISCUSSION By combining DNA footprinting data from different sources , we have identiﬁed a DNA sequence that correlates with H-NS binding at speciﬁc locations in the E. coli genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>The nucleotide sequence in the possible upstream regulatory region exhi-bits short homopolymeric dA and dT stretches , which can act as curved DNA sequences binding to the E. coli histone-like protein H-NS ( Tanaka et ai , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7815937" target="_blank">7815937</a>]<br></li>
<li>The apparent promiscuity of H-NS probably arises from the fact that it does not have a consensus sequence for DNA binding , but instead has a consensus structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The Cnu-H-NS protein complex binds to a specific sequence at the origin of chromosomal DNA replication in Escherichia coli [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>H-NS , a non-specific DNA binding protein , binds to a specific sequence of oriC once complexed with Cnu [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>Although there is ample evidence that H-NS binds preferentially to DNA containing curved regions , we show here that a major basis for this selectivity is the presence of a conserved sequence motif in H-NS target transcriptons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>H-NS binds to no specific target sequence in DNA but instead recognizes intrinsic DNA curva ¬ 3 The abbreviations used are : PR1 , region 1 promoter ; X-gal , 5-bromo-4-chloro-3-indolyl - - D-galactopyranoside ; EMSA , electrophoretic mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>We tested whether H-NS binds to this bent DNA sequence of lrp by DNA mobility shift experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>These results indicated that H-NS binds specifically to a bent DNA region of lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Rather than binding directly to specific DNA sequences in the hly operon , Hha exhibits nonspecific DNA binding ; therefore , it was proposed Hha binds to H-NS , which binds to specific regions of regulatory sequences in the hly operon [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>H-NS favors binding to DNA sequences that have the propensity to bend ( 101 ) , but there is no deﬁnitive DNA sequence that is recognized by H-NS , and it binds with considerably less speciﬁcity to rRNA promoters than expected for a classical repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15568992" target="_blank">15568992</a>]<br></li>
<li>LRP and H-NS bind to the same DNA sequences , yet they differ in binding mechanisms In comparison with H-NS , which also binds to all seven rRNA P1 promoter upstream regions ( Hillebrand et al. , 2005 ) , LRP complexes were formed at notably lower ( 2-to 20-fold ) protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>H-NS preferentially binds to two DNA regions , one of which includes sequences that map upstream of the known binding sites for the transcription factors that regulate the expression of nrdA ( FIS and IciA ) ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>( A ) Logo representation of the experimentally determined H-NS binding motif derived from sequences in Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>We tested severarlestriction fragments that carry theextended H-NS binding site in proV and different lengths of proU upstream sequences in a two-dimensional gel electrophoresis assay that can resolve DNA fragments according to their size and conformation ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>We therefore wondered whether theextended H-NS binding site at the 5 ' end of the E. coli proV gene also consists of curved DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Based on this definition , an H-NS nucleation site is distinct from other ordinary H-NS-preferred binding DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>Other ordinary H-NS-binding DNA sequences may lead to the binding of H-NS , but in the absence of an H-NS nucleation site , the bound H-NS is not necessarily well organized and hence is not transcriptionally repressive ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>0002-8080 G 2005 SGM Printed in Great Britain Silencing involves two sites ( Dole et al. , 2004b ) : H-NS represses transcription initiation at the cAMP receptor protein ( CRP ) - dependent promoter by binding to an AT-rich and presumably bent upstream silencer sequence ( Schnetz , 1995 ; Singh et al. , 1995 ; Schnetz & Wang , 1996 ; Mukerji & Mahadevan , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>A Cis-spreading Nucleoprotein Filament Is Responsible for the Bacterial Gene Silencing -- In addition to the nucleation site , the rest of the AT-rich DNA sequence in S. typhimurium or E. coli LCR-I appears to also possess a certain binding preference for H-NS because , with increasing H-NS concentrations , some of this AT-rich DNA was occupied by H-NS as well ( Figs. 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15582999" target="_blank">15582999</a>]<br></li>
<li>This sequence element is similarly AT-rich as the CRISPR leader regions and according to ChIP-chip analysis harbours H-NS sites in addition to RNA polymerase binding sites ( Oshima et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>However , by combining recent ﬁndings we derive here an AT-rich sequence motif for directing sequence-speciﬁc DNA binding by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>This sequence is highly A/T-rich and is therefore consistent with other reported binding targets for H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>This sequence is AT rich ( 68.4 % ) , a feature typically found in other well-character-ized H-NS binding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS binds to the AT-rich sequence within the 200 bp sequence upstream from the csgD promoter , and interferes with the binding of RNA polymerase ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>H-NS binds to DNA with no obvious sequence specificity , but specific binding sites tend to be AT-rich and intrinsically bend ( 13 -- 16 ) , as also observed for the upstream silencing region of the bgl operon ( 6 , 8 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>Once H-NS is recruited to the local site ( the AT-rich DNA sequence element , AT4 ) , the binding cooperativity of H-NS may cause a cis-spreading ( oligomerization ) of H-NS to the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121424" target="_blank">11121424</a>]<br></li>
<li>H-NS binds to upstream pnir sequences and represses FNR-dependent transcription We noted that the pnir upstream region contains a number of AT-rich tracts of sequence that are phased such that this region might be intrinsically curved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The DNA sequences of both fragments are AT rich , and in addition , the upstream fragment includes DNA sequences identical to the H-NS consensus sequence region described previously ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Interestingly , in the presence of LeuO the extended protection by H-NS is no longer visible ( Fig. 3B , compare lanes 2 and 4 ) , indicating that due to LeuO binding the DNA region containing the H-NS high-affinity binding site is no longer protected from DNase I cleavage , in agreement with decreased cooperative binding ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>The results of the binding experiments allow the foiiowing conclusions ; ( i ) Fis and H-NS binding sites probably overlap at sequence position - 75 ( Fis site I ) , and ( ii ) the UAS sequence , centred around - 90 , has a strong effect on gel mobility ( k-factors in Fig , 3A ) and Fis binding , while the H-NS target region , downstream from position - 9 0 , does not contribute very much to the intrinsic curvature , but binding of H-NS to this site can alter the conformation drastically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>As for the pyciGECO , two predicted H-NS binding sites ( agaATAtATT centered at -- 98.5 and AtaTTATCTc centered at -- 93.5 ) in pyciGSTM overlap the single YncC binding site ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>On the other hand , H-NS binds at a long sequence , possibly including four sites , covering almost the entire csgD promoter from 2201 to +28 ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In the promoter upstream region , centered at approximately -- 150 , there is an intrinsic DNA bend involved in the transcriptional auto-repression operated by H-NS which binds to three sites , two located on each side of the bend and the third partly overlapping the -- 35 box [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>It is suggested that H-NS occupation of the ftnA promoter depends upon the presence of H-NS bound at the upstream sites through an H-NS oligomerization and H-NS -- DNA looping mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>H-NS binds at multiple sites within , and upstream of , the ftnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>H-NS binds at multiple sites within the ftnA promoter region At 1 mM H-NS , protection of the ftnA promoter fragment was detected at six H-NS binding sites ( I -- VI ) ( Fig. 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Consequently , Fur displacement of H-NS from the upstream sites prevents cooperative H-NS binding at the downstream sites within the promoter , thus allowing Accepted 12 November , 2009 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Del-etion of sequences upstream of the micF promoter derepresses micF expression .58 These sequences could represent H-NS binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Schematic diagram of the H-NS and FIs binding sites at the upstream rRNA PI promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Blue squares represent high-afinity H-NS binding sequences ( nucleating sites ) and green and yellow circles represent bound H-NS on nominal distal and proximal sides , respectively , of a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Clustering of characterized H-NS - , IHF - and Fis-binding sites [ ± 100 bp ; ( 18 ) ] located within coding and noncoding sequences was not detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>H-NS binding to two target sequences located at a distance apart has been reported for most H-NS-modulated operons ( 17 , 42 ) , and it is not uncommon that one of the binding sites maps downstream of the promoter , as is the case for the bgl operon ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Both proteins , H-NS and LRP , bind to the same region of the rrnB P1 promoter upstream sequence First we performed a standard DNA binding assay with a 260 bp radiolabelled DNA fragment , containing the complete rrnB P1 UAS region and the core promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>According to the 10-bp consensus binding sequence reported for H-NS nucleation sites ( 44 ) , two predicted H-NS binding sites of high affinity are located in the upstream promoter region , precisely in the motifs recognized by YncC/McbR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>First , sequence analysis predicts a bend in the promoter region of both gad genes , which is a prerequisite for H-NS binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>( C ) Putative H-NS 50 bp binding sequence in the nrdAB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Furthermore , scanning of the DNA sequence for putative H-NS nucleation sites , using the H-NS consensus ( 28 ) , revealed several putative matches which cluster in the predicted bend region and downstream of the yjjQ promoter ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The function of H-NS at the csiD promoter would be to counteract this ﬂexibility and thereby improve the precision of activation from the optimal CRP box position , maybe by binding to several sites along this region as a ` local DNA scaffold ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>We propose that , like most preferred binding sites for H-NS , the cydAB promoter region contains one or more stretches of bent DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>We next performed ﬁne mapping of the putative H-NS binding sites in the PnrdA operon by using several overlapping DNA fragments from the promoter region and from the nrdA coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Alternatively , it has been suggested that H-NS-mediated action requires first the building up of a substructure resulting from the binding of H-NS at a curved sequence ( the nucleation step ) and then recruitment by a cooperative process of H-NS mol-ecules bound at other strategic sites , in particular at the − 10 region of a promoter for transcription repression ( Badaut et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>Binding of LeuO and H-NS to the DNA sequence upstream of casA The casA-cas3 intergenic region ( here denoted IGLB ) contains Pcas , for which H-NS has strong binding affinity as well as the divergently oriented anti-cas3 ( known as anti-Pcas ) promoter , that is located 80 bp upstream of Pcas and gives rise to an antisense transcript of unknown function ( Figs 1A and 3C ) ( Pul et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>Analysis of the chiA promoter sequence using the DNA structure prediction program ` BEND.IT ' ( ICGEB , Trieste , Italy ; Munteanu et al. , 1998 ) revealed the presence of a curved region that could correspond to the H-NS binding site ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>No consensus binding site for the CRP protein could be detected in the upstream region of sfm , conﬁrming that CRP probably does not bind to the promoter region of sfm ( Grainger et al. , 2005 ) and does not directly compete with H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Because the derived motif depended on the assumption that high-afinity binding sites are related in sequence we used the motif to predict three potential H-NS binding sites in the ﬁs promoter from positions 150 to +1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>These predicted sites , although imperfect ﬁts to the derived motif , conﬁrmed a footprinting analysis of this promoter ( Supplementary Data ) and thus provide a validation of the derived sequence template for H-NS binding in vitro ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>The binding sites of OmpR , RstA , CpxR , IHF and H-NS were identified by DNase I footprinting assays ( for gel patterns see Fig. 5 ) and are shown along the csgD promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The location of binding sites for LeuO and H-NS deduced from the yjjP promoter-lacZ fusions are summarized schematically in Fig. 5B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Binding of H-NS and LRP to the rrnB UAS occurs in a synergistic fashion The observation that both proteins , LRP and H-NS , are able to bind individually to the same rrn P1 promoter upstream DNA elements immediately raises the question whether or not the two proteins recognize the same binding sites and potentially compete each other , or if they were able to bind simultaneously , possibly at different faces of the DNA helix , or even stimulate the binding of each other in a cooperative manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>[ 15 ] D. Tippner , R. Wagner , Fluorescence analysis of the Escherichia coli transcription regulator H-NS reveals two distinguishable complexes dependent on binding to speciﬁc or nonspeciﬁc sites , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12504278" target="_blank">12504278</a>]<br></li>
<li>B. Determination of the H-NS activity to bind to speciﬁc DNA target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Figure 5 Determination of binding sites for H-NS , StpA , LRP and FIS by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>Analysis of the LRP and H-NS binding sites by DNase I footprinting To identify the region ( s ) at which LRP and H-NS bind to the rrn UAS DNA , we performed a preliminary DNase I footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>In contrast , H-NS-I119T binds DNA and forms higher-order assemblies but does not speciﬁcally recognize H-NS binding sites ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Competitive EMSA analyses indicate that despite their overlapping binding sites , SlyA and H-NS were able to bind to this promoter simultaneously , forming higher order SlyA DNA H-NS complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>High-affinity DNA binding sites for H-NS provide a molecular basis for selective silencing within proteobacterial genomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>White or shaded boxes indicate binding sites for Fis and H-NS , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>By binding to regulatory sites , Fis and H-NS change DNA topology and bending w96 ,97 x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The footprinting analysis shows that binding sites for SlyA overlap the AT-rich H-NS-binding sequences and that SlyA affects the ability of H-NS to bind to PhlyE , probably by binding ﬁrst to the higher-affinity SlyA II site to form an intermediate complex SlyA : H-NS : PhlyE in which the H-NS dimer interactions bridging H-NS II and H-NS I are disrupted ( Fig. 7 , C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The H-NS protected sequences overlap the Fnr and ArcA binding sites and the P1 , P2 , P3 and P4 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>The arrangement of binding sites suggests that H-NS forms an extended repressing nucleoprotein complex that prevents transcription initiation at both divergent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The locations of these putative sites are in agreement with speciﬁc binding of H-NS to fragments 2 and 4 , encompassing these two regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>These sites overlap AT-rich H-NS binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Indeed , both regions contain A/T-rich segments proximal to the 335 regions of their promoters which could serve as binding sites for H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>The average AT content of the high-conﬁdence H-NS bound sites was 61.4 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>The positions of the binding sites for the catabolite gene activator protein ( CAP ) , the osmoregulator OmpR and the H-NS protein ( black lines below sequence ) are taken from Shin and Park ( 1995 ) and Soutourina et al. ( 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>These include a CAP ( ` catabolite gene activator protein ' ) binding site centred around -- 71.5 ( class I site ) , two OmpR ( osmoregulation ) binding sites at positions -- 100 to -- 163 and +1 to +66 , respectively , and multiple binding sites for the H-NS protein from position -- 178 up to the coding region ( Shin and Park , 1995 ; Soutourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>Further investigation showed that the PhlyE contained overlapping high-affinity binding sites for both H-NS and SlyA ( Wyborn et al. , 2004b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>While a degenerated DNA consensus sequence has been characterized for LRP -- DNA interaction ( Brinkman et al. , 2003 ) , the view of a conserved DNA sequence for H-NS binding has been abandoned , although a high content of A : T base pairs is consistently found within H-NS binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>These sites include a majority of sequences containing the sites ATAAA ( 56 % ) and AATAA ( 68 % ) , which form part of the highafinity H-NS binding motifs identified by computational analysis by Lang et al. 83 and Kahramanoglou et al. 12 respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>The sizes of the H-NS binding sites on the ftnA promoter vary from 10 to 35 bp which resembles the protection pattern of H-NS within other promoters , e.g. proU , ﬂhDC and synthetic promoters ( Lucht et al. , 1994 ; Soutourina et al. , 1999 ; Rimsky et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The H-NS binding sites in the ftnA promoter are AT-rich with all sites containing > 65 % AT content and displaying an average AT content of 80 % , much greater than the genomic average of ~ 50 % and consistent with the marked preference of H-NS for AT-rich sequences ( Lucht et al. , 1994 ; Atlung and Ingmer , 1997 ; Grainger et al. , 2006 ; Lucchini et al. , 2006 ; Lang et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Indicated in the ﬁgure are : i ) the location of the four direct repeats ( identiﬁed by numbered arrows ) [ 6 ] ; ii ) the center of the intrinsic DNA bend ( ) identiﬁed by permutation experiments [ 9 ] ; iii ) the three H-NS binding sites ( striped boxes ) [ 9 ] ; iv ) the seven FIS binding sites [ 10 ] identiﬁed by footprinting ; and v ) the location of the presumed PhoB box [ 7 ] , whose sequence is given in the text ( thick frame ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>For determination of the number of H-NS sites within the minimal PhlyE sequence from pGS1065 , further gel shift and footprinting assays were undertaken .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by H-NS</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>- Galactosidase Ratio of activity Expression 37 ° C 20 ° C 37 ° C / 20 ° C Strain/WT Miller units P90C pDC6 1992.4 46.7 1112.1 22.2 P90CslyA : : kan 1055.7 14.2 543.9 19.0 pDC6 P90Chns : : kan 2136.4 19.8 1221.6 11.7 pDC6 regulated , is positively autoregulated , and is independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_2">
Antagonist involvement of FIS and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></label>
<input id="tu_similarity_2", type="checkbox">
<div class="TU_similar">
<li>Antagonistic involvement of FIS and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885445" target="_blank">16885445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li></div>
<li>Falconi , M. , Brandl , A. , La Teana , A. , Gualerzi , C.O. , and Pon , C.L. ( 1996 ) Antagonistic involvement of FIS and H-NS protein in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_5">
[ 4 ] Falconi M. , Brandi A. , La Taena A. , Gualerzi C.O. , Pon C.L. , Antagonistic involvement of FIS and H-NS proteins in the transcriptional control of hns expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></label>
<input id="tu_similarity_5", type="checkbox">
<div class="TU_similar">
<li>Falconi , M. , Brandi , A. , La Teana , A. , Gualerzi , C.O. , and Pon , C.L. ( 1996 ) Antagonistic involvement of Fis and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9680219" target="_blank">9680219</a>]<br></li>
<li>Falconi , M. , Brandi , A. , La Teana , A. , Gualerzi , C.O. , and Pon , C.L. ( 1996 ) Antagonistic involvement of FIS and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Falconi , M. , Brandi , A. , La Teana , A. , Gualerzi , C.O. and Pon , C.L. ( 1996 ) Antagonistic involvement of FIS and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li></div>
<li>Transcriptional monocopy lacZ fusion studies showed that both operons are coexpressed during anaerobic growth in the presence of carnitine , respond to common environmental stimuli ( like glucose and nitrate ) , and are modulated positively by the same general regulators , CRP and FNR , and negatively by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>In E. coli , operons caiTABCDE and fixABCX are modulated positively by general regulators , such as the AMPc receptor protein ( CRP ) or the transcriptional regulator responsible for anaerobic induction ( FNR ) , and negatively by the DNA-binding protein H-NS , glucose , or nitrate ( Eichler et al. , 1994a ; Unden and Trageser 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>This difference in regulation of bgl versus stpA and proU by H-NS-I119T and StpA may further imply that growth conditions which induce stpA may allow the maintenance of strong repression of bgl , even if H-NS is less active under such conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>1990 ) , the product of the bolA morphogene ( Aldea etai , 1990 ; Lange and Hengge-Aronis , 1990b ) , proteins stimulating glycogen synthesis ( Lange and Hengge-Aronis , 1991a ) , the nucleoid protein H-NS ( VanBogelen etai , 1990 ; Spaasky etai , 1984 ) , the putative starvation-inducible sigma factor encoded by the katF gene ( Mulvey and Loewen , 1989 ; Lange and Hengge-Aronis , 1991a ) , and the GroEL , GroES , DnaK , GrpE , HtpH , and HtpM proteins of the heat-shock response ( VanBogelen ef al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>Finally , proU synthesis is derepressed by an osmotic upshift in part by release of the abundant nucleoid-associated protein H-NS from an extended regulatory sequence located downstream from its promoter ( Lucht and Bremer 1994 ; Gowrishankar and Manna 1996 ; Jordi et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601034" target="_blank">10601034</a>]<br></li>
<li>This conﬁrms that H-NS will repress gadA and gadBC ; however , this control did not require gadX as reported for the W3110 strain and was not even evident in a gadX culture ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>A recent report has dem-onstrated that H-NS may work to repress the gadA and gadBC genes by repressing gadX ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>As shown in Fig. 4B , fragments 2 and 4 were shifted very efﬁciently by H-NS ; fragment 2 encompasses the predicted bend region located between the divergent yjjP and yjjQ promoters , while fragment 4 was mapped to a position downstream of the yjjQ promoter ( from positions 7 to 193 relative to that of the yjjQ transcription start site , which was mapped to 158 bp upstream of yjjQ ) ; fragment 1 , which encompasses the yjjP promoter and sequences located downstream to it ( from position 80 to 163 relative to that of the yjjP transcription start at 113 bp upstream of yjjP ) was also shifted speciﬁcally ( Fig. 4B ) ; fragment 3 , which encompasses the yjjQ promoter was not shifted compared to that of the lacZ control fragment , which suggests that in this region no sites or only weak H-NS sites are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Furthermore , the expression level directed by a yjjP promoter-lacZ fusion that encompasses the complete intergenic region ( from position 450 to 182 relative to that of the transcription start site mapping to a position 113 bp upstream of yjjP ) increased from 33 units in the wildtype strain to 263 units and 211 units in the hns and the hns leuO double mutants , respectively ( Fig. 5B ) , demonstrating that in the complete context , H-NS represses the yjjP promoter eightfold and suggesting that H-NS also binds in between the yjjP promoter and the yjjP gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>To analyze whether downstream sequences alone confer regulation by H-NS and/or StpA , the stpA coding region [ stpA ( orf ) ] ( including positions 1 to 466 relative to the transcription start ) was inserted between the constitutive lacUV5 promoter and the lacZ gene ( Fig. 1E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>Possibly , LeuO has a dual role in the regulation of the yjjP promoter ; it may repress the promoter directly and also counteract its repression by H-NS when the binding site downstream of the yjjP promoter is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>It is also possible that H-NS traps RNA polymerase at the yjjQ promoter by binding upstream and downstream and that LeuO prevents the formation of that repressing DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>It is possible , and not unprecedented , that H-NS binds downstream of the transcriptional start site and still affects transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Translation of bglG reduces the H-NS mediated repression via the downstream silencer It is possible that repression by H-NS via the downstream silencer affects transcription initiation at any promoter ( bgl and lacUV5 were tested ) located upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_23">
Repression by H-NS at the downstream silencer requires termination factor Rho and is reduced by translation of the bglG mRNA , but is independent of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></label>
<input id="tu_similarity_23", type="checkbox">
<div class="TU_similar">
<li>Repression by H-NS via the downstream silencer requires termination factor Rho , is enhanced when translation of the bglG gene is prevented , and is promoter independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li></div>
<li>H-NS binding to two target sequences located at a distance apart has been reported for most H-NS-modulated operons ( 17 , 42 ) , and it is not uncommon that one of the binding sites maps downstream of the promoter , as is the case for the bgl operon ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>H-NS mediated repression via the downstream silencer.The inﬂuence of the downstream silencer was analysed by its insertion in between the constitutive lacUV5 promoter ( UV5 ) and the lacZ gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The repression of the bgl operon by H-NS was analysed using chromosomally encoded bgl-lacZ fusion , which carry the bgl promoter including the upstream and the downstream silencer and its deletion derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Discussion The approximately 100-fold repression of the bgl operon by H-NS requires an AT-rich and presumably bent silencer sequence located directly upstream of the CRP-depen-dent bgl promoter , and a downstream silencer located within the coding region of the ﬁrst gene , bglG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The experimental data reported here are consistent with gene silencing models in which the curved DNA to which H-NS initially binds is located upstream or downstream of the promoter that is to be repressed ( Rimsky , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>The protection by H-NS at site I was lost , as was protection at sites V and VI , which are located on and just downstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Although it is possible that there are other promoters much further upstream which read through into the stpA region but which were not detected in this experiment due to inefﬁcient extension over long distances , it seems likely that the transcriptional start at 241 is the primary stpA promoter in vivo and is strongly repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>It should be noted that our data do not allow us to discern whether H-NS remains bound to the cydAB promoter region in microaerobiosis or whether it still has a repressing role under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>This suggests that in vivo , the most important role played by GadX ( GadW ) on the gadA promoter is as an H-NS countersilencer , not as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>Homers AM , van der Zeijst BAM , Gaastra W. Uh!in BE 11992 ) The positive regulator Clad overcomes the repression mediated by histone-like protein H-NS -LCB- HI ) in the CFA/i fimbrial operon of Escherichia colt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Regulation of the yjjQ-bglJ Operon , Encoding LuxR-Type Transcription Factors , and the Divergent yjjP Gene by H-NS and LeuO † Thomas Stratmann , S. Madhusudan , and Karin Schnetz * Institut für Genetik , Universität zu Köln , Zülpicher Str .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The divergent arrangement of transcription units at the yjjP-yjjQ-bglJ locus raises the possibility that transcription-induced supercoiling ( 34 ) plays a role in its antagonistic regulation by H-NS and LeuO , which again would relate to autoregulation of the leuO gene ( 20 , 21 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The yjjQ and bglJ genes encoding LuxR-type transcription factors form an operon whose repression by H-NS is counteracted by LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_38">
Regulation of the yjjQ-bglJ operon , encoding LuxR-type transcription factors , and the divergent yjjP gene by H-NS and LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></label>
<input id="tu_similarity_38", type="checkbox">
<div class="TU_similar">
<li>Stratmann , T. , Madhusudan , S. , and Schnetz , K. ( 2007 ) Regulation of the yjjQ-bglJ operon encoding LuxR-type transcription factors and the divergent yjjP gene by H-NS and LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>Stratmann , T. , Madhusudan , S. , and Schnetz , K. ( 2008 ) Regulation of the yjjQ-bglJ operon , encoding LuxR-type transcription factors , and the divergent yjjP gene by H-NS and LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843219" target="_blank">19843219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Stratmann , T. , Madhusudan , S. and Schnetz , K. ( 2008 ) Regulation of the yjjQ-bglJ operon , encoding LuxR-type transcription factors , and the divergent yjjP gene by H-NS and LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24335284" target="_blank">24335284</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li></div>
<li>However , while the relevance of LeuO and YjjQ for pathogenicity indicates that their genes are expressed under certain in vivo conditions in the host environment , both the leuO gene and the yjjQ-bglJ operon are repressed by H-NS under laboratory growth conditions ( 6 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>However , expression of leuO and bglJ is repressed by H-NS , at least under laboratory growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>YjjQ was provided plasmidically , as expression of the native yjjQ-bglJ operon is repressed by H-NS during exponential growth under laboratory conditions ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Furthermore , the data support the conclusion that LeuO , whose expression is likewise repressed by H-NS and which is a virulence factor in Salmonella enterica , is a master regulator that among other loci , also controls the yjjQ-bglJ operon and thus indirectly the presumptive targets of YjjQ and BglJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Here we show that yjjQ and bglJ form an operon carried by E. coli K-12 , whose expression is repressed by the histone-like nucleoid structuring ( H-NS ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The results show , furthermore , that H-NS represses the expression of this yjjQ-bglJ operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Transcription of the yjjQ-bglJ operon is repressed by the global repressor H-NS ( heat-stable nucleoid structuring protein ) , and it is activated by the pleiotropic transcriptional regulator LeuO ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Expression of the yjjQ gene , which is encoded in the yjjQ-bglJ operon , is repressed by the global repressor H-NS and activated by the pleiotropic transcriptional regulator LeuO , as shown for the laboratory E. coli K-12 strain ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>A further parallel is that both the rcsA gene and the bglJ gene ( within the yjjQ-bglJ operon ) are repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>Furthermore , the yjjP gene , encoding a membrane protein of unknown function and located upstream in divergent orientation to the yjjQ-bglJ operon , is likewise repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The yjjP gene located upstream of the yjjQ-bglJ operon and in a divergent orientation is likewise repressed by H-NS but is not activated by LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Taken together , the data conﬁrm the regulation of the yjjQ-bglJ operon by H-NS and LeuO , and they demonstrate the repression of yjjP by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Here we show that yjjQ and bglJ are expressed as an operon , which is repressed by H-NS and activated by LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>RESULTS H-NS represses and LeuO activates the yjjQ-bglJ operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Regulation of the yjjQ-bglJ operon by H-NS and LeuO adds an additional aspect to the role of LeuO in the emerging regulatory network important for in vivo induction of genes and for pathogenicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>However , regulation of the yjjQ-bglJ operon by H-NS and LeuO indicates that these genes are related to the control of stress responses and/or virulence-associ-ated traits ( 15 , 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>Further , it was proposed that LeuO acts as a positive regulator of the leu operon and leuO gene with activation based on transcription induced changes in DNA supercoiling and delimiting of H-NS spreading by LeuO ( Fang and Wu , 1998b ; Chen et al. , 2003 ; Chen and Wu , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>For Regulation of rRNA transcription by H-NS and LRP 873 instance , H-NS is involved in the LRP-dependent activation of the ilvIH operon ( Levinthal et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Similarly , the bgl operon , encoding the gene products necessary for the uptake and fermentation of aryl-b , D-glucosides is repressed ~ 100-fold by H-NS ( Higgins et al. , 1988 ; Lopilato and Wright , 1990 ; Schnetz , 1995 ; Caramel and Schnetz , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Results Repression of the bgl operon by H-NS H-NS represses the wild-type bgl operon encoding the positive regulator BglG , the b-glucoside speciﬁc permease BglF ( or EIIBgl ) , and the phospho-b , D-glucosidase BglB ( Fig. 1 ) ( Defez and de Felice , 1981 ; Schnetz et al. , 1987 ; Higgins et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>A negative regulatory element ( NRE ) is located within the ﬁrst gene of the proU operon and constitutes a speciﬁc binding site for H-NS ( 13 , 34 , 41 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626294" target="_blank">8626294</a>]<br></li>
<li>The proU operon , encoding an uptake system for the osmoprotectants glycine-betaine and proline , is repressed very speciﬁcally ( ~ 70-fold ) by H-NS in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>The second , downstream level of bgl operon repression by H-NS is not affected by LeuO , BglJ or RpoS/Crl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>The E. coli bgl operon is one example of a system whose repression by H-NS is exceptionally speciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>The bgl ( aryl - , D-glucoside ) operon of E. coli is a classical example of a locus that is tightly repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>In addition , the pap operon is controlled by cAMP-CAP and also regulated by H-NS [ 301 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Hence , it is not surprising that H-NS does not play a role in the temperature-dependent regulation of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>operon was expressed in cells depleted of both HU and H-NS as well as in cells depleted of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>and TP2600-1 ( H-NS '' , CJ ® - ) ( Fig. 7A ) all transcribed the csgBA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Furthermore , we demonstrate that an additionai operon , hdeAB , which is also dependent upon a ^ for transcription , can be transcribed by Ed ™ in vivo in the absence of IH-NS by utilizing the phdeAB promo-ter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Another common feature shared by the gsp and chiA genes and the bgl operon is the fact that their transcription is silenced by H-NS ( Defez and De Felice , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>E. coli does not normally ferment - glucosides such as salicin because of silencing by H-NS and transcription termination within the bgl operon ( Bglo cryptic phenotype ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576773" target="_blank">15576773</a>]<br></li>
<li>Later , it was found that repression of the bgl operon by H-NS can be relieved by the LysRtype transcription factor LeuO ( see below ) and by BglJ ( 25 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>S1 ) , indicating that transcription of the cas-ABCDE12 operon is tightly controlled by H-NS and LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>In an hns1 strain in the absence of Na1 , nhaA was repressed whether or not nhaR was present ( Fig. 2 and 3 ) ( 39 ) , suggesting that the repression by H-NS is NhaR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Thus , we conclude that IHF and Fis can repress pnir in the absence of H-NS ( Table 2 ) , and that repression by H-NS can not be reversed by the action of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Transcription of nhaA , the Main Na1/H1 Escherichia coli , Is Regulated b and Growth Phase Antiporter of y Na1 NIR DOVER AND ETANA PADAN * Division of Microbial and Molecular Ecology , Institute of Life Sciences , The Hebrew University of Jerusalem , 91904 Jerusalem , Israel Received 7 August 2000/Accepted 19 October 2000 The transcription of nhaA , encoding the main Na1/H1 antiporter of Escherichia coli , is induced by Na1 , regulated by NhaR , and affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Na - induced transcription of nhaA , which encodes an Na / H anti-porter in Escherichia coli , is positively regulated by NaR and affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867437" target="_blank">12867437</a>]<br></li>
<li>Consistent with this assignment , a number of genes transcribed by EsS in stationary phase are silenced by H-NS in the exponential phase ( Barth et al. , 1995 ) and hchA was recently identiﬁed as a sS-dependent gene in two independent microarray studies ( Patten et al. , 2004 ; Weber et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>A common denominator for the genes known to be regulated by H-NS ( e.g. regulation of genes at stationary growth phase , starvation , changes in high osmolarity , low temperature shock , and altered DNA topology ) seems to be their involvement in different kinds of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>Such an observation is consistent with the function of H-NS at osmotic shock or the regulation of genes expressed under stationary phase ( Higgins etai , 1988 ; Hengge-Aronis , 1993 ) , Thus H-NS may link rRNA expression to situations of rapid osmotic changes or reduced growth at stationary phase or even be effective under conditions of cold shock ( La Teana etai , 1991 ) , Obviously , acellular process as central as the regulation of rRNA synthesis must be controlled by many different parameters and sensors to be as effective and efficient as observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>H-NS has been referred to as a general transcription silencer , since it represses a large number of genes , which are not required during the exponential phase of growth under normal laboratory conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>The major nucleoid proteins , HU , IHF , H-NS and Fis , in exponential growth phase E. coli under aerobic conditions are all known to have functional dichotomy ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response to oxidative stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Expression of 19K , 17K , IOK-L , and IOK-S is under the control of H-NS at the transcriptional level To determine whether H-NS affects transcription of the genes coding for 19K , 17K , 10K-L , and 10K-S , cells harboring the set of promoter-lacZ fusion genes were grown until the mid-logarithmic and stationary phases , and then the / 3-galactosidase activity expressed by the cells was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Expression was induced to nearly the same extent in both strains , ﬁvefold in the wild type and fourfold in the mutant , suggesting that H-NS does not play a major role in stationary-phase induction of appY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Thus , the silencing of csgD by H-NS should be reduced in stationary phase because of the increase in IHF level ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The present work revealed that the P2 promoter of nhaA is a stationary phase-induced promoter which is activated by sS and neither NhaR nor Na1 or H-NS is involved in this induction ; in the stationary phase , transcription from P2 increased dramatically , and this increase was eliminated in an rpoS mutant strain ( Fig. 6A ) but not in a DnhaR strain ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Both Fis and H-NS selectively repress the dps promoter , preventing transcription initiation by RNA polymerase containing s70 , the housekeeping s factor , but not by RNA polymerase containing s38 , the stationary-phase s factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Interestingly , the three genes rcsA , gadE and bglJ , respectively , encoding the three RcsB heterodimerization partners are all repressed by H-NS ( Majdalani and Gottesman , 2005 ; Stratmann et al. , 2008 ; Sayed and Foster , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>In agreement with ﬁndings for repression by H-NS the intergenic region between the yjjQ and the yjjP genes located upstream and in a divergent orientation shows features reminiscent of H-NS-repressed loci ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Apart from the gadBC operon , the most important genes involved in acid resistance are present within the acid fitness island ( AFI ) , a 15 kb region both repressed by H-NS and under the control of RpoS [ 10,11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>In the 20-kb genomic region ( from yddS to ydeN ) encompassing the gadBC operon , none of the genes is known to be H-NS repressed or acid induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>The stabilization of S in the hns mutant is not affected by iraP ( data not shown ) , suggesting that at least one other anti-RssB regulator is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>]<br></li>
<li>The fact that most , if not all , H-NS is located in the nucleoid is shown by immunogold-labelling and electron microscopy [ 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Transcription of CRISPR-1 and casA -- E is repressed by H-NS [ 31 ] and de-repressed by LeuO [ 32 ] or BaeR [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21699496" target="_blank">21699496</a>]<br></li>
<li>16 ) and nhaA ( 37 ) , known to be repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>It is repressed by both LRP and H-NS ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>Transcription of micF is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>It has been noted that many of the genes repressed by H-NS require activation signals to overcome repression ( Schroder and Wagner 2002 ; Yu and DiRita 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>It is possible that other H-NS repressed genes are also regulated by this mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Our results conﬁrm that transcription of leuO is repressed by H-NS , and further show that StpA contrib-utes to this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>These results show that expression of leuO is directed by two H-NS and StpA repressed promoters ( PleuO and P2leuO ) and they suggest that RcsB-BglJ activates transcription from promoter P2leuO at position 84 304 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Under standard laboratory growth conditions transcription is repressed by H-NS and/or StpA ( repressed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Many stressresponse genes are both repressed by H-NS and activated by RpoS , and similar kinds of conditions both relieve H-NS silencing and stimulate transcription mediated by RpoS ( 15 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>Under aerobic condition , transcription of gadE was repressed by the global DNA silencer H-NS in M9 minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Whether or not NhaR can bind to DNA in the absence of Na1 in an hns1 strain is unknown , as transcription is already repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Addition of 2 mM H-NS repressed both hlyE and umuD transcription ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>In anaerobiosis , cydAB transcription was not repressed by H-NS ( Fig. 6 , compare lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Chromosomally located stpA is expressed primarily from a promoter immediately upstream of the gene which is severely repressed by the homologous nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Transcription from the distal promoter , osmCp1 , is independent of S and is repressed by both the leucine-responsive protein and H-NS in minimal me-dium ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867437" target="_blank">12867437</a>]<br></li>
<li>This explains the dominance of the bglJ and leuO mutants over the wild-type copy of the bglJ and leuO gene , respectively , also present in the strains , as they are repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Expression of the leuO gene is repressed by H-NS , and it is upregulated in stationary phase and under amino acid starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>However , the leuO gene is also repressed by H-NS and only moderately induced by branched amino acid starvation in a ppGpp-depen-dent manner ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>The stpA gene is known to be repressed by H-NS and by StpA , as shown by stpA promoter-lacZ fusions and Northern analyses ( 16 , 37 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>The divergent yjjP gene is likewise repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>The expression directed by the divergent yjjQ and yjjP promoters is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Promoter mapping revealed that two H-NS and StpA repressed promoters direct expression of the leuO gene and that one of these promoters is activated by RcsB-BglJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>First , the leuO gene itself is repressed by H-NS and is positively autoregulated ( 5 , 6 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>H-NS is one of the major factors involved in silencing the bgl promoter ( 16 , 25 , 31 ) ; both loss-of-function mutations in the hns gene and mutations that disrupt H-NS binding upstream of the bgl promoter activate the bgl operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843842" target="_blank">22843842</a>]<br></li>
<li>rcsA is repressed by the global regulator H-NS , and dsrA , a gene located upstream of rcsA , encodes a small RNA that controls H-NS synthesis by interfering with the hns mRNA ( Sledjeski & Gottesman , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Binding ot H-NS to the upstream region of Its own gene hns as revealed by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Alternatively , a bound transcription factor may form a barrier that prevents spreading of the H-NS nucleoprotein complex into the promoter , as proposed for autoregulation of the H-NS-repressed leuO gene and the divergent leuABCD operon encoding enzymes of the leucine synthesis pathway ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>U p o n induction of the synthesis of H-NS by depletion of tryptophan , the transcription of the hns-lacZ fusion gene was significantly repressed , while that of the tac-lacZ fusion gene was not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Provided that the synthesis of H-NS was induced in the cells on the omission of tryptophan , the transcription of the set of promoter-lacZ fusion genes carried on TY008 , TY010 , and TY012 was significantly repressed , while that of the tac-lacZ fusion gene was unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>A PCR-ampliﬁed DNA fragment extending from -474 to +222 relative to the hlyE transcript start containing the hlyE promoter and the promoter for the divergently transcribed umuD gene was used in in vitro transcription assays in the presence of increasing concentrations of H-NS as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>One possibility is that H-NS is inactive under anaerobic conditions and it disassembles from the gadE promoter , allowing ArcA which is active under this condition to bind the promoter of gadE and initiate gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>As seen from Fig. 4A , cat transcription from pKK660 was severely inhibited ( -75 % ) by purified H-NS , white transcription from pKK110 -LCB- Fig. 4B ) was not affected under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>Although gadA transcription is stimulated more than 2-fold by GadX alone and strongly repressed ( up to 70 % ) by H-NS alone , as also shown in Figs. 4 and 6 , the effect of simultaneously adding both proteins to the reaction mixture gives rise to the gadA transcription profiles shown in Fig. 7A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In fact , gadA transcription attains 226 % in the presence of GadX alone ( 45 nM ) and 189 and 135 % in samples containing the same amount of GadX and 468 and 936 nM H-NS , respectively ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>E. coli chromosomal cas3 is located upstream of casA -- E ( Figure 1A ) and is not controlled by H-NS , although cas3 transcription in Thermus thermophilus is increased 2.5-fold in response to infection by phage ϕYS40 [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21699496" target="_blank">21699496</a>]<br></li>
<li>Furthermore , the effects of the dksA strain do not appear to be mediated by two other proteins that affect rRNA transcription , Fis and H-NS ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15294157" target="_blank">15294157</a>]<br></li>
<li>Since this gene is part of an operon with hha and ybaJ is the first gene downstream of their common promoter , the microarray data indicate that Hha represses ybaJ as well as its own transcription , a common property among the histone-like regulatory proteins like IHF [ 29 ] , H-NS [ 30 ] , and HU [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_136">
Yang , J. , Tauschek , M. , Strugnell , R. and Robins-Browne , R.M. ( 2005 ) The H-NS protein represses transcription of the eltAB operon , which encodes heat-labile enterotoxin in enterotoxigenic Escherichia coli , by binding to regions downstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></label>
<input id="tu_similarity_136", type="checkbox">
<div class="TU_similar">
<li>The H-NS protein represses transcription of the eltAB operon , which encodes heat-labile enterotoxin in enterotoxigenic Escherichia coli , by binding to regions downstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li></div>
<li>The nucleoid-associated protein H-NS also represses osmC transcription from both promo-ters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>( 1994 ) reported that the nucleoid-associated protein H-NS prevents its transcription by Es , whereas Es is able to 70 s transcribe it even in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Because H-NS represses its own transcription ( 6 ) , a decrease in H-NS protein should lead to increased hns mRNA levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10954740" target="_blank">10954740</a>]<br></li>
<li>The H-NS and StpA proteins negatively autoregulate the transcription of their own genes and each represses the gene that encodes the other protein in trans 6,10 -- 15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>The Fis-polymerase interactions are disrupted and transcription is no longer activated by Fis , In ( c ) , high occupation of H-NS sites completely deforms the DNA structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>The repressive H-NS complex is strikingly local and highly specific to the DNA sequence in the proV promoter , because H-NS does not affect transcription from ptac on the same DNA molecule ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121424" target="_blank">11121424</a>]<br></li>
<li>H-NS might repress appY transcription by interacting with the curved DNA sequence upstream of the promoter , similarly to the way it represses its own synthesis ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>These results demonstrate that both Cnu and H-NS are required to make the host cells resistant to Sm , presumably because the Cnu-H-NS complex binds to the operator DNA ( Ori14 ) , which , in turn , represses transcription of the rpsL gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>These results suggest that H-NS in addition to repressing transcription initiation represses transcription elongation by inducing polarity ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Although it was assumed that H-NS positively affects ﬂhDC transcription , its regulation has not been clearly demonstrated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>H-NS is not involved in sS-dependent transcription of nhaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>The thermoregulator H-NS , but not RimJ , alters cirA and fes transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>Expression of the cps-lacZ fusion is dependent on RcsA ; transcription of rcsA is in turn negatively regulated by H-NS ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770508" target="_blank">9770508</a>]<br></li>
<li>H-NS negatively regulates evgA and gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>H-NS negatively regulates its own transcription ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9770507" target="_blank">9770507</a>]<br></li>
<li>Although gad transcription is abrogated during the stationary phase in the hns rpoS double mutant , the possibility also exists that gad promoters can be recognized by H-NS , and transcribed by s70 under certain conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Our data support this report and demonstrate that the increased content of H-NS in stationary phase cultures is largely caused by the genetic regulation of hns transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>A prediction of this autoregulatory model for hns transcription is that synthesis of H-NS should shut down as cells enter stationary phase and cease active DNA synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>In the present paper , we report that hns transcription is under autoregulation during the logarithmic growth phase , and that H-NS functions as a repressor for its own transcription Summary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>These in vitro results indicated that the autoregulation observed in vivo for the hns promoter can be explained by H-NS functioning as a selective repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>H-NS regulation of the ﬂhD operon is complex , however ; although an hns mutant was nonmotile , in vitro transcription of the ﬂhD operon by H-NS was negative ( 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>To determine if H-NS affects its own transcription , we constructed an isogenic derivative of CU236 , which carries a hns : : neo ( Kin r ) insertion ( CU237 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>To confirm that H-NS does indeed affect its own transcription , the hns gene was introduced into the hns deletion strain CU237 on a plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Though this appears to suggest a form of molecular backup between two homologous proteins , the backup is unequal : whereas the DstpA mutant hardly affects gene expression , a Dhns mutant leads to upregulation of at least 500 genes as agreed upon by several studies .12,13,15,34 Whereas the absence of stpA does not visibly affect in vivo DNA binding of H-NS on a genomic scale , that of hns abolishes binding of StpA to about two-thirds of its wildtype binding regions .32 Though both H-NS and StpA repress each other 's transcription , leading to the up-regulation of one in the absence of the other , the degree of up-regulation conferred on StpA by the absence of H-NS may not be suficient to compensate for the lack of H-NS .30 This in part might be due to the higher susceptibility of StpA to degradation by Lon in the absence of H-NS .33 A complete backup of H-NS activity by StpA might not be desirable as there are conditions , such as a late stationary phase , in which an attenuated H-NS is favoured .36 The close relative of E. coli , Salmonella enterica Typhimurium , encodes three H-NS-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>The ratio of 13-galactosidasespecificactivityin DMO9 to PS2529 is plotted on the ordinateversusthe leucineconcentration plotted on the abscissa.The triangles indicatedata fromculturesin stationaryphase ; the squares are for culturesin logphase.Thefilled are for culturesgrownin media of high osmotarity ( 0.3 M symbols NaC1 ) and the open symbols are for cultures grown at low osmolarity Extra copies of the hns + allele in trans reduce ilvIH transcription levels The experiments reported above demonstrated that the absence of H-NS of results in higher expression of the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The cadBA operon and the adiA gene , which encode a lysine decarboxylase , an antiporter and an arginine decarboxylase , respectively , have been shown to be regulated by H-NS in rich medium under anaerobic conditions ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>The clyA locus in the E. coli K-12 chromosome does not seem to be expressed under most laboratory growth conditions , and our present evidence established that the gene is subject to silencing by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053378" target="_blank">11053378</a>]<br></li>
<li>Similarly , the complex regulation of the acid stress response gene gadA , which is activated by GadE and RcsB , involves repression by H-NS ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>By contrast , we found that the expression of the dps gene , directly regulated by H-NS and known to protect cells against multiple stresses [ 29 ] , is essential to lysine - and arginine-dependent responses to acid stress , while its role is less important during the glutamate-dependent response ( Table 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>( i ) H-NS represses the gadE gene encoding the central activator of the glutamate-dependent acid resistance pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Like cai and ﬁx operons , the caiF gene appeared to be positively controlled by CRP and FNR and negatively controlled by H-NS and the NarL ( plus nitrate ) regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>For example , H-NS is known to interfere with ss and Lrp in transcription of the osmC gene ( 8 ) and with FNR and CAP in the regulation of cai and ﬁx operons ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>These results confirm the theory that H-NS negatively regulates its own gene in order to match protein production to DNA synthesis , and they suggest that it is important for cells to maintain a constant ratio of H-NS protein to DNA during logarithmic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>First , H-NS is a repressor of hdfR , a gene that codes for a LysR-like protein that negatively regulates the promoter of the flagellar master-regulator operon , flhDC29 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Similarly an activator , HdfR , of the master operon , is also negatively regulated by H-NS in vivo ( 33 ) and its gene is associated with a putative H-NS binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>Expression of the hdfR gene is negatively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>These results indicated that the lrp gene is not only feedback regulated by Lrp but is also controlled by H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>For example , the protein that is encoded by gene 5.5 of bacteriophage T7 inactivates H-NS , resulting in derepression of both host and viral promoters99 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>FIS corepresses transcription of crp ( encoding CAP ) in addition to its autoregulation by CAP ( GonzaÂlez-Gil et al. , 1998 ) , and it positively regulates transcription of hns encoding H-NS , antagonistically to its autoregulation by H-NS ( Falconi et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760165" target="_blank">10760165</a>]<br></li>
<li>The amount of H-NS within the cell is therefore likely to be under tight control , which is presumably achieved by the negative autoregulation of the hns gene ( Dersch et al. , 1993 ; Ueguchi and Mizuno , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The regulatory region upstream of hns has also been shown to contain intrinsic curvature , where H-NS has been shown to bind and , by autoregulation , affect its own expression ( Dersch et al. , 1993 ; Falconi et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Two others , fllO and fl12 between 70999-70225 , were previously identified as cotranscribed genes encoding soluble ( cytoplasmic/periplasmic ) proteins whose expression is affected by the nucleoid protein H-NS ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8041620" target="_blank">8041620</a>]<br></li>
<li>Yoshida , T. , Ueguchi , C , and Mizuno , T. ( 1993a ) Physicai map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucieoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_181">
Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426343" target="_blank">12426343</a>]<br></label>
<input id="tu_similarity_181", type="checkbox">
<div class="TU_similar">
<li>[ 35 ] Yoshida T. , Uegushi C. , Mizuno T. , Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278074" target="_blank">11278074</a>]<br></li>
<li>Yoshida , T. , Ueguchi , C. , and Mizuno , T. ( 1993c ) Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>[ 39 ] Yoshida , T. , Ueguchi , C. and Mizuno , T. ( 1993 ) Physical map location of a set of Escherichia coli genes ( hde ) whose expression is affected by the nucleoid protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li></div>
<li>Induction of the ferritin gene ( ftnA ) of Escherichia coli by Fe2 - Fur is mediated by reversal of H-NS silencing and is RyhB independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21908668" target="_blank">21908668</a>]<br></li>
<li>( Received forpublication , August 18 , 1993 , and in revised form , November 12 , 1993 ) The Escherichia coli hns gene encodes the abundant cuits control theintracellularconcentration of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>H-NS is encoded by the hns gene located at 27.5 min on the E. coli map ( 18 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>Transcription of the hns gene encoding the H-NS protein appears to be negatively regulated by H-NS itself both in vitro and in vivo , We have examined the role of this mode of regulation in wild-type cells in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Insertion elements , located from 145 to 13 bp relative to the transcription start site , activated the ade gene as did unlinked mutations in the hns gene , encoding the histone-like protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>and MCI 061 , can be activated for csgBA transcription by mutating the hns gene encoding H-NS ( Olsen et ai , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Both promoters can be activated in anCT ^ - negativebackground when the hns gene encoding H-NS is also inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>Identification of a cold shock transcriptional enhancer o Escherichia coli gene encoding nucleoid protein H-NS ABSTRACT The hns ( 27 min ) gene encoding the 15.4-kDa nucleoid protein H-NS was shown to belong to the cold shock regulon of Escherichia coli , its expression being enhanced 3 - to 4-fold during the growth lag that follows a shift from 37C to 100C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1961761" target="_blank">1961761</a>]<br></li>
<li>Mol Gen Genet ( 1993 ) 236:171 -178 OIGG © Springer-Verlag 1993 Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H - N S functions as a repressor of its own transcription Chiharu Ueguchi , Minoru Kakeda , and Takeshi Mizuno Laboratory of Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya464 , Japan ReceivedApril 22 , 1992 / AcceptedJuly 6 , 1992 properties of H-NS , was made by a different biochemical approach ( Bracco et al. 1989 ; Yamada et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Ueguchi , C , Kakeda , M. , and Mizuno , T , ( 1993 ) Autoregulatory expession of the Esherichia ccli hns gene encoding a nucleoid protein -- H-NS functions as a repressor of its own transcription , Mol Gen Genet 236 : 171 -178 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>EMBO J 12:1039 -1046 Ueguchi C , Kakeda M , Mizuno T ( 1993 ) Autoregulatory expression of the Eseheriehia eoli hns gene encoding a nucleoi protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Kaneda , M. and Mizuno , T. ( I 993 ) Autoregulatory expression of the Eschen ' chia co/i hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_197">
and Mizuno , T. ( 1993 ) Auto-regulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984093" target="_blank">7984093</a>]<br></label>
<input id="tu_similarity_197", type="checkbox">
<div class="TU_similar">
<li>Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071234" target="_blank">8071234</a>]<br></li></div>
<li>Agric Biol Chem 55:3139 -3141 Ueguchi C , Kakeda M , Mizuno T ( 1992 ) Autoregulatory expression of the Eseherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor for its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_200">
Kakeda , M. , and Mizuno , T , ( 1993 ) Autoregulatory expression of the Eschehctiia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></label>
<input id="tu_similarity_200", type="checkbox">
<div class="TU_similar">
<li>[ 22 ] Ueguchi C. , Kakeda M. , Mizuno T. , Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>Ueguchi , C. , Kakeda , M. , and Mizuno , T. ( 1993 ) Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Ueguchi , C , Kakeda , M. , and Mizuno , T. ( 1993 ) Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein : H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997164" target="_blank">7997164</a>]<br></li>
<li>Ueguchi , C. , Kakeda , M. , and Mizuno , T. ( 1993 ) Autoregulatory expression of the Escherichia coli hns gene encoding a nucleoid protein -- H-NS functions as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li></div>
<li>Falconi , M. , Higgins , N.P. , Spurio , R. , Pon , C.L. , and Gualerzi , C.O. ( 1993 ) Expression of the gene encoding the major bacterial nucleoid protein H-NS is subject to transcriptional autorepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>Transcription of hns , the gene encoding the abundant nucleoid-associated protein H-NS , is also subject to negative autoregulation ( Dersch et aL , 1993 ; Falconi et aL , 1993 ; Ueguchi et aL , 1993 ) but is unaffected by RpoS ( Dersch et aL , 1993 ; our unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Expression of the hemolysin operon in Escherichia coli is modulated by a nucleoid-protein complex that includes the proteins Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Hha hns mutants contain increased levels of HtrA and HdeA proteins in the periplasm Considering the previous characterization of the interaction of Hha and H-NS proteins [ 29 ] , and the participation of these proteins in the modulation of the expression of the operon encoding the toxin a-hemoly-sin ( Hly ) [ 29,32 ] , we further investigated other E. coli proteins modulated by the Hha -- H-NS complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Furthermore , it was found that hscA , encoding an HSP66 protein and DnaK homologue , modulates the H-NS-mediated repression of pilA and the bgl operon in an hns mutant with reduced levels of H-NS protein ( Kawula & Lelivelt , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>The operon is also activated by mutations within the hns locus , encoding the histone-like nucleoid structuring protein H-NS ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22020646" target="_blank">22020646</a>]<br></li>
<li>Like many otherwise unrelated stress-induced genes , the cad operon is derepressed by mutations in hns , the structural gene for the histone-like protein H-NS ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836317" target="_blank">7836317</a>]<br></li>
<li>The regulatory features of the stpA operon , being affected by H-NS and by temperature , resemble those found in other cases of transcriptional control subject to temperaturedependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>1992 ) In the present paper we have demonstrated Ihat hns is also subject to negative control ( auto-repression ) by its own product H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>x20 xlO0 ( b ) Transcription initiating at the putative promoter ( s ) for the operon encoding 10K-S and 10K-L is also under the control of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>It should be noted that this divergent promoter is also under the control of H-NS ( Fig. 6 B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Indeed , the opposite effect was observed , whereby the increased levels of Rho that were detected in the rho and nusG mutants were partially reversed upon H-NS 64 or YdgT expression ( Fig. 6A , compare lanes 5 and 8 to lanes 2 and 3 ) ; since it is known that Rho is autoregulated ( 31 ) , these results provide yet another line of evidence that H-NS 64 and YdgT increase the efﬁciency of Rho-dependent termination in the mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>ArcA ANTAGONIZES THE H-NS MEDIATED REPRESSION OF gadE-mdtEF DURING THE TRANSITION FROM AEROBIC TO ANAEROBIC GROWTH Our results above showed that the expression of gadE-mdtEF is largely repressed by the DNA silencer H-NS under aerobic conditions in M9 glucose medium , whereas under anaerobic condition in the same medium its expression is activated primarily in an ArcA dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Can other known n ^ - dependent genes be transcribed by En ^ ° in the absence of H-NS ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>uspB induction is dependent on sS and modulated by IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>Reconstituted EsS transcribes hchA in a reaction that is inhibited by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>RESULTS rcsA Transcription Is Silenced by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7534408" target="_blank">7534408</a>]<br></li>
<li>It is not presently known whether H-NS directly affects the regulation of micF and ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>It is activated by s70 and affected by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>( i ) It directly acts as transcriptional repressor of gadA and gadX promoter ; ( ii ) it exhibits an additional negative control over gadX expression possibly by reducing its transcription from the gadA promoter ; ( iii ) unlike gadA , gadB does not display any specific H-NS binding , and its promoter activity is not affected in vitro by H-NS ; however , H-NS might indirectly inhibit the GadX-mediated expression of gadBC by repressing gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>These results indicate that H-NS is involved in the negative transcriptional regulation of the gad genes and raise the possibility that H-NS would directly silence the gadA and gadB promoters and/or indirectly repress gad expression through the control of trans-acting factors , probably the stationary-phase factor ss .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>To further analyze the regulation of yjjQ by H-NS and LeuO , a series of fragments encompassing the putative yjjQ promoters was fused to a promoterless lacZ gene as the reporter for expression ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>In contrast , the expression of the yjjQ promoter-lacZ fusion encompassing position 347 to 26 was activated by LeuO and was repressed more efﬁciently by H-NS than the shorter constructs were ( Fig. 5A , PQ 347 to 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>Because it is known that FIS exerts its activating effect on rRNA transcription at P1 promoters , and the same speciﬁcity had been noted for H-NS-dependent repression in vitro ( Tippner et al. , 1994 ) , we asked whether or not P2-directed expression will be affected in vivo when FIS or H-NS are depleted from the cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></li>
<li>This increase of dicB and dicC expression was considerately less in the absence of H-NS : in MG1655ghns / pCnuK9E , dicB and dicC expression was increased 78 - and 24-fold , respectively , whereas that of dicA was decreased by half ( 0.55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>This suggests that during anaerobic growth in liquid cultures in the absence of glucose , FNR is the major regulator of hlyE expression , and that CRP-mediated hlyE expression requires the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>The ﬁndings that SlyA does not enhance hlyE expression in hns cultures grown in liquid medium under anaerobic conditions and that SlyA appears unable to activate hlyE expression in the absence of FNR and CRP and the determination of the location of the region of PhlyE occupied by SlyA ( Fig. 6 ) suggest that SlyA activates hlyE expression by antagonizing the negative action of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>As a control , the expression level directed by a lacUV5 -- t1-L -- lacZ fusion which carries the bgl-leader in between the lacUV5 promoter and lacZ but lacks the bglG coding region ( encompassing the downstream silencer , see below ) is not affected by H-NS ( Fig. 4E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Like SlyA at PhlyE , VirB has no positive effect on iscB expression in the absence of H-NS , but VirB alters the structure of PiscB in the region occupied by H-NS and this requires a DNA sequence located downstream of the iscB transcript start ( Turner and Dorman , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>Changes in growth and environmental conditions are known to correlate with alterations in DNA topology ( 1 , 7 , 27 ) , and it is thus possible that under certain conditions the DNA of the stpA promoter is in a favorable conformation for expression even in the presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>It has been shown that H-NS binds to the upstream and downstream regions of the hlyE promoter ( PhlyE ) in E. coli and is involved in silencing hlyE expression ( Westermark et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>RESULTS H-NS Binds to the proU Control Region-DNA segments located upstream and downstream of the proU promoter are known to influence proU expression ( 17 , 26 , 51 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>As an example , H-NS acts upstream of its own promoter to autoregulate its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Both cai and ® x operons require the presence of FNR and CRP for their expression , while they are partially repressed by H-NS ( Eichler et al. , 1995 ; Buchet et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li>
<li>The expression of the gad system and hdeAB is repressed by a nucleoid protein H-NS during the exponential growth phase ( Yoshida et al. , 1993a , b , c ; De Biase et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The expression of leuO is likewise repressed by H-NS , and not detectable during exponential growth ( Klauck et al. , 1997 ; Chen et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Remarkably , expression of the leuO gene is itself repressed by H-NS under standard laboratory growth conditions ( Klauck et al. , 1997 ; Chen et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>H-NS is an abundant , approximately 15 kDa nucleoid-associated protein that is present at about 20 000 copies per genome equivalent and is a well-known silencing protein that represses expression of cryptic genes such as the bgl operon involved in b-glucoside utilization ( Dorman and Deighan , 2003 ; Dole et al. , 2004 ; Dorman , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>The gadB and gadC genes form an operon and a number of factors , including H-NS , RpoS and CRP-cAMP , are involved in the transcriptional control of gad expression ( Castanie-Cornet & Foster , 2001 ; De Biase et al. , 1999 ; Rowbury , 1997 ; Rowland et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>One such is H-NS enhancement of the expression of the genes ¯ hC , ¯ hD and ¯ iA , which encode the ¯ agellar master control operon and the ¯ agellar sigma factor , respectively ( Bertin et al. 1994 ) Three lines of evidence con ® rm that the activity of the switch located ® mA promoter is a $ ected by hns background and enhanced when the switch is in the ON but not in the OFF orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>CHR activity was always high because the cai operon ( responsible for the expression of carnitine metabolism ) was controlled by protein regulators such as FNR ( transcriptional regulator under anaerobic conditions ) , catabolic repression via protein receptor ( CRP ) of AMPc , histones ( H-NS ) , and the transcriptional factor j 2 ( RpoS ) , which is expressed when the cell reaches the stationary phase of growth and inhibits carnitine metabolism ( Eichler et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14595781" target="_blank">14595781</a>]<br></li>
<li>The expression of gadX is activated by the alternative sigma factor RpoS during the stationary phase of growth [ 15,19,21 ] , but is repressed during the exponential phase by the nucleoid protein H-NS due to its binding to the gadX promoter or its destabilizing effect on RpoS [ 23-25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21314918" target="_blank">21314918</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_247">
The nucleoid protein H-NS is involved in the negative control of gadA and gadBC transcription during the exponential growth phase under oxidative conditions , while the alternative sigma factor RpoS is responsible for gad expression at the stationary phase but not during fermentative growth ( 6 , 7 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></label>
<input id="tu_similarity_247", type="checkbox">
<div class="TU_similar">
<li>The nucleoid protein H-NS is involved in the negative con trol of gadA and gadBC transcription during exponential phase , while the alternative sigma factor RpoS is responsible for Gad expression in stationary phase ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li></div>
<li>In the case of LRP and H-NS , derepression is discernible during the complete growth curve , while in the fis mutant expression of 6S RNA during exponential phase is not enhanced but rather slightly reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
<li>In summary , Hsp31 expression is highly sensitive to the growth temperature and H-NS plays an important role in modulating Hsp31 concentration in exponential phase cells , silencing hchA transcription more efﬁciently as the temperature decreases while allowing RNA polymerase access to its EsD - and EsS-dependent promoters in times of thermal stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16796689" target="_blank">16796689</a>]<br></li>
<li>Since it was demonstrated that H-NS is an inhibitory component of the sS stationary-phase induction network and also inhibits the expression of sS itself ( 3 ) and since we now know that sS also takes part in nhaA regulation , we examined the possible involvement of H-NS in rpoS-dependent stationary phase induction of nhaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Recently , it has been reported that Fis and H-NS modulate the expression of the dps gene by allowing preferential utilization of the dps promoter a putative dual promoter , by s over s when cells reach the 38 70 stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>There is also evidence that the intracellular concentration of H-NS , which like HU autoregulates its own expression , is at its highest point at the onset of the stationary phase ( Spassky et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>As H-NS is a strong negative regulator of stpA expression , it was possible that stpA expression also varied with growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>H-NS over-expression induces an artificial stationary phase by silencing global transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_256">
McGovern , V. , Higgins , N.P. , Chiz , R.S. , and Jaworski , A. ( 1994 ) H-NS over-expression induces an artiﬁcial stationary phase by silencing global transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9632265" target="_blank">9632265</a>]<br></label>
<input id="tu_similarity_256", type="checkbox">
<div class="TU_similar">
<li>McGovern , V. , Higgins , N.P. , Chiz , R.S. , and Jaworski , A. ( 1994 ) H-NS over-expression induces an artificial stationary phase by silencing global transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li></div>
<li>Thus , the accelerated production of RpoS under acidic conditions was not the result of an early entry into stationary phase but was a consequence of low pH. H-NS conditionally represses gadA and gadBC expression independent of GadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>In this respect , it is worth mentioning that the observed auto-regulatory expression of H-NS is dependent on the growth phase , i.e. its autoregulatory expression appears to be derepressed at stationary phase , and consequently the level of H-NS in cells appears to increase at this stage of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Finally , the expression of class III genes ( e.g. , csiD and csiE ) is only weakly increased in an hns mutant during the exponential phase of growth but seems to require H-NS as a positive factor for stationary-phase induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>H-NS represses the expression of many genes that are expressed in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
<li>Thus , it is known that H-NS is a pleiotropic exponentialphase inhibitor of the expression of many stationary-phase-responsive genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>On the other hand , H-NS somewhat ( approximately 40 % ) repressed the expression of cspD after entry into the stationary growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>]<br></li>
<li>H-NS activates ydaM expression during entry into stationary phase and represses yciR expression throughout the entire growth cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>They also participate in diverse roles such as regulation of DNA replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>DNA replication that H-NS inhibits the expression of the other arginine triggered by double-stranded breaks in E. coli : dependence o n decarboxylase , encoded by adiA , that is expressed under homologous recombination functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Since rcsB is transcribed either individually or within the rcsDB operon , the effect of H-NS inactivation on the expression of rcsB and rcsD genes was analyzed by real-time quantitative RT-PCR by comparison of their RNA levels between the wild-type and hns-deﬁcient strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>, t glyA glyA glyA hns : : Tn5seq hns : : Tn5seq 1 tdh : : cat Addition to minimal medium None Glycine ¬ + ¬ + + + + + + + + 300 200 , Temperature affects the expression of the kbiitdh operon 100 Certain genes that are expressed at 37 ° C but not at 30 ° C are controlled by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Despite a similar complete loss of ﬂagellin accumulation in hns and crp mutants , in vitro transcription assays showed that CAP activates expression of the master operon while H-NS represses it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>As an hns null mutant also produced anucleate cells [ 20 ] , it is possible that H-NS could regulate expression of the whole mukB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>Nevertheless , the ﬂhDC master operon is repressed in vitro by H-NS and expression from its promoter is increased in an hns mutant ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>When H-NS is absent , as in hns-ll8 , expression levels ( compared to hns + parental controls ) of the ilvIH operon increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>We also conﬁrmed that H-NS in strain CU305 is fully functional , because the expression of the proVWX operon is not derepressed by the LeuO overproduction ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422614" target="_blank">9422614</a>]<br></li>
<li>An example which could clearly point to such a role for HU is the proU operon , the expression of which is repressed by H-NS [ 66,67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>We show that expression of the kblltdh operon is temperature controlled and that this control is not mediated through H-NS action as has been shown for some other temperature controlled genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>H-NS is a positive regulator of ﬂhCD , the master operon which controls the expression of all ﬂagellar genes required for motility ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922259" target="_blank">9922259</a>]<br></li>
<li>Therefore the expression of the kbl / tdh operon is reduced by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>A direct test of these hypotheses can be carried out by furnishing additional H-NS and monitoring the expression of the iIvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Furthermore , we have shown that constitutive expression of transcription factors LeuO and BglJ relieves silencing of the bgl operon by counteracting repression of the promoter by H-NS through the upstream site , whilst BglJ and LeuO do not affect downstream repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Taken together , within the natural context of the bgl operon the relief of the H-NS mediated silencing of the promoter masks the H-NS mediated repression at the downstream silencer due to increased transcription and translation of bglG , positive autoregulation of BglG , and limitation of the maximum expression level of the operon by the PTS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Therefore , it was of interest to know whether the positive effect of H-NS on ﬂhDC expression was a direct consequence of its binding to the promoter region of the master operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>The increased transcription from these promoters observed in the absence of H-NS correlates with the increased expression of the gspA-lacZ and gspD-lacZ operon fusions ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>HdfR binds to the promoter region of the ﬂhDC operon , and its expression was negatively regulated by H-NS , suggesting that the apparent activation of ﬂhDC transcription by H-NS is mediated through the negative regulator HdfR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Perhaps under unstressed conditions when the expression of the operon is unnecessary , the promoter of gadE is primarily occupied by H-NS and transcription from the various promoters in this region is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>However , in hns mutants , the PgadE-lacZ activities increased more than 10-fold to about 250 Miller units , suggesting H-NS strongly represses the expression of gadE-mdtEF under aerobic condition in the M9 glucose minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>The expression of class I genes ( e.g. , osmY ) is no longer affected by an hns mutation when assayed in an rpoS mutant background , suggesting that H-NS affects these genes only indirectly via its role in sS regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>The hns mutation caused a two - to fourfold increase in ftnA -- lacZ expression in the absence of Dip , and a ﬁve - to eightfold increase in its presence , thus conﬁrming that H-NS represses ftnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The approximately ﬁvefold repression in ftnA -- lacZ expression caused by Dip in the wild type was reduced to approximately twofold by the fur and/or hns mutations suggesting that , at least in part , iron induction of ftnA is dependent on both Fur and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Plasmid pHM52 , expressing the dominant negative H-NS mutant protein G113D and plasmid pHOP11 , encoding wild-type H-NS protein were used to transform wild-type ( MG1655 ) and hns - strains ( PD32 ) from which total RNA was analysed before and after the induction of the expression of H-NS or its mutant variant by primer extension .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>]<br></li>
<li>A role for IHF or H-NS in regulating ftnA expression was tested by Western blot analysis of the corresponding himA ( one of two unlinked genes encoding the heterodimeric IHF protein ) and hns mutants , and parental strains ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Decreased induction of acrA expression by MarA in the hns mutant ( Fig. 2B ) did not involve alteration of marA expression ( Fig. 2A ) , and no protein-protein interaction between H-NS and MarA was found in two-hybrid experiments ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Since both the hns-lacZprotein and operon fusion strains show the same pattern of hns expression , the data presented in Fig. 2 and Fig. 3 strongly suggest that synthesis of the H-NS protei is regulated in response to growth phase at the level of hns transcription , Nuc/eoid-associated ONA-binding protein H-NS 877 A EcuRI + CAATTCCTTACATTCCTCGCT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>H-NS is clearly not essential for osmotic regulation ofproU transcription , since hns mutants still show osmotic regulation of proU expression ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>Our results clearly prove that H-NS regulates gad expression by acting at the transcriptional level because repression of gad transcription during exponential growth is relieved in the hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10383761" target="_blank">10383761</a>]<br></li>
<li>Growth-phase regulation of hns mRNA and H-NS protein levels A prediction of our model for autoregulatory coupling of H-NS synthesis to DNA synthesis is that hns expression should be repressed in stationary phase when DNA synthesis ceases but the protein remains present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596450" target="_blank">8596450</a>]<br></li>
<li>Although this hns mutation confers less H-NS protein in the cell , this reduced concentration did not affect ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Overproduction of the StpA protein affects expression from at least four hns regulated operons ( the papB , proU , bgl and hns operons ) , in both the presence and absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>It has been previously reported that , under normal conditions , expression of hdeA is ( denoting hns depending expression A ) repressed by H-NS protein [ 39,40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16143461" target="_blank">16143461</a>]<br></li>
<li>Genes repressed by H-NS often need a speci ® c activator for maximal expression , even in an hns background ( Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>We also showed that nmpC expression is repressed by H-NS ( threefold increased expression in the presence of an hns mutation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Detection of the YciE and YciF proteins in the hns mutant but not in the wild-type strain ( Fig. 4A ) suggested that expression of these proteins is silenced by H-NS in E. coli K-12 , in agreement with a previous finding ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Considering that the hns expression is under autoregulatory control , we test in vitro whether Hha could modify the afﬁnity of H-NS for its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>One speculative possibility is that Lon and DnaKJ modulate the H-NS repressing complex or change the level of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>Here we demonstrate that H-NS represses sS expression , that the absence of H-NS results in a failure to osmotically induce sS , and that most proteins overexpressed in an hns mutant are in fact sS dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>H-NS : moiecuiar anaiysis of a subset of proteins whose expression is enhanced in a hns deietion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854117" target="_blank">7854117</a>]<br></li>
<li>We succeeded in cloning and identifying genes encoding such proteins and demonstrate that the expression of these genes is negatively regulated by H-NS at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455549" target="_blank">8455549</a>]<br></li>
<li>Therefore , to study the role of H-NS and CAP in ﬂagellar-gene expression , we constructed transcriptional fusions between the cat gene and the ﬂiA gene , encoding the speciﬁc ﬂagellar sigma factor , as well as ﬂiL and ﬂhB , two genes also located at the second level in the ordered cascade and regulated by the FlhDC regulatory proteins ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>nhaA represents a new addition to the list of genes whose expression is regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>Whereas certain sS-dependent genes ( e.g. , osmY ) are only indirectly regulated by H-NS via its role in the control of sS expression , others are also H-NS-regulated in a sS-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>In order to determine the influence of H-NS on expression of these genes , fimB and timE promoter-lacZ fusions were employed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7510658" target="_blank">7510658</a>]<br></li>
<li>In a preliminary study we found , by using plasmid located cents m-lacZ fusions , that H-NS apparently downregulated the expression of the cents mB and cents mE genes [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>By their nature , microarray screens used to identify genes whose expression is hns sensitive would not necessarily reveal all genes under direct H-NS control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>In order to gain a deeper understanding of the genetic control of hns expression , we have constructed a set of chromosomal hns -- lacZ protein and operon fusions and have used these hybrid genes to study transcriptional control of hns in response to the growth phase and the synthesis of H-NS itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>In previous studies , it was found that expression of the mdtEF genes is up-regulated by the bacterial two-component signal transduction system EvgSA ( 20 ) and the AraC/XylS acid resistance regulator YdeO ( 14 ) , while their expression is repressed by a histone-like protein , H-NS ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>The transcription of csgA is positively regulated by the RpoS sigma factor and repressed by the histone-like protein H-NS , but the temperature and osmolarity regulation of curli expression involves other factors ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11466275" target="_blank">11466275</a>]<br></li>
<li>The H-NS and IHF histone-like factors also affected sodB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>Regulation of sodB expression by the histone-like proteins IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>It is therefore interesting and important to clarify the relationship between H-NS , a histone-like protein , and the expression of the mukB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>) , encoding the histone-like protein H-NS , which represses expression of many genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>]<br></li>
<li>Recently , we found that histone-like protein H-NS represses the expression of acrEF and mdtEF ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
<li>The rcsA promoter itself is H-NS repressed ; its expression is positively autoregulated by RcsA-RcsB and requires induction of the Rcs signaling pathway ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>It is known that the expression of a variety of genes is repressed by H-NS , and mutations in hns result in various phenotypes , but the role of H-NS in the drug resistance of Escherichia coli has not been known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>The expression of virulence genes in the LOCUS OF ENTEROCYTE EFFACEMENT ( LEE ) pathogenicity island of enteropatho-genic E. coli is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>In the absence of H-NS , the expression of `` epistatically controlled genes '' is repressed by StpA , whereas that of `` unilaterally controlled genes '' is not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>Plasmid-expressed DsrA increased expression of genes normally repressed by H-NS , including rcsA , proU , and papA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11222598" target="_blank">11222598</a>]<br></li>
<li>In contrast , expression from pnir carried by the pnir7061 fragment ( which extends upstream to position 261 ) was hardly repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Some recent studies suggest that ilvIH expression is repressed by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Indeed , we found that ydaM expression is under positive control of H-NS , whereas yciR is repressed by H-NS ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Beyond the silencing of rcsA expression by H-NS and the multicopy effect of DsrA on H-NS , little is known about the regulation of rcsA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>Regarding cadBA transcriptional regulation , it is known that H-NS represses cadBA expression under noninducing conditions ( 46 ) , and according to the current model , CadC binding dissolves the repressor complex formed by H-NS ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>These data suggested that hlyE expression is activated by FNR and CRP , repressed by H-NS , and that H-NS-mediated transcriptional silencing is relieved by SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17892462" target="_blank">17892462</a>]<br></li>
<li>These data indicate that expression of dnaKJ is required for downstream repression by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207917" target="_blank">16207917</a>]<br></li>
<li>These results indicated that $ S induces expression of yciGFE in E. coli K-12 only when H-NS mediated repression was relieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>This positive H-NS effect is also transmitted onto the expression of the downstream target csgB ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>In the absence of both H-NS and StpA , the expression of the divergently oriented malKlamBmalM and malEFG op-erons was almost completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>In the absence of added sugars , CRP indeed appeared to be an activator of sfm both in the presence and in the absence of H-NS , as shown by decreased sfm expression in crp mutants ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>In the absence of glucose , H-NS appeared to have a positive effect on hlyE expression ( Fig. 1c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Osmolarity did not inﬂuence nrdA expression ; accordingly , the modulatory role of H-NS with regard to nrdA was not affected by the osmolarity of the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>This suggests that H-NS has the potential to regulate cydAB expression negatively in both aerobiosis and anaerobiosis , but ArcA antagonizes its action in the absence of oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Taken together , these results strongly suggest that H-NS negatively regulates cydAB expression under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>Effect of ArcA and H-NS on induction of appY expression under oxygen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>H-NS represses expression both in aerobically and in anaerobically growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>In aerobically growing E. coli cells , H-NS represses nrdAB expression under all conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Examination of these data revealed that H-NS binds the yciGFEkatN locus and represses its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In this study , we found that H-NS represses the expression of acrEF and mdtEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973023" target="_blank">14973023</a>]<br></li>
<li>This occurred because H-NS represses acrEF , mdtEF , and emrKY expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>We assigned a role to the H-NS protein as a modulator of the expression of the nrdAB and nrdDG genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>We concluded that ﬁmB expression was signiﬁcantly induced in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>We show that H-NS modulates the expression of the nrdAB and nrdDG operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Although the degree of Irp derepression by Ahns was not especially high , at least under these experimental conditions , the effect was reproducible and may be interpreted as indicating that H-NS has the potential to repress Irp expression in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>The effect of H-NS was also tested by monitoring expression of nhaA9-9lacZ from P2 of plasmid pKRZ3 * in either OR200 or OR200D1 ( OR200 hns ) at various optical density ( OD ) values during both the logarithmic and stationary phases of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>The ﬁnding that H-NS is able to repress the expression of nrdAB and nrdDG in vivo prompted us to investigate its interaction with the nrdA , nrdH , and nrdD promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The magnitude of the effect is reasonable , since we are presumably observing an indirect effect of H-NS on the expression of the ilvIH operon and we do not know what other conditions might affect the synthesis of Lrp and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The effect of H-NS on cydA ± lacZ in vivo expression was subsequently determined under aerobic and anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>The expression of hdfR was negatively controlled by H-NS , which explains the negative effect on ﬂagellation of an H-NS mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>A single CRP binding site is located 2115.5 and 2215.5 bp upstream of the divergent transcription start sites for the papI and papB genes , respectively , and CRP positively regulates expression of these genes by inhibiting binding of the repressing H-NS protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>In addition to the bgl operon and other cryptic genes the H-NS protein also modulates the expression of a large number of active E. coli genes , usually by repressing transcription initiation ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>H-NS is best known for its ability to repress or silence the expression of a large number of genes in E. coli and other enterobacteria by acting at the level of transcription initiation of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21602341" target="_blank">21602341</a>]<br></li>
<li>These studies demonstrated that Ler directly activates transcription of the LEE genes by competing for binding with H-NS , which represses their expression in the absence of Ler .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12011002" target="_blank">12011002</a>]<br></li>
<li>It is generally assumed that H-NS represses expression by forming extended nucleoprotein complexes that interfere with transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066043" target="_blank">15066043</a>]<br></li>
<li>Here we observed that YdaM / YciR-mediated control is also connected to the action of the histone-like protein H-NS in csgD regulation : In the absence of H-NS , csgD transcription at 28 ° C is not only reduced ( conﬁrming a positive role of H-NS in csgD expression ) , but becomes nearly YdaM/YciR-independent ( Fig. 8A , compare lanes 4 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>A positive role for the environmental regulator H-NS was found for fes , cirA , hisJ , and srlE transcription , whereas it had a primarily negative effect on cysP and garP expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526711" target="_blank">17526711</a>]<br></li>
<li>In vitro transcription analysis of H-NS - and Fur-dependent control of ftnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Expression of the caiF gene is dependent on the same general regulatory proteins , H-NS , CRP , FNR , and transcription factor RpoN ( 11 ) , which control cai and ﬁx expression ( Table 2 ) ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></li>
<li>Regulators that affect ﬂhDC expression by controlling ﬂhDC transcription ( LrhA , OmpR-P , H-NS , cAMP/CAP ) or mRNA stability and translation ( CsrA ) positively ( + ) or negatively ( -- ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>As leuO expression is negatively regulated by H-NS and positively by LeuO itself ( Hommais et al. , 2001 ; Chen et al. , 2005 ) , this would further amplify the activating signal for cas gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20659289" target="_blank">20659289</a>]<br></li>
<li>In the present study we focused on the genetic control of the structural gene for H-NS and show that hns expression , in addition to its already known cold-shock regulation ( La Teana et al. , 1991 ) , is growth phase controlled and negatively autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>H-NS protein is involved in a number of cellular functions such as the modulated expression of numerous environmentally regulated genes The hns gene is not part of an operon in E. coli ( Ussery et al. , 1994 ; Atlung and Ingmer , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>The smtA gene is not essential for cell growth and its expression is positively regulated by H-NS , an Escherichia coli histone-like protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>Itappears to repress transcription of other genes as well.From DNA-binding studies , it has been found that the H-NS protein binds to curved DNA and can function as a nonspecific repressor of gene expression ( YAMADAet al. 1991 ; ZUBERet al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
<li>In support of this explanation , inactivation of the histone-like protein H-NS , which induces negative DNA supercoiling and binds to curved DNA regions , has been shown to abrogate the requirement of the rpoS gene for expression of curli by E. coli K-12 ( Olsén et al. , 1993 ; Arnqvist et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Additionally , we show that expression of slyA is temperature-dependent , autoregulated , and independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>While H-NS usually acts as a negative regulator of gene expression , hlyE is not the only example of a CRP-regulated gene whose expression is positively regulated by H-NS ; for example , expression of both the malT and csiD genes in E. coli is stimulated by H-NS ( 8 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>Second , we showed that expression of the E. coli slyA gene was temperature-regulated , being expressed at higher levels at 37 ° C than at 20 ° C and that this regulation was independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_375">
In addition , we show that expression of the E. coli slyA gene is tem-perature-regulated , positively autoregulated , and independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></label>
<input id="tu_similarity_375", type="checkbox">
<div class="TU_similar">
<li>In addition , we demonstrate that expression of the E. coli slyA gene is temperature-regulated , positively autoregulated , and independent of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li></div>
<li>Nevertheless , at least for the proU operon of S typhi-murium , [ 44 ] , the rrnB gene of E coli [ 48 ] , and the hns gene itself [ 13-15 ] , curved DNA sequences do define the site at which H-NS interacts to modulate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>This has been demonstrated in the proU system of E. coli and Salmonella typhimurium ( 34 , 43 ) , where H-NS binds within proV , the ﬁrst gene of the operon , and alters gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>H-NS interacts largely within the ﬁrst structural gene of the proU operon , proV , to strongly repress operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002611" target="_blank">8002611</a>]<br></li>
<li>Results gsp gene transcription in E.coli K-12 is silenced by H-NS To probe the expression of the two divergent operons of the main gsp locus , we inserted the promoterless lacZYA operon into the gspA and gspD genes using an R6K-based suicide vector ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11118204" target="_blank">11118204</a>]<br></li>
<li>With respect to temperature , it should be noted that the temperature-dependent modulation of gene expression by H-NS usually occurs in genes or operons that are downregulated at low temperature , which is not the case for the nrdAB operon , which is upregulated at low temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>The inclusion of an additional 853 base pairs of yicO sequence upstream of the ade promoter in pCN2534 did not significantly affect the regulation by H-NS but reduced gene expression 2-fold both in the wild type and hns mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12077137" target="_blank">12077137</a>]<br></li>
<li>Regulation of the hns gene The intracellular levels of H-NS are essentially constitutive and environmental signals such as osmolarity , temperature or anaerobic growth do not appear to influence hns expression [ 8 , 13 , 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The expression of over 80 % of these genes was induced in the hns mutant strain , which further supports the role of H-NS as a repressor of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>Moreover , the effect of H-NS on genes of the colanic acid operon could be mediated by the increased expression of their activator-encoding gene , rcsA ( Ebel and Trempy , 1999 ) , in the hns mutant ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>At present , it is not known if expression of the mlc gene is affected by H-NS and/or StpA , but evidently Mlc is subject to autoregulation and represses its own expression ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829919" target="_blank">9829919</a>]<br></li>
<li>tems in which H-NS represses transcription , appY gene expression in the wild-type strain was not affected by changes in osmolarity or by changes in temperature in the 30 to 428C range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>It is also involved in transcription termination ,20 and can bind and stabilise mRNA .21 As a transcriptional regulator , H-NS silences gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23661089" target="_blank">23661089</a>]<br></li>
<li>When H-NS is inactive , the expression of hdfR gene is increased , resulting in an overproduction of HdfR , which will reduce the transcription of ﬂhDC by binding to its upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Recently it was reported that expression of the gene cluster located near yhiUV is also increased by a deﬁcit of H-NS protein ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670992" target="_blank">12670992</a>]<br></li>
<li>Furthermore , we show that expression of fimB and timE is strongly influenced by the H-NS nucleoid protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7510658" target="_blank">7510658</a>]<br></li>
<li>Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-asso-ciated protein , H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></li>
<li>H-NS , another nucleoid-associated protein that is controlled by RpoS ( 10 ) , compacts DNA ( 22 ) and can modulate gene expression ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>gadX gene is under transcriptional control of H-NS and RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>In the present paper , we demonstrate that in vivo H-NS exerts a negative control on the transcriptional expression of its own gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934818" target="_blank">7934818</a>]<br></li>
<li>However , no link has yet been established between control of gene expression by H-NS and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>in a control experiment , the expression of hns ( the gene eoding ior the nueleoid-assoeiated protein H-NS ) was monitored under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7830553" target="_blank">7830553</a>]<br></li>
<li>Binding of H-NS to these sequences might explain the negative regulation of hns expression by its own gene product ( 20,21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8120010" target="_blank">8120010</a>]<br></li>
<li>H-NS , encoded by hns , is involved in the transcriptional regulation of gene expression , genetic recombination , and the topological organization of the chromosomal DNA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>]<br></li>
<li>In the first scenario , the interaction of H-NS • : , ith DNA would be influenced by specific environmental signals , such that it regulates gene expression in response to those signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Thus , H-NS does not seem to be restricted to a purely structural role but appears to influence gene expression , probably through its effect on DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Furthermore , the expression of the hdfR gene was shown to be negatively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Further , the expression of the orj30 gene , named smtA , was positively regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>To test whether expression of the smtA gene is regulated by H-NS or not , we prepared the smtA-1acZ fusion construct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8566713" target="_blank">8566713</a>]<br></li>
<li>C.F. ( 1990 ) Histone-iiko protein HI ( H-NS ) , DNA supercoiling , and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>( H-NS ) , DNA supercoiling , and gene expression in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Interestingly , these changes in gene expression were observed regardless of the presence of H-NS ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23028867" target="_blank">23028867</a>]<br></li>
<li>Role of H-NS in anaerobic regulation of appY gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>We investigated a possible functíon of H-NS in the regulatíon of the expression of its own structural gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>H-NS negatively modulates the expression of its own structural gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>The roles of additional regulatory proteins ( Lrp , H-NS ) in the control of csiD expression It seems that with the elucidation of the crucial role of cAMP-CRP and a perhaps modulatory role of negative DNA supercoiling , our understanding of the regulation of csiD is not yet complete .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>This implies that H-NS might indirectly participate in the regulation of GadE-mediated expression of gadA and gadBC by repressing gadE and/or components of the EvgA/S regulatory system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Previous reports utilizing E. coli strain W3110 convincingly showed that H-NS can also indirectly repress gadA and gadBC expression by inhibiting RpoS production ( 4 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>However , the expression of lrp is not regulated solely by Lrp but is also under the control of H-NS protein , as revealed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Therefore , it remains to be investigated if Lrp and H-NS bind the same sites on DNA and control lrp expression co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Given that gadX expression is also stationary-phase dependent , we investigated the involvement of both H-NS and RpoS in the transcriptional control of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976288" target="_blank">11976288</a>]<br></li>
<li>H-NS is also reported to inhibit gadX expression but appears capable of inhibiting gadA and gadBC expression independently of gadX control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446650" target="_blank">12446650</a>]<br></li>
<li>Under acidic condition , cAMP and Crp indirectly affect gadW gene expression by inhibiting the production of RpoS ( Ma et al. 2003 ) , where RpoS is also under control of H-NS ( Jung and Kim 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>H-NS also represses the expression of evgA and gadX ( Hommais et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Hommais et al. ( 12 ) compared the proteome and transcriptome of an H-NS-deﬁcient strain and its parent strain and reported that the expression of evgA is induced by H-NS deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>The expression of [ 3-galac-tosidase activity expressed by CU237/pMK1 was affected by tryptophan added to the medium , i.e. the level of [ ~ - galactosidase activity was markedly reduced when production of H-NS was appropriately induced by depletion of tryptophan in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437561" target="_blank">8437561</a>]<br></li>
<li>Wild-type cells are Bgl , i.e. , they can not utilize arbutin and salicin , at least under laboratory conditions , because the expression of the bgl genes is signiﬁcantly reduced by silencing elements that include DNA structural elements as well as the global repressor , H-NS ( 19 , 21 , 25 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>]<br></li>
<li>On the basis of this association with H-NS and the fact that Hsp70 proteins are typically induced under stress conditions , we examined hscA expression following cold shock , which induces H-NS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>H-NS autoregulates its own expression [ 13-15 ] , and the intraceilular concentration of H-NS does not appear to vary significantly in response to growth conditions , except that it is induced when cells are cold-shocked ( see below ; [ 16 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>The mechanism of H-NS 's function on specific genes is not quite clear , though it has been suggested to be due to its effects on DNA conformation or its ability to Biodegradative arginine decarboxylase and lysine decarboxylase , encoded by adi and cadA , respectively , are induced to maximal levels when Escherichia coli is grown anaerobically in rich medium at acidic pH. Mutants formed by transposon mutagenesis , namely , GNB725 , GNB729 , GNB88 , GNB824 , and GNB837 , exhibited considerably elevated expression at pH 8.0 compared with the corresponding parental strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8381784" target="_blank">8381784</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1996 , p. 3418 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology e Histone-Like Protein H-NS Acts as a Transcriptional Repressor for Expression of the Anaerobic and Growth Phase Th Activator AppY of Escherichia coli TOVE ATLUNG ,1 * SUSANNE SUND ,2 KIRSTEN OLESEN ,1 AND LONE BRØNDSTED2 Department of Chemistry and Life Sciences , Roskilde University , 4000 Roskilde ,1 and Department of Microbiology , Technical University of Denmark , DK-2800 Lyngby ,2 Denmark Received 12 October 1995/Accepted 9 April 1996 The transcriptional activator AppY is required for anaerobic and stationary-phase induction of the cyx-appA and hya operons of Escherichia coli , and expression of the appY gene itself is induced by these environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655536" target="_blank">8655536</a>]<br></li>
<li>In addition to a role in DNA compaction , H-NS is known to speci cents cally repress transcription of a whole set of genes by its ability to modulate DNA structure [ 53,54 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>In parallel , H-NS decreases acid resistance by repressing genes involved in general acid stress response ( e.g. dps encoding the DNA protection during starvation protein ) or in three amino-acid decarboxylase systems , dependent on glutamate , lysine and arginine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20435136" target="_blank">20435136</a>]<br></li>
<li>Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Rather than directly affecting the switching process per se , it appeared that the orientation of this element was affected by the differential temperature modulation of H-NS of the fimB and fimE genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>These results suggest that class Ia ( nrdAB ) and III ( nrdDG ) but not Ib ( nrdHIEF ) RNR genes could be modulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873909" target="_blank">23873909</a>]<br></li>
<li>Under non-cold shock conditions , H-NS acts as a direct transcriptional repressor of the osmotically induced proU operon ( 42 ) as well as of numerous other genes ( reviewed in reference 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665466" target="_blank">7665466</a>]<br></li>
<li>Approximately half the proteins encoded by genes controlled by H-NS were located in or associated with membranes or were present in the periplasmic space ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>It is remarkable that , for the H-NS family of proteins , two or more genes encoding members of the family can be present in the same bacterial cell at the same time .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458420" target="_blank">15458420</a>]<br></li>
<li>This suggests that overexpression of either GadX or GadE alone , in an FB8 genetic background , is not sufﬁcient to induce expression of genes involved in the bacterial response to low pH when cells are grown at neutral pH. Overexpression of gadX signiﬁcantly induced the expression of 48 genes of the H-NS regulon ( Table 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>Of 30 genes induced more than eightfold by EvgA overexpression , 18 , including ydePO and yhiE , are also induced by H-NS deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>