<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - HipB</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">HipB</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">HipB transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>Reaction mixtures depicted in the lanes contained the indicated concentrations of active HipB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>The final concentration of HipB in each reaction mixture was 0 , 0.013 , 0.026 , 0.053 , 0.13 , 0.26 , 0.53 , and 1.3 , Â± g/ml for lanes 1 to 8 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>There were no obvious sites of hypersensitive cleavage caused by the presence of HipB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>One stable-pair state is characterized by high HipA and low HipB concentrations , corresponding to the dormant persister state ( Falla and Chopra , 1998 ; Korch et al. , 2003 ; Correia et al. , 2006 ; Vazquez-Laslop et al. , 2006 ; Rotem et al. , 2010 ) ; another is characterized by low HipA and high HipB concentrations , corresponding to the normal growing state ( Falla and Chopra , 1999 ; Korch et al. , 2003 ; Hansen et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>This corresponds to 0.008 , ug/ml of HipB , assuming that the HipA-HipB prep-amounts of HipA-HipB were added to-binding reaction mixtures containing the end-labeled BamHI-BglII restriction fragment ( 3.2 x -35 -1 0 00 * 0 000 TTTCCAGATTGACTTATCCTCAC CTTATAATATCCCCTTAAGCGGATAAACTTGCTGTGGACGTATGACATGATGAGCTTTCAGAAGATCT - ' 3 01 v - 02 7 `` q - -- 000 -20 + ; 00000 0 +20 -40 +40 +60 000 Con5 servedC - TATCCNNNNNNNNGGATA -3 ' 3 ' - ATAGGNNNNNNNNCCTAT -5 ' 000 FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Although disruption of hipA slightly increased expression from the hip-lac fusion , in vitro studies suggest that HipA does not bind to the hip regulatory region directly but indirectly via HipB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Thus , HipA appears to be toxic in the absence of HipB , and an interaction between these gene products might be anticipated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Degradation of HipB would lead to release of previously bound HipA protein , which would in turn inhibit cell growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
<li>Microcolony analysis demonstrated that cell division in the propose that guillotining at dif destroys the expression of the adjacent hipBA genes and , in the absence of continued formation of HipB , HipA inhibits growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
<li>The multiple mechanisms for maintaining a low level of expression of the operon and little or no free HipA in the cell are probably related to the toxicity of HipA in the absence of HipB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Given that each strand of DNA has some phosphate groups , 54 Na + and 16 Cl counterions for the 2HipA + HipB2 + DNA model are added to achieve electroneutrality and to satisfy the experimental ionic strength , namely 100 mM for 2HipA + HipB dimer + DNA complex ( Schumacher et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>On the basis of the binding curve , 40-pM active HipB protein was required for .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>An increase of only about fourfold in HipB protein concentration is required to increase the bound fraction of DNA from 10 to 90 % ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Indeed , the binding of the HipA preparation was increased by even a small increment of HipB ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>These results will be helpful to understand the DNA molecule , as a regulative media , that might modulate the HipB dimer conformation , consequently increasing the interactions of HipB dimer with HipA proteins , which explains the mechanism of HipA sequestration in the nucleoid in detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>As expected , the structural change analysis as discussed in the Results section indicates that the DNA binding to the HipB dimer causes the extension of Î±2 / 3 helices in the HipB2-DNA interface ; furtherly , such movement of Î±2 / 3 helices structurally communicates to the Î±1 helix and induces the closeness of Î±1 helix to the corresponding HipA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Summarily , the DNA binding to the HipB dimer , as a regulative media , causes the extension of Î±2 / 3 of HipB dimer ; then such extension communicates the allostery to the closeness of Î±1 helices of HipB dimer to the HipA proteins , consequently increases the afï¬nity of HipB dimer for the HipA proteins , which can be a good example to understand the modulation of allostery of the protein -- protein interface by a DNA media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Therefore , HipB dimer + DNA complex in the normal growing state can interact with HipA protein to inhibit its kinase activity and persistence function through HipA sequestration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Dynamical ï¬uctuations and correlations from 2HipA + HipB2 model to 2HipA + HipB2 + DNA model To investigate dynamically the strong interaction between the HipA protein and HipB dimer induced by DNA binding , the dynamics of every residue in the HipBa subunit were determined and interpreted by the residue ï¬uctuations and correlations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Simultaneously , these simulations also explored that the thermodynamic binding order in a simulated physiological environment , that is , the HipB dimer ï¬rst bind to DNA to form HipB dimer + DNA complex , then capturing strongly the HipA proteins to form a ternary complex , 2HipA + HipB dimer + DNA , for sequestrating HipA in the nucleoid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>These results also propose a thermodynamic binding order , for the HipA protein , HipB dimer , and DNA molecule in a simulated physiological environment , that is , HipB dimer bind ï¬rst to DNA to form HipB2 + DNA complex ; then HipA protein binding to HipB2 + DNA complex to form HipA + HipB2 + DNA or 2HipA + HipB2 + DNA complex ; ï¬nally , the remaining HipA proteins combining with the HipB dimer to form the HipA + HipB2 or 2HipA + HipB2 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>The calculated data of free energies favor the binding of HipA to HipB2 + DNA complex over the HipB dimer In order to analyze the competitive binding properties of HipB dimer to DNA or to HipA protein , the binding free energies for the binding states of HipB dimer to HipA both in the presence and absence of DNA were calculated by using the MM-PBSA methodology when the average energies are converged .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>The two HipA proteins binding to the HipB2 + DNA complex simultaneously increase the afï¬nity of HipB dimer for DNA with the increased 1 energy of 23.06 kcal mol , which supports the experiment observation ( Schumacher et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>The DNA disturbance caused by binding of HipB dimer and HipA proteins The HipB dimer strongly binding to DNA at the DNA major groove is a driving force for the increase of interactions between the HipA protein and HipB dimer and the construction of allosteric communication network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Moreover , the HipB dimer and 2HipA + HipB2 complex , respectively , binding to DNA molecule cause the DNA helix bends of 72.1 Â° and 107.3 Â° towards the minor groove ( Figure 11 ( c ) ) , which may favor the hydrogen bond and the hydrophobic and ionic interactions in the HipA/HipB2-DNA interface to increase the afï¬nity of HipA/HipB protein for DNA molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">HipB protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>It belongs to COG1396 ( Clusters of Orthologous Groups of proteins ; http : / / www.ncbi.nlm.nih.gov/COG ) , which also contains the known antitoxins HipB and HigA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233923" target="_blank">20233923</a>]<br></li>
<li>Analysis of the interaction of HipB with the hip regulatory region was conducted with the following 32P-labeled DNA fragments : the BamHI-BglII fragment of pSP72SB , which contains bases -291 through +61 of hip ; the EcoRV-BglII fragment of pSP72SB , which contains bases -77 through +61 of hip ; and the BamHI-EcoRV fragment of pSP72SB , which contains bases -291 through -78 of hip ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>To investigate the extension of two Î±1 helices in the 2HipA + HipB2 + DNA model , the inï¬uence of DNA-binding on the structure of HipB dimer has been also analyzed through measuring the distances between two Î±2 / 3 helices , as the DNA-binding motif , in the HipB dimer over the simulation times for the 2HipA + HipB2 and 2HipA + HipB2 + DNA models .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Dynamical ï¬uctuations and correlations from 2HipA + HipB2 model to 2HipA + HipB2 + DNA model To investigate dynamically the strong interaction between the HipA protein and HipB dimer induced by DNA-binding , the dynamics of every residue in the HipBa subunit were determined and interpreted by the residue ï¬uctuations and correlations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>The movement of Î±1 helix results in the formations of new hydrogen bonds between the Î±1 helix of HipB and the residues Leu47 and Ser324 of HipA protein , which favors the combination of HipA proteins with the HipB dimer in the 2HipA + HipB2 + DNA model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Such movement of Î±1 helices results in the formations of extra hydrogen bonds between the residues Leu47 and Ser324 in the HipA proteins and the residues Asn15 and Gln12 of the Î±1 helices of the HipB dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>As expected , the structural change analysis as discussed in the Results section indicates that the DNA-binding to the HipB dimer causes the extension of Î±2 / 3 helices in the HipB2-DNA interface ; furtherly , such movement of Î±2 / 3 helices structurally communicates to the Î±1 helix and induces the closeness of Î±1 helix to the corresponding HipA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>It can be seen from the motion correlation analysis as discussed previously that the motions of Î±2 / 3 helices , around the residues Gln28 , Ser29 , Lys38 , Gln39 , Ala40 , Thr41 , and Ser43 , of HipB subunit correlate to the motions of Î±1 helix , around the residues Gln12 and Asn15 , of HipB subunit with the cross-correlation coefï¬cients of > 0.5 in the HipA-HipB2 interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>On the other hand , the motion correlations of the Î±4 helix with the Î±2 / 3 and Î±1 helices in the HipB subunit as discussed previously predict that the hydrogen bond formations between the residues Asn51 , Thr53 , Thr56 of Î±4 helix , and the base pairs C9 : G36 , T10 : A35 of DNA in the HipB2-DNA interface cause the interaction of Î±1 helix with the HipA protein in the HipA -- HipB2 interface through the allosteric communication of Î±2 / 3 helices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Statistical analysis using the methods of Brennan and Matthews ( 6 ) and Dodd and Egan ( 9 ) , as well as computer analysis ( 10 ) , support the view that HipB contains a helix-turn-helix motif near the N terminus and is a Cro-like , DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>HipB protein contains four Î± helices and one Î² strand with topology Î±1 ( residues 10 -- 24 ) Î±2 ( 28 -- 35 ) Î±3 ( 39 -- 48 ) Î±4 ( 54 -- 65 ) Î²1 ( 66 -- 72 ) conformation , in which Î±2 and Î±3 helices are a canonical helix-turn-helix motif capable of binding DNA through major groove contacts ( Black et al. , 1991 ; Schumacher et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Moreover , the C-terminal residues 75 -- 88 , which are near by the Î²1 sheet in the X-ray densities of HipB protein , are missing ( Schumacher et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>These studies showed that HipB forms dimers in solution and that the dimeric state is not caused by disulfide bonds between the cysteine residues of the mono-mers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Therefore , since dimethyl sulfate specifically methylates the N-7 position of guanine residues in the major groove and the N-3 position of adenine residues in the minor groove ( 31 ) , the interaction of HipB within an individual operator appears to occur in adjacent major grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Similarly , two HipB proteins in the HipB dimer of this model are respectively assigned as HipBa and HipBb subunits ; each of which contains the residues 1 -- 88 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>Examination of the overall spacing between operators and protected residues suggests that HipB binding occurs on the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>HipB interaction with specific purine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>HipB primarily modified the methylation of guanine resi-dues ( Fig. 4 ) ; 23 of the 27 affected guanine residues were protected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>The transcribed strand contains 12 guanine residues which were protected by HipB from methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>In addition , HipB enhanced the methylation of 2 guanine residues and 5 adenine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>The purine residues that were protected from methylation ( - ) and the purine residues whose methylation was enhanced in the presence of HipB on the basis of densitometric scanning ( + ) are indicated to the right of the gels 10 - li M ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>The 10 adenine residues that were affected by HipB were hypermethylated ; none was protected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Because the afï¬nity of HipB for HipA or for hipBA operator DNA was not altered when the last 16-residue residues ( Ala73-Trp88 ) of HipB were removed , the coordinates of HipB subunit containing the residues Met1-Asp74 were taken from the X-ray crystal structure ( PDB entry 3DNV ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>The missing residues ( Met1 , Asp135 -- 145 , Gly185-Asp195 , Gly438-Lys440 in the HipA segment , and Met1-Ser3 in the HipB subunit ) were repaired using the loop search method in the Swiss-PdbViewer ( also known as DeepView , http://spdbv.vital-it.ch/ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">HipB evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>It belongs to COG1396 ( Clusters of Orthologous Groups of proteins ; http : / / www.ncbi.nlm.nih.gov/COG ) , which also contains the known antitoxins HipB and HigA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233923" target="_blank">20233923</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by HipB</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Microcolony analysis demonstrated that cell-division in the propose that guillotining at dif destroys the expression of the adjacent hipBA genes and , in the absence of continued formation of HipB , HipA inhibits growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
<li>Catabolite response Deoxirribose assimilation D 87 135 Crp w 27 PeÌrez - Rueda and Collado-Vides ( 2000 ) 220 -- R 235 10 DeoR DicA 35 w PeÌrez - Rueda and Collado-Vides ( 2000 ) -- n/d 0 -- -- n/d HLH R 4 DicC DnaA 0 Transcription and replication regulation Positive regulator of cytochrome c552 77 0 -- PeÌrez - Rueda and Collado-Vides ( 2000 ) 41 -- n/d DniR 0 Kajie et al. ( 1991 ) 209 8 215 -- A Ebp FhlC 77 m PeÌrez - Rueda and Collado-Vides ( 2000 ) PeÌrez - Rueda and Collado-Vides ( 2000 ) n/d 0 -- 7 -- PeÌrez - Rueda and Collado-Vides ( 2000 ) Schnetz ( 2002 ) n/d R FhlD FrvR 0 -- Phosphoenolpyruvate sugar phosphotransferase system operon regulation Iron assimilation 8 0 -- -- 128a Fur R w Achenbach and Yang ( 1997 ) 107 460 35 Carbon sources assimilation Nguyen and Saier ( 1995 ) ; Weickert and Adhya ( 1992 ) 388 GalR/LacI 55 R c -- 13 GcvR GntR Rigali et al. ( 2002 ) Glycine cleavage operon repression 0 R 735 775 Ghrist and Stauffer ( 1998 ) 96 R w Yamada and Saier ( 1988 ) Activator of the glucitol operon in E. coli High-frequency persistence of the lethal effects of inhibition of either DNA or peptidoglycan-synthesis regulation ( HipAB ) -- 8 GutM 0 A -- Black et al. ( 1991 ) -- 1 HipA 1 0 n/d -- n/d HipB HtgA Black et al. ( 1991 ) Missiakas et al. ( 1993 ) 87 0 -- -- -- n/d 7 Heat shock regulation 0 -- HycA HypF 6 -- n/d ZnF A 0 -- 0 Bouhouche et al. ( 2000 ) Gui et al. ( 1996 ) 27 -- Glyoxylate bypass operon Integration host factor 31 106 161 w R -- IclR Ihf -- 15 8 PeÌrez - Rueda and Collado-Vides ( 2000 ) Campoy et al. ( 2002 ) -- D 43 -- DNA-repair 54 LexA R w 102 279 Miller and Bassler ( 2001 ) Schell ( 1993 541 LuxR/UhpA m A Quorum sensing 1548 LysR 1271 Amino acid biosynthesis regulation D w 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>E. coli hipA encodes a 440-residue protein , HipA , which is co-transcribed with a smaller upstream gene , hipB , encoding an 88-residue protein HipB ( Black et al. , 1991 ; Black et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">HipB binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Matrix Consensus ( IUPAC ) order sites Total IC IC per FPR50 % ( LOO ) FPR90 % ( LOO ) FPR100 % ( LOO ) FPR50 % FPR90 % FPR100 % ( matrix sites ) LOO/Matrix sites ( FPR 50 % ) ( matrix sites ) ( matrix sites ) colum n 1 0 1 1 e e e e mem mem mem mem TrpR CRP CysB NanR 24 24 30 20 6 tyGtACtmGykaACtaGTaCratr aaawwtgtGayryagaTCACawww gGAavGrrrtaAyKrmwtrmcarakykm ktATAmMwGktataMmrGww 13.62 7.29 10.67 10.28 0.57 0.30 0.36 0.51 6.67 E 1.38 E 9.29 E 1.53 E + + + + 03 00 03 05 1.30 E-10 6.40 E-242 1900 8.30 E-07 1.00 E 6.20 E 2.60 E 2.30 E -08 -05 -04 -06 7.70 E-06 3.70 E-03 1.40 E-01 1.60 E-03 7.70 E-06 2.80 E-02 1.40 E-01 1.60 E-03 1.50 E 4.50 E 2.80 E 1.50 E -12 -05 -08 -11 1.50 E 2.70 E 3.20 E 6.40 E -09 -03 -05 -07 1.50 E 2.10 E 3.20 E 6.40 E -09 -02 -05 -07 198 8 6 kt 1 1 0 meme meme LexA HipB FNR 21 4 72 22 18 20 wwtrCTGtayatamammCAGya ATCCsskagmGCGGATAA tdywwwtTGaTwwmraTCaa 11.65 10.95 6.94 0.53 0.61 0.35 1.33 E 3.33 E 1.71 E + + + 01 03 0 2.60 E-46 0.00022 2.09 E-71 2.00 E 5.00 E 1.30 E -08 -08 -04 4.60 E-04 4.30 E-06 5.30 E-03 3.10 E-03 4.30 E-06 1.80 E-02 1.50 E 1.50 E 7.60 E -09 -11 -05 1.50 E 3.10 E 2.90 E -05 -09 -03 1.50 E 3.10 E 9.80 E -04 -09 -03 consensus B C Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>( A ) Sequence logos from different matrices representing the binding motif for the TF HipB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>In addition , the areas protected by HipB encompass the three inverted-repeats with the conserved motif , TATCCN8GGATA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Thus , it is anticipated that HipB binds to each operator site as an oligomer of at least two subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Additional data supporting the notion that HipB binds cooperatively is the observation that the affinity of HipB for a fragment containing 01 , 02 , and 03 , the EcoRV-BglII ( bp -77 to +61 of hip ) fragment , is 10-fold less than for a fragment containing all four operator sites , the BamHI-BglII ( bp -291 to +61 of hip ) fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>HipB binds with a Kapp ( K apparent ) of 40 pM to four operator sites with the conserved sequence TATCCN8GGATA ( N represents any nucleotide ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>This identification was supported by in vitro studies demonstrating that HipB binds to four operator sites , with the conserved inverted-repeat sequence TATCCN8GGATA , protecting a region of approximately 100 bp from DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Moreover , HipB protein , in this state , either binds to HipA forming a stable non-toxic complex 2HipA + HipB dimer or binds tightly to a hipB operator with the consensus sequence TATCC ( N ) 8GGATA ( N = nucleotide ) forming a HipB dimer + DNA complex ( Black et al. , 1991 ; Black et al. , 1994 ; Schumacher et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
<li>The 2HipA + HipB2 + DNA model consists of a 22 bp palindromic DNA fragment , 5 ' - d ( ACTATCCCCTTAAGGGGATAGT ) -3 ' , two HipA proteins , and a HipB dimer ( shown in Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24089363" target="_blank">24089363</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by HipB</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Analysis of the role of HipA in the regulation of the hip operon is complicated by the toxicity of HipA in the absence of HipB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Third , overexpression of both hipB and hipA by using the Studier T7 expression system ( 35 ) results in a protein extract in which HipA is found exclusively associated with HipB in an equimolar ratio .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Studies by Black et al. ( 1994 ) have demonstrated that the expression of hipBA is autoregulated by HipB protein , that HipA protein is present in a tight complex with HipB , and that free HipA is toxic to cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
<li>Microcolony analysis demonstrated that cell division in the propose that guillotining at dif destroys the expression of the adjacent hipBA genes and , in the absence of continued formation of HipB , HipA inhibits growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
<li>Owing to degradation of the hipBA genes , which are immediately adjacent to dif , HipB protein would no longer be produced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>