<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - HU</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">HU</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">HU transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>When the parental strain GB5 and its derivatives were grown in this medium , it was supplemented with L-histidine , L-me-thionine , and uracil , all at a concentration of 50 jig/ml ( LB-MHU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=342494" target="_blank">342494</a>]<br></li>
<li>Fur J Biochem 77 , 69-75 9 PetersenHU , Wikman FP , SiboskaGE , Worm-LeonhardH , Clark BFC ( 1983 ) Interaction between methionine accepting tRNAs and proteins during initiation of procaryotic translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1720674" target="_blank">1720674</a>]<br></li>
<li>Since THU was shown to exhibit no significant affinity for Me-GlcU and since it is suspected that Me-GlcU has its own transport system , we attempted to elucidate this puzzling point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>Interference with Murein Turnover Has No Effect on Growth but Reduces b-Lactamase Induction in Escherichia coli ANGELIKA R. KRAFT , JULIA PRABHU , ASTRID URSINUS , AND JOACHIM-VOLKER HÖLTJE * Abteilung Biochemie , Max-Planck-Institut für Entwicklungsbiologie , 72076 Tübingen , Germany Received 19 July 1999/Accepted 21 September 1999 Physiological studies of a mutant of Escherichia coli lacking the three lytic transglycosylases Slt70 , MltA , and MltB revealed that interference with murein turnover can prevent AmpC b-lactamase induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572120" target="_blank">10572120</a>]<br></li>
<li>Science269,496-512 Fraser CM , Gocayne JD , White 0 , Adams MD , Clayton RA , Fleischmann , RD , Bult CJ , Kerlavage AR , Sutton G , Kelley JM , Fritchmann J L , Weidman J. F. , Small KV , Sandusky M , Furhmann J , Nguyen D , Utterback TR , Saudek DM , Phillips CA , Merrick JM , Tomm J , Dougherty BA , Bott KF , Hu P , Lucier TS , Peterson SN , Smith HO , Hutchison III CA and Venter JC ( 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9468603" target="_blank">9468603</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_6">
Fraser , C. M. , Gocayne , J. D. , White , O. , Adams , M. D. , Clayton , R. A. , Fleischmann , R. D. , Bult , C. J. , Kerlavage , A. R. , Sutton , G. , Kelley , J. M. , Fritchman , J. L. , Weidman , J. F. , Small , K. V. , Sandusky , M. , Fuhrmann , J. L. , Nguyen , D. T. , Utterback , T. R. , Saudek , D. M. , Phillips , C. A. , Merrick , J. M. , Tomb , J. F. , Dougherty , B. A. , Bott , K. F. , Hu , P. C. , Lucier , T. S. , Peterson , S. N. , Smith , H. O. , Hutchison , C. A. , and Venter , J. C. ( 1995 ) Science 270 , 397 -- 403 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748244" target="_blank">9748244</a>]<br></label>
<input id="act_similarity_6", type="checkbox">
<div class="ACT_similar">
<li>Fraser , C. M. , J. D. Gocayne , O. White , M. D. Adams , R. A. Clayton , R. D. Fleischmann , C. J. Bult , A. R. Kerlavage , G. Sutton , J. M. Kelley , J. L. Fritchman , J. F. Weidman , K. V. Small , M. Sandusky , J. Fuhrmann , D. Nguyen , T. R. Utterback , D. M. Saudek , C. A. Phillips , J. M. Merrick , J. F. Tomb , B. A. Dougherty , K. F. Bott , P. C. Hu , T. S. Lucier , S. N. Peterson , H. O. Smith , C. A. Hutchison , 3rd , and J. C. Venter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>Fraser , C. M. , J. D. Gocayne , O. White , M. D. Adams , R. A. Clayton , R. D. Fleischmann , C. J. Bult , A. R. Kerlavage , G. Sutton , J. M. Kelley , J. L. Fritchman , J. F. Weidman , K. V. Small , M. Sandusky , J. Fuhrmann , D. Nguyen , T. R. Utterback , D. M. Saudek , C. A. Phillips , J. M. Merrick , J. F. Tomb , B. A. Dougherty , K. F. Bott , P. C. Hu , T. S. Lucier , S. N. Peterson , H. O. Smith , C. A. Hutchison III , and J. C. Venter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15353568" target="_blank">15353568</a>]<br></li></div>
<li>Journal of Medical Microbiology ( 2013 ) , 62 , 1728 -- 1734 DOI 10.1099 / jmm.0.062893-0 Characterization of novel ybjG and dacC variants in Escherichia coli Dongguo Wang ,13 Enping Hu ,23 Jiayu Chen ,33 Xiulin Tao ,23 Katelyn Gutierrez4 and Yongxiao Qi3 Correspondence Dongguo Wang 1 Department of Clinical Lab Medicine , Taizhou University affiliated Taizhou Municipal Hospital , Taizhou , PR China wdgtzs@163.com Enping Hu 2 Department of Urology Surgery , Taizhou University affiliated Taizhou Municipal Hospital , Taizhou , PR China enpinghu@163.com 3 Department of Lab Medicine , Medical College of Taizhou University , Taizhou , PR China 4 Department of Neurosurgery , Stanford University , Palo Alto , CA , USA A total of 69 strains of Escherichia coli from patients in the Taizhou Municipal Hospital , China , were isolated , and 11 strains were identified that were resistant to bacitracin , chloramphenicol , tetracycline and erythromycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23912810" target="_blank">23912810</a>]<br></li>
<li>ANALYTICAL BIOCHEMISTRY 233 , 76 -- 86 ( 1996 ) Article No. 0010 Excision of Oxidative Cytosine Modiﬁcations from g-Irradiated N y Escherichia coli Endonuclease III and Hu Whole-Cell Extracts1 D A b man J. Richard Wagner , * ,2 Benjamin C. Blount , † and Michael Weinfeld ‡ * Centre de Recherche en Gérontologie et Gériatrie , Hôpital d'Youville , 1036 Belvédère Sud , Sherbrooke , Québec , Canada J1H 4C4 and Laboratoire de Spectrométrie de Masse , Département de Médecine Nuclèaire et Radiobiologie , Faculté de Médecine , Université de Sherbrooke , Sherbrooke , Québec , Canada J1H 5N4 ; † Biomedical Mass Spectrometry Unit , University of New South Wales , Sydney 2052 , Australia ; and ‡ Radiobiology Department , Cross Cancer Institute , Edmonton , Alberta , Canada T6G 1Z2 Received July 11 , 1995 The possible release from g-irradiated DNA of eight oxidatively modiﬁed cytosine bases by Escherichia coli endonuclease III was examined by trimethylsilylation and gas chromatography/electron impact/mass spectrometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8789150" target="_blank">8789150</a>]<br></li>
<li>Multiple Roles for the Twin Arginine Leader Sequence of Dimethyl Sulfoxide Reductase of Escherichia coli * Received for publication , November 17 , 1999 , and in revised form , March 3 , 2000 Published , JBC Papers in Press , May 8 , 2000 , DOI 10.1074 / jbc.M909289199 Damaraju Sambasivarao , Raymond J. Turner ‡ , Joanne L. Simala-Grant § , Gillian Shaw , Jing Hu ¶ , and Joel H. Weineri From the Medical Research Council Group in Molecular Biology of Membrane Proteins , Department of Biochemistry , University of Alberta , Edmonton , Alberta T6G 2H7 , Canada Dimethyl sulfoxide ( Me2SO ) reductase of Escherichia coli is a terminal electron transport chain enzyme that is expressed under anaerobic-growth conditions and is required for anaerobic-growth with Me2SO as the terminal electron acceptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10801884" target="_blank">10801884</a>]<br></li>
<li>of Salmonella enterica Is a LuxR Homolog Mixed Microbial Communities SdiA That Detects BINDHU MICHAEL ,1 JENEE N. SMITH ,1 SIMON SWIFT ,2 † FRED HEFFRON ,3 AND BRIAN M. M. AHMER1 * Department of Microbiology , The Ohio State University , Columbus , Ohio 43210-12921 ; Institute of Infections and Immunity , University of Nottingham , Queen 's Medical Centre , Nottingham NG7 2UH , United Kingdom2 ; and Department of Molecular Microbiology and Immunology , Oregon Health Sciences University , Portland , Oregon 97201-30983 Received 13 April 2001/Accepted 19 June 2001 Proteins of the LuxR family detect the presence of N-acylhomoserine lactones ( AHLs ) and regulate transcription accordingly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544237" target="_blank">11544237</a>]<br></li>
<li>GARY REINESS * , HUEY-LANG YANG * , GEOFFREY ZUBAY * , AND MICHAEL CASHELt * Department of Biological Sciences , Columbia University , New York , N.Y. 10027 ; and t National Institute of Child Health and Hu Institutes of Health , Bethesda , Maryland 20014 Guanosine 5 ' - diphosphate 3 ' - diphosphate ( ppGpp ) and pppGpp accumulate when stringent strains of Eschertchsa colt are deprived of amino acids but do not accumulate in similarly deprived relaxed mutant strains ( 1 ) ; such mutants have recently been denoted as relA ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1103124" target="_blank">1103124</a>]<br></li>
<li>Yuan JS , Abercrombie LLG , Cao Y , Halfhill MD , Zhou X , Peng Y , Hu J , Rao MR , Heck GR , Larosa TJ , Sammons RD , Wang X , Ranjan P , Johnson DH , Wadl PA , ScheZer BE , Rinehart TA , Trigiano RN , Stewart CN Jr ( 2010 ) Functional genomics analysis of Horseweed ( Conyza canadensis ) with special reference to the evolution of non-target-site glyphosate resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22089966" target="_blank">22089966</a>]<br></li>
<li>Contents lists available at ScienceDirect Biochemical Pharmacology journal hom epage : ww w.elsevier.com/locate/biochempharm Testing double mutants of the enzyme nitroreductase for enhanced cell sensitisation to prodrugs : Effects of combining beneﬁcial single mutations Mansooreh Jaberipour a ,1 , Simon O. Vass a , Christopher P. Guise a ,2 , Jane I. Grove a ,3 , Richard J. Knox c , Longqin Hu d , Eva I. Hyde b , Peter F. Searle a , * a Cancer Research UK Institute for Cancer Studies , University of Birmingham , Birmingham B15 2TT , UK b School of Biosciences , University of Birmingham , Birmingham B15 2TT , UK c Morvus Technology Ltd. , Carmarthen SA32 8HZ UK d Department of Pharmaceutical Chemistry , Ernest Mario School of Pharmacy , Rutgers , The State University of New Jersey , 160 Frelinghuysen Road , Piscataway , NJ 08854 , USA A B S T R A C T A R T I C L E I N F O Article history : Prodrug activation gene therapy for cancer involves expressing prodrug-activating enzymes in tumour cells , so they can be selectively killed by systemically administered prodrug .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19665450" target="_blank">19665450</a>]<br></li>
<li>Searle , P. F. , Chen , M. J. , Hu , L. , Race , P. R. , Lovering , A. L. , Grove , J. I. , Guise , C. , Jaberipour , M. , James , N. D. , Mautner , V. , Young , L. S. , Kerr , D. J. , Mountain , A. , White , S. A. , and Hyde , E. I. ( 2004 ) Nitroreductase : a prodrug-activating enzyme for cancer gene therapy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19580253" target="_blank">19580253</a>]<br></li>
<li>( 14 ) Miller , M. J. ; Zhu , H. ; Xu , Y. ; Wu , C. ; Walz , A. J. ; Vergne , A. ; Roosenberg , J. M. ; Moraski , G. ; Minnick , A. A. ; McKee-Dolence , J. ; Hu , J. ; Fennell , K. ; Dolence , E. K. ; Dong , L. ; Franzblau , S. ; Malouin , F. ; Möllmann , U. Biometals 2009 , 22 , 61 − 75 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23098193" target="_blank">23098193</a>]<br></li>
<li>Skerker JM , Leon D , Price MN , Mar JS , Tarjan DR , Wetmore KM , Deutschbauer AM , Baumohl JK , Bauer S , Ibáñez AB , Mitchell VD , Wu CH , Hu P , Hazen T , Arkin AP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25063650" target="_blank">25063650</a>]<br></li>
<li>DeSantis , T.Z. , Hugenholtz , P. , Larsen , N. , Rojas , M. , Brodie , E.L. , Keller , K. , Huber , T. , Dalevi , D. , Hu , P. and Andersen , G.L. ( 2006 ) Greengenes , a chimera-checked 16S rRNA gene database and workbench compatible with ARB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504983" target="_blank">21504983</a>]<br></li>
<li>Hu , P. , Janga , S.C. , Babu , M. , Diaz-Mejia , J.J. , Butland , G. , Yang , W. , Pogoutse , O. , Guo , X. , Phanse , S. , Wong , P. et al. ( 2009 ) Global functional atlas of Escherichia coli encompassing previously uncharacterized proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22411911" target="_blank">22411911</a>]<br></li>
<li>Hu , P. , Janga , S. C. , Babu , M. , Díaz-Mejía , J. J. , Butland , G. , Yang , W. , Pogoutse , O. , Guo , X. , Phanse , S. , Wong , P. , Chandran , S. , Christopoulos , C. , Nazarians-Armavil , A. , Nasseri , N. K. , Musso , G. , Ali , M. , Nazemof , N. , Eroukova , V. , Golshani , A. , Paccanaro , A. , Greenblatt , J. F. , Moreno-Ha - 25 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>( 13 ) Hu , P. ; Janga , S. C. ; Babu , M. ; Diaz-Mejia , J. J. ; Butland , G. ; Yang , W. ; Pogoutse , O. ; Guo , X. ; Phanse , S. ; Wong , P. ; Chandran , S. ; Christopoulos , C. ; Nazarians-Armavil , A. ; Nasseri , N. K. ; Musso , G. ; Ali , M. ; Nazemof , N. ; Eroukova , V. ; Golshani , A. ; Paccanaro , A. ; Greenblatt , J. F. ; Moreno-Hagelsieb , G. ; Emili , A. Global functional atlas of Escherichia coli encompassing previously uncharacterized proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21210718" target="_blank">21210718</a>]<br></li>
<li>; Getzoff , E. J. ; Hu , P. ; Herzfeldt , B. ; Roos , R. P. ; Wagner , C. ; Deng , G. ; Soriano , E. ; Smith , C. ; Parge , H. E. ; Ahmed , A. ; Roses , A. D. ; Hallewell , R. A. ; Pericak-Vance , M. A. ; Siddique , T. Science 1993 , 261 , 1047 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11800609" target="_blank">11800609</a>]<br></li>
<li>Todesco , M. , Balasubramanian , S. , Hu , T.T. , Traw , M.B. , Horton , M. , Epple , P. , Kuhns , C. , Sureshkumar , S. , Schwartz , C. , Lanz , C. et al. ( 2010 ) Natural allelic variation underlying a major ﬁtness trade-off in Arabidopsis thaliana .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22323519" target="_blank">22323519</a>]<br></li>
<li>For example , the lactose and arabinose promoters are known to be strictly `` on/off '' ( Siegele and Hu , 1997 ) ; however , intermediate levels of induction are clearly possible with these promoters ( Guzman et al. , 1995 ) and are attributed to induction in only part of a cell population .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10441348" target="_blank">10441348</a>]<br></li>
<li>Retailleau , P. , Huang , X. , Yin , Y.H. , Hu , M. , Weinreb , V. , Vachette , P. , Vonrhein , C. , Bricogne , G. , Roversi , P. , Ilyin , V. , et al. , ( 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18713004" target="_blank">18713004</a>]<br></li>
<li>Wu , C.H. , Yeh,L.-S.L. , Huang , H. , Arminski , L. , Castro-Alvear , J. , Chen , Y. , Hu , Z. , Kourtesis , P. , Ledley , R.S. , Suzek , B.E. et al. ( 2003 ) The Protein Information Resource .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15608167" target="_blank">15608167</a>]<br></li>
<li>Hu et al. ( 43 ) demonstrated that the rate of ATP hydrolysis by MinD is independent of the concentration of MinE ; however , the lag phase depends on MinE concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23012351" target="_blank">23012351</a>]<br></li>
<li>In particular , MinC is proposed to inhibit FtsZ -- the cytoskeletal framework of divisome assembly ( Hu et al. , 1999 ; Hu and Lutkenhaus , 2000 ; Shen and Lutkenhaus , 2009 , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25271375" target="_blank">25271375</a>]<br></li>
<li>The effector of the Min system is MinC , which blocks cell division by preventing Z ring formation ( de Boer et al. , 1989 , Hu et al. , 1999 , de Boer et al. , 1990 , Bi & Lutkenhaus , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>MinC blocks FtsZ polymerization and Z-ring formation ( Hu et al. , 1999 ; Pichoff and Lutkenhaus , 2001 ; Johnson et al. , 2002 ) , and it is the pole-to-pole oscillation of this division inhibitor that is proposed to direct assembly of the division apparatus to mid-cell ( Hale et al. , 2001 ; Howard et al. , 2001 ; Meinhardt and de Boer , 2001 ; Kruse , 2002 ; Huang et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15165230" target="_blank">15165230</a>]<br></li>
<li>When bound to ATP , MinD dimerizes and binds to the membrane ( Lackner et al. , 2003 , Hu et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>Both MinD and MinE bind the membrane via amphipathic helices that are critical to Min oscillation and divisome positioning in vivo ( Hu and Lutkenhaus , 2003 ; Szeto et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24930948" target="_blank">24930948</a>]<br></li>
<li>MinD is a peripheral membrane protein with ATPase activity that binds MinC and recruits it to the cell membrane ( de Boer et al. , 1991 ; Hu & Lutkenhaus , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12368455" target="_blank">12368455</a>]<br></li>
<li>MinC requires MinD for full activity ; in part , because MinD recruits MinC to the membrane ( Hu & Lutkenhaus , 2001 , Hu et al. , 2003 , Lackner et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>MinE does this by stimulating the pole to pole oscillation of MinC/MinD through its ability to stimulate the ATPase activity of MinD and thus , the release of MinD from the membrane ( Raskin & de Boer , 1999b , Raskin & de Boer , 1999a , Fu et al. , 2001 , Hale et al. , 2001 , Hu & Lutkenhaus , 2001 , Hu et al. , 2002 , Hu & Lutkenhaus , 1999 , Shih et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>MinC and MinD form a division inhibitor complex that rapidly oscillates from one cell pole to the other ( Hu and Lutkenhaus , 1999 ; Raskin and de Boer , 1999a , b ; Shih et al. , 2003 ) , in a process that is driven by interactions of MinD with ATP , itself , the membrane and MinE ( Hu and Lutkenhaus , 2001 ; Hu et al. , 2002 ; 2003 ; Suefuji et al. , 2002 ; Lackner et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15165230" target="_blank">15165230</a>]<br></li>
<li>Repression of XPR-IacZ expression was calculated as : 100-IOOX ( 0-galactosidase units in the presence of repressor or fusion protein / $ - galactosidase units in the absence of repressor ) ( Hu et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>However , we also observed that increased acetylation of RcsB in K154 represses transcription of rprA gene as described in the same study ( Hu et al , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25518064" target="_blank">25518064</a>]<br></li>
<li>a Results Chimeric XNtrC proteins are active in vivo and in vitro To test the dimerization properties of various response-regulator domains , we used the X system introduced by Hu et al. ( 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Although the exact function of MukB , the gene product of mukB , is not known , experimental evidence and its similarity in domain organization to eukaryotic SMC ( structural maintenance of the chromosome ) proteins , strongly suggest a positive role in chromosome condensation ( Niki et al. , 1992 ; Hu et al. , 1996 ; Hirano and Hirano , 1998 ; Melby et al. , 1998 ; Weitao et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931285" target="_blank">10931285</a>]<br></li>
<li>The MinD ATPase dimerizes in the presence of ATP and binds membrane via an amphipathic helix at its C-terminus ( Hu and Lutkenhaus , INTRODUCTION * Correspondence to : Kiyoshi Mizuuchi Laboratory of Molecular Biology , National Institute of Diabetes , and Digestive and Kidney Diseases , National Institutes of Health Bethesda , MD 20892-0540 , USA kmizu@helix.nih.gov .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24930948" target="_blank">24930948</a>]<br></li>
<li>Concentration dependent effect of MinCC/MinD on Z rings Overproduction of MinCC in the presence of MinD inhibits assembly of Z rings ( Shiomi & Margolin , 2007 ) even though MinCC does not affect the polymerization of FtsZ in vitro ( Hu & Lutkenhaus , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_44">
Hu , Z. , Gogol , E. P. , and Lutkenhaus , J. ( 2002 ) Dynamic assembly of MinD on phospholipid vesicles regulated by ATP and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23012351" target="_blank">23012351</a>]<br></label>
<input id="act_similarity_44", type="checkbox">
<div class="ACT_similar">
<li>[ PubMed : 10611296 ] Hu Z , Gogol EP , Lutkenhaus J. Dynamic assembly of MinD on phospholipid vesicles regulated by ATP and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15165230" target="_blank">15165230</a>]<br></li>
<li>[ PubMed : 10869074 ] Hu Z , Gogol EP , Lutkenhaus J. Dynamic assembly of MinD on phospholipid vesicles regulated by ATP and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24930948" target="_blank">24930948</a>]<br></li>
<li>[ PubMed : 11278571 ] Hu Z , Gogol EP , Lutkenhaus J. Dynamic assembly of MinD on phospholipid vesicles regulated by ATP and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li></div>
<li>Also , MinDΔ10 , which lacks the membrane targeting sequence , increases the toxicity of MinC , just not as well as the full length MinD ( Hu & Lutkenhaus , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>Based on the demonstration that MinE can activate the ATPase activity of MinD in vitro , it was proposed that the E-ring is responsible for the disassembly of the polar zone and that this is accomplished by the MinE-mediated activation of the ATPase activity of MinD molecules at the edge of the polar zone ( Hu and Lutkenhaus , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093736" target="_blank">12093736</a>]<br></li>
<li>Hu , Z. and Lutkenhaus , J. ( 2001 ) Topological regulation of cell division in E.coli : spatiotemporal oscillation of MinD occurs through stimulation of its ATPase activity by MinE and phospholipid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093736" target="_blank">12093736</a>]<br></li>
<li>[ 13 ] Hu Z. , Lutkenhaus J. , Topological regulation of cell division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11254986" target="_blank">11254986</a>]<br></li>
<li>Allegrucci M , Hu FZ , Shen K , Hayes J , Ehrlich GD , Post JC , Sauer K. 2006 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25192989" target="_blank">25192989</a>]<br></li>
<li>Hong , W. , Jiao , W. , Hu , J. , Zhang , J. , Liu , C. , Fu , X. , Shen , D. , Xia , B. , and Chang , Z. ( 2005 ) Periplasmic protein HdeA exhibits chaperone-like activity exclusively within stomach pH range by transforming into disordered conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25391835" target="_blank">25391835</a>]<br></li>
<li>Hong , W. , W. Jiao , J. Hu , J. Zhang , C. Liu , X. Fu , D. Shen , B. Xia , and Z. Chang .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17259322" target="_blank">17259322</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17085547" target="_blank">17085547</a>]<br></li>
<li>[ 29 ] M.L. Hu , A.L. Tappel , Potentiation of oxidative damage to proteins by ultraviolet-A and protection by antioxidants , Photo-chem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>Kuhn , M. L. , Zemaitaitis , B. , Hu , L. I. , Sahu , A. , Sorensen , D. , Minasov , G. , Lima , B. P. , Scholle , M. , Mrksich , M. & other authors ( 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>[ 27 ] R. Golbik , H. Neef , G. Hu `` bner , St. Ko `` nig , L. Meshalkina , G.A. Kochetov , A. Schellenberger , Bioorg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9655909" target="_blank">9655909</a>]<br></li>
<li>Triola , G. ; Wetzel , S. ; Ellinger , B. ; Koch , M. ; Hu -RCB- bel , K. ; Rauh , D. ; Waldmann , H. Bioorg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21831641" target="_blank">21831641</a>]<br></li>
<li>2 D. Sambasivarao , H. A. Dawson , J. Hu , and J. H. Weiner , manu-script in preparation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10801884" target="_blank">10801884</a>]<br></li>
<li>Sambasivarao , D. , R. J. Turner , J. L. Simala-Grant , G. Shaw , J. Hu , and J. H. Weiner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980457" target="_blank">16980457</a>]<br></li>
<li>Nakayama , S. , A. Kushiro , T. Asahara , R. Tanaka , L. Hu , D. J. Kopecko , and H. Watanabe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17122336" target="_blank">17122336</a>]<br></li>
<li>Sun , J. X. ; Hu , J. Y. ; Peng , H. ; Shi , J. ; Dong , Z. Chemosphere 2012 , 7 , 37 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24882676" target="_blank">24882676</a>]<br></li>
<li>Chen , L. , Walker , D. , Sun , B. , Hu , Y. , Walker , S. , and Kahne , D. ( 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15157881" target="_blank">15157881</a>]<br></li>
<li>Meng , F. , Chen , X. , Hu , Y. , Tang , H. , Dang , W. and Zhou , J. ( 2010 ) Sua5p is required for telomere recombination in Saccharomyces cerevisiae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23258706" target="_blank">23258706</a>]<br></li>
<li>Steen , E.J. , Kang , Y. , Bokinsky , G. , Hu , Z. , Schirmer , A. , McClure , A. , Del Cardayre , S.B. , Keasling , J.D. , 2010 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23026122" target="_blank">23026122</a>]<br></li>
<li>The control set of proteins contained HRTHBD , MYSLG , LZBPT4 , AZBR , XNCHDC , R5EC7 , LPHUMC , CCBN , IHKREV , CSBQ , ISCHT , LZBA , TISY , CVJB , JGECA , RCBPL , VCTNS , NTSR3C , IPPG , VBHU , KLBOI , CRBO2 , CYBOG2 , SYBSYF , PSPGA , UHHUQ , QXBOGL , ACHUA1 , NFRT1 , BHTLD , 04RBP2 , MCSW , HSSFBM , APBPML , RKEGL , HLMSAB , CSMSAT , IVMSA1 , AXPG , R3EC1 , CGB07S , ISECX1 , QXHU3M , WMEC15 , DSBYC , BMTD , YVBPMS , FJEC , FOFVlR , XYOFS , QRHULD , TXEC , RGBYA2 , and FSFB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042779" target="_blank">3042779</a>]<br></li>
<li>Therefore , THU and enzymes II , III ' , and IV ' are under the same negative control exerted by the exuR regulatory gene , and the exuT gene also belongs to the exu regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>Good candidates for premutagenic lesions recognized by endo III and endo VIII are oxidized cytosines such as 5-OHC , 5-OHU , and Ug .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171429" target="_blank">9171429</a>]<br></li>
<li>The enzyme endonuclease III ( Endo III ) mainly excises pyrimidine-derived lesions such as thymine glycol , uracil glycol , dHU , 5,6-dihydrothymine , 5-hydroxy-5 ,6 - dihydrothymine , 5-hydroxy-5 ,6 - dihydrouracil , 5-hydroxyuracil , 5-hydroxycyto-sine , alloxan , urea , 6-hydroxy-5-hydrouracil , 5,6-dihydroxycy-tosine , 6-hydroxy-5-hydrocytosine , and products of fragmentation of these lesions ( 9 -- 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
<li>Several years ago , we identified a third E. coli oxida ¬ 1 The abbreviations used are : Fpg , formamidopyrimidine DNA glycosylase ; endo , endonuclease ; Tg , thymine glycol ; DHT , dihydrothymine ; AP , abasic ; redAP , reduced AP ; 5-OHC , 5-hydroxycytosine ; 5-OHU , 5-hydroxyuracil ; Pol I , polymerase I ; dRp , deoxyribose phosphate ; ds , double-stranded ; FAPY , formamidopyrimidine ; dTgMP , thymidine glycol 59-monophosphate ; dDHTMP , dihydrothymidine 59-monophosphate ; 5-OHdCHP , 5-hydroxycytidine 59-monophosphate ; 5-OHdVMP , 5-hy-droxyuridine 59-monophosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9405426" target="_blank">9405426</a>]<br></li>
<li>2 The abbreviations used are : AP , apurinic/apyrimidinic ; EcoNth , Escherichia coli endonuclease III ; Tg , 5,6-dihydroxy-5 ,6 - dihydrothymine , thymine glycol ; 5-OHC , 5-hydroxy-cytosine ; 5-OHU , 5-hydroxyuracil ; DHU , 5,6-dihydrouracil ; DHT , 5,6-dihydrothymine ; HhH , helix-hairpin-helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16096281" target="_blank">16096281</a>]<br></li>
<li>Various studies of the inactive conformer suggest that the DHU loop and stem are denatured and fully accessible to solvent and the anticodon loop , which is free in the native conformer , is inaccessible and probably bound to the TqC loop and/or stem ( Webb & Fresco , 1973 ; Uhlenbeck et al. , 1974 ; Wong et al. , 1973 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6178431" target="_blank">6178431</a>]<br></li>
<li>Specifically , if UP1 molecules are bound to the accessible single-stranded DHU arm of inactive tRNA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6178431" target="_blank">6178431</a>]<br></li>
<li>[ 6 ] C.A. Conlin , S.L. Ton , H. Hu , M. Segar , The apeE gene of Salmonella enterica serovar Typhimurium is induced by phosphate limitation and regulated by PhoBR , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>The activities of THU are expressed in percentage of the value induced in strain 2500 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6357945" target="_blank">6357945</a>]<br></li>
<li>1 Abbreviations : R , ribonucleotide reductase large subunit ; ara , L-arabinose ; , ribonucleotide reductase small subunit ; apo - 2 , ironfree form of 2 ; met - 2 , radical-free 2 that contains a diiron ( III/III ) cluster ; RNR , ribonucleotide reductase ; wt , wild type ; Y • , tyrosyl radical ; YfaE , 2Fe2S ferredoxin ; Abs , antibodies ; HU , hydroxyurea ; buffer A , 100 mM Tris , 150 mM NaCl , and 5 % glycerol ( pH 7.6 ) ; N-S - , N-terminal StrepII-tagged ; C-S - , C-terminal StrepII-tagged ; PVDF , poly ( vinylidene ) diﬂuoride ; TR , thioredoxin ; TRR , thioredoxin reductase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18314964" target="_blank">18314964</a>]<br></li>
<li>The 2 mM ATP ; 0.5 mM each of GTP , CTP , and UTP ; 0.1 crude cell lysate was clariﬁed by centrifugation mM each of dATP , dGTP , dCTP , and [ a-32P ] dTTP ( 100,000 g , 30 min , 4 ° C ) , and the supernatant fraction ( 50 -- 150 cpm/pmol ) ; 200 ng ( 600 pmol as nucleotide ) which contained the soluble recombinant DnaA protein pBSoriC ; 450 ng E. coli SSB ; 10 ng protein HU ; 400 ng was retained for further puriﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_78">
When added , the ﬁnal concentrations of HU , TyrR-RQ10 , and tryptophan were 400 nM , 200 nM , and 1 mM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></label>
<input id="act_similarity_78", type="checkbox">
<div class="ACT_similar">
<li>When added , the ﬁnal concentrations of HU , TyrR , and phenylalanine were 400 , 200 , and 1 mM , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li></div>
<li>HU constrains negative supercoifs in DNA ( Broyles and Pettijohn , 1986 ) and is inencoding gene ; hupA and hupB , genes encoding HU-2 and HU - 1 , respectively ; Hy , hygromycm B ; IHF , integration host factor ; himA and himD , genes encoding IHFa and IHFB , respectively ; kb , kiiobase ( sf or 1000 bp ; Km , kan ~ y ~ in ; LB , Lucia-Bertani ( medium ) ; nt , nucleotidefs ) ; ohgo , oligodeoxyribonucleotide ; PA , pofyacfylam ~ de ; PAGE , PA-gel etectro-phoresis ; R , resistance/resistant ; s , sensitivity/sensitive ; SDS , sodium dodecyl sulfate ; Tc , tetracycline ; trp $ genes encoding enzymes for tryptophan synthesis ; wt , wild type ; : : , novel junction ( fusion or insertion ) ; [ 1 , denotes plasmid-carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Biochemistry 2005 , 44 , 5373-5380 5373 Gal Repressor-Operator-HU Ternary Complex : Pathway of Repressosome Formation Siddhartha Roy , ‡ , § ,  Emilios K. Dimitriadis , ‡ , ⊥ Sudeshna Kar ,  Mark Geanacopoulos ,  , # Marc S. Lewis , ⊥ and Sankar Adhya * ,  Department of Biophysics , Bose Institute , Calcutta 700019 , India , and Laboratory of Molecular Biology , National Cancer Institute , Bioengineering and Physical Science , ORS , OD , and Laboratory of Molecular Biology , National Heart , Lung , and Blood Institute , National Institutes of Health , Bethesda , Maryland 20892 ReceiVed October 26 , 2004 ; ReVised Manuscript ReceiVed January 26 , 2005 ABSTRACT : DNA transaction reactions require formation of nucleoprotein complexes that involve multifaceted DNA-protein and protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Histone-like HU as a speciﬁc transcriptional-regulator : cofactor role in the repression of gal transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>Aki , T. , Choy , H. E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a specific transcriptional-regulator : co-factor role in repression of gal transcription by GAL repressor , Genes Cells 1 , 179-188 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_84">
References Aki , T. , Choy , H.E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional-regulator : cofactor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></label>
<input id="act_similarity_84", type="checkbox">
<div class="ACT_similar">
<li>Aki , T. , Choy , H.E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional-regulator : co-factor role in repression of gal transcription by GAL repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Aki , T. , Choy , H.E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional-regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Aki , T. , Choy , H.E. and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional-regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li></div>
<li>The EMBO Journal Vol .16 No. 12 pp. 3666 -- 3674 , 1997 Repressor induced site-speciﬁc binding of HU for transcriptional regulation Tsunehiro Aki and Sankar Adhya1 Laboratory of Molecular Biology , Division of Basic Sciences , National Cancer Institute , National Institutes of Health , Building 37 , Room 2E16 , 37 Convent Drive MSC 4255 , Bethesda , MD 20892-4255 , USA 1Corresponding author Transcription from two overlapping gal promoters is repressed by Gal repressor binding to bipartite gal operators , O and O , which ﬂank the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_89">
Aki , T. and Adhya , S. ( 1997 ) Repressor induced site-specific binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24598256" target="_blank">24598256</a>]<br></label>
<input id="act_similarity_89", type="checkbox">
<div class="ACT_similar">
<li>Aki , T. , and Adhya , S. ( 1997 ) Repressor induced site-specific binding of HU for transcriptional regulation , EMBO J. 16 , 3666-3674 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_91">
Aki , T. and Adhya , S. ( 1997 ) Repressor induced site-speci ® c binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></label>
<input id="act_similarity_91", type="checkbox">
<div class="ACT_similar">
<li>References Aki , T. , and Adhya , S. ( 1997 ) Repressor induced site speciﬁc binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Aki , T. and Adhya , S. ( 1997 ) Repressor induced site-speciﬁc binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Aki , T. , and Adhya , S. ( 1997 ) Repressor induced site-speciﬁc binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_95">
Repressor induced site-specific binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></label>
<input id="act_similarity_95", type="checkbox">
<div class="ACT_similar">
<li>Repressor induced sitespeci ® c binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Repressor induced sitespeciﬁc binding of HU for transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li></div>
<li>HU also binds to a few RNAs , but the full spectrum of its binding targets in the cell is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Since we omitted any cross-linking , there is a chance that RNAs are free to dissociate and bind HU in the cell lysate and RNAs that may normally be sequestered in the cell and unavailable for binding to HU are potentially released in the lysis process and then bind to HU nonspeciﬁcally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>The previously proposed reduction in helical pitch as a consequence of HU binding [ 13 ] would lead to the introduction of negative supercoils .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>No indication could be found for the existence of nucleosome-like structures induced by the binding of HU ( Dame and Goosen , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>An alternative explanation , that NagC requires GalR/S to enhance its binding , as proposed for HU binding at galE ( Kar and Adhya , 2001 ) seems not to be the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>An alternative explanation is that this stimulation is mediated by bending induced by the binding of single HU dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Cells generate new copies even in the absence of horizontal gene transfer , presumably by duplication and genetic drift ( for example , the HU α - and β-subunits , and H-NS and StpA ) and one assumes that those novel arrangements that lead to optimum fitness are selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Since individual HU subunits represent substrates for Lon-dependent degradation ( 2 ) , the increase in rcsA expression can be explained with a stabilization of RcsA due to saturation of Lon with either HU subunit , leading to increased rcsA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>This model was supported by circularisation experiments , in which HU stimulates ligation of substrates as short as 78 bp [ 44,45 ] , whereas in the absence of HU substrates shorter than 150 bp are not e ⁄ ciently circularised [ 46 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>This outcome clearly demonstrates that the absence of HU results in a decrease in the concentration of micF RNA in cells growing in low osmolarity medium and causes , as a consequence , an increase in OmpF synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Probably most intriguing aspect is the resemblance between the properties of bacteria mutated in the mar locus and those of cells lacking HU , such as the decrease observed in the level of the small micF RNA , which leads to the increase in the amount of OmpF and increased susceptibility to antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>To investigate whether the increased level of OmpF observed in the absence of HU involved transcriptional or translational control of ompF RNA we extracted the total mRNA from hup1 and hupAB strains grown either in the absence or in the presence of 300 mM NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>When the bacteria were grown in LB with no salt , the level of micF RNA was strongly reduced ( around 6-fold ) in the absence of HU , whereas no difference was observed when the bacteria were grown in LB containing 300 mM NaCl ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Since the total amount of OmpF increased sharply in the absence of HU , we wondered whether the basal level of OmpF protein was dependent on the total amount of HU protein in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Clearly the absence of HU , which sharply increased the level of OmpF , did not abolish its downregulation at high-salt concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>After the lag period , when the cells start to divide , the level of both subunits of the protein rapidly decreases due to its dilution in the absence of massive de novo synthesis of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>While the complete absence of HU does not abolish the osmoregulation of OmpF protein synthesis , the steady-state level of micF RNA , the negative regulator of OmpF , decreases in bacteria lacking HU , increasing the basal level of this membrane protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>HU has also been found to in ¯ uence micF RNA levels under certain environmental conditions .60 In an HU-de ® cient E. coli strain , micF RNA levels are reduced approximately sixfold when cells are grown under low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>In the wild-type strain , C600 , a low level of OmpF is present compared with high levels of OmpC and OmpA , whereas in the absence of HU , under the same conditions of growth , OmpF is clearly the most abundant porin present in the OM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>To check if the OmpF overproduced in the hupAB mutant , due to the absence of HU , was also sensitive to the temperature of growth , we compared the profiles of porins extracted from either hup1 or hupAB cells grown at 30 ° C , 37 ° C , or 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>DISCUSSION In the course of the construction of the double hupAB mutant , we observed that the cells lacking the protein HU were abnormally sensitive to antibiotics , bile salts , and detergents , particularly in growth medium of low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Mutants in dam have been reported to be sensitive to UV and dam and recA are known to be synthetically lethal ( Marinus and Morris , 1974 ) , although the synergistic sensitivity to UV and to HU between dam and recA was unexpected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Oligomerization via the N-terminus of DnaA is also required for the initiation reaction .6 Initiation of replication occurs once per cell cycle and , during rapid growth , at all origins in the cell simultaneously .7 DnaA binding to the low-affinity sites , as well as binding of architectural elements ( Fis , IHF , and HU ) , may possibly contribute to the synchrony of initiation .8 -- 10 Two main mechanisms prevent initiation from reoccurring at already replicated origins and involve either inactivation of the newly replicated origin ( by sequestration ) or inactivation of the DnaA protein ( by hydrolysis of ATP-DnaA to ADP-DnaA ) .11,12 Specific inactivation of new origins occurs because new origins are hemimethylated ( i.e. , adenine nucleotides of GATC sites are methylated on only one strand ) .13,14 The SeqA protein binds to the hemimethylated GATC sites and prevents remethylation by Dam methylase .15 Regions on the chromosome that have high numbers of GATC sites ( oriC and dnaA promoter ) remain hemimethylated considerably longer ( for up to one-third of a cell cycle ) than regions with few GATC sites .14,16 Also a region with an artificially high number of GATC sites ( an oriC-like distribution of GATC sites placed in the tnaA gene ) was found to be sequestered as long as oriC , showing that sequestration depends on the concentration and distribution of GATC sites .17 During sequestration of newly replicated origins , ATP-DnaA is hydrolyzed to ADP-DnaA in a process that requires the Hda protein and the DNA-loaded β-clamp subunit of the DNA polymerase III holoenzyme .18 This process contributes to lowering the initiation potential by lowering the proportion of ATP-form DnaA .19 The total amount of free DnaA is also lowered because new DnaA binding sites are generated by replication .20,21 After sequestration , the initiation potential is low , and initiation does not occur until a generation has passed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>These rest & s , together with our previous Ending that cells deficient in both HU and IHF are viable at 30-37 ° C [ Kane and Imamoto , Gene 89 ( 1990 ) I33-1371 showed that E. cnfi cells deficient in any two of these three histone-like proteins are viable at 30-37 '' C , and suggested that simultaneous deficiency of all three of the proteins is lethal , There were no detectable differences in the levels of superhelicity of the reporter plasmids isolated from cells deficient in either IHF or H-NS , or from wild-type cells , but about 1576 decrease in negative superhehcity was detected for the reporter plasmid isolated from cells lacking HU and lacking both HU and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>By using the quantitative immuno-blotting ( 21 ) , we found the intracellular concentration of YgiP was very low in growing E. coli cells under aerobic conditions ; however , under hypoxic or anaerobic culture conditions , the YgiP level increased more than 100-fold , reaching to the level as high as those of major nucleoid proteins HU and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Goshima , N. , Kano , Y. , Tanaka , H. , Kohno , K. , Iwaki , T. , and Imamoto , F. ( 1994 ) IHF suppresses the inhibitory effect of H-NS on HU function in the hin inversion system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>This last procedure increases accuracy for quantitative measurement of the intracellular amounts of HU proteins -LCB- Goshima , Kitano , Kohno , Kano & Imamoto , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>For routine assays , the antibodies were diluted 200-fold for CbpA , Fis , H-NS , IHF , RpoD , SeqA and L7/L12 , and 500 fold for CbpB , Dps , Hfg , HU , IciA , StpA , RpoA , RpoS , Rsd and RMF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>While HU is widely distributed among the eubacteria , H-NS , FIS and IHF are restricted to the proteobacteria ( Oberto and Rouviere-Yaniv , 2001 ; Tendeng and Bertin , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>As well as identifying IHF and Fis , as expected , the sampling experiment ( Fig. 3 ) suggested that HU and YfhH also bind to the pnrf53 fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>HU and IHF are small , basic , histone-like proteins which are abundant in the cell and which are involved in a number of cellular processes , such as site-speciﬁc recombination , replication , and transcription ( 8 , 11 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>For operons involved in use of carbon sources in Escherichia coli , the most frequent global regulator is cAMP/CAP , responsible for catabolite repression , but histone-like proteins , IHF , H-NS , Fis and HU are often involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Gene 76 Kano , Y. , Ogawa , T. , Ogura , T. , Hiraga , S. , Okazaki , T. and Imamoto , F. : Participation of the histone-like protein HU and of IHF in mini-chromosomal maintenance in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>On this view FIS , and possibly also CRP , act to ﬁne tune a fundamental mode of transcriptional regulation , not only by their direct effects on DNA topology but also from the indirect effects of their transcriptional control of other nucleoid-associated proteins , such as HU and H-NS [ 3 , 4 ] , which also act as determinants of chromosomal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>The abundence of the small nucleoid-asso-ciated proteins , HU , IHF , Fis , Lrp , in the cell makes it likely that many more examples of their participation in transcription regulation of complex promoters will be found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The intracellular levels of the major DNA-binding proteins , Fis , H-NS , HU and StpA , in the growing cells decreased in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>The major nucleoid proteins , HU , IHF , H-NS and Fis , in exponential-growth-phase E. coli under aerobic conditions are all known to have functional dichotomy ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>All 12 DNA-binding proteins , CbpA , CbpB ( or Rob ) , DnaA , Dps , Fis , Hfq , H-NS , HU , IciA , IHF , Lrp and StpA , were over-expressed using pCU60 , pMK19 , pSY567 , pDPS1 , pRJ1077 , pHFQ607 , pHOP11 , pLhupAhupB , pICS1 , pSA5hiphimA , pMWD1 and pT7stpA , respectively , and puri ® ed to apparent homogeneity as previously described ( Talukder & Ishihama 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>This result suggests that , at high levels of chromosomal DNA supercoiling required for maximal IHF activation , the activating effects of other nonspecific in vivo chromosomal organizer proteins , such as HU or HN-S , might become apparent ( 24 -- 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>These proteins , among which HU , H-NS , FIS ( factor of inversion stimulation ) and IHF ( integration tiosf factor ) are the best characterized , are considered to be histone-like as they are small , relatively abundant and able to bend or wrap the DNA ( reviewed by Schmid , 1990 ) , HU and H-NS bind to DNA non-speoifically , although for Received 21 October , 1994 ; revised 5 December , 1994 : accepted 7 December , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Some sequences produce an intrinsic bend in the DNA or bind proteins such as Fis , H-NS , HU , or IHF that bend the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Some of these proteins bind nonspeci ® cally to DNA , e.g. HU and H-NS , while others are site-speci ® c , e.g. FIS , integration host factor ( IHF ) , and Lrp ( Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>HU thus plays an architectural role , as has been reported in nucleoprotein complexes involved in recombination and replication ( Johnson et al. , 1986 ; Hodges-Garcia et al. , 1989 ; Baker and Mizuuchi , 1992 ; Hwang and Kornberg , 1992 ; Lavoie and Chaconas , 1994 ; Seagall et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>In most cases of DNA transactions involving HU , it acts architecturally by assisting spatially separated sequence-speciﬁc DNA-bound proteins to associ-ate by binding to and bending the intervening DNA region ( Craigie et al. , 1985 ; Johnson et al. , 1986 ; Bramhill and Kornberg , 1988 ; Hodges-Garcia et al. , 1989 ; Ogawa et al. , 1989 ; Wada et al. , 1989 ; Baker and Mizuuchi , 1992 ; Hwang and Kornberg , 1992 ; Lavoie and Chaconas , 1990 ; Lavoie et al. , 1991 ; Lavoie and Chaconas , 1994 ; Segall et al. , 1994 ; Bètermier et al. , 1995 ; Pérez - Martın and deLorenzo , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>HU has been described as an accessory factor that assists the formation of speciﬁc nucleoprotein complexes in many other DNA transactions ( Johnson et al. , 1986 ; Bramhill and Kornberg , 1988 ; Hodges-Garcia et al. , 1989 ; Ogawa et al. , 1989 ; Baker and Mizuuchi , 1992 ; Lavoie and Chaconas , 1994 ; Segall , 1994 ; Betermier , et al. et al. 1995 ; Pérez - Martın and deLorenzo , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Although it et al is still difﬁcult to understand the physiological reasons for the diverse regulation of the two hup genes , it is conceivable that HU could be a sensor of growth , through , and metabolism , through hupA hupB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The tyrP promoter is dependent on supercoiling and its activity in vitro can be inhibited by the addition of HU ( Yang et al. , 1996b ) or by relaxing the supercoiled state with topoisomerase I ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15049824" target="_blank">15049824</a>]<br></li>
<li>Fis can also affect DNA supercoiling indirectly , because of its roles in repressing transcription of gyrA and gyrB to reduce the DNA gyrase activity ( Schneider et al. , 1999 ) , increasing topA transcription to enhance topoisomerase I activity ( WeinsteinFischer et al. , 2000 ) , and regulating the expression of other DNA-binding proteins such as H-NS , HU and CRP ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) that also affect DNA supercoiling directly or indirectly ( Broyles & Pettijohn , 1986 ; Gomez-Gomez et al. , 1996 ; Tupper et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>However , it is possible that the change in transcription activity of PguaB in a fis background occurs as a result of altered regulation of PguaB by a transcription factor other than CRP , for example H-NS or HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ) or by changes in supercoiling ( Schneider et al. , 1997 ; Weinstein-Fischer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524927" target="_blank">18524927</a>]<br></li>
<li>Architectural elements , such as the IHF and HU proteins ( Dixon and Kornberg , 1984 ; Skarstad et al. , 1990 ; Hwang and Kornberg , 1992 ) , negative DNA supercoiling and transcriptional activation ( Baker and Kornberg , 1988 ; Skarstad et al. , 1990 ) all stimulate the event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>We draw attention to the fact that the negative feedback control of hns described here is shared by the structural genes for the HU , IHF and Fis proteíns ( Miller , 1981 ; Kohno et al. , 1990 ; Ninnemann et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>During the transition from exponential to stationary phase , when Fis levels are continually declining while elevated levels of the nucleoid-associated proteins Dps , IHF , HU , Hfq and H-NS are maintained ( Ali Azam et al. , 1999 ) , a global redistribution of chromosomal DNA branching can affect the accessibility of RNAP to numerous promoters to indirectly alter gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The synthesis of IHF is indeed growth-phase regulated ( as is the case for the abundant H-NS and Dps proteins ; Almiron et al. , 1992 ) and an elevated concentration in stationary phase could help either to compact DNA , or protect it against chemical or physical attacks ( as was proposed for HU ; Boubrik & Rouviere-Yaniv , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>In exponential phase E. coli cells , the order of intracellular concentration of these DNA-binding proteins is , in descending order , Fis , HU , StpA , H-NS and IHF ( Talukder et al. 1999 ) , among which HU , StpA , H-NS and IHF have been considered to participate in the condensation of genome DNA to form the compact nucleoid ( for reviews see Pettijohn 1982 , 1988 , 1996 ; Drlica & Rouviere-Yaniv 1987 ; Rouviere-Yaniv & Yaniv 1979 ; Murphy & Zimmerman 1997a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>Nucleoid compaction and DNA-associated proteins DNA-associated proteins such as H-NS , HU , IHF , and Fis have often been suggested to be responsible for DNA compaction in E. coli , based among other things on their colocalization with DNA in vivo ( Azam et al. , 2000 ) , on their occurrence in isolated nucleoids that are prepared in the absence of high salt concentrations ( Murphy and Zimmerman , 1997 ; Pettijohn , 1996 ) , and on their ability to bend or compact DNA in vitro ( Dame , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>Also in accord , Brunetti et al. ( 2001 ) found that individual deletions of H-NS , HU , Fis , or IHF did not appear to change the large scale structure of the E. coli chromosome , based upon the lack of changes in the in vivo genomic fragmentation patterns by DNA gyrase in the presence of oxolinic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>The bending induced by HU facilitates optimal protein -- protein and/or protein -- DNA contacts in an otherwise physically recalcitrant structure , thereby orchestrating the regulated assembly of functional nucleoprotein complexes , such as those involved in DNA replication , recombination , transposition , transcription , and DNA repair ( Skarstad et al. 1990 ; Dri et al. 1992 ; Haykinson and Johnson 1993 ; Manna and Gowrishankar 1994 ; Li and Waters 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>This view derived primarily from the fact that Lrp tends to activate some operons involved in biosynthesis and repress others involved in catabolism .1 Another view advanced by D'Ari et al. 13 is that Lrp is primarily a DNA-organizing protein , much like HNS , IHF , and HU , which also affect gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>IHF is a heterodimer of J & rr .20,000 ( Nash & Robertson , 1981 ) with sequence homology to the histone-like protein HU , but unlike HU , binds to a specific DNA sequence ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Moreover , like the eukaryotic HMG1 protein , dimeric HU binds to double-stranded DNA in a non-sequence-speciﬁc manner and speciﬁcally recognizes DNA structures such as cruciform DNA ( Pontiggia et al. , 1993 ; Bonnefoy et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>It is now well established that non-speci ® c binding is an important feature of IHF ( Wang et al. , 1995 ; Yang & Nash , 1995 ; this work ) and it is interesting to note that the value of its Kc is in the same range as the dissociation constant of HU for linear DNA ( Bonnefoy et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>DNA is compacted at low HU concentrations , but becomes extended at high HU concentrations ( van Noort et al. , 2004 ; Skoko et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>The extent to which DNA is compacted by HU depends on the angle induced by the protein and the number of HU molecules bound ( van Noort et al. , 2004 ; Skoko et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>In E. coli exposed to cold-shock , an increase in the negative supercoiling of DNA is observed which is mediated by the function of Gyrase and the DNA binding protein HU ( Mizushima et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>This observation can be rationalized by looking into the HU -- DNA cocrystal structure , in which , besides bending , underwinding and negative writhe are induced ( Swinger et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Furthermore , HU binds preferentially to structural distortions in DNA , such as nicks , gaps and three - or four-way junctions , recognizing and stabilizing a pre-existing bend , rather than actively inducing the bend ( Pontiggia et al. , 1993 ; Castaing et al. , 1995 ; Kamashev et al. , 1999 ; Kamashev and Rouviere-Yaniv , 2000 ; Balandina et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Only the complete absence of HU is really deleterious to the cells ( Wada et al. , 1988 ; Huisman et al. , 1989 ; Boubrik & Rouviere-Yaniv , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Although HU shows little sequence speciﬁcity , it binds preferentially to supercoiled DNA ( Shindo et al. , 1992 ) and constrains negative supercoiling in vitro ( Rouviere-Yaniv et al. , 1979 ; Broyles and Pettijohn , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>In both models the ability of HU to induce or constrain supercoiling is explicitly taken into account ( Rouviere-Yaniv et al. , 1979 ; Broyles and Pettijohn , 1986 ; Shindo et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>HU has also been suggested to induce negative supercoiling ( Rouviere-Yaniv et al. , 1979 ; Tanaka et al. , 1995 ) either directly due to non-co-planar DNA bending , or indirectly by stimulation of DNA gyrase ( Swinger and Rice , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>a2 like ab , but not b2 , restrains DNA superhelicity To explore the possibility that the three alternative forms of HU differ in their biochemical activity , we examined the ability of HU to introduce negative supercoiling in the presence of topoisomerase I activity ( Rouviere-Yaniv et al. , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Like the four histones , HU is capable of condensing DNA and introducing negative supercoiling in a circular DNA molecule in the presence of topoisomerase I ( Rouviere-Yaniv et al. , 1979 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The HU-induced bend is not rigid as IHF-induced bends but better considered as a Xexible hinge and it seems that HU can induce and stabilize a range of diVerent bend angles similar to high mobility group ( HMG ) proteins in eukaryotes ( Paull et al. , 1993 ; Swinger and Rice , 2004 ; Swinger et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Similarly , HU is proposed to counteract the effects of H-NS , possibly by competing for preferential binding sites ( Dame and Goosen , 2002 ; van Noort et al. , 2004 and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Pérez - Martin , J. and De Lorenzo , V. ( 1995 ) The sigma 54-dependent promoters Ps of the TOL plasmid of Pseudomonas patida requires HU for transcriptional activation in vivo by XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Accordingly , in addition to restraining negative supercoils , HU may work in cooperation with topoisomerase I to regulate the in vivo level of supercoiling in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>In addition , it has recently been shown that a relationship exists between the level of HU protein in vivo and the activity of DNA topoisomerase I. Thus , in addition to restraining negative supercoils , HU may , in vivo , communicate with topoisomerase I to regulate the global level of supercoiling in E. coli ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>HU can stabilize or constrain negative supercoiling , which reduces writhe and thus prevents the formation of plectonemes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>The absence of HU could stabilize the repression loop by increasing the energy available from negative supercoiling for loop formation , thereby compensating in small part for the destabilizing effect caused by the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The HU cells have only slightly decreased negative supercoiling but show substantial changes in loop stability and character .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Figure 3 and Table 3 show the results when plasmids from WT , HU , IHF and H-NS strains were electrophor-esed in the presence of 1 , 2 , 3 , 5 , 7 or 9 mg/ml chloroquine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>In contrast , total negative supercoiling was strongly reduced in HU cells but strongly increased in IHF relative to WT ( this is most clear Figure 3 , 7 mg/ml chloroquine ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>These data show that levels of unrestrained negative supercoiling in the WT and HU backgrounds are similar , while negative supercoiling in H-NS and IHF strains was increased relative to WT ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>The increases in supercoiling levels in strains mutant in HU , FIS and gyrase are not seen if the cspE gene in the plasmid is inactivated by an insertion ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12904550" target="_blank">12904550</a>]<br></li>
<li>We demonstrate here that strains overexpressing AEB have an increase in supercoiling of resident plasmids and that these increased supercoiling levels can be seen in gyrase , HU and Fis mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12904550" target="_blank">12904550</a>]<br></li>
<li>In this scenario , overexpression of cspE would increase supercoiling levels of resident plasmids in both parental cells and cells defective in supercoiling levels ( gyrase , HU or Fis mutants ) by stabilizing the population of supercoiled DNA molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12904550" target="_blank">12904550</a>]<br></li>
<li>Unlike HU , this protein can self-associate to form dimers , trimers or larger oligomers and this property is probably essential for its architectural function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>The presence of ab enhances survival of E. coli during starvation Since the heterodimeric form of HU is predominant in stationary phase we wondered if this form of HU performs some speci ® c function in entero-bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Moreover not only is HU required for this function , but speci ® - cally its heterodimeric form is needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>In all cases , elimination of these genes led to an increase in survival in the presence of either amikacin or HU , similar to what is seen in the presence of a cpxA * mutation that constitutively activates the Cpx response , without any apparent overall change in viability ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>We assayed survival of increasing doses of UV irradiation , and the ability to form colonies in the presence of low levels of HU or AZT , inhibitors of replication elongation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Interestingly , deletion of nc1 leads to decondensation of the nucle-oid regardless of the presence of HU in the cell ( Fig. 6A to D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Another recent study demonstrates that in vivo the transcriptional activity of T7 RNA polymerase is enhanced by the presence of HU , and that this stimulation most likely occurs at the level of initiation [ 73 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Although the assay conditions were slightly different , the 45 kDa phosphoprotein we observe may correspond to the protein no longer phosphorylated in vitro in ihe presence of the histone-iike proteins HU and H-NS ( S. T. Sweeney , P , H. Williams and V. Norris , submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783627" target="_blank">7783627</a>]<br></li>
<li>The results show that in the presence of HU alone , transcription of mtr was severely inhibited in every case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>The degree of TyrR-mediated transcription activation ( fold activation ) was measured for each of the RNA polymerases by comparing the intensity of the mtr band produced in the presence of HU , TyrR , and phenylalanine with that produced in the presence of HU alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>a Intensity of the mtr band produced in the presence of HU , TyrR , and phenylalanine divided by that produced in the presence of HU alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>Previous in vitro studies have shown that the binding of the TyrR protein in the presence of its cofactors to the TyrR boxes of mtr results in relief of HU - or IHF-mediated inhibition of transcription ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209035" target="_blank">9209035</a>]<br></li>
<li>As occurred in the case of HU , the IHF-mediated transcription inhibition of both the mtr and tyrP13 * promoters could not be lifted by the TyrR protein in the presence of tyrosine ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>Furthermore , our observation that the BaCl2-mediated inhibition of transcription can be alleviated by the presence of TyrR protein and phenylalanine ( results not shown ) implies that the mechanism by which the TyrR protein activates transcription of the mtr and tyrP13 * promoters is to overcome the HU - or IHF-induced conformational change of DNA , which is undesirable for transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>As previously reported , in vitro activation of transcription of mtr by TyrR protein and phenylalanine is observed on supercoiled templates in the presence of HU , IHF , or BaCl2 ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>In the presence of phenylalanine , the wild-type TyrR protein , but not a mutant TyrR protein ( activation negative ) , was able to relieve the HU - or IHF-mediated inhibition of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>The in vitro transcription experiments using the mutant template were also performed in the presence of HU with and without puriﬁed TyrR protein and phenylalanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>These results suggest that it is the positive charge and not the hydrophobic side chain of arginine-265 that is required for the in vitro transcription of mtr in the presence of HU and TyrR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>This successive appearance of HU mRNAs is independent of FIS which regulates the hup genes FIS and CRP regulate the transcription of both hup genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>1.0 * 1.0 ** 0.3 2.2 / 2.5 3.3 The transcriptional regulation by CRP and FIS is mirrored in the synthesis of HU subunits To examine how variations at the level of HU transcription caused by the FIS and CRP regulation affected the a and b subunits , we performed Western blots to measure the accumulation of the two HU subunits in the absence of FIS and/or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The level of transcription of the HU genes was strongly in ¯ uenced by the absence of the FIS protein : clearly FIS stimulated the expression of the hupA gene while it negatively regulated the hupB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Effects of CRP and FIS on heterodimeric HU The results obtained on the hupA transcription show that the positive effects of CRP and FIS are additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The HU regulon is composed of genes responding to anaerobiosis , acid stress , high osmolarity and SOS induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>The speed of centrifugation was chosen for the expected species in cells 6 and 7 , and it was , therefore , too low for the mass of the HU tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>The P1 promoter showed little , if any , derepression in the presence of coumermycin ( Fig. 3 ) and only partial derepression in the absence of HU and/or OI in vivo ( Figs 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The effect of HU was apparent only in the presence of all four glpD operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The same publication conﬁrmed the presence of all the mRNA species that bind to HU in exponentially growing cells , except cspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Proteins were transfered onto a nitrocellulose membrane and HU was detected immuno-chemically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>We confirm that deletion of HU inhibits looping and that quantitative modeling suggests residual looping in the induced operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Several factors have been shown to influence the ability of DnaA protein to open the strands ( 3-6 ) : ( i ) binding of ATP or ADP to DnaA ; ( ii ) the level of negative superhelicity ; ( iii ) the temperature ; ( iv ) the presence of structure-modifying proteins ( e.g. HU and IHF ) ; ( v ) thepresence of theinhibitory IciA protein ; and ( vi ) transcriptional activationI l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>Thus far , three proteins have been identified with specific functions and binding sites in oriC ( i ) DnaA , the initiator protein , which binds four 9-mers and then interacts with a contiguous set of three 13-mersto open the duplex ; ( ii ) the IciA protein which binds these 13-mers to prevent DnaA protein from opening this region and thus inhibits initiation ; and ( iii ) IHF , which binds a sequence that facilitates the bending of oriC around an aggregate of DnaA proteins ; HU protein serves the same function as IHF buw t ithout an identified binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>heat unstable nucleoid protein ( HU ) - mediated DNA looping Giuseppe Lia * † , David Bensimon ‡ , Vincent Croquette ‡ , Jean-Francois Allemand ‡ § , David Dunlap ¶ , Dale E. A. Lewis , Sankar Adhya , and Laura Finzi * , ** * Department of Biology , University of Milan , 20133 Milan , Italy ; ‡ Laboratoire de Physique Statistique and Department de Biologie , and § Departement de Chimie , Ecole Normale Superieure , 75005 Paris , France ; ¶ San Raffaele Scientiﬁc Institute , 20132 Milan , Italy ; and Laboratory of Molecular Biology , National Cancer Institute , Bethesda , MD 20892 Contributed by Sankar Adhya , July 31 , 2003 The overall topology of DNA profoundly inﬂuences the regulation GalR dimers , and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_217">
Lia , G. , Bensimon , D. , Croquette , V. , Allemand , J.F. , Dunlap , D. , Lewis , D.E. , et al. ( 2003 ) Supercoiling and denaturation in Gal repressor/heat unstable nucleoid protein ( HU ) - mediated DNA looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></label>
<input id="act_similarity_217", type="checkbox">
<div class="ACT_similar">
<li>Lia , G. , Bensimon , D. , Croquette , V. , Allemand , J.F. , Dunlap , D. , Lewis , D.E. , Adhya , S. and Finzi , L. ( 2003 ) Supercoiling and denaturation in Gal repressor/heat unstable nucleoid protein ( HU ) - mediated DNA looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li></div>
<li>© 2000 Oxford University Press A functional assay in Escherichia coli to detect non-assisted interaction between galactose repressor dimers N. Perez , M. Rehault and M. Amouyal * Equipe ` Interactions à Distance ' , CNRS UMR 8532 , Institut Gustave Roussy ( PR2 ) , 39 Rue Camille Desmoulins , 94 805 Villejuif Cedex , France Received May 15 , 2000 ; Revised and Accepted July 21 , 2000 ABSTRACT Escherichia coli Among the operons repressed from multiple sites on DNA , the galactose operon is unique : its repression requires an auxiliary protein , HU , to assist cooperative repressor binding to two distant DNA sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>Further research will be required to establish this interpretation ; however , the results at this stage support the proposal that HU causes DNA structural changes in addition to t ( he restraint of negative supercoils ( c ) Binding of HC ; protein to negatively supercoiled 0s relaxed DNAs Alteration of DNA helical twist by HU protein in the manner suggested above would impose a substantial shift , in the angle bet ' ween adjacent DNA base-pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>Nevertheless , each protein functions speciﬁcally in one operon but has no effect in another operon , arguing that selective action of HU and IHF in different operons might enhance the ability for speciﬁc regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>However , the cryptic bgf operon , whose expression was reported to be regulated through topological changes of cellular DNA , could not be activated in cells depleted of HU or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>It is possible that IHF and an additional protein , such as FIS , H-NS or HU , along with CysB may be necessary for full activation of the positively regulated cysJIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>Protein HU binds speciﬁcally to kinked DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_225">
Pontiggia , A. , Negri , A. , Beltrame , M. & Bianchi , M.E. ( 1993 ) Protein HU binds speci ® cally to kinked DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></label>
<input id="act_similarity_225", type="checkbox">
<div class="ACT_similar">
<li>Pontiggia , A. , Negri , A. , Beltrame , M. and Bianchi , M.E. ( 1993 ) Protein HU binds speciﬁcally to kinked DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Pontiggia , A. , Negri , A. , Beltrame , M. , and Bianchi , M.E. ( 1993 ) Protein HU binds speciﬁcally to kinked DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li></div>
<li>HU binds to DNA without any sequence specificity and with low affinity , and it binds to bent or distorted DNA with higher affinity ( 8-12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>However , the fluorescence energy transfer measurements do not support a model in which HU also binds to the 20 bp DNA to which GalR2 binds with high affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>[ Keywords : Gal repressor ; transcription regulation ; DNA loop ; oriented heterodimers ; operators ; HU ] Received April 6 , 2004 ; revised version accepted May 25 , 2004 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Sensitivity to various types of DNA-damaging agents , such as UV irradiation , MMS , hydrogen peroxide , hydroxyurea ( HU ) , and phleomy-cin , were assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446634" target="_blank">12446634</a>]<br></li>
<li>Binding of single HU dimers at random positions can induce local DNA bending and increase the average # exibility of the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Pinson , V. , Takahashi , M. , and Rouviere-Yaniv , J. ( 1999 ) Differential binding of the Escherichia coli HU , homodimeric forms and heterodimeric form to linear , gapped and cruciform DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Tanaka , H. , Yasuzawa , K. , Kohno , K. , Goshima , N. , Kano , Y. , Saiki , T. and Imamoto , F. ( 1995 ) Role of HU proteins in forming and constraining supercoils of chromosomal DNA in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>Shimizu , M. , Miyake , M. , Kanke , F. , Matsumoto , U. , and Shindo , H. ( 1995 ) Characterization of the binding of HU and IHF , homologous histone-like proteins of Escherichia coli , to curved and uncurved DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>One obvious structural change occurring differently in the supercoiled and nicked DNA is t , he linking-number change in the nicked DNA competit , or after binding the HU that can not occur in t , he covalent , ly continuous DNA competitor using the experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>Another in vitro system , one that is reconstituted with the puriﬁed proteins , DnaA , DnaB helicase , DnaC , DnaG primase , single-strand binding protein SSB , histone-like protein HU , gyrase , and DNA polymer-ase III holoenzyme , also efﬁciently replicates oriC plasmids ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10497067" target="_blank">10497067</a>]<br></li>
<li>This sensor should also allow the monitoring of the concentrations of other DNA binding proteins like HI and HU which bind DNA non-specifically and because of this could affect the accessibility of FIS binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1547773" target="_blank">1547773</a>]<br></li>
<li>Destabilization of DNA looping in the absence of HU could imply a direct , sequence-non-speciﬁc but possibly structure-speciﬁc binding of HU to looped DNA , as is seen for the related Gal repressor ( 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>In vitro , the altered HU proteins showed substantial defect in repression of P2 transcription and exhibited no discernible difference in their DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Thus , an interaction of two DNA bound GalRs stabilizes site-speciﬁc binding of HU and vice versa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Kobryn , K. , Lavoie , B. , and Chaconas , G. ( 1999 ) Supercoiling-dependent site-specific binding of HU to naked Mu DNA , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>In contrast to HU and H-NS , FIS and IHF are site-specific binding proteins which , upon interaction with their respective recognition sites , induce a significant bend in the DNA with an estimated angle of 90 for Fis and 140 '' for IHF ( Thompson and Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>In contrast to HU , IHF binding to DNA is strongly sequencespecific .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Binding of HU is sequence unspeci cents c but , as is the case with IHF [ 7 ] , binding to DNA probably involves its two arms being inserted into the minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Assisted by protein HU or IHF , cooperative binding of 20 -- 40 monomers of ATP-bound DnaA protein per oriC stimulates duplex opening at the AT-rich 13-mer repeats in oriC , where several replication proteins are loaded to initiate DNA replication ( Kornberg & Baker , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Skoko , D. , Wong , B. , Johnson , R.C. , and Marko , J.F. ( 2004 ) Micromechanical analysis of the binding of DNA-bending proteins HMGB1 , NHP6A , and HU reveals their ability to form highly stable DNA -- protein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>To apply this method , purified HU protein was mixed with relaxed pMB9 DNA and incubated w & h a topo-isomerase to relax any torsional t , ension in the DNA induced by HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>HU is a conserved protein known for its role in binding to DNA and maintaining negative supercoils in the latter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Shindo , H. , Furubayashi , A. , Shimizu , M. , Miyake , M. , and Imamoto , F. ( 1992 ) Preferential binding of E. coli histonelike protein HU alpha to negatively supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Furthermore , unlike the case of NS ( HU ) , which was found to form a larger amount of large cross-linked products in the presence of DNA ( Losso e ( al. , 1986 ) , the cross-linking reaction with H-NS was found to be completely unaffected by the presence of DNA ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>We used this high-resolution technique to measure accurately the fragment sizes of DNA digestred by staphylocorcal nuclease and DNase I in the presence of HU ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>Moreover , the effect of HU could not occur through a change in DNA supercoiling , since it was shown that in E. coli the absence of HU causes only a marginal change in the superhelical density of the DNA : the absence of HU being correlated with an increased activity of topoisomerase I ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Recent in vitro experiments have shown that HU can introduce negative superhelical turns in a relaxed DNA in the presence of a nicking-closing enzyme in a similar fashion to the 4 core histones [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=227733" target="_blank">227733</a>]<br></li>
<li>As we have already shown , HU ab , when mixed with closed circular relaxed SV40 DNA in the presence of topoisomerase I present in ascites extract , introduces supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>HU , which shares with H-NS the property of introducing , in vitro , negative supercoiling in relaxed DNA in the presence of topoisomerase I ( 3 , 31 ) has clearly opposite effects on OmpF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Rouviére - Yaniv , J. , Kiseleva , E. , Bensaid , A. , Almeida , A. and Drlica , K. ( 1992 ) Protein HU and bacterial DNA supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Effect of DNA supercoiling and HU on transcription and activation of the mtr and tyrP13 * promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>HU ( heat-unstable nucleoid protein ) , considered a prokaryotic histone homologue , is one of the bacterial NAPs that covers the E. coli genome .15 HU 's ∼ 50 000 molecules in the cell support a separation of , on average , ∼ 350 bp between HU oligomers .15 HU induces DNA loop formation , where ∼ 100-mers cyclize in the presence of nonspeciﬁcally bound HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Previous characterization of this system for loop lengths of 63 -- 91 bp revealed that ( i ) DNA twisting rather than bending is the major obstacle to DNA looping ; ( ii ) weak repression loops are still detected in the presence of saturating concentrations of IPTG ; ( iii ) deletion of the E. coli architectural protein HU dramatically destabilizes repression loops ; ( iv ) replacement of E. coli HU with heterologous mammalian HMGB proteins can partially rescue DNA looping and ( v ) the effect of HU loss on DNA looping does not result from changes in DNA supercoiling ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Indeed the dimer HU must be bound to DNA by electrostatic bonds since during the isolation process , the protein HU adsorbed at low ionic strength on a DNA-cellulose column , is eluted by increasing the salt concentration [ l ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6987059" target="_blank">6987059</a>]<br></li>
<li>Increasing the level of negative supercoiling of the DNA in vivo by either overproducing the HU protein or introducing topoisomerase mutations caused a decrease in tdc ± lacZ expression ( Wu and Datta , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Thus , when present at a high enough concentration , TcbC might possibly have the same effect as HU protein in increasing the level of DNA supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The optimal operator spacing for uninduced DNA looping is 2 bp shorter in the absence of HU , suggesting a change in the optimum repression loop geometry , a reorganization of the DNA domain , or a change in the local supercoiling status in the absence of this nucleoid protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>The basis for context-dependent changes in the unrestrained negative supercoiling in the absence of HU protein is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>The e ¡ ect of HU on DNA topology is further underlined by the observation that the absence of this protein is compensated by mutations in gyrase [ 33 ] , the enzyme responsible for the induction of negative supercoils in DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Finallv , titrations using different concentrations of ethidium bromide during electrophoresis have shown that the purified DNA was negatively supercoiled after interaction wit ) h protein HU and topoisomerase ( Broyles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>It was also shown that HU binds preferentially to negatively supercoiled DNA [ 32 ] , that it constrains supercoils and that one negative superhelical turn is constrained every 120 to 290 bp [ 13,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>This would be incompatible with the fact that HU binds preferentially to negatively supercoiled DNA and that the protein can constrain negative supercoils .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>These results suggest that repression of glpD is mediated by a repression loop , with HU facilitating repression by bending the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>HU facilitates formation of a DNA repression loop in the gal operon , where the protein reportedly binds a speciﬁc sequence in the loop ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>However , the accessory protein HU contributes additionally to repression , presumably by assisting in DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The his-tone-like protein HU necessary for transcription repression has high afﬁnity to the cognate DNA ( hbs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>[ Key Words : Transcription repression ; DNA looping ; protein -- protein interaction ; HU mutants ] Received June 18 , 2001 ; revised version accepted July 16 , 2001 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>We also demonstrated weak DNA looping by Lac repressor even when saturated by IPTG , and showed evidence that loss of the nucleoid protein HU destabilized repression loops .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>No distinct region of protection ( between -200 and +40 ) was afforded by HU in DNase I footprints , although the DNA was more resistant to digestion in the presence of HU , which is consistent with the known properties of this protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Repression also requires supercoiled DNA and † Present address : Laboratoire de Physique Statistique , Ecole Normale Superieure , 75005 the presence of the nucleoid-associated protein HU ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>The relative affinity of HU protein for negatively supercoiled and relaxed DNA was compared using t ' he competit , ion linking-number assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>The process requires negatively supercoiled DNA and the presence of the histone-like protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Additional cross-linking experiments were carried out in which H-NS was reacted in the presence of DNA and/or in the presence of the other histone-like protein NS ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>Together the in vitro data were taken to provide evidence for DNA being wrapped around an HU protein core [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Unlike HU protein , which binds to DNA in a nonspecific manner , IHF binds to a specific DNA consensus sequence , 5 ' - WATCAANNNNTTR-3 ' ( where W = A or T , R = A or G , N = any base ) ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Next , we asked if host protein integration host factor ( IHF ) , which is closely related to HU but binds to DNA in a sequence-speciﬁc manner , can inhibit transcription initiation from the mtr or the tyrP13 * promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>Keywords : Nucleoid ; Bacterial chromatin ; DNA compaction ; Nucleoid-associated protein ; H-NS ; HU ; IHF ; Fis 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>HU is a structural homologue of IHF , a protein that can induce bends into DNA of up to 160 ‡ [ 7 ] when bound to its recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>fence is in contrast to HU , which binds linear DNA in preference to supercoiled DNA ; HU binds RNA and single-stranded DNA in preference to double-stranded DNA 1411 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748941" target="_blank">7748941</a>]<br></li>
<li>Castaing , B. , Zelwer , C. , Laval , J. , and Boiteux , S. ( 1995 ) HU protein of Escherichia coli binds specifically to DNA that contains-single-strand breaks or gaps , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>The archetype of prokaryotic histonelike protein , HU , binds to DNA with little or no sequence specificity , although it has a preference for DNA containing cruciforms or single-stranded breaks ( Bianchi 1994 ; Nash 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The histone-like protein HU binds speciﬁcally to DNA recombination and repair intermediates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Kamashev , D. , and Rouviere-Yaniv , J. ( 2000 ) The histonelike protein HU binds speciﬁcally to DNA recombination and repair intermediates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>enzyme , 0.5 , Ag ; gyrA protein , 0.1 jug ; gyrB protein , 0.5 , ug ; E. coli to-poisomerase I , 75 ng ; and protein HU , 0.2 , ug ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6310593" target="_blank">6310593</a>]<br></li>
<li>HU protein facilitates negative regulation of glpD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Finally and most importantly , FIS regulated the HU protein synthesis like it regulates that of the mRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The HU protein binds the rpoS leader region and stimulates translation ( 2 ) , while H-NS inhib-its RpoS induction ( 3 ) in a manner antagonized by DsrA ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142416" target="_blank">12142416</a>]<br></li>
<li>The key conclusion of these experiments was that the sequence-non-speciﬁc HU protein is required to stabilize small repression loops in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>It will be of interest to understand why less HU is produced during growth in glucose and why , in the absence of FIS in the cell , b becomes predominant while a decreases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>Surprisingly , E. coli cells survived , albeit poorly , in the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>While the P1 promoter was found to be repressible in the absence of DNA supercoiling and HU , the repression of P 2 was entirely dependent upon DNA supercoiling in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Although P2 repression is completely HU-dependent in vitro , P2 is not completely derepressed in vivo in the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>P 2 , but not P1 , was also totally derepressed by the absence of HU or the OI operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Regulation of P1 and P 2 by HU and DNA looping were further veriﬁed by measuring the expression of the promoters in the absence of OI in strains in which P1 was fused to the lacZ reporter gene and P 2 to gusA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Increasing the loop size ( Virnik et al. 2003 ) or strengthening the tetramerization interface of GalR dimers ( Geanacopoulos and Adhya 2002 ; Semsey et al. 2002 ) enhanced nonassisted loop formation , resulting in reduced dependency on the architectural contribution of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Energy values ( kcal/mole ) obtained in the presence of HU , bound at hbs ( +6.5 ) , are as described in Geanaco-poulos et al. ( 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Similar energy values were obtained for the AL2 loop in the absence or presence of HU binding to hbs , suggesting that HU binding at position +6.5 does not significantly contribute to the stability of the AL2 loop ( Table 2 ; Geanacopoulos et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Discussion In our attempt to de ® ne the structure of the Gal repressosome , we previously described the isolation and characterization of GalR mutants that bind to DNA operators normally but are defective in potential GalR ± GalR or GalR ± HU contacts during DNA loop formation ( Geanacopoulos et al. , 1999 , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The models for GalR bound to DNA were taken from the crystal structures of PurR -- DNA complex ( Schumacher et al. 1994 ) , and the model for HU bound to DNA was taken from the complex based on the affinity cleavage data ( Lavoie et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The functional signiﬁcance of strong DNA bending was recently exempliﬁed in a magnetic tweezers experiment in which the interaction between two GalR molecules separated by 115 bp distance was shown to be directly stimulated by the presence of HU ( Lia et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>The two operator-bound GalR molecules associate in the presence of the histonelike protein HU resulting in the formation of a DNA loop encompassing the promoters ( Aki et al. , 1996 ; Aki & Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>In contrast , dimeric GalR molecules bound to bipartite operators separated by 11 helical turns associate to form DNA loops on supercoiled DNA only in the presence of HU ( Choy et al. 1995 ; Aki et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>The presence of wild-type GalR and HU had the same effect , since the repressosome structure can not be formed on linear template ( Aki et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In the presence of GalR and HU , - glucuronidase synthesis is repressed from the P2 ∼ gus fusion ( Lewis et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Results gal repression requires HU In a puriﬁed transcription system , GalR activates the P2 promoter and represses , albeit less efﬁciently , the P1 promoter ( Choy and Adhya , 1992 ; Goodrich and McClure , 1992 ; Choy et al. , 1995a ; Figure 1 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>( i ) If DNA looping is prevented by the use of relaxed DNA or by the omission of HU , GalR binding to OE alone brings about partial repression of P1 ( Kuhnke et al. , 1986 ; Choy and Adhya , 1992 ; Aki et al. , 1996 ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>LacI tetramer forms a DNA loop and concurrently represses P1 and P2 transcription without HU ( Mandal et al. , 1989 ; Choy and Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In this case HU is necessary for formation of a repression loop with GalR ( Aki and Adhya , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>GalR ( like GalS ) is a dimer in solution but repression of galEP2 requires tetramerization of GalR , mediated by HU forming the complex called a repressosome ( Aki et al. , 1996 ; Lewis et al. , 1999 ; Geanacopoulos et al. , 2001 ; Kar and Adhya , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>HU binding requires occupancy of both operators by GalR The repression of the two overlapping gal promoters both in vivo and in vitro requires GalR binding to both gal operators ( Adhya and Miller , 1979 ; Irani et al. , 1983 ; Haber and Adhya , 1988 ; Aki et al. , 1996 ; Figure 1 , lanes 3 , 8 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Ef ® cient repression of the P2 promoter is totally dependent upon the presence of the HU protein ( Lewis et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>To investigate the mechanism and specificity of the weak HU-indepen-dent looping repression in the gal system , we investigated the effect of the GalR mutants GalRT322R and GalRH327R , defective in tetramerization , on the P2 activity in the absence of HU ( Geanacopoulos et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>GENES & DEVELOPMENT 15:2273 -- 2281 © 2001 by Cold Spring Harbor Laboratory Press ISSN 0890-9369/01 $ 5.00 ; www.genesdev.org The binding of GalR dimers to OE and OI in the absence of HU is noncooperative ( Brenowitz et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In the absence of HU , however , GalR binding to OE shows different effects on P1 and P2 ; it represses the former and activates the latter ( Cho 0022 ± 2836/97/380293 ± 08 $ 25.00 / 0/mb971221 & Adhya , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>In the absence of HU , GalR binding to OE partially represses the P1 promoter and activates the P2 promoter ( Choy and Adhya 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Geometry of loops Gal loops In contrast to the LacI , the GalR tetramer is likely to have a rather rigid V-shaped structure that is formed via stacking interactions of two GalR dimers with relatively weak afﬁnity for each other .18,19 These properties restrict the conditions ( e.g. phasing and spatial separation of operators along DNA , presence or absence of HU , negative superhelicity etc. ) in which the bound GalR dimers are able to form a loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Energy of DNA bending and twisting for the antiparallel loops Etotal Etwist Ebend Linear DNA + GalR AL1 loop AL2 loop 26.0 28.0 6.4 7.6 21.2 22.2 Linear DNA + GalR + HU bound to hbs ( +6.5 ) AL1 loop AL2 loop 9.4 8.7 5.8 25.5 14.9 13.6 Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>We conclude that GalR dimers interact directly and do not use HU as an adaptor in loop closure ; HU and DNA supercoiling act in concert to stabilize the GalR tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Besides induction ( loop breakdown ) of the operon by D ( ) - galactose , which acts by inactivating GalR , the loop must break down for expression in response to anabolic needs resulting from changes in DNA superhelicity or HU concentration ( 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>To study any critical GalR -- HU interaction , we performed site-directed mutagenesis of the hupA gene to identify any HU mutants that would be specifically defective in the formation of the repressosome while retaining their ability to-bind to DNA with normal affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Although a specific interaction of GalR and HU was shown to be essential for loop formation , the HU-GalR contact is unlikely to be present in the repressosome ; it appears that GalR piggybacks HU to its architecturally critical binding site on DNA ( Kar and Adhya 2001 ; S. Roy , E. Dimitriadis , S. Kar , M. Geanaco-poulos , M.S. Lewis , and S. Adhya , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>HU binding is considered to be not strictly sequence-speciﬁc ; HU has strong afﬁ-nity for distorted DNA structure .7,13 It shows tri-partite-binding cooperativity with GalR dimer binding to the operators in DNA looping .34 Our results support the model that DNA looping occurs via a direct interaction of two GalR dimers bound to the operators , and is helped by HU protein binding to the architecturally critical position hbs .12 Probably , increasing the distance between OE and OI reduces the persistence of initially short DNA element , its resistance to torsional and bending distortions , and shifts the energy balance in favor of a more stable GalR tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Recruitment of HU by piggyback : a special role of GalR in repressosome assembly Sudeshna Kar and Sankar Adhya1 Department of Molecular Biology , National Cancer Institute , National Institutes of Health , Bethesda , Maryland 20892-4255 , USA In Gal repressosome assembly , a DNA loop is formed by the interaction of two GalR , bound to two distal operators , and the binding of the histone-like protein , HU , to an architecturally critical position on DNA to facilitate the GalR -- GalR interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>GalR binds to the operators with high afﬁnity .10,11 Two conditions are necessary for realization of a stable loop : negatively supercoiled DNA and the presence of the histone-like nucleoid-structural protein HU .12 During DNA looping , the HU heterodimer binds to an architecturally critical site , hbs13 ( Figure 1 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>GalR represses transcription initiation from the two promoters , P1 and P2 , of the gal operon by binding to two spatially separated operators , OE and OI , which encompass the Abbreviations : HU , heat unstable nucleoid protein ; GalR , gal repressor dimer protein ; SSB , single-stranded binding protein ; ssDNA , single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>GalR dimers bound to two spatially separated operators , O and O , speci ® cally interact with the his-E I tone-like protein HU and close the loop in supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In vitro , DNA loopingmediated repression involves binding of the Gal repressor ( GalR ) to two gal operators ( OE and OI ) and binding of the histone-like protein HU to a speciﬁc locus ( hbs ) about the midpoint between OE and OI , and supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Formation of the repression loop by GalR absolutely requires the intervention of the histone-like protein , HU , binding between the two GalR operators and stabilising an antiparallel DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Footprinting experiments using iron -- EDTA-coupled HU show that HU binding to gal DNA is orientation speciﬁc and is speciﬁcally dependent upon binding of GalR to both OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The site-speciﬁc stable binding of HU to gal DNA is totally dependent upon GalR binding to both operators , OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Since HU does not bind to gal DNA in the absence of GalR ( 3 ) , HU binding may be dependent on direct GalR -- GalR cooperation and may stabilise an interaction which is weaker in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>binding of HU to hbs is totally dependent upon the binding of GalR to the operators , a protein-protein interaction between GalR and HU may explain the cooperativity in DNA binding ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>( 3 ) Finally , there is a tripartite cooperativity between GalR and HU in binding to gal DNA ; binding of HU to hbs is absolutely dependent on binding of GalR dimers to both operators and HU binding , in turn , results in increasing the strength of GalR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Since sitespeciﬁc HU binding to gal DNA is strictly dependent upon GalR interaction with both operators , we reinvestigated the co-operative binding of GalR using the ` HU-nuclease ' - dependent DNA cleavage as a diagnosis of GalR cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In the absence of HU or DNA 13 supercoiling the P1 promoter is repressed by a looping-independent mechanism by binding of GalR to an operator ( OE ) located upstream of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>In the absence of GalR , HU did not bind to the 20 bp DNA duplex in any detectable measure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>A model showing the DNA bent around HU bound to hbs with GalR dimers bound to OE and O facing away from each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>In this model , HU-bound GalR binds to the gal operators , thereby bringing HU to the vicinity of the gal region and facilitating its DNA binding , which in turn stabilizes DNA-bound GalR tetramerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In one model , the strong binding of GalR or GalR -- HU complex to the operator sequences precedes HU binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>HU binding , in turn , results in co-operative binding of GalR to OE and OI , presumably by an interaction between two DNA-bound GalRs as shown below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Repression of P2 requires the formation of the repressosome through cooperative binding of GalR to both OE and OI , and of HU to the interoperator segment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>This and the following results suggest that the binding of GalR to both OE and OI is required to facilitate HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In turn , HU helps co-operative binding of GalR to OE and OI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The synergistic binding of GalR and HU , the strength of HU binding , and the specificity of GalR and HU in the repressosome may originate from a functional interaction between GalR and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The simplest interpretation of such co-operative binding is an interaction between two GalR dimers in the presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Conversely , homologous repressors , like GalS and LacI , when provided with their cognate operators , can not replace GalR in bringing about HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Immunoprecipitation of GalR and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>( b ) GalR-HU complexes bind to the operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Binding of HU was totally dependent upon the binding of GalR to O and O ; conversely , HU E I increased the afﬁnity of GalR to O and O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>GalR dimer is also speciﬁc for inducing sitespeciﬁc binding of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Thus , GalR is speciﬁc in assisting HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>We report here that in restoring repression of transcription to P1 and P2 by GalR , HU binds site-speciﬁcally between the two gal operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The repression of the two gal promoters occurs due to repressosome formation when two GalR dimers bound to the operators tetramerize with the help of HU , resulting in a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The GalR and HU mediated repression of the two gal promoters requires both gal operators and is sensitive to the addition of inducer , D-galactose ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Taken together , these measurements confirm the widely held hypothesis that GalR and HU induce DNA looping in the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>S29R GalR appeared to be effective in repression when DNA looping was allowed in the presence of HU , suggesting that the mutant GalR was functional in DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>In addition , a possibility of the gal operon repression mediated by GalR in the absence of HU was shown in the new DNA constructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>To further investigate the reason for failure of repression in the AL2 loop , we constructed stereochemical models of the transient GalR -- DNA loops formed in the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Although HU has no effect on transcription in the absence of GalR , it forms a cooperative complex with GalR bound to the tandem operators , OE and OI , which effectively represses P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>In the presence of HU , GalR bound to OE and OI brings the two operators together to form a DNA loop , which simultaneously represses P1 and P2 [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>These results demonstrate that the repression of P2 by R282L in the absence of HU involves DNA looping by the bipartite operators bound by the mutant GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Lack of inhibition of P2 transcription upon destruction of OI demonstrated that the repression of transcription on the DNA templates with the inserts and two intact operators originated from DNA looping by GalR bound to the operators and were independent on HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In model B , the delivery of HU by piggyback to its destination on the DNA may be followed by one of the two scenarios ( Fig. 8B ) : ( 1 ) After HU binds to the specified position on the DNA , it dissociates from GalR , leaving behind a GalR dimer -- dimer contact required for DNA looping ; ( 2 ) HU binds to DNA while remaining GalR bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The nucleoprotein complex of GalR , HU and gal DNA prevents open complex formation We tested whether RNA polymerase binds to gal promoter in the presence of GalR and HU by DNase I protection analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Energy transfer experiments suggested that the DNA ends and the tryptophan residues in GalR2 are excluded to AEDANS labels in HU , indicating that HU binds to the GalR2-OE binary complex in the manner shown in the cartoon , i.e. , through GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>As an adaptor , HU binds simultaneously to OE - and OI-bound GalR , which are otherwise incapable of interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>We tested whether HU binds to the GalR dimer-OE complex observed above and , if so , at what stoichiometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>HU binding is entirely dependent upon binding of GalR to the two operators and is sensitive to exogenous signal D-galactose , thus revealing a novel way by which a HU-containing higher-order complex responds to transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>( ii ) The crystal structure of LacI is known .16,17 ( iii ) LacI tetramer forms stable DNA loops and does not require HU and supercoiled DNA although the latter enhances looping efﬁciency .1,33,36 Besides , LacI has higher speciﬁc binding afﬁnity to DNA than GalR .1,10,33 ( iv ) The gal promoters can be negatively regulated via DNA looping by binding of the LacI repressor to the gal operon in which the OE and OI operators were modiﬁed to be recognized by LacI .29 The C-terminal tetramerization domain of Lac repressor is ﬂexible and , therefore , depending on phasing and separation of operators along the DNA , the LacI-mediated DNA complex can potentially adopt various types of loops .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The fac - cause transcriptional repression of the gal operon , it would be tors required for gal DNA looping in single-molecule experiments possible to detect transitions between short ( looped ) and long ( HU , GalR and DNA supercoiling ) correspond exactly to those ( unlooped ) conformations of single , supercoiled gal DNA molnecessary for gal repression observed both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>The site-speciﬁc cleavages were attenuated by the addition of inducer , D-galactose ( lanes 5 and 10 ) , clearly indicating that GalR binding to the operators stimulates HU binding to speciﬁc sites in gal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The properties of site-speciﬁc stable binding of HU parallel the properties of HU in transcriptional repression -- the requirement for binding of GalR to both operators , the requirement for supercoiled DNA and the sensitivity to the presence of inducer , D-galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The chromosome contains speciﬁc structures , such as the gal repression complex , in which speciﬁc binding of HU is directed by the binding of sequence-speciﬁc regulatory proteins in the vicinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The total dependency of site-speciﬁc HU binding for gal repression and its responsiveness to inducer shows for the ﬁrst time that HU acts as an operon speciﬁc transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In this paper , we provide struc-ture-based genetic and biochemical evidence to show a specific and functional GalR -- HU interaction , both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In this paper , we report physical interactions between the different interacting partners , including GalR and HU , of the proposed repressosome complex , and determine the energetics of these interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>After induction of GalR , the cellular extracts were analyzed for GalR -- HU complexes by immunoprecipitating with anti-His antibody and immunoblotting with antiserum specific for HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>A GalR -- HU interaction was also demonstrated by sedimentation ultracentrifugation analysis and fluorescence studies ( S. Roy and M. Geanacopoulos , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>As shown in Figure 5A , lane 2 , LacIadi and HU complex formation was barely detectable under the lowest salt concentration that was used in reactions involving GalR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Equimolar amounts of GalR and HU were mixed together in buffer containing varying concentrations of salt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>( A ) A fixed concentration of GalR ( 5 nM ) was mixed with increasing concentrations of wild-type and mutant HU , as indicated in the figure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The concentration of GalR and HU was 80 nM as shown on the top .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Concentration of GalR was 80 nM ; concentrations of HU were 40 nM , 80 nM , and 140 nM , as shown at the top .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid-associated protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Three trans-acting proteins , GalR ( the gal repressor ) , cAMP receptor protein ( CRP ) - cAMP complex , and the histone-like protein HU , control transcription from these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Interestingly , HU binding in the presence of GalR neither showed protection from nor generated hyper-reactive sites by DNase I treatment ( lanes 4 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>This may be explained by the presence of the nick in these molecules , which can allow the release of any superhelical tension built up by HU binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Interestingly , the DNA cleavages caused by the EPD -- Fe ( III ) - derivatized HU pre-bound to the DNA in the presence of GalR were not affected by the subsequent addition of a 5-fold excess of ` cold ' HU before the Na-ascorbate treatment ( lane 4 ) , whereas pre-bound ` cold ' HU inhibited signiﬁcantly the cleavages by a 5-fold excess of derivatized HU added later ( lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Results and Discussion DNA molecules that were negatively supercoiled by at least 3 % ( 0.03 ) and stretched with a force ( F ) of 0.88 pN intermittently switched between two conformations in the presence of both GalR and HU ( Figs. 1b and 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Coimmunoprecipitation using purified GalR and HU with antiserum specific to GalR consistently revealed the presence of HU in these immunocomplexes , as ascertained by Western blotting with HU-specific antiserum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Our ecules under moderate tension only in the presence of the GalR single-molecule experiments revealed that negatively supercoiled and HU proteins ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>In the presence of 300 mM NaCl , the observed difference due to the absence of HU is virtually undetectable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Nevertheless , the repression of both promoters observed in the absence of HU was again not as effective as in the presence of HU ( compare second and third lanes in panels 2 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Both GalR mutants resulted in higher P2 promoter activity compared to that in the wild-type GalR in the absence of the HU protein , showing that the tetramerization interface of GalR dimers is involved in the residual repression of the P2 promoter ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>We demonstrated the ability of GalR to repress the P2 promoter by specific interaction of repressor dimers in the absence of HU ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The most effective repression of both P1 and P2 by GalR and HU occurred in the wild-type gal DNA with 113 bp distance between OE and OI , while in the plasmids with 31 -- 105 additional base-pairs the repression was partial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Presence of both GalR and wild-type HU caused simultaneous repression of both P1 and P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>( A ) Transcription of galP1 and galP2 promoters in the presence of GalR or wild-type HU and HU mutants , S17P , K18A , or T19D .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The presence of GalR ( wild-type or R282L ) and HU resulted in complete repression of both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The improper helical phasing of the operators inhibited the loopingmediated repression of P2 with wild-type GalR in the presence of HU ( Figure 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The amino acid residues S17 , K18 , and T19 in HU , which when substituted showed defect in repression of gal transcription and in interaction of GalR with HU , are located contiguously in a small turn between the first and second alpha helices ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The repression of gal transcription observed in the presence of HU is lifted by the addition of the inducer , D-galactose ( lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>In other words moderate enlargement ( up to 84 bp ) of the distance between the gal operators was an advantage to the loop formation by GalR both in the absence and presence of HU while further increase more diminished this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The most advantageous length of the insert in terms of loop formation among the insert-contain-ing templates in the presence of GalR and HU proteins was 84 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The footprinting reaction was performed in the presence of GalR with cold HU ( 100 nM ; lane 1 ) or iron -- EDTA-derivatized HU ( ~ 20 nM ; indicated in box in lanes 2 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The level of P2 transcription was the same both in the absence and presence of GalR ; the addition of HU also did not change the amount of P2 transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The assays for each plasmid were performed as described in Materials and Methods and included three transcription reactions : a control reaction without repressor GalR or HU ; a reaction in the presence of 80 nM GalR ; or in the presence of both 80 nM GalR and 80 nM HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The transcription reactions were performed in the absence and presence of 80 nM GalR and HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>The transcription reactions were performed in the absence and presence of 80 nM GalR and HU with OE -- OI DNA template , and in the presence of 80 nM LacI with OLE -- OL I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In a purified system , P2 transcription was enhanced by 80 nM GalR in the absence of HU , as observed previously , and D-galactose eliminated the activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>However , as was found in vivo , the puri ® ed R282L mutant GalR repressed transcription in the absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In this study , we report the physiological and biochemical characterization of a GalR mutant protein , which can repress gal transcription and thus form the Gal repressosome in the absence of HU and supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>In the presence of mutant GalR protein , however , transcription from both promoters decreased dramatically , in both the absence and presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>However , the mutant GalR protein , R282L , ef ® ciently repressed transcription from the altered DNA template regardless of the presence or absence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>We verified that no telegraphic signals were observed in the absence of GalR and or HU ( data not shown ) , or in the presence of 40 nM D ( ) - galactose , an inducer of gal transcription ( Fig. 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>( a ) Repression of in vitro transcription of the gal operon mediated by GalR in the absence and presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>Supercoiled P1 -- P21 gal template was incubated in the presence or absence of GalR , HU or RNA polymerase as indicated under conditions of the in vitro transcription except that dNTPs were omitted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Regulation by the iso-regulatory proteins of the gal regulon , GalR and GalS , was compared in in vitro transcription assays performed on the pRPGSM template in the presence of heat-unstable protein ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>In vitro transcription on the resulting plasmid DNAs in the presence of wild-type GalR protein showed that GalR homodimers could form repressosome and repress the gal promoters in the presence of HU ( Fig. 3 , A16T lanes 3,6,9,12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>In hupAhupB mutants proU expression is more strongly repressed , which could be due to to a higher e ¡ ective repressive activity of H-NS , whereas overproduction of HU apparently alleviates the repression by H-NS [ 68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Figure 8 shows that both HU subunits are subject to catabolic repression since both a and b subunits are less abundant when glucose is added to the medium ( lane 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>However , in the case of the three HU mutants , S17P , K18A , and T19D , when used at similar concentrations , repression of P2 was significantly reduced for all of them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Furthermore , we tested whether these HU mutants that were impaired in P2 repression of gal were compromised in their other cellular functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Genetic experiments also conﬁrmed the involvement of HU in repression of P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>As the results in Table 5 indicate , the loss of HU lowered repression fourfold ( 15 versus 56 ) when all four operators were present but had almost no effect on repression when OD4 was nonfunctional ( 5 versus 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>This is in contrast to our recent ﬁnding that IHF contributed twofold to repression of glpT in the presence of internal glpT operators while HU had no effect ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Recently , it was found that the binding of HU increased by two - to threefold the repression of the P2 promoter of the gal operon while IHF had no effect ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>HU protein was found to contribute about fourfold to repression while IHF had no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Despite the fact that HU and IHF have many structural and functional similarities , it was shown that IHF had little or no effect on glpD transcription while HU increased repression about fourfold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_437">
The substrates containing dHU are subject to the full enzymatic cycle , which includes DNA binding , N-glycosidic bond cleavage , - elimination , and product release .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></label>
<input id="act_similarity_437", type="checkbox">
<div class="ACT_similar">
<li>The substrates containing dHU are subject to the full enzymatic cycle , which includes DNA binding , N-glycosidic bond cleavage , - elimina-tion , and product release .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li></div>
<li>Endo III binding induces gross conforma ¬ 3 The abbreviations used are : dHU , 5,6-dihydrouracil ; aPu , 2-aminopurine ; tCO , 1,3-diaza-2-oxophenoxazine ; F , ( 3-hydroxytetrahydrofuran-2-yl ) - methyl phosphate ; hOGG1 , human 8-oxoguanine-DNA glycosylase ; HhH , helix-hairpin-helix ; Endo III , endonuclease III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
<li>Therefore , in the case of the dHU substrate the conformational perturbation induced in DNA by Endo III binding occur nearly simultaneously in both strands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
<li>aPu is quenched in a less polar environment , so the decrease in The variations in the quantum yield of tCO are not straightthe fluorescence of this reporter at the initial phases of reaction forward to interpret ; it seems to be stacking-dependent in sinwith the F ligand and AP and dHU substrates is obviously asso - gle-stranded DNA but relatively invariable in double-stranded ciated with the insertion of Gln41 into the space vacated by the DNA , and , if tCO is located between two G bases , the fluores-everted nucleotide and stacking of its amide group against aPu .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
<li>Therefore , we hypothesize that the first and AP tend to be intrahelical or slightly displaced toward the decrease in fluorescence in cases of F ligand and AP and dHU major or minor groove when the opposite base is well stacked substrates may be associated with DNA kinking upon Endo III FIGURE 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">HU protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Therefore , we constructed 12-base pair oligodeoxyribonucle-otide duplexes with an aPu reporter on the 3 - side of the site of unmodified G , or modified F , AP , and dHU residues as specific ligands and substrates for Endo III ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
<li>An aPu residue placed opposite dHU was insensitive to the binding by Endo III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25869130" target="_blank">25869130</a>]<br></li>
<li>The steric consequences of only one or two UP1 molecules bound to the DHU arm ( 17 residues ) could have a strong destabilizing effect on the interaction between the anticodon loop and the TqC loop , lowering the barrier enough for renaturation to proceed rapidly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6178431" target="_blank">6178431</a>]<br></li>
<li>conformer ( DHU loop and stem and CCA terminus ) are already accessible as single-stranded segments ( Uhlenbeck et al. , 1974 ) , and upon interaction with UP1 a third of the residues heretofore in base-paired structure become unpaired ( Karpel & Burchard , 1980 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6178431" target="_blank">6178431</a>]<br></li>
<li>However , due to our lack of knowledge about the RNA-binding domain of HU , such mutants are not available .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Ye , K. , Serganov , A. , Hu , W. , Garber , M. & Patel , D.J. Ribosome-associated factor Y adopts a fold resembling a double-stranded RNA-binding domain scaffold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15502846" target="_blank">15502846</a>]<br></li>
<li>Karginov , F.V. , Caruthers , J.M. , Hu , Y. , McKay , D.B. and Uhlenbeck , O.C. ( 2005 ) YxiN is a modular protein combining a DEx ( D/H ) core and a speciﬁc RNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734347" target="_blank">19734347</a>]<br></li>
<li>Wang , S. , Hu , Y. , Overgaard , M.T. , Karginov , F.V. , Uhlenbeck , O.C. and McKay , D.B. ( 2006 ) The domain of the Bacillus subtilis DEAD-box helicase YxiN that is responsible for speciﬁc binding of 23S rRNA has an RNA recognition motif fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734347" target="_blank">19734347</a>]<br></li>
<li>Another recent study demonstrates that in vivo the transcriptional-activity of T7 RNA polymerase is enhanced by the presence of HU , and that this stimulation most likely occurs at the level of initiation [ 73 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Purification of HU The wild-type hupA and hupA genes containing the mutations S17P , K18A , and T19D , after PCR amplification with primers containing Nde1 and BamH1 sites at the N-terminal and Cterminal ends , respectively , were cloned in expression vector plasmid pET15b ( Novagen ) , which contains an N-terminal hexa-histidine tag .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Fraction II contains ( Yand p ; fraction I contains cu ; fraction III contains p. Molecular weight markers are : E. cc & HU ( 1OK ) ; E. cofi helix destabi ~ ing protein ( 22 000 ) and E. coli RNA polymerase ( a subunit 39 000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=227733" target="_blank">227733</a>]<br></li>
<li>ncRNAs may interact simultaneously with BIME DNA sequences around the chromosome and HU to form ternary complexes potentially playing the role of anchoring points , which deﬁne different chromosomal domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Based on a modeled structure of E. coli HU homodimer derived from the Bacillus stearothermophilus HU homodimer X-ray and nuclear magnetic resonance structure ( Tanaka et al. 1984 ; Vis et al. 1995 ) , we identified 10 amino acid residues for targeted mutagenesis , following two criteria : ( 1 ) the amino acid residues were surface exposed , and ( 2 ) they were not on the DNA-bind-ing - sheet arms or the dimerization interface of the protein ( Nash 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>In the modeled structure of the HU dimer , the three residues , S17 , K18 , and T19 , whose replacements resulted in defective transcriptional repression and GalR interaction , are prominently exposed in a peptide turn between two alpha helices on the face of HU near the N-terminal region and farthest from the DNA-binding C-domain and are accessible for interaction with other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>These looping-defective amino acid substitutions in the GalR dimer were located in the DNA-distal domain , which preferably de ® ned a GalR ± GalR rather than a GalR ± HU contact area .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>By site-directed mutagenesis , we scanned the surface of the HU homodimer to identify amino acid residues that , when altered , would be defective in DNA looping without the loss of their intrinsic DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Three-dimensional model of the HU dimer showing the location and spatial arrangement of the substituted residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The amino acid residues S17 , K18 , and T19 in HU , which when substituted showed defect in repression of gal transcription and in interaction of GalR with HU , are located contiguously in a small turn between the first and second alpha helices ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Amino acid substitutions in HU Position Native amino acid Substituted amino acid The substituted amino acids were chosen from amino acids at the corresponding positions in HU from other species ( see text ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The amino acid residues at the chosen positions were substituted by residues found at corresponding positions in HU of other bacterial species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The list of native and substituted amino acid residues in HU is in Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Protein HU is indeed a heterodimer constituted of two polypeptide chains , termed HU-1 and HU-2 , of 90 residues each with closely related amino acid sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6987059" target="_blank">6987059</a>]<br></li>
<li>COMPLETE SEQUENCE OF PROTEINS HU-1 AND HU-2 The automated Edman degradation of the dimer HU allowed us to establish a sequence of 50 amino acid residues in which the identification of two residues at positions 4 , 9 , 12 , 13 , 15 , 16 , 20 , 22 , 23 , 26 , 29 , 31 , 32 , 41 , 43 led us to postulate the presence of two polypeptide chains of closely related sequence in the dimer H U [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6987059" target="_blank">6987059</a>]<br></li>
<li>Wild-type HU does not have any cysteine residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Protein HU is characterized by a high content in alanine and lacks cysteine , tyrosine and tryptophan [ l ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6987059" target="_blank">6987059</a>]<br></li>
<li>The protein HU is rich in alanine and lysine and is devoid of cysteine , tyrosine and tryptophan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=350619" target="_blank">350619</a>]<br></li>
<li>From the specificity of trypsin , the tryptic peptide T : z lacking a lysine or arginine residue must be assigned to the carboxy terminal position in the protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=350619" target="_blank">350619</a>]<br></li>
<li>Reactions in the right panel contained 80 nM HU protein to allow looping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>This is the first example of HU making a specific contact with another protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Discussion The sequence of the first 37 amino acids of the protein HU contains 19 hydrophobic residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=350619" target="_blank">350619</a>]<br></li>
<li>Carboxy-teminal sequence The hydrolysis of the protein HU with the carboxy-peptidase B for 2 h did not release any basic residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=350619" target="_blank">350619</a>]<br></li>
<li>The hydrolysis of the protein HU with the carboxy-peptidase A for 4 h released the following residues ( mol/mol protein ) : asparagine 0.48 ; valine 0.13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=350619" target="_blank">350619</a>]<br></li>
<li>However , IHF and HU share extensive sequence homology , especially in their DNA-binding domains , and both proteins are believed to interact with DNA in the minor groove ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>By analogy to the protein structure of HU , it has been proposed that an IHF-induced DNA bend may be facilitated by the interaction of two antiparallel @ - stranded arms , one extending from each subunit of the protein , with the minor groove of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Overall , the results suggest that the interaction-with-DNA does not influence the quaternary structure of H-NS and that the function of this protein does not entail a direct interaction with NS ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3135462" target="_blank">3135462</a>]<br></li>
<li>hioi Rev 51,301-319 7 Rouvi ~ re-Yaniv J. Yaniv M , Germond JE ( 1979 ) E cnli DNA-binding protein HU forms nucleosome-like structures with circular double-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>E. coli DNA-binding protein Hu forms nucleosome-like structure with circular double stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1688424" target="_blank">1688424</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_38">
Escherichia coli DNA-binding protein HU forms nucleosome-like structure with circular double-str & ed DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></label>
<input id="dom_similarity_38", type="checkbox">
<div class="DOM_similar">
<li>E. coli DNA-binding protein HU forms nucleosome-like structure with circular double-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>E. coli DNA-binding protein HU forms nucleosome like structure with circular double-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li></div>
<li>Escheridzia cdi contains an abundant DNA-binding protein , HU , [ l ] with a monomeric mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=227733" target="_blank">227733</a>]<br></li>
<li>Based on nucleotide and amino acid comparisons , IHF is assumed to be structurally related to the bacterial histone-like protein HU , a type II DNA-binding protein for which the crystal structure has been determined ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Previous studies of structural motifs important for HU binding revealed that HU recognizes a stem-loop structure ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Wojtuszewski , K. , and Mukerji , I. ( 2004 ) The HU -- DNA-binding interaction probed with UV resonance Raman spectros-copy : structural elements of speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>An adaptor role for HU raised the question of whether DNA-bound GalR directly interacts at all in loop closure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Looping was presumably mediated by an interaction between the two operator-bound GalR , with HU facilitating the interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The simplest interpretation of such co-operative binding is an interaction between two GalR dimers in the presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Although a specific interaction of GalR and HU was shown to be essential for loop formation , the HU-GalR contact is unlikely to be present in the repressosome ; it appears that GalR piggybacks HU to its architecturally critical binding site on DNA ( Kar and Adhya 2001 ; S. Roy , E. Dimitriadis , S. Kar , M. Geanaco-poulos , M.S. Lewis , and S. Adhya , in prep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Genetic and biochemical studies suggested that the higher order Gal repressosome structure , which governs the transcription of two tandem gal promoters in Escherichia coli , involves sequence-specific binding of GalR repressor dimers to two operators , OE and OI , located 113 bp apart , binding of GalR to the sequence-nonspecific DNA-binding protein HU , interaction of HU with an architecturally critical DNA site between the two operators , and interaction between two DNA-bound GalR dimers generating a loop of the intervening DNA segment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>We assume that under normal conditions the DNA loop is formed by transient interaction of operator-bound GalR dimers , and HU plays an architectural role by stabilizing the repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>The GalR -- HU contact that results in cooperative binding of the two proteins to DNA may be transient and absent in the final repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>The DNA-bound HU could then either maintain or lose contact with GalR in the final repressosome structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Although further studies are needed to distinguish between these two models , we note that energetic considerations favor a model in which HU is not in contact with GalR in the final repressosome structure ( Geanacopoulos et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>By similarly characterizing a HU mutant , we confirm a physiological role of the specific GalR-HU contact in repressosome formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>The demonstration of GalR tetramer formation by ultracentrifugation studies ( Table 1 ) , the ability of mutant GalR proteins to interact with each other in the absence of HU using a genetically defined interface ( 23 ) , and the existence of a specific HU-DNA contact between the two operators in repressosome formation ( 13 ) rule out a simple adapter model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>monomer ( d ) Partial m , odel for H L-D , VA interactions It is premature to propose a specific molecular model for the interaction of HU protein and DNA because several important details about the complex are still unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>HU was found to play important roles in the transposition of bacteriophage Mu by formation of a higher-order structure and site-speciﬁc inversion of the ﬂagel-lin promoter by assisting DNA loop formation ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>We propose that HU , in concert with GalR , forms a speciﬁc nucleoprotein higher order complex containing a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The three forms of HU migrated close to one another , but were separated enough to be ident-i ® ed ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>The three forms of HU are not equivalent , since b2 is incapable of promoting formation of DNA supercoiling like ab and a2 do .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Essentially only two forms of HU are detected , the homodimeric a2 and the heterodi-meric ab .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Since an HU heterodimer contacts two minor grooves in DNA through polypeptide loops , the center of gal DNA occupied by HU would be located at position 16.5 ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The marked segment below the horizontal double helix indicates the region -- 12 to 16 in contact with the HU α-subunit as discussed in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The ﬁlled circle shown in the double helix represents the center of the segment that is in contact with HU heterodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>( 5 ) According to biochemically established topological features of representative pro-karyotic APC superfamily members , both the N and C termini of the proteins are located in the cytoplasm with a 12 TMS topology ( Cosgriff & Pittard , 1997 ; Ellis et al. , 1995 ; Hu & King , 1998a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931886" target="_blank">10931886</a>]<br></li>
<li>Overproduction of a MalE-MinCN fusion blocks Z ring formation in vivo and the purified protein antagonizes FtsZ polymer assembly in vitro without affecting the GTPase activity of FtsZ ( Hu & Lutkenhaus , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>MinC is composed of two domains with distinct functions and both domains have division inhibitory activity when overexpressed : MinCN blocks Z ring formation in vivo and disrupts FtsZ polymer assembly in vitro as assayed by sedimentation ( Hu & Lutkenhaus , 2000 ) ; MinCC does not affect FtsZ sedimentation but decreases filament bundling and can localize to Z rings and block cell division when MinD is present ( Hu & Lutkenhaus , 2000 , Shiomi & Margolin , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>[ PubMed : 10611296 ] Hu Z , Lutkenhaus J. Analysis of MinC Reveals Two Independent Domains Involved in Interaction with MinD and FtsZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24930948" target="_blank">24930948</a>]<br></li>
<li>[ PubMed : 10540287 ] Hu Z , Lutkenhaus J. Analysis of MinC reveals two independent domains involved in interaction with MinD and FtsZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>To compare the activity of the two domains of MinC in the same context and under more physiological conditions , we took advantage of the two MinC mutants ( MinC-G10D and MinC-R172A ) described previously ( Hu et al. , 1999 , Zhou & Lutkenhaus , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>The N-terminal domain of MinC inhibits FtsZ polymerization by binding to an alpha-helix of FtsZ at the interface of FtsZ subunits in a filament ( Hu et al. , 1999 ; Shen and Lutkenhaus , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25271375" target="_blank">25271375</a>]<br></li>
<li>To test this hypothesis , we used a MinC mutant ( MinC-G10D ) with lower inhibitory activity of its N-terminal domain ( Hu et al. , 1999 ; Shen and Lutkenhaus , 2009 , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25271375" target="_blank">25271375</a>]<br></li>
<li>FtsZ-I374V is still sensitive the N-terminus of MinC MinC has two functional domains and each domain affects FtsZ function but the mechanism of action is quite different ( Hu & Lutkenhaus , 2000 , Shiomi & Margolin , 2007 , Hu et al. , 1999 , Dajkovic et al. , 2008a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>Structural , sequence and functional analyses of MinC reveal that it has two domains of approximately equal size ( Cordell et al. , 2001 , Hu & Lutkenhaus , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>MinC has a C-terminal and an N-terminal domain , ( Hu and Lutkenhaus , 2000 ; Cordell et al. , 2001 ) which are both involved in FtsZ inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25271375" target="_blank">25271375</a>]<br></li>
<li>a Results Chimeric XNtrC proteins are active in vivo and in vitro To test the dimerization properties of various response regulator domains , we used the X system introduced by Hu et al. ( 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Plasmid pJH391 ( Hu et al. , 1990 ) carries a fragment of the Wc ind I gene ( bp 1-396 ) encoding the N-terminal domain and the linker domain of k repressor fused to a truncated form of lacZ ( bp 4-1958 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>For example , Hu , which is required for efficient DNA replication in vitro , contains 11 % lysine residues ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537812" target="_blank">2537812</a>]<br></li>
<li>It is also noteworthy that the amino group of the repositioned lysine is located at the same position as one of the sodium ions in the sodium-dependent bile acid symporter ASBTNM , a structural homologue of NhaA and NapA ( Hu et al. , 2011 ; Fig. 9 B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25422503" target="_blank">25422503</a>]<br></li>
<li>Comparison with the literature As already mentioned , Hu et al. [ 4 ] also performed a sitedirected mutagenesis study of invariant residues T15 , H18 Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17692452" target="_blank">17692452</a>]<br></li>
<li>Using the NMR structure of the TatAd protein from Bacillus subtilis as a template ( Hu et al. , 2010 ; PDB code 2l16 ) , we generated a homology model of the transmembrane helix of E. coli TatB to visualize the positions of TatB residues involved in TatB -- TatB and TatB -- TatC interactions ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22742417" target="_blank">22742417</a>]<br></li>
<li>This region of s is located in the C-terminal 70 domain ( CTD ; amino acids 590 ± 613 ) , which also contains residues required for transcription-activation by Ada ( Landini and Busby , 1999 ) , lcI ( Kuldell and Hochschild , 1994 ; Li et al. , 1994 ) , AraC ( Hu and Gross , 1985 ) , RhaS ( Bhende and Egan , 2000 ) and the cyclic AMP receptor protein ( CRP ) ( Lonetto et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>It encodes the DNA-binding protein lcI with the C-terminus domain fused to a truncated version ( residues 831 to 1057 ) of the b subunit of RNAP connected by a short three alanine linker region ( Dove & Hochschild , 2004 ; Hu et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25998262" target="_blank">25998262</a>]<br></li>
<li>Structural data concerning the mutated residues and enzymatic effect of the mutations Residue Domain Motif X-ray analysis kcat/Km , this study ( % ) a b c d e kcat/Km , Hu et al. [ 4 ] ( % ) e UDP-GlcNAc Lipid I analogue UDP-GlcNAc Lipid I analogue N T15 I 0.10 0.10 0.072 0.25 Belongs to the GGTGGH loop and to the putative acceptor-binding site Belongs to the GGTGGH loop and to the putative acceptor-binding site Belongs to the GGY loop and to the putative acceptor-binding site Belongs to the HEQN loop and to the putative acceptor-binding site Belongs to the HEQN loop and to the putative acceptor-binding site Belongs to the HEQN loop and to the putative acceptor-binding site ; H-bonded with the C4 hydroxyl of GlcNAc Belongs to the a5 helix ; structural role Belongs with the GGS loop ; H-bonded with the b-phosphate of UDP-GlcNAc Belongs to the a1 helix ; structural role End of the b4 sheet ; structural role H-bonded with the 20 and 30 hydroxyls of ribose H-bonded with the C3 hydroxyl of GlcNAc H-bonded with the C4 hydroxyl f f I N H18 0.054 0.17 N.D. g II N Y105 9.3 8.2 N.D. II N H124 0.13 N.D. N.D. 0.072 II N E125 0.48 N.D. N.D. 0.055 f f II N N127 1.3 1.0 h h N134 II N N.D. N.D. C S191 0.34 III 5.2 0.097 0.72 2.4 N198 III C N.D. N.D. 3.6 0.61 0.51 R260 IV C N.D. N.D. f f IV C E268 3.7 70 f f V C Q288 0.36 1.1 V C N291 N.D. 0.25 2.1 N.D. of GlcNAc a See Section 3 for explanation of the amino acid numbering .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17692452" target="_blank">17692452</a>]<br></li>
<li>Retailleau , P. , Weinreb , V. , Hu , M. and Carter , C.W. ( 2007 ) Crystal structure of tryptophanyl-tRNA synthetase complexed with adenosine-5 ' tetraphosphate : evidence for distributed use of catalytic binding energy in amino acid activation by class I aminoacyl-tRNA synthetases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23275545" target="_blank">23275545</a>]<br></li>
<li>1999 Identiﬁcation of amino acid residues responsible for the pyrimidine and purine nucleoside speciﬁcities of hu - + man concentrative Na nucleoside cotransporters hCNT1 and hCNT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16718600" target="_blank">16718600</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_87">
D. Kiefer , X. Hu , R. E. Dalbey , and A. Kuhn , Negatively charged amino acid residues play an active role in orienting the Sec-independent Pf3 coat protein in the Echerichia coli inner membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11051763" target="_blank">11051763</a>]<br></label>
<input id="dom_similarity_87", type="checkbox">
<div class="DOM_similar">
<li>Kiefer , D. , Hu , X. , Dalbey , R. , and Kuhn , A. ( 1997 ) Negatively charged amino acid residues play an active role in orienting the Sec-independent Pf3 coat protein in the Escherichia coli inner membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22179606" target="_blank">22179606</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">HU evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>This is why the FCD family member identiﬁed to be most similar to McbR using the DALI structural homology search database changes depending on whether the search is performed with the McbR FCD domain alone ( PS5454 , PDB ID : 3C7J ) or fulllength McbR ( , PDB ID : 3IHU ; Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>FCD Family Members and Their Structural Similarity to McbRa FCD only full-length name PDB Z rmsd ( Å ) Z rmsd ( Å ) Id ( % ) subfamily metal binding FCD ligandsb McbR 4P9F VanR no UNK PS5454 3C7J 18.2 1.9 18.3 1.9 23 VanR yes Ni CGL2915 2DI3 15.5 2.6 15.5 4.0 20 FadR yes Zn RO03477 2HS5 15.4 2.6 19.7 3.1 19 VanR no Act 3IHU 15.4 2.5 20.3 2.6 16 VanR no TM0439 3FMS 5 14.8 2.4 4 16.6 3.1 22 VanR yes Act , Ni FadR 1H9G 6 12.8 2.6 6 12.6 6.2 14 FadR no CoA-Myr a R , Z-score rank .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25376905" target="_blank">25376905</a>]<br></li>
<li>Its hyphenated dyad symmetry resembles that of the binding sites for other regulators with helix-turn-helix motifs ( e.g , CRP and FNR ) but it differs from the known sites for other members of the GntR family which exhibit uninterrupted dyad symmetry , i.e. GntR -LCB- ATACnGTATACAAGTAT : Fujita and Miwa , 1989 ) and HutCpp -LCB- TTGJAATAtAc ; Hu et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7783622" target="_blank">7783622</a>]<br></li>
<li>D , Acid/urea/Triton-X 100/polyacryl-amide gel electrophoresis of the fractions ( 43 , 45 , 47 , 49 , 51 , 53 and 55 ) belonging to the two peaks of protein corresponding to the b and a subunit of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>cReported TBLASTN identity to HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>cReported TBLASTN identity to each NAP 53 HU 57 NC 009140 176473 14067 14333 99 HU 15070 148225 15336 148626 57 61 99 99 H-NS H-NS 111747 111346 64 99 query coverage ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>Our results ( fig. 2 ) show that the mixed dimer accounts for -90 % of the HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=227733" target="_blank">227733</a>]<br></li>
<li>Enteric bacteria have a hetero-dimeric HU ( ) that consists of two 9.5 kDa subunits , which share 70 % homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>HU and IHF have a similar structure consisting of a globular body from which two - ribbon arms protrude ( Fig. 2D and E ) and share about 30 -- 40 % homology ( Swinger et al. , 2003 ) HU binds non-speciWcally to DNA , but has a higher aYnity for supercoiled DNA and distorted DNA structures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>HU/IHF HU and its sequence speciﬁc homologue IHF consist of two ( identical or homologous ) subunits ( Swinger and Rice , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>IHF and HU monomers share 30 % sequence identity and are related in both structure and function ( 20,31,33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Interestingly , it has been shown that HU can be functionally replaced by members of the eukaryotic family of HMG1 / 2-proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Examples of members of this family include HU , IHF , H-NS , and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1459953" target="_blank">1459953</a>]<br></li>
<li>Fis is a member of the general family of nucleoid-associated proteins that includes HU , IHF , H-NS , LRP , and Dps ( 2 , 35 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768815" target="_blank">7768815</a>]<br></li>
<li>Role of his-tone-like proteins OsmZ and HU in homologous recombination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>IHF is a small heterodimeric protein that belongs to a class of prokaryotic histone-like proteins , together with HU , H-NS and FIS ( Schmid , 1990 ; Schmid & Johnson , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>B himA : + hip : - f-pDRC158 : ¬ IFI'G : 1 -60 - - 4 0 * - 4 0 * - 8 0 * - 2 0 * Discussion IHF belongs to a group of histone-like proteins which , together with HU , H-NS and Fis , appear to participate in the three-dimensional organization of the bacterial chromosome ( Driica and Rouviere-Yaniv , 1987 ; Schmid , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>The same extracts were also immunoblotted with anti-IHF antibody to check whether GalR coimmunoprecipitated with IHF , a protein that is highly homologous with HU ( Oberto et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>( 1990 ) The family of repetitive extragenic palindromic sequences : interaction with DNA gyrase and histone like protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The family of repetitive extragenic palindromic sequences : interaction with DNA gyrase and histonelike protein HU , p. 211-225 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400248" target="_blank">1400248</a>]<br></li>
<li>As the amino acid sequence of the subunits of IHF and those of the HU family are very similar , Yang and Nash ( 1989 ) proposed a model for the IHF-DNA interaction in which the crystal structure of HU from Bacillus stearothermophilus ( Tanaka et ai , 1984 ) was used to approximate the structure of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF as an asymmetric transcription factor in contrast to HU , the subunits of IHF show only a mode-rate ( 25 % ) identity at the amino acid level , indicating considerable evolutionary divergence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The complete amino acid sequences of these two proteins were determined [ 6 ] and found to be highly homologous with the N-terminal sequences of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=227733" target="_blank">227733</a>]<br></li>
<li>Integration host factor ( IHF ) is a member of a family of DNA-bending proteins that share amino acid similarity with the bacterial HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>The α - and β-subunits of the HU protein show approximately 70 % amino-acid-sequence similarity and HU can exist as an α-homodimer , a β-homodimer or an αβ-heterodimer87 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by HU</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>A , Aldohexuronic transport system ( THU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7007313" target="_blank">7007313</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , May 1998 , p. 2549 -- / 98 / $ 04.0010 Copyright © 1998 , American Society for Microbiology Regulation of a New Cell Wall Hydrolase Gene , cwlF , Which Affects Cell Separation in Bacillus subtilis SHU ISHIKAWA , YOSHIKO HARA , RYO OHNISHI , AND JUNICHI SEKIGUCHI * Department of Applied Biology , Faculty of Textile Science and Technology , Shinshu University , 3-15-1 Tokida , Ueda-shi , Nagano 386 , Japan Received 29 September 1997/Accepted 15 February 1998 Bacillus subtilis produces a 35-kDa cell wall hydrolase , CwlF , during vegetative growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573210" target="_blank">9573210</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_3">
E. coli strains harbored plasmids expressing the designated genes as follows : pAHG , UbiGK-12 ; pUbiGTHU , UbiGTHU ; pHZ1 , UbiXTHU ; and pPZ2 , UbiXK-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16923914" target="_blank">16923914</a>]<br></label>
<input id="rp_similarity_3", type="checkbox">
<div class="RP_similar">
<li>E. coli strains harbored plasmids expressing the designated genes as follows : pAHG , UbiGK-12 ; pUbiGTHU , UbiGTHU ; pHZ1 , UbiXTHU ; and pPZ2 , UbiXK-12 FIG. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16923914" target="_blank">16923914</a>]<br></li></div>
<li>To test whether Cpx-mediated downregulation of these genes could be involved in antibiotic resistance , we examined the impact of de-leting these genes on resistance to the aminoglycoside amikacin , to which hyperactivated CpxA * mutants are known to be resistant ( 86 ) , and the drug hydroxyurea ( HU ) ( resistance to HU is also impacted by mutations to cpxA ) ( 84 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>Interestingly , the randomly selected control gene ydiY also caused a slight increase in resistance to amikacin and HU ( Fig. 3B ) , although other strains carrying kanamycin insertion cassette mutations did not ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>To investigate the possibility that the HU genes may also be under stringent control , we determined the response of the hup genes to amino acid starvation , which induces the stringent-response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The HU regulon is composed of genes responding to anaerobiosis , acid stress , high osmolarity and SOS induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>HU constrains negative supercoifs in DNA ( Broyles and Pettijohn , 1986 ) and is inencoding gene ; hupA and hupB , genes encoding HU-2 and HU - 1 , respectively ; Hy , hygromycm B ; IHF , integration host factor ; himA and himD , genes encoding IHFa and IHFB , respectively ; kb , kiiobase ( sf or 1000 bp ; Km , kan ~ y ~ in ; LB , Lucia-Bertani ( medium ) ; nt , nucleotidefs ) ; ohgo , oligodeoxyribonucleotide ; PA , pofyacfylam ~ de ; PAGE , PA-gel etectro-phoresis ; R , resistance/resistant ; s , sensitivity/sensitive ; SDS , sodium dodecyl sulfate ; Tc , tetracycline ; trp $ genes encoding enzymes for tryptophan-synthesis ; wt , wild type ; : : , novel junction ( fusion or insertion ) ; [ 1 , denotes plasmid-carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Like the hns and stpA genes , the genes that encode the HU subunits , hupA ( α-subunit ) and hupB ( β-subunit ) , are subject to differential regulation87 ,88 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>Bacterial strains disrupted in genes encoding nucleoid proteins Strain Relevant genotype Designation Comment Strep derivative of CSH142 [ araD ( gpt-lac ) 5 ] R FW102 hupA hupB FW102 hns FW102 himD FW102 a BL643 BL706 BL707 WT HU U heterodimer Loss of both HU-1 and HU-2 subunits of H Loss of H-NS homodimer Loss of IHFb subunit of IHF heterodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>For example , cAMP-CRP , GlcIIa , BarA , polyphosphate , homoserine lactone , acetate , and the NADH/NAD ratio have been shown to affect rpoS transcription , whereas DnaK , DksA , Hfq , HU , StpA , LeuO , and several small regulatory RNAs control rpoS translation ( Hengge-Aro-nis 2002 ; Repoila et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17403784" target="_blank">17403784</a>]<br></li>
<li>`` Recon proteins '' indicates theinclusion of a mixture of DnaB helicase , DnaC , DnaG primase , SSB , HU , gyrase , pol III * , and b subunit of pol III * ( see Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9674428" target="_blank">9674428</a>]<br></li>
<li>Table 1 Correspondence among replication proteins from Escherichia coli , Bacillus subtilis , and bacteriophages T7 and T4 phage T4 phage T7 B. subtilis E. coli Function PolC , DnaEBS b2 t5 RTP2 SSB4 DnaA , B , D DnaC6 DnaI DnaG polymerase/exonuclease sliding clamp clamp loader terminator SSB initiator replicative helicase helicase loader primase aey b2 gp43 gp453 gp5 Trx ( E. coli ) -- -- gp2 .52 T7 RNAP gp4 -- gp4 dgt2d ' cw Tus gp444gp62 -- gp32 -- gp41 gp59 gp6 SSB4 DnaA , IHF , HU , FIS DnaB6 DnaC DnaG Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>Concerning HU , a recent transcriptome analysis of its regulon showed that HU activates nrf operon expression during exponential growth ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Besides induction ( loop breakdown ) of the operon by D ( ) - galactose , which acts by inactivating GalR , the loop must break down for expression in response to anabolic needs resulting from changes in DNA superhelicity or HU concentration ( 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Ferris HU , Furukawa Y , Minamino T , Kroetz MB , Kihara M , Namba K , Macnab RM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22435757" target="_blank">22435757</a>]<br></li>
<li>The fact that dnaA mutations that reduce the efﬁciency of initiation substantially suppress the sensitivity of seqA mutants to UV , AZT and HU indicate that SeqA 's role in controlling replication initiation is primarily responsible for promoting survival of DNA damage or replication inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>Mini-Pl plasmid replication and Mu bacteriophage development in looping-defective HU mutants No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>For example , in the double deletion mutant lacking both HU proteins , or/2-dependent DNA-replication of mini-F plasmids ( Wada et al. , 1988b ) , replicative transposition of bacteriophage Mu ( Kano et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>Indeed , HU can replace IHF in a number of systems including IS70 transposition and DNA-replication at oriC in E. co / / ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Furthermore , biochemical and genetic studies have shown that HU participates in several specific processes , including oriC-dependent DNA-replication , hin-mediated gene inversion , transposition of bacteriophage Mu , and the transposon 10 and DNA-repair ( 5 -- 9 ) , probably as a component of active multiprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>In vitro and in vivo studies suggest that HU is involved in several fundamental biological functions , including the initiation of DNA-replication ( 3 , 13 , 22 ) , sitespeciﬁc DNA inversion ( 34 , 49 ) , and transposition ( 21 , 34 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>Relative plating , Strain Genotype 5 mM HUa MG1655 NK6367 STL7210 STL7375 STL7377 STL7798 ( wild-type ) seqA10 0.26 0.042 0.041 0.34 0.053 0.036 dam : : EZ-Tn5 mutS215 : : Tn10 dam : : EZ-Tn5 mutS215 : : Tn10 dam : : EZ-Tn5 seqA10 Dam and SeqA epistasis Results Sensitivity of dam methylase mutants to replication inhibitors and DNA damaging agents , independent of mismatch-repair To isolate new factors involved in DNA-repair or damage tolerance , we screened EZ-Tn5 insertion mutants in a recAts background for increased sensitivity to UV or to the replication inhibitor HU at 30 ° C and 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>To identify new factors involved in replication fork repair , we undertook a genetic screen for Escherichia coli mutants sensitive to UV irradiation , which produces repli-cation-blocking pyrimidine dimers , or the replication elongation inhibitors , hydroxyurea ( HU ) and azidothymidine ( AZT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>We examined mutations in the radA/sms gene of Escherichia coli for effects on conjugational recombination and sensitivity to DNA-damaging agents , including UV irradiation , methyl meth-anesulfonate ( MMS ) , mitomycin C , phleomycin , hydrogen peroxide , and hydroxyurea ( HU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446634" target="_blank">12446634</a>]<br></li>
<li>Sensitivity to various types of DNA-damaging agents , such as UV irradiation , MMS , hydrogen peroxide , hydroxyurea ( HU ) , and phleomy-cin , were assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446634" target="_blank">12446634</a>]<br></li>
<li>This last characteristic might explain at least some of the numerous functions that HU has in the cell such as its role in recombination , DNA inversion ( see for a review Rouviere-Yaniv et al. , 1992 ) and DNA-repair ( Boubrik & Rouviere-Yaniv , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>HU thus plays an architectural role , as has been reported in nucleoprotein complexes involved in recombination and replication ( Johnson et al. , 1986 ; Hodges-Garcia et al. , 1989 ; Baker and Mizuuchi , 1992 ; Hwang and Kornberg , 1992 ; Lavoie and Chaconas , 1994 ; Seagall et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>The bending induced by HU facilitates optimal protein -- protein and/or protein -- DNA contacts in an otherwise physically recalcitrant structure , thereby orchestrating the regulated assembly of functional nucleoprotein complexes , such as those involved in DNA-replication , recombination , transposition , transcription , and DNA-repair ( Skarstad et al. 1990 ; Dri et al. 1992 ; Haykinson and Johnson 1993 ; Manna and Gowrishankar 1994 ; Li and Waters 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Several DNA-binding proteins , including H-NS ( Suzuki et al. , 1996 ) , HU ( Painbeni et al. , 1997 ) , Lrp ( Ferrario et al. , 1995 ) , Rob ( Ariza et al. , 1995 ) , SoxS ( Chou et al. , 1993 ) and MarA ( Cohen et al. , 1988 ; Jair et al. , 1995 ) , are known to regulate micF promoter activity and , consequently , modulate ompF mRNA translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Interestingly , expression of mutant HU induces activation of a subset of genes that is normally silenced by H-NS ( Kar et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>More recently , FIS has been shown to modulate expression of some RpoS-regulated genes ( Xu and Johnson , 1995a , b , c ) , to regulate the promoters of the hupA and hupB genes encoding HU ( Claret and Rouviere-Yaniv , 1996 ) and of the hns gene encoding H-NS ( Falconi et al. , 1996 ) , and to regulate expression of several operons involved in the catabolism of sugars and nucleic acids ( González - Gil et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582281" target="_blank">9582281</a>]<br></li>
<li>For instance , combined evidence from our work and that of others shows that Fis mediates the regulation of other transcription regulators , such as GadE , CusR ( Table 2 ) , cyclic AMP receptor protein ( CRP ) , H-NS and HU ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Fis can also affect DNA supercoiling indirectly , because of its roles in repressing transcription of gyrA and gyrB to reduce the DNA gyrase activity ( Schneider et al. , 1999 ) , increasing topA transcription to enhance topoisomerase I activity ( WeinsteinFischer et al. , 2000 ) , and regulating the expression of other DNA-binding proteins such as H-NS , HU and CRP ( Claret & Rouviere-Yaniv , 1996 ; Falconi et al. , 1996 ; Gonzalez-Gil et al. , 1998 ) that also affect DNA supercoiling directly or indirectly ( Broyles & Pettijohn , 1986 ; Gomez-Gomez et al. , 1996 ; Tupper et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Proc Natl Acad Sci USA 95 : 224 -- 228 Laursen BS , Sorensen HP , Mortensen KK , Sperling-Petersen HU ( 2005 ) Initiation of protein-synthesis in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16362046" target="_blank">16362046</a>]<br></li>
<li>Proc Natl Acad Sci USA 54:572 -579 Overath P , Pauli G , Schairer HU ( 1969 ) Fatty acid degradation in Escherichia coli : an inducible acyl CoA synthetase , the mapping of oM-mutations and the isolation of regulatory mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3116274" target="_blank">3116274</a>]<br></li>
<li>In this scenario , HU plays an architectural role in Gal repressosome formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_39">
Histone-lik protein HU as a speciﬁc transcriptional regulator : cofactor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></label>
<input id="rp_similarity_39", type="checkbox">
<div class="RP_similar">
<li>Histone-like HU as a speciﬁc transcriptional regulator : cofactor role in the repression of gal transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_41">
Histone-like protein HU as a speciﬁc transcriptional regulator : co-factor role in repression of gal transcription by GAL repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></label>
<input id="rp_similarity_41", type="checkbox">
<div class="RP_similar">
<li>Histone-like protein HU as a speciﬁc transcriptional regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Histone-like protein HU as a speci ® c transcriptional regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Histone-like protein HU as a specific transcriptional regulator : Cofactor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>Histone-like protein HU as a specific transcriptional regulator : Co-factor role in repression of gal transcription by gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Histone-like protein HU as a specific transcriptional regulator : Co-factor role in repression of gal transcription by GAL repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li></div>
<li>Aki , T. , Choy , H. E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a specific transcriptional regulator : co-factor role in repression of gal transcription by GAL repressor , Genes Cells 1 , 179-188 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_48">
References Aki , T. , Choy , H.E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional regulator : cofactor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></label>
<input id="rp_similarity_48", type="checkbox">
<div class="RP_similar">
<li>Aki , T. , Choy , H.E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional regulator : co-factor role in repression of gal transcription by GAL repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630975" target="_blank">17630975</a>]<br></li>
<li>Aki , T. , Choy , H.E. and Adhya , S. ( 1996 ) Histone-like protein HU as a speci ® c transcriptional regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>Aki , T. , Choy , H.E. , and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Aki , T. , Choy , H.E. and Adhya , S. ( 1996 ) Histone-like protein HU as a speciﬁc transcriptional regulator : co-factor role in repression of gal transcription by Gal repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li></div>
<li>E I Concurrent repression of the gal promoters also requires the bacterial histone-like protein HU which acts as a co-factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Three trans-acting proteins , GalR ( the gal repressor ) , cAMP receptor protein ( CRP ) - cAMP complex , and the histone-like protein HU , control transcription from these two promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21445255" target="_blank">21445255</a>]<br></li>
<li>Our results show that HU participates in a control pathway that down-modulates OmpF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Loss of HU protein causes a global 3-fold promoter derepression ( Figure 2B ) , reﬂected in a 3-fold decrease in estimated equilibrium constants for speciﬁc and non-speciﬁc loop formation ( K and K , max NSL Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>For example , unbalanced expression of HU was shown to induce mucoidy in E. coli ( Painbeni etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>As for the small DNA-binding proteins FIS and HU , bacteria lacking these proteins show defects in growth , cell-division , recombination , DNA-repair etc. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_59">
, Rouviere-Yaniv , J. , and Moreau , P.L. ( 1991 ) Inhibition of cell-division in hupA hupB mutant bacteria lacking HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809757" target="_blank">8809757</a>]<br></label>
<input id="rp_similarity_59", type="checkbox">
<div class="RP_similar">
<li>Inhibition of cell-division in hupA hupB mutant bacteria lacking HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li></div>
<li>The nucleoid-associated protein HU , the histone-like protein H-NS , and the small regulatory RNAs DsrA and OxyS in conjugation with the RNA chaperone Hfq control the translation of rpoS by modulating the secondary structure of the rpoS mRNA ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>Positive regulators of rpoS translation include the cold-shock proteins CspC and CspE ( Phadtare and Inouye , 2001 ) , RNA binding protein Hfq ( Soper et al. , 2010 ) , nucleoid protein HU ( Balandina et al. , 2001 ) , and some small regulatory RNAs ( DsrA , RprA and ArcZ ) ( Soper et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>This view derived primarily from the fact that Lrp tends to activate some operons involved in biosynthesis and repress others involved in catabolism .1 Another view advanced by D'Ari et al. 13 is that Lrp is primarily a DNA-organizing protein , much like HNS , IHF , and HU , which also affect gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>They also participate in diverse roles such as regulation-of-DNA-replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Panels ( A -- D ) show expression data reﬂecting DNA looping in ( A ) WT E. coli , ( B ) cells lacking HU , ( C ) cells lacking IHF and ( D ) cells lacking H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Our results suggest that the HU protein , the Min proteins , and the MukB protein play interconnected roles in different steps of the E. coli life cycle , among them DNA-replication-initiation , nucleoid topology , nucleoid partitioning , and septation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>Our results suggest that global nucleoid structure , determined in part by the histone-like protein HU , plays a role in DNA-replication-initiation , in proper chromosome partitioning directed by the MukFEB proteins , and in correct septum placement directed by the MinCDE proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>These results suggest that the HU protein , possibly by controlling in part the global structure of the nucleoid , is involved in DNA-replication-initiation , that it participates in chromosome partitioning ( as discussed in reference 51 ) via the MukB system , and that it affects septation via the MinCDE proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>Role of his-tone-like proteins OsmZ and HU in homologous recombination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>HU and IHF are small , basic , histone-like proteins which are abundant in the cell and which are involved in a number of cellular-processes , such as site-speciﬁc recombination , replication , and transcription ( 8 , 11 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>For operons involved in use of carbon sources in Escherichia coli , the most frequent global regulator is cAMP/CAP , responsible for catabolite repression , but histone-like proteins , IHF , H-NS , Fis and HU are often involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Histone-like proteins , such as IHF , H-NS , HU , and Fis , play a major role in modifying DNA conformation , changing the access of certain regulators and/or RNA polymerase to promoters ( 2 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>However , nucleoid-associated proteins ( NAPs ) , particularly histone-like proteins , such as histone-like nucleoid structuring ( H-NS ) protein , heat unstable nucleoid protein ( HU ) , factor for inversion stimulation ( Fis ) and integration host factor ( IHF ) , are believed to act like histones and play a signiﬁcant role in the organization of the nucleoid ( 14 -- 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>These proteins include ( in decreasing order of abundance during exponential growth ) : factor for inversion stimulation ( Fis ) , heat-unstable nucleoid protein ( HU ) , integration host factor ( IHF ) , histone-like , nucleoid-structuring protein ( H-NS ) , suppressor of td mutant phenotype A ( StpA ) and DNA-binding protein from starved cells ( Dps ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Protein HU is an alanineand lysine-rich protein which lacks cysteine , tyrosine and tryptophan [ l ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=350619" target="_blank">350619</a>]<br></li>
<li>Overexpression of either subunit of the HU protein ( HUa or HUb ) in E. coli lon1 cells induces expression of the cps genes ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>DISCUSSION The nucleoid-binding proteins of E. coli , assimilated to his-tone-like proteins , include HU , H-NS , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>Nash , H. A. ( 1996 ) The E. coli HU and IHF proteins : accessory factors for complex protein-DNA assemblies , in Regulation of Gene Expression in E. coli ( Lin , E. C. C. , and Lynch , A. S. , Eds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_79">
The E. coli HU and IHF proteins : Accessory factors for complex protein-DNA assemblies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></label>
<input id="rp_similarity_79", type="checkbox">
<div class="RP_similar">
<li>The HU and IHF proteins , accessory factors for complex protein -- DNA assemblies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li></div>
<li>Ogawa , T. , Wada , M. , Kano , Y. , Imamoto , F. and Okasaki , T. ( 1989 ) DNA-replication in E. coli mutants that lack protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Shindo , H. , Furubayashi , A. , Shimizu , M. , Miyake , M. , and Imamoto , F. ( 1992 ) Preferential binding of E. coli histonelike protein HU alpha to negatively supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>enzyme , 0.5 , Ag ; gyrA protein , 0.1 jug ; gyrB protein , 0.5 , ug ; E. coli to-poisomerase I , 75 ng ; and protein HU , 0.2 , ug ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6310593" target="_blank">6310593</a>]<br></li>
<li>The Escherichia coli histone-like protein HU affects DNA initiation , chromosome partitioning via MukB , and cell-division via MinCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004192" target="_blank">11004192</a>]<br></li>
<li>HU The histone-like protein from E. coli strain U93 ( HU ) ( Drlica and Rouviere-Yaniv , 1987 ) is closely related to IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Balandina , A. , Claret , L. , Hengge-Aronis , R. , and Rouviere-Yaniv , J. ( 2001 ) The Escherichia coli histone-like protein HU regulates rpoS translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_87">
The Escherichia coli his-tone-like protein HU regulates rpoS translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></label>
<input id="rp_similarity_87", type="checkbox">
<div class="RP_similar">
<li>The Escherichia coli histone-like protein HU regulates rpoS translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142416" target="_blank">12142416</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li></div>
<li>T. , Spassky , A. , But , H. and Higgins , C.F. : Histone-like protein Hl ( H-NS ) , DNA supercoiling , and gene Kano , Y. and Imamoto , F. : Requirement of integration host factor ( IHF ) for growth of Escherichia coli deficient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_90">
Requirement of integration host factor ( IHF ) for growth of Escherichia coli deWcient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></label>
<input id="rp_similarity_90", type="checkbox">
<div class="RP_similar">
<li>Requirement of integration host factor ( IHF ) for growth of Escherichia coli deﬁcient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li></div>
<li>[ 54 ] Huisman O. , Faelen M. , Girard D. , Jaffe A. , Toussaint A. , Rouviere-Yaniv J. , Multiple defects in Escherichia coli mutants lacking HU protein , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Cyanobacterial DNA-binding protein related to Escherichia coli HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_94">
Regulation of HU protein-synthesis in Escherichia coli , p. 247 -- 257 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></label>
<input id="rp_similarity_94", type="checkbox">
<div class="RP_similar">
<li>Regulation of HU protein-synthesis in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li></div>
<li>The HU protein has been shown to be involved in DNA-replication in vitro ; we show here that its inactivation in the hupA hupB double mutant disturbs the synchrony of replication initiation in vivo , as evaluated by ﬂow cytometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>HU is a very abundant DNA-binding protein that has been speculated to be involved in the formation of the bacterial nucleoid ( Driica and Rouviere-Yaniv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Therefore , HU was long considered as the primary protein factor involved in the organization of bacterial chromatin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>HU accumulates to 25 000 dimers per cell during exponential growth , and it has been proposed that the protein functions in many processes including DNA-replication , transcriptional repression and recombination ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>The ﬁnding that one of the genes coding for HU , also a highly expressed protein , is , like the stable RNA operons , under positive control of FIS , was interesting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>As in the case of H-NS , the HU protein affects the global pattern of gene transcription by modulating the conformation of genome DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>On the other hand , a small , histone-like protein , HU , which is known to bend and compact chromosomal DNA , drastically reduces tdc transcription ; mutational analyses of the genes encoding DNA gyrase and topoisomerase I also revealed that relaxation of supercoiled DNA significantly enhanced tdc transcription in vivo ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Kar , S. , Choi , E.J. , Guo , F. , Dimitriadis , E.K. , Kotova , S.L. and Adhya , S. ( 2006 ) Right-handed DNA supercoiling by an octmeric form of histone-like protein HU : modulation of cellular-transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>The histone-like protein HU binds speciﬁcally to DNA-recombination and repair intermediates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Recently , a protein that permitted galR-mediated inhibition of transcription from P2 in vitro was puriﬁed from cell extracts and identiﬁed as HU , a histone-like DNA-binding protein that imparts a sharp bend to sequences to which it binds ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>On this view FIS , and possibly also CRP , act to ﬁne tune a fundamental mode of transcriptional regulation , not only by their direct effects on DNA topology but also from the indirect effects of their transcriptional-control of other nucleoid-associated proteins , such as HU and H-NS [ 3 , 4 ] , which also act as determinants of chromosomal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></li>
<li>Like the bacterial HU family of proteins , HMG-1 proteins are small , basic , minor-groove-binding , DNA-bending proteins that play an architectural role in chromatin structure ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>DNA bending by HU A speci cents c regulatory role for HU , which involves ` stable ' DNA binding and bending at a speci cents c position , has been reported for a number of systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>Since the H-NS , HU , IHF and Fis proteins participate in a wide variety of cellular-processes , careful control of their intracellular amount at a given time during the bacterial growth cycle is obviously of great importance for cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Effects of mutations in global regulators IHF and HU on persister formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18519731" target="_blank">18519731</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_111">
Histones , HMG , HU , IHF : Meme combat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></label>
<input id="rp_similarity_111", type="checkbox">
<div class="RP_similar">
<li>Histones , HMG , HU , IHF : meme combat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li></div>
<li>Steen EJ , Kang Y , Bokinsky G , Hu Z , Schirmer A , McClure A , del Cardayre SB , Keasling JD : Microbial production of fatty-acid-derived fuels and chemicals from plant biomass .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23618128" target="_blank">23618128</a>]<br></li>
<li>Searle , P. F. , Chen , M. J. , Hu , L. , Race , P. R. , Lovering , A. L. , Grove , J. I. , Guise , C. , Jaberipour , M. , James , N. D. , Mautner , V. , Young , L. S. , Kerr , D. J. , Mountain , A. , White , S. A. , and Hyde , E. I. ( 2004 ) Nitroreductase : a prodrug-activating enzyme for cancer gene therapy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19580253" target="_blank">19580253</a>]<br></li>
<li>We are grateful to D. Geller , P. Hu , W. Jelinik , and Wendi Yajnikfor constant encouragementand criticism , G. DeVries for valuable help in computer analysis , and D. Saluja for suggestions on protein purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7685758" target="_blank">7685758</a>]<br></li>
<li>Res Microbiol 161 : 363 -- 371 Kuhn ML , Zemaitaitis B , Hu LI , Sahu A , Sorensen D , Minasov G , Lima BP , Scholle M , Mrksich M , Anderson WF , Gibson BW , Schilling B , Wolfe AJ ( 2014 ) Structural , kinetic and proteomic characterization of acetyl phosphate-dependent bacterial protein-acetylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25518064" target="_blank">25518064</a>]<br></li>
<li>Physiological implications of lysine acetylation in E. coli are subject of intense study and include altered activity of an array of different cellular machineries such as the acetyl-CoA synthetase , RNA polymerase , the chemotaxis response regulator ( CheY ) , the regulator of capsule synthesis ( RcsB ) , ribonuclease R ( RNase R ) and N-hydroxyarylamine O-acetyltransferase ( Li et al , 2010 ; Thao et al , 2010 ; Castaño - Cerezo et al , 2011 ; Lima et al , 2012 ; Hu et al , 2013 ; Zhang et al , 2013 ; Bernal et al , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25518064" target="_blank">25518064</a>]<br></li>
<li>The proteins compared are the P. aeruginosa and Neisseria gonorrhoeae PilD enzymes ( Nunn et al. , 1990 ; Lauer et al. , 1993 ; Dupuy and Pugsley , 1994 ) , the E. coli BfpP enzyme encoded by a gene in the plasmid-borne bundle-forming pilus operon ( Zhang et al. , 1994 ) , PulO ( Pugsley and Reyss , 1990 ) , the V. cholerae TcpJ enzyme involved in Tcp pilus-formation ( Kaufman et al. , 1991 ) , XpsO ( Hu et al. , 1995 ) , GspO ( Andrews et al. , 1989 ; Francetic and Pugsley , 1996 ) and PppA ( this study ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515702" target="_blank">9515702</a>]<br></li>
<li>PCC 7120 was also reported not to be involved in DNA segregation ( Hu et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17581120" target="_blank">17581120</a>]<br></li>
<li>cspE has been implicated in various cellular functions including chromosome partitioning ( Yamanaka et al. , 1994 ) , chromosome condensation ( Hu et al. , 1996 ; Johnston et al. , 2006 ) , growth in cold ( Mangoli et al. , 1997 ) , modulation of RNase E activity ( Feng et al. , 2001 ) , nucleic acid melting activity ( Phadtare et al. , 2002 ) , and regulation of expression for some of the cold-induced genes ( Phadtare et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177308" target="_blank">18177308</a>]<br></li>
<li>In this paper , we have analysed a number of factors responsible for the low-temperature regulation of cspE , a gene involved in various important cellular functions ( Yamanaka et al. , 1994 ; Hu et al. , 1996 ; Mangoli et al. , 1997 ; Feng et al. , 2001 ; Phadtare et al. , 2002 ; Johnston et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177308" target="_blank">18177308</a>]<br></li>
<li>Nguyen KT , Hu X , Colton C , Chakrabarti R , Zhu MX , Pei D. Characterization of a human peptide deformylase : implications for antibacterial drug design .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19922819" target="_blank">19922819</a>]<br></li>
<li>Hu J , Wang B , Fang X , MEans WJ , McCormick RJ , Gomelsky M , Zhu MJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26303830" target="_blank">26303830</a>]<br></li>
<li>Hu , M. X. , Zhang , X. , Li , E. L. , and Feng , Y. J. ( 2010 ) Recent advancements in toxin and antitoxin systems involved in bacterial programmed-cell-death .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24923448" target="_blank">24923448</a>]<br></li>
<li>Hu Z , Zhang A , Storz G , Gottesman S , Leppla SH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23318956" target="_blank">23318956</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25030700" target="_blank">25030700</a>]<br></li>
<li>Li Y , Hu YF , Fu WY , Xia B , Jin CW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21463637" target="_blank">21463637</a>]<br></li>
<li>Ye K , Serganov A , Hu W , Garber M , Patel DJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23475967" target="_blank">23475967</a>]<br></li>
<li>( g ) Liu , Y. M. ; Hu , X. H. J. Phys .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16910683" target="_blank">16910683</a>]<br></li>
<li>, Dougherty , B. A. , Bott , K. F. , Hu , P.-C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218434" target="_blank">9218434</a>]<br></li>
<li>Hu , L. A. & King , S. C. ( 1998a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931886" target="_blank">10931886</a>]<br></li>
<li>Wagner , J. R. , Hu , C.-C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8789150" target="_blank">8789150</a>]<br></li>
<li>Johnson , L. N. , Hu , S.-H .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9334229" target="_blank">9334229</a>]<br></li>
<li>Multiple Roles for the Twin Arginine Leader Sequence of Dimethyl Sulfoxide Reductase of Escherichia coli * Received for publication , November 17 , 1999 , and in revised form , March 3 , 2000 Published , JBC Papers in Press , May 8 , 2000 , DOI 10.1074 / jbc.M909289199 Damaraju Sambasivarao , Raymond J. Turner ‡ , Joanne L. Simala-Grant § , Gillian Shaw , Jing Hu ¶ , and Joel H. Weineri From the Medical Research Council Group in Molecular Biology of Membrane Proteins , Department of Biochemistry , University of Alberta , Edmonton , Alberta T6G 2H7 , Canada Dimethyl sulfoxide ( Me2SO ) reductase of Escherichia coli is a terminal electron-transport-chain enzyme that is expressed under anaerobic growth conditions and is required for anaerobic growth with Me2SO as the terminal electron acceptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10801884" target="_blank">10801884</a>]<br></li>
<li>6824 Biochemistry 2002 , 41 , 6824-6833 Intrinsic Lipid Preferences and Kinetic Mechanism of Escherichia coli MurG † Lan Chen , Hongbin Men , Sha Ha , Xiang-Yang Ye , Livia Brunner , Yanan Hu , and Suzanne W Department of Chemistry , Princeton UniVersity , Princeton , New Jersey 08544 alker * ReceiVed February 12 , 2002 ; ReVised Manuscript ReceiVed March 26 , 2002 ABSTRACT : MurG , the last enzyme involved in the intracellular phase of peptidoglycan-synthesis , is a membrane-associated glycosyltransferase that couples N-acetyl glucosamine to the C4 hydroxyl of a lipid-linked N-acetyl muramic acid derivative ( lipid I ) to form the â-linked disaccharide ( lipid II ) that is the minimal subunit of peptidoglycan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12022887" target="_blank">12022887</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_135">
D. Kiefer , X. Hu , R. E. Dalbey , and A. Kuhn , Negatively charged amino acid residues play an active role in orienting the Sec-independent Pf3 coat protein in the Echerichia coli inner membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11051763" target="_blank">11051763</a>]<br></label>
<input id="rp_similarity_135", type="checkbox">
<div class="RP_similar">
<li>Kiefer , D. , Hu , X. , Dalbey , R. , and Kuhn , A. ( 1997 ) Negatively charged amino acid residues play an active role in orienting the Sec-independent Pf3 coat protein in the Escherichia coli inner membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22179606" target="_blank">22179606</a>]<br></li></div>
<li>Hu , S. , Ohtsubo , E. and Davidson , N. : Electron microscope studies of sequence relations among plasmids of Escheri-chia coli : Structure of F13 and related F-primes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6265322" target="_blank">6265322</a>]<br></li>
<li>Membrane-associated MinD then recruits its passenger protein MinC ( Hu and Lutkenhaus , 2000 ) , which acts as a potent inhibitor of Z-ring formation - a key prerequisite for assembly of the cell-division machinery ( Bi and Lutkenhaus , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24930948" target="_blank">24930948</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_139">
Hu , Z. , and Lutkenhaus , J. ( 1999 ) Topological regulation in Escherichia coli involves rapid pole-to-pole oscillation of the division inhibitor MinC under the control of MinD and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16262780" target="_blank">16262780</a>]<br></label>
<input id="rp_similarity_139", type="checkbox">
<div class="RP_similar">
<li>[ PubMed : 11800919 ] Hu Z , Lutkenhaus J. Topological regulation-of-cell-division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15165230" target="_blank">15165230</a>]<br></li>
<li>[ PubMed : 20025670 ] Hu Z , Lutkenhaus J. Topological regulation-of-cell-division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24930948" target="_blank">24930948</a>]<br></li>
<li>[ PubMed : 11983867 ] Hu Z , Lutkenhaus J. Topological regulation-of-cell-division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>[ 13 ] Hu Z. , Lutkenhaus J. , Topological regulation-of-cell-division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11254986" target="_blank">11254986</a>]<br></li>
<li>Hu , Z. , and Lutkenhaus , J. ( 1999 ) Topological regulation-of-cell-division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23012351" target="_blank">23012351</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931285" target="_blank">10931285</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_145">
Hu , Z. , and Lutkenhaus , J. ( 2001 ) Topological regulation-of-cell-division in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23012351" target="_blank">23012351</a>]<br></label>
<input id="rp_similarity_145", type="checkbox">
<div class="RP_similar">
<li>[ PubMed : 10869074 ] Hu Z , Lutkenhaus J. Topological regulation-of-cell-division in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19415799" target="_blank">19415799</a>]<br></li>
<li>[ PubMed : 10540287 ] Hu Z , Lutkenhaus J. Topological regulation-of-cell-division in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15165230" target="_blank">15165230</a>]<br></li></div>
<li>Infect Dis Clin North Am 23:609 -- 23 Hu Y , Coates AR ( 2005 ) Transposon mutagenesis identiﬁes genes which control antimicrobial drug tolerance in stationary-phase Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23090599" target="_blank">23090599</a>]<br></li>
<li>Hu P , Janga SC , Babu M , Diaz-Mejia JJ , Butland G , Yang W , Pogoutse O , Guo X , Phanse S , Wong P , et al : Global functional atlas of Escherichia coli encompassing previously uncharacterized proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21810204" target="_blank">21810204</a>]<br></li>
<li>A. Diep , K. Hu , A. Iverson , H. Yang , J.H. Miller , Use of the rpoB gene to determine the speciﬁcity of base substitution mutations on the Escherichia coli chromosome , DNA Repair ( Amst .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17727900" target="_blank">17727900</a>]<br></li>
<li>Saedler , H. , H. J. Reif , S. Hu and N. Davidson , 1974 IS2 , a genetic element for turn-off and turn-on of gene activity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11014799" target="_blank">11014799</a>]<br></li>
<li>Hu , M. and Deonier , R.C. : Mapping of ISI elements flanking the argF gene region on the Escherichia coli K-12 chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1849492" target="_blank">1849492</a>]<br></li>
<li>Insights into the mechanisms of the selectivity filter of Escherichia coli aquaporin Z Guodong Hu , Department of Physics , University of Texas at San Antonio , San Antonio , TX 78249 , USA Shandong Provincial Key Laboratory of Functional Macromolecular Biophysics , Dezhou 253023 , China Department of Physics , Dezhou University , Dezhou 253023 , China Liao Y. Chen , and Department of Physics , University of Texas at San Antonio , San Antonio , TX 78249 , USA Jihua Wang Shandong Provincial Key Laboratory of Functional Macromolecular Biophysics , Dezhou 253023 , China Department of Physics , Dezhou University , Dezhou 253023 , China Abstract Aquaporin Z ( AQPZ ) is a tetrameric protein that forms water channels in the cell membrane of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22392432" target="_blank">22392432</a>]<br></li>
<li>Studies of DNA-synthesis in Escherichia coli have identified numerous functions required for chromosomal replication , including DNA polymerase III , which contains at least eight subunits , DNA gyrase , DNA primase , RNA polymer-ase , DNA helicases , and Hu , the major histonelike protein in the cell ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537812" target="_blank">2537812</a>]<br></li>
<li>Fur J Biochem 77 , 69-75 9 PetersenHU , Wikman FP , SiboskaGE , Worm-LeonhardH , Clark BFC ( 1983 ) Interaction between methionine accepting tRNAs and proteins during initiation of procaryotic translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1720674" target="_blank">1720674</a>]<br></li>
<li>The control set of proteins contained HRTHBD , MYSLG , LZBPT4 , AZBR , XNCHDC , R5EC7 , LPHUMC , CCBN , IHKREV , CSBQ , ISCHT , LZBA , TISY , CVJB , JGECA , RCBPL , VCTNS , NTSR3C , IPPG , VBHU , KLBOI , CRBO2 , CYBOG2 , SYBSYF , PSPGA , UHHUQ , QXBOGL , ACHUA1 , NFRT1 , BHTLD , 04RBP2 , MCSW , HSSFBM , APBPML , RKEGL , HLMSAB , CSMSAT , IVMSA1 , AXPG , R3EC1 , CGB07S , ISECX1 , QXHU3M , WMEC15 , DSBYC , BMTD , YVBPMS , FJEC , FOFVlR , XYOFS , QRHULD , TXEC , RGBYA2 , and FSFB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042779" target="_blank">3042779</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">HU binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Then using a genetic • is ribosomal RNA promoter region sequence hu selection , we have inserted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=344139" target="_blank">344139</a>]<br></li>
<li>Hu et al. have proposed that loop H EQN , conserved 124 127 in all the MurG sequences available , `` forms a key part of the acceptor-binding site '' [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17692452" target="_blank">17692452</a>]<br></li>
<li>Distances and twist angles between operators and putative HU binding sites in the antiparallel loops formed in the absence of HU AL1 loop AL2 loop OI to +6.5 bp/deg OE to +6.5 bp/deg OE to − 13.5 bp/deg OI to − 13.5 OE to − 14.5 bp/deg bp/deg OI to − 14.5 bp/deg 47 bp/108 .7 ° 66 bp / − 46.8 ° 47 bp/138 .5 ° 66 bp / − 84.2 ° 46 bp/101 .5 ° 67 bp / − 47.2 ° The experimentally observed position of HU binding , +6.5 , is favorable only in the AL1 loop , as shown by the energy calculations ( Geanacopoulos et al. 2001 ) , and it provides implausible HU binding site in the loop AL2 ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>It is possible that HU might recognize similar sequences and the corresponding structural motifs in both RNA species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>, 1981 ) , while HU shows no indication of co-operativity in the linking-number binding assay ( Fig. 2 ) or in a DIVA competition assay which shows that duplex-DXA fragments of all sizes down to 40 bp bind HU equally ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>Another possible explanation for the 8.5 bp repeat is that the DNA may lie on the surface of an HU complex which itself is screwed left-handed in space so that the bound DNA having a 10.5 bp helical repeat is optimally exposed every 8.5 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3514923" target="_blank">3514923</a>]<br></li>
<li>Such a sequence , which contains a dyad symmetry followed by a stretch of AT base pairs , very likely generates a cruciform structure that attracts HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>The family of repetitive extragenic palindromic sequences : interaction with DNA gyrase and histonelike protein HU , p. 211-225 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400248" target="_blank">1400248</a>]<br></li>
<li>The target sites of the HU proteins may be the tertiary structures formed by bending and looping the restricted DNA sequences , but at present there is no evidence for or against this possibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>Considering that the HU binding site at the apex of the gal DNA loop is AT rich and palindromic ( 9 , 17 ) , and that HU preferentially binds to altered DNA structures such as denatured or cruciform DNA ( 18 -- 22 ) , we reasoned that HU may facilitate GalR-mediated looping by tightly binding the single-stranded DNA ( ssDNA ) in the unwound segment induced by negative supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>Thus far , three proteins have been identified with specific functions and binding sites in oriC ( i ) DnaA , the initiator protein , which binds four 9-mers and then interacts with a contiguous set of three 13-mersto open the duplex ; ( ii ) the IciA protein which binds these 13-mers to prevent DnaA protein from opening this region and thus inhibits initiation ; and ( iii ) IHF , which binds a sequence that facilitates the bending of oriC around an aggregate of DnaA proteins ; HU protein serves the same function as IHF buw t ithout an identified binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>Recently , a protein that permitted galR-mediated inhibition of transcription from P2 in vitro was puriﬁed from cell extracts and identiﬁed as HU , a histone-like DNA-binding protein that imparts a sharp bend to sequences to which it binds ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8982002" target="_blank">8982002</a>]<br></li>
<li>HU binds at þ6 .5 , designated hbs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Although HU shows little sequence speciﬁcity , it binds preferentially to supercoiled DNA ( Shindo et al. , 1992 ) and constrains negative supercoiling in vitro ( Rouviere-Yaniv et al. , 1979 ; Broyles and Pettijohn , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>The archetype of prokaryotic histonelike protein , HU , binds to DNA with little or no sequence specificity , although it has a preference for DNA containing cruciforms or single-stranded breaks ( Bianchi 1994 ; Nash 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Moreover , like the eukaryotic HMG1 protein , dimeric HU binds to double-stranded DNA in a non-sequence-speciﬁc manner and speciﬁcally recognizes DNA structures such as cruciform DNA ( Pontiggia et al. , 1993 ; Bonnefoy et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>HU is not known to show DNA sequence speciﬁcity and has a tendency to bind to distorted regions of DNA , e.g. kinks , bends , single-stranded gaps and cruciforms ( Bianchi et al. , 1989 ; Pontiggia et al. , 1993 ; Bianchi , 1994 ; Bonnefoy et al. , 1994 ; Castaing et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Like HU , these proteins bind DNA without apparent sequence speci cents city and can induce bends into DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>These authors argue that HU binding overcomes the sequence dependence of the DNA bending energy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24598256" target="_blank">24598256</a>]<br></li>
<li>IHF is a heterodimer of J & rr .20,000 ( Nash & Robertson , 1981 ) with sequence homology to the histone-like protein HU , but unlike HU , binds to a specific DNA sequence ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>HU binds DNA without sequence speciﬁcity , but has been reported to bind with greater avidity to pre-bent or kinked DNA ( 25 -- 28 ) , consistent with expectations for a DNA-bending protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>HU binds to the underlined sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>HU binds to DNA without any sequence specificity and with low affinity , and it binds to bent or distorted DNA with higher affinity ( 8-12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>IHF shows considerable amino acid sequence similarity with HU but , unlike it , IHF binds to a speciﬁc DNA sequence ( Goosen and van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>An important difference between both proteins , however , is that HU binds the DNA nonwhereas IHF specifically , recognizes a specific sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Introduction [ 5 , 6 ] , although they have distinct DNA-binding activities : HU binds to DNA nonspeciﬁcally whereas IHF binds to a consensus sequence [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>Unlike HU protein , which binds to DNA in a nonspecific manner , IHF binds to a specific DNA consensus sequence , 5 ' - WATCAANNNNTTR-3 ' ( where W = A or T , R = A or G , N = any base ) ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>IHF is a sequence-speciﬁc DNA-binding protein which recognizes the core consensus sequence 59-WATCAANNNNTTR-39 ( 10 ) , whereas HU binds to DNA in a sequence-nonspeciﬁc manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>This site ( TTGAT -- CCCGG ) only matches the consensus sequence in one half-site and it is tempting to speculate that HU may improve FNR-binding at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>The minimal HU binding site is smaller compared to that of IHF ( '' 30 bp at a speciWc site ) and is about 9 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>In one model , the strong binding of GalR or GalR -- HU complex to the operator sequences precedes HU binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15289461" target="_blank">15289461</a>]<br></li>
<li>By assuming that binding of IHF to a non-speci ® c site does not induce the strong bend in DNA observed for speci ® c sites ( Rice et al. , 1996 ) we estimate that IHF covers the same amount of DNA as does HU ( Bonnefoy et al. , 1994 ) , i.e. about 10 bp per dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>In the case of MuA -- HU -- DNA transpososome structure , Lavoie and Chaconas ( 1993 , 1994 ) and Lavoie et al. ( 1996 ) have shown that one HU heterodimer binds around the center of the two MuA binding sites , creating a footprint of ~ 30 bp because of HU-induced DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Sequence analysis revealed that ubiG genes from THU and IS16 share ﬁve nucleotide variances in comparison to the ubiG gene from K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16923914" target="_blank">16923914</a>]<br></li>
<li>175 , No. 23 Ge netic and Molecular Analyses of the C-Terminal Region o the recE Gene from the Rac Prophage of Escherichia coli K-12 Reveal the recT Gene ALVIN J. CLARK , ' * VINEET SHARMA ,1 t STEPHAN BRENOWITZ , ' t CHARLES C. CHU ,2 STEVEN SANDLER , ' LESLIE SATIN , ' ANN TEMPLIN , ' § IRIT BERGER ,3 AND AMIKAM COHEN3 Department of Molecular and Cell Biology , Barker/Koshland ASU , University of California , Berkeley , California 947201 ; Laboratory of Immunology , National Institute ofAllergy and Infectious Diseases , Bethesda , Maryland 208932 ; and Department of Molecular Genetics , Haddassah Medical School , The Hebrew University , P.O. Box 1172 , Jerusalem , Israel3 Received 28 July 1993/Accepted 2 October 1993 f The nucleotide sequence of the C-terminal region of the recE gene of the Rac prophage of Escherichia coli K-12 reveals the presence of a partially overlapping reading frame we call recT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8244937" target="_blank">8244937</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by HU</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>GARY REINESS * , HUEY-LANG YANG * , GEOFFREY ZUBAY * , AND MICHAEL CASHELt * Department of Biological Sciences , Columbia University , New York , N.Y. 10027 ; and t National Institute of Child Health and Hu Institutes of Health , Bethesda , Maryland 20014 Guanosine 5 ' - diphosphate 3 ' - diphosphate ( ppGpp ) and pppGpp accumulate when stringent strains of Eschertchsa colt are deprived of amino acids but do not accumulate in similarly deprived relaxed mutant strains ( 1 ) ; such mutants have recently been denoted as relA ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1103124" target="_blank">1103124</a>]<br></li>
<li>Hu , W.S. , Wang , R.Y.H. , Shih , J.W.K. , and Lo , S.C. ( 1993 ) Identiﬁcation of a putative infC-rpmI-rplT operon ﬂanked by long inverted repeats in Mycoplasma fermentans ( incognitus strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12068810" target="_blank">12068810</a>]<br></li>
<li>The sensitivity of seqA mutants to UV irradiation and to the replication inhibitors hydroxyurea ( HU ) and azidothymidine ( AZT ) was suppressed by alleles of dnaA that reduce the efﬁciency of replication initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16556234" target="_blank">16556234</a>]<br></li>
<li>We draw attention to the fact that the negative feedback control of hns described here is shared by the structural genes for the HU , IHF and Fis proteíns ( Miller , 1981 ; Kohno et al. , 1990 ; Ninnemann et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>We do not yet know whether it is by a direct or indirect mechanism that HU affects micF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>Whether HU plays a significant part in the regulation of ndh expression is not yet known but it certainly has a greater affinity for the ndh promoter than for the ansB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>There is also evidence that the intracellular concentration of H-NS , which like HU autoregulates its own expression , is at its highest point at the onset of the stationary phase ( Spassky et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>These experiments also showed that osmoregulation switched off the transcription of ompF independently of HU , whereas ompC transcription was , also in both types of cells , at its highest level at 300 mM NaCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>A regulatory role for HU was suggested when it was found that repressor-mediated control of glpD transcription was increased fourfold in strains containing HU compared with that of strains deﬁcient in HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>a Intensity of the mtr band produced in the presence of HU , TyrR , and phenylalanine divided by that produced in the presence of HU alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>These results suggest that it is the positive charge and not the hydrophobic side chain of arginine-265 that is required for the in vitro transcription of mtr in the presence of HU and TyrR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>As occurred in the case of HU , the IHF-mediated transcription inhibition of both the mtr and tyrP13 * promoters could not be lifted by the TyrR protein in the presence of tyrosine ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>HU can not be replaced by IHF , Fis or HNS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Functional substitution of IHF by HU and HNS is not without precedent ( Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>They also participate in diverse roles such as regulation of DNA replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Since this gene is part of an operon with hha and ybaJ is the first gene downstream of their common promoter , the microarray data indicate that Hha represses ybaJ as well as its own transcription , a common property among the histone-like regulatory proteins like IHF [ 29 ] , H-NS [ 30 ] , and HU [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>This observation can be rationalized by looking into the HU -- DNA cocrystal structure , in which , besides bending , underwinding and negative writhe are induced ( Swinger et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>A. HU induces bends in DNA that can be exploited for compaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>It is not associated with HU in these systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10982882" target="_blank">10982882</a>]<br></li>
<li>HU is then not stably associated with the DNA and does not induce a static bend .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>It should be noted that the DNA substrate used for the HU -- DNA co-crystals was pre-distorted and that the observed bend does not necessarily reXect the extent of bending induced in native DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>The bending angle induced by HU into undistorted DNA can not be easily estimated because of the lack of sequence speci-ﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>We isolated and characterized a GalR mutant containing an amino acid substitution ( R282L ) that can repress transcription in the absence of HU and supercoiled DNA both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12169637" target="_blank">12169637</a>]<br></li>
<li>The level of P2 transcription was the same both in the absence and presence of GalR ; the addition of HU also did not change the amount of P2 transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>( a ) Repression of in vitro transcription of the gal operon mediated by GalR in the absence and presence of HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14596799" target="_blank">14596799</a>]<br></li>
<li>In the absence of HU or DNA 13 supercoiling the P1 promoter is repressed by a looping-independent mechanism by binding of GalR to an operator ( OE ) located upstream of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15533432" target="_blank">15533432</a>]<br></li>
<li>Regulation of P1 and P 2 by HU and DNA looping were further veriﬁed by measuring the expression of the promoters in the absence of OI in strains in which P1 was fused to the lacZ reporter gene and P 2 to gusA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>HU mutants defective in repression of gal promoters The ability of the different HU mutants to repress transcription of the gal operon by the formation of the DNA loop was determined in vivo in the reporter strain SK22 , which carries a chromosomal fusion of the P2 promoter of gal to the reporter gene gusA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>HU has recently been shown to play a role in the repression of the glpD operon in E. coli , presumably by DNA looping ( Yang and Larson , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Taken together , these measurements confirm the widely held hypothesis that GalR and HU induce DNA looping in the gal operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14500788" target="_blank">14500788</a>]<br></li>
<li>We confirm that deletion of HU inhibits looping and that quantitative modeling suggests residual looping in the induced operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid-associated protein HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>For example , unbalanced expression of HU was shown to induce mucoidy in E. coli ( Painbeni etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Concerning HU , a recent transcriptome analysis of its regulon showed that HU activates nrf operon expression during exponential growth ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>An example which could clearly point to such a role for HU is the proU operon , the expression of which is repressed by H-NS [ 66,67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>However , the cryptic bgf operon , whose expression was reported to be regulated through topological changes of cellular DNA , could not be activated in cells depleted of HU or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>operon was expressed in cells depleted of both HU and H-NS as well as in cells depleted of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>( Panel B ) HU protein was partially purified , and analyzed located by CBB staining .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>The incubation of HU with nc5 RNA was performed under high-salt conditions , and HU-RNA was analyzed by PAGE ( see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>To investigate the possibility that the HU genes may also be under stringent control , we determined the response of the hup genes to amino acid starvation , which induces the stringent response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The increases in supercoiling levels in strains mutant in HU , FIS and gyrase are not seen if the cspE gene in the plasmid is inactivated by an insertion ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12904550" target="_blank">12904550</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_42">
The hupA and the hupB genes , encoding the two polypeptide chains of HU ( HUa and HUb ) , do not form an operon : they are located far apart at 90 minutes and 10 minutes , respectively on the E. coli chromosome ( Kano et al. , 1985 , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></label>
<input id="tu_similarity_42", type="checkbox">
<div class="TU_similar">
<li>The two HU polypeptide chains ( HUa and HUb ) are encoded by the hupA and hupB genes , which do not form an operon but are localized at 90 minutes and 10 minutes , respectively , on the E. coli chromosome ( Kano et al. , 1985 , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li></div>
<li>Overexpression of either subunit of the HU protein ( HUa or HUb ) in E. coli lon1 cells induces expression of the cps genes ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882673" target="_blank">9882673</a>]<br></li>
<li>E. coli can vary the composition of its HU protein because it has two paralogous genes , hupA and hupB , that encode highly related but distinct monomers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15100692" target="_blank">15100692</a>]<br></li>
<li>We previously showed that FIS , which stimulates the expression of stable RNA genes , strongly in ¯ uences the expression of HU : expression from hupA is enhanced by FIS whereas that from hupB is repressed ( Claret & Rouviere-Yaniv , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>The ﬁnding that one of the genes coding for HU , also a highly expressed protein , is , like the stable RNA operons , under positive control of FIS , was interesting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8913296" target="_blank">8913296</a>]<br></li>
<li>The negative regulatory effects of the HU proteins seem to be specific for hup genes : at least , the HU proteins do not significantly affect the expressions of the true promoters of other genes such as CmR , Km R ( Fig. 7 ) , rrns ( Figs 4 , 7 and 8 ) , lac and ara ( our unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>The present results with the hupB promoter-CmR and hupA promoter-KinRfusion genes indicate that the regulatory regions of the hup genes for the negative regulation by HU proteins are located in upstream DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>This successive appearance of HU mRNAs is independent of FIS which regulates the hup genes FIS and CRP regulate the transcription of both hup genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>In fact , during overproduction of HU protein , transcription from the second promoter of the hupB gene was reduced less than that from the most upstream promoter ( Fig. 4 ( c ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>The level of transcription of the HU genes was strongly in ¯ uenced by the absence of the FIS protein : clearly FIS stimulated the expression of the hupA gene while it negatively regulated the hupB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Discussion The present results indicate that one HU protein regulates transcription of its own hup gene and also that one HU protein regulates transcription of the gene of the other HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2187099" target="_blank">2187099</a>]<br></li>
<li>Results Different promoters are used for transcription of the genes encoding HU during a lag-logarithmic-stationary growth cycle We have recently shown that while the hupA gene is transcribed from a single promoter , hupB transcription is controlled by a multiple promoter system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>Autoregulation of transcription of the hupA gene in Escherichia coli : evidence for steric hindrance of the functional promoter domains induced by HU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>The 5 ' - end of the purA mRNA was alsoindependently determined by the T4 DNA polymerase method of Hu and Davidson ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3058695" target="_blank">3058695</a>]<br></li>
<li>[ 6 ] C.A. Conlin , S.L. Ton , H. Hu , M. Segar , The apeE gene of Salmonella enterica serovar Typhimurium is induced by phosphate limitation and regulated by PhoBR , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>The interest raised by this situation from an evolutionary point of view was reinforced recently when it was discovered that argF was located within a 10-kb chromosomal fragment flanked by two ISI elements , suggesting that this gene might have been inherited laterally ( Hu and Deonier , 1981 ; York and Stodolsky , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6329710" target="_blank">6329710</a>]<br></li>
<li>It is also noteworthy that the amino group of the repositioned lysine is located at the same position as one of the sodium-ions in the sodium-dependent bile acid symporter ASBTNM , a structural homologue of NhaA and NapA ( Hu et al. , 2011 ; Fig. 9 B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25422503" target="_blank">25422503</a>]<br></li>
<li>However , we also observed that increased acetylation of RcsB in K154 represses transcription of rprA gene as described in the same study ( Hu et al , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25518064" target="_blank">25518064</a>]<br></li>
<li>The IS16 mutant strain was found to contain an ubiX gene sequence identical to that of its parental strain THU ( an E. coli 15 strain ) , encoding a single-amino-acid substitution ( S98R ) relative to the ubiX sequence from E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16923914" target="_blank">16923914</a>]<br></li>
<li>The ubiG gene from E. coli THU was ampliﬁed by PCR and inserted into pNoTA/T7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16923914" target="_blank">16923914</a>]<br></li>
<li>Since exuT9 is recessive to the wild-type allele , it is not a superrepressed allele ( 31 ) of a regulatory gene , and exuT is likely a structural gene coding for a THU component .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>THU activity is superrepressed in an exuR mutant in which the uxaC-uxaA operon and the uxaB operon are superrepressed ; exuR + / exuR merodiploids are also superrepressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>On the basis of these and other results , it is concluded that THU is coded for by the structural gene exuT , which is negatively controlled by the exuR gene product and which probably belongs to an operon distinct from the uxaA-uxaC operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>( iv ) The second regulatory gene , uxuR , of the hexuronate system , which primarily controls the uxu operon and also the structural gene of enzyme I but not the exu regulon , does not seem to govern THU biosynthesis : in the mutant strain PR1 ( uxuR - ) , which exhibits a constitutive synthesis only of enzymes I , III , and IV ( 15 , 25 , 29 ; Robert-Baudouy et al. , unpublished data ) , THU is not derepressed in the absence of any inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>Two features suggested that THU and the three enzymes determined by the exu regulon ( enzymes II , III ' , and IV ' ) are under the control of a common regulatory gene : inducers are identical and the exuT gene is very close to the presumed operon uxaC-uxaA ( enzymes II and IV ' ) ( 17 , 18 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
<li>Therefore , THU and enzymes II , III ' , and IV ' are under the same negative control exerted by the exuR regulatory gene , and the exuT gene also belongs to the exu regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=783117" target="_blank">783117</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>