<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - HyfR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">HyfR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">HyfR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Interestingly , the A3 subdomain of HyfR contains a cysteine-rich segment ( - C-X , - H-C-X-C-X-P-X-C-X-P - ; Fig. 4 ) which could serve as a binding site for a metal cofactor or iron-sulphur cluster , endowing the HyfR protein with the ability to respond to changes in redox and/or oxygen status as for the FNR , IRP and SoxR transcriptional-regulator proteins ( Hidalgo et al. , 1995 ; Rouault & Klausner , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>[ 3 ] Skibinski , D.A.G. , Golby , P. , Chang , Y.S. , Sargent , F. , Ho ¡ man , R. , Harper , R. , Guest , J.R. , Attwood , M.M. , Berks , B.C. and Andrews , S.C. ( 2002 ) Regulation of the hydrogenase-4 operon of Escherichia coli by the c54-dependent transcriptional-activators FhlA and HyfR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14612240" target="_blank">14612240</a>]<br></li>
<li>R egulation of the Hydrogenase-4 Operon of Escherichia coli by the 54-Dependent Transcriptional Activators FhlA and HyfR David A. G. Skibinski ,1,2 Paul Golby ,1 † Yung-Sheng Chang ,1 Frank Sargent ,3 Ralf Hoffman ,1 R. Harper ,1 John R. Guest ,2 Margaret M. Attwood ,2 Ben C. Berks ,4 and Simon C. Andrews1 * School of Animal and Microbial Sciences , The University of Reading , Reading RG6 6AJ ,1 Department of Molecular Biology & Biotechnology , University of Shefﬁeld , Shefﬁeld S10 2TN ,2 The Centre of Metalloprotein Spectroscopy & Biology , School of Biological Sciences , University of East Anglia , Norwich NR4 7TJ ,3 and Department of Biochemistry , University of Oxford , Oxford ,4 United Kingdom Received 6 May 2002/Accepted 30 August 2002 The hyf locus ( hyfABCDEFGHIJ-hyfR-focB ) of Escherichia coli encodes a putative 10-subunit hydrogenase complex ( hydrogenase-4 [ Hyf ] ) ; a potential 54-dependent transcriptional-activator , HyfR ( related to FhlA ) ; and a putative formate transporter , FocB ( related to FocA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>In the presence of HyfR , a relatively minor band of very low mobility was observed ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The intensity of the retarded band was directly related to the amount of the MalE-HyfR mixture included in the reaction and was not observed when MalE-HyfR was replaced by MalEHyfI ( Fig. 7 ) , suggesting that retardation depends on the presence of HyfR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>This difference in the responses between FhlA165 and HyfR to CRP-cAMP is probably related to the absence of N-terminal domain in the smaller protein , FhlA165 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Both probes identiﬁed the same proteins : AtoC , the product of b1201 , FhlA , HyfR , NRI , PrpR , PspF , RtcR , YfhA YgaA , YgeV , and ZraR ( HydH ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>This possibility will be tested in future experiments by using puriﬁed HyfR , FhlA , and FhlA165 proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The putative consensus sequences for the CRP , FNR , Fur , and FhlA/HyfR proteins are enclosed by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>These results indicate that transcriptional activation by HyfR is strongly inhibited by oxygen , strongly enhanced by low pH , and weakly stimulated by formate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Although strain WS266 , with HyfR as the activator , produced more than 25,000 U of - galactosidase activity when grown anaerobically in LB medium , aerobic cultures produced less than 200 U of - galactosidase activity ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>It should be noted that although FhlA132 and FhlA165 activated hyf-lac expression and apparently bind to the hyf upstream DNA at the same location as does HyfR , the cya mutation had a slightly positive effect on hyf-lacZ-dependent - galactosidase activity produced in the presence of FhlA165 ( data not presented ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The HyfR protein , produced independent of its native control system , activated the hyf operon , and the level of - galactosidase activity produced by strain WS266 ( with WS10 ) was about 8,000 U ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>However , a mutation in fnr increased the level of - galactosidase activity produced by the culture about threefold with HyfR as the activator ( Table 3 ) but had no signiﬁcant effect on expression activated by the mutant FhlA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Provision of the entire hyf operon in multicopy by transformation with the plasmid pGS1020 had no signiﬁcant effect on hyfA-lacZ expression levels under fermentative conditions in the presence of 50 mM formate ( data not shown ) , suggesting that hyfR expression from the hyf operon is too weak to produce levels of HyfR sufﬁcient to induce the hyfA-lacZ fusion , at least under the growth conditions employed here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>To determine whether HyfR , like its homologue , FhlA , regulates transcription of the hyf operon , the effect of hyfR inactivation on hyfA-lacZ expression was studied ( Fig. 5 ; note use of logarithmic scale ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>HyfR , an FhlA homolog encoded by the hyfR gene present at the end of the hyf operon , also activated transcription of hyf-lacZ but did so only when hyfR was expressed from a heterologous promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>HyfR does not activate transcription of hyf-lacZ in aerobically grown cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Although FhlA165 successfully activated transcription of hyf-lacZ in an in vitro transcription-translation experiment , aerobically puriﬁed HyfR protein was unable to activate transcription in vitro ( data not shown ) , a ﬁnding that was in agreement with the putative oxygen sensitivity of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Optimum expression of hyf-lacZ required the presence of cyclic AMP receptor protein-cyclic AMP complex and anaerobic conditions when HyfR was the activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The lack of expression of hyf-lac by HyfR when the cells were grown aerobically demonstrated that when activated by HyfR , expression of hyf is oxygen sensitive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>HyfR regulates hyf expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Overproduction of HyfR increased hyf expression by up to 1,000-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Even upon activation by either the FhlA165 or the HyfR protein , a hyc mutant that is hyf failed to produce detectable dihydrogen under any of the growth conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>HyfR-depen-dent induction was unaffected by the fhlA mutation , indicating that FhlA does not participate in the transcriptional activation of hyf by HyfR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The HyfR-dependent postexponential-phase induction of hyf was accompanied by a drop in the pH of the medium below 6.0 ( data not shown ) , which indicates that HyfR-mediated transcriptional activation requires a low external pH. In support of this suggestion , HyfR induction of hyf was reduced by about 20-fold when the pH of the growth medium was maintained above pH 7.0 ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The induction of hyf by HyfR was only observed when hyfR was placed under the control of the inducible Plac promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Overproduction of HyfR elicited a massive induction of the hyf operon under fermentative conditions at low pH , and this effect was 54 dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>This induction was highly dependent upon the growth conditions and appears to involve direct interaction of HyfR with the hyf operator/promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Nor would it appear that HyfR mediates the FhlA-independent formate induction of hydrogenase-1 ( Brerndsted & Atlung , 1994 ) , since a5 ' is not required for hya expression ( Birkmann et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>It is not clear whether this weak formate effect is signiﬁcant or whether , like FhlA , HyfR responds to intracellular formate concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">HyfR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Furthermore , this similarity is restricted to two subdomains : residues 173-234 / 139-202 of FhlA/HyfR ( A2 subdomain , 52 % identity ) , and residues 277-350/242 -3 17 of FhlA/HyfR ( A4 subdomain , 38 YO identity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>A 250-bp fragment containing the bcp-hyfA intergenic region was PCR ampliﬁed , radiolabeled , and incubated with factor Xa-treated MalE-HyfR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Gel retardation analysis of HyfR interaction with the hyfA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The putative helix-turn-helix motif in the D domain is conserved in HyfR and FhlA , as are the two ATP-binding motifs in the C domain ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>There is only one substitution in the predicted ` reading ` helix of HyfR relative to that of FhlA , suggesting that they recognize similar DNA-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>The presence of the highly conserved regions in the A domains would be consistent with HyfR interacting with formate or a similar metabolite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>HyfR , a protein with 663 amino acids , is missing the amino acids corresponding to the ﬁrst 43 amino acids of the FhlA protein , which contains the region similar to the ABC-ATPases ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Alignment of the FhlA and HyfR amino acid sequences showed that , although the N-terminal signal-sensing A domains of the two proteins are similar , the HyfR protein possesses an amino acid motif absent in FhlA ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Since the central and C-terminal domains of FhlA and HyfR are more than 60 % identical ( 70 % similar ) , it is possible that the effector-independent forms of the FhlA protein would activate hyfA-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>In the unique N-terminal domain of HyfR , a cysteine-rich amino acid sequence can be detected ( 200-CSDLSASHCAC LPRC-214 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>It is not related to any protein in the database except the equivalent region of HyfR ( residues 1-317 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>This difference in the responses between FhlA165 and HyfR to CRP-cAMP is probably related to the absence of N-terminal domain in the smaller protein , FhlA165 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">HyfR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Furthermore , this similarity is restricted to two subdomains : residues 173-234 / 139-202 of FhlA/HyfR ( A2 subdomain , 52 % identity ) , and residues 277-350/242 -3 17 of FhlA/HyfR ( A4 subdomain , 38 YO identity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>The HyfR protein is closely related ( 46YO identity ) to the homotetrameric FhlA protein of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>HyfR and FhlA possess good similarity in their central ( 6970 identity ) and C-terminal ( 69 % identity ) regions , but are less similar in their N-terminal regions ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>HyfR is closely similar to FhlA at the primary-structure level ( 46 % identity [ 1 ] ) and might therefore be expected to possess similar sensory properties .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Alignment of the FhlA and HyfR amino acid sequences showed that , although the N-terminal signal-sensing A domains of the two proteins are similar , the HyfR protein possesses an amino acid motif absent in FhlA ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Except for a stretch of about 60 amino acids ( 139 to 195 in HyfR and 179 to 234 in FhlA ) in which the two proteins are 56 % identical , HyfR and FhlA are dissimilar in their unique N-terminal regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>HyfR is similar to the FhlA protein ( 44 % identical and 54 % similar ) , which is the formate - and molybdenum-dependent activator of the hyc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>HyfR , a protein with 663 amino acids , is missing the amino acids corresponding to the ﬁrst 43 amino acids of the FhlA protein , which contains the region similar to the ABC-ATPases ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by HyfR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>From the above it would seem likely that HyfR regulates transcription of the hyf operon ( and other genes ) in response to formate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>Effect of HyfR on fdhF-lacZ and hycB-lacZ gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>To determine whether HyfR , like its homologue , FhlA , regulates transcription of the hyf operon , the effect of hyfR inactivation on hyfA-lacZ expression was studied ( Fig. 5 ; note use of logarithmic scale ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>HyfR regulates hyf expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Effect of HyfR on expression of the hyf operon ( hyfA-lacZ fusion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The studies described here have revealed that E. coli possesses an operon ( hyf ; bdrogenase four ) that appears to encode a new nine-subunit [ Ni-Fe ] hydrogenase ( Hyf ) complex , a formate-sensing transcription regulator ( HyfR , homologous to FhlA ) , and a formate-transporter ( FocB , homologous to FocA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>R egulation of the Hydrogenase-4 Operon of Escherichia coli by the 54-Dependent Transcriptional Activators FhlA and HyfR David A. G. Skibinski ,1,2 Paul Golby ,1 † Yung-Sheng Chang ,1 Frank Sargent ,3 Ralf Hoffman ,1 R. Harper ,1 John R. Guest ,2 Margaret M. Attwood ,2 Ben C. Berks ,4 and Simon C. Andrews1 * School of Animal and Microbial Sciences , The University of Reading , Reading RG6 6AJ ,1 Department of Molecular Biology & Biotechnology , University of Shefﬁeld , Shefﬁeld S10 2TN ,2 The Centre of Metalloprotein Spectroscopy & Biology , School of Biological Sciences , University of East Anglia , Norwich NR4 7TJ ,3 and Department of Biochemistry , University of Oxford , Oxford ,4 United Kingdom Received 6 May 2002/Accepted 30 August 2002 The hyf locus ( hyfABCDEFGHIJ-hyfR-focB ) of Escherichia coli encodes a putative 10-subunit hydrogenase complex ( hydrogenase-4 [ Hyf ] ) ; a potential 54-dependent transcriptional activator , HyfR ( related to FhlA ) ; and a putative formate-transporter , FocB ( related to FocA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">HyfR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>This indicates that the interaction of HyfR with the PhyfA fragment is nucleotide sequence speciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>It is possible that binding of FNR at this site may not have an impact on binding of the smaller constitutive activator FhlA165 but may minimize the ability of the larger HyfR protein to bind at the target sequence , which is only about 25 bases upstream of the predicted FNR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>A potential - dependent promoter is located upstream of the hyf operon and approximately 110 bp downstream of a putative FhlA ( and/or HyfR ) - binding site ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The putative consensus sequences for the CRP , FNR , Fur , and FhlA/HyfR proteins are enclosed by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by HyfR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>In a separate experiment , the hyfR gene was cloned and expressed from a heterologous promote to determine whether HyfR , once produced within the cell , would activate the expression of hyfA-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>It should be noted that although FhlA132 and FhlA165 activated hyf-lac expression and apparently bind to the hyf upstream DNA at the same location as does HyfR , the cya mutation had a slightly positive effect on hyf-lacZ-dependent - galactosidase activity produced in the presence of FhlA165 ( data not presented ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The induction of hyf by HyfR was only observed when hyfR was placed under the control of the inducible Plac promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>The HyfR protein , produced independent of its native control system , activated the hyf operon , and the level of - galactosidase activity produced by strain WS266 ( with WS10 ) was about 8,000 U ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Provision of the entire hyf operon in multicopy by transformation with the plasmid pGS1020 had no signiﬁcant effect on hyfA-lacZ expression levels under fermentative conditions in the presence of 50 mM formate ( data not shown ) , suggesting that hyfR expression from the hyf operon is too weak to produce levels of HyfR sufﬁcient to induce the hyfA-lacZ fusion , at least under the growth conditions employed here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Although the hyf operon is apparently silent in wild-type E. coli , two mutant FhlA proteins ( FhlA132 and FhlA165 ) and constitutively expressed HyfR protein were able to activate transcription of hyf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>HyfR does not activate transcription of hyf-lacZ in aerobically grown cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>The lack of expression of hyf-lac by HyfR when the cells were grown aerobically demonstrated that when activated by HyfR , expression of hyf is oxygen sensitive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>No HyfR-dependent induction of hyf was seen when HyfR production relied upon expression from the hyfR gene located within the single - or multicopy hyf operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>A potential - dependent promoter is located upstream of the hyf operon and approximately 110 bp downstream of a putative FhlA ( and/or HyfR ) - binding site ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>A gene coding for a putative transcriptional activator , HyfR , is located at the end of the hyf operon ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>From the above it would seem likely that HyfR regulates transcription of the hyf operon ( and other genes ) in response to formate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9387241" target="_blank">9387241</a>]<br></li>
<li>HyfR , an FhlA homolog encoded by the hyfR gene present at the end of the hyf operon , also activated transcription of hyf-lacZ but did so only when hyfR was expressed from a heterologous promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>To determine whether HyfR , like its homologue , FhlA , regulates transcription of the hyf operon , the effect of hyfR inactivation on hyfA-lacZ expression was studied ( Fig. 5 ; note use of logarithmic scale ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
<li>Effect of HyfR on expression of the hyf operon ( hyfA-lacZ fusion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12426353" target="_blank">12426353</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>