<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - IdnR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">IdnR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">IdnR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Among these predicted targets under the direct control of RgrR , as many as 11 genes are included , including PgrR , LeuO , YagP , MarR , YpdA , XapR , YhgF , GadX and IdnR ( indicated in star in Table 1 ) , indicating that a number of genes must be regulated indirectly by PgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23301696" target="_blank">23301696</a>]<br></li>
<li>Northern blot analysis of the idnK , idnD , idnO , idnT , and IdnR transcripts in E. coli W1485 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>Abreviations : 5KG , 5-keto-D-gluconate ; GntT and GntU , GntI permeases ; GntK and GntV , GntI and GntII gluconate kinases respectively ; IdnT idonate and gluconate permeases ; IdnD , idonate dehydrogenase ; IdnO , 5KG reductase ; IdnR , idnR regulon regulator ; GntR , gntR regulon regulator ; Edd , Entner-Doudoroff dehydratase ; Eda , Entner-Doudoroff aldolase [ GBTB ( Ist√∫riz , et al. , 1986 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
<li>The fact that gntV is also induced in wild type cells grown in gluconate led us to wonder whether this expression is also under the positive control of the 5KG-IdnR complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
<li>Alternatively , it could be that D-gluconate is itself the inducer of the idn genes but that IdnR requires a higher concentration of D-gluconate for induction than does GntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9658018" target="_blank">9658018</a>]<br></li>
<li>This suggests that growth of the pseudorevertants on gluconate may be the result of autoinduction , that is , L-idonate formation from gluconate , leading to induction of the IdnR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>In addition , utilization of idonate , a gluconate predecessor , is subject to positive regulation by IdnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Enzymes for utilization of the gluconate predecessor , idonate , are encoded by adjacent idnDOTR and gntV operons that are under positive regulation by IdnR ( Bausch et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>The position of the putative IdnR binding site is interesting because this location is atypical of negative control , despite the fact that IdnR and GntR belong to the GalR-LacI family of negative regulators ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>It includes all known and predicted members of the GntR , UxuR/ExuR , KdgR , UidR and IdnR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The last column ( * ) represents experimental data about regulation : S , known binding sites ; R , known regulation , binding site not known.Regulons : a. GntR ; b. IdnR ; c. UxuR/ExuR ; d. ExuR ; e. UidR ; f. KdgR ; g. CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">IdnR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The position of the putative IdnR binding site is interesting because this location is atypical of negative control , despite the fact that IdnR and GntR belong to the GalR-LacI family of negative regulators ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>The regulatory protein IdnR encoded by the idnDOTR operon is closely homologous to GntR ( 53 % identity ) and , thus , the appearance of the GntR boxes upstream of idnDOTR can reflect the similarity of signals recognized by these two transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>IdnK is 45 % identical to GntK ( 33 ) ; IdnD is 31 % identical to mammalian sorbitol dehydrogenase ( 14 ) ; IdnO is 56 % identical to Gno from G. oxydans ( 15 ) ; IdnT is 61 % identical to GntT ( 25 ) , a match which was the highest among a total of seven gluco-nate permease orthologs in E. coli ( 23 ) ; and the helix-turn-helix protein IdnR is 46 % identical to GntR , strongly suggesting that IdnR is involved in regulation of idnK and the idnDOTR operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9658018" target="_blank">9658018</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by IdnR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Sugar acid regulons on the E. coli genome Location ( kb ) Gene Function Regulon Operon Reference GntR gntR 65 20 3575.9 1931.7 1930.6 3544.8 3574.9 3574.2 4488.2 4492.5 4491.8 gntR edd eda gntT gntK gntU idnR idnK idnD Repressor : gluconate induction 6-Phosphogluconate dehydratase KDPG aldolase High-afÔ¨Ånity gluconate-transport Gluconate kinase Low-afÔ¨Ånity gluconate-transport Regulator : L-idonate induction Gluconate kinase L-Idonate 5-dehydrogenase edd-eda 20 52 65 65 8 , 9 8 , 9 8 , 9 gntT gntKU IdnR idnDOTR idnK ( gntV ) idnDOTR ( yjgVUTS ) 8 , 9 8 , 9 57 57 39 20 36 46 35 35 35 Gluconate 5-dehydrogenase L-Idonate transport 4490.6 4489.3 1908.2 3676.4 4099.8 1930.6 3244.7 3243.5 3241.7 3240.3 1608.0 idnO idnT kdgR kdgK kdgT eda exuR exuT uxaC uxaA uxaB KdgR kdgR kdgK Repressor : KDG induction KDG kinase KDG transport KDPG aldolase Repressor : hexuronate induction Glucuronate/galacturonate transport Glucuronate/galacturonate isomerase Altronate dehydratase ( galacturonate ) Altronate oxidoreductase ( galacturonate ) Repressor : glucuronate induction Mannonate dehydratase ( glucuronate ) Mannonate oxidoreductase ( glucuronate ) Hexonate/hexuronate-transport ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>In addition , E. coli has two local regulators , UidR and IdnR , which regulate single operons for the utilization of glucuronides and idonates respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>In addition , utilization of idonate , a gluconate predecessor , is subject to positive regulation by IdnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Enzymes for utilization of the gluconate predecessor , idonate , are encoded by adjacent idnDOTR and gntV operons that are under positive regulation by IdnR ( Bausch et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>For many years it was believed that the genes of the IdnR regulon encoded the so-called GntII ( or subsidiary ) system for gluconate-transport and phosphorylation ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>Abreviations : 5KG , 5-keto-D-gluconate ; GntT and GntU , GntI permeases ; GntK and GntV , GntI and GntII gluconate kinases respectively ; IdnT idonate and gluconate permeases ; IdnD , idonate dehydrogenase ; IdnO , 5KG reductase ; IdnR , idnR regulon regulator ; GntR , gntR regulon regulator ; Edd , Entner-Doudoroff dehydratase ; Eda , Entner-Doudoroff aldolase [ GBTB ( Ist√∫riz , et al. , 1986 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">IdnR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>A putative IdnR operator site ( ATGTTA-CGCA-TAACGT ) with homology to the GntR consensus binding sequence ( ATGTTA - [ N4 ] - TAACAT ) ( 21 ) is centered between the two promoters , 78.5 and 83.5 , with respect to the idnK and idnD transcription start sites , respectively , suggesting that this site may function as a regulatory element for both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>The location of the putative IdnR binding site centered between the idnD and idnK transcription start sites suggests that IdnR may coordinately regulate both promoters CB130 CB131 FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14973046" target="_blank">14973046</a>]<br></li>
<li>In addition , there are two binding sites for cAMP-CRP complex and another for GntR or IdnR ( Izu et al. , 1997 , Tsunedomi et al. , 2003b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
<li>There is also the possibility that the GntR regulon cross-talks with the IdnR regulon by way of two highly conserved GntR binding sites located within the divergent promoter region of the L-idonate regulon ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>This region includes promoter-operator sequences for the gntV gene and the idnDOTR operon , each with a binding element for GntR or IdnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by IdnR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Enzymes for utilization of the gluconate predecessor , idonate , are encoded by adjacent idnDOTR and gntV operons that are under positive regulation by IdnR ( Bausch et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The fact that gntV is also induced in wild type cells grown in gluconate led us to wonder whether this expression is also under the positive control of the 5KG-IdnR complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
<li>RESULTS Characteristics of E. coli mutants TK424 ( idnO < > kan , gntK , gnt177 ) and TK425 ( gntK , gnt177 ) Because E. coli mutant TK425 carries the gnt177 and gntK mutations , its growth in gluconate must depend on the inducible expression of the idnR regulon ( idnDOTR and gntV operons ) by the 5KG-IdnR complex ( Ram√≠rez , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22688914" target="_blank">22688914</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>