<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - IHF</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">IHF</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">IHF transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>sigma scores boxes evidence 1 glkP1 - P2 k n glk FruR C [ 36 ] , in this work GCGTTGTTGTTATGCCCCCAGGTATTTACAGTGTGAGAA 236 s38 2 s70 : 3.92 glkP3 - P4 n n s70 3 in this work A , C CCCCCAGGTATTTACAGTGTGAGAAAGAATTATTTTGAC 221 s38 : 2.95 pgiP1 - P2 k n [ 37 ] , in this work A , B , E ¬ pgi ACCATCACATTTTTCTGTGACTGGCGCTACAATCTTCCA 236 s70 : 6.9 ; s38 : 7.44 pfkAP1 - P2 k n ¬ A , B FruR pfkA [ 39 ] , in this work ATCAATTCAGCAGGAAGTGATTGTTATACTATTTGCACA 278 s70 : 8.61 ; s38 : 5.7 pfkAP3 - P4 n n s38 2 in this work A , E TCACTTCGATGTGCAAGAAGACTTCCGGCAACAGATTTC 229 s70 : 1.12 ki CTGCCCTGCGGGGCGGCCATCTTCCTTTATTCGCTTATA 219 ¬ tpiA tpiAP [ 40 ] ¬ ¬ tpiAP1 - P2 in this work n n ¬ A , E CAAAGCCTTTGTGCCGATGAATCTCTATACTGTTTCACA 262 s70 : 8.07 ; s38 : 7.4 gapA gapAP1 - P1a [ 41,43 ] , in k n this work gapAP2 [ 41,43 ] k ¬ A , B , D CGCTTGACGCTGCGTAAGGTTTTTGTAATTTTACAGGCA 236 s70 : 3.76 ; s38 : 0.82 ¬ A CCTTTAAAATTCGGGGCGCCGACCCCATGTGGTCTCAAG 2153 ¬ k TCACATTTTTATCGTAATTGCCCTTTAAAATTCGGGGCG 2174 - - ¬ gapAP3 [ 41,43 ] Crp k AACACCAACTGGCAAAATTTTGTCCTAAACTTGATCTCG 2245 - - ¬ gapAP4 [ 41,43 ] ACATTCCTTTTATTCCACGTTTCGCTTATCCTAGCTGAA 2132 no predictions s703 , s382 A , D epdP1 - P2 k FruR , Crp epd [ 45 ] , in this work pgkP1 k A pgk [ 45 ] AAGCAGCACAAGGTGCATTTCATGGTATAGTTGACTATA 2235 - ¬ n TGTTGCTTTCAGGTAAGACGCAAGCAGCGTCTGCAAAAC 226 no predictions ¬ pgkP2 in this work D , E n GCACACCTGATCAAAACGTTGGTCTGGTGCGATAACGAA 2113 no predictions s703 pgkP3 in this work A , D gpmA gpmAP1 - P2 [ 46 ] , in this ki n work gpmAP3 - P4 in this work n n Fur A ¬ AAGCATTGCTGTTGCTTCGTCGCGGCAATATAATGAGAA 237 s70 :7.56 ; s38 : 3.66 ¬ ¬ CCTTACACTGCGCCACTATTTTCGCTATGGTTATGCGTA 275 s70 :5.80 ; s38 : 7.48 eno enoP1 [ 47 ] k ¬ ¬ AATCGTGTCAGCGTCAACATCAAACTGATCGATTCACAA 2731 ¬ k CGAAGCACTGAAACACGGTGGGCTGAAGAATCGTGTCAG 2760 - - ¬ enoP2 [ 47 ] k TAAACGATTCAGCTTAAACTGCCCGGAAGCGAATCTGTC 2903 - - ¬ enoP3 [ 47 ] n GCGTACCCTGGGTACGCGTTGTTTGTCTGGAGTTTCAGT 227 no predictions s703 , s382 A , D , E enoP4 - P5 in this work n GAGTTCCAGAAACGTCAGGCGAAGTAAGTAAAAAAGTTA 277 no predictions - D enoP6 in this work n n GGGGATGATCAGTTGGTCGAGATCATCGAAGTTCCGAAT 2212 no predictions s703 , s382 D enoP7 - P8 in this work pykF pykFPPki [ 49 ] FruR TGTTTTCCTTTTGGATTAATTTCAGCGTATAATGCGCGC 2231 - -- pykFP1 - P2 in this work n n GGATTCGCTTTCCGGCAGTGCGCCCAGAAAGCAAGTTTC 234 no predictions s703 , s382 A , B , D n GATGTCACCTATCCTTAGAGCGAGGCACCACCACTTTCG 280 no predictions - D pykFP3 in this work pykFP4 - P5 n n in this work CCTCTGCACGCTTTTTCGATGTCACCTATCCTTAGAGCG 297 s70 : 4.79 ; s38 : 9.95 ¬ B , D pykA pykAP1 - P2 n n s38 2 in this work AAGTGACGATCGCTAAAAACGACTGTCACTGTCCTAATC 2105 s70 : 0.90 , D B n n GTCAAAGAAGCGCTGAAGGAATCGCGTTTTGATAAGCAG 2153 s70 : 1.76 s382 A , B , D pykAP3 - P4 in this work Crp , Fnr , GTATGGACATAAGGTGAATACTTTGTTACTTTAGCGTCA 258 no predictions s703 , s382 B , D PdhR s38 2 pdhR pdhRP1 - P2 k n [ 51 -- 52 ] , in this work aceE aceEP1 - P2 k n ¬ AACCTGTCTTATTGAGCTTTCCGGCGAGAGTTCAATGGG 248 s70 : 1.90 [ 50 ] , in this work poxB poxBP1 - P2 k n [ 57,58 ] , in his work B , C ¬ MarA , SoxS TCCCTTCCCCCTCCGTCAGATGAACTAAACTTGTTACCG 227 s70 : 5.31 ; s38 : 9.5 acs acsP1 k [ 60 -- 61 ] ¬ Fis , Crp , Ihf TCTTTAATCAATTGTAAGTGCATGTAAAATACCACTTTA 2224 ¬ k n CCCCTACATTTAACGCTTATGCCACATATTATTAACATC 220 s70 : 4.03 ; this work s38 : 4.97 B , C ¬ acsP2 - P3 [ 60 -- 61 ] , in Fis , Crp , Ihf consensus sigma 70 : TGGTATAATG sigma 38 : CTACACTT sequences Proposed promoters in which putative s and s overlapping recognition sequences have been determined , are presented as bold sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>This would activate RssB resulting in stimulation of os degradation and the subsequent reduction of the development of stationary phase-specific phenotypes observed in ihf mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>This showed that the critica1 factor was not the presence or absence of active fimB in the ihf mutant background , but the copy nurnber of the active fimB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
<li>We found that dps mRNA leveis are controiied by rpoS-encoded a ^ , the transcriptionai activator OxyR and the histone-iike iHF protein , in exponentiaily growing ceiis , dps is induced by treatment with hydrogen peroxide in an OxyR-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>However , the fact that expression of the rpoS , lrp and ihf genes is stimulated by ppGpp ( Aviv et al. , 1994 ; Lange et al. , 1995 ) and that many genes induced by carbon , phosphorus and nitrogen starvation are regulated by the stringent response ( Nystro $ m , 1995 ; Rao et al. , 1998 ) , is consistent with YeiL being involved in the response to nutrient starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Keywords : Ihf ; Integration host factor ; Northern analysis ; Polycistonic transcripts ; Growth rate-dependent regulation of transcription ; Transcriptional terminator ; Stationary-phase et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>A 10-bp insertion in the same position , which approximately restores the helical phasing of the ihf site and increases its distance to the promoter , partially increases transcription to 39 and 74 % of maximum levels in RZ211 and RJ1561 cells , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>A 5-bp insertion between ﬁs P and the ihf site caused about a fourfold reduction in transcription in the presence of Fis and about a twofold reduction in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>A 10-bp insertion between the ihf site and ﬁs P resulted in a partial restoration of transcription in the presence or absence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>BMC Microbiology BioMed Central ss Open Acce Research article Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , Crp , and ArgR Ligi Paul1 ,6 , Pankaj K Mishra2 , Robert M Rowena G Matthews1 ,4,5 Blumenthal * 2,3 and Address : 1Life Sciences Institute , University of Michigan , Ann Arbor , Michigan 48109 , USA , 2Department of Medical Microbiology and Immunology , University of Toledo Health Science Campus , Toledo , Ohio 43614-5806 , USA , 3Program in Bioinformatics and Proteomics / Genomics , University of Toledo Health Science Campus , Toledo , Ohio 43614-5806 , USA , 4Biophysics Research Division , University of Michigan , Ann Arbor , Michigan 48109 , USA , 5Department of Biological Chemistry , University of Michigan , Ann Arbor , Michigan 48109 , USA and 6Vitamin Metabolism and Aging Laboratory , Jean Mayer USDA Human Nutrition Research Center on Aging , Tufts University , Boston , Massachusetts 02111 , USA Email : Ligi Paul - Pankaj K Mishra - Pankaj.Mishra@utoledo.edu; Robert M Blumenthal * - Robert.Blumenthal@utoledo.edu; Rowena G Matthews - RMatthew@umich.edu * Corresponding author Received : 22 September 2006 Accepted : 18 January 2007 Published : 18 January 2007 BMC Microbiology 2007 , 7:2 doi :10.1186 / 1471-2180-7-2 This article is available from : http://www.biomedcentral.com/1471-2180/7/2 © 2007 Paul et al ; licensee BioMed Central Ltd. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The negative regulation of ompF by iHF might therefore be ascribed to inhibition of this OmpR-mediated activation , either by blocking of the binding of this protein to the DNA or by obstruction of its function ( Ramani et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Spot number IHF effect ID ProteinAC pl MW Descriptioonffunction number P39184 P08839 P03003 P06139 P08200 P15046 P24192 P06977 P04790 P17288 P25529 P02418 85.4 59.8 61.13 56.7 46.1 44.0 41.7 36.4 27.0 21.6 25.0 19.8 5.29 4.78 4,55 4.85 5.02 5.84 5.81 6.58 5.57 5.01 5.17 6.20 VIA PT1 negative no no no no negative positive positive positive no negative no 1 Phosphate acetyltransferase 2 Phosphoenolpyruvate-protein phosphotransferase Transcription factor Chaperon Isocitrate dehydrogenase ( NADP ) Acetate kinase Hydrogenase isoenzymes formation protein Glyceraldehyde 3-phosphate dehydrogenase A Triosophosphate isomerase Inorganic pyrophosphatase 7-Alpha hydroxysteroid dehydrogenase 50S ribosomal protein L NUSA GROEL IDH ACKA HYDP G3P1 TPIS IPYR HDHA RL9 3 4 5 6 7 8 9 10 11 12 I I ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>FNR-binding sitesc Other reported regulation Y/N Y/Y Y/N CRP-activated , NarLrepressed 46 47 48 Half-sites 63.5 , and 39.5 - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant and promoter truncations - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and EMSAd - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis , EMSA , and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis , and EMSA ArcA - and DcuSR-regulated ModE-repressed 33.5 , 113.5 moaABCDE Molybdopterin biosynthesis 0.50 1.06 0.34 1.52 0.43 0.97 0.59 hcp-hcr dmsABC Hybrid cluster protein and NADPH oxidoreductase Anaerobic dimethyl sulfoxide reductase 73.5 41.5 Y/Y Y/Y NarL - and NarP-regulated ModE-activated , NarLrepressed 0.72 1.12 0.66 30.26 1.60 47.81 2.54 0.12 0.27 0.10 0.10 0.07 0.70 0.07 49 50 -- 52 focA-pflB Formate transporter and pyruvate-formate lyase 0.24 0.30 0.34 0.35 0.22 0.87 0.31 41.5 Y/Y ArcA-activated 53 -- 55 narK Nitrate/nitrite transport protein 0.20 0.25 0.14 14.30 0.08 12.07 0.12 41.5 Y/Y NarL-activated 56 , 57 41.5 Y/Y narGHJI 0.07 0.07 0.11 33.98 0.14 3.91 0.14 Nitrate reductase A NarL-activated 58 , 59 Y/Y fdnGHI 0.33 0.40 0.20 15.19 0.15 2.16 0.13 Formate dehydrogenase N 42.5 and 97.5 NarL-activated 55 , 60 fliZY Putative cell density - 1.08 0.72 0.23 1.34 0.32 0.62 0.08 responsive regulator of 28 napFDAGHBC Periplasmic nitrate reductase 0.51 0.47 0.39 0.95 0.27 4.04 0.26 No predicted ts 64.5 FlhDC-regulated NarL-repressed , N activated Y/Y Y/N 61 30 arP ¬ glpABC yfiD 0.19 0.16 0.16 0.55 0.21 0.71 0.18 0.07 0.02 0.04 0.66 0.06 1.22 0.08 Glycerol-3-phosphate dehydrogenase Putative formate acetyltransferase 41.5 N/N Y/Y GlpR-repressed , CRP-activated PdhR-repressed 62 40.5 and 93.5 63 , 64 65 N/N hypBCDE 0.20 0.15 0.21 0.59 0.22 0.67 0.19 Hydrogenase maturation Half-sites ansB 41.5 Y/Y Periplasmic L-asparaginase II 0.01 0.02 0.01 0.25 0.01 0.63 0.01 CRP-activated 55 , 66 41.5 Y/Y nirBDC 0.74 0.48 0.38 9.99 0.23 8.80 0.38 Nitrite reductase NarL/NarP-activated , IHF - , Fis - , and H-NS-regulated 10 , 67 0.50 0.29 0.19 0.03 0.80 0.63 0.65 4.73 0.40 0.24 0.10 0.03 1.11 0.29 0.33 0.32 0.53 0.31 0.14 0.05 0.20 0.78 0.22 0.05 0.28 1.38 0.11 0.05 Ferrous iron transport Nickel transporter Putative cytochrome c peroxidase Periplasmic nitrite reductase No reported ts 41.5 61.5 41.5 Y/N Y/N Y/Y Y/Y Fur-repressed NikR-repressed feoAB 68 69 nikABCDE yhjA nrfABCDEFG 61 10 , 70 NarP-activated and NarLregulated TABLE 2 -- continued Ratiosa Position of FNR siteb FNR-binding Other reported regulation Product of operon Operon Evidence of FNR regulation Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>91.5 78 N/N glpTQ 0.46 0.28 0.30 0.64 0.35 0.68 0.34 GlpR-repressed , CRP-activated 82.5 5 Y/Y 79 N/N Y/Y ArcA - and NarL-activated ArcA-activated 80 ndh 81 Y/Y 0.30 0.99 6.60 1.21 6.16 1.18 5.80 ArcA-regulated acnA nrdAB hmpA Aconitate hydrase 1 Y/Y N/N Y/Y CRP-activated , ArcA-repressed Proposed anti-activation in stationary phase 2.58 4.16 75.91 1.43 2.23 20.42 2.51 3.79 3.02 1.36 5.99 1.68 2.02 6.32 2.04 1.18 1.73 29.89 2.44 3.04 149.7 40.5 82 75 Ribonucleoside diphosphate reductase 1 Flavohemoglobin H alf-site 5.5 37 , 85 Half-sites Fur-repressed 86 N/N IHF - and ArcA-activated ModE-activated ArcA-repressed N/N Y/Y N/N 57 , 87 83 88 83 , 84 Y/Y 89 Y/N IHF - , Lrp - , and YeiL-activated 22 N/N NorR-activated , NarL/NarP-repressed 73 N/Y IHF-repressed nitrite ) and strain ( wild type or fnr ) compared with transcript TABLE 3 Operons that display FNR regulation and that have putative FNR-binding sites upstream Ratiosa O2 O Ø 2 2 NO3 NO3 ] NO2 NO2 WT fnr fnr WT fnr WT fnr Location of putative TS FNR FNR site ( s ) b determined site IDsc Operon Function of operon Putative FNR-activated operons caiTABCDE Carnitine metabolism fixABCX Carnitine metabolism allB Allantoinase ybcW Hypothetical protein fes-entF-fepE Ferric enterobactin synthesis and transport Citrate lyase Hypothetical protein Hypothetical protein Putative peptidase T Hypothetical proteins Outer membrane protein ; colicin S4 receptor ydcA Hypothetical protein ydfZ Hypothetical protein ynfEFGHI Me2SO reductase paralogue ynfK Hypothetical protein ydhYVWXU Putative oxidoreductase ydjXYZynjABCDE Hypothetical proteins pykA Pyruvate kinase II tar tap Chemotaxis cheRBYZ yecR Hypothetical protein yedM Hypothetical protein ackA Acetate kinase yfcC Putative S-transferase upp-uraA Uracil salvage yqjI Hypothetical protein aer Aerotaxis sensor receptor uxaCA Glucuronate degradation tdcABCDEFG Threonine degradation garPLRK Glucarate degradation yhbTS Hypothetical proteins yhbUV Hypothetical protein and putative collagenase Hypothetical protein Maltose degradation Low affinity phosphate transport Putative 2-component regulator PTS system Catalase Hypothetical proteins Hypothetical protein Transcriptional activator of cad operon fimBEAICDFGH Type 1 fimbriae yjiD Hypothetical protein tsr Chemotaxis protein yjjIW Hypothetical protein Putative FNR-repressed operons pdhR-aceEF-lpdA Pyruvate dehydrogenase and regulator Glucose dehydrogenase Hypothetical protein Cytochrome o ubiquinol oxidase Rhs element protein / hypotheticals Hypothetical protein Outer membrane protein X Putative cysteine hydrolase 41.5 and 90.5 197.5 185.5 142.5 and 21.5 143.5 Mapped Mapped Predicted Predicted Mapped Y/N Y/N Y/N Y/N Y/N 0.79 1.35 1.16 0.29 1.39 0.94 1.31 0.77 0.29 1.32 0.51 0.40 0.57 0.44 0.30 0.82 0.82 1.20 0.66 0.21 0.39 0.56 0.56 0.28 0.16 0.74 0.81 0.84 0.89 0.29 0.60 0.64 0.41 0.33 0.14 32.5 30.5 Predicted Predicted Y/Y Y/N Y/Y Y/N Y/Y Y/Y citCDEFXG ycbJ yccM pepT ymfED ompW 0.19 0.23 0.27 0.33 0.46 0.09 0.19 0.30 0.42 0.59 0.21 0.06 0.37 0.41 0.43 0.39 0.69 0.27 0.49 0.39 1.64 0.66 1.49 0.03 0.30 0.33 0.44 0.35 0.60 0.02 0.48 0.74 2.80 0.85 0.91 0.53 0.34 0.36 0.36 0.36 0.68 0.02 25.5 83.5 124.5 2 putative sites Predicted Predicted Predicted No predicted ts 0.46 0.01 0.06 0.21 0.42 0.49 0.02 0.05 0.15 0.14 43.5 35.5 64.5 29.5 167.5 Predicted Predicted Predicted Predicted Predicted Y/N N/Y Y/Y N/Y Y/Y 0.40 0.01 0.03 0.11 0.21 0.66 0.52 0.03 0.50 0.23 0.39 0.01 0.02 0.09 0.25 0.84 0.73 0.25 0.77 0.31 0.45 0.01 0.03 0.09 0.16 2.81 0.82 0.96 1.23 1.19 0.54 0.48 0.48 0.51 1.78 0.48 0.39 0.18 0.42 0.39 0.33 0.25 0.78 0.39 0.44 0.57 0.42 0.91 1.63 0.55 0.24 0.29 0.10 0.61 1.15 0.88 0.26 0.80 1.39 0.80 0.31 0.23 0.34 0.71 1.91 79.5 117.5 Predicted Predicted Y/N Y/N Y/N Y/Y Y/N Y/N Y/N Y/Y Y/N Y/N 0.53 0.42 0.28 0.55 0.42 0.28 0.16 0.42 0.40 0.62 0.63 1.01 0.53 0.67 0.91 0.73 0.84 0.93 1.38 0.64 0.46 0.43 0.05 0.17 0.29 0.27 0.23 0.31 0.47 0.27 160.5 113.5 14.5 1 putative site 18.5 61.5 150.5 and 200.5 51.5 Predicted Predicted Predicted No predicted ts Predicted Mapped Predicted Mapped 0.57 0.44 0.60 0.37 0.24 1.12 0.81 1.30 0.80 0.97 0.42 0.28 0.46 0.35 0.34 0.64 0.60 0.59 0.58 0.88 0.47 0.45 0.49 0.34 0.56 1.14 1.04 1.67 0.63 0.71 0.85 0.75 0.99 0.53 0.46 58.5 Predicted Mapped Predicted Predicted Predicted Y/N Y/Y Y/N Y/N Y/N 10.5 and 148.5 154.5 76.5 45.5 yhcC 0.22 0.62 0.37 0.62 0.84 0.85 0.93 0.82 0.21 0.54 0.34 0.71 0.68 0.73 0.57 0.75 0.32 0.49 0.48 0.45 0.38 4.29 0.70 0.67 0.28 1.54 0.75 0.50 Predicted Mapped Predicted Predicted 70.5 127.5 66.5 37.5 Y/N Y/N Y/Y Y/N malPQ pitA yieNM frvABXR katG yjdAZ yjdK cadC 0.34 0.27 0.04 0.20 0.15 0.43 0.94 0.50 0.65 0.25 0.37 0.23 0.22 0.16 0.11 0.39 0.73 1.36 0.78 0.29 0.32 0.21 0.15 0.31 0.14 1.45 0.32 0.76 0.08 1.54 1.82 0.22 1.19 0.09 1.98 Predicted Mapped Predicted Predicted Mapped 11.5 Y/N Y/Y Y/N Y/Y Y/Y 96.5 and 112.5 50.5 and 56.5 14.5 156.5 0.40 0.59 0.05 0.06 0.86 1.06 0.62 0.70 0.70 0.37 0.19 0.04 1.06 0.67 1.22 0.62 0.93 0.50 0.16 0.06 0.35 0.71 0.78 0.11 0.50 0.44 1.55 0.11 Y/Y Y/Y Y/N Y/N Mapped Predicted Predicted Predicted 71.5 148.5 80.5 56.5 3.30 0.81 2.97 1.05 2.64 0.74 0.63 Y/Y Mapped 50.5 Y/N Y/N Y/Y gcd yaiB 1.52 1.96 43.04 0.75 1.09 1.12 1.85 2.04 61.33 0.91 1.04 13.07 1.07 2.30 60.68 5.27 2.48 33.75 10.29 1.54 74.26 Mapped Predicted Mapped 31.5 wrt p1 , 15.5 wrt p2 62.5 192.5 E cyoABCD rhsDybbCylbH-ybbDb0501 ybiJ ompX 2.05 2.17 1.98 0.90 0.42 1.44 1.13 2.46 1.94 0.55 1.05 1.61 1.52 1.44 2.15 0.72 1.68 1.70 0.49 2.84 1.81 Predicted Predicted Predicted 61.5 and 128.5 2.5 and 62.5 3.5 Y/Y Y/N Y/N ycaC Y/Y 134.5 Predicted 0.79 3.01 1.73 2.54 1.41 2.99 1.17 2 , part B , be confirmed ; these were cydDC , hlyE , adhE , glpTQ , and arcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Genes most highly repressed upon IbsC , ShoB , LdrD and TisB overexpression.a Average fold repression dusB/yhdG ﬁs yeeF apt efeU ( ycdN ) CRP , IHF , Fis tRNA dihydrouridine synthase B Transcriptional activator Putative transporter Adenine phosphoribosyl transferase Inactive ferrous iron permease 6.5 7.7 0.9 0.9 0.8 0.9 1.2 1.1 1.1 1.2 1.2 2.1 4.3 3.8 4.7 2.3 1.2 17.8 6.1 21.2 CpxR Inactive ferrous iron transporter Inactive ferrous iron transporter Porin Unknown efeO ( ycdO ) efeB ( ycdB ) ompF yagU 14.3 2.6 6.1 26.7 1.0 1.8 1.4 1.7 1.2 1.8 0.9 2.5 1.2 1.2 0.5 1.2 OmpR , EnvYd a. Genes and operons which are repressed 5-fold in three independent experiments are listed with the average fold repression for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage shock protein Regulator ; phage shock protein Regulator ; phage shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_20">
The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829938" target="_blank">9829938</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8779563" target="_blank">8779563</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></label>
<input id="act_similarity_20", type="checkbox">
<div class="ACT_similar">
<li>The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10892648" target="_blank">10892648</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631665" target="_blank">8631665</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542180" target="_blank">10542180</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751275" target="_blank">7751275</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss during stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li></div>
<li>We thank the Fondation pour la Recherche MeÂdicale for a fellowship to F.R. , and the NATO Science Program for a Collaborative Research Grant ( CRG 972150 ) References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>pKOXS B. Demple Media and cell growth conditions Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during P growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_28">
[ 5 ] S. Altuvia , M. Almiron , G. Huisman , R. Kolter , G. Storz , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></label>
<input id="act_similarity_28", type="checkbox">
<div class="ACT_similar">
<li>[ 24 ] Altuvia S. , Almiron M. , Huisman G. , Kolter R. , Storz G , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
<li>Referencec Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Ston , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>J Mol Biol 215 :403 410 Altuvia S , Almir6n M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709966" target="_blank">8709966</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_32">
Altuvia , S. , Almiron , M. , Gjalt , H. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></label>
<input id="act_similarity_32", type="checkbox">
<div class="ACT_similar">
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>References Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_35">
Genes Dev 6:2646 -2654 Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ~ rs in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476863" target="_blank">7476863</a>]<br></label>
<input id="act_similarity_35", type="checkbox">
<div class="ACT_similar">
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>References Altuvia S , Almiron M , Hulsman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and rS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>References Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during S growth and by IHF and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R & Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF S and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigmaS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and s S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li></div>
<li>Therefore , the relatively weak binding of IHF observed in vitro could have physiological signiﬁcance in the regulation of pst , i.e. , IHF binds only during the late stationary phase when IHF molecules are abundant and the cells are fully starved for Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Because this low-affinity CRP site also appears to be dependent on the supercoiling of the DNA , and the negative supercoiling of the DNA decreases as the cells reach stationary phase ( Travers and Muskhelishvili , 2005 ) , the affinity of CRP III would be expected to be at its lowest when IHF approaches its highest concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>This conclusion is supported by the growing number of reports of IHF involvement in cellular metabolism during stationary phase [ 21-25 ] * Correspondence and reprints Both ihfA and ihfB were shown to be negatively regulated by IHF [ 20 , 26-28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Although OxyR can activate transcription by Es70 , and IHF can enhance transcription by Es38 ( Altuvia et al. , 1994 ; Jeong et al. , 2006 ) , the molecular basis of how dps expression is regulated by growth-phase is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>As expected , occupancy clearly varied with the Kd and occupancy increased in stationary phase , con ® rming previous experiments that had shown that the amount of total IHF increases in stationary phase ( Ditto et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>1981 ; Mechulam et al. , 1987 ) and IHF levels are reported to increase as E , coli enters stationary phase ( M , Ditto and R , Weisberg , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The level of IHF increases in the stationary growth-phase ( Ditto et al. 1994 ) and several stress-speciﬁc rS-dependent promoters , such as ihfA , dps , osmY and sra , are known to be regulated by IHF ( Arﬁn et al. 2000 ; Lange et al. 1993 ; Izutsu et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>The level of IHF increases about ® ve - to six-fold when cells are in the stationary phase and then decreases when cells are diluted into fresh medium and allowed to resume growth ( Ditto et al. , 1994 ; Aviv et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>IHF levels increase 5 -- 10-fold during the stationary phase of growth ( Ditto et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>The synthesis of IHF is indeed growth-phase regulated ( as is the case for the abundant H-NS and Dps proteins ; Almiron et al. , 1992 ) and an elevated concentration in stationary phase could help either to compact DNA , or protect it against chemical or physical attacks ( as was proposed for HU ; Boubrik & Rouviere-Yaniv , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The requirements for Lrp and IHF ( especially in the absence of FNR ) are consistent with their greater abundance in slow-growing or stationary-phase organisms ( Landgraf et al. , 1996 ; Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Thus the growth-phase dependent transcriptional-regulators Fis and IHF stimulate the expression of the nuo operon and repress ndh , and the resulting increase of Nuo at the expense of Ndh raises the ATP yield in early exponential aerobic-growth ( Wackwitz et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>We emphasize , however , that the regulation of ® s expression is a complex phenomenon subject to growth-phase-dependent and stringent control , and involving in addition to IHF the effects of other factors such as FIS itself , CRP , the superhelical density of DNA and , perhaps , NTP pools ( this study ; Ball et al. , 1992 ; Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>As FIS levels are maximal after a shift to fresh medium but decrease during exponential-growth , while IHF is present at high levels in the stationary phase , CRP-dependent activation of acs transcription is maximal at the transition from the exponential to the stationary growth-phase ( Browning et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>We conclude that IHF plays a negative role in the control of osmY transcription during transition into stationary phase when the cells are grown in rich medium or in glucose-containing minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Therefore , the promoter of sra is partly dependent on sigma S. Furthermore , global transcriptional-regulators , such as FIS , IHF , cAMP , and ppGpp , were found to positively regulate directly or indirectly the transcription of sra during the stationary phase , and the region between 2455 and 2325 bp is likely to be responsible for the action of the regulators ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>After pair-wise promoter-binding assays , the increase in csgD transcription in the stationary phase was suggested to be due , at least in part , to the increase in IHF level cancelling the silencing effect of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Thus , except for the marked increase of IHF , the change in the intracellular concentrations of other transcription factors might not be the major cause of the activation of csgD promoter in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>As a result , the increased amount of IHF bound on the csgD promoter should lead to cancellation of the H-NS-mediated silencing of csgD in stationary phase ( Fig. 7d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Thus , the silencing of csgD by H-NS should be reduced in stationary phase because of the increase in IHF level ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response to oxidative-stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth-phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Overexpression of IHF during balanced growth was not sufﬁcient to induce these proteins , but it resulted in an increased synthesis of rpoH-dependent heat-shock proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>However , in no case was IHF overproduction sufﬁcient to induce these proteins in the exponential-growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Hence , this limited promoter sequence exhibits growth-phase-dependent regulation , demonstrating that while Fis and IHF exert antagonistic effects on the ﬁs promoter , they are not necessary for growth-phase-depen-dent regulation of ﬁs expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Although the relative levels of IHF and Fis in E. coli vary according to growth rate and growth-phase , to date , we have found no growth rate-dependent or growth-phase-dependent variations in regulation at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>It is now apparent that IHF also contributes to the growth-phase-dependent regulation of ndh .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>We find that the in viva level of the cy subunit of IHF increases 15-fold with a 4-fold increase in growth rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_73">
In contrast , IHF levels increase steadily as the growth rate decreases to become , in stationary phase , the second most abundant nucleoid-associated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></label>
<input id="act_similarity_73", type="checkbox">
<div class="ACT_similar">
<li>In contrast , IHF levels increase steadily as the growth rate decreases to become the second most abundant nucleoid-associated protein by stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>In stationary phase the concentration of free IHF in the cell increases about sevenfold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>In contrast , the level of H-NS and IHF increases as the cells enter the stationary phase of growth ( Spassky etai , 1984 ; Bushman efai , 1985 ; Ditto etai , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>A yeiL -- lacZ transcriptional fusion showed that aerobic yeiL expression increases at least sixfold during stationary phase , requires RpoS , and is positively autoregulated by YeiL , positively activated by Lrp ( and IHF in the absence of FNR ) and negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>These results suggest that FIS , IHF , cAMP , and ppGpp are involved in the positive regulation of sra expression during the stationary growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>However , details of the mechanisms of ﬁne control of sra expression in the exponential phase as well as in the stationary phase by sigma S , OmpR , FIS , IHF , cAMP , and/or ppGpp remain to be investigated further .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>The results presented above suggest that the increase in the level of IHF as the cells enter the stationary phase -LCB- Bushman etai , 1985 ; Ditto etai , 1994 ) is partly caused by the increase in the mRNA level of the two IHF genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>IHF stimulates the expression of dps in the stationary phase in a sigma S-dependent fashion ( 2 ) , and the level of IHF increases approximately 10-fold during the transition to the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>IHF levels were found to increase 5 - to 10-fold during the stationary phase of growth [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>The expression of ihfA and ihfB increases when the cells enter the stationary phase and as a result the concentration of IHF increases tenfold [ 3 , 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>The expression of ihfA-lacZ in the absence of the IHF heterodimers is increased 3 - to 5-fold in the exponential phase and 2.5 - to 3-fold in stationary phase , while the expression of ihfB-IacZ increases about two-fold in both exponential and stationary phases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>A possible explanation could be the observation that the concentration of IHF in the late stationary phase is increased tenfold [ 9 ] and that high concentrations of IHF are needed to affect the expression of phoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>( c ) IHF affected the expression of pst and phoA only in Pi-starved cells that reached the late stationary phase , when the concentration of IHF in the cell is maximal [ 9 , 10 ] , and in log-cells only when it was overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>' ; switch oH Converted 01 plB439 to fímS901 pHK1 04 by replacing BsrG I-Psp 14061 Iragment 01 fimS OLgonucleotide-directed 01 to change IHF fim site 1Iram to 5 ' - TAGAA-3 ' ( fimSl1 ) and site 11 Irorn 5 ' - ATCAA-3 ' lo 5 ' - ATCCC-3 ' ( fimS9 ) Converted fimS + 01 plB439 to fimS9 , 1101 plB478 by replacing Sphl-Psp 14061 Iragrnent 01 fimS Converted fimS + 01 plB439 to fimSI1 01 pHK112 by replacing Sphl-Psp 14061 01 fimS Oligonucleotide-directed 01 plB406 to change IHF fim site Ilrom to 5 ' - CCA 3 ' ( fimSl0 ) Oligonucleotide-directed to change IHF fim site 11 Irom 5 ' - ATCCC-3 ' ( fimS9 ) Converted fimS + 01 plB429 to fímSIO 01 pHK101 by replacing Sphl -- Psp 14061 01 fimS Oiigonuc1eotide-directed to change IHF tim site 1Irom to 5 ' - TAGAA-3 ' ( fimSl Converted fimS + 01 to fimSl1 01 10 by replacing 14061 fragcnent fimS Yang and Nash Yang and Nash plB478 amp ' p1B406 , fímS9 ,11 plB483 p1B439 , fimS9 ,11 plB485 p1B439 , fimSl1 pHK101 amp ' , let ' p1B406 , ( imSl0 pHK104 amp ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>00 carP6 Plasmids Apr , promoterless lacZ Apr , promoterless iacZ 133 ] [ 22 ] [ 181 pJELI22 pMMI pJST5 Apr , S typhimurium carA-lacZ fusion cloned in pJEL 122 ; P2 inactivated Apr , AST3 derivativc of pJST5 Apr , AST7 derivative of pJST5 Apr , derivative of pJST5 carrying the AST7 deletion and a 7bp duplication Apr , AST I 1 derivative of pJST5 EcoRI-HaeIIi S typhimurium carAB fragment from pMKs 15 carrying the ST5 mutation cloned in EcoRI and Smal digested MI3mpl9 pJST3ST5 pJSTSST7 pJSTSSTT-2 [ 181 This work This work pJST5ST I I M 13ST5 This work ' 181 IHF is equally requiredfor CarP and for pyrimidinemediated inhibition of Dam methylase modification of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>HU constrains negative supercoifs in DNA ( Broyles and Pettijohn , 1986 ) and is inencoding gene ; hupA and hupB , genes encoding HU-2 and HU - 1 , respectively ; Hy , hygromycm B ; IHF , integration host factor ; himA and himD , genes encoding IHFa and IHFB , respectively ; kb , kiiobase ( sf or 1000 bp ; Km , kan ~ y ~ in ; LB , Lucia-Bertani ( medium ) ; nt , nucleotidefs ) ; ohgo , oligodeoxyribonucleotide ; PA , pofyacfylam ~ de ; PAGE , PA-gel etectro-phoresis ; R , resistance/resistant ; s , sensitivity/sensitive ; SDS , sodium dodecyl sulfate ; Tc , tetracycline ; trp $ genes encoding enzymes for tryptophan synthesis ; wt , wild type ; : : , novel junction ( fusion or insertion ) ; [ 1 , denotes plasmid-carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>iscR b2531 4.3 Adaptation dinJ himD b0226 b0912 5.7 5.7 Predicted antitoxin of YafQ-DinJ toxin-antitoxin system Integration host factor ( IHF ) , b subunit ; site-specific recombination Stress-induced protein Periplasmic glutamine-binding protein ; permease Periplasmic adaptor protein , regulator of the Cpx response and possible chaperone involved in resistance to extracytoplasmic stress Specificity factor for ClpA-ClpP , ClpS shows OxyR-dependent induction of expression by hydrogen peroxide chaperone-protease complex Acidic protein suppresses mutants lacking function of protein export GroES , 10 kDa chaperone binds to Hsp60 in pres. Mg-ATP , suppressing its ATPase activity Lysine - , arginine - , ornithine-binding periplasmic protein Signal inhibitory factor-X , Phosphohistidine phosphatase affecting ArcB H-D phosphorelay ; expression of six A is dependent on the alternative sigma factor rpoE PTS heat stable , histidyl phosphorylatable protein Dipeptide transport protein uspF ( ynaF ) a glnH cpxP b1376 b0811 b3913 5.7 5.3 4.6 b0881 4.3 clpS ( yljA ) msyB 4.3 b1051 groS 4.3 b4142 argT sixA 4.0 4.0 b2310 b2340 4.0 4.0 b2415 b3544 ptsH dppAb Metabolism fldA acpP glgS ivbL acs Flavodoxin 1 Acyl carrier protein 5.3 4.3 4.3 4.3 4.0 b0684 b1094 b3049 b3672 b4069 Predicted glycogen synthesis protein , rpoS dependent ilvB operon leader peptide Acetyl-CoA synthetase RNA rpsV b 4.6 30S ribosomal subunit protein S22 b1480 Unknown function ybeL a ybiM a , b yjfO yahO ydiZ b yoaC yjdN b yacL ybaA ycfP ynhG yqjC yegP yobF smg b0643 b0806 b4189 b0329 b1724 b1810 b4107 b0119 b0456 b1108 b1678 b3097 b2080 b1824 b3284 Putative alpha helical protein Predicted periplasmic protein , YhcN Probable lipoprotein , YhcN family Predicted periplasmic protein , YhcN Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein 6.5 6.5 5.7 4.9 4.9 4.9 4.9 4.6 4.6 4.6 4.6 4.6 4.3 4.0 4.0 ily fam ily fam Abbreviation : LB , Luria -- Bertani medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18309357" target="_blank">18309357</a>]<br></li>
<li>Regulated expression of 21 small proteins Comparison of growth under the two indicated conditions Minimal glucose vs minimal glycerol Proteina M b ap c LB vs Aerobic vs Mock treatment vs pH 7.5 vs 30 ° C vs 37 ° C vs Mock Mock low oxygen SDS EDTA pH 5.5 45 ° C 10 ° C treatment treatment vs vs H2O2 diamide RegulonDB minimal glucose YpfM YccB Monocistronic None d appAB-yccB - 38 , 70 , AppY , appC ArcA Monocistronic None Monocistronic None cydAB-ybgTE 70 , ArcA , Fnr , FruR , HNS ykgMO 70 , Zur yqgB-speAB None Monocistronic None yjhBC-ythA None Monocistronic NL yqeKL None Monocistronic NL Monocistronic NL ilvXGMEDA 70 , Lrp , IHF yeaP-yoaKJ None Monocistronic NL yeaP-yoaKJ None yoeI-yeeF None Monocistronic None In IsrB sRNA NL yobF-cspC None YncL YneM - - e YbgT - ¬ YkgO YqgB YnhF YthA YobI YqeL YohP YkgR IlvX YoaJ YpdK YoaK YoeI YbhT AzuC YobF - - - - - ¬ - ¬ - ¬ - - - ¬ - - ¬ - - - ¬ - ¬ - -- ¬ -- - ¬ - ¬ - ¬ - - ¬ a All small proteins that had at least a 4-fold change in protein levels in at least one experiment are listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>In this case , labelled fragments were incubated with purified IHF ( a gift from H. A. Nash , NIH , USA , who suggested that it always be diluted in the presence of 100 mM KCl and 2 mg bovine serum albumin ml-l ) in 25 pl reaction mixture containing 50 mM Tris/HCl ( pH 7.9 , 7 mM KCl , 1-1mM EDTA , 1mM dithiothreitol , 7 mM MgCI , , 3 mM CaCI , , 10/o ' ( w/v ) glycerol , 200 pg nuclease-free bovine serum albumin ( ml reaction mixture ) - ' and 2 pg denatured , sonicated salmon sperm DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>Oligomerization via the N-terminus of DnaA is also required for the initiation reaction .6 Initiation of replication occurs once per cell cycle and , during rapid growth , at all origins in the cell simultaneously .7 DnaA binding to the low-affinity sites , as well as binding of architectural elements ( Fis , IHF , and HU ) , may possibly contribute to the synchrony of initiation .8 -- 10 Two main mechanisms prevent initiation from reoccurring at already replicated origins and involve either inactivation of the newly replicated origin ( by sequestration ) or inactivation of the DnaA protein ( by hydrolysis of ATP-DnaA to ADP-DnaA ) .11,12 Specific inactivation of new origins occurs because new origins are hemimethylated ( i.e. , adenine nucleotides of GATC sites are methylated on only one strand ) .13,14 The SeqA protein binds to the hemimethylated GATC sites and prevents remethylation by Dam methylase .15 Regions on the chromosome that have high numbers of GATC sites ( oriC and dnaA promoter ) remain hemimethylated considerably longer ( for up to one-third of a cell cycle ) than regions with few GATC sites .14,16 Also a region with an artificially high number of GATC sites ( an oriC-like distribution of GATC sites placed in the tnaA gene ) was found to be sequestered as long as oriC , showing that sequestration depends on the concentration and distribution of GATC sites .17 During sequestration of newly replicated origins , ATP-DnaA is hydrolyzed to ADP-DnaA in a process that requires the Hda protein and the DNA-loaded β-clamp subunit of the DNA polymerase III holoenzyme .18 This process contributes to lowering the initiation potential by lowering the proportion of ATP-form DnaA .19 The total amount of free DnaA is also lowered because new DnaA binding sites are generated by replication .20,21 After sequestration , the initiation potential is low , and initiation does not occur until a generation has passed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835566" target="_blank">18835566</a>]<br></li>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor biosynthesis , and DNA-RNA metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative-stress .6,11 -14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>The amounts of transcription factors added were : CpxR and IHF , 0 , 0.5 , 1.0 , 1.5 , 2.0 , 2.5 , 3.0 , 3.5 , 4.0 , 4.5 , 5.0 , 5.5 and 6.0 pmol for lanes 1 -- 13 ; H-NS , 0 , 2.5 , 5.0 , 7.5 , 10 , 12.5 , 15 , 17.5 , 20 , 22.5 , 25 , 27.5 and 30 pmol for lanes 1 -- 13 ; OmpR and RstA , 0 , 5 , 10 , 15 , 20 , 25 , 30 , 35 , 40 , 45 , 50 , 55 and 60 pmol for lanes 1 -- 13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The concentrations of Fis protein in the reactions were as follows : lanes 1 and 5 , no protein ; lanes 2 and 6 , 0.11 M ; lanes 3 and 7 , 0.22 M ; and lanes 4 and 8 , 0.44 M. Note that pnrf53 EcoRI-MfeI fragments carry pnrf sequences from positions 209 to 50 and , therefore , only carry the IHF II , IHF III , and Fis II binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>B. End-labelled AatII -- BamHI fragment was incubated with FNR DA154 in combination with Fis or IHF and treated with DNase I.The concentration of FNR was : lanes 2 , 5 and 8 , no protein ; lanes 3 , 6 and 9 , 0.25 mM ; lanes 4 , 7 and 10 , 0.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The concentration of Fis in each reaction mixture was as follows : lanes 1 to 4 , no protein ; lane 5 , 447 nM ; lane 6 , 894 nM ; lane 7 , 1.79 M. ( B ) End-labeled pnir7150 AatII-HindIII fragment was incubated with RNA polymerase ( RNAP ) , FNR DA154 , IHF , and Fis and subjected to potassium permanganate footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The concentration of RNAP was 20 nM ; the concentration of IHF was 2 nM in lanes 2 and 6 , 5 nM in lanes 3 and 7 , and 50 nM in lanes 4 and 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>The concentration of IHF protein in each reaction mixture was as follows : lanes 1 and 6 , no protein ; lanes 2 and 7 , 25 nM ; lanes 3 and 8 , 50 nM ; lanes 4 and 9 , 100 nM ; lanes 5 and 10 , 300 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The concentration of IHF in each reaction was as follows : lane 1 , 5 , 6 and 7 , no protein ; lanes 2 and 8 , 20 nM ; lanes 3 and 9 , 100 nM ; lanes 4 and 10 -- 13 , 200 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The concentration of IHF protein in each 10 ml preincubation mix , was as follows : lanes 1 and 7 , no protein ; lanes 2 and 8 , 0.5 nM IHF ; lanes 3 and 9 , 1 nM IHF ; lanes 4 and 10 , 5 nM IHF ; lanes 5 and 11 , 20 nM IHF ; lanes 6 and 12 , 100 nM IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The concentration of IHF in each reaction was as follows : lanes 2 , 6 , 7 , 11 and 12 , no protein ; lanes 3 , 8 and 13 , 0.46 mM ; lanes 4 , 9 and 14 , 0.92 mM ; lanes 5 , 10 m and 15 , 1.84 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_104">
The concentration of IHF protein in each reaction mixture was as follows : lanes 1 and 8 , no protein ; lanes 2 and 9 , 6 nM ; lanes 3 and 10 , 13 nM ; lanes 4 and 11 , 25 nM ; lanes 5 and 12 , 50 nM ; lanes 6 and 13 , 100 nM ; lanes 7 and 14 , 300 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></label>
<input id="act_similarity_104", type="checkbox">
<div class="ACT_similar">
<li>The concentration of IHF protein in each reaction was : lanes 1 , 7 and 13 , no protein ; lanes 2 , 8 and 14 , 25 nM ; lanes 3 , 9 and 15 , 50 nM ; lanes 4 , 10 and 16 , 100 nM ; lanes 5 , 11 and 17 , 200 nM ; lanes 6 , 12 and 18 , 300 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The concentration of IHF protein in each reaction was : lanes 1 , 6 and 11 , no protein ; lanes 2 , 7 and 12 , 25 nM ; lanes 3 , 8 and 13 , 50 nM ; lanes 4 , 9 and 14 , 100 nM ; lanes 5 , 10 and 15 , 200 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li></div>
<li>The concentration of IHF in each reaction was as follows : lanes 2 , 3 , 4 , 11 , 12 , 14 and 15 , no protein ; lanes 5 , 7 and 8 , 58 nM ; lanes 6 , 9 , 10 , 13 , 16 and 17 , 465 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The concentration of IHF in each reaction mixture was as follows : lanes 1 , 8 , and 15 , no protein ; lanes 2 , 9 , and 16 , 15 nM ; lanes 3 , 10 , and 17 , 30 nM ; lanes 4 , 11 , and 18 , 59 nM ; lanes 5 , 12 , and 19 , 117 nM ; lanes 6 , 13 , and 20 , 234 nM ; and lanes 7 , 14 , and 21 , 469 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The concentration of IHF in each reaction was as follows : lanes 2 -- 6 and 14 , no protein ; lanes 7 -- 13 and 15 -- 17 , 930 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The concentration of IHF in each reaction was : lanes 2 and 9 -- 14 , no protein ; lane 3 , 29 nM ; lane 4 , 58 nM ; lane 5 , 116 nM ; lane 6 , 233 nM , lane 7 , 465 nM ; lane 8 , 930 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The concentration of IHF was as follows : lanes 1 to 5 , no protein ; lanes 6 to 8 , 200 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_112">
The concentration of IHF in each reaction mixture was as follows : lanes 1 , 2 , and 5 to 7 , no protein ; lane 3 , 465 nM ; lane 4 , 930 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></label>
<input id="act_similarity_112", type="checkbox">
<div class="ACT_similar">
<li>The concentration of IHF in each reaction was as follows : lanes 2 -- 5 , no protein ; lanes 6 -- 9 , 465 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The concentration of IHF in each reaction mixture was as follows : lanes 1 , 2 , and 6 , no protein ; lanes 3 and 7 , 233 nM ; lanes 4 and 8 , 465 nM ; lanes 5 and 9 , 930 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_115">
The concentration of IHF was : lanes 1 -- 4 and 8 -- 11 , no protein ; lanes 5 and 12 , 0.23 mM ; lanes 6 and 13 , 0.46 mM ; lane 7 and 14 , 0.93 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></label>
<input id="act_similarity_115", type="checkbox">
<div class="ACT_similar">
<li>The concentration of IHF was : lanes 2 -- 6 and 10 -- 12 , no protein ; lane 7 , 0.23 mM ; lane 8 , 0.46 mM ; lane 9 , 0.93 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li></div>
<li>The concentration of FIS in each reaction was as follows : lanes 1 and 6 , no protein ; lanes 2 and 7 , 0.016 mM ; lanes 3 and 8 , 0.049 mM ; lanes 4 and 9 , 0.148 mM ; lanes 5 and 10 , 0.450 mM.The concentration of IHF in each reaction was as follows : lanes 1 -- 5 , no protein ; lanes 6 -- 10 , 0.450 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The concentration of IHF in each reaction was as follows : lanes 1 and 2 , no protein ; lanes 3 and 6 , 0.114 mM ; lanes 4 and 7 , 0.228 mM ; lanes 5 and 8 , 0.460 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The concentration of IHF was as follows : lanes 2 and 6 , no protein ; lanes 3 and 7 , 0.23 M ; lanes 4 and 8 , 0.46 M ; and lanes 5 and 9 , 0.93 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_120">
The concentration of IHF was : lanes 2 , 4 , 5 and 7 , no protein ; lanes 3 , 6 , 8 and 9 , 0.93 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></label>
<input id="act_similarity_120", type="checkbox">
<div class="ACT_similar">
<li>The concentration of IHF was : lanes 2 -- 4 , no protein ; lanes 5 -- 7 , 0.93 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li></div>
<li>The concentrations of IHF used were as follows : 0 nM , lanes 1 and 9 ; 1 nM , lanes 2 and 10 ; 2.5 nM , lanes 3 and 11 ; 5 nM , lanes 4 and 12 ; 10 nM , lanes 5 and 13 ; 20 nM , lanes 6 and 14 ; 40 nM , lanes 7 , 8 , 15 , and 16 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The concentrations of IHF protein in the reactions were as follows : lanes 1 , 6 , and 11 , no protein ; lanes 2 , 7 , and 12 , 25 nM ; lanes 3 , 8 , and 13 , 50 nM ; lanes 4 , 9 , and 14 , 0.1 M ; and lanes 5 , 10 , and 15 , 0.2 M. ( B ) 32P-end-labeled pnrf53 EcoRI-MfeI fragments were incubated with increasing concentrations of puriﬁed Fis protein , as follows : lanes 1 to 4 , wild-type pnrf53 ; and lanes 5 to 8 , pnrf53/p175G ( Fis II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_124">
The final protein concentrations used for IHF , IntS , and TorI in the reactions were 0.125 , 0.25 , and 0.5 M. G A sequencing ladders are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></label>
<input id="act_similarity_124", type="checkbox">
<div class="ACT_similar">
<li>The final protein concentrations used for IHF , IntS , and TorI in the reactions were 0.03 , 0.06 , 0.125 , and 0.25 M. G A sequencing ladders are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_126">
The following concentrations ( nanomolar ) of IHF were added : 0 , lanes 1 , 5 , and 9 ; 20 , lanes 2 and 6 ; 40 , lanes 3 and 7 ; 80 , lanes 4 and 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></label>
<input id="act_similarity_126", type="checkbox">
<div class="ACT_similar">
<li>The following concentrations ( nanomolar ) of IHF were added : 0 , lane 1 ; 20 , lanes 2 and 5 ; 40 , lanes 3 and 6 ; 80 , lanes 4 and 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li></div>
<li>Amount of IHF required in vitro for 50 % protection against DNase I. Approximately 0.1 pmol of fragment 1 , end labeled on the bottom strand , was incubated with the following concentrations ( nanomolar ) of IHF : lane 1 , 0 ; lane 2 , 20 ; lane 3 , 40 ; lane 4 , 60 ; lane 5 , 80 ; lane 6 , 100 ; lane 7 , 120 ; lane 8 , 140 ; lane 9 , 160 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>( B ) End-labeled pnir fragments were incubated with increasing concentrations of puriﬁed IHF protein : lanes 1 to 5 , pnirEC K-12 EcoRI-HindIII fragment ; lanes 6 to 10 , pnirSTM EcoRI-HindIII fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>End-labeled pnir fragments were incubated with increasing concentrations of IHF protein and subjected to DNase I footprinting : lanes 1 to 7 , pnir fragment from E. coli K-12 ( pnirEC K-12 ) ; lanes 8 to 14 , pnirEPEC ; lanes 15 to 21 , pnirSTM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>End-labelled pnir7150 EcoRI -- HindIII fragments were incubated with increasing concentrations of puriﬁed IHF protein : lanes 1 -- 6 , wild-type pnir7150 ; lanes 7 -- 12 , pnir7150/p112G ; lanes 13 -- 18 , pnir7150/p111A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>End-labelled EcoRI ± NsiI pnir fragments were incubated with increasing concentrations of purified IHF protein : lanes 1 ± 6 , wild-type pnir ; lanes , 7 ± 12 , pnir p99G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Together , these fragments were incubated with increasing concentrations of puriﬁed IHF protein : lanes 1 -- 5 , wild-type pnir7150 ; lanes 6 -- 10 , pnir7150/p112G ; lanes 11 -- 15 , pnir7150/p111A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>( A ) 32P-end-labeled pnrf53 EcoRI-MfeI fragments were incubated with increasing concentrations of puriﬁed IHF protein , as follows : lanes 1 to 5 , wild-type pnrf53 ; lanes 6 to 10 , pnrf53/p104Gp103C ( IHF II ) ; and lanes 11 to 15 , pnrf53/p124Gp123C ( IHF III ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>End-labelled pacs AatII -- HindIII fragment was incubated with increasing concentrations of IHF protein in the absence or presence of a constant amount of CRP or FIS proteins and subjected to DNase I footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>A. End-labelled pnir7150 AatII -- HindIII fragment ( carrying pnir sequences from positions -150 to +36 ) was incubated with increasing concentrations of IHF and Fis protein and subjected to DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>C. End-labelled pnir7150 EcoRI -- NsiI fragment was incubated with increasing concentrations of puriﬁed IHF and phospho-NarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>End-labelled pnir7150 AatII -- HindIII fragment was incubated with increasing concentrations of phospho-NarL and IHF protein and subjected to DNase I footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>70-Dependent Transcription ... 3 Activation of acsP2 by CRP ... 3 Negative Regulation by Nucleoid Proteins ... 3 Negative regulation by FIS ... 3 Negative regulation by IHF ... 3 Independent regulation of acs transcription by FIS and IHF ... 3 Other Regulatory Factors ... 3 Signals ... 3 OST-TRANSCRIPTIONAL CONTROL ... 3 Post-Transcriptional Control by the Csr System ... 3 Post-Translational Control by Acetylation-Deacetylation ... 3 AMP-ACS-Dependent Acetylation and Chemotaxis ... 3 CETATE SWITCH IN OTHER ORGANISMS ... 3 PTA-ACKA/AMP-ACS Acetate Switch of the Gram-Positive Bacterium B. subtilis ... 3 Acetate Assimilation by the PTA-ACK Pathway of an Industrial Corynebacterium sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>x Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentiﬁed o xidant-responsive factor Joon-Hee Lee , Won-Sik Yeo and Jung-Hye Roe * Laboratory of Molecular Microbiology , School of Biological Sciences , and Institute of Microbiology , Seoul National University , Seoul 151-742 , Korea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>, B-Galactosidase specific activity ( 28 ) from a pfl-lacZ fusion was determined for cells grown aerobically ( solid bars ) , anaerobically ( open bars ) , or anaerobically with pyruvate ( stippled bars ) in rich media with the following strains : wt ( wild type ) , parent , RM123 ; himD , IHF - , RM444 ; frzr , FNR - , RM135 ; arcA , ArcA - , RM320 ; fnr himD , FNR - IHF - , RM324 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>In a himA mutant with a defective a-subunit of IHF , b-galactosidase activity was elevated by three - to fourfold when either the sucA-lacZ or the sdhCDAB-a Cells containing lSJP43 or lSJP61 were grown in a minimal medium ( pH 7.0 ) with the indicated additions except when buffered LB was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209026" target="_blank">9209026</a>]<br></li>
<li>To analyse the effect of the deletion and insertion mutations altering the distance between the IHF target site and the in rivo protected GATC site on the ex-ression and regulation of promoter P l , we measured galactosidase specific activities of cultures of the Lac - strain MC4100 transformed with plasmids pJSTS , pJST5ST7 , pJST5STII and pJST5ST7-2 grown either in minimal medium or in minimal plus arginine and uracil supplemented medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>In addition to the operon-specific regulation mediated by the PaaX repressor , the paa catabolic cluster of E. coli is subjected to control by the two global regulators , CRP ( 23 ) and IHF ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>We have shown in this work that the specific PaaX-mediated control of the Pa and Pz promoters for PA degradation in E. coli is , in turn , subordinated to a superimposed regulation mediated by global regulators such as CRP and IHF , which connect the expression of the paa catabolic genes to the metabolic and energetic status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Known regulators involved in the control of acs expression in E. coli are cAMP receptor protein ( CRP ) , factor for inversion stimulation ( Fis ) , integration host factor ( IHF ) ( Wolfe 2005 ) and possibly the dual transcriptional-regulator for an-oxic redox control ( ArcA ) ( Covert et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li>
<li>We had shown previously that the Pa and Pz promoters of the two PA catabolic operons in E. coli were under control of global regulators such as the cyclic adenosine monophosphate ( cAMP ) receptor protein ( CRP ) , that accounts for the catabolite repression control mediated by glucose , and the integration host factor ( IHF ) ( Ferrández et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Superimposed in the specific PaaX-mediated repression , two global regulators , the cAMP receptor protein ( CRP ) and the integration host factor protein ( IHF ) , act as activators of the gene expression driven by Pa and Pz promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The 12 - to 15-foid anaerobic induction ot pfi Q & r \ e expression is mediated by seven co-ordinately regulated promoters and induction requires the concerted action of the ArcA , Fnr and integration host factor ( IHF ) proteins ( Sawers and Bock , 1988 : 1989 ; Sawers and Suppmann , 1992 ; Sirko etat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>The other two IscR binding regions have not yet been tested for their role in suf regulation in vivo ; only weak binding of region 3 was observed by high IscR concentrations in vitro , and integration host factor ( IHF ) , which activates suf in conjunction with OxyR in response to peroxide stress , exhibits stronger binding afﬁnity than does IscR for region 2 ( 11 , 12 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Laboratory of Genetics The Rockefeller University 1230 York Avenue , New York NY 10021 , USA # 1999 Academic Press Keywords : s54 ; PspF ; PspF HTH ; oligomerization ; psp enhancer * Corresponding author Introduction Transcription of the phage-shock protein ( psp ) operon ( pspA-E ) of Escherichia coli is under control of the alternative sigma factor , s54 , is stimulated by integration host factor ( IHF ) , and is dependent on a cis-acting sequence and the PspF activator protein ( Weiner et al. , 1991 , 1995 ; Jovanovic et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>In E.coli it is repressed by Fur , ArcA , Fnr and integration host factor IHF , and it is activated by the product of the soxRS locus and mutated alleles of soxQ locus ( such as CfxB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812853" target="_blank">11812853</a>]<br></li>
<li>Expression of this operon is induced by growth on glycolate and is under the control of an activator protein encoded by the divergently transcribed gene glcC ( see Fig. 1 ) Expression is strongly dependent on the integration host factor ( IHF ) and is repressed by the global respiratory regulator ArcA-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11283302" target="_blank">11283302</a>]<br></li>
<li>Yona-Nadler , C. , Umanski , T. , Aizawa , S. , Friedberg , D. , and Rosenshine , I. ( 2003 ) Integration host factor ( IHF ) mediates repression of ﬂagella in enteropathogenic and enterohaemorrhagic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Integration host factor ( IHF ) is required for the activation of developmentally regulated genes in Caulobacter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>J Mol Biol 213 : 109-121 Gober JW , Shapiro L ( 1990 ) Integration host factor ( IHF ) is required for the activation of developmentally regulated genes in Cuirlobncter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>These sites are located between 75 and 140 bp upstream of the norV promoter [ 4 ] and each contains the Key words : σ54 , Escherichia coli ﬂavorubredoxin , cGMP-speciﬁc and - stimulated phosphodiesterases , Anabaena adenylate cyclases and Escherichia coli FhlA domain ( GAF domain ) , nitric Abbreviations used : AAA + , ATPase associated with various cellular activities ; EBP , enhancer binding protein ; IHF , integration host factor ; GAF , cGMP-speciﬁc and - stimulated phosphodiesterases , Anabaena adenylate cyclases and Escherichia coli FhlA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Abbreviations used : AAA + , ATPase associated with various cellular activities ; GAF , cGMP speciﬁc and stimulated phosphodiesterases , Anabaena adenylate cyclases and Escherichia coli FhlA ; IHF , integration host factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>At 661 0 1991 Academic Press Limite t Abbreviations used : CAP = CRP , catabolite activator protein , CAMP receptor protein ; CAMP , cyclic AMP ; GlcNAc , N-acetylglucosamine ; PEP , phosphoenolpyruvate ; PTS , phosphotransferase system ; GlcN , glucosamine ; bp , base pair ( s ) ; kb , kilobase pairs ; DMS , dimethyl sulphoxide ; DTE , dithioerythritol ; IPTG , isopropyl-fi-n-thiogalactoside ; IHF , integration host factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Conversely , PspB and PspC act as positive regulators of psp operon transcription by overcoming the negative regulation 1 The abbreviations used are : pIV , protein IV ; IHF , integration host factor ; IPTG , isopropyl 1-thio - - D-galactopyranoside ; RT , reverse transcriptase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>1 The abbreviations used are : PGA , penicillin G acylase ; 6-APA , 6-aminopenicillanic acid ; PAA , phenylacetic acid ; PAA-CoA , phenylac-etic acid-coenzyme A ; CRP , cAMP-receptor protein ; IHF , integration host factor ; RT , reverse transcriptase ; EMSA , electrophoretic mobility shift assay ; ChIP , chromatin immunoprecipitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>1 The abbreviations used are : PA , phenylacetic acid ; bp , base pair ( s ) ; CRP , cAMP receptor protein ; CoA , coenzyme A ; IHF , integration host factor ; LH , left-half ; OR , operator region ; PA-CoA , phenylacetyl-coen-zyme A ; PCR , polymerase chain reaction ; RH , right-half .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>E. Gryson B-1070 Brussels , Belgium * Corresponding author Abbreviations used : CPSase , carbamoylphosphate synthetase ; IHF , integration host factor ; ATCase , aspartate carbamoyltransferase ; DHOase , dihydroorotase ; DHOdehase , dihydroorotate dehydrogenase ; OPRTase , orotate phosphoribosyltransferase ; UPRTase , uracil phosphoribosyltransferase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li>
<li>Abbreviations : CRP , CAMPreceptor protein ; FNR , fumarate and nitrate reduction regulator ; IHF , integration host factor ; ODH , 2-oxoglutarate dehydrogenase ; PDH , pyruvate dehydrogenase ; SCS , succinyl-CoA synthetase ; SDH , succinate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Abbreviations : ALDH , aldehyde dehydrogenase ; Crp , catabolite repression protein ; IHF , integration host factor ; tsp , transcriptional start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>Abbreviations : Ap , ampicillin ; H-NS , histone-like nucleoid-structuring protein ; IHF , integration host factor ; IRL , left inverted repeat ; Km , kanamycin ; Lrp , leucine-responsive regulatory protein ; Sm , spectinomycin ; UPEC , uropathogenic E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207912" target="_blank">16207912</a>]<br></li>
<li>Lrp , leucine-regulatory protein ; L-SD1 and L-SD2 , L-serine deaminases 1 and 2 ; orf , open-reading frame ; Crp , CAMP-regulatory protein ; IHF , integration host factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Abbreviations used : OmpF , outer membrane protein F ; OmpC , outer membrane protein C ; Lrp , leucine-responsive protein ; NO , nitric oxide ; IHF , integration host factor ; HTH , helix-turn-helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Abbreviations : HNS , nucleoid-associated protein ; IHF , integration host factor ; Lrp , leucine-responsive protein ; Fis , factor for inversion stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>atabolite Repression of the Propionate Catabolic Genes in C Escherichia coli and Salmonella enterica : Evidence for Involvement of the Cyclic AMP Receptor Protein Sung Kuk Lee ,1 Jack D. Newman ,1 and Jay D. Keasling1 ,2 * Department of Chemical Engineering , University of California ,1 and Synthetic Biology Department , Physical Biosciences Division , Lawrence Berkeley National Laboratory ,2 Berkeley , California Received 24 September 2004/Accepted 3 January 2005 Previous studies with Salmonella enterica serovar Typhimurium LT2 demonstrated that transcriptional activation of the prpBCDE operon requires the function of transcription factor PrpR , sigma-54 , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Communication Activation of the Escherichia coli Nitrate Reductase ( narGHJI ) Operon by NarL and Fnr Requires Integration Host Factor * ( Received for publication , August 26 , 1992 ) Imke SchroderS , Silvia Darie , and Robert P. Gunsaluss From the Department of Microbiologyand Molecular Genetics , and the Molecular Biology Institute , University of California , Los Angela , California 90024 Integration host factor protein ( IHF ) was shown to be required for Fnr - and NarL-dependent activation of the nitrate reductase ( narGHJI ) operon of Esche-richia coli in response to nitrate availability and ana-erobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Oct. 1995 , p. 5707 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Glucose Starvation Stimulon of Escherichia coli : Role o Integration Host Factor in Starvation Survival and Growth Phase-Dependent Protein Synthesis f THOMAS NYSTRÖM * Department of General and Marine Microbiology , Lundberg Laboratory , University of Göteborg , 413 90 Göteborg , Sweden Received 19 June 1995/Accepted 1 August 1995 The DNA-binding protein IHF was found to be required for starvation survival and for the induction of 14 proteins of the glucose-starvation stimulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>y Binding n-Regulation of the Escherichia coli K-12 nrf Promoter b of the NsrR Nitric Oxide-Sensing Transcription Repressor to an Upstream Site Douglas F. Browning ,1 * David J. Lee ,1 Stephen Spiro ,2 and Stephen J. W. Busby1 School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom ,1 and Department of Molecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 750802 ow D Received 1 March 2010/Accepted 9 May 2010 FNR-dependent activation of the Escherichia coli K-12 nrf promoter is downregulated by the nitric oxide-sensitive NsrR protein together with the nucleoid-associated protein IHF , which bind to overlapping targets adjacent to the DNA site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Regulation by Nucleoid-Associated Proteins at the Escherichia coli nir Operon Promoter Douglas F. Browning , * Jeffrey A. Cole , and Stephen J. W. Busby School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom Received 23 July 2008/Accepted 22 August 2008 The Escherichia coli K-12 nir operon promoter can be fully activated by binding of the regulator of fumarate and nitrate reduction ( FNR ) to a site centered at position 41.5 upstream of the transcript start , and this activation is modulated by upstream binding of the integration host factor ( IHF ) and Fis ( factor for inversion stimulation ) proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>These can be related to the maintenance , replication , partitioning and transfer of plasmids , to recombination and transposition , to phage packaging and to gene regulation where IHF can either stimulate or inhibit the action of RNA polymerase ( for a review and references see Friedman 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>H-NS , IHF and Fis sites do not exhibit spatial clustering Interaction set Genic regions Non-genic regions Clustering Interactions R Clustering Interactions Clustering Interactions R RS RS RS RS S CLS S CLS NAP a Fis Exp SHX Low ** NC Low ** Low ** ND ND ND ND H-NS Exp SHX Low ** Low * Low ** Low ** ND ND ND ND Exp SHX IHF Low ** Low ** ND ND ND ND Low * NC NAP ( 500 bp ) b Exp NC NC Low ** Low ** SHX Low * NC Low *** Low *** Fis , H-NS , IHF coding Fis , H-NS , IHF non-coding Exp Low ** Low * Low ** Low ** SHX Low ** NC Low ** Low ** a The summed interaction or clustering strength for each condition was compared with 1000 random data sets with RS or CLS between each element ( see ` Materials and Methods ' section ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>By using the quantitative immuno-blotting ( 21 ) , we found the intracellular concentration of YgiP was very low in growing E. coli cells under aerobic conditions ; however , under hypoxic or anaerobic culture conditions , the YgiP level increased more than 100-fold , reaching to the level as high as those of major nucleoid proteins HU and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Goshima , N. , Kano , Y. , Tanaka , H. , Kohno , K. , Iwaki , T. , and Imamoto , F. ( 1994 ) IHF suppresses the inhibitory effect of H-NS on HU function in the hin inversion system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123679" target="_blank">11123679</a>]<br></li>
<li>While HU is widely distributed among the eubacteria , H-NS , FIS and IHF are restricted to the proteobacteria ( Oberto and Rouviere-Yaniv , 2001 ; Tendeng and Bertin , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17116239" target="_blank">17116239</a>]<br></li>
<li>Many of these , notably Fis , IHF and H-NS , have been found to exert specific effects at particular promoters ( reviewed by Friedman , 1988 ; Finkel and Johnson , 1992 ; Williams and Rimsky , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Nonetheless , our results are supported by earlier analyses of the somewhat differing effects of gene disruptions in the global regulators H-NS [ 121-124 ] , IHF [ 125,126 ] , and Fis [ 127,128 ] in the closely-related genera Escherichia and Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>For operons involved in use of carbon sources in Escherichia coli , the most frequent global regulator is cAMP/CAP , responsible for catabolite repression , but histone-like proteins , IHF , H-NS , Fis and HU are often involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Bands identiWed by tandem mass spectrometry : a , partially resolved bands of ( above ) and ( below ) subunits of RNA polymerase ; b and c , - and - subunits of ATP synthase ; d , subunit of RNA polymerase ; e , Dps ; f , H-NS , StpA ; g , egg white lysozyme ; h , IHF - , IHF - , Fis ; i , HU-1 , HU-2 ; j , L4 ; k , S3 , S7 , S9 ; l , L13 , Fis , IHF - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>For routine assays , the antibodies were diluted 200-fold for CbpA , Fis , H-NS , IHF , RpoD , SeqA and L7/L12 , and 500 fold for CbpB , Dps , Hfg , HU , IciA , StpA , RpoA , RpoS , Rsd and RMF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>These rest & s , together with our previous Ending that cells deficient in both HU and IHF are viable at 30-37 ° C [ Kane and Imamoto , Gene 89 ( 1990 ) I33-1371 showed that E. cnfi cells deficient in any two of these three histone-like proteins are viable at 30-37 '' C , and suggested that simultaneous deficiency of all three of the proteins is lethal , There were no detectable differences in the levels of superhelicity of the reporter plasmids isolated from cells deficient in either IHF or H-NS , or from wild-type cells , but about 1576 decrease in negative superhehcity was detected for the reporter plasmid isolated from cells lacking HU and lacking both HU and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>In contrast , total negative supercoiling was strongly reduced in HU cells but strongly increased in IHF relative to WT ( this is most clear Figure 3 , 7 mg/ml chloroquine ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Figure 3 and Table 3 show the results when plasmids from WT , HU , IHF and H-NS strains were electrophor-esed in the presence of 1 , 2 , 3 , 5 , 7 or 9 mg/ml chloroquine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>These data show that levels of unrestrained negative supercoiling in the WT and HU backgrounds are similar , while negative supercoiling in H-NS and IHF strains was increased relative to WT ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic-growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic-growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Examples of transcriptional control in which the action of ArgR occurs in combination with ( an ) other regulatory protein ( s ) and co-factors have been documented previously for CRP , NtrC , Lrp , and IHF .22,23,29,50 Furthermore , artP mRNA levels were found to be induced in the stationary phase by EσS , the general stress-specific RNA polymerase ,51 upregulated ( 1.9-fold ) by the superoxide generating agent paraquat , an inducer of the soxRS system , and by sodium salicylate ( NaSal ) , an inducer of the marRA system ( 2.5-fold ) ,52 and down-regulated by Lrp ( about sixfold ) .53 Genome-wide expression profiling studies in E. coli indicate that argT and hisJ transcription are activated by NtrC ( GlnG ) ,45 and that in anaerobiosis hisJ expression is strongly increased in fnr ( 25-fold ) and arcA ( 60-fold ) mutants .54,55 None of the underlying molecular mechanisms has been unraveled yet , but the induction of E. coli artP in the stationary phase appears to depend on several secondary promoters ,51 upstream of PartP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></li>
<li>' - ' Iac2 by Transcriptional regulation regulators of respiration and IHF Transcriptional regulation of nuo ^ gyAB ' - ' lacZ by regulators of respiration and integration iiost factor ( IHF ) Regulatory mutants defective in the genes fnr , arcA , narL , narP , fhlA and himD were used to study the effects of transcriptional-regulators acting on respiratory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>We examined the effects of some global regulators ( factor inversion stimulation [ FIS ] , H-NS , OmpR , cAMP , ppGpp , and IHF ) on the expression of sra by using global regulator-defec-tive mutants lysogenized with lpF13 ( Psra-lacZ ) or carrying pFF6 ( Psra-lacZ ) and by measuring b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>In the exponential phase , only a few differences were observed between the wild-type strains and the mutants defective in FIS , H-NS , IHF , OmpR , cAMP , and ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Expression and purification of Lrp , ArgR , Crp and IHF In all cases , protein purity was assessed from single ( or , in the case of IHF , double ) Coomassie-stained bands when μg amounts were loaded onto SDS polyacrylamide gels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We report here that gltBDF is directly and negatively controlled by Crp and ArgR , in addition to the direct positive control by Lrp and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>In addition to the previously-demon-strated direct positive roles of Lrp and IHF , there are direct negative roles for Crp and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Lrp directly and strongly activates gltBD transcription in E. coli [ 74,92 ] , in a process that also requires the global regulator IHF [ 93 ] and is antagonized by Crp and ArgR [ 94 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In E. coli gltB is activated 30 -- 40 fold by Lrp when grown in MOPS glucose [ 67,69,74,92 ] , with the activation codependent on another global regulator , IHF [ 93,94 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In E. coli , PgltB is controlled by Lrp , Crp , IHF , and ArgR [ 67,74,93,94,112 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth-phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>During exponential phase , repression by Lrp , CRP , or IHF can not account for the low level of expression because similar low expression was found in mutants deficient for these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Beyond this point , osmYis finally strongly induced , which also implies that the repressing activity of cAMP-CRP , Lrp , and IHF is at least partially relieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Under these conditions , cAMP-CRP and IHF do not contribute to the specific regulatory pattern of starvation induction ( which is no longer found in a strain that is deficient for as and Lrp but has wild-type cya , crp , himA , and hip alleles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>None of the several regulatory genes mentioned above seems to be essential for hyperosmotic induction of osmY , which could still be observed in mutants deficient in as , Lrp , cAMP-CRP , or IHF and even in strains carrying rpoS lrp or rpoS hip double mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Two hypotheses could account for these results : ( i ) osmoregulation of osmY requires yet another unidentified regulatory mechanism that is totally independent of as , Lrp , cAMP-CRP , and IHF ; or ( ii ) osmotic induction could be mediated by several independently and additively acting components , some of which could be those identified in the present study as being involved in the regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Stationary-phase induction of osmY in rich medium is mediated by the combined action of as , Lrp , cAMP-CRP , and IHF , with the latter three proteins acting as transition state regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Whereas rs , Lrp , CRP , and IHF are clearly involved in stationary-phase induction , none of these regulators is essential for osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In contrast to station-ary-phase induction of osmY , which now can be explained by the concerted action of us and the negatively acting transitio state regulators Lrp , cAMP-CRP , and IHF , the problem of osmotic regulation of osmY still awaits its solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Therefore , osmY remains repressed by the concerted action of CRP , Lrp , and IHF until the growth rate is further reduced below a certain limit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>At present , we can not decide whether cAMP-CRP , Lrp , and IHF directly bind to the osmY regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>However , CRP , Lrp , and IHF are not essential for osmotic induction of osmY ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative-stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The activity of sS is ﬁne tuned by a number of global regulators , including CRP , Lrp , IHF , H-NS and Crl ( Hengge-Aronis , 1996b ; Loewen et al. , 1998 ; Pratt and Silhavy , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>A number of the rsdependent genes are also regulated through the action of one or several additional global regulators like CRP-cAMP , Fis , IHF , Lrp or H-NS ( see for instance Xu and Johnson 1995a , 1995b , 1995c ; Bouvier et al. 1998 ; Colland et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12395194" target="_blank">12395194</a>]<br></li>
<li>This activation is brought about in an RpoS-depen-dent manner in concert with a combination of one or more global regulators such as Lrp , H-NS , and IHF ( 11 , 14 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>]<br></li>
<li>This differentiates it from tlie many stereotypically regulated geiies controlled by IHF , Lrp , H-NS aiid RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108285" target="_blank">9108285</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_218">
Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></label>
<input id="act_similarity_218", type="checkbox">
<div class="ACT_similar">
<li>Regulatory pattern III ( anaerobically repressed , but not affected by ArcA ) and pattern IV ( anaerobically activated , but not affected by ArcA ) are most easily explained as genes controlled by the FNR protein or by an as yet unidentified global regulator such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li></div>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Thus , ArcA and FNR regulate respiration in response to oxygen availability ( the variable in these experiments ) , CRP and Lrp sense nutritional state via cAMP and L-leucine concentrations , and Fis , H-NS and IHF modify DNA topology , which is itself dependent on energy status [ 7,25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>By these criteria , ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp were designated as global TFs in E. coli K-12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , it is concluded that ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp are truly global regulators , at least as far as the response to oxygen availability is concerned , in the sense that their outputs create the transcriptional framework upon which local regulators operate in response to specific unstable environments Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>-80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC DISCUSSION osmY ( csi-5 ) is one of many stationary-phase-induced and osmotically regulated genes in E. coli that require us , the gene product of rpoS , for expression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Lrp Lrp -200 GACCGTGTCCTTGCAATAGGCCTGCAAGTCAAGGTGGTCTGGCGCTCGTAATTAAGAAC -140 -80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC ACKNOWLEDGMENTS We thank Erhard Bremer and his coworkers for providing bacterial strains and Merna Villarejo for sharing unpublished data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_226">
Weisberg , R.A. , Freundlich , M. , Friedman , D. , Gardner , J. , Goosen , N. , Nash , H. , Oppenheim , A. and Rouvi & e-Yaniv , J. ( 1996 ) Nomenclature of the genes encoding IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></label>
<input id="act_similarity_226", type="checkbox">
<div class="ACT_similar">
<li>[ 10 ] Weisberg R.A. , Freundlich M. , Friedman D. , Gardner J. , Goosen N. , Nash H. , Oppenheim A. , Rouviere-Yaniv J. , Nomenclature of the genes encoding IHF , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
<li>Acknowledgements We thank Hunner Lewis for computer programming , Kathy Squires , James Fee , Yuk-Ching Tse-Dmh , Ira Schwartz , Dave Laporte , Jacque LaPointe , Nicholas Kredich , Peter Model and Gary Sawers for plasmids or DNA fragments , and Howard Nash for purified IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Acknowledgments-We thank C. Gross and H. Miller for providing bacterial strains ; H. Nash for his gift of purified IHF ; L. Hansen , J. Winkelman , D. Heck , and J. Keener for comments on the manuscript ; and J. Winkelman for many helpful discussions concerning this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>This tetracycline-resistant plasmid can replicate only when functional IHF molecules are present in the cell ( Stenzel et al. , 1987 ; Biek t Cohen , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>This tetracycline-resistant plasmid can replicate only when functional IHF molecules are present in the cell ( 8 , 44 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>This role would therefore be similar to that of the other site-specific recombination accessory factors IHF , FIS , ( factor for inversion stimulation ; Kahmann et al. , 1985 ) and Tn3 resolvase , in organizing and providing specificity to other synaptic complexes ( Echols , 1986 ; Boocock et al. , 1987 ; Thompson et al. , 1987 ; Benjamin and Cozzarelli , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>This may compare with the diverse role of IHF , which appears to be involved in site-specific recombination , transposition , plasmid replication , conjugal transfer and transcriptional control of biosynthetic and catabolic operons ( Friedman et al. , 1984 ; Leong et al. , 1985 ; Dempsey , 1987 ; Morisato and Kleckner , 1987 ; Prentki et al. , 1987 ; Stenzel et al. , 1987 ; Thompson et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>This difference is entirely explained and predicted by our measured value of the non-speci ® c dissociation constant and the presence of four moderate IHF binding sites on the cloning vector , pUC18 ( Gamas et al. , 1987 ; Prentki et al. , 1987 ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>-1 +3 -12 -10 -9 -30 -29 -28 P2A P2 RNAP IHF + RNAP 200 nM CRP + IHF + RNAP 12 nM CRP + IHF + RNAP 200 nM KE101 + IHF + RNAP CRP stabilizes the TOC in the presence of binding to lower-affinity sites by IHF Because 20 nM IHF did not exert a large effect on the RNAP footprint , we speculated that the negative effect of IHF observed previously on pacs444 in vitro ( Browning et al. , 2004 ) might be due to its binding to lower-affinity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>These results , in conjunction with those from footprint analysis , suggest that CRP and FIS can bind simultaneously to CRP II , FIS II and CRP I. IHF binds the acs regulatory region simultaneously with either CRP or FIS Previously , we showed that IHF in vitro binds to a DNA site ( IHF I ) that overlaps acsP1 and in vivo represses transcription from both acsP1 and pnrfA ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>We attribute this to residual IHF and Fis repression in the absence of a functional IHF II site , as the p99G and p146A substitutions do not completely disrupt IHF and Fis binding ( Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>At pnirB , in contrast , FIS and IHF inhibit transcription by binding to upstream sites ( Browning et al. , 2000 ; 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The dotted grey ovals denote the binding of IHF to the IHF II site at position -115 that activates FNR-dependent transcription ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>FNR -41.5 IHF bound to the IHF II site activates FNR-dependent transcription Previously , we demonstrated that IHF binding to the IHF I site represses FNR-dependent transcription at pnir ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>In some cases , the presence of an IHF binding site in the promoter region favours transcription by the rS factor , while in others transcription is inhibited by IHF ( Aviv et al. 1994 ; Colland et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>1982 ; Peacock et al. , 1984 ; Peacock & Weisbach , 1985 ) and the control of phage Mu early transcription ( Krause & Higgins , 1986 ) where IHF binding to DNA resulted in the activation of a specific promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Hypersensitive site 1 ( -93 ) is associated with CRP binding to CRP I ( Beatty et al. , 2003 ) , site 2 ( -94 ) is associated with FIS binding to FIS II , and site 3 ( -95 ) occurs in the presence of CRP and a high concentration of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In a similar fashion , at the acs promoter , for example , the binding of CRP to similarly placed activating sites stabilizes RNAP binding in the presence of nucleoid proteins such as FIS and IHF ( Sclavi et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></li>
<li>CRP-dependent activation of the acs promoter acsP2 is counteracted by the binding of FIS or IHF ( Browning et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>It is also possible that IHF regulates-the-activity of other kinases or phosphatases which , under certain conditions , may alter the binding properties of OmpR ( Igo et al. 1989b ; Slauch and Silhavy 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Maximal expression of PnarG occurs under anaerobic-growth conditions in the presence of nitrate and , in addition to FNR , requires the DNA-binding proteins NarL and IHF , which bind to DNA sequences located upstream of the FNR site ( Rabin et al. , 1992 ; Schroder et al. , 1993 ) ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>In addition , himA and hip mutations result in various phenotypes ( Ile Ts in the presence of leucine , reduced supercoiling of DNA ) indicating that IHF could be involved in the regulation of the expression of some bacterial genes through chromosomal DNA binding ( Friedman et al. , 1984 ; Friden et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Since ® s expression is known to be modulated by three DNA architectural proteins , IHF , CRP and FIS itself ( Ball et al. , 1992 ; Ninnemann et al. , 1992 ; Pratt et al. , 1997 ; Nasser et al. , 2001 ) , the differences in relative b-galacto-sidase activities of deletion mutants could be explained by binding of these regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Measurement of IHF binding by UV footprinting We had reported before ( Engelhorn et al. , 1995 ) that IHF binding to its speci ® c site can be detected by an increase in band intensity of the guanine residue in the consensus pentamer sequence upon irradiation by UV light .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Moreover , the target sites for protein binding in the E. coli and S. typhimurium carAB operons contain the sequences 5 ' - ATTTAATTTATTG-3 ' and 5 ' - TTTTAATATATTG-3 ' , respectively , which resemble the IHF consensus binding site ( 5 ' - AATCAAN4TTA/G -3 ' ; Goodrich et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>It is now well established that non-speci ® c binding is an important feature of IHF ( Wang et al. , 1995 ; Yang & Nash , 1995 ; this work ) and it is interesting to note that the value of its Kc is in the same range as the dissociation constant of HU for linear DNA ( Bonnefoy et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF bends DNA upon binding to the speciﬁc sequences ( Nash , 1996 ; Radman-Livaja et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>As a vast majority of IHF in vivo is bound to DNA ( Engelhorn et al. , 1995 ; Yang and Nash , 1995 ) , non-speciWc binding is frequent and probably important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>The interaction of s54-RNAP and the activator bound to UAS ( s ) is facilitated by DNA-looping , either by local DNA topology produced by curved DNA and supercoiling , or by IHF bound to a site located between the promoter and the activator binding site ( s ) ( Kustu et al. , 1989 ; Hoover et al. , 1990 ; Collado-Vides et al. , 1991 ; Charlton et al. , 1993 ; Molina-Lopez et al. , 1994 ; Goosen & Putte , 1995 ; Carmona & Magasanik , 1996 ; Cases et al. , 1996 ) as is the case for psp ( Dworkin et al. , 1997 , 1998 ; Weiner et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>In the case of ~ 4-monitored promoters it has been demonstrated that binding of IHF to an upstream site induces DNA bending , thereby allowing a bound activator even further upstream to contact RNA polymerase ( Hoover et al. 1990 ; Claverie-Martin and Magasanik 1991 ; Gober and Shapiro 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The exact mechanism whereby IHF affects promoter activity is not yet known , but it is conceivable that in these cases where IHF binds near the promoter region it could either prevent RNA polymerase binding by steric hindrance or stimulate open complex formation by direct protein-protein contact or by local modification of the helical parameters in the promoter region ( Thompson and Mosig 1988 ; Griffo et al. 1989 ; Tsui and Freundlich 1988 ; Kur et al. 1989 ; Pereira et al. 1988 ; Giladi et al. 1990 , 1992 ; van Rijn et al. 1988 , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The interaction of IHF and NagC in vitro Binding of IHF to the element investigated above would be expected to produce a hairpin-like bend in the DNA midway between ONC1 and ONC2 ( see Rice et al. , 1996 for a review ) , and if so , should either facilitate or prevent contact between NagC bound to its two operator sites depending upon the phasing of the sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>3 , D and The extent of protec tíon produced by IHF bound to both sites I and 11 is consis tent with bindíng of a single dimer of IHF to each site ( Yang and Nash , 1989 ; Winkelman and Hatfield , 1990 ; Kur et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Transcription control of acs -- yjcH -- actP is complex and the intergenic region between acs and nrfA is bound by CRP , FNR , IHF , FIS and NarL or NarP and is folded into an ordered nucleoprotein structure that permits the two divergent promoters to be regulated independently ( Browning et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>The expression of acs encoding acetyl-coenzyme A synthetase is controlled by the regulator proteins CRP , Fis and IHF ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Results Our previous studies revealed that cells regulate acs transcription through the combined action of multiple proteins , including , but not limited to , IHF and CRP ( Kumari et al. , 2000a ; Browning et al. , 2002 ; 2004 ; 2005 ; 2006 ; Beatty et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Another example is the NarL / NarP - and FNR-stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>1 NarL and NarP counteract the negative effects of Fis and IHF but not H-NS Wu et al. ( 1998 ) proposed that NarL and NarP function at pnir , in response to nitrite , by counteracting the repressive effects of upstream factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The interaction of FNR-AR1 and FNR-AR3 with RNAP is apparently altered by NarL and/or IHF Although some FNR-dependent promoters appear to use only FNR for transcription activation , other promoters , such as PnarG , require the additional factors NarL and IHF for maximal expression ( Rabin et al. , 1992 ; Stewart , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Browning et al. ( 2000 ) proposed that , during anaerobic-growth , a complex nucleoprotein assembly , involving IHF and Fis , forms and that this represses FNR-dependent transcription activation ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Fis and IHF play roles in both activation and repression of transcription ( for examples , see Ross et al. , 1990 ; Bykowski and Sirko , 1998 ; Giladi et al. , 1998 ; Schneider et al. , 1999 ; Caramel and Schnetz , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>IHF increases acsBAK transcription by mediating the disengagement of the repressor IclR ( Resnik et al. , 1996 ) and facilitating UP element-dependent activation ( Negre et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>± 6700 ± 2200 pnir7150 pnir7150 p146A 1.0 1.0 0.8 0.8 2.5 1.9 1.1 1.6 pnir7150 p130G pnir7150 p123G pnir7150 p123G/p130G IHF binds to the nir promoter and represses FNR - dependent transcription Wu et al. ( 1998 ) demonstrated that , apart from Fis , at least one other protein factor binds to the upstream region of pnir and inhibits FNR-dependent transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The reason why the resulting multiprotein complex represses FNR-dependent activation is unclear , but we suppose that the sharp bend induced by IHF upon binding to its target ( Rice et al. , 1996 ) drives the promoter into an inhibitory structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>As at the nrf promoter , the nir promoter can be activated by FNR alone but FNR-dependent activation is repressed by IHF and Fis ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The binding of NarL or NarP to the nir promoter displaces IHF and overrides the repression of FNR-dependent activation by IHF and Fis ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Removal of OxyR , IHF , ORE-I , or ORE-II , did not abolish the ORE-III mediated oxidative induction of the sufA gene ( Lee et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The sufABCDSE operon is directly regulated by OxyR , IHF , Fur and an unidentiﬁed oxidant-responsive ( ORE ) activator ( Lee and Roe , 1997 ; Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>For example , gene expression of sufABCDSE is induced by H2O2 ( Zheng et al. 2001 ) , and oxidant induction of the suf operon is regulated by OxyR , IHF , Fur and IscR ( Lee et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>The positive effect of IHF on 054-dependent promoters is well documented ( Gober & Shapiro , 1990 ; Hoover et al. , 1990 ; Cannon et al. , 1991 ; Claverie-Martin & Magasanik , 1991 ; Lorenzo et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Beta-galficlosidase IHF Genotype Units The effect of direct ppGpp induction from a cloned truncated relA gene in wild-type and rpoS cells A direct way to induce high ppGpp levels in the cells is through the induction o -LCB- a truncated relA gene ( Schreiber et ai , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Since onipF transcription is dependent on OmpR ( Norioko et al. 1986 ; Mizuno et al. 1988 ) , it was of interest to examine whether increased amounts of this activator could reverse IHF inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>IHF binds to numerous regions in bacteriophage ( Krause and Higgins 1986 ; Kur et al. 1989 ; Giladi et al. 1990 ) and bacteria1 ( Pereira et al. 1988 ; Tsui and Freun-dlich 1988 ; Cannon et al. 1990 ; Hoover et al. 1990 ; Huang et al. 1990 ) promoters to influence transcription positively or negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Other examples exist where IHF and H-NS interplay to modulate gene expression : in the hilA promoter of the S. enterica , IHF is required to alleviate H-NS-mediated silencing depending upon growth conditions ( Queiroz et al. , 2011 ) ; P. van Ulsen and N. Goosen have reported that , in vivo , in E. coli the extent of IHF-mediated Pe promoter activation differs in the presence of H-NS ( van Ulsen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>In some of the cases in which IHF inhibits gene expression , IHF binds to a site in the promoter that overlaps the -10 and/or -35 region ( 15 , 23 , 42 ; Tsui et al. , submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Genetic regulation chiC locus narGHIJ operon hark gene Like the narGHJl operon , expression of the narK gene is positively regulated by Fnr , NarL and IHF ( Stewart & Parales 1988 ; Kolesnikow et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>Thus , neither Lrp nor IHF has a signiﬁcant effect on ﬁmB expression under the conditions described in this paper , and H-NS represses the expression of the recombinase ( Blomﬁeld et al. , 1993 ; 1997 ; Donato et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890032" target="_blank">12890032</a>]<br></li>
<li>Other regulators such as RpoS and IHF , known to aVect hmp expression , apparently are not involved in the hmpEc induction by NO ( Memb-rillo-Hernández et al. 1996 , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>IHF also inhibits the expression of ompC in vitro ( Huang et ai , 1990 ) , but the mechanism of inhibition is much less clear in this case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>We have found that ompB is negatively regulated by IHF ( Tsui et al. , submitted ) and that IHF mutants have increased levels and altered osmoregulation of OmpF ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Strains that lack IHF have increased expression and altered osmoregulation of the major outer membrane proteins , OmpF and OmpC ( Helu et al. , 1988 ; Huang et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF was previously shown to-bind to the ompC promoter region and to inhibit transcription ( Huang et al. 1990 ) and IHF mutants were found to have increased expression and altered osmoregulation of ompF ( Tsui et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>SeqA can exert a deleterious effect even when the origin is not fully methylated : overproduction of SeqA is lethal to a dam : : Tn9 strain , as it is to other mutants compromised for replication initiation ( dnaAts and IHF - ; Lu et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>Like the lambda recombinase and related bacteriophage integrases , recombination pro moted by timB is stimulated by IHF ; in mutants facking either the C \ ' ( ihtA ) or ¡ ) ( ihtB ) subunits 01 IHF , timB-pro moted switching is substantially redliced ( Dorman and Higgins , 1987 ; Eisenstein et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>We concluded that , although NarL and IHF were unable to-bind together , the other ﬁve pairs of proteins could bind together to form ternary complexes ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>This intergenic region contains sites for NarL , IHF and Fnr ( Spiro and Guest , 1990 ; Walker and DeMoss , 1991 ; Dong et al. , 1992 ; Rabin et al. , 1992 ; Schroder et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564515" target="_blank">10564515</a>]<br></li>
<li>Alterations in the IHF-bind-ing sites in each of the omp operons results in loss of IHF function in vitro and in vivo ( Tsui et al. , 1991 ; L. Huang and M. Freundlich , unpublished ; N. Ramani and M. Freundlich , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>For example , cyclic AMP receptor protein ( CRP ) and IHF are also known to play similar roles in gene regulation in other promoters ( Lobell and Schleif , 1990 ; Søgaard-Andersen et al. , 1991 ; Richet , 1996 ; Yang et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>Mutations leading to the asynchrony phenotype occur in the dnaA , dam , recA and fis genes as well as in the genes encoding the IHF protein complex ( Skarstad et al. , 1988 ; Boye et al. , 1988 , 1992 ; Skarstad and Boye , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168484" target="_blank">8168484</a>]<br></li>
<li>IHF and FIS proteins are suspected t play a role in replication initiation , it8 control in vivo , or both ( reviewed by Boye et al. , 1992 ; Finkel and Johnson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>We draw attention to the fact that the negative feedback control of hns described here is shared by the structural genes for the HU , IHF and Fis proteíns ( Miller , 1981 ; Kohno et al. , 1990 ; Ninnemann et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>coli transcriptional units and IHF acts as both a positive and negative regulator ( Freundlich et al. , 1992 ; Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>In the absence of IHF , this is true also for the psp cis-acting sequence ( Weiner et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>Furthermore , it is also possible that IHF levels are lower in Ihe defined-rich medium ( Neidhardt et al. , 1974 ) , used in our studies , than in L-broth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>d. Also regulated by Crp , IHF , Lrp , CpxR , RstA , Fur as well as sE ( Rhodius et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>IHF is also a positive effector of 070 promoters , some of which , for example hemA , narGHJZ , narK , pfl and sodA , are regulated in response to oxygen by FNR ( Guest et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>The Figure was prepared using the program MOLSCRIPT .48 Our analysis of PurR-mediated regulation of car P1 activity in various genetic backgrounds clearly demonstrates that PurR entirely relies on PepA and to a lower extent on IHF and PyrH , the architectural and regulatory proteins known to cooperate in the pyrimidine-speciﬁc control of carAB transcription .13 -- 15 This conclusion is further corroborated by the effects of selected cis-acting operator mutations ( ins1 , ins2 , del3 ) that do not alter the PUR box sequence and exhibit normal PurR binding in vitro , but are nevertheless deﬁcient for transcriptional control by both , pyrimidine resi-dues and purine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>Briefly , E. coli whole-cell lysates were prepared by sonication after lysozyme treatment , and treated with antibodies , which were prepared in rabbits against purified transcription factors , CsgD , RstA , OmpR , CpxR , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>At the times indicated , whole-cell lysates were prepared and subjected to quantitative Western blot analysis using antibodies against IHF , OmpR , H-NS , CpxR and RstA ( upper panels ) , and anti-RpoA ( bottom panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In growing cells , the intracellular concentrations of two nucleoid proteins , IHF and H-NS , are similar and higher than those of the specific regulators , RstA , OmpR and CpxR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Based on the results described herein , we propose a novel regulation model of cooperation between two positive factors ( OmpR and IHF ; and RstA and IHF ) and also between two negative factors ( CpxR and H-NS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In addition to the competition between positive and negative factors , we found unexpected cooperation between two negative factors , CpxR and H-NS , and between two pairs of positive factors , OmpR and IHF , and RstA and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Previously , we classiﬁed three factors ( OmpR , RstA and IHF ) as activators and two factors ( CpxR and H-NS ) as repressors , and found novel modes of their interplay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In principle , YdaM-generated c-di-GMP may modulate the action of any of these regulators ( i.e. OmpR , IHF , MlrA , CpxR , H-NS ) at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Interplay between three activators , MlrA , OmpR and IHF , was also analysed with respect to control of this complex promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>One explanation for the lack of csgD promoter activation in the mutants lacking OmpR and IHF is that MlrA requires these two activators for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>All five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , formed binary DNA complexes in vitro in the absence of other factors ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In addition , three factors , IHF , CpxR and H-NS , bind to the same upstream region between 2188 and 2159 , designated hot-spot I. Interplay between transcription factors in binding to these two hot-spots should influence the level of csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Interplay between MlrA , OmpR and IHF Three positive factors , IHF , OmpR and RstA , can associate simultaneously within the promoter-proximal hot-spot II region of transcription factor binding ( Fig. 6 ) , and cooperate with each other for activation of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In this report , we describe the regulatory modes of five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , and their interplays in binding in vitro to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>CRP , CpxR , CsgD , H-NS , IHF , MlrA , OmpR , and RstA on the csgD promoter ( Ogasawara et al. 2010a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The binding sites of MlrA , OmpR , RstA , CpxR , IHF and H-NS were determined by DNase-1 footprinting assays ( Fig. 2 ; and Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The apparent affinity for the csgD promoter was found to be high for IHF and CpxR , followed by OmpR , H-NS and RstA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The site of BasR binding on the csgD promoter overlaps with those of IHF , CpxR and H-NS ( Fig. 7 ) , implying that the silencing effect of H-NS might be antagonized by the combination of three regulators , BasR , IHF and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>In contrast , the second mechanism depends on the role of IHF as a nucleoid protein that , when present at higher concentrations , binds to lower-affinity , less speciﬁc , sites and , in the process , inhibits open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>IHF FimB ... FimB/FimE Sitel Sitel FF N Switch O Switch O Experimental procedures Bacterial slrains , plasmíds ; media and growth conditíons Bacterial strains , bacteriophage , and plasrnids used in this work are lisled in Table 1 ; all strains are derivatives 01 E coIíK-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Thus , it is possible that the induction of Hsps caused by overexpression of himA and hip is a direct effect of normal IHF functions rather than the result of overproduction artifacts or growth rate regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Strains lacking functional IHF will therefore under normal growth conditions have increased concentrations of the activator PspF and decreased concentrations of the negative regulator PspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>To examine whether other transcription factors regulate Pfnr , we tested whether the DNA-bending protein IHF had an effect on fnr transcription , since it has been shown to be involved in the regulation of several FNR-dependent promoters , such as narG , nir , nrfA , dmsA , ubiC , ndh , pﬂ , sodA , and narK ( 5 -- 7 , 9 , 19 , 31 , 36 , 43 , 48 , 51 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>More recently , Freundlich and coworkers reported that the E. coli ilvBN and ompB operons are subject to regulation by both CAP and IHF ( 7 , 12 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Proposed model for regulation of the sdhCDAB and sucABCD genes by the ArcA , Fnr , and IHF proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209026" target="_blank">9209026</a>]<br></li>
<li>Interestingly , IHF , PepA/CarP , PyrH and ArgR , all involved in the regulation of the tandem pair of carAB promoters , are one and all multifunctional proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li>
<li>There are so far no examples of wild-type 054-dependent promoters subject to negative regulation by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Additionally , the arginine repressor , ArgR , has been shown to be involved in the transcriptional regulation of the ast operon in S. enterica serovar Typhimurium ( 23 ) , and IHF is involved in the regulation of some NR 54 I - and - dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>Strains carrying null alleles of the IHF structural genes , himD and himA , had severe defects in nitrate induction of the nar operon but were normal for nitrate induction of the coordinately regulated fdn operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Overall expression of the narXL genes was only slightly reduced in the absence of IHF ( Table 3 ) , suggesting that limitation for NarL and/or NarX is not responsible for the severe deficiency in nar operon induction by nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>All four , as has been found for many anaerobic systems ( 38 ) , are regulated by global and specific control elements , and IHF is a strong positive effector for the specific factor in each operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>pspG is physically unlinked with the psp operon but is co-conserved 54 and co-regulated with the psp operon genes by , PspF , IHF , and PspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>It is striking that the psp operon and the pspG 54 promoters , which are physically unlinked , are both regulated by PspF , PspA , and IHF in exactly the same fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>However , the 2.3-fold induction ofpfl due to addition of pyruvate to cells grown anaerobically in rich media was completely absent in the IHF mutant ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Taken together , these data suggest that IHF is necessary for both pyruvate induction and the anaerobic induction mediated by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Taken together , these results suggest that IHF may alter anaerobic induction of pfl primarily by influencing how ArcA functions in this process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF also has a positive role in the anaerobic induction ofpfi , although this was not as obvious as was its function in pyruvate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>In addition to the role of IHF in pyruvate induction , the results in Fig. 5 also show that IHF is required for complete anaerobic regulation of pfl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF is required for pyruvate induction and ArcA-dependent anaerobic regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Both OxyR and IHF are required for suf induction in response to H2O2 To further characterize the suf transcriptional response to H2O2 and 2,2 cents - dipyridyl , we deﬁned the regulatory features of the suf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Therefore , IHF is required for the o ^ - but not the OxyR-dependent induction of dps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Nitrate induction of F ( narG-lacZ ) expression is strongly dependent on IHF , which binds to the narG operon control region ( 21 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>To test whether IHF is necessary for OxyR-dependent induction of the suf operon , we examined suf expression in strains lacking either himA or himD , which encode the subunits of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>These data indicated that the presence of IHF is required for the AtoC-mediated induction of the atoDAEB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>IHF is absolutely necessary for pyruvate-mediated induction of the pfl operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF is required for optimal induction of the aceBAK operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Regulation of the isc operon is mediated by IscR , whereas the suf operon requires OxyR and IHF for the response to oxidative-stress and Fur for induction by iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>From these results , we propose that the oxidant induction of the sufA operon is mediated through OxyR , IHF , plus an unidentiﬁed oxidant-responsive factor , and that the suf gene products are needed to Accepted 17 November , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>In addition to regulation by the hip gene products and possible regulation by IHF , the hip promoter is subject to inverse growth rate and , independently , inverse temperature control ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>In addition to this growth rate control , both promoters were negatively regulated by IHE This was shown by the increase of [ 5-galactosidase activities in ihfA or ihfB mutant strains and by reduced expression of each promoter under conditions of excess levels of IHF ( figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>However , in the presence of high salt ( LB supplemented with 0.3 M NaCl ) there is already a high proU expression ( Gowrishankar and Manna 1996 ) , and so there is no measurable difference between the wt and the himA ( − ) strain ( data not shown ) on the expression of the proU promoter suggesting that IHF is acting only when expression is down regulated by H-NS , in this case at low salt concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>When the resulting strains , S90CPX and DPB101PX ( Table S1 ) , were grown in glycerol-containing minimal medium in the presence of PA , the lack of IHF did not inﬂuence negatively the lacZ expression driven by the Px promoter ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>B , the addition of increasing concentrations ( 0 -- 400 nM ) of wild type PspF to the transcription assay in A. C , the addition of increasing concentrations ( 0 -- 200 nM ) of IHF to the transcription assay in A. DISCUSSION Our data show that expression of pIV in both E. coli and S. typhimurium results in the significant up-regulation of psp-ABCDE and pspG ( yjbO ) transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>70 In the presence of nitrate , PnarG shows reduced requirements for FNR-AR1 and AR3 To investigate the effect of the FNR-AR mutants on PnarG expression in the presence of NarL and IHF , b-galactosidase activity was assayed in strains grown anaerobically in the presence of 20 mM potassium nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Thus , in the presence of NarL and IHF , the requirements of FNR-AR1 and FNR-AR3 for PnarG expression appear to be less stringent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>As these growth conditions promote maximal activity of PnarG as a result of the combined function of NarL , IHF and FNR , these results suggest that the Accepted 30 August , 2000 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Thus , durin anaerobic-growth conditions IHF appears to act as an enhancer of narc-lac2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Thus , under some anaerobic-growth conditions Arr may compete with FNR and IHF for occupation of the ndh promoter and , by acting positively , Arr may counteract the negative effects of FNR and IHF to ` finetune ' ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>In this same report , FNR and IHF were reported to have roles as weak repressors under anaerobic conditions , and it was suggested that the effect of FNR may be indirect as a consequence of its role in regulating ArcA expression ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>FNR and IHF acted as weak repressors under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>Therefore , since the overall effect of IHF in fimA regulation and fimbriae expression is positive , and since both ihfA and fimA are induced upon hha deletion , we hypothesized that another part of the mechanism by which Hha represses biofilm formation is through the diminution of type I fimbriae through the repression of fimA , either directly or indirectly through the repression of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Thus , IHF effects a 2-fold overall increase in the level of expression of the ilvGMEDA operon when both IHF sites are occupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Thus , IHF is capable of regulating the expression of the ilvGMEDA operon in opposing manners ; it can activate transcription initiation of this operon from the ilvP , Z promoter 4-fold and increase the termination of this transcription at the downstream attenuator Z-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Thus , IHF is capable , at least under certain conditions , of regulating the expression of the ilvGMEDA operon in opposing independent manners ; IHF activates transcription of this operon 4-fold and can increase the termination of this transcription at the downstream attenuator site 2-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>However , these experiments do not differentiate between direct and indirect effects of IHF on this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>These results strongly suggest that IHF is a direct effector of the pfl operon and that the effects of IHF on pfl expression found in vivo are a direct consequence of the binding of IHF to the site characterized in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Because IHF has diverse effects on gene expression , and since these effects are usually not very strong , it will be important to prove by rigorous in vitro and / n vivo analysis that IHF Is a direct effector of a particular operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>We report here that the dependence ofpsp activation on IHF also varies with the inducing agent , as psp expression in response to high temperature and ethanol is more strongly stimulated by IHF than expression during osmotic shock .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>It is also not obvious why IHF is involved in regulation of the omp system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>It is also possible that , earlier in bacterial evolution , IHF was involved in ast operon expression and that ArgR preempted this function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>IHF is also a transcriptional-regulator of the ompB operon whose protein products , EnvZ and OmpR , are required for expression and osmoregulation of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Besides requiring sS and MlrA , csgD expression is also modulated by IHF and the two-component response-regulators OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>To date , nc conditions have been found that result in a decrease in the amount of IHF , which makes a physiological function for iHF in gene expression questionable ; so why do promoters like PG2 , Pe and pLI use IHF to enhance their activity ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Thus , not only does IHF stimulate both FimB and FimE recombination , as well as activate ﬁmbrial expression in phase ON cells , but it is now clear that it enhances ﬁmB expression too .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The results presented above show that IHF does not enhance the ability of NanR to activate ﬁmB expression in the wild-type background ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>It thus seems likely that the ability of IHF alone to activate ﬁmB expression is enhanced in the D3 mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , in contrast to the situation in the wild type , IHF also appears to enhance NanR activation of ﬁmB expression in the deletion mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>In contrast , IHF does not enhance the ability of NanR to activate ﬁmB expression in the wild-type background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>In contrast to the situation with NagC , IHF is not required for NanR to activate ﬁmB expression in the wildtype background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Sohanpal , B. , Friar , S. , Roobol , J. , Plumbridge , J. , and Blom-ﬁeld , I. ( 2007 ) Multiple co-regulatory elements and IHF are necessary for the control of ﬁmB expression in response to sialic acid and N-acetylglucosamine in E. coli-K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>However , if IHF has a positive effect on the expression of the wild-type pst operon , in the absence of IHF the expression of the pst genes will be diminished , a condition that partially mimics a constitutive mutant and will result in an increased activity of AP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Assuming that IHF is directly or indirectly involved in the negative regulation of pta and ackA , the increased levels of their expression in the ihf mutants could lead to elevated levels of acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Consistent with this data and the increased sensitivity of the pspG promoter to PspF in comparison to the pspA promoter ( Fig. 6B ) , PspG expression in an IHF mutant ( MC1068 ) is increased both before and after induction by pIV in vivo ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>However , in vivo the basal level of pspG expression in IHF mutants is increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Hence , the increased level of pspG expression in IHF mutants should be because of the high sensitivity of the pspG promoter to PspF activation and diminished negative regulation by PspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Results in Fig. 4 show that , in the Dlac narL narP ihfA strain , JCB3884 , pnir expression 1 is increased by both the p99G and the p146A substitutions , which disrupt the binding of IHF and Fis respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Data in Table 2 show that , as expected , expression from pnrf is increased when IHF I is inactivated , decreased when IHF III is inactivated , but largely unaltered when IHF II is inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Binding of IHF to IHF I and to IHF II has opposing effects , repressing and activating pnir expression , respectively ( 3 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>tion of narG-lac2 expression appears to require the concerted action of three transciptional activator proteins , NarL , Fnr , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>These results were consistent and show that the presence of IHF improves the expression of pstS , as judged by the bGal activity of the fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>These effects were both positive and negative , results that are consistent with the positive and negative influence of IHF on the expression of E. coli operons ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>These data strongly suggest that IHF is required to regulate expression from dmsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_397">
These results suggest that IHF activates yiaK-S expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></label>
<input id="act_similarity_397", type="checkbox">
<div class="ACT_similar">
<li>These results suggest that IHF activates aceBAK expression by binding to IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li></div>
<li>In vivo experiments showed that IHF is a negative effector of ompB expression in growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>IHF also has a direct negative effect on ompF and ompC ( 26 , 45 , 46 ) , and it is likely that IHF regulation of ompB contributes to the overall expression of these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>IHF is a direct negative effector of gene expression in a number of other systems ( 26 , 32 , 33 , 43 , 45 , 55 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>This results in a negative effect of IHF on gene expression [ 56 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>This shows that the effect of IHF on nar operon expression is specific for nitrate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Under aerobic conditions , a deletion in himA ( IHF subunit ~ ) resulted in a 60 % increase in sodA expression , while having no effect on induction by paraquat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>Thus , IHF appears to be a negative modulator of sodA expression , but has little effect on induction of sodA-lacZ by paraquat via the SoxRS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>3-Galactosidase activity was measured in the IHF mutant and in the parental strain grown under a variety of conditions that are known to alterpfl expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>There is no clear correlation between the conditions that activate via PspB and PspC , and the treatments that require IHF to induce high-level expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>In bacteria lacking IHF , psp expression is substantially reduced in response to high temperature and ethanol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>During osmotic shock in contrast , psp expression is only weakly stimulated by IHF , and IHF mutants can strongly induce the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>IHF stimulates psp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>IHF is required for strong psp expression during heat-shock and ethanol treatment , but only weakly stimulates psp induction by osmotic shock .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>Similarly , Fnr can function in mediating partial induction of expression in the absenoe of ArcA or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>IHF slightly repressed sodB expression independently of Fur regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>This is comparable to the increase found in chromosomal ompB expression in IHF mutants ( Fig. 7 ; Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>This result suggests that IHF activates aceBAK expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Perhaps high expression of ompB in IHF mutants leads to changed levels of EnvZ in relation to OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Thus , IHF may interact at many levels to negatively regulate the expression of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Inactivation of IHFb had little effect on aceB : : lacZ expression in this background , indicating that activation of aceBAK by IHF results largely from its effect on repression by IclR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Under inducing conditions , IHF activates aceB : : lacZ expression by opposing IclR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>IHF stimulates Pl-directed expression in minimal medium but this effect is reversed under conditions of P1 repression by pyrimidines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>It is interesting that an artificial increase of oniy one subunit of IHF can induce strong repression of IHF gene expression ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>IHF thus appears to be required for maximal expression as well as for establishment of full repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>In the present case a detailed model for the action of IHF on car expression can not be presented as yet ; however one thing is clear , IHF appears to be required for both maximal activity and for the establishment of full repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>To evaluate the relative efficiency of the excisive recombination on different substrates , as well as the contribution of the accessory proteins TorI and IHF , we ran the same kind of experiments as described above , except that they were followed by real-time PCR analysis as described under `` Experimental Procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>Protein purifications IntS , TorI and IHF proteins were overproduced and purified near homogeneity as described [ 26,59,60 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20949106" target="_blank">20949106</a>]<br></li>
<li>Equimolar DNA substrates were incubated with various combinations of recombination proteins ( IntS , TorI , and IHF ) for 1 h at 37 ° C , which constitute the optimal conditions for this reaction in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>In Vitro Integrative Recombination -- As for the excisive recombination assay , we used various DNA substrates and the purified IntS , TorI , and IHF proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>5 ' - end-labelled fragments were prepared using 7 `` 4 polynucleotide kinase and [ T-32p ] ATP with DNA previously dephosphorylated with calf intestinal phosphatase , and purified Binding of IHF to its target site in the carAB operon was detected by band-shift gel electrophoresis based on the method of Fried and Crothers [ 39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Thus far , three proteins have been identified with specific functions and binding sites in oriC ( i ) DnaA , the initiator protein , which binds four 9-mers and then interacts with a contiguous set of three 13-mersto open the duplex ; ( ii ) the IciA protein which binds these 13-mers to prevent DnaA protein from opening this region and thus inhibits initiation ; and ( iii ) IHF , which binds a sequence that facilitates the bending of oriC around an aggregate of DnaA proteins ; HU protein serves the same function as IHF buw t ithout an identified binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>Several factors have been shown to influence the ability of DnaA protein to open the strands ( 3-6 ) : ( i ) binding of ATP or ADP to DnaA ; ( ii ) the level of negative superhelicity ; ( iii ) the temperature ; ( iv ) the presence of structure-modifying proteins ( e.g. HU and IHF ) ; ( v ) thepresence of theinhibitory IciA protein ; and ( vi ) transcriptional activationI l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>Assisted by protein HU or IHF , cooperative binding of 20 -- 40 monomers of ATP-bound DnaA protein per oriC stimulates duplex opening at the AT-rich 13-mer repeats in oriC , where several replication proteins are loaded to initiate DNA replication ( Kornberg & Baker , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160809" target="_blank">11160809</a>]<br></li>
<li>Binding of ArcA to eitber of tbese sites could provide a plausible explanation for tbe ArcA-dependent repression of lacZ expression in the absence of IHF , Clearly , the inability of IHF to-bind to this construct will cause a significant alteration of the DNA topology at the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>Since IHF binds to the DNA via the minor groove ( 21 ) and NsrR is likely to-bind through the major groove , the simplest model suggests that both proteins bind simultaneously and repress expression from the nrf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Finally , although bending of promoter DNA by IHF is generally suggested to be critical to its role in gene expression , this event has as yet not been shown to be essential for the activity of IHF in any operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>These studies suggest that the IHF protein performs a DNA bending function at the narG promoter to allow nitrate-dependent activation by the NarL regulatory protein , and second , it enhances the Fnr-dependent expression from the narG promoter under anaerobic cell growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>H159L Because the expression on linear IC-55 in the absence of IHF was undetectable , the dramatic activation factor of IHF-mediated DNA bending on IC-55 can not be calculated ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Apparently , IHF binding to narK regulatory DNA is required for full activation of narK expression in response to anaerobiosis and to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>IHF and Fis proteins are also required for full activation of narK expression , and their roles in DNA bending are discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>Effect of upstream DNA , IHF , and Fis on narK-kacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>No significant effect of IHF on a l d A expression was detected under our conditions , although the possibility of its widely accepted role as stabilizer of DNA conformation remains open ( Nash & Granston , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>However , the cryptic bgf operon , whose expression was reported to be regulated through topological changes of cellular DNA , could not be activated in cells depleted of HU or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>This result suggests that , at high levels of chromosomal DNA supercoiling required for maximal IHF activation , the activating effects of other nonspecific in vivo chromosomal organizer proteins , such as HU or HN-S , might become apparent ( 24 -- 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>( 12950 ; A streptavidin ) ArgR , 16863 Lrp , 18756 IhfB , IhfA , 10520 11223 Lrp , 18756 B Crp , 23509 A second possibility , which we did not test , is that the Crp-induced bend in the DNA [ 23 ] opposes that generated by IHF , preventing proper activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Furthermore , the observation that in the absence of IHF , transcriptional activity did not continue to increase beyond a superhelical DNA density of s 5 20.09 ( Fig. 4 ) suggests that the activation role of IHF can not be replaced simply by increasing the superhelical density of the DNA template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>It was proposed that when the spacing between the UAS and a54 recognition sequence is altered , IHF may inhibit transcriptional activation by wrapping the DNA such that a loop of different size or orientation can not form ( 30,54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>A change of any one of these three sites to the opposite face of the DNA helix does not affect the activation of transcription in the absence of IHF , but results in the powerful inhibition of this activation by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>No proteins RNAP NA G C T A Discussion IHF R P+C rp Role of IHF in mediating activation of gltBD by Lrp IHF , a DNA-bending protein [ 30 ] , is required for activation of gltBDF transcription by Lrp [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>When supercoiled templates were used , the activation effect of DNA bending was about 1.6-fold for CRP and fourfold for IHF ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>We conclude , on the contrary , that transcriptional activation by IHF is due to bending of the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>On supercoiled DNA , IHF does not play an essenhial role , but enhances the activation of transcription by NR,-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>I ) i i -RCB- -RCB- li i Activation of transcription by IHF In order to assess the contribution of DNA bending to transcriptional activation by CRP , we replaced CRP by the heterologous DNA-bending protein IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>If IHF can activate transcription by bending the DNA it is quite likely that DNA-bending by CRP has the same effect , namely transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_453">
When IHF interacts with the target binding sites in the lambda attP region , it induces a severe bend in the DNA helix estimated to be greater than 140 '' ( Robertson & Nash , 1988 ; Thompson & Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></label>
<input id="act_similarity_453", type="checkbox">
<div class="ACT_similar">
<li>( c ) IHF-mediated activation of transcription from the ilvP $ promoter When IHF interacts with its target binding sites in the lambda attP region , it induces a severe bend in the DNA helix estimated to be greater than 140 ( Robertson & Nash , 1988 ; Thompson & Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li></div>
<li>In contrast to HU and H-NS , FIS and IHF are site-specific binding proteins which , upon interaction with their respective recognition sites , induce a significant bend in the DNA with an estimated angle of 90 for Fis and 140 '' for IHF ( Thompson and Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>For IHF , the binding of only one dimer to its recognition site is sufficient to induce a strong curvature , which was estimated to be - 1400 ( Thompson and Landy , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>DNA bending analysis Every one of the previously identiﬁed DNA-binding proteins that bind to the carAB control region ( ArgR , PurR , IHF , PepA ) induces a pronounced bending ( 97 -- 144 ) or even wrapping ( ± 260 wrapping around hexameric PepA ) of the operator ( 16,18 , and unpublished observations from this laboratory ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472642" target="_blank">20472642</a>]<br></li>
<li>While the reason for these unusual cleavages is unclear , it has been supposed that the altered DNA conformation wrought by the binding of IHF , or the IHFmolecule itself , could catalyze the transfer of a methyl group to a phosphodiester bond adjacent to these residues , rendering them susceptible to cleavage by the basic conditions of the piperidine-mediated , ring-cleavage reaction ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>We were unable to detect IHF binding to this mutated ﬁs P region in gel mobility shift assays , even when the IHF concentration was increased 64-fold over that required to detect binding with the wild-type DNA template ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>In control experiments , it was found that prior methylation of the DNA did not affect the affinity of NagC for binding to the operator region in the presence or absence of IHF ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , NagC and IHF binding to the DNA in vitro does not inhibit NanR binding to ONR or vice versa ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>This scenari seems unlikely , however , since the IHF-mediated activation is maintained when the DNA sequences upstream from the IHF binding site are replaced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>This particular arrangement strongly suggests a model where transcriptional activation is accomplished via the architectural function of IHF factor that is known to induce a sharp bend in the DNA upon binding to its site and thus facilitating long-range interactions between RNAP and FruR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512708" target="_blank">9512708</a>]<br></li>
<li>As the IHF-binding site does not overlap any of these promoters , and as the inhibition does not require the presence of other proteins besides IHF , the effect of IHF must be mediated through conformational changes in the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Both purified HimA and HimD proteins appear to-bind specifically to the ihf site , inducing a comparable bend in the DNA as IHF does , suggesting that they are binding as homodimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>We can thus quantify the interaction of IHF with its binding sites by measuring the occupancy of these sites ; ( iii ) the in vitro and in vivo pro ® les , in the absence of IHF , show differences in photoreactivity at certain residues , re ¯ ecting the fact that the DNA is in a different environment in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>DNA , this repression should be relieved in the absence Because of the large number of Lrp-binding sites in the of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>Because the effect is observed in DH10B , which does not carry prpR , the negative regulation might be due to a secondary DNA structure formed by binding of IHF to the region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Although the mechanism of action of this protein is not clearly understood , in many cases IHF appears to act as an accessory factor to influence the properties of other DNA binding proteins ( reviewed in Friedman 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>This type of regulation is effected by small DNA-binding proteins like IHF Ž integration host f actor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Compaction of single DNA molecules induced by binding of integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Ali , B.M. , Amit , R. , Braslavsky , I. , Oppenheim , A.B. , Gileadi , O. , and Stavans , J. ( 2001 ) Compaction of single DNA molecules induced by binding of integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Shimizu , M. , Miyake , M. , Kanke , F. , Matsumoto , U. , and Shindo , H. ( 1995 ) Characterization of the binding of HU and IHF , homologous histone-like proteins of Escherichia coli , to curved and uncurved DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Hence , binding of IHF to the lacZ and ptsG promoters is unspeciﬁc , which is in line with previous data reporting that IHF binds DNA with lower afﬁnity also in sequence-independent manner ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>However , unlike other members of this family which have no or little sequence specificity for DNA binding , IHF binds to the 5 ' - AAT-CAAN4TTA / G-3 ' consensus sequence ; flanking sequences contribute significantly to this binding [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Nevertheless , IHF and HU-DNA complexes are quite different ; binding of a single IHF molecule results in strong bending of the DNA , whereas adjacent binding of several molecules is required for HU-induced bending [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>It is possible that IHF binding helps to stabilize those CysB ± DNA ± RNA polymerase complexes which are stimulated by N-acetyl-L-serine ( inducer ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>In contrast to HU , IHF binding to DNA is strongly sequencespecific .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Binding of HU is sequence unspeci cents c but , as is the case with IHF [ 7 ] , binding to DNA probably involves its two arms being inserted into the minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>dC ) ( Promega ) at 5 mg/ml was added to inhibit non-specific binding of IHF to the DNA fragments ( Fried & Crothers , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>Poly ( dI - dC ) ( Promega Biotec , Madison , Wis. ) at 9 , g/ml was added to inhibit nonspecific binding of IHF to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Perhaps the bending of the DNA by IHF can enhance the binding of Fnr to the DNA or a putative interaction of Fnr with NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Upon IHF binding , a conformational change is introduced in the DNA that brings an upstrearmegion nearby the RNApolym ¬ YAANNNNTGAT ~ - Nash , '' '' '' `` I FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Separation of free and bound DNA was performed on 5 % and 6 % polyacrylamide gels for PepA and IHF binding , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>Lane 1 , free DNA ; lanes 2 through 4 and 7 through 9 , 50 , 100 , and 200 ng of CAP , respectively ; lane 5 , 100 ng of CAP plus 20 ng of IHF ; lanes 6 and 10 , 20 ng of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Lane 2 , free DNA ; lane 3 , CAP alone ; lane 4 , CAP plus cAMP ; lane 5 , CAP plus cAMP plus IHF ; lane 6 , IHF alone ; lane 7 , IHF plus CAP ; lane 8 , IHF plus cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Binding of IHF and bending the DNA results in a re-orientation of the planes of consecutive bases thus increasing or decreasing the probability of an intramolecular photoreaction such as the formation of a pyrimidine dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Plasmid DNA incubated with increasing amounts of IHF was irradiated in vitro and analyzed by primer extension ( lanes j to p ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>c-32P end-labelled W8-W9 DNA was incubated with increasing amounts of IHF protein , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>A and B , concentrations of purified IHF are indicated in mg/ml ; FP shows free probe DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10506196" target="_blank">10506196</a>]<br></li>
<li>IHF ÿ DNA where DNA * is the radiolabeled oligonucleotide , DNA is the corresponding un-labeled oligonucleotide , and IHF is the concentration of active IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Thus , we conclude that IHF forms different IHF-DNA complexes when it binds to the EPEC and Salmonella promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>How IHF acts to influence these events is not known , but it has been suggested that the ability of IHF to bend the DNA to which it binds is essential to its mode of action in these systems ( 18 , 35 , 40 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>It was thus suggested that IHF can bend DNA when it binds to DNA nonspeciﬁcally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>IHF binds to DNA in a sequence-speciﬁc manner , inducing a bend in the DNA ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Although the exact mechanism of action of this multifunctional protein is not completely understood , IHF has been shown to bend the DNA to which it binds ( 24 , 33 , 44 , 47 , 51 , 53 , 57 ) and in at least one case this property was found to be critical to its biological function ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>In many of these oases IHF was found to bend the DNA to which it binds , and in the case of X integrative recombination the ability of IHF to bend DNA was shown to be essential for one aspect of IHF function -LCB- Goodman and Nash .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Only when CAP and IHF are simultaneously bound to their respective sites do they bend tdcP DNA into a new conformation and act in concert to activate tdcP ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>It is anticipated that large differences in the local structure of the DNA and its degree of supercoiling would occur when IHF binds to all three sites ( Fig. 7F ) relative to when it binds only IHF III ( Fig. 7D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Signiﬁcant IHF stimulation of ﬁs P occurs in ﬁs cells without the need for speciﬁc DNA sequences upstream of 2135 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>For example , IHF can induce the bending of DNA to allow an activator bound upstream to interact with downstream -- bound σ 54-RNA polymerase [ 17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Thus , the interaction of IHF and NarL-Phosphate with the upstream region appears to form a highly ordered protein-DNA complex which places the bound NarL-Phosphate in a position to interact with and activate the transcription initiation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>A heterologous protein ( IHF ) that bends the DNA to a similar degree as does CRP activates transcription when bound at sites equivalent to activating positions for CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>IHF as an architectural element in transcription stimulation IHF was initially found to be required for the site-specific recombination of phags k with the chromosome , its function in this process is to organize DNA and the Integiase protein into an ordered nucleo-protein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Also , IHF binds with equal affinities to its target site on linear and supercoiled DNA templates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>HU is a structural homologue of IHF , a protein that can induce bends into DNA of up to 160 ‡ [ 7 ] when bound to its recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>During the transition from exponential to stationary phase , when Fis levels are continually declining while elevated levels of the nucleoid-associated proteins Dps , IHF , HU , Hfq and H-NS are maintained ( Ali Azam et al. , 1999 ) , a global redistribution of chromosomal DNA branching can affect the accessibility of RNAP to numerous promoters to indirectly alter gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>In exponential phase E. coli cells , the order of intracellular concentration of these DNA-binding proteins is , in descending order , Fis , HU , StpA , H-NS and IHF ( Talukder et al. 1999 ) , among which HU , StpA , H-NS and IHF have been considered to participate in the condensation of genome DNA to form the compact nucleoid ( for reviews see Pettijohn 1982 , 1988 , 1996 ; Drlica & Rouviere-Yaniv 1987 ; Rouviere-Yaniv & Yaniv 1979 ; Murphy & Zimmerman 1997a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>For instance , the sequence speciﬁc DNA bending proteins IHF and Fis can relieve repression by H-NS at speciﬁc promoters ( Tippner et al. , 1994 ; van Ulsen et al. , 1996 ; Afﬂerbach et al. , 1998 ; Falconi et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>Nucleoid compaction and DNA-associated proteins DNA-associated proteins such as H-NS , HU , IHF , and Fis have often been suggested to be responsible for DNA compaction in E. coli , based among other things on their colocalization with DNA in vivo ( Azam et al. , 2000 ) , on their occurrence in isolated nucleoids that are prepared in the absence of high salt concentrations ( Murphy and Zimmerman , 1997 ; Pettijohn , 1996 ) , and on their ability to bend or compact DNA in vitro ( Dame , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>Also in accord , Brunetti et al. ( 2001 ) found that individual deletions of H-NS , HU , Fis , or IHF did not appear to change the large scale structure of the E. coli chromosome , based upon the lack of changes in the in vivo genomic fragmentation patterns by DNA gyrase in the presence of oxolinic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16384714" target="_blank">16384714</a>]<br></li>
<li>All 12 DNA-binding proteins , CbpA , CbpB ( or Rob ) , DnaA , Dps , Fis , Hfq , H-NS , HU , IciA , IHF , Lrp and StpA , were over-expressed using pCU60 , pMK19 , pSY567 , pDPS1 , pRJ1077 , pHFQ607 , pHOP11 , pLhupAhupB , pICS1 , pSA5hiphimA , pMWD1 and pT7stpA , respectively , and puri ® ed to apparent homogeneity as previously described ( Talukder & Ishihama 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>These proteins , among which HU , H-NS , FIS ( factor of inversion stimulation ) and IHF ( integration tiosf factor ) are the best characterized , are considered to be histone-like as they are small , relatively abundant and able to bend or wrap the DNA ( reviewed by Schmid , 1990 ) , HU and H-NS bind to DNA non-speoifically , although for Received 21 October , 1994 ; revised 5 December , 1994 : accepted 7 December , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Some of these proteins bind nonspeci ® cally to DNA , e.g. HU and H-NS , while others are site-speci ® c , e.g. FIS , integration host factor ( IHF ) , and Lrp ( Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>Keywords : Nucleoid ; Bacterial chromatin ; DNA compaction ; Nucleoid-associated protein ; H-NS ; HU ; IHF ; Fis 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>To evaluate this possibility , we asked whether two proteins known to bend DNA with which they interact , Integration Host Factor ( IHF ; reviewed in Friedman , 1988 ) and Fis , could bind specifically to rrnB P1 upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2209559" target="_blank">2209559</a>]<br></li>
<li>Some sequences produce an intrinsic bend in the DNA or bind proteins such as Fis , H-NS , HU , or IHF that bend the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Host-encoded proteins are also involved in this process , including IHF and Fis that bind and bend DNA in order to assist intasome formation [ 9,18 -- 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20949106" target="_blank">20949106</a>]<br></li>
<li>172 , No. 9 Integration Host Factor Is aNegative Effector of In Vivo and In Vitro Expression of ompC in Escherichia coli LIN HUANG , PING TSUI , AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , Ne Received 24 April 1990/Accepted 15 June 1990 w York 11794-5215 Integration host factor ( IHF ) of Escherichia coli is a DNA-binding protein involved in gene expression and other cellular functions in E. coli and some of its bacteriophages and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>An interesting comparison can be made with the IHF protein , which is also involved in different types of cellular processes , including regulation , recombination and replication and which is also known to induce DNA bending ( for a review see Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1640456" target="_blank">1640456</a>]<br></li>
<li>Introduction Integration host factor ( IHF ) from Escherichia coli is a small , DNA-binding and bending heterodimeric protein involved in numerous bacterial processes including phage DNA packaging and partition , DNA replication , recombination , transposition , and gene expression ( Friedman , 1988 ; Freundlich et al. , 1992 ; Goosen and Van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>Since its initial identification as a host protein necessary for lambda site specific integration , IHF has been demonstrated to be involved in numerous bacterial processes including phage DNA packaging and partition , DNA replication , recombination , transposition , and gene expression ( for reviews see [ 7 , 11 , 12 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Subsequently , it was found to be engaged in other site-specific DNA recombination processes ( Ball & Johnson , 1991 ; Thompson et al. , 1987 ; Weinreich & Reznikoff , 1992 ) , to alter DNA supercoiling ( Schneider et al. , 1997 , 2001 ; Skoko et al. , 2005 ; Travers et al. , 2001 ) , to influence the initiation of DNA replication Abbreviations : CRP , cyclic AMP receptor protein ; H-NS , histone-like nucleoid structuring protein ; IHF , integration host factor ; qRT-PCR , reverse transcriptase quantitative PCR ; RNAP , RNA polymerase ; WMW , Wilcoxon -- Mann -- Whitney ; WT , wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The inversion of ﬁmS requires a site-speciﬁc recombination , catalysed by recombinases FimB ( inversion in either direction ) and FimE ( inversion from ON-to-OFF predominately ) and is controlled by DNA bending proteins IHF ( integration host factor ) and Lrp ( leucine-responsive regulatory protein ) ( Klemm , 1986 ; Dorman and Higgins , 1987 ; Eisenstein et al. , 1987 ; McClain et al. , 1991 ; 1993 ; Blomﬁeld et al. , 1993 ; Gally et al. , 1994 ; 1996 ; Blomﬁeld et al. , 1997 ; Roesch and Blomﬁeld , 1998 ; Lahooti et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Nitrate induction of the narGHJl operon not only requires activated NarL , but also an accessory DNA binding protein , integration host factor , IHF ( Rabin et al. 1992 ; Schr6der et al. 1993 ) which has been shown to-bind to specific DNA recognition sites and mediates the formation of DNA bending ( Freundlich et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>To do that , we used the cAMP receptor protein ( CRP , bends DNA by about 901 Lawson et al. , 2004 ) and the integration host factor ( IHF , bends DNA by 140 -- 1601 Rice et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>For example , both integration host factor ( IHF ) and the cAMP-catabolite gene activator protein ( CAP ) complex bind to specific DNA sequences in the tdc promoter and act in concert to positively regulate the tdc genes ( Wu and Datta 1992 ; Wu et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>This view derived primarily from the fact that Lrp tends to activate some operons involved in biosynthesis and repress others involved in catabolism .1 Another view advanced by D'Ari et al. 13 is that Lrp is primarily a DNA-organizing protein , much like HNS , IHF , and HU , which also affect gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>When IHF binds to its speciﬁc sites , it causes a large distortion of the DNA where the double helix bends up to 180 degrees and wraps around the protein ( Rice et al. , 1996 ; Travers , 1997 ; Lynch et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Unlike HU protein , which binds to DNA in a nonspecific manner , IHF binds to a specific DNA consensus sequence , 5 ' - WATCAANNNNTTR-3 ' ( where W = A or T , R = A or G , N = any base ) ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>IHF is a heterodimer of J & rr .20,000 ( Nash & Robertson , 1981 ) with sequence homology to the histone-like protein HU , but unlike HU , binds to a specific DNA sequence ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Unlike other , structurally related histone-like proteins , IHF is a sequence-specific DNA binding protein , which binds to the 5 ' - AATCAAN4TTA/G -3 ' consensus sequence ( Goodrich et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>In addition , ﬁs transcription is positively regulated by the DNA-bending protein , IHF , and is enhanced by increases in negative supercoiling ( Pratt et al. , 1997 ; Schneider et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Architectural elements , such as the IHF and HU proteins ( Dixon and Kornberg , 1984 ; Skarstad et al. , 1990 ; Hwang and Kornberg , 1992 ) , negative DNA supercoiling and transcriptional activation ( Baker and Kornberg , 1988 ; Skarstad et al. , 1990 ) all stimulate the event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931356" target="_blank">10931356</a>]<br></li>
<li>Synthetic curved DNA can functionally replace IHF - or CRP-induced bending , demonstrating that bending is not a mere side effect of DNA - protein complex formation , but can be of functional significance in biological processes ( for a review , see Bracco et al. , 1989 ; Goodman and Nash , 1989 ; Snyder et al. , 1989 ; Travers , 1989 ; Gartenberg and Crothers , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>CRP and IHF have been studied in detail and shown to bend DNA by 90 and - 1400 , respectively ( Wu and Crothers , 1984 ; Liu-Johnson et al. , 1986 ; Robertson and Nash , 1988 ; Thompson and Landy , 1988 ; Schultz et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>These observations have led to the suggestion that the ability of IHF to change DNA conformation may be central to the mechanism of action of this multifunctional protein ( Prentki et al. 1987 ; Robertson and Nash 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>IHF has the ability to bend DNA to a similar degree as does CRP ( Kosturko et al. , 1988 ) and thus allowed us to analyse the respective contributions of protein-protein contacts and DNA curvature to the mechanism of transcriptional activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The primary function of IHF appears to be architectural , since , where tested , IHF can be at least partly replaced by heterologous DNA-bending proteins or by intrinsically bent DNA ( Goodman et al. 1992 ; Molina-Lopez et al. 1994 ; Perez-Martin et al. 1994 ; Segall et al. 1994 ; Parekh and Hat ® eld 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>The mechanism of action of IHF is not fully understood , but it is well known that IHF bends the DNA by more than 140 '' upon ibinding ( Kosturko et al. , 1989 ; Prentki et al. , 1987 ; Stenzel et al. , 1987 ; Robertson & Nash , 1988 ; Thompson & Landy , 1988 ; Tsui & Freundlich , 1990 ) It is , therefore , possible that t ' he primary role of IHF in lambda integration and other processes is to bend the DNA into a functional-conformation ( Goodman & Nash , 1989 ; Snyder et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>The effect of IHF can be attributed to its ability to bend the DNA ( Goodman & Nash , 1989 ; Kosturko et al. , 1989 ; Snyder et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_542">
c Fold activation by IHF was obtained for each plasmid by dividing the bgalactosidase activity in MC4100 cells by that in HP4110 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></label>
<input id="act_similarity_542", type="checkbox">
<div class="ACT_similar">
<li>d Fold IHF activation was obtained for each plasmid by dividing the b-galactosidase activity in MC4100 cells by that in HP4110 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li></div>
<li>that the ability of GcvR to repress is dependent on Abbreviations : GCV , glycine cleavage ; GM , glucose minimal ; IHF , in - GcvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>Thus , activation by IHF appears to be position dependent , since it is sensitive to distance and possibly helical phasing relative to ﬁs P. Stimulation of ﬁs P by IHF may resemble the situation with the bacteriophage Mu Pe and bacteriophage lpL1 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>This requirement of IHF for transcriptional activation had been noted for several o - ' - dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Thus , on pacs236 crp II , the presence of IHF makes AR2 a positive component of CRP-dependent activation above that provided by AR1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>This substantial increase supports the hypothesis that IHF III contributes to IHF-dependent anti-activation of CRP-dependent activation of acsP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Intriguingly , IHF also enhanced CRP-dependent activation in both the pacs236 and full-length constructs ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>When both proteins were present ( Fig. 6C , lanes 8 -- 13 ) a distinct IHF -- NarL -- DNA complex was observed , indicating that NarL and IHF can bind to pnir together .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The third minor Fis -- NarL -- DNA complex , seen in Fig. 6B ( lanes 10 -- 13 ) , is likely NarL , FNR and Fis can bind to pnir together , but NarL interferes with the binding of IHF Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>This indicates that NarL displaces IHF from IHF I , and that NarL can bind to pnir even when excess IHF is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Thus , it is possible that NarL and NarP can displace IHF and that the consequence of this is to reposition pnir upstream sequences ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>If this were the case , IHF would not be expected to play a physiological role as a transcriptional-regulator of ﬁs P , unless other proteins ( such as Fis ) regulate the binding activity of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>IHF could have a physiological role in transcription regulation if other proteins that do respond to environmental signals influence the binding or activity of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Fis , IHF and FNR can bind together to the nir promoter Further DNase I footprinting experiments were performed to investigate the binding of puriﬁed Fis , IHF and FNR protein in different combinations at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Thus , we selected the p22C and p42T substitutions for further study and we used electromobility shift assays to measure the effects of these changes on the binding of Fis , FNR and IHF to the pnrf97 fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>To investigate the mechanism by which NarL counteracts IHF - and Fis-mediated repression at pnir , we used gel retardation assays and DNase I footprinting to examine the binding of NarL , FNR , Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>This has been demonstrated for the promoters of virF [ 55 ^ 57 ] , hns [ 58 ] , the early promoter of phage Mu [ 59 ] and the ribosomal RNA P1 promoters [ 60,61 ] , in which relief of repression occurs by the binding of Fis or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>This suggests that , when the IHF I and Fis I sites are disrupted , binding of IHF to IHF II hardly activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The simplest explanation for this is that IHF binding to the IHF II site decreases the repression resulting from IHF and Fis binding at the other sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Indeed , this appears to be the case for the nir promoter , in which binding of IHF to the IHF II site decreases the repression of nir mediated by IHF and Fis binding at other sites ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>However , in each case tested , Fis-dependent repression is less efﬁcient when one or both of the DNA sites for IHF is inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>However , in the presence of FNR the altered promoters were about three-fold more active than wildtype , which may in part be due to weaker anaerobic repression by Fis , IHF and FNR , because position -66 forms part of the Fis I1 and IHF I11 sites , and the -55 substitution removes a key FNR : DNA interaction at the FNR I site ( Figs 1 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Figure 2 illustrates such an experiment , in which the pnrf97 fragment was incubated with RNAP in the presence of puriﬁed Fis , IHF , FNR and NarL , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , in the presence of all three proteins , Fis binds at its target at position -15 , but the FNR and IHF footprints remain incomplete .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , the ability of IHF to stimulate transcription is dependent on its position relative to ﬁs P. ﬁs expression pattern in the absence of ihf and Fis sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Thus , Fis can repress transcription in the absence of IHF at IHF I and IHF II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Thus , repressive effects of Fis and IHF appear to be additive to the repressive effects of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>IHF and OxyR both function as activators , while Fis and H-NS repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Interestingly , in the ihfA strain , Fis bound at position 2142 appears unable to repress pnir , suggesting that IHF may be involved in the repression of pnir by upstreambound Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>For example , Fis I-mediated repression is more effective when upstream IHF sites are intact , suggesting that IHF may help position Fis for optimal repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>This supports the proposal that NarL and NarP function by counteracting the repressive effects of IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Our results clearly show that H-NS represses pnir and that the action of H-NS is independent of the action of Fis , IHF and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , when repression by both Fis and IHF is disrupted , nitrite-triggered NarL or NarP is no longer required for optimal activation of pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , we conclude that IHF and Fis can repress pnir in the absence of H-NS ( Table 2 ) , and that repression by H-NS can not be reversed by the action of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>NarL counteracts repression by IHF but is unable to alter repression by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>NarL and NarP can relieve both IHF - and Fis-mediated repression , but are unable to counteract H-NS - mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Escherichia coli prereplication complex assembly is regulated by dynamic interplay among Fis , IHF and DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>Rather , a Fis -- IHF -- FNR -- pnir complex forms in which pnir activity is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_581">
NarL can bind simultaneously to pnir with FNR , Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></label>
<input id="act_similarity_581", type="checkbox">
<div class="ACT_similar">
<li>FNR , Fis and IHF can bind to pnir simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li></div>
<li>Moreover , IHF appears to be necessary for downregulation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Upstream IHF and Fis sites must be correctly positioned to regulate pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>NarL and NarP activate FNR-dependent transcription by counteracting the repressive effects of IHF and Fis ( Fig. 1B ) , and thus the role of IHF and Fis is to set the co-dependence of pnir on both FNR and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>By disrupting Fis I , IHF II , and IHF I using the p146A , p112G , and p99G substitutions , respectively ( Fig. 1A ) , we showed that FNR-dependent transcription at pnir is repressed by Fis I and IHF I and stimulated by IHF II ( 3 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>We suppose that , in response to nitrite or nitrate ions , NarL is activated by phosphorylation by the NarX and NarQ sensor kinases , and the activated NarL displaces IHF bound at IHF I and relieves the repression of FNR-dependent transcription initiation at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>IHF III activates FNR-dependent transcription at pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Previously , we demonstrated that NarL activates FNR-dependent transcription at pnrf by displacing IHF bound to the IHF I site ( 3 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>In an IHF - strain , anaerobic , FNR-dependent activation of expression still occurred but NarL-dependent nitrate regulation was abolished ( 32 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>However , in the absence of nitrate or nitrite ions , some nir operon expression can be measured , and our studies have shown that the nucleoid-asso-ciated proteins IHF and Fis play a key role in setting the level of this expression by modulating FNR-dependent activation of the nir promoter ( 3 , 5 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>It is especially intriguing that binding of IHF to IHF I and to IHF II at the nir promoter produces opposing effects , and it is the balance between these effects that sets the level of FNR-dependent expression in the absence of nitrate/nitrite-dependent activation via NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>This suggests that occupation of the IHF II site increases FNR-depen-dent activation of pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>In previous work ( 3 -- 6 ) , we showed that the DNA target for FNR at pnrf is centered at position 41.5 ( i.e. , between 41 and 42 bp upstream from the transcript start point ) , that anaerobically induced FNR-dependent activation of pnrf is downregulated by the binding of the nucleoid-associated protein IHF to a target ( IHF I ) centered at position 54 , and that this repression is relieved by the binding of either NarL or NarP to a DNA target centered at position 74.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>This activation is dependent on the face of the DNA helix , independent of IHF binding at other locations , and found only when NarL/NarP are not bound at position 74.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>CRP-dependent activation of acsP2 is modulated by binding of the nucleoid-associated proteins IHF and Fis to tandem sites , namely , IHF * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Under these conditions , IHF has both negative and positive effects on FNR-dependent activation due to its binding at the IHF I and IHF III sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Taken together , these results argue that IHF , binding near position 299 at pnir , does downregulate FNR-dependent activation of pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Surprisingly , IHF binding at the IHF II site increases FNR-dependent activation by decreasing the repression mediated by IHF and Fis binding at the other sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Fis binding at this site appears to assist IHF to repress FNR-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , although Fis and IHF repress FNR-dependent activation of pnir , they do not prevent FNR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Moreover , the repression of FNR-dependent activation by IHF can be reversed by NarL ( Fig. 2B , lanes 7 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The primary role of IHF , mediated by binding at the IHF I target , is also to downregulate FNR-dependent activation , but this can be reversed by NarP or NarL in response to nitrite or nitrate ions in the environment ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>If nitrate or nitrite ions are present , NarL is activated ( via NarX or NarQ ) and , upon binding to its target site , IHF is displaced and the repression of FNR-dependent activation is lifted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Note that our observation that NarL can increase FNR-dependent open complex formation in the absence of IHF argues that it , and perhaps NarP , behave as genuine co-activators at the nrf promoter as well as anti-repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , NarL and NarP protect FNR-dependent transcription at pnrf from both the negative and positive effects of IHF and appear to function like insulator proteins rather than conventional transcription activators , which stimulate promoter activity by recruitment of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Binding of NarL/NarP appears to insulate the nrf promoter from the effects of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>In previous work , we found that NarL or NarP activates the nir promoter by binding to a site centred at position -69.5 and counteracting the repressive effects of IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The binding of NarL or NarP to the 7-2-7 element , centred at position -74.5 , displaces IHF and activates the promoter provided that Fis is not present ( see text ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The binding of NarL or NarP to the 7-2-7 element , centred at position -69.5 , counteracts the effects of IHF and Fis , and activates the promoter provided that Cra is not present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Our observation that NarL could displace IHF from its site at the nrf promoter suggested that NarL/NarP activates transcription by counteracting IHF-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Once phosphorylated , NarL or NarP binds to the same target site centered at position 69.5 ( Fig. 1A ) and activates transcription by displacing IHF from IHF I , thereby counteracting the repression by IHF and Fis from upstream sites ( 3 , 25 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Our results show that open complex formation is completely dependent on FNR and is enhanced by NarL , but is repressed by IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The results of this study support the proposals made by Wu and colleagues , but show that the repressing factors are more complex than initially supposed : FNR-dependent activation is repressed by three nucleoid-associated factors , Fis , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Other results in Fig. 2A show that FNR-dependent activation of the nrf promoter is repressed by Fis or IHF proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>We suspect that this may account for the downregulation of FNR-dependent activation of the nrf promoter by IHF , but this is unproven .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Our results conﬁrm that Fis binds independently of FNR and IHF and we suppose that Fis blocks access of RNAP to the nrf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Thus , activation involving FNR , NarL and IHF is overridden by Fis when it binds to its target at position -15 and blocks access to the nrf promoter -10 element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Integration of three signals at the Escherichia coli nrf ole f Summary Expression from the Escherichia coli nrf operon promoter is activated by the anaerobically triggered transcription factor , FNR , and by the nitrate/nitrite ioncontrolled response-regulators , NarL or NarP , but is repressed by the IHF and Fis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Incontrast , the / pel promoter was repressed by both glucose ( via a CRP-independentmechanism ) and anaerobiosis ( mediated by ArcA ) , and activated by Fis , but it was not regulated by FNR , FruR , IHF or These observations support the view that transcription of the sucABCD genes is primarily initiated and regulated at the upstream sdh promoter , and that the Ipd promoter is independently co-regulated with Pdh ( primarily by ArcAmediated repression ) rather than with PsucDirect evidence for co-transcription of the entire sdhCDAksucABCD region from PHNw t as obtained by detecting a 10 kb transcript inm c and me mutants , but not inthe parental strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Another example is the promoter of the nrfA operon that is activated by FNR and repressed by Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Expressionfrom the suc promoter was repressed by IHF and partially activated by d8but it was not regulated by ArcA , FNR , CRP , FruR or Fis , and not repressed by glucose or anaerobiosis , indicating that the well-established catabolite and anaerobic repression of ODH synthesis is imposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>] When the amount of H-NS added was increased in the presence of a saturating amount of IHF , H-NS was able to-bind by replacing IHF sites ( Fig. 7a , b , lanes 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>On the other hand , the addition of increasing amounts of IHF in the presence of a high concentration of H-NS to saturate the CD1 and CD3 probes did not change the gel pattern ( Fig. 7a , b , lanes 11 -- 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Increasing concentrations of IHF exerted little effect ( Fig. 5 , lanes 8 -- 10 ) , suggesting that IHF and CRP can bind simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Increasing concentrations of IHF exerted no apparent effect upon the binding of FIS to its sites ( Fig. 5 , lanes 13 -- 15 ) and vice versa ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>To determine the putative role of CRP and IHF in paa regulation , we have used both genetic and biochemical approaches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>This positive and negative control by cyclic AMP-CRP is apparent in both the IHF + and IHF - strains ( Fig. 2A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Many of the transcriptional changes observed were brought about by global TFs such as ArcA , FNR , CRP , IHF , pdhR and fur in ATPase and NOX ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>Transcription of acs by cells grown in glucose minimal medium relative to optical density , consumption of carbon sources , excretion of acetate , and intracellular concentrations of FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>For acsP2 , the most important signals appear to be those represented by the concentrations of CRP , IHF and FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_632">
Reactions contained puriﬁed RNA polymerase , puriﬁed CRP and increasing concentrations of puriﬁed IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></label>
<input id="act_similarity_632", type="checkbox">
<div class="ACT_similar">
<li>Reactions contained puriﬁed RNA polymerase , puriﬁed CRP , puriﬁed IHF and increasing concentrations of puriﬁed FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li></div>
<li>As can be seen in lanes j to p of Figures 2 , 3 , and 4 , the band intensity increased with increasing concentrations of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>However , the overall proﬁle was very similar , and at the increasing concentrations , the IHF effect was diminished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>For example , in the case of the yjbE site , the 50 TAA 30 sequence , which corresponds to the second conserved part of the IHF consensus , reacts differently in the presence of IHF ; the photo-reactivity of the two external resi-dues increased slightly in the presence of IHF ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Excision can be done in the absence of IHF , and the presence of IHF led to a 1.2 - to 2.9-fold increase in TABLE 2 KplE1 substrates No protein 0.1 0.1 0.1 0.1 IntS 0.1 0.1 0.1 0.1 IntS TorI 9 3 70 7 67 8 2 2 IntS IHF 0.1 0.1 0.1 0.1 IntS TorI IHF 17 9 85 11 85 12 67 9 HK620 substrates No protein 0.1 0.1 0.1 0.1 IntS 0.1 0.1 0.1 0.1 IntS TorI 13 3 47 6 69 8 10 4 IntS IHF 0.5 0.4 0.5 0.2 0.1 0.1 IntS TorI IHF 22 8 74 8 86 12 57 9 a Slash symbols indicate linear substrates , whereas circles indicate plasmid-cloned substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>Fluorescently labelled DNA probe of the csgD promoter CD1 fragment ( 1 pmol ) was incubated with increasing concentrations of purified IHF ( lanes 1 and 2 , 0 pmol ; lane 3 , 2.5 pmol , lane 4 , 5.0 pmol ; lanes 5 and 6 , 10 pmol ) in the absence ( lane 1 , 6 ) or presence ( lanes 2 -- 5 ) of OmpR ( 80 pmol ) and subjected to DNase I footprinting assays as described in Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Approximately 0.1 pmol of the end-labelled TaqI-BsmI fragment was incubated in the absence or presence of increasing concentrations of IHF at 28 ° C in the same buffer used in the gel retardation assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>Negatively supercoiled plasmid pDHAwt ( 17 nM ) was incubated in the absence and in increasing concentrations of IHF ( 3.5 , 7 , 14 , 28 , and 56 nM ) in 17 , ul of buffer A at 37 ° C for 10 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Briefly , 32 nM of attL and attR linear substrates were incubated at 37uC for 1 h in the presence of constant concentrations of TorI and IHF ( 1.6 mM and 0.25 mM , respectively ) and increasing concentrations of IntS ( 0.02 to 6.7 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20949106" target="_blank">20949106</a>]<br></li>
<li>In the presence of purified IHF protein , in vitro transcriptional pausing of RNA polymerase at the leader-attenuator pause site is increased Z-fold and , concomitantly , a 2-fold increase in transcriptional termination at the attenuator is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>As previously reported , in vitro activation of transcription of mtr by TyrR protein and phenylalanine is observed on supercoiled templates in the presence of HU , IHF , or BaCl2 ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>( a ) In vitro 1.6 4.0 25.0 ND Activation factor transcription assays on supercoiled or linear templates with native Es54 in the absence or presence of cAMP receptor protein ( CRP ) H159L or integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>A clear shift in migration could be observed in the presence of the protein for all DNAs ( fig 3 ) indicating that IHF still binds to the various mutant DNAs and with about the same affinity as to the wild-type sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Figure 8 ( lanes 2 -- 5 and 6 -- 9 ) shows controls in which NarL and IHF were added to the pnir7150 fragment in the absence of a preincubated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>We used in vivo laser footprinting to compare the local conformation of proU promoters in the presence or absence of IHF or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Reactions included NagC at ﬁnal concentrations of 100 , 50 , 25 , 12.5 nM ( indicated by the horizontal triangle ) in the presence ( + ) or absence ( -- ) of 50 nM IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>2 and 1 indicate the absence and presence of 250 nM purified IHF , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>C , sche-matic representation of the RNA transcription products from reactions performed in the absence or presence of purified IHF protein ( 50 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>B , schematic representation of the transcripts obtained either inthe absence or presence of50 nM IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>A , autoradiogram of a urea-poly-acrylamide gel in the absence of IHF protein or in the presence of 25 or 50 nM IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>A , autoradiogram of a urea-polyacryl-amide gel in the absence of IHF protein or in the presence of 5 or 50 nM IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>( B ) The Wild-type DNA from pYW65 was incubated separately with various concentrations of purified IHF in the presence or absence of 50 ng of purified CAP plus 20 WM cAMP prior to gel electrophoresis and autoradiography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>The Me,S04-mediated methylation pattern of the nontranscribed DNA strand , in the presence and absence of IHF protein , is shown in Fig. 2B and summarized in Fig. 3 ( top DNA strand ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Autoradiograms displayingthe positions of deox-yribose sugar residues protected fromhydroxylradical cleavage on the nontranscribed ( A ) and transcribed ( B ) DNA strands , displayed by brackets , in the presence and absence of IHF protein are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Determining how ArcA , Fnr , IHF and RNA polymerase bind to their respective recognition sequences in the presence of the other proteins will obviously bring us a step further towards achieving this objective .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>This may not be so surprising , since stable complex formation could possibly require the simultaneous presence of the PepA and/or IHF proteins ; moreover , the necessity for a supercoiled template should be considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li>
<li>Protein-DNA complexes were formed by incubation of 0.2 Hg purified IHF , with end-labelled DNA fragments in the presence of a 100-fold excess sonicated calf thymus DNA as described [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Labelled DNA fragments containing the hlmO/tvp and the tilmA P4 promoters were incubated with increasing amounts of IHF in the presence of excess amounts of non-specific DNA and subjected to eleotrophoresis in a non-denaturing poiyacryiamide ge !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Titrations of a double-stranded oligonucleotide with increasing concentrations of IHF ( indicated at the top of each lane ) were performed in the presence of different concentrations of E. coli chromosomal DNA ( 0 , 15 , 30 , 45 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>A , Binding of IHF to the BIME-1 gyrB site in the presence of increasing amount of E. coli chromosomal non-speci ® c DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Binding of IHF to four speci ® c binding sites in the presence of increasing concentrations of E. coli chromosomal DNA : titration curve relating K as a function of DNA competitor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>However , the reaction appeared to reach a maximum efficiency in the presence of IHF , illustrating the positive effect of DNA bending in this process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>In the presence of both Lrp and IHF in the reaction mixture , we observed an additional band corresponding to the ternary complex , which was not present when either Lrp or IHF alone was used in the reactions ; at saturating concentrations of Lrp and IHF , most of the DNA migrated in this band .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>In the presence of 60 ng of IHF , very weak DNase I protection could be observed in the region from 288 to 237 on the top DNA strand and from 279 to 249 on the bottom strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Migration of the DNA fragment was retarded in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>In the presence of saturating amounts of Lrp and IHF , most of the DNA was bound to both IHF and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Nevertheless , the relative mobility pattern of the ternary DNA-IHF-CAP complex shown in Fig. 7 clearly reveals the presence of bent DNA , and the extent of bending appears to exhibited by the binary DNA-be different from the pattern to IHF complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Binding of IHF to the 208-bp fragment devoid of the IHF2 site ( from 2144 to 164 ) gave a more rapidly migrating specific complex C1 and additional complexes that were lost in the presence of an excess of unspecific DNA ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10506196" target="_blank">10506196</a>]<br></li>
<li>Formation of complexes between this DNA and NagC in the presence of IHF prevented methylation by Dam methylase of GATCNagC , as judged by cleavage with MboI producing a band of 64 bp ( PM-GATCNagC ; Fig. 5 , lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , when the 400 bp probe CD6 , including both the IHF-1 and the IHF-2 sites , was used , IHF -- H-NS -- DNA ternary complexes were detected ( data not shown ) , implying that IHF was not ejected even in the presence of high concentrations of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In the simultaneous presence of MlrA , OmpR and IHF , all three regulators bind to the same CD6 probe forming MlrA -- OmpR -- IHF -- DNA quaternary complexes ( Fig. 5c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In contrast to the situation with NagC and IHF , we where unable to demonstrate methylation protection of ONR in the presence of NanR in the assay described above ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>H159L As control , the expression of native glnAp2 was not affected by the presence of either CRP or IHF ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>In contrast , IHF exerted no substantial effect on the affinity of CRP to either CRP I or CRP II ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Under these conditions , IHF inhibits TOC formation by RNAP , which can be overcome by the presence of low concentrations of CRP ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>This concentration of CRP suffices to stabilize the open complex in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In contrast , the combined presence of IHF and 12 nM CRP resulted in the formation of virtually no open complex at acsP2A ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In the presence of both RNAP and 20 nM IHF ( Fig. 4 , lane 3 ) , a decrease in the amount of modiﬁcations at P2A occurred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In contrast , IHF at this higher concentration does not inhibit the binding of RNAP in the presence of CRP , even when the transcription factor is present at the lower concentration ( 12 nM ) tested ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>( i ) The presence of IHF on its three speciﬁc sites in the full-length pacs444 , or on the IHF III site of the truncated pacs236 , does not inhibit the formation of an open complex in vitro in either the presence or the absence of CRP ( Figs 4 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In either case , the presence of low-affinity sites for both IHF and CRP raises a much larger issue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Second , with IHF bound at its speciﬁc sites , additional protection was observed between the two CRP sites , but only in the presence of the intact AR2 surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>However , in the presence of both RNAP and CRP , IHF does mediate increased protection of the region between the two CRP sites ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>As we observed similar behaviour with pacs444 in either the absence or presence of IHF ( Fig. 5 ) , this effect of CRP at the acs promoter appears to be general .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Two pairs of reactions with no CRP are represented : the ﬁrst was performed in the absence of IHF , and the second was performed in its presence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Planar representation ( 8,9,40 ) shows the Me2S0m 4 ethylation and hydroxyl radical protection patterns in the ilvGMEDA leader in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Fig. 3 summarizes the positions of purine residues that were either protected from methylation or showed an enhanced methylation pattern in the presence of IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Three adenine residues at base pair positions +134 , +137 , and +138 are protected from methylation in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>However , in the presence of NarL , the protection of the IHF I site was lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>( b ) Gel shift assay in the presence of both IHF and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Open circles indicate protection against cleavage in the presence of IHF , and the ﬁlled circles indicate unchanged cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>One particularly striking difference , as an example , is the DNase cleavage at position -19 , that shows hypersensitivity in the presence of Lrp + IHF but not in the presence of Crp ( compare lane 7 of Fig. 3B to lane 7 of Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>RNAP only yields a defined DNAse I footprint under two tested conditions : in the presence of both Lrp and IHF , or in the presence of Crp-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We also show that RNAP binds stably to the gltBDF promoter in the presence of Lrp and IHF , or in the presence of Crp-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>A. DNase I protection assay of gltBDF promoter region in the presence of both Lrp and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Although the in vivo pro ® les superimposed well onto those obtained in vitro , we noticed that , independently of the presence of IHF , the intensity of several signals was strikingly different in the two environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Under these conditions the levels of approximately 15-20 % of the poly-peptides were significantly different ( as judged by the intensity or presence of spots ) in the IHF mutant relative to the parent ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>In the absence of a functional IHF protomer , disruption of the DNA bend between the tandem promoters results in a sevenfold decrease in the level of galactokinase expression ( compare st ' rains IH140 and IH220 , Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>Down-regulation of the transcription from ptac in the absence of IHF , together with IHF - and SeqA-mediated effects on pR , suggest that DNA topology can not be underestimated when assessing in vivo promoters ' activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>( All plasmid constructs were maintained in the strain CH1483 , as the absence of IHF was found to prevent random DNA rearrangements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10781567" target="_blank">10781567</a>]<br></li>
<li>However , although the in vivo DNA supercoiling of the bacterial chromosome is slightly higher in a wild type strain in the presence of IHF than in its absence ( Table II ; Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>However , the reaction was still possible in the absence of IHF but with a very low efficiency ( Table 3 , compare IntS to IntS IHF rows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>In the absence of IHF ( himA , line 8 ) the activities in minimal and pyrimidine supplemented medium are slightly increased but reprczsibility is still insig-nificant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>H-NS does not directly a ¡ ect switching frequency In order to test whether H-NS plays a direct role in inversion of the phase switch , for example by blocking access of proteins required for switching such as IHF or Lrp , in vitro switching experiments were carried out in the absence or presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>The probability of thymidine formation increased significantly in the absence of IHF in comparison to the wt , while in the absence of H-NS the opposite effect was observed ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Primer extension assays showed that P1 induction by hydrogen peroxide occurs in the absence of Lrp , IHF and H-NS ( Fig. 4A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>The expression of the aceBAK operon , which codes for the key enzymes of the glyoxylate cycle , isocitrate lyase and malate synthase , and for isocitrate dehydrogenase kinase/phosphatase , is activated by FruR and IHF and is repressed by IclR , which in turn is activated by FadR ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>The expression of the prpBCDE operon of enteric bacteria , which codes for 2-meth-ylcitrate synthase , 2-methylcitrate dehydratase , and 2-methyl-citrate lyase , which are key enzymes of the methylcitrate cycle , and propionyl-CoA synthetase , depends on PrpR , IHF , and NtrA ( 10 , 27 , 51 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>The data in Fig. 4 demonstrate that all three promoters were still induced by anaerobiosis and that the IHF III site activated anaerobic expression in the pnrf-nir fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>The only noticeable effect of the combination of mutations resulting in a net increase of the interval IHF target site - GATC sequence by I bp is a roughly two-fold increase of PI expression in all growth conditions and genetic contexts analysed in this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>The effect IHF exerts on th ~ expression of a gene when bound to its control region may either be positive or negative and a number of different mechanisms appear to be involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>' The results of the in uitro transcriptional analyses presented in this report demonstrate that IHF , when bound simultaneously at both of the target sites , regulates the expression of this operon by independent interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The ompC gene , which was underexpressed in this condition ( Table 3 ) , has a complex regulation including protein regulators ( OmpR , CpxR , Lrp , IHF and EnvY ) and small regulatory RNAs ( micC , rseX and rybB ) that respond to different signals [ 42-46 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The data presented in Table I1 show that the interaction of IHF atthe leader-attenuator target sitedoes , in fact , cause a 2-fold decrease ignalactokinasespecificactivity ( comparethe galactokinase specific activities in strains IH1020 and IH109 in Table 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The low-level activity might be due to P1 activation by the decreased reiterative transcription of U clusters at P1 initiation site ( Han and Turnbough , 1998 ) owing to the decrease in UTP level or a change in the intracellular level of other four transcription factors , IHF , PepA ( CarP ) , PurR and ArgR ( see Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>The binding of IHF to its target , situated around position 304 , far upstream of the start point of P1 transcription ( Fig. 1 ) , mainly reinforces the pyrimidine-speciﬁc repression ( 8 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>IHF binds dmsA immediately upstream of ModE The vast majority of repressors bind their cognate promo-ters in a region that spans from position ¹ 70 to þ 20 relative to the start of transcription ( Gralla and Collado-Vides , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The observed influence of pyrimidines on the binding of IHF is rather unexpected ; we have obtained evidence ( to be published elsewhere ) that differences observed in the binding capacity of particular extracts are actually due to the presence of different amounts of IHF rather than to effects of cofactors or protein complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The presence of several binding sites for IHF in the intergenic region between the ssu operon and ycbQ suggested that IHF could also be involved in regulation of ssu , but we were unable to demonstrate it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10506196" target="_blank">10506196</a>]<br></li>
<li>The binding of OmpR to these regions may be facilitated by the presence of the bent DNA in the region between - 101 and -71 and by IHF binding at one or both of its sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>The repeatedly observed pyrimidine-enhanced protein binding to the upstream carAB region can at present best be explained by increased production of this DNA binding protein ( clearly identified below as the E. coli IHF ) in the presence of uracil in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The binding of IHF to speci ® c ihf sites was detected , in primer extension experiments of irradiated complexes , by an increase in the band intensity of the guanine residue in the consensus pentamer sequence TTGAW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The low affinity of iHF binding suggests a simple mechanism for Ihe negative autoregulation of IHF , in whiclT IHF acts as a repressor of transcripfion initiation only when its level in the celi is high .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>The replacement of TIO and Tll by two G residues , though resulting in a 50-fold decrease of the affinity , did no completely abolish the IHF specific binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>The IHF binding site , 5 ' - AAACAACAATTTA-3 ' , in upstream activating sequence ( UAS ) 1 is located between base pairs -82 and -96 ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>The IHF binding sites , IHF and IHF , were inactivated to determine a b whether they contributed to the activation of aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>The binding of IHF by itself also does not stimulate the isomerization to the open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The binding of IHF distorts this dinucleotide and further enhances the reactivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The binding of IHF to the csgD promoterproximal site IHF-2 should activate its transcription through protein -- protein contacts with either aCTD or sCTD ( Ishihama , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The stimulation that is observed in the absence of IHF binding under repression conditions is however more pronounced when the IHF protein itself is mutated ( himA ) than in the absence of the cognate binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The failure of HimA to form stable protein-DNA complexes , however , is probably due to the inability to form stable homodimers and not to the lack of capacity to-bind the DNA , since construction of an IHF fusion protein which contains only the putative DNA-binding domains of HimA and not those of HimD did specifically bind to the ihf site with a high efficiency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The IHF binding isotherms obtained for its ilvP target site on G the linearized and negatively supercoiled DNA plasmids are shown in Fig. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>The nitrate-dependent phosphorylation of NarL by the sensor kinases NarX/NarQ promotes the specific binding of NarL-phosphate to its sites , whereas DNA binding by IHF is apparently not regulated by oxygen or nitrate ( Walker and DeMoss , 1994 ; Darwin and Stewart , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The regulation of the nar operon has some aspects analogous to that reported here for the pfl operon , and it is tempting to speculate that pyruvate induction , like nitrate induction of the nar operon , is also mediated by a specific DNA-binding protein and that IHF functions as a coactivator facilitating interaction between the activator and either the FNR protein or RNA polymerase or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The repression observed in the GS852 IHF-C and IHF-D results are also consistent with a model where Lrp lysogens are due to IHF binding and bending the gcv normally plays primarily a structural role , and GcvA is DNA , this activation and repression should not be the gcv-specific regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>The error in the determination of the activity of IHF by this procedure is less than 10 % Scale qIHF 2 D N A K d ÿ ... IHF DNA K 2 d ÿ 4 IHF DNA ... 1 We thank Lucien Caro for continuous support during the course of this work , Griet van Houwe for expert technical assistance , and M. Buckle and H. Buc for allowing some of the irradiations to be performed in their laboratory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The results in Fig. 3 show that a single retarded DNA-protein complex was progressively formed as the concentration of IHF was increased from 0.5 to 40 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The protein concentration required for half-maximal binding is very close to the dissociation constant , Kd , because the concentration of DNA ( 50 pM ) is very much lower than the concentration of IHF ( 0.25 to 16 nM ) , and consequently the amount of free IHF approximates that of total IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The free IHF concentration in each sample was assumed to be the same as the total IHF concentration since the DNA template concentration was significantly lower than that of the total protein concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>The reduced stability of both complexes compared to that of the IHF - DNA complex makes it very unlikely that the homodimers specifically bind to ihf sites in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The IHF cells showed strongly increased unrestrained negative supercoiling but little change in DNA looping ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>The conformational change induced by IHF probably involves DNA bending ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>The levels of IHF-mediated transcriptional inhibition were also shown to be proportional to the levels of IHF present in the reaction mixtures , and below the concentration of 50 nM , no inhibition of transcription could be seen ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>The inhibition of transcription by IHF was relieved by the presence of TyrR in combination with-phenylalanine , but not by the presence of TyrR or phenylalanine alone nor by the presence of mutant TyrR protein TyrR-RQ10 with or without-phenylalanine ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>The mixture was incubated for 10 more minutes after addition 54 of ArgR and IHF in that order for reactions with or of ArgR and CRP S simultaneously for reactions with .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>The influence of CRP on the transcriptome progressively increased with increased aerobiosis , whereas the activities of ArcA , FNR , IHF and Lrp decreased as aerobiosis increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The action of many global transcription factors such as ArcA , Fnr , CRP , and IHF commonly involved both NADH and ATP , whereas others responded to either ATP or NADH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>Global transcription factors such as ArcA , Fnr , IHF , Fur , and CRP respond to both NADH and ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20299454" target="_blank">20299454</a>]<br></li>
<li>The transcript from either template was very strongly inhibited when IHF was added to the transcription reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The additional occupation of IHF III either increases or decreases transcription , depending upon the status of CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The amounts of proteins used were ( from lane 2 to 5 ) : 1.25 , 2.5 , 5 and 10 pmol for IHF ; 2.5 , 5 , 10 and 20 pmol for CpxR ; 5 , 10 , 20 and 40 pmol for OmpR and H-NS in the absence ( H-NS and IHF ) or presence ( CpxR , OmpR and RstA ) of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_754">
( a ) The gel shift assay was carried out using RstA and IHF with FITC-labelled CD6 probe DNA in the presence of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></label>
<input id="act_similarity_754", type="checkbox">
<div class="ACT_similar">
<li>( b ) The gel shift assay was carried out using RstA and IHF with FITC-labelled CD3 probe DNA in the presence of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li></div>
<li>The IHF mutations were introduced into a host carrying the sipA-lac fusion , and expression of , - galactosidase was monitored in the presence of the alpA + plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7511583" target="_blank">7511583</a>]<br></li>
<li>'' The resulting strains were grown in glycerol-containing minimal medium in the presence of PA , and the IHF effect on the lacZ expression driven by the Pa and Pz promoters was checked by b-galactosidase assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The combination of RNAP , IHF and CRP ( Fig. 3A , lanes 3 -- 5 ; Fig. 3C ) increased protection throughout the region downstream of CRP II to about -40 , especially between CRP II and CRP I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The presence of 20 nM IHF ( Fig. 3A , lanes 2 and 15 ; Fig. 3C ) caused the loss of the CRP III-associated HS sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The replacement of WT CRP with the KE101 mutant in the presence of IHF resulted in a signiﬁcant decrease in the protection between CRP I and CRP II ( Fig. 3A , lanes 16 -- 18 versus lanes 3 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The result presented in Fig. 5 clearly demonstrated that the oxidant induction of sufA mediated through ORE-I is dependent on the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The amount of the expected run-o $ product of 271 nucleotides formed in the presence of CysB and N-acetyl-L-serine was signi ® - cantly increased by the addition of IHF ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>The appearance of additional bands of unknown origin in the autoradiogram ( Fig. 5 ) are unaffected by the presence of IHF in these experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The percentages of transcriptional readthrough a t the attenuator , obtained from experiments performed in the presence or absence of IHF , are mean values f S.D. calculated from these corrected data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The IHF contribution was more important than in the excisive reactions , for example , in the reactions containing circular attP substrates , its presence led to a 4-fold increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>The DNAse I footprint of RNAP in the presence of Lrp and IHF is altered by ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The major difference between the pacs155 and pacs236 crp II was the presence of IHF III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The open stars indicate hypersensitive sites in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>The methylation of a guanine residue at base pair position +150 is enhanced in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The resulting plasmids were then transformed into the Dlac narL narP strain , JCB3884 ( Table 1 ) , in order to monitor the effects of IHF in the absence of activation by NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The effects of IHF are observed only in the absence of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>The phase-dependence of repression in the absence of IPTG is slightly reduced ( reﬂected in a smaller Capp ) , and the equilibrium constant K for the most stable loop is 2-fold reduced in the max absence of IHF ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>The absence of functional IHF in the transformed strains due to a himA mutation ( Table 2A , line 3 ) results in slightly reduced ~ - galactosidase and CPSase levels on minimal medium and minimal medium plus arginine ( for [ 3-galactosidase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The ﬁs P region from 236 to 17 , which is not directly regulated by either IHF or Fis , retained the characteristic ﬁs regulation pattern in response to a nutritional upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>The abundence of the small nucleoid-asso-ciated proteins , HU , IHF , Fis , Lrp , in the cell makes it likely that many more examples of their participation in transcription regulation of complex promoters will be found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The best characterised and most abundant members of this group of proteins are HU , H-NS , IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12372591" target="_blank">12372591</a>]<br></li>
<li>The same phenotype was also observed in an IHF + strain that lacked FNR , suggesting that IHF and FNR may work in consort to mediate pyruvate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The longest regulation path in the network is IHF → OmpR → FlhDC → FliA and further to seven operons ( marked yellow in Fig. 1B ) related to flagella motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15603590" target="_blank">15603590</a>]<br></li>
<li>The physiological role of IHF in E. coli is not known , and its concentration does not appear to vary in response to environmental change ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The question remains as to the mechanistic role IHF plays in the regulation of dmsA expression in response to anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The pattern of protection of either ModE and IHF alone , or together , when bound at dmsA are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The ilvG gene , which is responsible for the synthesis of isoleucine , is regulated by many factors , such as IHF , ppGpp , isoleucine , and valine ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10753455" target="_blank">10753455</a>]<br></li>
<li>The pst operon can be added to the list of genes and operons that are positively regulated by rS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>The function of IHF in these operons seems to be to enhance closed-complex formation ( Krause and Higgins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The absence of contamination of TorI and IntS by IHF was checked by Western blot using IHF antisera .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>The purification of proteins IHF , TorI , and IntS was described under `` Experimental Procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>The pyrimidine-speciﬁc repression was shown to be PepA , IHF , and PyrH dependent , and the purine-speciﬁc repression was shown to be PurR and PepA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>The releaseof repression of hyp by nitrate in a IHF - strain can bexplained on the same basis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>The effect of IHF on Fnr activity , however , is less clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The IHF target is even dispensable for purine-mediated regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>The possibitity is discussed that the ncgative effects of IHF in this system are mediated through OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>The Fis protein appears to also affect anaerobic and nitrate activation of narK expression in a way similar to that of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>16 , 1 ¬ ompB asmoiaiity ) OmpR IHF IHF '60 -35 -10 • 170 QmpR CmpROmpR IHF IHF -360 -60 -170 The architectural roie of IHF in gene expression is not only limited to o - ̂ ** promoters but is probably also involved in the activation of the c ' ^ ^ - dependent promoter of the narGHJI operon ( Schroder et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The mutations may preventpsp activation via IHF , and these results lead us to favor the possibility of IHF-dependent and IHF-independent activating factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>The activation of the gltBDF operon by Lrp is totally dependent on the presence of IHF , since introduction of an Lrp mutation in strain LP1048 did not lead to a further decrease in b-galactosidase level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The mechan-ism of transcriptional activation of micF and ompC by OmpR as well as a role of IHF in this activation need to be better understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The activation factors are smaller than for CRP either because additional interactions exist between the genuine activator CRP and RNA polymerase , or because the geometrical distortions introduced by IHF do not exactly mimic the DNA conformation induced by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The activation factor Position CRP ( activation ) Position IHF ( inhibition ) Activation factors are given as the ratio of luminosities in wt strains compared with strains deleted for the activator ( CRP or IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>The magnitude of IHF activation is smaller than the activation obtained with CRP bound at the same positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>In principle , bound IHF could enhance any step leading to a final complex in which bound polymerase and bound activator touch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>( ii ) In IHF cells we could still detect the increase in intensity of the G-signal ; as expected , the signal was absent in the IHF deletion derivative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>In vivo experiments indicated that the himA and himD genes were negatively autoregulated by IHF ( Miller etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>In vivo and in vitro evidence suggested that at least some of the effects by IHF are due to direct or indirect interactions of IHF with OmpR , the major transcriptional-regulator of these operons ( Huang etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Transcription from the bla control fragment was slightly increased by IHF ( 15 to 25 % ) , strongly suggesting that IHF inhibition of ompB was specific .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>As the concentration of IHF was increased another complex of lower mobility was formed ( `` complex a '' ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>The ﬁrst retarded complex , in which IHF I was occupied ( see below ) , had increased mobility compared to the corresponding complex with the E. coli K-12 pnir fragment , suggesting that IHF bends pnirSTM less when bound at IHF I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>In vitro binding of IHF The activity of the IHF preparations was assayed by titrating 50 nM of IHF bound to 1 nM of radio-labeled double-stranded 40-mer oligonucleotides with increasing amounts of the same non-radioactive 40-mer oligonucleotide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>In vitro footprinting was performed by irradiating 40 ml of binding reactions containing 5 nM of supercoiled plasmid and increasing amounts of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>When the concentration of IHF protein was increased to 50 nM in the footprinting assay , no additional protected regions were seen within the narc fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Concentrations of IHF are 5 , 25 , 50 , 100 , 125 , 200 , 250 , 500 and 1000 nM for lanes d to l , respectively ( coding strand ) and for lanes o to w ( non-coding strand ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>A + G sequencing reactions ; 2 , no IHF added ; 3 , 25 nM-IHF ; 4 , T+C sequencing reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>- e. no IHF ; O-O , 80 nM IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>no IHF ; lane 4 , 20 nM IHF ; lane 5 , 40 nM IHF ; lane 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Typically the concentration of IHF was around 50 nM , in roughly 1000-fold excess over the radio-labeled probe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>When present , the concentration of RNAP was 30 nM , Lrp 100 nM and IHF 100 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>When present , the final concentration of IHF in the assay was 50 nM unless otherwise noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Abolishing the IHF protein by mutation ( Table 2B , line 3 ) also results in reduced 13-galactosidase levels under conditions of derepression and increased levels upon pyrimidine supply .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>It is possible that IHF and an additional protein , such as FIS , H-NS or HU , along with CysB may be necessary for full activation of the positively regulated cysJIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>It involves repression by the GB regulator ICLR and activation by the nucleoid protein IHF and the global regulator Cra ( also known as FruR ) ( reviewed in reference 95 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Gene 76 Kano , Y. , Ogawa , T. , Ogura , T. , Hiraga , S. , Okazaki , T. and Imamoto , F. : Participation of the histone-like protein HU and of IHF in mini-chromosomal maintenance in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Nevertheless , each protein functions speciﬁcally in one operon but has no effect in another operon , arguing that selective action of HU and IHF in different operons might enhance the ability for speciﬁc regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>HU protein was found to contribute about fourfold to repression while IHF had no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>IHF protein binds to pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>It is possible that the presence of an excess of one subunit shifts the equilibrium of dissociation between fhe IHF subunits and thereby increases the effective IHF concentration in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>It was therefore proposed that the asymmetry of IHF is not related to its function but to the mechanism by which the protein recognizes its target , in a putative stepwise fashion ( Werner etai , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>It is possible that IHF plays a role in Hsp induction under one or more such conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>It has recently been shown that under certain conditions , IHF subunits can form homodimers ( Werner etai , 1994 ; Zulianello etai , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>This is in contrast to our recent ﬁnding that IHF contributed twofold to repression of glpT in the presence of internal glpT operators while HU had no effect ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Despite the fact that HU and IHF have many structural and functional similarities , it was shown that IHF had little or no effect on glpD transcription while HU increased repression about fourfold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Although evidence is lacking , it is conceivable that IHF could bind A+T - rich proximal sequences in other E. coli promoters such as fdhF and hyd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>We therefore investigated the activating effect of IHF both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>We were , therefore , unable to test in vitro the direct effect of IHF on the himD/hip promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>However , as shown in Figure 10 , in this case too , IHF exerts its severe inhibitory effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>NarL insulates pnrf from the effect of IHF III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Moreover , IHF exerts its effect on acetate , while hardly any effect is observed on glucose [ 47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19852855" target="_blank">19852855</a>]<br></li>
<li>IHF was displaced from the probe by increasing amounts of the identical , non-radioactive oligonucleo-tide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF is beginning to be displaced from the radiolabeled probe when there are roughly stoichiometric amounts of unlabeled oligonucleotide and active IHF in the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Thus , IHF III must be correctly positioned for IHF-mediated stimulation to occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Thus , it appears that IHF is required , but not sufﬁcient , for the induction of these Gsps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Thus regulation of sodA by the SoxRS regulon appears to be independent of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>We found that IHF is a negative effector of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Therefore , the negative effects of IHF on ompF and ompC could be due to the inhibitory action of IHF on ompB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>NsrR binds to a target that overlaps IHF I , and the effects of NsrR and IHF appear to be additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>IHF binds to sites centred at -179 , -85 , -42 , and +131 in ompF , ilvBN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>dps and sra are positively regulated , while osmY and ihfA are negatively affected by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>Both hiniA promoters were found to be negatively controlled by IHF ( Miller e! ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Additionally , physical interactions between Crp and/or Fur with other regulators such as IHF , Cra , and FadR could be important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659676" target="_blank">15659676</a>]<br></li>
<li>Lane 1 , hlu fragment only , no IHF ; lane 2 , onipF fragment only , no IHF ; lane 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>It is well known that bound IHF induces a sharp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Binding of IHF strongly increased the variation in the mobility of the permuted fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>b Binding of IHF to the pennuted fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Binding of UlaR , CRP , and IHF to the ulaG-ulaA intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Binding of IHF to the phoA control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>IHF , exemplified by the Pap pilins , are regulated by PapI and PapB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8102362" target="_blank">8102362</a>]<br></li>
<li>Transcription of the acc genes dose not appear to be regulated by Crp , IHF or FadR [ 294 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>However , this does not appear to be the mechanism by which IHF stimulates ﬁs P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Purity of IHF , TorI , and IntS fractions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>) , Fur , ArcA , IHF ( neg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Negative regulation by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>IHF contributes to the induction of aceBAK by opposing repression by IclR during growth on acetate ( inducing conditions ) but not on glucose ( repressing conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550439" target="_blank">8550439</a>]<br></li>
<li>The ability of IHF to relieve repression under inducing but not repressing conditions allows this protein to amplify the induction of aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>To study the inﬂuence of IHF on repression , strains BY4004 , BY1004 , BY4259 and BY1259 ( Table 3 ) were contructed by deletion of himA in strains BY4000 , BY1000 , BY4159 and BY1159 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>These results indicate that repression by nitrate is independent of the three transcription factors ArcA , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>It was anticipated that nitrate repression would not show a dependence on the presence of ArcA , FNR , or IHF , which proved to be the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Furthermore , we found that IHF repressed div P on pWN12 and pWN14 but not on the pminiDiv construct , implicating the sequences between ± 108 and ± 156 in mediating repression ( Figure 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>These antagonistic effects exerted by IHF lead to a two - to threefold reduction of the pyrimidine repression coefficient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Alternatively when one subunit is expressed at high level , homodimers can form and act in the repression of the IHF promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>These results suggested the participation of IHF in the repression of the ula regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Nitrate repression is independent of IHF , FNR , and ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Is tyrosine repression affected by IHF ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961453" target="_blank">7961453</a>]<br></li>
<li>IHF prevents repression by IclR under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Quantification of the data by scanning densitometry showed that IHF inhibition was lowered from 72 % to 27 % by increased amounts of OmpR ( Fig. 5b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>The data in Fig. 5a show that increasing the amount of partially purified OmpR in the in vitro transcription assay from 100 ng to 150 ng strongly reduced inhibition by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>free Consistent with this idea are the results showing that IHF inhibition of transcription can be strongly reduced by increasing the levels of partially purified OmpR normally added to the in vitro assay ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Data illustrated in Fig. 1B show that , as expected , pnir activity was increased by deletion of Fis I. Activity was then decreased by deletion of IHF II but restored by further deletion of IHF I. Effects of IHF are different at the nir promoter in EPEC and S. enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Transcripts from the pspG promoter increased with increasing concentrations of PspF ( Fig. 6B ) and it appears that in the absence of IHF the pspG promoter is much more sensitive to PspF concentration than the pspA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Accordingly , increasing concentrations of IHF progressively inhibited in vitro transcription from the ndh promoter whilst formation of the transcript was increased ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>In an IHF + background , the activity of CAT was mildly inhibited by the rpoS mutation ( Fig. 2 ; columns a , b ) , conﬁrming that rS contributes to the overall transcription level of pst ( Taschner et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>In other words , in the absence of IHF ErS has lost its ability to compete with ErD in recognizing the pst promoter causing it to behave like phoA and the other PHO genes , which are inhibited by the presence of rS ( Tas-chner et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>Abortive transcription initiation assays were performed in the presence or absence of IHF according to the procedures of Hawley et al. ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>In Vitro Transcriptions -- In vitro transcription reactions were performed according to the procedures of Hauser et al. ( 20 ) , with closedcircular supercoiled plasmid pDHDwt ( Table I ) , in the absence and presence of purified IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>Whether in the absence or the presence of IHF , FIS inﬂuenced transcription similarly : initially enhancing transcription and then decreasing it ( Fig. 6C , compare lanes 1 -- 5 with lanes 6 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Whether in the absence or the presence of FIS , IHF diminished transcription about two - to threefold ( compare Fig. 6B , lane 5 with Fig. 6C , lanes 6 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>However , in the absence of IHF , pst expression was signiﬁcantly inhibited by rS ( Fig. 2 , columns c , d ) , suggesting that IHF plays a po-sitive role in the rS-transcription initiation of pst .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>Moreover , the positive effect of IHF on overall pst transcription ( compare columns a , c in Fig. 2 ) is abolished in the absence of a functional rS ( compare columns b , d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>In HP4110 , b-galactosidase levels showed little variation irrespective of the presence of ihf sites , indicating that the deletions or point mutations did not signiﬁcantly affect transcription in the absence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>These results show that in vivo transcription from ﬁs P on a plasmid is about 3.8-fold higher in the presence of IHF than in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Furthermore , the results of the abortive transcription initiation assays performed in the presence and absence of IHF ( Table 4 ) show that IHF binding in UAS1 increases the rate of open complex formation at ilVPG four-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Here , we report effects both on fimA transcription and on fimA promoter segment inversion in the presence and absence of protein cofactors IHF and LRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
<li>We found that at a mutant glnHp2 promoter with increased affinity for 054 - RNA polymerase , the initiation of transcription ca , n be activated equally well by NR,-phosphate in the presence or absence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Inhibition of ompF transcription by IHF in the presence of OmpR ( ng ) : 100 , 72 % , 125 , 51 % ; 150 , 27 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>This transcription also requires s54-containing RNA polymerase and is stimulated by IHF and by the presence of the UASs and TABLE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260970" target="_blank">9260970</a>]<br></li>
<li>Transcription in vitro ( activated by PspF ) was slightly inhibited by IHF ( data not shown ) , which is consistent with the observation that IHF stimulates transcription only with a speciﬁc activator ( Dworkin et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>However , under the conditions of our experiment on in vitro transcription from P6 , with PspF as an activator , IHF slightly inhibited the reaction ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>In a number of cases in which IHF positively regulates gene expression , IHF does not activate transcription directly but enhances the effectiveness of an activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>This inhibition was reversed by increasing the concentration o $ OmpR , a positive activator required for ompF expression , suggesting that IHF may inhibit ompF transcription by altering how OmpR interacts with the onzpF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Thus IHF enhances both the activation of psp transcription and the negative autogenous control of pspF keeping the PspF concentration at a low level ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>However , it appears that the role of NarL at PnarG is more direct , as transcription activation of PnarG can not be detected in vitro , unless FNR , IHF and NarL are all present ( E. C. Ziegelhoffer and P. J. Kiley , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>The molecular mechanism by which NarL , IHF and FNR function together to stimulate transcription activation at PnarG is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></li>
<li>Regulation of ® s transcription by IHF The mechanism of activation of ® s transcription by IHF appears to be such an example of shifting the balance of relative activities of the ® s P3 , P2 , P1 and div promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Furthermore , the effect of IHF on tandem ® s promoters is selective , since IHF increases the transcription of ® s P2 and P1 but reduces that of ® s P3 in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>This notion is further reinforced by our in vitro data clearly showing that IHF speci ® cally increases both the opening and transcription of P2 and P1 promoters , but strongly represses those of div and P3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>In those cases in which IHF acts as an architectural element in transcription regulation it ts probably no !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Therefore , we conclude that transcription from both the tiimO/hlp and NmA P4 promoters is negatively regulated by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>IHF did not stimulate transcription and appeared to inhibit it when ArgR was also present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>IHF enhances transcription when the t.hree proteins are located on the same face of the helix , but strongly inhibits transcription when any one of the proteins is located on the opposite face .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>IHF , which binds around position -- 180 , inhibits transcription from PI and P3 , but not from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF inhibits transcription and the values represent inhibition factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>IHF inhibits onipF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Binding of CysB is necessary for e cient cysJIH transcription ( Hryniewicz and Kredich 1995 ) and IHF may act only as an accessory factor facilitating activation by CysB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>Schematic model for the stimulation or inhibition of glnHp2 transcription activation by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>These results support the supposition that IHF III contributes to inhibition of acs transcription and that sequences upstream of IHF III contribute to full activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>These results , taken together with previous data ( 32 , 47 ) , indicate that no fewer than four proteins ( IHF , 54 , PrpR , and CRP ) are involved in the regulation of prpB-CDE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>When CRP II is not occupied , the binding of IHF enhances transcription , perhaps by increasing the stability of the open complex ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>When both CRP sites are occupied , the binding of IHF to IHF III results in signiﬁcantly reduced transcription , possibly because compaction of the upstream elements overstabilizes the complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>When AR2 is disrupted by mutation , or when either the promoter-distal CRP II or the promoter-proximal IHF III sites remain unoccupied , some of these upstream contacts are lost , and transcription can proceed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>It was also reported that the nucleoproteins , FIS and IHF , bound directly to multiple regions of the acsP2 promoter , that their binding reduced CRP-dependent acs transcription , and that FIS competed with CRP to occupy the acsP2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>CRP activates acs transcription by binding tandem sites located upstream of the major promoter , acsP2 , whereas , both IHF and Fis each act at different times during growth as anti-activators of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Maximal transcription from pacs236 crp II ( which includes CRP I and IHF III , but not CRP II ) depended upon the presence of both IHF and AR2 , as well as the inability of CRP to-bind at CRP II ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In vivo , the presence of IHF III strongly reduces CRP-dependent transcription , sug gesting that the binding of IHF to this site interferes with the ability of CRP to activate transcription ( 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In vivo , the presence of IHF III strongly reduced CRP-dependent transcription , whereas the presence of additional upstream sequences increased acs transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In this study , we show that two nucleoid-associated proteins , FIS and IHF , independently modulate CRP-dependent activation of acsP2 transcription and present in vivo evidence to support that contention .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>A. Schematic showing acs transcription ( acs , TRXN ) , growth ( OD ) and the relative concentrations of active CRP , FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Increasing concentrations of IHF decreased CRP-dependent acsP2 transcription as much as twofold ( Fig. 6B , lanes 2 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>To activate acsP2 transcription ( in the absence of IHF ) , CRP performs two roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>In vitro , we demonstrated that FIS and IHF can bind simultaneously and inﬂuence CRP-dependent acsP2 transcription independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>DNase I analysis and in vitro transcription assays showed that IHF can bind simultaneously with CRP and can anti-activate CRP-dependent acsP2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>With IHF bound to its lower-affinity sites , RNAP can not form the closed complex , presumably due to the low stability of these initial interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The resulting complex ( composed of IHF bound at IHF III , CRP dimers bound at CRP II and CRP I , and RNAP bound at acsP2 ) could fold into a wrapped conformation that inhibits escape due to numerous interactions between RNAP and several upstream elements ( Fig. 7D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Thus , the CRP -- RNAP complex can successfully compete with this concentration of IHF for access to the acsP2 promoter , while RNAP alone can not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>It appears that CRP can stabilize the closed complex enough to compete successfully with IHF and , thus , form a TOC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>C. b-Galactosidase activity from promoter fragments pacs444 , pacs236 , pacs205 , pacs155 or pacs236 variants defective for CRP I , IHF III or CRP II fused to lacZYA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>IHF can bind simultaneously to the acs promoter region with either CRP or FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_934">
Sclavi , B. , Beatty , C. , Thach , D. , Fredericks , C. , Buckle , M. , and Wolfe , A. ( 2007 ) The multiple roles of CRP at the complex acs promoter depend on activation region 2 and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487274" target="_blank">20487274</a>]<br></label>
<input id="act_similarity_934", type="checkbox">
<div class="ACT_similar">
<li>Sclavi , B. , Beatty , C.M. , Thach , D.S. , Fredericks , C.E. , Buckle , M. , and Wolfe , A.J. ( 2007 ) The multiple roles of CRP at the complex acs promoter depend on activation region 2 and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22059728" target="_blank">22059728</a>]<br></li></div>
<li>The effects of alterations in the alIinity of the as4-RNA polymerase for the promoter on open complex formation : on linear and supercoiled DNA in the presence and absence of 7 HF , are shown in Figure i. Tt can be seen that the replacement , of the ` I ` by G has improved t.he abi1it.y of the promoter to form open complexes on supercoiled DNA in the absence of TH1 ' ; as a result of this improvement , there is little enhancement of open complex formation by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>The antagonistic effects exerted by IHF on the expression of carAB promoter PI could be explained by the IHF-stimulated formation of a protein-DNA complex around position -106 , if the interacting pre-tein ( which could be the carP gene product ) could act as an activator in the absence of a pyrimidine cofactor or as a repressor in the presence of it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Apparently , when NR,-phosphate and IHF are bound to the same face of the DNA helix , as in the wild-type promoter region , IHF enhances the ability of NR,-phosphate to activate open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>It can be seen that under our experimental conditions , NR,-phosphate in the absence of IHF was able to activate open complex formation on supercoiled , but not on linear DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>The addition of IHF enabled NR,-phosphat to activate open complex formation on linear DNA to a level approximat.ely equivalent to that attained on supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>It is of functional IHF site , were inserted upstream of the strong rrnB termi - significance to note that IHF half-maximally represses trannator sequence contained in the plasmid pDD3 to yield the scription from the iluPGl promoter at a concentration of plasmid constructspJP29 and pJP3 ( 0Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Binding of IHF to its target site about 300 nucleotides upstream of the startpoint of P1 transcription exerts antagonistic effects on promoter activity , i.e. stimulation of transcription in minimal medium but an enhancement of the repression in the pre-sence of pyrimidines ( Charlier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li>
<li>the GATC target We have shown previously [ 18 ] that binding of IHF to its target site centered around nucleotide -305 upstream of the PI startpoint of transcription in the E coli and S typhimurium carAB operons , enhances the repressibility by pyrimidines of this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Under conditions where P1 is actively expressed ( minimal and arginine-supplemented medium ) , this promoter is stimulated by binding of IHF to the upstream target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Remarkably , transcription in vivo from the mutant promoter was unaffected aerobically , which indicates that TdcA , bound to the presumptive co-inducer , together with IHF and TdcR are all essential for transcriptional activation , even when the cAMP ± CRP complex is not limiting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Lack of IHF leads to a decreased hyc operon transcription which may result ina higher occupancy of IR2 by FHLA and in an increaosfeP , promoter activation.A decreased formationof HYCA in the IHF-strain ( HYCAis a `` repressor '' counteractingFHLA in some unknown manner ; Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>In anaerobic conditions , FNR binds to its target site , but transcription activation is repressed by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>In this report , we provide data to show that the IHF interaction with the upstream iluPG1 promoter target site is , in fact , responsible for the repression of iluPGl promoter activity and the simultaneous 4-fold activation of transcription initiation from the iluPG2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>IHF binds just upstream from this o ^ '' - dependent promoter and strongly stimulates NifA activation of open complex formation In vitro ( Hoover et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>In contrast , ArcA regulation shows a strong dependence on IHF , indicating that IHF is necessary to allow ArcA to engage in transcriptional activation of this promoter system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>In the cases studied to date , IHF stimulates transcription only when the activator is present , and does not enhance the binding efficiency of either the activator or Ea5 ' 4 to the promoter ( 30,35,36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>In this report , we describe experiments designed to determine how IHF represses the activity of the ilvPo1 promoter and activates transcription from the ilvPo2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>In contrast , activity of the overlapping s70-promoter appears to be unaffected by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The association of Fis , IHF and H-NS suggests that nir promoter DNA is sequestrated into a highly ordered nucleo-protein structure that represses FNR-dependent transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The relatively strong promoter activity in the absence of IHF and the relatively weak influence of externally added IHcFan be explained as follows : Claverie-MartinandMagasanik ( 38 ) have found that on linear DNA the activation of transcription from the dependent glnHP2 promoter from E. coli by NTRC-P depends upon the presence of IHF , which binds to a site located between the NTRC binding sites and thepromoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>The increase up to 4-fold in proU-P2 promoter expression in the presence of IHF occurs at low salt concentrations where the promoter is under H-NS repression showing that IHF was partially alleviating H-NS repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>The relative increase in repression due to Fis or IHF for each promoter is shown in parentheses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The observation of notably decreased activity of an artiﬁcial ptac promoter in the absence of IHF has further implications for studies on in vivo promoters ' activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>The binding of IHF may therefore in ¯ uence the binding of H-NS in such a way that H-NS enhances transcription from the ® mA promoter only in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>The reporter plasmids used in our transcription-translation syste were in the supercoiled conformation which possibly reduced the influence of IHF on hyc promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>The activation of the csgD promoter by IHF associated with site IHF-1 might be attributable to a conformational change of the upstream segment of the csgD promoter , for instance through protein -- protein interaction between site-1 - and site-2-associated IHF molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The NarL-mediated nitrate regulation of the promoter requires IHF , which recognizes and binds a 12000 10000 8000 6000 E C ' U 4000 2000 L I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The role of IHF at the nrf promoter is to set its activity under conditions where nitrite/nitrate-dependent activation via NarL/NarP is minimal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>The proposed hairpin structure for this promoter is supported by the absolute dependence of its activation on IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>The best-characterized examples of this are at the glnHp2 promoter where activation by NtrC requires IHF ( 4 ) and IHF-dependent NifA activation of the nifH promoter ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The induction was dependent on OxyR , IHF and an unidenti-ﬁed factor that binds to the ORE-III site in close proximity to the -35 region of the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The two IHF target site deletions ( AST2 and AST3 ) and the P2 promoter mutation ( ST5 ) were , respectively , introduced using the oligonucleotides 5 ' - TATATTGATTTTTAAACGTTA-ACTTTCTTGATTTTG-3 ' , 5 ' - GTTTTTTATATTTTT-AATATAAAATTTTTTTGTCGTTAAC-3 ' and 5 ' - GC-AAATTTGCAAAACTTCATACTG-3 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Moreover , PspF and IHF bind cooperatively in the psp promoter-regulatory region , and this effect is dependent on the presence of an intact psp enhancer ( Jovanovic & Model , 1997b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>OxyR and IHF can simultaneously bind to the dps promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>864 F. W. Outten , O. Djaman and G. Storz Independent binding of IHF , OxyR and Fur to the suf promoter Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Possibly the dps promoter is not ideally bound by either holoenzyme leading to the need for IHF to enhance the binding of o ^ - RNA polymerase and OxyR to enhance the binding of a '' ° - RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Although genetic evidence indicates that both Fur and IHF regulate suf , it was not known where these two regulatory factors bind to the suf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>As seen in Fig. 7 , both promoter regions bind IHF specifically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>When the levels of the inducing nitrite or nitrate ions fall , the promoter must be reset to the inactive state by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Giladi , H. , Koby , S. , Prag , G. , Engelhorn , M. , Geiselmann , J. , and Oppenheim , A.B. ( 1998 ) Participation of IHF and a distant Up element in the stimulation of phage l PL promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Bertoni , G. , Fujita , N. , Ishihama , A. , and Lorenzo , V. ( 1998 ) Active recruitment of sigma54-RNA polymerase to the Pu promoter of Pseudomonas putida : role of IHF and alphaCTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Perhaps a change in DNA conformation when IHF binds in ompC results in alterations in DNA structure that lead to a reduction in P1 and P3 promoter function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>In separate experiments ( 7 , 30 ) , these authors found that CAP and IHF positively regulate ilvBN and that its promoter DNA binds CAP and IHF between positions -82 and -44 and between positions -99 and -73 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>IHF binds to three sites in the ompS promoter region centred at -102 , -37 , and +8 and inhibits transcription in vivo and in vitro ( Tsui eta !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>One such gene is nir ( 3 ) where H-NS binds to the upstream region in concert with FIS and IHF to repress promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li>
<li>- A -4 - B - C RESULTS IHF binds to the ompB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>IHF , which binds just upstream of the aceBAK promoter , enhances the response of IclR to the carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755903" target="_blank">8755903</a>]<br></li>
<li>This result adds another layer of complexity to a promoter already known to require two other proteins , IHF and PrpR , for transcriptional activation ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>We show furthermore that direct contacts between RNA polymerase and regions of IHF corresponding to ARI , ARII or ARIII of CRP are not responsible for activation : the class II promoter variants of malT are repressed by IHF binding , whereas the homologous CRP promoters are activated by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>It is interesting that the transcriptional activation of the lambda pL promoter by the binding of IHF to an upstream site also results in the repression of a second upstream promoter , pL2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>If differential evolution of HimA and HimD has indeed led to an activation domain in IHF , this would predict that the orientation of the IHF-binding site with respect to the promoter is the same in the three opetons that have been shown to be directly activated by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>We suggest that IHF site 1 is ideally located to allow this interaction to take place to facilitate transcriptional activation of the norVW promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Furthermore , we demonstrate an unusual activation mechanism by IHF , involving a coordinated shift in the balance of promoter activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>On the other hand , in strains where this repressibility is already reduced by mutation ( carP6 , PyrH ± ) , i.e. under conditions where P1 is still very active even in the presence of high pyrimidine concentrations , removal of the IHF target site ( AST3 ) or elimination of the DNA-binding protein itself ( in the himA context ) leads to reduced promoter activities in all conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Lanes 2 -- 5 and lanes 6 -- 9 are controls that show the binding of NarL and IHF to the promoter region in the absence of a second protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>It is important to note that the 4-fold increase in iluPo2 promoter activity , which we have shown to be mediated by IHF protein binding to the upstream iluPG1 promoter region , is dependent on the presence of this adenine-rich region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Characterization of protein binding To understand the unexpected behaviour caused by disruption of AR2 and/or the presence of IHF III , we characterized the binding of each protein to the acs promoter region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>IHF can also inhibit transcription indirectly through modulation of a regulator protein , in the ompF promoter region , IHF binds to two sites that flank the binding region of the OmpR regulator that is situated between - 6 0 and -100 ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF binding just upstream from this a ^ ' ^ promoter markedly enhances the ability of the NR1 protein to activate transcription ( Claverie-Martin and Magasanik , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>If more than one hexamer binds to the ast promoter , as implied by the mobility shift and footprinting results , they could cause a bend closer to 180 ° , which would be comparable to the bend caused by IHF binding ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>Lrp and IHF appear to stabilize RNAP binding to the promoter , as no RNAP-dependent footprint appears unless both Lrp and IHF are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Recently , it was found that the binding of HU increased by two - to threefold the repression of the P2 promoter of the gal operon while IHF had no effect ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>These antagonistic effects result in a two - to threefold reduction in the repressibility of promoter P1 by pyrimidines in the absence of IHF binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>However , the combined effects of IHF and CAP in altering promoter conformation remain to be established by binding experiments with IHF plus CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Cooperative binding of the two activators RstA and IHF to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1001">
( d ) Model of cooperation between OmpR and IHF in binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></label>
<input id="act_similarity_1001", type="checkbox">
<div class="ACT_similar">
<li>( c ) Model of cooperation between RstA and IHF in binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li></div>
<li>Binding sites for the regulatory proteins IHF , PepA , PurR and ArgR involved in the purine , pyrimidine and arginine-speciﬁc control of promoter activity are colored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>The antagonistic effects exerted by IHF under conditions of growth on minimal or uracil-supplemented medium result in an approximately two - to threefold contribution to the repressibility of promoter P1 by py-rimidines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>However , the b-galactosidase activity produced by the div promoter construct pWN3 increased by > 5-fold , suggesting that IHF represses div ( Figure 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>IHF stimulates Pi promoter activity in minimal medium , but also increases the repressibility of this promoter by py-rimidines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>IHF stimulates P1 promoter activity in minimal medium , but increases the repressibility of this promoter by pyrimidines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>In Vivo Effect of DNA Supercoiling on Transcription from the ilvPG Promoter in the Absence and Presence of IHF -- To determine if changes in DNA supercoiling affect promoter activity in vivo in the same manner as in vitro , promoter activities were measured in gyrase - ( gyrB ) and topoisomerase ( topA ) - deficient strains containing a functional or a mutated himA ( IHF ) gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>The results presented in Table II show that , like in vitro , the in vivo expression of the ilvPG promoter in the presence of IHF is proportional to the degree of negative DNA supercoiling in the physiological range of superhelical densities ( s 5 20.03 to 20.09 ; Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>We show that the activity of the ilvPG promoter increases with increased negative supercoiling and that this sensitivity to superhelical density is enhanced in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>These conclusions are based on the in vitro observations that : ( i ) although promoter activity is increased by DNA supercoiling , both in the presence and absence of IHF the relative activity of the promoter remains the same over a wide range of template DNA superhelical densities ( Fig. 5 ) ; and ( ii ) IHF binds with indistinguishable affinities to relaxed and supercoiled DNA templates ( Fig. 9 ; Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>These data suggest that IHF does not affect the overall response of the ilvPG promoter to changes in superhelical DNA density ; rather , in the presence of IHF , promoter activity is more sensitive to small changes in the superhelical density of the DNA template in the physiological range of s 5 20.03 to 20.09 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>In the absence of IHF , promoter activity increases with increased DNA supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>That is , the superhelical DNA density that produced half-maximal promoter activity in the presence of IHF was the same as the superhelical DNA density that produced half-maximal promoter activity in the absence of IHF ( Figs. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>In the presence of IHF , the same increases in superhelical DNA densities result in larger increases in promoter activity until a maximal activation of 5-fold is obtained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>However , in the presence of IHF , increases in superhelical DNA densities resulted in larger increases in promoter activity until a maximal activation of 5-fold was obtained at a superhelical DNA density of s 5 20.09 ( Figs. 4 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>IHF also activates transcription from this promoter by increasing the rate of the isomerization step and , in the presence of IHF , the activating effect of increased negative DNA supercoiling is enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>Thus , although both increased DNA supercoiling and IHF facilitate the unwinding of the DNA helix and the rate of promoter open complex formation ( 7 ) , our results suggest that the activation of this rate-limiting step in the transcription initiation reaction by DNA supercoiling and IHF are effected by distinctly different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1019">
Thus , IHF and increased DNA supercoiling activate transcription by different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></label>
<input id="act_similarity_1019", type="checkbox">
<div class="ACT_similar">
<li>Thus , IHF and increased negative supercoiling activate transcription from the ilvPG promoter by different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li></div>
<li>In this case the mechanism of IHF action is not known although , in view of the fact that IHF binds to DNA , it seems probable that IHF facilitates the adoption of the optimum DNA conformation for RNA polymerase binding and transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>Since IHF has also been shown to cause a strong bend upon binding DNA and both regulators bind to unusually long sequences , the combined binding of OxyR and IHF may lead to an unusual promoter structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>In vitro gel-shift experiments demonstrated direct binding of the promoter DNA to IHF and ArcA-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>Alternatively , binding of ModE and IHF may alter DNA conformation at the dmsA promoter to facilitate binding of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>On the basis of the observation that speciﬁc binding of IHF induces bending of DNA ( 3 , 4 , 8 ) , we proposed that a speciﬁc folded DNA-protein complex is generated when NARL-P and IHF bind which interacts with the RNA polymerase-narGp promoter complex and stimulates transcription ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>The fact that IHF can bind to very relaxed sequences and , to some extend , to DNA with no speciﬁc binding sites , indicates that this protein has a potential to regulate promoter 's activity both speciﬁcally and generally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>IHF bound between the PspF UASs and pspA-E s54-dependent promoter bend DNA and PspF molecules bound to the UASs interact with s54-RNAP ( arrow ) , activate pspA-E transcription and autogenously regulate pspF transcription ( bar ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>In other words , IHF might activate transcription simply by bending the DNA helix in the upstream activating region and altering the local conformation of the DNA in the nearby iZvPo2 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>In the context of the pacs236 promoter construct , where the binding of IHF to IHF III accentuates the bending of upstream DNA , these additional upstream stabilizing interactions can result in a decrease in the amount of transcription ( Fig. 7D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Thus , it is possible to envisage that while bending of promoter DNA by IHF or CAP , separately , may not create the optimum promoter conformation to activate of both bend DNA into transcription , binding proteins might a new conformation to allow transcription to proceed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Therefore , IHF binding does not activate transcription simply by increasing the local negative supercoiling of the DNA helix in the downstream promoter region or by differential binding to relaxed and supercoiled DNA templates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>31 ) , it is clear that IHF does not activate transcription simply by increasing DNA supercoiling or by binding better to supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1033">
Binding of IHF at this site , on a negatively supercoiled DNA template , activates transcription from the downstream ilvPG promoter an additional 3 to 4-fold ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></label>
<input id="act_similarity_1033", type="checkbox">
<div class="ACT_similar">
<li>Binding of IHF to this site on a negatively supercoiled DNA template activates transcription from the ilvPG promoter another 3 -- 5-fold ( 6 , 7 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li></div>
<li>When the IHF site in UAS1 is replaced with a LEF-1 site , LEF-1 activates transcription from the downstream ilVPG promoter in E. coli as well as it is activated by its natural activator , IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>IHF inhibits the PcM promoter in Mu when it is bound to a site 80 bp downstream from the start of transcription ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>These results suggest the interesting possibility that IHF activates transcription from the nearby downstream promoter simply by bending the DNA helix in the absence of specific IHF-RNA poiymerase or upstream DNA-RNA polymerase interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>These and other observations ( see `` Discussion '' ) coupled with the observations that IHF-mediated activation occurs in the absence of specific protein interactions between IHF and RNA polymerase , and that IHF-mediated activation requires a negatively supercoiled DNA template , support the hypothesis that IHF activates transcription from this promoter by an allosteric DNA mechanism that is influenced by the superhelical state of the DNA template ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>It is possible that the IHF-induced DNA bend , rather than the IHF protein itself , may be the functional component necessary for the activation of the & PC2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>If , indeed , specific protein interactions are unimportant , and if IHF-mediated activation of transcription from the ilvPG promoter is influenced by the IHF-induced DNA bend in the UAS1 region , then any heterologous protein that bends the DNA helix to the same approximate extent as IHF might be expected to activate transcription from the downstream ilVPG promoter site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Activation of transcription from the ilVPG promoter by a similar protein-protein interaction mechanism is unlikely because IHF-mediated activation of ilvPG is partially retained when the IHF binding site is reversed and placed on the opposite face of the DNA helix and because a heterologous DNA-bending protein , LEF-1 , can activate transcription from ilvPG as well as IHF ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>In this report , we show that this activation occurs in the absence of specific proteinprotein interactions and that the formation of an IHF-DNA nucleoprotein structure in the UAS1 region of a negatively supercoiled DNA template causes a conformational change in the DNA helix at the downstream promoter site and increases Abbreviations : IHF , integration host factor ; UAS , upstream activating sequence ; LEF , lymphoid enhancer-binding factor ; GST , glutathione S-transferase ; HMG , high mobility group ; IPTG , isopropyl f3-D-thiogalactoside ; ONP , o-nitrophenol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>This interaction is facilitated by DNA-looping promoted by IHF bound to a site located between the psp promoter and enhancer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>In many instances , interaction of the s54-RNAP bound to the promoter and the activator bound to the UAS is facilitated by DNA bending mediated by integration host factor ( IHF ) , which binds to a site between the UAS and the core promoter region ( 14 , 19 , 21 , 35 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>The integration host factor IHF , named for its role in phage integration , binds speciﬁc asymmetric DNA sites ( WATCAANNNNTTR ; W , A or T ; N , any base ; R , A or G ) as a heterodimer ( 31,32 ) , bending DNA by 1608 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Previously , we reported that cAMP receptor protein ( CRP ) activates acs transcription by binding tandem DNA sites located upstream of the major acsP2 promoter and that the nucleoid protein IHF ( integration host factor ) binds three speciﬁc sites located just upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Transcriptional activation by protein-induced DNA bending : Evidence for a DNA structural transmission model ABSTRACT Integration host factor ( IHF ) is a DNA-bending protein that binds to an upstream activating sequence ( UAS1 ) and , on a negatively supercoiled DNA template , activates transcription from the ilVPG promoter of the ilvG-MEDA operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Next , we asked if host protein integration host factor ( IHF ) , which is closely related to HU but binds to DNA in a sequence-speciﬁc manner , can inhibit transcription initiation from the mtr or the tyrP13 * promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>IHF bound to the phage ; II1A site Downloaded from www.microbiologyresearch.org by IP : 132.248.34.24 On : Fri , 05 Aug 2016 22:42:4 Lrp-binding analysis Role of Lrp in gcvT : : lacZexpression in the IHF-C , IHF-D and IHF-G mutants We used a gel mobility-shift assay to determine whether Lrp could still bind to the gcv control region carrying the Since all but the upstream 22 bp of the Lrp-binding insertion sequences , and this binding could be responregion was replaced in the IHF lysogens , we predicted sible for the negative effect of Lrp on g c v T : : l a c Z that Lrp would no longer be involved in regulation of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>We propose that these increases are primarily a result of relieving repression from P2 as , when IHF and ModE are bound at dmsA , they would be predicted to occlude the ¹ 35 and ¹ 10 consensus sequences respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>When the HimD protein binds as a homodimer to the ihf site , its specificity for recognition could be ascribed to the same conserved amino acids , since the arm region of one of the HimD subunits now probably lies in the minor groove that is normally occupied by the HimA arm as part of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>To determine if IHF binds to the potential site identiﬁed by mutagenesis above , and to investigate the effect of the protein on the interaction between NagC bound to ONC1 and ONC2 , DNase I footprinting was carried out on a 341 bp PCR generated fragment ( PM ; Figs 1 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>IHF does not participate directly in this site-specific recombination event and its essential function appears to be as an architectural element in facilitating Int activity ( Goodman and Nash , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Also induced were the putative transcriptional-regulator yiaG , induced by AI-2-mediator MqsR [ 27 ] , and ihfA ( himA ) , which together with ihfB ( himD ) , constitutes the global regulator IHF that is required for site-specific recombination , DNA replication , transcriptional control , and fimbriation [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>HU and IHF are small , basic , histone-like proteins which are abundant in the cell and which are involved in a number of cellular processes , such as site-speciﬁc recombination , replication , and transcription ( 8 , 11 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>8ite 11 was altered Irom Ihe consensus 5 ' - ATCAA lo 5 ' - ATCCC ( fimS9 ) , base subslitutions that were expected to greatly lower the affinity 01 IHF lor site 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Irrespective 01 the mechanism by which IHF bound to site 1 acts to promote switching , we lound no convincing evi dence that the site stimulates off-to-on versus on-to-off recombination promoted by FimB , as suggested by one model ( Oove and Dorman , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>For example , IHF has been reported to inhibit transcription when bound to a site that Crude OmpR ( ng ) zu CS-910 scanning densitometer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>An interesting possibility is that Fis bound to site V collaborates with IHF bound at 2114 to increase transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>At present , the mechanism by which IHF bound to the IHF III site stimulates transcription initiation is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>ModE and IHF bind dmsA in the vicinity of the P1 transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>We now show that NarL can displace IHF bound at the IHF I site , but IHF is unable to displace bound NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>NarL displaces IHF bound to site I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Binding of NARL-P to two sites adjacent to the IHF site did not induce bending or modify the apparent bending induced by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>Since the nrf-acs intergenic region contains three IHF sites , we investigated whether IHF bound to the IHF III site functioned independently of IHF bound to other sites within pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Therefore , we concluded that IHF bound to the IHF III site acts independently of IHF bound to other sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>The combined effects of mutations in IHF , NagC and NanR binding sites on ﬁmB expression and of IHF on methylation protection in vivo The results presented above show that IHF , bound to an element situated midway between the NagC operators ONC1 and ONC2 , forms a cooperative complex with NagC in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Taking these results together with those presented above , it is concluded that the cooperative interaction of NagC bound to ONC1 and ONC2 , with IHF bound to the site identiﬁed in this work , forms a nucleoprotein complex that activates ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Considered together , these data show that IHF promotes an interaction between NagC bound to ONC1 and ONC2 , forming a stable nucleoprotein complex that activates ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Examination of this and additional footprints allows us to conclude that IHF stimulates NagC binding to ONC1 at least 10-fold , and that IHF clearly facilitates in vitro the interaction between ONC1 and ONC2 predicted to occur in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Using genetic and biochemical approaches , IHF was shown to-bind to a site midway between ONC1 and ONC2 to facilitate strong cooperative binding of NagC to these elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , binding of IHF to this site considerably enhanced binding of NagC to both ONC1 and ONC2 ( compare lanes 2 -- 5 with 6 -- 9 ) , showing that IHF facilitated cooperative binding of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The ibs mutation described above eliminated IHF binding ( lanes 26 -- 30 ) , and prevented the enhanced binding of NagC to ONC1 , while mutation of the ONC2 site considerably reduced IHF-stimulated binding of NagC to both ONC1 and ONC2 ( compare lanes 6 -- 9 and 16 -- 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , the combination of NagC and IHF prevented Dam methylation of GATCNagC in vitro , and IHF was shown additionally to be required to prevent methylation of GATCNagC in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Mutations in either subunit of IHF resulted in the loss of methylation protection of GATCNagC ( Fig. 2B ) , and thus IHF does facilitate the interaction between NagC bound to ONC1 and ONC2 in vivo as it does in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , neither mutation of ONC1 , nor that of ONC2 , seems to enhance the methylation protection of GATCNanR relative to the wild type in vivo as would be expected if the binding of NagC and IHF inhibited NanR binding ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>To investigate whether IHF bound to the IHF II and IHF III sites inﬂuences this competition , we examined the binding of puriﬁed IHF and NarL to the pnrf53 fragment by using DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Thus , IH bound to the IHF II and IHF III sites does not interfere with the ability of NarL to-bind DNA and displace IHF from IHF I. To test how IHF III inﬂuences nitrite induction at pnrf , the promoter activities of pnrf53 , pnrf53 / 87 , and a pnrf53 fragment carrying the p124Gp123C substitution were measured in the narL narP strain JCB387 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Taken together , these results suggest that , with wild-type pnir sequences and extracts from JCB3884 ( ihfA ) cells , the second , more retarded band is due to the 1 binding of IHF , and that this binding is greatly decreased by the p99G substitution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>To identify putative binding sites for the proteins involved in site-specific recombination , namely IntS , TorI , and IHF , we analyzed the attL and attR sequences , upstream of intS and downstream of torI up to the respective core sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>To determine whether the proteins involved in recombination were actually binding to these predicted sites , we purified IntS , TorI , and IHF near homogeneity and we also verified the absence of contamination of IntS and TorI with IHF by Western blot using IHF antisera ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>Furthermore , the study of the effect of IHF on initiation of transcription on promoters with increased and decreased affinity for Q ~ ~ - RNA polymerase shows that in the former case , IHF does not further enhance transcription , while in the latter case , transcription has become dependent on the presence of IHF ( Fig. 7 ) B - 77 1 -5 p TCGTGCATCACGT-I-ITTGCCGCATCTCGAAAAATCAAGGA NRl-4 IHF - binding site ' ` - 54 -35 Figure 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Although DNase I footprinting revealed a weak binding site in this position , no increase in Pfnr transcription was observed in vitro in the presence of puriﬁed CRP-cAMP ( 0.5 to 1 M ) , either in the presence or in the absence of IHF ( 0.5 M ) ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Our data show that Crp-cAMP binding in vitro ( Fig. 5B , E ) is correlated with reduced transcription in vivo ( Fig. 6 ) , and that RNAP generates a protection pattern at PgltB in the presence of Crp-cAMP ( Fig. 7 , lane 7 ) different from the pattern yielded by RNAP in the presence of IHF + Lrp ( Fig. 3B , lane 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>These results indicate that the two nucleoid proteins , IHF and H-NS , compete with each other for binding to the respective target sequences , and in the presence of high concentrations of both proteins , only the H-NS complexes are formed because of the higher affinity of H-NS than IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Since there are two IHF-binding sites within IHF-2 , the CD3 probe formed multiple species of OmpR -- IHF -- CD3 complex , e.g. IHF -- OmpR -- CD3 and IHF -- IHF -- OmpR -- CD3 , should be formed in the simultaneous presence of two factors , thereby giving a broad distribution of complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In spite of the presence of multiple IHF binding sites in ihfB its auto-regulation , under the conditions examined , is less effective than the autoregulation of ihfA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Fig. 3A shows that the Lrp-binding region yields a subtly altered Lrp-dependent DNAse I footprint in the presence of IHF , though the IHF footprint is not obviously changed by the presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>This extra factor could be the presence of a functional IHF binding site , which is present in the control region of pst ( Spira and Yagil 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>This signal is probably due to the presence of a T ^ C pyr-imidine dimer on the opposite strand , and the increase of that signal was interpreted as an enhanced probability of forming this dimer upon IHF binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>This suggests that the ORE-I binding factor , most probably OxyR , exerts its activating role only in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Third , it stabilizes the TOC in the presence of non-speciﬁc IHF binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>To determine whether occupancy of these DNA sites for IHF affects the binding of CRP , we performed DNase I footprint analysis and compared the signatures of CRP alone or in combination with increasing concentrations of IHF ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>To determine whether the binding of IHF can inﬂuence the binding of FIS to FIS II or FIS III , we subjected pacs444 to DNase I footprint analysis and compared the signatures of FIS alone or in combination with increasing concentrations of IHF ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1095">
When CD6 was used as a probe , both OmpR and IHF bound to the same probe ( Fig. 9b ) , as expected from the map of binding sites ( Fig. 9a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></label>
<input id="act_similarity_1095", type="checkbox">
<div class="ACT_similar">
<li>When CD6 was used as a probe , both RstA and IHF bound to the probe ( Fig. 10a ) as expected from the map of binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li></div>
<li>Murtin , C. , Engelhorn , M. , Geiselmann , J. , and Boccard , F. ( 1998 ) A quantitative UV laser footprinting analysis of the interaction of IHF with speciﬁc binding sites : re-evaluation of the effective concentration of IHF in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>AM three sites show low binding affinity to IHF suggesting that Received 11 April , 1994 ; revised It August , 1994 ; accepted 18 August .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Here , we used electrophoretic mobility shift assays and DNase I footprint analyses to demonstrate that the nucleoid proteins FIS and IHF each bind multiple sites within the acs regulatory region , that FIS competes successfully with CRP for binding to their overlapping and neighbouring sites and that IHF binds independently of either FIS or CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>In Bacteria , an abrupt slope in the NWD curve reveals the presence of a handful of high-scoring binding sites for highly speciﬁc TFs ( e.g. TrpR , LexA ) , whereas a progressive increase of the NWD slope is indicative of global factors , such as CRP , FNR ( Figure 3 ) , Fur , FruR , IHF and FIS ( Supplementary Data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>Thus , the occupation of IHF III both increases the probability that a TOC will form at acsP2 and , as described above , increases the probability that the TOC eventually will become poised , i.e. when the CRP concentration rises to levels that favour binding at CRP II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>S1 and data not shown ) to determine whether the presence of IHF bound at IHF III and CRP bound to CRP II can affect the binding of RNAP and the structure of the open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>While we did not observe a substantial difference in the overall affinity of RNAP for acsP2 , or of CRP for its sites , we can not exclude the possibility that the presence of IHF exclusively at IHF III and the resulting changes in DNA structure might affect the on-and-off rates of binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Together with the observation that IHF can bind simultaneously with CRP , these data suggest a working model in which the binding of IHF to IHF III alters the interaction of one CRP dimer with the other and/or with RNAP itself ( C. M. Beatty , D. Thach , and A. J. Wolfe , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Taken together , these data show that RNAP , CRP and IHF can bind simultaneously at saturating concentrations for their respective speciﬁc binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Simultaneous binding of IHF , CRP and RNAP results in protection between the tandem CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Binding of IHF to low-affinity sites inhibits RNAP binding in the absence of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Moreover , its promoter carries a feature shared by many s - promoters S ( Hengge-Aronis , 2000 ) , namely , the presence of a functional IHF binding site that helps elevate its expression and thereby reduce the expression of AP ( Spira & Yagil , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>]<br></li>
<li>We conclude that IHF has a low , but speciﬁc affinity to the IHF-binding site in the pst promoter and that this binding may cause the increase in the expression of pstS when the cells have entered a strong Pi-starvation phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Although in late-stationaiy cultures the amount of IHF is increased by a factor of 5 to 10 , it is not very likely that the expression of the IHF-activated promoters would change , as the amount of IHF during exponential-growth should be enough for fuK occupancy of the specific IHF-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Differences in the upstream IHF and Fis binding sites at the nir promoter in related enteric bacteria ﬁx the level of nir operon expression under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>This assay , in combination with heparin agarose chromatography , enabled us to purify and characterize one of these binding activities as being IHF , and we show that IHF is an additional component required for maximal in vivo pfl expression under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Consistent with this hypothesis , when the IHF site upstream of aceBAK was inactivated , the response of aceBAK expression to the carbon source closely resembled that of iclR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550439" target="_blank">8550439</a>]<br></li>
<li>We also show that IHF binds downstream from the activator binding site , and that IHF is required for high level psp expression in response to some , but not all , inducing agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>IHF appears to be absolutely necessary for pyruvate induction since no increase in pfl expression was found when the IHF binding site mutant or a himD mutant was grown with pyruvate or in conditions that increase pyruvate endogenously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Although our studies indicate that the predicted upstream site is not an FNR binding site , we found that this sequence binds IHF and increases expression of fnr under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Here it is demonstrated that NagC binding to a second site ( ONC2 ) , located 212 bp closer to ﬁmB , also controls ﬁmB expression , and that integration host factor ( IHF ) , which binds midway between ONC1 and ONC2 , facilitates NagC binding to its two operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>One of these sites overlaps the pho box , suggesting that IHF may inhibit phoA expression by binding to this sequence and interfering with the binding of the phosphorylated PhoB protein , the activator of the PHO genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Indeed opposing effects of IHF on activation by NanR and NagC could explain why mutations that affect IHF function have less effect on ﬁmB expression than do those affecting NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , when NagC binding to its operator sites is disrupted , IHF bound to the regulatory site identiﬁed above has no apparent effect on ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , placing the NanR , IHF and NagC binding sites closer to the ﬁmB promoter enhances the ability of the regulators to activate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Effect of IHF on expression and binding to the Pa and Pz promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>This suggests that IHF binding to ibs has a small inhibitory effect on the ability of NanR to activate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>IHF therefore does seem to have a small activating effect on ﬁmB expression in the absence of both NagC and NanR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>In the absence of NagC binding , a residual activating effect of IHF on ﬁmB expression could thus be offset by an inhibi-tory effect of IHF on NanR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Hence , evolution appears to have adjusted the binding of IHF to the two targets in order to set the levels of anaerobic nir operon expression in the absence of NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Here we show that pnrf activity is stimulated by IHF binding to the IHF III site , but only in the absence of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>This indicates that NarL/NarP can compensate for the loss of IHF III and that the effect of IHF III is only observed in the absence of NarL/NarP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Thus , different levels of EnvZ in IHF mutants could lead to changes in the binding properties of OmpR in MH760 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Because there are no known IHF-binding sequences proximal to the hsp promoters , the effect of IHF overproduction on Hsp induction most likely results from increased levels of s32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Most strikingly , IHF binding very strongly increased the extent of the intrinsic sequence-induced bend ( Figs. 2 , 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>This residual activity has been reported to be mediated by ArcA ( 35 ) , and the arcA , himD , and IHF binding site mutants have very similar phenotypes in relation to anaerobic induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>This inhibition appears to be directly mediated by IHF binding to the ompB promoter , since a site-specific mutation in the IHF-C binding site prevents the negative action of IHF on in vivo ompB promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>negative effect of IHF found in vivo and that this effect is probably mediated primarily by IHF binding to the IHF-C site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>On the basis of these results , we propose that when IHF binds to its binding site , it has a negative effect on tyrosine-mediated repression which is relieved in +4 and himA mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961453" target="_blank">7961453</a>]<br></li>
<li>Interaction of IHF with this site might inhibit osmEp , a situation analogous to that of the osmotically regulated gene ompC whiob is inhibited by IHF binding to a site upstream ftom its promo-ter ( Huang efaf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li>
<li>Together these results demonstrate that binding of IHF to its target site about 200 bp upstream of the Dam methyl-ation site in the carAB operon is required to efficiently inhibit the modification of this site in vivo in the pre-sence of Carp and pyrimidines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Binding of IHF to its site does not increase the binding of 054-RNA polymerase to the glnHp2 promoter ; instead , our data suggest that IHF bends the DNA to align the activator with the closed c 54-RNA polymerase prom oter complex to facilitate the interactions that result in open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>We now confirm the stimulatory effect of IHF by using sitespecific mutagenesis to alter the IHF-binding site and by examining the effects of these alterations on the ability of THF to-bind to the site , to stimulate the initiation of transcription , and to enhance open RNA polymerase-promoter complex formation in intact cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>This activation of open complex formation was greatly reduced by the mutations in the IHF-binding site , indicating that IHF plays an important role in the activation of glnHp2 transcription in response to nitrogen deprivation in intact cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>When the binding site of the NR1 activator of the gInHPQ operon was moved several hundred base pairs further upstream , it could still activate open complex formation , but in this case IHF inhibited the activation ( Claverie-Martin and Magasanik , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>( A ) Binding and electrophoresis were as described in the text ; ( B ) relative mobilities of retarded fragments plotted as described in the legend to Fig. 2B This model for activation by the concerted binding of IHF and NARL-P to the narG promoter is similar to that proposed for the catabolite activator protein ( CAP ) - dependent activation of the maltose operons by MALT ( 12 ) in which speciﬁc CAP-induced bending is thought to promote a critical interaction between MALT proteins bound to sequences on either side of the bending site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>Therefore , we were surprised to find that the upstream activation by IHF is independent of the spatial orientation of these two sites , since it is difficult to reconcile this observation with either model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>IHF enhances psp operon transcription ( 20 , 46 ) and facilitates binding of PspF to its upstream activation sequences in the psp operon regulatory region , autogenously down-regulat-ing pspF transcription ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>IHF works to enhance transcriptional activation of the psp operon ( 20 , 46 ) and facilitates the binding of PspF to upstream activation sequences I and II in the psp operon regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>One possibility , given the proximity of the Crp and IHF binding sites ( Fig. 2A ) , and the requirement of IHF for activation by Lrp [ 14 ] , is that Crp interferes with IHF binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>IHF binding is required for Lrp-dependent activation of the gltBDF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Since the proximal Lrp binding site in gltBDF operon is positioned 110 bp upstream of the 235 hexamer , an intermediate IHF binding site appears to be essential to Lrpmediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">IHF protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>In order to directly measure the occupancy of speci ® c ihf sites in vivo , we irradiated cells containing plasmids that carry any one of the ihf sites used here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>lla , iHF has direct contact with a specific domain of RNAP , which promotes closed complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>However , when these lysates are mixed , a specific ihf binding activity is observed ( Figure 3 ) , indicating that the lysates contain HimA and HimD subunits that are properly folded to form an active heterodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>GCATGCCGIGGACGCAGGCATGTTTCTCAATAACGAAATTTGATAAAATCCCGCTCTTTCATAACATTATTTCAGCCTTCTTCAGGGCTGACTGTTTGCAT P2-10 P3-35 p3-10 AAAAATTCATCTGTATG CACAATAATGTTGTATCAACCACCATATCGGGTGACTTATGCGAAGCTCGGCTAAGCAAGAAGAACTAGTTAAAGCATTTAAA ARG ARG M R S S A K Q E E L V K A F K IHF G CATTACTTAAAGAAGAG AAATTTAGCTCCCAGGG CGAAATCGTCGCCGCGTTGCAGGAGCAAGGCTTTGACAATATTAATCAGTCTAAAGTCTCG CGGA A L L K E E K F S S Q G E I V A A L Q E Q G F D N T N Q S K V S R M cer - like TGTTGACCAAGTTTGGTGCTGTACGTACACGCAATGCCAAAATGGAAATGGTTTACTGCCTGCCAGCTGAACTGGGTGTACCAACCACCTCCAGTCCATT L T KA F G A V R T R N A K M E M V Y C L P A E L G V P T T S S P L GAAGAATCTGGTGCTGGATATCGACTACAACGATGCAGTTGTCGTGATTCATACCAGCCCTGGCGCGGCGCAGTTAATTGCTCGCCTGCTGGACTCACTG K N L V L D I D Y N D A V V V I H T S P G A A Q L I A R L L. D S L 1 101 201 3 4 01 TAGAGCTGTTCGACCAGGAGCTTTAATCTCTG CCCCGTCGTTTCTGACGGCGGGGAAAATGTTGCTTATCCCTCTCAACCCCCTGCTTTCCCCTGCG ATT E L F D Q E L AATTTAACGAATAGTGCGTTTTACTG CGACATGTCATTCACACAATGAATACATAAGGTAAAAAAAGCACATTATGCAAAATTCATTATCTAATTGAAAA AACTAGAATTAACGATAAATAACCGTATTTTTAATTCTTTTTTGTTATTAAAATTCACATTTTTAACACTTAGTATCAACTO AAACAGTTAGCGCGGTAT TAAT7 ` AGCTCAATAATTAGTGTATAC 6 residue and the other a potential 70 amino-acid residue protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor biosynthesis , and DNA-RNA metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative stress.6,11-14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Cluster four ( osmoregulated porin gene ) contains the global regulators IHF and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15603590" target="_blank">15603590</a>]<br></li>
<li>In the case of the IS1 site a second signal , at a G residue located 8 nt downstream of the pentamer signal , also increased in the presence of IHF ( indicated by a star in Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Two IHF footprints are seen within this site : one at the G residue of the consensus pentamer ( indicated by the arrow ) , and another one located 8 nt downstream ( indicated by a star ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>It is of interest to note the occurrence of a DNA cleavage site at or adjacent to a thymine residue at base pair position +141 and a cytidine residue at base pair position +158 on the nontranscribed DNA strand in the presence of IHF ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The methylation of a guanine residue at base pair position +150 is enhanced in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The N7 major groove-specific methylation of 2 guanine residues , at base pair positions +131 and +133 , and the N3 minor groove-specific methylation of a single adenine residue , at base pair position +132 , are enhanced by IHF addition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Fig. 3 summarizes the positions of purine residues that were either protected from methylation or showed an enhanced methylation pattern in the presence of IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Three adenine residues at base pair positions +134 , +137 , and +138 are protected from methylation in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The degree of protection or hyperreactivity of individual residues within the same IHF binding site can differ signi ® cantly , making subjective the choice of the residue for a standardized comparison ( see for example Figure 4 of Yang & Nash , 1995 ) ; ( ii ) the footprinting procedure is identical in vitro and in vivo and it is therefore possible to compare the primer extension pro ® les obtained in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF protection of deoxyribose residues in the ilu-GMEDA leader .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Such a domain was identified by Giladi etai -LCB- 1992b ) as mutant RNAP , in which the C-terminal portion of the f : / - SLibunit has been deleted , could no longer be activated by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The involvement of the same domain ot RNAP in IHF-mediated tianscription activation makes a direct contact between IHF and RNAP very likely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Bending or looping models do not rule out the possibility of direct RNAP-IHF contacts occurring as well [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>IHF does not participate directly in this site-specific recombination event and its essential function appears to be as an architectural element in facilitating Int activity ( Goodman and Nash , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Whereas IHF binding lo site 11 in all likelihood lullils a direct structural role in recombination , the function 01 sile 1 appears less obvious .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>We note the parallel with integration host factor ( IHF ) - facilitated Tn10 transposition , in which it has been proposed that during the transposition process IHF is ejected from a supercoiled loop and subsequently rebinds after rotation of the loop .31 It is notable that the upstream contact of RNAP is located within FIS site III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12054813" target="_blank">12054813</a>]<br></li>
<li>Alterations in the pR0 promoter of coliphage lambda modify both its activity and interaction with the integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Abbreviations used : OmpF , outer-membrane-protein F ; OmpC , outer-membrane-protein C ; Lrp , leucine-responsive protein ; NO , nitric oxide ; IHF , integration host factor ; HTH , helix-turn-helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Abbreviations used : IHF , integration host factor ; RNAP , RNA polymerase ; HTH , helix-turn-helix ; EBP , enhancer binding protein ; UAS , upstream activating sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>IHF , one of the histone-like proteins , is also a dimeric protein composed of two different subunits with 32 % of their amino acids being identical ( for a review see Driica and Rouviere-Yaniv , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984109" target="_blank">7984109</a>]<br></li>
<li>When IHF sites appear , they are always located between the polymerase and activator elements , as would be required if they assisted in bringing the two together by enhancing loop formation ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The function of IHF is probably fo stabilize this loop formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Combined , these elements indicate that fullrange transcriptional control of the carAB operon by arginine , pyrimidines and purines requires the elaboration of a speci ® c nucleoprotein structure comprising at least three speci ® c sensors ( ArgR , PyrH and PurR ) and two architectural elements ( IHF and PepA/CarP ) , which interact by establishing protein-DNA and protein-protein contacts in a complex interplay that requires also a pronounced susceptibility of the extended control region ( about 400 bp ) to undergo structural distortion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li>
<li>In order directly to investigate this possibility and to minimize possible contributions of direct protein-protein contacts , we reconstructed promoters that replace the CRP binding sites with recognition sites for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Interaction with other proteins Although this study revealed fractional occupancy for all sites tested , these sites were taken out of their context and it is possible that in some instances a cooperative interaction with another protein might stabilize IHF binding , and consequently displace the binding equilibrium of this particular site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>We have analyzed the in vitro and in vivo interaction of IHF with three speci ® c binding sites using UV laser footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>For these reasons , we have initiated a molecular analysis of the in vivo interaction of the integration host factor ( IHF ) with its speci ® c binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The replacement of TIO and Tll by two G residues , though resulting in a 50-fold decrease of the affinity , did no completely abolish the IHF specific binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>The corresponding region of P4 is AT rich and has a number of possible IHF binding motifs ( 13 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7511583" target="_blank">7511583</a>]<br></li>
<li>The pspG promoter region contains regions predicted to bind PspF , IHF , and 54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Within this dA-t-dT-rich region , a stretch of three dA residues located 9bp from the 5 ' site of the consensus of the ihf site in the regulatory region of phage Mu ( see also below ) was shown to be of paiiicuiar importance for IHF binding , as substitution of two of these dA residues for dC residues reduced the stability of the IHF-DNA complex at least 10-fold ( van Rijn e ( a / , , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>In the attR region , two IntS arm-type and one IHF binding sites were mapped ( Figs. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>This mutation lay within one of the two putative IHF binding sites that are located at 62 to 50 and 37 to 25 , between the AtoC and RNA polymerase binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>( ii ) DNase I protection studies revealed three IHF binding sites in the ompB pro-moter-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>We report here that IHF specifically protects two sites within the ilvGMEDA promoter-regulatory region against DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>The ompF regulatory region also contains two recognition sites for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>If differential evolution of HimA and HimD has indeed led to an activation domain in IHF , this would predict that the orientation of the IHF-binding site with respect to the promoter is the same in the three opetons that have been shown to be directly activated by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>This activation of open complex formation was greatly reduced by the mutations in the IHF-binding site , indicating that IHF plays an important role in the activation of glnHp2 transcription in response to nitrogen deprivation in intact cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>IHF binding at this site forms a nucleoprotein complex that alters the structure of a supercoiled DNA template in a way that unwinds the DNA helix in the 210 region of the downstream promoter and facilitates open complex formation ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068661" target="_blank">9068661</a>]<br></li>
<li>We suggest , therefore , that IHF activates transcription by forming a higher-order protein-DNA complex in the UAS1 region that structurally alters the DNA helix in a way that facilitates open complex formation at the downstream ilvPG promoter site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>We propose that IHF forms a higher-order pro-tein-DNA complex in the UAS1 region that structurally alters the DNA helix at the downstream ilVPG promoter site in a way that facilitates open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>This observation suggested that , unlike most activator proteins , IHF might not require a specific protein interaction with RNA polymerase for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Nevertheless , the results of these experiments suggest that the IHF-mediated activation of transcription from the ilvPG promoter does not require specific protein contacts between IHF and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>However , we have shown that IHF interacts with the bases and the sugar phosphate backbone of the DNA helix in the iZvPo1 promoter region on the opposite face of the DNA helix covered by RNA polymerase and that no DNA contacts are shared by these two molecules ( Winkelman & Hatfield , 1990 ; Winkelman , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>The narGHJl regulatory region appears to form a highly ordered structure where Fnr enables RNA polymerase to make essential contacts with the zone upstream from - 7 9 and where IHF bends the DNA in such a way that bound NarL-Phosphate can interact with one another and/or with the transcription initiation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>We suggest either that NagC contacts RNA polymerase directly ( or another regulatory protein bound closer to the promoter ) to enhance transcription-activation , or that the nucleoprotein complex that includes NagC and IHF alters the DNA structure nearer the promoter in such a way as to somehow facilitate transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>As this IHF binding site is located immediately upstream of the UP element , the binding of IHP may also interfere with the UP element -- aCTD interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>We suggest that IHF site 1 is ideally located to allow this interaction to take place to facilitate transcriptional activation of the norVW promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Alternatively , the IHF-induced altered DNA conformation in the tandem promoter region might bring IHF and RNA polymerase into close proxi-mity and transcriptional activation could be the consequence of an IHF-RNA polymerase interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>The binding of IHF to the folA promoter is presumably required for creating an ideal DNA conformation , which facilitates TyrR-RNA polymerase interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557822" target="_blank">17557822</a>]<br></li>
<li>IHF site 1 is located between the σ 54 promoter and the norV-proximal NorR binding site ( NorR1 ) consistent with a role in facilitating an interaction between DNA-bound NorR and σ 54-RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Speciﬁc interaction of IHF with the dmsA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>A similar pattern of DNA cleavage at a thymine residue was observed in the Me & O4 protection pattern for IHF interaction at the upstream ilvPo1 promoter region target site ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>This activation is modulated by the upstream binding of nucleoid-associated proteins , integration host factor ( IHF ) and Fis ( factor for inversion stimulation ) ( 3 ) , two DNA-binding proteins that play a role in shaping the folded bacterial chromosome ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Introduction Integration host factor ( IHF ) of Escherictiia coti is a small , basic , sequence-specific DNA-hinding protein composed of two non-identical subunits a and \ \ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Taken together , these results suggest that the function of IHF in the orrpB promoterregulatory region is quite complicated and involves interaction of IHF with a number of other DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>In the second observation , a region of the DNA-binding domain of IHF and the eukaryotic general transcription factor TFIID was found to be similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>A similar architecture has also been described in the formation of the intasome complex in bacteriophage λ , in which a bivalent Int protein contacts two distal DNA sites with an IHF molecule protein-binding in the intervening region and bending the DNA ( Segall et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Direct contact between these two proteins requires formation of a DNA loop between the UAS and theRNA polymerase at PC.This contact formatiocnould be facilitatedby binding of IHF between the UAS and PCwhich is assumed to cause DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>Footprinting studies have shown that , unlike most other sequence-specific-DNA-binding proteins , IHF primarily contacts the minor groove of DNA ( Yang & Nash : 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>However , IHF and HU share extensive sequence homology , especially in their DNA-binding domains , and both proteins are believed to interact with DNA in the minor groove ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>However , the structure shows that , in fact , the DNA-binding residues of IHF contact mostly the phosphodiester backbone rather than bases of the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>However , most of the IHF contacts on this DNA strand are with adenine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Based on nucleotide and amino acid comparisons , IHF is assumed to be structurally related to the bacterial histone-like protein HU , a type II DNA-binding protein for which the crystal structure has been determined ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>According to the model proposed by Yang and Nash ( 1989 ) , the DNA-binding domains of the IHF dimer are located in the C-terminal parts of the two subunits : the $ - sheet arms and possibly also the C-terminal a-helices ( a3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The binding specificity is established by different domains of each subunit ; the arm of the HimA protein participates i ted in the far upstream located IHF target site [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>In this paper , we have shown that an IHF fusion protein with two C-terminal regions of HimA binds much better to the DNA than a fusion protein with two C-terminal regions of HimD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>IHF is a site-specific DNA-binding and - bending protein which alters DNA topoiogy to facilitate protein-protein interaction ( Freundlich etat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>How does binding of IHF some 200 bp upstream affect the particular protein-DNA interaction at this site ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>We tried to make IHF fusion proteins that still contain the N-terminal dimerization domains of both HimA and HimD subunits , but the DNA-binding domains of HimA alone or HimD alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Apparently , an IHF protein which has its dimerization part inverted with respect to its binding domain can still stably bind the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Next , we tested the specific binding capacity of an IHF protein with the DNA-binding domains of HiniA alone by performing gel retardation with a HimA-HimDA heterodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Similarity between the DNA-binding domains of IHF protein and TFIID protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>By domain swapping between HimA and HimD , we have constructed an IHF fusion protein which has the putative DNA-binding domains of only HimA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>A comparable fusion protein with only the DNA-binding domains of HimD forms less stable complexes , suggesting that sequence-specific contacts betweeQ IHF and the ihf consensus are mainly provided by the HlimA subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The failure of HimA to form stable protein-DNA complexes , however , is probably due to the inability to form stable homodimers and not to the lack of capacity to bind the DNA , since construction of an IHF fusion protein which contains only the putative DNA-binding domains of HimA and not those of HimD did specifically bind to the ihf site with a high efficiency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Like IHF , HU employs the intercalation of two conserved proline residues that are inserted into the minor groove 9 bp apart and induce or stabilize DNA bending ( Fig. 2D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_83">
DNA looping generated by DNA bending protein IHF and the two domains of lambda integrase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8106311" target="_blank">8106311</a>]<br></label>
<input id="dom_similarity_83", type="checkbox">
<div class="DOM_similar">
<li>DNA looping generated by DNA bending protein IHF and two domains of lambda integrase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li></div>
<li>Cold Spring Harbor Symp Quant Biol 43:1121 -1126 Moitoso de Vargas L , Kim S , Landy A ( 1989 ) DNA looping generated by DNA bending protein IHF and the two domains of lambda integrase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>J Biol Chem 263 : 1008-1012 Moitoso de Vargas L , Kim S , Landy A ( 1989 ) DNA looping generated by DNA bending protein IHF and the two domains of lambda integrase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>The isolation of IHF mutants with altered , expanded DNA-binding specificities [ 14 ] and a genetic and biochemical analysis of the ~ - subunit of IHF [ 15 ] support this model and suggest that in contrast to the classical helix-turn-helix DNA-binding motif , the binding of IHF is not restricted to one domain , but requires the co-operative participation of several regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Thus , IHF II must be correctly positioned on one face of the DNA helix for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Therefore , since Me2S04 methylates the N7 position of guanine residues in the major groove of the DNA helix and the N3 position of adenine residues in the minor groove ( 52 ) , these results show that IHF spans two adjacent major grooves on the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The pattern of methylation of this strand of the DNA also suggests that IHF interacts with adjacent major and minor grooves on one face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>c A162T R24S Assays were carried out at 37 uC in 10 ml reactions containing the indicated amount of activator with 20 nM DNA template , 25 nM RNA polymerase core enzyme , 100 nM s protein , 4 mM ATP , 54 40 nM IHF protein , in Tris/acetate buffer ( 25 mM Tris/acetate , pH 7 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The presence of architectural factors , like IHF , is often required fo the optimal contact of the ELE-bound NtrC-type transcription factor with the promoter-bound 54 RNA polymerase ( 11 , 26 , 29 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>In these promoters , IHF stimulation is face-of-the-helix dependent ( 16 , 52 ) and requires the C-terminal domain of the asubunit ( a-CTD ) of RNA polymerase ( 17 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>We would hypothesize that the TyrR hexamer binds across both boxes and , with the assistance of IHF , produces a complex in which the N-terminal domain of one or more of the TyrR molecules is able to interact with the subunit of RNA polymerase and activate transcription ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557822" target="_blank">17557822</a>]<br></li>
<li>These results indicate that direct interactions between the N-terminal domain of TyrR and the C-terminal region of the a subunit of RNA polymerase are required for TyrR to counteract IHF - or HU-mediated transcription inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>( b ) Activation factors of protein mediated DNA ( b ) Template Activation factor DNA-binding protein H159L CRP Supercoiled CC-55 .5 Supercoiled IC-55 Linear CC-55 .5 Linear IC-55 1.6 3.8 IHF CRPH159L bending in vitro , on supercoiled or linear templates with aCTD ( C-terminal domain of the a subunit of RNA polymerase ) - truncated Es54 in the absence or presence of CRP or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Recall that at the phage PL promoter , IHF bends upstream DNA and allows the C-terminal domain of the subunit of RNA polymerase to contact a distal UP element ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">IHF evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>A similar distribution was observed in the DnaA , ArgR , and Ihf families ( where a high proportion in small genomes was observed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>These results are similar to those of Compan and Touati ( 1993 ) , who reported an increase in sodA-lacZ activity of about 40 % in an IHF mutant strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>Two additional regions ﬂanking ABS3 exhibit a higher degree of conservation and show similarity to binding sites of IHF , which are represented by the consensus WATCARXXXXTTR ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Site I and site 11 have similarity scores ( S.S. ) of 62 and 60 , respec tively , which are well within the range of known IHF sites ( 76.6-46 .2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>In addition to 54 , expression requires IHF and PrpR , which is homologous to NRI ( 122 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>IHF Integration host factor ( IHF ) is composed of two subunits , which share about 25 % homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Site-specific IHF binding IHF is a heterodimeric protein consisting of the subunits HimA ( / Wr 11 350 ) and HimD ( H 10 650 ) that share about 25 % amino acid identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF as an asymmetric transcription factor in contrast to HU , the subunits of IHF show only a mode-rate ( 25 % ) identity at the amino acid level , indicating considerable evolutionary divergence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>cReported TBLASTN identity to IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>IHF and HU monomers share 30 % sequence identity and are related in both structure and function ( 20,31,33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>As the amino acid sequence of the subunits of IHF and those of the HU family are very similar , Yang and Nash ( 1989 ) proposed a model for the IHF-DNA interaction in which the crystal structure of HU from Bacillus stearothermophilus ( Tanaka et ai , 1984 ) was used to approximate the structure of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>HU and IHF have a similar structure consisting of a globular body from which two - ribbon arms protrude ( Fig. 2D and E ) and share about 30 -- 40 % homology ( Swinger et al. , 2003 ) HU binds non-speciWcally to DNA , but has a higher aYnity for supercoiled DNA and distorted DNA structures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>IHF is a heterodimer consisting of two homologous ( 30 % similarity ) subunits , IHF - and IHF - ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>HU/IHF HU and its sequence speciﬁc homologue IHF consist of two ( identical or homologous ) subunits ( Swinger and Rice , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15853876" target="_blank">15853876</a>]<br></li>
<li>However , unlike other members of this family which have no or little sequence specificity for DNA binding , IHF binds to the 5 ' - AAT-CAAN4TTA / G-3 ' consensus sequence ; flanking sequences contribute significantly to this binding [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>B himA : + hip : - f-pDRC158 : ¬ IFI'G : 1 -60 - - 4 0 * - 4 0 * - 8 0 * - 2 0 * Discussion IHF belongs to a group of histone-like proteins which , together with HU , H-NS and Fis , appear to participate in the three-dimensional organization of the bacterial chromosome ( Driica and Rouviere-Yaniv , 1987 ; Schmid , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Examples of members of this family include HU , IHF , H-NS , and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1459953" target="_blank">1459953</a>]<br></li>
<li>Fis is a member of the general family of nucleoid-associated proteins that includes HU , IHF , H-NS , LRP , and Dps ( 2 , 35 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768815" target="_blank">7768815</a>]<br></li>
<li>Instead Dan is rather similar to those of major nucleoid proteins such as HU , IHF , H-NS and Fis ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>It has been suggested that this may be due to the partial functional substitution of IHF by other members of the histone-like family [ 11 , 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>IHF is considered to be a member of the E. coli histone-like family of proteins because it can compact DNA and because each of its Received 29 April , 1992 , revised and accepted 18 May .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Integration host factor ( IHF ) is a member of a family of DNA-bending proteins that share amino acid similarity with the bacterial HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Integration host factor ( IHF ) from Escherichia coli is a member of the histone-like family of proteins [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>IHF is a small heterodimeric protein that belongs to a class of prokaryotic histone-like proteins , together with HU , H-NS and FIS ( Schmid , 1990 ; Schmid & Johnson , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF is a heterodimeric protein that belongs to a class of prokaryotic histone-like proteins and is implicated in many cellular processes involving DNA ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>IHF belongs to a family of bacterial histone-like DNA binding proteins that facilitate the assembly of ordered protein-DNA structures by bringing distant DNA sequences , or proteins bound at distant DNA sites , into close proximity to one another ( for a review , see Schmid , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>IHF belongs to the family of historic-like proteins , able to wrap DNA into higher order structures [ 6 -- 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by IHF</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>This mutant promoter fusion ( pacs236 ihf III ) , and all others used in this study , was recombined into a hybrid l bacteriophage and introduced as a monolysogen into AJW678 , a lac strain that is wild-type ( WT ) for both acetate-metabolism and crp ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Pawinskiego5A , 02-106 Warsaw , Poland ( Received 3 July 1998 ; accepted 5 October 1998 ) Abstract - In attempts to identify subunit-specific phenotypes of ihf mutants we analyzed viability , thermoresistance and protein-synthesis patterns in ihfA and ihfB mutants and their respective parental strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Rifampicin-resistant , DnaA-dependent initiation of replication has also been observed in other genetic backgrounds such as ihf mutants ( von Freiesleben et al , 2000b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16041320" target="_blank">16041320</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_4">
prpR , ntrA , and ihf functions are required for expression of the prpBCDE operon , encoding enzymes that catabolize propionate in Salmonella enterica serovar typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></label>
<input id="rp_similarity_4", type="checkbox">
<div class="RP_similar">
<li>prpR , ntrA , and ihf functions are required for expression of the prpBCDE operon , encoding enzymes that catabolize propionate in Salmonella enterica serovar Typhi-murium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>prpR , ntrA , and ihf functions are required for expression of the prpBCDE operon , encoding enzymes that catabolize propionate in Salmonella enterica serovar Typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li></div>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage-shock protein Regulator ; phage-shock protein Regulator ; phage-shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate-biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Functionb Other regulator ( s ) Reference ( s ) Rich Minimal Medium Medium No 15 12 Yes 24 18 No 6 6 ybaY gabP ugpE Unknown GABA-transporter protein b0453 b2663 b3451 GabC ( ) , Nac ( ) PhoBR ( ) 55 , 66 , 93 15 G-3-P transporter membrane bound component ATP binding protein of G-3-P transporter Putative binding protein of ABC transporter Hypothetical ATP-binding protein of ABC transporter Beta subunit of nitrate reductase-II Subunit of cytochrome bd-II oxidase Lysine decarboxylase Induced by alkylating agents ; protects DNA from alkylation damage Hydroperoxidase II Universal stress protein B ; required for ethanol resistance Periplasmic protein induced by high osmolarity ; function unknown Multidrug efﬂux pump No 20 9 Yes 6 2 No 28 21 ugpC ydcS yehX b3450 b1440 b2129 PhoBR ( ) Nac ( ) 15 93 narY appB IdcC aidB b1467 b0979 b0186 b4187 13 23 Yes Yes Yes Yes 10 25 7 36 2 7 AppR ( ) 47 , 79 45 , 46 11 33 Lrp ( ) Yes 40 66 Yes 4 8 katE uspB b1732 b3494 74 26 Yes 35 64 osmY Lrp ( ) , Crp ( ) , IHF ( ) EvgA ( ) , Lrp ( ) b4376 20 , 90 , 91 No 31 yhiV b3514 56 , 57 , 74 , 78 56 , 57 , 78 25 72 , 74 12 77 24 4 yhiU mltB otsA ecnB argH aroM Multidrug efﬂux pump EvgA ( ) b3513 b2701 b1896 8 2 Yes No Yes Yes No No 39 2 31 23 2 15 Membrane-bound lytic transglycosylase B Trehalose phosphate synthase Bacteriolytic lipoprotein entericidin B Argino-succinate lyase Unknown ; may play a role in aromatic-amino-acid biosynthesis Putative lipase 2,5-diketo-D-gluconic acid reductase A ; catalyzes the conversion of 2,5-KGDR to 2-KLG Aldehyde dehydrogenase B ; converts lactaldehyde and NAD to lactate and NADH Succinate-semialdehyde dehydrogenase Membrane-bound subunit of carbonphosphorus lyase complex Transaldolase A ; converts Sed-7-P and Gly-3-P to Fru-6-P and Ery-4-P Putative glutathione S-transferase Putative dehydrogenase Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Lrp ( ) 57 74 5 78 ( ) EnvZ/OmpR ArgR ( ) TyrR ( ) b3960 b0390 No 15 No 8 yhjY yqhE b3548 b3012 9 92 11 aldB 22 Yes 21 b3588 74 , 87 gabD phnP B2661 B4092 16 8 No No GabC ( ) , Nac ( ) PhnF and PhnO ( ) 55 , 93 51 13 2 talA B2464 26 20 No CreBC ( ) 7 , 70 yfcG yliI yjbJ yjbE ygaU ygdI ygaF yjgR ydaM ydcK yhiN yebF yjiN yeaG yodC ybiO yhjG yphA yhjD B2302 B0837 B4045 B4026 B2665 B2809 B2660 B4263 B1341 B1428 B3492 B1847 B4336 B1783 B1957 B0808 B3524 B2543 B3522 52 37 37 17 22 22 4 7 3 6 16 15 5 41 4 13 41 13 17 24 5 No Yes No No No No No No No No No No No Yes No No No No Yes SdiA ( ) 84 66 179 58 12 5 7 3 23 9 7 25 6 10 17 26 15 18 34 Lrp ( ) 74 63 , 74 Lrp ( ) Lrp ( ) 63 a Genes were identiﬁed using the BLAST algorithm ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>FNR-binding sitesc Other reported regulation Y/N Y/Y Y/N CRP-activated , NarLrepressed 46 47 48 Half-sites 63.5 , and 39.5 - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant and promoter truncations - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter truncations , mutagenesis , and footprinting Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis and EMSAd - Galactosidase assay in fnr mutant , promoter mutagenesis and footprinting - Galactosidase assay in fnr mutant , promoter mutagenesis , EMSA , and footprinting - Galactosidase assay in fnr mutant - Galactosidase assay in fnr mutant Bioinformatic evidence only - Galactosidase assay in fnr mutant , promoter mutagenesis , and EMSA ArcA - and DcuSR-regulated ModE-repressed 33.5 , 113.5 moaABCDE Molybdopterin-biosynthesis 0.50 1.06 0.34 1.52 0.43 0.97 0.59 hcp-hcr dmsABC Hybrid cluster protein and NADPH oxidoreductase Anaerobic dimethyl sulfoxide reductase 73.5 41.5 Y/Y Y/Y NarL - and NarP-regulated ModE-activated , NarLrepressed 0.72 1.12 0.66 30.26 1.60 47.81 2.54 0.12 0.27 0.10 0.10 0.07 0.70 0.07 49 50 -- 52 focA-pflB Formate transporter and pyruvate-formate lyase 0.24 0.30 0.34 0.35 0.22 0.87 0.31 41.5 Y/Y ArcA-activated 53 -- 55 narK Nitrate/nitrite-transport protein 0.20 0.25 0.14 14.30 0.08 12.07 0.12 41.5 Y/Y NarL-activated 56 , 57 41.5 Y/Y narGHJI 0.07 0.07 0.11 33.98 0.14 3.91 0.14 Nitrate reductase A NarL-activated 58 , 59 Y/Y fdnGHI 0.33 0.40 0.20 15.19 0.15 2.16 0.13 Formate dehydrogenase N 42.5 and 97.5 NarL-activated 55 , 60 fliZY Putative cell density - 1.08 0.72 0.23 1.34 0.32 0.62 0.08 responsive regulator of 28 napFDAGHBC Periplasmic nitrate reductase 0.51 0.47 0.39 0.95 0.27 4.04 0.26 No predicted ts 64.5 FlhDC-regulated NarL-repressed , N activated Y/Y Y/N 61 30 arP ¬ glpABC yfiD 0.19 0.16 0.16 0.55 0.21 0.71 0.18 0.07 0.02 0.04 0.66 0.06 1.22 0.08 Glycerol-3-phosphate dehydrogenase Putative formate acetyltransferase 41.5 N/N Y/Y GlpR-repressed , CRP-activated PdhR-repressed 62 40.5 and 93.5 63 , 64 65 N/N hypBCDE 0.20 0.15 0.21 0.59 0.22 0.67 0.19 Hydrogenase maturation Half-sites ansB 41.5 Y/Y Periplasmic L-asparaginase II 0.01 0.02 0.01 0.25 0.01 0.63 0.01 CRP-activated 55 , 66 41.5 Y/Y nirBDC 0.74 0.48 0.38 9.99 0.23 8.80 0.38 Nitrite reductase NarL/NarP-activated , IHF - , Fis - , and H-NS-regulated 10 , 67 0.50 0.29 0.19 0.03 0.80 0.63 0.65 4.73 0.40 0.24 0.10 0.03 1.11 0.29 0.33 0.32 0.53 0.31 0.14 0.05 0.20 0.78 0.22 0.05 0.28 1.38 0.11 0.05 Ferrous iron-transport Nickel transporter Putative cytochrome c peroxidase Periplasmic nitrite reductase No reported ts 41.5 61.5 41.5 Y/N Y/N Y/Y Y/Y Fur-repressed NikR-repressed feoAB 68 69 nikABCDE yhjA nrfABCDEFG 61 10 , 70 NarP-activated and NarLregulated TABLE 2 -- continued Ratiosa Position of FNR siteb FNR-binding Other reported regulation Product of operon Operon Evidence of FNR regulation Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>91.5 78 N/N glpTQ 0.46 0.28 0.30 0.64 0.35 0.68 0.34 GlpR-repressed , CRP-activated 82.5 5 Y/Y 79 N/N Y/Y ArcA - and NarL-activated ArcA-activated 80 ndh 81 Y/Y 0.30 0.99 6.60 1.21 6.16 1.18 5.80 ArcA-regulated acnA nrdAB hmpA Aconitate hydrase 1 Y/Y N/N Y/Y CRP-activated , ArcA-repressed Proposed anti-activation in stationary phase 2.58 4.16 75.91 1.43 2.23 20.42 2.51 3.79 3.02 1.36 5.99 1.68 2.02 6.32 2.04 1.18 1.73 29.89 2.44 3.04 149.7 40.5 82 75 Ribonucleoside diphosphate reductase 1 Flavohemoglobin H alf-site 5.5 37 , 85 Half-sites Fur-repressed 86 N/N IHF - and ArcA-activated ModE-activated ArcA-repressed N/N Y/Y N/N 57 , 87 83 88 83 , 84 Y/Y 89 Y/N IHF - , Lrp - , and YeiL-activated 22 N/N NorR-activated , NarL/NarP-repressed 73 N/Y IHF-repressed nitrite ) and strain ( wild type or fnr ) compared with transcript TABLE 3 Operons that display FNR regulation and that have putative FNR-binding sites upstream Ratiosa O2 O Ø 2 2 NO3 NO3 ] NO2 NO2 WT fnr fnr WT fnr WT fnr Location of putative TS FNR FNR site ( s ) b determined site IDsc Operon Function of operon Putative FNR-activated operons caiTABCDE Carnitine metabolism fixABCX Carnitine metabolism allB Allantoinase ybcW Hypothetical protein fes-entF-fepE Ferric enterobactin-synthesis and transport Citrate lyase Hypothetical protein Hypothetical protein Putative peptidase T Hypothetical proteins Outer membrane protein ; colicin S4 receptor ydcA Hypothetical protein ydfZ Hypothetical protein ynfEFGHI Me2SO reductase paralogue ynfK Hypothetical protein ydhYVWXU Putative oxidoreductase ydjXYZynjABCDE Hypothetical proteins pykA Pyruvate kinase II tar tap Chemotaxis cheRBYZ yecR Hypothetical protein yedM Hypothetical protein ackA Acetate kinase yfcC Putative S-transferase upp-uraA Uracil salvage yqjI Hypothetical protein aer Aerotaxis sensor receptor uxaCA Glucuronate degradation tdcABCDEFG Threonine degradation garPLRK Glucarate degradation yhbTS Hypothetical proteins yhbUV Hypothetical protein and putative collagenase Hypothetical protein Maltose degradation Low affinity phosphate-transport Putative 2-component regulator PTS-system Catalase Hypothetical proteins Hypothetical protein Transcriptional activator of cad operon fimBEAICDFGH Type 1 fimbriae yjiD Hypothetical protein tsr Chemotaxis protein yjjIW Hypothetical protein Putative FNR-repressed operons pdhR-aceEF-lpdA Pyruvate dehydrogenase and regulator Glucose dehydrogenase Hypothetical protein Cytochrome o ubiquinol oxidase Rhs element protein / hypotheticals Hypothetical protein Outer membrane protein X Putative cysteine hydrolase 41.5 and 90.5 197.5 185.5 142.5 and 21.5 143.5 Mapped Mapped Predicted Predicted Mapped Y/N Y/N Y/N Y/N Y/N 0.79 1.35 1.16 0.29 1.39 0.94 1.31 0.77 0.29 1.32 0.51 0.40 0.57 0.44 0.30 0.82 0.82 1.20 0.66 0.21 0.39 0.56 0.56 0.28 0.16 0.74 0.81 0.84 0.89 0.29 0.60 0.64 0.41 0.33 0.14 32.5 30.5 Predicted Predicted Y/Y Y/N Y/Y Y/N Y/Y Y/Y citCDEFXG ycbJ yccM pepT ymfED ompW 0.19 0.23 0.27 0.33 0.46 0.09 0.19 0.30 0.42 0.59 0.21 0.06 0.37 0.41 0.43 0.39 0.69 0.27 0.49 0.39 1.64 0.66 1.49 0.03 0.30 0.33 0.44 0.35 0.60 0.02 0.48 0.74 2.80 0.85 0.91 0.53 0.34 0.36 0.36 0.36 0.68 0.02 25.5 83.5 124.5 2 putative sites Predicted Predicted Predicted No predicted ts 0.46 0.01 0.06 0.21 0.42 0.49 0.02 0.05 0.15 0.14 43.5 35.5 64.5 29.5 167.5 Predicted Predicted Predicted Predicted Predicted Y/N N/Y Y/Y N/Y Y/Y 0.40 0.01 0.03 0.11 0.21 0.66 0.52 0.03 0.50 0.23 0.39 0.01 0.02 0.09 0.25 0.84 0.73 0.25 0.77 0.31 0.45 0.01 0.03 0.09 0.16 2.81 0.82 0.96 1.23 1.19 0.54 0.48 0.48 0.51 1.78 0.48 0.39 0.18 0.42 0.39 0.33 0.25 0.78 0.39 0.44 0.57 0.42 0.91 1.63 0.55 0.24 0.29 0.10 0.61 1.15 0.88 0.26 0.80 1.39 0.80 0.31 0.23 0.34 0.71 1.91 79.5 117.5 Predicted Predicted Y/N Y/N Y/N Y/Y Y/N Y/N Y/N Y/Y Y/N Y/N 0.53 0.42 0.28 0.55 0.42 0.28 0.16 0.42 0.40 0.62 0.63 1.01 0.53 0.67 0.91 0.73 0.84 0.93 1.38 0.64 0.46 0.43 0.05 0.17 0.29 0.27 0.23 0.31 0.47 0.27 160.5 113.5 14.5 1 putative site 18.5 61.5 150.5 and 200.5 51.5 Predicted Predicted Predicted No predicted ts Predicted Mapped Predicted Mapped 0.57 0.44 0.60 0.37 0.24 1.12 0.81 1.30 0.80 0.97 0.42 0.28 0.46 0.35 0.34 0.64 0.60 0.59 0.58 0.88 0.47 0.45 0.49 0.34 0.56 1.14 1.04 1.67 0.63 0.71 0.85 0.75 0.99 0.53 0.46 58.5 Predicted Mapped Predicted Predicted Predicted Y/N Y/Y Y/N Y/N Y/N 10.5 and 148.5 154.5 76.5 45.5 yhcC 0.22 0.62 0.37 0.62 0.84 0.85 0.93 0.82 0.21 0.54 0.34 0.71 0.68 0.73 0.57 0.75 0.32 0.49 0.48 0.45 0.38 4.29 0.70 0.67 0.28 1.54 0.75 0.50 Predicted Mapped Predicted Predicted 70.5 127.5 66.5 37.5 Y/N Y/N Y/Y Y/N malPQ pitA yieNM frvABXR katG yjdAZ yjdK cadC 0.34 0.27 0.04 0.20 0.15 0.43 0.94 0.50 0.65 0.25 0.37 0.23 0.22 0.16 0.11 0.39 0.73 1.36 0.78 0.29 0.32 0.21 0.15 0.31 0.14 1.45 0.32 0.76 0.08 1.54 1.82 0.22 1.19 0.09 1.98 Predicted Mapped Predicted Predicted Mapped 11.5 Y/N Y/Y Y/N Y/Y Y/Y 96.5 and 112.5 50.5 and 56.5 14.5 156.5 0.40 0.59 0.05 0.06 0.86 1.06 0.62 0.70 0.70 0.37 0.19 0.04 1.06 0.67 1.22 0.62 0.93 0.50 0.16 0.06 0.35 0.71 0.78 0.11 0.50 0.44 1.55 0.11 Y/Y Y/Y Y/N Y/N Mapped Predicted Predicted Predicted 71.5 148.5 80.5 56.5 3.30 0.81 2.97 1.05 2.64 0.74 0.63 Y/Y Mapped 50.5 Y/N Y/N Y/Y gcd yaiB 1.52 1.96 43.04 0.75 1.09 1.12 1.85 2.04 61.33 0.91 1.04 13.07 1.07 2.30 60.68 5.27 2.48 33.75 10.29 1.54 74.26 Mapped Predicted Mapped 31.5 wrt p1 , 15.5 wrt p2 62.5 192.5 E cyoABCD rhsDybbCylbH-ybbDb0501 ybiJ ompX 2.05 2.17 1.98 0.90 0.42 1.44 1.13 2.46 1.94 0.55 1.05 1.61 1.52 1.44 2.15 0.72 1.68 1.70 0.49 2.84 1.81 Predicted Predicted Predicted 61.5 and 128.5 2.5 and 62.5 3.5 Y/Y Y/N Y/N ycaC Y/Y 134.5 Predicted 0.79 3.01 1.73 2.54 1.41 2.99 1.17 2 , part B , be confirmed ; these were cydDC , hlyE , adhE , glpTQ , and arcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Catabolite response Deoxirribose assimilation D 87 135 Crp w 27 Pérez - Rueda and Collado-Vides ( 2000 ) 220 -- R 235 10 DeoR DicA 35 w Pérez - Rueda and Collado-Vides ( 2000 ) -- n/d 0 -- -- n/d HLH R 4 DicC DnaA 0 Transcription and replication regulation Positive regulator of cytochrome c552 77 0 -- Pérez - Rueda and Collado-Vides ( 2000 ) 41 -- n/d DniR 0 Kajie et al. ( 1991 ) 209 8 215 -- A Ebp FhlC 77 m Pérez - Rueda and Collado-Vides ( 2000 ) Pérez - Rueda and Collado-Vides ( 2000 ) n/d 0 -- 7 -- Pérez - Rueda and Collado-Vides ( 2000 ) Schnetz ( 2002 ) n/d R FhlD FrvR 0 -- Phosphoenolpyruvate sugar phosphotransferase system operon regulation Iron assimilation 8 0 -- -- 128a Fur R w Achenbach and Yang ( 1997 ) 107 460 35 Carbon sources assimilation Nguyen and Saier ( 1995 ) ; Weickert and Adhya ( 1992 ) 388 GalR/LacI 55 R c -- 13 GcvR GntR Rigali et al. ( 2002 ) Glycine cleavage operon repression 0 R 735 775 Ghrist and Stauffer ( 1998 ) 96 R w Yamada and Saier ( 1988 ) Activator of the glucitol operon in E. coli High-frequency persistence of the lethal effects of inhibition of either DNA or peptidoglycan-synthesis regulation ( HipAB ) -- 8 GutM 0 A -- Black et al. ( 1991 ) -- 1 HipA 1 0 n/d -- n/d HipB HtgA Black et al. ( 1991 ) Missiakas et al. ( 1993 ) 87 0 -- -- -- n/d 7 Heat shock regulation 0 -- HycA HypF 6 -- n/d ZnF A 0 -- 0 Bouhouche et al. ( 2000 ) Gui et al. ( 1996 ) 27 -- Glyoxylate bypass operon Integration host factor 31 106 161 w R -- IclR Ihf -- 15 8 Pérez - Rueda and Collado-Vides ( 2000 ) Campoy et al. ( 2002 ) -- D 43 -- DNA-repair 54 LexA R w 102 279 Miller and Bassler ( 2001 ) Schell ( 1993 541 LuxR/UhpA m A Quorum sensing 1548 LysR 1271 Amino acid biosynthesis regulation D w 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Being a modulator of gene-expression , the relative increase in iHF level may be important for delicate tuning of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Tel. , ext. 1310 ; Fax ; e-mail : asirko@ibbrain.ibb.waw.pl Abbreviations : Bp , base pair ( s ) ; Cmk , Gene encoding cytidine mono-phosphate kinase ; Hima , See lhfa ; Hired , See Ihfb ; Hip , See Ihfb ; Ihf , integration host factor ; Ihfa , Gene encoding the ~ subunit of ihf ; lhfb , Gene encoding the 13subunit of ihf ; Infc , Structural gene for translation-initiation factor if3 ; Lb , luria-bertani ( medium ) ; Mssa , See Cmk ; Nt , nucleotide ( s ) ; Oligo , oligodeoxyribonucleotide ; Pcr , polymerase chain reaction ; Phes And Phet , Structural genes for the small and large subunits of phenylalanyl-trna synthetase ( e.c. 6.1.20 ) ; Ppgpp , guanosine 5 ' - diphosphate 3 ' - diphosphate ; Rplt , Structural gene for ribosomal protein 120 ; Rpsa , Structural gene for ribosomal protein sl ; Thrs , Gene encoding threonyl-trna synthetase ; Z3Z , [ Mathsoff ] stationary phase-specific sigma ( g ) subunit of rna polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>Knockout mutations in lrp and ihf genes affecting global regulation , or the mlc and nagC ( Plumbridge , 1989 ; 1998 ) genes involved in the regulation of GlcNAc uptake and metabo-lism , had no effect on chiA ± lacZ expression and ChiA production ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760150" target="_blank">10760150</a>]<br></li>
<li>However , the fact that expression of the rpoS , lrp and ihf genes is stimulated by ppGpp ( Aviv et al. , 1994 ; Lange et al. , 1995 ) and that many genes induced by carbon , phosphorus and nitrogen starvation are regulated by the stringent-response ( Nystro $ m , 1995 ; Rao et al. , 1998 ) , is consistent with YeiL being involved in the response-to-nutrient starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Click here for file [ http://www.biomedcentral.com/content/supplementary/1475-2859-8-54-S1.DOC] List of abbreviations used aceBAK : glyoxylate shunt operon ; aceE : subunit E1p of pyruvate dehydrogenase complex ; aceF : lipoate acetyltransferase subunit of pyruvate dehydrogenase complex ; Ack ( ack ) : acetate kinase ; Acs ( acs ) : acetyl-CoA synthetase ; ADH ( adhE ) : alcohol dehydrogenase ; cAMP : cyclic AMP ; CoASH : coenzyme A ; Cra/FruR ( cra ) : catabolite repressor activator ; Crp ( crp ) : catabolic repressor via cAMP receptor protein ; Icdh ( icd ) : isocitrate dehydrogenase ; IcIR ( iclR ) : glyoxylate shunt repressor ; Icl ( aceA ) : isocitrate lyase ; iD-Ldh ( dld ) : D-lactate dehydrogenase ( NAD + - independent ) ; α IHF subunit ( ihfA ) : subunit of integration host factor ; iL-Ldh ( lldD ) : L-lactate dehydrogenase ( NAD + - independ-ent ) ; Ldh ( ldhA ) : lactate dehydrogenase ( NAD + - depend-ent ) ; Lpd ( lpd ) : lipoamide dehydrogenase , pyruvate dehydrogenase E3 monomer ; MaeB ( maeB ) : malic enzyme NADP + - requiring ; Mdh ( mdh ) : malate dehydrogenase ; MS ( aceB ) : malate synthase ; OAA : oxaloacetate ; Pcc ( ppc ) : PEP carboxylase ; Pck ( pck ) : PEP carboxykinase ; Pdh : pyruvate dehydrogenase complex ; PdhR ( pdhR ) : pyruvate dehydrogenase complex regulator ; PEP : phosphoe-nolpyruvate ; Pfl ( pflB ) : pyruvate formate lyase ; PflA ( pflA ) : pyruvate formate lyase activating enzyme ; polA : DNA polymerase I ; PoxB ( poxB ) : pyruvate oxidase ; Pps ( pps ) : phosphoenolpyruvate synthase ; Pta ( pta ) : phosphotransacetylase ; PtsG ( ptsG ) : subunit of Enzyme IIglc ; Pyk ( pykF , pykA ) : pyruvate kinase ; RpoD ( or σ32 ) : sigma subu-nit of RNA polymerase ; RpoS ( or σS ) : sigma subunit of RNA polymerase ; Sdh ( sdhC ) : succinate dehydrogenase membrane protein ; SfcA ( sfcA/maeA ) : malic enzyme NAD + - requiring ; sucA : subunit of E1 ( 0 ) component of 2-oxoglutarato dehydrogenase ; TCA : tricarboxylic acids cycle ; Zwf ( zwf ) : glucose 6-phosphate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19852855" target="_blank">19852855</a>]<br></li>
<li>Expression analysis of ( ulaG-lacZ ) and ( ulaA-lacZ ) transcriptional fusions in different genetic backgrounds under anaerobic conditions - Galactosidase expressiona ( Miller units ) Relevant phenotype Strain ( ulaG-lacZ ) ( ulaA-lacZ ) CAA L-Ascorbate CAA L-Ascorbate 40 130 JA134 Wild type JA184 IHF JA185 IHF JA188 RpoS JA211 YjfQ JA216 YjfQ CRP JA182 FNR a ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>Hha partially represses biofilm-formation through type I fimbriation via fimA and ihfA Along with fimA , the main structural component of type I fimbriae , hha deletion induced ihfA ; ihfA is a subunit of the IHF global regulator that is required for the normal expression of fimA , as mutations in ihfA or in the IHF beta-subunit ihfB ( also induced upon hha deletion ) prevent the phase variation expression of fimA [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Other lines of evidence found here linking Hha to fimbriae include ( ii ) ihfA encoding a subunit of the IHF regulator that is an activator of type I fimbriae production via the fimA promoter [ 11 ] is activated in all the LB glu microarrays upon hha deletion ( Table S1 ) , ( iii ) Hha represses fimA and ihfA transcription in biofilm cells as shown by RT-PCR , ( iv ) fimA and ihfA mutations abolish biofilm-formation , ( v ) deleting hha along with either fimA and ihfA ( double mutants ) fails to increase biofilm-formation , ( vi ) transcription of fimA and ihfA is induced in the hha deletion mutant and is inhibited by overexpressing hha ( GFP reporter constructs ) ( Figure 4 ) , and ( vii ) type I fimbriae production dramatically increases in the absence of hha and Hha overexpression inhibits type I fimbriation ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Table 1 Correspondence among replication proteins from Escherichia coli , Bacillus subtilis , and bacteriophages T7 and T4 phage T4 phage T7 B. subtilis E. coli Function PolC , DnaEBS b2 t5 RTP2 SSB4 DnaA , B , D DnaC6 DnaI DnaG polymerase/exonuclease sliding clamp clamp loader terminator SSB initiator replicative helicase helicase loader primase aey b2 gp43 gp453 gp5 Trx ( E. coli ) -- -- gp2 .52 T7 RNAP gp4 -- gp4 dgt2d ' cw Tus gp444gp62 -- gp32 -- gp41 gp59 gp6 SSB4 DnaA , IHF , HU , FIS DnaB6 DnaC DnaG Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16036556" target="_blank">16036556</a>]<br></li>
<li>An International Journal RSpringer-Verlag New York Inc. 1999 The Integration Host Factor ( IHF ) Affects the Expression of the Phosphate-Binding Protein and of Alkaline Phosphatase in Escherichia coli Beny Spira , * Ezra Yagil Department of Biochemistry , Tel-Aviv University , Tel-Aviv 69978 , Israel Received : 4 May 1998 / Accepted : 19 August 1998 Abstract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>iscR b2531 4.3 Adaptation dinJ himD b0226 b0912 5.7 5.7 Predicted antitoxin of YafQ-DinJ toxin-antitoxin system Integration host factor ( IHF ) , b subunit ; site-specific recombination Stress-induced protein Periplasmic glutamine-binding protein ; permease Periplasmic adaptor protein , regulator of the Cpx response and possible chaperone involved in resistance to extracytoplasmic stress Specificity factor for ClpA-ClpP , ClpS shows OxyR-dependent induction of expression by hydrogen peroxide chaperone-protease complex Acidic protein suppresses mutants lacking function of protein export GroES , 10 kDa chaperone binds to Hsp60 in pres. Mg-ATP , suppressing its ATPase activity Lysine - , arginine - , ornithine-binding periplasmic protein Signal inhibitory factor-X , Phosphohistidine phosphatase affecting ArcB H-D phosphorelay ; expression of six A is dependent on the alternative sigma factor rpoE PTS heat stable , histidyl phosphorylatable protein Dipeptide-transport protein uspF ( ynaF ) a glnH cpxP b1376 b0811 b3913 5.7 5.3 4.6 b0881 4.3 clpS ( yljA ) msyB 4.3 b1051 groS 4.3 b4142 argT sixA 4.0 4.0 b2310 b2340 4.0 4.0 b2415 b3544 ptsH dppAb Metabolism fldA acpP glgS ivbL acs Flavodoxin 1 Acyl carrier protein 5.3 4.3 4.3 4.3 4.0 b0684 b1094 b3049 b3672 b4069 Predicted glycogen-synthesis protein , rpoS dependent ilvB operon leader peptide Acetyl-CoA synthetase RNA rpsV b 4.6 30S ribosomal subunit protein S22 b1480 Unknown function ybeL a ybiM a , b yjfO yahO ydiZ b yoaC yjdN b yacL ybaA ycfP ynhG yqjC yegP yobF smg b0643 b0806 b4189 b0329 b1724 b1810 b4107 b0119 b0456 b1108 b1678 b3097 b2080 b1824 b3284 Putative alpha helical protein Predicted periplasmic protein , YhcN Probable lipoprotein , YhcN family Predicted periplasmic protein , YhcN Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein Hypothetical protein 6.5 6.5 5.7 4.9 4.9 4.9 4.9 4.6 4.6 4.6 4.6 4.6 4.3 4.0 4.0 ily fam ily fam Abbreviation : LB , Luria -- Bertani medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18309357" target="_blank">18309357</a>]<br></li>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor-biosynthesis , and DNA-RNA-metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative stress.6,11-14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated Glycolysis , citric-acid-cycle , and respiratory chain genes times following the introduction of air 5 min 10 min 15 min 60 min 6.0 6.7 6.9 4.7 3.6 5.7 6.1 3.2 Relevant regulatory proteins aceE pyruvate dehydrogenase ( decarboxylase component ) aceF pyruvate dehydrogenase ( dihydrolipoyltransacetylase component ) acnA aconitase A acnB aconitase B cyoA cytochrome bo terminal oxidase subunit II cyoB cytochrome bo terminal oxidase subunit I cyoC cytochrome bo terminal oxidase subunit III cyoD cytochrome bo terminal oxidase subunit IV cyoE heme o synthase gltA citrate synthase gpmA phosphoglycerate mutase icdA isocitrate dehydrogenase ( NADP ) lpdA lipoamide dehydrogenase ( E3 monomer ) mdh malate dehydrogenase mqo malate dehydrogenase ( quinone ) ndh NADH dehydrogenase II pdhR pyruvate-responsive transcription regulator poxB pyruvate oxidase sdhA succinate dehydrogenase flavoprotein sdhB succinate dehydrogenase iron-sulfur protein PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) ArcA ( ) FNR ( ) Cra ( ) CRP ( ) SoxRS ( ) ArcA ( ) Cra ( ) CRP ( ) Fis ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA ( ) ArcA * ( ) Fur ( ) P1 , ArcA * ( ) P2 , Cra * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) Plpd-ArcA * ( ) FNR ( ) CRP ( ) PdhR ( ) Fis ( ) ArcA ( ) CRP ( ) FlhD ( ) ArcA ( ) FNR * ( ) PpdhR , FNR ( / ) PdhR * ( ) CRP ( ) SoxRS ( ) MarR ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2-CRP ( ) P1 - ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 - CRP ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) P1 , ArcA * ( / ) FNR * ( ) CRP ( ) Fur ( ) P2 , ArcA ( ) FNR ( ) IHF ( ) 2.0 3.1 2.6 2.9 6.8 8.6 9.9 7.3 5.2 2.7 1.9 5.0 3.3 3.2 7.4 3.2 5.8 7.3 9.1 11.7 7.8 6.5 4.7 2.1 6.1 4.9 10.4 12.9 9.2 5.7 4.4 1.5 4.8 8.6 10.0 7.9 6.3 4.0 2.1 5.3 3.0 2.4 4.8 2.9 3.5 3.9 3.4 6.9 4.2 4.6 3.5 8.6 5.0 5.0 2.2 3.5 2.4 6.6 3.1 3.0 4.7 2.1 3.7 6.8 9.9 5.5 9.1 5.8 8.7 4.3 2.9 5.8 2.8 sdhC succinate dehydrogenase membrane protein sdhD succinate dehydrogenase membrane protein sucA 2-oxoglutarate decarboxylase 16.6 4.6 3.2 3.6 7.8 1.9 2.0 2.1 2.1 3.2 3.6 3.1 sucB dihydrolipoamide succinate transferase sucC succinyl-CoA synthetase , subunit sucD succinyl-CoA synthetase , subunit 3.5 4.7 2.7 TABLE 2 Transcripts encoding proteins involved in the oxidative stress response that are present in increased abundance after the introduction of air to anaerobic cultures of E. coli MG1655 Fold regulation ( at least 2-fold at one or more time point ) is the ratio of transcript levels at the indicated time after the introduction of air to the transcript levels at t 0 ( the initial anaerobic state ; p 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 4.8 5.2 4.6 3.1 3.1 3.5 2.1 2.0 4.2 4.2 5.1 4.6 Oxidative stress and iron-sulfur-cluster-assembly genes Relevant regulatory proteins ahpC alkyl hydroperoxide reductase small subunit ahpF alkyl hydroperoxide reductase large subunit dadA D-amino acid dehydrogenase dadX alanine racemase OxyR * ( ) MetJ ( ) OxyR * ( ) MetJ ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) IscR * ( ) SoxRS * ( ) OxyR * ( ) OxyR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) OxyR * ( ) P1 , H-NS ( ) Lrp * ( ) NhaR ( ) RcsB ( P2 , H-NS ( ) Lrp * ( ) IscR * ( ) OxyR ( ) IHF ( ) Fur SoxR ( ) SoxS ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) 3.3 2.1 3.4 1.9 3.2 1.3 7.2 1.6 2.2 2.8 6.3 6.1 3.4 3.0 2.1 5.3 2.1 2.8 3.0 9.1 5.8 4.1 fdx ferredoxin fur ferric uptake regulator 2.1 3.4 2.4 2.8 3.3 10.1 7.0 5.1 2.0 1.9 1.8 2.7 1.8 1.8 2.8 2.5 11.2 5.7 4.4 1.8 1.1 3.6 grxA glutaredoxin 1 hscA co-chaperone involved in IscU iron-sulfur-cluster-assembly hscB co-chaperone involved in IscU iron-sulfur-cluster-assembly iscA iron-sulfur-cluster-assembly protein iscR transcriptional regulator iscS cysteine desulfurase iscU scaffold protein involved in iron-sulfur-cluster-assembly iscX possible role in iron-sulfur-cluster-biosynthesis katG bifunctional hydroperoxidase I osmC organic hydroperoxidase Within 5 min of air exposure the abundance of the soxS transcript is increased 8-fold ( Table 2 ) , indicating activation of the sensor SoxR , which enhances soxS expression , which in turn controls the SoxRS regulon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Significant electron flow is directed through both enzymes during glucose-limited aerobic growth ( Calhoun et al. , 1993 ) and recently it has been shown that NdhII is the Abbreviations : Arr , amino acid response regulator ; Fis , factor for inversion stimulation ; FNR , regulator of fumarate and nitrate reduction ; IHF , integrationhost factor ; NbpB ( NbpC ) , heat-stablefraction 1 ( 2 ) ; Ndhl ( Ndhll ) , NADH dehydrogenaseI ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Among a set of more than 12 characterized proteins , FIS , H-NS , IHF and LRP have been recognized as more speciﬁc regulators , acting as transcriptional activators or repressors for numerous genes ( Azam and Ishihama , 1999 ; Dame et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>2010 SGM Printed in Great Britain In concert with the complex mode of regulation for biofilm-formation , the expression of csgD has been proposed to be under the control of various stress-sensing regulatory proteins such as OmpR ( Romling et al. , 1998 ; Vidal et al. , 1998 ; Prigent-Combaret et al. , 1999 , 2000 , 2001 ; Gerstel & Romling , 2001 ; Gerstel et al. , 2003 ) , CpxR ( PrigentCombaret et al. , 2001 ; Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ; Robbe-Saule et al. , 2006 ) , CRP ( Zheng et al. , 2004 ) , H-NS ( Arnqvist et al. , 1994 ; Gerstel et al. , 2003 ; Olsen et al. , 1993 ) , IHF ( Gerstel et al. , 2003 ; Gerstel & Romling , 2001 ) , MlrA ( Brown et al. , 2001 ) and RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Furthermore , a number of transcription factors are involved in the regulation of the csgD promoter , including CpxR ( Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ) , H-NS ( Gerstel et al. , 2003 ) , IHF ( Gerstel et al. , 2003 , 2006 ) , OmpR ( Vidal et al. , 1998 ; Gerstel et al. , 2003 , 2006 ) , RstA ( Ogasawara et al. , 2007a ) , MlrA ( Brown et al. , 2001 ) , RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) and CRP ( Zheng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In wVo chemical protection studies suggest that IHF also helps NR1 to activate this promoter in growing cells ( F. Claverie-Martin and B. Magasanik , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>A , IHF effect on Pa and Pz promoter activity in vivo in isogenic E. coli IHF1 ( himA1D1 ) and IHF2 ( himA-D1 , himA1D2 ) strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Binding sites for the regulatory proteins IHF , PepA , PurR and ArgR involved in the purine , pyrimidine and arginine-speciﬁc control of promoter activity are colored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>In the carAB promoter , at least ﬁve regulatory proteins ( IHF , PepA , PurR , RutR and ArgR ) are involved in the purine , pyrimidine and arginine-speciﬁc control of the promoter activity ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21572102" target="_blank">21572102</a>]<br></li>
<li>The double mutation C150G / C153G , which disrupts binding of IHF to IHF III , was generated using the Gene Editor site-directed mutagenesis kit ( Promega , Madison , WI ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>We thank M. Chandler , D. Clark , and H. Nash for strains and plasmids ; M. Filutowicz for his protocol for IHF purification ; and M. Sauter for performing the large-scale fermentation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>We thank Howard A. Nash for generously providing puriﬁed IHF protein , S. Adhya for strain A5427 , G. W. Hatﬁeld for strain IH1-6 , and Ali T. van Loo-Bhattacharya for technical assistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>Acknowledgements We thank F.Boccard for puri ® ed IHF protein , M.Berkenkopf for technical assistance and R.Kahmann for critical comments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>We thank Valley Stewart for supplying bacterial strains , Reid Johnson and Rick Gourse for providing puriﬁed Fis protein , and Tamara Belyaeva and David Lee for help in purifying IHF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank Barry Wanner for providing plasmid pKD46 and E. coli strains BW25113 and BW25993 , Richard Gourse for puriﬁed CRP protein and strain DM0068 , and Marcin Filutowicz for puriﬁed IHF and strain DBP102 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>INTRODUCTION Keywords : Escherichia coli , ndh , IHF , amino acids , transcription regulation The facultative anaerobe Escherichia coli has two genetically distinct NADH dehydrogenases serving as primary dehydrogenasesin the aerobic respiratory chain ( Calhoun & Gennis , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>IHF has recently been shown to be a direct effector of a number of other operons in E. coli ( summarized in Table 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_43">
For several operons in IHF has E. coli , been shown to inhibit transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9315722" target="_blank">9315722</a>]<br></label>
<input id="rp_similarity_43", type="checkbox">
<div class="RP_similar">
<li>IHF as a negative regulator of transcription For several operons of E. coli , IHF has been shown to inhibit transcription in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_45">
Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , CRP and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></label>
<input id="rp_similarity_45", type="checkbox">
<div class="RP_similar">
<li>Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , Crp , and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_47">
Factor selectivity of Escherichia coli RNA polymerase : Role for CRP , IHF and lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></label>
<input id="rp_similarity_47", type="checkbox">
<div class="RP_similar">
<li>s factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>]<br></li>
<li>Sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621818" target="_blank">16621818</a>]<br></li>
<li>Sigma factor selectivity of Escherichia coli RNA polymerase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li></div>
<li>Gene 31:165 -- 171 Colland F , Barth M , Hengge-Aronis R , Kolb A ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_52">
Colland , F. , Barth , M. , Hengge-Aronis , R. , and Kolb , A. ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymer-ase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></label>
<input id="rp_similarity_52", type="checkbox">
<div class="RP_similar">
<li>177 , 3166 -- [ 30 ] Colland , F. , Barth , M. , Hengge-Aronis , R. and Kolb , A. ( 2000 ) r factor selectivity of Escherichia coli RNA polymerase , role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15251200" target="_blank">15251200</a>]<br></li>
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. , and Kolb , A. ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymerase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. and Kolb , A. ( 2000 ) sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li></div>
<li>HU constrains negative supercoifs in DNA ( Broyles and Pettijohn , 1986 ) and is inencoding gene ; hupA and hupB , genes encoding HU-2 and HU - 1 , respectively ; Hy , hygromycm B ; IHF , integration host factor ; himA and himD , genes encoding IHFa and IHFB , respectively ; kb , kiiobase ( sf or 1000 bp ; Km , kan ~ y ~ in ; LB , Lucia-Bertani ( medium ) ; nt , nucleotidefs ) ; ohgo , oligodeoxyribonucleotide ; PA , pofyacfylam ~ de ; PAGE , PA-gel etectro-phoresis ; R , resistance/resistant ; s , sensitivity/sensitive ; SDS , sodium dodecyl sulfate ; Tc , tetracycline ; trp $ genes encoding enzymes for tryptophan-synthesis ; wt , wild type ; : : , novel junction ( fusion or insertion ) ; [ 1 , denotes plasmid-carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>' - ' Iac2 by Transcriptional regulation regulators of respiration and IHF Transcriptional regulation of nuo ^ gyAB ' - ' lacZ by regulators of respiration and integration iiost factor ( IHF ) Regulatory mutants defective in the genes fnr , arcA , narL , narP , fhlA and himD were used to study the effects of transcriptional regulators acting on respiratory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>Proc Natl Acad Sci USA 77:122 -126 Thompson R , Mosig G ( 1988 ) Integration Host Factor ( IHF ) represses a Chlamydomonas chloroplast promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>We report the identification of Integration Host Factor ( IHF ) as a new element involved in modulation of P 1 , the upstream pyrimidine-specific promoter of the Escherichia coli K12 and Salmonella typhimurium earAB operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_60">
Charlier , D. , Roovers , M. , Gigot , D. , Huysveld , N. , Piérard , A. and Glansdorff , N. ( 1993 ) Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472642" target="_blank">20472642</a>]<br></label>
<input id="rp_similarity_60", type="checkbox">
<div class="RP_similar">
<li>Charlier D , Roovers M , Gigot D , Huysveld N , Pierard A , Glansdorff N ( 1993 ) Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Charlier , D. , Roovers , M. , Gigot , D. , Huysveld , N. , Pierard , A. , and Glansdorff , N. ( 1993 ) Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>Integration host factor ( IHF ) modulates the expression of the pyrimi-dine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>Integration host factor ( IHF ) modulates the expression of the pyrimi-dine-speci ® c promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li></div>
<li>ArcA ( aerobic-respiration control ) ( luchi and Lin , 1991 ) , SoxRS ( superoxide response ) ( Demple and Amabile-Cuevas , 1991 ; Tsaneva and Weiss , 1990 ) , SoxO -LCB- Greenberg et ai , 1991 ) , Fnr ( fumarate nitrate reductase ) ( Spiro and Guest , 1991 ) , and IHF ( integration host factor ) ( Craig and Nash , 1984 ; Freundlich ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412671" target="_blank">8412671</a>]<br></li>
<li>The carP1 control region harbours binding sites for the transcriptional regulators PurR ( purine repressor ) and RutR ( pyrimidine utilization regulator ) , and for the architectural proteins IHF ( integration host factor ) and PepA ( aminopeptidase A , a trigger enzyme ; Figure 1a ; 2,14 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472642" target="_blank">20472642</a>]<br></li>
<li>However , nucleoid-associated proteins ( NAPs ) , particularly histone-like proteins , such as histone-like nucleoid structuring ( H-NS ) protein , heat unstable nucleoid protein ( HU ) , factor for inversion stimulation ( Fis ) and integration host factor ( IHF ) , are believed to act like histones and play a signiﬁcant role in the organization of the nucleoid ( 14 -- 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>These proteins include ( in decreasing order of abundance during exponential growth ) : factor for inversion stimulation ( Fis ) , heat-unstable nucleoid protein ( HU ) , integration host factor ( IHF ) , histone-like , nucleoid-structuring protein ( H-NS ) , suppressor of td mutant phenotype A ( StpA ) and DNA-binding protein from starved cells ( Dps ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Candidate proteins include the nucleoid proteins FIS ( factor for inversion stimulation ) and IHF ( integration host factor ) , both of which participate directly in the regulation of acs transcription ( D. F. Browning , C. M. Beatty , S. J. W. Busby , and A. J. Wolfe , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923087" target="_blank">12923087</a>]<br></li>
<li>In addition , global regulators , such as factor inversion stimulation ( FIS ) , integration host factor ( IHF ) , cyclic AMP , and ppGpp , were found to play a role either directly or indirectly in the transcription of sra in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Conversely , PspB and PspC act as positive regulators of psp operon transcription by overcoming the negative regulation 1 The abbreviations used are : pIV , protein IV ; IHF , integration host factor ; IPTG , isopropyl 1-thio - - D-galactopyranoside ; RT , reverse transcriptase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>The Fur-dependent regulation of ftnA requires H-NS Nucleoid-associated proteins , such as IHF ( integration host factor ) and H-NS , play a major role in modifying DNA conformation and thus controlling the access of certain regulators and/or RNA polymerase to promoters ( Atlung and Ingmer , 1997 ; Dame , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>The described change in overall molecular shape may inﬂuence interaction with other proteins , such as integration host factor ( IHF ) , which has been shown to play a role in ModE-dependent regulation of the dmsABC operon in E. coli .17 Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>The integration requires also the host encoded architectural protein IHF , and the reverse reaction , excision , requires in addition to Int and IHF , the phage-encoded Cox protein ( Yu and Haggård-Ljungquist , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20627350" target="_blank">20627350</a>]<br></li>
<li>Some inducible processes , including pH-regulated ones , which show a DNA gyrase requirement , involve both integration host factor , IHF , which is hrmA/hzmD encoded , and H-NS ( Stim and Bennett 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588889" target="_blank">8588889</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Sept. 1991 , p. / 91/185800 -08 $ 02.00 / 0 Copyright © 1991 , American Society for Microbiology Integration Host Factor Binds Specifically to Multiple Sites in the ompB Promoter of Escherichia coli and Inhibits Transcription PING TSUI , LIN HUANG , AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , New York 11794-5212 Received 7 January 1991/Accepted 15 July 1991 Escherichia coli integration host factor ( IHF ) is a DNA-binding protein that participates in gene regulation , site-specific recombination , and other processes in E. coli and some of its bacteriophages and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>172 , No. 9 Integration Host Factor Is aNegative Effector of In Vivo and In Vitro Expression of ompC in Escherichia coli LIN HUANG , PING TSUI , AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , Ne Received 24 April 1990/Accepted 15 June 1990 w York 11794-5215 Integration host factor ( IHF ) of Escherichia coli is a DNA-binding protein involved in gene-expression and other cellular functions in E. coli and some of its bacteriophages and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>( 1988 ) 203 , 817-820 LETTERSTOTHEEDITOR Integration Host Factor Binds Specifically to Sites in the ilvGMEDA Operon in Escherichia coli Integration host factor ( IHF ) of Escherichia coli is a histone-like protein that is involved both in site-specific recombination and in regulating the expression of a number of phage and bacterial genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>Summary Integration host factor ( IHF ) of Escherichia coli is an asymmetric histone-like protein that binds and bends the DNA at specific sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Integration host factor ( IHF ) of Escherichia coli is a site-specific DNA binding protein involved in a wide variety of physiological activities in E. coli and its phages and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>Integration host factor ( IHF ) of Escherichia coli is a multifunctional DNA-binding protein ( 3 ) that is a direct participant in certain site-specific recombination events and other processes in E. coli and some of its bacteriophages and plasmids ( reviewed in references 5 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>We examined the dependence of prophage excision on integration host factor ( IHF ) , a protein known to be involved in many site-specific recombination reactions ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7511583" target="_blank">7511583</a>]<br></li>
<li>Although originally characterized as a protein essential for site-specific integration of the bacteriophage genome into the E. coli host chromosome ( Kikuchi and Nash 1978 ; Miller et al. 1979 ) , IHF is now known to be involved in many other cellular-processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>/ ` banscription-Integration host factor ( IHF ) is a se-quence-specific DNA-binding and - bending protein , which is implicated ina number of processes including DNA-replication , site-specific recombination , and geneexpression ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>Integration host factor ( IHF ) , a heterodimeric site-speci fic DNA-binding protein , has been shown to participate in a variety of processes in E. coN including recombination and transcriptional-control ( for reviews see Freundlich et al. , 1992 ; Oberto el al. , 1994 ; Nash , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Laboratory of Genetics The Rockefeller University 1230 York Avenue , New York NY 10021 , USA # 1999 Academic Press Keywords : s54 ; PspF ; PspF HTH ; oligomerization ; psp enhancer * Corresponding author Introduction Transcription of the phage-shock protein ( psp ) operon ( pspA-E ) of Escherichia coli is under control of the alternative sigma factor , s54 , is stimulated by integration host factor ( IHF ) , and is dependent on a cis-acting sequence and the PspF activator protein ( Weiner et al. , 1991 , 1995 ; Jovanovic et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>( 3 ) We have identified several possible integration host factor , IHF ( himA-hip complex , see ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3520487" target="_blank">3520487</a>]<br></li>
<li>We have shown specific binding of several E. coli proteins to sodA promoter sequences and identified one of these proteins as the integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>Integration host factor ( IHF ) mediates repression of ﬂagella in entero-pathogenic and enterohaemorrhagic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>Yona-Nadler , C. , Umanski , T. , Aizawa , S. , Friedberg , D. , and Rosenshine , I. ( 2003 ) Integration host factor ( IHF ) mediates repression of ﬂagella in enteropathogenic and enterohaemorrhagic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Mosig G ( 1988 ) Iiitegration host factor ( IHF ) represses a Clilaiiiydoi ~ ionaschloroplast promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_93">
Integration host factor ( IHF ) represses a Chlamydomonus chloroplast promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></label>
<input id="rp_similarity_93", type="checkbox">
<div class="RP_similar">
<li>Integration host factor ( IHF ) represses a Chlamydomonas chloroplast promoter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_95">
Involvement of integration host factor ( IHF ) in maintenance of plasmid pSClO1 in Escherichia coli : characterization of pSClO1 mutants that replicate in the absence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></label>
<input id="rp_similarity_95", type="checkbox">
<div class="RP_similar">
<li>Involvement of integration host factor ( IHF ) in maintenance of plasmid pSC101 in Escherichia coli : characterization of pSC101 mutants that replicate in the absence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_97">
[ 45 ] Biek D.P. , Cohen S.N. , Involvement of integration host factor ( IHF ) in the maintenance of plasmid pSC101 in Escherichia coli : mutations in the topA gene allow pSC101 replication in the absence of IHF , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278071" target="_blank">11278071</a>]<br></label>
<input id="rp_similarity_97", type="checkbox">
<div class="RP_similar">
<li>Involvement of integration host factor ( IHF ) in maintenance of plasmid pSC101 in Escherichia coli : mutations in the topA gene allow pSC101 replication in the absence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li></div>
<li>Integration host factor protein ( IHF ) acts to control the expression of many genes in E. coli ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209047" target="_blank">9209047</a>]<br></li>
<li>Previous work has shown that integration host factor ( IHF ) mutants havcincreased expression and altered osmoregulation of OmpF , a major Escherichia coli outer membrane protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_101">
J Bacteriol 177:4053 -- 4058 Spira B , Yagil E ( 1999 ) The integration host factor ( IHF ) affects the expression of the phosphate-binding protein and of alkaline phosphatase in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></label>
<input id="rp_similarity_101", type="checkbox">
<div class="RP_similar">
<li>The integration host factor ( IHF ) affects the expression of the phosphate-binding protein and of alkaline phosphatase in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645557" target="_blank">25645557</a>]<br></li></div>
<li>Mutants of Escherichia coli lacking the global regulatory protein integration host factor ( IHF ) show an increased level of alkaline phosphatase and a decreased level of Pst .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>T. , Spassky , A. , But , H. and Higgins , C.F. : Histone-like protein Hl ( H-NS ) , DNA supercoiling , and gene Kano , Y. and Imamoto , F. : Requirement of integration host factor ( IHF ) for growth of Escherichia coli deficient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_105">
Requirement of integration host factor ( IHF ) for growth of Escherichia coli deWcient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></label>
<input id="rp_similarity_105", type="checkbox">
<div class="RP_similar">
<li>Requirement of integration host factor ( IHF ) for growth of Escherichia coli deﬁcient in HU protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li></div>
<li>In addition to osmoregulation , the expression of ompF and ompC is affected by a variety of other factors , including growth temperature ( 33 ) , integration host factor ( IHF ) ( 27 , 44 , 48 , 49 ) , stress conditions ( 6 ) , and cyclic AMP ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Mangan MW , Lucchini S , Danino V , Croinin TO , Hinton JC , Dorman CJ : The integration host factor ( IHF ) integrates stationary-phase and virulence gene-expression in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Because the integration host factor protein ( IHF ) was demonstrated to be essential for nitrate-dependent anaerobic expression of the nitrate reductase ( narGHJI ) operon ( 34 ) , we also tested its role in nitrate-dependent narK-lacZ expression ( Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>Second , integration host factor ( IHF ) has recently been shown to be absolutely required for ArcA-dependent anaerobic regulation of pfl expression ( Sirko etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934836" target="_blank">7934836</a>]<br></li>
<li>Normal expression of fimA was shown to require the integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>Recent work suggests that an additional DNA-bind-ing protein , integration host factor ( IHF ) , plays a role in the expression of the omp operons ( Tsui et al. 1988 ; Huang et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Integration host factor ( IHF ) is a DNA-binding protein involved in the bending of DNA and the regulation-of-gene-expression , and a consensus binding sequence has been proposed ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>The DNA bending proteins that participate in - depen-dent gene-expression in E. coli are integration host factor ( IHF ) and ArgR , the arginine repressor ( 72 , 103 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>70 The E. coli nucleoid-associated proteins including Fis ( factor for inversion stimulation ) , H-NS , IHF ( integration host factor ) and Lrp ( leucine-responsive regulatory protein ) are known to be involved in DNA compaction as well as in transcriptional-control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>Genetic studies have identi ® ed cAMP-CRP , Lrp , integration host factor ( IHF ) and H-NS as such modulators of sS-dependent genes ( see Hengge-Aronis ( 1996b ) for review , and references cited therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Expression of this operon is induced by growth on glycolate and is under the control of an activator protein encoded by the divergently transcribed gene glcC ( see Fig. 1 ) Expression is strongly dependent on the integration host factor ( IHF ) and is repressed by the global respiratory regulator ArcA-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11283302" target="_blank">11283302</a>]<br></li>
<li>Abbreviations : CAA , casein acid hydrolysate ; CAT , chloramphenicol acetyltransferase ; LB , Luria -- Bertani broth ; IHF , integration host factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11283302" target="_blank">11283302</a>]<br></li>
<li>Abbreviations : CRP , CAMPreceptor protein ; FNR , fumarate and nitrate reduction regulator ; IHF , integration host factor ; ODH , 2-oxoglutarate dehydrogenase ; PDH , pyruvate dehydrogenase ; SCS , succinyl-CoA synthetase ; SDH , succinate dehydrogenase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Abbreviations : CAP , catabolite gene activator protein binding site ; IHF , integration host factor binding site ; = = , AGPyATAAA consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2573820" target="_blank">2573820</a>]<br></li>
<li>Abbreviations : ALDH , aldehyde dehydrogenase ; Crp , catabolite repression protein ; IHF , integration host factor ; tsp , transcriptional start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>We found that several global regulators ( DnaK , DnaJ , DksA , histone-like protein , integration host factor [ IHF ] ) , as well as the phosphatase YigB and the 5-formyl-tetrahydrofolate ( 5-formyl-THF ) cyclo-ligase YgfA , are involved in persister cell formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18519731" target="_blank">18519731</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We show here that in addition to crS , the global regulators Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex ( cAMP-CRP ) , and integration host factor ( IHF ) are involved in the control of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In recent studies we have shown that in addition to sS , the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex , Lrp , and integration host factor ( IHF ) are involved in modulating osmY transcription ( 33 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>Superimposed in the specific PaaX-mediated repression , two global regulators , the cAMP receptor protein ( CRP ) and the integration host factor protein ( IHF ) , act as activators of the gene-expression driven by Pa and Pz promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>We had shown previously that the Pa and Pz promoters of the two PA catabolic operons in E. coli were under control of global regulators such as the cyclic adenosine monophosphate ( cAMP ) receptor protein ( CRP ) , that accounts for the catabolite repression control mediated by glucose , and the integration host factor ( IHF ) ( Ferrández et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Known regulators involved in the control of acs expression in E. coli are cAMP receptor protein ( CRP ) , factor for inversion stimulation ( Fis ) , integration host factor ( IHF ) ( Wolfe 2005 ) and possibly the dual transcriptional regulator for an-oxic redox control ( ArcA ) ( Covert et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24633370" target="_blank">24633370</a>]<br></li>
<li>The synthesis and expression of these ﬂagella and motility genes are regulated by several environmental cues besides quorum-sensing , including temperature ( Adler and Templeton , 1967 ) , osmolarity ( Shin and Park , 1995 ) , cAMP-CRP ( Yokota and Gots , 1970 ; Silverman and Simon , 1974 ) , cell-cycle control ( Nishimura and Hirota , 1989 ) , RcsCDB ( Francez-Charlot et al. , 2003 ) , H-NS ( Bertin et al. , 1994 ) and integration host factor ( IHF ) ( Yona-Nadler et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Integration host factor ( IHF ) also appears to play a role in type 1 pilus expression ; mutations in either himA or hireD , which encode the a and [ 3 subunits of IHF , respectively , resulted in diminished inversion frequencies as well as reduced firnA transcription ( Dorman and Higgins 1987 ; Eisenstein et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8121417" target="_blank">8121417</a>]<br></li>
<li>The expression of the aceBAK operon , which codes for the key enzymes of the glyoxylate-cycle , isocitrate lyase and malate synthase , and for isocitrate dehydrogenase kinase/phosphatase , is activated by FruR and IHF and is repressed by IclR , which in turn is activated by FadR ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>The expression of the prpBCDE operon of enteric bacteria , which codes for 2-meth-ylcitrate synthase , 2-methylcitrate dehydratase , and 2-methyl-citrate lyase , which are key enzymes of the methylcitrate cycle , and propionyl-CoA synthetase , depends on PrpR , IHF , and NtrA ( 10 , 27 , 51 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>Model illustrating the roles of the TCS GlrR/GlrK , s54 and IHF for transcription of sRNA genes glmY and glmZ in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>These observations altogether indicate that YgiP has dual functions , playing the architectural role for genome folding and the global regulatory role in transcription of a number of genes , as in the case of the nucleoid proteins Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>mutant to investigate a possible role for IHF in TyrR-mediated regulation of folA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557822" target="_blank">17557822</a>]<br></li>
<li>IHF therefore plays an important auxiliary role in transcriptional regulation of certain genes ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>This is the first report of the role of IHF in enhancing biofilm-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>IHF plays an architectural role at several activated promoters , such as PilvP [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Prior studies have also shown that IHF stimulates the fim switch in E. coli K-12 strain CSH50 ( Eisenstein et al. , 1 Dorman and Higgins , 1987 ) , which is a timE mutant tha !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>IHF has also been implicated in packaging of I phage DNA ( Bear et al. , 1984 ; Feiss et al. , 1985 ) and in the modification of the expression of a number of E. coli ( Miller et al. , 1981 ; Gellert et al. , 1982 ; Friden et al. , 1984a ; Friedman et al. , 1984 ; Gamas et al. , 1987 ) and phage ( Hoyt et al. , 1982 ; Oppenheim et al. , 1982 ; Goosen & van de Putte , 1984 ; Krause & Higgins , 1986 ) genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>coli transcriptional units and IHF acts as both a positive and negative regulator ( Freundlich et al. , 1992 ; Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>For example , cyclic AMP receptor protein ( CRP ) and IHF are also known to play similar roles in gene regulation in other promoters ( Lobell and Schleif , 1990 ; Søgaard-Andersen et al. , 1991 ; Richet , 1996 ; Yang et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10027963" target="_blank">10027963</a>]<br></li>
<li>IHF is involved in transcription from some r54-dependent promoters ( Per-ez-Martin and deLorenzo 1997 ; Sze et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>IHF , a DNA-bending protein , plays a key role in the regulation of a number of stress response genes and is thought to be important for the cellular adaptations that occur upon entry into stationary phase ( Nash and Rob-ertson , 1981 ; Stenzel et al. , 1987 ; Altuvia et al. , 1994 ; Ditto et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>We emphasize , however , that the regulation of ® s expression is a complex phenomenon subject to growth phase-dependent and stringent control , and involving in addition to IHF the effects of other factors such as FIS itself , CRP , the superhelical density of DNA and , perhaps , NTP pools ( this study ; Ball et al. , 1992 ; Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>Also induced were the putative transcriptional regulator yiaG , induced by AI-2-mediator MqsR [ 27 ] , and ihfA ( himA ) , which together with ihfB ( himD ) , constitutes the global regulator IHF that is required for site-specific recombination , DNA-replication , transcriptional-control , and fimbriation [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>This may compare with the diverse role of IHF , which appears to be involved in site-specific recombination , transposition , plasmid replication , conjugal transfer and transcriptional-control of biosynthetic and catabolic operons ( Friedman et al. , 1984 ; Leong et al. , 1985 ; Dempsey , 1987 ; Morisato and Kleckner , 1987 ; Prentki et al. , 1987 ; Stenzel et al. , 1987 ; Thompson et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>Recent studies have shown that IHF is involved in other site-specific recombination events including integration of bacteriophage 4 80 and P22 ( Leong et al. , 1985 ) , transposition by IS1 ( Gamas et al. , 1985 ) and TnlO ( Morisato & Kleckner , 1987 ) , and phase variation in E. coli ( Dorman & Higgins , 1987 ; Eisenstein et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>IHF is a histone-like , sequence specific DNA-binding protein ( Craig and Nash 1984 ) which participates in a number of site-specific recombination events including integrative recombination of 2 ( Kikuchi and Nash 1978 ; * Present address : Department of Molecular Genetics , Smith Kline and French Labs , Philadelphia , PA 19101 , USA Miller et al. 1979 ) and q580 ( Leong et al. 1985 ) , transposition of TnlO ( Morisato and Kleckner 1987 ) and ISI ( Gamas et al. 1985 ) , and phase variation in E. coli ( Dorman and Higgins 1987 ; Eisenstein et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>IHF , a heterodimer encoded by the himA and himD ( hip ) genes , is a sequence-specific DNA-binding protein involved in bending DNA ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>IHF bends DNA by binding to specific nt sequences ( Nash , 1990 ) and is involved in site-specific rec ~ ) mbination , hns \ I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>Moreover , the binding site for IHF identiﬁed above is involved in this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>IHF is a sequence-speciﬁc , asymmetric , histone-like , heterodimeric protein ( encoded by himA and himD ) that plays important roles in replication , site-speciﬁc recombination , and transcription ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>HU and IHF are small , basic , histone-like proteins which are abundant in the cell and which are involved in a number of cellular-processes , such as site-speciﬁc recombination , replication , and transcription ( 8 , 11 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>Indeed , HU can replace IHF in a number of systems including IS70 transposition and DNA-replication at oriC in E. co / / ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Studies on the role of IHF in site-specific recombination in X have provided the basis for how IHF might participate in gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>In addition to its role in certain site-specific recombinations , IHF has also been implicated more directly in the control of gene-expression , for example , the regulation of the cIT gene of phage A ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>We have shown previously that this site has no role in uspA regulation ( 15 ) , and so we wanted to test if IHF and this site are involved in uspB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>Such regulation is not by site-specific recombination , and , indeed , IHF has not previously been shown to be involved in the control of gene-expression via phase variation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>While IHF was initially isolated as a protein of Escherichia coli involved in the higher-order assembly of a nucleoprotein complex required for the integrative recombination of bacteriophage A into the bacterial chromosome , it is now known that IHF is involved in a diverse set of cellular functions , including the regulation-of-gene-expression ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Extensive genetic and biochemical analyses have suggested a role for IHF in the regulation of several cellular functions , such as plasmid DNA-replication ( 8 , 13 , 14 ) , phage encapsidation ( l & 16 ) , recombination reactions involved in the movement of transposable elements ( 17 , 18 ) , fimbrial phase variation ( 19 , 20 ) , conjugal transfer of plasmids in E. coli ( 21 , 22 ) , and the posttranscriptional-control of bacteriophage X cI1 gene-expression ( 23,24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>IHF functions as an accessory factor in a wide variety of processes including replication , site-specific recombination and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF participates in a number of cellular-processes such as ) - site-specific recombination , transposition and inversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>IHF is involved in a variety of cellular-processes , such as site-speci ® c recombination of DNA , transposition of mobile genetic elements , replication of plasmids , nicking at oriT of conjugative plasmid DNA , or the control of the expression of several operons ( for a review , see Nash , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Subsequent work in X and other mobile genetic elements of E coli revealed that IHF participates in a wide range ot physiological activities including a variety of site-specific recombination events , phage packaging and partition , DNA-replication , and the expression of a number of genes -LCB- Friedman .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF participates in a number of processes involving DNA including site-specific recombination , DNA-replication , and positive or negative effects on gene-expression ( 10 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF also differs from typical E. coli regulatory factors by its participation in processes other than gene-expression , such as site-specific recombination , transposition , and DNA-replication ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF has subsequently been shown to participate in a wide variety of other processes in , including plasmid E. coli replication and partitioning , site-speciﬁc recombination , and gene-expression ( 4 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Besides the role it plays in the integration of the lambda genome into the host chromosome , IHF also participates in a variety of cellular-processes including other site-specific recombination events , phage packaging , replication , partitioning and transfer of plasmids and gene-expression [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Since its initial identification as a host protein necessary for lambda site specific integration , IHF has been demonstrated to be involved in numerous bacterial processes including phage DNA packaging and partition , DNA-replication , recombination , transposition , and gene-expression ( for reviews see [ 7 , 11 , 12 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>The integration host factor , IHF , of Escherichia coli participates in several cellular-processes , including gene-expression , site-speciﬁc recombination , transposition , phage packaging , and plasmid replication ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Less is known concerning the role of integration host factor ( IHF ) in E. coli , although it has been implicated in a number of processes in this organism including DNA-replication , site-specific recombination , and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Optical densities ( X ) and b-galactosidase activities ( ) were determined throughout the growth cycle in batch culture The e $ ect of ppGpp levels on the expression of hmp The e $ ect of IHF on rS-mediated regulation of hmp Integration host factor ( IHF ) of E. coli has been implicated in a number of processes including DNA-replication , site-speci ® c recombination and gene-expression ( Freundlich et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>Introduction Integration host factor ( IHF ) from Escherichia coli is a small , DNA-binding and bending heterodimeric protein involved in numerous bacterial processes including phage DNA packaging and partition , DNA-replication , recombination , transposition , and gene-expression ( Friedman , 1988 ; Freundlich et al. , 1992 ; Goosen and Van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>IHF is also involved in numerous other processes including packaging of lambdoid phages , DNA-replication in some plasmids and phages , and the regulation of expression of a number of phage and E. coli genes ( reviewed in Drlica and Rouviere-Yaniv 1987 ; Friedman 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>IHF has been found to play a role in a variety of cellular-processes , including the regulation-of-gene-expression ( for reviews see Drlica and Rouviere-Yaniv , 1987 ; Friedman 1988 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>An interesting comparison can be made with the IHF protein , which is also involved in different types of cellular-processes , including regulation , recombination and replication and which is also known to induce DNA bending ( for a review see Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1640456" target="_blank">1640456</a>]<br></li>
<li>( IHF is a small histonelike protein shown to be play a role in recombination , DNA-replication , and gene regulation [ for a review , see reference 8 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The ihfA ( himA ) gene , encoding the c ~ subunit of IHF , is located downstream from a group of genes ( thrS infC rplTpheS pheT ) coding for proteins involved in translation ( Miller , 1984 ; Mechulam et al. , 1985 ; Wertheimer * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>This lesion proved to be a mutation in the himD gene , encoding the , ( subunit of IHF ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>An insertion in the himD gene encoding the b subunit of IHF ( himD : : cat ) was transferred by P1 vir transduction into strain LP1000 , which contains a gltB : : lacZ transcriptional fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Ó Springer-Verlag 1997 Mol Gen Genet ( 1997 ) 254:599 ± 603 S H O R T C O M M U N I C A T I O N J. Membrillo-Herna Â ndez á G. M. Cook á R. K. Poole Roles of RpoS ( rS ) , IHF and ppGpp in the expression of the h encoding the ¯ avohemoglobin ( Hmp ) of Escherichia coli K-12 mp gene Received : 9 September 1996 / Accepted : 8 January 1997 Abstract The Escherichia coli K-12 gene hmp encodes the ¯ avohemoglobin Hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>Membrillo-Herna ́ndez , J. , Cook , G.M. , and Poole , R.K. ( 1997b ) Roles of ss , IHF and ppGpp in the expression of hmp encoding the ﬂavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_184">
Roles of RpoS ( S ) , IHF and ppGpp in the expression of the hmp gene encoding the ﬂavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></label>
<input id="rp_similarity_184", type="checkbox">
<div class="RP_similar">
<li>Membrillo-Hernandez , J. , Cook , G.M. , and Poole , R.K. ( 1997 ) Roles of RpoS ( sigmaS ) , IHF and ppGpp in the expression of the hmp gene encoding the ﬂavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>[ 23 ] Membrillo-Hernandez J. , Cook G.M. , Poole R.K. , Roles of RpoS ( sigmaS ) , IHF and ppGpp in the expression of the hmp gene encoding the flavohemoglobin ( Hmp ) of Escherichia coli K-I 2 , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
<li>Appl Environ Microbiol 78:3369 -- 3378 Membrillo-Hernández J , Cook GM , Poole RK ( 1997 ) Roles of RpoS ( sigmaS ) , IHF andppGpp in the expression of the hmp gene encoding the flavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23824318" target="_blank">23824318</a>]<br></li>
<li>FEBS Lett 382:141 -- 144 Membrillo-Hernández J , Cook GM , Poole RK ( 1997a ) Roles of RpoS ( S ) , IHF and ppGpp in the expression of the hmp gene encoding the Xavohaemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>-80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC DISCUSSION osmY ( csi-5 ) is one of many stationary-phase-induced and osmotically regulated genes in E. coli that require us , the gene product of rpoS , for expression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>R. ( 1993 ) Complex transcriptional-control of the a ^ - dependent stationary phase-induced osmY -LCB- csi-S ) gene suggests novel roles for Lrp , cAMP-CRP and IHF in the stationary phase response of Escherichia coli J Bacterioi MS : 7910-7917 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997184" target="_blank">7997184</a>]<br></li>
<li>Other abundant nucleoid-associated proteins such as IHF , H-NS , and Dps/PexB also have been shown to control the expression of a variety of speciﬁc genes ( 1 , 9 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7860604" target="_blank">7860604</a>]<br></li>
<li>Global regulators such as H-NS ( 27 , 28 ) , OmpR ( 30 ) , and IHF ( 36 ) control the expression of the curli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>In addition to these three two-component systems , global regulators such as H-NS ( 1 , 28 , 36 ) and IHF ( 22 ) control the expression of the curli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>We examined the effects of some global regulators ( factor inversion stimulation [ FIS ] , H-NS , OmpR , cAMP , ppGpp , and IHF ) on the expression of sra by using global regulator-defec-tive mutants lysogenized with lpF13 ( Psra-lacZ ) or carrying pFF6 ( Psra-lacZ ) and by measuring b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>These results suggest that FIS , IHF , cAMP , and ppGpp are involved in the positive regulation of sra expression during the stationary growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Thus , many global regulators , including cAMP , H-NS , IHF , and ppGpp , inﬂuence the expression of sigma S , which in turn may regulate the transcription of sra in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>DISCUSSION We have established that IHF is essentifaolr the anaerobic expression of the nitrate reductase ( narGHJZ ) operon of E. coli by the Fnr and NarL activator proteA inpsa.rtial deletion of the h i d gene , which encodes the a subunit of the heter-odimeric IHF protein ( 30 ) d , ecreased narG-lac2 expression in cells grown anaerobically , and it alssoeverely impaired nitrate induction of narG expression by NarL ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Analysis of this mutation , along with authentic himA ( IHF a subunit ) and himD null that IHF is essential for nitrate alleles , showed function induction of nar operon expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Global regulators , such as RpoS ( ss ) , IHF and Fnr , are also involved in the expression of hmp ( Poole et al. , 1996 ; Membrillo-Herna ́ndez et al. , 1997b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>This view derived primarily from the fact that Lrp tends to activate some operons involved in biosynthesis and repress others involved in catabolism .1 Another view advanced by D'Ari et al. 13 is that Lrp is primarily a DNA-organizing protein , much like HNS , IHF , and HU , which also affect gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>They also participate in diverse roles such as regulation-of-DNA-replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Panels ( A -- D ) show expression data reﬂecting DNA looping in ( A ) WT E. coli , ( B ) cells lacking HU , ( C ) cells lacking IHF and ( D ) cells lacking H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>We discuss the possible physiological relevance of the involvement of IHF in gene-expression and the hypothesis that IHF is a member of a class of functionally redundant proteins that participate in chromosome structure and multiple processes involving DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>They are involved in regulating gene-expression ( by functioning as transcription terminators , by stabilizing mRNA and by acting as topological insulators for transcription-induced positive supercoiling ( 2 -- 5 ) , and in structuring DNA ( by binding proteins such as IHF , PolI and DNA gyrase ) ( 6 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22199259" target="_blank">22199259</a>]<br></li>
<li>We have reported previously that IHF negatively regulates ompF ( 44 ) and ompB ( Tsui et al. , submitted ) , two additional operons involved in the expression of the E. coli major outer membrane proteins ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>It is interesting that of the 10 operons in E. coli whose expression is directly altered by IHF , 4 are involved in anaerobic metabolism ( 10 , 32 , 36 , 39 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Taken together with the published data these results indicate that IHF is involved in altering the expression of many genes in E. coli and that many of these effects may be mediated directly by IHF binding to promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>` IHF appears to be involved in the expression of an unusually large number of operons and many of these operons are not physiologically related in any obvious way ( Tables 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Also , IHF mutants were reported to have reduced expression of genes involved in the synthesis of flagellin ( Szekelye/a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>One-Sentence Summary : IHF is a new regulator of the proU promoter involved in the fine tuning of its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Sohanpal , B. , Friar , S. , Roobol , J. , Plumbridge , J. , and Blom-ﬁeld , I. ( 2007 ) Multiple co-regulatory elements and IHF are necessary for the control of ﬁmB expression in response to sialic acid and N-acetylglucosamine in E. coli-K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Of the seven most-global regulators , together affecting expression of about half of all E. coli genes , two were previously shown to exert direct , positive control on gltBDF transcription : Lrp and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Just seven proteins ( ArcA , Crp , FIS , Fnr , IHF , Lrp , and NarL ) directly control expression of about half of all genes in E. coli [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Firth Court , Western Bank , Sheffield SIO 2TN , UK Key words : Ubiquinone biosynthesis ; Ubi genes ; Gene expression ( E. coli ) ; IHF ; Fnr 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9315722" target="_blank">9315722</a>]<br></li>
<li>IHF influences the expression of a surprisingly large number of apparently physiologically unrelated genes and o , ~ erons in E coli [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>IHF has been shown to have a direct positive or negative role in the expression of a number of genes in E. coli , X , and Mu ( reviewed in references 5 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>This protein , which is essential for normal expression of narKlacZ in vivo , apparently binds to narK regulatory DNA site ( s ) and may bend it as has been demonstrated for IHF regulation of other genes in E. coli ( 32 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>IHF is a histone-like protein with an auxiliary role in the transcriptional-control of several genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Leucine represses the synthesis of the enzymes required for isoleucine-biosynthesis ( AHAS ) III , while expression of the other enzymes in the pathway requires active IHF ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>RESULTS Effectof a h i d Mutationon inVivo narGHJI Operon Expression-To determine if IHF protein has aphysiological role in regulation of the nitrate respiratory narGHJI operon , we tested the effect of a h i d mutation on the expression of a narG-lac2 reporter fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Evidence that IHF plays a role in the expression of the carAB operon of both E. coli and S. typhimurium was obtained by assaying CPSase ( chromosome encoded ) and [ 3-galactosidase ( plasmid encoded ) in different de-rivatives of a Car + Lac - E. coli strain harbouring a single-copy plasmid carrying a fusion between the car control region of S. typhimurium and the lacZ gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Thus , IHF plays a dual role in controlling fimA expression : it is required both for inversion of the fimA control region and for efficient expression from the fimA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>To examine the role of IHF in fimA expression , the himA82 : : TnlO insertion was introduced by Pl-mediated transduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>The question remains as to the mechanistic role IHF plays in the regulation of dmsA expression in response to anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>IHF plays a minor role in regulating napF-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>) To determine if IHF plays a role in regulating napF-lacZ expression , we transduced a himA allele into wild-type ( MC4100 ) and modE ( PM8 ) strains that harbor HW2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Precisely how IHF works inpfi and other anaerobic operons awaits the development of in vitro systems that will allow binding and transcription analysis with all of the factors that participate in the expression of each operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF often works in conjunction with other NAPs and transcription factors to tune gene-expression ( Browning et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>These observations are consistent with IHF acting as both an aerobic and anaerobic repressor of ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>However , both are regulated by IHF , which acts as an anaerobic repressor of both nuoA-N and ndh to provide a degree of co-ordinated expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>IHF was also required for Fnr-independent anaerobic control of narGHJI expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>IHF has also been shown to be required for positive regulation of tdc operon expression ( Wu et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>ihe IHF protein itself which tunes the expression of an operon , but the regulator protein that is supported in its function by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF is also a transcriptional regulator of the ompB operon whose protein products , EnvZ and OmpR , are required for expression and osmoregulation of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>We wished to determine whether the IHF protein is also involved in regulating F ( narP-9lacZ ) and F ( narQ-lacZ ) expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>The IHF protein is also involved in regulating sucABCD gene-expression ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209026" target="_blank">9209026</a>]<br></li>
<li>This and other observations indicate that IHF is directly involved in nar operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>It is also possible that , earlier in bacterial evolution , IHF was involved in ast operon expression and that ArgR preempted this function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>Under inducing conditions , IHF activates aceB : : lacZ expression by opposing IclR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>In contrast , IHF has little effect on aceB : : lacZ expression under repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>A defect in the himA gene that encodes one of the subunits of IHF resulted in a threefold elevation of icd-lacZ expression during anaerobic cell-growth ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209047" target="_blank">9209047</a>]<br></li>
<li>A strain deleted for the gene encoding the b-subunit of IHF also showed a drastic reduction in TcbC-dependent induction of anaerobic tdc expression ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>We have also investigated the effect of IHF on expression of the sodA gene by monitoring sodA-lacZ expression in a AhimA strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>As mutations in the structural gene for IHFa clearly affected expression of FD ( fimA-lacZ ) , we anticipated that mutations in himD ( also known as hip ) , which encodes the , B subunit of IHF , would have a similar effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>We constructed a series of isogenic strains carrying various himD and himA alleles to study the requirement of IHF for expression of 4D ( narG-lacZ ) and 4 ( fdnG-LacZ ) gene fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Mutations in either the himA or the himD ( hip ) gene , which encode the a and I8 subunits of IHF , respectively , prevented phase variation and locked expression of fimA in either the `` on '' or `` oilf '' phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>Our results do not exclude the possibility that IHF may also have an indirect e $ ect on cysJIH transcription , for example , by altering the expression of the cysB gene encoding the regulatory protein CysB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>We also discus evidence which indicates that the function of IHF in gene-expression in E. coti may be fundamentally different from that of classical global transcription regulators such as LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The objective of this study was to determine whether IHF by itself functions as a global regulator of stationary-phase-induced gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Recent experiments indicate that IHF participates in stationary-phase-induced expression of fhe osmY gene and the dsp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Most investigators have used single mutants ( either ihfA or ihfB ) to demonstrate the effects of IHF deficiency on gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>It has to be underscored that in this system the IHF effect is not mediated by a repression in the expression of the regulator gene glcC ( see Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>Therefore , we examined the role of IHF for expression of E. coli glmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Role of IHF for expression of glmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>IHF-binding sites and operon expression in E. coli Early work established that , in addition to its role in sitespecific recombination in X , IHF was also involved in influencing gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>As discussed above , IHF has few properties of a typical regulatory protein and it is not known why IHF participates in gene-expression in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>At approximately the same time studies suggested that IHF was involved in gene-expression in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>A number of hypotheses can be considered to account for the involvement of IHF in gene-expression in E , coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF also participates in regulation-of-gene-expression in a number of phages and in E. coli ( reviewed in references 10 and 14 ) and some other gram-negative bacteria ( 6 , 18 , 25 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>In addition to negative regulation , IHF can have a positive role in gene-expression ( reviewed in reference 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>IHF is a direct negative effector of gene-expression in a number of other systems ( 26 , 32 , 33 , 43 , 45 , 55 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Perhaps IHF has a special function in anaerobic gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF as a negative regulator of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>This results in a negative effect of IHF on gene-expression [ 56 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>However , a number of observations suggest a much broader role for IHF in gene-expression in this organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Therefore , IHF serves a nonessential role in modulating napF gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>As noted above , IHF acts as a positive regulator while H-NS is a potent repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Histones , HMG , HU , IHF : meme combat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22942248" target="_blank">22942248</a>]<br></li>
<li>Examples of IHF acting as a repressor of E .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>) , IHF ( neg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>) , Fur , ArcA , IHF ( neg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Bacterial strains containing lesions in either alpha ( IhfA ) or beta ( IhfB ) subunits of IHF show marked changes in expression of many genes and regulation of many cellular-processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Nomenclature of the genes encoding IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>Six proteins ( Fig. 2 , proteins 15 to 20 ) , including fumarase , glycerol-6-phosphate kinase , outer membrane protein C ( OmpC ) , and DnaN , exhibited higher expression levels in the himA background , indicating that IHF , directly or indirectly , acts as a negative modulator of the genes encoding these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>A role for IHF or H-NS in regulating ftnA expression was tested by Western blot analysis of the corresponding himA ( one of two unlinked genes encoding the heterodimeric IHF protein ) and hns mutants , and parental strains ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Since the H-NS , HU , IHF and Fis proteins participate in a wide variety of cellular-processes , careful control of their intracellular amount at a given time during the bacterial growth cycle is obviously of great importance for cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response-to-oxidative-stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>IhfB subunit of IHF , two other nucleoid binding proteins which are global regulators as well , also showed up as hits in the screen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18519731" target="_blank">18519731</a>]<br></li>
<li>It involves repression by the GB regulator ICLR and activation by the nucleoid protein IHF and the global regulator Cra ( also known as FruR ) ( reviewed in reference 95 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The global regulator CRP , like the nucleoid protein IHF , has many low-affinity sites on the chromosome ( Grainger et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Histone-like proteins , such as IHF , H-NS , HU , and Fis , play a major role in modifying DNA conformation , changing the access of certain regulators and/or RNA polymerase to promoters ( 2 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>Other regulators ( AppY , Fis , H-NS , StpA , and IHF ) The AppY protein Ž acid poly phosphatase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>For operons involved in use of carbon sources in Escherichia coli , the most frequent global regulator is cAMP/CAP , responsible for catabolite repression , but histone-like proteins , IHF , H-NS , Fis and HU are often involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>HU The histone-like protein from E. coli strain U93 ( HU ) ( Drlica and Rouviere-Yaniv , 1987 ) is closely related to IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>DISCUSSION The nucleoid-binding proteins of E. coli , assimilated to his-tone-like proteins , include HU , H-NS , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171392" target="_blank">9171392</a>]<br></li>
<li>The E. coli HU and IHF proteins : Accessory factors for complex protein-DNA assemblies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544184" target="_blank">11544184</a>]<br></li>
<li>Nash , H. A. ( 1996 ) The E. coli HU and IHF proteins : accessory factors for complex protein-DNA assemblies , in Regulation of Gene Expression in E. coli ( Lin , E. C. C. , and Lynch , A. S. , Eds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15807530" target="_blank">15807530</a>]<br></li>
<li>Recently , IHF has been implicated in regulation of superoxide dismutase ( SodA ) , an enzyme which protects cells from oxidative stress ( Compan and Touati 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>Some known systems include the regulation of nifDHK operon ( encoding a nitrogenase ) by NifA ( a specific activator ) and IHF ( a global modulator ) in Klebsiella pneurnoniae ( Hoover et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The complex transcriptional regulation of the csgDEFG operon will also have to be considered , which occurs in stationary phase and involves several global regulators such as RpoS , HNS , IHF and OmpR ( Barnhart and Chapman , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>Our set of 11 global regulators are transcription factors involved in carbon degradation ( mlc and lrp ) , redox status sensing ( arcA , narL and FNR ) , ion-transport regulation ( fur ) , environmental sensors ( cspA and CRP ) , a regulator of structural proteins ( hns ) and two general enhancers ( IHF and FIS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Similarly , it has been demonstrated that IHF is involved in stationary-phase regulation of the osmY ( 16 ) and dps ( 1 ) genes and that additional factors , including Lrp , Cap , and RpoS ( osmY [ 16 ] and dps [ 1 ] ) participate in their regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Whereas rs , Lrp , CRP , and IHF are clearly involved in stationary-phase induction , none of these regulators is essential for osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>IHF has been shown to be important in mediating AtcA-dependent regulation of the f / / gene ( Sirko etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>Additionally , the arginine repressor , ArgR , has been shown to be involved in the transcriptional regulation of the ast operon in S. enterica serovar Typhimurium ( 23 ) , and IHF is involved in the regulation of some NR 54 I - and - dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>To examine whether other transcription factors regulate Pfnr , we tested whether the DNA-bending protein IHF had an effect on fnr transcription , since it has been shown to be involved in the regulation of several FNR-dependent promoters , such as narG , nir , nrfA , dmsA , ubiC , ndh , pﬂ , sodA , and narK ( 5 -- 7 , 9 , 19 , 31 , 36 , 43 , 48 , 51 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Host-encoded proteins are also involved in this process , including IHF and Fis that bind and bend DNA in order to assist intasome formation [ 9,18 -- 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20949106" target="_blank">20949106</a>]<br></li>
<li>Editor : Andre Klier ABSTRACT IHF is a protein of the bacterial nucleoid proteins ( NAPs for nucleoid-associated proteins ) involved in DNA structuring and transcription regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>IHF has been shown to be involved in transcriptional regulation of many operons on its own or in concert with additional activator proteins ( 14 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>So far we have many promising links between phage activation and Hha ; for example , some of the prophage genes that Hha regulates are related to Hha toxicity and some of those genes , like alpA and ihfA , have a role in phage activation ( alpA is associated with the excision of CP4-57 [ 45 ] , and IHF is essential for the lysogenic life cycle of lambda phage [ 64 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>The appearance of the transformant colonies was reminiscent of Phase-locked-ON derivatives of parental VL386 harbouring mutations in the genes encoding IHF , a cofactor thought to be essential for fimA promoter segment inversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
<li>Bacterial strains disrupted in genes encoding nucleoid proteins Strain Relevant genotype Designation Comment Strep derivative of CSH142 [ araD ( gpt-lac ) 5 ] R FW102 hupA hupB FW102 hns FW102 himD FW102 a BL643 BL706 BL707 WT HU U heterodimer Loss of both HU-1 and HU-2 subunits of H Loss of H-NS homodimer Loss of IHFb subunit of IHF heterodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">IHF binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>sigma scores boxes evidence 1 glkP1 - P2 k n glk FruR C [ 36 ] , in this work GCGTTGTTGTTATGCCCCCAGGTATTTACAGTGTGAGAA 236 s38 2 s70 : 3.92 glkP3 - P4 n n s70 3 in this work A , C CCCCCAGGTATTTACAGTGTGAGAAAGAATTATTTTGAC 221 s38 : 2.95 pgiP1 - P2 k n [ 37 ] , in this work A , B , E ¬ pgi ACCATCACATTTTTCTGTGACTGGCGCTACAATCTTCCA 236 s70 : 6.9 ; s38 : 7.44 pfkAP1 - P2 k n ¬ A , B FruR pfkA [ 39 ] , in this work ATCAATTCAGCAGGAAGTGATTGTTATACTATTTGCACA 278 s70 : 8.61 ; s38 : 5.7 pfkAP3 - P4 n n s38 2 in this work A , E TCACTTCGATGTGCAAGAAGACTTCCGGCAACAGATTTC 229 s70 : 1.12 ki CTGCCCTGCGGGGCGGCCATCTTCCTTTATTCGCTTATA 219 ¬ tpiA tpiAP [ 40 ] ¬ ¬ tpiAP1 - P2 in this work n n ¬ A , E CAAAGCCTTTGTGCCGATGAATCTCTATACTGTTTCACA 262 s70 : 8.07 ; s38 : 7.4 gapA gapAP1 - P1a [ 41,43 ] , in k n this work gapAP2 [ 41,43 ] k ¬ A , B , D CGCTTGACGCTGCGTAAGGTTTTTGTAATTTTACAGGCA 236 s70 : 3.76 ; s38 : 0.82 ¬ A CCTTTAAAATTCGGGGCGCCGACCCCATGTGGTCTCAAG 2153 ¬ k TCACATTTTTATCGTAATTGCCCTTTAAAATTCGGGGCG 2174 - - ¬ gapAP3 [ 41,43 ] Crp k AACACCAACTGGCAAAATTTTGTCCTAAACTTGATCTCG 2245 - - ¬ gapAP4 [ 41,43 ] ACATTCCTTTTATTCCACGTTTCGCTTATCCTAGCTGAA 2132 no predictions s703 , s382 A , D epdP1 - P2 k FruR , Crp epd [ 45 ] , in this work pgkP1 k A pgk [ 45 ] AAGCAGCACAAGGTGCATTTCATGGTATAGTTGACTATA 2235 - ¬ n TGTTGCTTTCAGGTAAGACGCAAGCAGCGTCTGCAAAAC 226 no predictions ¬ pgkP2 in this work D , E n GCACACCTGATCAAAACGTTGGTCTGGTGCGATAACGAA 2113 no predictions s703 pgkP3 in this work A , D gpmA gpmAP1 - P2 [ 46 ] , in this ki n work gpmAP3 - P4 in this work n n Fur A ¬ AAGCATTGCTGTTGCTTCGTCGCGGCAATATAATGAGAA 237 s70 :7.56 ; s38 : 3.66 ¬ ¬ CCTTACACTGCGCCACTATTTTCGCTATGGTTATGCGTA 275 s70 :5.80 ; s38 : 7.48 eno enoP1 [ 47 ] k ¬ ¬ AATCGTGTCAGCGTCAACATCAAACTGATCGATTCACAA 2731 ¬ k CGAAGCACTGAAACACGGTGGGCTGAAGAATCGTGTCAG 2760 - - ¬ enoP2 [ 47 ] k TAAACGATTCAGCTTAAACTGCCCGGAAGCGAATCTGTC 2903 - - ¬ enoP3 [ 47 ] n GCGTACCCTGGGTACGCGTTGTTTGTCTGGAGTTTCAGT 227 no predictions s703 , s382 A , D , E enoP4 - P5 in this work n GAGTTCCAGAAACGTCAGGCGAAGTAAGTAAAAAAGTTA 277 no predictions - D enoP6 in this work n n GGGGATGATCAGTTGGTCGAGATCATCGAAGTTCCGAAT 2212 no predictions s703 , s382 D enoP7 - P8 in this work pykF pykFPPki [ 49 ] FruR TGTTTTCCTTTTGGATTAATTTCAGCGTATAATGCGCGC 2231 - -- pykFP1 - P2 in this work n n GGATTCGCTTTCCGGCAGTGCGCCCAGAAAGCAAGTTTC 234 no predictions s703 , s382 A , B , D n GATGTCACCTATCCTTAGAGCGAGGCACCACCACTTTCG 280 no predictions - D pykFP3 in this work pykFP4 - P5 n n in this work CCTCTGCACGCTTTTTCGATGTCACCTATCCTTAGAGCG 297 s70 : 4.79 ; s38 : 9.95 ¬ B , D pykA pykAP1 - P2 n n s38 2 in this work AAGTGACGATCGCTAAAAACGACTGTCACTGTCCTAATC 2105 s70 : 0.90 , D B n n GTCAAAGAAGCGCTGAAGGAATCGCGTTTTGATAAGCAG 2153 s70 : 1.76 s382 A , B , D pykAP3 - P4 in this work Crp , Fnr , GTATGGACATAAGGTGAATACTTTGTTACTTTAGCGTCA 258 no predictions s703 , s382 B , D PdhR s38 2 pdhR pdhRP1 - P2 k n [ 51 -- 52 ] , in this work aceE aceEP1 - P2 k n ¬ AACCTGTCTTATTGAGCTTTCCGGCGAGAGTTCAATGGG 248 s70 : 1.90 [ 50 ] , in this work poxB poxBP1 - P2 k n [ 57,58 ] , in his work B , C ¬ MarA , SoxS TCCCTTCCCCCTCCGTCAGATGAACTAAACTTGTTACCG 227 s70 : 5.31 ; s38 : 9.5 acs acsP1 k [ 60 -- 61 ] ¬ Fis , Crp , Ihf TCTTTAATCAATTGTAAGTGCATGTAAAATACCACTTTA 2224 ¬ k n CCCCTACATTTAACGCTTATGCCACATATTATTAACATC 220 s70 : 4.03 ; this work s38 : 4.97 B , C ¬ acsP2 - P3 [ 60 -- 61 ] , in Fis , Crp , Ihf consensus sigma 70 : TGGTATAATG sigma 38 : CTACACTT sequences Proposed promoters in which putative s and s overlapping recognition sequences have been determined , are presented as bold sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>A 81 bp fragment containing the K23 derivative of the ihf site located at the right extremity of IS1 was obtained by PCR ampli ® cation using as template pIS1IRR ( a pBluescript derivative carrying the right extremity of IS1 ; Murtin , 1996 ) and the oligonucleotides 50-CCGCGG-TGGCGGCCGCTC-30 and 50-CGACTCACTATAGGG-CG-30 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The black box depicts theIHF target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>The black boxes depict theIHF target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>end finE GMGAT1 TTT ATTG AAGCAAAACAATAATACAAGACAATTGGGGGCCAT1M GACTCATAGAGGAAAGCACGCGAACAAACT IM CAGM IRL VAL *** IHF SITE AITGTTGGCrTMATATTCTATTGTTATCMAMATAGATGMATATTGCATGAGGTGGI I I ITTGGAGAGAAGAATGAGGAAGATGCGTCGAGCCACAGAAACGT A TPACTFrACATAT/GCGGAGGTGATGTG TTAMA AA CAG TGCI I I I IGTCGT111T MTAITIFIATGCTTGA IHF SITE GMAAMTCGTMCAmATGTATGGCAGmGCCCCA ATGMCATCTITII ` GGGGGAAACTGTGCAGTGTTGGCAGTCAAACTCGMACAAAACAAA IRR GTGTACAGMCGACiGCCCATGTCGAMAIATMAG I I I ITAGAGAIGGNACAGCATGAA fIIELYS FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>These species-speciﬁc sequences represent a family of complex prokaryotic repetitive extragenic palindromes , called RIB ( stands for reiterative IHF BIME [ bacterial interspread mosaic elements ] ) ( 6 , 16 , 40 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_7">
In panel B the short vertical bar at the top and the long vertical bar at the bottom represent DNA sequences protected by CAP and IHF , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></label>
<input id="sit_similarity_7", type="checkbox">
<div class="SIT_similar">
<li>In panel A the short vertical bar at the top and the long vertical bar at the bottom represent DNA sequences protected by IHF and CAP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li></div>
<li>The pellet was discarded , the supematant was adjusted to 70 % saturation in ammonium sulfate and stirred for a further iragment A1 fragment # 2 fragment # 3 - b 00p 100 bp Miul Alul Asel Alul BgUi - IHF FNR ort pft A. L L ~ ~ ~ ~ ~ ~ ~ ~ LLL4 P7 PS PS PA P3 P2 P1 ... ... ... .132 44-94 GTTAATTTTTAAAATTTATTTTTATTTGGATAATCAAATATTTACTCCGTATTTGCATAAAA CAATTAAAAATTTTAAATAAAAATAAACCTATTAGTTTATAAATCACGCATAAACGTATTTT -141 -103 It B C. GTTAATTTTTAAAATTTATTTTTATTTGGA * ATCAAATATTTA TCCGTATTTGCATAAAA C TT C FIG. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>This DNA construct replaces most sequences upstream of the ihf site centered at 2114 with plasmid DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>( 1990 ) , in which not only the consensus sequences but also the flanking sequences ot 27 known ihf sites are used , reveals a higher score for the alternative sequence ( 39.3 ) than for the one that was published ( 31.2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Thus , the iow affinity of iHF binding to the himD/hip and himA P4 promoter regions could be the result of both the deviation from consensus and the sequence context ot the binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Upon closer examination of the DNA sequence from 2166 to 281 shown here to be required for maximum ﬁs P activity , we noticed a close match to the ihf consensus sequence , WATCAANNNNTTR ( 10 , 15 , 21 , 22 ) , located from 2114 to 2126 ( Fig. 2B , bottom strand ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>This region contains a sequence from 276 to 264 that also resembles the ihf consensus sequence ( 21 ) , although not as closely as the one from 2126 to 2114 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Nucleotide sequences resembling the ihf consensus sequence are underlined for both the ihf site at 2114 and a weaker site at 264 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>This region contains a good match to the ihf consensus sequence ( 10 , 15 , 21 ) , and indeed , DNase I protection experiments revealed an ihf-binding site extending from 2135 to 294 ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Effect of ihf site position on b-galactosidase activity from ﬁs P Insertion RZ211 RJ1561 MC4100 HP4110 Fold IHF ( bp ) a b b d b-Galb % Activityc b-Galb % Activityc b-Gal b-Gal activation Plasmida pRJ1028 0 126 6 4 100 722 6 6 35 119 6 9 3.8 pKW125 5 33 6 1 26 352 6 18 49 107 6 8 110 6 15 1.0 pKW232 10 49 6 2 39 532 6 22 74 189 6 20 107 6 30 1.8 a ﬁs P regions in these plasmids contained the wild-type sequence from 2373 to 180 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>A sequence pertaining to an ihf site is indicated by boldface type to transcription initiating from the previously designated P2 , P3 , and P4 promoters or any other promoter upstream of ﬁs P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679232" target="_blank">14679232</a>]<br></li>
<li>coZz ' strain or plasmid Relevant genotype or marker Reference Dente e t al. ( 1983 ) Auger et al. ( 1989 ) Auger e t al. ( 1989 ) Shi e t al. ( 1993 ) Casadaban & Cohen ( 1979 ) Atlung & Brandsted ( 1994 ) Flamm & Weisberg ( 1985 ) This work Ap ' Tc ' APT Sutcliffe ( 1978 ) Dente e t al. ( 1983 ) Auger et al. ( 1989 ) This work Mu d5005 meP adiA + Km ' pX52 with two mutations in the IHF consensus site in the adiA promoter 4.3 kb SalI fragment from pKER110 cloned in pEMBL8 + Same as pKS4 but cloned in the opposite orientation 1-23kb EcoRI PCR fragment containing adiY cloned in pEMBL8 + 0.945 kb EcoRI-PstI fragment containing envy cloned in pBR322 2 kb HindIII-SmaI fragment containing appY cloned in pUN121 Ap ' Tc ' Sm ' Sp ' 166 bp EcoRI-BamHI PCR fragment of the adiA promoter cloned in pXA 144 bp EcoRI-BamHI PCR fragment of the adiA promoter cloned in pXA 274 bp EcoRI-BamHI PCR fragment of the adiA promoter cloned in pXA 124 bp BamHI-Ss-1 PCR fragment of the adiA promoter cloned in pXA pKS4 Stim & Bennett ( 1993 ) pKSl1 This work pKS015 This work pML22 Lundrigan & Earhart ( 1984 ) pTAC3303 Atlung et al. ( 1989 ) pUN121 PXA PX1/4 Nilsson e t al. ( 1983 ) de Lorenzo e t al. ( 1988 ) This work This work PXU5 pX52 This work pX-B/S -1 Stim & Bennett ( 1993 ) 14000 r.p.m. for 30 s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>y Binding n-Regulation of the Escherichia coli K-12 nrf Promoter b of the NsrR Nitric Oxide-Sensing Transcription Repressor to an Upstream Site Douglas F. Browning ,1 * David J. Lee ,1 Stephen Spiro ,2 and Stephen J. W. Busby1 School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom ,1 and Department of Molecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 750802 ow D Received 1 March 2010/Accepted 9 May 2010 FNR-dependent activation of the Escherichia coli K-12 nrf promoter is downregulated by the nitric oxide-sensitive NsrR protein together with the nucleoid-associated protein IHF , which bind to overlapping targets adjacent to the DNA site for FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The sites for the two proteins overlap by five base pairs , but this overlap is probably tolerated because OxyR is likely to bind the major groove ( M , B , Toledano , et ai , unpublished ) and IHF binds the minor groove ( Yang and Nash , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Purification of the separate subunits of IHF showed that HimA and HimD can also specifically bind to ihf sites as homodimers ( Zulianello etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>To test if IHF can bind speciﬁcally to these putative target sites , we performed DNase I footprinting experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>Low similarity score [ 50 ] calculated for some of the footprinted IHF sites suggested that IHF might bind to some of these sites in the ` quasi specific ' or in the cooperative manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>IHF and Fis proteins bind to multiple sites within pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Finally , three IHF recognition consensus sequences ( Nash & Granston , 1991 ) were apparent a t positions 7 to -6 , -55 to -67 a n d -130 t o -142 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>Both sequences match the consensus eight out of ten base-pairs , and all the positions shown to be in close contact with the IHF are strictly conserved ( A5 , TlO and Tll ; Craig & Nash , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Location of the experimentally determined ModE and IHF recognition sequences at dmsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>We propose that these increases are primarily a result of relieving repression from P2 as , when IHF and ModE are bound at dmsA , they would be predicted to occlude the ¹ 35 and ¹ 10 consensus sequences respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Sequences protected by IHF and CAP during footprinting ( 24 , 26 ) are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>The regions protected by IHF are shown by solid lines and [ he proposed IHF-reoogniiion sequences are shown by a dotted underline , A lObp inverted-repeat in the himD/hip promoter region is shown by arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Open boxes denote sequences presumed to represent the 210 and 235 promoter regions for ﬁs P. DNA regions protected from DNase I cleavage by Fis ( ■ ) or IHF ( u ) are indicated below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>The boxes represent the sequences protected by IHF and NARL-P in DNase protection studies by Walker and DeMoss ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>Determining how ArcA , Fnr , IHF and RNA polymerase bind to their respective recognition sequences in the presence of the other proteins will obviously bring us a step further towards achieving this objective .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>For most of the nrf promoters , upstream sequences are conserved , but the S. enterica serovar Typhi-murium promoter possesses large insertions within IHF II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Further gel retardation assays using purified IHF protein ( Fig. 3C ) confirmed that IHF binds to the EcoRI ± NsiI fragment carrying upstream pnir sequences ( lanes 1 ± 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>In contrast , IHF binds to upstream sequences , performing some as yet unknown inhibitory role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>To evaluate this possibility , we asked whether two proteins known to bend DNA with which they interact , Integration Host Factor ( IHF ; reviewed in Friedman , 1988 ) and Fis , could bind specifically to rrnB P1 upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2209559" target="_blank">2209559</a>]<br></li>
<li>On the basis of the above results , we conclude that IHF , as well as Fis , can bind to upstream sequences at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Also , the DNA flanking this consensus seems to be important for the formation of stable IHF-DNA complexes , as sequences have been found which match the IHF consensus but which bind IHF only pcorly ( van Rijn etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Although evidence is lacking , it is conceivable that IHF could bind A+T - rich proximal sequences in other E. coli promoters such as fdhF and hyd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This suggests that IHF does not bind speciﬁcally to the linear DNA sequences of the mtr and tyrP13 * promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>If IHF plays an architectural role in bringing upstream DNA sequences in contact with RNA polymerase bound at ﬁs P , such interactions must be largely nonspeciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>It has been suggested that IHF sites in the att region for the related integrase from phage 480 are in the phage sequences upstream of atP ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7511583" target="_blank">7511583</a>]<br></li>
<li>Since it is likely that the IHF effect is mediated by sequences downstream of position 2170 , a computer search for IHF sites ( 30 ) was undertaken .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>Since DNA-bound IHF and Fis both bend their target sites sharply ( by 140 ° and 90 ° , respectively ) , the upstream sequences at pnir are likely to be folded ( 21 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>To measure the binding of NsrR to pnrf directly , we incubated an end-labeled DNA fragment that carries pnrf sequences from position 125 to position 20 with cell extracts from cells lacking NsrR and IHF that carried a plasmid encoding NsrR or the empty vector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>Since IHF has also been shown to cause a strong bend upon binding DNA and both regulators bind to unusually long sequences , the combined binding of OxyR and IHF may lead to an unusual promoter structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>IHF belongs to a family of bacterial histone-like DNA binding proteins that facilitate the assembly of ordered protein-DNA structures by bringing distant DNA sequences , or proteins bound at distant DNA sites , into close proximity to one another ( for a review , see Schmid , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>IHF is a likely candidate for the missing factor as , on binding DNA , it introduces a sharp bend and through this bend is able to bring together proteins that are bound to sequences separated by hundreds of basepairs ( Goosen and van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>In this context , the finding that IHF binds in the centre of a specific class of invertedly oriented repe-titive extragenic palindromic ( REP ) sequences is interesting ( Boccard and Prentki , 1993 ; Oppenheim et ai , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Recognition sequences for Fis , FNR , and IHF ( boxed ) were described previously by Jackson et al. ( 18 ) , Meng et al. ( 21 ) , and Green et al. ( 11 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513468" target="_blank">17513468</a>]<br></li>
<li>Competition has been reported between IHF bound at IHF I and both RNAP and FNR for their overlapping sites , and between IHF bound at IHF II and NarL/P for their overlapping sites ( Browning et al. , 2002 ; 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>IHF was recently shown to bind to two distinct sites in the switch region , one of which is located immediately downstream of ® mE ( Blom ® eld et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>This intergenic region contains sites for NarL , IHF and Fnr ( Spiro and Guest , 1990 ; Walker and DeMoss , 1991 ; Dong et al. , 1992 ; Rabin et al. , 1992 ; Schroder et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564515" target="_blank">10564515</a>]<br></li>
<li>This 31 bp region includes IHF III , the most proximal of three DNA sites for IHF ( Fig. 1A ) ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>From the structure of IHF bound to 35 bp of DNA , the bend has been measured to be greater than 180 ( Rice et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF recognizes specific sequences for binding and facilitates DNA bending with the bend angle of about 160u as analysed by crystallography ( Rice et al. , 1996 ) , thereby affecting transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>IHF bends DNA upon binding to the speciﬁc sequences ( Nash , 1996 ; Radman-Livaja et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>IHF is a heterodimer of J & rr .20,000 ( Nash & Robertson , 1981 ) with sequence homology to the histone-like protein HU , but unlike HU , binds to a specific DNA sequence ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Nevertheless , Eisenstein et al. ( 1987 ) and Dorman and Higgins ( 1987 ) proposed that IHF binds to sites both within , and adjacent to , the tim switch on the basis of DNA sequence analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Recent work has shown that IHF strongly bends DNA at sites where it binds in some phage ( Robertson and Nash 1988 ; Thompson and Landy 1988 ; Higgins et al. 1989 ) , transposon ( Prentki etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>IHF is an asymmetric protein that binds DNA at speciﬁc sites , causing a bent of 160 in the DNA molecule ( Rice et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>The integration host factor IHF , named for its role in phage integration , binds speciﬁc asymmetric DNA sites ( WATCAANNNNTTR ; W , A or T ; N , any base ; R , A or G ) as a heterodimer ( 31,32 ) , bending DNA by 1608 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17553830" target="_blank">17553830</a>]<br></li>
<li>Summary Integration host factor ( IHF ) of Escherichia coli is an asymmetric histone-like protein that binds and bends the DNA at specific sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Binding sites for integration host factor ( IHF ) , a histone-like protein that bends DNA ( 34 ) , are frequent features of a54-dependent promoters ( 23 ) and are usually located between the a154 and activator recognition sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>Previous EcoGene-based searches for intergenic repeats include the genomic mapping of repetitive extragenic palindromic ( REP ) sequences [ 12 ] , the identiﬁcation of a class of bacterial interspersed mosaic elements ( BIME-1s ) as repetitive integration host factor ( IHF ) - binding palindromes ( RIPs ) [ 31 ] , and a search for new direct and inverted-repeat DNA sequence classes [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10673004" target="_blank">10673004</a>]<br></li>
<li>A dramatic example of this occurs when integration host factor ( IHF ) binds speciﬁcally to its cognate DNA sequence .29 Upon complexation with IHF , an entire 35 bp duplex is bent by more than 1608 , which includes a short ( ,12 bp ) region bent by approximately 608 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15095980" target="_blank">15095980</a>]<br></li>
<li>The abbreviations used are : FHL , formate hydrogenlyase ; UAS , upstream activatory sequence ; IHF , integration host factor ; bp , base pair ( s ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8034728" target="_blank">8034728</a>]<br></li>
<li>T-A T-G T-C ` ¬ WT -6 - 71 + I ¬ IHF T-C T-A T+G WT IHF I insertions CGGGC + 5 bp TGGATCCGTC + lObp CGGGATCGATCCGTC +15 bp Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>IHF bind ~ '' to the E. coli and S. typhimurium carAB operons Band-shift experiments performed with purified IHF ( a gift from N.P. Higgins , Department of Biochemistry , University of Alabama at Birmingham ) clearly demonstrate that the purified protein binds to the 266 bp Sau3AI fragment and that DNA protein complexes formed with purified IHF migrate at the same level as complexes formed upon incubation with S-30 extracts ( Fig. 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>A single IHF molecule proiects a DNA region of about 40 bp against DNase i digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>IHF binds to speciﬁc DNA sequences of 30 to 35 bp ( 21 ) and directly or indirectly regulates the transcription of more than 100 genes ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18519731" target="_blank">18519731</a>]<br></li>
<li>The results of the alkylation protection experiments , depicted in Fig. 2 and summarized in Fig. 3 , showthat the IHF protein interacts specifically with the nucleotides ( 5 '' GAT-CAAGCCTTAA-3 ' ) locatedbetween basepair positions +133 and +145 in the iluGMEDA leader region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>IHF binds dmsA immediately upstream of ModE The vast majority of repressors bind their cognate promo-ters in a region that spans from position ¹ 70 to þ 20 relative to the start of transcription ( Gralla and Collado-Vides , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The area of protection from DNase I by 0.5 M IHF is indicated with a vertical line , and numbers indicate the distances relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>It is well known that bound IHF induces a sharp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>In previous work ( 3 -- 6 ) , we showed that the DNA target for FNR at pnrf is centered at position 41.5 ( i.e. , between 41 and 42 bp upstream from the transcript start point ) , that anaerobically induced FNR-dependent activation of pnrf is downregulated by the binding of the nucleoid-associated protein IHF to a target ( IHF I ) centered at position 54 , and that this repression is relieved by the binding of either NarL or NarP to a DNA target centered at position 74.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_79">
FNR and NarL/NarP binding sites are represented by inverted arrows , while IHF sites are depicted by boxes , with the positions of binding sites shown in parentheses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></label>
<input id="sit_similarity_79", type="checkbox">
<div class="SIT_similar">
<li>FNR and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>FNR , CRP , and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>FNR and NarL/NarP binding sites are represented by inverted arrows , and the IHF I binding site is depicted by a box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The locations of FNR and NarL / NarP binding sites are represented by inverted arrows , and the IHF binding site is shown by a box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li></div>
<li>IHF has been & own LO bind DNA specifically through contacts with t , he mino groove ( Craig & Nash , 1984 ; Yang & Nash , 1989 ) and its recognition sequence is A + T-rich .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Results of SELDI ProteinChip experiments suggested that IHF might bind to the gltBDF upstream sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>This sequence does not bind IHF in a gel mobility shift assay ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>The 10 bp AST2 and AST3 deletions in the S. typhimurium IHF binding site are indicate b S. typhimurium Hincll Hpall ( -33 ) ( -22 ) Haelll Hincll Nrul L L P1 P2 / - 1 1 4 b p - ~ -401 TCGCGAT [ GTAATrCTTARACATTGTCTTAAT [ qGCTLGI I I I IGTIGATTfTATGATGCA [ TrGATTTGTFITT TATATTTITAATATAFIGATI-fflAAAATT1-1qTTGTCGT - ~ AAC I I I -282 A ST3 A ST2 Lane 1 , without added protein ; lanes 2 and 3 , with 10 lag S-30 extract of P4X ) ~ - cells grown on minimal medium and minimal medium supplemented with uracil , respectively teraction of the extracts with fragments spanning the argG and pyrBI control regions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>There is a potential Lrp site ( YAGHAWATTWTDCTR ; Y ¯ T or C ; H ¯ A , C or T ; W ¯ A or T ; D ¯ G , A or T ; R ¯ A or G ; Cui et al. , 1995 ) centred at ® 120 bp just inside the yeiK coding region , and there are two sequences matching the IHF-site consensus ( WATCAANNNNTTR ; Freundlich et al. , 1992 ) , which are consistent with the observed effects of Lrp and IHF on yeiL gene expression ( Fig. 7c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>IHF and CRP Behave as Transcriptional Activators of the Pa and Pz Promoters -- Analysis of the paaZ-paaA intergenic region revealed the existence of two sequences , TATCAACTACTAA and GAGTGCGAAATCTGTCACAGTT , from positions 2125 to 2113 and 273 to 252 relative to the major transcription start site of paaA ( Fig. 4 ) , that significantly match the consensus sequences WATCAANNNNTTR ( where W is A or T , R is A or G , and N is any of the four bases ) and AANTGT ¬ FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>In the natural context of pR two relaxed IHF consensus sites are present : TATTTANNNNTTG at 64 and AATGCCNNNNTTG at +107 n relative to the +1 of pR transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>In general the IHF sites cataloged in Fig. 3 appear to occur in poor promoters , as evaluated by the deviation of promoter sequences from consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Also , regions with similarities to the Lrp ( 44 ) and IHF ( 15 ) consensus sites can be found ( Fig. 7 ) , and the location of all these sequences is consistent with the results of a recent deletion analysis of the osmY control region performed by Villarejo and coworkers ( 43a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The ihf consensus match ( 2126 to 2114 ) is asymmetrically positioned relative to the protected region , as has been shown for other ihf-binding sites ( 10 , 21 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>The dotted region below the IHF-protected nurG DNAsequences indicates a conserved 13-bp region located upstream of narG compared with the IHF consensus sites of Filutowicz and Inman ( 31 ) and Craig and Nash ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>The transcriptional start site ( indicated by xxx in the upper part of the figure and by the heavy arrow in the lower part ) and the most likely - 10 region are indicated , as well as regions with homology to the Lrp , CRP , and IHF consensus sites and a motif with the characteristics of curved DNA and palindromic symmetry ( thin arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>IHF bound to the phage ; II1A site Downloaded from www.microbiologyresearch.org by IP : 132.248.34.24 On : Fri , 05 Aug 2016 22:42:4 Lrp-binding analysis Role of Lrp in gcvT : : lacZexpression in the IHF-C , IHF-D and IHF-G mutants We used a gel mobility-shift assay to determine whether Lrp could still bind to the gcv control region carrying the Since all but the upstream 22 bp of the Lrp-binding insertion sequences , and this binding could be responregion was replaced in the IHF lysogens , we predicted sible for the negative effect of Lrp on g c v T : : l a c Z that Lrp would no longer be involved in regulation of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>Gel-retardation assay Plasmid pGP133 was digested with HindIl , providing a 180 bp DNA fragment of the regulatory region of phage Mu with the ihf binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The 575 bp BamHI fragment cents ncompassing the control region was 5 ' - end-labelled on both strands , aliquots were digested with Sau3Aland Mbol , the reaction products separated by electrophoresis on a 6 % polyacrylamide gel and revealed by autoradiography , B. in the absence of IHF binding , pJST3ST5 plasmid DNA carrying a 10 bp deletion in the IHF target site was extracted from a culture of MC4100 transformed with this plasmid and grown in the presence of arginine , uracil and cytidine , pJST5 plasmid DNA was extracted from a culture of MC4100 himA and of MC4100 ( as int cents ~ aal control ) transformed with this plasmid and gro ~ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>To verify this last hypothesis directly , we used oligo-nucleotide-directed mutagenesis and subcloning to construct pJST5 ( see Materials and methods ) , a derivative of pJST in which the arginine-specific P2 promoter has been inactivated by a 3 bp substitution mutation in its - 3 5 region , and the double mutant pJST3ST5 combining the inactivated P2 promoter and the 10 bp AST3 IHF binding site deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>( A ) Binding and electrophoresis were as described in the text ; ( B ) relative mobilities of retarded fragments plotted as described in the legend to Fig. 2B This model for activation by the concerted binding of IHF and NARL-P to the narG promoter is similar to that proposed for the catabolite activator protein ( CAP ) - dependent activation of the maltose operons by MALT ( 12 ) in which speciﬁc CAP-induced bending is thought to promote a critical interaction between MALT proteins bound to sequences on either side of the bending site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>To investigate whether the IHF III site from pnrf could regulate the nir promoter , the nrf upstream sequences ( positions 209 to 60 ) were fused to the nir core promoter sequences ( positions 60 to 36 ) to generate a pnrf-nir fusion promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Thus , we can predict that IHF binding at pnir will induce a bend centred at position 288 that will orient upstream sequences away from the promoter ( Fig. 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>On the other hand , when we introduce CRP - induced DNA bends at H159L different precise location between upstream enhancer sequences and the core promoter of glnAp2 promoter ( e.g. CRP binding site centered at 50.5 , 55.5 , 61.5 , ( a ) Supercoiled CC-55 .5 Supercoiled IC-55 Linear CC-55 .5 Linear IC-55 CRPH159L − + − − − + − − IHF − − − + − − − + Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>and CarPmediated protection of DNA modification in the carAB control region Although significant differences are observed between the respective E coli and S typhimurium carAB sequences upstream of the P l promoter , both control regions contain an IHF binding site located at exactly the same distance upstream of the P l startpoint of transcription [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Because IHF is known to bend DNA by as much as 140 ° ( 49 ) , it can sometimes function as an architectural element to bring speciﬁc DNA sequences upstream of the ihf site in close proximity to RNA polymerase at the target promoter ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>In the first model , the bending induced by IHF brings sequences located further upstream from the ihf site into contact with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , July 1996 , p. 3971 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Structure Modiﬁcation Induced in the narG Promoter by Binding of Integration Host Factor and NARL-P Received 18 December 1995/Accepted 16 April 1996 Interaction of integration host factor ( IHF ) with linear DNA fragments containing the narG promoter region induced an apparent sharp bend in the DNA centered at the IHF-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>The untranslated DNA segment between the tdcR and tdcA reading frames ( see Fig. 5 ) contains sequences implicated in tdc gene expression : the CAP site for transcription initiation by the cAMP-CAP complex which is inhibited by the E. coli integration host factor ( IHF ) , which binds to the DNA at the IHF site ( R.V. Patil and P. Datta , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2573820" target="_blank">2573820</a>]<br></li>
<li>For example , when multiple CAP-cyclic AMP dimers are bound upstream of a promoter , the CTD can reach to at least position 100 ( 7 , 33 ) , and at the Pu promoter in Pseudomonas putida , integration host factor ( IHF ) bends the DNA so that CTD can interact with speciﬁc DNA sequences located upstream of the IHF site ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11591675" target="_blank">11591675</a>]<br></li>
<li>Thus , since the biochemical data for the interaction of IHF with DNA sequences in the iZVP , 1 promoter region ( Winkelman & Hatfield , 1990 ) are in excellent agreement wit ) h the data of Yang & Nash ( 1989 ) for the interaction of THF with DNA sequences at the lambda attP sites , we assume that the IHF-induced bend in t ' he iZvPG1 promoter region may also be symmetrical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_111">
We have shown that IHF interacts with the bases and the sugar-phosphate backbone of the DNA helix at a near consensus target binding site in the iZvPo1 promoter region in a manner very similar to the way IHF interacts with its target sites in the lambda attP region ( Winkelman & Hatfield , 1990 ; Yang & Nash , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></label>
<input id="sit_similarity_111", type="checkbox">
<div class="SIT_similar">
<li>We have shown that IHF interacts with the bases and the sugar-phosphate backbone of the DNA helix at a near consensus target binding site in the iZvPo1 promoter region in a manner very similar to the way IHF interacts with its target sites in the lambda attP region ( Winkelman & Hatfield , 1990 : Yang & Nash , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li></div>
<li>In previous work , we found that NarL or NarP activates the nir promoter by binding to a site centred at position -69.5 and counteracting the repressive effects of IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The repressive effects of IHF and Fis at pnir are relieved by NarL or NarP binding to two target heptamers , organized as an inverted site centred at position 269.5 ( Tyson et al. , 1994 ; Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The interaction of s54-RNAP and the activator bound to UAS ( s ) is facilitated by DNA-looping , either by local DNA topology produced by curved DNA and supercoiling , or by IHF bound to a site located between the promoter and the activator binding site ( s ) ( Kustu et al. , 1989 ; Hoover et al. , 1990 ; Collado-Vides et al. , 1991 ; Charlton et al. , 1993 ; Molina-Lopez et al. , 1994 ; Goosen & Putte , 1995 ; Carmona & Magasanik , 1996 ; Cases et al. , 1996 ) as is the case for psp ( Dworkin et al. , 1997 , 1998 ; Weiner et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>In the case of ~ 4-monitored promoters it has been demonstrated that binding of IHF to an upstream site induces DNA bending , thereby allowing a bound activator even further upstream to contact RNA polymerase ( Hoover et al. 1990 ; Claverie-Martin and Magasanik 1991 ; Gober and Shapiro 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>By assuming that binding of IHF to a non-speci ® c site does not induce the strong bend in DNA observed for speci ® c sites ( Rice et al. , 1996 ) we estimate that IHF covers the same amount of DNA as does HU ( Bonnefoy et al. , 1994 ) , i.e. about 10 bp per dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The extended binding implíes that IHF bound to site I líes within 18 bp 01 IRL , only 10-14bp away Irom FimB and FimE bound to the external half-síte of IRL ( GalIy et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>FNR -41.5 IHF bound to the IHF II site activates FNR-dependent transcription Previously , we demonstrated that IHF binding to the IHF I site represses FNR-dependent transcription at pnir ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>In this model , the sharp bend ( U-turn ) introduced by IHF ( Rice et al. , 1996 ) on binding to the site centred at ± 114 precludes binding of RNAP at ® s P3 and div promoters with resultant ` chanelling ' of transcription towards ® s P2 and P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11847119" target="_blank">11847119</a>]<br></li>
<li>The open box indicates the position of the IHF binding site , while the filled triangle indicates the position of the 2 A-tract sequences centered around base-pair - 50 The electrophoretic mobilities of radiolabeled , circularly permuted DNA fragments ( diagrammed in Fig. 6 ) in a 5 % non-denaturing polyacrylamide gel are labeled `` free DNA '' and shown on the autoradiogram in Figure 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>For DNA bending experiments , the following three DNA fragments were cloned into the XbaI ( blunt ) - SalI cloning sites of plasmid pBend2 ( 14 ) ( a gift from S. Adhya ) ( Fig. 1 ) : ( i ) a 202-bp EcoRI ( blunt ) - SalI fragment from pYW65 ( nt 1722 to 1890 plus flanking vector sequences ) containing both IHF - and CAP-binding sites , yielding pYW100 ; ( ii ) a 123-bp AflII ( blunt ) - SalI fragment from pYW66 ( nt 1722 to 1829 plus flanking vector sequence at the SalI end ) containing an IHF-binding site but only one-half of the putative CAP-binding site , yielding pYW88 ; and ( iii ) a purified 126-bp PCR-amplified DNA fragment ( nt 1765 to 1890 ) containing the putative CAP-binding site and one-half of the IHF-binding site , yielding pYW99 ( the last fragment was generated by using primers F75 and R65 [ see below ] ) and pYW75 template DNA and then a filling-in reaction with DNA polymerase and digestion with Sall ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>To analyze CAP - and/or IHF-induced DNA bending , three plasmids , pYW88 ( A CAP , with an IHF-binding site ) , pYW99 ( A IHF , with a CAP-binding site ) , and pYW100 ( WT , with both IHF - and CAP-binding sites ) ( Fig. 1 ) were digested separately with individual restriction enzymes ( BglII , Nhel , XhoI , EcoRV , PvuII , StuI , and NruI ) to generate DNA fragments which had identical sizes but the protein-binding sites that were present in circularly permuted positions along the length of the DNA ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>To determine if the effect of IHF on ﬁs expression could be mediated by IHF binding to the higher-afﬁnity site , we generated four point mutations at positions known to be highly conserved among ihf-binding sites ( 21 ) , leaving the remaining DNA sequences in the region from 2373 to 1107 unaltered in pTP185 ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>To replace the IHF site with the LEF-1 site and to reorient and reposition the IHF site , 27-bp double-stranded SacI-Xho I DNA fragments containing the appropriate sequences were synthesized and ligated into these sites in the promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>This site overlaps the DNA sequence that encodes the iZvP , l promoter site and it has been demonstrated , both in vivo and in vitro , that the interaction of IHF with this target site represses transcription from the iZvP , l promoter ( Pereira et al. , 1988 ; Page1 & Hatfield , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>'' Quantitative DNase I footprinting was employed to determine the equilibrium dissociation constant of IHF to its target site on a linearized ( relaxed ) and a negatively supercoiled DNA plasmid ( pDHDwt ; Table I ) containing the ilvPG promoter DNA sequence from bp positions 2248 to 16 ( Figs. 1 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>DNase I footprinting was performed on PM DNA labelled at the nanC end corresponding to wild-type ﬁm sequence or mutated in ONC2 or the IHF binding site ( ibs ) as indicated across the tops of the lanes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>pX52 has an intact IHF site with a complete wild-type DNA sequence on both sides of this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>This suggests that the DNA bending activity of IHF may be required to bring the atypically distant OxyR site closer to RNA polymerase bound at the -10 and -35 promoter elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>This interaction is facilitated by DNA-looping promoted by IHF bound to a site located between the psp promoter and enhancer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>IHF bound to site 11 is likely to bend the DNA through approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>b b 165 ¬ ¬ 125 - +115 IHF with the minor groove of the DNA helix at the central WATCAA region of the target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>FIS is a sequencespecific DNA binding protein that , like IHF , induces a severe bend in the DNA helix at its target binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>IHF also induces a severe bend in the DNA at its target binding site in the upstream promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>In this report we show that IHF also induces a severe bend in the DNA helix at its target binding site in the iZvPo1 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>IHF ( dotled circ1es ) binding lo site 11 , and Lrp ( hatched circ1es ) binding wlthin the fim switch , help bend the DNA to align IRL and IRR in the correel orientation lor synapsis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>At pnrfA , FIS represses transcription by binding to a site that overlaps the -10 element , whereas IHF binds to a site that overlaps the DNA site for the activator FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Purified OxyR or IHF was incubated with the binding site DNA for 10 min , then the IHF or OxyR was added and assayed for protection from DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>The limited sequence conservation observed in the region designated the P3 10 promoter region may reﬂect the position of an ihf binding site previously identiﬁed in E. coli and shown to be required for transcription stimulation of ﬁs P ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679232" target="_blank">14679232</a>]<br></li>
<li>The +1 , -10 and -35 elements of the suf promoter are underlined , the start codon for sufA is shown in bold capital letters , the putative Fur site is indicated with asterisks under the sequence , and the putative IHF site is indicated with double asterisks under the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>The up to 2-fold variations in 13-galactosidase expression observed in Tables 2 and 3 might be due to sequence alterations in the extended promoter region and/or a steric inhibition of RNA polymerase binding due to the positioning of the IHF binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>The σ54 promoter consensus sequence and the putative IHF site in the intergenic region are most likely part of the yhcK promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10525738" target="_blank">10525738</a>]<br></li>
<li>The IHF 111 site was previously predicted after analysis of the sequence of the ndh promoter region and IHF-dependent retardation was observed ( Green & Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Bases conserved in all four genera are shown in bold , and the shaded regions indicate the proximal Lrp binding site , the poly ( A ) region upstream of the IHF binding site , the consensus IHF binding site ( 295 to 283 ) , and the 235 and 210 hexamers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Although the protected region is 12 bp smaller than that commonly seen at other IHF-binding sites , the IHF consensus sequence is located asymmetrically within the protected region ( Fig. 6A ; Goo-sen and van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>This site contains a sequence ( Fig. 4 ) identical to the 13-base-pair ( bp ) IHF DNA-binding consensus sequence found in known IHF-binding sites ( 3 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>This site differs from th consensus binding sequence of IHF by one base pair only ( lower case in Fig. 4 ) and , in contrast to other IHF binding sites , it is located in the transcribed region , between the transcription initiation site and the ribosomebinding site ( SD ) ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>RESULTS [ HF enhancespsp expression The consensus binding sequence for IHF is 5 ' - A/TAT-CAAN 4TTA/G ( 47,49 ) , and two such consensus sites are present in the psp promoter region between positions -60 and -33 with respect to the transcription start site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>Upstream of the start site of the yiaK-S operon the 210 and 235 consensus , high AT content region , IHF site , and Crp-binding sites are located by the corresponding boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
<li>To facilitate the reversal of the IHF binding site and replacement of factor ( LEF ) -1 the IHF binding site by the lymphoid enhancer-binding binding site , the 5 ' - TTTATT-3 ' sequence between base pairs -108 and -101 and the 5 ' - TTGAAA-3 ' sequence between base pairs -81 and -74 of the wild-type ilv promoter region ( 4 ) were changed to Sac I and Xho I endonuclease restriction sites by site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>To test this idea , the asymmetrical IHF binding site , 5 ' - AAACAACAATTTA-3 ' , encoded in the DNA sequence between the Sac I andXho I sites shown in Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>Boxed sequences represent the binding sites of IHF ( site-I ) and MlrA while the OmpR-binding sequence is shown by a bar below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Though the sequence of the IHF binding site is not conserved among the four genera , the cytosine that is present in all known natural IHF binding sites ( 16 ) is conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Asterisks mark the AT-rich sequence motifs , IHF indicates the putative binding site for IHF , I and II mark binding sites for FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1547773" target="_blank">1547773</a>]<br></li>
<li>These sites include the matches with the IHF binding site consensus sequence identiﬁed by sequence analysis ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>To assess whether the two sequence elements resembling IHF binding sites are important for s54-promoter activity , we mutated the putative IHF-site 1 in the E. coli glmY ' - lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Symbols : squares , no IHF ( fragment containing no IHF-binding sites ) ; closed circles , pR-wt ( fragment containing the pR promoter ) ; open circles , pR-IHF ( fragment containing pR promoter with SeqA-binding site replaced with the ideal IHF-binding sequence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>In addition to the consensus sS promoter , there is a sequence resembling an IHF binding site ( 29 ) situated almost exactly between the promoters of uspA and uspB ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>In this context , it is worthwhile to note that a nucleotide sequence homologous to the IHF binding site consensus ( Craig & Nash , 1984 ) is present between the - 35 and - 10 regions of p4 ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Consistent with this , we found a sequence homo-logous to the IHF binding site consensus ( Craig & Nash ; 1984 ) between the -35 and - 10 regions of t ' he p4 promoter ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>We chose the core of the well-characterized HI-site within the att region of bacteriophage lambda as a representative IHF binding sequence ( Yang and Nash , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Recerrt mutational analysis of the ihf site of pL1 confirms our hypothesis , as mutations that affect IHF binding map within our proposed sequence ( A. B. Oppenheim , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Extensive genetic analysis of the bacteriophage A H ' site for IHF binding has revealed the importance of nucleotides that lie both within and outside of the core consensus ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Due to the presence of a sequence near Pl that resembles the IHF binding site , we measured expression of galactokinase from several of our plasmid constructions in strains with and without the himA mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845101" target="_blank">2845101</a>]<br></li>
<li>Although our studies indicate that the predicted upstream site is not an FNR binding site , we found that this sequence binds IHF and increases expression of fnr under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>However , closer inspection of the sequence showed that ligation to the filled-in Hind111 site upstream has recreated a comparable IHF binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845101" target="_blank">2845101</a>]<br></li>
<li>Promoters derived from the A2 deletion possess the same sequence between -40 and -49 but they differ widely in the sequence further upstream ( due to different length spacers , insertion of a CRP or IHF binding site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>narHnarJ narl - ] chIZ locus .9 ERIC sequence REP sequence o Nar L binding site IHF binding site E ~ ~ '' ~ Fnr binding site Transcription initiation site If terDtermination replication site ~ ) rho-independent transcription termination site Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>A further conserved sequence motif exhibiting similarity to an IHF binding site ( Swinger and Rice , 2004 ) is present 5 bp downstream of ABS2 ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843219" target="_blank">19843219</a>]<br></li>
<li>The IHF binding site , 5 ' - AAACAACAATTTA-3 ' , in upstream activating sequence ( UAS ) 1 is located between base pairs -82 and -96 ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>The NarL-2 binding site partially overlaps the recognition sequence for IHF , while sites NarL-3a and NarL-3b are located between promoter 7 and the IHF-binding site ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>The NarL-2 site is located between the FNR-binding site of P6 and partially overlaps the IHF recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>The Crp binding is centered at -65.5 , in the predicted binding sequence ( -76 to -55 ) , and between the -35 hexamer and the IHF binding site ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>( C ) The nucleotide sequence of the noncoding strand around the IHF binding site in the pfl regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The approximate distance between the proximal Lrp binding site and the IHF site is conserved among the different genera , but the sequence of the intervening region is not highly conserved except for the poly ( A ) region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The BIME-1 L linker is well conserved and frequently carries an IHF binding site ( 6 ) while those of BIME-2 and atypical BIMEs vary both in length and sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22199259" target="_blank">22199259</a>]<br></li>
<li>The IHF binding site thus obtained possesses an affinity for IHF very close to the affinity of HI in its natural sequence context ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>Both sequences were defined by their ability to match major parts of the consensus sequence previously determined for the site of binding of IHF ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300522" target="_blank">8300522</a>]<br></li>
<li>Overlapping the putative Fnr-binding site is a sequence ( Fig. 2 ; boxed ) that bears resemblance to the consensus IHF box ( W A T C A A N N N N T T R ; W = A or T , R = A or G , N = any base [ 24 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9315722" target="_blank">9315722</a>]<br></li>
<li>Mutagenesis of the 13-bp consensus sequence ( 23 ) in the IHF binding site very strongly reduced IHF binding in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>A 4-bp insertion between TyrR box 1 and the TrpR binding site reduces this IHF binding site to 40 % homology with the consensus sequence , as well as placing TrpR and TyrR on different faces of the helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961453" target="_blank">7961453</a>]<br></li>
<li>The IHF binding site in the nar operon control region , as defined by DNase I protection , shares substantial similarity with the core IHF binding site consensus sequence but weaker similarity with the extended consensus ( Fig. 1C ; ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>The fact that a consensus IHF binding sequence is close both to the cer-like site within argR and to the crossover region 501 GGCAAAGCAGAAGGTATTCTGGGCACCATCGCTGGCGATGACACCATCTTTACCACCCCTGCTAACGGTTTCACAGTCAAAGACCTGTACGAAGCGATTT G K A E G I L G T I A G D D T I F T T P A N G F T V K D L Y E A I L 01 701 801 901 Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>The exchange of a natural SeqA-binding site with the ideal IHF consensus sequence resulted in 50 % reduction of pR transcription in the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>The putative IHF binding site in the a d i A promoter is ATTGCATAACAAA ( from bp 70 to 82 ; Stim & Bennett , 1993 ) and has been changed to ATTGCATCCCAAA in pKPS7 ( the changes are underlined ) , which is less like the IHF consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>Within this dA-t-dT-rich region , a stretch of three dA residues located 9bp from the 5 ' site of the consensus of the ihf site in the regulatory region of phage Mu ( see also below ) was shown to be of paiiicuiar importance for IHF binding , as substitution of two of these dA residues for dC residues reduced the stability of the IHF-DNA complex at least 10-fold ( van Rijn e ( a / , , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>A DNA fragment from 2406 to 1246 but with the region from 248 to 2128 replaced with an equivalent length of the cat gene sequence did not bind IHF under identical binding conditions ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>In addition , the DNA sequences that IHF binds to ( consensus site YAANNNNTTGAW , where W is A or T , Y is T or C , and N is any nucleotide ) ( 15 , 23 , 60 ) show some similarity to the sequence within this upstream region of the fnr promoter ( 110TTAAGGGTTTTCAA 97 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Further 5 ' to this promoter several sequence features are noteworthy : 6-7 bp long AT stretches are repeated eight times in irregular intervals , a sequence resembling the consensus binding site for IHF ( Integration Host Factor ; Friedman , 1988 ) is found at position -81 and two sequences with one mismatch to the FIS consensus binding site are located just upstream of the -35 region and downstream of + 13 ( Figure 2 , I and II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1547773" target="_blank">1547773</a>]<br></li>
<li>The IHF-binding sequences upstream of the Pe and ilv PG2 promoters have the same orientation ( Fig. 4A ) , The spacing between the ihf site and the -- 35 region differs by 12bp , which is consistent with the observation that the Pe promoter is still activated by IHF upon extension of the spacer with one helical turn of the DNA , In contrast , the sequence corresponding to the ihf consensus upstream of the pLI promoter , as proposed by Giladi et ai ( 1990 ) , is oppositely orientated with respect to the - 3 5 region ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>In this context , it is remarkable that two sequences homologous to the IHF binding site consensus are found ( one on each DNA strand ) , overlapping the -35 region of the p3 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>The second potential IHF binding site is part of a previously described bent DNA sequence ( Bent 111 ) located from nucleotides 134 to 497 and covering almost the entire 5 ' region of orfl and 58 bp of the orfl coding region ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300522" target="_blank">8300522</a>]<br></li>
<li>B Lrp ArgR ( -- ) + Crp IHF ( + ) ( -- ) RNAP gFlitgBDurFepr2omoter region Our inspection of the gltBDF DNA sequence revealed a potential Crp-binding site between -76 and -55 ( Fig. 5A , top two lines ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Lrp and IHF bind and , according to the model , bend gltBDF promoter DNA such that an upstream regulatory sequence or Lrp surface required for stable binding of RNAP is brought in contact with RNAP that is loosely associated with the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Polyacrylamide gel electrophoresis experiments performed with circularly permuted DNA fragments containing the tandem promoter region reveal the presence of a sequence-directed DNA bend in the 20 base-pair activating region between the promoters , and an IHF-induced DNA bend at the IHF binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>No further significant enhancement of transcription initiation from the ilvPo2 promoter is observed until the DNA sequence including the IHF binding site in the iZvPo1 promoter region is present upstream from the iZvPo2 promoter ( strain IH1055 ; Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>For example , at physiological superhelical densities , strand separation in a 43-bp A1T-rich sequence upstream of and including the IHF binding site in the UAS1 region contained in a negatively supercoiled pBR322 based plasmid is observed .2 Perhaps IHF binding decreases the probability of duplex destabilization at its binding site in UAS1 and increases the probability of unwinding the DNA helix in the 210 region at the downstream ilvPG promoter site ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>The positions of the zones of direct contact with PepA ( PEPA1 and PEPA2 ) generate three main linker regions in the carP1 operator : L1 ( 31 bp ) between the 35 promoter element and PEPA1 , L2 ( 65 bp ) comprised between PEPA1 and PEPA2 , and L3 ( 96 bp ) that connects PEPA2 to the IHF binding site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>Interpreted as a simple bending model , these results suggest that IHF induces a sharp bend centered at the IHF-binding site which may promote an interaction between NARL-Ps bound at two speciﬁc sites located on either side of the putative bending center .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>The fragment devoid of a functional IHF1 site was still able to form two complexes with IHF , which suggests the presence of at least three binding sites for this protein in the wild type ssu regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10506196" target="_blank">10506196</a>]<br></li>
<li>Putative regulatory sites : open arrows , dyad symmetry ; arrows , direct repeats ; Dcm methylation sites ; putative Fis protein binding sites ; AT-rich region and IHF binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748952" target="_blank">7748952</a>]<br></li>
<li>To identify putative binding sites for the proteins involved in site-specific recombination , namely IntS , TorI , and IHF , we analyzed the attL and attR sequences , upstream of intS and downstream of torI up to the respective core sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>Between tdcR and tdcA are found putative Crp binding sites , and also a functioning site for integration host factor ( IHF ) 110 bp upstream of the transcription start site ( Wu and Datta 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Possible IHF binding sites were identiﬁed between 46 and 68 bases upstream from the start site of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>Two putative NRI binding sites and IHF binding site in ygjG upstream region proposed to be involved into transcription regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12617754" target="_blank">12617754</a>]<br></li>
<li>On the other hand , moving the NR,-binding sites 5 bp closer to the IHF-binding site , or 5 or 15 bp away from that site , results in the almost complete inhibition of open complex formation by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>The binding sites for the activator may be moved as much as 12 bp closer to the IHF site or as much as 10 bp away from that site , without interfering with the ability of IHF to stimulate transcription ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>The Hi & I-Hinfl fragment ( see Fig. 1 ) containing the t ' wo IHF binding sites was digested with RsaT to yield two fragments containing either the upstream IHF site ( Hid-RsaI , -240 to -33 ) or the downstream site ( RsaI-Hinfl , -33 to +250 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>In addition , two IHF binding sites are present downstream from the glpT transcriptional start site at positions +15 to +51 and +193 to +227 [ 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21314918" target="_blank">21314918</a>]<br></li>
<li>In the gltBDF operon , IHF binds the region from -75 to-113 , while the three Lrp binding sites are centered at nucleotides -152 , -215 and -246 relative to the transcription start site [ 5,14 ] ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>In vitro mobility shift and DNase I footprinting experiments identified a single IHF binding site in thepfl upstream regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>In support of this notion , the results of DNase I footprinting experiments revealed that the region of DNA from bp 117 to 89 relative to the fnr transcription start site was protected in the presence of 0.5 M IHF ( Fig. 7 ) and includes the sequence of DNA that was originally predicted to be the upstream FNR binding site centered at bp 103.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>In these experiments , the IHF protected sites of DNase I cleavage were visualized by primer extension analysis employing a 32P-end-labeled 23-bp single-stranded , DNA oligonucleotide that hybridizes to a region immediately upstream of the IHF binding site at ilv bp positions 2155 to 2132 ( Fig. 1 ; Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>In this work we show by gel retardation analysis that IHF binds to bent i/vBN DNA and greatly enhances the bend located within or near the IHF binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>Since extracts of an IHF-deficient strain do not show any detectable binding , these results prove that IHF is the active DNA-binding protein present in our cell extracts and that it binds to the putative IHF binding sites located between 293 and 321 bp upstream of the transcription initiation site of the carAB promoter P1 in E. eoli and S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>IHF binds to a site located between the glnHpd promoter and the NR,-binding sites and .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>In the case of the pst operon , the IHF-binding site is located between the 210 promoter and the ribosome-binding site , and IHF binds speciﬁcally , though weakly , to a fragment that carries this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Because the where the Lrp-binding region was replaced with the IHF - precise direction of the Lrp-induced bend relative t o binding site were isolated and ligated into the EcoRI site of RNAP bound a t the gcu promoter is unknown , and since phage Agt2 ( Panasenko et af .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>In addition to the weak-strong box combination downstream of the promoter , there is a second strong box just upstream of the − 35 sequence , a TrpR binding site downstream of the TyrR boxes and a putative IHF site that overlaps the two downstream TyrR boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207706" target="_blank">12207706</a>]<br></li>
<li>In each case , at least one weak but possible IHF binding site , which overlaps the strong TyrR box region , was found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_222">
Identiﬁcation of an UP element within the IHF binding site at the PL1-PL2 tandem promoter of bacteriophage l. J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></label>
<input id="sit_similarity_222", type="checkbox">
<div class="SIT_similar">
<li>Identiﬁcation of an UP element within the IHF binding site at the P 1-P 2 L L tandem promoter of bacteriophage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li></div>
<li>Further mutational analysis of the nar control region IHF binding site will be necessary to define the crucial nucleotides for IHF action in this particular site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Sequence analysis of the hip regulatory region revealed a putative IHF binding site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>A sequence analysis of the pst control region has revealed a putative binding site for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>TO investigate the promoter region , a primer was designed to alter bases in the proposed IHF binding site of the a d d promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>One binding site for IHF ( Sirko et aL , 1993 ) and four binding sites for ArcA have been identified ( Drapal and Sawers , 1995 ) in the promoter-regulatory region of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709852" target="_blank">8709852</a>]<br></li>
<li>It has also been demonstrated that the pst promoter region possesses a functional IHF binding site that is located between the pst promoter and the Shine-Dalgarno sequence , and that IHF stimulates pst transcription ( Spira and Yagil 1999 ) N. P. Taschner Æ B. Spira ( & ) Departamento de Microbiologia , Instituto de Ciências Biomédicas , Universidade de São Paulo , Av .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>The A PL promoter is activated by the binding of IHF at a site 80 bp upstream of the transcription initiation site ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>The binding of IHF to another target site in the ilvPG1 promoter region represses transcription from this promoter and causes a 4-fold stimulation of transcription initiation from the downstream ilvPG2 promoter I-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>This IHF binding site is located at position -108 to -144 relative to the initiationsite for narG transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>A second IHF binding site was located in the 5 ' region of orfi at positions 339 to 356 , slightly downstream of the transcription initiation site ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8300522" target="_blank">8300522</a>]<br></li>
<li>100 % A +5 WT +5 +5 +10 r -- l -- + HF - + I - + -- I - RNA polymerase B -5 p - 25 GAAAAATCAAGGAGl-fGCAAAACTGGCACGArrrmCAT IHF - binding site glnHp2 t ' I Insertions GATCT + 5bp AATCGAAlX + lObp Figure 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Using genetic and biochemical approaches , IHF was shown to bind to a site midway between ONC1 and ONC2 to facilitate strong cooperative binding of NagC to these elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>When a conﬁrmed binding site for IHF is present , activators may bind farther upstream and retain function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Bending of the origin of replication of Escherichia coli by binding of IHF at a specific site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7553853" target="_blank">7553853</a>]<br></li>
<li>NarL binding at its target site prevents IHF binding ( Fig. 4A , lanes 5 -- 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>or the TTG trinucleotide by GGC in the THF binding site upstream from glnHp2 ( 5 ' - AATCAAGGAGT-TG ) greatly reduced the prot ' ection of this site by IHF ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>Our results show strong binding of IHF to a site centered approximately 80 bp upstream from the first of the three ompC promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Here we report the effect of IHF binding to the upstream ( -305 ) target site on CarP and pyrimidinemediated protection of the GATC site located 200 bp downstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Furthermore we have shown that the length of the intervening sequences separating the far upstream IHF binding site from the pyrimidine specific target about 200 bp downstream is crucial for the action of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>The results in Fig. 5 show that IHF alone protected an extended region of 100 bp , encompassing IHF I , IHF II , and IHF III ( lane 5 ) , while NarL alone protected the NarL/NarP binding site centered at position 74.5 ( lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>The approximate extent of protection by IHF for each binding site is indicated by a vertical line .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The ModE DNaseI footprinting experiments indicate that this proposed IHF site would overlap the left half of the ModE binding site by 5 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>A typical speciWc IHF binding site is approximately 30 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>The minimal HU binding site is smaller compared to that of IHF ( '' 30 bp at a speciWc site ) and is about 9 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Examination of the suf promoter revealed a putative binding site for integration host factor ( IHF ) located at -139 to -151 ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>A deletion of the upstream region in this fusion ( uspB131-lacZ ) was created by using a PCR primer binding to the putative integration host factor ( IHF ) binding site ( Fig. 1A ; 59 ATATCTGCAGAGTGGTTAACCTTCTGG ) with a primer within the lacZ gene and amplifying the uspB promoter from a plasmid carrying the fusion ( AF631 [ 15 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>Because enhancer proteins are absolutely necessary for σ54-dependent transcription [ 25 ] we reanalyzed the sequence of this fragment and found some putative binding sites for E. coli activators : two general nitrogen regulatory protein ( NRI ) binding sites and binding site for integration host factor ( IHF ) ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12617754" target="_blank">12617754</a>]<br></li>
<li>These sequences encompassed putative integration host factor ( IHF ) binding sites and are located just downstream of the repressor region we identified in nmpC and downstream of the negative binding site of OmpR in ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Abbreviations for the putative sites : CAP/CRP , cAMP-CRP consensus sequence ( 4 ) ; IHF , integration host factor ( 7 ) ; NRI , nitrogen regulatory protein binding site ( 11 ) ; RpoN , s54 promoter region ( 17 ) ; RpoS , 210 region of the sS promoter region ( 24 ) ; 210 and 235 , Es70 promoter sequences ( 18 ) ; S.D. , Shine-Dalgarno sequence ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696780" target="_blank">9696780</a>]<br></li>
<li>Here it is demonstrated that NagC binding to a second site ( ONC2 ) , located 212 bp closer to ﬁmB , also controls ﬁmB expression , and that integration host factor ( IHF ) , which binds midway between ONC1 and ONC2 , facilitates NagC binding to its two operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>UAS1 contains a DNA-binding site , centered at bp position 292 , for the DNA bending protein , integration host factor ( IHF ) ( 8 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>UAS1 , centered 92 bp upstream from the transcriptional start site , contains a binding site for integration host factor ( IHF ) ( 17 , 20 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068661" target="_blank">9068661</a>]<br></li>
<li>Besides the putative IHF binding site , another sequence between Pl and the cluster of promoters P2 , P3 , P4 that may be a potential recognition site for a regulatory factor , is a sequence of dyad symmetry that extends from nucleotide 1190 to 1207 ( TCGTGAACAGAGTCACGA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2845101" target="_blank">2845101</a>]<br></li>
<li>We found that the region from 2166 to 281 relative to the transcriptional start site , which contains a match to the consensus sequence for the integration host factor ( IHF ) , was required for maximum transcription from ﬁs P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Also within the argR gene ( Figure 2 ) we have noticed a 13 bp sequence showing very strong similarity to the consensus integration host-factor ( IHF ) binding site ( Leong et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>Sequences resembling the most highly conserved regions of the consensus binding site , AATCAAN4TTA , for the integration host factor ( IHF ) are indicated by boxes connected by the dotted lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9738023" target="_blank">9738023</a>]<br></li>
<li>Sequences resembling the IHF ( integration host factor ) consensus binding site are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Based on the presence of a consensus binding sequence , this region seemed likely to be a binding site for integration host factor ( IHF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Also present in the 5 ' noncoding region is a sequence nearly identical to the integration host factor ( IHF ) consensus binding site 5 ' - ( AfT ) ATC AAN4YF ( A/G ) -3 ' ( 11 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8021189" target="_blank">8021189</a>]<br></li>
<li>Abbreviations : CAP , catabolite gene activator protein binding site ; IHF , integration host factor binding site ; = = , AGPyATAAA consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2573820" target="_blank">2573820</a>]<br></li>
<li>By using the MacTargsearch program and the supplied consensus binding site sequence for integration host factor ( IHF ) ( 17 , 28 ) , a site with an overall homology of 48 % on the reverse strand from bases 172 to 219 was identified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961453" target="_blank">7961453</a>]<br></li>
<li>A DNA sequence resembling the integration host factor ( IHF ) binding site centered approximately at 2114 showed DNase I cleavage protection by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>We identiﬁed a putative integration host factor ( IHF ) binding site in the intervening sequence , and in vitro studies con-ﬁrmed that IHF bound this site with high afﬁnity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>We introduced deletions and insertions , and combinations thereof , in four linkers connecting the binding sites for integration host factor ( IHF ) , PepA , PurR , and RNA polymerase and studied the importance of phasing and spacing of the targets and the importance of the nucleotide sequence of the linkers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>In addition , an integration host factor ( IHF ) binding site has been identi ® ed at the ÿ66 position ( Figure 4 ( a ) ) .76 There is a high degree of phylogenetic conservation of nucleotide sequences in transcription factor binding sites .6 An understanding of the interaction of OmpR with the micF regulatory DNA sequences offers a particular challenge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Interestingly , two sequences resembling integration host factor ( IHF ) - binding sites ( Freundlich et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057842" target="_blank">8057842</a>]<br></li>
<li>This homology involves eight out of ten nucleotides , all the positions in close contact with the IHF ( Craig & Nash , 1984 ) being conserved , with the exception of the T at position 10 , which is replaced by a C. Gardner & Nash ( 1986 ) have introduced two base-pair alterations in the IHF binding sites of A at @ DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Equilibrium dissociation constants of speci ® c and non-speci ® c IHF binding sites Kd EMSAa K app app d UV footprint yjbE BIME-1 gyrB 0.28 0.57 2.25 1.9 0.18 nM 0.08 nM 0.75 nM 0.4 mM 11.1 18.8 56.5 0.9 nM 2.5 nM 9.6 nM ± IS1 E. coli DNA Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>CRP activates acs transcription by binding tandem sites located upstream of the major promoter , acsP2 , whereas , both IHF and Fis each act at different times during growth as anti-activators of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Previously , we reported that cAMP receptor protein ( CRP ) activates acs transcription by binding tandem DNA sites located upstream of the major acsP2 promoter and that the nucleoid protein IHF ( integration host factor ) binds three speciﬁc sites located just upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The role of IHF has remained less clear : while in vitro it is known to bind simultaneously with CRP and to cause inhibition of acsP2 transcription , in vivo the upstream sequence containing the three IHF sites has a positive effect on transcription ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>At pnirB , in contrast , FIS and IHF inhibit transcription by binding to upstream sites ( Browning et al. , 2000 ; 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>The heterobivalent integrase works as a dimer and is believed to bridge the arm and core-binding sites with the help of IHF or Cox , which bind cooperatively with the integrase ( Frumerie et al. , 2005a ; Frumerie et al. , 2005b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20627350" target="_blank">20627350</a>]<br></li>
<li>The initial band probably represents IHF binding : to a single cite while the slowest moving band probably comprises DNA-pro-tein complexes in which IHF is bound to two separate sites on the DNA ( Prentki et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Thus far , three proteins have been identified with specific functions and binding sites in oriC ( i ) DnaA , the initiator protein , which binds four 9-mers and then interacts with a contiguous set of three 13-mersto open the duplex ; ( ii ) the IciA protein which binds these 13-mers to prevent DnaA protein from opening this region and thus inhibits initiation ; and ( iii ) IHF , which binds a sequence that facilitates the bending of oriC around an aggregate of DnaA proteins ; HU protein serves the same function as IHF buw t ithout an identified binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449900" target="_blank">8449900</a>]<br></li>
<li>In this work , we have analyzed the interaction of IHF with three speci ® c binding sites : the ihf site found in the promoter region of the Escherichia coli yjbE open reading frame ( ORF ) , the site within the BIME-1 element located at the 30 end of the gyrB gene ( Boccard & Prentki , 1993 ; Oppenheim et al. , 1993 ) , and a derivative of the ihf site located at the right extremity of insertion sequence IS1 ( Gamas et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Two binding sites for LRP have been characterized within the invertible element ( Gally et al. , 1994 ) , one potential binding site for IHF occurs within the element and another lies immediately outside ( Dorman and Higgins , ~ 8 1 a ) nd purified IHF binds in vitro.to these sites ( S. L. Dove and C. J. Dorman , unpublished ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
<li>Furthermore , for activation of transcription to occur , the sequences which define the IHF and NarL binding sites must be correctly positioned relative to each other and to the downstream region where RNA polymerase binds ( Dong et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>These include an IHF binding region extending from - 1 0 6 to - 1 4 4 ( Rabin et al. 1992 ; Schr6der et al. 1993 ) and two NarL binding sites located on either side of the IHF binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>RESULTS AND DISCUSSION IHF binds specifically to the proU P2 promoter Analysis of the proU promoter region via Virtual Footprint ( Hales et al. , 1994 ) predicted two binding sites for IHF to the promoter region at positions − 25 / − 11 and +134 / +149 relative to the +1 site of the P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>The concentrations of Fis protein in the reactions were as follows : lanes 1 and 5 , no protein ; lanes 2 and 6 , 0.11 M ; lanes 3 and 7 , 0.22 M ; and lanes 4 and 8 , 0.44 M. Note that pnrf53 EcoRI-MfeI fragments carry pnrf sequences from positions 209 to 50 and , therefore , only carry the IHF II , IHF III , and Fis II binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_285">
The p99G and p146A substitutions disrupt , respectively , IHF and Fis binding to upstream pnir sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></label>
<input id="sit_similarity_285", type="checkbox">
<div class="SIT_similar">
<li>The p99G and p146A substitutions disrupt , respectively , binding of IHF and Fis to upstream sequences at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li></div>
<li>The sequences protected from DNase I digestion by OxyR and IHF binding are indicated by the brackets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Putative IHF and CRP binding sequences are shown by brackets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Based on the DNase I protected regions and the distribution of the ihf consensus sequence in a crystal structure of an IHF-DNA complex ( 45 ) , we estimated the centers of the higher - and lower-afﬁnity ihf-binding sites to reside at 2114 and at 264 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Sequences for a p54-promoter and for binding sites of the response regulator GlrR as well as of IHF are shared features of the glmY and glmZ promoter regions of many , but not all Enterobacteriaceae We performed sequence alignments of the promoter regions of the glmY as well as glmZ genes retrieved from 39 genome sequences representing the most important genera of Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Using the program fuzzpro ( EMBOSS programs ) and the consensus sequence WWWTCAA [ N4 ] TTR for IHF binding ( 45 ) and sequences GGCACGCAAATTGT for 54 binding and TAGTGTAATTCGCTAACT for PspF binding ( based on the 54 and PspF binding sites in the pspA promoter ) ( 20 , 46 ) we found potential binding sites for 54 , IHF , and PspF ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Consensus sequence for IHF binding is WWWTCAA [ N4 ] TTR ( 45 ) and the sequences for 54 ( GGCACGCAAAT-TGT ) and PspF ( TAGTGTAATTCGCTAACT ) binding are present in the pspA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Upstream of the gltBDF promoter , within the region replaced by the cat fragment in strain LP10048 , there are two regions resembling the consensus IHF binding sequence WAT CAANNNNTTR , where W is A or T and R is A or G ( 10 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Although sequences homologous to the IHF binding consensus sequence are also found at the level of the p3 promoter , the organization of this region appears different from that of the p4 operator , with two putative IHF binding sites in opposite orient & ions , forming a pseudo-palin-dromic structure on DNA ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>indication that IHF may affect aceBAK expression came from the identiﬁcation of two regions upstream of the promoter which matched the consensus sequence for IHF binding sites ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>( A ) Potential IHF binding sites near the aceBAK promoter are compared with the consensus sequence for IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>A single unique protection sequence was identified for OmpR and RstA , while two sites were identified with IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Within the 200 bp sequence upstream from the csgD promoter , IHF binds at least at two sites ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Based on both in vivo and in vitro analyses , we identiﬁed MlrA as a positive factor of the csgD promoter by directly binding to its upstream region ( 113 to 146 ) with a palindromic sequence of AAAATTGTA CA ( 12N ) TGTACAATTTT between the binding sites of two activators , IHF and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Two additional regions ﬂanking ABS3 exhibit a higher degree of conservation and show similarity to binding sites of IHF , which are represented by the consensus WATCARXXXXTTR ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>) , putative ® 35 and ® 10 hexamers ( underlined or overlined ) ; potential binding sites for FNR , Lrp and IHF ( boxed ) ; regions of partial dyad symmetry ( converging arrows ) ; ribosome-binding sites ( RBS , consensus bases italicized ) , and translation initiators ( fMet ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>We examined whether returning the NR,-binding sites to their original face of the DNA helix by inserting 5 bp of DNA between the binding sites for IHF and NR , would prevent inhibition of transcription by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>EMSA demonstrated IHF binding to the 277 bp DNA fragment W8-W9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Thusitappears that IHF can bind to quite different recognition sites with high affinity.The narG ' IHFbindingsite , which is centered -125 bp upstream of the narG transcription start ( 32 ) , is located in between the proposed binding sites for the two other essential regulatory proteins , NarL and Fnr ( 12 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>The binding sites for IHF and Fnr have been localised to a 450 bp non-translated regulatory region directly upstream of the pfi operon ( Fig. 1 ) , This segment of DNA includes two of the seven promoters , termed P6 and P7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565118" target="_blank">7565118</a>]<br></li>
<li>This protection extended ~ 30 bp upstream of the -35 hexamer , merging with the footprint of IHF , and some protection was visible even farther upstream between the IHF and Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>This mutation lay within one of the two putative IHF binding sites that are located at 62 to 50 and 37 to 25 , between the AtoC and RNA polymerase binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>The binding sites of IHF , OmpR , RstA and CpxR are indicated along the csgD promoter sequence while the H-NS-binding sequence is shown by a dotted line between the duplex sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The sequences of the putative IHF binding sites upstream of E. coli glmY and Y. pseudotuberculosis glmZ are shown and the nucleotide exchanges introduced in IHF site 1 of the E. coli glmY promoter are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>and an upstream DNA sequence , including the ilvPo1 promoter and IHF binding sites , which stimulates transcription initiation from the iEvP ,2 promoter approximately fourfold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>An obvious IHF-mediated transcriptional activation mechanism could be that , in collaboration with the sequence-directed DNA bend between the IHF and RNA polymerase binding sites , the IHF-induced DNA bend might bring upstream DNA sequences into contact with an RNA polymerase at the iZwPo2 promoter site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>The alignment in Fig. 2 shows that while core promoter sequences ( positions 60 to 36 ) are identical , differences occur in the upstream region ( positions 150 to 60 ) , especially within the Fis and IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The above data do not demonstrate , however , any requirement of the DNA bend for downstream activation , nor do they exclude the possibility that the region between the tandem promoters is required for the function of the upstream activating sequences beyond base-pair -60 that encode the iZvPo1 promoter and IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_314">
The second upstream activating region contains the DNA sequences including the ilvPo1 promoter and IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></label>
<input id="sit_similarity_314", type="checkbox">
<div class="SIT_similar">
<li>The other upstream activating region contains the DNA sequences including the overlapping iZvP , 1 promoter and IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li></div>
<li>The fact that IHF can bind to very relaxed sequences and , to some extend , to DNA with no speciﬁc binding sites , indicates that this protein has a potential to regulate promoter 's activity both speciﬁcally and generally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Since the DNA sites for IHF and Fis are distal to the core promoter sequences , it is unlikely that their binding blocks direct access of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The lower portion of the ﬁgure depicts the location of the binding sites for IHF , the FNR protein ( FNR1 and FNR2 ) , and DNA sequences protected by NarL-phosphate from DNase I attack ( NarL-1 through NarL-4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>In contrast , because pacs236 does not include the sequences for the two upstream sites , IHF protected only sequences that correspond to IHF III cent regions of the DNA ( data not shown ) , a behaviour indicative of binding to lower-affinity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Clustering of characterized H-NS - , IHF - and Fis-binding sites [ ± 100 bp ; ( 18 ) ] located within coding and noncoding sequences was not detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23632166" target="_blank">23632166</a>]<br></li>
<li>This suggests that sequences removed in this fragment , i.e. the IHF binding sites , are necessary for the positive role o -70 & nfI ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>However , not all consensus sites bind IHF and some good IHF binding sites do not fit the consensus , indicating that the flanking sequences are also important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>One of these sites overlaps the pho box , suggesting that IHF may inhibit phoA expression by binding to this sequence and interfering with the binding of the phosphorylated PhoB protein , the activator of the PHO genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Nucleotide sequence surrounding the IHF - and CAP-binding sites and base substitutions within the CAP TGTGA motif ( boldface letters ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>IHF exerts its activity after binding to DNA at the speciﬁc sites with relatively relaxed consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>DNA sequence analysis predicted three putative IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>The location of these sites relative to the DnaA box , R1 , and the IHF binding sequence coincides with a potential blocking of the initiation of DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631765" target="_blank">8631765</a>]<br></li>
<li>The presumptive IHF binding site is shown ( iI ) , as is the cer-like sequence ( ) ; nucleotides 301-364 ) and the two 18 bp arginine repressor binding sites ( ARG boxes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>The atoDAEB promoter contains two sequence stretches ( 62 to 50 and 37 to 25 ) that bear homology to IHF binding sites ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>The sequence alignment also detected two putative IHF binding sites that coincide with the occurrence of s54-promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The two sites identiﬁed by sequence alignment are likely candidates for these IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The binding of IHF to speci ® c ihf sites was detected , in primer extension experiments of irradiated complexes , by an increase in the band intensity of the guanine residue in the consensus pentamer sequence TTGAW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The 13 bp IHF consensus binding sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>Our results show that IHF and Fis bind to multiple sites at pnir but do not interfere with FNR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Taken together , these data show that RNAP , CRP and IHF can bind simultaneously at saturating concentrations for their respective speciﬁc binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Sequence motifs of a s54-promoter , three GlrR binding sites and two IHF binding sites are detect-able .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The two upstream IHF sites colocalize with binding sites for other transcription factors ( NarL/P and FNR ) and with at least one promoter , acsP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>The small open boxes indicate the positions of the putative FNR binding sites , and the small filled box indicates the binding site for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The EMSA allows us to directly measure the binding constant of a speci ® c IHF binding site without interference from other speci ® c or non-speci ® c sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>The location of deletion ( D ) and replacement ( rm ) mutations , binding sites for NanR , NagC and IHF , known promoters ( P ) and Dam methylation sites GATCNanR and GATCNagC are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The interaction of E. coli IHF protein with its speciﬁc binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>The IHF , PepA , and PurR binding sites are indicated with gray colored boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>The IHF and arginine repressor binding sites are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>+50 ~ , + 8 5 ACAGCAGCCGCTTAATTTGCCTTTAAGGAACCGGAGGAATOATG ~ CC T G T C G T C G G C G A A T T A A A C G G A A A T T C C T T G G C C T C C T T A G T A C T G G OB-2 = C BlBIB strong IHF b nd ng I sit ~ s ] I ] weak IHF binding i sites ~ , - , , ~ o ~ + '' ; `` i = ° I I I ATG ÷ 1To / hfA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>( B ) nrf-acs intergenic region , showing the location of binding sites for CRP , FIS , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>( A ) Location of the putative IHF and CRP binding sites in the intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>These sites are responsible for speciﬁc IHF binding , as demonstrated in gel-shift studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Mutagenesis of IHF binding sites and construction of ompBcat transcription fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Potential IHF binding sites upstream of aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>These correspond to IHF binding sites A , B , and C in Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Specific binding of IHF to multiple binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Footprinting IHF binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Bases 01 IHF sites I and 11 that lít the proposed consensus lor íHF bindirlg proposed by Goodrich el al , ( 1990 ) are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>The C-residue of the WATCAA part of the consensus , which is present in the alternative sequence but not In the sequence suggested by Giladi ef ai , is 100 % conserved in all / / i/sites that have been shown to bind IHF ( Goodrich et ai , 1990 ) - Between the two blocks of conserved sequences , many functional ihf siies contain a single A at the third position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>A. Alignment of pnir sequences with the IHF consensus sequence : the sequence around position 299 , which is similar to the IHF consensus ( Friedman , 1988 ; Goodrich et al. , 1990 ) , is indicated in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The generally accepted consensus sequence is WATCAANNNNTTR ( Friedman , 1988 ) , however , more relaxed versions are known to be able to bind IHF as well ( Giladi et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Sequences that exhibit a reasonable match to suggested consensus binding sequences for FIS ( Finkel and Johnson , 1992 ) , leucine-responsive protein ( Rex et al. , 1991 ) , IHF ( Goodrich et al. , 1990 ) , and DnaA proteins ( Schaefer and Messer , 1991 ) are indicated by solid underlining , dashed underlining , dotted overlining , and wavy overlining , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8011018" target="_blank">8011018</a>]<br></li>
<li>The observation that some sequences that match the ihf consensus only poorly can bind IHF ( Gamas et al. , 1987 ; van Rijn et al. , 1991 ) affirms the importance of the surrounding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Matches to the ModE ( Anderson et al. , 1997 ; McNicholas et al. , 1997 ) and IHF consensus sequences ( Freundlich et al. , 1992 ) are bracketed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>IHF sites used in this study : nucleotide sequence of the double-stranded oligonucleotides yjbE promoter 5 ` GATCTATTTTAAGTTTATGATTTTTATTGTTATATAAATA 3 ' 3 ' ATAAAATTCAAATACTAAAAATAACAATATATTTATCTAG 5 ' BIME-1 gyrB 5 ` GATCTCTTGCAATGTATTGAATTTTCATGATTTTGTAGGC 3 ' 3 ' AGAACGTTACATAACTTAAAAGTACTAAAACATCCGCTAG 5 ' IS1 5 ` GATCGACTCCAACTTATTGATAGTGTGAGTTCTATAGCTC 3 ' 3 ' CTGAGGTTGAATAACTATCACACTCAAGATATCGAGCTAG 5 ' l attP H ' 5 ` GATCCGTTGCAACAAATTGATAAGCAATGCTTTTTTATAA 3 ' 3 ' GCAACGTTGTTTAACTATTCGTTACGAAAAAATATTCTAG 5 ' The core consensus sequence is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Binding of IHF to the aldA promoter The promoter sequence presented above displayed three copies of a rather conserved IHF recognition sequence ( positions 7 to -6 , -55 to -67 and -130 to -142 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>The asymmetric nature of the IHF protein corresponds to the asymmetry of the recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>This implies that IHF recognizes DNA conformation ( speciﬁcally , the minor groove ) rather than the sequence spec-iﬁcity of the DNA ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>This sequence is located in one half of the DNA region protected by IHF from Dnase 1 digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>An extremely large region of DNA that extends from downstream of P7 and adjoins the IHF recognition sequence was also protected ( NarL-3a and -3 b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Integration host factor ( IHF ) is a heterodimeric protein from Escherichia coli which specifically binds to an asymmetric consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>An important difference between both proteins , however , is that HU binds the DNA nonwhereas IHF specifically , recognizes a specific sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>IHF shows considerable amino acid sequence similarity with HU but , unlike it , IHF binds to a speciﬁc DNA sequence ( Goosen and van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9987132" target="_blank">9987132</a>]<br></li>
<li>IHF binds to speci ® c sites on DNA ( ihf sites ) that approximate the consensus YAANNNNTTGAW ( where W A or T , Y T or C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF binds to deﬁned sites on the plasmid IHF consensus sequence is relatively relaxed , with many allowed exchanges .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>The stimulation of transcription promoted by NARL-P is dependent on the DNA-binding protein IHF ( 11 ) , which binds to and protects a sequence located between ( and partially overlapping ) the two NARL-P-binding regions ( 6 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>Both of the DNA bending proteins that facilitate the activation of 54-dependent genes in E. coli , IHF and ArgR , have an AT-rich consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>Unlike HU protein , which binds to DNA in a nonspecific manner , IHF binds to a specific DNA consensus sequence , 5 ' - WATCAANNNNTTR-3 ' ( where W = A or T , R = A or G , N = any base ) ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>However , unlike other members of this family which have no or little sequence specificity for DNA binding , IHF binds to the 5 ' - AAT-CAAN4TTA / G-3 ' consensus sequence ; flanking sequences contribute significantly to this binding [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Analysis of the DNA sequences to which IHF binds in phage A has revealed a consensus sequence for IHF binding : PyrAANNNNTTGAT ( 6 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>Introduction [ 5 , 6 ] , although they have distinct DNA-binding activities : HU binds to DNA nonspeciﬁcally whereas IHF binds to a consensus sequence [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>IHF is a sequence-speciﬁc DNA-binding protein which recognizes the core consensus sequence 59-WATCAANNNNTTR-39 ( 10 ) , whereas HU binds to DNA in a sequence-nonspeciﬁc manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892849" target="_blank">8892849</a>]<br></li>
<li>IHF ( i ) folds into a symmetric heteromeric complex , ( ii ) protects large DNA fragments ( 32 to 48 bp ) from DNase I footprinting with DNA bending , and ( iii ) has an asymmetric consensus binding sequence ( i.e. , WATCAANNNNTTR , located at the 3 - half of its footprint region ) ( 14 , 18 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>IHF acts either as a negative or a positive regulator by binding to the 13-bp consensus sequence WATCAANNNNTTR ( W 5A or T ; N 5any base ; R 5A or G ) and bending the DNA [ 11 , 12 , 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>A single IHF heterodimer binds to a site of approximately 35 bp which contains a 13 bp consensus sequence WATCAANNNNTTR -LCB- Craig and Nash , 1984 ; Leong etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>This site is located between the DraI and Sau96I restriction enzyme recognition sequences identified as the region bound by IHF purified from the wild-type extract ( Fig. 2 ) and contains a sequence ( Fig. 1B and C ) which is identical to the 13-bp IHF consensus sequence ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>, Accl , argR ( 261-273 bp ) CAAGGCTTTGACA IHF consensus CAANNNNTTGATA Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>Although site C requires only one-half as much IHF as site A for 50 % protection , the sequence in site C is 1 bp farther from the 13-bp consensus than is the site A sequence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>A 4-bp change was made in the 13-bp IHF consensus sequence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>For example , computer searches for the 13 bp IHF consensus sequence have found a very large number of these potential sites in the E. coli genome ( Mechulam etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>In these three cases the 13 bp IHF consensus sequence of Leong etai ( 1985 ) was used to determine the base score .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>pKPS7 has the same length as pX52 ( 274 bp ) but contains an alteration within the IHF consensus sequence , ATTGAANAACTAA ( Goodrich e t al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8704970" target="_blank">8704970</a>]<br></li>
<li>We have used computer analyses to search for an expanded ( 48 bp ) IHF consensus sequence ( Goodrich etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>b. Total of the score assigned each base in the expanded 48 bp IHF consensus sequence as described by Goodrich etai ( 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Since both IHF subunits are capable of specifically recognizing the ihf consensus ( WATCAANNNNTTR ) , it is conceivable that each subunit makes specific contacts with a part of this consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The asymmetric localization of the consensus sequence within the DNase I footprint ( Craig and Nash , 1984 ; van Rijn et al. , 1991 ) could indicate that only one of the IHF arm regions is recognizing this consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The smaller protein appeared not to be the IHF ( Driica and Rouviere-Yaniv , 1989 ) despite its comparable size and heat stability , and the presence of a sequence , '' '' A C A A A A C T T G A T T '' ^ ^ , overlapping the primary FNR site , which resembles the IHF consensus sequences : C/TAANNNNTTGATA/T ( Craig and Nash , 1984 ) ; and TAAAAGCTTTAAA ( Fiiutowicz and Inman , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8065261" target="_blank">8065261</a>]<br></li>
<li>A thick horizontal line specifies the nucleotides in the consensus IHF target sequence ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>Thus , it is likely that the definition of an IHF consensus sequence must be further refined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>For comparison , the IHF consensus sequence is shown below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The difference in stability could mean that in the IHF heterodimer it is mostly the HimD subunit that is recognizing the consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The extent of protection by IHF is shown by brackets , and the IHF consensus sequence is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_398">
The IHF consensus sequence is shown in bold : W = A or T , R = A or G , n = any base .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></label>
<input id="sit_similarity_398", type="checkbox">
<div class="SIT_similar">
<li>The IHF consensus sequence is shown : W A or T , R A or G , n any base .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li></div>
<li>IHF recognizes the consensus sequence WATCAA-NNNNTTR ( Craig and Nash , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>In relation with these studies , a consensus DNA sequence for the binding of IHF has been proposed ( Craig & Nash , 1984 ; Gardner & Nash , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>The OxyR binding region determined by footprinting ( Zheng et al. , 2001 ) , an IHF consensus sequence and the predicted Fur-box are also indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The presence of a consensus IHF binding sequence in this region further supported this idea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The two overlapping matches to the consensus IHF binding sequence in the glt upstream region ( 295 to 283 and 291 to 279 ) are protected in footprinting assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The binding of IHF ( consensus sequence is WATCAANNNTTR , where W is A or T and R is A or G [ 14 ] ) is known to bend the DNA to facilitate the formation of a local nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>The nucleotides corresponding to those in the 13-bp IHF consensus binding sequence ( 34 ) are enclosed in brackets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The IHF binding site consensus sequence ( Craig & Nash , 1984 ) is shown at the bottom of the Figure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>The IHF-recognilioti sequence within the binding sites deviate by one or more nucleotides from the established IHF consensus sequence ( Craig and Nash , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_409">
The nucleotides corresponding to those in the IHF consensus binding sequence ( Craig and Nash 1984 ; Leong e1 al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></label>
<input id="sit_similarity_409", type="checkbox">
<div class="SIT_similar">
<li>The nucleotides corresponding to those in the IHF consensus binding sequence ( Craig , & Nash .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li></div>
<li>Thus , we inspected the base sequence around position 299 at pnir and found a sequence element that resembles the consensus for IHF ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>For example , in the case of the yjbE site , the 50 TAA 30 sequence , which corresponds to the second conserved part of the IHF consensus , reacts differently in the presence of IHF ; the photo-reactivity of the two external resi-dues increased slightly in the presence of IHF ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>Nucieotide sequence 01 the fimE-fím switch-timA regíon and the position 01 IHF consensus sequences , site I and site 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>The narG IHF site contains4 base mismatches relative t o this consensus sequence whereas no strong similarity was observed to a different IHF consensus sequence proposed by CraigandNash ( YAANNNNTTGATW ( 19 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>In addition , two A. T base pairs ( -39 and -40 ) in the IHF consensus sequence in the IHF-B site were changed to G. C base pairs to create mutant template LH315 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>LH313 is the AvaI-BstBI fragment with the A T base pair at +3 deleted in the IHF site C consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>A second mutant template , LH313 , in which the A. T base pair at +3 in the IHF consensus sequence in the IHF-C site was deleted , was also constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The protected regions are similar in size ( approximately 30 bp ) to those observed at other IHF-binding sites ( Huang et al. , 1990 ; Oppenheim et al. , 1993 ; Resnik et al. , 1996 ) and they contain DNA sequences ( IHF I , +20 TAAcctgTTGTTA +3 * 2 IHF I , -23 CATtcatGT TATT-ll ; and IHF 11 , - `` A C A ~ ~ ~ C T T G A T T - ' w ~ h ) ich resemble the IHF consensus sequence , C/TAAnnnn-TTGATA/T ( Craig & Nash , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>In a number of these cases the physiological function of IHF has been correlated with a 13 bp IHF DNA-binding consensus sequence ( Craig & Nash , 1984 ; Leong et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>In addition , each of the binding sites contained a sequence similar to the 13 bp IHF DNA-binding sequence ( Craig & Nash , 1984 ; Leong et al. , 1985 ) found in known IHF binding sites ( see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>Both sites contain a sequence ( see Fig. 1 ) which differs by 1 bp from the 13 bp I H F DNA-binding consensus sequence derived from known IHF binding sites ( Craig and Nash 1984 ; Leong et al. 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>To test the hypothesis that IHF interacts with this sequence ( termed ibs for IHF binding site ; Fig. 1 ) to control ﬁmB expression , four bases known to be important for IHF binding ( Lee et al. , 1991 ) were altered by sitedirected mutagenesis ( 5 ′ - TTAACTATTTGTT changed to 5 ′ - TTGGCTATGGGTT at the bases underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Inspection of the sequence replaced in rm13 ( 5 ′ - TTAACTATTTGTT changed to 5 ′ - AACCGCGGAACAA ) highlights an element that resembles closely a consensus for IHF binding ( 5 ′ - TAANTNNTTGATT ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Unlike other , structurally related histone-like proteins , IHF is a sequence-specific DNA binding protein , which binds to the 5 ' - AATCAAN4TTA/G -3 ' consensus sequence ( Goodrich et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>Both putative IHF-binding sites show good homology to the highlyconserved bases proposed to form a consensus for IHF binding according to McClure and co-workers ( Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Moreover , the target sites for protein binding in the E. coli and S. typhimurium carAB operons contain the sequences 5 ' - ATTTAATTTATTG-3 ' and 5 ' - TTTTAATATATTG-3 ' , respectively , which resemble the IHF consensus binding site ( 5 ' - AATCAAN4TTA/G -3 ' ; Goodrich et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>DNA sites for IHF and Fis are shown in bold , aligned with their binding site consensus : W = A or T , R = A or G , n = any base ( Goodrich et al. , 1990 ; Finkel and Johnson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>When we searched the DNA sequence around site 1 using the extended IHF consensus ( Goodrich et al. 1990 ) we obtained a similarity score of 34.3 for site 1 , which is far below the value that is recommended as a cutoff for identifying potential IHF-binding sites ( i.e. , 46.2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>Potential regulatory features in the leader region include a ten-base direct repeat , two sequences of dyad symmetry that could confer stable RNA seconda ~ structure , a potential Fis-binding site and a sequence that strongly resembles the IHF binding site proposed by Goodrich et al [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748952" target="_blank">7748952</a>]<br></li>
<li>Analysis of the acs regulatory region revealed additional sequences ( denoted IHF II and IHF III ; Fig. 1 ) that resembled the IHF consensus binding site ( Friedman , 1988 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>s54 -- RNAP binding to the putative P6 promoter IHF , NtrC and PspF binding upstream of the P6 promoter Analysis of the regulatory region of the rpoH gene revealed the presence of a putative IHF-binding site , with remarkable homology to the consensus site ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Measurement of IHF binding by UV footprinting We had reported before ( Engelhorn et al. , 1995 ) that IHF binding to its speci ® c site can be detected by an increase in band intensity of the guanine residue in the consensus pentamer sequence upon irradiation by UV light .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>A dA-i-dT-rich region is often , but not always , found at the 5 ' side of the consensus in the other half of the DNA region occupied by IHF , Fusion of a dA + dT-rich region to a consensus sequence that natuialiy lacks such an element resulted in an ihf site with an increased affinity , suggesting an important role for the dA + dT-rich DNA in stability of the IHF binding ( Hales et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>However , studies of binding to mutant IHF binding sites in the regulatory region of bacteriophage Mu ( van Rijn et al. 1991 ) suggest that it is not the primary but rather the secondary structure of the flanking region that determines whether a consensus sequence is a good IHF binding site or not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The extended IHF consensus sequence of Goodrich al. ( 15 ) used obtain a similarity score of et was to 60.2 for the pfl IHF site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>The ﬁgure shows the pnir sequence from position -136 to -70 and the alignment of the IHF II site with the IHF consensus sequence ( Friedman , 1988 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The presence of an IHF-binding site that corresponded to the consensus sequence ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) and demonstration of binding to IHF , was consistent with previous ﬁndings 54 regarding the structure and functioning of most of the s promoter regulatory regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Comparison of the pnir base sequence with the IHF binding site consensus ( Friedman , 1988 ; Goodrich et al. , 1990 ) indicates that this second IHF site is centred at position -115 ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Moreover , an IHF -- binding site homologous to the consensus sequence was found at the position 2349 to 2362 ( Fig. 1b ) ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>The lootprint lor site I on the top strand ( Fig. 3 , A and E ) extends for 32 bp and overlaps with the consensus lor IHF binding proposed by McClure and co-workers ( Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>The more efficient binding may be a reflection of the greater match that t ' he upstream site has for the IHF consensus ( Fig. 1 ) and of the larger number of A and T residues that are found in this site and its flanking sequences ( Leong et al. , 1985 ; Gamas rt nl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>The REP element at 155279-155312 downstream of o417 , is the same structure as the ` RIP element upstream of avtA ' described by Oppenheim et al. ( 63 ) in that the spacer between the two parts of the REP element is similar to the IHF binding site consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8041620" target="_blank">8041620</a>]<br></li>
<li>The region that is protected from DNase I digestion by one IHF dimer , however , is much larger ( 30-40 bp ) , suggesting that sequences adjacent to the consensus are also involved in protein binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>The strong reduction in the binding affinity observed in the 10 bp AST2 deletion mutant ( retaining an intac consensus site ) illustrates the importance of the flanking sequences for DNA-protein interaction and emphasizes the importance of the A-rich region that flanks the IHF consensus site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>The RIB sequence in the pspF 39 ﬂanking region is composed of two converging REP ( stands for repetitive extragenic palindrome ) sequences , REP1 and REP2 , ﬂanking a conserved segment which carries a static DNA bend and a consensus sequence for the binding of E. coli IHF ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>A complex repetitive sequence ( RIB ) , composed of two imperfect REP sequences and an IHF binding consensus in between , follows the pspF gene immediately further downstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>Despite there being consensus IHF binding sequences close to the cer-like region of argR and within the crossover region of cer , we have not observed reduced recombination in IHF-deficient strains ( our unpublished work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3149585" target="_blank">3149585</a>]<br></li>
<li>It is known that IHF binding is affected by sequences within and outside of the consensus ( Goodrich el ai , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>There are three overlapping IHF binding consensus sequences in the area of footprint III in ihfB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>The putative ribosome binding site ( RBS ) , IHF consensus , and REP sequences ( REP1 and REP2 ) are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>The IHF binding consensus sequences inside of the footprinted areas are boxed and numbered ( roman numbers ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Binding of IHF to sequences between the distant specific activator site ( UAS ) and the 235 and 210 consensus sequences for RNA polymerase binding has been reported for several bending-dependent promoters ( 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>IHF bends DNA by binding to specific nt sequences ( Nash , 1990 ) and is involved in site-specific rec ~ ) mbination , hns \ I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>To do so we used plasmid pJST3ST5 carrying a 10 bp deletion in the consensus part of the IHF target site resulting in an almost complete loss of IHF binding ( fig !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>We have therkfore searched for sequences matching the IHF binding site consensus 14 out of 15 bp , in the E. coli chromosome ( 451,209 base-pairs available in the GENBANK data library , release 44 ) and have found an average of one potential IHF binding site every 2.1 kb in the E. coli genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Purified His6-ArcA ( 50 mg/ml ) was phosphorylated with 50 mM disodium carbamyl phosphate ( Sigma ) at 30 ° C for 1 h. DNA Binding Studies -- For the binding studies with ArcA-P or IHF , the 360-bp DNA fragment containing the glcD promoter sequence ( 2247 to 194 ) was obtained by PCR with primers GLCBam ( 59-CGCG-GATCCGTTCGATACTCTCTGCAACC-39 ) and GLCEco ( 59-GGGGAAT-TCTAAAGCGCCATCAAGACG-39 ) using plasmid pTP25 ( 7 ) as a template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>Thus , we prepared a probe which contained only the first 208 bp of the sodA sequence ( AvaI-StyI fragment of sodA , Fig. 2 ) and compared it with the 255 bp probe in binding IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>Lanes I , 3 , 5 and 7 without added protein , lanes 2 , 4 , 6 and 8 with 0.2 ~ tg purified IHE Mutations reducing the distance separating the binding sitefor IHF ( -305 ) from the GATC sequence ( -106 ) sererely affect the regulation by pyrimidines of promotor PI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>In addition , this putative IHF binding sequence is flanked by a 12 base direct repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>It should be noted that this IHF sequence is flanked by two 12 base regions in direct repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>( TATGCATTI'AA ) deletions altering the distance separating the IHF target site from the Pl promoter were introduced by oligonucleotide-divected mutagenesis using M13ST5 as template and the 25 - and 26-mer oligonucleotides 5 ' - GAATI `` I ` ITATGCTGGCATTGATGGC.Y ( AST7 ) and 5 ' - GATTTfGAA ~ GCATTGATC3GC .3 ' ( AST !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>HimA and HimD bind specifically to the ihf site Surprisingly , incubation of the DNA with HimD alone ( Figure 6 , lanes 5 and 8 ) or HimA alone ( Figure 6 , lanes 13 and 14 ) also results in the formation of a complex which migrates at exactly the same position as the IHF - DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>Despite the large excess of Sau3Al enzyme used in the digests , cleavage reactions were never complete and also some form of star activity was consistently observed at a site GAGTC located some 16 bp upstream of the IHF target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Tsui and Freundlich ( 39 ) have shown that IHF protects a 39-base pair region from DNase I attack at a second IHF target site located in the iluGMEDA leaderattenuator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Alkylation protection and hydroxyl radical footprinting techniques have been used to demonstrate that integration host factor ( IHF ) interacts with the nucleotides in a consensus-like DNA sequence located immediately downstream of the RNA polymerase transcriptional pause site in the leader-attenuator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>In this report , we have used alkylation protection and hydroxyl radical `` footprinting '' techniques to show that IHF protein interacts with the bases of a consensus IHF target sequence , previously suggested by Tsui and Freundlich ( 39 ) , located immediately downstream of the RNA polymerase transcriptional pause site in the ilvGMEDA leader-attenuator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>It is of functional IHF site , were inserted upstream of the strong rrnB termi - significance to note that IHF half-maximally represses trannator sequence contained in the plasmid pDD3 to yield the scription from the iluPGl promoter at a concentration of plasmid constructspJP29 and pJP3 ( 0Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>A second suppressing factor is IHF , that binds to a site located at position 288 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Here , we have shown that the unidentified factor is IHF , which binds to a site resembling the consensus that overlaps position 299 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>IHF also participates in the expression of some & 70-transcribed promoters which are regulated by two activator proteins , where one activator binds at a site proximal to the promoter and the other has a recognition sequence which is more remote ( 32 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF binds to a site between the UAS and s54-RNAP consensus sequence of the psp operon ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>In addition , psp expression is IHF dependent ( 57 ) , and IHF binds to a site between the UAS and the s54 recognition sequence ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>We further show that Integration host factor ( IHF ) binds In vitro to a site between the UAS and a94 recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>IHF binds in vitro to a site ( position -60 to -25 ) located between the UAS and a54 recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>The 243 bp HphI-BsmI fragment extends from positions -164 to +79 with respect to the transcription start site and includes the upstream activation region , IHF site , y54 recognition sequence , and first 13 codons of pspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>The IHF protein is required for maximum transcription from PR1 at 37 ° C and binds to a single site located 130 bp 3 to the transcription start point at PR1 ( Fig. 1 ) ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17827501" target="_blank">17827501</a>]<br></li>
<li>The ideal IHF consensus sequence was introduced to pTE103-pR derived construct , in the position +88 relative to the transcription start site , by site-directed mutagenesis , using oligonucleotides pR-IHF1 ( 50-GAC AGC TAA CAA TCA AGG CGT TTA TC-30 ) and pR-IHF2 ( 50-GAT AAA CGC CTT GAT TGT TAG CTG TC-30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>In vitro transcription from the pR promoter located on supercoiled or liner templates , containing wild-type pR sequence ( open circles ) or pR with IHF consensus sequence introduced downstream of transcription start site ( closed circles ) at indicated amounts of the IHF protein used in the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>Part of the oxidant induction is mediated by OxyR that binds to ORE-I site ( -236 to -197 nt upstream from the transcription start site ) and probably contacts RNA polymerase through a loop formation facilitated by IHF bound at ORE-II ( -156 to -127 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Accordingly , IHF bound al site 1may act to promote either ( i ) protein-prolein interactions between an unidentifísd prolein bound in Ihe 3 ' segment 01 timE and a proteín bound to the tim switch , or ( ii ) ONA-protein interactions across Ihe same region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Wilh these observa tions in mind , IHF bound lo site I might acl by sleric hindrance lO block Ihe binding 01 a second protein thal in turn blocks FimB , bul not Irom binding lo IRL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>^ TGCCTGATGCGCTCCGCTT.ATCAGGCCTACG.V > AATTCTGC .4 ATG IleGluEnd REP-L IHF site B 22 50 232 9 23 40 • rATTGAATTTGCACGATTTl ` GTAGGCCGGATAAGGCGTTAACGCCGCATCCGGCATAAAC ` AA.a.GCGCACTTTGTC.i ^ AC ^ i - ^ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536291" target="_blank">7536291</a>]<br></li>
<li>IHF protected a single site from approximately position -60 to -25 ( Fig. 4 ) ; one IHF molecule normally protects -35 bp from DNAse I digestion ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>Insertions of 5 or 10 bp between this ihf site and ﬁs P suggest that IHF functions in a position-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>In the pnrf53 / 5 and pnrf53 / 10 promoter fragments , the IHF I site has been disrupted by the insertion of 5 or 10 bp of DNA at position 56 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>IHF inhibits the PcM promoter in Mu when it is bound to a site 80 bp downstream from the start of transcription ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Previously , we demonstrated that NarL activates FNR-dependent transcription at pnrf by displacing IHF bound to the IHF I site ( 3 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>We now show that NarL can displace IHF bound at the IHF I site , but IHF is unable to displace bound NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>NarL displaces IHF bound to site I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>At present , the mechanism by which IHF bound to the IHF III site stimulates transcription initiation is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>We showed that IHF bound to a site centered approximately at 2114 , causing an increase in ﬁs P transcription in a position-depen-dent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>An interesting possibility is that Fis bound to site V collaborates with IHF bound at 2114 to increase transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Á T I r e \ 1 A \ 1 \ s ' s ' 5 ' E SITE 1 IR L Z590TTTGATTTAACTTATTGATAATAAAGTIAAAAAAACAAATAAATACAAGACAATTGGGGCCA AAACTAAATTGAATAACTATTATTTCAATTTTTTIGTTTATTTATGTTCTGTTAACCCCGGT F SITE II Z840TTACAATAGAAATAATTTACATATCAAACAGTTAGATGCTTTT AATGTTATCTTTATTAAATGTATAGTTTGTCAATCTACGAAAA ???? Fig. 3 , In sítu 1,10 phenanthroline copper footprinting 01 IHF bound lo site 1 ( panels A and B ) and site 1I ( panels e and D ) 01 the fim switch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>+10 + 1 -10 -20 '' 10 box ~ J IHF box t ~ FNR site Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9315722" target="_blank">9315722</a>]<br></li>
<li>This analysis conﬁrmed that IHF bound to the site identiﬁed by site-directed mutagenesis ( Fig. 4 , lanes 6 -- 10 and 16 -- 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Four highly conserved nucleotides ( Supplementary Figure S3 ) were exchanged within the putative IHF site 1 ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The hatched box represents the IHF footprint site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>The distance separating the IHF target site fi'om the Dam methylation site is crucial for IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>Since the IHF-2 region is almost twice the length of IHF-1 , two molecules of IHF should bind on the IHF-2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>ModE and IHF bind dmsA in the vicinity of the P1 transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_501">
Both OmpR and IHF are able to bind to the IHF-2 site simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></label>
<input id="sit_similarity_501", type="checkbox">
<div class="SIT_similar">
<li>Both RstA and IHF are able to bind to the IHF-2 site simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li></div>
<li>The carA promoter fragment of 500 bp in length upstream from the carA initiation codon , including the target sites for all the transcription factors , PepA , PurR , PepA , RutR and IHF ( see Fig. 3B ) , was inserted into TFP vector so to adjust the initiation codon to that of green ﬂuorescent protein ( GFP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></li>
<li>DNase I , hydroxyl radical and dimethylsulphate footprinting experiments allowed us to allocate the IHF binding site to a 38 bp , highly A+T - rich stretch , centred around nucleotide - 305 upstream of the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>IHF specifically binds to a 38 bp highly A-T rich stretch centered around nucleotide -305 upstream of the P I startpoint of transcription in the E coil and S typhimurium carAB operons [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748925" target="_blank">7748925</a>]<br></li>
<li>IHF protects against nuclease digestion a 38 bp very A + T-rich region centred around nucteotide - 3 0 5 upstream of the P1 promoter startpoint of transcription in both organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>At present , we can not decide whether cAMP-CRP , Lrp , and IHF directly bind to the osmY regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Here we identified that five regulatory proteins ( CpxR , OmpR , RstA , IHF and H-NS ) bind within a narrow region of about 229 bp along the csgD promoter ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>As seen in Fig. 7 , both promoter regions bind IHF specifically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>This implies that more Fis binds to the nrf promoter region p23C p10T p8T p7T p22C / p22A p21A p42T NarL / NarP pnrf FNR TAACTCTAAAGTGGTATTTTACATGCACTTACAATTGATTAAAGACAACATTCACAGTGTGGTTATTTGTTACACATAGGGGcGAGCA WWWWWWnnnnnnnnWATCAAnnnnTTR GntcAaaTttTganC +1 IHF Consensus tg tAa c Fis Consensus Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>the pLl promoter of bacteriophage I ( Giladi et al , 1990 ) , and the PQ2 promoter of the IlvGMEDA operon of E. coli ( Pagel and Hatfield , 1991 ) , in all three cases , IHF binds to a site located just upstream from the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>In some of the cases in which IHF inhibits gene expression , IHF binds to a site in the promoter that overlaps the -10 and/or -35 region ( 15 , 23 , 42 ; Tsui et al. , submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>We have previously found that IHF binds specifically to a site just upstream from the ilvBN promoter and strongly decreases transcriptional pausing and termination in the ilvBN leader .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>IHF also binds at promoter-distal site IHF-1 ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>These results showed that IHF in an nM concentration range binds specifically near the transcription start site on the proU promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25673659" target="_blank">25673659</a>]<br></li>
<li>IHF binds to thepfl promoter-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF binds to ompC promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>5769 In the present report , we show that IHF binds to a single site in the pfl promoter-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>IHF binds to a site centred at -103 in the tdc promoter region , and tdc expression was strongly reduced in IHF mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The ﬁrst two nucleotides in the 235 site of the P1 promoter overlap the IHF protected region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>The induction was dependent on OxyR , IHF and an unidenti-ﬁed factor that binds to the ORE-III site in close proximity to the -35 region of the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>This hypothesis was tested by replacing all but the upstream 22 bp of the Lrpbinding region in a gcvT : : lacZfusion with the I I A site from phage A. Integration host factor ( IHF ) binds the I I A site and bends the DNA about 140O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>Band-shift assays , performed with S-30 extracts of the wild type and a himA , hip double mutant or with purified IHF demonstrate that , in vitro , this factor binds to a region 300 bp upstream of the transcription initiation site of P1 in both organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>IHF binds to the gltBDF upstream region with an apparent Kd of 5.7 6 0.3 nM ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>IHF binds to ompC DNA and protects a region of 35 base pairs located upstream from the ompC promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Since the nrf-acs intergenic region contains three IHF sites , we investigated whether IHF bound to the IHF III site functioned independently of IHF bound to other sites within pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Within the nrfA-acs intergenic region , it binds to three higher-afﬁnity sites : IHF I to IHF III , centered at positions 226 , 180 , and 153 , respectively ( Fig. 11B ) ( 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>IHF bound te sites !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>It is clear that IHF binds to multiple sites in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Puriﬁed IHF protected a large region ( positions -238 to -140 ) , suggesting that IHF binds to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>IHF binds to two sites within pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>IHF binds to sites centred at -179 , -85 , -42 , and +131 in ompF , ilvBN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF was found to bind to two sites in this region and to block omnpF-specific transcription in a partially purified in vitro system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Within the acs promoter region , Fis and IHF each bind to three sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630973" target="_blank">17630973</a>]<br></li>
<li>Most of the nucleotides involved in CysB and IHF binding are conserved between the two species Discussion We demonstrated by gel retardation and footprinting that IHF binds to the cysJIH promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>We therefore tested whether IHF binds to the promoter region of phoA by performing a gel-retardation assay of a PCR fragment of 180 bp that includes the promoter region of phoA and the two putative IHF-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Gel retardation assays and DNase 1protection experiments showed that IHF binds to two sites in the ompF promoter region centered at positions -180 and -60 relative to the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>This suggests that the himD/hip promoter region binds IHF at two distinct sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>In the present report , we show by gel retardation analysis and DNase I footprinting that IHF binds specifically to three sites in the ompB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Gel retardation analysis and DNase 1 footprinting show that IHF binds to two sites in the ompF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Thus , the functional IHF site between nt -10 and +35 resides about 65 nucleotides downstream from the CAP site and overlaps the two major transcription start sites ( at nt +1 and +23 ) , which are inhibited and activated , respectively , by the cAMP-CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Using mutational analysis and gel retardation assays , we showed that IHF binds to a site centred at position -54 , overlapping the DNA sites for FNR and NarL/NarP , while Fis binds to a site at position -15 , overlapping the promoter -10 element ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Once phosphorylated , NarL or NarP binds to the same target site centered at position 69.5 ( Fig. 1A ) and activates transcription by displacing IHF from IHF I , thereby counteracting the repression by IHF and Fis from upstream sites ( 3 , 25 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>himD/hip site I himD/hip site II himA P4 site lliF consensus TTAGTGGTTGATT TAATTCTCTGACT CAACTACCTGATT YAANNMHTTGATW IHF binds to these sites af about a 10-fotd lower affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>It is clear that IHF binds to the fragment containing both IHF sites and to each of the fragments that carries a single site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>These values indicate that IHF binds 1000 to 10,000-fold more avidly to a speci ® c site than to random , non-speci ® c sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>IHF binds to the ompF and ompC promoter regions at sites centred at -179 and -176 , respectively , and to an additional site in ompFcentred at - 6 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF binds to three sites on the ompB promoter between positions -117 and -82 , -55 and -18 , and -10 and +35 , whereas CAP occupies a single site between positions -69 and -36 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by IHF</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Fragments containing the ihf sites located near the yjbE promoter , l attP H0 site , and BIME-1 gyrB were obtained by PCR ampli ® cation using as primers the oli-gonucleotides 50-GGGGGGTACCATTTCGGAAAATAC-GCAG-30 and 50-GCGCTCTAGAGCGATGGCGTTTTA-GG-30 ( yjbE ) , 50-GGGGGGTACCTACGTTTCTCGTTCA-GC-30 and 50-GGGCTCTAGACAGGGAGTGGGACAA-AATTG-30 ( l attP H0 ) , 50-CCGGGGTACCTATTTAA-TGGCGCTAACC-30 and 50-CCGCTCTAGATTCTTGCC-GGATGCGG-30 ( BIME-1 gyrB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837718" target="_blank">9837718</a>]<br></li>
<li>This showed that the critica1 factor was not the presence or absence of active fimB in the ihf mutant background , but the copy nurnber of the active fimB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_3">
prpR , ntrA , and ihf functions are required for expression of the prpBCDE operon , encoding enzymes that catabolize propionate in Salmonella enterica serovar typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></label>
<input id="tu_similarity_3", type="checkbox">
<div class="TU_similar">
<li>prpR , ntrA , and ihf functions are required for expression of the prpBCDE operon , encoding enzymes that catabolize propionate in Salmonella enterica serovar Typhi-murium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>prpR , ntrA , and ihf functions are required for expression of the prpBCDE operon , encoding enzymes that catabolize propionate in Salmonella enterica serovar Typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li></div>
<li>However , the fact that expression of the rpoS , lrp and ihf genes is stimulated by ppGpp ( Aviv et al. , 1994 ; Lange et al. , 1995 ) and that many genes induced by carbon , phosphorus and nitrogen starvation are regulated by the stringent response ( Nystro $ m , 1995 ; Rao et al. , 1998 ) , is consistent with YeiL being involved in the response to nutrient starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Genes most highly repressed upon IbsC , ShoB , LdrD and TisB overexpression.a Average fold repression dusB/yhdG ﬁs yeeF apt efeU ( ycdN ) CRP , IHF , Fis tRNA dihydrouridine synthase B Transcriptional activator Putative transporter Adenine phosphoribosyl transferase Inactive ferrous iron permease 6.5 7.7 0.9 0.9 0.8 0.9 1.2 1.1 1.1 1.2 1.2 2.1 4.3 3.8 4.7 2.3 1.2 17.8 6.1 21.2 CpxR Inactive ferrous iron transporter Inactive ferrous iron transporter Porin Unknown efeO ( ycdO ) efeB ( ycdB ) ompF yagU 14.3 2.6 6.1 26.7 1.0 1.8 1.4 1.7 1.2 1.8 0.9 2.5 1.2 1.2 0.5 1.2 OmpR , EnvYd a. Genes and operons which are repressed 5-fold in three independent experiments are listed with the average fold repression for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_9">
Lee , J.H. , Yeo , W.S. , and Roe , J.H. ( 2004 ) Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentiﬁed oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></label>
<input id="tu_similarity_9", type="checkbox">
<div class="TU_similar">
<li>Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentiﬁed oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hdyrogen peroxide : involvement of OxyR , IHF and an unidentified oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentified oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li></div>
<li>The sufABCDSE operon is directly regulated by OxyR , IHF , Fur and an unidentiﬁed oxidant-responsive ( ORE ) activator ( Lee and Roe , 1997 ; Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>From these results , we propose that the oxidant induction of the sufA operon is mediated through OxyR , IHF , plus an unidentiﬁed oxidant-responsive factor , and that the suf gene products are needed to Accepted 17 November , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Interestingly it has also been shown that both ihf genes ( Aviv et al. , 1994 ) as well as the hupB gene ( Claret & Rouviere-Yaniv , 1996 ) are controlled by ppGpp in addition to growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>The transcription pro ® les of both ihf genes are similar to that of HUb with maximal expression when the cell growth begins to slow down .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367749" target="_blank">9367749</a>]<br></li>
<li>The major nucleoid proteins , HU , IHF , H-NS and Fis , in exponential growth phase E. coli under aerobic conditions are all known to have functional dichotomy ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response to oxidative stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Surprisingly , the same promoter is also activated by IHF and a ^ at the onset of stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_20">
The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10892648" target="_blank">10892648</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631665" target="_blank">8631665</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542180" target="_blank">10542180</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></label>
<input id="tu_similarity_20", type="checkbox">
<div class="TU_similar">
<li>The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829938" target="_blank">9829938</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8779563" target="_blank">8779563</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751275" target="_blank">7751275</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss during stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li></div>
<li>pKOXS B. Demple Media and cell growth conditions Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during P growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Referencec Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Ston , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>J Mol Biol 215 :403 410 Altuvia S , Almir6n M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709966" target="_blank">8709966</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_29">
[ 5 ] S. Altuvia , M. Almiron , G. Huisman , R. Kolter , G. Storz , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></label>
<input id="tu_similarity_29", type="checkbox">
<div class="TU_similar">
<li>Altuvia , S. , Almiron , M. , Gjalt , H. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>References Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_33">
Genes Dev 6:2646 -2654 Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ~ rs in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476863" target="_blank">7476863</a>]<br></label>
<input id="tu_similarity_33", type="checkbox">
<div class="TU_similar">
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>References Altuvia S , Almiron M , Hulsman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and rS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>References Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during S growth and by IHF and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R & Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF S and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigmaS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and s S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li></div>
<li>Although OxyR can activate transcription by Es70 , and IHF can enhance transcription by Es38 ( Altuvia et al. , 1994 ; Jeong et al. , 2006 ) , the molecular basis of how dps expression is regulated by growth phase is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>The level of IHF increases in the stationary growth phase ( Ditto et al. 1994 ) and several stress-speciﬁc rS-dependent promoters , such as ihfA , dps , osmY and sra , are known to be regulated by IHF ( Arﬁn et al. 2000 ; Lange et al. 1993 ; Izutsu et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>Upon entry into stationary phase , transcription of dps is ensured by Es , stimulated by the nucleoids associated protein IHF and OxyR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>We conclude that it is the level of transcription initiation ot the IHF genes , and not mRNA stability , that increases upon entry of the cells into the stationary phase of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>As analyzed graphically in Fig. 4B , these results indicate that binding of NARL-P does not bend the DNA or alter the bending of DNA induced by IHF ; neither the apparent center of bending nor the degree of bending , which was calculated according to the method described by Thompson and Landy ( 17 ) , was signiﬁcantly modi-ﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>Transcription of acs by cells grown in glucose minimal medium relative to optical density , consumption of carbon sources , excretion of acetate , and intracellular concentrations of FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The boundaries of KplE1 prophage are 16-bp-duplicated core sequences ( CTGCAGGGGACACCAT ) separated by 10.216 kb outlined by the argW tRNA gene and the dsdC gene , and com ¬ 3 The abbreviations used are : Int , integrase ; IHF , integration host factor ; RDF , recombination directionality factor ; AU , arbitrary unit ( s ) ; Xis , excisionase ; Tricine , N - [ 2-hydroxy-1 ,1 - bis ( hydroxymethyl ) ethyl ] glycine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17545146" target="_blank">17545146</a>]<br></li>
<li>Expression of hemkA-lacZ was reduced by twofold aerobically and elevated by almost twofold anaerobically compared with that in the wildtype strain , suggesting that integration host factor protein ( IHF ) is somehow important in controlling the DNA topology of the hemA regulatory region , since IHF is known to cause bending of DNA upon binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071201" target="_blank">8071201</a>]<br></li>
<li>Under these conditions , cAMP-CRP and IHF do not contribute to the specific regulatory pattern of starvation induction ( which is no longer found in a strain that is deficient for as and Lrp but has wild-type cya , crp , himA , and hip alleles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>TcbC does not functionally replace any of the tdc operon regulators Expression of the tdc operon is controlled directly by a number of positive transcriptional regulators , including CRP , IHF , TdcA and TdcR ( Wu et al. , 1992 ; Hagewood et al. , 1994 ) , as well as being controlled indirectly by FNR ( Chattopadhyay et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>We draw attention to the fact that the negative feedback control of hns described here is shared by the structural genes for the HU , IHF and Fis proteíns ( Miller , 1981 ; Kohno et al. , 1990 ; Ninnemann et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>Membrillo-Herna ́ndez , J. , Cook , G.M. , and Poole , R.K. ( 1997b ) Roles of ss , IHF and ppGpp in the expression of hmp encoding the ﬂavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>Appl Environ Microbiol 78:3369 -- 3378 Membrillo-Hernández J , Cook GM , Poole RK ( 1997 ) Roles of RpoS ( sigmaS ) , IHF andppGpp in the expression of the hmp gene encoding the flavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23824318" target="_blank">23824318</a>]<br></li>
<li>FEBS Lett 382:141 -- 144 Membrillo-Hernández J , Cook GM , Poole RK ( 1997a ) Roles of RpoS ( S ) , IHF and ppGpp in the expression of the hmp gene encoding the Xavohaemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_58">
Roles of RpoS ( S ) , IHF and ppGpp in the expression of the hmp gene encoding the ﬂavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></label>
<input id="tu_similarity_58", type="checkbox">
<div class="TU_similar">
<li>Membrillo-Hernandez , J. , Cook , G.M. , and Poole , R.K. ( 1997 ) Roles of RpoS ( sigmaS ) , IHF and ppGpp in the expression of the hmp gene encoding the ﬂavohemoglobin ( Hmp ) of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>[ 23 ] Membrillo-Hernandez J. , Cook G.M. , Poole R.K. , Roles of RpoS ( sigmaS ) , IHF and ppGpp in the expression of the hmp gene encoding the flavohemoglobin ( Hmp ) of Escherichia coli K-I 2 , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
<li>Prof. Lineu Prestes 1374 , CEP 05508-900 São Paulo , Brazil E-mail : benys@usp.br Tel. : 55-11-30917347 Fax : 55-11-30917354 E. Yagil niversity , Department of Biochemistry , Tel-Aviv U 69978 Tel-Aviv , Israel In the present report we show that IHF plays a po-sitive role in the ErS-dependent transcription of the pst operon and that a 13T to 13C mutation in the IHF-independent promoter of phoA does not permit its transcription by ErS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Control of the sucABCD genes by Fnr and ArcA is from the sdhC promoter , while IHF control is primarily from the more weakly expressed sucA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209026" target="_blank">9209026</a>]<br></li>
<li>CRP activates acs transcription by binding tandem sites located upstream of the major promoter , acsP2 , whereas , both IHF and Fis each act at different times during growth as anti-activators of CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Integration of three signals at the Escherichia coli nrf ole f Summary Expression from the Escherichia coli nrf operon promoter is activated by the anaerobically triggered transcription factor , FNR , and by the nitrate/nitrite ioncontrolled response regulators , NarL or NarP , but is repressed by the IHF and Fis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>As at the nrf promoter , the nir promoter can be activated by FNR alone but FNR-dependent activation is repressed by IHF and Fis ( Wu et al. , 1998 ; Browning et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Another example is the promoter of the nrfA operon that is activated by FNR and repressed by Fis and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Expressionfrom the suc promoter was repressed by IHF and partially activated by d8but it was not regulated by ArcA , FNR , CRP , FruR or Fis , and not repressed by glucose or anaerobiosis , indicating that the well-established catabolite and anaerobic repression of ODH synthesis is imposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>Incontrast , the / pel promoter was repressed by both glucose ( via a CRP-independentmechanism ) and anaerobiosis ( mediated by ArcA ) , and activated by Fis , but it was not regulated by FNR , FruR , IHF or These observations support the view that transcription of the sucABCD genes is primarily initiated and regulated at the upstream sdh promoter , and that the Ipd promoter is independently co-regulated with Pdh ( primarily by ArcAmediated repression ) rather than with PsucDirect evidence for co-transcription of the entire sdhCDAksucABCD region from PHNw t as obtained by detecting a 10 kb transcript inm c and me mutants , but not inthe parental strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720032" target="_blank">9720032</a>]<br></li>
<li>One unit , prpBCDE , encodes most of the enzymes Transcription of the prpBCDE operon of S. enterica is known to require the functions of the PrpR , RpoN and IHF proteins ( Palacios & Escalante-Semerena , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Because our experiments indicated that pspG transcription is regulated by the same elements that regulate psp operon transcription , we used a bioinformatic approach to search the pspG promoter region for the control elements that are present in the psp operon promoter , which are binding sites for 54 , PspF , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>pspG is physically unlinked with the psp operon but is co-conserved 54 and co-regulated with the psp operon genes by , PspF , IHF , and PspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>It is striking that the psp operon and the pspG 54 promoters , which are physically unlinked , are both regulated by PspF , PspA , and IHF in exactly the same fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>Expression of the chromosomally encoded E. coli glmY ' - lacZ fusion was determined in mutants lacking ihfA or ihfB , which encode the subunits of IHF ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The ihfB ( himD ) gene , encoding the 13subunit of IHF , is located downstream of rpsA , coding for ribosomal protein S1 ( Pedersen et al. , 1984 ; Flamm and Weisberg , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8973312" target="_blank">8973312</a>]<br></li>
<li>The relative mobility shift plots of the same data ( Fig. 6C and D ) clearly reveal that both IHF and CAP induced significant DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>The same result was obtained when this construct was expressed in a himA mutant ( not shown ) indicating that the IHF effects are mediated by sequences upstream of position 254 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>The fusion carrying the sodB5 fragment , the 59 extremity of which was located at position 2110 , was repressed by IHF , whereas the fragment carrying the sodB4 fragment , the 59 extremity of which was located at position 239 , was not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>The ability of IHF to relieve repression under inducing but not repressing conditions allows this protein to amplify the induction of aceBAK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>IHF contributes to the induction of aceBAK by opposing repression by IclR during growth on acetate ( inducing conditions ) but not on glucose ( repressing conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550439" target="_blank">8550439</a>]<br></li>
<li>IHF prevents repression by IclR under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Thus , we conclude that IHF and Fis can repress pnir in the absence of H-NS ( Table 2 ) , and that repression by H-NS can not be reversed by the action of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>HU can not be replaced by IHF , Fis or HNS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
<li>Functional substitution of IHF by HU and HNS is not without precedent ( Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_85">
Charlier , D. , Roovers , M. , Gigot , D. , Huysveld , N. , Pierard , A. , and Glansdorff , N. ( 1993 ) Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17919280" target="_blank">17919280</a>]<br></label>
<input id="tu_similarity_85", type="checkbox">
<div class="TU_similar">
<li>Charlier , D. , Roovers , M. , Gigot , D. , Huysveld , N. , Piérard , A. and Glansdorff , N. ( 1993 ) Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472642" target="_blank">20472642</a>]<br></li>
<li>Integration host factor ( IHF ) modulates the expression of the pyrimidine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621816" target="_blank">16621816</a>]<br></li>
<li>Integration host factor ( IHF ) modulates the expression of the pyrimi-dine-speciﬁc promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14741201" target="_blank">14741201</a>]<br></li>
<li>Integration host factor ( IHF ) modulates the expression of the pyrimi-dine-speci ® c promoter of the carAB operons of Escherichia coli K12 and Salmonella typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li></div>
<li>Integration host factor ( IHF ) binds upstream of and stimulates transcription initiation at Pl , the pyrimidine specific promoter of the carAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1640456" target="_blank">1640456</a>]<br></li>
<li>On the role of the Escherichia coli intergration host factor ( IHF ) in repression at a distance of the pyrimi-dine-speci ® c promoter P1 of the carAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677289" target="_blank">9677289</a>]<br></li>
<li>Pﬁs is negatively autoregulated by Fis binding at two sites centered at 44 and 25 relative to the transcription start site ( 3 , 37 , 47 ) and positively controlled by integration host factor ( IHF ) binding at a single site centered at 94 ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885445" target="_blank">16885445</a>]<br></li>
<li>It is known ( 14 ) that Xp2 expresses one of the two subunits of the integration host factor ( IHF ) coded by himA , the other subunit being encoded by the himD gene ( also called hip [ 13 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2991205" target="_blank">2991205</a>]<br></li>
<li>Here it is demonstrated that NagC binding to a second site ( ONC2 ) , located 212 bp closer to ﬁmB , also controls ﬁmB expression , and that integration host factor ( IHF ) , which binds midway between ONC1 and ONC2 , facilitates NagC binding to its two operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The integration host factor IHF encoded by ihfA and ihfB also plays a dual role in the regulation of type I fimbriae expression since it is required both for inversion of the switching element in both directions and for efficient transcription of fimA [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>The expression also depends on the integrative host factor ( IHF ) and is repressed by the respiratory regulator ArcA-P [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22168595" target="_blank">22168595</a>]<br></li>
<li>Additional regulators of dmsABC expression have also been described that involve molybdenum dependent gene expression via the ModE transcription factor , and integration host factor ( IHF ) that interacts with DNA at a downstream site called P2 [ 4,6,11,12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079504" target="_blank">12079504</a>]<br></li>
<li>The DNA bending proteins that participate in - depen-dent gene expression in E. coli are integration host factor ( IHF ) and ArgR , the arginine repressor ( 72 , 103 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>Less is known concerning the role of integration host factor ( IHF ) in E. coli , although it has been implicated in a number of processes in this organism including DNA replication , site-specific recombination , and gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The promoter upstream of the sucA gene is repressed by integration host factor ( IHF ) and partially activated by S .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659676" target="_blank">15659676</a>]<br></li>
<li>In E.coli it is repressed by Fur , ArcA , Fnr and integration host factor IHF , and it is activated by the product of the soxRS locus and mutated alleles of soxQ locus ( such as CfxB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812853" target="_blank">11812853</a>]<br></li>
<li>Expression of the glc operon is strongly dependent on the integration host factor ( IHF ) and is repressed by the global respiratory regulator ArcA-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>Expression of this operon is induced by growth on glycolate and is under the control of an activator protein encoded by the divergently transcribed gene glcC ( see Fig. 1 ) Expression is strongly dependent on the integration host factor ( IHF ) and is repressed by the global respiratory regulator ArcA-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11283302" target="_blank">11283302</a>]<br></li>
<li>The integration host factor ( IHF ) is required for the induction of the atoDAEB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>Conversely , PspB and PspC act as positive regulators of psp operon transcription by overcoming the negative regulation 1 The abbreviations used are : pIV , protein IV ; IHF , integration host factor ; IPTG , isopropyl 1-thio - - D-galactopyranoside ; RT , reverse transcriptase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>This positive and negative control by cyclic AMP-CRP is apparent in both the IHF + and IHF - strains ( Fig. 2A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Arch Microbiol ( 2006 ) 185 : 234 -- 237 DOI 10.1007 / s00203-005-0082-4 SH O RT CO MM U N IC A T IO N Natalia Pasternak Taschner Æ Ezra Yagil Beny Spira S The effect of IHF on r of Escherichia coli phoA and pst promoters selectivity of the Abstract The pst operon , a member of the PHO regulon of Escherichia coli , encodes a high-afinity phosphate transport system whose expression is induced when the cells enter a phase of phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>dps and sra are positively regulated , while osmY and ihfA are negatively affected by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>This conclusion is supported by the growing number of reports of IHF involvement in cellular metabolism during stationary phase [ 21-25 ] * Correspondence and reprints Both ihfA and ihfB were shown to be negatively regulated by IHF [ 20 , 26-28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>We conclude that IHF plays a negative role in the control of osmY transcription during transition into stationary phase when the cells are grown in rich medium or in glucose-containing minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>And in fact , IHF has been identified as a positive factor directly activating the transcription of several stationary phase induced genes [ 21 , 22 , 24 , 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>The complex transcriptional regulation of the csgDEFG operon will also have to be considered , which occurs in stationary phase and involves several global regulators such as RpoS , HNS , IHF and OmpR ( Barnhart and Chapman , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>Thus , not only does IHF stimulate both FimB and FimE recombination , as well as activate ﬁmbrial expression in phase ON cells , but it is now clear that it enhances ﬁmB expression too .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Such regulation is not by site-specific recombination , and , indeed , IHF has not previously been shown to be involved in the control of gene expression via phase variation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>While IHF overproduction was not sufﬁcient to induce IHF-dependent Gsps in the exponential growth phase , it did signiﬁcantly induce a group of proteins whose expression is dependent on the heat shock regulator s32 ( Fig. 4 ) ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Thus the growth phase dependent transcriptional regulators Fis and IHF stimulate the expression of the nuo operon and repress ndh , and the resulting increase of Nuo at the expense of Ndh raises the ATP yield in early exponential aerobic growth ( Wackwitz et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931557" target="_blank">11931557</a>]<br></li>
<li>Thus , the silencing of csgD by H-NS should be reduced in stationary phase because of the increase in IHF level ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>A yeiL -- lacZ transcriptional fusion showed that aerobic yeiL expression increases at least sixfold during stationary phase , requires RpoS , and is positively autoregulated by YeiL , positively activated by Lrp ( and IHF in the absence of FNR ) and negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>10 600 ( a ) â-Galactosidase â-Galactosidase 400 1 wt yeiL rpoS 200 0 · 1 ihfA Studies with monolysogens lacking YeiL or various transcription factors indicated that in the stationary phase aerobic yeiL expression is positively autoregulated by YeiL , and signiﬁcantly dependent on RpoS ( σS ) and Lrp but not IHF ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>It has previously been observed that IHF mutants showed reduced growth on limiting amounts of inorganic sulfur sources and on djenkolate ( 33 , 34 ) , which supported the assumption that IHF is required for full expression of cysJIH in stationary phase but not during exponential phase ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10506196" target="_blank">10506196</a>]<br></li>
<li>These results suggest that FIS , IHF , cAMP , and ppGpp are involved in the positive regulation of sra expression during the stationary growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>However , the expression of sra during the stationary phase in the mutants defective in FIS , IHF , cAMP , and ppGpp was signiﬁcantly lower than that in the corresponding wild-type strains , as shown in Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>However , details of the mechanisms of ﬁne control of sra expression in the exponential phase as well as in the stationary phase by sigma S , OmpR , FIS , IHF , cAMP , and/or ppGpp remain to be investigated further .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Studieswith the ndh-lacZ fusion combined with Western blotting suggest that IHF represses ndh expression during the late-exponential and stationary phases of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>The growth phase-dependent ﬁs expression pattern can be observed in the absence of Fis ( 4 ) or IHF ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>The expression of ihfA and ihfB increases when the cells enter the stationary phase and as a result the concentration of IHF increases tenfold [ 3 , 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>The expression of ihfA-lacZ in the absence of the IHF heterodimers is increased 3 - to 5-fold in the exponential phase and 2.5 - to 3-fold in stationary phase , while the expression of ihfB-IacZ increases about two-fold in both exponential and stationary phases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>We conclude that IHF is needed for full expression of the positively regulated cysJIH promoter during stationary growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>The himA and hip genes are at least partly growth phase regulated in that the transition to stationary phase induces their expression ( 2 ) and IHF is accumulated in growth-arrested cells ( 3 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>The in vivo observations suggest that IHF is necessary for full expression of cysJIH in stationary phase but not during exponential growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>A possible explanation could be the observation that the concentration of IHF in the late stationary phase is increased tenfold [ 9 ] and that high concentrations of IHF are needed to affect the expression of phoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>( c ) IHF affected the expression of pst and phoA only in Pi-starved cells that reached the late stationary phase , when the concentration of IHF in the cell is maximal [ 9 , 10 ] , and in log-cells only when it was overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Thus , IHF does not play a signi ® cant role in stationary phase F ( hmp-lacZ ) expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>Recent experiments indicate that IHF participates in stationary-phase-induced expression of fhe osmY gene and the dsp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>IHF aetivation of dps expression Given the presence of a consensus sequence for the histone-like IHF protein in the dps promoter ( noted by A. Sirko and M. Freundlich ) , we also assayed the effect of himA and himD ( encoding the heterodlmeric IHF protein ) mutations on the expression of Dps in stationary phase cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>For example , the stationary phase expression of cgsA ( encoding the curlin subunit gene ) is modulated by both a ® and HN-S ( Olsen etai , 1993 ) , and osm / ( encoding a periplasmic protein ) is activated by a ^ and repressed by Lrp , cAMP-CRP and IHF in stationary phase cells ( Lange et ai , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>-80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC DISCUSSION osmY ( csi-5 ) is one of many stationary-phase-induced and osmotically regulated genes in E. coli that require us , the gene product of rpoS , for expression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We further show that the transcription of this operon requires the DNA-bending protein integration host factor ( IHF ) - along with KatF , encoded by the rpoS gene and thought be sigma factor to a for growth phase-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8102362" target="_blank">8102362</a>]<br></li>
<li>The previous observation ( Freundlich et al. 1992 ) that IHF mutants show reduced growth on a limiting amounts of inorganic sulfur and the fact that these mutants grow more slowly on djenkolate ( our unpublished results ) , which is a limiting sulfur source , is in agreement with the postulated role for IHF as a positive factor required for full expression of the cysJIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>The expression of the aceBAK operon , which codes for the key enzymes of the glyoxylate cycle , isocitrate lyase and malate synthase , and for isocitrate dehydrogenase kinase/phosphatase , is activated by FruR and IHF and is repressed by IclR , which in turn is activated by FadR ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>The pst operon can be added to the list of genes and operons that are positively regulated by rS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>The physiological role of IHF in E. coli is not known , and its concentration does not appear to vary in response to environmental change ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>The function of IHF in these operons seems to be to enhance closed-complex formation ( Krause and Higgins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Since this gene is part of an operon with hha and ybaJ is the first gene downstream of their common promoter , the microarray data indicate that Hha represses ybaJ as well as its own transcription , a common property among the histone-like regulatory proteins like IHF [ 29 ] , H-NS [ 30 ] , and HU [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>Introduction of a 13C residue in the promoter region of phoA , another PHO regulon gene that is not directly affected by IHF , did not affect its exclusive transcription initiation by ErD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>IHF binds weakly but speciﬁcally to a DNA fragment containing the promoter region of the pst operon but does not bind to a fragment that includes the promoter region of phoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>To determine if this decreased transcription into thegalK gene mightbe due to theinteraction of IHF at the leader-attenuator IHF-binding site , in vivo levels of galK expression from the ilvPc2-attenuat0r : : galK transcriptional fusion strains , whichdo not contain the upstream iluPC1 promoter region IHF-binding site , were compared itnhe presence and in the absence of functional IHF protein expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>In the case of the nonoverlapping promoters of the ilvG-MEDA operon , IHF represses transcription of the upstream ilvpG1 promoter and stimulates transcription of the downstream ilvpG2 promoter ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>However , the IHF-mediated activation of transcription is not dependent upon the helical phasing ( spatial orientation ) of the upstream IHF and downstream promoter sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>Surprisingly , a comparison of the galactokinase levels in strains containing the six base-pair insertion shows that the IHF-mediated activation of ilvGMEDA operon expression is not dependent upon the spatial orientation of the upstream IHF binding site in relation to the downstream iZwPo2 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569580" target="_blank">1569580</a>]<br></li>
<li>Therefore , since the overall effect of IHF in fimA regulation and fimbriae expression is positive , and since both ihfA and fimA are induced upon hha deletion , we hypothesized that another part of the mechanism by which Hha represses biofilm formation is through the diminution of type I fimbriae through the repression of fimA , either directly or indirectly through the repression of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18545668" target="_blank">18545668</a>]<br></li>
<li>In uitro transcriptions were in uiuo and in uitro , IHF decreases transcriptional readperformed at 23 `` C. Transcription reaction time ( seconds ) is shown through at the attenuator of the ilvGMEDA operon approxiat the top of each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Thus , it is clear that functional IHF is needed for efficient transcription of the ecpD-htrE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8102362" target="_blank">8102362</a>]<br></li>
<li>Mutants in htrE have reduced thermotolerance at 50 ° C and IHF along with RpoS were found to act as positive factors of the operon transcription [ 43 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>' It will be interesting to determine the correlation of these changes in IHF levels with the levels of transcription initiation and termination in the promoterattenuator region of the ilvGMEDA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Thus , IHF is capable of regulating the expression of the ilvGMEDA operon in opposing manners ; it can activate transcription initiation of this operon from the ilvP , Z promoter 4-fold and increase the termination of this transcription at the downstream attenuator Z-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>Thus , IHF is capable , at least under certain conditions , of regulating the expression of the ilvGMEDA operon in opposing independent manners ; IHF activates transcription of this operon 4-fold and can increase the termination of this transcription at the downstream attenuator site 2-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>DISCUSSION Transcription of the pspABCE operon in E. coli utilizes s54-RNAP , is facilitated by IHF , and is dependent on an upstream , cis-acting UAS ( 56 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>IHF has also been reported to activate the ace operon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>Transcription of the psp operon is also stimulated by IHF in vivo ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>IHF binding is required for Lrp-dependent activation of the gltBDF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>These data indicated that the presence of IHF is required for the AtoC-mediated induction of the atoDAEB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>IHF is required for optimal induction of the aceBAK operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>IHF is a positive effector of the anaerobically induced biodegradative threonine deaminase ( tdc ) operon ( Wu and Datta , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Because IHF has diverse effects on gene expression , and since these effects are usually not very strong , it will be important to prove by rigorous in vitro and / n vivo analysis that IHF Is a direct effector of a particular operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>We have recently found that IHF is a direct , positive effector for expression of the E. coli ilvBN operon ( Tsui and Freundlich , submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2250661" target="_blank">2250661</a>]<br></li>
<li>We report here that IHF is a direct negative effector of the ompC operon of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>It is also not obvious why IHF is involved in regulation of the omp system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>IHF also has a positive role in the anaerobic induction ofpfi , although this was not as obvious as was its function in pyruvate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>uspB induction is dependent on sS and modulated by IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829921" target="_blank">9829921</a>]<br></li>
<li>It is well known that bound IHF induces a sharp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>whose effect on sodA is enhanced by IHF and Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412671" target="_blank">8412671</a>]<br></li>
<li>Potential repression of ompC by IHF is not included ( 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Effect of IHF on narXL Expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>This suggests that not only are the HimA and HimD proteins recognizing the same sequence as IHF , but also these proteins induce the same amount of bending in the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8156991" target="_blank">8156991</a>]<br></li>
<li>They also participate in diverse roles such as regulation of DNA replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>IHF and Fis proteins are also required for full activation of narK expression , and their roles in DNA bending are discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>H159L Because the expression on linear IC-55 in the absence of IHF was undetectable , the dramatic activation factor of IHF-mediated DNA bending on IC-55 can not be calculated ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>When the resulting strains , S90CPX and DPB101PX ( Table S1 ) , were grown in glycerol-containing minimal medium in the presence of PA , the lack of IHF did not inﬂuence negatively the lacZ expression driven by the Px promoter ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>'' The resulting strains were grown in glycerol-containing minimal medium in the presence of PA , and the IHF effect on the lacZ expression driven by the Pa and Pz promoters was checked by b-galactosidase assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Most of the proteins whose expression levels were altered because of the himA mutation were not induced in the normal wild-type fashion during stasis , indicating that IHF may act as a positive modulator of expression for these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>For example , in the himA , topA strain ilvPG expression is nearly twice as high as it is in a himA strain ( compare strains IH-105 and IH-755 ) ; whereas , it is only slightly higher in a topA strain containing a functional IHF gene than it is in a wild type strain ( compare strains IH-100 and IH750 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>Another example is the NarL / NarP - and FNR-stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>A similar mechanism may account for the selective expression of osmY by E 38 : Its expression by E 70 is efficiently repressed by CRP , IHF , and lrp transcription factors ( Colland et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16204188" target="_blank">16204188</a>]<br></li>
<li>fis expression is also controlled by DNA supercoiling ( Schneider et al. , 2000 ; Walker et al. , 2004 ) , IHF ( Pratt et al. , 1997 ) and CRP ( Nasser et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17768236" target="_blank">17768236</a>]<br></li>
<li>The expression of acs encoding acetyl-coenzyme A synthetase is controlled by the regulator proteins CRP , Fis and IHF ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233826" target="_blank">17233826</a>]<br></li>
<li>Other regulators such as RpoS and IHF , known to aVect hmp expression , apparently are not involved in the hmpEc induction by NO ( Memb-rillo-Hernández et al. 1996 , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>For example , gene expression of sufABCDSE is induced by H2O2 ( Zheng et al. 2001 ) , and oxidant induction of the suf operon is regulated by OxyR , IHF , Fur and IscR ( Lee et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>Genetic regulation chiC locus narGHIJ operon hark gene Like the narGHJl operon , expression of the narK gene is positively regulated by Fnr , NarL and IHF ( Stewart & Parales 1988 ; Kolesnikow et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7747940" target="_blank">7747940</a>]<br></li>
<li>A role for IHF or H-NS in regulating ftnA expression was tested by Western blot analysis of the corresponding himA ( one of two unlinked genes encoding the heterodimeric IHF protein ) and hns mutants , and parental strains ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>]<br></li>
<li>Evidence that IHF plays a role in the expression of the carAB operon of both E. coli and S. typhimurium was obtained by assaying CPSase ( chromosome encoded ) and [ 3-galactosidase ( plasmid encoded ) in different de-rivatives of a Car + Lac - E. coli strain harbouring a single-copy plasmid carrying a fusion between the car control region of S. typhimurium and the lacZ gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8455562" target="_blank">8455562</a>]<br></li>
<li>DISCUSSION We have established that IHF is essentifaolr the anaerobic expression of the nitrate reductase ( narGHJZ ) operon of E. coli by the Fnr and NarL activator proteA inpsa.rtial deletion of the h i d gene , which encodes the a subunit of the heter-odimeric IHF protein ( 30 ) d , ecreased narG-lac2 expression in cells grown anaerobically , and it alssoeverely impaired nitrate induction of narG expression by NarL ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Overall expression of the narXL genes was only slightly reduced in the absence of IHF ( Table 3 ) , suggesting that limitation for NarL and/or NarX is not responsible for the severe deficiency in nar operon induction by nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>Results in Fig. 4 show that , in the Dlac narL narP ihfA strain , JCB3884 , pnir expression 1 is increased by both the p99G and the p146A substitutions , which disrupt the binding of IHF and Fis respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Furthermore , the IHF-dependent increase in fnr expression was not observed in the construct containing the mutant upstream site ( Fig. 6 ) , suggesting that IHF binds to the upstream element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>Repression of ndh expression by IHF Strains containing a Andh-ZacZ fusion prophage were the anaerobically grown ' ihfA fnr double mutant ( Fig. 4b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Thus , the ability of IHF to stimulate transcription is dependent on its position relative to ﬁs P. ﬁs expression pattern in the absence of ihf and Fis sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335285" target="_blank">9335285</a>]<br></li>
<li>Effect of multicopy fimB in strains deficient in IHF Loss of IHF expression has been shown previously ta : reduce fimA-lacZ transcription by a factor of seven ( Dorman and Higgins , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8898385" target="_blank">8898385</a>]<br></li>
<li>It is interesting that of the 10 operons in E. coli whose expression is directly altered by IHF , 4 are involved in anaerobic metabolism ( 10 , 32 , 36 , 39 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Concluding remarks Although the published number of operons in E. coli whose expression is influenced by IHF is small , the information presented in this review suggests a much wider role for IHF in gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>This becomes a particularly intriguing question when it is considered that no physiological regulatory role for IHF has been discerned for any of the many operons whose expression it is known to affect ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>This protein , which is essential for normal expression of narKlacZ in vivo , apparently binds to narK regulatory DNA site ( s ) and may bend it as has been demonstrated for IHF regulation of other genes in E. coli ( 32 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>IHF also differs from typical E. coli regulatory factors by its participation in processes other than gene expression , such as site-specific recombination , transposition , and DNA replication ( Friedman , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>As discussed above , IHF has few properties of a typical regulatory protein and it is not known why IHF participates in gene expression in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>It has to be underscored that in this system the IHF effect is not mediated by a repression in the expression of the regulator gene glcC ( see Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>It is interesting that an artificial increase of oniy one subunit of IHF can induce strong repression of IHF gene expression ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>In a number of cases in which IHF positively regulates gene expression , IHF does not activate transcription directly but enhances the effectiveness of an activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Our results do not exclude the possibility that IHF may also have an indirect e $ ect on cysJIH transcription , for example , by altering the expression of the cysB gene encoding the regulatory protein CysB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>However , it is not a general effect on any ostnotically regulated gene , as IHF does not affect the expression ol the proU operotT ( Lucht and Bremer , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565114" target="_blank">7565114</a>]<br></li>
<li>The effect of IHF on gene expression can be either positive or negative ( Friedman 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7862094" target="_blank">7862094</a>]<br></li>
<li>IHF can alter gene expression in a number of ways including positive and negative effects on transcription ( reviewed in Friedman 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>IHF also has a direct negative effect on ompF and ompC ( 26 , 45 , 46 ) , and it is likely that IHF regulation of ompB contributes to the overall expression of these operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Besides requiring sS and MlrA , csgD expression is also modulated by IHF and the two-component response regulators OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>In this same report , FNR and IHF were reported to have roles as weak repressors under anaerobic conditions , and it was suggested that the effect of FNR may be indirect as a consequence of its role in regulating ArcA expression ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Under inducing conditions , IHF activates aceB : : lacZ expression by opposing IclR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>In contrast , IHF has little effect on aceB : : lacZ expression under repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>The observation that the positive effect of IHF on the expression of pst is coupled with a decreased expression of phoA demonstrates again that the Pst system regulates the PHO genes , but it does not disclose any clue how it does so .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>The effects of narL , narP , fnr , and himD ( which encodes one of the subunits of the IHF protein ) null alleles on expression of the nar regulatory genes were also determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>The IHF protein is also involved in regulating sucABCD gene expression ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209026" target="_blank">9209026</a>]<br></li>
<li>The data in Fig. 4 demonstrate that all three promoters were still induced by anaerobiosis and that the IHF III site activated anaerobic expression in the pnrf-nir fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>DISCUSSION The data in this report clearly show that IHF is a negative effector of ompC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>In addition to this growth rate control , both promoters were negatively regulated by IHE This was shown by the increase of [ 5-galactosidase activities in ihfA or ihfB mutant strains and by reduced expression of each promoter under conditions of excess levels of IHF ( figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Consistent with this data and the increased sensitivity of the pspG promoter to PspF in comparison to the pspA promoter ( Fig. 6B ) , PspG expression in an IHF mutant ( MC1068 ) is increased both before and after induction by pIV in vivo ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>These data suggest that IHF exerts negative control of ompC expression in two ways-directly , by binding upstream from the ompC promoter region , and indirectly , by influencing the expression of OmpR and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Thus , under some anaerobic growth conditions Arr may compete with FNR and IHF for occupation of the ndh promoter and , by acting positively , Arr may counteract the negative effects of FNR and IHF to ` finetune ' ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Differences in the upstream IHF and Fis binding sites at the nir promoter in related enteric bacteria ﬁx the level of nir operon expression under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The results of this experiment ( Table 1 ) showed that the growth rate-related expression of the ilvGMEDA operon is not mediated by either IHF or ilvpG promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068661" target="_blank">9068661</a>]<br></li>
<li>Since sdhC-lacZ expression was not signiﬁcantly altered in a himA mutant ( 19 ) , this suggests that the IHF control is exerted primarily from the sucA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9209026" target="_blank">9209026</a>]<br></li>
<li>Thus , IHF plays a dual role in controlling fimA expression : it is required both for inversion of the fimA control region and for efficient expression from the fimA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>IHF factor partially relieved the repressor effect in the expression of ulaG or ulaA promoter fusions when cells grew in the absence of L-ascorbate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>FNR-dependent expression from the nrf promoter is downregulated in vivo by the Fis and IHF proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>It is clear from the results presented in this paper that expression from the fimA promoter is reduced by about sevenfold in the absence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>Thus , it is unclear how IHF increases the expression of the fnr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>IHF represses ndh expression by binding to three sites in the ndh promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>This pattern of expression results from positive regulation of transcription from the narGHJI operon by three transacting factors , FNR , NARL , and integration host factor ( IHF ) ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682807" target="_blank">8682807</a>]<br></li>
<li>Our results suggest that CAP and IHF act in concert as positive transcription factors for Wc operon expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Because the integration host factor protein ( IHF ) was demonstrated to be essential for nitrate-dependent anaerobic expression of the nitrate reductase ( narGHJI ) operon ( 34 ) , we also tested its role in nitrate-dependent narK-lacZ expression ( Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1429433" target="_blank">1429433</a>]<br></li>
<li>Recently , we reported ( 33 ) that expression of 3-galactosi-dase from a tdcB ' - ` lacZ fusion plasmid was drastically reduced in an integration host factor ( IHF ) - deficient strain and that purified IHF bound to a region around position -100 on the tdc promoter for positive activation of the tdc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>Since it is difficult to assay the products of the Pst transport system , we tested the effect of IHF on the expression of pst by in vivo constructions of chromosomal operon fusions between the reporter gene lacZ ( encoding bGal ) and some of the pst structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>However , the cryptic bgf operon , whose expression was reported to be regulated through topological changes of cellular DNA , could not be activated in cells depleted of HU or IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1452042" target="_blank">1452042</a>]<br></li>
<li>The results in Table 2 show that this fusion ( pPTS1 ) has low levels of fl-galactosidase activity and that operon expression is no longer IHF dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>' The results of the in uitro transcriptional analyses presented in this report demonstrate that IHF , when bound simultaneously at both of the target sites , regulates the expression of this operon by independent interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1703160" target="_blank">1703160</a>]<br></li>
<li>A model whereby three transcriptional activators , NarL , IHF , and Fnr , induce expression of a a70-dependent promoter for the narGHJI operon is discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>16 , 1 ¬ ompB asmoiaiity ) OmpR IHF IHF '60 -35 -10 • 170 QmpR CmpROmpR IHF IHF -360 -60 -170 The architectural roie of IHF in gene expression is not only limited to o - ̂ ** promoters but is probably also involved in the activation of the c ' ^ ^ - dependent promoter of the narGHJI operon ( Schroder et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Involvement of IHF in the expression of the glc operon was tested by assaying b-galactosidase in strain JA162 himA : : cat F ( glcD-lacZ ) lacking one of the IHF subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9880556" target="_blank">9880556</a>]<br></li>
<li>For example , IHF increases the expression of the ilvGMEDA operon of E. coli , which encodes four of the five enzymes required for the biosynthesis of L-isoleucine and L-valine ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8577735" target="_blank">8577735</a>]<br></li>
<li>IHF is also a transcriptional regulator of the ompB operon whose protein products , EnvZ and OmpR , are required for expression and osmoregulation of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF was also required for Fnr-independent anaerobic control of narGHJI expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>Using himA mutants , we demonstrate that IHF is also required for efﬁcient anaerobic expression of the dmsABC operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Nitrate induction of F ( narG-lacZ ) expression is strongly dependent on IHF , which binds to the narG operon control region ( 21 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>It is also possible that , earlier in bacterial evolution , IHF was involved in ast operon expression and that ArgR preempted this function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>ihe IHF protein itself which tunes the expression of an operon , but the regulator protein that is supported in its function by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>It is proposed that IHF is a positive regulator of the pst operon and as such controls indirectly the expression of phoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Effects of IHF and IclR on expression of aceB : : lacZ operon fusions FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>This and other observations indicate that IHF is directly involved in nar operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1528882" target="_blank">1528882</a>]<br></li>
<li>During osmotic shock in contrast , psp expression is only weakly stimulated by IHF , and IHF mutants can strongly induce the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>f. Reduced ( + ) or increased ( - ) operon expression in IHF mutants ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF-binding sites and operon expression in E. coli Early work established that , in addition to its role in sitespecific recombination in X , IHF was also involved in influencing gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>Finally , although bending of promoter DNA by IHF is generally suggested to be critical to its role in gene expression , this event has as yet not been shown to be essential for the activity of IHF in any operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>We have shown recently ( 33 ) that this IHF-binding site and a functional IHF protein are essential for tdc operon expression ; lack of either one reduces tdc expression by about 90 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>In ompF , the two IHF binding sites flank the OmpR sites that are necessary for operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Since we observed putative IHF-binding sites in the promoter region of phoA and of the pst operon , we examined the effect of IHF on the expression of both these members of the PHO regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>In such a case , employing IHF - and/or SeqA-de-void host can bring the expression to the level that is not harmful for the host cell metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>These effects were both positive and negative , results that are consistent with the positive and negative influence of IHF on the expression of E. coli operons ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>These results were consistent and show that the presence of IHF improves the expression of pstS , as judged by the bGal activity of the fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>These results demonstrate that rS-mediated regulation of F ( hmp-lacZ ) expression is not dependent on IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>These data and results with ilvGMEDA-lac promoter fusions suggest a direct role for IHF in expression of the ilvGMEDA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3145340" target="_blank">3145340</a>]<br></li>
<li>These results suggest that IHF does not positively contribute to ast operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>These data strongly suggest that IHF is required to regulate expression from dmsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>These results suggest that IHF activates aceBAK expression by binding to IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>These observations are consistent with IHF acting as both an aerobic and anaerobic repressor of ndh expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>However , both are regulated by IHF , which acts as an anaerobic repressor of both nuoA-N and ndh to provide a degree of co-ordinated expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>A role for IHF in tuning expression levels of uspA has been conﬁrmed by using a uspA-lacZ fusion ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>To examine the role of IHF in fimA expression , the himA82 : : TnlO insertion was introduced by Pl-mediated transduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2886490" target="_blank">2886490</a>]<br></li>
<li>Moreover , speciﬁc DNA topology affected by IHF and SeqA can inﬂuence the expression of recombinant genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18590762" target="_blank">18590762</a>]<br></li>
<li>The H-NS and IHF histone-like factors also affected sodB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>Regulation of sodB expression by the histone-like proteins IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>Therefore , the mechanism of IHF action and its effeot on gene expression are probably quite complex and variable among operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF as a negative regulator of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Therefore , IHF serves a nonessential role in modulating napF gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Regulation of expression of the ulaG and ulaA divergent promoters by CRP , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996803" target="_blank">14996803</a>]<br></li>
<li>IHF slightly repressed sodB expression independently of Fur regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850997" target="_blank">10850997</a>]<br></li>
<li>This result suggests that IHF activates aceBAK expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626344" target="_blank">8626344</a>]<br></li>
<li>Interestingly , IHF - strains are constitutive for ompF expression ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>It is not clear whether IHF regulates a l d A expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9202484" target="_blank">9202484</a>]<br></li>
<li>IHF in ompB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Role of IHF for expression of glmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In spite of the presence of multiple IHF binding sites in ihfB its auto-regulation , under the conditions examined , is less effective than the autoregulation of ihfA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Upstream of the start site of the yiaK-S operon the 210 and 235 consensus , high AT content region , IHF site , and Crp-binding sites are located by the corresponding boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
<li>IHF enhances psp operon transcription ( 20 , 46 ) and facilitates binding of PspF to its upstream activation sequences in the psp operon regulatory region , autogenously down-regulat-ing pspF transcription ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>In favor of direct IHF involvement in the transcription of the ackA pta operon is the presence of a potential IHF binding site in the regulatory region of these genes ( our unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li>
<li>Expression of iclR may be relatively insensitive to the carbon source because it does not have a binding site of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550439" target="_blank">8550439</a>]<br></li>
<li>In contrast to t , he glnH regulatory region , there is no IHF-binding site upstream from gZnAp2 and IHF does not stimulat , e t , ranscription init , iation ( Claverie-RZa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>In the first model , the bending induced by IHF brings sequences located further upstream from the ihf site into contact with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>One interpretation of this result is that IHF facilitates the formation of a DNA loop between the OmpR-binding site located at - 384 and -351 and the OmpR-binding site located at -56 and -41 and that this DNA loop is responsible for repressing ompF expression under conditions of high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>In anaerobic conditions , FNR binds to its target site , but transcription activation is repressed by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Although our studies indicate that the predicted upstream site is not an FNR binding site , we found that this sequence binds IHF and increases expression of fnr under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293415" target="_blank">17293415</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Action at a distance by IHF has also been suggested for the positive role of this protein in ilvBN transcription in E. coli ( P. Tsui and M. Freundlich , unpublished data ) , DNA replication in pSC101 ( 40 ) , and Mu transposition ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Interestingly , for several other IHF-regulated genes in E. coli and related bacteria , the IHF sities located about 100bp upstream of the transcription start sites ( 15 , 33 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8419352" target="_blank">8419352</a>]<br></li>
<li>These results indicate that repression by nitrate is independent of the three transcription factors ArcA , IHF , and FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Model illustrating the roles of the TCS GlrR/GlrK , s54 and IHF for transcription of sRNA genes glmY and glmZ in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>31 ) , it is clear that IHF does not activate transcription simply by increasing DNA supercoiling or by binding better to supercoiled DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8702758" target="_blank">8702758</a>]<br></li>
<li>If IHF can activate transcription by bending the DNA it is quite likely that DNA-bending by CRP has the same effect , namely transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8895588" target="_blank">8895588</a>]<br></li>
<li>pspG transcription requires 54-RNA polymerase , PspF , and ATP and is facilitated by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>The transcription of pspG in vitro is enhanced by IHF in a concentration-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15485810" target="_blank">15485810</a>]<br></li>
<li>IHF did not stimulate transcription and appeared to inhibit it when ArgR was also present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>A. Schematic showing acs transcription ( acs , TRXN ) , growth ( OD ) and the relative concentrations of active CRP , FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>Using in vitro transcription assays , we demonstrated that FIS and IHF independently reduce CRP-dependent acs transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14651625" target="_blank">14651625</a>]<br></li>
<li>It appears likely that IHF contributes to reduced acs transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Independent regulation of acs transcription by FIS and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>IHF is a sequence-speciﬁc , asymmetric , histone-like , heterodimeric protein ( encoded by himA and himD ) that plays important roles in replication , site-speciﬁc recombination , and transcription ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955388" target="_blank">8955388</a>]<br></li>
<li>IHF is a sequence-specific DNA-binding and - bending protein encoded by the himA and himD ( hip ) genes of E. coli ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>Activation of folA also requires IHF , which is not required for the simpler activation of genes such as mtr ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17557822" target="_blank">17557822</a>]<br></li>
<li>Although these mutations are not yet mapped , they do not appear to lie in the IHF genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596833" target="_blank">7596833</a>]<br></li>
<li>In vivo experiments indicated that the himA and himD genes were negatively autoregulated by IHF ( Miller etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>IHF can thus play an analogous role to that of CRP in facilitating NagC-dependent DNA loop formation at the divergent nagE-BACD operons ( Plumbridge and Kolb , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Recently , Bock and his associates reported that the fdhF and hyc operons can be transcribed and translated in an in vitro system with FHLA , IHF , and formate ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665461" target="_blank">7665461</a>]<br></li>
<li>Both operons are similarly controlled by the factors IHF and ArcA-P and can fulfil redundant roles via cross-induction [ 13 ] to avoid the toxic accumulation of glyoxylate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22168595" target="_blank">22168595</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>