<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - IscR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">IscR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">IscR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>200660410581075Original ArticleIscR regulates Fe-S and oxygen-regulated genesJ.L.Giel etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Copyright 2010 by the Genetics Society of America DOI : 10.1534 / genetics .110.114462 IscR Regulates RNase LS Activity by Repressing rnlA Transcription Yuichi Otsuka , * Kumiko Miki , * Mitsunori Koga , * Natsu Katayama , * Wakako Morimoto , * Yasuhiro Takahashi † and Tetsuro Yonesaki * ,1 * Department of Biological Sciences , Graduate School of Science , Osaka University , Osaka 560-0043 , Japan and † Division of Life Science , Graduate School of Science and Engineering , Saitama University , Saitama 338-8570 , Japan Manuscript received January 19 , 2010 Accepted for publication April 23 , 2010 ABSTRACT The Escherichia coli endoribonuclease LS was originally identiﬁed as a potential antagonist of bacteriophage T4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>This core network was extended to include local regulators ( CpxR , CusR , FhlA , Fur , IscR , ModE , NarL , NrdR and PdhR ) that exhibited responses in the stable environments o steady-state cultures across the aerobiosis scale and in the unstable environments of the transitions ( figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Three reporter strains , one lacking IscR , another lacking OxyR and their isogenic wild-type strain , were assayed after 60 min of wrwycr treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>This work was supported by the NIH Grant GM45844 to P.J.K A = Afree + ððAbound AfreeÞ 1/2 IscR nÞ = Knd + 1/2 IscR n ; ð1Þ Supplementary data associated with this article can be found , in the online version , at doi :10.1016 / j.jmb .2009.01.055 References 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In summary , this study extends the role of IscR in the regulation of Fe-S cluster biogenesis in E. coli to include the suf operon as well as the genes encoding the putative Fe-S cluster assembly proteins , YadR and YhgI , and suggests a similar function of IscR in other related bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_8">
IscR acts as an activator in response to oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></label>
<input id="act_similarity_8", type="checkbox">
<div class="ACT_similar">
<li>IscR acts as an activator in response to oxidative-stress for the suf operon encoding Fe -- S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>( 18 ) Yeo , W. S. ; Lee , J. H. ; Lee , K. C. ; Roe , J. H. IscR acts as an activator in response to oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Yeo , W.S. , Lee , J.H. , and Roe , J.H. ( 2006 ) IscR acts as an activator in response to oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_12">
Oxidant-responsive induction of the suf operon , encoding a FeS assembly system , through Fur and IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></label>
<input id="act_similarity_12", type="checkbox">
<div class="ACT_similar">
<li>Lee , K.C. , Yeo , W.S. , and Roe , J.H. ( 2008 ) Oxidant-responsive induction of the suf operon , encoding a Fe-S assembly system , through Fur and IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_14">
Oxidant-responsive induction of the suf operon , encoding a Fe-S assembly system , through Fur and IscR in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></label>
<input id="act_similarity_14", type="checkbox">
<div class="ACT_similar">
<li>Oxidant-responsive induction of the suf operon , encoding a Fe-S assembly system , through Fur and IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li></div>
<li>OxyR and IscR contributed almost equally to activate the sufA operon in response to oxidants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>This demonstrates that IscR positively contributes to induce the sufA gene as much as OxyR does in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Thus whereas isc induction reﬂects disruption of overall Fe -- S cluster status and the inability of IscS , IscU and IscA to maintain the cluster of IscR , suf induction is a speciﬁc response to oxidative-stress and iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>This result indicates that the IscR repressor accounts for the transcriptional induction of the isc promoter in response to oxidative-stress and iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Regulation of the isc operon is mediated by IscR , whereas the suf operon requires OxyR and IHF for the response to oxidative-stress and Fur for induction by iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>iscR is required for isc induction in response to H2O2 and 2,2 cents - dipyridyl To ascertain if the induction of the isc operon in response to oxidative-stress and iron starvation is mediated through the IscR repressor , we examined isc induction in response to H2O2 and 2,2 cents - dipyridyl in a mutant strain with a nonpolar deletion of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Contrary to wild-type cells , where IscR becomes active after 15 -- 20 min of dip addition , the three other backgrounds ( DryhB , rne131 , and hfq ) still express the isc operon as IscR lacks sufﬁcient iron to act as a repressor ( Apo-IscR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Finally , two genes of unknown function , ydiU and yjcB , also appeared to be activated by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The yfhJ-pepB-sseB genes , located directly downstream of the IscR-repressed isc-SUA-hscBA-fdx operon , also appeared to be repressed by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The only known regulator of isc is IscR , a transcriptional-repressor co-expressed with the other isc genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>The goal of this study was to investigate whether IscR regulates genes in addition to the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In this study , we investigated whether IscR regulates the expression of genes in addition to the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Even though its underlying mechanism is not clearly understood , oxi-dative stress could cause inactivation of IscR as a repressor , leading to derepression of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Provision of IscR from the multicopy plasmid lowered the basal level of the iscR Positive regulation of the suf operon by IscR 211 transcripts most likely due to increased amount of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>It is surprising , however , that suf is activated by IscR , unlike the isc operon , which is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>This suggests that IscR remains under the Apo form and does not repress the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscR contributes to the activation of the suf operon , but only under the Apo-IscR form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>The induction of suf operon promoter activity by wrwycr treatment agrees with this idea , and implicates IscR ( and not OxyR ) as being necessary for this induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>Peptide wrwycr treatment activates the Suf operon through IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>( b ) Suf operon activation by wrwycr is dependent upon the IscR activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>However , it is unlikely that IscR binds to ORE-III , considering that there is no similar sequence between the upstream region of the isc and suf operons , and the factor that binds to ORE-III under oxidative conditions is most likely to be an activator rather than a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>These results support the proposal that IscR positively acts to induce sufA expression in response to PMS via ORE sites , most likely ORE-III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>These results support the proposal that IscR binds directly to the ORE-III region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>These results support a model that IscR in its demetallated form directly activates sufA transcription , while it de-represses isc operon , under oxidative-stress condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The other part of oxidant induction is mediated by IscR , as demonstrated in this study , which binds to ORE-III site ( -56 to -35 ) and directly activates sufA transcription when cells are under oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Fur and IscR regulate nrdHIEF transcription We sought to identify the regulatory mechanism that triggers expression of nrdHIEF during oxidative H2O2 stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Thus these three regulatory proteins act independently of one another : NrdR : dATP and Fur : Fe2 + repress nrdHIEF transcription , while apo-IscR activates it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>While isc and suf are both direct targets of the IscR regulon and are expressed in response to similar signals ( 11 -- 15 ) , IscR controls expression of these two operons by different mechanisms : IscR represses isc expression , whereas IscR activates transcription of suf ( 14 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Finally , IscR decreased the expression of dctA , encoding a C4-dicarboxylate transporter ; soxS , encoding a transcription factor that activates genes in response to superoxide stress ; ﬁmAICDFGH and ﬂu , two operons encoding surface structures ; and several operons with genes of unknown function : ycgZ , the putative yjiH-yjiG-iadA-yjiE operon , yjiM and yqjI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR decreased expression of soxS , encoding a general regulator of oxidative-stress , but increased expression of a superoxide dismutase The levels of transcription of two genes involved in the response to oxidative-stress also were affected by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Lanes 1 and 4 lack IscR or ArcA ; lanes 2 and 3 contain 800 nM and 400 nM phosphorylated ArcA , respectively ; lane 5 contains 800 nM as-puriﬁed IscR ; lane 6 is a Maxam and Gilbert A G ladder made from the same DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Lanes 1 and 2 contain 800 nM and 400 nM phosphorylated ArcA , respectively ; lane 3 contains 200 nM as-puriﬁed IscR and 800 nM phosphorylated ArcA ; lane 4 contains 200 nM as-puriﬁed IscR ; lane 5 contains no protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>His-tagged IscR proteins bound to Ni-NTA were eluted by step gradient using 10 volumes of elution buffer containing 100 , 200 and 300 mM imidazole .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>For wild-type IscR , fraction bound was determined using the equation : Fraction Bound = ðA AfreeÞ = ððAbound AÞ4q + A AfreeÞ ; ð2Þ where A is the anisotropy of the sample , Afree is the anisotropy with no IscR , Abound is the anisotropy with 1600 nM IscR , and q is the quenching factor , which was between 0.87 and 0.91 for wild-type IscR .41 q was calculated by dividing the total fluorescence of the 1600-nM IscR sample by the total fluorescence of the 0-nM IscR sample .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Fraction bound ( y-axis ) was calculated from the anisotropy and is plotted as a function of [ IscR monomers ] divided by [ 250 nM DNA ] ( x-axis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The fraction of fluorescent probe bound in the presence of competitors was calculated as above except that Abound is the anisotropy at 55 nM IscR in the absence of competitors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Because deprivation of intracellular iron is known to induce the expression of IscR , we measured the burst size of a T4 dmd mutant on MH1 cells in the presence of 0.25 mm 2 , 29-dipyridyl , a membrane-permeant ferrous iron chelator that causes deprivation of intracellular iron ( supporting information , Table S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>It did not respond to manganese availability ; it was not induced by redox-cycling drugs , which indicates that it is not regulated by SoxR ( 16 , 47 ) ; it was not affected by Fur , IscR , Hfq , RpoS , or YaaA itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Thus , the ability of a common regulator , IscR , to coordinate transcription of both the Isc and Suf pathways by derepression and activation , respectively , is a vital homeostatic mechanism that ensures cell survival under a variety of growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>In summary , our data provide additional insight into the mechanisms by which IscR coordinates regulation of the Isc and Suf pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>IscR regulates the Suf pathway in addition to the Isc pathway Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>R and Y are puRine and pYrimidine , respectively ; S is G or C ; W is A or T ; K is T or G ; and M is A or C. Discussion A broader role for IscR in regulating Fe-S biogenesis Our ﬁnding that IscR , in addition to negatively regulating the Isc Fe-S biogenesis pathway , also positively regulates an alternative Fe-S biogenesis pathway , Suf , suggests that IscR has a more general role in regulation of this key metabolic process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Keywords IscR ; Fe-S biogenesis ; Isc ; Suf ; autoregulation INTRODUCTION Iron-sulfur ( Fe-S ) proteins are nearly ubiquitous across all domains of life and carry out many important cellular functions such as respiration , photosynthesis , nitrogen fixation , and gene regulation ( Beinert , 2000 , Fontecave , 2006 , Kiley and Beinert , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_59">
Role of IscR in regulation of iron-sulfur biogenesis in Esch-erichia coli : identiﬁcation of the IscR regulon and mechanisms of autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></label>
<input id="act_similarity_59", type="checkbox">
<div class="ACT_similar">
<li>Role of IscR in regulation of iron-sulfur biogenesis in Escherichia coli : Identification of the IscR regulon and mechanisms of autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li></div>
<li>These results suggest that although RyhB downregulates IscS , the IscR transcriptional-regulator remains expressed , however , under the Apo-IscR form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>In addition to the fact that iron is limited when RyhB is expressed , our results suggest that the downregulation of iscS , iscU , and iscA by RyhB increases the ratio of Apo-IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>These results show a fourfold increase in the IscR/IscS ratio after 4 h 10 8 6 4 2 0 RyhB -- + -- + Wild-type ΔiscR iscR ′ - lacZ Figure 7 Effect of RyhB on IscS and IscR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>We believe that in the absence of functional RyhB , the intracellular iron will be sequestered by non-essential proteins , which results in iron shortage for essential iron-dependent proteins , such as IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>NO is a damaging agent of iron-sulfur clusters , and apart from the wide variety of iron-sulfur-containing proteins involved in relevant pathways , the formation of these clusters is essential at least for the transcriptionally active form of regulators such as FNR , IscR , and SoxR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>This suggests that iron -- sulfur clusters are being damaged and are unavailable to-bind to apo-IscR following wrwycr treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>When iron-sulfur synthesis is impeded -- the situation that is sensed by IscR -- YfaE may become unavailable to activate NrdAB , and so a shift to NrdEF could be salutary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>We also explore the sequence specificity of IscR binding to the hya IscR-binding site as a model for describing IscR interactions with Type 2 sequences since IscR shows the strongest interaction with the hya site in footprinting assays .2 To investigate regulation by IscR in vivo , we analyzed promoter fusions to lacZ along with IscR mutants lacking the ligand residues for the [ 2Fe -- 2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In summary , our data showed that IscR does not require its cluster for regulation of Type 2 promoters , and future work will focus on the role of the [ 2Fe -- 2S ] cluster for direct regulation of Type 1 promoters or indirect regulation of the Type 2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In this study , we investigated the requirement for the ligation of the [ 2Fe -- 2S ] cluster of IscR to regulate a subset of IscR target promoters ( PhyaA , PydiU , PnapF , and PhybO ) and defined the requirements for sequence-specific binding to the IscR target site in the hyaA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Results The [ 2Fe -- 2S ] cluster of IscR is dispensable for regulation of promoters containing Type 2 sites Previous studies7 showed that IscR does not require its cluster to activate PsufA , which has a Type 2 IscR-binding site similar to that of PydiU and P 2 hyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Because SoxR binds independently of its [ 2Fe -- 2S ] cluster , it also represses transcription of its own promoter independently of its [ 2Fe -- 2S ] cluster .32 However , SoxR can only activate transcription from the divergent PsoxS when its [ 2Fe -- 2S ] cluster is present and oxidized .9 Although SoxR and IscR may be similar in their ability to function as trans-cription factors in the absence of their Fe -- S clusters , SoxR only binds a single DNA site , whereas IscR has multiple targets containing either the Type 1 site or the Type 2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>While PiscR contains two Type 1 sites and requires [ 2Fe -- 2S ] - IscR for its repression , PsufA contains a Type 2 site and does not require the Fe -- S cluster for its activation .1,7 This suggests that the type of IscR site specifies whether apo - and/or [ 2Fe -- 2S ] - IscR binds to the site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>This property of IscR was surprising since other known Fe -- S transcription factors ( e.g. , FNR , SoxR ) have only one type of site that either does ( FNR ) or does not ( SoxR ) require the Fe -- S cluster for DNA binding .8,9 The proposed Fe -- S ligands for IscR ( Cys92 , Cys98 , and Cys104 ) are distal to the predicted DNA-binding site ( Fig. 1 ) , and upon anaerobic isolation , the cluster is partially reduced ( [ 2Fe -- 2S ] 1 + - IscR ) but is readily oxidized in air to [ 2Fe -- 2S ] 2 + - IscR .1 Because IscR appears to require its Fe -- S cluster for regulation of PiscR and not for PsufA , this raises the question of whether IscR requires its [ 2Fe -- 2S ] cluster for regulation of other Type 2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The differential repressor activities of [ 2Fe-2S ] - and apo-IscR are reminiscent of [ 2Fe-2S ] - SoxR , which binds DNA in both an active and inactive state but regulates transcription only in the active state ( 12 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>This observation could not be explained by differences in DNA binding afﬁnities or IscR levels , so we conclude that [ 2Fe-2S ] - IscR is a weaker repressor of PhyaA than clusterless IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Since in vitro DNA binding experiments have shown that [ 2Fe-2S ] - IscR and apo-IscR bind the hyaA DNA site with the same afﬁnities ( 16 , 28 ) , our data imply that [ 2Fe-2S ] - IscR is a weaker repressor than the apo form at PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Characterization of the oligomeric state of IscR showed that both apo-IscR and [ 2Fe -- 2S ] - IscR were dimers in solution , and four protomers of either form bound to the hya site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Also , binding of either apo - or [ 2Fe -- 2S ] - IscR to the hya site showed cooperativity , suggesting that both forms interact similarly with the target site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In vitro assays showed that both [ 2Fe -- 2S ] - IscR and an IscR mutant lacking the cluster ( IscR-C92A / C98A/C104A ) bound the hya site with similar affinity , explaining why the mutant protein retained its ability to repress PhyaA in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In addition , both [ 2Fe -- 2S ] - IscR and apo-IscR bound the hya site with similar affinity in vitro ( Fig. 3 ) , suggesting that both forms of IscR can regulate PhyaA and other Type 2 promoters in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The requirement for the [ 2Fe-2S ] form of IscR for repression also correlated well with the increased affinity of [ 2Fe-2S ] - IscR for either site A or B. Nevertheless , the in vivo concentration of the clusterless mutant IscR-C92A / C98A / C104A is sufficiently high ( ≥ 17 µM ; Table 2 ) such that some binding of the mutant might be expected even though its binding characteristics in vitro were poor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>[ 2Fe -- 2S ] - IscR and IscR-C92A / C98A/C104A show no significant differences in binding affinity to the hya site To address whether there is a preference for which form of IscR binds Type 2 sites , we compared DNA binding by IscR-C92A / C98A/C104A and the aspurified wild-type protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>[ 2Fe-2S ] - IscR binds with greater affinity to the PiscR sites A and B in vitro than does apo-IscR Previous in vitro studies demonstrated that [ 2Fe-2S ] - IscR and IscR-C92A / C98A/C104A bind with equal affinity to PhyaA ( Nesbit et al. , 2009 ) , which contains a Type 2 IscR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>C ) Fraction bound corrected for fluorescence quenching of labeled PiscR sites A or B bound by [ 2Fe-2S ] - IscR as a function of protein concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In To double-check this result , we indirectly monitored the presence of [ 2Fe-2S ] clusters in the IscR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Thus , a general feature of Type 2 IscR-binding sites is the ability of either apo - or [ 2Fe -- 2S ] - IscR to-bind and regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Discussion Characterization of IscR has expanded from its function in feedback regulation of its own promoter1 to that of a global regulator with two different binding sites2 and now to the surprising finding that both [ 2Fe -- 2S ] - and apo-IscR can bind Type 2 sites in a sequence-specific manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>[ PubMed : 11443091 ] In vitro binding of anaerobically purified [ 2Fe-2S ] - IscR to wild-type and mutated IscR binding sites within PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>However , the above in vivo data raise the question as to whether [ 2Fe-2S ] - IscR binds with higher affinity than apo-IscR to the Type 1 sites within PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Furthermore , the observed difference in [ 2Fe-2S ] - IscR and clusterless IscR binding affinity for the two PiscR Type 1 sites provides an explanation as to why the [ 2Fe-2S ] cluster is required for IscR to repress PiscR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Our data indicate that a negative feedback loop operates to repress transcription of the iscRSUA-hscBA-fdx operon , encoding IscR and the Isc machinery , through binding of [ 2Fe-2S ] - IscR to two upstream binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Furthermore , these in vivo ﬁndings are consistent with previous in vitro experiments demonstrating that [ 2Fe-2S ] - IscR and IscR-C92A both activate sufA transcription in vitro ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>These genes are strongly induced when iron-sulfur synthesis is hindered , due to conversion of the IscR [ 2Fe-2S ] transcription factor to its apoprotein form [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>Since apo-IscR is sufficient to activate PsufA transcription ( Nesbit et al. , 2009 , Yeo et al. , 2006 ) , we propose that in the ΔiscSUA-hscBA-fdx strain , upregulation of the Suf pathway by apo-IscR may explain the residual [ 2Fe-2S ] - IscR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>When the Isc pathway was absent , however , a small amount of [ 2Fe-2S ] - IscR activity remained , suggesting that IscR could acquire [ 2Fe-2S ] clusters from another source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Therefore , in the fur mutant , levels of the Isc machinery may not be sufﬁcient to fully mature IscR , yet corresponding upregulation of the Suf pathway by increased levels of apo-IscR could at least partially account for IscR [ 2Fe-2S ] cluster assembly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , we considered the possibility that upregulation of the Suf machinery by apo-IscR and subsequent transfer of a Suf-generated [ 2Fe-2S ] cluster to IscR may explain the residual [ 2Fe-2S ] - IscR-dependent activity measured in the ΔiscSUA-hscBA-fdx strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Thus , the Suf pathway , when expressed at elevated levels , appears to be able to substitute for the Isc pathway by inserting a [ 2Fe-2S ] cluster into IscR , and this may account for the residual [ 2Fe-2S ] - IscR activity measured in the strain lacking the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Regulation of iron-sulfur cluster homeostasis through transcriptional control of the Isc pathway by [ 2Fe-2S ] - IscR in Escherichia coli Jennifer L. Giel1 ,2,3 , * , April D. Nesbit2 ,4 , * , Erin L. Mettert2 , * , Angela S. Fleischhacker2 , Brendan T. Wanta2 ,5 , and Patricia J. Kiley2 ,6 1Microbiology Doctoral Training Program , University of Wisconsin , Madison , Wisconsin 53706 , USA 2Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin 53706 , USA SUMMARY Fe-S clusters are essential across the biological world , yet how cells regulate expression of Fe-S cluster biogenesis pathways to cope with changes in Fe-S cluster demand is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Regulation of iron-sulfur cluster homeostasis through transcriptional control of the Isc pathway by [ 2Fe-2S ] - IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Giel , J.L. , Nesbit , A.D. , Mettert , E.L. , Fleischhacker , A.S. , Wanta , B.T. , and Kiley , P.J. ( 2013 ) Regulation of ironsulphur cluster homeostasis through transcriptional control of the Isc pathway by [ 2Fe-2S ] - IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Together , these data show how transcriptional control of iscRSUA-hscBA-fdx by [ 2Fe-2S ] - IscR allows E. coli to respond efficiently to varying Fe-S demands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Together , our data indicate that [ 2Fe-2S ] - IscR regulates expression of the Isc pathway via a negative feedback mechanism based on the cellular Fe-S demand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Finally , because active IscR can be abundantly produced in E. coli by using recombinant DNA methods , and the active form contains an [ Fe-S ] cluster , it should be possible to examine the role of each [ Fe-S ] cluster assembly protein in vivo by using whole-cell Mossbauer and electron paramagnetic resonance spectroscopies to monitor the assembly , and perhaps , disassembly of the [ 2Fe-2S ] cluster contained in IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>Conversely , expression of the isc operon is controlled by one of its products , IscR , and an attractive model for the feedback regulation by the [ 2Fe-2S ] form of IscR has been proposed ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>Oxidative stress conditions increase suf and isc expression , which was attributed to a shift from the cluster-containing form to the apoprotein form of IscR ( 14 , 15 ) , although whether apo-IscR and [ 2Fe-2S ] - IscR function similarly in activating suf has not been tested in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Furthermore , the [ 2Fe-2S ] cluster is not required for IscR to activate suf ( 14 , 15 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The global regulator IscR coordinates cluster synthesis by repressing transcription of the isc operon by [ 2Fe-2S ] - IscR and activating expression of the suf operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>We show that either [ 2Fe-2S ] - IscR or apo-IscR can activate suf , making expression sensitive to mainly IscR levels and not the cluster state , unlike isc expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , isc expression is coupled to Fe-S demand through levels of the IscR [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Our finding that IscR contains a [ 2Fe-2S ] cluster raises the question of how this cluster might function in regulation of IscR repressor activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>The regulatory mechanism for iscR transcription is not well characterized , except that the repressor activity of IscR depends on the Fe-S cluster formation , most likely a [ 2Fe-2S ] , as observed in anaerobically puriﬁed IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The observation that IscR activity is decreased in mutants lacking the Fe-S cluster assembly proteins , IscS and Hsc66 , has led us to consider that the ability of IscR to function as a repressor depends on whether it contains a [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Because IscR requires its Fe -- S cluster to repress the iscR promoter ,1 any condition that leads to a decrease in [ 2Fe -- 2S ] - IscR should increase the amount of total IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Upon acquisition of a [ 2Fe-2S ] cluster from the Isc machinery , the cluster-containing form of IscR can directly bind the iscR promoter ( PiscR ) to repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Since these are the only cysteine residues in IscR , it is likely , although not proven , that they provide three of the ligands to the [ 2Fe-2S ] cluster ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>This model would also imply that the [ 2Fe-2S ] cluster would have one noncysteinyl ligand , because only three Cys are present in IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>s a functional Isc system to fully repress PiscR in vivo ire u q re R c Is Other Fe-S biogenesis proteins are not necessary for [ 2Fe-2S ] - IscR activity when the Isc pathway is functional We also examined whether other Fe-S biogenesis proteins contribute to IscR cluster assembly since a small amount of [ 2Fe-2S ] - IscR activity remained in the strain lacking the Isc pathway ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>way is the major system for IscR [ 2Fe-2S ] cluster biogenesis th a p c Is e h T Deletion of the iscSUA-hscBA-fdx operon resulted in decreased PiscR repression , suggesting that IscR primarily receives its [ 2Fe-2S ] cluster from the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In this model , IscR [ 2Fe-2S ] cluster occupancy would be a reporter of the cellular Fe-S status , tuning synthesis of the Isc pathway in response to changing demands for Fe-S biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>As a result , there is competition between these substrates and IscR in acquiring Fe-S clusters from the Isc pathway , resulting in low IscR [ 2Fe-2S ] cluster occupancy and thus less repression of iscR-SUA-hscBA-fdx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The low demand for Fe-S biogenesis in the absence of O2 leads to increased IscR [ 2Fe-2S ] cluster occupancy , and in turn , more repression of the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Why IscR may not compete as well with other substrate proteins is not known but may be related to the fact that IscR binds its [ 2Fe-2S ] cluster with an atypical ligation scheme of In sum , we propose that under aerobic conditions , the sensitivity of some Fe-S clusters to O2 and/or reactive oxygen species results in a higher rate of cluster turnover , which in turn leads to increased synthesis of the isc operon to meet the demands for Fe-S biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Together , the unique features of the [ 2Fe-2S ] cluster-dependent and - independent modes of IscR regulation have led to the proposal that conditions that increase the Fe-S demand will lead to decreased [ 2Fe-2S ] - IscR levels , de-repression of the isc operon , elevated apo-IscR levels , and accordingly , increased regulation of promoters with Type 2 sites by apo-IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>assembly proteins would subsequently lead to an increase in the rate of Fe-S cluster formation , causing levels of [ 2Fe-2S ] IscR to rise , thus reestablishing repression of the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>If IscR senses the cellular Fe-S status through its [ 2Fe-2S ] cluster , then conditions that increase the demand for Fe-S clusters are predicted to affect [ 2Fe-2S ] - IscR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This situation could make it either more difficult to assemble the [ 2Fe-2S ] cluster within IscR relative to other [ Fe-S ] proteins or lead to instability of the cluster once inserted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>β-Galactosidase assays using the IscR mutant lacking the [ 2Fe -- 2S ] cluster , IscR-C92A , showed that both PnapF and PhybO were repressed by the mutant 20 - and 2.1-fold , respectively ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Thus , the corresponding 2-fold increase in PiscR-lacZ repression observed in the fur iscSUA-hscBA-fdx mutant suggests that the increased levels of Suf machinery can provide IscR with [ 2Fe-2S ] clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Interestingly , we observed a small defect ( 2-fold ) in [ 2Fe-2S ] - IscR-dependent repression of P - lacZ in the fur strain iscR ( Fig. 4 ) , consistent with IscR protein levels being 2 - to 3-fold elevated in this mutant relative to the wild type ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>[ 2Fe-2S ] - IscR ( 20 to 718 nM ) or IscR-C92A / C98A/C104A ( 35 to 1000 nM ) was incubated with 5 nM Texas Red labeled-DNA , 40 mM Tris-Cl ( pH 7.9 ) , and 150 mM KCl for 10 min at room temperature under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Thus , in addition to decreased IscR protein levels under anaerobic conditions , the poor repressing function of the [ 2Fe-2S ] form of IscR , which predominates under anaerobic conditions , contributes to overall regulation of PhyaA under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Our studies revealed that IscR repressor function is diminished under anaerobic conditions because of antirepression by ArcA and AppY , decreased IscR levels , and decreased repressor function of [ 2Fe-2S ] - IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In sum , a combination of ArcA and AppY antirepression of IscR function , lower levels of IscR , and weak repression by [ 2Fe-2S ] - IscR leads to increased PhyaA expression under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Antirepression by the anaerobic transcription factors ArcA and AppY , in addition to a combination of decreased levels of wild-type IscR and the weak repressing function of [ 2Fe-2S ] - IscR , resulted in the lack of repression by IscR under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Because the Isc requirement for biogenesis of the [ 4Fe-4S ] cluster of FNR is greater under aerobic than anaerobic conditions ( Mettert et al. , 2008 ) , we considered the possibility that acquisition of the [ 2Fe-2S ] cluster by IscR is reduced under aerobic conditions because of the competition with other substrates for the Isc machinery .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Thus , we conclude that the difference in the amount of PiscR repression under aerobic and anaerobic conditions can be explained by O2-dependent changes in the cluster occupancy of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Thus , in the presence of the Isc pathway , other Fe-S assembly proteins do not have a major effect on IscR-dependent repression of PiscR , thus supporting our model that the Isc pathway provides the [ 2Fe-2S ] cluster required for IscR to fully repress PiscR expression under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In the absence of the Isc machinery , elevated expression of the Suf pathway can fully restore [ 2Fe-2S ] - IscR activity IscR [ 2Fe-2S ] cluster occupancy is greater under anaerobic-growth conditions In the wild-type strain , repression of PiscR was ∼ 3 -- 5-fold greater under anaerobic conditions than aerobic conditions despite the fact that there is ∼ 10-fold less IscR protein under anaerobic conditions ( Figs. 2 , 3 , and 5 , Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Assuming that the concentration of [ 2Fe-2S ] - IscR is equivalent to the IscR protein concentration required for 50 % repression , our data indicate that under aerobic conditions IscR is at least 9-fold less occupied with [ 2Fe-2S ] clusters than under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>To determine whether cluster occupancy affects the ability of IscR to repress under anaerobic conditions , we measured PhyaA-lacZ expression while increasing the levels of IscR with the C92A mutation ( IscRC92A ) , which lacks the [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Since we have previously observed a difference in IscR cluster occupancy between anaerobic and aerobic conditions ( 13 ) , we examined the possibility that [ 2Fe-2S ] - and apo-IscR might function differently in their abilities to repress PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The only reported difference regarding IscR between aerobic and an-aerobic conditions is that under anaerobic conditions IscR appears to be more highly occupied with the [ 2Fe-2S ] cluster than its aerobic counterpart ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Under either aerobic or anaerobic conditions , strains containing chromosomally encoded IscR-C92A , IscR-C98A , IscR-C104A , or IscRC92A/C98A/C104A exhibited a nearly complete defect in PiscR-lacZ repression ( Fig. 3 ) , indicating that under both growth conditions , IscR requires its [ 2Fe-2S ] cluster to repress PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>IscR requires its Fe-S cluster to repress expression from the iscR promoter in vivo under aerobic and anaerobic conditions To establish the extent of the IscR [ 2Fe-2S ] cluster requirement for negative autoregulation under both aerobic and anaerobic-growth conditions , we measured PiscR repression in strains expressing IscR mutants that are defective in cluster binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>To address whether differences in [ 2Fe-2S ] cluster occupancy for IscR exist between aerobic and anaerobic-growth , an in vivo titration experiment was performed to determine the level of IscR protein required for PiscR repression for both conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Examination of PiscR repression under aerobic and anaerobic-growth has revealed that the increased repression of isc under anaerobic conditions is due to increased IscR [ 2Fe-2S ] cluster occupancy under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>We observed that wild-type and apoprotein ( IscR-C92A ) forms of IscR behaved similarly in activating PsufA-lacZ ( Fig. 3 ) even under an-aerobic conditions , where wild-type IscR is predominantly in the [ 2Fe-2S ] cluster form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>When we tested the ability of different levels of wild-type IscR ( containing the [ 2Fe-2S ] cluster ) and IscR-C92A ( apo mutant ) to repress the synthetic promoter under anaerobic conditions , we found that the apo mutant repressed more efﬁciently than the wildtype IscR ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In addition , no difference was observed in the apparent binding affinity of wild-type IscR assayed under anaerobic conditions , which contains some reduced [ 2Fe -- 2S ] 1 + cluster ,1 versus under aerobic conditions , which rapidly oxidizes to the [ 2Fe -- 2S ] 2 + cluster .1 Since there was little difference in binding between preparations of the as-purified wild-type protein and the mutant lacking the cluster , these data argue that the Fe -- S form has a similar affinity for the hya site as apo-IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>B ) DNA binding isotherms of wild-type [ 2Fe-2S ] - IscR ( open symbols ) and the clusterless mutant protein IscR-C92A / C98A/C104A ( closed symbols ) measured as a change in anisotropy under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Unexpectedly , IscR with a C92A mutation ( IscR-C92A ) , which mimics the clusterless form of the protein that is present predominantly under aerobic conditions , was a better repressor under anaerobic conditions of both PhyaA and a constitutive promoter containing the IscR binding site from PhyaA than wild-type IscR , which is predominantly in the [ 2Fe-2S ] form under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Since in vitro DNA binding by [ 2Fe-2S ] - IscR is not sensitive to the O2-dependent change in the [ 2Fe-2S ] cluster oxidation state ( Fleischhacker et al. , 2012 ) , a simple interpretation of our in vivo results is that the fraction of IscR containing a [ 2Fe-2S ] cluster may be higher under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Also , stress conditions may affect the oxidation state of the [ 2Fe -- 2S ] cluster , but similar in vitro binding of IscR under anaerobic and aerobic conditions to the hya DNA site suggests that O2-mediated oxidation of [ 2Fe -- 2S ] 1 + - IscR to [ 2Fe -- 2S ] 2 + - IscR1 does not affect IscR binding to Type 2 sites ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The requirement of the [ 2Fe-2S ] - containing form of IscR for site-specific binding in vitro and regulation of PiscR under aerobic and anaerobic conditions in vivo was analyzed using IscR mutants that can not ligate an Fe-S cluster but are otherwise functional .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Despite this , we found that under anaerobic-growth conditions when IscR is predominantly in the [ 2Fe-2S ] form , IscR did not regulate most promoters with type 2 sites , including P ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Taken together , our data support the notion that IscR requires its [ 2Fe-2S ] cluster to negatively regulate PiscR under both aerobic and anaerobic conditions and this occurs by [ 2Fe-S ] - IscR binding to the two Type 1 sites within the iscR promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Although recent data indicate that the [ 2Fe-2S ] cluster occupancy of IscR is decreased under aerobic conditions ( 13 ) , we did not expect any differences in IscR function between aerobic and anaerobic conditions for promoters containing type 2 sites since the afﬁnities of the two forms of IscR for the hyaA promoter region were similar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>MALDI-TOF mass analysis identiﬁed it with IscR , known to serve as a repressor of the iscRSUA gene expression under anaerobic condition as a [ 2Fe-2S ] - bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Using aerobically or anaerobically grown cultures , expression of PiscR-lacZ , which requires the [ 2Fe-2S ] cluster-containing form of IscR for repression , was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>We subsequently measured PiscR-lacZ repression which , according to our in vivo data , requires the [ 2Fe-2S ] form of IscR , and the amount of IscR present at each IPTG concentration was quantified by Western blots .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Recent studies made use of similar strains to demonstrate that , while repression of the iscR promoter requires [ 2Fe-2S ] - IscR ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>[ 2Fe-2S ] - IscR exhibits decreased repressor function at the hyaA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>β-galactosidase assays For the FNR overexpression experiment , strains were grown in M9 minimal medium ( described in the above paragragh ) containing 50 µg ml − 1 Ap , and 20 µg ml − 1 Cm under lo b tern es W t analysis Strains were grown as for β-galactosidase assays , and IscR , SufD , or FNR levels were measured by Western blots as described previously ( Nesbit et al. , 2009 , Mettert et al. , 2008 , Sutton et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Transcripts originating from P1 and P2 were present at such low levels in the absence of additional factors , as has been previously reported ( Darwin et al. , 1998 ) , that it could not be determined whether IscR repressed these transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>For simplicity , all protein concentrations herein are reported as monomers , although IscR is mainly a dimer in solution ( Nesbit et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Connection of other O - regulated genes to 2 the IscR regulon In addition to the hya , hyb and nap operons , a recent global gene expression study ( Kang et al. , 2005 ) indicates that the expression of other candidate promoters identiﬁed in this study ( ﬂu , sufABCDSE , sodA , yjiH and ﬁmICDFGH ) is also regulated by O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Nevertheless , expression of csdA-ygdK , encoding another IscS/SufS homologue , CsdA , and a second SufE homologue , YgdK , was not regulated by either IscR or O2 in our experiments ; YgdK enhances the cysteine desulphurase activity of CsdA in a manner analogous to that of SufE and SufS ( Liu et al. , 2005 ; Loiseau et al. , 2005 ) , although no in vivo function has been established for these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In support of the model , the presence of [ 2Fe-2S ] - IscR was demonstrated in vivo by Mössbauer spectroscopy and repression of PiscR was shown to require [ 2Fe-2S ] - IscR , since clusterless IscR mutants failed to repress PiscR ( Fleischhacker et al. , 2012 , Yeo et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In addition , whether the decrease in in vivo repression observed with the clusterless mutants can be explained by a lack of DNA binding is of particular interest because IscR target promoters containing Type 2 sites ( such as those in the sufABCDSE , napFDAGHBC , hyaABCDEF , and hybOABCDEFG operons ) do not require the [ 2Fe-2S ] cluster form of IscR either for binding DNA in vitro ( Nesbit et al. , 2009 ) or for transcriptional regulation in vivo ( Nesbit et al. , 2009 , Yeo et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>sites A and B are critical for IscR-mediated repression of PiscR Of the three IscR binding sites present within PiscR ( Giel et al. , 2006 ) , we found that only the two adjacent Type 1 IscR binding sites , A and B , are responsible for PiscR repression in vivo under standard growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Since most IscR-regulated promoters contain just a single IscR binding site ( Giel et al. , 2006 ) , perhaps the presence of multiple IscR binding sites within the PiscR region may be important to extend the dynamic range to regulate Fe-S biogenesis in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>When the iscR promoter region from -96 to -27 was examined for IscR binding in parallel , some weak binding complexes were also formed with cell extracts from oxidant-treated cells , suggesting that IscR can also bind to its own promoter region under aerobic condition , consistent with previous observation by Schwartz et al. ( 2001 ) ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>However , earlier in vivo studies in E. coli reported no effect of the iscX mutation on GltBD and Sdh activities , or more recently a slight defect on IscR maturation ( Tokumoto and Takahashi , 2001 ; Giel et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Finally , deletion of the genes encoding IscX , PepB , and SseB , which are located downstream of iscRSUA-hscBA-fdx and are repressed by IscR ( Giel et al. , 2006 ) , were also tested for their effects on IscR function , although only IscX has been suggested to have a possible function in Fe-S biogenesis ( Pastore et al. , 2006 , Shimomura et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Thus , under conditions of oxidative-stress , IscR would be involved in the upregulation of both the isc and suf operons ( Zheng et al. , 2001 ) , although whether this occurs by the IscR feedback mechanism is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The Suf pathway has been suggested to function under conditions of oxidative and iron stress ( Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) via OxyR and Fur , respectively , but the ﬁnding that IscR also controls the Suf pathway suggests that this alternate pathway is involved in Fe-S biogenesis under normal aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Recent work showed that IscR regulates about 40 genes involved in anaerobic respiration , oxidative-stress response , Fe -- S biogenesis , and other metabolic pathways ( Giel et al , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Recently , Nesbit et al. ( 2009 ) reported that the Fe-S cluster is dispensable for IscR regulation of some of its target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Cys92 , Cys98 , and Cys104 are predicted Fe-cluster ligands in IscR and indeed , IscR-C92A with an alanine substitution did not contain an Fe-S cluster , when it was puriﬁed under anaerobic condition ( Nesbit et al. 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Conﬁning the function of IscR to the Fe-S form provides an attractive feedback mechanism for Fe-S biogenesis to be sensitive to the Fe-S cluster status of cells ; as previously proposed ( Schwartz et al. , 2001 ) , when the Isc and Hsc proteins and the building blocks of Fe-S clusters ( Fe2 + , cysteine ) increase to the appropriate cellular concentrations , IscR itself will acquire an Fe-S cluster , thus enabling it to repress transcription of the iscRSUA-hscBA-fdx operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The analysis identiﬁed both bands from E1 and E2 with IscR , known as a transcriptional-repressor of its own gene , iscR , and the Fe-S cluster assembly genes of the iscRSUA oepron ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>This iscRSUA auto-repression depends on IscR loaded with Fe -- S clusters ( Holo ¬ IscR ) , which are synthesised by IscS , IscU , and IscA ( Schwartz et al , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscS , IscU , and IscA are responsible for Fe -- S cluster biogenesis and are thought to transfer these clusters to IscR ( Schwartz et al , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Therefore , when Fe is insufﬁcient , or when IscSUA activity is limited , the IscR regulator , now free of the Fe -- S cluster ( Apo-IscR ) , relieves its repression on the isc promoter , thus allowing transcription ( Schwartz et al , 2000 ; Outten et al , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>As the Fe-S form of IscR has only been shown to be necessary for repression of the isc operon ( Schwartz et al. , 2001 ) , it is possible that IscR lacking an Fe-S cluster may also bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR itself is an Fe-S cluster protein and an Fe-S cluster in IscR is necessary for transcriptional repression of the isc operon ( Schwartz et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>When IscR is bound to a Fe -- S cluster ( Holo-IscR ) , it represses the isc promoter as a feedback control ( Schwartz et al , 2001 see Introduction for details ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscR is a transcriptional-regulator that in its Fe-S cluster-bound form represses the expression of the isc operon ( Schwartz et al. , 2001 ; Giel et al. , 2013 ; Rajagopalan et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Possible involvement of IscR , which negatively regulates the iscSUA genes encoding another Fe-S assembly system in E. coli , can be considered , based on the possibility that IscR , being a 2Fe-2S protein , could respond to oxidants ( Schwartz et al. , 2001 ; Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Furthermore , inactivation of the Isc Fe-S biogenesis pathway decreases repression of the iscR promoter ( Schwartz et al. , 2001 ) , suggesting that the [ 2Fe-2S ] cluster is required for IscR repressor function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In addition , strains lacking IscS and HscA showed decreased IscR repression , indicating a role of the Isc pathway in maintaining the [ 2Fe-2S ] form of IscR ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>PiscR repression requires [ 2Fe-2S ] - IscR and is decreased in strains lacking iscS or hscA ( Schwartz et al. , 2001 ) , supporting the model that the Isc machinery synthesizes the [ 2Fe-2S ] cluster for IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Hydrogen peroxide also affects the function of IscR , a transcriptional-regulator that can coordinate a [ 2Fe-2S ] cluster ( Yeo et al. , 2006 ; Lee et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Unlike regulation of PiscR , regulation of the suf operon by IscR does not require IscR to ligate a [ 2Fe-2S ] cluster ( Nesbit et al. , 2009 , Yeo et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Although transcription of the isc operon is repressed by IscR ( Schwartz et al. , 2001 ) , a [ 2Fe-2S ] cluster-containing protein encoded by the first gene of this operon , a detailed understanding of how this autoregulatory mechanism occurs and ultimately how Fe-S biogenesis is regulated in vivo remains to be elucidated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>One direct regulator of the iscRSUA operon is IscR that contains a [ 2Fe-2S ] cluster and represses the operon under anaerobic condition ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Brieﬂy , IscR is a [ 2Fe-2S ] transcriptional-regulator that under its holo-form acts as a repressor of the isc operon ( Schwartz et al. , 2001 ; Giel et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Repression of isc expression by IscR is relieved when a 2Fe-2S cluster present in IscR is damaged or fails to form , and a non-polar deletion in iscR causes a 17-fold increase in basal expression of the isc promoter ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>In fact isc expression was somewhat higher in the H2O2-stressed cells than in unstressed cells , presumably reflecting substantial conversion of [ 2Fe-2S ] − IscR , a repressor of isc , to the apo-IscR form ( Giel et al. , 2006 , Schwartz et al. , 2001 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Fur and IscR are the other global regulators that are perturbed by H2O2 stress ( Varghese et al. , 2007 ; Yeo et al. , 2006 ; Lee et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Furthermore , it was shown recently that Apo-IscR could function as a transcriptional-activator of the suf operon ( Giel et al , 2006 ; Yeo et al , 2006 ; Lee et al , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Lee et al. previously demonstrated that suf transcription is repressed by Fur primarily by antagonizing IscR binding to PsufA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The suf promoter is activated independently by apo-IscR and OxyR ( Yeo et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>In contrast , the iscR mutation derepressed the level of iscR gene expression in the absence of any oxidant treatment , consistent with the proposal that IscR acts as a repressor of its own operon ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The mutant forms of IscR derepressed the iscR transcripts to the full level in the absence of oxidative treatments in accordance with the model proposed by Schwartz et al. ( 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Within this group were the genes of the isc operon ( iscS , iscU , iscA , hscB and fdx ) , which are known to be repressed by IscR ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Contrary to suf , the isc operon is not regulated by Fur but by IscR , a transcriptional-repressor encoded as part of the iscRSUA polycistron ( Schwartz et al , 2001 ; Giel et al , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscR has been reported to act as a repressor for its own operon iscRSUA ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>IscR was originally discovered by its role in the negative control of this operon ( Schwartz et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>In agreement with this notion , the induction of the isc operon by H2O2 is dependent on IscR ( Outten et al. , 2004 ) , and is independent of OxyR , unlike the induction of the suf operon ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>For example , gene expression of sufABCDSE is induced by H2O2 ( Zheng et al. 2001 ) , and oxidant induction of the suf operon is regulated by OxyR , IHF , Fur and IscR ( Lee et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_214">
Schwartz , C.J. , Giel , J.L. , Patschkowski , T , Luther , C , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S cluster-con-taining transcription factor , represses expression ofEscherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983079" target="_blank">11983079</a>]<br></label>
<input id="act_similarity_214", type="checkbox">
<div class="ACT_similar">
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T. , Luther , C. , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S clus-ter-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T. , Luther , C. , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li></div>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor biosynthesis , and DNA-RNA metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative-stress .6,11 -14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Furthermore , iscS is potentially important for global transcriptional regulation because several transcriptional-regulators , such as SoxR [ 19 ] , IscR [ 20 ] , Fnr [ 21 ] , and YeiL [ 22 ] , carry Fe -- S clusters as an essential prosthetic group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>In contrast to previous results with the iscR promoter , we found that the Fe -- S cluster is dispensable for IscR regulation of PhyaA , PydiU , PnapF , and PhybO , since IscR mutants containing alanine substitutions of the cysteine Fe -- S ligands retained IscR-dependent regulation of these promoters in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Independent alanine substitutions of the predicted Fe -- S cysteine ligands of IscR ( C92 , C98 , and C104 ) were tested for their ability to regulate PydiU and PhyaA , in addition to PsufA , to determine whether this lack of a requirement for the Fe -- S cluster was common to other Type 2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>While these data indicate that IscR lacking a cluster binds the hya site with high affinity , we could not precisely determine the affinity of the Fe -- S-only form because all of our preparations of wild-type IscR are a mixture of apo - and Fe -- Scontaining forms , despite attempts to remove the apo-protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Surprisingly , we also found that even in the absence of AppY and ArcA , wild-type IscR could not repress PhyaA anaerobically as well as an IscR mutant that lacked the Fe-S cluster or as well as wild-type IscR aerobically at the same protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The need for tight control of iron-sulfur cluster assembly is further emphasized by the finding that the transcription factor IscR , which serves as the intracellular sensor of the demand for FeS cluster biosynthesis , and the FeS cluster scaffolding protein IscU were captured with Clptrap ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23091052" target="_blank">23091052</a>]<br></li>
<li>The level of H2O2 that accumulates in Hpx − mutants is sufficient to disrupt Isc-mediated cluster assembly , leading to the conversion of substantial IscR to its apoprotein form ( Jang and Imlay , 2010 ) , with consequent effects on the operons it regulates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Thus , PsufA is unlike PhyaA in that both cluster-containing and apoprotein forms of IscR are sufﬁcient to regulate suf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Because IscR levels and Fe-S cluster occupancy vary depending on the genetic backgrounds used in the previous experiment , we constructed strains where iscR was ectopically expressed to systematically examine the effect of IscR levels on Fur repression of suf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>We extended this analysis to compare the effects of IscR cluster occupancy on Fur repression of suf by comparing wild-type IscR to IscR-C92A , the variant lacking a Fe-S cluster , as a mimic for the apoprotein form of IscR that is generated in strains lacking the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , any alteration that may lead to increased IscR levels , such as an increase in the Fe-S demand via oxidative-stress or Fe limitation , would lead to an increase in suf expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>It is likely that the loss of IscR repression under nickel stress ( when iron and Fe-S ho-meostasis are disrupted ) leads to increased expression of apo-YqjI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>The ﬁndings presented here indicate a more general role of IscR in the regulation of Fe-S cluster biogenesis and that IscR contributes to the O2 regulation of several promoters controlling the expression of anaerobic Fe-S proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>However , as the IscS level decreases after RyhB expression , we hypothesised that the IscR protein produced in these conditions is under the Apo form ( without a Fe -- S cluster ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>The IscR protein produced under these conditions is likely to be under the Apo form ( lacking Fe-S ) and may activate the expression of the suf operon , which would then overtake the Fe -- S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>The increase in IscR protein levels in the mutant strains likely arises from the increase in expression of the iscR promoter , since the Fe -- S form of IscR is required to repress P 1 iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>It was previously reported that deletion of the IscR Fe-S regulatory protein leads to upregulation of yqjI , suggesting that IscR may normally repress yqjI expression ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>DISCUSSION The findings of this study have revealed new mechanistic insight as to how IscR regulates transcription of PiscR to maintain Fe-S cluster homeostasis in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This would cause IscR to lose its Fe-S cluster , thereby inactivating it and relieving the repression of yqjI transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Therefore , IscR appears able to repress rnlA transcription regardless of the presence of an Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Puriﬁed IscR that lacked Fe-S cluster bound to the ORE-III site and activated transcription from the sufA promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Dispensability of Fe-S-binding sites in IscR for activating sufA transcription IscR homologues in bacteria contain three highly conserved cysteine residues ( C92 , C98 and C104 in E. coli IscR ) which is thought to coordinate Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>When sufﬁcient amount of Fe-S cluster is provided , as sensed by forming [ Fe-S ] - IscR , iscR expression gets repressed to a certain basal level , and sufA expression is no longer activated and returns to near ground level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>C. Transcription of the iscR and sufA genes was not induced during manganese intoxication , indicating that IscR remained in its cluster-containing form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>The status of Fe-S cluster including its redox state could affect the activity of IscR as a repressor and an activator , thereby sensing the intracellular state of Fe-S assembly and maintenance , and thus coordinating gene expression as cell needs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The Fe-S cluster appears to be important for IscR function , because repression of iscR expression is significantly reduced in iscS-null and hscA-null strains ( Schwartz et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>The Fe-S cluster appears to be important for IscR function , because repression of iscR expression is signiﬁcantly reduced in strains containing null mutations of the Fe-S cluster assembly genes iscS or hscA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Given the proximity of iscR to the Fe-S cluster assembly genes iscSUA , the similarity of IscR to MarA , and the possibility that IscR contains a Fe-S cluster , we characterized the properties of IscR and investigated whether this protein regulates the transcription of the isc gene cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Third , deletion of iscR results in elevated expression of the [ Fe-S ] cluster assembly genes in vivo and isolated IscR inhibits their expression in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>When cells are under attack by exogenous reactive oxygen species as encountered in phagocytic environment , oxidation of Fe-S cluster and its subsequent disassembly in IscR will actively induce the sufABCDSE operon , and will derepress the iscRSUA operon at the same time , maximizing the cellular capacity for Fe-S assembly and maintenance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Iron-sulfur ( Fe-S ) clusters are ancient and ubiquitous prosthetic groups involved in a wide variety of biological processes .1 In prokaryotes , the ISC ( iron-sulfur cluster ) biogenesis system , encoded by the isc operon ( IscR , IscS , IscU , IscA , HscB , HscA , and ferredoxin ) is mainly responsible for the ` house-keeping ' generation of iron-sulfur clusters .2 The complex interactions of these proteins are essential for efficient Fe-S cluster biogenesis .3 A very similar ISC system is found in the mitochondria of eukaryotes .4 IscU is the scaffold protein for cluster assembly and transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22734684" target="_blank">22734684</a>]<br></li>
<li>The transcriptional proﬁling data suggested that the alternate Fe-S cluster biogenesis system encoded by the sufABCDSE operon is also regulated by IscR ; however , unlike the isc operon , the suf operon appears to be activated by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The ISC operon is controlled by IscR , a Fe-S cluster binding transcriptional-repressor , and contains genes for a cysteine desulfurase ( IscS ) , scaffold protein ( IscU ) , ferredoxin ( Fdx ) , IscA , molecular chaperones ( hscA and hscB ) , and IscX ( also known as yfhj and ORF3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22352884" target="_blank">22352884</a>]<br></li>
<li>This cleavage gives rise to the remaining 50-section of the transcript encoding IscR , a transcriptional-regulator responsible for activation and repression of several genes depending on the cellular Fe -- S level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>By direct RyhB action , the downstream section encoding the Fe -- S cluster formation machinery , iscSUA , is degraded , giving rise to the remaining fragment encoding IscR , a transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Because our data also suggest that the Fe-S cluster of IscR may be important for its function as a transcriptional-regulator , we propose that IscR participates in an autoregulatory loop to maintain the appropriate levels of cellular Fe-S cluster synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>It is important to recall that we tested IscR , a transcriptional-regulator with an oxygen sensitive Fe-S cluster , an oxygen-resistant variant of FNR , another transcriptional-regulator , and the Nuo and Sdh complexes that both contain buried oxygen resistant Fe-S clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>For example , only residual Fe-S cluster-dependent activities were detected for IscR , fumarate nitrate regulator ( FNR ) , NsrR , glutamate synthase , isopropylmalate isomerase , NADH dehydrogenase I , and succinate dehydrogenase in strains lacking a functional Isc pathway ( 10 , 20 , 23 , 29 , 42 -- 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Although the fourth ligand to the IscR Fe -- S cluster is not known , variations in this ligand could explain the ability of different Rrf2 proteins to res-pond to specific signals ( e.g. , Fe levels , nitric oxide , or Fe -- S cluster status ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>While the role of IscR in repressing the housekeeping Isc pathway is clearly established , how IscR activity is regulated to maintain an appropriate amount of Fe-S biogenesis is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR was shown to require primarily the Isc pathway for synthesis of its Fe-S cluster , providing a link between IscR activity and demands for Fe-S clusters through the levels of the Isc system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>We have shown that in the absence of the Isc pathway , Fe-S cluster biogenesis and , hence , viability depend on activation of the suf operon by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Our data indicate that the elevated levels of apo-IscR in a mutant lacking the Isc pathway maintain cellular Fe-S cluster homeostasis by promoting transcription of the suf operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Whether IscR regulates other functions involved in Fe-S biogenesis had not previously been addressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Surprisingly , IscR activates the Suf pathway yet represses the other Fe-S biogenesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In particular , our results shed new insight on how IscR responds to the presence of O2 to satisfy the cellular need for Fe-S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Conditions that increase the Fe-S cluster demand ( e.g. , growth in the presence of O2 ) are predicted to also increase competition between IscR and other substrate proteins for the Isc machinery such that derepression of isc occurs ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , given their genomic location , regulation by IscR , and ability to speciﬁcally bind Isc and Hsc proteins , the yfhJ-pepB-sseB gene products may also contribute to Fe-S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Together , these ﬁndings highlight the role of IscR as the master regulator of Fe-S cluster biogenesis due to the ability of IscR to differentially regulate the Isc and Suf pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>In this study , we dissected the mechanism by which IscR negatively regulates PiscR to better understand how cells maintain Fe-S homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Finally , to examine whether IscR responds to cellular demands for Fe-S clusters , we tested whether overexpression of an Fe-S protein affected transcription from PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>One explanation for this is based on the fact that the Fe-S protein IscR represses transcription of yqjI .17 When the suf operon is expressed , the increased synthesis of Fe-S clusters by the SUF system likely decreases the concentration of free iron within the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Under the culture conditions , however , it is unknown whether the IscR , IscU , IscA , and Fdx proteins fused to GAD or GBD can accommodate the Fe-S clusters in yeast cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983079" target="_blank">11983079</a>]<br></li>
<li>ple , the activation of IscR could be used to assess in vitro transfer of [ Fe-S ] clusters from the assembly machinery to the immature form of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>To examine how ArcA interferes with IscR repression of PhyaA , in vitro DNA binding experiments were carried out with isolated ArcA and IscR ( Fig. 2A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , much greater repression was observed aerobically than anaerobically at the same concentrations of IPTG , consistent with direct IscR repression at PhyaA without competition from ArcA and AppY aerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In sum , we have found that a combinatorial strategy leads to ArcA and AppY overcoming IscR repression at PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Although isolation of AppY will be required before we can elucidate how this protein inhibits IscR repression at PhyaA , the simplest explanation is that both AppY and ArcA block IscR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The ability of ArcA and AppY to act as antirepressors of IscR repression of PhyaA depended on IscR levels , suggesting that IscR competes with ArcA and/or AppY for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Therefore , while the role of AppY in preventing repression by IscR is unclear , the simplest model from the combined results is that both AppY and ArcA act as antirepressors , competing with the repressor IscR for binding to PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Our data indicate that the decrease in expression upon elimination of ArcA or AppY is not due to activation by either transcription factor , but , rather , to repression by IscR in the absence of ArcA and/or AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Therefore , the decreased expression in strains lacking ArcA and/or AppY under anaerobic conditions does not appear to result from a defect in activation by ArcA or AppY but , rather , from unmasking IscR repression in the absence of these other transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , the role of ArcA as an anaerobic antirepressor of IscR at PhyaA raises the question of whether the decreased expression of other promoters in strains lacking ArcA is due to the loss of ArcA activation or to the unmasking of repression by another transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The repression by IscR in the absence of either ArcA or AppY appears to be a direct result of IscR binding , since the increase in expression observed from a mutation that is known to disrupt the IscR binding site ( C-to-G mutation at position 50 in PhyaA [ PhyaA ( 50G ) ] ) ( 28 ) mimicked the increase in expression of the iscR strains ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>For example , the decreased expression of PhyaA that arises when strains lack ArcA ( AppY and IscR ) or AppY ( ArcA and IscR ) is reversed by the further elimination of IscR from either strain ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In order to test IscR repression of the hyaA site in the absence of other factors that might unknowingly affect PhyaA regulation anaerobically , we incorporated the IscR binding site from PhyaA into a constitutively active promoter [ Psyn ( hya ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Since removing the IscR binding site from the context of PhyaA results in a similar preference for promoter repression by the apo form of IscR , it is unlikely that there is another factor dictating the cluster dependence of PhyaA repression by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>This was surprising because the iron-dependent repressor Fur , whose higher-afﬁnity binding at the sufA promoter should occlude IscR binding , showed only partial repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>g in d in b cR Is Since both IscR sites A and B span the region that RNA polymerase binds , the mechanism of IscR repression can be simply explained by promoter occlusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Additional studies are necessary to determine what sequences in the iscR promoter are critical for IscR DNA binding and repression and the mechanism of IscR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In support of this competition model , electrophoretic mobility shift assays and DNase I footprinting showed that the ArcA and IscR binding sites overlap and that binding of ArcA and IscR is mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Based on the DNase I footprint and electro-phoretic mobility shift assays presented in this study , we propose that ArcA antagonizes IscR repression by partially occluding IscR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>DNA binding assays were used to determine whether IscR and ArcA bind simultaneously to the hya promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>To test whether IscR and ArcA can bind simultaneously , electrophoretic mobility shift assays were performed with radiolabeled hyaA DNA , IscR , and/or phosphorylated ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>As IscR protected a region of the hyaA promoter that overlaps the − 35 promoter element and site ( s ) where an activator may bind , one mechanism to explain the lack of IscR repression of the hyaA and hybO promoters under anaerobic conditions would be the occlusion of IscR binding by anaerobic regulators ( ArcA , AppY or FNR ) and/or RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>However , no complex with a size larger than the ArcA-DNA complex was observed when both proteins were present in the binding assay , indicating that binding of ArcA and IscR to hyaA promoter DNA is mutually exclusive ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Thus , ArcA and IscR appear to compete for binding to the hyaA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Lower IscR levels also contribute to decreased IscR regula ¬ FIG 2 ( A ) DNase I footprinting of ArcA and IscR binding to the hyaA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA and AppY Antagonize IscR Repression of Hydrogenase-1 Expression under Anaerobic Conditions , Revealing a Novel Mode O2 Regulation of Gene Expression in Escherichia coli of A. D. Nesbit , * A. S. Fleischhacker , S. J. Teter , and P. J. Kiley Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , USA Transcription of the Escherichia coli hydrogenase-1 operon ( hyaABCDEF ) is increased by the transcription factors ArcA and AppY under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Fluorescence anisotropy DNA binding isotherms were generated under anaerobic conditions by measuring changes in fluorescence polarization when IscR bound dsDNA as described ( Nesbit et al. , 2009 ) , and assays were repeated on three independent occasions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In contrast , for the hyaA and hybO promoters , IscR repressed the previously identiﬁed O2-regulated transcription start sites ; under anaerobic-growth conditions , AppY and the ArcA/B two-component regulatory system have been implicated in the activation of transcription of the hya operon ( Brondsted and Atlung , 1994 ; Richard et al. , 1999 ) , whereas FNR appears to contribute to activation of the hybO promoter ( Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>However , three new genes of unknown function ( yabI , yadR and yhgI ) were identiﬁed as candidates for repression by IscR speciﬁcally under anaerobic conditions.YabI is a predicted inner membrane protein , while both YadR and YhgI are in the HesB/YadR/YfhF protein family ( Dombrecht et al. , Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In a strain lacking NfuA , an IscA/NifU-like protein that facilitates Fe-S cluster transfer ( Bandyopadhyay et al. , 2008 , Angelini et al. , 2008 ) and whose expression is repressed by IscR ( Giel et al. , 2006 ) , there was a small defect ( ∼ 2-fold ) in IscR repression , but only under anaerobic conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The ﬁnding that IscR represses the expression of the hya , hyb and nap operons under aerobic conditions was surprising as previous work indicated that the regulation of these genes by O2 resulted from speciﬁc activation under anaerobic conditions ( Choe and Reznikoff , 1993 ; Brondsted and Atlung , 1994 ; Darwin and Stewart , 1995 ; Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>As expected from previous results ( Giel et al. , 2006 ) , IscR repressed PiscR expression more under anaerobic ( 17-fold ) than aerobic ( 6-fold ) growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>) Many of the 37 genes are in predicted ( Bockhorst et al. , 2003 ) or known operons , narrowing the number of candidate promoters regulated by IscR under aerobic conditions to 17 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Transcriptome analysis identiﬁed 40 genes in 20 predicted operons , which were regulated by IscR under aerobic and anaerobic conditions ( Giel et al. 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>In this study , global transcriptional proﬁling of Escherichia coli IscR + and IscR -- strains grown under aerobic and anaerobic conditions indicated that 40 genes in 20 predicted operons were regulated by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The only genes that appeared to be regulated by IscR under both aerobic and anaerobic conditions were iscSUA-hscBA-fdx-yfhJ-sseB , ﬁmIF and yqjI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR directly represses yadR and yhgI , predicted Fe-S biogenesis proteins Promoter -- lacZ fusions were constructed for yadR and yhgI , and assays of β-galactosidase activity ( Table 1 ) indicated that IscR decreased expression of these genes only under anaerobic-growth conditions as was found with the transcriptional proﬁling experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Transcription of a subset of anaerobically induced genes is repressed by IscR under aerobic conditions Unexpectedly , the transcriptional proﬁling data indicated that three operons , hyaABCDEF , hybOABCDEFG and napFDAGHBC , encoding respiratory enzymes that are known to contain Fe-S clusters and are active only under anaerobic conditions , were derepressed under aerobic conditions in the absence of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>As the promoters of other anaerobically induced Fe-S enzymes ( frd , fdnG , narG , dmsA , driving the expression of operons encoding fuma-rate reductase , formate dehydrogenase , nitrate reductase and DMSO reductase respectively ) are known to be poorly expressed under aerobic conditions and were below the limits of detection in the microarray experiments , we tested whether any of these other anaerobic respiratory enzymes were also controlled by IscR using the more sensitive β-galactosidase assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Expression of hyaA , hybO and napF did not appear to be repressed by IscR under anaerobic conditions as the amount of β-galactosidase activity was similar in strains containing or lacking IscR , suggesting that IscR might be an aerobic-speciﬁc repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Perhaps transcriptional repression of the hyaA , hybO and napF promoters by IscR under aerobic conditions prevents inappropriate Fe-S biogenesis of the Fe-S IscR regulates Fe-S and oxygen-regulated genes 1069 subunits under aerobic conditions , which might lead to their destruction and increase oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The ﬁnding that IscR repressed expression of the hyaA , hybO and napF promoters speciﬁcally under aerobic-growth conditions suggests a new mechanism to explain their upregulation under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR does not repress expression of all anaerobic Fe-S respiratory enzymes We considered the possibility that transcriptional repression of the hyaA , hybO and napF promoters by IscR under aerobic conditions may be a general mechanism to prevent inappropriate Fe-S cluster insertion into anaerobic Fe-S respiratory enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR regulates a diverse set of genes , including some encoding Fe-S proteins as well as genes encoding additional Fe-S biogenesis proteins Thirty-one genes showed an increase in transcript levels in the strain lacking IscR compared with the wild-type strain under aerobic conditions , indicating that these genes are candidates for repression by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR senses and responds to the different Fe-S demands that exist between aerobic and anaerobic-growth conditions Our finding that IscR is more sensitive to changes in Fe-S demand under aerobic conditions makes physiological sense .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>We report the role of other Isc pathway components and the Suf system in providing Fe-S clusters to IscR under aerobic and anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Taken together , the different needs for Isc Fe-S biogenesis under aerobic and anaerobic conditions emphasize the important role of IscR as an Fe-S sensor , enabling E. coli to respond appropriately to environments of varying O2 tension .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Furthermore , decreased isc repression under aerobic conditions suggests that there may be more competition between IscR and substrate proteins for the Isc machinery when O2 is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>As a result , there would be more competition between IscR and substrate proteins for the Isc machinery under aerobic conditions than anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Surprisingly , the isc operon was more repressed under anaerobic conditions , indicating increased Fe-S cluster occupancy of IscR and decreased Fe-S cluster biogenesis demand relative to aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Thus , under aerobic conditions , an increase in the Fe-S cluster demand via overexpression of FNR relieves repression of the isc operon , presumably due to the competition between FNR and IscR for acquiring Fe-S clusters from the Isc machinery .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This suggests that the Isc machinery can meet an increase in Fe-S demand under anaerobic conditions such that no competition between FNR and IscR was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Because a previous study established that under aerobic and anaerobic conditions IscR is elevated to the same high level when negative autoregulation is mostly eliminated by the absence of the Isc pathway ( 10 ) , we reasoned that increased IscR protein levels may be responsible for the increased suf expression observed in the absence of the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Under either aerobic or anaerobic conditions , deletion of the suf operon had no effect on PiscR repression ( Fig. 5 ) , indicating that the Suf pathway plays no apparent role in IscR function under nonstress conditions when the Isc system is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>For several genes ( dctA , ﬂu , hyaABCDEF , hybOABC-DEFG , napFDAGHBC , soxS , ycgZ , yjiM ) , repression by IscR was observed only under aerobic-growth conditions even though IscR can function anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>To test whether increasing the concentration of IscR could overcome antirepression by ArcA and AppY , we varied the levels of both the wild-type and a clusterless mutant of IscR and monitored the expression of the hya promoter under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>doi :10.1128 / JB.01757-12 Journal of Bacteriology p. 6892 -- 6899 IscR repression of PhyaA expression under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA and AppY antagonize IscR repression of hydrogenase-1 expression under anaerobic conditions , revealing a novel mode of O2 regulation of gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Rather , hya expression in S. enterica is regulated not by IscR or ArcA but , rather , by FNR under anaerobic conditions ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , elimination of all three transcription factors ( IscR , AppY , and ArcA ) resulted in an unexpectedly high level of expression under anaerobic conditions , which was slightly greater than the level of expression in the parent strain ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>hyaA In this study , we examined the roles of IscR , ArcA , and AppY in regulating the expression of the hyaA promoter under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA , AppY , and IscR act combinatorially to control hyaA promoter expression levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>RESULTS AppY and ArcA prevent IscR repression of the hyaA promoter under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Despite the fact that we observed more repression of the hyaA promoter with increasing levels of IscR , the extent of PhyaA repression under anaerobic conditions was never as great as that measured under aerobic conditions , even in the absence of AppY and ArcA ( Fig. 1 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>A further , but small , increase in repression of PhyaA was observed with IscRC92A when ArcA and AppY were eliminated ( Fig. 3C ) , indicating that the ability of apo-IscR to repress PhyaA under anaerobic conditions is largely indifferent to the presence of ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>These data also show that both AppY and ArcA must be present to prevent IscR repression of PhyaA under anaerobic conditions , since loss of either transcription factor is sufﬁcient to permit repression by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Consistent with the notion that PhyaA expression is regulated by IscR repression aerobically and by ArcA and AppY antirepression of IscR anaerobically , PhyaA-lacZ expression levels in the absence of any transcription factors were similar under anaerobic and aerobic conditions ( Fig. 1A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>We found that under anaerobic conditions two transcription factors , ArcA and AppY , work together to overcome the repression of PhyaA by IscR that occurs under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Here , we report that ArcA and AppY increase PhyaA expression under anaerobic conditions by antagonizing IscR binding at PhyaA , since IscR repression is observed when either ArcA or AppY is eliminated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , ArcA repression is not observed when all three transcription factors are present , suggesting that the major effect of ArcA under anaerobic conditions is to increase expression of PhyaA by preventing IscR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Therefore , downregulation of IscR levels under anaerobic conditions appears to be important for proper regulation of PhyaA , such that IscR competition with ArcA and AppY for PhyaA binding is maintained at a low level , keeping PhyaA from being repressed anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Hence , the concentration of IscR , in addition to regulation by ArcA and AppY , is important for determining the anaerobic expression levels of PhyaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The ﬁnding that PhyaA was repressed by IscR under aerobic conditions was surprising since activation of PhyaA under anaerobic conditions by ArcA and AppY would make an aerobic repressor unnecessary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Since ArcA competes with IscR for binding to PhyaA , we examined whether the proposed antirepression mechanism is dependent on the low IscR protein levels observed under anaerobic conditions compared to levels observed under aerobic conditions ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , the possibility that IscR is non-functional in the absence of O2 was excluded as IscR-dependent repression of the iscR promoter was detectable anaerobically and , in fact , was enhanced under anaerobic-growth conditions ( Fig. 1 , Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>However , IscR , which represses transcription of the hyaA promoter ( PhyaA ) under aerobic conditions , was not known to repress transcription of this promoter under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>More recently , another transcription factor , IscR , has been shown to directly repress the hyaA promoter ( PhyaA ) but only under aerobic conditions when ArcA and AppY are not active ( 14 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Identiﬁcation of promoter regions regulated by IscR under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Thus , the effect of IscR on the upstream promoter is only observed under aerobic conditions , when the downstream promoter is not functional .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>As the hya , hyb and nap operons are known to be upregulated under anaerobic conditions , we also examined whether IscR had any effect under anaerobic-growth conditions ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In addition , we found that under anaerobic conditions , expression of some operons ( sodA , soxS , sufABCDSE , ycgZ , yjcB ) was decreased relative to aerobic conditions , but regulation by IscR could not be determined as expression levels of these operons in both strains were below the limit of detection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR decreased expression of soxS twofold under aerobic conditions but had no effect under anaerobic conditions , which paralleled the results of the transcriptional proﬁling experiments ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Using a promoter -- lacZ fusion , we found that IscR increased expression of ydiU under both aerobic and anaerobic conditions ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR-C92A was purified from PK7879 under anaerobic conditions following the same protocol as for wild-type IscR , and IscR-C92A / C98A/C104A was purified from PK7881 similar to wild type but under aerobic conditions at 4 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>However , in same conditions , IscR levels remain stable even in the presence of RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>We found that under anaerobic conditions , more repression of PhyaA was observed with IscR-C92A than with wild-type IscR at the same IPTG concentrations ( Fig. 3B ) , despite the fact that protein levels were similar for wild-type IscR and IscR-C92A ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , even in the presence of 320 M IPTG , which produced 3 - to 50-fold more IscR protein than wild-type levels under aerobic and anaerobic conditions , respectively ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>This was the case over the entire range of protein concentrations tested , including when levels of wild-type IscR and IscR-C92A ( 80 M IPTG ) were comparable to chromosomal IscR levels under aerobic conditions ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The observed difference in repression between anaerobic and aerobic-growth conditions was not due to insufﬁcient IscR concentrations since the highest IPTG concentrations resulted in IscR protein levels well above those observed in aerobic cells ( determined by Western blotting ) ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The lack of repression of PhyaA under anaerobic conditions appears to be linked to the lower anaerobic levels of IscR ( 10-fold less than aerobic protein levels [ 13 ] ) since artiﬁcially increasing the levels of wild-type IscR to those found under aerobic conditions resulted in increased repression of PhyaA by IscR , even in the presence of ArcA and AppY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Measurements of IscR protein in these strains showed that each of the IscR variants was present at levels ∼ 2-fold and ∼ 8-fold greater than wild-type IscR under aerobic and anaerobic conditions , respectively , ( Table 2 ) ( Nesbit et al. , 2009 ) , indicating that the loss in PiscR repression did not result from a defect in protein accumulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Quantitative Western blots revealed that under aerobic-growth conditions , levels of IscR are increased about 4 - and 2-fold in ΔiscU and ΔiscA strains , respectively ( data not shown ) , indicating that IscR protein was present but unable to fully repress PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This notion was tested by measuring the levels of IscR protein required for PiscR repression under both growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Under both growth conditions , the amount of PiscR repression increased with increasing levels of IscR ; however , more IscR protein was needed to fully repress PiscR under aerobic conditions compared to anaerobic conditions ( Fig S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Fold repression was determined by dividing the β-galactosidase activity present in the strain lacking IscR ( PK6512 ) , grown in the presence or absence of nicotinic acid and thiamine , by the β-galactosidase activity measured for each strain and error bars represent the propagation of standard errors for three biological replicates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_366">
Fold repression was determined by dividing the β-galactosidase activity present in the strain lacking IscR ( DV915 ) by the β-galactosidase activity measured for each strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></label>
<input id="act_similarity_366", type="checkbox">
<div class="ACT_similar">
<li>The amount of IscR-dependent repression ( fold repression ) was determined by dividing the β-galactosidase activity present in the strain lacking IscR ( DV915 ) by the β-galactosidase activity measured for each strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li></div>
<li>No significant activity of IscR and FNR was predicted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>β-galactosidase values were expressed as fold repression for the level of IscR activity , and as fold activation for the level of FNR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>When sites A and B were both mutated , repression was eliminated , similar to that observed in a strain lacking IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In Fig. 3 and 4 , fold repression was calculated by dividing the promoter activity in a strain lacking IscR by the promoter activity in the indicated strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In M9 medium supplemented with FeSO4 , the repression activity of IscR depended on the presence of the cyaY gene ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Therefore , a strain containing both iscR under the control of Ptac and iscSUA hscBA fdx under the control of PBAD was grown aerobically ( Fig. 3A ) or anaerobically ( Fig. 3B ) in the presence of different IPTG concentrations to determine the effect of various IscR levels within the range normally produced under either condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In the case where Fur binding to PsufA was disrupted , providing a measure of only IscR activation , anaerobic - galactosidase activity was slightly increased ( 2-fold ) in the strain expressing IscR-C92A compared to that expressing wild-type IscR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Substitution of either -50 ( C to G ) or -38 ( A to T ) nearly eliminated repression by IscRC92A/C98A/C104A since only ∼ 2-fold repression was observed as compared to the typical ∼ 70-fold repression observed for wild-type PhyaA ( Fig. 7 ) , consistent with the N 25-fold defect in IscR binding to these mutant DNA sites in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The dispensability of the cluster was not specific to promoters that are activated by IscR since IscR-dependent repression of PhyaA-lacZ was even greater ( ∼ 3-fold ) with the IscR Cys-to-Ala substitutions than with wild-type IscR ( Fig. 2c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Strains containing IscR-C92A , IscR-C98A , or IscR-C104A showed ∼ 2-fold higher levels of activation of PsufA-lacZ than did wild-type IscR ( Fig. 2a ) , indicating that the Fe -- S cluster is not required for the transcription activation by IscR , in agreement with previous findings .7 The Fe -- S cluster of IscR was also dispensable for activation of PydiU since IscR-C92A , IscR-C98A , and IscR-C104A showed no defect in activation of PydiUlacZ ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The defect in repression caused by these site mutations was not specific to the IscR mutant C92A/C98A/C104A since strains containing wild-type IscR also showed a similar increase in expression ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In addition , transcript levels of hyaABCDEF and hybOABCDEFG , encoding hydrogenases-1 and -2 , respectively , and napFDAGHBC , encoding the periplasmic nitrate reductase , were increased in the strain lacking IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In addition , this appears to be an IscR-responsive transcript , because levels were increased in a strain lacking IscR ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In addition , the small increase in activation of PsufA and repression of PhyaA could be explained by the ∼ 2-fold increase in IscR protein levels in strains containing IscR-C92A , IscR-C98A , or IscR-C104A , as determined by quantitative Western blots using α-IscR antibody ( Fig. 2d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>We found that as the levels of IscR were increased ( by increasing the IPTG concentration ) under both conditions , there was a corresponding increase in repression by IscR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>thermore , by similarly increasing the levels of IscR in a strain lacking ArcA and AppY , we found that less IscR was required for repression anaerobically ( Fig. 3C ) than in strains containing AppY and ArcA ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The low level of the sufA transcripts in the absence of oxidants is attributable partly to repression by Fur , suggesting that sufA induction occurs only in the absence of Fur repression and in the presence of apo-IscR as dem-onstrated in Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Nor did a ΔnrdAB Δfur iscR - ( C92A/C98A/C104A ) mutant , despite the simultaneous deactivation of NrdR , absence of Fur , and presence of apo-IscR ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>We also noted that IscR overproduction caused a slight increase in the uninduced level of sufA transcripts , consistent with observations with the lacZ fusion constructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Results in Fig. 7 demonstrate that the exogenously provided wildtype IscR restored sufA induction in DoxyRiscR mutant to the level comparable to that in DoxyR mutant ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>In comparison , DoxyR mutation reduced the sufA -- lacZ induction to about 40 % of the wild-type level in SPD256 , and retained induction in SPD184 and SPD61 constructs , representing the contribution from IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Compared with the wild-type IscR , the basal level of the sufA transcripts slightly increased when mutant IscR proteins were provided .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>This was also the case when we introduced a more conservative disruption of the IscR binding site within P 50 sufA ( C changed to G ) that eliminates IscR-dependent suf activation in vivo ( Fig. 2B and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>This ﬁnding led us to question the mechanism by which this upregulation occurred since IscR was either not present or not able to activate suf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Furthermore , both the apo - and holoprotein forms of IscR are capable of activating suf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The contribution of apo-IscR to suf induction during H2O2 stress was moderate ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>The other two IscR binding regions have not yet been tested for their role in suf regulation in vivo ; only weak binding of region 3 was observed by high IscR concentrations in vitro , and integration host factor ( IHF ) , which activates suf in conjunction with OxyR in response to peroxide stress , exhibits stronger binding afﬁnity than does IscR for region 2 ( 11 , 12 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Previous studies have demonstrated that a clusterless IscR variant ( IscRC92A/C98A/C104A ) binds this site with high afﬁnity in vitro and is sufﬁcient to activate suf expression both in vivo and in vitro ( 14 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Fur binds to PsufA with higher afﬁnity than does holo - or apo-IscR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , consistent with previous in vitro studies ( 15 ) , the higher binding afﬁnity of Fur can not be outcompeted by physiological changes in IscR levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Despite the role of Fur in repressing suf in the absence of IscR , no colonies or only suppressor mutants were recovered when iscRSUA-hscBA-fdx or iscRSUA alleles were introduced into a fur strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>In addition , Fur derepression was not sufﬁcient for viability in the absence of IscR and the Isc pathway , highlighting the importance of direct IscR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>We also demonstrate that in the absence of isc , IscR-dependent suf activation is essential since strains lacking both the Isc pathway and IscR were not viable unless Suf was expressed ectopically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Since IscR levels also increase in mutant strains lacking components of the Isc pathway ( 10 ) , we address here whether this increase in suf expression speciﬁcally relies on IscR-dependent activation and is essential for viability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>IscR-dependent suf activation can partially restore holo-IscR activity when the Isc pathway is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Since our data show that IscR-dependent suf activation is increased in the absence of Fur ( Fig. 2 ) , we tested whether this induction resulted in increased [ 2Fe-2S ] - IscR activity in a strain also lacking the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , suf expression in the wild-type strain reﬂects contributions from both IscR activation and Fur repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Taken together , these data show that in the absence of the Isc pathway , suf expression is higher than wild-type levels , which can be explained by both increased IscR activation and diminished Fur repression , providing sufﬁcient suf expression to support viability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Using - galactosidase assays , we examined the sensitivity of PsufA-lacZ expression to elevated IscR protein levels and addressed whether equal levels of wild-type IscR and a clusterless IscR mutant ( IscR-C92A ) activate suf to the same extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Thus , to compare the abilities of holo-IscR and apo-IscR to activate suf , wild-type IscR or IscR-C92A was produced from Ptac and PsufA-lacZ and PsufA ( Fur * ) - lacZ expression was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Under nonstress conditions , Fur antagonizes IscR binding to PsufA , thus keeping suf expression low under conditions when the Isc system is functional ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>In contrast , under Fe-limiting conditions , loss of the Fur Fe2 cofactor eliminates Fur binding , allowing apo-IscR to fully activate suf expression ( 11 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Finally , a mutant lacking Fur and the Isc pathway increased suf expression to the highest observed levels and nearly restored [ 2Fe-2S ] - IscR activity , providing a mechanism for regulating IscR activity under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Finally , we measured how [ 2Fe-2S ] - IscR activity is affected by the increased suf expression observed in a fur iscSUA-hscBA-fdx double mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The increased IscR protein levels present in a iscSUA-hscBA-fdx mutant are critical for this regulation ; we demonstrated that by elevating the cellular concentration of IscR , a corresponding increase in suf expression occurs even when Fur is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Furthermore , suf expression was increased in a isc mutant , presumably due to increased IscR levels in this mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The increased expression of suf in the absence of the Isc pathway requires IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Together , these ﬁndings demonstrate that the increased expression of suf in a mutant lacking the Isc pathway correlates with increased IscR protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>However , in strains lacking the Isc pathway , Fur repressed suf expression only 4-fold , presumably because increased IscR levels are able to compete with Fur for its binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>IscR also appears to increase the expression of a small subset of genes , including sufABCDSE ; argE , involved in arginine biosynthesis ; and sodA , the Mn-containing superoxide dismutase , as the transcript intensities decreased by at least twofold in this assay ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Experiments that address whether hscBAfdx expression is regulated by IscR in E. coli should provide additional insight as to how expression of these genes is controlled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>To address this question , we used quantitative western blots to measure the levels of IscR and IscS proteins after RyhB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>This indeed appears to be the case for PsufA , PhyaA , and PnapF , since changes in expression in strains containing IscR mutants lacking the cluster were correlated with increased IscR levels , indicating that none of these IscR-binding sites are completely occupied under our standard growth condition ( Figs. 2 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>However , the physiological relevance of the increase in soxS expression in strains lacking IscR is questionable as no corresponding increase in SoxS target gene expression was observed in the strain lacking IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR is an iron-binding repressor ( 42 ) , and it is plausible that changes in intracellular iron availability upon RyhB expression could lead to increased IscR repression activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16199566" target="_blank">16199566</a>]<br></li>
<li>Regardless of the 3 endpoint fused to lacZ , expression was increased in strains lacking IscR , demonstrating that IscR regulates transcription of the entire iscRSUA gene cluster ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>These results showed that expression of the iscR - lacZ fusion increased 17-fold in the absence of IscR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>As shown in Fig. 7 , iscR - lacZ expression was also increased in cells lacking Hsc66 , indicating that IscR activity was decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In this study , we found that IscR functions as a repressor of the iscRSUA operon , because strains deleted for iscR have increased expression of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Hence , the level of IscR transcriptional-repressor activity was assayed by monitoring iscR-lacZ fusion expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Thus , changes in oxygen availability ( sensed directly by FNR , and indirectly by ArcA ) affect the redox state of the system , leading to consequences for metal ion homeostasis ( copper , molybdate , iron ; sensed by CusR , ModE and Fur ) , iron -- sulphur cluster turnover ( sensed by IscR ) , ribonucleotide reductase activity 20 0 31 0 10 aerobiosis ( % ) time ( min ) Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Transcription start sites for rnlA and rnlB were identiﬁed by primer extension analysis : We tested whether IscR can directly inhibit RNase LS activity in vitro using S30 extract from MH1 cells , but IscR could not inhibit RNA cleavages by RNase LS ( Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Because expression of IscR is induced under oxidative-stress or iron deprivation , RNase LS activity would be repressed under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Otsuka , Y. , K. Miki , M. Koga , N. Katayama , W. Morimoto et al. , 2010 IscR regulates RNase LS activity by repressing rnlA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20980243" target="_blank">20980243</a>]<br></li>
<li>From these results , we conclude that IscR negatively regulates transcription of rnlA and represses RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Taken together , we conclude that IscR represses the transcription of rnlA via its binding to the promoter region and thus reduces RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>These results demonstrated that ligation of the IscR with Fe-S cluster had almost no effect on IscR-mediated repression of RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>-- Effect of an Fe-S cluster in IscR on RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Next , we investigated another possibility that IscR regulates RNase LS activity via transcriptional control Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Repression of downstream genes in this operon by IscR might be responsible for inhibition of RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>This result suggested that an increase of intracellular IscR counteracts RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>-- Effect of IscR on RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Other promoters repressed by IscR in vivo We also examined whether the promoters for yqjI and the putative yjiH-yjiG-iadA-yjiE operons were directly repressed by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The sufA DNA probes , labelled at the 5 end of either the top or bottom strand , were incubated with increasing concentrations of IscR , followed by DNase I treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The in vitro transcription and footprinting data suggest at least two plausible mechanisms to explain how IscR can contribute to the O2 regulation of these genes without requiring a change in IscR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR contributes to the O2 regulation of several genes IscR appears to-bind to two different classes of DNA sites The use of bioinformatic methods in addition to DNase I footprinting of IscR binding sites was critical in identifying conserved sequences for binding of IscR as it appears that IscR can recognize two different DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The differential regulation of isc and suf by IscR is in part due to the ability of IscR to recognize two classes of DNA sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>However , changing -48 C to T to improve the symmetry relative to -38 A did not increase IscRC92A/C98A/C104A binding ( Table 1 ) , suggesting that -38 A is not part of the palindrome bound by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Given that we found that IscR binds to a site with an almost identical location as ORE-III and that nucleotides − 56 to − 51 were necessary for IscR binding , it is likely that IscR is the third oxidant-responsive regulator in the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Another surprising finding is that the IscR-binding site in PhyaA , PsufA , and PydiU is located in a similar position ( within 2 -- 3 bp ) with respect to the-35 hexamer of each promoter even though PhyaA is repressed by IscR and PsufA and PydiU are activated by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The ability of IscR to repress and activate from a similar location within the promoter may be due to the slight differences in spacing to the predicted -35 elements or the weak binding of IscR to the sufA and ydiU sites compared to the strong binding of IscR to the hya site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Effect of iscR mutation on ORE-III DNA binding in vitro To conﬁrm direct involvement of IscR in ORE-III-mediated regulation , we examined the effect of iscR deletion and complementation by IscR overproduction in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Rajagopalan , S. , Teter , S.J. , Zwart , P.H. , Brennan , R.G. , Phillips , K.J. , and Kiley , P.J. ( 2013 ) Studies of IscR reveal a unique mechanism for metal-dependent regulation of DNA binding speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Furthermore , the location of the putative binding sites upstream of iscR led us to hypothesize that IscR might regulate its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>If these represent sequences to which IscR binds , then these sites are in a position atypical for binding of a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Mutations in Fe-S-binding sites of IscR enabled sufA activation in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The analysis of IscR binding sites suggests two modes of DNA binding , one of which is speciﬁc for repression of the Fe-S biogenesis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Whether this indicates that the Fe-S form of IscR is speciﬁc for sites that function in repression of Fe-S biogenesis functions and that IscR lacking an Fe-S cluster recognizes the second class of binding sites awaits additional studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Thus , IscR is unusual in its ability to differentially bind its target sites depending on the presence or absence of its cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Thus , if IscR has DNA-binding domains organized in a manner similar to other members of this family , the presence of the Fe-S cluster might affect IscR repressor function by influencing its DNA binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Surprisingly , the DNA-binding isotherms yielded a sigmoidal saturation curve when plotted as a linear function of IscR concentration ( data not shown ) , indicating that binding of IscR to the 30-mer hya dsDNA sequence was cooperative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Further , because QacR DNA binding widens the major groove and undertwists the DNA , a mechanism was suggested in which binding of one dimer causes a conformational change in the DNA , which favors binding of a second dimer .30 In summary , given the similar DNA-binding properties of IscR to QacR , the simplest explanation to account for the data reported here is that two IscR dimers bind cooperatively to the hya DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In the presence of increasing amounts of IscR , binding increased as a linear function because proteins bind DNA in a 1:1 ratio when the DNA and protein are present at concentrations well over the apparent dissociation constant .10,11 At the concentration of IscR in which all of the DNA in the assay is bound ( indicated by the plateau ) , the stoichiometry of IscR binding to the DNA sequence was calculated to be 4.1 ± 0.4 wildtype IscR protomers per hya site DNA and 4.2 ± 0.2 IscR-C92A / C98A/C104A protomers per hya site DNA ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Thus , our data indicated that , unexpectedly , apo-IscR is a site-specific DNA-binding protein , and the role of apo-IscR needs to be considered in developing models for how IscR globally regulates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>We also note that the IscR binding site overlaps with that of ModE , which increases napF transcription in the presence of molybdate ( McNicholas and Gunsalus , 2002 ) , although it has not been established which start site ModE regulates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>For puriﬁed protein , 0.1 -- 50 pmol of His-tagged IscR protein was used in each binding reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Assuming that all of the protein in the preparation is active in binding , the simplest explanation to account for the solution behavior of IscR is that two IscR dimers are cooperatively binding the hya site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The region bound by IscR protein is shown , but the sequence speciﬁcity determinants for IscR binding to this site have not been deﬁned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>Despite this difference in DNA sequences , the occupancy of the upstream ( − 67 to − 13 ) sites was the same as the downstream ( +9 to +26 ) site upon titration of IscR , indicating either that these sites bind IscR equally well or that binding is cooperative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>How do four subunits of IscR bind to the hya site DNA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>First , one tetramer of IscR bound to a single operator DNA site , which would be similar to Lac repressor binding ( reviewed in Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>RT -- PCR analysis demonstrated that endogenous rnlA mRNA was reduced by overexpression of IscR and increased by deletion of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Finally , rnlA mRNA was reduced by overexpression of IscR and increased by deletion of iscR in vivo ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>As shown in Figure 5 , a shifted rnlA DNA probe was apparent in the presence of 50 nm IscR , although 200 nm IscR did not signiﬁcantly shift the rnlB DNA probe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>IscR can bind to rnlA promoter : Since the above results suggested that IscR speciﬁcally binds to the rnlA promoter , we performed gel mobility shift assays with different concentrations of puriﬁed His-tagged IscR and 32P-labeled DNA probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Furthermore , His-tagged IscR bound to DNA containing the rnlA promoter ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>-- Binding of His-tagged IscR to rnlA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Parallel samples were removed and analyzed for IscR protein levels by Western blotting to correlate β-galactosi-dase activity with IscR protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Immunoblot analysis showed that the level of IscR in the ΔcyaY mutant was higher than in the wt strain , consistent with reduced repressor activity and ruling out that reduced repressor activity of IscR was the result of decreased protein abundance ( Fig. 2E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Future work will aim to elucidate the mechanism of IscR repression , determine whether other targets exist for IscR , and study how the activity of this protein is modulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In fact , half-maximal PiscR repression required ∼ 9-fold more IscR protein aerobically than anaerobically ( 955 ± 93 nM vs. 107 ± 14 nM monomeric IscR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This , in turn , would lead to more PiscR repression and less IscR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Perhaps an additional mechanism exists in vivo to prevent repression of PiscR by IscR apo-protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The most important observation in our work is that by sRNA-induced differential degradation of the iscRSUA polycistron , the IscR protein is still produced even when iron is depleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscR protein concentrations are given from left to right in terms of nM total protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The protein concentrations of ArcA and IscR and the iron and sulﬁde content of IscR were determined colorimetrically , as described previously ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Interestingly , comparing studies in M9 medium supplemented or not with FeSO4 indicated that IscX contributes to IscR maturation solely in iron-rich conditions ( Fig. 8F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>( E ) Cell extracts of indicated strains , cultured in LB medium , were subjected to immunoblot analysis using antibodies raised against IscR , FNR , NuoF and NuoC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Such was the case when considering IscR and Sdh , as in the double ΔcyaYΔiscX mutant , the IscR and Sdh activities were lower than in the single parental mutant strains ( Fig. 8A and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>To check against this , and to assess the robustness of our predictions , we repeated our analysis including three further regulators , FNR , ArcA and IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>pRKC92AR , pRKC98AR , and pRKC104AR encoded alaninesubstituted IscR at Cys92 , Cys98 , and Cys104 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>These features suggest that IscR binds to this sequence for transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>No interactions were seen for IscR with any ISC proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983079" target="_blank">11983079</a>]<br></li>
<li>Concentrations of puri-ﬁed IscR proteins were determined by the Bradford method .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Results IscR Negatively Regulates iscR Expression in Vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR absorbance values were recorded at A280 and A230 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Transcription proﬁling of IscR + and IscR -- strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Novel features of IscR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>IscR Acts as a Repressor in Vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Also , the effect of CyaY on IscR maturation was found to be the same under aerobiosis and anaerobiosis ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Contribution of CyaY to the IscR , FNR , Nuo and Sdh maturation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>IscR , FNR , Nuo and Sdh maturation in ΔcyaY , ΔiscX and ΔcyaYΔiscX mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>However , although the β-galactosidase assays indicated that hybO is repressed by IscR , the magnitude of the regulation for the hybO promoter was lower than that observed when measuring transcript abundance via microarrays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR directly represses hyaA , hybO and napF transcription To test whether the effect of IscR at the hyaA , hybO and napF promoters was direct , in vitro transcription assays were performed ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Addition of IscR led to a decrease in transcript levels from the previously characterized hyaA and hybO transcription start sites , indicating that IscR directly repressed the known hyaA and hybO promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In agreement with the transcriptional proﬁling data , IscR decreased expression of the hyaA , hybO and napF promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Thus , two possible mecha ¬ IscR regulates Fe-S and oxygen-regulated genes 1065 nisms have emerged to explain the O2 and IscR regulation of the hyaA , hybO and napF promoters : in the case of the hyaA and hybO promoters , IscR represses expression of the previously known O2-regulated promoters , but in the case of napF , a previously unidentiﬁed promoter is subject to regulation by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The observation that C to A substitutions in IscR still allowed sufA induction in DoxyR background conﬁrms that IscR in its apo-form is able to activate sufA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Effect of cysteine to alanine substitutions in IscR on sufA induction in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>As demonstrated in Fig. 6 , addition of IscR dramatically increased the amount of sufA transcripts , conﬁrming its role as a direct activator of sufA transcription ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The results presented in Fig. 5 demonstrates that IscR binds to at least three sites in the upstream of the sufA promoter ( Fig. 5A , top and bottom strands ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The transcript initiating from the sufA promoter increased by four - to sixfold when IscR was added to the reaction , demonstrating that IscR directly activates transcription of the sufA promoter ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The increase in transcription was speciﬁc for the sufA promoter as no effect of IscR addition was observed for the control RNA-1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In addition , transcription from the rnlA promoter is negatively regulated by IscR ( Otsuka et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20980243" target="_blank">20980243</a>]<br></li>
<li>In addition , in vitro transcription reactions established a direct role for IscR in repression of the iscR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Taken together , these data indicate that IscR negatively regulates transcription from a promoter located upstream of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Because this increase does not require IscR , other transcription factors may function in this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>IscR directly activates ydiU , a gene of unknown function We examined the effect of IscR on ydiU expression as , like the sufA promoter , the transcription proﬁling data suggested that IscR activates this gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR directly activates sufA transcription in vitro Effect of puriﬁed IscR on sufA transcription was examined by in vitro transcription assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>These data suggested that IscR might activate nrdHIEF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>IscR binds to the promoter region of sufA and activates its transcription as a demetallated apo-form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>It appears that apo-IscR activates transcription of nrdHIEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>To further characterize IscR and determine whether it directly represses iscR transcription , we purified the protein from cells that overexpress IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>This result indicates that IscR directly represses transcription from its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR represses transcription from chromosomal rnlA : Finally , we conﬁrmed that IscR represses transcription of rnlA in the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>When the C92/98/104 A triple mutant form of IscR was puriﬁed and assessed for its activity in transcriptional activation in vitro , similar amount of activation was observed as described in Fig. 6 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>However , preliminary experiments suggest that oxidation of IscR does not cause any large changes in the ability of IscR to repress the iscR promoter in vitro ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>However , when the same promoters were tested by in vitro transcription assays , no effect of IscR was observed even at IscR protein concentrations of 2.2 µM ( Fig. 2A , data not shown ) ; a transcript from yjiH was not even detected in this assay despite the fact that the control RNA-1 transcript was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Additionally , in vitro transcription assays showed no effect of IscR addition on soxS transcription even when IscR was added to 2.2 µM ( Fig. 2A , data not shown ) , suggesting that IscR may not directly regulate the soxS promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>A similar analysis of the sodA promoter indicated that IscR increased expression from a sodA promoter -- lacZ fusion in vivo , but like the soxS promoter , the sodA promoter exhibited no change in in vitro transcript levels even upon addition of IscR to 2.2 µM ( Fig. 2A , data not shown ) , indicating that IscR regulation may be indirect or that an additional sodA transcription factor ( Compan and Touati , 1993 ) is necessary to observe an effect of IscR in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">IscR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Materials and Methods Strain construction Chromosomally encoded mutants of IscR in which cysteine residues were substituted with alanine were constructed in several steps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Previously constructed alleles encoding IscR mutants in which cysteine residues were substituted with alanine ( Nesbit et al. , 2009 ) were transduced into PK6364 using P1 vir and selecting for KanR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Effect of cysteine to alanine substitutions in IscR on sufA induction in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Site-directed mutagenesis of cysteine-to-alanine substitutions in iscR Site-directed mutagenesis of the cysteine residues in IscR was carried out with QuikChangeTM Site-directed Mutagenesis Kit ( Stratagene ) according to the manufacturer 's instruction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Assessment of metals in the puriﬁed IscR sample by inductively coupled plasma-atomic emission spectroscopy ( ICP-AES ) revealed that it does not contain any detectable level of iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Lanes 1 and 2 contain 800 nM and 400 nM phosphorylated ArcA , respectively ; lane 3 contains 200 nM as-puriﬁed IscR and 800 nM phosphorylated ArcA ; lane 4 contains 200 nM as-puriﬁed IscR ; lane 5 contains no protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>We also demonstrate that in the absence of isc , IscR-dependent suf activation is essential since strains lacking both the Isc pathway and IscR were not viable unless Suf was expressed ectopically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>IscR-dependent suf activation can partially restore holo-IscR activity when the Isc pathway is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor biosynthesis , and DNA-RNA metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative stress.6,11-14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>When the C92/98/104 A triple mutant form of IscR was puriﬁed and assessed for its activity in transcriptional activation in vitro , similar amount of activation was observed as described in Fig. 6 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Rrf2 from Desulfovibrio vulgaris , the founding member of this family , was discovered for its role in expression of the high-molecular-mass cytochrome redox protein complex .14 Other members include RirA , an iron-responsive regulator found in the symbiotic bacterium Rhizobium leguminosarum ,15 and NsrR , the nitric oxide sensor discovered in Nitrosomonas europaea .16 Most Rrf2-family members contain three Cys residues in the C-terminal region ‡ , similar to IscR , suggesting that they may ligate an Fe -- S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In this model proteins destined to contain [ Fe-S ] clusters establish synthetic homeostasis by effectively competing with IscR for the available clusters produced by the biosynthetic machinery .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>ple , the activation of IscR could be used to assess in vitro transfer of [ Fe-S ] clusters from the assembly machinery to the immature form of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>These possibilities are supported by the observation that the population of as-isolated IscR does not contain a full complement of [ 2Fe-2S ] clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>Thus , we considered the possibility that upregulation of the Suf machinery by apo-IscR and subsequent transfer of a Suf-generated [ 2Fe-2S ] cluster to IscR may explain the residual [ 2Fe-2S ] - IscR-dependent activity measured in the ΔiscSUA-hscBA-fdx strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Since in vitro DNA-binding by [ 2Fe-2S ] - IscR is not sensitive to the O2-dependent change in the [ 2Fe-2S ] cluster oxidation state ( Fleischhacker et al. , 2012 ) , a simple interpretation of our in vivo results is that the fraction of IscR containing a [ 2Fe-2S ] cluster may be higher under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Diagram of IscR showing the predicted helix -- turn -- helix motif ( http://www.uniprot.org/uniprot/ P0AGK8?id = P0AGK8 ) and a [ 2Fe -- 2S ] cluster ligation region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Since these are the only cysteine residues in IscR , it is likely , although not proven , that they provide three of the ligands to the [ 2Fe-2S ] cluster ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>How this is accomplished has not yet been evaluated , although it might be significant there are only three cysteine residues located within the E. coli IscR primary sequence , whereas four cysteines are usually involved in the coordination of a [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>Also , the elution profile of the [ 2Fe -- 2S ] cluster containing fractions of wild-type IscR , monitored at 420 nm , was similar to the protein fractions , monitored at 280 nm ( data not shown ) , providing additional evidence that the [ 2Fe -- 2S ] cluster does not alter the oligomeric state of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In addition , IscR had a reddish color , and the broad absorption features between 500 and 600 nm in the absorption spectrum were typical of a protein containing a [ 2Fe-2S ] cluster ( data not shown ) ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In addition , no difference was observed in the apparent binding affinity of wild-type IscR assayed under anaerobic conditions , which contains some reduced [ 2Fe -- 2S ] 1 + cluster ,1 versus under aerobic conditions , which rapidly oxidizes to the [ 2Fe -- 2S ] 2 + cluster .1 Since there was little difference in binding between preparations of the as-purified wild-type protein and the mutant lacking the cluster , these data argue that the Fe -- S form has a similar affinity for the hya site as apo-IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Thus , take together , these data indicate that IscR containing the [ 2Fe -- 2S ] cluster interacts similarly with the hya site as the mutant lacking the cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>However , since iscS and hscA are reported to have roles beyond Fe-S cluster assembly ( Py and Barras , 2010 , Johnson et al. , 2005 ) , we characterized the role of other members of the Isc pathway in supplying IscR its [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Finally , because active IscR can be abundantly produced in E. coli by using recombinant DNA methods , and the active form contains an [ Fe-S ] cluster , it should be possible to examine the role of each [ Fe-S ] cluster assembly protein in vivo by using whole-cell Mossbauer and electron paramagnetic resonance spectroscopies to monitor the assembly , and perhaps , disassembly of the [ 2Fe-2S ] cluster contained in IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>The regulatory mechanism for iscR transcription is not well characterized , except that the repressor activity of IscR depends on the Fe-S cluster formation , most likely a [ 2Fe-2S ] , as observed in anaerobically puriﬁed IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The observation that IscR activity is decreased in mutants lacking the Fe-S cluster assembly proteins , IscS and Hsc66 , has led us to consider that the ability of IscR to function as a repressor depends on whether it contains a [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Nonetheless , the finding that IscR activity is decreased in mutants lacking the Fe-S cluster assembly proteins , IscS and Hsc66 , has led us to consider that there may be a link between the levels of [ 2Fe-2S ] IscR and the rate of Fe-S cluster formation catalyzed by the Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>This situation could make it either more difficult to assemble the [ 2Fe-2S ] cluster within IscR relative to other [ Fe-S ] proteins or lead to instability of the cluster once inserted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>If IscR senses the cellular Fe-S status through its [ 2Fe-2S ] cluster , then conditions that increase the demand for Fe-S clusters are predicted to affect [ 2Fe-2S ] - IscR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In summary , these results demonstrate that IscR contains a [ 2Fe-2S ] cluster , but that these preparations are not completely occupied with Fe-S clusters , a finding not unusual for overexpressed Fe-S proteins ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Nevertheless , it is also possible that the oxidation state of the cluster further modulates IscR function as has been found with the [ 2Fe-2S ] - containing transcription factor , SoxR ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In this case , we would expect that IscR is a monomer like other members of this family ( 14 ) and contains one [ 2Fe-2S ] cluster , although this hypothesis has not been tested experimentally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>This model would also imply that the [ 2Fe-2S ] cluster would have one noncysteinyl ligand , because only three Cys are present in IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR Contains a [ 2Fe-2S ] Cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>The Role of the [ 2Fe-2S ] Cluster in Controlling IscR Function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Thus , determining whether the [ 2Fe-2S ] cluster occupancy of IscR differs between aerobic and anaerobic conditions is an important question to address .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>These features together , the optical spectra , the spin vs. sulfide stoichiometry , and the low spin relaxation rate , leave no doubt that IscR contains a [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>We have shown that IscR contains a [ 2Fe-2S ] cluster and is a transcriptional repressor of the iscRSUA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Our finding that IscR contains a [ 2Fe-2S ] cluster raises the question of how this cluster might function in regulation of IscR repressor activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Electron paramagnetic resonance data show that as-isolated IscR protein contains a [ 2Fe-2S ] cluster that can be reversibly oxidized and reduced ( Schwartz et al. , 2001 ) , a property characteristic of most Fe-S clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Analysis of IscR by electron paramagnetic resonance showed that the anaerobically isolated protein contains a [ 2Fe-2S ] 1 cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Anaerobically isolated IscR contains a [ 2Fe-2S ] + cluster ( Schwartz et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>First , a fraction of isolated IscR contains a [ 2Fe-2S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>Strains containing IscR-C92A , IscR-C98A , or IscR-C104A showed ∼ 2-fold higher levels of activation of PsufA-lacZ than did wild-type IscR ( Fig. 2a ) , indicating that the Fe -- S cluster is not required for the transcription-activation by IscR , in agreement with previous findings .7 The Fe -- S cluster of IscR was also dispensable for activation of PydiU since IscR-C92A , IscR-C98A , and IscR-C104A showed no defect in activation of PydiUlacZ ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Since most IscR-regulated promoters contain just a single IscR binding site ( Giel et al. , 2006 ) , perhaps the presence of multiple IscR binding sites within the PiscR region may be important to extend the dynamic range to regulate Fe-S biogenesis in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>This suppressive effect did not require the ligation of Fe-S cluster into IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_48">
Sequence-specific binding to a subset of IscR-regulated promoters does not require IscR Fe-S cluster ligation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></label>
<input id="dom_similarity_48", type="checkbox">
<div class="DOM_similar">
<li>Sequence-speciﬁc binding to a subset of IscR-regulated promoters does not require IscR Fe-S cluster ligation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li></div>
<li>The analysis of IscR binding sites suggests two modes of DNA-binding , one of which is speciﬁc for repression of the Fe-S biogenesis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Whether this indicates that the Fe-S form of IscR is speciﬁc for sites that function in repression of Fe-S biogenesis functions and that IscR lacking an Fe-S cluster recognizes the second class of binding sites awaits additional studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>These mutants contain one or more alanine substitutions of three of the four IscR [ 2Fe-2S ] cluster ligands ( C92 , C98 , C104 ) ( Fleischhacker et al. , 2012 ) and were previously shown to contain no detectable Fe-S cluster upon isolation ( Nesbit et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Cys92 , Cys98 , and Cys104 are predicted Fe-cluster ligands in IscR and indeed , IscR-C92A with an alanine substitution did not contain an Fe-S cluster , when it was puriﬁed under anaerobic condition ( Nesbit et al. 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Because IscR contained a series of Cys residues , we reasoned that it might contain a Fe-S cluster ; therefore , the protein was purified under anaerobic conditions by using the overexpression scheme previously shown to allow isolation of the FNR protein , which contains an oxygen-sensitive Fe-S cluster ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>It is important to recall that we tested IscR , a transcriptional regulator with an oxygen sensitive Fe-S cluster , an oxygen-resistant variant of FNR , another transcriptional regulator , and the Nuo and Sdh complexes that both contain buried oxygen resistant Fe-S clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Iron-sulfur ( Fe-S ) clusters are ancient and ubiquitous prosthetic groups involved in a wide variety of biological processes .1 In prokaryotes , the ISC ( iron-sulfur cluster ) biogenesis system , encoded by the isc operon ( IscR , IscS , IscU , IscA , HscB , HscA , and ferredoxin ) is mainly responsible for the ` house-keeping ' generation of iron-sulfur clusters .2 The complex interactions of these proteins are essential for efficient Fe-S cluster biogenesis .3 A very similar ISC system is found in the mitochondria of eukaryotes .4 IscU is the scaffold protein for cluster assembly and transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22734684" target="_blank">22734684</a>]<br></li>
<li>Thus whereas isc induction reﬂects disruption of overall Fe -- S cluster status and the inability of IscS , IscU and IscA to maintain the cluster of IscR , suf induction is a speciﬁc response to oxidative stress and iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>encoding Fe-S cluster assembly proteins IscR ( iron-sulfur cluster regulator ) is encoded by an ORF located immediately upstream of genes coding for the Escherichia coli Fe-S cluster assembly proteins , IscS , IscU , and IscA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscS , IscU , and IscA are responsible for Fe -- S cluster biogenesis and are thought to transfer these clusters to IscR ( Schwartz et al , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Dispensability of Fe-S-binding sites in IscR for activating sufA transcription IscR homologues in bacteria contain three highly conserved cysteine residues ( C92 , C98 and C104 in E. coli IscR ) which is thought to coordinate Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Interestingly , IscR contains a stretch of amino acids not found in MarA , which includes cysteine residues that may coordinate a Fe-S cluster ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>This is a particularly attractive hypothesis , because the three IscR cysteine residues , which may coordinate the Fe-S cluster , are located in a region that in MarA separates the two helix-turn-helix DNA-binding motifs ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Not only do our data underscore IscR 's critical role in adjusting expression of the Isc and Suf pathways to meet the cellular Fe-S cluster demand , they also support previous ﬁndings demonstrating that Fur plays an important role in maintaining Fe-S homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Because our data also suggest that the Fe-S cluster of IscR may be important for its function as a transcriptional regulator , we propose that IscR participates in an autoregulatory loop to maintain the appropriate levels of cellular Fe-S cluster synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR activity is decreased in strains in which Fe-S cluster assembly is impaired , and this finding suggests that IscR may be part of a novel autoregulatory mechanism that senses the Fe-S cluster assembly status of cells ( Frazzon and Dean 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>The ﬁnding that IscR activity is decreased in strain backgrounds in which Fe-S cluster assembly is impaired suggests that this protein may be part of a novel auto-regulatory mechanism that senses the Fe-S cluster assembly status of cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>The Fe-S cluster appears to be important for IscR function , because repression of iscR expression is signiﬁcantly reduced in strains containing null mutations of the Fe-S cluster assembly genes iscS or hscA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Strains containing null mutations in hscA lack Hsc66 and are defective in Fe-S cluster assembly , similar to strains containing an iscS deletion ( 3 , 5 ) , so we also assayed IscR activity in a strain containing an insertion in hscA ( hscA2-cat ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Thus , if IscR has DNA-binding domains organized in a manner similar to other members of this family , the presence of the Fe-S cluster might affect IscR repressor function by influencing its DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Therefore , we wanted to test whether the defects in Fe-S cluster assembly that occur in an iscS deletion strain would affect IscR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Therefore , IscR appears able to repress rnlA transcription regardless of the presence of an Fe-S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>IscR maintains Fe-S cluster homeostasis in the absence of the Isc pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>CyaY favors Fe-S cluster biogenesis in vivo In order to investigate the in vivo contribution of CyaY to Fe-S cluster biogenesis , we used IscR activity as a readout .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>Ligation of the Fe-S cluster with IscR was not required for suppression of RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>-- Effect of an Fe-S cluster in IscR on RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>IscR is a member of the helix -- turn -- helix Rrf2-family of proteins The Rrf2-family of transcription factors are widely distributed in bacteria but , overall , have not been well studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>For example , NsrR is also found in E. coli along with IscR ,17 but NsrR only matches three amino acids ( underlined ) in the predicted helix -- turn -- helix of IscR : 28-LADISER-QGIS-LSYLEQLFSRLRK-51 ‖ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>IscR is mainly a dimer in solution Since cooperative interactions are typically exerted through subunit interactions of oligomeric proteins , we investigated whether IscR was a dimer in solution , as is often the case for prokaryotic DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Taken together with the size-exclusion data , IscR appears to be mainly a dimer in solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Rather , the properties of IscR observed in this study are more consistent with those of QacR , IdeR , and DtxR , in which two dimers have been shown by X-ray crystallography and DNase I footprinting to contact longer operator sites ( ∼ 30 bp ) that are composed of two overlapping binding sites .24 -- 28 As observed with IscR , all of these proteins bind their sites cooperatively .24,26,29,30 Surprisingly , the cooperativity of DtxR , IdeR , and QacR binding to their DNA target does not arise from dimer -- dimer interactions since the dimers bind on different faces of the DNA helix .25,26,28 In addition , while QacR , DtxR , and IdeR bind symmetrical elements , each helix -- turn -- helix motif of QacR contacts different bases within each half site , resulting in asymmetric § http://ca.expasy.org/cgi-bin/prosite-search-ac ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>As expected from the low sequence similarity between the IscR and NsrR helix -- turn -- helix residues , the DNA-binding motif for NsrR in γ-proteobacteria ( gATGyATTTxAAATrCAtc ) 18 does not share obvious similarity with the Type 2 IscR-binding motif ( 1Axx-xCCxxAxxxXxxxTAxGGxxxT25 ; Table 1 ) , and only one promoter , PnapF , is common to both the IscR and NsrR regulons .2,19 Thus , while protein structure prediction algorithms suggest that Rrf2-family members share a common structural organization , divergence at the protein sequence level may explain their ability to recognize distinct DNA target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>We first established roles for the three predicted IscR binding sites in repressing PiscR transcription in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>How do four subunits of IscR bind to the hya site DNA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Thus , the correlation between binding strength and the number of matches to the motif provides additional evidence for the function of this sequence motif in IscR DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Our studies revealed unexpected DNA-binding properties of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Additional studies are necessary to determine the mechanism by which IscR recognizes different DNA-binding sites and which nucleotides in the two identiﬁed motifs are important for recognition by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Since a monomer of IscR is only 17 kDa with a single predicted helix -- turn -- helix motif ( Fig. 1 ) , the interaction with two apparently different DNA elements was unexpected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Furthermore , alignment of the sequences that were protected by IscR in the additional six promoter regions did not reveal a highly conserved inverted repeat sequence typical of sites recognized by helix -- turn -- helix DNA-binding domains , a motif also predicted to be in IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>We considered it a reasonable possibility that IscR might bind these sequences , because IscR contains regions similar to the helix-turn-helix DNA-binding domains of MarA ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">IscR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>IscR , a Member of the Mar Sox Rob Family , Functions as a Repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Rrf2 from Desulfovibrio vulgaris , the founding member of this family , was discovered for its role in expression of the high-molecular-mass cytochrome redox protein complex .14 Other members include RirA , an iron-responsive regulator found in the symbiotic bacterium Rhizobium leguminosarum ,15 and NsrR , the nitric oxide sensor discovered in Nitrosomonas europaea .16 Most Rrf2 family members contain three Cys residues in the C-terminal region ‡ , similar to IscR , suggesting that they may ligate an Fe -- S cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>NsrR belongs to the Rrf2 family [ 16 ] that includes the [ 2Fe-2S ] containing transcription factor IscR and the iron regulator RirA [ 17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18989365" target="_blank">18989365</a>]<br></li>
<li>NsrR is a member of the Rrf2 family of transcriptional regulators , which includes a putative Rrf2 regulator for a redox operon in D. vulgaris [ 25 ] , the iron-responsive repressor RirA , which controls iron uptake in rhizobia [ 26 ] , and the IscR repressor for the Fe-S cluster assembly operon in E. coli [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The characterized members of the PF02082 family are the Rrf2 repressor for the electron transport operon hmc in Desulfovibrio vulgaris [ 25 ] , the ironsulfur cluster repressor IscR in E. coli [ 27 ] , the iron-responsive regulator RirA in rhizobia [ 26 ] , and the nitrite-sensitive repressor NsrR for the nitrite reductase operon nirK in Nitrosomonas europeae [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Thus , if IscR has DNA-binding domains organized in a manner similar to other members of this family , the presence of the Fe-S cluster might affect IscR repressor function by influencing its DNA binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>At least one other Rrf2 family transcription factor , IscR , can also function both as a repressor and as a transcription activator ( 19 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Our finding that IscR is a repressor of iscRSUA does not exclude the possibility that IscR may also function as a transcription activator like other members of the AraC family ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR is a member of the helix -- turn -- helix Rrf2 family of proteins The Rrf2 family of transcription factors are widely distributed in bacteria but , overall , have not been well studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>The best-characterized member of the Rrf2 family is the IscR protein of E. coli , which contains a [ 2Fe-2S ] cluster , and regulates the transcription of isc genes , the products of which have roles in the biogenesis of [ Fe-S ] clusters ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>Second , the IscR protein bears primary sequence identity with a family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>IscR , a newly recognized member of the AraC family of transcriptional regulators and a `` distant '' MarA relative ( 20 % identity , 40 % amino acid similarity ) , was able to function in vitro as a transcriptional repressor of the iscRSUA operon ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>IscR is the first MarA Rob SoxS family member , to our knowledge , that has been shown to directly function as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR shares a similar amino acid sequence with MarA , a member of the Mar/Sox/Rob family of transcription factors ( Martin and Rosner 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>IscR shares amino acid similarity with MarA , a member of the MarA SoxS Rob family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by IscR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Reference or source Genea Descriptionb Wild fur type mutant FeSO4 acnB b0118 4.4 2.0 1.4 Aconitate hydrase B [ 4Fe-4S ] , major , exponential phase cueOc b0123 2.1 1.3 1.2 Multicopper oxidase , phenoloxidase and ferroxidase activities sdhC b0721 6.1 2.5 4.5 Succinate dehydrogenase , cytochrome b556 sdhD b0722 6.1 2.3 5.0 Succinate dehydrogenase , hydrophobic subunit sdhA b0723 4.0 2.6 3.0 Succinate dehydrogenase , ﬂavoprotein subunit sdhB b0724 5.1 2.8 1.9 Succinate dehydrogenase , 2Fe-2S , 3Fe-4S , 4Fe-4S cydA b0733 1.9 2.2 2.4 Cytochrome d terminal oxidase , polypeptide subunit I cydB b0734 1.7 1.8 2.2 Cytochrome d terminal oxidase , polypeptide subunit II pﬂA b0902 3.8 2.9 2.8 Pyruvate formate lyase activating enzyme 1 , 4Fe-4S acnAd b1276 1.7 1.3 3.8 Aconitate hydrase A ydbK b1378 2.9 2.8 1.3 Putative oxidoreductase , 4Fe-4S fumA b1612 7.6 3.8 3.2 Fumarase A fumarate hydratase Class I , 4Fe-4S ydhD b1654 3.0 3.1 1.7 Similar to some glutaredoxins sodB b1656 19.3 10.6 20.9 Iron superoxide dismutase yeaC b1777 2.0 2.3 1.6 Hypothetical protein msrB b1778 2.8 1.9 2.2 Methionine sulfoxide reductase mrp/apbC b2113 4.7 3.0 3.3 Fe-S cluster formation/repair in Salmonella enterica nuoN b2276 3.0 2.5 4.1 NADH dehydrogenase I chain N nuoM b2277 4.1 3.5 4.2 NADH dehydrogenase I chain M nuoL b2278 4.3 3.2 4.8 NADH dehydrogenase I chain L nuoK b2279 5.0 3.3 5.0 NADH dehydrogenase I chain K nuoJ b2280 5.5 3.9 6.6 NADH dehydrogenase I chain J nuoI b2281 8.2 3.5 7.7 NADH dehydrogenase I chain I , 4Fe-4S nuoH b2282 6.8 3.5 5.8 NADH dehydrogenase I chain H nuoG b2283 4.9 3.7 4.3 NADH dehydrogenase I chain G , 2Fe-2S , 4Fe-4S nuoF b2284 3.2 2.4 2.5 NADH dehydrogenase I chain F , potential 4Fe-4S nuoE b2285 3.6 2.5 3.0 NADH dehydrogenase I chain E , potential 2Fe-2S nuoC b2286 4.5 3.0 4.7 NADH dehydrogenase I chain C , D nuoB b2287 4.4 2.3 3.9 NADH dehydrogenase I chain B , potential 4Fe-4S nuoA b2288 3.6 2.5 3.3 NADH dehydrogenase I chain A sseB b2522 2.5 2.0 1.3 Enhanced serine sensitivity pepB b2523 2.7 2.1 2.5 Peptidase B iscX b2524 5.6 3.5 4.7 Open reading frame , hypothetical protein fdx b2525 4.3 3.6 3.1 Ferredoxin [ 2FE-2S ] , electron carrer protein hscA b2526 2.9 2.6 2.4 Heat shock protein , chaperone , member of Hsp70 protein family hscB b2527 3.7 2.6 4.4 Scaffold protein involved in iron-sulfur-cluster-assembly iscA b2528 3.5 2.3 2.5 Putative regulator iscU b2529 4.9 2.8 3.2 Scaffold protein involved in iron-sulfur-cluster-assembly iscS b2530 6.1 3.8 5.1 Subunit of cysteine desulfurase iscR b2531 1.6 1.2 1.3 IscR transcriptional regulator yggG b2936 3.0 2.1 2.7 Putative metalloprotease hybG b2990 0.9 1.1 1.1 Maturation of large subunit of hydrogenase-2 hybF b2991 0.9 2.8 2.3 May modulate levels of hydrogenase-2 hybE b2992 1.2 1.4 1.5 Member of hyb operon hybD b2993 1.2 0.9 1.4 Probable processing element for hydrogenase-2 hybC b2994 1.3 6.1 9.2 Large subunit , hydrogenase-2 hybB b2995 2.1 1.9 3.0 Probable cytochrome Ni/Fe component of hydrogenase-2 hybA b2996 2.3 1.6 2.1 Hydrogenase-2 small subunit , 4Fe-4S , 3Fe-4S hybO b2997 2.4 1.6 8.8 Small hydrogenase subunit , 4Fe-4S , 3Fe-4S fdoI b3892 3.8 2.6 3.4 Formate dehydrogenase , cytochrome b556 subunit fdoH b3893 5.6 3.1 5.2 Formate dehydrogenase-O , iron-sulfur subunit fdoG b3894 5.0 3.3 4.2 Formate dehydrogenase-O , major subunit frdD b4151 6.0 2.6 2.3 Fumarate reductase , anaerobic , membrane anchor polypeptide frdC b4152 10.7 3.3 2.5 Fumarate reductase , anaerobic , membrane anchor polypeptide frdB b4153 5.3 1.8 1.9 Fumarate reductase , anaerobic , iron-sulfur protein subunit frdA b4154 4.4 2.6 2.6 Fumarate reductase , anaerobic , ﬂavoprotein subunit a Individual genes or operons are grouped by lightface or boldface gene names .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16199566" target="_blank">16199566</a>]<br></li>
<li>B. Expression of sufAp -- lacZ in wild type , DoxyR , DiscR background and in DiscR provided with pTac-IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Because deprivation of intracellular iron is known to induce the expression of IscR , we measured the burst size of a T4 dmd mutant on MH1 cells in the presence of 0.25 mm 2 , 29-dipyridyl , a membrane-permeant ferrous iron chelator that causes deprivation of intracellular iron ( supporting information , Table S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>IscR may function as a regulator of mRNA-degradation ( T. Yonesaki , personal communication ) in addition to its regulatory role as a repressor of the isc operon ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>Thus , the ability of a common regulator , IscR , to coordinate transcription of both the Isc and Suf pathways by derepression and activation , respectively , is a vital homeostatic mechanism that ensures cell survival under a variety of growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>To incorporate a role for IscR , we would suggest that for the napF promoter , anaerobic activation of the downstream promoter ( s ) via FNR , a bona ﬁde anaerobic global regulator , and NarP , a nitrate-dependent activator ( Darwin et al. , 1998 ; Stewart et al. , 2003 ) , masks detection of IscR-dependent changes in expression from the upstream promoter ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>ArcA and AppY Antagonize IscR Repression of Hydrogenase-1 Expression under Anaerobic Conditions , Revealing a Novel Mode O2 Regulation of Gene Expression in Escherichia coli of A. D. Nesbit , * A. S. Fleischhacker , S. J. Teter , and P. J. Kiley Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , USA Transcription of the Escherichia coli hydrogenase-1 operon ( hyaABCDEF ) is increased by the transcription factors ArcA and AppY under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>To determine whether IscR had an effect on expression of these genes under anaerobic growth conditions , global transcriptional proﬁling experiments were carried out on strains grown anaerobically ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In sum , a combination of ArcA and AppY antirepression of IscR function , lower levels of IscR , and weak repression by [ 2Fe-2S ] - IscR leads to increased PhyaA expression under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>hyaA In this study , we examined the roles of IscR , ArcA , and AppY in regulating the expression of the hyaA promoter under aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA , AppY , and IscR act combinatorially to control hyaA promoter expression levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>NO is a damaging agent of iron-sulfur clusters , and apart from the wide variety of iron-sulfur-containing proteins involved in relevant pathways , the formation of these clusters is essential at least for the transcriptionally active form of regulators such as FNR , IscR , and SoxR ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>Furthermore , iscS is potentially important for global transcriptional regulation because several transcriptional regulators , such as SoxR [ 19 ] , IscR [ 20 ] , Fnr [ 21 ] , and YeiL [ 22 ] , carry Fe -- S clusters as an essential prosthetic group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor-biosynthesis , and DNA-RNA-metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative stress.6,11-14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 4.8 5.2 4.6 3.1 3.1 3.5 2.1 2.0 4.2 4.2 5.1 4.6 Oxidative stress and iron-sulfur-cluster-assembly genes Relevant regulatory proteins ahpC alkyl hydroperoxide reductase small subunit ahpF alkyl hydroperoxide reductase large subunit dadA D-amino acid dehydrogenase dadX alanine racemase OxyR * ( ) MetJ ( ) OxyR * ( ) MetJ ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) IscR * ( ) SoxRS * ( ) OxyR * ( ) OxyR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) OxyR * ( ) P1 , H-NS ( ) Lrp * ( ) NhaR ( ) RcsB ( P2 , H-NS ( ) Lrp * ( ) IscR * ( ) OxyR ( ) IHF ( ) Fur SoxR ( ) SoxS ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) 3.3 2.1 3.4 1.9 3.2 1.3 7.2 1.6 2.2 2.8 6.3 6.1 3.4 3.0 2.1 5.3 2.1 2.8 3.0 9.1 5.8 4.1 fdx ferredoxin fur ferric uptake regulator 2.1 3.4 2.4 2.8 3.3 10.1 7.0 5.1 2.0 1.9 1.8 2.7 1.8 1.8 2.8 2.5 11.2 5.7 4.4 1.8 1.1 3.6 grxA glutaredoxin 1 hscA co-chaperone involved in IscU iron-sulfur-cluster-assembly hscB co-chaperone involved in IscU iron-sulfur-cluster-assembly iscA iron-sulfur-cluster-assembly protein iscR transcriptional regulator iscS cysteine desulfurase iscU scaffold protein involved in iron-sulfur-cluster-assembly iscX possible role in iron-sulfur-cluster-biosynthesis katG bifunctional hydroperoxidase I osmC organic hydroperoxidase Within 5 min of air exposure the abundance of the soxS transcript is increased 8-fold ( Table 2 ) , indicating activation of the sensor SoxR , which enhances soxS expression , which in turn controls the SoxRS regulon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Thus , in addition to its putative role in transcript elongation ( 67 ) , it is possible that YacL might mod ¬ 1.8 2.5 ) pepB aminopeptidase B sodC Cu-Zn superoxide dismutase 2.1 2.0 2.3 soxS transcriptional activator sseB putative role in sulfur-metabolism sufA scaffold protein for iron-sulfur-cluster-assembly sufB component of Suf cysteine desulfurase activator complex sufC ATPase component of Suf cysteine desulfurase activator complex sufE sulfur acceptor that interacts and increases the activity of SufS sufS L-selenocysteine lyase and cysteine desulfurase tpx thioredoxin-dependent thiol peroxidase trxC thioredoxin udhA cytoplasmic transhydrogenase yhgI IscA-related protein 9.6 8.0 13.4 2.3 3.6 11.8 8.3 3.0 3.9 2.2 3.7 2.7 4.6 O O A O xyR xyR rcA xyR * ( ) IH * ( ) IH ( ) FN * ( ) F * ( ) F F * ( ) F R ( ) ur ( ) IscR * ( ) ur ( ) IscR * ( ) 2.6 5.8 2.1 8.0 1.9 14.7 2.1 1.6 1.7 1.6 2.1 TABLE 3 Transcripts encoding proteins involved in methionine-biosynthesis and putrescine utilization that exhibit altered abundance after the introduction of air to anaerobic cultures of E. coli MG1655 Fold regulation is the ratio of transcript levels at the indicated time after the introduction of air to the transcript levels at t 0 ( the initial anaerobic state ; p 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_17">
IscR acts as an activator in response-to-oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></label>
<input id="rp_similarity_17", type="checkbox">
<div class="RP_similar">
<li>IscR acts as an activator in response-to-oxidative-stress for the suf operon encoding Fe -- S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li></div>
<li>PLoS Biol 6 : e64 Yeo WS , Lee JH , Lee KC , Roe JH ( 2006 ) IscR acts as an activator in response-to-oxidative-stress for the suf operon encoding Fe -- S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_20">
( 18 ) Yeo , W. S. ; Lee , J. H. ; Lee , K. C. ; Roe , J. H. IscR acts as an activator in response-to-oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></label>
<input id="rp_similarity_20", type="checkbox">
<div class="RP_similar">
<li>Yeo , W.S. , Lee , J.H. , and Roe , J.H. ( 2006 ) IscR acts as an activator in response-to-oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li></div>
<li>This demonstrates that IscR positively contributes to induce the sufA gene as much as OxyR does in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Thus whereas isc induction reﬂects disruption of overall Fe -- S cluster status and the inability of IscS , IscU and IscA to maintain the cluster of IscR , suf induction is a speciﬁc response-to-oxidative-stress and iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>iscR is required for isc induction in response to H2O2 and 2,2 cents - dipyridyl To ascertain if the induction of the isc operon in response-to-oxidative-stress and iron starvation is mediated through the IscR repressor , we examined isc induction in response to H2O2 and 2,2 cents - dipyridyl in a mutant strain with a nonpolar deletion of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>This result indicates that the IscR repressor accounts for the transcriptional induction of the isc promoter in response-to-oxidative-stress and iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Hydrogen peroxide also affects the function of IscR , a transcriptional regulator that can coordinate a [ 2Fe-2S ] cluster ( Yeo et al. , 2006 ; Lee et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Brieﬂy , IscR is a [ 2Fe-2S ] transcriptional regulator that under its holo-form acts as a repressor of the isc operon ( Schwartz et al. , 2001 ; Giel et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>While this makes expression of the iscR promoter the key factor in determining the overall global regulatory impact of IscR , it also reveals the mechanistic complexity of this system and underscores the need to determine the requirement for [ 2Fe -- 2S ] - IscR for each promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>MALDI-TOF mass analysis identiﬁed it with IscR , known to serve as a repressor of the iscRSUA gene-expression under anaerobic condition as a [ 2Fe-2S ] - bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>IscR also appears to increase the expression of a small subset of genes , including sufABCDSE ; argE , involved in arginine-biosynthesis ; and sodA , the Mn-containing superoxide dismutase , as the transcript intensities decreased by at least twofold in this assay ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Connection of other O - regulated genes to 2 the IscR regulon In addition to the hya , hyb and nap operons , a recent global gene-expression study ( Kang et al. , 2005 ) indicates that the expression of other candidate promoters identiﬁed in this study ( ﬂu , sufABCDSE , sodA , yjiH and ﬁmICDFGH ) is also regulated by O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Keywords IscR ; Fe-S biogenesis ; Isc ; Suf ; autoregulation INTRODUCTION Iron-sulfur ( Fe-S ) proteins are nearly ubiquitous across all domains of life and carry out many important cellular functions such as respiration , photosynthesis , nitrogen-fixation , and gene regulation ( Beinert , 2000 , Fontecave , 2006 , Kiley and Beinert , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>IscR also directly activates the expression of the alternate Fe-S biogenesis pathway encoded by the sufABCDSE ( suf ) operon ( Nesbit et al. , 2009 , Yeo et al. , 2006 ) , which includes proteins that exhibit functional similarity to those of the Isc pathway ( Py and Barras , 2010 , Py et al. , 2011 , Johnson et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The Suf pathway has been suggested to function under conditions of oxidative and iron stress ( Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) via OxyR and Fur , respectively , but the ﬁnding that IscR also controls the Suf pathway suggests that this alternate pathway is involved in Fe-S biogenesis under normal aerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Recent work showed that IscR regulates about 40 genes involved in anaerobic-respiration , oxidative stress response , Fe -- S biogenesis , and other metabolic pathways ( Giel et al , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Not only do our data underscore IscR 's critical role in adjusting expression of the Isc and Suf pathways to meet the cellular Fe-S cluster demand , they also support previous ﬁndings demonstrating that Fur plays an important role in maintaining Fe-S homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Together , our data indicate that [ 2Fe-2S ] - IscR regulates expression of the Isc pathway via a negative feedback mechanism based on the cellular Fe-S demand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>R and Y are puRine and pYrimidine , respectively ; S is G or C ; W is A or T ; K is T or G ; and M is A or C. Discussion A broader role for IscR in regulating Fe-S biogenesis Our ﬁnding that IscR , in addition to negatively regulating the Isc Fe-S biogenesis pathway , also positively regulates an alternative Fe-S biogenesis pathway , Suf , suggests that IscR has a more general role in regulation of this key metabolic-process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>We report the role of other Isc pathway components and the Suf system in providing Fe-S clusters to IscR under aerobic and anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The system also holds promise for the development of in vitro DNA-directed formation of IscR as a way to test whether molecular chaperones or other components are involved in the maturation of [ Fe-S ] proteins posttranslationally or cotranslationally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11752417" target="_blank">11752417</a>]<br></li>
<li>The ISC operon is controlled by IscR , a Fe-S cluster binding transcriptional repressor , and contains genes for a cysteine desulfurase ( IscS ) , scaffold protein ( IscU ) , ferredoxin ( Fdx ) , IscA , molecular chaperones ( hscA and hscB ) , and IscX ( also known as yfhj and ORF3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22352884" target="_blank">22352884</a>]<br></li>
<li>From these results , we conclude that IscR negatively regulates transcription of rnlA and represses RNase LS activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>In wild-type cells treated with dip , RyhB expression reduces the expression of non-essential iron-using proteins , leaving the available iron to essential proteins ( such as IscR , repressing the iscRSUA transcription after 15-20 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Role of IscR in regulation of iron -- sulfur biogenesis in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>IscR controls iron-dependent biofilm-formation in Escherichia coli by regulating type I fimbria expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23666921" target="_blank">23666921</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>]<br></li>
<li>Here , we describe the mechanism by which IscR , a [ 2Fe-2S ] cluster-containing regulator of Escherichia coli , adjusts the synthesis of the Isc Fe-S biogenesis pathway to maintain Fe-S homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Giel , J.L. , Nesbit , A.D. , Mettert , E.L. , Fleischhacker , A.S. , Wanta , B.T. , and Kiley , P.J. ( 2013 ) Regulation of ironsulphur cluster homeostasis through transcriptional-control of the Isc pathway by [ 2Fe-2S ] - IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_48">
Regulation of iron-sulphur cluster homeostasis through transcriptional-control of the Isc pathway by [ 2Fe-2S ] - IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></label>
<input id="rp_similarity_48", type="checkbox">
<div class="RP_similar">
<li>Regulation of iron-sulfur cluster homeostasis through transcriptional-control of the Isc pathway by [ 2Fe-2S ] - IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li></div>
<li>Therefore , E. coli employs two major global regulators , IscR and Fur , to integrate Fe availability with Fe-S cluster homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>NsrR is a member of the Rrf2 family of transcriptional regulators , which includes a putative Rrf2 regulator for a redox operon in D. vulgaris [ 25 ] , the iron-responsive repressor RirA , which controls iron uptake in rhizobia [ 26 ] , and the IscR repressor for the Fe-S cluster assembly operon in E. coli [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>This cleavage gives rise to the remaining 50-section of the transcript encoding IscR , a transcriptional regulator responsible for activation and repression of several genes depending on the cellular Fe -- S level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscR regulates a diverse set of genes , including some encoding Fe-S proteins as well as genes encoding additional Fe-S biogenesis proteins Thirty-one genes showed an increase in transcript levels in the strain lacking IscR compared with the wild-type strain under aerobic conditions , indicating that these genes are candidates for repression by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Whether IscR regulates other functions involved in Fe-S biogenesis had not previously been addressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Surprisingly , IscR activates the Suf pathway yet represses the other Fe-S biogenesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Results Identiﬁcation of IscR-regulated genes in E. coli K-12 Candidate IscR-regulated genes were identiﬁed by comparing the transcription proﬁles of strains PK6597 ( an IscR -- mutant containing a non-polar deletion of iscR ) and its E. coli K-12 parent , MG1655 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Finally , IscR decreased the expression of dctA , encoding a C4-dicarboxylate-transporter ; soxS , encoding a transcription factor that activates genes in response-to-superoxide stress ; ﬁmAICDFGH and ﬂu , two operons encoding surface structures ; and several operons with genes of unknown function : ycgZ , the putative yjiH-yjiG-iadA-yjiE operon , yjiM and yqjI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR decreased expression of soxS , encoding a general regulator of oxidative stress , but increased expression of a superoxide dismutase The levels of transcription of two genes involved in the response-to-oxidative-stress also were affected by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>To analyze the effect of the loss of IscR , IscS , and HscA on isc gene-expression , null mutations of the genes encoding these proteins ( Table 1 ) were introduced into the appropriate strains by P1 transduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>coli .1,2 IscR was first discovered for its role in regulating expression of the Isc ( iron -- sulfur cluster ) biogenesis pathway ( reviewed in Refs .3 and 4 ) , which is transcribed as an operon ( iscRSUA-hscBA-fdx ) that includes iscR .1 The finding that IscR requires a functional Isc Fe -- S biogenesis pathway to repress the iscR promoter ( PiscR ) linked IscR [ 2Fe -- 2S ] cluster occupancy to its transcription function , suggesting that Fe -- S biogenesis is controlled by a homeostatic mechanism directed by [ 2Fe -- 2S ] - IscR .1 Recent genome-wide transcription profiling data extended these initial observations to show that IscR also controls the expression of a subset of anaerobic Fe -- S-containing enzymes [ hydrogenase-1 ( Hya ) , hydrogenase-2 ( Hyb ) , and the periplasmic nitrate reductase ( Nap ) ] , in addition to other proteins with Fe -- S biogenesis functions [ YadR ( now ErpA ) , YhgI Introduction IscR , a [ 2Fe -- 2S ] cluster-containing transcription factor , acts as a repressor or activator to control the expression of more than 40 genes in Escherichia 0022-2836 / $ - see front matter © 2009 Elsevier Ltd. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Other genes in the operon encode a [ 2Fe-2S ] ferredoxin ( Fdx ) and the transcription factor IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The global regulator IscR coordinates cluster synthesis by repressing transcription of the isc operon by [ 2Fe-2S ] - IscR and activating expression of the suf operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The isc operon also encodes IscR , a [ 2Fe-2S ] transcription factor that modulates expression of the Isc pathway according to the cellular needs for Fe-S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>While IscR was first discovered for its role in regulating transcription of the Isc system ( Schwartz et al. , 2001 ) , subsequent genome-wide transcription profiling data revealed that IscR controls the expression of more than 40 genes in E. coli ( Giel et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>While neither the mechanism nor the regulation of Fe-S biogenesis is well characterized , it has been established that , in Escherichia coli , IscR ( iron-sulphur cluster regulator ) represses the transcription of iscRSUA-hscBA-fdx , the operon encoding the housekeeping Fe-S biogenesis system ( Isc ) and the regulator IscR ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In its holoprotein form , IscR [ 2Fe-2S ] represses a broad set of genes that includes those that encode the housekeeping iron-sulfur-cluster assembly system ( iscSUA-fdx-hscAB ) ( Schwartz et al. , 2001 ; Giel et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Possible involvement of IscR , which negatively regulates the iscSUA genes encoding another Fe-S assembly system in E. coli , can be considered , based on the possibility that IscR , being a 2Fe-2S protein , could respond to oxidants ( Schwartz et al. , 2001 ; Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>In its apoprotein form , IscR is a positive activator of expression of other genes , including the alternative iron-sulfur-cluster assembly system that is encoded by the sufABCDSE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>In summary , this study extends the role of IscR in the regulation of Fe-S cluster biogenesis in E. coli to include the suf operon as well as the genes encoding the putative Fe-S cluster assembly proteins , YadR and YhgI , and suggests a similar function of IscR in other related bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In this study , we found that a previously uncharacterized regulator , IscR , regulates expression of the genes encoding these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>For instance , a number of genes encoding Fe -- S containing enzymes are coordinately regulated by Fe -- S containing IscR ( Fleischhacker et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_72">
IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></label>
<input id="rp_similarity_72", type="checkbox">
<div class="RP_similar">
<li>IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12486045" target="_blank">12486045</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16199566" target="_blank">16199566</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>IscR , an Fe -- S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe -- S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_75">
Proc Natl Acad Sci USA 97 : 9009 -- 9014 Schwartz CJ , Giel JL , Patschkowski T , Luther C , Ruzicka FJ , Beinert H , Kiley PJ ( 2001 ) IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></label>
<input id="rp_similarity_75", type="checkbox">
<div class="RP_similar">
<li>Proc Natl Acad Sci USA 97 : 9009 -- 9014 Schwartz CJ , Giel JL , Patschkowski T , Luther C , Ruzicka FJ , Beinert H , Kiley PJ ( 2001 ) IscR , an Fe -- S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe -- S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_77">
Schwartz CJ , Giel JL , Patschkowski T , Luther C , Ruzicka FJ , et al. ( 2001 ) IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></label>
<input id="rp_similarity_77", type="checkbox">
<div class="RP_similar">
<li>Schwartz , C. J. , J. L. Giel , T. Patschkowski , C. Luther , F. J. Ruzicka et al. , 2001 IscR , an Fe -- S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe -- S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T , Luther , C , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S cluster-con-taining transcription factor , represses expression ofEscherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983079" target="_blank">11983079</a>]<br></li>
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T. , Luther , C. , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S clus-ter-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T. , Luther , C. , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li></div>
<li>IscR-dependent gen expression links iron-sulphur e o the Summary IscR is an iron-sulphur ( Fe-S ) cluster-containing transcription factor that represses transcription of the operon containing its own gene and the iscSUA-hscBA-fdx genes , whose products are involved in Fe-S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>An Fe-S cluster protein , IscR , acts as a transcription factor and controls the expression of genes that are necessary for Fe-S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">IscR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Thus , we used a computational approach guided by the DNA footprinting results to identify sequences recognized by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Further systematic studies with extensive mutations are necessary to reveal the recognition sequences of apo-IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>In addition , the presence of two well-deﬁned regions separated by roughly 20 bp suggests that multiple oligomers of IscR may bind to the iscR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>We found no consensus recognition sequence of known Fe -- S cluster-containing regulators , namely , Fnr [ 21 ] , YeiL [ 22 ] , SoxRS [ 44 ] , and IscR [ 20 ] , in the promoter regions of udp , cdd , and yeiT -- yeiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18482579" target="_blank">18482579</a>]<br></li>
<li>( 1 ) ] indicated that wild-type IscR and IscR-C92A / C98A/C104A had Hill coefficients of 3.8 and 4.0 , respectively , establishing that both forms of IscR bind cooperatively to this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>These features suggest that IscR binds to this sequence for transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>C ) Fraction bound corrected for fluorescence quenching of labeled PiscR sites A or B bound by [ 2Fe-2S ] - IscR as a function of protein concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Interestingly , the NsrR homologue IscR also binds to two types of site ( Type 1 and Type 2 ) , though in this case they are unrelated sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>IscR also binds efﬁciently to a region between -132 and -164 ( site 2 ) which overlaps with ORE-II site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The λ-PiscR-lacZ reporter fusion ( − 476 to +74 bp relative to the IscR start codon ) ( Schwartz et al. , 2001 ) is located within the λatt site , whereas the PiscR-lacZ reporter fusion ( − 428 to +144 bp relative to the IscR start codon ) replaces the wildtype lac promoter and was constructed by a method previously described ( Giel et al. , 2006 , Kang et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>In vitro transcription assays showed that IscR directly activates transcription of ydiU ( Fig. 2B ) , and primer extension assays identiﬁed the transcription start site 10 nucleotides upstream of the ATG start codon ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Inspection of the hybO regulatory region revealed a site from + 2 to +27 , matching seven of the nine nuc-leotides within the recognition motif ( Fig. 8a ) , bu this site was not observed in the IscR footprints ,12 which only visualized from − 5 to − 197 relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>At the napF promoter , IscR bound a sequence from − 130 to − 102 relative to the previously identiﬁed P1 transcription start site ( − 19 to +19 relative to the novel P3 napF transcription initiation site ) with a hypersensitive site at − 122 , and at higher protein concentrations , partial protection from − 152 to − 102 was observed ( Figs 3C and 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In this case , it seems likely that IscR may act by blocking RNA polymerase from binding to the promoter elements of the novel start site or from clearing the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The regions protected by IscR are indicated with vertical lines and numbers indicate the distance relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In contrast , DNase I footprinting results have shown that IscR binds up to three regions within PsufA , with the strongest binding region located at bp 60 to 26 relative to the transcription start site ( TSS ) , which encompasses a type 2 IscR binding site ( designated region 1 ) ( Fig. 1 ) ( 14 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Using a wild-type DNA fragment for comparison ( Fig. 1B ) , DNase I footprinting showed that when site A was mutated , anaerobically isolated [ 2Fe-2S ] - IscR no longer protected the region ( − 67 to − 41 bp relative to the +1 iscR transcription start site ) encompassing site A , but still protected the regions ( − 40 to − 14 bp and +9 to +26 bp ) encompassing sites B and C in a protein concentration-dependent manner ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>To identify the region of the sufA promoter controlled by IscR , either nucleotides − 393 to +90 ( containing the previously described OxyR binding site located from − 254 to − 198 ; Zheng et al. , 2001 ) or − 200 to +40 relative to the transcription start site ( lacking the OxyR binding site ) was fused to lacZ on the chromosome , and β-galactosidase activity was assayed from cells grown under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>For the hyaA promoter , IscR protected a region from − 55 to − 29 relative to the transcription start site with a hypersensitive site at − 36 , indicating a bend or structural change in the DNA ( Figs 3B and 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Although the exact location of the IscR-binding site ( s ) within the iscR promoter region is unknown , it must reside within the region from 161 to 38 nucleotides relative to the start site of transcription , because this region was sufficient for IscR to decrease transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>DNase I footprinting localized the IscR binding sites to − 40 to +3 nucleotides relative to the A transcription start site of the yadR promoter ( Figs 3D and 4D ) and to − 8 to +19 nucleotides relative to the T transcription start site of the yhgI promoter ( Figs 3E and 4E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Using DNase I footprinting , an IscR binding site was identiﬁed upstream of the sufA promoter , spanning nucleotides − 57 to − 28 relative to the transcription start site ( Figs 3A and 4A ) , with a weakly hypersensitive site at − 57 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>DNase I footprinting indicates that IscR binds a region located from − 58 to − 28 relative to the transcription start site ( Figs 3F and 4F ) , a region with similar spacing to that of the IscR binding site in the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>( a ) Comparison of IscR-binding sites to the IscR-binding motif with the original Type 2 sequence logo as a reference .2 The number listed after the promoter name is the most upstream base of the IscR site relative to the transcription start site ( P3 of PnapF is − 121 bp relative to P1 in P 2 napF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>One region of protection , which encompasses nucleotides − 67 to − 14 relative to the +1 transcription start site , includes two Type 1 IscR binding sites : site A ( − 65 to − 41 ) and site B ( − 40 to − 16 ) ( Giel et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>QuikChange ( Stratagene ) was used to disrupt IscR binding to the sufA promoter region by de-leting bp 56 to 51 or by replacing the C at bp 50 with a G , forming pPK8644 and pPK11186 , respectively ; similarly , the Fur binding site was disrupted ( 26ATA 24 changed to 26TAT 24 ) to form pPK10874 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Given that we found that IscR binds to a site with an almost identical location as ORE-III and that nucleotides − 56 to − 51 were necessary for IscR binding , it is likely that IscR is the third oxidant-responsive regulator in the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Puriﬁed IscR that lacked Fe-S cluster bound to the ORE-III site and activated transcription from the sufA promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Whether this indicates that the Fe-S form of IscR is speciﬁc for sites that function in repression of Fe-S biogenesis functions and that IscR lacking an Fe-S cluster recognizes the second class of binding sites awaits additional studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Of the promoters shown to be directly regulated by IscR , only PnapF and PhybO were not classified as either a Type 1 or a Type 2 promoter .2 Comparison of the IscR footprint at PnapF from Giel et al. 2 with the IscR-binding motif showed that the region contains five of the nine nucleotides ( Fig. 8a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>None of the footprinted sites contained the marbox element previously noted ( Schwartz et al. , 2001 ; Martin et al. , 2002 ) in the − 82 to − 63 region of the iscR promoter , indicating that these sequences are not relevant to IscR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Since sites A and B overlap sequences ( − 65 to − 16 bp ) of the − 35 and extended − 10 promoter elements required for σ70 binding ( Record et al. , 1996 ) and a region potentially encompassing an UP element important for recognition by αCTD ( Gourse et al. , 2000 ) , it appears that IscR binding of both Type 1 sites is required to fully prevent RNA polymerase from binding PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Because IscR regulation of promoters containing type 2 sites appears not to require the ligation of an Fe-S cluster , we searched the sequence around the rnlA promoter for a sequence homologous to type 2 sites and found a sequence with 65 % ( 17/26 bp ) homology to the type 2 consensus ( Figure 7A ) , which also partly matches `` AxxxCCxxAxxxxxxxTAxGGxxxT '' suggested for IscR-binding motif ( Nesbit et al. 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>The nucleotide sequences vary between Type 1 ( Giel et al. , 2006 ) and Type 2 ( Nesbit et al. , 2009 ) sites , and it is not yet clear how a dimer of IscR distinguishes between them ( Nesbit et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The remaining IscR site , located from +9 to +26 , does not share sequence similarity with other known IscR binding sites and is referred to as PiscR site C ( Giel et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Nucle-otides matching the derived IscR consensus sites are highlighted in green .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Identifying additional IscR sites using genomic analysis should provide further support for the role of this sequence element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>IscR and downstream ArcA sites are boxed in the E. coli sequence ( top ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Sequence logos for the IscR binding sites identiﬁed for ( A ) the iscR , yhgI and yadR genes ( Type 1 ) ; and for ( B ) the sufA , ydiU and hyaA genes ( Type 2 ) from E. coli and related Enterobacteria generated using the SIGNALX program and WebLogo package .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In comparison , both the sufA and the ydiU sites contain a G in that position because PsufA and PydiU predicted -35 elements are 3 and 2 bp , respectively , further downstream with respect to the IscR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Interestingly , the distances between the IscR motifs and transcription start sites of sufA and ydiU are almost the same , 29 and 30 nucleotides respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Additional studies are necessary to determine the mechanism by which IscR recognizes different DNA binding sites and which nucleotides in the two identiﬁed motifs are important for recognition by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Determining the nucleotides within the hya site critical for IscR binding Inspection of the logo for Type 2 sites does not readily reveal any single or overlapping symmetrical DNA-binding elements ( Fig. 8 and Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>If these represent sequences to which IscR binds , then these sites are in a position atypical for binding of a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Marked with asterisks are nucleotides of the IscR and Fur binding sites that were deleted or mutated in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>A ) Sequences of IscR binding sites A ( underlined ) and B ( double underlined ) within PiscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Despite this difference in DNA sequences , the occupancy of the upstream ( − 67 to − 13 ) sites was the same as the downstream ( +9 to +26 ) site upon titration of IscR , indicating either that these sites bind IscR equally well or that binding is cooperative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR contributes to the O2 regulation of several genes IscR appears to bind to two different classes of DNA sites The use of bioinformatic methods in addition to DNase I footprinting of IscR binding sites was critical in identifying conserved sequences for binding of IscR as it appears that IscR can recognize two different DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The region bound by IscR protein is shown , but the sequence speciﬁcity determinants for IscR binding to this site have not been deﬁned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>The sequence of the protected site indicated that it belongs to the type 2 class of IscR binding sites , which binds with similar afﬁnities to either the [ 2Fe-2S ] cluster or the apo ( clusterless ) form of IscR ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>The two classes of IscR-binding sites were inferred from DNase I footprinting of six IscR-regulated promoters and were further supported by phylogenetic comparisons .2 A 25-bp Type 1 site ( ATASYYGACTRwwwYAGTCRRSTAT ) was compiled from the IscR target sites in the promoter regions of iscR , yadR , and yhgI , whereas a 26-bp Type 2 site ( AWARCCCYTSnGTTTGMnGKKKTKWA ) was compiled from the IscR sites upstream of the promoters for hyaA , ydiU , and sufA .2 For either Type 1 or Type 2 sites , IscR protects a region of ∼ 28 -- 30 bp , and each promoter contained a single site except for the iscR promoter that contained three binding sites .2 The length of the footprint is longer than expected for most dimeric helix -- turn -- helix DNA-binding proteins that recognize a single palindromic sequence , and neither site revealed a common symmetrical element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Furthermore , alignment of the sequences that were protected by IscR in the additional six promoter regions did not reveal a highly conserved inverted-repeat sequence typical of sites recognized by helix -- turn -- helix DNA-binding domains , a motif also predicted to be in IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>A string of two such palindromic sites were located in the iscR promoter within the upstream IscR-protected region ( from − 65 to − 41 and − 40 IscR regulates Fe-S and oxygen-regulated genes 1067 to − 16 ) and both of them were highly conserved in 10 other species of Enterobacteria ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The results presented in Fig. 5 demonstrates that IscR binds to at least three sites in the upstream of the sufA promoter ( Fig. 5A , top and bottom strands ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Neither form of IscR bound the DNA containing a rando-mized sequence ( Fig. 3 ) , which indicates that both preparations of IscR bind the hya site in a sitespecific manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In the presence of increasing amounts of IscR , binding increased as a linear function because proteins bind DNA in a 1:1 ratio when the DNA and protein are present at concentrations well over the apparent dissociation constant .10,11 At the concentration of IscR in which all of the DNA in the assay is bound ( indicated by the plateau ) , the stoichiometry of IscR binding to the DNA sequence was calculated to be 4.1 ± 0.4 wildtype IscR protomers per hya site DNA and 4.2 ± 0.2 IscR-C92A / C98A/C104A protomers per hya site DNA ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>In view of the stoichiometry data of four subunits of IscR bound per DNA site ( Fig. 5 ) , we also considered the possibility that the hya site might be composed of two imperfect overlapping palindromes to accommodate the binding of two IscR dimers and the large DNase I footprint .2 Two sequences were constructed to test whether symme-trical combinations of potential adjacent half-sites TCCA and TACA improved binding of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Neither palindrome 1 , containing two adjacent TACA halfsites , nor palindrome 2 , containing two adjacent TCCA half-sites , bound IscR as well as the wild-type sequence , requiring 7 - and 35-fold more DNA for 50 % inhibition of binding , respectively ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Taken together , these data suggest that , in addition to -38 A , IscR can recognize key palindromic nucleotides within a larger palindrome ( 1AAATCCACACAGtTTGTATGGTTTT25 ; bold , key nucleotides ; underlined , palindrome ; lowercase , center of symmetry ) but leaves open the question of how four subunits of IscR bind this site since this sequence has only one axis of symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Analysis of mutations in the hya site using DNA competition assays showed that apo-IscR most likely recognizes an imperfect palindrome within the hya promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>IscR binds to an imperfect palindrome within the hya promoter Since the two most critical nucleotides , -50 C and -38 A , are not in the palindrome , we determined whether binding would increase by extending the symmetry to include these positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by IscR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>It did not respond to manganese availability ; it was not induced by redox-cycling drugs , which indicates that it is not regulated by SoxR ( 16 , 47 ) ; it was not affected by Fur , IscR , Hfq , RpoS , or YaaA itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Cell extracts from exponentially grown DoxyR cells treated with 30 mM phenazine methosulphate ( PMS ) for 1 h were partially puriﬁed through heparin and Q-sepharose column chromatographies as described Positive regulation of the suf operon by IscR 207 in Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Results IscR Negatively Regulates iscR Expression in Vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>RT -- PCR analysis demonstrated that endogenous rnlA mRNA was reduced by overexpression of IscR and increased by deletion of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Finally , rnlA mRNA was reduced by overexpression of IscR and increased by deletion of iscR in vivo ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>Other promoters repressed by IscR in vivo We also examined whether the promoters for yqjI and the putative yjiH-yjiG-iadA-yjiE operons were directly repressed by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>OxyR and IscR contributed almost equally to activate the sufA operon in response to oxidants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>This demonstrates that IscR positively contributes to induce the sufA gene as much as OxyR does in response to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>These results support the proposal that IscR positively acts to induce sufA expression in response to PMS via ORE sites , most likely ORE-III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>These results showed that expression of the iscR - lacZ fusion increased 17-fold in the absence of IscR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In vitro transcription assays showed that IscR directly repressed transcription from the same promoter regions of yadR and yhgI that were fused to lacZ ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>It appears that apo-IscR activates transcription of nrdHIEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>To further characterize IscR and determine whether it directly represses iscR transcription , we purified the protein from cells that overexpress IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>This result indicates that IscR directly represses transcription from its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Our data indicate that IscR represses transcription initiation from a single promoter upstream of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>In wild-type cells treated with dip , RyhB expression reduces the expression of non-essential iron-using proteins , leaving the available iron to essential proteins ( such as IscR , repressing the iscRSUA transcription after 15-20 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Thus , two possible mecha ¬ IscR regulates Fe-S and oxygen-regulated genes 1065 nisms have emerged to explain the O2 and IscR regulation of the hyaA , hybO and napF promoters : in the case of the hyaA and hybO promoters , IscR represses expression of the previously known O2-regulated promoters , but in the case of napF , a previously unidentiﬁed promoter is subject to regulation by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>While isc and suf are both direct targets of the IscR regulon and are expressed in response to similar signals ( 11 -- 15 ) , IscR controls expression of these two operons by different mechanisms : IscR represses isc expression , whereas IscR activates transcription of suf ( 14 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The repression of the transcription factor IscR may be part of the observed O2-dependent expression of the iscR regulon ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Finally , IscR decreased the expression of dctA , encoding a C4-dicarboxylate transporter ; soxS , encoding a transcription factor that activates genes in response to superoxide stress ; ﬁmAICDFGH and ﬂu , two operons encoding surface structures ; and several operons with genes of unknown function : ycgZ , the putative yjiH-yjiG-iadA-yjiE operon , yjiM and yqjI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR decreased expression of soxS , encoding a general regulator of oxidative stress , but increased expression of a superoxide dismutase The levels of transcription of two genes involved in the response to oxidative stress also were affected by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>When sufﬁcient amount of Fe-S cluster is provided , as sensed by forming [ Fe-S ] - IscR , iscR expression gets repressed to a certain basal level , and sufA expression is no longer activated and returns to near ground level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>C. Transcription of the iscR and sufA genes was not induced during manganese intoxication , indicating that IscR remained in its cluster-containing form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>Because SoxR binds independently of its [ 2Fe -- 2S ] cluster , it also represses transcription of its own promoter independently of its [ 2Fe -- 2S ] cluster .32 However , SoxR can only activate transcription from the divergent PsoxS when its [ 2Fe -- 2S ] cluster is present and oxidized .9 Although SoxR and IscR may be similar in their ability to function as trans-cription factors in the absence of their Fe -- S clusters , SoxR only binds a single DNA site , whereas IscR has multiple targets containing either the Type 1 site or the Type 2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>coli .1,2 IscR was first discovered for its role in regulating expression of the Isc ( iron -- sulfur cluster ) biogenesis pathway ( reviewed in Refs .3 and 4 ) , which is transcribed as an operon ( iscRSUA-hscBA-fdx ) that includes iscR .1 The finding that IscR requires a functional Isc Fe -- S biogenesis pathway to repress the iscR promoter ( PiscR ) linked IscR [ 2Fe -- 2S ] cluster occupancy to its transcription function , suggesting that Fe -- S biogenesis is controlled by a homeostatic mechanism directed by [ 2Fe -- 2S ] - IscR .1 Recent genome-wide transcription profiling data extended these initial observations to show that IscR also controls the expression of a subset of anaerobic Fe -- S-containing enzymes [ hydrogenase-1 ( Hya ) , hydrogenase-2 ( Hyb ) , and the periplasmic nitrate reductase ( Nap ) ] , in addition to other proteins with Fe -- S biogenesis functions [ YadR ( now ErpA ) , YhgI Introduction IscR , a [ 2Fe -- 2S ] cluster-containing transcription factor , acts as a repressor or activator to control the expression of more than 40 genes in Escherichia 0022-2836 / $ - see front matter © 2009 Elsevier Ltd. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li>
<li>Oxidant-responsive induction of the suf operon , encoding a Fe-S assembly system , through Fur and IscR in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>IscR-dependent gen expression links iron-sulphur e o the Summary IscR is an iron-sulphur ( Fe-S ) cluster-containing transcription factor that represses transcription of the operon containing its own gene and the iscSUA-hscBA-fdx genes , whose products are involved in Fe-S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_28">
IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></label>
<input id="tu_similarity_28", type="checkbox">
<div class="TU_similar">
<li>IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12486045" target="_blank">12486045</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16199566" target="_blank">16199566</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>IscR , an Fe -- S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe -- S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19361432" target="_blank">19361432</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_31">
Schwartz , C.J. , Giel , J.L. , Patschkowski , T , Luther , C , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S cluster-con-taining transcription factor , represses expression ofEscherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11983079" target="_blank">11983079</a>]<br></label>
<input id="tu_similarity_31", type="checkbox">
<div class="TU_similar">
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T. , Luther , C. , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S clus-ter-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>Schwartz , C.J. , Giel , J.L. , Patschkowski , T. , Luther , C. , Ruzicka , F.J. , Beinert , H. , and Kiley , P.J. ( 2001 ) IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li></div>
<li>The yfhJ-pepB-sseB genes , located directly downstream of the IscR-repressed isc-SUA-hscBA-fdx operon , also appeared to be repressed by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The only genes that appeared to be regulated by IscR under both aerobic and anaerobic conditions were iscSUA-hscBA-fdx-yfhJ-sseB , ﬁmIF and yqjI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Finally , two genes of unknown function , ydiU and yjcB , also appeared to be activated by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Finally , deletion of the genes encoding IscX , PepB , and SseB , which are located downstream of iscRSUA-hscBA-fdx and are repressed by IscR ( Giel et al. , 2006 ) , were also tested for their effects on IscR function , although only IscX has been suggested to have a possible function in Fe-S biogenesis ( Pastore et al. , 2006 , Shimomura et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>When cells are under attack by exogenous reactive oxygen species as encountered in phagocytic environment , oxidation of Fe-S cluster and its subsequent disassembly in IscR will actively induce the sufABCDSE operon , and will derepress the iscRSUA operon at the same time , maximizing the cellular capacity for Fe-S assembly and maintenance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>MALDI-TOF mass analysis identiﬁed it with IscR , known to serve as a repressor of the iscRSUA gene expression under anaerobic condition as a [ 2Fe-2S ] - bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>One explanation for this is based on the fact that the Fe-S protein IscR represses transcription of yqjI .17 When the suf operon is expressed , the increased synthesis of Fe-S clusters by the SUF system likely decreases the concentration of free iron within the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>IscR also directly activates the expression of the alternate Fe-S biogenesis pathway encoded by the sufABCDSE ( suf ) operon ( Nesbit et al. , 2009 , Yeo et al. , 2006 ) , which includes proteins that exhibit functional similarity to those of the Isc pathway ( Py and Barras , 2010 , Py et al. , 2011 , Johnson et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The transcriptional proﬁling data suggested that the alternate Fe-S cluster biogenesis system encoded by the sufABCDSE operon is also regulated by IscR ; however , unlike the isc operon , the suf operon appears to be activated by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>It is surprising , however , that suf is activated by IscR , unlike the isc operon , which is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The ISC operon is controlled by IscR , a Fe-S cluster binding transcriptional repressor , and contains genes for a cysteine desulfurase ( IscS ) , scaffold protein ( IscU ) , ferredoxin ( Fdx ) , IscA , molecular chaperones ( hscA and hscB ) , and IscX ( also known as yfhj and ORF3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22352884" target="_blank">22352884</a>]<br></li>
<li>Interestingly , all of these genes are repressed by IscR and are known or implicated in Fe-S biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Surprisingly , the isc operon was more repressed under anaerobic conditions , indicating increased Fe-S cluster occupancy of IscR and decreased Fe-S cluster biogenesis demand relative to aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>The global regulator IscR coordinates cluster synthesis by repressing transcription of the isc operon by [ 2Fe-2S ] - IscR and activating expression of the suf operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The IscR protein produced under these conditions is likely to be under the Apo form ( lacking Fe-S ) and may activate the expression of the suf operon , which would then overtake the Fe -- S cluster biogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>In addition , we found that the expression of two genes , yadR and yhgI , encoding proteins related to IscA are also repressed by IscR , similar to the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In its apoprotein form , IscR is a positive activator of expression of other genes , including the alternative iron-sulfur-cluster assembly system that is encoded by the sufABCDSE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>These results support a model that IscR in its demetallated form directly activates sufA transcription , while it de-represses isc operon , under oxidative stress condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Conversely , expression of the isc operon is controlled by one of its products , IscR , and an attractive model for the feedback regulation by the [ 2Fe-2S ] form of IscR has been proposed ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>This suggests that IscR remains under the Apo form and does not repress the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>Our data indicate that a negative feedback loop operates to repress transcription of the iscRSUA-hscBA-fdx operon , encoding IscR and the Isc machinery , through binding of [ 2Fe-2S ] - IscR to two upstream binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Other genes in the operon encode a [ 2Fe-2S ] ferredoxin ( Fdx ) and the transcription factor IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>These genes are strongly induced when iron-sulfur synthesis is hindered , due to conversion of the IscR [ 2Fe-2S ] transcription factor to its apoprotein form [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>For example , gene expression of sufABCDSE is induced by H2O2 ( Zheng et al. 2001 ) , and oxidant induction of the suf operon is regulated by OxyR , IHF , Fur and IscR ( Lee et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>Although transcription of the isc operon is repressed by IscR ( Schwartz et al. , 2001 ) , a [ 2Fe-2S ] cluster-containing protein encoded by the first gene of this operon , a detailed understanding of how this autoregulatory mechanism occurs and ultimately how Fe-S biogenesis is regulated in vivo remains to be elucidated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>One direct regulator of the iscRSUA operon is IscR that contains a [ 2Fe-2S ] cluster and represses the operon under anaerobic condition ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>IscR is a transcriptional regulator that in its Fe-S cluster-bound form represses the expression of the isc operon ( Schwartz et al. , 2001 ; Giel et al. , 2013 ; Rajagopalan et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>]<br></li>
<li>In contrast , the iscR mutation derepressed the level of iscR gene expression in the absence of any oxidant treatment , consistent with the proposal that IscR acts as a repressor of its own operon ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>Within this group were the genes of the isc operon ( iscS , iscU , iscA , hscB and fdx ) , which are known to be repressed by IscR ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Contrary to suf , the isc operon is not regulated by Fur but by IscR , a transcriptional repressor encoded as part of the iscRSUA polycistron ( Schwartz et al , 2001 ; Giel et al , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19407815" target="_blank">19407815</a>]<br></li>
<li>IscR has been reported to act as a repressor for its own operon iscRSUA ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>IscR was originally discovered by its role in the negative control of this operon ( Schwartz et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>( b ) Suf operon activation by wrwycr is dependent upon the IscR activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>IscR : a repressor protein of the iscRSUA operon IscR is encoded by an ORF located immediately upstream of iscSUA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>However , the physiological relevance of the increase in soxS expression in strains lacking IscR is questionable as no corresponding increase in SoxS target gene expression was observed in the strain lacking IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>ArcA and AppY Antagonize IscR Repression of Hydrogenase-1 Expression under Anaerobic Conditions , Revealing a Novel Mode O2 Regulation of Gene Expression in Escherichia coli of A. D. Nesbit , * A. S. Fleischhacker , S. J. Teter , and P. J. Kiley Department of Biomolecular Chemistry , University of Wisconsin , Madison , Wisconsin , USA Transcription of the Escherichia coli hydrogenase-1 operon ( hyaABCDEF ) is increased by the transcription factors ArcA and AppY under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>For several genes ( dctA , ﬂu , hyaABCDEF , hybOABC-DEFG , napFDAGHBC , soxS , ycgZ , yjiM ) , repression by IscR was observed only under aerobic growth conditions even though IscR can function anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>However , the possibility that IscR is non-functional in the absence of O2 was excluded as IscR-dependent repression of the iscR promoter was detectable anaerobically and , in fact , was enhanced under anaerobic growth conditions ( Fig. 1 , Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>As the hya , hyb and nap operons are known to be upregulated under anaerobic conditions , we also examined whether IscR had any effect under anaerobic growth conditions ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Transcription of a subset of anaerobically induced genes is repressed by IscR under aerobic conditions Unexpectedly , the transcriptional proﬁling data indicated that three operons , hyaABCDEF , hybOABCDEFG and napFDAGHBC , encoding respiratory enzymes that are known to contain Fe-S clusters and are active only under anaerobic conditions , were derepressed under aerobic conditions in the absence of IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>As the promoters of other anaerobically induced Fe-S enzymes ( frd , fdnG , narG , dmsA , driving the expression of operons encoding fuma-rate reductase , formate dehydrogenase , nitrate reductase and DMSO reductase respectively ) are known to be poorly expressed under aerobic conditions and were below the limits of detection in the microarray experiments , we tested whether any of these other anaerobic respiratory enzymes were also controlled by IscR using the more sensitive β-galactosidase assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Because expression of IscR is induced under oxidative stress or iron deprivation , RNase LS activity would be repressed under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20421606" target="_blank">20421606</a>]<br></li>
<li>The ﬁnding that IscR represses the expression of the hya , hyb and nap operons under aerobic conditions was surprising as previous work indicated that the regulation of these genes by O2 resulted from speciﬁc activation under anaerobic conditions ( Choe and Reznikoff , 1993 ; Brondsted and Atlung , 1994 ; Darwin and Stewart , 1995 ; Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In contrast , for the hyaA and hybO promoters , IscR repressed the previously identiﬁed O2-regulated transcription start sites ; under anaerobic growth conditions , AppY and the ArcA/B two-component regulatory system have been implicated in the activation of transcription of the hya operon ( Brondsted and Atlung , 1994 ; Richard et al. , 1999 ) , whereas FNR appears to contribute to activation of the hybO promoter ( Richard et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In contrast , IscR was able to repress PhyaA-lacZ expression under aerobic conditions , as previously reported ( 14 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>However , IscR , which represses transcription of the hyaA promoter ( PhyaA ) under aerobic conditions , was not known to repress transcription of this promoter under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>More recently , another transcription factor , IscR , has been shown to directly repress the hyaA promoter ( PhyaA ) but only under aerobic conditions when ArcA and AppY are not active ( 14 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Rather , hya expression in S. enterica is regulated not by IscR or ArcA but , rather , by FNR under anaerobic conditions ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>Using a promoter -- lacZ fusion , we found that IscR increased expression of ydiU under both aerobic and anaerobic conditions ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Consistent with the notion that PhyaA expression is regulated by IscR repression aerobically and by ArcA and AppY antirepression of IscR anaerobically , PhyaA-lacZ expression levels in the absence of any transcription factors were similar under anaerobic and aerobic conditions ( Fig. 1A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>ArcA , AppY , and IscR act combinatorially to control hyaA promoter expression levels under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In addition , we found that under anaerobic conditions , expression of some operons ( sodA , soxS , sufABCDSE , ycgZ , yjcB ) was decreased relative to aerobic conditions , but regulation by IscR could not be determined as expression levels of these operons in both strains were below the limit of detection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR decreased expression of soxS twofold under aerobic conditions but had no effect under anaerobic conditions , which paralleled the results of the transcriptional proﬁling experiments ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR requires its Fe-S cluster to repress expression from the iscR promoter in vivo under aerobic and anaerobic conditions To establish the extent of the IscR [ 2Fe-2S ] cluster requirement for negative autoregulation under both aerobic and anaerobic growth conditions , we measured PiscR repression in strains expressing IscR mutants that are defective in cluster binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></li>
<li>Expression of hyaA , hybO and napF did not appear to be repressed by IscR under anaerobic conditions as the amount of β-galactosidase activity was similar in strains containing or lacking IscR , suggesting that IscR might be an aerobic-speciﬁc repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>The ﬁnding that IscR repressed expression of the hyaA , hybO and napF promoters speciﬁcally under aerobic growth conditions suggests a new mechanism to explain their upregulation under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>IscR does not repress expression of all anaerobic Fe-S respiratory enzymes We considered the possibility that transcriptional repression of the hyaA , hybO and napF promoters by IscR under aerobic conditions may be a general mechanism to prevent inappropriate Fe-S cluster insertion into anaerobic Fe-S respiratory enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>