<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - KdgR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">KdgR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">KdgR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>We predicted that PhoP regulates cysB encoding a global regulator of cysteine biosynthesis , KdgR putatively regulates edd encoding the gluconate dehydratase , Fis putatively regulates rpsI and rplM encoding two ribosomal proteins , and PdhR regulates lipA encoding the lipoate synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>To determine whether CRP and KdgR were involved in one of these modes of regulation , activities of setA - lacZ and sgrS - lacZ fusions were monitored in two different host strains : host 1 , an sgrR host in which stress was induced by addition of MG to cul tures , and host 2 , a sgrR host complemented with a stress signal-independent sgrR allele ( i.e. , an sgrR mutant that does not require stress to activate the sgrS promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>To investigate whether KdgR and CRP directly or indirectly regulate sgrS promoter activity , we again utilized the plasmid carrying the sgrR1 mutant allele .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Collectively , these results suggested that under glucose-phosphate stress conditions , KdgR represses the activation of sgrS ( and very slightly of setA ) indirectly by altering signaling through the stress response system and therefore activity of SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Together , these results suggest that while CRP may repress basal levels of setA transcription , neither CRP nor KdgR is likely to exert any direct effect on setA transcription under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Both KdgR and CRP had modest effects on the activation of setA - lacZ and more signiﬁcant effects on sgrS - lacZ under stress conditions ( Fig. 4A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>We therefore tested the effects of CRP and KdgR on regulation of sgrS and setA in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Effects of CRP and KdgR on setA and sgrS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Previously , the only in vivo information regarding SET family members was limited to sotA of D. dadantii : the transcription of sotA is activated by the cyclic AMP ( cAMP ) receptor protein ( CRP ) and weakly repressed by the 2-keto-3-deoxygluconate repressor KdgR ( 4 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>CRP activated expression of sotA ( 4 ) , while KdgR , which represses transcription of many genes involved in transport and catabolism of 2-keto-3-deoxy-gluconate in E. coli ( 22 , 27 ) , was shown to weakly repress the expression of sotA in D. dadantii ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>In E. chrysanthemi , the KdgR repressor induces pel genes in response to 2-keto-3-deoxyglu-conate , and this induction is reinforced by competition with Pir activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
<li>Nasser , W. , Reverchon , S. , Condemine , G. , and RobertBaudouy , J. ( 1994 ) Specific interactions of Erwinia chry-santhemi KdgR repressor with different operators of genes involved in pectinolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Moreover , almost all genes from the KdgR regulon are regulated by CRP in E. chrysanthemi ( Nasser et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_14">
Antagonistic effect of CRP and KdgR in the transcription control of the Erwinia chrysanthemi pectinolysis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9573142" target="_blank">9573142</a>]<br></label>
<input id="act_similarity_14", type="checkbox">
<div class="ACT_similar">
<li>Nasser , W. , Robert-Baudouy , J. , and Reverchon , S. ( 1997 ) Antagonistic effect of CRP and KdgR in the transcription control of the Erwinia chrysanthemi pectinolysis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>]<br></li></div>
<li>Glucuronate catabolism requires ﬁrst the ExuR regulon , which controls transport of glucuronate and galactu-ronate and their conversion to fructuronate and tagaturonate , respectively , and then the UxuR and KdgR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>It includes all known and predicted members of the GntR , UxuR/ExuR , KdgR , UidR and IdnR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The last column ( * ) represents experimental data about regulation : S , known binding sites ; R , known regulation , binding site not known.Regulons : a. GntR ; b. IdnR ; c. UxuR/ExuR ; d. ExuR ; e. UidR ; f. KdgR ; g. CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>These data indicate that eda P2 has a high basal level of transcription , is induced by glucuronate , and is repressed by KdgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>P2 controls eda induction on glucuronate and galacturonate and is regulated by KdgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Regulation by KdgR induces eda if glucuronate , galacturonate , or methyl-b-d-glucuronide are present in the growth media [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>P2 is induced by growth on glucuronate , galacturonate , and methyl - - D-glucuronide and is mediated directly by KdgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>We obtained evidence that P1 is under negative control of GntR , P2 is under negative control of KdgR , and P4 is under negative control of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Since Eda is required for growth on hexonates and hexuronates , its induction by these sugar acids and inclusion of eda in both the GntR and KdgR regulons are easily understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>This dual control facilitates the induction of Eda only for growth on hexuronates , which is mediated by KdgR and does not require Edd , which is only necessary for converting 6-phosphoglu-conate to KDPG ; coordinate induction of Edd and Eda for growth on hexonates is controlled by GntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Lanes 1 to 10 , 0 , 0.07 , 0.14 , 0.21 , 0.28 , 0.35 , 0.49 , 0.63 , 1.41 , and 2.11 pmol of KdgR , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Therefore , the ppsA gene is activated by at least three dual-mode catabolic regulators ( HexR , FruR , and KdgR ) in three lineages of - proteobacteria ( Shewanella , Escherichia , and Erwinia , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">KdgR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The C-terminal domain retains its structure and small molecule binding function when expressed without the DNA-binding domain and was structurally characterized for three E. coli representatives of IclR-family , KdgR ( PDB code 1YSP ) , YiaJ ( PDB code 1YSQ ) , and AllR3 ( Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17426033" target="_blank">17426033</a>]<br></li>
<li>possessing the KdgR TF : one copy is located in the uxaC-kdgK-eda-ygjV cluster ( not regulated by KdgR ) , another one belongs to the kduD-ygjV-kdgF-spiX cluster preceded by two candidate KdgR sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">KdgR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>continued Subcellular localization f Function lrp a 30.69 c 2.72 2.69 8.72 2.60 23.70 2 2.22 2 2.00 2 2.00 23.33 2 2.22 2 2.08 2 4.00 2 2.00 2 2.70 2 2.77 2 3.13 2 2.08 2 2.00 2 2.50 3.75 2 2.04 2 2.99 2 2.00 2 2.17 2 2.50 22.86 9.76 14.00 Cytoplasmic Cytoplasmic Cytoplasmic f Cytoplasmic f Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic Cytoplasmic f Leucine-responsive regulatory protein Regulator of the two-component system PhoP/PhoQ 5E-2 1E-2 7E-3 3E-3 1E-3 2E-2 3E-2 1E-2 1E-2 5E-2 4E-3 2E-3 5E-2 4E-4 2E-2 2E-2 1E-2 8E-3 6E-3 5E-2 5E-2 5E-2 2E-2 2E-2 5E-2 4E-2 2E-2 5E-2 4E-2 P19494 P23836 P77170 P77574 P24210 P02384 P02387 P02386 P02390 P02418 P02409 P02392 P02411 P02413 P02416 P02424 P02425 P02429 P02435 P02436 P13445 P02361 P02363 P02369 P02370 P02373 P02375 P30017 P75989 phoP b pinQ b pinR b rcsA ab rplA b rplB rplC rplF rplI b rplK rplL rplN rplO rplQ rplW rplX rpmC rpmF rpmG rpoS a rpsH rpsI rpsM rpsN rpsQ rpsS b stpA a ycgE DNA invertase DNA invertase Regulatory protein 50S ribosomal subunit protein L1 50S ribosomal subunit protein L2 50S ribosomal subunit protein L3 50S ribosomal subunit protein L6 50S ribosomal subunit protein L9 50S ribosomal subunit protein L11 50S ribosomal subunit protein L7/L12 50S ribosomal subunit protein L14 50S ribosomal subunit protein L15 50S ribosomal subunit protein L17 50S ribosomal subunit protein L23 50S ribosomal subunit protein L24 50S ribosomal subunit protein L29 50S ribosomal subunit protein L32 50S ribosomal subunit protein L33 RNA polymerase sigma factor 38 30S ribosomal subunit protein S8 30S ribosomal subunit protein S9 30S ribosomal subunit protein S13 30S ribosomal subunit protein S14 30S ribosomal subunit protein S17 30S ribosomal subunit protein S19 DNA-binding protein Similar to P33358 , YehV putative transcriptional regulator ( E. coli ) PS00552 : Bacterial regulatory protein MerR family signature Similar to P05052 , AppY regulatory protein ( E. coli ) PS00041 : Bacterial regulatory proteins AraC family signature Similar to O44006 , regulatory protein CzcR ( Ralstonia eutropha ) Similar to U32711 , sigma54 modulation protein , putative ( H. influenzae ) Similar to P05052 , AppY regulatory protein ( E. coli ) PS00041 : Bacterial regulatory proteins AraC family signature Similar to P10805 , regulatory protein EnvY ( E. coli ) PS00041 : Bacterial regulatory proteins AraC family signature Similar to P76268 , KdgR transcriptional regulator ( E. coli ) PS01051 : Bacterial regulatory proteins , IclR family signature ydeO b P76135 2E-2 3.87 Cytoplasmic 35.36 c 3.72 32.41 c yedW b yfiA yhiW b P76340 P11285 P37638 5E-2 4E-2 2E-2 Cytoplasmic probable Cytoplasmic f Cytoplasmic f yhiX b Cytoplasmic f P37639 4E-4 13.92 24.07 c yjhI b Cytoplasmic f P39360 4E-2 Other functions 2.50 d 4.10 D appA b cspB 3E-2 5E-2 Acid phosphatase , pH 2.5 Similar to cold shock protein P07102 P36995 Periplasmic Cytoplasmic probable ain signature PS00352 : Cold shock DNA-binding dom Similar to cold shock protein PS00352 : Cold shock DNA-binding dom P36996 4.73 1E-2 cspC Cytoplasmic probable ain signatur Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>In studies of sotA , a SET family member in D. dadantii , the global regulators CRP and KdgR were implicated in transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>The YiaJ repressor was also found to have a high similarity to KdgR and Pir of Erwinia chrysanthemi ( 27 , 29 ) , both involved in pectin degradation and classiﬁed into the IclR family of regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
<li>The cisacting element recognized by E. coli IclR repressor was shown to be AATTAAAATGGAAATTGTTTTTGATTTTGCATTTT ( 27 ) , and highly homologous sequences have been reported in E. chrysanthemi for other operator regions recognized by proteins of this family , such as KdgR or Pir ( 27 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by KdgR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Sugar acid regulons on the E. coli genome Location ( kb ) Gene Function Regulon Operon Reference GntR gntR 65 20 3575.9 1931.7 1930.6 3544.8 3574.9 3574.2 4488.2 4492.5 4491.8 gntR edd eda gntT gntK gntU idnR idnK idnD Repressor : gluconate induction 6-Phosphogluconate dehydratase KDPG aldolase High-afﬁnity gluconate-transport Gluconate kinase Low-afﬁnity gluconate-transport Regulator : L-idonate induction Gluconate kinase L-Idonate 5-dehydrogenase edd-eda 20 52 65 65 8 , 9 8 , 9 8 , 9 gntT gntKU IdnR idnDOTR idnK ( gntV ) idnDOTR ( yjgVUTS ) 8 , 9 8 , 9 57 57 39 20 36 46 35 35 35 Gluconate 5-dehydrogenase L-Idonate transport 4490.6 4489.3 1908.2 3676.4 4099.8 1930.6 3244.7 3243.5 3241.7 3240.3 1608.0 idnO idnT kdgR kdgK kdgT eda exuR exuT uxaC uxaA uxaB KdgR kdgR kdgK Repressor : KDG induction KDG kinase KDG transport KDPG aldolase Repressor : hexuronate induction Glucuronate/galacturonate transport Glucuronate/galacturonate isomerase Altronate dehydratase ( galacturonate ) Altronate oxidoreductase ( galacturonate ) Repressor : glucuronate induction Mannonate dehydratase ( glucuronate ) Mannonate oxidoreductase ( glucuronate ) Hexonate/hexuronate-transport ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>We predicted that PhoP regulates cysB encoding a global regulator of cysteine-biosynthesis , KdgR putatively regulates edd encoding the gluconate dehydratase , Fis putatively regulates rpsI and rplM encoding two ribosomal proteins , and PdhR regulates lipA encoding the lipoate synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>We obtained evidence that P1 is under negative control of GntR , P2 is under negative control of KdgR , and P4 is under negative control of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Two transcriptional regulators , CRP and KdgR , which inﬂuence the expression of sotA , encoding a SetA paralog in D. dadantii ( 4 , 29 ) , were tested for their roles in regulating setA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Global regulators ExpA ( GacA ) and KdgR modulate extracellular enzyme gene-expression through the RsmA-rsmB system in Erwinia carotovora subsp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12533459" target="_blank">12533459</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12193630" target="_blank">12193630</a>]<br></li>
<li>Interestingly , the KDG-responsive KdgR regulon negatively controlling the pectin-catabolism in plant pathogenic Erwinia is largely extended in comparison with other enterobacteria and include the positively regulated gene ppsA ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21849503" target="_blank">21849503</a>]<br></li>
<li>KdgR regulates the genes for the catabolism of KDG ( 2-keto-3-deoxyglu-conate ) and its predecessors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>CRP activated expression of sotA ( 4 ) , while KdgR , which represses transcription of many genes involved in transport and catabolism of 2-keto-3-deoxy-gluconate in E. coli ( 22 , 27 ) , was shown to weakly repress the expression of sotA in D. dadantii ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>In E. chrysanthemi , KdgR also controls the majority of the pectinase genes ( pelA , B , C , D and E ) and the out genes involved in pectinase secretion ( Nasser et al. , 1994 ; Rouanet et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Nasser , W. , Reverchon , S. , Condemine , G. , and RobertBaudouy , J. ( 1994 ) Specific interactions of Erwinia chry-santhemi KdgR repressor with different operators of genes involved in pectinolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Specific interactions of Erwinia chrysanthemi KdgR repressor with different operators of genes involved in pectinolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16672238" target="_blank">16672238</a>]<br></li>
<li>In E. coli , the downstream KDG metabolism is controlled by the KdgR repressor from the IclR family , which is also regulating the pectin utilization in plant-pathogenic species of the Entero-bacteriales ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>We believe that the KdgR regulon of the related E. chrysanthemi also contains an as yet unsequenced transport system orthologous to ogtABCD that is required for the uptake of the pectin-degradation products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The YiaJ repressor was also found to have a high similarity to KdgR and Pir of Erwinia chrysanthemi ( 27 , 29 ) , both involved in pectin-degradation and classiﬁed into the IclR family of regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
<li>In E. chrysanthemi , KdgR regulates almost all genes involved in pectin-degradation ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The KdgR regulon of the plant pathogen E. chrysanthemi is the largest and contains an array of genes for the degradation and subsequent utilization of plant pectin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The predicted new member of the KdgR regulon , an ABC transport system ogtABCD in Y. pestis and K. pneumoniae , is a missing link for the utilization of oligogalacturonates in the metabolic map .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Glucuronate catabolism requires ﬁrst the ExuR regulon , which controls transport of glucuronate and galactu-ronate and their conversion to fructuronate and tagaturonate , respectively , and then the UxuR and KdgR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516583" target="_blank">15516583</a>]<br></li>
<li>This duplication may be a result of a recent specialization of the paralogs toward catab-olism of either pectin ( regulated by KdgR ) or hexuronate , and its localization suggests that yjgV could be a part of the UxuR regulon , involved in the transport of some intermediates of the hexuronate catabolic pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">KdgR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Operons Position Score Binding site ( s ) * A. GntR regulon E. coli gntKU R a 2211 11 2116 5.84 5.3 5.69 4.42 6 4.65 4.82 cAATGTTACCGaTAACAgTT AAcaGTTACCcGTAACATTT AgATGTTACCcGTAtCATTc TgAcGTTACCcaTAACAaaT TtATtTTACCGGTAACATgA AAtTGTTACgcGTAACAaaT TcAcGTTAtgcGTAACATag S a S a R a gntT edd-eda R b idnDOTR/gntV S. typhi gntKU 2213 9 5.84 4.62 5.78 4.36 5.94 4.48 cAATGTTACCGaTAACAgTT AAcaGTTACCcGTAACAaaT AtATGTTACCcGTAtCATTc TgAcGTTACCcaTAACAaag gtATtTTACCGGTAACATgA AttTGTTACgcGTAACAcag gntT edd-eda Y. pestis gntT/gntK 6.05 5.48 4.39 5.72 ccATGTTACCGGTAtCATgA TcATGaTACCGGTAACAaaT AtATaTTAtCGcTAtCAggA TcAcGTTACCGGTAACATgT idnO-gntV sdsX V. cholerae gntU-eda / gntK-edd 5.8 ± 81 TcATGTTAtgGGTAACATgT 5.57 267 AcATGTTACCGGTAACtTcA saATGTTACCGGTAACATtw Consensus B. UxuR/ExuR regulon E. coli exuR uxaB uxuAB/gntP R c R c R d 5.26 5.24 5.24 4.5 5.06 4.28 3.82 5.19 4.14 261 AAAgTGGTATAaCAAaTa AAtgTGtTcTACCAcTTT gAtgTGGTtaACCAATTT AAATTGGTcaACCAtTgT tAtTTGGTtgACCAgTTT AAAgTtGTATgaCAAgTT AatTTttTtaACtAcgTT ctATTGGTtaACCAATTT gAATTGGTtaACtAATca R d R c uxuR exuT/uxaCA uidR uidAB R , R e c S. typhi exuT/uxuABC AAATTGGctagCCAAaTc tgATTGGTcaACCAATTT ttgTTGGTtgACCAgTTT AtATTGGTAaACCAATaT 4.42 5.07 4.57 5.25 uxuR Y. pestis uxuB uxuA uxuR exuR gntP AAAcTGGTcTAaCAAcTT AAATcGGTATAaCAAcaT AAgTTGtTtgACCAcaTg tAAgTGGTATAaCAAaTc AAcTTGGTtaACCAATac cAtTTGGTtatCCAcTTa AAcTTGtcATACagATTT 4.99 4.44 4.66 4.93 4.78 4.54 3.72 uxaCBA/exuT H. influenzae uxuRA AAATTGGaATACCAATTT tttTTGGaATACCAAaTa AAATTGGTATACCAATTT 5.47 4.39 223 273 Consensus C. KdgR regulon E. chrysanthemi ogl f AAATGAAAgAATGTTTtATaa AAATGAAACgTTGTTTCtaca AAATaAAACATTaTTTCATTT AAtcaAAACAATGTTTCtaTT AAATaAAACATcGTTTCATcg AAAaGAAACATTGTTTCATTT tAATGAAACggTGTTTtATTa AttTaAAACATcGTTTCATTa tAATGAAAtggcaTTTCAaTT tAATGAAAttAcGTTTCAacT AgATGAAAtggTaTTTCgTTT S S S R S S S S S S S 5.15 5.63 6.05 5.42 5.64 6.07 5.83 5.52 5.72 5.44 5.59 f f kduID pelW kdgK kdgT outT pelA pelB pelC pelE f f f f f f f f Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>The derived consensus sequence of the palindromic KdgR box is AAATGAAACAnTGTTTCATTT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>Puriﬁed KdgR retarded the mobility of a labeled DNA probe corresponding to the eda P2 promoter region , which contains the putative KdgR binding right half-site GTGTTTCAAA ( Fig. 1 ) , a close match to the consensus sequence , AAATGAAACAnTGTTTCATTT ( 24 ) ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>UxuR binding sites are indicated as black diamonds , ExuR binding sites are indicated as light blue diamonds , sites predicted to be recognized by both UxuR and ExuR are indicated as violet diamonds , and KdgR binding sites ( upstream of ygjV ) are indicated as dark blue squares .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>The cisacting element recognized by E. coli IclR repressor was shown to be AATTAAAATGGAAATTGTTTTTGATTTTGCATTTT ( 27 ) , and highly homologous sequences have been reported in E. chrysanthemi for other operator regions recognized by proteins of this family , such as KdgR or Pir ( 27 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by KdgR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>In E. chrysanthemi , the KdgR repressor induces pel genes in response to 2-keto-3-deoxyglu-conate , and this induction is reinforced by competition with Pir activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913096" target="_blank">10913096</a>]<br></li>
<li>The genes for the catabolism of KDG and its predecessors are regulated by transcription factor KdgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115104" target="_blank">11115104</a>]<br></li>
<li>have two copies of both kdgK and eda genes , one of which is regulated by UxuR , whereas the other one is under the KdgR control ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>also have additional copies of the kdgK and eda genes that are members of the KdgR regulon ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21622752" target="_blank">21622752</a>]<br></li>
<li>Since Eda is required for growth on hexonates and hexuronates , its induction by these sugar acids and inclusion of eda in both the GntR and KdgR regulons are easily understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Regulation by KdgR induces eda if glucuronate , galacturonate , or methyl-b-d-glucuronide are present in the growth media [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>Interestingly , eda is also a member of the KdgR regulon and is induced approximately fourfold by growth on glucuronate ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9657988" target="_blank">9657988</a>]<br></li>
<li>P2 controls eda induction on glucuronate and galacturonate and is regulated by KdgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>These data indicate that eda P2 has a high basal level of transcription , is induced by glucuronate , and is repressed by KdgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>