<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - KdpE</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">KdpE</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">KdpE transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The RR-bound 32P was stably retained for the A1 group RRs such as pArcA , pCpxR , pCusR , pKdpE , and pNarL but was rapidly released for the A2 group RRs , including pBaeR , pCheB , pCheY , pCreB , pDcuR , pNarP , pNtrC , pTorR , pUhpA , pYehT , and pYfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The first approximately 200 residues of KdpE and three other effector-proteins , PhoB ( 28 ) , CreB ( formerly PhoM-orf2 , the hypothetical PhoB protein of PhoM/Pho2 ) ( 2 , 48 ) , and OmpR ( 8 ) of E. coli are aligned with the TREE program ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>lPV 1LLTARsava 1eaatrkpDL iILDlgLPdgdGiefirdI.Rqvsav ... PV ivLsARseEs ArcA VirG PhoP Va `` R PhOB CreB TctD KdpE CO `` SEX `` SUS DL LD LP G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>pvirighFeLnepaaqisw fDtp.LaLTryEFlLLktL1 kspgrVwSRq hevqqlgeLi Fhdegy.fll qgqp.LaLTp rEqaLLtvb yrrtrpvSRq dplvkfsdvtvdlaar.vih rgeeavhLTp iEFrLagrcstmpekyspsg ~ PR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE and Silverman , with the second viously published by Weber and Silverman ( Weber 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>Members of this class are NtrC and PhoB , other candidates are ArcA , OmpR and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>227 OmpR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE kINnlaRgre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>Asterisks ( * ) indicate the conserved aa residues crucial for the transcriptional regula ¬ 232 CpxR nvsas ... OmpR ArcA VirG PhoP V & `` R PhoB CreB TctD KdpE hrpdgska .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>PG-V-SR -- LL 60 70 80 90 i00 SRV RDEEVYDRS IDVLILRLRR KLEADPSSPQLIKTARGAGYFF SRV REEEVYDRS IDVLIFRLRR KLEGDPTSPQLIKTARGAGYFF SRV REEEVYDRS IDVLI LRLRR KLEGDPTTPQLIKTARGAGYFF VTG RELKPHDRTVDVTIRRIRK HFESTPDTPEIIATIHGEGYRF '' ~ 2 , ~ GY TPERHL DTRVVDVHISRLRA KLEDDPSNPELILTSRGTGYLF \ q , ~ GY TAERQV DTRVVDVHISRLRA KLEDDPSNPDLILTARGTGYLF LQLY PDAELRESHTIDVLMGRLRK KIQAQYPH DVITTVRGQGYLF '' , ~ G PNAVEHSHYLRIYMGHLRQ KLEQDPARPRHFITETGIGYRF 2 , GR EYSAME RSIDVQISRLRR MVEEDPAHPRYIQTVWGLGYVF , ~ A2RSASAEEAAGDRAVDVQ ITRLRR KIEPDPREPRYLQTVRGLGYML '' , ' > ~ G TNVYVEDRTVDVHIRRLRK ALEPG GHDRMVQTVRGTGYRF \ ~ ' , ; G GNVYVEERTVDMD IRRLRK ALGE VYENLVQTVRGTGYRF V > ~ E DAQDTYDRTVDTHIKTLRA KLRAINPDLSPINTHRGMGYSL VS 'N YDFAGDTRIVDVHISHLRPTKIENNTKKPIYIKTIRGLGYKL VLG KRLTPFDRAIDMHISNLRR KLPDRKDGHPWFKTLRGRGYLM AWD ENTDPFTNVVRVTVMTLRR KLGE PPVIVTVPGSGYRI EQGFEVDDSSLTQAISTLRK MLKDSTKSPQYVKTVPKRGYQL VirG Atu6 VirG Arh VirG Atu5 ArcA Eco RRHM CYca RRHM POae PhoP Sty KdpE Eco OmpR Eco PetR Rca PhoB Eco PhoB Pae CreB Eco PhoB Bsu YjjE Eco CutR Sli ToxR Vch 119 Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response-regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Proteins in classes 4 , 5 and 6 are exceptions 7O ENAF I0 20 30 40 50 60 NtrC NtrC NifRl NtrC PgtA DctD FixJ NodW CheY SpoOF RRHM RRHM OmpR PhoB CreB ArcA PetR VirG CutR KdpE PhoB PhoP TctD AIgB YLB3 RcsC BvgS HydG CheY FrzE BarA TrpO HvgA FimZ DegU NarL AIgRI YecB UhpA RcsB Agm & CheB CheB FrzG Spo0A MrkE AgrA Coma F!bD Bpa Rme Rca Eco Sty Rle Rme Bja Bsu Bsu Cca Pae Eco Bsu ECO Eco Rca Atu Sli Eco Eco Sty Sty Pae Lin Eco Bpe Eco Eco M ~ a Eco Pae Bpe Eco Bsu Eco Pae Eco Eco Eco Pae Bsu Eco Mxa Bsu Kpn Sau Bsu Ccr 6 ILVADDDTAIKTVLNQA 6 ILVADDD ~ IRTVLNQA 5 VLVADDDRTIRTVLTQA 6 V % ~ VDDDSSIR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>) Major clusters 1 , 2 , and 3 are associated primarily or exclusively with the three major families of DNA binding NtrC Bpa NtrC Rme NifRl Rca Nt rC Eco PgtA Sty DetD Rle FixJ Kme NodW Bja CheY Bsu SpoOF BSu RRHM Cca RRHM Pae OmpR Eco PhoB Bsu CreB Eco ArcA Eco PetR Rca VirG Atu CutR Sli KdpE ECO PhoB Eco PhoP Sty TctD Sty A!gB Pae YLB3 Lin RcsC Eco BvgS Bpe HydG Eco CheY Eco FrzE Nc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>On the other hand , nine species of RR ( AtoC , CheY , CusR , HydG , KdpE , NarL , NarP , NtrC , and YfhA ) were phosphoryl-ated by non-cognate HKs besides their cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Although , KdpE does not contain identical residues ( it is Y191 , I194 , H198 and T217 respectively at positions equivalent to those of PhoB shown in the previous line ) , the properties of three side chains are similar except for KdpE T217 and PhoB R219 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>However , the superposed structures reveal KdpE H198 is less-likely to participate in base specific interactions due to its shorter side-chain length when compared to R201 of PhoB ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Mol Microbiol 6:1777 -- 1784 Ohwada T , Sagisaka S ( 1987 ) An intermediate and steep increase in ATP concentration in response to reduced turgor pressure in Escherichia coli B. Arch Biochem Biophys 259:157 -- 163 Nakashima K , Sugiura A , Kanamaru K , Mizuno T ( 1993a ) Signal transduction between the two regulatory components involved in the regulation of the kdpABC operon in Escherichia coli : phosphorylation-dependent functioning of the positive regulator , KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>Effects of mutation of conserved residues in the wHTH motif of KdpE In contrast to wild-type KdpE , the three mutants ( R193A , R200A and T215A ) tested in the context of the full-length KdpE protein were incapable of responding to stress induced by changing the K concentration from 10 mM to 0 mM ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>However , this particular preparation probably represents a non-phos-phorylated form of KdpE , because the protein was purified from oells artificially overproducing KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>To examine the effect of UspC on binding of KdpE ∼ P to the KdpE binding site , double-stranded DNA fragment comprising the KdpE ∼ P-binding site21 were generated by heating a biotinylated primer and the complementary primer to 95 ° C for 5 min followed by mixing and annealing at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Therefore , residues R193 , R200 and T214 of KdpE were mutated and the ability to promote gene expression in vivo and DNA binding was analyzed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>These results suggest that the phosphorylation of KdpE results not only in enhancement of its DNA-binding ability with respect to the kdpABC promoter region , but also in stimulation of its transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>These results suggest that the phosphorylation of KdpE results in enhancement of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Therefore , it was revealed that although both forms of KdpE exhibit a specific DNA-binding ability , the phosphorylation of KdpE results in enhancement of its specific DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>It is possible that ArcA-P binds to the icdA promoter as a dimer of dimers ; one dimer binds DR1 and DR2 , as depicted in the PhoB and KdpE DNA cocrystal structures ( 15 , 18 ) , and the second dimer binds DR3 but only weakly to adjacent DNA sequence , such that no footprint is observed ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Residues in KdpE targeted for mutagenesis namely R193 , R200 and T215 are indicated by stars , whereas the + sign points to R219 of PhoB that interacts with the minor groove of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Unlike OmpR , the DBDs of both PhoB and KdpE bind their cognate DNA with moderate affinities ( range of Kds 100 -- 400 nM ) ; however , only the DBD of PhoB can initiate transcription [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>For T215 of KdpE , variants with mutations at equivalent residues in OmpR and PhoB are also defective in DNA binding due to loss of H-bonding with the DNA backbone [ 28,49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The inability of mutant KdpE R193A to-bind to DNA is interesting because in most other family members the equivalent residue is of opposite charge as observed in PhoB ( D196 ) and OmpR ( D202 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To elucidate biochemical properties of phospho-KdpE , we then needed to explore in vitro conditions under which phos-pho-KdpE is stably prepared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>When KdpE is phosphorylated , it takes on an active form that exhibits relatively high affinity with the 23 bp sequence ( nucleotide positions - 7 2 to -50 ) immediately upstream of the canonical - 3 5 and - 1 0 regions of the kdpABC operon ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In this study , we obtained biochemical evidence supporting the view that when KdpE is phosphorylated , it takes on an active form that exhibits relatively high affinity for the kdpABC promoter , which in turn results in activation of the kdpABC operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Consequently , KdpE may be extensively phosphorylated and then take on an active form that exhibits high affinity for the cognate promoter DNA , which in turn results in activation of the kdpABC operon ( Walderhaug ef al. , 1992 ; Nakashima et aL , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In any event , the results presented in Fig. 6 clearly demonstrated that transcription of the kdpABC promoter was significantly stimulated under the conditions in which the phosphorylated form of KdpE was supplied , while the lacUV5 promoter was transcribed in a KdpE-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Consequently , the phosphorylated form of KdpE triggers kdp transcription efficiently in response to hyperosmotio stress ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>More recently , biochemical evidence supporting the view that KdpD is a sensory protein kinase exhibiting autophosphorylation and KdpE phosphotransfer characteristics and that KdpE binds to DNA immediately upstream of the promoter for the kdpABC structural genes , in a region which is necessary in cis for normal regulation of the operon , has also been obtained ( 30 , increases transiently upon instantaneous addition of impermeable solutes has also been confirmed recently by Sugiura et al. ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8331081" target="_blank">8331081</a>]<br></li>
<li>In this respect , we demonstrated previously that KdpE is indeed a DNA-bind-ing protein that specifically and directly binds to the upstream region of the kdpABC promoter ( Sugiura ef al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Phosphorylated KdpE binds to a sequence upstream of the kdpFABC promoter and stimulates transcription initiation ( Sugiura et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>This indicated that the KdpE protein binds directly to the sequence upstream of the -35 and -10 regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></li>
<li>In other words , kdp expression appears to be enhanced upon binding of the KdpE protein to the upstream region of the canonical -35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></li>
<li>It was thus of interest to address the crucial issue of the biological consequence of KdpE phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Of particular interest is the conservation of Asp-9 and Asp-52 in KdpE , since these residues are in positions homologous to those of the residues identified as being important in the phosphorylation of the homologous CheY protein ( 6 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Asp-57 , the phosphorylated residue of CheY , is homologous to Asp-52 of KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>( 2009 ) 386 , 134 -- 148 Available online at www.sciencedirect.com The Universal Stress Protein UspC Scaffolds the dp ignaling Cascad cherichia KdpD/K E S e of Es coli under Salt Stress Ralf Heermann1 ,2 , Arnim Weber1 , Bettina Mayer1 , Melanie Ott1 , Elisabeth Hauser1 , Günther Gabriel1 ,2 , Torsten Pirch1 ,2 and Kirsten Jung1 ,2 1Ludwig-Maximilians-Universität München , The sensor kinase KdpD and the response-regulator KdpE control induction of the kdpFABC operon encoding the high-affinity K + - transport system KdpFABC in response to K + limitation or salt stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Induction of the corresponding kdpFABC operon is under control of the two-component system KdpD/KdpE , which induces kdpFABC expression under K + - limiting conditions or under osmotic stress imposed by a salt .2,3 Upon perception of the stimulus , the mem-brane-integrated sensor kinase KdpD undergoes autophosphorylation and subsequently the phosphoryl group is transferred to the cytoplasmic res-ponse regulator KdpE .4 Phosphorylated KdpE exhi-bits increased affinity to a 23-bp sequence upstream of the canonical -- 35 and -- 10 regions of the kdpFABC promoter , and interaction with the RNA-polymer-ase triggers kdpFABC expression .5 KdpD has been shown to be the only protein with KdpE ∼ P phosphatase activity .4 The nature of the stimulus sensed by KdpD has been the subject of numerous studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Expression of the Kdp system is controlled by the two-component system ( TCS ) KdpD/KdpE , which activates kdpFABC transcription under conditions of K + limitation or osmotic salt stress ( Walderhaug et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>In this paper , we report several lines of biochemical evidence supporting this scheme Results Purification of a truncated form of KdpD as a soluble form It was demonstrated previously in vitro that KdpE is capable of undergoing phosphorylation upon incubation with KdpD in the isolated nnembrane ( Nakashima et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In this class , KdpD/KdpE , NarP and NtrB/NtrC are known to be involved in response to turgor pressure , nitrate and nitrogen availability respectively ( Laimins et al. , 1981 ; Darwin and Stewart , 1995 ; Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>In response to appropriate environmental stimuli , KdpD is able to phosphorylate KdpE , and KdpE ~ P brings about transcriptional activation of the kdpFABC operon by binding more tightly to a region upstream of the kdpFABC promoter ( Nakashima et al. 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>The phosphoryl group is then transferred to the response-regulator KdpE thereby transforming it into a transcriptional-activator ( Nakashima et al. 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>Upon signal perception , the membrane-bound sensor kinase KdpD autophosphorylates and transfers the phosphoryl-group to the response-regulator KdpE ( Nakashima et al. , 1992 ; Jung et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>KdpE of Clostridium acetobutylicum is a highly speciﬁc response-regulator controlling only the expression of the kdp operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>This increases phosphorylation of response-regulator KdpE , which enhances transcription initiation at the kdpFABC promoter and thereby synthesis of the high-affinity K + transporter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>In RH003 cells , which were engineered by removing the kdpD and kdpE genes and by fusing a b-galactosidase reporter gene to the kdpFABC promoter , b-galactosidase production was observed under K limiting conditions only when wild-type KdpD and + KdpE were co-expressed ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>However , co-expression of KdpD and KdpEDBD failed to elicit b-galactosidase production , indicating the need for the RD of KdpE for gene transcription despite KdpEDBD retaining its primary biochemical function of specific binding to kdpFABCBS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To address this issue , phospho-KdpD * and phospho-KdpE prepared in vitro were subjected to acid or alkali treatment in a solution containing SDS and 2-mercaptoethanol , the remaining radioactivity in KdpD * and KdpE being monitored by SDS-PAGE analysis ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>At the indicated times , the reaction was stopped by adding SDS-sample buffer , the samples were subjected to SDS-PAGE , and phosphorylated KdpD and KdpE were detected by autoradiography using a phosphor screen and quantified with a PhosphorImager Storm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>At the indicated times , the reactions were stopped , the samples were subjected to SDS-PAGE , and phosphorylated KdpD and KdpE were detected by phospho-imaging and quantiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>We suggest that phosphoKdpE is a positive effector of Kdp expression and that low turgor pressure causes KdpD to phosphorylate KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>The sequences of KdpD and KdpE show they are members of the sensor-effector class of regulatory proteins : the C-terminal half of KdpD is homologous to sensors such as EnvZ and PhoR , and KdpE is homologous to effectors such as OmpR and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Homology of reported effector-proteins with KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Moreover , this domain stabilizes binding of KdpE to its target site on the DNA and contains the UspC interaction surface ( Heermann et al. , 2003 ; 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Recently , it has been shown that the universal stress protein UspC binds to KdpD and stabilizes the KdpD / KdpE ~ P/DNA complex , which leads to enhanced transcription of the kdp operon ( Heermann et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Heermann , R. , Weber , A. , Mayer , B. , Ott , M. , Hauser , E. , Gabriel , G. , et al. ( 2009 ) The universal stress protein UspC scaffolds the KdpD/KdpE signaling cascade of Escherichia coli under salt stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Here , we report a specific interaction between one of the soluble single-domain Usp proteins , UspC , with a Usp domain in KdpD and its role to scaffold KdpD/KdpE signaling in response to salt stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>This is , to our knowledge , the first example for an interaction between one of the soluble Usp proteins with an Usp domain integrated within a larger protein , although it is known that Usp proteins are able to form dimers or higher oligomers .22,26 In summary , binding of UspC to KdpD enhances the amount of phosphorylated KdpE at a concentration of K + that would otherwise almost block phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Earlier , it was shown that the kinase activity of KdpD is inhibited by K +.14 The inhibitory effect of K + is an exclusive property of KdpD , since other biochemically characterized sensor kinases are sti-mulated by increased concentrations of K +.40 -- 42 Moreover , it is known that the equilibrium is shifted towards the phosphorylated form of KdpE when phosphorylation is tested in the presence of DNA fragments comprising the KdpE-binding site .43 This effect is believed to be due to a stabilization of the interaction between KdpE and its corresponding DNA-binding site by the N-terminal domain of KdpD .21 When the reconstructed phosphorylation cascade of KdpD/KdpE was tested at 250 mM K + , which corresponds to the freely available K + under salt stress , the amount of phosphorylated KdpE was reduced to about 10 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>It is important to note that the amount of phosphorylated KdpE on the chips was constant , independent of the addition of KdpD / 1-395 in proteoliposomes with or without UspC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Then , KdpE , phosphorylated KdpE , or the corresponding KdpD/KdpE/UspC mixtures were injected to determine protein binding to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Since UspC did not affect any of the enzymatic activities catalyzed by KdpD , we proposed that binding of UspC to KdpD scaffolds the KdpD/KdpE phosphorylation cascade , resulting in a shift to the phosphorylated form of KdpE by further stabilizing the KdpD/KdpE ∼ P/DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Binding of UspC to the N-terminal domain of KdpD stabilizes the complex between KdpD , phosphorylated KdpE and the DNA under this condition and thus attenuates the inhibitory effect of K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_73">
The Nterminal domain of KdpD was found to be important for the stabilization of the KdpE ∼ P/DNA complex that results in a shift of the equilibrium to the phosphorylated form of KdpE .21 Therefore , it was suggested that UspC increases and/or stabilizes the phosphorylated form of KdpE in a complex with the DNA ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></label>
<input id="act_similarity_73", type="checkbox">
<div class="ACT_similar">
<li>Instead , the Nterminal domain of KdpD was earlier found to be important for the stabilization of the KdpE ∼ P/DNA complex that results in a shift of the equilibrium to the phosphorylated form of KdpE .21 Therefore , we proposed that binding of UspC to KdpD increases and/or stabilizes the phosphorylated form of KdpE in a complex with the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li></div>
<li>Very little unphosphorylated KdpE bound to the Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Upon the incubation of KdpE with KdpD * inthepresenceof ATP , its DNA-bind-ing ability with respect to the kdpABC promoter sequence was greatly enhanced ( more than 10-fold ) , as shown in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>These samples were immediately analysed by SDS-PAGE , followed by autotradiography KdpD - + KdpD - + KdpE + + KdpE + + Phospho-KdpE efficiently stimulates the in vitro Kdp transcription To examine directly the biochemical consequences of phosphorylation of KdpE , we performed an in wYro trans-cription experiment , involving phospho-KdpE and the kdpABC promoter DNA as a template , as shown in Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Although the precise signal ( s ) remain unknown , KdpD integrates multiple inputs , including drops in extracellular K concentration [ 19 ] , changes + resulting from ionic hyperosmolarity [ 20 ] , changes in membrane lipid composition [ 21 ] and ATP levels [ 22 ] , to activate KdpE by phosphorylation [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>When 20 mM MgCl2 was added to the reaction mixture , however , radiolabelled KdpE was found to decrease rapidly , suggesting that the phosphoryl group in KdpE is less stable , being hydrotysed readily in the presence of MgCl2 and KdpD * , under the in vitro conditions used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In any event , when KdpE was added to the reaction mixture , the amount of radiolabelled KdpD * decreased , and KdpE was concomitantly and stably radiolabelled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In any case , it was revealed that phosphoKdpE was stably accumulated in vitro by incubation with KdpD * only under conditions in which Mg ^ ^ ions were omitted from the reaction mixture , i.e. we succeeded in preparing phospho-KdpE in a stable form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Importantly , the presence of UspC significantly enhanced the fraction of phosphorylated KdpE under this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Quantiﬁcation revealed that the KdpE ~ P signal was twofold enhanced in the presence of IIANtr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>First of all , to confirm that the KdpD * purified in this study is capable of phosphorylating the purified KdpE in the presence of ATP , KdpD * was incubated with 0.02 mM [ - Y - ^ ^ PJ-ATP in the presence of 50 mM KCI and 5mM CaCb - The time-oourse of the presumed KdpD * autophosphorylation was followed by means of SDS-PAGE and autoradiography ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In the absence of K + or at a low concentration of K + ( 50 mM ; Fig. 2a ) , phosphorylation of KdpE was unaffected by the presence of UspC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Further aliquots were removed at different times and mixed with SDS-sample buffer .61 The phosphorylation was done in the presence or in the absence of UspC ( equimolar to KdpE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>The phosphorylation was carried out in the absence or presence of IIANtr in concentrations , which were equimolar to 4 mM KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>The reactions were performed in the absence or presence of IIANtr ( equimolar to KdpE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>To test kinase activity , KdpD was ﬁrst phosphorylated in the absence or presence of IIANtr for 3.5 min and subsequently nonphosphorylated KdpE protein was added to the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>To test the effect of IIANtr on KdpD phosphatase activity , phosphorylated KdpE protein was isolated and subsequently inverted membrane vesicles containing KdpD were added in the absence or presence of IIANtr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Previous control experiments have shown that dephosphorylation of KdpE ~ P depends on the presence of KdpD in these assays ( Jung and Altendorf , 1998a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Chemical stability of phospho-KdpD * and phospho - KdpE Phosphorylation of both KdpD * and KdpE was carried out with [ • V - ̂ ^ Pj-ATP under the same conditions as described in the section on phosphorylation experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>was incubated with KdpD ' and KdpE in the absence and presence of ATP ( 0.02 mM ) in 20 ( il of a buffer for 10 min , as described in the Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Various amounts of KdpE ( 5-100 pmol ) were incubated at 25 ° C for 2 h with KdpD * ( 12 pmol ) in the absence and presence of ATP ( 0.02 mM ) under the same conditions as given in the section on phosphorylation experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Phosphorylated KdpE was incubated with proteoliposomes containing KdpD in the absence and in the presence of UspC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Non-phospho-KdpE was mixed with either the kdpABC promoter DNA or the lacUV5 promoter DNA in the presence and absence of KdpD * , and then incubated for 120 min in the presence of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Under conditions of salt stress , which trigger an increase of the intracellular K + content far above the normal level and almost prevent phosphorylation of KdpE due to the inhibitory effect of K + on the kinase activity of KdpD , UspC scaffolds KdpD/KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>UspC activates the KdpD/KdpE phosphorylation cascade in vitro at a high concentration of K + The effect of either UspA , UspC , or UspD on the in vitro reconstructed KdpD/KdpE signaling cascade was tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Among these proteins only UspC stimulated the in vitro reconstructed signaling cascade ( KdpD → KdpE → DNA ) resulting in phosphorylation of KdpE at a K + concentration that would otherwise almost prevent phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>According to this model , UspC which is induced under salt stress24 ( see also Fig. 4b ) scaffolds the KdpD/KdpE signaling cascade resulting in an increase of kdpFABC expression at an internal concentration of K + that would otherwise almost prevent phosphorylation of KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Despite the inhibitory effect of K + , the pre-sence of UspC stimulated the phosphorylation cascade , resulting in a fourfold higher phosphorylation of KdpE ( 0.12 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Intriguingly , the presence of puriﬁed IIANtr stimulated the phosphorylation cascade at all tested K + concentrations , resulting in a stable phosphorylation of KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Nevertheless , it was found that this mutant KdpD can not function normally in vivo , i.e. it appears to be unable to activate KdpE in response to hyperosmotic stress ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>KdpD then phosphorylates KdpE , which activates the kdp operon ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576059" target="_blank">8576059</a>]<br></li>
<li>Thus , the total cell lysate was KdpD - + KdpD - + KdpE + + KdpE + + Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>However , it was rather difficult to determine the in vitro properties of the phosphorylated form of KdpE itself , because we used a crude membrane fraction enriched with KdpD to demonstrate KdpE phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Indeed , presence of IIANtr strongly enhanced autophosphorylation of KdpD ( i.e. 10-fold , when compared at t = 2 min ) and the subsequent phospho-transfer to KdpE ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>As a result , IIANtr increased the phosphorylation level of KdpD/KdpE twofold ( no K + ) , threefold ( 50 mM K + ) and 10-fold ( 250 mM K + ) respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Fig. 6B , addition of IIANtr-H73A enhanced the phosphorylation level of KdpD/KdpE 10-fold in the presence of 250 mM K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>As it is known that phosphorylation of the in vitro reconstructed KdpD/KdpE signalling cascade is inhibited by K + ( Heermann et al. , 2009 ) , the inﬂuence of IIANtr on the phosphorylation cascade was tested at different K + concentrations ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>In vitro experiments using KdpD reconstituted in proteoliposomes and puriﬁed response-regulator KdpE showed that phosphorylation of both proteins is strongly enhanced in the presence of IIANtr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Fig. 3A , B In vitro phosphorylation of KdpD and phosphotransfer to KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>IIANtr ( A ) and IIANtr-H73A ( B ) stimulate phosphorylation of KdpD and KdpE in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>We demonstrated previously that the response-regulator , KdpE , is capable of undergoing phosphorylation mediated by the sensory protein kinase , KdpD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Ntr IIA increases the phosphorylation of KdpD and KdpE in vitro Our data demonstrated that IIANtr enhances kdpFABC transcription and interacts with the sensor kinase KdpD , which controls together with the response-regulator KdpE activity of the corresponding promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Therefore , we tested whether IIANtr interacts with the histidine kinase KdpD or the response-regulator KdpE , which control expression of the kdp operon in response to K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Therefore , the most likely explanation was that IIANtr modulates KdpD / KdpE signalling , resulting in increased levels of phosphorylated response-regulator KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Two-hybrid data and biochemical analysis revealed that IIANtr interacts with sensor kinase KdpD and stimulates kinase activity , resulting in increased levels of phosphorylated response-regulator KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>This interaction strongly stimulates KdpD kinase activity resulting in increased levels of phosphorylated response-regulator KdpE , which in turn increases kdpFABC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>It was also suggested that the central hydrophobic domain of KdpD , which is conceivably responsible for membrane anchoring of this protein , ptays a role in the signalling mechanism underlying KdpE phosphorylation in response to hyperosmotic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Although it is clear that the phosphorylation of KdpE mediated by KdpD plays a central role in activation of the kdpABC operon , and it is also probable that KdpD changes its own kinase activity in response to hyperosmotio stress , many mechanistic questions remain to be answered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>The divalent cation , Mg ^ '' ^ , may stimulate the presumed phosphatase activity of KdpD * or the presumed autophosphatase activity of KdpE , although the mechanism underlying the effect of Mg ^ ^ remains to be examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>The KdpD-KdpE pair is typical in that the larger is sensor membrane bound while the smaller has extensive homology with other effectors and appears to act by stimulating transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The KdpD-KdpE pair appears to be the sole system that responds to turgor as such and is distinct from responses to the osmolarity of the medium that are independent of changes in turgor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Fig. 5 ( legend on previous page Discussion The sensor kinase KdpD catalyzes autophosphor-ylation by ATP , transfer of the phosphoryl group to KdpE and dephosphorylation of KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>The auto ¬ phosphorylation of KdpD and hence the phosphorylation of KdpE are maximal when the intracellular K + content is low , the concentration of ATP is high , and the ionic strength is increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>The results showed that radiolabelled KdpD ' increased progressively in the absence of KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">KdpE protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The corresponding residue in KdpE is the bbranched residue T217 , the branch point sits close to the peptide backbone , which makes it unlikely to penetrate the minor groove formed by T-rich sequences between the S1 and S2 half-sites of kdpFABCBS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To test kinase activity , KdpD was ﬁrst phosphorylated in the absence or presence of IIANtr for 3.5 min and subsequently nonphosphorylated KdpE protein was added to the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>The gap in the lower panel represents a three residue insertion in few of the KdpE orthologs used in sequence alignment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The 15-residue-long , solvent exposed helix a8 of KdpE contains five positively charged residues that can potentially interact with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The residues targeted for mutagenesis in KdpE are boxed , the triangles represent residues involved in base specific interactions in PhoB-DNA complex ( PDB code : 1GXP ) , and the residue numbering is that of KdpE sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The recent structural characterization of full-length KdpE , another OmpR/PhoB family member , bound to its direct repeat site conﬁrmed these different domain symmetries while identifying an additional level of asymmetry resulting from intramolecular contacts between the receiver and DNA-binding domains within one KdpE subunit ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>For T215 of KdpE , variants with mutations at equivalent residues in OmpR and PhoB are also defective in DNA-binding due to loss of H-bonding with the DNA backbone [ 28,49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Further studies are necessary to identify the role of phosphorylation of the receiver-domain in either enhancing the affinity of binding to DNA and/or in cooperative interactions at the two half-sites of the KdpE recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Therefore , residues R193 , R200 and T214 of KdpE were mutated and the ability to promote gene expression in vivo and DNA-binding was analyzed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Therefore , it was revealed that although both forms of KdpE exhibit a specific DNA-binding ability , the phosphorylation of KdpE results in enhancement of its specific DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>DNA-binding analysis of phospho-KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Mutant KdpE were incapable of DBD binding to DNA in EMSA ( Fig. 3B ) , even at a 1:8 molar ratio of DNA to protein , which underscores the importance of these residues in stabilizing DNA-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>In conclusion , our studies identified residues of KdpE that DBD participate in DNA-binding , the location of the half-sites on the DNA ; and the nucleotide bases essential for protein-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Because R193 is conserved among the KdpE family , and when mutated abrogates DNA interaction , it may play a role in base-specific recognition as suggested by the position in the superposition ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Residues in KdpE targeted for mutagenesis namely R193 , R200 and T215 are indicated by stars , whereas the + sign points to R219 of PhoB that interacts with the minor groove of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Analysis KdpEDBD structure superposed on PhoB-DNA complex revealed : ( i ) R193 , conserved only among KdpE orthologs ( Fig. 2B lower panel ) had the potential to form base specific contacts ( Fig. 2C ) and ( ii ) residues conserved in OmpR/PhoB family namely R200 and T214 of KdpE superpose well onto R203 and T217 of PhoB that form a hydrogen bond ( R203 NH1 T217OG1 in PhoB ) in addition to salt bridges to the backbone in PhoB-DNA structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>In the PhoB-DNA complex structure , R203 , the residue equivalent to R200 of KdpE forms a salt bridge with O1P on the backbone of nucleotide T14 [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>These stimuli are converted to signals via a series of phosphotransfer events involving autophosphorylation , transphosphorylation , and dephosphorylation reactions : The first two steps constitute the activation process of signaling , and the latter involves termination Expression of KdpFABC , a K pump that restores osmotic balance , is controlled by binding of the response regulator KdpE to + a specific DNA sequence ( kdpFABCBS ) via the winged-helix-turn-helix type DNA-binding domain ( KdpEDBD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To maintain continuity with the structure of the N-terminal receiver-domain of KdpE [ 25 ] , the b-strands and a-helices of KdpEDBD are labeled starting with b-6 and a-6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>It was also suggested that the central hydrophobic domain of KdpD , which is conceivably responsible for membrane anchoring of this protein , ptays a role in the signalling mechanism underlying KdpE phosphorylation in response to hyperosmotic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>The C terminus is surely cytoplasmic , since it has the kinase domain that must interact with ATP and with KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Functional importance of the hydrophobic domain of KdpD in signal transduction In this study , so far we have used KdpD * as a practical aid for preparing phospho-KdpE in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>The purified polypeptide , KdpD * , containing the intact C-terminal catalytic domain , is able to exhibit KdpE phosphorytation ability , at least In vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>UspC does not affect the enzymatic activities of KdpD/KdpE directly , but binding of UspC to the N-terminal domain of KdpD shifts the equilibrium to the phosphorylated form of KdpE by stabilizing the KdpD/KdpE ∼ P/DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Binding of UspC to the N-terminal domain of KdpD stabilizes the complex between KdpD , phosphorylated KdpE and the DNA under this condition and thus attenuates the inhibitory effect of K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Besides stimulus perception this domain has an important role in stabilizing the interaction between KdpE and its corresponding DNA-binding site .21 Moreover , within this part of KdpD , a domain was identified which shares similarities to the Usp proteins of the UspA subfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Moreover , this domain stabilizes binding of KdpE to its target site on the DNA and contains the UspC interaction surface ( Heermann et al. , 2003 ; 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_28">
The N-terminal input domain of the sensor kinase KdpD of Escherichia coli stabilizes the interaction between the cognate response regulator KdpE and the corresponding DNA-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></label>
<input id="dom_similarity_28", type="checkbox">
<div class="DOM_similar">
<li>Heermann , R. , Altendorf , K. , and Jung , K. ( 2003 ) The N-terminal input domain of the sensor kinase KdpD of Escherichia coli stabilizes the interaction between the cognate response regulator KdpE and the corresponding DNA-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li></div>
<li>This is , to our knowledge , the first example for an interaction between one of the soluble Usp proteins with an Usp domain integrated within a larger protein , although it is known that Usp proteins are able to form dimers or higher oligomers .22,26 In summary , binding of UspC to KdpD enhances the amount of phosphorylated KdpE at a concentration of K + that would otherwise almost block phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Here , we report a specific interaction between one of the soluble single-domain Usp proteins , UspC , with a Usp domain in KdpD and its role to scaffold KdpD/KdpE signaling in response to salt stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Therefore , we hypothesized that the Usp domain within KdpD might interact with one of the soluble UspA-subfamily proteins of E. coli and thereby might influence the KdpD/KdpE signaling cascade .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Thus , phos-pho-KdpE characterized in this study appears to contain a phosphoryl group at an aspartate residue ( probably at Asp-52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Of particular interest is the conservation of Asp-9 and Asp-52 in KdpE , since these residues are in positions homologous to those of the residues identified as being important in the phosphorylation of the homologous CheY protein ( 6 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Asp-57 , the phosphorylated residue of CheY , is homologous to Asp-52 of KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The fact that residues beginning with the fourth amino acid of KdpE are identical to those of its homologs is a strong argument that the protein begins with the start codon suggested here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Such an arginine residue is conserved at structurally equivalent positions in many members of the OmpR/PhoB family , with the exception of KdpE , OmpR , and DrrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>After alignment is optimized by adding a maximum of two gaps in any one sequence , 36 % of the residues of KdpE are identical to those of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Although , KdpE does not contain identical residues ( it is Y191 , I194 , H198 and T217 respectively at positions equivalent to those of PhoB shown in the previous line ) , the properties of three side chains are similar except for KdpE T217 and PhoB R219 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">KdpE evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>B. Conservation of the sequence in the wHTH motif across members of the OmpR/PhoB family ( upper panel ) and between KdpE orthologs ( lower panel ) presented in logo format derived from multiple sequence alignments [ 61 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>ompFPro represents the recognition site of OmpR , a RR belonging to the same protein family as KdpE ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Such an arginine residue is conserved at structurally equivalent positions in many members of the OmpR/PhoB family , with the exception of KdpE , OmpR , and DrrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The inability of mutant KdpE R193A to bind to DNA is interesting because in most other family members the equivalent residue is of opposite charge as observed in PhoB ( D196 ) and OmpR ( D202 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>, 1989 , and references therein ) , it was suspec-ted that KdpD and KdpE may belong to such a family of adaptive response systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>174 , No. 7 KdpD an kdpABC d KdpE , Proteins That Control Expression of the Operon , Are Members of the Two-Component Sensor-Effector Class of Regulators MARK 0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The adjacent kdpDE operon codes for the two proteins KdpD and KdpE , which regulate the expression of the structural genes ( 18 ) and which belong to the family of sensor kinase/response regulator systems ( 19 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608856" target="_blank">10608856</a>]<br></li>
<li>The regulatory proteins KdpD and KdpE , which belong to the family of sensor kinase and response regulator systems , respectively , control kdpFABC expression in E. coli ( Polarek et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>The sequences of KdpD and KdpE show they are members of the sensor-effector class of regulatory proteins : the C-terminal half of KdpD is homologous to sensors such as EnvZ and PhoR , and KdpE is homologous to effectors such as OmpR and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_10">
KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-compo-nent sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></label>
<input id="evo_similarity_10", type="checkbox">
<div class="EVO_similar">
<li>K. and Epstein , W. : KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8331081" target="_blank">8331081</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282684" target="_blank">8282684</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576059" target="_blank">8576059</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8388874" target="_blank">8388874</a>]<br></li></div>
<li>J Bacteriol 171:1192 -- 1195 Walderhaug MO , Polarek JW , Voelkner P , Daniel JM , Hesse JE , Altendorf K , Epstein W ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_14">
Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M. , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensoreffector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></label>
<input id="evo_similarity_14", type="checkbox">
<div class="EVO_similar">
<li>Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M. , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by KdpE</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>These stimuli are converted to signals via a series of phosphotransfer events involving autophosphorylation , transphosphorylation , and dephosphorylation reactions : The first two steps constitute the activation process of signaling , and the latter involves termination Expression of KdpFABC , a K pump that restores osmotic balance , is controlled by binding of the response regulator KdpE to + a specific DNA sequence ( kdpFABCBS ) via the winged helix-turn-helix type DNA binding domain ( KdpEDBD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The residues targeted for mutagenesis in KdpE are boxed , the triangles represent residues involved in base specific interactions in PhoB-DNA complex ( PDB code : 1GXP ) , and the residue numbering is that of KdpE sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The samples were : tane 1 , total cell lysate of MH1160 recA/vector ; lane 2 , total cell lysate of MH1160 recA/pRUN-SD ; lane3 , precipitate from total cell lysate of MH1160 reo4/pRUN-SD ; lane 4 , DEAE-cellulose chromatography ; lane 5 , Super-ose-12 chromatography ; lane M , moiecular mass markers in kDa NaOH Tris-HCI HCi 10 20 30 10 20 30 10 20 30 Hme ( min ) Phospho-KdpD Chemistry of phosphorylation of KdpD and KdpE The following crucial issue had to be addressed before we oould determine the biochemical properties of phosphoKdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>To identify YehT-regulated genes , we compared the transcriptomes of E. coli cells overproducing either YehT or the RR KdpE ( control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>To validate the data obtained by the transcriptome analysis , we probed the transcription levels of all genes identiﬁed in strains MG1655 and MG2 following overproduction of YehT-D54E or KdpE ( negative control ) by Northern blotting ( see Table S2 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li>
<li>We reasoned that comparative biochemical studies on the EnvZ-OmpR and KdpD-KdpE systems should provide a general insight into signal-transduction in response to environmental osmotic stimuli , in which a mechanism that involves coupling of an external physical force to internal genetic regulation might be implied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In this class , KdpD/KdpE , NarP and NtrB/NtrC are known to be involved in response to turgor pressure , nitrate and nitrogen availability respectively ( Laimins et al. , 1981 ; Darwin and Stewart , 1995 ; Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Previous control experiments have shown that dephosphorylation of KdpE ~ P depends on the presence of KdpD in these assays ( Jung and Altendorf , 1998a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>The expression of the kdp genes is under control of the regulatory proteins KdpD and KdpE , which are chromosomally encoded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608856" target="_blank">10608856</a>]<br></li>
<li>In this case the expression of kdpFABC genes was under control of the chromosomally encoded regulatory proteins KdpD and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608856" target="_blank">10608856</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_11">
KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-compo-nent sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></label>
<input id="rp_similarity_11", type="checkbox">
<div class="RP_similar">
<li>K. and Epstein , W. : KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8388874" target="_blank">8388874</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576059" target="_blank">8576059</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8331081" target="_blank">8331081</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282684" target="_blank">8282684</a>]<br></li></div>
<li>J Bacteriol 171:1192 -- 1195 Walderhaug MO , Polarek JW , Voelkner P , Daniel JM , Hesse JE , Altendorf K , Epstein W ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_15">
Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M. , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensoreffector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></label>
<input id="rp_similarity_15", type="checkbox">
<div class="RP_similar">
<li>Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M. , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li></div>
<li>We suggest that phosphoKdpE is a positive effector of Kdp expression and that low turgor pressure causes KdpD to phosphorylate KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Identification and localization of KdpD and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>To see if the phosphorylation state of IIANtr is important for triggering phosphorylation of KdpD / KdpE , we also tested the puriﬁed non-phosphorylatable IIANtr-H73A variant ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Heermann , R. , Weber , A. , Mayer , B. , Ott , M. , Hauser , E. , Gabriel , G. , et al. ( 2009 ) The universal stress protein UspC scaffolds the KdpD/KdpE signaling-cascade of Escherichia coli under salt stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>The universal stress protein uspC scaffolds the KdpD/KdpE signaling-cascade of Escherichia coli under salt stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376594" target="_blank">23376594</a>]<br></li>
<li>It was also suggested that the central hydrophobic domain of KdpD , which is conceivably responsible for membrane anchoring of this protein , ptays a role in the signalling mechanism underlying KdpE phosphorylation in response to hyperosmotic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Expression requires KdpD , a 99-kDa membrane protein , and KdpE , a 25-kDa soluble cytoplasmic protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>We then demonstrated that KdpD exhibits protein-kinase-activity , and that KdpE is capable of undergoing phosphorylation through phosphotransfer from KdpD ( Nakashima ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>This increases phosphorylation of response regulator KdpE , which enhances transcription initiation at the kdpFABC promoter and thereby synthesis of the high-affinity K + transporter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>b-Galactosidase assay Methods Protein expression and purification DNA sequences encoding E. coli KdpE ( Escherichia coli str .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Mol Microbiol 6:1777 -- 1784 Ohwada T , Sagisaka S ( 1987 ) An intermediate and steep increase in ATP concentration in response to reduced turgor pressure in Escherichia coli B. Arch Biochem Biophys 259:157 -- 163 Nakashima K , Sugiura A , Kanamaru K , Mizuno T ( 1993a ) Signal transduction between the two regulatory components involved in the regulation of the kdpABC operon in Escherichia coli : phosphorylation-dependent functioning of the positive regulator , KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>Molecular Microbiology ( 1993 ) 7 ( 1 ) , 109-116 Signal transduction between the two regulatory components involved in the regulation of the kdpABC operon in Escherichia coli : phosphorylation-dependent functioning of the positive regulator , KdpE K. Nakashima , A. Sugiura , K. Kanamaru and T. Mjzuno * Laboratory of Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464 , Japan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_30">
Nakashima , K. , Sugiura , A. , Kanamaru , K. and Mizuno , T. ( 1993 ) Signal transduction between the 2 regulatory components involved in the regulation of the kdpABC operon in Escherichia coli-phosphorylation-dependent functioning of the positive regulator , KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></label>
<input id="rp_similarity_30", type="checkbox">
<div class="RP_similar">
<li>Signal transduction between the two regulatory components involved in the regulation of the kdpABC operon in Escherichia coli-phosphorylation-dependent functioning of the positive regulator KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8331081" target="_blank">8331081</a>]<br></li>
<li>Signal transduction between the two regulatory components involved in the regulation of the kdpABC operon in Escherichia coli : phosphorylation-dependent functioning of the positive regulator , KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282684" target="_blank">8282684</a>]<br></li></div>
<li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">KdpE binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The variations in sequences between a8 helices of PhoB and KdpE possibly reflect the differences in recognition sites of the two proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>These results concur with footprinting analysis identifying a 23 bp KdpE binding element situated between 272 and 250 of the transcription start site for the kdpFABC operon [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>EMSA analyses of KdpEDBD with truncated DNA molecules showed that a 21 bp fragment with only 3 bp beyond the 59 end of S1 and only 1 bp beyond the 39 end of S2 is sufficient for binding to KdpE ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Electrophoretic Mobility Shift Assay ( EMSA ) A 30 bp double-stranded DNA representing the binding site ( kdpFABCBS ) for KdpE [ 29 ] synthesized and purified by HPLC by IDT Inc. ( www.idtdna.com ) for use in EMSA and AUC analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>B. EMSA showing effects of mutations in KdpE on interaction with the 30 bp DNA fragment representing its + + binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Very little unphosphorylated KdpE bound to the Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>The residues targeted for mutagenesis in KdpE are boxed , the triangles represent residues involved in base specific interactions in PhoB-DNA complex ( PDB code : 1GXP ) , and the residue numbering is that of KdpE sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>It is possible that ArcA-P binds to the icdA promoter as a dimer of dimers ; one dimer binds DR1 and DR2 , as depicted in the PhoB and KdpE DNA cocrystal structures ( 15 , 18 ) , and the second dimer binds DR3 but only weakly to adjacent DNA sequence , such that no footprint is observed ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>When KdpE is phosphorylated , it takes on an active form that exhibits relatively high affinity with the 23 bp sequence ( nucleotide positions - 7 2 to -50 ) immediately upstream of the canonical - 3 5 and - 1 0 regions of the kdpABC operon ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Identifying the binding sites for KdpE in kdpFABCBS DNA To identify KdpE binding half-sites and their specific sequences , a multiple sequence alignment of regions upstream of the kdpFABC operon from a variety of bacteria was generated ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>A 270 bp DNA fragment end-labelled with ^ ^ P , which encompasses the kdpABC promoter region and its upstream sequence , was incubated with either non-phos-pho-KdpE or phospho-KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>The results of preliminary DNase I footprinting analysis with phospho-KdpE revealed that it recognizes the same specific nuoleotide sequence as that recognized by non-phospho-KdpE , namely , the region between - 7 2 and - 5 0 in the kdpABC promoter ( the transcriptional start site was taken as +1 ) ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>In this respect , we demonstrated previously that KdpE is indeed a DNA-bind-ing protein that specifically and directly binds to the upstream region of the kdpABC promoter ( Sugiura ef al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Phosphorylated KdpE binds to a sequence upstream of the kdpFABC promoter and stimulates transcription initiation ( Sugiura et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_16">
This upstream sequence was demonstrated to be the target site for the frans-acting activator , KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></label>
<input id="sit_similarity_16", type="checkbox">
<div class="SIT_similar">
<li>This c/s-acting upstream sequence was revealed to be the target site for the frans-acting activator , KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></li></div>
<li>As predicted from its sequence , KdpE is a soluble protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>This indicated that the KdpE protein binds directly to the sequence upstream of the -35 and -10 regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by KdpE</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Although it is clear that the phosphorylation of KdpE mediated by KdpD plays a central role in activation of the kdpABC operon , and it is also probable that KdpD changes its own kinase activity in response to hyperosmotio stress , many mechanistic questions remain to be answered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>Transcription initiation at the kdpFABC promoter is positively controlled by the two-component system KdpD/KdpE in response to K + availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>Expression of the Kdp system is controlled by the two-component system ( TCS ) KdpD/KdpE , which activates kdpFABC transcription under conditions of K + limitation or osmotic salt stress ( Walderhaug et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400808" target="_blank">19400808</a>]<br></li>
<li>The mechanism by which UspC interacts with the KdpD/KdpE system and therefore modulates finetuning of kdpFABC expression is reminiscent of the scaffolding proteins known from eukaryotes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></li>
<li>Walderhaug , M.O. , Polarek , J.W. , Voelkner , P. , Daniel , J.M , Hesse , J.E. , Altendorf , K. , and Epstein , W. ( 1992 ) KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_6">
KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-compo-nent sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101563" target="_blank">19101563</a>]<br></label>
<input id="tu_similarity_6", type="checkbox">
<div class="TU_similar">
<li>KdpD and KdpE , proteins that control expression of the kdpABC operon , are members of the two-component sensor-effector class of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8388874" target="_blank">8388874</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8576059" target="_blank">8576059</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8331081" target="_blank">8331081</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282684" target="_blank">8282684</a>]<br></li></div>
<li>Two proteins , KdpD and KdpE , encoded by genes in an operon adjacent to that encoding the Kdp ATPase , are required to express Kdp ( 34 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The expression of the kdp genes is under control of the regulatory proteins KdpD and KdpE , which are chromosomally encoded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608856" target="_blank">10608856</a>]<br></li>
<li>In this case the expression of kdpFABC genes was under control of the chromosomally encoded regulatory proteins KdpD and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608856" target="_blank">10608856</a>]<br></li>
<li>Mutation of these residues renders KdpE incapable of inducing expression of the kdpFABC operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Expression of the operon is regulated by the membrane-bound sensor kinase KdpD and the cytosolic response regulator KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115059" target="_blank">12115059</a>]<br></li>
<li>KdpE of Clostridium acetobutylicum is a highly speciﬁc response regulator controlling only the expression of the kdp operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>