<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - LexA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">LexA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">LexA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>umuDC and mucAB operons whose products are required for UV light - and chemical-induced mutagenesis : UmuD , MucA , and LexA proteins share homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682790" target="_blank">8682790</a>]<br></li>
<li>Perry , K.L. , S.J. Elledge , B.B. Mitchell , L. Marsh and G.C. Walker ( 1985 ) umuDC and muchB operons whose products are required for UV light - and chemical-induced mutagenesis : UmuD , MucA , and LexA proteins share homology , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>Cell 37 : 675-682 Perry KL , Elledge SJ , Mitchel BB , Marsh L , Walker GC ( 1985 ) umuDC and mueAB operons whose products are required for UV light and chemical-induced mutagenesis , UmuD , MucA and LexA proteins share homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2972909" target="_blank">2972909</a>]<br></li>
<li>Ann Rev Biochem 3:293 -321 Perry KL , Elledge SJ , Mitchell BB , Marsh L , Walker GC ( 1985 ) umuDC and mucAB operons whose products are required for UV light - and chemical-induced mutagenesis : UmuD , MucA , and LexA proteins share homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3025556" target="_blank">3025556</a>]<br></li>
<li>Since the predicted targets of LexA have already been experimentally verified in [ 12,16 ] , we thus chose the regulation of lipA by PdhR as best candidate for an experimental validation of a predicted interaction To confirm the presence of a PdhR binding site in the promoter region of lipA , we performed electro-phoretic mobility shift assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>s simultaneously drives and represses the 70 induction of promoters controlled by LexA and Fur To test the hypothesis that when s is underexpressed , 70 silencing of certain s - driven promoters is diminished , we 70 further analyzed the activity of a set of s - driven promo-70 ter : : luxCDABE in normal and low s cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21396442" target="_blank">21396442</a>]<br></li>
<li>gcvA allele * β-Galactosidase activity † in cells grown in : ‡ GM Glycine Inosine GcvA mutants unable to repress gcv expression are impaired in their ability to interact with GcvR None 21 To test whether the gcvA mutants described above are defective in their ability to interact with GcvR , each mutant allele was cloned into pMS604 and its interaction with GcvR measured using the LexA heterodimerization system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>of genes involved in the response to DNA damage ( SOS response ) , including RecA and LexA ; in the presence of single-stranded DNA , RecA interacts with LexA , causing an autocatalytic cleavage which disrupts the DNA-binding part of LexA , leading to derepression of the SOS regulon and eventually DNA repair Catalyzes the formation of phosphodiester linkages between 5 - phosphoryl and 3 - hydroxyl groups in double-stranded DNA using NAD as a coenzyme and as the energy source for the reaction ; essential for DNA replication and repair of damaged DNA Component of the leucine , isoleucine , valine ( threonine ) transport system , which is one of the two periplasmic binding-protein-dependent transport systems of the high-afﬁnity transport of the branched-chain amino acids ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol and in the physiological short-term adaptation to glucose limitation ; decreases after benzoic acid treatment and phosphate limitation late and in the presence of leucine Component of the leucine-speciﬁc transport system , which is one of the two periplasmic binding-protein-dependent transport systems of the high-afﬁnity transport of the branched-chain amino acids ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol ; decreases after benzoic acid treatment 5.42 / 44,439 5.93 / 50,340 ( DIGE 4.5 -- 6.5 ) LeuD 5.16 / 22,356.29 P30126 3-Isopropylmalate dehydratase small subunit Probable HTH-type transcriptional-regulator Leucyl-tRNA synthetase 5.20 / 23,000 ( DIGE 4.5 -- 6.5 ) 179 , 321 5.80 / 35,694.73 P10151 LeuO 5.77 / 34,515 293 5.11 / 98,831 5.16 / 97,233.76 P07813 LeuS LexA 6.23 / 22,357.72 P0A7C2 LexA repressor 179 5.39 / 73,606.07 DNA ligase P15042 LigA 79 1 LivJ 5.28 / 41,960 5.28 / 36,772.53 P0AD96 Leu/Ile/Val-binding protein 29 , 61 , 71 , 179 , 228 , 258 , 293 , 294 , 311 , 321 5.18 / 39,439 ( DIGE 4.5 -- 6.5 ) 5.30 / 38,773 ( DIGE 4.5 -- 6.5 ) 5.31 / 40,279 ( 4.5 -- 5.5 ) 5.21 / 16,964 ( 4.5 -- 5.5 ) 5.09 / 50,544 ( 5 -- 6 ) 5.23 / 50,424 ( 5 -- 6 ) 5.67 / 30,753 ( 5 -- 6 ) 5.33 / 23,392 ( 5 -- 6 ) 5.73 / 30,106 ( 5.5 -- 6.7 ) LivK 5.00 / 41,464 5.07 / 36,982.71 Leucine-speciﬁc binding protein P04816 71 , 179 , 212 , 228 , 258 , 287 , 294 , 321 5.09 / 39,439 ( DIGE 4.5 -- 6.5 ) 5.08 / 39,792 ( 4.5 -- 5.5 ) 5.56 / 28,995 ( 4.5 -- 5.5 ) 5.25 / 59,333 ( 5 -- 6 ) Continued on facing pag Induced by L-lactate aerobically Participates in the translocation of lipoproteins from the inner membrane to the outer membrane Plays a critical role in the incorporation of lipoproteins in the outer membrane after they are released by the LolA protein ; essential for E. coli viability Degrades short-lived regulatory and abnormal proteins in the presence of ATP ; induced by stress conditions such as high temperatures , nalidixic acid , and accumulation of abnormal proteins ; positively regulated by htpR Component of the glycine cleavage system as well as of the alpha-ketoacid dehydrogenase complexes ; increases during aerobic-growth and the low temp at 10 ° C ; induced by low pH during anaerobic-growth Interacts with the peptidoglycan both covalently and noncovalently ; contributes to the maintenance of the structural and functional integrity of the cell envelope Catalyzes the transformation of S-ribosylhomocysteine to homocysteine and 4,5-dihydroxy-2 ,3 - pentadione ; involved in the synthesis of autoinducer 2 , which is secreted by bacteria and is used to communicate both the cell density and the metabolic potential of the environment ; changes in cell density in a process called quorum sensing ; induced by low pH Involved in lysine biosynthesis Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Synthesizes a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The phosphoryl group from PEP is transferred to the phosphoryl carrier protein HPr by enzyme I ( PtsI ) ; phosphoHPr then transfers it to the permease ( enzymes II/III ) ; HPr is common to all PTS ; regulated by the cAMP-CRP complex and also by growth-on-glucose ; induced by low pH and phosphonate growth See PtsH ; decreases after benzoic acid treatment but increases at phosphonate growth Plays an important role in de novo pathway of purine nucleotide biosynthesis : AMP biosynthesis ; increases after benzoic acid treatment Involved in IMP biosynthesis : 5-amino-1 - ( 5-phospho-D-ribosyl ) imidazole-4 - carboxamide from N ( 2 ) - formyl-N ( 1 ) - ( 5-phospho - dribosyl ) glycinamide ( step 5 ) See PurB : step 4 ; increases after benzoic acid treatment See PurB : step 2 ; increases after benzoic acid treatment See PurB : step 3 PstS P0AG82 Phosphate-binding periplasmic 6.92 / 34,421.76 6.85 / 45,258 ( 6 -- 11 ) protein Pta P0A9M8 Phosphate acetyltransferase 5.28 / 77,040.90 5.29 / 85,417 179 , 228 , 274 , 293 , 294 PtrA 179 Protease III P05458 5.66 / 105,113.29 5.65 / 9,119.37 5.65 / 10,366 P0AA04 Phosphocarrier protein HPr PtsH 27 , 228 , 293 , 294 PtsI 4.78 / 59,810 Phosphoenolpyruvate-protein phosphotransferase P08839 56 , 179 , 228 , 293 , 294 , 321 4.78 / 63,561.84 4.91 / 63,671 ( DIGE 4.5 -- 6.5 ) PurA P0A7D4 Adenylosuccinate synthetase 179 , 321 5.30 / 44,524 ( DIGE 4.5 -- 6.5 ) 5.30 / 43,494 ( DIGE 4.5 -- 6.5 ) 5.32 / 47,213.76 PurB 179 5.68 / 51,542.81 P0AB89 Adenylosuccinate lyase PurC 5.09 / 26,148 ( DIGE 4.5 -- 6.5 ) 5.07 / 26,995.00 P0A7D7 Phosphoribosylaminoimidazolesuccinocarboxamide synthase Phosphoribosylamine-glycine ligase Phosphoribosylaminoimidazole carboxylase catalytic subunit Bifunctional purine biosynthesis protein Phosphoribosylaminoimidazole carboxylase ATPase subunit 179 , 321 PurD 179 , 321 5.06 / 47,667 ( DIGE 4.5 -- 6.5 ) 4.96 / 45,940.36 P15640 PurE 179 , 321 6.05 / 18,899 ( DIGE 4.5 -- 6.5 ) 6.03 / 17,649.13 P0AG18 See PurB : ﬁnal step ; increases after benzoic acid treatment See PurB : step 3 ; possesses an ATPase activity that is dependent on the presence of AIR ( aminoimidazole ribonucleotide ) ; the association of PurK and PurE produces an enzyme complex capable of converting AIR to CAIR efﬁciently under physiological condition See PurB : step 2 PurH 179 , 321 5.50 / 57,000 ( DIGE 4.5 -- 6.5 ) 5.53 / 57,329.21 P15639 5.75 / 28,100 ( 5 -- 6 ) 5.60 / 39,461.12 P09029 PurK 287 179 P08178 Phosphoribosylformylgly-cinamidine cyclo-ligase HTH-type transcriptional-repressor 4.83 / 36,722.85 PurM Repressor that binds to the purF operator and coregulates other genes for de novo purine nucleotide synthesis ; involved in regulation of purB , purC , purEK , purHD , purL , purMN , and guaBA expression ; binds hypoxanthine and guanine as inducers 4.83 / 36,722.85 PurM P0ACP7 6.3 / 38,043.62 PurR 179 Oxidizes proline to glutamate for use as a carbon and nitrogen source and also functions as a transcriptional-repressor of the put operon ; induced by proline Involved in ﬁnal step of glycolysis Involved in UMP biosynthesis : UMP from HCO3 ( step 2 ) See PyrB : step 3 See PyrB : step 4 See PyrB : step 6 ( ﬁnal ) Bifunctional protein 5.69 / 143,815.16 PykF Pyruvate kinase I P0AD61 5.77 / 50,729.42 5.65 / 56,143 ( DIGE 4.5 -- 6.5 ) 179 , 321 PyrB Aspartate carbamoyltransferase catalytic chain Dihydroorotase Dihydroorotate dehydrogenase Orotidine 5 -- phosphate decarboxylase CTP synthase P0A786 6.13 / 34,296.17 5.70 / 93,664 6.13 / 35,321 179 , 212 , 228 , 294 P05020 P0A7E1 P08244 PyrC 5.77 / 38,696.19 7.66 / 36,774.52 5.81 / 26,350.24 179 179 , 287 PyrD PyrF 7.28 / 47,786 ( 6 -- 11 ) 5.76 / 56,695 179 , 228 , 294 , 321 5.83 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.61 / 58,475 ( DIGE 4.5 -- 6.5 ) Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen ; allosterically activated by GTP when glutamine is the substrate and also activated by magnesium but inhibited by CTP and by divalent metal ions such as copper and zinc Catalyzes the phosphorylation of UMP to UDP , with ATP as the preferred donor Involved in allosteric regulation of aspartate carbamoyltransferase Essential for efﬁcient processing of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; cold-shock protein essential for E. coli cells to adapt to low temp PyrG P0A7E5 5.63 / 60,242.96 179 , 321 7.07 / 25,839.07 P0A7E9 Uridylate kinase PyrH PyrI P0A7F3 Aspartate carbamoyltransferase 6.84 / 16,989.44 6.85 / 17,023 regulatory chain 7.01 / 17,642 ( 6 -- 11 ) 179 , 212 , 228 , 287 , 294 RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 Involved in the high-afﬁnity D-ribose membrane transport system and also serves as the primary chemoreceptor for chemotaxis ; repressed in the presence of glucose but induced in the physiological short-term adaptation to glucose limitation Involved in ribose metabolism Catalyzes the hydrolysis of ATP in the presence of single-stranded DNA , the ATP-dependent uptake of single-stranded DNA by duplex DNA , and the ATP-dependent hybridization of homologous single-stranded DNAs ; interacts with LexA , causing its activation and leading to its autocatalytic cleavage ; induced by low temp at 10 ° C as well as by cadmium chloride , hydrogen peroxide , and ACDQ Counteracts the effect of relE by means of direct proteinprotein interaction , enabling the reversion of translation inhibition ; acts as an autorepressor of relBE transcription ; increased transcription rate of relBE and activation of relE is consistent with a lower level of relB in starved cells due to degradation of relB by protease lon ; induced by amino acid starvation RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 RbsB P02925 6.85 / 30,900 5.99 / 28,474.47 D-Ribose-binding periplasmic protein 99 , 179 , 311 , 321 E 4.5 -- 6.5 ) IG 5.92 / 29,066 ( D 4.99 / 32,290.52 P0A9J6 Ribokinase P0A7G6 Protein RecA RbsK RecA 179 5.09 / 37,842.18 5.08 / 41,137 133 , 179 , 228 , 294 , 297 4.81 / 9,071.48 4.81 / 9,100 Antitoxin P0C079 RelB 99 Continued on facing pag Catalyzes the interconversion between ADP-D-glycero-beta-D-manno-heptose and ADP-L-glycero-beta-D-manno-heptose via an epimerization at carbon 6 of the heptose ; completely inhibited by ADP and ADP-glucose and partially inhibited by ATP and NADH ; induced by heat-shock Involved in carbohydrate biosynthesis and dTDP-L-rhamnose biosynthesis See RfbB RfaD P67910 ADP-L-glycero-D-manno-heptose-6-epimerase 4.80 / 34,893.17 4.85 / 36,820 4.94 / 36,106 ( DIGE 4.5 -- 6.5 ) 4.98 / 35,245 ( 4 -- 5 ) P37759 RfbB 179 dTDP-glucose 4,6-dehydratase 5.47 / 40,558.33 RfbC P37745 dTDP-4-dehydrorhamnose 3,5 - 5.48 / 21,270.11 epimerase P0AG30 Transcription termination factor 6.75 / 47,004.21 179 Facilitates transcription Rho termination by a mechanism that involves rho binding to the nascent RNA , activation of rho 's RNA-dependent ATPase activity , and release of the mRNA from the DNA template Involved in riboﬂavin biosynthesis ; repressed by heat-shock but induced by low pH Involved in ﬁnal steps of riboﬂavin synthesis ; riboﬂavin synthase is a bifunctional enzyme complex catalyzing the formation of riboﬂavin from 5-amino-6 - ( 1 - D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione and L-3 ,4 - dihydrohy-2-butanone-4 - phosphate via 6,7-dimethyl-8-lumazine ; the alpha subunit catalyzes the dismutation of 6,7-dimethyl-8-lumazine to riboﬂavin and 5-amino-6 - ( 1-D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione See RibE ; decreases after benzoic acid treatment Acetylates the N-terminal serine of ribosomal protein L7/L12 ; increases during phosphate limitation and phosphonate growth Essential for efﬁcient processing of 16S rRNA ; probably part of the 30S subunit prior to or during the ﬁnal step in the processing of 16S free 30S ribosomal subunits ; could be some accessory protein needed for efﬁcient assembly of the 30S subunit ; needed in a step prior to RbfA during the maturation of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; decreases after benzoic acid treatment P0A7J0 3,4-Dihydroxy-2-butanone 4-phosphate synthase 4.90 / 23,353.47 RibB 274 RibE P0AFU8 Riboﬂavin synthase alpha chain 5.64 / 23,444.90 5.67 / 26,821 ( 5 -- 6 ) 5.73 / 25,731 ( 5.5 -- 6.7 ) 179 , 287 RibH 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.19 / 12,875 ( 4.5 -- 5.5 ) 5.42 / 40,651 5.15 / 16,156.51 P61714 6,7-Dimethyl-8-ribityllumazine synthase Ribosomal protein-serine acetyltransferase 179 , 287 , 321 5.86 / 20,680.55 P13857 RimL 228 , 293 , 294 4.68 / 24,436 ( DIGE 4.5 -- 6.5 ) 4.61 / 20,605.43 P0A7X6 16S rRNA-processing protein RimM 321 179 321 5.09 / 35,712.71 5.39 / 32,693.56 Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA Catalyzes the formation of dTDP-glucose from dTTP and glucose-1-phosphate as well as its pyrophosphorolysis ; decreases after benzoic acid treatment Digests double-stranded RNA ; involved in the processing of rRNA precursors and of some mRNAs Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA 5.41 / 32,544 ( DIGE 4.5 -- 6.5 ) RmlA1 6.4 / 25,550.04 P0A7Y0 RNase III Rnc 179 Matures 5S rRNA from its precursors from all the rRNA genes ; cleaves RNA I , a molecule that controls the replication of colE1 plasmid DNA ; the major endoribonuclease participating in mRNA turnover in E. coli May be part of a membrane complex involved in electron transport Responsible for the end turnover of tRNA : speciﬁcally removes the terminal AMP residue from uncharged tRNA ( tRNA-C-C-A ) ; involved in tRNA biosynthesis , especially in strains lacking other exoribonucleases P21513 Rne RNase E 5.48 / 118,196.73 Electron transport complex protein P77285 RnfG 179 6.59 / 21,911.91 RNase T P30014 Rnt 179 5.19 / 23,522.7 5.13 / 24,554.25 Ribulose-phosphate 3-epimerase Ribose-5-phosphate isomerase A P0AG07 Rpe 179 Involved in nonoxidative branch of the pentose phosphate pathway ; base-induced protein One of the primary rRNA-binding proteins ; binds very close to the 3 end of the 23S rRNA and forms part of the L1 stalk ; translational repressor protein ; controls the translation of the L11 operon by binding to its mRNA See RplA ; increases at phosphonate growth RpiA 5.20 / 22,860.40 5.06 / 25,943 P0A7Z0 179 , 274 , 286 RplA 50S ribosomal protein 9.64 / 24,598.44 8.24 / 25,577 ( 6 -- 11 ) L1 P0A7L0 179 , 287 RplI 6.20 / 19,831 6.17 / 15,769 6.17 / 15,769.06 50S ribosomal protein L9 P0A7R1 29 , 179 , 228 , 286 , 287 , 293 , 294 6.71 / 10,366 5.10 / 8,830 ( 4.5 -- 5.5 ) 6.36 / 15,124 ( 6 -- 11 ) 9.04 / 17,580.4 Translational repressor protein ; controls the translation of the rplJL-rpoBC operon by binding to its mRNA Seems to be the binding site for several of the factors involved in protein synthesis and appears to be essential for accurate translation ; decreases during phosphate limitation Binds the 5S rRNA ; required for the late stages of subunit assembly Binds to 23S rRNA in the presence of protein L20 One of the proteins that binds to the 5S RNA in the ribosome , where it forms part of the central protuberance ; binds to the 5S rRNA independently of L5 and L18 ; not required for binding of the 5S rRNA/L5/L18 subcomplex to 23S rRNA May be induced under zinc-limiting conditions ; may be repressed in that case by the zinc uptake regulation protein Zur DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>[ 25 ] A. Fernandez de Henestrosa , T. Ogi , S. Aoyagi , D. Chaﬁn , J.J. Hayes , H. Ohmori , R. Woodgate , Identiﬁcation of additional genes belonging [ 26 ] J. Courcelle , A. Khodursky , B. Peter , P.O. Brown , C. Hanawalt , [ 27 ] C. Janion , Some aspects of SOS response system -- a critical survey , [ 28 ] R.P. Fuchs , S. Fujii , J. Wagner , Properties and functions of Escherichia [ 29 ] R. Costa de Oliveira , J. Laval , S. Boiteux , Induction of SOS and adaptive [ 30 ] E. Śledziewska - Gójska , C. Janion , Alternative pathways of methyl [ 31 ] E. Grzesiuk , C. Janion , The frequency of MMS induced , umuDC - [ 32 ] B.J. Bachmann , Derivation and genotype of some mutant derivatives of [ 33 ] B. Sedgwick , Oxidation of methylhydrazines to mutagenic methy - [ 34 ] M.R. Volkert , D.C. Nguyen , C. Beard , Escherichia coli gene induction [ 35 ] R. Woodgate , Construction of a umuDC operon substitution mutation in [ 36 ] C. Ho , O.I. Kulaeva , A.S. Levine , R. Woodgate , A rapid method for to the LexA regulon in Escherichia coli , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>GeÂrard , F. , Dri , A.M. and Moreau , P.L. ( 1999 ) Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>Tolerating some redundancy RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>That the enhance the DNA-repair efficiency by increasing thk be exposed to high levels of ROS generated by enzymes efficiency of induction of the LexA regulon in Pi-starved of the respiratory chain while glucose is metabolized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Finally , the finding that induction in cultures starved for Pi , survival and metabolism of of the LexA regulon in Pi-starved cells is totally starved cells was dependent upon RecBCD , RecA , RecG prevented by introduction of a recB mutation clearly and RuvABC , which are required to repair DSB ( Lloyd indicates that DSB are the main DNA damage produced & Low , 1996 ) , a damage produced mainly through during starvation for Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Second , pyruvate H-NS helps maintain a rapid degradation of arginine oxidase ( PoxB ) , which may be preferentially used to and glucose produce acetate when bacteria approach the stationary phase , is unlikely to play a key role under Pi-starvation Because H-NS is required for a full induction of the conditions because the expression of poxB is strictly LexA regulon in Pi-starved cells ( Dri & Moreau , 1993 ; controlled by RpoS ( Chang et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>This idea is supported by the nation proteins helps explain ( i ) the requirement in these finding that the level of induction of the LexA regulon in cells of the induction of the LexA regulon ( ZexA ' Pi-starved cells is inversely related to RpoS activity , genotype ) , and ( ii ) the fact that the LexA regulon is which suggests that the higher the RpoS activity , the actually induced in Pi-starved cells ( Dri & Moreau , lower the number of DSB , and thus the lower the level of Downloaded from www.microbiologyresearch.org by IP : 132.248.34.24 On : Fri , 05 Aug 2016 22:55:5 arid argininc observed under PI-starvation conditions may be explained by rhe synthesis of putrescine and spermidine ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Copyright 2001 by the Genetics Society of America Comparative Gene Expression Proﬁles Following UV Exposure in Wild-Type and SOS-Deﬁcient Escherichia coli Justin Courcelle , * ,1 Arkady Khodursky , † ,1 Brian Peter , ‡ Patrick O. Brown † and Philip C. Hanawalt § † Department of Biochemistry , Howard Hughes Medical Institute , Stanford University , Stanford , California 94305 , ‡ Department of MCB , UC-Berkeley , Berkeley , California 94720 , * Department of Biological Science , Mississippi State University , Mississippi State , Mississippi 39762 and § Department of Biological Sciences , Stanford University , Stanford , California 94305 Manuscript received October 4 , 2000 Accepted for publication January 29 , 2001 ABSTRACT The SOS response in UV-irradiated Escherichia coli includes the upregulation of several dozen genes that are negatively regulated by the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>As the cellular concentration of LexA diminishes , Corresponding author : Justin Courcelle , Department of Biological Science , P.O. Box GY , Mississippi State University , Mississippi State , MS 39762 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>As division in this region was not inhibited in lexA3 mutants ( Fig. 7 ) , this inhibition was apparently controlled by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844683" target="_blank">10844683</a>]<br></li>
<li>As expected , UV irradiation causes decreased LexA association and increased RNAP association with the dinJ promoter in a wild-type strain , but not in a lexA1 strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The LexA system uses two reporter genes ( LEU2 and lacZ ) under the control of multiple LexA operators to eliminate false positives ; the assay for LEU2 activity tests for growth of transformed yeast cells in the absence of leucine ( Leu plates ) and the assay for lacZ measures the specific activity of its encoded product , b-galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19386726" target="_blank">19386726</a>]<br></li>
<li>It is likely that a complicated network of interactions between the RecX , DinI , LexA , UmuD , SSB , RecA , and possibly other unidentified proteins , acts to regulate the RecA nucleoprotein filament and RecA function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>Since it has been inferred ( 13 ) and directly shown ( J. Withey and R.F.S. , unpublished work ) that arabinose does not affect dimerization of AraC monomers , the effect of arabinose on the binding of AraCDimCl-LexADNA to the LexA operator most likely is the result of a conformational change in the N-terminal part ofthe protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>To determine , experimentally , whether these sequences bind to LexA protein we conducted gel retardation experiments ( Fried and Crothers , 1981 ; Garner and Revzin , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>Sancar , A. , G.B. Sancar , J.W. Little , D.W. Mount and W.D. Rupp ( 1982 ) LexA protein inhibits transcription of the E. coil UvrA gene in vitro , Nature ( London ) , 298 , 96-98 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>In vitro transcription of the umu operon was specifically inhibited by the LexA protein ( Fig. 5 ) , and the LexA protein bound to a specific region of the operator of the umu operon , as demonstrated by DNase I protection by LexA protein ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>This value is the smallest among those reported to date , and might suggest that the LexA protein binds most tightly to the operator of the umu operon to prevent mutagenesis in noninduced cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>The ability of the AraCDimer-LexADNA chimera to-bind specifically to the LexA operator was assayed in vivo by the ability of the protein to repress P-galactosidase expression from psuI-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>According to this model , LexA would ordinarily repress ftsK expression by binding its cognate operator at dinH , transient displacement of LexA from LexA-1 to LexA-2 , in response to an uncharacterized cellular signal , would free dinH and allow transcription of ftsK in response to the cell 's requirement for FtsK protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>These transformants were picked and transferred to duplicate MacConkey agar plates containing either 1 mM IPTG to induce LexA expression or no IPTG as a repressed control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Galactokinase activity in cells harboring pLKL59 ( wild type ) was 3.5 times higher when LexA expression was repressed by lacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Analysis of recN : : lacZ fusion strains showed that recN is regulated directly by LexA and that its expression can be induced by ultraviolet light , gamma-radiation , and nalidixic acid , in addition to mitomycin C ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>The results summarised in Table 2 show that CAT expression is activated in pSP103 and is regulated by LexA , from which we conclude that the 0.9 kb PstI fragment does contain the recN promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>Changes in the operator conformation , for example DNA supercoilhg , modification of LexA abolishing its ability to-bind the operator or removal of LexA from the operator by competing activator ( s ) produced or activated at the late exponential-growth-phase , could account for the high derepression of sbmC in starved stationary-phase cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>Since LexA is presumed to be functional in RecA - cells , the most simple explanation for this result is that the mutation precludes or reduces the binding of LexA to the sbmC operator , thus allowing constitutive expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>Since sbmC activation occurs with the same intensity in recA56 and in recA ÷ cells ( Fig. 5 ) , a hypothetical RecA-stimulated LexA cleavage occurring at late log phase can not be invoked to explain this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>We used Western blot analysis to monitor degradation of the LexA repressor protein after exposure to UV light under conditions where de novo protein synthesis was inhibited by chloramphenicol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>Notably , the ` helixpositioning ' contacts between Ile31 , Ala32 and Ala43 in LexA are very similar to the equivalent contacts in , for example , lac repressor or catabolic activator protein ( CAP ; see legend to Figure 9 for protein names and references ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Such a strong derepression is typical of SOS operons such as sfiA and umuDC , whose LexA-boxes are efficiently recognized and tightly bound by LexA ( Schnarr et aL , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>Binding of LexA protein to potential LexA binding sites Although the HI value of dinJ and yigN is quite low and is below the general ` cut-off ' of , 10 for most LexA-regulated genes , one explanation for the lack of LexA regulation is that the proposed site simply does not bind LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>For some target genes , FNR is the only regulator , whereas most are regulated coordinately by FNR and the regulators ArcA , NarL , NarP ( nitrate/nitrite regulation ) , Fur ( Fe regulator ) , SoxRS and Q ( regulation by O2̇ ¯ ) , LexA ( SOS regulation ) , or CRP ( glucose regulation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>In summary , we found families involved in the regulation of processes that require unique regulators in bacteria and probably different from those in archaea , such as the global regulator protein , CRP , the positive regulator of the cytochrome c552 , DniR , or the DNA repair system regulated by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>c Fusions to oraA and yigN were LexA dependent and generally DNA damage responsive , while those to ycgH , intG , the lpxA lpxB rnhB dnaE nalidixic acid upregulated and not generally DNA damage inducible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544210" target="_blank">11544210</a>]<br></li>
<li>Certain recA mutants , such as recA1 or recA2OO ( Ts ) at nonpermissive temperature , also block induction of SOS functions because they are not capable of promoting LexA repressor cleavage ; they too are inviable with dam - ( Marinus and Morris 1974 ; Marinus and Morris 1975 ; Roberts et al. 1978 ; McGraw and Marinus 1980 ; Hickson et al. 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3932821" target="_blank">3932821</a>]<br></li>
<li>It was also previously hypothesized as being LexA regulated and was called sosF ( Lewis et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>While the induction of these genes is co-ordinated by LexA repression , the products of individual genes may be needed by the cell in different amounts and may be necessary for different lengths of time after DNA is damaged .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>To do so , we have made oligonucleotide probes to each of the genes identified in Table 2 that have not already been shown to be LexA regulated ( those lacking an asterisk in Table 2 ) and have analysed the pattern of gene expression in three different lexA backgrounds in the absence and presence of exo-genous DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In addition , there are several examples of genes , such as dnaA , dnaN , dnaQ , phr , and recQ ( 28 ) , that appear to be regulated in a recA - and lexA-dependent manner but have not been shown to be directly repressed by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>additional genes ( phr , recQ , and dnaA , dnaN , dnaQ ) exhibit recA - , lexA-dependent DNA induction but have not been shown to be damage directly LexA repressed by ( 13 , 14 , 38 , 39 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Five other genes ( phr , recQ , dnaA , dnaN , and dnaQ ) require recA and lexA for transcriptional induction by DNA damaging agents but may not be directly repressed by LexA ( see text ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>It is required for mediating the autodigestion of LexA repressor , thereby increasing expression of the SOS regulon including the umuDC operon , for mediating the auto-digestion of UmuD to UmuD9 , which is the active form in SOS mutagenesis , and for a third , as yet poorly understood role in the process of translesion synthesis ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>Overexpression of RecX inhibited induction of two din ( damage-inducible ) genes and cleavage of the UmuD and LexA repressor proteins ; however , recX inactivation had no effect on any of these processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>The common denominator of these promoters is that they are all negatively regulated by a repressor protein : lacZ is controlled by LacI that dissociates from the lac operator upon its interaction with lactose ( Reznikoff , 1992 ) ; SOS promoters are controlled by LexA , which is self-cleaved upon interaction with RecA single-strand DNA ﬁlament ( Butala et al. , 2009 ) ; iron acquisition genes are repressed by Fur when it is bound to Fe , and are derepressed þ2 when iron is scarce ( Brickman et al. , 1990 ; Braun et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21396442" target="_blank">21396442</a>]<br></li>
<li>These physiological changes are the consequence of the derepression of at least seventeen unlinked operons which are negatively regulated by LexA repressor ( Brent and Ptashne 1981 ; Little et al. 1981 ) , collectively called the SOS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3932821" target="_blank">3932821</a>]<br></li>
<li>Recent biochemical studies on the RecA protein have shown than it can act as a specific protease , after activation by some signal , such as single-strand D N A fragments generated by D N A damage , cleaving both the repressor that controls phage induction ( Roberts et al. , 1978 ) and 0167-8817/86 / $ 03.50 © 1986 Elsevier Science Publishers B.V. ( Biomedical Division STRAINS OF Escherichia coli K-12 USED IN THIS WORK the LexA protein , which in normal conditions represses the recA gene along with the other genes involved in the SOS response ( Brent and Ptashne , 1981 : Little et al. , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425255" target="_blank">2425255</a>]<br></li>
<li>Under normal growth conditions the LexA protein represses expression of the SOS genes by binding to a specific recognition sequence , at the operator , known as the SOS box ( Little et al. 1981 ; Brent and Ptashne 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>Many of the DNA damage-induced genes are negatively regulated by the LexA repressor protein , which binds to a 20-bp consensus sequence in the operator region of the genes , suppressing their expression ( Brent and Ptashne 1981 ; Little et al. 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Although the SOS response , in its broadest sense , represents all damage-inducible genes , it quickly became synonymous with a dual-component system in which RecA protein is the activator and LexA a negatively acting transcriptional-regulator ( Little et al. , 1980 ; 1981 ; Brent and Ptashne , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>It is known that LexA binds to the sulA promoter tightly as compared to other SOS genes and is induced 100-fold during the SOS response ( Friedberg et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>Similar to tisB and symE , which are both repressed by LexA and are induced by DNA damage ( Vogel et al. , 2004 ; Kawano et al. , 2007 ) , transcription of the ibs and shoB genes may also be induced by speciﬁc growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>This decreases the concentration of the LexA repressor in the cell and increases the level of transcription of LexA-repressed genes ( Little et al. , 1980 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>Strain AB2463 is SOS repressed , and LexA protein is bound to the operators of all the SOS genes , including uvrD , while DM1411 has derepressed SOS genes , since the LexA protein is defective .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038838" target="_blank">3038838</a>]<br></li>
<li>Its expression , like that of other SOS genes , is repressed by the LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>In the uninduced state , LexA protein represses SOS genes by binding to SOS boxes upstream of each gene ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489426" target="_blank">15489426</a>]<br></li>
<li>Continued synthesis of LexA protein restores repression of the SOS genes and thereby returns the cell to the uninduced state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489426" target="_blank">15489426</a>]<br></li>
<li>SOS gene promoters synthesized higher levels of GalK when expression of the LexA repressor was inhibited by LacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The recA protease activity , activated after an SOS inducing treatment , may cleave LexA sitting on dnaG P2 and allow increased expression of dnaG for initiation of DNA replication and rpoD for increased transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>The SOS response requires some level of transcription to activate the 50 plus LexA regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>The expression of the ruvAB operon is normally repressed by LexA unless the SOS response is activated ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142427" target="_blank">12142427</a>]<br></li>
<li>TisB is a small , 29-amino acid hydrophobic peptide that binds to the membrane and leads to a drop in ATP levels Cipro DNA topoisomerase , gyrase DNA damage Canonical SOS response , induction of repair enzymes RecA activation LexA cleavage ( tisA ) tisB lstr-1 Cat + Cat + TisB lstr-1 RNA pmf ATP Dormancy , multidrug tolerant persisters Figure 19.3 A model of TisB-dependent persister formation in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23084949" target="_blank">23084949</a>]<br></li>
<li>These were categorized by responses to mitomycin C and the effect of a mutation that eliminates the ability of the LexA protein to be cleaved in response to DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544210" target="_blank">11544210</a>]<br></li>
<li>of the sulA Gene and Repression by LexA The Escherichia coli sulA gene product is a highly unstable protein , whose synthesis in response to DNA damage is associated with an inhibition of septation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>Upon DNA damage , RecA bound to single-stranded DNA at blocked recombination forks stimulates LexA autoproteolysis , resulting in the derepression of LexA-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>If progression of an active replication fork is blocked by DNA damage or mutation , RecA is activated to promote the cleavage of the LexA repressor , a negative regulator of approximately 20 unlinked genes or operons involved in DNA repair , recombination , and mutagenesis ( 51 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002613" target="_blank">8002613</a>]<br></li>
<li>Coordinated derepression of these genes by the recA and lexA genes results when RecA is activated by DNA damage to promote cleavage of LexA ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Members of the SOS regulon may be differentiated from other genes which exhibit increased transcription in response to DNA damage , e.g. , the heat-shock , adaptive response , oxy , and sox regulons ( 32 , 55 , 56 , 59 ) , through their regulation by the RecA and LexA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Because the damage has been repaired , the level of the activated RecA drops , LexA accumulates , and a decrease in the activation of SOS genes is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>In Escherichia coli , RecA and LexA regulate , at the level of transcription , the cellular reaction to DNA damage called the SOS Response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19555451" target="_blank">19555451</a>]<br></li>
<li>Therefore , when the DNA damage is repaired and RecA is no longer bound to ssDNA , the level of LexA will increase in the cell turning off the SOS response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>Discussion RecA and LexA regulate SOS expression in response to DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19116657" target="_blank">19116657</a>]<br></li>
<li>However , as a result of DNA damage the protease activity of the RecA protein is stimulated and this cleaves the LexA protein ( Horii etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>When the SOS response is induced ( which occurs after damage to DNA or halting of DNA synthesis ) , the RecA protein acquires protease activity , cleaves the LexA repressor and leads to derepression of LexA-repressed genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3937030" target="_blank">3937030</a>]<br></li>
<li>Upon DNA damage , such as that caused by UV light , the LexA repressor undergoes self-cleavage and the expression of SOS genes are thus activated [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21106118" target="_blank">21106118</a>]<br></li>
<li>During this `` SOS response '' the RecA protein is converted into its active form , which stimulates the cleavage of the LexA repressor resulting in derepression of the SOS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The SOS response is initiated when RecA senses damaged DNA and activates cleavage of the global repressor LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20186264" target="_blank">20186264</a>]<br></li>
<li>RecA protein is activated to RecA * by damaged DNA , and RecA * then promotes the cleavage of the X cI repressor and the LexA protein , the transcriptional-repressor of the SOS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3036779" target="_blank">3036779</a>]<br></li>
<li>LexA is responsible for the transcriptional repression of ª40 -- 60 genes in E. coli that are induced in response to DNA damage ( the SOS response ) ( Fernandez De Henestrosa et al. , 2000 ; Wade et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19943910" target="_blank">19943910</a>]<br></li>
<li>Although the SOS-response after nalidixic acid treatment is not induced by chemical damage to DNA but rather by the stalling of the DNA replication fork ( Sugino et al. , 1977 ) , it is nevertheless evident that nalidixic acid treatment of RecA + LexA + bacteria leads to the activation of RecA protease and derepression of LexA regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>]<br></li>
<li>In vivo , we showed that recX is under control of the LexA repressor and is up-regulated in response to DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>ruvA and ruvB form an operon and are regulated by LexA repressor as part of the SOS response to DNA damage ( Shurvinton and Lloyd , 1982 ; Benson et al. , 1988 ; Shinagawa et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8428576" target="_blank">8428576</a>]<br></li>
<li>The gene that encodes the antitoxin , dinJ ( damage inducible gene ) was predicted to be regulated by the LexA repressor ( Lewis et al. , 1994 ) ; LexA acts as a transcriptional-repressor of SOS-regulated genes but is inactivated in response to DNA damage resulting in their induction ( Butala et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>The transcriptional-repressor LexA inhi-bits expression of genes involved in response to DNA damage and DNA replication inhibition in the so-called SOS response [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22168595" target="_blank">22168595</a>]<br></li>
<li>LexA directly regulates ∼ 30 E. coli transcription units involved in the `` SOS '' response whose transcription is induced in response to DNA damage ( Little and Mount 1982 ; Walker 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Overview of the istR/tisAB Locus LexA control and thus induced by DNA damage/SOS conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17499044" target="_blank">17499044</a>]<br></li>
<li>Investigation of 0-galactosidase synthesis in the ruv : : lac fusion strains demonstrated that expression of ruv is induced by damage to DNA and is regulated by LexA as part of the SOS response ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>Less well characterized and underappreciated , however , is the fact that many other proteins are induced by DNA damage that are not directly regulated by RecA or LexA ( see Koch and Woodgate , 1998 and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>When cellular DNA damage is perceived by the binding of RecA to ssDNA , the resulting RecA-ssDNA ﬁlament acts as an allosteric-effector of LexA autocleavage or proteolysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>The resulting nucleoprotein complex will activate RecA and lend it the ability to stimulate the autoproteolytic activity of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16313623" target="_blank">16313623</a>]<br></li>
<li>Similar to what was proposed to occur with the uncleavable LexA repressor protein ( 36 ) , the interaction of RecX and RecA may disrupt the equilibrium of RecA and SSB binding to ssDNA , favoring the binding of SSB to ssDNA and resulting in collapse of the nucleoprotein filament and inhibition of ATPase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>It is not known whether such a cascade occurs in vivo under conditions when RecA protein is in excess , for example , when the LexA regulon , which includes recA , is derepressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8080625" target="_blank">8080625</a>]<br></li>
<li>Similar to the LexA repressor , the CI repressor protein encoded by lambdoid prophages , which maintains a lysogenic state , will cleave itself in the presence of activated RecA , leading to initiation of the lytic cycle and the production and release of phage particles ( Roberts and Devoret , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16313623" target="_blank">16313623</a>]<br></li>
<li>RecA is activated in the presence of ssDNA ( Anderson & Kowalczykowski , 1998 ) and greatly enhances LexA selfcleavage ( Slilaty & Little , 1987 ) to allow high level expression of the SOS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>Since LexA is cleaved and inactivated by an activated form of the RecA protein , it is thought that the elevated levels of the LexA protein result in a rapid restoration of repression as soon as LexA protein production exceeds its cleavage , thereby allowing intact , functional LexA to accumulate ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>Surprisingly , the lexA1 mutant ( which produces a LexA repressor protein resistant to the action of the RecA protease ) also showed a decrease in the ATP level , about 20 rain after the ATP levels increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425255" target="_blank">2425255</a>]<br></li>
<li>RecA promotes proteolytic inactivation of LexA in induced cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>It is shown that the ﬂuorescence in sulAp-gfp cells is regulated by RecA and LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>SOS induction requires activated RecA to function as a coprotease to facilitate LexA self-cleavage , thereby derepressing genes of the SOS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>It is repressed by LexA , and its expression increases following SOS-induced cleavage of LexA by RecA * , the activated form of RecA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>The lexA3 mutation renders LexA resistant to RecA * - induced cleavage , and thus , the SOS regulon is constitutively repressed ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18502852" target="_blank">18502852</a>]<br></li>
<li>This increase subsequently decays after 20 30 rain as a consequence of LexA cleavage by the activated RecA protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425255" target="_blank">2425255</a>]<br></li>
<li>Autocatalytic LexA cleavage in the presence of RecA D112R .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143322" target="_blank">21143322</a>]<br></li>
<li>RecA removes repression by promoting the cleavage of LexA , which can no longer form the dimer needed for repression ( Walker , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>This activated form of RecA facilitates the proteolytic cleavage of LexA repressor , which results in derepression of the regulon ( Little et al. 1980 ; Horii et al. 1981 ; Little 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3932821" target="_blank">3932821</a>]<br></li>
<li>This response is controlled by the regulatory RecA protein , which in activated form promotes the cleavage of LexA repressor , allowing for the increased transcription of about 17 repressed genes ( Little and Mount , 1982 ; Walker , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>This activates the RecA protein which in turns activates the LexA repressor , causing it to cleave .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23084949" target="_blank">23084949</a>]<br></li>
<li>This induction occurs through the activation of the proteolytic activity of the RecA protein , which allows it to cleave some proteins such as the lambda eI repressor and LexA , the repressor of the SOS genes ( Little & Mount , 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>RecA then becomes activated and induces autocleavage of the repressor LexA , leading to an increase in the expression of roughly 40 genes in E. coli ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19648247" target="_blank">19648247</a>]<br></li>
<li>RecA bound to single-strand DNA becomes conformationally active , serving as a coprotease to cleave the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>The active form of RecA facilitates the inactivation of the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>We do not know which of the two forms is a functional form required for mutagenesis , The results shown in Table III and IV are consistent with the idea that the umu operon is under the negative control of the LexA repressor and the DNA-damaging treatments activate the proteolytic activity of the RecA protein to inactivate the repressor by proteolytic cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6311684" target="_blank">6311684</a>]<br></li>
<li>RdgC inhibits RecA protein-promoted DNA strand exchange , ATPase activity , and RecA-dependent LexA cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377615" target="_blank">16377615</a>]<br></li>
<li>If RdgC protein was added first , an RdgC concentration of 1.4 M completely blocked the access of RecA protein to the DNA and thereby inhibited LexA cleavage ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377615" target="_blank">16377615</a>]<br></li>
<li>Similar phenomena were observed in the s - driven promoters of sulA , uvrA , recA , fecI , entC and lacZ , 70 70 the transcription of which is directly controlled by a repressor protein ( LexA , Fur and LacI ) , but not in promoters of the housekeeping genes ftsA and ftsY , or in s - driven treA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21396442" target="_blank">21396442</a>]<br></li>
<li>The resulting protein fusion retained the DNA binding speci ® city of LexA and was inducible by long-chain fatty acids demonstrating the ligand binding function contributed by FadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>HU acts as a regulator by promoting and inhibiting the DNA binding of transcriptional-regulators , such as the cyclic AMP repressor protein , the Lac repressor , the Gal repressor , the Trp repressor , and LexA ( 19 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18519731" target="_blank">18519731</a>]<br></li>
<li>In summary , strains lacking D N A adenine methyl-ase activity require recA + function for LexA repressor inactivation and probably recombination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3932821" target="_blank">3932821</a>]<br></li>
<li>Butala , M. , Žgur-Bertok , D. , and Busby , S.J.W. ( 2009 ) The bacterial LexA transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>The bacterial LexA transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21396442" target="_blank">21396442</a>]<br></li>
<li>In this work , we reconstruct the activity level of the transcriptional-repressor , LexA , from the expression time profiles of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>The synthesis of many of these genes is negatively regulated by the LexA repressor , whose activity is in turn regulated by the product of the recA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>This is in agreement with a previous report showing that despite the presence of a putative LexA binding box , the dinJ/yafQ locus may not be under control of the SOS response [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20186264" target="_blank">20186264</a>]<br></li>
<li>In fact , ( p ) ppGpp up-regulation was seen to increase levels of mRNA transcripts involved in both the stringent and SOS responses such as RelBE , DinJ , RecA , and LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>( A ) In the absence of SOS , IstR-1 is present in high excess over its target , tisAB mRNA , which is expressed at low levels because of incomplete repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15620655" target="_blank">15620655</a>]<br></li>
<li>Lesca , C. , Petit , C. , and Defais , M. ( 1991 ) UV induction of LexA independent proteins which could be involved in SOS repair .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In particular , recAC mutants may resemble those filaments that induce the SOS Response ( after UV treatment ) and this is the reason why when they are loaded onto the DNA , they promote LexA auto-cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19555451" target="_blank">19555451</a>]<br></li>
<li>This reduction is followed by a recovery phase , wherein LexA goes back to its levels under normal conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>Surprisingly , it shows us that , in the absence of LexA regulation , there are no other major responses to UV irradiation at the level of transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Tn the same experiment , rec .4 transcription is repressed only to one-fifth of the level in the absence of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>Western analysis of the gel shifts conﬁrmed the presence of LexA in both the primary and secondary retarded complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The resulting vector , pLKL5 , was then used as a cloning vehicle for the specific isolation of E. coli promoters which are transcriptionally repressed in the presence of high levels of LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The presence of LexA within the primary retardation complex was conﬁrmed by carrying out the binding reactions in the presence of anti-LexA antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The presence or absence of protein extract and anti-LexA antibodies is indicated by plus and/or minus symbols under each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The SymE-SPA levels were signiﬁcantly elevated in the double mutant and were induced even further upon deactivation of the LexA repressor with mitomycin C treatment indicating that the contributions of LexA , SymR and Lon to SymE repression are additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17462020" target="_blank">17462020</a>]<br></li>
<li>The activity of LexA , here referred to as transcription factor activity , is expressed in arbitrary units and can be interpreted as relative `` levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>The degree of repression suggests that there is very little transcription of the sulA operon in the presence of repressing levels of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>CulD , N-terminal-fused with the LexA activation domain , was expressed under control of the constitutive ADH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408908" target="_blank">23408908</a>]<br></li>
<li>DinQ is under LexA control and antisense regulation by a novel small RNA , agrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>These results demonstrate that the LexA-2 operator-like sequence can be recognized and bound by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>In each case , LexA only binds in the presence of the ectopic DNA site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>LexA is in red , while the other proteins are in yellow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>LexA is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>Thus , for conventional LexA sites , the level of association in vivo is strongly related to the inherent affinity of LexA for these DNA sites in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Evidence is provided in the present work that this site is bound by LexA in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>LexA binds in vitro to canonical , but not noncanonical , targets To determine which in vivo targets are bound directly by LexA in vitro , we incubated a GST-LexA fusion protein or GST alone with sheared genomic DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The fold repression ( indicated by the bars ) , as a measure for dimerization , was calculated as the ratio of the - galactosidase activities measured without and with induction of the LexA fusion proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>LexA binds directly to the second operator of the dinJ promoter region in vitro ( 31 ) ; however , it is unclear whether LexA-mediated repression occurs in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>While it is impossible from these experiments to determine if the repression is directly due to a LexA regulation , we did observe cases where functional LexA boxes appeared to result in inhibition of gene expression following UV irradiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>This regulation also includes repression by LexA binding to three identified SOS boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22168595" target="_blank">22168595</a>]<br></li>
<li>Demonstration of either LexA or the DinJ -- YafQ binding to the LexA box upstream of the dinJ -- yafQ module also suggests that competition occurs and both contribute to its repression and autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>Transcriptional repression and direct binding of DinJ and LexA at the dinJ promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>Strain DM1180 served as a positive control , as light emission should be negatively proportional to the LexA repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Scheme of transcriptional repression by LexA -- LexA408 heterodimer formed through interaction of CysB and a-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>In all cases , LexA binds these targets in vitro and is required for repression of the target gene in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Mizusawa , S. , Court , D. and Gottesman , S. ( 1983 ) Transcription of the sulA gene and repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Transcription of the sulA gene and repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Transcription of the sit/A gene and repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666395" target="_blank">2666395</a>]<br></li>
<li>C , - galactosidase assays testing repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>Transcriptional repression by LexA heterodimer formation of GcvA and GcvR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>In Bacteria , an abrupt slope in the NWD curve reveals the presence of a handful of high-scoring binding sites for highly speciﬁc TFs ( e.g. TrpR , LexA ) , whereas a progressive increase of the NWD slope is indicative of global factors , such as CRP , FNR ( Figure 3 ) , Fur , FruR , IHF and FIS ( Supplementary Data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>We repeated the analysis , using equally stringent search criteria , for other global DNA-binding proteins ( LexA , Fur , FruR , FNR , CRP , NarL and Fis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>Conversely , these results conﬂict with another recent report in which a LexA binding site upstream of yigN was identiﬁed but no increase in mRNA formation upon mitomycin C treatment or regulation by LexA was observed ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544210" target="_blank">11544210</a>]<br></li>
<li>In a few cases , adjacent LexA sites are bound in a cooperative manner , further enhancing the sensitivity to changes in signal ( 36 ) , as hypothesized for the ArcA binding sites located upstream of acs and astC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">LexA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Since sbmC activation occurs with the same intensity in recA56 and in recA ÷ cells ( Fig. 5 ) , a hypothetical RecA-stimulated LexA cleavage occurring at late log phase can not be invoked to explain this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>Only three LexA motifs with ScanACE score > 16.5 occur in coding regions , suggesting that LexA motifs in functionally inappropriate locations may be negatively selected during evolution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>GH5 and HNF-3 have a three-stranded antiparallel , B-sheet with the third strand formed by the loop between helices I and II , whereas in LexA and BirA this loop runs approximately parallel to the n-sheet strands without forming part of the sheet .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>As is the case for LexA , the loop is flexible in GH5 , where it is found in two different conformations in the two molecules in the unit cell , and in HSF , where the central part of the loop shows no discernible electron density .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The EMBO Joumal vol .13 no. 17 pp.3936-3944 , 1994 Solution structure of the LexA repressor D binding domain determined by 1H NMR spectroscopy NA R.H.Fogh , G.Ottleben1 , H.Ruterjansl , M.Schnarr2 , R.Boelens and R.Kaptein3 Bijvoet Center for Biomolecular Research , Utrecht University , Padualaan 8 , 3584-CH Utrecht , The Netherlands , 1Institute of Biophysical Chemistry , Johann Wolfgang Goethe-Universitat , TheodorStem Kai 7 , Haus 75A , D-6000 Frankfurt 70 , Germany and 2Institut de Biologie Moleculaire et Cellulaire du CNRS , 15 Rue Rene Descartes , 67084 Strasbourg c6dex , France 3Corresponding author Communicated by R.Kaptein The structure of the 84 residue DNA-binding domain of the Escherichia coli LexA repressor has been determined from NMR data using distance geometry and restrained molecular dynamics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The amino-terminal domain of LexA repressor is a-helical but differs from canonical helix-turn-helix proteins : A two-dimensional 1H NMR study R. M. J. N. LAMERICHS * , A. PADILLA * , R. BOELENS * , R. KAPTEIN * , G. OTTLEBENt , H. RUTERJANSt , M. GRANGER-SCHNARRt , P. OERTELO , AND M. SCHNARR * * Department of Chemistry , University of Utrecht , Padualaan 8 , 3584 CH Utrecht , The Netherlands ; tInstitut fur Biophysikalische Chemie , Johann Wolfgang Goethe-Universitit , Theodor-Stern Kai 7 , Haus 75A , D-6000 Frankfurt 70 , Federal Republic of Germany ; and tInstitut de Biologie Moldculaire et Cellulaire du Centre National de la Recherche Scientifique , Laboratoire de Biochimie , 15 rue Rend Descartes , 67084 Strasbourg C6dex , France Communicated by Peter H. von Hippel , June 5 , 1989 ABSTRACT The amino-terminal DNA bin g domain of LexA repressor consisting of 84 amino acid residues has been studied by two-dimensional ' H NMR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>In summary , strains lacking D N A adenine methyl-ase activity require recA + function for LexA repressor inactivation and probably recombination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3932821" target="_blank">3932821</a>]<br></li>
<li>These data suggest that the decrease in the ATP level observed in the wild-type strain must be attributed to two factors : LexA repressor cleavage and an important recombinant activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425255" target="_blank">2425255</a>]<br></li>
<li>Note the insert in LexA at residue 38/39 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The observed structure falls completely within the parameters of a classic HTH motif ( with the important difference that the LexA HTH motif contains a one residue insert in the turn between the helices , at residue 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The data show that LexA contains at least three helices , one with a kink in the middle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The LexA DBD contains three a-helices extending over residues 6-21 ( helix I ) , 28-35 ( helix II ) and 40-53 ( helix III ) , according to the Kabsch-Sander secondary structure definition rules , as incorporated in the program PROCHECK ( Morris et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>LexA DBD contains three tight turns according to the definition of Wilmot and Thornton ( 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Sequential NOEs used to assign LexA are shown as a hatched box and medium-range NOEs as a thick bar connecting the relevant residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>First , potential LexA boxes should have less than five G or C residues in the general ( N ) 10 region ( as all of the previously characterized LexA boxes only have three or less guanine or cytosine residues within this region ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The main reason for this difference is that in LexA the loop region is two residues longer and as a result of this the end of helix 2 and the amino-terminal start of helix 3 are further apart with Phe-37 located between the two helices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>Information on this orientation in LexA can be derived from NOEs between residues in the two helices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>For LexA most NOEs in this region are between residues in the turn and residues in both helices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>Accordingly , structures of the LexA DBD were generated only for residues 1-72 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Ramachandran plot for residues 4-69 from the final set of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The final set of structures used to represent the conformation of the LexA DBD were now selected , superimposed over the C , N and Ca atoms of residues 4-69 and averaged to give the average structure , AV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>This prediction is in better agreement with the NMR data since the second helix in LexA indeed begins at Arg-28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>Of the six proteins , CAP and BirA contain a classic HTH motif , while GH5 , HNF-3 and HSF , like LexA , have an insert in the turn between the helices .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>GH5 , HNF-3 and HSF also have differences in the position and orientation of the first helix in the HTH motif relative to CAP , BirA and LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>All of these mutants contain amino acid substitutions in a region of LexA ( residues 28 to 46 ) where recent nuclear magnetic resonance studies have identified a modified helix-turn-helix motif which is quite different from the classical structures seen in the X , 434 , and Trp-repressor-proteins ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>However , other studies ( 23 ) predict that LexA has only a low probability to have a helix-turn-helix domain and that , if it is present at all , it would start at Thr-5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>For LexA it has been suggested that a helix-turn-helix motif would start at Arg-28 ( 22 ) , based on weak sequence homology in the first helix of the motif ( i.e. , Ala-32 ) and the highly conserved Gly-36 in the turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>Thus , at the start of this work it was an open question whether or not LexA recognizes its various operators through a helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>As mentioned in the Introduction it was not clear from the sequence homology whether the structure of LexA contains a helix-turn-helix motif or not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The conserved van der Waals contact between the two helices of the motif-i.e. , between Ala-10 and Val-20 in lac repressor ( 18 ) - is replaced in the case of LexA by a network of hydrophobic contacts involving Phe-37 and alanines 32 , 42 , and 43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>With the complete and detailed LexA structure presented for the first time in this work , it is clear that LexA contains a HTH motif at residues 28-48 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Notably , Pabo and Sauer ( 1984 ) predict that LexA should contain a HTH motif at residues 28-47 , while Dodd and Egan ( 1987 , 1990 ) find a low probability for an HTH motif starting at residue 5 , but predict that the motif is most probably absent in LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The r.m.s. deviation for C , N and Ca of LexA residues 28-37 and 40-48 [ residues 1-10 and 12-20 of the motif in the nomenclature of Pabo and Sauer ( 1984 ) ] is 0.96 ± 0.16 A comparing LexA with the proteins depicted in Figure 9 , as compared with 0.87 ± 0.16 A between the proteins themselves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>HTH motif of LexA and selected other proteins superimposed on C , N and Ca of LexA residues 28-37 and 40-48 , corresponding to residues 1-10 and 12-20 of the HTH motif in the nomenclature of Pabo and Sauer ( 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>We have defined two open reading frames 5 ' to uvrC , one encoding a 23 kDa protein and one encoding Department of Cell Biology , Baylor College of Medicine , One Baylor Plaza , Houston , TX 77030 , USA Received 11September 1989 We have studied the in vitro interaction of the LexA repressor protein and the uvrC regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>According to the studies realized on AraC ( 72 , 74 , 75 ) ( XylS/AraC HTH family ) and LexA ( 73 ) , both from E. coli , such a restricted mobility is thought to be due to interactions between the D-b and E-b/O domains and/or to interactions of part of the linker region with one of the two structural domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>LexA has two domains connected by a hinge region containing the Ala-84-Gly-85 cleavage site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>In vivo studies of the promoter region of colicins E1 ( Salles & Weinstock , 1989 ) and E7 ( Lu & Chak , 1996 ) show that mutations in either LexA binding motif result in an increased level of colicin production , suggesting that the colicin operon contains two distinct SOS boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>All previously characterized chromosomal operators for LexA have been found to contain the symmetrical sequence CTG ( N1O ) CAG , with the central region typically very AT rich .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The sequence deduced for the promoter region of dinG contains a potential 20-bp operator segment which does not retain the symmetrical CTG ( N1O ) CAG motif found in all other chromosomal LexA binding sites but instead contains the asymmetric sequence TTG ( N1O ) CAG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>This analysis suggests that , except for the disruption in the highly conserved CTG ... CAG motif ( which diminishes LexA binding at two other sites ) , the dinG sequence is otherwise very similar to known SOS operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Furthermore , we show that LexA binds comparably to an ectopic canonical LexA motif introduced at various positions in the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Conversely , analysis of canonical sites was performed with primers flanking the LexA motif , therefore maximizing the observed fold-enrichment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Unconventional in vivo targets of LexA that lack a canonical LexA motif As mentioned above , 19 of the novel target regions did not contain a significant match to the canonical LexA motif as defined by the PWM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>This region does not contain the best match to the consensus LexA motif within 1000 bp of the predicted site location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>It does , however , contain a sequence that is a good match to the consensus LexA motif except that the near-invariant CTG motifs in each half-site are replaced by ATG ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>St r u cture -- function relationship of CysB transcription factor 799 interaction between protein domains fused to these LexA DBDs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>This result strongly suggests that interaction between CysBwt and a-CTDwt domains fused to LexA DBDs allows effective recognition of the hybrid LexA operator through the LexAwt -- LexA408 hybrid DBD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>Notably , the ` helixpositioning ' contacts between Ile31 , Ala32 and Ala43 in LexA are very similar to the equivalent contacts in , for example , lac repressor or catabolic activator protein ( CAP ; see legend to Figure 9 for protein names and references ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>These two mutations modify the structural properties of RecA protein in a similar way rendering it proteolytically active on the LexA repressor ( at 42 ° C in the presence of adenine for the recA441 mutation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2972909" target="_blank">2972909</a>]<br></li>
<li>A domain search with b1145 protein shows that it belongs to the LexA group of SOS-response transcriptional repressors similar to the lambda repressor ( Interpro id : IPR006198 , PFAM id : PF00717 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14733619" target="_blank">14733619</a>]<br></li>
<li>As a consequence , we have not excluded potential LexA binding sites from our study if they are located in a hypothetical ORF with no known function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Thirdly , the potential LexA box should not be located inside a gene with a known function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The N-terminal domain of the molecule , LexA 1-84 , binds strongly to at least one of the SOS operators ( Bertrand-Burggraf et al. , 1987 ) , and forms the same contacts with DNA as the complete protein according to methylation protection and hydroxyl radical footprinting studies ( Hurstel et al. , 1986 , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Methylation protection and hydroxyl radical footprinting experiments ( 5 , 6 ) have shown that intact LexA and the isolated amino-terminal domain form the same specific complex with recA operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>This causes a frameshift in LexA at amino acid 160 and completely changes the amino acid sequence of the C-terminal domain , which is required for dimerization of LexA on its operator sequences ( Kim and Little , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>CulD , N-terminal-fused with the LexA activation domain , was expressed under control of the constitutive ADH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408908" target="_blank">23408908</a>]<br></li>
<li>The amino-terminal domain of LexA repressor i ot-helical but differs from canonical helix-turn-helix proteins : a two-dimensional 1H NMR study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>In this paper we report on the secondary structure of the amino-terminal domain of the LexA repressor derived from two-dimensional ( 2D ) NMR measurements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>Although the amino-terminal domain of LexA is of comparable size , 84 amino acids and a molecular mass of 9.1 kDa , the analysis of the 2D NMR spectra of LexA was not trivial .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The sequential and medium-range NOEs that could be identified in the 2D NOE spectra of the amino-terminal domain of LexA are shown in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>It should be noted that such an interaction would place residues 59-63 in close proximity to the dyad axis , so that some of the activity-modifying mutations found here ( Figure 2 ) could also be explained as the result of interactions with the same region of the other LexA monomer rather than protein-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>There are 12 families of enzyme domains , of which at least three are certainly catalytically active : the signal peptidase domain in LexA , the methyl-DNA protein methyltransferase as in Ada and the P-loop nucleotide triphosphate hydrolase in DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>We demonstrate a near-universal relationship between the presence of a LexA sequence motif , LexA binding in vitro , and LexA binding in vivo , suggesting that a suitable recognition site for LexA is sufficient for binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Analysis of one such noncanonical target in the ptrA promoter region reveals an aberrant , but specific , version of a LexA motif , and it suggests that another factor is important for LexA binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Thus , LexA association at the ptrA promoter is mediated by an aberrant , but specific , LexA motif-like sequence in which key residues directly contacted by LexA are altered so as to preclude DNA-binding in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The result of this search was that the two-helical structure of LexA is not more closely related to the canonical helix-turn-helix motif than it is to similar substructures found in other classes of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>Since the helix-turn-helix motifs found in various DNA-binding proteins are very similar ( 29 ) , within 0.8 A , the result of the search suggests that the structure of the segment 28-47 of LexA is not more closely related to the helix-turn-helix motif than it is to similar but different structures found in proteins of other classes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The side-chain packing around the HTH motif in LexA corresponds to what is found in other HTH proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Relationship between LexA binding in vivo and similarity to the LexA motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The remarkable similarity between the HTH motif of LexA and some acknowledged HTH DNA-binding proteins can be seen in Figure 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The possible presence of the helix-turn-helix ( HTH ) DNA-binding motif in LexA has been the subject of discussion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>A comparison of the primary of structure LexA with that of other DNA-binding proteins gives a possible explanation why this structure in LexA differs from the helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The HTH motif The structure of the LexA DBD has been the focus of some interest given the different opinions about the nature and location of the DNA-binding motif ( Pabo and Sauer , 1984 ; Dodd and Egan , 1987 , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>The similarities between LexA DBD and other HTH DNA-binding proteins , as well as the impact of the LexA structure on the sequence homology recognition pattern for HTH proteins , will be discussed in more detail elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>In A cI this motif is formed by helices 2 and 3 , in lac repressor by helices I and II , and in trp by helices D and E. Further , in Fig. 3 some of the highly conserved residues are underlined and the mutants that affect the DNA-binding of LexA are indicated by an asterisk .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>In striking contrast , GST-LexA binding to two unconventional targets ( i.e. , those lacking the LexA motif ) is indistinguishable from that of the GST control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The 23 kDa protein overlies two promoters ( P3 and P4 ) proximal to the uvrC structural gene and also overlies two potential LexA binding sites , neither of which is canonical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>It is also possible that the binding requires the interaction of two sites because neither fexA2 nor IexA3 fits the canonical LexA binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>A contains no LexA protein and B-F contain 2gM LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>The carboxy terminus of the more distal 28 kDa protein lies close to an additional LexA binding site which is canonical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>Analysis of LexA protein-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>DISCUSSION The results presented support the conclusion that LexA protein interacts with potential binding sites in the uvrCregulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>Considering that HNF-3 does not bind as a dimer and that the monomers of CAP are oriented opposite to those of LexA , the LexA-DNA complex may well turn out to be significantly different from DNA complexes of HNF-3 and CAP , in spite of the close similarities in topology and structure between the three DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>Structural classification of proteins [ http://scop.mrc-lmb.cam.ac.uk/scop/data/scop.1.001.004.004.html] Fogh RH , Ottleben G , Ruterjans H , Schnarr M , Boelens R , Kaptein R : Solution structure of the LexA repressor DNA-binding domain determined by 1H NMR spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11305941" target="_blank">11305941</a>]<br></li>
<li>Since mutations in residues 40-45 can alter the binding specificity of LexA ( Thliveris and Mount , 1992 ) , these residues must be involved in base-specific contacts with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>These symmetric mutations might be important for binding another protein that cooperates with LexA and/or alters the precise nature of the LexA : DNA contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Other groups have also applied similar computational analyses to look for the recognition sites of DNA-binding proteins , including LexA ( Robison et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>A similar trans-domain ligand-induced response has been observed in fusion proteins carrying the LexA DNA-binding domain and the hormone-binding site of the glucocorticoid receptor ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>( b ) Alignment of AraC motifs similar to LexA and Myb DNA-binding motifs ( Suzuki & Brenner , 1995 ) 669 Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8980677" target="_blank">8980677</a>]<br></li>
<li>Some regulators , including TrpR repressor , Met repressor , and Cro repressor , are compact and globular ( 3-5 ) , but others , such as the LexA , phage A , and phage 434 repressors , consist of a DNA-binding domain connected to a dimerization domain ( 6-12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>Table 1 shows that the LexA DNA-binding domain itself repressed only by a factor of 2 but that fusing this to the C/EBP leucine-zipper dimerization domain or the N-terminal half of AraC led to repression by a factor of 14 or 12 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>We therefore constructed two types of chimeras utilizing parts of AraC : the AraC N-terminal half fused to the LexA DNA-binding domain and the AraC C-terminal half fused to the leucine-zipper from the eukaryotic transcriptional activator C/EBP ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>DNA was constructed encoding the N-termi-nal half of AraC fused to the LexA DNA-binding domain ( AraCDimer-LexADNA ) , the C-terminal half of AraC fused to the leucine-zipper region of C/EBP ( ZipDimer-AraCDNA ) , and the C/EBP leucine-zipper fused to the LexA DNA-binding domain ( ZiPDimCr-LexADNA ) ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>In vivo and in vitro experiments showed that a chimeric protein consisting of the N-terminal half of the AraC protein and the DNA-binding domain of the LexA repressor dimerizes , binds to a LexA operator , and represses the expression of a LexA operator in an arabinose-responsive manner ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Notwithstanding the results in this paper showing that the N-terminal half of AraC can participate in specific dimer formation , results do not eliminate the possibility that in our vivo the DNA-binding domains themselves of both AraC and LexA are unstable and are stabilized , but not dimerized , by additional protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>The finding that the fusion protein efficiently repressed the AraCDimer-LexADNA expression of PSUIA-lacZ , whereas the LexA DNA-binding domain alone did not , suggested that the addition of the AraC dimerization region did indeed dimerize the chimeric protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>As shown in Fig. 4 , when fused to the LexA DNA-binding domain , both full-length MtfA and MtfA 1-83 , which is missing the amino-terminal part of the protein , are able to interact with Mlc fused to LexA * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>]<br></li>
<li>LexA repressor and iron uptake regulator from Escherichia coli : new members of the CAP-like DNA-binding domain superfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931550" target="_blank">11931550</a>]<br></li>
<li>One-half of this artiﬁcial operator is recognized by the helix-turn-helix DNA-binding domain of the LexA wild-type repressor , the other half by a LexA * mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>]<br></li>
<li>Works on LexA show that the DNA-binding ability of a specific domain can be enhanced or diminished by fusing the D-b domain with some alternative dimerization domains ( 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11756427" target="_blank">11756427</a>]<br></li>
<li>Golemis , E.A. , and Brent , R. ( 1992 ) Fused protein domains inhibit DNA-binding by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>Thus , the ability of AraCDjIIXC-LexADNA , but not the LexA DNA-binding domain , to repress PsuU-lacZ was more likely the result of the protein 's dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>Genetic analysis of the DNA-binding domain of E. coli LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>In this paper the 3-D structure of the DNA-binding domain of LexA will be presented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>This was determined to be the case by phenotypic analyses of protein fusions between the DNA-binding domain of LexA ( amino acids 1 ± 87 ) and amino acids 102 ± 239 of FadR [ 118 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>For heterodimerization analysis , fusions of protein X to the wild-type LexA DNA-binding domain ( lexA1-87 / X ) and of protein Y to the LexA408 mutant DNA-binding domain ( lexA4081-87 / Y ) were coexpressed from compatible plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20952573" target="_blank">20952573</a>]<br></li>
<li>¶ The ` winged helix ' superfamily is defined in the SCOP database [ 40 ] ; LexA [ 41 ] and ArgR [ 42-44 ] DNA-binding domains have been resolved by X-ray crystallography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11305941" target="_blank">11305941</a>]<br></li>
<li>Using a LexA-based two-hybrid system designed by Dmitrova et al. ( 1998 ) , we were able to show that CysBwt and a-CTDwt are very likely to interact in vivo , as they restore LexA repressor activity when fused to LexA DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>This possibility appears unlikely because other studies utilizing the DNA-binding domains of A repressor or LexA found that dimerization domains from by GNC4 , Fos , and Jun proteins was required for binding to their respective operators ( 10 , 14 , 15 ) , the same result on which we base our conclusion of dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>n Also , the DNA-binding domains of the dimeric A phage repressor and LexA repressor have been used as DNA-binding modules ( 6 , 10 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8516313" target="_blank">8516313</a>]<br></li>
<li>Furthermore , as an internal control , we fused Mlc to both different DNA-binding domains of the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>]<br></li>
<li>Only in the case of cross talk between the two hybrid proteins did the two different DNA-binding domains of LexA and LexA * come together to repress lacZ expression , as was seen in the two control plasmids pMS604 ( encoding a LexA-Fos-hybrid protein ) and pDL804 ( encoding a LexA * - Jun-hybrid protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>]<br></li>
<li>Cells with only one of the two LexA-Mlc hybrid proteins always showed a very high - galactosidase activity , indicating a high speciﬁcity of the different LexA DNA-binding domains for their cognate operator sites ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">LexA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>We did not observe LexA ( His ) 6 binding to the relBE palindrome ( data not shown ) which suggested that ( i ) the binding of LexA to the dinJ -- yafQ palindrome we observed was speciﬁc to the dinJ -- yafQ module and ( ii ) the in vivo regulation of dinJ -- yafQ is likely distinct from its family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>Closer examination of the AccI -RCB- BglII region reveals a 19 bp motif , 5 `` - CTG AAcAgTcATgTTT CAG-3 `` ( uppercase letters represent bases which show identity with the LexA operator consensus ; bold letters represent the invariable palindrome found within the LexA operator sequence ) , from ® 671 to ® 689 bp that shows similarity to the consensus sequence of a LexA operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>Lastly , it may be noteworthy that the ORFfs protein of R67 f shows a similarity to the known N-terminal amino acid sequence of the phage 186 Tum protein ( Fig. 7 ) , which is also under control of LexA and functions as an antirepressor upon UV induction ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682790" target="_blank">8682790</a>]<br></li>
<li>Consequently , the LexA protein has a large set of correctly predicted targets and exhibits a highly signiﬁcant motif almost identical to the known canonical LexA motif ( Figure S3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></li>
<li>Note that for both methods , most of the genes here show much smaller scores than genes belonging to the LexA regulon ( see Table 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11897025" target="_blank">11897025</a>]<br></li>
<li>A domain search with b1145 protein shows that it belongs to the LexA group of SOS-response transcriptional repressors similar to the lambda repressor ( Interpro id : IPR006198 , PFAM id : PF00717 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14733619" target="_blank">14733619</a>]<br></li>
<li>DinQ belongs to the LexA regulon in E. coli [ 1 ] which regulates the SOS response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>There are 12 families of enzyme domains , of which at least three are certainly catalytically active : the signal peptidase domain in LexA , the methyl-DNA protein methyltransferase as in Ada and the P-loop nucleotide triphosphate hydrolase in DnaA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>( b ) Alignment of AraC motifs similar to LexA and Myb DNA-binding motifs ( Suzuki & Brenner , 1995 ) 669 Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8980677" target="_blank">8980677</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by LexA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Description b0532 4 Putative outer membrane protein , export function ; sequence similarity suggests that it is a member of the ﬁmbrial usher protein family Hydrogenase 4 membrane subunit Curli production assembly/transport component , second curli operon 867 sfmD b2483 4 b1039 322 129 hyfC csgE 4 RNA related ybcJ ORF , hypothetical protein ; putative RNA-binding protein Putative transcriptional regulator Transcriptional repression of hyc and hyp operons ; involved in the formate hydrogenylase system Fimbrial Z protein ; probable signal transducer ; positive DNA-binding transcriptional regulator b0528 b3264 b2725 17.1 4.9 4 77 envR hycA 220 153 b0535 ﬁmZ 4 210 Phage related alpA b2624 b1140 b1557 b1157 Prophage CP4-57 regulatory protein AlpA Prophage e14 intergrase 6.5 4.6 4.6 4 70 375 71 179 intE cspB stfE Cold shock protein ; may affect transcription ; transcriptional regulator Putative tail ﬁber protein Cell processes ampC pphB ibpB b4150 b2734 b3686 - Lactamase ; penicillin resistance Protein phosphatase 2 Heat shock protein 4.3 4 4 ORFs with unknown function yjiP yghG yfjR 19.7 18.4 16 ORF , hypothetical protein ORF , hypothetical protein b4338 b2971 b2634 ORF , hypothetical protein ; putative transcriptional repressor , probably related to phage ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ; cold-induced member of CspA family Putative pectinase Putative oxidoreductase ORF , hypothetical protein ; LexA regulated ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ; conserved protein with phophatase-like domain ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ; gene is in an operon associated with acid resistance ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein ; putative membrane protein possibly involved in transport ORF , hypothetical protein ORF , hypothetical protein Putative oxidoreductase Putative enzyme ORF , hypothetical protein ; conserved hypothetical protein , related to 2Fe-2S ferredoxin ORF , hypothetical protein ORF , hypothetical protein ORF , hypothetical protein 12.1 11.3 11.3 8.6 7.5 7.5 7 6.1 6.1 5.3 4.9 4.9 4.9 4.6 4.6 4.6 4.6 4.3 120 70 65 b0690 b1058 b0248 b0519 b4026 b1742 b0689 b4249 b0685 b3948 b4047 b1470 b0844 b4185 b1877 b0787 b0309 b1500 ybfG yceO yafX ylbE yjbE ves ybfP yjgI ybfE yijI yjbL yddJ ybjI yjfM yecT ybhM b0309 b1500 91 4.3 4 4 4 b1228 b4205 b1796 b4257 b1228 ytfA yoaG yjgN 108 60 398 4 4 4 4 4 84 b4066 b3121 b2902 b2373 b2236 yjcF yhaC ygfF yfdU yfaE 4 yfaA b2230 yadS b0157 b0370 b0370 578 207 89 4 4 a CoA , coenzyme A. b FNT , formate and nitrate-transporter family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16385049" target="_blank">16385049</a>]<br></li>
<li>Catabolite response Deoxirribose assimilation D 87 135 Crp w 27 Pérez - Rueda and Collado-Vides ( 2000 ) 220 -- R 235 10 DeoR DicA 35 w Pérez - Rueda and Collado-Vides ( 2000 ) -- n/d 0 -- -- n/d HLH R 4 DicC DnaA 0 Transcription and replication regulation Positive regulator of cytochrome c552 77 0 -- Pérez - Rueda and Collado-Vides ( 2000 ) 41 -- n/d DniR 0 Kajie et al. ( 1991 ) 209 8 215 -- A Ebp FhlC 77 m Pérez - Rueda and Collado-Vides ( 2000 ) Pérez - Rueda and Collado-Vides ( 2000 ) n/d 0 -- 7 -- Pérez - Rueda and Collado-Vides ( 2000 ) Schnetz ( 2002 ) n/d R FhlD FrvR 0 -- Phosphoenolpyruvate sugar phosphotransferase system operon regulation Iron assimilation 8 0 -- -- 128a Fur R w Achenbach and Yang ( 1997 ) 107 460 35 Carbon sources assimilation Nguyen and Saier ( 1995 ) ; Weickert and Adhya ( 1992 ) 388 GalR/LacI 55 R c -- 13 GcvR GntR Rigali et al. ( 2002 ) Glycine cleavage operon repression 0 R 735 775 Ghrist and Stauffer ( 1998 ) 96 R w Yamada and Saier ( 1988 ) Activator of the glucitol operon in E. coli High-frequency persistence of the lethal effects of inhibition of either DNA or peptidoglycan-synthesis regulation ( HipAB ) -- 8 GutM 0 A -- Black et al. ( 1991 ) -- 1 HipA 1 0 n/d -- n/d HipB HtgA Black et al. ( 1991 ) Missiakas et al. ( 1993 ) 87 0 -- -- -- n/d 7 Heat shock regulation 0 -- HycA HypF 6 -- n/d ZnF A 0 -- 0 Bouhouche et al. ( 2000 ) Gui et al. ( 1996 ) 27 -- Glyoxylate bypass operon Integration host factor 31 106 161 w R -- IclR Ihf -- 15 8 Pérez - Rueda and Collado-Vides ( 2000 ) Campoy et al. ( 2002 ) -- D 43 -- DNA-repair 54 LexA R w 102 279 Miller and Bassler ( 2001 ) Schell ( 1993 541 LuxR/UhpA m A Quorum sensing 1548 LysR 1271 Amino acid biosynthesis regulation D w 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>I Keywords : H-NS , LexA , phosphate starvation , reactive-oxygen-species , RpoS INTRODUCTION respiration , or fermentation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Tolerating some redundancy RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_5">
J Bacteriol 188 : Gerard F , Dri AM & Moreau PL ( 1999 ) Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></label>
<input id="rp_similarity_5", type="checkbox">
<div class="RP_similar">
<li>GeÂrard , F. , Dri , A.M. and Moreau , P.L. ( 1999 ) Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li>
<li>Role of Escherichia coli RpoS , LexA and H-NS global regulators in metabolism and survival under aerobic , phosphate-starvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li></div>
<li>Mutations that prevent induction of the LexA regulon ( lexA-Ind - ; Fig. 6a ) or that inhibit To assess the possible role of RpoS during Pi starvation , recombinational-repair , i.e. recA ( Fig. 6b ) and ruuA ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Second , pyruvate H-NS helps maintain a rapid degradation of arginine oxidase ( PoxB ) , which may be preferentially used to and glucose produce acetate when bacteria approach the stationary phase , is unlikely to play a key role under Pi-starvation Because H-NS is required for a full induction of the conditions because the expression of poxB is strictly LexA regulon in Pi-starved cells ( Dri & Moreau , 1993 ; controlled by RpoS ( Chang et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Besides RpoS and LexA , show that H-NS also plays we a key role in Pi-starved cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Repair proteinsa E ( g ) Group Gene E. coli V. cholerae H. inﬂuenzae B. subtilis Direct repair 0.40 0.51 -- 0.53 0.38 -- 0.62 -- -- -- -- Deoxyribodipyrimidine photolyase O6 -- Methylguanine DNA alkyltransferase 0.79 0.59 0.76 0.68 -- Methylated DNA-protein-cysteine methyltransferase Regulatory protein , methyltransferase , alkyltransferase Ada Base excision repair 0.47 0.53 0.57 0.53 0.61 0.56 0.64 0.53 0.46 0.40 0.44 0.43 0.46 0.85 0.66 0.57 0.40 0.68 0.38 0.47 0.46 0.57 0.51 0.54 -- 0.53 0.47 0.35 0.37 0.53 0.44 0.48 0.78 0.35 -- b 0.38 0.50 0.41 0.69 0.55 0.62 0.58 0.68 -- 0.70 -- 0.48 0.47 0.62 0.63 0.56 0.88 0.44 0.56 0.53 0.51 0.45 0.53 0.49 0.50 0.54 -- 0.69 -- 0.57 0.35 0.42 -- -- 0.39 1.09 0.40 0.50 0.61 0.51 0.37 Uracil DNA glycosylase A/G-speci ﬁc adenine glycosylase Endonuclease III Formamidopyrimidine DNA glycosylase 3-Methyladenine DNA glycosylase I DNA 3-methyladenine glycosidase II Exonuclease III , apurinic-apyrimidinic endonuclease Endonuclease IV DNA mismatch-repair protein MutS DNA mismatch-repair protein MutL GATC endonuclease GATC methylase Exodeoxyribonuclease large subunit Exodeoxyribonuclease small subunit Excision nuclease UvrA Excision nuclease UvrB Excision nuclease UvrC DNA helicase II Transcription repair coupling factor AP endonucleases Mismatch excision repair Nucleotide excision repair Recombinational repair 1.48 0.28 0.32 0.39 0.51 0.39 0.59 0.71 0.56 0.45 0.40 0.35 0.45 0.50 0.80 -- -- -- 0.43 0.37 -- 0.72 0.39 0.48 -- 0.39 0.52 0.51 0.69 0.39 0.37 0.42 0.64 0.48 0.63 0.75 0.53 0.53 0.53 -- -- 0.63 0.88 0.41 0.46 0.41 0.47 0.44 0.43 0.55 0.49 0.53 0.43 0.37 0.47 0.61 RecA protein ExoV helicase RecB ExoV nuclease RecC ExoV helicase RecD Single-stranded DNA binding protein RecF 59 3 39 single-stranded DNA exonuclease RecJ RecO protein RecR protein ATP binding protein RecN DNA helicase RecQ Exodeoxyribonuclease I Double-stranded DNA exonuclease SbcC Double-stranded DNA exonuclease SbcD DNA-repair protein RadC RuvA protein RuvB protein 0.55 0.64 -- -- 0.65 0.65 0.52 0.58 0.55 0.47 0.40 0.50 0.59 0.56 0.48 0.61 Branch migration resolution RecG protein Holliday junction endonuclease RuvC NAD-dependent DNA ligase GTPase triphosphatase MutT 0.51 0.54 0.89 0.44 0.74 0.71 0.39 0.63 -- 0.56 0.72 0.97 DNA ligases Nucleotide pools Deoxyuridine 59-triphosphate nucleotide hydrolase 0.48 0.32 0.42 0.40 DNA polymerase I 0.43 0.36 -- -- DNA polymerase II 0.64 0.34 0.41 0.30 DNA polymerase III alpha chain Replication 0.78 0.56 0.78 0.53 SOS regulon transcription repressor LexA 0.50 0.53 0.56 0.56 ATP-dependent protease Sms/RadA Other repair proteins 0.40 0.37 0.48 0.36 Primosomal protein replication factor PriA 1.48 0.66 0.94 0.80 Single-stranded DNA binding protein Ssb 0.99 0.52 0.49 -- ATP-dependent helicase HepA a The classiﬁcation of repair proteins was adopted in part from data in reference 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489855" target="_blank">11489855</a>]<br></li>
<li>of genes involved in the response to DNA damage ( SOS-response ) , including RecA and LexA ; in the presence of single-stranded DNA , RecA interacts with LexA , causing an autocatalytic cleavage which disrupts the DNA-binding part of LexA , leading to derepression of the SOS regulon and eventually DNA-repair Catalyzes the formation of phosphodiester linkages between 5 - phosphoryl and 3 - hydroxyl groups in double-stranded DNA using NAD as a coenzyme and as the energy source for the reaction ; essential for DNA-replication and repair of damaged DNA Component of the leucine , isoleucine , valine ( threonine ) transport system , which is one of the two periplasmic binding-protein-dependent transport systems of the high-afﬁnity transport of the branched-chain amino acids ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and in the physiological short-term adaptation to glucose limitation ; decreases after benzoic acid treatment and phosphate limitation late and in the presence of leucine Component of the leucine-speciﬁc transport system , which is one of the two periplasmic binding-protein-dependent transport systems of the high-afﬁnity transport of the branched-chain amino acids ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol ; decreases after benzoic acid treatment 5.42 / 44,439 5.93 / 50,340 ( DIGE 4.5 -- 6.5 ) LeuD 5.16 / 22,356.29 P30126 3-Isopropylmalate dehydratase small subunit Probable HTH-type transcriptional regulator Leucyl-tRNA synthetase 5.20 / 23,000 ( DIGE 4.5 -- 6.5 ) 179 , 321 5.80 / 35,694.73 P10151 LeuO 5.77 / 34,515 293 5.11 / 98,831 5.16 / 97,233.76 P07813 LeuS LexA 6.23 / 22,357.72 P0A7C2 LexA repressor 179 5.39 / 73,606.07 DNA ligase P15042 LigA 79 1 LivJ 5.28 / 41,960 5.28 / 36,772.53 P0AD96 Leu/Ile/Val-binding protein 29 , 61 , 71 , 179 , 228 , 258 , 293 , 294 , 311 , 321 5.18 / 39,439 ( DIGE 4.5 -- 6.5 ) 5.30 / 38,773 ( DIGE 4.5 -- 6.5 ) 5.31 / 40,279 ( 4.5 -- 5.5 ) 5.21 / 16,964 ( 4.5 -- 5.5 ) 5.09 / 50,544 ( 5 -- 6 ) 5.23 / 50,424 ( 5 -- 6 ) 5.67 / 30,753 ( 5 -- 6 ) 5.33 / 23,392 ( 5 -- 6 ) 5.73 / 30,106 ( 5.5 -- 6.7 ) LivK 5.00 / 41,464 5.07 / 36,982.71 Leucine-speciﬁc binding protein P04816 71 , 179 , 212 , 228 , 258 , 287 , 294 , 321 5.09 / 39,439 ( DIGE 4.5 -- 6.5 ) 5.08 / 39,792 ( 4.5 -- 5.5 ) 5.56 / 28,995 ( 4.5 -- 5.5 ) 5.25 / 59,333 ( 5 -- 6 ) Continued on facing pag Induced by L-lactate aerobically Participates in the translocation of lipoproteins from the inner membrane to the outer membrane Plays a critical role in the incorporation of lipoproteins in the outer membrane after they are released by the LolA protein ; essential for E. coli viability Degrades short-lived regulatory and abnormal proteins in the presence of ATP ; induced by stress conditions such as high temperatures , nalidixic acid , and accumulation of abnormal proteins ; positively regulated by htpR Component of the glycine-cleavage-system as well as of the alpha-ketoacid dehydrogenase complexes ; increases during aerobic growth and the low temp at 10 ° C ; induced by low pH during anaerobic growth Interacts with the peptidoglycan both covalently and noncovalently ; contributes to the maintenance of the structural and functional integrity of the cell envelope Catalyzes the transformation of S-ribosylhomocysteine to homocysteine and 4,5-dihydroxy-2 ,3 - pentadione ; involved in the synthesis of autoinducer 2 , which is secreted by bacteria and is used to communicate both the cell density and the metabolic potential of the environment ; changes in cell density in a process called quorum-sensing ; induced by low pH Involved in lysine-biosynthesis Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Synthesizes a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The phosphoryl group from PEP is transferred to the phosphoryl carrier protein HPr by enzyme I ( PtsI ) ; phosphoHPr then transfers it to the permease ( enzymes II/III ) ; HPr is common to all PTS ; regulated by the cAMP-CRP complex and also by growth on glucose ; induced by low pH and phosphonate growth See PtsH ; decreases after benzoic acid treatment but increases at phosphonate growth Plays an important role in de novo pathway of purine-nucleotide-biosynthesis : AMP-biosynthesis ; increases after benzoic acid treatment Involved in IMP-biosynthesis : 5-amino-1 - ( 5-phospho-D-ribosyl ) imidazole-4 - carboxamide from N ( 2 ) - formyl-N ( 1 ) - ( 5-phospho - dribosyl ) glycinamide ( step 5 ) See PurB : step 4 ; increases after benzoic acid treatment See PurB : step 2 ; increases after benzoic acid treatment See PurB : step 3 PstS P0AG82 Phosphate-binding periplasmic 6.92 / 34,421.76 6.85 / 45,258 ( 6 -- 11 ) protein Pta P0A9M8 Phosphate acetyltransferase 5.28 / 77,040.90 5.29 / 85,417 179 , 228 , 274 , 293 , 294 PtrA 179 Protease III P05458 5.66 / 105,113.29 5.65 / 9,119.37 5.65 / 10,366 P0AA04 Phosphocarrier protein HPr PtsH 27 , 228 , 293 , 294 PtsI 4.78 / 59,810 Phosphoenolpyruvate-protein phosphotransferase P08839 56 , 179 , 228 , 293 , 294 , 321 4.78 / 63,561.84 4.91 / 63,671 ( DIGE 4.5 -- 6.5 ) PurA P0A7D4 Adenylosuccinate synthetase 179 , 321 5.30 / 44,524 ( DIGE 4.5 -- 6.5 ) 5.30 / 43,494 ( DIGE 4.5 -- 6.5 ) 5.32 / 47,213.76 PurB 179 5.68 / 51,542.81 P0AB89 Adenylosuccinate lyase PurC 5.09 / 26,148 ( DIGE 4.5 -- 6.5 ) 5.07 / 26,995.00 P0A7D7 Phosphoribosylaminoimidazolesuccinocarboxamide synthase Phosphoribosylamine-glycine ligase Phosphoribosylaminoimidazole carboxylase catalytic subunit Bifunctional purine biosynthesis protein Phosphoribosylaminoimidazole carboxylase ATPase subunit 179 , 321 PurD 179 , 321 5.06 / 47,667 ( DIGE 4.5 -- 6.5 ) 4.96 / 45,940.36 P15640 PurE 179 , 321 6.05 / 18,899 ( DIGE 4.5 -- 6.5 ) 6.03 / 17,649.13 P0AG18 See PurB : ﬁnal step ; increases after benzoic acid treatment See PurB : step 3 ; possesses an ATPase activity that is dependent on the presence of AIR ( aminoimidazole ribonucleotide ) ; the association of PurK and PurE produces an enzyme complex capable of converting AIR to CAIR efﬁciently under physiological condition See PurB : step 2 PurH 179 , 321 5.50 / 57,000 ( DIGE 4.5 -- 6.5 ) 5.53 / 57,329.21 P15639 5.75 / 28,100 ( 5 -- 6 ) 5.60 / 39,461.12 P09029 PurK 287 179 P08178 Phosphoribosylformylgly-cinamidine cyclo-ligase HTH-type transcriptional repressor 4.83 / 36,722.85 PurM Repressor that binds to the purF operator and coregulates other genes for de novo purine-nucleotide-synthesis ; involved in regulation of purB , purC , purEK , purHD , purL , purMN , and guaBA expression ; binds hypoxanthine and guanine as inducers 4.83 / 36,722.85 PurM P0ACP7 6.3 / 38,043.62 PurR 179 Oxidizes proline to glutamate for use as a carbon and nitrogen source and also functions as a transcriptional repressor of the put operon ; induced by proline Involved in ﬁnal step of glycolysis Involved in UMP-biosynthesis : UMP from HCO3 ( step 2 ) See PyrB : step 3 See PyrB : step 4 See PyrB : step 6 ( ﬁnal ) Bifunctional protein 5.69 / 143,815.16 PykF Pyruvate kinase I P0AD61 5.77 / 50,729.42 5.65 / 56,143 ( DIGE 4.5 -- 6.5 ) 179 , 321 PyrB Aspartate carbamoyltransferase catalytic chain Dihydroorotase Dihydroorotate dehydrogenase Orotidine 5 -- phosphate decarboxylase CTP synthase P0A786 6.13 / 34,296.17 5.70 / 93,664 6.13 / 35,321 179 , 212 , 228 , 294 P05020 P0A7E1 P08244 PyrC 5.77 / 38,696.19 7.66 / 36,774.52 5.81 / 26,350.24 179 179 , 287 PyrD PyrF 7.28 / 47,786 ( 6 -- 11 ) 5.76 / 56,695 179 , 228 , 294 , 321 5.83 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.61 / 58,475 ( DIGE 4.5 -- 6.5 ) Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen ; allosterically activated by GTP when glutamine is the substrate and also activated by magnesium but inhibited by CTP and by divalent metal ions such as copper and zinc Catalyzes the phosphorylation of UMP to UDP , with ATP as the preferred donor Involved in allosteric regulation of aspartate carbamoyltransferase Essential for efﬁcient processing of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; cold shock protein essential for E. coli cells to adapt to low temp PyrG P0A7E5 5.63 / 60,242.96 179 , 321 7.07 / 25,839.07 P0A7E9 Uridylate kinase PyrH PyrI P0A7F3 Aspartate carbamoyltransferase 6.84 / 16,989.44 6.85 / 17,023 regulatory chain 7.01 / 17,642 ( 6 -- 11 ) 179 , 212 , 228 , 287 , 294 RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 Involved in the high-afﬁnity D-ribose membrane transport system and also serves as the primary chemoreceptor for chemotaxis ; repressed in the presence of glucose but induced in the physiological short-term adaptation to glucose limitation Involved in ribose metabolism Catalyzes the hydrolysis of ATP in the presence of single-stranded DNA , the ATP-dependent uptake of single-stranded DNA by duplex DNA , and the ATP-dependent hybridization of homologous single-stranded DNAs ; interacts with LexA , causing its activation and leading to its autocatalytic cleavage ; induced by low temp at 10 ° C as well as by cadmium chloride , hydrogen peroxide , and ACDQ Counteracts the effect of relE by means of direct proteinprotein interaction , enabling the reversion of translation inhibition ; acts as an autorepressor of relBE transcription ; increased transcription rate of relBE and activation of relE is consistent with a lower level of relB in starved cells due to degradation of relB by protease lon ; induced by amino acid starvation RbfA P0A7G2 Ribosome-binding factor A 5.96 / 15,023.29 5.79 / 17,433 6.00 / 15,635 286 , 319 RbsB P02925 6.85 / 30,900 5.99 / 28,474.47 D-Ribose-binding periplasmic protein 99 , 179 , 311 , 321 E 4.5 -- 6.5 ) IG 5.92 / 29,066 ( D 4.99 / 32,290.52 P0A9J6 Ribokinase P0A7G6 Protein RecA RbsK RecA 179 5.09 / 37,842.18 5.08 / 41,137 133 , 179 , 228 , 294 , 297 4.81 / 9,071.48 4.81 / 9,100 Antitoxin P0C079 RelB 99 Continued on facing pag Catalyzes the interconversion between ADP-D-glycero-beta-D-manno-heptose and ADP-L-glycero-beta-D-manno-heptose via an epimerization at carbon 6 of the heptose ; completely inhibited by ADP and ADP-glucose and partially inhibited by ATP and NADH ; induced by heat shock Involved in carbohydrate-biosynthesis and dTDP-L-rhamnose-biosynthesis See RfbB RfaD P67910 ADP-L-glycero-D-manno-heptose-6-epimerase 4.80 / 34,893.17 4.85 / 36,820 4.94 / 36,106 ( DIGE 4.5 -- 6.5 ) 4.98 / 35,245 ( 4 -- 5 ) P37759 RfbB 179 dTDP-glucose 4,6-dehydratase 5.47 / 40,558.33 RfbC P37745 dTDP-4-dehydrorhamnose 3,5 - 5.48 / 21,270.11 epimerase P0AG30 Transcription termination factor 6.75 / 47,004.21 179 Facilitates transcription Rho termination by a mechanism that involves rho binding to the nascent RNA , activation of rho 's RNA-dependent ATPase activity , and release of the mRNA from the DNA template Involved in riboﬂavin biosynthesis ; repressed by heat shock but induced by low pH Involved in ﬁnal steps of riboﬂavin synthesis ; riboﬂavin synthase is a bifunctional enzyme complex catalyzing the formation of riboﬂavin from 5-amino-6 - ( 1 - D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione and L-3 ,4 - dihydrohy-2-butanone-4 - phosphate via 6,7-dimethyl-8-lumazine ; the alpha subunit catalyzes the dismutation of 6,7-dimethyl-8-lumazine to riboﬂavin and 5-amino-6 - ( 1-D ) - ribityl-amino-2 ,4 ( 1H ,3 H ) - pyrimidinedione See RibE ; decreases after benzoic acid treatment Acetylates the N-terminal serine of ribosomal protein L7/L12 ; increases during phosphate limitation and phosphonate growth Essential for efﬁcient processing of 16S rRNA ; probably part of the 30S subunit prior to or during the ﬁnal step in the processing of 16S free 30S ribosomal subunits ; could be some accessory protein needed for efﬁcient assembly of the 30S subunit ; needed in a step prior to RbfA during the maturation of 16S rRNA ; has afﬁnity for free ribosomal 30S subunits but not for 70S ribosomes ; decreases after benzoic acid treatment P0A7J0 3,4-Dihydroxy-2-butanone 4-phosphate synthase 4.90 / 23,353.47 RibB 274 RibE P0AFU8 Riboﬂavin synthase alpha chain 5.64 / 23,444.90 5.67 / 26,821 ( 5 -- 6 ) 5.73 / 25,731 ( 5.5 -- 6.7 ) 179 , 287 RibH 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.19 / 12,875 ( 4.5 -- 5.5 ) 5.42 / 40,651 5.15 / 16,156.51 P61714 6,7-Dimethyl-8-ribityllumazine synthase Ribosomal protein-serine acetyltransferase 179 , 287 , 321 5.86 / 20,680.55 P13857 RimL 228 , 293 , 294 4.68 / 24,436 ( DIGE 4.5 -- 6.5 ) 4.61 / 20,605.43 P0A7X6 16S rRNA-processing protein RimM 321 179 321 5.09 / 35,712.71 5.39 / 32,693.56 Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA Catalyzes the formation of dTDP-glucose from dTTP and glucose-1-phosphate as well as its pyrophosphorolysis ; decreases after benzoic acid treatment Digests double-stranded RNA ; involved in the processing of rRNA precursors and of some mRNAs Rare lipoprotein A Glucose-1-phosphate thymidylyltransferase 1 P10100 P37744 RlpA 5.41 / 32,544 ( DIGE 4.5 -- 6.5 ) RmlA1 6.4 / 25,550.04 P0A7Y0 RNase III Rnc 179 Matures 5S rRNA from its precursors from all the rRNA genes ; cleaves RNA I , a molecule that controls the replication of colE1 plasmid DNA ; the major endoribonuclease participating in mRNA turnover in E. coli May be part of a membrane complex involved in electron transport Responsible for the end turnover of tRNA : speciﬁcally removes the terminal AMP residue from uncharged tRNA ( tRNA-C-C-A ) ; involved in tRNA-biosynthesis , especially in strains lacking other exoribonucleases P21513 Rne RNase E 5.48 / 118,196.73 Electron transport complex protein P77285 RnfG 179 6.59 / 21,911.91 RNase T P30014 Rnt 179 5.19 / 23,522.7 5.13 / 24,554.25 Ribulose-phosphate 3-epimerase Ribose-5-phosphate isomerase A P0AG07 Rpe 179 Involved in nonoxidative branch of the pentose-phosphate-pathway ; base-induced protein One of the primary rRNA-binding proteins ; binds very close to the 3 end of the 23S rRNA and forms part of the L1 stalk ; translational repressor protein ; controls the translation of the L11 operon by binding to its mRNA See RplA ; increases at phosphonate growth RpiA 5.20 / 22,860.40 5.06 / 25,943 P0A7Z0 179 , 274 , 286 RplA 50S ribosomal protein 9.64 / 24,598.44 8.24 / 25,577 ( 6 -- 11 ) L1 P0A7L0 179 , 287 RplI 6.20 / 19,831 6.17 / 15,769 6.17 / 15,769.06 50S ribosomal protein L9 P0A7R1 29 , 179 , 228 , 286 , 287 , 293 , 294 6.71 / 10,366 5.10 / 8,830 ( 4.5 -- 5.5 ) 6.36 / 15,124 ( 6 -- 11 ) 9.04 / 17,580.4 Translational repressor protein ; controls the translation of the rplJL-rpoBC operon by binding to its mRNA Seems to be the binding site for several of the factors involved in protein-synthesis and appears to be essential for accurate translation ; decreases during phosphate limitation Binds the 5S rRNA ; required for the late stages of subunit assembly Binds to 23S rRNA in the presence of protein L20 One of the proteins that binds to the 5S RNA in the ribosome , where it forms part of the central protuberance ; binds to the 5S rRNA independently of L5 and L18 ; not required for binding of the 5S rRNA/L5/L18 subcomplex to 23S rRNA May be induced under zinc-limiting conditions ; may be repressed in that case by the zinc uptake regulation protein Zur DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>We also discus evidence which indicates that the function of IHF in gene-expression in E. coti may be fundamentally different from that of classical global transcription regulators such as LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>We also observed a clear decrease in reporter expression ( down to 16 % ) in the presence of a pair of plasmids encoding LexA DBD -- a-CTDwt and LexA408 DBD -- CysBwt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>The LexA system uses two reporter genes ( LEU2 and lacZ ) under the control of multiple LexA operators to eliminate false positives ; the assay for LEU2 activity tests for growth of transformed yeast cells in the absence of leucine ( Leu plates ) and the assay for lacZ measures the specific activity of its encoded product , b-galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19386726" target="_blank">19386726</a>]<br></li>
<li>Thus , although LexA serves as a negative regulator for polB , it essentially behaves as a positive regulator of araDAB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Control samples [ SU202 , test strain without any plasmid as a negative control ; pMS ( LexA-Fos-hybrid protein ) / pDL ( LexA - Jun-hybrid protein ) as a positive control ] are shown in gray .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>]<br></li>
<li>[ 25 ] A. Fernandez de Henestrosa , T. Ogi , S. Aoyagi , D. Chaﬁn , J.J. Hayes , H. Ohmori , R. Woodgate , Identiﬁcation of additional genes belonging [ 26 ] J. Courcelle , A. Khodursky , B. Peter , P.O. Brown , C. Hanawalt , [ 27 ] C. Janion , Some aspects of SOS-response system -- a critical survey , [ 28 ] R.P. Fuchs , S. Fujii , J. Wagner , Properties and functions of Escherichia [ 29 ] R. Costa de Oliveira , J. Laval , S. Boiteux , Induction of SOS and adaptive [ 30 ] E. Śledziewska - Gójska , C. Janion , Alternative pathways of methyl [ 31 ] E. Grzesiuk , C. Janion , The frequency of MMS induced , umuDC - [ 32 ] B.J. Bachmann , Derivation and genotype of some mutant derivatives of [ 33 ] B. Sedgwick , Oxidation of methylhydrazines to mutagenic methy - [ 34 ] M.R. Volkert , D.C. Nguyen , C. Beard , Escherichia coli gene induction [ 35 ] R. Woodgate , Construction of a umuDC operon substitution mutation in [ 36 ] C. Ho , O.I. Kulaeva , A.S. Levine , R. Woodgate , A rapid method for to the LexA regulon in Escherichia coli , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16226494" target="_blank">16226494</a>]<br></li>
<li>Using DNA microarrays containing ampliﬁed DNA fragments from 95.5 % of all open reading frames identiﬁed on the E. coli chromosome , we have examined the changes in gene-expression following UV exposure in both wild-type cells and lexA1 mutants , which are unable to induce genes under LexA control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>These authors examined the changes in gene-expression after UV exposure ( 40 J/m2 ) in both wild-type cells and lexA1 mutants , which are unable to induce genes under LexA control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>Most of the differentially expressed genes ( e.g. , dinF , dinI , lexA , ruvA , ruvB , yebG ) in the LexA regulatory module show significant up-regulation after UV radiation and subsequent decline in the gene-expression levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>Future experiments will characterize the molecular basis of the regulation and examine the signiﬁcance of LexA control with respect to cell-division gene-expression in normal growth conditions , and on SOS induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>It provides some generalization with respect to the role of LexA in the regulation ( up and possibly down ) of genes in response to one type of environmental stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>This report demonstrates that LexA has an important and unexpected general role in cellular regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>In addition to LexA regulation of transcription , posttranscriptional-controls may play an important role in modulating gene activity in the SOS system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>In addition to RecA and LexA , SSB is also believed to play a key role in this cellular response ( 8,9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2991853" target="_blank">2991853</a>]<br></li>
<li>Copyright 2001 by the Genetics Society of America Comparative Gene Expression Proﬁles Following UV Exposure in Wild-Type and SOS-Deﬁcient Escherichia coli Justin Courcelle , * ,1 Arkady Khodursky , † ,1 Brian Peter , ‡ Patrick O. Brown † and Philip C. Hanawalt § † Department of Biochemistry , Howard Hughes Medical Institute , Stanford University , Stanford , California 94305 , ‡ Department of MCB , UC-Berkeley , Berkeley , California 94720 , * Department of Biological Science , Mississippi State University , Mississippi State , Mississippi 39762 and § Department of Biological Sciences , Stanford University , Stanford , California 94305 Manuscript received October 4 , 2000 Accepted for publication January 29 , 2001 ABSTRACT The SOS-response in UV-irradiated Escherichia coli includes the upregulation of several dozen genes that are negatively regulated by the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>The control for these reactions was the polB ( dinA ) LexA binding site , which has an HI value of 12.09 and is therefore in the uppermost range of LexA-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The AccI/BglII region is essential for ftsK transcription and binds LexA Expression of genes controlled by LexA has a genetic dependence on recA function , yet our observations show transcription of ftsK from dinH in a recA minus background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>From these additional properties , it was suggested that ruv , along with other LexA regulated recombination genes such as recA , recN , and recQ ( 6,7,8 ) , is involved in a recombinational process for repair of damaged DNA ( 4,5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>In summary , we found families involved in the regulation of processes that require unique regulators in bacteria and probably different from those in archaea , such as the global regulator protein , CRP , the positive regulator of the cytochrome c552 , DniR , or the DNA-repair system regulated by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>LexA , a major regulator of DNA-repair , is known to have a single well-conserved DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></li>
<li>In fact , ( p ) ppGpp up-regulation was seen to increase levels of mRNA transcripts involved in both the stringent and SOS-responses such as RelBE , DinJ , RecA , and LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>Lesca , C. , Petit , C. , and Defais , M. ( 1991 ) UV induction of LexA independent proteins which could be involved in SOS repair .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>E. coli strain phase ; and the LexA regulon , including recA and ruuAB , EN2361 is a derivative of strain AB1157 ( Bachmann , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>Henestrosa ARFD , Ogi T , Aoyagi S , Chafin D , Hayes JJ , Ohmori H , Woodgate R : Identification of additional genes belonging to the LexA regulon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>bringing the present number of genes in the E. coli LexA regulon to 31 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The LexA protein represses transcription of over 30 SOS genes in the E. coli SOS regulon ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>DinQ belongs to the LexA regulon in E. coli [ 1 ] which regulates the SOS-response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>Conjugational hyperrecombination achieved by derepressing the LexA regulon , altering the properties of RecA protein and inactivating mismatch-repair in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143322" target="_blank">21143322</a>]<br></li>
<li>Control of the LexA regulon by pH ; evidence for a reversible inactivation of the LexA repressor during the growth cycle of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>Control of the LexA regulon by recombination of bacterial plasmid DNA : effect of RecF pathpH : evidence for a reversible inactivation of the LexA repressor way mutations on plasmid-recombination in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>It is required for mediating the autodigestion of LexA repressor , thereby increasing expression of the SOS regulon including the umuDC operon , for mediating the auto-digestion of UmuD to UmuD9 , which is the active form in SOS mutagenesis , and for a third , as yet poorly understood role in the process of translesion-synthesis ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>EMBO J 3:707 -712 Calsou P , Villaverde A , Defais M ( 1987 ) Activated RecA protein may induce the expression of a gene that is not controlled by LexA repressor and whose function is required for mutagenesis and repair of UV-irradiated lambda phage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2972909" target="_blank">2972909</a>]<br></li>
<li>A property these damage-induc-ible genes have in common is a unique operator sequence to which the LexA protein binds , thereby repressing expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>The second mutant host , strain DM1187 ( Kim & Little , 1992 ) , served as a negative control , as it encodes a LexA protein unable to bind to the operator , resulting in unregulated expression of the SOS-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>If the latter is the case , these genes must be under the negative control of a repressor protein other than LexA , since the strains used in these experiments are lexA ( Def ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>The recA protease activity , activated after an SOS inducing treatment , may cleave LexA sitting on dnaG P2 and allow increased expression of dnaG for initiation of DNA-replication and rpoD for increased transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>The pleiotro-pic SOS reaction to DNA damage entails the derepression of a set of functions , including enhanced synthesis of RecA , LexA , Uvr and other Din proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2991853" target="_blank">2991853</a>]<br></li>
<li>* E-mail : magnar.bjoras@rr-research.no ( MB ) ; knut.ivan.kristiansen@rr-research.no ( KIK ) Introduction Exposure of E. coli to DNA damaging agents induces the SOS-response , which is under control of the RecA and LexA regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>The lexA3 mutation leads to the synthesis of a LexA protein that is resistant to proteolytic cleavage and thus prevents the induction of DNA-repair genes in the cell ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2435706" target="_blank">2435706</a>]<br></li>
<li>First , by cata-lyzing cleavage of LexA protein , the spiral filament derepressesthe LexA regulon , which consists largely of genes involved in repair of replicative bypass of DNA damage .55 Second , by catalyzing cleavage of the UmuD which is encoded by the LexA regulon gene umuD , the spiral filament activates a mutagenic replicative damage bypass process , Third , by catalyzing cleavage of certain phage repressor proteins , the spiral filament derepresses the phage regulons and initiates phage multiplicationand cell-death .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8080625" target="_blank">8080625</a>]<br></li>
<li>Recent biochemical studies on the RecA protein have shown than it can act as a specific protease , after activation by some signal , such as single-strand D N A fragments generated by D N A damage , cleaving both the repressor that controls phage induction ( Roberts et al. , 1978 ) and 0167-8817/86 / $ 03.50 © 1986 Elsevier Science Publishers B.V. ( Biomedical Division STRAINS OF Escherichia coli K-12 USED IN THIS WORK the LexA protein , which in normal conditions represses the recA gene along with the other genes involved in the SOS-response ( Brent and Ptashne , 1981 : Little et al. , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2425255" target="_blank">2425255</a>]<br></li>
<li>At the same time , the LexA repressor that regulates expression of all SOS genes also controls transcription of the TisAB toxin/antitoxin module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23090599" target="_blank">23090599</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23090599" target="_blank">23090599</a>]<br></li>
<li>Decreasing the concentration of LexA , the repressor of the SOS Response , up-regulates a large set of genes ( 50 or more ) that have both known functions ( i.e. , DNA-repair , mutagenesis and delay of cell-division ) and yet unknown functions [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19116657" target="_blank">19116657</a>]<br></li>
<li>We consider an example of an SOS repair system , and computationally infer the regulator activity of its master repressor , LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>In the E. coli SOS repair system , the master repressor LexA regulates a SIM , which consists of several targets shown in Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>RecA then becomes activated and induces autocleavage of the repressor LexA , leading to an increase in the expression of roughly 40 genes in E. coli ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19648247" target="_blank">19648247</a>]<br></li>
<li>Activated RecA in turn triggers autocleavage of LexA repressor and thus expression of repair functions , including RecA itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928948" target="_blank">7928948</a>]<br></li>
<li>If progression of an active replication fork is blocked by DNA damage or mutation , RecA is activated to promote the cleavage of the LexA repressor , a negative regulator of approximately 20 unlinked genes or operons involved in DNA-repair , recombination , and mutagenesis ( 51 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002613" target="_blank">8002613</a>]<br></li>
<li>This activates the RecA protein which in turns activates the LexA repressor , causing it to cleave .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23084949" target="_blank">23084949</a>]<br></li>
<li>This response is controlled by the regulatory RecA protein , which in activated form promotes the cleavage of LexA repressor , allowing for the increased transcription of about 17 repressed genes ( Little and Mount , 1982 ; Walker , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>Additional activity of activated RecA other than cleavage of LexA repressor has been suggested to be involved in SOS mutagenesis ( 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>DNA damage which blocks the replication fork , induces a modification of the RecA protein into an activated form ( RecA * ) that promotes the cleavage of the LexA repressor and thereby allows the expression of SOS functions ( for review see Walker 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2972909" target="_blank">2972909</a>]<br></li>
<li>When the SOS-response is induced ( which occurs after damage to DNA or halting of DNA-synthesis ) , the RecA protein acquires protease activity , cleaves the LexA repressor and leads to derepression of LexA-repressed genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3937030" target="_blank">3937030</a>]<br></li>
<li>RecA protein acts as a positive regulator of the SOS system : in response to blockage of the replication fork , it cleaves the LexA protein , a transcriptional repressor of the genes of the regulon ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3298208" target="_blank">3298208</a>]<br></li>
<li>RecA coprotease activity facilitates self-cleavage of the LexA repressor protein , enhancing expression of SOS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>Activated RecA protein functions as a coprotease , mediating cleavage of LexA repressor and thus activating transcription of SOS regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489426" target="_blank">15489426</a>]<br></li>
<li>As a control , we used antibodies against LexA , another stress response protein that is not directly linked to oxidative stress but involved in the SOS DNA damage control pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>LexA is responsible for the transcriptional repression of ª40 -- 60 genes in E. coli that are induced in response to DNA damage ( the SOS-response ) ( Fernandez De Henestrosa et al. , 2000 ; Wade et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19943910" target="_blank">19943910</a>]<br></li>
<li>Key words : distance geometry/molecular dynamics/NMR spectrum assignment/protein structure Introduction The LexA repressor from Escherichia coli is a 202 residue protein that regulates transcription of -20 genes known SOS as genes , which are involved in DNA-repair and replication , mutagenesis and cell-division [ for reviews see Little and Mount ( 1982 ) , Walker ( 1984 ) , Little ( 1991 ) , Schnarr et al. ( 1991 ) and Schnarr and Granger-Schnarr ( 1993 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>LexA directly regulates ∼ 30 E. coli transcription units involved in the `` SOS '' response whose transcription is induced in response to DNA damage ( Little and Mount 1982 ; Walker 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The LexA repressor plays a central role in initiating the SOS-response upon DNA damage ( Shinagawa , 1996 ) , suggesting that regulation of dinJ -- yafQ transcription is also linked to DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>LexA repressor plays an important role in the regulation of about 20 different `` SOS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>The SOS-response requires some level of transcription to activate the 50 plus LexA regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>Induction of the SOS-response results in induction of lexA , which encodes the LexA repressor of SOS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>]<br></li>
<li>The lexA3 ( Ind ) allele encodes an uncleavable LexA repressor protein that prevents derepression of LexA-controlled SOS genes during an SOS-response ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19648247" target="_blank">19648247</a>]<br></li>
<li>The gene that encodes the antitoxin , dinJ ( damage inducible gene ) was predicted to be regulated by the LexA repressor ( Lewis et al. , 1994 ) ; LexA acts as a transcriptional repressor of SOS-regulated genes but is inactivated in response to DNA damage resulting in their induction ( Butala et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>LexA Represses Transcription at the Second Operator of the DinJ Promoter -- The LexA repressor regulates the transcription of genes involved in the SOS-response mediated by DNA damage ( 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>The transcriptional repressor LexA inhi-bits expression of genes involved in response to DNA damage and DNA-replication inhibition in the so-called SOS-response [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22168595" target="_blank">22168595</a>]<br></li>
<li>LexA protein is a repressor of its own gene , recA and a number of other , commonly called din ( damage inducible ) genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3937030" target="_blank">3937030</a>]<br></li>
<li>Brieﬂy , LexA is a transcriptional repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>Green arrowheads , promoters under LexA control ; black arrowhead , constitutive promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17499044" target="_blank">17499044</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , May 1992 , p. / 92/103377 -09 $ 02.00 / 0 Copyright C 1992 , American Society for Microbiology Isolation of DNA Damage-Inducible Promoters in Escherichia coli : Regulation of polB ( dinA ) , dinG , and dinH by LexA Repressor L. KEVIN LEWIS , t MICHAEL E. JENKINS , AND DAVID W. MOUNT * Molecular and Cellular Biology Department , University ofArizona , Tucson , Arizona 85721 Received 25 November 1991/Accepted 9 March 1992 A new genetic screening method has been developed to isolate Escherichia coli promoters which are components of the SOS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_86">
Isolation of DNA damage-inducible promoters in Escherichia coli : Regulation of polB ( dinA ) , dinG , and dinH by LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751638" target="_blank">11751638</a>]<br></label>
<input id="rp_similarity_86", type="checkbox">
<div class="RP_similar">
<li>Isolation of DNA damage-inducible promoters in Escherichia coli : regulation of polB ( dinA ) , dinG , and dinH by LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11567013" target="_blank">11567013</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592387" target="_blank">7592387</a>]<br></li>
<li>Isolation of DNA-damage inducible promoters in E. coli : regulation of polB ( dinA ) , dinG , and dinH by LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>Isolation of DNA damage-inducible promoters in E. coli : regulation of polB ( dinA ) , dinG , and dinH by LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682790" target="_blank">8682790</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">LexA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Matrix Consensus ( IUPAC ) order sites Total IC IC per FPR50 % ( LOO ) FPR90 % ( LOO ) FPR100 % ( LOO ) FPR50 % FPR90 % FPR100 % ( matrix sites ) LOO/Matrix sites ( FPR 50 % ) ( matrix sites ) ( matrix sites ) colum n 1 0 1 1 e e e e mem mem mem mem TrpR CRP CysB NanR 24 24 30 20 6 tyGtACtmGykaACtaGTaCratr aaawwtgtGayryagaTCACawww gGAavGrrrtaAyKrmwtrmcarakykm ktATAmMwGktataMmrGww 13.62 7.29 10.67 10.28 0.57 0.30 0.36 0.51 6.67 E 1.38 E 9.29 E 1.53 E + + + + 03 00 03 05 1.30 E-10 6.40 E-242 1900 8.30 E-07 1.00 E 6.20 E 2.60 E 2.30 E -08 -05 -04 -06 7.70 E-06 3.70 E-03 1.40 E-01 1.60 E-03 7.70 E-06 2.80 E-02 1.40 E-01 1.60 E-03 1.50 E 4.50 E 2.80 E 1.50 E -12 -05 -08 -11 1.50 E 2.70 E 3.20 E 6.40 E -09 -03 -05 -07 1.50 E 2.10 E 3.20 E 6.40 E -09 -02 -05 -07 198 8 6 kt 1 1 0 meme meme LexA HipB FNR 21 4 72 22 18 20 wwtrCTGtayatamammCAGya ATCCsskagmGCGGATAA tdywwwtTGaTwwmraTCaa 11.65 10.95 6.94 0.53 0.61 0.35 1.33 E 3.33 E 1.71 E + + + 01 03 0 2.60 E-46 0.00022 2.09 E-71 2.00 E 5.00 E 1.30 E -08 -08 -04 4.60 E-04 4.30 E-06 5.30 E-03 3.10 E-03 4.30 E-06 1.80 E-02 1.50 E 1.50 E 7.60 E -09 -11 -05 1.50 E 3.10 E 2.90 E -05 -09 -03 1.50 E 3.10 E 9.80 E -04 -09 -03 consensus B C Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>SocZ - s ~ p2 -- - ssP3 -- ~ oP2 - , , - AAAAG ~ G ~ GCAGCAGCA ~ TGCAC ~ TAAAAI'IFCCGC TTTTCC ~ GCGCC ~ CGAAATGTTTCGTCGTCGTTAACGTCATTTTAAGGCG - loP3 -- ~ - LexA ACCATTI ~ GC!GGCGTTGATGCCAGCGGCAAACCGAATTAATC TGGTAAAACTTTATTCGACCGCAACTACGGTCGCCGTTTGGCTTAATTAG rpsU ~ MetProVal AAAGGTGAGAGGCACATGCCGGTA Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6206376" target="_blank">6206376</a>]<br></li>
<li>Another promoter has been identified by sequence analysis on the opposite strand at positions - 3 2 4 to - 3 5 3 -91 -81 -71 -61 Cla I -51 -41 -31 -21 - I I ATCGATAGACTCGCCGAGCAGCTCGTACTGACCAATGCCAGTCACGCTGATTAACTGCG -801 TATGA • • G • • GGAAA • AAG • AGCGCAACAAA • GGAAATTC • GGCGGGTTATCTTCCAGCATCGG • GCTAACAGATGCCCTTCCATATGGTGTACAGGGAT -701 CGCCGGAACGTCCCAGGCAAACGCCAGAGAACGCCCCACGGTCGCGCCAACCAGTAGCGCGCCGACTAATCCAGGGCCTGCGGTATAGGCCACAGCATCA -601 ATGTcTTTT ~ CC ~ TTAAAcCAGACTcCTTTA ~ CGCCGCcT ~ ATcAAcG ~ TACG ~ TTTTAC ~ cACAT ~ ATc ~ cG ~ A ~ CcA ~ TTcA ~ CAC ~ ACGCcGC -501 CGTAGTCAGCGTGCAATTTCACCTGACTATACAATTGGTTGGC TAACAAACCTTTTTCATCGTCGTAAATGGCGATGC CGGTTTCATCGCAGGAAGTTTC -401 AATACCCAGTACACGCATGACTTGTTTTACCTCGCTTTATTACCGC ~ GACCAATGCGGGTTG ~ CTTTGTTCGC CCCTGGAGAAAG -301 Sacll `` 35P1 -35 P2 P1 CCTCGTGTATACTCCTCACCCTTATAAAAGTCCCTTTCAAAAAAGGC CGCGGTG ~ AAGCAGCAGCAA ~ TGCA ( ~ AAAAT ~ CCGCACCATT ~ -201 P2 LexA AATAjAGCT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>Nucleotide Sequence of the ClaI-HindIII Restriction Fragment 5 ' to dnaG Gene : Existence of a 2 nut Site and LexA Binding site Plasmid pBS105 was digested with restriction endonucleases ClaI and PstI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>Gene LexA box TACTGTATATAAAAACAGTA TACTGTATATAAAAACAGTA AACTGTATATAAATACAGTT TACTGTATGCTCATACAGTA TACTGTACATCCATACAGTA TACTGTATATAAAACCAGTT AACTGTTTTTTTATCCAGTA ACCTGTATAAATAACCAGTA TGCTGTATATACTCACAGCA TACTGTATATTCATTCAGGT ACCTGAATGAATATACAGTA TACTGTATAAAATCACAGTT AACTGTATATACACCCAGGG TCCTGTTAATCCATACAGCA ATCTGTATATATACCCAGCT TATTGGCTGTTTATACAGTA CGCTGGATATCTATCCAGCA TACTGTACACAATAACAGTA TAATGGTTTTTCATACAGGA GACTGTATAAAACCACAGCC CACTGTATACTTTACCAGTG TACTGTATATATATACAGTA TACTGTATATATATACAGTA CTGNNTNNNNNNNCAG CTGDNTDNNHNNHCAG umuDC sbmC pcsA recA sulA recN ( 1 ) uvrB dinI lexA/dinF ( 1 ) uvrA ssb yebG lexA/dinF ( 2 ) ftsK uvrD dinG ruvAB recN ( 2 ) recN ( 3 ) polB dinB Consensus Search pattern 1 Search pattern 2 Search pattern 3 Results Computational searches and criteria for the identification of LexA binding sites To identify additional genes in the LexA regulon , we have used three different computer analyses and search parameters to identify potential LexA binding sites within the entire E. coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In vivo binding of LexA to the sulA LexA motif in its natural location ( genome position 1,020,172 ) or in each of seven ectopic locations in the coding sequences of lacZ ( 365,099 ) , gltA ( 753,123 ) , araG ( 1,982,342 ) , araE ( 2,979,212 ) , malQ ( 3,547,000 ) , melA ( 4,340,520 ) , and fecD ( 4,509,915 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Independent confirmation of the permissive nature of the E. coli genome comes from the observation that LexA binds comparably to the sulA site artificially introduced within seven different coding sequences representing different levels of transcriptional activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Consistent with this observation , LexA binds comparably to ectopic target sites introduced at various positions in the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Furthermore , we show that LexA binds comparably to an ectopic canonical LexA motif introduced at various positions in the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>It should be emphasized that these search programs and parameters identified all the previously identified LexA-regulated genes or those predicted to be regulated by LexA ( Lewis et al. , 1994 ) , and it seems The underlined sequences in the dinG and recN ( 3 ) LexA boxes highlight the fact that they deviate from the conserved ` CTG0 motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Previous studies found that lexA sequences with an HI value of ,15 generally bound LexA ( Lewis et al. 1994 ) and recently this method was used to identify seven new lexA-regulated genes ( Fernandez De Henestrosa et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>On the basis of earlier calculations that were based upon a smaller set of lexA sequences , Fernandez De Henes-trosa et al. ( 2000 ) found that although dinJ had an HI value of 7.06 , it did not bind LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Q 2000 Blackwell Science Lt In 1994 , using a mathematical formula described previously by Berg and Von Hippel ( 1988 ) to determine the degree of divergence of any 20 nucleotide sequence from the consensus LexA box , Lewis et al. ( 1994 ) defined the term ` heterology index ' ( HI ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Sequences with a low HI value are more similar to the consensus , and thus are presumed to have a greater affinity for the LexA protein than sites with higher HI values ( Fernandez De Henestrosa et al. , 2000 ; Lewis et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Those sequences with a low HI value are closer to the consensus LexA box and are predicted to bind LexA with greater affinity than those sites with a higher HI score .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Thus , although the vast majority of bona fide LexA target sequences are located in intergenic regions , LexA is fully capable of associating with all transcriptionally inactive or active protein-coding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>In this regard , the average distance between the LexA motif and the nearest translational start codon is only 58 bp , a distance very close to the promoter , and in accord with the view that LexA blocks transcription by steric interference with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Thus , at the start of this work it was an open question whether or not LexA recognizes its various operators through a helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>For LexA it has been suggested that a helix-turn-helix motif would start at Arg-28 ( 22 ) , based on weak sequence homology in the first helix of the motif ( i.e. , Ala-32 ) and the highly conserved Gly-36 in the turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>As mentioned in the Introduction it was not clear from the sequence homology whether the structure of LexA contains a helix-turn-helix motif or not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2780544" target="_blank">2780544</a>]<br></li>
<li>A. Schematic of the dinJ -- yafQ palindrome positioned at -16 to -33 from the translation start site ( imperfect repeat ; mismatched residues in lowercase ) relative to the LexA box ; arrows indicate axes of symmetry of repeated sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>DNA sequences that exhibit a close match to the consensus are said to have a low heterology index and bind LexA tightly , whereas those that are more diverged have a high heterology index and are not expected to bind LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_23">
Transcription of the SOS regulon is controlled by the LexA repressor ( Little & Mount , 1982 ) , which binds with high afinity to a consensus operator , the LexA box ( Berg & Von Hippel , 1987 ) , located within or very close to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></label>
<input id="sit_similarity_23", type="checkbox">
<div class="SIT_similar">
<li>LexA binds with high afinity to a consensus operator , the LexA box , located within or very close to the promoter of each SOS gene ( Berg & Von Hippel , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li></div>
<li>To determine , experimentally , whether these sequences bind to LexA protein we conducted gel retardation experiments ( Fried and Crothers , 1981 ; Garner and Revzin , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>We also conducted D N A a s e I footprinting experiments on the phr fragment with the putative SOS boxes in order to uncover weak specific bind ¬ recA R.E. [ LexA ] r Lane uvrA Lan ~ uvrB Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>A comparison of all the sequenced enteric bacteriocin promoters reveals that over 75 % are regulated by dual , overlapping SOS boxes , which serve to bind two LexA repressor proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>In fact a second SOS box-like sequence in the ruv gene which has lower heterology index than the two phr SOS boxes also fails to bind LexA protein ( Shinagawa et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>In conclusion , both theoretical considerations and experimental data indicate that the two SOS box-like sequences immediately upstream of phr do not bind to LexA protein specifically and therefore phr can not be considered a member of the SOS regulon based on available evidence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>The LexA protein does not bind specifically to the two SOS box-like sequences immediately 5 ' to the phr gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Individual SOS boxes vary in their DNA sequences and in their ability to bind the LexA protein ( Walker , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The putative promoter overlaps two 16 bp regions ( positions 326-341 and 348-363 ) that match the consensus for sequences ( SOS boxes ) known to bind LexA protein ( 26,27,28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>LexA protein protected the DNA segment spanning nucleotides 284 to 316 that covered the SOS box at nucleotides 293 to 308 ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>LexA protein bound to this SOS box would prevent R N A polymerase from transcribing from the sulA promoter by partially obscuring th -- 10 region and by completely covering the transcriptional initiation sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>Labelled DNA fragments containing ydjM promoter sequences for hydroxyl radical footprint analysis Primers complementary to sequences bracketing the putative LexA sites in the ydjM promoter were designed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In addition to being present at all previously known LexA targets at their expected coordinates , this LexA motif is also detected by the MEME analysis at five Class II targets : in the promoter regions of minC , fadE , and yoaC , and the coding sequences of ydeQ and b3776 ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>For the LexA matrix , built from 23 binding sites , the ROC curve shows a gradual increase ( Figure 3 ) ; at 50 % sensitivity the expected FPR remains reasonably low ( FPR50 % = 1.3 10 ) , whereas 90 % sensitivity includes 5 almost 1FP per 100 bp ( FPR90 % = 8.3 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>Volume 16 Number 4 1988 Nucleotide sequencing of the ruv region of Eschenchia col K-12 reveals a LexA regulated operon encoding two genes Fiona E.Benson , Graham T.llling , Gary J.Sharples and Robert G.Lloyd Department of Genetics , Queen 's Medical Centre , University of Nottingham , Nottingham NG7 2UH , UK Received December 10 , 1987 ; Revised and Accepted January 21 , 1988 ABSTRACT The nucleotide sequence of a 2505 bp region of the Escherichia coli chromosome containing the LexA regulated ruv gene has been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>All previously characterized chromosomal operators for LexA have been found to contain the symmetrical sequence CTG ( N1O ) CAG , with the central region typically very AT rich .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Interestingly , two operators have a greatly reduced ability to bind LexA when converted from their natural CTG ... CAG sequence to the asymmetric TTG ... CAG form found in dinG ( 21 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Finally , interactions are inferred either at a certain precision ( see Information conten 2 1.5 1 0.5 0 9 10 11 12 13 14 15 16 17 18 19 20 21 8 7 6 5 4 3 2 1 Position Figure 2 Sequence logo of the position weight matrix of the LexA binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>( 1 ) TABLE 1 AFFINITY OF SOS BOXES TO LexA PROTEIN Site Sequence XE Consensus TACTGTATATATATACAGTA ¬ In K recA TACTGTATGAGCATACAGTA 6.7 a 20.0 a uvrA TACTGTATATTCATTCAGGT 7.9 a 17.2 a uvrB AACTGTTTTTTTATCCAGTA 5.6 a 17.7 a In K = l n Ko - ( l/X ) ~ ln [ ( N , o +0.5 ) / l = 1 ruv ( 1 ) CGCTGGATATCTATCCAGCA 10.9 b.c 16.8 b ruv ( 2 ) TACTGTGCCATTTTTCAGTT 16.2 c 8.6 c ( N m + 0.5 ) ] ( 2 ) phr ( 1 ) GGCTGCGCTTATCGACAGTT 20.8 c 6.7 ~ phr ( 2 ) GCCTGGCTTTCAGGGCAGCG 26.8 c 5.2 c ethidium bromide ) only the fragment containing the binding site is retarded at appropriate protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>Sequence determinants for LexA association at the ptrA promoter The strongest unconventional in vivo LexA target is located near the ptrA promoter , and we mapped this site to a 40-bp region using ChIP and quantitative PCR with six primer pairs surrounding the predicted location ( Fig. 7A ; Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Fourth , the LexA site at the ptrA promoter is located downstream of the known 70-dependent transcription start site and upstream of a putative 54-dependent transcription start site ( Fig. 7A ; Claverie-Martin et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The latter observation indicated that both LexA boxes bind LexA protein independently , as would be expected from the fact that they are separated by 37 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>The dyad symmetry present in these sequences and those from past experiments have suggested that LexA binds to each site as a multimer , probably a dimer ( 24-26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>The simplest model is that LexA binds to these sites cooperatively with another protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The actual site at which the LexA protein binds was determined by DNase I protection by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>The region protected from DNase I digestion by the LexA protein spans from positions -11 to + 12 , including the putative Pribnow box and the initiation site of the transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Analysis of binding of the LexA protein to the umu operator , the 159-base-pair ( bp ) Hinfl/Fnu4HI and 167-bp Fok I/Xho II restriction fragments , which were labeled with 32P at the 5 ' end , was performed as described by Ebina et al. ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Under normal growth conditions , LexA binds to a specific 20-base-pair ( bp ) sequence within the promoter regions of these genes , repressing transcription by sterically occluding RNA polymerase ( RNAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>LexA dimerizes with a rather weak dissociation constant of 50 , uM ( Schnarr et al. , 1985 , 1988 ) to form an active dimer that binds to a 16 bp palindromic sequence ( Wertman and Mount , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8076591" target="_blank">8076591</a>]<br></li>
<li>LexA binds at a similar level to the 144 - and 16-bp sequences but does not bind the 14-bp sequence ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Closer examination of the AccI -RCB- BglII region reveals a 19 bp motif , 5 `` - CTG AAcAgTcATgTTT CAG-3 `` ( uppercase letters represent bases which show identity with the LexA operator consensus ; bold letters represent the invariable palindrome found within the LexA operator sequence ) , from ® 671 to ® 689 bp that shows similarity to the consensus sequence of a LexA operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>Closer examination of the AccI -RCB- BglII region reveals a sequence within it that shows similarity to the LexA consensus operator motif and that this putative operator ( LexA-2 ) is able to bind LexA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>This region does not contain the best match to the consensus LexA motif within 1000 bp of the predicted site location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>It does , however , contain a sequence that is a good match to the consensus LexA motif except that the near-invariant CTG motifs in each half-site are replaced by ATG ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>) As shown in Fig. 4b a region of dyad symmetry containing the consensus LexA recognition sequence CTGT ( X ) 9GAG is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>This unique , highly conserved region was further characterized by determination of the degree of divergence from the consensus sequence and by a predicted higher affinity for the LexA protein ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Recent experiments have provided evidence that operators which are most similar to this consensus sequence are bound most tightly by LexA protein in vitro ( 5 , 14 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Lower values of AEN20 indicate a better fit to the consensus sequence and suggest a higher affinity to LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>ACAG ) seen in the consensus sequence for LexA operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>It remains , however , to be established whether the consensus sequence observed is involved in the binding of LexA Alterrepressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>]<br></li>
<li>For this experiment , gcvA was fused downstream of the wildtype LexA DBD sequence of pMS604 and gcvR was fused downstream of the LexA408 DBD sequence of pDP804 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>DinQ is predicted to encode an open reading frame ( ORF ) of 49 aa 139 nt downstream from a LexA operator sequence [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>IstR-2 is a longer small RNA transcript that is LexA controlled and contains the entire IstR-1 RNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20186264" target="_blank">20186264</a>]<br></li>
<li>LexA boxes are identified by asterisks above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>Chromosomally encoded E. coli LexA-regulated genes and the sequence of their respective LexA boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>As expected this search identified the dinQ LexA operator sequence identified earlier in a screen for LexA regulated genes in E. coli [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>These results demonstrate that the LexA-2 operator-like sequence can be recognized and bound by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>Interaction of LexA repressor with the asymmetric dinG operator and complete nucleotide sequence of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002613" target="_blank">8002613</a>]<br></li>
<li>Each of the changes occurred in the putative asymmetric LexA operator sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Low HI values predict that a potential sequence is more likely to bind LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>We examined , by polyacrylamide gel electrophoresis , whether LexA protein would bind to the DNA fragment containing the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>A property these damage-induc-ible genes have in common is a unique operator sequence to which the LexA protein binds , thereby repressing expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Many of the DNA damage-induced genes are negatively regulated by the LexA repressor protein , which binds to a 20-bp consensus sequence in the operator region of the genes , suppressing their expression ( Brent and Ptashne 1981 ; Little et al. 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Under normal growth conditions the LexA protein represses expression of the SOS genes by binding to a specific recognition sequence , at the operator , known as the SOS box ( Little et al. 1981 ; Brent and Ptashne 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>The 134-bp noncoding gap has been reported to contain an SOS-inducible promoter ( dinH ) and a sequence conforming to the consensus sequence for a LexA box ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592387" target="_blank">7592387</a>]<br></li>
<li>The LexA protein is the repressor of the SOS genes and it binds to the operator ( SOS box ) of these genes which has the consensus sequence 5 ' - TACTGTATATATATACAGTA-3 ' ( Berg , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>The LexA protein did not bind to the other SOS box-like sequence , however .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Finally , it should be pointed out that there is extensive sequence homology between a promoter proximal region of suIA and the second SOS box of the LexA operator ( Fig. 4c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>A DNA sequence that is homologous to the SOS box and bound by LexA protein was found in the regulatory region of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>In vitro transcription of the regulatory region of the umu operon revealed that ( l ) upstream of the coding region there is a promoter-operator complex having the consensus sequence , CTGTATATAAAAACAG , of an SOS box , ( it ) transcription of the umu operon begins at an adenine residue in the SOS box , and ( iii ) the LexA protein binds to the operator region , with an equilibrium dissociation constant ( Kd ) of 0.2 , thereby repressing the transcription of the umuDC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Finally , the region surrounding each potential LexA box was analysed for the presence of potential ORFs immediately upstream or downstream of each analysed sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>We will refer to this sequence as LexA-2 , and the LexA box associated with dinH as LexA-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The first , designated LexA box 1 in Fig. 2 , extends from bp 197 to bp 212 , while the second , designated LexA box 2 , extends from bp 250 to bp 265 and overlaps the proposed -35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>The dinQ gene was found in the 823 bp arsR-gor intergenic region ( Figure 1A ) , encoding a ,330 nt transcript with a putative LexA operator sequence ( heterology index ( HI ) = 4.83 ) in the promoter region and two putative ORFs of 18 and 49 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>The ® 10 and ® 35 consensus sequences for the dinH promoter are indicated by lines above the sequence and the LexA operator is boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The ® 10 and ® 35 consensus sequences are capitalized and the LexA operator is boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>Sequence data demonstrates : 1 ) tandem promoters with proper - 3 5 sequences located on the same SacII-HindIII restriction fragment demonstrated to promote expression of Tc ; 2 ) presence of a nut equivalent , pN like binding site ( deCrombrugghe et al. 1974 ) , sequence between the promoter and terminator which could explain the results of Nakamura and Yura ( 1976 ) demonstrating an increase of a synthesis in the presence of the 2 N gene product ; 3 ) an open reading frame with a proper ribosome binding site whose amino acid sequence , when translated by the genetic code , is equal to ribosome protein $ 21 , the rpsU gene product ( Burton et al. 1983 ) ; 4 ) a sequence homologous to the binding site for LexA protein ( Little and Mount 1982 ) located on the tandem promoters and ; 5 ) upstream from the tandem dnaG promoters ( P1 P2 ) is another active promoter ( P3 ) on the opposite DNA strand and transcribing divergently in the opposite direction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>We have determined the complete nucleotide sequence of this controlling region and report : 1 ) tandem promoters on the same SacII-HindIII restriction fragment which promotes tet expression in the gene fusion experiments , 2 ) an open reading frame between these promoters and the dnaG gene which is the rpsU ( ribosomal protein $ 21 ) gene , 3 ) a sequence homologous to the 2 nut site , 4 ) a possible LexA protein binding site on one of the dnaG promoters , This places the order on the E. coli genetic map at 66 min in the clockwise direction as rpsU-dnaG-rpoD which are all contained in a single macromolecular synthesis operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>From the gene fusion and nucleotide sequence experiments reported here , and from experiments previously reported ( Lupski et al. 1982 ; Wold and McMacken 1982 ) and nucleotide sequence ( Burton et al. 1981 ; Smiley et al. 1982 ) previously published and independent confirmed by Burton et al. ( 1983 ) , the order of relevent loci on the E. coli chromosomal region containing the rpsU , dnaG , and rpoD genes is : tandem promoters , LexA binding site , rpsU gene , nut equivalent , transcription terminator , dnaG structural gene , rpoD and a terminator ending the operon ( dnaG P1 , dnaG P2 , lex , rpsU , nuteq , term , dnaG , rpoD , term .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>( C ) Association of LexA with the indicated regions of the ptrA promoter region present at an ectopic location in the coding sequence of melA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_94">
Occupancy was measured as a ratio of binding of LexA or to the tested region and to a control region located within the coding sequence of the predicted ORF sgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></label>
<input id="sit_similarity_94", type="checkbox">
<div class="SIT_similar">
<li>Occupancy was measured as a ratio of binding of GST or GST-LexA to the indicated region and to a control region located within the coding sequence of the predicted ORF sgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li></div>
				<label class="SIT_collapse_similarity" for="sit_similarity_96">
Occupancy was measured as a ratio of binding of LexA to the melA coding sequence and to a control region located around the binding site for LexA at the sulA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></label>
<input id="sit_similarity_96", type="checkbox">
<div class="SIT_similar">
<li>Occupancy was measured as a ratio of binding of LexA to the tested region to a control region located around the natural binding site for LexA at the sulA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Occupancy was measured as a ratio of binding of LexA to the tested region and to a control region located around the binding site for LexA at the sulA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li></div>
<li>These observations suggest that a suitable recognition site for LexA is sufficient for binding in vivo , regardless of its genomic location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Interestingly , our analysis of the unconventional site at the ptrA promoter suggests that a particular suboptimal version of the LexA motif with symmetric mutations in each half-site , not merely a weakened binding site , is necessary and sufficient for LexA binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>[ 18 ] L.K. Lewis , G.R. Harlow , L.A. Greeg-Jolly , D.W. Mount , Identiﬁcation of hight afﬁnity binding sites for LexA which deﬁne new DNA damage-inducible genes in Escherichia coli , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12547390" target="_blank">12547390</a>]<br></li>
<li>Identi ® cation of high af ® nity binding sites for LexA which de ® ne new DNA damage-inducible genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>Furthermore , a recent search for LexA operator sequences in E. coli has identified at least three additional high-affinity binding sites in the genome ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Using equally stringent search conditions for each factor , we screened the E. coli MG1655 genome for DNA sequences that resembled the DNA-binding sites of RutR , LexA , Fur , FruR , FNR , CRP , NarL and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>Finally , another noteworthy observation arising from these studies is that ydjM has two closely spaced LexA binding sites with HI values of 9.07 and 13.60 respectively ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Genes that have an appropriately positioned LexA box with a good heterology index , but do not appear to be LexA regulated Our initial computational search was for potential LexA binding sites that had an HI , 15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In a few cases , adjacent LexA sites are bound in a cooperative manner , further enhancing the sensitivity to changes in signal ( 36 ) , as hypothesized for the ArcA binding sites located upstream of acs and astC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>The search for phylogenetically conserved binding sites for predicted interactions identified five new targets of the transcription factor LexA : cho , dinB , dinI , dinD and yebG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>The first motif was found in 15 bacteriocin gene clusters and is unique in possessing an A-rich region between the − 35 and − 10 boxes ( ∼ 10 bp ) , double LexA binding sites , a relatively long ( ∼ 18 bp ) T-rich region , and an additional short ( ∼ 7 bp ) highly conserved T-rich region ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The bacteriocin dual SOS boxes consist of a highly conserved A tract ( Fig. 2 ) found at the distal and proximal LexA binding sites ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Alveocins A and B ( produced by Hafnia alvei ) and klebicin C ( produced by Klebsiella pneumoniae ) share the presence of double overlapping LexA binding sites and a variable T-rich region , but lack the short consensus T-rich region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Colicin A and klebicin B ( produced by K. pneumoniae ) share an insertion of about 60 bp located between dual LexA binding sites and the SD box ( Riley et al. , 2001 ) , and lack the T-rich region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>nrd4B shows elevated synthesis in lex4 ( Def - ) hosts but does not contain a recognizable LexA binding site in its sequenced regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Direct LexA regulation of these genes has not yet been demonstrated , and their sequenced regulatory regions do not contain recognizable LexA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Alignment of 20 LexA binding sites found in the E. coli chromosome reveals a consensus of 50-TACTG ( TA ) 0 5CAGTA-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>All the search parameters take advantage of the fact that over 20 LexA-regulated genes/LexA binding sites have been analysed experimentally , thereby allowing us to use a consensus LexA binding site ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In contrast to what was found for yigN , there is no predicted LexA binding site immediately upstream of oraA ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544210" target="_blank">11544210</a>]<br></li>
<li>In consequence , the evidence from a detected binding site of Fis in the promoter regions of rplM and rpsI is not as strong as in the case of the detected binding sites of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>Due to the resolution afforded by the arrays , we were able to make relatively accurate predictions as to the location of the LexA-binding site at each target region identified : For previously identified LexA targets , our predictions were accurate to within an average of 167 bp of the known sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>n values for each position of the 20-bp LexA binding site were determined using the known LexA-binding sites shown in Figure 1A and their respective complementary sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>However , because of the short 2 bp spacing between the two LexA boxes in the ydjM promoter , the two patterns of protection resulting from repressor dimer binding to each site overlap , essentially resulting in a ` sharing ' of one weaker area of protection between the two sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>In order to identify potential LexA binding sites ( SOS boxes ) , the computer program N-match was used to search each regulatory region ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Hypothetical ribosome binding sites ( RBS ) are underlined , whereas LexA boxes are either underlined or overlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Comparative analysis of the dinG operator and previously identified LexA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Inter-bacteriocin consensus for 22 bacteriocin LexA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Competition for LexA binding to lexA & lexA3 sites of the uvrCgene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>Sequences in the region upstream of the recN coding region that are homologous with LexA binding sites are overlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>As a consequence , we have not excluded potential LexA binding sites from our study if they are located in a hypothetical ORF with no known function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>As a consequence , we did not eliminate potential LexA binding sites from our analysis even if they were located in the coding region of hypothetical genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Putative LexA binding sites ( overlined and designated [ SOS ] ) and promoter sequences ( underlined with -35 and -10 regions labelled ) were identified as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Third , the deduced nucleotide sequences of the fragments contained recognizable binding sites for the LexA protein located near likely promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>This region also contains two sequences that provide possible sites for binding LexA protein ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>The two dotted 16-bp sequences upstream of ruvA are the LexA binding sites described by Benson et al. ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657895" target="_blank">1657895</a>]<br></li>
<li>The - 1 0 region of the presumptive recQ promoter overlapped the putative terminator for the upstream gene pldA , and was immediately followed by a 15 bp stretch of DNA bearing a strong resemblance to the reported sequences of LexA repressor binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3027506" target="_blank">3027506</a>]<br></li>
<li>Both the uvrA and uvrB genes are inducible by several DNA damaging agents ( Kenyon and Walker , 1981 ; Fogliano and Schendel , 1981 ) and in the regulatory regions of the uvrA and uvrB genes , a LexA repressor binding site ( SOS box ) has been identified ( Van den Berg et al. , 1981 ; Sancar et al. , 1982a , 1982b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>]<br></li>
<li>These results suggest that LexA proteins bind most tightly to a promoter which contains the dual SOS box ( + / + ) , less tightly to a promoter with a single LexA binding site ( + / − and − / + mutants ) , and not at all to a promoter that lacks an SOS box ( − / − ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Recently it was noticed ( Ihara et al. , 1987 ) that in this region there also exist two SOS box ( LexA binding site ) - like sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>Lack of specific binding of SOS box-like sequences of phr to LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>LexA is a repressor protein which inhibits transcription by binding to highly conserved 20-bp sequences located upstream of most or all SOS coding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>The majority of known SOS genes contain a highly conserved binding site for LexA within or adjacent to their promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>As intact LexA binds to the promoter region of SOS genes and acts as a global repressor of the SOS regulon , its RecA-stimulated autocleavage will titrate the repressor away from its DNA binding site and allow transcription of the SOS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16313623" target="_blank">16313623</a>]<br></li>
<li>LexA acts as a transcriptional repressor of these unlinked genes by binding to specific sequences ( LexA boxes ) located within the promoter region of each LexA-regulated gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Interestingly , operon of the closely related organism S. typhimurium has been and does not contain a site for LexA sequenced binding or a potential polB coding sequence downstream of araD ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>This conclusion is supported by the DNA sequence analysis presented in the accompanying paper ( Finch and Emmerson , 1983 ) which reveals a potential LexA binding site in the control region of the uvrD gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6319240" target="_blank">6319240</a>]<br></li>
<li>The deduced sequence of the dinG regulatory region , which is located at 17.8 min on the E. coli genetic map , contains a potential binding site for LexA that is quite different from previously characterized nonoverlapping operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>To conﬁrm that DinJ binds PcspE , we mutated the LexA binding sequence from CTGG to GACC and found that DinJ no longer binds PcspE ( Fig. 3F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>We used a modified form of the standard gel-retardation assay : instead of mixing the binding proteins with a purified DNA fragment containing the binding site we incubated the LexA protein with plasmid DNAs that had been digested with a restriction enzyme of 4-bp recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2509902" target="_blank">2509902</a>]<br></li>
<li>Additionally , LexA binds to a very specific DNA sequence ( Fig. 2 ) : the LexA binding site is flanked by three very well preserved nucleotides containing a stretch of AT repeats .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>As shown in Fig. 4 , the 5 ' sequence of the transcript 1 is pppAAAAACAGUAU ... , which matches the DNA sequence starting with a stretch of five adenine residues in the LexA protein binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>The poorer homology of LexA-2 to the consensus binding motif might reﬂect the lower differential binding afinity of the site for LexA and ensure that the repressor binds preferentially to LexA-1 at dinH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The consensus LexA binding site ( 5 ′ - TACTG ( TA ) 5CAGTA-3 ′ ) , which also is the DinJ binding site , is highlighted in green with the spacer indicated in yellow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>i. Sequences underlined represent deviations from the consensus core LexA binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Light green box , binding site for the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17499044" target="_blank">17499044</a>]<br></li>
<li>This putative SD sequence is preceded by 5 ' - CACTGTATACT ` r-TACCAGTG-3 ' at 8506-8525 ( Fig. 2A ) , a sequence similar to the perfectly symmetrical consensus sequence for the LexA binding site ( 5 ' - TACTGTATATATATACAGTA-Y ; Berg , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>Colicins Ia and Ib possess a single LexA binding site , which differs from the E. coli consensus sequence ( Table 3 ) , and have relatively short conserved T-rich regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The resulting bacteriocin-based consensus sequence varied from the traditional E. coli LexA site consensus ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The second approach used the program MACVECTOR 6.0 ( Oxford Molecular ) to search the E. coli genome for a 16 nucleotide motif , CTGNNT ( N ) 7CAG , which was derived from the consensus LexA binding site sequence shown in Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The central 16-bp sequence of the other LexA binding site in the S. typhimurium lexA gene is identical to that of the E. coli lexA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The location of the putative gene for the 30 kDa protein , close to and upstream of , raises the possibility that it may be involved itself in l214 control contain a LexA protein binding site matching the consensus sequence ( A/T ) - CTGT ( A/T ) Y - ( A/T ) - Y ( A/T ) Y-CAG , where Y is a pyrimidine ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3520484" target="_blank">3520484</a>]<br></li>
<li>The LexA binding site from recA , lexA , uvrA , uvrB , and colE1 are shown with the consensus below them ( taken from Little and Mount 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>The sequence from - 1 9 4 to - 1 7 9 exactly matches the consensus LexA protein binding site ( Little and Mount 1982 ) , of CTG followed 11 bp later by C A G in a six bp inverted-repeat ( see Fig. 4 and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>In gammaproteobacteria , LexA represses transcription by binding to a 16-mer consensus sequence ( CTG-N10-CAG ) called the LexA binding site or SOS box ( Erill et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Upstream of ORF1 , two sequences , CTGTgccatTtTtCAG and CTGgATATcTATcCAG , that were similar to the consensus sequence for the LexA repressor-binding site ( SOS box ) , CTGTATATATATACAG ( 33 ) , were found between nucleotides 240 and 255 and nucleotides 293 and 308 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>Upstream of the umuD gene ( Fig. 1B ) there are two stretches separated by 4 bp that show homology with the consensus sequence for the LexA protein binding site , CTGTatata-a-aCAG , which is called the SOS box ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>The yafNO locus belongs to the SOS regulon of E. coli By inspection of the yafNO promoter sequence , we found that it contains a putative LexA binding site that overlaps with the transcriptional start site and the -10 box ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19943910" target="_blank">19943910</a>]<br></li>
<li>Although the -35 or -10 sequence of most E.coZi din promoters is overlapping with the LexA binding site , the SOS box of the inducible ssb promoter is located upstream of and adjacent to the -35 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2991853" target="_blank">2991853</a>]<br></li>
<li>Table 1 shows that these LexA boxes match the consensus sequence for LexA binding established from a comparison of similar sequences found in the promoter regions of other SOS regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>The LexA repressor downregulates more than 50 SOS genes by binding to the operator sequence in their promoter regions [ 1,2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23408903" target="_blank">23408903</a>]<br></li>
<li>Comparison of the sequence around the sulA promoter with the operator sequences of other SOS genes revealed the presence of an SOS box , the LexA protein binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>Contrasting with the other genes under SOS control , the LexA protein binding site consensus sequence could not be found in the two promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>Two 16 bp sequences ( SOS boxes ) that match the consensus sequence for binding LexA protein are located 5 ' to ruvA in a region that provides a possible single promoter for expression of both ruvA and ruvB , with the second SOS box overlapping the putative -35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>Inspecting the nucleotide sequence around the orfY promoter region , we noticed that it contained near the transcriptional start site a sequence highly homologous ( 17 of 20 ) to the consensus sequence of the SOS box ( 35 ) , the binding site of the LexA protein ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002613" target="_blank">8002613</a>]<br></li>
<li>Its coding region is preceded by a sequence similar to the consensus binding sequence for LexA , the so-called SOS box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>LexA represses SOS genes by binding , as a dimer , to a consensus sequence ( the LexA-box ) located at the promoter region of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li>
<li>Tn t , he nucleotide sequence of the sulA region ( Beck & Bremer , 1980 ) , a number of possible promoter sequences exist , as well as a sequence that matches , at 16 out of 19 bases , the consensus sequence for the LexA repressor binding site ( Little & Mount .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>Overlapping our proposed start point for transcription and t , he - 10 region of the promoter is a nucleotide sequence that has 16 of 18 bases homologous to the consensus sequence for LexA binding sites ( Little & , Mount .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>In the DNA region of the P2 promoter no sequence can be indicated which matches the consensus sequence for LexA binding ( CTGN 1CAG ) ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3295776" target="_blank">3295776</a>]<br></li>
<li>In the DNA sequence a putative LexA binding site could be indicated , overlapping the P3 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3295776" target="_blank">3295776</a>]<br></li>
<li>In order to define a minimal binding site for LexA at the ptrA promoter , we created derivatives of MG1655 that contain an 144 - , 16 - , or 14-bp sequence centered around the putative LexA-binding site at the ptrA promoter , introduced into the coding sequence of melA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Thus , we have defined a minimal 16-bp sequence that is sufficient for LexA binding at the ptrA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The presence of the LexA consensus site in the dinJ promoter suggests a possible link between the SOS and the stringent response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>A , wild-type promoter region of dinJ with each IR of operator 1 and 2 , the 35 and 10 promoter regions , the ribosome-binding site ( RBS ) , and the LexA consensus box indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>Analysis of the promoter/operator regions of these din genes , as well as other genes known to be regulated by LexA , revealed a 0 consensus LexA binding site of 5 - TACTG ( TA ) 5CAGTA-30 ( Walker , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Third , analysis of the sequence of the lexA gene of S. typhimurium has revealed that one of the two LexA binding sites upstream of this gene ( box 2 ) also contains TTG ( N10 ) CAG ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>This analysis suggests that , except for the disruption in the highly conserved CTG ... CAG motif ( which diminishes LexA binding at two other sites ) , the dinG sequence is otherwise very similar to known SOS operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>The premise that LexA binds operator DNA as a multimeric complex is also supported by the finding that all chromosomal binding sites which have been characterized are imperfect 20-bp palindromes containing the sequence CTG ( N1O ) CAG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The sequence deduced for the promoter region of dinG contains a potential 20-bp operator segment which does not retain the symmetrical CTG ( N1O ) CAG motif found in all other chromosomal LexA binding sites but instead contains the asymmetric sequence TTG ( N1O ) CAG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>rte o ep R The consensus sequence of enteric bacteriocin promoters contains a σ70 promoter region ( identified as − 35 and − 10 boxes ) , one or two LexA binding sites , a thymine ( T ) - rich region and a Shine -- Dalgarno ( SD ) box ( Riley et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>A 16-bp sequence extending across the proposed -35 region from bp 422 to 437 has some similarity with LexA binding sites ( CTGN10CAG ) found in the promoters of SOS-in-ducible genes ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657895" target="_blank">1657895</a>]<br></li>
<li>The LexA binding sites of the proximal recN and distal umuDC sites are similar in sequence to the colicin SOS box motif ( HI values of 1.1 and 2.6 , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The highly conserved region downstream of the σ70 box compared to the upstream sequence ( data not shown ) in most of the colicin promoters suggests that the LexA binding sites play a major role in enhancing promoter strength and control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The location and sequence of the predicted LexA binding sites ( box 1 and box 2 ) are indicated above the scans , while the region of protection afforded by each LexA dimer bound to each box is indicated by the hatched bar below the scan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The finding that the recQ region contained a sequence analogous to those of LexA repressor binding sites strongly suggested that SOS regulation operates on recQ gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3027506" target="_blank">3027506</a>]<br></li>
<li>The nucleotide sequence of the dinH regulatory region contains a segment which is very similar to previously characterized binding sites for LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The region whose sequence is nearly homologous with LexA binding sites in the recA and lexA genes is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>Nucleotide sequence binding specificity of the LexA repressor of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19555451" target="_blank">19555451</a>]<br></li>
<li>The predicted LexA binding sites for polB and dinH contain the consensus sequence CTGT ( N8 ) ACAG with an AT-rich central segment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>In contrast to LexA , the binding sites of Fis are less preserved in their sequence ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20718955" target="_blank">20718955</a>]<br></li>
<li>We have not seen any obvious homologies to consensus sequence binding sites for LexA , SoxS , OxyR , or KatF in the promoter regions of the fpg or mutY operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>However , in vitro binding studies show that the DinJ-YafQ complex binds to the first operator but not the second operator , although the SOS-activated , transcriptional repressor LexA binds to the second operator that also contains an overlapping LexA consensus sequence ( 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>Binding constants determined experimentally have indicated that those sites which this consensus match operator sequence most closely also exhibit the tightest binding to the LexA protein ( 2 , 25 , 47 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Past studies have indicated that all functional LexA binding sites exhibit great similarity to the palindromic 20-bp consensus sequence shown in the figure and that those operator sequences which are most similar to the consensus site generally interact with LexA protein most strongly ( 5 , 14 , 24 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>Triple asterisks below bases , termination codons ; solid bar below sequence , putative LexA repressor binding site ; boxes , -35 and -10 hexanucleotide sequences of potential promoter ; solid and open arrows , hyphenated dyad symmetry of possible terminators of pldA and recQ , respectively ; underlining , possible initiation codons ; linesabovesequence , possible Shine-Dalgarno ( SD ) sequences ; dashedlines abovesequence , restriction enzyme cleavage sites ( those referred to in the text specified by numbers in parentheses ) ; inverted triangles , Tn3-chromosome boundaries at the recQ promoter distal ends of the inserts 2500 2695 bp i i 0 500 1000 i 1500 i 2000 i i i 1 ) E ( 1 ) ~ ( 1 ) A ( 2 ) i E ( 2 ) i E E P -LCB- A ) C A ( 3 ) i i i i i A ( 4 ) I Analysis of the nucleotide sequence of the recQ region The complete nucleotide sequence of the PvuII ( 1 ) - AluI ( 4 ) fragment consisting of 2,695 bp was determined ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3027506" target="_blank">3027506</a>]<br></li>
<li>The significance of this homology is not yet clear , but Blanar et al. ( 25 ) suggested that in RecA , the a-3 helix together with the ATP-binding site ( a ¬ Titration of the 566 bp Sau3A fragment containing the recN promoter ( lanes a-h ) and the 530 bp tet fragment of ~ pACYC184 ( lanes i-o ) with LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>( B ) The DNA sequence from position ( boxed ) and the putative 190-310 containing the ' - 35 ' and - 10 ' regions LexA binding site ( underlined ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>]<br></li>
<li>The exact location of that promoter has not been established , but there is a putative LexA binding site 35 bp downstream of the end of the lIp gene ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The slow derepression of the uvrC promoter could be ascribed to the slight alteration in the structure of the putative LexA binding site where the dyad symmetry is separated by 11 bp instead of the 10-bp distance as is generally observed in other din genes ( Cole , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>]<br></li>
<li>Furthermore , the gel retardation assay of Fried and Crothers ( 17 ) revealed specific binding of LexA protein to this 590 bp Sau3AI fragment and to the two sub-fragments of this region generated by cleavage at the TaqI site beginning at bp 217 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>Regions of dyad symmetry are depicted by arrows in opposing orientation and demarcate 1 ) the putative LexA protein binding site , 2 ) a possible nut equivalent site and 3 ) the terminator we previously reported ( Smiley et al. 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>The LexA protein binding site of the umu operon has an extensive homology with those found in other SOS genes so far sequenced ( 24 , 31-36 ) and has a near-perfect dyad symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>At the bottom is the putative LexA protein binding site from the rpsU-dnaG-rpoD macromolecular synthesis operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>A new regulatory DNA motif of the gamma subclass proteobacteria : identiﬁcation of the LexA protein binding site of the plant pathogen Xylella fastidiosa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>The fact that DNA sequence is a remarkably accurate predictor of LexA binding in vivo , as indicated by the ChIP -- chip score , is in stark contrast to the situation in yeast and mammalian cells , where numerous high-quality DNA sequence motifs are not bound by the relevant protein in vivo ( Iyer et al. 2001 ; Lieb et al. 2001 ; Martone et al. 2003 ; Cawley et al. 2004 ; Euskirchen et al. 2004 ; Harbison et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>The value for the SOS box-like sequence upstream of dinP was calculated to be 12.8 , which means that the site has a much weaker affinity to LexA than the SOS box sequence in the umuDC operon with the value of 1.60 ( Lewis et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>The consensus sequence forms an almost perfect palindrome , the distal LexA site is far more conserved than the proximal site , a highly conserved A motif rather than AT motif is present , and the proximal SOS box has a CAC motif in place of the CAG usually encountered in E. coli ( Fernandez De Henestrosa et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>SOS box analysis All dual SOS boxes were aligned according to the consensus LexA binding motif N2CTGTN9CAGN2 ( Fernandez De Henestrosa et al. , 2000 ) and employing CLUSTALW ( Thompson et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>The majority of enteric bacteriocins possess dual LexA binding sites ; however , these sites deviate considerably from the E. coli SOS box consensus ( Fernandez De Henestrosa et al. , 2000 ; Lewis et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>This study has been aided by the fact that ( i ) the entire E. coli genome has been sequenced ( Blattner et al. , 1997 ) and ( ii ) the consensus binding site for LexA provides a motif that can be used for computational searches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Recent studies by Gervais et al. ( 21 ) also identified a ( 770 promoter and a LexA binding site upstream ( positions -202 to -218 ) of rcsB in E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8366025" target="_blank">8366025</a>]<br></li>
<li>DinJ binds the cspE promoter To determine if antitoxin DinJ directly controls the transcription of cspE by binding , an electrophoretic mobility shift assay ( EMSA ) was used for the 35 bp ( from -87 to -53 ) and 176 bp fragments of PcspE ( from -123 to +53 ) , both of which contain the consensus LexA binding sequence ( 5 ′ - TACTG ( TA ) 5CAGTA-3 ′ ) ( Fernández De Henestrosa et al. , 2000 ) as shown in Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>YafQ and DinJ form a stable complex ( Motiejūnaite et al. , 2007 ) , which binds the dinJ-yafQ palindrome ( 5 ′ - CTGAATAAATATACAG-3 ′ , -16 to -33 from the translation start site ) ( Prysak et al. , 2009 ) which overlaps the consensus LexA binding site ( 5 ′ - TACTG ( TA ) 5CAGTA-3 ′ ) ( Fernández De Henestrosa et al. , 2000 ) , suggesting that this module is regulated by DNA damage ( Prysak et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22026739" target="_blank">22026739</a>]<br></li>
<li>The palindrome upstream of dinJ -- yafQ is unique among known E. coli proteic TA modules because this sequence has similarity to a consensus LexA binding site ( Lewis et al. , 1994 ; Fernandez De Henestrosa et al. , 2000 ) , suggesting that this module is regulated by DNA damage ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19210620" target="_blank">19210620</a>]<br></li>
<li>Sequence analysis of LexA-regulated promoters revealed the consensus sequence TACTG ( TA ) 5CAGTA as the binding site for LexA ( Walker 1984 ) , and computational analysis has identified additional LexA targets in the E. coli genome ( Lewis et al. 1994 ; Fernandez de Henestrosa et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>These were located upstream of all eight class II flagellar promoters and 39 non-flagellar promoters , a figure comparable with the 69 putative E. coli LexA regulon binding sites identified with an HI below 15 ( Lewis et al. , 1994 ; Fernandez de Henestrosa et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15941987" target="_blank">15941987</a>]<br></li>
<li>ChIP-chip data LexA ChIP-chip detected binding sequences were obtained from the Supplementary Material of Wade et al. ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20923783" target="_blank">20923783</a>]<br></li>
<li>One method of predicting whether a LexA-like sequence will bind LexA is to examine its heterology index ( HI ) , which is a value derived from a mathematical formula ranking the relatedness of a potential sequence to that of known LexA-binding sequences ( Lewis et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Lewis et al. ( 1994 ) tested their hypothesis directly by performing LexA-dependent gel mobility shift assays on a number of potential LexA binding sites and found that LexA bound to sequences with an HI value of 12.6 or lower , but not to sequences with an HI value .15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>However , no LexA boxes are evident upstream of the mfd gene and although a computer algorithm that identifies potential transcription factor binding sites in DNA sequences by cross-species comparison predicted three potential regulatory sites in the ycfT -- mfd intragenic region , none of the sequences correspond to the binding sites of known transcription factors ( McCue et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12658334" target="_blank">12658334</a>]<br></li>
<li>In this downstream sequence , Lewis et al. ( 23 ) found one of the high-afﬁnity binding sites for LexA , which overlapped with a probable 210 signal sequence for transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8682790" target="_blank">8682790</a>]<br></li>
<li>Immediately following the presumptive - 1 0 region , there was a 15 base sequence ( CTGTTTTTATTTCAG ) with close similarity to the sequences ( ) proved or thought to be LexA repressor binding sites ( van Sluis et al. 1983 ; Walker 1984 ; Mechulam et al. 1985 ) , suggesting that the expression of the recQ gene is under the control of the SOS system ( Witkin 1976 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3027506" target="_blank">3027506</a>]<br></li>
<li>Strong positive and possible negative evolutionary selection for genomic locations of canonical LexA sites Based on the PWM derived from our genomic analysis , a canonical LexA sequence should occur by chance on average approximately nine times in the E. coli genome ( the average number of sites with ScanACE score > 16.5 in a randomized genome ) , with approximately two sites in intergenic regions and seven sites in protein-coding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Indeed , across 100 coding sequence randomizations , the number of such high-scoring LexA sites found was always > 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>We identify 49 high-confidence LexA target regions in E. coli with low false-positive and false-negative rates , and map these sites to a resolution of 167 bp without any sequence information ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Recently , Wade and co-workers ( 11 ) demonstrated that LexA binds to artiﬁcial sites introduced within coding DNA and concluded that , since few such LexA sites exist in the wild type E. coli genome sequence , they must have been removed as the genome evolved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18515344" target="_blank">18515344</a>]<br></li>
<li>We determined whether LexA binds to these sites in vivo using ChIP and quantitative PCR , defining a true bound region as having more than twofold enrichment of target DNA relative to the control region in the sgrR coding sequence ( equivalent to 1 Occupancy Unit ; see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>eDistance between predicted site and canonical LexA motif for Class I and Class II targets in base pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>To determine whether this sequence is required for LexA association in vivo , we analyzed , in an otherwise MG1655 background , 1 - , 20 - , and 80-bp chromosomal deletions centered at this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_238">
We also identify ∼ 20 novel LexA targets that lack a canonical LexA sequence motif , are not bound by LexA in vitro , and presumably require an additional factor for binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></label>
<input id="sit_similarity_238", type="checkbox">
<div class="SIT_similar">
<li>We also identify ∼ 20 novel LexA targets that lack a canonical LexA sequence motif and are not bound by LexA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li></div>
<li>In addition , using MEME and a variety of other motif-finding programs , we were unable to identify a common DNA sequence motif among LexA targets that lack a canonical LexA motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>In this vein , our analysis reveals a novel , and unexpected , class of 19 LexA targets that lack a LexA sequence motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Specifically , virtually all canonical LexA sequence motifs that are bound by LexA in vitro associate with LexA in vivo , indicating that the presence of a suitable DNA sequence is sufficient for LexA association in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>A canonical LexA sequence recognition motif is sufficient for LexA binding in vivo We used MEME ( Bailey and Elkan 1994 ) to identify common DNA sequence motifs among genomic regions representing 1000 bp centered at each of the 49 predicted LexA sites ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Thus , LexA association at the ptrA promoter is mediated by an aberrant , but specific , LexA motif-like sequence in which key residues directly contacted by LexA are altered so as to preclude DNA binding in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>We demonstrate a near-universal relationship between the presence of a LexA sequence motif , LexA binding in vitro , and LexA binding in vivo , suggesting that a suitable recognition site for LexA is sufficient for binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>We demonstrate a remarkable correlation between the correspondence of a LexA sequence motif to the consensus , the extent of LexA binding in vitro , and LexA binding in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by LexA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Perry , K.L. , S.J. Elledge , B.B. Mitchell , L. Marsh and G.C. Walker ( 1985 ) umuDC and muchB operons whose products are required for UV light - and chemical-induced mutagenesis : UmuD , MucA , and LexA proteins share homology , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>Ann Rev Biochem 3:293 -321 Perry KL , Elledge SJ , Mitchell BB , Marsh L , Walker GC ( 1985 ) umuDC and mucAB operons whose products are required for UV light - and chemical-induced mutagenesis : UmuD , MucA , and LexA proteins share homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3025556" target="_blank">3025556</a>]<br></li>
<li>Furthermore , strains containing uncleavable LexA do not exhibit the expected ﬁlamentation phenotype , suggesting that even when LexA is irreversibly bound to its operators , ftsK transcription is unaffected and cell division proceeds normally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The LexA system uses two reporter genes ( LEU2 and lacZ ) under the control of multiple LexA operators to eliminate false positives ; the assay for LEU2 activity tests for growth of transformed yeast cells in the absence of leucine ( Leu plates ) and the assay for lacZ measures the specific activity of its encoded product , b-galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19386726" target="_blank">19386726</a>]<br></li>
<li>An operon structure of this nature is supported by the fact that 8-galactosidase synthesis in ruv-61 : : Mud ( Aprlac ) insertion strains ( 2,3 ) is subject to LexA regulation ( 3 ) and that this insertion is located in the KpnI-BglII region in the correct orientation for expression of lac from the suggested promoter ( unpublished work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>Thus , under the conditions tested , the dinS and ybfE promoters are not true LexA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Robust expression of the 19-kDa RecX band was detected in strain DE192 , a lexA51 ( Def ) strain that does not produce a functional LexA repressor , but not in the isogenic strain DE192 recX ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12427742" target="_blank">12427742</a>]<br></li>
<li>Since it has been experimentally shown that LexA repression controls expression of functions in both the e14 early left and early right operons [ 18,19 ] , and transcription of the majority of genes tested in these operons have been found to increase following UV irradiation [ 35 ] , the promoters and operators on both sides of b1145 appear to have remained intact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14733619" target="_blank">14733619</a>]<br></li>
<li>Sequence data demonstrates : 1 ) tandem promoters with proper - 3 5 sequences located on the same SacII-HindIII restriction fragment demonstrated to promote expression of Tc ; 2 ) presence of a nut equivalent , pN like binding site ( deCrombrugghe et al. 1974 ) , sequence between the promoter and terminator which could explain the results of Nakamura and Yura ( 1976 ) demonstrating an increase of a synthesis in the presence of the 2 N gene product ; 3 ) an open reading frame with a proper ribosome binding site whose amino acid sequence , when translated by the genetic code , is equal to ribosome protein $ 21 , the rpsU gene product ( Burton et al. 1983 ) ; 4 ) a sequence homologous to the binding site for LexA protein ( Little and Mount 1982 ) located on the tandem promoters and ; 5 ) upstream from the tandem dnaG promoters ( P1 P2 ) is another active promoter ( P3 ) on the opposite DNA strand and transcribing divergently in the opposite direction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6222240" target="_blank">6222240</a>]<br></li>
<li>Although wd have ' Qbserved that the rep promoter is slightly inducitle by DNA-damaging agents , our genetic data indicate that - the LexA ptotein does notrepress the expression of the rep gene ( data-not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2999067" target="_blank">2999067</a>]<br></li>
<li>Since the AEN20 value of 12.8 is slightly higher than that ( 12.1 ) for the dinA gene and well in the range lower than those for the operator-like sequences which do not footprint with LexA ( above 15.5 ; see Lewis et al. , 1994 ) , it is most likely that dinP gene expression is under the control of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596361" target="_blank">7596361</a>]<br></li>
<li>These transformants were picked and transferred to duplicate MacConkey agar plates containing either 1 mM IPTG to induce LexA expression or no IPTG as a repressed control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>These authors examined the changes in gene expression after UV exposure ( 40 J/m2 ) in both wild-type cells and lexA1 mutants , which are unable to induce genes under LexA control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17121995" target="_blank">17121995</a>]<br></li>
<li>Galactokinase activity in cells harboring pLKL59 ( wild type ) was 3.5 times higher when LexA expression was repressed by lacI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>On the other hand , LexA expression is also increased upon H2O2 treatment ( 31 ) , but it did not co-purify with YchF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Thus , although LexA serves as a negative regulator for polB , it essentially behaves as a positive regulator of araDAB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>natively , the results described here are not in disagreement with model where uvrC expression is directly a not modulated by LexA but rather by damage-inducible positive effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6321159" target="_blank">6321159</a>]<br></li>
<li>Coordinated derepression of these genes by the recA and lexA genes results when RecA is activated by DNA damage to promote cleavage of LexA ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>In addition , there are several examples of genes , such as dnaA , dnaN , dnaQ , phr , and recQ ( 28 ) , that appear to be regulated in a recA - and lexA-dependent manner but have not been shown to be directly repressed by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>additional genes ( phr , recQ , and dnaA , dnaN , dnaQ ) exhibit recA - , lexA-dependent DNA induction but have not been shown to be damage directly LexA repressed by ( 13 , 14 , 38 , 39 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>Five other genes ( phr , recQ , dnaA , dnaN , and dnaQ ) require recA and lexA for transcriptional induction by DNA damaging agents but may not be directly repressed by LexA ( see text ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>To do so , we have made oligonucleotide probes to each of the genes identified in Table 2 that have not already been shown to be LexA regulated ( those lacking an asterisk in Table 2 ) and have analysed the pattern of gene expression in three different lexA backgrounds in the absence and presence of exo-genous DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Known LexA-regulated genes , such as recA and umuDC , which we used as positive controls in these experiments , exhibit the following response in the three backgrounds : in a wild-type lexA cell , there is basal 1 expression of the gene , which is enhanced by exposure to DNA damage ; in a lexA3 ( Ind ) strain , which has a G85D 2 change at the cleavage site of LexA ( Markham et al. , 1981 ; Little , 1984 ) , thus rendering the repressor non-cleavable ( Lin and Little , 1989 ) , no damage-inducible induction of LexA-regulated genes is observed ( Mount et al. , 1972 ) ; in a lexA51 ( Def ) mutant strain that encodes a defective repressor ( Mount , 1977 ; see Experimental procedures ) , all LexA-regulated genes are expressed at derepressed levels , irrespective of whether the cell has been exposed to exogenous DNA damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Thirdly , the potential LexA box should not be located inside a gene with a known function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>LexA is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>Mizusawa , S. , Court , D. and Gottesman , S. ( 1983 ) Transcription of the sulA gene and repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Transcription of the sulA gene and repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Transcription of the sit/A gene and repression by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666395" target="_blank">2666395</a>]<br></li>
<li>Interestingly , uvrC is the only gene of the UvrABCD excision repair complex that is not apparently regulated by LexA , and it is possible that ydjQ is an ancestral homologue of uvrC that retains its LexA regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>The downstream gene , designated hepA in this report , is not regulated by the LexA The repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_31">
dinH , that is regulated by LexA , is immediately a gene downstream of lrp and is transcribed in the direction ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="tu_similarity_31", type="checkbox">
<div class="TU_similar">
<li>dinH , a gene that is regulated by LexA , is immediately downstream of lip and is transcribed in the same direction ( 83 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li></div>
<li>nucleotides which was not regulated by LexA ( Fig. 6 ) , it was likely transcribed from the opposite DNA strand to the template for the P1 and P2 transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>Other genes from the E. coli genome that are LexA regulated are ysdAB , dinQ , hokE , ydjM , dinS and ybfE ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>According to this model , LexA would ordinarily repress ftsK expression by binding its cognate operator at dinH , transient displacement of LexA from LexA-1 to LexA-2 , in response to an uncharacterized cellular signal , would free dinH and allow transcription of ftsK in response to the cell 's requirement for FtsK protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>It is not known whether such a cascade occurs in vivo under conditions when RecA protein is in excess , for example , when the LexA regulon , which includes recA , is derepressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8080625" target="_blank">8080625</a>]<br></li>
<li>A recent study of one of these genes , phr , indicated that LexA does not bind to its upstream promoter region ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>In vitro transcription of the regulatory region of the umu operon revealed that ( l ) upstream of the coding region there is a promoter-operator complex having the consensus sequence , CTGTATATAAAAACAG , of an SOS box , ( it ) transcription of the umu operon begins at an adenine residue in the SOS box , and ( iii ) the LexA protein binds to the operator region , with an equilibrium dissociation constant ( Kd ) of 0.2 , thereby repressing the transcription of the umuDC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Similar phenomena were observed in the s - driven promoters of sulA , uvrA , recA , fecI , entC and lacZ , 70 70 the transcription of which is directly controlled by a repressor protein ( LexA , Fur and LacI ) , but not in promoters of the housekeeping genes ftsA and ftsY , or in s - driven treA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21396442" target="_blank">21396442</a>]<br></li>
<li>The AccI/BglII region is essential for ftsK transcription and binds LexA Expression of genes controlled by LexA has a genetic dependence on recA function , yet our observations show transcription of ftsK from dinH in a recA minus background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>]<br></li>
<li>The expectation was that expression of SOS-responsive gene fusions controlled by LexA-regulated promoters would be induced by both nalidixic acid and mitomycin C in a manner that is dependent on normal LexA function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544210" target="_blank">11544210</a>]<br></li>
<li>Analysis of recN : : lacZ fusion strains showed that recN is regulated directly by LexA and that its expression can be induced by ultraviolet light , gamma-radiation , and nalidixic acid , in addition to mitomycin C ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>The results summarised in Table 2 show that CAT expression is activated in pSP103 and is regulated by LexA , from which we conclude that the 0.9 kb PstI fragment does contain the recN promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3037486" target="_blank">3037486</a>]<br></li>
<li>Since expression of ruv is known to be regulated by LexA ( 3 ) , it seems likely that the region immediately upstream of the first open reading frame is the promoter for expression of ruv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>This response is controlled by the regulatory RecA protein , which in activated form promotes the cleavage of LexA repressor , allowing for the increased transcription of about 17 repressed genes ( Little and Mount , 1982 ; Walker , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3009022" target="_blank">3009022</a>]<br></li>
<li>LexA directly regulates ∼ 30 E. coli transcription units involved in the `` SOS '' response whose transcription is induced in response to DNA damage ( Little and Mount 1982 ; Walker 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
<li>Similar to tisB and symE , which are both repressed by LexA and are induced by DNA damage ( Vogel et al. , 2004 ; Kawano et al. , 2007 ) , transcription of the ibs and shoB genes may also be induced by speciﬁc growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Overview of the istR/tisAB Locus LexA control and thus induced by DNA damage/SOS conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17499044" target="_blank">17499044</a>]<br></li>
<li>Investigation of 0-galactosidase synthesis in the ruv : : lac fusion strains demonstrated that expression of ruv is induced by damage to DNA and is regulated by LexA as part of the SOS response ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>ruvA and ruvB form an operon and are regulated by LexA repressor as part of the SOS response to DNA damage ( Shurvinton and Lloyd , 1982 ; Benson et al. , 1988 ; Shinagawa et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8428576" target="_blank">8428576</a>]<br></li>
<li>ruvA and ruvB form a single operon ( 4 , 28 ) which is regulated by LexA as part of the SOS response ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657895" target="_blank">1657895</a>]<br></li>
<li>form part of an operon regulated by LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3279394" target="_blank">3279394</a>]<br></li>
<li>Since the nucleotide sequence of this region suggests that the ruv gene and ORF1 are closely located in the same orientation and are flanked by a putative operator ( s ) ( SOS box ) and terminator ( s ) , ruv and ORF1 may constitute an operon that is regulated by the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>Therefore , the orfi and ruv genes appears to constitute an operon which is regulated by the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2842314" target="_blank">2842314</a>]<br></li>
<li>Both uvrA and ssb genes are organized in the SOS regulon and regulated by the LexA repressor [ 15,29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16140472" target="_blank">16140472</a>]<br></li>
<li>It is required for mediating the autodigestion of LexA repressor , thereby increasing expression of the SOS regulon including the umuDC operon , for mediating the auto-digestion of UmuD to UmuD9 , which is the active form in SOS mutagenesis , and for a third , as yet poorly understood role in the process of translesion synthesis ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>Together , these observations suggest that , in contrast with its classical regulatory role in the SOS response , LexA can not directly act as a repressor of the pheST-himA operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3326947" target="_blank">3326947</a>]<br></li>
<li>It is repressed by LexA , and its expression increases following SOS-induced cleavage of LexA by RecA * , the activated form of RecA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8755866" target="_blank">8755866</a>]<br></li>
<li>The expression of the ruvAB operon is normally repressed by LexA unless the SOS response is activated ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142427" target="_blank">12142427</a>]<br></li>
<li>It is not yet known whether other genes belonging to the mismatch repair system are also under the control of the LexA repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3937030" target="_blank">3937030</a>]<br></li>
<li>It has been found that the promoter of the gene uvrD is under the control of the LexA repressor ( Nakayama et al. 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3937030" target="_blank">3937030</a>]<br></li>
<li>It is known that LexA binds to the sulA promoter tightly as compared to other SOS genes and is induced 100-fold during the SOS response ( Friedberg et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15387814" target="_blank">15387814</a>]<br></li>
<li>EMBO J 3:707 -712 Calsou P , Villaverde A , Defais M ( 1987 ) Activated RecA protein may induce the expression of a gene that is not controlled by LexA repressor and whose function is required for mutagenesis and repair of UV-irradiated lambda phage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2972909" target="_blank">2972909</a>]<br></li>
<li>At the same time , the LexA repressor that regulates expression of all SOS genes also controls transcription of the TisAB toxin/antitoxin module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23090599" target="_blank">23090599</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23090599" target="_blank">23090599</a>]<br></li>
<li>Homologies are indicated by * Huisman and D'Ari ( 1981 ) elegantly demonstrated that expression of sulA is regulated at the transcriptional level by the LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>There have been interesting reports for the 35-kDa protein gene-ftsL region , a sequence that has high homology with the LexA repressor-binding sequence ( 28 ) and a negative control by mreB of the ftsI expression ( 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294438" target="_blank">9294438</a>]<br></li>
<li>Colicin-E1 structural gene expression is repressed by the LexA protein ( Ebina et al. 1982 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6206376" target="_blank">6206376</a>]<br></li>
<li>Its expression , like that of other SOS genes , is repressed by the LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>First , expression from the din : : galK fusions was repressed by high cellular levels of LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1577702" target="_blank">1577702</a>]<br></li>
<li>The second mutant host , strain DM1187 ( Kim & Little , 1992 ) , served as a negative control , as it encodes a LexA protein unable to bind to the operator , resulting in unregulated expression of the SOS-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18524933" target="_blank">18524933</a>]<br></li>
<li>Many of the DNA damage-induced genes are negatively regulated by the LexA repressor protein , which binds to a 20-bp consensus sequence in the operator region of the genes , suppressing their expression ( Brent and Ptashne 1981 ; Little et al. 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11333217" target="_blank">11333217</a>]<br></li>
<li>Under normal growth conditions the LexA protein represses expression of the SOS genes by binding to a specific recognition sequence , at the operator , known as the SOS box ( Little et al. 1981 ; Brent and Ptashne 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6306396" target="_blank">6306396</a>]<br></li>
<li>We found that the divergently transcribed ssb and uvrA genes are controlled by a common LexA binding site ( SOS box ) , indicating that also ssb gene expression is SOS inducible ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2991853" target="_blank">2991853</a>]<br></li>
<li>LexA protein is a repressor of its own gene , recA and a number of other , commonly called din ( damage inducible ) genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3937030" target="_blank">3937030</a>]<br></li>
<li>A property these damage-induc-ible genes have in common is a unique operator sequence to which the LexA protein binds , thereby repressing expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>First we tested two strains in which the SOS response can not be induced either because the LexA repressor in uncleavable ( lexA ( Ind − ) strain ) or by lack of the RecA protein ( the recA99 strain contains an amber codon early in the recA gene coding sequence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12547390" target="_blank">12547390</a>]<br></li>
<li>The common denominator of these promoters is that they are all negatively regulated by a repressor protein : lacZ is controlled by LacI that dissociates from the lac operator upon its interaction with lactose ( Reznikoff , 1992 ) ; SOS promoters are controlled by LexA , which is self-cleaved upon interaction with RecA single-strand DNA ﬁlament ( Butala et al. , 2009 ) ; iron acquisition genes are repressed by Fur when it is bound to Fe , and are derepressed þ2 when iron is scarce ( Brickman et al. , 1990 ; Braun et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21396442" target="_blank">21396442</a>]<br></li>
<li>Expression of many of the genes in the SOS regulon are controlled by the LexA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Expression of the SOS regulon is controlled by the RecA and LexA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489426" target="_blank">15489426</a>]<br></li>
<li>Expression of all SOS genes is controlled by two regulatory proteins , RecA and LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1629168" target="_blank">1629168</a>]<br></li>
<li>[ 25 ] The LexA protein can not interact with LexA-boxes , the SOS system is constitutively induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12547390" target="_blank">12547390</a>]<br></li>
<li>The apparent Kd value of the LexA protein and the umuDC operator , which was determined under the same conditions ( 37 ° C , 200 nM KCl ) used for other SOS genes is = 0.2 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2989817" target="_blank">2989817</a>]<br></li>
<li>Although LexA has been reported to interact with the dinJ promoter ( 31 ) , our results extend this observation by demonstrating that LexA represses the dinJ operon in vivo ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24898247" target="_blank">24898247</a>]<br></li>
<li>Tn fact , transcription of the sulA fragment in the presence of LexA ( purified by and obtained from J. Little ) , is reduced to less than one-tenth the level found in the absence of LexA ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>The physiological significance of the LexA effect occuring only in the absence of Pz ( Fig. 5 ) is not immediately apparent but , nevertheless it suggests that SOS induction of the operon does not occur in vivo unless P ~ is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6206376" target="_blank">6206376</a>]<br></li>
<li>The degree of repression suggests that there is very little transcription of the sulA operon in the presence of repressing levels of LexA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317868" target="_blank">6317868</a>]<br></li>
<li>The affinity of LexA for the LexA boxes located immediately upstream of dinJ , ydjM , yigN and polB ( used as the positive control ) was analysed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760155" target="_blank">10760155</a>]<br></li>
<li>Fourth , the LexA site at the ptrA promoter is located downstream of the known 70-dependent transcription start site and upstream of a putative 54-dependent transcription start site ( Fig. 7A ; Claverie-Martin et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16264194" target="_blank">16264194</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>